

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":1,"applicationTime":2159,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/4/397/3815695","@type":"ScholarlyArticle","name":"Heap: a highly sensitive and accurate SNP detection tool for low-coverage high-throughput sequencing data","datePublished":"2017-05-11","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2017-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx012","keywords":["single nucleotide polymorphism (SNP)","next-generation sequencing (NGS)","restriction-site associated DNA sequencing (RAD-seq)","genome-wide association studies (GWAS)","genomic prediction (GP)"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/4/397/3815695","author":[{"name":"Kobayashi, Masaaki","affiliation":"1 Bioinformatics Laboratory, Department of Life Sciences, School of Agriculture, Meiji University, Kanagawa 214-8571, Japan","@type":"Person"},{"name":"Ohyanagi, Hajime","affiliation":"1 Bioinformatics Laboratory, Department of Life Sciences, School of Agriculture, Meiji University, Kanagawa 214-8571, Japan   2 King Abdullah University of Science and Technology (KAUST), Computational Bioscience Research Center (CBRC), Thuwal 23955-6900, Kingdom of Saudi Arabia","@type":"Person"},{"name":"Takanashi, Hideki","affiliation":"3 Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo 113-8657, Japan","@type":"Person"},{"name":"Asano, Satomi","affiliation":"1 Bioinformatics Laboratory, Department of Life Sciences, School of Agriculture, Meiji University, Kanagawa 214-8571, Japan","@type":"Person"},{"name":"Kudo, Toru","affiliation":"1 Bioinformatics Laboratory, Department of Life Sciences, School of Agriculture, Meiji University, Kanagawa 214-8571, Japan","@type":"Person"},{"name":"Kajiya-Kanegae, Hiromi","affiliation":"3 Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo 113-8657, Japan","@type":"Person"},{"name":"Nagano, Atsushi J.","affiliation":"4 Faculty of Agriculture, Ryukoku University, Shiga 520-2194, Japan   5 PRESTO, Japan Science and Technology Agency, Japan   6 Center for Ecological Research, Kyoto University, Shiga 520-2113, Japan","@type":"Person"},{"name":"Tainaka, Hitoshi","affiliation":"3 Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo 113-8657, Japan","@type":"Person"},{"name":"Tokunaga, Tsuyoshi","affiliation":"7 EARTHNOTE Co. Ltd., Nago, Okinawa 905-1152, Japan","@type":"Person"},{"name":"Sazuka, Takashi","affiliation":"8 Bioscience and Biotechnology Center, Nagoya University, Aichi 464-8601, Japan","@type":"Person"},{"name":"Iwata, Hiroyoshi","affiliation":"3 Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo 113-8657, Japan","@type":"Person"},{"name":"Tsutsumi, Nobuhiro","affiliation":"3 Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo 113-8657, Japan","@type":"Person"},{"name":"Yano, Kentaro","affiliation":"1 Bioinformatics Laboratory, Department of Life Sciences, School of Agriculture, Meiji University, Kanagawa 214-8571, Japan","@type":"Person"}],"description":"Abstract.  Recent availability of large-scale genomic resources enables us to conduct so called genome-wide association studies (GWAS) and genomic prediction (G","pageStart":"397","pageEnd":"405","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/4/1/m_dnares_24_4cover.png?Expires=2147483647&Signature=Bn68nrHzMnl3Hf0TBxDUSpIiE3ALYvws4lUuOsXgZQeZw~qHN97KSJf53QRVRMw9UIEwsJhU~oIqlbc~bNDrFxzLTkzmyUjnAZ7sS1Ope7v8mL4P5X8GIjyJTBx96xx3yzaFdQ9im8bglRhXw7RCd3PHFDaTYDb24rkCGDLkkLUBVi~BntzWxZrnH-LI5Yr~cYB3mJnIO~ISuJYAH~ZtK9rwdBnTT~rg4DIDb~I6Y2wsrkFjsIdVm-sIz91ZlsrHMWugLRT8lXkXDxrnR3~Cwmkt8-yfu-WWcXNP8U4bmwXopL-J4wp6s4jITl~IopCYMRAqzqwzkZxKkX6zWcYGMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Heap: a highly sensitive and accurate SNP detection tool for low-coverage high-throughput sequencing data","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/4/1/m_dnares_24_4cover.png?Expires=2147483647&Signature=Bn68nrHzMnl3Hf0TBxDUSpIiE3ALYvws4lUuOsXgZQeZw~qHN97KSJf53QRVRMw9UIEwsJhU~oIqlbc~bNDrFxzLTkzmyUjnAZ7sS1Ope7v8mL4P5X8GIjyJTBx96xx3yzaFdQ9im8bglRhXw7RCd3PHFDaTYDb24rkCGDLkkLUBVi~BntzWxZrnH-LI5Yr~cYB3mJnIO~ISuJYAH~ZtK9rwdBnTT~rg4DIDb~I6Y2wsrkFjsIdVm-sIz91ZlsrHMWugLRT8lXkXDxrnR3~Cwmkt8-yfu-WWcXNP8U4bmwXopL-J4wp6s4jITl~IopCYMRAqzqwzkZxKkX6zWcYGMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Heap: a highly sensitive and accurate SNP detection tool for low-coverage high-throughput sequencing data" />
    <meta property="og:description" content="Abstract.  Recent availability of large-scale genomic resources enables us to conduct so called genome-wide association studies (GWAS) and genomic prediction (G" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/4/397/3815695" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-05-11" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Kobayashi, Masaaki" /><meta name="citation_author_institution" content="Bioinformatics Laboratory, Department of Life Sciences, School of Agriculture, Meiji University, Kanagawa 214-8571, Japan" /><meta name="citation_author" content="Ohyanagi, Hajime" /><meta name="citation_author_institution" content="Bioinformatics Laboratory, Department of Life Sciences, School of Agriculture, Meiji University, Kanagawa 214-8571, Japan" /><meta name="citation_author_institution" content="King Abdullah University of Science and Technology (KAUST), Computational Bioscience Research Center (CBRC), Thuwal 23955-6900, Kingdom of Saudi Arabia" /><meta name="citation_author" content="Takanashi, Hideki" /><meta name="citation_author_institution" content="Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo 113-8657, Japan" /><meta name="citation_author" content="Asano, Satomi" /><meta name="citation_author_institution" content="Bioinformatics Laboratory, Department of Life Sciences, School of Agriculture, Meiji University, Kanagawa 214-8571, Japan" /><meta name="citation_author" content="Kudo, Toru" /><meta name="citation_author_institution" content="Bioinformatics Laboratory, Department of Life Sciences, School of Agriculture, Meiji University, Kanagawa 214-8571, Japan" /><meta name="citation_author" content="Kajiya-Kanegae, Hiromi" /><meta name="citation_author_institution" content="Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo 113-8657, Japan" /><meta name="citation_author" content="Nagano, Atsushi J." /><meta name="citation_author_institution" content="Faculty of Agriculture, Ryukoku University, Shiga 520-2194, Japan" /><meta name="citation_author_institution" content="PRESTO, Japan Science and Technology Agency, Japan" /><meta name="citation_author_institution" content="Center for Ecological Research, Kyoto University, Shiga 520-2113, Japan" /><meta name="citation_author" content="Tainaka, Hitoshi" /><meta name="citation_author_institution" content="Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo 113-8657, Japan" /><meta name="citation_author" content="Tokunaga, Tsuyoshi" /><meta name="citation_author_institution" content="EARTHNOTE Co. Ltd., Nago, Okinawa 905-1152, Japan" /><meta name="citation_author" content="Sazuka, Takashi" /><meta name="citation_author_institution" content="Bioscience and Biotechnology Center, Nagoya University, Aichi 464-8601, Japan" /><meta name="citation_author" content="Iwata, Hiroyoshi" /><meta name="citation_author_institution" content="Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo 113-8657, Japan" /><meta name="citation_author" content="Tsutsumi, Nobuhiro" /><meta name="citation_author_institution" content="Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo 113-8657, Japan" /><meta name="citation_author" content="Yano, Kentaro" /><meta name="citation_author_institution" content="Bioinformatics Laboratory, Department of Life Sciences, School of Agriculture, Meiji University, Kanagawa 214-8571, Japan" /><meta name="citation_title" content="Heap: a highly sensitive and accurate SNP detection tool for low-coverage high-throughput sequencing data" /><meta name="citation_firstpage" content="397" /><meta name="citation_lastpage" content="405" /><meta name="citation_doi" content="10.1093/dnares/dsx012" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2017/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Genomic prediction in maize breeding populations with genotyping-by-sequencing;  citation_journal_title=G3 (Bethesda);  citation_year=2013;  citation_volume=3;  citation_pages=1903-26; " /><meta name="citation_reference" content="citation_title=Population genomic and genome-wide association studies of agroclimatic traits in sorghum;  citation_journal_title=Proc. Natl. Acad. Sci;  citation_year=2013;  citation_volume=110;  citation_pages=453-8; " /><meta name="citation_reference" content="citation_title=Genotyping by sequencing for genomic prediction in a soybean breeding population;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=740" /><meta name="citation_reference" content="citation_title=Identification of loci governing eight agronomic traits using a GBS-GWAS approach and validation by QTL mapping in soya bean;  citation_journal_title=Plant Biotechnol. J;  citation_year=2015;  citation_volume=13;  citation_pages=211-21; " /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; " /><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=357-9; " /><meta name="citation_reference" content="citation_title=The sequence alignment/map format and SAMtools;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-79; " /><meta name="citation_reference" content="citation_title=A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2987-93; " /><meta name="citation_reference" content="citation_title=The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data;  citation_journal_title=Genome Res;  citation_year=2010;  citation_volume=20;  citation_pages=1297-1303; " /><meta name="citation_reference" content="citation_title=A framework for variation discovery and genotyping using next-generation DNA sequencing data;  citation_journal_title=Nat. Genet;  citation_year=2011;  citation_volume=43;  citation_pages=491-8; " /><meta name="citation_reference" content="citation_publisher=John Wiley &amp; Sons, Inc, Hoboken, NJ, USA;  citation_title=From FastQ Data to High-Confidence Variant Calls: The Genome Analysis Toolkit Best Practices Pipeline. Current Protocols in Bioinformatics;  citation_year=2013; " /><meta name="citation_reference" content="citation_title=Assessing single nucleotide variant detection and genotype calling on whole-genome sequenced individuals;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=1707-13; " /><meta name="citation_reference" content="citation_title=Rapid SNP Discovery and Genetic Mapping Using Sequenced RAD Markers;  citation_journal_title=PLoS One;  citation_year=2008;  citation_volume=3;  citation_pages=e3376" /><meta name="citation_reference" content="citation_title=SNP discovery and genotyping using restriction-site-associated DNA sequencing in chickens;  citation_journal_title=Anim. Genet;  citation_year=2015;  citation_volume=46;  citation_pages=216-9; " /><meta name="citation_reference" content="citation_title=Stacks: building and genotyping loci de novo from short-read sequences;  citation_journal_title=G3 Genes Genomes Genetics;  citation_year=2011;  citation_volume=1;  citation_pages=171-82; " /><meta name="citation_reference" content="citation_title=Stacks: an analysis tool set for population genomics;  citation_journal_title=Mol. Ecol;  citation_year=2013;  citation_volume=22;  citation_pages=3124-40; " /><meta name="citation_reference" content="citation_title=Plant omics data center: an integrated web repository for interspecies gene expression networks with NLP-based curation;  citation_journal_title=Plant Cell Physiol;  citation_year=2015;  citation_volume=56;  citation_pages=e9" /><meta name="citation_reference" content="citation_title=Phytozome: a comparative platform for green plant genomics;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=D1178-86; " /><meta name="citation_reference" content="citation_title=Improvement of the Oryza sativa Nipponbare reference genome using next generation sequence and optical map data;  citation_journal_title=Rice;  citation_year=2013;  citation_volume=6;  citation_pages=4" /><meta name="citation_reference" content="citation_title=Rice annotation project database (RAP-DB): an integrative and interactive database for rice genomics;  citation_journal_title=Plant Cell Physiol;  citation_year=2013;  citation_volume=54;  citation_pages=e6" /><meta name="citation_reference" content="citation_title=The Sorghum bicolor genome and the diversification of grasses;  citation_journal_title=Nature;  citation_year=2009;  citation_volume=457;  citation_pages=551-6; " /><meta name="citation_reference" content="citation_title=The draft genome of Hop (Humulus lupulus), an essence for brewing;  citation_journal_title=Plant Cell Physiol;  citation_year=2015;  citation_volume=56;  citation_pages=428-41; " /><meta name="citation_reference" content="citation_title=The Vigna Genome Server, ‘VigGS’: A genomic knowledge base of the genus Vigna based on high-quality, annotated genome sequence of the Azuki Bean, Vigna angularis (Willd.) Ohwi &amp; Ohashi;  citation_journal_title=Plant Cell Physiol;  citation_year=2016;  citation_volume=57;  citation_pages=e2" /><meta name="citation_reference" content="citation_title=MTGD: The Medicago truncatula Genome Database;  citation_journal_title=Plant Cell Physiol;  citation_year=2015;  citation_volume=56;  citation_pages=e1" /><meta name="citation_reference" content="citation_title=OryzaGenome: genome diversity database of wild Oryza species;  citation_journal_title=Plant Cell Physiol;  citation_year=2016;  citation_volume=57;  citation_pages=e1" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/4/397/22893140/dsx012.pdf" /><meta name="description" content="Abstract.  Recent availability of large-scale genomic resources enables us to conduct so called genome-wide association studies (GWAS) and genomic prediction (G" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/4/397/3815695" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/4/397/3815695" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/4/397/3815695");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx012';
    </script>
<title>
	Heap: a highly sensitive and accurate SNP detection tool for low-coverage high-throughput sequencing data | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./3815695" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bc1SPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLb2JheWFzaGksIE1hc2Fha2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW9pbmZvcm1hdGljcyBMYWJvcmF0b3J5LCBEZXBhcnRtZW50IG9mIExpZmUgU2NpZW5jZXMsIFNjaG9vbCBvZiBBZ3JpY3VsdHVyZSwgTWVpamkgVW5pdmVyc2l0eSwgS2FuYWdhd2EgMjE0LTg1NzEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT2h5YW5hZ2ksIEhhamltZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb2luZm9ybWF0aWNzIExhYm9yYXRvcnksIERlcGFydG1lbnQgb2YgTGlmZSBTY2llbmNlcywgU2Nob29sIG9mIEFncmljdWx0dXJlLCBNZWlqaSBVbml2ZXJzaXR5LCBLYW5hZ2F3YSAyMTQtODU3MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLaW5nIEFiZHVsbGFoIFVuaXZlcnNpdHkgb2YgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSAoS0FVU1QpLCBDb21wdXRhdGlvbmFsIEJpb3NjaWVuY2UgUmVzZWFyY2ggQ2VudGVyIChDQlJDKSwgVGh1d2FsIDIzOTU1LTY5MDAsIEtpbmdkb20gb2YgU2F1ZGkgQXJhYmlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFrYW5hc2hpLCBIaWRla2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgQWdyaWN1bHR1cmFsIGFuZCBMaWZlIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgVG9reW8gMTEzLTg2NTcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQXNhbm8sIFNhdG9taSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb2luZm9ybWF0aWNzIExhYm9yYXRvcnksIERlcGFydG1lbnQgb2YgTGlmZSBTY2llbmNlcywgU2Nob29sIG9mIEFncmljdWx0dXJlLCBNZWlqaSBVbml2ZXJzaXR5LCBLYW5hZ2F3YSAyMTQtODU3MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLdWRvLCBUb3J1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvaW5mb3JtYXRpY3MgTGFib3JhdG9yeSwgRGVwYXJ0bWVudCBvZiBMaWZlIFNjaWVuY2VzLCBTY2hvb2wgb2YgQWdyaWN1bHR1cmUsIE1laWppIFVuaXZlcnNpdHksIEthbmFnYXdhIDIxNC04NTcxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikthaml5YS1LYW5lZ2FlLCBIaXJvbWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgQWdyaWN1bHR1cmFsIGFuZCBMaWZlIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgVG9reW8gMTEzLTg2NTcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFnYW5vLCBBdHN1c2hpIEouIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRmFjdWx0eSBvZiBBZ3JpY3VsdHVyZSwgUnl1a29rdSBVbml2ZXJzaXR5LCBTaGlnYSA1MjAtMjE5NCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJQUkVTVE8sIEphcGFuIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3kgQWdlbmN5LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNlbnRlciBmb3IgRWNvbG9naWNhbCBSZXNlYXJjaCwgS3lvdG8gVW5pdmVyc2l0eSwgU2hpZ2EgNTIwLTIxMTMsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFpbmFrYSwgSGl0b3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBBZ3JpY3VsdHVyYWwgYW5kIExpZmUgU2NpZW5jZXMsIFRoZSBVbml2ZXJzaXR5IG9mIFRva3lvLCBUb2t5byAxMTMtODY1NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUb2t1bmFnYSwgVHN1eW9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJFQVJUSE5PVEUgQ28uIEx0ZC4sIE5hZ28sIE9raW5hd2EgOTA1LTExNTIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2F6dWthLCBUYWthc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvc2NpZW5jZSBhbmQgQmlvdGVjaG5vbG9neSBDZW50ZXIsIE5hZ295YSBVbml2ZXJzaXR5LCBBaWNoaSA0NjQtODYwMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJd2F0YSwgSGlyb3lvc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEFncmljdWx0dXJhbCBhbmQgTGlmZSBTY2llbmNlcywgVGhlIFVuaXZlcnNpdHkgb2YgVG9reW8sIFRva3lvIDExMy04NjU3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRzdXRzdW1pLCBOb2J1aGlybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBBZ3JpY3VsdHVyYWwgYW5kIExpZmUgU2NpZW5jZXMsIFRoZSBVbml2ZXJzaXR5IG9mIFRva3lvLCBUb2t5byAxMTMtODY1NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZYW5vLCBLZW50YXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvaW5mb3JtYXRpY3MgTGFib3JhdG9yeSwgRGVwYXJ0bWVudCBvZiBMaWZlIFNjaWVuY2VzLCBTY2hvb2wgb2YgQWdyaWN1bHR1cmUsIE1laWppIFVuaXZlcnNpdHksIEthbmFnYXdhIDIxNC04NTcxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iSGVhcDogYSBoaWdobHkgc2Vuc2l0aXZlIGFuZCBhY2N1cmF0ZSBTTlAgZGV0ZWN0aW9uIHRvb2wgZm9yIGxvdy1jb3ZlcmFnZSBoaWdoLXRocm91Z2hwdXQgc2VxdWVuY2luZyBkYXRhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMzk3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI0MDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3gwMTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNy8wOC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgcHJlZGljdGlvbiBpbiBtYWl6ZSBicmVlZGluZyBwb3B1bGF0aW9ucyB3aXRoIGdlbm90eXBpbmctYnktc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RzMgKEJldGhlc2RhKTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MTkwMy0yNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9wdWxhdGlvbiBnZW5vbWljIGFuZCBnZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBzdHVkaWVzIG9mIGFncm9jbGltYXRpYyB0cmFpdHMgaW4gc29yZ2h1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTQ1My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vdHlwaW5nIGJ5IHNlcXVlbmNpbmcgZm9yIGdlbm9taWMgcHJlZGljdGlvbiBpbiBhIHNveWJlYW4gYnJlZWRpbmcgcG9wdWxhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9NzQwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgbG9jaSBnb3Zlcm5pbmcgZWlnaHQgYWdyb25vbWljIHRyYWl0cyB1c2luZyBhIEdCUy1HV0FTIGFwcHJvYWNoIGFuZCB2YWxpZGF0aW9uIGJ5IFFUTCBtYXBwaW5nIGluIHNveWEgYmVhbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQmlvdGVjaG5vbC4gSjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTIxMS0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCBhbmQgYWNjdXJhdGUgc2hvcnQgcmVhZCBhbGlnbm1lbnQgd2l0aCBCdXJyb3dzLVdoZWVsZXIgdHJhbnNmb3JtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTE3NTQtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QgZ2FwcGVkLXJlYWQgYWxpZ25tZW50IHdpdGggQm93dGllIDI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MzU3LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBzZXF1ZW5jZSBhbGlnbm1lbnQvbWFwIGZvcm1hdCBhbmQgU0FNdG9vbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MjA3OC03OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzdGF0aXN0aWNhbCBmcmFtZXdvcmsgZm9yIFNOUCBjYWxsaW5nLCBtdXRhdGlvbiBkaXNjb3ZlcnksIGFzc29jaWF0aW9uIG1hcHBpbmcgYW5kIHBvcHVsYXRpb24gZ2VuZXRpY2FsIHBhcmFtZXRlciBlc3RpbWF0aW9uIGZyb20gc2VxdWVuY2luZyBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTI5ODctOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBHZW5vbWUgQW5hbHlzaXMgVG9vbGtpdDogQSBNYXBSZWR1Y2UgZnJhbWV3b3JrIGZvciBhbmFseXppbmcgbmV4dC1nZW5lcmF0aW9uIEROQSBzZXF1ZW5jaW5nIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0xMjk3LTEzMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZnJhbWV3b3JrIGZvciB2YXJpYXRpb24gZGlzY292ZXJ5IGFuZCBnZW5vdHlwaW5nIHVzaW5nIG5leHQtZ2VuZXJhdGlvbiBETkEgc2VxdWVuY2luZyBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9NDkxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1Kb2huIFdpbGV5ICZhbXA7IFNvbnMsIEluYywgSG9ib2tlbiwgTkosIFVTQTsgIGNpdGF0aW9uX3RpdGxlPUZyb20gRmFzdFEgRGF0YSB0byBIaWdoLUNvbmZpZGVuY2UgVmFyaWFudCBDYWxsczogVGhlIEdlbm9tZSBBbmFseXNpcyBUb29sa2l0IEJlc3QgUHJhY3RpY2VzIFBpcGVsaW5lLiBDdXJyZW50IFByb3RvY29scyBpbiBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzZXNzaW5nIHNpbmdsZSBudWNsZW90aWRlIHZhcmlhbnQgZGV0ZWN0aW9uIGFuZCBnZW5vdHlwZSBjYWxsaW5nIG9uIHdob2xlLWdlbm9tZSBzZXF1ZW5jZWQgaW5kaXZpZHVhbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MTcwNy0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmFwaWQgU05QIERpc2NvdmVyeSBhbmQgR2VuZXRpYyBNYXBwaW5nIFVzaW5nIFNlcXVlbmNlZCBSQUQgTWFya2VyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPWUzMzc2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U05QIGRpc2NvdmVyeSBhbmQgZ2Vub3R5cGluZyB1c2luZyByZXN0cmljdGlvbi1zaXRlLWFzc29jaWF0ZWQgRE5BIHNlcXVlbmNpbmcgaW4gY2hpY2tlbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFuaW0uIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9MjE2LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0YWNrczogYnVpbGRpbmcgYW5kIGdlbm90eXBpbmcgbG9jaSBkZSBub3ZvIGZyb20gc2hvcnQtcmVhZCBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUczIEdlbmVzIEdlbm9tZXMgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTE3MS04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhY2tzOiBhbiBhbmFseXNpcyB0b29sIHNldCBmb3IgcG9wdWxhdGlvbiBnZW5vbWljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBFY29sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MzEyNC00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGxhbnQgb21pY3MgZGF0YSBjZW50ZXI6IGFuIGludGVncmF0ZWQgd2ViIHJlcG9zaXRvcnkgZm9yIGludGVyc3BlY2llcyBnZW5lIGV4cHJlc3Npb24gbmV0d29ya3Mgd2l0aCBOTFAtYmFzZWQgY3VyYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGwgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01NjsgIGNpdGF0aW9uX3BhZ2VzPWU5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5dG96b21lOiBhIGNvbXBhcmF0aXZlIHBsYXRmb3JtIGZvciBncmVlbiBwbGFudCBnZW5vbWljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz1EMTE3OC04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1wcm92ZW1lbnQgb2YgdGhlIE9yeXphIHNhdGl2YSBOaXBwb25iYXJlIHJlZmVyZW5jZSBnZW5vbWUgdXNpbmcgbmV4dCBnZW5lcmF0aW9uIHNlcXVlbmNlIGFuZCBvcHRpY2FsIG1hcCBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1SaWNlOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz00IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmljZSBhbm5vdGF0aW9uIHByb2plY3QgZGF0YWJhc2UgKFJBUC1EQik6IGFuIGludGVncmF0aXZlIGFuZCBpbnRlcmFjdGl2ZSBkYXRhYmFzZSBmb3IgcmljZSBnZW5vbWljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTU0OyAgY2l0YXRpb25fcGFnZXM9ZTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgU29yZ2h1bSBiaWNvbG9yIGdlbm9tZSBhbmQgdGhlIGRpdmVyc2lmaWNhdGlvbiBvZiBncmFzc2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDU3OyAgY2l0YXRpb25fcGFnZXM9NTUxLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBkcmFmdCBnZW5vbWUgb2YgSG9wIChIdW11bHVzIGx1cHVsdXMpLCBhbiBlc3NlbmNlIGZvciBicmV3aW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsIFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NTY7ICBjaXRhdGlvbl9wYWdlcz00MjgtNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBWaWduYSBHZW5vbWUgU2VydmVyLCDigJhWaWdHU+KAmTogQSBnZW5vbWljIGtub3dsZWRnZSBiYXNlIG9mIHRoZSBnZW51cyBWaWduYSBiYXNlZCBvbiBoaWdoLXF1YWxpdHksIGFubm90YXRlZCBnZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIEF6dWtpIEJlYW4sIFZpZ25hIGFuZ3VsYXJpcyAoV2lsbGQuKSBPaHdpICZhbXA7IE9oYXNoaTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTU3OyAgY2l0YXRpb25fcGFnZXM9ZTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NVEdEOiBUaGUgTWVkaWNhZ28gdHJ1bmNhdHVsYSBHZW5vbWUgRGF0YWJhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGwgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01NjsgIGNpdGF0aW9uX3BhZ2VzPWUxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3J5emFHZW5vbWU6IGdlbm9tZSBkaXZlcnNpdHkgZGF0YWJhc2Ugb2Ygd2lsZCBPcnl6YSBzcGVjaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsIFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NTc7ICBjaXRhdGlvbl9wYWdlcz1lMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNC80LzM5Ny8yMjg5MzE0MC9kc3gwMTIucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFJlY2VudCBhdmFpbGFiaWxpdHkgb2YgbGFyZ2Utc2NhbGUgZ2Vub21pYyByZXNvdXJjZXMgZW5hYmxlcyB1cyB0byBjb25kdWN0IHNvIGNhbGxlZCBnZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBzdHVkaWVzIChHV0FTKSBhbmQgZ2Vub21pYyBwcmVkaWN0aW9uIChHIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zODE1Njk1JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZFxWNq5sfNVoTp1IU33I21OEVpz1" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f4%2f397%2f3815695" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f4%2f397%2f3815695" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=98073" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/4/1/m_dnares_24_4cover.png?Expires=2147483647&amp;Signature=Bn68nrHzMnl3Hf0TBxDUSpIiE3ALYvws4lUuOsXgZQeZw~qHN97KSJf53QRVRMw9UIEwsJhU~oIqlbc~bNDrFxzLTkzmyUjnAZ7sS1Ope7v8mL4P5X8GIjyJTBx96xx3yzaFdQ9im8bglRhXw7RCd3PHFDaTYDb24rkCGDLkkLUBVi~BntzWxZrnH-LI5Yr~cYB3mJnIO~ISuJYAH~ZtK9rwdBnTT~rg4DIDb~I6Y2wsrkFjsIdVm-sIz91ZlsrHMWugLRT8lXkXDxrnR3~Cwmkt8-yfu-WWcXNP8U4bmwXopL-J4wp6s4jITl~IopCYMRAqzqwzkZxKkX6zWcYGMA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/4">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="106356825">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356825">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106356827">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356827">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106356833">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356833">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106356871">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356871">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106356888">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356888">Availability of data and program</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106356890">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356890">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106356894">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356894">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106356896">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356896">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106356898">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106356898">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/4/387/3064184">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/4/407/3744590">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Heap: a highly sensitive and accurate SNP detection tool for low-coverage high-throughput sequencing data
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masaaki Kobayashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Bioinformatics Laboratory, Department of Life Sciences, School of Agriculture, Meiji University, Kanagawa 214-8571, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masaaki+Kobayashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kobayashi M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kobayashi M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Masaaki Kobayashi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hajime Ohyanagi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Bioinformatics Laboratory, Department of Life Sciences, School of Agriculture, Meiji University, Kanagawa 214-8571, Japan</div><div class="aff"><div class="label -label">2</div>King Abdullah University of Science and Technology (KAUST), Computational Bioscience Research Center (CBRC), Thuwal 23955-6900, Kingdom of Saudi Arabia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hajime+Ohyanagi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohyanagi H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohyanagi H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hajime Ohyanagi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Takanashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo 113-8657, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Takanashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takanashi H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takanashi H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hideki Takanashi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satomi Asano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Bioinformatics Laboratory, Department of Life Sciences, School of Agriculture, Meiji University, Kanagawa 214-8571, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satomi+Asano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Asano S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Asano S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Satomi Asano</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Toru Kudo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Bioinformatics Laboratory, Department of Life Sciences, School of Agriculture, Meiji University, Kanagawa 214-8571, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Toru+Kudo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kudo T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kudo T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Toru Kudo</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiromi Kajiya-Kanegae</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo 113-8657, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiromi+Kajiya-Kanegae">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kajiya-Kanegae H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kajiya-Kanegae H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hiromi Kajiya-Kanegae</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Atsushi J. Nagano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div>Faculty of Agriculture, Ryukoku University, Shiga 520-2194, Japan</div><div class="aff"><div class="label -label">5</div>PRESTO, Japan Science and Technology Agency, Japan</div><div class="aff"><div class="label -label">6</div>Center for Ecological Research, Kyoto University, Shiga 520-2113, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Atsushi+J.+Nagano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nagano A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nagano A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Atsushi J. Nagano</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hitoshi Tainaka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo 113-8657, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hitoshi+Tainaka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tainaka H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tainaka H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hitoshi Tainaka</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tsuyoshi Tokunaga</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div>EARTHNOTE Co. Ltd., Nago, Okinawa 905-1152, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tsuyoshi+Tokunaga">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tokunaga T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tokunaga T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Tsuyoshi Tokunaga</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takashi Sazuka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">8</div>Bioscience and Biotechnology Center, Nagoya University, Aichi 464-8601, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takashi+Sazuka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sazuka T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sazuka T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Takashi Sazuka</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroyoshi Iwata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo 113-8657, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroyoshi+Iwata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Iwata H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Iwata H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Hiroyoshi Iwata</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nobuhiro Tsutsumi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Graduate School of Agricultural and Life Sciences, The University of Tokyo, Tokyo 113-8657, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nobuhiro+Tsutsumi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tsutsumi N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tsutsumi N&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Nobuhiro Tsutsumi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kentaro Yano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Bioinformatics Laboratory, Department of Life Sciences, School of Agriculture, Meiji University, Kanagawa 214-8571, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx012-cor1" data-open="dsx012-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 44 934 7046. Fax. +81 44 934 7046. Email: <a href="mailto:kyano@isc.meiji.ac.jp" target="_blank">kyano@isc.meiji.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kentaro+Yano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yano K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yano K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Kentaro Yano</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx012-FM1"><div class="fn"><div class="footnote-content">Edited by Prof. Hiroyuki Toh</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 4, 1 August 2017, Pages 397–405, <a href='https://doi.org/10.1093/dnares/dsx012'>https://doi.org/10.1093/dnares/dsx012</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">11 May 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">15 June 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">20 April 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="3815695" href="/dnaresearch/article-pdf/24/4/397/22893140/dsx012.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Masaaki Kobayashi, Hajime Ohyanagi, Hideki Takanashi, Satomi Asano, Toru Kudo, Hiromi Kajiya-Kanegae, Atsushi J. Nagano, Hitoshi Tainaka, Tsuyoshi Tokunaga, Takashi Sazuka, Hiroyoshi Iwata, Nobuhiro Tsutsumi, Kentaro Yano;  Heap: a highly sensitive and accurate SNP detection tool for low-coverage high-throughput sequencing data, <em>DNA Research</em>, Volume 24, Issue 4, 1 August 2017, Pages 397–405, <a href='https://doi.org/10.1093/dnares/dsx012'>https://doi.org/10.1093/dnares/dsx012</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/3815695?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/3815695?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/3815695?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/3815695?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/3815695?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/3815695?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/3815695?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Heap%3A%20a%20highly%20sensitive%20and%20accurate%20SNP%20detection%20tool%20for%20low-coverage%20high-throughput%20sequencing%20data&amp;publicationDate=2017-05-11&amp;volumeNum=24&amp;issueNum=4&amp;author=Kobayashi%2C%20Masaaki%3B%20Ohyanagi%2C%20Hajime&amp;startPage=397&amp;endPage=405&amp;contentId=10.1093%2Fdnares%2Fdsx012&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f4%2f397%2f3815695" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=106356825 id="106356825" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Recent availability of large-scale genomic resources enables us to conduct so called genome-wide association studies (GWAS) and genomic prediction (GP) studies, particularly with next-generation sequencing (NGS) data. The effectiveness of GWAS and GP depends on not only their mathematical models, but the quality and quantity of variants employed in the analysis. In NGS single nucleotide polymorphism (SNP) calling, conventional tools ideally require more reads for higher SNP sensitivity and accuracy. In this study, we aimed to develop a tool, Heap, that enables robustly sensitive and accurate calling of SNPs, particularly with a low coverage NGS data, which must be aligned to the reference genome sequences in advance. To reduce false positive SNPs, Heap determines genotypes and calls SNPs at each site except for sites at the both ends of reads or containing a minor allele supported by only one read. Performance comparison with existing tools showed that Heap achieved the highest <em>F</em>-scores with low coverage (7X) restriction-site associated DNA sequencing reads of sorghum and rice individuals. This will facilitate cost-effective GWAS and GP studies in this NGS era. Code and documentation of Heap are freely available from <a class="link link-uri" href="https://github.com/meiji-bioinf/heap" target="_blank">https://github.com/meiji-bioinf/heap</a> (29 March 2017, date last accessed) and our web site (<a class="link link-uri" href="http://bioinf.mind.meiji.ac.jp/lab/en/tools.html" target="_blank">http://bioinf.mind.meiji.ac.jp/lab/en/tools.html</a> (29 March 2017, date last accessed)).</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;single nucleotide polymorphism (SNP)&quot;">single nucleotide polymorphism (SNP)</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;next-generation sequencing (NGS)&quot;">next-generation sequencing (NGS)</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;restriction-site associated DNA sequencing (RAD-seq)&quot;">restriction-site associated DNA sequencing (RAD-seq)</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome-wide association studies (GWAS)&quot;">genome-wide association studies (GWAS)</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genomic prediction (GP)&quot;">genomic prediction (GP)</a></div>                            <h2 scrollto-destination=106356827 id="106356827" class="section-title" >1. Introduction</h2>
<p>DNA polymorphisms including single nucleotide polymorphisms (SNPs), insertions and deletions (INDELs), among inbred lines or collected individuals have been employed to identify loci associated with traits or to predict phenotypes by genotypes. To identify genes associated with phenotypes and implement high-throughput breeding, genome-wide association studies (GWAS) and genomic prediction (GP) studies have been performed with next-generation sequencing (NGS) data<a reveal-id="dsx012-B1" data-open="dsx012-B1" class="link link-ref link-reveal xref-bibr"><sup>1–4</sup></a>. In GWAS and GP studies, SNPs are widely used as genetic markers. SNPs are determined by mapping NGS reads and consequent variant calls with conventional tools. Widely used software, such as Burrows-Wheeler Aligner (BWA) or Bowtie 2 performs alignments of NGS reads to reference genomes<a reveal-id="dsx012-B5" data-open="dsx012-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsx012-B6" data-open="dsx012-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a>, whereas SAMtools/BCFtools or Genome Analysis Toolkit (GATK) call variants based on alignments produced during the mapping steps<a reveal-id="dsx012-B7" data-open="dsx012-B7" class="link link-ref link-reveal xref-bibr"><sup>7–11</sup></a>. The accuracy of both mapping and variant calling should be key factors for the effectiveness of GWAS and GP.</p><p>SAMtools/BCFtools and GATK accurately call SNPs if provided with enough number of reads. When read coverage (depth) is 20X or more, SNPs have been called with sufficient sensitivity in human genome resequencing<a reveal-id="dsx012-B12" data-open="dsx012-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a>. It is also reported that SNP calling becomes difficult under low read coverage (7X or lower)<a reveal-id="dsx012-B12" data-open="dsx012-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a>. In large-scale GWAS or GP studies, the number of NGS reads in each individual tends to be small in order to genotype as many individuals as possible with a limited budget. In these cases, methodologies that accurately detect large number of SNPs based on a relatively low number of reads are paramount for an efficient implementation of GWAS and GP studies.</p><p>Restriction-site associated DNA sequencing, or genotype-by-sequencing (RAD-seq/GBS), is an economical strategy to identify genome wide polymorphisms<a reveal-id="dsx012-B13" data-open="dsx012-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a>. While whole genome shotgun (WGS) sequencing methods provide sequencing information from the whole genome, the RAD-seq/GBS methods confine sequencing regions to the fragments neighbouring particular restriction sites. To be precise, in RAD-seq/GBS, regions neighbouring particular restriction sites are sequenced in combination with high-throughput sequencing methods and with subsequent mapping and variant calling steps on reference genome sequences to identify genomic polymorphisms. For example, RAD-seq/GBS reads sets of <em>Hin</em>dIII which covers only 0.24% of the chicken genome on average (i.e. coverage = 0.0024x) could show 3× or more depth of coverage on the RAD fragments<a reveal-id="dsx012-B14" data-open="dsx012-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a>. Compared with WGS sequencing methods, the RAD-seq/GBS strategy under the same sequencing cost generally assures a relatively higher coverage at each sequencing region, even when the total number of reads is lower in each sample (e.g. inbred lines, individuals, or libraries) with the RAD-seq/GBS strategy.</p><p>Stacks is a specialized tool for identifying SNPs from RAD-seq/GBS reads<a reveal-id="dsx012-B15" data-open="dsx012-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsx012-B16" data-open="dsx012-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a>. In the Stacks pipeline, identical reads, which are called ‘stack’, are collected from each sample. Then, SNPs among samples are identified by comparing among stacks obtained from multiple samples. Stacks are capable of calling many SNPs compared to SAMtools/BCFtools and GATK with low coverage RAD-seq/GBS reads. However, Stacks can call false positive SNPs more frequently than other methods (as is shown in this work).</p><p>In this study, we developed a new tool, Heap, that identifies numerous SNPs with high accuracy from RAD-seq/GBS or WGS sequencing reads. Heap is capable of detecting SNPs with high sensitivity, which is the ratio of correctly identified SNPs (True positives) to all the existing SNPs (True positives + False negatives), and high positive predictive values (PPVs), which is the ratio of True positives to all the identified SNPs (True positives + False positives), from even reads with a low coverage aligned to the reference genome sequences. To confirm the performance of Heap, we identified SNPs from RAD-seq/GBS reads and calculated an accuracy index <em>F</em>-score, which is the harmonic mean between sensitivity and PPV, of SNP calling in 17 inbred sorghum lines and 4 inbred rice lines.</p>                            <h2 scrollto-destination=106356833 id="106356833" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=106356834 id="106356834" class="section-title" >2.1. Software details</h3>
                            <h4 scrollto-destination=106356835 id="106356835" class="section-title" >2.1.1. Algorithms of Heap</h4>
<p>We designed Heap to identify SNPs from short read sequences of diploid species. In Heap analysis, short read sequences must be aligned to reference genome sequences in advance, and information on aligned reads, which is stored in either Sequence Alignment/MAP (SAM) format files or the binary version of SAM (BAM) format files<a reveal-id="dsx012-B7" data-open="dsx012-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a>, must be employed (<a reveal-id="dsx012-F1" data-open="dsx012-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). </p>                        <a id="106356837" scrollto-destination="106356837"></a>
<div id="dsx012-F1" data-id="dsx012-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/m_dsx012f1.png?Expires=2147483647&Signature=e1V-eee9nYu~ENMkS-dtW8WLGv~Ep4WLh6CDwat4AdIemhKekiwsZ2~M42A0CZVf0IQXr63sb6ak76-dzMHlZEJDSVTTmg6ruN80HA8PNeJIs6oXYudjUQqRiHIhV-GgqcXrTRQ3tYBneR9PKmshiAkPKRLuZN~JrtvVnIFdHcefyd54EXXDKhW87QZBMWNojqtZ7S15VgymeDlExU0Mbg43SHlG7IqpMIqMoEyIydl~h14cDvmxLtgAnxuKPX11IMLt5SjwelwLIUFep2DMazQ-2rwQUdUY9CG84ksvlJly5NGtlowkjATwGPI-JvTXMWGyNCLwf5H9op-SdSrDAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Workflow of Heap algorithm." path-from-xml="dsx012f1.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356837/dsx012f1.png" path-from-xml="dsx012f1.png" target="_blank">View large</a><a section="106356837" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/dsx012f1.png?Expires=2147483647&Signature=r5ubapGE1MoZkmiyCO1AzUwdqXNlu808YAWjsYtNl1WncHCz8yqN6B7AfYi2gmGJNNtBe2b216Co5eRSIB39xwTEgP0PlqoJrNLpv18Fwb51QhWuo4rLkxl83xxsDtDGEpB2COscT3l-cymUh1Nf58p6hRD0WClyRjysbX2fADHAFzNTnHjzmh0UZ6CqKa-8cjOSTLE8zI8MmMWiFG89WMQKLpVeW9wBPeQSA7Yyqva6ftW2yJgL5Ijkoh6LBKHR6BSVcLBYF2ZhJX6SNL8BT~5ZA8wHXfwGevDuyJ6IZSdf07NSgGKOO9x2JihRNUd7u9QPcDUVFRGm7~4Hfm-GJw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106356837&ar=3815695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx012f1.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Workflow of Heap algorithm.</p></div></div><div content-id="dsx012-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/m_dsx012f1.png?Expires=2147483647&Signature=e1V-eee9nYu~ENMkS-dtW8WLGv~Ep4WLh6CDwat4AdIemhKekiwsZ2~M42A0CZVf0IQXr63sb6ak76-dzMHlZEJDSVTTmg6ruN80HA8PNeJIs6oXYudjUQqRiHIhV-GgqcXrTRQ3tYBneR9PKmshiAkPKRLuZN~JrtvVnIFdHcefyd54EXXDKhW87QZBMWNojqtZ7S15VgymeDlExU0Mbg43SHlG7IqpMIqMoEyIydl~h14cDvmxLtgAnxuKPX11IMLt5SjwelwLIUFep2DMazQ-2rwQUdUY9CG84ksvlJly5NGtlowkjATwGPI-JvTXMWGyNCLwf5H9op-SdSrDAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Workflow of Heap algorithm." path-from-xml="dsx012f1.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356837/dsx012f1.png" path-from-xml="dsx012f1.png" target="_blank">View large</a><a section="106356837" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/dsx012f1.png?Expires=2147483647&Signature=r5ubapGE1MoZkmiyCO1AzUwdqXNlu808YAWjsYtNl1WncHCz8yqN6B7AfYi2gmGJNNtBe2b216Co5eRSIB39xwTEgP0PlqoJrNLpv18Fwb51QhWuo4rLkxl83xxsDtDGEpB2COscT3l-cymUh1Nf58p6hRD0WClyRjysbX2fADHAFzNTnHjzmh0UZ6CqKa-8cjOSTLE8zI8MmMWiFG89WMQKLpVeW9wBPeQSA7Yyqva6ftW2yJgL5Ijkoh6LBKHR6BSVcLBYF2ZhJX6SNL8BT~5ZA8wHXfwGevDuyJ6IZSdf07NSgGKOO9x2JihRNUd7u9QPcDUVFRGm7~4Hfm-GJw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106356837&ar=3815695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx012f1.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Workflow of Heap algorithm.</p></div></div><p>After importing information on read alignments from a SAM or BAM file, Heap performs read filtering to obtain high quality reads (<a reveal-id="dsx012-F1" data-open="dsx012-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>). By default setting of Heap, reads with a phred scaled mapping quality score (MAPQ) below 20 are removed. Bases with a phred scaled quality score in base calling below 13 are also eliminated from the search scope of valid SNP sites. Heap also trims both ends of each read before mining SNPs, because it is empirically observed that these regions contain many base calling errors. By default setting, both ends with <em>n </em>=<em> </em>2bp are not counted for SNP searches. Moreover, flanking regions within <em>i </em>=<em> </em>5bp of each INDEL site are also removed in SNP calling. To determine the default setting of the <em>n</em> and <em>i</em> values, we examined the empirical distribution of <em>F</em>-scores under multiple conditions (for <em>F</em>-scores, see Performance comparison among SNP calling tools with RAD-seq reads in sorghum in results and discussions) (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/supplementary_fig_s1_dsx012.ppt?Expires=2147483647&Signature=JVLIv080j1q1nKNRXVXwhmeXeGLHiCUjfQsSpTJzvYxfwyoG~nxiKs47pvg6BUyTlW1-nQoJFDeVUHQRfL3zc5Rzjihm0lqCEk-8pz8QmgpJR57NPmQ3-L1JwVvwc2y2PRl0do9cQOxxj9XN57tTh91WFjIPktzec0YyafzMuFKbii-v9Xpg91b-aZpH3GuOe9VUipRkPhPQ37In1UT864YgBD-0PsxPCkUNVIxz2RxIZBeiFOk7~dwvAn9Hjh7grrxIUSGgSDanLR5KnyyUCbXYZjbiCY271VaVRyuREEtSNvcG9477cOoweQBI5ZDrb~pFlOS1fHM1xcVqUFK4~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). The highest <em>F</em>-score was obtained under around <em>n </em>=<em> </em>2–3, while <em>F</em>-score reached a plateau under around <em>i </em>=<em> </em>4–5. Therefore, we determined that <em>n </em>=<em> </em>2 and <em>i </em>=<em> </em>5 as default values.</p><p>Next, Heap determines each sample's genotype in every site that passes quality filtering (<a reveal-id="dsx012-F1" data-open="dsx012-F1" class="link link-reveal link-table xref-fig">Fig. 1C</a>). On each nucleotide site, the allele frequency is calculated from the number of nucleotide bases (A, T, C or G) aligned on it. Heap ignores any sites with three or more allele variants, since any allele possibilities above 2 on diploid organisms are likely due to sequencing errors. To determine the zygosity of each allele, a binomial test with allele frequency is performed (the null hypothesis H<sub>0</sub>: the allele is heterozygous). When the <em>P</em> value of the binomial test is &lt;0.05, the null hypothesis is rejected thus the zygosity on the site is determined as homozygous. Conversely, when the P value is ≥0.05 and the minor allele is supported by two or more NGS reads, the zygosity is determined as heterozygous. When neither is the case, the genotype is included in subsequent analyses as missing genotype (./.). This genotyping step is repeated for all samples.</p><p>Heap then performs SNP calling by comparing the genotypes between the sample (e.g. an inbred line) and the reference genome (<a reveal-id="dsx012-F1" data-open="dsx012-F1" class="link link-reveal link-table xref-fig">Fig. 1D</a>). The information on all SNPs between the sample and the reference genome is stored in a Variant Call Format (VCF) file. In a VCF file, reference allele, first alternative allele, second alternative allele, and missing is presented as ‘0’, ‘1’, ‘2’, and ‘.’ in the genotype field, respectively. Finally, to determine SNPs among all samples, the VCF files for all samples are merged in a single VCF file by BCFtools (<a reveal-id="dsx012-F1" data-open="dsx012-F1" class="link link-reveal link-table xref-fig">Fig. 1E</a>).</p>                            <h4 scrollto-destination=106356841 id="106356841" class="section-title" >2.1.2. Software implementation and requirements</h4>
<p>Heap is implemented in C ++ programming languages and is able to be installed on a Linux/UNIX-like operating system including, but not limited to, CentOS, Ubuntu and Mac OS X. The following additional software and libraries are required: Boost C ++ libraries, BCFtools (≥ ver. 1.2), HTSlib (≥ ver. 1.2), and g ++ compiler (≥ ver. 4.1.2). All steps of Heap algorithm can be invoked with a single command line. For the usage and options of Heap, see the README text file. Heap is released under the GNU General Public License Version 3 (see the COPYING text file).</p>                            <h3 scrollto-destination=106356843 id="106356843" class="section-title" >2.2. Wet lab procedures</h3>
                            <h4 scrollto-destination=106356844 id="106356844" class="section-title" >2.2.1. DNA extraction from sorghum and rice plants</h4>
<p>DNA samples were extracted from 17 inbred lines of <em>Sorghum bicolor</em> (GULUM ABIAD, A-6129, AGIRA, LR 399, CRIOLLO CABEZA APRETADA, B2AR3043, RTx430, B3Tx2817, ZIRA-EL-SABI, ITALIAN, CAPRICORN, SOR 1, 15065, SAP-155, RCV, SC 56, and SIL-05) and 4 inbred lines of <em>Oryza sativa</em> (Omachi, Yamada Nishiki, Hitomebore, and Kameji). Methods of DNA extraction are described in <span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/supplementary_methods_dsx012.doc?Expires=2147483647&Signature=ry9MHxyf0iwBvLnDJBhlaTtzY7tamI8iOj-3AG40Qi4~~VtNszR8UcO4qPkHJC6AC2FcOEDaXKJ~09gz1bUkOs-VxX8WxpcoDn1gwuAP3vJLZU-8HtYFRX48BEBbCcHV8HjApl4Bxwg7uE9sJ35p0REzRfK-iOYhDO09BGE~VuoHv749sza0UWbMPyXwWgPxyTqf4eB7n3tHa3APWhgaQpIUWD2hHgERaUWttTrAdHHuYausFequfsXiMYwNCKclFJGSLbu3Bn~Yjqg756v-ZYdWKgXNexIy6DerrsuGHrWIOnpS-3gSMOfAd~bl9vGhPEvzGZusrhUA8k6Af181hg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Methods</a></span>.</p>                            <h4 scrollto-destination=106356846 id="106356846" class="section-title" >2.2.2. RAD-seq of sorghum</h4>
<p>We performed RAD-seq in the 17 inbred sorghum lines using an Illumina HiSeq 2000 platform with 100 cycles with single end layout. Detailed methods for DNA libraries construction and RAD-seq are described in <span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/supplementary_methods_dsx012.doc?Expires=2147483647&Signature=ry9MHxyf0iwBvLnDJBhlaTtzY7tamI8iOj-3AG40Qi4~~VtNszR8UcO4qPkHJC6AC2FcOEDaXKJ~09gz1bUkOs-VxX8WxpcoDn1gwuAP3vJLZU-8HtYFRX48BEBbCcHV8HjApl4Bxwg7uE9sJ35p0REzRfK-iOYhDO09BGE~VuoHv749sza0UWbMPyXwWgPxyTqf4eB7n3tHa3APWhgaQpIUWD2hHgERaUWttTrAdHHuYausFequfsXiMYwNCKclFJGSLbu3Bn~Yjqg756v-ZYdWKgXNexIy6DerrsuGHrWIOnpS-3gSMOfAd~bl9vGhPEvzGZusrhUA8k6Af181hg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Methods</a></span>. These RAD-seq data have been submitted to DDBJ Sequence Read Archive (DRA) (DRR045054-DRR045070).</p>                            <h4 scrollto-destination=106356848 id="106356848" class="section-title" >2.2.3. WGS sequencing of sorghum</h4>
<p>WGS sequencing was performed in the 17 inbred sorghum lines using an Illumina HiSeq 2000 platform with 100 cycles with paired end layout. Detailed methods for DNA libraries construction and WGS sequencing are described in <span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/supplementary_methods_dsx012.doc?Expires=2147483647&Signature=ry9MHxyf0iwBvLnDJBhlaTtzY7tamI8iOj-3AG40Qi4~~VtNszR8UcO4qPkHJC6AC2FcOEDaXKJ~09gz1bUkOs-VxX8WxpcoDn1gwuAP3vJLZU-8HtYFRX48BEBbCcHV8HjApl4Bxwg7uE9sJ35p0REzRfK-iOYhDO09BGE~VuoHv749sza0UWbMPyXwWgPxyTqf4eB7n3tHa3APWhgaQpIUWD2hHgERaUWttTrAdHHuYausFequfsXiMYwNCKclFJGSLbu3Bn~Yjqg756v-ZYdWKgXNexIy6DerrsuGHrWIOnpS-3gSMOfAd~bl9vGhPEvzGZusrhUA8k6Af181hg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Methods</a></span>. These WGS sequencing data have been submitted to DRA (DRR045071-DRR045087).</p>                            <h4 scrollto-destination=106356850 id="106356850" class="section-title" >2.2.4. RAD-seq of rice</h4>
<p>We performed RAD-seq in the 4 inbred rice lines using an Illumina HiSeq 2000 platform with 50 cycles with single end layout. Detailed methods for DNA libraries construction and RAD-seq are described in <span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/supplementary_methods_dsx012.doc?Expires=2147483647&Signature=ry9MHxyf0iwBvLnDJBhlaTtzY7tamI8iOj-3AG40Qi4~~VtNszR8UcO4qPkHJC6AC2FcOEDaXKJ~09gz1bUkOs-VxX8WxpcoDn1gwuAP3vJLZU-8HtYFRX48BEBbCcHV8HjApl4Bxwg7uE9sJ35p0REzRfK-iOYhDO09BGE~VuoHv749sza0UWbMPyXwWgPxyTqf4eB7n3tHa3APWhgaQpIUWD2hHgERaUWttTrAdHHuYausFequfsXiMYwNCKclFJGSLbu3Bn~Yjqg756v-ZYdWKgXNexIy6DerrsuGHrWIOnpS-3gSMOfAd~bl9vGhPEvzGZusrhUA8k6Af181hg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Methods</a></span>. These RAD-seq data have been submitted to DRA (DRR045088-DRR045091).</p>                            <h4 scrollto-destination=106356852 id="106356852" class="section-title" >2.2.5. WGS sequencing data of rice</h4>
<p>Besides the sequencing data mentioned above, WGS sequencing data of the 4 inbred rice lines (DRR000719, DRR000720, DRR003652, DRR003655, DRR004451, DRR004452, DRR004453, DRR003648, DRR003649, and DRR003658) were downloaded from DRA. Each of them corresponds to each of 4 inbred lines mentioned in RAD-seq of rice.</p>                            <h3 scrollto-destination=106356854 id="106356854" class="section-title" >2.3. Benchmarking</h3>
                            <h4 scrollto-destination=106356855 id="106356855" class="section-title" >2.3.1. Preprocessing of reads</h4>
<p>In order to correctly map the WGS sequencing reads and the RAD-seq reads to the reference genome sequences, we performed adapter trimming and quality filtering as described previously<a reveal-id="dsx012-B17" data-open="dsx012-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a>. After quality control by FastQC (<a class="link link-uri" href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc/" target="_blank">http://www.bioinformatics.babraham.ac.uk/projects/fastqc/</a> (29 March 2017, date last accessed)), we trimmed adaptor sequences by cutadapt (<a class="link link-uri" href="https://cutadapt.readthedocs.io/en/stable/" target="_blank">https://cutadapt.readthedocs.io/en/stable/</a> (29 March 2017, date last accessed)). Low-quality reads were also filtered out by an empirically optimized custom Perl script as the following: (i) both ends of each read must have QV ≥10 (if not, the end base with QV &lt;10 is trimmed away until QV ≥10 is exposed); (ii) each read must have average QV ≥17 (if not, the read is discarded); (iii) final length of each read must be ≥20 bp (if not, the read is discarded); (iv) each read must have low-quality positions (QV &lt;10) no more than 10% of final length (if not, the read is discarded); and (v) each read must not contain any N bases (if not, the read is discarded). The Perl script for filtering out low-quality reads are available from our website (<a class="link link-uri" href="http://bioinf.mind.meiji.ac.jp/lab/en/download/readPreprocessingScripts.tar.gz" target="_blank">http://bioinf.mind.meiji.ac.jp/lab/en/download/readPreprocessingScripts.tar.gz</a> (29 March 2017, date last accessed)).</p>                            <h4 scrollto-destination=106356857 id="106356857" class="section-title" >2.3.2. Mapping of reads to the reference genome sequences</h4>
<p>We downloaded the genome sequences of <em>S. bicolor</em> (cv. BTx623) (Sbicolor_v2.1) and <em>O. sativa</em> (Japonica group, cv. Nipponbare) (Os-Nipponbare-Reference-IRGSP-1.0) from Phytozome (<a class="link link-uri" href="http://www.phytozome.net/" target="_blank">http://www.phytozome.net/</a> (29 March 2017, date last accessed)) and the Rice Annotation Project Database (RAP-DB) (<a class="link link-uri" href="http://rapdb.dna.affrc.go.jp/" target="_blank">http://rapdb.dna.affrc.go.jp/</a> (29 March 2017, date last accessed)), respectively<a reveal-id="dsx012-B18" data-open="dsx012-B18" class="link link-ref link-reveal xref-bibr"><sup>18–21</sup></a>. We aligned the preprocessed reads on the reference genome sequences using BWA with the default options. We then performed realignment of the reads of neighbouring INDELs using the RealignerTargetCreator and IndelRealigner commands of GATK (with default settings). Finally, we selected uniquely mapped reads with the X0:i:1 and X1:i:0 tags in the optional fields of the SAM files.</p>                            <h4 scrollto-destination=106356859 id="106356859" class="section-title" >2.3.3. SNP calling with RAD-seq reads</h4>
<p>To compare performance of Heap (ver. 0.7.8), Stacks (ver. 1.29), SAMtools/BCFtools (ver. 1.1), and GATK (ver. 3.2) under similar conditions, we called SNPs with RAD-seq reads as follows.</p><p>Analysis with Heap, Stacks and GATK, SNPs supported by 3 or more reads were used. In SAMtools/BCFtools and GATK analysis, SNPs with low genotype quality scores (GQs) (GQ &lt;13) were excluded.</p><p>In Heap analysis, we executed ‘heap’ command with the parameter of a minimum read depth (-d 3). Consequently, Heap provides an SNP list, which contains SNPs between samples.</p><p>In Stacks analysis, to obtain SNPs between samples, we executed by the pipeline via the ref_map.pl. To obtain SNPs supported stacks, we executed the ref_map.pl with the parameter of a minimum stack depth (-m 3). SNPs sites were obtained by the ‘populations’ command of Stacks in VCF format.</p><p>In SAMtools/BCFtools analysis, we ran the ‘mpileup’ command of SAMtools and the ‘call’ command of BCFtools. To describe GQs in FORMAT fields of the VCF file for subsequent filtering, the ‘call’ command was performed with the format fields parameter (-f GQ).</p><p>In GATK analysis, for each sample, we obtained genotypes at all nucleotide sites by the ‘HaplotypeCaller’ command. This command provides genotypes for all sites in Genomic VCF (gVCF) format, therefore, it contains both homozygous and heterozygous SNPs in each site. We then created a VCF file containing information of all SNPs between samples, which was integrated with all gVCF files by the ‘GenotypeGVCFs’ command using the default parameters.</p><p>To compare the SNPs derived by the multiple tools, we filtered the SNP sites, because conditions of SNP calling are different between Stacks and the other three tools (SAMtools/BCFtools, GATK, and Heap). Stacks calls SNPs at only nucleotide sites containing more than one allele among multiple samples. On the other hand, SAMtools/BCFtools, GATK, and Heap call SNPs at nucleotide sites containing one or more alternative alleles among multiple samples. For example, in three samples, a site containing genotypes ‘0/1’, ‘0/1’, and ‘./.’, is reported by the four tools. On the other hand, a site containing genotypes ‘1/1’, ‘1/1’, and ‘./.’, is not reported by Stacks but reported by the other tools. To equalize the conditions of the SNP lists derived by the multiple tools, we selected polymorphic SNP sites which contained 2 or more genotypes with more than one allele among samples, from the VCF files by a custom AWK script (<a class="link link-uri" href="http://bioinf.mind.meiji.ac.jp/lab/en/download/awkScriptToExtractPolySNPsitesFromVcf.sh" target="_blank">http://bioinf.mind.meiji.ac.jp/lab/en/download/awkScriptToExtractPolySNPsitesFromVcf.sh</a> (29 March 2017, date last accessed)).</p>                            <h4 scrollto-destination=106356867 id="106356867" class="section-title" >2.3.4. Mapping of the WGS reads and Genotyping of the ‘definitive answer genotypes’</h4>
<p>To determine if genotypes obtained from the low coverage RAD-seq reads are correct or not at each nucleotide site, we determined more probable genotypes as ‘definitive answer genotypes’ by using the high coverage WGS sequencing reads. In order to establish the definitive answer genotypes at each nucleotide site in each sorghum and rice line, we aligned WGS sequencing reads to the reference genome sequences and determined the definitive answer genotypes stringently at each nucleotide site with the aligned WGS sequencing reads (<a reveal-id="dsx012-F2" data-open="dsx012-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). </p>                        <a id="106356869" scrollto-destination="106356869"></a>
<div id="dsx012-F2" data-id="dsx012-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/m_dsx012f2.png?Expires=2147483647&Signature=jMxb9~SQrqYAefsG9ToGqiXhpLlVy4RCbdmEJZ4QD0YcoTmIvdznqAcULVDNmtvHiPUwd0~s20V0ut-TJOaJjFXO57Hi5qXaDhbyt3IXdnfsMuaiW~3He8f47LbwqsdVOIQTZA14-REK82cePuYKVlHTA-~6ISN8W2qcx1eC5kVWdzTONWsrvmGS~UryNv1leHfDsqRYU1wJXhH9muI5kwa33kSdHGKAaYbe82E1keZmzs7ksKwAlaD8xz5UQLdaXwCZtxxHrog1mstL5mXzFFSODZbV0n3pmdx6Adr01mR-rkeAegs-IK750bahwP8jP45eNUKeOqqODhTn~MCt4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of performance comparison of SNP calling by Heap, Stacks, SAMtools/BCFtools, and GATK. (A) Schematic view of the calculation of definitive answer genotypes with WGS sequencing reads. MAPQ are measures of mapping quality. (B) Schematic view of SNP calling from RAD-seq reads employing Heap, Stacks, SAMtools, and GATK. GQ indicates genotype quality. (C) Definitions of sensitivity, positive predictive value (PPV), and F-score for SNP calling." path-from-xml="dsx012f2.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356869/dsx012f2.png" path-from-xml="dsx012f2.png" target="_blank">View large</a><a section="106356869" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/dsx012f2.png?Expires=2147483647&Signature=iREdbTKXAwdfzn8-B9kT8qP2TYqeLpTDh1RwPpcx-6dMETPyy8PgqQc46mN7YiIuL3dcQfooLC7~cTu8IqYt7tqRdK95w3TGxs9scJddf1FID9nWMLYD0xLt73XeFx2-q07ZMy5FqmrrqYhBvJ97BoawCVtzdqoeQXCAgAn5YMcEDm8EC6QRGKwBKZKx7i0CcM2nmSQx8IMIfR-9BZk9PStGERoyqjHGVlz6bb2KTaUleNfEqaVXtYjk0SEJ~joa-9kmbQdwASms43QVhR7CWuZUNbAf5rjwCmlcfE4v1AB8YB~MGi1bKfbhJ68RPQQjypoj97y0KBOLA9DJ0f7XBQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106356869&ar=3815695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx012f2.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of performance comparison of SNP calling by Heap, Stacks, SAMtools/BCFtools, and GATK. (A) Schematic view of the calculation of definitive answer genotypes with WGS sequencing reads. MAPQ are measures of mapping quality. (B) Schematic view of SNP calling from RAD-seq reads employing Heap, Stacks, SAMtools, and GATK. GQ indicates genotype quality. (C) Definitions of sensitivity, positive predictive value (PPV), and <em>F</em>-score for SNP calling.</p></div></div><div content-id="dsx012-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/m_dsx012f2.png?Expires=2147483647&Signature=jMxb9~SQrqYAefsG9ToGqiXhpLlVy4RCbdmEJZ4QD0YcoTmIvdznqAcULVDNmtvHiPUwd0~s20V0ut-TJOaJjFXO57Hi5qXaDhbyt3IXdnfsMuaiW~3He8f47LbwqsdVOIQTZA14-REK82cePuYKVlHTA-~6ISN8W2qcx1eC5kVWdzTONWsrvmGS~UryNv1leHfDsqRYU1wJXhH9muI5kwa33kSdHGKAaYbe82E1keZmzs7ksKwAlaD8xz5UQLdaXwCZtxxHrog1mstL5mXzFFSODZbV0n3pmdx6Adr01mR-rkeAegs-IK750bahwP8jP45eNUKeOqqODhTn~MCt4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of performance comparison of SNP calling by Heap, Stacks, SAMtools/BCFtools, and GATK. (A) Schematic view of the calculation of definitive answer genotypes with WGS sequencing reads. MAPQ are measures of mapping quality. (B) Schematic view of SNP calling from RAD-seq reads employing Heap, Stacks, SAMtools, and GATK. GQ indicates genotype quality. (C) Definitions of sensitivity, positive predictive value (PPV), and F-score for SNP calling." path-from-xml="dsx012f2.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356869/dsx012f2.png" path-from-xml="dsx012f2.png" target="_blank">View large</a><a section="106356869" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/dsx012f2.png?Expires=2147483647&Signature=iREdbTKXAwdfzn8-B9kT8qP2TYqeLpTDh1RwPpcx-6dMETPyy8PgqQc46mN7YiIuL3dcQfooLC7~cTu8IqYt7tqRdK95w3TGxs9scJddf1FID9nWMLYD0xLt73XeFx2-q07ZMy5FqmrrqYhBvJ97BoawCVtzdqoeQXCAgAn5YMcEDm8EC6QRGKwBKZKx7i0CcM2nmSQx8IMIfR-9BZk9PStGERoyqjHGVlz6bb2KTaUleNfEqaVXtYjk0SEJ~joa-9kmbQdwASms43QVhR7CWuZUNbAf5rjwCmlcfE4v1AB8YB~MGi1bKfbhJ68RPQQjypoj97y0KBOLA9DJ0f7XBQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106356869&ar=3815695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx012f2.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of performance comparison of SNP calling by Heap, Stacks, SAMtools/BCFtools, and GATK. (A) Schematic view of the calculation of definitive answer genotypes with WGS sequencing reads. MAPQ are measures of mapping quality. (B) Schematic view of SNP calling from RAD-seq reads employing Heap, Stacks, SAMtools, and GATK. GQ indicates genotype quality. (C) Definitions of sensitivity, positive predictive value (PPV), and <em>F</em>-score for SNP calling.</p></div></div><p>After adapter trimming and quality filtering, we aligned the WGS sequencing reads to the reference genome sequences by using BWA. We obtained bases of the mapped WGS sequencing reads using the mpileup command of SAMtools. To ensure the high-quality calls from accurately mapped reads, we used the MAPQ 20 and the minimum base quality 20 for the mpileup command. Additionally, to obtain bases that had sufficient read coverages from the mpileup result, nucleotide sites supported with 20 or more reads were extracted by the awk command (awk -F ′\t′ ′{if($4 &gt; =20){print $0}}′). We counted reads for each nucleotide base (A, T, G, or C) and calculated frequencies of the nucleotide bases at each nucleotide site using custom Perl scripts. We then determined each genotype according to the following rigorous conditions: (i) when the major allele frequency is ≥ 0.95, the genotype is homozygous; (ii) when the major allele frequency is ≥ 0.5 and ≤ 0.6, the genotype is heterozygous; (iii) otherwise, the genotype is not available (NA). Finally, we filtered out the monomorphic sites, which contained a single genotype among samples. A shell script and Perl scripts for making definitive answer genotypes are available from our website (<a class="link link-uri" href="http://bioinf.mind.meiji.ac.jp/lab/en/download/benchMarkingScripts.tar.gz" target="_blank">http://bioinf.mind.meiji.ac.jp/lab/en/download/benchMarkingScripts.tar.gz</a> (29 March 2017, date last accessed)).</p>                            <h2 scrollto-destination=106356871 id="106356871" class="section-title" >3. Results and discussion</h2>
<p>To compare the sensitivities and PPVs of SNP calling for Heap, Stacks, SAMtools/BCFtools, and GATK with low read coverage, we detected SNPs using low coverage RAD-seq reads by the tools in two plant species, sorghum and rice. Sensitivities and PPVs of the SNPs were calculated by comparing the SNPs detected with RAD-seq reads, and SNPs rigorously detected by a conventional strategy with more than adequate amounts of WGS sequencing reads, i.e. definitive answer genotypes. Then, we calculated the <em>F</em>-score, which represents a total performance index for sensitivity and accuracy of SNP mining.</p>                            <h3 scrollto-destination=106356873 id="106356873" class="section-title" >3.1. Genotyping with WGS sequencing reads for the ‘definitive answer genotypes’</h3>
<p>Firstly, to establish the ‘definitive answer genotypes’, we aligned the WGS sequencing read to the reference genome sequences in sorghum and rice. An overview of the preprocessing and the mapping for the WGS sequencing reads are presented in <a reveal-id="dsx012-T1" data-open="dsx012-T1" class="link link-reveal link-table xref-fig">Tables 1</a> and <a reveal-id="dsx012-T2" data-open="dsx012-T2" class="link link-reveal link-table xref-fig">2</a>. Averages of the WGS read coverages were 21.5 and 38.5 in the 17 sorghum lines and in the 4 rice lines, respectively. These read coverages (around 20–40) should be sufficient to perform definitively accurate genotyping<a reveal-id="dsx012-B12" data-open="dsx012-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a>. Subsequently, we conservatively determined the definitive answer genotypes at each nucleotide site. In average, 4,045 SNP sites and 2,955,720 non-SNP sites were obtained in sorghum; 122 SNP sites and 2,779,362 non-SNP sites were obtained in rice. These results indicated that the number of non-SNP sites was very larger than the SNP sites. In this situation, specificity shows a large value, and is not informative value. In fact, all specificity values were over than 0.999 in this study (data not shown). Therefore, we adopted PPV to examine rate of precisely called SNPs in all SNPs. <div content-id="dsx012-T1" class="table-modal"><div class="table-wrap"><div id="dsx012-T1" data-id="dsx012-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Summary of WGS sequencing reads and mapping of the reads in Sorghum</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="3">Sample </th><th colspan="2">Raw reads<hr /> </th><th colspan="2">Preprocessed reads<hr /> </th><th colspan="2">Mapped reads<hr /> </th><th colspan="6">Uniquely mapped reads<hr /> </th></tr><tr><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="4">Coverage<hr /> </th></tr><tr><th>Mean </th><th>1st quartile </th><th>Median </th><th>3rd quartile </th></tr></thead><tbody><tr><td><strong>GULUM ABIAD</strong> </td><td>219.2 </td><td>22.1 </td><td>200.5 </td><td>19.9 </td><td>189.0 </td><td>94.3 </td><td>100.3 </td><td>50.0 </td><td>21.2 </td><td>11 </td><td>20 </td><td>27 </td></tr><tr><td><strong>A-6129</strong> </td><td>205.1 </td><td>20.7 </td><td>191.9 </td><td>19.1 </td><td>181.1 </td><td>94.4 </td><td>97.0 </td><td>50.5 </td><td>20.2 </td><td>12 </td><td>19 </td><td>26 </td></tr><tr><td><strong>AGIRA</strong> </td><td>214.0 </td><td>21.6 </td><td>195.4 </td><td>19.4 </td><td>185.5 </td><td>94.9 </td><td>104.5 </td><td>53.5 </td><td>18.7 </td><td>10 </td><td>18 </td><td>24 </td></tr><tr><td><strong>LR 399</strong> </td><td>232.7 </td><td>23.5 </td><td>213.7 </td><td>21.2 </td><td>202.6 </td><td>94.8 </td><td>113.4 </td><td>53.1 </td><td>20.8 </td><td>11 </td><td>20 </td><td>27 </td></tr><tr><td><strong>CRIOLLO CABEZA APRETADA</strong> </td><td>260.9 </td><td>26.4 </td><td>239.2 </td><td>23.8 </td><td>230.0 </td><td>96.2 </td><td>131.1 </td><td>54.8 </td><td>24.8 </td><td>14 </td><td>24 </td><td>32 </td></tr><tr><td><strong>B2AR3043</strong> </td><td>206.7 </td><td>20.9 </td><td>181.4 </td><td>18.4 </td><td>171.9 </td><td>94.8 </td><td>98.5 </td><td>54.3 </td><td>19.1 </td><td>12 </td><td>18 </td><td>24 </td></tr><tr><td><strong>RTx430</strong> </td><td>215.2 </td><td>21.7 </td><td>196.4 </td><td>19.5 </td><td>188.4 </td><td>95.9 </td><td>105.8 </td><td>53.8 </td><td>20.4 </td><td>12 </td><td>19 </td><td>25 </td></tr><tr><td><strong>B3Tx2817</strong> </td><td>226.5 </td><td>22.9 </td><td>209.8 </td><td>20.9 </td><td>203.3 </td><td>96.9 </td><td>118.7 </td><td>56.6 </td><td>21.7 </td><td>12 </td><td>21 </td><td>28 </td></tr><tr><td><strong>ZIRA-EL-SABI</strong> </td><td>235.7 </td><td>23.8 </td><td>218.1 </td><td>21.7 </td><td>210.3 </td><td>96.4 </td><td>118.0 </td><td>54.1 </td><td>22.9 </td><td>14 </td><td>22 </td><td>29 </td></tr><tr><td><strong>ITALIAN</strong> </td><td>192.2 </td><td>19.4 </td><td>177.0 </td><td>17.6 </td><td>170.2 </td><td>96.1 </td><td>97.5 </td><td>55.1 </td><td>19.2 </td><td>12 </td><td>18 </td><td>24 </td></tr><tr><td><strong>CAPRICORN</strong> </td><td>250.0 </td><td>25.3 </td><td>233.1 </td><td>23.2 </td><td>225.2 </td><td>96.6 </td><td>129.1 </td><td>55.4 </td><td>24.6 </td><td>16 </td><td>24 </td><td>31 </td></tr><tr><td><strong>SOR 1</strong> </td><td>270.0 </td><td>27.3 </td><td>237.4 </td><td>23.4 </td><td>225.8 </td><td>95.1 </td><td>120.1 </td><td>50.6 </td><td>23.8 </td><td>14 </td><td>23 </td><td>30 </td></tr><tr><td><strong>15065</strong> </td><td>232.4 </td><td>23.5 </td><td>207.2 </td><td>20.5 </td><td>196.7 </td><td>94.9 </td><td>105.9 </td><td>51.1 </td><td>20.8 </td><td>12 </td><td>20 </td><td>26 </td></tr><tr><td><strong>SAP-155</strong> </td><td>217.6 </td><td>22.0 </td><td>200.4 </td><td>19.8 </td><td>191.7 </td><td>95.6 </td><td>107.9 </td><td>53.9 </td><td>20.9 </td><td>13 </td><td>20 </td><td>26 </td></tr><tr><td><strong>RCV</strong> </td><td>216.8 </td><td>21.9 </td><td>197.0 </td><td>19.4 </td><td>188.7 </td><td>95.8 </td><td>108.9 </td><td>55.3 </td><td>20.8 </td><td>12 </td><td>20 </td><td>26 </td></tr><tr><td><strong>SC 56</strong> </td><td>203.8 </td><td>20.6 </td><td>177.8 </td><td>17.4 </td><td>170.3 </td><td>95.8 </td><td>96.9 </td><td>54.5 </td><td>19.3 </td><td>12 </td><td>18 </td><td>24 </td></tr><tr><td><strong>SIL-05</strong> </td><td>272.4 </td><td>27.5 </td><td>239.2 </td><td>23.5 </td><td>229.6 </td><td>96.0 </td><td>129.9 </td><td>54.3 </td><td>25.7 </td><td>15 </td><td>25 </td><td>33 </td></tr><tr><td><strong>Total</strong> </td><td>3871.3 </td><td>391.0 </td><td>3515.5 </td><td>348.6 </td><td>3360.2 </td><td>– </td><td>1883.3 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td><strong>Average</strong> </td><td>227.7 </td><td>23.0 </td><td>206.8 </td><td>20.5 </td><td>197.7 </td><td>95.6 </td><td>110.8 </td><td>53.6 </td><td>21.5 </td><td>12.6 </td><td>20.5 </td><td>27.2 </td></tr></tbody></table></div><div content-id="dsx012-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="3">Sample </th><th colspan="2">Raw reads<hr /> </th><th colspan="2">Preprocessed reads<hr /> </th><th colspan="2">Mapped reads<hr /> </th><th colspan="6">Uniquely mapped reads<hr /> </th></tr><tr><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="4">Coverage<hr /> </th></tr><tr><th>Mean </th><th>1st quartile </th><th>Median </th><th>3rd quartile </th></tr></thead><tbody><tr><td><strong>GULUM ABIAD</strong> </td><td>219.2 </td><td>22.1 </td><td>200.5 </td><td>19.9 </td><td>189.0 </td><td>94.3 </td><td>100.3 </td><td>50.0 </td><td>21.2 </td><td>11 </td><td>20 </td><td>27 </td></tr><tr><td><strong>A-6129</strong> </td><td>205.1 </td><td>20.7 </td><td>191.9 </td><td>19.1 </td><td>181.1 </td><td>94.4 </td><td>97.0 </td><td>50.5 </td><td>20.2 </td><td>12 </td><td>19 </td><td>26 </td></tr><tr><td><strong>AGIRA</strong> </td><td>214.0 </td><td>21.6 </td><td>195.4 </td><td>19.4 </td><td>185.5 </td><td>94.9 </td><td>104.5 </td><td>53.5 </td><td>18.7 </td><td>10 </td><td>18 </td><td>24 </td></tr><tr><td><strong>LR 399</strong> </td><td>232.7 </td><td>23.5 </td><td>213.7 </td><td>21.2 </td><td>202.6 </td><td>94.8 </td><td>113.4 </td><td>53.1 </td><td>20.8 </td><td>11 </td><td>20 </td><td>27 </td></tr><tr><td><strong>CRIOLLO CABEZA APRETADA</strong> </td><td>260.9 </td><td>26.4 </td><td>239.2 </td><td>23.8 </td><td>230.0 </td><td>96.2 </td><td>131.1 </td><td>54.8 </td><td>24.8 </td><td>14 </td><td>24 </td><td>32 </td></tr><tr><td><strong>B2AR3043</strong> </td><td>206.7 </td><td>20.9 </td><td>181.4 </td><td>18.4 </td><td>171.9 </td><td>94.8 </td><td>98.5 </td><td>54.3 </td><td>19.1 </td><td>12 </td><td>18 </td><td>24 </td></tr><tr><td><strong>RTx430</strong> </td><td>215.2 </td><td>21.7 </td><td>196.4 </td><td>19.5 </td><td>188.4 </td><td>95.9 </td><td>105.8 </td><td>53.8 </td><td>20.4 </td><td>12 </td><td>19 </td><td>25 </td></tr><tr><td><strong>B3Tx2817</strong> </td><td>226.5 </td><td>22.9 </td><td>209.8 </td><td>20.9 </td><td>203.3 </td><td>96.9 </td><td>118.7 </td><td>56.6 </td><td>21.7 </td><td>12 </td><td>21 </td><td>28 </td></tr><tr><td><strong>ZIRA-EL-SABI</strong> </td><td>235.7 </td><td>23.8 </td><td>218.1 </td><td>21.7 </td><td>210.3 </td><td>96.4 </td><td>118.0 </td><td>54.1 </td><td>22.9 </td><td>14 </td><td>22 </td><td>29 </td></tr><tr><td><strong>ITALIAN</strong> </td><td>192.2 </td><td>19.4 </td><td>177.0 </td><td>17.6 </td><td>170.2 </td><td>96.1 </td><td>97.5 </td><td>55.1 </td><td>19.2 </td><td>12 </td><td>18 </td><td>24 </td></tr><tr><td><strong>CAPRICORN</strong> </td><td>250.0 </td><td>25.3 </td><td>233.1 </td><td>23.2 </td><td>225.2 </td><td>96.6 </td><td>129.1 </td><td>55.4 </td><td>24.6 </td><td>16 </td><td>24 </td><td>31 </td></tr><tr><td><strong>SOR 1</strong> </td><td>270.0 </td><td>27.3 </td><td>237.4 </td><td>23.4 </td><td>225.8 </td><td>95.1 </td><td>120.1 </td><td>50.6 </td><td>23.8 </td><td>14 </td><td>23 </td><td>30 </td></tr><tr><td><strong>15065</strong> </td><td>232.4 </td><td>23.5 </td><td>207.2 </td><td>20.5 </td><td>196.7 </td><td>94.9 </td><td>105.9 </td><td>51.1 </td><td>20.8 </td><td>12 </td><td>20 </td><td>26 </td></tr><tr><td><strong>SAP-155</strong> </td><td>217.6 </td><td>22.0 </td><td>200.4 </td><td>19.8 </td><td>191.7 </td><td>95.6 </td><td>107.9 </td><td>53.9 </td><td>20.9 </td><td>13 </td><td>20 </td><td>26 </td></tr><tr><td><strong>RCV</strong> </td><td>216.8 </td><td>21.9 </td><td>197.0 </td><td>19.4 </td><td>188.7 </td><td>95.8 </td><td>108.9 </td><td>55.3 </td><td>20.8 </td><td>12 </td><td>20 </td><td>26 </td></tr><tr><td><strong>SC 56</strong> </td><td>203.8 </td><td>20.6 </td><td>177.8 </td><td>17.4 </td><td>170.3 </td><td>95.8 </td><td>96.9 </td><td>54.5 </td><td>19.3 </td><td>12 </td><td>18 </td><td>24 </td></tr><tr><td><strong>SIL-05</strong> </td><td>272.4 </td><td>27.5 </td><td>239.2 </td><td>23.5 </td><td>229.6 </td><td>96.0 </td><td>129.9 </td><td>54.3 </td><td>25.7 </td><td>15 </td><td>25 </td><td>33 </td></tr><tr><td><strong>Total</strong> </td><td>3871.3 </td><td>391.0 </td><td>3515.5 </td><td>348.6 </td><td>3360.2 </td><td>– </td><td>1883.3 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td><strong>Average</strong> </td><td>227.7 </td><td>23.0 </td><td>206.8 </td><td>20.5 </td><td>197.7 </td><td>95.6 </td><td>110.8 </td><td>53.6 </td><td>21.5 </td><td>12.6 </td><td>20.5 </td><td>27.2 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>All read counts and lengths are shown in millions and billions, respectively.</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>a</sup>Mapping rates are calculated as the ratio of the number of the mapped reads against the number of the preprocessed reads.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx012-T1" data-id="dsx012-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Summary of WGS sequencing reads and mapping of the reads in Sorghum</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="3">Sample </th><th colspan="2">Raw reads<hr /> </th><th colspan="2">Preprocessed reads<hr /> </th><th colspan="2">Mapped reads<hr /> </th><th colspan="6">Uniquely mapped reads<hr /> </th></tr><tr><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="4">Coverage<hr /> </th></tr><tr><th>Mean </th><th>1st quartile </th><th>Median </th><th>3rd quartile </th></tr></thead><tbody><tr><td><strong>GULUM ABIAD</strong> </td><td>219.2 </td><td>22.1 </td><td>200.5 </td><td>19.9 </td><td>189.0 </td><td>94.3 </td><td>100.3 </td><td>50.0 </td><td>21.2 </td><td>11 </td><td>20 </td><td>27 </td></tr><tr><td><strong>A-6129</strong> </td><td>205.1 </td><td>20.7 </td><td>191.9 </td><td>19.1 </td><td>181.1 </td><td>94.4 </td><td>97.0 </td><td>50.5 </td><td>20.2 </td><td>12 </td><td>19 </td><td>26 </td></tr><tr><td><strong>AGIRA</strong> </td><td>214.0 </td><td>21.6 </td><td>195.4 </td><td>19.4 </td><td>185.5 </td><td>94.9 </td><td>104.5 </td><td>53.5 </td><td>18.7 </td><td>10 </td><td>18 </td><td>24 </td></tr><tr><td><strong>LR 399</strong> </td><td>232.7 </td><td>23.5 </td><td>213.7 </td><td>21.2 </td><td>202.6 </td><td>94.8 </td><td>113.4 </td><td>53.1 </td><td>20.8 </td><td>11 </td><td>20 </td><td>27 </td></tr><tr><td><strong>CRIOLLO CABEZA APRETADA</strong> </td><td>260.9 </td><td>26.4 </td><td>239.2 </td><td>23.8 </td><td>230.0 </td><td>96.2 </td><td>131.1 </td><td>54.8 </td><td>24.8 </td><td>14 </td><td>24 </td><td>32 </td></tr><tr><td><strong>B2AR3043</strong> </td><td>206.7 </td><td>20.9 </td><td>181.4 </td><td>18.4 </td><td>171.9 </td><td>94.8 </td><td>98.5 </td><td>54.3 </td><td>19.1 </td><td>12 </td><td>18 </td><td>24 </td></tr><tr><td><strong>RTx430</strong> </td><td>215.2 </td><td>21.7 </td><td>196.4 </td><td>19.5 </td><td>188.4 </td><td>95.9 </td><td>105.8 </td><td>53.8 </td><td>20.4 </td><td>12 </td><td>19 </td><td>25 </td></tr><tr><td><strong>B3Tx2817</strong> </td><td>226.5 </td><td>22.9 </td><td>209.8 </td><td>20.9 </td><td>203.3 </td><td>96.9 </td><td>118.7 </td><td>56.6 </td><td>21.7 </td><td>12 </td><td>21 </td><td>28 </td></tr><tr><td><strong>ZIRA-EL-SABI</strong> </td><td>235.7 </td><td>23.8 </td><td>218.1 </td><td>21.7 </td><td>210.3 </td><td>96.4 </td><td>118.0 </td><td>54.1 </td><td>22.9 </td><td>14 </td><td>22 </td><td>29 </td></tr><tr><td><strong>ITALIAN</strong> </td><td>192.2 </td><td>19.4 </td><td>177.0 </td><td>17.6 </td><td>170.2 </td><td>96.1 </td><td>97.5 </td><td>55.1 </td><td>19.2 </td><td>12 </td><td>18 </td><td>24 </td></tr><tr><td><strong>CAPRICORN</strong> </td><td>250.0 </td><td>25.3 </td><td>233.1 </td><td>23.2 </td><td>225.2 </td><td>96.6 </td><td>129.1 </td><td>55.4 </td><td>24.6 </td><td>16 </td><td>24 </td><td>31 </td></tr><tr><td><strong>SOR 1</strong> </td><td>270.0 </td><td>27.3 </td><td>237.4 </td><td>23.4 </td><td>225.8 </td><td>95.1 </td><td>120.1 </td><td>50.6 </td><td>23.8 </td><td>14 </td><td>23 </td><td>30 </td></tr><tr><td><strong>15065</strong> </td><td>232.4 </td><td>23.5 </td><td>207.2 </td><td>20.5 </td><td>196.7 </td><td>94.9 </td><td>105.9 </td><td>51.1 </td><td>20.8 </td><td>12 </td><td>20 </td><td>26 </td></tr><tr><td><strong>SAP-155</strong> </td><td>217.6 </td><td>22.0 </td><td>200.4 </td><td>19.8 </td><td>191.7 </td><td>95.6 </td><td>107.9 </td><td>53.9 </td><td>20.9 </td><td>13 </td><td>20 </td><td>26 </td></tr><tr><td><strong>RCV</strong> </td><td>216.8 </td><td>21.9 </td><td>197.0 </td><td>19.4 </td><td>188.7 </td><td>95.8 </td><td>108.9 </td><td>55.3 </td><td>20.8 </td><td>12 </td><td>20 </td><td>26 </td></tr><tr><td><strong>SC 56</strong> </td><td>203.8 </td><td>20.6 </td><td>177.8 </td><td>17.4 </td><td>170.3 </td><td>95.8 </td><td>96.9 </td><td>54.5 </td><td>19.3 </td><td>12 </td><td>18 </td><td>24 </td></tr><tr><td><strong>SIL-05</strong> </td><td>272.4 </td><td>27.5 </td><td>239.2 </td><td>23.5 </td><td>229.6 </td><td>96.0 </td><td>129.9 </td><td>54.3 </td><td>25.7 </td><td>15 </td><td>25 </td><td>33 </td></tr><tr><td><strong>Total</strong> </td><td>3871.3 </td><td>391.0 </td><td>3515.5 </td><td>348.6 </td><td>3360.2 </td><td>– </td><td>1883.3 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td><strong>Average</strong> </td><td>227.7 </td><td>23.0 </td><td>206.8 </td><td>20.5 </td><td>197.7 </td><td>95.6 </td><td>110.8 </td><td>53.6 </td><td>21.5 </td><td>12.6 </td><td>20.5 </td><td>27.2 </td></tr></tbody></table></div><div content-id="dsx012-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="3">Sample </th><th colspan="2">Raw reads<hr /> </th><th colspan="2">Preprocessed reads<hr /> </th><th colspan="2">Mapped reads<hr /> </th><th colspan="6">Uniquely mapped reads<hr /> </th></tr><tr><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="4">Coverage<hr /> </th></tr><tr><th>Mean </th><th>1st quartile </th><th>Median </th><th>3rd quartile </th></tr></thead><tbody><tr><td><strong>GULUM ABIAD</strong> </td><td>219.2 </td><td>22.1 </td><td>200.5 </td><td>19.9 </td><td>189.0 </td><td>94.3 </td><td>100.3 </td><td>50.0 </td><td>21.2 </td><td>11 </td><td>20 </td><td>27 </td></tr><tr><td><strong>A-6129</strong> </td><td>205.1 </td><td>20.7 </td><td>191.9 </td><td>19.1 </td><td>181.1 </td><td>94.4 </td><td>97.0 </td><td>50.5 </td><td>20.2 </td><td>12 </td><td>19 </td><td>26 </td></tr><tr><td><strong>AGIRA</strong> </td><td>214.0 </td><td>21.6 </td><td>195.4 </td><td>19.4 </td><td>185.5 </td><td>94.9 </td><td>104.5 </td><td>53.5 </td><td>18.7 </td><td>10 </td><td>18 </td><td>24 </td></tr><tr><td><strong>LR 399</strong> </td><td>232.7 </td><td>23.5 </td><td>213.7 </td><td>21.2 </td><td>202.6 </td><td>94.8 </td><td>113.4 </td><td>53.1 </td><td>20.8 </td><td>11 </td><td>20 </td><td>27 </td></tr><tr><td><strong>CRIOLLO CABEZA APRETADA</strong> </td><td>260.9 </td><td>26.4 </td><td>239.2 </td><td>23.8 </td><td>230.0 </td><td>96.2 </td><td>131.1 </td><td>54.8 </td><td>24.8 </td><td>14 </td><td>24 </td><td>32 </td></tr><tr><td><strong>B2AR3043</strong> </td><td>206.7 </td><td>20.9 </td><td>181.4 </td><td>18.4 </td><td>171.9 </td><td>94.8 </td><td>98.5 </td><td>54.3 </td><td>19.1 </td><td>12 </td><td>18 </td><td>24 </td></tr><tr><td><strong>RTx430</strong> </td><td>215.2 </td><td>21.7 </td><td>196.4 </td><td>19.5 </td><td>188.4 </td><td>95.9 </td><td>105.8 </td><td>53.8 </td><td>20.4 </td><td>12 </td><td>19 </td><td>25 </td></tr><tr><td><strong>B3Tx2817</strong> </td><td>226.5 </td><td>22.9 </td><td>209.8 </td><td>20.9 </td><td>203.3 </td><td>96.9 </td><td>118.7 </td><td>56.6 </td><td>21.7 </td><td>12 </td><td>21 </td><td>28 </td></tr><tr><td><strong>ZIRA-EL-SABI</strong> </td><td>235.7 </td><td>23.8 </td><td>218.1 </td><td>21.7 </td><td>210.3 </td><td>96.4 </td><td>118.0 </td><td>54.1 </td><td>22.9 </td><td>14 </td><td>22 </td><td>29 </td></tr><tr><td><strong>ITALIAN</strong> </td><td>192.2 </td><td>19.4 </td><td>177.0 </td><td>17.6 </td><td>170.2 </td><td>96.1 </td><td>97.5 </td><td>55.1 </td><td>19.2 </td><td>12 </td><td>18 </td><td>24 </td></tr><tr><td><strong>CAPRICORN</strong> </td><td>250.0 </td><td>25.3 </td><td>233.1 </td><td>23.2 </td><td>225.2 </td><td>96.6 </td><td>129.1 </td><td>55.4 </td><td>24.6 </td><td>16 </td><td>24 </td><td>31 </td></tr><tr><td><strong>SOR 1</strong> </td><td>270.0 </td><td>27.3 </td><td>237.4 </td><td>23.4 </td><td>225.8 </td><td>95.1 </td><td>120.1 </td><td>50.6 </td><td>23.8 </td><td>14 </td><td>23 </td><td>30 </td></tr><tr><td><strong>15065</strong> </td><td>232.4 </td><td>23.5 </td><td>207.2 </td><td>20.5 </td><td>196.7 </td><td>94.9 </td><td>105.9 </td><td>51.1 </td><td>20.8 </td><td>12 </td><td>20 </td><td>26 </td></tr><tr><td><strong>SAP-155</strong> </td><td>217.6 </td><td>22.0 </td><td>200.4 </td><td>19.8 </td><td>191.7 </td><td>95.6 </td><td>107.9 </td><td>53.9 </td><td>20.9 </td><td>13 </td><td>20 </td><td>26 </td></tr><tr><td><strong>RCV</strong> </td><td>216.8 </td><td>21.9 </td><td>197.0 </td><td>19.4 </td><td>188.7 </td><td>95.8 </td><td>108.9 </td><td>55.3 </td><td>20.8 </td><td>12 </td><td>20 </td><td>26 </td></tr><tr><td><strong>SC 56</strong> </td><td>203.8 </td><td>20.6 </td><td>177.8 </td><td>17.4 </td><td>170.3 </td><td>95.8 </td><td>96.9 </td><td>54.5 </td><td>19.3 </td><td>12 </td><td>18 </td><td>24 </td></tr><tr><td><strong>SIL-05</strong> </td><td>272.4 </td><td>27.5 </td><td>239.2 </td><td>23.5 </td><td>229.6 </td><td>96.0 </td><td>129.9 </td><td>54.3 </td><td>25.7 </td><td>15 </td><td>25 </td><td>33 </td></tr><tr><td><strong>Total</strong> </td><td>3871.3 </td><td>391.0 </td><td>3515.5 </td><td>348.6 </td><td>3360.2 </td><td>– </td><td>1883.3 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td><strong>Average</strong> </td><td>227.7 </td><td>23.0 </td><td>206.8 </td><td>20.5 </td><td>197.7 </td><td>95.6 </td><td>110.8 </td><td>53.6 </td><td>21.5 </td><td>12.6 </td><td>20.5 </td><td>27.2 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>All read counts and lengths are shown in millions and billions, respectively.</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>a</sup>Mapping rates are calculated as the ratio of the number of the mapped reads against the number of the preprocessed reads.</p></div></div></div></div><div content-id="dsx012-T2" class="table-modal"><div class="table-wrap"><div id="dsx012-T2" data-id="dsx012-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Summary of WGS sequencing reads and mapping of the reads in rice</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="3">Sample </th><th colspan="2">Raw reads<hr /> </th><th colspan="2">Preprocessed reads<hr /> </th><th colspan="2">Mapped reads<hr /> </th><th colspan="6">Uniquely mapped reads<hr /> </th></tr><tr><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="4">Coverage<hr /> </th></tr><tr><th>Mean </th><th>1st quartile </th><th>Median </th><th>3rd quartile </th></tr></thead><tbody><tr><td><strong>Omachi</strong> </td><td>297.9 </td><td>22.3 </td><td>260.7 </td><td>18.3 </td><td>257.0 </td><td>98.6 </td><td>185.4 </td><td>71.1 </td><td>41.6 </td><td>27 </td><td>42 </td><td>56 </td></tr><tr><td><strong>Yamada Nishiki</strong> </td><td>218.3 </td><td>19.3 </td><td>184.7 </td><td>15.8 </td><td>181.3 </td><td>98.2 </td><td>140.6 </td><td>76.1 </td><td>37.4 </td><td>30 </td><td>40 </td><td>47 </td></tr><tr><td><strong>Hitomebore</strong> </td><td>221.9 </td><td>16.6 </td><td>202.2 </td><td>14.9 </td><td>198.0 </td><td>97.9 </td><td>144.3 </td><td>71.4 </td><td>34.2 </td><td>18 </td><td>35 </td><td>48 </td></tr><tr><td><strong>Kameji</strong> </td><td>233.8 </td><td>20.7 </td><td>219.4 </td><td>18.7 </td><td>214.0 </td><td>97.6 </td><td>154.3 </td><td>70.4 </td><td>40.5 </td><td>27 </td><td>44 </td><td>54 </td></tr><tr><td><strong>Total</strong> </td><td>971.9 </td><td>79.1 </td><td>866.8 </td><td>67.7 </td><td>850.2 </td><td>– </td><td>624.6 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td><strong>Average</strong> </td><td>243.0 </td><td>19.8 </td><td>216.7 </td><td>16.9 </td><td>212.6 </td><td>98.1 </td><td>156.1 </td><td>72.2 </td><td>38.5 </td><td>25.5 </td><td>40.3 </td><td>51.3 </td></tr></tbody></table></div><div content-id="dsx012-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="3">Sample </th><th colspan="2">Raw reads<hr /> </th><th colspan="2">Preprocessed reads<hr /> </th><th colspan="2">Mapped reads<hr /> </th><th colspan="6">Uniquely mapped reads<hr /> </th></tr><tr><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="4">Coverage<hr /> </th></tr><tr><th>Mean </th><th>1st quartile </th><th>Median </th><th>3rd quartile </th></tr></thead><tbody><tr><td><strong>Omachi</strong> </td><td>297.9 </td><td>22.3 </td><td>260.7 </td><td>18.3 </td><td>257.0 </td><td>98.6 </td><td>185.4 </td><td>71.1 </td><td>41.6 </td><td>27 </td><td>42 </td><td>56 </td></tr><tr><td><strong>Yamada Nishiki</strong> </td><td>218.3 </td><td>19.3 </td><td>184.7 </td><td>15.8 </td><td>181.3 </td><td>98.2 </td><td>140.6 </td><td>76.1 </td><td>37.4 </td><td>30 </td><td>40 </td><td>47 </td></tr><tr><td><strong>Hitomebore</strong> </td><td>221.9 </td><td>16.6 </td><td>202.2 </td><td>14.9 </td><td>198.0 </td><td>97.9 </td><td>144.3 </td><td>71.4 </td><td>34.2 </td><td>18 </td><td>35 </td><td>48 </td></tr><tr><td><strong>Kameji</strong> </td><td>233.8 </td><td>20.7 </td><td>219.4 </td><td>18.7 </td><td>214.0 </td><td>97.6 </td><td>154.3 </td><td>70.4 </td><td>40.5 </td><td>27 </td><td>44 </td><td>54 </td></tr><tr><td><strong>Total</strong> </td><td>971.9 </td><td>79.1 </td><td>866.8 </td><td>67.7 </td><td>850.2 </td><td>– </td><td>624.6 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td><strong>Average</strong> </td><td>243.0 </td><td>19.8 </td><td>216.7 </td><td>16.9 </td><td>212.6 </td><td>98.1 </td><td>156.1 </td><td>72.2 </td><td>38.5 </td><td>25.5 </td><td>40.3 </td><td>51.3 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p>All read counts and lengths are shown in millions and billions, respectively.</p></div></div><div content-id="tblfn4"><div class="fn"><p><sup>a</sup>Mapping rates are calculated as the ratio of the number of the mapped reads against the number of the preprocessed reads.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx012-T2" data-id="dsx012-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Summary of WGS sequencing reads and mapping of the reads in rice</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="3">Sample </th><th colspan="2">Raw reads<hr /> </th><th colspan="2">Preprocessed reads<hr /> </th><th colspan="2">Mapped reads<hr /> </th><th colspan="6">Uniquely mapped reads<hr /> </th></tr><tr><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="4">Coverage<hr /> </th></tr><tr><th>Mean </th><th>1st quartile </th><th>Median </th><th>3rd quartile </th></tr></thead><tbody><tr><td><strong>Omachi</strong> </td><td>297.9 </td><td>22.3 </td><td>260.7 </td><td>18.3 </td><td>257.0 </td><td>98.6 </td><td>185.4 </td><td>71.1 </td><td>41.6 </td><td>27 </td><td>42 </td><td>56 </td></tr><tr><td><strong>Yamada Nishiki</strong> </td><td>218.3 </td><td>19.3 </td><td>184.7 </td><td>15.8 </td><td>181.3 </td><td>98.2 </td><td>140.6 </td><td>76.1 </td><td>37.4 </td><td>30 </td><td>40 </td><td>47 </td></tr><tr><td><strong>Hitomebore</strong> </td><td>221.9 </td><td>16.6 </td><td>202.2 </td><td>14.9 </td><td>198.0 </td><td>97.9 </td><td>144.3 </td><td>71.4 </td><td>34.2 </td><td>18 </td><td>35 </td><td>48 </td></tr><tr><td><strong>Kameji</strong> </td><td>233.8 </td><td>20.7 </td><td>219.4 </td><td>18.7 </td><td>214.0 </td><td>97.6 </td><td>154.3 </td><td>70.4 </td><td>40.5 </td><td>27 </td><td>44 </td><td>54 </td></tr><tr><td><strong>Total</strong> </td><td>971.9 </td><td>79.1 </td><td>866.8 </td><td>67.7 </td><td>850.2 </td><td>– </td><td>624.6 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td><strong>Average</strong> </td><td>243.0 </td><td>19.8 </td><td>216.7 </td><td>16.9 </td><td>212.6 </td><td>98.1 </td><td>156.1 </td><td>72.2 </td><td>38.5 </td><td>25.5 </td><td>40.3 </td><td>51.3 </td></tr></tbody></table></div><div content-id="dsx012-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="3">Sample </th><th colspan="2">Raw reads<hr /> </th><th colspan="2">Preprocessed reads<hr /> </th><th colspan="2">Mapped reads<hr /> </th><th colspan="6">Uniquely mapped reads<hr /> </th></tr><tr><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="4">Coverage<hr /> </th></tr><tr><th>Mean </th><th>1st quartile </th><th>Median </th><th>3rd quartile </th></tr></thead><tbody><tr><td><strong>Omachi</strong> </td><td>297.9 </td><td>22.3 </td><td>260.7 </td><td>18.3 </td><td>257.0 </td><td>98.6 </td><td>185.4 </td><td>71.1 </td><td>41.6 </td><td>27 </td><td>42 </td><td>56 </td></tr><tr><td><strong>Yamada Nishiki</strong> </td><td>218.3 </td><td>19.3 </td><td>184.7 </td><td>15.8 </td><td>181.3 </td><td>98.2 </td><td>140.6 </td><td>76.1 </td><td>37.4 </td><td>30 </td><td>40 </td><td>47 </td></tr><tr><td><strong>Hitomebore</strong> </td><td>221.9 </td><td>16.6 </td><td>202.2 </td><td>14.9 </td><td>198.0 </td><td>97.9 </td><td>144.3 </td><td>71.4 </td><td>34.2 </td><td>18 </td><td>35 </td><td>48 </td></tr><tr><td><strong>Kameji</strong> </td><td>233.8 </td><td>20.7 </td><td>219.4 </td><td>18.7 </td><td>214.0 </td><td>97.6 </td><td>154.3 </td><td>70.4 </td><td>40.5 </td><td>27 </td><td>44 </td><td>54 </td></tr><tr><td><strong>Total</strong> </td><td>971.9 </td><td>79.1 </td><td>866.8 </td><td>67.7 </td><td>850.2 </td><td>– </td><td>624.6 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td><strong>Average</strong> </td><td>243.0 </td><td>19.8 </td><td>216.7 </td><td>16.9 </td><td>212.6 </td><td>98.1 </td><td>156.1 </td><td>72.2 </td><td>38.5 </td><td>25.5 </td><td>40.3 </td><td>51.3 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p>All read counts and lengths are shown in millions and billions, respectively.</p></div></div><div content-id="tblfn4"><div class="fn"><p><sup>a</sup>Mapping rates are calculated as the ratio of the number of the mapped reads against the number of the preprocessed reads.</p></div></div></div></div></p>                            <h3 scrollto-destination=106356875 id="106356875" class="section-title" >3.2. Performance comparison among SNP calling tools with RAD-seq reads in sorghum</h3>
<div class="block-child-p">We compared performances of Heap, Stacks, SAMtools/BCFtools, and GATK with RAD-seq reads in sorghum. At first, in 17 sorghum lines, we conducted RAD-seq and mapped the RAD-seq reads on the reference genome sequences (<a reveal-id="dsx012-T3" data-open="dsx012-T3" class="link link-reveal link-table xref-fig">Table 3</a>). The average RAD-seq read coverage in RAD-regions was around 7.4, which is low for conventional SNP calling using SAMtools/BCFtools or GATK<a reveal-id="dsx012-B12" data-open="dsx012-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a>. We detected 47,901, 53,410, 30,316 and 26,728 SNP sites among the 17 inbred sorghum lines using Heap, Stacks, SAMtools/BCFtools, and GATK with the RAD-seq reads (<a reveal-id="dsx012-F2" data-open="dsx012-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>), respectively. We then calculated sensitivities and PPVs of the SNPs detected from RAD-seq reads by referring to the definitive answer genotypes, as shown in <a reveal-id="dsx012-F2" data-open="dsx012-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>. The results indicated that Heap demonstrated a fairly high sensitivity (0.884) and the highest PPV (0.956) (<a reveal-id="dsx012-F3" data-open="dsx012-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>) among the 4 tools. Additionally, we calculated F-score (<em>F</em>), which is the harmonic mean between sensitivity (<em>S</em>) and PPV (<em>P</em>) (<a reveal-id="dsx012-F2" data-open="dsx012-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>). <a id="E1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">F</mi><mo xmlns="">=</mo><mfrac xmlns=""><mrow><mn>2</mn><mi>S</mi><mi>P</mi></mrow><mrow><mi>S</mi><mo>+</mo><mi>P</mi></mrow></mfrac></math></div></div></div><div content-id="dsx012-T3" class="table-modal"><div class="table-wrap"><div id="dsx012-T3" data-id="dsx012-T3" class="table-wrap-title"><div class="label -label">Table 3</div><div class="caption"><p>Summary of RAD-seq reads and mapping of the reads in Sorghum</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="3">Sample </th><th colspan="2">Raw reads<hr /> </th><th colspan="2">Preprocessed reads<hr /> </th><th colspan="2">Mapped reads<hr /> </th><th colspan="6">Uniquely mapped reads<hr /> </th></tr><tr><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="4">Coverage in RAD-region<hr /> </th></tr><tr><th>Mean </th><th>1st quartile </th><th>Median </th><th>3rd quartile </th></tr></thead><tbody><tr><td><strong>GULUM ABIAD</strong> </td><td>1.6 </td><td>0.2 </td><td>1.5 </td><td>0.1 </td><td>1.4 </td><td>91.4 </td><td>0.6 </td><td>41.8 </td><td>5.8 </td><td>1 </td><td>2 </td><td>4 </td></tr><tr><td><strong>A-6129</strong> </td><td>1.9 </td><td>0.2 </td><td>1.9 </td><td>0.2 </td><td>1.8 </td><td>92.7 </td><td>0.9 </td><td>45.0 </td><td>7.8 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>AGIRA</strong> </td><td>2.2 </td><td>0.2 </td><td>2.1 </td><td>0.2 </td><td>2.0 </td><td>94.3 </td><td>1.0 </td><td>47.8 </td><td>8.1 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>LR 399</strong> </td><td>1.4 </td><td>0.1 </td><td>1.4 </td><td>0.1 </td><td>1.3 </td><td>94.0 </td><td>0.6 </td><td>42.4 </td><td>5.5 </td><td>1 </td><td>2 </td><td>5 </td></tr><tr><td><strong>CRIOLLO CABEZA APRETADA</strong> </td><td>1.5 </td><td>0.2 </td><td>1.4 </td><td>0.1 </td><td>1.3 </td><td>93.8 </td><td>0.6 </td><td>45.2 </td><td>6.2 </td><td>1 </td><td>2 </td><td>6 </td></tr><tr><td><strong>B2AR3043</strong> </td><td>2.7 </td><td>0.3 </td><td>2.6 </td><td>0.3 </td><td>2.5 </td><td>93.6 </td><td>1.3 </td><td>48.1 </td><td>9.2 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>RTx430</strong> </td><td>2.3 </td><td>0.2 </td><td>2.2 </td><td>0.2 </td><td>2.1 </td><td>93.3 </td><td>1.1 </td><td>47.7 </td><td>8.5 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>B3Tx2817</strong> </td><td>1.7 </td><td>0.2 </td><td>1.7 </td><td>0.2 </td><td>1.6 </td><td>94.0 </td><td>0.8 </td><td>48.3 </td><td>6.8 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>ZIRA-EL-SABI</strong> </td><td>2.4 </td><td>0.2 </td><td>2.3 </td><td>0.2 </td><td>2.2 </td><td>94.2 </td><td>1.1 </td><td>48.6 </td><td>7.9 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>ITALIAN</strong> </td><td>2.3 </td><td>0.2 </td><td>2.2 </td><td>0.2 </td><td>2.1 </td><td>91.9 </td><td>1.1 </td><td>48.6 </td><td>9.3 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>CAPRICORN</strong> </td><td>1.4 </td><td>0.2 </td><td>1.4 </td><td>0.1 </td><td>1.3 </td><td>93.1 </td><td>0.7 </td><td>45.8 </td><td>6.4 </td><td>1 </td><td>3 </td><td>6 </td></tr><tr><td><strong>SOR 1</strong> </td><td>2.9 </td><td>0.3 </td><td>2.9 </td><td>0.3 </td><td>2.7 </td><td>92.2 </td><td>1.4 </td><td>47.8 </td><td>9.8 </td><td>1 </td><td>2 </td><td>6 </td></tr><tr><td><strong>15065</strong> </td><td>1.8 </td><td>0.2 </td><td>1.8 </td><td>0.2 </td><td>1.6 </td><td>92.3 </td><td>0.8 </td><td>45.2 </td><td>6.4 </td><td>1 </td><td>2 </td><td>5 </td></tr><tr><td><strong>SAP-155</strong> </td><td>2.2 </td><td>0.2 </td><td>2.1 </td><td>0.2 </td><td>2.0 </td><td>93.7 </td><td>1.0 </td><td>48.0 </td><td>7.9 </td><td>1 </td><td>2 </td><td>6 </td></tr><tr><td><strong>RCV</strong> </td><td>2.0 </td><td>0.2 </td><td>2.0 </td><td>0.2 </td><td>1.9 </td><td>93.9 </td><td>1.0 </td><td>48.8 </td><td>7.4 </td><td>1 </td><td>2 </td><td>6 </td></tr><tr><td><strong>SC 56</strong> </td><td>1.1 </td><td>0.1 </td><td>1.1 </td><td>0.1 </td><td>1.0 </td><td>93.3 </td><td>0.5 </td><td>45.7 </td><td>5.4 </td><td>1 </td><td>2 </td><td>6 </td></tr><tr><td><strong>SIL-05</strong> </td><td>4.1 </td><td>0.4 </td><td>4.0 </td><td>0.4 </td><td>3.5 </td><td>86.4 </td><td>1.9 </td><td>47.5 </td><td>6.9 </td><td>1 </td><td>1 </td><td>3 </td></tr><tr><td><strong>Total</strong> </td><td>35.3 </td><td>3.6 </td><td>34.8 </td><td>3.3 </td><td>32.1 </td><td>– </td><td>16.3 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td><strong>Average</strong> </td><td>2.1 </td><td>0.2 </td><td>2.1 </td><td>0.2 </td><td>1.9 </td><td>92.8 </td><td>1.0 </td><td>46.6 </td><td>7.4 </td><td>1.0 </td><td>2.0 </td><td>6.0 </td></tr></tbody></table></div><div content-id="dsx012-T3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="3">Sample </th><th colspan="2">Raw reads<hr /> </th><th colspan="2">Preprocessed reads<hr /> </th><th colspan="2">Mapped reads<hr /> </th><th colspan="6">Uniquely mapped reads<hr /> </th></tr><tr><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="4">Coverage in RAD-region<hr /> </th></tr><tr><th>Mean </th><th>1st quartile </th><th>Median </th><th>3rd quartile </th></tr></thead><tbody><tr><td><strong>GULUM ABIAD</strong> </td><td>1.6 </td><td>0.2 </td><td>1.5 </td><td>0.1 </td><td>1.4 </td><td>91.4 </td><td>0.6 </td><td>41.8 </td><td>5.8 </td><td>1 </td><td>2 </td><td>4 </td></tr><tr><td><strong>A-6129</strong> </td><td>1.9 </td><td>0.2 </td><td>1.9 </td><td>0.2 </td><td>1.8 </td><td>92.7 </td><td>0.9 </td><td>45.0 </td><td>7.8 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>AGIRA</strong> </td><td>2.2 </td><td>0.2 </td><td>2.1 </td><td>0.2 </td><td>2.0 </td><td>94.3 </td><td>1.0 </td><td>47.8 </td><td>8.1 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>LR 399</strong> </td><td>1.4 </td><td>0.1 </td><td>1.4 </td><td>0.1 </td><td>1.3 </td><td>94.0 </td><td>0.6 </td><td>42.4 </td><td>5.5 </td><td>1 </td><td>2 </td><td>5 </td></tr><tr><td><strong>CRIOLLO CABEZA APRETADA</strong> </td><td>1.5 </td><td>0.2 </td><td>1.4 </td><td>0.1 </td><td>1.3 </td><td>93.8 </td><td>0.6 </td><td>45.2 </td><td>6.2 </td><td>1 </td><td>2 </td><td>6 </td></tr><tr><td><strong>B2AR3043</strong> </td><td>2.7 </td><td>0.3 </td><td>2.6 </td><td>0.3 </td><td>2.5 </td><td>93.6 </td><td>1.3 </td><td>48.1 </td><td>9.2 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>RTx430</strong> </td><td>2.3 </td><td>0.2 </td><td>2.2 </td><td>0.2 </td><td>2.1 </td><td>93.3 </td><td>1.1 </td><td>47.7 </td><td>8.5 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>B3Tx2817</strong> </td><td>1.7 </td><td>0.2 </td><td>1.7 </td><td>0.2 </td><td>1.6 </td><td>94.0 </td><td>0.8 </td><td>48.3 </td><td>6.8 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>ZIRA-EL-SABI</strong> </td><td>2.4 </td><td>0.2 </td><td>2.3 </td><td>0.2 </td><td>2.2 </td><td>94.2 </td><td>1.1 </td><td>48.6 </td><td>7.9 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>ITALIAN</strong> </td><td>2.3 </td><td>0.2 </td><td>2.2 </td><td>0.2 </td><td>2.1 </td><td>91.9 </td><td>1.1 </td><td>48.6 </td><td>9.3 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>CAPRICORN</strong> </td><td>1.4 </td><td>0.2 </td><td>1.4 </td><td>0.1 </td><td>1.3 </td><td>93.1 </td><td>0.7 </td><td>45.8 </td><td>6.4 </td><td>1 </td><td>3 </td><td>6 </td></tr><tr><td><strong>SOR 1</strong> </td><td>2.9 </td><td>0.3 </td><td>2.9 </td><td>0.3 </td><td>2.7 </td><td>92.2 </td><td>1.4 </td><td>47.8 </td><td>9.8 </td><td>1 </td><td>2 </td><td>6 </td></tr><tr><td><strong>15065</strong> </td><td>1.8 </td><td>0.2 </td><td>1.8 </td><td>0.2 </td><td>1.6 </td><td>92.3 </td><td>0.8 </td><td>45.2 </td><td>6.4 </td><td>1 </td><td>2 </td><td>5 </td></tr><tr><td><strong>SAP-155</strong> </td><td>2.2 </td><td>0.2 </td><td>2.1 </td><td>0.2 </td><td>2.0 </td><td>93.7 </td><td>1.0 </td><td>48.0 </td><td>7.9 </td><td>1 </td><td>2 </td><td>6 </td></tr><tr><td><strong>RCV</strong> </td><td>2.0 </td><td>0.2 </td><td>2.0 </td><td>0.2 </td><td>1.9 </td><td>93.9 </td><td>1.0 </td><td>48.8 </td><td>7.4 </td><td>1 </td><td>2 </td><td>6 </td></tr><tr><td><strong>SC 56</strong> </td><td>1.1 </td><td>0.1 </td><td>1.1 </td><td>0.1 </td><td>1.0 </td><td>93.3 </td><td>0.5 </td><td>45.7 </td><td>5.4 </td><td>1 </td><td>2 </td><td>6 </td></tr><tr><td><strong>SIL-05</strong> </td><td>4.1 </td><td>0.4 </td><td>4.0 </td><td>0.4 </td><td>3.5 </td><td>86.4 </td><td>1.9 </td><td>47.5 </td><td>6.9 </td><td>1 </td><td>1 </td><td>3 </td></tr><tr><td><strong>Total</strong> </td><td>35.3 </td><td>3.6 </td><td>34.8 </td><td>3.3 </td><td>32.1 </td><td>– </td><td>16.3 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td><strong>Average</strong> </td><td>2.1 </td><td>0.2 </td><td>2.1 </td><td>0.2 </td><td>1.9 </td><td>92.8 </td><td>1.0 </td><td>46.6 </td><td>7.4 </td><td>1.0 </td><td>2.0 </td><td>6.0 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn5"><div class="fn"><p>All read counts and lengths are shown in millions and billions, respectively.</p></div></div><div content-id="tblfn6"><div class="fn"><p><sup>a</sup>Mapping rates are calculated as the ratio of the number of the mapped reads against the number of the preprocessed reads.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx012-T3" data-id="dsx012-T3" class="table-wrap-title"><div class="label -label">Table 3</div><div class="caption"><p>Summary of RAD-seq reads and mapping of the reads in Sorghum</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="3">Sample </th><th colspan="2">Raw reads<hr /> </th><th colspan="2">Preprocessed reads<hr /> </th><th colspan="2">Mapped reads<hr /> </th><th colspan="6">Uniquely mapped reads<hr /> </th></tr><tr><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="4">Coverage in RAD-region<hr /> </th></tr><tr><th>Mean </th><th>1st quartile </th><th>Median </th><th>3rd quartile </th></tr></thead><tbody><tr><td><strong>GULUM ABIAD</strong> </td><td>1.6 </td><td>0.2 </td><td>1.5 </td><td>0.1 </td><td>1.4 </td><td>91.4 </td><td>0.6 </td><td>41.8 </td><td>5.8 </td><td>1 </td><td>2 </td><td>4 </td></tr><tr><td><strong>A-6129</strong> </td><td>1.9 </td><td>0.2 </td><td>1.9 </td><td>0.2 </td><td>1.8 </td><td>92.7 </td><td>0.9 </td><td>45.0 </td><td>7.8 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>AGIRA</strong> </td><td>2.2 </td><td>0.2 </td><td>2.1 </td><td>0.2 </td><td>2.0 </td><td>94.3 </td><td>1.0 </td><td>47.8 </td><td>8.1 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>LR 399</strong> </td><td>1.4 </td><td>0.1 </td><td>1.4 </td><td>0.1 </td><td>1.3 </td><td>94.0 </td><td>0.6 </td><td>42.4 </td><td>5.5 </td><td>1 </td><td>2 </td><td>5 </td></tr><tr><td><strong>CRIOLLO CABEZA APRETADA</strong> </td><td>1.5 </td><td>0.2 </td><td>1.4 </td><td>0.1 </td><td>1.3 </td><td>93.8 </td><td>0.6 </td><td>45.2 </td><td>6.2 </td><td>1 </td><td>2 </td><td>6 </td></tr><tr><td><strong>B2AR3043</strong> </td><td>2.7 </td><td>0.3 </td><td>2.6 </td><td>0.3 </td><td>2.5 </td><td>93.6 </td><td>1.3 </td><td>48.1 </td><td>9.2 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>RTx430</strong> </td><td>2.3 </td><td>0.2 </td><td>2.2 </td><td>0.2 </td><td>2.1 </td><td>93.3 </td><td>1.1 </td><td>47.7 </td><td>8.5 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>B3Tx2817</strong> </td><td>1.7 </td><td>0.2 </td><td>1.7 </td><td>0.2 </td><td>1.6 </td><td>94.0 </td><td>0.8 </td><td>48.3 </td><td>6.8 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>ZIRA-EL-SABI</strong> </td><td>2.4 </td><td>0.2 </td><td>2.3 </td><td>0.2 </td><td>2.2 </td><td>94.2 </td><td>1.1 </td><td>48.6 </td><td>7.9 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>ITALIAN</strong> </td><td>2.3 </td><td>0.2 </td><td>2.2 </td><td>0.2 </td><td>2.1 </td><td>91.9 </td><td>1.1 </td><td>48.6 </td><td>9.3 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>CAPRICORN</strong> </td><td>1.4 </td><td>0.2 </td><td>1.4 </td><td>0.1 </td><td>1.3 </td><td>93.1 </td><td>0.7 </td><td>45.8 </td><td>6.4 </td><td>1 </td><td>3 </td><td>6 </td></tr><tr><td><strong>SOR 1</strong> </td><td>2.9 </td><td>0.3 </td><td>2.9 </td><td>0.3 </td><td>2.7 </td><td>92.2 </td><td>1.4 </td><td>47.8 </td><td>9.8 </td><td>1 </td><td>2 </td><td>6 </td></tr><tr><td><strong>15065</strong> </td><td>1.8 </td><td>0.2 </td><td>1.8 </td><td>0.2 </td><td>1.6 </td><td>92.3 </td><td>0.8 </td><td>45.2 </td><td>6.4 </td><td>1 </td><td>2 </td><td>5 </td></tr><tr><td><strong>SAP-155</strong> </td><td>2.2 </td><td>0.2 </td><td>2.1 </td><td>0.2 </td><td>2.0 </td><td>93.7 </td><td>1.0 </td><td>48.0 </td><td>7.9 </td><td>1 </td><td>2 </td><td>6 </td></tr><tr><td><strong>RCV</strong> </td><td>2.0 </td><td>0.2 </td><td>2.0 </td><td>0.2 </td><td>1.9 </td><td>93.9 </td><td>1.0 </td><td>48.8 </td><td>7.4 </td><td>1 </td><td>2 </td><td>6 </td></tr><tr><td><strong>SC 56</strong> </td><td>1.1 </td><td>0.1 </td><td>1.1 </td><td>0.1 </td><td>1.0 </td><td>93.3 </td><td>0.5 </td><td>45.7 </td><td>5.4 </td><td>1 </td><td>2 </td><td>6 </td></tr><tr><td><strong>SIL-05</strong> </td><td>4.1 </td><td>0.4 </td><td>4.0 </td><td>0.4 </td><td>3.5 </td><td>86.4 </td><td>1.9 </td><td>47.5 </td><td>6.9 </td><td>1 </td><td>1 </td><td>3 </td></tr><tr><td><strong>Total</strong> </td><td>35.3 </td><td>3.6 </td><td>34.8 </td><td>3.3 </td><td>32.1 </td><td>– </td><td>16.3 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td><strong>Average</strong> </td><td>2.1 </td><td>0.2 </td><td>2.1 </td><td>0.2 </td><td>1.9 </td><td>92.8 </td><td>1.0 </td><td>46.6 </td><td>7.4 </td><td>1.0 </td><td>2.0 </td><td>6.0 </td></tr></tbody></table></div><div content-id="dsx012-T3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="3">Sample </th><th colspan="2">Raw reads<hr /> </th><th colspan="2">Preprocessed reads<hr /> </th><th colspan="2">Mapped reads<hr /> </th><th colspan="6">Uniquely mapped reads<hr /> </th></tr><tr><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn6" data-open="tblfn6" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="4">Coverage in RAD-region<hr /> </th></tr><tr><th>Mean </th><th>1st quartile </th><th>Median </th><th>3rd quartile </th></tr></thead><tbody><tr><td><strong>GULUM ABIAD</strong> </td><td>1.6 </td><td>0.2 </td><td>1.5 </td><td>0.1 </td><td>1.4 </td><td>91.4 </td><td>0.6 </td><td>41.8 </td><td>5.8 </td><td>1 </td><td>2 </td><td>4 </td></tr><tr><td><strong>A-6129</strong> </td><td>1.9 </td><td>0.2 </td><td>1.9 </td><td>0.2 </td><td>1.8 </td><td>92.7 </td><td>0.9 </td><td>45.0 </td><td>7.8 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>AGIRA</strong> </td><td>2.2 </td><td>0.2 </td><td>2.1 </td><td>0.2 </td><td>2.0 </td><td>94.3 </td><td>1.0 </td><td>47.8 </td><td>8.1 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>LR 399</strong> </td><td>1.4 </td><td>0.1 </td><td>1.4 </td><td>0.1 </td><td>1.3 </td><td>94.0 </td><td>0.6 </td><td>42.4 </td><td>5.5 </td><td>1 </td><td>2 </td><td>5 </td></tr><tr><td><strong>CRIOLLO CABEZA APRETADA</strong> </td><td>1.5 </td><td>0.2 </td><td>1.4 </td><td>0.1 </td><td>1.3 </td><td>93.8 </td><td>0.6 </td><td>45.2 </td><td>6.2 </td><td>1 </td><td>2 </td><td>6 </td></tr><tr><td><strong>B2AR3043</strong> </td><td>2.7 </td><td>0.3 </td><td>2.6 </td><td>0.3 </td><td>2.5 </td><td>93.6 </td><td>1.3 </td><td>48.1 </td><td>9.2 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>RTx430</strong> </td><td>2.3 </td><td>0.2 </td><td>2.2 </td><td>0.2 </td><td>2.1 </td><td>93.3 </td><td>1.1 </td><td>47.7 </td><td>8.5 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>B3Tx2817</strong> </td><td>1.7 </td><td>0.2 </td><td>1.7 </td><td>0.2 </td><td>1.6 </td><td>94.0 </td><td>0.8 </td><td>48.3 </td><td>6.8 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>ZIRA-EL-SABI</strong> </td><td>2.4 </td><td>0.2 </td><td>2.3 </td><td>0.2 </td><td>2.2 </td><td>94.2 </td><td>1.1 </td><td>48.6 </td><td>7.9 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>ITALIAN</strong> </td><td>2.3 </td><td>0.2 </td><td>2.2 </td><td>0.2 </td><td>2.1 </td><td>91.9 </td><td>1.1 </td><td>48.6 </td><td>9.3 </td><td>1 </td><td>2 </td><td>7 </td></tr><tr><td><strong>CAPRICORN</strong> </td><td>1.4 </td><td>0.2 </td><td>1.4 </td><td>0.1 </td><td>1.3 </td><td>93.1 </td><td>0.7 </td><td>45.8 </td><td>6.4 </td><td>1 </td><td>3 </td><td>6 </td></tr><tr><td><strong>SOR 1</strong> </td><td>2.9 </td><td>0.3 </td><td>2.9 </td><td>0.3 </td><td>2.7 </td><td>92.2 </td><td>1.4 </td><td>47.8 </td><td>9.8 </td><td>1 </td><td>2 </td><td>6 </td></tr><tr><td><strong>15065</strong> </td><td>1.8 </td><td>0.2 </td><td>1.8 </td><td>0.2 </td><td>1.6 </td><td>92.3 </td><td>0.8 </td><td>45.2 </td><td>6.4 </td><td>1 </td><td>2 </td><td>5 </td></tr><tr><td><strong>SAP-155</strong> </td><td>2.2 </td><td>0.2 </td><td>2.1 </td><td>0.2 </td><td>2.0 </td><td>93.7 </td><td>1.0 </td><td>48.0 </td><td>7.9 </td><td>1 </td><td>2 </td><td>6 </td></tr><tr><td><strong>RCV</strong> </td><td>2.0 </td><td>0.2 </td><td>2.0 </td><td>0.2 </td><td>1.9 </td><td>93.9 </td><td>1.0 </td><td>48.8 </td><td>7.4 </td><td>1 </td><td>2 </td><td>6 </td></tr><tr><td><strong>SC 56</strong> </td><td>1.1 </td><td>0.1 </td><td>1.1 </td><td>0.1 </td><td>1.0 </td><td>93.3 </td><td>0.5 </td><td>45.7 </td><td>5.4 </td><td>1 </td><td>2 </td><td>6 </td></tr><tr><td><strong>SIL-05</strong> </td><td>4.1 </td><td>0.4 </td><td>4.0 </td><td>0.4 </td><td>3.5 </td><td>86.4 </td><td>1.9 </td><td>47.5 </td><td>6.9 </td><td>1 </td><td>1 </td><td>3 </td></tr><tr><td><strong>Total</strong> </td><td>35.3 </td><td>3.6 </td><td>34.8 </td><td>3.3 </td><td>32.1 </td><td>– </td><td>16.3 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td><strong>Average</strong> </td><td>2.1 </td><td>0.2 </td><td>2.1 </td><td>0.2 </td><td>1.9 </td><td>92.8 </td><td>1.0 </td><td>46.6 </td><td>7.4 </td><td>1.0 </td><td>2.0 </td><td>6.0 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn5"><div class="fn"><p>All read counts and lengths are shown in millions and billions, respectively.</p></div></div><div content-id="tblfn6"><div class="fn"><p><sup>a</sup>Mapping rates are calculated as the ratio of the number of the mapped reads against the number of the preprocessed reads.</p></div></div></div></div>The <em>F</em>-score is commonly used in the field of statistical classification and employed as a measure of a test’s accuracy. We found that Heap exhibited the significantly highest <em>F</em>-score (0.918) compared to the other tools according to the Tukey-Kramer honestly significant difference (HSD) test (<a reveal-id="dsx012-F3" data-open="dsx012-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). In SAMtools/BCFtools and GATK analysis, <em>F</em>-scores were low due to the low sensitivities of these tools. Here it should be noted that, in SAMtools/BCFtools and GATK analyses, not negligible amounts of SNPs with lower GQ (&lt;13) had been eliminated by the filtering on VCF files in advance. This could lead to an underestimation of sensitivities for the 2 tools.</div>                        <a id="106356877" scrollto-destination="106356877"></a>
<div id="dsx012-F3" data-id="dsx012-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/m_dsx012f3.png?Expires=2147483647&Signature=rXRUldCWHNBzjbbD9ukMipH1c5hzbs3Y2AbNNCiSxH45D0cbPfPVIjKwivqSgfEDnyzn684KTptbm7sCoM68rc5vFuzzR7Zmd4-KOXilrysRYCGxYW1L6pZMCfSqkHfzAhq1l2TqNkbQByLubtQdCKzNofwrp~SfvbHxEKljbbQsYi4AkHm1QfZO-VuO667ydCGtfW16JWLIUYh4f8vzjOMbddkxr9LijA5nKjUTQqKGlH4~BGXBHYrlHWNoCGOo0H1jANDPMugAIpGU-QqIJqRUifXnCgXqko~Fh2FpASav-RVJ4Lto2WrevAR7in9sIXbdMDh1yzo5F1-Dd5bZ5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Performance comparison among SNP calling tools with RAD-seq reads in 17 inbred sorghum lines. Mean values of sensitivities (left), positive predictive values (PPVs) (center), and F-scores (right) of SNP calling by Heap, Stacks, SAMtools, and GATK from RAD-seq reads in 17 inbred sorghum lines are shown. Statistical analysis was performed using the Tukey-Kramer HSD test. Letters above the bars indicate groups that are significantly different (P &lt; 0.05)." path-from-xml="dsx012f3.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356877/dsx012f3.png" path-from-xml="dsx012f3.png" target="_blank">View large</a><a section="106356877" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/dsx012f3.png?Expires=2147483647&Signature=iamzAnqXj-aq5cXApc5v~xn7J9ervOXJB2CaoZ8qq5YR4i-MnKFHXE4ptQwg8oymmO2LYSDWmdpdEbxA3tsc7LQXc~kUL4d3Tx5Bt5xR4XuOHS9X58Hsklo6--keQ3E6DRogiOUeRD8VTWm9AwrGxs5rXhJ6r83yvUa7gLQ8sPP7F2qpiZTXQCR6oss2CefXzAVidP5H~eZZEnyzozwS7HjQcO9kfzCbnmszw9pocj2WZeZ3atZTGoG2TJ16NPXZfo4oT6aiJJuULlL6BVTLvea~BG1Ve7yFdAtzpO77-la7lwwpoKzKn9GrA~lLkyKGeoW7MNzrrOv90NCPQAYfsA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106356877&ar=3815695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx012f3.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Performance comparison among SNP calling tools with RAD-seq reads in 17 inbred sorghum lines. Mean values of sensitivities (left), positive predictive values (PPVs) (center), and <em>F</em>-scores (right) of SNP calling by Heap, Stacks, SAMtools, and GATK from RAD-seq reads in 17 inbred sorghum lines are shown. Statistical analysis was performed using the Tukey-Kramer HSD test. Letters above the bars indicate groups that are significantly different (<em>P &lt;</em> 0.05).</p></div></div><div content-id="dsx012-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/m_dsx012f3.png?Expires=2147483647&Signature=rXRUldCWHNBzjbbD9ukMipH1c5hzbs3Y2AbNNCiSxH45D0cbPfPVIjKwivqSgfEDnyzn684KTptbm7sCoM68rc5vFuzzR7Zmd4-KOXilrysRYCGxYW1L6pZMCfSqkHfzAhq1l2TqNkbQByLubtQdCKzNofwrp~SfvbHxEKljbbQsYi4AkHm1QfZO-VuO667ydCGtfW16JWLIUYh4f8vzjOMbddkxr9LijA5nKjUTQqKGlH4~BGXBHYrlHWNoCGOo0H1jANDPMugAIpGU-QqIJqRUifXnCgXqko~Fh2FpASav-RVJ4Lto2WrevAR7in9sIXbdMDh1yzo5F1-Dd5bZ5A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Performance comparison among SNP calling tools with RAD-seq reads in 17 inbred sorghum lines. Mean values of sensitivities (left), positive predictive values (PPVs) (center), and F-scores (right) of SNP calling by Heap, Stacks, SAMtools, and GATK from RAD-seq reads in 17 inbred sorghum lines are shown. Statistical analysis was performed using the Tukey-Kramer HSD test. Letters above the bars indicate groups that are significantly different (P &lt; 0.05)." path-from-xml="dsx012f3.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356877/dsx012f3.png" path-from-xml="dsx012f3.png" target="_blank">View large</a><a section="106356877" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/dsx012f3.png?Expires=2147483647&Signature=iamzAnqXj-aq5cXApc5v~xn7J9ervOXJB2CaoZ8qq5YR4i-MnKFHXE4ptQwg8oymmO2LYSDWmdpdEbxA3tsc7LQXc~kUL4d3Tx5Bt5xR4XuOHS9X58Hsklo6--keQ3E6DRogiOUeRD8VTWm9AwrGxs5rXhJ6r83yvUa7gLQ8sPP7F2qpiZTXQCR6oss2CefXzAVidP5H~eZZEnyzozwS7HjQcO9kfzCbnmszw9pocj2WZeZ3atZTGoG2TJ16NPXZfo4oT6aiJJuULlL6BVTLvea~BG1Ve7yFdAtzpO77-la7lwwpoKzKn9GrA~lLkyKGeoW7MNzrrOv90NCPQAYfsA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106356877&ar=3815695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx012f3.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Performance comparison among SNP calling tools with RAD-seq reads in 17 inbred sorghum lines. Mean values of sensitivities (left), positive predictive values (PPVs) (center), and <em>F</em>-scores (right) of SNP calling by Heap, Stacks, SAMtools, and GATK from RAD-seq reads in 17 inbred sorghum lines are shown. Statistical analysis was performed using the Tukey-Kramer HSD test. Letters above the bars indicate groups that are significantly different (<em>P &lt;</em> 0.05).</p></div></div><p>With the aim of conducting a fair benchmarking for each tool, we also conducted GQ filtering-free analyses of SAMtools/BCFtools and GATK. The results showed that their sensitivities increased (SAMtools/BCFtools: 0.925, GATK: 0.686), but PPVs decreased (SAMtools/BCFtools: 0.843) or did not change considerably (GATK: 0.937) in the GQ filtering-free condition (<a reveal-id="dsx012-F3" data-open="dsx012-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). Importantly, Heap still showed the highest <em>F</em>-score (0.918) in this benchmarking, while the <em>F</em>-scores of GQ filtering-free SAMtools/BCFtools and GATK were 0.882 and 0.792, respectively. These results demonstrated that Heap is the best performance tool for SNP calling from RAD-seq reads, showing sufficient sensitivity and accuracy among the 17 inbred sorghum lines.</p>                            <h3 scrollto-destination=106356879 id="106356879" class="section-title" >3.3. Performance comparison among SNP calling tools with RAD-seq reads in rice</h3>
<p>To confirm the advantages of Heap with multiple datasets, we also compared performances of the 4 tools in 4 inbred lines of rice, Omachi, Yamada Nishiki, Hitomebore, and Kameji. Prior to the SNP calling, we conducted RAD-seq and reference mapping of the RAD-seq reads (<a reveal-id="dsx012-T4" data-open="dsx012-T4" class="link link-reveal link-table xref-fig">Table 4</a>). The average RAD-seq read coverage in RAD-regions was low (9.7) for conventional SNP calling. We detected 842, 1,021, 437 and 242 SNP sites among the 4 rice lines using Heap, Stacks, SAMtools/BCFtools and GATK with the RAD-seq reads, respectively. Again, we calculated sensitivities, PPVs, and <em>F</em>-scores as mentioned above. Heap showed a high sensitivity (0.803) (<a reveal-id="dsx012-F4" data-open="dsx012-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>) with a sufficiently high PPV (0.722). Here we would mention that although Stacks showed the highest sensitivity (0.843), the sensitivities of Heap and Stacks were not significantly different, according to the Tukey-Kramer HSD test (<a reveal-id="dsx012-F4" data-open="dsx012-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). Additionally, the PPV of SNPs detected by Heap (0.722) was higher than that by Stacks (0.553). Importantly, the <em>F</em>-score of Heap (0.757) was the highest among the tools examined. These results indicate that Heap calls SNPs with a high sensitivity and high PPV with a reasonable balance between them in rice samples, too. <div content-id="dsx012-T4" class="table-modal"><div class="table-wrap"><div id="dsx012-T4" data-id="dsx012-T4" class="table-wrap-title"><div class="label -label">Table 4</div><div class="caption"><p>Summary of RAD-seq reads and mapping of the reads in rice</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="3">Sample </th><th colspan="2">Raw reads<hr /> </th><th colspan="2">Preprocessed reads<hr /> </th><th colspan="2">Mapped reads<hr /> </th><th colspan="6">Uniquely mapped reads<hr /> </th></tr><tr><th rowspan="2">Count  (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="4">Coverage in RAD-region (average)<hr /> </th></tr><tr><th>Mean </th><th>1st quartile </th><th>Median </th><th>3rd quartile </th></tr></thead><tbody><tr><td><strong>Omachi</strong> </td><td>3.6 </td><td>0.2 </td><td>3.6 </td><td>0.2 </td><td>3.5 </td><td>98.4 </td><td>2.4 </td><td>67.3 </td><td>11.2 </td><td>1 </td><td>2 </td><td>4 </td></tr><tr><td><strong>Yamada Nishiki</strong> </td><td>4.2 </td><td>0.2 </td><td>4.2 </td><td>0.2 </td><td>4.1 </td><td>98.1 </td><td>2.8 </td><td>66.9 </td><td>11.7 </td><td>1 </td><td>1 </td><td>4 </td></tr><tr><td><strong>Hitomebore</strong> </td><td>0.7 </td><td>0.0 </td><td>0.7 </td><td>0.0 </td><td>0.7 </td><td>98.7 </td><td>0.4 </td><td>62.0 </td><td>6.5 </td><td>1 </td><td>2 </td><td>4 </td></tr><tr><td><strong>Kameji</strong> </td><td>1.9 </td><td>0.1 </td><td>1.9 </td><td>0.1 </td><td>1.8 </td><td>98.4 </td><td>1.3 </td><td>67.2 </td><td>9.2 </td><td>1 </td><td>2 </td><td>4 </td></tr><tr><td><strong>Total</strong> </td><td>10.3 </td><td>0.5 </td><td>10.3 </td><td>0.5 </td><td>10.1 </td><td>– </td><td>6.9 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td><strong>Average</strong> </td><td>2.6 </td><td>0.1 </td><td>2.6 </td><td>0.1 </td><td>2.5 </td><td>98.4 </td><td>1.7 </td><td>65.9 </td><td>9.7 </td><td>1.0 </td><td>1.8 </td><td>4.0 </td></tr></tbody></table></div><div content-id="dsx012-T4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="3">Sample </th><th colspan="2">Raw reads<hr /> </th><th colspan="2">Preprocessed reads<hr /> </th><th colspan="2">Mapped reads<hr /> </th><th colspan="6">Uniquely mapped reads<hr /> </th></tr><tr><th rowspan="2">Count  (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="4">Coverage in RAD-region (average)<hr /> </th></tr><tr><th>Mean </th><th>1st quartile </th><th>Median </th><th>3rd quartile </th></tr></thead><tbody><tr><td><strong>Omachi</strong> </td><td>3.6 </td><td>0.2 </td><td>3.6 </td><td>0.2 </td><td>3.5 </td><td>98.4 </td><td>2.4 </td><td>67.3 </td><td>11.2 </td><td>1 </td><td>2 </td><td>4 </td></tr><tr><td><strong>Yamada Nishiki</strong> </td><td>4.2 </td><td>0.2 </td><td>4.2 </td><td>0.2 </td><td>4.1 </td><td>98.1 </td><td>2.8 </td><td>66.9 </td><td>11.7 </td><td>1 </td><td>1 </td><td>4 </td></tr><tr><td><strong>Hitomebore</strong> </td><td>0.7 </td><td>0.0 </td><td>0.7 </td><td>0.0 </td><td>0.7 </td><td>98.7 </td><td>0.4 </td><td>62.0 </td><td>6.5 </td><td>1 </td><td>2 </td><td>4 </td></tr><tr><td><strong>Kameji</strong> </td><td>1.9 </td><td>0.1 </td><td>1.9 </td><td>0.1 </td><td>1.8 </td><td>98.4 </td><td>1.3 </td><td>67.2 </td><td>9.2 </td><td>1 </td><td>2 </td><td>4 </td></tr><tr><td><strong>Total</strong> </td><td>10.3 </td><td>0.5 </td><td>10.3 </td><td>0.5 </td><td>10.1 </td><td>– </td><td>6.9 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td><strong>Average</strong> </td><td>2.6 </td><td>0.1 </td><td>2.6 </td><td>0.1 </td><td>2.5 </td><td>98.4 </td><td>1.7 </td><td>65.9 </td><td>9.7 </td><td>1.0 </td><td>1.8 </td><td>4.0 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn7"><div class="fn"><p>All read counts and lengths are shown in millions and billions, respectively.</p></div></div><div content-id="tblfn8"><div class="fn"><p><sup>a</sup>Mapping rates are calculated as the ratio of the number of the mapped reads against the number of the preprocessed reads.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx012-T4" data-id="dsx012-T4" class="table-wrap-title"><div class="label -label">Table 4</div><div class="caption"><p>Summary of RAD-seq reads and mapping of the reads in rice</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="3">Sample </th><th colspan="2">Raw reads<hr /> </th><th colspan="2">Preprocessed reads<hr /> </th><th colspan="2">Mapped reads<hr /> </th><th colspan="6">Uniquely mapped reads<hr /> </th></tr><tr><th rowspan="2">Count  (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="4">Coverage in RAD-region (average)<hr /> </th></tr><tr><th>Mean </th><th>1st quartile </th><th>Median </th><th>3rd quartile </th></tr></thead><tbody><tr><td><strong>Omachi</strong> </td><td>3.6 </td><td>0.2 </td><td>3.6 </td><td>0.2 </td><td>3.5 </td><td>98.4 </td><td>2.4 </td><td>67.3 </td><td>11.2 </td><td>1 </td><td>2 </td><td>4 </td></tr><tr><td><strong>Yamada Nishiki</strong> </td><td>4.2 </td><td>0.2 </td><td>4.2 </td><td>0.2 </td><td>4.1 </td><td>98.1 </td><td>2.8 </td><td>66.9 </td><td>11.7 </td><td>1 </td><td>1 </td><td>4 </td></tr><tr><td><strong>Hitomebore</strong> </td><td>0.7 </td><td>0.0 </td><td>0.7 </td><td>0.0 </td><td>0.7 </td><td>98.7 </td><td>0.4 </td><td>62.0 </td><td>6.5 </td><td>1 </td><td>2 </td><td>4 </td></tr><tr><td><strong>Kameji</strong> </td><td>1.9 </td><td>0.1 </td><td>1.9 </td><td>0.1 </td><td>1.8 </td><td>98.4 </td><td>1.3 </td><td>67.2 </td><td>9.2 </td><td>1 </td><td>2 </td><td>4 </td></tr><tr><td><strong>Total</strong> </td><td>10.3 </td><td>0.5 </td><td>10.3 </td><td>0.5 </td><td>10.1 </td><td>– </td><td>6.9 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td><strong>Average</strong> </td><td>2.6 </td><td>0.1 </td><td>2.6 </td><td>0.1 </td><td>2.5 </td><td>98.4 </td><td>1.7 </td><td>65.9 </td><td>9.7 </td><td>1.0 </td><td>1.8 </td><td>4.0 </td></tr></tbody></table></div><div content-id="dsx012-T4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="3">Sample </th><th colspan="2">Raw reads<hr /> </th><th colspan="2">Preprocessed reads<hr /> </th><th colspan="2">Mapped reads<hr /> </th><th colspan="6">Uniquely mapped reads<hr /> </th></tr><tr><th rowspan="2">Count  (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Total length (Gb) </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th rowspan="2">Count (×10<sup>6</sup>) </th><th rowspan="2">Rate (%)<a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th colspan="4">Coverage in RAD-region (average)<hr /> </th></tr><tr><th>Mean </th><th>1st quartile </th><th>Median </th><th>3rd quartile </th></tr></thead><tbody><tr><td><strong>Omachi</strong> </td><td>3.6 </td><td>0.2 </td><td>3.6 </td><td>0.2 </td><td>3.5 </td><td>98.4 </td><td>2.4 </td><td>67.3 </td><td>11.2 </td><td>1 </td><td>2 </td><td>4 </td></tr><tr><td><strong>Yamada Nishiki</strong> </td><td>4.2 </td><td>0.2 </td><td>4.2 </td><td>0.2 </td><td>4.1 </td><td>98.1 </td><td>2.8 </td><td>66.9 </td><td>11.7 </td><td>1 </td><td>1 </td><td>4 </td></tr><tr><td><strong>Hitomebore</strong> </td><td>0.7 </td><td>0.0 </td><td>0.7 </td><td>0.0 </td><td>0.7 </td><td>98.7 </td><td>0.4 </td><td>62.0 </td><td>6.5 </td><td>1 </td><td>2 </td><td>4 </td></tr><tr><td><strong>Kameji</strong> </td><td>1.9 </td><td>0.1 </td><td>1.9 </td><td>0.1 </td><td>1.8 </td><td>98.4 </td><td>1.3 </td><td>67.2 </td><td>9.2 </td><td>1 </td><td>2 </td><td>4 </td></tr><tr><td><strong>Total</strong> </td><td>10.3 </td><td>0.5 </td><td>10.3 </td><td>0.5 </td><td>10.1 </td><td>– </td><td>6.9 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td><strong>Average</strong> </td><td>2.6 </td><td>0.1 </td><td>2.6 </td><td>0.1 </td><td>2.5 </td><td>98.4 </td><td>1.7 </td><td>65.9 </td><td>9.7 </td><td>1.0 </td><td>1.8 </td><td>4.0 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn7"><div class="fn"><p>All read counts and lengths are shown in millions and billions, respectively.</p></div></div><div content-id="tblfn8"><div class="fn"><p><sup>a</sup>Mapping rates are calculated as the ratio of the number of the mapped reads against the number of the preprocessed reads.</p></div></div></div></div></p>                        <a id="106356881" scrollto-destination="106356881"></a>
<div id="dsx012-F4" data-id="dsx012-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/m_dsx012f4.png?Expires=2147483647&Signature=b584t4UqEZijHha8EQTxQxdMkeXtXzJ2bMaSASPc~aEy2pDr0bAjCFV6Gd1L-b5hsiJNSxb4pSlXl0jlf~Symi5-Azp54VUMSHy~VZBJTaMV5E8qaYDGGdsBYi2cspaq~VGeG6DGL5nLytFqxkNaV6Hm0QPSN2EpLkRAiNKYxZPpDNx0WmYLM0kKlzm3ZSFCi1Ib7HU8UDW71M1cSb4ghhyf3rU-j2RFPjsJojBe8xbqaQQ3c7z0QSq1DpRybEmRnVy4m8HUzlRDp7c~46uA18~GfUhlQyN2WCsYT6SRRLdOTctOixN-SHNnPcIrc--n~1SCapqsGRUvr-C-ryMOfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Performance comparison among SNP calling tools with RAD-seq reads in 4 inbred rice lines. Mean values of sensitivities (left), PPVs (center), and F-scores (right) of SNP calling by Heap, Stacks, SAMtools, and GATK from RAD-seq reads in 4 inbred rice lines are shown. Letters above the bars indicate groups that are significantly different (P &lt; 0.05), according to the Tukey-Kramer HSD test." path-from-xml="dsx012f4.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356881/dsx012f4.png" path-from-xml="dsx012f4.png" target="_blank">View large</a><a section="106356881" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/dsx012f4.png?Expires=2147483647&Signature=cXG34grH5TupW9DjcQvjllJiLf29Bw8Nnvx-Bvuz2x4V7uilbCxaUyopf~SP42u77HIGjDYCTXCKT~i5urdHfTURbtOLSWBZsJ3PNLUT-MMpw40t7yyAl1QCSmEFqEE3TTVZqJ4S0bzQaCMyoOQMIXk1g8T5fPBkQnzNCzEZSuhOsujMQ6G9ttr4Qh2IS1bDyC7qgg0tQ5heYHR9qRYjyC4GdhbKC0v~6Jnaifm7lDZaL0pieFiIs0EhEHJ0xWHKUgxALKU6ZUFe4jrSVxKp~I~Qv3KZZ5DcQdSYPDMXPTmAWa3pWKpatfvYbBbg7Ms7z5iOaaP0rwNQrFQXzvHpsw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106356881&ar=3815695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx012f4.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Performance comparison among SNP calling tools with RAD-seq reads in 4 inbred rice lines. Mean values of sensitivities (left), PPVs (center), and <em>F</em>-scores (right) of SNP calling by Heap, Stacks, SAMtools, and GATK from RAD-seq reads in 4 inbred rice lines are shown. Letters above the bars indicate groups that are significantly different (<em>P &lt;</em> 0.05), according to the Tukey-Kramer HSD test.</p></div></div><div content-id="dsx012-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/m_dsx012f4.png?Expires=2147483647&Signature=b584t4UqEZijHha8EQTxQxdMkeXtXzJ2bMaSASPc~aEy2pDr0bAjCFV6Gd1L-b5hsiJNSxb4pSlXl0jlf~Symi5-Azp54VUMSHy~VZBJTaMV5E8qaYDGGdsBYi2cspaq~VGeG6DGL5nLytFqxkNaV6Hm0QPSN2EpLkRAiNKYxZPpDNx0WmYLM0kKlzm3ZSFCi1Ib7HU8UDW71M1cSb4ghhyf3rU-j2RFPjsJojBe8xbqaQQ3c7z0QSq1DpRybEmRnVy4m8HUzlRDp7c~46uA18~GfUhlQyN2WCsYT6SRRLdOTctOixN-SHNnPcIrc--n~1SCapqsGRUvr-C-ryMOfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Performance comparison among SNP calling tools with RAD-seq reads in 4 inbred rice lines. Mean values of sensitivities (left), PPVs (center), and F-scores (right) of SNP calling by Heap, Stacks, SAMtools, and GATK from RAD-seq reads in 4 inbred rice lines are shown. Letters above the bars indicate groups that are significantly different (P &lt; 0.05), according to the Tukey-Kramer HSD test." path-from-xml="dsx012f4.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106356881/dsx012f4.png" path-from-xml="dsx012f4.png" target="_blank">View large</a><a section="106356881" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/dsx012f4.png?Expires=2147483647&Signature=cXG34grH5TupW9DjcQvjllJiLf29Bw8Nnvx-Bvuz2x4V7uilbCxaUyopf~SP42u77HIGjDYCTXCKT~i5urdHfTURbtOLSWBZsJ3PNLUT-MMpw40t7yyAl1QCSmEFqEE3TTVZqJ4S0bzQaCMyoOQMIXk1g8T5fPBkQnzNCzEZSuhOsujMQ6G9ttr4Qh2IS1bDyC7qgg0tQ5heYHR9qRYjyC4GdhbKC0v~6Jnaifm7lDZaL0pieFiIs0EhEHJ0xWHKUgxALKU6ZUFe4jrSVxKp~I~Qv3KZZ5DcQdSYPDMXPTmAWa3pWKpatfvYbBbg7Ms7z5iOaaP0rwNQrFQXzvHpsw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106356881&ar=3815695&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx012f4.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Performance comparison among SNP calling tools with RAD-seq reads in 4 inbred rice lines. Mean values of sensitivities (left), PPVs (center), and <em>F</em>-scores (right) of SNP calling by Heap, Stacks, SAMtools, and GATK from RAD-seq reads in 4 inbred rice lines are shown. Letters above the bars indicate groups that are significantly different (<em>P &lt;</em> 0.05), according to the Tukey-Kramer HSD test.</p></div></div>                            <h3 scrollto-destination=106356882 id="106356882" class="section-title" >3.4. Performance comparison among SNP calling tools with high read coverages in sorghum</h3>
<p>To assess the performance of Heap even with high read coverage, we identified SNPs with WGS sequencing reads among the 17 inbred sorghum lines by Heap, SAMtools/BCFtools, and GATK, and benchmarked them in sensitivities, PPVs, and <em>F</em>-scores. We did not adopt Stacks in this test, because Stacks is incompatible with WGS sequencing reads. As a result, we detected 6,153,145, 5,160,730 and 5,587,400 SNP sites using Heap, SAMtools/BCFtools and GATK, respectively. The results showed that Heap had a high <em>F</em>-score of 0.9949, which was not significantly different from the highest <em>F</em>-score of SAMtools/BCFtools (0.9952) (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/supplementary_fig_s2_dsx012.ppt?Expires=2147483647&Signature=nYvf5ZiEizDns-NPoEzgEfsXrcqsipEQmhUrDTn1zx07mDhXpIpLG03rCSQHN86CBdCfsWadiF5qcTqHKoNFo~dMQ0q-Ge61i0w33k5MAqjJOCk-UbwEbxP3dummPasKAZn1wAmZSR3RfWy56GmaqAtkzxZvKrx0a~UruS~y2JR2rpNKWlzH49zJKbOIQns7QABUGMjcpEq2ykanROPhEd6kHYR97z1iCPvgNuACXzG57sV~OG7XUjRAukEaYe55YcDJdWyRvoUZVlYZUd-suYhFpUzLsjR6YgBHCYDSXV~MgAB667otXu7MgRDkAffAjjKXt~ws0a8TewKeh40HnQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Compared to the <em>F</em>-scores under low read coverage, the <em>F</em>-scores of SNPs detected by SAMtools/BCFtools and GATK turned out to be considerably high (<a reveal-id="dsx012-F3" data-open="dsx012-F3" class="link link-reveal link-table xref-fig">Fig. 3</a> and <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/supplementary_fig_s2_dsx012.ppt?Expires=2147483647&Signature=nYvf5ZiEizDns-NPoEzgEfsXrcqsipEQmhUrDTn1zx07mDhXpIpLG03rCSQHN86CBdCfsWadiF5qcTqHKoNFo~dMQ0q-Ge61i0w33k5MAqjJOCk-UbwEbxP3dummPasKAZn1wAmZSR3RfWy56GmaqAtkzxZvKrx0a~UruS~y2JR2rpNKWlzH49zJKbOIQns7QABUGMjcpEq2ykanROPhEd6kHYR97z1iCPvgNuACXzG57sV~OG7XUjRAukEaYe55YcDJdWyRvoUZVlYZUd-suYhFpUzLsjR6YgBHCYDSXV~MgAB667otXu7MgRDkAffAjjKXt~ws0a8TewKeh40HnQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Also in Heap’s case, the more read coverage available, the higher the <em>F</em>-score achieved. This result reconfirms the importance of read coverage for accurate SNP calling. However, surprisingly, the <em>F</em>-scores of SNPs detected by Heap with the high read coverage were not substantially different from that of low read coverage (<a reveal-id="dsx012-F3" data-open="dsx012-F3" class="link link-reveal link-table xref-fig">Fig. 3</a> and <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/supplementary_fig_s2_dsx012.ppt?Expires=2147483647&Signature=nYvf5ZiEizDns-NPoEzgEfsXrcqsipEQmhUrDTn1zx07mDhXpIpLG03rCSQHN86CBdCfsWadiF5qcTqHKoNFo~dMQ0q-Ge61i0w33k5MAqjJOCk-UbwEbxP3dummPasKAZn1wAmZSR3RfWy56GmaqAtkzxZvKrx0a~UruS~y2JR2rpNKWlzH49zJKbOIQns7QABUGMjcpEq2ykanROPhEd6kHYR97z1iCPvgNuACXzG57sV~OG7XUjRAukEaYe55YcDJdWyRvoUZVlYZUd-suYhFpUzLsjR6YgBHCYDSXV~MgAB667otXu7MgRDkAffAjjKXt~ws0a8TewKeh40HnQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). These results demonstrate that Heap is applicable not only to SNP calling with high read coverage but also to that with low read coverage, with fairly high performance.</p>                            <h3 scrollto-destination=106356884 id="106356884" class="section-title" >3.5. Scope of Heap</h3>
<p>In this study, Heap exhibited the highest <em>F</em>-score in SNP calling with low read coverage when compared with the conventional tools. Heap will contribute to reducing costs of work requiring the identification of many SNP markers among multiple samples, such as with GWAS and GP studies. Heap also demonstrated sufficiently high performance compared to the other tools in situations with high read coverage.</p>                            <h3 scrollto-destination=106356886 id="106356886" class="section-title" >3.6. Conclusion</h3>
<p>In GWAS and GP studies, a large number of SNP markers are required to detect associations between SNP markers and phenotypes. On the other hand, false positive SNPs would disturb precise association or prediction. In this study, we have developed a new tool Heap. In the low read coverage condition, we demonstrated Heap’s advantages in sensitivity and PPV by calculating and benchmarking <em>F</em>-scores of SNPs as long as the sorghum and the rice datasets were used. Therefore, Heap would offer fairly reliable SNPs with special reference to GWAS and GP studies. As genomic information becomes available in many species<a reveal-id="dsx012-B22" data-open="dsx012-B22" class="link link-ref link-reveal xref-bibr"><sup>22–25</sup></a>, their SNP information serves as a useful platform on comparative functional analyses. In the future, we will be maintaining and updating the function for SNP mining in Heap.</p>                            <h2 scrollto-destination=106356888 id="106356888" class="section-title" >Availability of data and program</h2>
<p>The datasets supporting the conclusions of this article are available in the DDBJ DRA repository (<a class="link link-uri" href="http://trace.ddbj.nig.ac.jp/dra/index_e.html" target="_blank">http://trace.ddbj.nig.ac.jp/dra/index_e.html</a> (29 March 2017, date last accessed)), under accession numbers DRR045054-DRR045091. The source code of Heap is freely available from the git repository (<a class="link link-uri" href="https://github.com/meiji-bioinf/heap" target="_blank">https://github.com/meiji-bioinf/heap</a> (29 March 2017, date last accessed)) and our web site (<a class="link link-uri" href="http://bioinf.mind.meiji.ac.jp/lab/en/tools.html" target="_blank">http://bioinf.mind.meiji.ac.jp/lab/en/tools.html</a> (29 March 2017, date last accessed)).</p>                            <h2 scrollto-destination=106356890 id="106356890" class="backacknowledgements-title" >Acknowledgements</h2>
<p>This work was supported by JST CREST Grant Number JPMJCR12B5, Japan. Computations were partially performed on the NIG (National Institute of Genetics) SuperComputer Facilities hosted at NIG/ROIS (Research Organization of Information and Systems).</p>                            <h3 scrollto-destination=106356892 id="106356892" class="section-title" >Conflicts of Interest</h3>
<p>None declared.</p>                            <h2 scrollto-destination=106356894 id="106356894" class="backsection-title" >Supplementary data</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/supplementary_table_s1_dsx012.ppt?Expires=2147483647&Signature=otolKOrNIyYzL~2fGzbrrOfrvsSQJ~JFj8SYL-CIrRdeqf6uQobEGK3xT-QMCmRMn17BbXjTRPSsws27Fj92U7zIY4so-m2Sdn15NC9l~EY9FhvL8pFiQNhJGVlECtlyWmQ7IBwo2-cQ9gCpDpBTY5~xOhMxM6uDrbws9TmgxiNVznHt0afxTlPEYvcRBUHGJrIcsqmQsjwN7InYtCipipVA8-F-dRq3mt2yRYGSlQoKP-7zSPsQZC-2ubpfgKk9Y7j5Lm5VMB4qEZN3L5b9RKvXQXkf8prtoiu2dB3zqIIQkRZNbVedHtZQrNsL8Ndk8SkQDooIuIHN8Z-53K20lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary material</a></span> is available at <em>DNARES</em> online.</p>                            <h2 scrollto-destination=106356896 id="106356896" class="backsection-title" >Funding</h2>
<p>This work was partially supported by CREST, JST to NT, the Japan Society for the Promotion of Science (JSPS) KAENHI [Grants-in-Aid for Scientific Research on Innovative Areas (No. 26113716)], MEXT-Supported Program for the Strategic Research Foundation at Private Universities (2014–2018), and Research Funding for Computational Software Supporting Program from Meiji University to KY.</p>                            <h2 scrollto-destination=106356898 id="106356898" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx012-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Crossa</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Beyene</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kassa</div> <div class="given-names">S.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Genomic prediction in maize breeding populations with genotyping-by-sequencing</div>. <div class="source">G3 (Bethesda)</div> , <div class="volume">3</div>, <div class="fpage">1903</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20prediction%20in%20maize%20breeding%20populations%20with%20genotyping-by-sequencing&amp;author=J.%20Crossa&amp;author=Y.%20Beyene&amp;author=S.%20Kassa&amp;publication_year=2013&amp;journal=G3%20(Bethesda)&amp;volume=3&amp;pages=1903-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.113.008227" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.113.008227" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.113.008227"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24022750" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx012-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Morris</div>
               <div class="given-names">G. P.</div>
            </div>, <div class="name string-name"><div class="surname">Ramu</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Deshpande</div> <div class="given-names">S. P.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Population genomic and genome-wide association studies of agroclimatic traits in sorghum</div>. <div class="source">Proc. Natl. Acad. Sci</div> ., <div class="volume">110</div>, <div class="fpage">453</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20genomic%20and%20genome-wide%20association%20studies%20of%20agroclimatic%20traits%20in%20sorghum&amp;author=G.%20P.%20Morris&amp;author=P.%20Ramu&amp;author=S.%20P.%20Deshpande&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci&amp;volume=110&amp;pages=453-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1215985110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1215985110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1215985110"> </span></div> </div></div></div></div></div><div content-id="dsx012-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jarquín</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Kocak</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Posadas</div> <div class="given-names">L.</div></div></div>, et al.  <div class="year">2014</div>, 
<div class="article-title">Genotyping by sequencing for genomic prediction in a soybean breeding population</div>. <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">740</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genotyping%20by%20sequencing%20for%20genomic%20prediction%20in%20a%20soybean%20breeding%20population&amp;author=D.%20Jarqu%C3%ADn&amp;author=K.%20Kocak&amp;author=L.%20Posadas&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=740" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-740" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-740" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-740"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25174348" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx012-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sonah</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">O’Donoughue</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Cober</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Rajcan</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Belzile</div> <div class="given-names">F.</div></div></div>
          <div class="year">2015</div>, 
<div class="article-title">Identification of loci governing eight agronomic traits using a GBS-GWAS approach and validation by QTL mapping in soya bean</div>. <div class="source">Plant Biotechnol. J</div> ., <div class="volume">13</div>, <div class="fpage">211</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20loci%20governing%20eight%20agronomic%20traits%20using%20a%20GBS-GWAS%20approach%20and%20validation%20by%20QTL%20mapping%20in%20soya%20bean&amp;author=H.%20Sonah&amp;author=L.%20O%E2%80%99Donoughue&amp;author=E.%20Cober&amp;author=I.%20Rajcan&amp;author=F.%20Belzile&amp;publication_year=2015&amp;journal=Plant%20Biotechnol.%20J&amp;volume=13&amp;pages=211-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.2015.13.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2015.13.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.2015.13.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25213593" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx012-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div>
          <div class="year">2009</div>, 
<div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</div>. <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx012-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Langmead</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div></div>
          <div class="year">2012</div>, 
<div class="article-title">Fast gapped-read alignment with Bowtie 2</div>. <div class="source">Nat. Methods</div> , <div class="volume">9</div>, <div class="fpage">357</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B.%20Langmead&amp;author=S.%20L.%20Salzberg&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=357-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx012-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Handsaker</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wysoker</div> <div class="given-names">A.</div></div></div>, et al.  <div class="year">2009</div>, 
<div class="article-title">The sequence alignment/map format and SAMtools</div>. <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2078</div>–<div class="lpage">79</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20alignment%2Fmap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx012-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="year">2011</div>, 
<div class="article-title">A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data</div>. <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">2987</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20statistical%20framework%20for%20SNP%20calling%2C%20mutation%20discovery%2C%20association%20mapping%20and%20population%20genetical%20parameter%20estimation%20from%20sequencing%20data&amp;author=H.%20Li&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2987-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr509" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr509" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr509"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21903627" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx012-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">McKenna</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Hanna</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Banks</div> <div class="given-names">E.</div></div></div>, et al.  <div class="year">2010</div>, 
<div class="article-title">The Genome Analysis Toolkit: A MapReduce framework for analyzing next-generation DNA sequencing data</div>. <div class="source">Genome Res</div> ., <div class="volume">20</div>, <div class="fpage">1297</div>–<div class="lpage">1303</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Genome%20Analysis%20Toolkit%3A%20A%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&amp;author=A.%20McKenna&amp;author=M.%20Hanna&amp;author=E.%20Banks&amp;publication_year=2010&amp;journal=Genome%20Res&amp;volume=20&amp;pages=1297-1303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.107524.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.107524.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.107524.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20644199" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx012-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">DePristo</div>
               <div class="given-names">M.A.</div>
            </div>, <div class="name string-name"><div class="surname">Banks</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Poplin</div> <div class="given-names">R.</div></div></div>, et al.  <div class="year">2011</div>, 
<div class="article-title">A framework for variation discovery and genotyping using next-generation DNA sequencing data</div>. <div class="source">Nat. Genet</div> ., <div class="volume">43</div>, <div class="fpage">491</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20framework%20for%20variation%20discovery%20and%20genotyping%20using%20next-generation%20DNA%20sequencing%20data&amp;author=M.A.%20DePristo&amp;author=E.%20Banks&amp;author=R.%20Poplin&amp;publication_year=2011&amp;journal=Nat.%20Genet&amp;volume=43&amp;pages=491-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.806" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.806" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.806"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21478889" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx012-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Van der Auwera</div>
               <div class="given-names">G. A.</div>
            </div>, <div class="name string-name"><div class="surname">Carneiro</div> <div class="given-names">M. O.</div></div>, <div class="name string-name"><div class="surname">Hartl</div> <div class="given-names">C.</div></div></div>, et al.  <div class="year">2013</div>, <div class="source">From FastQ Data to High-Confidence Variant Calls: The Genome Analysis Toolkit Best Practices Pipeline. Current Protocols in Bioinformatics</div> . 
<div class="publisher-name">John Wiley &amp; Sons, Inc</div>., 
<div class="publisher-loc">Hoboken, NJ, USA</div>, pp. 11.10.1-11.10.33.</div></div></div></div><div content-id="dsx012-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cheng</div>
               <div class="given-names">A. Y.</div>
            </div>, <div class="name string-name"><div class="surname">Teo</div> <div class="given-names">Y.-Y.</div></div>, <div class="name string-name"><div class="surname">Ong</div> <div class="given-names">R. T.-H.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">Assessing single nucleotide variant detection and genotype calling on whole-genome sequenced individuals</div>. <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">1707</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessing%20single%20nucleotide%20variant%20detection%20and%20genotype%20calling%20on%20whole-genome%20sequenced%20individuals&amp;author=A.%20Y.%20Cheng&amp;author=Y.-Y.%20Teo&amp;author=R.%20T.-H.%20Ong&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=1707-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu067" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu067" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu067"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24558117" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx012-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Baird</div>
               <div class="given-names">N. A.</div>
            </div>, <div class="name string-name"><div class="surname">Etter</div> <div class="given-names">P. D.</div></div>, <div class="name string-name"><div class="surname">Atwood</div> <div class="given-names">T. S.</div></div></div>, et al.  <div class="year">2008</div>, 
<div class="article-title">Rapid SNP Discovery and Genetic Mapping Using Sequenced RAD Markers</div>. <div class="source">PLoS One</div> , <div class="volume">3</div>, <div class="fpage">e3376</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20SNP%20Discovery%20and%20Genetic%20Mapping%20Using%20Sequenced%20RAD%20Markers&amp;author=N.%20A.%20Baird&amp;author=P.%20D.%20Etter&amp;author=T.%20S.%20Atwood&amp;publication_year=2008&amp;journal=PLoS%20One&amp;volume=3&amp;pages=e3376" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0003376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0003376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0003376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18852878" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx012-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhai</div>
               <div class="given-names">Z.</div>
            </div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">He</div> <div class="given-names">C.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">SNP discovery and genotyping using restriction-site-associated DNA sequencing in chickens</div>. <div class="source">Anim. Genet</div> ., <div class="volume">46</div>, <div class="fpage">216</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20discovery%20and%20genotyping%20using%20restriction-site-associated%20DNA%20sequencing%20in%20chickens&amp;author=Z.%20Zhai&amp;author=W.%20Zhao&amp;author=C.%20He&amp;publication_year=2015&amp;journal=Anim.%20Genet&amp;volume=46&amp;pages=216-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/age.2015.46.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fage.2015.46.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fage.2015.46.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25591076" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx012-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Catchen</div>
               <div class="given-names">J. M.</div>
            </div>, <div class="name string-name"><div class="surname">Amores</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Hohenlohe</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Cresko</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Postlethwait</div> <div class="given-names">J. H.</div></div>, <div class="name string-name"><div class="surname">De Koning</div> <div class="given-names">D.-J.</div></div></div>
          <div class="year">2011</div>, 
<div class="article-title">Stacks: building and genotyping loci <em>de novo</em> from short-read sequences</div>. <div class="source">G3 Genes Genomes Genetics</div> , <div class="volume">1</div>, <div class="fpage">171</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stacks%3A%20building%20and%20genotyping%20loci%20de%20novo%20from%20short-read%20sequences&amp;author=J.%20M.%20Catchen&amp;author=A.%20Amores&amp;author=P.%20Hohenlohe&amp;author=W.%20Cresko&amp;author=J.%20H.%20Postlethwait&amp;author=D.-J.%20De%20Koning&amp;publication_year=2011&amp;journal=G3%20Genes%20Genomes%20Genetics&amp;volume=1&amp;pages=171-82" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22384329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx012-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Catchen</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Hohenlohe</div> <div class="given-names">P. A.</div></div>, <div class="name string-name"><div class="surname">Bassham</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Amores</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Cresko</div> <div class="given-names">W. A.</div></div></div>
          <div class="year">2013</div>, 
<div class="article-title">Stacks: an analysis tool set for population genomics</div>. <div class="source">Mol. Ecol</div> ., <div class="volume">22</div>, <div class="fpage">3124</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stacks%3A%20an%20analysis%20tool%20set%20for%20population%20genomics&amp;author=J.%20Catchen&amp;author=P.%20A.%20Hohenlohe&amp;author=S.%20Bassham&amp;author=A.%20Amores&amp;author=W.%20A.%20Cresko&amp;publication_year=2013&amp;journal=Mol.%20Ecol&amp;volume=22&amp;pages=3124-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.12354" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.12354" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.12354"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23701397" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx012-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ohyanagi</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Takano</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Terashima</div> <div class="given-names">S.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Plant omics data center: an integrated web repository for interspecies gene expression networks with NLP-based curation</div>. <div class="source">Plant Cell Physiol</div> ., <div class="volume">56</div>, <div class="fpage">e9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20omics%20data%20center%3A%20an%20integrated%20web%20repository%20for%20interspecies%20gene%20expression%20networks%20with%20NLP-based%20curation&amp;author=H.%20Ohyanagi&amp;author=T.%20Takano&amp;author=S.%20Terashima&amp;publication_year=2015&amp;journal=Plant%20Cell%20Physiol&amp;volume=56&amp;pages=e9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcu188" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcu188" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcu188"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25505034" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx012-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Goodstein</div>
               <div class="given-names">D. M.</div>
            </div>, <div class="name string-name"><div class="surname">Shu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Howson</div> <div class="given-names">R.</div></div></div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Phytozome: a comparative platform for green plant genomics</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">D1178</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phytozome%3A%20a%20comparative%20platform%20for%20green%20plant%20genomics&amp;author=D.%20M.%20Goodstein&amp;author=S.%20Shu&amp;author=R.%20Howson&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=D1178-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr944" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr944" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr944"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22110026" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx012-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kawahara</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">de la Bastide</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hamilton</div> <div class="given-names">J. P.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Improvement of the <em>Oryza sativa</em> Nipponbare reference genome using next generation sequence and optical map data</div>. <div class="source">Rice</div> , <div class="volume">6</div>, <div class="fpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improvement%20of%20the%20Oryza%20sativa%20Nipponbare%20reference%20genome%20using%20next%20generation%20sequence%20and%20optical%20map%20data&amp;author=Y.%20Kawahara&amp;author=M.%20de%20la%20Bastide&amp;author=J.%20P.%20Hamilton&amp;publication_year=2013&amp;journal=Rice&amp;volume=6&amp;pages=4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1939-8433-6-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1939-8433-6-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1939-8433-6-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24280374" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx012-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sakai</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">S. S.</div></div>, <div class="name string-name"><div class="surname">Tanaka</div> <div class="given-names">T.</div></div></div>, et al.  <div class="year">2013</div>, 
<div class="article-title">Rice annotation project database (RAP-DB): an integrative and interactive database for rice genomics</div>. <div class="source">Plant Cell Physiol</div> ., <div class="volume">54</div>, <div class="fpage">e6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rice%20annotation%20project%20database%20(RAP-DB)%3A%20an%20integrative%20and%20interactive%20database%20for%20rice%20genomics&amp;author=H.%20Sakai&amp;author=S.%20S.%20Lee&amp;author=T.%20Tanaka&amp;publication_year=2013&amp;journal=Plant%20Cell%20Physiol&amp;volume=54&amp;pages=e6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcs183" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcs183" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcs183"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23299411" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx012-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Paterson</div>
               <div class="given-names">A. H.</div>
            </div>, <div class="name string-name"><div class="surname">Bowers</div> <div class="given-names">J. E.</div></div>, <div class="name string-name"><div class="surname">Bruggmann</div> <div class="given-names">R.</div></div></div>, et al.  <div class="year">2009</div>, 
<div class="article-title">The <em>Sorghum bicolor</em> genome and the diversification of grasses</div>. <div class="source">Nature</div> , <div class="volume">457</div>, <div class="fpage">551</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Sorghum%20bicolor%20genome%20and%20the%20diversification%20of%20grasses&amp;author=A.%20H.%20Paterson&amp;author=J.%20E.%20Bowers&amp;author=R.%20Bruggmann&amp;publication_year=2009&amp;journal=Nature&amp;volume=457&amp;pages=551-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07723" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07723" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07723"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19189423" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx012-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Natsume</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Takagi</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Shiraishi</div> <div class="given-names">A.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">The draft genome of Hop (<em>Humulus lupulus</em>), an essence for brewing</div>. <div class="source">Plant Cell Physiol</div> ., <div class="volume">56</div>, <div class="fpage">428</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20of%20Hop%20(Humulus%20lupulus)%2C%20an%20essence%20for%20brewing&amp;author=S.%20Natsume&amp;author=H.%20Takagi&amp;author=A.%20Shiraishi&amp;publication_year=2015&amp;journal=Plant%20Cell%20Physiol&amp;volume=56&amp;pages=428-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcu169" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcu169" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcu169"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25416290" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx012-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sakai</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Naito</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Takahashi</div> <div class="given-names">Y.</div></div></div>, et al.  <div class="year">2016</div>, 
<div class="article-title">The <em>Vigna</em> Genome Server, ‘<em>Vig</em>GS’: A genomic knowledge base of the genus <em>Vigna</em> based on high-quality, annotated genome sequence of the Azuki Bean, <em>Vigna angularis</em> (Willd.) Ohwi &amp; Ohashi</div>. <div class="source">Plant Cell Physiol</div> ., <div class="volume">57</div>, <div class="fpage">e2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Vigna%20Genome%20Server%2C%20%E2%80%98VigGS%E2%80%99%3A%20A%20genomic%20knowledge%20base%20of%20the%20genus%20Vigna%20based%20on%20high-quality%2C%20annotated%20genome%20sequence%20of%20the%20Azuki%20Bean%2C%20Vigna%20angularis%20(Willd.)%20Ohwi%20%26%20Ohashi&amp;author=H.%20Sakai&amp;author=K.%20Naito&amp;author=Y.%20Takahashi&amp;publication_year=2016&amp;journal=Plant%20Cell%20Physiol&amp;volume=57&amp;pages=e2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcv189" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcv189" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcv189"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26644460" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx012-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Krishnakumar</div>
               <div class="given-names">V.</div>
            </div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Rosen</div> <div class="given-names">B. D.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">MTGD: The <em>Medicago truncatula</em> Genome Database</div>. <div class="source">Plant Cell Physiol</div> ., <div class="volume">56</div>, <div class="fpage">e1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MTGD%3A%20The%20Medicago%20truncatula%20Genome%20Database&amp;author=V.%20Krishnakumar&amp;author=M.%20Kim&amp;author=B.%20D.%20Rosen&amp;publication_year=2015&amp;journal=Plant%20Cell%20Physiol&amp;volume=56&amp;pages=e1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcu179" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcu179" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcu179"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25432968" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx012-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx012-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ohyanagi</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Ebata</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">X.</div></div></div>, et al.  <div class="year">2016</div>, 
<div class="article-title">OryzaGenome: genome diversity database of wild <em>Oryza</em> species</div>. <div class="source">Plant Cell Physiol</div> ., <div class="volume">57</div>, <div class="fpage">e1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=OryzaGenome%3A%20genome%20diversity%20database%20of%20wild%20Oryza%20species&amp;author=H.%20Ohyanagi&amp;author=T.%20Ebata&amp;author=X.%20Huang&amp;publication_year=2016&amp;journal=Plant%20Cell%20Physiol&amp;volume=57&amp;pages=e1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcv171" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcv171" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcv171"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26578696" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=3815695&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/supplementary_table_s1_dsx012.ppt?Expires=2147483647&amp;Signature=otolKOrNIyYzL~2fGzbrrOfrvsSQJ~JFj8SYL-CIrRdeqf6uQobEGK3xT-QMCmRMn17BbXjTRPSsws27Fj92U7zIY4so-m2Sdn15NC9l~EY9FhvL8pFiQNhJGVlECtlyWmQ7IBwo2-cQ9gCpDpBTY5~xOhMxM6uDrbws9TmgxiNVznHt0afxTlPEYvcRBUHGJrIcsqmQsjwN7InYtCipipVA8-F-dRq3mt2yRYGSlQoKP-7zSPsQZC-2ubpfgKk9Y7j5Lm5VMB4qEZN3L5b9RKvXQXkf8prtoiu2dB3zqIIQkRZNbVedHtZQrNsL8Ndk8SkQDooIuIHN8Z-53K20lg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S1</div></a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/supplementary_table_s2_dsx012.ppt?Expires=2147483647&amp;Signature=OWDbKaivdo-oimzIfN8q4QMT9aKVHedUBPUgw-YrDfOAbC4wsa02OG-wwpoZVNvEMYzufTi7aBHpRu4Csj8K4vGsPbV2zNmKbnzesg9JzS7hH0ohHyGnpLUNjUqO4zhyPlHvDSxgqWT2jNoLmSCjPiguAM5vY7MVCX5slkeFWArslw~8ZYFN~2lbTdJXE9NOVo0T~9ZkdiEvFdPmu3U7kvyLPiSnyGJxwDk0e21bULAK6c3dkkdSChlAVNGOjmhnfZpvbmceXLZ5~ElkYavGPK06h4inpewyezBoMtl-FOfC5HbWodNSr6Q63pVZXeLCwi~G4PkE3Yr~9SFH5ufVRQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S2</div></a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/supplementary_fig_s1_dsx012.ppt?Expires=2147483647&amp;Signature=JVLIv080j1q1nKNRXVXwhmeXeGLHiCUjfQsSpTJzvYxfwyoG~nxiKs47pvg6BUyTlW1-nQoJFDeVUHQRfL3zc5Rzjihm0lqCEk-8pz8QmgpJR57NPmQ3-L1JwVvwc2y2PRl0do9cQOxxj9XN57tTh91WFjIPktzec0YyafzMuFKbii-v9Xpg91b-aZpH3GuOe9VUipRkPhPQ37In1UT864YgBD-0PsxPCkUNVIxz2RxIZBeiFOk7~dwvAn9Hjh7grrxIUSGgSDanLR5KnyyUCbXYZjbiCY271VaVRyuREEtSNvcG9477cOoweQBI5ZDrb~pFlOS1fHM1xcVqUFK4~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S1</div></a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/supplementary_fig_s2_dsx012.ppt?Expires=2147483647&amp;Signature=nYvf5ZiEizDns-NPoEzgEfsXrcqsipEQmhUrDTn1zx07mDhXpIpLG03rCSQHN86CBdCfsWadiF5qcTqHKoNFo~dMQ0q-Ge61i0w33k5MAqjJOCk-UbwEbxP3dummPasKAZn1wAmZSR3RfWy56GmaqAtkzxZvKrx0a~UruS~y2JR2rpNKWlzH49zJKbOIQns7QABUGMjcpEq2ykanROPhEd6kHYR97z1iCPvgNuACXzG57sV~OG7XUjRAukEaYe55YcDJdWyRvoUZVlYZUd-suYhFpUzLsjR6YgBHCYDSXV~MgAB667otXu7MgRDkAffAjjKXt~ws0a8TewKeh40HnQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S2</div></a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx012/2/supplementary_methods_dsx012.doc?Expires=2147483647&amp;Signature=ry9MHxyf0iwBvLnDJBhlaTtzY7tamI8iOj-3AG40Qi4~~VtNszR8UcO4qPkHJC6AC2FcOEDaXKJ~09gz1bUkOs-VxX8WxpcoDn1gwuAP3vJLZU-8HtYFRX48BEBbCcHV8HjApl4Bxwg7uE9sJ35p0REzRfK-iOYhDO09BGE~VuoHv749sza0UWbMPyXwWgPxyTqf4eB7n3tHa3APWhgaQpIUWD2hHgERaUWttTrAdHHuYausFequfsXiMYwNCKclFJGSLbu3Bn~Yjqg756v-ZYdWKgXNexIy6DerrsuGHrWIOnpS-3gSMOfAd~bl9vGhPEvzGZusrhUA8k6Af181hg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Methods</div></a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,412</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx012"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="3815695"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f4%2f397%2f3815695">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F4%2F397%2F3815695" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/3815695'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Heap: a highly sensitive and accurate SNP detection tool for low-coverage high-throughput sequencing data" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx012");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f4%2f397%2f3815695">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

