

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/12/6/403/344476","@type":"ScholarlyArticle","name":" Characterization of a Whole Set of tRNA Molecules in an Aerobic Hyper-thermophilic Crenarchaeon,  Aeropyrum pernix  K1 ","datePublished":"2005-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/12/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2005-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsi023","keywords":["tRNA","intron","splicing","bulge–helix–bulge structure","genome","Crenarchaeon","<italic>Aeropyrum pernix</italic> K1 "],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/12/6/403/344476","author":[{"name":"Yamazaki, Syuji","affiliation":"1  National Institute of Technology and Evaluation  2-49-10 Nishihara, Shibuya, Tokyo 151-0066, Japan","@type":"Person"},{"name":"Kikuchi, Hisashi","affiliation":"1  National Institute of Technology and Evaluation  2-49-10 Nishihara, Shibuya, Tokyo 151-0066, Japan","@type":"Person"},{"name":"Kawarabayasi, Yutaka","affiliation":"2  National Institute of Advanced Industrial Science and Technology (AIST)  Higashi 1-1, Tsukuba, Ibaraki 305-8566, Japan","@type":"Person"}],"description":"Abstract.   The tRNA molecule has an important role in translation, the function of which is to carry amino acids to the ribosomes. It is known that tRNA is tra","pageStart":"403","pageEnd":"416","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/6/0/m_dnaresearch12_6.cover.gif?Expires=2147483647&Signature=vmBj-dEpEdSISGGZZVQQ800UdbaQ61VgN5SO6nMO30mtIcmmHegLeBqq~JmeB3Cl71F-HZSxEtC05awn7JLvbmS~R8ehKSKlG67TzUmDNfl2uJ9iwd8X3XVTg535TrTFouPO8FrSAc3H4exNAvHuAshR8~cghIB3WqxBjwF3q4V-gWs6yVOhRl523VyJV6-aueYhJA8pI1jFRHUmPhSY-RKzDUrwgfwrVrH153kk1IiVH3chlu~YmBF~XczCNeM~pNFI-0DwLreboeiSvqsHnGPMQGgfWos-lNOnEyjyXTWeiJBGXgUTUflQE1fArs2l0eo950LpKbLah37z6bPoSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":" Characterization of a Whole Set of tRNA Molecules in an Aerobic Hyper-thermophilic Crenarchaeon,  Aeropyrum pernix  K1 ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/6/0/m_dnaresearch12_6.cover.gif?Expires=2147483647&Signature=vmBj-dEpEdSISGGZZVQQ800UdbaQ61VgN5SO6nMO30mtIcmmHegLeBqq~JmeB3Cl71F-HZSxEtC05awn7JLvbmS~R8ehKSKlG67TzUmDNfl2uJ9iwd8X3XVTg535TrTFouPO8FrSAc3H4exNAvHuAshR8~cghIB3WqxBjwF3q4V-gWs6yVOhRl523VyJV6-aueYhJA8pI1jFRHUmPhSY-RKzDUrwgfwrVrH153kk1IiVH3chlu~YmBF~XczCNeM~pNFI-0DwLreboeiSvqsHnGPMQGgfWos-lNOnEyjyXTWeiJBGXgUTUflQE1fArs2l0eo950LpKbLah37z6bPoSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content=" Characterization of a Whole Set of tRNA Molecules in an Aerobic Hyper-thermophilic Crenarchaeon,  Aeropyrum pernix  K1 " />
    <meta property="og:description" content="Abstract.   The tRNA molecule has an important role in translation, the function of which is to carry amino acids to the ribosomes. It is known that tRNA is tra" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/12/6/403/344476" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2005-01-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Yamazaki, Syuji" /><meta name="citation_author_institution" content="1National Institute of Technology and Evaluation2-49-10 Nishihara, Shibuya, Tokyo 151-0066, Japan" /><meta name="citation_author" content="Kikuchi, Hisashi" /><meta name="citation_author_institution" content="1National Institute of Technology and Evaluation2-49-10 Nishihara, Shibuya, Tokyo 151-0066, Japan" /><meta name="citation_author" content="Kawarabayasi, Yutaka" /><meta name="citation_author_institution" content="2National Institute of Advanced Industrial Science and Technology (AIST)Higashi 1-1, Tsukuba, Ibaraki 305-8566, Japan" /><meta name="citation_title" content=" Characterization of a Whole Set of tRNA Molecules in an Aerobic Hyper-thermophilic Crenarchaeon, Aeropyrum pernix K1 " /><meta name="citation_firstpage" content="403" /><meta name="citation_lastpage" content="416" /><meta name="citation_doi" content="10.1093/dnares/dsi023" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="12" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2005/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Splicing of tRNA Precursors; citation_publisher=American Society for Microbiology, Washington, DC; tRNA: Structure, Biosynthesis, and Function;  citation_year=1995;  citation_pages=79-92; " /><meta name="citation_reference" content="citation_title=Intron sequence and structure requirements for tRNA splicing in Saccharomyces cerevisiae; J. Biol. Chem.;  citation_year=1988;  citation_volume=263;  citation_pages=13839-13847; " /><meta name="citation_reference" content="citation_title=Structure of intron-containing tRNA precursors. Analysis of solution conformation using chemical and enzymatic probes; J. Biol. Chem.;  citation_year=1984;  citation_volume=259;  citation_pages=5197-5207; " /><meta name="citation_reference" content="citation_title=Substrate recognition and splice site determination in yeast tRNA splicing; Cell;  citation_year=1988;  citation_volume=55;  citation_pages=719-730; " /><meta name="citation_reference" content="citation_title=Site selection by the splicing endonuclease of Xenopus Laevis; Cell;  citation_year=1988;  citation_volume=55;  citation_pages=731-738; " /><meta name="citation_reference" content="citation_title=A tRNA(Trp) intron endonuclease from Halobacterium volcanii . Unique substrate recognition properties; J. Biol. Chem.;  citation_year=1988;  citation_volume=263;  citation_pages=17951-17959; " /><meta name="citation_reference" content="citation_title=Recognition of exon–intron boundaries by the Halobacterium volcanii tRNA intron endonuclease; J. Biol. Chem.;  citation_year=1990;  citation_volume=265;  citation_pages=18104-18111; " /><meta name="citation_reference" content="citation_title=Transcription and excision of a large intron in the tRNA Trp gene of an archaebacterium, Halobacterium volcanii; J. Biol. Chem.;  citation_year=1985;  citation_volume=260;  citation_pages=3132-3134; " /><meta name="citation_reference" content="citation_title=Presence of an intron in elongator methionine-tRNA of Halobacterium volcanii; Can. J. Microbiol.;  citation_year=1989;  citation_volume=35;  citation_pages=189-194; " /><meta name="citation_reference" content="citation_title=Putative introns in tRNA genes of prokaryotes; Proc. Natl Acad. Sci. USA;  citation_year=1983;  citation_volume=80;  citation_pages=3309-3312; " /><meta name="citation_reference" content="citation_title=A unique tRNA intron in the variable loop of the extreme thermophile Thermofilm pendens and its possible evolutionary implications; J. Biol. Chem.;  citation_year=1989;  citation_volume=264;  citation_pages=17834-17837; " /><meta name="citation_reference" content="citation_title=Comparison of transfer RNA and ribosomal RNA intron splicing in the extreme thermophile and archaebacterium Desulfurococcus mobilis; Can. J. Microbiol.;  citation_year=1989;  citation_volume=35;  citation_pages=210-214; " /><meta name="citation_reference" content="citation_title=Genes for stable RNA in the extreme thermophile Thermoproteus tenax : introns and transcription signals; EMBO J.;  citation_year=1987;  citation_volume=6;  citation_pages=523-528; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of an aerobic hyper-thermophilic crenarchaeota, Aeropyrum pernix K1; citation_text=147–155; DNA Res.;  citation_year=1999;  citation_volume=6;  citation_pages=84-76; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of an aerobic thermoacidophilic crenarchaeon, Sulfolobus tokodaii strain7; DNA Res.;  citation_year=2001;  citation_volume=8;  citation_pages=123-140; " /><meta name="citation_reference" content="citation_title=Aeropurum pernix gen. nov., sp. nov., a novel aerobic hyper thermophilic archaeon growing at temperatures up to 100°C; Int. J. Syst. Bacteriol.;  citation_year=1996;  citation_volume=46;  citation_pages=1070-1077; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of the methanogenic archaeon, Methanococcus jannaschii; Science;  citation_year=1996;  citation_volume=273;  citation_pages=1058-1073; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of Methanobacterium thermoautotrophicum deltaH: functional analysis and comparative genomics; J. Bacteriol.;  citation_year=1997;  citation_volume=179;  citation_pages=7135-7155; " /><meta name="citation_reference" content="citation_title=Participation of the intron in the reaction catalyzed by the Xenopus tRNA splicing endonuclease; Science;  citation_year=1992;  citation_volume=255;  citation_pages=1404-1408; " /><meta name="citation_reference" content="citation_title=Identification of BHB splicing motifs in intron-containing tRNAs from 18 archaea: evolutionary implications; RNA;  citation_year=2003;  citation_volume=9;  citation_pages=1516-1531; " /><meta name="citation_reference" content="citation_title=Molecular characterization and postsplicing fate of three introns within the single rRNA operon of the hyperthermophilic archaeon Aeropyrum pernix K1; J. Bacteriol.;  citation_year=1998;  citation_volume=180;  citation_pages=3635-3643; " /><meta name="citation_reference" content="citation_title=Introns in protein-coding genes in Archaea; FEBS Lett.;  citation_year=2002;  citation_volume=510;  citation_pages=27-30; " /><meta name="citation_reference" content="citation_title=Properties of H. volcanii tRNA intron endonuclease reveal a relationship between the archaeal and eucaryal tRNA intron processing systems; Cell;  citation_year=1997;  citation_volume=89;  citation_pages=839-848; " /><meta name="citation_reference" content="citation_title=Saccharomyces cerevisiae tRNA ligase. Purification of the protein and isolation of the structural gene; J. Biol. Chem.;  citation_year=1986;  citation_volume=261;  citation_pages=2978-2986; " /><meta name="citation_reference" content="citation_title=tRNAscan-SE: A program for improved detection of transfer RNA genes in genomic sequence; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=955-964; " /><meta name="citation_reference" content="citation_title=Structure and transcription of eukaryotic tRNA genes; CRC Crit. Biochem.;  citation_year=1985;  citation_volume=19;  citation_pages=107-144; " /><meta name="citation_reference" content="citation_title=Concerted changes in the nucleotide sequences of the intragenic promoter regions of eukaryotic genes for tRNAs of all specificities; J. Mol. Evol.;  citation_year=2003;  citation_volume=57;  citation_pages=520-532; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/12/6/403/6718749/dsi023.pdf" /><meta name="description" content="Abstract.   The tRNA molecule has an important role in translation, the function of which is to carry amino acids to the ribosomes. It is known that tRNA is tra" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/12/6/403/344476" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/12/6/403/344476" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/12/6/403/344476");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsi023';
    </script>
<title>
	 Characterization of a Whole Set of tRNA Molecules in an Aerobic Hyper-thermophilic Crenarchaeon, Aeropyrum pernix K1  | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./344476" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BetAPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZYW1hemFraSwgU3l1amkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxTmF0aW9uYWwgSW5zdGl0dXRlIG9mIFRlY2hub2xvZ3kgYW5kIEV2YWx1YXRpb24yLTQ5LTEwIE5pc2hpaGFyYSwgU2hpYnV5YSwgVG9reW8gMTUxLTAwNjYsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2lrdWNoaSwgSGlzYXNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjFOYXRpb25hbCBJbnN0aXR1dGUgb2YgVGVjaG5vbG9neSBhbmQgRXZhbHVhdGlvbjItNDktMTAgTmlzaGloYXJhLCBTaGlidXlhLCBUb2t5byAxNTEtMDA2NiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYXdhcmFiYXlhc2ksIFl1dGFrYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjJOYXRpb25hbCBJbnN0aXR1dGUgb2YgQWR2YW5jZWQgSW5kdXN0cmlhbCBTY2llbmNlIGFuZCBUZWNobm9sb2d5IChBSVNUKUhpZ2FzaGkgMS0xLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS04NTY2LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iIENoYXJhY3Rlcml6YXRpb24gb2YgYSBXaG9sZSBTZXQgb2YgdFJOQSBNb2xlY3VsZXMgaW4gYW4gQWVyb2JpYyBIeXBlci10aGVybW9waGlsaWMgQ3JlbmFyY2hhZW9uLCBBZXJvcHlydW0gcGVybml4IEsxICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjQwMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNDE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNpMDIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjEyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDUvMDEvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TcGxpY2luZyBvZiB0Uk5BIFByZWN1cnNvcnM7IGNpdGF0aW9uX3B1Ymxpc2hlcj1BbWVyaWNhbiBTb2NpZXR5IGZvciBNaWNyb2Jpb2xvZ3ksIFdhc2hpbmd0b24sIERDOyB0Uk5BOiBTdHJ1Y3R1cmUsIEJpb3N5bnRoZXNpcywgYW5kIEZ1bmN0aW9uOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fcGFnZXM9NzktOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludHJvbiBzZXF1ZW5jZSBhbmQgc3RydWN0dXJlIHJlcXVpcmVtZW50cyBmb3IgdFJOQSBzcGxpY2luZyBpbiBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWU7IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTI2MzsgIGNpdGF0aW9uX3BhZ2VzPTEzODM5LTEzODQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmUgb2YgaW50cm9uLWNvbnRhaW5pbmcgdFJOQSBwcmVjdXJzb3JzLiBBbmFseXNpcyBvZiBzb2x1dGlvbiBjb25mb3JtYXRpb24gdXNpbmcgY2hlbWljYWwgYW5kIGVuenltYXRpYyBwcm9iZXM7IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0xOTg0OyAgY2l0YXRpb25fdm9sdW1lPTI1OTsgIGNpdGF0aW9uX3BhZ2VzPTUxOTctNTIwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3Vic3RyYXRlIHJlY29nbml0aW9uIGFuZCBzcGxpY2Ugc2l0ZSBkZXRlcm1pbmF0aW9uIGluIHllYXN0IHRSTkEgc3BsaWNpbmc7IENlbGw7ICBjaXRhdGlvbl95ZWFyPTE5ODg7ICBjaXRhdGlvbl92b2x1bWU9NTU7ICBjaXRhdGlvbl9wYWdlcz03MTktNzMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaXRlIHNlbGVjdGlvbiBieSB0aGUgc3BsaWNpbmcgZW5kb251Y2xlYXNlIG9mIFhlbm9wdXMgTGFldmlzOyBDZWxsOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTU1OyAgY2l0YXRpb25fcGFnZXM9NzMxLTczODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSB0Uk5BKFRycCkgaW50cm9uIGVuZG9udWNsZWFzZSBmcm9tIEhhbG9iYWN0ZXJpdW0gdm9sY2FuaWkgLiBVbmlxdWUgc3Vic3RyYXRlIHJlY29nbml0aW9uIHByb3BlcnRpZXM7IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTI2MzsgIGNpdGF0aW9uX3BhZ2VzPTE3OTUxLTE3OTU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNvZ25pdGlvbiBvZiBleG9u4oCTaW50cm9uIGJvdW5kYXJpZXMgYnkgdGhlIEhhbG9iYWN0ZXJpdW0gdm9sY2FuaWkgdFJOQSBpbnRyb24gZW5kb251Y2xlYXNlOyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjU7ICBjaXRhdGlvbl9wYWdlcz0xODEwNC0xODExMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdGlvbiBhbmQgZXhjaXNpb24gb2YgYSBsYXJnZSBpbnRyb24gaW4gdGhlIHRSTkEgVHJwIGdlbmUgb2YgYW4gYXJjaGFlYmFjdGVyaXVtLCBIYWxvYmFjdGVyaXVtIHZvbGNhbmlpOyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk4NTsgIGNpdGF0aW9uX3ZvbHVtZT0yNjA7ICBjaXRhdGlvbl9wYWdlcz0zMTMyLTMxMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZXNlbmNlIG9mIGFuIGludHJvbiBpbiBlbG9uZ2F0b3IgbWV0aGlvbmluZS10Uk5BIG9mIEhhbG9iYWN0ZXJpdW0gdm9sY2FuaWk7IENhbi4gSi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPTE4OS0xOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVB1dGF0aXZlIGludHJvbnMgaW4gdFJOQSBnZW5lcyBvZiBwcm9rYXJ5b3RlczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk4MzsgIGNpdGF0aW9uX3ZvbHVtZT04MDsgIGNpdGF0aW9uX3BhZ2VzPTMzMDktMzMxMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSB1bmlxdWUgdFJOQSBpbnRyb24gaW4gdGhlIHZhcmlhYmxlIGxvb3Agb2YgdGhlIGV4dHJlbWUgdGhlcm1vcGhpbGUgVGhlcm1vZmlsbSBwZW5kZW5zIGFuZCBpdHMgcG9zc2libGUgZXZvbHV0aW9uYXJ5IGltcGxpY2F0aW9uczsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl92b2x1bWU9MjY0OyAgY2l0YXRpb25fcGFnZXM9MTc4MzQtMTc4Mzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmlzb24gb2YgdHJhbnNmZXIgUk5BIGFuZCByaWJvc29tYWwgUk5BIGludHJvbiBzcGxpY2luZyBpbiB0aGUgZXh0cmVtZSB0aGVybW9waGlsZSBhbmQgYXJjaGFlYmFjdGVyaXVtIERlc3VsZnVyb2NvY2N1cyBtb2JpbGlzOyBDYW4uIEouIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz0yMTAtMjE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lcyBmb3Igc3RhYmxlIFJOQSBpbiB0aGUgZXh0cmVtZSB0aGVybW9waGlsZSBUaGVybW9wcm90ZXVzIHRlbmF4IDogaW50cm9ucyBhbmQgdHJhbnNjcmlwdGlvbiBzaWduYWxzOyBFTUJPIEouOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz01MjMtNTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBnZW5vbWUgc2VxdWVuY2Ugb2YgYW4gYWVyb2JpYyBoeXBlci10aGVybW9waGlsaWMgY3JlbmFyY2hhZW90YSwgQWVyb3B5cnVtIHBlcm5peCBLMTsgY2l0YXRpb25fdGV4dD0xNDfigJMxNTU7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz04NC03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgZ2Vub21lIHNlcXVlbmNlIG9mIGFuIGFlcm9iaWMgdGhlcm1vYWNpZG9waGlsaWMgY3JlbmFyY2hhZW9uLCBTdWxmb2xvYnVzIHRva29kYWlpIHN0cmFpbjc7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xMjMtMTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZXJvcHVydW0gcGVybml4IGdlbi4gbm92Liwgc3AuIG5vdi4sIGEgbm92ZWwgYWVyb2JpYyBoeXBlciB0aGVybW9waGlsaWMgYXJjaGFlb24gZ3Jvd2luZyBhdCB0ZW1wZXJhdHVyZXMgdXAgdG8gMTAwwrBDOyBJbnQuIEouIFN5c3QuIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9NDY7ICBjaXRhdGlvbl9wYWdlcz0xMDcwLTEwNzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgbWV0aGFub2dlbmljIGFyY2hhZW9uLCBNZXRoYW5vY29jY3VzIGphbm5hc2NoaWk7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MjczOyAgY2l0YXRpb25fcGFnZXM9MTA1OC0xMDczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBnZW5vbWUgc2VxdWVuY2Ugb2YgTWV0aGFub2JhY3Rlcml1bSB0aGVybW9hdXRvdHJvcGhpY3VtIGRlbHRhSDogZnVuY3Rpb25hbCBhbmFseXNpcyBhbmQgY29tcGFyYXRpdmUgZ2Vub21pY3M7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MTc5OyAgY2l0YXRpb25fcGFnZXM9NzEzNS03MTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYXJ0aWNpcGF0aW9uIG9mIHRoZSBpbnRyb24gaW4gdGhlIHJlYWN0aW9uIGNhdGFseXplZCBieSB0aGUgWGVub3B1cyB0Uk5BIHNwbGljaW5nIGVuZG9udWNsZWFzZTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0yNTU7ICBjaXRhdGlvbl9wYWdlcz0xNDA0LTE0MDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIEJIQiBzcGxpY2luZyBtb3RpZnMgaW4gaW50cm9uLWNvbnRhaW5pbmcgdFJOQXMgZnJvbSAxOCBhcmNoYWVhOiBldm9sdXRpb25hcnkgaW1wbGljYXRpb25zOyBSTkE7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTE1MTYtMTUzMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGNoYXJhY3Rlcml6YXRpb24gYW5kIHBvc3RzcGxpY2luZyBmYXRlIG9mIHRocmVlIGludHJvbnMgd2l0aGluIHRoZSBzaW5nbGUgclJOQSBvcGVyb24gb2YgdGhlIGh5cGVydGhlcm1vcGhpbGljIGFyY2hhZW9uIEFlcm9weXJ1bSBwZXJuaXggSzE7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTgwOyAgY2l0YXRpb25fcGFnZXM9MzYzNS0zNjQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRyb25zIGluIHByb3RlaW4tY29kaW5nIGdlbmVzIGluIEFyY2hhZWE7IEZFQlMgTGV0dC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NTEwOyAgY2l0YXRpb25fcGFnZXM9MjctMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb3BlcnRpZXMgb2YgSC4gdm9sY2FuaWkgdFJOQSBpbnRyb24gZW5kb251Y2xlYXNlIHJldmVhbCBhIHJlbGF0aW9uc2hpcCBiZXR3ZWVuIHRoZSBhcmNoYWVhbCBhbmQgZXVjYXJ5YWwgdFJOQSBpbnRyb24gcHJvY2Vzc2luZyBzeXN0ZW1zOyBDZWxsOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTg5OyAgY2l0YXRpb25fcGFnZXM9ODM5LTg0ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIHRSTkEgbGlnYXNlLiBQdXJpZmljYXRpb24gb2YgdGhlIHByb3RlaW4gYW5kIGlzb2xhdGlvbiBvZiB0aGUgc3RydWN0dXJhbCBnZW5lOyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk4NjsgIGNpdGF0aW9uX3ZvbHVtZT0yNjE7ICBjaXRhdGlvbl9wYWdlcz0yOTc4LTI5ODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXRSTkFzY2FuLVNFOiBBIHByb2dyYW0gZm9yIGltcHJvdmVkIGRldGVjdGlvbiBvZiB0cmFuc2ZlciBSTkEgZ2VuZXMgaW4gZ2Vub21pYyBzZXF1ZW5jZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9OTU1LTk2NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJlIGFuZCB0cmFuc2NyaXB0aW9uIG9mIGV1a2FyeW90aWMgdFJOQSBnZW5lczsgQ1JDIENyaXQuIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0xOTg1OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MTA3LTE0NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uY2VydGVkIGNoYW5nZXMgaW4gdGhlIG51Y2xlb3RpZGUgc2VxdWVuY2VzIG9mIHRoZSBpbnRyYWdlbmljIHByb21vdGVyIHJlZ2lvbnMgb2YgZXVrYXJ5b3RpYyBnZW5lcyBmb3IgdFJOQXMgb2YgYWxsIHNwZWNpZmljaXRpZXM7IEouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NTc7ICBjaXRhdGlvbl9wYWdlcz01MjAtNTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTIvNi80MDMvNjcxODc0OS9kc2kwMjMucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gICBUaGUgdFJOQSBtb2xlY3VsZSBoYXMgYW4gaW1wb3J0YW50IHJvbGUgaW4gdHJhbnNsYXRpb24sIHRoZSBmdW5jdGlvbiBvZiB3aGljaCBpcyB0byBjYXJyeSBhbWlubyBhY2lkcyB0byB0aGUgcmlib3NvbWVzLiBJdCBpcyBrbm93biB0aGF0IHRSTkEgaXMgdHJhIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNDQ0NzYmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkZfQZXDscLXG2rzJHnmvakzlaDE4=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f6%2f403%2f344476" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f6%2f403%2f344476" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13225" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/12/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/6/0/m_dnaresearch12_6.cover.gif?Expires=2147483647&amp;Signature=vmBj-dEpEdSISGGZZVQQ800UdbaQ61VgN5SO6nMO30mtIcmmHegLeBqq~JmeB3Cl71F-HZSxEtC05awn7JLvbmS~R8ehKSKlG67TzUmDNfl2uJ9iwd8X3XVTg535TrTFouPO8FrSAc3H4exNAvHuAshR8~cghIB3WqxBjwF3q4V-gWs6yVOhRl523VyJV6-aueYhJA8pI1jFRHUmPhSY-RKzDUrwgfwrVrH153kk1IiVH3chlu~YmBF~XczCNeM~pNFI-0DwLreboeiSvqsHnGPMQGgfWos-lNOnEyjyXTWeiJBGXgUTUflQE1fArs2l0eo950LpKbLah37z6bPoSQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/12/6">
                        <div class="volume trailing-comma">Volume 12</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
2005        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="35870519">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#35870519">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="35870521">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#35870521">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="35870531">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#35870531">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="35870551">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#35870551">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="35870581">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#35870581">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="35870609">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#35870609">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/12/6/389/344376">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/12/6/417/343854">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main"> Characterization of a Whole Set of tRNA Molecules in an Aerobic Hyper-thermophilic Crenarchaeon, <em>Aeropyrum pernix</em> K1 
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Syuji Yamazaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">National Institute of Technology and Evaluation</div><div class="addr-line">2-49-10 Nishihara, Shibuya, Tokyo 151-0066, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Syuji+Yamazaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamazaki S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamazaki S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Syuji Yamazaki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hisashi Kikuchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">National Institute of Technology and Evaluation</div><div class="addr-line">2-49-10 Nishihara, Shibuya, Tokyo 151-0066, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hisashi+Kikuchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kikuchi H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kikuchi H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hisashi Kikuchi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yutaka Kawarabayasi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">National Institute of Advanced Industrial Science and Technology (AIST)</div><div class="addr-line">Higashi 1-1, Tsukuba, Ibaraki 305-8566, Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <sup>*</sup> To whom correspondence should be addressed. Tel: +81-29-861-6040, Fax: +81-29-861-6423, Email: <a href="mailto:kawarabayasi.yutaka@aist.go.jp" target="_blank">kawarabayasi.yutaka@aist.go.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yutaka+Kawarabayasi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawarabayasi Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kawarabayasi Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yutaka Kawarabayasi</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Communicated by Michio Oishi</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 12, Issue 6, 1 January 2005, Pages 403–416, <a href='https://doi.org/10.1093/dnares/dsi023'>https://doi.org/10.1093/dnares/dsi023</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 January 2005</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">13 August 2004</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">29 November 2005</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="344476" href="/dnaresearch/article-pdf/12/6/403/6718749/dsi023.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Syuji Yamazaki, Hisashi Kikuchi, Yutaka Kawarabayasi;   Characterization of a Whole Set of tRNA Molecules in an Aerobic Hyper-thermophilic Crenarchaeon, <em>Aeropyrum pernix</em> K1 , <em>DNA Research</em>, Volume 12, Issue 6, 1 January 2005, Pages 403–416, <a href='https://doi.org/10.1093/dnares/dsi023'>https://doi.org/10.1093/dnares/dsi023</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/344476?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/344476?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/344476?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/344476?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/344476?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/344476?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/344476?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=%20Characterization%20of%20a%20Whole%20Set%20of%20tRNA%20Molecules%20in%20an%20Aerobic%20Hyper-thermophilic%20Crenarchaeon%2C%20%26lt%3Bem%26gt%3BAeropyrum%20pernix%26lt%3B%2Fem%26gt%3B%20K1%20&amp;publicationDate=2005-01-01&amp;volumeNum=12&amp;issueNum=6&amp;author=Yamazaki%2C%20Syuji%3B%20Kikuchi%2C%20Hisashi&amp;startPage=403&amp;endPage=416&amp;contentId=10.1093%2Fdnares%2Fdsi023&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f6%2f403%2f344476" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=35870519 id="35870519" class="abstract-title" >Abstract</h2>
<section class="abstract"><p> The tRNA molecule has an important role in translation, the function of which is to carry amino acids to the ribosomes. It is known that tRNA is transcribed from tRNA genes, some of which, in Eukarya and Archaea, contain introns. A computational analysis of the complete genome of <em>Aeropyrum pernix</em> K1 predicted the presence of 14 intron-containing tRNA genes. To elucidate whether these introns are actually processed in living cells and what mechanism detects the intron regions, cDNAs for premature and mature forms of the tRNA molecules transcribed from the intron-containing tRNA genes in the model aerobic acidothermophilic crenarchaeon, <em>A. pernix</em> K1 were identified and analyzed. A comparison between the nucleotide sequences of these two types of cDNAs indicated that the intron regions of the tRNA molecules were indeed processed in <em>A. pernix</em> K1 living cells. Some cDNA clones showed that the actual splicing positions were different from those predicted by computational analysis. However, the bulge–helix–bulge structure, which has been previously identified in exon–intron boundaries of archaeal tRNA genes, was evident in all boundary regions confirmed in this work. These results indicate that the generally described mechanism for tRNA processing in Archaea is utilized for processing the intron region of the tRNA molecules in <em>A. pernix</em> K1. </p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="tRNA">tRNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="intron">intron</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="splicing">splicing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;bulge–helix–bulge structure&quot;">bulge–helix–bulge structure</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="genome">genome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Crenarchaeon">Crenarchaeon</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Aeropyrum pernix K1 &quot;"><em>Aeropyrum pernix</em> K1 </a></div>                            <h2 scrollto-destination=35870521 id="35870521" class="section-title" >1. Introduction</h2>
<p> The intron region, which is processed immediately after transcription, has been identified in protein-coding and RNA-coding genes in many organisms. As shown in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a> , the mechanisms for splicing of the introns already characterized were classified in the group I, II, III, nuclear mRNA and tRNA introns. </p>                        <a id="35870523" scrollto-destination="35870523"></a>
<div id="fig1" data-id="fig1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi023/3/m_dsi023f1.jpeg?Expires=2147483647&Signature=Sa7xwXmYcukDm6USnvR4D5dqCaGvuKs0~ksNTB4Y64o~Ytw2CZ8YLereeioULf4AHM~AJxqEQVo5PoZUdTdF-OputT4oMngMRQfT9VRymStdutbetySS8wkVxBTjmh-vcFPR0hNsj3uV9cj5mApCNR-cWDso5JhSk1kNlAETUcWyabKDSgZ7gBx1H8NuEVS8CiEcEEgDAhZQxUGaW-5BIs2OGz0Vqs9DSPS9uhlWB3eSnUWxQj4BrbKazCwaTgotNCuySMX4CHa2dzWcmp9djn12~6kapEycSV0zxQqUxCOBooPgUj7-RDdXfBSkK2FO0n2QhEhIE0JbseW8ExyMpA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Mechanisms for splicing of Group I, II, III and tRNA introns. Schematic outline of splicing mechanism for Group I intron is shown in A, for Group II, III and nuclear mRNA introns in B and for archaeal and nuclear tRNA introns in C. As shown in A, Group I intron requires the attack and cleavage by the hydroxyl group in GTP. In Group II or III intron, self-attacking by hydroxyl group in adenosine in intron region is necessary, as shown in B. In spite of these two mechanisms, the specific endonuclease is required for recognition and cleavage of intron portion in the archaeal and eukaryotic tRNA genes, and following enzymatic ligation is occurred." path-from-xml="dsi023f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35870523/dsi023f1.jpeg" path-from-xml="dsi023f1.jpeg" target="_blank">View large</a><a section="35870523" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi023/3/dsi023f1.jpeg?Expires=2147483647&Signature=vY-WE6AHv0tYv0Pg2P2JlFwXF-Xyq9uxDW-GBfQeyOOF3m3nEWWgEz3njZGbf8bZwUf1uf31WIKsR-NBAkd8fwCD8kh8LyML7jxDaQxaYHXoHrxpua95q996VUICbQmAw73Ki6rGWsPFR-G8CNtFCgpsHezTxj6YMOPUN56OzdMftY3S7aJbR5cZ-ff9Pf~-XYlSmCnYRbIJVgMIxeYV5LmEoqs-IFrqTN7MXOgDgxByY50fdOxJ-ahBbsdZFzW2e8u3M1u33ST-EqjizMRxYzfxAdcb-qczBsx4yNcB82lesbJgVMFitbdjiGvOE6LTUvK9YPUz3oog4P2XUCWJ6Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35870523&ar=344476&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi023f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Mechanisms for splicing of Group I, II, III and tRNA introns. Schematic outline of splicing mechanism for Group I intron is shown in A, for Group II, III and nuclear mRNA introns in B and for archaeal and nuclear tRNA introns in C. As shown in A, Group I intron requires the attack and cleavage by the hydroxyl group in GTP. In Group II or III intron, self-attacking by hydroxyl group in adenosine in intron region is necessary, as shown in B. In spite of these two mechanisms, the specific endonuclease is required for recognition and cleavage of intron portion in the archaeal and eukaryotic tRNA genes, and following enzymatic ligation is occurred.</p></div></div><div content-id="fig1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi023/3/m_dsi023f1.jpeg?Expires=2147483647&Signature=Sa7xwXmYcukDm6USnvR4D5dqCaGvuKs0~ksNTB4Y64o~Ytw2CZ8YLereeioULf4AHM~AJxqEQVo5PoZUdTdF-OputT4oMngMRQfT9VRymStdutbetySS8wkVxBTjmh-vcFPR0hNsj3uV9cj5mApCNR-cWDso5JhSk1kNlAETUcWyabKDSgZ7gBx1H8NuEVS8CiEcEEgDAhZQxUGaW-5BIs2OGz0Vqs9DSPS9uhlWB3eSnUWxQj4BrbKazCwaTgotNCuySMX4CHa2dzWcmp9djn12~6kapEycSV0zxQqUxCOBooPgUj7-RDdXfBSkK2FO0n2QhEhIE0JbseW8ExyMpA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Mechanisms for splicing of Group I, II, III and tRNA introns. Schematic outline of splicing mechanism for Group I intron is shown in A, for Group II, III and nuclear mRNA introns in B and for archaeal and nuclear tRNA introns in C. As shown in A, Group I intron requires the attack and cleavage by the hydroxyl group in GTP. In Group II or III intron, self-attacking by hydroxyl group in adenosine in intron region is necessary, as shown in B. In spite of these two mechanisms, the specific endonuclease is required for recognition and cleavage of intron portion in the archaeal and eukaryotic tRNA genes, and following enzymatic ligation is occurred." path-from-xml="dsi023f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35870523/dsi023f1.jpeg" path-from-xml="dsi023f1.jpeg" target="_blank">View large</a><a section="35870523" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi023/3/dsi023f1.jpeg?Expires=2147483647&Signature=vY-WE6AHv0tYv0Pg2P2JlFwXF-Xyq9uxDW-GBfQeyOOF3m3nEWWgEz3njZGbf8bZwUf1uf31WIKsR-NBAkd8fwCD8kh8LyML7jxDaQxaYHXoHrxpua95q996VUICbQmAw73Ki6rGWsPFR-G8CNtFCgpsHezTxj6YMOPUN56OzdMftY3S7aJbR5cZ-ff9Pf~-XYlSmCnYRbIJVgMIxeYV5LmEoqs-IFrqTN7MXOgDgxByY50fdOxJ-ahBbsdZFzW2e8u3M1u33ST-EqjizMRxYzfxAdcb-qczBsx4yNcB82lesbJgVMFitbdjiGvOE6LTUvK9YPUz3oog4P2XUCWJ6Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35870523&ar=344476&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi023f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Mechanisms for splicing of Group I, II, III and tRNA introns. Schematic outline of splicing mechanism for Group I intron is shown in A, for Group II, III and nuclear mRNA introns in B and for archaeal and nuclear tRNA introns in C. As shown in A, Group I intron requires the attack and cleavage by the hydroxyl group in GTP. In Group II or III intron, self-attacking by hydroxyl group in adenosine in intron region is necessary, as shown in B. In spite of these two mechanisms, the specific endonuclease is required for recognition and cleavage of intron portion in the archaeal and eukaryotic tRNA genes, and following enzymatic ligation is occurred.</p></div></div><p> It is known that introns in tRNA genes, which encode RNA molecules necessary to transfer the amino acids to ribosome, are identified in many organisms, Eukarya, Bacteria and Archaea. The tRNA introns in Bacteria can be classified into group I or II intron, that are processed with GTP or by intron itself without other factor, respectively ( <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1A and B</a> ). Conversely, the tRNA introns in Eukarya or Archaea lack any identifiable sequence or structure to group I, II, III or mRNA introns as indicated in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1C</a> . <a reveal-id="b1" data-open="b1" class="link link-ref link-reveal xref-bibr">1</a></p><p> Previous analysis indicated that tRNA introns in Eukarya were recognized their mature structure by tRNA endonuclease during the cleavage of the introns. <a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a> In this structure, the tRNA portion of the precursor maintains an L-shaped conformation, stabilized by the interaction between the D and TΨC loops. The 3′ splice site junction is always single-stranded. <a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a>,<em>In vitro</em> and <em>in vivo</em> analyses have shown that the cleavage sites of intron in eukaryotic tRNA molecules are identified by a measurement mechanism that senses the distance from the center of the molecule, possibly the top of the anticodon stem, to the 5′ and 3′ cleavage sites, as shown in <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2A</a> . <a reveal-id="b4" data-open="b4" class="link link-ref link-reveal xref-bibr">4</a> , <a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> Because of this mechanism for the processing of intron portion in Eukarya, the intron in eukaryotic tRNA genes is located only at one base 3′ from anticodon region. </p>                        <a id="35870526" scrollto-destination="35870526"></a>
<div id="fig2" data-id="fig2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi023/3/m_dsi023f2.jpeg?Expires=2147483647&Signature=PM4pD8WXxSMDDfa~9QVBRhRG~QPaf4uaCGeKKNboIyjVHkM~ekJSzj0XGn59ShMAYW9i0eySzwNuFrVkalMmZccg~KwtCuwqzqCnS8EIWACNlk9ylkMCVoSpqhoHpSc9xwezI1qvtVKaEj1D41TVESTonrCwooAytIVaVJXb8gdHjvo3uqVxff3sh6D967an7S4yZOCtJCm9QCOq8jK1qZZ~8wAWb~fKalvhNDNj0MY1Qo-SoDI8~13NqM4eYHufdU7tzEDtOi4ymo1GgdXHAs7Ap~Er6KKnS9yPhX86Ec5Z-LObGct4xqiggQV5a-4OaU5mKXrq1yc9oGX9vWPuHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Two different mechanisms recognizing introns in tRNA genes. A: In the measurement mechanism, which is utilized in eukaryotic tRNA genes, the enzymes processing the tRNA introns recognize the distance between anticodon stem and both cleavage sites. B: In the archaeal tRNA genes, the bulge–helix–bulge motif is necessary for recognition of splicing site. Splicing sites are indicated by short arrow. Y indicates pyrimidine nucleotides; R indicates purine nucleotides. O and X indicate the non-conserved nucleotides in the regions with conserved structure and non-conserved structure, respectively." path-from-xml="dsi023f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35870526/dsi023f2.jpeg" path-from-xml="dsi023f2.jpeg" target="_blank">View large</a><a section="35870526" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi023/3/dsi023f2.jpeg?Expires=2147483647&Signature=ax9j3qruI4UmuY--UpO86o-X5sPtCP3ydqyPOJMos0TvktKyQamUPLy3oW-~o-rliwAAGjahFAWY1pFdO2CJUPSbcbDrtjDrebxYT~gy9lIA7xMqri9Le8q~v2Uml5HDnZQ3-WYL4S05bDk0TX5U4poey3~9OQj2StpFw1q1MBMKOSfLyAv-sRA9BJ-OpmefQq5bMvEi4WhG8LsHmfVsEHNOaSFvB6qpKp97Mti520lBz~AnDupjIxNdKXLD6~cU4rWKPt9sZDPr1bdp8Rb7SyOxNm6VqqRLT1HAKRJHNgTrqRCOjqCg4dMj4tjYh3CSGiOHj8O~IMfNf2gSRSZOcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35870526&ar=344476&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi023f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Two different mechanisms recognizing introns in tRNA genes. A: In the measurement mechanism, which is utilized in eukaryotic tRNA genes, the enzymes processing the tRNA introns recognize the distance between anticodon stem and both cleavage sites. B: In the archaeal tRNA genes, the bulge–helix–bulge motif is necessary for recognition of splicing site. Splicing sites are indicated by short arrow. Y indicates pyrimidine nucleotides; R indicates purine nucleotides. O and X indicate the non-conserved nucleotides in the regions with conserved structure and non-conserved structure, respectively.</p></div></div><div content-id="fig2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi023/3/m_dsi023f2.jpeg?Expires=2147483647&Signature=PM4pD8WXxSMDDfa~9QVBRhRG~QPaf4uaCGeKKNboIyjVHkM~ekJSzj0XGn59ShMAYW9i0eySzwNuFrVkalMmZccg~KwtCuwqzqCnS8EIWACNlk9ylkMCVoSpqhoHpSc9xwezI1qvtVKaEj1D41TVESTonrCwooAytIVaVJXb8gdHjvo3uqVxff3sh6D967an7S4yZOCtJCm9QCOq8jK1qZZ~8wAWb~fKalvhNDNj0MY1Qo-SoDI8~13NqM4eYHufdU7tzEDtOi4ymo1GgdXHAs7Ap~Er6KKnS9yPhX86Ec5Z-LObGct4xqiggQV5a-4OaU5mKXrq1yc9oGX9vWPuHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Two different mechanisms recognizing introns in tRNA genes. A: In the measurement mechanism, which is utilized in eukaryotic tRNA genes, the enzymes processing the tRNA introns recognize the distance between anticodon stem and both cleavage sites. B: In the archaeal tRNA genes, the bulge–helix–bulge motif is necessary for recognition of splicing site. Splicing sites are indicated by short arrow. Y indicates pyrimidine nucleotides; R indicates purine nucleotides. O and X indicate the non-conserved nucleotides in the regions with conserved structure and non-conserved structure, respectively." path-from-xml="dsi023f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35870526/dsi023f2.jpeg" path-from-xml="dsi023f2.jpeg" target="_blank">View large</a><a section="35870526" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi023/3/dsi023f2.jpeg?Expires=2147483647&Signature=ax9j3qruI4UmuY--UpO86o-X5sPtCP3ydqyPOJMos0TvktKyQamUPLy3oW-~o-rliwAAGjahFAWY1pFdO2CJUPSbcbDrtjDrebxYT~gy9lIA7xMqri9Le8q~v2Uml5HDnZQ3-WYL4S05bDk0TX5U4poey3~9OQj2StpFw1q1MBMKOSfLyAv-sRA9BJ-OpmefQq5bMvEi4WhG8LsHmfVsEHNOaSFvB6qpKp97Mti520lBz~AnDupjIxNdKXLD6~cU4rWKPt9sZDPr1bdp8Rb7SyOxNm6VqqRLT1HAKRJHNgTrqRCOjqCg4dMj4tjYh3CSGiOHj8O~IMfNf2gSRSZOcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35870526&ar=344476&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi023f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Two different mechanisms recognizing introns in tRNA genes. A: In the measurement mechanism, which is utilized in eukaryotic tRNA genes, the enzymes processing the tRNA introns recognize the distance between anticodon stem and both cleavage sites. B: In the archaeal tRNA genes, the bulge–helix–bulge motif is necessary for recognition of splicing site. Splicing sites are indicated by short arrow. Y indicates pyrimidine nucleotides; R indicates purine nucleotides. O and X indicate the non-conserved nucleotides in the regions with conserved structure and non-conserved structure, respectively.</p></div></div><p> In contrast to eukaryotic mechanism, it is found that the recognition and cleavage of the intron region by the archaeal intron endonuclease does not require the complete mature tRNA structure, but requires a defined structural element at the exon–intron boundaries, referred to as the bulge–helix–bulge motif shown in <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2B</a> . <a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> , <a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a> This mechanism is well suited for the fact that archaeal tRNA introns are not restricted to a single location. The previous researches were targeted toward some specific intron-containing tRNA molecules in Archaea, which led to the discovery of an archaeal tRNA intron in the anticodon loop, the anticodon stem and the extra arm. <a reveal-id="b8  b13" data-open="b8  b13" class="link link-ref link-reveal xref-bibr">8–13</a> The previous analyses of the splicing mechanism of the tRNA introns in Archaea have been also performed for the restricted tRNA genes in some archaeal species, <a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> most of which were halophilic Archaea. <a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> , <a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a> Experimental analysis of splicing mechanism of archaeal tRNA intron and experimental confirmation of splicing pattern of tRNA molecule using the entire set of tRNA genes in one microorganism have not been previously performed. Hence, it is necessary to examine the entire tRNA system in one microorganism in order to understand the actual expression and splicing mechanism in the complete tRNA system. </p><p> To date, the entire nucleotide sequences for 16 archaeal genomes have been determined. From this genomic data, the tRNA genes, that were interrupted by an intron, were identified in both the Euryarchaeota and the Crenarchaeota. Though the genomes of the Euryarchaeota contain a limited number of interrupted tRNA genes, computational analysis predicted 14 and 24 interrupted tRNA genes in two Crenarchaea, <em>Aeropyrum pernix</em> K1 and <em>Sulfolobus tokodaii</em> strain7, respectively. <a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> , <a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a> Also noteworthy was the observation that tRNA genes containing an intron longer than 40 nt were predicted in both genomes and that the tRNA genes interrupted by the intron at the other position than the general one, one base 3′ from anticodon region, were also predicted in both genomes. This data led to two questions: (i) Are these introns isolated in archaeal tRNA genes actually cleaved out during tRNA processing in living cells? (ii) Are the all intron regions in tRNA genes, including those located at other position than one base 3′ from anticodon region, correctly recognized by the tRNA–intron splicing machinery in archaeal living cells? </p><p> In this study, to determine whether all tRNA molecules predicted were expressed, whether the intron regions predicted were actually processed in archaeal living cells and the mechanism for tRNA–intron splicing in Archaea, <em>A. pernix</em> K1 was selected as a target. The reasons why this archaeon was selected are shown in the following: (i) the entire genomic sequence was already determined <a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> ; (ii) 14 tRNA genes were identified as interrupted tRNA genes in this genome; (iii) tRNA genes that contain the long intron or the introns within anticodon loop and anticodon stem were predicted and (iv) this microorganism can be easily cultured in aerobic conditions. The cDNA molecules were synthesized from tRNA molecules of this Crenarchaeon by RT–PCR using the specific primers designed from the genomic data. </p><p> The plasmid clones containing the cDNA fragments were used for determination of the nucleotide sequences of the mature and premature types of the cDNA molecules. A comparison between these two types of sequences indicated that the introns predicted in tRNA genes of <em>A. pernix</em> K1 were actually processed in living cells and that the bulge–helix–bulge motif was required for recognition of exon–intron boundaries and cleavage of the intron region in <em>A. pernix</em> K1 living cells. </p>                            <h2 scrollto-destination=35870531 id="35870531" class="section-title" >2. Materials and Methods</h2>
                            <h3 scrollto-destination=35870532 id="35870532" class="section-title" >2.1. Bacterial strains and vector</h3>
<p> The <em>A. pernix</em> strain K1, deposited in the Japan Collection of Microorganisms (JCM number 9820), was cultured using JXT medium. <a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a><em>Escherichia coli</em> TOP10F′ (Invitrogen, Carlsbad, CA, USA) was used for cloning and preparation of plasmid DNA. pCR2.1 Vector (Invitrogen, Carlsbad, CA, USA) was used for the cloning of fragments amplified by PCR. </p>                            <h3 scrollto-destination=35870534 id="35870534" class="section-title" >2.2. Preparation of total RNA and small-sized RNA</h3>
<p> From <em>A. pernix</em> K1 cells, which were grown to late log phase in 40 ml of JXT medium, <a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a> a crude total RNA sample was prepared with Trizol Reagent (Invitrogen, Carlsbad, CA, USA) according to the manufacturer's instruction. The prepared RNA was dissolved in 10 µl of RNase-free water. To remove the contaminated DNA, the total RNA was incubated with 10 U of the RNase-free DNase I (Worthington biochemical, Lakewood, NJ, USA) in 50 µl of the reaction buffer [50 mM Tris–HCl (pH 7.8), 1 mM MgCl <sub>2</sub> and 1 mM CaCl <sub>2</sub> ] at 37°C for 30 min. After incubation, purified RNA was treated by an equal volume of phenol chloroform solution and precipitated with ethanol. The recovered RNA molecules were dissolved in 10 µl of RNase-free water and stored at −80°C. </p><p>A mixture of small-sized RNA, including the crude tRNA and 5S RNA, was prepared from 40 µg of total RNA by fractionation with the QIAGEN RNA/DNA Kit (QIAGEN, Hilden, Germany) according to the manufacturer's instruction.</p>                            <h3 scrollto-destination=35870537 id="35870537" class="section-title" >2.3. Preparation of the beads covalently binding the sequence-specific DNA molecules</h3>
<p> To enrich the less abundant tRNA molecules, the specific tRNA molecules were purified by affinity chromatography. Magnetic beads covalently binding the poly(dT) oligo nucleotide were used as the starting materials. Since the nucleotide sequence of the 3′ half of tRNA <sup>Ser</sup> (CGA) and tRNA <sup>Ser</sup> (UGA) were initially same, two 40mer oligo nucleotides, which contained the 3′ half of tRNA <sup>Trp</sup> (CCA) and tRNA <sup>Ser</sup> (CGA) or tRNA <sup>Ser</sup> (UGA), with a 10 nt long poly(dA) sequences at the 3′ end of this oligo DNA were synthesized. The sequences of the synthesized DNA are GACCCGTAGGTCGGGGGTTCAAATCCCCCCGGGCCCACCAAAAAAAAAAA for tRNA <sup>Trp</sup> (CCA), and CGCGAGGGGCGCGCGGGTTCGAATCCCGCCCCCGGCGCCAAAAAAAAAAA for tRNA <sup>Ser</sup> (CGA) and tRNA <sup>Ser</sup> (UGA). </p><p> For construction of the matrix for affinity chromatography, 10 µg of synthesized oligo nucleotides were heated at 94°C for 3 min with 15 µl of the Oligotex™-MAG (Takara, Kyoto, Japan) in 50 µl of the kit's 2× binding buffer. After treatment, the solution was slowly cooled to room temperature and the Oligotex™-MAG beads were washed with 350 µl of the kit's wash solution to remove any non-hybridized oligo DNA. For synthesis of the DNA strands complimentary to the oligo DNA on the beads, the Oligotex™-MAG beads with hybridized oligo DNA were incubated in 80 µl of klenow DNA polymerase buffer [10 mM Tris–HCl (pH 7.5), 7 mM MgCl <sub>2</sub> , 0.1 mM DTT and 0.125 mM each of dATP, dGTP, dCTP and dTTP] at 37°C for 30 min with 12 U of klenow DNA polymerase. After incubation, the Oligotex™-MAG beads in the solution were heated at 94°C for 2 min and quickly chilled. The resulting magnetic beads covalently bind the tRNA-specific DNA strand, and hence were used for affinity concentration of the less-abundant tRNA molecules from small-sized tRNA. </p>                            <h3 scrollto-destination=35870540 id="35870540" class="section-title" >2.4. Purification of the species-specific tRNA molecules</h3>
<p>Small-sized RNA (0.5 µg) was incubated at 94°C for 3 min with the Oligotex™-MAG beads prepared as above in 50 µl of 2× binding buffer used in the Oligotex™-MAG kit. After heating, the solution was incubated at 25°C for 10 min. The non-hybridized tRNA molecules were removed by washing twice with 350 µl of the kit's wash buffer. The species-specific tRNA molecules were recovered in 20 µl of RNase-free pure water by heating at 94°C for 2 min and quickly chilled in ice water. Recovered species-specific concentrated tRNA molecules were stored at −80°C and used for the synthesis of species-specific cDNA.</p>                            <h3 scrollto-destination=35870542 id="35870542" class="section-title" >2.5. RT–PCR</h3>
<p> Synthesis and amplification of cDNA for specific tRNA molecules were carried out by the Thermoscript™ RT–PCR System (Invitrogen, Carlsbad, CA, USA) according to the supplier's instruction with some modification. One microgram of total RNA, 1 µg of small-sized RNA or 0.05 µg of species-specific concentrated RNA and 10 pmol of primer were incubated at 95°C for 5 min in 19 µl of Thermoscript™ RT–PCR buffer [50 mM Tris–acetate (pH 8.4), 75 mM potassium acetate, 8 mM magnesium acetate, stabilizer, 5 mM DTT, 40 U RNaseOUT (Invitrogen, Carlsbad, CA, USA) and 1 mM each of dATP, dGTP, dCTP and dTTP] and cooled to the <em>T</em><sub>m</sub> of each primer set used. After cooling to this temperature, 15 U ThermoscriptRT enzyme was added and the reaction mixture was kept at this temperature for 30 min to allow synthesis of the first strand of cDNA. </p><p> The subsequent PCR amplification was performed in 50 µl of reaction mixture containing 2 µl of solution from the first reaction, 20 mM Tris–HCl (pH 8.4), 50 mM KCl, 1.5 mM MgCl <sub>2</sub> , 0.2 mM each of dATP, dGTP, dCTP and dTTP, 10 pmol of the appropriate primer set (as detailed in <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Tables 1</a> and <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">2</a> ) and 2 U of Platinum <em>Taq</em> DNA polymerase (Invitrogen, Carlsbad, CA, USA). The PCR was performed under standard conditions as specified in the manual accompanying the Thermoscript™ RT–PCR kit. After purification of the amplified fragments by 3% Nusive GTG agarose (FMC, Rockland, MD, USA) gel electrophoresis, the fragments were dissolved in the 10 µl of RNase-free water and stored in −20°C. </p>                        <a id="35870545" scrollto-destination="35870545"></a>
<div content-id="tbl1" class="table-modal"><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>List of primers used for RT–PCR of tRNA molecules</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="2">(A) Primers for synthesis of cDNA </th><th colspan="2">(B) Primers for amplification by PCR </th></tr><tr><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th></tr><tr><th>Primer name </th><th>Sequences </th><th>Primer name </th><th>Sequences </th></tr></thead><tbody><tr><td>cP01 </td><td>TGGAGCCGCCGGCGGGAT </td><td>aP01 </td><td>GCCGCCGTAGCTCAGCGG </td></tr><tr><td>cP02 </td><td>TGGTGGGCCCGCGGGGAT </td><td>aP02 </td><td>GGGCCCGTCGTCTAGCCT </td></tr><tr><td>cP03 </td><td>TGGCGGGCCCGCCGGGAT </td><td>aP03 </td><td>GGGCCCGTAGCTCAGCCT </td></tr><tr><td>cP04 </td><td>TGGTGCCGCCGCCGGGAT </td><td>aP04 </td><td>GCCGCCGTAGCTCAGCAG </td></tr><tr><td>cP05 </td><td>TGGTGGACCGGCCGGGAT </td><td>aP05 </td><td>GCCGCCGTAGCTCAGCCT </td></tr><tr><td>cP06 </td><td>TGGTGGGGCCGCGGGGAT </td><td>aP06 </td><td>GGGCCGGTAGCTCAGCCT </td></tr><tr><td>cP07 </td><td>TGGCGCCGGGGGCGGGAT </td><td>aP07 </td><td>GCCGGGGTGGCCGAGCGG </td></tr><tr><td>cP08 </td><td>TGGTGCGGGGGGTGGGAT </td><td>aP08 </td><td>GCGGGGGTGCCCGAGCCT </td></tr><tr><td>cP09 </td><td>TGGCGCCGCGGCCGGGAT </td><td>aP09 </td><td>GCCGCGGTAGTATAGCCT </td></tr><tr><td>cP10 </td><td>TGGCGCCGGGGAGGGGAC </td><td>aP10 </td><td>GCCGGGGTAGCTCAGCTA </td></tr><tr><td>cP11 </td><td>TGGTGGGCCCGGGGGGAT </td><td>aP11 </td><td>GGGCCCGTAGCTCAGCCA </td></tr><tr><td>cP12 </td><td>TGGTGCGGCGGCCGGGAT </td><td>aP12 </td><td>GCGGGGGTGCCCGAGCCA </td></tr><tr><td>cP13 </td><td>TGGAGCCGCCGGGATTTA </td><td>aP13 </td><td>GCGGCGGTCGTCTAGCCT </td></tr><tr><td>cP14 </td><td>TGGTAGCCGGGCCGGGAT </td><td>aP14 </td><td>CCCGCCGTAGCTCAGCGG </td></tr><tr><td>cP15 </td><td>TGGTAGCGGGGGGTGGAT </td><td>aP15 </td><td>CACGCAGTAGCTAAGCTG </td></tr><tr><td>cP16 </td><td>TGGTGCCGCGGCCGGGAT </td><td>aP16 </td><td>AGCCGGGTCGTCTAGCGG </td></tr><tr><td>cP17 </td><td>TGGTGGGGCCGCCGGGAT </td><td>aP17 </td><td>AGCGGGGTAGGGTAGCCT </td></tr><tr><td>cP18 </td><td>TGGAGCCGGGGCCGGGAT </td><td>aP18 </td><td>GCCGCGGTAGTATAGCCC </td></tr><tr><td>cP19 </td><td>TGGGGCCCGGGCCGGGAT </td><td>aP19 </td><td>GGGGCCGTCGTCTAGCCT </td></tr><tr><td>cP20 </td><td>TGGTCCCGCCGCGGGGAT </td><td>aP20 </td><td>GGGCCCGTAGCTCAGCCC </td></tr><tr><td>cP21 </td><td>TGGTAGCCGGGCGGGGAT </td><td>aP21 </td><td>GCCCGGGTAGCTCAGCCT </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="2">(A) Primers for synthesis of cDNA </th><th colspan="2">(B) Primers for amplification by PCR </th></tr><tr><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th></tr><tr><th>Primer name </th><th>Sequences </th><th>Primer name </th><th>Sequences </th></tr></thead><tbody><tr><td>cP01 </td><td>TGGAGCCGCCGGCGGGAT </td><td>aP01 </td><td>GCCGCCGTAGCTCAGCGG </td></tr><tr><td>cP02 </td><td>TGGTGGGCCCGCGGGGAT </td><td>aP02 </td><td>GGGCCCGTCGTCTAGCCT </td></tr><tr><td>cP03 </td><td>TGGCGGGCCCGCCGGGAT </td><td>aP03 </td><td>GGGCCCGTAGCTCAGCCT </td></tr><tr><td>cP04 </td><td>TGGTGCCGCCGCCGGGAT </td><td>aP04 </td><td>GCCGCCGTAGCTCAGCAG </td></tr><tr><td>cP05 </td><td>TGGTGGACCGGCCGGGAT </td><td>aP05 </td><td>GCCGCCGTAGCTCAGCCT </td></tr><tr><td>cP06 </td><td>TGGTGGGGCCGCGGGGAT </td><td>aP06 </td><td>GGGCCGGTAGCTCAGCCT </td></tr><tr><td>cP07 </td><td>TGGCGCCGGGGGCGGGAT </td><td>aP07 </td><td>GCCGGGGTGGCCGAGCGG </td></tr><tr><td>cP08 </td><td>TGGTGCGGGGGGTGGGAT </td><td>aP08 </td><td>GCGGGGGTGCCCGAGCCT </td></tr><tr><td>cP09 </td><td>TGGCGCCGCGGCCGGGAT </td><td>aP09 </td><td>GCCGCGGTAGTATAGCCT </td></tr><tr><td>cP10 </td><td>TGGCGCCGGGGAGGGGAC </td><td>aP10 </td><td>GCCGGGGTAGCTCAGCTA </td></tr><tr><td>cP11 </td><td>TGGTGGGCCCGGGGGGAT </td><td>aP11 </td><td>GGGCCCGTAGCTCAGCCA </td></tr><tr><td>cP12 </td><td>TGGTGCGGCGGCCGGGAT </td><td>aP12 </td><td>GCGGGGGTGCCCGAGCCA </td></tr><tr><td>cP13 </td><td>TGGAGCCGCCGGGATTTA </td><td>aP13 </td><td>GCGGCGGTCGTCTAGCCT </td></tr><tr><td>cP14 </td><td>TGGTAGCCGGGCCGGGAT </td><td>aP14 </td><td>CCCGCCGTAGCTCAGCGG </td></tr><tr><td>cP15 </td><td>TGGTAGCGGGGGGTGGAT </td><td>aP15 </td><td>CACGCAGTAGCTAAGCTG </td></tr><tr><td>cP16 </td><td>TGGTGCCGCGGCCGGGAT </td><td>aP16 </td><td>AGCCGGGTCGTCTAGCGG </td></tr><tr><td>cP17 </td><td>TGGTGGGGCCGCCGGGAT </td><td>aP17 </td><td>AGCGGGGTAGGGTAGCCT </td></tr><tr><td>cP18 </td><td>TGGAGCCGGGGCCGGGAT </td><td>aP18 </td><td>GCCGCGGTAGTATAGCCC </td></tr><tr><td>cP19 </td><td>TGGGGCCCGGGCCGGGAT </td><td>aP19 </td><td>GGGGCCGTCGTCTAGCCT </td></tr><tr><td>cP20 </td><td>TGGTCCCGCCGCGGGGAT </td><td>aP20 </td><td>GGGCCCGTAGCTCAGCCC </td></tr><tr><td>cP21 </td><td>TGGTAGCCGGGCGGGGAT </td><td>aP21 </td><td>GCCCGGGTAGCTCAGCCT </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/35870545" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>List of primers used for RT–PCR of tRNA molecules</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="2">(A) Primers for synthesis of cDNA </th><th colspan="2">(B) Primers for amplification by PCR </th></tr><tr><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th></tr><tr><th>Primer name </th><th>Sequences </th><th>Primer name </th><th>Sequences </th></tr></thead><tbody><tr><td>cP01 </td><td>TGGAGCCGCCGGCGGGAT </td><td>aP01 </td><td>GCCGCCGTAGCTCAGCGG </td></tr><tr><td>cP02 </td><td>TGGTGGGCCCGCGGGGAT </td><td>aP02 </td><td>GGGCCCGTCGTCTAGCCT </td></tr><tr><td>cP03 </td><td>TGGCGGGCCCGCCGGGAT </td><td>aP03 </td><td>GGGCCCGTAGCTCAGCCT </td></tr><tr><td>cP04 </td><td>TGGTGCCGCCGCCGGGAT </td><td>aP04 </td><td>GCCGCCGTAGCTCAGCAG </td></tr><tr><td>cP05 </td><td>TGGTGGACCGGCCGGGAT </td><td>aP05 </td><td>GCCGCCGTAGCTCAGCCT </td></tr><tr><td>cP06 </td><td>TGGTGGGGCCGCGGGGAT </td><td>aP06 </td><td>GGGCCGGTAGCTCAGCCT </td></tr><tr><td>cP07 </td><td>TGGCGCCGGGGGCGGGAT </td><td>aP07 </td><td>GCCGGGGTGGCCGAGCGG </td></tr><tr><td>cP08 </td><td>TGGTGCGGGGGGTGGGAT </td><td>aP08 </td><td>GCGGGGGTGCCCGAGCCT </td></tr><tr><td>cP09 </td><td>TGGCGCCGCGGCCGGGAT </td><td>aP09 </td><td>GCCGCGGTAGTATAGCCT </td></tr><tr><td>cP10 </td><td>TGGCGCCGGGGAGGGGAC </td><td>aP10 </td><td>GCCGGGGTAGCTCAGCTA </td></tr><tr><td>cP11 </td><td>TGGTGGGCCCGGGGGGAT </td><td>aP11 </td><td>GGGCCCGTAGCTCAGCCA </td></tr><tr><td>cP12 </td><td>TGGTGCGGCGGCCGGGAT </td><td>aP12 </td><td>GCGGGGGTGCCCGAGCCA </td></tr><tr><td>cP13 </td><td>TGGAGCCGCCGGGATTTA </td><td>aP13 </td><td>GCGGCGGTCGTCTAGCCT </td></tr><tr><td>cP14 </td><td>TGGTAGCCGGGCCGGGAT </td><td>aP14 </td><td>CCCGCCGTAGCTCAGCGG </td></tr><tr><td>cP15 </td><td>TGGTAGCGGGGGGTGGAT </td><td>aP15 </td><td>CACGCAGTAGCTAAGCTG </td></tr><tr><td>cP16 </td><td>TGGTGCCGCGGCCGGGAT </td><td>aP16 </td><td>AGCCGGGTCGTCTAGCGG </td></tr><tr><td>cP17 </td><td>TGGTGGGGCCGCCGGGAT </td><td>aP17 </td><td>AGCGGGGTAGGGTAGCCT </td></tr><tr><td>cP18 </td><td>TGGAGCCGGGGCCGGGAT </td><td>aP18 </td><td>GCCGCGGTAGTATAGCCC </td></tr><tr><td>cP19 </td><td>TGGGGCCCGGGCCGGGAT </td><td>aP19 </td><td>GGGGCCGTCGTCTAGCCT </td></tr><tr><td>cP20 </td><td>TGGTCCCGCCGCGGGGAT </td><td>aP20 </td><td>GGGCCCGTAGCTCAGCCC </td></tr><tr><td>cP21 </td><td>TGGTAGCCGGGCGGGGAT </td><td>aP21 </td><td>GCCCGGGTAGCTCAGCCT </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="2">(A) Primers for synthesis of cDNA </th><th colspan="2">(B) Primers for amplification by PCR </th></tr><tr><th colspan="2"><hr /> </th><th colspan="2"><hr /> </th></tr><tr><th>Primer name </th><th>Sequences </th><th>Primer name </th><th>Sequences </th></tr></thead><tbody><tr><td>cP01 </td><td>TGGAGCCGCCGGCGGGAT </td><td>aP01 </td><td>GCCGCCGTAGCTCAGCGG </td></tr><tr><td>cP02 </td><td>TGGTGGGCCCGCGGGGAT </td><td>aP02 </td><td>GGGCCCGTCGTCTAGCCT </td></tr><tr><td>cP03 </td><td>TGGCGGGCCCGCCGGGAT </td><td>aP03 </td><td>GGGCCCGTAGCTCAGCCT </td></tr><tr><td>cP04 </td><td>TGGTGCCGCCGCCGGGAT </td><td>aP04 </td><td>GCCGCCGTAGCTCAGCAG </td></tr><tr><td>cP05 </td><td>TGGTGGACCGGCCGGGAT </td><td>aP05 </td><td>GCCGCCGTAGCTCAGCCT </td></tr><tr><td>cP06 </td><td>TGGTGGGGCCGCGGGGAT </td><td>aP06 </td><td>GGGCCGGTAGCTCAGCCT </td></tr><tr><td>cP07 </td><td>TGGCGCCGGGGGCGGGAT </td><td>aP07 </td><td>GCCGGGGTGGCCGAGCGG </td></tr><tr><td>cP08 </td><td>TGGTGCGGGGGGTGGGAT </td><td>aP08 </td><td>GCGGGGGTGCCCGAGCCT </td></tr><tr><td>cP09 </td><td>TGGCGCCGCGGCCGGGAT </td><td>aP09 </td><td>GCCGCGGTAGTATAGCCT </td></tr><tr><td>cP10 </td><td>TGGCGCCGGGGAGGGGAC </td><td>aP10 </td><td>GCCGGGGTAGCTCAGCTA </td></tr><tr><td>cP11 </td><td>TGGTGGGCCCGGGGGGAT </td><td>aP11 </td><td>GGGCCCGTAGCTCAGCCA </td></tr><tr><td>cP12 </td><td>TGGTGCGGCGGCCGGGAT </td><td>aP12 </td><td>GCGGGGGTGCCCGAGCCA </td></tr><tr><td>cP13 </td><td>TGGAGCCGCCGGGATTTA </td><td>aP13 </td><td>GCGGCGGTCGTCTAGCCT </td></tr><tr><td>cP14 </td><td>TGGTAGCCGGGCCGGGAT </td><td>aP14 </td><td>CCCGCCGTAGCTCAGCGG </td></tr><tr><td>cP15 </td><td>TGGTAGCGGGGGGTGGAT </td><td>aP15 </td><td>CACGCAGTAGCTAAGCTG </td></tr><tr><td>cP16 </td><td>TGGTGCCGCGGCCGGGAT </td><td>aP16 </td><td>AGCCGGGTCGTCTAGCGG </td></tr><tr><td>cP17 </td><td>TGGTGGGGCCGCCGGGAT </td><td>aP17 </td><td>AGCGGGGTAGGGTAGCCT </td></tr><tr><td>cP18 </td><td>TGGAGCCGGGGCCGGGAT </td><td>aP18 </td><td>GCCGCGGTAGTATAGCCC </td></tr><tr><td>cP19 </td><td>TGGGGCCCGGGCCGGGAT </td><td>aP19 </td><td>GGGGCCGTCGTCTAGCCT </td></tr><tr><td>cP20 </td><td>TGGTCCCGCCGCGGGGAT </td><td>aP20 </td><td>GGGCCCGTAGCTCAGCCC </td></tr><tr><td>cP21 </td><td>TGGTAGCCGGGCGGGGAT </td><td>aP21 </td><td>GCCCGGGTAGCTCAGCCT </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/35870545" target="_blank">View Large</a></div></div>                        <a id="35870546" scrollto-destination="35870546"></a>
<div content-id="tbl2" class="table-modal"><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Primer sets used for amplification of each tRNA molecule</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">Primer sets </th><th colspan="4">tRNA species </th></tr></thead><tbody><tr><td>1 </td><td>cP05 </td><td>aP06 </td><td> tRNA <sup>Ala</sup> (CGC)  </td><td> tRNA <sup>Ala</sup> (GGC)  </td><td> tRNA <sup>Ala</sup> (UGC)  </td><td> </td></tr><tr><td>2 </td><td>cP06 </td><td>aP19 </td><td> tRNA <sup>Pro</sup> (UGG)  </td><td> </td><td> </td><td> </td></tr><tr><td>3 </td><td>cP08 </td><td>aP08 </td><td> tRNA <sup>Leu</sup> (UAG)  </td><td> tRNA <sup>Leu</sup> (UAA)  </td><td> tRNA <sup>Leu</sup> (GAG)  </td><td> tRNA <sup>Leu</sup> (CAG)  </td></tr><tr><td>4 </td><td>cP10 </td><td>aP10 </td><td> tRNA <sup>Asn</sup> (GUU)  </td><td> </td><td> </td><td> </td></tr><tr><td>5 </td><td>cP08 </td><td>aP12 </td><td> tRNA <sup>Leu</sup> (CAA)  </td><td> </td><td> </td><td> </td></tr><tr><td>6 </td><td>cP12 </td><td>aP13 </td><td> tRNA <sup>Gly</sup> (GCC)  </td><td> </td><td> </td><td> </td></tr><tr><td>7 </td><td>cP11 </td><td>aP03 </td><td> tRNA <sup>Ile</sup> (GAU)  </td><td> </td><td> </td><td> </td></tr><tr><td>8 </td><td>cP13 </td><td>aP15 </td><td> tRNA <sup>Thr</sup> (UGU-2) <a reveal-id="tf2-1" data-open="tf2-1" class="link link-ref link-reveal xref-fn">a</a> </td><td> </td><td> </td><td> </td></tr><tr><td>9 </td><td>cP14 </td><td>aP16 </td><td> tRNA <sup>Gln</sup> (UUG)  </td><td> </td><td> </td><td> </td></tr><tr><td>10 </td><td>cP15 </td><td>aP17 </td><td> tRNA <sup>Met</sup> (CAU-3) <a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">b</a> </td><td> </td><td> </td><td> </td></tr><tr><td>11 </td><td>cP16 </td><td>aP18 </td><td> tRNA <sup>Glu</sup> (CUC)  </td><td> tRNA <sup>Glu</sup> (UUC)  </td><td> </td><td> </td></tr><tr><td>12 </td><td>cP19 </td><td>aP21 </td><td> tRNA <sup>His</sup> (GUG)  </td><td> </td><td> </td><td> </td></tr><tr><td>13 </td><td>cP21 </td><td>aP16 </td><td> tRNA <sup>Gln</sup> (CUG)  </td><td> tRNA <sup>Thr</sup> (GGU)  </td><td> </td><td> </td></tr><tr><td>14 </td><td>cP03 </td><td>aP11 </td><td> tRNA <sup>Arg</sup> (CCG)  </td><td> tRNA <sup>Arg</sup> (UCG)  </td><td> </td><td> </td></tr><tr><td>15 </td><td>cP01 </td><td>aP01 </td><td><span class="underline"> tRNA <sup>Thr</sup> (CGU) </span> </td><td> </td><td> </td><td> </td></tr><tr><td>16 </td><td>cP02 </td><td>aP02 </td><td><span class="underline"> tRNA <sup>Pro</sup> (CGG) </span> </td><td> tRNA <sup>Val</sup> (CAC)  </td><td> tRNA <sup>Val</sup> (UAC)  </td><td> tRNA <sup>Val</sup> (GAC)  </td></tr><tr><td>17 </td><td>cP03 </td><td>aP03 </td><td><span class="underline"> tRNA <sup>Lys</sup> (CUU) </span> </td><td> </td><td> </td><td> </td></tr><tr><td>18 </td><td>cP01 </td><td>aP04 </td><td><span class="underline"> tRNA <sup>Thr</sup> (UGU-1) </span><a reveal-id="tf2-1" data-open="tf2-1" class="link link-ref link-reveal xref-fn">a</a> </td><td> </td><td> </td><td> </td></tr><tr><td>19 </td><td>cP04 </td><td>aP05 </td><td><span class="underline"> tRNA <sup>Met</sup> (CAU-1) </span><a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">b</a> </td><td> </td><td> </td><td> </td></tr><tr><td>20 </td><td>cP07 </td><td>aP07 </td><td><span class="underline"> tRNA <sup>Ser</sup> (CGA) </span> </td><td> tRNA <sup>Ser</sup> (UGA)  </td><td> tRNA <sup>Ser</sup> (GGA)  </td><td> </td></tr><tr><td>21 </td><td>cP09 </td><td>aP09 </td><td><span class="underline"> tRNA <sup>Asp</sup> (GUC) </span> </td><td> tRNA <sup>Ser</sup> (GCU)  </td><td> </td><td> </td></tr><tr><td>22 </td><td>cP11 </td><td>aP11 </td><td><span class="underline"> tRNA <sup>Trp</sup> (CCA) </span> </td><td><span class="underline"> tRNA <sup>Met</sup> (CAU-2) </span><a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">b</a> </td><td> </td><td> </td></tr><tr><td>23 </td><td>cP03 </td><td>aP11 </td><td> tRNA <sup>Arg</sup> (GCG)  </td><td><span class="underline"> tRNA <sup>Arg</sup> (UCU) </span> </td><td> tRNA <sup>Arg</sup> (CCU)  </td><td> </td></tr><tr><td>24 </td><td>cP17 </td><td>aP19 </td><td> tRNA <sup>Gly</sup> (CCC)  </td><td> tRNA <sup>Gly</sup> (UCC)  </td><td><span class="underline"> tRNA <sup>Pro</sup> (GGG) </span> </td><td> </td></tr><tr><td>25 </td><td>cP03 </td><td>aP20 </td><td><span class="underline"> tRNA <sup>Lys</sup> (UUU) </span> </td><td> </td><td> </td><td> </td></tr><tr><td>26 </td><td>cP18 </td><td>aP07 </td><td><span class="underline"> tRNA <sup>Cys</sup> (GCA) </span> </td><td> </td><td> </td><td> </td></tr><tr><td>27 </td><td>cP20 </td><td>aP14 </td><td><span class="underline"> tRNA <sup>Tyr</sup> (GUA) </span> </td><td> tRNA <sup>Phe</sup> (GAA)  </td><td> </td><td> </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">Primer sets </th><th colspan="4">tRNA species </th></tr></thead><tbody><tr><td>1 </td><td>cP05 </td><td>aP06 </td><td> tRNA <sup>Ala</sup> (CGC)  </td><td> tRNA <sup>Ala</sup> (GGC)  </td><td> tRNA <sup>Ala</sup> (UGC)  </td><td> </td></tr><tr><td>2 </td><td>cP06 </td><td>aP19 </td><td> tRNA <sup>Pro</sup> (UGG)  </td><td> </td><td> </td><td> </td></tr><tr><td>3 </td><td>cP08 </td><td>aP08 </td><td> tRNA <sup>Leu</sup> (UAG)  </td><td> tRNA <sup>Leu</sup> (UAA)  </td><td> tRNA <sup>Leu</sup> (GAG)  </td><td> tRNA <sup>Leu</sup> (CAG)  </td></tr><tr><td>4 </td><td>cP10 </td><td>aP10 </td><td> tRNA <sup>Asn</sup> (GUU)  </td><td> </td><td> </td><td> </td></tr><tr><td>5 </td><td>cP08 </td><td>aP12 </td><td> tRNA <sup>Leu</sup> (CAA)  </td><td> </td><td> </td><td> </td></tr><tr><td>6 </td><td>cP12 </td><td>aP13 </td><td> tRNA <sup>Gly</sup> (GCC)  </td><td> </td><td> </td><td> </td></tr><tr><td>7 </td><td>cP11 </td><td>aP03 </td><td> tRNA <sup>Ile</sup> (GAU)  </td><td> </td><td> </td><td> </td></tr><tr><td>8 </td><td>cP13 </td><td>aP15 </td><td> tRNA <sup>Thr</sup> (UGU-2) <a reveal-id="tf2-1" data-open="tf2-1" class="link link-ref link-reveal xref-fn">a</a> </td><td> </td><td> </td><td> </td></tr><tr><td>9 </td><td>cP14 </td><td>aP16 </td><td> tRNA <sup>Gln</sup> (UUG)  </td><td> </td><td> </td><td> </td></tr><tr><td>10 </td><td>cP15 </td><td>aP17 </td><td> tRNA <sup>Met</sup> (CAU-3) <a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">b</a> </td><td> </td><td> </td><td> </td></tr><tr><td>11 </td><td>cP16 </td><td>aP18 </td><td> tRNA <sup>Glu</sup> (CUC)  </td><td> tRNA <sup>Glu</sup> (UUC)  </td><td> </td><td> </td></tr><tr><td>12 </td><td>cP19 </td><td>aP21 </td><td> tRNA <sup>His</sup> (GUG)  </td><td> </td><td> </td><td> </td></tr><tr><td>13 </td><td>cP21 </td><td>aP16 </td><td> tRNA <sup>Gln</sup> (CUG)  </td><td> tRNA <sup>Thr</sup> (GGU)  </td><td> </td><td> </td></tr><tr><td>14 </td><td>cP03 </td><td>aP11 </td><td> tRNA <sup>Arg</sup> (CCG)  </td><td> tRNA <sup>Arg</sup> (UCG)  </td><td> </td><td> </td></tr><tr><td>15 </td><td>cP01 </td><td>aP01 </td><td><span class="underline"> tRNA <sup>Thr</sup> (CGU) </span> </td><td> </td><td> </td><td> </td></tr><tr><td>16 </td><td>cP02 </td><td>aP02 </td><td><span class="underline"> tRNA <sup>Pro</sup> (CGG) </span> </td><td> tRNA <sup>Val</sup> (CAC)  </td><td> tRNA <sup>Val</sup> (UAC)  </td><td> tRNA <sup>Val</sup> (GAC)  </td></tr><tr><td>17 </td><td>cP03 </td><td>aP03 </td><td><span class="underline"> tRNA <sup>Lys</sup> (CUU) </span> </td><td> </td><td> </td><td> </td></tr><tr><td>18 </td><td>cP01 </td><td>aP04 </td><td><span class="underline"> tRNA <sup>Thr</sup> (UGU-1) </span><a reveal-id="tf2-1" data-open="tf2-1" class="link link-ref link-reveal xref-fn">a</a> </td><td> </td><td> </td><td> </td></tr><tr><td>19 </td><td>cP04 </td><td>aP05 </td><td><span class="underline"> tRNA <sup>Met</sup> (CAU-1) </span><a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">b</a> </td><td> </td><td> </td><td> </td></tr><tr><td>20 </td><td>cP07 </td><td>aP07 </td><td><span class="underline"> tRNA <sup>Ser</sup> (CGA) </span> </td><td> tRNA <sup>Ser</sup> (UGA)  </td><td> tRNA <sup>Ser</sup> (GGA)  </td><td> </td></tr><tr><td>21 </td><td>cP09 </td><td>aP09 </td><td><span class="underline"> tRNA <sup>Asp</sup> (GUC) </span> </td><td> tRNA <sup>Ser</sup> (GCU)  </td><td> </td><td> </td></tr><tr><td>22 </td><td>cP11 </td><td>aP11 </td><td><span class="underline"> tRNA <sup>Trp</sup> (CCA) </span> </td><td><span class="underline"> tRNA <sup>Met</sup> (CAU-2) </span><a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">b</a> </td><td> </td><td> </td></tr><tr><td>23 </td><td>cP03 </td><td>aP11 </td><td> tRNA <sup>Arg</sup> (GCG)  </td><td><span class="underline"> tRNA <sup>Arg</sup> (UCU) </span> </td><td> tRNA <sup>Arg</sup> (CCU)  </td><td> </td></tr><tr><td>24 </td><td>cP17 </td><td>aP19 </td><td> tRNA <sup>Gly</sup> (CCC)  </td><td> tRNA <sup>Gly</sup> (UCC)  </td><td><span class="underline"> tRNA <sup>Pro</sup> (GGG) </span> </td><td> </td></tr><tr><td>25 </td><td>cP03 </td><td>aP20 </td><td><span class="underline"> tRNA <sup>Lys</sup> (UUU) </span> </td><td> </td><td> </td><td> </td></tr><tr><td>26 </td><td>cP18 </td><td>aP07 </td><td><span class="underline"> tRNA <sup>Cys</sup> (GCA) </span> </td><td> </td><td> </td><td> </td></tr><tr><td>27 </td><td>cP20 </td><td>aP14 </td><td><span class="underline"> tRNA <sup>Tyr</sup> (GUA) </span> </td><td> tRNA <sup>Phe</sup> (GAA)  </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf2-1"><div class="fn"><p><sup>a</sup> tRNA <sup>Thr</sup> (UGU-1) was originally designated as APEtRNA06; tRNA <sup>Thr</sup> (UGU-2) was originally designated as APEtRNA26. </p></div></div><div content-id="tf2-2"><div class="fn"><p><sup>b</sup> tRNA <sup>Met</sup> (CAU-1) was originally designated as APEtRNA07; tRNA <sup>Met</sup> (CAU-2) was originally designated as APEtRNA16; tRNA <sup>Met</sup> (CAU-3) was originally designated as APEtRNA31. </p></div></div><div content-id=""><div class="fn"><p>tRNAs underlined indicate the interrupted tRNA genes.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/35870546" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>Primer sets used for amplification of each tRNA molecule</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">Primer sets </th><th colspan="4">tRNA species </th></tr></thead><tbody><tr><td>1 </td><td>cP05 </td><td>aP06 </td><td> tRNA <sup>Ala</sup> (CGC)  </td><td> tRNA <sup>Ala</sup> (GGC)  </td><td> tRNA <sup>Ala</sup> (UGC)  </td><td> </td></tr><tr><td>2 </td><td>cP06 </td><td>aP19 </td><td> tRNA <sup>Pro</sup> (UGG)  </td><td> </td><td> </td><td> </td></tr><tr><td>3 </td><td>cP08 </td><td>aP08 </td><td> tRNA <sup>Leu</sup> (UAG)  </td><td> tRNA <sup>Leu</sup> (UAA)  </td><td> tRNA <sup>Leu</sup> (GAG)  </td><td> tRNA <sup>Leu</sup> (CAG)  </td></tr><tr><td>4 </td><td>cP10 </td><td>aP10 </td><td> tRNA <sup>Asn</sup> (GUU)  </td><td> </td><td> </td><td> </td></tr><tr><td>5 </td><td>cP08 </td><td>aP12 </td><td> tRNA <sup>Leu</sup> (CAA)  </td><td> </td><td> </td><td> </td></tr><tr><td>6 </td><td>cP12 </td><td>aP13 </td><td> tRNA <sup>Gly</sup> (GCC)  </td><td> </td><td> </td><td> </td></tr><tr><td>7 </td><td>cP11 </td><td>aP03 </td><td> tRNA <sup>Ile</sup> (GAU)  </td><td> </td><td> </td><td> </td></tr><tr><td>8 </td><td>cP13 </td><td>aP15 </td><td> tRNA <sup>Thr</sup> (UGU-2) <a reveal-id="tf2-1" data-open="tf2-1" class="link link-ref link-reveal xref-fn">a</a> </td><td> </td><td> </td><td> </td></tr><tr><td>9 </td><td>cP14 </td><td>aP16 </td><td> tRNA <sup>Gln</sup> (UUG)  </td><td> </td><td> </td><td> </td></tr><tr><td>10 </td><td>cP15 </td><td>aP17 </td><td> tRNA <sup>Met</sup> (CAU-3) <a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">b</a> </td><td> </td><td> </td><td> </td></tr><tr><td>11 </td><td>cP16 </td><td>aP18 </td><td> tRNA <sup>Glu</sup> (CUC)  </td><td> tRNA <sup>Glu</sup> (UUC)  </td><td> </td><td> </td></tr><tr><td>12 </td><td>cP19 </td><td>aP21 </td><td> tRNA <sup>His</sup> (GUG)  </td><td> </td><td> </td><td> </td></tr><tr><td>13 </td><td>cP21 </td><td>aP16 </td><td> tRNA <sup>Gln</sup> (CUG)  </td><td> tRNA <sup>Thr</sup> (GGU)  </td><td> </td><td> </td></tr><tr><td>14 </td><td>cP03 </td><td>aP11 </td><td> tRNA <sup>Arg</sup> (CCG)  </td><td> tRNA <sup>Arg</sup> (UCG)  </td><td> </td><td> </td></tr><tr><td>15 </td><td>cP01 </td><td>aP01 </td><td><span class="underline"> tRNA <sup>Thr</sup> (CGU) </span> </td><td> </td><td> </td><td> </td></tr><tr><td>16 </td><td>cP02 </td><td>aP02 </td><td><span class="underline"> tRNA <sup>Pro</sup> (CGG) </span> </td><td> tRNA <sup>Val</sup> (CAC)  </td><td> tRNA <sup>Val</sup> (UAC)  </td><td> tRNA <sup>Val</sup> (GAC)  </td></tr><tr><td>17 </td><td>cP03 </td><td>aP03 </td><td><span class="underline"> tRNA <sup>Lys</sup> (CUU) </span> </td><td> </td><td> </td><td> </td></tr><tr><td>18 </td><td>cP01 </td><td>aP04 </td><td><span class="underline"> tRNA <sup>Thr</sup> (UGU-1) </span><a reveal-id="tf2-1" data-open="tf2-1" class="link link-ref link-reveal xref-fn">a</a> </td><td> </td><td> </td><td> </td></tr><tr><td>19 </td><td>cP04 </td><td>aP05 </td><td><span class="underline"> tRNA <sup>Met</sup> (CAU-1) </span><a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">b</a> </td><td> </td><td> </td><td> </td></tr><tr><td>20 </td><td>cP07 </td><td>aP07 </td><td><span class="underline"> tRNA <sup>Ser</sup> (CGA) </span> </td><td> tRNA <sup>Ser</sup> (UGA)  </td><td> tRNA <sup>Ser</sup> (GGA)  </td><td> </td></tr><tr><td>21 </td><td>cP09 </td><td>aP09 </td><td><span class="underline"> tRNA <sup>Asp</sup> (GUC) </span> </td><td> tRNA <sup>Ser</sup> (GCU)  </td><td> </td><td> </td></tr><tr><td>22 </td><td>cP11 </td><td>aP11 </td><td><span class="underline"> tRNA <sup>Trp</sup> (CCA) </span> </td><td><span class="underline"> tRNA <sup>Met</sup> (CAU-2) </span><a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">b</a> </td><td> </td><td> </td></tr><tr><td>23 </td><td>cP03 </td><td>aP11 </td><td> tRNA <sup>Arg</sup> (GCG)  </td><td><span class="underline"> tRNA <sup>Arg</sup> (UCU) </span> </td><td> tRNA <sup>Arg</sup> (CCU)  </td><td> </td></tr><tr><td>24 </td><td>cP17 </td><td>aP19 </td><td> tRNA <sup>Gly</sup> (CCC)  </td><td> tRNA <sup>Gly</sup> (UCC)  </td><td><span class="underline"> tRNA <sup>Pro</sup> (GGG) </span> </td><td> </td></tr><tr><td>25 </td><td>cP03 </td><td>aP20 </td><td><span class="underline"> tRNA <sup>Lys</sup> (UUU) </span> </td><td> </td><td> </td><td> </td></tr><tr><td>26 </td><td>cP18 </td><td>aP07 </td><td><span class="underline"> tRNA <sup>Cys</sup> (GCA) </span> </td><td> </td><td> </td><td> </td></tr><tr><td>27 </td><td>cP20 </td><td>aP14 </td><td><span class="underline"> tRNA <sup>Tyr</sup> (GUA) </span> </td><td> tRNA <sup>Phe</sup> (GAA)  </td><td> </td><td> </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="2">Primer sets </th><th colspan="4">tRNA species </th></tr></thead><tbody><tr><td>1 </td><td>cP05 </td><td>aP06 </td><td> tRNA <sup>Ala</sup> (CGC)  </td><td> tRNA <sup>Ala</sup> (GGC)  </td><td> tRNA <sup>Ala</sup> (UGC)  </td><td> </td></tr><tr><td>2 </td><td>cP06 </td><td>aP19 </td><td> tRNA <sup>Pro</sup> (UGG)  </td><td> </td><td> </td><td> </td></tr><tr><td>3 </td><td>cP08 </td><td>aP08 </td><td> tRNA <sup>Leu</sup> (UAG)  </td><td> tRNA <sup>Leu</sup> (UAA)  </td><td> tRNA <sup>Leu</sup> (GAG)  </td><td> tRNA <sup>Leu</sup> (CAG)  </td></tr><tr><td>4 </td><td>cP10 </td><td>aP10 </td><td> tRNA <sup>Asn</sup> (GUU)  </td><td> </td><td> </td><td> </td></tr><tr><td>5 </td><td>cP08 </td><td>aP12 </td><td> tRNA <sup>Leu</sup> (CAA)  </td><td> </td><td> </td><td> </td></tr><tr><td>6 </td><td>cP12 </td><td>aP13 </td><td> tRNA <sup>Gly</sup> (GCC)  </td><td> </td><td> </td><td> </td></tr><tr><td>7 </td><td>cP11 </td><td>aP03 </td><td> tRNA <sup>Ile</sup> (GAU)  </td><td> </td><td> </td><td> </td></tr><tr><td>8 </td><td>cP13 </td><td>aP15 </td><td> tRNA <sup>Thr</sup> (UGU-2) <a reveal-id="tf2-1" data-open="tf2-1" class="link link-ref link-reveal xref-fn">a</a> </td><td> </td><td> </td><td> </td></tr><tr><td>9 </td><td>cP14 </td><td>aP16 </td><td> tRNA <sup>Gln</sup> (UUG)  </td><td> </td><td> </td><td> </td></tr><tr><td>10 </td><td>cP15 </td><td>aP17 </td><td> tRNA <sup>Met</sup> (CAU-3) <a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">b</a> </td><td> </td><td> </td><td> </td></tr><tr><td>11 </td><td>cP16 </td><td>aP18 </td><td> tRNA <sup>Glu</sup> (CUC)  </td><td> tRNA <sup>Glu</sup> (UUC)  </td><td> </td><td> </td></tr><tr><td>12 </td><td>cP19 </td><td>aP21 </td><td> tRNA <sup>His</sup> (GUG)  </td><td> </td><td> </td><td> </td></tr><tr><td>13 </td><td>cP21 </td><td>aP16 </td><td> tRNA <sup>Gln</sup> (CUG)  </td><td> tRNA <sup>Thr</sup> (GGU)  </td><td> </td><td> </td></tr><tr><td>14 </td><td>cP03 </td><td>aP11 </td><td> tRNA <sup>Arg</sup> (CCG)  </td><td> tRNA <sup>Arg</sup> (UCG)  </td><td> </td><td> </td></tr><tr><td>15 </td><td>cP01 </td><td>aP01 </td><td><span class="underline"> tRNA <sup>Thr</sup> (CGU) </span> </td><td> </td><td> </td><td> </td></tr><tr><td>16 </td><td>cP02 </td><td>aP02 </td><td><span class="underline"> tRNA <sup>Pro</sup> (CGG) </span> </td><td> tRNA <sup>Val</sup> (CAC)  </td><td> tRNA <sup>Val</sup> (UAC)  </td><td> tRNA <sup>Val</sup> (GAC)  </td></tr><tr><td>17 </td><td>cP03 </td><td>aP03 </td><td><span class="underline"> tRNA <sup>Lys</sup> (CUU) </span> </td><td> </td><td> </td><td> </td></tr><tr><td>18 </td><td>cP01 </td><td>aP04 </td><td><span class="underline"> tRNA <sup>Thr</sup> (UGU-1) </span><a reveal-id="tf2-1" data-open="tf2-1" class="link link-ref link-reveal xref-fn">a</a> </td><td> </td><td> </td><td> </td></tr><tr><td>19 </td><td>cP04 </td><td>aP05 </td><td><span class="underline"> tRNA <sup>Met</sup> (CAU-1) </span><a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">b</a> </td><td> </td><td> </td><td> </td></tr><tr><td>20 </td><td>cP07 </td><td>aP07 </td><td><span class="underline"> tRNA <sup>Ser</sup> (CGA) </span> </td><td> tRNA <sup>Ser</sup> (UGA)  </td><td> tRNA <sup>Ser</sup> (GGA)  </td><td> </td></tr><tr><td>21 </td><td>cP09 </td><td>aP09 </td><td><span class="underline"> tRNA <sup>Asp</sup> (GUC) </span> </td><td> tRNA <sup>Ser</sup> (GCU)  </td><td> </td><td> </td></tr><tr><td>22 </td><td>cP11 </td><td>aP11 </td><td><span class="underline"> tRNA <sup>Trp</sup> (CCA) </span> </td><td><span class="underline"> tRNA <sup>Met</sup> (CAU-2) </span><a reveal-id="tf2-2" data-open="tf2-2" class="link link-ref link-reveal xref-fn">b</a> </td><td> </td><td> </td></tr><tr><td>23 </td><td>cP03 </td><td>aP11 </td><td> tRNA <sup>Arg</sup> (GCG)  </td><td><span class="underline"> tRNA <sup>Arg</sup> (UCU) </span> </td><td> tRNA <sup>Arg</sup> (CCU)  </td><td> </td></tr><tr><td>24 </td><td>cP17 </td><td>aP19 </td><td> tRNA <sup>Gly</sup> (CCC)  </td><td> tRNA <sup>Gly</sup> (UCC)  </td><td><span class="underline"> tRNA <sup>Pro</sup> (GGG) </span> </td><td> </td></tr><tr><td>25 </td><td>cP03 </td><td>aP20 </td><td><span class="underline"> tRNA <sup>Lys</sup> (UUU) </span> </td><td> </td><td> </td><td> </td></tr><tr><td>26 </td><td>cP18 </td><td>aP07 </td><td><span class="underline"> tRNA <sup>Cys</sup> (GCA) </span> </td><td> </td><td> </td><td> </td></tr><tr><td>27 </td><td>cP20 </td><td>aP14 </td><td><span class="underline"> tRNA <sup>Tyr</sup> (GUA) </span> </td><td> tRNA <sup>Phe</sup> (GAA)  </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf2-1"><div class="fn"><p><sup>a</sup> tRNA <sup>Thr</sup> (UGU-1) was originally designated as APEtRNA06; tRNA <sup>Thr</sup> (UGU-2) was originally designated as APEtRNA26. </p></div></div><div content-id="tf2-2"><div class="fn"><p><sup>b</sup> tRNA <sup>Met</sup> (CAU-1) was originally designated as APEtRNA07; tRNA <sup>Met</sup> (CAU-2) was originally designated as APEtRNA16; tRNA <sup>Met</sup> (CAU-3) was originally designated as APEtRNA31. </p></div></div><div content-id=""><div class="fn"><p>tRNAs underlined indicate the interrupted tRNA genes.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/35870546" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=35870547 id="35870547" class="section-title" >2.6. Cloning of cDNA</h3>
<p> For cloning the cDNA fragments amplified, 2 µl of cDNA molecules purified were incubated in 10 µl of reaction solution at 14°C by Original TA Cloning kit (Invitrogen, Carlsbad, CA, USA) with 0.05 µg of pCR2.1 Vector (Invitrogen, Carlsbad, CA, USA). For transformation of the ligated DNA, the DNA was introduced into the <em>E. coli</em> TOP10F′ competent cells (Invitrogen, Carlsbad, CA, USA). </p>                            <h3 scrollto-destination=35870549 id="35870549" class="section-title" >2.7. Sequencing</h3>
<p> The plasmid DNA was prepared from <em>E. coli</em> cells, which were cultured in 1.2 ml of liquid Luria–Bertani broth at 37°C for 18 h, by QIAprep turbo Miniprep KIT (QIAGEN, Hilden, Germany). The sequencing reaction was performed with the BigDye Terminator Cycle Sequencing Kit (ABI, Foster city, CA, USA), according to the supplier manual and the signal was detected using BASE station (MJ research, Waltham, MA, USA). </p>                            <h2 scrollto-destination=35870551 id="35870551" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=35870552 id="35870552" class="section-title" >3.1. RNA molecules prepared</h3>
<p> As shown in Materials and Methods section, three different qualities of RNA molecules, total RNA, small RNA and species-specific concentrated tRNA molecules, were prepared from freshly cultured cells of <em>A. pernix</em> K1. </p><p> Approximately 120 µg of the total RNA molecule was recovered from <em>A. pernix</em> K1 cells cultured in 40 ml culture broth. Approximately 1.8 µg of small RNA was recovered using the QIAGEN RNA/DNA Kit from 40 µg of DNase-treated total RNA. From 0.5 µg of small RNA, all species-specific concentrated tRNA molecules for tRNA <sup>Trp</sup> (CCA), tRNA <sup>Ser</sup> (CGA) and tRNA <sup>Ser</sup> (UGA) were used for the cDNA synthesis. </p>                            <h3 scrollto-destination=35870555 id="35870555" class="section-title" >3.2. Design of primers</h3>
<p>The experiment in this report was designed to utilize the specific primers, each of which possessed the sequence identical to corresponding tRNA molecule. The specific primers were capable of annealing only to the specific sequence of the tRNA molecules. For amplification and cloning of the specific cDNA from tRNA molecules, selection of length and sequence of the primer was thought to be important.</p><p> For the design of the primers utilized for the synthesis of the first strand of the cDNA from the 47 species of tRNA molecules assigned from the genomic sequence, <a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> the 3′ end sequences of the tRNA genes were compared with each other. As some identical 3′ end sequences were recognized, 21 independent 18mer oligonucleotides were designed and synthesized. As indicated in <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a> (column A), these synthesized oligo DNAs were referred as cP plus a two-digit number. </p><p> To amplify the cDNA molecules synthesized as the first strand of cDNA from the tRNA molecules, the primers, named aP plus a two-digit number, were designed and synthesized according to the 5′ end sequence of tRNA genes predicted. As listed in <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a> (column B), 21 independent 18mer oligonucleotides were synthesized, because of the presence of the identical 5′ sequences. PCR with 27 different combinations of cP and aP primers, as listed in <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a> , were planned for amplification of all 47 tRNA molecules present in <em>A. pernix</em> . </p>                            <h3 scrollto-destination=35870559 id="35870559" class="section-title" >3.3. Detection of amplified fragments</h3>
<p> During synthesis of the first strand of cDNA, the RNA template and cP-type primer was not chilled below the <em>T</em><sub>m</sub> of the primer used, to minimize the non-specific annealing of primers to the unrelated region or unrelated RNA molecules. After heat treatment, at 95°C for 5 min, of a reaction solution containing both the RNA and cP-type primer, the solution was cooled only to the annealing temperature and this temperature was maintained until addition of reverse transcriptase. </p><p>Among 27 RT–PCRs with the total RNA, all the amplified cDNA products were detected except for reaction by primer set 8. The cDNA fragments synthesized from the total RNA by 26 primer sets were used for construction of independent libraries by cloning into pCR2.1 vector. Each library constructed was designated as the APcDNAT plus number of primer set.</p><p> Since some cDNA fragments which were expected to be involved in the libraries APcDNAT08, 15, 20, 22 and 27 were not recovered, RT–PCRs and library construction were performed with the small RNA to obtain the cDNAs not identified from the total RNA. By this reaction, the four libraries, referred as the APcDNAS plus number of primer set, were constructed with products amplified by the primer set 15, 20, 22 and 27, but a fragment was not amplified by the primer set 8. Although the plasmid clones made from mature form of tRNA <sup>Tyr</sup> (GUA), tRNA <sup>Ser</sup> (GGA) and tRNA <sup>Phe</sup> (GAA) were identified from the APcDNAS libraries, cDNA fragments for mature forms of tRNA <sup>Trp</sup> (CCA) and tRNA <sup>Ser</sup> (CGA) and tRNA <sup>Ser</sup> (UGA) were still not obtained from this type of libraries. </p><p> To obtain the cDNA fragments synthesized from the mature forms of tRNA <sup>Trp</sup> (CCA) and tRNA <sup>Ser</sup> (CGA) and tRNA <sup>Ser</sup> (UGA), affinity chromatography was performed to separate and concentrate these three specific tRNA molecules. The cDNA clone for the mature form of tRNA <sup>Trp</sup> (CCA) was identified in the library constructed from species-specific concentrated tRNA molecules. However, cDNA fragments for mature form of tRNA <sup>Ser</sup> (CGA) and tRNA <sup>Ser</sup> (UGA) were not detected in the libraries constructed from this concentrated RNA. </p>                            <h3 scrollto-destination=35870564 id="35870564" class="section-title" >3.4. Identification of tRNA molecules without introns by sequence determination</h3>
<p> The cDNA libraries APcDNAT02, 04, 05, 06, 07, 09, 10 and 12 contained only one uninterrupted cDNA species as shown in <a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a> . To confirm these tRNA molecules, five plasmid clones, each of one library, were randomly selected and used for determination of the nucleotide sequence of the insert portion. Sequencing results indicated that these eight tRNA genes actually produced the tRNA molecules identical to the prediction. </p><p> Since the cDNA libraries APcDNAT01, 03, 11, 13 and 14 contained the multiple tRNA molecules transcribed from the uninterrupted tRNA genes, 30, 40, 20, 20 and 20 plasmid DNAs used for determination of sequence were prepared from the cDNA libraries APcDNAT01, 03, 11, 13 and 14, respectively. Sequencing results indicated that all 13 uninterrupted tRNA molecules recognized by these five primer sets were actually present in the living <em>A. pernix</em> cells, and that the sequence of the tRNA molecules actually present in the living <em>A. pernix</em> cells were identical to previous prediction of these tRNA genes. </p><p> The sequencing results for clones from APcDNAT01 to 07 and from APcDNA09 to 14 indicated that the nucleotide sequences of the 21 tRNA molecules actually produced from the uninterrupted tRNA genes in <em>A. pernix</em> were identical to those predicted in the previous genomic paper. <a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a></p><p> Each library APcDNAT16, 20, 21, 23, 24 and 27 contained two or more cDNA clones corresponding to the premature and mature forms of tRNA molecules, because these primer sets were specific to both the intron-containing and uncontaining tRNA genes. Thus, 80, 60, 40, 60, 60 and 40 clones randomly selected were used for determination of the sequence of insert portion of clones in libraries APcDNAT16, 20, 21, 23, 24 and 27, respectively. Sequencing results indicated that among 11 tRNA molecules expected to be included in these libraries, 8 cDNA clones for uninterrupted molecules for tRNA <sup>Val</sup> (CAC), tRNA <sup>Val</sup> (UAC), tRNA <sup>Val</sup> (GAC), tRNA <sup>Ser</sup> (GCU), tRNA <sup>Arg</sup> (GCG), tRNA <sup>Arg</sup> (CCU), tRNA <sup>Gly</sup> (CCC) and tRNA <sup>Gly</sup> (UCC), were identified in these APcDNAT type of libraries. The APcDNAS libraries were constructed with cDNA synthesized from the small RNA with primer set 20 and 27, to obtain the remaining cDNAs for three uninterrupted tRNA molecules. Sequencing results showed that the cDNA clones for tRNA <sup>Ser</sup> (GGA) and tRNA <sup>Phe</sup> (GAA) were identified from the APcDNAS libraries. </p><p> In the libraries constructed from the total or small RNA, totally 31 cDNA clones corresponding to the tRNA molecules produced from the uninterrupted tRNA genes were identified. Although cDNA clones for tRNA <sup>Ser</sup> (UGA) and tRNA <sup>Thr</sup> (UGU-2) were not detected in any libraries, the over all results revealed that previous prediction of the uninterrupted tRNA genes was accurate. </p>                            <h3 scrollto-destination=35870570 id="35870570" class="section-title" >3.5. Confirmation of actual intron position</h3>
<p>To determine the actual splicing patterns of tRNA molecules transcribed from each interrupted tRNA gene, sequence comparison of the premature form with mature form of the tRNA molecules transcribed from same interrupted tRNA gene was performed.</p><p> As the length of the cDNA synthesized from premature form of tRNA molecules was longer than those from mature form of tRNA molecules, five longer clones, each of one library, APcDNAT15 to 27, were used for the determination of their sequences for the premature form. The cloning and sequencing results indicated that all premature forms of tRNA molecules transcribed from the interrupted tRNA genes were identified from the APcDNAT library except for tRNA <sup>Thr</sup> (CGU), which was still not identified from APcDNAS library. These results are summarized in <a reveal-id="tbl3" data-open="tbl3" class="link link-reveal link-table xref-fig">Table 3</a> . </p>                        <a id="35870573" scrollto-destination="35870573"></a>
<div content-id="tbl3" class="table-modal"><div class="table-wrap"><div id="tbl3" class="table-wrap-title" data-id="tbl3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Summary of introns detected and unidentified tRNA molecules</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>tRNA species </th><th>Gene size (nt) </th><th> Intron start <a reveal-id="tf3-1" data-open="tf3-1" class="link link-ref link-reveal xref-fn">a</a> </th><th> Intron end <a reveal-id="tf3-1" data-open="tf3-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Intron size (nt) </th><th>Location </th><th colspan="2">Identified molecule </th></tr><tr><th> </th><th> </th><th> </th><th> </th><th> </th><th> </th><th colspan="2"><hr /> </th></tr><tr><th> </th><th> </th><th> </th><th> </th><th> </th><th> </th><th>Premature </th><th>Mature </th></tr></thead><tbody><tr><td> tRNA <sup>Arg</sup> (UCU)  </td><td>122 </td><td>40 </td><td>83 </td><td>44 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td><span class="underline"> tRNA <sup>Asp</sup> (GUC) </span> </td><td>199 </td><td>41 </td><td>161 </td><td>121 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Cys</sup> (GCA)  </td><td>96 </td><td>39 </td><td>56 </td><td>18 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Lys</sup> (CUU)  </td><td>113 </td><td><strong>47</strong> </td><td><strong>82</strong> </td><td>36 </td><td>V loop </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Lys</sup> (UUU)  </td><td>113 </td><td><strong>47</strong> </td><td><strong>82</strong> </td><td>36 </td><td>V loop </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Met</sup> (CAU-1) <a reveal-id="tf3-2" data-open="tf3-2" class="link link-ref link-reveal xref-fn">b</a> </td><td>125 </td><td>39 </td><td>86 </td><td>48 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Met</sup> (CAU-2) <a reveal-id="tf3-2" data-open="tf3-2" class="link link-ref link-reveal xref-fn">b</a> </td><td>115 </td><td>39 </td><td>76 </td><td>38 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Pro</sup> (GGG)  </td><td>115 </td><td>40 </td><td>76 </td><td>37 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Pro</sup> (CGG)  </td><td>122 </td><td><strong>35</strong> </td><td><strong>78</strong> </td><td>44 </td><td>A loop 5′ </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td><span class="underline"> tRNA <sup>Thr</sup> (UGU-1) </span><a reveal-id="tf3-3" data-open="tf3-3" class="link link-ref link-reveal xref-fn">c</a> </td><td>95 </td><td>23 </td><td>41 </td><td>19 </td><td>D stem </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td><span class="underline"> tRNA <sup>Trp</sup> (CCA) </span> </td><td>114 </td><td>32 </td><td>68 </td><td>37 </td><td>A stem 5′ </td><td><strong>t</strong> </td><td><strong>c</strong> </td></tr><tr><td> tRNA <sup>Tyr</sup> (GUA)  </td><td>114 </td><td>37 </td><td>75 </td><td>39 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>s</strong> </td></tr><tr><td> tRNA <sup>Thr</sup> (CGU)  </td><td>124 </td><td>37 </td><td>85 </td><td>49 </td><td>Normal </td><td><strong>n. d.</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Ser</sup> (CGA)  </td><td>125 </td><td><strong>n. a.</strong> </td><td><strong>n. a.</strong> </td><td><strong>n. a</strong> .  </td><td><strong>n. a.</strong> </td><td><strong>t</strong> </td><td><strong>n. d.</strong> </td></tr><tr><td> tRNA <sup>Ser</sup> (UGA)  </td><td>89 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td><td><strong>n. d.</strong> </td></tr><tr><td> tRNA <sup>Thr</sup> (UGU-2) <a reveal-id="tf3-3" data-open="tf3-3" class="link link-ref link-reveal xref-fn">c</a> </td><td>73 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td><td><strong>n. d.</strong> </td></tr></tbody></table></div><div content-id="tbl3" class="table-modal"><table><thead name="thead" align=""><tr><th>tRNA species </th><th>Gene size (nt) </th><th> Intron start <a reveal-id="tf3-1" data-open="tf3-1" class="link link-ref link-reveal xref-fn">a</a> </th><th> Intron end <a reveal-id="tf3-1" data-open="tf3-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Intron size (nt) </th><th>Location </th><th colspan="2">Identified molecule </th></tr><tr><th> </th><th> </th><th> </th><th> </th><th> </th><th> </th><th colspan="2"><hr /> </th></tr><tr><th> </th><th> </th><th> </th><th> </th><th> </th><th> </th><th>Premature </th><th>Mature </th></tr></thead><tbody><tr><td> tRNA <sup>Arg</sup> (UCU)  </td><td>122 </td><td>40 </td><td>83 </td><td>44 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td><span class="underline"> tRNA <sup>Asp</sup> (GUC) </span> </td><td>199 </td><td>41 </td><td>161 </td><td>121 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Cys</sup> (GCA)  </td><td>96 </td><td>39 </td><td>56 </td><td>18 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Lys</sup> (CUU)  </td><td>113 </td><td><strong>47</strong> </td><td><strong>82</strong> </td><td>36 </td><td>V loop </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Lys</sup> (UUU)  </td><td>113 </td><td><strong>47</strong> </td><td><strong>82</strong> </td><td>36 </td><td>V loop </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Met</sup> (CAU-1) <a reveal-id="tf3-2" data-open="tf3-2" class="link link-ref link-reveal xref-fn">b</a> </td><td>125 </td><td>39 </td><td>86 </td><td>48 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Met</sup> (CAU-2) <a reveal-id="tf3-2" data-open="tf3-2" class="link link-ref link-reveal xref-fn">b</a> </td><td>115 </td><td>39 </td><td>76 </td><td>38 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Pro</sup> (GGG)  </td><td>115 </td><td>40 </td><td>76 </td><td>37 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Pro</sup> (CGG)  </td><td>122 </td><td><strong>35</strong> </td><td><strong>78</strong> </td><td>44 </td><td>A loop 5′ </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td><span class="underline"> tRNA <sup>Thr</sup> (UGU-1) </span><a reveal-id="tf3-3" data-open="tf3-3" class="link link-ref link-reveal xref-fn">c</a> </td><td>95 </td><td>23 </td><td>41 </td><td>19 </td><td>D stem </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td><span class="underline"> tRNA <sup>Trp</sup> (CCA) </span> </td><td>114 </td><td>32 </td><td>68 </td><td>37 </td><td>A stem 5′ </td><td><strong>t</strong> </td><td><strong>c</strong> </td></tr><tr><td> tRNA <sup>Tyr</sup> (GUA)  </td><td>114 </td><td>37 </td><td>75 </td><td>39 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>s</strong> </td></tr><tr><td> tRNA <sup>Thr</sup> (CGU)  </td><td>124 </td><td>37 </td><td>85 </td><td>49 </td><td>Normal </td><td><strong>n. d.</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Ser</sup> (CGA)  </td><td>125 </td><td><strong>n. a.</strong> </td><td><strong>n. a.</strong> </td><td><strong>n. a</strong> .  </td><td><strong>n. a.</strong> </td><td><strong>t</strong> </td><td><strong>n. d.</strong> </td></tr><tr><td> tRNA <sup>Ser</sup> (UGA)  </td><td>89 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td><td><strong>n. d.</strong> </td></tr><tr><td> tRNA <sup>Thr</sup> (UGU-2) <a reveal-id="tf3-3" data-open="tf3-3" class="link link-ref link-reveal xref-fn">c</a> </td><td>73 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td><td><strong>n. d.</strong> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><strong>t</strong> indicates ‘isolated from total RNA’; <strong>s</strong> indicates ‘isolated from small RNA’; <strong>c</strong> indicates ‘isolated from species-specific concentrated RNA’; <strong>n. d.</strong> indicates ‘not detected’ and <strong>n. a.</strong> indicates ‘not assigned’. </p></div></div><div content-id="tf3-1"><div class="fn"><p><sup>a</sup> Intron position was indicated the nucleotides distance from 5′ end of the gene. </p></div></div><div content-id="tf3-2"><div class="fn"><p><sup>b</sup> tRNA <sup>Met</sup> (CAU-1) was originally designated as APEtRNA07; tRNA <sup>Met</sup> (CAU-2) was originally designated as APEtRNA16. </p></div></div><div content-id="tf3-3"><div class="fn"><p><sup>c</sup> tRNA <sup>Thr</sup> (UGU-1) was originally designated as APEtRNA06; tRNA <sup>Thr</sup> (UGU-2) was originally designated as APEtRNA26. </p></div></div><div content-id=""><div class="fn"><p>Numbers in bold indicate the intron locations different against prediction.</p></div></div><div content-id=""><div class="fn"><p>tRNAs underlined indicate those or the intron of those predicted manually.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/35870573" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl3" class="table-wrap-title" data-id="tbl3"><div class="label -label"><strong>Table 3</strong></div><div class="caption"><p>Summary of introns detected and unidentified tRNA molecules</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>tRNA species </th><th>Gene size (nt) </th><th> Intron start <a reveal-id="tf3-1" data-open="tf3-1" class="link link-ref link-reveal xref-fn">a</a> </th><th> Intron end <a reveal-id="tf3-1" data-open="tf3-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Intron size (nt) </th><th>Location </th><th colspan="2">Identified molecule </th></tr><tr><th> </th><th> </th><th> </th><th> </th><th> </th><th> </th><th colspan="2"><hr /> </th></tr><tr><th> </th><th> </th><th> </th><th> </th><th> </th><th> </th><th>Premature </th><th>Mature </th></tr></thead><tbody><tr><td> tRNA <sup>Arg</sup> (UCU)  </td><td>122 </td><td>40 </td><td>83 </td><td>44 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td><span class="underline"> tRNA <sup>Asp</sup> (GUC) </span> </td><td>199 </td><td>41 </td><td>161 </td><td>121 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Cys</sup> (GCA)  </td><td>96 </td><td>39 </td><td>56 </td><td>18 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Lys</sup> (CUU)  </td><td>113 </td><td><strong>47</strong> </td><td><strong>82</strong> </td><td>36 </td><td>V loop </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Lys</sup> (UUU)  </td><td>113 </td><td><strong>47</strong> </td><td><strong>82</strong> </td><td>36 </td><td>V loop </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Met</sup> (CAU-1) <a reveal-id="tf3-2" data-open="tf3-2" class="link link-ref link-reveal xref-fn">b</a> </td><td>125 </td><td>39 </td><td>86 </td><td>48 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Met</sup> (CAU-2) <a reveal-id="tf3-2" data-open="tf3-2" class="link link-ref link-reveal xref-fn">b</a> </td><td>115 </td><td>39 </td><td>76 </td><td>38 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Pro</sup> (GGG)  </td><td>115 </td><td>40 </td><td>76 </td><td>37 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Pro</sup> (CGG)  </td><td>122 </td><td><strong>35</strong> </td><td><strong>78</strong> </td><td>44 </td><td>A loop 5′ </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td><span class="underline"> tRNA <sup>Thr</sup> (UGU-1) </span><a reveal-id="tf3-3" data-open="tf3-3" class="link link-ref link-reveal xref-fn">c</a> </td><td>95 </td><td>23 </td><td>41 </td><td>19 </td><td>D stem </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td><span class="underline"> tRNA <sup>Trp</sup> (CCA) </span> </td><td>114 </td><td>32 </td><td>68 </td><td>37 </td><td>A stem 5′ </td><td><strong>t</strong> </td><td><strong>c</strong> </td></tr><tr><td> tRNA <sup>Tyr</sup> (GUA)  </td><td>114 </td><td>37 </td><td>75 </td><td>39 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>s</strong> </td></tr><tr><td> tRNA <sup>Thr</sup> (CGU)  </td><td>124 </td><td>37 </td><td>85 </td><td>49 </td><td>Normal </td><td><strong>n. d.</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Ser</sup> (CGA)  </td><td>125 </td><td><strong>n. a.</strong> </td><td><strong>n. a.</strong> </td><td><strong>n. a</strong> .  </td><td><strong>n. a.</strong> </td><td><strong>t</strong> </td><td><strong>n. d.</strong> </td></tr><tr><td> tRNA <sup>Ser</sup> (UGA)  </td><td>89 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td><td><strong>n. d.</strong> </td></tr><tr><td> tRNA <sup>Thr</sup> (UGU-2) <a reveal-id="tf3-3" data-open="tf3-3" class="link link-ref link-reveal xref-fn">c</a> </td><td>73 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td><td><strong>n. d.</strong> </td></tr></tbody></table></div><div content-id="tbl3" class="table-modal"><table><thead name="thead" align=""><tr><th>tRNA species </th><th>Gene size (nt) </th><th> Intron start <a reveal-id="tf3-1" data-open="tf3-1" class="link link-ref link-reveal xref-fn">a</a> </th><th> Intron end <a reveal-id="tf3-1" data-open="tf3-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Intron size (nt) </th><th>Location </th><th colspan="2">Identified molecule </th></tr><tr><th> </th><th> </th><th> </th><th> </th><th> </th><th> </th><th colspan="2"><hr /> </th></tr><tr><th> </th><th> </th><th> </th><th> </th><th> </th><th> </th><th>Premature </th><th>Mature </th></tr></thead><tbody><tr><td> tRNA <sup>Arg</sup> (UCU)  </td><td>122 </td><td>40 </td><td>83 </td><td>44 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td><span class="underline"> tRNA <sup>Asp</sup> (GUC) </span> </td><td>199 </td><td>41 </td><td>161 </td><td>121 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Cys</sup> (GCA)  </td><td>96 </td><td>39 </td><td>56 </td><td>18 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Lys</sup> (CUU)  </td><td>113 </td><td><strong>47</strong> </td><td><strong>82</strong> </td><td>36 </td><td>V loop </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Lys</sup> (UUU)  </td><td>113 </td><td><strong>47</strong> </td><td><strong>82</strong> </td><td>36 </td><td>V loop </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Met</sup> (CAU-1) <a reveal-id="tf3-2" data-open="tf3-2" class="link link-ref link-reveal xref-fn">b</a> </td><td>125 </td><td>39 </td><td>86 </td><td>48 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Met</sup> (CAU-2) <a reveal-id="tf3-2" data-open="tf3-2" class="link link-ref link-reveal xref-fn">b</a> </td><td>115 </td><td>39 </td><td>76 </td><td>38 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Pro</sup> (GGG)  </td><td>115 </td><td>40 </td><td>76 </td><td>37 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Pro</sup> (CGG)  </td><td>122 </td><td><strong>35</strong> </td><td><strong>78</strong> </td><td>44 </td><td>A loop 5′ </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td><span class="underline"> tRNA <sup>Thr</sup> (UGU-1) </span><a reveal-id="tf3-3" data-open="tf3-3" class="link link-ref link-reveal xref-fn">c</a> </td><td>95 </td><td>23 </td><td>41 </td><td>19 </td><td>D stem </td><td><strong>t</strong> </td><td><strong>t</strong> </td></tr><tr><td><span class="underline"> tRNA <sup>Trp</sup> (CCA) </span> </td><td>114 </td><td>32 </td><td>68 </td><td>37 </td><td>A stem 5′ </td><td><strong>t</strong> </td><td><strong>c</strong> </td></tr><tr><td> tRNA <sup>Tyr</sup> (GUA)  </td><td>114 </td><td>37 </td><td>75 </td><td>39 </td><td>Normal </td><td><strong>t</strong> </td><td><strong>s</strong> </td></tr><tr><td> tRNA <sup>Thr</sup> (CGU)  </td><td>124 </td><td>37 </td><td>85 </td><td>49 </td><td>Normal </td><td><strong>n. d.</strong> </td><td><strong>t</strong> </td></tr><tr><td> tRNA <sup>Ser</sup> (CGA)  </td><td>125 </td><td><strong>n. a.</strong> </td><td><strong>n. a.</strong> </td><td><strong>n. a</strong> .  </td><td><strong>n. a.</strong> </td><td><strong>t</strong> </td><td><strong>n. d.</strong> </td></tr><tr><td> tRNA <sup>Ser</sup> (UGA)  </td><td>89 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td><td><strong>n. d.</strong> </td></tr><tr><td> tRNA <sup>Thr</sup> (UGU-2) <a reveal-id="tf3-3" data-open="tf3-3" class="link link-ref link-reveal xref-fn">c</a> </td><td>73 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td><td><strong>n. d.</strong> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><strong>t</strong> indicates ‘isolated from total RNA’; <strong>s</strong> indicates ‘isolated from small RNA’; <strong>c</strong> indicates ‘isolated from species-specific concentrated RNA’; <strong>n. d.</strong> indicates ‘not detected’ and <strong>n. a.</strong> indicates ‘not assigned’. </p></div></div><div content-id="tf3-1"><div class="fn"><p><sup>a</sup> Intron position was indicated the nucleotides distance from 5′ end of the gene. </p></div></div><div content-id="tf3-2"><div class="fn"><p><sup>b</sup> tRNA <sup>Met</sup> (CAU-1) was originally designated as APEtRNA07; tRNA <sup>Met</sup> (CAU-2) was originally designated as APEtRNA16. </p></div></div><div content-id="tf3-3"><div class="fn"><p><sup>c</sup> tRNA <sup>Thr</sup> (UGU-1) was originally designated as APEtRNA06; tRNA <sup>Thr</sup> (UGU-2) was originally designated as APEtRNA26. </p></div></div><div content-id=""><div class="fn"><p>Numbers in bold indicate the intron locations different against prediction.</p></div></div><div content-id=""><div class="fn"><p>tRNAs underlined indicate those or the intron of those predicted manually.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/35870573" target="_blank">View Large</a></div></div><p> Since the primer sets 15, 17, 18, 19, 25 and 26 were able to recognize only one interrupted tRNA gene, it was thought that each library, APcDNAT15, 17, 18, 19, 25 and 26, contained only two different-sized cDNA clones made from the premature and mature forms of tRNA molecules. Each of the five plasmids with the short and long insert were selected from each library and used for sequencing analyses. All cDNA clones corresponding to the mature form of these interrupted tRNA molecules were successfully identified from these six libraries. The cDNA clone for the mature form of the tRNA <sup>Thr</sup> (CGU) molecule was identified from the APcDNAT library, but that for the premature form of tRNA <sup>Thr</sup> (CGU) was not identified from the APcDNAT or APcDNAS libraries. </p><p> As the APcDNAT22 library contained cDNAs corresponding to the two interrupted tRNA genes, tRNA <sup>Trp</sup> (CCA) and tRNA <sup>Met</sup> (CAU-2), each of the 30 clones for the shorter and longer clones from the library were used for the determination of the sequence. The sequencing results indicated that both tRNA <sup>Met</sup> (CAU-2) molecules were present in <em>A. pernix</em> , but the evidence for the maturation of tRNA <sup>Trp</sup> (CCA) was not obtained from APcDNAT22 library. Then, the library APcDNAS22 was constructed from the small RNA, but the cDNA clone corresponding to the mature form of tRNA <sup>Trp</sup> (CCA) was not identified in this library. </p><p> The libraries APcDNAT16, 20, 21, 23, 24 and 27 contained the multiple cDNAs synthesized with primer sets specific to both the interrupted and uninterrupted tRNA genes. The length of the cDNA fragment was confirmed by PCR analyses of insert portion, and 60–100 independent plasmid clones with short insert were used for the determination of sequence. All shorter cDNA clones were detected in these sequenced clones in APcDNAT libraries except for the APcDNAT20 and 27 libraries. Two cDNA libraries, APcDNAS20 and 27, were constructed to obtain these missing cDNA clones in the libraries APcDNAT20 and APcDNAT27. The cDNA clones corresponding to the uninterrupted tRNA molecules, tRNA <sup>Ser</sup> (GGA) and tRNA <sup>Phe</sup> (GAA), and mature form of tRNA <sup>Tyr</sup> (GUA) were identified from the APcDNAS libraries. However, two cDNA clones corresponding to the mature form of tRNA <sup>Ser</sup> (CGA) and the tRNA <sup>Ser</sup> (UGA) molecule were still not identified from the APcDNAS library. </p><p> As the cDNAs for two mature forms of the tRNA molecules, tRNA <sup>Trp</sup> (CCA) and tRNA <sup>Ser</sup> (CGA), were not identified from the APcDNAS library, the concentration of species-specific tRNA molecules for tRNA <sup>Trp</sup> (CCA) and tRNA <sup>Ser</sup> (CGA) were performed as described in Materials and Methods section. In the libraries constructed from the species-specific concentrated tRNA molecules, the mature form of tRNA <sup>Trp</sup> (CCA) was identified, but cDNA for tRNA <sup>Ser</sup> (CGA) was not obtained. </p><p> Among 14 interrupted tRNA genes, 12 sets of both the premature and mature forms, 1 mature form and 1 premature form of cDNA, were identified as indicated in <a reveal-id="tbl3" data-open="tbl3" class="link link-reveal link-table xref-fig">Table 3</a> . These results indicated that the tRNA molecules, that were actually transcribed from the interrupted tRNA genes predicted, were actually processed in <em>A. pernix</em> K1 living cells. Since the mature form of tRNA <sup>Thr</sup> (CGU) molecule was cloned and sequenced, sequence comparison data of this clone and 12 sets of both forms of tRNA molecules provided the information for actual splicing position. </p><p> In the previous genomic paper of <em>A. pernix</em> K1, <a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> the intron region of tRNA <sup>Trp</sup> (CCA), tRNA <sup>Thr</sup> (UGU-1) and tRNA <sup>Asp</sup> (GUC) were manually predicted to locate at three bases 5′ from the anticodon, within the D-loop and at one base 3′ from anticodon with the 121 bp long intron, which was 1.5 times longer than the tRNA molecule itself, respectively. The comparison of cDNA sequences corresponding to the mature and premature forms of these tRNA molecules indicated that the intron region predicted in these tRNA genes were actually processed in <em>A. pernix</em> K1 cells. </p><p> However, 36 nt long introns in the tRNA <sup>Lys</sup> (CUU) and the tRNA <sup>Lys</sup> (UUU) were identified at 8 bases 3′ from the anticodon region as shown in bold in <a reveal-id="tbl3" data-open="tbl3" class="link link-reveal link-table xref-fig">Table 3</a> . Also, the 44 nt long intron in the tRNA <sup>Pro</sup> (CGG) was identified at one base 5′ from the anticodon region. Totally, five interrupted tRNA genes in <em>A. pernix</em> contained the intron portion at other position than one base 3′ from anticodon and other eight tRNA introns, including the 121 bp long intron in tRNA <sup>Asp</sup> (GUC), were inserted into the position at one base 3′ from anticodon, which is common to the eukaryotic tRNA intron. </p>                            <h2 scrollto-destination=35870581 id="35870581" class="section-title" >4. Discussion</h2>
<p> Our work was able to show both the actual transcription of the tRNA molecules and the actual processing of tRNA–intron portions by cloning a whole set of cDNA for tRNA molecules in <em>A. pernix</em> living cells. This is one of the first cases of identification of a whole set of tRNA molecules present in one organism as cDNA, including the premature and mature forms of tRNA molecules. </p><p> Our success in cloning cDNA clones for most of tRNA molecules was mainly due to the improvement of the methods used for synthesis of the first strand of cDNA from RNA molecules. In our work, the solution containing both template RNA and primer was not chilled below the <em>T</em><sub>m</sub> of the primers, which should allow primers to anneal to specific sites of tRNA molecules only. It is noteworthy that the annealing temperature of primers and RNA template is important for synthesis and amplification of specific cDNA from similar molecules such as tRNA. </p><p> In this work, four tRNA molecules, tRNA <sup>Thr</sup> (UGU-2), tRNA <sup>Ser</sup> (UGA), mature form of tRNA <sup>Ser</sup> (CGA) and premature form of tRNA <sup>Thr</sup> (CGU), were not identified as cDNA molecules from any qualities of RNA. The mature form of tRNA <sup>Tyr</sup> (GUA), tRNA <sup>Ser</sup> (GGA) and tRNA <sup>Phe</sup> (GAA) were isolated from small RNA molecules. The cDNA for the mature form of tRNA <sup>Trp</sup> (CCA) was successfully cloned from the species-specific tRNA molecules concentrated by the affinity chromatography technique, indicating that affinity chromatography was effective for concentration of the species-specific tRNA molecules. These results revealed that the concentration of one kind of tRNA molecule and usage of small RNA were effective for the cloning of relatively rare tRNA molecules. </p><p> The insertion position and length of the introns identified in the tRNA <sup>Lys</sup> (CUU) and tRNA <sup>Lys</sup> (UUU) genes were identical. And it was shown that these two tRNA genes shared the same sequence except for 2 nt, even though the different length and sequences of the intron portions were confirmed in other tRNA genes. In the genomic data of the two methanogen, <em>Methanocaldococcus jannaschii</em><a reveal-id="b17" data-open="b17" class="link link-ref link-reveal xref-bibr">17</a> and <em>Methanothermobacter thermautotrophicus</em> , <a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> the tRNA <sup>Lys</sup> (CUU) gene was not present. These facts proposed one hypothesis that the two introns identified in the tRNA <sup>Lys</sup> (CUU) and tRNA <sup>Lys</sup> (UUU) genes were derived from the same origin. It can be thought that the intron was introduced into the tRNA <sup>Lys</sup> (UUU) gene before duplication or separation of the tRNA <sup>Lys</sup> (CUU) gene, then the tRNA <sup>Lys</sup> (CUU) was duplicated or separated from tRNA <sup>Lys</sup> (UUU) recently in <em>A. pernix</em> , accompanied with the transfer of the same intron to the tRNA <sup>Lys</sup> (CUU). </p><p> According to the tRNA <sup>Thr</sup> (UGU-2), it was thought that this tRNA gene was overestimated by tRNA detection software. The reason for this conclusion is due to the facts that cDNA for tRNA <sup>Thr</sup> (UGU-2) was not amplified even when specific primer set 8 was used, the promoter motif sequence was not identified at upstream of the tRNA <sup>Thr</sup> (UGU-2) gene (shown below), both the premature and mature products from the tRNA gene for threonine with same anticodon, tRNA <sup>Thr</sup> (UGU-1), were identified and the score for this tRNA gene indicated by tRNA detection software was lower than those for other tRNA genes. </p><p> The previous analyses proposed two types of mechanisms for the splicing of the intron region in tRNA molecule. One is the eukaryotic type, in which the intron region in the tRNA molecule was recognized by the distance from the top of the anticodon stem to the 5′ and 3′ cleavage sites as shown in <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2A</a><a reveal-id="b4" data-open="b4" class="link link-ref link-reveal xref-bibr">4</a> , <a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> and cleavage of the eukaryotic intron requires the formation of a 3 nt bulge loop at the 3′ intron–exon boundary site. <a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> This mechanism is well suited for the fact that all eukaryal tRNA introns are located in one base 3′ from anticodon. </p><p> The other is the archaeal type of splicing mechanism of tRNA intron. In contrast with the eukaryotic type, studies for archaeal tRNA intron showed that complete mature tRNA structure was not required for digestion of intron by intron endonuclease. <a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> , <a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a> Cleavage of the archaeal tRNA introns requires a structural motif element at the exon–intron boundaries, the bulge–helix–bulge motif structure as shown in <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2B</a> . <a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a> Since the tRNA introns in <em>A. pernix</em> were confirmed at six different positions, the distance between the top of the anticodon stem and cleavage sites was widely variable. Hence, we searched the bulge–helix–bulge motif structures at the exon–intron boundaries confirmed in this work. </p><p> For the eight interrupted tRNA genes, of which introns were confirmed to locate at one base 3′ from anticodon, the bulge–helix–bulge motif structures were detected at the exon–intron boundaries as shown in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3</a> . </p>                        <a id="35870590" scrollto-destination="35870590"></a>
<div id="fig3" data-id="fig3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi023/3/m_dsi023f3.jpeg?Expires=2147483647&Signature=pmg85U-PpjMX8T0bmDQ7GyhSq5Dir~ec6-M78ucrY~kY9tQ2~DWDpw5NOH6BrU60oL65r79KRU6G3Q4puX-Fg47Y1jx7uaphgS6LavEkj2Qq2T2qW6w2WYrJgg0u0bvhi8NPjrKHEVJ31G1eSuTh4h7T~Y5FVTu0JY7UKZN6WtZeSEXky~AmlFpHD64w8721Teaa6bWMosaU3zMIFPrD8LOb7NfAqMZoZCamiDsGXRO9osOaJjJAUfbvXRNYH4SPKrRmtLVB3XF5AnRffJSV3Pqfr8LkoTXWUZy3KRolp~sKS0rBBMJficfszOpTJHB9Dug0CiCuOQ7NLps1DzLQtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Structure of the premature tRNA molecules with introns at one base 3′ from anticodon region. Structures of eight premature tRNA molecules with introns at one base 3′ from anticodon region. Splicing sites are indicated by short arrow. The anticodon regions are shown in the boxes. A: tRNA Arg (UCU), B: tRNA Asp (GUC), C: tRNA Met (CAU-1), D: tRNA Met (CAU-2), E: tRNA Thr (CGU), F: tRNA Pro (GGG), G: tRNA Cys (GCA), H: tRNA Tyr (GUA). " path-from-xml="dsi023f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35870590/dsi023f3.jpeg" path-from-xml="dsi023f3.jpeg" target="_blank">View large</a><a section="35870590" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi023/3/dsi023f3.jpeg?Expires=2147483647&Signature=h92vWQB-aRJm4trF6XoTzO9iuRsErAjsmPtIZAE6ddTmReYKV9T27rvnX-LXTzpD3Fuj70FxY5oge1DeewKuN0EUtz9dABACgY-WQF7afXY40w2ianUP-FIJr4Rxykga1HBw57Uvh3soL629qVFi4q7E7cwC3b0xBtNafCCE~ihW~Rj9nem~nsbyCKSCtEYZm5m7olXC6ZqMwfut3sYGpnhNHMoesKPc-4wJDI~fmWUadifVXOjrImPTDB-xb7hC5kxbT7nAO-w4zNm2mGNzovZRU81XJTF5PlCta1-9xSM32-iJb8~RUxyhGXJmli0N5Cn6QDV~qIch7hXRgBkczA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35870590&ar=344476&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi023f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Structure of the premature tRNA molecules with introns at one base 3′ from anticodon region. Structures of eight premature tRNA molecules with introns at one base 3′ from anticodon region. Splicing sites are indicated by short arrow. The anticodon regions are shown in the boxes. A: tRNA <sup>Arg</sup> (UCU), B: tRNA <sup>Asp</sup> (GUC), C: tRNA <sup>Met</sup> (CAU-1), D: tRNA <sup>Met</sup> (CAU-2), E: tRNA <sup>Thr</sup> (CGU), F: tRNA <sup>Pro</sup> (GGG), G: tRNA <sup>Cys</sup> (GCA), H: tRNA <sup>Tyr</sup> (GUA). </p></div></div><div content-id="fig3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi023/3/m_dsi023f3.jpeg?Expires=2147483647&Signature=pmg85U-PpjMX8T0bmDQ7GyhSq5Dir~ec6-M78ucrY~kY9tQ2~DWDpw5NOH6BrU60oL65r79KRU6G3Q4puX-Fg47Y1jx7uaphgS6LavEkj2Qq2T2qW6w2WYrJgg0u0bvhi8NPjrKHEVJ31G1eSuTh4h7T~Y5FVTu0JY7UKZN6WtZeSEXky~AmlFpHD64w8721Teaa6bWMosaU3zMIFPrD8LOb7NfAqMZoZCamiDsGXRO9osOaJjJAUfbvXRNYH4SPKrRmtLVB3XF5AnRffJSV3Pqfr8LkoTXWUZy3KRolp~sKS0rBBMJficfszOpTJHB9Dug0CiCuOQ7NLps1DzLQtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Structure of the premature tRNA molecules with introns at one base 3′ from anticodon region. Structures of eight premature tRNA molecules with introns at one base 3′ from anticodon region. Splicing sites are indicated by short arrow. The anticodon regions are shown in the boxes. A: tRNA Arg (UCU), B: tRNA Asp (GUC), C: tRNA Met (CAU-1), D: tRNA Met (CAU-2), E: tRNA Thr (CGU), F: tRNA Pro (GGG), G: tRNA Cys (GCA), H: tRNA Tyr (GUA). " path-from-xml="dsi023f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35870590/dsi023f3.jpeg" path-from-xml="dsi023f3.jpeg" target="_blank">View large</a><a section="35870590" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi023/3/dsi023f3.jpeg?Expires=2147483647&Signature=h92vWQB-aRJm4trF6XoTzO9iuRsErAjsmPtIZAE6ddTmReYKV9T27rvnX-LXTzpD3Fuj70FxY5oge1DeewKuN0EUtz9dABACgY-WQF7afXY40w2ianUP-FIJr4Rxykga1HBw57Uvh3soL629qVFi4q7E7cwC3b0xBtNafCCE~ihW~Rj9nem~nsbyCKSCtEYZm5m7olXC6ZqMwfut3sYGpnhNHMoesKPc-4wJDI~fmWUadifVXOjrImPTDB-xb7hC5kxbT7nAO-w4zNm2mGNzovZRU81XJTF5PlCta1-9xSM32-iJb8~RUxyhGXJmli0N5Cn6QDV~qIch7hXRgBkczA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35870590&ar=344476&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi023f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Structure of the premature tRNA molecules with introns at one base 3′ from anticodon region. Structures of eight premature tRNA molecules with introns at one base 3′ from anticodon region. Splicing sites are indicated by short arrow. The anticodon regions are shown in the boxes. A: tRNA <sup>Arg</sup> (UCU), B: tRNA <sup>Asp</sup> (GUC), C: tRNA <sup>Met</sup> (CAU-1), D: tRNA <sup>Met</sup> (CAU-2), E: tRNA <sup>Thr</sup> (CGU), F: tRNA <sup>Pro</sup> (GGG), G: tRNA <sup>Cys</sup> (GCA), H: tRNA <sup>Tyr</sup> (GUA). </p></div></div><p> The same bulge–helix–bulge motif structures were searched on the two genes for tRNA <sup>Thr</sup> (UGU-1) and tRNA <sup>Trp</sup> (CCA), their actual processing occurred at the D stem and the anticodon stem which were at the same position as predicted, respectively. The bulge–helix–bulge motif structures were clearly recognized at the confirmed exon–intron boundaries of these tRNA genes as shown in <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4</a> , even though those introns were identified at the D stem and the anticodon stem. </p>                        <a id="35870592" scrollto-destination="35870592"></a>
<div id="fig4" data-id="fig4" class="fig fig-section"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi023/3/m_dsi023f4.jpeg?Expires=2147483647&Signature=fXVEZenIp6F5RH3h9A0O5HgD3j6KfORqn6L6jfcBbDY6x1g-gTPNiFZZEmdWR~s7Gt-KupyBtkzfMtZwJq8IIkHhJMtryDA8KhNI~foICDVXL800dgjrTPq9XBwgXaul0itopPsblGd2klNgTXhCicS0TvKe0~gRW-IXySlF5duKv38ZFQUg88nIaOSF2tu~c6T~mGZGEBAtLSqGebwJw3OSLZduavXz4GapYV3BQ-RIKsJJKOhNXoeZixzYCVfRqiol-E652UMBEW-6TaHJPYlZnX2GknFVqqIit6kod9jSPcGeUv4IqSeQVvqYRjtlJzn7ujIiv~PQe08A~mLevg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Structures of the premature tRNA molecules with introns at the predicted positions other than one base 3′ from anticodon region. Structures of two premature tRNA molecules with introns at D stem region (A) and anticodon stem (B). Splicing sites are indicated by short arrow. The anticodon regions are shown in the boxes. A: tRNA Thr (UGU-1), B: tRNA Trp (CCA). " path-from-xml="dsi023f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35870592/dsi023f4.jpeg" path-from-xml="dsi023f4.jpeg" target="_blank">View large</a><a section="35870592" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi023/3/dsi023f4.jpeg?Expires=2147483647&Signature=pcyT0VLn04H3Yr26dvXj4N9tQqY79kTw9wPIDbajT~CkLpj4mn3nRfWxW0KYfN9MVSsBICloNvLRctjM6g34MX5qt2WehpGaYAWD0W3gampUzBJPrsT7hGAIlzfu6BhAL2OTFyu5GZS64jXnVZfRIH4J3q6HRNVTDxc9-CcLZQjFHDmhqKlxDX87ObJgTBvsx8omFyW~l-RTHrDwJgDojzeHRkqk2LDUxmDiMVa2tr-KsyGYXvrTvdc8Vo5AZZG45ec7L0toZmvX3vzM~FgB0kGXbUmOgb2L1ZBojT4L7juOaznOHmr-ACiYnzL6Wm0prW5AF02fTYPPmAoMVlKJGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35870592&ar=344476&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi023f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Structures of the premature tRNA molecules with introns at the predicted positions other than one base 3′ from anticodon region. Structures of two premature tRNA molecules with introns at D stem region (A) and anticodon stem (B). Splicing sites are indicated by short arrow. The anticodon regions are shown in the boxes. A: tRNA <sup>Thr</sup> (UGU-1), B: tRNA <sup>Trp</sup> (CCA). </p></div></div><div content-id="fig4" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi023/3/m_dsi023f4.jpeg?Expires=2147483647&Signature=fXVEZenIp6F5RH3h9A0O5HgD3j6KfORqn6L6jfcBbDY6x1g-gTPNiFZZEmdWR~s7Gt-KupyBtkzfMtZwJq8IIkHhJMtryDA8KhNI~foICDVXL800dgjrTPq9XBwgXaul0itopPsblGd2klNgTXhCicS0TvKe0~gRW-IXySlF5duKv38ZFQUg88nIaOSF2tu~c6T~mGZGEBAtLSqGebwJw3OSLZduavXz4GapYV3BQ-RIKsJJKOhNXoeZixzYCVfRqiol-E652UMBEW-6TaHJPYlZnX2GknFVqqIit6kod9jSPcGeUv4IqSeQVvqYRjtlJzn7ujIiv~PQe08A~mLevg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Structures of the premature tRNA molecules with introns at the predicted positions other than one base 3′ from anticodon region. Structures of two premature tRNA molecules with introns at D stem region (A) and anticodon stem (B). Splicing sites are indicated by short arrow. The anticodon regions are shown in the boxes. A: tRNA Thr (UGU-1), B: tRNA Trp (CCA). " path-from-xml="dsi023f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35870592/dsi023f4.jpeg" path-from-xml="dsi023f4.jpeg" target="_blank">View large</a><a section="35870592" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi023/3/dsi023f4.jpeg?Expires=2147483647&Signature=pcyT0VLn04H3Yr26dvXj4N9tQqY79kTw9wPIDbajT~CkLpj4mn3nRfWxW0KYfN9MVSsBICloNvLRctjM6g34MX5qt2WehpGaYAWD0W3gampUzBJPrsT7hGAIlzfu6BhAL2OTFyu5GZS64jXnVZfRIH4J3q6HRNVTDxc9-CcLZQjFHDmhqKlxDX87ObJgTBvsx8omFyW~l-RTHrDwJgDojzeHRkqk2LDUxmDiMVa2tr-KsyGYXvrTvdc8Vo5AZZG45ec7L0toZmvX3vzM~FgB0kGXbUmOgb2L1ZBojT4L7juOaznOHmr-ACiYnzL6Wm0prW5AF02fTYPPmAoMVlKJGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35870592&ar=344476&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi023f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Structures of the premature tRNA molecules with introns at the predicted positions other than one base 3′ from anticodon region. Structures of two premature tRNA molecules with introns at D stem region (A) and anticodon stem (B). Splicing sites are indicated by short arrow. The anticodon regions are shown in the boxes. A: tRNA <sup>Thr</sup> (UGU-1), B: tRNA <sup>Trp</sup> (CCA). </p></div></div><p> For the three tRNA genes encoding tRNA <sup>Lys</sup> (CUU), tRNA <sup>Lys</sup> (UUU) and tRNA <sup>Pro</sup> (CGG), of which the actual cleavage site were differed from that predicted by software, we compared the structures at the exon–intron boundaries confirmed in this work with those found surround the exon–intron boundaries predicted. <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Figure 5A–C</a> indicates the structures of premature forms of tRNA molecules including introns at the position predicted by software and <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5D–F</a> indicates those including introns at the confirmed insertion position. <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Figure 5A–C</a> showed that the bulge–helix–bulge motif structure was not recognized at the exon–intron boundary. However, the bulge–helix–bulge motif structure was clearly identified at the exon–intron boundary confirmed by this work, as shown in <a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5D–F</a> . </p>                        <a id="35870594" scrollto-destination="35870594"></a>
<div id="fig5" data-id="fig5" class="fig fig-section"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi023/3/m_dsi023f5.jpeg?Expires=2147483647&Signature=kP9ghQJ1VLb0-tL9UWHx2tt6ZHSKNNGPfraMf27tfKYCDu~WqmMtAWubYHi1D5r8pa35kBj1qX1e~9YKKR7DTV3sVwPSyy5TfUfFh~E9od2WtRt8ddSSpm3oWCD54hKWqobMrb8GSpMH0MJe3kZWjkCJaO5VULqNUhx2Z3sdjxTzu24EYLJFCQKOaC2OCu~Hf84U9yLcCCfg7UMjvOOxBtKEm6Vbv56QLg6pHZoTVtBOLQZxz6pzFLsUYYUDTHDIhZylIdYify-dweBCd2yDxgJFvUTwih6sWrCzKh8fKjymJdr9VPHoCJLTAwMgGJ-zidhhLMRMbx23PXUvfsJapQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Comparison of structures at the exon–intron boundary according to the sequence predicted by computer and experimentally identified. A–C indicate the predicted structures of tRNA molecules with the exon–intron boundary assigned according to the software. D–F indicate the predicted structure of the same tRNA molecules with the exon–intron boundary experimentally confirmed in this work. The experimentally confirmed cleavage sites are indicated by short arrows. The anticodon regions are shown in the boxes. A and D: tRNA Lys (CUU), B and E: tRNA Lys (UUU), C and F: tRNA Pro (CGG). " path-from-xml="dsi023f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35870594/dsi023f5.jpeg" path-from-xml="dsi023f5.jpeg" target="_blank">View large</a><a section="35870594" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi023/3/dsi023f5.jpeg?Expires=2147483647&Signature=hCIb2x96fkmVvqRbhYfP2vqGXbdZlKQk~~dYWmISwKyWUJGrUvUG3lPqy6myDheYcjwhzNBWll9lkt4fC8AHdx2jvyzJJ9cXgKFYAnfj4TFORj1wBSd9S6YPIuCRjqQSLc1zVQWCFcmoURCLNnRyX8lCubL8s31zzGP0M-mgiOiWGRA9xLlPsRp~HIGdXzVAn6XNrIYecrMHSX-mxfQJ2SNko5zkzS~lbfvav7yweJxhQc6HCjXpl~vHuhTf2exdQzJ-hTj2gT7qIRrj5Sn6jBeuulhfMY-E8MF2wjWVtNZaHkshJZNxnqAJZ~Q6T4eU6a-L7dkEylFYqsqbBkcb5Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35870594&ar=344476&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi023f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Comparison of structures at the exon–intron boundary according to the sequence predicted by computer and experimentally identified. A–C indicate the predicted structures of tRNA molecules with the exon–intron boundary assigned according to the software. D–F indicate the predicted structure of the same tRNA molecules with the exon–intron boundary experimentally confirmed in this work. The experimentally confirmed cleavage sites are indicated by short arrows. The anticodon regions are shown in the boxes. A and D: tRNA <sup>Lys</sup> (CUU), B and E: tRNA <sup>Lys</sup> (UUU), C and F: tRNA <sup>Pro</sup> (CGG). </p></div></div><div content-id="fig5" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi023/3/m_dsi023f5.jpeg?Expires=2147483647&Signature=kP9ghQJ1VLb0-tL9UWHx2tt6ZHSKNNGPfraMf27tfKYCDu~WqmMtAWubYHi1D5r8pa35kBj1qX1e~9YKKR7DTV3sVwPSyy5TfUfFh~E9od2WtRt8ddSSpm3oWCD54hKWqobMrb8GSpMH0MJe3kZWjkCJaO5VULqNUhx2Z3sdjxTzu24EYLJFCQKOaC2OCu~Hf84U9yLcCCfg7UMjvOOxBtKEm6Vbv56QLg6pHZoTVtBOLQZxz6pzFLsUYYUDTHDIhZylIdYify-dweBCd2yDxgJFvUTwih6sWrCzKh8fKjymJdr9VPHoCJLTAwMgGJ-zidhhLMRMbx23PXUvfsJapQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Comparison of structures at the exon–intron boundary according to the sequence predicted by computer and experimentally identified. A–C indicate the predicted structures of tRNA molecules with the exon–intron boundary assigned according to the software. D–F indicate the predicted structure of the same tRNA molecules with the exon–intron boundary experimentally confirmed in this work. The experimentally confirmed cleavage sites are indicated by short arrows. The anticodon regions are shown in the boxes. A and D: tRNA Lys (CUU), B and E: tRNA Lys (UUU), C and F: tRNA Pro (CGG). " path-from-xml="dsi023f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/35870594/dsi023f5.jpeg" path-from-xml="dsi023f5.jpeg" target="_blank">View large</a><a section="35870594" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi023/3/dsi023f5.jpeg?Expires=2147483647&Signature=hCIb2x96fkmVvqRbhYfP2vqGXbdZlKQk~~dYWmISwKyWUJGrUvUG3lPqy6myDheYcjwhzNBWll9lkt4fC8AHdx2jvyzJJ9cXgKFYAnfj4TFORj1wBSd9S6YPIuCRjqQSLc1zVQWCFcmoURCLNnRyX8lCubL8s31zzGP0M-mgiOiWGRA9xLlPsRp~HIGdXzVAn6XNrIYecrMHSX-mxfQJ2SNko5zkzS~lbfvav7yweJxhQc6HCjXpl~vHuhTf2exdQzJ-hTj2gT7qIRrj5Sn6jBeuulhfMY-E8MF2wjWVtNZaHkshJZNxnqAJZ~Q6T4eU6a-L7dkEylFYqsqbBkcb5Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=35870594&ar=344476&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi023f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Comparison of structures at the exon–intron boundary according to the sequence predicted by computer and experimentally identified. A–C indicate the predicted structures of tRNA molecules with the exon–intron boundary assigned according to the software. D–F indicate the predicted structure of the same tRNA molecules with the exon–intron boundary experimentally confirmed in this work. The experimentally confirmed cleavage sites are indicated by short arrows. The anticodon regions are shown in the boxes. A and D: tRNA <sup>Lys</sup> (CUU), B and E: tRNA <sup>Lys</sup> (UUU), C and F: tRNA <sup>Pro</sup> (CGG). </p></div></div><p> These results indicate that the bulge–helix–bulge motif structure plays a most important role for recognition and cleavage of intron region in the tRNA genes in <em>A. pernix</em> . This fact indicates that the intron in the tRNA molecules is cleaved by the mechanism generally used in Archaea. Marck and Grosjean <a reveal-id="b20" data-open="b20" class="link link-ref link-reveal xref-bibr">20</a> reported that the clear bulge–helix–bulge motif structure was not recognized at the intron–exon boundary in some archaeal-interrupted tRNA genes. However, our results indicate that clear bulge–helix–bulge motif structure is necessary for correct recognition and cleavage of the intron region in archaeal tRNA genes, and that the data for insertion position of the intron used in analyses by Marck and Grosjean <a reveal-id="b20" data-open="b20" class="link link-ref link-reveal xref-bibr">20</a> might not be accurate, because of the absence of experimental confirmation for those data. Also it is noteworthy that a feature such as the bulge–helix–bulge motif structure conserved at exon–intron boundaries in archaeal tRNA genes must be introduced for computational prediction of the correct intron position in archaeal tRNA genes. </p><p> In both the rRNA genes of <em>A. pernix</em> K1, the intron sequences were previously identified. <a reveal-id="b21" data-open="b21" class="link link-ref link-reveal xref-bibr">21</a> The structure at the exon–intron boundary in these rRNA genes was predicted as the bulge–helix–bulge motif structure similar to those recognized at the exon–intron boundary in the tRNA molecules. Watanabe et al. <a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> indicated the evidence that an intron-containing protein-coding gene, APE0989 and APE0990, was present in <em>A. pernix</em> K1. The exon–intron boundary sequence of this protein-coding gene, which encodes a subunit of a small nucleolar ribonucleoprotein involved in eukaryotic rRNA, indicated the typical bulge–helix–bulge motif structure. The structural similarity of intron boundary sequences in archaeal genes encoding tRNA, rRNA and protein suggests the hypothesis that the introns of archaeal tRNA gene transfered to rRNA genes and protein-coding genes. If so, it can be thought that the intron region in archaeal tRNA gene is the ancestral intron in the protein-coding genes in Eukaryotes. </p><p> The tRNA–intron endonucleases identified from archaeal genomic data are classified into two types, the one is over 300 amino acid residues and the other is shorter than 190 residues. This difference in size suggests that these two types of endonucleases will exhibit different functions or mechanisms of cleavage of introns in tRNA molecules. From the genome data of <em>A. pernix</em> K1, the 187 residues long APE1646 protein was identified as a protein similar to the tRNA–intron endonuclease of <em>Haloferax volcanii</em> reported by Kleman-Leyer et al. <a reveal-id="b23" data-open="b23" class="link link-ref link-reveal xref-bibr">23</a></p><p> The tRNA ligase capable of joining exon ends was identified from yeast, <a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a> but not detected in halophiles. <a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a> We searched for this ligase in the genomic information of <em>A. pernix</em> K1. However, an ORF encoding the related enzyme was not detected as similar to the other Archaea. This ligase would be necessary for the joining of two exons cleaved by the tRNA endonuclease. Hence, identification and characterization of this ligase is the next important step for understanding the mechanism of tRNA splicing in Archaea. </p><p> The first prediction of tRNA genes in <em>A. pernix</em> K1 was performed by tRNAscan, which was developed by Lowe and Eddy, <a reveal-id="b25" data-open="b25" class="link link-ref link-reveal xref-bibr">25</a> as described in the previous genomic paper. Forty-five tRNA genes were predicted by this software, and two genes, tRNA <sup>Asp</sup> (GUC) and tRNA <sup>Trp</sup> (CCA), were predicted manually. The intron in tRNA <sup>Thr</sup> (UGU-1) and tRNA <sup>Trp</sup> (CCA) genes were manually predicted within the D stem, regions which were not detected by tRNAscan as intron, and at three bases 5′ from anticodon, respectively. The intron location of the other 12 interrupted tRNA genes was predicted at one base 3′ from anticodon in the tRNA gene. </p><p>According to the 33 uninterrupted tRNA genes, 31 tRNA molecules transcribed were confirmed in this work as same molecules as prediction. This result indicated the reliability of the assignment of the uninterrupted tRNA genes by tRNAscan.</p><p> In contrast, the result for confirmation of transcript from the interrupted tRNA genes predicted by tRNAscan was not in the same situation. Among 14 interrupted tRNA genes, the correct position of the intron portion in tRNA <sup>Ser</sup> (CGA) was not confirmed by this experiment, because of the failure of obtaining the mature type of its cDNA. The actual location of the intron for the remaining 13 interrupted tRNA genes were confirmed in this work, as both the premature and mature forms of cDNA molecules for 12 interrupted tRNA genes and the mature form of cDNA molecule for the tRNA <sup>Thr</sup> (CGU) were identified. A comparison of the mature sequence with the premature sequence of cDNA molecules indicated that transcripts from seven tRNA genes, tRNA <sup>Arg</sup> (UCU), tRNA <sup>Cys</sup> (GCA), tRNA <sup>Met</sup> (CAU-1), tRNA <sup>Met</sup> (CAU-2), tRNA <sup>Pro</sup> (GGG), tRNA <sup>Tyr</sup> (GUA) and tRNA <sup>Thr</sup> (CGU), were spliced at the exon–intron boundary predicted by software. However, the introns in six tRNA genes, shown in bold and underlined in <a reveal-id="tbl3" data-open="tbl3" class="link link-reveal link-table xref-fig">Table 3</a> , were processed at the other positions than those predicted by software. The ratio of correct assignment of the intron position is only 54%. This result shows that tRNAscan is reliable for assignment of the uninterrupted tRNA genes, but not reliable for assignment of the interrupted tRNA genes in <em>A. pernix</em> K1. </p><p> The tRNA genes are transcribed by RNA polymerase III, which recognizes the promoter sequence located in the coding part of the tRNA genes in Eukaryotes and that located upstream from the transcription start point in prokaryotic tRNA genes. The two distinct recognition boxes, 11 bp long A box: TRRYNNARYGG and B box: GGTTCGANTCC, were identified in the coding part of the Eukaryotic tRNA genes, <a reveal-id="b26" data-open="b26" class="link link-ref link-reveal xref-bibr">26</a> and similar sequences were also detected upstream of prokaryotic tRNA genes. <a reveal-id="b27" data-open="b27" class="link link-ref link-reveal xref-bibr">27</a> These boxed sequences were searched in the coding and upstream region of the tRNA genes of <em>A. pernix</em> K1, but no identical or related boxed sequence was identified. Instead of the absence of the known boxed sequences, the novel consensus motif sequences were identified in the upstream region of the tRNA genes. The consensus motif sequences identified and the original sequences of the upstream region of tRNA genes are summarized in <a reveal-id="tbl4" data-open="tbl4" class="link link-reveal link-table xref-fig">Table 4</a> . </p>                        <a id="35870603" scrollto-destination="35870603"></a>
<div content-id="tbl4" class="table-modal"><div class="table-wrap"><div id="tbl4" class="table-wrap-title" data-id="tbl4"><div class="label -label"><strong>Table 4</strong></div><div class="caption"><p>Comparison of 5′ non-coding region of tRNA genes and deduced consensus sequence</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>tRNA species </th><th>Motif A </th><th> Space <a reveal-id="tf4-1" data-open="tf4-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Motif B </th><th> Space <a reveal-id="tf4-1" data-open="tf4-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Motif C </th><th> Space <a reveal-id="tf4-2" data-open="tf4-2" class="link link-ref link-reveal xref-fn">b</a> </th></tr></thead><tbody><tr><td> tRNA <sup>Thr</sup> (CGU)  </td><td>aaa </td><td>10 </td><td>aata </td><td>17 </td><td>gagg </td><td>0 </td></tr><tr><td> tRNA <sup>Pro</sup> (CGG)  </td><td>gga </td><td>3 </td><td>gata </td><td>7 </td><td>gcgg </td><td>0 </td></tr><tr><td> tRNA <sup>Ser</sup> (GCU)  </td><td>ggagaa </td><td>6 </td><td>aata </td><td>3 </td><td>ctgg </td><td>2 </td></tr><tr><td> tRNA <sup>Arg</sup> (CCG)  </td><td>aaaaa </td><td>6 </td><td>aata </td><td>13 </td><td>gagg </td><td>0 </td></tr><tr><td> tRNA <sup>Lys</sup> (CUU)  </td><td>gggggaaaa </td><td>20 </td><td>gata </td><td>8 </td><td>ccgg </td><td>7 </td></tr><tr><td> tRNA <sup>Thr</sup> (UGU-1)  </td><td>aaa </td><td>15 </td><td>cata </td><td>11 </td><td>gagg </td><td>0 </td></tr><tr><td> tRNA <sup>Met</sup> (CAU-1)  </td><td>agaaa </td><td>4 </td><td>cata </td><td>2 </td><td>cagg </td><td>2 </td></tr><tr><td> tRNA <sup>Val</sup> (CAC)  </td><td>agagaa </td><td>28 </td><td>aata </td><td>11 </td><td>ggag </td><td>7 </td></tr><tr><td> tRNA <sup>Ala</sup> (CGC)  </td><td>gggagg </td><td>30 </td><td>aata </td><td>12 </td><td>ctgg </td><td>2 </td></tr><tr><td> tRNA <sup>Ser</sup> (CGA)  </td><td>aaggga </td><td>30 </td><td>gatg </td><td>14 </td><td>gagg </td><td>1 </td></tr><tr><td> tRNA <sup>Leu</sup> (UAG)  </td><td>gaaag </td><td>12 </td><td>acta </td><td>1 </td><td>acgg </td><td>1 </td></tr><tr><td> tRNA <sup>Asp</sup> (GUC)  </td><td>aaag </td><td>20 </td><td>gata </td><td>6 </td><td>ccgg </td><td>0 </td></tr><tr><td> tRNA <sup>Asn</sup> (GUU)  </td><td>ggaaa </td><td>26 </td><td>gata </td><td>2 </td><td>gagg </td><td>2 </td></tr><tr><td> tRNA <sup>Trp</sup> (CCA)  </td><td>gggaa </td><td>5 </td><td>aata </td><td>7 </td><td>gggg </td><td>2 </td></tr><tr><td> tRNA <sup>Met</sup> (CAU-2)  </td><td>gggaggaa </td><td>5 </td><td>gata </td><td>7 </td><td>gagg </td><td>3 </td></tr><tr><td> tRNA <sup>Leu</sup> (CAA)  </td><td>gggaaggaga </td><td>29 </td><td>aata </td><td>2 </td><td>atgg </td><td>2 </td></tr><tr><td> tRNA <sup>Ala</sup> (GGC)  </td><td>aaaaaga </td><td>3 </td><td>gata </td><td>33 </td><td>gcgg </td><td>4 </td></tr><tr><td> tRNA <sup>Gly</sup> (GCC)  </td><td>ggaaa </td><td>15 </td><td>agta </td><td>34 </td><td>ctgg </td><td>2 </td></tr><tr><td> tRNA <sup>Leu</sup> (UAA)  </td><td>ggggaga </td><td>0 </td><td>aata </td><td>7 </td><td>atgg </td><td>1 </td></tr><tr><td> tRNA <sup>Val</sup> (UAC)  </td><td>aaga </td><td>3 </td><td>gata </td><td>3 </td><td>ctgg </td><td>6 </td></tr><tr><td> tRNA <sup>Arg</sup> (GCG)  </td><td>aaggga </td><td>7 </td><td>aata </td><td>3 </td><td> cagg <a reveal-id="tf4-3" data-open="tf4-3" class="link link-ref link-reveal xref-fn">c</a> </td><td>0 </td></tr><tr><td> tRNA <sup>Ile</sup> (GAU)  </td><td>aaa </td><td>5 </td><td>aata </td><td>5 </td><td>ctgg </td><td>0 </td></tr><tr><td> tRNA <sup>Ser</sup> (UGA)  </td><td>agaggaga </td><td>37 </td><td>gata </td><td>12 </td><td>gagg </td><td>2 </td></tr><tr><td> tRNA <sup>Phe</sup> (GAA)  </td><td>aaaaa </td><td>14 </td><td>tcta </td><td>5 </td><td>gtgg </td><td>9 </td></tr><tr><td> tRNA <sup>Gln</sup> (UUG)  </td><td>agga </td><td>7 </td><td>ggta </td><td>10 </td><td>tcgg </td><td>6 </td></tr><tr><td> tRNA <sup>Gly</sup> (CCC)  </td><td>gggagga </td><td>59 </td><td>aata </td><td>2 </td><td>atgg </td><td>1 </td></tr><tr><td> tRNA <sup>Thr</sup> (GGU)  </td><td>ggaag </td><td>13 </td><td>tata </td><td>3 </td><td>gagg </td><td>1 </td></tr><tr><td> tRNA <sup>Arg</sup> (UCU)  </td><td>aaagg </td><td>14 </td><td>aata </td><td>6 </td><td>tcgg </td><td>3 </td></tr><tr><td> tRNA <sup>Met</sup> (CAU-3)  </td><td>aaaa </td><td>32 </td><td>aata </td><td>7 </td><td>gggg </td><td>6 </td></tr><tr><td> tRNA <sup>Glu</sup> (CUC)  </td><td>ggaga </td><td>2 </td><td>aata </td><td>41 </td><td>gcgg </td><td>2 </td></tr><tr><td> tRNA <sup>Pro</sup> (GGG)  </td><td>agaa </td><td>3 </td><td>gata </td><td>1 </td><td>acgg </td><td>3 </td></tr><tr><td> tRNA <sup>Ser</sup> (GGA)  </td><td>ggggg </td><td>51 </td><td>cata </td><td>14 </td><td>tagg </td><td>1 </td></tr><tr><td> tRNA <sup>Lys</sup> (UUU)  </td><td>aaag </td><td>12 </td><td>aata </td><td>9 </td><td>gagg </td><td>8 </td></tr><tr><td> tRNA <sup>Leu</sup> (GAG)  </td><td>gag </td><td>21 </td><td>aata </td><td>14 </td><td>acgg </td><td>1 </td></tr><tr><td> tRNA <sup>Ala</sup> (UGC)  </td><td>aaaaa </td><td>10 </td><td>cata </td><td>12 </td><td>ctgg </td><td>2 </td></tr><tr><td> tRNA <sup>Arg</sup> (CCU)  </td><td>gaggagg </td><td>23 </td><td>gata </td><td>18 </td><td>gagg </td><td>12 </td></tr><tr><td> tRNA <sup>Val</sup> (GAC)  </td><td>aagaa </td><td>2 </td><td>gata </td><td>5 </td><td>gggg </td><td>4 </td></tr><tr><td> tRNA <sup>Leu</sup> (CAG)  </td><td>aaagg </td><td>12 </td><td>aatc </td><td>7 </td><td>tcgg </td><td>1 </td></tr><tr><td> tRNA <sup>Glu</sup> (UUC)  </td><td>aaag </td><td>13 </td><td>tata </td><td>7 </td><td>gggg </td><td>1 </td></tr><tr><td> tRNA <sup>Cys</sup> (GCA)  </td><td>agagagaa </td><td>52 </td><td>acta </td><td>1 </td><td>ctgg </td><td>1 </td></tr><tr><td> tRNA <sup>His</sup> (GUG)  </td><td>gaggag </td><td>6 </td><td>agta </td><td>16 </td><td>aagg </td><td>3 </td></tr><tr><td> tRNA <sup>Tyr</sup> (GUA)  </td><td>agaaa </td><td>9 </td><td>aata </td><td>0 </td><td>gtgg </td><td>1 </td></tr><tr><td> tRNA <sup>Gln</sup> (CUG)  </td><td>agg </td><td>5 </td><td>agtg </td><td>6 </td><td>cagg </td><td>1 </td></tr><tr><td> tRNA <sup>Gly</sup> (UCC)  </td><td>aaaaa </td><td>12 </td><td>gata </td><td>17 </td><td>aagg </td><td>12 </td></tr><tr><td> tRNA <sup>Arg</sup> (UCG)  </td><td>gggaaaa </td><td>1 </td><td>cata </td><td>9 </td><td>cagg </td><td>18 </td></tr><tr><td> tRNA <sup>Pro</sup> (UGG)  </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> tRNA <sup>Thr</sup> (UGU-2)  </td><td>gggaaaaggaagaaggaag </td><td> </td><td> </td><td> </td><td>cagg </td><td>4 </td></tr><tr><td><strong>Consensus</strong><a reveal-id="tf4-4" data-open="tf4-4" class="link link-ref link-reveal xref-fn">d</a> </td><td><strong> Rn ( <em>n</em> = 3–10) </strong> </td><td><strong>0 –53</strong> </td><td>RRTA </td><td><strong>0–41</strong> </td><td>SRGG </td><td> </td></tr></tbody></table></div><div content-id="tbl4" class="table-modal"><table><thead name="thead" align=""><tr><th>tRNA species </th><th>Motif A </th><th> Space <a reveal-id="tf4-1" data-open="tf4-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Motif B </th><th> Space <a reveal-id="tf4-1" data-open="tf4-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Motif C </th><th> Space <a reveal-id="tf4-2" data-open="tf4-2" class="link link-ref link-reveal xref-fn">b</a> </th></tr></thead><tbody><tr><td> tRNA <sup>Thr</sup> (CGU)  </td><td>aaa </td><td>10 </td><td>aata </td><td>17 </td><td>gagg </td><td>0 </td></tr><tr><td> tRNA <sup>Pro</sup> (CGG)  </td><td>gga </td><td>3 </td><td>gata </td><td>7 </td><td>gcgg </td><td>0 </td></tr><tr><td> tRNA <sup>Ser</sup> (GCU)  </td><td>ggagaa </td><td>6 </td><td>aata </td><td>3 </td><td>ctgg </td><td>2 </td></tr><tr><td> tRNA <sup>Arg</sup> (CCG)  </td><td>aaaaa </td><td>6 </td><td>aata </td><td>13 </td><td>gagg </td><td>0 </td></tr><tr><td> tRNA <sup>Lys</sup> (CUU)  </td><td>gggggaaaa </td><td>20 </td><td>gata </td><td>8 </td><td>ccgg </td><td>7 </td></tr><tr><td> tRNA <sup>Thr</sup> (UGU-1)  </td><td>aaa </td><td>15 </td><td>cata </td><td>11 </td><td>gagg </td><td>0 </td></tr><tr><td> tRNA <sup>Met</sup> (CAU-1)  </td><td>agaaa </td><td>4 </td><td>cata </td><td>2 </td><td>cagg </td><td>2 </td></tr><tr><td> tRNA <sup>Val</sup> (CAC)  </td><td>agagaa </td><td>28 </td><td>aata </td><td>11 </td><td>ggag </td><td>7 </td></tr><tr><td> tRNA <sup>Ala</sup> (CGC)  </td><td>gggagg </td><td>30 </td><td>aata </td><td>12 </td><td>ctgg </td><td>2 </td></tr><tr><td> tRNA <sup>Ser</sup> (CGA)  </td><td>aaggga </td><td>30 </td><td>gatg </td><td>14 </td><td>gagg </td><td>1 </td></tr><tr><td> tRNA <sup>Leu</sup> (UAG)  </td><td>gaaag </td><td>12 </td><td>acta </td><td>1 </td><td>acgg </td><td>1 </td></tr><tr><td> tRNA <sup>Asp</sup> (GUC)  </td><td>aaag </td><td>20 </td><td>gata </td><td>6 </td><td>ccgg </td><td>0 </td></tr><tr><td> tRNA <sup>Asn</sup> (GUU)  </td><td>ggaaa </td><td>26 </td><td>gata </td><td>2 </td><td>gagg </td><td>2 </td></tr><tr><td> tRNA <sup>Trp</sup> (CCA)  </td><td>gggaa </td><td>5 </td><td>aata </td><td>7 </td><td>gggg </td><td>2 </td></tr><tr><td> tRNA <sup>Met</sup> (CAU-2)  </td><td>gggaggaa </td><td>5 </td><td>gata </td><td>7 </td><td>gagg </td><td>3 </td></tr><tr><td> tRNA <sup>Leu</sup> (CAA)  </td><td>gggaaggaga </td><td>29 </td><td>aata </td><td>2 </td><td>atgg </td><td>2 </td></tr><tr><td> tRNA <sup>Ala</sup> (GGC)  </td><td>aaaaaga </td><td>3 </td><td>gata </td><td>33 </td><td>gcgg </td><td>4 </td></tr><tr><td> tRNA <sup>Gly</sup> (GCC)  </td><td>ggaaa </td><td>15 </td><td>agta </td><td>34 </td><td>ctgg </td><td>2 </td></tr><tr><td> tRNA <sup>Leu</sup> (UAA)  </td><td>ggggaga </td><td>0 </td><td>aata </td><td>7 </td><td>atgg </td><td>1 </td></tr><tr><td> tRNA <sup>Val</sup> (UAC)  </td><td>aaga </td><td>3 </td><td>gata </td><td>3 </td><td>ctgg </td><td>6 </td></tr><tr><td> tRNA <sup>Arg</sup> (GCG)  </td><td>aaggga </td><td>7 </td><td>aata </td><td>3 </td><td> cagg <a reveal-id="tf4-3" data-open="tf4-3" class="link link-ref link-reveal xref-fn">c</a> </td><td>0 </td></tr><tr><td> tRNA <sup>Ile</sup> (GAU)  </td><td>aaa </td><td>5 </td><td>aata </td><td>5 </td><td>ctgg </td><td>0 </td></tr><tr><td> tRNA <sup>Ser</sup> (UGA)  </td><td>agaggaga </td><td>37 </td><td>gata </td><td>12 </td><td>gagg </td><td>2 </td></tr><tr><td> tRNA <sup>Phe</sup> (GAA)  </td><td>aaaaa </td><td>14 </td><td>tcta </td><td>5 </td><td>gtgg </td><td>9 </td></tr><tr><td> tRNA <sup>Gln</sup> (UUG)  </td><td>agga </td><td>7 </td><td>ggta </td><td>10 </td><td>tcgg </td><td>6 </td></tr><tr><td> tRNA <sup>Gly</sup> (CCC)  </td><td>gggagga </td><td>59 </td><td>aata </td><td>2 </td><td>atgg </td><td>1 </td></tr><tr><td> tRNA <sup>Thr</sup> (GGU)  </td><td>ggaag </td><td>13 </td><td>tata </td><td>3 </td><td>gagg </td><td>1 </td></tr><tr><td> tRNA <sup>Arg</sup> (UCU)  </td><td>aaagg </td><td>14 </td><td>aata </td><td>6 </td><td>tcgg </td><td>3 </td></tr><tr><td> tRNA <sup>Met</sup> (CAU-3)  </td><td>aaaa </td><td>32 </td><td>aata </td><td>7 </td><td>gggg </td><td>6 </td></tr><tr><td> tRNA <sup>Glu</sup> (CUC)  </td><td>ggaga </td><td>2 </td><td>aata </td><td>41 </td><td>gcgg </td><td>2 </td></tr><tr><td> tRNA <sup>Pro</sup> (GGG)  </td><td>agaa </td><td>3 </td><td>gata </td><td>1 </td><td>acgg </td><td>3 </td></tr><tr><td> tRNA <sup>Ser</sup> (GGA)  </td><td>ggggg </td><td>51 </td><td>cata </td><td>14 </td><td>tagg </td><td>1 </td></tr><tr><td> tRNA <sup>Lys</sup> (UUU)  </td><td>aaag </td><td>12 </td><td>aata </td><td>9 </td><td>gagg </td><td>8 </td></tr><tr><td> tRNA <sup>Leu</sup> (GAG)  </td><td>gag </td><td>21 </td><td>aata </td><td>14 </td><td>acgg </td><td>1 </td></tr><tr><td> tRNA <sup>Ala</sup> (UGC)  </td><td>aaaaa </td><td>10 </td><td>cata </td><td>12 </td><td>ctgg </td><td>2 </td></tr><tr><td> tRNA <sup>Arg</sup> (CCU)  </td><td>gaggagg </td><td>23 </td><td>gata </td><td>18 </td><td>gagg </td><td>12 </td></tr><tr><td> tRNA <sup>Val</sup> (GAC)  </td><td>aagaa </td><td>2 </td><td>gata </td><td>5 </td><td>gggg </td><td>4 </td></tr><tr><td> tRNA <sup>Leu</sup> (CAG)  </td><td>aaagg </td><td>12 </td><td>aatc </td><td>7 </td><td>tcgg </td><td>1 </td></tr><tr><td> tRNA <sup>Glu</sup> (UUC)  </td><td>aaag </td><td>13 </td><td>tata </td><td>7 </td><td>gggg </td><td>1 </td></tr><tr><td> tRNA <sup>Cys</sup> (GCA)  </td><td>agagagaa </td><td>52 </td><td>acta </td><td>1 </td><td>ctgg </td><td>1 </td></tr><tr><td> tRNA <sup>His</sup> (GUG)  </td><td>gaggag </td><td>6 </td><td>agta </td><td>16 </td><td>aagg </td><td>3 </td></tr><tr><td> tRNA <sup>Tyr</sup> (GUA)  </td><td>agaaa </td><td>9 </td><td>aata </td><td>0 </td><td>gtgg </td><td>1 </td></tr><tr><td> tRNA <sup>Gln</sup> (CUG)  </td><td>agg </td><td>5 </td><td>agtg </td><td>6 </td><td>cagg </td><td>1 </td></tr><tr><td> tRNA <sup>Gly</sup> (UCC)  </td><td>aaaaa </td><td>12 </td><td>gata </td><td>17 </td><td>aagg </td><td>12 </td></tr><tr><td> tRNA <sup>Arg</sup> (UCG)  </td><td>gggaaaa </td><td>1 </td><td>cata </td><td>9 </td><td>cagg </td><td>18 </td></tr><tr><td> tRNA <sup>Pro</sup> (UGG)  </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> tRNA <sup>Thr</sup> (UGU-2)  </td><td>gggaaaaggaagaaggaag </td><td> </td><td> </td><td> </td><td>cagg </td><td>4 </td></tr><tr><td><strong>Consensus</strong><a reveal-id="tf4-4" data-open="tf4-4" class="link link-ref link-reveal xref-fn">d</a> </td><td><strong> Rn ( <em>n</em> = 3–10) </strong> </td><td><strong>0 –53</strong> </td><td>RRTA </td><td><strong>0–41</strong> </td><td>SRGG </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf4-1"><div class="fn"><p><sup>a</sup> Space between motifs indicated by number of nucleotides. </p></div></div><div content-id="tf4-2"><div class="fn"><p><sup>b</sup> Space from first nucleotide of tRNA coding region. </p></div></div><div content-id="tf4-3"><div class="fn"><p><sup>c</sup> Last g is overlapped with first coding nucleotide. </p></div></div><div content-id="tf4-4"><div class="fn"><p><sup>d</sup> R indicates purine nucleotides; A or G, N indicates any nucleotides; S indicates G or C. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/35870603" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl4" class="table-wrap-title" data-id="tbl4"><div class="label -label"><strong>Table 4</strong></div><div class="caption"><p>Comparison of 5′ non-coding region of tRNA genes and deduced consensus sequence</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>tRNA species </th><th>Motif A </th><th> Space <a reveal-id="tf4-1" data-open="tf4-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Motif B </th><th> Space <a reveal-id="tf4-1" data-open="tf4-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Motif C </th><th> Space <a reveal-id="tf4-2" data-open="tf4-2" class="link link-ref link-reveal xref-fn">b</a> </th></tr></thead><tbody><tr><td> tRNA <sup>Thr</sup> (CGU)  </td><td>aaa </td><td>10 </td><td>aata </td><td>17 </td><td>gagg </td><td>0 </td></tr><tr><td> tRNA <sup>Pro</sup> (CGG)  </td><td>gga </td><td>3 </td><td>gata </td><td>7 </td><td>gcgg </td><td>0 </td></tr><tr><td> tRNA <sup>Ser</sup> (GCU)  </td><td>ggagaa </td><td>6 </td><td>aata </td><td>3 </td><td>ctgg </td><td>2 </td></tr><tr><td> tRNA <sup>Arg</sup> (CCG)  </td><td>aaaaa </td><td>6 </td><td>aata </td><td>13 </td><td>gagg </td><td>0 </td></tr><tr><td> tRNA <sup>Lys</sup> (CUU)  </td><td>gggggaaaa </td><td>20 </td><td>gata </td><td>8 </td><td>ccgg </td><td>7 </td></tr><tr><td> tRNA <sup>Thr</sup> (UGU-1)  </td><td>aaa </td><td>15 </td><td>cata </td><td>11 </td><td>gagg </td><td>0 </td></tr><tr><td> tRNA <sup>Met</sup> (CAU-1)  </td><td>agaaa </td><td>4 </td><td>cata </td><td>2 </td><td>cagg </td><td>2 </td></tr><tr><td> tRNA <sup>Val</sup> (CAC)  </td><td>agagaa </td><td>28 </td><td>aata </td><td>11 </td><td>ggag </td><td>7 </td></tr><tr><td> tRNA <sup>Ala</sup> (CGC)  </td><td>gggagg </td><td>30 </td><td>aata </td><td>12 </td><td>ctgg </td><td>2 </td></tr><tr><td> tRNA <sup>Ser</sup> (CGA)  </td><td>aaggga </td><td>30 </td><td>gatg </td><td>14 </td><td>gagg </td><td>1 </td></tr><tr><td> tRNA <sup>Leu</sup> (UAG)  </td><td>gaaag </td><td>12 </td><td>acta </td><td>1 </td><td>acgg </td><td>1 </td></tr><tr><td> tRNA <sup>Asp</sup> (GUC)  </td><td>aaag </td><td>20 </td><td>gata </td><td>6 </td><td>ccgg </td><td>0 </td></tr><tr><td> tRNA <sup>Asn</sup> (GUU)  </td><td>ggaaa </td><td>26 </td><td>gata </td><td>2 </td><td>gagg </td><td>2 </td></tr><tr><td> tRNA <sup>Trp</sup> (CCA)  </td><td>gggaa </td><td>5 </td><td>aata </td><td>7 </td><td>gggg </td><td>2 </td></tr><tr><td> tRNA <sup>Met</sup> (CAU-2)  </td><td>gggaggaa </td><td>5 </td><td>gata </td><td>7 </td><td>gagg </td><td>3 </td></tr><tr><td> tRNA <sup>Leu</sup> (CAA)  </td><td>gggaaggaga </td><td>29 </td><td>aata </td><td>2 </td><td>atgg </td><td>2 </td></tr><tr><td> tRNA <sup>Ala</sup> (GGC)  </td><td>aaaaaga </td><td>3 </td><td>gata </td><td>33 </td><td>gcgg </td><td>4 </td></tr><tr><td> tRNA <sup>Gly</sup> (GCC)  </td><td>ggaaa </td><td>15 </td><td>agta </td><td>34 </td><td>ctgg </td><td>2 </td></tr><tr><td> tRNA <sup>Leu</sup> (UAA)  </td><td>ggggaga </td><td>0 </td><td>aata </td><td>7 </td><td>atgg </td><td>1 </td></tr><tr><td> tRNA <sup>Val</sup> (UAC)  </td><td>aaga </td><td>3 </td><td>gata </td><td>3 </td><td>ctgg </td><td>6 </td></tr><tr><td> tRNA <sup>Arg</sup> (GCG)  </td><td>aaggga </td><td>7 </td><td>aata </td><td>3 </td><td> cagg <a reveal-id="tf4-3" data-open="tf4-3" class="link link-ref link-reveal xref-fn">c</a> </td><td>0 </td></tr><tr><td> tRNA <sup>Ile</sup> (GAU)  </td><td>aaa </td><td>5 </td><td>aata </td><td>5 </td><td>ctgg </td><td>0 </td></tr><tr><td> tRNA <sup>Ser</sup> (UGA)  </td><td>agaggaga </td><td>37 </td><td>gata </td><td>12 </td><td>gagg </td><td>2 </td></tr><tr><td> tRNA <sup>Phe</sup> (GAA)  </td><td>aaaaa </td><td>14 </td><td>tcta </td><td>5 </td><td>gtgg </td><td>9 </td></tr><tr><td> tRNA <sup>Gln</sup> (UUG)  </td><td>agga </td><td>7 </td><td>ggta </td><td>10 </td><td>tcgg </td><td>6 </td></tr><tr><td> tRNA <sup>Gly</sup> (CCC)  </td><td>gggagga </td><td>59 </td><td>aata </td><td>2 </td><td>atgg </td><td>1 </td></tr><tr><td> tRNA <sup>Thr</sup> (GGU)  </td><td>ggaag </td><td>13 </td><td>tata </td><td>3 </td><td>gagg </td><td>1 </td></tr><tr><td> tRNA <sup>Arg</sup> (UCU)  </td><td>aaagg </td><td>14 </td><td>aata </td><td>6 </td><td>tcgg </td><td>3 </td></tr><tr><td> tRNA <sup>Met</sup> (CAU-3)  </td><td>aaaa </td><td>32 </td><td>aata </td><td>7 </td><td>gggg </td><td>6 </td></tr><tr><td> tRNA <sup>Glu</sup> (CUC)  </td><td>ggaga </td><td>2 </td><td>aata </td><td>41 </td><td>gcgg </td><td>2 </td></tr><tr><td> tRNA <sup>Pro</sup> (GGG)  </td><td>agaa </td><td>3 </td><td>gata </td><td>1 </td><td>acgg </td><td>3 </td></tr><tr><td> tRNA <sup>Ser</sup> (GGA)  </td><td>ggggg </td><td>51 </td><td>cata </td><td>14 </td><td>tagg </td><td>1 </td></tr><tr><td> tRNA <sup>Lys</sup> (UUU)  </td><td>aaag </td><td>12 </td><td>aata </td><td>9 </td><td>gagg </td><td>8 </td></tr><tr><td> tRNA <sup>Leu</sup> (GAG)  </td><td>gag </td><td>21 </td><td>aata </td><td>14 </td><td>acgg </td><td>1 </td></tr><tr><td> tRNA <sup>Ala</sup> (UGC)  </td><td>aaaaa </td><td>10 </td><td>cata </td><td>12 </td><td>ctgg </td><td>2 </td></tr><tr><td> tRNA <sup>Arg</sup> (CCU)  </td><td>gaggagg </td><td>23 </td><td>gata </td><td>18 </td><td>gagg </td><td>12 </td></tr><tr><td> tRNA <sup>Val</sup> (GAC)  </td><td>aagaa </td><td>2 </td><td>gata </td><td>5 </td><td>gggg </td><td>4 </td></tr><tr><td> tRNA <sup>Leu</sup> (CAG)  </td><td>aaagg </td><td>12 </td><td>aatc </td><td>7 </td><td>tcgg </td><td>1 </td></tr><tr><td> tRNA <sup>Glu</sup> (UUC)  </td><td>aaag </td><td>13 </td><td>tata </td><td>7 </td><td>gggg </td><td>1 </td></tr><tr><td> tRNA <sup>Cys</sup> (GCA)  </td><td>agagagaa </td><td>52 </td><td>acta </td><td>1 </td><td>ctgg </td><td>1 </td></tr><tr><td> tRNA <sup>His</sup> (GUG)  </td><td>gaggag </td><td>6 </td><td>agta </td><td>16 </td><td>aagg </td><td>3 </td></tr><tr><td> tRNA <sup>Tyr</sup> (GUA)  </td><td>agaaa </td><td>9 </td><td>aata </td><td>0 </td><td>gtgg </td><td>1 </td></tr><tr><td> tRNA <sup>Gln</sup> (CUG)  </td><td>agg </td><td>5 </td><td>agtg </td><td>6 </td><td>cagg </td><td>1 </td></tr><tr><td> tRNA <sup>Gly</sup> (UCC)  </td><td>aaaaa </td><td>12 </td><td>gata </td><td>17 </td><td>aagg </td><td>12 </td></tr><tr><td> tRNA <sup>Arg</sup> (UCG)  </td><td>gggaaaa </td><td>1 </td><td>cata </td><td>9 </td><td>cagg </td><td>18 </td></tr><tr><td> tRNA <sup>Pro</sup> (UGG)  </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> tRNA <sup>Thr</sup> (UGU-2)  </td><td>gggaaaaggaagaaggaag </td><td> </td><td> </td><td> </td><td>cagg </td><td>4 </td></tr><tr><td><strong>Consensus</strong><a reveal-id="tf4-4" data-open="tf4-4" class="link link-ref link-reveal xref-fn">d</a> </td><td><strong> Rn ( <em>n</em> = 3–10) </strong> </td><td><strong>0 –53</strong> </td><td>RRTA </td><td><strong>0–41</strong> </td><td>SRGG </td><td> </td></tr></tbody></table></div><div content-id="tbl4" class="table-modal"><table><thead name="thead" align=""><tr><th>tRNA species </th><th>Motif A </th><th> Space <a reveal-id="tf4-1" data-open="tf4-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Motif B </th><th> Space <a reveal-id="tf4-1" data-open="tf4-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Motif C </th><th> Space <a reveal-id="tf4-2" data-open="tf4-2" class="link link-ref link-reveal xref-fn">b</a> </th></tr></thead><tbody><tr><td> tRNA <sup>Thr</sup> (CGU)  </td><td>aaa </td><td>10 </td><td>aata </td><td>17 </td><td>gagg </td><td>0 </td></tr><tr><td> tRNA <sup>Pro</sup> (CGG)  </td><td>gga </td><td>3 </td><td>gata </td><td>7 </td><td>gcgg </td><td>0 </td></tr><tr><td> tRNA <sup>Ser</sup> (GCU)  </td><td>ggagaa </td><td>6 </td><td>aata </td><td>3 </td><td>ctgg </td><td>2 </td></tr><tr><td> tRNA <sup>Arg</sup> (CCG)  </td><td>aaaaa </td><td>6 </td><td>aata </td><td>13 </td><td>gagg </td><td>0 </td></tr><tr><td> tRNA <sup>Lys</sup> (CUU)  </td><td>gggggaaaa </td><td>20 </td><td>gata </td><td>8 </td><td>ccgg </td><td>7 </td></tr><tr><td> tRNA <sup>Thr</sup> (UGU-1)  </td><td>aaa </td><td>15 </td><td>cata </td><td>11 </td><td>gagg </td><td>0 </td></tr><tr><td> tRNA <sup>Met</sup> (CAU-1)  </td><td>agaaa </td><td>4 </td><td>cata </td><td>2 </td><td>cagg </td><td>2 </td></tr><tr><td> tRNA <sup>Val</sup> (CAC)  </td><td>agagaa </td><td>28 </td><td>aata </td><td>11 </td><td>ggag </td><td>7 </td></tr><tr><td> tRNA <sup>Ala</sup> (CGC)  </td><td>gggagg </td><td>30 </td><td>aata </td><td>12 </td><td>ctgg </td><td>2 </td></tr><tr><td> tRNA <sup>Ser</sup> (CGA)  </td><td>aaggga </td><td>30 </td><td>gatg </td><td>14 </td><td>gagg </td><td>1 </td></tr><tr><td> tRNA <sup>Leu</sup> (UAG)  </td><td>gaaag </td><td>12 </td><td>acta </td><td>1 </td><td>acgg </td><td>1 </td></tr><tr><td> tRNA <sup>Asp</sup> (GUC)  </td><td>aaag </td><td>20 </td><td>gata </td><td>6 </td><td>ccgg </td><td>0 </td></tr><tr><td> tRNA <sup>Asn</sup> (GUU)  </td><td>ggaaa </td><td>26 </td><td>gata </td><td>2 </td><td>gagg </td><td>2 </td></tr><tr><td> tRNA <sup>Trp</sup> (CCA)  </td><td>gggaa </td><td>5 </td><td>aata </td><td>7 </td><td>gggg </td><td>2 </td></tr><tr><td> tRNA <sup>Met</sup> (CAU-2)  </td><td>gggaggaa </td><td>5 </td><td>gata </td><td>7 </td><td>gagg </td><td>3 </td></tr><tr><td> tRNA <sup>Leu</sup> (CAA)  </td><td>gggaaggaga </td><td>29 </td><td>aata </td><td>2 </td><td>atgg </td><td>2 </td></tr><tr><td> tRNA <sup>Ala</sup> (GGC)  </td><td>aaaaaga </td><td>3 </td><td>gata </td><td>33 </td><td>gcgg </td><td>4 </td></tr><tr><td> tRNA <sup>Gly</sup> (GCC)  </td><td>ggaaa </td><td>15 </td><td>agta </td><td>34 </td><td>ctgg </td><td>2 </td></tr><tr><td> tRNA <sup>Leu</sup> (UAA)  </td><td>ggggaga </td><td>0 </td><td>aata </td><td>7 </td><td>atgg </td><td>1 </td></tr><tr><td> tRNA <sup>Val</sup> (UAC)  </td><td>aaga </td><td>3 </td><td>gata </td><td>3 </td><td>ctgg </td><td>6 </td></tr><tr><td> tRNA <sup>Arg</sup> (GCG)  </td><td>aaggga </td><td>7 </td><td>aata </td><td>3 </td><td> cagg <a reveal-id="tf4-3" data-open="tf4-3" class="link link-ref link-reveal xref-fn">c</a> </td><td>0 </td></tr><tr><td> tRNA <sup>Ile</sup> (GAU)  </td><td>aaa </td><td>5 </td><td>aata </td><td>5 </td><td>ctgg </td><td>0 </td></tr><tr><td> tRNA <sup>Ser</sup> (UGA)  </td><td>agaggaga </td><td>37 </td><td>gata </td><td>12 </td><td>gagg </td><td>2 </td></tr><tr><td> tRNA <sup>Phe</sup> (GAA)  </td><td>aaaaa </td><td>14 </td><td>tcta </td><td>5 </td><td>gtgg </td><td>9 </td></tr><tr><td> tRNA <sup>Gln</sup> (UUG)  </td><td>agga </td><td>7 </td><td>ggta </td><td>10 </td><td>tcgg </td><td>6 </td></tr><tr><td> tRNA <sup>Gly</sup> (CCC)  </td><td>gggagga </td><td>59 </td><td>aata </td><td>2 </td><td>atgg </td><td>1 </td></tr><tr><td> tRNA <sup>Thr</sup> (GGU)  </td><td>ggaag </td><td>13 </td><td>tata </td><td>3 </td><td>gagg </td><td>1 </td></tr><tr><td> tRNA <sup>Arg</sup> (UCU)  </td><td>aaagg </td><td>14 </td><td>aata </td><td>6 </td><td>tcgg </td><td>3 </td></tr><tr><td> tRNA <sup>Met</sup> (CAU-3)  </td><td>aaaa </td><td>32 </td><td>aata </td><td>7 </td><td>gggg </td><td>6 </td></tr><tr><td> tRNA <sup>Glu</sup> (CUC)  </td><td>ggaga </td><td>2 </td><td>aata </td><td>41 </td><td>gcgg </td><td>2 </td></tr><tr><td> tRNA <sup>Pro</sup> (GGG)  </td><td>agaa </td><td>3 </td><td>gata </td><td>1 </td><td>acgg </td><td>3 </td></tr><tr><td> tRNA <sup>Ser</sup> (GGA)  </td><td>ggggg </td><td>51 </td><td>cata </td><td>14 </td><td>tagg </td><td>1 </td></tr><tr><td> tRNA <sup>Lys</sup> (UUU)  </td><td>aaag </td><td>12 </td><td>aata </td><td>9 </td><td>gagg </td><td>8 </td></tr><tr><td> tRNA <sup>Leu</sup> (GAG)  </td><td>gag </td><td>21 </td><td>aata </td><td>14 </td><td>acgg </td><td>1 </td></tr><tr><td> tRNA <sup>Ala</sup> (UGC)  </td><td>aaaaa </td><td>10 </td><td>cata </td><td>12 </td><td>ctgg </td><td>2 </td></tr><tr><td> tRNA <sup>Arg</sup> (CCU)  </td><td>gaggagg </td><td>23 </td><td>gata </td><td>18 </td><td>gagg </td><td>12 </td></tr><tr><td> tRNA <sup>Val</sup> (GAC)  </td><td>aagaa </td><td>2 </td><td>gata </td><td>5 </td><td>gggg </td><td>4 </td></tr><tr><td> tRNA <sup>Leu</sup> (CAG)  </td><td>aaagg </td><td>12 </td><td>aatc </td><td>7 </td><td>tcgg </td><td>1 </td></tr><tr><td> tRNA <sup>Glu</sup> (UUC)  </td><td>aaag </td><td>13 </td><td>tata </td><td>7 </td><td>gggg </td><td>1 </td></tr><tr><td> tRNA <sup>Cys</sup> (GCA)  </td><td>agagagaa </td><td>52 </td><td>acta </td><td>1 </td><td>ctgg </td><td>1 </td></tr><tr><td> tRNA <sup>His</sup> (GUG)  </td><td>gaggag </td><td>6 </td><td>agta </td><td>16 </td><td>aagg </td><td>3 </td></tr><tr><td> tRNA <sup>Tyr</sup> (GUA)  </td><td>agaaa </td><td>9 </td><td>aata </td><td>0 </td><td>gtgg </td><td>1 </td></tr><tr><td> tRNA <sup>Gln</sup> (CUG)  </td><td>agg </td><td>5 </td><td>agtg </td><td>6 </td><td>cagg </td><td>1 </td></tr><tr><td> tRNA <sup>Gly</sup> (UCC)  </td><td>aaaaa </td><td>12 </td><td>gata </td><td>17 </td><td>aagg </td><td>12 </td></tr><tr><td> tRNA <sup>Arg</sup> (UCG)  </td><td>gggaaaa </td><td>1 </td><td>cata </td><td>9 </td><td>cagg </td><td>18 </td></tr><tr><td> tRNA <sup>Pro</sup> (UGG)  </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> tRNA <sup>Thr</sup> (UGU-2)  </td><td>gggaaaaggaagaaggaag </td><td> </td><td> </td><td> </td><td>cagg </td><td>4 </td></tr><tr><td><strong>Consensus</strong><a reveal-id="tf4-4" data-open="tf4-4" class="link link-ref link-reveal xref-fn">d</a> </td><td><strong> Rn ( <em>n</em> = 3–10) </strong> </td><td><strong>0 –53</strong> </td><td>RRTA </td><td><strong>0–41</strong> </td><td>SRGG </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tf4-1"><div class="fn"><p><sup>a</sup> Space between motifs indicated by number of nucleotides. </p></div></div><div content-id="tf4-2"><div class="fn"><p><sup>b</sup> Space from first nucleotide of tRNA coding region. </p></div></div><div content-id="tf4-3"><div class="fn"><p><sup>c</sup> Last g is overlapped with first coding nucleotide. </p></div></div><div content-id="tf4-4"><div class="fn"><p><sup>d</sup> R indicates purine nucleotides; A or G, N indicates any nucleotides; S indicates G or C. </p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/35870603" target="_blank">View Large</a></div></div><p> The consensus motif sequence identified consisted of three independent consensus elements, motif A is poly-purines, motif B is RRTA and motif C is SRGG, and each consensus motif was separated by the spacer sequences with different length. The consensus motif sequences were not identified at the upstream region of the tRNA <sup>Thr</sup> (UGU-2) gene, which supported the suggestion that this region was not worked as tRNA gene. </p><p> Three tRNA gene clusters, tRNA <sup>Met</sup> (CAU-1)-tRNA <sup>Thr</sup> (UGU-1), tRNA <sup>Ser</sup> (CGA)-tRNA <sup>Pro</sup> (UGG) and tRNA <sup>Cys</sup> (GCA)-tRNA <sup>Glu</sup> (UUC), were predicted to be transcribed as a single transcript. The promoter motif sequences were identified at the upstream region of the gene clusters, but conservation of the promoter motif sequences was not clearly identified at the upstream region of the downstream genes. As the tRNA <sup>Pro</sup> (UGG) gene was located downstream of the tRNA <sup>Ser</sup> (CGA) gene with a distance of just 7 nt, the cDNA for tRNA <sup>Pro</sup> (UGG) and premature form of tRNA <sup>Ser</sup> (CGA) were obtained from the total RNA. However, the cDNA for the mature form of tRNA <sup>Ser</sup> (CGA) was not obtained, suggesting that splicing of the premature form of the tRNA <sup>Ser</sup> (CGA) molecule should not have efficiently occurred. </p><p> Our study emphasizes that genomic information is a powerful source from which is investigated the analyses of a family of molecules or reactions in one organism. Our results provide the opportunity for future research on the more detailed understanding of a variety of biological events in Archaea. Since the tRNA ligase was not identified from Archaea, it is necessary to identify the tRNA ligase from Archaea, which also provides the information for recognition mechanism of the exon portion and construction mechanism of the mature tRNA molecules. As three tRNA gene clusters were identified in <em>A. pernix</em> , it is also targeted to understand the actual cleavage mechanism of the transcript from the cluster tRNA genes into each independent tRNA molecule in Archaea cell. Since the similar intron portion was identified in the tRNA, rRNA and protein-coding genes in <em>A. pernix</em> , this microorganism is thought to be one of the most convenient materials for understanding the mechanism for evolution of intron region and the machinery utilized in <em>A. pernix</em> for processing the independent but similar intron portion possessed in the different types of genes. </p><p> We thank Y. Hino for technical support of the nucleotide sequencing and T. Tanaka for assistance with the computational analysis of the genomic data of <em>A. pernix</em> K1. This work was supported by the Ministry of Economy, Trade and Industry. </p>                            <h2 scrollto-destination=35870609 id="35870609" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Westaway</div> <div class="given-names">S. K.</div></div>,  <div class="name"><div class="surname">Abelson</div> <div class="given-names">J.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Söll</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">RajBhandary</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Splicing of tRNA Precursors</div>, <div class="source">tRNA: Structure, Biosynthesis, and Function</div> , <div class="year">1995</div><div class="publisher-loc">Washington, DC</div><div class="publisher-name">American Society for Microbiology</div>(pg. <div class="fpage">79</div>-<div class="lpage">92</div>)</div></div></div></div><div content-id="b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Szekely</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Belford</div> <div class="given-names">H. G.</div></div>,  <div class="name"><div class="surname">Greer</div> <div class="given-names">C. L.</div></div>. </div><div class="article-title"> Intron sequence and structure requirements for tRNA splicing in <em>Saccharomyces cerevisiae</em></div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1988</div>, vol. <div class="volume">263</div> (pg. <div class="fpage">13839</div>-<div class="lpage">13847</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intron%20sequence%20and%20structure%20requirements%20for%20tRNA%20splicing%20in%20Saccharomyces%20cerevisiae&amp;author=E.%20Szekely&amp;author=H.%20G.%20Belford&amp;author=C.%20L.%20Greer&amp;publication_year=1988&amp;journal=J.%20Biol.%20Chem.&amp;volume=263&amp;pages=13839-13847" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2843539" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Swerdlow</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Guthrie</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Structure of intron-containing tRNA precursors. Analysis of solution conformation using chemical and enzymatic probes</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1984</div>, vol. <div class="volume">259</div> (pg. <div class="fpage">5197</div>-<div class="lpage">5207</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20of%20intron-containing%20tRNA%20precursors.%20Analysis%20of%20solution%20conformation%20using%20chemical%20and%20enzymatic%20probes&amp;author=H.%20Swerdlow&amp;author=C.%20Guthrie&amp;publication_year=1984&amp;journal=J.%20Biol.%20Chem.&amp;volume=259&amp;pages=5197-5207" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6371001" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Reyes</div> <div class="given-names">V. M.</div></div>,  <div class="name"><div class="surname">Abelson</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Substrate recognition and splice site determination in yeast tRNA splicing</div>, <div class="source">Cell</div> , <div class="year">1988</div>, vol. <div class="volume">55</div> (pg. <div class="fpage">719</div>-<div class="lpage">730</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Substrate%20recognition%20and%20splice%20site%20determination%20in%20yeast%20tRNA%20splicing&amp;author=V.%20M.%20Reyes&amp;author=J.%20Abelson&amp;publication_year=1988&amp;journal=Cell&amp;volume=55&amp;pages=719-730" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(88)90230-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(88)90230-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(88)90230-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3141064" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mattoccia</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Baldi</div> <div class="given-names">I. M.</div></div>,  <div class="name"><div class="surname">Gandini-Attardi</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Ciafre</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tocchini-valentini</div> <div class="given-names">G. P.</div></div>. </div><div class="article-title"> Site selection by the splicing endonuclease of <em>Xenopus Laevis</em></div>, <div class="source">Cell</div> , <div class="year">1988</div>, vol. <div class="volume">55</div> (pg. <div class="fpage">731</div>-<div class="lpage">738</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Site%20selection%20by%20the%20splicing%20endonuclease%20of%20Xenopus%20Laevis&amp;author=E.%20Mattoccia&amp;author=I.%20M.%20Baldi&amp;author=D.%20Gandini-Attardi&amp;author=S.%20Ciafre&amp;author=G.%20P.%20Tocchini-valentini&amp;publication_year=1988&amp;journal=Cell&amp;volume=55&amp;pages=731-738" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(88)90231-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(88)90231-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(88)90231-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3180224" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">L. D.</div></div>,  <div class="name"><div class="surname">Daniels</div> <div class="given-names">C. J.</div></div>. </div><div class="article-title"> A tRNA(Trp) intron endonuclease from <em>Halobacterium volcanii</em> . Unique substrate recognition properties </div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1988</div>, vol. <div class="volume">263</div> (pg. <div class="fpage">17951</div>-<div class="lpage">17959</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20tRNA(Trp)%20intron%20endonuclease%20from%20Halobacterium%20volcanii%20.%20Unique%20substrate%20recognition%20properties&amp;author=L.%20D.%20Thompson&amp;author=C.%20J.%20Daniels&amp;publication_year=1988&amp;journal=J.%20Biol.%20Chem.&amp;volume=263&amp;pages=17951-17959" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3192521" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">L. D.</div></div>,  <div class="name"><div class="surname">Daniels</div> <div class="given-names">C. J.</div></div>. </div><div class="article-title"> Recognition of exon–intron boundaries by the <em>Halobacterium volcanii</em> tRNA intron endonuclease </div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1990</div>, vol. <div class="volume">265</div> (pg. <div class="fpage">18104</div>-<div class="lpage">18111</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recognition%20of%20exon%E2%80%93intron%20boundaries%20by%20the%20Halobacterium%20volcanii%20tRNA%20intron%20endonuclease&amp;author=L.%20D.%20Thompson&amp;author=C.%20J.%20Daniels&amp;publication_year=1990&amp;journal=J.%20Biol.%20Chem.&amp;volume=265&amp;pages=18104-18111" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1698785" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Daniels</div> <div class="given-names">C. J.</div></div>,  <div class="name"><div class="surname">Gupta</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Doolittle</div> <div class="given-names">W. F.</div></div>. </div><div class="article-title"> Transcription and excision of a large intron in the tRNA <sup>Trp</sup> gene of an archaebacterium, <em>Halobacterium volcanii</em></div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1985</div>, vol. <div class="volume">260</div> (pg. <div class="fpage">3132</div>-<div class="lpage">3134</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription%20and%20excision%20of%20a%20large%20intron%20in%20the%20tRNA%20Trp%20gene%20of%20an%20archaebacterium%2C%20Halobacterium%20volcanii&amp;author=C.%20J.%20Daniels&amp;author=R.%20Gupta&amp;author=W.%20F.%20Doolittle&amp;publication_year=1985&amp;journal=J.%20Biol.%20Chem.&amp;volume=260&amp;pages=3132-3134" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3844407" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Datta</div> <div class="given-names">P. K.</div></div>,  <div class="name"><div class="surname">Hawkins</div> <div class="given-names">L. K.</div></div>,  <div class="name"><div class="surname">Gupta</div> <div class="given-names">R.</div></div>. </div><div class="article-title"> Presence of an intron in elongator methionine-tRNA of <em>Halobacterium volcanii</em></div>, <div class="source">Can. J. Microbiol.</div> , <div class="year">1989</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">189</div>-<div class="lpage">194</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Presence%20of%20an%20intron%20in%20elongator%20methionine-tRNA%20of%20Halobacterium%20volcanii&amp;author=P.%20K.%20Datta&amp;author=L.%20K.%20Hawkins&amp;author=R.%20Gupta&amp;publication_year=1989&amp;journal=Can.%20J.%20Microbiol.&amp;volume=35&amp;pages=189-194" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/m89-029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fm89-029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fm89-029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2470482" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaine</div> <div class="given-names">B. P.</div></div>,  <div class="name"><div class="surname">Gupta</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Woese</div> <div class="given-names">C. R.</div></div>. </div><div class="article-title">Putative introns in tRNA genes of prokaryotes</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1983</div>, vol. <div class="volume">80</div> (pg. <div class="fpage">3309</div>-<div class="lpage">3312</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Putative%20introns%20in%20tRNA%20genes%20of%20prokaryotes&amp;author=B.%20P.%20Kaine&amp;author=R.%20Gupta&amp;author=C.%20R.%20Woese&amp;publication_year=1983&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=80&amp;pages=3309-3312" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.80.11.3309" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.80.11.3309" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.80.11.3309"> </span></div> </div></div></div></div></div><div content-id="b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kjems</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Leffers</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Olesen</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Garrett</div> <div class="given-names">R. A.</div></div>. </div><div class="article-title"> A unique tRNA intron in the variable loop of the extreme thermophile <em>Thermofilm pendens</em> and its possible evolutionary implications </div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1989</div>, vol. <div class="volume">264</div> (pg. <div class="fpage">17834</div>-<div class="lpage">17837</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20unique%20tRNA%20intron%20in%20the%20variable%20loop%20of%20the%20extreme%20thermophile%20Thermofilm%20pendens%20and%20its%20possible%20evolutionary%20implications&amp;author=J.%20Kjems&amp;author=H.%20Leffers&amp;author=T.%20Olesen&amp;author=R.%20A.%20Garrett&amp;publication_year=1989&amp;journal=J.%20Biol.%20Chem.&amp;volume=264&amp;pages=17834-17837" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2808354" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kjems</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Jensen</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Olesen</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Garrett</div> <div class="given-names">R. A.</div></div>. </div><div class="article-title"> Comparison of transfer RNA and ribosomal RNA intron splicing in the extreme thermophile and archaebacterium <em>Desulfurococcus mobilis</em></div>, <div class="source">Can. J. Microbiol.</div> , <div class="year">1989</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">210</div>-<div class="lpage">214</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20transfer%20RNA%20and%20ribosomal%20RNA%20intron%20splicing%20in%20the%20extreme%20thermophile%20and%20archaebacterium%20Desulfurococcus%20mobilis&amp;author=J.%20Kjems&amp;author=J.%20Jensen&amp;author=T.%20Olesen&amp;author=R.%20A.%20Garrett&amp;publication_year=1989&amp;journal=Can.%20J.%20Microbiol.&amp;volume=35&amp;pages=210-214" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/m89-033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fm89-033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fm89-033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2470485" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wich</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Leinfelder</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Bock</div> <div class="given-names">A.</div></div>. </div><div class="article-title"> Genes for stable RNA in the extreme thermophile <em>Thermoproteus tenax</em> : introns and transcription signals </div>, <div class="source">EMBO J.</div> , <div class="year">1987</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">523</div>-<div class="lpage">528</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genes%20for%20stable%20RNA%20in%20the%20extreme%20thermophile%20Thermoproteus%20tenax%20%3A%20introns%20and%20transcription%20signals&amp;author=G.%20Wich&amp;author=W.%20Leinfelder&amp;author=A.%20Bock&amp;publication_year=1987&amp;journal=EMBO%20J.&amp;volume=6&amp;pages=523-528" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15981338" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawarabayasi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hino</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Horikawa</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title"> Complete genome sequence of an aerobic hyper-thermophilic crenarchaeota, <em>Aeropyrum pernix</em> K1 </div>, <div class="source">DNA Res.</div> , <div class="year">1999</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">84</div>-<div class="lpage">76</div>) <div class="comment">147–155</div></div></div></div></div><div content-id="b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawarabayasi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hino</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Horikawa</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title"> Complete genome sequence of an aerobic thermoacidophilic crenarchaeon, <em>Sulfolobus tokodaii</em> strain7 </div>, <div class="source">DNA Res.</div> , <div class="year">2001</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">123</div>-<div class="lpage">140</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20an%20aerobic%20thermoacidophilic%20crenarchaeon%2C%20Sulfolobus%20tokodaii%20strain7&amp;author=Y.%20Kawarabayasi&amp;author=Y.%20Hino&amp;author=H.%20Horikawa&amp;publication_year=2001&amp;journal=DNA%20Res.&amp;volume=8&amp;pages=123-140" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/8.4.123" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F8.4.123" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F8.4.123"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11572479" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sako</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nomura</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Uchida</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title"><em>Aeropurum pernix</em> gen. nov., sp. nov., a novel aerobic hyper thermophilic archaeon growing at temperatures up to 100°C </div>, <div class="source">Int. J. Syst. Bacteriol.</div> , <div class="year">1996</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">1070</div>-<div class="lpage">1077</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aeropurum%20pernix%20gen.%20nov.%2C%20sp.%20nov.%2C%20a%20novel%20aerobic%20hyper%20thermophilic%20archaeon%20growing%20at%20temperatures%20up%20to%20100%C2%B0C&amp;author=Y.%20Sako&amp;author=N.%20Nomura&amp;author=A.%20Uchida&amp;publication_year=1996&amp;journal=Int.%20J.%20Syst.%20Bacteriol.&amp;volume=46&amp;pages=1070-1077" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00207713-46-4-1070" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00207713-46-4-1070" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00207713-46-4-1070"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8863437" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bult</div> <div class="given-names">C. J.</div></div>,  <div class="name"><div class="surname">White</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Olsen</div> <div class="given-names">G. J.</div></div>, et al. </div><div class="article-title"> Complete genome sequence of the methanogenic archaeon, <em>Methanococcus jannaschii</em></div>, <div class="source">Science</div> , <div class="year">1996</div>, vol. <div class="volume">273</div> (pg. <div class="fpage">1058</div>-<div class="lpage">1073</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20the%20methanogenic%20archaeon%2C%20Methanococcus%20jannaschii&amp;author=C.%20J.%20Bult&amp;author=O.%20White&amp;author=G.%20J.%20Olsen&amp;publication_year=1996&amp;journal=Science&amp;volume=273&amp;pages=1058-1073" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.273.5278.1058" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.273.5278.1058" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.273.5278.1058"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8688087" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smith</div> <div class="given-names">D. R.</div></div>,  <div class="name"><div class="surname">Doucette-Stamm</div> <div class="given-names">L. A.</div></div>,  <div class="name"><div class="surname">Deloughery</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title"> Complete genome sequence of <em>Methanobacterium thermoautotrophicum</em> deltaH: functional analysis and comparative genomics </div>, <div class="source">J. Bacteriol.</div> , <div class="year">1997</div>, vol. <div class="volume">179</div> (pg. <div class="fpage">7135</div>-<div class="lpage">7155</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20Methanobacterium%20thermoautotrophicum%20deltaH%3A%20functional%20analysis%20and%20comparative%20genomics&amp;author=D.%20R.%20Smith&amp;author=L.%20A.%20Doucette-Stamm&amp;author=C.%20Deloughery&amp;publication_year=1997&amp;journal=J.%20Bacteriol.&amp;volume=179&amp;pages=7135-7155" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9371463" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baldi</div> <div class="given-names">M. I.</div></div>,  <div class="name"><div class="surname">Mattoccia</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Bufardeci</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Fabbri</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tocchini-Valentini</div> <div class="given-names">G. P.</div></div>. </div><div class="article-title"> Participation of the intron in the reaction catalyzed by the <em>Xenopus</em> tRNA splicing endonuclease </div>, <div class="source">Science</div> , <div class="year">1992</div>, vol. <div class="volume">255</div> (pg. <div class="fpage">1404</div>-<div class="lpage">1408</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Participation%20of%20the%20intron%20in%20the%20reaction%20catalyzed%20by%20the%20Xenopus%20tRNA%20splicing%20endonuclease&amp;author=M.%20I.%20Baldi&amp;author=E.%20Mattoccia&amp;author=E.%20Bufardeci&amp;author=S.%20Fabbri&amp;author=G.%20P.%20Tocchini-Valentini&amp;publication_year=1992&amp;journal=Science&amp;volume=255&amp;pages=1404-1408" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1542788" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1542788" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1542788"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1542788" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marck</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Grosjean</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Identification of BHB splicing motifs in intron-containing tRNAs from 18 archaea: evolutionary implications</div>, <div class="source">RNA</div> , <div class="year">2003</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">1516</div>-<div class="lpage">1531</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20BHB%20splicing%20motifs%20in%20intron-containing%20tRNAs%20from%2018%20archaea%3A%20evolutionary%20implications&amp;author=C.%20Marck&amp;author=H.%20Grosjean&amp;publication_year=2003&amp;journal=RNA&amp;volume=9&amp;pages=1516-1531" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.5132503" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.5132503" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.5132503"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14624007" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nomura</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Sako</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Uchida</div> <div class="given-names">A.</div></div>. </div><div class="article-title"> Molecular characterization and postsplicing fate of three introns within the single rRNA operon of the hyperthermophilic archaeon <em>Aeropyrum pernix</em> K1 </div>, <div class="source">J. Bacteriol.</div> , <div class="year">1998</div>, vol. <div class="volume">180</div> (pg. <div class="fpage">3635</div>-<div class="lpage">3643</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20characterization%20and%20postsplicing%20fate%20of%20three%20introns%20within%20the%20single%20rRNA%20operon%20of%20the%20hyperthermophilic%20archaeon%20Aeropyrum%20pernix%20K1&amp;author=N.%20Nomura&amp;author=Y.%20Sako&amp;author=A.%20Uchida&amp;publication_year=1998&amp;journal=J.%20Bacteriol.&amp;volume=180&amp;pages=3635-3643" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9658008" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Watanabe</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yokobori</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Inaba</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Introns in protein-coding genes in Archaea</div>, <div class="source">FEBS Lett.</div> , <div class="year">2002</div>, vol. <div class="volume">510</div> (pg. <div class="fpage">27</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Introns%20in%20protein-coding%20genes%20in%20Archaea&amp;author=Y.%20Watanabe&amp;author=S.%20Yokobori&amp;author=T.%20Inaba&amp;publication_year=2002&amp;journal=FEBS%20Lett.&amp;volume=510&amp;pages=27-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0014-5793(01)03219-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0014-5793(01)03219-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0014-5793(01)03219-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11755525" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kleman-Leyer</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Armbruster</div> <div class="given-names">D. W.</div></div>,  <div class="name"><div class="surname">Daniels</div> <div class="given-names">C. J.</div></div>. </div><div class="article-title"> Properties of <em>H. volcanii</em> tRNA intron endonuclease reveal a relationship between the archaeal and eucaryal tRNA intron processing systems </div>, <div class="source">Cell</div> , <div class="year">1997</div>, vol. <div class="volume">89</div> (pg. <div class="fpage">839</div>-<div class="lpage">848</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Properties%20of%20H.%20volcanii%20tRNA%20intron%20endonuclease%20reveal%20a%20relationship%20between%20the%20archaeal%20and%20eucaryal%20tRNA%20intron%20processing%20systems&amp;author=K.%20Kleman-Leyer&amp;author=D.%20W.%20Armbruster&amp;author=C.%20J.%20Daniels&amp;publication_year=1997&amp;journal=Cell&amp;volume=89&amp;pages=839-848" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(00)80269-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(00)80269-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(00)80269-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9200602" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Phizicky</div> <div class="given-names">E. M.</div></div>,  <div class="name"><div class="surname">Schwartz</div> <div class="given-names">R. C.</div></div>,  <div class="name"><div class="surname">Abelson</div> <div class="given-names">J.</div></div>. </div><div class="article-title"><em>Saccharomyces cerevisiae</em> tRNA ligase. Purification of the protein and isolation of the structural gene </div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1986</div>, vol. <div class="volume">261</div> (pg. <div class="fpage">2978</div>-<div class="lpage">2986</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Saccharomyces%20cerevisiae%20tRNA%20ligase.%20Purification%20of%20the%20protein%20and%20isolation%20of%20the%20structural%20gene&amp;author=E.%20M.%20Phizicky&amp;author=R.%20C.%20Schwartz&amp;author=J.%20Abelson&amp;publication_year=1986&amp;journal=J.%20Biol.%20Chem.&amp;volume=261&amp;pages=2978-2986" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3512545" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lowe</div> <div class="given-names">T. M.</div></div>,  <div class="name"><div class="surname">Eddy</div> <div class="given-names">S. R.</div></div>. </div><div class="article-title">tRNAscan-SE: A program for improved detection of transfer RNA genes in genomic sequence</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">955</div>-<div class="lpage">964</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20A%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence&amp;author=T.%20M.%20Lowe&amp;author=S.%20R.%20Eddy&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=955-964" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9023104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">S. J.</div></div>,  <div class="name"><div class="surname">Schaak</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Cooley</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Burke</div> <div class="given-names">D. J.</div></div>,  <div class="name"><div class="surname">Söll</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Structure and transcription of eukaryotic tRNA genes</div>, <div class="source">CRC Crit. Biochem.</div> , <div class="year">1985</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">107</div>-<div class="lpage">144</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20and%20transcription%20of%20eukaryotic%20tRNA%20genes&amp;author=S.%20J.%20Sharp&amp;author=J.%20Schaak&amp;author=L.%20Cooley&amp;author=D.%20J.%20Burke&amp;author=D.%20S%C3%B6ll&amp;publication_year=1985&amp;journal=CRC%20Crit.%20Biochem.&amp;volume=19&amp;pages=107-144" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3109/10409238509082541" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3109%2F10409238509082541" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3109%2F10409238509082541"> </span></div> </div></div></div></div></div><div content-id="b27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nayokova</div> <div class="given-names">T. M.</div></div>,  <div class="name"><div class="surname">Kondrakhin</div> <div class="given-names">Y. V.</div></div>,  <div class="name"><div class="surname">Rogozin</div> <div class="given-names">I. B.</div></div>,  <div class="name"><div class="surname">Voevoda</div> <div class="given-names">M. I.</div></div>,  <div class="name"><div class="surname">Yudin</div> <div class="given-names">N. S.</div></div>,  <div class="name"><div class="surname">Romaschenko</div> <div class="given-names">A. G.</div></div>. </div><div class="article-title">Concerted changes in the nucleotide sequences of the intragenic promoter regions of eukaryotic genes for tRNAs of all specificities</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">2003</div>, vol. <div class="volume">57</div> (pg. <div class="fpage">520</div>-<div class="lpage">532</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Concerted%20changes%20in%20the%20nucleotide%20sequences%20of%20the%20intragenic%20promoter%20regions%20of%20eukaryotic%20genes%20for%20tRNAs%20of%20all%20specificities&amp;author=T.%20M.%20Nayokova&amp;author=Y.%20V.%20Kondrakhin&amp;author=I.%20B.%20Rogozin&amp;author=M.%20I.%20Voevoda&amp;author=N.%20S.%20Yudin&amp;author=A.%20G.%20Romaschenko&amp;publication_year=2003&amp;journal=J.%20Mol.%20Evol.&amp;volume=57&amp;pages=520-532" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-003-2504-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-003-2504-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-003-2504-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14738310" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2006. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=344476&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">178</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsi023"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="344476"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f12%2f6%2f403%2f344476">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F12%2F6%2F403%2F344476" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/344476'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value=" Characterization of a Whole Set of tRNA Molecules in an Aerobic Hyper-thermophilic Crenarchaeon, Aeropyrum pernix K1 " />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsi023");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f12%2f6%2f403%2f344476">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

