

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/3/285/393995","@type":"ScholarlyArticle","name":"Assessing Diversity of DNA Structure-Related Sequence Features in Prokaryotic Genomes","datePublished":"2014-01-09","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2014-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst057","keywords":["sequence patterns","Z-DNA","DNA curvature","sequence repeats","palindromes"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/3/285/393995","author":[{"name":"Huang, Yongjie","affiliation":"1  Institute of Bioinformatics ,  University of Georgia ,  Athens, GA 30602 ,  USA","@type":"Person"},{"name":"Mrázek, Jan","affiliation":"1  Institute of Bioinformatics ,  University of Georgia ,  Athens, GA 30602 ,  USA    2  Department of Microbiology ,  University of Georgia ,  Athens, GA 30602 ,  USA","@type":"Person"}],"description":"Abstract.  Prokaryotic genomes are diverse in terms of their nucleotide and oligonucleotide composition as well as presence of various sequence features that ca","pageStart":"285","pageEnd":"297","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/3/1/m_cover.gif?Expires=2147483647&Signature=eD74u2chqR7m7Q4xVEpu-B2CnS8a5UZybuUBj9IiQcs7vcn-sLThVTH2YkcAPlEiODArsSqGH041ZPg5RakmGwkyHxqVC4LdBdXxFui4WCeLWHefwauec7mA5UKsie6Bk-ZRp7yPg10FfeUTBrjN7buCiodIP00VlbrJgVNTJFuLyaUUxPgJ3yahCwwX4TAcK2CRCX-3Y~LqokrGvJCW5URLAAo4GxWdmNUAJqBhdMREkIh~McyOhMfYKJNLDTxW9vnvEbKMxNW5048gQKGgF7kMxJ3~NsfswZCCrwYr2quaHFHHORZM0v4Eh4XRYgKpFeDHa6jnY4OeJPxjcOPrMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Assessing Diversity of DNA Structure-Related Sequence Features in Prokaryotic Genomes","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/3/1/m_cover.gif?Expires=2147483647&Signature=eD74u2chqR7m7Q4xVEpu-B2CnS8a5UZybuUBj9IiQcs7vcn-sLThVTH2YkcAPlEiODArsSqGH041ZPg5RakmGwkyHxqVC4LdBdXxFui4WCeLWHefwauec7mA5UKsie6Bk-ZRp7yPg10FfeUTBrjN7buCiodIP00VlbrJgVNTJFuLyaUUxPgJ3yahCwwX4TAcK2CRCX-3Y~LqokrGvJCW5URLAAo4GxWdmNUAJqBhdMREkIh~McyOhMfYKJNLDTxW9vnvEbKMxNW5048gQKGgF7kMxJ3~NsfswZCCrwYr2quaHFHHORZM0v4Eh4XRYgKpFeDHa6jnY4OeJPxjcOPrMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Assessing Diversity of DNA Structure-Related Sequence Features in Prokaryotic Genomes" />
    <meta property="og:description" content="Abstract.  Prokaryotic genomes are diverse in terms of their nucleotide and oligonucleotide composition as well as presence of various sequence features that ca" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/3/285/393995" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2014-01-09" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Huang, Yongjie" /><meta name="citation_author_institution" content="Institute of Bioinformatics, University of Georgia, Athens, GA 30602, USA" /><meta name="citation_author" content="Mrázek, Jan" /><meta name="citation_author_institution" content="Institute of Bioinformatics, University of Georgia, Athens, GA 30602, USA" /><meta name="citation_author_institution" content="Department of Microbiology, University of Georgia, Athens, GA 30602, USA" /><meta name="citation_title" content="Assessing Diversity of DNA Structure-Related Sequence Features in Prokaryotic Genomes" /><meta name="citation_firstpage" content="285" /><meta name="citation_lastpage" content="297" /><meta name="citation_doi" content="10.1093/dnares/dst057" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2014/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Bias of purine stretches in sequenced chromosomes; Comput. Chem.;  citation_year=2002;  citation_volume=26;  citation_pages=531-41; " /><meta name="citation_reference" content="citation_publisher=Academic Press, San Diego; DNA Structure and Function;  citation_year=1994; " /><meta name="citation_reference" content="citation_title=Stretches of alternating pyrimidine/purines and purines are respectively linked with pathogenicity and growth temperature in prokaryotes; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=346" /><meta name="citation_reference" content="citation_title=Genome-wide prediction of G4 DNA as regulatory motifs: role in Escherichia coli global regulation; Genome Res.;  citation_year=2006;  citation_volume=16;  citation_pages=644-55; " /><meta name="citation_reference" content="citation_title=Involvement of DNA curvature in intergenic regions of prokaryotes; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=2316-27; " /><meta name="citation_reference" content="citation_title=Comparative analysis of sequence periodicity among prokaryotic genomes points to differences in nucleoid structure and a relationship to gene expression; J. Bacteriol.;  citation_year=2010;  citation_volume=192;  citation_pages=3763-72; " /><meta name="citation_reference" content="citation_title=Simple sequence repeats in prokaryotic genomes; Proc. Natl Acad. Sci. USA;  citation_year=2007;  citation_volume=104;  citation_pages=8472-7; " /><meta name="citation_reference" content="citation_title=The bacterial nucleoid: a highly organized and dynamic structure; J. Cell Biochem.;  citation_year=2005;  citation_volume=96;  citation_pages=506-21; " /><meta name="citation_reference" content="citation_title=Bacterial nucleoid-associated proteins, nucleoid structure and gene expression; Nat. Rev. Microbiol.;  citation_year=2010;  citation_volume=8;  citation_pages=185-95; " /><meta name="citation_reference" content="citation_title=Adaptive evolution of highly mutable loci in pathogenic bacteria; Curr. Biol.;  citation_year=1994;  citation_volume=4;  citation_pages=24-33; " /><meta name="citation_reference" content="citation_title=Phase and antigenic variation in bacteria; citation_text=table of contents; Clin. Microbiol. Rev.;  citation_year=2004;  citation_volume=17;  citation_pages=581-611; " /><meta name="citation_reference" content="citation_title=Cruciform structures are a common DNA feature important for regulating biological processes; BMC Mol. Biol.;  citation_year=2011;  citation_volume=12;  citation_pages=33" /><meta name="citation_reference" content="citation_title=Slipped-strand mispairing: a major mechanism for DNA sequence evolution; Mol. Biol. Evol.;  citation_year=1987;  citation_volume=4;  citation_pages=203-21; " /><meta name="citation_reference" content="citation_title=Cis and trans-acting effects on a mutational hotspot involving a replication template switch; J. Mol. Biol.;  citation_year=2006;  citation_volume=356;  citation_pages=300-11; " /><meta name="citation_reference" content="citation_title=Encoded errors: mutations and rearrangements mediated by misalignment at repetitive DNA sequences; Mol. Microbiol.;  citation_year=2004;  citation_volume=52;  citation_pages=1243-53; " /><meta name="citation_reference" content="citation_title=Translocation and gross deletion breakpoints in human inherited disease and cancer II: potential involvement of repetitive sequence elements in secondary structure formation between DNA ends; Hum. Mutat.;  citation_year=2003;  citation_volume=22;  citation_pages=245-51; " /><meta name="citation_reference" content="citation_title=Quadruplex DNA: sequence, topology and structure; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=5402-15; " /><meta name="citation_reference" content="citation_title=Intramolecular and intermolecular guanine quadruplexes of DNA in aqueous salt and ethanol solutions; Biopolymers;  citation_year=2007;  citation_volume=86;  citation_pages=1-10; " /><meta name="citation_reference" content="citation_title=Biological aspects of DNA/RNA quadruplexes; Biopolymers;  citation_year=2000;  citation_volume=56;  citation_pages=209-27; " /><meta name="citation_reference" content="citation_title=The fragile X mental retardation protein binds specifically to its mRNA via a purine quartet motif; EMBO J.;  citation_year=2001;  citation_volume=20;  citation_pages=4803-13; " /><meta name="citation_reference" content="citation_title=Base-tetrad swapping results in dimerization of RNA quadruplexes: implications for formation of the i-motif RNA octaplex; Proc. Natl Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=3130-4; " /><meta name="citation_reference" content="citation_title=Functional and dysfunctional roles of quadruplex DNA in cells; Chem. Biol.;  citation_year=2001;  citation_volume=8;  citation_pages=221-30; " /><meta name="citation_reference" content="citation_title=Evidence for interstrand quadruplex formation in the dimerization of human immunodeficiency virus 1 genomic RNA; Proc. Natl Acad. Sci. USA;  citation_year=1993;  citation_volume=90;  citation_pages=3393-7; " /><meta name="citation_reference" content="citation_title=Formation of intramolecular triplex in homopurine–homopyrimidine mirror repeats with point substitutions; Nucleic Acids Res.;  citation_year=1990;  citation_volume=18;  citation_pages=6621-4; " /><meta name="citation_reference" content="citation_title=A polypyrimidine/polypurine tract within the Hmga2 minimal promoter: a common feature of many growth-related genes; Biochemistry;  citation_year=2002;  citation_volume=41;  citation_pages=1229-40; " /><meta name="citation_reference" content="citation_title=Do natural DNA triple-helical structures occur and function in vivo?; Cell Mol. Life Sci.;  citation_year=2003;  citation_volume=60;  citation_pages=862-70; " /><meta name="citation_reference" content="citation_title=Stabilization and detection of natural left-handed Z-DNA; J. Biomol. Struct. Dyn.;  citation_year=1983;  citation_volume=1;  citation_pages=1-19; " /><meta name="citation_reference" content="citation_title=Unusual DNA structures, chromatin and transcription; Bioessays;  citation_year=1994;  citation_volume=16;  citation_pages=59-68; " /><meta name="citation_reference" content="citation_title=Left-handed Z-DNA: structure and function; Genetica;  citation_year=1999;  citation_volume=106;  citation_pages=37-47; " /><meta name="citation_reference" content="citation_title=Timeline: Z-DNA: the long road to biological function; Nat. Rev. Genet.;  citation_year=2003;  citation_volume=4;  citation_pages=566-72; " /><meta name="citation_reference" content="citation_title=Z-DNA, an active element in the genome; Front Biosci.;  citation_year=2007;  citation_volume=12;  citation_pages=4424-38; " /><meta name="citation_reference" content="citation_title=DNA triple helices: biological consequences and therapeutic potential; Biochimie;  citation_year=2008;  citation_volume=90;  citation_pages=1117-30; " /><meta name="citation_reference" content="citation_title=A-tract clusters may facilitate DNA packaging in bacterial nucleoid; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=3907-18; " /><meta name="citation_reference" content="citation_title=Sequence periodicity in complete genomes of archaea suggests positive supercoiling; J. Biomol. Struct. Dyn.;  citation_year=1998;  citation_volume=16;  citation_pages=341-5; " /><meta name="citation_reference" content="citation_title=The Genomes OnLine Database (GOLD) v.4: status of genomic and metagenomic projects and their associated metadata; Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=D571-9; " /><meta name="citation_reference" content="citation_title=Pattern locator: a new tool for finding local sequence patterns in genomic DNA sequences; Bioinformatics;  citation_year=2006;  citation_volume=22;  citation_pages=3099-100; " /><meta name="citation_reference" content="citation_title=Analysis of distribution indicates diverse functions of simple sequence repeats in Mycoplasma genomes; Mol. Biol. Evol.;  citation_year=2006;  citation_volume=23;  citation_pages=1370-85; " /><meta name="citation_reference" content="citation_title=Stability and kinetics of G-quadruplex structures; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=5482-515; " /><meta name="citation_reference" content="citation_title=Guanine tetraplex topology of human telomere DNA is governed by the number of (TTAGGG) repeats; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=5851-60; " /><meta name="citation_reference" content="citation_title=AIMIE: a web-based environment for detection and interpretation of significant sequence motifs in prokaryotic genomes; Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=1041-8; " /><meta name="citation_reference" content="citation_title=Methyl-directed repair of mismatched small heterologous sequences in cell extracts from Escherichia coli; J. Biol. Chem.;  citation_year=1997;  citation_volume=272;  citation_pages=22714-20; " /><meta name="citation_reference" content="citation_title=Repair of DNA heteroduplexes containing small heterologous sequences in Escherichia coli; Proc. Natl Acad. Sci. USA;  citation_year=1992;  citation_volume=89;  citation_pages=1730-4; " /><meta name="citation_reference" content="citation_title=The excess of small inverted repeats in prokaryotes; J. Mol. Evol.;  citation_year=2008;  citation_volume=67;  citation_pages=291-300; " /><meta name="citation_reference" content="citation_title=Widespread selection for local RNA secondary structure in coding regions of bacterial genes; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=2042-51; " /><meta name="citation_reference" content="citation_title=Adaptive role of increased frequency of polypurine tracts in mRNA sequences of thermophilic prokaryotes; Proc. Natl Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=2951-6; " /><meta name="citation_reference" content="citation_title=Thermal adaptation analyzed by comparison of protein sequences from mesophilic and extremely thermophilic Methanococcus species; Proc. Natl Acad. Sci. USA;  citation_year=1999;  citation_volume=96;  citation_pages=3578-83; " /><meta name="citation_reference" content="citation_title=Genomic correlates of hyperthermostability, an update; J. Biol. Chem.;  citation_year=2003;  citation_volume=278;  citation_pages=17198-202; " /><meta name="citation_reference" content="citation_title=Archaeal adaptation to higher temperatures revealed by genomic sequence of Thermoplasma volcanium; Proc. Natl Acad. Sci. USA;  citation_year=2000;  citation_volume=97;  citation_pages=14257-62; " /><meta name="citation_reference" content="citation_title=Ecologic genomics of DNA: upstream bending in prokaryotic promoters; Genome Res.;  citation_year=2000;  citation_volume=10;  citation_pages=1185-93; " /><meta name="citation_reference" content="citation_title=Genome analysis of Escherichia coli promoter sequences evidences that DNA static curvature plays a more important role in gene transcription than has previously been anticipated; Genomics;  citation_year=2006;  citation_volume=87;  citation_pages=329-37; " /><meta name="citation_reference" content="citation_title=Conservation of DNA curvature signals in regulatory regions of prokaryotic genes; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=6770-7; " /><meta name="citation_reference" content="citation_title=Environmental influences on DNA curvature; J. Biomol. Struct. Dyn.;  citation_year=1999;  citation_volume=16;  citation_pages=811-23; " /><meta name="citation_reference" content="citation_title=Avoiding dangerous missense: thermophiles display especially low mutation rates; PLoS Genet.;  citation_year=2009;  citation_volume=5;  citation_pages=e1000520" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/3/285/1108818/dst057.pdf" /><meta name="description" content="Abstract.  Prokaryotic genomes are diverse in terms of their nucleotide and oligonucleotide composition as well as presence of various sequence features that ca" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/3/285/393995" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/3/285/393995" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/3/285/393995");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst057';
    </script>
<title>
	Assessing Diversity of DNA Structure-Related Sequence Features in Prokaryotic Genomes | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./393995" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BepqPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIdWFuZywgWW9uZ2ppZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBCaW9pbmZvcm1hdGljcywgVW5pdmVyc2l0eSBvZiBHZW9yZ2lhLCBBdGhlbnMsIEdBIDMwNjAyLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNcsOhemVrLCBKYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgQmlvaW5mb3JtYXRpY3MsIFVuaXZlcnNpdHkgb2YgR2VvcmdpYSwgQXRoZW5zLCBHQSAzMDYwMiwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNaWNyb2Jpb2xvZ3ksIFVuaXZlcnNpdHkgb2YgR2VvcmdpYSwgQXRoZW5zLCBHQSAzMDYwMiwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJBc3Nlc3NpbmcgRGl2ZXJzaXR5IG9mIEROQSBTdHJ1Y3R1cmUtUmVsYXRlZCBTZXF1ZW5jZSBGZWF0dXJlcyBpbiBQcm9rYXJ5b3RpYyBHZW5vbWVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjg1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyOTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3QwNTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNC8wNi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpYXMgb2YgcHVyaW5lIHN0cmV0Y2hlcyBpbiBzZXF1ZW5jZWQgY2hyb21vc29tZXM7IENvbXB1dC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz01MzEtNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1BY2FkZW1pYyBQcmVzcywgU2FuIERpZWdvOyBETkEgU3RydWN0dXJlIGFuZCBGdW5jdGlvbjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RyZXRjaGVzIG9mIGFsdGVybmF0aW5nIHB5cmltaWRpbmUvcHVyaW5lcyBhbmQgcHVyaW5lcyBhcmUgcmVzcGVjdGl2ZWx5IGxpbmtlZCB3aXRoIHBhdGhvZ2VuaWNpdHkgYW5kIGdyb3d0aCB0ZW1wZXJhdHVyZSBpbiBwcm9rYXJ5b3RlczsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MzQ2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgcHJlZGljdGlvbiBvZiBHNCBETkEgYXMgcmVndWxhdG9yeSBtb3RpZnM6IHJvbGUgaW4gRXNjaGVyaWNoaWEgY29saSBnbG9iYWwgcmVndWxhdGlvbjsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz02NDQtNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludm9sdmVtZW50IG9mIEROQSBjdXJ2YXR1cmUgaW4gaW50ZXJnZW5pYyByZWdpb25zIG9mIHByb2thcnlvdGVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz0yMzE2LTI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBhbmFseXNpcyBvZiBzZXF1ZW5jZSBwZXJpb2RpY2l0eSBhbW9uZyBwcm9rYXJ5b3RpYyBnZW5vbWVzIHBvaW50cyB0byBkaWZmZXJlbmNlcyBpbiBudWNsZW9pZCBzdHJ1Y3R1cmUgYW5kIGEgcmVsYXRpb25zaGlwIHRvIGdlbmUgZXhwcmVzc2lvbjsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xOTI7ICBjaXRhdGlvbl9wYWdlcz0zNzYzLTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW1wbGUgc2VxdWVuY2UgcmVwZWF0cyBpbiBwcm9rYXJ5b3RpYyBnZW5vbWVzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEwNDsgIGNpdGF0aW9uX3BhZ2VzPTg0NzItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGJhY3RlcmlhbCBudWNsZW9pZDogYSBoaWdobHkgb3JnYW5pemVkIGFuZCBkeW5hbWljIHN0cnVjdHVyZTsgSi4gQ2VsbCBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT05NjsgIGNpdGF0aW9uX3BhZ2VzPTUwNi0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFjdGVyaWFsIG51Y2xlb2lkLWFzc29jaWF0ZWQgcHJvdGVpbnMsIG51Y2xlb2lkIHN0cnVjdHVyZSBhbmQgZ2VuZSBleHByZXNzaW9uOyBOYXQuIFJldi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MTg1LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZGFwdGl2ZSBldm9sdXRpb24gb2YgaGlnaGx5IG11dGFibGUgbG9jaSBpbiBwYXRob2dlbmljIGJhY3RlcmlhOyBDdXJyLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MjQtMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoYXNlIGFuZCBhbnRpZ2VuaWMgdmFyaWF0aW9uIGluIGJhY3RlcmlhOyBjaXRhdGlvbl90ZXh0PXRhYmxlIG9mIGNvbnRlbnRzOyBDbGluLiBNaWNyb2Jpb2wuIFJldi47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz01ODEtNjExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DcnVjaWZvcm0gc3RydWN0dXJlcyBhcmUgYSBjb21tb24gRE5BIGZlYXR1cmUgaW1wb3J0YW50IGZvciByZWd1bGF0aW5nIGJpb2xvZ2ljYWwgcHJvY2Vzc2VzOyBCTUMgTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTMzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2xpcHBlZC1zdHJhbmQgbWlzcGFpcmluZzogYSBtYWpvciBtZWNoYW5pc20gZm9yIEROQSBzZXF1ZW5jZSBldm9sdXRpb247IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTIwMy0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2lzIGFuZCB0cmFucy1hY3RpbmcgZWZmZWN0cyBvbiBhIG11dGF0aW9uYWwgaG90c3BvdCBpbnZvbHZpbmcgYSByZXBsaWNhdGlvbiB0ZW1wbGF0ZSBzd2l0Y2g7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzU2OyAgY2l0YXRpb25fcGFnZXM9MzAwLTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbmNvZGVkIGVycm9yczogbXV0YXRpb25zIGFuZCByZWFycmFuZ2VtZW50cyBtZWRpYXRlZCBieSBtaXNhbGlnbm1lbnQgYXQgcmVwZXRpdGl2ZSBETkEgc2VxdWVuY2VzOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTI7ICBjaXRhdGlvbl9wYWdlcz0xMjQzLTUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2xvY2F0aW9uIGFuZCBncm9zcyBkZWxldGlvbiBicmVha3BvaW50cyBpbiBodW1hbiBpbmhlcml0ZWQgZGlzZWFzZSBhbmQgY2FuY2VyIElJOiBwb3RlbnRpYWwgaW52b2x2ZW1lbnQgb2YgcmVwZXRpdGl2ZSBzZXF1ZW5jZSBlbGVtZW50cyBpbiBzZWNvbmRhcnkgc3RydWN0dXJlIGZvcm1hdGlvbiBiZXR3ZWVuIEROQSBlbmRzOyBIdW0uIE11dGF0LjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTI0NS01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhZHJ1cGxleCBETkE6IHNlcXVlbmNlLCB0b3BvbG9neSBhbmQgc3RydWN0dXJlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz01NDAyLTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRyYW1vbGVjdWxhciBhbmQgaW50ZXJtb2xlY3VsYXIgZ3VhbmluZSBxdWFkcnVwbGV4ZXMgb2YgRE5BIGluIGFxdWVvdXMgc2FsdCBhbmQgZXRoYW5vbCBzb2x1dGlvbnM7IEJpb3BvbHltZXJzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg2OyAgY2l0YXRpb25fcGFnZXM9MS0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlvbG9naWNhbCBhc3BlY3RzIG9mIEROQS9STkEgcXVhZHJ1cGxleGVzOyBCaW9wb2x5bWVyczsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT01NjsgIGNpdGF0aW9uX3BhZ2VzPTIwOS0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGZyYWdpbGUgWCBtZW50YWwgcmV0YXJkYXRpb24gcHJvdGVpbiBiaW5kcyBzcGVjaWZpY2FsbHkgdG8gaXRzIG1STkEgdmlhIGEgcHVyaW5lIHF1YXJ0ZXQgbW90aWY7IEVNQk8gSi47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz00ODAzLTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXNlLXRldHJhZCBzd2FwcGluZyByZXN1bHRzIGluIGRpbWVyaXphdGlvbiBvZiBSTkEgcXVhZHJ1cGxleGVzOiBpbXBsaWNhdGlvbnMgZm9yIGZvcm1hdGlvbiBvZiB0aGUgaS1tb3RpZiBSTkEgb2N0YXBsZXg7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9MzEzMC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5jdGlvbmFsIGFuZCBkeXNmdW5jdGlvbmFsIHJvbGVzIG9mIHF1YWRydXBsZXggRE5BIGluIGNlbGxzOyBDaGVtLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MjIxLTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmlkZW5jZSBmb3IgaW50ZXJzdHJhbmQgcXVhZHJ1cGxleCBmb3JtYXRpb24gaW4gdGhlIGRpbWVyaXphdGlvbiBvZiBodW1hbiBpbW11bm9kZWZpY2llbmN5IHZpcnVzIDEgZ2Vub21pYyBSTkE7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9OTA7ICBjaXRhdGlvbl9wYWdlcz0zMzkzLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZvcm1hdGlvbiBvZiBpbnRyYW1vbGVjdWxhciB0cmlwbGV4IGluIGhvbW9wdXJpbmXigJNob21vcHlyaW1pZGluZSBtaXJyb3IgcmVwZWF0cyB3aXRoIHBvaW50IHN1YnN0aXR1dGlvbnM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTY2MjEtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwb2x5cHlyaW1pZGluZS9wb2x5cHVyaW5lIHRyYWN0IHdpdGhpbiB0aGUgSG1nYTIgbWluaW1hbCBwcm9tb3RlcjogYSBjb21tb24gZmVhdHVyZSBvZiBtYW55IGdyb3d0aC1yZWxhdGVkIGdlbmVzOyBCaW9jaGVtaXN0cnk7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz0xMjI5LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EbyBuYXR1cmFsIEROQSB0cmlwbGUtaGVsaWNhbCBzdHJ1Y3R1cmVzIG9jY3VyIGFuZCBmdW5jdGlvbiBpbiB2aXZvPzsgQ2VsbCBNb2wuIExpZmUgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT02MDsgIGNpdGF0aW9uX3BhZ2VzPTg2Mi03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhYmlsaXphdGlvbiBhbmQgZGV0ZWN0aW9uIG9mIG5hdHVyYWwgbGVmdC1oYW5kZWQgWi1ETkE7IEouIEJpb21vbC4gU3RydWN0LiBEeW4uOyAgY2l0YXRpb25feWVhcj0xOTgzOyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0xLTE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbnVzdWFsIEROQSBzdHJ1Y3R1cmVzLCBjaHJvbWF0aW4gYW5kIHRyYW5zY3JpcHRpb247IEJpb2Vzc2F5czsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTU5LTY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MZWZ0LWhhbmRlZCBaLUROQTogc3RydWN0dXJlIGFuZCBmdW5jdGlvbjsgR2VuZXRpY2E7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTA2OyAgY2l0YXRpb25fcGFnZXM9MzctNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRpbWVsaW5lOiBaLUROQTogdGhlIGxvbmcgcm9hZCB0byBiaW9sb2dpY2FsIGZ1bmN0aW9uOyBOYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz01NjYtNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVotRE5BLCBhbiBhY3RpdmUgZWxlbWVudCBpbiB0aGUgZ2Vub21lOyBGcm9udCBCaW9zY2kuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NDQyNC0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIHRyaXBsZSBoZWxpY2VzOiBiaW9sb2dpY2FsIGNvbnNlcXVlbmNlcyBhbmQgdGhlcmFwZXV0aWMgcG90ZW50aWFsOyBCaW9jaGltaWU7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTA7ICBjaXRhdGlvbl9wYWdlcz0xMTE3LTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BLXRyYWN0IGNsdXN0ZXJzIG1heSBmYWNpbGl0YXRlIEROQSBwYWNrYWdpbmcgaW4gYmFjdGVyaWFsIG51Y2xlb2lkOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz0zOTA3LTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZSBwZXJpb2RpY2l0eSBpbiBjb21wbGV0ZSBnZW5vbWVzIG9mIGFyY2hhZWEgc3VnZ2VzdHMgcG9zaXRpdmUgc3VwZXJjb2lsaW5nOyBKLiBCaW9tb2wuIFN0cnVjdC4gRHluLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTM0MS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgR2Vub21lcyBPbkxpbmUgRGF0YWJhc2UgKEdPTEQpIHYuNDogc3RhdHVzIG9mIGdlbm9taWMgYW5kIG1ldGFnZW5vbWljIHByb2plY3RzIGFuZCB0aGVpciBhc3NvY2lhdGVkIG1ldGFkYXRhOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz1ENTcxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhdHRlcm4gbG9jYXRvcjogYSBuZXcgdG9vbCBmb3IgZmluZGluZyBsb2NhbCBzZXF1ZW5jZSBwYXR0ZXJucyBpbiBnZW5vbWljIEROQSBzZXF1ZW5jZXM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MzA5OS0xMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIGRpc3RyaWJ1dGlvbiBpbmRpY2F0ZXMgZGl2ZXJzZSBmdW5jdGlvbnMgb2Ygc2ltcGxlIHNlcXVlbmNlIHJlcGVhdHMgaW4gTXljb3BsYXNtYSBnZW5vbWVzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MTM3MC04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhYmlsaXR5IGFuZCBraW5ldGljcyBvZiBHLXF1YWRydXBsZXggc3RydWN0dXJlczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9NTQ4Mi01MTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUd1YW5pbmUgdGV0cmFwbGV4IHRvcG9sb2d5IG9mIGh1bWFuIHRlbG9tZXJlIEROQSBpcyBnb3Zlcm5lZCBieSB0aGUgbnVtYmVyIG9mIChUVEFHR0cpIHJlcGVhdHM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTU4NTEtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFJTUlFOiBhIHdlYi1iYXNlZCBlbnZpcm9ubWVudCBmb3IgZGV0ZWN0aW9uIGFuZCBpbnRlcnByZXRhdGlvbiBvZiBzaWduaWZpY2FudCBzZXF1ZW5jZSBtb3RpZnMgaW4gcHJva2FyeW90aWMgZ2Vub21lczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xMDQxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGh5bC1kaXJlY3RlZCByZXBhaXIgb2YgbWlzbWF0Y2hlZCBzbWFsbCBoZXRlcm9sb2dvdXMgc2VxdWVuY2VzIGluIGNlbGwgZXh0cmFjdHMgZnJvbSBFc2NoZXJpY2hpYSBjb2xpOyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNzI7ICBjaXRhdGlvbl9wYWdlcz0yMjcxNC0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVwYWlyIG9mIEROQSBoZXRlcm9kdXBsZXhlcyBjb250YWluaW5nIHNtYWxsIGhldGVyb2xvZ291cyBzZXF1ZW5jZXMgaW4gRXNjaGVyaWNoaWEgY29saTsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT04OTsgIGNpdGF0aW9uX3BhZ2VzPTE3MzAtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGV4Y2VzcyBvZiBzbWFsbCBpbnZlcnRlZCByZXBlYXRzIGluIHByb2thcnlvdGVzOyBKLiBNb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTY3OyAgY2l0YXRpb25fcGFnZXM9MjkxLTMwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2lkZXNwcmVhZCBzZWxlY3Rpb24gZm9yIGxvY2FsIFJOQSBzZWNvbmRhcnkgc3RydWN0dXJlIGluIGNvZGluZyByZWdpb25zIG9mIGJhY3RlcmlhbCBnZW5lczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0yMDQyLTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZGFwdGl2ZSByb2xlIG9mIGluY3JlYXNlZCBmcmVxdWVuY3kgb2YgcG9seXB1cmluZSB0cmFjdHMgaW4gbVJOQSBzZXF1ZW5jZXMgb2YgdGhlcm1vcGhpbGljIHByb2thcnlvdGVzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTI5NTEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlcm1hbCBhZGFwdGF0aW9uIGFuYWx5emVkIGJ5IGNvbXBhcmlzb24gb2YgcHJvdGVpbiBzZXF1ZW5jZXMgZnJvbSBtZXNvcGhpbGljIGFuZCBleHRyZW1lbHkgdGhlcm1vcGhpbGljIE1ldGhhbm9jb2NjdXMgc3BlY2llczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT05NjsgIGNpdGF0aW9uX3BhZ2VzPTM1NzgtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgY29ycmVsYXRlcyBvZiBoeXBlcnRoZXJtb3N0YWJpbGl0eSwgYW4gdXBkYXRlOyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yNzg7ICBjaXRhdGlvbl9wYWdlcz0xNzE5OC0yMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFyY2hhZWFsIGFkYXB0YXRpb24gdG8gaGlnaGVyIHRlbXBlcmF0dXJlcyByZXZlYWxlZCBieSBnZW5vbWljIHNlcXVlbmNlIG9mIFRoZXJtb3BsYXNtYSB2b2xjYW5pdW07IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9OTc7ICBjaXRhdGlvbl9wYWdlcz0xNDI1Ny02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWNvbG9naWMgZ2Vub21pY3Mgb2YgRE5BOiB1cHN0cmVhbSBiZW5kaW5nIGluIHByb2thcnlvdGljIHByb21vdGVyczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xMTg1LTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgYW5hbHlzaXMgb2YgRXNjaGVyaWNoaWEgY29saSBwcm9tb3RlciBzZXF1ZW5jZXMgZXZpZGVuY2VzIHRoYXQgRE5BIHN0YXRpYyBjdXJ2YXR1cmUgcGxheXMgYSBtb3JlIGltcG9ydGFudCByb2xlIGluIGdlbmUgdHJhbnNjcmlwdGlvbiB0aGFuIGhhcyBwcmV2aW91c2x5IGJlZW4gYW50aWNpcGF0ZWQ7IEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTg3OyAgY2l0YXRpb25fcGFnZXM9MzI5LTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zZXJ2YXRpb24gb2YgRE5BIGN1cnZhdHVyZSBzaWduYWxzIGluIHJlZ3VsYXRvcnkgcmVnaW9ucyBvZiBwcm9rYXJ5b3RpYyBnZW5lczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9Njc3MC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbnZpcm9ubWVudGFsIGluZmx1ZW5jZXMgb24gRE5BIGN1cnZhdHVyZTsgSi4gQmlvbW9sLiBTdHJ1Y3QuIER5bi47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz04MTEtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUF2b2lkaW5nIGRhbmdlcm91cyBtaXNzZW5zZTogdGhlcm1vcGhpbGVzIGRpc3BsYXkgZXNwZWNpYWxseSBsb3cgbXV0YXRpb24gcmF0ZXM7IFBMb1MgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMDUyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMS8zLzI4NS8xMTA4ODE4L2RzdDA1Ny5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgUHJva2FyeW90aWMgZ2Vub21lcyBhcmUgZGl2ZXJzZSBpbiB0ZXJtcyBvZiB0aGVpciBudWNsZW90aWRlIGFuZCBvbGlnb251Y2xlb3RpZGUgY29tcG9zaXRpb24gYXMgd2VsbCBhcyBwcmVzZW5jZSBvZiB2YXJpb3VzIHNlcXVlbmNlIGZlYXR1cmVzIHRoYXQgY2EiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM5Mzk5NSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGTC4NFk82zacu20g2cPicpQ3fvnUQ==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f3%2f285%2f393995" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f3%2f285%2f393995" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=15369" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/3/1/m_cover.gif?Expires=2147483647&amp;Signature=eD74u2chqR7m7Q4xVEpu-B2CnS8a5UZybuUBj9IiQcs7vcn-sLThVTH2YkcAPlEiODArsSqGH041ZPg5RakmGwkyHxqVC4LdBdXxFui4WCeLWHefwauec7mA5UKsie6Bk-ZRp7yPg10FfeUTBrjN7buCiodIP00VlbrJgVNTJFuLyaUUxPgJ3yahCwwX4TAcK2CRCX-3Y~LqokrGvJCW5URLAAo4GxWdmNUAJqBhdMREkIh~McyOhMfYKJNLDTxW9vnvEbKMxNW5048gQKGgF7kMxJ3~NsfswZCCrwYr2quaHFHHORZM0v4Eh4XRYgKpFeDHa6jnY4OeJPxjcOPrMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/3">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5813676">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5813676">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5813681">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5813681">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5813696">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5813696">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5813723">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5813723">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5813770">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5813770">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5813807">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5813807">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5813809">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5813809">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5813811">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5813811">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5813813">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5813813">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/3/267/393841">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/21/3/299/394334">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Assessing Diversity of DNA Structure-Related Sequence Features in Prokaryotic Genomes
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yongjie Huang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Institute of Bioinformatics</div>, <div class="insititution">University of Georgia</div>, <div class="addr-line">Athens, GA 30602</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yongjie+Huang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Huang Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Huang Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yongjie Huang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jan Mr&#225;zek</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Institute of Bioinformatics</div>, <div class="insititution">University of Georgia</div>, <div class="addr-line">Athens, GA 30602</div>, <div class="country">USA</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Microbiology</div>, <div class="insititution">University of Georgia</div>, <div class="addr-line">Athens, GA 30602</div>, <div class="country">USA</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +1 706-542-1065. Fax. +1 706-542-2674. Email: <a href="mailto:mrazek@uga.edu" target="_blank">mrazek@uga.edu</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jan+Mr%c3%a1zek">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mr&#225;zek J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mr&#225;zek J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jan Mrázek</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 3, 1 June 2014, Pages 285–297, <a href='https://doi.org/10.1093/dnares/dst057'>https://doi.org/10.1093/dnares/dst057</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">09 January 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">25 July 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">02 December 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="393995" href="/dnaresearch/article-pdf/21/3/285/1108818/dst057.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yongjie Huang, Jan Mrázek;  Assessing Diversity of DNA Structure-Related Sequence Features in Prokaryotic Genomes, <em>DNA Research</em>, Volume 21, Issue 3, 1 June 2014, Pages 285–297, <a href='https://doi.org/10.1093/dnares/dst057'>https://doi.org/10.1093/dnares/dst057</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/393995?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/393995?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/393995?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/393995?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/393995?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/393995?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/393995?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Assessing%20Diversity%20of%20DNA%20Structure-Related%20Sequence%20Features%20in%20Prokaryotic%20Genomes&amp;publicationDate=2014-01-09&amp;volumeNum=21&amp;issueNum=3&amp;author=Huang%2C%20Yongjie%3B%20Mr%C3%A1zek%2C%20Jan&amp;startPage=285&amp;endPage=297&amp;contentId=10.1093%2Fdnares%2Fdst057&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f3%2f285%2f393995" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5813676 id="5813676" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Prokaryotic genomes are diverse in terms of their nucleotide and oligonucleotide composition as well as presence of various sequence features that can affect physical properties of the DNA molecule. We present a survey of local sequence patterns which have a potential to promote non-canonical DNA conformations (i.e. different from standard B-DNA double helix) and interpret the results in terms of relationships with organisms' habitats, phylogenetic classifications, and other characteristics. Our present work differs from earlier similar surveys not only by investigating a wider range of sequence patterns in a large number of genomes but also by using a more realistic null model to assess significant deviations. Our results show that simple sequence repeats and Z-DNA-promoting patterns are generally suppressed in prokaryotic genomes, whereas palindromes and inverted repeats are over-represented. Representation of patterns that promote Z-DNA and intrinsic DNA curvature increases with increasing optimal growth temperature (OGT), and decreases with increasing oxygen requirement. Additionally, representations of close direct repeats, palindromes and inverted repeats exhibit clear negative trends with increasing OGT. The observed relationships with environmental characteristics, particularly OGT, suggest possible evolutionary scenarios of structural adaptation of DNA to particular environmental niches.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;sequence patterns&quot;">sequence patterns</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Z-DNA">Z-DNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DNA curvature&quot;">DNA curvature</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;sequence repeats&quot;">sequence repeats</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="palindromes">palindromes</a></div>                            <h2 scrollto-destination=5813681 id="5813681" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Prokaryotic genomes are extremely diverse in terms of their nucleotide and oligonucleotide composition as well as presence of various forms of sequence repeats and patterns that can affect physical properties of the DNA molecule. For example, Ussery <em>et al</em>.<sup><a reveal-id="DST057C1" data-open="DST057C1" class="link link-ref link-reveal xref-bibr">1</a></sup> analysed oligopurine/oligopyrimidine runs and alternating purine/pyrimidine patterns in prokaryotic genomes and reported large differences among different organisms. Alternating purine/pyrimidine patterns promote Z-DNA conformation, whereas oligopurine/oligopurimidine runs can facilitate formation of A-DNA or H-DNA.<sup><a reveal-id="DST057C2" data-open="DST057C2" class="link link-ref link-reveal xref-bibr">2</a></sup> They found that differences among different prokaryotes were related to optimal growth temperature (OGT) and pathogenicity, possibly as a result of structural adaptations of DNA of these organisms.<sup><a reveal-id="DST057C3" data-open="DST057C3" class="link link-ref link-reveal xref-bibr">3</a></sup> In another example, potential G-DNA-forming sequences were found to occur at vastly different frequencies in different prokaryotic genomes.<sup><a reveal-id="DST057C4" data-open="DST057C4" class="link link-ref link-reveal xref-bibr">4</a></sup> The amount of intrinsic DNA curvature in promoter regions was found to be related to OGT.<sup><a reveal-id="DST057C5" data-open="DST057C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Our own analysis of more than 1000 prokaryotic chromosomes suggested a large variance among different genomes in DNA curvature-related 10–11 bp sequence periodicity, which could reflect differences in chromosomal structure.<sup><a reveal-id="DST057C6" data-open="DST057C6" class="link link-ref link-reveal xref-bibr">6</a></sup> The presence of long simple sequence repeats (SSRs) in a genome is strongly correlated with the organism's dependence on a eukaryotic host.<sup><a reveal-id="DST057C7" data-open="DST057C7" class="link link-ref link-reveal xref-bibr">7</a></sup> These results indicate a significant variance of general DNA properties among different prokaryotes, which in some cases appear to be related to the organisms' habitats and lifestyles.</p><p>Advances in DNA sequencing technologies over the past decades led to a situation where complete genomes are available for many microbes about which very little is known apart from the information that can be derived from the genomic DNA sequence. In particular, despite the diversity of DNA properties mentioned above, our knowledge about chromosome structure and organization in the cell is limited to studies of a few model organisms. While it is reasonable to assume that the general model of bacterial nucleoid composed of dynamic, supercoiled DNA loops stabilized by nucleoid-associated proteins is probably universal,<sup><a reveal-id="DST057C8" data-open="DST057C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DST057C9" data-open="DST057C9" class="link link-ref link-reveal xref-bibr">9</a></sup> the differences in DNA sequence properties suggest that subtle variations may exist among bacteria with completely sequenced genomes. Such differences in physical characteristics of the chromosomes could play roles in the organisms' physiology and adaptations to their particular environments.</p><p>We present a survey of local sequence patterns in prokaryotic genomes with a potential to generate local irregularities in DNA structure. Our goal was to assess diversity of the prokaryotic genomes in terms of abundance of sequence patterns indicative of possible structural transitions in the DNA molecule. While physiological functions of sequence patterns promoting non-canonical DNA conformations and sequence patterns promoting conformational transitions in DNA are not well understood, there is increasing evidence that they can play important roles. For example, SSRs have been implicated in phase variation, a mechanism that promotes reversible switching of phenotypes.<sup><a reveal-id="DST057C10" data-open="DST057C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DST057C11" data-open="DST057C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Palindromes and close inverted repeats form stem-loop structures in RNA, which can function in transcription terminators and riboswitches, and they can promote formation of cruciform structures in DNA, which influence replication, regulation of gene expression, and recombination.<sup><a reveal-id="DST057C12" data-open="DST057C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Close direct and inverted repeats have mutagenic effects on DNA.<sup><a reveal-id="DST057C13  DST057C16" data-open="DST057C13  DST057C16" class="link link-ref link-reveal xref-bibr">13–16</a></sup> Specific guanine-rich patterns can promote formation of quadruplex G-DNA.<sup><a reveal-id="DST057C17  DST057C19" data-open="DST057C17  DST057C19" class="link link-ref link-reveal xref-bibr">17–19</a></sup> The G-DNA formation in telomeres is well documented, but G-DNA may also play a role in gene expression, replication, recombination, and integration of viruses.<sup><a reveal-id="DST057C19  DST057C23" data-open="DST057C19  DST057C23" class="link link-ref link-reveal xref-bibr">19–23</a></sup> Alternating purine–pyrimidine patterns can under favourable conditions promote transitions to the Z-DNA conformation, while oligopurine/oligopyrimidine runs promote formation of A-DNA or triple-stranded H-DNA.<sup><a reveal-id="DST057C2" data-open="DST057C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DST057C24  DST057C28" data-open="DST057C24  DST057C28" class="link link-ref link-reveal xref-bibr">24–28</a></sup> Transient formation of left-handed Z-DNA can influence transcription and promote genome instability,<sup><a reveal-id="DST057C28  DST057C31" data-open="DST057C28  DST057C31" class="link link-ref link-reveal xref-bibr">28–31</a></sup> and indirect evidence suggests similar roles for triple-helical H-DNA.<sup><a reveal-id="DST057C26" data-open="DST057C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DST057C32" data-open="DST057C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Intrinsic DNA curvature is largely related to periodic spacings of A-tracts and can influence DNA–protein interactions in regulatory regions, aid DNA compaction in the nucleoid, and possibly promote a particular mode of supercoiling.<sup><a reveal-id="DST057C5" data-open="DST057C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DST057C33" data-open="DST057C33" class="link link-ref link-reveal xref-bibr">33</a>,<a reveal-id="DST057C34" data-open="DST057C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Although the physiological significance of the unusual DNA conformations in different organisms is still poorly understood, it is reasonable to ask how common the sequence patterns that promote their formation are in different genomes because their unusually high or low occurrence may indicate its functional importance.</p><p>Our present work differs from earlier similar surveys not only by investigating a wider range of sequence patterns in a large number of genomes but also by using a more realistic null model to assess significant anomalies. Significant deviations from expected pattern occurrences could indicate that the pattern <em>per se</em> is subject to selective constraint and therefore functionally significant in a given organism but the pattern over- or under-representation could also arise as a tolerated artefact of other biases affecting the DNA sequence. Our null model reflects the genome-specific nearest neighbour preferences, codon biases, and heterogeneity of the DNA sequence; therefore, deviations from expected occurrences are more likely to reflect their functional significance of the investigated sequence patterns. We interpret our results in terms of relationships with organisms' habitats, phylogenetic classifications, and other characteristics.</p>                            <h2 scrollto-destination=5813696 id="5813696" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5813700 id="5813700" class="section-title" data-legacy-id=s2a>2.1. Data sets</h3>
<p>Complete prokaryotic genomes were downloaded from the National Center for Biotechnology Information (NCBI) ftp server (<a class="link link-uri" href="http://ftp://ftp.ncbi.nih.gov/genomes/Bacteria/">ftp://ftp.ncbi.nih.gov/genomes/Bacteria/</a>). Pattern representations were assessed for the complete genomes, as well as for 1 Mb segments randomly selected from each genome. The purpose of the latter approach was to reduce the effect of statistical artefacts from comparing sequences of different sizes. The results presented in this paper are based on the 1 Mb segments, while the data for the complete genomes are shown in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data</a> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S5–S7</a>). Genomes smaller than 1 Mb were excluded from the analysis. Our final data set included 1424 complete genomes of 941 species, 519 genera, and 37 phyla or subphyla (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). We used the existing annotation (the ‘CDS’ keywords) to differentiate protein-coding and non-coding sequences.</p><p>The OGT and oxygen requirement classifications for each genome were obtained from the Genomes Online Database (<a class="link link-uri" href="http://www.genomesonline.org">http://www.genomesonline.org</a>).<sup><a reveal-id="DST057C35" data-open="DST057C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Among the 1424 complete genomes, 1378 genomes have the OGT classification available and 1304 genomes have the oxygen requirement classification available. To eliminate sampling biases towards genera represented by many completely sequenced genomes, we performed statistical assessments at the level of genera rather than individual genomes. Accordingly, we apply a single classification to each genus, which is determined by the majority of species within the genus. This procedure yielded OGT classification for 498 genera (including 18 psychrophiles and psychrotolerant organisms, 382 mesophiles, 72 thermophiles, and 26 hyperthermophiles) and oxygen requirement classification for 465 genera (159 anaerobes, 202 aerobes, 95 facultative, and 9 microaerophilic organisms).</p>                            <h3 scrollto-destination=5813708 id="5813708" class="section-title" data-legacy-id=s2b>2.2. Patterns of interest</h3>
<p>Table <a reveal-id="DST057TB1" data-open="DST057TB1" class="link link-reveal link-table xref-fig">1</a> provides a list of sequence patterns whose occurrences in prokaryotic genomes were investigated in this work. The patterns were selected based on the available information about sequences that promote various forms of structural transitions or mutations in DNA under favourable circumstances. However, because exact rules governing structural transitions in DNA are not fully understood, we use multiple forms of similar patterns, in most cases pertaining to varying length of the pattern and number of tolerated mismatches. The specific parameters used in the definition of the patterns were in part dictated by the size of the analysed genomes in order to obtain sufficient data sample for statistical evaluations. </p>                        <a id="5813714" scrollto-destination="5813714"></a>
<div content-id="DST057TB1" class="table-modal"><div class="table-wrap"><div id="DST057TB1" class="table-wrap-title" data-id="DST057TB1"><div class="label -label">Table 1.</div><div class="caption"><p>List of sequence patterns investigated in this work</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Pattern </th><th>Code </th><th>Meaning </th><th>Example<sup>a</sup> </th></tr></thead><tbody><tr><td rowspan="11">Simple sequence repeats </td><td>1n8 </td><td>A single nucleotide repeated 8+ times in a row </td><td>G<span class="underline">CAAAAAAAAA</span>TA </td></tr><tr><td>2n5 </td><td>A dinucleotide repeated 5+ times in a row </td><td>AC<span class="underline">CACACACACA</span>TA </td></tr><tr><td>3n4 </td><td rowspan="9">Analogous to the two examples above </td><td> </td></tr><tr><td>4n4 </td><td> </td></tr><tr><td>5n4 </td><td> </td></tr><tr><td>6n3 </td><td> </td></tr><tr><td>7n3 </td><td> </td></tr><tr><td>8n3 </td><td> </td></tr><tr><td>9n2 </td><td> </td></tr><tr><td>10n2 </td><td> </td></tr><tr><td>11n2 </td><td> </td></tr><tr><td rowspan="5">Close direct repeats </td><td>4n6g12 </td><td>A tetranucleotide repeated 6+ times with gaps ≤12 nt </td><td>TA<span class="underline">CCAT</span>GCT<span class="underline">CCAT</span>TA<span class="underline">CCAT</span>AG<span class="underline">CCAT</span>… </td></tr><tr><td>6n6g24 </td><td>A 6-mer repeated 6+ times with gaps ≤24 nt </td><td> </td></tr><tr><td>8n4g24 </td><td>An 8-mer repeated 4+ times with gaps ≤24 nt </td><td> </td></tr><tr><td>cd8g6 </td><td>An 8-mer repeated within 6 nt </td><td><span class="underline">CTTAGGCA</span>TCAC<span class="underline">CTTAGGCA</span> </td></tr><tr><td>cd10g50 </td><td>A 10-mer repeated within 50 nt </td><td> </td></tr><tr><td rowspan="5">Palindromes and inverted repeats </td><td>cp8g6 </td><td>Inverted repeat of an 8-mer separated by no more than 6 nt </td><td><span class="underline">CTTAGGCA</span>TCAC<span class="underline">TGCCTAAG</span> </td></tr><tr><td>cp10g50 </td><td>Inverted repeat of a 10-mer separated by ≤50 nt </td><td> </td></tr><tr><td>pals9 </td><td>9 nt inverted repeat (no separation) OR 12 nt inverted repeat allowing 1 mismatch OR 15 nt inverted repeat allowing 2 mismatches OR … (one mismatch added for every 3 nt length) </td><td>CTGGATCAGGCTAAA⋮TTCAGCCTCATCCAG </td></tr><tr><td>pals9g12 </td><td>Like pals9 but allowing separation up to 12 bp </td><td> </td></tr><tr><td>pals12g20 </td><td>Analogous to the example above </td><td> </td></tr><tr><td rowspan="10">H-DNA-related patterns </td><td>cm8g6 </td><td>Mirror repeat of an 8-mer separated by ≤6 nt </td><td><span class="underline">CTTAGGCA</span>TCAC<span class="underline">ACGGATTC</span> </td></tr><tr><td>cm10g50 </td><td>Mirror repeat of a 10-mer separated by ≤50 nt </td><td> </td></tr><tr><td>mirs9 </td><td>9 nt mirror repeat (no separation) OR 12 nt mirror repeat allowing 1 mismatch OR 15 nt mirror repeat allowing 2 mismatches OR … (one mismatch added for every 3 nt length) </td><td>CTGGATCAGGCTAAA⋮AACTCGGACTGGGTC </td></tr><tr><td>mirs9g12 </td><td>Like mirs9 but allowing separation up to 12 bp </td><td> </td></tr><tr><td>mirs12g20 </td><td>Analogous to mirs9g12 </td><td> </td></tr><tr><td>R15 </td><td>Run of ≥15 purines or pyrimidines </td><td>AAGGGAGGGAGGAGA </td></tr><tr><td>R30 </td><td>Run of ≥30 purines or pyrimidines </td><td> </td></tr><tr><td>R30e3 </td><td>Run of ≥30 purines or pyrimidines allowing ≤3 errors </td><td> </td></tr><tr><td>R45e6 </td><td rowspan="2">Analogous to the example above </td><td> </td></tr><tr><td>R60e9 </td><td> </td></tr><tr><td rowspan="3">G-DNA-related patterns </td><td>GG8g4 </td><td>8 or more GG dimers separated by ≤4 nt from each other </td><td><span class="underline">GG</span>A<span class="underline">GG</span>CT<span class="underline">GG</span>C<span class="underline">GGGG</span>C<span class="underline">GG</span>T<span class="underline">GGGG</span> </td></tr><tr><td>GGG4g6 </td><td rowspan="2">Analogous to the example above </td><td> </td></tr><tr><td>GGGG4g6 </td><td> </td></tr><tr><td rowspan="6">Z-DNA-related patterns </td><td>GC6 </td><td>Alternating G–C, ≥6 nt length </td><td>GCGCGC </td></tr><tr><td>GC8 </td><td>Alternating G–C, ≥8 nt length </td><td> </td></tr><tr><td>RY12 </td><td>Alternating R-Y, ≥12 nt length </td><td>TGTACGTGTGCA </td></tr><tr><td>RY12e1 </td><td>Like RY12 but allowing 1 error </td><td>TGTACGAGTGCA </td></tr><tr><td>RY18e2 </td><td>Alternating R-Y, ≥18 nt length, ≤2 errors </td><td> </td></tr><tr><td>RY24e3 </td><td>Alternating R-Y, ≥24 nt length, ≤3 errors </td><td> </td></tr><tr><td rowspan="3">DNA bending </td><td>bend45w60 </td><td>Predicted bend of ≥45° within a ≤60 bp segment </td><td> </td></tr><tr><td>bend60w100 </td><td>Predicted bend of ≥60° within a ≤100 bp segment </td><td> </td></tr><tr><td>bend90w120 </td><td>Predicted bend of ≥90° within a ≤120 bp segment </td><td> </td></tr></tbody></table></div><div content-id="DST057TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Pattern </th><th>Code </th><th>Meaning </th><th>Example<sup>a</sup> </th></tr></thead><tbody><tr><td rowspan="11">Simple sequence repeats </td><td>1n8 </td><td>A single nucleotide repeated 8+ times in a row </td><td>G<span class="underline">CAAAAAAAAA</span>TA </td></tr><tr><td>2n5 </td><td>A dinucleotide repeated 5+ times in a row </td><td>AC<span class="underline">CACACACACA</span>TA </td></tr><tr><td>3n4 </td><td rowspan="9">Analogous to the two examples above </td><td> </td></tr><tr><td>4n4 </td><td> </td></tr><tr><td>5n4 </td><td> </td></tr><tr><td>6n3 </td><td> </td></tr><tr><td>7n3 </td><td> </td></tr><tr><td>8n3 </td><td> </td></tr><tr><td>9n2 </td><td> </td></tr><tr><td>10n2 </td><td> </td></tr><tr><td>11n2 </td><td> </td></tr><tr><td rowspan="5">Close direct repeats </td><td>4n6g12 </td><td>A tetranucleotide repeated 6+ times with gaps ≤12 nt </td><td>TA<span class="underline">CCAT</span>GCT<span class="underline">CCAT</span>TA<span class="underline">CCAT</span>AG<span class="underline">CCAT</span>… </td></tr><tr><td>6n6g24 </td><td>A 6-mer repeated 6+ times with gaps ≤24 nt </td><td> </td></tr><tr><td>8n4g24 </td><td>An 8-mer repeated 4+ times with gaps ≤24 nt </td><td> </td></tr><tr><td>cd8g6 </td><td>An 8-mer repeated within 6 nt </td><td><span class="underline">CTTAGGCA</span>TCAC<span class="underline">CTTAGGCA</span> </td></tr><tr><td>cd10g50 </td><td>A 10-mer repeated within 50 nt </td><td> </td></tr><tr><td rowspan="5">Palindromes and inverted repeats </td><td>cp8g6 </td><td>Inverted repeat of an 8-mer separated by no more than 6 nt </td><td><span class="underline">CTTAGGCA</span>TCAC<span class="underline">TGCCTAAG</span> </td></tr><tr><td>cp10g50 </td><td>Inverted repeat of a 10-mer separated by ≤50 nt </td><td> </td></tr><tr><td>pals9 </td><td>9 nt inverted repeat (no separation) OR 12 nt inverted repeat allowing 1 mismatch OR 15 nt inverted repeat allowing 2 mismatches OR … (one mismatch added for every 3 nt length) </td><td>CTGGATCAGGCTAAA⋮TTCAGCCTCATCCAG </td></tr><tr><td>pals9g12 </td><td>Like pals9 but allowing separation up to 12 bp </td><td> </td></tr><tr><td>pals12g20 </td><td>Analogous to the example above </td><td> </td></tr><tr><td rowspan="10">H-DNA-related patterns </td><td>cm8g6 </td><td>Mirror repeat of an 8-mer separated by ≤6 nt </td><td><span class="underline">CTTAGGCA</span>TCAC<span class="underline">ACGGATTC</span> </td></tr><tr><td>cm10g50 </td><td>Mirror repeat of a 10-mer separated by ≤50 nt </td><td> </td></tr><tr><td>mirs9 </td><td>9 nt mirror repeat (no separation) OR 12 nt mirror repeat allowing 1 mismatch OR 15 nt mirror repeat allowing 2 mismatches OR … (one mismatch added for every 3 nt length) </td><td>CTGGATCAGGCTAAA⋮AACTCGGACTGGGTC </td></tr><tr><td>mirs9g12 </td><td>Like mirs9 but allowing separation up to 12 bp </td><td> </td></tr><tr><td>mirs12g20 </td><td>Analogous to mirs9g12 </td><td> </td></tr><tr><td>R15 </td><td>Run of ≥15 purines or pyrimidines </td><td>AAGGGAGGGAGGAGA </td></tr><tr><td>R30 </td><td>Run of ≥30 purines or pyrimidines </td><td> </td></tr><tr><td>R30e3 </td><td>Run of ≥30 purines or pyrimidines allowing ≤3 errors </td><td> </td></tr><tr><td>R45e6 </td><td rowspan="2">Analogous to the example above </td><td> </td></tr><tr><td>R60e9 </td><td> </td></tr><tr><td rowspan="3">G-DNA-related patterns </td><td>GG8g4 </td><td>8 or more GG dimers separated by ≤4 nt from each other </td><td><span class="underline">GG</span>A<span class="underline">GG</span>CT<span class="underline">GG</span>C<span class="underline">GGGG</span>C<span class="underline">GG</span>T<span class="underline">GGGG</span> </td></tr><tr><td>GGG4g6 </td><td rowspan="2">Analogous to the example above </td><td> </td></tr><tr><td>GGGG4g6 </td><td> </td></tr><tr><td rowspan="6">Z-DNA-related patterns </td><td>GC6 </td><td>Alternating G–C, ≥6 nt length </td><td>GCGCGC </td></tr><tr><td>GC8 </td><td>Alternating G–C, ≥8 nt length </td><td> </td></tr><tr><td>RY12 </td><td>Alternating R-Y, ≥12 nt length </td><td>TGTACGTGTGCA </td></tr><tr><td>RY12e1 </td><td>Like RY12 but allowing 1 error </td><td>TGTACGAGTGCA </td></tr><tr><td>RY18e2 </td><td>Alternating R-Y, ≥18 nt length, ≤2 errors </td><td> </td></tr><tr><td>RY24e3 </td><td>Alternating R-Y, ≥24 nt length, ≤3 errors </td><td> </td></tr><tr><td rowspan="3">DNA bending </td><td>bend45w60 </td><td>Predicted bend of ≥45° within a ≤60 bp segment </td><td> </td></tr><tr><td>bend60w100 </td><td>Predicted bend of ≥60° within a ≤100 bp segment </td><td> </td></tr><tr><td>bend90w120 </td><td>Predicted bend of ≥90° within a ≤120 bp segment </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn1"><div class="fn"><p><sup>a</sup>Segments matching the sequence pattern are underscored, mismatches are shaded, and symmetrical segments are separated by a vertical dashed line.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5813714" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST057TB1" class="table-wrap-title" data-id="DST057TB1"><div class="label -label">Table 1.</div><div class="caption"><p>List of sequence patterns investigated in this work</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Pattern </th><th>Code </th><th>Meaning </th><th>Example<sup>a</sup> </th></tr></thead><tbody><tr><td rowspan="11">Simple sequence repeats </td><td>1n8 </td><td>A single nucleotide repeated 8+ times in a row </td><td>G<span class="underline">CAAAAAAAAA</span>TA </td></tr><tr><td>2n5 </td><td>A dinucleotide repeated 5+ times in a row </td><td>AC<span class="underline">CACACACACA</span>TA </td></tr><tr><td>3n4 </td><td rowspan="9">Analogous to the two examples above </td><td> </td></tr><tr><td>4n4 </td><td> </td></tr><tr><td>5n4 </td><td> </td></tr><tr><td>6n3 </td><td> </td></tr><tr><td>7n3 </td><td> </td></tr><tr><td>8n3 </td><td> </td></tr><tr><td>9n2 </td><td> </td></tr><tr><td>10n2 </td><td> </td></tr><tr><td>11n2 </td><td> </td></tr><tr><td rowspan="5">Close direct repeats </td><td>4n6g12 </td><td>A tetranucleotide repeated 6+ times with gaps ≤12 nt </td><td>TA<span class="underline">CCAT</span>GCT<span class="underline">CCAT</span>TA<span class="underline">CCAT</span>AG<span class="underline">CCAT</span>… </td></tr><tr><td>6n6g24 </td><td>A 6-mer repeated 6+ times with gaps ≤24 nt </td><td> </td></tr><tr><td>8n4g24 </td><td>An 8-mer repeated 4+ times with gaps ≤24 nt </td><td> </td></tr><tr><td>cd8g6 </td><td>An 8-mer repeated within 6 nt </td><td><span class="underline">CTTAGGCA</span>TCAC<span class="underline">CTTAGGCA</span> </td></tr><tr><td>cd10g50 </td><td>A 10-mer repeated within 50 nt </td><td> </td></tr><tr><td rowspan="5">Palindromes and inverted repeats </td><td>cp8g6 </td><td>Inverted repeat of an 8-mer separated by no more than 6 nt </td><td><span class="underline">CTTAGGCA</span>TCAC<span class="underline">TGCCTAAG</span> </td></tr><tr><td>cp10g50 </td><td>Inverted repeat of a 10-mer separated by ≤50 nt </td><td> </td></tr><tr><td>pals9 </td><td>9 nt inverted repeat (no separation) OR 12 nt inverted repeat allowing 1 mismatch OR 15 nt inverted repeat allowing 2 mismatches OR … (one mismatch added for every 3 nt length) </td><td>CTGGATCAGGCTAAA⋮TTCAGCCTCATCCAG </td></tr><tr><td>pals9g12 </td><td>Like pals9 but allowing separation up to 12 bp </td><td> </td></tr><tr><td>pals12g20 </td><td>Analogous to the example above </td><td> </td></tr><tr><td rowspan="10">H-DNA-related patterns </td><td>cm8g6 </td><td>Mirror repeat of an 8-mer separated by ≤6 nt </td><td><span class="underline">CTTAGGCA</span>TCAC<span class="underline">ACGGATTC</span> </td></tr><tr><td>cm10g50 </td><td>Mirror repeat of a 10-mer separated by ≤50 nt </td><td> </td></tr><tr><td>mirs9 </td><td>9 nt mirror repeat (no separation) OR 12 nt mirror repeat allowing 1 mismatch OR 15 nt mirror repeat allowing 2 mismatches OR … (one mismatch added for every 3 nt length) </td><td>CTGGATCAGGCTAAA⋮AACTCGGACTGGGTC </td></tr><tr><td>mirs9g12 </td><td>Like mirs9 but allowing separation up to 12 bp </td><td> </td></tr><tr><td>mirs12g20 </td><td>Analogous to mirs9g12 </td><td> </td></tr><tr><td>R15 </td><td>Run of ≥15 purines or pyrimidines </td><td>AAGGGAGGGAGGAGA </td></tr><tr><td>R30 </td><td>Run of ≥30 purines or pyrimidines </td><td> </td></tr><tr><td>R30e3 </td><td>Run of ≥30 purines or pyrimidines allowing ≤3 errors </td><td> </td></tr><tr><td>R45e6 </td><td rowspan="2">Analogous to the example above </td><td> </td></tr><tr><td>R60e9 </td><td> </td></tr><tr><td rowspan="3">G-DNA-related patterns </td><td>GG8g4 </td><td>8 or more GG dimers separated by ≤4 nt from each other </td><td><span class="underline">GG</span>A<span class="underline">GG</span>CT<span class="underline">GG</span>C<span class="underline">GGGG</span>C<span class="underline">GG</span>T<span class="underline">GGGG</span> </td></tr><tr><td>GGG4g6 </td><td rowspan="2">Analogous to the example above </td><td> </td></tr><tr><td>GGGG4g6 </td><td> </td></tr><tr><td rowspan="6">Z-DNA-related patterns </td><td>GC6 </td><td>Alternating G–C, ≥6 nt length </td><td>GCGCGC </td></tr><tr><td>GC8 </td><td>Alternating G–C, ≥8 nt length </td><td> </td></tr><tr><td>RY12 </td><td>Alternating R-Y, ≥12 nt length </td><td>TGTACGTGTGCA </td></tr><tr><td>RY12e1 </td><td>Like RY12 but allowing 1 error </td><td>TGTACGAGTGCA </td></tr><tr><td>RY18e2 </td><td>Alternating R-Y, ≥18 nt length, ≤2 errors </td><td> </td></tr><tr><td>RY24e3 </td><td>Alternating R-Y, ≥24 nt length, ≤3 errors </td><td> </td></tr><tr><td rowspan="3">DNA bending </td><td>bend45w60 </td><td>Predicted bend of ≥45° within a ≤60 bp segment </td><td> </td></tr><tr><td>bend60w100 </td><td>Predicted bend of ≥60° within a ≤100 bp segment </td><td> </td></tr><tr><td>bend90w120 </td><td>Predicted bend of ≥90° within a ≤120 bp segment </td><td> </td></tr></tbody></table></div><div content-id="DST057TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Pattern </th><th>Code </th><th>Meaning </th><th>Example<sup>a</sup> </th></tr></thead><tbody><tr><td rowspan="11">Simple sequence repeats </td><td>1n8 </td><td>A single nucleotide repeated 8+ times in a row </td><td>G<span class="underline">CAAAAAAAAA</span>TA </td></tr><tr><td>2n5 </td><td>A dinucleotide repeated 5+ times in a row </td><td>AC<span class="underline">CACACACACA</span>TA </td></tr><tr><td>3n4 </td><td rowspan="9">Analogous to the two examples above </td><td> </td></tr><tr><td>4n4 </td><td> </td></tr><tr><td>5n4 </td><td> </td></tr><tr><td>6n3 </td><td> </td></tr><tr><td>7n3 </td><td> </td></tr><tr><td>8n3 </td><td> </td></tr><tr><td>9n2 </td><td> </td></tr><tr><td>10n2 </td><td> </td></tr><tr><td>11n2 </td><td> </td></tr><tr><td rowspan="5">Close direct repeats </td><td>4n6g12 </td><td>A tetranucleotide repeated 6+ times with gaps ≤12 nt </td><td>TA<span class="underline">CCAT</span>GCT<span class="underline">CCAT</span>TA<span class="underline">CCAT</span>AG<span class="underline">CCAT</span>… </td></tr><tr><td>6n6g24 </td><td>A 6-mer repeated 6+ times with gaps ≤24 nt </td><td> </td></tr><tr><td>8n4g24 </td><td>An 8-mer repeated 4+ times with gaps ≤24 nt </td><td> </td></tr><tr><td>cd8g6 </td><td>An 8-mer repeated within 6 nt </td><td><span class="underline">CTTAGGCA</span>TCAC<span class="underline">CTTAGGCA</span> </td></tr><tr><td>cd10g50 </td><td>A 10-mer repeated within 50 nt </td><td> </td></tr><tr><td rowspan="5">Palindromes and inverted repeats </td><td>cp8g6 </td><td>Inverted repeat of an 8-mer separated by no more than 6 nt </td><td><span class="underline">CTTAGGCA</span>TCAC<span class="underline">TGCCTAAG</span> </td></tr><tr><td>cp10g50 </td><td>Inverted repeat of a 10-mer separated by ≤50 nt </td><td> </td></tr><tr><td>pals9 </td><td>9 nt inverted repeat (no separation) OR 12 nt inverted repeat allowing 1 mismatch OR 15 nt inverted repeat allowing 2 mismatches OR … (one mismatch added for every 3 nt length) </td><td>CTGGATCAGGCTAAA⋮TTCAGCCTCATCCAG </td></tr><tr><td>pals9g12 </td><td>Like pals9 but allowing separation up to 12 bp </td><td> </td></tr><tr><td>pals12g20 </td><td>Analogous to the example above </td><td> </td></tr><tr><td rowspan="10">H-DNA-related patterns </td><td>cm8g6 </td><td>Mirror repeat of an 8-mer separated by ≤6 nt </td><td><span class="underline">CTTAGGCA</span>TCAC<span class="underline">ACGGATTC</span> </td></tr><tr><td>cm10g50 </td><td>Mirror repeat of a 10-mer separated by ≤50 nt </td><td> </td></tr><tr><td>mirs9 </td><td>9 nt mirror repeat (no separation) OR 12 nt mirror repeat allowing 1 mismatch OR 15 nt mirror repeat allowing 2 mismatches OR … (one mismatch added for every 3 nt length) </td><td>CTGGATCAGGCTAAA⋮AACTCGGACTGGGTC </td></tr><tr><td>mirs9g12 </td><td>Like mirs9 but allowing separation up to 12 bp </td><td> </td></tr><tr><td>mirs12g20 </td><td>Analogous to mirs9g12 </td><td> </td></tr><tr><td>R15 </td><td>Run of ≥15 purines or pyrimidines </td><td>AAGGGAGGGAGGAGA </td></tr><tr><td>R30 </td><td>Run of ≥30 purines or pyrimidines </td><td> </td></tr><tr><td>R30e3 </td><td>Run of ≥30 purines or pyrimidines allowing ≤3 errors </td><td> </td></tr><tr><td>R45e6 </td><td rowspan="2">Analogous to the example above </td><td> </td></tr><tr><td>R60e9 </td><td> </td></tr><tr><td rowspan="3">G-DNA-related patterns </td><td>GG8g4 </td><td>8 or more GG dimers separated by ≤4 nt from each other </td><td><span class="underline">GG</span>A<span class="underline">GG</span>CT<span class="underline">GG</span>C<span class="underline">GGGG</span>C<span class="underline">GG</span>T<span class="underline">GGGG</span> </td></tr><tr><td>GGG4g6 </td><td rowspan="2">Analogous to the example above </td><td> </td></tr><tr><td>GGGG4g6 </td><td> </td></tr><tr><td rowspan="6">Z-DNA-related patterns </td><td>GC6 </td><td>Alternating G–C, ≥6 nt length </td><td>GCGCGC </td></tr><tr><td>GC8 </td><td>Alternating G–C, ≥8 nt length </td><td> </td></tr><tr><td>RY12 </td><td>Alternating R-Y, ≥12 nt length </td><td>TGTACGTGTGCA </td></tr><tr><td>RY12e1 </td><td>Like RY12 but allowing 1 error </td><td>TGTACGAGTGCA </td></tr><tr><td>RY18e2 </td><td>Alternating R-Y, ≥18 nt length, ≤2 errors </td><td> </td></tr><tr><td>RY24e3 </td><td>Alternating R-Y, ≥24 nt length, ≤3 errors </td><td> </td></tr><tr><td rowspan="3">DNA bending </td><td>bend45w60 </td><td>Predicted bend of ≥45° within a ≤60 bp segment </td><td> </td></tr><tr><td>bend60w100 </td><td>Predicted bend of ≥60° within a ≤100 bp segment </td><td> </td></tr><tr><td>bend90w120 </td><td>Predicted bend of ≥90° within a ≤120 bp segment </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn1"><div class="fn"><p><sup>a</sup>Segments matching the sequence pattern are underscored, mismatches are shaded, and symmetrical segments are separated by a vertical dashed line.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5813714" target="_blank">View Large</a></div></div><p>It is worthwhile to note that we are not aiming to predict accurately every site in the genome that is likely to undergo a conformational transition under favourable conditions. We are only asking how common are sequences favouring certain structural transitions in each genome and whether their frequency could be indicative of selective constraints acting on such sequences. At the same time, the extensive simulations with randomized genomes described below require that the patterns are sufficiently simple that they can be identified quickly. Given this limitation, we selected the sequence patterns to be approximately representative of the sequences known to undergo the specific structural transitions under favourable conditions.</p>                            <h3 scrollto-destination=5813716 id="5813716" class="section-title" data-legacy-id=s2c>2.3. Evaluation of pattern representations</h3>
<p>Pattern Locator<sup><a reveal-id="DST057C36" data-open="DST057C36" class="link link-ref link-reveal xref-bibr">36</a></sup> was used to count loci in each analysed sequence that matched the sequence pattern at hand (the observed count). The expected count of matching loci was determined as the average number of matching loci in 20 randomized sequences. The randomized sequences were generated by the ‘m1c1’ model of the Genome Randomizer software previously developed in our laboratory<sup><a reveal-id="DST057C37" data-open="DST057C37" class="link link-ref link-reveal xref-bibr">37</a></sup> (available for download at <a class="link link-uri" href="http://www.cmbl.uga.edu/software.html">http://www.cmbl.uga.edu/software.html</a>). In this model, the analysed sequence is first divided into segments corresponding to annotated protein-coding genes and intergenic regions. For each intergenic segment, a random sequence of the same length is generated as the first-order Markov chain, thus preserving the nucleotide and dinucleotide composition of that specific intergenic segment. Analogously, a random sequence is generated for each gene as a first-order Markov chain using the codon alphabet. The final randomized genome is constructed by reassembling the randomized genes and intergenic segments in their original order. The resulting randomized sequence mimics not only the overall nucleotide, dinucleotide, and codon frequencies of the complete genome but rather of each individual gene and intergenic region. Thus, the null model incorporates the compositional heterogeneity of the genomic sequence at the scale of individual genes. Because we factor the codon usage biases, dinucleotide preferences, and differences between protein-coding and non-coding sequences into the null model, we are more likely to detect anomalous usage of sequence patterns that arises from direct selection on the sequence patterns as opposed to anomalous usage that is a simple consequence of codon or dinucleotide biases characteristic of the particular genome or its various segments. As a result, our assessments are more likely to reflect selective constraints and functional significance of the analysed sequence pattern than assessments utilizing a simpler null model.</p><p>The pattern representations are classified into nine levels, from −4 (extremely under-represented), through 0 (normally represented) to +4 (extremely over-represented) based on the <em>P</em>-value and the observed/expected ratio (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). We use the combination of the two criteria because the observed/expected ratio is independent of the sample size and measures the deviation from the null model in more absolute terms but does not directly provide assessment of statistical significance. Using <em>P</em>-value alone could emphasize small deviations in large samples (e.g. large genomes or more frequent patterns), whereas using the observed/expected ratio alone could lead to over-interpretation of results that lack statistical significance. Assigning representation categories based on both criteria is a compromise designed to avoid these potential pitfalls. The <em>P</em>-values are assessed based on an assumption that the counts of matching loci follow the Poisson distribution. This is a reasonable approximation for long sequences and patterns that occur at low frequencies.</p><p>Statistical assessments could be skewed by inclusion of observations that are not independent, such as several closely related genomes (e.g. different strains of the same species or closely related species), which are likely to feature similar levels of representations of various sequence patterns simply as a result of insufficient evolutionary divergence. To reduce the effect of dependent observations resulting from insufficient divergence, we combined the results for all genomes of the same genus into a single ‘observation’ by averaging pattern representation categories for all available genomes belonging to that genus.</p>                            <h2 scrollto-destination=5813723 id="5813723" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=5813725 id="5813725" class="section-title" data-legacy-id=s3a>3.1. Simple sequence repeats</h3>
<p>A strong avoidance of tandem repeats of mono-, di-, and tri-nucleotides spreads over all phyla, especially in the whole genome and protein-coding regions, while tandem repeats of longer oligomers are generally normally represented and sometimes weakly over-represented (Table <a reveal-id="DST057TB2" data-open="DST057TB2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>). However, tandem repeats of mono-, di-, and tri-nucleotides are generally less suppressed in the intergenic regions when compared with protein-coding regions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S10 and S11</a>). <em>Trichodesmium erythraeum</em> and <em>Methanosphaera stadtmanae</em> are extreme examples with strongly over-represented mono-nucleotide repeats 1n8 (a single nucleotide repeated ≥8 times) in the intergenic regions (<em>P</em> &lt; 10<sup>−12</sup>), but normally represented in the whole genome and extremely under-represented in the protein-coding regions. Most Chlorobi (6 species out of 10) also tend to have over-represented 1n8 pattern in the intergenic regions. Tandem repeats of longer oligonucleotides (6–11 bp) are extremely over-represented in intergenic regions as well as complete genomes of <em>Methanococcus voltae</em>, <em>Methanococcus aeolicus</em>, and <em>Natrialba magadii</em> (and to a lesser extent in several other genomes) but not in the protein-coding segments (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S2–S4</a>). </p>                        <a id="5813728" scrollto-destination="5813728"></a>
<div content-id="DST057TB2" class="table-modal"><div class="table-wrap"><div id="DST057TB2" class="table-wrap-title" data-id="DST057TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Representation of sequence patterns in different phyla</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Pattern name </th><th rowspan="2">Pattern code </th><th>AlPr </th><th>BePr </th><th>GaPr </th><th>DePr </th><th>EpPr </th><th>Firm </th><th>Acti </th><th>Cyan </th><th>Bact </th><th>Chlb </th><th>Chlf </th><th>Dein </th><th>Fuso </th><th>Chla </th><th>Spir </th><th>Acid </th><th>Verr </th><th>Defe </th><th>Plan </th><th>Aqui </th><th>Ther </th><th>Eury </th><th>Cren </th></tr><tr><th>58 </th><th>39 </th><th>76 </th><th>23 </th><th>11 </th><th>61 </th><th>63 </th><th>12 </th><th>34 </th><th>5 </th><th>8 </th><th>6 </th><th>5 </th><th>6 </th><th>4 </th><th>4 </th><th>4 </th><th>4 </th><th>4 </th><th>8 </th><th>5 </th><th>44 </th><th>16 </th></tr></thead><tbody><tr><td rowspan="11">Simple sequence repeats </td><td>1n8 </td><td>−4.00 </td><td>−3.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−1.67 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−3.79 </td><td>−3.75 </td><td>−3.00 </td><td>−3.00 </td><td>−4.00 </td><td>−3.50 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td></tr><tr><td>2n5 </td><td>−2.21 </td><td>−2.00 </td><td>−3.00 </td><td>−3.00 </td><td>−4.00 </td><td>−4.00 </td><td>−3.00 </td><td>−3.63 </td><td>−4.00 </td><td>−3.00 </td><td>−3.00 </td><td>−2.60 </td><td>−4.00 </td><td>−4.00 </td><td>−3.50 </td><td>−2.00 </td><td>−2.50 </td><td>−4.00 </td><td>−2.00 </td><td>−4.00 </td><td>−4.00 </td><td>−3.00 </td><td>−3.00 </td></tr><tr><td>3n4 </td><td>−0.78 </td><td>−2.00 </td><td>−2.09 </td><td>−1.00 </td><td>−3.00 </td><td>−3.00 </td><td>−2.00 </td><td>−1.50 </td><td>−2.67 </td><td>−2.00 </td><td>−1.50 </td><td>−1.50 </td><td>−4.00 </td><td>−3.00 </td><td>−3.17 </td><td>−0.25 </td><td>−1.00 </td><td>−3.50 </td><td>−1.00 </td><td>−3.50 </td><td>−4.00 </td><td>−3.00 </td><td>−3.00 </td></tr><tr><td>4n4 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>5n4 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.07 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>6n3 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.07 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>7n3 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.75 </td><td>0.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.50 </td><td>1.00 </td><td>2.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>8n3 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.50 </td><td>0.00 </td><td>0.00 </td><td>0.50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>9n2 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.36 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>10n2 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.17 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>11n2 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.59 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td rowspan="5">Close direct repeats </td><td>4n6g12 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.13 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>6n6g24 </td><td>0.42 </td><td>0.25 </td><td>0.88 </td><td>1.00 </td><td>0.00 </td><td>0.30 </td><td>2.00 </td><td>1.50 </td><td>1.00 </td><td>1.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.98 </td><td>1.50 </td><td>0.00 </td><td>1.50 </td><td>1.25 </td><td>0.00 </td><td>0.00 </td><td>0.10 </td><td>0.00 </td></tr><tr><td>8n4g24 </td><td>0.50 </td><td>1.00 </td><td>1.14 </td><td>2.00 </td><td>0.00 </td><td>1.00 </td><td>3.00 </td><td>3.00 </td><td>1.50 </td><td>2.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.25 </td><td>2.25 </td><td>0.50 </td><td>1.50 </td><td>2.75 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>0.00 </td></tr><tr><td>cd8g6 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>1.00 </td><td>0.00 </td><td>2.00 </td><td>1.92 </td><td>0.00 </td><td>0.00 </td><td>0.75 </td><td>1.20 </td><td>0.00 </td><td>0.00 </td><td>0.02 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>0.60 </td><td>1.00 </td></tr><tr><td>cd10g50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>0.60 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.19 </td><td>0.15 </td><td>0.00 </td><td>0.00 </td><td>−1.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td rowspan="5">Palindromes and inverted repeats </td><td>cp8g6 </td><td>3.00 </td><td>3.00 </td><td>4.00 </td><td>3.50 </td><td>3.00 </td><td>4.00 </td><td>3.00 </td><td>1.81 </td><td>3.00 </td><td>2.00 </td><td>4.00 </td><td>2.63 </td><td>3.00 </td><td>4.00 </td><td>2.86 </td><td>3.50 </td><td>4.00 </td><td>3.00 </td><td>1.50 </td><td>2.00 </td><td>4.00 </td><td>2.00 </td><td>2.00 </td></tr><tr><td>cp10g50 </td><td>3.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>3.83 </td><td>4.00 </td><td>4.00 </td><td>3.54 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>3.75 </td><td>4.00 </td><td>4.00 </td><td>3.69 </td><td>3.50 </td><td>2.00 </td><td>3.00 </td><td>3.50 </td><td>1.00 </td><td>4.00 </td><td>1.00 </td><td>1.00 </td></tr><tr><td>pals9 </td><td>1.09 </td><td>2.00 </td><td>3.35 </td><td>3.00 </td><td>2.00 </td><td>4.00 </td><td>4.00 </td><td>2.00 </td><td>4.00 </td><td>3.00 </td><td>3.50 </td><td>3.00 </td><td>3.00 </td><td>3.96 </td><td>2.17 </td><td>1.50 </td><td>2.00 </td><td>1.50 </td><td>1.00 </td><td>0.00 </td><td>3.50 </td><td>0.10 </td><td>0.00 </td></tr><tr><td>pals9g12 </td><td>3.66 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>3.72 </td><td>4.00 </td><td>4.00 </td><td>3.71 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>3.75 </td><td>4.00 </td><td>4.00 </td><td>3.77 </td><td>4.00 </td><td>4.00 </td><td>3.00 </td><td>4.00 </td><td>1.00 </td><td>4.00 </td><td>2.00 </td><td>1.00 </td></tr><tr><td>pals12g20 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>3.77 </td><td>4.00 </td><td>4.00 </td><td>3.00 </td><td>4.00 </td><td>0.00 </td><td>4.00 </td><td>1.00 </td><td>0.22 </td></tr><tr><td rowspan="10">H-DNA-related patterns </td><td>cm8g6 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.34 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>cm10g50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.34 </td><td>0.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>mirs9 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>mirs9g12 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.25 </td><td>0.17 </td><td>0.00 </td><td>0.00 </td><td>0.34 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.25 </td><td>0.50 </td><td>0.00 </td><td>0.00 </td><td>0.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>mirs12g20 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.09 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.13 </td><td>0.00 </td><td>0.50 </td><td>0.00 </td><td>0.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>R15 </td><td>−0.20 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−1.00 </td><td>−0.20 </td><td>0.00 </td><td>−1.07 </td><td>0.00 </td><td>0.00 </td><td>0.25 </td><td>−1.00 </td><td>−4.00 </td><td>0.00 </td><td>−0.90 </td><td>−0.50 </td><td>1.00 </td><td>−3.00 </td><td>0.00 </td><td>−2.00 </td><td>−3.00 </td><td>−1.00 </td><td>−1.00 </td></tr><tr><td>R30 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.38 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>R30e3 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−2.00 </td><td>−0.25 </td><td>−0.13 </td><td>0.00 </td><td>0.50 </td><td>−1.00 </td><td>0.00 </td><td>−0.50 </td><td>−1.50 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>R45e6 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−2.00 </td><td>0.00 </td><td>0.15 </td><td>0.00 </td><td>0.50 </td><td>0.00 </td><td>0.50 </td><td>−0.50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>R60e9 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.07 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td rowspan="3">G-DNA-related patterns </td><td>GG8g4 </td><td>1.00 </td><td>1.33 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>2.70 </td><td>0.00 </td><td>0.00 </td><td>0.54 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>1.50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>GGG4g6 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.63 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.31 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>−0.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.50 </td><td>0.00 </td><td>−0.42 </td><td>0.00 </td></tr><tr><td>GGGG4g6 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.17 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td rowspan="6">Z-DNA-related patterns </td><td>GC6 </td><td>−1.00 </td><td>−1.00 </td><td>−2.00 </td><td>−2.00 </td><td>−1.00 </td><td>−1.00 </td><td>−2.00 </td><td>−2.00 </td><td>−1.00 </td><td>−4.00 </td><td>−2.00 </td><td>−1.20 </td><td>0.00 </td><td>0.00 </td><td>−0.77 </td><td>−1.50 </td><td>−1.00 </td><td>−0.50 </td><td>−2.75 </td><td>−0.25 </td><td>0.00 </td><td>−2.00 </td><td>−1.00 </td></tr><tr><td>GC8 </td><td>−0.50 </td><td>−1.00 </td><td>−1.00 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>−3.00 </td><td>−0.25 </td><td>0.00 </td><td>−2.00 </td><td>−1.34 </td><td>−1.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.50 </td><td>0.00 </td><td>0.00 </td><td>−3.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>RY12 </td><td>−2.50 </td><td>−2.00 </td><td>−3.00 </td><td>−1.00 </td><td>−0.89 </td><td>−0.33 </td><td>−2.00 </td><td>−2.00 </td><td>−0.35 </td><td>−2.00 </td><td>−3.25 </td><td>−0.25 </td><td>0.00 </td><td>0.42 </td><td>−0.09 </td><td>−0.75 </td><td>0.00 </td><td>−1.00 </td><td>−3.00 </td><td>0.00 </td><td>0.00 </td><td>−0.72 </td><td>0.00 </td></tr><tr><td>RY12e1 </td><td>−2.00 </td><td>−1.40 </td><td>−3.00 </td><td>−1.00 </td><td>−1.00 </td><td>−0.25 </td><td>−2.00 </td><td>−3.00 </td><td>0.00 </td><td>−3.00 </td><td>−2.25 </td><td>0.00 </td><td>0.00 </td><td>0.84 </td><td>0.05 </td><td>−0.75 </td><td>0.00 </td><td>−1.00 </td><td>−3.00 </td><td>0.50 </td><td>0.00 </td><td>−0.64 </td><td>−0.98 </td></tr><tr><td>RY18e2 </td><td>−2.65 </td><td>−2.00 </td><td>−2.53 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>−2.00 </td><td>−2.00 </td><td>0.00 </td><td>−2.00 </td><td>−3.00 </td><td>−1.00 </td><td>0.00 </td><td>0.75 </td><td>0.00 </td><td>−0.75 </td><td>0.00 </td><td>0.00 </td><td>−3.50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>RY24e3 </td><td>−1.03 </td><td>−1.50 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−1.00 </td><td>−1.00 </td><td>0.00 </td><td>−1.00 </td><td>−1.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.17 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td rowspan="3">DNA bending </td><td>bend45w60 </td><td>0.28 </td><td>0.00 </td><td>0.07 </td><td>1.00 </td><td>3.00 </td><td>2.00 </td><td>0.00 </td><td>1.50 </td><td>2.00 </td><td>1.00 </td><td>0.17 </td><td>0.45 </td><td>2.00 </td><td>1.50 </td><td>0.50 </td><td>0.00 </td><td>0.00 </td><td>2.50 </td><td>0.00 </td><td>2.00 </td><td>2.00 </td><td>2.00 </td><td>0.00 </td></tr><tr><td>bend60w100 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.50 </td><td>2.97 </td><td>2.00 </td><td>0.00 </td><td>1.00 </td><td>2.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>2.00 </td><td>0.75 </td><td>0.25 </td><td>0.00 </td><td>0.00 </td><td>3.00 </td><td>0.00 </td><td>2.25 </td><td>3.00 </td><td>2.00 </td><td>0.00 </td></tr><tr><td>bend90w120 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>3.00 </td><td>1.00 </td><td>0.00 </td><td>1.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>3.00 </td><td>0.00 </td><td>0.20 </td><td>0.00 </td><td>0.00 </td><td>3.50 </td><td>0.00 </td><td>2.00 </td><td>3.00 </td><td>1.00 </td><td>0.00 </td></tr></tbody></table></div><div content-id="DST057TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Pattern name </th><th rowspan="2">Pattern code </th><th>AlPr </th><th>BePr </th><th>GaPr </th><th>DePr </th><th>EpPr </th><th>Firm </th><th>Acti </th><th>Cyan </th><th>Bact </th><th>Chlb </th><th>Chlf </th><th>Dein </th><th>Fuso </th><th>Chla </th><th>Spir </th><th>Acid </th><th>Verr </th><th>Defe </th><th>Plan </th><th>Aqui </th><th>Ther </th><th>Eury </th><th>Cren </th></tr><tr><th>58 </th><th>39 </th><th>76 </th><th>23 </th><th>11 </th><th>61 </th><th>63 </th><th>12 </th><th>34 </th><th>5 </th><th>8 </th><th>6 </th><th>5 </th><th>6 </th><th>4 </th><th>4 </th><th>4 </th><th>4 </th><th>4 </th><th>8 </th><th>5 </th><th>44 </th><th>16 </th></tr></thead><tbody><tr><td rowspan="11">Simple sequence repeats </td><td>1n8 </td><td>−4.00 </td><td>−3.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−1.67 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−3.79 </td><td>−3.75 </td><td>−3.00 </td><td>−3.00 </td><td>−4.00 </td><td>−3.50 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td></tr><tr><td>2n5 </td><td>−2.21 </td><td>−2.00 </td><td>−3.00 </td><td>−3.00 </td><td>−4.00 </td><td>−4.00 </td><td>−3.00 </td><td>−3.63 </td><td>−4.00 </td><td>−3.00 </td><td>−3.00 </td><td>−2.60 </td><td>−4.00 </td><td>−4.00 </td><td>−3.50 </td><td>−2.00 </td><td>−2.50 </td><td>−4.00 </td><td>−2.00 </td><td>−4.00 </td><td>−4.00 </td><td>−3.00 </td><td>−3.00 </td></tr><tr><td>3n4 </td><td>−0.78 </td><td>−2.00 </td><td>−2.09 </td><td>−1.00 </td><td>−3.00 </td><td>−3.00 </td><td>−2.00 </td><td>−1.50 </td><td>−2.67 </td><td>−2.00 </td><td>−1.50 </td><td>−1.50 </td><td>−4.00 </td><td>−3.00 </td><td>−3.17 </td><td>−0.25 </td><td>−1.00 </td><td>−3.50 </td><td>−1.00 </td><td>−3.50 </td><td>−4.00 </td><td>−3.00 </td><td>−3.00 </td></tr><tr><td>4n4 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>5n4 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.07 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>6n3 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.07 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>7n3 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.75 </td><td>0.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.50 </td><td>1.00 </td><td>2.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>8n3 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.50 </td><td>0.00 </td><td>0.00 </td><td>0.50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>9n2 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.36 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>10n2 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.17 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>11n2 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.59 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td rowspan="5">Close direct repeats </td><td>4n6g12 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.13 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>6n6g24 </td><td>0.42 </td><td>0.25 </td><td>0.88 </td><td>1.00 </td><td>0.00 </td><td>0.30 </td><td>2.00 </td><td>1.50 </td><td>1.00 </td><td>1.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.98 </td><td>1.50 </td><td>0.00 </td><td>1.50 </td><td>1.25 </td><td>0.00 </td><td>0.00 </td><td>0.10 </td><td>0.00 </td></tr><tr><td>8n4g24 </td><td>0.50 </td><td>1.00 </td><td>1.14 </td><td>2.00 </td><td>0.00 </td><td>1.00 </td><td>3.00 </td><td>3.00 </td><td>1.50 </td><td>2.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.25 </td><td>2.25 </td><td>0.50 </td><td>1.50 </td><td>2.75 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>0.00 </td></tr><tr><td>cd8g6 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>1.00 </td><td>0.00 </td><td>2.00 </td><td>1.92 </td><td>0.00 </td><td>0.00 </td><td>0.75 </td><td>1.20 </td><td>0.00 </td><td>0.00 </td><td>0.02 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>0.60 </td><td>1.00 </td></tr><tr><td>cd10g50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>0.60 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.19 </td><td>0.15 </td><td>0.00 </td><td>0.00 </td><td>−1.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td rowspan="5">Palindromes and inverted repeats </td><td>cp8g6 </td><td>3.00 </td><td>3.00 </td><td>4.00 </td><td>3.50 </td><td>3.00 </td><td>4.00 </td><td>3.00 </td><td>1.81 </td><td>3.00 </td><td>2.00 </td><td>4.00 </td><td>2.63 </td><td>3.00 </td><td>4.00 </td><td>2.86 </td><td>3.50 </td><td>4.00 </td><td>3.00 </td><td>1.50 </td><td>2.00 </td><td>4.00 </td><td>2.00 </td><td>2.00 </td></tr><tr><td>cp10g50 </td><td>3.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>3.83 </td><td>4.00 </td><td>4.00 </td><td>3.54 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>3.75 </td><td>4.00 </td><td>4.00 </td><td>3.69 </td><td>3.50 </td><td>2.00 </td><td>3.00 </td><td>3.50 </td><td>1.00 </td><td>4.00 </td><td>1.00 </td><td>1.00 </td></tr><tr><td>pals9 </td><td>1.09 </td><td>2.00 </td><td>3.35 </td><td>3.00 </td><td>2.00 </td><td>4.00 </td><td>4.00 </td><td>2.00 </td><td>4.00 </td><td>3.00 </td><td>3.50 </td><td>3.00 </td><td>3.00 </td><td>3.96 </td><td>2.17 </td><td>1.50 </td><td>2.00 </td><td>1.50 </td><td>1.00 </td><td>0.00 </td><td>3.50 </td><td>0.10 </td><td>0.00 </td></tr><tr><td>pals9g12 </td><td>3.66 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>3.72 </td><td>4.00 </td><td>4.00 </td><td>3.71 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>3.75 </td><td>4.00 </td><td>4.00 </td><td>3.77 </td><td>4.00 </td><td>4.00 </td><td>3.00 </td><td>4.00 </td><td>1.00 </td><td>4.00 </td><td>2.00 </td><td>1.00 </td></tr><tr><td>pals12g20 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>3.77 </td><td>4.00 </td><td>4.00 </td><td>3.00 </td><td>4.00 </td><td>0.00 </td><td>4.00 </td><td>1.00 </td><td>0.22 </td></tr><tr><td rowspan="10">H-DNA-related patterns </td><td>cm8g6 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.34 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>cm10g50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.34 </td><td>0.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>mirs9 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>mirs9g12 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.25 </td><td>0.17 </td><td>0.00 </td><td>0.00 </td><td>0.34 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.25 </td><td>0.50 </td><td>0.00 </td><td>0.00 </td><td>0.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>mirs12g20 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.09 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.13 </td><td>0.00 </td><td>0.50 </td><td>0.00 </td><td>0.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>R15 </td><td>−0.20 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−1.00 </td><td>−0.20 </td><td>0.00 </td><td>−1.07 </td><td>0.00 </td><td>0.00 </td><td>0.25 </td><td>−1.00 </td><td>−4.00 </td><td>0.00 </td><td>−0.90 </td><td>−0.50 </td><td>1.00 </td><td>−3.00 </td><td>0.00 </td><td>−2.00 </td><td>−3.00 </td><td>−1.00 </td><td>−1.00 </td></tr><tr><td>R30 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.38 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>R30e3 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−2.00 </td><td>−0.25 </td><td>−0.13 </td><td>0.00 </td><td>0.50 </td><td>−1.00 </td><td>0.00 </td><td>−0.50 </td><td>−1.50 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>R45e6 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−2.00 </td><td>0.00 </td><td>0.15 </td><td>0.00 </td><td>0.50 </td><td>0.00 </td><td>0.50 </td><td>−0.50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>R60e9 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.07 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td rowspan="3">G-DNA-related patterns </td><td>GG8g4 </td><td>1.00 </td><td>1.33 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>2.70 </td><td>0.00 </td><td>0.00 </td><td>0.54 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>1.50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>GGG4g6 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.63 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.31 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>−0.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.50 </td><td>0.00 </td><td>−0.42 </td><td>0.00 </td></tr><tr><td>GGGG4g6 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.17 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td rowspan="6">Z-DNA-related patterns </td><td>GC6 </td><td>−1.00 </td><td>−1.00 </td><td>−2.00 </td><td>−2.00 </td><td>−1.00 </td><td>−1.00 </td><td>−2.00 </td><td>−2.00 </td><td>−1.00 </td><td>−4.00 </td><td>−2.00 </td><td>−1.20 </td><td>0.00 </td><td>0.00 </td><td>−0.77 </td><td>−1.50 </td><td>−1.00 </td><td>−0.50 </td><td>−2.75 </td><td>−0.25 </td><td>0.00 </td><td>−2.00 </td><td>−1.00 </td></tr><tr><td>GC8 </td><td>−0.50 </td><td>−1.00 </td><td>−1.00 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>−3.00 </td><td>−0.25 </td><td>0.00 </td><td>−2.00 </td><td>−1.34 </td><td>−1.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.50 </td><td>0.00 </td><td>0.00 </td><td>−3.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>RY12 </td><td>−2.50 </td><td>−2.00 </td><td>−3.00 </td><td>−1.00 </td><td>−0.89 </td><td>−0.33 </td><td>−2.00 </td><td>−2.00 </td><td>−0.35 </td><td>−2.00 </td><td>−3.25 </td><td>−0.25 </td><td>0.00 </td><td>0.42 </td><td>−0.09 </td><td>−0.75 </td><td>0.00 </td><td>−1.00 </td><td>−3.00 </td><td>0.00 </td><td>0.00 </td><td>−0.72 </td><td>0.00 </td></tr><tr><td>RY12e1 </td><td>−2.00 </td><td>−1.40 </td><td>−3.00 </td><td>−1.00 </td><td>−1.00 </td><td>−0.25 </td><td>−2.00 </td><td>−3.00 </td><td>0.00 </td><td>−3.00 </td><td>−2.25 </td><td>0.00 </td><td>0.00 </td><td>0.84 </td><td>0.05 </td><td>−0.75 </td><td>0.00 </td><td>−1.00 </td><td>−3.00 </td><td>0.50 </td><td>0.00 </td><td>−0.64 </td><td>−0.98 </td></tr><tr><td>RY18e2 </td><td>−2.65 </td><td>−2.00 </td><td>−2.53 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>−2.00 </td><td>−2.00 </td><td>0.00 </td><td>−2.00 </td><td>−3.00 </td><td>−1.00 </td><td>0.00 </td><td>0.75 </td><td>0.00 </td><td>−0.75 </td><td>0.00 </td><td>0.00 </td><td>−3.50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>RY24e3 </td><td>−1.03 </td><td>−1.50 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−1.00 </td><td>−1.00 </td><td>0.00 </td><td>−1.00 </td><td>−1.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.17 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td rowspan="3">DNA bending </td><td>bend45w60 </td><td>0.28 </td><td>0.00 </td><td>0.07 </td><td>1.00 </td><td>3.00 </td><td>2.00 </td><td>0.00 </td><td>1.50 </td><td>2.00 </td><td>1.00 </td><td>0.17 </td><td>0.45 </td><td>2.00 </td><td>1.50 </td><td>0.50 </td><td>0.00 </td><td>0.00 </td><td>2.50 </td><td>0.00 </td><td>2.00 </td><td>2.00 </td><td>2.00 </td><td>0.00 </td></tr><tr><td>bend60w100 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.50 </td><td>2.97 </td><td>2.00 </td><td>0.00 </td><td>1.00 </td><td>2.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>2.00 </td><td>0.75 </td><td>0.25 </td><td>0.00 </td><td>0.00 </td><td>3.00 </td><td>0.00 </td><td>2.25 </td><td>3.00 </td><td>2.00 </td><td>0.00 </td></tr><tr><td>bend90w120 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>3.00 </td><td>1.00 </td><td>0.00 </td><td>1.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>3.00 </td><td>0.00 </td><td>0.20 </td><td>0.00 </td><td>0.00 </td><td>3.50 </td><td>0.00 </td><td>2.00 </td><td>3.00 </td><td>1.00 </td><td>0.00 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn2"><div class="fn"><p>Numbers in the table refer to the medians of pattern representation among all genera of the corresponding phylum. The pattern representations were categorized into nine categories from −4 (extremely under-represented), through 0 (normally represented), to +4 (extremely over-represented). See Materials and methods for details. A colour version of this table is presented in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Materials as Table S8</a>. Codes in the second column refer to specific sequence patterns (see Table <a reveal-id="DST057TB1" data-open="DST057TB1" class="link link-reveal link-table xref-fig">1</a>). Columns represent different phyla abbreviated as follows: AlPr, α-proteobacteria; BePr, β-proteobacteria; GaPr, γ-proteobacteria; DePr, δ-proteobacteria; EpPr, ɛ-proteobacteria; Firm, Firmicutes; Acti, Actinobacteria; Cyan, Cyanobacteria; Bact, Bacteroidetes; Chlb, Chlorobi; Chlf, Chloroflexi; Dein, Deinococcus-Thermus; Fuso, Fusobacteria; Chla, Chlamydiae; Spir, Spirochaetes; Acid, Acidobacteria; Verr, Verrucomicrobia; Defe, Deferribacteres; Plan, Planctomycetes; Aqui, Aquificales; Ther, Thermotogae; Eury, Euryarchaeota; Cren, Crenarchaeota. Numbers in the second row indicate the number of genera available for each phylum. Only phyla represented by three or more genera are shown.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5813728" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST057TB2" class="table-wrap-title" data-id="DST057TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Representation of sequence patterns in different phyla</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Pattern name </th><th rowspan="2">Pattern code </th><th>AlPr </th><th>BePr </th><th>GaPr </th><th>DePr </th><th>EpPr </th><th>Firm </th><th>Acti </th><th>Cyan </th><th>Bact </th><th>Chlb </th><th>Chlf </th><th>Dein </th><th>Fuso </th><th>Chla </th><th>Spir </th><th>Acid </th><th>Verr </th><th>Defe </th><th>Plan </th><th>Aqui </th><th>Ther </th><th>Eury </th><th>Cren </th></tr><tr><th>58 </th><th>39 </th><th>76 </th><th>23 </th><th>11 </th><th>61 </th><th>63 </th><th>12 </th><th>34 </th><th>5 </th><th>8 </th><th>6 </th><th>5 </th><th>6 </th><th>4 </th><th>4 </th><th>4 </th><th>4 </th><th>4 </th><th>8 </th><th>5 </th><th>44 </th><th>16 </th></tr></thead><tbody><tr><td rowspan="11">Simple sequence repeats </td><td>1n8 </td><td>−4.00 </td><td>−3.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−1.67 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−3.79 </td><td>−3.75 </td><td>−3.00 </td><td>−3.00 </td><td>−4.00 </td><td>−3.50 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td></tr><tr><td>2n5 </td><td>−2.21 </td><td>−2.00 </td><td>−3.00 </td><td>−3.00 </td><td>−4.00 </td><td>−4.00 </td><td>−3.00 </td><td>−3.63 </td><td>−4.00 </td><td>−3.00 </td><td>−3.00 </td><td>−2.60 </td><td>−4.00 </td><td>−4.00 </td><td>−3.50 </td><td>−2.00 </td><td>−2.50 </td><td>−4.00 </td><td>−2.00 </td><td>−4.00 </td><td>−4.00 </td><td>−3.00 </td><td>−3.00 </td></tr><tr><td>3n4 </td><td>−0.78 </td><td>−2.00 </td><td>−2.09 </td><td>−1.00 </td><td>−3.00 </td><td>−3.00 </td><td>−2.00 </td><td>−1.50 </td><td>−2.67 </td><td>−2.00 </td><td>−1.50 </td><td>−1.50 </td><td>−4.00 </td><td>−3.00 </td><td>−3.17 </td><td>−0.25 </td><td>−1.00 </td><td>−3.50 </td><td>−1.00 </td><td>−3.50 </td><td>−4.00 </td><td>−3.00 </td><td>−3.00 </td></tr><tr><td>4n4 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>5n4 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.07 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>6n3 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.07 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>7n3 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.75 </td><td>0.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.50 </td><td>1.00 </td><td>2.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>8n3 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.50 </td><td>0.00 </td><td>0.00 </td><td>0.50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>9n2 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.36 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>10n2 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.17 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>11n2 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.59 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td rowspan="5">Close direct repeats </td><td>4n6g12 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.13 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>6n6g24 </td><td>0.42 </td><td>0.25 </td><td>0.88 </td><td>1.00 </td><td>0.00 </td><td>0.30 </td><td>2.00 </td><td>1.50 </td><td>1.00 </td><td>1.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.98 </td><td>1.50 </td><td>0.00 </td><td>1.50 </td><td>1.25 </td><td>0.00 </td><td>0.00 </td><td>0.10 </td><td>0.00 </td></tr><tr><td>8n4g24 </td><td>0.50 </td><td>1.00 </td><td>1.14 </td><td>2.00 </td><td>0.00 </td><td>1.00 </td><td>3.00 </td><td>3.00 </td><td>1.50 </td><td>2.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.25 </td><td>2.25 </td><td>0.50 </td><td>1.50 </td><td>2.75 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>0.00 </td></tr><tr><td>cd8g6 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>1.00 </td><td>0.00 </td><td>2.00 </td><td>1.92 </td><td>0.00 </td><td>0.00 </td><td>0.75 </td><td>1.20 </td><td>0.00 </td><td>0.00 </td><td>0.02 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>0.60 </td><td>1.00 </td></tr><tr><td>cd10g50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>0.60 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.19 </td><td>0.15 </td><td>0.00 </td><td>0.00 </td><td>−1.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td rowspan="5">Palindromes and inverted repeats </td><td>cp8g6 </td><td>3.00 </td><td>3.00 </td><td>4.00 </td><td>3.50 </td><td>3.00 </td><td>4.00 </td><td>3.00 </td><td>1.81 </td><td>3.00 </td><td>2.00 </td><td>4.00 </td><td>2.63 </td><td>3.00 </td><td>4.00 </td><td>2.86 </td><td>3.50 </td><td>4.00 </td><td>3.00 </td><td>1.50 </td><td>2.00 </td><td>4.00 </td><td>2.00 </td><td>2.00 </td></tr><tr><td>cp10g50 </td><td>3.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>3.83 </td><td>4.00 </td><td>4.00 </td><td>3.54 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>3.75 </td><td>4.00 </td><td>4.00 </td><td>3.69 </td><td>3.50 </td><td>2.00 </td><td>3.00 </td><td>3.50 </td><td>1.00 </td><td>4.00 </td><td>1.00 </td><td>1.00 </td></tr><tr><td>pals9 </td><td>1.09 </td><td>2.00 </td><td>3.35 </td><td>3.00 </td><td>2.00 </td><td>4.00 </td><td>4.00 </td><td>2.00 </td><td>4.00 </td><td>3.00 </td><td>3.50 </td><td>3.00 </td><td>3.00 </td><td>3.96 </td><td>2.17 </td><td>1.50 </td><td>2.00 </td><td>1.50 </td><td>1.00 </td><td>0.00 </td><td>3.50 </td><td>0.10 </td><td>0.00 </td></tr><tr><td>pals9g12 </td><td>3.66 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>3.72 </td><td>4.00 </td><td>4.00 </td><td>3.71 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>3.75 </td><td>4.00 </td><td>4.00 </td><td>3.77 </td><td>4.00 </td><td>4.00 </td><td>3.00 </td><td>4.00 </td><td>1.00 </td><td>4.00 </td><td>2.00 </td><td>1.00 </td></tr><tr><td>pals12g20 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>3.77 </td><td>4.00 </td><td>4.00 </td><td>3.00 </td><td>4.00 </td><td>0.00 </td><td>4.00 </td><td>1.00 </td><td>0.22 </td></tr><tr><td rowspan="10">H-DNA-related patterns </td><td>cm8g6 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.34 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>cm10g50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.34 </td><td>0.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>mirs9 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>mirs9g12 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.25 </td><td>0.17 </td><td>0.00 </td><td>0.00 </td><td>0.34 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.25 </td><td>0.50 </td><td>0.00 </td><td>0.00 </td><td>0.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>mirs12g20 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.09 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.13 </td><td>0.00 </td><td>0.50 </td><td>0.00 </td><td>0.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>R15 </td><td>−0.20 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−1.00 </td><td>−0.20 </td><td>0.00 </td><td>−1.07 </td><td>0.00 </td><td>0.00 </td><td>0.25 </td><td>−1.00 </td><td>−4.00 </td><td>0.00 </td><td>−0.90 </td><td>−0.50 </td><td>1.00 </td><td>−3.00 </td><td>0.00 </td><td>−2.00 </td><td>−3.00 </td><td>−1.00 </td><td>−1.00 </td></tr><tr><td>R30 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.38 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>R30e3 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−2.00 </td><td>−0.25 </td><td>−0.13 </td><td>0.00 </td><td>0.50 </td><td>−1.00 </td><td>0.00 </td><td>−0.50 </td><td>−1.50 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>R45e6 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−2.00 </td><td>0.00 </td><td>0.15 </td><td>0.00 </td><td>0.50 </td><td>0.00 </td><td>0.50 </td><td>−0.50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>R60e9 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.07 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td rowspan="3">G-DNA-related patterns </td><td>GG8g4 </td><td>1.00 </td><td>1.33 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>2.70 </td><td>0.00 </td><td>0.00 </td><td>0.54 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>1.50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>GGG4g6 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.63 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.31 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>−0.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.50 </td><td>0.00 </td><td>−0.42 </td><td>0.00 </td></tr><tr><td>GGGG4g6 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.17 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td rowspan="6">Z-DNA-related patterns </td><td>GC6 </td><td>−1.00 </td><td>−1.00 </td><td>−2.00 </td><td>−2.00 </td><td>−1.00 </td><td>−1.00 </td><td>−2.00 </td><td>−2.00 </td><td>−1.00 </td><td>−4.00 </td><td>−2.00 </td><td>−1.20 </td><td>0.00 </td><td>0.00 </td><td>−0.77 </td><td>−1.50 </td><td>−1.00 </td><td>−0.50 </td><td>−2.75 </td><td>−0.25 </td><td>0.00 </td><td>−2.00 </td><td>−1.00 </td></tr><tr><td>GC8 </td><td>−0.50 </td><td>−1.00 </td><td>−1.00 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>−3.00 </td><td>−0.25 </td><td>0.00 </td><td>−2.00 </td><td>−1.34 </td><td>−1.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.50 </td><td>0.00 </td><td>0.00 </td><td>−3.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>RY12 </td><td>−2.50 </td><td>−2.00 </td><td>−3.00 </td><td>−1.00 </td><td>−0.89 </td><td>−0.33 </td><td>−2.00 </td><td>−2.00 </td><td>−0.35 </td><td>−2.00 </td><td>−3.25 </td><td>−0.25 </td><td>0.00 </td><td>0.42 </td><td>−0.09 </td><td>−0.75 </td><td>0.00 </td><td>−1.00 </td><td>−3.00 </td><td>0.00 </td><td>0.00 </td><td>−0.72 </td><td>0.00 </td></tr><tr><td>RY12e1 </td><td>−2.00 </td><td>−1.40 </td><td>−3.00 </td><td>−1.00 </td><td>−1.00 </td><td>−0.25 </td><td>−2.00 </td><td>−3.00 </td><td>0.00 </td><td>−3.00 </td><td>−2.25 </td><td>0.00 </td><td>0.00 </td><td>0.84 </td><td>0.05 </td><td>−0.75 </td><td>0.00 </td><td>−1.00 </td><td>−3.00 </td><td>0.50 </td><td>0.00 </td><td>−0.64 </td><td>−0.98 </td></tr><tr><td>RY18e2 </td><td>−2.65 </td><td>−2.00 </td><td>−2.53 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>−2.00 </td><td>−2.00 </td><td>0.00 </td><td>−2.00 </td><td>−3.00 </td><td>−1.00 </td><td>0.00 </td><td>0.75 </td><td>0.00 </td><td>−0.75 </td><td>0.00 </td><td>0.00 </td><td>−3.50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>RY24e3 </td><td>−1.03 </td><td>−1.50 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−1.00 </td><td>−1.00 </td><td>0.00 </td><td>−1.00 </td><td>−1.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.17 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td rowspan="3">DNA bending </td><td>bend45w60 </td><td>0.28 </td><td>0.00 </td><td>0.07 </td><td>1.00 </td><td>3.00 </td><td>2.00 </td><td>0.00 </td><td>1.50 </td><td>2.00 </td><td>1.00 </td><td>0.17 </td><td>0.45 </td><td>2.00 </td><td>1.50 </td><td>0.50 </td><td>0.00 </td><td>0.00 </td><td>2.50 </td><td>0.00 </td><td>2.00 </td><td>2.00 </td><td>2.00 </td><td>0.00 </td></tr><tr><td>bend60w100 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.50 </td><td>2.97 </td><td>2.00 </td><td>0.00 </td><td>1.00 </td><td>2.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>2.00 </td><td>0.75 </td><td>0.25 </td><td>0.00 </td><td>0.00 </td><td>3.00 </td><td>0.00 </td><td>2.25 </td><td>3.00 </td><td>2.00 </td><td>0.00 </td></tr><tr><td>bend90w120 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>3.00 </td><td>1.00 </td><td>0.00 </td><td>1.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>3.00 </td><td>0.00 </td><td>0.20 </td><td>0.00 </td><td>0.00 </td><td>3.50 </td><td>0.00 </td><td>2.00 </td><td>3.00 </td><td>1.00 </td><td>0.00 </td></tr></tbody></table></div><div content-id="DST057TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Pattern name </th><th rowspan="2">Pattern code </th><th>AlPr </th><th>BePr </th><th>GaPr </th><th>DePr </th><th>EpPr </th><th>Firm </th><th>Acti </th><th>Cyan </th><th>Bact </th><th>Chlb </th><th>Chlf </th><th>Dein </th><th>Fuso </th><th>Chla </th><th>Spir </th><th>Acid </th><th>Verr </th><th>Defe </th><th>Plan </th><th>Aqui </th><th>Ther </th><th>Eury </th><th>Cren </th></tr><tr><th>58 </th><th>39 </th><th>76 </th><th>23 </th><th>11 </th><th>61 </th><th>63 </th><th>12 </th><th>34 </th><th>5 </th><th>8 </th><th>6 </th><th>5 </th><th>6 </th><th>4 </th><th>4 </th><th>4 </th><th>4 </th><th>4 </th><th>8 </th><th>5 </th><th>44 </th><th>16 </th></tr></thead><tbody><tr><td rowspan="11">Simple sequence repeats </td><td>1n8 </td><td>−4.00 </td><td>−3.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−1.67 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−3.79 </td><td>−3.75 </td><td>−3.00 </td><td>−3.00 </td><td>−4.00 </td><td>−3.50 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td><td>−4.00 </td></tr><tr><td>2n5 </td><td>−2.21 </td><td>−2.00 </td><td>−3.00 </td><td>−3.00 </td><td>−4.00 </td><td>−4.00 </td><td>−3.00 </td><td>−3.63 </td><td>−4.00 </td><td>−3.00 </td><td>−3.00 </td><td>−2.60 </td><td>−4.00 </td><td>−4.00 </td><td>−3.50 </td><td>−2.00 </td><td>−2.50 </td><td>−4.00 </td><td>−2.00 </td><td>−4.00 </td><td>−4.00 </td><td>−3.00 </td><td>−3.00 </td></tr><tr><td>3n4 </td><td>−0.78 </td><td>−2.00 </td><td>−2.09 </td><td>−1.00 </td><td>−3.00 </td><td>−3.00 </td><td>−2.00 </td><td>−1.50 </td><td>−2.67 </td><td>−2.00 </td><td>−1.50 </td><td>−1.50 </td><td>−4.00 </td><td>−3.00 </td><td>−3.17 </td><td>−0.25 </td><td>−1.00 </td><td>−3.50 </td><td>−1.00 </td><td>−3.50 </td><td>−4.00 </td><td>−3.00 </td><td>−3.00 </td></tr><tr><td>4n4 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>5n4 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.07 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>6n3 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.07 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>7n3 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.75 </td><td>0.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.50 </td><td>1.00 </td><td>2.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>8n3 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.50 </td><td>0.00 </td><td>0.00 </td><td>0.50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>9n2 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.36 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>10n2 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.17 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>11n2 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.59 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td rowspan="5">Close direct repeats </td><td>4n6g12 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.13 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>6n6g24 </td><td>0.42 </td><td>0.25 </td><td>0.88 </td><td>1.00 </td><td>0.00 </td><td>0.30 </td><td>2.00 </td><td>1.50 </td><td>1.00 </td><td>1.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.98 </td><td>1.50 </td><td>0.00 </td><td>1.50 </td><td>1.25 </td><td>0.00 </td><td>0.00 </td><td>0.10 </td><td>0.00 </td></tr><tr><td>8n4g24 </td><td>0.50 </td><td>1.00 </td><td>1.14 </td><td>2.00 </td><td>0.00 </td><td>1.00 </td><td>3.00 </td><td>3.00 </td><td>1.50 </td><td>2.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.25 </td><td>2.25 </td><td>0.50 </td><td>1.50 </td><td>2.75 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>0.00 </td></tr><tr><td>cd8g6 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>1.00 </td><td>0.00 </td><td>2.00 </td><td>1.92 </td><td>0.00 </td><td>0.00 </td><td>0.75 </td><td>1.20 </td><td>0.00 </td><td>0.00 </td><td>0.02 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>0.60 </td><td>1.00 </td></tr><tr><td>cd10g50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>0.60 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.19 </td><td>0.15 </td><td>0.00 </td><td>0.00 </td><td>−1.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td rowspan="5">Palindromes and inverted repeats </td><td>cp8g6 </td><td>3.00 </td><td>3.00 </td><td>4.00 </td><td>3.50 </td><td>3.00 </td><td>4.00 </td><td>3.00 </td><td>1.81 </td><td>3.00 </td><td>2.00 </td><td>4.00 </td><td>2.63 </td><td>3.00 </td><td>4.00 </td><td>2.86 </td><td>3.50 </td><td>4.00 </td><td>3.00 </td><td>1.50 </td><td>2.00 </td><td>4.00 </td><td>2.00 </td><td>2.00 </td></tr><tr><td>cp10g50 </td><td>3.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>3.83 </td><td>4.00 </td><td>4.00 </td><td>3.54 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>3.75 </td><td>4.00 </td><td>4.00 </td><td>3.69 </td><td>3.50 </td><td>2.00 </td><td>3.00 </td><td>3.50 </td><td>1.00 </td><td>4.00 </td><td>1.00 </td><td>1.00 </td></tr><tr><td>pals9 </td><td>1.09 </td><td>2.00 </td><td>3.35 </td><td>3.00 </td><td>2.00 </td><td>4.00 </td><td>4.00 </td><td>2.00 </td><td>4.00 </td><td>3.00 </td><td>3.50 </td><td>3.00 </td><td>3.00 </td><td>3.96 </td><td>2.17 </td><td>1.50 </td><td>2.00 </td><td>1.50 </td><td>1.00 </td><td>0.00 </td><td>3.50 </td><td>0.10 </td><td>0.00 </td></tr><tr><td>pals9g12 </td><td>3.66 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>3.72 </td><td>4.00 </td><td>4.00 </td><td>3.71 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>3.75 </td><td>4.00 </td><td>4.00 </td><td>3.77 </td><td>4.00 </td><td>4.00 </td><td>3.00 </td><td>4.00 </td><td>1.00 </td><td>4.00 </td><td>2.00 </td><td>1.00 </td></tr><tr><td>pals12g20 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>4.00 </td><td>3.77 </td><td>4.00 </td><td>4.00 </td><td>3.00 </td><td>4.00 </td><td>0.00 </td><td>4.00 </td><td>1.00 </td><td>0.22 </td></tr><tr><td rowspan="10">H-DNA-related patterns </td><td>cm8g6 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.34 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>cm10g50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.34 </td><td>0.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>mirs9 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>mirs9g12 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.25 </td><td>0.17 </td><td>0.00 </td><td>0.00 </td><td>0.34 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.25 </td><td>0.50 </td><td>0.00 </td><td>0.00 </td><td>0.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>mirs12g20 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.09 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.13 </td><td>0.00 </td><td>0.50 </td><td>0.00 </td><td>0.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>R15 </td><td>−0.20 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−1.00 </td><td>−0.20 </td><td>0.00 </td><td>−1.07 </td><td>0.00 </td><td>0.00 </td><td>0.25 </td><td>−1.00 </td><td>−4.00 </td><td>0.00 </td><td>−0.90 </td><td>−0.50 </td><td>1.00 </td><td>−3.00 </td><td>0.00 </td><td>−2.00 </td><td>−3.00 </td><td>−1.00 </td><td>−1.00 </td></tr><tr><td>R30 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.38 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>R30e3 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−2.00 </td><td>−0.25 </td><td>−0.13 </td><td>0.00 </td><td>0.50 </td><td>−1.00 </td><td>0.00 </td><td>−0.50 </td><td>−1.50 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>R45e6 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−2.00 </td><td>0.00 </td><td>0.15 </td><td>0.00 </td><td>0.50 </td><td>0.00 </td><td>0.50 </td><td>−0.50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>R60e9 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.07 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td rowspan="3">G-DNA-related patterns </td><td>GG8g4 </td><td>1.00 </td><td>1.33 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>1.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>2.70 </td><td>0.00 </td><td>0.00 </td><td>0.54 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>1.50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>GGG4g6 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.63 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.31 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>−0.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.50 </td><td>0.00 </td><td>−0.42 </td><td>0.00 </td></tr><tr><td>GGGG4g6 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.17 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td rowspan="6">Z-DNA-related patterns </td><td>GC6 </td><td>−1.00 </td><td>−1.00 </td><td>−2.00 </td><td>−2.00 </td><td>−1.00 </td><td>−1.00 </td><td>−2.00 </td><td>−2.00 </td><td>−1.00 </td><td>−4.00 </td><td>−2.00 </td><td>−1.20 </td><td>0.00 </td><td>0.00 </td><td>−0.77 </td><td>−1.50 </td><td>−1.00 </td><td>−0.50 </td><td>−2.75 </td><td>−0.25 </td><td>0.00 </td><td>−2.00 </td><td>−1.00 </td></tr><tr><td>GC8 </td><td>−0.50 </td><td>−1.00 </td><td>−1.00 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>−3.00 </td><td>−0.25 </td><td>0.00 </td><td>−2.00 </td><td>−1.34 </td><td>−1.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.50 </td><td>0.00 </td><td>0.00 </td><td>−3.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>RY12 </td><td>−2.50 </td><td>−2.00 </td><td>−3.00 </td><td>−1.00 </td><td>−0.89 </td><td>−0.33 </td><td>−2.00 </td><td>−2.00 </td><td>−0.35 </td><td>−2.00 </td><td>−3.25 </td><td>−0.25 </td><td>0.00 </td><td>0.42 </td><td>−0.09 </td><td>−0.75 </td><td>0.00 </td><td>−1.00 </td><td>−3.00 </td><td>0.00 </td><td>0.00 </td><td>−0.72 </td><td>0.00 </td></tr><tr><td>RY12e1 </td><td>−2.00 </td><td>−1.40 </td><td>−3.00 </td><td>−1.00 </td><td>−1.00 </td><td>−0.25 </td><td>−2.00 </td><td>−3.00 </td><td>0.00 </td><td>−3.00 </td><td>−2.25 </td><td>0.00 </td><td>0.00 </td><td>0.84 </td><td>0.05 </td><td>−0.75 </td><td>0.00 </td><td>−1.00 </td><td>−3.00 </td><td>0.50 </td><td>0.00 </td><td>−0.64 </td><td>−0.98 </td></tr><tr><td>RY18e2 </td><td>−2.65 </td><td>−2.00 </td><td>−2.53 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>−2.00 </td><td>−2.00 </td><td>0.00 </td><td>−2.00 </td><td>−3.00 </td><td>−1.00 </td><td>0.00 </td><td>0.75 </td><td>0.00 </td><td>−0.75 </td><td>0.00 </td><td>0.00 </td><td>−3.50 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td>RY24e3 </td><td>−1.03 </td><td>−1.50 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−1.00 </td><td>−1.00 </td><td>0.00 </td><td>−1.00 </td><td>−1.25 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>−0.17 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>−1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td></tr><tr><td rowspan="3">DNA bending </td><td>bend45w60 </td><td>0.28 </td><td>0.00 </td><td>0.07 </td><td>1.00 </td><td>3.00 </td><td>2.00 </td><td>0.00 </td><td>1.50 </td><td>2.00 </td><td>1.00 </td><td>0.17 </td><td>0.45 </td><td>2.00 </td><td>1.50 </td><td>0.50 </td><td>0.00 </td><td>0.00 </td><td>2.50 </td><td>0.00 </td><td>2.00 </td><td>2.00 </td><td>2.00 </td><td>0.00 </td></tr><tr><td>bend60w100 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.50 </td><td>2.97 </td><td>2.00 </td><td>0.00 </td><td>1.00 </td><td>2.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>2.00 </td><td>0.75 </td><td>0.25 </td><td>0.00 </td><td>0.00 </td><td>3.00 </td><td>0.00 </td><td>2.25 </td><td>3.00 </td><td>2.00 </td><td>0.00 </td></tr><tr><td>bend90w120 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>3.00 </td><td>1.00 </td><td>0.00 </td><td>1.00 </td><td>1.00 </td><td>0.00 </td><td>0.00 </td><td>0.00 </td><td>3.00 </td><td>0.00 </td><td>0.20 </td><td>0.00 </td><td>0.00 </td><td>3.50 </td><td>0.00 </td><td>2.00 </td><td>3.00 </td><td>1.00 </td><td>0.00 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn2"><div class="fn"><p>Numbers in the table refer to the medians of pattern representation among all genera of the corresponding phylum. The pattern representations were categorized into nine categories from −4 (extremely under-represented), through 0 (normally represented), to +4 (extremely over-represented). See Materials and methods for details. A colour version of this table is presented in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Materials as Table S8</a>. Codes in the second column refer to specific sequence patterns (see Table <a reveal-id="DST057TB1" data-open="DST057TB1" class="link link-reveal link-table xref-fig">1</a>). Columns represent different phyla abbreviated as follows: AlPr, α-proteobacteria; BePr, β-proteobacteria; GaPr, γ-proteobacteria; DePr, δ-proteobacteria; EpPr, ɛ-proteobacteria; Firm, Firmicutes; Acti, Actinobacteria; Cyan, Cyanobacteria; Bact, Bacteroidetes; Chlb, Chlorobi; Chlf, Chloroflexi; Dein, Deinococcus-Thermus; Fuso, Fusobacteria; Chla, Chlamydiae; Spir, Spirochaetes; Acid, Acidobacteria; Verr, Verrucomicrobia; Defe, Deferribacteres; Plan, Planctomycetes; Aqui, Aquificales; Ther, Thermotogae; Eury, Euryarchaeota; Cren, Crenarchaeota. Numbers in the second row indicate the number of genera available for each phylum. Only phyla represented by three or more genera are shown.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5813728" target="_blank">View Large</a></div></div><p>Mono- and dinucleotide repeats are about equally suppressed in psychrophiles, mesophiles, thermophiles, and hyperthermophiles (Table <a reveal-id="DST057TB3" data-open="DST057TB3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S12</a>). However, the suppression of trinucleotide tandem repeats is more pronounced in thermophiles and hyperthermophiles than in mesophiles and psychrophiles (significant at <em>P</em> &lt; 10<sup>−5</sup>; the <em>P</em>-value is based on Fisher's exact test for 2-by-2 contingency tables). Oxygen requirement appears to have no relationship to tandem repeat representations. </p>                        <a id="5813731" scrollto-destination="5813731"></a>
<div content-id="DST057TB3" class="table-modal"><div class="table-wrap"><div id="DST057TB3" class="table-wrap-title" data-id="DST057TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Representation of sequence patterns in different OGT and oxygen requirement classes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Pattern name </th><th rowspan="2">Pattern code </th><th>Psychrophile </th><th>Mesophile </th><th>Thermophile </th><th>Hyperthermophile </th><th>Anaerobe </th><th>Aerobe </th><th>Facultative </th><th>Microaerophile </th></tr><tr><th>18 </th><th>382 </th><th>72 </th><th>26 </th><th>159 </th><th>202 </th><th>95 </th><th>9 </th></tr></thead><tbody><tr><td rowspan="11">Simple sequence repeats </td><td>1n8 </td><td>−3.83 </td><td>−3.54 </td><td>−3.89 </td><td>−4.00 </td><td>−3.73 </td><td>−3.61 </td><td>−3.65 </td><td>−3.67 </td></tr><tr><td>2n5 </td><td>−3.15 </td><td>−2.89 </td><td>−3.40 </td><td>−3.45 </td><td>−3.30 </td><td>−2.74 </td><td>−3.02 </td><td>−3.11 </td></tr><tr><td>3n4 </td><td>−1.93 </td><td>−1.92 </td><td>−2.83 </td><td>−3.09 </td><td>−2.56 </td><td>−1.75 </td><td>−2.15 </td><td>−2.33 </td></tr><tr><td>4n4 </td><td>−0.06 </td><td>0.01 </td><td>−0.06 </td><td>−0.08 </td><td>−0.04 </td><td>0.01 </td><td>0.00 </td><td>−0.10 </td></tr><tr><td>5n4 </td><td>0.00 </td><td>0.07 </td><td>0.00 </td><td>0.00 </td><td>0.02 </td><td>0.10 </td><td>0.01 </td><td>0.11 </td></tr><tr><td>6n3 </td><td>0.20 </td><td>0.19 </td><td>0.06 </td><td>−0.03 </td><td>0.05 </td><td>0.25 </td><td>0.08 </td><td>0.31 </td></tr><tr><td>7n3 </td><td>1.27 </td><td>0.48 </td><td>0.45 </td><td>0.00 </td><td>0.42 </td><td>0.59 </td><td>0.35 </td><td>0.89 </td></tr><tr><td>8n3 </td><td>0.56 </td><td>0.30 </td><td>0.12 </td><td>0.01 </td><td>0.25 </td><td>0.27 </td><td>0.20 </td><td>0.33 </td></tr><tr><td>9n2 </td><td>−0.08 </td><td>0.09 </td><td>0.22 </td><td>0.04 </td><td>0.09 </td><td>0.19 </td><td>−0.11 </td><td>−0.11 </td></tr><tr><td>10n2 </td><td>0.09 </td><td>0.25 </td><td>0.25 </td><td>0.28 </td><td>0.27 </td><td>0.24 </td><td>0.16 </td><td>0.00 </td></tr><tr><td>11n2 </td><td>0.21 </td><td>0.32 </td><td>0.30 </td><td>0.40 </td><td>0.42 </td><td>0.26 </td><td>0.24 </td><td>0.03 </td></tr><tr><td rowspan="5">Close direct repeats </td><td>4n6g12 </td><td>0.45 </td><td>0.58 </td><td>0.48 </td><td>0.82 </td><td>0.55 </td><td>0.73 </td><td>0.31 </td><td>0.50 </td></tr><tr><td>6n6g24 </td><td>1.35 </td><td>1.16 </td><td>0.71 </td><td>0.34 </td><td>1.00 </td><td>1.21 </td><td>0.81 </td><td>1.39 </td></tr><tr><td>8n4g24 </td><td>2.02 </td><td>1.61 </td><td>0.80 </td><td>0.28 </td><td>1.27 </td><td>1.69 </td><td>1.18 </td><td>1.13 </td></tr><tr><td>cd8g6 </td><td>0.39 </td><td>0.73 </td><td>0.80 </td><td>1.20 </td><td>0.87 </td><td>0.82 </td><td>0.41 </td><td>0.70 </td></tr><tr><td>cd10g50 </td><td>0.46 </td><td>0.41 </td><td>0.05 </td><td>−0.07 </td><td>0.25 </td><td>0.60 </td><td>0.01 </td><td>0.07 </td></tr><tr><td rowspan="5">Palindromes and inverted repeats </td><td>cp8g6 </td><td>3.57 </td><td>2.78 </td><td>2.81 </td><td>1.99 </td><td>2.98 </td><td>2.39 </td><td>3.21 </td><td>3.22 </td></tr><tr><td>cp10g50 </td><td>3.56 </td><td>3.26 </td><td>2.98 </td><td>1.64 </td><td>3.15 </td><td>3.03 </td><td>3.45 </td><td>3.20 </td></tr><tr><td>pals9 </td><td>3.39 </td><td>2.58 </td><td>2.13 </td><td>0.71 </td><td>2.47 </td><td>2.27 </td><td>2.89 </td><td>2.47 </td></tr><tr><td>pals9g12 </td><td>3.83 </td><td>3.46 </td><td>3.05 </td><td>1.79 </td><td>3.33 </td><td>3.22 </td><td>3.55 </td><td>3.64 </td></tr><tr><td>pals12g20 </td><td>3.72 </td><td>3.58 </td><td>2.97 </td><td>1.17 </td><td>3.16 </td><td>3.47 </td><td>3.62 </td><td>3.00 </td></tr><tr><td rowspan="10">H-DNA-related patterns </td><td>cm8g6 </td><td>0.02 </td><td>0.20 </td><td>0.21 </td><td>0.33 </td><td>0.24 </td><td>0.24 </td><td>0.15 </td><td>0.04 </td></tr><tr><td>cm10g50 </td><td>0.02 </td><td>0.16 </td><td>0.10 </td><td>0.15 </td><td>0.16 </td><td>0.18 </td><td>0.10 </td><td>0.00 </td></tr><tr><td>mirs9 </td><td>0.00 </td><td>0.13 </td><td>0.19 </td><td>0.17 </td><td>0.14 </td><td>0.18 </td><td>0.08 </td><td>−0.02 </td></tr><tr><td>mirs9g12 </td><td>0.15 </td><td>0.30 </td><td>0.35 </td><td>0.44 </td><td>0.33 </td><td>0.38 </td><td>0.20 </td><td>0.00 </td></tr><tr><td>mirs12g20 </td><td>0.15 </td><td>0.27 </td><td>0.28 </td><td>0.20 </td><td>0.26 </td><td>0.33 </td><td>0.16 </td><td>0.22 </td></tr><tr><td>R15 </td><td>−0.47 </td><td>−0.52 </td><td>−0.96 </td><td>−1.77 </td><td>−0.91 </td><td>−0.55 </td><td>−0.34 </td><td>−1.78 </td></tr><tr><td>R30 </td><td>0.03 </td><td>0.07 </td><td>0.08 </td><td>0.11 </td><td>0.10 </td><td>0.04 </td><td>0.07 </td><td>0.00 </td></tr><tr><td>R30e3 </td><td>0.08 </td><td>−0.01 </td><td>−0.18 </td><td>−0.66 </td><td>−0.23 </td><td>0.00 </td><td>0.14 </td><td>−0.66 </td></tr><tr><td>R45e6 </td><td>0.17 </td><td>0.08 </td><td>−0.03 </td><td>−0.32 </td><td>−0.06 </td><td>0.10 </td><td>0.15 </td><td>−0.02 </td></tr><tr><td>R60e9 </td><td>0.15 </td><td>0.13 </td><td>0.11 </td><td>−0.03 </td><td>0.14 </td><td>0.10 </td><td>0.12 </td><td>0.00 </td></tr><tr><td rowspan="3">G-DNA-related patterns </td><td>GG8g4 </td><td>0.22 </td><td>0.72 </td><td>0.24 </td><td>0.14 </td><td>0.20 </td><td>1.03 </td><td>0.48 </td><td>0.44 </td></tr><tr><td>GGG4g6 </td><td>−0.08 </td><td>−0.26 </td><td>−0.78 </td><td>−0.65 </td><td>−0.67 </td><td>−0.18 </td><td>−0.26 </td><td>−0.22 </td></tr><tr><td>GGGG4g6 </td><td>0.00 </td><td>0.10 </td><td>−0.06 </td><td>−0.12 </td><td>0.01 </td><td>0.11 </td><td>0.02 </td><td>0.11 </td></tr><tr><td rowspan="6">Z-DNA-related patterns </td><td>GC6 </td><td>−1.37 </td><td>−1.50 </td><td>−1.35 </td><td>−0.95 </td><td>−1.41 </td><td>−1.38 </td><td>−1.71 </td><td>−1.42 </td></tr><tr><td>GC8 </td><td>−0.75 </td><td>−1.10 </td><td>−0.84 </td><td>−0.18 </td><td>−0.47 </td><td>−1.42 </td><td>−1.13 </td><td>−0.37 </td></tr><tr><td>RY12 </td><td>−1.79 </td><td>−1.58 </td><td>−0.88 </td><td>−0.16 </td><td>−0.78 </td><td>−1.73 </td><td>−2.03 </td><td>−0.99 </td></tr><tr><td>RY12e1 </td><td>−1.81 </td><td>−1.40 </td><td>−0.71 </td><td>−0.31 </td><td>−0.85 </td><td>−1.43 </td><td>−1.93 </td><td>−0.44 </td></tr><tr><td>RY18e2 </td><td>−1.66 </td><td>−1.45 </td><td>−0.72 </td><td>−0.04 </td><td>−0.58 </td><td>−1.60 </td><td>−2.07 </td><td>−0.37 </td></tr><tr><td>RY24e3 </td><td>−0.60 </td><td>−0.78 </td><td>−0.25 </td><td>0.05 </td><td>−0.30 </td><td>−0.79 </td><td>−1.12 </td><td>−0.44 </td></tr><tr><td rowspan="3">DNA Bending </td><td>bend45w60 </td><td>0.60 </td><td>0.93 </td><td>1.51 </td><td>1.11 </td><td>1.57 </td><td>0.59 </td><td>0.85 </td><td>1.70 </td></tr><tr><td>bend60w100 </td><td>0.32 </td><td>0.79 </td><td>1.40 </td><td>1.17 </td><td>1.42 </td><td>0.50 </td><td>0.74 </td><td>1.48 </td></tr><tr><td>bend90w120 </td><td>0.28 </td><td>0.58 </td><td>1.23 </td><td>0.94 </td><td>1.11 </td><td>0.34 </td><td>0.66 </td><td>1.44 </td></tr></tbody></table></div><div content-id="DST057TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Pattern name </th><th rowspan="2">Pattern code </th><th>Psychrophile </th><th>Mesophile </th><th>Thermophile </th><th>Hyperthermophile </th><th>Anaerobe </th><th>Aerobe </th><th>Facultative </th><th>Microaerophile </th></tr><tr><th>18 </th><th>382 </th><th>72 </th><th>26 </th><th>159 </th><th>202 </th><th>95 </th><th>9 </th></tr></thead><tbody><tr><td rowspan="11">Simple sequence repeats </td><td>1n8 </td><td>−3.83 </td><td>−3.54 </td><td>−3.89 </td><td>−4.00 </td><td>−3.73 </td><td>−3.61 </td><td>−3.65 </td><td>−3.67 </td></tr><tr><td>2n5 </td><td>−3.15 </td><td>−2.89 </td><td>−3.40 </td><td>−3.45 </td><td>−3.30 </td><td>−2.74 </td><td>−3.02 </td><td>−3.11 </td></tr><tr><td>3n4 </td><td>−1.93 </td><td>−1.92 </td><td>−2.83 </td><td>−3.09 </td><td>−2.56 </td><td>−1.75 </td><td>−2.15 </td><td>−2.33 </td></tr><tr><td>4n4 </td><td>−0.06 </td><td>0.01 </td><td>−0.06 </td><td>−0.08 </td><td>−0.04 </td><td>0.01 </td><td>0.00 </td><td>−0.10 </td></tr><tr><td>5n4 </td><td>0.00 </td><td>0.07 </td><td>0.00 </td><td>0.00 </td><td>0.02 </td><td>0.10 </td><td>0.01 </td><td>0.11 </td></tr><tr><td>6n3 </td><td>0.20 </td><td>0.19 </td><td>0.06 </td><td>−0.03 </td><td>0.05 </td><td>0.25 </td><td>0.08 </td><td>0.31 </td></tr><tr><td>7n3 </td><td>1.27 </td><td>0.48 </td><td>0.45 </td><td>0.00 </td><td>0.42 </td><td>0.59 </td><td>0.35 </td><td>0.89 </td></tr><tr><td>8n3 </td><td>0.56 </td><td>0.30 </td><td>0.12 </td><td>0.01 </td><td>0.25 </td><td>0.27 </td><td>0.20 </td><td>0.33 </td></tr><tr><td>9n2 </td><td>−0.08 </td><td>0.09 </td><td>0.22 </td><td>0.04 </td><td>0.09 </td><td>0.19 </td><td>−0.11 </td><td>−0.11 </td></tr><tr><td>10n2 </td><td>0.09 </td><td>0.25 </td><td>0.25 </td><td>0.28 </td><td>0.27 </td><td>0.24 </td><td>0.16 </td><td>0.00 </td></tr><tr><td>11n2 </td><td>0.21 </td><td>0.32 </td><td>0.30 </td><td>0.40 </td><td>0.42 </td><td>0.26 </td><td>0.24 </td><td>0.03 </td></tr><tr><td rowspan="5">Close direct repeats </td><td>4n6g12 </td><td>0.45 </td><td>0.58 </td><td>0.48 </td><td>0.82 </td><td>0.55 </td><td>0.73 </td><td>0.31 </td><td>0.50 </td></tr><tr><td>6n6g24 </td><td>1.35 </td><td>1.16 </td><td>0.71 </td><td>0.34 </td><td>1.00 </td><td>1.21 </td><td>0.81 </td><td>1.39 </td></tr><tr><td>8n4g24 </td><td>2.02 </td><td>1.61 </td><td>0.80 </td><td>0.28 </td><td>1.27 </td><td>1.69 </td><td>1.18 </td><td>1.13 </td></tr><tr><td>cd8g6 </td><td>0.39 </td><td>0.73 </td><td>0.80 </td><td>1.20 </td><td>0.87 </td><td>0.82 </td><td>0.41 </td><td>0.70 </td></tr><tr><td>cd10g50 </td><td>0.46 </td><td>0.41 </td><td>0.05 </td><td>−0.07 </td><td>0.25 </td><td>0.60 </td><td>0.01 </td><td>0.07 </td></tr><tr><td rowspan="5">Palindromes and inverted repeats </td><td>cp8g6 </td><td>3.57 </td><td>2.78 </td><td>2.81 </td><td>1.99 </td><td>2.98 </td><td>2.39 </td><td>3.21 </td><td>3.22 </td></tr><tr><td>cp10g50 </td><td>3.56 </td><td>3.26 </td><td>2.98 </td><td>1.64 </td><td>3.15 </td><td>3.03 </td><td>3.45 </td><td>3.20 </td></tr><tr><td>pals9 </td><td>3.39 </td><td>2.58 </td><td>2.13 </td><td>0.71 </td><td>2.47 </td><td>2.27 </td><td>2.89 </td><td>2.47 </td></tr><tr><td>pals9g12 </td><td>3.83 </td><td>3.46 </td><td>3.05 </td><td>1.79 </td><td>3.33 </td><td>3.22 </td><td>3.55 </td><td>3.64 </td></tr><tr><td>pals12g20 </td><td>3.72 </td><td>3.58 </td><td>2.97 </td><td>1.17 </td><td>3.16 </td><td>3.47 </td><td>3.62 </td><td>3.00 </td></tr><tr><td rowspan="10">H-DNA-related patterns </td><td>cm8g6 </td><td>0.02 </td><td>0.20 </td><td>0.21 </td><td>0.33 </td><td>0.24 </td><td>0.24 </td><td>0.15 </td><td>0.04 </td></tr><tr><td>cm10g50 </td><td>0.02 </td><td>0.16 </td><td>0.10 </td><td>0.15 </td><td>0.16 </td><td>0.18 </td><td>0.10 </td><td>0.00 </td></tr><tr><td>mirs9 </td><td>0.00 </td><td>0.13 </td><td>0.19 </td><td>0.17 </td><td>0.14 </td><td>0.18 </td><td>0.08 </td><td>−0.02 </td></tr><tr><td>mirs9g12 </td><td>0.15 </td><td>0.30 </td><td>0.35 </td><td>0.44 </td><td>0.33 </td><td>0.38 </td><td>0.20 </td><td>0.00 </td></tr><tr><td>mirs12g20 </td><td>0.15 </td><td>0.27 </td><td>0.28 </td><td>0.20 </td><td>0.26 </td><td>0.33 </td><td>0.16 </td><td>0.22 </td></tr><tr><td>R15 </td><td>−0.47 </td><td>−0.52 </td><td>−0.96 </td><td>−1.77 </td><td>−0.91 </td><td>−0.55 </td><td>−0.34 </td><td>−1.78 </td></tr><tr><td>R30 </td><td>0.03 </td><td>0.07 </td><td>0.08 </td><td>0.11 </td><td>0.10 </td><td>0.04 </td><td>0.07 </td><td>0.00 </td></tr><tr><td>R30e3 </td><td>0.08 </td><td>−0.01 </td><td>−0.18 </td><td>−0.66 </td><td>−0.23 </td><td>0.00 </td><td>0.14 </td><td>−0.66 </td></tr><tr><td>R45e6 </td><td>0.17 </td><td>0.08 </td><td>−0.03 </td><td>−0.32 </td><td>−0.06 </td><td>0.10 </td><td>0.15 </td><td>−0.02 </td></tr><tr><td>R60e9 </td><td>0.15 </td><td>0.13 </td><td>0.11 </td><td>−0.03 </td><td>0.14 </td><td>0.10 </td><td>0.12 </td><td>0.00 </td></tr><tr><td rowspan="3">G-DNA-related patterns </td><td>GG8g4 </td><td>0.22 </td><td>0.72 </td><td>0.24 </td><td>0.14 </td><td>0.20 </td><td>1.03 </td><td>0.48 </td><td>0.44 </td></tr><tr><td>GGG4g6 </td><td>−0.08 </td><td>−0.26 </td><td>−0.78 </td><td>−0.65 </td><td>−0.67 </td><td>−0.18 </td><td>−0.26 </td><td>−0.22 </td></tr><tr><td>GGGG4g6 </td><td>0.00 </td><td>0.10 </td><td>−0.06 </td><td>−0.12 </td><td>0.01 </td><td>0.11 </td><td>0.02 </td><td>0.11 </td></tr><tr><td rowspan="6">Z-DNA-related patterns </td><td>GC6 </td><td>−1.37 </td><td>−1.50 </td><td>−1.35 </td><td>−0.95 </td><td>−1.41 </td><td>−1.38 </td><td>−1.71 </td><td>−1.42 </td></tr><tr><td>GC8 </td><td>−0.75 </td><td>−1.10 </td><td>−0.84 </td><td>−0.18 </td><td>−0.47 </td><td>−1.42 </td><td>−1.13 </td><td>−0.37 </td></tr><tr><td>RY12 </td><td>−1.79 </td><td>−1.58 </td><td>−0.88 </td><td>−0.16 </td><td>−0.78 </td><td>−1.73 </td><td>−2.03 </td><td>−0.99 </td></tr><tr><td>RY12e1 </td><td>−1.81 </td><td>−1.40 </td><td>−0.71 </td><td>−0.31 </td><td>−0.85 </td><td>−1.43 </td><td>−1.93 </td><td>−0.44 </td></tr><tr><td>RY18e2 </td><td>−1.66 </td><td>−1.45 </td><td>−0.72 </td><td>−0.04 </td><td>−0.58 </td><td>−1.60 </td><td>−2.07 </td><td>−0.37 </td></tr><tr><td>RY24e3 </td><td>−0.60 </td><td>−0.78 </td><td>−0.25 </td><td>0.05 </td><td>−0.30 </td><td>−0.79 </td><td>−1.12 </td><td>−0.44 </td></tr><tr><td rowspan="3">DNA Bending </td><td>bend45w60 </td><td>0.60 </td><td>0.93 </td><td>1.51 </td><td>1.11 </td><td>1.57 </td><td>0.59 </td><td>0.85 </td><td>1.70 </td></tr><tr><td>bend60w100 </td><td>0.32 </td><td>0.79 </td><td>1.40 </td><td>1.17 </td><td>1.42 </td><td>0.50 </td><td>0.74 </td><td>1.48 </td></tr><tr><td>bend90w120 </td><td>0.28 </td><td>0.58 </td><td>1.23 </td><td>0.94 </td><td>1.11 </td><td>0.34 </td><td>0.66 </td><td>1.44 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn3"><div class="fn"><p>Numbers in the table refer to the average significance category for all genera within each class of organisms. Numbers below the class description indicate numbers of available genera of each class. Anaerobe includes both obligate anaerobes and anaerobes; Aerobe includes both obligate aerobes and aerobes. See <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a> for coloured version of this table.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5813731" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST057TB3" class="table-wrap-title" data-id="DST057TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Representation of sequence patterns in different OGT and oxygen requirement classes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Pattern name </th><th rowspan="2">Pattern code </th><th>Psychrophile </th><th>Mesophile </th><th>Thermophile </th><th>Hyperthermophile </th><th>Anaerobe </th><th>Aerobe </th><th>Facultative </th><th>Microaerophile </th></tr><tr><th>18 </th><th>382 </th><th>72 </th><th>26 </th><th>159 </th><th>202 </th><th>95 </th><th>9 </th></tr></thead><tbody><tr><td rowspan="11">Simple sequence repeats </td><td>1n8 </td><td>−3.83 </td><td>−3.54 </td><td>−3.89 </td><td>−4.00 </td><td>−3.73 </td><td>−3.61 </td><td>−3.65 </td><td>−3.67 </td></tr><tr><td>2n5 </td><td>−3.15 </td><td>−2.89 </td><td>−3.40 </td><td>−3.45 </td><td>−3.30 </td><td>−2.74 </td><td>−3.02 </td><td>−3.11 </td></tr><tr><td>3n4 </td><td>−1.93 </td><td>−1.92 </td><td>−2.83 </td><td>−3.09 </td><td>−2.56 </td><td>−1.75 </td><td>−2.15 </td><td>−2.33 </td></tr><tr><td>4n4 </td><td>−0.06 </td><td>0.01 </td><td>−0.06 </td><td>−0.08 </td><td>−0.04 </td><td>0.01 </td><td>0.00 </td><td>−0.10 </td></tr><tr><td>5n4 </td><td>0.00 </td><td>0.07 </td><td>0.00 </td><td>0.00 </td><td>0.02 </td><td>0.10 </td><td>0.01 </td><td>0.11 </td></tr><tr><td>6n3 </td><td>0.20 </td><td>0.19 </td><td>0.06 </td><td>−0.03 </td><td>0.05 </td><td>0.25 </td><td>0.08 </td><td>0.31 </td></tr><tr><td>7n3 </td><td>1.27 </td><td>0.48 </td><td>0.45 </td><td>0.00 </td><td>0.42 </td><td>0.59 </td><td>0.35 </td><td>0.89 </td></tr><tr><td>8n3 </td><td>0.56 </td><td>0.30 </td><td>0.12 </td><td>0.01 </td><td>0.25 </td><td>0.27 </td><td>0.20 </td><td>0.33 </td></tr><tr><td>9n2 </td><td>−0.08 </td><td>0.09 </td><td>0.22 </td><td>0.04 </td><td>0.09 </td><td>0.19 </td><td>−0.11 </td><td>−0.11 </td></tr><tr><td>10n2 </td><td>0.09 </td><td>0.25 </td><td>0.25 </td><td>0.28 </td><td>0.27 </td><td>0.24 </td><td>0.16 </td><td>0.00 </td></tr><tr><td>11n2 </td><td>0.21 </td><td>0.32 </td><td>0.30 </td><td>0.40 </td><td>0.42 </td><td>0.26 </td><td>0.24 </td><td>0.03 </td></tr><tr><td rowspan="5">Close direct repeats </td><td>4n6g12 </td><td>0.45 </td><td>0.58 </td><td>0.48 </td><td>0.82 </td><td>0.55 </td><td>0.73 </td><td>0.31 </td><td>0.50 </td></tr><tr><td>6n6g24 </td><td>1.35 </td><td>1.16 </td><td>0.71 </td><td>0.34 </td><td>1.00 </td><td>1.21 </td><td>0.81 </td><td>1.39 </td></tr><tr><td>8n4g24 </td><td>2.02 </td><td>1.61 </td><td>0.80 </td><td>0.28 </td><td>1.27 </td><td>1.69 </td><td>1.18 </td><td>1.13 </td></tr><tr><td>cd8g6 </td><td>0.39 </td><td>0.73 </td><td>0.80 </td><td>1.20 </td><td>0.87 </td><td>0.82 </td><td>0.41 </td><td>0.70 </td></tr><tr><td>cd10g50 </td><td>0.46 </td><td>0.41 </td><td>0.05 </td><td>−0.07 </td><td>0.25 </td><td>0.60 </td><td>0.01 </td><td>0.07 </td></tr><tr><td rowspan="5">Palindromes and inverted repeats </td><td>cp8g6 </td><td>3.57 </td><td>2.78 </td><td>2.81 </td><td>1.99 </td><td>2.98 </td><td>2.39 </td><td>3.21 </td><td>3.22 </td></tr><tr><td>cp10g50 </td><td>3.56 </td><td>3.26 </td><td>2.98 </td><td>1.64 </td><td>3.15 </td><td>3.03 </td><td>3.45 </td><td>3.20 </td></tr><tr><td>pals9 </td><td>3.39 </td><td>2.58 </td><td>2.13 </td><td>0.71 </td><td>2.47 </td><td>2.27 </td><td>2.89 </td><td>2.47 </td></tr><tr><td>pals9g12 </td><td>3.83 </td><td>3.46 </td><td>3.05 </td><td>1.79 </td><td>3.33 </td><td>3.22 </td><td>3.55 </td><td>3.64 </td></tr><tr><td>pals12g20 </td><td>3.72 </td><td>3.58 </td><td>2.97 </td><td>1.17 </td><td>3.16 </td><td>3.47 </td><td>3.62 </td><td>3.00 </td></tr><tr><td rowspan="10">H-DNA-related patterns </td><td>cm8g6 </td><td>0.02 </td><td>0.20 </td><td>0.21 </td><td>0.33 </td><td>0.24 </td><td>0.24 </td><td>0.15 </td><td>0.04 </td></tr><tr><td>cm10g50 </td><td>0.02 </td><td>0.16 </td><td>0.10 </td><td>0.15 </td><td>0.16 </td><td>0.18 </td><td>0.10 </td><td>0.00 </td></tr><tr><td>mirs9 </td><td>0.00 </td><td>0.13 </td><td>0.19 </td><td>0.17 </td><td>0.14 </td><td>0.18 </td><td>0.08 </td><td>−0.02 </td></tr><tr><td>mirs9g12 </td><td>0.15 </td><td>0.30 </td><td>0.35 </td><td>0.44 </td><td>0.33 </td><td>0.38 </td><td>0.20 </td><td>0.00 </td></tr><tr><td>mirs12g20 </td><td>0.15 </td><td>0.27 </td><td>0.28 </td><td>0.20 </td><td>0.26 </td><td>0.33 </td><td>0.16 </td><td>0.22 </td></tr><tr><td>R15 </td><td>−0.47 </td><td>−0.52 </td><td>−0.96 </td><td>−1.77 </td><td>−0.91 </td><td>−0.55 </td><td>−0.34 </td><td>−1.78 </td></tr><tr><td>R30 </td><td>0.03 </td><td>0.07 </td><td>0.08 </td><td>0.11 </td><td>0.10 </td><td>0.04 </td><td>0.07 </td><td>0.00 </td></tr><tr><td>R30e3 </td><td>0.08 </td><td>−0.01 </td><td>−0.18 </td><td>−0.66 </td><td>−0.23 </td><td>0.00 </td><td>0.14 </td><td>−0.66 </td></tr><tr><td>R45e6 </td><td>0.17 </td><td>0.08 </td><td>−0.03 </td><td>−0.32 </td><td>−0.06 </td><td>0.10 </td><td>0.15 </td><td>−0.02 </td></tr><tr><td>R60e9 </td><td>0.15 </td><td>0.13 </td><td>0.11 </td><td>−0.03 </td><td>0.14 </td><td>0.10 </td><td>0.12 </td><td>0.00 </td></tr><tr><td rowspan="3">G-DNA-related patterns </td><td>GG8g4 </td><td>0.22 </td><td>0.72 </td><td>0.24 </td><td>0.14 </td><td>0.20 </td><td>1.03 </td><td>0.48 </td><td>0.44 </td></tr><tr><td>GGG4g6 </td><td>−0.08 </td><td>−0.26 </td><td>−0.78 </td><td>−0.65 </td><td>−0.67 </td><td>−0.18 </td><td>−0.26 </td><td>−0.22 </td></tr><tr><td>GGGG4g6 </td><td>0.00 </td><td>0.10 </td><td>−0.06 </td><td>−0.12 </td><td>0.01 </td><td>0.11 </td><td>0.02 </td><td>0.11 </td></tr><tr><td rowspan="6">Z-DNA-related patterns </td><td>GC6 </td><td>−1.37 </td><td>−1.50 </td><td>−1.35 </td><td>−0.95 </td><td>−1.41 </td><td>−1.38 </td><td>−1.71 </td><td>−1.42 </td></tr><tr><td>GC8 </td><td>−0.75 </td><td>−1.10 </td><td>−0.84 </td><td>−0.18 </td><td>−0.47 </td><td>−1.42 </td><td>−1.13 </td><td>−0.37 </td></tr><tr><td>RY12 </td><td>−1.79 </td><td>−1.58 </td><td>−0.88 </td><td>−0.16 </td><td>−0.78 </td><td>−1.73 </td><td>−2.03 </td><td>−0.99 </td></tr><tr><td>RY12e1 </td><td>−1.81 </td><td>−1.40 </td><td>−0.71 </td><td>−0.31 </td><td>−0.85 </td><td>−1.43 </td><td>−1.93 </td><td>−0.44 </td></tr><tr><td>RY18e2 </td><td>−1.66 </td><td>−1.45 </td><td>−0.72 </td><td>−0.04 </td><td>−0.58 </td><td>−1.60 </td><td>−2.07 </td><td>−0.37 </td></tr><tr><td>RY24e3 </td><td>−0.60 </td><td>−0.78 </td><td>−0.25 </td><td>0.05 </td><td>−0.30 </td><td>−0.79 </td><td>−1.12 </td><td>−0.44 </td></tr><tr><td rowspan="3">DNA Bending </td><td>bend45w60 </td><td>0.60 </td><td>0.93 </td><td>1.51 </td><td>1.11 </td><td>1.57 </td><td>0.59 </td><td>0.85 </td><td>1.70 </td></tr><tr><td>bend60w100 </td><td>0.32 </td><td>0.79 </td><td>1.40 </td><td>1.17 </td><td>1.42 </td><td>0.50 </td><td>0.74 </td><td>1.48 </td></tr><tr><td>bend90w120 </td><td>0.28 </td><td>0.58 </td><td>1.23 </td><td>0.94 </td><td>1.11 </td><td>0.34 </td><td>0.66 </td><td>1.44 </td></tr></tbody></table></div><div content-id="DST057TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Pattern name </th><th rowspan="2">Pattern code </th><th>Psychrophile </th><th>Mesophile </th><th>Thermophile </th><th>Hyperthermophile </th><th>Anaerobe </th><th>Aerobe </th><th>Facultative </th><th>Microaerophile </th></tr><tr><th>18 </th><th>382 </th><th>72 </th><th>26 </th><th>159 </th><th>202 </th><th>95 </th><th>9 </th></tr></thead><tbody><tr><td rowspan="11">Simple sequence repeats </td><td>1n8 </td><td>−3.83 </td><td>−3.54 </td><td>−3.89 </td><td>−4.00 </td><td>−3.73 </td><td>−3.61 </td><td>−3.65 </td><td>−3.67 </td></tr><tr><td>2n5 </td><td>−3.15 </td><td>−2.89 </td><td>−3.40 </td><td>−3.45 </td><td>−3.30 </td><td>−2.74 </td><td>−3.02 </td><td>−3.11 </td></tr><tr><td>3n4 </td><td>−1.93 </td><td>−1.92 </td><td>−2.83 </td><td>−3.09 </td><td>−2.56 </td><td>−1.75 </td><td>−2.15 </td><td>−2.33 </td></tr><tr><td>4n4 </td><td>−0.06 </td><td>0.01 </td><td>−0.06 </td><td>−0.08 </td><td>−0.04 </td><td>0.01 </td><td>0.00 </td><td>−0.10 </td></tr><tr><td>5n4 </td><td>0.00 </td><td>0.07 </td><td>0.00 </td><td>0.00 </td><td>0.02 </td><td>0.10 </td><td>0.01 </td><td>0.11 </td></tr><tr><td>6n3 </td><td>0.20 </td><td>0.19 </td><td>0.06 </td><td>−0.03 </td><td>0.05 </td><td>0.25 </td><td>0.08 </td><td>0.31 </td></tr><tr><td>7n3 </td><td>1.27 </td><td>0.48 </td><td>0.45 </td><td>0.00 </td><td>0.42 </td><td>0.59 </td><td>0.35 </td><td>0.89 </td></tr><tr><td>8n3 </td><td>0.56 </td><td>0.30 </td><td>0.12 </td><td>0.01 </td><td>0.25 </td><td>0.27 </td><td>0.20 </td><td>0.33 </td></tr><tr><td>9n2 </td><td>−0.08 </td><td>0.09 </td><td>0.22 </td><td>0.04 </td><td>0.09 </td><td>0.19 </td><td>−0.11 </td><td>−0.11 </td></tr><tr><td>10n2 </td><td>0.09 </td><td>0.25 </td><td>0.25 </td><td>0.28 </td><td>0.27 </td><td>0.24 </td><td>0.16 </td><td>0.00 </td></tr><tr><td>11n2 </td><td>0.21 </td><td>0.32 </td><td>0.30 </td><td>0.40 </td><td>0.42 </td><td>0.26 </td><td>0.24 </td><td>0.03 </td></tr><tr><td rowspan="5">Close direct repeats </td><td>4n6g12 </td><td>0.45 </td><td>0.58 </td><td>0.48 </td><td>0.82 </td><td>0.55 </td><td>0.73 </td><td>0.31 </td><td>0.50 </td></tr><tr><td>6n6g24 </td><td>1.35 </td><td>1.16 </td><td>0.71 </td><td>0.34 </td><td>1.00 </td><td>1.21 </td><td>0.81 </td><td>1.39 </td></tr><tr><td>8n4g24 </td><td>2.02 </td><td>1.61 </td><td>0.80 </td><td>0.28 </td><td>1.27 </td><td>1.69 </td><td>1.18 </td><td>1.13 </td></tr><tr><td>cd8g6 </td><td>0.39 </td><td>0.73 </td><td>0.80 </td><td>1.20 </td><td>0.87 </td><td>0.82 </td><td>0.41 </td><td>0.70 </td></tr><tr><td>cd10g50 </td><td>0.46 </td><td>0.41 </td><td>0.05 </td><td>−0.07 </td><td>0.25 </td><td>0.60 </td><td>0.01 </td><td>0.07 </td></tr><tr><td rowspan="5">Palindromes and inverted repeats </td><td>cp8g6 </td><td>3.57 </td><td>2.78 </td><td>2.81 </td><td>1.99 </td><td>2.98 </td><td>2.39 </td><td>3.21 </td><td>3.22 </td></tr><tr><td>cp10g50 </td><td>3.56 </td><td>3.26 </td><td>2.98 </td><td>1.64 </td><td>3.15 </td><td>3.03 </td><td>3.45 </td><td>3.20 </td></tr><tr><td>pals9 </td><td>3.39 </td><td>2.58 </td><td>2.13 </td><td>0.71 </td><td>2.47 </td><td>2.27 </td><td>2.89 </td><td>2.47 </td></tr><tr><td>pals9g12 </td><td>3.83 </td><td>3.46 </td><td>3.05 </td><td>1.79 </td><td>3.33 </td><td>3.22 </td><td>3.55 </td><td>3.64 </td></tr><tr><td>pals12g20 </td><td>3.72 </td><td>3.58 </td><td>2.97 </td><td>1.17 </td><td>3.16 </td><td>3.47 </td><td>3.62 </td><td>3.00 </td></tr><tr><td rowspan="10">H-DNA-related patterns </td><td>cm8g6 </td><td>0.02 </td><td>0.20 </td><td>0.21 </td><td>0.33 </td><td>0.24 </td><td>0.24 </td><td>0.15 </td><td>0.04 </td></tr><tr><td>cm10g50 </td><td>0.02 </td><td>0.16 </td><td>0.10 </td><td>0.15 </td><td>0.16 </td><td>0.18 </td><td>0.10 </td><td>0.00 </td></tr><tr><td>mirs9 </td><td>0.00 </td><td>0.13 </td><td>0.19 </td><td>0.17 </td><td>0.14 </td><td>0.18 </td><td>0.08 </td><td>−0.02 </td></tr><tr><td>mirs9g12 </td><td>0.15 </td><td>0.30 </td><td>0.35 </td><td>0.44 </td><td>0.33 </td><td>0.38 </td><td>0.20 </td><td>0.00 </td></tr><tr><td>mirs12g20 </td><td>0.15 </td><td>0.27 </td><td>0.28 </td><td>0.20 </td><td>0.26 </td><td>0.33 </td><td>0.16 </td><td>0.22 </td></tr><tr><td>R15 </td><td>−0.47 </td><td>−0.52 </td><td>−0.96 </td><td>−1.77 </td><td>−0.91 </td><td>−0.55 </td><td>−0.34 </td><td>−1.78 </td></tr><tr><td>R30 </td><td>0.03 </td><td>0.07 </td><td>0.08 </td><td>0.11 </td><td>0.10 </td><td>0.04 </td><td>0.07 </td><td>0.00 </td></tr><tr><td>R30e3 </td><td>0.08 </td><td>−0.01 </td><td>−0.18 </td><td>−0.66 </td><td>−0.23 </td><td>0.00 </td><td>0.14 </td><td>−0.66 </td></tr><tr><td>R45e6 </td><td>0.17 </td><td>0.08 </td><td>−0.03 </td><td>−0.32 </td><td>−0.06 </td><td>0.10 </td><td>0.15 </td><td>−0.02 </td></tr><tr><td>R60e9 </td><td>0.15 </td><td>0.13 </td><td>0.11 </td><td>−0.03 </td><td>0.14 </td><td>0.10 </td><td>0.12 </td><td>0.00 </td></tr><tr><td rowspan="3">G-DNA-related patterns </td><td>GG8g4 </td><td>0.22 </td><td>0.72 </td><td>0.24 </td><td>0.14 </td><td>0.20 </td><td>1.03 </td><td>0.48 </td><td>0.44 </td></tr><tr><td>GGG4g6 </td><td>−0.08 </td><td>−0.26 </td><td>−0.78 </td><td>−0.65 </td><td>−0.67 </td><td>−0.18 </td><td>−0.26 </td><td>−0.22 </td></tr><tr><td>GGGG4g6 </td><td>0.00 </td><td>0.10 </td><td>−0.06 </td><td>−0.12 </td><td>0.01 </td><td>0.11 </td><td>0.02 </td><td>0.11 </td></tr><tr><td rowspan="6">Z-DNA-related patterns </td><td>GC6 </td><td>−1.37 </td><td>−1.50 </td><td>−1.35 </td><td>−0.95 </td><td>−1.41 </td><td>−1.38 </td><td>−1.71 </td><td>−1.42 </td></tr><tr><td>GC8 </td><td>−0.75 </td><td>−1.10 </td><td>−0.84 </td><td>−0.18 </td><td>−0.47 </td><td>−1.42 </td><td>−1.13 </td><td>−0.37 </td></tr><tr><td>RY12 </td><td>−1.79 </td><td>−1.58 </td><td>−0.88 </td><td>−0.16 </td><td>−0.78 </td><td>−1.73 </td><td>−2.03 </td><td>−0.99 </td></tr><tr><td>RY12e1 </td><td>−1.81 </td><td>−1.40 </td><td>−0.71 </td><td>−0.31 </td><td>−0.85 </td><td>−1.43 </td><td>−1.93 </td><td>−0.44 </td></tr><tr><td>RY18e2 </td><td>−1.66 </td><td>−1.45 </td><td>−0.72 </td><td>−0.04 </td><td>−0.58 </td><td>−1.60 </td><td>−2.07 </td><td>−0.37 </td></tr><tr><td>RY24e3 </td><td>−0.60 </td><td>−0.78 </td><td>−0.25 </td><td>0.05 </td><td>−0.30 </td><td>−0.79 </td><td>−1.12 </td><td>−0.44 </td></tr><tr><td rowspan="3">DNA Bending </td><td>bend45w60 </td><td>0.60 </td><td>0.93 </td><td>1.51 </td><td>1.11 </td><td>1.57 </td><td>0.59 </td><td>0.85 </td><td>1.70 </td></tr><tr><td>bend60w100 </td><td>0.32 </td><td>0.79 </td><td>1.40 </td><td>1.17 </td><td>1.42 </td><td>0.50 </td><td>0.74 </td><td>1.48 </td></tr><tr><td>bend90w120 </td><td>0.28 </td><td>0.58 </td><td>1.23 </td><td>0.94 </td><td>1.11 </td><td>0.34 </td><td>0.66 </td><td>1.44 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn3"><div class="fn"><p>Numbers in the table refer to the average significance category for all genera within each class of organisms. Numbers below the class description indicate numbers of available genera of each class. Anaerobe includes both obligate anaerobes and anaerobes; Aerobe includes both obligate aerobes and aerobes. See <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a> for coloured version of this table.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5813731" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5813732 id="5813732" class="section-title" data-legacy-id=s3b>3.2. Close direct repeats</h3>
<p>In general, close direct repeats are normally represented or slightly over-represented in complete genomes of most prokaryotes and in the intergenic regions, and they are mostly normally represented in genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S9–S11</a>). The cd10g50 repeats (two copies of the same 10-mer separated by ≤50 nucleotides) exhibit the strongest trends among the investigated direct repeat structures. They are mostly over-represented in the intergenic regions, weakly under-represented in protein-coding regions, and normally represented in the whole genome. It is interesting to note the contrast in representations of close pairs of 10 bp repeats (cd10g50) and multiple close copies of shorter oligonucleotides (4n6g12, 6n6g24, 8n4g24). While the former are often strongly over-represented in intergenic regions and normally represented to weakly under-represented in genes, the latter are normally represented or moderately over-represented in both genes and intergenic regions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S10 and S11</a>).</p><p>Representations of close direct repeats exhibit clear negative trends with increasing OGT (Table <a reveal-id="DST057TB3" data-open="DST057TB3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S12</a>). For example, clustered repeats 8n4g24 (at least four copies of the same octamer separated by gaps of no more than 24 bp) are over-represented in ∼50% of psychrophiles and mesophiles but in few thermophiles and virtually no hyperthermophiles (Fig. <a reveal-id="DST057F1" data-open="DST057F1" class="link link-reveal link-table xref-fig">1</a>; the difference is significant at <em>P</em> &lt; 10<sup>−6</sup>). A significant decrease in pattern representations with increasing OGT is seen also for patterns 6n6g24 and8n4g24, indicating a general trend of lower local repetitiveness in thermophiles and hyperthermophiles. There is no clear relationship between the representation of close direct repeats and oxygen requirement. </p>                        <a id="5813738" scrollto-destination="5813738"></a>
<div id="DST057F1" data-id="DST057F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst057/2/m_dst05701.png?Expires=2147483647&Signature=Q5fZ2CH7f~GKxERfHO618Vojj~XLtygnK~LZ-NR1M8CoeEu3Y5Nb7XBJ3KtI2zVWMlo4Om13TML9W9bd-ZR6kkSmSVJsLzqjCBelrfoyKC5YuKYg2iPPgN7UN6QXGaMwFVES41wsV6Dl2b8qmwD2EuPcOTD7WAdaGSL22jfKRgVhjzsQZI1HWG8G83ovb9v54fF6Fm66DrPBjHjh34ZShuoDKXeyhljNyXf9yHo6VpP8RfFEaLW~j~09v1TYYhYxECKsl41VnmHCLvVhkcbWPXBtuR~UvAbzCqM2ZX3A3exKGXo4rmQwbe0DEUskTP4nUS87WofT3viQ7zhCKYWnOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of representations of selected patterns in different OGT classes. Bars show the percentage of species in each OGT class which have the given pattern under-represented, normally represented, or over-represented. The pattern is considered over-represented if the P-value is &lt;10−4 and observed to expected ratio &gt;1.10 (representation level 2 or higher) for majority of the complete genomes available for that genera, it is deemed under-represented if the P-value is &lt;10−4 and observed to expected ratio &lt;0.91, and normally represented otherwise. See Materials and methods and Supplementary Table S8. The four patterns for which the data are shown are representative of close repeat structures (8n4g24, top left), palindromes and close inverted repeats (pals9g12, top right), potential Z-DNA-promoting patterns (RY12, bottom left), and DNA bending pattrens (bend60w100, bottom right). See Table 1 for description of the pattern codes." path-from-xml="dst05701.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5813738/dst05701.tiff" path-from-xml="dst05701.tiff" target="_blank">View large</a><a section="5813738" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst057/2/dst05701.png?Expires=2147483647&Signature=p2s8mBdPKECYT5yvTikZAvOx-uhtKBNo6q1kfgRAiDpav~JrOWjhG1g26bNbyewwBRglvv97xFxrNVmn4KEvViGXQFVw~2b8J8ejRB9TG4~LBwJq-m3FXsK6jolLcrnSdBXhbwYiQEG3sSS0Wtqza5h1q-g-2C~vpXZPK7T-k4lC5aREwNqvXLGXRr5nEDAIC8vYMvkyD19gutdtBMZK0qwiuicJoaX-I1qOhXEEGxFCoCdVPGuKjPUv3EKCMxCp0cqVtEv3PaREJRtQc~LMDc1IqMPbY7Jev0EiDBbTBbzSWBU3DXyVBwkwQssINFwuq95KIp6SzE~18HBzJm-e1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5813738&ar=393995&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05701.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of representations of selected patterns in different OGT classes. Bars show the percentage of species in each OGT class which have the given pattern under-represented, normally represented, or over-represented. The pattern is considered over-represented if the <em>P</em>-value is &lt;10<sup>−4</sup> and observed to expected ratio &gt;1.10 (representation level 2 or higher) for majority of the complete genomes available for that genera, it is deemed under-represented if the <em>P</em>-value is &lt;10<sup>−4</sup> and observed to expected ratio &lt;0.91, and normally represented otherwise. See Materials and methods and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>. The four patterns for which the data are shown are representative of close repeat structures (8n4g24, top left), palindromes and close inverted repeats (pals9g12, top right), potential Z-DNA-promoting patterns (RY12, bottom left), and DNA bending pattrens (bend60w100, bottom right). See Table <a reveal-id="DST057TB1" data-open="DST057TB1" class="link link-reveal link-table xref-fig">1</a> for description of the pattern codes.</p></div></div><div content-id="DST057F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst057/2/m_dst05701.png?Expires=2147483647&Signature=Q5fZ2CH7f~GKxERfHO618Vojj~XLtygnK~LZ-NR1M8CoeEu3Y5Nb7XBJ3KtI2zVWMlo4Om13TML9W9bd-ZR6kkSmSVJsLzqjCBelrfoyKC5YuKYg2iPPgN7UN6QXGaMwFVES41wsV6Dl2b8qmwD2EuPcOTD7WAdaGSL22jfKRgVhjzsQZI1HWG8G83ovb9v54fF6Fm66DrPBjHjh34ZShuoDKXeyhljNyXf9yHo6VpP8RfFEaLW~j~09v1TYYhYxECKsl41VnmHCLvVhkcbWPXBtuR~UvAbzCqM2ZX3A3exKGXo4rmQwbe0DEUskTP4nUS87WofT3viQ7zhCKYWnOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of representations of selected patterns in different OGT classes. Bars show the percentage of species in each OGT class which have the given pattern under-represented, normally represented, or over-represented. The pattern is considered over-represented if the P-value is &lt;10−4 and observed to expected ratio &gt;1.10 (representation level 2 or higher) for majority of the complete genomes available for that genera, it is deemed under-represented if the P-value is &lt;10−4 and observed to expected ratio &lt;0.91, and normally represented otherwise. See Materials and methods and Supplementary Table S8. The four patterns for which the data are shown are representative of close repeat structures (8n4g24, top left), palindromes and close inverted repeats (pals9g12, top right), potential Z-DNA-promoting patterns (RY12, bottom left), and DNA bending pattrens (bend60w100, bottom right). See Table 1 for description of the pattern codes." path-from-xml="dst05701.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5813738/dst05701.tiff" path-from-xml="dst05701.tiff" target="_blank">View large</a><a section="5813738" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst057/2/dst05701.png?Expires=2147483647&Signature=p2s8mBdPKECYT5yvTikZAvOx-uhtKBNo6q1kfgRAiDpav~JrOWjhG1g26bNbyewwBRglvv97xFxrNVmn4KEvViGXQFVw~2b8J8ejRB9TG4~LBwJq-m3FXsK6jolLcrnSdBXhbwYiQEG3sSS0Wtqza5h1q-g-2C~vpXZPK7T-k4lC5aREwNqvXLGXRr5nEDAIC8vYMvkyD19gutdtBMZK0qwiuicJoaX-I1qOhXEEGxFCoCdVPGuKjPUv3EKCMxCp0cqVtEv3PaREJRtQc~LMDc1IqMPbY7Jev0EiDBbTBbzSWBU3DXyVBwkwQssINFwuq95KIp6SzE~18HBzJm-e1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5813738&ar=393995&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05701.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of representations of selected patterns in different OGT classes. Bars show the percentage of species in each OGT class which have the given pattern under-represented, normally represented, or over-represented. The pattern is considered over-represented if the <em>P</em>-value is &lt;10<sup>−4</sup> and observed to expected ratio &gt;1.10 (representation level 2 or higher) for majority of the complete genomes available for that genera, it is deemed under-represented if the <em>P</em>-value is &lt;10<sup>−4</sup> and observed to expected ratio &lt;0.91, and normally represented otherwise. See Materials and methods and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>. The four patterns for which the data are shown are representative of close repeat structures (8n4g24, top left), palindromes and close inverted repeats (pals9g12, top right), potential Z-DNA-promoting patterns (RY12, bottom left), and DNA bending pattrens (bend60w100, bottom right). See Table <a reveal-id="DST057TB1" data-open="DST057TB1" class="link link-reveal link-table xref-fig">1</a> for description of the pattern codes.</p></div></div>                            <h3 scrollto-destination=5813739 id="5813739" class="section-title" data-legacy-id=s3c>3.3. Palindromes and inverted repeats</h3>
<p>Palindromes and inverted repeats are strongly over-represented in the complete genomes across almost all bacterial phyla but less so in archaea and Aquificales (Table <a reveal-id="DST057TB2" data-open="DST057TB2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>). There are few organisms that strongly avoid palindromes in protein-coding regions and the complete genomes, including cyanobacteria <em>Synechosystis</em> and <em>Thermosynechococcus</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S2 and S3</a>). Representations of palindromes and close inverted repeats in protein-coding sequences vary significantly among different genera, from extremely over-represented (e.g. <em>Cyanobacterium</em>, <em>Thermoanaerobacter</em>, <em>Allivibrio</em>, <em>Azobacteroides</em>) to strongly suppressed (e.g. <em>Rhodomicrobium</em>, <em>Phenylobacterium</em>, <em>Jannaschia</em>, <em>Clavibacter</em>). Many α-proteobacteria exhibit some level of palindrome suppression. <em>Clavibacter</em> is an outlier among Actinobacteria, which generally tend to have palindromes normally represented and in some cases over-represented (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>).</p><p>A comparison between genes and intergenic regions shows that inverted repeats are almost invariably over-represented in the intergenic regions and generally normally represented in the protein-coding regions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S3 and S4</a>). The high concentration of palindromes and inverted repeats in the intergenic region is not necessarily surprising because of their function in transcription termination and as regulatory elements, which are generally located outside the protein-coding segments of genes.</p><p>There is a general trend of decreasing representations of palindromes and inverted repeats with increasing OGT (Fig. <a reveal-id="DST057F1" data-open="DST057F1" class="link link-reveal link-table xref-fig">1</a>, Table <a reveal-id="DST057TB3" data-open="DST057TB3" class="link link-reveal link-table xref-fig">3</a>, and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S12</a>). In contrast, there is only a weak or no relationship between representations of palindromes and oxygen requirement.</p>                            <h3 scrollto-destination=5813746 id="5813746" class="section-title" data-legacy-id=s3d>3.4. Oligopurine/oligopyrimidine runs and triplex DNA-promoting patterns</h3>
<p>H-DNA triplexes form under favourable conditions in oligopurine/oligopyrimidine sequences with a mirror symmetry,<sup><a reveal-id="DST057C24" data-open="DST057C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DST057C26" data-open="DST057C26" class="link link-ref link-reveal xref-bibr">26</a></sup> whereas some oligopurine/oligopyrimidine segments can also promote formation of A-like DNA.<sup><a reveal-id="DST057C2" data-open="DST057C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Mirror repeats and extended oligopurine/oligopyrimidine stretches are generally normally represented in complete genomes of most phyla (Table <a reveal-id="DST057TB2" data-open="DST057TB2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>), with moderate over-representations in some Cyanobacteria (<em>Anabaena, Nostoc</em>, <em>Microcystis, Trichodesmium</em>), Chloroflexi (<em>Chloroflexus</em> and <em>Roseiflexus</em>), Planctomycetes (<em>Isosphaera</em>), and Verrucomicrobia (<em>Methylacidiphilum</em>) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). However, oligopurine and oligopyrimidine stretches, mainly patterns R15 and R30e3, are suppressed in complete genomes of some phyla. Separate analysis of protein-coding and non-coding regions shows that oligopurine/oligopyrimidine stretches tend to be normally represented in intergenic regions and that anomalous representations of these patterns in some phyla arise from biases in protein-coding segments (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S10 and S11</a>).</p><p>Suppression of oligopurine/oligopyrimidine stretches, especially R15, is most common among hyperthermophiles, whereas mesophiles and psychrophiles exhibit normal representations of these patterns (Table <a reveal-id="DST057TB3" data-open="DST057TB3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S12</a>). The R15 pattern is also more likely to be under-represented in anaerobes than in aerobes (Table <a reveal-id="DST057TB3" data-open="DST057TB3" class="link link-reveal link-table xref-fig">3</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S12, and Supplementary Fig. S1</a>). On the other hand, mirror repeats cm10g50 and mirs12g20 are more likely to be over-represented in the coding regions of anaerobes than aerobes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S13 and Supplementary Fig. S2</a>).</p>                            <h3 scrollto-destination=5813751 id="5813751" class="section-title" data-legacy-id=s3e>3.5. Guanine-rich patterns and G-DNA-promoting sequences</h3>
<p>Formation of intrastrand G-DNA quadruplex is promoted by clustered short runs of guanine.<sup><a reveal-id="DST057C17" data-open="DST057C17" class="link link-ref link-reveal xref-bibr">17</a>–<a reveal-id="DST057C19" data-open="DST057C19" class="link link-ref link-reveal xref-bibr">19</a></sup> We investigated several forms of such G-rich clusters (G-patterns; Table <a reveal-id="DST057TB1" data-open="DST057TB1" class="link link-reveal link-table xref-fig">1</a>), among which the GGG4g6 pattern (four G-triplets separated by gaps of no more than six nucleotides) represents best the sequences known to form G-quadruplexes.<sup><a reveal-id="DST057C2" data-open="DST057C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DST057C38" data-open="DST057C38" class="link link-ref link-reveal xref-bibr">38</a>,<a reveal-id="DST057C39" data-open="DST057C39" class="link link-ref link-reveal xref-bibr">39</a></sup> In general, the G-patterns are normally represented in prokaryotic genomes (Table <a reveal-id="DST057TB2" data-open="DST057TB2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>). GG dimer clusters (GG8g4, eight or more GG dimers separated by ≤4 nucleotides from each other) tend to be moderately over-represented in protein-coding regions of α-, β-, and δ-proteobacteria, Actinobacteria, Cyanobacteria, Deincoccus-Thermus group, and Planctomycetes (especially for <em>Isosphaera</em>) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S10</a>). The other guanine patterns are generally normally represented with most notable exceptions among Cyanobacteria where species of <em>Microcystis</em>, <em>Anabaena</em>, and <em>Nostoc</em> feature extreme over-representation of all forms of G-patterns (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). The planctomycete <em>Isosphaera pallida</em> also has G-patterns strongly over-represented. With these few exceptions, the GGG4g6 pattern, which is most directly related to G-DNA formation, is mostly normally represented or slightly under-represented, suggesting that if G-DNA-promoting sequences have significant physiological roles in bacteria, such roles are probably limited to only a few species or genera.</p><p>Most organisms with over-represented GG8g4 pattern are aerobes, whereas virtually no anaerobes have excess of GG8g4 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). However, over-represented GG8g4 pattern could also reflect excess of glycine-rich segments (encoded by the GGN codons) or proline-rich segments (encoded by CCN with GG dinucleotides in the complementary strand) in proteins, whereas the GGG4g6 does not exhibit a relationship with oxygen requirement. There is no apparent relationship between G-pattern representations and the OGT (Table <a reveal-id="DST057TB3" data-open="DST057TB3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S12</a>).</p>                            <h3 scrollto-destination=5813756 id="5813756" class="section-title" data-legacy-id=s3f>3.6. Z-DNA-promoting patterns</h3>
<p>The left-handed Z-DNA conformation is most commonly adopted by runs of alternating G–C and more generally by alternating purine–pyrimidine (RY) patterns. The RY patterns are often under-represented but to different extent in different phyla—most strongly in α-, β-, and γ-proteobacteria, Actinobacteria, Cyanobacteria, Chlorobi, Chloroflexi, and Planctomycetes (Table <a reveal-id="DST057TB2" data-open="DST057TB2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>). In contrast, the RY patterns are normally represented in Fusobacteria, Aquificales, and Thermotogae and even slightly over-represented in Chlamydiae. In general, the suppression of RY patterns is stronger in protein-coding regions, although several phyla exhibit significant RY pattern suppression in intergenic regions as well (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S10 and S11</a>). Interestingly, <em>Treponema pallidum</em> and <em>Treponema paraluiscuniculi</em> exhibit a strong over-representation of all forms of RY patterns, whereas other spirochaetes, including other species of <em>Treponema</em>, have RY patterns normally represented or weakly under-represented. Similar extreme over-representation of RY patterns applies to <em>Helicobacter felis</em> and to a lesser extent to <em>Helicobacter bizzozeronii</em> but not to other <em>Helicobacter</em> species. The archaeon <em>Thermofilum pendens</em> also exhibits a strong over-representation of RY patterns and several other genomes show a weaker RY-pattern over-representation (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>).</p><p>We used Pattern Locator and associated software tools<sup><a reveal-id="DST057C36" data-open="DST057C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DST057C40" data-open="DST057C40" class="link link-ref link-reveal xref-bibr">40</a></sup> (<a class="link link-uri" href="http://www.cmbl.uga.edu/software/patloc.html">http://www.cmbl.uga.edu/software/patloc.html</a>) to investigate in detail the distribution of RY patterns in several specific genomes, including the above-mentioned genomes with over-represented RY patterns. We did not find any significant anomalies in the distribution of the RY patterns with respect to the origin or terminus of replication, with respect to the 3′ ends of genes (stop codons), or any strong association with a particular class of genes. However, we noted increased numbers of RY patterns overlapping with start codons, such that the ATG/GTG start codons are embedded in RY patterns which sometimes extend several codons deep into the protein-coding region (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). This tendency appears to be widespread among prokaryotic genomes. However, comparison between ATG and GTG triplets that function as start codons and those that are not translation start sites shows no significant difference in fractions of ATG/GTG triplets overlapping with extended RY patterns between the two groups (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S18</a>). This suggests that the increased number of RY patterns overlapping with translation start sites is not directly related to the translation but rather a simple consequence of ATG and GTG themselves having the form of a short RY pattern (RYR), thus increasing the likelihood of finding a longer RY pattern at the same site.</p><p>Representations of RY patterns tend to increase with increasing OGT (Table <a reveal-id="DST057TB3" data-open="DST057TB3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S12</a>). For example, the pattern RY12 is under-represented in 60% of psychrophiles, 50% of mesophiles, ∼25% of thermophiles but only a single hyperthermophilic genus, <em>Thermaerobacter</em> (Fig. <a reveal-id="DST057F1" data-open="DST057F1" class="link link-reveal link-table xref-fig">1</a>; significant at <em>P</em> &lt; 10<sup>−6</sup>).</p><p>Among all analysed sequence patterns, the RY patterns exhibit the most notable trend with respect to the level of oxygen requirement (Table <a reveal-id="DST057TB3" data-open="DST057TB3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S12</a>). Interestingly, facultative species exhibit the strongest suppression of RY patterns followed by aerobes, whereas RY pattern representations in anaerobes and microaerophilic organisms are close to normal (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>c for pattern RY12). The same trend was observed when the comparisons between aerobes and anaerobes were restricted to mesophiles, or bacteria (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S15 and S17</a>), indicating that the relationship between RY patterns and oxygen requirement cannot be attributed to increased number of anaerobes among thermophiles or different representations of anaerobes and aerobes among bacteria and archaea.</p>                            <h3 scrollto-destination=5813762 id="5813762" class="section-title" data-legacy-id=s3g>3.7. Patterns contributing to DNA curvature</h3>
<p>Sequence patterns related to DNA curvature (the bend-patterns) range generally from normal to over-represented. The over-representation is most pronounced in ɛ-proteobacteria, Fusobacteria, Deferribacteres, and Thermotogae (Table <a reveal-id="DST057TB2" data-open="DST057TB2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). This trend applies to both protein-coding and intergenic regions, although differences among phyla are more prominent in protein-coding regions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S9 and S11</a>). At the opposite extreme, <em>Mycoplasma haemofelis</em>, <em>Mycoplasma penetrans</em>, <em>Cytophaga hutchinsonii</em>, and <em>Pelagibacter</em> sp. show moderate to strong suppression of DNA bends in their genomes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). The genus <em>Mycoplasma</em> is particularly interesting, featuring species with extreme over-representation of DNA bends as well as strong under-representation. This is consistent with our previous report that various genome properties among <em>Mycoplasma</em> vary more than in other genera.<sup><a reveal-id="DST057C37" data-open="DST057C37" class="link link-ref link-reveal xref-bibr">37</a></sup> In addition, <em>C. hutchinsonii</em> and <em>Pelagibacter</em> sp. feature extreme suppression of bend-patterns in protein-coding regions but not in intergenic regions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S3 and S4</a>).</p><p>With respect to OGT, intrinsic DNA bends tend to be more over-represented in hyperthermophiles and thermophiles than in mesophiles and psychrophiles (Table <a reveal-id="DST057TB3" data-open="DST057TB3" class="link link-reveal link-table xref-fig">3</a> and Fig. <a reveal-id="DST057F1" data-open="DST057F1" class="link link-reveal link-table xref-fig">1</a>). Surprisingly, the number of genera with over-represented bend-patterns in protein-coding segments increases with increasing OGT in protein-coding regions but decreases for bend-patterns in non-coding regions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). The same trend is shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>, where genomes of thermophiles and hyperthermophiles tend to have more protein-coding bends than intergenic bends, whereas the opposite is true for most psychrophiles. Statistical significance of this trend has been confirmed by the Mann–Whitney <em>U</em>-test (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S19</a>).</p><p>With respect to oxygen requirement, representation of DNA bending patterns tends to decrease with the increasing level of oxygen (Table <a reveal-id="DST057TB3" data-open="DST057TB3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S12</a>). Specifically, the bend-patterns tend to be over-represented in anaerobes and microaerophiles but not in aerobes. For example, the pattern bend60w100 is over-represented in half of the anaerobes, but &lt;20% of aerobes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. 1</a>; significant at <em>P</em> &lt; 10<sup>−6</sup>). This trend holds when the data are restricted to mesophiles or thermophiles, suggesting that the trend with respect to oxygen requirement is independent of the trend with respect to OGT (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S15 and S16</a>).</p>                            <h2 scrollto-destination=5813770 id="5813770" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=5813774 id="5813774" class="section-title" data-legacy-id=s4a>4.1. Comparisons with previous work</h3>
<p>The work presented here differs from similar earlier surveys in scope (both the number of genomes used in the analysis and the number of different types of sequence patterns investigated) as well as methodology. The most important methodological difference is in the null model used to assess whether a sequence pattern is anomalously represented in a given genome. Our null model takes into account the nearest-neighbour biases and codon usage propensities of all individual genes and intergenic regions, which likely have separate underlying causes not related to potential functional significance of the investigated sequence patterns. In terms of our results, one significant difference relative to earlier work concerns representations of alternating purine–pyrimidine patterns. Bohlin <em>et al.</em><sup><a reveal-id="DST057C3" data-open="DST057C3" class="link link-ref link-reveal xref-bibr">3</a></sup> reported that alternating RY patterns were generally suppressed across different phyla except β-proteobactria, where they were mostly over-represented, with a particularly strong surplus of RY patterns in <em>Burkholderia</em>. These authors used the i.i.d. model (independently drawn and independently distributed letters) as a benchmark. Using our more realistic null model, we found that β-proteobactria including <em>Burkholderia</em> species suppress the RY patterns to the same extent as other bacteria (Table <a reveal-id="DST057TB2" data-open="DST057TB2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S2 and S9</a>). We therefore conclude that the increased amount of RY patterns in β-proteobactria arises from two opposite evolutionary constraints: biased codon and dinucleotide usages that favour RY-patterns and are specific for β-proteobactria, which are partially offset by suppression of long RY-patterns, which is rather universal among bacterial genomes. Similarly, the over-representation of oligopurine/oligopyrimidine stretches reported by Bohlin <em>et al.</em> for some phyla arises from the biases at the level of dinucleotide and codon usages, as we found the oligopurine/oligopyrimidine stretches normally represented or under-represented in all phyla (Table <a reveal-id="DST057TB2" data-open="DST057TB2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>).</p><p>For SSRs, our results are consistent with previous works including our own survey of SSRs in prokaryotic genomes.<sup><a reveal-id="DST057C7" data-open="DST057C7" class="link link-ref link-reveal xref-bibr">7</a></sup> In particular, our present data confirm that SSR comprised tandem repeats of very short units (mono-, di-, and trinucleotides; patterns 1n8, 2n5, and 3n4) tend to be strongly suppressed in prokaryotic genomes, while repeats of longer units (tetranucleotides through 11-mers) tend to be normally represented or weakly over-represented (Table <a reveal-id="DST057TB2" data-open="DST057TB2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>). This strong difference points to likely functional difference between tandem repeats of very short (1–3 bp) and longer (4–11 bp) units. Our present results differ from the earlier data in that we previously included repeats of tetranucleotides in the same group as mono-, di, and trinucleotides,<sup><a reveal-id="DST057C7" data-open="DST057C7" class="link link-ref link-reveal xref-bibr">7</a></sup> while the present results indicate that tetranucleotide SSRs may be more accurately included in the same group as SSRs composed of pentanucleotides and longer units, which are generally not suppressed. This result suggests that tandem repeats of tetranucleotides and longer units may not have the same harmful effects as repeats of mono-, di-, and trinucleotides. The difference between tandem repeats of very short units (1–3 bp) and longer units (≥4 bp) could also arise from properties of the methyl-directed repair pathway, which is very efficient in repairing heterologous loops of 1–3 bp, but its efficiency dramatically drops as the length of the loop increases.<sup><a reveal-id="DST057C41" data-open="DST057C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DST057C42" data-open="DST057C42" class="link link-ref link-reveal xref-bibr">42</a></sup></p><p>Ladoukakis and Eyre-Walker<sup><a reveal-id="DST057C43" data-open="DST057C43" class="link link-ref link-reveal xref-bibr">43</a></sup> reported a small but significant excess of short inverted repeats (6–9 bp) in protein-coding sequences and proffered that the inverted repeats could arise by sequence-directed mutagenesis where an imperfect palindrome or inverted repeat converts to a perfect one due to template switching during replication.<sup><a reveal-id="DST057C15" data-open="DST057C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Their result is consistent with that of Katz and Burge,<sup><a reveal-id="DST057C44" data-open="DST057C44" class="link link-ref link-reveal xref-bibr">44</a></sup> who found that native mRNA sequences in many bacterial genomes possess a significantly higher potential to form stable RNA secondary structures than random sequences preserving the codon usage, dinucleotide frequencies, and the protein sequences encoded by the native mRNAs. Katz and Burge proposed that the excess of base pairing in mRNA molecules is due to selection for stable mRNAs. We analysed occurrences of slightly larger palindromes than Ladoukakis and Eyre-Walker but with a similar result that palindromes and close inverted repeats are strongly over-represented in both protein-coding and non-coding regions of genomes of most prokaryotic phyla.</p><p>The excess of palindromes is weaker in the archaea and in the Aquificae than in other bacterial phyla. This result is similar to that of Katz and Burge, who reported a significant excess of mRNA secondary structures in most bacteria but few archaea, and not in <em>Aquifex aeolicus</em> (note that their study was based on a much smaller sample of genomes available at that time and <em>A. aeolicus</em> was a sole representative of Aquificae in their data set). The weaker over-representation of palindromes in archaea and Aquificae is related to general decrease in palindrome representations with increasing OGT (Fig. <a reveal-id="DST057F1" data-open="DST057F1" class="link link-reveal link-table xref-fig">1</a>). The decrease in palindrome representations with increasing OGT might seem counterintuitive if selection for mRNA stability drives the excess of palindromes in mRNA sequences. One possible explanation is that higher temperatures prevent formation of stable mRNA structures even with an increased content of inverted repeats, thus making selection for inverted repeats moot, or perhaps higher temperature decreases the efficiency of palindrome conversion via sequence-directed mutagenesis. Yet another potential explanation for weaker over-representation of inverted repeats in thermophiles stems from the work by Paz <em>et al.</em>,<sup><a reveal-id="DST057C45" data-open="DST057C45" class="link link-ref link-reveal xref-bibr">45</a></sup> who noted that thermophilic mRNA sequences have increased purine-to-pyrimidine ratios compared with mesophiles, as well as excess of short oligopurine tracts. These authors proposed that purine-loading of mRNA sequences could increase their thermal stability. The resulting purine–pyrimidine imbalance could decrease the base-pairing potential in the mRNA sequences and the purine bias could therefore contribute to lower excess of palindromes and inverted repeats in thermophiles.</p>                            <h3 scrollto-destination=5813790 id="5813790" class="section-title" data-legacy-id=s4b>4.2. Relationship with OGT and oxygen requirement</h3>
<p>Extreme temperature as well as presence of oxygen (via reactive oxygen species) cause damage to DNA and require cellular mechanisms to prevent or correct the damage in order to keep the cells viable. Such protection can be enzymatic (e.g. detoxification pathways) but could also involve adaptations in DNA composition and structure. One particularly puzzling question is how thermophiles prevent their DNA from denaturing. The simple explanation that DNA of thermophile is more GC-rich and thus more stable was mostly rejected as more data became available.<sup><a reveal-id="DST057C46" data-open="DST057C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DST057C47" data-open="DST057C47" class="link link-ref link-reveal xref-bibr">47</a></sup> Kawashima <em>et al</em>.<sup><a reveal-id="DST057C48" data-open="DST057C48" class="link link-ref link-reveal xref-bibr">48</a></sup> proffered excess of RR and YY dinucleotides over RY and YR dinucleotides as a characteristic of thermophiles but this has later been shown a poor indicator of thermophily.<sup><a reveal-id="DST057C47" data-open="DST057C47" class="link link-ref link-reveal xref-bibr">47</a></sup> We were therefore interested in finding whether any of the sequence patterns we analysed could be related to differences in OGT and oxygen requirement, and possibly contribute to the protection of DNA from the effects of extreme temperature and/or oxygen damage.</p><p>One of our more surprising results concerns the relationship of representations of intrinsic DNA bends with OGT. Bolshoy and coworkers<sup><a reveal-id="DST057C5" data-open="DST057C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DST057C49" data-open="DST057C49" class="link link-ref link-reveal xref-bibr">49</a></sup> reported an excess of intrinsically bent DNA in intergenic regions of prokaryotic genomes, including segments containing putative promoters and transcription terminators. Notably, the amount of bent DNA in intergenic regions was significantly weaker in thermophiles than in mesophiles. In a separate work, Tolstorukov <em>et al</em>.<sup><a reveal-id="DST057C33" data-open="DST057C33" class="link link-ref link-reveal xref-bibr">33</a></sup> found intrinsic DNA bends distributed widely throughout bacterial genomes, including both intergenic and protein-coding regions, and they proposed that the intrinsic bends could play a role in compaction of the bacterial nucleoid. Our present data show that predicted intrinsic DNA bends are over-represented in many phyla in both protein-coding and non-coding segments. Interestingly, the protein-coding and non-coding regions exhibit opposite trends with respect to OGT. For non-coding sequences, our results are consistent with the decrease in curved DNA in thermophiles observed by Kozobay-Avraham <em>et al.</em><sup><a reveal-id="DST057C5" data-open="DST057C5" class="link link-ref link-reveal xref-bibr">5</a></sup> However, the representation of DNA bends in protein-coding sequences tends to increase with increasing OGT.</p><p>We propose that the opposite trends in protein-coding and non-coding DNA segments reflect different roles of DNA bends. Intergenic bends are often associated with regulatory elements and could facilitate opening of the DNA double helix at the transcription initiation sites.<sup><a reveal-id="DST057C50" data-open="DST057C50" class="link link-ref link-reveal xref-bibr">50</a>,<a reveal-id="DST057C51" data-open="DST057C51" class="link link-ref link-reveal xref-bibr">51</a></sup> Decreased amount of intrinsic DNA bends in intergenic regions of thermophiles is consistent with experiments that showed that anomalous gel mobility characteristic of intrinsically curved DNA disappears at increased temperatures.<sup><a reveal-id="DST057C52" data-open="DST057C52" class="link link-ref link-reveal xref-bibr">52</a></sup> If sequence-directed intrinsic DNA bending is ineffective at high temperatures, then selection for intrinsic bends at regulatory sites could be weak or absent. It is also possible that DNA bending is less important to transcription initiation at high temperatures. Surprisingly, the excess content of intragenic bends, which are less likely to have direct regulatory functions but may contribute to establishing and maintaining the nucleoid structure, increases with increasing OGT. The latter result could possibly indicate that the intrinsic bends maintain their role in stabilizing the nucleoid structure even at high temperatures and that high OGT may require increased amount of intrinsic DNA bending to maintain adequate structural stability of the chromosome.</p><p>In addition to palindromes and close inverted repeats discussed above, some direct repeat structures also feature decreasing representations with increasing OGT (Table <a reveal-id="DST057TB3" data-open="DST057TB3" class="link link-reveal link-table xref-fig">3</a> and Fig. <a reveal-id="DST057F1" data-open="DST057F1" class="link link-reveal link-table xref-fig">1</a>). We speculate that higher temperature might make the thermophile DNA more susceptible to illegitimate recombination, DNA polymerase slippage, or other forms of mutations facilitated by short repeats, and that reducing the amount of repeats, both direct and inverted, could be a strategy to counteract increased recombination rates. Along these lines, thermophiles were reported to have lower mutation rates than mesophiles, possibly driven by selection to maintain thermostability of the encoded proteins.<sup><a reveal-id="DST057C53" data-open="DST057C53" class="link link-ref link-reveal xref-bibr">53</a></sup> Because close repeats are often sources of mutations, suppression of close repeats could be a part of the strategy to decrease overall mutation rates in thermophiles.</p><p>Alternating RY patterns exhibit trends related to both OGT and oxygen requirement and these trends are independent of each other. The relationship with oxygen requirement is particularly intriguing. Facultative organisms suppress RY patterns more strongly than both aerobes and anaerobes. The main known structural effect of RY patterns is that they can facilitate transitions to left-handed Z-DNA under favourable conditions. The B-to-Z transition in such sequences can be induced by torsional stress arising from processes that require DNA unwinding, such as transcription or replication.<sup><a reveal-id="DST057C28" data-open="DST057C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DST057C31" data-open="DST057C31" class="link link-ref link-reveal xref-bibr">31</a></sup> The general suppression of RY patterns in prokaryotes suggests that B-to-Z DNA transitions are generally undesirable. It is intriguing to speculate that Z-DNA could be more detrimental to facultative organisms perhaps due to interference with effective regulation of a diverse ensemble of metabolic pathways related to the ability to grow in both aerobic and anaerobic conditions. However, the specific mechanism of such interference between Z-DNA formation and transcriptional regulation is not clear.</p><p>Overall, our data indicate potential new mechanisms how genome properties adapt to particular environments. Most of these mechanisms are related to adaptations to growth at high temperatures, which appear to be accompanied by reduction in overall repetitiveness of the DNA sequence, reduced excess of palindromes, and reduced DNA curvature in regulatory regions accompanied by general increase in intrinsically curved DNA in protein-coding sequences.</p>                            <h2 scrollto-destination=5813807 id="5813807" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=5813809 id="5813809" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work is supported by the grant number DBI-0950266 from National Science Foundation.</p>                            <h2 scrollto-destination=5813811 id="5813811" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We wish to thank Dr Barny Whitman and Hao Tong for stimulating suggestions, and Orion Mantione for his help in writing the program to predict DNA bends.</p>                            <h2 scrollto-destination=5813813 id="5813813" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST057C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ussery</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Soumpasis</div> <div class="given-names">D.M.</div></div>,  <div class="name"><div class="surname">Brunak</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Staerfeldt</div> <div class="given-names">H.H.</div></div>,  <div class="name"><div class="surname">Worning</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Krogh</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Bias of purine stretches in sequenced chromosomes</div>, <div class="source">Comput. Chem.</div> , <div class="year">2002</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">531</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bias%20of%20purine%20stretches%20in%20sequenced%20chromosomes&amp;author=D.%20Ussery&amp;author=D.M.%20Soumpasis&amp;author=S.%20Brunak&amp;author=H.H.%20Staerfeldt&amp;author=P.%20Worning&amp;author=A.%20Krogh&amp;publication_year=2002&amp;journal=Comput.%20Chem.&amp;volume=26&amp;pages=531-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0097-8485(02)00013-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0097-8485(02)00013-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0097-8485(02)00013-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12144181" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sinden</div> <div class="given-names">R.R.</div></div>. </div>, <div class="source">DNA Structure and Function</div> , <div class="year">1994</div><div class="publisher-loc">San Diego</div><div class="publisher-name">Academic Press</div></div></div></div></div><div content-id="DST057C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bohlin</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Hardy</div> <div class="given-names">S.P.</div></div>,  <div class="name"><div class="surname">Ussery</div> <div class="given-names">D.W.</div></div>. </div><div class="article-title">Stretches of alternating pyrimidine/purines and purines are respectively linked with pathogenicity and growth temperature in prokaryotes</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">346</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stretches%20of%20alternating%20pyrimidine%2Fpurines%20and%20purines%20are%20respectively%20linked%20with%20pathogenicity%20and%20growth%20temperature%20in%20prokaryotes&amp;author=J.%20Bohlin&amp;author=S.P.%20Hardy&amp;author=D.W.%20Ussery&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=346" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-346" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-346" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-346"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19646265" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rawal</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Kummarasetti</div> <div class="given-names">V.B.</div></div>,  <div class="name"><div class="surname">Ravindran</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Genome-wide prediction of G4 DNA as regulatory motifs: role in <em>Escherichia coli</em> global regulation</div>, <div class="source">Genome Res.</div> , <div class="year">2006</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">644</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20prediction%20of%20G4%20DNA%20as%20regulatory%20motifs%3A%20role%20in%20Escherichia%20coli%20global%20regulation&amp;author=P.%20Rawal&amp;author=V.B.%20Kummarasetti&amp;author=J.%20Ravindran&amp;publication_year=2006&amp;journal=Genome%20Res.&amp;volume=16&amp;pages=644-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.4508806" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.4508806" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.4508806"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16651665" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kozobay-Avraham</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Hosid</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bolshoy</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Involvement of DNA curvature in intergenic regions of prokaryotes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">2316</div>-<div class="lpage">27</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Involvement%20of%20DNA%20curvature%20in%20intergenic%20regions%20of%20prokaryotes&amp;author=L.%20Kozobay-Avraham&amp;author=S.%20Hosid&amp;author=A.%20Bolshoy&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=2316-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl230" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl230" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl230"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16679450" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mrázek</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Comparative analysis of sequence periodicity among prokaryotic genomes points to differences in nucleoid structure and a relationship to gene expression</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2010</div>, vol. <div class="volume">192</div> (pg. <div class="fpage">3763</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20sequence%20periodicity%20among%20prokaryotic%20genomes%20points%20to%20differences%20in%20nucleoid%20structure%20and%20a%20relationship%20to%20gene%20expression&amp;author=J.%20Mr%C3%A1zek&amp;publication_year=2010&amp;journal=J.%20Bacteriol.&amp;volume=192&amp;pages=3763-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00149-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00149-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00149-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20494989" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mrázek</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Guo</div> <div class="given-names">X.X.</div></div>,  <div class="name"><div class="surname">Shah</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Simple sequence repeats in prokaryotic genomes</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2007</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">8472</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20sequence%20repeats%20in%20prokaryotic%20genomes&amp;author=J.%20Mr%C3%A1zek&amp;author=X.X.%20Guo&amp;author=A.%20Shah&amp;publication_year=2007&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=104&amp;pages=8472-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0702412104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0702412104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0702412104"> </span></div> </div></div></div></div></div><div content-id="DST057C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thanbichler</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Shapiro</div> <div class="given-names">L.</div></div>. </div><div class="article-title">The bacterial nucleoid: a highly organized and dynamic structure</div>, <div class="source">J. Cell Biochem.</div> , <div class="year">2005</div>, vol. <div class="volume">96</div> (pg. <div class="fpage">506</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20bacterial%20nucleoid%3A%20a%20highly%20organized%20and%20dynamic%20structure&amp;author=M.%20Thanbichler&amp;author=S.C.%20Wang&amp;author=L.%20Shapiro&amp;publication_year=2005&amp;journal=J.%20Cell%20Biochem.&amp;volume=96&amp;pages=506-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-4644" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-4644" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-4644"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15988757" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dillon</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Dorman</div> <div class="given-names">C.J.</div></div>. </div><div class="article-title">Bacterial nucleoid-associated proteins, nucleoid structure and gene expression</div>, <div class="source">Nat. Rev. Microbiol.</div> , <div class="year">2010</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">185</div>-<div class="lpage">95</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20nucleoid-associated%20proteins%2C%20nucleoid%20structure%20and%20gene%20expression&amp;author=S.C.%20Dillon&amp;author=C.J.%20Dorman&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Microbiol.&amp;volume=8&amp;pages=185-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro2261" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro2261" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro2261"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20140026" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moxon</div> <div class="given-names">E.R.</div></div>,  <div class="name"><div class="surname">Rainey</div> <div class="given-names">P.B.</div></div>,  <div class="name"><div class="surname">Nowak</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Lenski</div> <div class="given-names">R.E.</div></div>. </div><div class="article-title">Adaptive evolution of highly mutable loci in pathogenic bacteria</div>, <div class="source">Curr. Biol.</div> , <div class="year">1994</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">24</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adaptive%20evolution%20of%20highly%20mutable%20loci%20in%20pathogenic%20bacteria&amp;author=E.R.%20Moxon&amp;author=P.B.%20Rainey&amp;author=M.A.%20Nowak&amp;author=R.E.%20Lenski&amp;publication_year=1994&amp;journal=Curr.%20Biol.&amp;volume=4&amp;pages=24-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0960-9822(00)00005-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0960-9822(00)00005-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0960-9822(00)00005-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7922307" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">van der Woude</div> <div class="given-names">M.W.</div></div>,  <div class="name"><div class="surname">Baumler</div> <div class="given-names">A.J.</div></div>. </div><div class="article-title">Phase and antigenic variation in bacteria</div>, <div class="source">Clin. Microbiol. Rev.</div> , <div class="year">2004</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">581</div>-<div class="lpage">611</div>) <div class="comment">table of contents</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phase%20and%20antigenic%20variation%20in%20bacteria&amp;author=M.W.%20van%20der%20Woude&amp;author=A.J.%20Baumler&amp;publication_year=2004&amp;journal=Clin.%20Microbiol.%20Rev.&amp;volume=17&amp;pages=581-611" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/CMR.17.3.581-611.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FCMR.17.3.581-611.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FCMR.17.3.581-611.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15258095" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brázda</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Laister</div> <div class="given-names">R.C.</div></div>,  <div class="name"><div class="surname">Jagelská</div> <div class="given-names">E.B.</div></div>,  <div class="name"><div class="surname">Arrowsmith</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Cruciform structures are a common DNA feature important for regulating biological processes</div>, <div class="source">BMC Mol. Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">33</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cruciform%20structures%20are%20a%20common%20DNA%20feature%20important%20for%20regulating%20biological%20processes&amp;author=V.%20Br%C3%A1zda&amp;author=R.C.%20Laister&amp;author=E.B.%20Jagelsk%C3%A1&amp;author=C.%20Arrowsmith&amp;publication_year=2011&amp;journal=BMC%20Mol.%20Biol.&amp;volume=12&amp;pages=33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2199-12-33" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2199-12-33" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2199-12-33"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21816114" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Levinson</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Gutman</div> <div class="given-names">G.A.</div></div>. </div><div class="article-title">Slipped-strand mispairing: a major mechanism for DNA sequence evolution</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1987</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">203</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Slipped-strand%20mispairing%3A%20a%20major%20mechanism%20for%20DNA%20sequence%20evolution&amp;author=G.%20Levinson&amp;author=G.A.%20Gutman&amp;publication_year=1987&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=4&amp;pages=203-21" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3328815" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dutra</div> <div class="given-names">B.E.</div></div>,  <div class="name"><div class="surname">Lovett</div> <div class="given-names">S.T.</div></div>. </div><div class="article-title">Cis and trans-acting effects on a mutational hotspot involving a replication template switch</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">356</div> (pg. <div class="fpage">300</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cis%20and%20trans-acting%20effects%20on%20a%20mutational%20hotspot%20involving%20a%20replication%20template%20switch&amp;author=B.E.%20Dutra&amp;author=S.T.%20Lovett&amp;publication_year=2006&amp;journal=J.%20Mol.%20Biol.&amp;volume=356&amp;pages=300-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmb.2005.11.071" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmb.2005.11.071" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmb.2005.11.071"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16376936" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lovett</div> <div class="given-names">S.T.</div></div>. </div><div class="article-title">Encoded errors: mutations and rearrangements mediated by misalignment at repetitive DNA sequences</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2004</div>, vol. <div class="volume">52</div> (pg. <div class="fpage">1243</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Encoded%20errors%3A%20mutations%20and%20rearrangements%20mediated%20by%20misalignment%20at%20repetitive%20DNA%20sequences&amp;author=S.T.%20Lovett&amp;publication_year=2004&amp;journal=Mol.%20Microbiol.&amp;volume=52&amp;pages=1243-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04076.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-2958.2004.04076.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-2958.2004.04076.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15165229" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chuzhanova</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Abeysinghe</div> <div class="given-names">S.S.</div></div>,  <div class="name"><div class="surname">Krawczak</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Cooper</div> <div class="given-names">D.N.</div></div>. </div><div class="article-title">Translocation and gross deletion breakpoints in human inherited disease and cancer II: potential involvement of repetitive sequence elements in secondary structure formation between DNA ends</div>, <div class="source">Hum. Mutat.</div> , <div class="year">2003</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">245</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translocation%20and%20gross%20deletion%20breakpoints%20in%20human%20inherited%20disease%20and%20cancer%20II%3A%20potential%20involvement%20of%20repetitive%20sequence%20elements%20in%20secondary%20structure%20formation%20between%20DNA%20ends&amp;author=N.%20Chuzhanova&amp;author=S.S.%20Abeysinghe&amp;author=M.%20Krawczak&amp;author=D.N.%20Cooper&amp;publication_year=2003&amp;journal=Hum.%20Mutat.&amp;volume=22&amp;pages=245-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1098-1004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1098-1004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1098-1004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12938089" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Burge</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Parkinson</div> <div class="given-names">G.N.</div></div>,  <div class="name"><div class="surname">Hazel</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Todd</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Neidle</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Quadruplex DNA: sequence, topology and structure</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">5402</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quadruplex%20DNA%3A%20sequence%2C%20topology%20and%20structure&amp;author=S.%20Burge&amp;author=G.N.%20Parkinson&amp;author=P.%20Hazel&amp;author=A.K.%20Todd&amp;author=S.%20Neidle&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=5402-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl655" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl655" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl655"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17012276" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vorlíčková</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Bednářová</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kejnovská</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Kypr</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Intramolecular and intermolecular guanine quadruplexes of DNA in aqueous salt and ethanol solutions</div>, <div class="source">Biopolymers</div> , <div class="year">2007</div>, vol. <div class="volume">86</div> (pg. <div class="fpage">1</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intramolecular%20and%20intermolecular%20guanine%20quadruplexes%20of%20DNA%20in%20aqueous%20salt%20and%20ethanol%20solutions&amp;author=M.%20Vorl%C3%AD%C4%8Dkov%C3%A1&amp;author=K.%20Bedn%C3%A1%C5%99ov%C3%A1&amp;author=I.%20Kejnovsk%C3%A1&amp;author=J.%20Kypr&amp;publication_year=2007&amp;journal=Biopolymers&amp;volume=86&amp;pages=1-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0282" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0282" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0282"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17211886" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shafer</div> <div class="given-names">R.H.</div></div>,  <div class="name"><div class="surname">Smirnov</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Biological aspects of DNA/RNA quadruplexes</div>, <div class="source">Biopolymers</div> , <div class="year">2000</div>, vol. <div class="volume">56</div> (pg. <div class="fpage">209</div>-<div class="lpage">27</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biological%20aspects%20of%20DNA%2FRNA%20quadruplexes&amp;author=R.H.%20Shafer&amp;author=I.%20Smirnov&amp;publication_year=2000&amp;journal=Biopolymers&amp;volume=56&amp;pages=209-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0282" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0282" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0282"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11745112" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schaeffer</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Bardoni</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Mandel</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Ehresmann</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Ehresmann</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Moine</div> <div class="given-names">H.</div></div>. </div><div class="article-title">The fragile X mental retardation protein binds specifically to its mRNA via a purine quartet motif</div>, <div class="source">EMBO J.</div> , <div class="year">2001</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">4803</div>-<div class="lpage">13</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20fragile%20X%20mental%20retardation%20protein%20binds%20specifically%20to%20its%20mRNA%20via%20a%20purine%20quartet%20motif&amp;author=C.%20Schaeffer&amp;author=B.%20Bardoni&amp;author=J.L.%20Mandel&amp;author=B.%20Ehresmann&amp;author=C.%20Ehresmann&amp;author=H.%20Moine&amp;publication_year=2001&amp;journal=EMBO%20J.&amp;volume=20&amp;pages=4803-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/emboj/20.17.4803" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Femboj%2F20.17.4803" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Femboj%2F20.17.4803"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11532944" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pan</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Shi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Sundaralingam</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Base-tetrad swapping results in dimerization of RNA quadruplexes: implications for formation of the i-motif RNA octaplex</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2006</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">3130</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Base-tetrad%20swapping%20results%20in%20dimerization%20of%20RNA%20quadruplexes%3A%20implications%20for%20formation%20of%20the%20i-motif%20RNA%20octaplex&amp;author=B.%20Pan&amp;author=K.%20Shi&amp;author=M.%20Sundaralingam&amp;publication_year=2006&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=3130-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0507730103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0507730103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0507730103"> </span></div> </div></div></div></div></div><div content-id="DST057C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Arthanari</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Bolton</div> <div class="given-names">P.H.</div></div>. </div><div class="article-title">Functional and dysfunctional roles of quadruplex DNA in cells</div>, <div class="source">Chem. Biol.</div> , <div class="year">2001</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">221</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20and%20dysfunctional%20roles%20of%20quadruplex%20DNA%20in%20cells&amp;author=H.%20Arthanari&amp;author=P.H.%20Bolton&amp;publication_year=2001&amp;journal=Chem.%20Biol.&amp;volume=8&amp;pages=221-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1074-5521(01)00007-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1074-5521(01)00007-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1074-5521(01)00007-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11306347" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sundquist</div> <div class="given-names">W.I.</div></div>,  <div class="name"><div class="surname">Heaphy</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Evidence for interstrand quadruplex formation in the dimerization of human immunodeficiency virus 1 genomic RNA</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1993</div>, vol. <div class="volume">90</div> (pg. <div class="fpage">3393</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20interstrand%20quadruplex%20formation%20in%20the%20dimerization%20of%20human%20immunodeficiency%20virus%201%20genomic%20RNA&amp;author=W.I.%20Sundquist&amp;author=S.%20Heaphy&amp;publication_year=1993&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=90&amp;pages=3393-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.90.8.3393" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.90.8.3393" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.90.8.3393"> </span></div> </div></div></div></div></div><div content-id="DST057C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Belotserkovskii</div> <div class="given-names">B.P.</div></div>,  <div class="name"><div class="surname">Veselkov</div> <div class="given-names">A.G.</div></div>,  <div class="name"><div class="surname">Filippov</div> <div class="given-names">S.A.</div></div>,  <div class="name"><div class="surname">Dobrynin</div> <div class="given-names">V.N.</div></div>,  <div class="name"><div class="surname">Mirkin</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Frank-Kamenetskii</div> <div class="given-names">M.D.</div></div>. </div><div class="article-title">Formation of intramolecular triplex in homopurine–homopyrimidine mirror repeats with point substitutions</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1990</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">6621</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Formation%20of%20intramolecular%20triplex%20in%20homopurine%E2%80%93homopyrimidine%20mirror%20repeats%20with%20point%20substitutions&amp;author=B.P.%20Belotserkovskii&amp;author=A.G.%20Veselkov&amp;author=S.A.%20Filippov&amp;author=V.N.%20Dobrynin&amp;author=S.M.%20Mirkin&amp;author=M.D.%20Frank-Kamenetskii&amp;publication_year=1990&amp;journal=Nucleic%20Acids%20Res.&amp;volume=18&amp;pages=6621-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/18.22.6621" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F18.22.6621" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F18.22.6621"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2251122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rustighi</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Tessari</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Vascotto</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Sgarra</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Giancotti</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Manfioletti</div> <div class="given-names">G.</div></div>. </div><div class="article-title">A polypyrimidine/polypurine tract within the Hmga2 minimal promoter: a common feature of many growth-related genes</div>, <div class="source">Biochemistry</div> , <div class="year">2002</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">1229</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20polypyrimidine%2Fpolypurine%20tract%20within%20the%20Hmga2%20minimal%20promoter%3A%20a%20common%20feature%20of%20many%20growth-related%20genes&amp;author=A.%20Rustighi&amp;author=M.A.%20Tessari&amp;author=F.%20Vascotto&amp;author=R.%20Sgarra&amp;author=V.%20Giancotti&amp;author=G.%20Manfioletti&amp;publication_year=2002&amp;journal=Biochemistry&amp;volume=41&amp;pages=1229-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/bi011666o" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fbi011666o" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fbi011666o"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11802722" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zain</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">J.S.</div></div>. </div><div class="article-title">Do natural DNA triple-helical structures occur and function in vivo?</div>, <div class="source">Cell Mol. Life Sci.</div> , <div class="year">2003</div>, vol. <div class="volume">60</div> (pg. <div class="fpage">862</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Do%20natural%20DNA%20triple-helical%20structures%20occur%20and%20function%20in%20vivo%3F&amp;author=R.%20Zain&amp;author=J.S.%20Sun&amp;publication_year=2003&amp;journal=Cell%20Mol.%20Life%20Sci.&amp;volume=60&amp;pages=862-70" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12827276" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rich</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Nordheim</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Azorin</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Stabilization and detection of natural left-handed Z-DNA</div>, <div class="source">J. Biomol. Struct. Dyn.</div> , <div class="year">1983</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">1</div>-<div class="lpage">19</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stabilization%20and%20detection%20of%20natural%20left-handed%20Z-DNA&amp;author=A.%20Rich&amp;author=A.%20Nordheim&amp;author=F.%20Azorin&amp;publication_year=1983&amp;journal=J.%20Biomol.%20Struct.%20Dyn.&amp;volume=1&amp;pages=1-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/07391102.1983.10507424" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F07391102.1983.10507424" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F07391102.1983.10507424"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6101083" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">van Holde</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Zlatanova</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Unusual DNA structures, chromatin and transcription</div>, <div class="source">Bioessays</div> , <div class="year">1994</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">59</div>-<div class="lpage">68</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unusual%20DNA%20structures%2C%20chromatin%20and%20transcription&amp;author=K.%20van%20Holde&amp;author=J.%20Zlatanova&amp;publication_year=1994&amp;journal=Bioessays&amp;volume=16&amp;pages=59-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1521-1878" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1521-1878" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1521-1878"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8141807" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Herbert</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Rich</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Left-handed Z-DNA: structure and function</div>, <div class="source">Genetica</div> , <div class="year">1999</div>, vol. <div class="volume">106</div> (pg. <div class="fpage">37</div>-<div class="lpage">47</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Left-handed%20Z-DNA%3A%20structure%20and%20function&amp;author=A.%20Herbert&amp;author=A.%20Rich&amp;publication_year=1999&amp;journal=Genetica&amp;volume=106&amp;pages=37-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1003768526018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1003768526018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1003768526018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10710708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rich</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Timeline: Z-DNA: the long road to biological function</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2003</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">566</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Timeline%3A%20Z-DNA%3A%20the%20long%20road%20to%20biological%20function&amp;author=A.%20Rich&amp;author=S.%20Zhang&amp;publication_year=2003&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=4&amp;pages=566-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1115" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1115" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1115"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12838348" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Vasquez</div> <div class="given-names">K.M.</div></div>. </div><div class="article-title">Z-DNA, an active element in the genome</div>, <div class="source">Front Biosci.</div> , <div class="year">2007</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">4424</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Z-DNA%2C%20an%20active%20element%20in%20the%20genome&amp;author=G.%20Wang&amp;author=K.M.%20Vasquez&amp;publication_year=2007&amp;journal=Front%20Biosci.&amp;volume=12&amp;pages=4424-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2741/2399" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2741%2F2399" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2741%2F2399"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17485386" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jain</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Vasquez</div> <div class="given-names">K.M.</div></div>. </div><div class="article-title">DNA triple helices: biological consequences and therapeutic potential</div>, <div class="source">Biochimie</div> , <div class="year">2008</div>, vol. <div class="volume">90</div> (pg. <div class="fpage">1117</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20triple%20helices%3A%20biological%20consequences%20and%20therapeutic%20potential&amp;author=A.%20Jain&amp;author=G.%20Wang&amp;author=K.M.%20Vasquez&amp;publication_year=2008&amp;journal=Biochimie&amp;volume=90&amp;pages=1117-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biochi.2008.02.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biochi.2008.02.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biochi.2008.02.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18331847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tolstorukov</div> <div class="given-names">M.Y.</div></div>,  <div class="name"><div class="surname">Virnik</div> <div class="given-names">K.M.</div></div>,  <div class="name"><div class="surname">Adhya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zhurkin</div> <div class="given-names">V.B.</div></div>. </div><div class="article-title">A-tract clusters may facilitate DNA packaging in bacterial nucleoid</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">3907</div>-<div class="lpage">18</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A-tract%20clusters%20may%20facilitate%20DNA%20packaging%20in%20bacterial%20nucleoid&amp;author=M.Y.%20Tolstorukov&amp;author=K.M.%20Virnik&amp;author=S.%20Adhya&amp;author=V.B.%20Zhurkin&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=3907-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki699" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki699" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki699"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16024741" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Herzel</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Weiss</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Trifonov</div> <div class="given-names">E.N.</div></div>. </div><div class="article-title">Sequence periodicity in complete genomes of archaea suggests positive supercoiling</div>, <div class="source">J. Biomol. Struct. Dyn.</div> , <div class="year">1998</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">341</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20periodicity%20in%20complete%20genomes%20of%20archaea%20suggests%20positive%20supercoiling&amp;author=H.%20Herzel&amp;author=O.%20Weiss&amp;author=E.N.%20Trifonov&amp;publication_year=1998&amp;journal=J.%20Biomol.%20Struct.%20Dyn.&amp;volume=16&amp;pages=341-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/07391102.1998.10508251" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F07391102.1998.10508251" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F07391102.1998.10508251"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9833672" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pagani</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Liolios</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Jansson</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">The Genomes OnLine Database (GOLD) v.4: status of genomic and metagenomic projects and their associated metadata</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2012</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">D571</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Genomes%20OnLine%20Database%20(GOLD)%20v.4%3A%20status%20of%20genomic%20and%20metagenomic%20projects%20and%20their%20associated%20metadata&amp;author=I.%20Pagani&amp;author=K.%20Liolios&amp;author=J.%20Jansson&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=D571-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1100"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22135293" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mrázek</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Xie</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Pattern locator: a new tool for finding local sequence patterns in genomic DNA sequences</div>, <div class="source">Bioinformatics</div> , <div class="year">2006</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">3099</div>-<div class="lpage">100</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pattern%20locator%3A%20a%20new%20tool%20for%20finding%20local%20sequence%20patterns%20in%20genomic%20DNA%20sequences&amp;author=J.%20Mr%C3%A1zek&amp;author=S.%20Xie&amp;publication_year=2006&amp;journal=Bioinformatics&amp;volume=22&amp;pages=3099-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btl551" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtl551" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtl551"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17095514" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mrázek</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Analysis of distribution indicates diverse functions of simple sequence repeats in Mycoplasma genomes</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2006</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">1370</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20distribution%20indicates%20diverse%20functions%20of%20simple%20sequence%20repeats%20in%20Mycoplasma%20genomes&amp;author=J.%20Mr%C3%A1zek&amp;publication_year=2006&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=23&amp;pages=1370-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msk023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsk023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsk023"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16618962" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lane</div> <div class="given-names">A.N.</div></div>,  <div class="name"><div class="surname">Chaires</div> <div class="given-names">J.B.</div></div>,  <div class="name"><div class="surname">Gray</div> <div class="given-names">R.D.</div></div>,  <div class="name"><div class="surname">Trent</div> <div class="given-names">J.O.</div></div>. </div><div class="article-title">Stability and kinetics of G-quadruplex structures</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">5482</div>-<div class="lpage">515</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stability%20and%20kinetics%20of%20G-quadruplex%20structures&amp;author=A.N.%20Lane&amp;author=J.B.%20Chaires&amp;author=R.D.%20Gray&amp;author=J.O.%20Trent&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=5482-515" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn517" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn517" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn517"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18718931" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vorlíčková</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Chládková</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kejnovská</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Fialová</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kypr</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Guanine tetraplex topology of human telomere DNA is governed by the number of (TTAGGG) repeats</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">5851</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Guanine%20tetraplex%20topology%20of%20human%20telomere%20DNA%20is%20governed%20by%20the%20number%20of%20(TTAGGG)%20repeats&amp;author=M.%20Vorl%C3%AD%C4%8Dkov%C3%A1&amp;author=J.%20Chl%C3%A1dkov%C3%A1&amp;author=I.%20Kejnovsk%C3%A1&amp;author=M.%20Fialov%C3%A1&amp;author=J.%20Kypr&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=5851-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki898" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki898" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki898"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16221978" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mrázek</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Xie</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Guo</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Srivastava</div> <div class="given-names">A.</div></div>. </div><div class="article-title">AIMIE: a web-based environment for detection and interpretation of significant sequence motifs in prokaryotic genomes</div>, <div class="source">Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">1041</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AIMIE%3A%20a%20web-based%20environment%20for%20detection%20and%20interpretation%20of%20significant%20sequence%20motifs%20in%20prokaryotic%20genomes&amp;author=J.%20Mr%C3%A1zek&amp;author=S.%20Xie&amp;author=X.%20Guo&amp;author=A.%20Srivastava&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=1041-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn077" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn077" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn077"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18304933" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fang</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">J.Y.</div></div>,  <div class="name"><div class="surname">Su</div> <div class="given-names">M.J.</div></div>. </div><div class="article-title">Methyl-directed repair of mismatched small heterologous sequences in cell extracts from Escherichia coli</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1997</div>, vol. <div class="volume">272</div> (pg. <div class="fpage">22714</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Methyl-directed%20repair%20of%20mismatched%20small%20heterologous%20sequences%20in%20cell%20extracts%20from%20Escherichia%20coli&amp;author=W.%20Fang&amp;author=J.Y.%20Wu&amp;author=M.J.%20Su&amp;publication_year=1997&amp;journal=J.%20Biol.%20Chem.&amp;volume=272&amp;pages=22714-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.272.36.22714" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.272.36.22714" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.272.36.22714"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9278429" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parker</div> <div class="given-names">B.O.</div></div>,  <div class="name"><div class="surname">Marinus</div> <div class="given-names">M.G.</div></div>. </div><div class="article-title">Repair of DNA heteroduplexes containing small heterologous sequences in Escherichia coli</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1992</div>, vol. <div class="volume">89</div> (pg. <div class="fpage">1730</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repair%20of%20DNA%20heteroduplexes%20containing%20small%20heterologous%20sequences%20in%20Escherichia%20coli&amp;author=B.O.%20Parker&amp;author=M.G.%20Marinus&amp;publication_year=1992&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=89&amp;pages=1730-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.89.5.1730" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.89.5.1730" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.89.5.1730"> </span></div> </div></div></div></div></div><div content-id="DST057C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ladoukakis</div> <div class="given-names">E.D.</div></div>,  <div class="name"><div class="surname">Eyre-Walker</div> <div class="given-names">A.</div></div>. </div><div class="article-title">The excess of small inverted repeats in prokaryotes</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">2008</div>, vol. <div class="volume">67</div> (pg. <div class="fpage">291</div>-<div class="lpage">300</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20excess%20of%20small%20inverted%20repeats%20in%20prokaryotes&amp;author=E.D.%20Ladoukakis&amp;author=A.%20Eyre-Walker&amp;publication_year=2008&amp;journal=J.%20Mol.%20Evol.&amp;volume=67&amp;pages=291-300" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-008-9151-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-008-9151-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-008-9151-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18696026" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Katz</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Burge</div> <div class="given-names">C.B.</div></div>. </div><div class="article-title">Widespread selection for local RNA secondary structure in coding regions of bacterial genes</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">2042</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Widespread%20selection%20for%20local%20RNA%20secondary%20structure%20in%20coding%20regions%20of%20bacterial%20genes&amp;author=L.%20Katz&amp;author=C.B.%20Burge&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=2042-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1257503" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1257503" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1257503"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12952875" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Paz</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Mester</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Baca</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Nevo</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Korol</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Adaptive role of increased frequency of polypurine tracts in mRNA sequences of thermophilic prokaryotes</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">2951</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adaptive%20role%20of%20increased%20frequency%20of%20polypurine%20tracts%20in%20mRNA%20sequences%20of%20thermophilic%20prokaryotes&amp;author=A.%20Paz&amp;author=D.%20Mester&amp;author=I.%20Baca&amp;author=E.%20Nevo&amp;author=A.%20Korol&amp;publication_year=2004&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=2951-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0308594100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0308594100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0308594100"> </span></div> </div></div></div></div></div><div content-id="DST057C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Haney</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Badger</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Buldak</div> <div class="given-names">G.L.</div></div>,  <div class="name"><div class="surname">Reich</div> <div class="given-names">C.I.</div></div>,  <div class="name"><div class="surname">Woese</div> <div class="given-names">C.R.</div></div>,  <div class="name"><div class="surname">Olsen</div> <div class="given-names">G.J.</div></div>. </div><div class="article-title">Thermal adaptation analyzed by comparison of protein sequences from mesophilic and extremely thermophilic <em>Methanococcus</em> species</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1999</div>, vol. <div class="volume">96</div> (pg. <div class="fpage">3578</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Thermal%20adaptation%20analyzed%20by%20comparison%20of%20protein%20sequences%20from%20mesophilic%20and%20extremely%20thermophilic%20Methanococcus%20species&amp;author=P.J.%20Haney&amp;author=J.H.%20Badger&amp;author=G.L.%20Buldak&amp;author=C.I.%20Reich&amp;author=C.R.%20Woese&amp;author=G.J.%20Olsen&amp;publication_year=1999&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=96&amp;pages=3578-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.96.7.3578" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.96.7.3578" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.96.7.3578"> </span></div> </div></div></div></div></div><div content-id="DST057C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suhre</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Claverie</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">Genomic correlates of hyperthermostability, an update</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2003</div>, vol. <div class="volume">278</div> (pg. <div class="fpage">17198</div>-<div class="lpage">202</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20correlates%20of%20hyperthermostability%2C%20an%20update&amp;author=K.%20Suhre&amp;author=J.M.%20Claverie&amp;publication_year=2003&amp;journal=J.%20Biol.%20Chem.&amp;volume=278&amp;pages=17198-202" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M301327200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M301327200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M301327200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12600994" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawashima</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Amano</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Koike</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Archaeal adaptation to higher temperatures revealed by genomic sequence of Thermoplasma volcanium</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2000</div>, vol. <div class="volume">97</div> (pg. <div class="fpage">14257</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Archaeal%20adaptation%20to%20higher%20temperatures%20revealed%20by%20genomic%20sequence%20of%20Thermoplasma%20volcanium&amp;author=T.%20Kawashima&amp;author=N.%20Amano&amp;author=H.%20Koike&amp;publication_year=2000&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=97&amp;pages=14257-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.97.26.14257" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.97.26.14257" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.97.26.14257"> </span></div> </div></div></div></div></div><div content-id="DST057C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bolshoy</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Nevo</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Ecologic genomics of DNA: upstream bending in prokaryotic promoters</div>, <div class="source">Genome Res.</div> , <div class="year">2000</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">1185</div>-<div class="lpage">93</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ecologic%20genomics%20of%20DNA%3A%20upstream%20bending%20in%20prokaryotic%20promoters&amp;author=A.%20Bolshoy&amp;author=E.%20Nevo&amp;publication_year=2000&amp;journal=Genome%20Res.&amp;volume=10&amp;pages=1185-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.10.8.1185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.10.8.1185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.10.8.1185"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10958636" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Olivares-Zavaleta</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Jauregui</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Merino</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Genome analysis of Escherichia coli promoter sequences evidences that DNA static curvature plays a more important role in gene transcription than has previously been anticipated</div>, <div class="source">Genomics</div> , <div class="year">2006</div>, vol. <div class="volume">87</div> (pg. <div class="fpage">329</div>-<div class="lpage">37</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20analysis%20of%20Escherichia%20coli%20promoter%20sequences%20evidences%20that%20DNA%20static%20curvature%20plays%20a%20more%20important%20role%20in%20gene%20transcription%20than%20has%20previously%20been%20anticipated&amp;author=N.%20Olivares-Zavaleta&amp;author=R.%20Jauregui&amp;author=E.%20Merino&amp;publication_year=2006&amp;journal=Genomics&amp;volume=87&amp;pages=329-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygeno.2005.11.023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2005.11.023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygeno.2005.11.023"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16413165" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jauregui</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Abreu-Goodger</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Moreno-Hagelsieb</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Collado-Vides</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Merino</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Conservation of DNA curvature signals in regulatory regions of prokaryotic genes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">6770</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conservation%20of%20DNA%20curvature%20signals%20in%20regulatory%20regions%20of%20prokaryotic%20genes&amp;author=R.%20Jauregui&amp;author=C.%20Abreu-Goodger&amp;author=G.%20Moreno-Hagelsieb&amp;author=J.%20Collado-Vides&amp;author=E.%20Merino&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=6770-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg882" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg882" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg882"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14627810" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ussery</div> <div class="given-names">D.W.</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">C.F.</div></div>,  <div class="name"><div class="surname">Bolshoy</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Environmental influences on DNA curvature</div>, <div class="source">J. Biomol. Struct. Dyn.</div> , <div class="year">1999</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">811</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Environmental%20influences%20on%20DNA%20curvature&amp;author=D.W.%20Ussery&amp;author=C.F.%20Higgins&amp;author=A.%20Bolshoy&amp;publication_year=1999&amp;journal=J.%20Biomol.%20Struct.%20Dyn.&amp;volume=16&amp;pages=811-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/07391102.1999.10508294" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F07391102.1999.10508294" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F07391102.1999.10508294"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10217451" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST057C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST057C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Drake</div> <div class="given-names">J.W.</div></div>. </div><div class="article-title">Avoiding dangerous missense: thermophiles display especially low mutation rates</div>, <div class="source">PLoS Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e1000520</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Avoiding%20dangerous%20missense%3A%20thermophiles%20display%20especially%20low%20mutation%20rates&amp;author=J.W.%20Drake&amp;publication_year=2009&amp;journal=PLoS%20Genet.&amp;volume=5&amp;pages=e1000520" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000520" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000520" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000520"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19543367" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id="fn1"><div class="fn"><div class="footnote-content">Edited by Dr Mikita Suyama</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=393995&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst057/2/dst057-dst057supp_table8.pdf?Expires=2147483647&amp;Signature=LDa0CN2~CywtX1JX~viT1AMABI23x2MKw2dycLaPxmjgn3sMbbfpAElAJfZQEnn3tTvZul90234VW3pp81dnZYhkkilUOh~vjVvIzN7CMKOxNC9UN2Mkr7Y~nvgwXRCMu9--gVjT8ZNp2K6mirOnTO-fO5ygCLHFW9Mg5QznT~zAs7x0BAjNhftjSh~brEa779P0iNEb-8mRLM4E5xQWyhIKi0RenEOkKYpCYl4b5HXPDC3NYRB~a3QlnUb4fe6bVtXtyupVe7wLIHPBJlTG82TG75TKFKjll3cGEHI8MN96mWc0QmPkY-M6fEO0InYZBGgK7DVaQ1bhAu9FENMUNg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-9</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst057/2/dst057-dst057supp_table7.xlsx?Expires=2147483647&amp;Signature=sgpzp3Avs4megSys3e998AOenHW97rKegtEBKEVc4n2onPDVD7Rvtsq-7yJGGjDm~lN5pARap3ea7g5ukTkUpk3Q9gU1-eLv3hyBrrQNxsd5ZxDh1-vyWoK9WWCsLPh30YUImLCKMqHSpdwiwwlNLqm~Sp1rRzNIlcQKItvI2d-IDkPvDB8WuoLDnKbNVdbPiSCpJ3xnUAWu5JTISeCNUNftggnJRyNr8O42M7MY231VeilygstURn6VBfDp~r-UWjfKDYn6DARfhLlEWfNlbNRbr1m8WcOndE9q-lG9XE6MloFy2K9Blcpx3UP0XJ5cX-3Xg8IGH5rTyXPW9UoBDg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst057/2/dst057-dst057supp_table6.xlsx?Expires=2147483647&amp;Signature=vbmSFfUcGZx31XN0CHnpQ7q6x17-huL1zXjRR~ZuR8AQACyYMO0BbNDE1FpzjSpySgrSFbdarHdLWDijvHxh62i9ohsyOmLzvzb06HKfqL-mrWxYdRF8j1jg9~otuxzRY3HZUaJnnU1zJykVkEspitfbWabpKe1TBMa3VlfKioPVyEADHgkyMHATexWiPHLiJ05rPgt6v0X2~pdUTeymuLh3oDyldeEhq50NMiSBHisnF4OjwClbYGwcqYHBj4FEgyxL5ujuLqODCShFB6ZiY5nNnv6B5-6x1ax4XbHaRurdUe8dWfbDa0yBAwYyTElxPvvL4QuQadChSkWVaG1Ejw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst057/2/dst057-dst057supp_table5.xlsx?Expires=2147483647&amp;Signature=qrf0~~IrvOuNxDeXQ480F9V0z61d5ktOON8oxHbbnIjtQELK~j8LdBQ~iXWu8ymqN6fysoiuJQ7ig26yp29vjIZddWO-IIR34-r9PM~kTFg7V3ErQZfCQiyLImHYZEjn-TJpobezW8QYK9Ks44xze-kxhX0JtDJHVppMlXS-LMPkA2IrclPRvCI-Z5Kt8M1zGTJGorCC1JnJdblUyr7NddwpM8w18wDrklNfOlJ42IDGJMLmNiWzntP5SgbFdzEK8RB1K~s~NxOr9jMiX-AIrvnos7AblNkPzAkB4H4-ilD6IwEdQpI1XzIItL3vW4Glpxvil5VBLDDulVTkDWSILg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst057/2/dst057-dst057supp_table4.xlsx?Expires=2147483647&amp;Signature=UnEdM2bP77hNwlvRnEwxNsF~ZmrsCmrGecci1Ao8UIkZHcw8z0fQTNHJu1k~JZTbNavX3~9eUQsKnLW24ieYgO5L9BovBXbxuXn3cDIhWVvNBpTIniTnJN29HghyVw8wFVjDYq6JDYvRo4ulD9GxRFbuewja156M~XAs5aDSNhM0M0a7xi4vbvyCdvzzv~XicWlpRzD93CYA7c9PUX60ScSNs5u86XnrrcS9QmrPb7xNeXznws9GDB1jsFVPw0HiDjP1PFRUWWx4-qtnwhynRjdBC~JP3UWQviq5ergun2ecSuCvo~--uechGrOjoHU5y8zSY2XEUfG67E1S3BFB7w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst057/2/dst057-dst057supp_table3.xlsx?Expires=2147483647&amp;Signature=GOUhf1Q3jO-VYE6dkS3DjgbAok3pvE3YsCkiBJWSjVMuX2GbIRkD5SeqkaKhxfDR5i4x9U4OFlQOR0gZ1AWQZtzkjQ~NN9C1OJfDsjA8OeVxSOVVuS83iEgVRlXhGS6Obj2IsGEGvOOY-Kv0VJqiioKQlINkObmOu6QulyYcyxgZ6OVHMm008C0CIPNW7hUJE6u1zIgdqBNlTv2tZH6Htx3OeetCaixbvPXHM8wGX-sIsro9O94yTpMjWrDi53FH7aeP6MPmlDyRZwuSVZfLJYiq00~1nipGkXMtXk4yAC0h~P692o-f76CbBUGWcNvifhb9oeeTI5AySYj-Qsu3jQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst057/2/dst057-dst057supp_table2.xlsx?Expires=2147483647&amp;Signature=Srren2iUGx6yfsqIV-l97VZwwhm1r--Fwyz-EhHeIGMW6yebp8JwPNRv3KEZ~S9IXi5kAFOCdr4Mo7mcSEAoSym51Uddls4G3-45CwRdimBk2xeysVqsdy7k~Zs~SL~xcqdbU6aY-lUUDzAvDR8HeYw5NXM9LZD-o-ktqxwDKbCeoPiFP4e2KhMtWf~6NSddZc8J9zUpBoqHfY0afjh1lZHjOqARnw6dM6QBum0SWvgxiCvcxueerP40kSBTeXWESe1FUZHwWs~iwnzsRhkCsKdV1MRlp6GjnoBsAmex1DUab7-2hVE4ceX5XTgs5CCGBoFfo9XjktqHXTVAA4zhGQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst057/2/dst057-dst057supp_table1.xlsx?Expires=2147483647&amp;Signature=xbgut5P5jlAlEQ4iCoTpqXeY8UL7pNYxjPL94yHBK0ndo1TbsbO8gq6IND6crcSRB3v-VirVRwkOi~O21JrPjElrocsu-En3Mp~nJDNqkJjN~ghT1BISdNtRQzRklALACdonH3A6H~dlDNLIWhNOyDKmlW-U1SwZ~xiw83VVkXQJnKHkLoI8Zip4j6nBSE9kjrp3lOe24j9~LvVPeOtvGTiW7S3CLrAc5hfs8j-QeHUfdz72y4~p44v2KrwtTn4hFsu-ZK2VbuGTDmRPZgK6uJQ1-VGTOi~NAuVUU1oXkQ6rc3qywnsywkJG-I3ngLz495gfAUbe6XUZvNKxVRk6Tg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst057/2/dst057-dst057supp.pdf?Expires=2147483647&amp;Signature=3iDxAgXIL4ctv-wm5w~H73~RBVolOtrekA-7boka0OB0bHODr0G02Gkf5ZOEn82Gc0pSIHWTg35r1fUCOlyLJsCBhPObrwN8EHxuC4xmrDM4dq9dCCVwEa9dXPKzk2enLfQwzapgvtLAst7YsqWQySQiJ-ZIx4XF-EtDLbanR4Vq-ceobByVeSRZSEodh9X6D9oGqSwMXaByV8w-Lu7R7yvMKfHRBPj9ddgXFXS3GfnmlWZMRaXbYMMBaqB1llvCVrmeEWzvurLwndSPy~iNWnAMrRzVW5N6iNoVwz4hBD~F0CWhicZtNEK2apq4Z9kMTAQ75A5hFJMZy9aRc6r17Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">334</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897900005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">7</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst057"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="393995"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897900005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f3%2f285%2f393995">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000339897900005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (7)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F3%2F285%2F393995" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/393995'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Assessing Diversity of DNA Structure-Related Sequence Features in Prokaryotic Genomes" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst057");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f3%2f285%2f393995">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

