

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/3/219/591394","@type":"ScholarlyArticle","name":"Dissecting the fungal biology of  Bipolaris papendorfii : from phylogenetic to comparative genomic analysis","datePublished":"2015-04-27","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2015-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv007","keywords":["dematiaceous","<italic>Bipolaris papendorfii</italic>","<italic>Cochliobolus</italic>","<italic>Curvularia</italic>","LysM domain"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/3/219/591394","author":[{"name":"Kuan, Chee Sian","affiliation":"1  Department of Medical Microbiology, Faculty of Medicine ,  University of Malaya ,  Kuala Lumpur 50603 ,  Malaysia","@type":"Person"},{"name":"Yew, Su Mei","affiliation":"1  Department of Medical Microbiology, Faculty of Medicine ,  University of Malaya ,  Kuala Lumpur 50603 ,  Malaysia","@type":"Person"},{"name":"Toh, Yue Fen","affiliation":"1  Department of Medical Microbiology, Faculty of Medicine ,  University of Malaya ,  Kuala Lumpur 50603 ,  Malaysia","@type":"Person"},{"name":"Chan, Chai Ling","affiliation":"1  Department of Medical Microbiology, Faculty of Medicine ,  University of Malaya ,  Kuala Lumpur 50603 ,  Malaysia","@type":"Person"},{"name":"Ngeow, Yun Fong","affiliation":"1  Department of Medical Microbiology, Faculty of Medicine ,  University of Malaya ,  Kuala Lumpur 50603 ,  Malaysia","@type":"Person"},{"name":"Lee, Kok Wei","affiliation":"2  Codon Genomics SB, Jalan Bandar Lapan Belas ,  Selangor Darul Ehsan 47160 ,  Malaysia","@type":"Person"},{"name":"Na, Shiang Ling","affiliation":"1  Department of Medical Microbiology, Faculty of Medicine ,  University of Malaya ,  Kuala Lumpur 50603 ,  Malaysia","@type":"Person"},{"name":"Yee, Wai-Yan","affiliation":"2  Codon Genomics SB, Jalan Bandar Lapan Belas ,  Selangor Darul Ehsan 47160 ,  Malaysia","@type":"Person"},{"name":"Hoh, Chee-Choong","affiliation":"2  Codon Genomics SB, Jalan Bandar Lapan Belas ,  Selangor Darul Ehsan 47160 ,  Malaysia","@type":"Person"},{"name":"Ng, Kee Peng","affiliation":"1  Department of Medical Microbiology, Faculty of Medicine ,  University of Malaya ,  Kuala Lumpur 50603 ,  Malaysia","@type":"Person"}],"description":"Abstract.  Bipolaris papendorfii has been reported as a fungal plant pathogen that rarely causes opportunistic infection in humans. Secondary metabolites isolat","pageStart":"219","pageEnd":"232","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/3/1/m_cover.gif?Expires=2147483647&Signature=o~ckdHMyoALozjazxFdJARrSll6HI0qZKNez1rp60DiaVqyzX7KX3gffSJBpNovfc7x0FBbw5M~q-oXkBI0BrEKD0pgvQT5eVNC4LZt8HYW1wCNn7TYsIJ~iamj5~PWBqwfQdRx7Lwp~5y0cfA6QZfzXvHT-c2y8KfWBVWCVg9vuL7VY9lDugsDIpyrHTWTMWH4KR~-txVxPM4XBFIVE2oVLkX6QYMzija1YcuQl4yisR1R~qhzzHGrjVvva6IGLptvPDIFfjhAoXGwaWNy8qGaYVTkt5JCmjQmDV1APqpGYq6f9-3DKz0Py3DzfqJ6xp8-SqVKAnV4V9jMcimV1eA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Dissecting the fungal biology of  Bipolaris papendorfii : from phylogenetic to comparative genomic analysis","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/3/1/m_cover.gif?Expires=2147483647&Signature=o~ckdHMyoALozjazxFdJARrSll6HI0qZKNez1rp60DiaVqyzX7KX3gffSJBpNovfc7x0FBbw5M~q-oXkBI0BrEKD0pgvQT5eVNC4LZt8HYW1wCNn7TYsIJ~iamj5~PWBqwfQdRx7Lwp~5y0cfA6QZfzXvHT-c2y8KfWBVWCVg9vuL7VY9lDugsDIpyrHTWTMWH4KR~-txVxPM4XBFIVE2oVLkX6QYMzija1YcuQl4yisR1R~qhzzHGrjVvva6IGLptvPDIFfjhAoXGwaWNy8qGaYVTkt5JCmjQmDV1APqpGYq6f9-3DKz0Py3DzfqJ6xp8-SqVKAnV4V9jMcimV1eA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Dissecting the fungal biology of  Bipolaris papendorfii : from phylogenetic to comparative genomic analysis" />
    <meta property="og:description" content="Abstract.  Bipolaris papendorfii has been reported as a fungal plant pathogen that rarely causes opportunistic infection in humans. Secondary metabolites isolat" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/3/219/591394" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-04-27" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Kuan, Chee Sian" /><meta name="citation_author_institution" content="Department of Medical Microbiology, Faculty of Medicine, University of Malaya, Kuala Lumpur 50603, Malaysia" /><meta name="citation_author" content="Yew, Su Mei" /><meta name="citation_author_institution" content="Department of Medical Microbiology, Faculty of Medicine, University of Malaya, Kuala Lumpur 50603, Malaysia" /><meta name="citation_author" content="Toh, Yue Fen" /><meta name="citation_author_institution" content="Department of Medical Microbiology, Faculty of Medicine, University of Malaya, Kuala Lumpur 50603, Malaysia" /><meta name="citation_author" content="Chan, Chai Ling" /><meta name="citation_author_institution" content="Department of Medical Microbiology, Faculty of Medicine, University of Malaya, Kuala Lumpur 50603, Malaysia" /><meta name="citation_author" content="Ngeow, Yun Fong" /><meta name="citation_author_institution" content="Department of Medical Microbiology, Faculty of Medicine, University of Malaya, Kuala Lumpur 50603, Malaysia" /><meta name="citation_author" content="Lee, Kok Wei" /><meta name="citation_author_institution" content="Codon Genomics SB, Jalan Bandar Lapan Belas, Selangor Darul Ehsan 47160, Malaysia" /><meta name="citation_author" content="Na, Shiang Ling" /><meta name="citation_author_institution" content="Department of Medical Microbiology, Faculty of Medicine, University of Malaya, Kuala Lumpur 50603, Malaysia" /><meta name="citation_author" content="Yee, Wai-Yan" /><meta name="citation_author_institution" content="Codon Genomics SB, Jalan Bandar Lapan Belas, Selangor Darul Ehsan 47160, Malaysia" /><meta name="citation_author" content="Hoh, Chee-Choong" /><meta name="citation_author_institution" content="Codon Genomics SB, Jalan Bandar Lapan Belas, Selangor Darul Ehsan 47160, Malaysia" /><meta name="citation_author" content="Ng, Kee Peng" /><meta name="citation_author_institution" content="Department of Medical Microbiology, Faculty of Medicine, University of Malaya, Kuala Lumpur 50603, Malaysia" /><meta name="citation_title" content="Dissecting the fungal biology of Bipolaris papendorfii: from phylogenetic to comparative genomic analysis" /><meta name="citation_firstpage" content="219" /><meta name="citation_lastpage" content="232" /><meta name="citation_doi" content="10.1093/dnares/dsv007" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2015/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Diversity of Bipolaris species in clinical samples in the United States and their antifungal susceptibility profiles; citation_author=da Cunha  K.C.; citation_author=Sutton  D.A.; citation_author=Fothergill  A.W.;  citation_journal_title=J. Clin. Microbiol.;  citation_year=2012;  citation_volume=50;  citation_pages=4061-6; " /><meta name="citation_reference" content="citation_title=A phylogenetic and taxonomic re-evaluation of the Bipolaris - Cochliobolus - Curvularia complex; citation_author=Manamgoda  D.S.; citation_author=Cai  L.; citation_author=McKenzie  E.H.C.;  citation_journal_title=Fungal Diver.;  citation_year=2012;  citation_volume=56;  citation_pages=131-44; " /><meta name="citation_reference" content="citation_title=Cochliobolus: an overview and current status of species; citation_author=Manamgoda  D.S.; citation_author=Cai  L.; citation_author=Bahkali  A.H.; citation_author=Chukeatirote  E.; citation_author=Hyde  K.D.;  citation_journal_title=Fungal Divers;  citation_year=2011;  citation_volume=51;  citation_pages=3-42; " /><meta name="citation_reference" content="citation_title=The southern corn leaf blight epidemic; citation_author=Tatum  L.A.;  citation_journal_title=Science;  citation_year=1971;  citation_volume=171;  citation_pages=1113-6; " /><meta name="citation_reference" content="citation_author=Scheffer  R.P.; citation_publisher=Cambridge University Press, New York;  citation_title=The nature of disease in plants;  citation_year=1997; " /><meta name="citation_reference" content="citation_title=Comparative genome structure, secondary metabolite, and effector coding capacity across Cochliobolus pathogens; citation_author=Condon  B.J.; citation_author=Leng  Y.; citation_author=Wu  D.;  citation_journal_title=PLoS Genet.;  citation_year=2013;  citation_volume=9;  citation_pages=e1003233" /><meta name="citation_reference" content="citation_title=Epidemiology, clinical manifestations, and therapy of infections caused by dematiaceous fungi; citation_author=Brandt  M.E.; citation_author=Warnock  D.W.;  citation_journal_title=J. Chemother.;  citation_year=2003;  citation_volume=15;  citation_issue=(Suppl. 2);  citation_pages=36-47; " /><meta name="citation_reference" content="citation_author=de Hoog  G.S.; citation_author=Guarro  J.; citation_author=Gené  J.; citation_author=Figueras  M.J.; citation_publisher=Centraalbureau Voor Schimmelcultures, Baarn, the Netherlands;  citation_title=Atlas of clinical fungi;  citation_year=2005; " /><meta name="citation_reference" content="citation_title=First report of Bipolaris papendorfii causing corn leaf spot in China; citation_author=Li  P.P.; citation_author=Cao  Z.Y.;  citation_journal_title=Plant Dis.;  citation_year=2013;  citation_volume=97;  citation_pages=1506" /><meta name="citation_reference" content="citation_title=Antiproliferative activity of hamigerone and radicinol isolated from Bipolaris papendorfii; citation_author=Giridharan  P.; citation_author=Verekar  S.A.; citation_author=Gohil  A.R.; citation_author=Mishra  P.D.; citation_author=Khanna  A.; citation_author=Deshmukh  S.K.;  citation_journal_title=Biomed. Res. Int.;  citation_year=2014;  citation_volume=2014;  citation_pages=890904" /><meta name="citation_reference" content="citation_title=A five-year survey of dematiaceous fungi in a tropical hospital reveals potential opportunistic species; citation_author=Yew  S.M.; citation_author=Chan  C.L.; citation_author=Lee  K.W.;  citation_journal_title=PLoS ONE;  citation_year=2014;  citation_volume=9;  citation_pages=e104352" /><meta name="citation_reference" content="citation_title=The M-Coffee web server: a meta-method for computing multiple sequence alignments by combining alternative alignment methods; citation_author=Moretti  S.; citation_author=Armougom  F.; citation_author=Wallace  I.M.; citation_author=Higgins  D.G.; citation_author=Jongeneel  C.V.; citation_author=Notredame  C.;  citation_journal_title=Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=W645-8; " /><meta name="citation_reference" content="citation_title=MrBayes 3: Bayesian phylogenetic inference under mixed models; citation_author=Ronquist  F.; citation_author=Huelsenbeck  J.P.;  citation_journal_title=Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=1572-4; " /><meta name="citation_reference" content="citation_title=An efficient method for DNA extraction from Cladosporioid fungi; citation_author=Moslem  M.A.; citation_author=Bahkali  A.H.; citation_author=Abd-Elsalam  K.A.; citation_author=Wit  P.J.;  citation_journal_title=Genet. Mol. Res.;  citation_year=2010;  citation_volume=9;  citation_pages=2283-91; " /><meta name="citation_reference" content="citation_title=Error correction and assembly complexity of single molecule sequencing reads; citation_author=Lee  H.; citation_author=Gurtowski  J.; citation_author=Yoo  S.; citation_author=Marcus  S.; citation_author=McCombie  W.R.; citation_author=Schatz  M.;  citation_year=2014;  citation_pages=1-17; " /><meta name="citation_reference" content="citation_title=Quake: quality-aware detection and correction of sequencing errors; citation_author=Kelley  D.R.; citation_author=Schatz  M.C.; citation_author=Salzberg  S.L.;  citation_journal_title=Genome Biol.;  citation_year=2010;  citation_volume=11;  citation_pages=R116" /><meta name="citation_reference" content="citation_title=A whole-genome assembly of Drosophila; citation_author=Myers  E.W.; citation_author=Sutton  G.G.; citation_author=Delcher  A.L.;  citation_journal_title=Science;  citation_year=2000;  citation_volume=287;  citation_pages=2196-204; " /><meta name="citation_reference" content="citation_title=Gene identification in novel eukaryotic genomes by self-training algorithm; citation_author=Lomsadze  A.; citation_author=Ter-Hovhannisyan  V.; citation_author=Chernoff  Y.O.; citation_author=Borodovsky  M.;  citation_journal_title=Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=6494-506; " /><meta name="citation_reference" content="citation_title=Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research; citation_author=Conesa  A.; citation_author=Gotz  S.; citation_author=Garcia-Gomez  J.M.; citation_author=Terol  J.; citation_author=Talon  M.; citation_author=Robles  M.;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=3674-6; " /><meta name="citation_reference" content="citation_title=The COG database: an updated version includes eukaryotes; citation_author=Tatusov  R.L.; citation_author=Fedorova  N.D.; citation_author=Jackson  J.D.;  citation_journal_title=BMC Bioinformatics;  citation_year=2003;  citation_volume=4;  citation_pages=41" /><meta name="citation_reference" content="citation_title=InterProScan: protein domains identifier; citation_author=Quevillon  E.; citation_author=Silventoinen  V.; citation_author=Pillai  S.;  citation_journal_title=Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=W116-20; " /><meta name="citation_reference" content="citation_title=RNAmmer: consistent and rapid annotation of ribosomal RNA genes; citation_author=Lagesen  K.; citation_author=Hallin  P.; citation_author=Rodland  E.A.; citation_author=Staerfeldt  H.H.; citation_author=Rognes  T.; citation_author=Ussery  D.W.;  citation_journal_title=Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=3100-8; " /><meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence; citation_author=Lowe  T.M.; citation_author=Eddy  S.R.;  citation_journal_title=Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=955-64; " /><meta name="citation_reference" content="citation_title=dbCAN: a web resource for automated carbohydrate-active enzyme annotation; citation_author=Yin  Y.; citation_author=Mao  X.; citation_author=Yang  J.; citation_author=Chen  X.; citation_author=Mao  F.; citation_author=Xu  Y.;  citation_journal_title=Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=W445-51; " /><meta name="citation_reference" content="citation_title=SignalP 4.0: discriminating signal peptides from transmembrane regions; citation_author=Petersen  T.N.; citation_author=Brunak  S.; citation_author=von Heijne  G.; citation_author=Nielsen  H.;  citation_journal_title=Nat. Methods;  citation_year=2011;  citation_volume=8;  citation_pages=785-6; " /><meta name="citation_reference" content="citation_title=Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes; citation_author=Krogh  A.; citation_author=Larsson  B.; citation_author=von Heijne  G.; citation_author=Sonnhammer  E.L.;  citation_journal_title=J. Mol. Biol.;  citation_year=2001;  citation_volume=305;  citation_pages=567-80; " /><meta name="citation_reference" content="citation_title=SMURF: Genomic mapping of fungal secondary metabolite clusters; citation_author=Khaldi  N.; citation_author=Seifuddin  F.T.; citation_author=Turner  G.;  citation_journal_title=Fungal Genet. Biol.;  citation_year=2010;  citation_volume=47;  citation_pages=736-41; " /><meta name="citation_reference" content="citation_title=Diverse lifestyles and strategies of plant pathogenesis encoded in the genomes of eighteen Dothideomycetes fungi; citation_author=Ohm  R.A.; citation_author=Feau  N.; citation_author=Henrissat  B.;  citation_journal_title=PLoS Pathog.;  citation_year=2012;  citation_volume=8;  citation_pages=e1003037" /><meta name="citation_reference" content="citation_author=Dean  R.A.; citation_author=Ann  L.P.; citation_author=Chittaranjan  K.; citation_publisher=Springer, Berlin, Heidelberg;  citation_title=Genomics of plant-associated fungi: monocot pathogens;  citation_year=2014;  citation_pages=41-67; " /><meta name="citation_reference" content="citation_title=Protein annotation and modelling servers at University College London; citation_author=Buchan  D.W.; citation_author=Ward  S.M.; citation_author=Lobley  A.E.; citation_author=Nugent  T.C.; citation_author=Bryson  K.; citation_author=Jones  D.T.;  citation_journal_title=Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=W563-8; " /><meta name="citation_reference" content="citation_title=SWISS-MODEL: modelling protein tertiary and quaternary structure using evolutionary information; citation_author=Biasini  M.; citation_author=Bienert  S.; citation_author=Waterhouse  A.;  citation_journal_title=Nucleic Acids Res.;  citation_year=2014;  citation_volume=42;  citation_pages=W252-8; " /><meta name="citation_reference" content="citation_title=OrthoMCL: identification of ortholog groups for eukaryotic genomes; citation_author=Li  L.; citation_author=Stoeckert  C.J.,; citation_author=Roos  D.S.;  citation_journal_title=Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=2178-89; " /><meta name="citation_reference" content="citation_title=Clustal W and Clustal X version 2.0; citation_author=Larkin  M.A.; citation_author=Blackshields  G.; citation_author=Brown  N.P.;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2947-8; " /><meta name="citation_reference" content="citation_title=Fungal effector Ecp6 outcompetes host immune receptor for chitin binding through intrachain LysM dimerization; citation_author=Sanchez-Vallet  A.; citation_author=Saleem-Batcha  R.; citation_author=Kombrink  A.;  citation_journal_title=Elife;  citation_year=2013;  citation_volume=2;  citation_pages=e00790" /><meta name="citation_reference" content="citation_title=SWISS-MODEL and the Swiss-PdbViewer: an environment for comparative protein modeling; citation_author=Guex  N.; citation_author=Peitsch  M.C.;  citation_journal_title=Electrophoresis;  citation_year=1997;  citation_volume=18;  citation_pages=2714-23; " /><meta name="citation_reference" content="citation_title=Sequence analysis and homology modeling of glyceraldehyde-3-phosphate dehydrogenase of an antarctic psychrophilic Pseudomonas sp; citation_author=Kuan  C.S.; citation_author=Tien  C.F.; citation_author=See Too  W.C.; citation_author=Few  L.L.;  citation_journal_title=Afr. J. Microbiol. Res.;  citation_year=2012;  citation_volume=6;  citation_pages=881-91; " /><meta name="citation_reference" content="citation_title=Artemis: sequence visualization and annotation; citation_author=Rutherford  K.; citation_author=Parkhill  J.; citation_author=Crook  J.;  citation_journal_title=Bioinformatics;  citation_year=2000;  citation_volume=16;  citation_pages=944-5; " /><meta name="citation_reference" content="citation_title=Characterization and potential evolutionary impact of transposable elements in the genome of Cochliobolus heterostrophus; citation_author=Santana  M.F.; citation_author=Silva  J.C.; citation_author=Mizubuti  E.S.;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=536" /><meta name="citation_reference" content="citation_title=Non-LTR retrotransposons in fungi; citation_author=Novikova  O.; citation_author=Fet  V.; citation_author=Blinov  A.;  citation_journal_title=Funct. Integr. Genomics;  citation_year=2009;  citation_volume=9;  citation_pages=27-42; " /><meta name="citation_reference" content="citation_title=LTR retrotransposons in fungi; citation_author=Muszewska  A.; citation_author=Hoffman-Sommer  M.; citation_author=Grynberg  M.;  citation_journal_title=PLoS ONE;  citation_year=2011;  citation_volume=6;  citation_pages=e29425" /><meta name="citation_reference" content="citation_title=Assembly of the Tc1 and mariner transposition initiation complexes depends on the origins of their transposase DNA binding domains; citation_author=Brillet  B.; citation_author=Bigot  Y.; citation_author=Auge-Gouillou  C.;  citation_journal_title=Genetica;  citation_year=2007;  citation_volume=130;  citation_pages=105-20; " /><meta name="citation_reference" content="citation_title=The catalytic domain of all eukaryotic cut-and-paste transposase superfamilies; citation_author=Yuan  Y.W.; citation_author=Wessler  S.R.;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2011;  citation_volume=108;  citation_pages=7884-9; " /><meta name="citation_reference" content="citation_title=Posttranslational modifications of proteins in the pathobiology of medically relevant fungi; citation_author=Leach  M.D.; citation_author=Brown  A.J.;  citation_journal_title=Eukaryot. Cell;  citation_year=2012;  citation_volume=11;  citation_pages=98-108; " /><meta name="citation_reference" content="citation_title=Multifunction of autophagy-related genes in filamentous fungi; citation_author=Khan  I.A.; citation_author=Lu  J.P.; citation_author=Liu  X.H.; citation_author=Rehman  A.; citation_author=Lin  F.C.;  citation_journal_title=Microbiol. Res.;  citation_year=2012;  citation_volume=167;  citation_pages=339-45; " /><meta name="citation_reference" content="citation_title=Correction: comparative analysis of fungal genomes reveals different plant cell wall degrading capacity in fungi; citation_author=Zhao  Z.; citation_author=Liu  H.; citation_author=Wang  C.; citation_author=Xu  J.R.;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=6" /><meta name="citation_reference" content="citation_title=Bioinformatic characterisation of genes encoding cell wall degrading enzymes in the Phytophthora parasitica genome; citation_author=Blackman  L.M.; citation_author=Cullerne  D.P.; citation_author=Hardham  A.R.;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=785" /><meta name="citation_reference" content="citation_title=A symbiont-independent endo-1,4-beta-xylanase from the plant-parasitic nematode Meloidogyne incognita; citation_author=Mitreva-Dautova  M.; citation_author=Roze  E.; citation_author=Overmars  H.;  citation_journal_title=Mol. Plant Microb. Interact.;  citation_year=2006;  citation_volume=19;  citation_pages=521-9; " /><meta name="citation_reference" content="citation_title=Molecular cloning of fungal xylanases: an overview; citation_author=Ahmed  S.; citation_author=Riaz  S.; citation_author=Jamil  A.;  citation_journal_title=Appl. Microbiol. Biotechnol.;  citation_year=2009;  citation_volume=84;  citation_pages=19-35; " /><meta name="citation_reference" content="citation_title=Exploring laccase-like multicopper oxidase genes from the ascomycete Trichoderma reesei: a functional, phylogenetic and evolutionary study; citation_author=Levasseur  A.; citation_author=Saloheimo  M.; citation_author=Navarro  D.;  citation_journal_title=BMC Biochem.;  citation_year=2010;  citation_volume=11;  citation_pages=32" /><meta name="citation_reference" content="citation_title=Lignin peroxidase structure and function; citation_author=Piontek  K.; citation_author=Smith  A.T.; citation_author=Blodig  W.;  citation_journal_title=Biochem. Soc. Trans.;  citation_year=2001;  citation_volume=29;  citation_pages=111-6; " /><meta name="citation_reference" content="citation_title=LysM effectors: secreted proteins supporting fungal life; citation_author=Kombrink  A.; citation_author=Thomma  B.P.;  citation_journal_title=PLoS Pathog.;  citation_year=2013;  citation_volume=9;  citation_pages=e1003769" /><meta name="citation_reference" content="citation_title=The novel Cladosporium fulvum lysin motif effector Ecp6 is a virulence factor with orthologues in other fungal species; citation_author=Bolton  M.D.; citation_author=van Esse  H.P.; citation_author=Vossen  J.H.;  citation_journal_title=Mol. Microbiol.;  citation_year=2008;  citation_volume=69;  citation_pages=119-36; " /><meta name="citation_reference" content="citation_title=Fungal LysM effectors: extinguishers of host immunity?; citation_author=de Jonge  R.; citation_author=Thomma  B.P.;  citation_journal_title=Trends Microbiol.;  citation_year=2009;  citation_volume=17;  citation_pages=151-7; " /><meta name="citation_reference" content="citation_title=Application of mating type gene technology to problems in fungal biology; citation_author=Turgeon  B.G.;  citation_journal_title=Annu. Rev. Phytopathol.;  citation_year=1998;  citation_volume=36;  citation_pages=115-37; " /><meta name="citation_reference" content="citation_title=Fungal secondary metabolism - from biochemistry to genomics; citation_author=Keller  N.P.; citation_author=Turner  G.; citation_author=Bennett  J.W.;  citation_journal_title=Nat. Rev. Microbiol.;  citation_year=2005;  citation_volume=3;  citation_pages=937-47; " /><meta name="citation_reference" content="citation_title=The structure of dimethylallyl tryptophan synthase reveals a common architecture of aromatic prenyltransferases in fungi and bacteria; citation_author=Metzger  U.; citation_author=Schall  C.; citation_author=Zocher  G.;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=14309-14; " /><meta name="citation_reference" content="citation_title=Ergot alkaloids--biology and molecular biology; citation_author=Schardl  C.L.; citation_author=Panaccione  D.G.; citation_author=Tudzynski  P.;  citation_journal_title=Alkaloids Chem. Biol.;  citation_year=2006;  citation_volume=63;  citation_pages=45-86; " /><meta name="citation_reference" content="citation_title=Genetic and genomic dissection of the Cochliobolus heterostrophus Tox1 locus controlling biosynthesis of the polyketide virulence factor T-toxin; citation_author=Turgeon  B.G.; citation_author=Baker  S.E.;  citation_journal_title=Adv. Genet.;  citation_year=2007;  citation_volume=57;  citation_pages=219-61; " /><meta name="citation_reference" content="citation_title=Ionizing radiation changes the electronic properties of melanin and enhances the growth of melanized fungi; citation_author=Dadachova  E.; citation_author=Bryan  R.A.; citation_author=Huang  X.;  citation_journal_title=PLoS ONE;  citation_year=2007;  citation_volume=2;  citation_pages=e457" /><meta name="citation_reference" content="citation_title=Melanin biosynthesis in the maize pathogen Cochliobolus heterostrophus depends on two mitogen-activated protein kinases, Chk1 and Mps1, and the transcription factor Cmr1; citation_author=Eliahu  N.; citation_author=Igbaria  A.; citation_author=Rose  M.S.; citation_author=Horwitz  B.A.; citation_author=Lev  S.;  citation_journal_title=Eukaryot. Cell;  citation_year=2007;  citation_volume=6;  citation_pages=421-9; " /><meta name="citation_reference" content="citation_title=Novel fungal transcriptional activators, Cmr1p of Colletotrichum lagenarium and pig1p of Magnaporthe grisea, contain Cys2His2 zinc finger and Zn(II)2Cys6 binuclear cluster DNA-binding motifs and regulate transcription of melanin biosynthesis genes in a developmentally specific manner; citation_author=Tsuji  G.; citation_author=Kenmochi  Y.; citation_author=Takano  Y.;  citation_journal_title=Mol. Microbiol.;  citation_year=2000;  citation_volume=38;  citation_pages=940-54; " /><meta name="citation_reference" content="citation_title=The influence of natural products upon drug discovery; citation_author=Newman  D.J.; citation_author=Cragg  G.M.; citation_author=Snader  K.M.;  citation_journal_title=Nat. Prod. Rep.;  citation_year=2000;  citation_volume=17;  citation_pages=215-34; " /><meta name="citation_reference" content="citation_title=Caffeine consumption; citation_author=Gilbert  R.M.;  citation_journal_title=Prog. Clin. Biol. Res.;  citation_year=1984;  citation_volume=158;  citation_pages=185-213; " /><meta name="citation_reference" content="citation_title=Catabolic pathways and biotechnological applications of microbial caffeine degradation; citation_author=Dash  S.S.; citation_author=Gummadi  S.N.;  citation_journal_title=Biotechnol. Lett.;  citation_year=2006;  citation_volume=28;  citation_pages=1993-2002; " /><meta name="citation_reference" content="citation_title=Catabolism of caffeine in plants and microorganisms; citation_author=Mazzafera  P.;  citation_journal_title=Front Biosci.;  citation_year=2004;  citation_volume=9;  citation_pages=1348-59; " /><meta name="citation_reference" content="citation_title=Comparison of the cytochrome P-450 containing monooxigenases originating from two different yeasts; citation_author=Sauer  M.;  citation_journal_title=Dev. Biochem.;  citation_year=1982;  citation_volume=23;  citation_pages=452-7; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/3/219/1593822/dsv007.pdf" /><meta name="description" content="Abstract.  Bipolaris papendorfii has been reported as a fungal plant pathogen that rarely causes opportunistic infection in humans. Secondary metabolites isolat" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/3/219/591394" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/3/219/591394" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/3/219/591394");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv007';
    </script>
<title>
	Dissecting the fungal biology of Bipolaris papendorfii: from phylogenetic to comparative genomic analysis | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./591394" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbvJATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3VhbiwgQ2hlZSBTaWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNZWRpY2FsIE1pY3JvYmlvbG9neSwgRmFjdWx0eSBvZiBNZWRpY2luZSwgVW5pdmVyc2l0eSBvZiBNYWxheWEsIEt1YWxhIEx1bXB1ciA1MDYwMywgTWFsYXlzaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZZXcsIFN1IE1laSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTWVkaWNhbCBNaWNyb2Jpb2xvZ3ksIEZhY3VsdHkgb2YgTWVkaWNpbmUsIFVuaXZlcnNpdHkgb2YgTWFsYXlhLCBLdWFsYSBMdW1wdXIgNTA2MDMsIE1hbGF5c2lhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVG9oLCBZdWUgRmVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNZWRpY2FsIE1pY3JvYmlvbG9neSwgRmFjdWx0eSBvZiBNZWRpY2luZSwgVW5pdmVyc2l0eSBvZiBNYWxheWEsIEt1YWxhIEx1bXB1ciA1MDYwMywgTWFsYXlzaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDaGFuLCBDaGFpIExpbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1lZGljYWwgTWljcm9iaW9sb2d5LCBGYWN1bHR5IG9mIE1lZGljaW5lLCBVbml2ZXJzaXR5IG9mIE1hbGF5YSwgS3VhbGEgTHVtcHVyIDUwNjAzLCBNYWxheXNpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5nZW93LCBZdW4gRm9uZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTWVkaWNhbCBNaWNyb2Jpb2xvZ3ksIEZhY3VsdHkgb2YgTWVkaWNpbmUsIFVuaXZlcnNpdHkgb2YgTWFsYXlhLCBLdWFsYSBMdW1wdXIgNTA2MDMsIE1hbGF5c2lhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGVlLCBLb2sgV2VpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ29kb24gR2Vub21pY3MgU0IsIEphbGFuIEJhbmRhciBMYXBhbiBCZWxhcywgU2VsYW5nb3IgRGFydWwgRWhzYW4gNDcxNjAsIE1hbGF5c2lhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmEsIFNoaWFuZyBMaW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNZWRpY2FsIE1pY3JvYmlvbG9neSwgRmFjdWx0eSBvZiBNZWRpY2luZSwgVW5pdmVyc2l0eSBvZiBNYWxheWEsIEt1YWxhIEx1bXB1ciA1MDYwMywgTWFsYXlzaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZZWUsIFdhaS1ZYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDb2RvbiBHZW5vbWljcyBTQiwgSmFsYW4gQmFuZGFyIExhcGFuIEJlbGFzLCBTZWxhbmdvciBEYXJ1bCBFaHNhbiA0NzE2MCwgTWFsYXlzaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIb2gsIENoZWUtQ2hvb25nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ29kb24gR2Vub21pY3MgU0IsIEphbGFuIEJhbmRhciBMYXBhbiBCZWxhcywgU2VsYW5nb3IgRGFydWwgRWhzYW4gNDcxNjAsIE1hbGF5c2lhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmcsIEtlZSBQZW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNZWRpY2FsIE1pY3JvYmlvbG9neSwgRmFjdWx0eSBvZiBNZWRpY2luZSwgVW5pdmVyc2l0eSBvZiBNYWxheWEsIEt1YWxhIEx1bXB1ciA1MDYwMywgTWFsYXlzaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkRpc3NlY3RpbmcgdGhlIGZ1bmdhbCBiaW9sb2d5IG9mIEJpcG9sYXJpcyBwYXBlbmRvcmZpaTogZnJvbSBwaHlsb2dlbmV0aWMgdG8gY29tcGFyYXRpdmUgZ2Vub21pYyBhbmFseXNpcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjIxOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMjMyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN2MDA3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTUvMDYvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXZlcnNpdHkgb2YgQmlwb2xhcmlzIHNwZWNpZXMgaW4gY2xpbmljYWwgc2FtcGxlcyBpbiB0aGUgVW5pdGVkIFN0YXRlcyBhbmQgdGhlaXIgYW50aWZ1bmdhbCBzdXNjZXB0aWJpbGl0eSBwcm9maWxlczsgY2l0YXRpb25fYXV0aG9yPWRhIEN1bmhhICBLLkMuOyBjaXRhdGlvbl9hdXRob3I9U3V0dG9uICBELkEuOyBjaXRhdGlvbl9hdXRob3I9Rm90aGVyZ2lsbCAgQS5XLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQ2xpbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT01MDsgIGNpdGF0aW9uX3BhZ2VzPTQwNjEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwaHlsb2dlbmV0aWMgYW5kIHRheG9ub21pYyByZS1ldmFsdWF0aW9uIG9mIHRoZSBCaXBvbGFyaXMgLSBDb2NobGlvYm9sdXMgLSBDdXJ2dWxhcmlhIGNvbXBsZXg7IGNpdGF0aW9uX2F1dGhvcj1NYW5hbWdvZGEgIEQuUy47IGNpdGF0aW9uX2F1dGhvcj1DYWkgIEwuOyBjaXRhdGlvbl9hdXRob3I9TWNLZW56aWUgIEUuSC5DLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnVuZ2FsIERpdmVyLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT01NjsgIGNpdGF0aW9uX3BhZ2VzPTEzMS00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29jaGxpb2JvbHVzOiBhbiBvdmVydmlldyBhbmQgY3VycmVudCBzdGF0dXMgb2Ygc3BlY2llczsgY2l0YXRpb25fYXV0aG9yPU1hbmFtZ29kYSAgRC5TLjsgY2l0YXRpb25fYXV0aG9yPUNhaSAgTC47IGNpdGF0aW9uX2F1dGhvcj1CYWhrYWxpICBBLkguOyBjaXRhdGlvbl9hdXRob3I9Q2h1a2VhdGlyb3RlICBFLjsgY2l0YXRpb25fYXV0aG9yPUh5ZGUgIEsuRC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZ1bmdhbCBEaXZlcnM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NTE7ICBjaXRhdGlvbl9wYWdlcz0zLTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc291dGhlcm4gY29ybiBsZWFmIGJsaWdodCBlcGlkZW1pYzsgY2l0YXRpb25fYXV0aG9yPVRhdHVtICBMLkEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTcxOyAgY2l0YXRpb25fdm9sdW1lPTE3MTsgIGNpdGF0aW9uX3BhZ2VzPTExMTMtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fYXV0aG9yPVNjaGVmZmVyICBSLlAuOyBjaXRhdGlvbl9wdWJsaXNoZXI9Q2FtYnJpZGdlIFVuaXZlcnNpdHkgUHJlc3MsIE5ldyBZb3JrOyAgY2l0YXRpb25fdGl0bGU9VGhlIG5hdHVyZSBvZiBkaXNlYXNlIGluIHBsYW50czsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgZ2Vub21lIHN0cnVjdHVyZSwgc2Vjb25kYXJ5IG1ldGFib2xpdGUsIGFuZCBlZmZlY3RvciBjb2RpbmcgY2FwYWNpdHkgYWNyb3NzIENvY2hsaW9ib2x1cyBwYXRob2dlbnM7IGNpdGF0aW9uX2F1dGhvcj1Db25kb24gIEIuSi47IGNpdGF0aW9uX2F1dGhvcj1MZW5nICBZLjsgY2l0YXRpb25fYXV0aG9yPVd1ICBELjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAzMjMzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXBpZGVtaW9sb2d5LCBjbGluaWNhbCBtYW5pZmVzdGF0aW9ucywgYW5kIHRoZXJhcHkgb2YgaW5mZWN0aW9ucyBjYXVzZWQgYnkgZGVtYXRpYWNlb3VzIGZ1bmdpOyBjaXRhdGlvbl9hdXRob3I9QnJhbmR0ICBNLkUuOyBjaXRhdGlvbl9hdXRob3I9V2Fybm9jayAgRC5XLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQ2hlbW90aGVyLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX2lzc3VlPShTdXBwbC4gMik7ICBjaXRhdGlvbl9wYWdlcz0zNi00NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fYXV0aG9yPWRlIEhvb2cgIEcuUy47IGNpdGF0aW9uX2F1dGhvcj1HdWFycm8gIEouOyBjaXRhdGlvbl9hdXRob3I9R2Vuw6kgIEouOyBjaXRhdGlvbl9hdXRob3I9RmlndWVyYXMgIE0uSi47IGNpdGF0aW9uX3B1Ymxpc2hlcj1DZW50cmFhbGJ1cmVhdSBWb29yIFNjaGltbWVsY3VsdHVyZXMsIEJhYXJuLCB0aGUgTmV0aGVybGFuZHM7ICBjaXRhdGlvbl90aXRsZT1BdGxhcyBvZiBjbGluaWNhbCBmdW5naTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Rmlyc3QgcmVwb3J0IG9mIEJpcG9sYXJpcyBwYXBlbmRvcmZpaSBjYXVzaW5nIGNvcm4gbGVhZiBzcG90IGluIENoaW5hOyBjaXRhdGlvbl9hdXRob3I9TGkgIFAuUC47IGNpdGF0aW9uX2F1dGhvcj1DYW8gIFouWS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IERpcy47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9OTc7ICBjaXRhdGlvbl9wYWdlcz0xNTA2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW50aXByb2xpZmVyYXRpdmUgYWN0aXZpdHkgb2YgaGFtaWdlcm9uZSBhbmQgcmFkaWNpbm9sIGlzb2xhdGVkIGZyb20gQmlwb2xhcmlzIHBhcGVuZG9yZmlpOyBjaXRhdGlvbl9hdXRob3I9R2lyaWRoYXJhbiAgUC47IGNpdGF0aW9uX2F1dGhvcj1WZXJla2FyICBTLkEuOyBjaXRhdGlvbl9hdXRob3I9R29oaWwgIEEuUi47IGNpdGF0aW9uX2F1dGhvcj1NaXNocmEgIFAuRC47IGNpdGF0aW9uX2F1dGhvcj1LaGFubmEgIEEuOyBjaXRhdGlvbl9hdXRob3I9RGVzaG11a2ggIFMuSy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb21lZC4gUmVzLiBJbnQuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTIwMTQ7ICBjaXRhdGlvbl9wYWdlcz04OTA5MDQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGZpdmUteWVhciBzdXJ2ZXkgb2YgZGVtYXRpYWNlb3VzIGZ1bmdpIGluIGEgdHJvcGljYWwgaG9zcGl0YWwgcmV2ZWFscyBwb3RlbnRpYWwgb3Bwb3J0dW5pc3RpYyBzcGVjaWVzOyBjaXRhdGlvbl9hdXRob3I9WWV3ICBTLk0uOyBjaXRhdGlvbl9hdXRob3I9Q2hhbiAgQy5MLjsgY2l0YXRpb25fYXV0aG9yPUxlZSAgSy5XLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWUxMDQzNTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgTS1Db2ZmZWUgd2ViIHNlcnZlcjogYSBtZXRhLW1ldGhvZCBmb3IgY29tcHV0aW5nIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudHMgYnkgY29tYmluaW5nIGFsdGVybmF0aXZlIGFsaWdubWVudCBtZXRob2RzOyBjaXRhdGlvbl9hdXRob3I9TW9yZXR0aSAgUy47IGNpdGF0aW9uX2F1dGhvcj1Bcm1vdWdvbSAgRi47IGNpdGF0aW9uX2F1dGhvcj1XYWxsYWNlICBJLk0uOyBjaXRhdGlvbl9hdXRob3I9SGlnZ2lucyAgRC5HLjsgY2l0YXRpb25fYXV0aG9yPUpvbmdlbmVlbCAgQy5WLjsgY2l0YXRpb25fYXV0aG9yPU5vdHJlZGFtZSAgQy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPVc2NDUtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXJCYXllcyAzOiBCYXllc2lhbiBwaHlsb2dlbmV0aWMgaW5mZXJlbmNlIHVuZGVyIG1peGVkIG1vZGVsczsgY2l0YXRpb25fYXV0aG9yPVJvbnF1aXN0ICBGLjsgY2l0YXRpb25fYXV0aG9yPUh1ZWxzZW5iZWNrICBKLlAuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTE1NzItNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gZWZmaWNpZW50IG1ldGhvZCBmb3IgRE5BIGV4dHJhY3Rpb24gZnJvbSBDbGFkb3Nwb3Jpb2lkIGZ1bmdpOyBjaXRhdGlvbl9hdXRob3I9TW9zbGVtICBNLkEuOyBjaXRhdGlvbl9hdXRob3I9QmFoa2FsaSAgQS5ILjsgY2l0YXRpb25fYXV0aG9yPUFiZC1FbHNhbGFtICBLLkEuOyBjaXRhdGlvbl9hdXRob3I9V2l0ICBQLkouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldC4gTW9sLiBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0yMjgzLTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FcnJvciBjb3JyZWN0aW9uIGFuZCBhc3NlbWJseSBjb21wbGV4aXR5IG9mIHNpbmdsZSBtb2xlY3VsZSBzZXF1ZW5jaW5nIHJlYWRzOyBjaXRhdGlvbl9hdXRob3I9TGVlICBILjsgY2l0YXRpb25fYXV0aG9yPUd1cnRvd3NraSAgSi47IGNpdGF0aW9uX2F1dGhvcj1Zb28gIFMuOyBjaXRhdGlvbl9hdXRob3I9TWFyY3VzICBTLjsgY2l0YXRpb25fYXV0aG9yPU1jQ29tYmllICBXLlIuOyBjaXRhdGlvbl9hdXRob3I9U2NoYXR6ICBNLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3BhZ2VzPTEtMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVF1YWtlOiBxdWFsaXR5LWF3YXJlIGRldGVjdGlvbiBhbmQgY29ycmVjdGlvbiBvZiBzZXF1ZW5jaW5nIGVycm9yczsgY2l0YXRpb25fYXV0aG9yPUtlbGxleSAgRC5SLjsgY2l0YXRpb25fYXV0aG9yPVNjaGF0eiAgTS5DLjsgY2l0YXRpb25fYXV0aG9yPVNhbHpiZXJnICBTLkwuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz1SMTE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSB3aG9sZS1nZW5vbWUgYXNzZW1ibHkgb2YgRHJvc29waGlsYTsgY2l0YXRpb25fYXV0aG9yPU15ZXJzICBFLlcuOyBjaXRhdGlvbl9hdXRob3I9U3V0dG9uICBHLkcuOyBjaXRhdGlvbl9hdXRob3I9RGVsY2hlciAgQS5MLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODc7ICBjaXRhdGlvbl9wYWdlcz0yMTk2LTIwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBpZGVudGlmaWNhdGlvbiBpbiBub3ZlbCBldWthcnlvdGljIGdlbm9tZXMgYnkgc2VsZi10cmFpbmluZyBhbGdvcml0aG07IGNpdGF0aW9uX2F1dGhvcj1Mb21zYWR6ZSAgQS47IGNpdGF0aW9uX2F1dGhvcj1UZXItSG92aGFubmlzeWFuICBWLjsgY2l0YXRpb25fYXV0aG9yPUNoZXJub2ZmICBZLk8uOyBjaXRhdGlvbl9hdXRob3I9Qm9yb2RvdnNreSAgTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTY0OTQtNTA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CbGFzdDJHTzogYSB1bml2ZXJzYWwgdG9vbCBmb3IgYW5ub3RhdGlvbiwgdmlzdWFsaXphdGlvbiBhbmQgYW5hbHlzaXMgaW4gZnVuY3Rpb25hbCBnZW5vbWljcyByZXNlYXJjaDsgY2l0YXRpb25fYXV0aG9yPUNvbmVzYSAgQS47IGNpdGF0aW9uX2F1dGhvcj1Hb3R6ICBTLjsgY2l0YXRpb25fYXV0aG9yPUdhcmNpYS1Hb21leiAgSi5NLjsgY2l0YXRpb25fYXV0aG9yPVRlcm9sICBKLjsgY2l0YXRpb25fYXV0aG9yPVRhbG9uICBNLjsgY2l0YXRpb25fYXV0aG9yPVJvYmxlcyAgTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MzY3NC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQ09HIGRhdGFiYXNlOiBhbiB1cGRhdGVkIHZlcnNpb24gaW5jbHVkZXMgZXVrYXJ5b3RlczsgY2l0YXRpb25fYXV0aG9yPVRhdHVzb3YgIFIuTC47IGNpdGF0aW9uX2F1dGhvcj1GZWRvcm92YSAgTi5ELjsgY2l0YXRpb25fYXV0aG9yPUphY2tzb24gIEouRC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9NDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlclByb1NjYW46IHByb3RlaW4gZG9tYWlucyBpZGVudGlmaWVyOyBjaXRhdGlvbl9hdXRob3I9UXVldmlsbG9uICBFLjsgY2l0YXRpb25fYXV0aG9yPVNpbHZlbnRvaW5lbiAgVi47IGNpdGF0aW9uX2F1dGhvcj1QaWxsYWkgIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz1XMTE2LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1STkFtbWVyOiBjb25zaXN0ZW50IGFuZCByYXBpZCBhbm5vdGF0aW9uIG9mIHJpYm9zb21hbCBSTkEgZ2VuZXM7IGNpdGF0aW9uX2F1dGhvcj1MYWdlc2VuICBLLjsgY2l0YXRpb25fYXV0aG9yPUhhbGxpbiAgUC47IGNpdGF0aW9uX2F1dGhvcj1Sb2RsYW5kICBFLkEuOyBjaXRhdGlvbl9hdXRob3I9U3RhZXJmZWxkdCAgSC5ILjsgY2l0YXRpb25fYXV0aG9yPVJvZ25lcyAgVC47IGNpdGF0aW9uX2F1dGhvcj1Vc3NlcnkgIEQuVy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPTMxMDAtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9dFJOQXNjYW4tU0U6IGEgcHJvZ3JhbSBmb3IgaW1wcm92ZWQgZGV0ZWN0aW9uIG9mIHRyYW5zZmVyIFJOQSBnZW5lcyBpbiBnZW5vbWljIHNlcXVlbmNlOyBjaXRhdGlvbl9hdXRob3I9TG93ZSAgVC5NLjsgY2l0YXRpb25fYXV0aG9yPUVkZHkgIFMuUi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTk1NS02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9ZGJDQU46IGEgd2ViIHJlc291cmNlIGZvciBhdXRvbWF0ZWQgY2FyYm9oeWRyYXRlLWFjdGl2ZSBlbnp5bWUgYW5ub3RhdGlvbjsgY2l0YXRpb25fYXV0aG9yPVlpbiAgWS47IGNpdGF0aW9uX2F1dGhvcj1NYW8gIFguOyBjaXRhdGlvbl9hdXRob3I9WWFuZyAgSi47IGNpdGF0aW9uX2F1dGhvcj1DaGVuICBYLjsgY2l0YXRpb25fYXV0aG9yPU1hbyAgRi47IGNpdGF0aW9uX2F1dGhvcj1YdSAgWS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPVc0NDUtNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpZ25hbFAgNC4wOiBkaXNjcmltaW5hdGluZyBzaWduYWwgcGVwdGlkZXMgZnJvbSB0cmFuc21lbWJyYW5lIHJlZ2lvbnM7IGNpdGF0aW9uX2F1dGhvcj1QZXRlcnNlbiAgVC5OLjsgY2l0YXRpb25fYXV0aG9yPUJydW5hayAgUy47IGNpdGF0aW9uX2F1dGhvcj12b24gSGVpam5lICBHLjsgY2l0YXRpb25fYXV0aG9yPU5pZWxzZW4gIEguOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTc4NS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW5nIHRyYW5zbWVtYnJhbmUgcHJvdGVpbiB0b3BvbG9neSB3aXRoIGEgaGlkZGVuIE1hcmtvdiBtb2RlbDogYXBwbGljYXRpb24gdG8gY29tcGxldGUgZ2Vub21lczsgY2l0YXRpb25fYXV0aG9yPUtyb2doICBBLjsgY2l0YXRpb25fYXV0aG9yPUxhcnNzb24gIEIuOyBjaXRhdGlvbl9hdXRob3I9dm9uIEhlaWpuZSAgRy47IGNpdGF0aW9uX2F1dGhvcj1Tb25uaGFtbWVyICBFLkwuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTMwNTsgIGNpdGF0aW9uX3BhZ2VzPTU2Ny04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U01VUkY6IEdlbm9taWMgbWFwcGluZyBvZiBmdW5nYWwgc2Vjb25kYXJ5IG1ldGFib2xpdGUgY2x1c3RlcnM7IGNpdGF0aW9uX2F1dGhvcj1LaGFsZGkgIE4uOyBjaXRhdGlvbl9hdXRob3I9U2VpZnVkZGluICBGLlQuOyBjaXRhdGlvbl9hdXRob3I9VHVybmVyICBHLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnVuZ2FsIEdlbmV0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00NzsgIGNpdGF0aW9uX3BhZ2VzPTczNi00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGl2ZXJzZSBsaWZlc3R5bGVzIGFuZCBzdHJhdGVnaWVzIG9mIHBsYW50IHBhdGhvZ2VuZXNpcyBlbmNvZGVkIGluIHRoZSBnZW5vbWVzIG9mIGVpZ2h0ZWVuIERvdGhpZGVvbXljZXRlcyBmdW5naTsgY2l0YXRpb25fYXV0aG9yPU9obSAgUi5BLjsgY2l0YXRpb25fYXV0aG9yPUZlYXUgIE4uOyBjaXRhdGlvbl9hdXRob3I9SGVucmlzc2F0ICBCLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBQYXRob2cuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMzAzNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX2F1dGhvcj1EZWFuICBSLkEuOyBjaXRhdGlvbl9hdXRob3I9QW5uICBMLlAuOyBjaXRhdGlvbl9hdXRob3I9Q2hpdHRhcmFuamFuICBLLjsgY2l0YXRpb25fcHVibGlzaGVyPVNwcmluZ2VyLCBCZXJsaW4sIEhlaWRlbGJlcmc7ICBjaXRhdGlvbl90aXRsZT1HZW5vbWljcyBvZiBwbGFudC1hc3NvY2lhdGVkIGZ1bmdpOiBtb25vY290IHBhdGhvZ2VuczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3BhZ2VzPTQxLTY3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZWluIGFubm90YXRpb24gYW5kIG1vZGVsbGluZyBzZXJ2ZXJzIGF0IFVuaXZlcnNpdHkgQ29sbGVnZSBMb25kb247IGNpdGF0aW9uX2F1dGhvcj1CdWNoYW4gIEQuVy47IGNpdGF0aW9uX2F1dGhvcj1XYXJkICBTLk0uOyBjaXRhdGlvbl9hdXRob3I9TG9ibGV5ICBBLkUuOyBjaXRhdGlvbl9hdXRob3I9TnVnZW50ICBULkMuOyBjaXRhdGlvbl9hdXRob3I9QnJ5c29uICBLLjsgY2l0YXRpb25fYXV0aG9yPUpvbmVzICBELlQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz1XNTYzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNXSVNTLU1PREVMOiBtb2RlbGxpbmcgcHJvdGVpbiB0ZXJ0aWFyeSBhbmQgcXVhdGVybmFyeSBzdHJ1Y3R1cmUgdXNpbmcgZXZvbHV0aW9uYXJ5IGluZm9ybWF0aW9uOyBjaXRhdGlvbl9hdXRob3I9Qmlhc2luaSAgTS47IGNpdGF0aW9uX2F1dGhvcj1CaWVuZXJ0ICBTLjsgY2l0YXRpb25fYXV0aG9yPVdhdGVyaG91c2UgIEEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz1XMjUyLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9ydGhvTUNMOiBpZGVudGlmaWNhdGlvbiBvZiBvcnRob2xvZyBncm91cHMgZm9yIGV1a2FyeW90aWMgZ2Vub21lczsgY2l0YXRpb25fYXV0aG9yPUxpICBMLjsgY2l0YXRpb25fYXV0aG9yPVN0b2Vja2VydCAgQy5KLiw7IGNpdGF0aW9uX2F1dGhvcj1Sb29zICBELlMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTIxNzgtODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsdXN0YWwgVyBhbmQgQ2x1c3RhbCBYIHZlcnNpb24gMi4wOyBjaXRhdGlvbl9hdXRob3I9TGFya2luICBNLkEuOyBjaXRhdGlvbl9hdXRob3I9QmxhY2tzaGllbGRzICBHLjsgY2l0YXRpb25fYXV0aG9yPUJyb3duICBOLlAuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTI5NDctODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuZ2FsIGVmZmVjdG9yIEVjcDYgb3V0Y29tcGV0ZXMgaG9zdCBpbW11bmUgcmVjZXB0b3IgZm9yIGNoaXRpbiBiaW5kaW5nIHRocm91Z2ggaW50cmFjaGFpbiBMeXNNIGRpbWVyaXphdGlvbjsgY2l0YXRpb25fYXV0aG9yPVNhbmNoZXotVmFsbGV0ICBBLjsgY2l0YXRpb25fYXV0aG9yPVNhbGVlbS1CYXRjaGEgIFIuOyBjaXRhdGlvbl9hdXRob3I9S29tYnJpbmsgIEEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FbGlmZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9ZTAwNzkwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U1dJU1MtTU9ERUwgYW5kIHRoZSBTd2lzcy1QZGJWaWV3ZXI6IGFuIGVudmlyb25tZW50IGZvciBjb21wYXJhdGl2ZSBwcm90ZWluIG1vZGVsaW5nOyBjaXRhdGlvbl9hdXRob3I9R3VleCAgTi47IGNpdGF0aW9uX2F1dGhvcj1QZWl0c2NoICBNLkMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FbGVjdHJvcGhvcmVzaXM7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0yNzE0LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZSBhbmFseXNpcyBhbmQgaG9tb2xvZ3kgbW9kZWxpbmcgb2YgZ2x5Y2VyYWxkZWh5ZGUtMy1waG9zcGhhdGUgZGVoeWRyb2dlbmFzZSBvZiBhbiBhbnRhcmN0aWMgcHN5Y2hyb3BoaWxpYyBQc2V1ZG9tb25hcyBzcDsgY2l0YXRpb25fYXV0aG9yPUt1YW4gIEMuUy47IGNpdGF0aW9uX2F1dGhvcj1UaWVuICBDLkYuOyBjaXRhdGlvbl9hdXRob3I9U2VlIFRvbyAgVy5DLjsgY2l0YXRpb25fYXV0aG9yPUZldyAgTC5MLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWZyLiBKLiBNaWNyb2Jpb2wuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTg4MS05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXJ0ZW1pczogc2VxdWVuY2UgdmlzdWFsaXphdGlvbiBhbmQgYW5ub3RhdGlvbjsgY2l0YXRpb25fYXV0aG9yPVJ1dGhlcmZvcmQgIEsuOyBjaXRhdGlvbl9hdXRob3I9UGFya2hpbGwgIEouOyBjaXRhdGlvbl9hdXRob3I9Q3Jvb2sgIEouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTk0NC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIGFuZCBwb3RlbnRpYWwgZXZvbHV0aW9uYXJ5IGltcGFjdCBvZiB0cmFuc3Bvc2FibGUgZWxlbWVudHMgaW4gdGhlIGdlbm9tZSBvZiBDb2NobGlvYm9sdXMgaGV0ZXJvc3Ryb3BodXM7IGNpdGF0aW9uX2F1dGhvcj1TYW50YW5hICBNLkYuOyBjaXRhdGlvbl9hdXRob3I9U2lsdmEgIEouQy47IGNpdGF0aW9uX2F1dGhvcj1NaXp1YnV0aSAgRS5TLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9NTM2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm9uLUxUUiByZXRyb3RyYW5zcG9zb25zIGluIGZ1bmdpOyBjaXRhdGlvbl9hdXRob3I9Tm92aWtvdmEgIE8uOyBjaXRhdGlvbl9hdXRob3I9RmV0ICBWLjsgY2l0YXRpb25fYXV0aG9yPUJsaW5vdiAgQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZ1bmN0LiBJbnRlZ3IuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0yNy00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TFRSIHJldHJvdHJhbnNwb3NvbnMgaW4gZnVuZ2k7IGNpdGF0aW9uX2F1dGhvcj1NdXN6ZXdza2EgIEEuOyBjaXRhdGlvbl9hdXRob3I9SG9mZm1hbi1Tb21tZXIgIE0uOyBjaXRhdGlvbl9hdXRob3I9R3J5bmJlcmcgIE0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTI5NDI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzZW1ibHkgb2YgdGhlIFRjMSBhbmQgbWFyaW5lciB0cmFuc3Bvc2l0aW9uIGluaXRpYXRpb24gY29tcGxleGVzIGRlcGVuZHMgb24gdGhlIG9yaWdpbnMgb2YgdGhlaXIgdHJhbnNwb3Nhc2UgRE5BIGJpbmRpbmcgZG9tYWluczsgY2l0YXRpb25fYXV0aG9yPUJyaWxsZXQgIEIuOyBjaXRhdGlvbl9hdXRob3I9Qmlnb3QgIFkuOyBjaXRhdGlvbl9hdXRob3I9QXVnZS1Hb3VpbGxvdSAgQy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNhOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEzMDsgIGNpdGF0aW9uX3BhZ2VzPTEwNS0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNhdGFseXRpYyBkb21haW4gb2YgYWxsIGV1a2FyeW90aWMgY3V0LWFuZC1wYXN0ZSB0cmFuc3Bvc2FzZSBzdXBlcmZhbWlsaWVzOyBjaXRhdGlvbl9hdXRob3I9WXVhbiAgWS5XLjsgY2l0YXRpb25fYXV0aG9yPVdlc3NsZXIgIFMuUi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTA4OyAgY2l0YXRpb25fcGFnZXM9Nzg4NC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3N0dHJhbnNsYXRpb25hbCBtb2RpZmljYXRpb25zIG9mIHByb3RlaW5zIGluIHRoZSBwYXRob2Jpb2xvZ3kgb2YgbWVkaWNhbGx5IHJlbGV2YW50IGZ1bmdpOyBjaXRhdGlvbl9hdXRob3I9TGVhY2ggIE0uRC47IGNpdGF0aW9uX2F1dGhvcj1Ccm93biAgQS5KLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXVrYXJ5b3QuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz05OC0xMDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11bHRpZnVuY3Rpb24gb2YgYXV0b3BoYWd5LXJlbGF0ZWQgZ2VuZXMgaW4gZmlsYW1lbnRvdXMgZnVuZ2k7IGNpdGF0aW9uX2F1dGhvcj1LaGFuICBJLkEuOyBjaXRhdGlvbl9hdXRob3I9THUgIEouUC47IGNpdGF0aW9uX2F1dGhvcj1MaXUgIFguSC47IGNpdGF0aW9uX2F1dGhvcj1SZWhtYW4gIEEuOyBjaXRhdGlvbl9hdXRob3I9TGluICBGLkMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NaWNyb2Jpb2wuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTY3OyAgY2l0YXRpb25fcGFnZXM9MzM5LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db3JyZWN0aW9uOiBjb21wYXJhdGl2ZSBhbmFseXNpcyBvZiBmdW5nYWwgZ2Vub21lcyByZXZlYWxzIGRpZmZlcmVudCBwbGFudCBjZWxsIHdhbGwgZGVncmFkaW5nIGNhcGFjaXR5IGluIGZ1bmdpOyBjaXRhdGlvbl9hdXRob3I9WmhhbyAgWi47IGNpdGF0aW9uX2F1dGhvcj1MaXUgIEguOyBjaXRhdGlvbl9hdXRob3I9V2FuZyAgQy47IGNpdGF0aW9uX2F1dGhvcj1YdSAgSi5SLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpb2luZm9ybWF0aWMgY2hhcmFjdGVyaXNhdGlvbiBvZiBnZW5lcyBlbmNvZGluZyBjZWxsIHdhbGwgZGVncmFkaW5nIGVuenltZXMgaW4gdGhlIFBoeXRvcGh0aG9yYSBwYXJhc2l0aWNhIGdlbm9tZTsgY2l0YXRpb25fYXV0aG9yPUJsYWNrbWFuICBMLk0uOyBjaXRhdGlvbl9hdXRob3I9Q3VsbGVybmUgIEQuUC47IGNpdGF0aW9uX2F1dGhvcj1IYXJkaGFtICBBLlIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz03ODUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHN5bWJpb250LWluZGVwZW5kZW50IGVuZG8tMSw0LWJldGEteHlsYW5hc2UgZnJvbSB0aGUgcGxhbnQtcGFyYXNpdGljIG5lbWF0b2RlIE1lbG9pZG9neW5lIGluY29nbml0YTsgY2l0YXRpb25fYXV0aG9yPU1pdHJldmEtRGF1dG92YSAgTS47IGNpdGF0aW9uX2F1dGhvcj1Sb3plICBFLjsgY2l0YXRpb25fYXV0aG9yPU92ZXJtYXJzICBILjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBQbGFudCBNaWNyb2IuIEludGVyYWN0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTUyMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgY2xvbmluZyBvZiBmdW5nYWwgeHlsYW5hc2VzOiBhbiBvdmVydmlldzsgY2l0YXRpb25fYXV0aG9yPUFobWVkICBTLjsgY2l0YXRpb25fYXV0aG9yPVJpYXogIFMuOyBjaXRhdGlvbl9hdXRob3I9SmFtaWwgIEEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcHBsLiBNaWNyb2Jpb2wuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTg0OyAgY2l0YXRpb25fcGFnZXM9MTktMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGxvcmluZyBsYWNjYXNlLWxpa2UgbXVsdGljb3BwZXIgb3hpZGFzZSBnZW5lcyBmcm9tIHRoZSBhc2NvbXljZXRlIFRyaWNob2Rlcm1hIHJlZXNlaTogYSBmdW5jdGlvbmFsLCBwaHlsb2dlbmV0aWMgYW5kIGV2b2x1dGlvbmFyeSBzdHVkeTsgY2l0YXRpb25fYXV0aG9yPUxldmFzc2V1ciAgQS47IGNpdGF0aW9uX2F1dGhvcj1TYWxvaGVpbW8gIE0uOyBjaXRhdGlvbl9hdXRob3I9TmF2YXJybyAgRC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTMyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGlnbmluIHBlcm94aWRhc2Ugc3RydWN0dXJlIGFuZCBmdW5jdGlvbjsgY2l0YXRpb25fYXV0aG9yPVBpb250ZWsgIEsuOyBjaXRhdGlvbl9hdXRob3I9U21pdGggIEEuVC47IGNpdGF0aW9uX2F1dGhvcj1CbG9kaWcgIFcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9jaGVtLiBTb2MuIFRyYW5zLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTExMS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MeXNNIGVmZmVjdG9yczogc2VjcmV0ZWQgcHJvdGVpbnMgc3VwcG9ydGluZyBmdW5nYWwgbGlmZTsgY2l0YXRpb25fYXV0aG9yPUtvbWJyaW5rICBBLjsgY2l0YXRpb25fYXV0aG9yPVRob21tYSAgQi5QLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBQYXRob2cuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMzc2OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBub3ZlbCBDbGFkb3Nwb3JpdW0gZnVsdnVtIGx5c2luIG1vdGlmIGVmZmVjdG9yIEVjcDYgaXMgYSB2aXJ1bGVuY2UgZmFjdG9yIHdpdGggb3J0aG9sb2d1ZXMgaW4gb3RoZXIgZnVuZ2FsIHNwZWNpZXM7IGNpdGF0aW9uX2F1dGhvcj1Cb2x0b24gIE0uRC47IGNpdGF0aW9uX2F1dGhvcj12YW4gRXNzZSAgSC5QLjsgY2l0YXRpb25fYXV0aG9yPVZvc3NlbiAgSi5ILjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTY5OyAgY2l0YXRpb25fcGFnZXM9MTE5LTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5nYWwgTHlzTSBlZmZlY3RvcnM6IGV4dGluZ3Vpc2hlcnMgb2YgaG9zdCBpbW11bml0eT87IGNpdGF0aW9uX2F1dGhvcj1kZSBKb25nZSAgUi47IGNpdGF0aW9uX2F1dGhvcj1UaG9tbWEgIEIuUC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MTUxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFwcGxpY2F0aW9uIG9mIG1hdGluZyB0eXBlIGdlbmUgdGVjaG5vbG9neSB0byBwcm9ibGVtcyBpbiBmdW5nYWwgYmlvbG9neTsgY2l0YXRpb25fYXV0aG9yPVR1cmdlb24gIEIuRy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubnUuIFJldi4gUGh5dG9wYXRob2wuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9MTE1LTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5nYWwgc2Vjb25kYXJ5IG1ldGFib2xpc20gLSBmcm9tIGJpb2NoZW1pc3RyeSB0byBnZW5vbWljczsgY2l0YXRpb25fYXV0aG9yPUtlbGxlciAgTi5QLjsgY2l0YXRpb25fYXV0aG9yPVR1cm5lciAgRy47IGNpdGF0aW9uX2F1dGhvcj1CZW5uZXR0ICBKLlcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9OTM3LTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc3RydWN0dXJlIG9mIGRpbWV0aHlsYWxseWwgdHJ5cHRvcGhhbiBzeW50aGFzZSByZXZlYWxzIGEgY29tbW9uIGFyY2hpdGVjdHVyZSBvZiBhcm9tYXRpYyBwcmVueWx0cmFuc2ZlcmFzZXMgaW4gZnVuZ2kgYW5kIGJhY3RlcmlhOyBjaXRhdGlvbl9hdXRob3I9TWV0emdlciAgVS47IGNpdGF0aW9uX2F1dGhvcj1TY2hhbGwgIEMuOyBjaXRhdGlvbl9hdXRob3I9Wm9jaGVyICBHLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDY7ICBjaXRhdGlvbl9wYWdlcz0xNDMwOS0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXJnb3QgYWxrYWxvaWRzLS1iaW9sb2d5IGFuZCBtb2xlY3VsYXIgYmlvbG9neTsgY2l0YXRpb25fYXV0aG9yPVNjaGFyZGwgIEMuTC47IGNpdGF0aW9uX2F1dGhvcj1QYW5hY2Npb25lICBELkcuOyBjaXRhdGlvbl9hdXRob3I9VHVkenluc2tpICBQLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWxrYWxvaWRzIENoZW0uIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTYzOyAgY2l0YXRpb25fcGFnZXM9NDUtODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgYW5kIGdlbm9taWMgZGlzc2VjdGlvbiBvZiB0aGUgQ29jaGxpb2JvbHVzIGhldGVyb3N0cm9waHVzIFRveDEgbG9jdXMgY29udHJvbGxpbmcgYmlvc3ludGhlc2lzIG9mIHRoZSBwb2x5a2V0aWRlIHZpcnVsZW5jZSBmYWN0b3IgVC10b3hpbjsgY2l0YXRpb25fYXV0aG9yPVR1cmdlb24gIEIuRy47IGNpdGF0aW9uX2F1dGhvcj1CYWtlciAgUy5FLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWR2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NTc7ICBjaXRhdGlvbl9wYWdlcz0yMTktNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlvbml6aW5nIHJhZGlhdGlvbiBjaGFuZ2VzIHRoZSBlbGVjdHJvbmljIHByb3BlcnRpZXMgb2YgbWVsYW5pbiBhbmQgZW5oYW5jZXMgdGhlIGdyb3d0aCBvZiBtZWxhbml6ZWQgZnVuZ2k7IGNpdGF0aW9uX2F1dGhvcj1EYWRhY2hvdmEgIEUuOyBjaXRhdGlvbl9hdXRob3I9QnJ5YW4gIFIuQS47IGNpdGF0aW9uX2F1dGhvcj1IdWFuZyAgWC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz1lNDU3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWVsYW5pbiBiaW9zeW50aGVzaXMgaW4gdGhlIG1haXplIHBhdGhvZ2VuIENvY2hsaW9ib2x1cyBoZXRlcm9zdHJvcGh1cyBkZXBlbmRzIG9uIHR3byBtaXRvZ2VuLWFjdGl2YXRlZCBwcm90ZWluIGtpbmFzZXMsIENoazEgYW5kIE1wczEsIGFuZCB0aGUgdHJhbnNjcmlwdGlvbiBmYWN0b3IgQ21yMTsgY2l0YXRpb25fYXV0aG9yPUVsaWFodSAgTi47IGNpdGF0aW9uX2F1dGhvcj1JZ2JhcmlhICBBLjsgY2l0YXRpb25fYXV0aG9yPVJvc2UgIE0uUy47IGNpdGF0aW9uX2F1dGhvcj1Ib3J3aXR6ICBCLkEuOyBjaXRhdGlvbl9hdXRob3I9TGV2ICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXVrYXJ5b3QuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTQyMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob3ZlbCBmdW5nYWwgdHJhbnNjcmlwdGlvbmFsIGFjdGl2YXRvcnMsIENtcjFwIG9mIENvbGxldG90cmljaHVtIGxhZ2VuYXJpdW0gYW5kIHBpZzFwIG9mIE1hZ25hcG9ydGhlIGdyaXNlYSwgY29udGFpbiBDeXMySGlzMiB6aW5jIGZpbmdlciBhbmQgWm4oSUkpMkN5czYgYmludWNsZWFyIGNsdXN0ZXIgRE5BLWJpbmRpbmcgbW90aWZzIGFuZCByZWd1bGF0ZSB0cmFuc2NyaXB0aW9uIG9mIG1lbGFuaW4gYmlvc3ludGhlc2lzIGdlbmVzIGluIGEgZGV2ZWxvcG1lbnRhbGx5IHNwZWNpZmljIG1hbm5lcjsgY2l0YXRpb25fYXV0aG9yPVRzdWppICBHLjsgY2l0YXRpb25fYXV0aG9yPUtlbm1vY2hpICBZLjsgY2l0YXRpb25fYXV0aG9yPVRha2FubyAgWS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPTk0MC01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGluZmx1ZW5jZSBvZiBuYXR1cmFsIHByb2R1Y3RzIHVwb24gZHJ1ZyBkaXNjb3Zlcnk7IGNpdGF0aW9uX2F1dGhvcj1OZXdtYW4gIEQuSi47IGNpdGF0aW9uX2F1dGhvcj1DcmFnZyAgRy5NLjsgY2l0YXRpb25fYXV0aG9yPVNuYWRlciAgSy5NLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBQcm9kLiBSZXAuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MjE1LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DYWZmZWluZSBjb25zdW1wdGlvbjsgY2l0YXRpb25fYXV0aG9yPUdpbGJlcnQgIFIuTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2cuIENsaW4uIEJpb2wuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5ODQ7ICBjaXRhdGlvbl92b2x1bWU9MTU4OyAgY2l0YXRpb25fcGFnZXM9MTg1LTIxMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2F0YWJvbGljIHBhdGh3YXlzIGFuZCBiaW90ZWNobm9sb2dpY2FsIGFwcGxpY2F0aW9ucyBvZiBtaWNyb2JpYWwgY2FmZmVpbmUgZGVncmFkYXRpb247IGNpdGF0aW9uX2F1dGhvcj1EYXNoICBTLlMuOyBjaXRhdGlvbl9hdXRob3I9R3VtbWFkaSAgUy5OLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvdGVjaG5vbC4gTGV0dC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0xOTkzLTIwMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNhdGFib2xpc20gb2YgY2FmZmVpbmUgaW4gcGxhbnRzIGFuZCBtaWNyb29yZ2FuaXNtczsgY2l0YXRpb25fYXV0aG9yPU1henphZmVyYSAgUC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZyb250IEJpb3NjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTEzNDgtNTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmlzb24gb2YgdGhlIGN5dG9jaHJvbWUgUC00NTAgY29udGFpbmluZyBtb25vb3hpZ2VuYXNlcyBvcmlnaW5hdGluZyBmcm9tIHR3byBkaWZmZXJlbnQgeWVhc3RzOyBjaXRhdGlvbl9hdXRob3I9U2F1ZXIgIE0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1EZXYuIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0xOTgyOyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9NDUyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMi8zLzIxOS8xNTkzODIyL2RzdjAwNy5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgQmlwb2xhcmlzIHBhcGVuZG9yZmlpIGhhcyBiZWVuIHJlcG9ydGVkIGFzIGEgZnVuZ2FsIHBsYW50IHBhdGhvZ2VuIHRoYXQgcmFyZWx5IGNhdXNlcyBvcHBvcnR1bmlzdGljIGluZmVjdGlvbiBpbiBodW1hbnMuIFNlY29uZGFyeSBtZXRhYm9saXRlcyBpc29sYXQiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTU5MTM5NCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRLVUlAGtbUNQEDZ4bzm8cvUfeLNA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f3%2f219%2f591394" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f3%2f219%2f591394" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=23905" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/3/1/m_cover.gif?Expires=2147483647&amp;Signature=o~ckdHMyoALozjazxFdJARrSll6HI0qZKNez1rp60DiaVqyzX7KX3gffSJBpNovfc7x0FBbw5M~q-oXkBI0BrEKD0pgvQT5eVNC4LZt8HYW1wCNn7TYsIJ~iamj5~PWBqwfQdRx7Lwp~5y0cfA6QZfzXvHT-c2y8KfWBVWCVg9vuL7VY9lDugsDIpyrHTWTMWH4KR~-txVxPM4XBFIVE2oVLkX6QYMzija1YcuQl4yisR1R~qhzzHGrjVvva6IGLptvPDIFfjhAoXGwaWNy8qGaYVTkt5JCmjQmDV1APqpGYq6f9-3DKz0Py3DzfqJ6xp8-SqVKAnV4V9jMcimV1eA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/3">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="9633617">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9633617">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9633623">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9633623">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9633633">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9633633">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9633675">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9633675">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9633790">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9633790">4. Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9633798">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9633798">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9633805">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9633805">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="9633807">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#9633807">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/3/205/591312">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/3/233/591590">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Dissecting the fungal biology of <em>Bipolaris papendorfii</em>: from phylogenetic to comparative genomic analysis
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chee Sian Kuan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Medical Microbiology, Faculty of Medicine</div>, <div class="insititution">University of Malaya</div>, <div class="addr-line">Kuala Lumpur 50603</div>, <div class="country">Malaysia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chee+Sian+Kuan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kuan C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kuan C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Chee Sian Kuan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Su Mei Yew</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Medical Microbiology, Faculty of Medicine</div>, <div class="insititution">University of Malaya</div>, <div class="addr-line">Kuala Lumpur 50603</div>, <div class="country">Malaysia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Su+Mei+Yew">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yew S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yew S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Su Mei Yew</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yue Fen Toh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Medical Microbiology, Faculty of Medicine</div>, <div class="insititution">University of Malaya</div>, <div class="addr-line">Kuala Lumpur 50603</div>, <div class="country">Malaysia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yue+Fen+Toh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Toh Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Toh Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yue Fen Toh</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chai Ling Chan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Medical Microbiology, Faculty of Medicine</div>, <div class="insititution">University of Malaya</div>, <div class="addr-line">Kuala Lumpur 50603</div>, <div class="country">Malaysia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chai+Ling+Chan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chan C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chan C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Chai Ling Chan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yun Fong Ngeow</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Medical Microbiology, Faculty of Medicine</div>, <div class="insititution">University of Malaya</div>, <div class="addr-line">Kuala Lumpur 50603</div>, <div class="country">Malaysia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yun+Fong+Ngeow">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ngeow Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ngeow Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yun Fong Ngeow</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kok Wei Lee</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Codon Genomics SB, Jalan Bandar Lapan Belas</div>, <div class="addr-line">Selangor Darul Ehsan 47160</div>, <div class="country">Malaysia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kok+Wei+Lee">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lee K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lee K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kok Wei Lee</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shiang Ling Na</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Medical Microbiology, Faculty of Medicine</div>, <div class="insititution">University of Malaya</div>, <div class="addr-line">Kuala Lumpur 50603</div>, <div class="country">Malaysia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shiang+Ling+Na">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Na S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Na S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shiang Ling Na</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wai-Yan Yee</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Codon Genomics SB, Jalan Bandar Lapan Belas</div>, <div class="addr-line">Selangor Darul Ehsan 47160</div>, <div class="country">Malaysia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wai-Yan+Yee">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yee W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yee W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Wai-Yan Yee</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chee-Choong Hoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Codon Genomics SB, Jalan Bandar Lapan Belas</div>, <div class="addr-line">Selangor Darul Ehsan 47160</div>, <div class="country">Malaysia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chee-Choong+Hoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hoh C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hoh C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Chee-Choong Hoh</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kee Peng Ng</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Medical Microbiology, Faculty of Medicine</div>, <div class="insititution">University of Malaya</div>, <div class="addr-line">Kuala Lumpur 50603</div>, <div class="country">Malaysia</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +60 3-79492418. Fax. +60 3-79494881. E-mail: <a href="mailto:kpng@ummc.edu.my" target="_blank">kpng@ummc.edu.my</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kee+Peng+Ng">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ng K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ng K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kee Peng Ng</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Prof. Takashi Ito</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 3, 1 June 2015, Pages 219–232, <a href='https://doi.org/10.1093/dnares/dsv007'>https://doi.org/10.1093/dnares/dsv007</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">27 April 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">17 January 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">28 March 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="591394" href="/dnaresearch/article-pdf/22/3/219/1593822/dsv007.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Chee Sian Kuan, Su Mei Yew, Yue Fen Toh, Chai Ling Chan, Yun Fong Ngeow, Kok Wei Lee, Shiang Ling Na, Wai-Yan Yee, Chee-Choong Hoh, Kee Peng Ng;  Dissecting the fungal biology of <em>Bipolaris papendorfii</em>: from phylogenetic to comparative genomic analysis, <em>DNA Research</em>, Volume 22, Issue 3, 1 June 2015, Pages 219–232, <a href='https://doi.org/10.1093/dnares/dsv007'>https://doi.org/10.1093/dnares/dsv007</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/591394?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/591394?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/591394?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/591394?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/591394?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/591394?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/591394?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Dissecting%20the%20fungal%20biology%20of%20%26lt%3Bem%26gt%3BBipolaris%20papendorfii%26lt%3B%2Fem%26gt%3B%3A%20from%20phylogenetic%20to%20comparative%20genomic%20analysis&amp;publicationDate=2015-04-27&amp;volumeNum=22&amp;issueNum=3&amp;author=Kuan%2C%20Chee%20Sian%3B%20Yew%2C%20Su%20Mei&amp;startPage=219&amp;endPage=232&amp;contentId=10.1093%2Fdnares%2Fdsv007&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f3%2f219%2f591394" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=9633617 id="9633617" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Bipolaris papendorfii</em> has been reported as a fungal plant pathogen that rarely causes opportunistic infection in humans. Secondary metabolites isolated from this fungus possess medicinal and anticancer properties. However, its genetic fundamental and basic biology are largely unknown. In this study, we report the first draft genome sequence of <em>B. papendorfii</em> UM 226 isolated from the skin scraping of a patient. The assembled 33.4 Mb genome encodes 11,015 putative coding DNA sequences, of which, 2.49% are predicted transposable elements. Multilocus phylogenetic and phylogenomic analyses showed <em>B. papendorfii</em> UM 226 clustering with <em>Curvularia</em> species, apart from other plant pathogenic <em>Bipolaris</em> species. Its genomic features suggest that it is a heterothallic fungus with a putative unique gene encoding the LysM-containing protein which might be involved in fungal virulence on host plants, as well as a wide array of enzymes involved in carbohydrate metabolism, degradation of polysaccharides and lignin in the plant cell wall, secondary metabolite biosynthesis (including dimethylallyl tryptophan synthase, non-ribosomal peptide synthetase, polyketide synthase), the terpenoid pathway and the caffeine metabolism. This first genomic characterization of <em>B. papendorfii</em> provides the basis for further studies on its biology, pathogenicity and medicinal potential.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="dematiaceous">dematiaceous</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Bipolaris papendorfii&quot;"><em>Bipolaris papendorfii</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Cochliobolus"><em>Cochliobolus</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Curvularia"><em>Curvularia</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;LysM domain&quot;">LysM domain</a></div>                            <h2 scrollto-destination=9633623 id="9633623" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p><em>Bipolaris</em> is a large genus of dematiaceous hyphomycetes belonging to the order Pleosporales in the class Dothideomycetes.<sup><a reveal-id="DSV007C1" data-open="DSV007C1" class="link link-ref link-reveal xref-bibr">1</a></sup> It is closely related to two other genera, <em>Cochliobolus</em> and <em>Curvularia</em>. The taxonomy of these three genera has caused confusion for many mycologists as their sexual and asexual states have undergone several nomenclatural changes.<sup><a reveal-id="DSV007C2" data-open="DSV007C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Manamgoda <em>et al.</em><sup><a reveal-id="DSV007C3" data-open="DSV007C3" class="link link-ref link-reveal xref-bibr">3</a></sup> showed that <em>Cochliobolus</em> species can be separated broadly into two subgroups, one with <em>Bipolaris</em> species and another with <em>Curvularia</em> species. Although <em>Bipolaris</em> and <em>Curvularia</em> share many morphological similarities including the same teleomorph of <em>Cochliobolus,</em> some species demonstrate intermediate morphology and some <em>Bipolaris</em> species cluster phylogenetically with <em>Curvularia</em> species instead of with other <em>Bipolaris</em> species.<sup><a reveal-id="DSV007C3" data-open="DSV007C3" class="link link-ref link-reveal xref-bibr">3</a></sup></p><p><em>Bipolaris</em> species have been frequently isolated from soil, plant debri, as well as living plants on which they may exist as saprophytes or phytopathogens. The fungal genus <em>Bipolaris</em> has a asexual state named <em>Cochliobolus</em> are notorious for causing devastating disease epidemics of crop plants such as maize, oat, wheat, rice and sorghum,<sup><a reveal-id="DSV007C4  DSV007C6" data-open="DSV007C4  DSV007C6" class="link link-ref link-reveal xref-bibr">4–6</a></sup> leading to problems of starvation, malnutrition and world economic crisis.<sup><a reveal-id="DSV007C4" data-open="DSV007C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSV007C5" data-open="DSV007C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Some plant pathogens such as <em>Bipolaris australiensis</em>, <em>Bipolaris hawaiiensis</em> and <em>Bipolaris spicifera</em> occasionally cause disease in both immunocompetent and immunocompromised humans,<sup><a reveal-id="DSV007C7" data-open="DSV007C7" class="link link-ref link-reveal xref-bibr">7</a></sup> manifesting as allergic or chronic, superficial or deep-seated, localized or disseminated infections.<sup><a reveal-id="DSV007C8" data-open="DSV007C8" class="link link-ref link-reveal xref-bibr">8</a></sup></p><p><em>Bipolaris papendorfii</em> has received little attention, compared with other virulent species that are pathogenic in food crops and humans. It is a rare opportunistic pathogen that has been reported in only three cases of keratitis.<sup><a reveal-id="DSV007C8" data-open="DSV007C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Like other plant pathogenic <em>Bipolaris</em> species, it causes corn leaf spot,<sup><a reveal-id="DSV007C9" data-open="DSV007C9" class="link link-ref link-reveal xref-bibr">9</a></sup> but the underlying pathogenic mechanism is totally unclear. More interestingly, a recent report described two bioactive compounds, hamigerone and radicinol, from <em>B. papendorfii</em> that exhibited anti-proliferative activity in various cancer cells.<sup><a reveal-id="DSV007C10" data-open="DSV007C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Both of these compounds also activated the apoptosis pathway by regulating the expression of BCL-2, caspase 3 and p53.<sup><a reveal-id="DSV007C10" data-open="DSV007C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p><p>The development of high-throughput next-generation sequencing technologies (NGS) has provided massive genomic nucleotide sequence data for the study of microbes. Although there are whole genome sequence data for several <em>Cochliobolus</em> species in international databases,<sup><a reveal-id="DSV007C6" data-open="DSV007C6" class="link link-ref link-reveal xref-bibr">6</a></sup> the whole genome sequence of <em>B. papendorfii</em> has not yet been determined. The genetic fundamental and basic biology of this fungus are largely unknown. In this study, we report the <em>de nov</em>o draft genome sequence of <em>B. papendorfii</em> UM 226 that was isolated from the skin scraping of a patient with dermatomycosis. The draft genome was generated by a hybrid assembly of Illumina and PacBio sequencing technologies to improve genome assembly. To our knowledge, this report is the first detailed description of the <em>B. papendorfii</em> genome. It should provide useful information on the biology, pathogenicity and medicinal potential of this fungus.</p>                            <h2 scrollto-destination=9633633 id="9633633" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=9633636 id="9633636" class="section-title" data-legacy-id=s2a>2.1. Ethics statement</h3>
<p>The isolate used in this study was obtained from an archived fungal collection. All labels on the source sample have been erased with the exception of sample type and clinical diagnosis. Thus, there is no information traceable back to the patient from whom the isolate was obtained. As such, this genomic study is exempt from ethical approval in our teaching hospital. (<a class="link link-uri" href="http://www.ummc.edu.my/view/content.php?ID=VGxSWlBRPT0=">http://www.ummc.edu.my/view/content.php?ID=VGxSWlBRPT0=</a>).</p>                            <h3 scrollto-destination=9633640 id="9633640" class="section-title" data-legacy-id=s2b>2.2. Fungal isolate</h3>
<p>UM 226 was isolated from the skin scraping of a patient with suspected dermatomycosis in the University of Malaya Medical Centre (UMMC), Kuala Lumpur, Malaysia. The isolate was processed according to the laboratory's standard operating procedures (SOP)<sup><a reveal-id="DSV007C11" data-open="DSV007C11" class="link link-ref link-reveal xref-bibr">11</a></sup> with direct wet mount microscopy followed by inoculation on Sabouraud Dextrose Agar (SDA) for incubation at 30°C up to 7 days, with alternate day examination for fungal growth. Macroscopic examination was carried out to observe colonial characteristics, including color, texture and topography. Tease mount and slide culture were performed to identify the arrangement of conidia and conidiophores. The isolate was viewed under the scanning electron microscope (SEM) to examine wall ornamentation and fine surface features of fungal conidia.</p>                            <h3 scrollto-destination=9633643 id="9633643" class="section-title" data-legacy-id=s2c>2.3. DNA sequencing and phylogenetic analysis</h3>
<p>The internal transcribed spacer region (ITS), small subunit of the ribosomal RNA gene (SSU) and large subunit of the ribosomal RNA gene (LSU) were used as targets for molecular identification of the fungal isolate. Total DNA extraction, PCR amplification and sequencing were performed as described previously.<sup><a reveal-id="DSV007C11" data-open="DSV007C11" class="link link-ref link-reveal xref-bibr">11</a></sup> The primers used are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. Annealing temperatures used to amplify ITS, SSU and LSU genes were 58, 52 and 46°C, respectively. The sequenced data were subjected to BLASTn search against the non-redundant (nr) NCBI-nucleotide database for fungal identification. Unique ITS, SSU and LSU sequences from the isolate, together with an additional 145 species of <em>Cochliobolus</em>, <em>Bipolaris</em> and <em>Curvularia</em> and an outgroup strain of <em>Alternaria brassicicola</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>) were subjected to phylogenetic analysis. Multiple sequence alignments of collected ITS, LSU and SSU sequences were generated using M-Coffee.<sup><a reveal-id="DSV007C12" data-open="DSV007C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Individual alignments were concatenated for Bayesian Markov Chain Monte Carlo (MCMC) analysis partitioned by gene. Bayesian tree analyses were performed using MrBayes v3.2.1<sup><a reveal-id="DSV007C13" data-open="DSV007C13" class="link link-ref link-reveal xref-bibr">13</a></sup> with reversible jump MCMC averaging over the entire general time reversible (GTR) rates and gamma-distributed rate heterogeneity for all subsets of partitioned scheme. A total of 5,000,000 generations were run with a sampling frequency of 100, and diagnostics were calculated for every 1,000 generations. The first 12,500 trees were discarded with burn-in setting of 25%. Convergence was assessed with a standard deviation of split frequencies below 0.01, no obvious trend for the plot of the generation versus the log probability of the data, and the potential scale reduction factor (PSRF) close to 1.0 for all parameters.</p>                            <h3 scrollto-destination=9633647 id="9633647" class="section-title" data-legacy-id=s2d>2.4. Genomic DNA extraction</h3>
<p>Genomic DNA was extracted from fresh fungal mycelia in accordance with the protocol of Moslem <em>et al.</em><sup><a reveal-id="DSV007C14" data-open="DSV007C14" class="link link-ref link-reveal xref-bibr">14</a></sup> with slight modification. Briefly, mycelia were scraped off from the agar surface and crushed into fine powder with liquid nitrogen. One gram of crushed fungal powder was suspended in 6 ml of DNA extraction buffer [200 mM Tris–HCl, (pH 8.5), 250 mM NaCl, 25 mM EDTA, 0.6% (w/v) SDS] in a 50-ml centrifuge tube and incubated at 65°C for 10 min. Next, 1.4 ml of 3 M sodium acetate (pH 5.3) was added to the mixture and followed by incubation at 80°C for 20 min. The tube was then centrifuged at 4,000 × <em>g</em> for 5 min and the supernatant was transferred into a new 50-ml centrifuge tube. Genomic DNA was then precipitated with equal volume of 100% (v/v) isopropanol. Genomic DNA was washed from the pellet by adding 500 µl of ice-cold 70% (v/v) ethanol and finally dissolved in nuclease-free distilled water. The purified genomic DNA was treated with RNAase at 37°C for overnight. Then, the mixture was incubated at 65°C for 10 min, followed by centrifugation at 28,433 × <em>g</em> for 5 min. The supernatant was transferred into a new 1.5-ml microcentrifuge tube. The quantity and quality of the extracted genomic DNA were determined using NanoDrop 2000c spectrophotometer (Thermo Fisher Scientific).</p>                            <h3 scrollto-destination=9633650 id="9633650" class="section-title" data-legacy-id=s2e>2.5. Genome sequencing and <em>de novo</em> hybrid assembly</h3>
<p>Illumina library was prepared using TruSeq v3 Reagent Kits (Illumina). UM 226 genomic DNA was sheared into smaller fragments by Covaris S/E210 or Bioruptor. The 500-bp fragments were purified though gel electrophoresis, which then selectively enriched and amplified by PCR. Next, qualified sequencing library were sequenced using Illumina HiSeq 2000 Sequencer (Illumina) with 2 × 90 bp paired-end mode.</p><p>PacBio RS sequencing was performed using the PacBio RS II system (Pacific Biosciences). UM 226 genomic DNA was used for 10-kb library preparation using PacBio DNA Template Prep Kit 2.0 (3–10 kb). SMRTbell templates were then annealed using PacBio DNA/Polymerase Binding Kit P4. Three SMRT cells were used for sequencing with MagBead standard Seq v2 collection protocol using 180-min movies.</p><p>Genome assembly was performed using the hybrid approach using ECtools<sup><a reveal-id="DSV007C15" data-open="DSV007C15" class="link link-ref link-reveal xref-bibr">15</a></sup> with Illumina contigs and PacBio long reads. Illumina short reads were first pre-processed trimming bases with a Phred quality below Qv20 from the 3′ end of the reads, retaining reads ≥ 50 bp and reads with 40% bases having Qv ≤20 were filtered out using FASTX-Toolkit (<a class="link link-uri" href="http://hannonlab.cshl.edu/fastx_toolkit/">http://hannonlab.cshl.edu/fastx_toolkit/</a>). The pre-processed reads were corrected using Quake software<sup><a reveal-id="DSV007C16" data-open="DSV007C16" class="link link-ref link-reveal xref-bibr">16</a></sup> with 16-mer setting. Corrected reads were then assembled into preliminary contigs using Velvet version 1.2.0715 with k-mer setting = 57, scaffolding = no, insert_length = 462 and ins_length_sd = 60.</p><p>ECTools then utilize velvet assembled contigs with length ≥500 bp to correct errors of the PacBio long reads. Error corrected long reads were then assembled using Celera assembler.<sup><a reveal-id="DSV007C17" data-open="DSV007C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Both ECTools and Celebra assembly were performed according to the software instruction with default parameters setting. Celera assembled contigs smaller than 10 kb were discarded, and Quiver was used to polish the remaining contigs. The assembly was imported into SMRT analysis software version 2.1.1 through SMRT portal interface as the reference, and three SMRT cells data generated were aligned to the genome using the RS_Resequencing protocol iteratively. Through the polishing steps, low-quality contigs with low coverage information from Quiver output were discarded from the final assembly.</p>                            <h3 scrollto-destination=9633658 id="9633658" class="section-title" data-legacy-id=s2f>2.6. Gene prediction and annotation</h3>
<p>Protein coding sequences of <em>B. papendorfii</em> UM 226 were predicted using GeneMark-ES version 2.3e.<sup><a reveal-id="DSV007C18" data-open="DSV007C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Annotation of coding sequences for UM 226 was completed using BLAST (Basic Local Alignment Search Tool) searches against the NCBI nr protein and SwissProt databases. Gene Ontology and KEGG metabolic pathways matches were carried out using local BLAST2GO tools.<sup><a reveal-id="DSV007C19" data-open="DSV007C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Protein classifications were performed by EuKaryotic Orthologous Group (KOG).<sup><a reveal-id="DSV007C20" data-open="DSV007C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The total number of predicted proteins involved in post-translational modification, protein turnover, chaperones (KOG annotation for other <em>Cochliobolus</em> species) was obtained from the Joint Genome Institute (JGI) website, <a class="link link-uri" href="http://genome.jgi.doe.gov/">http://genome.jgi.doe.gov/</a>. Protein domain families were matched to Pfam database using InterProScan 5.<sup><a reveal-id="DSV007C21" data-open="DSV007C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Individual rRNA and tRNA sequences were identified using RNAmmer v1.2<sup><a reveal-id="DSV007C22" data-open="DSV007C22" class="link link-ref link-reveal xref-bibr">22</a></sup> and tRNAscan-SE v1.3.1,<sup><a reveal-id="DSV007C23" data-open="DSV007C23" class="link link-ref link-reveal xref-bibr">23</a></sup> respectively. Putative transposable elements were identified using Transposon-PSI (<a class="link link-uri" href="http://transposonpsi.sourceforge.net">http://transposonpsi.sourceforge.net</a>).</p><p>Carbohydrate-Active enzymes (CAZymes) were functionally annotated by submitting the predicted protein models to the databases of automated Carbohydrate-active enzyme ANnotation (dbCAN).<sup><a reveal-id="DSV007C24" data-open="DSV007C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Small secreted proteins were identified by the prediction of cleavage sites and signal peptide/non-signal peptide using SignalP version 4.1<sup><a reveal-id="DSV007C25" data-open="DSV007C25" class="link link-ref link-reveal xref-bibr">25</a></sup> after discarding proteins with transmembrane domains that were determined by TMHMM version 2.0.<sup><a reveal-id="DSV007C26" data-open="DSV007C26" class="link link-ref link-reveal xref-bibr">26</a></sup> However, one transmembrane domain located at in the N-terminal 40 amino acids is allowed, because the domain is responsible to the secretion signal. Genomic mapping of fungal secondary metabolite clusters was performed using web-based SMURF (Secondary Metabolite Unknown Regions Finder) (<a class="link link-uri" href="http://www.jcvi.org/smurf/">www.jcvi.org/smurf/</a>).<sup><a reveal-id="DSV007C27" data-open="DSV007C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Amino acid sequences with e-value threshold ≤1e−5, alignment length over 70% of its own length and over 50% match identity were selected and assigned as the annotation of predicted genes.</p>                            <h3 scrollto-destination=9633661 id="9633661" class="section-title" data-legacy-id=s2g>2.7. Orthologous genes and comparative genomic analysis</h3>
<p>The protein sequences of all current publicly available <em>Cochliobolus</em> genomes (<em>Cochliobolus heterostrophus</em> C5,<sup><a reveal-id="DSV007C6" data-open="DSV007C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSV007C28" data-open="DSV007C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSV007C29" data-open="DSV007C29" class="link link-ref link-reveal xref-bibr">29</a></sup><em>C. heterostrophus</em> C4,<sup><a reveal-id="DSV007C4" data-open="DSV007C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSV007C30" data-open="DSV007C30" class="link link-ref link-reveal xref-bibr">30</a>,<a reveal-id="DSV007C31" data-open="DSV007C31" class="link link-ref link-reveal xref-bibr">31</a></sup><em>C. carbonum</em> 26-R-13,<sup><a reveal-id="DSV007C6" data-open="DSV007C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSV007C29" data-open="DSV007C29" class="link link-ref link-reveal xref-bibr">29</a></sup><em>C. victoriae</em> FI3,<sup><a reveal-id="DSV007C6" data-open="DSV007C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSV007C29" data-open="DSV007C29" class="link link-ref link-reveal xref-bibr">29</a></sup><em>C. miyabeanus</em> ATCC 44560,<sup><a reveal-id="DSV007C6" data-open="DSV007C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSV007C29" data-open="DSV007C29" class="link link-ref link-reveal xref-bibr">29</a></sup><em>C. sativus</em> ND90Pr<sup><a reveal-id="DSV007C4" data-open="DSV007C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSV007C30" data-open="DSV007C30" class="link link-ref link-reveal xref-bibr">30</a>,<a reveal-id="DSV007C31" data-open="DSV007C31" class="link link-ref link-reveal xref-bibr">31</a></sup> and <em>C. lunatus</em> m118<sup><a reveal-id="DSV007C29" data-open="DSV007C29" class="link link-ref link-reveal xref-bibr">29</a></sup>) were used to determine the orthologous genes in <em>B. papendorfii</em> UM 226. The genome sequences of the seven <em>Cochliobolus</em> species were downloaded from JGI. The OrthoMCL version 2.02<sup><a reveal-id="DSV007C32" data-open="DSV007C32" class="link link-ref link-reveal xref-bibr">32</a></sup> was used in the analysis of protein sequences clustering (≥33 amino acids) for <em>B. papendorfii</em> UM 226 and the seven genome references by all-against-all BLASTp searches of all proteins. The reciprocal best hits from distinct genomes were identified as orthologous genes.</p>                            <h3 scrollto-destination=9633663 id="9633663" class="section-title" data-legacy-id=s2h>2.8. Phylogenomic analysis</h3>
<p>A phylogenomic tree was built using all proteome clusters of the seven <em>Cochliobolus</em> species generated from comparative analysis. The tree is rooted with <em>A. brassicicola</em> ATCC 96836 as outgroup. A total of 6,051 single-copy orthologous genes containing one member in each species was subjected to individual sequence alignments using ClustalW version 2.0.<sup><a reveal-id="DSV007C33" data-open="DSV007C33" class="link link-ref link-reveal xref-bibr">33</a></sup> All spurious sequences or poorly aligned regions were discarded with trimAL (with –gt 0.5). The filtered multiple alignments were then concatenated into a superalignment with 2,991,675 characters. Bayesian phylogenetic analysis was run using MrBayes v3.2.1 with mixed amino acid model, gamma-distributed rate variation across sites and a proportion of invariable sites. The MCMC was run using a sampling frequency of 100 for 100,000 generations with a burn-in setting of 25%.</p>                            <h3 scrollto-destination=9633667 id="9633667" class="section-title" data-legacy-id=s2i>2.9. Prediction of secondary structure and homology modelling</h3>
<p>PSIPRED v3.3<sup><a reveal-id="DSV007C30" data-open="DSV007C30" class="link link-ref link-reveal xref-bibr">30</a></sup> was used to predict the secondary structure of the LysM domains. Three-dimensional structural models of the LysM domains were generated using SWISS-MODEL server.<sup><a reveal-id="DSV007C31" data-open="DSV007C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Amino acid sequence alignment between UM226_683 domains and Ecp6 was performed using ClustalW version 2.0. The template used in this modelling was selected from template identification function in the SWISS-MODEL program. In this study, a <em>Cladosporium fulvum</em> LysM effector Ecp6 (Protein Data Bank entry 4b8v)<sup><a reveal-id="DSV007C34" data-open="DSV007C34" class="link link-ref link-reveal xref-bibr">34</a></sup> was selected as the template. The predicted models were stored as a PDB output file, and the structures were visualized by Swiss-PdbViewer 4.0.1.<sup><a reveal-id="DSV007C35" data-open="DSV007C35" class="link link-ref link-reveal xref-bibr">35</a></sup> The quality of the protein model generated was validated by using the PROCHECK program as described previously.<sup><a reveal-id="DSV007C36" data-open="DSV007C36" class="link link-ref link-reveal xref-bibr">36</a></sup></p>                            <h3 scrollto-destination=9633670 id="9633670" class="section-title" data-legacy-id=s2j>2.10. Mating type locus identification and phylogenetic analysis</h3>
<p><em>MAT1-2</em> mating type regions immediately 5 kb upstream and downstream were retrieved from the sequenced genome using Artemis v12.0 sequence viewer.<sup><a reveal-id="DSV007C37" data-open="DSV007C37" class="link link-ref link-reveal xref-bibr">37</a></sup> Bayesian tree analysis was carried out by using a procedure similar to ribosomal molecular identification with 500,000 generations run. A phylogenetic tree was constructed using <em>MAT1-2</em> nucleotide sequences from <em>B. papendorfii</em> UM 226, together with additional nine species of <em>Cochliobolus</em> and <em>Bipolaris</em>. The tree is rooted with <em>Alternaria alternata</em> as outgroup.</p>                            <h2 scrollto-destination=9633675 id="9633675" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=9633678 id="9633678" class="section-title" data-legacy-id=s3a>3.1. Morphological and molecular identification</h3>
<p>The <em>B. papendorfii</em> UM 226 colonies on SDA were fast growing, velvety and compact with abundant short aerial mycelium (Fig. <a reveal-id="DSV007F1" data-open="DSV007F1" class="link link-reveal link-table xref-fig">1</a>a). The surface of the colonies was initially grayish white to olivaceous green and becomes black with a raised grayish periphery after being cultured for 7 days (Fig. <a reveal-id="DSV007F1" data-open="DSV007F1" class="link link-reveal link-table xref-fig">1</a>a). It had a black-green coloration on the reverse side (Fig. <a reveal-id="DSV007F1" data-open="DSV007F1" class="link link-reveal link-table xref-fig">1</a>b). The diameter of the colony was 60 mm after 7 days of incubation at 30°C. Microscopic morphology demonstrated simple or flexuose branched, pale to medium brown conidiophores (up to 200 µm long, 4–9 µm wide) with zig-zag rachis bearing conidium in a sympodial pattern (Fig. <a reveal-id="DSV007F1" data-open="DSV007F1" class="link link-reveal link-table xref-fig">1</a>c). Conidiogenous nodes were verruculose, distinct, swollen and up to 5 µm wide (Fig. <a reveal-id="DSV007F1" data-open="DSV007F1" class="link link-reveal link-table xref-fig">1</a>d). The conidia (24–40 × 10–20 µm) were curved, kidney shaped or obpyriform (Fig. <a reveal-id="DSV007F1" data-open="DSV007F1" class="link link-reveal link-table xref-fig">1</a>c). The conidia had three pseudoseptates with the widest at the second cell from the base. SEM observation demonstrated typical, roughed, zig-zag conidiophores with several conidia (Fig. <a reveal-id="DSV007F1" data-open="DSV007F1" class="link link-reveal link-table xref-fig">1</a>e). Unlike other <em>Bipolaris</em> spp. that have typical smooth walled conidia,<sup><a reveal-id="DSV007C8" data-open="DSV007C8" class="link link-ref link-reveal xref-bibr">8</a></sup> the SEM microphotograph of the UM 226 conidia showed that they were verrucose walled with thickenings at places of pseudosepta (Fig. <a reveal-id="DSV007F1" data-open="DSV007F1" class="link link-reveal link-table xref-fig">1</a>e). On the basis of these characteristics, the UM 226 clinical isolate was found to be morphologically related to <em>B. papendorfii</em>.<sup><a reveal-id="DSV007C8" data-open="DSV007C8" class="link link-ref link-reveal xref-bibr">8</a></sup></p>                        <a id="9633684" scrollto-destination="9633684"></a>
<div id="DSV007F1" data-id="DSV007F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/m_dsv00701.jpeg?Expires=2147483647&Signature=jfn4jErTglMPWzk2IUfOPhXIdpyMiOGlvsLvDlajQAPVlFA8kdbDCOIH~0tvX2GwrupJygnpygVPNJHfMBh~OuxRczwATceJ7RdxuytkthZJXlQmTrBn9zfHgFq4xgjPPOJ8hh-pkHy5NneD4LJv9v3nc9IShI69q9Hd9VNddbKcBCPVRLZ8HxioTkZ0Za0JLb2HK1wHBEa4dWAlatA7RIuKsvRkJXL-xcFeURnuJzsgQaYc~M8rau2jo7elLolU4BEpEoGzkb3Dt6h1ZyIHoqs-pRivY83OQrNccEG0aPY0nwPSJCCspmHhY2sVuWzJSLRZO5FY2kT-dXezl1O5yA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Colonial characteristic and microscopic morphology of Bipolaris papendorfii UM 226. The surface (A) and reverse (B) colony morphology of B. papendorfii UM 226 after being cultured for 7 days. Light micrograph showing (C) typical zig-zag conidiophore with several conidia and (D) conidia with three pseudoseptates (×400 magnification, bars 20 µm). Scanning electron micrograph showing (E) zig-zag conidiophores with verruculose walled conidia (×3,065 magnification, bars 10 µm). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9633684/dsv00701.jpeg" path-from-xml="dsv00701.jpeg" target="_blank">View large</a><a section="9633684" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/dsv00701.jpeg?Expires=2147483647&Signature=pZmq3z5zuP6KWff~GHU3DgwPhupVXJ2kvuETuCf4GEUhTg4YORw37KkwBywUGieLStp8WM5qWQS4XWlf9ixnW9-bF8SOtHH8sWEsfI0NXfNwwi7si89eOmNZAtMm~OWJG8azPZHhXniqqQ2qXv30cMoBkie0FRADfDmOlSeiOsAe8O7OYL5LdIC2SuOvtrJQAZPjqa8ARJN9jvPiUr54pTIIp9-2AbZReA5O8oWJNLQgqYqJTrA8rYojgLB2C1OLVmKPHTm-l5HBoMvfHr01n2q8AKDVnxRDTdT6PD3VfxQoD6i7zvkxPAwJ5R9oglMNbV0jjCFcNn0MNCbljyJwfQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9633684&ar=591394&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Colonial characteristic and microscopic morphology of <em>Bipolaris papendorfii</em> UM 226. The surface (A) and reverse (B) colony morphology of <em>B. papendorfii</em> UM 226 after being cultured for 7 days. Light micrograph showing (C) typical zig-zag conidiophore with several conidia and (D) conidia with three pseudoseptates (×400 magnification, bars 20 µm). Scanning electron micrograph showing (E) zig-zag conidiophores with verruculose walled conidia (×3,065 magnification, bars 10 µm). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV007F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/m_dsv00701.jpeg?Expires=2147483647&Signature=jfn4jErTglMPWzk2IUfOPhXIdpyMiOGlvsLvDlajQAPVlFA8kdbDCOIH~0tvX2GwrupJygnpygVPNJHfMBh~OuxRczwATceJ7RdxuytkthZJXlQmTrBn9zfHgFq4xgjPPOJ8hh-pkHy5NneD4LJv9v3nc9IShI69q9Hd9VNddbKcBCPVRLZ8HxioTkZ0Za0JLb2HK1wHBEa4dWAlatA7RIuKsvRkJXL-xcFeURnuJzsgQaYc~M8rau2jo7elLolU4BEpEoGzkb3Dt6h1ZyIHoqs-pRivY83OQrNccEG0aPY0nwPSJCCspmHhY2sVuWzJSLRZO5FY2kT-dXezl1O5yA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Colonial characteristic and microscopic morphology of Bipolaris papendorfii UM 226. The surface (A) and reverse (B) colony morphology of B. papendorfii UM 226 after being cultured for 7 days. Light micrograph showing (C) typical zig-zag conidiophore with several conidia and (D) conidia with three pseudoseptates (×400 magnification, bars 20 µm). Scanning electron micrograph showing (E) zig-zag conidiophores with verruculose walled conidia (×3,065 magnification, bars 10 µm). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9633684/dsv00701.jpeg" path-from-xml="dsv00701.jpeg" target="_blank">View large</a><a section="9633684" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/dsv00701.jpeg?Expires=2147483647&Signature=pZmq3z5zuP6KWff~GHU3DgwPhupVXJ2kvuETuCf4GEUhTg4YORw37KkwBywUGieLStp8WM5qWQS4XWlf9ixnW9-bF8SOtHH8sWEsfI0NXfNwwi7si89eOmNZAtMm~OWJG8azPZHhXniqqQ2qXv30cMoBkie0FRADfDmOlSeiOsAe8O7OYL5LdIC2SuOvtrJQAZPjqa8ARJN9jvPiUr54pTIIp9-2AbZReA5O8oWJNLQgqYqJTrA8rYojgLB2C1OLVmKPHTm-l5HBoMvfHr01n2q8AKDVnxRDTdT6PD3VfxQoD6i7zvkxPAwJ5R9oglMNbV0jjCFcNn0MNCbljyJwfQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9633684&ar=591394&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Colonial characteristic and microscopic morphology of <em>Bipolaris papendorfii</em> UM 226. The surface (A) and reverse (B) colony morphology of <em>B. papendorfii</em> UM 226 after being cultured for 7 days. Light micrograph showing (C) typical zig-zag conidiophore with several conidia and (D) conidia with three pseudoseptates (×400 magnification, bars 20 µm). Scanning electron micrograph showing (E) zig-zag conidiophores with verruculose walled conidia (×3,065 magnification, bars 10 µm). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>The preliminary morphological identification of the UM 226 isolate was confirmed by PCR amplification of the ITS, SSU and LSU gene regions, which yielded specific amplicons of ∼540 (ITS), 1,200 (SSU) and 900 bp (LSU), respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). The sequenced data were used to construct a phylogram using combined gene analysis with the additional 145 ex-type strains of <em>Cochliobolus</em>, <em>Bipolaris</em> and <em>Curvularia</em> spp. (Fig. <a reveal-id="DSV007F2" data-open="DSV007F2" class="link link-reveal link-table xref-fig">2</a>). As previously noted,<sup><a reveal-id="DSV007C2" data-open="DSV007C2" class="link link-ref link-reveal xref-bibr">2</a></sup> the phylogenetic tree analysis separated the <em>Curvularia</em>, <em>Bipolaris</em> and <em>Cochliobolus</em> species into two major, well-supported groups: Groups 1 and 2 (Fig. <a reveal-id="DSV007F2" data-open="DSV007F2" class="link link-reveal link-table xref-fig">2</a>). Phylogenetic analysis of the combined dataset of the three loci revealed that 59 species were clustered in Group 1 and 87 species were clustered in Group 2. Group 1 consisted of <em>Bipolaris</em> and <em>Cochliobolus</em> spp. while Group 2 comprised species of <em>Curvularia</em>, <em>Bipolaris</em> and <em>Cochliobolus</em>. All the plant pathogenic and economically important fungi are identified in genus <em>Bipolaris</em>, which were grouped into Group 1.<sup><a reveal-id="DSV007C2" data-open="DSV007C2" class="link link-ref link-reveal xref-bibr">2</a></sup> The <em>Bipolaris</em> species that clustered in Group 2 (Fig. <a reveal-id="DSV007F2" data-open="DSV007F2" class="link link-reveal link-table xref-fig">2</a>). Multilocus phylogenetic analysis revealed that UM 226 was clustered together with <em>B. papendorfii</em> strain CMON22 in Group 2 (Fig. <a reveal-id="DSV007F2" data-open="DSV007F2" class="link link-reveal link-table xref-fig">2</a>). Based on the morphological examination and multigene phylogeny, the isolate was identified as <em>B. papendorfii</em>. </p>                        <a id="9633690" scrollto-destination="9633690"></a>
<div id="DSV007F2" data-id="DSV007F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/m_dsv00702.jpeg?Expires=2147483647&Signature=UiOIENcngSbKrtURmp7~363kbnP1v99TFqvofX4bpJh~vZThpPDY9i9nQv50-psn8HaphY264lDypqZN7UAzd2~Jik2MaArgQw3c-qEYLI1gIN-9wtQmp2rMccdPUMVkta5uH2QcKdUFiWJ7tUbjk5JqIK5oZhzHi0tF1T~8XLcnncUd2yaF0Rv6-jSwnk8X0zy0vuGaSdj4JFbqeKy0Rq8QVCaiK~jqyTvzk0lBemZBYS82V0PUmAUrRF6gM9cSaJFP1CjO~8cvNBRxGPNi~5Kq3VqlDauHiA2wcRaWUhfiUZGKrReT99UxslE-tAF656uKGBNTzRE0cWwh25g4lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Bayesian phylogram generated based on combined genes of ITS, SSU and LSU sequenced data. The tree is rooted with Alternaria brassicicola as outgroup. Numbers on the nodes indicate Bayesian posterior probability based on 100 sampling frequency for a total of 150,000 generations. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9633690/dsv00702.jpeg" path-from-xml="dsv00702.jpeg" target="_blank">View large</a><a section="9633690" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/dsv00702.jpeg?Expires=2147483647&Signature=o4HDY03zMZNxBuYkg-enHH8dd72j06x1pWXFYPJUb9EkfSvzZ9sLkOYOShL7SaU6fxQpz3BhW10nM~eHJR60wHyvP19zB2hyLBLjhG5B9axTzB05eTp7v2azjE6dc02X2lLnh1ku34pZ8byw6iW0kYFrOi3YAyZArIPU8Uu7n56E8syEMUcx-g7bgcxM7WKSssUN4bHwa4ZXAmV5VOy4TQDw6LV87fXzyE-AnCzfI1vEqbRSbdaUtNmUaOxkUj8InUu1nzzXY6ocC1mc4CHt-By8nnvSZTY5edneE~Oj9~dBMjgVqm16bKRDarbaUq4zXA1Li4as508qpufqpgLmUw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9633690&ar=591394&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Bayesian phylogram generated based on combined genes of ITS, SSU and LSU sequenced data. The tree is rooted with <em>Alternaria brassicicola</em> as outgroup. Numbers on the nodes indicate Bayesian posterior probability based on 100 sampling frequency for a total of 150,000 generations. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV007F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/m_dsv00702.jpeg?Expires=2147483647&Signature=UiOIENcngSbKrtURmp7~363kbnP1v99TFqvofX4bpJh~vZThpPDY9i9nQv50-psn8HaphY264lDypqZN7UAzd2~Jik2MaArgQw3c-qEYLI1gIN-9wtQmp2rMccdPUMVkta5uH2QcKdUFiWJ7tUbjk5JqIK5oZhzHi0tF1T~8XLcnncUd2yaF0Rv6-jSwnk8X0zy0vuGaSdj4JFbqeKy0Rq8QVCaiK~jqyTvzk0lBemZBYS82V0PUmAUrRF6gM9cSaJFP1CjO~8cvNBRxGPNi~5Kq3VqlDauHiA2wcRaWUhfiUZGKrReT99UxslE-tAF656uKGBNTzRE0cWwh25g4lg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Bayesian phylogram generated based on combined genes of ITS, SSU and LSU sequenced data. The tree is rooted with Alternaria brassicicola as outgroup. Numbers on the nodes indicate Bayesian posterior probability based on 100 sampling frequency for a total of 150,000 generations. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9633690/dsv00702.jpeg" path-from-xml="dsv00702.jpeg" target="_blank">View large</a><a section="9633690" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/dsv00702.jpeg?Expires=2147483647&Signature=o4HDY03zMZNxBuYkg-enHH8dd72j06x1pWXFYPJUb9EkfSvzZ9sLkOYOShL7SaU6fxQpz3BhW10nM~eHJR60wHyvP19zB2hyLBLjhG5B9axTzB05eTp7v2azjE6dc02X2lLnh1ku34pZ8byw6iW0kYFrOi3YAyZArIPU8Uu7n56E8syEMUcx-g7bgcxM7WKSssUN4bHwa4ZXAmV5VOy4TQDw6LV87fXzyE-AnCzfI1vEqbRSbdaUtNmUaOxkUj8InUu1nzzXY6ocC1mc4CHt-By8nnvSZTY5edneE~Oj9~dBMjgVqm16bKRDarbaUq4zXA1Li4as508qpufqpgLmUw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9633690&ar=591394&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Bayesian phylogram generated based on combined genes of ITS, SSU and LSU sequenced data. The tree is rooted with <em>Alternaria brassicicola</em> as outgroup. Numbers on the nodes indicate Bayesian posterior probability based on 100 sampling frequency for a total of 150,000 generations. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=9633695 id="9633695" class="section-title" data-legacy-id=s3b>3.2. Genome features of <em>B. papendorfii</em> UM 226</h3>
<p>In this study, Illumina HiSeq 2000 and PacBio sequencing technologies were used to produce hybrid genome assembly for <em>B. papendorfii</em> UM 226. The total assembly size of <em>B. papendorfii</em> UM 226 genome is 33.4 Mb. The sequencing reads were assembled into 374 contigs with an N50 contig size of 146,099 kb. Hybrid assembly of <em>B. papendorfii</em> UM 226 showed a lower number of contigs and a higher N50 value (Table <a reveal-id="DSV007TB1" data-open="DSV007TB1" class="link link-reveal link-table xref-fig">1</a>). Summary of the sequence statistics is presented in Table <a reveal-id="DSV007TB1" data-open="DSV007TB1" class="link link-reveal link-table xref-fig">1</a>. The genome had an average G + C content of 50.65%. </p>                        <a id="9633700" scrollto-destination="9633700"></a>
<div content-id="DSV007TB1" class="table-modal"><div class="table-wrap"><div id="DSV007TB1" class="table-wrap-title" data-id="DSV007TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Genome statistics of <em>Bipolaris papendorfii</em> UM 226</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Details </th><th>Illumina HiSeq 2000 </th><th>PacBio RS II system </th><th>Hybrid assembly of Illumina and PacBio </th></tr></thead><tbody><tr><td>Sequencing depth </td><td>∼87× </td><td>∼34× </td><td>— </td></tr><tr><td>Read type </td><td>Small insert </td><td>Single end </td><td>— </td></tr><tr><td>Library size (bp) </td><td>500 </td><td>20,000 </td><td>— </td></tr><tr><td>Read length (bp) </td><td>90 </td><td>2,742 </td><td>— </td></tr><tr><td>Total number of reads </td><td>33,655,556 </td><td>405,257 </td><td>— </td></tr><tr><td>Total number of bases </td><td>3,029,000,040 </td><td>1,111,215,948 </td><td>— </td></tr><tr><td>Assembly size (Mb) </td><td>33,620,574 </td><td>— </td><td>33,379,470 </td></tr><tr><td>Contigs (≥500 bp) </td><td>730 </td><td>— </td><td>374 </td></tr><tr><td>Contigs (N50) (kb) </td><td>127,284 </td><td>— </td><td>146,099 </td></tr><tr><td>Contigs G + C (%) </td><td>50.55 </td><td>— </td><td>50.65 </td></tr></tbody></table></div><div content-id="DSV007TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Details </th><th>Illumina HiSeq 2000 </th><th>PacBio RS II system </th><th>Hybrid assembly of Illumina and PacBio </th></tr></thead><tbody><tr><td>Sequencing depth </td><td>∼87× </td><td>∼34× </td><td>— </td></tr><tr><td>Read type </td><td>Small insert </td><td>Single end </td><td>— </td></tr><tr><td>Library size (bp) </td><td>500 </td><td>20,000 </td><td>— </td></tr><tr><td>Read length (bp) </td><td>90 </td><td>2,742 </td><td>— </td></tr><tr><td>Total number of reads </td><td>33,655,556 </td><td>405,257 </td><td>— </td></tr><tr><td>Total number of bases </td><td>3,029,000,040 </td><td>1,111,215,948 </td><td>— </td></tr><tr><td>Assembly size (Mb) </td><td>33,620,574 </td><td>— </td><td>33,379,470 </td></tr><tr><td>Contigs (≥500 bp) </td><td>730 </td><td>— </td><td>374 </td></tr><tr><td>Contigs (N50) (kb) </td><td>127,284 </td><td>— </td><td>146,099 </td></tr><tr><td>Contigs G + C (%) </td><td>50.55 </td><td>— </td><td>50.65 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/9633700" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV007TB1" class="table-wrap-title" data-id="DSV007TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Genome statistics of <em>Bipolaris papendorfii</em> UM 226</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Details </th><th>Illumina HiSeq 2000 </th><th>PacBio RS II system </th><th>Hybrid assembly of Illumina and PacBio </th></tr></thead><tbody><tr><td>Sequencing depth </td><td>∼87× </td><td>∼34× </td><td>— </td></tr><tr><td>Read type </td><td>Small insert </td><td>Single end </td><td>— </td></tr><tr><td>Library size (bp) </td><td>500 </td><td>20,000 </td><td>— </td></tr><tr><td>Read length (bp) </td><td>90 </td><td>2,742 </td><td>— </td></tr><tr><td>Total number of reads </td><td>33,655,556 </td><td>405,257 </td><td>— </td></tr><tr><td>Total number of bases </td><td>3,029,000,040 </td><td>1,111,215,948 </td><td>— </td></tr><tr><td>Assembly size (Mb) </td><td>33,620,574 </td><td>— </td><td>33,379,470 </td></tr><tr><td>Contigs (≥500 bp) </td><td>730 </td><td>— </td><td>374 </td></tr><tr><td>Contigs (N50) (kb) </td><td>127,284 </td><td>— </td><td>146,099 </td></tr><tr><td>Contigs G + C (%) </td><td>50.55 </td><td>— </td><td>50.65 </td></tr></tbody></table></div><div content-id="DSV007TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Details </th><th>Illumina HiSeq 2000 </th><th>PacBio RS II system </th><th>Hybrid assembly of Illumina and PacBio </th></tr></thead><tbody><tr><td>Sequencing depth </td><td>∼87× </td><td>∼34× </td><td>— </td></tr><tr><td>Read type </td><td>Small insert </td><td>Single end </td><td>— </td></tr><tr><td>Library size (bp) </td><td>500 </td><td>20,000 </td><td>— </td></tr><tr><td>Read length (bp) </td><td>90 </td><td>2,742 </td><td>— </td></tr><tr><td>Total number of reads </td><td>33,655,556 </td><td>405,257 </td><td>— </td></tr><tr><td>Total number of bases </td><td>3,029,000,040 </td><td>1,111,215,948 </td><td>— </td></tr><tr><td>Assembly size (Mb) </td><td>33,620,574 </td><td>— </td><td>33,379,470 </td></tr><tr><td>Contigs (≥500 bp) </td><td>730 </td><td>— </td><td>374 </td></tr><tr><td>Contigs (N50) (kb) </td><td>127,284 </td><td>— </td><td>146,099 </td></tr><tr><td>Contigs G + C (%) </td><td>50.55 </td><td>— </td><td>50.65 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/9633700" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=9633702 id="9633702" class="section-title" data-legacy-id=s3c>3.3. Transposable elements</h3>
<p>Transposable elements (TEs), also known as jumping genes, were important in the variability, gene structure and evolution of the genome. Recently, Santana <em>et al.</em><sup><a reveal-id="DSV007C38" data-open="DSV007C38" class="link link-ref link-reveal xref-bibr">38</a></sup> demonstrated that <em>C. heterostrophus</em> race O genome (∼36 Mb) harbours 5.9% of TEs. Here, our bioinformatics prediction revealed that 2.49% (831,552 bp) of the assembled genome of <em>B. papendorfii</em> UM 226 consisted of TEs (Table <a reveal-id="DSV007TB2" data-open="DSV007TB2" class="link link-reveal link-table xref-fig">2</a>). The transposable elements of Class I, retrotransposons comprised 2.31% of the genome, whereas Class II transposable elements, DNA transposons comprised only 0.19% of the genome. LINE (Long Interspersed Nuclear Element)- and LTR (Long Terminal Repeat)-type retrotransposons are mostly presented in the fungal genomes.<sup><a reveal-id="DSV007C39" data-open="DSV007C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DSV007C40" data-open="DSV007C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Gypsy and Copia elements are the LTR superfamilies that widely distributed in these genomes. As previously noted,<sup><a reveal-id="DSV007C38" data-open="DSV007C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Gypsy elements were identified to be located upstream of the <em>PKS1</em>, <em>OXI1</em> and <em>LAM1</em> genes, downstream of <em>PKS2</em> gene as well as between the <em>RED2</em> and <em>RED3</em> genes in the <em>C. heterostrophus</em> race O genome; Copia element was located downstream of the <em>TOX9</em> gene in the genome. In this study, our analysis showed that 0.24, 0.67 and 0.29% of the sequenced <em>B. papendorfii</em> UM 226 genome, estimated 33.4 Mb, consists of LINE, gypsy and TY1_Copia, respectively. However, the locations of these TEs in the genome remain unknown at this stage of knowledge. The most predominant TE families, both in terms of number of copies and element size, are DDE_1 transposases (Table <a reveal-id="DSV007TB2" data-open="DSV007TB2" class="link link-reveal link-table xref-fig">2</a>). The DDE_1 transposases contain a functional domain to mediate DNA-binding, protein–protein interaction and ‘cut-and-paste’ activities.<sup><a reveal-id="DSV007C41" data-open="DSV007C41" class="link link-ref link-reveal xref-bibr">41</a></sup> The DDE domain has been identified in all eukaryotic cut-and-paste TEs, suggesting their common evolutionary origin.<sup><a reveal-id="DSV007C42" data-open="DSV007C42" class="link link-ref link-reveal xref-bibr">42</a></sup></p>                        <a id="9633708" scrollto-destination="9633708"></a>
<div content-id="DSV007TB2" class="table-modal"><div class="table-wrap"><div id="DSV007TB2" class="table-wrap-title" data-id="DSV007TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Putative transposable elements in the genome sequence of <em>Bipolaris papendorfii</em> UM 226</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Class </th><th>Family name </th><th>Total number </th><th>Total bases </th><th>Percentage of assembled genome </th></tr></thead><tbody><tr><td rowspan="5">I </td><td>DDE_1 </td><td>422 </td><td>369,615 </td><td>1.11 </td></tr><tr><td>gypsy </td><td>279 </td><td>223,716 </td><td>0.67 </td></tr><tr><td>LINE </td><td>54 </td><td>78,738 </td><td>0.24 </td></tr><tr><td>ltr_Roo </td><td>2 </td><td>348 </td><td>0.00001 </td></tr><tr><td>TY1_Copia </td><td>87 </td><td>95,805 </td><td>0.29 </td></tr><tr><td rowspan="7">II </td><td>helitronORF </td><td>4 </td><td>2,673 </td><td>0.01 </td></tr><tr><td>hAT </td><td>21 </td><td>15,879 </td><td>0.05 </td></tr><tr><td>mariner </td><td>55 </td><td>36,201 </td><td>0.11 </td></tr><tr><td>mariner_ant1 </td><td>6 </td><td>2,793 </td><td>0.01 </td></tr><tr><td>MuDR_A_B </td><td>19 </td><td>4,701 </td><td>0.01 </td></tr><tr><td>cacta </td><td>10 </td><td>1,083 </td><td>0.00003 </td></tr><tr><td>Total </td><td> </td><td>831,552 </td><td>2.49 </td></tr></tbody></table></div><div content-id="DSV007TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Class </th><th>Family name </th><th>Total number </th><th>Total bases </th><th>Percentage of assembled genome </th></tr></thead><tbody><tr><td rowspan="5">I </td><td>DDE_1 </td><td>422 </td><td>369,615 </td><td>1.11 </td></tr><tr><td>gypsy </td><td>279 </td><td>223,716 </td><td>0.67 </td></tr><tr><td>LINE </td><td>54 </td><td>78,738 </td><td>0.24 </td></tr><tr><td>ltr_Roo </td><td>2 </td><td>348 </td><td>0.00001 </td></tr><tr><td>TY1_Copia </td><td>87 </td><td>95,805 </td><td>0.29 </td></tr><tr><td rowspan="7">II </td><td>helitronORF </td><td>4 </td><td>2,673 </td><td>0.01 </td></tr><tr><td>hAT </td><td>21 </td><td>15,879 </td><td>0.05 </td></tr><tr><td>mariner </td><td>55 </td><td>36,201 </td><td>0.11 </td></tr><tr><td>mariner_ant1 </td><td>6 </td><td>2,793 </td><td>0.01 </td></tr><tr><td>MuDR_A_B </td><td>19 </td><td>4,701 </td><td>0.01 </td></tr><tr><td>cacta </td><td>10 </td><td>1,083 </td><td>0.00003 </td></tr><tr><td>Total </td><td> </td><td>831,552 </td><td>2.49 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/9633708" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV007TB2" class="table-wrap-title" data-id="DSV007TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Putative transposable elements in the genome sequence of <em>Bipolaris papendorfii</em> UM 226</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Class </th><th>Family name </th><th>Total number </th><th>Total bases </th><th>Percentage of assembled genome </th></tr></thead><tbody><tr><td rowspan="5">I </td><td>DDE_1 </td><td>422 </td><td>369,615 </td><td>1.11 </td></tr><tr><td>gypsy </td><td>279 </td><td>223,716 </td><td>0.67 </td></tr><tr><td>LINE </td><td>54 </td><td>78,738 </td><td>0.24 </td></tr><tr><td>ltr_Roo </td><td>2 </td><td>348 </td><td>0.00001 </td></tr><tr><td>TY1_Copia </td><td>87 </td><td>95,805 </td><td>0.29 </td></tr><tr><td rowspan="7">II </td><td>helitronORF </td><td>4 </td><td>2,673 </td><td>0.01 </td></tr><tr><td>hAT </td><td>21 </td><td>15,879 </td><td>0.05 </td></tr><tr><td>mariner </td><td>55 </td><td>36,201 </td><td>0.11 </td></tr><tr><td>mariner_ant1 </td><td>6 </td><td>2,793 </td><td>0.01 </td></tr><tr><td>MuDR_A_B </td><td>19 </td><td>4,701 </td><td>0.01 </td></tr><tr><td>cacta </td><td>10 </td><td>1,083 </td><td>0.00003 </td></tr><tr><td>Total </td><td> </td><td>831,552 </td><td>2.49 </td></tr></tbody></table></div><div content-id="DSV007TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Class </th><th>Family name </th><th>Total number </th><th>Total bases </th><th>Percentage of assembled genome </th></tr></thead><tbody><tr><td rowspan="5">I </td><td>DDE_1 </td><td>422 </td><td>369,615 </td><td>1.11 </td></tr><tr><td>gypsy </td><td>279 </td><td>223,716 </td><td>0.67 </td></tr><tr><td>LINE </td><td>54 </td><td>78,738 </td><td>0.24 </td></tr><tr><td>ltr_Roo </td><td>2 </td><td>348 </td><td>0.00001 </td></tr><tr><td>TY1_Copia </td><td>87 </td><td>95,805 </td><td>0.29 </td></tr><tr><td rowspan="7">II </td><td>helitronORF </td><td>4 </td><td>2,673 </td><td>0.01 </td></tr><tr><td>hAT </td><td>21 </td><td>15,879 </td><td>0.05 </td></tr><tr><td>mariner </td><td>55 </td><td>36,201 </td><td>0.11 </td></tr><tr><td>mariner_ant1 </td><td>6 </td><td>2,793 </td><td>0.01 </td></tr><tr><td>MuDR_A_B </td><td>19 </td><td>4,701 </td><td>0.01 </td></tr><tr><td>cacta </td><td>10 </td><td>1,083 </td><td>0.00003 </td></tr><tr><td>Total </td><td> </td><td>831,552 </td><td>2.49 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/9633708" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=9633711 id="9633711" class="section-title" data-legacy-id=s3d>3.4. Gene prediction</h3>
<p>Our pipeline showed that there are 128 tRNAs, 12 5S rRNA, 3 18S rRNA and 3 28S rRNA in the genomic sequence. In addition, the <em>B. papendorfii</em> UM 226 genome revealed a total of 11,015 putative coding DNA sequences (CDS) with an average gene length of 1,425 bp. Gene density of protein coding genes is 3.29 genes/10 kb. The genome size of <em>B. papendorfii</em> UM 226, number of predicted genes and gene density are comparable to the previously sequenced <em>Cochliobolus</em> species genomes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). On average, there are 2.95 exons per gene. A total of 10,708, 6,963 and 8,283 gene-coding sequences are homologous to currently available proteins in the NCBI nr, SwissProt and InterPro databases with 7,759 hypothetical proteins identified based on the top hit of the BLAST result against NCBI nr database. The large amount of hypothetical proteins in the genome indicated the uniqueness of <em>B. papendorfii</em> isolate.</p><p>To further characterize the predicted proteins, the genome was mapped to KOG and KEGG databases. All the proteins were ascribed to 26 different functional groups based on KOG classification (Fig. <a reveal-id="DSV007F3" data-open="DSV007F3" class="link link-reveal link-table xref-fig">3</a>a and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). Apart from the poorly characterized categories: categories [R] General functions prediction only, [S] Function unknown and [X] Unnamed protein, the top five KOG groups they were annotated in were [O] Post-translational modification, protein turnover, chaperones (512 genes), [T] Signal transduction mechanisms (396 genes), [I] Lipid transport and metabolism (382 genes), [C] Energy production and conversion (344 genes) and [Q] Secondary metabolites biosynthesis, transport and catabolism (334 genes) (Fig. <a reveal-id="DSV007F3" data-open="DSV007F3" class="link link-reveal link-table xref-fig">3</a>a). Like other <em>Cochliobolus</em> species [<em>C. heterostrophus</em> C4 (560 copies), <em>C. heterostrophus</em> C5 (608 copies), <em>C. victoriae</em> F13 (597 copies), <em>C. sativus</em> ND90Pr (585 copies), <em>C. carbonum</em> 26-R-13 (597 copies), <em>C. lunatus</em> m118 (587 copies)] that have been previously sequenced by JGI, the majority of the genes in <em>B. papendorfii</em> UM 226 genome was involved in post-translational modifications (Fig. <a reveal-id="DSV007F3" data-open="DSV007F3" class="link link-reveal link-table xref-fig">3</a>a). Post-translational modifications of proteins regulate a wide variety of basic cellular functions in fungi, such as cellular growth, adaptive and developmental processes.<sup><a reveal-id="DSV007C43" data-open="DSV007C43" class="link link-ref link-reveal xref-bibr">43</a></sup> Autophagy plays a vital role in cellular growth, degradation as well as cell survival in filamentous fungi, especially in starvation condition.<sup><a reveal-id="DSV007C44" data-open="DSV007C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Interestingly, we identified 103 putative serine/threonine-protein kinase involved in autophagy from the KOG group O (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). Moreover, various fungal cell surface proteins post-translationally modified in fungi are involved in fungal virulence.<sup><a reveal-id="DSV007C43" data-open="DSV007C43" class="link link-ref link-reveal xref-bibr">43</a></sup> To gain insight into the gene functions, we managed to assign 3,074 predicted proteins to their orthologous genes in the KEGG database. At the first glance, there were 1,125 predicted proteins of <em>B. papendorfii</em> UM 226 involved in metabolic pathways in the KEGG database (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). The top five categories in KEGG metabolic pathway are carbohydrate metabolism, amino acid metabolism, lipid metabolism, nucleotide metabolism and xenobiotics biodegradation and metabolism (Fig <a reveal-id="DSV007F3" data-open="DSV007F3" class="link link-reveal link-table xref-fig">3</a>b). There are 293 unique reactions corresponding to carbohydrate metabolism. This result suggests that there are enriched carbohydrate mechanisms such as glycolysis, gluconeogenesis and citrate cycle (TCA cycle), that can facilitate the use of diverse substrates of <em>B. papendorfii</em> UM 226 in exposure to various carbohydrate sources in the environment. </p>                        <a id="9633721" scrollto-destination="9633721"></a>
<div id="DSV007F3" data-id="DSV007F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/m_dsv00703.jpeg?Expires=2147483647&Signature=RQL0uAg83anxP4q5HOLhZ5lYz4Ui5yM7ZEo2aCpYTP34v1Mk2J6TpMwoczabJbP~KSl9JzzfiqSOc1p3N44a1N~4ueBsmwmtsseP36SWsvtTtF7C6s9pbz1TlldAbnZqOeYWlrNmBTUi4tgOTJeQQi7pLI9rA3rutTrp0Cp2OhEp5419FNgcTZW02Lk5Mm9UnMKm~WnF4g1baux7~y655pDYdqMdZjRiOujo0lnCbXi~zl1G~C2O7ZTX47dUSdZf7dPgIdpOUO9SclJc6QjeSiPStbl6ryqHGiZ4MOTM~JjO7PpO38nWdZTSBmmuQntYvliJqIeCpc9ne5W9tM8V5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="KOG and KEGG classifications of proteins in Bipolaris papendorfii UM 226 (A) KOG class annotation distribution of B. papendorfii UM 226 genome. A, RNA processing and modification; B, Chromatin structure and dynamics; C, Energy production and conversion; D, Cell cycle control, cell division, chromosome partitioning; E, Amino acid transport and metabolism; F, Nucleotide transport and metabolism; G, Carbohydrate transport and metabolism; H, Coenzyme transport and metabolism; I, Lipid transport and metabolism; J, Translation, ribosomal structure and biogenesis; K, Transcription; L, Replication, recombination and repair; M, Cell wall/membrane/envelope biogenesis; N, Cell motility; O, Post-translational modification, protein turnover, chaperones; P, Inorganic ion transport and metabolism; Q, Secondary metabolites biosynthesis, transport and catabolism; R, General function prediction only; S, Function unknown; T, Signal transduction mechanisms; U, Intracellular trafficking, secretion, and vesicular transport; V, Defense mechanisms; W, Extracellular structures; X, Unnamed protein; Y, Nuclear structure and Z, Cytoskeleton. (B) Distribution of predicted proteins from B. papendorfii UM 226 genome that involved in in metabolic pathway by KEGG database." path-from-xml="dsv00703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9633721/dsv00703.jpeg" path-from-xml="dsv00703.jpeg" target="_blank">View large</a><a section="9633721" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/dsv00703.jpeg?Expires=2147483647&Signature=AqjGKlSQSOQKXSLOEGn8SQQio86qnF87myp1i92-Y23~YfPbnjbVisCEP--7aSTQICNW1xSIeSiakxzpN6N0TxPTui9fztdg4FXSFITwi-ShTrMTxAr5Xn5HzS9R0NbwBmOtdUg5M-DuZSIbi8G1lOuTfou67gNXaECSuAxEmMLNwH49Qqkzu1hOycpXlbZuj9GDp~WAMpMzu2~VxlOtTG9gD4KLjF9P~7uygcuH9QXjdQ-aQug~2qgf4ySnjGEhpDm85uNCfmKg6-yTdJgRuKyB8Lv~WYMYbU0VzG39M28RA5VUvEAdTVhTSHflxcH8XxjJ4K9N1pSnRW~7N6XLUg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9633721&ar=591394&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>KOG and KEGG classifications of proteins in <em>Bipolaris papendorfii</em> UM 226 (A) KOG class annotation distribution of <em>B. papendorfii</em> UM 226 genome. A, RNA processing and modification; B, Chromatin structure and dynamics; C, Energy production and conversion; D, Cell cycle control, cell division, chromosome partitioning; E, Amino acid transport and metabolism; F, Nucleotide transport and metabolism; G, Carbohydrate transport and metabolism; H, Coenzyme transport and metabolism; I, Lipid transport and metabolism; J, Translation, ribosomal structure and biogenesis; K, Transcription; L, Replication, recombination and repair; M, Cell wall/membrane/envelope biogenesis; N, Cell motility; O, Post-translational modification, protein turnover, chaperones; P, Inorganic ion transport and metabolism; Q, Secondary metabolites biosynthesis, transport and catabolism; R, General function prediction only; S, Function unknown; T, Signal transduction mechanisms; U, Intracellular trafficking, secretion, and vesicular transport; V, Defense mechanisms; W, Extracellular structures; X, Unnamed protein; Y, Nuclear structure and Z, Cytoskeleton. (B) Distribution of predicted proteins from <em>B. papendorfii</em> UM 226 genome that involved in in metabolic pathway by KEGG database.</p></div></div><div content-id="DSV007F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/m_dsv00703.jpeg?Expires=2147483647&Signature=RQL0uAg83anxP4q5HOLhZ5lYz4Ui5yM7ZEo2aCpYTP34v1Mk2J6TpMwoczabJbP~KSl9JzzfiqSOc1p3N44a1N~4ueBsmwmtsseP36SWsvtTtF7C6s9pbz1TlldAbnZqOeYWlrNmBTUi4tgOTJeQQi7pLI9rA3rutTrp0Cp2OhEp5419FNgcTZW02Lk5Mm9UnMKm~WnF4g1baux7~y655pDYdqMdZjRiOujo0lnCbXi~zl1G~C2O7ZTX47dUSdZf7dPgIdpOUO9SclJc6QjeSiPStbl6ryqHGiZ4MOTM~JjO7PpO38nWdZTSBmmuQntYvliJqIeCpc9ne5W9tM8V5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="KOG and KEGG classifications of proteins in Bipolaris papendorfii UM 226 (A) KOG class annotation distribution of B. papendorfii UM 226 genome. A, RNA processing and modification; B, Chromatin structure and dynamics; C, Energy production and conversion; D, Cell cycle control, cell division, chromosome partitioning; E, Amino acid transport and metabolism; F, Nucleotide transport and metabolism; G, Carbohydrate transport and metabolism; H, Coenzyme transport and metabolism; I, Lipid transport and metabolism; J, Translation, ribosomal structure and biogenesis; K, Transcription; L, Replication, recombination and repair; M, Cell wall/membrane/envelope biogenesis; N, Cell motility; O, Post-translational modification, protein turnover, chaperones; P, Inorganic ion transport and metabolism; Q, Secondary metabolites biosynthesis, transport and catabolism; R, General function prediction only; S, Function unknown; T, Signal transduction mechanisms; U, Intracellular trafficking, secretion, and vesicular transport; V, Defense mechanisms; W, Extracellular structures; X, Unnamed protein; Y, Nuclear structure and Z, Cytoskeleton. (B) Distribution of predicted proteins from B. papendorfii UM 226 genome that involved in in metabolic pathway by KEGG database." path-from-xml="dsv00703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9633721/dsv00703.jpeg" path-from-xml="dsv00703.jpeg" target="_blank">View large</a><a section="9633721" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/dsv00703.jpeg?Expires=2147483647&Signature=AqjGKlSQSOQKXSLOEGn8SQQio86qnF87myp1i92-Y23~YfPbnjbVisCEP--7aSTQICNW1xSIeSiakxzpN6N0TxPTui9fztdg4FXSFITwi-ShTrMTxAr5Xn5HzS9R0NbwBmOtdUg5M-DuZSIbi8G1lOuTfou67gNXaECSuAxEmMLNwH49Qqkzu1hOycpXlbZuj9GDp~WAMpMzu2~VxlOtTG9gD4KLjF9P~7uygcuH9QXjdQ-aQug~2qgf4ySnjGEhpDm85uNCfmKg6-yTdJgRuKyB8Lv~WYMYbU0VzG39M28RA5VUvEAdTVhTSHflxcH8XxjJ4K9N1pSnRW~7N6XLUg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9633721&ar=591394&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>KOG and KEGG classifications of proteins in <em>Bipolaris papendorfii</em> UM 226 (A) KOG class annotation distribution of <em>B. papendorfii</em> UM 226 genome. A, RNA processing and modification; B, Chromatin structure and dynamics; C, Energy production and conversion; D, Cell cycle control, cell division, chromosome partitioning; E, Amino acid transport and metabolism; F, Nucleotide transport and metabolism; G, Carbohydrate transport and metabolism; H, Coenzyme transport and metabolism; I, Lipid transport and metabolism; J, Translation, ribosomal structure and biogenesis; K, Transcription; L, Replication, recombination and repair; M, Cell wall/membrane/envelope biogenesis; N, Cell motility; O, Post-translational modification, protein turnover, chaperones; P, Inorganic ion transport and metabolism; Q, Secondary metabolites biosynthesis, transport and catabolism; R, General function prediction only; S, Function unknown; T, Signal transduction mechanisms; U, Intracellular trafficking, secretion, and vesicular transport; V, Defense mechanisms; W, Extracellular structures; X, Unnamed protein; Y, Nuclear structure and Z, Cytoskeleton. (B) Distribution of predicted proteins from <em>B. papendorfii</em> UM 226 genome that involved in in metabolic pathway by KEGG database.</p></div></div>                            <h3 scrollto-destination=9633729 id="9633729" class="section-title" data-legacy-id=s3e>3.5. The CAZymes family</h3>
<p>All types of CAZymes, especially plant cell wall degrading enzymes, are commonly produced by fungal plant pathogens to penetrate the rigid barrier of the plant cell wall.<sup><a reveal-id="DSV007C45" data-open="DSV007C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Most of the fungal CAZymes were known to degrade plant cell wall materials to obtain carbon source for fungal growth as well as for penetration and infection of their host plants.<sup><a reveal-id="DSV007C45" data-open="DSV007C45" class="link link-ref link-reveal xref-bibr">45</a></sup> As <em>B. papendorfii</em> has been reported to cause corn infection,<sup><a reveal-id="DSV007C9" data-open="DSV007C9" class="link link-ref link-reveal xref-bibr">9</a></sup> the <em>B. papendorfii</em> UM 226 genome was mapped to the CAZy database to determine the presence of CAZyme-coding homologous genes. A total of 728 putative CAZyme-coding homologous genes were determined in the genome. This comprises 122 with auxiliary activities (AA), 69 carbohydrate-binding module (CBM), 150 carbohydrate esterases (CE), 266 glycoside hydrolases (GH), 104 glycosyl transferases (GT) and 17 polysaccharide lyases (PL) (Fig. <a reveal-id="DSV007F4" data-open="DSV007F4" class="link link-reveal link-table xref-fig">4</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). The number of GH, GT and PL family candidates identified are comparable to the several reported in several studies for necrotrophic and hemibiotrophic <em>Cochliobolus</em> species.<sup><a reveal-id="DSV007C28" data-open="DSV007C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSV007C45" data-open="DSV007C45" class="link link-ref link-reveal xref-bibr">45</a></sup> The number of GHs in <em>C. heterostrophus</em> C4, <em>C. heterostrophus</em> C5 and <em>C. sativus</em> ND90Pr are 276, 292 and 272, respectively; the number of GTs in <em>C. heterostrophus</em> C4, <em>C. heterostrophus</em> C5 and <em>C. sativus</em> ND90Pr are 96, 103 and 99, respectively; the number of PLs in <em>C. heterostrophus</em> C4, <em>C. heterostrophus</em> C5 and <em>C. sativus</em> ND90Pr are 15, 15 and 15, respectively.<sup><a reveal-id="DSV007C28" data-open="DSV007C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSV007C44" data-open="DSV007C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Interestingly, the <em>B. papendorfii</em> UM 226 genome contains a higher number of CE-coding gene homologs (150 copies) compared with that of the reported <em>Cochliobolus</em> species, such as <em>C. heterostrophus</em> C4 (46 copies), <em>C. heterostrophus</em> C5 (49 copies) and <em>C. sativus</em> ND90Pr (47 copies).<sup><a reveal-id="DSV007C28" data-open="DSV007C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSV007C45" data-open="DSV007C45" class="link link-ref link-reveal xref-bibr">45</a></sup> Our findings revealed that the fungus has 12 of the 16 CE families, with family CE2, CE6, CE11 and CE13 missing. The high number of CE1 (42 copies) and CE10 (57 copies) indicated that <em>B. papendorfii</em> UM 226 might produce many putative plant degrading enzymes containing carboxylesterase and endo-1,4-β-xylanase activities, respectively.<sup><a reveal-id="DSV007C45" data-open="DSV007C45" class="link link-ref link-reveal xref-bibr">45</a></sup> CE1 and CE10 members were reported to have a wide range of substrate specificities. CE1 and CE10 are hemicellulose-targeting CEs.<sup><a reveal-id="DSV007C46" data-open="DSV007C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DSV007C47" data-open="DSV007C47" class="link link-ref link-reveal xref-bibr">47</a></sup> In this study, we identified seven copies of putative endo-1,4-β-xylanases, which are responsible to degrade the 1,4-glycosidic bond and thus depolymerize the xylan polymer (a major carbohydrate component of hemicellulose) into its monomer constituents (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>).<sup><a reveal-id="DSV007C48" data-open="DSV007C48" class="link link-ref link-reveal xref-bibr">48</a></sup> Our result also demonstrated a high number of GH and AA genes, postulating the ability of our fungal isolate to degrade lignocellulose, which is a main structural constituent of non-woody or woody plants. A total of 16 putative ligninolytic genes: multicopper oxidase (6 copies) and heme peroxidase (10 copies) were identified in this work.<sup><a reveal-id="DSV007C49" data-open="DSV007C49" class="link link-ref link-reveal xref-bibr">49</a>,<a reveal-id="DSV007C50" data-open="DSV007C50" class="link link-ref link-reveal xref-bibr">50</a></sup> Multicopper oxidases and heme peroxidases involved in lignin degradation by oxidizing lignin subunits using molecular oxygen (electron acceptor) and extracellular hydrogen peroxide (co-substrate), respectively.<sup><a reveal-id="DSV007C48" data-open="DSV007C48" class="link link-ref link-reveal xref-bibr">48</a>,<a reveal-id="DSV007C49" data-open="DSV007C49" class="link link-ref link-reveal xref-bibr">49</a></sup> Collectively, <em>B. papendorfii</em> UM 226 possesses a wide array of CAZymes that are involved in the degradation of the complex carbohydrates in the plant cell walls. </p>                        <a id="9633736" scrollto-destination="9633736"></a>
<div id="DSV007F4" data-id="DSV007F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/m_dsv00704.jpeg?Expires=2147483647&Signature=4jfEQ-kTEbWAAPVYdrjP2agK8~VaiJXTYQfIPsNOT0Bx7TSBTleFsiouHwKMYNS1Hth-UGRAx64IA3JXc~PL4yZO565A2rjRFTDBO6vVyuUNaFZthps6lrKJpzPtrw0QlgpOVnI~2OEKCyj-L0l-3wnzuK0Pbko5OpPKgekJlw~zK1rVfFna1uqY1N2lvGvs3adZJo~LBvvI5Z7fr2jc0ckf2670oNxJfdomF81G5EZ--UiB7e9Wa8iZfv6Kkpx085pWKZDiYcqAyEUNIzqpyxcuf1I-uJCIMpbGXsqAJWEOy1HGz4nzdIIyFPryuzQwloqjq-tWz~~eOLTdHFCuTQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CAZyme class annotation distribution of Bipolaris papendorfii UM 226 genome. AA, auxiliary activities; CBM, carbohydrate-binding module; CE, carbohydrate esterase; GH, glycoside hydrolases; GT, glycosyltransferase; PL, polysaccharide lyase." path-from-xml="dsv00704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9633736/dsv00704.jpeg" path-from-xml="dsv00704.jpeg" target="_blank">View large</a><a section="9633736" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/dsv00704.jpeg?Expires=2147483647&Signature=Pe67jIJSNR~YSxAmtcUklPEkDYqJJLrb~F7AOSVLAbBp1MoL62iEK4i1RkRtkKeYAbt6gILZNspjBcBPxAEReONmEhpoylc9r~1IUIF35F3ycL4Ew-a~gDaEZdCuxaFNroBUdTaKx43zVndu4zUBihxvJ0d7tYnLpTbEXYh5uDtncV1xwzWGJKExhlyCqpgA~7NgFVXo9FUg-kozfA3ZVxXVa95hf6kFPjH6PAsiA7IyTfO6wSAhI8t0bbDoE1BMm5t2ZBc69kTBezIdqaqJkEhUvAozn30OMIyIr4B1uMAixxT183ui46n0l5ri8gB-tZII~HDYg2d5ns46-zH5gw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9633736&ar=591394&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CAZyme class annotation distribution of <em>Bipolaris papendorfii</em> UM 226 genome. AA, auxiliary activities; CBM, carbohydrate-binding module; CE, carbohydrate esterase; GH, glycoside hydrolases; GT, glycosyltransferase; PL, polysaccharide lyase.</p></div></div><div content-id="DSV007F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/m_dsv00704.jpeg?Expires=2147483647&Signature=4jfEQ-kTEbWAAPVYdrjP2agK8~VaiJXTYQfIPsNOT0Bx7TSBTleFsiouHwKMYNS1Hth-UGRAx64IA3JXc~PL4yZO565A2rjRFTDBO6vVyuUNaFZthps6lrKJpzPtrw0QlgpOVnI~2OEKCyj-L0l-3wnzuK0Pbko5OpPKgekJlw~zK1rVfFna1uqY1N2lvGvs3adZJo~LBvvI5Z7fr2jc0ckf2670oNxJfdomF81G5EZ--UiB7e9Wa8iZfv6Kkpx085pWKZDiYcqAyEUNIzqpyxcuf1I-uJCIMpbGXsqAJWEOy1HGz4nzdIIyFPryuzQwloqjq-tWz~~eOLTdHFCuTQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CAZyme class annotation distribution of Bipolaris papendorfii UM 226 genome. AA, auxiliary activities; CBM, carbohydrate-binding module; CE, carbohydrate esterase; GH, glycoside hydrolases; GT, glycosyltransferase; PL, polysaccharide lyase." path-from-xml="dsv00704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9633736/dsv00704.jpeg" path-from-xml="dsv00704.jpeg" target="_blank">View large</a><a section="9633736" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/dsv00704.jpeg?Expires=2147483647&Signature=Pe67jIJSNR~YSxAmtcUklPEkDYqJJLrb~F7AOSVLAbBp1MoL62iEK4i1RkRtkKeYAbt6gILZNspjBcBPxAEReONmEhpoylc9r~1IUIF35F3ycL4Ew-a~gDaEZdCuxaFNroBUdTaKx43zVndu4zUBihxvJ0d7tYnLpTbEXYh5uDtncV1xwzWGJKExhlyCqpgA~7NgFVXo9FUg-kozfA3ZVxXVa95hf6kFPjH6PAsiA7IyTfO6wSAhI8t0bbDoE1BMm5t2ZBc69kTBezIdqaqJkEhUvAozn30OMIyIr4B1uMAixxT183ui46n0l5ri8gB-tZII~HDYg2d5ns46-zH5gw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9633736&ar=591394&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CAZyme class annotation distribution of <em>Bipolaris papendorfii</em> UM 226 genome. AA, auxiliary activities; CBM, carbohydrate-binding module; CE, carbohydrate esterase; GH, glycoside hydrolases; GT, glycosyltransferase; PL, polysaccharide lyase.</p></div></div>                            <h3 scrollto-destination=9633742 id="9633742" class="section-title" data-legacy-id=s3f>3.6. Comparative genomics of <em>B. papendorfii</em> UM 226 and other <em>Cochliobolus</em> species</h3>
<p>In this work, we selected seven publicly available <em>Cochliobolus</em> species genomes and an <em>A. brassicicola</em> genome for rooting the phylogenomic tree (Fig. <a reveal-id="DSV007F5" data-open="DSV007F5" class="link link-reveal link-table xref-fig">5</a>). A total of 101,438 proteins were clustered into 14,296 orthologous clusters with 6,051 single-copy orthologous genes determined. It should be noted that <em>B. papendorfii</em> UM 226 was tightly clustered with <em>C. lunatus</em> m118. Among the 14,296 gene family clusters, 222 gene families were conserved between <em>B. papendorfii</em> UM 226 and <em>C. lunatus</em> m118. In agreement with the multilocus phylogenetic analysis (Fig. <a reveal-id="DSV007F2" data-open="DSV007F2" class="link link-reveal link-table xref-fig">2</a>), these data showed a close evolutionary relationship between <em>B. papendorfii</em> UM 226 and <em>Curvularia</em> species, which are separated from all plant pathogenic <em>Cochliobolus</em> (anamorph <em>Bipolaris</em>) species (Fig. <a reveal-id="DSV007F5" data-open="DSV007F5" class="link link-reveal link-table xref-fig">5</a>). </p>                        <a id="9633747" scrollto-destination="9633747"></a>
<div id="DSV007F5" data-id="DSV007F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/m_dsv00705.jpeg?Expires=2147483647&Signature=Y6hWaSZFOp8C~Lk10Q1d3WgHvSVW1xHwMLBDVZesvJQ8G-pbtTIoXlL~k06jfyyMwRARvyc8-c~1IPwDbcHwRA87w3QdDdvoOrglRFWZXM87GH6ceVLgZytzei1NHsMuz~dpR~U7PSgwfCnWqq4se-vZNqOA8sDxBL3fJgthGplLC6LXh5I1R4GBbHSnDK~JwZPuFw51S0hHBf8V1pW-poOv42h1K~L9vJ~gW3lZzHqr8dx69g8JW~euypBSneVA1fhqhCc0n4yszHvdxjqmsXneN1PZxFnDwUB0s7BVQoidb4IpTFMSpXCZu7phAaQKKXjAPBwz9SQT3gJE~gc27A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparative phylogenomic analysis of Bipolaris papendorfii UM 226 along with seven previously published Cochliobolus genomes by using Bayesian analysis. Number at the node referring to Bayesian posterior probability. The tree is rooted with Alternaria brassicicola ATCC 96836 as outgroup." path-from-xml="dsv00705.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9633747/dsv00705.jpeg" path-from-xml="dsv00705.jpeg" target="_blank">View large</a><a section="9633747" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/dsv00705.jpeg?Expires=2147483647&Signature=IlzAyK~N10ivzraH2tJktIwlAaVeocW10rVCHztExOdsHXsemoFxXyld9iWMAPjDnLuan6izzqsgHdwi-OPjlERrLU~fe6hO2NTW3P8kbXjCve9AqNpIFXpmKrLAR0r2wPr2RyDJRbtxfdlXgElmkSzfk~FIKUi9iiq-oozhnPNUYP1a-CSYbpYzK5gJLe~lEQJL4MrZoT43D9~lS0bhFSe~SWEwfzOiuXEpWm1BtPSwS9ZcqBUbGfknyLiHhoPm1P97VqobZHkoS7TszC01stk3sSY6e00ArmFqp-rbm0ZT05qnzx2fQPFddGL0GjO1CKfHBpZ0yFSs7uqt1eidBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9633747&ar=591394&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00705.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative phylogenomic analysis of <em>Bipolaris papendorfii</em> UM 226 along with seven previously published <em>Cochliobolus</em> genomes by using Bayesian analysis. Number at the node referring to Bayesian posterior probability. The tree is rooted with <em>Alternaria brassicicola</em> ATCC 96836 as outgroup.</p></div></div><div content-id="DSV007F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/m_dsv00705.jpeg?Expires=2147483647&Signature=Y6hWaSZFOp8C~Lk10Q1d3WgHvSVW1xHwMLBDVZesvJQ8G-pbtTIoXlL~k06jfyyMwRARvyc8-c~1IPwDbcHwRA87w3QdDdvoOrglRFWZXM87GH6ceVLgZytzei1NHsMuz~dpR~U7PSgwfCnWqq4se-vZNqOA8sDxBL3fJgthGplLC6LXh5I1R4GBbHSnDK~JwZPuFw51S0hHBf8V1pW-poOv42h1K~L9vJ~gW3lZzHqr8dx69g8JW~euypBSneVA1fhqhCc0n4yszHvdxjqmsXneN1PZxFnDwUB0s7BVQoidb4IpTFMSpXCZu7phAaQKKXjAPBwz9SQT3gJE~gc27A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparative phylogenomic analysis of Bipolaris papendorfii UM 226 along with seven previously published Cochliobolus genomes by using Bayesian analysis. Number at the node referring to Bayesian posterior probability. The tree is rooted with Alternaria brassicicola ATCC 96836 as outgroup." path-from-xml="dsv00705.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9633747/dsv00705.jpeg" path-from-xml="dsv00705.jpeg" target="_blank">View large</a><a section="9633747" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/dsv00705.jpeg?Expires=2147483647&Signature=IlzAyK~N10ivzraH2tJktIwlAaVeocW10rVCHztExOdsHXsemoFxXyld9iWMAPjDnLuan6izzqsgHdwi-OPjlERrLU~fe6hO2NTW3P8kbXjCve9AqNpIFXpmKrLAR0r2wPr2RyDJRbtxfdlXgElmkSzfk~FIKUi9iiq-oozhnPNUYP1a-CSYbpYzK5gJLe~lEQJL4MrZoT43D9~lS0bhFSe~SWEwfzOiuXEpWm1BtPSwS9ZcqBUbGfknyLiHhoPm1P97VqobZHkoS7TszC01stk3sSY6e00ArmFqp-rbm0ZT05qnzx2fQPFddGL0GjO1CKfHBpZ0yFSs7uqt1eidBA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9633747&ar=591394&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00705.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative phylogenomic analysis of <em>Bipolaris papendorfii</em> UM 226 along with seven previously published <em>Cochliobolus</em> genomes by using Bayesian analysis. Number at the node referring to Bayesian posterior probability. The tree is rooted with <em>Alternaria brassicicola</em> ATCC 96836 as outgroup.</p></div></div><p>Whole proteome alignment of the <em>B. papendorfii</em> UM 226 and the seven <em>Cochliobolus</em> gene contents indicated that <em>B. papendorfii</em> UM 226 contained 37 gene family clusters composed of 83 unique genes. All of these unique genes encode for hypothetical proteins or unknown proteins with no apparent homologs to the currently existing gene sequences deposited in NCBI nr and SwissProt databases. Protein domain analysis revealed that seven of the family clusters were related to resistance-associated proteins, retrotransposon, peptidoglycan-binding function, hydrolytic functions, aspartic peptidase and DNA-binding function (Table <a reveal-id="DSV007TB3" data-open="DSV007TB3" class="link link-reveal link-table xref-fig">3</a>). Among them, particular attention was paid to a hypothetical protein (UM226_638) with multiple lysin motifs (LysMs) domains. Based on the CAZyme annotation, UM226_638 hypothetical protein belongs to the carbohydrate-binding module (CBM50) of CAZyme which is involved in the degradation of chitin or peptidoglycan. Fungal LysM effectors had been shown to have chitin-binding affinity to perturb chitin-induced immunity from their host plant.<sup><a reveal-id="DSV007C51" data-open="DSV007C51" class="link link-ref link-reveal xref-bibr">51</a></sup> A novel virulence factor with LysM domains, known as Ecp6 effector, was well characterized from the tomato pathogenic fungus <em>C. fulvum</em>.<sup><a reveal-id="DSV007C52" data-open="DSV007C52" class="link link-ref link-reveal xref-bibr">52</a></sup> The presence of LysM domains in Ecp6 effector was postulated to protect fungal cell walls from plant chitinases hydrolysis by competing with host receptors for chitin.<sup><a reveal-id="DSV007C52" data-open="DSV007C52" class="link link-ref link-reveal xref-bibr">52</a></sup> As previously noted,<sup><a reveal-id="DSV007C52" data-open="DSV007C52" class="link link-ref link-reveal xref-bibr">52</a></sup> the LysMs-containing UM226_638 protein belongs to type A fungal LysM protein (PF01467) and contains three LysM domains. Many of the type A LysM proteins play an important role in infection process.<sup><a reveal-id="DSV007C53" data-open="DSV007C53" class="link link-ref link-reveal xref-bibr">53</a></sup> For convenient description hereafter, these three domains were assigned as LysM domain I (75–118 amino acids), LysM domain II (338–384 amino acids) and LysM domain III (454–502 amino acids) (Fig. <a reveal-id="DSV007F6" data-open="DSV007F6" class="link link-reveal link-table xref-fig">6</a>a). Unlike <em>C. fulvum</em> secreted Ecp6,<sup><a reveal-id="DSV007C38" data-open="DSV007C38" class="link link-ref link-reveal xref-bibr">38</a>,<a reveal-id="DSV007C39" data-open="DSV007C39" class="link link-ref link-reveal xref-bibr">39</a></sup> we found that the UM226_638 protein do not have a signal peptide for secretion. Thus, UM226_638 protein might not be a secreted protein. Moreover, secondary structure prediction showed that only LysM domain II and domain III have a βααβ secondary structure (Fig. <a reveal-id="DSV007F6" data-open="DSV007F6" class="link link-reveal link-table xref-fig">6</a>b). On the contrary, LysM domain I composed of three α-helices and a β-sheet (Fig. <a reveal-id="DSV007F6" data-open="DSV007F6" class="link link-reveal link-table xref-fig">6</a>b). </p>                        <a id="9633757" scrollto-destination="9633757"></a>
<div content-id="DSV007TB3" class="table-modal"><div class="table-wrap"><div id="DSV007TB3" class="table-wrap-title" data-id="DSV007TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Specific family clusters in <em>Bipolaris papendorfii</em> UM 226</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Families </th><th>Gene ID </th><th>InterPro ID//domain </th></tr></thead><tbody><tr><td rowspan="3">Bipap11034 </td><td>UM226_39 </td><td>IPR023213//Chloramphenicol acetyltransferase-like domain<br />IPR003480//Transferase </td></tr><tr><td>UM226_347 </td><td>IPR023213//Chloramphenicol acetyltransferase-like domain </td></tr><tr><td>UM226_1504 </td><td>IPR023213//Chloramphenicol acetyltransferase-like domain<br />IPR003480//Transferase </td></tr><tr><td rowspan="2">Bipap12004 </td><td>UM226_638 </td><td>IPR018392//LysM domain </td></tr><tr><td>UM226_6391 </td><td>IPR018392//LysM domain </td></tr><tr><td rowspan="2">Bipap11994 </td><td>UM226_10258 </td><td>IPR000383//Alpha/Beta hydrolase fold<br />IPR005674//CocE/NonD hydrolase<br />IPR008979//Galactose-binding domain-like<br />IPR013736//Peptidase S15/CocE/NonD, C-terminal<br />IPR029058//X-Pro dipeptidyl-peptidase-like domain </td></tr><tr><td>UM226_11642 </td><td>IPR000383//Alpha/Beta hydrolase fold<br />IPR005674//CocE/NonD hydrolase<br />IPR029058//X-Pro dipeptidyl-peptidase-like domain </td></tr><tr><td rowspan="3">Bipap10443 </td><td>UM226_9098 </td><td>— </td></tr><tr><td>UM226_9156 </td><td>— </td></tr><tr><td>UM226_10550 </td><td>IPR005162//Retrotransposon gag domain </td></tr><tr><td rowspan="2">Bipap12068 </td><td>UM226_9099 </td><td>IPR021109//Aspartic peptidase domain </td></tr><tr><td>UM226_10987 </td><td>IPR021109//Aspartic peptidase domain </td></tr><tr><td rowspan="2">Bipap12005 </td><td>UM226_6690 </td><td>IPR006600//HTH CenpB-type DNA-binding domain </td></tr><tr><td>UM226_9523 </td><td>IPR006600//HTH CenpB-type DNA-binding domain<br />IPR004875/DDE superfamily endonuclease, CENP-B-like </td></tr><tr><td rowspan="2">Bipap11992 </td><td>UM226_10253 </td><td>IPR029068//Glyoxalase/Bleomycin resistance protein/Dihydroxybiphenyl dioxygenase </td></tr><tr><td>UM226_11078 </td><td>IPR029068//Glyoxalase/Bleomycin resistance protein/Dihydroxybiphenyl dioxygenase </td></tr></tbody></table></div><div content-id="DSV007TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Families </th><th>Gene ID </th><th>InterPro ID//domain </th></tr></thead><tbody><tr><td rowspan="3">Bipap11034 </td><td>UM226_39 </td><td>IPR023213//Chloramphenicol acetyltransferase-like domain<br />IPR003480//Transferase </td></tr><tr><td>UM226_347 </td><td>IPR023213//Chloramphenicol acetyltransferase-like domain </td></tr><tr><td>UM226_1504 </td><td>IPR023213//Chloramphenicol acetyltransferase-like domain<br />IPR003480//Transferase </td></tr><tr><td rowspan="2">Bipap12004 </td><td>UM226_638 </td><td>IPR018392//LysM domain </td></tr><tr><td>UM226_6391 </td><td>IPR018392//LysM domain </td></tr><tr><td rowspan="2">Bipap11994 </td><td>UM226_10258 </td><td>IPR000383//Alpha/Beta hydrolase fold<br />IPR005674//CocE/NonD hydrolase<br />IPR008979//Galactose-binding domain-like<br />IPR013736//Peptidase S15/CocE/NonD, C-terminal<br />IPR029058//X-Pro dipeptidyl-peptidase-like domain </td></tr><tr><td>UM226_11642 </td><td>IPR000383//Alpha/Beta hydrolase fold<br />IPR005674//CocE/NonD hydrolase<br />IPR029058//X-Pro dipeptidyl-peptidase-like domain </td></tr><tr><td rowspan="3">Bipap10443 </td><td>UM226_9098 </td><td>— </td></tr><tr><td>UM226_9156 </td><td>— </td></tr><tr><td>UM226_10550 </td><td>IPR005162//Retrotransposon gag domain </td></tr><tr><td rowspan="2">Bipap12068 </td><td>UM226_9099 </td><td>IPR021109//Aspartic peptidase domain </td></tr><tr><td>UM226_10987 </td><td>IPR021109//Aspartic peptidase domain </td></tr><tr><td rowspan="2">Bipap12005 </td><td>UM226_6690 </td><td>IPR006600//HTH CenpB-type DNA-binding domain </td></tr><tr><td>UM226_9523 </td><td>IPR006600//HTH CenpB-type DNA-binding domain<br />IPR004875/DDE superfamily endonuclease, CENP-B-like </td></tr><tr><td rowspan="2">Bipap11992 </td><td>UM226_10253 </td><td>IPR029068//Glyoxalase/Bleomycin resistance protein/Dihydroxybiphenyl dioxygenase </td></tr><tr><td>UM226_11078 </td><td>IPR029068//Glyoxalase/Bleomycin resistance protein/Dihydroxybiphenyl dioxygenase </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/9633757" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV007TB3" class="table-wrap-title" data-id="DSV007TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Specific family clusters in <em>Bipolaris papendorfii</em> UM 226</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Families </th><th>Gene ID </th><th>InterPro ID//domain </th></tr></thead><tbody><tr><td rowspan="3">Bipap11034 </td><td>UM226_39 </td><td>IPR023213//Chloramphenicol acetyltransferase-like domain<br />IPR003480//Transferase </td></tr><tr><td>UM226_347 </td><td>IPR023213//Chloramphenicol acetyltransferase-like domain </td></tr><tr><td>UM226_1504 </td><td>IPR023213//Chloramphenicol acetyltransferase-like domain<br />IPR003480//Transferase </td></tr><tr><td rowspan="2">Bipap12004 </td><td>UM226_638 </td><td>IPR018392//LysM domain </td></tr><tr><td>UM226_6391 </td><td>IPR018392//LysM domain </td></tr><tr><td rowspan="2">Bipap11994 </td><td>UM226_10258 </td><td>IPR000383//Alpha/Beta hydrolase fold<br />IPR005674//CocE/NonD hydrolase<br />IPR008979//Galactose-binding domain-like<br />IPR013736//Peptidase S15/CocE/NonD, C-terminal<br />IPR029058//X-Pro dipeptidyl-peptidase-like domain </td></tr><tr><td>UM226_11642 </td><td>IPR000383//Alpha/Beta hydrolase fold<br />IPR005674//CocE/NonD hydrolase<br />IPR029058//X-Pro dipeptidyl-peptidase-like domain </td></tr><tr><td rowspan="3">Bipap10443 </td><td>UM226_9098 </td><td>— </td></tr><tr><td>UM226_9156 </td><td>— </td></tr><tr><td>UM226_10550 </td><td>IPR005162//Retrotransposon gag domain </td></tr><tr><td rowspan="2">Bipap12068 </td><td>UM226_9099 </td><td>IPR021109//Aspartic peptidase domain </td></tr><tr><td>UM226_10987 </td><td>IPR021109//Aspartic peptidase domain </td></tr><tr><td rowspan="2">Bipap12005 </td><td>UM226_6690 </td><td>IPR006600//HTH CenpB-type DNA-binding domain </td></tr><tr><td>UM226_9523 </td><td>IPR006600//HTH CenpB-type DNA-binding domain<br />IPR004875/DDE superfamily endonuclease, CENP-B-like </td></tr><tr><td rowspan="2">Bipap11992 </td><td>UM226_10253 </td><td>IPR029068//Glyoxalase/Bleomycin resistance protein/Dihydroxybiphenyl dioxygenase </td></tr><tr><td>UM226_11078 </td><td>IPR029068//Glyoxalase/Bleomycin resistance protein/Dihydroxybiphenyl dioxygenase </td></tr></tbody></table></div><div content-id="DSV007TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Families </th><th>Gene ID </th><th>InterPro ID//domain </th></tr></thead><tbody><tr><td rowspan="3">Bipap11034 </td><td>UM226_39 </td><td>IPR023213//Chloramphenicol acetyltransferase-like domain<br />IPR003480//Transferase </td></tr><tr><td>UM226_347 </td><td>IPR023213//Chloramphenicol acetyltransferase-like domain </td></tr><tr><td>UM226_1504 </td><td>IPR023213//Chloramphenicol acetyltransferase-like domain<br />IPR003480//Transferase </td></tr><tr><td rowspan="2">Bipap12004 </td><td>UM226_638 </td><td>IPR018392//LysM domain </td></tr><tr><td>UM226_6391 </td><td>IPR018392//LysM domain </td></tr><tr><td rowspan="2">Bipap11994 </td><td>UM226_10258 </td><td>IPR000383//Alpha/Beta hydrolase fold<br />IPR005674//CocE/NonD hydrolase<br />IPR008979//Galactose-binding domain-like<br />IPR013736//Peptidase S15/CocE/NonD, C-terminal<br />IPR029058//X-Pro dipeptidyl-peptidase-like domain </td></tr><tr><td>UM226_11642 </td><td>IPR000383//Alpha/Beta hydrolase fold<br />IPR005674//CocE/NonD hydrolase<br />IPR029058//X-Pro dipeptidyl-peptidase-like domain </td></tr><tr><td rowspan="3">Bipap10443 </td><td>UM226_9098 </td><td>— </td></tr><tr><td>UM226_9156 </td><td>— </td></tr><tr><td>UM226_10550 </td><td>IPR005162//Retrotransposon gag domain </td></tr><tr><td rowspan="2">Bipap12068 </td><td>UM226_9099 </td><td>IPR021109//Aspartic peptidase domain </td></tr><tr><td>UM226_10987 </td><td>IPR021109//Aspartic peptidase domain </td></tr><tr><td rowspan="2">Bipap12005 </td><td>UM226_6690 </td><td>IPR006600//HTH CenpB-type DNA-binding domain </td></tr><tr><td>UM226_9523 </td><td>IPR006600//HTH CenpB-type DNA-binding domain<br />IPR004875/DDE superfamily endonuclease, CENP-B-like </td></tr><tr><td rowspan="2">Bipap11992 </td><td>UM226_10253 </td><td>IPR029068//Glyoxalase/Bleomycin resistance protein/Dihydroxybiphenyl dioxygenase </td></tr><tr><td>UM226_11078 </td><td>IPR029068//Glyoxalase/Bleomycin resistance protein/Dihydroxybiphenyl dioxygenase </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/9633757" target="_blank">View Large</a></div></div>                        <a id="9633758" scrollto-destination="9633758"></a>
<div id="DSV007F6" data-id="DSV007F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/m_dsv00706.jpeg?Expires=2147483647&Signature=1kPMZPcA1HF-PEkQhQ21BST~Aha0XMIkItPw57HKXGjxTpJAaaVLaqF3BO3M8fAz46PwW7haGfLtNN0oeLsYJwyyVhO7i8l5OhHLpjxJ9p5tYf28lihjJ7hgwtheDlpIQMM1uggPo8wB6KODOuGRy4STMhE19comhv72SmVD0suaxOOneKE08QAeYQp4ISZMeWpSaTuzv6xXMuBI2Sl7VKAzFpAIpqRIl21-Y5lYHE0PjZCF0raDW1pxZ4ZZygvLXoY~F-052oka4U2soYjqiHvv~1ckcjfTSYthl4UjBUZIt2DOKOjtc0DjSIaRWQhE~m4992oINgowO0Il~~ctAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="LysM domain of Bipolaris papendorfii UM 226_683 hypothetical protein. (A) Schematic representation of the locations of LysM domains in UM226_683 hypothetical protein. I, II and III represent the LysM domain I, II and III. (B) Secondary structure prediction of the LysM domains. Amino acid sequence alignment between UM 226_683 domains and Ecp6 was performed using ClustalW version 2.0. Asterisk (*) indicates positions that have a single, fully conserved residue; colon (:) indicates conservation between groups of strongly similar properties; period (.) indicates conservation between groups of weakly similar properties. (C) Three-dimensional ribbon structures of the LysM domains. Panels 1 and 2 show LysM domain II and III, respectively. Panel 4 displays the molecular surface of LysM domain III. The arrow in panel 4 shows the shallow groove described as binding site of chitin oligomers.52 α-helices are shown in red; β-sheets are shown in yellow and random coils are shown in black. The N- and C-terminal ends are labelled." path-from-xml="dsv00706.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9633758/dsv00706.jpeg" path-from-xml="dsv00706.jpeg" target="_blank">View large</a><a section="9633758" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/dsv00706.jpeg?Expires=2147483647&Signature=Qk-KzsmtNHu-JV8rvUm363E88aVV-83ym0qF~KfT~OQtkzixf8kIVKb3qp9iMAw48UgsYc8CDL-skYP1yhbas5kLdtJoM2jAh17OAuiEdolGGVwFCdOvuAqd4MgwLtghj0DMZ2kMz0P2wPESYYLPH2E5Pv3XZt4RcPIT0mayzMYfAIr3MCzm2U3mrObY1ZdmtptXR0osGMiceeeoEMD99EP~m8WAfx~R003qCPsIJAyRbE8H5KaBz5xJJInLmwh~X5JksaWASfLDl0x7npdaQ1COauyv1ittwkzGBfkO0UboMc31FFraVyxvn62LaZvMV7MLY~JTFJxdmOiR3QTq1Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9633758&ar=591394&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00706.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>LysM domain of <em>Bipolaris papendorfii</em> UM 226_683 hypothetical protein. (A) Schematic representation of the locations of LysM domains in UM226_683 hypothetical protein. I, II and III represent the LysM domain I, II and III. (B) Secondary structure prediction of the LysM domains. Amino acid sequence alignment between UM 226_683 domains and Ecp6 was performed using ClustalW version 2.0. Asterisk (*) indicates positions that have a single, fully conserved residue; colon (:) indicates conservation between groups of strongly similar properties; period (.) indicates conservation between groups of weakly similar properties. (C) Three-dimensional ribbon structures of the LysM domains. Panels 1 and 2 show LysM domain II and III, respectively. Panel 4 displays the molecular surface of LysM domain III. The arrow in panel 4 shows the shallow groove described as binding site of chitin oligomers.<sup><a reveal-id="DSV007C52" data-open="DSV007C52" class="link link-ref link-reveal xref-bibr">52</a></sup> α-helices are shown in red; β-sheets are shown in yellow and random coils are shown in black. The N- and C-terminal ends are labelled.</p></div></div><div content-id="DSV007F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/m_dsv00706.jpeg?Expires=2147483647&Signature=1kPMZPcA1HF-PEkQhQ21BST~Aha0XMIkItPw57HKXGjxTpJAaaVLaqF3BO3M8fAz46PwW7haGfLtNN0oeLsYJwyyVhO7i8l5OhHLpjxJ9p5tYf28lihjJ7hgwtheDlpIQMM1uggPo8wB6KODOuGRy4STMhE19comhv72SmVD0suaxOOneKE08QAeYQp4ISZMeWpSaTuzv6xXMuBI2Sl7VKAzFpAIpqRIl21-Y5lYHE0PjZCF0raDW1pxZ4ZZygvLXoY~F-052oka4U2soYjqiHvv~1ckcjfTSYthl4UjBUZIt2DOKOjtc0DjSIaRWQhE~m4992oINgowO0Il~~ctAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="LysM domain of Bipolaris papendorfii UM 226_683 hypothetical protein. (A) Schematic representation of the locations of LysM domains in UM226_683 hypothetical protein. I, II and III represent the LysM domain I, II and III. (B) Secondary structure prediction of the LysM domains. Amino acid sequence alignment between UM 226_683 domains and Ecp6 was performed using ClustalW version 2.0. Asterisk (*) indicates positions that have a single, fully conserved residue; colon (:) indicates conservation between groups of strongly similar properties; period (.) indicates conservation between groups of weakly similar properties. (C) Three-dimensional ribbon structures of the LysM domains. Panels 1 and 2 show LysM domain II and III, respectively. Panel 4 displays the molecular surface of LysM domain III. The arrow in panel 4 shows the shallow groove described as binding site of chitin oligomers.52 α-helices are shown in red; β-sheets are shown in yellow and random coils are shown in black. The N- and C-terminal ends are labelled." path-from-xml="dsv00706.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9633758/dsv00706.jpeg" path-from-xml="dsv00706.jpeg" target="_blank">View large</a><a section="9633758" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/dsv00706.jpeg?Expires=2147483647&Signature=Qk-KzsmtNHu-JV8rvUm363E88aVV-83ym0qF~KfT~OQtkzixf8kIVKb3qp9iMAw48UgsYc8CDL-skYP1yhbas5kLdtJoM2jAh17OAuiEdolGGVwFCdOvuAqd4MgwLtghj0DMZ2kMz0P2wPESYYLPH2E5Pv3XZt4RcPIT0mayzMYfAIr3MCzm2U3mrObY1ZdmtptXR0osGMiceeeoEMD99EP~m8WAfx~R003qCPsIJAyRbE8H5KaBz5xJJInLmwh~X5JksaWASfLDl0x7npdaQ1COauyv1ittwkzGBfkO0UboMc31FFraVyxvn62LaZvMV7MLY~JTFJxdmOiR3QTq1Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9633758&ar=591394&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00706.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>LysM domain of <em>Bipolaris papendorfii</em> UM 226_683 hypothetical protein. (A) Schematic representation of the locations of LysM domains in UM226_683 hypothetical protein. I, II and III represent the LysM domain I, II and III. (B) Secondary structure prediction of the LysM domains. Amino acid sequence alignment between UM 226_683 domains and Ecp6 was performed using ClustalW version 2.0. Asterisk (*) indicates positions that have a single, fully conserved residue; colon (:) indicates conservation between groups of strongly similar properties; period (.) indicates conservation between groups of weakly similar properties. (C) Three-dimensional ribbon structures of the LysM domains. Panels 1 and 2 show LysM domain II and III, respectively. Panel 4 displays the molecular surface of LysM domain III. The arrow in panel 4 shows the shallow groove described as binding site of chitin oligomers.<sup><a reveal-id="DSV007C52" data-open="DSV007C52" class="link link-ref link-reveal xref-bibr">52</a></sup> α-helices are shown in red; β-sheets are shown in yellow and random coils are shown in black. The N- and C-terminal ends are labelled.</p></div></div><p>Tertiary structures of the LysM domain II and LysM domain III were then generated by using the SWISS-MODEL server based on the experimentally solved structural homolog. As previously reported,<sup><a reveal-id="DSV007C38" data-open="DSV007C38" class="link link-ref link-reveal xref-bibr">38</a>,<a reveal-id="DSV007C39" data-open="DSV007C39" class="link link-ref link-reveal xref-bibr">39</a></sup> the three-dimensional structural prediction showed that the domains comprised two α-helices packed on the same side of the anti-parallel two β-sheets (Fig. <a reveal-id="DSV007F6" data-open="DSV007F6" class="link link-reveal link-table xref-fig">6</a>c, panel 1 and 2). The molecular surface of the third LysM domain of UM226_638 was computed (Fig. <a reveal-id="DSV007F6" data-open="DSV007F6" class="link link-reveal link-table xref-fig">6</a>c, panel 3). In consistent with the structural homology of <em>C. fulvum</em> Ecp6,<sup><a reveal-id="DSV007C52" data-open="DSV007C52" class="link link-ref link-reveal xref-bibr">52</a></sup> a shallow groove was observed in the LysM domain III that provides a binding site for chitin oligomers. Accuracy of the domain models generated was verified by PROCHECK program, a protein structure validation program (data not shown). The stereochemical parameters were all indicated to be within the acceptable limit, and it can serve as a good hypothetical domain structures for further functional study. Taken together, the putative unique LysMs-containing UM226_638 protein might be a fungal avirulence protein that involved in fungal virulence on its host.</p>                            <h3 scrollto-destination=9633768 id="9633768" class="section-title" data-legacy-id=s3g>3.7. Sexual reproduction</h3>
<p>Sexual reproduction is well characterized for most of the known <em>Cochliobolus</em> species.<sup><a reveal-id="DSV007C29" data-open="DSV007C29" class="link link-ref link-reveal xref-bibr">29</a></sup> In <em>Cochliobolus</em> species, a single mating type locus (MAT) was important to regulate sexual development and production of sexual spores.<sup><a reveal-id="DSV007C54" data-open="DSV007C54" class="link link-ref link-reveal xref-bibr">54</a></sup> The coexistence of heterothallic and homothallic species are common in the genus of <em>Cochliobolus</em>. Thus, sexual <em>Cochliobolus</em> species can be either heterothallic fungi that require two strains of opposite mating type to participate in the sexual process (<em>MAT1-1</em> or <em>MAT1-2</em> gene) or homothallic fungi that carry both <em>MAT1-1</em> and <em>MAT1-2</em> genes.<sup><a reveal-id="DSV007C29" data-open="DSV007C29" class="link link-ref link-reveal xref-bibr">29</a></sup> In this study, we successfully identified a putative <em>MAT1-2</em> (UM226_825) gene but no <em>MAT1-1</em> gene in the UM 226 genome. UM226_825 shared a homology (81% identical) and domain architecture (DNA domain of the high mobility group, HMG domain) with the MAT-2 from <em>Curvularia ellisii</em>. The structural organization of 12 kb at the <em>B. papendorfii</em> UM 226 <em>MAT1-2</em> region was examined (Fig. <a reveal-id="DSV007F7" data-open="DSV007F7" class="link link-reveal link-table xref-fig">7</a>a). As previously described,<sup><a reveal-id="DSV007C6" data-open="DSV007C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSV007C54" data-open="DSV007C54" class="link link-ref link-reveal xref-bibr">54</a></sup> three other putative genes have been identified adjacent to the <em>MAT1-2</em>: (i) a β-glucosidase homolog gene (3′ flank of <em>MAT1-2</em>), (ii) a homolog of a yeast gene (ORF1) with unknown function (5′ flank of <em>MAT1-2</em>) and (iii) a GTPase-activating homolog gene (5′ flank of <em>MAT1-2</em>) (Fig. <a reveal-id="DSV007F7" data-open="DSV007F7" class="link link-reveal link-table xref-fig">7</a>a). However, these genes were not involved in sexual reproduction.<sup><a reveal-id="DSV007C54" data-open="DSV007C54" class="link link-ref link-reveal xref-bibr">54</a></sup> Turgeon <em>et al.</em><sup><a reveal-id="DSV007C54" data-open="DSV007C54" class="link link-ref link-reveal xref-bibr">54</a></sup> showed that the <em>MAT1-2</em> gene evolved at a faster rate than other nucleotide sequences in the genome of <em>Cochliobolus</em> species. In this study, a phylogenetic tree was built to reveal evolutionary relationships between <em>B. papendorfii</em> and other related species (Fig. <a reveal-id="DSV007F7" data-open="DSV007F7" class="link link-reveal link-table xref-fig">7</a>b). <em>MAT1-2</em> genes from three homothallic fungi and six heterothallic fungi were included in the analysis (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>). Based on the phylogenetic analysis, the <em>MAT1-2</em> genes can be divided into two groups, which agree well with the tree previously discussed (Fig. <a reveal-id="DSV007F2" data-open="DSV007F2" class="link link-reveal link-table xref-fig">2</a>). <em>B. papendorfii MAT1-2</em> is tightly clustered together with the <em>MAT1-2</em> of heterothallic <em>C. ellisii</em>, which are separated from another group that consisted of virulent plant pathogens. The close relationship of asexual <em>B. papendorfii</em> to the sexual <em>C. ellisii</em> suggests that <em>B. papendorfii</em> can also be reproduced sexually in field. </p>                        <a id="9633770" scrollto-destination="9633770"></a>
<div id="DSV007F7" data-id="DSV007F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/m_dsv00707.jpeg?Expires=2147483647&Signature=eKAioiqCcGcMjZxZIs6nL5XNaP1bIrCZMBBW73BD5Gpl9OfoHyDm~8JCIBhwfk3u8~s~gfLMkEkq2rHpzcuXcDJWUO-jDSxFafhavmEKhzCQPl3ywtrLbZsP91TIK~D7pGKc5m3acl~bybMbjU8b6vEugIfsQ~sMy~SrSkaawZIN3OsrDWit4mMLGWraR5yTOYhh~TD7CAHAeB~TCcHOH1iCnVgcJ4eFd2PiwNtIu58z8wiU4IOejvLOoYhRT0j6RlwCZPvVzTqBDFiTjrRyyggrzxRj1X6px0bRASH6dKGHSEGu5Oi8NVs1mEWVB97hA-cmlJcV978-YqyrKjr6fg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="MAT1-2 gene of Bipolaris papendorfii UM 226. (A) Schematic representation of major open reading frames (ORFs) identified in the MAT regions of B. papendorfii UM 226 and Cochliobolus heterostrophus C4. Numbers are in kilobases. (B) Bayesian phylogram generated based on MAT1-2 nucleotide sequences. The tree is rooted with Alternaria alternata as outgroup. Numbers on the nodes indicate Bayesian posterior probability based on 100 sampling frequency for a total of 150,000 generations. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00707.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9633770/dsv00707.jpeg" path-from-xml="dsv00707.jpeg" target="_blank">View large</a><a section="9633770" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/dsv00707.jpeg?Expires=2147483647&Signature=eBmVqLnbAuEnuBaa5PINErHA-C01pWi6sN~KmMv9h8-xeEOzFttk9NHOIw43shxa0-RHRpgXMNEzVks0VY0uoUaVmoE26yLM-Ev4lA~X8Chbpi~AM6V9RTQNOqmFmhlITIPup7Dv6mYL83G8iDYddXFE2CwbHrlxmRvjL-lXhsdFyf156X0UfrYT98iuOzSFSWqdjGImI4r59MPZnAuE-JHMJkljOzSC-Z-DiFkDYRoyirDr5mObVhSL5hHMFMrSx46JKZP~p4tPsQ65d4dB5zx5qzkMG0XfiA3ySn61L9KLuw7fi~270ByZUqmbLqkbaT~2d6A1DE5-fk1wPr1CDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9633770&ar=591394&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00707.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>MAT1-2</em> gene of <em>Bipolaris papendorfii</em> UM 226. (A) Schematic representation of major open reading frames (ORFs) identified in the <em>MAT</em> regions of <em>B. papendorfii</em> UM 226 and <em>Cochliobolus heterostrophus</em> C4. Numbers are in kilobases. (B) Bayesian phylogram generated based on <em>MAT1-2</em> nucleotide sequences. The tree is rooted with <em>Alternaria alternata</em> as outgroup. Numbers on the nodes indicate Bayesian posterior probability based on 100 sampling frequency for a total of 150,000 generations. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV007F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/m_dsv00707.jpeg?Expires=2147483647&Signature=eKAioiqCcGcMjZxZIs6nL5XNaP1bIrCZMBBW73BD5Gpl9OfoHyDm~8JCIBhwfk3u8~s~gfLMkEkq2rHpzcuXcDJWUO-jDSxFafhavmEKhzCQPl3ywtrLbZsP91TIK~D7pGKc5m3acl~bybMbjU8b6vEugIfsQ~sMy~SrSkaawZIN3OsrDWit4mMLGWraR5yTOYhh~TD7CAHAeB~TCcHOH1iCnVgcJ4eFd2PiwNtIu58z8wiU4IOejvLOoYhRT0j6RlwCZPvVzTqBDFiTjrRyyggrzxRj1X6px0bRASH6dKGHSEGu5Oi8NVs1mEWVB97hA-cmlJcV978-YqyrKjr6fg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="MAT1-2 gene of Bipolaris papendorfii UM 226. (A) Schematic representation of major open reading frames (ORFs) identified in the MAT regions of B. papendorfii UM 226 and Cochliobolus heterostrophus C4. Numbers are in kilobases. (B) Bayesian phylogram generated based on MAT1-2 nucleotide sequences. The tree is rooted with Alternaria alternata as outgroup. Numbers on the nodes indicate Bayesian posterior probability based on 100 sampling frequency for a total of 150,000 generations. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv00707.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/9633770/dsv00707.jpeg" path-from-xml="dsv00707.jpeg" target="_blank">View large</a><a section="9633770" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/dsv00707.jpeg?Expires=2147483647&Signature=eBmVqLnbAuEnuBaa5PINErHA-C01pWi6sN~KmMv9h8-xeEOzFttk9NHOIw43shxa0-RHRpgXMNEzVks0VY0uoUaVmoE26yLM-Ev4lA~X8Chbpi~AM6V9RTQNOqmFmhlITIPup7Dv6mYL83G8iDYddXFE2CwbHrlxmRvjL-lXhsdFyf156X0UfrYT98iuOzSFSWqdjGImI4r59MPZnAuE-JHMJkljOzSC-Z-DiFkDYRoyirDr5mObVhSL5hHMFMrSx46JKZP~p4tPsQ65d4dB5zx5qzkMG0XfiA3ySn61L9KLuw7fi~270ByZUqmbLqkbaT~2d6A1DE5-fk1wPr1CDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=9633770&ar=591394&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv00707.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>MAT1-2</em> gene of <em>Bipolaris papendorfii</em> UM 226. (A) Schematic representation of major open reading frames (ORFs) identified in the <em>MAT</em> regions of <em>B. papendorfii</em> UM 226 and <em>Cochliobolus heterostrophus</em> C4. Numbers are in kilobases. (B) Bayesian phylogram generated based on <em>MAT1-2</em> nucleotide sequences. The tree is rooted with <em>Alternaria alternata</em> as outgroup. Numbers on the nodes indicate Bayesian posterior probability based on 100 sampling frequency for a total of 150,000 generations. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=9633776 id="9633776" class="section-title" data-legacy-id=s3h>3.8. Secondary metabolism</h3>
<p>Fungal secondary metabolites are a double-edged sword. They are the source of medicinal biologically active compounds. They are also notorious mycotoxins that cause diseases on plants, animals and humans. Armed with the knowledge that secondary metabolite biosynthetic genes often occur in clusters with a key backbone gene.<sup><a reveal-id="DSV007C55" data-open="DSV007C55" class="link link-ref link-reveal xref-bibr">55</a></sup> In this study, we used SMURF prediction to annotate potential gene cluster backbone enzymes in the <em>B. papendorfii</em> UM 226 genome. There are 32 gene clusters encoding core enzymes, including 19 genes encoding polyketide synthase (PKS) or PKS-like, 11 genes encoding non-ribosomal peptide synthetase (NRPS) or NRPS-like, a gene encoding polyketide synthase/non-ribosomal peptide synthetase hybrid (PKS-NRPS) and a gene encoding dimethylallyl tryptophan synthase (DMATS), that are important for the biosynthesis of melanin, bioactive compounds and mycotoxins. The only <em>DMATS</em> gene identified in <em>B. papendorfii</em> UM 226 is UM226_1500 (48% identical), which is often involved in the first committed step of ergot alkaloids biosynthesis.<sup><a reveal-id="DSV007C56" data-open="DSV007C56" class="link link-ref link-reveal xref-bibr">56</a></sup> Ergot alkaloids are mycotoxins but also important natural pharmaceuticals for human use.<sup><a reveal-id="DSV007C57" data-open="DSV007C57" class="link link-ref link-reveal xref-bibr">57</a></sup></p><p>Like most <em>Cochliobolus</em> plant pathogens reported,<sup><a reveal-id="DSV007C6" data-open="DSV007C6" class="link link-ref link-reveal xref-bibr">6</a></sup><em>B. papendorfii</em> UM 226 is enriched with multidomain NRPS encoding genes (NPS) and PKS genes. <em>NPS2</em> (UM226_2338, 74% identical), <em>NPS6</em> (UM226_3398, 87% identical), <em>NPS4</em> (UM226_5568, 82% identical) and <em>NPS10</em> (UM226_3258, 93% identical) orthologous genes found in <em>B. papendorfii</em> UM 226 were reported to be conserved across all <em>Cochliobolus</em> species.<sup><a reveal-id="DSV007C29" data-open="DSV007C29" class="link link-ref link-reveal xref-bibr">29</a></sup> NPS2, NPS6, NPS4 and NPS10 harbour multiple domains, including AMP-binding domain, condensation domain, phosphopantetheine-binding domain and amino acid adenylation domain. NPS2 is likely responsible for the synthesis of siderophore and intracellular iron storage.<sup><a reveal-id="DSV007C28" data-open="DSV007C28" class="link link-ref link-reveal xref-bibr">28</a></sup> All of the reported <em>Cochliobolus</em> pathogens are necrotrophs, which produce Host Selective Toxins (HSTs) to infect cereal crops.<sup><a reveal-id="DSV007C58" data-open="DSV007C58" class="link link-ref link-reveal xref-bibr">58</a></sup> Like <em>C. carbonum</em> (<em>Bipolaris zeicola</em>), <em>B. papendorfii</em> has been reported to cause Corn Leaf Spot disease.<sup><a reveal-id="DSV007C9" data-open="DSV007C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Thus, it is not surprising that <em>B. papendorfii</em> UM 226 contains genes encoding HST. Interestingly, UM226_5568 shared a homology (69% identical) with the HC-toxin synthetase from <em>Pyrenophora tritici-repentis</em> (strain Pt-1C-BFP). As noted previously,<sup><a reveal-id="DSV007C6" data-open="DSV007C6" class="link link-ref link-reveal xref-bibr">6</a></sup> the NPS4 encoded by UM226_5568 might be responsible for producing HC-toxin. In addition, four putative HC-toxin efflux carrier TOXAs were identified in the genome, including UM226_2212 (77% identical), UM226_5722 (74% identical), UM226_9947 (55% identical) and UM226_6817 (50% identical). These putative HC-toxin efflux carrier TOXAs have a potential role in self-protection against secreted HC-toxin.</p><p>Melanin production, a hallmark of dematiaceous fungi, is responsible for the protection of the fungi from UV irradiation, high temperatures as well as oxidants.<sup><a reveal-id="DSV007C59" data-open="DSV007C59" class="link link-ref link-reveal xref-bibr">59</a></sup><em>PKS18</em>, encoding a PKS for melanin biosynthesis, is conserved in all <em>Cochliobolus</em> plant pathogens.<sup><a reveal-id="DSV007C29" data-open="DSV007C29" class="link link-ref link-reveal xref-bibr">29</a></sup> However, this <em>PKS18</em> gene was not identified in <em>B. papendorfii</em> UM 226, implying that that there must be other genes, such as PKS1 (UM226_1411, 93% identical), PKS2 (UM226_3388, 84% identical), PKS4 (UM226_8230, 79% identical), PKS7 (UM226_6241, 65% identical), PKS9 (UM226_7130, 77% identical) and PKS11 (UM226_7830, 85% identical) that can be responsible for melanin biosynthesis. Among these, <em>PKS1</em> (UM226_1411), <em>BRN1</em> (UM226_1408, 98%) and transcription factor <em>CMR1</em> (UM226_1409, 84%) are located in a close group, and their mutual orientation is the same as found in <em>C. heterostrophus</em> and <em>A. brassicicola</em>.<sup><a reveal-id="DSV007C60" data-open="DSV007C60" class="link link-ref link-reveal xref-bibr">60</a></sup> Moreover, two important melanin biosynthetic genes, scytalone dehydratases (<em>SCD1</em>), UM226_59, (98% identical) and UM226_7826 (65% identical), were identified. PKS1, BRN1 and SCD1 are the key enzymes involved in the 1,8-dihydroxynaphthalene (DHN) biosynthesis pathway for the melanin biosynthesis.<sup><a reveal-id="DSV007C60" data-open="DSV007C60" class="link link-ref link-reveal xref-bibr">60</a></sup> Transcription factor CMR1 plays an important role to control <em>BRN1</em> and <em>SCD1</em> genes expression during melanin production.<sup><a reveal-id="DSV007C61" data-open="DSV007C61" class="link link-ref link-reveal xref-bibr">61</a></sup> Eliahu <em>et al.</em><sup><a reveal-id="DSV007C60" data-open="DSV007C60" class="link link-ref link-reveal xref-bibr">60</a></sup> indicated that the expression of <em>CMR1</em> is regulated by two mitogen-activated protein kinases (MAPKs) known as Mps1 and Chk1 for normal pigmentation; both enzymes were found in our genome. Two Mps1 (UM226_10400, 95% identical and UM226_10401, 93% identical) and two Chk1 (UM226_10509, 87% identical and UM226_10016, 50% identical) were identified. Overall, these findings suggested that like most of the dematiaceous fungi, melanin is biosynthesized via the DHN-melanin pathway, which is mediated by the transcription factor CMR1 and two signaling components, Mps1 and Chk1.</p><p>Terpenoids, also referred to as isoprenoids, are a diverse and the largest group of natural bioactive compounds. Many terpenoids such as the anticancer drug paclitaxel and the antimalarial drug Artemisinin are well recognized for human disease therapy.<sup><a reveal-id="DSV007C62" data-open="DSV007C62" class="link link-ref link-reveal xref-bibr">62</a></sup> In this study, we identified 10 core enzymes involved in the mevalonate (MVA) pathway for terpenoid biosynthesis (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). The enzymes 3-hydroxy-3-methylglutaryl-coenzyme A reductase and acetyl-CoA C-acetyltransferase are encoded by two and four copies of the genes, respectively. The remaining eight enzymes are encoded by single-copy genes. Taken together, terpenoid backbone biosynthesis in <em>B. papendorfii</em> UM 226 might occur mainly through the MVA pathway.</p><p>Caffeine (1,3,7-trimethylxanthine), a purine alkaloid, which is one of the world's most popular psychoactive drug in pharmaceuticals, with an estimated consumption of 120,000 tons worldwide per year.<sup><a reveal-id="DSV007C63" data-open="DSV007C63" class="link link-ref link-reveal xref-bibr">63</a></sup> Caffeine is a large environmental concern because of the production of millions tons of caffeine-containing waste products every year. There are few publication about microorganisms capable of caffeine degradation have been isolated from different environments.<sup><a reveal-id="DSV007C64" data-open="DSV007C64" class="link link-ref link-reveal xref-bibr">64</a>,<a reveal-id="DSV007C65" data-open="DSV007C65" class="link link-ref link-reveal xref-bibr">65</a></sup> Sauer <em>et al.</em><sup><a reveal-id="DSV007C66" data-open="DSV007C66" class="link link-ref link-reveal xref-bibr">66</a></sup> indicated that caffeine was degraded by cytochrome P450 enzymes in yeast, postulating that the catabolic pathway might be more similar to that in animals. In this work, we searched the <em>B. papendorfii</em> UM226 genome for potential genes involved in caffeine metabolism. Two cytochrome P450 enzymes (UM226_867, 78% identical and UM226_7104, 39% identical) were identified in the genome based on the KEGG metabolic pathway analysis. In addition, we also found two cytochrome P450 2E1 (UM226_1044, 81% identical and UM226_4157, 79% identical) that catalyze caffeine metabolism via N-demethylation pathway in human liver cells. The production of methylxanthines during the course of caffeine degradation can be further converted to the methyluric acids by xanthine oxidase.<sup><a reveal-id="DSV007C64" data-open="DSV007C64" class="link link-ref link-reveal xref-bibr">64</a></sup> In this work, two xanthine oxidases (UM226_7607, 92% identical and UM226_7880, 86% identical) were found in the <em>B. papendorfii</em> UM 226 genome. Moreover, we also identified an uricase (UM226_10622, 83% identical) that might responsible to convert free xanthine into simpler compounds.<sup><a reveal-id="DSV007C64" data-open="DSV007C64" class="link link-ref link-reveal xref-bibr">64</a></sup> This suggests that <em>B. papendorfii</em> UM 226 may be a potential caffeine-resistant strain of fungus.</p><p>In this study, we successfully generated a high-quality draft genome sequence of <em>B. papendorfii</em> UM 226 using hybrid assembly of Illumina and PacBio sequencing technologies. This is the first genome assembly of <em>B. papendorfii</em>, which allows us to carry out phylogenetic and comparative genomic analyses. Phylogenomic analysis revealed that <em>B. papendorfii</em> UM 226 tightly clustered with <em>C. lunatus</em> m118, which is consistent with the multilocus phylogenetic analysis. However, at this stage of knowledge, the long standing confusion in the taxonomic grouping of <em>Bipolaris</em>, <em>Curvularia</em> and <em>Cochliobolus</em> remains unsolved. In this work, we identified a unique gene encoding LysM-containing protein. The LysM-containing protein has three LysM domains which are predicted to bind chitin to perturb chitin-induced immunity from the host. Furthermore, <em>B. papendorfii</em> UM226 are enriched with putative DMAT, PKS and NRPS encoding genes. The presence of these key enzymes will serve as a landmark for future discovery of bioactive molecules as well as mycotoxins. We hope that the dissection of the <em>B. papendorfii</em> UM226 genome sequence and in-depth analysis of the genome content will help to decipher its basic biology and provide a valuable platform for future fungal research.</p>                            <h2 scrollto-destination=9633790 id="9633790" class="section-title" data-legacy-id=s4>4. Availability</h2>
<p>This Whole Genome Shotgun project has been deposited at DDBJ/EMBL/GenBank under the accession JXCC00000000. The version described in this paper is version JXCC01000000.</p>                            <h2 scrollto-destination=9633798 id="9633798" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=9633805 id="9633805" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This study was supported by High Impact Research MoE Grant UM.C/625/1/HIR/MOHE/MED/31 (No. H-20001-00-E000070). Funding to pay the Open Access publication charges for this article was provided by High Impact Research MoE Grant UM.C/625/1/HIR/MOHE/MED/31 (No. H-20001-00-E000070).</p>                            <h2 scrollto-destination=9633807 id="9633807" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV007C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">da Cunha</div> <div class="given-names">K.C.</div></div> <div class="name"><div class="surname">Sutton</div> <div class="given-names">D.A.</div></div> <div class="name"><div class="surname">Fothergill</div> <div class="given-names">A.W.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">Diversity of <em>Bipolaris</em> species in clinical samples in the United States and their antifungal susceptibility profiles</div>, <div class="source">J. Clin. Microbiol.</div> , <div class="volume">50</div>, <div class="fpage">4061</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20of%20Bipolaris%20species%20in%20clinical%20samples%20in%20the%20United%20States%20and%20their%20antifungal%20susceptibility%20profiles&amp;author=K.C.%20da%20Cunha&amp;author=D.A.%20Sutton&amp;author=A.W.%20Fothergill&amp;publication_year=2012&amp;journal=J.%20Clin.%20Microbiol.&amp;volume=50&amp;pages=4061-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JCM.01965-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJCM.01965-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJCM.01965-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23052310" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Manamgoda</div> <div class="given-names">D.S.</div></div> <div class="name"><div class="surname">Cai</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">McKenzie</div> <div class="given-names">E.H.C.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">A phylogenetic and taxonomic re-evaluation of the <em>Bipolaris</em> - <em>Cochliobolus</em> - <em>Curvularia</em> complex</div>, <div class="source">Fungal Diver.</div> , <div class="volume">56</div>, <div class="fpage">131</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20phylogenetic%20and%20taxonomic%20re-evaluation%20of%20the%20Bipolaris%20-%20Cochliobolus%20-%20Curvularia%20complex&amp;author=D.S.%20Manamgoda&amp;author=L.%20Cai&amp;author=E.H.C.%20McKenzie&amp;publication_year=2012&amp;journal=Fungal%20Diver.&amp;volume=56&amp;pages=131-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s13225-012-0189-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs13225-012-0189-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs13225-012-0189-2"> </span></div> </div></div></div></div></div><div content-id="DSV007C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Manamgoda</div> <div class="given-names">D.S.</div></div> <div class="name"><div class="surname">Cai</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Bahkali</div> <div class="given-names">A.H.</div></div> <div class="name"><div class="surname">Chukeatirote</div> <div class="given-names">E.</div></div> <div class="name"><div class="surname">Hyde</div> <div class="given-names">K.D.</div></div></div> <div class="year">2011</div>, <div class="article-title"><em>Cochliobolus</em>: an overview and current status of species</div>, <div class="source">Fungal Divers</div> , <div class="volume">51</div>, <div class="fpage">3</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cochliobolus%3A%20an%20overview%20and%20current%20status%20of%20species&amp;author=D.S.%20Manamgoda&amp;author=L.%20Cai&amp;author=A.H.%20Bahkali&amp;author=E.%20Chukeatirote&amp;author=K.D.%20Hyde&amp;publication_year=2011&amp;journal=Fungal%20Divers&amp;volume=51&amp;pages=3-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s13225-011-0139-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs13225-011-0139-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs13225-011-0139-4"> </span></div> </div></div></div></div></div><div content-id="DSV007C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Tatum</div> <div class="given-names">L.A.</div></div></div> <div class="year">1971</div>, <div class="article-title">The southern corn leaf blight epidemic</div>, <div class="source">Science</div> , <div class="volume">171</div>, <div class="fpage">1113</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20southern%20corn%20leaf%20blight%20epidemic&amp;author=L.A.%20Tatum&amp;publication_year=1971&amp;journal=Science&amp;volume=171&amp;pages=1113-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.171.3976.1113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.171.3976.1113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.171.3976.1113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17777595" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Scheffer</div> <div class="given-names">R.P.</div></div></div> <div class="year">1997</div>, <div class="source">The nature of disease in plants</div> . <div class="publisher-name">Cambridge University Press</div>: <div class="publisher-loc">New York</div>.</div></div></div></div><div content-id="DSV007C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Condon</div> <div class="given-names">B.J.</div></div> <div class="name"><div class="surname">Leng</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Wu</div> <div class="given-names">D.</div></div>et al.  </div> <div class="year">2013</div>, <div class="article-title">Comparative genome structure, secondary metabolite, and effector coding capacity across <em>Cochliobolus</em> pathogens</div>, <div class="source">PLoS Genet.</div> , <div class="volume">9</div>, <div class="fpage">e1003233</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genome%20structure%2C%20secondary%20metabolite%2C%20and%20effector%20coding%20capacity%20across%20Cochliobolus%20pathogens&amp;author=B.J.%20Condon&amp;author=Y.%20Leng&amp;author=D.%20Wu&amp;publication_year=2013&amp;journal=PLoS%20Genet.&amp;volume=9&amp;pages=e1003233" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1003233" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1003233" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1003233"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23357949" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Brandt</div> <div class="given-names">M.E.</div></div> <div class="name"><div class="surname">Warnock</div> <div class="given-names">D.W.</div></div></div> <div class="year">2003</div>, <div class="article-title">Epidemiology, clinical manifestations, and therapy of infections caused by dematiaceous fungi</div>, <div class="source">J. Chemother.</div> , <div class="volume">15</div><div class="issue">(Suppl. 2)</div>, <div class="fpage">36</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epidemiology%2C%20clinical%20manifestations%2C%20and%20therapy%20of%20infections%20caused%20by%20dematiaceous%20fungi&amp;author=M.E.%20Brandt&amp;author=D.W.%20Warnock&amp;publication_year=2003&amp;journal=J.%20Chemother.&amp;volume=15&amp;pages=36-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1179/joc.2003.15.Supplement-2.36" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1179%2Fjoc.2003.15.Supplement-2.36" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1179%2Fjoc.2003.15.Supplement-2.36"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14708965" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">de Hoog</div> <div class="given-names">G.S.</div></div> <div class="name"><div class="surname">Guarro</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Gené</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Figueras</div> <div class="given-names">M.J.</div></div></div> <div class="year">2005</div>, <div class="source">Atlas of clinical fungi</div> , <div class="edition">2nd edn</div>. <div class="publisher-name">Centraalbureau Voor Schimmelcultures</div>: <div class="publisher-loc">Baarn, the Netherlands</div>.</div></div></div></div><div content-id="DSV007C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">P.P.</div></div> <div class="name"><div class="surname">Cao</div> <div class="given-names">Z.Y.</div></div></div> <div class="year">2013</div>, <div class="article-title">First report of <em>Bipolaris papendorfii</em> causing corn leaf spot in China</div>, <div class="source">Plant Dis.</div> , <div class="volume">97</div>, <div class="fpage">1506</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=First%20report%20of%20Bipolaris%20papendorfii%20causing%20corn%20leaf%20spot%20in%20China&amp;author=P.P.%20Li&amp;author=Z.Y.%20Cao&amp;publication_year=2013&amp;journal=Plant%20Dis.&amp;volume=97&amp;pages=1506" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/PDIS-02-13-0203-PDN" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FPDIS-02-13-0203-PDN" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FPDIS-02-13-0203-PDN"> </span></div> </div></div></div></div></div><div content-id="DSV007C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Giridharan</div> <div class="given-names">P.</div></div> <div class="name"><div class="surname">Verekar</div> <div class="given-names">S.A.</div></div> <div class="name"><div class="surname">Gohil</div> <div class="given-names">A.R.</div></div> <div class="name"><div class="surname">Mishra</div> <div class="given-names">P.D.</div></div> <div class="name"><div class="surname">Khanna</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Deshmukh</div> <div class="given-names">S.K.</div></div></div> <div class="year">2014</div>, <div class="article-title">Antiproliferative activity of hamigerone and radicinol isolated from <em>Bipolaris papendorfii</em></div>, <div class="source">Biomed. Res. Int.</div> , <div class="volume">2014</div>, <div class="fpage">890904</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Antiproliferative%20activity%20of%20hamigerone%20and%20radicinol%20isolated%20from%20Bipolaris%20papendorfii&amp;author=P.%20Giridharan&amp;author=S.A.%20Verekar&amp;author=A.R.%20Gohil&amp;author=P.D.%20Mishra&amp;author=A.%20Khanna&amp;author=S.K.%20Deshmukh&amp;publication_year=2014&amp;journal=Biomed.%20Res.%20Int.&amp;volume=2014&amp;pages=890904" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1155/2014/890904" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1155%2F2014%2F890904" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1155%2F2014%2F890904"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25184147" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Yew</div> <div class="given-names">S.M.</div></div> <div class="name"><div class="surname">Chan</div> <div class="given-names">C.L.</div></div> <div class="name"><div class="surname">Lee</div> <div class="given-names">K.W.</div></div>et al.  </div> <div class="year">2014</div>, <div class="article-title">A five-year survey of dematiaceous fungi in a tropical hospital reveals potential opportunistic species</div>, <div class="source">PLoS ONE</div> , <div class="volume">9</div>, <div class="fpage">e104352</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20five-year%20survey%20of%20dematiaceous%20fungi%20in%20a%20tropical%20hospital%20reveals%20potential%20opportunistic%20species&amp;author=S.M.%20Yew&amp;author=C.L.%20Chan&amp;author=K.W.%20Lee&amp;publication_year=2014&amp;journal=PLoS%20ONE&amp;volume=9&amp;pages=e104352" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0104352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0104352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0104352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25098697" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Moretti</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Armougom</div> <div class="given-names">F.</div></div> <div class="name"><div class="surname">Wallace</div> <div class="given-names">I.M.</div></div> <div class="name"><div class="surname">Higgins</div> <div class="given-names">D.G.</div></div> <div class="name"><div class="surname">Jongeneel</div> <div class="given-names">C.V.</div></div> <div class="name"><div class="surname">Notredame</div> <div class="given-names">C.</div></div></div> <div class="year">2007</div>, <div class="article-title">The M-Coffee web server: a meta-method for computing multiple sequence alignments by combining alternative alignment methods</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">35</div>, <div class="fpage">W645</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20M-Coffee%20web%20server%3A%20a%20meta-method%20for%20computing%20multiple%20sequence%20alignments%20by%20combining%20alternative%20alignment%20methods&amp;author=S.%20Moretti&amp;author=F.%20Armougom&amp;author=I.M.%20Wallace&amp;author=D.G.%20Higgins&amp;author=C.V.%20Jongeneel&amp;author=C.%20Notredame&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=W645-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm333" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm333" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm333"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17526519" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Ronquist</div> <div class="given-names">F.</div></div> <div class="name"><div class="surname">Huelsenbeck</div> <div class="given-names">J.P.</div></div></div> <div class="year">2003</div>, <div class="article-title">MrBayes 3: Bayesian phylogenetic inference under mixed models</div>, <div class="source">Bioinformatics</div> , <div class="volume">19</div>, <div class="fpage">1572</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MrBayes%203%3A%20Bayesian%20phylogenetic%20inference%20under%20mixed%20models&amp;author=F.%20Ronquist&amp;author=J.P.%20Huelsenbeck&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=1572-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg180" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg180" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg180"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12912839" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Moslem</div> <div class="given-names">M.A.</div></div> <div class="name"><div class="surname">Bahkali</div> <div class="given-names">A.H.</div></div> <div class="name"><div class="surname">Abd-Elsalam</div> <div class="given-names">K.A.</div></div> <div class="name"><div class="surname">Wit</div> <div class="given-names">P.J.</div></div></div> <div class="year">2010</div>, <div class="article-title">An efficient method for DNA extraction from <em>Cladosporioid</em> fungi</div>, <div class="source">Genet. Mol. Res.</div> , <div class="volume">9</div>, <div class="fpage">2283</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20efficient%20method%20for%20DNA%20extraction%20from%20Cladosporioid%20fungi&amp;author=M.A.%20Moslem&amp;author=A.H.%20Bahkali&amp;author=K.A.%20Abd-Elsalam&amp;author=P.J.%20Wit&amp;publication_year=2010&amp;journal=Genet.%20Mol.%20Res.&amp;volume=9&amp;pages=2283-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4238/vol9-4gmr936" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4238%2Fvol9-4gmr936" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4238%2Fvol9-4gmr936"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21128208" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Gurtowski</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Yoo</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Marcus</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">McCombie</div> <div class="given-names">W.R.</div></div> <div class="name"><div class="surname">Schatz</div> <div class="given-names">M.</div></div></div> <div class="year">2014</div>, <div class="article-title">Error correction and assembly complexity of single molecule sequencing reads</div>, <div class="comment">Cold Spring Harbor Laboratory</div>, <div class="fpage">1</div>–<div class="lpage">17</div>.</div></div></div></div><div content-id="DSV007C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kelley</div> <div class="given-names">D.R.</div></div> <div class="name"><div class="surname">Schatz</div> <div class="given-names">M.C.</div></div> <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div> <div class="year">2010</div>, <div class="article-title">Quake: quality-aware detection and correction of sequencing errors</div>, <div class="source">Genome Biol.</div> , <div class="volume">11</div>, <div class="fpage">R116</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quake%3A%20quality-aware%20detection%20and%20correction%20of%20sequencing%20errors&amp;author=D.R.%20Kelley&amp;author=M.C.%20Schatz&amp;author=S.L.%20Salzberg&amp;publication_year=2010&amp;journal=Genome%20Biol.&amp;volume=11&amp;pages=R116" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2010-11-11-r116" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2010-11-11-r116" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2010-11-11-r116"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21114842" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Myers</div> <div class="given-names">E.W.</div></div> <div class="name"><div class="surname">Sutton</div> <div class="given-names">G.G.</div></div> <div class="name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div>et al.  </div> <div class="year">2000</div>, <div class="article-title">A whole-genome assembly of <em>Drosophila</em></div>, <div class="source">Science</div> , <div class="volume">287</div>, <div class="fpage">2196</div>–<div class="lpage">204</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20whole-genome%20assembly%20of%20Drosophila&amp;author=E.W.%20Myers&amp;author=G.G.%20Sutton&amp;author=A.L.%20Delcher&amp;publication_year=2000&amp;journal=Science&amp;volume=287&amp;pages=2196-204" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.287.5461.2196" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.287.5461.2196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.287.5461.2196"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10731133" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Lomsadze</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Ter-Hovhannisyan</div> <div class="given-names">V.</div></div> <div class="name"><div class="surname">Chernoff</div> <div class="given-names">Y.O.</div></div> <div class="name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div></div> <div class="year">2005</div>, <div class="article-title">Gene identification in novel eukaryotic genomes by self-training algorithm</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">33</div>, <div class="fpage">6494</div>–<div class="lpage">506</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20identification%20in%20novel%20eukaryotic%20genomes%20by%20self-training%20algorithm&amp;author=A.%20Lomsadze&amp;author=V.%20Ter-Hovhannisyan&amp;author=Y.O.%20Chernoff&amp;author=M.%20Borodovsky&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=6494-506" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki937" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki937" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki937"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16314312" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Conesa</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Gotz</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Garcia-Gomez</div> <div class="given-names">J.M.</div></div> <div class="name"><div class="surname">Terol</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Talon</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Robles</div> <div class="given-names">M.</div></div></div> <div class="year">2005</div>, <div class="article-title">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research</div>, <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">3674</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Blast2GO%3A%20a%20universal%20tool%20for%20annotation%2C%20visualization%20and%20analysis%20in%20functional%20genomics%20research&amp;author=A.%20Conesa&amp;author=S.%20Gotz&amp;author=J.M.%20Garcia-Gomez&amp;author=J.%20Terol&amp;author=M.%20Talon&amp;author=M.%20Robles&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=3674-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti610" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti610" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti610"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16081474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Tatusov</div> <div class="given-names">R.L.</div></div> <div class="name"><div class="surname">Fedorova</div> <div class="given-names">N.D.</div></div> <div class="name"><div class="surname">Jackson</div> <div class="given-names">J.D.</div></div>et al.  </div> <div class="year">2003</div>, <div class="article-title">The COG database: an updated version includes eukaryotes</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">4</div>, <div class="fpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20COG%20database%3A%20an%20updated%20version%20includes%20eukaryotes&amp;author=R.L.%20Tatusov&amp;author=N.D.%20Fedorova&amp;author=J.D.%20Jackson&amp;publication_year=2003&amp;journal=BMC%20Bioinformatics&amp;volume=4&amp;pages=41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-4-41" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-4-41" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-4-41"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12969510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Quevillon</div> <div class="given-names">E.</div></div> <div class="name"><div class="surname">Silventoinen</div> <div class="given-names">V.</div></div> <div class="name"><div class="surname">Pillai</div> <div class="given-names">S.</div></div>et al.  </div> <div class="year">2005</div>, <div class="article-title">InterProScan: protein domains identifier</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">33</div>, <div class="fpage">W116</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterProScan%3A%20protein%20domains%20identifier&amp;author=E.%20Quevillon&amp;author=V.%20Silventoinen&amp;author=S.%20Pillai&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=W116-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki442" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki442"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15980438" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Lagesen</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Hallin</div> <div class="given-names">P.</div></div> <div class="name"><div class="surname">Rodland</div> <div class="given-names">E.A.</div></div> <div class="name"><div class="surname">Staerfeldt</div> <div class="given-names">H.H.</div></div> <div class="name"><div class="surname">Rognes</div> <div class="given-names">T.</div></div> <div class="name"><div class="surname">Ussery</div> <div class="given-names">D.W.</div></div></div> <div class="year">2007</div>, <div class="article-title">RNAmmer: consistent and rapid annotation of ribosomal RNA genes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">35</div>, <div class="fpage">3100</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNAmmer%3A%20consistent%20and%20rapid%20annotation%20of%20ribosomal%20RNA%20genes&amp;author=K.%20Lagesen&amp;author=P.%20Hallin&amp;author=E.A.%20Rodland&amp;author=H.H.%20Staerfeldt&amp;author=T.%20Rognes&amp;author=D.W.%20Ussery&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=3100-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm160" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm160" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm160"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17452365" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Lowe</div> <div class="given-names">T.M.</div></div> <div class="name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div></div> <div class="year">1997</div>, <div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">25</div>, <div class="fpage">955</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence&amp;author=T.M.%20Lowe&amp;author=S.R.%20Eddy&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=955-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9023104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Yin</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Mao</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Yang</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Chen</div> <div class="given-names">X.</div></div> <div class="name"><div class="surname">Mao</div> <div class="given-names">F.</div></div> <div class="name"><div class="surname">Xu</div> <div class="given-names">Y.</div></div></div> <div class="year">2012</div>, <div class="article-title">dbCAN: a web resource for automated carbohydrate-active enzyme annotation</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">40</div>, <div class="fpage">W445</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=dbCAN%3A%20a%20web%20resource%20for%20automated%20carbohydrate-active%20enzyme%20annotation&amp;author=Y.%20Yin&amp;author=X.%20Mao&amp;author=J.%20Yang&amp;author=X.%20Chen&amp;author=F.%20Mao&amp;author=Y.%20Xu&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=W445-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks479" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks479" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks479"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22645317" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Petersen</div> <div class="given-names">T.N.</div></div> <div class="name"><div class="surname">Brunak</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">von Heijne</div> <div class="given-names">G.</div></div> <div class="name"><div class="surname">Nielsen</div> <div class="given-names">H.</div></div></div> <div class="year">2011</div>, <div class="article-title">SignalP 4.0: discriminating signal peptides from transmembrane regions</div>, <div class="source">Nat. Methods</div> , <div class="volume">8</div>, <div class="fpage">785</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SignalP%204.0%3A%20discriminating%20signal%20peptides%20from%20transmembrane%20regions&amp;author=T.N.%20Petersen&amp;author=S.%20Brunak&amp;author=G.%20von%20Heijne&amp;author=H.%20Nielsen&amp;publication_year=2011&amp;journal=Nat.%20Methods&amp;volume=8&amp;pages=785-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1701" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1701" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1701"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21959131" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Krogh</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Larsson</div> <div class="given-names">B.</div></div> <div class="name"><div class="surname">von Heijne</div> <div class="given-names">G.</div></div> <div class="name"><div class="surname">Sonnhammer</div> <div class="given-names">E.L.</div></div></div> <div class="year">2001</div>, <div class="article-title">Predicting transmembrane protein topology with a hidden Markov model: application to complete genomes</div>, <div class="source">J. Mol. Biol.</div> , <div class="volume">305</div>, <div class="fpage">567</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20transmembrane%20protein%20topology%20with%20a%20hidden%20Markov%20model%3A%20application%20to%20complete%20genomes&amp;author=A.%20Krogh&amp;author=B.%20Larsson&amp;author=G.%20von%20Heijne&amp;author=E.L.%20Sonnhammer&amp;publication_year=2001&amp;journal=J.%20Mol.%20Biol.&amp;volume=305&amp;pages=567-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.2000.4315" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.2000.4315" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.2000.4315"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11152613" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Khaldi</div> <div class="given-names">N.</div></div> <div class="name"><div class="surname">Seifuddin</div> <div class="given-names">F.T.</div></div> <div class="name"><div class="surname">Turner</div> <div class="given-names">G.</div></div>et al.  </div> <div class="year">2010</div>, <div class="article-title">SMURF: Genomic mapping of fungal secondary metabolite clusters</div>, <div class="source">Fungal Genet. Biol.</div> , <div class="volume">47</div>, <div class="fpage">736</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SMURF%3A%20Genomic%20mapping%20of%20fungal%20secondary%20metabolite%20clusters&amp;author=N.%20Khaldi&amp;author=F.T.%20Seifuddin&amp;author=G.%20Turner&amp;publication_year=2010&amp;journal=Fungal%20Genet.%20Biol.&amp;volume=47&amp;pages=736-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fgb.2010.06.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fgb.2010.06.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fgb.2010.06.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20554054" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Ohm</div> <div class="given-names">R.A.</div></div> <div class="name"><div class="surname">Feau</div> <div class="given-names">N.</div></div> <div class="name"><div class="surname">Henrissat</div> <div class="given-names">B.</div></div>et al.  </div> <div class="year">2012</div>, <div class="article-title">Diverse lifestyles and strategies of plant pathogenesis encoded in the genomes of eighteen Dothideomycetes fungi</div>, <div class="source">PLoS Pathog.</div> , <div class="volume">8</div>, <div class="fpage">e1003037</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diverse%20lifestyles%20and%20strategies%20of%20plant%20pathogenesis%20encoded%20in%20the%20genomes%20of%20eighteen%20Dothideomycetes%20fungi&amp;author=R.A.%20Ohm&amp;author=N.%20Feau&amp;author=B.%20Henrissat&amp;publication_year=2012&amp;journal=PLoS%20Pathog.&amp;volume=8&amp;pages=e1003037" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.ppat.1003037" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.ppat.1003037" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.ppat.1003037"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23236275" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Dean</div> <div class="given-names">R.A.</div></div> <div class="name"><div class="surname">Ann</div> <div class="given-names">L.P.</div></div> <div class="name"><div class="surname">Chittaranjan</div> <div class="given-names">K.</div></div></div> <div class="year">2014</div>, <div class="source">Genomics of plant-associated fungi: monocot pathogens</div> . <div class="publisher-name">Springer</div>: <div class="publisher-loc">Berlin, Heidelberg</div>, pp. <div class="fpage">41</div>–<div class="lpage">67</div>.</div></div></div></div><div content-id="DSV007C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Buchan</div> <div class="given-names">D.W.</div></div> <div class="name"><div class="surname">Ward</div> <div class="given-names">S.M.</div></div> <div class="name"><div class="surname">Lobley</div> <div class="given-names">A.E.</div></div> <div class="name"><div class="surname">Nugent</div> <div class="given-names">T.C.</div></div> <div class="name"><div class="surname">Bryson</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Jones</div> <div class="given-names">D.T.</div></div></div> <div class="year">2010</div>, <div class="article-title">Protein annotation and modelling servers at University College London</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">38</div>, <div class="fpage">W563</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20annotation%20and%20modelling%20servers%20at%20University%20College%20London&amp;author=D.W.%20Buchan&amp;author=S.M.%20Ward&amp;author=A.E.%20Lobley&amp;author=T.C.%20Nugent&amp;author=K.%20Bryson&amp;author=D.T.%20Jones&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=W563-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq427" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq427" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq427"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20507913" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Biasini</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Bienert</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Waterhouse</div> <div class="given-names">A.</div></div>et al.  </div> <div class="year">2014</div>, <div class="article-title">SWISS-MODEL: modelling protein tertiary and quaternary structure using evolutionary information</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">42</div>, <div class="fpage">W252</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SWISS-MODEL%3A%20modelling%20protein%20tertiary%20and%20quaternary%20structure%20using%20evolutionary%20information&amp;author=M.%20Biasini&amp;author=S.%20Bienert&amp;author=A.%20Waterhouse&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res.&amp;volume=42&amp;pages=W252-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku340" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku340" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku340"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24782522" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">L.</div></div> <div class="name"><div class="surname">Stoeckert</div> <div class="given-names">C.J.,</div>Jr</div> <div class="name"><div class="surname">Roos</div> <div class="given-names">D.S.</div></div></div> <div class="year">2003</div>, <div class="article-title">OrthoMCL: identification of ortholog groups for eukaryotic genomes</div>, <div class="source">Genome Res.</div> , <div class="volume">13</div>, <div class="fpage">2178</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=OrthoMCL%3A%20identification%20of%20ortholog%20groups%20for%20eukaryotic%20genomes&amp;author=L.%20Li&amp;author=C.J.%2C%20Stoeckert&amp;author=D.S.%20Roos&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=2178-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1224503" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1224503" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1224503"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12952885" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Larkin</div> <div class="given-names">M.A.</div></div> <div class="name"><div class="surname">Blackshields</div> <div class="given-names">G.</div></div> <div class="name"><div class="surname">Brown</div> <div class="given-names">N.P.</div></div>et al.  </div> <div class="year">2007</div>, <div class="article-title">Clustal W and Clustal X version 2.0</div>, <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">2947</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Clustal%20W%20and%20Clustal%20X%20version%202.0&amp;author=M.A.%20Larkin&amp;author=G.%20Blackshields&amp;author=N.P.%20Brown&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2947-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm404" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17846036" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Sanchez-Vallet</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Saleem-Batcha</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Kombrink</div> <div class="given-names">A.</div></div>et al.  </div> <div class="year">2013</div>, <div class="article-title">Fungal effector Ecp6 outcompetes host immune receptor for chitin binding through intrachain LysM dimerization</div>, <div class="source">Elife</div> , <div class="volume">2</div>, <div class="fpage">e00790</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fungal%20effector%20Ecp6%20outcompetes%20host%20immune%20receptor%20for%20chitin%20binding%20through%20intrachain%20LysM%20dimerization&amp;author=A.%20Sanchez-Vallet&amp;author=R.%20Saleem-Batcha&amp;author=A.%20Kombrink&amp;publication_year=2013&amp;journal=Elife&amp;volume=2&amp;pages=e00790" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7554/eLife.00790" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7554%2FeLife.00790" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7554%2FeLife.00790"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23840930" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Guex</div> <div class="given-names">N.</div></div> <div class="name"><div class="surname">Peitsch</div> <div class="given-names">M.C.</div></div></div> <div class="year">1997</div>, <div class="article-title">SWISS-MODEL and the Swiss-PdbViewer: an environment for comparative protein modeling</div>, <div class="source">Electrophoresis</div> , <div class="volume">18</div>, <div class="fpage">2714</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SWISS-MODEL%20and%20the%20Swiss-PdbViewer%3A%20an%20environment%20for%20comparative%20protein%20modeling&amp;author=N.%20Guex&amp;author=M.C.%20Peitsch&amp;publication_year=1997&amp;journal=Electrophoresis&amp;volume=18&amp;pages=2714-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1522-2683" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1522-2683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1522-2683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9504803" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kuan</div> <div class="given-names">C.S.</div></div> <div class="name"><div class="surname">Tien</div> <div class="given-names">C.F.</div></div> <div class="name"><div class="surname">See Too</div> <div class="given-names">W.C.</div></div> <div class="name"><div class="surname">Few</div> <div class="given-names">L.L.</div></div></div> <div class="year">2012</div>, <div class="article-title">Sequence analysis and homology modeling of glyceraldehyde-3-phosphate dehydrogenase of an antarctic psychrophilic <em>Pseudomonas</em> sp</div>, <div class="source">Afr. J. Microbiol. Res.</div> , <div class="volume">6</div>, <div class="fpage">881</div>–<div class="lpage">91</div>.</div></div></div></div><div content-id="DSV007C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Rutherford</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Parkhill</div> <div class="given-names">J.</div></div> <div class="name"><div class="surname">Crook</div> <div class="given-names">J.</div></div>et al.  </div> <div class="year">2000</div>, <div class="article-title">Artemis: sequence visualization and annotation</div>, <div class="source">Bioinformatics</div> , <div class="volume">16</div>, <div class="fpage">944</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Artemis%3A%20sequence%20visualization%20and%20annotation&amp;author=K.%20Rutherford&amp;author=J.%20Parkhill&amp;author=J.%20Crook&amp;publication_year=2000&amp;journal=Bioinformatics&amp;volume=16&amp;pages=944-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/16.10.944" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F16.10.944" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F16.10.944"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11120685" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Santana</div> <div class="given-names">M.F.</div></div> <div class="name"><div class="surname">Silva</div> <div class="given-names">J.C.</div></div> <div class="name"><div class="surname">Mizubuti</div> <div class="given-names">E.S.</div></div>et al.  </div> <div class="year">2014</div>, <div class="article-title">Characterization and potential evolutionary impact of transposable elements in the genome of Cochliobolus heterostrophus</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">536</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20and%20potential%20evolutionary%20impact%20of%20transposable%20elements%20in%20the%20genome%20of%20Cochliobolus%20heterostrophus&amp;author=M.F.%20Santana&amp;author=J.C.%20Silva&amp;author=E.S.%20Mizubuti&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=536" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-536" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-536" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-536"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24973942" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Novikova</div> <div class="given-names">O.</div></div> <div class="name"><div class="surname">Fet</div> <div class="given-names">V.</div></div> <div class="name"><div class="surname">Blinov</div> <div class="given-names">A.</div></div></div> <div class="year">2009</div>, <div class="article-title">Non-LTR retrotransposons in fungi</div>, <div class="source">Funct. Integr. Genomics</div> , <div class="volume">9</div>, <div class="fpage">27</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Non-LTR%20retrotransposons%20in%20fungi&amp;author=O.%20Novikova&amp;author=V.%20Fet&amp;author=A.%20Blinov&amp;publication_year=2009&amp;journal=Funct.%20Integr.%20Genomics&amp;volume=9&amp;pages=27-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10142-008-0093-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10142-008-0093-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10142-008-0093-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18677522" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Muszewska</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Hoffman-Sommer</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Grynberg</div> <div class="given-names">M.</div></div></div> <div class="year">2011</div>, <div class="article-title">LTR retrotransposons in fungi</div>, <div class="source">PLoS ONE</div> , <div class="volume">6</div>, <div class="fpage">e29425</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LTR%20retrotransposons%20in%20fungi&amp;author=A.%20Muszewska&amp;author=M.%20Hoffman-Sommer&amp;author=M.%20Grynberg&amp;publication_year=2011&amp;journal=PLoS%20ONE&amp;volume=6&amp;pages=e29425" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0029425" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0029425" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0029425"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22242120" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Brillet</div> <div class="given-names">B.</div></div> <div class="name"><div class="surname">Bigot</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Auge-Gouillou</div> <div class="given-names">C.</div></div></div> <div class="year">2007</div>, <div class="article-title">Assembly of the Tc1 and mariner transposition initiation complexes depends on the origins of their transposase DNA binding domains</div>, <div class="source">Genetica</div> , <div class="volume">130</div>, <div class="fpage">105</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assembly%20of%20the%20Tc1%20and%20mariner%20transposition%20initiation%20complexes%20depends%20on%20the%20origins%20of%20their%20transposase%20DNA%20binding%20domains&amp;author=B.%20Brillet&amp;author=Y.%20Bigot&amp;author=C.%20Auge-Gouillou&amp;publication_year=2007&amp;journal=Genetica&amp;volume=130&amp;pages=105-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10709-006-0025-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10709-006-0025-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10709-006-0025-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16912840" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Yuan</div> <div class="given-names">Y.W.</div></div> <div class="name"><div class="surname">Wessler</div> <div class="given-names">S.R.</div></div></div> <div class="year">2011</div>, <div class="article-title">The catalytic domain of all eukaryotic cut-and-paste transposase superfamilies</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">108</div>, <div class="fpage">7884</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20catalytic%20domain%20of%20all%20eukaryotic%20cut-and-paste%20transposase%20superfamilies&amp;author=Y.W.%20Yuan&amp;author=S.R.%20Wessler&amp;publication_year=2011&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=7884-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1104208108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1104208108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1104208108"> </span></div> </div></div></div></div></div><div content-id="DSV007C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Leach</div> <div class="given-names">M.D.</div></div> <div class="name"><div class="surname">Brown</div> <div class="given-names">A.J.</div></div></div> <div class="year">2012</div>, <div class="article-title">Posttranslational modifications of proteins in the pathobiology of medically relevant fungi</div>, <div class="source">Eukaryot. Cell</div> , <div class="volume">11</div>, <div class="fpage">98</div>–<div class="lpage">108</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Posttranslational%20modifications%20of%20proteins%20in%20the%20pathobiology%20of%20medically%20relevant%20fungi&amp;author=M.D.%20Leach&amp;author=A.J.%20Brown&amp;publication_year=2012&amp;journal=Eukaryot.%20Cell&amp;volume=11&amp;pages=98-108" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/EC.05238-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FEC.05238-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FEC.05238-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22158711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Khan</div> <div class="given-names">I.A.</div></div> <div class="name"><div class="surname">Lu</div> <div class="given-names">J.P.</div></div> <div class="name"><div class="surname">Liu</div> <div class="given-names">X.H.</div></div> <div class="name"><div class="surname">Rehman</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Lin</div> <div class="given-names">F.C.</div></div></div> <div class="year">2012</div>, <div class="article-title">Multifunction of autophagy-related genes in filamentous fungi</div>, <div class="source">Microbiol. Res.</div> , <div class="volume">167</div>, <div class="fpage">339</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multifunction%20of%20autophagy-related%20genes%20in%20filamentous%20fungi&amp;author=I.A.%20Khan&amp;author=J.P.%20Lu&amp;author=X.H.%20Liu&amp;author=A.%20Rehman&amp;author=F.C.%20Lin&amp;publication_year=2012&amp;journal=Microbiol.%20Res.&amp;volume=167&amp;pages=339-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.micres.2012.01.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.micres.2012.01.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.micres.2012.01.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22554685" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Zhao</div> <div class="given-names">Z.</div></div> <div class="name"><div class="surname">Liu</div> <div class="given-names">H.</div></div> <div class="name"><div class="surname">Wang</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Xu</div> <div class="given-names">J.R.</div></div></div> <div class="year">2014</div>, <div class="article-title">Correction: comparative analysis of fungal genomes reveals different plant cell wall degrading capacity in fungi</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correction%3A%20comparative%20analysis%20of%20fungal%20genomes%20reveals%20different%20plant%20cell%20wall%20degrading%20capacity%20in%20fungi&amp;author=Z.%20Zhao&amp;author=H.%20Liu&amp;author=C.%20Wang&amp;author=J.R.%20Xu&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24422981" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Blackman</div> <div class="given-names">L.M.</div></div> <div class="name"><div class="surname">Cullerne</div> <div class="given-names">D.P.</div></div> <div class="name"><div class="surname">Hardham</div> <div class="given-names">A.R.</div></div></div> <div class="year">2014</div>, <div class="article-title">Bioinformatic characterisation of genes encoding cell wall degrading enzymes in the Phytophthora parasitica genome</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">785</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bioinformatic%20characterisation%20of%20genes%20encoding%20cell%20wall%20degrading%20enzymes%20in%20the%20Phytophthora%20parasitica%20genome&amp;author=L.M.%20Blackman&amp;author=D.P.%20Cullerne&amp;author=A.R.%20Hardham&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=785" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-785" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-785" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-785"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25214042" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Mitreva-Dautova</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Roze</div> <div class="given-names">E.</div></div> <div class="name"><div class="surname">Overmars</div> <div class="given-names">H.</div></div>et al.  </div> <div class="year">2006</div>, <div class="article-title">A symbiont-independent endo-1,4-beta-xylanase from the plant-parasitic nematode Meloidogyne incognita</div>, <div class="source">Mol. Plant Microb. Interact.</div> , <div class="volume">19</div>, <div class="fpage">521</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20symbiont-independent%20endo-1%2C4-beta-xylanase%20from%20the%20plant-parasitic%20nematode%20Meloidogyne%20incognita&amp;author=M.%20Mitreva-Dautova&amp;author=E.%20Roze&amp;author=H.%20Overmars&amp;publication_year=2006&amp;journal=Mol.%20Plant%20Microb.%20Interact.&amp;volume=19&amp;pages=521-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-19-0521" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-19-0521" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-19-0521"> </span></div> </div></div></div></div></div><div content-id="DSV007C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Ahmed</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Riaz</div> <div class="given-names">S.</div></div> <div class="name"><div class="surname">Jamil</div> <div class="given-names">A.</div></div></div> <div class="year">2009</div>, <div class="article-title">Molecular cloning of fungal xylanases: an overview</div>, <div class="source">Appl. Microbiol. Biotechnol.</div> , <div class="volume">84</div>, <div class="fpage">19</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20cloning%20of%20fungal%20xylanases%3A%20an%20overview&amp;author=S.%20Ahmed&amp;author=S.%20Riaz&amp;author=A.%20Jamil&amp;publication_year=2009&amp;journal=Appl.%20Microbiol.%20Biotechnol.&amp;volume=84&amp;pages=19-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-009-2079-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-009-2079-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-009-2079-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19568746" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Levasseur</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Saloheimo</div> <div class="given-names">M.</div></div> <div class="name"><div class="surname">Navarro</div> <div class="given-names">D.</div></div>et al.  </div> <div class="year">2010</div>, <div class="article-title">Exploring laccase-like multicopper oxidase genes from the ascomycete Trichoderma reesei: a functional, phylogenetic and evolutionary study</div>, <div class="source">BMC Biochem.</div> , <div class="volume">11</div>, <div class="fpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploring%20laccase-like%20multicopper%20oxidase%20genes%20from%20the%20ascomycete%20Trichoderma%20reesei%3A%20a%20functional%2C%20phylogenetic%20and%20evolutionary%20study&amp;author=A.%20Levasseur&amp;author=M.%20Saloheimo&amp;author=D.%20Navarro&amp;publication_year=2010&amp;journal=BMC%20Biochem.&amp;volume=11&amp;pages=32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2091-11-32" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2091-11-32" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2091-11-32"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20735824" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Piontek</div> <div class="given-names">K.</div></div> <div class="name"><div class="surname">Smith</div> <div class="given-names">A.T.</div></div> <div class="name"><div class="surname">Blodig</div> <div class="given-names">W.</div></div></div> <div class="year">2001</div>, <div class="article-title">Lignin peroxidase structure and function</div>, <div class="source">Biochem. Soc. Trans.</div> , <div class="volume">29</div>, <div class="fpage">111</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lignin%20peroxidase%20structure%20and%20function&amp;author=K.%20Piontek&amp;author=A.T.%20Smith&amp;author=W.%20Blodig&amp;publication_year=2001&amp;journal=Biochem.%20Soc.%20Trans.&amp;volume=29&amp;pages=111-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/bst0290111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2Fbst0290111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2Fbst0290111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11356137" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Kombrink</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Thomma</div> <div class="given-names">B.P.</div></div></div> <div class="year">2013</div>, <div class="article-title">LysM effectors: secreted proteins supporting fungal life</div>, <div class="source">PLoS Pathog.</div> , <div class="volume">9</div>, <div class="fpage">e1003769</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LysM%20effectors%3A%20secreted%20proteins%20supporting%20fungal%20life&amp;author=A.%20Kombrink&amp;author=B.P.%20Thomma&amp;publication_year=2013&amp;journal=PLoS%20Pathog.&amp;volume=9&amp;pages=e1003769" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.ppat.1003769" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.ppat.1003769" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.ppat.1003769"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24348247" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Bolton</div> <div class="given-names">M.D.</div></div> <div class="name"><div class="surname">van Esse</div> <div class="given-names">H.P.</div></div> <div class="name"><div class="surname">Vossen</div> <div class="given-names">J.H.</div></div>et al.  </div> <div class="year">2008</div>, <div class="article-title">The novel <em>Cladosporium fulvum</em> lysin motif effector Ecp6 is a virulence factor with orthologues in other fungal species</div>, <div class="source">Mol. Microbiol.</div> , <div class="volume">69</div>, <div class="fpage">119</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20novel%20Cladosporium%20fulvum%20lysin%20motif%20effector%20Ecp6%20is%20a%20virulence%20factor%20with%20orthologues%20in%20other%20fungal%20species&amp;author=M.D.%20Bolton&amp;author=H.P.%20van%20Esse&amp;author=J.H.%20Vossen&amp;publication_year=2008&amp;journal=Mol.%20Microbiol.&amp;volume=69&amp;pages=119-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.2008.69.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2008.69.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.2008.69.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18452583" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">de Jonge</div> <div class="given-names">R.</div></div> <div class="name"><div class="surname">Thomma</div> <div class="given-names">B.P.</div></div></div> <div class="year">2009</div>, <div class="article-title">Fungal LysM effectors: extinguishers of host immunity?</div> <div class="source">Trends Microbiol.</div> , <div class="volume">17</div>, <div class="fpage">151</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fungal%20LysM%20effectors%3A%20extinguishers%20of%20host%20immunity%3F&amp;author=R.%20de%20Jonge&amp;author=B.P.%20Thomma&amp;publication_year=2009&amp;journal=Trends%20Microbiol.&amp;volume=17&amp;pages=151-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tim.2009.01.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tim.2009.01.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tim.2009.01.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19299132" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Turgeon</div> <div class="given-names">B.G.</div></div></div> <div class="year">1998</div>, <div class="article-title">Application of mating type gene technology to problems in fungal biology</div>, <div class="source">Annu. Rev. Phytopathol.</div> , <div class="volume">36</div>, <div class="fpage">115</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Application%20of%20mating%20type%20gene%20technology%20to%20problems%20in%20fungal%20biology&amp;author=B.G.%20Turgeon&amp;publication_year=1998&amp;journal=Annu.%20Rev.%20Phytopathol.&amp;volume=36&amp;pages=115-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.phyto.36.1.115" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.phyto.36.1.115" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.phyto.36.1.115"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15012495" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Keller</div> <div class="given-names">N.P.</div></div> <div class="name"><div class="surname">Turner</div> <div class="given-names">G.</div></div> <div class="name"><div class="surname">Bennett</div> <div class="given-names">J.W.</div></div></div> <div class="year">2005</div>, <div class="article-title">Fungal secondary metabolism - from biochemistry to genomics</div>, <div class="source">Nat. Rev. Microbiol.</div> , <div class="volume">3</div>, <div class="fpage">937</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fungal%20secondary%20metabolism%20-%20from%20biochemistry%20to%20genomics&amp;author=N.P.%20Keller&amp;author=G.%20Turner&amp;author=J.W.%20Bennett&amp;publication_year=2005&amp;journal=Nat.%20Rev.%20Microbiol.&amp;volume=3&amp;pages=937-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro1286" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro1286" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro1286"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16322742" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Metzger</div> <div class="given-names">U.</div></div> <div class="name"><div class="surname">Schall</div> <div class="given-names">C.</div></div> <div class="name"><div class="surname">Zocher</div> <div class="given-names">G.</div></div>et al.  </div> <div class="year">2009</div>, <div class="article-title">The structure of dimethylallyl tryptophan synthase reveals a common architecture of aromatic prenyltransferases in fungi and bacteria</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">106</div>, <div class="fpage">14309</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20structure%20of%20dimethylallyl%20tryptophan%20synthase%20reveals%20a%20common%20architecture%20of%20aromatic%20prenyltransferases%20in%20fungi%20and%20bacteria&amp;author=U.%20Metzger&amp;author=C.%20Schall&amp;author=G.%20Zocher&amp;publication_year=2009&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=14309-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0904897106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0904897106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0904897106"> </span></div> </div></div></div></div></div><div content-id="DSV007C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Schardl</div> <div class="given-names">C.L.</div></div> <div class="name"><div class="surname">Panaccione</div> <div class="given-names">D.G.</div></div> <div class="name"><div class="surname">Tudzynski</div> <div class="given-names">P.</div></div></div> <div class="year">2006</div>, <div class="article-title">Ergot alkaloids--biology and molecular biology</div>, <div class="source">Alkaloids Chem. Biol.</div> , <div class="volume">63</div>, <div class="fpage">45</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ergot%20alkaloids--biology%20and%20molecular%20biology&amp;author=C.L.%20Schardl&amp;author=D.G.%20Panaccione&amp;author=P.%20Tudzynski&amp;publication_year=2006&amp;journal=Alkaloids%20Chem.%20Biol.&amp;volume=63&amp;pages=45-86" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17133714" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Turgeon</div> <div class="given-names">B.G.</div></div> <div class="name"><div class="surname">Baker</div> <div class="given-names">S.E.</div></div></div> <div class="year">2007</div>, <div class="article-title">Genetic and genomic dissection of the <em>Cochliobolus heterostrophus</em> Tox1 locus controlling biosynthesis of the polyketide virulence factor T-toxin</div>, <div class="source">Adv. Genet.</div> , <div class="volume">57</div>, <div class="fpage">219</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20and%20genomic%20dissection%20of%20the%20Cochliobolus%20heterostrophus%20Tox1%20locus%20controlling%20biosynthesis%20of%20the%20polyketide%20virulence%20factor%20T-toxin&amp;author=B.G.%20Turgeon&amp;author=S.E.%20Baker&amp;publication_year=2007&amp;journal=Adv.%20Genet.&amp;volume=57&amp;pages=219-61" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17352906" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Dadachova</div> <div class="given-names">E.</div></div> <div class="name"><div class="surname">Bryan</div> <div class="given-names">R.A.</div></div> <div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>et al.  </div> <div class="year">2007</div>, <div class="article-title">Ionizing radiation changes the electronic properties of melanin and enhances the growth of melanized fungi</div>, <div class="source">PLoS ONE</div> , <div class="volume">2</div>, <div class="fpage">e457</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ionizing%20radiation%20changes%20the%20electronic%20properties%20of%20melanin%20and%20enhances%20the%20growth%20of%20melanized%20fungi&amp;author=E.%20Dadachova&amp;author=R.A.%20Bryan&amp;author=X.%20Huang&amp;publication_year=2007&amp;journal=PLoS%20ONE&amp;volume=2&amp;pages=e457" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0000457" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0000457" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0000457"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17520016" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Eliahu</div> <div class="given-names">N.</div></div> <div class="name"><div class="surname">Igbaria</div> <div class="given-names">A.</div></div> <div class="name"><div class="surname">Rose</div> <div class="given-names">M.S.</div></div> <div class="name"><div class="surname">Horwitz</div> <div class="given-names">B.A.</div></div> <div class="name"><div class="surname">Lev</div> <div class="given-names">S.</div></div></div> <div class="year">2007</div>, <div class="article-title">Melanin biosynthesis in the maize pathogen Cochliobolus heterostrophus depends on two mitogen-activated protein kinases, Chk1 and Mps1, and the transcription factor Cmr1</div>, <div class="source">Eukaryot. Cell</div> , <div class="volume">6</div>, <div class="fpage">421</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Melanin%20biosynthesis%20in%20the%20maize%20pathogen%20Cochliobolus%20heterostrophus%20depends%20on%20two%20mitogen-activated%20protein%20kinases%2C%20Chk1%20and%20Mps1%2C%20and%20the%20transcription%20factor%20Cmr1&amp;author=N.%20Eliahu&amp;author=A.%20Igbaria&amp;author=M.S.%20Rose&amp;author=B.A.%20Horwitz&amp;author=S.%20Lev&amp;publication_year=2007&amp;journal=Eukaryot.%20Cell&amp;volume=6&amp;pages=421-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/EC.00264-06" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FEC.00264-06" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FEC.00264-06"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17237364" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Tsuji</div> <div class="given-names">G.</div></div> <div class="name"><div class="surname">Kenmochi</div> <div class="given-names">Y.</div></div> <div class="name"><div class="surname">Takano</div> <div class="given-names">Y.</div></div>et al.  </div> <div class="year">2000</div>, <div class="article-title">Novel fungal transcriptional activators, Cmr1p of <em>Colletotrichum lagenarium</em> and pig1p of <em>Magnaporthe grisea</em>, contain Cys2His2 zinc finger and Zn(II)2Cys6 binuclear cluster DNA-binding motifs and regulate transcription of melanin biosynthesis genes in a developmentally specific manner</div>, <div class="source">Mol. Microbiol.</div> , <div class="volume">38</div>, <div class="fpage">940</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20fungal%20transcriptional%20activators%2C%20Cmr1p%20of%20Colletotrichum%20lagenarium%20and%20pig1p%20of%20Magnaporthe%20grisea%2C%20contain%20Cys2His2%20zinc%20finger%20and%20Zn(II)2Cys6%20binuclear%20cluster%20DNA-binding%20motifs%20and%20regulate%20transcription%20of%20melanin%20biosynthesis%20genes%20in%20a%20developmentally%20specific%20manner&amp;author=G.%20Tsuji&amp;author=Y.%20Kenmochi&amp;author=Y.%20Takano&amp;publication_year=2000&amp;journal=Mol.%20Microbiol.&amp;volume=38&amp;pages=940-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2000.02181.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2000.02181.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2000.02181.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11123670" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Newman</div> <div class="given-names">D.J.</div></div> <div class="name"><div class="surname">Cragg</div> <div class="given-names">G.M.</div></div> <div class="name"><div class="surname">Snader</div> <div class="given-names">K.M.</div></div></div> <div class="year">2000</div>, <div class="article-title">The influence of natural products upon drug discovery</div>, <div class="source">Nat. Prod. Rep.</div> , <div class="volume">17</div>, <div class="fpage">215</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20influence%20of%20natural%20products%20upon%20drug%20discovery&amp;author=D.J.%20Newman&amp;author=G.M.%20Cragg&amp;author=K.M.%20Snader&amp;publication_year=2000&amp;journal=Nat.%20Prod.%20Rep.&amp;volume=17&amp;pages=215-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1039/a902202c" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1039%2Fa902202c" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1039%2Fa902202c"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10888010" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Gilbert</div> <div class="given-names">R.M.</div></div></div> <div class="year">1984</div>, <div class="article-title">Caffeine consumption</div>, <div class="source">Prog. Clin. Biol. Res.</div> , <div class="volume">158</div>, <div class="fpage">185</div>–<div class="lpage">213</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Caffeine%20consumption&amp;author=R.M.%20Gilbert&amp;publication_year=1984&amp;journal=Prog.%20Clin.%20Biol.%20Res.&amp;volume=158&amp;pages=185-213" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6396645" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Dash</div> <div class="given-names">S.S.</div></div> <div class="name"><div class="surname">Gummadi</div> <div class="given-names">S.N.</div></div></div> <div class="year">2006</div>, <div class="article-title">Catabolic pathways and biotechnological applications of microbial caffeine degradation</div>, <div class="source">Biotechnol. Lett.</div> , <div class="volume">28</div>, <div class="fpage">1993</div>–<div class="lpage">2002</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Catabolic%20pathways%20and%20biotechnological%20applications%20of%20microbial%20caffeine%20degradation&amp;author=S.S.%20Dash&amp;author=S.N.%20Gummadi&amp;publication_year=2006&amp;journal=Biotechnol.%20Lett.&amp;volume=28&amp;pages=1993-2002" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10529-006-9196-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10529-006-9196-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10529-006-9196-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17009088" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Mazzafera</div> <div class="given-names">P.</div></div></div> <div class="year">2004</div>, <div class="article-title">Catabolism of caffeine in plants and microorganisms</div>, <div class="source">Front Biosci.</div> , <div class="volume">9</div>, <div class="fpage">1348</div>–<div class="lpage">59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Catabolism%20of%20caffeine%20in%20plants%20and%20microorganisms&amp;author=P.%20Mazzafera&amp;publication_year=2004&amp;journal=Front%20Biosci.&amp;volume=9&amp;pages=1348-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2741/1339" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2741%2F1339" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2741%2F1339"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14977550" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV007C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV007C66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name"><div class="surname">Sauer</div> <div class="given-names">M.</div></div></div> <div class="year">1982</div>, <div class="article-title">Comparison of the cytochrome P-450 containing monooxigenases originating from two different yeasts</div>, <div class="source">Dev. Biochem.</div> , <div class="volume">23</div>, <div class="fpage">452</div>–<div class="lpage">7</div>.</div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/4.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=591394&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/dsv007-dsv007supp_table8.xlsx?Expires=2147483647&amp;Signature=yxTvs~sPH97KRWcUfIXIWcesoXSGEMLW-BVKbF-zxforlUEUKfZsjCNn6~ac4UBjLcGIcWA~AUXehJXPKLwVJei2pbBY5NOReXy1DFmfcPBBC-Cis947J04vRSb6FEWqCTU6bbSzJnO3TPv0FLHlYwBGUmGyMKU5cVrKaXh-wH5eG48-RPViLIP-jVj0MwTc3BD9IRoiCOHe2lsSo5AGuxdF5DlGlgQsyDIa1qd9lJhnMWXwE6UNljWKadp5YMHBzB9itQJH7Oj7eX-KgIRgn8JaNsiZskSTG9o3FmWxhrvkvk~y6aPqxCtSVnLSC2m8FQWrFkzNisIGBX-Rd2DAqA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-9</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/dsv007-dsv007supp_table7.xlsx?Expires=2147483647&amp;Signature=svllo5coxlgZZ5g8t8SjyEulw5bMCD2gTLXZbvzQQkz9ZewAVxnho7NM0HiyOymXx8dXtD94CUbye8ehtVWWpwxNAvmEgaMptHHQk8Hq0xDf19339SLN2OCWC3M0cTsf~liMo0zByjSI3uyJZxWWJsO6LZsr6P-hsDVMD5DqX2vcFs~qZP3zGD8k7mZZX14eft2p7YVtiGD9BmRaC~3bxFbs-z8xGnIZQgqLjQMZf0U4JmlhiO17TbtgbSucKMNsHtyVgGH3YeE061RZHNCUbvXdMKJBcr8gAtVDMQ3K5U-Nctg95baLhivWRKCpsXP3~LG2ncwwLQpLgbDNYLed6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/dsv007-dsv007supp_table6.xlsx?Expires=2147483647&amp;Signature=RU7230HUPhI~5mNTRS2e6PVKWxCfKSYuE84~Hnp-6sNDU~~5IzIz7k~14fyZefoPzSPvAtoRz5V1aayF9IVvMbFOH8zL4BcCqptJ7nTaTJEmsIDCn6fKTDQHucZx0I0uBy12OT~wWIkBx4F6B58mrAfQn~7gByWlEhIcl7Zgk6DpE2X3XOFu50cChGMU6rz16zb10nj7MaBvLQvx8pNjRRBUfJWb-8obcNRnJmLUdYHnj~lD9GTepEPKZKsXiZ7zm6HNSZaKA~JHQS~VI9cPU71QGXy4JmDNRt~PkpRxposqNFlfW5BLddx63K9ZI4qyrntp5KsPSrsrnuw0Ad~-zA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/dsv007-dsv007supp_table5.xlsx?Expires=2147483647&amp;Signature=OoA7sLMrMMCEv0Q3EucRfGIFPoP7nI3CawmJDu4~lFF5vNeJ1tZBh0nAf~DWUrIau~SIhkZx893dvmBOi44WhmY0EFCwtmLiVTpQWDkrisu-ZYitETGj0rmJsErfqvO1rdxCTBjCvHO4yLYv2QCOlp8zirWY42lqiPeW55ElirF-HaXONslv3H9JK0i8ayg96rALChIN4U60r48XCUxpyYd52TWwr6a2x7IVP1Om77WS~k~9~~NlTY9-MrTTuTIvgl14nkvF6GcBJp1Efy66d0BKMLJiB5lf4nWffDKgMmYsqEl7UXM6k1oY3rVJfVrASIN3F7Ki-qAKXTzbMZWSLA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/dsv007-dsv007supp_table4.xlsx?Expires=2147483647&amp;Signature=h-9hjF5J7b7Zx4iIEKWituFeYYwdLt2GRVkmVtM9FsvTY~TKwUrjUL0pBysLiZwYekQho4aXHG24Tm0kvlGzvNffjUGbb2c0j13LVqWmicO0G8EhnHJFP7JlWd2GXjojtW~yNNfV-unWb2t4eF07pzNOG1j8q4otQ9KOxsKstPVJxNmisj9XgQYTXmcoOkrYKu5CosuJsd60TEWaJFZaAgXyeGNAPaESm1e4UFRHQkzrQRkpEk-nqVFdHlXetinz9e2YTqzlo3lblL4wTDdS8JNZBuXrB34r7rGTHEx6rAOrPgsMY0mDIoGWO9~Yg-Igs36yW1lxu9F-PqohQ5D9QA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/dsv007-dsv007supp_table3.xlsx?Expires=2147483647&amp;Signature=xi6JfxSkvHVV0BIXakpWECsqnMarv3ErDGboP8uL~HE5calqlQA~XO1eaoWhrABuaTAKMTf1RqLZzObw3WJh-ZBnTox6-Ri8eOw2nd0WI3X0CUJg2H-50cZFPJq0kYW2XZI5VBHDrS4RIiVzYVdPcgw0GnRBb44K2BWoSD73PozI-nSXO4xa~qERRsLEqk9SaThOQhgDe1lPgB5IMuB~nUzyIHrAu9mVM21Ba5roafuYjlYFYnGT67qLvlVrC99c3U5mJVr~WjU6kJ2KCP87it6dkH9LmFPRofxFCsik2LLfFd~ZHhg26c0TKdNHPm8qd3EuwRDxUn6M3ZpFHT4~tw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/dsv007-dsv007supp_table2.xlsx?Expires=2147483647&amp;Signature=llDUuqOpK-ErU6L-zQwLZ-x4224eakmnu5SoSS5x8Vkjw6lwl0YiqpotOeokHWpBP9pPxQ9~7Er02Rls0yR1C2ahy24DOPCRT2RRmYk0A0fPIKqKUsfYeiHualNK5sOoq-PbWam025zggJFq5JNlXvTPvMxP4X3zPNCu8Kpz7igb-3xZLrZZOmgrBAWE20Qoz8f4e9GDXaqfkrcmUy9sfinpJUqUy9V9jebUD8nXpV5MmJ~XBfekdcKhGJKnTyrJ0nfYKFe4AcyqwmHK0MVysXIT4FcIEy~SR1FoFLck64mbu5j2q5s3VsnmbvsYtC30tY6DH2uz6di1q2TBT-g5AQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/dsv007-dsv007supp_table1.xlsx?Expires=2147483647&amp;Signature=NZ6DdRskV7zPEzL8uT-bfXzNW8IK929nrMcheabmG8zte58~lIXa8t8KZkMHDKuXC4RsEyfU7eSlH43vkIwYrFPcjG7yQQOLrvwiGUsu9~88zmHsPKtcnzP~E6b2r9qidYQQqJLrhEzbNS2WoVrzyBVbaz0ekEDnQwwsazD2iQRnSRkwV6VLdlwDGIIwxNZS95iX0tgnpIz6kQ4n7j5aOvUinzaQmJl7X8SDVKNpd93uPb9mx2zAbopAKBe41YPAgLCI~VGS4oLQE8-Jr89D-u4CBPFbJ4er~Cfa0~OR1g9nxwc54xD2j3hHjgUyINU5NyRQg9sv9p4TirVFlTPjcg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/3/10.1093/dnares/dsv007/2/dsv007-dsv007supp_fig1.png?Expires=2147483647&amp;Signature=1UlPTenfHgiEVl~ks2SsmuwyUENXnbukSMfQTM6BqNJzQ9Wwkx-kOh34EB0Xsg8BkXMw35ZWCwgn-HP5NM3YPDYKzGCT~gyENnBaj1jJoU7tJWTxZiQ3exV4N1iGT8fiiSlu-tBNpIm6Vb4qI3WYK~cCU8cxjj24xlY8RUermk-5FJqb9lpA51IIVubm5lJh~IKR7J914w4haKSaH1cmJi1qyGr~-nTdwBPxBOWrqDbgI6-S6fP9mwgABc0fM4kG0J3uO7Chsdadl5xCSob14nQ442tp00YL7x2Dln5vAk3wZ8DmeEnfJBdWnIOfF3V5AJEoSDYRxg8oQTG~tWrlKw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - png file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">471</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000356658900004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">9</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv007"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="591394"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000356658900004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f3%2f219%2f591394">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000356658900004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (9)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F3%2F219%2F591394" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/591394'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Dissecting the fungal biology of Bipolaris papendorfii: from phylogenetic to comparative genomic analysis" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv007");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f3%2f219%2f591394">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

