

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":2344,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/2/129/2901809","@type":"ScholarlyArticle","name":"Long non-coding RNA exchange during the oocyte-to-embryo transition in mice","datePublished":"2017-01-12","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2017-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw058","keywords":["lncRNA","oocyte","zygote","polyadenylation","endo-siRNA"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/2/129/2901809","author":[{"name":"Karlic, Rosa","affiliation":"1 Bioinformatics Group, Division of Molecular Biology, Department of Biology, Faculty of Science, University of Zagreb, Horvatovac 102a, Zagreb, Croatia","@type":"Person"},{"name":"Ganesh, Sravya","affiliation":"2 Institute of Molecular Genetics, Academy of Sciences of the Czech Republic, Videnska 1083, 142 20 Prague 4, Czech Republic","@type":"Person"},{"name":"Franke, Vedran","affiliation":"1 Bioinformatics Group, Division of Molecular Biology, Department of Biology, Faculty of Science, University of Zagreb, Horvatovac 102a, Zagreb, Croatia","@type":"Person"},{"name":"Svobodova, Eliska","affiliation":"2 Institute of Molecular Genetics, Academy of Sciences of the Czech Republic, Videnska 1083, 142 20 Prague 4, Czech Republic","@type":"Person"},{"name":"Urbanova, Jana","affiliation":"2 Institute of Molecular Genetics, Academy of Sciences of the Czech Republic, Videnska 1083, 142 20 Prague 4, Czech Republic","@type":"Person"},{"name":"Suzuki, Yutaka","affiliation":"3 Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Japan","@type":"Person"},{"name":"Aoki, Fugaku","affiliation":"4 Department of Integrated Biosciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Japan","@type":"Person"},{"name":"Vlahovicek, Kristian","affiliation":"1 Bioinformatics Group, Division of Molecular Biology, Department of Biology, Faculty of Science, University of Zagreb, Horvatovac 102a, Zagreb, Croatia","@type":"Person"},{"name":"Svoboda, Petr","affiliation":"2 Institute of Molecular Genetics, Academy of Sciences of the Czech Republic, Videnska 1083, 142 20 Prague 4, Czech Republic","@type":"Person"}],"description":"Abstract.  The oocyte-to-embryo transition (OET) transforms a differentiated gamete into pluripotent blastomeres. The accompanying maternal-zygotic RNA exchange","pageStart":"129","pageEnd":"141","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Long non-coding RNA exchange during the oocyte-to-embryo transition in mice","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Long non-coding RNA exchange during the oocyte-to-embryo transition in mice" />
    <meta property="og:description" content="Abstract.  The oocyte-to-embryo transition (OET) transforms a differentiated gamete into pluripotent blastomeres. The accompanying maternal-zygotic RNA exchange" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/2/129/2901809" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-01-12" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Karlic, Rosa" /><meta name="citation_author_institution" content="Bioinformatics Group, Division of Molecular Biology, Department of Biology, Faculty of Science, University of Zagreb, Horvatovac 102a, Zagreb, Croatia" /><meta name="citation_author" content="Ganesh, Sravya" /><meta name="citation_author_institution" content="Institute of Molecular Genetics, Academy of Sciences of the Czech Republic, Videnska 1083, 142 20 Prague 4, Czech Republic" /><meta name="citation_author" content="Franke, Vedran" /><meta name="citation_author_institution" content="Bioinformatics Group, Division of Molecular Biology, Department of Biology, Faculty of Science, University of Zagreb, Horvatovac 102a, Zagreb, Croatia" /><meta name="citation_author" content="Svobodova, Eliska" /><meta name="citation_author_institution" content="Institute of Molecular Genetics, Academy of Sciences of the Czech Republic, Videnska 1083, 142 20 Prague 4, Czech Republic" /><meta name="citation_author" content="Urbanova, Jana" /><meta name="citation_author_institution" content="Institute of Molecular Genetics, Academy of Sciences of the Czech Republic, Videnska 1083, 142 20 Prague 4, Czech Republic" /><meta name="citation_author" content="Suzuki, Yutaka" /><meta name="citation_author_institution" content="Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Japan" /><meta name="citation_author" content="Aoki, Fugaku" /><meta name="citation_author_institution" content="Department of Integrated Biosciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Japan" /><meta name="citation_author" content="Vlahovicek, Kristian" /><meta name="citation_author_institution" content="Bioinformatics Group, Division of Molecular Biology, Department of Biology, Faculty of Science, University of Zagreb, Horvatovac 102a, Zagreb, Croatia" /><meta name="citation_author" content="Svoboda, Petr" /><meta name="citation_author_institution" content="Institute of Molecular Genetics, Academy of Sciences of the Czech Republic, Videnska 1083, 142 20 Prague 4, Czech Republic" /><meta name="citation_title" content="Long non-coding RNA exchange during the oocyte-to-embryo transition in mice" /><meta name="citation_firstpage" content="129" /><meta name="citation_lastpage" content="141" /><meta name="citation_doi" content="10.1093/dnares/dsw058" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2017/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Sculpting the transcriptome during the oocyte-to-embryo transition in mouse;  citation_journal_title=Curr. Top. Dev. Biol;  citation_year=2015;  citation_volume=113;  citation_pages=305-49; " /><meta name="citation_reference" content="citation_title=Intergenic lncRNAs and the evolution of gene expression;  citation_journal_title=Curr. Opin. Genet. Dev;  citation_year=2014;  citation_volume=27;  citation_pages=48-53; " /><meta name="citation_reference" content="citation_title=Modular regulatory principles of large non-coding RNAs;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=482;  citation_pages=339-46; " /><meta name="citation_reference" content="citation_title=Structure and function of long noncoding RNAs in epigenetic regulation;  citation_journal_title=Nat. Struct. Mol. Biol;  citation_year=2013;  citation_volume=20;  citation_pages=300-7; " /><meta name="citation_reference" content="citation_title=lincRNAs: genomics, evolution, and mechanisms;  citation_journal_title=Cell;  citation_year=2013;  citation_volume=154;  citation_pages=26-46; " /><meta name="citation_reference" content="citation_title=Rapid turnover of long noncoding RNAs and the evolution of gene expression;  citation_journal_title=PLoS Genet;  citation_year=2012;  citation_volume=8;  citation_pages=e1002841" /><meta name="citation_reference" content="citation_title=Long non-coding RNAs in stem cell pluripotency;  citation_journal_title=Wiley Interdiscip. Rev. RNA;  citation_year=2013;  citation_volume=4;  citation_pages=121-8; " /><meta name="citation_reference" content="citation_title=Single-cell RNA-Seq profiling of human preimplantation embryos and embryonic stem cells;  citation_journal_title=Nat. Struct. Mol. Biol;  citation_year=2013;  citation_volume=20;  citation_pages=1131-9; " /><meta name="citation_reference" content="citation_title=Identification and functional analysis of long non-coding RNAs in mouse cleavage stage embryonic development based on single cell transcriptome data;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=845" /><meta name="citation_reference" content="citation_title=Gene activation-associated long noncoding RNAs function in mouse preimplantation development;  citation_journal_title=Development;  citation_year=2015;  citation_volume=142;  citation_pages=910-20; " /><meta name="citation_reference" content="citation_title=Deep sequencing and de novo assembly of the mouse oocyte transcriptome define the contribution of transcription to the DNA methylation landscape;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=209" /><meta name="citation_reference" content="citation_title=The first murine zygotic transcription is promiscuous and uncoupled from splicing and 3' processing;  citation_journal_title=EMBO J;  citation_year=2015;  citation_volume=34;  citation_pages=1523-37; " /><meta name="citation_reference" content="citation_publisher=Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY;  citation_title=Manipulating the Mouse Embryo: A Laboratory Manual;  citation_year=2003; " /><meta name="citation_reference" content="citation_title=A retrotransposon-driven dicer isoform directs endogenous small interfering RNA production in mouse oocytes;  citation_journal_title=Cell;  citation_year=2013;  citation_volume=155;  citation_pages=807-16; " /><meta name="citation_reference" content="citation_title=Multiplex genome engineering using CRISPR/Cas systems;  citation_journal_title=Science;  citation_volume=339;  citation_pages=819-23; " /><meta name="citation_reference" content="citation_title=Dynamic imaging of genomic loci in living human cells by an optimized CRISPR/Cas system;  citation_journal_title=Cell;  citation_volume=155;  citation_pages=1479-91; " /><meta name="citation_reference" content="citation_title=Inferring the choreography of parental genomes during fertilization from ultralarge-scale whole-transcriptome analysis;  citation_journal_title=Genes Dev;  citation_year=2013;  citation_volume=27;  citation_pages=2736-48; " /><meta name="citation_reference" content="citation_title=Genetic programs in human and mouse early embryos revealed by single-cell RNA sequencing;  citation_journal_title=Nature;  citation_year=2013;  citation_volume=500;  citation_pages=593-7; " /><meta name="citation_reference" content="citation_title=Ab initio reconstruction of cell type-specific transcriptomes in mouse reveals the conserved multi-exonic structure of lincRNAs;  citation_journal_title=Nat. Biotechnol;  citation_year=2010;  citation_volume=28;  citation_pages=503-10; " /><meta name="citation_reference" content="citation_title=CPAT: coding-potential assessment tool using an alignment-free logistic regression model;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=e74" /><meta name="citation_reference" content="citation_title=Using RepeatMasker to identify repetitive elements in genomic sequences;  citation_journal_title=Curr. Protoc. Bioinformatics;  citation_year=2009; " /><meta name="citation_reference" content="citation_title=Essential Role for endogenous siRNAs during meiosis in mouse oocytes;  citation_journal_title=PLoS Genet;  citation_year=2015;  citation_volume=11;  citation_pages=e1005013" /><meta name="citation_reference" content="citation_title=Pseudogene-derived small interfering RNAs regulate gene expression in mouse oocytes;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=453;  citation_pages=534-8; " /><meta name="citation_reference" content="citation_title=P-body loss is concomitant with formation of a messenger RNA storage domain in mouse oocytes;  citation_journal_title=Biol. Reprod;  citation_year=2010; " /><meta name="citation_reference" content="citation_title=Chromatin signature reveals over a thousand highly conserved large non-coding RNAs in mammals;  citation_journal_title=Nature;  citation_year=2009;  citation_volume=458;  citation_pages=223-7; " /><meta name="citation_reference" content="citation_title=Transposable elements reveal a stem cell-specific class of long noncoding RNAs;  citation_journal_title=Genome Biol;  citation_year=2012;  citation_volume=13;  citation_pages=R107" /><meta name="citation_reference" content="citation_title=Quantitative changes in total Rna, total Poly(a), and ribosomes in early mouse embryos;  citation_journal_title=Dev. Biol;  citation_year=1982;  citation_volume=89;  citation_pages=362-78; " /><meta name="citation_reference" content="citation_title=Integrative annotation of human large intergenic noncoding RNAs reveals global properties and specific subclasses;  citation_journal_title=Genes Dev;  citation_year=2011;  citation_volume=25;  citation_pages=1915-27; " /><meta name="citation_reference" content="citation_title=The GENCODE v7 catalog of human long noncoding RNAs: analysis of their gene structure, evolution, and expression;  citation_journal_title=Genome Res;  citation_year=2012;  citation_volume=22;  citation_pages=1775-89; " /><meta name="citation_reference" content="citation_title=Conserved function of lincRNAs in vertebrate embryonic development despite rapid sequence evolution;  citation_journal_title=Cell;  citation_year=2011;  citation_volume=147;  citation_pages=1537-50; " /><meta name="citation_reference" content="citation_title=Dynamic CpG island methylation landscape in oocytes and preimplantation embryos;  citation_journal_title=Nat. Genet;  citation_year=2011;  citation_volume=43;  citation_pages=811-4; " /><meta name="citation_reference" content="citation_title=The human and mouse replication-dependent histone genes;  citation_journal_title=Genomics;  citation_year=2002;  citation_volume=80;  citation_pages=487-98; " /><meta name="citation_reference" content="citation_title=Translational control in oocyte development;  citation_journal_title=Cold Spring Harb. Perspect. Biol;  citation_year=2011;  citation_volume=3;  citation_pages=a002758" /><meta name="citation_reference" content="citation_title=Synthesis and function of Mos: the control switch of vertebrate oocyte meiosis;  citation_journal_title=Bioessays;  citation_year=1997;  citation_volume=19;  citation_pages=23-8; " /><meta name="citation_reference" content="citation_title=Meiotic maturation of mouse oocytes triggers the translation and polyadenylation of dormant tissue-type plasminogen activator mRNA;  citation_journal_title=Genes Dev;  citation_year=1987;  citation_volume=1;  citation_pages=1201-11; " /><meta name="citation_reference" content="citation_title=Acquisition of meiotic competence in growing mouse oocytes is controlled at both translational and posttranslational levels;  citation_journal_title=Dev. Biol;  citation_year=1997;  citation_volume=187;  citation_pages=43-54; " /><meta name="citation_reference" content="citation_title=Recruitment of Orc6l, a dormant maternal mRNA in mouse oocytes, is essential for DNA replication in 1-cell embryos;  citation_journal_title=Dev. Biol;  citation_year=2010;  citation_volume=341;  citation_pages=205-12; " /><meta name="citation_reference" content="citation_title=Maternally recruited DCP1A and DCP2 contribute to messenger RNA degradation during oocyte maturation and genome activation in mouse;  citation_journal_title=Biol. Reprod;  citation_year=2013;  citation_volume=88;  citation_pages=11" /><meta name="citation_reference" content="citation_title=Cytoplasmic polyadenylation element (CPE)- and CPE-binding protein (CPEB)-independent mechanisms regulate early class maternal mRNA translational activation in Xenopus oocytes;  citation_journal_title=J. Biol. Chem;  citation_year=2004;  citation_volume=279;  citation_pages=17650-9; " /><meta name="citation_reference" content="citation_title=Degradation of maternal mRNA in mouse embryos: selective degradation of specific mRNAs after fertilization;  citation_journal_title=Mol. Reprod. Dev;  citation_year=2005;  citation_volume=72;  citation_pages=281-90; " /><meta name="citation_reference" content="citation_title=Diverse patterns of poly(A) tail elongation and shortening of murine maternal mRNAs from fully grown oocyte to 2-cell embryo stages;  citation_journal_title=Biochem. Biophys. Res. Commun;  citation_year=2005;  citation_volume=336;  citation_pages=1181-9; " /><meta name="citation_reference" content="citation_title=Cytoplasmic long noncoding RNAs are frequently bound to and degraded at ribosomes in human cells;  citation_journal_title=RNA;  citation_year=2016;  citation_volume=22;  citation_pages=867-82; " /><meta name="citation_reference" content="citation_title=Long non-coding RNAs as a source of new peptides;  citation_journal_title=Elife;  citation_year=2014;  citation_volume=3;  citation_pages=e03523" /><meta name="citation_reference" content="citation_title=A comparative encyclopedia of DNA elements in the mouse genome;  citation_journal_title=Nature;  citation_year=2014;  citation_volume=515;  citation_pages=355-64; " /><meta name="citation_reference" content="citation_title=MALAT1 – a paradigm for long noncoding RNA function in cancer;  citation_journal_title=J. Mol. Med. (Berl);  citation_year=2013;  citation_volume=91;  citation_pages=791-801; " /><meta name="citation_reference" content="citation_title=Loss of the abundant nuclear non-coding RNA MALAT1 is compatible with life and development;  citation_journal_title=RNA Biol;  citation_year=2012;  citation_volume=9;  citation_pages=1076-87; " /><meta name="citation_reference" content="citation_title=The lncRNA Malat1 is dispensable for mouse development but its transcription plays a cis-regulatory role in the adult;  citation_journal_title=Cell Rep;  citation_year=2012;  citation_volume=2;  citation_pages=111-23; " /><meta name="citation_reference" content="citation_title=Malat1 is not an essential component of nuclear speckles in mice;  citation_journal_title=RNA;  citation_year=2012;  citation_volume=18;  citation_pages=1487-99; " /><meta name="citation_reference" content="citation_title=Endogenous siRNAs from naturally formed dsRNAs regulate transcripts in mouse oocytes;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=453;  citation_pages=539-43; " /><meta name="citation_reference" content="citation_title=Diversity and functional convergence of small noncoding RNAs in male germ cell differentiation and fertilization;  citation_journal_title=RNA;  citation_year=2015;  citation_volume=21;  citation_pages=946-62; " /><meta name="citation_reference" content="citation_title=Global characterization and target identification of piRNAs and endo-siRNAs in mouse gametes and zygotes;  citation_journal_title=Biochim. Biophys. Acta;  citation_year=2014;  citation_volume=1839;  citation_pages=463-75; " /><meta name="citation_reference" content="citation_title=The role of miRNAs and endogenous siRNAs in maternal-to-zygotic reprogramming and the establishment of pluripotency;  citation_journal_title=EMBO Rep;  citation_year=2010;  citation_volume=11;  citation_pages=590-7; " /><meta name="citation_reference" content="citation_title=MicroRNA activity is suppressed in mouse oocytes;  citation_journal_title=Curr. Biol;  citation_year=2010;  citation_volume=20;  citation_pages=265-70; " /><meta name="citation_reference" content="citation_title=Retrotransposons and pseudogenes regulate mRNAs and lncRNAs via the piRNA pathway in the germline;  citation_journal_title=Genome Res;  citation_year=2015;  citation_volume=25;  citation_pages=368-80; " /><meta name="citation_reference" content="citation_title=dsRNA expression in the mouse elicits RNAi in oocytes and low adenosine deamination in somatic cells;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=399-413; " /><meta name="citation_reference" content="citation_title=A molecular timescale for vertebrate evolution;  citation_journal_title=Nature;  citation_year=1998;  citation_volume=392;  citation_pages=917-20; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/2/129/13688935/dsw058.pdf" /><meta name="description" content="Abstract.  The oocyte-to-embryo transition (OET) transforms a differentiated gamete into pluripotent blastomeres. The accompanying maternal-zygotic RNA exchange" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/2/129/2901809" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/2/129/2901809" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/2/129/2901809");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw058';
    </script>
<title>
	Long non-coding RNA exchange during the oocyte-to-embryo transition in mice | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2901809" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bb+HATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2FybGljLCBSb3NhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvaW5mb3JtYXRpY3MgR3JvdXAsIERpdmlzaW9uIG9mIE1vbGVjdWxhciBCaW9sb2d5LCBEZXBhcnRtZW50IG9mIEJpb2xvZ3ksIEZhY3VsdHkgb2YgU2NpZW5jZSwgVW5pdmVyc2l0eSBvZiBaYWdyZWIsIEhvcnZhdG92YWMgMTAyYSwgWmFncmViLCBDcm9hdGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR2FuZXNoLCBTcmF2eWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgTW9sZWN1bGFyIEdlbmV0aWNzLCBBY2FkZW15IG9mIFNjaWVuY2VzIG9mIHRoZSBDemVjaCBSZXB1YmxpYywgVmlkZW5za2EgMTA4MywgMTQyIDIwIFByYWd1ZSA0LCBDemVjaCBSZXB1YmxpYyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZyYW5rZSwgVmVkcmFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvaW5mb3JtYXRpY3MgR3JvdXAsIERpdmlzaW9uIG9mIE1vbGVjdWxhciBCaW9sb2d5LCBEZXBhcnRtZW50IG9mIEJpb2xvZ3ksIEZhY3VsdHkgb2YgU2NpZW5jZSwgVW5pdmVyc2l0eSBvZiBaYWdyZWIsIEhvcnZhdG92YWMgMTAyYSwgWmFncmViLCBDcm9hdGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3ZvYm9kb3ZhLCBFbGlza2EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgTW9sZWN1bGFyIEdlbmV0aWNzLCBBY2FkZW15IG9mIFNjaWVuY2VzIG9mIHRoZSBDemVjaCBSZXB1YmxpYywgVmlkZW5za2EgMTA4MywgMTQyIDIwIFByYWd1ZSA0LCBDemVjaCBSZXB1YmxpYyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlVyYmFub3ZhLCBKYW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIE1vbGVjdWxhciBHZW5ldGljcywgQWNhZGVteSBvZiBTY2llbmNlcyBvZiB0aGUgQ3plY2ggUmVwdWJsaWMsIFZpZGVuc2thIDEwODMsIDE0MiAyMCBQcmFndWUgNCwgQ3plY2ggUmVwdWJsaWMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTdXp1a2ksIFl1dGFrYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQ29tcHV0YXRpb25hbCBCaW9sb2d5IGFuZCBNZWRpY2FsIFNjaWVuY2VzLCBHcmFkdWF0ZSBTY2hvb2wgb2YgRnJvbnRpZXIgU2NpZW5jZXMsIFRoZSBVbml2ZXJzaXR5IG9mIFRva3lvLCBLYXNoaXdhLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFva2ksIEZ1Z2FrdSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgSW50ZWdyYXRlZCBCaW9zY2llbmNlcywgR3JhZHVhdGUgU2Nob29sIG9mIEZyb250aWVyIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgS2FzaGl3YSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJWbGFob3ZpY2VrLCBLcmlzdGlhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb2luZm9ybWF0aWNzIEdyb3VwLCBEaXZpc2lvbiBvZiBNb2xlY3VsYXIgQmlvbG9neSwgRGVwYXJ0bWVudCBvZiBCaW9sb2d5LCBGYWN1bHR5IG9mIFNjaWVuY2UsIFVuaXZlcnNpdHkgb2YgWmFncmViLCBIb3J2YXRvdmFjIDEwMmEsIFphZ3JlYiwgQ3JvYXRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlN2b2JvZGEsIFBldHIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgTW9sZWN1bGFyIEdlbmV0aWNzLCBBY2FkZW15IG9mIFNjaWVuY2VzIG9mIHRoZSBDemVjaCBSZXB1YmxpYywgVmlkZW5za2EgMTA4MywgMTQyIDIwIFByYWd1ZSA0LCBDemVjaCBSZXB1YmxpYyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iTG9uZyBub24tY29kaW5nIFJOQSBleGNoYW5nZSBkdXJpbmcgdGhlIG9vY3l0ZS10by1lbWJyeW8gdHJhbnNpdGlvbiBpbiBtaWNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMTI5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIxNDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3cwNTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNy8wNC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNjdWxwdGluZyB0aGUgdHJhbnNjcmlwdG9tZSBkdXJpbmcgdGhlIG9vY3l0ZS10by1lbWJyeW8gdHJhbnNpdGlvbiBpbiBtb3VzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gVG9wLiBEZXYuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTEzOyAgY2l0YXRpb25fcGFnZXM9MzA1LTQ5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlcmdlbmljIGxuY1JOQXMgYW5kIHRoZSBldm9sdXRpb24gb2YgZ2VuZSBleHByZXNzaW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBHZW5ldC4gRGV2OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NDgtNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vZHVsYXIgcmVndWxhdG9yeSBwcmluY2lwbGVzIG9mIGxhcmdlIG5vbi1jb2RpbmcgUk5BczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ4MjsgIGNpdGF0aW9uX3BhZ2VzPTMzOS00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJlIGFuZCBmdW5jdGlvbiBvZiBsb25nIG5vbmNvZGluZyBSTkFzIGluIGVwaWdlbmV0aWMgcmVndWxhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBTdHJ1Y3QuIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTMwMC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1saW5jUk5BczogZ2Vub21pY3MsIGV2b2x1dGlvbiwgYW5kIG1lY2hhbmlzbXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTU0OyAgY2l0YXRpb25fcGFnZXM9MjYtNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhcGlkIHR1cm5vdmVyIG9mIGxvbmcgbm9uY29kaW5nIFJOQXMgYW5kIHRoZSBldm9sdXRpb24gb2YgZ2VuZSBleHByZXNzaW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMjg0MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvbmcgbm9uLWNvZGluZyBSTkFzIGluIHN0ZW0gY2VsbCBwbHVyaXBvdGVuY3k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVdpbGV5IEludGVyZGlzY2lwLiBSZXYuIFJOQTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MTIxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbmdsZS1jZWxsIFJOQS1TZXEgcHJvZmlsaW5nIG9mIGh1bWFuIHByZWltcGxhbnRhdGlvbiBlbWJyeW9zIGFuZCBlbWJyeW9uaWMgc3RlbSBjZWxsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBTdHJ1Y3QuIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTExMzEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gYW5kIGZ1bmN0aW9uYWwgYW5hbHlzaXMgb2YgbG9uZyBub24tY29kaW5nIFJOQXMgaW4gbW91c2UgY2xlYXZhZ2Ugc3RhZ2UgZW1icnlvbmljIGRldmVsb3BtZW50IGJhc2VkIG9uIHNpbmdsZSBjZWxsIHRyYW5zY3JpcHRvbWUgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9ODQ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBhY3RpdmF0aW9uLWFzc29jaWF0ZWQgbG9uZyBub25jb2RpbmcgUk5BcyBmdW5jdGlvbiBpbiBtb3VzZSBwcmVpbXBsYW50YXRpb24gZGV2ZWxvcG1lbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPURldmVsb3BtZW50OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE0MjsgIGNpdGF0aW9uX3BhZ2VzPTkxMC0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGVlcCBzZXF1ZW5jaW5nIGFuZCBkZSBub3ZvIGFzc2VtYmx5IG9mIHRoZSBtb3VzZSBvb2N5dGUgdHJhbnNjcmlwdG9tZSBkZWZpbmUgdGhlIGNvbnRyaWJ1dGlvbiBvZiB0cmFuc2NyaXB0aW9uIHRvIHRoZSBETkEgbWV0aHlsYXRpb24gbGFuZHNjYXBlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTIwOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBmaXJzdCBtdXJpbmUgenlnb3RpYyB0cmFuc2NyaXB0aW9uIGlzIHByb21pc2N1b3VzIGFuZCB1bmNvdXBsZWQgZnJvbSBzcGxpY2luZyBhbmQgMycgcHJvY2Vzc2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RU1CTyBKOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9MTUyMy0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUNvbGQgU3ByaW5nIEhhcmJvciBMYWJvcmF0b3J5IFByZXNzLCBDb2xkIFNwcmluZyBIYXJib3IsIE5ZOyAgY2l0YXRpb25fdGl0bGU9TWFuaXB1bGF0aW5nIHRoZSBNb3VzZSBFbWJyeW86IEEgTGFib3JhdG9yeSBNYW51YWw7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcmV0cm90cmFuc3Bvc29uLWRyaXZlbiBkaWNlciBpc29mb3JtIGRpcmVjdHMgZW5kb2dlbm91cyBzbWFsbCBpbnRlcmZlcmluZyBSTkEgcHJvZHVjdGlvbiBpbiBtb3VzZSBvb2N5dGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE1NTsgIGNpdGF0aW9uX3BhZ2VzPTgwNy0xNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXVsdGlwbGV4IGdlbm9tZSBlbmdpbmVlcmluZyB1c2luZyBDUklTUFIvQ2FzIHN5c3RlbXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl92b2x1bWU9MzM5OyAgY2l0YXRpb25fcGFnZXM9ODE5LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EeW5hbWljIGltYWdpbmcgb2YgZ2Vub21pYyBsb2NpIGluIGxpdmluZyBodW1hbiBjZWxscyBieSBhbiBvcHRpbWl6ZWQgQ1JJU1BSL0NhcyBzeXN0ZW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl92b2x1bWU9MTU1OyAgY2l0YXRpb25fcGFnZXM9MTQ3OS05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5mZXJyaW5nIHRoZSBjaG9yZW9ncmFwaHkgb2YgcGFyZW50YWwgZ2Vub21lcyBkdXJpbmcgZmVydGlsaXphdGlvbiBmcm9tIHVsdHJhbGFyZ2Utc2NhbGUgd2hvbGUtdHJhbnNjcmlwdG9tZSBhbmFseXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXMgRGV2OyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MjczNi00ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBwcm9ncmFtcyBpbiBodW1hbiBhbmQgbW91c2UgZWFybHkgZW1icnlvcyByZXZlYWxlZCBieSBzaW5nbGUtY2VsbCBSTkEgc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTUwMDsgIGNpdGF0aW9uX3BhZ2VzPTU5My03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BYiBpbml0aW8gcmVjb25zdHJ1Y3Rpb24gb2YgY2VsbCB0eXBlLXNwZWNpZmljIHRyYW5zY3JpcHRvbWVzIGluIG1vdXNlIHJldmVhbHMgdGhlIGNvbnNlcnZlZCBtdWx0aS1leG9uaWMgc3RydWN0dXJlIG9mIGxpbmNSTkFzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz01MDMtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNQQVQ6IGNvZGluZy1wb3RlbnRpYWwgYXNzZXNzbWVudCB0b29sIHVzaW5nIGFuIGFsaWdubWVudC1mcmVlIGxvZ2lzdGljIHJlZ3Jlc3Npb24gbW9kZWw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9ZTc0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXNpbmcgUmVwZWF0TWFza2VyIHRvIGlkZW50aWZ5IHJlcGV0aXRpdmUgZWxlbWVudHMgaW4gZ2Vub21pYyBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIFByb3RvYy4gQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzc2VudGlhbCBSb2xlIGZvciBlbmRvZ2Vub3VzIHNpUk5BcyBkdXJpbmcgbWVpb3NpcyBpbiBtb3VzZSBvb2N5dGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9ZTEwMDUwMTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qc2V1ZG9nZW5lLWRlcml2ZWQgc21hbGwgaW50ZXJmZXJpbmcgUk5BcyByZWd1bGF0ZSBnZW5lIGV4cHJlc3Npb24gaW4gbW91c2Ugb29jeXRlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ1MzsgIGNpdGF0aW9uX3BhZ2VzPTUzNC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QLWJvZHkgbG9zcyBpcyBjb25jb21pdGFudCB3aXRoIGZvcm1hdGlvbiBvZiBhIG1lc3NlbmdlciBSTkEgc3RvcmFnZSBkb21haW4gaW4gbW91c2Ugb29jeXRlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvbC4gUmVwcm9kOyAgY2l0YXRpb25feWVhcj0yMDEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaHJvbWF0aW4gc2lnbmF0dXJlIHJldmVhbHMgb3ZlciBhIHRob3VzYW5kIGhpZ2hseSBjb25zZXJ2ZWQgbGFyZ2Ugbm9uLWNvZGluZyBSTkFzIGluIG1hbW1hbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00NTg7ICBjaXRhdGlvbl9wYWdlcz0yMjMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNwb3NhYmxlIGVsZW1lbnRzIHJldmVhbCBhIHN0ZW0gY2VsbC1zcGVjaWZpYyBjbGFzcyBvZiBsb25nIG5vbmNvZGluZyBSTkFzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPVIxMDciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RdWFudGl0YXRpdmUgY2hhbmdlcyBpbiB0b3RhbCBSbmEsIHRvdGFsIFBvbHkoYSksIGFuZCByaWJvc29tZXMgaW4gZWFybHkgbW91c2UgZW1icnlvczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RGV2LiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTgyOyAgY2l0YXRpb25fdm9sdW1lPTg5OyAgY2l0YXRpb25fcGFnZXM9MzYyLTc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlZ3JhdGl2ZSBhbm5vdGF0aW9uIG9mIGh1bWFuIGxhcmdlIGludGVyZ2VuaWMgbm9uY29kaW5nIFJOQXMgcmV2ZWFscyBnbG9iYWwgcHJvcGVydGllcyBhbmQgc3BlY2lmaWMgc3ViY2xhc3NlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXMgRGV2OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTkxNS0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEdFTkNPREUgdjcgY2F0YWxvZyBvZiBodW1hbiBsb25nIG5vbmNvZGluZyBSTkFzOiBhbmFseXNpcyBvZiB0aGVpciBnZW5lIHN0cnVjdHVyZSwgZXZvbHV0aW9uLCBhbmQgZXhwcmVzc2lvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTE3NzUtODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnNlcnZlZCBmdW5jdGlvbiBvZiBsaW5jUk5BcyBpbiB2ZXJ0ZWJyYXRlIGVtYnJ5b25pYyBkZXZlbG9wbWVudCBkZXNwaXRlIHJhcGlkIHNlcXVlbmNlIGV2b2x1dGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDc7ICBjaXRhdGlvbl9wYWdlcz0xNTM3LTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EeW5hbWljIENwRyBpc2xhbmQgbWV0aHlsYXRpb24gbGFuZHNjYXBlIGluIG9vY3l0ZXMgYW5kIHByZWltcGxhbnRhdGlvbiBlbWJyeW9zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9ODExLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBodW1hbiBhbmQgbW91c2UgcmVwbGljYXRpb24tZGVwZW5kZW50IGhpc3RvbmUgZ2VuZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTgwOyAgY2l0YXRpb25fcGFnZXM9NDg3LTk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2xhdGlvbmFsIGNvbnRyb2wgaW4gb29jeXRlIGRldmVsb3BtZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Db2xkIFNwcmluZyBIYXJiLiBQZXJzcGVjdC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9YTAwMjc1OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5bnRoZXNpcyBhbmQgZnVuY3Rpb24gb2YgTW9zOiB0aGUgY29udHJvbCBzd2l0Y2ggb2YgdmVydGVicmF0ZSBvb2N5dGUgbWVpb3NpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvZXNzYXlzOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MjMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWVpb3RpYyBtYXR1cmF0aW9uIG9mIG1vdXNlIG9vY3l0ZXMgdHJpZ2dlcnMgdGhlIHRyYW5zbGF0aW9uIGFuZCBwb2x5YWRlbnlsYXRpb24gb2YgZG9ybWFudCB0aXNzdWUtdHlwZSBwbGFzbWlub2dlbiBhY3RpdmF0b3IgbVJOQTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXMgRGV2OyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0xMjAxLTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY3F1aXNpdGlvbiBvZiBtZWlvdGljIGNvbXBldGVuY2UgaW4gZ3Jvd2luZyBtb3VzZSBvb2N5dGVzIGlzIGNvbnRyb2xsZWQgYXQgYm90aCB0cmFuc2xhdGlvbmFsIGFuZCBwb3N0dHJhbnNsYXRpb25hbCBsZXZlbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPURldi4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0xODc7ICBjaXRhdGlvbl9wYWdlcz00My01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjcnVpdG1lbnQgb2YgT3JjNmwsIGEgZG9ybWFudCBtYXRlcm5hbCBtUk5BIGluIG1vdXNlIG9vY3l0ZXMsIGlzIGVzc2VudGlhbCBmb3IgRE5BIHJlcGxpY2F0aW9uIGluIDEtY2VsbCBlbWJyeW9zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1EZXYuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MzQxOyAgY2l0YXRpb25fcGFnZXM9MjA1LTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXRlcm5hbGx5IHJlY3J1aXRlZCBEQ1AxQSBhbmQgRENQMiBjb250cmlidXRlIHRvIG1lc3NlbmdlciBSTkEgZGVncmFkYXRpb24gZHVyaW5nIG9vY3l0ZSBtYXR1cmF0aW9uIGFuZCBnZW5vbWUgYWN0aXZhdGlvbiBpbiBtb3VzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvbC4gUmVwcm9kOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg4OyAgY2l0YXRpb25fcGFnZXM9MTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DeXRvcGxhc21pYyBwb2x5YWRlbnlsYXRpb24gZWxlbWVudCAoQ1BFKS0gYW5kIENQRS1iaW5kaW5nIHByb3RlaW4gKENQRUIpLWluZGVwZW5kZW50IG1lY2hhbmlzbXMgcmVndWxhdGUgZWFybHkgY2xhc3MgbWF0ZXJuYWwgbVJOQSB0cmFuc2xhdGlvbmFsIGFjdGl2YXRpb24gaW4gWGVub3B1cyBvb2N5dGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCaW9sLiBDaGVtOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTI3OTsgIGNpdGF0aW9uX3BhZ2VzPTE3NjUwLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlZ3JhZGF0aW9uIG9mIG1hdGVybmFsIG1STkEgaW4gbW91c2UgZW1icnlvczogc2VsZWN0aXZlIGRlZ3JhZGF0aW9uIG9mIHNwZWNpZmljIG1STkFzIGFmdGVyIGZlcnRpbGl6YXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gUmVwcm9kLiBEZXY7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NzI7ICBjaXRhdGlvbl9wYWdlcz0yODEtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpdmVyc2UgcGF0dGVybnMgb2YgcG9seShBKSB0YWlsIGVsb25nYXRpb24gYW5kIHNob3J0ZW5pbmcgb2YgbXVyaW5lIG1hdGVybmFsIG1STkFzIGZyb20gZnVsbHkgZ3Jvd24gb29jeXRlIHRvIDItY2VsbCBlbWJyeW8gc3RhZ2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9jaGVtLiBCaW9waHlzLiBSZXMuIENvbW11bjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzY7ICBjaXRhdGlvbl9wYWdlcz0xMTgxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN5dG9wbGFzbWljIGxvbmcgbm9uY29kaW5nIFJOQXMgYXJlIGZyZXF1ZW50bHkgYm91bmQgdG8gYW5kIGRlZ3JhZGVkIGF0IHJpYm9zb21lcyBpbiBodW1hbiBjZWxsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Uk5BOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9ODY3LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Mb25nIG5vbi1jb2RpbmcgUk5BcyBhcyBhIHNvdXJjZSBvZiBuZXcgcGVwdGlkZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVsaWZlOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz1lMDM1MjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbXBhcmF0aXZlIGVuY3ljbG9wZWRpYSBvZiBETkEgZWxlbWVudHMgaW4gdGhlIG1vdXNlIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTUxNTsgIGNpdGF0aW9uX3BhZ2VzPTM1NS02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUFMQVQxIOKAkyBhIHBhcmFkaWdtIGZvciBsb25nIG5vbmNvZGluZyBSTkEgZnVuY3Rpb24gaW4gY2FuY2VyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIE1lZC4gKEJlcmwpOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTkxOyAgY2l0YXRpb25fcGFnZXM9NzkxLTgwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG9zcyBvZiB0aGUgYWJ1bmRhbnQgbnVjbGVhciBub24tY29kaW5nIFJOQSBNQUxBVDEgaXMgY29tcGF0aWJsZSB3aXRoIGxpZmUgYW5kIGRldmVsb3BtZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1STkEgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MTA3Ni04NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGxuY1JOQSBNYWxhdDEgaXMgZGlzcGVuc2FibGUgZm9yIG1vdXNlIGRldmVsb3BtZW50IGJ1dCBpdHMgdHJhbnNjcmlwdGlvbiBwbGF5cyBhIGNpcy1yZWd1bGF0b3J5IHJvbGUgaW4gdGhlIGFkdWx0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsIFJlcDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MTExLTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYWxhdDEgaXMgbm90IGFuIGVzc2VudGlhbCBjb21wb25lbnQgb2YgbnVjbGVhciBzcGVja2xlcyBpbiBtaWNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1STkE7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xNDg3LTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbmRvZ2Vub3VzIHNpUk5BcyBmcm9tIG5hdHVyYWxseSBmb3JtZWQgZHNSTkFzIHJlZ3VsYXRlIHRyYW5zY3JpcHRzIGluIG1vdXNlIG9vY3l0ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00NTM7ICBjaXRhdGlvbl9wYWdlcz01MzktNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpdmVyc2l0eSBhbmQgZnVuY3Rpb25hbCBjb252ZXJnZW5jZSBvZiBzbWFsbCBub25jb2RpbmcgUk5BcyBpbiBtYWxlIGdlcm0gY2VsbCBkaWZmZXJlbnRpYXRpb24gYW5kIGZlcnRpbGl6YXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVJOQTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTk0Ni02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIGNoYXJhY3Rlcml6YXRpb24gYW5kIHRhcmdldCBpZGVudGlmaWNhdGlvbiBvZiBwaVJOQXMgYW5kIGVuZG8tc2lSTkFzIGluIG1vdXNlIGdhbWV0ZXMgYW5kIHp5Z290ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2NoaW0uIEJpb3BoeXMuIEFjdGE7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTgzOTsgIGNpdGF0aW9uX3BhZ2VzPTQ2My03NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJvbGUgb2YgbWlSTkFzIGFuZCBlbmRvZ2Vub3VzIHNpUk5BcyBpbiBtYXRlcm5hbC10by16eWdvdGljIHJlcHJvZ3JhbW1pbmcgYW5kIHRoZSBlc3RhYmxpc2htZW50IG9mIHBsdXJpcG90ZW5jeTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RU1CTyBSZXA7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz01OTAtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9STkEgYWN0aXZpdHkgaXMgc3VwcHJlc3NlZCBpbiBtb3VzZSBvb2N5dGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MjY1LTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXRyb3RyYW5zcG9zb25zIGFuZCBwc2V1ZG9nZW5lcyByZWd1bGF0ZSBtUk5BcyBhbmQgbG5jUk5BcyB2aWEgdGhlIHBpUk5BIHBhdGh3YXkgaW4gdGhlIGdlcm1saW5lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzY4LTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1kc1JOQSBleHByZXNzaW9uIGluIHRoZSBtb3VzZSBlbGljaXRzIFJOQWkgaW4gb29jeXRlcyBhbmQgbG93IGFkZW5vc2luZSBkZWFtaW5hdGlvbiBpbiBzb21hdGljIGNlbGxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPTM5OS00MTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbW9sZWN1bGFyIHRpbWVzY2FsZSBmb3IgdmVydGVicmF0ZSBldm9sdXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0zOTI7ICBjaXRhdGlvbl9wYWdlcz05MTctMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNC8yLzEyOS8xMzY4ODkzNS9kc3cwNTgucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFRoZSBvb2N5dGUtdG8tZW1icnlvIHRyYW5zaXRpb24gKE9FVCkgdHJhbnNmb3JtcyBhIGRpZmZlcmVudGlhdGVkIGdhbWV0ZSBpbnRvIHBsdXJpcG90ZW50IGJsYXN0b21lcmVzLiBUaGUgYWNjb21wYW55aW5nIG1hdGVybmFsLXp5Z290aWMgUk5BIGV4Y2hhbmdlIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0yOTAxODA5JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZFhC2hodd175qzbtugE2QwctPz8s" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f129%2f2901809" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f129%2f2901809" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=97104" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&amp;Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/2">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="78055336">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055336">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055338">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055338">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055342">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055342">2. Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055368">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055368">3. Results and Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055407">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055407">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055415">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055415">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/2/117/2949521">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/2/143/2870506">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Long non-coding RNA exchange during the oocyte-to-embryo transition in mice
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rosa Karlic</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Bioinformatics Group, Division of Molecular Biology, Department of Biology, Faculty of Science, University of Zagreb, Horvatovac 102a, Zagreb, Croatia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rosa+Karlic">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Karlic R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Karlic R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Rosa Karlic</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sravya Ganesh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Institute of Molecular Genetics, Academy of Sciences of the Czech Republic, Videnska 1083, 142 20 Prague 4, Czech Republic</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sravya+Ganesh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ganesh S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ganesh S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sravya Ganesh</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Vedran Franke</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Bioinformatics Group, Division of Molecular Biology, Department of Biology, Faculty of Science, University of Zagreb, Horvatovac 102a, Zagreb, Croatia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Vedran+Franke">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Franke V">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Franke V&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Vedran Franke</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eliska Svobodova</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Institute of Molecular Genetics, Academy of Sciences of the Czech Republic, Videnska 1083, 142 20 Prague 4, Czech Republic</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eliska+Svobodova">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Svobodova E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Svobodova E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Eliska Svobodova</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jana Urbanova</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Institute of Molecular Genetics, Academy of Sciences of the Czech Republic, Videnska 1083, 142 20 Prague 4, Czech Republic</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jana+Urbanova">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Urbanova J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Urbanova J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jana Urbanova</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yutaka Suzuki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yutaka+Suzuki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suzuki Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suzuki Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yutaka Suzuki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fugaku Aoki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div>Department of Integrated Biosciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw058-cor1" data-open="dsw058-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +420 241063147. Email: <a href="mailto:svobodap@img.cas.cz" target="_blank">svobodap@img.cas.cz</a>; Tel. +385 1 4606306, Email: <a href="mailto:kristian@bioinfo.hr" target="_blank">kristian@bioinfo.hr</a>; Tel. +81-4-7136-3695, Email: <a href="mailto:aokif@k.u-tokyo.ac.jp" target="_blank">aokif@k.u-tokyo.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fugaku+Aoki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Aoki F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Aoki F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Fugaku Aoki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kristian Vlahovicek</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Bioinformatics Group, Division of Molecular Biology, Department of Biology, Faculty of Science, University of Zagreb, Horvatovac 102a, Zagreb, Croatia</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw058-cor1" data-open="dsw058-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +420 241063147. Email: <a href="mailto:svobodap@img.cas.cz" target="_blank">svobodap@img.cas.cz</a>; Tel. +385 1 4606306, Email: <a href="mailto:kristian@bioinfo.hr" target="_blank">kristian@bioinfo.hr</a>; Tel. +81-4-7136-3695, Email: <a href="mailto:aokif@k.u-tokyo.ac.jp" target="_blank">aokif@k.u-tokyo.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kristian+Vlahovicek">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Vlahovicek K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Vlahovicek K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kristian Vlahovicek</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Petr Svoboda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Institute of Molecular Genetics, Academy of Sciences of the Czech Republic, Videnska 1083, 142 20 Prague 4, Czech Republic</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw058-cor1" data-open="dsw058-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +420 241063147. Email: <a href="mailto:svobodap@img.cas.cz" target="_blank">svobodap@img.cas.cz</a>; Tel. +385 1 4606306, Email: <a href="mailto:kristian@bioinfo.hr" target="_blank">kristian@bioinfo.hr</a>; Tel. +81-4-7136-3695, Email: <a href="mailto:aokif@k.u-tokyo.ac.jp" target="_blank">aokif@k.u-tokyo.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Petr+Svoboda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Svoboda P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Svoboda P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Petr Svoboda</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr. Minoru Ko</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 2, 1 April 2017, Pages 129–141, <a href='https://doi.org/10.1093/dnares/dsw058'>https://doi.org/10.1093/dnares/dsw058</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">12 January 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">30 June 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">28 November 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2901809" href="/dnaresearch/article-pdf/24/2/129/13688935/dsw058.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Rosa Karlic, Sravya Ganesh, Vedran Franke, Eliska Svobodova, Jana Urbanova, Yutaka Suzuki, Fugaku Aoki, Kristian Vlahovicek, Petr Svoboda;  Long non-coding RNA exchange during the oocyte-to-embryo transition in mice, <em>DNA Research</em>, Volume 24, Issue 2, 1 April 2017, Pages 129–141, <a href='https://doi.org/10.1093/dnares/dsw058'>https://doi.org/10.1093/dnares/dsw058</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2901809?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2901809?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2901809?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2901809?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2901809?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2901809?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2901809?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Long%20non-coding%20RNA%20exchange%20during%20the%20oocyte-to-embryo%20transition%20in%20mice&amp;publicationDate=2017-01-12&amp;volumeNum=24&amp;issueNum=2&amp;author=Karlic%2C%20Rosa%3B%20Ganesh%2C%20Sravya&amp;startPage=129&amp;endPage=141&amp;contentId=10.1093%2Fdnares%2Fdsw058&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f129%2f2901809" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=78055336 id="78055336" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The oocyte-to-embryo transition (OET) transforms a differentiated gamete into pluripotent blastomeres. The accompanying maternal-zygotic RNA exchange involves remodeling of the long non-coding RNA (lncRNA) pool. Here, we used next generation sequencing and <em>de novo</em> transcript assembly to define the core population of 1,600 lncRNAs expressed during the OET (lncRNAs). Relative to mRNAs, OET lncRNAs were less expressed and had shorter transcripts, mainly due to fewer exons and shorter 5′ terminal exons. Approximately half of OET lncRNA promoters originated in retrotransposons suggesting their recent emergence. Except for a small group of ubiquitous lncRNAs, maternal and zygotic lncRNAs formed two distinct populations. The bulk of maternal lncRNAs was degraded before the zygotic genome activation. Interestingly, maternal lncRNAs seemed to undergo cytoplasmic polyadenylation observed for dormant mRNAs. We also identified lncRNAs giving rise to trans-acting short interfering RNAs, which represent a novel lncRNA category. Altogether, we defined the core OET lncRNA transcriptome and characterized its remodeling during early development. Our results are consistent with the notion that rapidly evolving lncRNAs constitute signatures of cells-of-origin while a minority plays an active role in control of gene expression across OET. Our data presented here provide an excellent source for further OET lncRNA studies.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="lncRNA">lncRNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="oocyte">oocyte</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="zygote">zygote</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="polyadenylation">polyadenylation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="endo-siRNA">endo-siRNA</a></div>                            <h2 scrollto-destination=78055338 id="78055338" class="section-title" >1. Introduction</h2>
<p>The oocyte-to-embryo transition (OET), the transformation of a differentiated oocyte into a developing embryo, involves massive reprogramming of gene expression where zygotic genome activation (ZGA) initiates production of zygotic mRNA to replace maternal RNAs in control of development. Although changes of mRNA and small RNA populations during OET were characterized in a considerable detail (reviewed in<a reveal-id="dsw058-B1" data-open="dsw058-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a>), much less is known about composition and temporal changes of spliced long non-coding RNAs (lncRNAs). LncRNAs (reviewed for example in<a reveal-id="dsw058-B2 dsw058-B3 dsw058-B4 dsw058-B5" data-open="dsw058-B2 dsw058-B3 dsw058-B4 dsw058-B5" class="link link-ref link-reveal xref-bibr"><sup>2–5</sup></a>) add another layer to the transcriptome complexity. LncRNAs are an arbitrary category adopted for spliced transcripts not encoding proteins that are longer than 200 nucleotides (nt). LncRNAs represent an assorted group of RNAs implicated in transcriptional, post-transcriptional, translational, and epigenetic regulations or, importantly, without any apparent functions. LncRNAs evolve rapidly, showing little if any sequence conservation.<a reveal-id="dsw058-B6" data-open="dsw058-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> It is assumed that a relatively minor fraction of these transcripts is functional while the rest might represent transcriptional noise and/or lncRNAs that have appeared recently in evolution and have not acquired a function (reviewed in<a reveal-id="dsw058-B2" data-open="dsw058-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a>). However, specific lncRNAs are functionally important for different processes, which include the maintenance and induction of stem cell pluripotency (reviewed in<a reveal-id="dsw058-B7" data-open="dsw058-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a>).</p><p>Next generation sequencing (NGS)-based studies provided partial insights into some aspects of lncRNA biology during the mammalian OET. Upon single-cell RNA profiling of human preimplantation embryos, Yan et al.<a reveal-id="dsw058-B8" data-open="dsw058-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> reported 2,733 novel expressed lncRNAs. Zhang et al.<a reveal-id="dsw058-B9" data-open="dsw058-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> used single-cell SOLiD NGS data from OET stages and reported 5,563 novel lncRNAs. Hamazaki et al.<a reveal-id="dsw058-B10" data-open="dsw058-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> studied a specific lncRNA group termed promoter associated non-coding RNAs (pancRNAs) in ovulated oocytes and two-cell zygotes. So far, the most detailed analysis of OET lncRNAs has been provided by Veselovska et al.,<a reveal-id="dsw058-B11" data-open="dsw058-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> who produced <em>de novo</em> transcriptome assembly that included lncRNAs. However, their main research focus was the contribution of transcription to the DNA methylation landscape and not a thorough annotation and analysis of lncRNAs.</p><p>Understanding the composition of maternal and zygotic non-coding RNA pools is pre-requisite for understanding their biological roles during OET. In this study, we sought to provide a highly reliable set of <em>de novo</em> assembled lncRNAs present during OET (referred to as OET lncRNAs hereafter) and perform its characterization in terms of structure and expression. Accordingly, we identified, annotated, and characterized 1,600 OET lncRNA loci, including their transcriptional and post-transcriptional temporal dynamics. OET lncRNAs exhibited typical features of lncRNAs: lower expression levels than mRNAs, highly variable splicing, and restricted expression. Remarkably, the OET lncRNA expression largely falls into mutually exclusive maternal and zygotic expression patterns but rarely into the maternal-zygotic expression, which is common for mRNAs. Finally, we produced CRISPR-mediated knockouts of two maternal conserved lncRNAs without an effect on fertility.</p>                            <h2 scrollto-destination=78055342 id="78055342" class="section-title" >2. Methods</h2>
                            <h3 scrollto-destination=78055343 id="78055343" class="section-title" >2.1. RNA extraction, preparation of the NGS library and sequencing</h3>
<p>Total RNA was extracted from 3,000 fully grown germinal vesicle (GV)-intact oocytes obtained from C57BL6/J mice, respectively, using Isogen (Nippon Gene, Tokyo, Japan), according to the manufacturer’s instructions. PolyA RNA was isolated by using mRNA purification kit (Invitrogen, Carlsbad, CA; cat no. 610.06). High-throughput sequencing of size-selected RNA (&gt;200 nt) was performed using Genome Analyzer IIx (Illumina) and 76-nt paired-end-sequencing reads as described previously in.<a reveal-id="dsw058-B12" data-open="dsw058-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> The complete set of NGS data is available in the Array Express database under accession IDs E-MTAB-2950 and E-MTAB-4775.</p>                            <h3 scrollto-destination=78055345 id="78055345" class="section-title" >2.2. Analysis of lncRNA expression in oocytes and early embryos by real-time PCR</h3>
<p>Oocytes and early embryos were obtained from C57Bl/6 mice as described previously in.<a reveal-id="dsw058-B13" data-open="dsw058-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsw058-B14" data-open="dsw058-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> Resumption of meiosis during collection of GV oocytes was prevented with 0.2 mM 3-isobutyl-1-methyl-xanthine (IBMX, Sigma). RNA from a chosen number of oocytes or early embryos was released upon incubation in water with RNase inhibitor for 5 min at 85 °C. RNA was reverse-transcribed using <em>RevertAid</em> First Strand cDNA Synthesis Kit (Fermentas). Maxima SYBR Green qPCR Master Mix (Fermentas) was used for qPCR. The primers and PCR conditions are shown in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>.</p>                            <h3 scrollto-destination=78055347 id="78055347" class="section-title" >2.3. Production of lncRNA knockout models</h3>
<p>LncRNA knockout models were produced in the Transgenic Unit of the Institute of Molecular Genetics ASCR, Czech Centre for Phenogenomics using Cas9-mediated deletion of lncRNA promoters (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs. S7 and S8</a></span>).<a reveal-id="dsw058-B15" data-open="dsw058-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsw058-B16" data-open="dsw058-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> All animal experiments were approved by the Institutional Animal Use and Care Committees (project number 58-2015) and were carried out in accordance with the law.</p><p>Sequences of guide RNAs are listed in the Table S5. To produce guide RNAs, synthetic 128 nt guide RNA templates including T7 promoter, 18 nt sgRNA and tracrRNA sequences were amplified using T7 and TracrRNA primers (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Guide RNAs were produced <em>in vitro</em> using the Ambion mMESSAGE mMACHINE T7 Transcription Kit, and purified using the mirPremier microRNA Isolation Kit (Sigma). The Cas9 mRNA was synthesized from pSp Cas9-puro plasmid using Ambion mMESSAGE mMACHINE T7 Transcription Kit, and purified using the Qiagen RNasy mini kit. A sample for microinjection was prepared by mixing two guide RNAs in ultra-pure water at a concentration of 25 ng/µl for each one together with Cas9 RNA (100 ng/µl) . Five picoliters of the microinjection mixture were injected into male pronuclei of C57Bl/6 zygotes and transferred into pseudopregnant recipient mice. PCR genotyping was performed on tail biopsies from 4 weeks-old animals. Primers are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>.</p>                            <h3 scrollto-destination=78055350 id="78055350" class="section-title" >2.4. Bioinformatics analyses</h3>
                            <h4 scrollto-destination=78055351 id="78055351" class="section-title" >2.4.1. Mapping of Illumina NGS reads on the mouse genome</h4>
<p>Mapping of NGS data was performed as described previously in.<a reveal-id="dsw058-B12" data-open="dsw058-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> Briefly, adapters were removed using the Trimmomatic software (doi: 10.1093/bioinformatics/btu170). The filtered reads were mapped onto the mm9/NCBI37 version of the mouse genome using the STAR mapper (doi: 10.1093/bioinformatics/bts635) and the genome index was constructed with the addition of the mm9 Ensembl gene annotation, downloaded on 20 September 2013 from the Ensembl database. The dynamic ranges of read counts permitted us to use counts per million normalization for downstream analyses as they did not vary significantly across experiments.<a reveal-id="dsw058-B12" data-open="dsw058-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> Data were visualized in the UCSC browser by constructing bigWig tracks using the Bedtools software (10.1093/bioinformatics/btq033).</p>                            <h4 scrollto-destination=78055353 id="78055353" class="section-title" >2.4.2. Transcript model assembly from NGS data</h4>
<p>For assembling lncRNA transcript models, we used 76-nt paired end (76PE) non-directional NGS data with depths 33–58 × 10<sup>6</sup> sequence reads/sample from oocytes and early embryos (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Except of fully grown GV-intact oocytes data, other NGS datasets were published.<a reveal-id="dsw058-B12" data-open="dsw058-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a></p><p>For transcript model assembly, we combined total RNA NGS datasets (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) into three sets as follows: (i) the maternal set: fully grown GV oocyte and MII egg NGS data, (ii) the ZGA (zygotic) set: two- and four-cell NGS data, and (iii) the late preimplantation (embryonic) set: morula and blastocyst NGS data. Transcript model assembly was performed for each set separately to reduce artifacts from degraded maternal RNAs and to achieve accurate assembly of overlapping sense and antisense transcripts. We tested several combinations of data pooling including pooling with published shorter single-end read NGS data (e.g.<a reveal-id="dsw058-B17" data-open="dsw058-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsw058-B18" data-open="dsw058-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a>). However, the aforementioned grouping of 76PE samples into three independent sets yielded by far the best results in assembling transcript models of a diagnostic set of 20 lncRNA loci. Although addition of published 35SE sets would increase the sequencing depth and reveal additional lncRNAs, it also introduced numerous artifacts into transcript models when compared with transcript assembly based on 76PE sets, which yielded the best exon-intron junction prediction.</p><p>To build transcript models from NGS data, we used Scripture,<a reveal-id="dsw058-B19" data-open="dsw058-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> which performed better than Cufflinks or Stringtie in assembling the aforementioned diagnostic set of annotated and novel lncRNAs (data not shown). Transcript models generated by Scripture were refined to eliminate various artifacts. We filled introns &lt;20 nt length and removed transcript models containing introns &gt;250 kb, which were typically repeat-derived artifacts and disturbed assembly of transcript model clusters. We also removed single-exon transcripts and transcripts shorter than 200 nucleotides.</p><p>Refined transcript models in each developmental set were then used to build transcript clusters where each lncRNA cluster accommodated all transcript models with the same orientation sharing at least one splice site. Thus, a lncRNA cluster represents a locus containing a group of exons found in clustered transcript models. Transcript models in clusters were subsequently analysed for coding potential by CPAT<a reveal-id="dsw058-B20" data-open="dsw058-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). Building clusters and CPAT pre-filtering performed separately on the three sets was chosen because it simplified data management, allowing to work in parallel with smaller volumes of developmentally relevant data. All clusters containing any CPAT positive transcript model were removed from the lncRNA annotation in order to eliminate transcript model artifacts derived from poorly assembled mRNAs. Although one promoter could produce a shorter lncRNA and a longer mRNA (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>), we decided to remove entire clusters ‘contaminated’ with any CPAT-positive transcript models. In any case, lncRNAs in such clusters overlap with mRNAs in sense, which represented an lncRNA category omitted from our annotation and further analysis.</p><p>Next, we merged clusters containing the same transcript models from the three NGS sets. We first merged pools from the maternal lncRNA NGS set with the ZGA lncRNAs NGS set while we removed partial ZGA transcript models matching maternal transcript models as they typically came from poor assembly of degraded maternal transcripts. Then we added transcript models from late pre-implantation NGS clusters and generated a single non-redundant set of transcript model clusters for final refining.</p><p>Transcript models in each cluster were further refined by revising terminal exon predictions since we noticed that Scripture tends to produce truncated terminal exon variants despite an apparent support from NGS or expressed sequence tag (EST) data. Thus, when Scripture predicted terminal exon variants sharing the same splice donor or splice acceptor, we calculated read densities (FPKM - fragments per kilobase of exon per million fragments mapped) for those exon variants and if they did not differ &gt;2-fold, we collapsed shorter exon variants to retain only the longer one as the terminal exon for transcript models. We also refined predicted transcript models using mm9-annotated ESTs, which originated from oocytes and early embryos. We compared the annotated exon-intron structure of these ESTs with our transcript models and annotated lncRNAs. Truncated/partial transcript models were extended according to ESTs if both the ESTs and previously annotated lncRNAs supported such extensions. Further refining included the following filtering steps: transcript models were removed, which contained sequences of known highly abundant non-coding RNAs (e.g. rRNA, U1-U7 RNA, 7SLRNA, SSU-rRNA). We also removed all clusters where &gt;75% of sequences were recognized by Repeatmasker.<a reveal-id="dsw058-B21" data-open="dsw058-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> Although some true lncRNAs are made of &gt;75% of repetitive sequences, repeats were causing artifacts yielding spliced transcript models (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). We also added a filter to eliminate strange artifacts where sequence reads derived from exons of a highly abundant mRNA were assembled into an antisense lncRNA transcript model, not following the canonical splice donor and acceptor sequence rules. Finally, refined transcript models were analysed again for coding potential using CPAT and clusters carrying CPAT positive transcript models were removed (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>).</p><p>Then, we individually assessed all clusters which passed filtering until this point but their highest exon maximum FPKM in any of the developmental stages was &lt;1 and we retained those, which seemed to be supported by NGS data display in the UCSC browser (476 transcript clusters were retained). The expression of each cluster was calculated as the maximum exon FPKM, after masking the parts of lncRNA exons, which overlapped exons of protein-coding genes.</p><div class="block-child-p">The clusters (and their individual transcript models as well) were classified in six categories according to their positions in the genome (<a reveal-id="dsw058-F1" data-open="dsw058-F1" class="link link-reveal link-table xref-fig">Fig. 1C</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&amp;Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). The classification system used a simple dichotomic annotation based on boundaries of clusters (transcript models) and their orientations relative to transcripts from protein-coding genes. If there was no overlap, clusters were considered Intergenic. When there was an overlap, we distinguished sense and antisense overlap and we further recognized whether or not an overlap formed between mature (spliced) transcripts. When one lncRNA was associated with more than gene, we opted for classifying such lncRNAs simply as Complex (in total, there were 30 such loci). Clusters from the two sense categories were omitted from the further analysis because they contained too many artifacts that were hindering the annotation effort. <div data-id="dsw058-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/m_dsw058f1.png?Expires=2147483647&amp;Signature=BraSkiaTRyIM31K~pOvHEupR-aI2ovlsTJ84CElm9acyuqKBnOVw5A~9JAECpD12hIxyo0rzRJ2cOiZmmXAf6ahzx4xh5oxhybqwFL-0IuvUp3Y9ypJhlq04HA7eILYYzNLilaE~dzgStjttMfGd6A~LoAIAg9dtfp7y-Br0UPhUXWnON6~-X6oc9BdwaiV2nHs0IjOEDYJr9pazJ--Y~0AottIaNJIoOdaraS5qS~R80PClHeql3K~9wsi8RJF0uUTVEAbvUYnNYQDE3tG8BgTyfPfI2N2vk~HhdxfkUhYaY~NiTOHky6iGPfV2o2s9UoelV94l7~Td4V3KJ~Ya2Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overview of lncRNA analysis. (A) Samples analysed by 76PE NGS and their grouping for assembling transcript models. (B) Workflow of identification of lncRNA clusters from oocytes and early embryos NGS data. (C) Classification system used for annotating lncRNA transcript models and clusters. (D) Non-annotated lncRNA loci identified in this study. Exons from transcript models from each clusters were compared with data in ENCODE and Refseq databases. Clusters, in which none of the exon-exon junctions from transcript models matched an exon-exon junction annotated in these databases were considered novel. (E) Origin of transcript models in lncRNA clusters. The Venn diagram depicts which set produced transcript models for lncRNA clusters. For example, 19 lncRNA clusters contain transcript models assembled in all three developmental sets while 973 clusters comprise of exclusively oocyte-derived transcript models." path-from-xml="dsw058f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055361/dsw058f1.tif" path-from-xml="dsw058f1.tif" target="_blank">View large</a><a section="78055361" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058f1.png?Expires=2147483647&amp;Signature=F2Qt7bslywWpgkWl6a29SmDbvINJ8fpPsHQcqW-EXzGGEwt1IPJiYJKGJSFCHJIBfUPW7iDo4GhrvmbKVWUeMWg-bNFdQxQODgh2tI1dXBWSljt1SRQ4gPFqDngLYROG2sLyNJrJ44kl1h3VDEolZEGntR0-FV4FU2ldV56vpprxogkcuc1ICEyl5z-kD2RnxgeM0lbaf14LTbb8cqckT8-lW~OmwRtaub8JVMqEQAZcg0t0byq~pILEmvLi0Lb3eG-HiPfAKDmPhCntOei7PVLAoj35QQwBrhOT2kmMAW0rZaL4RwBu9uxmd2tJeeb477CbMITpuiVE6XODSyq1Tw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055361&amp;ar=2901809&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw058f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Overview of lncRNA analysis. <strong>(A)</strong> Samples analysed by 76PE NGS and their grouping for assembling transcript models. <strong>(B)</strong> Workflow of identification of lncRNA clusters from oocytes and early embryos NGS data. <strong>(C)</strong> Classification system used for annotating lncRNA transcript models and clusters. <strong>(D)</strong> Non-annotated lncRNA loci identified in this study. Exons from transcript models from each clusters were compared with data in ENCODE and Refseq databases. Clusters, in which none of the exon-exon junctions from transcript models matched an exon-exon junction annotated in these databases were considered novel. <strong>(E)</strong> Origin of transcript models in lncRNA clusters. The Venn diagram depicts which set produced transcript models for lncRNA clusters. For example, 19 lncRNA clusters contain transcript models assembled in all three developmental sets while 973 clusters comprise of exclusively oocyte-derived transcript models.</p></div></div><div content-id="dsw058-F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/m_dsw058f1.png?Expires=2147483647&amp;Signature=BraSkiaTRyIM31K~pOvHEupR-aI2ovlsTJ84CElm9acyuqKBnOVw5A~9JAECpD12hIxyo0rzRJ2cOiZmmXAf6ahzx4xh5oxhybqwFL-0IuvUp3Y9ypJhlq04HA7eILYYzNLilaE~dzgStjttMfGd6A~LoAIAg9dtfp7y-Br0UPhUXWnON6~-X6oc9BdwaiV2nHs0IjOEDYJr9pazJ--Y~0AottIaNJIoOdaraS5qS~R80PClHeql3K~9wsi8RJF0uUTVEAbvUYnNYQDE3tG8BgTyfPfI2N2vk~HhdxfkUhYaY~NiTOHky6iGPfV2o2s9UoelV94l7~Td4V3KJ~Ya2Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overview of lncRNA analysis. (A) Samples analysed by 76PE NGS and their grouping for assembling transcript models. (B) Workflow of identification of lncRNA clusters from oocytes and early embryos NGS data. (C) Classification system used for annotating lncRNA transcript models and clusters. (D) Non-annotated lncRNA loci identified in this study. Exons from transcript models from each clusters were compared with data in ENCODE and Refseq databases. Clusters, in which none of the exon-exon junctions from transcript models matched an exon-exon junction annotated in these databases were considered novel. (E) Origin of transcript models in lncRNA clusters. The Venn diagram depicts which set produced transcript models for lncRNA clusters. For example, 19 lncRNA clusters contain transcript models assembled in all three developmental sets while 973 clusters comprise of exclusively oocyte-derived transcript models." path-from-xml="dsw058f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055361/dsw058f1.tif" path-from-xml="dsw058f1.tif" target="_blank">View large</a><a section="78055361" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058f1.png?Expires=2147483647&amp;Signature=F2Qt7bslywWpgkWl6a29SmDbvINJ8fpPsHQcqW-EXzGGEwt1IPJiYJKGJSFCHJIBfUPW7iDo4GhrvmbKVWUeMWg-bNFdQxQODgh2tI1dXBWSljt1SRQ4gPFqDngLYROG2sLyNJrJ44kl1h3VDEolZEGntR0-FV4FU2ldV56vpprxogkcuc1ICEyl5z-kD2RnxgeM0lbaf14LTbb8cqckT8-lW~OmwRtaub8JVMqEQAZcg0t0byq~pILEmvLi0Lb3eG-HiPfAKDmPhCntOei7PVLAoj35QQwBrhOT2kmMAW0rZaL4RwBu9uxmd2tJeeb477CbMITpuiVE6XODSyq1Tw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055361&amp;ar=2901809&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw058f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Overview of lncRNA analysis. <strong>(A)</strong> Samples analysed by 76PE NGS and their grouping for assembling transcript models. <strong>(B)</strong> Workflow of identification of lncRNA clusters from oocytes and early embryos NGS data. <strong>(C)</strong> Classification system used for annotating lncRNA transcript models and clusters. <strong>(D)</strong> Non-annotated lncRNA loci identified in this study. Exons from transcript models from each clusters were compared with data in ENCODE and Refseq databases. Clusters, in which none of the exon-exon junctions from transcript models matched an exon-exon junction annotated in these databases were considered novel. <strong>(E)</strong> Origin of transcript models in lncRNA clusters. The Venn diagram depicts which set produced transcript models for lncRNA clusters. For example, 19 lncRNA clusters contain transcript models assembled in all three developmental sets while 973 clusters comprise of exclusively oocyte-derived transcript models.</p></div></div></div>                            <h4 scrollto-destination=78055362 id="78055362" class="section-title" >2.4.3. Comparison with published transcript annotations</h4>
<p>We considered an lncRNA locus to be supported by a previous annotation<a reveal-id="dsw058-B9" data-open="dsw058-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsw058-B11" data-open="dsw058-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> if it shared at least one exon-exon junction with an annotated transcript model. For analysing exon counts and locus lengths, we grouped all the overlapping transcripts for each OET lncRNA locus and calculated the number of unique exons in the transcripts and the length of the transcript group (defined as the distance between the 5′ end of the most upstream 5′ exon in the transcript group to the 3′ end of the most downstream exon in the transcript group). We then compared the number of unique exons and length of the transcript groups to the number of unique exons and the length of our OET lncRNA loci (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>).</p>                            <h4 scrollto-destination=78055364 id="78055364" class="section-title" >2.4.4. Transcriptome analysis of Dicer and AGO2 mutant oocytes</h4>
<p>The NGS data from Stein et al.<a reveal-id="dsw058-B22" data-open="dsw058-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> were mapped to the mm9 version of the mouse genome using the STAR aligner, with the following parameters: –outFilterMultimapNmax 10, –out FilterMismatch Nover Lmax 0.2, –sjdbScore 2. The genome was indexed, prior to mapping, with the addition of the mm9 Ensembl gene annotation. For each gene we chose the transcript model with the higher number of exons and counted the reads using the SummarizeOverlaps function. The estimated expression levels per transcript were obtained by normalizing for the library size using the sizeFactors from the DESeq2 package, and the transcript width.</p>                            <h4 scrollto-destination=78055366 id="78055366" class="section-title" >2.4.5. Mapping and display of 21-23 nt RNAs from oocytes</h4>
<p>Mapping small RNAs from Tam et al.<a reveal-id="dsw058-B23" data-open="dsw058-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> was performed as described previously in.<a reveal-id="dsw058-B24" data-open="dsw058-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> Data were visualized in the UCSC browser by constructing bigWig tracks using the Bedtools software (10.1093/bioinformatics/btq033).</p>                            <h2 scrollto-destination=78055368 id="78055368" class="section-title" >3. Results and Discussion</h2>
                            <h3 scrollto-destination=78055369 id="78055369" class="section-title" >3.1. Identification of OET lncRNAs</h3>
<p>We built lncRNA annotation from 76PE non-directional NGS of total non-amplified RNA from seven different OET stages in mice (<a reveal-id="dsw058-F1" data-open="dsw058-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). Total RNA NGS enabled us to explore the entire lncRNA population including non-polyadenylated lncRNAs. Given the depth of our NGS datasets (∼4–9 × 10<sup>6</sup> mapped non-rRNA reads, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>A), we focused on annotating well-expressed lncRNAs to reduce annotation artifacts emerging when annotating as many loci as possible. The annotation pipeline (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. 1B</a></span>) started with grouping NGS data into three biologically relevant datasets representing distinct types of OET transcriptomes: (i) <strong>maternal</strong>, containing data (∼16.1 × 10<sup>6</sup> mapped non-rRNA reads) from ovarian GV oocytes and ovulated MII eggs. (ii) ZGA (also referred to as ‘<strong>zygotic</strong>’)—containing two- and four-cell embryo data (∼10.0 × 10<sup>6</sup> mapped non-rRNA reads), and (iii) late preimplantation embryo (also referred to as ‘<strong>embryonic</strong>’)—containing morula and blastocyst data (∼9.3 × 10<sup>6</sup> mapped non-rRNA reads). The maternal, zygotic, and embryonic datasets were used for a separate transcript model assembly, filtering, and clustering (a cluster is a group of exons from clustered transcript models from one locus). Clusters containing a transcript model with a predicted coding potential were removed. Clusters from the maternal, zygotic, and embryonic datasets were merged and refined (including a manual inspection of ∼1,200 clusters) to produce a non-redundant set of transcript model clusters. In total, we obtained 1,600 lncRNA clusters classified into four categories according to their positions relative to protein-coding genes (<a reveal-id="dsw058-F1" data-open="dsw058-F1" class="link link-reveal link-table xref-fig">Fig. 1C</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). 1,142 (71%) lncRNA loci were intergenic, 30 (2%) overlapped with more than one protein-coding gene, 193 (12%) resided within introns, and 235 (15%) contained transcript models antisense overlapping with mRNA exons.</p><p>The majority of the 1,600 lncRNA clusters were assembled from the maternal NGS set (<a reveal-id="dsw058-F1" data-open="dsw058-F1" class="link link-reveal link-table xref-fig">Fig. 1D</a>), including 973 clusters assembled exclusively from the maternal set. Around 600 clusters were assembled from the ZGA set (362 clusters exclusively from the ZGA set), while mere 93 lncRNA clusters were assembled from the embryonic set (only 24 of those clusters were exclusively from the embryonic set). This result, which is discordant with the high count of lncRNAs annotated from ESCs,<a reveal-id="dsw058-B25" data-open="dsw058-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a><sup>,</sup><a reveal-id="dsw058-B26" data-open="dsw058-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> can be explained by sample heterogeneity and changing RNA content during early development. Abundance of lncRNAs specific for embryonic and extraembryonic lineages in morulae and blastocysts would be diluted by transcriptomes of non-expressing cells. Furthermore, the embryonic set had a lower depth than the maternal set (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>) while a blastocyst has ∼ three times more total RNA (∼1.5 ng) than an oocyte (∼0.5 ng).<a reveal-id="dsw058-B27" data-open="dsw058-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> Accordingly, a transcript with an identical copy number in the blastocyst and the oocyte has a three times lower FPKM value in the blastocyst. Thus, the same FPKM cut-off value is more stringent when selecting the blastocyst-expressed genes. This would especially affect the analysis of low-level transcripts such as lncRNAs.</p><p>Of the 1,600 clusters, 350 (22%) clusters contained exons or transcript models annotated in the ENCODE or Refseq databases (<a reveal-id="dsw058-F1" data-open="dsw058-F1" class="link link-reveal link-table xref-fig">Fig. 1E</a>). A comparison with recently published 19,617 non-coding transcript models<a reveal-id="dsw058-B11" data-open="dsw058-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> and 5,563 lncRNA transcripts from 3,492 loci<a reveal-id="dsw058-B9" data-open="dsw058-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> showed that 24% (390 lncRNA loci) were not annotated by either of these studies (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>), 44% were supported exclusively by Veselovska et al.,<a reveal-id="dsw058-B11" data-open="dsw058-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> 7% exclusively by Zhang et al.,<a reveal-id="dsw058-B9" data-open="dsw058-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> and 25% by both datasets. Thus, almost 70% of the clusters overlapped with Veselovska et al. who used a much deeper maternal NGS set and annotated longer transcript models with more splicing variants (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). In contrast, lncRNA loci by Zhang et al. were shorter and had fewer exons per transcript and per lncRNA locus (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>).</p><p>Finally, we examined expression of annotated lncRNA loci (clusters) in other available data: SOLiD NGS data by Park et al., which we used previously in,<a reveal-id="dsw058-B12" data-open="dsw058-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsw058-B17" data-open="dsw058-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> and unpublished data by Yu et al. (GSE71257) from GV oocytes, MII eggs, one-, and two-cell stages sequenced on the Illumina HiSeq 2500 platform. We found that only 34 of our lncRNA clusters produced transcript models with expression values &lt;1 FPKM (maximum exon FPKM for each OET lncRNA locus); expression of four clusters was not detected at all in the examined datasets (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Altogether, these data suggest that our OET lncRNA clusters form a high-quality collection of the most expressed maternal and zygotic lncRNAs.</p>                            <h3 scrollto-destination=78055374 id="78055374" class="section-title" >3.2. Structure and expression of OET lncRNAs</h3>
<div class="block-child-p">OET lncRNA loci were stochastically distributed across the genome (<a reveal-id="dsw058-F2" data-open="dsw058-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). The highest density of lncRNA loci was on chromosome 10 (0.98 lncRNA/Mb) while the lowest density on chromosome 17 (0.18 lncRNA/Mb), which contained just 17 lncRNAs (<a reveal-id="dsw058-F2" data-open="dsw058-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). These densities significantly differed (Holm corrected <em>t</em>-test <em>P</em>-value &lt; 10<sup>−</sup><sup>6</sup>) from the mean lncRNA density across all chromosomes (0.61 lncRNA/Mb) while protein-coding gene density did not (Holm corrected <em>t</em>-test <em>P</em>-value &gt; 0.05). <div data-id="dsw058-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/m_dsw058f2.png?Expires=2147483647&Signature=MukDKxph2-VI1Rb9epsbIQw5iN6tm74VOHIKeb9tQf5EqaHt-0Ogl327pTAOuiKeNk8~43wcywugLyUbgh~fUhuPE9qFGzd4KKBzLqD7GLvLu7mE~10q4so1IuZVuAkY42r6GmJND1hmTe-1~V8Gf65GIX08uUtmIWM4PCmuDyXDANOixB~hWFo335gISuopXx6vYXCFp7YrN8sjqXFlz3xC8-giyc~7laiwVM17-~aRaQgSAgayT3KVxcQ3nDwGPEHw98d76A2Z~KLy~~PIq5s-~jEzHoI63~AUO3PQyuw544FHhOJc7AKb6wKoLVaIQAhocAwTUmPW9MbriBZBeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Structural features of OET lncRNAs. (A) Genomic distribution of 1,600 OET lncRNA loci across the mouse genome. The color-coding indicates the highest expression (maternal, GV oocyte or MII egg; zygotic, two- or four-cell stage; late preimplantation, morula or blastocyst). (B) Number of exons in OET lncRNAs; (C) OET lncRNA locus lengths; (D) Median transcript length produced from an OET lncRNA locus; (D) Length distribution of OET lncRNA exons and introns; (F) Distribution of LTR-derived first exon sequences. (B–F) All features of OET lncRNAs (depicted in red) are compared with oocyte mRNA data (depicted in blue). (G) Contribution of retrotransposons to OET lncRNA transcriptional regulation. The graph depicts fractions of 5′ OET lncRNA exons, which contain a given type of a repetitive sequence over the putative transcription start site and 50 bp upstream. (I) Contribution of repetitive sequences to mature (spliced) OET lncRNA sequences." path-from-xml="dsw058f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055375/dsw058f2.tif" path-from-xml="dsw058f2.tif" target="_blank">View large</a><a section="78055375" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058f2.png?Expires=2147483647&Signature=aIutpITNhb~qmIUF68ku6RoKO1oYqftKlCZOeeeC~CeFQF6sLdpCL6kzFNKc4hA6FgWkO6USLrnZUdbSKNk~i7c46T94IYxKRTBLI8NQXEXCwmw8qcWsx5pbpCza6ahhMQBaIJyTjSGGnHFGi0NJjv59sI~~-WbbuQFF78R52e8ALBdMzHQ7lqqNOVrriw4STuchuTcq-STwnlCp4YTm1NAGw3Psc8wdWVtCy0tQuTLUPhIzHSguN5l00sRB2bEth9a8U4G1Gooi1HLxiclPvdY4~TBXtRuX-ovkkU3Dp~ErtPknznmqQRBZBIQvFE5ZUy65O8AfCePkBiUxS3lsRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055375&ar=2901809&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw058f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Structural features of OET lncRNAs. <strong>(A)</strong> Genomic distribution of 1,600 OET lncRNA loci across the mouse genome. The color-coding indicates the highest expression (maternal, GV oocyte or MII egg; zygotic, two- or four-cell stage; late preimplantation, morula or blastocyst). <strong>(B)</strong> Number of exons in OET lncRNAs; <strong>(C)</strong> OET lncRNA locus lengths; <strong>(D)</strong> Median transcript length produced from an OET lncRNA locus; <strong>(D)</strong> Length distribution of OET lncRNA exons and introns; <strong>(F)</strong> Distribution of LTR-derived first exon sequences. (B–F) All features of OET lncRNAs (depicted in red) are compared with oocyte mRNA data (depicted in blue). <strong>(G)</strong> Contribution of retrotransposons to OET lncRNA transcriptional regulation. The graph depicts fractions of 5′ OET lncRNA exons, which contain a given type of a repetitive sequence over the putative transcription start site and 50 bp upstream. <strong>(I)</strong> Contribution of repetitive sequences to mature (spliced) OET lncRNA sequences.</p></div></div><div content-id="dsw058-F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/m_dsw058f2.png?Expires=2147483647&Signature=MukDKxph2-VI1Rb9epsbIQw5iN6tm74VOHIKeb9tQf5EqaHt-0Ogl327pTAOuiKeNk8~43wcywugLyUbgh~fUhuPE9qFGzd4KKBzLqD7GLvLu7mE~10q4so1IuZVuAkY42r6GmJND1hmTe-1~V8Gf65GIX08uUtmIWM4PCmuDyXDANOixB~hWFo335gISuopXx6vYXCFp7YrN8sjqXFlz3xC8-giyc~7laiwVM17-~aRaQgSAgayT3KVxcQ3nDwGPEHw98d76A2Z~KLy~~PIq5s-~jEzHoI63~AUO3PQyuw544FHhOJc7AKb6wKoLVaIQAhocAwTUmPW9MbriBZBeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Structural features of OET lncRNAs. (A) Genomic distribution of 1,600 OET lncRNA loci across the mouse genome. The color-coding indicates the highest expression (maternal, GV oocyte or MII egg; zygotic, two- or four-cell stage; late preimplantation, morula or blastocyst). (B) Number of exons in OET lncRNAs; (C) OET lncRNA locus lengths; (D) Median transcript length produced from an OET lncRNA locus; (D) Length distribution of OET lncRNA exons and introns; (F) Distribution of LTR-derived first exon sequences. (B–F) All features of OET lncRNAs (depicted in red) are compared with oocyte mRNA data (depicted in blue). (G) Contribution of retrotransposons to OET lncRNA transcriptional regulation. The graph depicts fractions of 5′ OET lncRNA exons, which contain a given type of a repetitive sequence over the putative transcription start site and 50 bp upstream. (I) Contribution of repetitive sequences to mature (spliced) OET lncRNA sequences." path-from-xml="dsw058f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055375/dsw058f2.tif" path-from-xml="dsw058f2.tif" target="_blank">View large</a><a section="78055375" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058f2.png?Expires=2147483647&Signature=aIutpITNhb~qmIUF68ku6RoKO1oYqftKlCZOeeeC~CeFQF6sLdpCL6kzFNKc4hA6FgWkO6USLrnZUdbSKNk~i7c46T94IYxKRTBLI8NQXEXCwmw8qcWsx5pbpCza6ahhMQBaIJyTjSGGnHFGi0NJjv59sI~~-WbbuQFF78R52e8ALBdMzHQ7lqqNOVrriw4STuchuTcq-STwnlCp4YTm1NAGw3Psc8wdWVtCy0tQuTLUPhIzHSguN5l00sRB2bEth9a8U4G1Gooi1HLxiclPvdY4~TBXtRuX-ovkkU3Dp~ErtPknznmqQRBZBIQvFE5ZUy65O8AfCePkBiUxS3lsRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055375&ar=2901809&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw058f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Structural features of OET lncRNAs. <strong>(A)</strong> Genomic distribution of 1,600 OET lncRNA loci across the mouse genome. The color-coding indicates the highest expression (maternal, GV oocyte or MII egg; zygotic, two- or four-cell stage; late preimplantation, morula or blastocyst). <strong>(B)</strong> Number of exons in OET lncRNAs; <strong>(C)</strong> OET lncRNA locus lengths; <strong>(D)</strong> Median transcript length produced from an OET lncRNA locus; <strong>(D)</strong> Length distribution of OET lncRNA exons and introns; <strong>(F)</strong> Distribution of LTR-derived first exon sequences. (B–F) All features of OET lncRNAs (depicted in red) are compared with oocyte mRNA data (depicted in blue). <strong>(G)</strong> Contribution of retrotransposons to OET lncRNA transcriptional regulation. The graph depicts fractions of 5′ OET lncRNA exons, which contain a given type of a repetitive sequence over the putative transcription start site and 50 bp upstream. <strong>(I)</strong> Contribution of repetitive sequences to mature (spliced) OET lncRNA sequences.</p></div></div></div><p>When compared with maternal mRNAs, loci encoding lncRNAs were shorter and produced shorter transcripts (<a reveal-id="dsw058-F2" data-open="dsw058-F2" class="link link-reveal link-table xref-fig">Fig. 2B–D</a>); this difference could be attributed to a higher number of exons in mRNAs. OET lncRNAs had markedly shorter 5′ exons but lengths of internal and 3′ exons of mRNAs and lncRNAs were much closer to each other (<a reveal-id="dsw058-F2" data-open="dsw058-F2" class="link link-reveal link-table xref-fig">Fig. 2E</a>). Shorter 5′ exons also came from long terminal repeat (LTR) retrotransposons, which gave rise to ∼third of 5′ exons (<a reveal-id="dsw058-F1 dsw058-F2" data-open="dsw058-F1 dsw058-F2" class="link link-reveal link-table xref-fig">Fig. 2G and H</a>). The analysis of contribution of repetitive elements to lncRNA genes showed that SINE and LINE elements contributed to mature lncRNA sequences more often than to lncRNA promoters and transcription start sites. In contrast, LTR elements, especially the MaLR class, made a strong contribution to lncRNA promoters (<a reveal-id="dsw058-F1 dsw058-F2" data-open="dsw058-F1 dsw058-F2" class="link link-reveal link-table xref-fig">Fig. 2G and H</a>). A similar observation was also made by Veselovska et al.<a reveal-id="dsw058-B11" data-open="dsw058-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a></p><div class="block-child-p">The most-expressed 1,600 OET lncRNAs were more than an order of magnitude less expressed than the 1,600 most expressed OET mRNAs (<a reveal-id="dsw058-F3" data-open="dsw058-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>), which is consistent with low lncRNA expression reported elsewhere.<a reveal-id="dsw058-B19" data-open="dsw058-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><sup>,</sup><a reveal-id="dsw058-B25" data-open="dsw058-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a><sup>,</sup><a reveal-id="dsw058-B28 dsw058-B29 dsw058-B30" data-open="dsw058-B28 dsw058-B29 dsw058-B30" class="link link-ref link-reveal xref-bibr"><sup>28–30</sup></a> It is possible that OET lncRNAs evolved to have lower expression than mRNAs. Lower lncRNA expression could stem from different requirements for functioning than those applied to mRNAs. Alternatively, lower lncRNA expression could reflect a minimal selective pressure on high expression levels of evolving lncRNAs lacking a function. At the same time, highest expressed mRNAs may represent a derived trait, which questions whether the comparison reflects properties of highly expressed mRNAs or an lncRNA feature. If there were no major difference in control of expression between lncRNAs and mRNAs, a random set of 1,600 mRNAs would have expression similar to OET lncRNAs. Thus, we compared expression of 1,600 OET lncRNAs with 1,000 random selections of 1,600 OET mRNAs. We observed that mRNAs generally retained higher expression than lncRNAs while levels of both types of RNAs were essentially the same within the least expressed quartile (<a reveal-id="dsw058-F3" data-open="dsw058-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a><a reveal-id="dsw058-B56" data-open="dsw058-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a>). Whether the differences in expression levels between lncRNAs and mRNAs reflect lncRNA-specific features in transcriptional or post-transcriptional regulations remains unclear. Given the arbitrary definition and functional heterogeneity of lncRNAs, it is difficult to envision some feature underlying lower lncRNA expression levels except for one: a lack of function. Expression of non-functional lncRNAs would not be maintained and would most likely decline over time due to mutations affecting transcriptional control elements. <div data-id="dsw058-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/m_dsw058f3.png?Expires=2147483647&amp;Signature=sFVC4QRSaI~X8DeP97r17kG5xyCerzE2oK0jHvFWqOvPqUfQhuYE48oXabOB0WN~ZtnioomWp~lSTTA2HNbpQPb4A71wnrbADOJ-V4nrpG~2PWAuqb4mSWxd1Tx7RG4gwPEEHd~iCIYM9RqKVLYwpDDXKSMgmtdAlSu-pYCNy-OXCuhjZS8nOtqdOOHksTUV3uHG5KQBit23wCk-KxLwVfhxOyoN6TtCzxJc8PKYwRZVXO~yFXOuntVcorU52-mw~vFTy6l1NG0Qn3NjzSdDA0bhlnkx70KOGiSLA~qaj9y9tshcw256FcOc7c5T3AJrGqR1ZsN3ajo~~n3ux5vlTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression features of OET lncRNAs. (A) Comparison of OET lncRNA expression levels with OET mRNAs. The Y axis shows log10 FPKM expression, the Y axis are rank-sorted RNAs as follows: blue, 1,600 most expressed maternal RNAs; red, 1,600 OET lncRNAs. The broad grey curve represents values for thousand random selections of 1,600 RNAs. The FPKM values were calculated as the maximum exon FPKM per cluster (B) Distribution of polyA scores in GV and MII oocytes. The Y axis depicts the polyA score calculated as the ratio of polyA NGS FPKM/total RNA FPKM. PolyA RNA data for GV oocytes were taken from the literature,31 for MII polyA RNA we used our own data (Supplementary Table S1). The X axis represents rank-sorted OET lncRNAs that had FPKM  &gt;0 (1131 for GV and 1273 for MII). Dashed red and blue lines represent median polyA score values for lncRNAs and mRNAs, respectively. Green points on the right site indicate polyA scores of histone mRNAS, black points on the curve indicate the rank of polyA scores of dormant maternal mRNAs. (C) Examples of putative CPE elements found among OET lncRNAs. (D) polyadenylation changes upon fertilization. The Y axis depicts relative changes of gene expression in total RNA upon fertilization (log2[1-cell/MII total RNA FPKM]), the X axis shows relative changes in polyA RNA (log2[1-cell/MII polyA FPKM]). The left scatterplot displays only OET lncRNAs, the right plot shows OET lncRNAs in red superimposed onto mRNAs (black). Each point represents expression of one gene." path-from-xml="dsw058f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055377/dsw058f3.tif" path-from-xml="dsw058f3.tif" target="_blank">View large</a><a section="78055377" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058f3.png?Expires=2147483647&amp;Signature=ONeGdGe4LGIFgZjpuApKcW0Ix9PVjfEsaAMXS8RxwmWRpJWLNkNKbL39042O8EUi10pv~k1VhmJB-Fbb95djwd6yV-uN~x4k3Gs6IpHj0NA~S5DgWMzJcu~IrnmQJSIneyNr6u85Z6kHkMNQ5np1tQBSzkurj-FXqZZ~NzUHvGGzxficUGvNT9AqqaYd-bZvezX9itLMrZT4FQeCnGHnbPbm4-WaUBX3hi7LyQXsGhsx5zR-xru~MWWTrT4r21l4X1WKjohzPe5VtHb2VdBUXhXa7qjTCGNOAD5FOezhD~VaPlzYxryqgHJGd9QPPuz~E43CZO8NTi0ZgdDzbiDpEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055377&amp;ar=2901809&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw058f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression features of OET lncRNAs. <strong>(A)</strong> Comparison of OET lncRNA expression levels with OET mRNAs. The Y axis shows log<sub>10</sub> FPKM expression, the Y axis are rank-sorted RNAs as follows: blue, 1,600 most expressed maternal RNAs; red, 1,600 OET lncRNAs. The broad grey curve represents values for thousand random selections of 1,600 RNAs. The FPKM values were calculated as the maximum exon FPKM per cluster <strong>(B)</strong> Distribution of polyA scores in GV and MII oocytes. The Y axis depicts the polyA score calculated as the ratio of polyA NGS FPKM/total RNA FPKM. PolyA RNA data for GV oocytes were taken from the literature,<a reveal-id="dsw058-B31" data-open="dsw058-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> for MII polyA RNA we used our own data (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&amp;Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). The X axis represents rank-sorted OET lncRNAs that had FPKM  &gt;0 (1131 for GV and 1273 for MII). Dashed red and blue lines represent median polyA score values for lncRNAs and mRNAs, respectively. Green points on the right site indicate polyA scores of histone mRNAS, black points on the curve indicate the rank of polyA scores of dormant maternal mRNAs. <strong>(C)</strong> Examples of putative CPE elements found among OET lncRNAs. <strong>(D)</strong> polyadenylation changes upon fertilization. The Y axis depicts relative changes of gene expression in total RNA upon fertilization (log<sub>2</sub>[1-cell/MII total RNA FPKM]), the X axis shows relative changes in polyA RNA (log<sub>2</sub>[1-cell/MII polyA FPKM]). The left scatterplot displays only OET lncRNAs, the right plot shows OET lncRNAs in red superimposed onto mRNAs (black). Each point represents expression of one gene.</p></div></div><div content-id="dsw058-F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/m_dsw058f3.png?Expires=2147483647&amp;Signature=sFVC4QRSaI~X8DeP97r17kG5xyCerzE2oK0jHvFWqOvPqUfQhuYE48oXabOB0WN~ZtnioomWp~lSTTA2HNbpQPb4A71wnrbADOJ-V4nrpG~2PWAuqb4mSWxd1Tx7RG4gwPEEHd~iCIYM9RqKVLYwpDDXKSMgmtdAlSu-pYCNy-OXCuhjZS8nOtqdOOHksTUV3uHG5KQBit23wCk-KxLwVfhxOyoN6TtCzxJc8PKYwRZVXO~yFXOuntVcorU52-mw~vFTy6l1NG0Qn3NjzSdDA0bhlnkx70KOGiSLA~qaj9y9tshcw256FcOc7c5T3AJrGqR1ZsN3ajo~~n3ux5vlTQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression features of OET lncRNAs. (A) Comparison of OET lncRNA expression levels with OET mRNAs. The Y axis shows log10 FPKM expression, the Y axis are rank-sorted RNAs as follows: blue, 1,600 most expressed maternal RNAs; red, 1,600 OET lncRNAs. The broad grey curve represents values for thousand random selections of 1,600 RNAs. The FPKM values were calculated as the maximum exon FPKM per cluster (B) Distribution of polyA scores in GV and MII oocytes. The Y axis depicts the polyA score calculated as the ratio of polyA NGS FPKM/total RNA FPKM. PolyA RNA data for GV oocytes were taken from the literature,31 for MII polyA RNA we used our own data (Supplementary Table S1). The X axis represents rank-sorted OET lncRNAs that had FPKM  &gt;0 (1131 for GV and 1273 for MII). Dashed red and blue lines represent median polyA score values for lncRNAs and mRNAs, respectively. Green points on the right site indicate polyA scores of histone mRNAS, black points on the curve indicate the rank of polyA scores of dormant maternal mRNAs. (C) Examples of putative CPE elements found among OET lncRNAs. (D) polyadenylation changes upon fertilization. The Y axis depicts relative changes of gene expression in total RNA upon fertilization (log2[1-cell/MII total RNA FPKM]), the X axis shows relative changes in polyA RNA (log2[1-cell/MII polyA FPKM]). The left scatterplot displays only OET lncRNAs, the right plot shows OET lncRNAs in red superimposed onto mRNAs (black). Each point represents expression of one gene." path-from-xml="dsw058f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055377/dsw058f3.tif" path-from-xml="dsw058f3.tif" target="_blank">View large</a><a section="78055377" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058f3.png?Expires=2147483647&amp;Signature=ONeGdGe4LGIFgZjpuApKcW0Ix9PVjfEsaAMXS8RxwmWRpJWLNkNKbL39042O8EUi10pv~k1VhmJB-Fbb95djwd6yV-uN~x4k3Gs6IpHj0NA~S5DgWMzJcu~IrnmQJSIneyNr6u85Z6kHkMNQ5np1tQBSzkurj-FXqZZ~NzUHvGGzxficUGvNT9AqqaYd-bZvezX9itLMrZT4FQeCnGHnbPbm4-WaUBX3hi7LyQXsGhsx5zR-xru~MWWTrT4r21l4X1WKjohzPe5VtHb2VdBUXhXa7qjTCGNOAD5FOezhD~VaPlzYxryqgHJGd9QPPuz~E43CZO8NTi0ZgdDzbiDpEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055377&amp;ar=2901809&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw058f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression features of OET lncRNAs. <strong>(A)</strong> Comparison of OET lncRNA expression levels with OET mRNAs. The Y axis shows log<sub>10</sub> FPKM expression, the Y axis are rank-sorted RNAs as follows: blue, 1,600 most expressed maternal RNAs; red, 1,600 OET lncRNAs. The broad grey curve represents values for thousand random selections of 1,600 RNAs. The FPKM values were calculated as the maximum exon FPKM per cluster <strong>(B)</strong> Distribution of polyA scores in GV and MII oocytes. The Y axis depicts the polyA score calculated as the ratio of polyA NGS FPKM/total RNA FPKM. PolyA RNA data for GV oocytes were taken from the literature,<a reveal-id="dsw058-B31" data-open="dsw058-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> for MII polyA RNA we used our own data (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&amp;Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). The X axis represents rank-sorted OET lncRNAs that had FPKM  &gt;0 (1131 for GV and 1273 for MII). Dashed red and blue lines represent median polyA score values for lncRNAs and mRNAs, respectively. Green points on the right site indicate polyA scores of histone mRNAS, black points on the curve indicate the rank of polyA scores of dormant maternal mRNAs. <strong>(C)</strong> Examples of putative CPE elements found among OET lncRNAs. <strong>(D)</strong> polyadenylation changes upon fertilization. The Y axis depicts relative changes of gene expression in total RNA upon fertilization (log<sub>2</sub>[1-cell/MII total RNA FPKM]), the X axis shows relative changes in polyA RNA (log<sub>2</sub>[1-cell/MII polyA FPKM]). The left scatterplot displays only OET lncRNAs, the right plot shows OET lncRNAs in red superimposed onto mRNAs (black). Each point represents expression of one gene.</p></div></div></div>                            <h3 scrollto-destination=78055378 id="78055378" class="section-title" >3.3. Polyadenylation of OET lncRNAs</h3>
<p>Biogenesis of lncRNAs and mRNAs is common—they are spliced polymerase II transcripts whose transcription would utilize the same set of transcription factors. One of the features, by which lncRNAs could differ, is polyadenylation of the 3′ end. Consequently, we examined whether a comparison of total RNA and polyA RNA FPKM values would be indicative of the polyadenylation status (polyA FPKM/total RNA FPKM, for simplicity referred as polyA score, <a reveal-id="dsw058-F3" data-open="dsw058-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). GV oocytes and MII eggs are an excellent model system for testing this idea because of two possible internal controls: (i) Replication dependent histone mRNAs carrying at their 3′ ends unique stem loop structures instead of polyA tails (reviewed in<a reveal-id="dsw058-B32" data-open="dsw058-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a>). Presence of these transcripts within polyA-selected RNA would reflect the extent of contamination with non-polyadenylated mRNA. (ii) Dormant maternal mRNAs, translationally repressed mRNAs with short polyA tails stored in the GV oocyte, which are readenylated and translated during meiotic maturation (reviewed in<a reveal-id="dsw058-B33" data-open="dsw058-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a>). Thus, we selected 20 highly expressed replication-dependent histone genes lacking alternative polyadenylated transcript isoforms and five dormant maternal mRNAs, for which the cytoplasmic polyadenylation during meiosis was demonstrated: <em>Mos</em>, <em>Plat</em>, <em>Cyclin B1</em>, <em>Orc6l</em>, and <em>Dcp1a</em>.<a reveal-id="dsw058-B34 dsw058-B35 dsw058-B36 dsw058-B37 dsw058-B38" data-open="dsw058-B34 dsw058-B35 dsw058-B36 dsw058-B37 dsw058-B38" class="link link-ref link-reveal xref-bibr"><sup>34–38</sup></a></p><p>The distribution of the polyA score for lncRNAs from GV and MII stages yielded sigmoidal curves with slightly different slopes (<a reveal-id="dsw058-F3" data-open="dsw058-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>), which were reproduced with mRNA data (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>A). Although the difference in slopes might reflect intrinsic differences of GV and MII transcriptomes, polyA scores accurately reflected the lack of polyadenylation of histone mRNAs and cytoplasmic polyadenylation of dormant maternal mRNAs during meiotic maturation (<a reveal-id="dsw058-F3" data-open="dsw058-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary S5A</a></span>). The average polyA score of dormant maternal mRNAs, which was 1.106 in GV oocytes, increased to 7.895 in MII eggs. This difference also manifested as a shift of dormant mRNAs along the polyA score rank (<a reveal-id="dsw058-F3" data-open="dsw058-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>A). Taken together, the behavior of polyA scores appeared indicative of the polyA status. Based on this strategy, we estimated that only a minority of OET lncRNAs lacked the polyA tail. Since lncRNAs detectable in oocytes and/or two-cell zygotes (&gt;1 FPKM) represented ∼95% of the 1,600 lncRNAs, we added normalized polyA scores (median = 0 and variance = 1) from GV oocytes, MII eggs, one-, two-, and four-cell embryos to lncRNA annotation (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). However, given the diverse origin of total RNA and polyA RNA NGS data, polyA scores should be taken as an indication rather than an annotation of the polyadenylation status.</p><p>The dynamics of polyA scores of dormant maternal mRNAs raised a question whether similar behavior could also be found among maternal lncRNAs. Remarkably, we identified 91 maternal lncRNAs with expression &gt;1 FPKM whose polyA scores increased more than 5-fold during meiosis. Next, we analysed which of the 91 maternal lncRNAs carried putative cytoplasmic polyadenylation elements (CPEs). CPEs mediate the recruitment of dormant deadenylated mRNAs for translation. A CPE is bound by a CPE-binding protein (CPEB), which recognizes a canonical consensus site 5′-UUUUUAU-3′ art the 3′ end of RNAs. However, CPE variations such as UUUUAU, UUUUUAAU, or UUUUAACA were also reported.<a reveal-id="dsw058-B39" data-open="dsw058-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> We found that at least 14 lncRNAs carried a combination of a canonical AAUAAA signal site and a CPE-like motif at their 3′ ends (<a reveal-id="dsw058-F3" data-open="dsw058-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a>). lncRNAs resembling dormant maternal RNAs are remarkable because they suggest that cytoplasmic polyadenylation and dormancy could play a more general role in RNA regulation, i.e. a role that goes beyond translational control of maternal mRNAs. We hypothesize that controlled cytoplasmic polyadenylation during transcriptional quiescence could ‘activate’ stored maternal lncRNAs. Thus, putative dormant maternal lncRNAs represent excellent candidates for further functional studies of lncRNAs functioning between ovulation and ZGA.</p><p>Importantly, major changes in cytoplasmic RNA polyadenylation during OET occur also post-fertilization.<a reveal-id="dsw058-B40" data-open="dsw058-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a><sup>,</sup><a reveal-id="dsw058-B41" data-open="dsw058-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> In fact, cytoplasmic RNA polyadenylation in fertilized mouse eggs is so extensive that it manifests as an increase in total polyA RNA content.<a reveal-id="dsw058-B27" data-open="dsw058-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> A scatter plot of relative changes of lncRNAs in polyA and total RNA NGS sets in MII eggs and one-cell embryos showed a relative enrichment in lncRNA polyadenylation upon fertilization; this was similar to changes observed for mRNAs (<a reveal-id="dsw058-F3" data-open="dsw058-F3" class="link link-reveal link-table xref-fig">Fig. 3D</a>). In contrast to mRNAs, the number of lncRNAs showing a stronger decrease in polyA RNA upon fertilization was minimal. Increased polyadenylation did not seem to be an artifact of our samples, as it also showed when other published data were used (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>B).</p><p>Taken together, our data suggest that the bulk of OET lncRNAs are polyadenylated at their 3′ end and that maternal lncRNAs utilize the same cytoplasmic polyadenylation mechanisms as mRNAs. In case of mRNAs, cytoplasmic polyadenylation regulates translation and RNA turnover. Although lncRNAs seem to be recognized by the translation machinery,<a reveal-id="dsw058-B42" data-open="dsw058-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a><sup>,</sup><a reveal-id="dsw058-B43" data-open="dsw058-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> their coding capacity is highly restricted. Therefore, the interaction of lncRNAs with cytoplasmic polyadenylation and translation machinery likely regulates their functional availability and stability.</p>                            <h3 scrollto-destination=78055384 id="78055384" class="section-title" >3.4. Expression of OET lncRNAs in other tissues</h3>
<div class="block-child-p">Since the bulk of the 1,600 OET lncRNAs appeared polyadenylated, we examined their expression across 22 tissues selected from the ENCODE polyA RNA NGS mouse tissue panel (GSE49417<a reveal-id="dsw058-B44" data-open="dsw058-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a>). To increase the specificity of expression analysis, we included only clusters with four or more spliced reads in the tissue panel and expression &gt;4 FPKM in at least one of the tissues. The cut-off 4 FPKM for polyA RNA was used because it is an approximate of 1 FPKM in total RNA NGS from mouse oocytes, where mRNAs make 24% of all reads (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&amp;Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). Under these filtering conditions, we obtained expression values for 356 clusters (<a reveal-id="dsw058-F4" data-open="dsw058-F4" class="link link-reveal link-table xref-fig">Fig. 4 and</a><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&amp;Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). The analysis revealed a small population of ubiquitously expressed lncRNA clusters (28 having expression &gt;4 FPKM in all tissues, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&amp;Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). This is consistent with the notion that mammalian lncRNAs typically have a cell type-restricted expression.<a reveal-id="dsw058-B29" data-open="dsw058-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> Of the 28 lncRNA clusters ubiquitously expressed &gt;4 FPKM, 26 were annotated; they were from small nucleolar RNA host genes and other lncRNAs, such as <em>Malat</em>, <em>Firre</em>, or <em>Rian</em>. Remarkably, OET lncRNAs were mostly expressed in the testis. Within the tissue panel (which also included the ovary and the placenta), the testis stood out as the tissue that had the highest number of maximum expressions of clusters across tissues (121 clusters, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&amp;Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). The testis also yielded the highest number of expressed clusters among the tissues (202 clusters). The ovary ranked second after the testis, having the maximum expression of 19 clusters, while the total number of ovary-expressed clusters was 110 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&amp;Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). <div data-id="dsw058-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/m_dsw058f4.png?Expires=2147483647&amp;Signature=aJxfIB0N8bbLeapb~WyUsoyAFIkMy-3UDiieQW4Pi5HgFY9PAbE908WktkGDOo88wN4YVaAEIaqC7PVcaxnrNn3q~oQNJT-8-6QQ1vfwMoWg-vj7f9KjAdxSqjVcNypH8~ukD6uygJwlTi1h6KzgAwxld7X~-y9UVD6~t9imzMoY8lHHlIW-ptA6rwaCsdDL1V6c3cd8-3m0qUNu1blh6DUya7ACFwkGb2KrvyHZfk3bhTKIJ7dOb851C3IfagnvJ0~hexbr2cSPnsTUEoHf1Im94TZNHmKw5XXC4cp0wZaBFdoMB6KsYCT7y1kixmLVetZfiuf8RpevA9dsFTZJYQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="OET lncRNA expression in different tissues. The heatmap displays expression of 356 clusters with expression values &gt;4 FPKM in at least one of 22 tissues selected from the ENCODE polyA RNA NGS mouse tissue panel (GSE49417 [37])." path-from-xml="dsw058f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055385/dsw058f4.tif" path-from-xml="dsw058f4.tif" target="_blank">View large</a><a section="78055385" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058f4.png?Expires=2147483647&amp;Signature=I1Gm59ep7nlFuQyzi86Zr22~buGN0JsEMkFETLn0ssxoz2CRhsvbENPAFXTH6ibqXQSGwjPbWN0u1QTCN-XpO8k0Vmk2tsZaEUxezrOPJbboRE-Yvx1f1E3-CmNKFbWRlj9R9aP7PX8Dh0XaEDIuLB7efGMjhSU2bRp5w2KDOkKxYnDgEK1AtRYRc4wnryinRuJiAWwbzWBwb9hKI6HJHag~m0n3bXuNmhkXhXrD76NXSiu2AtQD1kWML9T8ZfiyRxPQwiocRgwf6L-8zDn8tbBR9mW8UkFCyUtQURaNpajr2lc8Np-ZkOy2~ZfouFWa~1fOfVhpABlR~se2HWCGoA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055385&amp;ar=2901809&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw058f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>OET lncRNA expression in different tissues. The heatmap displays expression of 356 clusters with expression values &gt;4 FPKM in at least one of 22 tissues selected from the ENCODE polyA RNA NGS mouse tissue panel (GSE49417 [37]).</p></div></div><div content-id="dsw058-F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/m_dsw058f4.png?Expires=2147483647&amp;Signature=aJxfIB0N8bbLeapb~WyUsoyAFIkMy-3UDiieQW4Pi5HgFY9PAbE908WktkGDOo88wN4YVaAEIaqC7PVcaxnrNn3q~oQNJT-8-6QQ1vfwMoWg-vj7f9KjAdxSqjVcNypH8~ukD6uygJwlTi1h6KzgAwxld7X~-y9UVD6~t9imzMoY8lHHlIW-ptA6rwaCsdDL1V6c3cd8-3m0qUNu1blh6DUya7ACFwkGb2KrvyHZfk3bhTKIJ7dOb851C3IfagnvJ0~hexbr2cSPnsTUEoHf1Im94TZNHmKw5XXC4cp0wZaBFdoMB6KsYCT7y1kixmLVetZfiuf8RpevA9dsFTZJYQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="OET lncRNA expression in different tissues. The heatmap displays expression of 356 clusters with expression values &gt;4 FPKM in at least one of 22 tissues selected from the ENCODE polyA RNA NGS mouse tissue panel (GSE49417 [37])." path-from-xml="dsw058f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055385/dsw058f4.tif" path-from-xml="dsw058f4.tif" target="_blank">View large</a><a section="78055385" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058f4.png?Expires=2147483647&amp;Signature=I1Gm59ep7nlFuQyzi86Zr22~buGN0JsEMkFETLn0ssxoz2CRhsvbENPAFXTH6ibqXQSGwjPbWN0u1QTCN-XpO8k0Vmk2tsZaEUxezrOPJbboRE-Yvx1f1E3-CmNKFbWRlj9R9aP7PX8Dh0XaEDIuLB7efGMjhSU2bRp5w2KDOkKxYnDgEK1AtRYRc4wnryinRuJiAWwbzWBwb9hKI6HJHag~m0n3bXuNmhkXhXrD76NXSiu2AtQD1kWML9T8ZfiyRxPQwiocRgwf6L-8zDn8tbBR9mW8UkFCyUtQURaNpajr2lc8Np-ZkOy2~ZfouFWa~1fOfVhpABlR~se2HWCGoA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055385&amp;ar=2901809&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw058f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>OET lncRNA expression in different tissues. The heatmap displays expression of 356 clusters with expression values &gt;4 FPKM in at least one of 22 tissues selected from the ENCODE polyA RNA NGS mouse tissue panel (GSE49417 [37]).</p></div></div></div><p>LncRNAs expressed in the testis and during OET represent an interesting group of germ-line lncRNAs. We took a closer look at the transcriptional control of the 121 lncRNA clusters expressed &gt;4 FPKMs to determine (i) if they are associated with maternal or zygotic expression, and (ii) whether they share the same promoters in the testis and OET stages or whether they have testis-specific promoters not utilized during OET. Most of the 121 lncRNA clusters during OET were highly expressed maternally (93 clusters), while 25 clusters had the highest expression in the zygotic stage and three clusters in the embryonic stage. We examined the promoters of the 121 clusters and found that approximately half of them (58 of 121 examined promoters) controlled the expression in testes and OET stages. In 37 cases, a unique non-repetitive lncRNA promoter functioned in testes, while oocytes or early embryos employed a different unique promoter (19/37) or a retrotransposon-derived one, such as a maternally active MaLR class LTR promoter (12/37). In any case, the 93 lncRNA loci expressed in oocytes and testes are prime candidates for an analysis of lncRNAs with germline-specific functions.</p>                            <h3 scrollto-destination=78055387 id="78055387" class="section-title" >3.5. lncRNA dynamics during OET</h3>
<p>Gene expression during OET can be divided into three basic classes reflecting the replacement of maternal RNAs with zygotic transcripts:</p><p>(i) maternal—concerns genes active only in oocytes whose transcripts survive until different time-points/stages of OET. They are not replaced with zygotic/embryonic transcripts. These transcripts may be important just for oocyte development or they may function during meiotic maturation or after fertilization, where they might contribute to ZGA and to the initiation of development.</p><p>(ii) zygotic—concerns genes expressed during ZGA and not active in the oocyte. Zygotic transcripts may be made just transiently during ZGA or they may remain expressed during early development. These are represented, for example, by transcripts of genes involved in the establishment and maintenance of totipotency.</p><p>(iii) maternal-zygotic—concerns genes expressed in both oocytes and early embryos. This category can be exemplified by housekeeping genes. Within this category, maternal gene expression may be much higher than that observed in early embryos or vice versa.</p><div class="block-child-p">To characterize lncRNA dynamics during OET, we divided lncRNA clusters into the three classes of gene expression described earlier. First, we reduced the complexity of the model system by stage grouping (as it was used for transcript model assembly) into three basic expression states: maternal (M), zygotic (Z), and embryonic (E). The M level was calculated as an average lncRNA level in GV and MII oocytes; it represented lncRNA expression before fertilization. The Z level was calculated as an average lncRNA level in two- and four-cell stages; it represented the transitive period of gene expression during ZGA. The E level was calculated as an average lncRNA level in morulae and blastocysts; it represented gene expression at a later embryonic stage during which maternal RNAs were cleared up (and so was ZGA-specific expression) and replaced by zygotic/embryonic transcripts. Next, we adjusted M, Z, and E lncRNA values so that the highest value was set to one, and we displayed values for all lncRNA clusters in a single plot (<a reveal-id="dsw058-F5" data-open="dsw058-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>). <div data-id="dsw058-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/m_dsw058f5.png?Expires=2147483647&Signature=J0FszJL3uSVUzThvR5-khdVZYhWnY3oDqf8wLjGjmH9r3E4usIFDAwAnnza1Mk~yNd31Zceuv7dfoKLTPAFC8d0T9u7Mk0wcgM~uTrerprYqwq5aP55v8K322EGKkCR2RYPUj-3sjsLUC3zC6r8~SmLPVAOErwGXN2g6WCWl3~y4OONRq~uJZOpLW4uNw7RafLULhI09Oc90rqauUlAZSXdUSTaOFGgUVGUoZchnAQojGMwVLukipDGCFibNC8HSprm60SAmyx8YMeyQb9ofYOUrZUvSJDJsOqLDpN6-44ZkVGqkrT8EcGmvDNJXoU51b3ZVwFGTu~fWbfhYQun1WA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="OET lncRNA population dynamics during early development. (A) Overview of expression patterns of OET lncRNAs. To simplify expression pattern classification, we used average FPKM values: M, maternal (GV and MII oocytes); Z, zygotic (two- and four-cell stages); and E, late preimplantation embryo (morula and blastocyst). The plot shows dynamics of all clusters where the maximum average FPKM value of each cluster in M, Z, E was set to 1. (B) Main expression patterns of OET lncRNAs. The six panels display six basic patterns separating maternal (top left panel), zygotic (top middle and top right panels), and maternal-zygotic lncRNA (bottom panels) expression. The red lines represent the average values per each panel. (C) Expression patterns of 1,600 OET lncRNAs and 19741 mRNAs. The heatmap for lncRNAs and mRNAs was assembled from the six basic patterns (shown in (B) and schematically depicted between the heatmaps with maternal in red, zygotic in blue and maternal-zygotic in grey). Clusters with M, Z, E maxima were ordered from the left to the right and ranked based on the Z value for M and E patterns and M value for Z patterns. (D) Expression correlations estimated from reads matching different types of sequences—exons of 1,600 lncRNA and exons and introns of protein-coding genes. The color scale on the left indicates the correlation coefficient for the analysed features. Note the negative correlation for lncRNA expression between maternal and zygotic/embryonic stages reflects the apparently mutually exclusive expression patterns observed in the upper heatmap in (C). Temporal expression patterns of miR-290-295 primary precursor (E) and lncRNAs carrying antisense sequences of processed pseudogenes (F). Graphs depict expression values (FPKMs) for indicated lncRNAs. In (F), lncRNAs are labeled by gene names from which the pseudogene sequences in lncRNAs originated." path-from-xml="dsw058f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055392/dsw058f5.tif" path-from-xml="dsw058f5.tif" target="_blank">View large</a><a section="78055392" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058f5.png?Expires=2147483647&Signature=yMYBiSiN~Y4~Xxbq9ONVWRq0ZXknYTMrAkQr25gncxPHk6r8FhOgjSOMUictSZuYQThYZv9kFlh-08MPQgJSxYnnrCUTXe6gTHTTHcM-Q7OhqTA2sgyxLD8UReju9jhB-HunjsvA7rnjIQ-MI98AB~DXNBS6g1U2q6pEUwuR4jyTW5eFPnag2-ilioOgdB3sTJg7LlApD8hsIpmsFseB7zWsTIotXaur1NGIcIi59r9chbaxZre-4tWZT-r8YVKWN6npa8P2UEakoJlveHLtXC4Kj8wLuntkpeY7~69ce7Yh1cm3w0djRFWxyGRVwjiCfOWLpuJyj25IkIHaLBXQtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055392&ar=2901809&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw058f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>OET lncRNA population dynamics during early development. <strong>(A)</strong> Overview of expression patterns of OET lncRNAs. To simplify expression pattern classification, we used average FPKM values: M, maternal (GV and MII oocytes); Z, zygotic (two- and four-cell stages); and E, late preimplantation embryo (morula and blastocyst). The plot shows dynamics of all clusters where the maximum average FPKM value of each cluster in M, Z, E was set to 1. <strong>(B)</strong> Main expression patterns of OET lncRNAs. The six panels display six basic patterns separating maternal (top left panel), zygotic (top middle and top right panels), and maternal-zygotic lncRNA (bottom panels) expression. The red lines represent the average values per each panel. <strong>(C)</strong> Expression patterns of 1,600 OET lncRNAs and 19741 mRNAs. The heatmap for lncRNAs and mRNAs was assembled from the six basic patterns (shown in (B) and schematically depicted between the heatmaps with maternal in red, zygotic in blue and maternal-zygotic in grey). Clusters with M, Z, E maxima were ordered from the left to the right and ranked based on the Z value for M and E patterns and M value for Z patterns. <strong>(D)</strong> Expression correlations estimated from reads matching different types of sequences—exons of 1,600 lncRNA and exons and introns of protein-coding genes. The color scale on the left indicates the correlation coefficient for the analysed features. Note the negative correlation for lncRNA expression between maternal and zygotic/embryonic stages reflects the apparently mutually exclusive expression patterns observed in the upper heatmap in (C). Temporal expression patterns of miR-290-295 primary precursor <strong>(E)</strong> and lncRNAs carrying antisense sequences of processed pseudogenes <strong>(F)</strong>. Graphs depict expression values (FPKMs) for indicated lncRNAs. In (F), lncRNAs are labeled by gene names from which the pseudogene sequences in lncRNAs originated.</p></div></div><div content-id="dsw058-F5" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/m_dsw058f5.png?Expires=2147483647&Signature=J0FszJL3uSVUzThvR5-khdVZYhWnY3oDqf8wLjGjmH9r3E4usIFDAwAnnza1Mk~yNd31Zceuv7dfoKLTPAFC8d0T9u7Mk0wcgM~uTrerprYqwq5aP55v8K322EGKkCR2RYPUj-3sjsLUC3zC6r8~SmLPVAOErwGXN2g6WCWl3~y4OONRq~uJZOpLW4uNw7RafLULhI09Oc90rqauUlAZSXdUSTaOFGgUVGUoZchnAQojGMwVLukipDGCFibNC8HSprm60SAmyx8YMeyQb9ofYOUrZUvSJDJsOqLDpN6-44ZkVGqkrT8EcGmvDNJXoU51b3ZVwFGTu~fWbfhYQun1WA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="OET lncRNA population dynamics during early development. (A) Overview of expression patterns of OET lncRNAs. To simplify expression pattern classification, we used average FPKM values: M, maternal (GV and MII oocytes); Z, zygotic (two- and four-cell stages); and E, late preimplantation embryo (morula and blastocyst). The plot shows dynamics of all clusters where the maximum average FPKM value of each cluster in M, Z, E was set to 1. (B) Main expression patterns of OET lncRNAs. The six panels display six basic patterns separating maternal (top left panel), zygotic (top middle and top right panels), and maternal-zygotic lncRNA (bottom panels) expression. The red lines represent the average values per each panel. (C) Expression patterns of 1,600 OET lncRNAs and 19741 mRNAs. The heatmap for lncRNAs and mRNAs was assembled from the six basic patterns (shown in (B) and schematically depicted between the heatmaps with maternal in red, zygotic in blue and maternal-zygotic in grey). Clusters with M, Z, E maxima were ordered from the left to the right and ranked based on the Z value for M and E patterns and M value for Z patterns. (D) Expression correlations estimated from reads matching different types of sequences—exons of 1,600 lncRNA and exons and introns of protein-coding genes. The color scale on the left indicates the correlation coefficient for the analysed features. Note the negative correlation for lncRNA expression between maternal and zygotic/embryonic stages reflects the apparently mutually exclusive expression patterns observed in the upper heatmap in (C). Temporal expression patterns of miR-290-295 primary precursor (E) and lncRNAs carrying antisense sequences of processed pseudogenes (F). Graphs depict expression values (FPKMs) for indicated lncRNAs. In (F), lncRNAs are labeled by gene names from which the pseudogene sequences in lncRNAs originated." path-from-xml="dsw058f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055392/dsw058f5.tif" path-from-xml="dsw058f5.tif" target="_blank">View large</a><a section="78055392" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058f5.png?Expires=2147483647&Signature=yMYBiSiN~Y4~Xxbq9ONVWRq0ZXknYTMrAkQr25gncxPHk6r8FhOgjSOMUictSZuYQThYZv9kFlh-08MPQgJSxYnnrCUTXe6gTHTTHcM-Q7OhqTA2sgyxLD8UReju9jhB-HunjsvA7rnjIQ-MI98AB~DXNBS6g1U2q6pEUwuR4jyTW5eFPnag2-ilioOgdB3sTJg7LlApD8hsIpmsFseB7zWsTIotXaur1NGIcIi59r9chbaxZre-4tWZT-r8YVKWN6npa8P2UEakoJlveHLtXC4Kj8wLuntkpeY7~69ce7Yh1cm3w0djRFWxyGRVwjiCfOWLpuJyj25IkIHaLBXQtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055392&ar=2901809&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw058f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>OET lncRNA population dynamics during early development. <strong>(A)</strong> Overview of expression patterns of OET lncRNAs. To simplify expression pattern classification, we used average FPKM values: M, maternal (GV and MII oocytes); Z, zygotic (two- and four-cell stages); and E, late preimplantation embryo (morula and blastocyst). The plot shows dynamics of all clusters where the maximum average FPKM value of each cluster in M, Z, E was set to 1. <strong>(B)</strong> Main expression patterns of OET lncRNAs. The six panels display six basic patterns separating maternal (top left panel), zygotic (top middle and top right panels), and maternal-zygotic lncRNA (bottom panels) expression. The red lines represent the average values per each panel. <strong>(C)</strong> Expression patterns of 1,600 OET lncRNAs and 19741 mRNAs. The heatmap for lncRNAs and mRNAs was assembled from the six basic patterns (shown in (B) and schematically depicted between the heatmaps with maternal in red, zygotic in blue and maternal-zygotic in grey). Clusters with M, Z, E maxima were ordered from the left to the right and ranked based on the Z value for M and E patterns and M value for Z patterns. <strong>(D)</strong> Expression correlations estimated from reads matching different types of sequences—exons of 1,600 lncRNA and exons and introns of protein-coding genes. The color scale on the left indicates the correlation coefficient for the analysed features. Note the negative correlation for lncRNA expression between maternal and zygotic/embryonic stages reflects the apparently mutually exclusive expression patterns observed in the upper heatmap in (C). Temporal expression patterns of miR-290-295 primary precursor <strong>(E)</strong> and lncRNAs carrying antisense sequences of processed pseudogenes <strong>(F)</strong>. Graphs depict expression values (FPKMs) for indicated lncRNAs. In (F), lncRNAs are labeled by gene names from which the pseudogene sequences in lncRNAs originated.</p></div></div></div><p>LncRNA expression patterns during OET were classified into six groups matching the maternal, zygotic, and maternal-zygotic expression types (<a reveal-id="dsw058-F5" data-open="dsw058-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>). First, we used the highest expression states to define M, Z, and E groups. Then, we defined maternal lncRNA clusters in the M group as those with a minimal E level (E values &lt;5% of M values), while the remaining lncRNA clusters in the M group were considered maternal-zygotic lncRNA clusters. Analogically, we defined zygotic lncRNA clusters in the Z and E groups as those with a minimal M level (M values &lt;5% of Z or E values), while the remaining lncRNA clusters in the Z and E groups were considered maternal-zygotic lncRNAs. The distribution of maximum values in M, Z, E correlated with the number of clusters defined from maternal, ZGA and embryonic sets (<a reveal-id="dsw058-F1" data-open="dsw058-F1" class="link link-reveal link-table xref-fig">Fig. 1D</a>).</p><p>To obtain a comprehensive view of temporal dynamics of OET lncRNAs, we organized all lncRNAs clusters into a heatmap based on six basic patterns while displaying expression in all sequenced stages (<a reveal-id="dsw058-F5" data-open="dsw058-F5" class="link link-reveal link-table xref-fig">Fig.5C</a>). Most clusters (1,166, displayed on the left) reached a maximum in M. The majority of those declined rapidly during ZGA, reaching low levels in the blastocyst. Of the 1,166 lncRNA cluster with the maximum in M, 993 transcripts exhibited the E value &lt;5% of M. These represent candidates for class I—maternal lncRNA clusters. This class is clearly the most abundant one in our dataset. In total 393 and 131 lncRNA clusters had maximum values in Z and E, respectively; 251 of those had minimal maternal expression (M values &lt;5% of Z values), thus representing class II—zygotic lncRNA clusters (<a reveal-id="dsw058-F5" data-open="dsw058-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a>). Of these, 107 lncRNAs were only transiently expressed during ZGA. 446 lncRNA clusters were considered class III—maternal-zygotic transcripts. Maternal-zygotic lncRNAs could be divided into two categories: (i) those constantly present during OET, i.e. zygotic transcripts appearing before maternal ones were eliminated, and (ii) those whose maternal transcripts were strongly reduced before zygotic/embryonic transcripts emerged—there was a distinct minor group of 59 maternal-zygotic clusters whose expression reached the minimum at the two- and four-cell stages (Z value &gt;0.05 FPKM). The dynamics of lncRNA expression differed from mRNAs (<a reveal-id="dsw058-F5" data-open="dsw058-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a> bottom) mainly in the proportion of maternal and maternal-zygotic expression. Although 62% of the 1,600 OET lncRNA loci were maternal (class I), maternally expressed mRNAs made 20% of all OET mRNAs. Furthermore, maternal-zygotic lncRNAs were a minor fraction of OET lncRNAs (28%), while this class was highly abundant (68%) among mRNAs, which was not surprising considering the multitudes of housekeeping roles of encoded proteins (<a reveal-id="dsw058-F5" data-open="dsw058-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a>).</p><p>Interesting results emerged from an analysis of RNA expression correlations between individual stages (<a reveal-id="dsw058-F5" data-open="dsw058-F5" class="link link-reveal link-table xref-fig">Fig. 5D</a>). We compared expression of lncRNAs and mRNA exons and introns (introns-derived reads reveal the presence of nascent transcripts, i.e. of ongoing transcription<a reveal-id="dsw058-B12" data-open="dsw058-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a>). Remarkably, lncRNAs showed positive correlations among stages with a strong contribution of maternal RNA (GV oocyte and unfertilized/fertilized eggs) and among zygotic stages (two-, four-cell, morula, and blastocyst), but not between any two stages from the two groups. Although two-cell stage lncRNAs showed no correlation with preceding stages, the later stages even had negative correlations (<a reveal-id="dsw058-F5" data-open="dsw058-F5" class="link link-reveal link-table xref-fig">Fig. 5D</a>). This apparently reflected the extensive mutually exclusive nature of maternal and zygotic lncRNA transcriptions, which was also apparent from the expression heatmap (<a reveal-id="dsw058-F5" data-open="dsw058-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a>).</p><p>In contrast, an analysis of exons of protein coding revealed good correlations between any two of the analysed stages (<a reveal-id="dsw058-F5" data-open="dsw058-F5" class="link link-reveal link-table xref-fig">Fig. 5D</a>). This was apparently due to abundant maternal-zygotic expression of protein-coding genes (68% of mRNA transcriptome in <a reveal-id="dsw058-F5" data-open="dsw058-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a>). The maternal-to-zygotic transcriptome transition manifested as higher correlations among stages containing high levels of maternal transcripts (GV oocytes, unfertilized eggs, and fertilized eggs) and among stages expressing zygotic transcripts (two-cell and later). An intron-based analysis showed similar correlations within maternal and zygotic groups but their borderline was shifted to an earlier developmental stage (fertilized egg/one-cell stage), apparently reflecting the minimal amount of intron-derived reads in the maternal transcriptome and the effects of minor ZGA, which took place during the one-cell stage.<a reveal-id="dsw058-B12" data-open="dsw058-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a></p><p>Our results are consistent with a previous observation that lncRNA expression varies at different stages of cleavage stage embryos, suggesting cleavage stage-specific expression.<a reveal-id="dsw058-B9" data-open="dsw058-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Our results show two main expression patterns—maternal expression, which does not come back during early development, and zygotic expression dominated by a transient major ZGA expression pattern. This implies that expression of the bulk of OET lncRNA clusters is driven by oocyte- and ZGA-specific transcription factors, because ubiquitous transcription factors would control only a minority of OET lncRNA clusters. This is consistent with stereotypical observations of high numbers of cell-type-specific lncRNAs.<a reveal-id="dsw058-B29" data-open="dsw058-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> But why would lncRNAs adapt their expression for tissue or stage-specific transcription factors? We speculate that lncRNAs emerge from random transcription of genic and intergenic regions and that this random expression by ubiquitous transcription factors is under stronger selective pressure than expression restricted to a specific cell type/developmental stage.</p>                            <h3 scrollto-destination=78055398 id="78055398" class="section-title" >3.6. Inferring biological roles of OET lncRNAs from NGS data</h3>
<p>The role of most of the 1,600 OET lncRNAs is unknown and it is possible that the majority of them have no function. However, several suggestive observations emerged while annotating OET lncRNA. For example, we found two novel maternally expressed lncRNA clusters located in imprinted loci, whose expressions correlate with the maternal pattern of expression. lncRNA-OET-17-106 overlaps antisense with 3′ end of <em>Airn</em> lncRNA, which is maternally silenced (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>A); lncRNA-OET-12-253 is expressed just downstream of an imprinted miRNA cluster, which is expressed from the maternal allele (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>A). We also found interesting expression patterns in several known lncRNA loci, such as <em>Malat1</em>, <em>Neat1</em>, or <em>Cyrano</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>B). Metastasis-associated lung adenocarcinoma transcript 1 (<em>Malat1</em>, reviewed in<a reveal-id="dsw058-B45" data-open="dsw058-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a>) is among the most studied lncRNA after <em>Xist</em>. <em>Malat1</em> is a conserved extremely abundant lncRNA non-essential for normal development.<a reveal-id="dsw058-B46 dsw058-B47 dsw058-B48" data-open="dsw058-B46 dsw058-B47 dsw058-B48" class="link link-ref link-reveal xref-bibr"><sup>46–48</sup></a><sup>,</sup><em>Malat1</em> transcript levels are minimal in the oocyte relative to later preimplantation stages (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>B). Similarly, expression of <em>Neat1</em>, lncRNA encoded adjacent to <em>Malat1</em>, starts at ZGA and a shorter <em>Neat1</em> transcript isoform accumulates from the four-cell stage on (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>A). Thus, the <em>Malat1</em>/<em>Neat1</em> locus transcription is a zygotic component of OET. In contrast, <em>Cyrano</em> lncRNA, which has been implicated in embryonic development in zebrafish,<a reveal-id="dsw058-B30" data-open="dsw058-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> exhibits relatively rare maternal and zygotic expression pattern while zygotic expression in the locus extents into a conserved the 3′ end region (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>B).</p><p>Among the lncRNA types, which can be identified, are precursors for small RNAs in RNA silencing pathways, since they can be matched with small RNAs cloned from mouse oocytes and early embryos.<a reveal-id="dsw058-B23" data-open="dsw058-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a><sup>,</sup><a reveal-id="dsw058-B49 dsw058-B50 dsw058-B51" data-open="dsw058-B49 dsw058-B50 dsw058-B51" class="link link-ref link-reveal xref-bibr"><sup>49–51</sup></a> We detected the primary miRNA precursor (pri-miRNA) carrying the miR-290-295 miRNA cluster (lncRNA-OET-07-048) whose expression starts at the two-cell stage and is present during early development (<a reveal-id="dsw058-F5" data-open="dsw058-F5" class="link link-reveal link-table xref-fig">Fig. 5E</a>). The miR-290-295 family is associated with pluripotency and is closely related to the miR-430 family, which functions in zebrafish embryos (reviewed in<a reveal-id="dsw058-B52" data-open="dsw058-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a>). The miR-290-295 family most likely does not contribute strongly to maternal mRNA clearance since most maternal mRNAs become eliminated before the miR-290-295 miRNAs accumulate enough to have an impact on the cellular transcriptome. At the same time, we did not detect any pri-miRNAs of the let-7 family, the most abundant miRNA family expressed in mouse oocytes.<a reveal-id="dsw058-B53" data-open="dsw058-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> This might be explained by the fact that the analysed fully grown oocytes were transcriptionally already quiescent, thus it would be possible that let-7 precursors were already processed into miRNAs. Consequently, we would not observe precursor sequences in oocyte NGS data similarly to the absence of nascent transcripts of maternally expressed genes.<a reveal-id="dsw058-B12" data-open="dsw058-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a></p><p>Although a miRNA precursor yields miRNA(s) with defined sequence(s), short interfering RNAs (siRNAs), which guide endonucleolytic cleavage of cognate RNAs in the RNA interference (RNAi), are produced from a precursor as a population of 21–23 nt RNAs. Mouse oocytes are unique among mammalian cells since they produce high amounts of endogenous siRNAs from double-stranded RNA (dsRNA).<a reveal-id="dsw058-B23" data-open="dsw058-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a><sup>,</sup><a reveal-id="dsw058-B54" data-open="dsw058-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> Endogenous dsRNA can form through (i) a transcription of an inverted repeat, (ii) a convergent transcription, and (iii) basepairing of mRNA and antisense RNA originating, e.g. from a processed pseudogene. Interestingly, of the 13 genes, for which Tam et al. predicted basepiring with transcripts from processed pseudogenes, 7 were annotated among the OET lncRNAs. All of them were maternally expressed (<a reveal-id="dsw058-F5" data-open="dsw058-F5" class="link link-reveal link-table xref-fig">Fig. 5F</a>). Antisense sequences of processed pseudogenes can be found in lncRNAs expressed elsewhere, but efficient siRNA production in mice requires a unique maternal isoform of the Dicer enzyme.<a reveal-id="dsw058-B14" data-open="dsw058-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsw058-B55" data-open="dsw058-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> Therefore, lncRNAs carrying antisense sequences of processed pseudogenes have unique functionality restricted to oocytes and early embryos, where they can be efficiently converted to endo-siRNAs.</p><p>Remarkably, two of these lncRNAs matched the predicted dormant lncRNAs shown in <a reveal-id="dsw058-F3" data-open="dsw058-F3" class="link link-reveal link-table xref-fig">Figure 3C</a> (lncRNA-OET-08-341 complementary to <em>Ppp4r1</em> mRNA and lncRNA-OET-02-259 complementary to <em>Traip</em> mRNA). We hypothesize that cytoplasmic polyadenylation could regulate the availability of the siRNA substrate and thus control the pace of clearance of specific maternal mRNAs.</p>                            <h3 scrollto-destination=78055403 id="78055403" class="section-title" >3.7. Functional analysis of two OET lncRNAs</h3>
<div class="block-child-p">For a functional analysis, we chose two maternal lncRNAs (lncRNA-OET-19-199 and lncRNA-OET-06-154, <a reveal-id="dsw058-F6" data-open="dsw058-F6" class="link link-reveal link-table xref-fig">Fig. 6A and B</a>), which had good expression, used a dominant promoter, showed sequence conservation among mammals, and were syntenic relative to adjacent genes. Both lncRNAs were maternally expressed and degraded after fertilization (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&amp;Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs. S7 and S8</a></span>). Upon confirming lncRNA expression patterns by qPCR (<a reveal-id="dsw058-F6" data-open="dsw058-F6" class="link link-reveal link-table xref-fig">Fig. 6C and E</a>), we created mouse knockout models using RNA-guided CRISPR Cas9 system.<a reveal-id="dsw058-B15" data-open="dsw058-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsw058-B16" data-open="dsw058-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> We aimed at deleting the promoter and exon1 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&amp;Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs. S7D and S8C</a></span>) in order to suppress lncRNA transcription in the locus, not just the accumulation of mature lncRNA. We successfully produced lncRNA knockouts and confirmed the loss of lncRNA expression by a qPCR analysis (<a reveal-id="dsw058-F6" data-open="dsw058-F6" class="link link-reveal link-table xref-fig">Fig. 6D and F</a>). <div data-id="dsw058-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/m_dsw058f6.png?Expires=2147483647&amp;Signature=u-wdKVnBkBPg3kzhBkpNNG~hMJmBPzcqf7MEdnpCvV7k7fq8XbTpHlQjowF3QspfLSm7OfT4xJhPL6AyC29tLqKztK00yMetgj1FDwp9LaQbhkDtPop2eW~W9wViVytlPZGUqH9m6A-C6fdLg674pbncIun55SKA-LFgpi3djgfybL0~B6fmjwkv7uwsfi72csPt2bh3RKPsLsnrnvBMsma0tBawNVgFwuuP87xRBGv~ix9wv3dGIFn~YTVx4tUiMfD3qzZKNJKNmaWJrhHftXvco~taXB8GkTmDLolxTQo5P8JJrMkkyCvbv3bzA7y3qx4Kn~RTtxfyoa4dQTw7kQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CRISPR-mediated lncRNA knockouts of. (A, B) Genomic structure of lncRNA-OET-19-199 and lncRNA-OET-06-154. Filled rectangles represent 5′ terminal, internal, and 3’ terminal exons. The frame over the first exon indicates the region targeted by CRISPR-mediated deletion (cleavage positions are depicted in Supplementary Figs. S7 and S8). (C, E) Relative expression of targeted lncRNA in oocytes and zygotes. RT-qPCR analysis of RNA from a constant number of oocytes/zygotes. These results are consistent with NGS (Supplementary Figs. S7B and S8B) and microarray analysis data (Supplementary Fig. S7C). (D, F) RNA expression at the targeted locus in oocytes from knockout animals. Shown is the RT-qPCR expression analysis of the targeted lncRNA and the nearest two upstream and two downstream genes (maps of the loci are available in Supplementary Figs. S7F and S8E. Error bar = SEM. (G) Characterization of lncRNA-OET-06-154 of the distal 3′ terminal exon region (corresponds to the framed region at the 3′ end of lncRN-OET-06-154 in the panel B). (H) Analysis of NGS data from Dicer and Ago2 knockout oocytes.22 The Y-scale depicts the FPKM difference in Ago2 knockouts (catalytically dead mutant22), the X-scale in Dicer knockouts. The FPKM difference was used because it better reflects the effect of suppressed RNAi on the transcriptome than the ratio, which is distorted by varying expression levels. In other words, if absence of RNAi results in stabilization of 1,000 molecules of a hypothetical mRNA, the graph will display such mRNA in the same position regardless if there is 100, 1 000, or 10 000 molecules of this mRNA in the wild-type oocyte." path-from-xml="dsw058f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055404/dsw058f6.tif" path-from-xml="dsw058f6.tif" target="_blank">View large</a><a section="78055404" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058f6.png?Expires=2147483647&amp;Signature=e-bcIscYkmxSfnJTmWymiwaABg5Dvkz63wTpHAX1rRn76YvVsEPWZoaBtRcbEMjmoKkI9T~pKpGu23DYcdoiCi8kGqBBye68NQ3~3f5wp6MM1CXyA83N1L808Hx1HDs8Z9QKf3WGHq-wzPk9zGUO-yKdGAlOaJxWkZ3vRe48In-G6yaAe6LvqGXX9ojWBAFBxwryqq3ByED6cWzd~7kcHDMgGnhL6F0OVUi45pEIRqkFqnzy9qoJDUFULIUckNZWCmSEJGmKm~Jd8P4aQxqtONnH2HfnGOhqc2SXMcIRql~QANHvDA9pPTHZ2HQVI8H22eTT5Cw8LCZRCxJCRXZNaA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055404&amp;ar=2901809&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw058f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CRISPR-mediated lncRNA knockouts of. <strong>(A, B)</strong> Genomic structure of lncRNA-OET-19-199 and lncRNA-OET-06-154. Filled rectangles represent 5′ terminal, internal, and 3’ terminal exons. The frame over the first exon indicates the region targeted by CRISPR-mediated deletion (cleavage positions are depicted in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&amp;Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs. S7 and S8</a></span>). <strong>(C, E)</strong> Relative expression of targeted lncRNA in oocytes and zygotes. RT-qPCR analysis of RNA from a constant number of oocytes/zygotes. These results are consistent with NGS (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&amp;Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs. S7B and S8B</a></span>) and microarray analysis data (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&amp;Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>C). <strong>(D, F)</strong> RNA expression at the targeted locus in oocytes from knockout animals. Shown is the RT-qPCR expression analysis of the targeted lncRNA and the nearest two upstream and two downstream genes (maps of the loci are available in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&amp;Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs. S7F and S8E</a></span>. Error bar = SEM. <strong>(G)</strong> Characterization of lncRNA-OET-06-154 of the distal 3′ terminal exon region (corresponds to the framed region at the 3′ end of lncRN-OET-06-154 in the panel B). <strong>(H)</strong> Analysis of NGS data from <em>Dicer</em> and <em>Ago2</em> knockout oocytes.<a reveal-id="dsw058-B22" data-open="dsw058-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> The Y-scale depicts the FPKM difference in <em>Ago2</em> knockouts (catalytically dead mutant<a reveal-id="dsw058-B22" data-open="dsw058-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a>), the X-scale in <em>Dicer</em> knockouts. The FPKM difference was used because it better reflects the effect of suppressed RNAi on the transcriptome than the ratio, which is distorted by varying expression levels. In other words, if absence of RNAi results in stabilization of 1,000 molecules of a hypothetical mRNA, the graph will display such mRNA in the same position regardless if there is 100, 1 000, or 10 000 molecules of this mRNA in the wild-type oocyte.</p></div></div><div content-id="dsw058-F6" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/m_dsw058f6.png?Expires=2147483647&amp;Signature=u-wdKVnBkBPg3kzhBkpNNG~hMJmBPzcqf7MEdnpCvV7k7fq8XbTpHlQjowF3QspfLSm7OfT4xJhPL6AyC29tLqKztK00yMetgj1FDwp9LaQbhkDtPop2eW~W9wViVytlPZGUqH9m6A-C6fdLg674pbncIun55SKA-LFgpi3djgfybL0~B6fmjwkv7uwsfi72csPt2bh3RKPsLsnrnvBMsma0tBawNVgFwuuP87xRBGv~ix9wv3dGIFn~YTVx4tUiMfD3qzZKNJKNmaWJrhHftXvco~taXB8GkTmDLolxTQo5P8JJrMkkyCvbv3bzA7y3qx4Kn~RTtxfyoa4dQTw7kQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CRISPR-mediated lncRNA knockouts of. (A, B) Genomic structure of lncRNA-OET-19-199 and lncRNA-OET-06-154. Filled rectangles represent 5′ terminal, internal, and 3’ terminal exons. The frame over the first exon indicates the region targeted by CRISPR-mediated deletion (cleavage positions are depicted in Supplementary Figs. S7 and S8). (C, E) Relative expression of targeted lncRNA in oocytes and zygotes. RT-qPCR analysis of RNA from a constant number of oocytes/zygotes. These results are consistent with NGS (Supplementary Figs. S7B and S8B) and microarray analysis data (Supplementary Fig. S7C). (D, F) RNA expression at the targeted locus in oocytes from knockout animals. Shown is the RT-qPCR expression analysis of the targeted lncRNA and the nearest two upstream and two downstream genes (maps of the loci are available in Supplementary Figs. S7F and S8E. Error bar = SEM. (G) Characterization of lncRNA-OET-06-154 of the distal 3′ terminal exon region (corresponds to the framed region at the 3′ end of lncRN-OET-06-154 in the panel B). (H) Analysis of NGS data from Dicer and Ago2 knockout oocytes.22 The Y-scale depicts the FPKM difference in Ago2 knockouts (catalytically dead mutant22), the X-scale in Dicer knockouts. The FPKM difference was used because it better reflects the effect of suppressed RNAi on the transcriptome than the ratio, which is distorted by varying expression levels. In other words, if absence of RNAi results in stabilization of 1,000 molecules of a hypothetical mRNA, the graph will display such mRNA in the same position regardless if there is 100, 1 000, or 10 000 molecules of this mRNA in the wild-type oocyte." path-from-xml="dsw058f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055404/dsw058f6.tif" path-from-xml="dsw058f6.tif" target="_blank">View large</a><a section="78055404" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058f6.png?Expires=2147483647&amp;Signature=e-bcIscYkmxSfnJTmWymiwaABg5Dvkz63wTpHAX1rRn76YvVsEPWZoaBtRcbEMjmoKkI9T~pKpGu23DYcdoiCi8kGqBBye68NQ3~3f5wp6MM1CXyA83N1L808Hx1HDs8Z9QKf3WGHq-wzPk9zGUO-yKdGAlOaJxWkZ3vRe48In-G6yaAe6LvqGXX9ojWBAFBxwryqq3ByED6cWzd~7kcHDMgGnhL6F0OVUi45pEIRqkFqnzy9qoJDUFULIUckNZWCmSEJGmKm~Jd8P4aQxqtONnH2HfnGOhqc2SXMcIRql~QANHvDA9pPTHZ2HQVI8H22eTT5Cw8LCZRCxJCRXZNaA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055404&amp;ar=2901809&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw058f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CRISPR-mediated lncRNA knockouts of. <strong>(A, B)</strong> Genomic structure of lncRNA-OET-19-199 and lncRNA-OET-06-154. Filled rectangles represent 5′ terminal, internal, and 3’ terminal exons. The frame over the first exon indicates the region targeted by CRISPR-mediated deletion (cleavage positions are depicted in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&amp;Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs. S7 and S8</a></span>). <strong>(C, E)</strong> Relative expression of targeted lncRNA in oocytes and zygotes. RT-qPCR analysis of RNA from a constant number of oocytes/zygotes. These results are consistent with NGS (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&amp;Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs. S7B and S8B</a></span>) and microarray analysis data (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&amp;Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>C). <strong>(D, F)</strong> RNA expression at the targeted locus in oocytes from knockout animals. Shown is the RT-qPCR expression analysis of the targeted lncRNA and the nearest two upstream and two downstream genes (maps of the loci are available in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&amp;Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs. S7F and S8E</a></span>. Error bar = SEM. <strong>(G)</strong> Characterization of lncRNA-OET-06-154 of the distal 3′ terminal exon region (corresponds to the framed region at the 3′ end of lncRN-OET-06-154 in the panel B). <strong>(H)</strong> Analysis of NGS data from <em>Dicer</em> and <em>Ago2</em> knockout oocytes.<a reveal-id="dsw058-B22" data-open="dsw058-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> The Y-scale depicts the FPKM difference in <em>Ago2</em> knockouts (catalytically dead mutant<a reveal-id="dsw058-B22" data-open="dsw058-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a>), the X-scale in <em>Dicer</em> knockouts. The FPKM difference was used because it better reflects the effect of suppressed RNAi on the transcriptome than the ratio, which is distorted by varying expression levels. In other words, if absence of RNAi results in stabilization of 1,000 molecules of a hypothetical mRNA, the graph will display such mRNA in the same position regardless if there is 100, 1 000, or 10 000 molecules of this mRNA in the wild-type oocyte.</p></div></div></div><p>Breeding of mutant mice did not reveal any effects on viability and fertility of homozygotes although breeding of lncRNA-OET-19-199 heterozygotes produced heterozygotes with a lower frequency than expected (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs. S7E and S8D</a></span>). The basis of this effect is currently under investigation. Nonetheless, homozygous null females for each lncRNA knockout were fertile and breeding of null animals produced viable offspring in both cases. Ovarian histology of knockout animals appeared normal and normal amounts of fully grown oocytes were recovered from null females (data not shown). We also examined expression of nearest genes (<a reveal-id="dsw058-F6" data-open="dsw058-F6" class="link link-reveal link-table xref-fig">Fig. 6D and F</a>). We observed a significant difference in one of the neighbouring genes of lncRNA-OET-19-199 (<a reveal-id="dsw058-F6" data-open="dsw058-F6" class="link link-reveal link-table xref-fig">Fig. 6D</a>). However, whether this reflected the biological role of lncRNA-OET-19-199 or whether it was a consequence of the introduced DNA deletion remains unknown.</p><p>Importantly, we assigned a biological function to one of the transcript isoforms of lncRNA-OET-06-154, even though the loss expression of lncRNA-OET-06-154 had no effect on fertility (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>D) and there was no effect on neighbouring genes (<a reveal-id="dsw058-F6" data-open="dsw058-F6" class="link link-reveal link-table xref-fig">Fig. 6F</a>). The reason was that the most downstream terminal exon of lncRNA-OET-06-154 carried an antisense sequences from the <em>Eef1g</em> pseudogene (<a reveal-id="dsw058-F6" data-open="dsw058-F6" class="link link-reveal link-table xref-fig">Fig. 6G</a>). The pseudogene insertion happened already in the common ancestor of mice and rats, and the pseudogene fragment was subsequently disrupted by several LTR insertions in the mouse lineage. The antisense pseudogene sequence generates an endo-siRNAs as evidenced by mapping small RNAs from NGS data<a reveal-id="dsw058-B23" data-open="dsw058-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> to the locus (<a reveal-id="dsw058-F6" data-open="dsw058-F6" class="link link-reveal link-table xref-fig">Fig. 6G</a>). Small RNAs targeting <em>Eef1g</em> are biologically active as evidenced by <em>Eef1g</em> upregulation in both <em>Dicer</em> and <em>Ago2</em> knockout oocytes (<a reveal-id="dsw058-F6" data-open="dsw058-F6" class="link link-reveal link-table xref-fig">Fig. 6H</a>). Taken together, lncRNA-OET-06-154 represents an example of a locus expressing multiple transcript isoforms that might differ in function: those carrying the most downstream 3’ terminal exon would be engaged in RNAi-mediated repression of <em>Eef1g</em> in the oocyte, while others might have another function (or no function at all). This lncRNA example also shows that functional siRNAs may have originated from a pseudogene insertion more than 40 million years old as suggested by molecular dating of the common ancestor of mice and rats based on 658 nuclear genes.<a reveal-id="dsw058-B56" data-open="dsw058-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a></p>                            <h2 scrollto-destination=78055407 id="78055407" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Radek Malik, Radek Jankele, Martin Moravec, Helena Fulkova, Josef Pasulka, and other members of our laboratories for discussions and assistance, the Mediterranean Institute for Life Sciences for hosting the data mining sessions, and the Transgenic and Archiving Module of the Czech Centre for Phenogenomics, Institute of Molecular Genetics ASCR whose work was supported by LM2011032 and LM2015040 (MEYS) and the BIOCEV European Regional Development Fund CZ.1.05/1.1.00/02.0109.</p>                            <h3 scrollto-destination=78055409 id="78055409" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=78055411 id="78055411" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h3 scrollto-destination=78055413 id="78055413" class="section-title" >Funding</h3>
<p>The main support for this research was provided through a Marie Curie Initial Training Network (project no. 607720, RNATRAIN) funding for S.G. Research of PS lab was further supported by the Czech Science Foundation grant GACR P305/12/G034, and Ministry of Education, Youth, and Sports project NPU1 LO1419. The IMG institutional support was provided by RVO: 68378050. R.K., V.F., and K.V. were supported through the European Commission Seventh Framework Program (Integra-Life; grant 315997 to KV), and Croatian Science Foundation (grant IP-2014-09-6400 to KV). The work of FA was supported by Grants-in-Aid from the Ministry of Education, Culture, Sports, Science and Technology of Japan (no. 20062002, no. 25252054).</p>                            <h2 scrollto-destination=78055415 id="78055415" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw058-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Svoboda</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Franke</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Schultz</div> <div class="given-names">R.M.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Sculpting the transcriptome during the oocyte-to-embryo transition in mouse</div>. <div class="source">Curr. Top. Dev. Biol</div> ., <div class="volume">113</div>, <div class="fpage">305</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sculpting%20the%20transcriptome%20during%20the%20oocyte-to-embryo%20transition%20in%20mouse&amp;author=P.%20Svoboda&amp;author=V.%20Franke&amp;author=R.M.%20Schultz&amp;publication_year=2015&amp;journal=Curr.%20Top.%20Dev.%20Biol&amp;volume=113&amp;pages=305-49" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26358877" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marques</div> <div class="given-names">A.C.</div></div>, <div class="name string-name"><div class="surname">Ponting</div> <div class="given-names">C.P.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Intergenic lncRNAs and the evolution of gene expression</div>. <div class="source">Curr. Opin. Genet. Dev</div> ., <div class="volume">27</div>, <div class="fpage">48</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intergenic%20lncRNAs%20and%20the%20evolution%20of%20gene%20expression&amp;author=A.C.%20Marques&amp;author=C.P.%20Ponting&amp;publication_year=2014&amp;journal=Curr.%20Opin.%20Genet.%20Dev&amp;volume=27&amp;pages=48-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gde.2014.03.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2014.03.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gde.2014.03.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24852186" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guttman</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Rinn</div> <div class="given-names">J.L.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Modular regulatory principles of large non-coding RNAs</div>. <div class="source">Nature</div> , <div class="volume">482</div>, <div class="fpage">339</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Modular%20regulatory%20principles%20of%20large%20non-coding%20RNAs&amp;author=M.%20Guttman&amp;author=J.L.%20Rinn&amp;publication_year=2012&amp;journal=Nature&amp;volume=482&amp;pages=339-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10887" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10887" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10887"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22337053" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mercer</div> <div class="given-names">T.R.</div></div>, <div class="name string-name"><div class="surname">Mattick</div> <div class="given-names">J.S.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Structure and function of long noncoding RNAs in epigenetic regulation</div>. <div class="source">Nat. Struct. Mol. Biol</div> ., <div class="volume">20</div>, <div class="fpage">300</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20and%20function%20of%20long%20noncoding%20RNAs%20in%20epigenetic%20regulation&amp;author=T.R.%20Mercer&amp;author=J.S.%20Mattick&amp;publication_year=2013&amp;journal=Nat.%20Struct.%20Mol.%20Biol&amp;volume=20&amp;pages=300-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.2480" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.2480" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.2480"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23463315" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ulitsky</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Bartel</div> <div class="given-names">D.P.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">lincRNAs: genomics, evolution, and mechanisms</div>. <div class="source">Cell</div> , <div class="volume">154</div>, <div class="fpage">26</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=lincRNAs%3A%20genomics%2C%20evolution%2C%20and%20mechanisms&amp;author=I.%20Ulitsky&amp;author=D.P.%20Bartel&amp;publication_year=2013&amp;journal=Cell&amp;volume=154&amp;pages=26-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2013.06.020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2013.06.020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2013.06.020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23827673" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kutter</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Watt</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Stefflova</div> <div class="given-names">K.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Rapid turnover of long noncoding RNAs and the evolution of gene expression</div>. <div class="source">PLoS Genet</div> ., <div class="volume">8</div>, <div class="fpage">e1002841</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20turnover%20of%20long%20noncoding%20RNAs%20and%20the%20evolution%20of%20gene%20expression&amp;author=C.%20Kutter&amp;author=S.%20Watt&amp;author=K.%20Stefflova&amp;publication_year=2012&amp;journal=PLoS%20Genet&amp;volume=8&amp;pages=e1002841" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002841" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002841" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002841"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22844254" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ng</div> <div class="given-names">S.Y.</div></div>, <div class="name string-name"><div class="surname">Stanton</div> <div class="given-names">L.W.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Long non-coding RNAs in stem cell pluripotency</div>. <div class="source">Wiley Interdiscip. Rev. RNA</div> , <div class="volume">4</div>, <div class="fpage">121</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Long%20non-coding%20RNAs%20in%20stem%20cell%20pluripotency&amp;author=S.Y.%20Ng&amp;author=L.W.%20Stanton&amp;publication_year=2013&amp;journal=Wiley%20Interdiscip.%20Rev.%20RNA&amp;volume=4&amp;pages=121-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/wrna.1146" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fwrna.1146" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fwrna.1146"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23139157" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yan</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">H.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Single-cell RNA-Seq profiling of human preimplantation embryos and embryonic stem cells</div>. <div class="source">Nat. Struct. Mol. Biol</div> ., <div class="volume">20</div>, <div class="fpage">1131</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single-cell%20RNA-Seq%20profiling%20of%20human%20preimplantation%20embryos%20and%20embryonic%20stem%20cells&amp;author=L.%20Yan&amp;author=M.%20Yang&amp;author=H.%20Guo&amp;publication_year=2013&amp;journal=Nat.%20Struct.%20Mol.%20Biol&amp;volume=20&amp;pages=1131-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.2660" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.2660" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.2660"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23934149" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Luo</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">S.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Identification and functional analysis of long non-coding RNAs in mouse cleavage stage embryonic development based on single cell transcriptome data</div>. <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">845</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20functional%20analysis%20of%20long%20non-coding%20RNAs%20in%20mouse%20cleavage%20stage%20embryonic%20development%20based%20on%20single%20cell%20transcriptome%20data&amp;author=K.%20Zhang&amp;author=K.%20Huang&amp;author=Y.%20Luo&amp;author=S.%20Li&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=845" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-845" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-845" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-845"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25277336" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hamazaki</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Uesaka</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Nakashima</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Agata</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Imamura</div> <div class="given-names">T.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Gene activation-associated long noncoding RNAs function in mouse preimplantation development</div>. <div class="source">Development</div> , <div class="volume">142</div>, <div class="fpage">910</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20activation-associated%20long%20noncoding%20RNAs%20function%20in%20mouse%20preimplantation%20development&amp;author=N.%20Hamazaki&amp;author=M.%20Uesaka&amp;author=K.%20Nakashima&amp;author=K.%20Agata&amp;author=T.%20Imamura&amp;publication_year=2015&amp;journal=Development&amp;volume=142&amp;pages=910-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/dev.116996" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fdev.116996" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fdev.116996"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25633350" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Veselovska</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Smallwood</div> <div class="given-names">S. A.</div></div>, <div class="name string-name"><div class="surname">Saadeh</div> <div class="given-names">H.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Deep sequencing and de novo assembly of the mouse oocyte transcriptome define the contribution of transcription to the DNA methylation landscape</div>. <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">209</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deep%20sequencing%20and%20de%20novo%20assembly%20of%20the%20mouse%20oocyte%20transcriptome%20define%20the%20contribution%20of%20transcription%20to%20the%20DNA%20methylation%20landscape&amp;author=L.%20Veselovska&amp;author=S.%20A.%20Smallwood&amp;author=H.%20Saadeh&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=209" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-015-0769-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0769-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-015-0769-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26408185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Abe</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Yamamoto</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Franke</div> <div class="given-names">V.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">The first murine zygotic transcription is promiscuous and uncoupled from splicing and 3' processing</div>. <div class="source">EMBO J</div> ., <div class="volume">34</div>, <div class="fpage">1523</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20first%20murine%20zygotic%20transcription%20is%20promiscuous%20and%20uncoupled%20from%20splicing%20and%203'%20processing&amp;author=K.%20Abe&amp;author=R.%20Yamamoto&amp;author=V.%20Franke&amp;publication_year=2015&amp;journal=EMBO%20J&amp;volume=34&amp;pages=1523-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.15252/embj.201490648" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.15252%2Fembj.201490648" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.15252%2Fembj.201490648"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25896510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nagy</div> <div class="given-names">A.</div></div></div> <div class="year">2003</div>, <div class="source">Manipulating the Mouse Embryo: A Laboratory Manual</div> . 
<div class="publisher-name">Cold Spring Harbor Laboratory Press</div>: 
<div class="publisher-loc">Cold Spring Harbor, NY</div>.</div></div></div></div><div content-id="dsw058-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Flemr</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Malik</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Franke</div> <div class="given-names">V.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">A retrotransposon-driven dicer isoform directs endogenous small interfering RNA production in mouse oocytes</div>. <div class="source">Cell</div> , <div class="volume">155</div>, <div class="fpage">807</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20retrotransposon-driven%20dicer%20isoform%20directs%20endogenous%20small%20interfering%20RNA%20production%20in%20mouse%20oocytes&amp;author=M.%20Flemr&amp;author=R.%20Malik&amp;author=V.%20Franke&amp;publication_year=2013&amp;journal=Cell&amp;volume=155&amp;pages=807-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2013.10.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2013.10.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2013.10.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24209619" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cong</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Ran</div> <div class="given-names">F.A.</div></div>, <div class="name string-name"><div class="surname">Cox</div> <div class="given-names">D.</div></div></div>, et al.  . 
<div class="article-title">Multiplex genome engineering using CRISPR/Cas systems</div>. <div class="source">Science</div> , <div class="volume">339</div>, <div class="fpage">819</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1231143" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1231143" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1231143"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23287718" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Gilbert</div> <div class="given-names">L. A.</div></div>, <div class="name string-name"><div class="surname">Cimini</div> <div class="given-names">B. A.</div></div></div>, et al.  . 
<div class="article-title">Dynamic imaging of genomic loci in living human cells by an optimized CRISPR/Cas system</div>. <div class="source">Cell</div> , <div class="volume">155</div>, <div class="fpage">1479</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2013.12.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2013.12.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2013.12.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24360272" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Park</div> <div class="given-names">S.J.</div></div>, <div class="name string-name"><div class="surname">Komata</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Inoue</div> <div class="given-names">F.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Inferring the choreography of parental genomes during fertilization from ultralarge-scale whole-transcriptome analysis</div>. <div class="source">Genes Dev</div> ., <div class="volume">27</div>, <div class="fpage">2736</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inferring%20the%20choreography%20of%20parental%20genomes%20during%20fertilization%20from%20ultralarge-scale%20whole-transcriptome%20analysis&amp;author=S.J.%20Park&amp;author=M.%20Komata&amp;author=F.%20Inoue&amp;publication_year=2013&amp;journal=Genes%20Dev&amp;volume=27&amp;pages=2736-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.227926.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.227926.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.227926.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24352427" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xue</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Cai</div> <div class="given-names">C.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Genetic programs in human and mouse early embryos revealed by single-cell RNA sequencing</div>. <div class="source">Nature</div> , <div class="volume">500</div>, <div class="fpage">593</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20programs%20in%20human%20and%20mouse%20early%20embryos%20revealed%20by%20single-cell%20RNA%20sequencing&amp;author=Z.%20Xue&amp;author=K.%20Huang&amp;author=C.%20Cai&amp;publication_year=2013&amp;journal=Nature&amp;volume=500&amp;pages=593-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature12364" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature12364" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature12364"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23892778" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guttman</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Garber</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Levin</div> <div class="given-names">J. Z.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">Ab initio reconstruction of cell type-specific transcriptomes in mouse reveals the conserved multi-exonic structure of lincRNAs</div>. <div class="source">Nat. Biotechnol</div> ., <div class="volume">28</div>, <div class="fpage">503</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ab%20initio%20reconstruction%20of%20cell%20type-specific%20transcriptomes%20in%20mouse%20reveals%20the%20conserved%20multi-exonic%20structure%20of%20lincRNAs&amp;author=M.%20Guttman&amp;author=M.%20Garber&amp;author=J.%20Z.%20Levin&amp;publication_year=2010&amp;journal=Nat.%20Biotechnol&amp;volume=28&amp;pages=503-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1633" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1633" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1633"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20436462" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">H.J.</div></div>, <div class="name string-name"><div class="surname">Dasari</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kocher</div> <div class="given-names">J.P.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">W.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">CPAT: coding-potential assessment tool using an alignment-free logistic regression model</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">e74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CPAT%3A%20coding-potential%20assessment%20tool%20using%20an%20alignment-free%20logistic%20regression%20model&amp;author=L.%20Wang&amp;author=H.J.%20Park&amp;author=S.%20Dasari&amp;author=S.%20Wang&amp;author=J.P.%20Kocher&amp;author=W.%20Li&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=e74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23335781" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tarailo-Graovac</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">N.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Using RepeatMasker to identify repetitive elements in genomic sequences</div>. <div class="source">Curr. Protoc. Bioinformatics</div> , <strong>Chapter 4</strong>, Unit 4 10.</div></div></div></div><div content-id="dsw058-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stein</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Rozhkov</div> <div class="given-names">N.V.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">F.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Essential Role for endogenous siRNAs during meiosis in mouse oocytes</div>. <div class="source">PLoS Genet</div> ., <div class="volume">11</div>, <div class="fpage">e1005013</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Essential%20Role%20for%20endogenous%20siRNAs%20during%20meiosis%20in%20mouse%20oocytes&amp;author=P.%20Stein&amp;author=N.V.%20Rozhkov&amp;author=F.%20Li&amp;publication_year=2015&amp;journal=PLoS%20Genet&amp;volume=11&amp;pages=e1005013" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1005013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1005013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1005013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25695507" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tam</div> <div class="given-names">O.H.</div></div>, <div class="name string-name"><div class="surname">Aravin</div> <div class="given-names">A.A.</div></div>, <div class="name string-name"><div class="surname">Stein</div> <div class="given-names">P.</div></div></div>, et al.  . <div class="year">2008</div>, 
<div class="article-title">Pseudogene-derived small interfering RNAs regulate gene expression in mouse oocytes</div>. <div class="source">Nature</div> , <div class="volume">453</div>, <div class="fpage">534</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pseudogene-derived%20small%20interfering%20RNAs%20regulate%20gene%20expression%20in%20mouse%20oocytes&amp;author=O.H.%20Tam&amp;author=A.A.%20Aravin&amp;author=P.%20Stein&amp;publication_year=2008&amp;journal=Nature&amp;volume=453&amp;pages=534-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06904" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06904" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06904"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18404147" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Flemr</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Ma</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Schultz</div> <div class="given-names">R. M.</div></div>, <div class="name string-name"><div class="surname">Svoboda</div> <div class="given-names">P.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">P-body loss is concomitant with formation of a messenger RNA storage domain in mouse oocytes</div>. <div class="source">Biol. Reprod</div> , <strong>82</strong>, 1008–17.</div></div></div></div><div content-id="dsw058-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guttman</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Amit</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Garber</div> <div class="given-names">M.</div></div></div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">Chromatin signature reveals over a thousand highly conserved large non-coding RNAs in mammals</div>. <div class="source">Nature</div> , <div class="volume">458</div>, <div class="fpage">223</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromatin%20signature%20reveals%20over%20a%20thousand%20highly%20conserved%20large%20non-coding%20RNAs%20in%20mammals&amp;author=M.%20Guttman&amp;author=I.%20Amit&amp;author=M.%20Garber&amp;publication_year=2009&amp;journal=Nature&amp;volume=458&amp;pages=223-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07672" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07672" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07672"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19182780" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kelley</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Rinn</div> <div class="given-names">J.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Transposable elements reveal a stem cell-specific class of long noncoding RNAs</div>. <div class="source">Genome Biol</div> , <div class="volume">13</div>, <div class="fpage">R107</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transposable%20elements%20reveal%20a%20stem%20cell-specific%20class%20of%20long%20noncoding%20RNAs&amp;author=D.%20Kelley&amp;author=J.%20Rinn&amp;publication_year=2012&amp;journal=Genome%20Biol&amp;volume=13&amp;pages=R107" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2012-13-11-r107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2012-13-11-r107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2012-13-11-r107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23181609" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Piko</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Clegg</div> <div class="given-names">K.B.</div></div></div> <div class="year">1982</div>, 
<div class="article-title">Quantitative changes in total Rna, total Poly(a), and ribosomes in early mouse embryos</div>. <div class="source">Dev. Biol</div> ., <div class="volume">89</div>, <div class="fpage">362</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20changes%20in%20total%20Rna%2C%20total%20Poly(a)%2C%20and%20ribosomes%20in%20early%20mouse%20embryos&amp;author=L.%20Piko&amp;author=K.B.%20Clegg&amp;publication_year=1982&amp;journal=Dev.%20Biol&amp;volume=89&amp;pages=362-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0012-1606(82)90325-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0012-1606(82)90325-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0012-1606(82)90325-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6173273" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cabili</div> <div class="given-names">M.N.</div></div>, <div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Goff</div> <div class="given-names">L.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">Integrative annotation of human large intergenic noncoding RNAs reveals global properties and specific subclasses</div>. <div class="source">Genes Dev</div> ., <div class="volume">25</div>, <div class="fpage">1915</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrative%20annotation%20of%20human%20large%20intergenic%20noncoding%20RNAs%20reveals%20global%20properties%20and%20specific%20subclasses&amp;author=M.N.%20Cabili&amp;author=C.%20Trapnell&amp;author=L.%20Goff&amp;publication_year=2011&amp;journal=Genes%20Dev&amp;volume=25&amp;pages=1915-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.17446611" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.17446611" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.17446611"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21890647" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Derrien</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Johnson</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Bussotti</div> <div class="given-names">G.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">The GENCODE v7 catalog of human long noncoding RNAs: analysis of their gene structure, evolution, and expression</div>. <div class="source">Genome Res</div> ., <div class="volume">22</div>, <div class="fpage">1775</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20GENCODE%20v7%20catalog%20of%20human%20long%20noncoding%20RNAs%3A%20analysis%20of%20their%20gene%20structure%2C%20evolution%2C%20and%20expression&amp;author=T.%20Derrien&amp;author=R.%20Johnson&amp;author=G.%20Bussotti&amp;publication_year=2012&amp;journal=Genome%20Res&amp;volume=22&amp;pages=1775-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.132159.111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.132159.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.132159.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22955988" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ulitsky</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Shkumatava</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Jan</div> <div class="given-names">C. H.</div></div>, <div class="name string-name"><div class="surname">Sive</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Bartel</div> <div class="given-names">D.P.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Conserved function of lincRNAs in vertebrate embryonic development despite rapid sequence evolution</div>. <div class="source">Cell</div> , <div class="volume">147</div>, <div class="fpage">1537</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conserved%20function%20of%20lincRNAs%20in%20vertebrate%20embryonic%20development%20despite%20rapid%20sequence%20evolution&amp;author=I.%20Ulitsky&amp;author=A.%20Shkumatava&amp;author=C.%20H.%20Jan&amp;author=H.%20Sive&amp;author=D.P.%20Bartel&amp;publication_year=2011&amp;journal=Cell&amp;volume=147&amp;pages=1537-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2011.11.055" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2011.11.055" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2011.11.055"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22196729" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Smallwood</div> <div class="given-names">S.A.</div></div>, <div class="name string-name"><div class="surname">Tomizawa</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Krueger</div> <div class="given-names">F.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">Dynamic CpG island methylation landscape in oocytes and preimplantation embryos</div>. <div class="source">Nat. Genet</div> ., <div class="volume">43</div>, <div class="fpage">811</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20CpG%20island%20methylation%20landscape%20in%20oocytes%20and%20preimplantation%20embryos&amp;author=S.A.%20Smallwood&amp;author=S.%20Tomizawa&amp;author=F.%20Krueger&amp;publication_year=2011&amp;journal=Nat.%20Genet&amp;volume=43&amp;pages=811-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.864" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.864" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.864"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21706000" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marzluff</div> <div class="given-names">W.F.</div></div>, <div class="name string-name"><div class="surname">Gongidi</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Woods</div> <div class="given-names">K.R.</div></div>, <div class="name string-name"><div class="surname">Jin</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Maltais</div> <div class="given-names">L.J.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">The human and mouse replication-dependent histone genes</div>. <div class="source">Genomics</div> , <div class="volume">80</div>, <div class="fpage">487</div>–<div class="lpage">98</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20human%20and%20mouse%20replication-dependent%20histone%20genes&amp;author=W.F.%20Marzluff&amp;author=P.%20Gongidi&amp;author=K.R.%20Woods&amp;author=J.%20Jin&amp;author=L.J.%20Maltais&amp;publication_year=2002&amp;journal=Genomics&amp;volume=80&amp;pages=487-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/geno.2002.6850" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fgeno.2002.6850" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fgeno.2002.6850"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12408966" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Richter</div> <div class="given-names">J.D.</div></div>, <div class="name string-name"><div class="surname">Lasko</div> <div class="given-names">P.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Translational control in oocyte development</div>. <div class="source">Cold Spring Harb. Perspect. Biol</div> ., <div class="volume">3</div>, <div class="fpage">a002758</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translational%20control%20in%20oocyte%20development&amp;author=J.D.%20Richter&amp;author=P.%20Lasko&amp;publication_year=2011&amp;journal=Cold%20Spring%20Harb.%20Perspect.%20Biol&amp;volume=3&amp;pages=a002758" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/cshperspect.a002758" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fcshperspect.a002758" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fcshperspect.a002758"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21690213" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gebauer</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Richter</div> <div class="given-names">J.D.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Synthesis and function of Mos: the control switch of vertebrate oocyte meiosis</div>. <div class="source">Bioessays</div> , <div class="volume">19</div>, <div class="fpage">23</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synthesis%20and%20function%20of%20Mos%3A%20the%20control%20switch%20of%20vertebrate%20oocyte%20meiosis&amp;author=F.%20Gebauer&amp;author=J.D.%20Richter&amp;publication_year=1997&amp;journal=Bioessays&amp;volume=19&amp;pages=23-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1521-1878" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1521-1878" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1521-1878"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9008414" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huarte</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Belin</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Vassalli</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Strickland</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Vassalli</div> <div class="given-names">J.D.</div></div></div> <div class="year">1987</div>, 
<div class="article-title">Meiotic maturation of mouse oocytes triggers the translation and polyadenylation of dormant tissue-type plasminogen activator mRNA</div>. <div class="source">Genes Dev</div> ., <div class="volume">1</div>, <div class="fpage">1201</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Meiotic%20maturation%20of%20mouse%20oocytes%20triggers%20the%20translation%20and%20polyadenylation%20of%20dormant%20tissue-type%20plasminogen%20activator%20mRNA&amp;author=J.%20Huarte&amp;author=D.%20Belin&amp;author=A.%20Vassalli&amp;author=S.%20Strickland&amp;author=J.D.%20Vassalli&amp;publication_year=1987&amp;journal=Genes%20Dev&amp;volume=1&amp;pages=1201-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1.10.1201" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1.10.1201" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1.10.1201"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3123315" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">de Vantery</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Stutz</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Vassalli</div> <div class="given-names">J.D.</div></div>, <div class="name string-name"><div class="surname">Schorderet-Slatkine</div> <div class="given-names">S.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Acquisition of meiotic competence in growing mouse oocytes is controlled at both translational and posttranslational levels</div>. <div class="source">Dev. Biol</div> ., <div class="volume">187</div>, <div class="fpage">43</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Acquisition%20of%20meiotic%20competence%20in%20growing%20mouse%20oocytes%20is%20controlled%20at%20both%20translational%20and%20posttranslational%20levels&amp;author=C.%20de%20Vantery&amp;author=A.%20Stutz&amp;author=J.D.%20Vassalli&amp;author=S.%20Schorderet-Slatkine&amp;publication_year=1997&amp;journal=Dev.%20Biol&amp;volume=187&amp;pages=43-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/dbio.1997.8599" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fdbio.1997.8599" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fdbio.1997.8599"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9224673" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Murai</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Stein</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Buffone</div> <div class="given-names">M.G.</div></div>, <div class="name string-name"><div class="surname">Yamashita</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Schultz</div> <div class="given-names">R.M.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Recruitment of Orc6l, a dormant maternal mRNA in mouse oocytes, is essential for DNA replication in 1-cell embryos</div>. <div class="source">Dev. Biol</div> ., <div class="volume">341</div>, <div class="fpage">205</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recruitment%20of%20Orc6l%2C%20a%20dormant%20maternal%20mRNA%20in%20mouse%20oocytes%2C%20is%20essential%20for%20DNA%20replication%20in%201-cell%20embryos&amp;author=S.%20Murai&amp;author=P.%20Stein&amp;author=M.G.%20Buffone&amp;author=S.%20Yamashita&amp;author=R.M.%20Schultz&amp;publication_year=2010&amp;journal=Dev.%20Biol&amp;volume=341&amp;pages=205-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ydbio.2010.02.027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ydbio.2010.02.027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ydbio.2010.02.027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20219456" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ma</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Flemr</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Strnad</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Svoboda</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Schultz</div> <div class="given-names">R.M.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Maternally recruited DCP1A and DCP2 contribute to messenger RNA degradation during oocyte maturation and genome activation in mouse</div>. <div class="source">Biol. Reprod</div> ., <div class="volume">88</div>, <div class="fpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Maternally%20recruited%20DCP1A%20and%20DCP2%20contribute%20to%20messenger%20RNA%20degradation%20during%20oocyte%20maturation%20and%20genome%20activation%20in%20mouse&amp;author=J.%20Ma&amp;author=M.%20Flemr&amp;author=H.%20Strnad&amp;author=P.%20Svoboda&amp;author=R.M.%20Schultz&amp;publication_year=2013&amp;journal=Biol.%20Reprod&amp;volume=88&amp;pages=11" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23136299" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Charlesworth</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Cox</div> <div class="given-names">L.L.</div></div>, <div class="name string-name"><div class="surname">MacNicol</div> <div class="given-names">A.M.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Cytoplasmic polyadenylation element (CPE)- and CPE-binding protein (CPEB)-independent mechanisms regulate early class maternal mRNA translational activation in Xenopus oocytes</div>. <div class="source">J. Biol. Chem</div> ., <div class="volume">279</div>, <div class="fpage">17650</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytoplasmic%20polyadenylation%20element%20(CPE)-%20and%20CPE-binding%20protein%20(CPEB)-independent%20mechanisms%20regulate%20early%20class%20maternal%20mRNA%20translational%20activation%20in%20Xenopus%20oocytes&amp;author=A.%20Charlesworth&amp;author=L.L.%20Cox&amp;author=A.M.%20MacNicol&amp;publication_year=2004&amp;journal=J.%20Biol.%20Chem&amp;volume=279&amp;pages=17650-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M313837200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M313837200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M313837200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14752101" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Alizadeh</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Kageyama</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Aoki</div> <div class="given-names">F.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Degradation of maternal mRNA in mouse embryos: selective degradation of specific mRNAs after fertilization</div>. <div class="source">Mol. Reprod. Dev</div> ., <div class="volume">72</div>, <div class="fpage">281</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Degradation%20of%20maternal%20mRNA%20in%20mouse%20embryos%3A%20selective%20degradation%20of%20specific%20mRNAs%20after%20fertilization&amp;author=Z.%20Alizadeh&amp;author=S.%20Kageyama&amp;author=F.%20Aoki&amp;publication_year=2005&amp;journal=Mol.%20Reprod.%20Dev&amp;volume=72&amp;pages=281-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1098-2795" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1098-2795" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1098-2795"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16094646" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sakurai</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Sato</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kimura</div> <div class="given-names">M.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Diverse patterns of poly(A) tail elongation and shortening of murine maternal mRNAs from fully grown oocyte to 2-cell embryo stages</div>. <div class="source">Biochem. Biophys. Res. Commun</div> ., <div class="volume">336</div>, <div class="fpage">1181</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diverse%20patterns%20of%20poly(A)%20tail%20elongation%20and%20shortening%20of%20murine%20maternal%20mRNAs%20from%20fully%20grown%20oocyte%20to%202-cell%20embryo%20stages&amp;author=T.%20Sakurai&amp;author=M.%20Sato&amp;author=M.%20Kimura&amp;publication_year=2005&amp;journal=Biochem.%20Biophys.%20Res.%20Commun&amp;volume=336&amp;pages=1181-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2005.08.250" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2005.08.250" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2005.08.250"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16169522" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Carlevaro-Fita</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Rahim</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Guigo</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Vardy</div> <div class="given-names">L.A.</div></div>, <div class="name string-name"><div class="surname">Johnson</div> <div class="given-names">R.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Cytoplasmic long noncoding RNAs are frequently bound to and degraded at ribosomes in human cells</div>. <div class="source">RNA</div> , <div class="volume">22</div>, <div class="fpage">867</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytoplasmic%20long%20noncoding%20RNAs%20are%20frequently%20bound%20to%20and%20degraded%20at%20ribosomes%20in%20human%20cells&amp;author=J.%20Carlevaro-Fita&amp;author=A.%20Rahim&amp;author=R.%20Guigo&amp;author=L.A.%20Vardy&amp;author=R.%20Johnson&amp;publication_year=2016&amp;journal=RNA&amp;volume=22&amp;pages=867-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.053561.115" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.053561.115" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.053561.115"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27090285" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ruiz-Orera</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Messeguer</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Subirana</div> <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Alba</div> <div class="given-names">M.M.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Long non-coding RNAs as a source of new peptides</div>. <div class="source">Elife</div> , <div class="volume">3</div>, <div class="fpage">e03523</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Long%20non-coding%20RNAs%20as%20a%20source%20of%20new%20peptides&amp;author=J.%20Ruiz-Orera&amp;author=X.%20Messeguer&amp;author=J.A.%20Subirana&amp;author=M.M.%20Alba&amp;publication_year=2014&amp;journal=Elife&amp;volume=3&amp;pages=e03523" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7554/eLife.03523" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7554%2FeLife.03523" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7554%2FeLife.03523"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25233276" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yue</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Breschi</div> <div class="given-names">A.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">A comparative encyclopedia of DNA elements in the mouse genome</div>. <div class="source">Nature</div> , <div class="volume">515</div>, <div class="fpage">355</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comparative%20encyclopedia%20of%20DNA%20elements%20in%20the%20mouse%20genome&amp;author=F.%20Yue&amp;author=Y.%20Cheng&amp;author=A.%20Breschi&amp;publication_year=2014&amp;journal=Nature&amp;volume=515&amp;pages=355-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature13992" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature13992" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature13992"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25409824" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gutschner</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Hammerle</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Diederichs</div> <div class="given-names">S.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">MALAT1 – a paradigm for long noncoding RNA function in cancer</div>. <div class="source">J. Mol. Med. (Berl)</div> , <div class="volume">91</div>, <div class="fpage">791</div>–<div class="lpage">801</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MALAT1%20%E2%80%93%20a%20paradigm%20for%20long%20noncoding%20RNA%20function%20in%20cancer&amp;author=T.%20Gutschner&amp;author=M.%20Hammerle&amp;author=S.%20Diederichs&amp;publication_year=2013&amp;journal=J.%20Mol.%20Med.%20(Berl)&amp;volume=91&amp;pages=791-801" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00109-013-1028-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00109-013-1028-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00109-013-1028-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23529762" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Eissmann</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Gutschner</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Hammerle</div> <div class="given-names">M.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Loss of the abundant nuclear non-coding RNA MALAT1 is compatible with life and development</div>. <div class="source">RNA Biol</div> ., <div class="volume">9</div>, <div class="fpage">1076</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Loss%20of%20the%20abundant%20nuclear%20non-coding%20RNA%20MALAT1%20is%20compatible%20with%20life%20and%20development&amp;author=M.%20Eissmann&amp;author=T.%20Gutschner&amp;author=M.%20Hammerle&amp;publication_year=2012&amp;journal=RNA%20Biol&amp;volume=9&amp;pages=1076-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/rna.21089" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Frna.21089" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Frna.21089"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22858678" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Arun</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Mao</div> <div class="given-names">Y. S.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">The lncRNA Malat1 is dispensable for mouse development but its transcription plays a cis-regulatory role in the adult</div>. <div class="source">Cell Rep</div> ., <div class="volume">2</div>, <div class="fpage">111</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20lncRNA%20Malat1%20is%20dispensable%20for%20mouse%20development%20but%20its%20transcription%20plays%20a%20cis-regulatory%20role%20in%20the%20adult&amp;author=B.%20Zhang&amp;author=G.%20Arun&amp;author=Y.%20S.%20Mao&amp;publication_year=2012&amp;journal=Cell%20Rep&amp;volume=2&amp;pages=111-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.celrep.2012.06.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.celrep.2012.06.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.celrep.2012.06.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22840402" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nakagawa</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ip</div> <div class="given-names">J. Y.</div></div>, <div class="name string-name"><div class="surname">Shioi</div> <div class="given-names">G.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Malat1 is not an essential component of nuclear speckles in mice</div>. <div class="source">RNA</div> , <div class="volume">18</div>, <div class="fpage">1487</div>–<div class="lpage">99</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Malat1%20is%20not%20an%20essential%20component%20of%20nuclear%20speckles%20in%20mice&amp;author=S.%20Nakagawa&amp;author=J.%20Y.%20Ip&amp;author=G.%20Shioi&amp;publication_year=2012&amp;journal=RNA&amp;volume=18&amp;pages=1487-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.033217.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.033217.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.033217.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22718948" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Watanabe</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Totoki</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Toyoda</div> <div class="given-names">A.</div></div></div>, et al.  . <div class="year">2008</div>, 
<div class="article-title">Endogenous siRNAs from naturally formed dsRNAs regulate transcripts in mouse oocytes</div>. <div class="source">Nature</div> , <div class="volume">453</div>, <div class="fpage">539</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Endogenous%20siRNAs%20from%20naturally%20formed%20dsRNAs%20regulate%20transcripts%20in%20mouse%20oocytes&amp;author=T.%20Watanabe&amp;author=Y.%20Totoki&amp;author=A.%20Toyoda&amp;publication_year=2008&amp;journal=Nature&amp;volume=453&amp;pages=539-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06908" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06908" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06908"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18404146" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Garcia-Lopez</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Alonso</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Cardenas</div> <div class="given-names">D.B.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Diversity and functional convergence of small noncoding RNAs in male germ cell differentiation and fertilization</div>. <div class="source">RNA</div> , <div class="volume">21</div>, <div class="fpage">946</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20and%20functional%20convergence%20of%20small%20noncoding%20RNAs%20in%20male%20germ%20cell%20differentiation%20and%20fertilization&amp;author=J.%20Garcia-Lopez&amp;author=L.%20Alonso&amp;author=D.B.%20Cardenas&amp;publication_year=2015&amp;journal=RNA&amp;volume=21&amp;pages=946-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.048215.114" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.048215.114" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.048215.114"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25805854" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Garcia-Lopez</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Hourcade Jde</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Alonso</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Cardenas</div> <div class="given-names">D.B.</div></div>, <div class="name string-name"><div class="surname">del Mazo</div> <div class="given-names">J.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Global characterization and target identification of piRNAs and endo-siRNAs in mouse gametes and zygotes</div>. <div class="source">Biochim. Biophys. Acta</div> , <div class="volume">1839</div>, <div class="fpage">463</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20characterization%20and%20target%20identification%20of%20piRNAs%20and%20endo-siRNAs%20in%20mouse%20gametes%20and%20zygotes&amp;author=J.%20Garcia-Lopez&amp;author=D.%20Hourcade%20Jde&amp;author=L.%20Alonso&amp;author=D.B.%20Cardenas&amp;author=J.%20del%20Mazo&amp;publication_year=2014&amp;journal=Biochim.%20Biophys.%20Acta&amp;volume=1839&amp;pages=463-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbagrm.2014.04.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbagrm.2014.04.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbagrm.2014.04.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24769224" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Svoboda</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Flemr</div> <div class="given-names">M.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">The role of miRNAs and endogenous siRNAs in maternal-to-zygotic reprogramming and the establishment of pluripotency</div>. <div class="source">EMBO Rep</div> ., <div class="volume">11</div>, <div class="fpage">590</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20miRNAs%20and%20endogenous%20siRNAs%20in%20maternal-to-zygotic%20reprogramming%20and%20the%20establishment%20of%20pluripotency&amp;author=P.%20Svoboda&amp;author=M.%20Flemr&amp;publication_year=2010&amp;journal=EMBO%20Rep&amp;volume=11&amp;pages=590-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/embor.2010.102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fembor.2010.102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fembor.2010.102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20651740" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ma</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Flemr</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Stein</div> <div class="given-names">P.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">MicroRNA activity is suppressed in mouse oocytes</div>. <div class="source">Curr. Biol</div> ., <div class="volume">20</div>, <div class="fpage">265</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MicroRNA%20activity%20is%20suppressed%20in%20mouse%20oocytes&amp;author=J.%20Ma&amp;author=M.%20Flemr&amp;author=P.%20Stein&amp;publication_year=2010&amp;journal=Curr.%20Biol&amp;volume=20&amp;pages=265-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cub.2009.12.042" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cub.2009.12.042" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cub.2009.12.042"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20116252" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Watanabe</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">E.C.</div></div>, <div class="name string-name"><div class="surname">Zhong</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Lin</div> <div class="given-names">H.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Retrotransposons and pseudogenes regulate mRNAs and lncRNAs via the piRNA pathway in the germline</div>. <div class="source">Genome Res</div> ., <div class="volume">25</div>, <div class="fpage">368</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Retrotransposons%20and%20pseudogenes%20regulate%20mRNAs%20and%20lncRNAs%20via%20the%20piRNA%20pathway%20in%20the%20germline&amp;author=T.%20Watanabe&amp;author=E.C.%20Cheng&amp;author=M.%20Zhong&amp;author=H.%20Lin&amp;publication_year=2015&amp;journal=Genome%20Res&amp;volume=25&amp;pages=368-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.180802.114" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.180802.114" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.180802.114"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25480952" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nejepinska</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Malik</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Filkowski</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Flemr</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Filipowicz</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Svoboda</div> <div class="given-names">P.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">dsRNA expression in the mouse elicits RNAi in oocytes and low adenosine deamination in somatic cells</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">399</div>–<div class="lpage">413</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=dsRNA%20expression%20in%20the%20mouse%20elicits%20RNAi%20in%20oocytes%20and%20low%20adenosine%20deamination%20in%20somatic%20cells&amp;author=J.%20Nejepinska&amp;author=R.%20Malik&amp;author=J.%20Filkowski&amp;author=M.%20Flemr&amp;author=W.%20Filipowicz&amp;author=P.%20Svoboda&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=399-413" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr702" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr702" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr702"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21908396" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw058-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw058-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hedges</div> <div class="given-names">S.B.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">A molecular timescale for vertebrate evolution</div>. <div class="source">Nature</div> , <div class="volume">392</div>, <div class="fpage">917</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20molecular%20timescale%20for%20vertebrate%20evolution&amp;author=S.%20Kumar&amp;author=S.B.%20Hedges&amp;publication_year=1998&amp;journal=Nature&amp;volume=392&amp;pages=917-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/31927" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F31927" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F31927"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9582070" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2901809&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw058/2/dsw058_Supp.zip?Expires=2147483647&amp;Signature=HM4NUYZbucKAOvCQqFu2PJyS4acHoUwYOmt4K9UORxR3pJrjRC49C6Re7bwmvuboJyK6oVNWSBgpBu04akr670UcGB6fZiHoOANNIepB0F3uW31wIsNX7InQaKyw8Ut5n3T0cGXp-fY9iIV~P5k7Q5vyo1wbHvVmhrKd6V5XmupzxeuXhXmP5R9HzkQsWgw-3qJW5Qr5uO9JWssHofUcCz~P2vpNPkuszfyK7TDsS41thIj1dg1oSb4I4bb9-akQ0bM~pz7dfDm5nLUTcut5FM~VtmSWgOeigQNRLdks54S3hjT5qfQsZcPu5AUvJSVd3JMXyCspNSSoontsSbmW~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,317</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000400913000003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">5</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw058"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2901809"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000400913000003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f2%2f129%2f2901809">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000400913000003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (5)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F2%2F129%2F2901809" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2901809'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Long non-coding RNA exchange during the oocyte-to-embryo transition in mice" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw058");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f129%2f2901809">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

