

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/3/221/368082","@type":"ScholarlyArticle","name":"Genome-Wide SNP Genotyping to Infer the Effects on Gene Functions in Tomato","datePublished":"2013-03-12","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2013-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst005","keywords":["single nucleotide polymorphism (SNP)","<italic>Solanum lycopersicum</italic>","infinium assay","goldengate assay","homology modelling"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/3/221/368082","author":[{"name":"Hirakawa, Hideki","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Shirasawa, Kenta","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Ohyama, Akio","affiliation":"2  National Agriculture and Food Research Organization ,  NARO Institute of Vegetable and Tea Science (NIVTS) ,  360 Kusawa, Ano, Tsu, Mie 514-2392 ,  Japan","@type":"Person"},{"name":"Fukuoka, Hiroyuki","affiliation":"2  National Agriculture and Food Research Organization ,  NARO Institute of Vegetable and Tea Science (NIVTS) ,  360 Kusawa, Ano, Tsu, Mie 514-2392 ,  Japan","@type":"Person"},{"name":"Aoki, Koh","affiliation":"3  Graduate School of Life &amp; Environmental Sciences ,  Osaka Prefecture University ,  1-1 Gakuen-cho, Nakaku, Sakai, Osaka 599-8531 ,  Japan","@type":"Person"},{"name":"Rothan, Christophe","affiliation":"4  Unité Mixte de Recherche 619, Biologie du Fruit ,  Institut National de la Recherche Agronomique (INRA Bordeaux), Universités de Bordeaux ,  33883 Villenave d'Ornon cedex ,  France","@type":"Person"},{"name":"Sato, Shusei","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Isobe, Sachiko","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Tabata, Satoshi","affiliation":"1  Kazusa DNA Research Institute ,  2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"}],"description":"Abstract.  The genotype data of 7054 single nucleotide polymorphism (SNP) loci in 40 tomato lines, including inbred lines, F1 hybrids, and wild relatives, were ","pageStart":"221","pageEnd":"233","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/3/1/m_cover.gif?Expires=2147483647&Signature=3JNT-3uWcoFSRSDjpyRS683cAWcv-uPBeo5zUjK0TFHyw5pdGPApuAnjwKw9hziMynOyeupAu8qXTBcWNNZ5zwCIK3wDotNETRf1C0gT1NeaWwrIBlHGIKu6bYD8JmeQgULL4qaQcRAHnyo7FyqKjoT8e0y4GLL7JPBIG8ClvgMIpmjvKRKJfK-rkhDpE9pXuYo8ex4C6o9vQhlNIWM8gIl4ygPsB2y8BMTIUXCEDS8KZ8W94~SHycoNAwx1VF1wN~gGUhCSNthMU3E6MsUONjgAq-FWmQ8kuAdsyKdxNSJy8mhuDD9SBTDoUvN5zzz~5gwbw0AeOX0S-zArQW9D7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome-Wide SNP Genotyping to Infer the Effects on Gene Functions in Tomato","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/3/1/m_cover.gif?Expires=2147483647&Signature=3JNT-3uWcoFSRSDjpyRS683cAWcv-uPBeo5zUjK0TFHyw5pdGPApuAnjwKw9hziMynOyeupAu8qXTBcWNNZ5zwCIK3wDotNETRf1C0gT1NeaWwrIBlHGIKu6bYD8JmeQgULL4qaQcRAHnyo7FyqKjoT8e0y4GLL7JPBIG8ClvgMIpmjvKRKJfK-rkhDpE9pXuYo8ex4C6o9vQhlNIWM8gIl4ygPsB2y8BMTIUXCEDS8KZ8W94~SHycoNAwx1VF1wN~gGUhCSNthMU3E6MsUONjgAq-FWmQ8kuAdsyKdxNSJy8mhuDD9SBTDoUvN5zzz~5gwbw0AeOX0S-zArQW9D7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome-Wide SNP Genotyping to Infer the Effects on Gene Functions in Tomato" />
    <meta property="og:description" content="Abstract.  The genotype data of 7054 single nucleotide polymorphism (SNP) loci in 40 tomato lines, including inbred lines, F1 hybrids, and wild relatives, were " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/3/221/368082" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-03-12" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Hirakawa, Hideki" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Shirasawa, Kenta" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Ohyama, Akio" /><meta name="citation_author_institution" content="National Agriculture and Food Research Organization, NARO Institute of Vegetable and Tea Science (NIVTS), 360 Kusawa, Ano, Tsu, Mie 514-2392, Japan" /><meta name="citation_author" content="Fukuoka, Hiroyuki" /><meta name="citation_author_institution" content="National Agriculture and Food Research Organization, NARO Institute of Vegetable and Tea Science (NIVTS), 360 Kusawa, Ano, Tsu, Mie 514-2392, Japan" /><meta name="citation_author" content="Aoki, Koh" /><meta name="citation_author_institution" content="Graduate School of Life &amp; Environmental Sciences, Osaka Prefecture University, 1-1 Gakuen-cho, Nakaku, Sakai, Osaka 599-8531, Japan" /><meta name="citation_author" content="Rothan, Christophe" /><meta name="citation_author_institution" content="Unité Mixte de Recherche 619, Biologie du Fruit, Institut National de la Recherche Agronomique (INRA Bordeaux), Universités de Bordeaux, 33883 Villenave d'Ornon cedex, France" /><meta name="citation_author" content="Sato, Shusei" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Isobe, Sachiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Tabata, Satoshi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_title" content="Genome-Wide SNP Genotyping to Infer the Effects on Gene Functions in Tomato" /><meta name="citation_firstpage" content="221" /><meta name="citation_lastpage" content="233" /><meta name="citation_doi" content="10.1093/dnares/dst005" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2013/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Next-generation sequencing technologies and their implications for crop genetics and breeding; Trends Biotechnol.;  citation_year=2009;  citation_volume=27;  citation_pages=522-30; " /><meta name="citation_reference" content="citation_title=Whole-genome sequencing of multiple Arabidopsis thaliana populations; Nat. Genet.;  citation_year=2011;  citation_volume=43;  citation_pages=956-63; " /><meta name="citation_reference" content="citation_title=Genome-wide SNP detection, validation, and development of an 8K SNP array for apple; PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e31745" /><meta name="citation_reference" content="citation_title=F-SNP: computationally predicted functional SNPs for disease association studies; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D820-4; " /><meta name="citation_reference" content="citation_title=SNPs3D: candidate gene and SNP selection for association studies; BMC Bioinformatics;  citation_year=2006;  citation_volume=7;  citation_pages=166" /><meta name="citation_reference" content="citation_title=An efficient computational method for screening functional SNPs in plants; J. Theor. Biol.;  citation_year=2010;  citation_volume=265;  citation_pages=55-62; " /><meta name="citation_reference" content="citation_title=The tomato genome sequence provides insights into fleshy fruit evolution; citation_author=Tomato Genome Consortium; Nature;  citation_year=2012;  citation_volume=485;  citation_pages=635-41; " /><meta name="citation_reference" content="citation_title=Single nucleotide polymorphism discovery in cultivated tomato via sequencing by synthesis; The Plant Genome;  citation_year=2012;  citation_volume=5;  citation_pages=17-29; " /><meta name="citation_reference" content="citation_title=High-density SNP genotyping of tomato (Solanum lycopersicum L.) reveals patterns of genetic variation due to breeding; PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e45520" /><meta name="citation_reference" content="citation_title=The Tomato Sequencing Project, the first cornerstone of the International Solanaceae Project (SOL); Comp. Funct. Genomics;  citation_year=2005;  citation_volume=6;  citation_pages=153-8; " /><meta name="citation_reference" content="citation_title=MiBASE: a database of a miniature tomato cultivar Micro-Tom; Plant Biotechnol. J.;  citation_year=2006;  citation_volume=23;  citation_pages=195-8; " /><meta name="citation_reference" content="citation_title=SNP discovery and linkage map construction in cultivated tomato; DNA Res.;  citation_year=2010;  citation_volume=17;  citation_pages=381-91; " /><meta name="citation_reference" content="citation_title=Inferring HIV transmission dynamics from phylogenetic sequence relationships; PLoS Med.;  citation_year=2008;  citation_volume=5;  citation_pages=e69" /><meta name="citation_reference" content="citation_title=MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods; Mol. Biol. Evol.;  citation_year=2011;  citation_volume=28;  citation_pages=2731-9; " /><meta name="citation_reference" content="citation_title=TASSEL: software for association mapping of complex traits in diverse samples; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2633-5; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=Improved tools for biological sequence comparison; Proc. Natl. Acad. Sci. USA;  citation_year=1988;  citation_volume=85;  citation_pages=2444-8; " /><meta name="citation_reference" content="citation_title=Comparative protein structure modeling using Modeller; citation_text=Chapter 2: Unit 2.9; Curr. Protoc. Bioinformatics;  citation_year=2006; " /><meta name="citation_reference" content="citation_title=Fpocket: an open source platform for ligand pocket detection; BMC Bioinformatics;  citation_year=2009;  citation_volume=10;  citation_pages=168" /><meta name="citation_reference" content="citation_title=TAIR: a resource for integrated Arabidopsis data; Funct. Integr. Genomics;  citation_year=2002;  citation_volume=2;  citation_pages=239-53; " /><meta name="citation_reference" content="citation_title=The COG database: an updated version includes eukaryotes; BMC Bioinformatics;  citation_year=2003;  citation_volume=4;  citation_pages=41-54; " /><meta name="citation_reference" content="citation_title=KEGG: Kyoto Encyclopedia of Genes and Genomes; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=29-34; " /><meta name="citation_reference" content="citation_title=The Pfam protein families database; Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=D290-301; " /><meta name="citation_reference" content="citation_title=Accelerated profile HMM searches; PLoS Comput. Biol.;  citation_year=2011;  citation_volume=7;  citation_pages=e1002195" /><meta name="citation_reference" content="citation_title=‘Tomato Chuukanbohon Nou 11’, a tomato parental line with a short-internode trait; Bull. Natl. Inst. Veg. Tea Sci.;  citation_year=2007;  citation_volume=6;  citation_pages=65-76; " /><meta name="citation_reference" content="citation_title=Micro-Tom A miniature dwarf tomato; Florida Agr. Expt. Sta. Circ.;  citation_year=1989;  citation_volume=370;  citation_pages=1-6; " /><meta name="citation_reference" content="citation_title=A high throughput system for transposon tagging and promoter trapping in tomato; Plant J.;  citation_year=2000;  citation_volume=22;  citation_pages=265-74; " /><meta name="citation_reference" content="citation_title=Combined use of male sterility and resistance to tobacco mosaic virus in tomato; Ann. Amelio. Plant;  citation_year=1976;  citation_volume=26;  citation_pages=517-21; " /><meta name="citation_reference" content="citation_title=A recombination hotspot delimits a wild-species quantitative trait locus for tomato sugar content to 484 bp within an invertase gene; Proc. Natl. Acad. Sci. USA;  citation_year=2000;  citation_volume=97;  citation_pages=4718-23; " /><meta name="citation_reference" content="citation_title=Zooming in on a quantitative trait for tomato yield using interspecific introgressions; Science;  citation_year=2004;  citation_volume=305;  citation_pages=1786-9; " /><meta name="citation_reference" content="citation_title=Protein modeling and active site binding mode interactions of myrosinase-sinigrin in Brassica juncea–an in silico approach; J. Mol. Graph. Model;  citation_year=2011;  citation_volume=29;  citation_pages=740-6; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/3/221/1049411/dst005.pdf" /><meta name="description" content="Abstract.  The genotype data of 7054 single nucleotide polymorphism (SNP) loci in 40 tomato lines, including inbred lines, F1 hybrids, and wild relatives, were " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/3/221/368082" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/3/221/368082" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/3/221/368082");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst005';
    </script>
<title>
	Genome-Wide SNP Genotyping to Infer the Effects on Gene Functions in Tomato | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./368082" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BchNPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIaXJha2F3YSwgSGlkZWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1LYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNoaXJhc2F3YSwgS2VudGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLUthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT2h5YW1hLCBBa2lvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgQWdyaWN1bHR1cmUgYW5kIEZvb2QgUmVzZWFyY2ggT3JnYW5pemF0aW9uLCBOQVJPIEluc3RpdHV0ZSBvZiBWZWdldGFibGUgYW5kIFRlYSBTY2llbmNlIChOSVZUUyksIDM2MCBLdXNhd2EsIEFubywgVHN1LCBNaWUgNTE0LTIzOTIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRnVrdW9rYSwgSGlyb3l1a2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBBZ3JpY3VsdHVyZSBhbmQgRm9vZCBSZXNlYXJjaCBPcmdhbml6YXRpb24sIE5BUk8gSW5zdGl0dXRlIG9mIFZlZ2V0YWJsZSBhbmQgVGVhIFNjaWVuY2UgKE5JVlRTKSwgMzYwIEt1c2F3YSwgQW5vLCBUc3UsIE1pZSA1MTQtMjM5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBb2tpLCBLb2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgTGlmZSAmYW1wOyBFbnZpcm9ubWVudGFsIFNjaWVuY2VzLCBPc2FrYSBQcmVmZWN0dXJlIFVuaXZlcnNpdHksIDEtMSBHYWt1ZW4tY2hvLCBOYWtha3UsIFNha2FpLCBPc2FrYSA1OTktODUzMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJSb3RoYW4sIENocmlzdG9waGUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJVbml0w6kgTWl4dGUgZGUgUmVjaGVyY2hlIDYxOSwgQmlvbG9naWUgZHUgRnJ1aXQsIEluc3RpdHV0IE5hdGlvbmFsIGRlIGxhIFJlY2hlcmNoZSBBZ3Jvbm9taXF1ZSAoSU5SQSBCb3JkZWF1eCksIFVuaXZlcnNpdMOpcyBkZSBCb3JkZWF1eCwgMzM4ODMgVmlsbGVuYXZlIGQnT3Jub24gY2VkZXgsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNhdG8sIFNodXNlaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJc29iZSwgU2FjaGlrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWJhdGEsIFNhdG9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLUthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJHZW5vbWUtV2lkZSBTTlAgR2Vub3R5cGluZyB0byBJbmZlciB0aGUgRWZmZWN0cyBvbiBHZW5lIEZ1bmN0aW9ucyBpbiBUb21hdG8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjIzMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdDAwNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEzLzA2LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmcgdGVjaG5vbG9naWVzIGFuZCB0aGVpciBpbXBsaWNhdGlvbnMgZm9yIGNyb3AgZ2VuZXRpY3MgYW5kIGJyZWVkaW5nOyBUcmVuZHMgQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz01MjItMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdob2xlLWdlbm9tZSBzZXF1ZW5jaW5nIG9mIG11bHRpcGxlIEFyYWJpZG9wc2lzIHRoYWxpYW5hIHBvcHVsYXRpb25zOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTk1Ni02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgU05QIGRldGVjdGlvbiwgdmFsaWRhdGlvbiwgYW5kIGRldmVsb3BtZW50IG9mIGFuIDhLIFNOUCBhcnJheSBmb3IgYXBwbGU7IFBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lMzE3NDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GLVNOUDogY29tcHV0YXRpb25hbGx5IHByZWRpY3RlZCBmdW5jdGlvbmFsIFNOUHMgZm9yIGRpc2Vhc2UgYXNzb2NpYXRpb24gc3R1ZGllczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9RDgyMC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TTlBzM0Q6IGNhbmRpZGF0ZSBnZW5lIGFuZCBTTlAgc2VsZWN0aW9uIGZvciBhc3NvY2lhdGlvbiBzdHVkaWVzOyBCTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTE2NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGVmZmljaWVudCBjb21wdXRhdGlvbmFsIG1ldGhvZCBmb3Igc2NyZWVuaW5nIGZ1bmN0aW9uYWwgU05QcyBpbiBwbGFudHM7IEouIFRoZW9yLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjU7ICBjaXRhdGlvbl9wYWdlcz01NS02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHRvbWF0byBnZW5vbWUgc2VxdWVuY2UgcHJvdmlkZXMgaW5zaWdodHMgaW50byBmbGVzaHkgZnJ1aXQgZXZvbHV0aW9uOyBjaXRhdGlvbl9hdXRob3I9VG9tYXRvIEdlbm9tZSBDb25zb3J0aXVtOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDg1OyAgY2l0YXRpb25fcGFnZXM9NjM1LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gZGlzY292ZXJ5IGluIGN1bHRpdmF0ZWQgdG9tYXRvIHZpYSBzZXF1ZW5jaW5nIGJ5IHN5bnRoZXNpczsgVGhlIFBsYW50IEdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTctMjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2gtZGVuc2l0eSBTTlAgZ2Vub3R5cGluZyBvZiB0b21hdG8gKFNvbGFudW0gbHljb3BlcnNpY3VtIEwuKSByZXZlYWxzIHBhdHRlcm5zIG9mIGdlbmV0aWMgdmFyaWF0aW9uIGR1ZSB0byBicmVlZGluZzsgUExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWU0NTUyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBUb21hdG8gU2VxdWVuY2luZyBQcm9qZWN0LCB0aGUgZmlyc3QgY29ybmVyc3RvbmUgb2YgdGhlIEludGVybmF0aW9uYWwgU29sYW5hY2VhZSBQcm9qZWN0IChTT0wpOyBDb21wLiBGdW5jdC4gR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTE1My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaUJBU0U6IGEgZGF0YWJhc2Ugb2YgYSBtaW5pYXR1cmUgdG9tYXRvIGN1bHRpdmFyIE1pY3JvLVRvbTsgUGxhbnQgQmlvdGVjaG5vbC4gSi47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xOTUtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U05QIGRpc2NvdmVyeSBhbmQgbGlua2FnZSBtYXAgY29uc3RydWN0aW9uIGluIGN1bHRpdmF0ZWQgdG9tYXRvOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTM4MS05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5mZXJyaW5nIEhJViB0cmFuc21pc3Npb24gZHluYW1pY3MgZnJvbSBwaHlsb2dlbmV0aWMgc2VxdWVuY2UgcmVsYXRpb25zaGlwczsgUExvUyBNZWQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lNjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NRUdBNTogbW9sZWN1bGFyIGV2b2x1dGlvbmFyeSBnZW5ldGljcyBhbmFseXNpcyB1c2luZyBtYXhpbXVtIGxpa2VsaWhvb2QsIGV2b2x1dGlvbmFyeSBkaXN0YW5jZSwgYW5kIG1heGltdW0gcGFyc2ltb255IG1ldGhvZHM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0yNzMxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRBU1NFTDogc29mdHdhcmUgZm9yIGFzc29jaWF0aW9uIG1hcHBpbmcgb2YgY29tcGxleCB0cmFpdHMgaW4gZGl2ZXJzZSBzYW1wbGVzOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTI2MzMtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2FwcGVkIEJMQVNUIGFuZCBQU0ktQkxBU1Q6IGEgbmV3IGdlbmVyYXRpb24gb2YgcHJvdGVpbiBkYXRhYmFzZSBzZWFyY2ggcHJvZ3JhbXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTMzODktNDAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbXByb3ZlZCB0b29scyBmb3IgYmlvbG9naWNhbCBzZXF1ZW5jZSBjb21wYXJpc29uOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT04NTsgIGNpdGF0aW9uX3BhZ2VzPTI0NDQtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgcHJvdGVpbiBzdHJ1Y3R1cmUgbW9kZWxpbmcgdXNpbmcgTW9kZWxsZXI7IGNpdGF0aW9uX3RleHQ9Q2hhcHRlciAyOiBVbml0IDIuOTsgQ3Vyci4gUHJvdG9jLiBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnBvY2tldDogYW4gb3BlbiBzb3VyY2UgcGxhdGZvcm0gZm9yIGxpZ2FuZCBwb2NrZXQgZGV0ZWN0aW9uOyBCTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xNjgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UQUlSOiBhIHJlc291cmNlIGZvciBpbnRlZ3JhdGVkIEFyYWJpZG9wc2lzIGRhdGE7IEZ1bmN0LiBJbnRlZ3IuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0yMzktNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBDT0cgZGF0YWJhc2U6IGFuIHVwZGF0ZWQgdmVyc2lvbiBpbmNsdWRlcyBldWthcnlvdGVzOyBCTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTQxLTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1LRUdHOiBLeW90byBFbmN5Y2xvcGVkaWEgb2YgR2VuZXMgYW5kIEdlbm9tZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTI5LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgUGZhbSBwcm90ZWluIGZhbWlsaWVzIGRhdGFiYXNlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz1EMjkwLTMwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWNjZWxlcmF0ZWQgcHJvZmlsZSBITU0gc2VhcmNoZXM7IFBMb1MgQ29tcHV0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDIxOTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT3igJhUb21hdG8gQ2h1dWthbmJvaG9uIE5vdSAxMeKAmSwgYSB0b21hdG8gcGFyZW50YWwgbGluZSB3aXRoIGEgc2hvcnQtaW50ZXJub2RlIHRyYWl0OyBCdWxsLiBOYXRsLiBJbnN0LiBWZWcuIFRlYSBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz02NS03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm8tVG9tIEEgbWluaWF0dXJlIGR3YXJmIHRvbWF0bzsgRmxvcmlkYSBBZ3IuIEV4cHQuIFN0YS4gQ2lyYy47ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl92b2x1bWU9MzcwOyAgY2l0YXRpb25fcGFnZXM9MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGhpZ2ggdGhyb3VnaHB1dCBzeXN0ZW0gZm9yIHRyYW5zcG9zb24gdGFnZ2luZyBhbmQgcHJvbW90ZXIgdHJhcHBpbmcgaW4gdG9tYXRvOyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTI2NS03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tYmluZWQgdXNlIG9mIG1hbGUgc3RlcmlsaXR5IGFuZCByZXNpc3RhbmNlIHRvIHRvYmFjY28gbW9zYWljIHZpcnVzIGluIHRvbWF0bzsgQW5uLiBBbWVsaW8uIFBsYW50OyAgY2l0YXRpb25feWVhcj0xOTc2OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9NTE3LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHJlY29tYmluYXRpb24gaG90c3BvdCBkZWxpbWl0cyBhIHdpbGQtc3BlY2llcyBxdWFudGl0YXRpdmUgdHJhaXQgbG9jdXMgZm9yIHRvbWF0byBzdWdhciBjb250ZW50IHRvIDQ4NCBicCB3aXRoaW4gYW4gaW52ZXJ0YXNlIGdlbmU7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTk3OyAgY2l0YXRpb25fcGFnZXM9NDcxOC0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Wm9vbWluZyBpbiBvbiBhIHF1YW50aXRhdGl2ZSB0cmFpdCBmb3IgdG9tYXRvIHlpZWxkIHVzaW5nIGludGVyc3BlY2lmaWMgaW50cm9ncmVzc2lvbnM7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzA1OyAgY2l0YXRpb25fcGFnZXM9MTc4Ni05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZWluIG1vZGVsaW5nIGFuZCBhY3RpdmUgc2l0ZSBiaW5kaW5nIG1vZGUgaW50ZXJhY3Rpb25zIG9mIG15cm9zaW5hc2Utc2luaWdyaW4gaW4gQnJhc3NpY2EganVuY2Vh4oCTYW4gaW4gc2lsaWNvIGFwcHJvYWNoOyBKLiBNb2wuIEdyYXBoLiBNb2RlbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTc0MC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjAvMy8yMjEvMTA0OTQxMS9kc3QwMDUucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFRoZSBnZW5vdHlwZSBkYXRhIG9mIDcwNTQgc2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtIChTTlApIGxvY2kgaW4gNDAgdG9tYXRvIGxpbmVzLCBpbmNsdWRpbmcgaW5icmVkIGxpbmVzLCBGMSBoeWJyaWRzLCBhbmQgd2lsZCByZWxhdGl2ZXMsIHdlcmUgIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNjgwODImeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkiWuovA+43y1tSAsZ3YrWNmJrfbA=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f3%2f221%2f368082" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f3%2f221%2f368082" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=14337" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/3/1/m_cover.gif?Expires=2147483647&amp;Signature=3JNT-3uWcoFSRSDjpyRS683cAWcv-uPBeo5zUjK0TFHyw5pdGPApuAnjwKw9hziMynOyeupAu8qXTBcWNNZ5zwCIK3wDotNETRf1C0gT1NeaWwrIBlHGIKu6bYD8JmeQgULL4qaQcRAHnyo7FyqKjoT8e0y4GLL7JPBIG8ClvgMIpmjvKRKJfK-rkhDpE9pXuYo8ex4C6o9vQhlNIWM8gIl4ygPsB2y8BMTIUXCEDS8KZ8W94~SHycoNAwx1VF1wN~gGUhCSNthMU3E6MsUONjgAq-FWmQ8kuAdsyKdxNSJy8mhuDD9SBTDoUvN5zzz~5gwbw0AeOX0S-zArQW9D7g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/3">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5345162">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5345162">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5345166">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5345166">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5345177">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5345177">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5345239">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5345239">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5345377">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5345377">4. Conclusion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5345396">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5345396">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5345402">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5345402">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5345409">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5345409">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5345418">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5345418">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/3/209/367502">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/20/3/235/367771">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome-Wide SNP Genotyping to Infer the Effects on Gene Functions in Tomato
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Hirakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 438-52-3951. Fax. +81 438-52-3918. Email: <a href="mailto:hh@kazusa.or.jp" target="_blank">hh@kazusa.or.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Hirakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirakawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirakawa H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hideki Hirakawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenta Shirasawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenta+Shirasawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shirasawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shirasawa K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kenta Shirasawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akio Ohyama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">National Agriculture and Food Research Organization</div>, <div class="insititution">NARO Institute of Vegetable and Tea Science (NIVTS)</div>, <div class="addr-line">360 Kusawa, Ano, Tsu, Mie 514-2392</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akio+Ohyama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohyama A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohyama A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Akio Ohyama</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroyuki Fukuoka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">National Agriculture and Food Research Organization</div>, <div class="insititution">NARO Institute of Vegetable and Tea Science (NIVTS)</div>, <div class="addr-line">360 Kusawa, Ano, Tsu, Mie 514-2392</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroyuki+Fukuoka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fukuoka H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fukuoka H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiroyuki Fukuoka</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Koh Aoki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Graduate School of Life &amp; Environmental Sciences</div>, <div class="insititution">Osaka Prefecture University</div>, <div class="addr-line">1-1 Gakuen-cho, Nakaku, Sakai, Osaka 599-8531</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Koh+Aoki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Aoki K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Aoki K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Koh Aoki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Christophe Rothan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Unité Mixte de Recherche 619, Biologie du Fruit</div>, <div class="insititution">Institut National de la Recherche Agronomique (INRA Bordeaux), Universités de Bordeaux</div>, <div class="addr-line">33883 Villenave d'Ornon cedex</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Christophe+Rothan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rothan C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Rothan C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Christophe Rothan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shusei Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shusei+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shusei Sato</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sachiko Isobe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sachiko+Isobe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Isobe S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Isobe S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sachiko Isobe</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Tabata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-Kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Tabata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tabata S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tabata S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Satoshi Tabata</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 3, 1 June 2013, Pages 221–233, <a href='https://doi.org/10.1093/dnares/dst005'>https://doi.org/10.1093/dnares/dst005</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">12 March 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">09 December 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">14 February 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="368082" href="/dnaresearch/article-pdf/20/3/221/1049411/dst005.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Hideki Hirakawa, Kenta Shirasawa, Akio Ohyama, Hiroyuki Fukuoka, Koh Aoki, Christophe Rothan, Shusei Sato, Sachiko Isobe, Satoshi Tabata;  Genome-Wide SNP Genotyping to Infer the Effects on Gene Functions in Tomato, <em>DNA Research</em>, Volume 20, Issue 3, 1 June 2013, Pages 221–233, <a href='https://doi.org/10.1093/dnares/dst005'>https://doi.org/10.1093/dnares/dst005</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/368082?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/368082?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/368082?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/368082?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/368082?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/368082?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/368082?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome-Wide%20SNP%20Genotyping%20to%20Infer%20the%20Effects%20on%20Gene%20Functions%20in%20Tomato&amp;publicationDate=2013-03-12&amp;volumeNum=20&amp;issueNum=3&amp;author=Hirakawa%2C%20Hideki%3B%20Shirasawa%2C%20Kenta&amp;startPage=221&amp;endPage=233&amp;contentId=10.1093%2Fdnares%2Fdst005&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f3%2f221%2f368082" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5345162 id="5345162" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The genotype data of 7054 single nucleotide polymorphism (SNP) loci in 40 tomato lines, including inbred lines, F<sub>1</sub> hybrids, and wild relatives, were collected using Illumina's Infinium and GoldenGate assay platforms, the latter of which was utilized in our previous study. The dendrogram based on the genotype data corresponded well to the breeding types of tomato and wild relatives. The SNPs were classified into six categories according to their positions in the genes predicted on the tomato genome sequence. The genes with SNPs were annotated by homology searches against the nucleotide and protein databases, as well as by domain searches, and they were classified into the functional categories defined by the NCBI's eukaryotic orthologous groups (KOG). To infer the SNPs' effects on the gene functions, the three-dimensional structures of the 843 proteins that were encoded by the genes with SNPs causing missense mutations were constructed by homology modelling, and 200 of these proteins were considered to carry non-synonymous amino acid substitutions in the predicted functional sites. The SNP information obtained in this study is available at the Kazusa Tomato Genomics Database (<a class="link link-uri" href="http://plant1.kazusa.or.jp/tomato/">http://plant1.kazusa.or.jp/tomato/</a>).</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;single nucleotide polymorphism (SNP)&quot;">single nucleotide polymorphism (SNP)</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Solanum lycopersicum&quot;"><em>Solanum lycopersicum</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;infinium assay&quot;">infinium assay</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;goldengate assay&quot;">goldengate assay</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;homology modelling&quot;">homology modelling</a></div>                            <h2 scrollto-destination=5345166 id="5345166" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The emergence of massive parallel sequencers known as next generation sequencers (NGSs) has drastically changed DNA sequencing strategy and has accelerated the collection of genome sequences not only of unicellular and model plants, but also of plants that have large and complicated genomes. Moreover, NGSs can be used to detect huge numbers of sequence variations or polymorphisms, e.g. single nucleotide polymorphisms (SNPs), insertions and deletions (indels), and copy-number variations (CNV), among different species, germplasms and varieties.<sup><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a></sup> The usual strategy for detecting such variations using NGSs is to re-sequence portions of the genome by mapping the sequence reads from the NGSs onto the accurate reference genome sequences.<sup><a reveal-id="DST005C2" data-open="DST005C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Along with the advances associated with NGSs, high-throughput SNP genotyping methods based on microbeads or microarrays have also been developed, e.g. GoldenGate and Infinium assays (Illumina Inc., San Diego, CA, USA) and GeneChip Mapping/SNP arrays (Affymetrix, Santa Clara, CA, USA). Using various combinations of these technologies for large-scale DNA sequencing and genotyping, genome-wide genetic studies have been extended from models to various plant species.<sup><a reveal-id="DST005C3" data-open="DST005C3" class="link link-ref link-reveal xref-bibr">3</a></sup></p><p>SNPs are the most abundant DNA polymorphisms in genome sequences and are thought to play major roles in the induction of phenotypic variations. In addition, the positions of SNPs on genome sequences affect gene expression and functions. In particular, nonsense mutations in exon regions are considered to be the most critical to phenotypic variations, and missense mutations accompanied by non-synonymous amino acid substitutions might influence protein functions. Moreover, the substitutions at splice junction sites produce truncated proteins. SNPs in upstream and downstream regions might cause phenotypic variations because they often activate or suppress gene expression by causing substitutions in regulatory sequences, e.g. promoters and terminators. Therefore, investigations of the SNPs related to functional changes of genes are considered essential for clarifying the functional genomics of organisms, including plants. In human genetics, the categorized SNPs associated with human diseases have been released from the F-SNP database (<a class="link link-uri" href="http://compbio.cs.queensu.ca/F-SNP/">http://compbio.cs.queensu.ca/F-SNP/</a>).<sup><a reveal-id="DST005C4" data-open="DST005C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Based on the support vector machine models, the sequence profiles and the structural stability of proteins whose functions are changed by non-synonymous substitutions have also been determined and are reported on the SNP3D website (<a class="link link-uri" href="http://www.snps3d.org">http://www.snps3d.org</a>).<sup><a reveal-id="DST005C5" data-open="DST005C5" class="link link-ref link-reveal xref-bibr">5</a></sup> In plants, the functional SNPs on ‘granule bound starch synthase I’ (<em>GBSSI</em>) have been screened by computational analyses including sequence alignments and the modelling of three-dimensional structures.<sup><a reveal-id="DST005C6" data-open="DST005C6" class="link link-ref link-reveal xref-bibr">6</a></sup></p><p>Cultivated tomato (<em>Solanum lycopersicum</em>) is one of the world's most important vegetable crops. It is characterized by a self-crossing reproduction system and a diploid genome (2<em>n</em> = 2X = 24) consisting of approximately 950 Mb. The whole-genome sequence of tomato was published recently by the International Tomato Genome Consortium,<sup><a reveal-id="DST005C7" data-open="DST005C7" class="link link-ref link-reveal xref-bibr">7</a></sup> and a total of 34 727 protein-coding genes were predicted by the International Tomato Annotation Group (ITAG) (<a class="link link-uri" href="http://www.uk-sol.org">http://www.uk-sol.org</a>). Using the whole-genome sequence as a reference, a large number of SNPs in tomato have been identified by the re-sequencing strategy.<sup><a reveal-id="DST005C8" data-open="DST005C8" class="link link-ref link-reveal xref-bibr">8</a></sup> In a subsequent study, 7720 genome-wide SNPs were genotyped for 426 tomato lines, consisting of 410 inbred and 16 hybrid lines, using a high-throughput genotyping system, Illumina's Infinium assay. The SNPs on the assay were called ‘scorable SNPs’ because they could be used to score the tomato germplasm in genotyping.<sup><a reveal-id="DST005C9" data-open="DST005C9" class="link link-ref link-reveal xref-bibr">9</a></sup> In our previous study, we discovered 1337 SNPs by a comparative analysis of transcriptome sequences derived from Micro-Tom and 19 tomato lines [SOL Genomics Network (SGN),<sup><a reveal-id="DST005C10" data-open="DST005C10" class="link link-ref link-reveal xref-bibr">10</a></sup><a class="link link-uri" href="http://solgenomics.net">http://solgenomics.net</a>; MiBASE,<sup><a reveal-id="DST005C11" data-open="DST005C11" class="link link-ref link-reveal xref-bibr">11</a></sup><a class="link link-uri" href="http://www.pgb.kazusa.or.jp/mibase/">http://www.pgb.kazusa.or.jp/mibase/</a>] and used them for polymorphism analysis of 27 tomato lines by the GoldenGate assay.<sup><a reveal-id="DST005C12" data-open="DST005C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Although a large number of SNPs in tomato have been discovered in previous studies, their functions have not been surveyed.</p><p>In this study, we used the Infinium assay developed by Sim et al.<sup><a reveal-id="DST005C9" data-open="DST005C9" class="link link-ref link-reveal xref-bibr">9</a></sup> to perform SNP genotyping for 40 tomato lines, including the 27 lines used in our previous study.<sup><a reveal-id="DST005C12" data-open="DST005C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The SNP genotype data obtained in the present and previous studies<sup><a reveal-id="DST005C12" data-open="DST005C12" class="link link-ref link-reveal xref-bibr">12</a></sup> were merged. To infer their effects on gene functions, SNPs can be classified into six categories: cSNPs (SNPs in a coding region that cause amino acid substitution), sSNPs (SNPs in a coding region that do not cause amino acid substitution), iSNPs (SNPs in an intron region), uSNPs (SNPs within the 1 Kb region upstream from the start codon), dSNPs (SNPs within the 1 Kb region downstream from the stop codon), and gSNPs (SNPs in intergenic regions). Furthermore, the three-dimensional structures of proteins encoded by genes having cSNPs were constructed by homology modelling. Based on the constructed protein structures, candidates for the functional sites were predicted to identify the SNPs that directly affect protein functions. We expected that the SNPs located on the candidate functional sites of proteins would be applicable to gene functional analysis and molecular marker-assisted selection as functional markers.</p>                            <h2 scrollto-destination=5345177 id="5345177" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5345181 id="5345181" class="section-title" data-legacy-id=s2a>2.1. Plant materials</h3>
<p>Forty tomato lines, consisting of 30 lines of cultivated tomato (<em>S. lycopersicum</em>), 2 lines of <em>S. lycopersicum</em> var. cerasiforme, and 8 lines of wild relatives (2 each of <em>Solanum pimpinellifolium</em>, <em>Solanum pennellii</em>, <em>Solanum peruvianum</em>, and <em>Solanum chilense</em>), were used for the genotyping assays. The 30 <em>S. lycopersicum</em> lines included 5 ‘Micro-Tom’ lines that were obtained from different resource centres. Micro-Tom is the experimental line most frequently used in the study of cultivated tomato. Natural variations were found in our previous study within two lines<em>,</em> one derived from NIVTS, Japan and the other from INRA, France.<sup><a reveal-id="DST005C12" data-open="DST005C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Two lines of ‘M82’ derived from different sources (NIVTS and INRA) were also included in the 30 <em>S. lycopersicum</em> lines to investigate the existence of SNPs within M82 because M82 has been widely used in tomato genetics. The names of the lines, accession numbers, sources, and breeding types are listed in Table <a reveal-id="DST005TB1" data-open="DST005TB1" class="link link-reveal link-table xref-fig">1</a>. All lines were grown in a greenhouse, and the DNA of each line was isolated from leaves using a DNeasy Plant Mini Kit (Qiagen, Hilden, Germany). </p>                        <a id="5345186" scrollto-destination="5345186"></a>
<div content-id="DST005TB1" class="table-modal"><div class="table-wrap"><div id="DST005TB1" class="table-wrap-title" data-id="DST005TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Descriptions of the plant materials</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Line names </th><th>Accession Number </th><th>Sources<sup>a</sup> </th><th>Breeding type </th><th>Array type </th><th>Descriptions </th><th>References<sup>b</sup> </th></tr></thead><tbody><tr><td><em>S. lycopersicum</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td>Micro-Tom MM </td><td> </td><td>INRA </td><td>Inbred </td><td>I &amp; G </td><td>A parental line of MMF2 </td><td>1 </td></tr><tr><td> </td><td>Micro-Tom KDRI </td><td> </td><td>KDRI </td><td>Inbred </td><td>I </td><td>Genome sequenced line </td><td><a reveal-id="DST005C2" data-open="DST005C2" class="link link-ref link-reveal xref-bibr">2</a> </td></tr><tr><td> </td><td>Micro-Tom NBRP (TOMJPF00001) </td><td>TOMJPF00001 </td><td>NBRP </td><td>Inbred </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td>Micro-Tom AM </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td>A parental line of AMF2 </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Micro-Tom TGRC (LA3911) </td><td>LA3911 </td><td>TGRC </td><td>Inbred </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td>Ailsa Craig </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td>A parental line of AMF2 </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Aichi First </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Best Of All </td><td>LS3908 </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Earliana </td><td>LA3238 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Fruit </td><td>LS1100 </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Furikoma </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Heinz 1706-BG </td><td>LA4345 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td>Genome sequenced line </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST005C3" data-open="DST005C3" class="link link-ref link-reveal xref-bibr">3</a> </td></tr><tr><td> </td><td>LA925 </td><td>LA925 </td><td>CU </td><td>Inbred </td><td>I &amp; G </td><td>A parental line of Tomato-EXPEN2000 </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST005C4" data-open="DST005C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DST005C5" data-open="DST005C5" class="link link-ref link-reveal xref-bibr">5</a> </td></tr><tr><td> </td><td>M82_MM </td><td> </td><td>INRA </td><td>Inbred </td><td>I &amp; G </td><td>A parental line of MMF2 </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>M82 </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td>Marglobe </td><td>LA0502 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Money Maker </td><td>LA2706 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Tomato Chuukanbohon Nou 11 </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Tomato Chuukanbohon Nou 9 </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Ponderosa </td><td>LS1728 </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Rio Grande </td><td>LA3343 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Rutgers </td><td>LA1090 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>San Marzano </td><td>LS4956 </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Geronimo </td><td> </td><td>De Ruiter Seeds Co. </td><td>F<sub>1</sub> hybrid </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Labell </td><td> </td><td>De Ruiter Seeds Co. </td><td>F<sub>1</sub> hybrid </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Matrix </td><td> </td><td>De Ruiter Seeds Co. </td><td>F<sub>1</sub> hybrid </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Momotaro 8 </td><td> </td><td>Takii Seeds Co. </td><td>F<sub>1</sub> hybrid </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Reika </td><td> </td><td>Sakata Seeds Co. </td><td>F<sub>1</sub> hybrid </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Sweet 100 </td><td> </td><td>Vilmorin Seeds Co. </td><td>F<sub>1</sub> hybrid, cherry type </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Regina </td><td> </td><td>Sakata Seeds Co. </td><td>Inbred, cherry type </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td><em>S. lycopersicum</em> var. cerasiforme </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td>var. cerasiforme (TOMJPF00009) </td><td>TOMJPF00009 </td><td>NBRP </td><td> </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td>var. cerasiforme (WIR25/30) </td><td>WIR25/30 </td><td>NIVTS </td><td> </td><td>I </td><td>Canadian line </td><td> </td></tr><tr><td><em>S. chilense</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. chilense</em> (LA1958) </td><td>LA1958 80L </td><td>NIVTS </td><td> </td><td>I </td><td> </td><td> </td></tr><tr><td><em>S. pennellii</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. pennellii</em> (LA716) </td><td>LA716 </td><td>CU </td><td> </td><td>I &amp; G </td><td>A parental line of Tomato-EXPEN2000 </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST005C4" data-open="DST005C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DST005C5" data-open="DST005C5" class="link link-ref link-reveal xref-bibr">5</a> </td></tr><tr><td> </td><td><em>S. pennellii</em> (TOMJPF00008) </td><td>TOMJPF00008 </td><td>NBRP </td><td> </td><td>I </td><td> </td><td> </td></tr><tr><td><em>S. peruvianum</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. peruvianum</em> (TOMJPF00010) </td><td>TOMJPF00010 </td><td>NBRP </td><td> </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. peruvianum</em> (WIR2020) </td><td>WIR2020 </td><td>NIVTS </td><td> </td><td>I </td><td>American line </td><td> </td></tr><tr><td><em>S. pimpinellifolium</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. pimpinellifolium</em> (TOMJPF00012) </td><td>TOMJPF00012 </td><td>NBRP </td><td> </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. pimpinellifolium</em> (PI344102) </td><td>PI344102 </td><td>NIVTS </td><td> </td><td>I </td><td>American line </td><td> </td></tr><tr><td> </td><td><em>S. pimpinellifolium</em> (LS1513) </td><td>LS1513 </td><td>NIVTS </td><td> </td><td>I </td><td>Peru line </td><td> </td></tr></tbody></table></div><div content-id="DST005TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Line names </th><th>Accession Number </th><th>Sources<sup>a</sup> </th><th>Breeding type </th><th>Array type </th><th>Descriptions </th><th>References<sup>b</sup> </th></tr></thead><tbody><tr><td><em>S. lycopersicum</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td>Micro-Tom MM </td><td> </td><td>INRA </td><td>Inbred </td><td>I &amp; G </td><td>A parental line of MMF2 </td><td>1 </td></tr><tr><td> </td><td>Micro-Tom KDRI </td><td> </td><td>KDRI </td><td>Inbred </td><td>I </td><td>Genome sequenced line </td><td><a reveal-id="DST005C2" data-open="DST005C2" class="link link-ref link-reveal xref-bibr">2</a> </td></tr><tr><td> </td><td>Micro-Tom NBRP (TOMJPF00001) </td><td>TOMJPF00001 </td><td>NBRP </td><td>Inbred </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td>Micro-Tom AM </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td>A parental line of AMF2 </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Micro-Tom TGRC (LA3911) </td><td>LA3911 </td><td>TGRC </td><td>Inbred </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td>Ailsa Craig </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td>A parental line of AMF2 </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Aichi First </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Best Of All </td><td>LS3908 </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Earliana </td><td>LA3238 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Fruit </td><td>LS1100 </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Furikoma </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Heinz 1706-BG </td><td>LA4345 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td>Genome sequenced line </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST005C3" data-open="DST005C3" class="link link-ref link-reveal xref-bibr">3</a> </td></tr><tr><td> </td><td>LA925 </td><td>LA925 </td><td>CU </td><td>Inbred </td><td>I &amp; G </td><td>A parental line of Tomato-EXPEN2000 </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST005C4" data-open="DST005C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DST005C5" data-open="DST005C5" class="link link-ref link-reveal xref-bibr">5</a> </td></tr><tr><td> </td><td>M82_MM </td><td> </td><td>INRA </td><td>Inbred </td><td>I &amp; G </td><td>A parental line of MMF2 </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>M82 </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td>Marglobe </td><td>LA0502 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Money Maker </td><td>LA2706 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Tomato Chuukanbohon Nou 11 </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Tomato Chuukanbohon Nou 9 </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Ponderosa </td><td>LS1728 </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Rio Grande </td><td>LA3343 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Rutgers </td><td>LA1090 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>San Marzano </td><td>LS4956 </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Geronimo </td><td> </td><td>De Ruiter Seeds Co. </td><td>F<sub>1</sub> hybrid </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Labell </td><td> </td><td>De Ruiter Seeds Co. </td><td>F<sub>1</sub> hybrid </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Matrix </td><td> </td><td>De Ruiter Seeds Co. </td><td>F<sub>1</sub> hybrid </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Momotaro 8 </td><td> </td><td>Takii Seeds Co. </td><td>F<sub>1</sub> hybrid </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Reika </td><td> </td><td>Sakata Seeds Co. </td><td>F<sub>1</sub> hybrid </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Sweet 100 </td><td> </td><td>Vilmorin Seeds Co. </td><td>F<sub>1</sub> hybrid, cherry type </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Regina </td><td> </td><td>Sakata Seeds Co. </td><td>Inbred, cherry type </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td><em>S. lycopersicum</em> var. cerasiforme </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td>var. cerasiforme (TOMJPF00009) </td><td>TOMJPF00009 </td><td>NBRP </td><td> </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td>var. cerasiforme (WIR25/30) </td><td>WIR25/30 </td><td>NIVTS </td><td> </td><td>I </td><td>Canadian line </td><td> </td></tr><tr><td><em>S. chilense</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. chilense</em> (LA1958) </td><td>LA1958 80L </td><td>NIVTS </td><td> </td><td>I </td><td> </td><td> </td></tr><tr><td><em>S. pennellii</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. pennellii</em> (LA716) </td><td>LA716 </td><td>CU </td><td> </td><td>I &amp; G </td><td>A parental line of Tomato-EXPEN2000 </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST005C4" data-open="DST005C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DST005C5" data-open="DST005C5" class="link link-ref link-reveal xref-bibr">5</a> </td></tr><tr><td> </td><td><em>S. pennellii</em> (TOMJPF00008) </td><td>TOMJPF00008 </td><td>NBRP </td><td> </td><td>I </td><td> </td><td> </td></tr><tr><td><em>S. peruvianum</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. peruvianum</em> (TOMJPF00010) </td><td>TOMJPF00010 </td><td>NBRP </td><td> </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. peruvianum</em> (WIR2020) </td><td>WIR2020 </td><td>NIVTS </td><td> </td><td>I </td><td>American line </td><td> </td></tr><tr><td><em>S. pimpinellifolium</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. pimpinellifolium</em> (TOMJPF00012) </td><td>TOMJPF00012 </td><td>NBRP </td><td> </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. pimpinellifolium</em> (PI344102) </td><td>PI344102 </td><td>NIVTS </td><td> </td><td>I </td><td>American line </td><td> </td></tr><tr><td> </td><td><em>S. pimpinellifolium</em> (LS1513) </td><td>LS1513 </td><td>NIVTS </td><td> </td><td>I </td><td>Peru line </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>CU, Cornell University; INRA, Institut National de la Recherche Agronomique; KDRI, Kazusa DNA Research Institute; NBRP, National BioResource Project, University of Tsukuba; NIVTS, National Institute of Vegetable and Tea Sciences, National Agriculture and Food Research Organization; TGRC, Tomato Genetic Resource Center, University of California, Davis.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>1: Shirasawa et al. (2010) <em>DNA Res</em>., 17, 381-391.; 2: Aoki et al. (2010) <em>BMC Genomics</em>, 11, 210.; 3: Tomato Genome Consortium (2012) Nature, 485, 635-641.; 4: Shirasawa et al. (2010) Theor. Appl. Genet., 121,731-739.; 5: Fulton et al. (2002) <em>Plant Cell</em>, 14,1457-1467.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5345186" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST005TB1" class="table-wrap-title" data-id="DST005TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Descriptions of the plant materials</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Line names </th><th>Accession Number </th><th>Sources<sup>a</sup> </th><th>Breeding type </th><th>Array type </th><th>Descriptions </th><th>References<sup>b</sup> </th></tr></thead><tbody><tr><td><em>S. lycopersicum</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td>Micro-Tom MM </td><td> </td><td>INRA </td><td>Inbred </td><td>I &amp; G </td><td>A parental line of MMF2 </td><td>1 </td></tr><tr><td> </td><td>Micro-Tom KDRI </td><td> </td><td>KDRI </td><td>Inbred </td><td>I </td><td>Genome sequenced line </td><td><a reveal-id="DST005C2" data-open="DST005C2" class="link link-ref link-reveal xref-bibr">2</a> </td></tr><tr><td> </td><td>Micro-Tom NBRP (TOMJPF00001) </td><td>TOMJPF00001 </td><td>NBRP </td><td>Inbred </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td>Micro-Tom AM </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td>A parental line of AMF2 </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Micro-Tom TGRC (LA3911) </td><td>LA3911 </td><td>TGRC </td><td>Inbred </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td>Ailsa Craig </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td>A parental line of AMF2 </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Aichi First </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Best Of All </td><td>LS3908 </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Earliana </td><td>LA3238 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Fruit </td><td>LS1100 </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Furikoma </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Heinz 1706-BG </td><td>LA4345 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td>Genome sequenced line </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST005C3" data-open="DST005C3" class="link link-ref link-reveal xref-bibr">3</a> </td></tr><tr><td> </td><td>LA925 </td><td>LA925 </td><td>CU </td><td>Inbred </td><td>I &amp; G </td><td>A parental line of Tomato-EXPEN2000 </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST005C4" data-open="DST005C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DST005C5" data-open="DST005C5" class="link link-ref link-reveal xref-bibr">5</a> </td></tr><tr><td> </td><td>M82_MM </td><td> </td><td>INRA </td><td>Inbred </td><td>I &amp; G </td><td>A parental line of MMF2 </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>M82 </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td>Marglobe </td><td>LA0502 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Money Maker </td><td>LA2706 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Tomato Chuukanbohon Nou 11 </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Tomato Chuukanbohon Nou 9 </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Ponderosa </td><td>LS1728 </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Rio Grande </td><td>LA3343 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Rutgers </td><td>LA1090 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>San Marzano </td><td>LS4956 </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Geronimo </td><td> </td><td>De Ruiter Seeds Co. </td><td>F<sub>1</sub> hybrid </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Labell </td><td> </td><td>De Ruiter Seeds Co. </td><td>F<sub>1</sub> hybrid </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Matrix </td><td> </td><td>De Ruiter Seeds Co. </td><td>F<sub>1</sub> hybrid </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Momotaro 8 </td><td> </td><td>Takii Seeds Co. </td><td>F<sub>1</sub> hybrid </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Reika </td><td> </td><td>Sakata Seeds Co. </td><td>F<sub>1</sub> hybrid </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Sweet 100 </td><td> </td><td>Vilmorin Seeds Co. </td><td>F<sub>1</sub> hybrid, cherry type </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Regina </td><td> </td><td>Sakata Seeds Co. </td><td>Inbred, cherry type </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td><em>S. lycopersicum</em> var. cerasiforme </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td>var. cerasiforme (TOMJPF00009) </td><td>TOMJPF00009 </td><td>NBRP </td><td> </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td>var. cerasiforme (WIR25/30) </td><td>WIR25/30 </td><td>NIVTS </td><td> </td><td>I </td><td>Canadian line </td><td> </td></tr><tr><td><em>S. chilense</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. chilense</em> (LA1958) </td><td>LA1958 80L </td><td>NIVTS </td><td> </td><td>I </td><td> </td><td> </td></tr><tr><td><em>S. pennellii</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. pennellii</em> (LA716) </td><td>LA716 </td><td>CU </td><td> </td><td>I &amp; G </td><td>A parental line of Tomato-EXPEN2000 </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST005C4" data-open="DST005C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DST005C5" data-open="DST005C5" class="link link-ref link-reveal xref-bibr">5</a> </td></tr><tr><td> </td><td><em>S. pennellii</em> (TOMJPF00008) </td><td>TOMJPF00008 </td><td>NBRP </td><td> </td><td>I </td><td> </td><td> </td></tr><tr><td><em>S. peruvianum</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. peruvianum</em> (TOMJPF00010) </td><td>TOMJPF00010 </td><td>NBRP </td><td> </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. peruvianum</em> (WIR2020) </td><td>WIR2020 </td><td>NIVTS </td><td> </td><td>I </td><td>American line </td><td> </td></tr><tr><td><em>S. pimpinellifolium</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. pimpinellifolium</em> (TOMJPF00012) </td><td>TOMJPF00012 </td><td>NBRP </td><td> </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. pimpinellifolium</em> (PI344102) </td><td>PI344102 </td><td>NIVTS </td><td> </td><td>I </td><td>American line </td><td> </td></tr><tr><td> </td><td><em>S. pimpinellifolium</em> (LS1513) </td><td>LS1513 </td><td>NIVTS </td><td> </td><td>I </td><td>Peru line </td><td> </td></tr></tbody></table></div><div content-id="DST005TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Line names </th><th>Accession Number </th><th>Sources<sup>a</sup> </th><th>Breeding type </th><th>Array type </th><th>Descriptions </th><th>References<sup>b</sup> </th></tr></thead><tbody><tr><td><em>S. lycopersicum</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td>Micro-Tom MM </td><td> </td><td>INRA </td><td>Inbred </td><td>I &amp; G </td><td>A parental line of MMF2 </td><td>1 </td></tr><tr><td> </td><td>Micro-Tom KDRI </td><td> </td><td>KDRI </td><td>Inbred </td><td>I </td><td>Genome sequenced line </td><td><a reveal-id="DST005C2" data-open="DST005C2" class="link link-ref link-reveal xref-bibr">2</a> </td></tr><tr><td> </td><td>Micro-Tom NBRP (TOMJPF00001) </td><td>TOMJPF00001 </td><td>NBRP </td><td>Inbred </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td>Micro-Tom AM </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td>A parental line of AMF2 </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Micro-Tom TGRC (LA3911) </td><td>LA3911 </td><td>TGRC </td><td>Inbred </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td>Ailsa Craig </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td>A parental line of AMF2 </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Aichi First </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Best Of All </td><td>LS3908 </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Earliana </td><td>LA3238 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Fruit </td><td>LS1100 </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Furikoma </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Heinz 1706-BG </td><td>LA4345 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td>Genome sequenced line </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST005C3" data-open="DST005C3" class="link link-ref link-reveal xref-bibr">3</a> </td></tr><tr><td> </td><td>LA925 </td><td>LA925 </td><td>CU </td><td>Inbred </td><td>I &amp; G </td><td>A parental line of Tomato-EXPEN2000 </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST005C4" data-open="DST005C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DST005C5" data-open="DST005C5" class="link link-ref link-reveal xref-bibr">5</a> </td></tr><tr><td> </td><td>M82_MM </td><td> </td><td>INRA </td><td>Inbred </td><td>I &amp; G </td><td>A parental line of MMF2 </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>M82 </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td>Marglobe </td><td>LA0502 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Money Maker </td><td>LA2706 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Tomato Chuukanbohon Nou 11 </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Tomato Chuukanbohon Nou 9 </td><td> </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Ponderosa </td><td>LS1728 </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Rio Grande </td><td>LA3343 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Rutgers </td><td>LA1090 </td><td>TGRC </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>San Marzano </td><td>LS4956 </td><td>NIVTS </td><td>Inbred </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Geronimo </td><td> </td><td>De Ruiter Seeds Co. </td><td>F<sub>1</sub> hybrid </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Labell </td><td> </td><td>De Ruiter Seeds Co. </td><td>F<sub>1</sub> hybrid </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Matrix </td><td> </td><td>De Ruiter Seeds Co. </td><td>F<sub>1</sub> hybrid </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Momotaro 8 </td><td> </td><td>Takii Seeds Co. </td><td>F<sub>1</sub> hybrid </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Reika </td><td> </td><td>Sakata Seeds Co. </td><td>F<sub>1</sub> hybrid </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Sweet 100 </td><td> </td><td>Vilmorin Seeds Co. </td><td>F<sub>1</sub> hybrid, cherry type </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td> </td><td>Regina </td><td> </td><td>Sakata Seeds Co. </td><td>Inbred, cherry type </td><td>I &amp; G </td><td> </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a> </td></tr><tr><td><em>S. lycopersicum</em> var. cerasiforme </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td>var. cerasiforme (TOMJPF00009) </td><td>TOMJPF00009 </td><td>NBRP </td><td> </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td>var. cerasiforme (WIR25/30) </td><td>WIR25/30 </td><td>NIVTS </td><td> </td><td>I </td><td>Canadian line </td><td> </td></tr><tr><td><em>S. chilense</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. chilense</em> (LA1958) </td><td>LA1958 80L </td><td>NIVTS </td><td> </td><td>I </td><td> </td><td> </td></tr><tr><td><em>S. pennellii</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. pennellii</em> (LA716) </td><td>LA716 </td><td>CU </td><td> </td><td>I &amp; G </td><td>A parental line of Tomato-EXPEN2000 </td><td><a reveal-id="DST005C1" data-open="DST005C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DST005C4" data-open="DST005C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DST005C5" data-open="DST005C5" class="link link-ref link-reveal xref-bibr">5</a> </td></tr><tr><td> </td><td><em>S. pennellii</em> (TOMJPF00008) </td><td>TOMJPF00008 </td><td>NBRP </td><td> </td><td>I </td><td> </td><td> </td></tr><tr><td><em>S. peruvianum</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. peruvianum</em> (TOMJPF00010) </td><td>TOMJPF00010 </td><td>NBRP </td><td> </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. peruvianum</em> (WIR2020) </td><td>WIR2020 </td><td>NIVTS </td><td> </td><td>I </td><td>American line </td><td> </td></tr><tr><td><em>S. pimpinellifolium</em> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. pimpinellifolium</em> (TOMJPF00012) </td><td>TOMJPF00012 </td><td>NBRP </td><td> </td><td>I </td><td> </td><td> </td></tr><tr><td> </td><td><em>S. pimpinellifolium</em> (PI344102) </td><td>PI344102 </td><td>NIVTS </td><td> </td><td>I </td><td>American line </td><td> </td></tr><tr><td> </td><td><em>S. pimpinellifolium</em> (LS1513) </td><td>LS1513 </td><td>NIVTS </td><td> </td><td>I </td><td>Peru line </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>CU, Cornell University; INRA, Institut National de la Recherche Agronomique; KDRI, Kazusa DNA Research Institute; NBRP, National BioResource Project, University of Tsukuba; NIVTS, National Institute of Vegetable and Tea Sciences, National Agriculture and Food Research Organization; TGRC, Tomato Genetic Resource Center, University of California, Davis.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>1: Shirasawa et al. (2010) <em>DNA Res</em>., 17, 381-391.; 2: Aoki et al. (2010) <em>BMC Genomics</em>, 11, 210.; 3: Tomato Genome Consortium (2012) Nature, 485, 635-641.; 4: Shirasawa et al. (2010) Theor. Appl. Genet., 121,731-739.; 5: Fulton et al. (2002) <em>Plant Cell</em>, 14,1457-1467.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5345186" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5345188 id="5345188" class="section-title" data-legacy-id=s2b>2.2. SNP genotyping</h3>
<p>Genome-wide SNP genotyping was performed using the Infinium assay (Illumina Inc.) that was developed by the Solanaceae Coordinated Agricultural Project (SolCAP),<sup><a reveal-id="DST005C8" data-open="DST005C8" class="link link-ref link-reveal xref-bibr">8</a></sup> according to the manufacturer's standard protocol. The probe sequences and SNP information are available from SolCAP (<a class="link link-uri" href="http://solcap.msu.edu">http://solcap.msu.edu</a>) and in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. The data obtained from the assays were analysed using the Genotyping Module of the GenomeStudio software (Illumina Inc.).</p>                            <h3 scrollto-destination=5345195 id="5345195" class="section-title" data-legacy-id=s2c>2.3. Genetic diversity analysis and detection of linkage disequilibrium (LD)</h3>
<div class="block-child-p">The Hamming distances<sup><a reveal-id="DST005C13" data-open="DST005C13" class="link link-ref link-reveal xref-bibr">13</a></sup> between each pair of the 40 tomato lines tested were calculated to generate the distance matrix based on the genotyping data obtained in this study. A neighbour-joining tree was generated using MEGA 5 software.<sup><a reveal-id="DST005C14" data-open="DST005C14" class="link link-ref link-reveal xref-bibr">14</a></sup> The heterozygosity (HZ) values were calculated by the equation <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/dst005ueq1.gif?Expires=2147483647&Signature=co6MtOlXHrRqUT4LJuEda6635Bg5WrKI81aFmnXrJ1Sjk6oGv2NHNABqlV4iuxEm7GTJRUCpHYAfZzBwUAMYSg8Uc6xp~DSSqJ1QXgd4OdDU0iHzTmbT-btw1Rm51ELiG3wDXCWZwlw~0F6xFToO1OYXLzxbm6hHibrjK5nr9-6p5CrfFWFnC4FuAyG3EUjC~S-0eDmZ7NyephRdv9C2rX~vkSa-fXnQevTlVWLLM78dCKdVV6HHbdigfMMTXFR5byGNwkxIWdbS3QEkRTZj3PIi2f-G9Zuqfbo~aE70NluWiS~lFwLG4bFW-IJdO4JXz8ha8VBC0RHkL1Hds2BOmw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dst005ueq1.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/m_dst005ueq1.gif?Expires=2147483647&Signature=x7TqfsZJgtHJHTkIfSwITtb-ywayZ~BmjHm0vHsDCtj0M~eq06EuZKwcU7z2hl~K4gRHbfZnjtAJCqW~XTap~7MnoIV5xtLFWSi~oiw3-WH4d7AwqIPsEAknloV5tQokrR0FwdRHg1YJ2INShcN6qNWnZWVfuQssH~C9N8qxNooglGwVq3qx1RebgahEsuI5q~eFR6W3lrWi3zs86dlsl~FLqvAXZVTD4HpvoY4q8auu~WU0uFM6n0RXunPT8Oiy1sDHIhDSmL~zxIXG0g2AonzqmdIanEnlB1ckNvjxnAo9bI0bjmj87hQGY~lWSvZZvZM4KrX4z-~JYQLHLZpfFw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dst005ueq1.gif" /></a></div></div></div> where <em>p<sub>ij</sub></em> is the frequency of the <em>j</em>th of the <em>i</em> allele.</div><p>Linkage disequilibrium (LD) in the tomato lines was detected by the <em>r</em><sup>2</sup> and <em>D</em>′ values calculated using the Tassel program.<sup><a reveal-id="DST005C15" data-open="DST005C15" class="link link-ref link-reveal xref-bibr">15</a></sup></p>                            <h3 scrollto-destination=5345210 id="5345210" class="section-title" data-legacy-id=s2d>2.4. Classification of SNPs according to their locations on the genome sequence</h3>
<p>The locations of SNPs on the tomato genome genotyped using the two assay platforms, Infinium and GoldenGate, were identified by the following method. The probe sequences in both assays were searched against the tomato genome sequence SL2.40 using Blastn<sup><a reveal-id="DST005C16" data-open="DST005C16" class="link link-ref link-reveal xref-bibr">16</a></sup> with an E-value cutoff of 1E-20. According to the relative positions between the SNPs and the genes predicted on the tomato genome sequence, the SNPs were classified into six groups, as described above: cSNPs, sSNPs, iSNPs, uSNPs, dSNPs, and gSNPs.</p>                            <h3 scrollto-destination=5345217 id="5345217" class="section-title" data-legacy-id=s2e>2.5. Inference of SNP effects on protein and gene functions</h3>
<p>The amino acid sequences of the genes were searched against the Protein Data Bank (PDB) (<a class="link link-uri" href="http://www.rcsb.org/pdb/home/home.do">http://www.rcsb.org/pdb/home/home.do</a>) by Blastp<sup><a reveal-id="DST005C16" data-open="DST005C16" class="link link-ref link-reveal xref-bibr">16</a></sup> with an E-value cutoff of 1E-10. The amino acid sequences were further aligned on those of the homologous proteins by Fasta36 program,<sup><a reveal-id="DST005C17" data-open="DST005C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and homology modelling was performed by Modeller 9.10 software<sup><a reveal-id="DST005C18" data-open="DST005C18" class="link link-ref link-reveal xref-bibr">18</a></sup> based on the alignment with default parameters. The functional sites on the constructed protein structures were predicted using FPocket software<sup><a reveal-id="DST005C19" data-open="DST005C19" class="link link-ref link-reveal xref-bibr">19</a></sup> with default parameters. The functions of proteins were predicted by combining the results obtained by the Blastp searches with an E-value cutoff of 1E-10 against the databases NR (non-redundant amino acid sequence database released by NCBI) (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a>), TAIR10,<sup><a reveal-id="DST005C20" data-open="DST005C20" class="link link-ref link-reveal xref-bibr">20</a></sup> KOG,<sup><a reveal-id="DST005C21" data-open="DST005C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and KEGG.<sup><a reveal-id="DST005C22" data-open="DST005C22" class="link link-ref link-reveal xref-bibr">22</a></sup> The functional domains of proteins were searched against Pfam database release 26.0<sup><a reveal-id="DST005C23" data-open="DST005C23" class="link link-ref link-reveal xref-bibr">23</a></sup> using HMMscan program.<sup><a reveal-id="DST005C24" data-open="DST005C24" class="link link-ref link-reveal xref-bibr">24</a></sup></p>                            <h2 scrollto-destination=5345239 id="5345239" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5345243 id="5345243" class="section-title" data-legacy-id=s3a>3.1. SNP genotyping by high-throughput assay technologies</h3>
<p>Of the 7720 scorable SNPs defined by the Infinium assay, 7617 (98.7%) were successfully genotyped, and 6374 (82.6%) showed polymorphism among the 40 tested lines. In our previous study using the GoldenGate assay, 916 of 1338 SNP loci (68.5%) showed polymorphism among the 27 tomato lines.<sup><a reveal-id="DST005C12" data-open="DST005C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Among the 916 SNP loci on the GoldenGate assay, 236 were duplicated on the Infinium assay; therefore, the genotype data of the specific 680 SNP loci on the GoldenGate assay were analysed with the 6374 SNPs of the Infinium assay. In total, polymorphic data were successfully obtained at 7054 SNP loci. The numbers of SNPs identified between each pair of the 40 lines are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>. The average number of polymorphic loci between each pair of the 40 lines was 4270, whereas that between each pair of the 30 cultivated tomatoes, the 2 lines of <em>S. lycopersicum</em> var. cerasiforme, and the 5 lines of Micro-Tom, was 930.</p><p>The heterozygous alleles were found at 2292, 967, 921, 901, 772, and 1200 SNP loci within the F<sub>1</sub> hybrid varieties of ‘Sweet 100’, ‘Geronimo’, ‘Matrix’, ‘Labell’, ‘Momotaro 8’, and ‘Reika’, respectively. In two of the five lines of Micro-Tom, i.e. ‘Micro-Tom NBRP’ and ‘Micro-Tom KDRI,’ all genotypes were completely identical. In the other pairs of Micro-Tom, the numbers of identified SNPs ranged from 275 (between ‘Micro-Tom TGRC’ and Micro-Tom KDRI/Micro-Tom NBRP) to 1008 (between ‘Micro-Tom MM’ and Micro-Tom KDRI/Micro-Tom NBRP).</p>                            <h3 scrollto-destination=5345256 id="5345256" class="section-title" data-legacy-id=s3b>3.2. Classification of tomato lines based on genotyping data</h3>
<p>The genetic distances between all the combinations of any two lines were calculated based on the genotyping data of the 7054 SNP loci, and a dendrogram was then constructed (Fig. <a reveal-id="DST005F1" data-open="DST005F1" class="link link-reveal link-table xref-fig">1</a>). Of the 40 lines examined, 36 were classified into 4 large clusters, which consisted of 8 wild relatives (Cluster 1), 5 lines of Micro-Tom (Cluster 2), 5 F<sub>1</sub> hybrid cultivars, and an inbred line, ‘Tomato Chuukanbohon Nou 11′ (Cluster 3), and the 15 inbred lines and 2 lines of <em>S. lycopersicum</em> var. cerasiforme (Cluster 4). Because Momotaro 8 was used as a backcross parent in the breeding program of Tomato Chuukannbohon Nou 11,<sup><a reveal-id="DST005C25" data-open="DST005C25" class="link link-ref link-reveal xref-bibr">25</a></sup> 75% of the genome of the Tomato Chuukanbohon Nou 11 might be identical to that of Momotaro 8, one of the tested F<sub>1</sub> cultivars. Two cherry types, ‘Sweet 100’ and ‘Regina,’ and the two lines of M82 were separated from the four clusters. Sweet 100 showed the largest genetic distances in comparison with the other cultivated tomato lines. The graphical genotypes of the 40 tomato lines with the 7054 SNPs are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>. Most of the SNP loci were located on the ends of chromosomes corresponding to the euchromatic regions, where the gene density is higher than that on the heterochromatic regions, except for the three regions of the heterochromatins of chromosomes 3, 11, and 12. The biased locations of the SNPs on the genome have been found. These results are valid because most of the SNPs on the Infinium and GoldenGate assays were selected from the transcriptome sequences.<sup><a reveal-id="DST005C9" data-open="DST005C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DST005C12" data-open="DST005C12" class="link link-ref link-reveal xref-bibr">12</a></sup></p>                        <a id="5345264" scrollto-destination="5345264"></a>
<div id="DST005F1" data-id="DST005F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/m_dst00501.jpeg?Expires=2147483647&Signature=nkjicrxxGm~LYi53r2K35aYIYC90IByPE5pr3sas~iUbG4-MkJt~CdcEtUfiymoSpKdhyVaqHhyb9Pxl-6ySjLza0LZeWI0iY9AJgwiWmKl87h2MOI2hp0DVWRWLKtgHMVbQHdBYY3JTtYYsemSw5BTOEHctt9QmfZfa6SqJkr6YpjzA1NHYe5W9fGhgZbsq2AV0vCjXza7ErO7eptxdAil6~9agx4JBz9v4W4n5X10cgdOJWmRpVUTOG9gy40Uy2pm~wEL6qBrFzCNoND~Gc3U2P~rPSHZLedd4LFkTgXYsFnfHVSm41XZd~gjjeDJ6T8cA-uhNCT-ykAqWuUn5JQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A dendrogram for the 40 lines of tomatoes based on the 7054 SNP genotypes." path-from-xml="dst00501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5345264/dst00501.jpeg" path-from-xml="dst00501.jpeg" target="_blank">View large</a><a section="5345264" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/dst00501.jpeg?Expires=2147483647&Signature=rmth4DD3MEswTpeQn2nEWm1E368XLvraDpBSO9D~UKDWbDI3uq-fGM1O1z98loCjK5YyQgOV9~iUH3B8b62rnqnr5MQ-mySZfmAbRyp0li3lxKL0BxD-SMtQbSAdzpisoGHqo17M~2W-s7STYnbF0up6V4-3OFSq0SfIP7VYnIKSUweFJnqZ1qgz~lVIe1-56nAGh9OCWH8-EBgVxrywjJgeYv7Jm68PTlt81428DLCwH-3-kT-LlzuBxQzw5JqobZKfc1kIWFXxEZrFZmtl0rwUpPhLyzsYRGBYnAhkUxAD~1mZoPj4tFlenxIQ8eWLrKfVZQ3mDmj96qpmzzdEIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5345264&ar=368082&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A dendrogram for the 40 lines of tomatoes based on the 7054 SNP genotypes.</p></div></div><div content-id="DST005F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/m_dst00501.jpeg?Expires=2147483647&Signature=nkjicrxxGm~LYi53r2K35aYIYC90IByPE5pr3sas~iUbG4-MkJt~CdcEtUfiymoSpKdhyVaqHhyb9Pxl-6ySjLza0LZeWI0iY9AJgwiWmKl87h2MOI2hp0DVWRWLKtgHMVbQHdBYY3JTtYYsemSw5BTOEHctt9QmfZfa6SqJkr6YpjzA1NHYe5W9fGhgZbsq2AV0vCjXza7ErO7eptxdAil6~9agx4JBz9v4W4n5X10cgdOJWmRpVUTOG9gy40Uy2pm~wEL6qBrFzCNoND~Gc3U2P~rPSHZLedd4LFkTgXYsFnfHVSm41XZd~gjjeDJ6T8cA-uhNCT-ykAqWuUn5JQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A dendrogram for the 40 lines of tomatoes based on the 7054 SNP genotypes." path-from-xml="dst00501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5345264/dst00501.jpeg" path-from-xml="dst00501.jpeg" target="_blank">View large</a><a section="5345264" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/dst00501.jpeg?Expires=2147483647&Signature=rmth4DD3MEswTpeQn2nEWm1E368XLvraDpBSO9D~UKDWbDI3uq-fGM1O1z98loCjK5YyQgOV9~iUH3B8b62rnqnr5MQ-mySZfmAbRyp0li3lxKL0BxD-SMtQbSAdzpisoGHqo17M~2W-s7STYnbF0up6V4-3OFSq0SfIP7VYnIKSUweFJnqZ1qgz~lVIe1-56nAGh9OCWH8-EBgVxrywjJgeYv7Jm68PTlt81428DLCwH-3-kT-LlzuBxQzw5JqobZKfc1kIWFXxEZrFZmtl0rwUpPhLyzsYRGBYnAhkUxAD~1mZoPj4tFlenxIQ8eWLrKfVZQ3mDmj96qpmzzdEIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5345264&ar=368082&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A dendrogram for the 40 lines of tomatoes based on the 7054 SNP genotypes.</p></div></div><p>The HZ value of the 7054 SNP loci was 0.30 on average, ranging from 0.25 for chromosome 10 to 0.37 for chromosome 11 (Table <a reveal-id="DST005TB2" data-open="DST005TB2" class="link link-reveal link-table xref-fig">2</a>). The average HZ value of Cluster 1, which consisted of the eight wild species, was the highest among the four clusters. In Cluster 2, which included the five lines of Micro-Tom, the HZ values were greater than zero, except for chromosome 11. In other words, SNPs were observed on 11 of 12 chromosomes within the Micro-Tom lines. Micro-Tom was developed by Scott and Harbaugh in 1989<sup><a reveal-id="DST005C26" data-open="DST005C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and has been used as a model line in tomato genetics since 1997.<sup><a reveal-id="DST005C27" data-open="DST005C27" class="link link-ref link-reveal xref-bibr">27</a></sup> The heterozygous regions in the five Micro-Tom lines might be caused by the existence of heterozygous regions in the first generation of Micro-Tom or mutations that arose during generation changes. In Cluster 3, which included the F<sub>1</sub> hybrid cultivars, high HZ values were found in the large regions of chromosomes 6 (8.7–30.0 Mb), 9 (4.8–58.9 Mb), and 11 (7.7–49.3 Mb) and in small segmental regions on all chromosomes. On the other hand, in Cluster 4, which included the 16 inbred lines, high HZ values were found in chromosomes 3 (60.8–61.3 Mb), 7 (60.6–61.0 Mb), and 11 (3.5–4.6 Mb). The high HZ regions in chromosomes 3 (60.8–61.3 Mb) and 7 (60.6–61.0 Mb) overlapped between Clusters 3 and 4, whereas those in chromosome 11 were specific to Cluster 4. The tested F<sub>1</sub> hybrids might be an introgressed disease-resistance gene, <em>I-2</em>,<sup><a reveal-id="DST005C28" data-open="DST005C28" class="link link-ref link-reveal xref-bibr">28</a></sup> from wild relatives in the process of breeding. The gene is located on the specific high-HZ regions of Cluster 3 on chromosome 11. Therefore, we inferred that the genomic regions derived from wild relatives remained in the genomes of F<sub>1</sub> hybrids and caused high heterozygosity in the regions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). </p>                        <a id="5345277" scrollto-destination="5345277"></a>
<div content-id="DST005TB2" class="table-modal"><div class="table-wrap"><div id="DST005TB2" class="table-wrap-title" data-id="DST005TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Average HZ values in each chromosome investigated on the 40 tomato lines and the 4 clusters</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>Total </th><th>Cluster 1 </th><th>Cluster 2 </th><th>Cluster 3 </th><th>Cluster 4 </th></tr></thead><tbody><tr><td>1 </td><td>0.28 </td><td>0.15 </td><td>&gt;0.00 </td><td>0.11 </td><td>0.13 </td></tr><tr><td>2 </td><td>0.30 </td><td>0.20 </td><td>0.01 </td><td>0.09 </td><td>0.09 </td></tr><tr><td>3 </td><td>0.28 </td><td>0.16 </td><td>0.03 </td><td>0.10 </td><td>0.15 </td></tr><tr><td>4 </td><td>0.31 </td><td>0.18 </td><td>0.08 </td><td>0.17 </td><td>0.08 </td></tr><tr><td>5 </td><td>0.32 </td><td>0.20 </td><td>0.02 </td><td>0.09 </td><td>0.05 </td></tr><tr><td>6 </td><td>0.28 </td><td>0.23 </td><td>0.06 </td><td>0.22 </td><td>0.07 </td></tr><tr><td>7 </td><td>0.26 </td><td>0.18 </td><td>0.06 </td><td>0.09 </td><td>0.07 </td></tr><tr><td>8 </td><td>0.27 </td><td>0.18 </td><td>0.01 </td><td>0.05 </td><td>0.13 </td></tr><tr><td>9 </td><td>0.29 </td><td>0.17 </td><td>&gt;0.00 </td><td>0.18 </td><td>0.11 </td></tr><tr><td>10 </td><td>0.25 </td><td>0.21 </td><td>&gt;0.00 </td><td>0.08 </td><td>0.09 </td></tr><tr><td>11 </td><td>0.37 </td><td>0.20 </td><td>0.00 </td><td>0.18 </td><td>0.11 </td></tr><tr><td>12 </td><td>0.29 </td><td>0.16 </td><td>0.19 </td><td>0.16 </td><td>0.13 </td></tr><tr><td>0 </td><td>0.19 </td><td>0.00 </td><td>0.00 </td><td>0.05 </td><td>0.00 </td></tr><tr><td>Total </td><td>0.30 </td><td>0.19 </td><td>0.04 </td><td>0.13 </td><td>0.10 </td></tr></tbody></table></div><div content-id="DST005TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>Total </th><th>Cluster 1 </th><th>Cluster 2 </th><th>Cluster 3 </th><th>Cluster 4 </th></tr></thead><tbody><tr><td>1 </td><td>0.28 </td><td>0.15 </td><td>&gt;0.00 </td><td>0.11 </td><td>0.13 </td></tr><tr><td>2 </td><td>0.30 </td><td>0.20 </td><td>0.01 </td><td>0.09 </td><td>0.09 </td></tr><tr><td>3 </td><td>0.28 </td><td>0.16 </td><td>0.03 </td><td>0.10 </td><td>0.15 </td></tr><tr><td>4 </td><td>0.31 </td><td>0.18 </td><td>0.08 </td><td>0.17 </td><td>0.08 </td></tr><tr><td>5 </td><td>0.32 </td><td>0.20 </td><td>0.02 </td><td>0.09 </td><td>0.05 </td></tr><tr><td>6 </td><td>0.28 </td><td>0.23 </td><td>0.06 </td><td>0.22 </td><td>0.07 </td></tr><tr><td>7 </td><td>0.26 </td><td>0.18 </td><td>0.06 </td><td>0.09 </td><td>0.07 </td></tr><tr><td>8 </td><td>0.27 </td><td>0.18 </td><td>0.01 </td><td>0.05 </td><td>0.13 </td></tr><tr><td>9 </td><td>0.29 </td><td>0.17 </td><td>&gt;0.00 </td><td>0.18 </td><td>0.11 </td></tr><tr><td>10 </td><td>0.25 </td><td>0.21 </td><td>&gt;0.00 </td><td>0.08 </td><td>0.09 </td></tr><tr><td>11 </td><td>0.37 </td><td>0.20 </td><td>0.00 </td><td>0.18 </td><td>0.11 </td></tr><tr><td>12 </td><td>0.29 </td><td>0.16 </td><td>0.19 </td><td>0.16 </td><td>0.13 </td></tr><tr><td>0 </td><td>0.19 </td><td>0.00 </td><td>0.00 </td><td>0.05 </td><td>0.00 </td></tr><tr><td>Total </td><td>0.30 </td><td>0.19 </td><td>0.04 </td><td>0.13 </td><td>0.10 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5345277" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST005TB2" class="table-wrap-title" data-id="DST005TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Average HZ values in each chromosome investigated on the 40 tomato lines and the 4 clusters</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>Total </th><th>Cluster 1 </th><th>Cluster 2 </th><th>Cluster 3 </th><th>Cluster 4 </th></tr></thead><tbody><tr><td>1 </td><td>0.28 </td><td>0.15 </td><td>&gt;0.00 </td><td>0.11 </td><td>0.13 </td></tr><tr><td>2 </td><td>0.30 </td><td>0.20 </td><td>0.01 </td><td>0.09 </td><td>0.09 </td></tr><tr><td>3 </td><td>0.28 </td><td>0.16 </td><td>0.03 </td><td>0.10 </td><td>0.15 </td></tr><tr><td>4 </td><td>0.31 </td><td>0.18 </td><td>0.08 </td><td>0.17 </td><td>0.08 </td></tr><tr><td>5 </td><td>0.32 </td><td>0.20 </td><td>0.02 </td><td>0.09 </td><td>0.05 </td></tr><tr><td>6 </td><td>0.28 </td><td>0.23 </td><td>0.06 </td><td>0.22 </td><td>0.07 </td></tr><tr><td>7 </td><td>0.26 </td><td>0.18 </td><td>0.06 </td><td>0.09 </td><td>0.07 </td></tr><tr><td>8 </td><td>0.27 </td><td>0.18 </td><td>0.01 </td><td>0.05 </td><td>0.13 </td></tr><tr><td>9 </td><td>0.29 </td><td>0.17 </td><td>&gt;0.00 </td><td>0.18 </td><td>0.11 </td></tr><tr><td>10 </td><td>0.25 </td><td>0.21 </td><td>&gt;0.00 </td><td>0.08 </td><td>0.09 </td></tr><tr><td>11 </td><td>0.37 </td><td>0.20 </td><td>0.00 </td><td>0.18 </td><td>0.11 </td></tr><tr><td>12 </td><td>0.29 </td><td>0.16 </td><td>0.19 </td><td>0.16 </td><td>0.13 </td></tr><tr><td>0 </td><td>0.19 </td><td>0.00 </td><td>0.00 </td><td>0.05 </td><td>0.00 </td></tr><tr><td>Total </td><td>0.30 </td><td>0.19 </td><td>0.04 </td><td>0.13 </td><td>0.10 </td></tr></tbody></table></div><div content-id="DST005TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Chromosome </th><th>Total </th><th>Cluster 1 </th><th>Cluster 2 </th><th>Cluster 3 </th><th>Cluster 4 </th></tr></thead><tbody><tr><td>1 </td><td>0.28 </td><td>0.15 </td><td>&gt;0.00 </td><td>0.11 </td><td>0.13 </td></tr><tr><td>2 </td><td>0.30 </td><td>0.20 </td><td>0.01 </td><td>0.09 </td><td>0.09 </td></tr><tr><td>3 </td><td>0.28 </td><td>0.16 </td><td>0.03 </td><td>0.10 </td><td>0.15 </td></tr><tr><td>4 </td><td>0.31 </td><td>0.18 </td><td>0.08 </td><td>0.17 </td><td>0.08 </td></tr><tr><td>5 </td><td>0.32 </td><td>0.20 </td><td>0.02 </td><td>0.09 </td><td>0.05 </td></tr><tr><td>6 </td><td>0.28 </td><td>0.23 </td><td>0.06 </td><td>0.22 </td><td>0.07 </td></tr><tr><td>7 </td><td>0.26 </td><td>0.18 </td><td>0.06 </td><td>0.09 </td><td>0.07 </td></tr><tr><td>8 </td><td>0.27 </td><td>0.18 </td><td>0.01 </td><td>0.05 </td><td>0.13 </td></tr><tr><td>9 </td><td>0.29 </td><td>0.17 </td><td>&gt;0.00 </td><td>0.18 </td><td>0.11 </td></tr><tr><td>10 </td><td>0.25 </td><td>0.21 </td><td>&gt;0.00 </td><td>0.08 </td><td>0.09 </td></tr><tr><td>11 </td><td>0.37 </td><td>0.20 </td><td>0.00 </td><td>0.18 </td><td>0.11 </td></tr><tr><td>12 </td><td>0.29 </td><td>0.16 </td><td>0.19 </td><td>0.16 </td><td>0.13 </td></tr><tr><td>0 </td><td>0.19 </td><td>0.00 </td><td>0.00 </td><td>0.05 </td><td>0.00 </td></tr><tr><td>Total </td><td>0.30 </td><td>0.19 </td><td>0.04 </td><td>0.13 </td><td>0.10 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5345277" target="_blank">View Large</a></div></div><p>The LD on the 40 lines and on each cluster across the chromosomes is shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>. Although significant LD was not clearly observed along with the diagonal lines on any of the LD maps of the 40 tomato lines, high LD values between adjacent SNPs were segmentally observed across the genomes. In Cluster 1, high LD values were observed on all of the chromosomes, suggesting that the wild relatives were independently evolved with few inter-crossings. In Cluster 2, small LD blocks were found on chromosomes 3, 4, 6, 7, and 12 that were caused by heterozygous regions within the five Micro-Tom lines. In Clusters 3 and 4, no significant LD was observed between adjacent SNPs in most of the genomic regions. This suggested that the genomes of cultivated tomato have been well admixed in germplasm collections. Large LD blocks were found on chromosomes 4, 6, 9, and 11 of Cluster 3, whereas a clear large LD block was observed on chromosome 11 in Cluster 4. Within each block, higher LD values were observed along with the oblong bars, and not diagonally. This implied that selection biases had occurred in partial regions of the genomes during the breeding processes. For example, the genes controlling disease resistance are located on chromosomes 6 (<em>Cf-2</em> and <em>Mi</em>), 9 (<em>Tm2</em><sup>2</sup>), and 11 (<em>I-2</em>), where LD blocks were observed (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). The selection of specific alleles of these genes might induce the LD blocks.</p>                            <h3 scrollto-destination=5345285 id="5345285" class="section-title" data-legacy-id=s3c>3.3. Classification of SNPs based on the positions on the tomato genome</h3>
<p>The SNPs were classified into six categories according to their positions on the tomato genome sequence (Fig. <a reveal-id="DST005F2" data-open="DST005F2" class="link link-reveal link-table xref-fig">2</a>). The numbers of cSNPs and sSNPs located within the coding regions were 2212 (31.4%) and 2296 (32.5%), respectively. The numbers of uSNPs and dSNPs were 516 (7.3%) and 1311 (18.6%), respectively. There were fewer iSNPs and gSNPs than the other types of SNPs, i.e. 485 (6.9%) and 234 (3.3%), respectively, because the SNPs on the probes of the assays were selected from the transcriptome sequences.<sup><a reveal-id="DST005C9" data-open="DST005C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DST005C12" data-open="DST005C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The total percentage of cSNPs, uSNPs, and dSNPs having the capacity to alter gene functions was 57.3%. Most of these SNPs were located on the gene-rich euchromatic regions in each chromosome (Fig. <a reveal-id="DST005F3" data-open="DST005F3" class="link link-reveal link-table xref-fig">3</a>). There were no significant differences in the distribution on the genome among the six SNP categories. </p>                        <a id="5345291" scrollto-destination="5345291"></a>
<div id="DST005F2" data-id="DST005F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/m_dst00502.jpeg?Expires=2147483647&Signature=lizpbvcLpTfK7EKUa9ZuvC6Ve-4w~scdKQD3f-5ln1F-Z7HwOOZLD2HDF03IC9Vux-rmrlK9psYcizPrq5pI-t0-n0EQomDtWg9Yo1NgNfuPic-L3T2hDRXdBhCzN-BjLR0~ZkjzFpZbhsKmqgOwHtrH-ZXfFB1yDOuJtqp1IRRXzW7z4vqBYXjJJUMmuL0dIgvJWZ2wxGeelYE~pThn9EMBylf5BxNv6bGDVTPDEin-Gs9mI9XQ1Idn-QbDcD9wDc9ViHQTuOxl5bIKLzSxGP8XVF4D399u5bNa0N99gL-eoVdhHXuCPjLoILwYTHlqNTTlChs6Q6NRdVIBIrOfWg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Numbers and percentages of the SNPs classified into six categories." path-from-xml="dst00502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5345291/dst00502.jpeg" path-from-xml="dst00502.jpeg" target="_blank">View large</a><a section="5345291" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/dst00502.jpeg?Expires=2147483647&Signature=i-hikjqRhc267MmxIkAVJNpokJH2uJyVXZa4AbPz-Za5AV-2sF0T~Ukn19TyigZGket1C3GLscbroEz~Tt5i~oCsI3yUp9Xgdy7Id5U3gM9IAHzt~UdxVZBB3y~ffL2KO81PfDKw5c0rFuv7VPUgdHRorPjeblt5-qdacr15qyYyAEi2ORElKjKZi6CCAwzB8R4yRt0ILtAMRxgkutsuRk18g8o5jMQBff0RjiLnYWeBwq9Cgy8PGmOV2zdlBk1gYb5FSy-G8UWJ6tbUjYrdYiT9~4729dAO14hX2n50cfZwsC818EHH3CFH40E7gYtsYImaKbc~QnxE03Ugb0wj0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5345291&ar=368082&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Numbers and percentages of the SNPs classified into six categories.</p></div></div><div content-id="DST005F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/m_dst00502.jpeg?Expires=2147483647&Signature=lizpbvcLpTfK7EKUa9ZuvC6Ve-4w~scdKQD3f-5ln1F-Z7HwOOZLD2HDF03IC9Vux-rmrlK9psYcizPrq5pI-t0-n0EQomDtWg9Yo1NgNfuPic-L3T2hDRXdBhCzN-BjLR0~ZkjzFpZbhsKmqgOwHtrH-ZXfFB1yDOuJtqp1IRRXzW7z4vqBYXjJJUMmuL0dIgvJWZ2wxGeelYE~pThn9EMBylf5BxNv6bGDVTPDEin-Gs9mI9XQ1Idn-QbDcD9wDc9ViHQTuOxl5bIKLzSxGP8XVF4D399u5bNa0N99gL-eoVdhHXuCPjLoILwYTHlqNTTlChs6Q6NRdVIBIrOfWg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Numbers and percentages of the SNPs classified into six categories." path-from-xml="dst00502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5345291/dst00502.jpeg" path-from-xml="dst00502.jpeg" target="_blank">View large</a><a section="5345291" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/dst00502.jpeg?Expires=2147483647&Signature=i-hikjqRhc267MmxIkAVJNpokJH2uJyVXZa4AbPz-Za5AV-2sF0T~Ukn19TyigZGket1C3GLscbroEz~Tt5i~oCsI3yUp9Xgdy7Id5U3gM9IAHzt~UdxVZBB3y~ffL2KO81PfDKw5c0rFuv7VPUgdHRorPjeblt5-qdacr15qyYyAEi2ORElKjKZi6CCAwzB8R4yRt0ILtAMRxgkutsuRk18g8o5jMQBff0RjiLnYWeBwq9Cgy8PGmOV2zdlBk1gYb5FSy-G8UWJ6tbUjYrdYiT9~4729dAO14hX2n50cfZwsC818EHH3CFH40E7gYtsYImaKbc~QnxE03Ugb0wj0w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5345291&ar=368082&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Numbers and percentages of the SNPs classified into six categories.</p></div></div>                        <a id="5345298" scrollto-destination="5345298"></a>
<div id="DST005F3" data-id="DST005F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/m_dst00503.jpeg?Expires=2147483647&Signature=3hD~WzkFVWUT5WiRtQXGKikTDi0OTYa0D9MaIVvJz5QeZuKae-1gK1zWA-dNE6uNw70s9RhxJVMIZv0y1gDGRKJCxm6I3tYlB8JZxTFoiMg-zDFHqItMLufBuM0vPzgQdvXk3PhicjM~CwWqP-6Ij3ihv4H06soybpD-09g9h~QvKvU~fvZPsbp-9DU6bymgqMwUyEwQoYXtjFHk5l-lDRWQqYzHWqgCZHBzWmW8~19oaorl0D3GWNV7YXUOwq4A~bGA6EJIFixpwQqTMGU8eDkUsiHkM8WV9OP2b4246DKmGGBDhyQQQ5hFzXWu6i0a-wvMmxFTiuRAjWJ0OzEjFA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Locations of the 7054 SNPs on the tomato genome sequence (SL2.40). The locations of the six groups of SNPs—cSNPs, sSNPs, iSNPs, uSNPs, dSNPs, and gSNPs—are indicated by bars coloured red, blue, sky blue, orange, green, and purple, respectively. The bars below the boxes indicate the regions of the euchromatins (magenta) and heterochromatins (grey)." path-from-xml="dst00503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5345298/dst00503.jpeg" path-from-xml="dst00503.jpeg" target="_blank">View large</a><a section="5345298" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/dst00503.jpeg?Expires=2147483647&Signature=oTqB5DBYIBmRObiJpbmHSB6-aiCsAYFJUdqCv~X50SapO8l4-ZGdu9~0J9cSvv7mklX7AxTSRpnIDvI3eNu~t4IkwfRVTcEi4ynTvMSeQYlHNzEIRuej-wxLSPlcwd1yZKmo~2k0DSaP7-zt5GXeJvFKJXBB-1mg80xCwq41uLAMBD7q3SQB7ijMyI2WJ9NwgqKq9eGno~6lu~5RZ3F8oVXXgdti7sriPHVLMYUF30nJs2ara-LhqsdlSwWUPYXrSg8ww7Tf-zW6td24dIKfHCMRr6rtiPfOochPFfo~tHOHZ3nRrqh~dYkkm5pYecc8SY-Ayny7SCZhpl8yjD8wDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5345298&ar=368082&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Locations of the 7054 SNPs on the tomato genome sequence (SL2.40). The locations of the six groups of SNPs—cSNPs, sSNPs, iSNPs, uSNPs, dSNPs, and gSNPs—are indicated by bars coloured red, blue, sky blue, orange, green, and purple, respectively. The bars below the boxes indicate the regions of the euchromatins (magenta) and heterochromatins (grey).</p></div></div><div content-id="DST005F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/m_dst00503.jpeg?Expires=2147483647&Signature=3hD~WzkFVWUT5WiRtQXGKikTDi0OTYa0D9MaIVvJz5QeZuKae-1gK1zWA-dNE6uNw70s9RhxJVMIZv0y1gDGRKJCxm6I3tYlB8JZxTFoiMg-zDFHqItMLufBuM0vPzgQdvXk3PhicjM~CwWqP-6Ij3ihv4H06soybpD-09g9h~QvKvU~fvZPsbp-9DU6bymgqMwUyEwQoYXtjFHk5l-lDRWQqYzHWqgCZHBzWmW8~19oaorl0D3GWNV7YXUOwq4A~bGA6EJIFixpwQqTMGU8eDkUsiHkM8WV9OP2b4246DKmGGBDhyQQQ5hFzXWu6i0a-wvMmxFTiuRAjWJ0OzEjFA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Locations of the 7054 SNPs on the tomato genome sequence (SL2.40). The locations of the six groups of SNPs—cSNPs, sSNPs, iSNPs, uSNPs, dSNPs, and gSNPs—are indicated by bars coloured red, blue, sky blue, orange, green, and purple, respectively. The bars below the boxes indicate the regions of the euchromatins (magenta) and heterochromatins (grey)." path-from-xml="dst00503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5345298/dst00503.jpeg" path-from-xml="dst00503.jpeg" target="_blank">View large</a><a section="5345298" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/dst00503.jpeg?Expires=2147483647&Signature=oTqB5DBYIBmRObiJpbmHSB6-aiCsAYFJUdqCv~X50SapO8l4-ZGdu9~0J9cSvv7mklX7AxTSRpnIDvI3eNu~t4IkwfRVTcEi4ynTvMSeQYlHNzEIRuej-wxLSPlcwd1yZKmo~2k0DSaP7-zt5GXeJvFKJXBB-1mg80xCwq41uLAMBD7q3SQB7ijMyI2WJ9NwgqKq9eGno~6lu~5RZ3F8oVXXgdti7sriPHVLMYUF30nJs2ara-LhqsdlSwWUPYXrSg8ww7Tf-zW6td24dIKfHCMRr6rtiPfOochPFfo~tHOHZ3nRrqh~dYkkm5pYecc8SY-Ayny7SCZhpl8yjD8wDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5345298&ar=368082&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Locations of the 7054 SNPs on the tomato genome sequence (SL2.40). The locations of the six groups of SNPs—cSNPs, sSNPs, iSNPs, uSNPs, dSNPs, and gSNPs—are indicated by bars coloured red, blue, sky blue, orange, green, and purple, respectively. The bars below the boxes indicate the regions of the euchromatins (magenta) and heterochromatins (grey).</p></div></div><p>Among the 2212 cSNP loci, nonsense mutations were found in 15 genes (Table <a reveal-id="DST005TB3" data-open="DST005TB3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Of the 15 nonsense mutations, 3 were found in the alleles of the wild relatives, i.e. Solyc01g106570.2.1 [cystathionine beta-synthase (IPR000644)], Solyc03g083470.2.1 [protein kinase core (IPR000719)], and Solyc01g080340.2.1 [BSD (IPR005607)], and 2 mutations were found on the alleles of the Micro-Tom lines, i.e. Solyc05g025810.2.1 [six-bladed beta-propeller TolB-like (IPR011042)] and Solyc08g076000.2.1 [epidermal growth factor (EGF)-like region conserved site (IPR013032)]. Three mutations on the alleles of Solyc04g014400.2.1 [leucine-rich repeat (IPR001611)], Solyc04g064900.1.1 [glycoside hydrolase family 9 (IPR001701)], and Solyc09g090350.2.1 [long-chain fatty alcohol dehydrogenase (IPR012400)] were distributed in the inbred lines. The other 7 mutations were found across the 40 tomato lines. In ‘Heinz 1706-BG,’ whose genomic sequence has been determined,<sup><a reveal-id="DST005C7" data-open="DST005C7" class="link link-ref link-reveal xref-bibr">7</a></sup> the five nonsense mutations were found on Solyc01g080340.2.1 [BSD (IPR005607)], Solyc01g091770.2.1 [zinc finger RING/FYVE/PHD-type (IPR013083)], Solyc02g050330.1.1 (hypothetical protein), Solyc04g056630.2.1 [cystathionine beta-synthase core (IPR000644)], and Solyc11g071510.1.1 (unnamed protein product). The 15 genes that contained cSNP-caused nonsense mutations are expected to help reveal the divergence and evolution between wild and cultivated tomato lines as well as within cultivated tomato lines. </p>                        <a id="5345310" scrollto-destination="5345310"></a>
<div content-id="DST005TB3" class="table-modal"><div class="table-wrap"><div id="DST005TB3" class="table-wrap-title" data-id="DST005TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Annotations of the genes in which nonsense mutations caused by cSNPs were identified</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>ITAG gene model </th><th>ITAG 2.3 (InterPro) </th><th>Annotation (KDRI) </th></tr></thead><tbody><tr><td>Solyc01g080340.2.1 </td><td>BSD (IPR005607) </td><td>Similar to BSD domain-containing protein </td></tr><tr><td>Solyc01g091770.2.1 </td><td>Zinc finger RING/FYVE/PHD-type (IPR013083) </td><td>Ring finger protein, putative </td></tr><tr><td>Solyc01g106570.2.1 </td><td>Cystathionine beta-synthase (IPR000644) </td><td>Predicted membrane protein contains two CBS domains </td></tr><tr><td>Solyc01g112110.2.1 </td><td>- </td><td>Hypothetical protein </td></tr><tr><td>Solyc02g050330.1.1 </td><td>- </td><td>Hypothetical protein </td></tr><tr><td>Solyc02g092220.1.1 </td><td>- </td><td>Hypothetical protein </td></tr><tr><td>Solyc03g083470.2.1 </td><td>Protein kinase core (IPR000719) </td><td>Serine/threonine protein kinase </td></tr><tr><td>Solyc04g014400.2.1 </td><td>Leucine-rich repeat (IPR001611) </td><td>Leucine-rich repeat </td></tr><tr><td>Solyc04g056630.2.1 </td><td>Cystathionine beta-synthase core (IPR000644) </td><td>Conserved hypothetical protein </td></tr><tr><td>Solyc04g064900.1.1 </td><td>Glycoside hydrolase family 9 (IPR001701) </td><td>Hypothetical protein </td></tr><tr><td>Solyc05g025810.2.1 </td><td>Six-bladed beta-propeller TolB-like (IPR011042) </td><td>Predicted alkaloid synthase/surface mucin hemomucin </td></tr><tr><td>Solyc07g008760.2.1 </td><td>Tetratricopeptide repeat (IPR019734) </td><td>Uncharacterized conserved protein contains tetratricopeptide repeat </td></tr><tr><td>Solyc08g076000.2.1 </td><td>EGF-like region conserved site (IPR013032) </td><td>Apoptotic ATPase </td></tr><tr><td>Solyc09g090350.2.1 </td><td>Long-chain fatty alcohol dehydrogenase (IPR012400) </td><td>Hypothetical protein </td></tr><tr><td>Solyc11g071510.1.1 </td><td>- </td><td>Unnamed protein product </td></tr></tbody></table></div><div content-id="DST005TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>ITAG gene model </th><th>ITAG 2.3 (InterPro) </th><th>Annotation (KDRI) </th></tr></thead><tbody><tr><td>Solyc01g080340.2.1 </td><td>BSD (IPR005607) </td><td>Similar to BSD domain-containing protein </td></tr><tr><td>Solyc01g091770.2.1 </td><td>Zinc finger RING/FYVE/PHD-type (IPR013083) </td><td>Ring finger protein, putative </td></tr><tr><td>Solyc01g106570.2.1 </td><td>Cystathionine beta-synthase (IPR000644) </td><td>Predicted membrane protein contains two CBS domains </td></tr><tr><td>Solyc01g112110.2.1 </td><td>- </td><td>Hypothetical protein </td></tr><tr><td>Solyc02g050330.1.1 </td><td>- </td><td>Hypothetical protein </td></tr><tr><td>Solyc02g092220.1.1 </td><td>- </td><td>Hypothetical protein </td></tr><tr><td>Solyc03g083470.2.1 </td><td>Protein kinase core (IPR000719) </td><td>Serine/threonine protein kinase </td></tr><tr><td>Solyc04g014400.2.1 </td><td>Leucine-rich repeat (IPR001611) </td><td>Leucine-rich repeat </td></tr><tr><td>Solyc04g056630.2.1 </td><td>Cystathionine beta-synthase core (IPR000644) </td><td>Conserved hypothetical protein </td></tr><tr><td>Solyc04g064900.1.1 </td><td>Glycoside hydrolase family 9 (IPR001701) </td><td>Hypothetical protein </td></tr><tr><td>Solyc05g025810.2.1 </td><td>Six-bladed beta-propeller TolB-like (IPR011042) </td><td>Predicted alkaloid synthase/surface mucin hemomucin </td></tr><tr><td>Solyc07g008760.2.1 </td><td>Tetratricopeptide repeat (IPR019734) </td><td>Uncharacterized conserved protein contains tetratricopeptide repeat </td></tr><tr><td>Solyc08g076000.2.1 </td><td>EGF-like region conserved site (IPR013032) </td><td>Apoptotic ATPase </td></tr><tr><td>Solyc09g090350.2.1 </td><td>Long-chain fatty alcohol dehydrogenase (IPR012400) </td><td>Hypothetical protein </td></tr><tr><td>Solyc11g071510.1.1 </td><td>- </td><td>Unnamed protein product </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The ‘Annotation (KDRI)’ indicates the deduced product name based on homology searches against the following databases: NCBI's NR, TAIR10, KOG, and KEGG.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5345310" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST005TB3" class="table-wrap-title" data-id="DST005TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Annotations of the genes in which nonsense mutations caused by cSNPs were identified</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>ITAG gene model </th><th>ITAG 2.3 (InterPro) </th><th>Annotation (KDRI) </th></tr></thead><tbody><tr><td>Solyc01g080340.2.1 </td><td>BSD (IPR005607) </td><td>Similar to BSD domain-containing protein </td></tr><tr><td>Solyc01g091770.2.1 </td><td>Zinc finger RING/FYVE/PHD-type (IPR013083) </td><td>Ring finger protein, putative </td></tr><tr><td>Solyc01g106570.2.1 </td><td>Cystathionine beta-synthase (IPR000644) </td><td>Predicted membrane protein contains two CBS domains </td></tr><tr><td>Solyc01g112110.2.1 </td><td>- </td><td>Hypothetical protein </td></tr><tr><td>Solyc02g050330.1.1 </td><td>- </td><td>Hypothetical protein </td></tr><tr><td>Solyc02g092220.1.1 </td><td>- </td><td>Hypothetical protein </td></tr><tr><td>Solyc03g083470.2.1 </td><td>Protein kinase core (IPR000719) </td><td>Serine/threonine protein kinase </td></tr><tr><td>Solyc04g014400.2.1 </td><td>Leucine-rich repeat (IPR001611) </td><td>Leucine-rich repeat </td></tr><tr><td>Solyc04g056630.2.1 </td><td>Cystathionine beta-synthase core (IPR000644) </td><td>Conserved hypothetical protein </td></tr><tr><td>Solyc04g064900.1.1 </td><td>Glycoside hydrolase family 9 (IPR001701) </td><td>Hypothetical protein </td></tr><tr><td>Solyc05g025810.2.1 </td><td>Six-bladed beta-propeller TolB-like (IPR011042) </td><td>Predicted alkaloid synthase/surface mucin hemomucin </td></tr><tr><td>Solyc07g008760.2.1 </td><td>Tetratricopeptide repeat (IPR019734) </td><td>Uncharacterized conserved protein contains tetratricopeptide repeat </td></tr><tr><td>Solyc08g076000.2.1 </td><td>EGF-like region conserved site (IPR013032) </td><td>Apoptotic ATPase </td></tr><tr><td>Solyc09g090350.2.1 </td><td>Long-chain fatty alcohol dehydrogenase (IPR012400) </td><td>Hypothetical protein </td></tr><tr><td>Solyc11g071510.1.1 </td><td>- </td><td>Unnamed protein product </td></tr></tbody></table></div><div content-id="DST005TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>ITAG gene model </th><th>ITAG 2.3 (InterPro) </th><th>Annotation (KDRI) </th></tr></thead><tbody><tr><td>Solyc01g080340.2.1 </td><td>BSD (IPR005607) </td><td>Similar to BSD domain-containing protein </td></tr><tr><td>Solyc01g091770.2.1 </td><td>Zinc finger RING/FYVE/PHD-type (IPR013083) </td><td>Ring finger protein, putative </td></tr><tr><td>Solyc01g106570.2.1 </td><td>Cystathionine beta-synthase (IPR000644) </td><td>Predicted membrane protein contains two CBS domains </td></tr><tr><td>Solyc01g112110.2.1 </td><td>- </td><td>Hypothetical protein </td></tr><tr><td>Solyc02g050330.1.1 </td><td>- </td><td>Hypothetical protein </td></tr><tr><td>Solyc02g092220.1.1 </td><td>- </td><td>Hypothetical protein </td></tr><tr><td>Solyc03g083470.2.1 </td><td>Protein kinase core (IPR000719) </td><td>Serine/threonine protein kinase </td></tr><tr><td>Solyc04g014400.2.1 </td><td>Leucine-rich repeat (IPR001611) </td><td>Leucine-rich repeat </td></tr><tr><td>Solyc04g056630.2.1 </td><td>Cystathionine beta-synthase core (IPR000644) </td><td>Conserved hypothetical protein </td></tr><tr><td>Solyc04g064900.1.1 </td><td>Glycoside hydrolase family 9 (IPR001701) </td><td>Hypothetical protein </td></tr><tr><td>Solyc05g025810.2.1 </td><td>Six-bladed beta-propeller TolB-like (IPR011042) </td><td>Predicted alkaloid synthase/surface mucin hemomucin </td></tr><tr><td>Solyc07g008760.2.1 </td><td>Tetratricopeptide repeat (IPR019734) </td><td>Uncharacterized conserved protein contains tetratricopeptide repeat </td></tr><tr><td>Solyc08g076000.2.1 </td><td>EGF-like region conserved site (IPR013032) </td><td>Apoptotic ATPase </td></tr><tr><td>Solyc09g090350.2.1 </td><td>Long-chain fatty alcohol dehydrogenase (IPR012400) </td><td>Hypothetical protein </td></tr><tr><td>Solyc11g071510.1.1 </td><td>- </td><td>Unnamed protein product </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The ‘Annotation (KDRI)’ indicates the deduced product name based on homology searches against the following databases: NCBI's NR, TAIR10, KOG, and KEGG.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5345310" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=5345313 id="5345313" class="section-title" data-legacy-id=s3d>3.4. Substitution effects on the genes induced by cSNP, uSNP, and dSNP</h3>
<p>To estimate the substitution effects on protein and gene functions, the KOG functional categories<sup><a reveal-id="DST005C21" data-open="DST005C21" class="link link-ref link-reveal xref-bibr">21</a></sup> were assigned to the corresponding genes classified into the categories cSNP, sSNP, iSNP, uSNP, and dSNP. Fig. <a reveal-id="DST005F4" data-open="DST005F4" class="link link-reveal link-table xref-fig">4</a> shows the percentage of each of these types of SNPs on the 25 KOG categories. In addition, the standard deviations (s) of the percentages of the 25 KOG categories were calculated in each SNP type (data not shown). Hereafter, KOG categories that showed more or &lt;1.5 s (s: sigma distribution) in the percentages were considered as significantly different KOGs in the distribution of each type of SNP. In the case of cSNPs, large ratios were observed for KOG L (replication, recombination, and repair, 41.0%) and KOG Q (secondary metabolites biosynthesis, transport, and catabolism, 40.7%), whereas a smaller ratio was identified for KOG F (nucleotide transport and metabolism, 18.8%). In the case of uSNP, significantly higher distributions were observed for KOG W (extracellular structures, 12.0%) and KOG H (coenzyme transport and metabolism, 12.5%), whereas lower distributions were identified for KOG J (translation, ribosomal structure, and biogenesis, 3.4%), KOG L (replication, recombination, and repair, 3.6%) and KOG Q (secondary metabolites biosynthesis, transport, and catabolism, 2.5%). In the case of dSNP, high and low ratios were observed for KOG Z (cytoskeleton, 35.7%), and KOG Y (nuclear structure, 11.1%), respectively. If promoters or terminators were located in the 1 Kb regions upstream or downstream from the start or stop codons of the genes, uSNP and dSNP might affect their gene expressions. For a more substantial estimation of the substitution effects on uSNP and dSNP, it is necessary to detect the promoter and terminator regions on each corresponding gene. </p>                        <a id="5345323" scrollto-destination="5345323"></a>
<div id="DST005F4" data-id="DST005F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/m_dst00504.jpeg?Expires=2147483647&Signature=yFb1AlWe5DvuBq2qSrm~m1KodXtDUFhm4Wu7nLcwPXOCnf7HKcaK1t1vs6UZ-0xdo29j~IzqDS2YzA7iaRnrC71fvuvZ9D5pJgs7mzU6ZViB8b~nhbX1rp0s4jggOsdw0XcYO-PnbN~5ShiUKiGPAnQTTAlU5PQdXOKFA4nxXNz1ElZ74CEroyFR76SzKy~Tvu9rDnGUGtWiTAQbVJQ3jp1UY6dsj867ynGKP-gFGZs0q6jGIm2Z3pova5Vl7kviCD1T7Ni4dQyWW-HBEglYx7w0yxRVXvzzaKSZSf~~Ng~3D~gKOW3I2xvOOvAyTeEW8ReSbpvUU4LZaaWzUU-nRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Percentages of the 5 types of SNPs on each of the 25 KOG categories. The percentages of the observed cSNPs, sSNPs, uSNPs, dSNPs, and iSNPs are indicated by white, grey, dots, thick dots, and slated bars, respectively." path-from-xml="dst00504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5345323/dst00504.jpeg" path-from-xml="dst00504.jpeg" target="_blank">View large</a><a section="5345323" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/dst00504.jpeg?Expires=2147483647&Signature=Q2GK3JL~xF2P-PhLkJeqnpb~dSfLoplBR4cX3VYVwIElmvxehNTmk6b9BBOSmxyGbiVTvvE7SwJGNVMPC3O1Dg65UuhdAsJ~8GtfuEpgLEocoECBp52ZTI6XCrzBs2IwbR9gXcp6P2HND1JnD-YTi7YiHXavbHxqjiF2IONoTSXJCMcV3cX5xHoR~0juUTxQCzSKhsvu8oeKETCsTBFLdpwV8-POIQXnbWq~1vuR8TW~y1JVSKGkWVkyA1ytgQ160oecRgte8MuOtEkZiEzAYZY7~TIDD3DyDJvF3RaFGq3C4lKV9vaok6ffBPYGXkpO2x1qQ7ROnWvpC7Q9c3b6ag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5345323&ar=368082&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Percentages of the 5 types of SNPs on each of the 25 KOG categories. The percentages of the observed cSNPs, sSNPs, uSNPs, dSNPs, and iSNPs are indicated by white, grey, dots, thick dots, and slated bars, respectively.</p></div></div><div content-id="DST005F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/m_dst00504.jpeg?Expires=2147483647&Signature=yFb1AlWe5DvuBq2qSrm~m1KodXtDUFhm4Wu7nLcwPXOCnf7HKcaK1t1vs6UZ-0xdo29j~IzqDS2YzA7iaRnrC71fvuvZ9D5pJgs7mzU6ZViB8b~nhbX1rp0s4jggOsdw0XcYO-PnbN~5ShiUKiGPAnQTTAlU5PQdXOKFA4nxXNz1ElZ74CEroyFR76SzKy~Tvu9rDnGUGtWiTAQbVJQ3jp1UY6dsj867ynGKP-gFGZs0q6jGIm2Z3pova5Vl7kviCD1T7Ni4dQyWW-HBEglYx7w0yxRVXvzzaKSZSf~~Ng~3D~gKOW3I2xvOOvAyTeEW8ReSbpvUU4LZaaWzUU-nRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Percentages of the 5 types of SNPs on each of the 25 KOG categories. The percentages of the observed cSNPs, sSNPs, uSNPs, dSNPs, and iSNPs are indicated by white, grey, dots, thick dots, and slated bars, respectively." path-from-xml="dst00504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5345323/dst00504.jpeg" path-from-xml="dst00504.jpeg" target="_blank">View large</a><a section="5345323" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/dst00504.jpeg?Expires=2147483647&Signature=Q2GK3JL~xF2P-PhLkJeqnpb~dSfLoplBR4cX3VYVwIElmvxehNTmk6b9BBOSmxyGbiVTvvE7SwJGNVMPC3O1Dg65UuhdAsJ~8GtfuEpgLEocoECBp52ZTI6XCrzBs2IwbR9gXcp6P2HND1JnD-YTi7YiHXavbHxqjiF2IONoTSXJCMcV3cX5xHoR~0juUTxQCzSKhsvu8oeKETCsTBFLdpwV8-POIQXnbWq~1vuR8TW~y1JVSKGkWVkyA1ytgQ160oecRgte8MuOtEkZiEzAYZY7~TIDD3DyDJvF3RaFGq3C4lKV9vaok6ffBPYGXkpO2x1qQ7ROnWvpC7Q9c3b6ag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5345323&ar=368082&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Percentages of the 5 types of SNPs on each of the 25 KOG categories. The percentages of the observed cSNPs, sSNPs, uSNPs, dSNPs, and iSNPs are indicated by white, grey, dots, thick dots, and slated bars, respectively.</p></div></div><p>Most of the substitutions involving changes in the charges or polarity of the amino acid residues were related to alterations in the physicochemical properties of proteins. Therefore, these alterations in the physicochemical properties of the amino acid residues by substitutions were investigated for 2212 cSNP loci. The amino acid properties were not altered by the substitutions at 926 (41.9%) of the 2212 cSNP loci (grey bars in Fig. <a reveal-id="DST005F5" data-open="DST005F5" class="link link-reveal link-table xref-fig">5</a>), whereas they were altered to non-polar (slanted bars) or polar (dotted bars) in 861 (38.9%) and 346 (15.6%) of the 2212 SNP loci, respectively. Amino acid residues whose properties were altered on a change in polarity were found in 64 (2.9%) SNP loci (white bars). In total, 1271 (57.5%) cSNP loci were predicted to alter the properties of amino acid residues. </p>                        <a id="5345334" scrollto-destination="5345334"></a>
<div id="DST005F5" data-id="DST005F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/m_dst00505.jpeg?Expires=2147483647&Signature=w7jhdjs2DCGaTBbwPRQ7qCMmPEyQFt8dus7~Rh~o~Z3~9coQMX3OLENTPuB9X4XIsbuHDSj~SKATS3Wfysr8mkUPcUeZRSdVXcvaQbFpEfG-XSgTL9Dz82d~-TtX6br-Eo95SXt27JodGk7F9XH0WYDwfeYlL-SzwPtr-9BUvJdSLAdc2T~DKBmcx0Hg7MPocPXDfUKtTtfLlvUZg36tqekEXF4OxjQhMPeaiU~H1Etj7ogMidA7PsPBmy-3JQ1OBokg3S7gGd8EXKZiSykt68W5lem-F8OGSCke-lb-YRwPUvOdGKVcgI1CpzkXMsyowW2t8-NUYylGlBpobChx-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Substitution patterns of the properties of amino acid residues caused by cSNPs. The substitutions that did not alter the properties of amino acid residues are indicated by grey bars. The substitutions related to changes in polarity, e.g. to polar and to non-polar, are shown by dotted and slated bars, respectively. The substitutions related to changes in charges or amino acid residues, i.e. negatively charged (acidic) or positively charged (basic), are shown by white bars. The substitutions related to nonsense mutations are shown by black bars." path-from-xml="dst00505.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5345334/dst00505.jpeg" path-from-xml="dst00505.jpeg" target="_blank">View large</a><a section="5345334" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/dst00505.jpeg?Expires=2147483647&Signature=zl1050N8PeCURLGXLuhQ7uYGH3Hg88Jun61hAcNWi4XVEAPcmvD9ica9r5FVhof6ynmidvMSL1EfoGeN9aUqewDA8Fc6c2RtR0YfUyxA3K3fqEa0seFsZZczsefIYA~TET~sO8g69x5XuLHPCkf09XmRMGIujXdE3pROiPdX5hcsVeEvIqRFm60S6LQ61e~SHbdgftNv7YSvHf9QqYX9JUzVCjg2O6R90SibjZpZUBkHP8LML~T3P6feAu4CDm7cpdmyacViZEhLFKC~Lz0jLmY5tj9ZnA1nuIREqlEWtsGfcdUYZBNS6EYRnjuyVDIjVHANz~b-NW-dBILpLev5-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5345334&ar=368082&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00505.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Substitution patterns of the properties of amino acid residues caused by cSNPs. The substitutions that did not alter the properties of amino acid residues are indicated by grey bars. The substitutions related to changes in polarity, e.g. to polar and to non-polar, are shown by dotted and slated bars, respectively. The substitutions related to changes in charges or amino acid residues, i.e. negatively charged (acidic) or positively charged (basic), are shown by white bars. The substitutions related to nonsense mutations are shown by black bars.</p></div></div><div content-id="DST005F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/m_dst00505.jpeg?Expires=2147483647&Signature=w7jhdjs2DCGaTBbwPRQ7qCMmPEyQFt8dus7~Rh~o~Z3~9coQMX3OLENTPuB9X4XIsbuHDSj~SKATS3Wfysr8mkUPcUeZRSdVXcvaQbFpEfG-XSgTL9Dz82d~-TtX6br-Eo95SXt27JodGk7F9XH0WYDwfeYlL-SzwPtr-9BUvJdSLAdc2T~DKBmcx0Hg7MPocPXDfUKtTtfLlvUZg36tqekEXF4OxjQhMPeaiU~H1Etj7ogMidA7PsPBmy-3JQ1OBokg3S7gGd8EXKZiSykt68W5lem-F8OGSCke-lb-YRwPUvOdGKVcgI1CpzkXMsyowW2t8-NUYylGlBpobChx-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Substitution patterns of the properties of amino acid residues caused by cSNPs. The substitutions that did not alter the properties of amino acid residues are indicated by grey bars. The substitutions related to changes in polarity, e.g. to polar and to non-polar, are shown by dotted and slated bars, respectively. The substitutions related to changes in charges or amino acid residues, i.e. negatively charged (acidic) or positively charged (basic), are shown by white bars. The substitutions related to nonsense mutations are shown by black bars." path-from-xml="dst00505.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5345334/dst00505.jpeg" path-from-xml="dst00505.jpeg" target="_blank">View large</a><a section="5345334" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/dst00505.jpeg?Expires=2147483647&Signature=zl1050N8PeCURLGXLuhQ7uYGH3Hg88Jun61hAcNWi4XVEAPcmvD9ica9r5FVhof6ynmidvMSL1EfoGeN9aUqewDA8Fc6c2RtR0YfUyxA3K3fqEa0seFsZZczsefIYA~TET~sO8g69x5XuLHPCkf09XmRMGIujXdE3pROiPdX5hcsVeEvIqRFm60S6LQ61e~SHbdgftNv7YSvHf9QqYX9JUzVCjg2O6R90SibjZpZUBkHP8LML~T3P6feAu4CDm7cpdmyacViZEhLFKC~Lz0jLmY5tj9ZnA1nuIREqlEWtsGfcdUYZBNS6EYRnjuyVDIjVHANz~b-NW-dBILpLev5-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5345334&ar=368082&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00505.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Substitution patterns of the properties of amino acid residues caused by cSNPs. The substitutions that did not alter the properties of amino acid residues are indicated by grey bars. The substitutions related to changes in polarity, e.g. to polar and to non-polar, are shown by dotted and slated bars, respectively. The substitutions related to changes in charges or amino acid residues, i.e. negatively charged (acidic) or positively charged (basic), are shown by white bars. The substitutions related to nonsense mutations are shown by black bars.</p></div></div>                            <h3 scrollto-destination=5345341 id="5345341" class="section-title" data-legacy-id=s3e>3.5. Inference of direct substitution effects on protein functions</h3>
<p>The amino acid residues directly altering protein functions were inferred according to the locations of substitutions on the functional sites of protein structures. A total of 1271 cSNP loci that altered the charges or polarities on amino acid residues were identified in 1108 genes. Of these 1108 genes, the three-dimensional structures of 843 corresponding proteins were constructed by homology modelling using Modeller v9.10 software.<sup><a reveal-id="DST005C18" data-open="DST005C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Furthermore, the candidate functional sites were predicted by FPocket software.<sup><a reveal-id="DST005C19" data-open="DST005C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p><p>These analyses together revealed 215 cSNP loci that were candidate functional sites on 200 proteins (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The genes whose cSNPs were located on candidate functional sites have the potential to be applied as functional markers because they could directly affect the protein functions. Therefore, differences in the cSNP distribution on the 25 KOG categories were investigated according to the cSNP positions on functional or other sites (Fig. <a reveal-id="DST005F6" data-open="DST005F6" class="link link-reveal link-table xref-fig">6</a>). Significantly different distributions were observed on KOG O (posttranslational modification, protein turnover, and chaperones), KOG C (energy production and conversion), KOG G (carbohydrate transport and metabolism), KOG E (amino acid transport and metabolism), and KOG Q (secondary metabolites biosynthesis, transport, and catabolism). </p>                        <a id="5345355" scrollto-destination="5345355"></a>
<div id="DST005F6" data-id="DST005F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/m_dst00506.jpeg?Expires=2147483647&Signature=VIxlK0P5yQHjDc9c2CIbw7nu2VgbBosp~fVCey4RKx8LacfNK9EMY3jUgWIxqTSbdtbdB5cxOL9vO-4xWqv5TU60T4X5nO7t0ff4qKsTtBGQ8ObDTWzSoPQfxHxV2O4PA4-iyCSSTFcupPTNojCqRJeD8JeIbSF9sSyuIm2NJ1Cws96pjHA8qudq1mFpWv538cRPYjB1PS5o2pFOSBU8wH04t0xzd3V8Z2mct461jdpNxi6JjhA~a3M73Ivj9GoSM5Har-5YZSdun0Mi32PI1QlI8Lfb-XDlKX9Fx6WPyaKLV2CWwIDxMuzynn8XkxElh4E6PFTTdTqSLyf-wv16OA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of the annotated KOGs in the genes with identified cSNPs inside or outside of functional sites. The numbers of annotated genes with cSNP located inside or outside of the functional sites are shown by white and grey bars, respectively." path-from-xml="dst00506.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5345355/dst00506.jpeg" path-from-xml="dst00506.jpeg" target="_blank">View large</a><a section="5345355" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/dst00506.jpeg?Expires=2147483647&Signature=K4RzsuOsjYFp0zE47O9T-bILjkv93K5xWH4nyvXqLwbbLe~liqVZhQ2uwRkM~BF82cMHtZkFcChNLuN~WpLai-H~avi-~Cf5eZDVimQ9QqPSSRSdhLPMYnjZl1lw5TzMdiTObgN4v3U929eKjwQO5DVkaNoVFQT3C5EXzQSm0-K8xO33oaowbOwkpnK0m7Fhqgbf3X4oFykL0sPqUFylEc43eEQnCT~FXQSz9RH4V9t1E0J-15RRqqWtMGaTLTvOvV2GHe15qvwRuNGgpsKkeKs1f5LaeNI4YLR4gKjXLopFh1MXnDIFAG7REFbmUgcyzXnTcQj3X9uunM6K94SzGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5345355&ar=368082&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00506.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of the annotated KOGs in the genes with identified cSNPs inside or outside of functional sites. The numbers of annotated genes with cSNP located inside or outside of the functional sites are shown by white and grey bars, respectively.</p></div></div><div content-id="DST005F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/m_dst00506.jpeg?Expires=2147483647&Signature=VIxlK0P5yQHjDc9c2CIbw7nu2VgbBosp~fVCey4RKx8LacfNK9EMY3jUgWIxqTSbdtbdB5cxOL9vO-4xWqv5TU60T4X5nO7t0ff4qKsTtBGQ8ObDTWzSoPQfxHxV2O4PA4-iyCSSTFcupPTNojCqRJeD8JeIbSF9sSyuIm2NJ1Cws96pjHA8qudq1mFpWv538cRPYjB1PS5o2pFOSBU8wH04t0xzd3V8Z2mct461jdpNxi6JjhA~a3M73Ivj9GoSM5Har-5YZSdun0Mi32PI1QlI8Lfb-XDlKX9Fx6WPyaKLV2CWwIDxMuzynn8XkxElh4E6PFTTdTqSLyf-wv16OA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of the annotated KOGs in the genes with identified cSNPs inside or outside of functional sites. The numbers of annotated genes with cSNP located inside or outside of the functional sites are shown by white and grey bars, respectively." path-from-xml="dst00506.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5345355/dst00506.jpeg" path-from-xml="dst00506.jpeg" target="_blank">View large</a><a section="5345355" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/dst00506.jpeg?Expires=2147483647&Signature=K4RzsuOsjYFp0zE47O9T-bILjkv93K5xWH4nyvXqLwbbLe~liqVZhQ2uwRkM~BF82cMHtZkFcChNLuN~WpLai-H~avi-~Cf5eZDVimQ9QqPSSRSdhLPMYnjZl1lw5TzMdiTObgN4v3U929eKjwQO5DVkaNoVFQT3C5EXzQSm0-K8xO33oaowbOwkpnK0m7Fhqgbf3X4oFykL0sPqUFylEc43eEQnCT~FXQSz9RH4V9t1E0J-15RRqqWtMGaTLTvOvV2GHe15qvwRuNGgpsKkeKs1f5LaeNI4YLR4gKjXLopFh1MXnDIFAG7REFbmUgcyzXnTcQj3X9uunM6K94SzGA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5345355&ar=368082&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00506.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of the annotated KOGs in the genes with identified cSNPs inside or outside of functional sites. The numbers of annotated genes with cSNP located inside or outside of the functional sites are shown by white and grey bars, respectively.</p></div></div><p>Solyc09g010080.2.1, also known as <em>brix9-2-5</em>,<sup><a reveal-id="DST005C29" data-open="DST005C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DST005C30" data-open="DST005C30" class="link link-ref link-reveal xref-bibr">30</a></sup> controls fruit sugar content and is involved in KOG G (carbohydrate transport and metabolism). It is also annotated as beta-fructofuranosidase. The three-dimensional structure of Solyc09g010080 was built by homology modelling based on the crystal structure of a cell-wall invertase from <em>Arabidopsis thaliana</em> (PDB id: 2ac1) (Fig. <a reveal-id="DST005F7" data-open="DST005F7" class="link link-reveal link-table xref-fig">7</a>). The identity of the amino acid sequences between Solyc09g010080 and 2ac1 was 51.0% in 92.6% length coverage. In the constructed structure, 28 pockets were found as candidate functional sites by the FPocket program.<sup><a reveal-id="DST005C19" data-open="DST005C19" class="link link-ref link-reveal xref-bibr">19</a></sup> This program usually detects several candidates for functional sites, which are called pockets, in a protein structure because there are many small or large clefts on the surface of proteins. In a previous study,<sup><a reveal-id="DST005C29" data-open="DST005C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DST005C30" data-open="DST005C30" class="link link-ref link-reveal xref-bibr">30</a></sup> three amino acid substitutions (Gln297, Asn315, and Leu322) caused by cSNPs were found in <em>brix9-2-5</em> between <em>S. lycopersicum</em> and <em>S. pennellii.</em> Asn315, which is one of the three amino acid residues, was identified on pocket 8 that was a relatively larger pocket that included 15 amino acid residues (Asn315, Leu316, Ser317, Lys319, Ser430, Tyr431, Lys432, Ile433, Val459, Asp460, Val461, Asp462, Leu463, Asp465, and Lys488). The amino acid residue at the 315th position was altered from Asn to Asp, and the amino acid property was changed from ‘polar’ to ‘negatively charged (acidic).’ The amino acid residue at the 315th position was Asp in the wild tomatoes and <em>S. lycopersicum</em> var cerasiforme (WIR25/30 and TOMJPF00009), whereas it was Asn in the other tomatoes. Aspartic acid residues play important roles in the catalytic sites of proteins, and, therefore, the activities of Solyc09g010080 might differ between the wild and cultivated tomatoes. </p>                        <a id="5345366" scrollto-destination="5345366"></a>
<div id="DST005F7" data-id="DST005F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/m_dst00507.jpeg?Expires=2147483647&Signature=Ph~m3Vbl0Q5qXCCoWXz0TIbBEcwx1lzjirPm-zAgo47xy27D9-0B60IJwRzP0NAJ9ErSeu67PfSTSPHIX9Jf6HaSVtmamcy1JDWx0GuJG9aQa63Fnf71W3xLVtxqabXljcABbp2uu0UEL4pxqEmX4sEIzQX6Zvpes0owjZFrKAVP8ba1ZSw5lGkNDrdElSWXRSeacAzmKTwIqj6KzMUQY5tXUYjwOMlZ94q7eEkg6WtWQoAM358BcT17OxPl4A1HoEVHhj5a-KhMp6uHY1jcp9WY8fucIpcRdHVTboBvOD5NGE0sOk9e4GQRcgD33U3zJmynF~CSmWXqlIwsFjklrw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Locations of the amino acid residues substituted by a cSNP on Solyc09g010080. The amino acid residue (number 315) substituted by a cSNP was located on pocket 8. The α-helices, β-sheets, and turns are shown in orange, blue, and yellow, respectively. Pocket 8 is displayed with a mesh model. In the mesh model, the carbon, oxygen, and nitrogen atoms are coloured grey, red, and blue, respectively." path-from-xml="dst00507.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5345366/dst00507.jpeg" path-from-xml="dst00507.jpeg" target="_blank">View large</a><a section="5345366" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/dst00507.jpeg?Expires=2147483647&Signature=kjF4B4L2x9N4QJvIAFqViLz11dzlnh4aj-pNBNkPtV6Y-5EzY01odw7EcB6nqbwOCR0FVngCDqcADjkALMWb~aZZ-3~8pExyfeX1GFaixJ19MfGmzkTUFwNZdl9OvhesPz-k8qns5jNtjOzD96QsUoblGQSCZzkF9VK46CJsHCEyyp3l3n9tQX99X3nO-uFY5JXeuCRUlx1X31O9jhF0OG4X8KzlaktR2lEMZYxjK2MOwfK9by1I4mlmBXdEJeCZooJPadfKhHHL-8laxCQfVtGsmWJNqzM3mnULRtcJlNUYbTy2qjnj3QqR~XI5QTb3SGA2GON8Hn1k3dOc0A5ssg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5345366&ar=368082&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00507.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Locations of the amino acid residues substituted by a cSNP on Solyc09g010080. The amino acid residue (number 315) substituted by a cSNP was located on pocket 8. The α-helices, β-sheets, and turns are shown in orange, blue, and yellow, respectively. Pocket 8 is displayed with a mesh model. In the mesh model, the carbon, oxygen, and nitrogen atoms are coloured grey, red, and blue, respectively.</p></div></div><div content-id="DST005F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/m_dst00507.jpeg?Expires=2147483647&Signature=Ph~m3Vbl0Q5qXCCoWXz0TIbBEcwx1lzjirPm-zAgo47xy27D9-0B60IJwRzP0NAJ9ErSeu67PfSTSPHIX9Jf6HaSVtmamcy1JDWx0GuJG9aQa63Fnf71W3xLVtxqabXljcABbp2uu0UEL4pxqEmX4sEIzQX6Zvpes0owjZFrKAVP8ba1ZSw5lGkNDrdElSWXRSeacAzmKTwIqj6KzMUQY5tXUYjwOMlZ94q7eEkg6WtWQoAM358BcT17OxPl4A1HoEVHhj5a-KhMp6uHY1jcp9WY8fucIpcRdHVTboBvOD5NGE0sOk9e4GQRcgD33U3zJmynF~CSmWXqlIwsFjklrw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Locations of the amino acid residues substituted by a cSNP on Solyc09g010080. The amino acid residue (number 315) substituted by a cSNP was located on pocket 8. The α-helices, β-sheets, and turns are shown in orange, blue, and yellow, respectively. Pocket 8 is displayed with a mesh model. In the mesh model, the carbon, oxygen, and nitrogen atoms are coloured grey, red, and blue, respectively." path-from-xml="dst00507.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5345366/dst00507.jpeg" path-from-xml="dst00507.jpeg" target="_blank">View large</a><a section="5345366" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/dst00507.jpeg?Expires=2147483647&Signature=kjF4B4L2x9N4QJvIAFqViLz11dzlnh4aj-pNBNkPtV6Y-5EzY01odw7EcB6nqbwOCR0FVngCDqcADjkALMWb~aZZ-3~8pExyfeX1GFaixJ19MfGmzkTUFwNZdl9OvhesPz-k8qns5jNtjOzD96QsUoblGQSCZzkF9VK46CJsHCEyyp3l3n9tQX99X3nO-uFY5JXeuCRUlx1X31O9jhF0OG4X8KzlaktR2lEMZYxjK2MOwfK9by1I4mlmBXdEJeCZooJPadfKhHHL-8laxCQfVtGsmWJNqzM3mnULRtcJlNUYbTy2qjnj3QqR~XI5QTb3SGA2GON8Hn1k3dOc0A5ssg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5345366&ar=368082&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst00507.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Locations of the amino acid residues substituted by a cSNP on Solyc09g010080. The amino acid residue (number 315) substituted by a cSNP was located on pocket 8. The α-helices, β-sheets, and turns are shown in orange, blue, and yellow, respectively. Pocket 8 is displayed with a mesh model. In the mesh model, the carbon, oxygen, and nitrogen atoms are coloured grey, red, and blue, respectively.</p></div></div><p>In this study, we predicted the functions of 7054 SNPs identified by array assays. Although this was the total number of SNPs we investigated, we expect that greater numbers of SNPs exist on the tomato genome. The advance of NGS technologies has enabled the discovery of genome-wide SNPs by the re-sequencing of multiple tomato lines. The approach demonstrated in this study is expected to be applied to larger-scale functional analysis of SNPs identified by re-sequencing.</p>                            <h2 scrollto-destination=5345377 id="5345377" class="section-title" data-legacy-id=s4>4. Conclusion</h2>
<p>A total of 7054 SNP loci were genotyped in 40 tomato lines, including <em>S. lycopersicum</em>, <em>S. lycopersicum</em> var. cerasiforme, <em>S. pimpinellifolium</em>, <em>S. pennellii</em>, <em>S. peruvianum</em>, and <em>S. chilense</em>. Based on the SNP genotypes, these 40 lines were classified into 4 clusters, and the LDs were detected on the 40 lines and each cluster. The SNP effects on gene expression were inferred by the sequences within 1 Kb upstream and downstream from the start codons because there was no information about the promoter and terminator regions in the annotations on ITAG. To investigate the effects of SNP more precisely, promoter and terminator regions should be identified by computational and experimental procedures. On the other hand, the SNP effects on protein functions were inferred by modelled structures because the molecular modelling method has been found useful for estimating the effects of the substitution of amino acid residues on protein function.<sup><a reveal-id="DST005C6" data-open="DST005C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DST005C31" data-open="DST005C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Therefore, homology modelling was applied to the tomato genes with cSNPs. As a result, it was predicted that the protein functions (or catalytic activities) of the 200 genes might be altered by the substitutions because the SNPs were located on the candidate functional sites. The uSNPs, dSNPs, and cSNPs have the capacity to affect gene expressions and protein functions, so they might be related to the differences in phenotypes among tomato lines. These SNPs are expected to be applied to marker-assisted selection because they could be considered functional markers. Recently, the large-scale genotyping of SNPs has been achieved using the NGS or array platform. The identification of the SNPs that directly affect protein and gene functions is quite important for molecular breeding and enzymology.</p>                            <h3 scrollto-destination=5345386 id="5345386" class="section-title" data-legacy-id=s4a>4.1. Availability</h3>
<p>The genotyping data, annotations, and modelled protein structures with cSNPs are available at the Kazusa Tomato Genomics DataBase (<a class="link link-uri" href="http://plant1.kazusa.or.jp/tomato/">http://plant1.kazusa.or.jp/tomato/</a>).</p>                            <h2 scrollto-destination=5345396 id="5345396" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at</a><a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5345402 id="5345402" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the Kazusa DNA Research Institute Foundation and the Ministry of Agriculture, Forestry, and Fisheries of Japan (Genomics for Agricultural Innovation Foundation, DD-4010 and SGE-1001), and MEXT KAKENHI Grant number 24510286, Grant-in-Aid for Scientific Research (C).</p>                            <h2 scrollto-destination=5345409 id="5345409" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are grateful to Dr Ariizumi (National BioResource Project, Japan), Dr Saito (NARO Institute of Vegetable and Tea Sciences, Japan), Dr Tam (Tomato Genetic Resource Center, USA), and Dr Tanksley (Cornell University, USA) for providing plant materials. The SNP BeadChip was developed by Illumina's technology and the USDA-NIFA Agriculture and Food Research Initiative (AFRI) sponsored <em>Solanaceae</em> Coordinated Agricultural Project.</p>                            <h2 scrollto-destination=5345418 id="5345418" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST005C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Nayak</div> <div class="given-names">S.N.</div></div>,  <div class="name"><div class="surname">May</div> <div class="given-names">G.D.</div></div>,  <div class="name"><div class="surname">Jackson</div> <div class="given-names">S.A.</div></div>. </div><div class="article-title">Next-generation sequencing technologies and their implications for crop genetics and breeding</div>, <div class="source">Trends Biotechnol.</div> , <div class="year">2009</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">522</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next-generation%20sequencing%20technologies%20and%20their%20implications%20for%20crop%20genetics%20and%20breeding&amp;author=R.K.%20Varshney&amp;author=S.N.%20Nayak&amp;author=G.D.%20May&amp;author=S.A.%20Jackson&amp;publication_year=2009&amp;journal=Trends%20Biotechnol.&amp;volume=27&amp;pages=522-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2009.05.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2009.05.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2009.05.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19679362" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST005C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cao</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Schneeberger</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ossowski</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Whole-genome sequencing of multiple <em>Arabidopsis thaliana</em> populations</div>, <div class="source">Nat. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">956</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20sequencing%20of%20multiple%20Arabidopsis%20thaliana%20populations&amp;author=J.%20Cao&amp;author=K.%20Schneeberger&amp;author=S.%20Ossowski&amp;publication_year=2011&amp;journal=Nat.%20Genet.&amp;volume=43&amp;pages=956-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.911" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.911" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.911"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21874002" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST005C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chagné</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Crowhurst</div> <div class="given-names">R.N.</div></div>,  <div class="name"><div class="surname">Troggio</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Genome-wide SNP detection, validation, and development of an 8K SNP array for apple</div>, <div class="source">PLoS One</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e31745</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20SNP%20detection%2C%20validation%2C%20and%20development%20of%20an%208K%20SNP%20array%20for%20apple&amp;author=D.%20Chagn%C3%A9&amp;author=R.N.%20Crowhurst&amp;author=M.%20Troggio&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e31745" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0031745" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0031745" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0031745"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22363718" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST005C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">P.H.</div></div>,  <div class="name"><div class="surname">Shatkay</div> <div class="given-names">H.</div></div>. </div><div class="article-title">F-SNP: computationally predicted functional SNPs for disease association studies</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D820</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=F-SNP%3A%20computationally%20predicted%20functional%20SNPs%20for%20disease%20association%20studies&amp;author=P.H.%20Lee&amp;author=H.%20Shatkay&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D820-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm904" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm904" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm904"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17986460" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST005C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yue</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Melamud</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Moult</div> <div class="given-names">J.</div></div>. </div><div class="article-title">SNPs3D: candidate gene and SNP selection for association studies</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">166</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNPs3D%3A%20candidate%20gene%20and%20SNP%20selection%20for%20association%20studies&amp;author=P.%20Yue&amp;author=E.%20Melamud&amp;author=J.%20Moult&amp;publication_year=2006&amp;journal=BMC%20Bioinformatics&amp;volume=7&amp;pages=166" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-7-166" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-7-166" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-7-166"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16551372" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST005C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kharabian</div> <div class="given-names">A.</div></div>. </div><div class="article-title">An efficient computational method for screening functional SNPs in plants</div>, <div class="source">J. Theor. Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">265</div> (pg. <div class="fpage">55</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20efficient%20computational%20method%20for%20screening%20functional%20SNPs%20in%20plants&amp;author=A.%20Kharabian&amp;publication_year=2010&amp;journal=J.%20Theor.%20Biol.&amp;volume=265&amp;pages=55-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jtbi.2010.04.017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jtbi.2010.04.017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jtbi.2010.04.017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20406646" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST005C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="collab">Tomato Genome Consortium</div><div class="article-title">The tomato genome sequence provides insights into fleshy fruit evolution</div>, <div class="source">Nature</div> , <div class="year">2012</div>, vol. <div class="volume">485</div> (pg. <div class="fpage">635</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22660326" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST005C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hamilton</div> <div class="given-names">J.P.</div></div>,  <div class="name"><div class="surname">Sim</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Stoffel</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Single nucleotide polymorphism discovery in cultivated tomato via sequencing by synthesis</div>, <div class="source">The Plant Genome</div> , <div class="year">2012</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">17</div>-<div class="lpage">29</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single%20nucleotide%20polymorphism%20discovery%20in%20cultivated%20tomato%20via%20sequencing%20by%20synthesis&amp;author=J.P.%20Hamilton&amp;author=S.C.%20Sim&amp;author=K.%20Stoffel&amp;publication_year=2012&amp;journal=The%20Plant%20Genome&amp;volume=5&amp;pages=17-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3835/plantgenome2011.12.0033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3835%2Fplantgenome2011.12.0033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3835%2Fplantgenome2011.12.0033"> </span></div> </div></div></div></div></div><div content-id="DST005C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sim</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Van Deynze</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Stoffel</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">High-density SNP genotyping of tomato (<em>Solanum lycopersicum</em> L.) reveals patterns of genetic variation due to breeding</div>, <div class="source">PLoS One</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e45520</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-density%20SNP%20genotyping%20of%20tomato%20(Solanum%20lycopersicum%20L.)%20reveals%20patterns%20of%20genetic%20variation%20due%20to%20breeding&amp;author=S.C.%20Sim&amp;author=A.%20Van%20Deynze&amp;author=K.%20Stoffel&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e45520" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0045520" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0045520" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0045520"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23029069" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST005C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mueller</div> <div class="given-names">L.A.</div></div>,  <div class="name"><div class="surname">Tanksley</div> <div class="given-names">S.D.</div></div>,  <div class="name"><div class="surname">Giovannoni</div> <div class="given-names">J.J.</div></div>, et al. </div><div class="article-title">The Tomato Sequencing Project, the first cornerstone of the International Solanaceae Project (SOL)</div>, <div class="source">Comp. Funct. Genomics</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">153</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Tomato%20Sequencing%20Project%2C%20the%20first%20cornerstone%20of%20the%20International%20Solanaceae%20Project%20(SOL)&amp;author=L.A.%20Mueller&amp;author=S.D.%20Tanksley&amp;author=J.J.%20Giovannoni&amp;publication_year=2005&amp;journal=Comp.%20Funct.%20Genomics&amp;volume=6&amp;pages=153-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/cfg.468" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fcfg.468" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fcfg.468"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18629226" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST005C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yano</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Watanabe</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">MiBASE: a database of a miniature tomato cultivar Micro-Tom</div>, <div class="source"><em>Plant Biotechnol. J</em>.</div> , <div class="year">2006</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">195</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MiBASE%3A%20a%20database%20of%20a%20miniature%20tomato%20cultivar%20Micro-Tom&amp;author=K.%20Yano&amp;author=M.%20Watanabe&amp;author=N.%20Yamamoto&amp;publication_year=2006&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=23&amp;pages=195-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.5511/plantbiotechnology.23.195" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.5511%2Fplantbiotechnology.23.195" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.5511%2Fplantbiotechnology.23.195"> </span></div> </div></div></div></div></div><div content-id="DST005C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">SNP discovery and linkage map construction in cultivated tomato</div>, <div class="source">DNA Res.</div> , <div class="year">2010</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">381</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20discovery%20and%20linkage%20map%20construction%20in%20cultivated%20tomato&amp;author=K.%20Shirasawa&amp;author=S.%20Isobe&amp;author=H.%20Hirakawa&amp;publication_year=2010&amp;journal=DNA%20Res.&amp;volume=17&amp;pages=381-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsq024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsq024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsq024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21044984" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST005C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pilcher</div> <div class="given-names">C.D.</div></div>,  <div class="name"><div class="surname">Wong</div> <div class="given-names">J.K.</div></div>,  <div class="name"><div class="surname">Pillai</div> <div class="given-names">S.K.</div></div>. </div><div class="article-title">Inferring HIV transmission dynamics from phylogenetic sequence relationships</div>, <div class="source">PLoS Med.</div> , <div class="year">2008</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e69</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inferring%20HIV%20transmission%20dynamics%20from%20phylogenetic%20sequence%20relationships&amp;author=C.D.%20Pilcher&amp;author=J.K.%20Wong&amp;author=S.K.%20Pillai&amp;publication_year=2008&amp;journal=PLoS%20Med.&amp;volume=5&amp;pages=e69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pmed.0050069" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pmed.0050069" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pmed.0050069"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18351799" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST005C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Peterson</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Stecher</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>. </div><div class="article-title">MEGA5: molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2011</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">2731</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA5%3A%20molecular%20evolutionary%20genetics%20analysis%20using%20maximum%20likelihood%2C%20evolutionary%20distance%2C%20and%20maximum%20parsimony%20methods&amp;author=K.%20Tamura&amp;author=D.%20Peterson&amp;author=N.%20Peterson&amp;author=G.%20Stecher&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=28&amp;pages=2731-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21546353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST005C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bradbury</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Kroon</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Casstevens</div> <div class="given-names">T.M.</div></div>,  <div class="name"><div class="surname">Ramdoss</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Buckler</div> <div class="given-names">E.S.</div></div>. </div><div class="article-title">TASSEL: software for association mapping of complex traits in diverse samples</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">2633</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TASSEL%3A%20software%20for%20association%20mapping%20of%20complex%20traits%20in%20diverse%20samples&amp;author=P.J.%20Bradbury&amp;author=Z.%20Zhang&amp;author=D.E.%20Kroon&amp;author=T.M.%20Casstevens&amp;author=Y.%20Ramdoss&amp;author=E.S.%20Buckler&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2633-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm308" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm308" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm308"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17586829" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST005C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Schäffer</div> <div class="given-names">A.A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Sch%C3%A4ffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST005C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pearson</div> <div class="given-names">W.R.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">Improved tools for biological sequence comparison</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">1988</div>, vol. <div class="volume">85</div> (pg. <div class="fpage">2444</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20tools%20for%20biological%20sequence%20comparison&amp;author=W.R.%20Pearson&amp;author=D.J.%20Lipman&amp;publication_year=1988&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=85&amp;pages=2444-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.85.8.2444" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.85.8.2444" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.85.8.2444"> </span></div> </div></div></div></div></div><div content-id="DST005C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eswar</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Webb</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Marti-Renom</div> <div class="given-names">M.A.</div></div>, et al. </div><div class="article-title">Comparative protein structure modeling using Modeller</div>, <div class="source">Curr. Protoc. Bioinformatics</div> , <div class="year">2006</div> <div class="comment">Chapter 2: Unit 2.9</div></div></div></div></div><div content-id="DST005C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Le Guilloux</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Schmidtke</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Tuffery</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Fpocket: an open source platform for ligand pocket detection</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">168</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fpocket%3A%20an%20open%20source%20platform%20for%20ligand%20pocket%20detection&amp;author=V.%20Le%20Guilloux&amp;author=P.%20Schmidtke&amp;author=P.%20Tuffery&amp;publication_year=2009&amp;journal=BMC%20Bioinformatics&amp;volume=10&amp;pages=168" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-10-168" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-10-168" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-10-168"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19486540" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST005C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garcia-Hernandez</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Berardini</div> <div class="given-names">T.Z.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">TAIR: a resource for integrated Arabidopsis data</div>, <div class="source"><em>Funct. Integr. Genomic</em>s</div> , <div class="year">2002</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">239</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TAIR%3A%20a%20resource%20for%20integrated%20Arabidopsis%20data&amp;author=M.%20Garcia-Hernandez&amp;author=T.Z.%20Berardini&amp;author=G.%20Chen&amp;publication_year=2002&amp;journal=Funct.%20Integr.%20Genomics&amp;volume=2&amp;pages=239-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10142-002-0077-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10142-002-0077-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10142-002-0077-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12444417" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST005C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tatusov</div> <div class="given-names">R.L.</div></div>,  <div class="name"><div class="surname">Fedorova</div> <div class="given-names">N.D.</div></div>,  <div class="name"><div class="surname">Jackson</div> <div class="given-names">J.D.</div></div>, et al. </div><div class="article-title">The COG database: an updated version includes eukaryotes</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2003</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">41</div>-<div class="lpage">54</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20COG%20database%3A%20an%20updated%20version%20includes%20eukaryotes&amp;author=R.L.%20Tatusov&amp;author=N.D.%20Fedorova&amp;author=J.D.%20Jackson&amp;publication_year=2003&amp;journal=BMC%20Bioinformatics&amp;volume=4&amp;pages=41-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-4-41" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-4-41" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-4-41"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12969510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST005C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ogata</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Goto</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Fujibuchi</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Bono</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div>. </div><div class="article-title">KEGG: Kyoto Encyclopedia of Genes and Genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">29</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KEGG%3A%20Kyoto%20Encyclopedia%20of%20Genes%20and%20Genomes&amp;author=H.%20Ogata&amp;author=S.%20Goto&amp;author=K.%20Sato&amp;author=W.%20Fujibuchi&amp;author=H.%20Bono&amp;author=M.%20Kanehisa&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=29-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.1.29" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.1.29" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.1.29"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9847135" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST005C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Punta</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Coggill</div> <div class="given-names">P.C.</div></div>,  <div class="name"><div class="surname">Eberhardt</div> <div class="given-names">R.Y.</div></div>, et al. </div><div class="article-title">The Pfam protein families database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2012</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">D290</div>-<div class="lpage">301</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Pfam%20protein%20families%20database&amp;author=M.%20Punta&amp;author=P.C.%20Coggill&amp;author=R.Y.%20Eberhardt&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=D290-301" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1065" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1065" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1065"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22127870" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST005C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">Accelerated profile HMM searches</div>, <div class="source">PLoS Comput. Biol.</div> , <div class="year">2011</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e1002195</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accelerated%20profile%20HMM%20searches&amp;author=S.R.%20Eddy&amp;publication_year=2011&amp;journal=PLoS%20Comput.%20Biol.&amp;volume=7&amp;pages=e1002195" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1002195" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1002195" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1002195"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22039361" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST005C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saito</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Matsunaga</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Yoshida</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">‘Tomato Chuukanbohon Nou 11’, a tomato parental line with a short-internode trait</div>, <div class="source">Bull. Natl. Inst. Veg. Tea Sci.</div> , <div class="year">2007</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">65</div>-<div class="lpage">76</div>)</div></div></div></div><div content-id="DST005C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Scott</div> <div class="given-names">J.W.</div></div>,  <div class="name"><div class="surname">Harbaugh</div> <div class="given-names">B.K.</div></div>. </div><div class="article-title">Micro-Tom A miniature dwarf tomato</div>, <div class="source">Florida Agr. Expt. Sta. Circ.</div> , <div class="year">1989</div>, vol. <div class="volume">370</div> (pg. <div class="fpage">1</div>-<div class="lpage">6</div>)</div></div></div></div><div content-id="DST005C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Meissner</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Chague</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Emmanuel</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Elkind</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Levy</div> <div class="given-names">A.A.</div></div>. </div><div class="article-title">A high throughput system for transposon tagging and promoter trapping in tomato</div>, <div class="source">Plant J.</div> , <div class="year">2000</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">265</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high%20throughput%20system%20for%20transposon%20tagging%20and%20promoter%20trapping%20in%20tomato&amp;author=R.%20Meissner&amp;author=V.%20Chague&amp;author=Q.%20Zhu&amp;author=E.%20Emmanuel&amp;author=Y.%20Elkind&amp;author=A.A.%20Levy&amp;publication_year=2000&amp;journal=Plant%20J.&amp;volume=22&amp;pages=265-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313x.2000.00735.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313x.2000.00735.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313x.2000.00735.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10849344" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST005C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Laterrot</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Combined use of male sterility and resistance to tobacco mosaic virus in tomato</div>, <div class="source">Ann. Amelio. Plant</div> , <div class="year">1976</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">517</div>-<div class="lpage">21</div>)</div></div></div></div><div content-id="DST005C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fridman</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Pleban</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Zamir</div> <div class="given-names">D.</div></div>. </div><div class="article-title">A recombination hotspot delimits a wild-species quantitative trait locus for tomato sugar content to 484 bp within an invertase gene</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2000</div>, vol. <div class="volume">97</div> (pg. <div class="fpage">4718</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20recombination%20hotspot%20delimits%20a%20wild-species%20quantitative%20trait%20locus%20for%20tomato%20sugar%20content%20to%20484%20bp%20within%20an%20invertase%20gene&amp;author=E.%20Fridman&amp;author=T.%20Pleban&amp;author=D.%20Zamir&amp;publication_year=2000&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=97&amp;pages=4718-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.97.9.4718" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.97.9.4718" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.97.9.4718"> </span></div> </div></div></div></div></div><div content-id="DST005C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fridman</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Carrari</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">Y.S.</div></div>,  <div class="name"><div class="surname">Fernie</div> <div class="given-names">A.R.</div></div>,  <div class="name"><div class="surname">Zamir</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Zooming in on a quantitative trait for tomato yield using interspecific introgressions</div>, <div class="source">Science</div> , <div class="year">2004</div>, vol. <div class="volume">305</div> (pg. <div class="fpage">1786</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Zooming%20in%20on%20a%20quantitative%20trait%20for%20tomato%20yield%20using%20interspecific%20introgressions&amp;author=E.%20Fridman&amp;author=F.%20Carrari&amp;author=Y.S.%20Liu&amp;author=A.R.%20Fernie&amp;author=D.%20Zamir&amp;publication_year=2004&amp;journal=Science&amp;volume=305&amp;pages=1786-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1101666" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1101666" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1101666"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15375271" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST005C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST005C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kumar</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sangwan</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yadav</div> <div class="given-names">I.S.</div></div>,  <div class="name"><div class="surname">Yadav</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Protein modeling and active site binding mode interactions of myrosinase-sinigrin in <em>Brassica juncea</em>–an <em>in silico</em> approach</div>, <div class="source">J. Mol. Graph. Model</div> , <div class="year">2011</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">740</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20modeling%20and%20active%20site%20binding%20mode%20interactions%20of%20myrosinase-sinigrin%20in%20Brassica%20juncea%E2%80%93an%20in%20silico%20approach&amp;author=R.%20Kumar&amp;author=S.%20Kumar&amp;author=S.%20Sangwan&amp;author=I.S.%20Yadav&amp;author=R.%20Yadav&amp;publication_year=2011&amp;journal=J.%20Mol.%20Graph.%20Model&amp;volume=29&amp;pages=740-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jmgm.2010.12.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jmgm.2010.12.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jmgm.2010.12.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21236711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Prof. Hiroyuki Toh</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by-nc/3.0/), which permits non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=368082&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/dst005-dst005supp_table2.xlsx?Expires=2147483647&amp;Signature=KO-BkpQ1F7yRjCBxtToqv~7oai7nGzSr0AgikfarmQG87HWv-5hZdS08YhJj1dhg~dxOjIXBIz108UiohN0PaXRoMYpkmEw1gyyerRyygwnp~j2MndUnlEMcXvCIOjQuupWSibI5IsbMcWnUkTBzb50GgVrcUxW2J-JfuOzT1Z6QO2UJ9fOCUqBtKGtjqePqSL9B46PEKzDacxnAWU9WadBasxqssDvafryzqQTgpvSqnKvjVE12M-oeGfLOVIgnXD3l8HWUiDFUl71s9dPs4UuD2v8hPYxY1BdvxJDiiSeEkVFSaquekDp8r9IBGQzN9E-AIw8BIeZaPtYkEVEXfg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/dst005-dst005supp_table1.xlsx?Expires=2147483647&amp;Signature=YfzwpHvSYOeFuXT5wN41COYS6OfwgL9KluSiEvym3sV-CwmbMtqXvePvdcBHhnQRu~Lu8wUHZeU-dBR3KLsjAZr~LkOCs4ExIyjU2wkdtXPMNg5MFRugWmEBisbqzE3vQTHZPy3DGF5Sm33Q7gDiqSO6feOy3jNsKPgW6TVXSfnOnCmLMdjZji0lMiXLd0WIY7e-E8u5m~BGLQTBpW8Ux8TtXenzeOuhFNNK7TzUttAqViCuDr9ZOavXSz1BjYwxWzhsNV8ekQ4iYtGN8-gQU~vYsSLEWKsaovGCWL8pnocXxOlogXsr4T26xVXhCDhXvHmVf6iZEBZYxz4cPijLeA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/dst005-dst005supp_fig2.pdf?Expires=2147483647&amp;Signature=jD2oBTXWNaD8QRcSt3LXaqhsCx2~Qp1iC0VCDBz0~ghItPceFpGqiEutOkUtObeZjJ8ITy~4FTDzo5o1g3yxxk5aISR57Ntabj64T-5me~GDRS~Os9QB2b5rGB7uLSHiNyT0funGL-AjBflQfdKjtWueNEoHO2vPH0mJrGAO47rRKahGug00R5sK-3reuPi9bWIh~B-pDyMxGPSnd5Iy~Mnyq86RaTUuSiEu87ajhXbmP1olKAtCPKN~enY38Aesw7r8rbPJKzrrDMEVycOQZqP3Ujw-UPhsEnWqZMhmMIXexg1EyT5jhEm6tWSrFQcdspkX8mVOKQindW0uZhXpfg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/dst005-dst005supp_fig1.pdf?Expires=2147483647&amp;Signature=dISG0yXuZvCN6UEtnKvgiDSm5cRjtVzuiuujXmBkmN-NdzQPRH8S~cfJW~YQCYNPOaxwq6ado7uIFWGkPgWMlvDP3fQNWahuCMYC9hPjhL1R2P8WOT6uxUsHp8J80coQ1XL2fO5BT16~ykBex0~lcH2thg6qdC73aiEwhMfa9MpRgw8GsPuIqr~8w2sMgIQLsAqVG6xD7zcVjoWpTozSq9Zb6f0NHnX2~M~3wxj~Ehpt-bWsSn7WFaBaIuGffM8yax0Meq1YF7Su2FKiofhvhgBrWvx4cprF1hA8dLI6sbcMFcO6waYV94qPMv15jfK2dpW8U8d3waa4lUQ1G3Zyrw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/3/10.1093/dnares/dst005/2/dst005-dst005supp.doc?Expires=2147483647&amp;Signature=dJYFFCO9b-Nbi8RWZokqzf5AK988cnzjeLa4H~poTe1k6OzIjkqXzjOsp7lw~dZn8Z3LXHxK~1nzKxeM1sKZeeH8oOsIlg4PLG6HrxLoLixEW0ESysI0Zgw1ZxiUPTUNoCH6kijmS8FYGUrkQO~Z8ibwRuaT59-PhxZKFNPq59BIDtPexfraJF9z3u~l0kWGZHmbQgF6anZz7XoJERRjBxBZ4ZedrrXmopEdf~BMIkSO5XnwEAMy7L1WVWOH6F0wgS7OPf6EcQCVo~G0RVkoG0PlAqSDs7Grgw5gmcYBUE~X-SsQW4OvHdQxdvVjis08ezy6WpIjfmHs4WmSqBjj4g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">940</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000321404200002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">28</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst005"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="368082"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000321404200002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f3%2f221%2f368082">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000321404200002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (28)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F3%2F221%2F368082" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/368082'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome-Wide SNP Genotyping to Infer the Effects on Gene Functions in Tomato" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst005");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f3%2f221%2f368082">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

