

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/3/253/1745562","@type":"ScholarlyArticle","name":"Diversity and population-genetic properties of copy number variations and multicopy genes in cattle","datePublished":"2016-04-15","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2016-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw013","keywords":["cattle genome","population sequencing","copy number variation","taurine","indicine"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/3/253/1745562","author":[{"name":"Bickhart, Derek M.","affiliation":"1  USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705 ,  USA","@type":"Person"},{"name":"Xu, Lingyang","affiliation":"1  USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705 ,  USA    2  Department of Animal and Avian Sciences ,  University of Maryland ,  College Park, MD 20742 ,  USA","@type":"Person"},{"name":"Hutchison, Jana L.","affiliation":"1  USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705 ,  USA","@type":"Person"},{"name":"Cole, John B.","affiliation":"1  USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705 ,  USA","@type":"Person"},{"name":"Null, Daniel J.","affiliation":"1  USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705 ,  USA","@type":"Person"},{"name":"Schroeder, Steven G.","affiliation":"1  USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705 ,  USA","@type":"Person"},{"name":"Song, Jiuzhou","affiliation":"2  Department of Animal and Avian Sciences ,  University of Maryland ,  College Park, MD 20742 ,  USA","@type":"Person"},{"name":"Garcia, Jose Fernando","affiliation":"3  Universidade Estadual Paulista (UNESP) ,  Araçatuba, São Paulo ,  Brazil","@type":"Person"},{"name":"Sonstegard, Tad S.","affiliation":"1  USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705 ,  USA","@type":"Person"},{"name":"Van Tassell, Curtis P.","affiliation":"1  USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705 ,  USA","@type":"Person"},{"name":"Schnabel, Robert D.","affiliation":"4  Division of Animal Sciences ,  University of Missouri ,  Columbia, MO 65211 ,  USA    5  Informatics Institute ,  University of Missouri ,  Columbia, MO ,  USA","@type":"Person"},{"name":"Taylor, Jeremy F.","affiliation":"4  Division of Animal Sciences ,  University of Missouri ,  Columbia, MO 65211 ,  USA","@type":"Person"},{"name":"Lewin, Harris A.","affiliation":"6  Department of Evolution and Ecology ,  University of California ,  Davis, CA 95616 ,  USA","@type":"Person"},{"name":"Liu, George E.","affiliation":"1  USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705 ,  USA","@type":"Person"}],"description":"Abstract.  The diversity and population genetics of copy number variation (CNV) in domesticated animals are not well understood. In this study, we analysed 75 g","pageStart":"253","pageEnd":"262","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/3/1/m_cover.gif?Expires=2147483647&Signature=LvDBTdLDdppm3i5tRNwSqz~B0k8-VHsmqHIBBCnnRcYDTL18U11IK0gTUDCr2aaiXR5ym0b2CrICUCmysLmWh9~qZCnuy6WtlRXMtjn3dUPX5JDysxfXgqaVimgPXdDj9MvRKYJHTpxKiiGBqdmD60Vyu-3HYo2rC7nhbU2Csw3AsHXqH2b5-on-VVv5O5CbzJ9cFVKqO2JUMAOtEwy~eR47JxqnpIruQ5TsFm9yeuF0a6K97NPEMZUdg8UsfzWvyLfc5Oo3QfFl0mocgffkA9IZdbrSQKGCcRcbah0WxTU9-qdiJP1Uf~c0qXmpfTlvciHM7TJsqSeOwpYzS5R1dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Diversity and population-genetic properties of copy number variations and multicopy genes in cattle","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/3/1/m_cover.gif?Expires=2147483647&Signature=LvDBTdLDdppm3i5tRNwSqz~B0k8-VHsmqHIBBCnnRcYDTL18U11IK0gTUDCr2aaiXR5ym0b2CrICUCmysLmWh9~qZCnuy6WtlRXMtjn3dUPX5JDysxfXgqaVimgPXdDj9MvRKYJHTpxKiiGBqdmD60Vyu-3HYo2rC7nhbU2Csw3AsHXqH2b5-on-VVv5O5CbzJ9cFVKqO2JUMAOtEwy~eR47JxqnpIruQ5TsFm9yeuF0a6K97NPEMZUdg8UsfzWvyLfc5Oo3QfFl0mocgffkA9IZdbrSQKGCcRcbah0WxTU9-qdiJP1Uf~c0qXmpfTlvciHM7TJsqSeOwpYzS5R1dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Diversity and population-genetic properties of copy number variations and multicopy genes in cattle" />
    <meta property="og:description" content="Abstract.  The diversity and population genetics of copy number variation (CNV) in domesticated animals are not well understood. In this study, we analysed 75 g" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/3/253/1745562" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-04-15" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Bickhart, Derek M." /><meta name="citation_author_institution" content="USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705, USA" /><meta name="citation_author" content="Xu, Lingyang" /><meta name="citation_author_institution" content="USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705, USA" /><meta name="citation_author_institution" content="Department of Animal and Avian Sciences, University of Maryland, College Park, MD 20742, USA" /><meta name="citation_author" content="Hutchison, Jana L." /><meta name="citation_author_institution" content="USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705, USA" /><meta name="citation_author" content="Cole, John B." /><meta name="citation_author_institution" content="USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705, USA" /><meta name="citation_author" content="Null, Daniel J." /><meta name="citation_author_institution" content="USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705, USA" /><meta name="citation_author" content="Schroeder, Steven G." /><meta name="citation_author_institution" content="USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705, USA" /><meta name="citation_author" content="Song, Jiuzhou" /><meta name="citation_author_institution" content="Department of Animal and Avian Sciences, University of Maryland, College Park, MD 20742, USA" /><meta name="citation_author" content="Garcia, Jose Fernando" /><meta name="citation_author_institution" content="Universidade Estadual Paulista (UNESP), Araçatuba, São Paulo, Brazil" /><meta name="citation_author" content="Sonstegard, Tad S." /><meta name="citation_author_institution" content="USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705, USA" /><meta name="citation_author" content="Van Tassell, Curtis P." /><meta name="citation_author_institution" content="USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705, USA" /><meta name="citation_author" content="Schnabel, Robert D." /><meta name="citation_author_institution" content="Division of Animal Sciences, University of Missouri, Columbia, MO 65211, USA" /><meta name="citation_author_institution" content="Informatics Institute, University of Missouri, Columbia, MO, USA" /><meta name="citation_author" content="Taylor, Jeremy F." /><meta name="citation_author_institution" content="Division of Animal Sciences, University of Missouri, Columbia, MO 65211, USA" /><meta name="citation_author" content="Lewin, Harris A." /><meta name="citation_author_institution" content="Department of Evolution and Ecology, University of California, Davis, CA 95616, USA" /><meta name="citation_author" content="Liu, George E." /><meta name="citation_author_institution" content="USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705, USA" /><meta name="citation_title" content="Diversity and population-genetic properties of copy number variations and multicopy genes in cattle" /><meta name="citation_firstpage" content="253" /><meta name="citation_lastpage" content="262" /><meta name="citation_doi" content="10.1093/dnares/dsw013" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2016/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Mapping copy number variation by population-scale genome sequencing;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=470;  citation_pages=59-65; " /><meta name="citation_reference" content="citation_title=An integrated map of structural variation in 2,504 human genomes;  citation_journal_title=Nature;  citation_year=2015;  citation_volume=526;  citation_pages=75-81; " /><meta name="citation_reference" content="citation_title=Diversity of human copy number variation and multicopy genes;  citation_journal_title=Science;  citation_year=2010;  citation_volume=330;  citation_pages=641-6; " /><meta name="citation_reference" content="citation_title=Human copy number variation and complex genetic disease;  citation_journal_title=Annu. Rev. Genet.;  citation_year=2011;  citation_volume=45;  citation_pages=203-26; " /><meta name="citation_reference" content="citation_title=Identification of copy number variations and common deletion polymorphisms in cattle;  citation_journal_title=BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=232" /><meta name="citation_reference" content="citation_title=Mapping DNA structural variation in dogs;  citation_journal_title=Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=500-9; " /><meta name="citation_reference" content="citation_title=Copy number variation in the bovine genome;  citation_journal_title=BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=284" /><meta name="citation_reference" content="citation_title=Copy number variation and missense mutations of the agouti signaling protein (ASIP) gene in goat breeds with different coat colors;  citation_journal_title=Cytogenet. Genome Res.;  citation_year=2009;  citation_volume=126;  citation_pages=333-47; " /><meta name="citation_reference" content="citation_title=A first comparative map of copy number variations in the sheep genome;  citation_journal_title=Genomics;  citation_year=2011;  citation_volume=97;  citation_pages=158-65; " /><meta name="citation_reference" content="citation_title=Analysis of copy number variations among diverse cattle breeds;  citation_journal_title=Genome Res.;  citation_year=2010;  citation_volume=20;  citation_pages=693-703; " /><meta name="citation_reference" content="citation_title=The genomic architecture of segmental duplications and associated copy number variants in dogs;  citation_journal_title=Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=491-9; " /><meta name="citation_reference" content="citation_title=Copy number variation in the porcine genome inferred from a 60 k SNP BeadChip;  citation_journal_title=BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=593" /><meta name="citation_reference" content="citation_title=Analysis of copy number variants in the cattle genome;  citation_journal_title=Gene;  citation_year=2011;  citation_volume=482;  citation_pages=73-7; " /><meta name="citation_reference" content="citation_title=Personal genome sequencing: current approaches and challenges;  citation_journal_title=Genes Dev.;  citation_year=2010;  citation_volume=24;  citation_pages=423-31; " /><meta name="citation_reference" content="citation_title=Whole-genome resequencing reveals loci under selection during chicken domestication;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=464;  citation_pages=587-91; " /><meta name="citation_reference" content="citation_title=Copy number variation of individual cattle genomes using next-generation sequencing;  citation_journal_title=Genome Res.;  citation_year=2012;  citation_volume=22;  citation_pages=778-90; " /><meta name="citation_reference" content="citation_title=Serial translocation by means of circular intermediates underlies colour sidedness in cattle;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=482;  citation_pages=81-4; " /><meta name="citation_reference" content="citation_title=The population genetics of structural variation;  citation_journal_title=Nat. Genet;  citation_year=2007;  citation_volume=39;  citation_pages=S30-6; " /><meta name="citation_reference" content="citation_title=Population-genetic nature of copy number variations in the human genome;  citation_journal_title=Hum. Mol. Genet.;  citation_year=2010;  citation_volume=19;  citation_pages=761-73; " /><meta name="citation_reference" content="citation_title=Global diversity, population stratification, and selection of human copy-number variation;  citation_journal_title=Science;  citation_year=2015;  citation_volume=349;  citation_pages=aab3761" /><meta name="citation_reference" content="citation_title=Global variation in copy number in the human genome;  citation_journal_title=Nature;  citation_year=2006;  citation_volume=444;  citation_pages=444-54; " /><meta name="citation_reference" content="citation_title=Origins and functional impact of copy number variation in the human genome;  citation_journal_title=Nature;  citation_year=2009;  citation_volume=464;  citation_pages=704-12; " /><meta name="citation_reference" content="citation_title=Genotype, haplotype and copy-number variation in worldwide human populations;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=451;  citation_pages=998-1003; " /><meta name="citation_reference" content="citation_title=Exploring the role of copy number variants in human adaptation;  citation_journal_title=Trends Genet.;  citation_year=2012;  citation_volume=28;  citation_pages=245-57; " /><meta name="citation_reference" content="citation_title=Copy number variation in human health, disease, and evolution;  citation_journal_title=Annu. Rev. Genomics Hum. Genet.;  citation_year=2009;  citation_volume=10;  citation_pages=451-81; " /><meta name="citation_reference" content="citation_title=Copy number variation and evolution in humans and chimpanzees;  citation_journal_title=Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=1698-710; " /><meta name="citation_reference" content="citation_title=Evolutionary dynamics of copy number variation in pig genomes in the context of adaptation and domestication;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=449" /><meta name="citation_reference" content="citation_title=Extensive genetic diversity and substructuring among zebrafish strains revealed through copy number variant analysis;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2012;  citation_volume=109;  citation_pages=529-34; " /><meta name="citation_reference" content="citation_title=Novel origins of copy number variation in the dog genome;  citation_journal_title=Genome Biol.;  citation_year=2012;  citation_volume=13;  citation_pages=R73" /><meta name="citation_reference" content="citation_title=Diet and the evolution of human amylase gene copy number variation;  citation_journal_title=Nat. Genet.;  citation_year=2007;  citation_volume=39;  citation_pages=1256-60; " /><meta name="citation_reference" content="citation_title=Population stratification of a common APOBEC gene deletion polymorphism;  citation_journal_title=PLoS Genet.;  citation_year=2007;  citation_volume=3;  citation_pages=e63" /><meta name="citation_reference" content="citation_title=Adaptive evolution of UGT2B17 copy-number variation;  citation_journal_title=Am. J. Hum. Genet.;  citation_year=2008;  citation_volume=83;  citation_pages=337-46; " /><meta name="citation_reference" content="citation_title=Haplotype differences for copy number variants in the 22q11.23 region among human populations: a pigmentation-based model for selective pressure;  citation_journal_title=Eur. J. Hum. Genet.;  citation_year=2015;  citation_volume=23;  citation_pages=116-23; " /><meta name="citation_reference" content="citation_title=Common deletion polymorphisms in the human genome;  citation_journal_title=Nat. Genet.;  citation_year=2006;  citation_volume=38;  citation_pages=86-92; " /><meta name="citation_reference" content="citation_title=Genome wide CNV analysis reveals additional variants associated with milk production traits in Holsteins;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=683" /><meta name="citation_reference" content="citation_title=Mitochondrial diversity and the origins of African and European cattle;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=1996;  citation_volume=93;  citation_pages=5131-5; " /><meta name="citation_reference" content="citation_title=Genome wide survey of SNP variation uncovers the genetic structure of cattle breeds; citation_author=The Bovine HapMap Consortium;  citation_journal_title=Science;  citation_year=2009;  citation_volume=324;  citation_pages=528-32; " /><meta name="citation_reference" content="citation_title=Microsatellite DNA variation and the evolution, domestication and phylogeography of taurine and zebu cattle (Bos taurus and Bos indicus);  citation_journal_title=Genetics;  citation_year=1997;  citation_volume=146;  citation_pages=1071-86; " /><meta name="citation_reference" content="citation_title=Complete mitochondrial genomes of Bos taurus and Bos indicus provide new insights into intra-species variation, taxonomy and domestication;  citation_journal_title=Cytogenet. Genome Res.;  citation_year=2008;  citation_volume=120;  citation_pages=150-6; " /><meta name="citation_reference" content="citation_title=Evidence for two independent domestications of cattle;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=1994;  citation_volume=91;  citation_pages=2757-61; " /><meta name="citation_reference" content="citation_title=Current perspectives and the future of domestication studies;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2014;  citation_volume=111;  citation_pages=6139-46; " /><meta name="citation_reference" content="citation_title=Genetic evidence for Near-Eastern origins of European cattle;  citation_journal_title=Nature;  citation_year=2001;  citation_volume=410;  citation_pages=1088-91; " /><meta name="citation_reference" content="citation_title=African pastoralism: genetic imprints of origins and migrations;  citation_journal_title=Science;  citation_year=2002;  citation_volume=296;  citation_pages=336-9; " /><meta name="citation_reference" content="citation_title=Worldwide patterns of ancestry, divergence, and admixture in domesticated cattle;  citation_journal_title=PLoS Genet.;  citation_year=2014;  citation_volume=10;  citation_pages=e1004254" /><meta name="citation_reference" content="citation_title=Domestic-animal genomics: deciphering the genetics of complex traits;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2004;  citation_volume=5;  citation_pages=202-12; " /><meta name="citation_reference" content="citation_title=Invited review: reliability of genomic predictions for North American Holstein bulls;  citation_journal_title=J. Dairy Sci.;  citation_year=2009;  citation_volume=92;  citation_pages=16-24; " /><meta name="citation_reference" content="citation_title=Genomic divergence of zebu and taurine cattle identified through high-density SNP genotyping;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=876" /><meta name="citation_reference" content="citation_title=Personalized copy number and segmental duplication maps using next-generation sequencing;  citation_journal_title=Nat. Genet.;  citation_year=2009;  citation_volume=41;  citation_pages=1061-7; " /><meta name="citation_reference" content="citation_title=Analysis of copy number variants and segmental duplications in the human genome: evidence for a change in the process of formation in recent evolutionary history;  citation_journal_title=Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=1865-74; " /><meta name="citation_reference" content="citation_title=Clustering rules: a comparison of partitioning and hierarchical clustering algorithms;  citation_journal_title=J. Math. Model. Algor.;  citation_year=2006;  citation_volume=5;  citation_pages=475-504; " /><meta name="citation_reference" content="citation_title=Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies;  citation_journal_title=Genetics;  citation_year=2003;  citation_volume=164;  citation_pages=1567-87; " /><meta name="citation_reference" content="citation_title=Inference of population structure using multilocus genotype data;  citation_journal_title=Genetics;  citation_year=2000;  citation_volume=155;  citation_pages=945-59; " /><meta name="citation_reference" content="citation_title=Fine mapping of copy number variations on two cattle genome assemblies using high density SNP array;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=376" /><meta name="citation_reference" content="citation_title=A new statistical method for haplotype reconstruction from population data;  citation_journal_title=Am. J. Hum. Genet.;  citation_year=2001;  citation_volume=68;  citation_pages=978-89; " /><meta name="citation_reference" content="citation_title=Evidence for substantial fine-scale variation in recombination rates across the human genome;  citation_journal_title=Nat. Genet.;  citation_year=2004;  citation_volume=36;  citation_pages=700-6; " /><meta name="citation_reference" content="citation_title=Analysis of recent segmental duplications in the bovine genome;  citation_journal_title=BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=571" /><meta name="citation_reference" content="citation_title=Large multiallelic copy number variations in humans;  citation_journal_title=Nat. Genet.;  citation_year=2015;  citation_volume=47;  citation_pages=296-303; " /><meta name="citation_reference" content="citation_title=Digital genotyping of macrosatellites and multicopy genes reveals novel biological functions associated with copy number variation of large tandem repeats;  citation_journal_title=PLoS Genet.;  citation_year=2014;  citation_volume=10;  citation_pages=e1004418" /><meta name="citation_reference" content="citation_title=The landscape of human STR variation;  citation_journal_title=Genome Res.;  citation_year=2014;  citation_volume=24;  citation_pages=1894-904; " /><meta name="citation_reference" content="citation_title=MIPSTR: a method for multiplex genotyping of germline and somatic STR variation across many individuals;  citation_journal_title=Genome Res.;  citation_year=2015;  citation_volume=25;  citation_pages=750-61; " /><meta name="citation_reference" content="citation_title=Accurate typing of short tandem repeats from genome-wide sequencing data and its applications;  citation_journal_title=Genome Res.;  citation_year=2015;  citation_volume=25;  citation_pages=736-49; " /><meta name="citation_reference" content="citation_title=Lower linkage disequilibrium at CNVs is due to both recurrent mutation and transposing duplications;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2010;  citation_volume=27;  citation_pages=103-11; " /><meta name="citation_reference" content="citation_title=Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources;  citation_journal_title=Nat. Protoc.;  citation_year=2009;  citation_volume=4;  citation_pages=44-57; " /><meta name="citation_reference" content="citation_title=Mammalian TOR complex 2 controls the actin cytoskeleton and is rapamycin insensitive;  citation_journal_title=Nat. Cell Biol.;  citation_year=2004;  citation_volume=6;  citation_pages=1122-8; " /><meta name="citation_reference" content="citation_title=The trafficking protein Tmed2/p24β(1) is required for morphogenesis of the mouse embryo and placenta;  citation_journal_title=Dev. Biol.;  citation_year=2010;  citation_volume=341;  citation_pages=154-66; " /><meta name="citation_reference" content="citation_title=Chronic overexpression of PNPLA3I148M in mouse liver causes hepatic steatosis;  citation_journal_title=J. Clin. Invest.;  citation_year=2012;  citation_volume=122;  citation_pages=4130-44; " /><meta name="citation_reference" content="citation_title=Concerted and birth-and-death evolution of multigene families;  citation_journal_title=Annu. Rev. Genet.;  citation_year=2005;  citation_volume=39;  citation_pages=121-52; " /><meta name="citation_reference" content="citation_title=Overdominance in the human genome and olfactory receptor activity;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2008;  citation_volume=25;  citation_pages=997-1001; " /><meta name="citation_reference" content="citation_title=Combinatorial receptor codes for odors;  citation_journal_title=Cell;  citation_year=1999;  citation_volume=96;  citation_pages=713-23; " /><meta name="citation_reference" content="citation_title=Massive screening of copy number population-scale variation in Bos taurus genome;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=124" /><meta name="citation_reference" content="citation_title=A 660-Kb deletion with antagonistic effects on fertility and milk production segregates at high frequency in Nordic Red cattle: additional evidence for the common occurrence of balancing selection in livestock;  citation_journal_title=PLoS Genet.;  citation_year=2014;  citation_volume=10;  citation_pages=e1004049" /><meta name="citation_reference" content="citation_title=The coalescent with selection on copy number variants;  citation_journal_title=Genetics;  citation_year=2012;  citation_volume=190;  citation_pages=1077-86; " /><meta name="citation_reference" content="citation_title=Multi-locus inference of population structure: a comparison between single nucleotide polymorphisms and microsatellites;  citation_journal_title=Heredity (Edinb.);  citation_year=2011;  citation_volume=106;  citation_pages=158-71; " /><meta name="citation_reference" content="citation_title=Challenges in analysis and interpretation of microsatellite data for population genetic studies;  citation_journal_title=Ecol. Evol.;  citation_year=2014;  citation_volume=4;  citation_pages=4399-428; " /><meta name="citation_reference" content="citation_title=Whole-genome sequencing of 234 bulls facilitates mapping of monogenic and complex traits in cattle;  citation_journal_title=Nat. Genet.;  citation_year=2014;  citation_volume=46;  citation_pages=858-65; " /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/3/253/17463150/dsw013.pdf" /><meta name="description" content="Abstract.  The diversity and population genetics of copy number variation (CNV) in domesticated animals are not well understood. In this study, we analysed 75 g" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/3/253/1745562" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/3/253/1745562" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/3/253/1745562");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw013';
    </script>
<title>
	Diversity and population-genetic properties of copy number variations and multicopy genes in cattle | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./1745562" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZKwATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQmlja2hhcnQsIERlcmVrIE0uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVVNEQS1BUlMsIEFuaW1hbCBHZW5vbWljcyBhbmQgSW1wcm92ZW1lbnQgTGFib3JhdG9yeSwgQmVsdHN2aWxsZSwgTUQgMjA3MDUsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ilh1LCBMaW5neWFuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVTREEtQVJTLCBBbmltYWwgR2Vub21pY3MgYW5kIEltcHJvdmVtZW50IExhYm9yYXRvcnksIEJlbHRzdmlsbGUsIE1EIDIwNzA1LCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFuaW1hbCBhbmQgQXZpYW4gU2NpZW5jZXMsIFVuaXZlcnNpdHkgb2YgTWFyeWxhbmQsIENvbGxlZ2UgUGFyaywgTUQgMjA3NDIsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikh1dGNoaXNvbiwgSmFuYSBMLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVTREEtQVJTLCBBbmltYWwgR2Vub21pY3MgYW5kIEltcHJvdmVtZW50IExhYm9yYXRvcnksIEJlbHRzdmlsbGUsIE1EIDIwNzA1LCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDb2xlLCBKb2huIEIuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVVNEQS1BUlMsIEFuaW1hbCBHZW5vbWljcyBhbmQgSW1wcm92ZW1lbnQgTGFib3JhdG9yeSwgQmVsdHN2aWxsZSwgTUQgMjA3MDUsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik51bGwsIERhbmllbCBKLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVTREEtQVJTLCBBbmltYWwgR2Vub21pY3MgYW5kIEltcHJvdmVtZW50IExhYm9yYXRvcnksIEJlbHRzdmlsbGUsIE1EIDIwNzA1LCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTY2hyb2VkZXIsIFN0ZXZlbiBHLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVTREEtQVJTLCBBbmltYWwgR2Vub21pY3MgYW5kIEltcHJvdmVtZW50IExhYm9yYXRvcnksIEJlbHRzdmlsbGUsIE1EIDIwNzA1LCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTb25nLCBKaXV6aG91IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBBbmltYWwgYW5kIEF2aWFuIFNjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIE1hcnlsYW5kLCBDb2xsZWdlIFBhcmssIE1EIDIwNzQyLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHYXJjaWEsIEpvc2UgRmVybmFuZG8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJVbml2ZXJzaWRhZGUgRXN0YWR1YWwgUGF1bGlzdGEgKFVORVNQKSwgQXJhw6dhdHViYSwgU8OjbyBQYXVsbywgQnJhemlsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU29uc3RlZ2FyZCwgVGFkIFMuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVVNEQS1BUlMsIEFuaW1hbCBHZW5vbWljcyBhbmQgSW1wcm92ZW1lbnQgTGFib3JhdG9yeSwgQmVsdHN2aWxsZSwgTUQgMjA3MDUsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlZhbiBUYXNzZWxsLCBDdXJ0aXMgUC4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJVU0RBLUFSUywgQW5pbWFsIEdlbm9taWNzIGFuZCBJbXByb3ZlbWVudCBMYWJvcmF0b3J5LCBCZWx0c3ZpbGxlLCBNRCAyMDcwNSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2NobmFiZWwsIFJvYmVydCBELiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIEFuaW1hbCBTY2llbmNlcywgVW5pdmVyc2l0eSBvZiBNaXNzb3VyaSwgQ29sdW1iaWEsIE1PIDY1MjExLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbmZvcm1hdGljcyBJbnN0aXR1dGUsIFVuaXZlcnNpdHkgb2YgTWlzc291cmksIENvbHVtYmlhLCBNTywgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGF5bG9yLCBKZXJlbXkgRi4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEaXZpc2lvbiBvZiBBbmltYWwgU2NpZW5jZXMsIFVuaXZlcnNpdHkgb2YgTWlzc291cmksIENvbHVtYmlhLCBNTyA2NTIxMSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGV3aW4sIEhhcnJpcyBBLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgRXZvbHV0aW9uIGFuZCBFY29sb2d5LCBVbml2ZXJzaXR5IG9mIENhbGlmb3JuaWEsIERhdmlzLCBDQSA5NTYxNiwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGl1LCBHZW9yZ2UgRS4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJVU0RBLUFSUywgQW5pbWFsIEdlbm9taWNzIGFuZCBJbXByb3ZlbWVudCBMYWJvcmF0b3J5LCBCZWx0c3ZpbGxlLCBNRCAyMDcwNSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJEaXZlcnNpdHkgYW5kIHBvcHVsYXRpb24tZ2VuZXRpYyBwcm9wZXJ0aWVzIG9mIGNvcHkgbnVtYmVyIHZhcmlhdGlvbnMgYW5kIG11bHRpY29weSBnZW5lcyBpbiBjYXR0bGUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyNTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjI2MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdzAxMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE2LzA2LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwcGluZyBjb3B5IG51bWJlciB2YXJpYXRpb24gYnkgcG9wdWxhdGlvbi1zY2FsZSBnZW5vbWUgc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQ3MDsgIGNpdGF0aW9uX3BhZ2VzPTU5LTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbnRlZ3JhdGVkIG1hcCBvZiBzdHJ1Y3R1cmFsIHZhcmlhdGlvbiBpbiAyLDUwNCBodW1hbiBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NTI2OyAgY2l0YXRpb25fcGFnZXM9NzUtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpdmVyc2l0eSBvZiBodW1hbiBjb3B5IG51bWJlciB2YXJpYXRpb24gYW5kIG11bHRpY29weSBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zMzA7ICBjaXRhdGlvbl9wYWdlcz02NDEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SHVtYW4gY29weSBudW1iZXIgdmFyaWF0aW9uIGFuZCBjb21wbGV4IGdlbmV0aWMgZGlzZWFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz0yMDMtMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIGNvcHkgbnVtYmVyIHZhcmlhdGlvbnMgYW5kIGNvbW1vbiBkZWxldGlvbiBwb2x5bW9ycGhpc21zIGluIGNhdHRsZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MjMyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwcGluZyBETkEgc3RydWN0dXJhbCB2YXJpYXRpb24gaW4gZG9nczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz01MDAtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29weSBudW1iZXIgdmFyaWF0aW9uIGluIHRoZSBib3ZpbmUgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0yODQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db3B5IG51bWJlciB2YXJpYXRpb24gYW5kIG1pc3NlbnNlIG11dGF0aW9ucyBvZiB0aGUgYWdvdXRpIHNpZ25hbGluZyBwcm90ZWluIChBU0lQKSBnZW5lIGluIGdvYXQgYnJlZWRzIHdpdGggZGlmZmVyZW50IGNvYXQgY29sb3JzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DeXRvZ2VuZXQuIEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEyNjsgIGNpdGF0aW9uX3BhZ2VzPTMzMy00NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBmaXJzdCBjb21wYXJhdGl2ZSBtYXAgb2YgY29weSBudW1iZXIgdmFyaWF0aW9ucyBpbiB0aGUgc2hlZXAgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT05NzsgIGNpdGF0aW9uX3BhZ2VzPTE1OC02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgY29weSBudW1iZXIgdmFyaWF0aW9ucyBhbW9uZyBkaXZlcnNlIGNhdHRsZSBicmVlZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9NjkzLTcwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9taWMgYXJjaGl0ZWN0dXJlIG9mIHNlZ21lbnRhbCBkdXBsaWNhdGlvbnMgYW5kIGFzc29jaWF0ZWQgY29weSBudW1iZXIgdmFyaWFudHMgaW4gZG9nczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz00OTEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29weSBudW1iZXIgdmFyaWF0aW9uIGluIHRoZSBwb3JjaW5lIGdlbm9tZSBpbmZlcnJlZCBmcm9tIGEgNjAgayBTTlAgQmVhZENoaXA7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTU5MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIGNvcHkgbnVtYmVyIHZhcmlhbnRzIGluIHRoZSBjYXR0bGUgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5lOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQ4MjsgIGNpdGF0aW9uX3BhZ2VzPTczLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBlcnNvbmFsIGdlbm9tZSBzZXF1ZW5jaW5nOiBjdXJyZW50IGFwcHJvYWNoZXMgYW5kIGNoYWxsZW5nZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmVzIERldi47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz00MjMtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdob2xlLWdlbm9tZSByZXNlcXVlbmNpbmcgcmV2ZWFscyBsb2NpIHVuZGVyIHNlbGVjdGlvbiBkdXJpbmcgY2hpY2tlbiBkb21lc3RpY2F0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDY0OyAgY2l0YXRpb25fcGFnZXM9NTg3LTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db3B5IG51bWJlciB2YXJpYXRpb24gb2YgaW5kaXZpZHVhbCBjYXR0bGUgZ2Vub21lcyB1c2luZyBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz03NzgtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcmlhbCB0cmFuc2xvY2F0aW9uIGJ5IG1lYW5zIG9mIGNpcmN1bGFyIGludGVybWVkaWF0ZXMgdW5kZXJsaWVzIGNvbG91ciBzaWRlZG5lc3MgaW4gY2F0dGxlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDgyOyAgY2l0YXRpb25fcGFnZXM9ODEtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHBvcHVsYXRpb24gZ2VuZXRpY3Mgb2Ygc3RydWN0dXJhbCB2YXJpYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz1TMzAtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9wdWxhdGlvbi1nZW5ldGljIG5hdHVyZSBvZiBjb3B5IG51bWJlciB2YXJpYXRpb25zIGluIHRoZSBodW1hbiBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUh1bS4gTW9sLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz03NjEtNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdsb2JhbCBkaXZlcnNpdHksIHBvcHVsYXRpb24gc3RyYXRpZmljYXRpb24sIGFuZCBzZWxlY3Rpb24gb2YgaHVtYW4gY29weS1udW1iZXIgdmFyaWF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTM0OTsgIGNpdGF0aW9uX3BhZ2VzPWFhYjM3NjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HbG9iYWwgdmFyaWF0aW9uIGluIGNvcHkgbnVtYmVyIGluIHRoZSBodW1hbiBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00NDQ7ICBjaXRhdGlvbl9wYWdlcz00NDQtNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9yaWdpbnMgYW5kIGZ1bmN0aW9uYWwgaW1wYWN0IG9mIGNvcHkgbnVtYmVyIHZhcmlhdGlvbiBpbiB0aGUgaHVtYW4gZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDY0OyAgY2l0YXRpb25fcGFnZXM9NzA0LTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vdHlwZSwgaGFwbG90eXBlIGFuZCBjb3B5LW51bWJlciB2YXJpYXRpb24gaW4gd29ybGR3aWRlIGh1bWFuIHBvcHVsYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDUxOyAgY2l0YXRpb25fcGFnZXM9OTk4LTEwMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGxvcmluZyB0aGUgcm9sZSBvZiBjb3B5IG51bWJlciB2YXJpYW50cyBpbiBodW1hbiBhZGFwdGF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MjQ1LTU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db3B5IG51bWJlciB2YXJpYXRpb24gaW4gaHVtYW4gaGVhbHRoLCBkaXNlYXNlLCBhbmQgZXZvbHV0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIEdlbm9taWNzIEh1bS4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NDUxLTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db3B5IG51bWJlciB2YXJpYXRpb24gYW5kIGV2b2x1dGlvbiBpbiBodW1hbnMgYW5kIGNoaW1wYW56ZWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTE2OTgtNzEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgZHluYW1pY3Mgb2YgY29weSBudW1iZXIgdmFyaWF0aW9uIGluIHBpZyBnZW5vbWVzIGluIHRoZSBjb250ZXh0IG9mIGFkYXB0YXRpb24gYW5kIGRvbWVzdGljYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTQ0OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4dGVuc2l2ZSBnZW5ldGljIGRpdmVyc2l0eSBhbmQgc3Vic3RydWN0dXJpbmcgYW1vbmcgemVicmFmaXNoIHN0cmFpbnMgcmV2ZWFsZWQgdGhyb3VnaCBjb3B5IG51bWJlciB2YXJpYW50IGFuYWx5c2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEwOTsgIGNpdGF0aW9uX3BhZ2VzPTUyOS0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm92ZWwgb3JpZ2lucyBvZiBjb3B5IG51bWJlciB2YXJpYXRpb24gaW4gdGhlIGRvZyBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPVI3MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZXQgYW5kIHRoZSBldm9sdXRpb24gb2YgaHVtYW4gYW15bGFzZSBnZW5lIGNvcHkgbnVtYmVyIHZhcmlhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz0xMjU2LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3B1bGF0aW9uIHN0cmF0aWZpY2F0aW9uIG9mIGEgY29tbW9uIEFQT0JFQyBnZW5lIGRlbGV0aW9uIHBvbHltb3JwaGlzbTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPWU2MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFkYXB0aXZlIGV2b2x1dGlvbiBvZiBVR1QyQjE3IGNvcHktbnVtYmVyIHZhcmlhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW0uIEouIEh1bS4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTgzOyAgY2l0YXRpb25fcGFnZXM9MzM3LTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IYXBsb3R5cGUgZGlmZmVyZW5jZXMgZm9yIGNvcHkgbnVtYmVyIHZhcmlhbnRzIGluIHRoZSAyMnExMS4yMyByZWdpb24gYW1vbmcgaHVtYW4gcG9wdWxhdGlvbnM6IGEgcGlnbWVudGF0aW9uLWJhc2VkIG1vZGVsIGZvciBzZWxlY3RpdmUgcHJlc3N1cmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV1ci4gSi4gSHVtLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xMTYtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbW1vbiBkZWxldGlvbiBwb2x5bW9ycGhpc21zIGluIHRoZSBodW1hbiBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9ODYtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSB3aWRlIENOViBhbmFseXNpcyByZXZlYWxzIGFkZGl0aW9uYWwgdmFyaWFudHMgYXNzb2NpYXRlZCB3aXRoIG1pbGsgcHJvZHVjdGlvbiB0cmFpdHMgaW4gSG9sc3RlaW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz02ODMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaXRvY2hvbmRyaWFsIGRpdmVyc2l0eSBhbmQgdGhlIG9yaWdpbnMgb2YgQWZyaWNhbiBhbmQgRXVyb3BlYW4gY2F0dGxlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTkzOyAgY2l0YXRpb25fcGFnZXM9NTEzMS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgd2lkZSBzdXJ2ZXkgb2YgU05QIHZhcmlhdGlvbiB1bmNvdmVycyB0aGUgZ2VuZXRpYyBzdHJ1Y3R1cmUgb2YgY2F0dGxlIGJyZWVkczsgY2l0YXRpb25fYXV0aG9yPVRoZSBCb3ZpbmUgSGFwTWFwIENvbnNvcnRpdW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzI0OyAgY2l0YXRpb25fcGFnZXM9NTI4LTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb3NhdGVsbGl0ZSBETkEgdmFyaWF0aW9uIGFuZCB0aGUgZXZvbHV0aW9uLCBkb21lc3RpY2F0aW9uIGFuZCBwaHlsb2dlb2dyYXBoeSBvZiB0YXVyaW5lIGFuZCB6ZWJ1IGNhdHRsZSAoQm9zIHRhdXJ1cyBhbmQgQm9zIGluZGljdXMpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDY7ICBjaXRhdGlvbl9wYWdlcz0xMDcxLTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBtaXRvY2hvbmRyaWFsIGdlbm9tZXMgb2YgQm9zIHRhdXJ1cyBhbmQgQm9zIGluZGljdXMgcHJvdmlkZSBuZXcgaW5zaWdodHMgaW50byBpbnRyYS1zcGVjaWVzIHZhcmlhdGlvbiwgdGF4b25vbXkgYW5kIGRvbWVzdGljYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN5dG9nZW5ldC4gR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTIwOyAgY2l0YXRpb25fcGFnZXM9MTUwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2aWRlbmNlIGZvciB0d28gaW5kZXBlbmRlbnQgZG9tZXN0aWNhdGlvbnMgb2YgY2F0dGxlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTkxOyAgY2l0YXRpb25fcGFnZXM9Mjc1Ny02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3VycmVudCBwZXJzcGVjdGl2ZXMgYW5kIHRoZSBmdXR1cmUgb2YgZG9tZXN0aWNhdGlvbiBzdHVkaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTExMTsgIGNpdGF0aW9uX3BhZ2VzPTYxMzktNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgZXZpZGVuY2UgZm9yIE5lYXItRWFzdGVybiBvcmlnaW5zIG9mIEV1cm9wZWFuIGNhdHRsZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTQxMDsgIGNpdGF0aW9uX3BhZ2VzPTEwODgtOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFmcmljYW4gcGFzdG9yYWxpc206IGdlbmV0aWMgaW1wcmludHMgb2Ygb3JpZ2lucyBhbmQgbWlncmF0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOTY7ICBjaXRhdGlvbl9wYWdlcz0zMzYtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V29ybGR3aWRlIHBhdHRlcm5zIG9mIGFuY2VzdHJ5LCBkaXZlcmdlbmNlLCBhbmQgYWRtaXh0dXJlIGluIGRvbWVzdGljYXRlZCBjYXR0bGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9ZTEwMDQyNTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Eb21lc3RpYy1hbmltYWwgZ2Vub21pY3M6IGRlY2lwaGVyaW5nIHRoZSBnZW5ldGljcyBvZiBjb21wbGV4IHRyYWl0czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MjAyLTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnZpdGVkIHJldmlldzogcmVsaWFiaWxpdHkgb2YgZ2Vub21pYyBwcmVkaWN0aW9ucyBmb3IgTm9ydGggQW1lcmljYW4gSG9sc3RlaW4gYnVsbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIERhaXJ5IFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9OTI7ICBjaXRhdGlvbl9wYWdlcz0xNi0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBkaXZlcmdlbmNlIG9mIHplYnUgYW5kIHRhdXJpbmUgY2F0dGxlIGlkZW50aWZpZWQgdGhyb3VnaCBoaWdoLWRlbnNpdHkgU05QIGdlbm90eXBpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTg3NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBlcnNvbmFsaXplZCBjb3B5IG51bWJlciBhbmQgc2VnbWVudGFsIGR1cGxpY2F0aW9uIG1hcHMgdXNpbmcgbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9MTA2MS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBjb3B5IG51bWJlciB2YXJpYW50cyBhbmQgc2VnbWVudGFsIGR1cGxpY2F0aW9ucyBpbiB0aGUgaHVtYW4gZ2Vub21lOiBldmlkZW5jZSBmb3IgYSBjaGFuZ2UgaW4gdGhlIHByb2Nlc3Mgb2YgZm9ybWF0aW9uIGluIHJlY2VudCBldm9sdXRpb25hcnkgaGlzdG9yeTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xODY1LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbHVzdGVyaW5nIHJ1bGVzOiBhIGNvbXBhcmlzb24gb2YgcGFydGl0aW9uaW5nIGFuZCBoaWVyYXJjaGljYWwgY2x1c3RlcmluZyBhbGdvcml0aG1zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNYXRoLiBNb2RlbC4gQWxnb3IuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz00NzUtNTA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmZlcmVuY2Ugb2YgcG9wdWxhdGlvbiBzdHJ1Y3R1cmUgdXNpbmcgbXVsdGlsb2N1cyBnZW5vdHlwZSBkYXRhOiBsaW5rZWQgbG9jaSBhbmQgY29ycmVsYXRlZCBhbGxlbGUgZnJlcXVlbmNpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE2NDsgIGNpdGF0aW9uX3BhZ2VzPTE1NjctODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluZmVyZW5jZSBvZiBwb3B1bGF0aW9uIHN0cnVjdHVyZSB1c2luZyBtdWx0aWxvY3VzIGdlbm90eXBlIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE1NTsgIGNpdGF0aW9uX3BhZ2VzPTk0NS01OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmluZSBtYXBwaW5nIG9mIGNvcHkgbnVtYmVyIHZhcmlhdGlvbnMgb24gdHdvIGNhdHRsZSBnZW5vbWUgYXNzZW1ibGllcyB1c2luZyBoaWdoIGRlbnNpdHkgU05QIGFycmF5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0zNzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5ldyBzdGF0aXN0aWNhbCBtZXRob2QgZm9yIGhhcGxvdHlwZSByZWNvbnN0cnVjdGlvbiBmcm9tIHBvcHVsYXRpb24gZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW0uIEouIEh1bS4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTY4OyAgY2l0YXRpb25fcGFnZXM9OTc4LTg5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmlkZW5jZSBmb3Igc3Vic3RhbnRpYWwgZmluZS1zY2FsZSB2YXJpYXRpb24gaW4gcmVjb21iaW5hdGlvbiByYXRlcyBhY3Jvc3MgdGhlIGh1bWFuIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz03MDAtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgcmVjZW50IHNlZ21lbnRhbCBkdXBsaWNhdGlvbnMgaW4gdGhlIGJvdmluZSBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTU3MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxhcmdlIG11bHRpYWxsZWxpYyBjb3B5IG51bWJlciB2YXJpYXRpb25zIGluIGh1bWFuczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz0yOTYtMzAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWdpdGFsIGdlbm90eXBpbmcgb2YgbWFjcm9zYXRlbGxpdGVzIGFuZCBtdWx0aWNvcHkgZ2VuZXMgcmV2ZWFscyBub3ZlbCBiaW9sb2dpY2FsIGZ1bmN0aW9ucyBhc3NvY2lhdGVkIHdpdGggY29weSBudW1iZXIgdmFyaWF0aW9uIG9mIGxhcmdlIHRhbmRlbSByZXBlYXRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPWUxMDA0NDE4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGxhbmRzY2FwZSBvZiBodW1hbiBTVFIgdmFyaWF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTE4OTQtOTA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NSVBTVFI6IGEgbWV0aG9kIGZvciBtdWx0aXBsZXggZ2Vub3R5cGluZyBvZiBnZXJtbGluZSBhbmQgc29tYXRpYyBTVFIgdmFyaWF0aW9uIGFjcm9zcyBtYW55IGluZGl2aWR1YWxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTc1MC02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWNjdXJhdGUgdHlwaW5nIG9mIHNob3J0IHRhbmRlbSByZXBlYXRzIGZyb20gZ2Vub21lLXdpZGUgc2VxdWVuY2luZyBkYXRhIGFuZCBpdHMgYXBwbGljYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTczNi00OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG93ZXIgbGlua2FnZSBkaXNlcXVpbGlicml1bSBhdCBDTlZzIGlzIGR1ZSB0byBib3RoIHJlY3VycmVudCBtdXRhdGlvbiBhbmQgdHJhbnNwb3NpbmcgZHVwbGljYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MTAzLTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeXN0ZW1hdGljIGFuZCBpbnRlZ3JhdGl2ZSBhbmFseXNpcyBvZiBsYXJnZSBnZW5lIGxpc3RzIHVzaW5nIERBVklEIGJpb2luZm9ybWF0aWNzIHJlc291cmNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBQcm90b2MuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz00NC01NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFtbWFsaWFuIFRPUiBjb21wbGV4IDIgY29udHJvbHMgdGhlIGFjdGluIGN5dG9za2VsZXRvbiBhbmQgaXMgcmFwYW15Y2luIGluc2Vuc2l0aXZlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIENlbGwgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTExMjItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHRyYWZmaWNraW5nIHByb3RlaW4gVG1lZDIvcDI0zrIoMSkgaXMgcmVxdWlyZWQgZm9yIG1vcnBob2dlbmVzaXMgb2YgdGhlIG1vdXNlIGVtYnJ5byBhbmQgcGxhY2VudGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPURldi4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MzQxOyAgY2l0YXRpb25fcGFnZXM9MTU0LTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaHJvbmljIG92ZXJleHByZXNzaW9uIG9mIFBOUExBM0kxNDhNIGluIG1vdXNlIGxpdmVyIGNhdXNlcyBoZXBhdGljIHN0ZWF0b3NpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQ2xpbi4gSW52ZXN0LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjI7ICBjaXRhdGlvbl9wYWdlcz00MTMwLTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25jZXJ0ZWQgYW5kIGJpcnRoLWFuZC1kZWF0aCBldm9sdXRpb24gb2YgbXVsdGlnZW5lIGZhbWlsaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPTEyMS01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3ZlcmRvbWluYW5jZSBpbiB0aGUgaHVtYW4gZ2Vub21lIGFuZCBvbGZhY3RvcnkgcmVjZXB0b3IgYWN0aXZpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz05OTctMTAwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tYmluYXRvcmlhbCByZWNlcHRvciBjb2RlcyBmb3Igb2RvcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9OTY7ICBjaXRhdGlvbl9wYWdlcz03MTMtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hc3NpdmUgc2NyZWVuaW5nIG9mIGNvcHkgbnVtYmVyIHBvcHVsYXRpb24tc2NhbGUgdmFyaWF0aW9uIGluIEJvcyB0YXVydXMgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIDY2MC1LYiBkZWxldGlvbiB3aXRoIGFudGFnb25pc3RpYyBlZmZlY3RzIG9uIGZlcnRpbGl0eSBhbmQgbWlsayBwcm9kdWN0aW9uIHNlZ3JlZ2F0ZXMgYXQgaGlnaCBmcmVxdWVuY3kgaW4gTm9yZGljIFJlZCBjYXR0bGU6IGFkZGl0aW9uYWwgZXZpZGVuY2UgZm9yIHRoZSBjb21tb24gb2NjdXJyZW5jZSBvZiBiYWxhbmNpbmcgc2VsZWN0aW9uIGluIGxpdmVzdG9jazsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1lMTAwNDA0OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb2FsZXNjZW50IHdpdGggc2VsZWN0aW9uIG9uIGNvcHkgbnVtYmVyIHZhcmlhbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xOTA7ICBjaXRhdGlvbl9wYWdlcz0xMDc3LTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aS1sb2N1cyBpbmZlcmVuY2Ugb2YgcG9wdWxhdGlvbiBzdHJ1Y3R1cmU6IGEgY29tcGFyaXNvbiBiZXR3ZWVuIHNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbXMgYW5kIG1pY3Jvc2F0ZWxsaXRlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SGVyZWRpdHkgKEVkaW5iLik7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTA2OyAgY2l0YXRpb25fcGFnZXM9MTU4LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFsbGVuZ2VzIGluIGFuYWx5c2lzIGFuZCBpbnRlcnByZXRhdGlvbiBvZiBtaWNyb3NhdGVsbGl0ZSBkYXRhIGZvciBwb3B1bGF0aW9uIGdlbmV0aWMgc3R1ZGllczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RWNvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTQzOTktNDI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZS1nZW5vbWUgc2VxdWVuY2luZyBvZiAyMzQgYnVsbHMgZmFjaWxpdGF0ZXMgbWFwcGluZyBvZiBtb25vZ2VuaWMgYW5kIGNvbXBsZXggdHJhaXRzIGluIGNhdHRsZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDY7ICBjaXRhdGlvbl9wYWdlcz04NTgtNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjMvMy8yNTMvMTc0NjMxNTAvZHN3MDEzLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBUaGUgZGl2ZXJzaXR5IGFuZCBwb3B1bGF0aW9uIGdlbmV0aWNzIG9mIGNvcHkgbnVtYmVyIHZhcmlhdGlvbiAoQ05WKSBpbiBkb21lc3RpY2F0ZWQgYW5pbWFscyBhcmUgbm90IHdlbGwgdW5kZXJzdG9vZC4gSW4gdGhpcyBzdHVkeSwgd2UgYW5hbHlzZWQgNzUgZyIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MTc0NTU2MiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRxNJSHOfYMy0wAhTyozg7UV7QGPw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f253%2f1745562" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f253%2f1745562" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=69063" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/3/1/m_cover.gif?Expires=2147483647&amp;Signature=LvDBTdLDdppm3i5tRNwSqz~B0k8-VHsmqHIBBCnnRcYDTL18U11IK0gTUDCr2aaiXR5ym0b2CrICUCmysLmWh9~qZCnuy6WtlRXMtjn3dUPX5JDysxfXgqaVimgPXdDj9MvRKYJHTpxKiiGBqdmD60Vyu-3HYo2rC7nhbU2Csw3AsHXqH2b5-on-VVv5O5CbzJ9cFVKqO2JUMAOtEwy~eR47JxqnpIruQ5TsFm9yeuF0a6K97NPEMZUdg8UsfzWvyLfc5Oo3QfFl0mocgffkA9IZdbrSQKGCcRcbah0WxTU9-qdiJP1Uf~c0qXmpfTlvciHM7TJsqSeOwpYzS5R1dw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/3">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87199581">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199581">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199584">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199584">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199593">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199593">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199622">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199622">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199671">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199671">Authors' contributions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199673">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199673">Conflict of interest statement</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199675">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199675">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199677">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199677">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199679">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199679">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/3/241/1745420">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/3/263/1745725">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Diversity and population-genetic properties of copy number variations and multicopy genes in cattle
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Derek M. Bickhart</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705</div>, <div class="country">USA</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +1 301-504-8592. Fax. +1 301-504-8092. E-mail: <a href="mailto:derek.bickhart@ars.usda.gov" target="_blank">derek.bickhart@ars.usda.gov</a> (D.M.B.); Tel. +1 301-504-9843. Fax. +1 301-504-8414. E-mail: <a href="mailto:george.liu@ars.usda.gov" target="_blank">george.liu@ars.usda.gov</a> (G.E.L.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Derek+M.+Bickhart">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bickhart D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bickhart D&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Derek M. Bickhart</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Lingyang Xu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705</div>, <div class="country">USA</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Animal and Avian Sciences</div>, <div class="insititution">University of Maryland</div>, <div class="addr-line">College Park, MD 20742</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Lingyang+Xu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Xu L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Xu L&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Lingyang Xu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jana L. Hutchison</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jana+L.+Hutchison">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hutchison J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hutchison J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jana L. Hutchison</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">John B. Cole</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=John+B.+Cole">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cole J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cole J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">John B. Cole</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Daniel J. Null</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Daniel+J.+Null">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Null D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Null D&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Daniel J. Null</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Steven G. Schroeder</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Steven+G.+Schroeder">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Schroeder S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Schroeder S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Steven G. Schroeder</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jiuzhou Song</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Animal and Avian Sciences</div>, <div class="insititution">University of Maryland</div>, <div class="addr-line">College Park, MD 20742</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jiuzhou+Song">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Song J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Song J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jiuzhou Song</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jose Fernando Garcia</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Universidade Estadual Paulista (UNESP)</div>, <div class="addr-line">Araçatuba, São Paulo</div>, <div class="country">Brazil</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jose+Fernando+Garcia">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Garcia J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Garcia J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jose Fernando Garcia</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tad S. Sonstegard</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tad+S.+Sonstegard">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sonstegard T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sonstegard T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Tad S. Sonstegard</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Curtis P. Van Tassell</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Curtis+P.+Van+Tassell">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Van Tassell C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Van Tassell C&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Curtis P. Van Tassell</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Robert D. Schnabel</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Division of Animal Sciences</div>, <div class="insititution">University of Missouri</div>, <div class="addr-line">Columbia, MO 65211</div>, <div class="country">USA</div></div><div class="aff"><div class="label -label">5</div><div class="addr-line">Informatics Institute</div>, <div class="insititution">University of Missouri</div>, <div class="addr-line">Columbia, MO</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Robert+D.+Schnabel">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Schnabel R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Schnabel R&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Robert D. Schnabel</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jeremy F. Taylor</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Division of Animal Sciences</div>, <div class="insititution">University of Missouri</div>, <div class="addr-line">Columbia, MO 65211</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jeremy+F.+Taylor">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Taylor J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Taylor J&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Jeremy F. Taylor</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Harris A. Lewin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="addr-line">Department of Evolution and Ecology</div>, <div class="insititution">University of California</div>, <div class="addr-line">Davis, CA 95616</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Harris+A.+Lewin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lewin H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lewin H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Harris A. Lewin</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">George E. Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">USDA-ARS, Animal Genomics and Improvement Laboratory, Beltsville, MD 20705</div>, <div class="country">USA</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +1 301-504-8592. Fax. +1 301-504-8092. E-mail: <a href="mailto:derek.bickhart@ars.usda.gov" target="_blank">derek.bickhart@ars.usda.gov</a> (D.M.B.); Tel. +1 301-504-9843. Fax. +1 301-504-8414. E-mail: <a href="mailto:george.liu@ars.usda.gov" target="_blank">george.liu@ars.usda.gov</a> (G.E.L.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=George+E.+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu G&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">George E. Liu</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors are co-first authors.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="footnote-content">Edited by Dr Osamu Ohara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 3, 1 June 2016, Pages 253–262, <a href='https://doi.org/10.1093/dnares/dsw013'>https://doi.org/10.1093/dnares/dsw013</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">15 April 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">05 November 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">29 February 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="1745562" href="/dnaresearch/article-pdf/23/3/253/17463150/dsw013.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Derek M. Bickhart, Lingyang Xu, Jana L. Hutchison, John B. Cole, Daniel J. Null, Steven G. Schroeder, Jiuzhou Song, Jose Fernando Garcia, Tad S. Sonstegard, Curtis P. Van Tassell, Robert D. Schnabel, Jeremy F. Taylor, Harris A. Lewin, George E. Liu;  Diversity and population-genetic properties of copy number variations and multicopy genes in cattle, <em>DNA Research</em>, Volume 23, Issue 3, 1 June 2016, Pages 253–262, <a href='https://doi.org/10.1093/dnares/dsw013'>https://doi.org/10.1093/dnares/dsw013</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/1745562?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745562?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745562?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745562?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745562?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745562?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745562?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Diversity%20and%20population-genetic%20properties%20of%20copy%20number%20variations%20and%20multicopy%20genes%20in%20cattle&amp;publicationDate=2016-04-15&amp;volumeNum=23&amp;issueNum=3&amp;author=Bickhart%2C%20Derek%20M.%3B%20Xu%2C%20Lingyang&amp;startPage=253&amp;endPage=262&amp;contentId=10.1093%2Fdnares%2Fdsw013&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f253%2f1745562" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87199581 id="87199581" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The diversity and population genetics of copy number variation (CNV) in domesticated animals are not well understood. In this study, we analysed 75 genomes of major taurine and indicine cattle breeds (including Angus, Brahman, Gir, Holstein, Jersey, Limousin, Nelore, and Romagnola), sequenced to 11-fold coverage to identify 1,853 non-redundant CNV regions. Supported by high validation rates in array comparative genomic hybridization (CGH) and qPCR experiments, these CNV regions accounted for 3.1% (87.5 Mb) of the cattle reference genome, representing a significant increase over previous estimates of the area of the genome that is copy number variable (∼2%). Further population genetics and evolutionary genomics analyses based on these CNVs revealed the population structures of the cattle taurine and indicine breeds and uncovered potential diversely selected CNVs near important functional genes, including <em>AOX1</em>, <em>ASZ1</em>, <em>GAT</em>, <em>GLYAT</em>, and <em>KRTAP9-1</em>. Additionally, 121 CNV gene regions were found to be either breed specific or differentially variable across breeds, such as <em>RICTOR</em> in dairy breeds and <em>PNPLA3</em> in beef breeds. In contrast, clusters of the <em>PRP</em> and <em>PAG</em> genes were found to be duplicated in all sequenced animals, suggesting that subfunctionalization, neofunctionalization, or overdominance play roles in diversifying those fertility-related genes. These CNV results provide a new glimpse into the diverse selection histories of cattle breeds and a basis for correlating structural variation with complex traits in the future.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;cattle genome&quot;">cattle genome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;population sequencing&quot;">population sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;copy number variation&quot;">copy number variation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="taurine">taurine</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="indicine">indicine</a></div>                            <h2 scrollto-destination=87199584 id="87199584" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Copy number variations (CNVs) are deletions and insertions of genomic sequence between two individuals of a species.<sup><a reveal-id="DSW013C1  DSW013C3" data-open="DSW013C1  DSW013C3" class="link link-ref link-reveal xref-bibr">1–3</a></sup> Substantial progress has been made in understanding the impacts of CNVs on both normal phenotypic variability and disease susceptibility in human.<sup><a reveal-id="DSW013C4" data-open="DSW013C4" class="link link-ref link-reveal xref-bibr">4</a></sup> The majority of previous studies of CNV in domesticated animals have been based on array comparative genomic hybridization (CGH) experiments or single-nucleotide polymorphism (SNP) arrays.<sup><a reveal-id="DSW013C5  DSW013C13" data-open="DSW013C5  DSW013C13" class="link link-ref link-reveal xref-bibr">5–13</a></sup> As in human,<sup><a reveal-id="DSW013C14" data-open="DSW013C14" class="link link-ref link-reveal xref-bibr">14</a></sup> sequence-based approaches are becoming popular for the study of CNVs in domesticated animals.<sup><a reveal-id="DSW013C15  DSW013C17" data-open="DSW013C15  DSW013C17" class="link link-ref link-reveal xref-bibr">15–17</a></sup></p><p>The CNV distribution within and among species seems to be shaped by mutation, selection, and demographic history.<sup><a reveal-id="DSW013C18" data-open="DSW013C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSW013C19" data-open="DSW013C19" class="link link-ref link-reveal xref-bibr">19</a></sup> However, unlike SNPs and microsatellites, the population genetics of CNV is largely unknown.<sup><a reveal-id="DSW013C1" data-open="DSW013C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSW013C20  DSW013C23" data-open="DSW013C20  DSW013C23" class="link link-ref link-reveal xref-bibr">20–23</a></sup> Several studies have explored the evolution and adaptation aspects of CNVs in human,<sup><a reveal-id="DSW013C24" data-open="DSW013C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSW013C25" data-open="DSW013C25" class="link link-ref link-reveal xref-bibr">25</a></sup> and other species.<sup><a reveal-id="DSW013C26  DSW013C29" data-open="DSW013C26  DSW013C29" class="link link-ref link-reveal xref-bibr">26–29</a></sup> Only a few cases of recent positive selection were found near <em>AMY1</em>, <em>APOBEC</em>, <em>MAPT</em>, <em>MIF</em>, and <em>UGT2B17</em> within human populations.<sup><a reveal-id="DSW013C30  DSW013C33" data-open="DSW013C30  DSW013C33" class="link link-ref link-reveal xref-bibr">30–33</a></sup> While most of simple deletions and simple duplications (75–90%) display extensive linkage disequilibrium (LD) with SNPs,<sup><a reveal-id="DSW013C1" data-open="DSW013C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSW013C34" data-open="DSW013C34" class="link link-ref link-reveal xref-bibr">34</a>,<a reveal-id="DSW013C35" data-open="DSW013C35" class="link link-ref link-reveal xref-bibr">35</a></sup> the properties of CNVs not tagged by SNPs remain unexplored in cattle. This is mainly due to the difficulty of genotyping CNV and to the limited sample size in the published CNV studies.</p><p>As one of the most important farm animals, cattle are used for a variety of purposes including dairy, beef, leather, and labour. The majority of the global cattle population can be classified into one of two subspecies: humpless (taurine, <em>Bos taurus taurus</em>) and humped (indicine or zebu, <em>Bos taurus indicus</em>) cattle with dramatic phenotypic differences between them.<sup><a reveal-id="DSW013C36" data-open="DSW013C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DSW013C37" data-open="DSW013C37" class="link link-ref link-reveal xref-bibr">37</a></sup> Earlier studies indicated that these two subspecies diverged from the last common ancestor between 0.6 and 2 million yrs ago.<sup><a reveal-id="DSW013C38" data-open="DSW013C38" class="link link-ref link-reveal xref-bibr">38</a>,<a reveal-id="DSW013C39" data-open="DSW013C39" class="link link-ref link-reveal xref-bibr">39</a></sup> There appears to have been two separate domestication events, with taurine cattle likely being domesticated in the Fertile Crescent ∼8,000–10,000 yrs ago and indicine cattle in the Indus Valley ∼6,000–8,000 yrs ago.<sup><a reveal-id="DSW013C40" data-open="DSW013C40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="DSW013C41" data-open="DSW013C41" class="link link-ref link-reveal xref-bibr">41</a></sup> A third independent domestication was proposed in Africa;<sup><a reveal-id="DSW013C42" data-open="DSW013C42" class="link link-ref link-reveal xref-bibr">42</a>,<a reveal-id="DSW013C43" data-open="DSW013C43" class="link link-ref link-reveal xref-bibr">43</a></sup> however, a recent study did not support this hypothesis.<sup><a reveal-id="DSW013C44" data-open="DSW013C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Since the early 1800s, breed development has primarily been based on phenotypic selection on coat colour and polled phenotypes. More recently, the adoption of effective genetic selection programs and widespread use of artificial insemination resulted in bottlenecks followed by breed expansion. During the last 50 yrs, animal breeding based on quantitative genetics has resulted in remarkable progress in improving milk and meat production traits.<sup><a reveal-id="DSW013C45" data-open="DSW013C45" class="link link-ref link-reveal xref-bibr">45</a>,<a reveal-id="DSW013C46" data-open="DSW013C46" class="link link-ref link-reveal xref-bibr">46</a></sup> Therefore, selective (natural and human-imposed) and non-selective forces (demographic events and introgression) have driven changes within the cattle genome. Their combined effects have created exceptional phenotypic diversity and genetic adaptation to local environments across the globe within the modern cattle breeds. For example, indicine cattle are better adapted to warm climates and demonstrate superior resistance to tick infestation than do taurine breeds.<sup><a reveal-id="DSW013C47" data-open="DSW013C47" class="link link-ref link-reveal xref-bibr">47</a></sup> Likewise, beef and dairy cattle breeds display distinct patterns in selected metabolic pathways related to muscling, marbling, and milk composition traits. Although cattle genome evolution and demographic history have been explored from multiple aspects, the diversity and population genetic properties of CNV in cattle are still unexplored.</p><p>In this study, we compare the diversity and population-genetic properties of CNVs in ∼70 cattle individuals sequenced to medium coverage (mean ∼11.8×). The data set includes multiple individuals from eight representative cattle breeds, representing both the major taurine and indicine breeds used for both dairy and beef purposes. It provides unprecedented genome-wide resolution to interrogate CNV and a unique opportunity to fully explore the population-genetic properties and evolutionary contributions of multicopy genes related to breed-specific traits.</p>                            <h2 scrollto-destination=87199593 id="87199593" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87199594 id="87199594" class="section-title" data-legacy-id=s2a>2.1. CNV calling, distribution, and association with other genomic features</h3>
<p>We used a previously described segmentation algorithm to call CNVs.<sup><a reveal-id="DSW013C16" data-open="DSW013C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSW013C48" data-open="DSW013C48" class="link link-ref link-reveal xref-bibr">48</a></sup> Detailed sample selection and CNV calling methods can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material online</a></span>. Association between CNVs and SDs was tested by Spearman's rank correlation using 100 kb windows as previously described.<sup><a reveal-id="DSW013C49" data-open="DSW013C49" class="link link-ref link-reveal xref-bibr">49</a></sup> Additional genomic features were obtained from public databases. Determination of the overlap between CNVRs and genomic features was performed as previously described.<sup><a reveal-id="DSW013C10" data-open="DSW013C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p>                            <h3 scrollto-destination=87199598 id="87199598" class="section-title" data-legacy-id=s2b>2.2. Population-genetic analyses and heatmap analysis</h3>
<p>Inbreeding is a common feature in livestock due to selective mating and widespread use of artificial insemination. We filtered our samples based on known pedigrees constraining Wright's coefficient of relationship (<em>r</em>) to &lt;0.25 to identify 69 unrelated individuals for the population-genetic analyses.</p><p>During the CNV discovery phase, a total of 1,148,528 windows of 1 kb were identified across the whole genome. Population-specific CNVs were estimated using the statistic <em>V</em><sub>ST</sub> developed by Redon et al.<sup><a reveal-id="DSW013C21" data-open="DSW013C21" class="link link-ref link-reveal xref-bibr">21</a></sup>. <em>V</em><sub>ST</sub> is calculated by considering (<em>V</em><sub>T</sub> − <em>V</em><sub>S</sub>)/<em>V</em><sub>T</sub>, where <em>V</em><sub>T</sub> is the variance in normalized copy numbers among all unrelated individuals and <em>V</em><sub>S</sub> is the average variance within each population, weighted for population size.</p><p>We next selected the top 1% diverse 1 kb windows (<em>n</em> = 80) from the distinct CNV regions to perform CNV genotyping using the partitioning around medoids (PAM) function in R.<sup><a reveal-id="DSW013C50" data-open="DSW013C50" class="link link-ref link-reveal xref-bibr">50</a></sup> A PAM procedure was used to cluster copy number ratios into discrete CN genotypes.<sup><a reveal-id="DSW013C21" data-open="DSW013C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Similarly, we partitioned the copy numbers of each 1 kb window into three clusters, representing the low, mid, and high ranges and then coded them using the 0, 1, and 2 matrix for SNP genotyping. Population clustering was then performed using STRUCTURE v2.3.3,<sup><a reveal-id="DSW013C51" data-open="DSW013C51" class="link link-ref link-reveal xref-bibr">51</a>,<a reveal-id="DSW013C52" data-open="DSW013C52" class="link link-ref link-reveal xref-bibr">52</a></sup> assuming three ancestral populations (<em>k</em> = 3). This analysis between taurine and indicine cattle was initially run for values of the number of clusters (<em>k</em>) between 2 and 8. Each analysis was performed using 100,000 replicates and 100,000 burn-in cycles under admixture and correlated allele frequencies models. Reynolds' genetic distances among breeds were calculated using PHYLIP 3.69. To provide statistical support for the resulting clades, 10,000 bootstrap simulations were performed. The phylogenetic trees were visualized in FigTree 1.3.1 (<a class="link link-uri" href="http://tree.bio.ed.ac.uk/software/figtree/">http://tree.bio.ed.ac.uk/software/figtree/</a>). Multidimensional scaling (MDS) analysis was conducted in PLINK 1.07 based on either CNV genotypes generated in this study or SNP genotypes generated using the same bovine HapMap populations as described previously.<sup><a reveal-id="DSW013C53" data-open="DSW013C53" class="link link-ref link-reveal xref-bibr">53</a></sup></p><p>NimbleGen array CGH log<sub>2</sub> ratios were calculated for each probe on a custom 2.1 million probe array for all animals. The reference animal, in all cases, was the Hereford cow used to generate the cattle reference assembly, L1 Dominette. All log<sub>2</sub> ratio values that spanned the <em>GAT</em>, <em>GLYAT,</em> and <em>KRTAP9-1</em> genes were averaged across the gene's length for each animal individually. <em>V</em><sub>ST</sub> values contrasting taurine and indicine populations were calculated for these log<sub>2</sub> ratios as previously described.<sup><a reveal-id="DSW013C21" data-open="DSW013C21" class="link link-ref link-reveal xref-bibr">21</a></sup></p><p>Heatmaps were generated using the estimated CN windows for each animal as described previously.<sup><a reveal-id="DSW013C16" data-open="DSW013C16" class="link link-ref link-reveal xref-bibr">16</a></sup> The gplots (v 2.14.2) R package (<a class="link link-uri" href="http://cran.r-project.org/web/packages/gplots/index.html">http://cran.r-project.org/web/packages/gplots/index.html</a>) was used to graph the CN values and generate heatmap representation of all lineage-specific gene duplications, deletions, and expansions identified in cattle breeds.</p>                            <h3 scrollto-destination=87199610 id="87199610" class="section-title" data-legacy-id=s2c>2.3. Gene analyses</h3>
<p>Gene content of cattle CNVRs was assessed as previously described.<sup><a reveal-id="DSW013C10" data-open="DSW013C10" class="link link-ref link-reveal xref-bibr">10</a></sup> We performed DAVID analysis to test whether the terms were under- or overrepresented in CNVRs after Bonferroni correction.<sup><a reveal-id="DSW013C10" data-open="DSW013C10" class="link link-ref link-reveal xref-bibr">10</a></sup> We identified the lineage-specific or lineage-differential gene families using a heuristic approach with our 75 analysed individual animals. We divided the animals into breed, subspecies, and purpose groups as listed in Table <a reveal-id="DSW013TB1" data-open="DSW013TB1" class="link link-reveal link-table xref-fig">1</a> and used a weighted search algorithm to highlight CNVRs with a high tendency to exist solely within a specific group. The weighted search was accomplished as follows: for each CNVR we calculated a sum score that represented a hypothesis that the CNVR was unique to a specific breed/subspecies based on the animals that shared the CNVR. For each breed/subspecies/purpose group (G), we counted the number of animals (A) from G that shared the CNVR and imposed a penalty (P) for each animal that was not a member of the current G. The sets of G that were tested consisted of dairy, beef, Angus, Holstein, Limousin, Jersey, Romagnola, Nelore, Gir, Brahman, Taurus, and Indicus (membership was not mutually exclusive within the groups). The summed weight of A–P was calculated for each G, and if it exceeded a threshold of 3, it was selected as a putative subspecies/breed-specific or differential CNVR. We also employed a statistical method (<em>V</em><sub>ST</sub>) to identify copy number variable genes within our data set. Based on the gene CNs from each animal, we identified gene families that were stratified by subspecies differences using the statistic <em>V</em><sub>ST</sub>, as described above. </p>                        <a id="87199612" scrollto-destination="87199612"></a>
<div content-id="DSW013TB1" class="table-modal"><div class="table-wrap"><div id="DSW013TB1" class="table-wrap-title" data-id="DSW013TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Samples and sequence data sets</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Breed </th><th>Subspecies </th><th>Purpose </th><th>Animal count </th><th>Coverage range </th><th>CNV count </th><th>Average CNVs per animal<sup>a</sup> </th></tr></thead><tbody><tr><td>Brahman (BRM) </td><td><em>Bos t. indicus</em> </td><td>Beef </td><td>7 </td><td>5–9× </td><td>3,836 </td><td>548 (86) </td></tr><tr><td>Gir (GIR) </td><td><em>Bos t. indicus</em> </td><td>Beef/dairy </td><td>6 </td><td>5–14× </td><td>3,724 </td><td>621 (30) </td></tr><tr><td>Nelore (NEL) </td><td><em>Bos t. indicus</em> </td><td>Beef </td><td>8 </td><td>6–20× </td><td>4,855 </td><td>607 (38) </td></tr><tr><td>Angus (ANG) </td><td><em>Bos t. taurus</em> </td><td>Beef </td><td>16 </td><td>5–30× </td><td>11,657 </td><td>729 (52) </td></tr><tr><td>Holstein (HOL) </td><td><em>Bos t. taurus</em> </td><td>Dairy </td><td>22 </td><td>4–20× </td><td>12,430 </td><td>565 (80) </td></tr><tr><td>Jersey (JER) </td><td><em>Bos t. taurus</em> </td><td>Dairy </td><td>6 </td><td>4–13× </td><td>3,487 </td><td>581 (46) </td></tr><tr><td>Limousin (LIM) </td><td><em>Bos t. taurus</em> </td><td>Beef </td><td>6 </td><td>5–10× </td><td>3,650 </td><td>608 (48) </td></tr><tr><td>Romagnola (ROM) </td><td><em>Bos t. taurus</em> </td><td>Beef/draft </td><td>4 </td><td>6–10× </td><td>2,708 </td><td>677 (20) </td></tr></tbody></table></div><div content-id="DSW013TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Breed </th><th>Subspecies </th><th>Purpose </th><th>Animal count </th><th>Coverage range </th><th>CNV count </th><th>Average CNVs per animal<sup>a</sup> </th></tr></thead><tbody><tr><td>Brahman (BRM) </td><td><em>Bos t. indicus</em> </td><td>Beef </td><td>7 </td><td>5–9× </td><td>3,836 </td><td>548 (86) </td></tr><tr><td>Gir (GIR) </td><td><em>Bos t. indicus</em> </td><td>Beef/dairy </td><td>6 </td><td>5–14× </td><td>3,724 </td><td>621 (30) </td></tr><tr><td>Nelore (NEL) </td><td><em>Bos t. indicus</em> </td><td>Beef </td><td>8 </td><td>6–20× </td><td>4,855 </td><td>607 (38) </td></tr><tr><td>Angus (ANG) </td><td><em>Bos t. taurus</em> </td><td>Beef </td><td>16 </td><td>5–30× </td><td>11,657 </td><td>729 (52) </td></tr><tr><td>Holstein (HOL) </td><td><em>Bos t. taurus</em> </td><td>Dairy </td><td>22 </td><td>4–20× </td><td>12,430 </td><td>565 (80) </td></tr><tr><td>Jersey (JER) </td><td><em>Bos t. taurus</em> </td><td>Dairy </td><td>6 </td><td>4–13× </td><td>3,487 </td><td>581 (46) </td></tr><tr><td>Limousin (LIM) </td><td><em>Bos t. taurus</em> </td><td>Beef </td><td>6 </td><td>5–10× </td><td>3,650 </td><td>608 (48) </td></tr><tr><td>Romagnola (ROM) </td><td><em>Bos t. taurus</em> </td><td>Beef/draft </td><td>4 </td><td>6–10× </td><td>2,708 </td><td>677 (20) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>Numbers in parentheses indicate 1 SD.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199612" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSW013TB1" class="table-wrap-title" data-id="DSW013TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Samples and sequence data sets</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Breed </th><th>Subspecies </th><th>Purpose </th><th>Animal count </th><th>Coverage range </th><th>CNV count </th><th>Average CNVs per animal<sup>a</sup> </th></tr></thead><tbody><tr><td>Brahman (BRM) </td><td><em>Bos t. indicus</em> </td><td>Beef </td><td>7 </td><td>5–9× </td><td>3,836 </td><td>548 (86) </td></tr><tr><td>Gir (GIR) </td><td><em>Bos t. indicus</em> </td><td>Beef/dairy </td><td>6 </td><td>5–14× </td><td>3,724 </td><td>621 (30) </td></tr><tr><td>Nelore (NEL) </td><td><em>Bos t. indicus</em> </td><td>Beef </td><td>8 </td><td>6–20× </td><td>4,855 </td><td>607 (38) </td></tr><tr><td>Angus (ANG) </td><td><em>Bos t. taurus</em> </td><td>Beef </td><td>16 </td><td>5–30× </td><td>11,657 </td><td>729 (52) </td></tr><tr><td>Holstein (HOL) </td><td><em>Bos t. taurus</em> </td><td>Dairy </td><td>22 </td><td>4–20× </td><td>12,430 </td><td>565 (80) </td></tr><tr><td>Jersey (JER) </td><td><em>Bos t. taurus</em> </td><td>Dairy </td><td>6 </td><td>4–13× </td><td>3,487 </td><td>581 (46) </td></tr><tr><td>Limousin (LIM) </td><td><em>Bos t. taurus</em> </td><td>Beef </td><td>6 </td><td>5–10× </td><td>3,650 </td><td>608 (48) </td></tr><tr><td>Romagnola (ROM) </td><td><em>Bos t. taurus</em> </td><td>Beef/draft </td><td>4 </td><td>6–10× </td><td>2,708 </td><td>677 (20) </td></tr></tbody></table></div><div content-id="DSW013TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Breed </th><th>Subspecies </th><th>Purpose </th><th>Animal count </th><th>Coverage range </th><th>CNV count </th><th>Average CNVs per animal<sup>a</sup> </th></tr></thead><tbody><tr><td>Brahman (BRM) </td><td><em>Bos t. indicus</em> </td><td>Beef </td><td>7 </td><td>5–9× </td><td>3,836 </td><td>548 (86) </td></tr><tr><td>Gir (GIR) </td><td><em>Bos t. indicus</em> </td><td>Beef/dairy </td><td>6 </td><td>5–14× </td><td>3,724 </td><td>621 (30) </td></tr><tr><td>Nelore (NEL) </td><td><em>Bos t. indicus</em> </td><td>Beef </td><td>8 </td><td>6–20× </td><td>4,855 </td><td>607 (38) </td></tr><tr><td>Angus (ANG) </td><td><em>Bos t. taurus</em> </td><td>Beef </td><td>16 </td><td>5–30× </td><td>11,657 </td><td>729 (52) </td></tr><tr><td>Holstein (HOL) </td><td><em>Bos t. taurus</em> </td><td>Dairy </td><td>22 </td><td>4–20× </td><td>12,430 </td><td>565 (80) </td></tr><tr><td>Jersey (JER) </td><td><em>Bos t. taurus</em> </td><td>Dairy </td><td>6 </td><td>4–13× </td><td>3,487 </td><td>581 (46) </td></tr><tr><td>Limousin (LIM) </td><td><em>Bos t. taurus</em> </td><td>Beef </td><td>6 </td><td>5–10× </td><td>3,650 </td><td>608 (48) </td></tr><tr><td>Romagnola (ROM) </td><td><em>Bos t. taurus</em> </td><td>Beef/draft </td><td>4 </td><td>6–10× </td><td>2,708 </td><td>677 (20) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>Numbers in parentheses indicate 1 SD.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199612" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87199614 id="87199614" class="section-title" data-legacy-id=s2d>2.4. Haplotype network analysis</h3>
<p>To explore the diversity of haplotypes and evolutionary relationships across populations, we retrieved the high-density SNP array data for these eight breeds generated by the Illumina BovineHD SNP Consortium as described previously.<sup><a reveal-id="DSW013C53" data-open="DSW013C53" class="link link-ref link-reveal xref-bibr">53</a></sup> Haplotypes and their frequencies were estimated separately for each breed using PHASE 2.1.<sup><a reveal-id="DSW013C54" data-open="DSW013C54" class="link link-ref link-reveal xref-bibr">54</a>,<a reveal-id="DSW013C55" data-open="DSW013C55" class="link link-ref link-reveal xref-bibr">55</a></sup> To obtain reliable results, we employed an iterative scheme to perform inference with 10,000 iterations and 10,000 burn-ins, also we increased the number of iterations of the final run of the algorithm using option -X100, for details see <a class="link link-uri" href="http://stephenslab.uchicago.edu/instruct2.1.pdf">http://stephenslab.uchicago.edu/instruct2.1.pdf</a>. Haplotype networks were constructed near functional genes such as <em>GAT</em>/<em>GLYAT</em>, <em>ASZ1</em>, <em>AOX1,</em> and <em>FZD3</em>. Phylogenetic relationships among the identified haplotypes were inferred through a median-joining network analysis using Network 4.6.12 (<a class="link link-uri" href="http://www.fluxus-engineering.com/">http://www.fluxus-engineering.com/</a>).</p>                            <h3 scrollto-destination=87199618 id="87199618" class="section-title" data-legacy-id=s2e>2.5. Data release</h3>
<p>All array CGH data have been submitted to the gene expression omnibus (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/geo/">http://www.ncbi.nlm.nih.gov/geo/</a>) under the accession number GSE62990. Raw data and population genetic and evolutionary analysis results are available upon request for research purposes. Raw reads were deposited under the SRA Bioproject PRJNA277147 in SRA (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/sra/">http://www.ncbi.nlm.nih.gov/sra/</a>).</p>                            <h2 scrollto-destination=87199622 id="87199622" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=87199625 id="87199625" class="section-title" data-legacy-id=s3a>3.1. CNV discovery and experimental validations</h3>
<p>After carefully excluding samples with low coverage in our initial survey, we focused on the remaining 75 individuals in our final data set (see Table <a reveal-id="DSW013TB1" data-open="DSW013TB1" class="link link-reveal link-table xref-fig">1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> for sample information and sequence coverage). We identified CNVs using a sliding window approach based on the previously published MrsFAST-WSSD method.<sup><a reveal-id="DSW013C16" data-open="DSW013C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSW013C48" data-open="DSW013C48" class="link link-ref link-reveal xref-bibr">48</a></sup> We discovered comparable average numbers of CNVs per individual across taurine (626.7) and indicine (591.2) cattle, suggesting our results based on the taurine reference assembly (UMD3.1) were not particularly biased against the indicine cattle. A full list of CNV calls (47,511) is presented in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>. After merged across samples, these CNVs yielded 1,853 CNV regions (CNVR), which represent 87.5 Mb (3.1%) of the cattle genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). We then calculated absolute copy number values for 1 kb windows across the genome for each sequenced individual (see Materials and methods). As anticipated, the average normalized genome-wide copy number was 2.15 ± 0.1 for all copy number windows. We successfully performed 75 quantitative PCR (qPCR, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>) and 25 array CGH experiments (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>) to assess the false-positive discovery rate for our data set as previously described.<sup><a reveal-id="DSW013C3" data-open="DSW013C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSW013C53" data-open="DSW013C53" class="link link-ref link-reveal xref-bibr">53</a></sup> Detailed experimental validations can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material online</a></span>. In summary, CNV calls made with sequence data were strongly correlated with array CGH data (<em>r</em><sup>2</sup> = 0.761) and had an estimated 12% false-positive rate and a 19% false-negative rate based upon qPCR and array CGH, respectively.</p><p>We found that a large proportion of identified CNVRs (43.3%; 49.5 Mb) overlapped with the segmental duplication (SD) regions. We estimated pair-wise Spearman's rank correlations (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>) of 0.084 and 0.098 for indicine and taurine CNVs and SD regions (both <span class="inline-formula no-formula-id"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">P</mi><mo xmlns="">&lt;</mo><mn xmlns="">0.001</mn></math>⁠</span>), which were similar to the previously reported human results.<sup><a reveal-id="DSW013C49" data-open="DSW013C49" class="link link-ref link-reveal xref-bibr">49</a></sup> A strong correlation of CNVs and SDs in cattle confirms that their formation mechanisms are mainly due to non-allelic homologous recombination (NAHR).<sup><a reveal-id="DSW013C10" data-open="DSW013C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSW013C56" data-open="DSW013C56" class="link link-ref link-reveal xref-bibr">56</a></sup> In the following analyses, we mainly focused on the characterization of the high-confidence CNVs from autosomes.</p>                            <h3 scrollto-destination=87199630 id="87199630" class="section-title" data-legacy-id=s3b>3.2. Population genetics of cattle CNVs</h3>
<p>To investigate the population genetics of CNVs, we first identified the frequencies of CNVRs in our data set (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). The average CNVR had a frequency of 29.3% (22 animals out of 75 total); however, the CNVR frequency tended towards a parabolic distribution with 835 CNVRs having a frequency ≤5% and 189 CNVRs having a frequency≥95% in our data set (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). As expected, rare events were often observed in only one subspecies/breed, whereas common CNVs (frequency &gt;5%) were usually shared across subspecies/breeds.</p><p>To explore the population differentiation of CNVs between taurine and indicine cattle, we applied statistical measures of population differentiation using <em>V</em><sub>ST</sub><sup><a reveal-id="DSW013C21" data-open="DSW013C21" class="link link-ref link-reveal xref-bibr">21</a></sup> to our dataset in three ways: (i) estimation of <em>V</em><sub>ST</sub> for genome-wide 1 kb CN windows; (ii) clustering of the top 1% of <em>V</em><sub>ST</sub> values for 1 kb CN windows; and (iii) estimation of <em>V</em><sub>ST</sub> using the average CN for annotated genes. Estimates of <em>V</em><sub>ST</sub> for all genome-wide CN windows and all CNVRs revealed a number of outliers with levels of population differentiation suggestive of population-specific selective pressures (Fig. <a reveal-id="DSW013F1" data-open="DSW013F1" class="link link-reveal link-table xref-fig">1</a>). Among these outliers were CNVs near <em>AOX1</em>, <em>GAT/GLYAT</em>, <em>ASZ1</em>, <em>KRTAP9-1</em>, and <em>MCM4</em> (Fig. <a reveal-id="DSW013F1" data-open="DSW013F1" class="link link-reveal link-table xref-fig">1</a>A). </p>                        <a id="87199636" scrollto-destination="87199636"></a>
<div id="DSW013F1" data-id="DSW013F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/m_dsw01301.png?Expires=2147483647&Signature=NrzwGLju0BDls~DDp-tzdVEIClc1R2-vgvHWQszU4TW-K5oWA12WI~SISz--giXZ7OCpI6qkaLWWn0L4weGpPcNVgnFjtGChYqaAYuXD7RavmaUSz95fRfmBdhHwTbGppAXLYFz~qVFlfwRU~OHP7yU8wcIM1WwbplUtuJ9jRXHQbn7MT0KvO1RXjDaK5oNHN1qKYCR6Dkme3j5w20NZpPRKo-WWvPQ76DVo~~2Dpiw7jUHmpSaDgKynPY1QrkSdOOBXMT6J4V4JILnUuuShErprkrpjI71x1LAf1xsqxVvm~AveyH8wK0VnnQp90t01arnIaSmTSNVxOVB9Pj7vSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Population differentiation for copy number variation. Population differentiation, estimated by VST, is plotted along each chromosome for the two taurine and indicine comparisons: (A) RefSeq genes and (B) genome-wide 1 kb windows. Example CNVs exhibiting high population differentiation are labelled. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01301.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199636/dsw01301.tiff" path-from-xml="dsw01301.tiff" target="_blank">View large</a><a section="87199636" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw01301.png?Expires=2147483647&Signature=hryvaBf5K4fhzoAHs-Fhp4oPuFFWQCUHvodTTYoenPfbb45F~fyJOWyweWx9TJrpLGSpt8l5qU1ZGx374ZvCaUMxbuJaG54KEi5l~BdQQFHZenP5PYVRc0NP5OKY8dkxtFbuE~o4T4xMG5KFjXV5jKP5CNgmgaPsUvK-cLjLgJd2QKFzOUkh9NFy4Rwf4YRMlWe14u5pQcpCe-urifdCZxdAKx1zAz3867b6Hc4CvH65MhbZ4kDO3HSZPP7W88pU7EG-ShWgGKwCnxhLwKL8PDtaSPeAu2iETPDmQxDgD7rdnSlOVYmQmFXqBybDmHre~oXzFOk7u2-QEydHsaOEPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199636&ar=1745562&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01301.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Population differentiation for copy number variation. Population differentiation, estimated by <em>V<sub>ST</sub></em>, is plotted along each chromosome for the two taurine and indicine comparisons: (A) RefSeq genes and (B) genome-wide 1 kb windows. Example CNVs exhibiting high population differentiation are labelled. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW013F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/m_dsw01301.png?Expires=2147483647&Signature=NrzwGLju0BDls~DDp-tzdVEIClc1R2-vgvHWQszU4TW-K5oWA12WI~SISz--giXZ7OCpI6qkaLWWn0L4weGpPcNVgnFjtGChYqaAYuXD7RavmaUSz95fRfmBdhHwTbGppAXLYFz~qVFlfwRU~OHP7yU8wcIM1WwbplUtuJ9jRXHQbn7MT0KvO1RXjDaK5oNHN1qKYCR6Dkme3j5w20NZpPRKo-WWvPQ76DVo~~2Dpiw7jUHmpSaDgKynPY1QrkSdOOBXMT6J4V4JILnUuuShErprkrpjI71x1LAf1xsqxVvm~AveyH8wK0VnnQp90t01arnIaSmTSNVxOVB9Pj7vSA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Population differentiation for copy number variation. Population differentiation, estimated by VST, is plotted along each chromosome for the two taurine and indicine comparisons: (A) RefSeq genes and (B) genome-wide 1 kb windows. Example CNVs exhibiting high population differentiation are labelled. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01301.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199636/dsw01301.tiff" path-from-xml="dsw01301.tiff" target="_blank">View large</a><a section="87199636" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw01301.png?Expires=2147483647&Signature=hryvaBf5K4fhzoAHs-Fhp4oPuFFWQCUHvodTTYoenPfbb45F~fyJOWyweWx9TJrpLGSpt8l5qU1ZGx374ZvCaUMxbuJaG54KEi5l~BdQQFHZenP5PYVRc0NP5OKY8dkxtFbuE~o4T4xMG5KFjXV5jKP5CNgmgaPsUvK-cLjLgJd2QKFzOUkh9NFy4Rwf4YRMlWe14u5pQcpCe-urifdCZxdAKx1zAz3867b6Hc4CvH65MhbZ4kDO3HSZPP7W88pU7EG-ShWgGKwCnxhLwKL8PDtaSPeAu2iETPDmQxDgD7rdnSlOVYmQmFXqBybDmHre~oXzFOk7u2-QEydHsaOEPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199636&ar=1745562&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01301.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Population differentiation for copy number variation. Population differentiation, estimated by <em>V<sub>ST</sub></em>, is plotted along each chromosome for the two taurine and indicine comparisons: (A) RefSeq genes and (B) genome-wide 1 kb windows. Example CNVs exhibiting high population differentiation are labelled. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>We next selected the top 1% diverse 1 kb windows (<em>n</em> = 80)—identified by our <em>V</em><sub>ST</sub> calculations—from the distinct outlier CNVRs to perform CNV genotyping. The PAM algorithm is the most common implementation of <em>k</em>-medoid clustering, which is related to the <em>k</em>-means algorithm and the medoid shift algorithm.<sup><a reveal-id="DSW013C50" data-open="DSW013C50" class="link link-ref link-reveal xref-bibr">50</a></sup> Previously, a PAM algorithm was used to cluster copy number derived from array CGH log<sub>2</sub> ratios into discrete CN genotypes.<sup><a reveal-id="DSW013C21" data-open="DSW013C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Similarly, we partitioned the copy numbers of each 1 kb window into three discrete value clusters, representing low, mid, and high ranges and then coded them as 0, 1, and 2 values within a matrix for subsequent genotyping (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>).</p><p>Using CNV genotype calls from the top 1% diverse 1 kb windows within CNVRs, we performed population clustering (Fig. <a reveal-id="DSW013F2" data-open="DSW013F2" class="link link-reveal link-table xref-fig">2</a>). The proximity of an individual to each apex of the triangle indicates the proportion of that individual's genome that is estimated to have ancestry from each of the three inferred ancestral populations. The clustering of most indicine cattle (BRM, GIR, NEL) in the right bottom apex reveals the clear discrimination between indicine and taurine cattle. In contrast, the taurine cattle were scattered along the opposing side with the exception of ROM near the centre. This distribution of ROM individuals agreed with previous results based on SNP genotypes,<sup><a reveal-id="DSW013C44" data-open="DSW013C44" class="link link-ref link-reveal xref-bibr">44</a></sup> confirming that ROM has both taurine and indicine ancestries. It is also noted that ANG individuals were clustered together in the upper apex, while the other taurine cattle (HOL, LMS, JER) were dispersed around the left bottom corner, suggesting a distinction between different taurine breeds. It is possible that these two clusters differentiate between continental European breeds of cattle from UK breeds or beef breeds from dairy breeds; however, it is also possible that our selected CNV markers may be subject to cryptic founder effects within our ANG individuals. We suspect that the addition of African cattle breeds to this dataset will better resolve the taurine cluster by providing a third distinct lineage. Still, we note that the clustering results were structurally similar to our results obtained with high-density SNP data derived from the same bovine HapMap samples (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>) and other recently published results.<sup><a reveal-id="DSW013C44" data-open="DSW013C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Based on the genotypes for these 80 loci (i.e. the top 1 kb windows) and using a neighbor-joining algorithm, we obtained a phylogenetic tree that generally agrees with the known cattle breed history (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). We also performed a MDS analysis based on CNV genotypes and compared it with the plot based on SNPs. Our plot confidently separated the indicine from the taurine cattle (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>); however, the separation and clustering of the taurine cattle using CNVs were not superior to those based on SNPs, suggesting that CNV genotyping still has room for improvement. </p>                        <a id="87199643" scrollto-destination="87199643"></a>
<div id="DSW013F2" data-id="DSW013F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/m_dsw01302.png?Expires=2147483647&Signature=d4ZeCsIfqS9XsbA5DseoJmWFclrKrZe-kzjIXp07RSJWXU2pJFKdlJqi2-7VC73oxRAd7ccmsMFJrSAUoSz4FA2iWbq2buNvF4hDHYKDFSt62IgSA6wcBLQ5yiHKTHjYvTWeW3FKOAVuZRVogE9n-4k5A1O64RgZHvX8JgFZg1OrSDIH-yY76eeUJwXVYrSSpBwgZ7Gc3ja-l8I3m4zsKiDSuRcMeY8zQJIBdGqoYPfbdtnsgGw3KEZ6QY3YJkZVaZ9Myi2V0jYvGOVrBrJnMyrD~V320wOrL2LUFKZ7UV4GKf9W6~ti9o2s8BLigQ4hrazu0uM3wOU9GWtI-m4AeQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Population clustering based on CNV genotypes. A triangle plot showing the clustering of 69 lowly related cattle individuals assuming three ancestral populations (k = 3). The proximity of an individual to each apex of the triangle indicates the proportion of that genome that is estimated to have ancestry in each of the three inferred ancestral populations. The clustering together of most indicine individuals (BRM, GIR, NEL) in the right bottom apex indicates the clear discrimination between indicine and taurine cattle. In contrast, taurine cattle are scattered along the opposing side with the exception of ROM in the centre. ANG individuals were clustered together in the upper apex, while the other taurine cattle (HOL, LMS, JER) were dispersed around the left bottom corner, suggesting a possible discrimination between beef and dairy cattle. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01302.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199643/dsw01302.tiff" path-from-xml="dsw01302.tiff" target="_blank">View large</a><a section="87199643" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw01302.png?Expires=2147483647&Signature=JXlqOlu4~PUXl6ryisj0tdnKG7hRfvNI~dzJtSpB5BqMAo5Ye~L2jQgYib6d0qo64bU2P~5nNzzPJMboXiMo8~YuTB8JSewOdZjLui3kSmNdDWtbzOWqVD41U2JepJkRjuNkZf1d6E-c6VeA9nyO7LVSGIsyRU8U9GP6fbCFuxEuRcE8bDwRMG4SLsP4pto5avp6Bs-kXyg7DOs6ElH85RJbZjWZ91WsH-E9DT-nOdlR1u3AwiV8gNLc8lOkrU1IZZVt9wMXJevDJ0HRQZ4yHRcOu3nuA5x~VTTc1pAVno70CPaTm6FH436Z3iNmaRWaxv0paB88pnVwey4ukN84ZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199643&ar=1745562&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01302.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Population clustering based on CNV genotypes. A triangle plot showing the clustering of 69 lowly related cattle individuals assuming three ancestral populations (<em>k</em> = 3). The proximity of an individual to each apex of the triangle indicates the proportion of that genome that is estimated to have ancestry in each of the three inferred ancestral populations. The clustering together of most indicine individuals (BRM, GIR, NEL) in the right bottom apex indicates the clear discrimination between indicine and taurine cattle. In contrast, taurine cattle are scattered along the opposing side with the exception of ROM in the centre. ANG individuals were clustered together in the upper apex, while the other taurine cattle (HOL, LMS, JER) were dispersed around the left bottom corner, suggesting a possible discrimination between beef and dairy cattle. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW013F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/m_dsw01302.png?Expires=2147483647&Signature=d4ZeCsIfqS9XsbA5DseoJmWFclrKrZe-kzjIXp07RSJWXU2pJFKdlJqi2-7VC73oxRAd7ccmsMFJrSAUoSz4FA2iWbq2buNvF4hDHYKDFSt62IgSA6wcBLQ5yiHKTHjYvTWeW3FKOAVuZRVogE9n-4k5A1O64RgZHvX8JgFZg1OrSDIH-yY76eeUJwXVYrSSpBwgZ7Gc3ja-l8I3m4zsKiDSuRcMeY8zQJIBdGqoYPfbdtnsgGw3KEZ6QY3YJkZVaZ9Myi2V0jYvGOVrBrJnMyrD~V320wOrL2LUFKZ7UV4GKf9W6~ti9o2s8BLigQ4hrazu0uM3wOU9GWtI-m4AeQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Population clustering based on CNV genotypes. A triangle plot showing the clustering of 69 lowly related cattle individuals assuming three ancestral populations (k = 3). The proximity of an individual to each apex of the triangle indicates the proportion of that genome that is estimated to have ancestry in each of the three inferred ancestral populations. The clustering together of most indicine individuals (BRM, GIR, NEL) in the right bottom apex indicates the clear discrimination between indicine and taurine cattle. In contrast, taurine cattle are scattered along the opposing side with the exception of ROM in the centre. ANG individuals were clustered together in the upper apex, while the other taurine cattle (HOL, LMS, JER) were dispersed around the left bottom corner, suggesting a possible discrimination between beef and dairy cattle. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01302.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199643/dsw01302.tiff" path-from-xml="dsw01302.tiff" target="_blank">View large</a><a section="87199643" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw01302.png?Expires=2147483647&Signature=JXlqOlu4~PUXl6ryisj0tdnKG7hRfvNI~dzJtSpB5BqMAo5Ye~L2jQgYib6d0qo64bU2P~5nNzzPJMboXiMo8~YuTB8JSewOdZjLui3kSmNdDWtbzOWqVD41U2JepJkRjuNkZf1d6E-c6VeA9nyO7LVSGIsyRU8U9GP6fbCFuxEuRcE8bDwRMG4SLsP4pto5avp6Bs-kXyg7DOs6ElH85RJbZjWZ91WsH-E9DT-nOdlR1u3AwiV8gNLc8lOkrU1IZZVt9wMXJevDJ0HRQZ4yHRcOu3nuA5x~VTTc1pAVno70CPaTm6FH436Z3iNmaRWaxv0paB88pnVwey4ukN84ZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199643&ar=1745562&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01302.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Population clustering based on CNV genotypes. A triangle plot showing the clustering of 69 lowly related cattle individuals assuming three ancestral populations (<em>k</em> = 3). The proximity of an individual to each apex of the triangle indicates the proportion of that genome that is estimated to have ancestry in each of the three inferred ancestral populations. The clustering together of most indicine individuals (BRM, GIR, NEL) in the right bottom apex indicates the clear discrimination between indicine and taurine cattle. In contrast, taurine cattle are scattered along the opposing side with the exception of ROM in the centre. ANG individuals were clustered together in the upper apex, while the other taurine cattle (HOL, LMS, JER) were dispersed around the left bottom corner, suggesting a possible discrimination between beef and dairy cattle. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>Out of these 80 loci, 62 can be reliably assessed for their variable patterns and 54 of these loci, in turn, (87.10%, 54/62) are located in or near tandem duplications (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). This estimate was consistent with our initial genome-wide results that 90% of SD in cattle are tandem duplications in contrast to human and other primates, which show a preponderance of interspersed duplications.<sup><a reveal-id="DSW013C56" data-open="DSW013C56" class="link link-ref link-reveal xref-bibr">56</a></sup> This led us to speculate that while it is challenging to systematically genotype cattle duplication CNV events as shown by Genome STRiP results in human,<sup><a reveal-id="DSW013C57" data-open="DSW013C57" class="link link-ref link-reveal xref-bibr">57</a></sup> our relatively high cattle CNV genotyping accuracy is likely due to the vast majority of cattle CNV being tandem duplicates. Large tandem repeats or duplication CNVs in cattle could behave similarly like human tandem macrosatellites and multicopy genes. For these tandem duplications, it is likely that we made reasonable approximations of CNV genotyping calls by simply clustering the normalized copy numbers, as shown traditionally for macrosatellites and microsatellites.<sup><a reveal-id="DSW013C58  DSW013C61" data-open="DSW013C58  DSW013C61" class="link link-ref link-reveal xref-bibr">58–61</a></sup> Additionally, the tandem distribution pattern could contribute to the high LD at CNV loci as suggested previously,<sup><a reveal-id="DSW013C62" data-open="DSW013C62" class="link link-ref link-reveal xref-bibr">62</a></sup> thus the majority of CNV genotype calls could better represent local alleles. Combining these two factors, it is not surprising that our CNV-based results generally agree with SNP-based results. Of course, this hypothesis certainly warrants more investigation using larger sample sizes and other mammals like mouse and dog to further validate and improve CNV genotyping approaches.</p><p>To provide an evolutionary perspective to our analyses, we also created heatmaps using the CN values for regions within selected gene loci (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>). These analyses of lineage-specific or lineage-differential CNVs separate subspecies/breeds into groupings that are consistent with the generally accepted cattle history.<sup><a reveal-id="DSW013C44" data-open="DSW013C44" class="link link-ref link-reveal xref-bibr">44</a></sup></p>                            <h3 scrollto-destination=87199646 id="87199646" class="section-title" data-legacy-id=s3c>3.3. Gene analyses</h3>
<p>We evaluated genes overlapped by cattle CNVs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>) and selected genes with known functions (Table <a reveal-id="DSW013TB2" data-open="DSW013TB2" class="link link-reveal link-table xref-fig">2</a>). We observed an enrichment of CNVs intersecting with genes (<em>P</em> &lt; 0.0001; Spearman's rank sum correlation), consistent with reduced evolutionary constraints acting on functionally redundant gene categories. We next used DAVID to identify basic biological functional categories for 361 genes overlapped by our identified CNVRs.<sup><a reveal-id="DSW013C63" data-open="DSW013C63" class="link link-ref link-reveal xref-bibr">63</a></sup> Like other mammals (human, mouse, and dog), statistically significant overrepresentations were observed for multiple categories including chromosome maintenance, immunity and cytoskeleton components (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). We then studied how variable genes were distributed across subspecies/breeds using either a heuristic approach based on CNV presence/absence or gene CN per individual (<em>V</em><sub>ST</sub>). </p>                        <a id="87199650" scrollto-destination="87199650"></a>
<div content-id="DSW013TB2" class="table-modal"><div class="table-wrap"><div id="DSW013TB2" class="table-wrap-title" data-id="DSW013TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Selected copy number variable genes identified from population sequence data</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene name </th><th>Function </th><th>Gene UMD3.1 coordinates </th><th><em>V</em><sub>ST</sub><sup>a</sup> </th><th>Identified<sup>b</sup> </th></tr></thead><tbody><tr><td><em>AOX1</em> </td><td>Detoxification </td><td>chr2:89517708-89589232 </td><td>0.5094 </td><td>Hou, Bickhart, and this study </td></tr><tr><td><em>ASZ1</em> </td><td>Spermatogenesis </td><td>chr4:51294534-51370343 </td><td>0.2109 </td><td>Only this study </td></tr><tr><td><em>CA1</em> </td><td>Carbonic anhydrase </td><td>chr14:79520632-79530892 </td><td>0.3270 </td><td>Hou, Bickhart, and this study </td></tr><tr><td><em>CFH</em> </td><td>Complement factor </td><td>chr16:5486704-6172566 </td><td>0.0483 </td><td>Hou, Bickhart, and this study </td></tr><tr><td><em>DDX21</em> </td><td>Translation initiation </td><td>chr28:25376358-25399769 </td><td>0.2375 </td><td>Only this study </td></tr><tr><td><em>DENR</em> </td><td>Translation initiation </td><td>chr29:7723699-7725004 </td><td>0.3285 </td><td>Only this study </td></tr><tr><td><em>FBXO16</em> </td><td>Ubiquitin protein ligase </td><td>chr8:10095869-10128675 </td><td>0.3558 </td><td>Bickhart and this study </td></tr><tr><td><em>FZD3</em> </td><td>Nervous system </td><td>chr8:10002971-10091175 </td><td>0.3334 </td><td>Bickhart and this study </td></tr><tr><td><em>GAL3ST1</em> </td><td>Glycolipid catalysis </td><td>chr17:71660016-71678806 </td><td>0.2435 </td><td>Hou and this study </td></tr><tr><td><em>GAT</em> </td><td>Detoxification </td><td>chr15:83472190-83493607 </td><td>0.4336 </td><td>Liu, Hou, Bickhart, and this study </td></tr><tr><td><em>GLYAT</em> </td><td>Detoxification </td><td>chr15:83455512-83469280 </td><td>0.4083 </td><td>Liu, Hou, Bickhart, and this study </td></tr><tr><td><em>GLYATL2</em> </td><td>Biological oxidation </td><td>chr15:83508339-83515102 </td><td>0.2257 </td><td>Bickhart and this study </td></tr><tr><td><em>KRTAP9-1</em> </td><td>Keratin family </td><td>chr19:42101853-42103421 </td><td>0.4578 </td><td>Bickhart and this study </td></tr><tr><td><em>LMBRD2</em> </td><td>Function unknown </td><td>chr20:38116509-38163145 </td><td>0.2573 </td><td>Only this study </td></tr><tr><td><em>PGR</em> </td><td>Progesterone receptor </td><td>chr15:8207682-8222806 </td><td>0.0103 </td><td>Only this study </td></tr><tr><td><em>PNPLA2</em> </td><td>Adipose tissue regulation </td><td>chr29:50742384-50747161 </td><td>0.0000 </td><td>Only this study </td></tr><tr><td><em>PRG3</em> </td><td>Carbohydrate binding </td><td>chr15:81920283-81926082 </td><td>0.2041 </td><td>Liu, Bickhart, and this study </td></tr><tr><td><em>RAET1G/ULBP17</em> </td><td>MHC class 1 related </td><td>chr9:88231932-88402262 </td><td>0.0000 </td><td>Liu, Hou, Bickhart, and this study </td></tr><tr><td><em>RICTOR</em> </td><td>Cell growth </td><td>chr20:35376523-35514753 </td><td>0.1048 </td><td>Only this study </td></tr><tr><td><em>SEC23A</em> </td><td>Vesicle transport </td><td>chr21:49489514-49555507 </td><td>0.3050 </td><td>Only this study </td></tr><tr><td><em>SERPINB4</em> </td><td>Protease inhibitor </td><td>chr24:62364701-62371668 </td><td>0.2418 </td><td>Liu, Hou, Bickhart, and this study </td></tr><tr><td><em>SUB1</em> </td><td>Transcriptional activation </td><td>chr20:41122022-41143914 </td><td>0.2282 </td><td>Only this study </td></tr><tr><td><em>TMED2</em> </td><td>Secretory vesicle transport </td><td>chr17:54330420-54338333 </td><td>0.0000 </td><td>Only this study </td></tr><tr><td><em>UFM1</em> </td><td>Ubiquitin </td><td>chr6:71051155-71053533 </td><td>0.5121 </td><td>Only this study </td></tr><tr><td><em>ZNF280B</em> </td><td>Negative regulation of p53 </td><td>chr17:51251538-51262528 </td><td>0.2658 </td><td>Liu, Hou, and this study </td></tr></tbody></table></div><div content-id="DSW013TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene name </th><th>Function </th><th>Gene UMD3.1 coordinates </th><th><em>V</em><sub>ST</sub><sup>a</sup> </th><th>Identified<sup>b</sup> </th></tr></thead><tbody><tr><td><em>AOX1</em> </td><td>Detoxification </td><td>chr2:89517708-89589232 </td><td>0.5094 </td><td>Hou, Bickhart, and this study </td></tr><tr><td><em>ASZ1</em> </td><td>Spermatogenesis </td><td>chr4:51294534-51370343 </td><td>0.2109 </td><td>Only this study </td></tr><tr><td><em>CA1</em> </td><td>Carbonic anhydrase </td><td>chr14:79520632-79530892 </td><td>0.3270 </td><td>Hou, Bickhart, and this study </td></tr><tr><td><em>CFH</em> </td><td>Complement factor </td><td>chr16:5486704-6172566 </td><td>0.0483 </td><td>Hou, Bickhart, and this study </td></tr><tr><td><em>DDX21</em> </td><td>Translation initiation </td><td>chr28:25376358-25399769 </td><td>0.2375 </td><td>Only this study </td></tr><tr><td><em>DENR</em> </td><td>Translation initiation </td><td>chr29:7723699-7725004 </td><td>0.3285 </td><td>Only this study </td></tr><tr><td><em>FBXO16</em> </td><td>Ubiquitin protein ligase </td><td>chr8:10095869-10128675 </td><td>0.3558 </td><td>Bickhart and this study </td></tr><tr><td><em>FZD3</em> </td><td>Nervous system </td><td>chr8:10002971-10091175 </td><td>0.3334 </td><td>Bickhart and this study </td></tr><tr><td><em>GAL3ST1</em> </td><td>Glycolipid catalysis </td><td>chr17:71660016-71678806 </td><td>0.2435 </td><td>Hou and this study </td></tr><tr><td><em>GAT</em> </td><td>Detoxification </td><td>chr15:83472190-83493607 </td><td>0.4336 </td><td>Liu, Hou, Bickhart, and this study </td></tr><tr><td><em>GLYAT</em> </td><td>Detoxification </td><td>chr15:83455512-83469280 </td><td>0.4083 </td><td>Liu, Hou, Bickhart, and this study </td></tr><tr><td><em>GLYATL2</em> </td><td>Biological oxidation </td><td>chr15:83508339-83515102 </td><td>0.2257 </td><td>Bickhart and this study </td></tr><tr><td><em>KRTAP9-1</em> </td><td>Keratin family </td><td>chr19:42101853-42103421 </td><td>0.4578 </td><td>Bickhart and this study </td></tr><tr><td><em>LMBRD2</em> </td><td>Function unknown </td><td>chr20:38116509-38163145 </td><td>0.2573 </td><td>Only this study </td></tr><tr><td><em>PGR</em> </td><td>Progesterone receptor </td><td>chr15:8207682-8222806 </td><td>0.0103 </td><td>Only this study </td></tr><tr><td><em>PNPLA2</em> </td><td>Adipose tissue regulation </td><td>chr29:50742384-50747161 </td><td>0.0000 </td><td>Only this study </td></tr><tr><td><em>PRG3</em> </td><td>Carbohydrate binding </td><td>chr15:81920283-81926082 </td><td>0.2041 </td><td>Liu, Bickhart, and this study </td></tr><tr><td><em>RAET1G/ULBP17</em> </td><td>MHC class 1 related </td><td>chr9:88231932-88402262 </td><td>0.0000 </td><td>Liu, Hou, Bickhart, and this study </td></tr><tr><td><em>RICTOR</em> </td><td>Cell growth </td><td>chr20:35376523-35514753 </td><td>0.1048 </td><td>Only this study </td></tr><tr><td><em>SEC23A</em> </td><td>Vesicle transport </td><td>chr21:49489514-49555507 </td><td>0.3050 </td><td>Only this study </td></tr><tr><td><em>SERPINB4</em> </td><td>Protease inhibitor </td><td>chr24:62364701-62371668 </td><td>0.2418 </td><td>Liu, Hou, Bickhart, and this study </td></tr><tr><td><em>SUB1</em> </td><td>Transcriptional activation </td><td>chr20:41122022-41143914 </td><td>0.2282 </td><td>Only this study </td></tr><tr><td><em>TMED2</em> </td><td>Secretory vesicle transport </td><td>chr17:54330420-54338333 </td><td>0.0000 </td><td>Only this study </td></tr><tr><td><em>UFM1</em> </td><td>Ubiquitin </td><td>chr6:71051155-71053533 </td><td>0.5121 </td><td>Only this study </td></tr><tr><td><em>ZNF280B</em> </td><td>Negative regulation of p53 </td><td>chr17:51251538-51262528 </td><td>0.2658 </td><td>Liu, Hou, and this study </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p><sup>a</sup><em>V</em><sub>ST</sub> was calculated from the comparison between the taurine and indicine individuals.</p></div></div><div content-id="tblfn3"><div class="fn"><p><sup>b</sup>Liu, Hou, and Bickhart: we focused on the comparisons with the published CNV results based on the same bovine HapMap samples using array CGH,<sup><a reveal-id="DSW013C10" data-open="DSW013C10" class="link link-ref link-reveal xref-bibr">10</a></sup> BovineHD SNP array,<sup><a reveal-id="DSW013C53" data-open="DSW013C53" class="link link-ref link-reveal xref-bibr">53</a></sup> and individual NGS,<sup><a reveal-id="DSW013C16" data-open="DSW013C16" class="link link-ref link-reveal xref-bibr">16</a></sup> respectively.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199650" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSW013TB2" class="table-wrap-title" data-id="DSW013TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Selected copy number variable genes identified from population sequence data</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene name </th><th>Function </th><th>Gene UMD3.1 coordinates </th><th><em>V</em><sub>ST</sub><sup>a</sup> </th><th>Identified<sup>b</sup> </th></tr></thead><tbody><tr><td><em>AOX1</em> </td><td>Detoxification </td><td>chr2:89517708-89589232 </td><td>0.5094 </td><td>Hou, Bickhart, and this study </td></tr><tr><td><em>ASZ1</em> </td><td>Spermatogenesis </td><td>chr4:51294534-51370343 </td><td>0.2109 </td><td>Only this study </td></tr><tr><td><em>CA1</em> </td><td>Carbonic anhydrase </td><td>chr14:79520632-79530892 </td><td>0.3270 </td><td>Hou, Bickhart, and this study </td></tr><tr><td><em>CFH</em> </td><td>Complement factor </td><td>chr16:5486704-6172566 </td><td>0.0483 </td><td>Hou, Bickhart, and this study </td></tr><tr><td><em>DDX21</em> </td><td>Translation initiation </td><td>chr28:25376358-25399769 </td><td>0.2375 </td><td>Only this study </td></tr><tr><td><em>DENR</em> </td><td>Translation initiation </td><td>chr29:7723699-7725004 </td><td>0.3285 </td><td>Only this study </td></tr><tr><td><em>FBXO16</em> </td><td>Ubiquitin protein ligase </td><td>chr8:10095869-10128675 </td><td>0.3558 </td><td>Bickhart and this study </td></tr><tr><td><em>FZD3</em> </td><td>Nervous system </td><td>chr8:10002971-10091175 </td><td>0.3334 </td><td>Bickhart and this study </td></tr><tr><td><em>GAL3ST1</em> </td><td>Glycolipid catalysis </td><td>chr17:71660016-71678806 </td><td>0.2435 </td><td>Hou and this study </td></tr><tr><td><em>GAT</em> </td><td>Detoxification </td><td>chr15:83472190-83493607 </td><td>0.4336 </td><td>Liu, Hou, Bickhart, and this study </td></tr><tr><td><em>GLYAT</em> </td><td>Detoxification </td><td>chr15:83455512-83469280 </td><td>0.4083 </td><td>Liu, Hou, Bickhart, and this study </td></tr><tr><td><em>GLYATL2</em> </td><td>Biological oxidation </td><td>chr15:83508339-83515102 </td><td>0.2257 </td><td>Bickhart and this study </td></tr><tr><td><em>KRTAP9-1</em> </td><td>Keratin family </td><td>chr19:42101853-42103421 </td><td>0.4578 </td><td>Bickhart and this study </td></tr><tr><td><em>LMBRD2</em> </td><td>Function unknown </td><td>chr20:38116509-38163145 </td><td>0.2573 </td><td>Only this study </td></tr><tr><td><em>PGR</em> </td><td>Progesterone receptor </td><td>chr15:8207682-8222806 </td><td>0.0103 </td><td>Only this study </td></tr><tr><td><em>PNPLA2</em> </td><td>Adipose tissue regulation </td><td>chr29:50742384-50747161 </td><td>0.0000 </td><td>Only this study </td></tr><tr><td><em>PRG3</em> </td><td>Carbohydrate binding </td><td>chr15:81920283-81926082 </td><td>0.2041 </td><td>Liu, Bickhart, and this study </td></tr><tr><td><em>RAET1G/ULBP17</em> </td><td>MHC class 1 related </td><td>chr9:88231932-88402262 </td><td>0.0000 </td><td>Liu, Hou, Bickhart, and this study </td></tr><tr><td><em>RICTOR</em> </td><td>Cell growth </td><td>chr20:35376523-35514753 </td><td>0.1048 </td><td>Only this study </td></tr><tr><td><em>SEC23A</em> </td><td>Vesicle transport </td><td>chr21:49489514-49555507 </td><td>0.3050 </td><td>Only this study </td></tr><tr><td><em>SERPINB4</em> </td><td>Protease inhibitor </td><td>chr24:62364701-62371668 </td><td>0.2418 </td><td>Liu, Hou, Bickhart, and this study </td></tr><tr><td><em>SUB1</em> </td><td>Transcriptional activation </td><td>chr20:41122022-41143914 </td><td>0.2282 </td><td>Only this study </td></tr><tr><td><em>TMED2</em> </td><td>Secretory vesicle transport </td><td>chr17:54330420-54338333 </td><td>0.0000 </td><td>Only this study </td></tr><tr><td><em>UFM1</em> </td><td>Ubiquitin </td><td>chr6:71051155-71053533 </td><td>0.5121 </td><td>Only this study </td></tr><tr><td><em>ZNF280B</em> </td><td>Negative regulation of p53 </td><td>chr17:51251538-51262528 </td><td>0.2658 </td><td>Liu, Hou, and this study </td></tr></tbody></table></div><div content-id="DSW013TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene name </th><th>Function </th><th>Gene UMD3.1 coordinates </th><th><em>V</em><sub>ST</sub><sup>a</sup> </th><th>Identified<sup>b</sup> </th></tr></thead><tbody><tr><td><em>AOX1</em> </td><td>Detoxification </td><td>chr2:89517708-89589232 </td><td>0.5094 </td><td>Hou, Bickhart, and this study </td></tr><tr><td><em>ASZ1</em> </td><td>Spermatogenesis </td><td>chr4:51294534-51370343 </td><td>0.2109 </td><td>Only this study </td></tr><tr><td><em>CA1</em> </td><td>Carbonic anhydrase </td><td>chr14:79520632-79530892 </td><td>0.3270 </td><td>Hou, Bickhart, and this study </td></tr><tr><td><em>CFH</em> </td><td>Complement factor </td><td>chr16:5486704-6172566 </td><td>0.0483 </td><td>Hou, Bickhart, and this study </td></tr><tr><td><em>DDX21</em> </td><td>Translation initiation </td><td>chr28:25376358-25399769 </td><td>0.2375 </td><td>Only this study </td></tr><tr><td><em>DENR</em> </td><td>Translation initiation </td><td>chr29:7723699-7725004 </td><td>0.3285 </td><td>Only this study </td></tr><tr><td><em>FBXO16</em> </td><td>Ubiquitin protein ligase </td><td>chr8:10095869-10128675 </td><td>0.3558 </td><td>Bickhart and this study </td></tr><tr><td><em>FZD3</em> </td><td>Nervous system </td><td>chr8:10002971-10091175 </td><td>0.3334 </td><td>Bickhart and this study </td></tr><tr><td><em>GAL3ST1</em> </td><td>Glycolipid catalysis </td><td>chr17:71660016-71678806 </td><td>0.2435 </td><td>Hou and this study </td></tr><tr><td><em>GAT</em> </td><td>Detoxification </td><td>chr15:83472190-83493607 </td><td>0.4336 </td><td>Liu, Hou, Bickhart, and this study </td></tr><tr><td><em>GLYAT</em> </td><td>Detoxification </td><td>chr15:83455512-83469280 </td><td>0.4083 </td><td>Liu, Hou, Bickhart, and this study </td></tr><tr><td><em>GLYATL2</em> </td><td>Biological oxidation </td><td>chr15:83508339-83515102 </td><td>0.2257 </td><td>Bickhart and this study </td></tr><tr><td><em>KRTAP9-1</em> </td><td>Keratin family </td><td>chr19:42101853-42103421 </td><td>0.4578 </td><td>Bickhart and this study </td></tr><tr><td><em>LMBRD2</em> </td><td>Function unknown </td><td>chr20:38116509-38163145 </td><td>0.2573 </td><td>Only this study </td></tr><tr><td><em>PGR</em> </td><td>Progesterone receptor </td><td>chr15:8207682-8222806 </td><td>0.0103 </td><td>Only this study </td></tr><tr><td><em>PNPLA2</em> </td><td>Adipose tissue regulation </td><td>chr29:50742384-50747161 </td><td>0.0000 </td><td>Only this study </td></tr><tr><td><em>PRG3</em> </td><td>Carbohydrate binding </td><td>chr15:81920283-81926082 </td><td>0.2041 </td><td>Liu, Bickhart, and this study </td></tr><tr><td><em>RAET1G/ULBP17</em> </td><td>MHC class 1 related </td><td>chr9:88231932-88402262 </td><td>0.0000 </td><td>Liu, Hou, Bickhart, and this study </td></tr><tr><td><em>RICTOR</em> </td><td>Cell growth </td><td>chr20:35376523-35514753 </td><td>0.1048 </td><td>Only this study </td></tr><tr><td><em>SEC23A</em> </td><td>Vesicle transport </td><td>chr21:49489514-49555507 </td><td>0.3050 </td><td>Only this study </td></tr><tr><td><em>SERPINB4</em> </td><td>Protease inhibitor </td><td>chr24:62364701-62371668 </td><td>0.2418 </td><td>Liu, Hou, Bickhart, and this study </td></tr><tr><td><em>SUB1</em> </td><td>Transcriptional activation </td><td>chr20:41122022-41143914 </td><td>0.2282 </td><td>Only this study </td></tr><tr><td><em>TMED2</em> </td><td>Secretory vesicle transport </td><td>chr17:54330420-54338333 </td><td>0.0000 </td><td>Only this study </td></tr><tr><td><em>UFM1</em> </td><td>Ubiquitin </td><td>chr6:71051155-71053533 </td><td>0.5121 </td><td>Only this study </td></tr><tr><td><em>ZNF280B</em> </td><td>Negative regulation of p53 </td><td>chr17:51251538-51262528 </td><td>0.2658 </td><td>Liu, Hou, and this study </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p><sup>a</sup><em>V</em><sub>ST</sub> was calculated from the comparison between the taurine and indicine individuals.</p></div></div><div content-id="tblfn3"><div class="fn"><p><sup>b</sup>Liu, Hou, and Bickhart: we focused on the comparisons with the published CNV results based on the same bovine HapMap samples using array CGH,<sup><a reveal-id="DSW013C10" data-open="DSW013C10" class="link link-ref link-reveal xref-bibr">10</a></sup> BovineHD SNP array,<sup><a reveal-id="DSW013C53" data-open="DSW013C53" class="link link-ref link-reveal xref-bibr">53</a></sup> and individual NGS,<sup><a reveal-id="DSW013C16" data-open="DSW013C16" class="link link-ref link-reveal xref-bibr">16</a></sup> respectively.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87199650" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87199653 id="87199653" class="section-title" data-legacy-id=s3d>3.4. Lineage-specific CNV genes based on a heuristic approach</h3>
<p>We first identified lineage specific, copy number variable genes (CNV genes) using a heuristic approach (see Materials and methods). Dairy cattle-specific CNVs tended to be present at low frequencies in our 28 dairy cattle, and they manifested as small copy number changes of affected genes. Several of these dairy-specific CNVRs were found to intersect genes related to cellular growth and development pathways, including <em>RICTOR</em> (rapamycin-insensitive companion of mTOR)<sup><a reveal-id="DSW013C64" data-open="DSW013C64" class="link link-ref link-reveal xref-bibr">64</a></sup> and <em>TMED2</em>.<sup><a reveal-id="DSW013C65" data-open="DSW013C65" class="link link-ref link-reveal xref-bibr">65</a></sup> We also identified several lipid metabolism genes that overlapped CNVRs exclusive to beef cattle (over 40 samples). Within our Angus data set, we discovered six animals that had a predicted heterozygous duplication of <em>PNPLA3</em> (the Patatin-like phospholipase domain-containing protein 3). This gene is expressed in adipose tissue and liver, and is associated with the <em>de novo</em> synthesis of fatty acids.<sup><a reveal-id="DSW013C66" data-open="DSW013C66" class="link link-ref link-reveal xref-bibr">66</a></sup> In indicine beef cattle (Nelore, Brahman, and Gir), we also detected a duplication in a predicted Ensembl gene (ENSBTAT00000043749) containing functional domains related to lipid metabolism.</p>                            <h3 scrollto-destination=87199657 id="87199657" class="section-title" data-legacy-id=s3e>3.5. Gene family expansion, diversity, and evolution</h3>
<p>We identified copy number variable genes among the different subspecies/breeds using <em>V</em><sub>ST</sub> statistics. We defined highly stratified genes as genes having <em>V</em><sub>ST</sub> values &gt;0.2. CN plots for these stratified genes showed clear differences in the average CN value for taurine and indicine animals (Fig. <a reveal-id="DSW013F3" data-open="DSW013F3" class="link link-reveal link-table xref-fig">3</a>A). Based on <em>V<sub>ST</sub></em> values, <em>ZNF280B</em>, <em>FBXO16</em>, <em>KRTAP9-1</em>, <em>MCM4</em>, <em>SERPINB4</em>, <em>CA1</em>, <em>FZD3</em>, <em>GLAYAT</em>, <em>GAT</em>, <em>MANBA</em>, and <em>DENR</em> were the most stratified genes. To provide orthogonal experimental support for the sequence-based <em>V</em><sub>ST</sub> results, we also retrieved log<sub>2</sub> ratios from array CGH data for the same animals. Representative results are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span> for the <em>GAT</em>, <em>GLYAT,</em> and <em>KRTAP9-1</em> genes, further confirming the sequence-based <em>V</em><sub>ST</sub> results. </p>                        <a id="87199659" scrollto-destination="87199659"></a>
<div id="DSW013F3" data-id="DSW013F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/m_dsw01303.png?Expires=2147483647&amp;Signature=I7HTB3JxmuTt3vhcU2iG6kszwXNNcERLeILOSqDUM4gxDEkTxTb56OD~xq34sUpHzfGFHZ4Ff6bpb1ASUbJCyjZ4UnuRPUXoOs5UOrJSvRstRZXuLozQ-zODrz2Yz0Fn4mp5UU~OQ8JAVZW7CYMinIKTrwJxpkkvO38Bbkbc50xAQcW6ybt7CWmo63p1znP6W~iO6RAyz8MEjNgsphx3CuBf1bvVRBvbNVGh31ORVLN8~MLd-DOW0sjPVbWWrKPiwtqdP1Afjd0exoX5YJ3Dj7JW72lMl7mbhubACyLT4GO3uYMOCEjjD5tXjl7rpLVy08HkvSfDjaUj5GXiUSVPEA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Cattle gene family copy number diversity and evolution. The genes most stratified by copy number on the basis of VST analysis of taurine and indicine cattle (A). The most copy number variable genes in both taurine and indicine subspecies (legend insets denote group colors) tended to be immune system-related genes. Histograms showing the distributions of copy numbers among the unrelated individuals in each group are plotted for the KRTAP9-1 gene (B) and the MCM4 gene (C). X-axis values indicate copy number and Y-axis values indicate sample count. Individual copy number values for each gene can be found in Supplementary Table S7. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01303.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199659/dsw01303.tiff" path-from-xml="dsw01303.tiff" target="_blank">View large</a><a section="87199659" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw01303.png?Expires=2147483647&amp;Signature=ACUJiFDy-GA-3UCs5Gq2oLNXwNYUcJRyzvoPExGTA8WBw1eeJ5fwVjAKOqZ0zcVRqQt1Ju6v6DHsiCQWkaO3R3Z4mdKvdgCqipAWuxjt~lLIpIJqbi5HDWj83lxshVZ2spPOqWya~Ry5rrOiPcYHqr1Yym4ZD-Tl2ABvIvrZCFBF8HCr1IGJPUhWH8v-5Z2ViJWzsGNWVPxYzAIwPzMndLfm7Oi42NlGorI0tj8zIQYx6FL~dJ1p6xenn8adR7oLeuD9xhpFjG~6s5StivzXEeVlZcBMjohvwoYFzjArm4p850ZSXa5cQtMtmv6gQ0E9WUAtDMYgXBXP2NgZkxo-AQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199659&amp;ar=1745562&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw01303.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Cattle gene family copy number diversity and evolution. The genes most stratified by copy number on the basis of <em>V</em><sub>ST</sub> analysis of taurine and indicine cattle (A). The most copy number variable genes in both taurine and indicine subspecies (legend insets denote group colors) tended to be immune system-related genes. Histograms showing the distributions of copy numbers among the unrelated individuals in each group are plotted for the <em>KRTAP9-1</em> gene (B) and the <em>MCM4</em> gene (C). <em>X</em>-axis values indicate copy number and <em>Y</em>-axis values indicate sample count. Individual copy number values for each gene can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&amp;Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW013F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/m_dsw01303.png?Expires=2147483647&amp;Signature=I7HTB3JxmuTt3vhcU2iG6kszwXNNcERLeILOSqDUM4gxDEkTxTb56OD~xq34sUpHzfGFHZ4Ff6bpb1ASUbJCyjZ4UnuRPUXoOs5UOrJSvRstRZXuLozQ-zODrz2Yz0Fn4mp5UU~OQ8JAVZW7CYMinIKTrwJxpkkvO38Bbkbc50xAQcW6ybt7CWmo63p1znP6W~iO6RAyz8MEjNgsphx3CuBf1bvVRBvbNVGh31ORVLN8~MLd-DOW0sjPVbWWrKPiwtqdP1Afjd0exoX5YJ3Dj7JW72lMl7mbhubACyLT4GO3uYMOCEjjD5tXjl7rpLVy08HkvSfDjaUj5GXiUSVPEA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Cattle gene family copy number diversity and evolution. The genes most stratified by copy number on the basis of VST analysis of taurine and indicine cattle (A). The most copy number variable genes in both taurine and indicine subspecies (legend insets denote group colors) tended to be immune system-related genes. Histograms showing the distributions of copy numbers among the unrelated individuals in each group are plotted for the KRTAP9-1 gene (B) and the MCM4 gene (C). X-axis values indicate copy number and Y-axis values indicate sample count. Individual copy number values for each gene can be found in Supplementary Table S7. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01303.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199659/dsw01303.tiff" path-from-xml="dsw01303.tiff" target="_blank">View large</a><a section="87199659" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw01303.png?Expires=2147483647&amp;Signature=ACUJiFDy-GA-3UCs5Gq2oLNXwNYUcJRyzvoPExGTA8WBw1eeJ5fwVjAKOqZ0zcVRqQt1Ju6v6DHsiCQWkaO3R3Z4mdKvdgCqipAWuxjt~lLIpIJqbi5HDWj83lxshVZ2spPOqWya~Ry5rrOiPcYHqr1Yym4ZD-Tl2ABvIvrZCFBF8HCr1IGJPUhWH8v-5Z2ViJWzsGNWVPxYzAIwPzMndLfm7Oi42NlGorI0tj8zIQYx6FL~dJ1p6xenn8adR7oLeuD9xhpFjG~6s5StivzXEeVlZcBMjohvwoYFzjArm4p850ZSXa5cQtMtmv6gQ0E9WUAtDMYgXBXP2NgZkxo-AQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199659&amp;ar=1745562&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw01303.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Cattle gene family copy number diversity and evolution. The genes most stratified by copy number on the basis of <em>V</em><sub>ST</sub> analysis of taurine and indicine cattle (A). The most copy number variable genes in both taurine and indicine subspecies (legend insets denote group colors) tended to be immune system-related genes. Histograms showing the distributions of copy numbers among the unrelated individuals in each group are plotted for the <em>KRTAP9-1</em> gene (B) and the <em>MCM4</em> gene (C). <em>X</em>-axis values indicate copy number and <em>Y</em>-axis values indicate sample count. Individual copy number values for each gene can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&amp;Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>CN stratified genes tended to be immune system related, which is expected given the different environmental challenges in the history of evolution of taurine and indicine cattle. One of these stratified CNVs represents a significantly higher duplication of the <em>KRTAP9-1</em> gene in taurine cattle, which is a paralog of <em>KRTAP9-2</em> that was previously reported to likely be involved in indicine tick resistance (Fig. <a reveal-id="DSW013F3" data-open="DSW013F3" class="link link-reveal link-table xref-fig">3</a>B). A duplication of the <em>MCM4</em> gene (Fig. <a reveal-id="DSW013F3" data-open="DSW013F3" class="link link-reveal link-table xref-fig">3</a>C) was found in indicine cattle compared with taurine cattle. We also confirmed several other gene families appearing to be copy number variable, including lysozyme, defensin, and unique long binding protein (<em>ULBP</em>) families and the major histocompatibility complex (<em>MHC</em>).</p><p>Discoveries of high-frequency gene duplications suggest that the affected gene families are currently expanding in a ‘gene family birth and death’ model as described by Nei and Rooney.<sup><a reveal-id="DSW013C67" data-open="DSW013C67" class="link link-ref link-reveal xref-bibr">67</a></sup> Such multicopy genes, if present in a sufficiently large proportion of the population, can be thought of as signs of diversifying selection or selection by overdominance.<sup><a reveal-id="DSW013C68" data-open="DSW013C68" class="link link-ref link-reveal xref-bibr">68</a></sup> One example of this can be found in the Olfactory Receptor (OR) gene family, which has several member genes that detect odorant molecules through combinatorial binding across other paralogous family members.<sup><a reveal-id="DSW013C69" data-open="DSW013C69" class="link link-ref link-reveal xref-bibr">69</a></sup> Therefore, the duplication and subsequent mutation of OR gene members allow for a greater range of odorant detection for a host organism. Indeed, out of 134 annotated OR genes, we have identified 31 (23.1%) individual genes that have predicted duplications in our data set.</p><p>We have detected several additional gene families that appear to be subject to a high degree of duplication in our data set, and these families likely represent classes of genes that are in the processes of subfunctionalization and neofunctionalization in cattle. They include a cluster of prolactin-related protein family (<em>PRP</em>) genes that appears to be duplicated in 96% (74/75) animals. It was previously discovered using the BovineSNP50 array<sup><a reveal-id="DSW013C70" data-open="DSW013C70" class="link link-ref link-reveal xref-bibr">70</a></sup>; however, we refined the event from 2.4 down to 0.7 megabases in size. Another locus containing several pregnancy-associated glycoprotein (<em>PAG</em>) family members was found to be duplicated in all animals within CNVR 1717.</p>                            <h3 scrollto-destination=87199663 id="87199663" class="section-title" data-legacy-id=s3f>3.6. Haplotype network analyses near selected multicopy genes</h3>
<p>It is important to note that CNVs in some loci may have different alleles. Earlier results also suggest that the diversity of a subset of multicopy genes like human OR genes may have been maintained by balancing selection, in the form of overdominance.<sup><a reveal-id="DSW013C68" data-open="DSW013C68" class="link link-ref link-reveal xref-bibr">68</a></sup> For example, a 660 kb deletion with antagonistic effects on fertility and milk production was recently found at high frequency in Nordic Red cattle, providing evidence for balancing selection of CNVs in livestock.<sup><a reveal-id="DSW013C71" data-open="DSW013C71" class="link link-ref link-reveal xref-bibr">71</a></sup> To investigate the potential effect of overdominance on selection and evolution of multicopy genes, we further investigated haplotype evolution pattern using the BovineHD SNP array. We obtained 11 haplotypes within the 50.3 kb haploblock region near the <em>GLYAT/GAT</em> locus (Fig. <a reveal-id="DSW013F4" data-open="DSW013F4" class="link link-reveal link-table xref-fig">4</a>A). The most common haplotype, H1 (with frequency of 70.06%), was mainly found in taurine cattle (HOL, ANG, JER, and LMS) and only minor portions were found in indicine cattle (ROM, BRM, GIR, and NEL) (Fig. <a reveal-id="DSW013F4" data-open="DSW013F4" class="link link-reveal link-table xref-fig">4</a>A). H2 (with frequency of 10.56%) included a large proportion of taurine cattle (ANG, LMS, and ROM). We also observed two haplotypes exclusive to indicine cattle with a combined frequency of 10.54%: H3 and H4. Altogether, this pattern indicated that separate haplotypes were clustered only for the indicine cattle (BRM, GIR, and NEL), while other common haplotypes were identified for taurine cattle (HOL, ANG, LMS, JER, and ROM). BRM and ROM were the only exceptions, as they were often associated with both taurine and indicine cattle. This was not unexpected, as it mirrors the complex ancestral backgrounds of these two breeds, since BRM cattle are a known indicine breed with taurine influence and ROM share distinctive genetic ancestry with indicine cattle.<sup><a reveal-id="DSW013C44" data-open="DSW013C44" class="link link-ref link-reveal xref-bibr">44</a></sup> We also found similar results for other copy number variable genes, such as <em>ASZ1</em> (Fig. <a reveal-id="DSW013F4" data-open="DSW013F4" class="link link-reveal link-table xref-fig">4</a>B), <em>AOX1,</em> and <em>FZD3</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S9 and S10</a></span>). These haplotype network analyses suggest that for a subset of multicopy genes: (i) common overlapping allelic haplotypes were often present within the taurine cattle, while separate distinct haplotypes were present in the indicine cattle, suggesting different evolutionary history for these two cattle subspecies; and (ii) there was high allelic diversity near multicopy genes maintained by balancing selection, in the form of overdominance, suggesting that they have been under different selection pressures in these two cattle subspecies. </p>                        <a id="87199665" scrollto-destination="87199665"></a>
<div id="DSW013F4" data-id="DSW013F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/m_dsw01304.png?Expires=2147483647&Signature=3GZ5CmoEdp5DUUeRknOdXtpqFI6YitM4m4CLrwiSxjBpRApbM~sgdRe8hgrNhtE3Owx6VmzuNbGQhV4C1rweMl5p~5dsOkyrufqI5ZcG2EheadnOwUE5q1wkXUXfOwEJ9sSrXFnoEhkFKwH~~OzfS-A5U9pBIOkvUx1RkRoEj1jLAROKaMOk3VAhEQ1hLtFsvyJLH22vAHS~d7NZosDD6OvhSWufJCjGZIQUFzHAEvERWlO1~kLywXfGllwxSR983RwYzabMkJjp0Z8siZlOw-PMEz~L4YEE~NS2UJVy9eRdZ6sIdmzml23O2aOaMvN97304cBBfeDFeAq-nAj51FA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Haplotype networks of two loci. (A) The GAT/GLYAT locus and (B) the ASZ1 locus. Each node represents a different haplotype, with the size of the circle proportional to frequency. Circles are colour coded according to breeds. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01304.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199665/dsw01304.tiff" path-from-xml="dsw01304.tiff" target="_blank">View large</a><a section="87199665" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw01304.png?Expires=2147483647&Signature=yIz~IRor~eObZk99xgEAqq6uMwCYUupKezbIUuDKHBHRmwqnBbo-kDLlZ2TcesEeTyGeNt8IYsFAhaQw82DdgM1lZ4USNccrbulJNDeBPF20vx9oFih-SUyB0LBzJsuS0JvXvBbUClbx9YoCgsEVU2rpjg-pw0thxV83YbpyBnX8Upr-TuLhzX5Fh1KwH76bIN96Tz~CqR~UfPRNkbkJRwAS3F7Z8LKiGGza63SpMd-iUCwWY2wDW7aw6YjPtuU6KnUCBOOikfTwGJmR6I8dkpSB~3peH9CFsw4OQkO7cfuxR-CxFuEVFGarq1y4x5O0~ir2g6tsyD-Kh-rHep5eSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199665&ar=1745562&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01304.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Haplotype networks of two loci. (A) The GAT/GLYAT locus and (B) the ASZ1 locus. Each node represents a different haplotype, with the size of the circle proportional to frequency. Circles are colour coded according to breeds. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSW013F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/m_dsw01304.png?Expires=2147483647&Signature=3GZ5CmoEdp5DUUeRknOdXtpqFI6YitM4m4CLrwiSxjBpRApbM~sgdRe8hgrNhtE3Owx6VmzuNbGQhV4C1rweMl5p~5dsOkyrufqI5ZcG2EheadnOwUE5q1wkXUXfOwEJ9sSrXFnoEhkFKwH~~OzfS-A5U9pBIOkvUx1RkRoEj1jLAROKaMOk3VAhEQ1hLtFsvyJLH22vAHS~d7NZosDD6OvhSWufJCjGZIQUFzHAEvERWlO1~kLywXfGllwxSR983RwYzabMkJjp0Z8siZlOw-PMEz~L4YEE~NS2UJVy9eRdZ6sIdmzml23O2aOaMvN97304cBBfeDFeAq-nAj51FA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Haplotype networks of two loci. (A) The GAT/GLYAT locus and (B) the ASZ1 locus. Each node represents a different haplotype, with the size of the circle proportional to frequency. Circles are colour coded according to breeds. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsw01304.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199665/dsw01304.tiff" path-from-xml="dsw01304.tiff" target="_blank">View large</a><a section="87199665" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw01304.png?Expires=2147483647&Signature=yIz~IRor~eObZk99xgEAqq6uMwCYUupKezbIUuDKHBHRmwqnBbo-kDLlZ2TcesEeTyGeNt8IYsFAhaQw82DdgM1lZ4USNccrbulJNDeBPF20vx9oFih-SUyB0LBzJsuS0JvXvBbUClbx9YoCgsEVU2rpjg-pw0thxV83YbpyBnX8Upr-TuLhzX5Fh1KwH76bIN96Tz~CqR~UfPRNkbkJRwAS3F7Z8LKiGGza63SpMd-iUCwWY2wDW7aw6YjPtuU6KnUCBOOikfTwGJmR6I8dkpSB~3peH9CFsw4OQkO7cfuxR-CxFuEVFGarq1y4x5O0~ir2g6tsyD-Kh-rHep5eSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87199665&ar=1745562&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw01304.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Haplotype networks of two loci. (A) The GAT/GLYAT locus and (B) the ASZ1 locus. Each node represents a different haplotype, with the size of the circle proportional to frequency. Circles are colour coded according to breeds. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87199666 id="87199666" class="section-title" data-legacy-id=s3g>3.7. The impacts of the reference genome assembly</h3>
<p>Different versions of cattle reference genome assemblies (Btau_4.0 and UMD3.1) have different RefSeq gene annotations, particularly in CN variable regions of the cattle genome. For example, the <em>CATHL4</em> gene, for which we previously reported copy number change,<sup><a reveal-id="DSW013C16" data-open="DSW013C16" class="link link-ref link-reveal xref-bibr">16</a></sup> was located on chrUn of UMD3.1. Since our RD method uses a window approach that relies on large genomic segments, this prevented us from assessing the copy number status of <em>CATHL4</em> on chrUn. However, we did find <em>CATHL1</em> was copy number variable in this study. Similarly, we were unable to assess the copy number of <em>KRTAP9-2</em>; however<em>,</em> we detected copy number changes for one of its paralogs, <em>KRTAP9-1</em>, in this study.</p>                            <h3 scrollto-destination=87199668 id="87199668" class="section-title" data-legacy-id=s3h>3.8. Limitations and future directions</h3>
<p>Similar to SNP and microsatellite, CNV distribution within and among populations seems to be shaped by mutation, recombination, gene conversion, selection, and demographic history.<sup><a reveal-id="DSW013C18" data-open="DSW013C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSW013C19" data-open="DSW013C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSW013C28" data-open="DSW013C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSW013C72" data-open="DSW013C72" class="link link-ref link-reveal xref-bibr">72</a></sup> However, CNV genetic markers may not be currently compatible with current population analyses, because CNVs violate the classical population genetics assumptions based on the infinite allele model and the infinite site model for SNP. Compared with SNPs, limitations of CNVs as markers were observed in this study probably due to their distinct mutation mechanisms, high mutation rates, heterogeneities among loci, and uncertainties related to allele calling. Similar observations were also reported for microsatellite—primarily due to similar limitations of detection and variability.<sup><a reveal-id="DSW013C73" data-open="DSW013C73" class="link link-ref link-reveal xref-bibr">73</a>,<a reveal-id="DSW013C74" data-open="DSW013C74" class="link link-ref link-reveal xref-bibr">74</a></sup> For example, although NAHR is believed to be responsible for most of large duplication CNVs in cattle, inference, and predictions on the forces influencing populations require the modelling of the mutational process generating CNV. We currently lack such a model for the large duplications present in cattle. This is also compounded by the fact that homoplasy caused by recurrent events is expected to occur relatively often for CNV compared with SNP because of their high mutation rates.</p><p>Despite the aforementioned limitations, this study represents one of the first attempts to genotype CNVs within large, diverse cattle populations using sequence data. Although beyond the scope of this study, a comparison with human-centric CNV genotyping methods using cattle sequence data will provide a useful contrast in approaches. Our results provide a new glimpse into the diversity of selective pressures during cattle speciation. We confirmed that cattle are strikingly diverse, despite relatively low estimated current population sizes for several taurine cattle as shown previously.<sup><a reveal-id="DSW013C37" data-open="DSW013C37" class="link link-ref link-reveal xref-bibr">37</a></sup> Our population-genetic analyses based on CNVs reveal the population structures of these taurine and indicine cattle and uncover hundreds of CNVs showing elevated population differentiation near important functional genes. We highlighted several subspecies specific or differential CNV gene overlaps that are likely subject to subfunctionalization and neofunctionalization. We also identified key regions of the cattle genome that are subject to variation and reported several potential genes affecting productive traits. These discoveries provide a basis for future efforts to genotype and track large CNVs in cattle. More sequencing data from the 1000 Bull Genomes Project<sup><a reveal-id="DSW013C75" data-open="DSW013C75" class="link link-ref link-reveal xref-bibr">75</a></sup> or the analysis of additional outlier groups (e.g. African cattle breeds) will help to validate and refine the link between genomic copy number in these regions or different alleles with production and health traits.</p>                            <h2 scrollto-destination=87199671 id="87199671" class="section-title" data-legacy-id=s4>Authors' contributions</h2>
<p>G.E.L. and D.M.B. conceived and designed the experiments. D.M.B., L.X., J.L.H., J.B.C., and J.S. performed <em>in silico</em> prediction and computational analyses. D.M.B. and L.X. performed aCGH, qPCR confirmation. D.J.N., S.G.S., J.F.G., C.P.V.T., T.S.S., R.D.S., J.F.T., and H.A.L. collected samples and generated sequence data. G.E.L. and D.M.B. wrote the paper.</p>                            <h2 scrollto-destination=87199673 id="87199673" class="section-title" data-legacy-id=s5>Conflict of interest statement</h2>
<p>T.S.S. is an employee of Acceligen Inc. of Animal Agriculture Subsidiary of Recombinetics, Inc. All other authors declare no potential conflict of interest.</p>                            <h2 scrollto-destination=87199675 id="87199675" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This work was supported in part by Agriculture and Food Research Initiative (AFRI) competitive grant No. 2011-67015-30183 from the USDA National Institute of Food and Agriculture (NIFA) Animal Genome Program. J.F.T. and C.P.V.T. were supported by AFRI competitive grant No. 2009-65205-05635 from the USDA NIFA. J.F.T. and R.D.S. were further supported by AFRI competitive grants No. 2011-68004-30367, 2011-68004-30214 and 2013-68004-20364 from the USDA NIFA Animal Genome Program. Funding to pay the Open Access publication charges for this article was provided by the USDA National Institute of Food and Agriculture (NIFA) Animal Genome Program.</p>                            <h2 scrollto-destination=87199677 id="87199677" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank members of the Illumina BovineHD SNP Consortium for sharing their samples and data. We would thank the Council on Dairy Cattle Breeding (Reynoldsburg, OH) for access to animal phenotypic information, and we thank the Cooperative Dairy DNA Repository (Beltsville, MD) for access to semen stocks used to generate Illumina DNA sequence libraries. We also thank T. Brown, R. Anderson, and A. Dimtchev for technical assistance. Mention of trade names or commercial products in this article is solely for the purpose of providing specific information and does not imply recommendation or endorsement by the US Department of Agriculture.</p>                            <h2 scrollto-destination=87199679 id="87199679" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSW013C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mills</div>
               <div class="given-names">R.E.</div>
            </div>, <div class="name string-name"><div class="surname">Walter</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Stewart</div> <div class="given-names">C.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Mapping copy number variation by population-scale genome sequencing</div>, <div class="source">Nature</div> , <div class="volume">470</div>, <div class="fpage">59</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20copy%20number%20variation%20by%20population-scale%20genome%20sequencing&amp;author=R.E.%20Mills&amp;author=K.%20Walter&amp;author=C.%20Stewart&amp;publication_year=2011&amp;journal=Nature&amp;volume=470&amp;pages=59-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature09708" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature09708" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature09708"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21293372" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sudmant</div>
               <div class="given-names">P.H.</div>
            </div>, <div class="name string-name"><div class="surname">Rausch</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Gardner</div> <div class="given-names">E.J.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">An integrated map of structural variation in 2,504 human genomes</div>, <div class="source">Nature</div> , <div class="volume">526</div>, <div class="fpage">75</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20map%20of%20structural%20variation%20in%202%2C504%20human%20genomes&amp;author=P.H.%20Sudmant&amp;author=T.%20Rausch&amp;author=E.J.%20Gardner&amp;publication_year=2015&amp;journal=Nature&amp;volume=526&amp;pages=75-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature15394" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature15394" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature15394"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26432246" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sudmant</div>
               <div class="given-names">P.H.</div>
            </div>, <div class="name string-name"><div class="surname">Kitzman</div> <div class="given-names">J.O.</div></div>, <div class="name string-name"><div class="surname">Antonacci</div> <div class="given-names">F.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Diversity of human copy number variation and multicopy genes</div>, <div class="source">Science</div> , <div class="volume">330</div>, <div class="fpage">641</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20of%20human%20copy%20number%20variation%20and%20multicopy%20genes&amp;author=P.H.%20Sudmant&amp;author=J.O.%20Kitzman&amp;author=F.%20Antonacci&amp;publication_year=2010&amp;journal=Science&amp;volume=330&amp;pages=641-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1197005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1197005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1197005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21030649" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Girirajan</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Campbell</div> <div class="given-names">C.D.</div></div>, <div class="name string-name"><div class="surname">Eichler</div> <div class="given-names">E.E.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Human copy number variation and complex genetic disease</div>, <div class="source">Annu. Rev. Genet.</div> , <div class="volume">45</div>, <div class="fpage">203</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20copy%20number%20variation%20and%20complex%20genetic%20disease&amp;author=S.%20Girirajan&amp;author=C.D.%20Campbell&amp;author=E.E.%20Eichler&amp;publication_year=2011&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=45&amp;pages=203-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-genet-102209-163544" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-genet-102209-163544" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-genet-102209-163544"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21854229" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bae</div>
               <div class="given-names">J.S.</div>
            </div>, <div class="name string-name"><div class="surname">Cheong</div> <div class="given-names">H.S.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">L.H.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Identification of copy number variations and common deletion polymorphisms in cattle</div>, <div class="source">BMC Genomics</div> , <div class="volume">11</div>, <div class="fpage">232</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20copy%20number%20variations%20and%20common%20deletion%20polymorphisms%20in%20cattle&amp;author=J.S.%20Bae&amp;author=H.S.%20Cheong&amp;author=L.H.%20Kim&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=232" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-232" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-232" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-232"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20377913" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chen</div>
               <div class="given-names">W.K.</div>
            </div>, <div class="name string-name"><div class="surname">Swartz</div> <div class="given-names">J.D.</div></div>, <div class="name string-name"><div class="surname">Rush</div> <div class="given-names">L.J.</div></div>, <div class="name string-name"><div class="surname">Alvarez</div> <div class="given-names">C.E.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Mapping DNA structural variation in dogs</div>, <div class="source">Genome Res.</div> , <div class="volume">19</div>, <div class="fpage">500</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20DNA%20structural%20variation%20in%20dogs&amp;author=W.K.%20Chen&amp;author=J.D.%20Swartz&amp;author=L.J.%20Rush&amp;author=C.E.%20Alvarez&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=500-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.083741.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.083741.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.083741.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19015322" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fadista</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Thomsen</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Holm</div> <div class="given-names">L.E.</div></div>, <div class="name string-name"><div class="surname">Bendixen</div> <div class="given-names">C.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Copy number variation in the bovine genome</div>, <div class="source">BMC Genomics</div> , <div class="volume">11</div>, <div class="fpage">284</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Copy%20number%20variation%20in%20the%20bovine%20genome&amp;author=J.%20Fadista&amp;author=B.%20Thomsen&amp;author=L.E.%20Holm&amp;author=C.%20Bendixen&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=284" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-284" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-284" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-284"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20459598" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fontanesi</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Beretti</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Riggio</div> <div class="given-names">V.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">Copy number variation and missense mutations of the agouti signaling protein (ASIP) gene in goat breeds with different coat colors</div>, <div class="source">Cytogenet. Genome Res.</div> , <div class="volume">126</div>, <div class="fpage">333</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Copy%20number%20variation%20and%20missense%20mutations%20of%20the%20agouti%20signaling%20protein%20(ASIP)%20gene%20in%20goat%20breeds%20with%20different%20coat%20colors&amp;author=L.%20Fontanesi&amp;author=F.%20Beretti&amp;author=V.%20Riggio&amp;publication_year=2009&amp;journal=Cytogenet.%20Genome%20Res.&amp;volume=126&amp;pages=333-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000268089" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000268089" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000268089"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20016133" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fontanesi</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Beretti</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Martelli</div> <div class="given-names">P.L.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">A first comparative map of copy number variations in the sheep genome</div>, <div class="source">Genomics</div> , <div class="volume">97</div>, <div class="fpage">158</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20first%20comparative%20map%20of%20copy%20number%20variations%20in%20the%20sheep%20genome&amp;author=L.%20Fontanesi&amp;author=F.%20Beretti&amp;author=P.L.%20Martelli&amp;publication_year=2011&amp;journal=Genomics&amp;volume=97&amp;pages=158-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygeno.2010.11.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2010.11.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygeno.2010.11.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21111040" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">G.E.</div>
            </div>, <div class="name string-name"><div class="surname">Hou</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">B.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Analysis of copy number variations among diverse cattle breeds</div>, <div class="source">Genome Res.</div> , <div class="volume">20</div>, <div class="fpage">693</div>–<div class="lpage">703</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20copy%20number%20variations%20among%20diverse%20cattle%20breeds&amp;author=G.E.%20Liu&amp;author=Y.%20Hou&amp;author=B.%20Zhu&amp;publication_year=2010&amp;journal=Genome%20Res.&amp;volume=20&amp;pages=693-703" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.105403.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.105403.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.105403.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20212021" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nicholas</div>
               <div class="given-names">T.J.</div>
            </div>, <div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Ventura</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Mealey</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Eichler</div> <div class="given-names">E.E.</div></div>, <div class="name string-name"><div class="surname">Akey</div> <div class="given-names">J.M.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">The genomic architecture of segmental duplications and associated copy number variants in dogs</div>, <div class="source">Genome Res.</div> , <div class="volume">19</div>, <div class="fpage">491</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genomic%20architecture%20of%20segmental%20duplications%20and%20associated%20copy%20number%20variants%20in%20dogs&amp;author=T.J.%20Nicholas&amp;author=Z.%20Cheng&amp;author=M.%20Ventura&amp;author=K.%20Mealey&amp;author=E.E.%20Eichler&amp;author=J.M.%20Akey&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=491-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.084715.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.084715.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.084715.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19129542" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ramayo-Caldas</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Castello</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Pena</div> <div class="given-names">R.N.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Copy number variation in the porcine genome inferred from a 60 k SNP BeadChip</div>, <div class="source">BMC Genomics</div> , <div class="volume">11</div>, <div class="fpage">593</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Copy%20number%20variation%20in%20the%20porcine%20genome%20inferred%20from%20a%2060%20k%20SNP%20BeadChip&amp;author=Y.%20Ramayo-Caldas&amp;author=A.%20Castello&amp;author=R.N.%20Pena&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=593" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-593" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-593" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-593"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20969757" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kijas</div>
               <div class="given-names">J.W.</div>
            </div>, <div class="name string-name"><div class="surname">Barendse</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Barris</div> <div class="given-names">W.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Analysis of copy number variants in the cattle genome</div>, <div class="source">Gene</div> , <div class="volume">482</div>, <div class="fpage">73</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20copy%20number%20variants%20in%20the%20cattle%20genome&amp;author=J.W.%20Kijas&amp;author=W.%20Barendse&amp;author=W.%20Barris&amp;publication_year=2011&amp;journal=Gene&amp;volume=482&amp;pages=73-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2011.04.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2011.04.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2011.04.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21620936" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Snyder</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Du</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Gerstein</div> <div class="given-names">M.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Personal genome sequencing: current approaches and challenges</div>, <div class="source">Genes Dev.</div> , <div class="volume">24</div>, <div class="fpage">423</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Personal%20genome%20sequencing%3A%20current%20approaches%20and%20challenges&amp;author=M.%20Snyder&amp;author=J.%20Du&amp;author=M.%20Gerstein&amp;publication_year=2010&amp;journal=Genes%20Dev.&amp;volume=24&amp;pages=423-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1864110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1864110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1864110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20194435" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rubin</div>
               <div class="given-names">C.J.</div>
            </div>, <div class="name string-name"><div class="surname">Zody</div> <div class="given-names">M.C.</div></div>, <div class="name string-name"><div class="surname">Eriksson</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Whole-genome resequencing reveals loci under selection during chicken domestication</div>, <div class="source">Nature</div> , <div class="volume">464</div>, <div class="fpage">587</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20resequencing%20reveals%20loci%20under%20selection%20during%20chicken%20domestication&amp;author=C.J.%20Rubin&amp;author=M.C.%20Zody&amp;author=J.%20Eriksson&amp;publication_year=2010&amp;journal=Nature&amp;volume=464&amp;pages=587-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08832" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08832" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08832"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20220755" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bickhart</div>
               <div class="given-names">D.M.</div>
            </div>, <div class="name string-name"><div class="surname">Hou</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Schroeder</div> <div class="given-names">S.G.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Copy number variation of individual cattle genomes using next-generation sequencing</div>, <div class="source">Genome Res.</div> , <div class="volume">22</div>, <div class="fpage">778</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Copy%20number%20variation%20of%20individual%20cattle%20genomes%20using%20next-generation%20sequencing&amp;author=D.M.%20Bickhart&amp;author=Y.%20Hou&amp;author=S.G.%20Schroeder&amp;publication_year=2012&amp;journal=Genome%20Res.&amp;volume=22&amp;pages=778-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.133967.111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.133967.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.133967.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22300768" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Durkin</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Coppieters</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Drogemuller</div> <div class="given-names">C.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Serial translocation by means of circular intermediates underlies colour sidedness in cattle</div>, <div class="source">Nature</div> , <div class="volume">482</div>, <div class="fpage">81</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Serial%20translocation%20by%20means%20of%20circular%20intermediates%20underlies%20colour%20sidedness%20in%20cattle&amp;author=K.%20Durkin&amp;author=W.%20Coppieters&amp;author=C.%20Drogemuller&amp;publication_year=2012&amp;journal=Nature&amp;volume=482&amp;pages=81-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10757" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10757" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10757"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22297974" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Conrad</div>
               <div class="given-names">D.F.</div>
            </div>, <div class="name string-name"><div class="surname">Hurles</div> <div class="given-names">M.E.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">The population genetics of structural variation</div>, <div class="source">Nat. Genet</div> , <div class="volume">39</div>, <div class="fpage">S30</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20population%20genetics%20of%20structural%20variation&amp;author=D.F.%20Conrad&amp;author=M.E.%20Hurles&amp;publication_year=2007&amp;journal=Nat.%20Genet&amp;volume=39&amp;pages=S30-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng2042" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng2042" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng2042"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17597779" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kato</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Kawaguchi</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Ishikawa</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Population-genetic nature of copy number variations in the human genome</div>, <div class="source">Hum. Mol. Genet.</div> , <div class="volume">19</div>, <div class="fpage">761</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population-genetic%20nature%20of%20copy%20number%20variations%20in%20the%20human%20genome&amp;author=M.%20Kato&amp;author=T.%20Kawaguchi&amp;author=S.%20Ishikawa&amp;publication_year=2010&amp;journal=Hum.%20Mol.%20Genet.&amp;volume=19&amp;pages=761-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/ddp541" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2Fddp541" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2Fddp541"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19966329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sudmant</div>
               <div class="given-names">P.H.</div>
            </div>, <div class="name string-name"><div class="surname">Mallick</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Nelson</div> <div class="given-names">B.J.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Global diversity, population stratification, and selection of human copy-number variation</div>, <div class="source">Science</div> , <div class="volume">349</div>, <div class="fpage">aab3761</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20diversity%2C%20population%20stratification%2C%20and%20selection%20of%20human%20copy-number%20variation&amp;author=P.H.%20Sudmant&amp;author=S.%20Mallick&amp;author=B.J.%20Nelson&amp;publication_year=2015&amp;journal=Science&amp;volume=349&amp;pages=aab3761" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.aab3761" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.aab3761" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.aab3761"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26249230" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Redon</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Ishikawa</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Fitch</div> <div class="given-names">K.R.</div></div>et al.  </div>. <div class="year">2006</div>, <div class="article-title">Global variation in copy number in the human genome</div>, <div class="source">Nature</div> , <div class="volume">444</div>, <div class="fpage">444</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20variation%20in%20copy%20number%20in%20the%20human%20genome&amp;author=R.%20Redon&amp;author=S.%20Ishikawa&amp;author=K.R.%20Fitch&amp;publication_year=2006&amp;journal=Nature&amp;volume=444&amp;pages=444-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature05329" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05329" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature05329"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17122850" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Conrad</div>
               <div class="given-names">D.F.</div>
            </div>, <div class="name string-name"><div class="surname">Pinto</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Redon</div> <div class="given-names">R.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">Origins and functional impact of copy number variation in the human genome</div>, <div class="source">Nature</div> , <div class="volume">464</div>, <div class="fpage">704</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origins%20and%20functional%20impact%20of%20copy%20number%20variation%20in%20the%20human%20genome&amp;author=D.F.%20Conrad&amp;author=D.%20Pinto&amp;author=R.%20Redon&amp;publication_year=2009&amp;journal=Nature&amp;volume=464&amp;pages=704-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08516" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08516" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08516"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19812545" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jakobsson</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Scholz</div> <div class="given-names">S.W.</div></div>, <div class="name string-name"><div class="surname">Scheet</div> <div class="given-names">P.</div></div>et al.  </div>. <div class="year">2008</div>, <div class="article-title">Genotype, haplotype and copy-number variation in worldwide human populations</div>, <div class="source">Nature</div> , <div class="volume">451</div>, <div class="fpage">998</div>–<div class="lpage">1003</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genotype%2C%20haplotype%20and%20copy-number%20variation%20in%20worldwide%20human%20populations&amp;author=M.%20Jakobsson&amp;author=S.W.%20Scholz&amp;author=P.%20Scheet&amp;publication_year=2008&amp;journal=Nature&amp;volume=451&amp;pages=998-1003" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06742" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06742" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06742"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18288195" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Iskow</div>
               <div class="given-names">R.C.</div>
            </div>, <div class="name string-name"><div class="surname">Gokcumen</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">C.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Exploring the role of copy number variants in human adaptation</div>, <div class="source">Trends Genet.</div> , <div class="volume">28</div>, <div class="fpage">245</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploring%20the%20role%20of%20copy%20number%20variants%20in%20human%20adaptation&amp;author=R.C.%20Iskow&amp;author=O.%20Gokcumen&amp;author=C.%20Lee&amp;publication_year=2012&amp;journal=Trends%20Genet.&amp;volume=28&amp;pages=245-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2012.03.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2012.03.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2012.03.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22483647" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Gu</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Hurles</div> <div class="given-names">M.E.</div></div>, <div class="name string-name"><div class="surname">Lupski</div> <div class="given-names">J.R.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Copy number variation in human health, disease, and evolution</div>, <div class="source">Annu. Rev. Genomics Hum. Genet.</div> , <div class="volume">10</div>, <div class="fpage">451</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Copy%20number%20variation%20in%20human%20health%2C%20disease%2C%20and%20evolution&amp;author=F.%20Zhang&amp;author=W.%20Gu&amp;author=M.E.%20Hurles&amp;author=J.R.%20Lupski&amp;publication_year=2009&amp;journal=Annu.%20Rev.%20Genomics%20Hum.%20Genet.&amp;volume=10&amp;pages=451-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genom.9.081307.164217" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genom.9.081307.164217" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genom.9.081307.164217"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19715442" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Perry</div>
               <div class="given-names">G.H.</div>
            </div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Marques-Bonet</div> <div class="given-names">T.</div></div>et al.  </div>. <div class="year">2008</div>, <div class="article-title">Copy number variation and evolution in humans and chimpanzees</div>, <div class="source">Genome Res.</div> , <div class="volume">18</div>, <div class="fpage">1698</div>–<div class="lpage">710</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Copy%20number%20variation%20and%20evolution%20in%20humans%20and%20chimpanzees&amp;author=G.H.%20Perry&amp;author=F.%20Yang&amp;author=T.%20Marques-Bonet&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=1698-710" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.082016.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.082016.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.082016.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18775914" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Paudel</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Madsen</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Megens</div> <div class="given-names">H.J.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Evolutionary dynamics of copy number variation in pig genomes in the context of adaptation and domestication</div>, <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">449</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20dynamics%20of%20copy%20number%20variation%20in%20pig%20genomes%20in%20the%20context%20of%20adaptation%20and%20domestication&amp;author=Y.%20Paudel&amp;author=O.%20Madsen&amp;author=H.J.%20Megens&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=449" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-449" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-449" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-449"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23829399" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Brown</div>
               <div class="given-names">K.H.</div>
            </div>, <div class="name string-name"><div class="surname">Dobrinski</div> <div class="given-names">K.P.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">A.S.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Extensive genetic diversity and substructuring among zebrafish strains revealed through copy number variant analysis</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">109</div>, <div class="fpage">529</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20genetic%20diversity%20and%20substructuring%20among%20zebrafish%20strains%20revealed%20through%20copy%20number%20variant%20analysis&amp;author=K.H.%20Brown&amp;author=K.P.%20Dobrinski&amp;author=A.S.%20Lee&amp;publication_year=2012&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=109&amp;pages=529-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1112163109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1112163109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1112163109"> </span></div> </div></div></div></div></div><div content-id="DSW013C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Berglund</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Nevalainen</div> <div class="given-names">E.M.</div></div>, <div class="name string-name"><div class="surname">Molin</div> <div class="given-names">A.M.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Novel origins of copy number variation in the dog genome</div>, <div class="source">Genome Biol.</div> , <div class="volume">13</div>, <div class="fpage">R73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20origins%20of%20copy%20number%20variation%20in%20the%20dog%20genome&amp;author=J.%20Berglund&amp;author=E.M.%20Nevalainen&amp;author=A.M.%20Molin&amp;publication_year=2012&amp;journal=Genome%20Biol.&amp;volume=13&amp;pages=R73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2012-13-8-r73" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2012-13-8-r73" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2012-13-8-r73"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22916802" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Perry</div>
               <div class="given-names">G.H.</div>
            </div>, <div class="name string-name"><div class="surname">Dominy</div> <div class="given-names">N.J.</div></div>, <div class="name string-name"><div class="surname">Claw</div> <div class="given-names">K.G.</div></div>et al.  </div>. <div class="year">2007</div>, <div class="article-title">Diet and the evolution of human amylase gene copy number variation</div>, <div class="source">Nat. Genet.</div> , <div class="volume">39</div>, <div class="fpage">1256</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diet%20and%20the%20evolution%20of%20human%20amylase%20gene%20copy%20number%20variation&amp;author=G.H.%20Perry&amp;author=N.J.%20Dominy&amp;author=K.G.%20Claw&amp;publication_year=2007&amp;journal=Nat.%20Genet.&amp;volume=39&amp;pages=1256-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng2123" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng2123" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng2123"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17828263" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kidd</div>
               <div class="given-names">J.M.</div>
            </div>, <div class="name string-name"><div class="surname">Newman</div> <div class="given-names">T.L.</div></div>, <div class="name string-name"><div class="surname">Tuzun</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Kaul</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Eichler</div> <div class="given-names">E.E.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Population stratification of a common APOBEC gene deletion polymorphism</div>, <div class="source">PLoS Genet.</div> , <div class="volume">3</div>, <div class="fpage">e63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20stratification%20of%20a%20common%20APOBEC%20gene%20deletion%20polymorphism&amp;author=J.M.%20Kidd&amp;author=T.L.%20Newman&amp;author=E.%20Tuzun&amp;author=R.%20Kaul&amp;author=E.E.%20Eichler&amp;publication_year=2007&amp;journal=PLoS%20Genet.&amp;volume=3&amp;pages=e63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.0030063" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.0030063" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.0030063"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17447845" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Xue</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Daly</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2008</div>, <div class="article-title">Adaptive evolution of UGT2B17 copy-number variation</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="volume">83</div>, <div class="fpage">337</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adaptive%20evolution%20of%20UGT2B17%20copy-number%20variation&amp;author=Y.%20Xue&amp;author=D.%20Sun&amp;author=A.%20Daly&amp;publication_year=2008&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=83&amp;pages=337-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ajhg.2008.08.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ajhg.2008.08.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ajhg.2008.08.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18760392" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Polimanti</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Piacentini</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Iorio</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Haplotype differences for copy number variants in the 22q11.23 region among human populations: a pigmentation-based model for selective pressure</div>, <div class="source">Eur. J. Hum. Genet.</div> , <div class="volume">23</div>, <div class="fpage">116</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Haplotype%20differences%20for%20copy%20number%20variants%20in%20the%2022q11.23%20region%20among%20human%20populations%3A%20a%20pigmentation-based%20model%20for%20selective%20pressure&amp;author=R.%20Polimanti&amp;author=S.%20Piacentini&amp;author=A.%20Iorio&amp;publication_year=2015&amp;journal=Eur.%20J.%20Hum.%20Genet.&amp;volume=23&amp;pages=116-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ejhg.2014.47" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fejhg.2014.47" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fejhg.2014.47"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24667780" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">McCarroll</div>
               <div class="given-names">S.A.</div>
            </div>, <div class="name string-name"><div class="surname">Hadnott</div> <div class="given-names">T.N.</div></div>, <div class="name string-name"><div class="surname">Perry</div> <div class="given-names">G.H.</div></div>et al.  </div>. <div class="year">2006</div>, <div class="article-title">Common deletion polymorphisms in the human genome</div>, <div class="source">Nat. Genet.</div> , <div class="volume">38</div>, <div class="fpage">86</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Common%20deletion%20polymorphisms%20in%20the%20human%20genome&amp;author=S.A.%20McCarroll&amp;author=T.N.%20Hadnott&amp;author=G.H.%20Perry&amp;publication_year=2006&amp;journal=Nat.%20Genet.&amp;volume=38&amp;pages=86-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1696" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1696" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1696"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16468122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Xu</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Cole</div> <div class="given-names">J.B.</div></div>, <div class="name string-name"><div class="surname">Bickhart</div> <div class="given-names">D.M.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Genome wide CNV analysis reveals additional variants associated with milk production traits in Holsteins</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">683</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20wide%20CNV%20analysis%20reveals%20additional%20variants%20associated%20with%20milk%20production%20traits%20in%20Holsteins&amp;author=L.%20Xu&amp;author=J.B.%20Cole&amp;author=D.M.%20Bickhart&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=683" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-683" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25128478" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bradley</div>
               <div class="given-names">D.G.</div>
            </div>, <div class="name string-name"><div class="surname">Machugh</div> <div class="given-names">D.E.</div></div>, <div class="name string-name"><div class="surname">Cunningham</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Loftus</div> <div class="given-names">R.T.</div></div></div>
          <div class="year">1996</div>, <div class="article-title">Mitochondrial diversity and the origins of African and European cattle</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">93</div>, <div class="fpage">5131</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mitochondrial%20diversity%20and%20the%20origins%20of%20African%20and%20European%20cattle&amp;author=D.G.%20Bradley&amp;author=D.E.%20Machugh&amp;author=P.%20Cunningham&amp;author=R.T.%20Loftus&amp;publication_year=1996&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=93&amp;pages=5131-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.93.10.5131" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.93.10.5131" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.93.10.5131"> </span></div> </div></div></div></div></div><div content-id="DSW013C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="collab">The Bovine HapMap Consortium</div>
          <div class="year">2009</div>, <div class="article-title">Genome wide survey of SNP variation uncovers the genetic structure of cattle breeds</div>, <div class="source">Science</div> , <div class="volume">324</div>, <div class="fpage">528</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1167936" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1167936" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1167936"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19390050" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">MacHugh</div>
               <div class="given-names">D.E.</div>
            </div>, <div class="name string-name"><div class="surname">Shriver</div> <div class="given-names">M.D.</div></div>, <div class="name string-name"><div class="surname">Loftus</div> <div class="given-names">R.T.</div></div>, <div class="name string-name"><div class="surname">Cunningham</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Bradley</div> <div class="given-names">D.G.</div></div></div>
          <div class="year">1997</div>, <div class="article-title">Microsatellite DNA variation and the evolution, domestication and phylogeography of taurine and zebu cattle (<em>Bos taurus</em> and <em>Bos indicus</em>)</div>, <div class="source">Genetics</div> , <div class="volume">146</div>, <div class="fpage">1071</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellite%20DNA%20variation%20and%20the%20evolution%2C%20domestication%20and%20phylogeography%20of%20taurine%20and%20zebu%20cattle%20(Bos%20taurus%20and%20Bos%20indicus)&amp;author=D.E.%20MacHugh&amp;author=M.D.%20Shriver&amp;author=R.T.%20Loftus&amp;author=P.%20Cunningham&amp;author=D.G.%20Bradley&amp;publication_year=1997&amp;journal=Genetics&amp;volume=146&amp;pages=1071-86" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9215909" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hiendleder</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Lewalski</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Janke</div> <div class="given-names">A.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Complete mitochondrial genomes of <em>Bos taurus</em> and <em>Bos indicus</em> provide new insights into intra-species variation, taxonomy and domestication</div>, <div class="source">Cytogenet. Genome Res.</div> , <div class="volume">120</div>, <div class="fpage">150</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20mitochondrial%20genomes%20of%20Bos%20taurus%20and%20Bos%20indicus%20provide%20new%20insights%20into%20intra-species%20variation%2C%20taxonomy%20and%20domestication&amp;author=S.%20Hiendleder&amp;author=H.%20Lewalski&amp;author=A.%20Janke&amp;publication_year=2008&amp;journal=Cytogenet.%20Genome%20Res.&amp;volume=120&amp;pages=150-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000118756" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000118756" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000118756"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18467841" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Loftus</div>
               <div class="given-names">R.T.</div>
            </div>, <div class="name string-name"><div class="surname">MacHugh</div> <div class="given-names">D.E.</div></div>, <div class="name string-name"><div class="surname">Bradley</div> <div class="given-names">D.G.</div></div>, <div class="name string-name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div>, <div class="name string-name"><div class="surname">Cunningham</div> <div class="given-names">P.</div></div></div>
          <div class="year">1994</div>, <div class="article-title">Evidence for two independent domestications of cattle</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">91</div>, <div class="fpage">2757</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20two%20independent%20domestications%20of%20cattle&amp;author=R.T.%20Loftus&amp;author=D.E.%20MacHugh&amp;author=D.G.%20Bradley&amp;author=P.M.%20Sharp&amp;author=P.%20Cunningham&amp;publication_year=1994&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=91&amp;pages=2757-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.91.7.2757" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.91.7.2757" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.91.7.2757"> </span></div> </div></div></div></div></div><div content-id="DSW013C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Larson</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Piperno</div> <div class="given-names">D.R.</div></div>, <div class="name string-name"><div class="surname">Allaby</div> <div class="given-names">R.G.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Current perspectives and the future of domestication studies</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">111</div>, <div class="fpage">6139</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Current%20perspectives%20and%20the%20future%20of%20domestication%20studies&amp;author=G.%20Larson&amp;author=D.R.%20Piperno&amp;author=R.G.%20Allaby&amp;publication_year=2014&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=111&amp;pages=6139-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1323964111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1323964111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1323964111"> </span></div> </div></div></div></div></div><div content-id="DSW013C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Troy</div>
               <div class="given-names">C.S.</div>
            </div>, <div class="name string-name"><div class="surname">MacHugh</div> <div class="given-names">D.E.</div></div>, <div class="name string-name"><div class="surname">Bailey</div> <div class="given-names">J.F.</div></div>et al.  </div>. <div class="year">2001</div>, <div class="article-title">Genetic evidence for Near-Eastern origins of European cattle</div>, <div class="source">Nature</div> , <div class="volume">410</div>, <div class="fpage">1088</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20evidence%20for%20Near-Eastern%20origins%20of%20European%20cattle&amp;author=C.S.%20Troy&amp;author=D.E.%20MacHugh&amp;author=J.F.%20Bailey&amp;publication_year=2001&amp;journal=Nature&amp;volume=410&amp;pages=1088-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35074088" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35074088" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35074088"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11323670" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hanotte</div>
               <div class="given-names">O.</div>
            </div>, <div class="name string-name"><div class="surname">Bradley</div> <div class="given-names">D.G.</div></div>, <div class="name string-name"><div class="surname">Ochieng</div> <div class="given-names">J.W.</div></div>, <div class="name string-name"><div class="surname">Verjee</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Hill</div> <div class="given-names">E.W.</div></div>, <div class="name string-name"><div class="surname">Rege</div> <div class="given-names">J.E.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">African pastoralism: genetic imprints of origins and migrations</div>, <div class="source">Science</div> , <div class="volume">296</div>, <div class="fpage">336</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=African%20pastoralism%3A%20genetic%20imprints%20of%20origins%20and%20migrations&amp;author=O.%20Hanotte&amp;author=D.G.%20Bradley&amp;author=J.W.%20Ochieng&amp;author=Y.%20Verjee&amp;author=E.W.%20Hill&amp;author=J.E.%20Rege&amp;publication_year=2002&amp;journal=Science&amp;volume=296&amp;pages=336-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1069878" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1069878" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1069878"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11951043" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Decker</div>
               <div class="given-names">J.E.</div>
            </div>, <div class="name string-name"><div class="surname">McKay</div> <div class="given-names">S.D.</div></div>, <div class="name string-name"><div class="surname">Rolf</div> <div class="given-names">M.M.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Worldwide patterns of ancestry, divergence, and admixture in domesticated cattle</div>, <div class="source">PLoS Genet.</div> , <div class="volume">10</div>, <div class="fpage">e1004254</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Worldwide%20patterns%20of%20ancestry%2C%20divergence%2C%20and%20admixture%20in%20domesticated%20cattle&amp;author=J.E.%20Decker&amp;author=S.D.%20McKay&amp;author=M.M.%20Rolf&amp;publication_year=2014&amp;journal=PLoS%20Genet.&amp;volume=10&amp;pages=e1004254" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1004254" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1004254" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1004254"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24675901" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Andersson</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Georges</div> <div class="given-names">M.</div></div></div>
          <div class="year">2004</div>, <div class="article-title">Domestic-animal genomics: deciphering the genetics of complex traits</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">5</div>, <div class="fpage">202</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Domestic-animal%20genomics%3A%20deciphering%20the%20genetics%20of%20complex%20traits&amp;author=L.%20Andersson&amp;author=M.%20Georges&amp;publication_year=2004&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=5&amp;pages=202-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1294" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1294" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1294"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14970822" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">VanRaden</div>
               <div class="given-names">P.M.</div>
            </div>, <div class="name string-name"><div class="surname">Van Tassell</div> <div class="given-names">C.P.</div></div>, <div class="name string-name"><div class="surname">Wiggans</div> <div class="given-names">G.R.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">Invited review: reliability of genomic predictions for North American Holstein bulls</div>, <div class="source">J. Dairy Sci.</div> , <div class="volume">92</div>, <div class="fpage">16</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Invited%20review%3A%20reliability%20of%20genomic%20predictions%20for%20North%20American%20Holstein%20bulls&amp;author=P.M.%20VanRaden&amp;author=C.P.%20Van%20Tassell&amp;author=G.R.%20Wiggans&amp;publication_year=2009&amp;journal=J.%20Dairy%20Sci.&amp;volume=92&amp;pages=16-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3168/jds.2008-1514" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3168%2Fjds.2008-1514" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3168%2Fjds.2008-1514"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19109259" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Porto-Neto</div>
               <div class="given-names">L.R.</div>
            </div>, <div class="name string-name"><div class="surname">Sonstegard</div> <div class="given-names">T.S.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">G.E.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Genomic divergence of zebu and taurine cattle identified through high-density SNP genotyping</div>, <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">876</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20divergence%20of%20zebu%20and%20taurine%20cattle%20identified%20through%20high-density%20SNP%20genotyping&amp;author=L.R.%20Porto-Neto&amp;author=T.S.%20Sonstegard&amp;author=G.E.%20Liu&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=876" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-876" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-876" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-876"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24330634" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Alkan</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Kidd</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Marques-Bonet</div> <div class="given-names">T.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">Personalized copy number and segmental duplication maps using next-generation sequencing</div>, <div class="source">Nat. Genet.</div> , <div class="volume">41</div>, <div class="fpage">1061</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Personalized%20copy%20number%20and%20segmental%20duplication%20maps%20using%20next-generation%20sequencing&amp;author=C.%20Alkan&amp;author=J.M.%20Kidd&amp;author=T.%20Marques-Bonet&amp;publication_year=2009&amp;journal=Nat.%20Genet.&amp;volume=41&amp;pages=1061-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.437" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.437" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.437"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19718026" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kim</div>
               <div class="given-names">P.M.</div>
            </div>, <div class="name string-name"><div class="surname">Lam</div> <div class="given-names">H.Y.</div></div>, <div class="name string-name"><div class="surname">Urban</div> <div class="given-names">A.E.</div></div>et al.  </div>. <div class="year">2008</div>, <div class="article-title">Analysis of copy number variants and segmental duplications in the human genome: evidence for a change in the process of formation in recent evolutionary history</div>, <div class="source">Genome Res.</div> , <div class="volume">18</div>, <div class="fpage">1865</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20copy%20number%20variants%20and%20segmental%20duplications%20in%20the%20human%20genome%3A%20evidence%20for%20a%20change%20in%20the%20process%20of%20formation%20in%20recent%20evolutionary%20history&amp;author=P.M.%20Kim&amp;author=H.Y.%20Lam&amp;author=A.E.%20Urban&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=1865-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.081422.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.081422.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.081422.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18842824" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Reynolds</div>
               <div class="given-names">A.P.</div>
            </div>, <div class="name string-name"><div class="surname">Richards</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">de la Iglesia</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Rayward-Smith</div> <div class="given-names">V.J.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">Clustering rules: a comparison of partitioning and hierarchical clustering algorithms</div>, <div class="source">J. Math. Model. Algor.</div> , <div class="volume">5</div>, <div class="fpage">475</div>–<div class="lpage">504</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Clustering%20rules%3A%20a%20comparison%20of%20partitioning%20and%20hierarchical%20clustering%20algorithms&amp;author=A.P.%20Reynolds&amp;author=G.%20Richards&amp;author=B.%20de%20la%20Iglesia&amp;author=V.J.%20Rayward-Smith&amp;publication_year=2006&amp;journal=J.%20Math.%20Model.%20Algor.&amp;volume=5&amp;pages=475-504" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10852-005-9022-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10852-005-9022-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10852-005-9022-1"> </span></div> </div></div></div></div></div><div content-id="DSW013C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Falush</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Stephens</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pritchard</div> <div class="given-names">J.K.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">Inference of population structure using multilocus genotype data: linked loci and correlated allele frequencies</div>, <div class="source">Genetics</div> , <div class="volume">164</div>, <div class="fpage">1567</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data%3A%20linked%20loci%20and%20correlated%20allele%20frequencies&amp;author=D.%20Falush&amp;author=M.%20Stephens&amp;author=J.K.%20Pritchard&amp;publication_year=2003&amp;journal=Genetics&amp;volume=164&amp;pages=1567-87" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12930761" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pritchard</div>
               <div class="given-names">J.K.</div>
            </div>, <div class="name string-name"><div class="surname">Stephens</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Donnelly</div> <div class="given-names">P.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Inference of population structure using multilocus genotype data</div>, <div class="source">Genetics</div> , <div class="volume">155</div>, <div class="fpage">945</div>–<div class="lpage">59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inference%20of%20population%20structure%20using%20multilocus%20genotype%20data&amp;author=J.K.%20Pritchard&amp;author=M.%20Stephens&amp;author=P.%20Donnelly&amp;publication_year=2000&amp;journal=Genetics&amp;volume=155&amp;pages=945-59" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10835412" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hou</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Bickhart</div> <div class="given-names">D.M.</div></div>, <div class="name string-name"><div class="surname">Hvinden</div> <div class="given-names">M.L.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Fine mapping of copy number variations on two cattle genome assemblies using high density SNP array</div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">376</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fine%20mapping%20of%20copy%20number%20variations%20on%20two%20cattle%20genome%20assemblies%20using%20high%20density%20SNP%20array&amp;author=Y.%20Hou&amp;author=D.M.%20Bickhart&amp;author=M.L.%20Hvinden&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=376" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22866901" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stephens</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Smith</div> <div class="given-names">N.J.</div></div>, <div class="name string-name"><div class="surname">Donnelly</div> <div class="given-names">P.</div></div></div>
          <div class="year">2001</div>, <div class="article-title">A new statistical method for haplotype reconstruction from population data</div>, <div class="source">Am. J. Hum. Genet.</div> , <div class="volume">68</div>, <div class="fpage">978</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20statistical%20method%20for%20haplotype%20reconstruction%20from%20population%20data&amp;author=M.%20Stephens&amp;author=N.J.%20Smith&amp;author=P.%20Donnelly&amp;publication_year=2001&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;volume=68&amp;pages=978-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/319501" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F319501" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F319501"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11254454" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Crawford</div>
               <div class="given-names">D.C.</div>
            </div>, <div class="name string-name"><div class="surname">Bhangale</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">N.</div></div>et al.  </div>. <div class="year">2004</div>, <div class="article-title">Evidence for substantial fine-scale variation in recombination rates across the human genome</div>, <div class="source">Nat. Genet.</div> , <div class="volume">36</div>, <div class="fpage">700</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20substantial%20fine-scale%20variation%20in%20recombination%20rates%20across%20the%20human%20genome&amp;author=D.C.%20Crawford&amp;author=T.%20Bhangale&amp;author=N.%20Li&amp;publication_year=2004&amp;journal=Nat.%20Genet.&amp;volume=36&amp;pages=700-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15184900" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">G.E.</div>
            </div>, <div class="name string-name"><div class="surname">Ventura</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Cellamare</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">Analysis of recent segmental duplications in the bovine genome</div>, <div class="source">BMC Genomics</div> , <div class="volume">10</div>, <div class="fpage">571</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20recent%20segmental%20duplications%20in%20the%20bovine%20genome&amp;author=G.E.%20Liu&amp;author=M.%20Ventura&amp;author=A.%20Cellamare&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=571" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-571" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-571" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-571"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19951423" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Handsaker</div>
               <div class="given-names">R.E.</div>
            </div>, <div class="name string-name"><div class="surname">Van Doren</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Berman</div> <div class="given-names">J.R.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Large multiallelic copy number variations in humans</div>, <div class="source">Nat. Genet.</div> , <div class="volume">47</div>, <div class="fpage">296</div>–<div class="lpage">303</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large%20multiallelic%20copy%20number%20variations%20in%20humans&amp;author=R.E.%20Handsaker&amp;author=V.%20Van%20Doren&amp;author=J.R.%20Berman&amp;publication_year=2015&amp;journal=Nat.%20Genet.&amp;volume=47&amp;pages=296-303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.3200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.3200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.3200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25621458" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Brahmachary</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Guilmatre</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Quilez</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Digital genotyping of macrosatellites and multicopy genes reveals novel biological functions associated with copy number variation of large tandem repeats</div>, <div class="source">PLoS Genet.</div> , <div class="volume">10</div>, <div class="fpage">e1004418</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Digital%20genotyping%20of%20macrosatellites%20and%20multicopy%20genes%20reveals%20novel%20biological%20functions%20associated%20with%20copy%20number%20variation%20of%20large%20tandem%20repeats&amp;author=M.%20Brahmachary&amp;author=A.%20Guilmatre&amp;author=J.%20Quilez&amp;publication_year=2014&amp;journal=PLoS%20Genet.&amp;volume=10&amp;pages=e1004418" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1004418" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1004418" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1004418"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24945355" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Willems</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Gymrek</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Highnam</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Mittelman</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Erlich</div> <div class="given-names">Y.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">The landscape of human STR variation</div>, <div class="source">Genome Res.</div> , <div class="volume">24</div>, <div class="fpage">1894</div>–<div class="lpage">904</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20landscape%20of%20human%20STR%20variation&amp;author=T.%20Willems&amp;author=M.%20Gymrek&amp;author=G.%20Highnam&amp;author=D.%20Mittelman&amp;author=Y.%20Erlich&amp;publication_year=2014&amp;journal=Genome%20Res.&amp;volume=24&amp;pages=1894-904" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.177774.114" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.177774.114" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.177774.114"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25135957" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Carlson</div>
               <div class="given-names">K.D.</div>
            </div>, <div class="name string-name"><div class="surname">Sudmant</div> <div class="given-names">P.H.</div></div>, <div class="name string-name"><div class="surname">Press</div> <div class="given-names">M.O.</div></div>, <div class="name string-name"><div class="surname">Eichler</div> <div class="given-names">E.E.</div></div>, <div class="name string-name"><div class="surname">Shendure</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Queitsch</div> <div class="given-names">C.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">MIPSTR: a method for multiplex genotyping of germline and somatic STR variation across many individuals</div>, <div class="source">Genome Res.</div> , <div class="volume">25</div>, <div class="fpage">750</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MIPSTR%3A%20a%20method%20for%20multiplex%20genotyping%20of%20germline%20and%20somatic%20STR%20variation%20across%20many%20individuals&amp;author=K.D.%20Carlson&amp;author=P.H.%20Sudmant&amp;author=M.O.%20Press&amp;author=E.E.%20Eichler&amp;author=J.%20Shendure&amp;author=C.%20Queitsch&amp;publication_year=2015&amp;journal=Genome%20Res.&amp;volume=25&amp;pages=750-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.182212.114" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.182212.114" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.182212.114"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25659649" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fungtammasan</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Ananda</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Hile</div> <div class="given-names">S.E.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Accurate typing of short tandem repeats from genome-wide sequencing data and its applications</div>, <div class="source">Genome Res.</div> , <div class="volume">25</div>, <div class="fpage">736</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accurate%20typing%20of%20short%20tandem%20repeats%20from%20genome-wide%20sequencing%20data%20and%20its%20applications&amp;author=A.%20Fungtammasan&amp;author=G.%20Ananda&amp;author=S.E.%20Hile&amp;publication_year=2015&amp;journal=Genome%20Res.&amp;volume=25&amp;pages=736-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.185892.114" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.185892.114" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.185892.114"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25823460" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schrider</div>
               <div class="given-names">D.R.</div>
            </div>, <div class="name string-name"><div class="surname">Hahn</div> <div class="given-names">M.W.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Lower linkage disequilibrium at CNVs is due to both recurrent mutation and transposing duplications</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">27</div>, <div class="fpage">103</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lower%20linkage%20disequilibrium%20at%20CNVs%20is%20due%20to%20both%20recurrent%20mutation%20and%20transposing%20duplications&amp;author=D.R.%20Schrider&amp;author=M.W.%20Hahn&amp;publication_year=2010&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=27&amp;pages=103-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msp210" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsp210" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsp210"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19745000" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Huang</div>
               <div class="given-names">D.W.</div>
            </div>, <div class="name string-name"><div class="surname">Sherman</div> <div class="given-names">B.T.</div></div>, <div class="name string-name"><div class="surname">Lempicki</div> <div class="given-names">R.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources</div>, <div class="source">Nat. Protoc.</div> , <div class="volume">4</div>, <div class="fpage">44</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20and%20integrative%20analysis%20of%20large%20gene%20lists%20using%20DAVID%20bioinformatics%20resources&amp;author=D.W.%20Huang&amp;author=B.T.%20Sherman&amp;author=R.%20Lempicki&amp;publication_year=2009&amp;journal=Nat.%20Protoc.&amp;volume=4&amp;pages=44-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2008.211" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2008.211" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2008.211"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19131956" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jacinto</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Loewith</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Schmidt</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2004</div>, <div class="article-title">Mammalian TOR complex 2 controls the actin cytoskeleton and is rapamycin insensitive</div>, <div class="source">Nat. Cell Biol.</div> , <div class="volume">6</div>, <div class="fpage">1122</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mammalian%20TOR%20complex%202%20controls%20the%20actin%20cytoskeleton%20and%20is%20rapamycin%20insensitive&amp;author=E.%20Jacinto&amp;author=R.%20Loewith&amp;author=A.%20Schmidt&amp;publication_year=2004&amp;journal=Nat.%20Cell%20Biol.&amp;volume=6&amp;pages=1122-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncb1183" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncb1183" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncb1183"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15467718" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jerome-Majewska</div>
               <div class="given-names">L.A.</div>
            </div>, <div class="name string-name"><div class="surname">Achkar</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Luo</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Lupu</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Lacy</div> <div class="given-names">E.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">The trafficking protein Tmed2/p24β(1) is required for morphogenesis of the mouse embryo and placenta</div>, <div class="source">Dev. Biol.</div> , <div class="volume">341</div>, <div class="fpage">154</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20trafficking%20protein%20Tmed2%2Fp24%CE%B2(1)%20is%20required%20for%20morphogenesis%20of%20the%20mouse%20embryo%20and%20placenta&amp;author=L.A.%20Jerome-Majewska&amp;author=T.%20Achkar&amp;author=L.%20Luo&amp;author=F.%20Lupu&amp;author=E.%20Lacy&amp;publication_year=2010&amp;journal=Dev.%20Biol.&amp;volume=341&amp;pages=154-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ydbio.2010.02.019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ydbio.2010.02.019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ydbio.2010.02.019"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20178780" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">J.Z.</div>
            </div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Karaman</div> <div class="given-names">R.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Chronic overexpression of PNPLA3I148M in mouse liver causes hepatic steatosis</div>, <div class="source">J. Clin. Invest.</div> , <div class="volume">122</div>, <div class="fpage">4130</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chronic%20overexpression%20of%20PNPLA3I148M%20in%20mouse%20liver%20causes%20hepatic%20steatosis&amp;author=J.Z.%20Li&amp;author=Y.%20Huang&amp;author=R.%20Karaman&amp;publication_year=2012&amp;journal=J.%20Clin.%20Invest.&amp;volume=122&amp;pages=4130-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1172/JCI65179" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1172%2FJCI65179" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1172%2FJCI65179"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23023705" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nei</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Rooney</div> <div class="given-names">A.P.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">Concerted and birth-and-death evolution of multigene families</div>, <div class="source">Annu. Rev. Genet.</div> , <div class="volume">39</div>, <div class="fpage">121</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Concerted%20and%20birth-and-death%20evolution%20of%20multigene%20families&amp;author=M.%20Nei&amp;author=A.P.%20Rooney&amp;publication_year=2005&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=39&amp;pages=121-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.39.073003.112240" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.39.073003.112240" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.39.073003.112240"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16285855" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Alonso</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Lopez</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Izagirre</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">de la Rúa</div> <div class="given-names">C.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Overdominance in the human genome and olfactory receptor activity</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="volume">25</div>, <div class="fpage">997</div>–<div class="lpage">1001</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Overdominance%20in%20the%20human%20genome%20and%20olfactory%20receptor%20activity&amp;author=S.%20Alonso&amp;author=S.%20Lopez&amp;author=N.%20Izagirre&amp;author=C.%20de%20la%20R%C3%BAa&amp;publication_year=2008&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=25&amp;pages=997-1001" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msn049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsn049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsn049"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18296703" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Malnic</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Hirono</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Sato</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Buck</div> <div class="given-names">L.B.</div></div></div>
          <div class="year">1999</div>, <div class="article-title">Combinatorial receptor codes for odors</div>, <div class="source">Cell</div> , <div class="volume">96</div>, <div class="fpage">713</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Combinatorial%20receptor%20codes%20for%20odors&amp;author=B.%20Malnic&amp;author=J.%20Hirono&amp;author=T.%20Sato&amp;author=L.B.%20Buck&amp;publication_year=1999&amp;journal=Cell&amp;volume=96&amp;pages=713-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0092-8674(00)80581-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0092-8674(00)80581-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0092-8674(00)80581-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10089886" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cicconardi</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Chillemi</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Tramontano</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Massive screening of copy number population-scale variation in Bos taurus genome</div>, <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">124</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Massive%20screening%20of%20copy%20number%20population-scale%20variation%20in%20Bos%20taurus%20genome&amp;author=F.%20Cicconardi&amp;author=G.%20Chillemi&amp;author=A.%20Tramontano&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=124" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-124" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-124" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-124"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23442185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kadri</div>
               <div class="given-names">N.K.</div>
            </div>, <div class="name string-name"><div class="surname">Sahana</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Charlier</div> <div class="given-names">C.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">A 660-Kb deletion with antagonistic effects on fertility and milk production segregates at high frequency in Nordic Red cattle: additional evidence for the common occurrence of balancing selection in livestock</div>, <div class="source">PLoS Genet.</div> , <div class="volume">10</div>, <div class="fpage">e1004049</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20660-Kb%20deletion%20with%20antagonistic%20effects%20on%20fertility%20and%20milk%20production%20segregates%20at%20high%20frequency%20in%20Nordic%20Red%20cattle%3A%20additional%20evidence%20for%20the%20common%20occurrence%20of%20balancing%20selection%20in%20livestock&amp;author=N.K.%20Kadri&amp;author=G.%20Sahana&amp;author=C.%20Charlier&amp;publication_year=2014&amp;journal=PLoS%20Genet.&amp;volume=10&amp;pages=e1004049" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1004049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1004049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1004049"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24391517" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Teshima</div>
               <div class="given-names">K.M.</div>
            </div>, <div class="name string-name"><div class="surname">Innan</div> <div class="given-names">H.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">The coalescent with selection on copy number variants</div>, <div class="source">Genetics</div> , <div class="volume">190</div>, <div class="fpage">1077</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20coalescent%20with%20selection%20on%20copy%20number%20variants&amp;author=K.M.%20Teshima&amp;author=H.%20Innan&amp;publication_year=2012&amp;journal=Genetics&amp;volume=190&amp;pages=1077-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.111.135343" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.111.135343" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.111.135343"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22174068" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Haasl</div>
               <div class="given-names">R.J.</div>
            </div>, <div class="name string-name"><div class="surname">Payseur</div> <div class="given-names">B.A.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Multi-locus inference of population structure: a comparison between single nucleotide polymorphisms and microsatellites</div>, <div class="source">Heredity (Edinb.)</div> , <div class="volume">106</div>, <div class="fpage">158</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multi-locus%20inference%20of%20population%20structure%3A%20a%20comparison%20between%20single%20nucleotide%20polymorphisms%20and%20microsatellites&amp;author=R.J.%20Haasl&amp;author=B.A.%20Payseur&amp;publication_year=2011&amp;journal=Heredity%20(Edinb.)&amp;volume=106&amp;pages=158-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/hdy.2010.21" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhdy.2010.21" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fhdy.2010.21"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20332809" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Putman</div>
               <div class="given-names">A.I.</div>
            </div>, <div class="name string-name"><div class="surname">Carbone</div> <div class="given-names">I.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Challenges in analysis and interpretation of microsatellite data for population genetic studies</div>, <div class="source">Ecol. Evol.</div> , <div class="volume">4</div>, <div class="fpage">4399</div>–<div class="lpage">428</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Challenges%20in%20analysis%20and%20interpretation%20of%20microsatellite%20data%20for%20population%20genetic%20studies&amp;author=A.I.%20Putman&amp;author=I.%20Carbone&amp;publication_year=2014&amp;journal=Ecol.%20Evol.&amp;volume=4&amp;pages=4399-428" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25540699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW013C75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW013C75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Daetwyler</div>
               <div class="given-names">H.D.</div>
            </div>, <div class="name string-name"><div class="surname">Capitan</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Pausch</div> <div class="given-names">H.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Whole-genome sequencing of 234 bulls facilitates mapping of monogenic and complex traits in cattle</div>, <div class="source">Nat. Genet.</div> , <div class="volume">46</div>, <div class="fpage">858</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20sequencing%20of%20234%20bulls%20facilitates%20mapping%20of%20monogenic%20and%20complex%20traits%20in%20cattle&amp;author=H.D.%20Daetwyler&amp;author=A.%20Capitan&amp;author=H.%20Pausch&amp;publication_year=2014&amp;journal=Nat.%20Genet.&amp;volume=46&amp;pages=858-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.3034" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.3034" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.3034"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25017103" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">Published by Oxford University Press on behalf of Kazusa DNA Research Institute 2016. This work is written by (a) US Government employee(s) and is in the public domain in the US.</div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=1745562&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/3/10.1093_dnares_dsw013/1/dsw013_Supplementary_Data.zip?Expires=2147483647&amp;Signature=L0XM7g-6~z0T6IEfjfMINM1K4KfCQiIJ3CWqbjFUqfGhfuG7dCmyzN6~QYIwnG1pBE0d4JIhuZ9x8jd3eG4zp-KsOYL2aiw7KCxMw38qN4eBaKW9OfZo1HqXcCgiFy8u62DIibanmpZwB02weA218psrXQWJr~AfS4ZnzOxvPfak2KAjWCBpooidhD09y9~VB1M73tQhfeMPEGq2Wa~5aHxAW83ziT73r8ZvBMzWyF~pXwLqyUH8~mrIYD4JR2DwqCM-z6YUeRdl9yEZ-qmojll4DX6i04iVwL8Jf~6bjHCqE7J1SQ-E9IJtLAICpwbUsgGBkYE8qNxTEaVUpwESAw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">947</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000379741500007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">14</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw013"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="1745562"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000379741500007&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f3%2f253%2f1745562">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000379741500007&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (14)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F3%2F253%2F1745562" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/1745562'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Diversity and population-genetic properties of copy number variations and multicopy genes in cattle" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw013");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f3%2f253%2f1745562">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

