

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/13/6/255/464686","@type":"ScholarlyArticle","name":"Positional Cloning Identifies  Lotus japonicus  NSP2, A Putative Transcription Factor of the GRAS Family, Required for  NIN  and  ENOD40  Gene Expression in Nodule Initiation","datePublished":"2006-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/13/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2006-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsl017","keywords":["positional cloning","<italic>Lotus japonicus</italic>","transcription factor","nodule initiation"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/13/6/255/464686","author":[{"name":"Murakami, Yasuhiro","affiliation":"1  National Institute of Agrobiological Sciences, 2-1-2 Kannondai  Tsukuba, Ibaraki 305-8602, Japan    3  Department of Biological Sciences, Graduate School of Science, The University of Tokyo  7-3-1 Hongo, Bunkyo, Tokyo 113-0033, Japan    4  Bio-oriented Technology Research Advancement Institution (BRAIN), Tokyo Office  3-18-19 Toranomon, Minato-ku, Tokyo 105-0001, Japan","@type":"Person"},{"name":"Miwa, Hiroki","affiliation":"2  John Innes Centre  Colney Lane, Norwich NR4 7UH, UK","@type":"Person"},{"name":"Imaizumi-Anraku, Haruko","affiliation":"1  National Institute of Agrobiological Sciences, 2-1-2 Kannondai  Tsukuba, Ibaraki 305-8602, Japan    4  Bio-oriented Technology Research Advancement Institution (BRAIN), Tokyo Office  3-18-19 Toranomon, Minato-ku, Tokyo 105-0001, Japan","@type":"Person"},{"name":"Kouchi, Hiroshi","affiliation":"1  National Institute of Agrobiological Sciences, 2-1-2 Kannondai  Tsukuba, Ibaraki 305-8602, Japan","@type":"Person"},{"name":"Downie, J. Allan","affiliation":"2  John Innes Centre  Colney Lane, Norwich NR4 7UH, UK","@type":"Person"},{"name":"Kawaguchi, Masayoshi","affiliation":"3  Department of Biological Sciences, Graduate School of Science, The University of Tokyo  7-3-1 Hongo, Bunkyo, Tokyo 113-0033, Japan","@type":"Person"},{"name":"Kawasaki, Shinji","affiliation":"1  National Institute of Agrobiological Sciences, 2-1-2 Kannondai  Tsukuba, Ibaraki 305-8602, Japan    4  Bio-oriented Technology Research Advancement Institution (BRAIN), Tokyo Office  3-18-19 Toranomon, Minato-ku, Tokyo 105-0001, Japan","@type":"Person"}],"description":"Abstract.  Rhizobia-secreted Nod-factors (NFs) are required for nodulation. In the early developmental process of nodulation, a large number of changes occur in","pageStart":"255","pageEnd":"265","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/6/0/m_dnaresearch13_6.cover.gif?Expires=2147483647&Signature=fcRvTZS-Esu1iTIRpBwNtJBvy2AJRBCdQJyTXwh60M7JIQlqTrbnOTGA7L-7i6L4UYLG4JUSar16AKIY53IuiLWx-2mOh3JMXxdjEKlHYOAqNyuibso-D4bG3vOwDPxfhcZLg45cv2YcwYaGQxhzHrL5TjKyhZ4mwYb7GjG0uGA0KhpFZicZMiPYJGB~oynsMfd94bPAIrtQQPPmGzVGv0FtPTWthom~pPxQD8WgJhn5dGBE8EuXCW7bGuh87zOMErdi4RM6l1hIpsCmjH99FUAMlBlnn1Obqs6yCaDyJuJdsESPq4-uwP4YJIZZyz6U843ECQydjMQgSO4nkuzNbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Positional Cloning Identifies  Lotus japonicus  NSP2, A Putative Transcription Factor of the GRAS Family, Required for  NIN  and  ENOD40  Gene Expression in Nodule Initiation","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/6/0/m_dnaresearch13_6.cover.gif?Expires=2147483647&Signature=fcRvTZS-Esu1iTIRpBwNtJBvy2AJRBCdQJyTXwh60M7JIQlqTrbnOTGA7L-7i6L4UYLG4JUSar16AKIY53IuiLWx-2mOh3JMXxdjEKlHYOAqNyuibso-D4bG3vOwDPxfhcZLg45cv2YcwYaGQxhzHrL5TjKyhZ4mwYb7GjG0uGA0KhpFZicZMiPYJGB~oynsMfd94bPAIrtQQPPmGzVGv0FtPTWthom~pPxQD8WgJhn5dGBE8EuXCW7bGuh87zOMErdi4RM6l1hIpsCmjH99FUAMlBlnn1Obqs6yCaDyJuJdsESPq4-uwP4YJIZZyz6U843ECQydjMQgSO4nkuzNbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Positional Cloning Identifies  Lotus japonicus  NSP2, A Putative Transcription Factor of the GRAS Family, Required for  NIN  and  ENOD40  Gene Expression in Nodule Initiation" />
    <meta property="og:description" content="Abstract.  Rhizobia-secreted Nod-factors (NFs) are required for nodulation. In the early developmental process of nodulation, a large number of changes occur in" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/13/6/255/464686" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2006-01-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Murakami, Yasuhiro" /><meta name="citation_author_institution" content="1National Institute of Agrobiological Sciences, 2-1-2 KannondaiTsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_author_institution" content="3Department of Biological Sciences, Graduate School of Science, The University of Tokyo7-3-1 Hongo, Bunkyo, Tokyo 113-0033, Japan" /><meta name="citation_author_institution" content="4Bio-oriented Technology Research Advancement Institution (BRAIN), Tokyo Office3-18-19 Toranomon, Minato-ku, Tokyo 105-0001, Japan" /><meta name="citation_author" content="Miwa, Hiroki" /><meta name="citation_author_institution" content="2John Innes CentreColney Lane, Norwich NR4 7UH, UK" /><meta name="citation_author" content="Imaizumi-Anraku, Haruko" /><meta name="citation_author_institution" content="1National Institute of Agrobiological Sciences, 2-1-2 KannondaiTsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_author_institution" content="4Bio-oriented Technology Research Advancement Institution (BRAIN), Tokyo Office3-18-19 Toranomon, Minato-ku, Tokyo 105-0001, Japan" /><meta name="citation_author" content="Kouchi, Hiroshi" /><meta name="citation_author_institution" content="1National Institute of Agrobiological Sciences, 2-1-2 KannondaiTsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_author" content="Downie, J. Allan" /><meta name="citation_author_institution" content="2John Innes CentreColney Lane, Norwich NR4 7UH, UK" /><meta name="citation_author" content="Kawaguchi, Masayoshi" /><meta name="citation_author_institution" content="3Department of Biological Sciences, Graduate School of Science, The University of Tokyo7-3-1 Hongo, Bunkyo, Tokyo 113-0033, Japan" /><meta name="citation_author" content="Kawasaki, Shinji" /><meta name="citation_author_institution" content="1National Institute of Agrobiological Sciences, 2-1-2 KannondaiTsukuba, Ibaraki 305-8602, Japan" /><meta name="citation_author_institution" content="4Bio-oriented Technology Research Advancement Institution (BRAIN), Tokyo Office3-18-19 Toranomon, Minato-ku, Tokyo 105-0001, Japan" /><meta name="citation_title" content="Positional Cloning Identifies Lotus japonicus NSP2, A Putative Transcription Factor of the GRAS Family, Required for NIN and ENOD40 Gene Expression in Nodule Initiation" /><meta name="citation_firstpage" content="255" /><meta name="citation_lastpage" content="265" /><meta name="citation_doi" content="10.1093/dnares/dsl017" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="13" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2006/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Calcium, kinases and nodulation signalling in legumes; Nat. Rev. Mol. Cell Biol.;  citation_year=2004;  citation_volume=5;  citation_pages=566-576; " /><meta name="citation_reference" content="citation_title=Four genes of Medicago truncatula controlling components of a Nod factor transduction pathway; Plant Cell;  citation_year=2000;  citation_volume=12;  citation_pages=1647-1665; " /><meta name="citation_reference" content="citation_title=Root, root hair, and symbiotic mutants of the model legume Lotus japonicus; Mol. Plant-Microbe Interact.;  citation_year=2002;  citation_volume=15;  citation_pages=17-26; " /><meta name="citation_reference" content="citation_title=Nodule organogenesis and symbiotic mutants of the model legume Lotus japonicus; Mol. Plant-Microbe Interact.;  citation_year=1998;  citation_volume=11;  citation_pages=684-697; " /><meta name="citation_reference" content="citation_title=Plant recognition of symbiotic bacteria requires two LysM receptor-like kinases; Nature;  citation_year=2003;  citation_volume=425;  citation_pages=585-592; " /><meta name="citation_reference" content="citation_title=A receptor kinase gene of the LysM type is involved in legume perception of rhizobial signals; Nature;  citation_year=2003;  citation_volume=425;  citation_pages=637-640; " /><meta name="citation_reference" content="citation_title=LysM domain receptor kinases regulating rhizobial Nod factor-induced infection; Science;  citation_year=2003;  citation_volume=302;  citation_pages=630-633; " /><meta name="citation_reference" content="citation_title=A plant receptor-like kinase required for both bacterial and fungal symbiosis; Nature;  citation_year=2002;  citation_volume=417;  citation_pages=959-962; " /><meta name="citation_reference" content="citation_title=Plastid proteins crucial for symbiotic fungal and bacterial entry into plant roots; Nature;  citation_year=2005;  citation_volume=433;  citation_pages=527-531; " /><meta name="citation_reference" content="citation_title=Medicago truncatula DMI1 required for bacterial and fungal symbioses in legumes; Science;  citation_year=2004;  citation_volume=303;  citation_pages=1364-1367; " /><meta name="citation_reference" content="citation_title=A receptor kinase gene regulating symbiotic nodule development; Nature;  citation_year=2002;  citation_volume=417;  citation_pages=962-966; " /><meta name="citation_reference" content="citation_title=A putative Ca2+ and Calmodulin-dependent protein kinase required for bacterial and fungal symbiosis; Science;  citation_year=2004;  citation_volume=303;  citation_pages=1361-1364; " /><meta name="citation_reference" content="citation_title=Deregulation of a Ca2+/calmodulin-dependent kinase leads to spontaneous nodule development; Nature;  citation_year=2006;  citation_volume=441;  citation_pages=1153-1156; " /><meta name="citation_reference" content="citation_title=A nucleoporin is required for induction of Ca2+ spiking in legume nodule development and essential for rhizobial and fungal symbiosis; Proc. Natl Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=359-364; " /><meta name="citation_reference" content="citation_title=A plant regulator controlling development of symbiotic root nodules; Nature;  citation_year=1999;  citation_volume=402;  citation_pages=191-195; " /><meta name="citation_reference" content="citation_title=Analysis of Nod-factor-induced calcium signaling in root hairs of symbiotically defective mutants of Lotus japonicus; Mol. Plant-Microbe Interact.;  citation_year=2006;  citation_volume=19;  citation_pages=914-923; " /><meta name="citation_reference" content="citation_title=NSP1 of the GRAS protein family is essential for rhizobial Nod factor-induced transcription; Science;  citation_year=2005;  citation_volume=303;  citation_pages=1789-1791; " /><meta name="citation_reference" content="citation_title=Nodulation signaling in legumes requires NSP2, a member of the GRAS family of transcriptional regulators; Science;  citation_year=2005;  citation_volume=303;  citation_pages=1786-1789; " /><meta name="citation_reference" content="citation_title=The maize transposable element Ac is mobile in the legume Lotus japonicus; Plant Mol. Biol.;  citation_year=1995;  citation_volume=27;  citation_pages=981-993; " /><meta name="citation_reference" content="citation_title=Lotus japonicus ‘Miyakojima’ MG-20: an early flowering accession suitable for indoor genetics; J. Plant Res.;  citation_year=2000;  citation_volume=113;  citation_pages=507-509; " /><meta name="citation_reference" content="citation_title=Lotus burttii takes a position of the third corner in the Lotus molecular genetics triangle; DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=69-77; " /><meta name="citation_reference" content="citation_title=Construction of high-density map, genome library, and saturation mapping of nodulation genes; citation_publisher=Brassicas and Legumes, ; Biotechnology in Agriculture and Forestry;  citation_year=2003;  citation_volume=Vol. 52;  citation_pages=183-202; " /><meta name="citation_reference" content="citation_title=Genome analysis of Lotus japonicus; J. Plant Res.;  citation_year=2000;  citation_volume=113;  citation_pages=497-506; " /><meta name="citation_reference" content="citation_title=Engineered GFP as a vital reporter in plants; Curr. Biol.;  citation_year=1996;  citation_volume=6;  citation_pages=325-330; " /><meta name="citation_reference" content="citation_title=Root lectin as a determinant of host-plant specificity in the Rhizobium-legume symbiosis; Nature;  citation_year=1989;  citation_volume=338;  citation_pages=579-581; " /><meta name="citation_reference" content="citation_title=Gene silencing by expression of Hairpin RNA in Lotus japonicus roots and root nodules; Mol. Plant-Microbe Interact.;  citation_year=2003;  citation_volume=16;  citation_pages=663-668; " /><meta name="citation_reference" content="citation_title=Rapid isolation of high molecular weight plant DNA; Nucleic Acids Res.;  citation_year=1980;  citation_volume=8;  citation_pages=4321-4325; " /><meta name="citation_reference" content="citation_title=Large-scale analysis of gene expression profiles during early stages of root nodule formation in a model legume, Lotus japonicus; DNA Res.;  citation_year=2004;  citation_volume=11;  citation_pages=263-274; " /><meta name="citation_reference" content="citation_title=Engineered GFP as a vital reporter in plants; Curr. Biol.;  citation_year=1996;  citation_volume=6;  citation_pages=325-330; " /><meta name="citation_reference" content="citation_title=Isolation of two different phenotypes of mycorrhizal mutants in the model legume plant Lotus japonicus after EMS-treatment; Plant Cell Physiol.;  citation_year=2000;  citation_volume=41;  citation_pages=726-732; " /><meta name="citation_reference" content="citation_title=Genetics of symbiosis in Lotus japonicus: recombinant inbred lines, comparative genetic maps, and map position of 35 symbiotic loci; Mol. Plant-Microbe Interact.;  citation_year=2006;  citation_volume=19;  citation_pages=80-92; " /><meta name="citation_reference" content="citation_title=The role of GRAS proteins in plant signal transduction and development; Planta;  citation_year=2004;  citation_volume=218;  citation_pages=683-692; " /><meta name="citation_reference" content="citation_title=The GRAS gene family in Arabidopsis: sequence characterization and basic expression analysis of the SCARECROW-LIKE genes; Plant J.;  citation_year=1999;  citation_volume=18;  citation_pages=111-119; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of the GRAS gene family in rice and Arabidopsis; Plant Mol. Biol.;  citation_year=2004;  citation_volume=54;  citation_pages=519-532; " /><meta name="citation_reference" content="citation_title=Plant GRAS and metazoan STATs: only family?; BioEssays;  citation_year=2000;  citation_volume=22;  citation_pages=573-577; " /><meta name="citation_reference" content="citation_title=Isolation and characterization of novel noduline cDNAs representing genes expressed at early stages of soybean nodule development; Mol. Gen. Genet.;  citation_year=1993;  citation_volume=238;  citation_pages=106-119; " /><meta name="citation_reference" content="citation_title=Characterization of GmENOD40, a gene showing novel patterns of cell-specific expression during soybean nodule development; Plant J.;  citation_year=1993;  citation_volume=3;  citation_pages=573-585; " /><meta name="citation_reference" content="citation_title=Expression of the early nodulin, ENOD40, in soybean roots in response to various lipo-chitin signal molecules; Plant J.;  citation_year=1996;  citation_volume=10;  citation_pages=23-32; " /><meta name="citation_reference" content="citation_title=RNAi knock-down of ENOD40s leads to significant suppression of nodule formation in Lotus japonicus; Plant Cell Physiol.;  citation_year=2006;  citation_volume=47;  citation_pages=1102-1111; " /><meta name="citation_reference" content="citation_title=Alteration of enod40 expression modifies Medicago truncatula root nodule development induced by Sinorhizobium meliloti; Plant Cell;  citation_year=1999;  citation_volume=11;  citation_pages=1953-1966; " /><meta name="citation_reference" content="citation_title=Lotus japonicus nodulation required two GRAS domain regulators, one of which is functionally conserved in a non-legume; Plant Physiol.;  citation_year=2006;  citation_volume=142;  citation_pages=1739-1750; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/13/6/255/1260637/dsl017.pdf" /><meta name="description" content="Abstract.  Rhizobia-secreted Nod-factors (NFs) are required for nodulation. In the early developmental process of nodulation, a large number of changes occur in" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/13/6/255/464686" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/13/6/255/464686" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/13/6/255/464686");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsl017';
    </script>
<title>
	Positional Cloning Identifies Lotus japonicus NSP2, A Putative Transcription Factor of the GRAS Family, Required for NIN and ENOD40 Gene Expression in Nodule Initiation | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./464686" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BdFkPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNdXJha2FtaSwgWWFzdWhpcm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEFncm9iaW9sb2dpY2FsIFNjaWVuY2VzLCAyLTEtMiBLYW5ub25kYWlUc3VrdWJhLCBJYmFyYWtpIDMwNS04NjAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjNEZXBhcnRtZW50IG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMsIEdyYWR1YXRlIFNjaG9vbCBvZiBTY2llbmNlLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bzctMy0xIEhvbmdvLCBCdW5reW8sIFRva3lvIDExMy0wMDMzLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjRCaW8tb3JpZW50ZWQgVGVjaG5vbG9neSBSZXNlYXJjaCBBZHZhbmNlbWVudCBJbnN0aXR1dGlvbiAoQlJBSU4pLCBUb2t5byBPZmZpY2UzLTE4LTE5IFRvcmFub21vbiwgTWluYXRvLWt1LCBUb2t5byAxMDUtMDAwMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNaXdhLCBIaXJva2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIySm9obiBJbm5lcyBDZW50cmVDb2xuZXkgTGFuZSwgTm9yd2ljaCBOUjQgN1VILCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkltYWl6dW1pLUFucmFrdSwgSGFydWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMU5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBZ3JvYmlvbG9naWNhbCBTY2llbmNlcywgMi0xLTIgS2Fubm9uZGFpVHN1a3ViYSwgSWJhcmFraSAzMDUtODYwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSI0QmlvLW9yaWVudGVkIFRlY2hub2xvZ3kgUmVzZWFyY2ggQWR2YW5jZW1lbnQgSW5zdGl0dXRpb24gKEJSQUlOKSwgVG9reW8gT2ZmaWNlMy0xOC0xOSBUb3Jhbm9tb24sIE1pbmF0by1rdSwgVG9reW8gMTA1LTAwMDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS291Y2hpLCBIaXJvc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMU5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBZ3JvYmlvbG9naWNhbCBTY2llbmNlcywgMi0xLTIgS2Fubm9uZGFpVHN1a3ViYSwgSWJhcmFraSAzMDUtODYwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEb3duaWUsIEouIEFsbGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMkpvaG4gSW5uZXMgQ2VudHJlQ29sbmV5IExhbmUsIE5vcndpY2ggTlI0IDdVSCwgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYXdhZ3VjaGksIE1hc2F5b3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjNEZXBhcnRtZW50IG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMsIEdyYWR1YXRlIFNjaG9vbCBvZiBTY2llbmNlLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bzctMy0xIEhvbmdvLCBCdW5reW8sIFRva3lvIDExMy0wMDMzLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikthd2FzYWtpLCBTaGluamkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEFncm9iaW9sb2dpY2FsIFNjaWVuY2VzLCAyLTEtMiBLYW5ub25kYWlUc3VrdWJhLCBJYmFyYWtpIDMwNS04NjAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjRCaW8tb3JpZW50ZWQgVGVjaG5vbG9neSBSZXNlYXJjaCBBZHZhbmNlbWVudCBJbnN0aXR1dGlvbiAoQlJBSU4pLCBUb2t5byBPZmZpY2UzLTE4LTE5IFRvcmFub21vbiwgTWluYXRvLWt1LCBUb2t5byAxMDUtMDAwMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IlBvc2l0aW9uYWwgQ2xvbmluZyBJZGVudGlmaWVzIExvdHVzIGphcG9uaWN1cyBOU1AyLCBBIFB1dGF0aXZlIFRyYW5zY3JpcHRpb24gRmFjdG9yIG9mIHRoZSBHUkFTIEZhbWlseSwgUmVxdWlyZWQgZm9yIE5JTiBhbmQgRU5PRDQwIEdlbmUgRXhwcmVzc2lvbiBpbiBOb2R1bGUgSW5pdGlhdGlvbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjI1NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMjY1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNsMDE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjEzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDYvMDEvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DYWxjaXVtLCBraW5hc2VzIGFuZCBub2R1bGF0aW9uIHNpZ25hbGxpbmcgaW4gbGVndW1lczsgTmF0LiBSZXYuIE1vbC4gQ2VsbCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NTY2LTU3NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Rm91ciBnZW5lcyBvZiBNZWRpY2FnbyB0cnVuY2F0dWxhIGNvbnRyb2xsaW5nIGNvbXBvbmVudHMgb2YgYSBOb2QgZmFjdG9yIHRyYW5zZHVjdGlvbiBwYXRod2F5OyBQbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTY0Ny0xNjY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Sb290LCByb290IGhhaXIsIGFuZCBzeW1iaW90aWMgbXV0YW50cyBvZiB0aGUgbW9kZWwgbGVndW1lIExvdHVzIGphcG9uaWN1czsgTW9sLiBQbGFudC1NaWNyb2JlIEludGVyYWN0LjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTE3LTI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob2R1bGUgb3JnYW5vZ2VuZXNpcyBhbmQgc3ltYmlvdGljIG11dGFudHMgb2YgdGhlIG1vZGVsIGxlZ3VtZSBMb3R1cyBqYXBvbmljdXM7IE1vbC4gUGxhbnQtTWljcm9iZSBJbnRlcmFjdC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz02ODQtNjk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFudCByZWNvZ25pdGlvbiBvZiBzeW1iaW90aWMgYmFjdGVyaWEgcmVxdWlyZXMgdHdvIEx5c00gcmVjZXB0b3ItbGlrZSBraW5hc2VzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDI1OyAgY2l0YXRpb25fcGFnZXM9NTg1LTU5MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSByZWNlcHRvciBraW5hc2UgZ2VuZSBvZiB0aGUgTHlzTSB0eXBlIGlzIGludm9sdmVkIGluIGxlZ3VtZSBwZXJjZXB0aW9uIG9mIHJoaXpvYmlhbCBzaWduYWxzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDI1OyAgY2l0YXRpb25fcGFnZXM9NjM3LTY0MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9THlzTSBkb21haW4gcmVjZXB0b3Iga2luYXNlcyByZWd1bGF0aW5nIHJoaXpvYmlhbCBOb2QgZmFjdG9yLWluZHVjZWQgaW5mZWN0aW9uOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMwMjsgIGNpdGF0aW9uX3BhZ2VzPTYzMC02MzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcGxhbnQgcmVjZXB0b3ItbGlrZSBraW5hc2UgcmVxdWlyZWQgZm9yIGJvdGggYmFjdGVyaWFsIGFuZCBmdW5nYWwgc3ltYmlvc2lzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NDE3OyAgY2l0YXRpb25fcGFnZXM9OTU5LTk2MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGxhc3RpZCBwcm90ZWlucyBjcnVjaWFsIGZvciBzeW1iaW90aWMgZnVuZ2FsIGFuZCBiYWN0ZXJpYWwgZW50cnkgaW50byBwbGFudCByb290czsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQzMzsgIGNpdGF0aW9uX3BhZ2VzPTUyNy01MzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lZGljYWdvIHRydW5jYXR1bGEgRE1JMSByZXF1aXJlZCBmb3IgYmFjdGVyaWFsIGFuZCBmdW5nYWwgc3ltYmlvc2VzIGluIGxlZ3VtZXM7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzAzOyAgY2l0YXRpb25fcGFnZXM9MTM2NC0xMzY3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHJlY2VwdG9yIGtpbmFzZSBnZW5lIHJlZ3VsYXRpbmcgc3ltYmlvdGljIG5vZHVsZSBkZXZlbG9wbWVudDsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTQxNzsgIGNpdGF0aW9uX3BhZ2VzPTk2Mi05NjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcHV0YXRpdmUgQ2EyKyBhbmQgQ2FsbW9kdWxpbi1kZXBlbmRlbnQgcHJvdGVpbiBraW5hc2UgcmVxdWlyZWQgZm9yIGJhY3RlcmlhbCBhbmQgZnVuZ2FsIHN5bWJpb3NpczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMDM7ICBjaXRhdGlvbl9wYWdlcz0xMzYxLTEzNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlcmVndWxhdGlvbiBvZiBhIENhMisvY2FsbW9kdWxpbi1kZXBlbmRlbnQga2luYXNlIGxlYWRzIHRvIHNwb250YW5lb3VzIG5vZHVsZSBkZXZlbG9wbWVudDsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTQ0MTsgIGNpdGF0aW9uX3BhZ2VzPTExNTMtMTE1NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBudWNsZW9wb3JpbiBpcyByZXF1aXJlZCBmb3IgaW5kdWN0aW9uIG9mIENhMisgc3Bpa2luZyBpbiBsZWd1bWUgbm9kdWxlIGRldmVsb3BtZW50IGFuZCBlc3NlbnRpYWwgZm9yIHJoaXpvYmlhbCBhbmQgZnVuZ2FsIHN5bWJpb3NpczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDM7ICBjaXRhdGlvbl9wYWdlcz0zNTktMzY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHBsYW50IHJlZ3VsYXRvciBjb250cm9sbGluZyBkZXZlbG9wbWVudCBvZiBzeW1iaW90aWMgcm9vdCBub2R1bGVzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9NDAyOyAgY2l0YXRpb25fcGFnZXM9MTkxLTE5NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgTm9kLWZhY3Rvci1pbmR1Y2VkIGNhbGNpdW0gc2lnbmFsaW5nIGluIHJvb3QgaGFpcnMgb2Ygc3ltYmlvdGljYWxseSBkZWZlY3RpdmUgbXV0YW50cyBvZiBMb3R1cyBqYXBvbmljdXM7IE1vbC4gUGxhbnQtTWljcm9iZSBJbnRlcmFjdC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz05MTQtOTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OU1AxIG9mIHRoZSBHUkFTIHByb3RlaW4gZmFtaWx5IGlzIGVzc2VudGlhbCBmb3Igcmhpem9iaWFsIE5vZCBmYWN0b3ItaW5kdWNlZCB0cmFuc2NyaXB0aW9uOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMwMzsgIGNpdGF0aW9uX3BhZ2VzPTE3ODktMTc5MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm9kdWxhdGlvbiBzaWduYWxpbmcgaW4gbGVndW1lcyByZXF1aXJlcyBOU1AyLCBhIG1lbWJlciBvZiB0aGUgR1JBUyBmYW1pbHkgb2YgdHJhbnNjcmlwdGlvbmFsIHJlZ3VsYXRvcnM7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzAzOyAgY2l0YXRpb25fcGFnZXM9MTc4Ni0xNzg5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbWFpemUgdHJhbnNwb3NhYmxlIGVsZW1lbnQgQWMgaXMgbW9iaWxlIGluIHRoZSBsZWd1bWUgTG90dXMgamFwb25pY3VzOyBQbGFudCBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9OTgxLTk5MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG90dXMgamFwb25pY3VzIOKAmE1peWFrb2ppbWHigJkgTUctMjA6IGFuIGVhcmx5IGZsb3dlcmluZyBhY2Nlc3Npb24gc3VpdGFibGUgZm9yIGluZG9vciBnZW5ldGljczsgSi4gUGxhbnQgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTM7ICBjaXRhdGlvbl9wYWdlcz01MDctNTA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Mb3R1cyBidXJ0dGlpIHRha2VzIGEgcG9zaXRpb24gb2YgdGhlIHRoaXJkIGNvcm5lciBpbiB0aGUgTG90dXMgbW9sZWN1bGFyIGdlbmV0aWNzIHRyaWFuZ2xlOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTY5LTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgaGlnaC1kZW5zaXR5IG1hcCwgZ2Vub21lIGxpYnJhcnksIGFuZCBzYXR1cmF0aW9uIG1hcHBpbmcgb2Ygbm9kdWxhdGlvbiBnZW5lczsgY2l0YXRpb25fcHVibGlzaGVyPUJyYXNzaWNhcyBhbmQgTGVndW1lcywgOyBCaW90ZWNobm9sb2d5IGluIEFncmljdWx0dXJlIGFuZCBGb3Jlc3RyeTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT1Wb2wuIDUyOyAgY2l0YXRpb25fcGFnZXM9MTgzLTIwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIGFuYWx5c2lzIG9mIExvdHVzIGphcG9uaWN1czsgSi4gUGxhbnQgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTM7ICBjaXRhdGlvbl9wYWdlcz00OTctNTA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbmdpbmVlcmVkIEdGUCBhcyBhIHZpdGFsIHJlcG9ydGVyIGluIHBsYW50czsgQ3Vyci4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTMyNS0zMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJvb3QgbGVjdGluIGFzIGEgZGV0ZXJtaW5hbnQgb2YgaG9zdC1wbGFudCBzcGVjaWZpY2l0eSBpbiB0aGUgUmhpem9iaXVtLWxlZ3VtZSBzeW1iaW9zaXM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzg7ICBjaXRhdGlvbl9wYWdlcz01NzktNTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIHNpbGVuY2luZyBieSBleHByZXNzaW9uIG9mIEhhaXJwaW4gUk5BIGluIExvdHVzIGphcG9uaWN1cyByb290cyBhbmQgcm9vdCBub2R1bGVzOyBNb2wuIFBsYW50LU1pY3JvYmUgSW50ZXJhY3QuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NjYzLTY2ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmFwaWQgaXNvbGF0aW9uIG9mIGhpZ2ggbW9sZWN1bGFyIHdlaWdodCBwbGFudCBETkE7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk4MDsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NDMyMS00MzI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXJnZS1zY2FsZSBhbmFseXNpcyBvZiBnZW5lIGV4cHJlc3Npb24gcHJvZmlsZXMgZHVyaW5nIGVhcmx5IHN0YWdlcyBvZiByb290IG5vZHVsZSBmb3JtYXRpb24gaW4gYSBtb2RlbCBsZWd1bWUsIExvdHVzIGphcG9uaWN1czsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0yNjMtMjc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbmdpbmVlcmVkIEdGUCBhcyBhIHZpdGFsIHJlcG9ydGVyIGluIHBsYW50czsgQ3Vyci4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTMyNS0zMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzb2xhdGlvbiBvZiB0d28gZGlmZmVyZW50IHBoZW5vdHlwZXMgb2YgbXljb3JyaGl6YWwgbXV0YW50cyBpbiB0aGUgbW9kZWwgbGVndW1lIHBsYW50IExvdHVzIGphcG9uaWN1cyBhZnRlciBFTVMtdHJlYXRtZW50OyBQbGFudCBDZWxsIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9NzI2LTczMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpY3Mgb2Ygc3ltYmlvc2lzIGluIExvdHVzIGphcG9uaWN1czogcmVjb21iaW5hbnQgaW5icmVkIGxpbmVzLCBjb21wYXJhdGl2ZSBnZW5ldGljIG1hcHMsIGFuZCBtYXAgcG9zaXRpb24gb2YgMzUgc3ltYmlvdGljIGxvY2k7IE1vbC4gUGxhbnQtTWljcm9iZSBJbnRlcmFjdC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz04MC05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJvbGUgb2YgR1JBUyBwcm90ZWlucyBpbiBwbGFudCBzaWduYWwgdHJhbnNkdWN0aW9uIGFuZCBkZXZlbG9wbWVudDsgUGxhbnRhOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIxODsgIGNpdGF0aW9uX3BhZ2VzPTY4My02OTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBHUkFTIGdlbmUgZmFtaWx5IGluIEFyYWJpZG9wc2lzOiBzZXF1ZW5jZSBjaGFyYWN0ZXJpemF0aW9uIGFuZCBiYXNpYyBleHByZXNzaW9uIGFuYWx5c2lzIG9mIHRoZSBTQ0FSRUNST1ctTElLRSBnZW5lczsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xMTEtMTE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBvZiB0aGUgR1JBUyBnZW5lIGZhbWlseSBpbiByaWNlIGFuZCBBcmFiaWRvcHNpczsgUGxhbnQgTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01NDsgIGNpdGF0aW9uX3BhZ2VzPTUxOS01MzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYW50IEdSQVMgYW5kIG1ldGF6b2FuIFNUQVRzOiBvbmx5IGZhbWlseT87IEJpb0Vzc2F5czsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTU3My01Nzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzb2xhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBub3ZlbCBub2R1bGluZSBjRE5BcyByZXByZXNlbnRpbmcgZ2VuZXMgZXhwcmVzc2VkIGF0IGVhcmx5IHN0YWdlcyBvZiBzb3liZWFuIG5vZHVsZSBkZXZlbG9wbWVudDsgTW9sLiBHZW4uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzg7ICBjaXRhdGlvbl9wYWdlcz0xMDYtMTE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIEdtRU5PRDQwLCBhIGdlbmUgc2hvd2luZyBub3ZlbCBwYXR0ZXJucyBvZiBjZWxsLXNwZWNpZmljIGV4cHJlc3Npb24gZHVyaW5nIHNveWJlYW4gbm9kdWxlIGRldmVsb3BtZW50OyBQbGFudCBKLjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9NTczLTU4NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwcmVzc2lvbiBvZiB0aGUgZWFybHkgbm9kdWxpbiwgRU5PRDQwLCBpbiBzb3liZWFuIHJvb3RzIGluIHJlc3BvbnNlIHRvIHZhcmlvdXMgbGlwby1jaGl0aW4gc2lnbmFsIG1vbGVjdWxlczsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0yMy0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Uk5BaSBrbm9jay1kb3duIG9mIEVOT0Q0MHMgbGVhZHMgdG8gc2lnbmlmaWNhbnQgc3VwcHJlc3Npb24gb2Ygbm9kdWxlIGZvcm1hdGlvbiBpbiBMb3R1cyBqYXBvbmljdXM7IFBsYW50IENlbGwgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz0xMTAyLTExMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdGVyYXRpb24gb2YgZW5vZDQwIGV4cHJlc3Npb24gbW9kaWZpZXMgTWVkaWNhZ28gdHJ1bmNhdHVsYSByb290IG5vZHVsZSBkZXZlbG9wbWVudCBpbmR1Y2VkIGJ5IFNpbm9yaGl6b2JpdW0gbWVsaWxvdGk7IFBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xOTUzLTE5NjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvdHVzIGphcG9uaWN1cyBub2R1bGF0aW9uIHJlcXVpcmVkIHR3byBHUkFTIGRvbWFpbiByZWd1bGF0b3JzLCBvbmUgb2Ygd2hpY2ggaXMgZnVuY3Rpb25hbGx5IGNvbnNlcnZlZCBpbiBhIG5vbi1sZWd1bWU7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE0MjsgIGNpdGF0aW9uX3BhZ2VzPTE3MzktMTc1MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzEzLzYvMjU1LzEyNjA2MzcvZHNsMDE3LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBSaGl6b2JpYS1zZWNyZXRlZCBOb2QtZmFjdG9ycyAoTkZzKSBhcmUgcmVxdWlyZWQgZm9yIG5vZHVsYXRpb24uIEluIHRoZSBlYXJseSBkZXZlbG9wbWVudGFsIHByb2Nlc3Mgb2Ygbm9kdWxhdGlvbiwgYSBsYXJnZSBudW1iZXIgb2YgY2hhbmdlcyBvY2N1ciBpbiIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDY0Njg2JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZC+kk0CB+4c9NrSlLewI7dV0qDK7" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f6%2f255%2f464686" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f6%2f255%2f464686" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=18198" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/13/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/6/0/m_dnaresearch13_6.cover.gif?Expires=2147483647&amp;Signature=fcRvTZS-Esu1iTIRpBwNtJBvy2AJRBCdQJyTXwh60M7JIQlqTrbnOTGA7L-7i6L4UYLG4JUSar16AKIY53IuiLWx-2mOh3JMXxdjEKlHYOAqNyuibso-D4bG3vOwDPxfhcZLg45cv2YcwYaGQxhzHrL5TjKyhZ4mwYb7GjG0uGA0KhpFZicZMiPYJGB~oynsMfd94bPAIrtQQPPmGzVGv0FtPTWthom~pPxQD8WgJhn5dGBE8EuXCW7bGuh87zOMErdi4RM6l1hIpsCmjH99FUAMlBlnn1Obqs6yCaDyJuJdsESPq4-uwP4YJIZZyz6U843ECQydjMQgSO4nkuzNbQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/13/6">
                        <div class="volume trailing-comma">Volume 13</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
2006        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6982037">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6982037">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6982042">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6982042">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6982055">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6982055">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6982120">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6982120">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6982174">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6982174">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6982187">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6982187">REFERENCES</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/13/6/245/464398">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/13/6/267/464883">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Positional Cloning Identifies <em>Lotus japonicus</em> NSP2, A Putative Transcription Factor of the GRAS Family, Required for <em>NIN</em> and <em>ENOD40</em> Gene Expression in Nodule Initiation
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasuhiro Murakami</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">National Institute of Agrobiological Sciences, 2-1-2 Kannondai</div><div class="addr-line">Tsukuba, Ibaraki 305-8602, Japan</div></div><div class="aff"><div class="label -label">3</div><sup>3</sup><div class="insititution">Department of Biological Sciences, Graduate School of Science, The University of Tokyo</div><div class="addr-line">7-3-1 Hongo, Bunkyo, Tokyo 113-0033, Japan</div></div><div class="aff"><div class="label -label">4</div><sup>4</sup><div class="insititution">Bio-oriented Technology Research Advancement Institution (BRAIN), Tokyo Office</div><div class="addr-line">3-18-19 Toranomon, Minato-ku, Tokyo 105-0001, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasuhiro+Murakami">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Murakami Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Murakami Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yasuhiro Murakami</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroki Miwa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">John Innes Centre</div><div class="addr-line">Colney Lane, Norwich NR4 7UH, UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroki+Miwa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Miwa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Miwa H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiroki Miwa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Haruko Imaizumi-Anraku</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">National Institute of Agrobiological Sciences, 2-1-2 Kannondai</div><div class="addr-line">Tsukuba, Ibaraki 305-8602, Japan</div></div><div class="aff"><div class="label -label">4</div><sup>4</sup><div class="insititution">Bio-oriented Technology Research Advancement Institution (BRAIN), Tokyo Office</div><div class="addr-line">3-18-19 Toranomon, Minato-ku, Tokyo 105-0001, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Haruko+Imaizumi-Anraku">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Imaizumi-Anraku H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Imaizumi-Anraku H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Haruko Imaizumi-Anraku</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroshi Kouchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">National Institute of Agrobiological Sciences, 2-1-2 Kannondai</div><div class="addr-line">Tsukuba, Ibaraki 305-8602, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroshi+Kouchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kouchi H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kouchi H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiroshi Kouchi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">J. Allan Downie</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">John Innes Centre</div><div class="addr-line">Colney Lane, Norwich NR4 7UH, UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=J.+Allan+Downie">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Downie J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Downie J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">J. Allan Downie</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masayoshi Kawaguchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><sup>3</sup><div class="insititution">Department of Biological Sciences, Graduate School of Science, The University of Tokyo</div><div class="addr-line">7-3-1 Hongo, Bunkyo, Tokyo 113-0033, Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <sup>*</sup>To whom correspondence should be addressed. Tel./Fax: +81-3-5841-4458, E-mail: <a href="mailto:masayosi@biol.s.u-tokyo.ac.jp" target="_blank">masayosi@biol.s.u-tokyo.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masayoshi+Kawaguchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawaguchi M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kawaguchi M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masayoshi Kawaguchi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shinji Kawasaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">National Institute of Agrobiological Sciences, 2-1-2 Kannondai</div><div class="addr-line">Tsukuba, Ibaraki 305-8602, Japan</div></div><div class="aff"><div class="label -label">4</div><sup>4</sup><div class="insititution">Bio-oriented Technology Research Advancement Institution (BRAIN), Tokyo Office</div><div class="addr-line">3-18-19 Toranomon, Minato-ku, Tokyo 105-0001, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shinji+Kawasaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawasaki S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kawasaki S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shinji Kawasaki</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="fn1"><div class="fn"><div class="footnote-content"><sup>†</sup>Present address: Biotechnology Research Center, The University of Tokyo, 1-1-1 Yayoi, Bunkyo, Tokyo 113-8657, Japan</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Communicated by Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 13, Issue 6, 1 January 2006, Pages 255–265, <a href='https://doi.org/10.1093/dnares/dsl017'>https://doi.org/10.1093/dnares/dsl017</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 January 2006</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">30 November 2006</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">04 December 2006</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="464686" href="/dnaresearch/article-pdf/13/6/255/1260637/dsl017.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yasuhiro Murakami, Hiroki Miwa, Haruko Imaizumi-Anraku, Hiroshi Kouchi, J. Allan Downie, Masayoshi Kawaguchi, Shinji Kawasaki;  Positional Cloning Identifies <em>Lotus japonicus</em> NSP2, A Putative Transcription Factor of the GRAS Family, Required for <em>NIN</em> and <em>ENOD40</em> Gene Expression in Nodule Initiation, <em>DNA Research</em>, Volume 13, Issue 6, 1 January 2006, Pages 255–265, <a href='https://doi.org/10.1093/dnares/dsl017'>https://doi.org/10.1093/dnares/dsl017</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/464686?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/464686?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/464686?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/464686?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/464686?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/464686?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/464686?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Positional%20Cloning%20Identifies%20%26lt%3Bem%26gt%3BLotus%20japonicus%26lt%3B%2Fem%26gt%3B%20NSP2%2C%20A%20Putative%20Transcription%20Factor%20of%20the%20GRAS%20Family%2C%20Required%20for%20%26lt%3Bem%26gt%3BNIN%26lt%3B%2Fem%26gt%3B%20and%20%26lt%3Bem%26gt%3BENOD40%26lt%3B%2Fem%26gt%3B%20Gene%20Expression%20in%20Nodule%20Initiation&amp;publicationDate=2006-01-01&amp;volumeNum=13&amp;issueNum=6&amp;author=Murakami%2C%20Yasuhiro%3B%20Miwa%2C%20Hiroki&amp;startPage=255&amp;endPage=265&amp;contentId=10.1093%2Fdnares%2Fdsl017&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f6%2f255%2f464686" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6982037 id="6982037" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Rhizobia-secreted Nod-factors (NFs) are required for nodulation. In the early developmental process of nodulation, a large number of changes occur in gene expression. <em>Lotus japonicus nsp2</em> mutants isolated from Gifu B-129 ecotype have defects in nodule initiation and display non-nodulating phenotype. Here, we describe positional cloning of <em>LjNSP2</em> as a component of the nodulation-specific signaling pathway. <em>LjNSP2</em> was mapped near the translocation site of chromosome 1 where the recombination is severely suppressed. To circumvent this problem, we introduced <em>Lotus burttii</em> as an alternative crossing partner in place of <em>L. japonicus</em> Miyakojima. The development of the high-resolution map using a total of 11 481 F2 plants, in combination with newly developed DNA markers and construction of BAC library, enabled us to identify the gene responsible for mutant phenotype. <em>LjNSP2</em> encodes a putative transcription factor of the GRAS family that constitutes a subfamily with <em>Medicago truncatula NSP2</em>. <em>LjNSP2</em> was expressed in roots and early nodules, but strongly suppressed in matured nodules. The expression analysis of <em>NIN</em> and <em>LjENOD40-1</em> genes in <em>Ljnsp2</em> mutants indicates that LjNSP2 functions upstream of these genes. These results suggest that LjNSP2 acts as a transcription factor to directly or indirectly switch on the NF-induced genes required for nodule initiation.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;positional cloning&quot;">positional cloning</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Lotus japonicus&quot;"><em>Lotus japonicus</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcription factor&quot;">transcription factor</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;nodule initiation&quot;">nodule initiation</a></div>                            <h2 scrollto-destination=6982042 id="6982042" class="section-title" >1. Introduction</h2>
<p>The legume–rhizobia symbioses lead to the formation of novel organs, termed nodules, which arise from division of cortical cells in the root and the infection of these nodules by rhizobia. Rhizobia within nodule cells differentiate to bacteroids which fix atmospheric nitrogen. Lipochitin oligosaccharides, Nod-Factors (NFs) secreted by rhizobia are responsible for nodule formation and induce a variety of responses in a host-specific manner, including root hair deformation and cortical cell division, during the early steps of nodulation. Ca<sup>2+</sup> spiking in root hair cells is one of the most early responses to NFs.<a reveal-id="b1" data-open="b1" class="link link-ref link-reveal xref-bibr">1</a></p><p>In the past few years, the phenotypes of an increasing number of symbiotically defective mutants have been analyzed in the model legumes, <em>Lotus japonicus</em> and <em>Medicago truncatula</em> to dissect the NF signaling pathway.<a reveal-id="b2  b4" data-open="b2  b4" class="link link-ref link-reveal xref-bibr">2–4</a> Several genes required for nodulation have been identified by positional cloning. <em>L. japonicus NFR1</em>,<a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a>,<em>NFR5</em><a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> and <em>Medicago LYK3</em><a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a> encode transmembrane receptor-like serine/threonine kinases with putative extracellular regions similar to LysM domains, which are thought to be directly involved in perception of NF signal. In subsequent signal transduction, <em>L. japonicus SYMRK</em>,<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a>,<em>CASTOR</em> and <em>POLLUX</em>,<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a>,<em>M. truncatula DMI1</em>,<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a>,<em>DMI2</em><a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> and <em>DMI3</em><a reveal-id="b12" data-open="b12" class="link link-ref link-reveal xref-bibr">12</a>, and <em>L. japonicus CCaMK</em><a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> and <em>Nup133</em><a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> have been identified as components required for the common symbiosis pathway shared between the fungal and bacterial endosymbiotic systems. The <em>L. japonicus nin</em> mutant showed normal mycorrhization and early responses following rhizobial inoculation, including root hair deformation,<a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a> Ca<sup>2+</sup> influx and Ca<sup>2+</sup> spiking.<a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a>,<em>NIN</em> was cloned as a putative transcription factor gene and shown to be expressed during both early and late stages of nodule development, suggesting that NIN controls various developmental aspects of nodulation after the perception of rhizobia in root hairs. However, little is known about what signal component relays the signal from the common symbiotic pathway into the nodulation-specific program in <em>L. japonicus</em>. Recently, GRAS family genes, <em>nodulation signaling pathway 1</em> (<em>NSP1</em>) and <em>NSP2</em>, have been identified as putative transcription factors functioning downstream of Ca<sup>2+</sup> spiking and CCaMK in <em>Medicago truncatula</em>.<a reveal-id="b17" data-open="b17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a></p><p>Here, we describe the positional cloning and characterization of nodulation-specific <em>LjNSP2</em> gene encoding a plant-specific GRAS protein most similar to <em>MtNSP2</em>, which might lead to the induction of expression of genes required for rhizobial infection and early nodule development.</p>                            <h2 scrollto-destination=6982055 id="6982055" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=6982057 id="6982057" class="section-title" >2.1. Plant materials</h3>
<p>The mutant carrying <em>Ljnsp2-1</em> was isolated from the EMS-mutagenesis experiments of <em>L. japonicus</em> Gifu B-129.<a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a> The mutant <em>Ljsym35</em> was isolated from a population of Gifu derived from a transposon-/DNA-tagging trial, as a non-tagged culture mutant,<a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> and was kindly provided by Prof. Jens Stougaard (Arrhus University, Denmark).</p>                            <h3 scrollto-destination=6982064 id="6982064" class="section-title" >2.2. Root hair deformation and assays of Ca<sup>2+</sup> spiking</h3>
<p>Seeds of <em>L. japonicus</em> were germinated and grown on BNM agar medium essentially as described previously,<sup>9</sup> except that the roots were grown between two filter papers (grade 0860; Schleicher and Schűll, UK), one of which was on the agar surface. Root hair deformation was scored as described previously following 16 h exposure to 10<sup>−8</sup> M NF in 1 ml BNM medium in a chamber on a microscope slide. Images were taken with a digital-camera attached to an inverted microscope. Ca<sup>2+</sup> spiking was assayed as described previously,<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a> following the addition of 10<sup>−8</sup> M NF. Representative traces were selected from at least 10 independent cells.</p>                            <h3 scrollto-destination=6982071 id="6982071" class="section-title" >2.3. Genetic mapping population and genomic DNA isolation</h3>
<p>Genetic mapping of the <em>LjNSP2</em> gene was performed with F2/F3 population derived from cross between <em>Ljnsp2-1</em> (Gifu B-129) and the early flowering ecotype Miyakojima MG-20<a reveal-id="b20" data-open="b20" class="link link-ref link-reveal xref-bibr">20</a> or <em>Lotus burttii</em> B-303.<a reveal-id="b21" data-open="b21" class="link link-ref link-reveal xref-bibr">21</a> For AFLP analysis, genomic DNA was extracted from 0.1 g leaves using a Qiagen DNeasy plant kit according to the manufacture's protocol. For PCR markers analysis, DNA was extracted from one young leaf in 100 μl of PEB (200 mM Tris–HCl pH 7.5, 250 mM NaCl, 25 mM EDTA, 0.5% SDS), precipitated with isopropanol, washed and dissolved in 100 μl of TE (pH 8.0).</p>                            <h3 scrollto-destination=6982078 id="6982078" class="section-title" >2.4. High efficiency genome scanning (HEGS)/AFLP or PCR-based screening of LjNSP2-inked markers</h3>
<p>Genomic DNA (100 ng) was digested with <em>Eco</em>RI and <em>Mse</em>I, ligated to <em>Eco</em>RI- and <em>Mse</em>I-adaptors and preamplified by using <em>Eco</em>RI- and <em>Mse</em>I-adaptor primers. Preamplified DNA was prepared at 0.05 mg/μl concentration before amplification with selective primers. A bulked segregant analysis was performed to identify markers linked to <em>LjNSP2</em>. Bulks were constructed from preamplified DNAs of 10 recessive (Nod<sup>−</sup>) or dominant (Nod<sup>+</sup>) homozygous F2 plants, 4096 selective primer combinations of <em>Eco</em>RI+3/<em>Mse</em>I+3 were screened to identify markers present only in the bulk of dominant homozygotes.</p><p>Subsequently, <em>LjNSP2</em>-linked HEGS/AFLP markers were excised directly from polyacrylamide gels and cloned with the TOPO TA Cloning Kit (Invitrogen). The cloned markers were sequenced and primers were designed by using the software Primer 3 (Whitehead Institute, Cambridge, MA). SCAR (sequence characterized amplified region) markers, which revealed polymorphism between <em>L</em>. <em>japonicus</em> accession Gifu and Miyakojima MG-20 or <em>L</em>. <em>burttii</em>, were analyzed in 2989 or 8472 F2/3 individuals from Miyakojima MG-20 or <em>L</em>. <em>burttii</em>, respectively.</p><p>To facilitate the efficiency of electrophoresis of AFLP or PCR products, the HEGS system<a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> was adapted. In this system, a set of electrophoresis apparatus is equipped with two sets of 24.5 × 26.5 cm glass plates, each accommodating a gel with 100 lanes and analysis of 400 samples is practicable in single run. After electrophoresis, the gels were stained with Vistra Green (Amersham Biosciences) and scanned by fluorescent gel scanner (FluorImager 595; Amersham Biosciences).</p>                            <h3 scrollto-destination=6982089 id="6982089" class="section-title" >2.5. BAC contig development</h3>
<p>Three-dimensional BAC DNA pools prepared from our BAC library<a reveal-id="b23" data-open="b23" class="link link-ref link-reveal xref-bibr">23</a> were screened with <em>LjNSP2</em>-linked HEGS/AFLP or SCAR markers as described above. <em>Hin</em>dIII-digested DNA from positive BACs was fractionated on an agarose gel for fingerprinting and determination of overlaps. BAC ends were sequenced using M13 reverse or forward primer and non-repetitive sequences in BAC were used for chromosome walking to screen 3-D BAC pools with PCR or AFLP. The process was repeated as needed to complete the BAC contig. Polymorphic PCR fragments in BAC sequences were analyzed to directly score recombinants in F2/F3 populations.</p><p>BAC clone 188C5 containing the <em>LjNSP2</em> gene was shotgun-sequenced and then annotated by Rice-GAAS (Rice Genome Automated Annotation System) (<a class="link link-uri" href="http://ricegaas.dna.affrc.go.jp/" target="_blank"></a>).</p>                            <h3 scrollto-destination=6982097 id="6982097" class="section-title" >2.6. Complementation experiments</h3>
<p>For complementation, a 6.9 kb <em>Pst</em>I fragment, carrying only the wild-type <em>LjNSP2</em> 1500 bp ORF and 4486 and 899 bp of upstream and downstream sequence, respectively, was cloned into the hairy root transformation vector, which was made from pCAMBIA1300 by replacing the hygromycin-resistant gene with <em>sGFP(S65T)</em>.<a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a> The resulting <em>LjNSP2</em> recombinant plasmid was introduced into <em>A. rhizogenes</em> LBA1334<a reveal-id="b25" data-open="b25" class="link link-ref link-reveal xref-bibr">25</a> by electroporation. Hairy root transformation of <em>Ljnsp2-1</em> mutant was performed as described.<a reveal-id="b26" data-open="b26" class="link link-ref link-reveal xref-bibr">26</a> The plants with transgenic hairy roots were grown in vermiculite pots and inoculated with <em>Mesorhizobium loti</em> TONO. GFP fluorescence and nodule formations were confirmed 4 weeks after inoculation.</p>                            <h3 scrollto-destination=6982103 id="6982103" class="section-title" >2.7. Southern and northern hybridizations</h3>
<p>Genomic DNA was extracted from leaves using CTAB method<a reveal-id="b27" data-open="b27" class="link link-ref link-reveal xref-bibr">27</a> from <em>L. japonicus</em>, and 2.5 μg DNA were digested by <em>Eco</em>RI, electrophoresed on 0.8% agarose gel and blotted to nylon membrane (Biodyne A, Pall).</p><p>Twelve-day-old plants were inoculated with <em>M. loti</em> TONO. Infected roots at 4 days post inoculation (dpi) and nodules at 8–32 dpi were immediately frozen in liquid nitrogen. Total RNA was isolated from flower, shoot, root and nodule tissue, and 5 μg aliquots were electrophoresed by denaturing agarose gel and blotted as above.</p><p><em>LjNSP2</em> probe (1322 bp) was amplified from 188C5 BAC DNA with primers 5′-ACTTCCACCACCTCATCGAC-3′ and 5′-ACAAGTCCAAAGGGATGCAG-3′, and labeled with <sup>32</sup>P using a random primer labeling kit (Takara). Hybridization was done at 63°C in Church buffer [0.5 M sodium phosphate, pH 7.2, 7% (w/v) SDS, 1 mM EDTA]<a reveal-id="b28" data-open="b28" class="link link-ref link-reveal xref-bibr">28</a> and the filters were washed once in 2× SSC containing 0.5% SDS at room temperature for 10 min, and twice in 0.2× SSC, 0.1% SDS at 63°C for 15 min. The hybridized membranes were then exposed for 3 days to phosphor imaging plates (Fuji, Tokyo, Japan), which were then scanned by a phosphor imaging scanner (Storm840, Amersham Bioscience). After stripping, the same filters were reprobed with the 400 bp fragment from <em>L. japonicus</em> ubiquitin cDNA as a loading control.<a reveal-id="b28" data-open="b28" class="link link-ref link-reveal xref-bibr">28</a></p>                            <h3 scrollto-destination=6982113 id="6982113" class="section-title" >2.8. Transient expression of the LjNSP2-GFP fusion protein in onion epidermal cells</h3>
<p>The 1.5 kb ORF of <em>LjNSP2</em> was amplified from 188C5 BAC DNA using the primers: [5′-ACGC<span class="underline">GTCGAC</span>ATGGAAATGGATATAGATTGCATCC-3′ (<em>Sal</em>I-site underlined), 5′-CATG<span class="underline">TCATGA</span>ATGCACAATCTGATTCTGAAGAAC-3′ (<em>Bsp</em>HI-site underlined)], digested with <em>Sal</em>I and <em>Bsp</em>HI, and cloned into pUC18-CaMV35SΩ-sGFP(s65T)-NOS plasmid<a reveal-id="b29" data-open="b29" class="link link-ref link-reveal xref-bibr">29</a> at the <em>Sal</em>I/<em>Nco</em>I sites just upstream of GFP gene. Onion epidermal cells were bombarded with DNA-coated particles using a Biolistic PDS-1000/He Particle Delivery System (Bio-Rad).<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a> About 18–24 h after bombardment, the cells were observed using a Bio-Rad Radiance2000 confocal laser scanning microscope.</p><p>The sequence data of <em>LjNSP2</em> and genomic sequences (B-129 Gifu) of <em>LjNSP2</em> have been deposited with the DDBJ data library under accession numbers AB241456 and AB241457, respectively.</p>                            <h2 scrollto-destination=6982120 id="6982120" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=6982123 id="6982123" class="section-title" >3.1. Early infection phenotypes of a Ljnsp2 mutant</h3>
<p>The <em>Ljnsp2-1</em> mutant of <em>L. japonicus</em> grew normally in nitrogen-rich compost, and established a normal symbiosis with the mycorrhizal fungus,<a reveal-id="b30" data-open="b30" class="link link-ref link-reveal xref-bibr">30</a> but did not form nodules under nitrogen limitation when inoculated with <em>M. loti</em>. The addition of <em>M. loti</em> to <em>Ljnsp2-1</em> seedlings induced no phenotypes, such as root hair curling, infection thread formation and cortical cell division, typically seen in the wild-type (data not shown). However, NF did induce swelling and branching in root hair tips, although at a reduced level compared with the wild-type (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1A and H</a>). Cytological staining of <em>M. loti</em> expressing <em>lacZ</em> revealed no infection foci and no infection thread formation (data not shown). Intracellular Ca<sup>2+</sup> spiking, which is induced by NF and has been proposed to be integrated by a Ca<sup>2+</sup>-calmodulin-dependent protein kinase required for activation of early nodulation gene expression and mycorrhization<a reveal-id="b1" data-open="b1" class="link link-ref link-reveal xref-bibr">1</a>, was indistinguishable in the mutant from the wild-type (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1I</a>). The observed induction of Ca<sup>2+</sup> spiking and normal mycorrhization of the mutant indicates that the mutation in <em>Ljnsp2</em> affects a nodulation-specific signaling component that is downstream of the common pathway for mycorrhizal and rhizobial symbioses.</p>                        <a id="6982127" scrollto-destination="6982127"></a>
<div id="fig1" data-id="fig1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/m_dsl017f1.jpeg?Expires=2147483647&Signature=WLVJ4mJVwOMzYnvEZVtC-zcn697jx~KmwQassmwz9S65sWWbawqwyFdZkW~9pHvOtM1YfPnl4P13i6kDAPUVC0QbOmRYznPwhULTysSI0eeVcVKUAdhJNSUzsbGzWFiVe7Dq4Hodyi6LvFgaQokAgypts38lB22cAaGLrLyOhW7Sny7naOkBhUu1bS~2gLTjxEtKbeB8HD1985LXVjmIF7NIk0j1xmCylM6VHHbSb173vHo26tvhjaxB96x~Mi1pMawalumFY9oFVQ8HOkgirpdiKhiYNjnHWwbA8jf6~5gzVCXo32g56jm1zige3NE~P4F3RRwRyvlHTbGM6Sjmxg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="L. japonicus root hair responses induced by NF. (A–D) Root hair deformation induced with 10−8 M NF. Slight swelling and branching were observed also in Ljnsp2-1 mutant (C and D), although to a lesser extent than the wild-type (Gifu B-129; A and B). (E–H) Non-treated root hairs look similar between the mutant (G and H) and wild-type (E and F). Scale bar: 50 μm. (I) Analysis of Ca2+ spiking in root hairs. Each trace is from a single root hair using seedlings of wild-type (Gifu B-129), or the Ljnsp2 mutant. After about 20 min from injection of the Ca2+-sensitive dye Oregon green-dextran to the root hairs, NF was added to the medium at 10−8 M. Both wild-type (upper) and the mutant (lower) responded positively for the addition of NF in 10–20 min and similar effects were observed with at least 10 independent seedlings." path-from-xml="dsl017f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6982127/dsl017f1.jpeg" path-from-xml="dsl017f1.jpeg" target="_blank">View large</a><a section="6982127" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/dsl017f1.jpeg?Expires=2147483647&Signature=wF7DA7a9JjUi69gANTEox7UfYTy-hMnM2y0DSSVo3sTmRRlfgSCrp6fn831~LA7AhoVbTW7P-NNhm-r4GtAUkOLumV24EMy3vpfH8fnbP7L0s4cOKQiIFSEjNSf-HfiQgt93y4QZ8iFUiT7AcDBfoh2eh1IWLrKJeNOcKZCMO83geswuPe98-9WKpS~b94jDRXBmvSJPB7MYFAy6zmqMhDqAZRINKuwtgWBte-WJyAhMmG-w1VXk32QZyOoXdLb2s4JgI2HGWGsfdAzCtiTw6PHi9iJ-n-xGcmPS3zR73xT1Wwplz3BEdR-s490I5~c3mQUAhCZdcW~lKTykQjrbyQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6982127&ar=464686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl017f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>L. japonicus</em> root hair responses induced by NF. (A–D) Root hair deformation induced with 10<sup>−8</sup> M NF. Slight swelling and branching were observed also in <em>Ljnsp2-1</em> mutant (C and D), although to a lesser extent than the wild-type (Gifu B-129; A and B). (E<strong>–</strong>H) Non-treated root hairs look similar between the mutant (G and H) and wild-type (E and F). Scale bar: 50 μm. (I) Analysis of Ca<sup>2+</sup> spiking in root hairs. Each trace is from a single root hair using seedlings of wild-type (Gifu B-129), or the <em>Ljnsp2</em> mutant. After about 20 min from injection of the Ca<sup>2+</sup>-sensitive dye Oregon green-dextran to the root hairs, NF was added to the medium at 10<sup>−8</sup> M. Both wild-type (upper) and the mutant (lower) responded positively for the addition of NF in 10–20 min and similar effects were observed with at least 10 independent seedlings.</p></div></div><div content-id="fig1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/m_dsl017f1.jpeg?Expires=2147483647&Signature=WLVJ4mJVwOMzYnvEZVtC-zcn697jx~KmwQassmwz9S65sWWbawqwyFdZkW~9pHvOtM1YfPnl4P13i6kDAPUVC0QbOmRYznPwhULTysSI0eeVcVKUAdhJNSUzsbGzWFiVe7Dq4Hodyi6LvFgaQokAgypts38lB22cAaGLrLyOhW7Sny7naOkBhUu1bS~2gLTjxEtKbeB8HD1985LXVjmIF7NIk0j1xmCylM6VHHbSb173vHo26tvhjaxB96x~Mi1pMawalumFY9oFVQ8HOkgirpdiKhiYNjnHWwbA8jf6~5gzVCXo32g56jm1zige3NE~P4F3RRwRyvlHTbGM6Sjmxg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="L. japonicus root hair responses induced by NF. (A–D) Root hair deformation induced with 10−8 M NF. Slight swelling and branching were observed also in Ljnsp2-1 mutant (C and D), although to a lesser extent than the wild-type (Gifu B-129; A and B). (E–H) Non-treated root hairs look similar between the mutant (G and H) and wild-type (E and F). Scale bar: 50 μm. (I) Analysis of Ca2+ spiking in root hairs. Each trace is from a single root hair using seedlings of wild-type (Gifu B-129), or the Ljnsp2 mutant. After about 20 min from injection of the Ca2+-sensitive dye Oregon green-dextran to the root hairs, NF was added to the medium at 10−8 M. Both wild-type (upper) and the mutant (lower) responded positively for the addition of NF in 10–20 min and similar effects were observed with at least 10 independent seedlings." path-from-xml="dsl017f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6982127/dsl017f1.jpeg" path-from-xml="dsl017f1.jpeg" target="_blank">View large</a><a section="6982127" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/dsl017f1.jpeg?Expires=2147483647&Signature=wF7DA7a9JjUi69gANTEox7UfYTy-hMnM2y0DSSVo3sTmRRlfgSCrp6fn831~LA7AhoVbTW7P-NNhm-r4GtAUkOLumV24EMy3vpfH8fnbP7L0s4cOKQiIFSEjNSf-HfiQgt93y4QZ8iFUiT7AcDBfoh2eh1IWLrKJeNOcKZCMO83geswuPe98-9WKpS~b94jDRXBmvSJPB7MYFAy6zmqMhDqAZRINKuwtgWBte-WJyAhMmG-w1VXk32QZyOoXdLb2s4JgI2HGWGsfdAzCtiTw6PHi9iJ-n-xGcmPS3zR73xT1Wwplz3BEdR-s490I5~c3mQUAhCZdcW~lKTykQjrbyQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6982127&ar=464686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl017f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>L. japonicus</em> root hair responses induced by NF. (A–D) Root hair deformation induced with 10<sup>−8</sup> M NF. Slight swelling and branching were observed also in <em>Ljnsp2-1</em> mutant (C and D), although to a lesser extent than the wild-type (Gifu B-129; A and B). (E<strong>–</strong>H) Non-treated root hairs look similar between the mutant (G and H) and wild-type (E and F). Scale bar: 50 μm. (I) Analysis of Ca<sup>2+</sup> spiking in root hairs. Each trace is from a single root hair using seedlings of wild-type (Gifu B-129), or the <em>Ljnsp2</em> mutant. After about 20 min from injection of the Ca<sup>2+</sup>-sensitive dye Oregon green-dextran to the root hairs, NF was added to the medium at 10<sup>−8</sup> M. Both wild-type (upper) and the mutant (lower) responded positively for the addition of NF in 10–20 min and similar effects were observed with at least 10 independent seedlings.</p></div></div>                            <h3 scrollto-destination=6982129 id="6982129" class="section-title" >3.2. High-resolution genetic mapping of LjNSP2</h3>
<p>The mutant allele <em>Ljnsp2-1</em> (Gifu B-129) was crossed to the early flowering ecotype Miyakojima MG-20. The <em>LjNSP2</em> locus was mapped near translocation site of the short arm of chromosome 1.<a reveal-id="b31" data-open="b31" class="link link-ref link-reveal xref-bibr">31</a> Since the recombination is significantly suppressed in the chromosomal segment, total of 4096 <em>Eco</em>RI/<em>Mse</em>I primer combinations were examined for bulked segregant analysis on bulks 10 Nod<sup>−</sup> recessive and Nod<sup>+</sup> dominant homozygous F2 plants in the cross of <em>Ljnsp2-1</em> and Miyakojima MG-20. <em>LjNSP2</em>-linked AFLP markers from this screen were further analyzed in additional 2989 F2 plants with HEGS/AFLP system. Nineteen markers are located on the southern side of <em>LjNSP2</em> locus at distances of 0.07 cM and seven markers on the northern side of <em>LjNSP2</em> locus at distances of 0.45 cM, respectively, while nine markers cosegregated with <em>LjNSP2</em> locus (<a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>, <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). The screened markers were converted into the 11 polymorphic SCAR markers (<a reveal-id="tbl2" data-open="tbl2" class="link link-reveal link-table xref-fig">Table 2</a>).</p>                        <a id="6982131" scrollto-destination="6982131"></a>
<div id="fig2" data-id="fig2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/m_dsl017f2.jpeg?Expires=2147483647&Signature=BgpKx0LQbn74FOqio0FJGVWdudcOy-DA2q-NGOBMkMS4lm5b5h4Yw6sVmWKMD-HkPT9aFlYmQMfuM4k7PuNyCpRzX1OEJQABNOTuDJcCNYXKXuXFvHfP5r429SdwQvVapresTqsqGMReDrlRmazJUe-Tx3GfuGR8cMav~w9Mk~rYR44sC5OoQ5gg02xewHsuGc8Cm0MRGGJfrjKxyttD~-KU2aOfv4c-3uhWGcMfhfEw8XGF7vrVTn-ZZX2HIi4my-WhBPA-kkmZ00pAbh7q2eguRZea1jrG5zYYhZZ68X2fEuJXAWCFWmQGpnPW~7WJLlkoxgmUsg4nyudOX4jNMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Positional cloning of the LjNSP2 gene (A) Genetic map of LjNSP2 region. This map is based on the analysis of 3 009 F2 plant of a cross between Ljnsp2-1 and Miyakojima MG-20. These HEGS/AFLP makers were selected from 4 096 of AFLP primer combinations with HEGS system. The markers conversed into polymorphic SCAR markers are underlined. (B) About 2 Mb of BAC contig around LjNSP2 genomic region. The numbers below the markers indicates recombination events detected in 16 944 chromosomes from 8 472 siblings of the heterozygotes from crosses of Ljnsp2-1 × L. burttii21. Only overlapping BAC clones spanning LjNSP2 are displayed in this figure. Finally, LjNSP2 region was delimited within a 130 kb region on a 211 kb BAC clone (188C5). (C) Putative genes in LjNSP2 region. ORFs of putative genes (closed arrows) and retro-elements (RE; open arrows) are marked in this 130 kb region. Predicted genes are GTK, glutamine transferase K; P4P5K, phosphatidilinositol-4-phosphate-5-kinase; UP, unknown proteins. There were no introns in the 1 500 bp of the predicted LjNSP2 ORF." path-from-xml="dsl017f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6982131/dsl017f2.jpeg" path-from-xml="dsl017f2.jpeg" target="_blank">View large</a><a section="6982131" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/dsl017f2.jpeg?Expires=2147483647&Signature=YZp93bOGmAVE7dIq-BSvAeyYXKX9lhcm9aDgZrK1ZvwvJoU8BG6j~qwc~4qU6Y~ssLn6plK38EJ84TCPb9gDY6xpR6NrUMlA48RmFLvizoR6~ixWKjjDPur~A6e9PPEHRA34rj5IN7CuksiRkJxM18yU0zcUmWTMY-QZfwdtGR65lY~7f2S5lTD3e4p0H4hxNhseCInSq5DZLPFi00HG3w9XB1Qt5bzF96tIZszQ7tTD9MImW09Mq~cOKyRX4AkCpX8x4vys9a4yPgxhazaQkgEDWvNkjlw9Ww1j9i13wRcOKLo0f1Ul24qRCIaCNRhb3L7UIOASrBM8G8GRHYxcIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6982131&ar=464686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl017f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Positional cloning of the <em>LjNSP2</em> gene (A) Genetic map of <em>LjNSP2</em> region. This map is based on the analysis of 3 009 F2 plant of a cross between <em>Ljnsp2-1</em> and Miyakojima MG-20. These HEGS/AFLP makers were selected from 4 096 of AFLP primer combinations with HEGS system. The markers conversed into polymorphic SCAR markers are underlined. (B) About 2 Mb of BAC contig around <em>LjNSP2</em> genomic region. The numbers below the markers indicates recombination events detected in 16 944 chromosomes from 8 472 siblings of the heterozygotes from crosses of <em>Ljnsp2-1</em> × <em>L. burttii</em><sup>21</sup>. Only overlapping BAC clones spanning <em>LjNSP2</em> are displayed in this figure. Finally, <em>LjNSP2</em> region was delimited within a 130 kb region on a 211 kb BAC clone (188C5). (C) Putative genes in <em>LjNSP2</em> region. ORFs of putative genes (closed arrows) and retro-elements (RE; open arrows) are marked in this 130 kb region. Predicted genes are GTK, glutamine transferase K; P4P5K, phosphatidilinositol-4-phosphate-5-kinase; UP, unknown proteins. There were no introns in the 1 500 bp of the predicted <em>LjNSP2</em> ORF.</p></div></div><div content-id="fig2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/m_dsl017f2.jpeg?Expires=2147483647&Signature=BgpKx0LQbn74FOqio0FJGVWdudcOy-DA2q-NGOBMkMS4lm5b5h4Yw6sVmWKMD-HkPT9aFlYmQMfuM4k7PuNyCpRzX1OEJQABNOTuDJcCNYXKXuXFvHfP5r429SdwQvVapresTqsqGMReDrlRmazJUe-Tx3GfuGR8cMav~w9Mk~rYR44sC5OoQ5gg02xewHsuGc8Cm0MRGGJfrjKxyttD~-KU2aOfv4c-3uhWGcMfhfEw8XGF7vrVTn-ZZX2HIi4my-WhBPA-kkmZ00pAbh7q2eguRZea1jrG5zYYhZZ68X2fEuJXAWCFWmQGpnPW~7WJLlkoxgmUsg4nyudOX4jNMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Positional cloning of the LjNSP2 gene (A) Genetic map of LjNSP2 region. This map is based on the analysis of 3 009 F2 plant of a cross between Ljnsp2-1 and Miyakojima MG-20. These HEGS/AFLP makers were selected from 4 096 of AFLP primer combinations with HEGS system. The markers conversed into polymorphic SCAR markers are underlined. (B) About 2 Mb of BAC contig around LjNSP2 genomic region. The numbers below the markers indicates recombination events detected in 16 944 chromosomes from 8 472 siblings of the heterozygotes from crosses of Ljnsp2-1 × L. burttii21. Only overlapping BAC clones spanning LjNSP2 are displayed in this figure. Finally, LjNSP2 region was delimited within a 130 kb region on a 211 kb BAC clone (188C5). (C) Putative genes in LjNSP2 region. ORFs of putative genes (closed arrows) and retro-elements (RE; open arrows) are marked in this 130 kb region. Predicted genes are GTK, glutamine transferase K; P4P5K, phosphatidilinositol-4-phosphate-5-kinase; UP, unknown proteins. There were no introns in the 1 500 bp of the predicted LjNSP2 ORF." path-from-xml="dsl017f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6982131/dsl017f2.jpeg" path-from-xml="dsl017f2.jpeg" target="_blank">View large</a><a section="6982131" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/dsl017f2.jpeg?Expires=2147483647&Signature=YZp93bOGmAVE7dIq-BSvAeyYXKX9lhcm9aDgZrK1ZvwvJoU8BG6j~qwc~4qU6Y~ssLn6plK38EJ84TCPb9gDY6xpR6NrUMlA48RmFLvizoR6~ixWKjjDPur~A6e9PPEHRA34rj5IN7CuksiRkJxM18yU0zcUmWTMY-QZfwdtGR65lY~7f2S5lTD3e4p0H4hxNhseCInSq5DZLPFi00HG3w9XB1Qt5bzF96tIZszQ7tTD9MImW09Mq~cOKyRX4AkCpX8x4vys9a4yPgxhazaQkgEDWvNkjlw9Ww1j9i13wRcOKLo0f1Ul24qRCIaCNRhb3L7UIOASrBM8G8GRHYxcIg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6982131&ar=464686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl017f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Positional cloning of the <em>LjNSP2</em> gene (A) Genetic map of <em>LjNSP2</em> region. This map is based on the analysis of 3 009 F2 plant of a cross between <em>Ljnsp2-1</em> and Miyakojima MG-20. These HEGS/AFLP makers were selected from 4 096 of AFLP primer combinations with HEGS system. The markers conversed into polymorphic SCAR markers are underlined. (B) About 2 Mb of BAC contig around <em>LjNSP2</em> genomic region. The numbers below the markers indicates recombination events detected in 16 944 chromosomes from 8 472 siblings of the heterozygotes from crosses of <em>Ljnsp2-1</em> × <em>L. burttii</em><sup>21</sup>. Only overlapping BAC clones spanning <em>LjNSP2</em> are displayed in this figure. Finally, <em>LjNSP2</em> region was delimited within a 130 kb region on a 211 kb BAC clone (188C5). (C) Putative genes in <em>LjNSP2</em> region. ORFs of putative genes (closed arrows) and retro-elements (RE; open arrows) are marked in this 130 kb region. Predicted genes are GTK, glutamine transferase K; P4P5K, phosphatidilinositol-4-phosphate-5-kinase; UP, unknown proteins. There were no introns in the 1 500 bp of the predicted <em>LjNSP2</em> ORF.</p></div></div>                        <a id="6982133" scrollto-destination="6982133"></a>
<div content-id="tbl1" class="table-modal"><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>AFLP markers linked to <em>LjNSP2</em> locus</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>AFLP marker </th><th>Primer combination </th><th>Approximate length of the AFLP fragment (bp)<sup>a</sup> </th><th>Marker type </th></tr></thead><tbody><tr><td>EM117 </td><td>E-CGA/M-GTG </td><td>700 </td><td>Dominant </td></tr><tr><td>EM140 </td><td>E-CTT/M-CAA </td><td>700 </td><td>Co-dominant </td></tr><tr><td>EM157 </td><td>E-GAC/M-GCC </td><td>130 </td><td>Dominant </td></tr><tr><td>EM177 </td><td>E-GCA/M-CAA </td><td>550 </td><td>Dominant </td></tr><tr><td>EM205 </td><td>E-GGC/M-TGG </td><td>1500 </td><td>Dominant </td></tr><tr><td>EM242 </td><td>E-TAG/M-CTC </td><td>130 </td><td>Dominant </td></tr><tr><td>EM262 </td><td>E-TCT/M-ACT </td><td>500 </td><td>Co-dominant </td></tr><tr><td>EM265 </td><td>E-TCT/M-GGC </td><td>250 </td><td>Co-dominant </td></tr><tr><td>EM299 </td><td>E-TTA/M-TGT </td><td>250 </td><td>Dominant </td></tr><tr><td>EM341 </td><td>E-AAT/M-GTT </td><td>500 </td><td>Co-dominant </td></tr><tr><td>EM367 </td><td>E-ACG/M-GGA </td><td>400 </td><td>Dominant </td></tr><tr><td>EM390 </td><td>E-AGC/M-AAA </td><td>1000 </td><td>Co-dominant </td></tr><tr><td>EM480 </td><td>E-CCA/M-GCA </td><td>190 </td><td>Dominant </td></tr><tr><td>EM527 </td><td>E-CGT/M-CTC </td><td>70 </td><td>Dominant </td></tr><tr><td>EM582 </td><td>E-GAC/M-TGG </td><td>800 </td><td>Dominant </td></tr><tr><td>EM603 </td><td>E-GCA/M-TTA </td><td>800 </td><td>Dominant </td></tr><tr><td>EM713 </td><td>E-TAC/M-AAA </td><td>400 </td><td>Dominant </td></tr><tr><td>EM784 </td><td>E-TCT/M-TTC </td><td>100 </td><td>Dominant </td></tr><tr><td>EM807 </td><td>E-TGG/M-CGA </td><td>90 </td><td>Dominant </td></tr><tr><td>EM011 </td><td>E-AAG/M-TTC </td><td>200 </td><td>Dominant </td></tr><tr><td>EM013 </td><td>E-AAT/M-CCT </td><td>120 </td><td>Dominant </td></tr><tr><td>EM116 </td><td>E-CGA/M-GGA </td><td>500 </td><td>Dominant </td></tr><tr><td>EM160 </td><td>E-GAG/M-AAT </td><td>250 </td><td>Co-dominant </td></tr><tr><td>EM190 </td><td>E-GCT/M-CAA </td><td>350 </td><td>Co-dominant </td></tr><tr><td>EM261 </td><td>E-TCG/M-TTG </td><td>250 </td><td>Co-dominant </td></tr><tr><td>EM283 </td><td>E-TGG/M-CTG </td><td>500 </td><td>Dominant </td></tr><tr><td>EM350 </td><td>E-ACA/M-TCA </td><td>250 </td><td>Dominant </td></tr><tr><td>EM466 </td><td>E-CAT/M-ACT </td><td>800 </td><td>Dominant </td></tr><tr><td>EM130 </td><td>E-CGT/M-TGG </td><td>600 </td><td>Dominant </td></tr><tr><td>EM686 </td><td>E-GTC/M-GAT </td><td>600 </td><td>Dominant </td></tr><tr><td>EM163 </td><td>E-GAG/M-GTC </td><td>300 </td><td>Co-dominant </td></tr><tr><td>EM115 </td><td>E-CGA/M-AGT </td><td>400 </td><td>Co-dominant </td></tr><tr><td>EM214 </td><td>E-GGG/M-TTA </td><td>850 </td><td>Dominant </td></tr><tr><td>EM231 </td><td>E-GTT/M-TCT </td><td>80 </td><td>Dominant </td></tr><tr><td>EM802 </td><td>E-TGG/M-AAT </td><td>200 </td><td>Dominant </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th>AFLP marker </th><th>Primer combination </th><th>Approximate length of the AFLP fragment (bp)<sup>a</sup> </th><th>Marker type </th></tr></thead><tbody><tr><td>EM117 </td><td>E-CGA/M-GTG </td><td>700 </td><td>Dominant </td></tr><tr><td>EM140 </td><td>E-CTT/M-CAA </td><td>700 </td><td>Co-dominant </td></tr><tr><td>EM157 </td><td>E-GAC/M-GCC </td><td>130 </td><td>Dominant </td></tr><tr><td>EM177 </td><td>E-GCA/M-CAA </td><td>550 </td><td>Dominant </td></tr><tr><td>EM205 </td><td>E-GGC/M-TGG </td><td>1500 </td><td>Dominant </td></tr><tr><td>EM242 </td><td>E-TAG/M-CTC </td><td>130 </td><td>Dominant </td></tr><tr><td>EM262 </td><td>E-TCT/M-ACT </td><td>500 </td><td>Co-dominant </td></tr><tr><td>EM265 </td><td>E-TCT/M-GGC </td><td>250 </td><td>Co-dominant </td></tr><tr><td>EM299 </td><td>E-TTA/M-TGT </td><td>250 </td><td>Dominant </td></tr><tr><td>EM341 </td><td>E-AAT/M-GTT </td><td>500 </td><td>Co-dominant </td></tr><tr><td>EM367 </td><td>E-ACG/M-GGA </td><td>400 </td><td>Dominant </td></tr><tr><td>EM390 </td><td>E-AGC/M-AAA </td><td>1000 </td><td>Co-dominant </td></tr><tr><td>EM480 </td><td>E-CCA/M-GCA </td><td>190 </td><td>Dominant </td></tr><tr><td>EM527 </td><td>E-CGT/M-CTC </td><td>70 </td><td>Dominant </td></tr><tr><td>EM582 </td><td>E-GAC/M-TGG </td><td>800 </td><td>Dominant </td></tr><tr><td>EM603 </td><td>E-GCA/M-TTA </td><td>800 </td><td>Dominant </td></tr><tr><td>EM713 </td><td>E-TAC/M-AAA </td><td>400 </td><td>Dominant </td></tr><tr><td>EM784 </td><td>E-TCT/M-TTC </td><td>100 </td><td>Dominant </td></tr><tr><td>EM807 </td><td>E-TGG/M-CGA </td><td>90 </td><td>Dominant </td></tr><tr><td>EM011 </td><td>E-AAG/M-TTC </td><td>200 </td><td>Dominant </td></tr><tr><td>EM013 </td><td>E-AAT/M-CCT </td><td>120 </td><td>Dominant </td></tr><tr><td>EM116 </td><td>E-CGA/M-GGA </td><td>500 </td><td>Dominant </td></tr><tr><td>EM160 </td><td>E-GAG/M-AAT </td><td>250 </td><td>Co-dominant </td></tr><tr><td>EM190 </td><td>E-GCT/M-CAA </td><td>350 </td><td>Co-dominant </td></tr><tr><td>EM261 </td><td>E-TCG/M-TTG </td><td>250 </td><td>Co-dominant </td></tr><tr><td>EM283 </td><td>E-TGG/M-CTG </td><td>500 </td><td>Dominant </td></tr><tr><td>EM350 </td><td>E-ACA/M-TCA </td><td>250 </td><td>Dominant </td></tr><tr><td>EM466 </td><td>E-CAT/M-ACT </td><td>800 </td><td>Dominant </td></tr><tr><td>EM130 </td><td>E-CGT/M-TGG </td><td>600 </td><td>Dominant </td></tr><tr><td>EM686 </td><td>E-GTC/M-GAT </td><td>600 </td><td>Dominant </td></tr><tr><td>EM163 </td><td>E-GAG/M-GTC </td><td>300 </td><td>Co-dominant </td></tr><tr><td>EM115 </td><td>E-CGA/M-AGT </td><td>400 </td><td>Co-dominant </td></tr><tr><td>EM214 </td><td>E-GGG/M-TTA </td><td>850 </td><td>Dominant </td></tr><tr><td>EM231 </td><td>E-GTT/M-TCT </td><td>80 </td><td>Dominant </td></tr><tr><td>EM802 </td><td>E-TGG/M-AAT </td><td>200 </td><td>Dominant </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>36 <em>LjNSP2</em>-linked AFLP markers were selected from 3009 F2/F3 plants in the cross of <em>Ljnsp2-1</em> (Gifu B-129) and Miyakojima MG-20.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>AFLP fragments were derived from Miyakojima MG-20.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6982133" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>AFLP markers linked to <em>LjNSP2</em> locus</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>AFLP marker </th><th>Primer combination </th><th>Approximate length of the AFLP fragment (bp)<sup>a</sup> </th><th>Marker type </th></tr></thead><tbody><tr><td>EM117 </td><td>E-CGA/M-GTG </td><td>700 </td><td>Dominant </td></tr><tr><td>EM140 </td><td>E-CTT/M-CAA </td><td>700 </td><td>Co-dominant </td></tr><tr><td>EM157 </td><td>E-GAC/M-GCC </td><td>130 </td><td>Dominant </td></tr><tr><td>EM177 </td><td>E-GCA/M-CAA </td><td>550 </td><td>Dominant </td></tr><tr><td>EM205 </td><td>E-GGC/M-TGG </td><td>1500 </td><td>Dominant </td></tr><tr><td>EM242 </td><td>E-TAG/M-CTC </td><td>130 </td><td>Dominant </td></tr><tr><td>EM262 </td><td>E-TCT/M-ACT </td><td>500 </td><td>Co-dominant </td></tr><tr><td>EM265 </td><td>E-TCT/M-GGC </td><td>250 </td><td>Co-dominant </td></tr><tr><td>EM299 </td><td>E-TTA/M-TGT </td><td>250 </td><td>Dominant </td></tr><tr><td>EM341 </td><td>E-AAT/M-GTT </td><td>500 </td><td>Co-dominant </td></tr><tr><td>EM367 </td><td>E-ACG/M-GGA </td><td>400 </td><td>Dominant </td></tr><tr><td>EM390 </td><td>E-AGC/M-AAA </td><td>1000 </td><td>Co-dominant </td></tr><tr><td>EM480 </td><td>E-CCA/M-GCA </td><td>190 </td><td>Dominant </td></tr><tr><td>EM527 </td><td>E-CGT/M-CTC </td><td>70 </td><td>Dominant </td></tr><tr><td>EM582 </td><td>E-GAC/M-TGG </td><td>800 </td><td>Dominant </td></tr><tr><td>EM603 </td><td>E-GCA/M-TTA </td><td>800 </td><td>Dominant </td></tr><tr><td>EM713 </td><td>E-TAC/M-AAA </td><td>400 </td><td>Dominant </td></tr><tr><td>EM784 </td><td>E-TCT/M-TTC </td><td>100 </td><td>Dominant </td></tr><tr><td>EM807 </td><td>E-TGG/M-CGA </td><td>90 </td><td>Dominant </td></tr><tr><td>EM011 </td><td>E-AAG/M-TTC </td><td>200 </td><td>Dominant </td></tr><tr><td>EM013 </td><td>E-AAT/M-CCT </td><td>120 </td><td>Dominant </td></tr><tr><td>EM116 </td><td>E-CGA/M-GGA </td><td>500 </td><td>Dominant </td></tr><tr><td>EM160 </td><td>E-GAG/M-AAT </td><td>250 </td><td>Co-dominant </td></tr><tr><td>EM190 </td><td>E-GCT/M-CAA </td><td>350 </td><td>Co-dominant </td></tr><tr><td>EM261 </td><td>E-TCG/M-TTG </td><td>250 </td><td>Co-dominant </td></tr><tr><td>EM283 </td><td>E-TGG/M-CTG </td><td>500 </td><td>Dominant </td></tr><tr><td>EM350 </td><td>E-ACA/M-TCA </td><td>250 </td><td>Dominant </td></tr><tr><td>EM466 </td><td>E-CAT/M-ACT </td><td>800 </td><td>Dominant </td></tr><tr><td>EM130 </td><td>E-CGT/M-TGG </td><td>600 </td><td>Dominant </td></tr><tr><td>EM686 </td><td>E-GTC/M-GAT </td><td>600 </td><td>Dominant </td></tr><tr><td>EM163 </td><td>E-GAG/M-GTC </td><td>300 </td><td>Co-dominant </td></tr><tr><td>EM115 </td><td>E-CGA/M-AGT </td><td>400 </td><td>Co-dominant </td></tr><tr><td>EM214 </td><td>E-GGG/M-TTA </td><td>850 </td><td>Dominant </td></tr><tr><td>EM231 </td><td>E-GTT/M-TCT </td><td>80 </td><td>Dominant </td></tr><tr><td>EM802 </td><td>E-TGG/M-AAT </td><td>200 </td><td>Dominant </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th>AFLP marker </th><th>Primer combination </th><th>Approximate length of the AFLP fragment (bp)<sup>a</sup> </th><th>Marker type </th></tr></thead><tbody><tr><td>EM117 </td><td>E-CGA/M-GTG </td><td>700 </td><td>Dominant </td></tr><tr><td>EM140 </td><td>E-CTT/M-CAA </td><td>700 </td><td>Co-dominant </td></tr><tr><td>EM157 </td><td>E-GAC/M-GCC </td><td>130 </td><td>Dominant </td></tr><tr><td>EM177 </td><td>E-GCA/M-CAA </td><td>550 </td><td>Dominant </td></tr><tr><td>EM205 </td><td>E-GGC/M-TGG </td><td>1500 </td><td>Dominant </td></tr><tr><td>EM242 </td><td>E-TAG/M-CTC </td><td>130 </td><td>Dominant </td></tr><tr><td>EM262 </td><td>E-TCT/M-ACT </td><td>500 </td><td>Co-dominant </td></tr><tr><td>EM265 </td><td>E-TCT/M-GGC </td><td>250 </td><td>Co-dominant </td></tr><tr><td>EM299 </td><td>E-TTA/M-TGT </td><td>250 </td><td>Dominant </td></tr><tr><td>EM341 </td><td>E-AAT/M-GTT </td><td>500 </td><td>Co-dominant </td></tr><tr><td>EM367 </td><td>E-ACG/M-GGA </td><td>400 </td><td>Dominant </td></tr><tr><td>EM390 </td><td>E-AGC/M-AAA </td><td>1000 </td><td>Co-dominant </td></tr><tr><td>EM480 </td><td>E-CCA/M-GCA </td><td>190 </td><td>Dominant </td></tr><tr><td>EM527 </td><td>E-CGT/M-CTC </td><td>70 </td><td>Dominant </td></tr><tr><td>EM582 </td><td>E-GAC/M-TGG </td><td>800 </td><td>Dominant </td></tr><tr><td>EM603 </td><td>E-GCA/M-TTA </td><td>800 </td><td>Dominant </td></tr><tr><td>EM713 </td><td>E-TAC/M-AAA </td><td>400 </td><td>Dominant </td></tr><tr><td>EM784 </td><td>E-TCT/M-TTC </td><td>100 </td><td>Dominant </td></tr><tr><td>EM807 </td><td>E-TGG/M-CGA </td><td>90 </td><td>Dominant </td></tr><tr><td>EM011 </td><td>E-AAG/M-TTC </td><td>200 </td><td>Dominant </td></tr><tr><td>EM013 </td><td>E-AAT/M-CCT </td><td>120 </td><td>Dominant </td></tr><tr><td>EM116 </td><td>E-CGA/M-GGA </td><td>500 </td><td>Dominant </td></tr><tr><td>EM160 </td><td>E-GAG/M-AAT </td><td>250 </td><td>Co-dominant </td></tr><tr><td>EM190 </td><td>E-GCT/M-CAA </td><td>350 </td><td>Co-dominant </td></tr><tr><td>EM261 </td><td>E-TCG/M-TTG </td><td>250 </td><td>Co-dominant </td></tr><tr><td>EM283 </td><td>E-TGG/M-CTG </td><td>500 </td><td>Dominant </td></tr><tr><td>EM350 </td><td>E-ACA/M-TCA </td><td>250 </td><td>Dominant </td></tr><tr><td>EM466 </td><td>E-CAT/M-ACT </td><td>800 </td><td>Dominant </td></tr><tr><td>EM130 </td><td>E-CGT/M-TGG </td><td>600 </td><td>Dominant </td></tr><tr><td>EM686 </td><td>E-GTC/M-GAT </td><td>600 </td><td>Dominant </td></tr><tr><td>EM163 </td><td>E-GAG/M-GTC </td><td>300 </td><td>Co-dominant </td></tr><tr><td>EM115 </td><td>E-CGA/M-AGT </td><td>400 </td><td>Co-dominant </td></tr><tr><td>EM214 </td><td>E-GGG/M-TTA </td><td>850 </td><td>Dominant </td></tr><tr><td>EM231 </td><td>E-GTT/M-TCT </td><td>80 </td><td>Dominant </td></tr><tr><td>EM802 </td><td>E-TGG/M-AAT </td><td>200 </td><td>Dominant </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>36 <em>LjNSP2</em>-linked AFLP markers were selected from 3009 F2/F3 plants in the cross of <em>Ljnsp2-1</em> (Gifu B-129) and Miyakojima MG-20.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>AFLP fragments were derived from Miyakojima MG-20.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6982133" target="_blank">View Large</a></div></div>                        <a id="6982135" scrollto-destination="6982135"></a>
<div content-id="tbl2" class="table-modal"><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>PCR-based markers linked to <em>LjNSP2</em> locus</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>PCR marker </th><th>Origin </th><th>Sequence (5′→3′) </th><th>Anealing temperature (°C) </th><th>Length (bp)<sup>a</sup> </th><th>Marker type </th></tr></thead><tbody><tr><td>SEM140 </td><td>EM140 </td><td>GAATTCCTTCCCGGTTCTTC </td><td>65 </td><td>750 </td><td>Dominant (M)<sup>b</sup> </td></tr><tr><td> </td><td> </td><td>TTAACAACAGCAACTTCACT </td><td> </td><td> </td><td> </td></tr><tr><td>SEM299 </td><td>EM299 </td><td>GAATTCTTACCGAGTGAGAT </td><td>60 </td><td>250 </td><td>Dominant (M, b) </td></tr><tr><td> </td><td> </td><td>TTAATGTTTCTGACAAAGGC </td><td> </td><td> </td><td> </td></tr><tr><td>SEM341 </td><td>EM341 </td><td>GAATTCAATGATTGTTGGGA </td><td>50 </td><td>500 </td><td>Co-dominant (M, b) </td></tr><tr><td> </td><td> </td><td>TTAAGTTGTTCGAATTGATA </td><td> </td><td> </td><td> </td></tr><tr><td>SEM390 </td><td>EM390 </td><td>GAATTCAGCCACAGCCTCTG </td><td>50 </td><td>230 </td><td>Dominant (M) </td></tr><tr><td> </td><td> </td><td>AACCACAAGTCAAATGCAAC </td><td> </td><td> </td><td> </td></tr><tr><td>SEM160 </td><td>EM160 </td><td>GAATTCGAGAATTGGAGAAG </td><td>60 </td><td>220 </td><td>Dominant (M) </td></tr><tr><td> </td><td> </td><td>TTAAAATCAAACCCTAACAA </td><td> </td><td> </td><td> </td></tr><tr><td>SEM190 </td><td>EM190 </td><td>GAATTCGCTTGCAAGTAGTG </td><td>50 </td><td>340 </td><td>Co-dominant (M) </td></tr><tr><td> </td><td> </td><td>TTAACAAGAGTCTCATAATC </td><td> </td><td> </td><td> </td></tr><tr><td>SEM261 </td><td>EM261 </td><td>TTCTCGGCGGTTCCACCAAT </td><td>68 </td><td>290 </td><td>Dominant (M) </td></tr><tr><td> </td><td> </td><td>GCTTGTTTGAAAGTGGCTTT </td><td> </td><td> </td><td>Co-dominant (b) </td></tr><tr><td>SEM115 </td><td>EM115 </td><td>CCGATTACATTTATGAGTAC </td><td>50 </td><td>350 </td><td>Co-dominant (M, b) </td></tr><tr><td> </td><td> </td><td>TTAAAGTCGATAGACCGACG </td><td> </td><td> </td><td> </td></tr><tr><td>SEM163 </td><td>EM163 </td><td>GAATTCGAGATCCAGAATCT </td><td>60 </td><td>290 </td><td>Dominant (M) </td></tr><tr><td> </td><td> </td><td>TTAAGTCTGAACCATTGCTA </td><td> </td><td> </td><td> </td></tr><tr><td>SEM214 </td><td>EM214 </td><td>GAATTCGGGGTGTTACACCTC </td><td>65 </td><td>580 </td><td>Dominant (M) </td></tr><tr><td> </td><td> </td><td>TTAATTACCCCCTCGTTCTC </td><td> </td><td> </td><td> </td></tr><tr><td>SEM802 </td><td>EM802 </td><td>GAATTCTGGAAGCACGTTGA </td><td>60 </td><td>190 </td><td>Co-dominant (M) </td></tr><tr><td> </td><td> </td><td>TTAAAATCATCATCTGAGAG </td><td> </td><td> </td><td> </td></tr><tr><td>s26d </td><td>188C5 </td><td>TCATGTTTAGCGCTCTGATT </td><td>60 </td><td>100 </td><td>Co-dominant (M, b) </td></tr><tr><td> </td><td> </td><td>GCACCGTAGTACCGTCAGTTTCC </td><td> </td><td> </td><td> </td></tr><tr><td>S11 </td><td>188C5 </td><td>TTGCAGGTTTTCAGGAACATT </td><td>60 </td><td>520 </td><td>Co-dominant (M, b) </td></tr><tr><td> </td><td> </td><td>GAAGTTCATTGTAACTCTCAAACG </td><td> </td><td> </td><td> </td></tr><tr><td>183R </td><td>183A2 </td><td>CATCGTATGTTGAAAAGAAGAATGAT </td><td>60 </td><td>580 </td><td>Co-dominant (M, b) </td></tr><tr><td> </td><td> </td><td>TCTTCAGTTCTTCCCTTTATAGCC </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th>PCR marker </th><th>Origin </th><th>Sequence (5′→3′) </th><th>Anealing temperature (°C) </th><th>Length (bp)<sup>a</sup> </th><th>Marker type </th></tr></thead><tbody><tr><td>SEM140 </td><td>EM140 </td><td>GAATTCCTTCCCGGTTCTTC </td><td>65 </td><td>750 </td><td>Dominant (M)<sup>b</sup> </td></tr><tr><td> </td><td> </td><td>TTAACAACAGCAACTTCACT </td><td> </td><td> </td><td> </td></tr><tr><td>SEM299 </td><td>EM299 </td><td>GAATTCTTACCGAGTGAGAT </td><td>60 </td><td>250 </td><td>Dominant (M, b) </td></tr><tr><td> </td><td> </td><td>TTAATGTTTCTGACAAAGGC </td><td> </td><td> </td><td> </td></tr><tr><td>SEM341 </td><td>EM341 </td><td>GAATTCAATGATTGTTGGGA </td><td>50 </td><td>500 </td><td>Co-dominant (M, b) </td></tr><tr><td> </td><td> </td><td>TTAAGTTGTTCGAATTGATA </td><td> </td><td> </td><td> </td></tr><tr><td>SEM390 </td><td>EM390 </td><td>GAATTCAGCCACAGCCTCTG </td><td>50 </td><td>230 </td><td>Dominant (M) </td></tr><tr><td> </td><td> </td><td>AACCACAAGTCAAATGCAAC </td><td> </td><td> </td><td> </td></tr><tr><td>SEM160 </td><td>EM160 </td><td>GAATTCGAGAATTGGAGAAG </td><td>60 </td><td>220 </td><td>Dominant (M) </td></tr><tr><td> </td><td> </td><td>TTAAAATCAAACCCTAACAA </td><td> </td><td> </td><td> </td></tr><tr><td>SEM190 </td><td>EM190 </td><td>GAATTCGCTTGCAAGTAGTG </td><td>50 </td><td>340 </td><td>Co-dominant (M) </td></tr><tr><td> </td><td> </td><td>TTAACAAGAGTCTCATAATC </td><td> </td><td> </td><td> </td></tr><tr><td>SEM261 </td><td>EM261 </td><td>TTCTCGGCGGTTCCACCAAT </td><td>68 </td><td>290 </td><td>Dominant (M) </td></tr><tr><td> </td><td> </td><td>GCTTGTTTGAAAGTGGCTTT </td><td> </td><td> </td><td>Co-dominant (b) </td></tr><tr><td>SEM115 </td><td>EM115 </td><td>CCGATTACATTTATGAGTAC </td><td>50 </td><td>350 </td><td>Co-dominant (M, b) </td></tr><tr><td> </td><td> </td><td>TTAAAGTCGATAGACCGACG </td><td> </td><td> </td><td> </td></tr><tr><td>SEM163 </td><td>EM163 </td><td>GAATTCGAGATCCAGAATCT </td><td>60 </td><td>290 </td><td>Dominant (M) </td></tr><tr><td> </td><td> </td><td>TTAAGTCTGAACCATTGCTA </td><td> </td><td> </td><td> </td></tr><tr><td>SEM214 </td><td>EM214 </td><td>GAATTCGGGGTGTTACACCTC </td><td>65 </td><td>580 </td><td>Dominant (M) </td></tr><tr><td> </td><td> </td><td>TTAATTACCCCCTCGTTCTC </td><td> </td><td> </td><td> </td></tr><tr><td>SEM802 </td><td>EM802 </td><td>GAATTCTGGAAGCACGTTGA </td><td>60 </td><td>190 </td><td>Co-dominant (M) </td></tr><tr><td> </td><td> </td><td>TTAAAATCATCATCTGAGAG </td><td> </td><td> </td><td> </td></tr><tr><td>s26d </td><td>188C5 </td><td>TCATGTTTAGCGCTCTGATT </td><td>60 </td><td>100 </td><td>Co-dominant (M, b) </td></tr><tr><td> </td><td> </td><td>GCACCGTAGTACCGTCAGTTTCC </td><td> </td><td> </td><td> </td></tr><tr><td>S11 </td><td>188C5 </td><td>TTGCAGGTTTTCAGGAACATT </td><td>60 </td><td>520 </td><td>Co-dominant (M, b) </td></tr><tr><td> </td><td> </td><td>GAAGTTCATTGTAACTCTCAAACG </td><td> </td><td> </td><td> </td></tr><tr><td>183R </td><td>183A2 </td><td>CATCGTATGTTGAAAAGAAGAATGAT </td><td>60 </td><td>580 </td><td>Co-dominant (M, b) </td></tr><tr><td> </td><td> </td><td>TCTTCAGTTCTTCCCTTTATAGCC </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>PCR was performed using the indicated primer pairs with the following cycling parameter: 4 min at 94°C; 35 cycles of 30 s at 94°C, 30 s at the temperature indicated, 1 min at 72°C; 10 min at 72°C; and hold at 4°C.</p></div></div><div content-id=""><div class="fn"><p>PCR fragments were electrophoresed on 13% acrylamide gels with HEGS system.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>PCR fragments are derived from Miyakojima MG-20.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>M and b indicates the polymorphism of Miyakojima MG-20 and <em>L. burttii</em>, respectively, compared with Gifu B-129.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6982135" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl2" class="table-wrap-title" data-id="tbl2"><div class="label -label"><strong>Table 2</strong></div><div class="caption"><p>PCR-based markers linked to <em>LjNSP2</em> locus</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>PCR marker </th><th>Origin </th><th>Sequence (5′→3′) </th><th>Anealing temperature (°C) </th><th>Length (bp)<sup>a</sup> </th><th>Marker type </th></tr></thead><tbody><tr><td>SEM140 </td><td>EM140 </td><td>GAATTCCTTCCCGGTTCTTC </td><td>65 </td><td>750 </td><td>Dominant (M)<sup>b</sup> </td></tr><tr><td> </td><td> </td><td>TTAACAACAGCAACTTCACT </td><td> </td><td> </td><td> </td></tr><tr><td>SEM299 </td><td>EM299 </td><td>GAATTCTTACCGAGTGAGAT </td><td>60 </td><td>250 </td><td>Dominant (M, b) </td></tr><tr><td> </td><td> </td><td>TTAATGTTTCTGACAAAGGC </td><td> </td><td> </td><td> </td></tr><tr><td>SEM341 </td><td>EM341 </td><td>GAATTCAATGATTGTTGGGA </td><td>50 </td><td>500 </td><td>Co-dominant (M, b) </td></tr><tr><td> </td><td> </td><td>TTAAGTTGTTCGAATTGATA </td><td> </td><td> </td><td> </td></tr><tr><td>SEM390 </td><td>EM390 </td><td>GAATTCAGCCACAGCCTCTG </td><td>50 </td><td>230 </td><td>Dominant (M) </td></tr><tr><td> </td><td> </td><td>AACCACAAGTCAAATGCAAC </td><td> </td><td> </td><td> </td></tr><tr><td>SEM160 </td><td>EM160 </td><td>GAATTCGAGAATTGGAGAAG </td><td>60 </td><td>220 </td><td>Dominant (M) </td></tr><tr><td> </td><td> </td><td>TTAAAATCAAACCCTAACAA </td><td> </td><td> </td><td> </td></tr><tr><td>SEM190 </td><td>EM190 </td><td>GAATTCGCTTGCAAGTAGTG </td><td>50 </td><td>340 </td><td>Co-dominant (M) </td></tr><tr><td> </td><td> </td><td>TTAACAAGAGTCTCATAATC </td><td> </td><td> </td><td> </td></tr><tr><td>SEM261 </td><td>EM261 </td><td>TTCTCGGCGGTTCCACCAAT </td><td>68 </td><td>290 </td><td>Dominant (M) </td></tr><tr><td> </td><td> </td><td>GCTTGTTTGAAAGTGGCTTT </td><td> </td><td> </td><td>Co-dominant (b) </td></tr><tr><td>SEM115 </td><td>EM115 </td><td>CCGATTACATTTATGAGTAC </td><td>50 </td><td>350 </td><td>Co-dominant (M, b) </td></tr><tr><td> </td><td> </td><td>TTAAAGTCGATAGACCGACG </td><td> </td><td> </td><td> </td></tr><tr><td>SEM163 </td><td>EM163 </td><td>GAATTCGAGATCCAGAATCT </td><td>60 </td><td>290 </td><td>Dominant (M) </td></tr><tr><td> </td><td> </td><td>TTAAGTCTGAACCATTGCTA </td><td> </td><td> </td><td> </td></tr><tr><td>SEM214 </td><td>EM214 </td><td>GAATTCGGGGTGTTACACCTC </td><td>65 </td><td>580 </td><td>Dominant (M) </td></tr><tr><td> </td><td> </td><td>TTAATTACCCCCTCGTTCTC </td><td> </td><td> </td><td> </td></tr><tr><td>SEM802 </td><td>EM802 </td><td>GAATTCTGGAAGCACGTTGA </td><td>60 </td><td>190 </td><td>Co-dominant (M) </td></tr><tr><td> </td><td> </td><td>TTAAAATCATCATCTGAGAG </td><td> </td><td> </td><td> </td></tr><tr><td>s26d </td><td>188C5 </td><td>TCATGTTTAGCGCTCTGATT </td><td>60 </td><td>100 </td><td>Co-dominant (M, b) </td></tr><tr><td> </td><td> </td><td>GCACCGTAGTACCGTCAGTTTCC </td><td> </td><td> </td><td> </td></tr><tr><td>S11 </td><td>188C5 </td><td>TTGCAGGTTTTCAGGAACATT </td><td>60 </td><td>520 </td><td>Co-dominant (M, b) </td></tr><tr><td> </td><td> </td><td>GAAGTTCATTGTAACTCTCAAACG </td><td> </td><td> </td><td> </td></tr><tr><td>183R </td><td>183A2 </td><td>CATCGTATGTTGAAAAGAAGAATGAT </td><td>60 </td><td>580 </td><td>Co-dominant (M, b) </td></tr><tr><td> </td><td> </td><td>TCTTCAGTTCTTCCCTTTATAGCC </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div content-id="tbl2" class="table-modal"><table><thead name="thead" align=""><tr><th>PCR marker </th><th>Origin </th><th>Sequence (5′→3′) </th><th>Anealing temperature (°C) </th><th>Length (bp)<sup>a</sup> </th><th>Marker type </th></tr></thead><tbody><tr><td>SEM140 </td><td>EM140 </td><td>GAATTCCTTCCCGGTTCTTC </td><td>65 </td><td>750 </td><td>Dominant (M)<sup>b</sup> </td></tr><tr><td> </td><td> </td><td>TTAACAACAGCAACTTCACT </td><td> </td><td> </td><td> </td></tr><tr><td>SEM299 </td><td>EM299 </td><td>GAATTCTTACCGAGTGAGAT </td><td>60 </td><td>250 </td><td>Dominant (M, b) </td></tr><tr><td> </td><td> </td><td>TTAATGTTTCTGACAAAGGC </td><td> </td><td> </td><td> </td></tr><tr><td>SEM341 </td><td>EM341 </td><td>GAATTCAATGATTGTTGGGA </td><td>50 </td><td>500 </td><td>Co-dominant (M, b) </td></tr><tr><td> </td><td> </td><td>TTAAGTTGTTCGAATTGATA </td><td> </td><td> </td><td> </td></tr><tr><td>SEM390 </td><td>EM390 </td><td>GAATTCAGCCACAGCCTCTG </td><td>50 </td><td>230 </td><td>Dominant (M) </td></tr><tr><td> </td><td> </td><td>AACCACAAGTCAAATGCAAC </td><td> </td><td> </td><td> </td></tr><tr><td>SEM160 </td><td>EM160 </td><td>GAATTCGAGAATTGGAGAAG </td><td>60 </td><td>220 </td><td>Dominant (M) </td></tr><tr><td> </td><td> </td><td>TTAAAATCAAACCCTAACAA </td><td> </td><td> </td><td> </td></tr><tr><td>SEM190 </td><td>EM190 </td><td>GAATTCGCTTGCAAGTAGTG </td><td>50 </td><td>340 </td><td>Co-dominant (M) </td></tr><tr><td> </td><td> </td><td>TTAACAAGAGTCTCATAATC </td><td> </td><td> </td><td> </td></tr><tr><td>SEM261 </td><td>EM261 </td><td>TTCTCGGCGGTTCCACCAAT </td><td>68 </td><td>290 </td><td>Dominant (M) </td></tr><tr><td> </td><td> </td><td>GCTTGTTTGAAAGTGGCTTT </td><td> </td><td> </td><td>Co-dominant (b) </td></tr><tr><td>SEM115 </td><td>EM115 </td><td>CCGATTACATTTATGAGTAC </td><td>50 </td><td>350 </td><td>Co-dominant (M, b) </td></tr><tr><td> </td><td> </td><td>TTAAAGTCGATAGACCGACG </td><td> </td><td> </td><td> </td></tr><tr><td>SEM163 </td><td>EM163 </td><td>GAATTCGAGATCCAGAATCT </td><td>60 </td><td>290 </td><td>Dominant (M) </td></tr><tr><td> </td><td> </td><td>TTAAGTCTGAACCATTGCTA </td><td> </td><td> </td><td> </td></tr><tr><td>SEM214 </td><td>EM214 </td><td>GAATTCGGGGTGTTACACCTC </td><td>65 </td><td>580 </td><td>Dominant (M) </td></tr><tr><td> </td><td> </td><td>TTAATTACCCCCTCGTTCTC </td><td> </td><td> </td><td> </td></tr><tr><td>SEM802 </td><td>EM802 </td><td>GAATTCTGGAAGCACGTTGA </td><td>60 </td><td>190 </td><td>Co-dominant (M) </td></tr><tr><td> </td><td> </td><td>TTAAAATCATCATCTGAGAG </td><td> </td><td> </td><td> </td></tr><tr><td>s26d </td><td>188C5 </td><td>TCATGTTTAGCGCTCTGATT </td><td>60 </td><td>100 </td><td>Co-dominant (M, b) </td></tr><tr><td> </td><td> </td><td>GCACCGTAGTACCGTCAGTTTCC </td><td> </td><td> </td><td> </td></tr><tr><td>S11 </td><td>188C5 </td><td>TTGCAGGTTTTCAGGAACATT </td><td>60 </td><td>520 </td><td>Co-dominant (M, b) </td></tr><tr><td> </td><td> </td><td>GAAGTTCATTGTAACTCTCAAACG </td><td> </td><td> </td><td> </td></tr><tr><td>183R </td><td>183A2 </td><td>CATCGTATGTTGAAAAGAAGAATGAT </td><td>60 </td><td>580 </td><td>Co-dominant (M, b) </td></tr><tr><td> </td><td> </td><td>TCTTCAGTTCTTCCCTTTATAGCC </td><td> </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>PCR was performed using the indicated primer pairs with the following cycling parameter: 4 min at 94°C; 35 cycles of 30 s at 94°C, 30 s at the temperature indicated, 1 min at 72°C; 10 min at 72°C; and hold at 4°C.</p></div></div><div content-id=""><div class="fn"><p>PCR fragments were electrophoresed on 13% acrylamide gels with HEGS system.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>PCR fragments are derived from Miyakojima MG-20.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>M and b indicates the polymorphism of Miyakojima MG-20 and <em>L. burttii</em>, respectively, compared with Gifu B-129.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6982135" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6982137 id="6982137" class="section-title" >3.3. Construction of a 2 Mb physical BAC contig spanning LjNSP2</h3>
<p>At the first step, all SCAR markers were used to screen 3-D BAC DNA pools prepared from our <em>L. japonicus</em> Gifu BAC library. The contigs containing these SCAR markers were extended by the chromosome walking with the screening of 3-D BAC DNA pool using PCR primer combinations based on end sequences of BAC clones. In several cases, however, we could not obtain non-repetitive PCR fragments from BAC end sequences and screen BAC library. As an alternative strategy, AFLP fragments from BAC with <em>Eco</em>RI+1 selective and <em>Mse</em>I+1 selective primers were searched for non-repetitive sequences of BAC inner. Subsequently, the corresponding <em>Eco</em>RI+3 selective and <em>Mse</em>I+3 selective primers were used to screen 3-D BAC DNA pools with AFLP system. Finally, we constructed a ∼2 Mb physical BAC contig that spans the <em>LjNSP2</em> locus (<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2B</a>).</p>                            <h3 scrollto-destination=6982140 id="6982140" class="section-title" >3.4. The narrowing of LjNSP2 genomic region using L. burttii as an alternative crossing partner and identification of LjNSP2 gene</h3>
<p>As no more recombination was found from the population of this cross of <em>Ljnsp2-1</em> × Miyakojima MG-20, even with new markers developed from the BAC clones in the contig, we made another cross with <em>L. burttii</em>.<a reveal-id="b21" data-open="b21" class="link link-ref link-reveal xref-bibr">21</a> Among 8 472 F2 progenies from this cross, six recombination events were found between the flanking markers S26d and 183R derived from the BAC clone 188C5; this located the mutation within a 130 kb region (<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). Among the 5 ORFs (<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2C</a>) predicted from the sequence in this region (excluding transposable elements), only one was identified as having a mutation in the mutant. A 6.9 kb fragment including this ORF complemented the mutant for nodulation in hairy roots transformed by the <em>Agrobacterium rhizogenes</em> carrying the cloned region (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3</a>). No nodules were formed using empty vector, confirming that mutation of this gene caused the mutant phenotype.</p>                        <a id="6982143" scrollto-destination="6982143"></a>
<div id="fig3" data-id="fig3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/m_dsl017f3.jpeg?Expires=2147483647&Signature=HmwY-YHFHqwQgfN3dGDVljW4gH9Fhe~1Qs2WcCW3NR6Y-0sENfDC0-wJ8gqhF7~8pwf2kgi99wHQgDYpDu50wwmjIYhoK78HTPQGBUndSHx0ryw4OTnxzxPN8XkaXigbG5TmZpjKaidfh95lAsOS1W1qVgsqvJE8AwPM8rhZDp5GO8YIptrfHo2myya9s4PbQosk~fBYg8hyxCuNC0fcsTP-1F1CWYyVN1SF8e~vvadbXY8ERAHMKZyvx~W-RkkKYtmZbXovQrsKnZ1YnTcx50CNsFbtxJGEDYs8EaVNcsMeNuETSQ1tmfOoGtvWdiOjba4vBucg8vzdTpiDIHaZlQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Complementation of Ljnsp2-1 mutant. Roots of the Ljnsp2 mutant were transformed by Agrobacterium rhizogenes strain LBA1334 carrying appropriate plasmids carrying the 1 500 bp LjNSP2 ORF and 4 486 and 899 bp of upstream and downstream sequences and a transformation indicator 35S-GFP. On inoculation with Mesorhizobium loti, the GFP-marked transformants formed nodules complementing the mutant phenotype. Scale bar: 1 mm." path-from-xml="dsl017f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6982143/dsl017f3.jpeg" path-from-xml="dsl017f3.jpeg" target="_blank">View large</a><a section="6982143" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/dsl017f3.jpeg?Expires=2147483647&Signature=PYgfec1rudneWIpOPSS37eq~PhJhpWNrVtIo0HaMc5JPZ3yWGJ8DnCA1AjlHR~padD3Thv3eWCO5gYwL~ijIpgzCAI36qKuGpYfKCMb7LjgL8SsHrbuk57cIZbvLbXtL2menIIscMp-yoFBlHVd4kWLzoTx5F51cqDCTHHvAkZfN4C82yH6mn-0bO3q4DQkZptkpHmuJ--ZwFXcsJQl3XKqJyqnYpzrf~52U77KLRjmrwIB6Nhn62aobVYXDDdVbDDxOk5wN9rey8OWmva9LLFLd2XwI8jHkJZYOs2M5Tg72igD9daXKp1oNV1p18rW47gJ3vHi0EMvUpVWrZcRz0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6982143&ar=464686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl017f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Complementation of <em>Ljnsp2-1</em> mutant. Roots of the <em>Ljnsp2</em> mutant were transformed by <em>Agrobacterium rhizogenes</em> strain LBA1334 carrying appropriate plasmids carrying the 1 500 bp <em>LjNSP2</em> ORF and 4 486 and 899 bp of upstream and downstream sequences and a transformation indicator 35S-<em>GFP</em>. On inoculation with <em>Mesorhizobium loti</em>, the GFP-marked transformants formed nodules complementing the mutant phenotype. Scale bar: 1 mm.</p></div></div><div content-id="fig3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/m_dsl017f3.jpeg?Expires=2147483647&Signature=HmwY-YHFHqwQgfN3dGDVljW4gH9Fhe~1Qs2WcCW3NR6Y-0sENfDC0-wJ8gqhF7~8pwf2kgi99wHQgDYpDu50wwmjIYhoK78HTPQGBUndSHx0ryw4OTnxzxPN8XkaXigbG5TmZpjKaidfh95lAsOS1W1qVgsqvJE8AwPM8rhZDp5GO8YIptrfHo2myya9s4PbQosk~fBYg8hyxCuNC0fcsTP-1F1CWYyVN1SF8e~vvadbXY8ERAHMKZyvx~W-RkkKYtmZbXovQrsKnZ1YnTcx50CNsFbtxJGEDYs8EaVNcsMeNuETSQ1tmfOoGtvWdiOjba4vBucg8vzdTpiDIHaZlQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Complementation of Ljnsp2-1 mutant. Roots of the Ljnsp2 mutant were transformed by Agrobacterium rhizogenes strain LBA1334 carrying appropriate plasmids carrying the 1 500 bp LjNSP2 ORF and 4 486 and 899 bp of upstream and downstream sequences and a transformation indicator 35S-GFP. On inoculation with Mesorhizobium loti, the GFP-marked transformants formed nodules complementing the mutant phenotype. Scale bar: 1 mm." path-from-xml="dsl017f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6982143/dsl017f3.jpeg" path-from-xml="dsl017f3.jpeg" target="_blank">View large</a><a section="6982143" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/dsl017f3.jpeg?Expires=2147483647&Signature=PYgfec1rudneWIpOPSS37eq~PhJhpWNrVtIo0HaMc5JPZ3yWGJ8DnCA1AjlHR~padD3Thv3eWCO5gYwL~ijIpgzCAI36qKuGpYfKCMb7LjgL8SsHrbuk57cIZbvLbXtL2menIIscMp-yoFBlHVd4kWLzoTx5F51cqDCTHHvAkZfN4C82yH6mn-0bO3q4DQkZptkpHmuJ--ZwFXcsJQl3XKqJyqnYpzrf~52U77KLRjmrwIB6Nhn62aobVYXDDdVbDDxOk5wN9rey8OWmva9LLFLd2XwI8jHkJZYOs2M5Tg72igD9daXKp1oNV1p18rW47gJ3vHi0EMvUpVWrZcRz0g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6982143&ar=464686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl017f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Complementation of <em>Ljnsp2-1</em> mutant. Roots of the <em>Ljnsp2</em> mutant were transformed by <em>Agrobacterium rhizogenes</em> strain LBA1334 carrying appropriate plasmids carrying the 1 500 bp <em>LjNSP2</em> ORF and 4 486 and 899 bp of upstream and downstream sequences and a transformation indicator 35S-<em>GFP</em>. On inoculation with <em>Mesorhizobium loti</em>, the GFP-marked transformants formed nodules complementing the mutant phenotype. Scale bar: 1 mm.</p></div></div><p>The ORF in this region corresponding to LjNSP2 encodes a protein belonging to the plant GRAS family of putative transcription factors<a reveal-id="b32  b34" data-open="b32  b34" class="link link-ref link-reveal xref-bibr">32–34</a> and analysis of the major plant GRAS family protein sequences indicated that the closest in sequence to LjNSP2 were the <em>M. truncatula</em> and <em>Pisum sativum</em> NSP2 proteins.<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> AtSCL26 of <em>Arabidopsis</em> was about twice distant from them, and together these proteins made an apparent subfamily of GRAS proteins (<a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4</a>). However, genome sequences around the <em>LjNSP2</em> and <em>M. truncatula NSP2</em> revealed no clear co-linearity based on the available data.</p>                        <a id="6982147" scrollto-destination="6982147"></a>
<div id="fig4" data-id="fig4" class="fig fig-section"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/m_dsl017f4.jpeg?Expires=2147483647&Signature=ObNa6qeqaSZEhzuHAjB1k~a0p6SoeXgqIu7HACzl976uhISxAhD7qqXt-uUq7jhplZQMH6OQoKNuC3Rimbbat6QIFxx3xGc1M7EtH8fk01R6j~n9QZVhWfml0bxUT6ELnekZ7ySf1QXiMKbrNRMY0ziroLbEljabNy39UMHeCIECGf~N8FpiGP2kvWD5FiMrx2HE4nro7It0kpgrDZRdL5XPw0tEIo8GUGjVIX9fX-w~9dMlDdMrchIGNvsPI2FdODkVKdLVxmrY9p4JAUq-cNUnaD1o7E1QD49m-qIfIRhtEBIe7wzsKIJb3k1HABCUBRV8ZiWuSfX4fmGw8tsXIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree of plant GRAS family proteins. Members of the family of plant GRAS proteins aligned using Clustal W are shown as a neighbor-joining dendrogram with 1000 bootstrap replicates. At, Arabidopsis thaliana; Mt, Medicago truncatula; Ph, Petunia hybrida; Ps, Pisum sativum; Le, Lycopersicum esculentum; NSP, nodulation signaling pathway; SCR, SCARECROW; SCL, SCARECROW-like; SHR, short-root; HAM, hairy meristem; Ls, lateral suppressor; RGA, repressor of ga1-3; PAT, phytochrome A signal transduction." path-from-xml="dsl017f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6982147/dsl017f4.jpeg" path-from-xml="dsl017f4.jpeg" target="_blank">View large</a><a section="6982147" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/dsl017f4.jpeg?Expires=2147483647&Signature=Lwr-5AKuii-3H2FpTqnvfNLfFslshy4yqlqv-d~RbtrFO2MKFHNV~QlqsEh9UQ2iI4Ovxc5qkTcr6e1CY3oQGpVyN5dmDOJIsstAB4-zfIxJhmMPvVHaGoJROnlOoAaIWbwp-Gc1FCA0EUJreKYcVn0wtJQXfWV~Vor7JdMy-LYBG1DoIqY08q33IeaRLP7JmkmVUMyURy1Uq3TUN9I~vmi0TQFA2UxDcA32M1N6ehq8nGDRrGpb7tFMtVaPD0zXdj1Ig2aHX2uhW2LHMaBAhLSTBFra0TZLgcAkmQywQr5pdvP~q6j9o8ToM1aBSabf-jQaMM4YVSsMV6AUNR5DOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6982147&ar=464686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl017f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of plant GRAS family proteins. Members of the family of plant GRAS proteins aligned using Clustal W are shown as a neighbor-joining dendrogram with 1000 bootstrap replicates. At, <em>Arabidopsis thaliana</em>; Mt, <em>Medicago truncatula</em>; Ph, <em>Petunia hybrida</em>; Ps, <em>Pisum sativum</em>; Le, <em>Lycopersicum esculentum</em>; NSP, nodulation signaling pathway; SCR, SCARECROW; SCL, SCARECROW-like; SHR, short-root; HAM, hairy meristem; Ls, lateral suppressor; RGA, repressor of ga1-3; PAT, phytochrome A signal transduction.</p></div></div><div content-id="fig4" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/m_dsl017f4.jpeg?Expires=2147483647&Signature=ObNa6qeqaSZEhzuHAjB1k~a0p6SoeXgqIu7HACzl976uhISxAhD7qqXt-uUq7jhplZQMH6OQoKNuC3Rimbbat6QIFxx3xGc1M7EtH8fk01R6j~n9QZVhWfml0bxUT6ELnekZ7ySf1QXiMKbrNRMY0ziroLbEljabNy39UMHeCIECGf~N8FpiGP2kvWD5FiMrx2HE4nro7It0kpgrDZRdL5XPw0tEIo8GUGjVIX9fX-w~9dMlDdMrchIGNvsPI2FdODkVKdLVxmrY9p4JAUq-cNUnaD1o7E1QD49m-qIfIRhtEBIe7wzsKIJb3k1HABCUBRV8ZiWuSfX4fmGw8tsXIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree of plant GRAS family proteins. Members of the family of plant GRAS proteins aligned using Clustal W are shown as a neighbor-joining dendrogram with 1000 bootstrap replicates. At, Arabidopsis thaliana; Mt, Medicago truncatula; Ph, Petunia hybrida; Ps, Pisum sativum; Le, Lycopersicum esculentum; NSP, nodulation signaling pathway; SCR, SCARECROW; SCL, SCARECROW-like; SHR, short-root; HAM, hairy meristem; Ls, lateral suppressor; RGA, repressor of ga1-3; PAT, phytochrome A signal transduction." path-from-xml="dsl017f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6982147/dsl017f4.jpeg" path-from-xml="dsl017f4.jpeg" target="_blank">View large</a><a section="6982147" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/dsl017f4.jpeg?Expires=2147483647&Signature=Lwr-5AKuii-3H2FpTqnvfNLfFslshy4yqlqv-d~RbtrFO2MKFHNV~QlqsEh9UQ2iI4Ovxc5qkTcr6e1CY3oQGpVyN5dmDOJIsstAB4-zfIxJhmMPvVHaGoJROnlOoAaIWbwp-Gc1FCA0EUJreKYcVn0wtJQXfWV~Vor7JdMy-LYBG1DoIqY08q33IeaRLP7JmkmVUMyURy1Uq3TUN9I~vmi0TQFA2UxDcA32M1N6ehq8nGDRrGpb7tFMtVaPD0zXdj1Ig2aHX2uhW2LHMaBAhLSTBFra0TZLgcAkmQywQr5pdvP~q6j9o8ToM1aBSabf-jQaMM4YVSsMV6AUNR5DOQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6982147&ar=464686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl017f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of plant GRAS family proteins. Members of the family of plant GRAS proteins aligned using Clustal W are shown as a neighbor-joining dendrogram with 1000 bootstrap replicates. At, <em>Arabidopsis thaliana</em>; Mt, <em>Medicago truncatula</em>; Ph, <em>Petunia hybrida</em>; Ps, <em>Pisum sativum</em>; Le, <em>Lycopersicum esculentum</em>; NSP, nodulation signaling pathway; SCR, SCARECROW; SCL, SCARECROW-like; SHR, short-root; HAM, hairy meristem; Ls, lateral suppressor; RGA, repressor of ga1-3; PAT, phytochrome A signal transduction.</p></div></div><p>The <em>LjNSP2</em> gene consists of a 1500 bp exon with no introns, encoding a predicted 499 amino acids protein of 55 kDa, containing the following GRAS family-specific domains: homopolymeric stretches (HPS) of polyE and polyT; first leucine heptad repeat (LHRI); a VHIID DNA-binding sequence, second leucine heptad repeat (LHRII); and a <em>Src</em>-homology 2 (SH2)-like domain<a reveal-id="b35" data-open="b35" class="link link-ref link-reveal xref-bibr">35</a> (<a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5A</a>). About 15% of the N-terminal region is divergent among most GRAS family members but that of LjNSP2 showed strong homology with the <em>M. truncatula</em> and <em>Pisum sativam</em> NSP2 proteins<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> suggesting functional equivalence and that they are probably orthologs. Furthermore, LjNSP2 has a well-conserved SH2-like domain among GRAS proteins (<a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5B</a>). The <em>Ljnsp2-1</em> mutation causes a substitution of a conserved valine (V) to glutamate (E) in this SH2-like domain (<a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5C</a>).</p>                        <a id="6982151" scrollto-destination="6982151"></a>
<div id="fig5" data-id="fig5" class="fig fig-section"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/m_dsl017f5.jpeg?Expires=2147483647&Signature=UUNZ7YfLMsj8AyEId2835jsYrR4imjF2iopzxyoV8sLwcRYlqxA66ctFD4yvryK2tm4tLXepfFDjfV7Fd9h2V90VJTTfbuD-qsNvaNDPn9fVuVMJgg5W-9pOXFha6PtT-SQaf7QhqHPnxj5In7E2k~ghxj-IGdSzUNwNlVVNVmoYpQ5JY59M8CU2azfPMFBI6rNLN-TDSs57URIc~4OHadoq3vQDG-hJ57ypEaakEmjSSpl9IsE4Q0mExdyaB9UjjxIx~iBiiPNmNyi-gWJ3Upt1GMT5KjJUC5v3VZYgzqFBF0qXguswvAQYDcXZjrZW-K0WGCMdaK2OlilCgPL35w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Domain structure of the LjNSP2 protein. (A) Amino acid sequence of 499 residues and predicted functional domains of LjNSP2. HPS; homopolymeric stretches characteristic for GRAS protein near to the N-terminal, LHR (leucine heptad repeat) 1 and 2; putative leucine zipper, VHIID; putative DNA-binding sites, SH2-like region, GY; GY, Y of which is phosphorylated in STATs, is conserved about 100 residues downstream from SH2-like domain. (B) Comparison of GRAS and STAT family proteins with LjNSP2 in their SH2(-like) domains. STAT2: human STAT2; P52630, Ce-STAT: Caenorhabditis elegans STAT; Z70754, D-STAT: Drosophila stat; Q24151, Dd-STAT: Dictyostelium discoideum STAT; Y13097, GAI: Arabidopsis GAI (gibberellin insensitive); At1g14920, SCR: Arabidopsis SCR (SCARECROW); At3g54220. GAI and SCR are the representatives of plant GRAS family proteins. Conserved amino acids are indicated by red boxes. A missense mutation in Ljnsp2-1 is indicated by arrowhead." path-from-xml="dsl017f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6982151/dsl017f5.jpeg" path-from-xml="dsl017f5.jpeg" target="_blank">View large</a><a section="6982151" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/dsl017f5.jpeg?Expires=2147483647&Signature=1zCalOdqPsI~6EL37BOASrUufHa5mq5KFvuSX0hf0pt5SmpREXpaGvEBPlh46UuvRcGYl4BAAcFQCJ967MsKDCJKTPB0UmiJKB46igyq031ZEItg1lEInCI07QXQ-CYzPWVd4M3qQ6uKzytsb9yNE3VkVB2q-3fnIgf6o6uXlGppxznpNGA2OpXdO9w352AU8dMB7Chkwr0eUb5rft6rpktRYDlwAPIdBTTa-~l-6BqIoy7G99-5GqMLQ9oa3QJCLFWusTFZTeipKlGH1ox2hrHsmx8v4JAYDrBLPZWrHwchKbv8nxsnDbU4uF1UU7VFt7ACPJgReljOjrgrRMr6GA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6982151&ar=464686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl017f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Domain structure of the LjNSP2 protein. (A) Amino acid sequence of 499 residues and predicted functional domains of LjNSP2. HPS; homopolymeric stretches characteristic for GRAS protein near to the N-terminal, LHR (leucine heptad repeat) 1 and 2; putative leucine zipper, VHIID; putative DNA-binding sites, SH2-like region, GY; GY, Y of which is phosphorylated in STATs, is conserved about 100 residues downstream from SH2-like domain. (B) Comparison of GRAS and STAT family proteins with LjNSP2 in their SH2(-like) domains. STAT2: human STAT2; P52630, <em>Ce</em>-STAT: <em>Caenorhabditis elegans</em> STAT; Z70754, <em>D</em>-STAT: <em>Drosophila</em> stat; Q24151, <em>Dd-</em>STAT: <em>Dictyostelium discoideum</em> STAT; Y13097, GAI: <em>Arabidopsis</em> GAI (gibberellin insensitive); <em>At</em>1g14920, SCR: <em>Arabidopsis</em> SCR (SCARECROW); <em>At</em>3g54220. GAI and SCR are the representatives of plant GRAS family proteins. Conserved amino acids are indicated by red boxes. A missense mutation in <em>Ljnsp2-1</em> is indicated by arrowhead.</p></div></div><div content-id="fig5" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/m_dsl017f5.jpeg?Expires=2147483647&Signature=UUNZ7YfLMsj8AyEId2835jsYrR4imjF2iopzxyoV8sLwcRYlqxA66ctFD4yvryK2tm4tLXepfFDjfV7Fd9h2V90VJTTfbuD-qsNvaNDPn9fVuVMJgg5W-9pOXFha6PtT-SQaf7QhqHPnxj5In7E2k~ghxj-IGdSzUNwNlVVNVmoYpQ5JY59M8CU2azfPMFBI6rNLN-TDSs57URIc~4OHadoq3vQDG-hJ57ypEaakEmjSSpl9IsE4Q0mExdyaB9UjjxIx~iBiiPNmNyi-gWJ3Upt1GMT5KjJUC5v3VZYgzqFBF0qXguswvAQYDcXZjrZW-K0WGCMdaK2OlilCgPL35w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Domain structure of the LjNSP2 protein. (A) Amino acid sequence of 499 residues and predicted functional domains of LjNSP2. HPS; homopolymeric stretches characteristic for GRAS protein near to the N-terminal, LHR (leucine heptad repeat) 1 and 2; putative leucine zipper, VHIID; putative DNA-binding sites, SH2-like region, GY; GY, Y of which is phosphorylated in STATs, is conserved about 100 residues downstream from SH2-like domain. (B) Comparison of GRAS and STAT family proteins with LjNSP2 in their SH2(-like) domains. STAT2: human STAT2; P52630, Ce-STAT: Caenorhabditis elegans STAT; Z70754, D-STAT: Drosophila stat; Q24151, Dd-STAT: Dictyostelium discoideum STAT; Y13097, GAI: Arabidopsis GAI (gibberellin insensitive); At1g14920, SCR: Arabidopsis SCR (SCARECROW); At3g54220. GAI and SCR are the representatives of plant GRAS family proteins. Conserved amino acids are indicated by red boxes. A missense mutation in Ljnsp2-1 is indicated by arrowhead." path-from-xml="dsl017f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6982151/dsl017f5.jpeg" path-from-xml="dsl017f5.jpeg" target="_blank">View large</a><a section="6982151" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/dsl017f5.jpeg?Expires=2147483647&Signature=1zCalOdqPsI~6EL37BOASrUufHa5mq5KFvuSX0hf0pt5SmpREXpaGvEBPlh46UuvRcGYl4BAAcFQCJ967MsKDCJKTPB0UmiJKB46igyq031ZEItg1lEInCI07QXQ-CYzPWVd4M3qQ6uKzytsb9yNE3VkVB2q-3fnIgf6o6uXlGppxznpNGA2OpXdO9w352AU8dMB7Chkwr0eUb5rft6rpktRYDlwAPIdBTTa-~l-6BqIoy7G99-5GqMLQ9oa3QJCLFWusTFZTeipKlGH1ox2hrHsmx8v4JAYDrBLPZWrHwchKbv8nxsnDbU4uF1UU7VFt7ACPJgReljOjrgrRMr6GA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6982151&ar=464686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl017f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Domain structure of the LjNSP2 protein. (A) Amino acid sequence of 499 residues and predicted functional domains of LjNSP2. HPS; homopolymeric stretches characteristic for GRAS protein near to the N-terminal, LHR (leucine heptad repeat) 1 and 2; putative leucine zipper, VHIID; putative DNA-binding sites, SH2-like region, GY; GY, Y of which is phosphorylated in STATs, is conserved about 100 residues downstream from SH2-like domain. (B) Comparison of GRAS and STAT family proteins with LjNSP2 in their SH2(-like) domains. STAT2: human STAT2; P52630, <em>Ce</em>-STAT: <em>Caenorhabditis elegans</em> STAT; Z70754, <em>D</em>-STAT: <em>Drosophila</em> stat; Q24151, <em>Dd-</em>STAT: <em>Dictyostelium discoideum</em> STAT; Y13097, GAI: <em>Arabidopsis</em> GAI (gibberellin insensitive); <em>At</em>1g14920, SCR: <em>Arabidopsis</em> SCR (SCARECROW); <em>At</em>3g54220. GAI and SCR are the representatives of plant GRAS family proteins. Conserved amino acids are indicated by red boxes. A missense mutation in <em>Ljnsp2-1</em> is indicated by arrowhead.</p></div></div><p>Complementation tests with the various <em>L. japonicus</em> nodulation mutants carrying mutations mapped to linkage group 1 revealed that <em>Ljsym35</em> carried a mutation allelic to <em>Ljnsp2</em>. DNA hybridizations detected no signal in <em>Ljsym35</em> (<a reveal-id="fig6" data-open="fig6" class="link link-reveal link-table xref-fig">Fig. 6A</a>) and PCR analyses indicated that it has a deletion of &gt;100 kb around the <em>LjNSP2</em> gene; this allele was renamed <em>Ljnsp2-2</em>.</p>                        <a id="6982154" scrollto-destination="6982154"></a>
<div id="fig6" data-id="fig6" class="fig fig-section"><div class="label fig-label"><strong>Figure 6</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/m_dsl017f6.jpeg?Expires=2147483647&Signature=bTIVrCRfHaz6t0-okq7yC9tXE6pG-X7cUux1Wpnf-yJNRPayTpts~U-ECricfCV4JUpZgg2x-UHKK6p5DhisX9dk4W~VWWCh7lHxDWj2PkqUqugb5LKWdIvsS6muEcwuC2QLAHjtwhi2HXnMlFc37oXnpYH9H975Ie-BM-~dpeKIHtabS~Tyt-1r1rbEmxlq4lYHQEUVzKz3PDyq7IOfr5vls1LDvvp8ZV4RkuFF95drjM3Yk4-DAQkqwqV21j8y69oX7HW9PM3JUbcJYVPkHsM9ewd0Fb8GJ06ucJwzhuX3WGLvtTxO5XrH44-qFFZuM-qfsjUo-Gfm8Is3v82IwA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNA, RNA hybridization and RT–PCR analyses of LjNSP2. (A) Genomic hybridization with the full-length LjNSP2 ORF probe to the wild-type (WT; Gifu B-129) and a null deletion mutant Ljnsp2-2. Genomic DNA was digested with EcoRI. The LjNSP2 homolog seems to be unique in WT, but cannot be detected in Ljnsp2-2. (B and C) Northern analysis of LjNSP2 expression in various organs (B), and infected roots at 4 dpi of M. loti TONO and nodules at 8–32 dpi of M. loti TONO, respectively (C). DPI: days post inoculation." path-from-xml="dsl017f6.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6982154/dsl017f6.jpeg" path-from-xml="dsl017f6.jpeg" target="_blank">View large</a><a section="6982154" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/dsl017f6.jpeg?Expires=2147483647&Signature=jbfbK08JoPKEOaW3j6REsRkPw8-OX786J~5oWN4XWcAt0SOAv08PJ4YCNHY8kTLxXIM~iSc2k1jXXlg5ZOLyQG~6n-lklsNRrqiOwWOxcehyddMlMpuPKg7hZiCCN-UINphQdzSlkYdpXruNVDI1pstXTHSCgF5NXF-ZP-~UkR13K~CeZGnOIso4Z8tns54Nmy5GIxy2M5DqUFccBjg6R-GSUbiR0wFcp0I5SLVbE8RjhngxXVlwto6LrSO1LxMCbvXPlAqSfr4308-yMsbgIIQCKxvex4plo4bMFqWpuHci4u1UrvQfUWlcdTThF9Lflr939QN54BRAzs7p-iR1cQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6982154&ar=464686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl017f6.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNA, RNA hybridization and RT–PCR analyses of <em>LjNSP2</em>. (A) Genomic hybridization with the full-length <em>LjNSP2</em> ORF probe to the wild-type (WT; Gifu B-129) and a null deletion mutant <em>Ljnsp2-2</em>. Genomic DNA was digested with <em>Eco</em>RI. The <em>LjNSP2</em> homolog seems to be unique in WT, but cannot be detected in <em>Ljnsp2-2</em>. (B and C) Northern analysis of <em>LjNSP2</em> expression in various organs (B), and infected roots at 4 dpi of <em>M. loti</em> TONO and nodules at 8–32 dpi of <em>M. loti</em> TONO, respectively (C). DPI: days post inoculation.</p></div></div><div content-id="fig6" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 6</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/m_dsl017f6.jpeg?Expires=2147483647&Signature=bTIVrCRfHaz6t0-okq7yC9tXE6pG-X7cUux1Wpnf-yJNRPayTpts~U-ECricfCV4JUpZgg2x-UHKK6p5DhisX9dk4W~VWWCh7lHxDWj2PkqUqugb5LKWdIvsS6muEcwuC2QLAHjtwhi2HXnMlFc37oXnpYH9H975Ie-BM-~dpeKIHtabS~Tyt-1r1rbEmxlq4lYHQEUVzKz3PDyq7IOfr5vls1LDvvp8ZV4RkuFF95drjM3Yk4-DAQkqwqV21j8y69oX7HW9PM3JUbcJYVPkHsM9ewd0Fb8GJ06ucJwzhuX3WGLvtTxO5XrH44-qFFZuM-qfsjUo-Gfm8Is3v82IwA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNA, RNA hybridization and RT–PCR analyses of LjNSP2. (A) Genomic hybridization with the full-length LjNSP2 ORF probe to the wild-type (WT; Gifu B-129) and a null deletion mutant Ljnsp2-2. Genomic DNA was digested with EcoRI. The LjNSP2 homolog seems to be unique in WT, but cannot be detected in Ljnsp2-2. (B and C) Northern analysis of LjNSP2 expression in various organs (B), and infected roots at 4 dpi of M. loti TONO and nodules at 8–32 dpi of M. loti TONO, respectively (C). DPI: days post inoculation." path-from-xml="dsl017f6.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6982154/dsl017f6.jpeg" path-from-xml="dsl017f6.jpeg" target="_blank">View large</a><a section="6982154" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/dsl017f6.jpeg?Expires=2147483647&Signature=jbfbK08JoPKEOaW3j6REsRkPw8-OX786J~5oWN4XWcAt0SOAv08PJ4YCNHY8kTLxXIM~iSc2k1jXXlg5ZOLyQG~6n-lklsNRrqiOwWOxcehyddMlMpuPKg7hZiCCN-UINphQdzSlkYdpXruNVDI1pstXTHSCgF5NXF-ZP-~UkR13K~CeZGnOIso4Z8tns54Nmy5GIxy2M5DqUFccBjg6R-GSUbiR0wFcp0I5SLVbE8RjhngxXVlwto6LrSO1LxMCbvXPlAqSfr4308-yMsbgIIQCKxvex4plo4bMFqWpuHci4u1UrvQfUWlcdTThF9Lflr939QN54BRAzs7p-iR1cQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6982154&ar=464686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl017f6.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNA, RNA hybridization and RT–PCR analyses of <em>LjNSP2</em>. (A) Genomic hybridization with the full-length <em>LjNSP2</em> ORF probe to the wild-type (WT; Gifu B-129) and a null deletion mutant <em>Ljnsp2-2</em>. Genomic DNA was digested with <em>Eco</em>RI. The <em>LjNSP2</em> homolog seems to be unique in WT, but cannot be detected in <em>Ljnsp2-2</em>. (B and C) Northern analysis of <em>LjNSP2</em> expression in various organs (B), and infected roots at 4 dpi of <em>M. loti</em> TONO and nodules at 8–32 dpi of <em>M. loti</em> TONO, respectively (C). DPI: days post inoculation.</p></div></div>                            <h3 scrollto-destination=6982157 id="6982157" class="section-title" >3.5. Expression of LjNSP2 during nodulation and in different organs</h3>
<p>RNA hybridization showed that <em>LjNSP2</em> expression was detectable in roots but not in shoots and flowers (<a reveal-id="fig6" data-open="fig6" class="link link-reveal link-table xref-fig">Fig. 6B</a>), in contrast to the ubiquitously expressed <em>Medicago NSP2</em>.<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a>,<em>LjNSP2</em> was also expressed in infected roots at 4 dpi and early nodules but strongly suppressed in matured nodules (<a reveal-id="fig6" data-open="fig6" class="link link-reveal link-table xref-fig">Fig. 6C</a>). These expression patterns of <em>LjNSP2</em> are similar to that of the putative Nod factor receptor kinase genes, <em>NFR1</em><a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> and <em>NFR5</em>,<a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> and the signal transduction components <em>CASTOR</em>, although the decrease in expression of the latter was slight.<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a> This contrasts with the expression patterns of the genes encoding the nodulation signaling pathway components <em>M. truncatula NSP1</em>,<a reveal-id="b17" data-open="b17" class="link link-ref link-reveal xref-bibr">17</a>,<em>M. truncatula NSP2</em><a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> and <em>L. japonicus NIN</em><a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a>(<a reveal-id="fig6" data-open="fig6" class="link link-reveal link-table xref-fig">Fig. 6A</a>) whose expression increased following inoculation.</p>                            <h3 scrollto-destination=6982162 id="6982162" class="section-title" >3.6. Expression of NIN and LjENOD40-1 genes in Ljnsp2-2 mutant</h3>
<p>In the <em>Ljnsp2-2</em> null mutant, RT–PCR revealed that even before inoculation with <em>M. loti</em> the expression levels of early nodulation genes, <em>NIN</em> and <em>LjENOD40-1</em>, were both &lt;40% of the wild-type (<a reveal-id="fig7" data-open="fig7" class="link link-reveal link-table xref-fig">Fig. 7A and B</a>). Upon inoculation of the wild-type with <em>M. loti</em>, <em>NIN</em> and <em>LjENOD40-1</em> increased by up to 65- or 3.5-fold, respectively. In contrast, the levels of <em>NIN</em> and <em>LjENOD40-1</em> transcripts in the <em>Ljnsp2-2</em> mutant remained low after inoculation reaching only about 5–20% of wild-type levels (<a reveal-id="fig7" data-open="fig7" class="link link-reveal link-table xref-fig">Fig. 7A and B</a>). This demonstrates that LjNSP2 function is required, directly or indirectly, for either the expression and/or the induction of these early nodulins.</p>                        <a id="6982165" scrollto-destination="6982165"></a>
<div id="fig7" data-id="fig7" class="fig fig-section"><div class="label fig-label"><strong>Figure 7</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/m_dsl017f7.jpeg?Expires=2147483647&Signature=OkXUr5xczYUZRpEe~o3x9rNCP5N3XD0m5z20anqBpjc-YGIErqwGMLGmSblYF47WVBjhb2EnLPCeU0e0xCXvKwQQeqF-deCssqqnY~hQRnBQ1~fJZRrwH6SbABZnx-V4qoh2TObqKQBtUhZPrUrHTMnEUxUotzmz~t1w~MYc8PJ1Egg5IdBXq4HK274xk804uzM9NZP~VnezLk8sMKzmls7R2F8p72L5KjsnwKc~RW9zryOl5ai2WXkHXsBztmeP4MuFwwFP0w2z058lrsMBqOWqlnYqcxuO4mHLoBC4eJwUHHlfNzafBQFRWaQGpA3uYjTJfupo8oIYWD0ayrinrA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="RT–PCR analyses of expression of LjNSP2 and the early nodulins. (A and B) RT–PCR analysis of the change of expression of NIN (A) and LjENOD40-1 (B) after M. loti inoculation, in wild-type and Ljnsp2. Suppression of LjNSP2 expression after inoculation was confirmed, and almost complete loss of induction in NIN and LjENOD40-1 were apparent. Ubiquitin is as a loading control. Relative expression levels were normalized against the amount of ubiquitin (set as 105). Standard deviations of three independent experiments are indicated by error bars. DPI: days post inoculation." path-from-xml="dsl017f7.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6982165/dsl017f7.jpeg" path-from-xml="dsl017f7.jpeg" target="_blank">View large</a><a section="6982165" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/dsl017f7.jpeg?Expires=2147483647&Signature=bZFaoXx09-awfjUuxlyKdDrS8BCFjNR0suO-AafQFRWwYiIbMhKFMw8~MOTNfy5ZXvEbn-7Q3cJkLGIA5wIJmr3V-4-RpybaOrKcRH5maxsFiRi~8QSnrZDNVPGGpL0KC2mbOgafOj~YBytLDUDpM8LWu~QZGiinu99D0OBIzGNcv1fpRbbQ93f4hrxPdCAQgBwZNSLq4Syooha5kQbemQAb7lAF~i2F07EgYtR-txn2-TWhtvPMin9XI0DWDTmkZGTuGZK2vQoccjF1E-e~7q7dLuwTTX58H~p6bkc4EH0ksPFttA2vZwSHWmm800bZm9W7UvqJQfeLCP-2Qbj6xg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6982165&ar=464686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl017f7.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>RT–PCR analyses of expression of <em>LjNSP2</em> and the early nodulins. (A and B) RT–PCR analysis of the change of expression of <em>NIN</em> (A) and <em>LjENOD40-1</em> (B) after <em>M</em>. <em>loti</em> inoculation, in wild-type and <em>Ljnsp2</em>. Suppression of <em>LjNSP2</em> expression after inoculation was confirmed, and almost complete loss of induction in <em>NIN</em> and <em>LjENOD40-1</em> were apparent. <em>Ubiquitin</em> is as a loading control. Relative expression levels were normalized against the amount of ubiquitin (set as 10<sup>5</sup>). Standard deviations of three independent experiments are indicated by error bars. DPI: days post inoculation.</p></div></div><div content-id="fig7" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 7</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/m_dsl017f7.jpeg?Expires=2147483647&Signature=OkXUr5xczYUZRpEe~o3x9rNCP5N3XD0m5z20anqBpjc-YGIErqwGMLGmSblYF47WVBjhb2EnLPCeU0e0xCXvKwQQeqF-deCssqqnY~hQRnBQ1~fJZRrwH6SbABZnx-V4qoh2TObqKQBtUhZPrUrHTMnEUxUotzmz~t1w~MYc8PJ1Egg5IdBXq4HK274xk804uzM9NZP~VnezLk8sMKzmls7R2F8p72L5KjsnwKc~RW9zryOl5ai2WXkHXsBztmeP4MuFwwFP0w2z058lrsMBqOWqlnYqcxuO4mHLoBC4eJwUHHlfNzafBQFRWaQGpA3uYjTJfupo8oIYWD0ayrinrA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="RT–PCR analyses of expression of LjNSP2 and the early nodulins. (A and B) RT–PCR analysis of the change of expression of NIN (A) and LjENOD40-1 (B) after M. loti inoculation, in wild-type and Ljnsp2. Suppression of LjNSP2 expression after inoculation was confirmed, and almost complete loss of induction in NIN and LjENOD40-1 were apparent. Ubiquitin is as a loading control. Relative expression levels were normalized against the amount of ubiquitin (set as 105). Standard deviations of three independent experiments are indicated by error bars. DPI: days post inoculation." path-from-xml="dsl017f7.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6982165/dsl017f7.jpeg" path-from-xml="dsl017f7.jpeg" target="_blank">View large</a><a section="6982165" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/dsl017f7.jpeg?Expires=2147483647&Signature=bZFaoXx09-awfjUuxlyKdDrS8BCFjNR0suO-AafQFRWwYiIbMhKFMw8~MOTNfy5ZXvEbn-7Q3cJkLGIA5wIJmr3V-4-RpybaOrKcRH5maxsFiRi~8QSnrZDNVPGGpL0KC2mbOgafOj~YBytLDUDpM8LWu~QZGiinu99D0OBIzGNcv1fpRbbQ93f4hrxPdCAQgBwZNSLq4Syooha5kQbemQAb7lAF~i2F07EgYtR-txn2-TWhtvPMin9XI0DWDTmkZGTuGZK2vQoccjF1E-e~7q7dLuwTTX58H~p6bkc4EH0ksPFttA2vZwSHWmm800bZm9W7UvqJQfeLCP-2Qbj6xg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6982165&ar=464686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl017f7.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>RT–PCR analyses of expression of <em>LjNSP2</em> and the early nodulins. (A and B) RT–PCR analysis of the change of expression of <em>NIN</em> (A) and <em>LjENOD40-1</em> (B) after <em>M</em>. <em>loti</em> inoculation, in wild-type and <em>Ljnsp2</em>. Suppression of <em>LjNSP2</em> expression after inoculation was confirmed, and almost complete loss of induction in <em>NIN</em> and <em>LjENOD40-1</em> were apparent. <em>Ubiquitin</em> is as a loading control. Relative expression levels were normalized against the amount of ubiquitin (set as 10<sup>5</sup>). Standard deviations of three independent experiments are indicated by error bars. DPI: days post inoculation.</p></div></div>                            <h3 scrollto-destination=6982167 id="6982167" class="section-title" >3.7. Nuclear localization of LjNSP2 in onion cells</h3>
<p>Although most GRAS family proteins have a putative nuclear localization sequence (NLS)<a reveal-id="b34" data-open="b34" class="link link-ref link-reveal xref-bibr">34</a>, PSORT II analysis did not identify an NLS-like sequence in <em>LjNSP2</em>. However, LjNSP2-GFP fusion delivered into onion (<em>Allium cepa</em>) epidermal cells by particle bombardment revealed it to be exclusively localized in nuclei, although not in nucleoli (<a reveal-id="fig8" data-open="fig8" class="link link-reveal link-table xref-fig">Fig. 8A</a>). The mutation in the SH-2-like domain reduced nuclear localization of Ljnsp2-1-GFP and fluorescence in the cytoplasm became noticeable in at least 30 GFP-expressed cells (<a reveal-id="fig8" data-open="fig8" class="link link-reveal link-table xref-fig">Fig. 8B</a>), but not as strong as seen with GFP alone (<a reveal-id="fig8" data-open="fig8" class="link link-reveal link-table xref-fig">Fig. 8C</a>). These observations are consistent with LjNSP2 acting as a transcription factor, and the SH2-like domain facilitating its nuclear localization.</p>                        <a id="6982170" scrollto-destination="6982170"></a>
<div id="fig8" data-id="fig8" class="fig fig-section"><div class="label fig-label"><strong>Figure 8</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/m_dsl017f8.jpeg?Expires=2147483647&Signature=P9m4mRSVugBioQtNId9p-gEziZslc-0KYSBRsbZ8o6SAIYSHySSNoyrJBgBbHtImpmvgsHQ33d0Wiubmj3SiCbwz8Dmu0Ek72tYcAw2RWJQHJGaZGxZaoktbJ~iqUJJpjUoUVfSMX1yJY3WpWDwnGeuNp7Zn6jphnvf0MkUYYIgnfkiMVP0jTkmXuafc~8q-jHMHwLhp60G558r2QJxzLHffLHr054cPgRpSCQdzz66wweriqwpL~arhZRjZYJn7c1LZFjXI8rfbKZAiG8fhGMBfPw3ayR3YMPMc2~~1tRh-vkR1aB0bMKBp38iL~pzGvgkUo-7JWXwuPpiRUEkLoQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Nuclear localization of the LjNSP2-GFP. LjNSP2-GFP (A), Ljnsp2-1-GFP (B) fusion genes and control GFP gene (C), delivered by particle bombardment, were transiently expressed in onion epidermal cells and observed with a laser confocal microscope 24 h after bombardment. The nuclear localization of the Ljnsp2-1 mutant gene product is significantly suppressed compared with LjNSP2. Scale: 10 μm." path-from-xml="dsl017f8.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6982170/dsl017f8.jpeg" path-from-xml="dsl017f8.jpeg" target="_blank">View large</a><a section="6982170" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/dsl017f8.jpeg?Expires=2147483647&Signature=hvIMeJJq1czN5ZC9JySxVkRiDPh7c0b1qfXmpgFJyj3Fj-Qlaic8YY8RKm~5mh9c4zP82Qn8zuNM~Vn6~HX9rLxSbNXdoECpKqrHxN488AVRZJrfV~Im2cX6uCR3qDbT11Oi~4jKNiRGreNkpgB-3XWOOBc1SGMPXdfDf4W8Jra3hbRsCMM1X0Mi3TUxGicu5qCfqz9vAv8Dh1Xf6Okr2ces6RPuwo8Dq0NNWfd73gzb~m2hl3Px6z-t5CvHWj1-~IT69RC9zTB1lub5TVB0eEXAP8CzV4naBVj~NTZtZ~pYwxp9nbhS2wcZjgpwlJZydaeIMIN0PADAIzUdNRNYQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6982170&ar=464686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl017f8.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Nuclear localization of the LjNSP2-GFP. <em>LjNSP2</em>-<em>GFP</em> (A), <em>Ljnsp2-1</em>-<em>GFP</em> (B) fusion genes and control GFP gene (C), delivered by particle bombardment, were transiently expressed in onion epidermal cells and observed with a laser confocal microscope 24 h after bombardment. The nuclear localization of the <em>Ljnsp2-1</em> mutant gene product is significantly suppressed compared with LjNSP2. Scale: 10 μm.</p></div></div><div content-id="fig8" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 8</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/m_dsl017f8.jpeg?Expires=2147483647&Signature=P9m4mRSVugBioQtNId9p-gEziZslc-0KYSBRsbZ8o6SAIYSHySSNoyrJBgBbHtImpmvgsHQ33d0Wiubmj3SiCbwz8Dmu0Ek72tYcAw2RWJQHJGaZGxZaoktbJ~iqUJJpjUoUVfSMX1yJY3WpWDwnGeuNp7Zn6jphnvf0MkUYYIgnfkiMVP0jTkmXuafc~8q-jHMHwLhp60G558r2QJxzLHffLHr054cPgRpSCQdzz66wweriqwpL~arhZRjZYJn7c1LZFjXI8rfbKZAiG8fhGMBfPw3ayR3YMPMc2~~1tRh-vkR1aB0bMKBp38iL~pzGvgkUo-7JWXwuPpiRUEkLoQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Nuclear localization of the LjNSP2-GFP. LjNSP2-GFP (A), Ljnsp2-1-GFP (B) fusion genes and control GFP gene (C), delivered by particle bombardment, were transiently expressed in onion epidermal cells and observed with a laser confocal microscope 24 h after bombardment. The nuclear localization of the Ljnsp2-1 mutant gene product is significantly suppressed compared with LjNSP2. Scale: 10 μm." path-from-xml="dsl017f8.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6982170/dsl017f8.jpeg" path-from-xml="dsl017f8.jpeg" target="_blank">View large</a><a section="6982170" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/6/10.1093/dnares/dsl017/2/dsl017f8.jpeg?Expires=2147483647&Signature=hvIMeJJq1czN5ZC9JySxVkRiDPh7c0b1qfXmpgFJyj3Fj-Qlaic8YY8RKm~5mh9c4zP82Qn8zuNM~Vn6~HX9rLxSbNXdoECpKqrHxN488AVRZJrfV~Im2cX6uCR3qDbT11Oi~4jKNiRGreNkpgB-3XWOOBc1SGMPXdfDf4W8Jra3hbRsCMM1X0Mi3TUxGicu5qCfqz9vAv8Dh1Xf6Okr2ces6RPuwo8Dq0NNWfd73gzb~m2hl3Px6z-t5CvHWj1-~IT69RC9zTB1lub5TVB0eEXAP8CzV4naBVj~NTZtZ~pYwxp9nbhS2wcZjgpwlJZydaeIMIN0PADAIzUdNRNYQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6982170&ar=464686&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl017f8.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Nuclear localization of the LjNSP2-GFP. <em>LjNSP2</em>-<em>GFP</em> (A), <em>Ljnsp2-1</em>-<em>GFP</em> (B) fusion genes and control GFP gene (C), delivered by particle bombardment, were transiently expressed in onion epidermal cells and observed with a laser confocal microscope 24 h after bombardment. The nuclear localization of the <em>Ljnsp2-1</em> mutant gene product is significantly suppressed compared with LjNSP2. Scale: 10 μm.</p></div></div><p>The above LjNSP2-GFP fusion under the control of the CaMV 35S promoter or the <em>LjNSP2</em> promoter containing a 5.1 kb fragment of the 5′ flanking sequence was constructed and introduced into <em>Ljnsp2-2</em> mutants. These constructs complemented the <em>Ljnsp2-2</em> mutant phenotype, indicating that this fusion protein retained the LjNSP2 activity for nodulation. However, the transgenic roots showed no detectable GFP fluorescence (data not shown).</p>                            <h2 scrollto-destination=6982174 id="6982174" class="section-title" >4. Discussion</h2>
<p>So far the only described nodulation-specific mutants of <em>L. japonicus</em>, which completely lack nodules but have normal mycorrhization, are <em>Ljnsp2</em>, <em>nfr1</em>,<a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a>,<em>nfr5</em><a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> and <em>nin</em>.<a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a> One key difference among these mutants is the induction of Ca<sup>2+</sup> spiking, which might be required for the early common response in activation of the symbioses with both rhizobia and mycorrhizal fungi. The <em>Ljnsp2</em> mutants are normal for Ca<sup>2+</sup> spiking but the <em>nfr1</em> and <em>nfr5</em> mutants are blocked. Recently, <em>NFR1</em> and <em>NFR5</em> were shown to encode LysM-receptor-like-kinases, that were predicted to function in NF perception.<sup>5,6</sup> NIN functions downstream of LjNSP2 as indicated in this study. It is possible that LjNSP2 is earliest known protein executing nodulation-specific gene expression from Ca<sup>2+</sup> spiking induced through NFR1 and NFR5 in <em>L. japonicus</em>.</p><p>We have established a ∼2 Mb physical BAC contig that spans the <em>LjNSP2</em> locus (<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). However, <em>LjNSP2</em> region was only closed to the minimum of 14 BAC clones even in the population of 3 009 F2/F3 plants of the cross with Miyakojima MG-20. This corresponds to 24 Mb/cM, ∼78-fold greater than the average physical to genetic distance found in our high-density map of the <em>L. japonicus</em> (Wang et al., manuscript in preparation). This indicates that recombination in the chromosomal segment near the translocation site is highly suppressed in the cross combination of Gifu B-129 × Miyakojima MG-20. Therefore, we changed the crossing partner from Miyakojima MG-20 to <em>L. burttii</em>.<a reveal-id="b21" data-open="b21" class="link link-ref link-reveal xref-bibr">21</a> As a result, <em>LjNSP2</em> region narrowed up to 130 kb in one BAC clone (188C5) using the population of 8 472 F2 plants of the cross with <em>L. burttii</em>, indicating that <em>L. burttii</em> is significantly useful as alternative crossing partner of Gifu especially near the translocation site of chromosome 1.</p><p><em>LjNSP2</em> encodes a GRAS family protein of a putative transcription factor. The phylogenetic analysis revealed that LjNSP2 is most closely related to <em>M. truncatula</em> NSP2 and <em>P. sativum</em> NSP2.<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> The comparison of the mutant phenotypes suggests that the <em>LjNSP2</em>, <em>MtNSP2</em> and <em>PsNSP2</em> genes function at similar or parallel positions in the nodulation signal transduction. However, the expression pattern is different between <em>LjNSP2</em> and <em>M. truncatula NSP2</em>. <em>LjNSP2</em> is predominantly expressed in roots and its expression decreases in developed nodules. The expression of <em>M. truncatula NSP2</em> is observed in shoots as well as roots and is induced after rhizobial inoculation. These results suggest that, in contrast with a possible role for <em>M. truncatula NSP2</em> in another organ development in addition to nodulation, <em>LjNSP2</em> seems to be specialized in nodule initiation.</p><p>The expression of <em>NIN</em> and <em>ENOD40</em> is induced rapidly after rhizobial inoculation and NF-treatment. The <em>NIN</em> transcripts are detected in different tissues during various nodule stages, such as the dividing cells of the nodule primordia and the nodule vascular bundles. The <em>Ljnin</em> mutants are nodulation-minus and blocked in both the infection thread formation and the cortical cell division.<a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a>,<em>ENOD40</em> is induced in the root pericycle a few hours after rhizobial inoculation, and subsequently in the dividing cortical cells of the root and nodule primordia.<a reveal-id="b36  b38" data-open="b36  b38" class="link link-ref link-reveal xref-bibr">36–38</a>,<em>ENOD40</em> RNAi knock-down lines<a reveal-id="b39" data-open="b39" class="link link-ref link-reveal xref-bibr">39</a> or the possible co-suppression<a reveal-id="b40" data-open="b40" class="link link-ref link-reveal xref-bibr">40</a> suppressed nodule primordium formation, resulting in very poor nodulation. Despite of their function in nodule initiation, however, little is known about what kinds of transcription factors in the root activate <em>NIN</em> and <em>ENOD40</em> gene expression in response to rhizobia. In this study, we demonstrate that the induction of <em>NIN</em> and <em>LjENOD40-1</em> gene expression is clearly cancelled by the <em>Ljnsp2</em> mutation (<a reveal-id="fig7" data-open="fig7" class="link link-reveal link-table xref-fig">Fig. 7</a>). On the basis of these findings, we speculate that LjNSP2 may function as a transcriptional activator to directly or indirectly switch on the <em>NIN</em> and <em>LjENOD40-1</em> gene expression in nodule initiation. In order to address this issue, identification of the promoter region and subsequent binding assay using LjNSP2 protein would be of great importance in future.</p><p>Kaló et al.<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> reported that <em>M. truncatula</em> NSP2-GFP localizes in the endoplasmic reticulum and nuclear envelope and re-localizes into the nucleus rapidly after NF-treatment. In this case, they made a functional C-terminal GFP fusion under control of the constitutive CaMV 35S promoter that was introduced in <em>Medicago nsp2</em> mutant plants. Here, we attempted to detect LjNSP2-GFP, -YFP or -DsRED2 fusion in <em>L. japonicus</em> hairy roots. These fusions could complement the mutant phenotype but no fluorescence was detected even under the control of CaMV35S promoter. In place of <em>L. japonicus</em> we delivered LjNSP2-GFP fusion into onion epidermal cells by particle bombardment. The fusion exclusively localizes in the nucleus but not in the nuclear envelope. Although there is no way to explain the peculiar difference of subcellular localization of NSP2 between <em>Medicago</em> and onion, putative nucleoporins such as NUP133<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> and NUP85 (Saito et al. unpublished data) required for rhizobial and arbuscular mycorrhizal symbioses may retain the NSP2 putative transcription factor in the nuclear envelope of <em>M. truncatula</em>. Most recently, Heckmann et al. reported that non-nodulating mutant, SL781-3, carrying an allele of <em>LjNSP2</em> was found by TILLING.<a reveal-id="b41" data-open="b41" class="link link-ref link-reveal xref-bibr">41</a></p><p>We thank Prof. Jens Stougaard for kind providing the mutant seeds of <em>Ljsym35</em>. We wish to thank Krzyztof Szczyglowski and Martin Parniske for valuable comments on the manuscript and Anne Heckman for helpful discussion. This work was supported by the fund of Promotion of Basic Research Activities for Innovative Biosciences, (BRAIN), a grant in aid from the BBSRC (to J.A.D.), and a John Innes Foundation studentship and an Overseas Student Award (to H.M.). This work was also supported in part by Grants-in-Aid for Scientific Research on Priority Areas from the Ministry of Education, Culture, Sports, Science and Technology of Japan.</p>                            <h2 scrollto-destination=6982187 id="6982187" class="backreferences-title" >REFERENCES</h2>
<div class="ref-list"><div content-id="b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Oldroyd</div> <div class="given-names">G. E. D.</div></div>,  <div class="name"><div class="surname">Downie</div> <div class="given-names">J. A.</div></div>. </div><div class="article-title">Calcium, kinases and nodulation signalling in legumes</div>, <div class="source">Nat. Rev. Mol. Cell Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">566</div>-<div class="lpage">576</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Calcium%2C%20kinases%20and%20nodulation%20signalling%20in%20legumes&amp;author=G.%20E.%20D.%20Oldroyd&amp;author=J.%20A.%20Downie&amp;publication_year=2004&amp;journal=Nat.%20Rev.%20Mol.%20Cell%20Biol.&amp;volume=5&amp;pages=566-576" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrm1424" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrm1424" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrm1424"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15232574" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Catorina</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Galera</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">de Billy</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Four genes of <em>Medicago truncatula</em> controlling components of a Nod factor transduction pathway</div>, <div class="source">Plant Cell</div> , <div class="year">2000</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">1647</div>-<div class="lpage">1665</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Four%20genes%20of%20Medicago%20truncatula%20controlling%20components%20of%20a%20Nod%20factor%20transduction%20pathway&amp;author=R.%20Catorina&amp;author=C.%20Galera&amp;author=F.%20de%20Billy&amp;publication_year=2000&amp;journal=Plant%20Cell&amp;volume=12&amp;pages=1647-1665" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.12.9.1647" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.12.9.1647" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.12.9.1647"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11006338" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawaguchi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Imaizumi-Anraku</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Koiwa</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Root, root hair, and symbiotic mutants of the model legume <em>Lotus japonicus</em></div>, <div class="source">Mol. Plant-Microbe Interact.</div> , <div class="year">2002</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">17</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Root%2C%20root%20hair%2C%20and%20symbiotic%20mutants%20of%20the%20model%20legume%20Lotus%20japonicus&amp;author=M.%20Kawaguchi&amp;author=H.%20Imaizumi-Anraku&amp;author=H.%20Koiwa&amp;publication_year=2002&amp;journal=Mol.%20Plant-Microbe%20Interact.&amp;volume=15&amp;pages=17-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI.2002.15.1.17" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI.2002.15.1.17" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI.2002.15.1.17"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11843301" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Szczyglowski</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Shaw</div> <div class="given-names">R.S.</div></div>,  <div class="name"><div class="surname">Wopereis</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Nodule organogenesis and symbiotic mutants of the model legume <em>Lotus japonicus</em></div>, <div class="source">Mol. Plant-Microbe Interact.</div> , <div class="year">1998</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">684</div>-<div class="lpage">697</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nodule%20organogenesis%20and%20symbiotic%20mutants%20of%20the%20model%20legume%20Lotus%20japonicus&amp;author=K.%20Szczyglowski&amp;author=R.S.%20Shaw&amp;author=J.%20Wopereis&amp;publication_year=1998&amp;journal=Mol.%20Plant-Microbe%20Interact.&amp;volume=11&amp;pages=684-697" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI.1998.11.7.684" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI.1998.11.7.684" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI.1998.11.7.684"> </span></div> </div></div></div></div></div><div content-id="b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rodutoiu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Madsen</div> <div class="given-names">L. H.</div></div>,  <div class="name"><div class="surname">Madsen</div> <div class="given-names">E. B.</div></div>, et al. </div><div class="article-title">Plant recognition of symbiotic bacteria requires two LysM receptor-like kinases</div>, <div class="source">Nature</div> , <div class="year">2003</div>, vol. <div class="volume">425</div> (pg. <div class="fpage">585</div>-<div class="lpage">592</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20recognition%20of%20symbiotic%20bacteria%20requires%20two%20LysM%20receptor-like%20kinases&amp;author=S.%20Rodutoiu&amp;author=L.%20H.%20Madsen&amp;author=E.%20B.%20Madsen&amp;publication_year=2003&amp;journal=Nature&amp;volume=425&amp;pages=585-592" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02039" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02039" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02039"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14534578" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Madsen</div> <div class="given-names">L. H.</div></div>,  <div class="name"><div class="surname">Madsen</div> <div class="given-names">E. B.</div></div>,  <div class="name"><div class="surname">Rodutoiu</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">A receptor kinase gene of the LysM type is involved in legume perception of rhizobial signals</div>, <div class="source">Nature</div> , <div class="year">2003</div>, vol. <div class="volume">425</div> (pg. <div class="fpage">637</div>-<div class="lpage">640</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20receptor%20kinase%20gene%20of%20the%20LysM%20type%20is%20involved%20in%20legume%20perception%20of%20rhizobial%20signals&amp;author=L.%20H.%20Madsen&amp;author=E.%20B.%20Madsen&amp;author=S.%20Rodutoiu&amp;publication_year=2003&amp;journal=Nature&amp;volume=425&amp;pages=637-640" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02045" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02045" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02045"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14534591" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Limpens</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Franken</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Smit</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Willemse</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bisseling</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Geurts</div> <div class="given-names">R.</div></div>. </div><div class="article-title">LysM domain receptor kinases regulating rhizobial Nod factor-induced infection</div>, <div class="source">Science</div> , <div class="year">2003</div>, vol. <div class="volume">302</div> (pg. <div class="fpage">630</div>-<div class="lpage">633</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LysM%20domain%20receptor%20kinases%20regulating%20rhizobial%20Nod%20factor-induced%20infection&amp;author=E.%20Limpens&amp;author=C.%20Franken&amp;author=P.%20Smit&amp;author=J.%20Willemse&amp;author=T.%20Bisseling&amp;author=R.%20Geurts&amp;publication_year=2003&amp;journal=Science&amp;volume=302&amp;pages=630-633" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1090074" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1090074" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1090074"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12947035" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stracke</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kistner</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Yoshida</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">A plant receptor-like kinase required for both bacterial and fungal symbiosis</div>, <div class="source">Nature</div> , <div class="year">2002</div>, vol. <div class="volume">417</div> (pg. <div class="fpage">959</div>-<div class="lpage">962</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20plant%20receptor-like%20kinase%20required%20for%20both%20bacterial%20and%20fungal%20symbiosis&amp;author=S.%20Stracke&amp;author=C.%20Kistner&amp;author=S.%20Yoshida&amp;publication_year=2002&amp;journal=Nature&amp;volume=417&amp;pages=959-962" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature00841" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature00841" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature00841"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12087405" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Imaizumi-Anraku</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Takeda</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Charpentier</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Plastid proteins crucial for symbiotic fungal and bacterial entry into plant roots</div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">433</div> (pg. <div class="fpage">527</div>-<div class="lpage">531</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plastid%20proteins%20crucial%20for%20symbiotic%20fungal%20and%20bacterial%20entry%20into%20plant%20roots&amp;author=H.%20Imaizumi-Anraku&amp;author=N.%20Takeda&amp;author=M.%20Charpentier&amp;publication_year=2005&amp;journal=Nature&amp;volume=433&amp;pages=527-531" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03237" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03237" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03237"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15616514" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Anė</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Kiss</div> <div class="given-names">G. B.</div></div>,  <div class="name"><div class="surname">Riely</div> <div class="given-names">B. K.</div></div>, et al. </div><div class="article-title"><em>Medicago truncatula DMI1</em> required for bacterial and fungal symbioses in legumes</div>, <div class="source">Science</div> , <div class="year">2004</div>, vol. <div class="volume">303</div> (pg. <div class="fpage">1364</div>-<div class="lpage">1367</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Medicago%20truncatula%20DMI1%20required%20for%20bacterial%20and%20fungal%20symbioses%20in%20legumes&amp;author=J.%20M.%20An%C4%97&amp;author=G.%20B.%20Kiss&amp;author=B.%20K.%20Riely&amp;publication_year=2004&amp;journal=Science&amp;volume=303&amp;pages=1364-1367" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1092986" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1092986" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1092986"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14963334" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Endre</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Kereszt</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kevei</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Mihacea</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kaló</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Kiss</div> <div class="given-names">G. B.</div></div>. </div><div class="article-title">A receptor kinase gene regulating symbiotic nodule development</div>, <div class="source">Nature</div> , <div class="year">2002</div>, vol. <div class="volume">417</div> (pg. <div class="fpage">962</div>-<div class="lpage">966</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20receptor%20kinase%20gene%20regulating%20symbiotic%20nodule%20development&amp;author=G.%20Endre&amp;author=A.%20Kereszt&amp;author=Z.%20Kevei&amp;author=S.%20Mihacea&amp;author=P.%20Kal%C3%B3&amp;author=G.%20B.%20Kiss&amp;publication_year=2002&amp;journal=Nature&amp;volume=417&amp;pages=962-966" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature00842" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature00842" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature00842"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12087406" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lėvy</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bres</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Geurts</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">A putative Ca<sup>2+</sup> and Calmodulin-dependent protein kinase required for bacterial and fungal symbiosis</div>, <div class="source">Science</div> , <div class="year">2004</div>, vol. <div class="volume">303</div> (pg. <div class="fpage">1361</div>-<div class="lpage">1364</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20putative%20Ca2%2B%20and%20Calmodulin-dependent%20protein%20kinase%20required%20for%20bacterial%20and%20fungal%20symbiosis&amp;author=J.%20L%C4%97vy&amp;author=C.%20Bres&amp;author=R.%20Geurts&amp;publication_year=2004&amp;journal=Science&amp;volume=303&amp;pages=1361-1364" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1093038" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1093038" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1093038"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14963335" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tirichine</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Imaizumi-Anraku</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Yoshida</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Deregulation of a Ca<sup>2+</sup>/calmodulin-dependent kinase leads to spontaneous nodule development</div>, <div class="source">Nature</div> , <div class="year">2006</div>, vol. <div class="volume">441</div> (pg. <div class="fpage">1153</div>-<div class="lpage">1156</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deregulation%20of%20a%20Ca2%2B%2Fcalmodulin-dependent%20kinase%20leads%20to%20spontaneous%20nodule%20development&amp;author=L.%20Tirichine&amp;author=H.%20Imaizumi-Anraku&amp;author=S.%20Yoshida&amp;publication_year=2006&amp;journal=Nature&amp;volume=441&amp;pages=1153-1156" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature04862" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature04862" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature04862"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16810257" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kanamori</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Madsen</div> <div class="given-names">L. H.</div></div>,  <div class="name"><div class="surname">Radutoiu</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">A nucleoporin is required for induction of Ca<sup>2+</sup> spiking in legume nodule development and essential for rhizobial and fungal symbiosis</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2006</div>, vol. <div class="volume">103</div> (pg. <div class="fpage">359</div>-<div class="lpage">364</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20nucleoporin%20is%20required%20for%20induction%20of%20Ca2%2B%20spiking%20in%20legume%20nodule%20development%20and%20essential%20for%20rhizobial%20and%20fungal%20symbiosis&amp;author=N.%20Kanamori&amp;author=L.%20H.%20Madsen&amp;author=S.%20Radutoiu&amp;publication_year=2006&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=359-364" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0508883103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0508883103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0508883103"> </span></div> </div></div></div></div></div><div content-id="b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schauser</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Roussis</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Stiller</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Stougaard</div> <div class="given-names">J.</div></div>. </div><div class="article-title">A plant regulator controlling development of symbiotic root nodules</div>, <div class="source">Nature</div> , <div class="year">1999</div>, vol. <div class="volume">402</div> (pg. <div class="fpage">191</div>-<div class="lpage">195</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20plant%20regulator%20controlling%20development%20of%20symbiotic%20root%20nodules&amp;author=L.%20Schauser&amp;author=A.%20Roussis&amp;author=J.%20Stiller&amp;author=J.%20Stougaard&amp;publication_year=1999&amp;journal=Nature&amp;volume=402&amp;pages=191-195" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/46058" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F46058" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F46058"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10647012" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miwa</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Oldroyd</div> <div class="given-names">G. E. D.</div></div>,  <div class="name"><div class="surname">Downie</div> <div class="given-names">J. A.</div></div>. </div><div class="article-title">Analysis of Nod-factor-induced calcium signaling in root hairs of symbiotically defective mutants of <em>Lotus japonicus</em></div>, <div class="source">Mol. Plant-Microbe Interact.</div> , <div class="year">2006</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">914</div>-<div class="lpage">923</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20Nod-factor-induced%20calcium%20signaling%20in%20root%20hairs%20of%20symbiotically%20defective%20mutants%20of%20Lotus%20japonicus&amp;author=H.%20Miwa&amp;author=J.%20Sun&amp;author=G.%20E.%20D.%20Oldroyd&amp;author=J.%20A.%20Downie&amp;publication_year=2006&amp;journal=Mol.%20Plant-Microbe%20Interact.&amp;volume=19&amp;pages=914-923" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-19-0914" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-19-0914" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-19-0914"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16903357" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Smit</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Raedts</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Portyanko</div> <div class="given-names">V.</div></div>, et al. </div><div class="article-title">NSP1 of the GRAS protein family is essential for rhizobial Nod factor-induced transcription</div>, <div class="source">Science</div> , <div class="year">2005</div>, vol. <div class="volume">303</div> (pg. <div class="fpage">1789</div>-<div class="lpage">1791</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NSP1%20of%20the%20GRAS%20protein%20family%20is%20essential%20for%20rhizobial%20Nod%20factor-induced%20transcription&amp;author=P.%20Smit&amp;author=J.%20Raedts&amp;author=V.%20Portyanko&amp;publication_year=2005&amp;journal=Science&amp;volume=303&amp;pages=1789-1791" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1111025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1111025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1111025"> </span></div> </div></div></div></div></div><div content-id="b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaló</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Gleason</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Edwards</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Nodulation signaling in legumes requires NSP2, a member of the GRAS family of transcriptional regulators</div>, <div class="source">Science</div> , <div class="year">2005</div>, vol. <div class="volume">303</div> (pg. <div class="fpage">1786</div>-<div class="lpage">1789</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nodulation%20signaling%20in%20legumes%20requires%20NSP2%2C%20a%20member%20of%20the%20GRAS%20family%20of%20transcriptional%20regulators&amp;author=P.%20Kal%C3%B3&amp;author=C.%20Gleason&amp;author=A.%20Edwards&amp;publication_year=2005&amp;journal=Science&amp;volume=303&amp;pages=1786-1789" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1110951" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1110951" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1110951"> </span></div> </div></div></div></div></div><div content-id="b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thykjær</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Stiller</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Handberg</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Stougaard</div> <div class="given-names">J.</div></div>. </div><div class="article-title">The maize transposable element Ac is mobile in the legume <em>Lotus japonicus</em></div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">1995</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">981</div>-<div class="lpage">993</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20maize%20transposable%20element%20Ac%20is%20mobile%20in%20the%20legume%20Lotus%20japonicus&amp;author=T.%20Thykj%C3%A6r&amp;author=J.%20Stiller&amp;author=K.%20Handberg&amp;author=J.%20Jones&amp;author=J.%20Stougaard&amp;publication_year=1995&amp;journal=Plant%20Mol.%20Biol.&amp;volume=27&amp;pages=981-993" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00037025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00037025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00037025"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7766887" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawaguchi</div> <div class="given-names">M.</div></div>. </div><div class="article-title"><em>Lotus japonicus</em> ‘Miyakojima’ MG-20: an early flowering accession suitable for indoor genetics</div>, <div class="source">J. Plant Res.</div> , <div class="year">2000</div>, vol. <div class="volume">113</div> (pg. <div class="fpage">507</div>-<div class="lpage">509</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lotus%20japonicus%20%E2%80%98Miyakojima%E2%80%99%20MG-20%3A%20an%20early%20flowering%20accession%20suitable%20for%20indoor%20genetics&amp;author=M.%20Kawaguchi&amp;publication_year=2000&amp;journal=J.%20Plant%20Res.&amp;volume=113&amp;pages=507-509" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00013961" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00013961" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00013961"> </span></div> </div></div></div></div></div><div content-id="b21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawaguchi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Pedrosa-Harand</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Yano</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title"><em>Lotus burttii</em> takes a position of the third corner in the <em>Lotus</em> molecular genetics triangle</div>, <div class="source">DNA Res.</div> , <div class="year">2005</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">69</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lotus%20burttii%20takes%20a%20position%20of%20the%20third%20corner%20in%20the%20Lotus%20molecular%20genetics%20triangle&amp;author=M.%20Kawaguchi&amp;author=A.%20Pedrosa-Harand&amp;author=K.%20Yano&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=69-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/12.1.69" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F12.1.69" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F12.1.69"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16106754" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawasaki</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Murakami</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Imaizumi-Anraku</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Shimizu</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Mikami</div> <div class="given-names">I.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Nagata</div></div>,  <div class="name"><div class="surname">Tabata</div></div>. </div><div class="article-title">Construction of high-density map, genome library, and saturation mapping of nodulation genes</div>, <div class="source">Biotechnology in Agriculture and Forestry</div> , <div class="year">2003</div>, vol. <div class="volume">Vol. 52</div> <div class="publisher-name">Brassicas and Legumes</div>(pg. <div class="fpage">183</div>-<div class="lpage">202</div>)</div></div></div></div><div content-id="b23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kawasaki</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Murakami</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Genome analysis of <em>Lotus japonicus</em></div>, <div class="source">J. Plant Res.</div> , <div class="year">2000</div>, vol. <div class="volume">113</div> (pg. <div class="fpage">497</div>-<div class="lpage">506</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20analysis%20of%20Lotus%20japonicus&amp;author=S.%20Kawasaki&amp;author=Y.%20Murakami&amp;publication_year=2000&amp;journal=J.%20Plant%20Res.&amp;volume=113&amp;pages=497-506" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00013960" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00013960" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00013960"> </span></div> </div></div></div></div></div><div content-id="b24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chiu</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Niwa</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Zeng</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Hirano</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kobayashi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sheen</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Engineered GFP as a vital reporter in plants</div>, <div class="source">Curr. Biol.</div> , <div class="year">1996</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">325</div>-<div class="lpage">330</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Engineered%20GFP%20as%20a%20vital%20reporter%20in%20plants&amp;author=W.%20Chiu&amp;author=Y.%20Niwa&amp;author=W.%20Zeng&amp;author=T.%20Hirano&amp;author=H.%20Kobayashi&amp;author=J.%20Sheen&amp;publication_year=1996&amp;journal=Curr.%20Biol.&amp;volume=6&amp;pages=325-330" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0960-9822(02)00483-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0960-9822(02)00483-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0960-9822(02)00483-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8805250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Diaz</div> <div class="given-names">C. L.</div></div>,  <div class="name"><div class="surname">Melchers</div> <div class="given-names">L. S.</div></div>,  <div class="name"><div class="surname">Hooykaas</div> <div class="given-names">P. J. J.</div></div>,  <div class="name"><div class="surname">Lugtenberg</div> <div class="given-names">B. J. J.</div></div>,  <div class="name"><div class="surname">Kijne</div> <div class="given-names">J. W.</div></div>. </div><div class="article-title">Root lectin as a determinant of host-plant specificity in the <em>Rhizobium</em>-legume symbiosis</div>, <div class="source">Nature</div> , <div class="year">1989</div>, vol. <div class="volume">338</div> (pg. <div class="fpage">579</div>-<div class="lpage">581</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Root%20lectin%20as%20a%20determinant%20of%20host-plant%20specificity%20in%20the%20Rhizobium-legume%20symbiosis&amp;author=C.%20L.%20Diaz&amp;author=L.%20S.%20Melchers&amp;author=P.%20J.%20J.%20Hooykaas&amp;author=B.%20J.%20J.%20Lugtenberg&amp;author=J.%20W.%20Kijne&amp;publication_year=1989&amp;journal=Nature&amp;volume=338&amp;pages=579-581" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/338579a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F338579a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F338579a0"> </span></div> </div></div></div></div></div><div content-id="b26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kumagai</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kouchi</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Gene silencing by expression of Hairpin RNA in <em>Lotus japonicus</em> roots and root nodules</div>, <div class="source">Mol. Plant-Microbe Interact.</div> , <div class="year">2003</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">663</div>-<div class="lpage">668</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20silencing%20by%20expression%20of%20Hairpin%20RNA%20in%20Lotus%20japonicus%20roots%20and%20root%20nodules&amp;author=H.%20Kumagai&amp;author=H.%20Kouchi&amp;publication_year=2003&amp;journal=Mol.%20Plant-Microbe%20Interact.&amp;volume=16&amp;pages=663-668" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI.2003.16.8.663" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI.2003.16.8.663" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI.2003.16.8.663"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12906110" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Murray</div> <div class="given-names">M. G.</div></div>,  <div class="name"><div class="surname">Thompson</div> <div class="given-names">W. F.</div></div>. </div><div class="article-title">Rapid isolation of high molecular weight plant DNA</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1980</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">4321</div>-<div class="lpage">4325</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20isolation%20of%20high%20molecular%20weight%20plant%20DNA&amp;author=M.%20G.%20Murray&amp;author=W.%20F.%20Thompson&amp;publication_year=1980&amp;journal=Nucleic%20Acids%20Res.&amp;volume=8&amp;pages=4321-4325" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/8.19.4321" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F8.19.4321" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F8.19.4321"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7433111" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kouchi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Shimomura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hata</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Large-scale analysis of gene expression profiles during early stages of root nodule formation in a model legume, <em>Lotus japonicus</em></div>, <div class="source">DNA Res.</div> , <div class="year">2004</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">263</div>-<div class="lpage">274</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20analysis%20of%20gene%20expression%20profiles%20during%20early%20stages%20of%20root%20nodule%20formation%20in%20a%20model%20legume%2C%20Lotus%20japonicus&amp;author=H.%20Kouchi&amp;author=K.%20Shimomura&amp;author=S.%20Hata&amp;publication_year=2004&amp;journal=DNA%20Res.&amp;volume=11&amp;pages=263-274" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/11.4.263" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F11.4.263" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F11.4.263"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15500251" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chiu</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Niwa</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Zeng</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Hirano</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kobayashi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sheen</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Engineered GFP as a vital reporter in plants</div>, <div class="source">Curr. Biol.</div> , <div class="year">1996</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">325</div>-<div class="lpage">330</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Engineered%20GFP%20as%20a%20vital%20reporter%20in%20plants&amp;author=W.%20Chiu&amp;author=Y.%20Niwa&amp;author=W.%20Zeng&amp;author=T.%20Hirano&amp;author=H.%20Kobayashi&amp;author=J.%20Sheen&amp;publication_year=1996&amp;journal=Curr.%20Biol.&amp;volume=6&amp;pages=325-330" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0960-9822(02)00483-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0960-9822(02)00483-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0960-9822(02)00483-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8805250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Senoo</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Solaiman</div> <div class="given-names">M. Z.</div></div>,  <div class="name"><div class="surname">Kawaguchi</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Isolation of two different phenotypes of mycorrhizal mutants in the model legume plant <em>Lotus japonicus</em> after EMS-treatment</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2000</div>, vol. <div class="volume">41</div> (pg. <div class="fpage">726</div>-<div class="lpage">732</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20of%20two%20different%20phenotypes%20of%20mycorrhizal%20mutants%20in%20the%20model%20legume%20plant%20Lotus%20japonicus%20after%20EMS-treatment&amp;author=K.%20Senoo&amp;author=M.%20Z.%20Solaiman&amp;author=M.%20Kawaguchi&amp;publication_year=2000&amp;journal=Plant%20Cell%20Physiol.&amp;volume=41&amp;pages=726-732" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/41.6.726" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2F41.6.726" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2F41.6.726"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10945342" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sandal</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Petersen</div> <div class="given-names">T. R.</div></div>,  <div class="name"><div class="surname">Murray</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Genetics of symbiosis in <em>Lotus japonicus</em>: recombinant inbred lines, comparative genetic maps, and map position of 35 symbiotic loci</div>, <div class="source">Mol. Plant-Microbe Interact.</div> , <div class="year">2006</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">80</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetics%20of%20symbiosis%20in%20Lotus%20japonicus%3A%20recombinant%20inbred%20lines%2C%20comparative%20genetic%20maps%2C%20and%20map%20position%20of%2035%20symbiotic%20loci&amp;author=N.%20Sandal&amp;author=T.%20R.%20Petersen&amp;author=J.%20Murray&amp;publication_year=2006&amp;journal=Mol.%20Plant-Microbe%20Interact.&amp;volume=19&amp;pages=80-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-19-0080" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-19-0080" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-19-0080"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16404956" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bolle</div> <div class="given-names">C.</div></div>. </div><div class="article-title">The role of GRAS proteins in plant signal transduction and development</div>, <div class="source">Planta</div> , <div class="year">2004</div>, vol. <div class="volume">218</div> (pg. <div class="fpage">683</div>-<div class="lpage">692</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20GRAS%20proteins%20in%20plant%20signal%20transduction%20and%20development&amp;author=C.%20Bolle&amp;publication_year=2004&amp;journal=Planta&amp;volume=218&amp;pages=683-692" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00425-004-1203-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00425-004-1203-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00425-004-1203-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14760535" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pysh</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Wysocka-Diller</div> <div class="given-names">J. W.</div></div>,  <div class="name"><div class="surname">Camilleri</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Bouchez</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Benfey</div> <div class="given-names">P. N.</div></div>. </div><div class="article-title">The GRAS gene family in <em>Arabidopsis</em>: sequence characterization and basic expression analysis of the <em>SCARECROW-LIKE</em> genes</div>, <div class="source">Plant J.</div> , <div class="year">1999</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">111</div>-<div class="lpage">119</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20GRAS%20gene%20family%20in%20Arabidopsis%3A%20sequence%20characterization%20and%20basic%20expression%20analysis%20of%20the%20SCARECROW-LIKE%20genes&amp;author=L.%20Pysh&amp;author=J.%20W.%20Wysocka-Diller&amp;author=C.%20Camilleri&amp;author=D.%20Bouchez&amp;author=P.%20N.%20Benfey&amp;publication_year=1999&amp;journal=Plant%20J.&amp;volume=18&amp;pages=111-119" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.1999.00431.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.1999.00431.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.1999.00431.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10341448" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tian</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Wan</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Sun</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Genome-wide analysis of the GRAS gene family in rice and <em>Arabidopsis</em></div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">519</div>-<div class="lpage">532</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20the%20GRAS%20gene%20family%20in%20rice%20and%20Arabidopsis&amp;author=C.%20Tian&amp;author=P.%20Wan&amp;author=S.%20Sun&amp;author=J.%20Li&amp;author=M.%20Chen&amp;publication_year=2004&amp;journal=Plant%20Mol.%20Biol.&amp;volume=54&amp;pages=519-532" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/B:PLAN.0000038256.89809.57" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FB:PLAN.0000038256.89809.57" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FB:PLAN.0000038256.89809.57"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15316287" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Richards</div> <div class="given-names">D. E.</div></div>,  <div class="name"><div class="surname">Peng</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Harberd</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Plant GRAS and metazoan STATs: only family?</div>, <div class="source">BioEssays</div> , <div class="year">2000</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">573</div>-<div class="lpage">577</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20GRAS%20and%20metazoan%20STATs%3A%20only%20family%3F&amp;author=D.%20E.%20Richards&amp;author=J.%20Peng&amp;author=N.%20Harberd&amp;publication_year=2000&amp;journal=BioEssays&amp;volume=22&amp;pages=573-577" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(SICI)1521-1878(200006)22:6&lt;&gt;1.0.CO;2-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(SICI)1521-1878(200006)22:6&lt;&gt;1.0.CO;2-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(SICI)1521-1878(200006)22:6&lt;&gt;1.0.CO;2-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10842311" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kouchi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Hata</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Isolation and characterization of novel noduline cDNAs representing genes expressed at early stages of soybean nodule development</div>, <div class="source">Mol. Gen. Genet.</div> , <div class="year">1993</div>, vol. <div class="volume">238</div> (pg. <div class="fpage">106</div>-<div class="lpage">119</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20novel%20noduline%20cDNAs%20representing%20genes%20expressed%20at%20early%20stages%20of%20soybean%20nodule%20development&amp;author=H.%20Kouchi&amp;author=S.%20Hata&amp;publication_year=1993&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=238&amp;pages=106-119" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7683079" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">W. C.</div></div>,  <div class="name"><div class="surname">Katinakis</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Hendriks</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Characterization of GmENOD40, a gene showing novel patterns of cell-specific expression during soybean nodule development</div>, <div class="source">Plant J.</div> , <div class="year">1993</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">573</div>-<div class="lpage">585</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20GmENOD40%2C%20a%20gene%20showing%20novel%20patterns%20of%20cell-specific%20expression%20during%20soybean%20nodule%20development&amp;author=W.%20C.%20Yang&amp;author=P.%20Katinakis&amp;author=P.%20Hendriks&amp;publication_year=1993&amp;journal=Plant%20J.&amp;volume=3&amp;pages=573-585" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.1993.03040573.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.1993.03040573.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.1993.03040573.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8220464" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Minami</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Kouchi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Cohn</div> <div class="given-names">J. R.</div></div>,  <div class="name"><div class="surname">Ogawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Stancey</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Expression of the early nodulin, ENOD40, in soybean roots in response to various lipo-chitin signal molecules</div>, <div class="source">Plant J.</div> , <div class="year">1996</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">23</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20of%20the%20early%20nodulin%2C%20ENOD40%2C%20in%20soybean%20roots%20in%20response%20to%20various%20lipo-chitin%20signal%20molecules&amp;author=E.%20Minami&amp;author=H.%20Kouchi&amp;author=J.%20R.%20Cohn&amp;author=T.%20Ogawa&amp;author=G.%20Stancey&amp;publication_year=1996&amp;journal=Plant%20J.&amp;volume=10&amp;pages=23-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.1996.10010023.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.1996.10010023.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.1996.10010023.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8758977" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kumagai</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kinoshita</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Ridge</div> <div class="given-names">R. W.</div></div>,  <div class="name"><div class="surname">Kouchi</div> <div class="given-names">H.</div></div>. </div><div class="article-title">RNAi knock-down of <em>ENOD40s</em> leads to significant suppression of nodule formation in <em>Lotus japonicus</em></div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2006</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">1102</div>-<div class="lpage">1111</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNAi%20knock-down%20of%20ENOD40s%20leads%20to%20significant%20suppression%20of%20nodule%20formation%20in%20Lotus%20japonicus&amp;author=H.%20Kumagai&amp;author=E.%20Kinoshita&amp;author=R.%20W.%20Ridge&amp;author=H.%20Kouchi&amp;publication_year=2006&amp;journal=Plant%20Cell%20Physiol.&amp;volume=47&amp;pages=1102-1111" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcj081" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcj081" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcj081"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16816411" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Charon</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Sousa</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Crespi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kondorosi</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Alteration of enod40 expression modifies <em>Medicago truncatula</em> root nodule development induced by <em>Sinorhizobium meliloti</em></div>, <div class="source">Plant Cell</div> , <div class="year">1999</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1953</div>-<div class="lpage">1966</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alteration%20of%20enod40%20expression%20modifies%20Medicago%20truncatula%20root%20nodule%20development%20induced%20by%20Sinorhizobium%20meliloti&amp;author=C.%20Charon&amp;author=C.%20Sousa&amp;author=M.%20Crespi&amp;author=A.%20Kondorosi&amp;publication_year=1999&amp;journal=Plant%20Cell&amp;volume=11&amp;pages=1953-1966" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10521525" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Heckmann</div> <div class="given-names">A. B.</div></div>,  <div class="name"><div class="surname">Lombardo</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Miwa</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title"><em>Lotus japonicus</em> nodulation required two GRAS domain regulators, one of which is functionally conserved in a non-legume</div>, <div class="source">Plant Physiol.</div> , <div class="year">2006</div>, vol. <div class="volume">142</div> (pg. <div class="fpage">1739</div>-<div class="lpage">1750</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lotus%20japonicus%20nodulation%20required%20two%20GRAS%20domain%20regulators%2C%20one%20of%20which%20is%20functionally%20conserved%20in%20a%20non-legume&amp;author=A.%20B.%20Heckmann&amp;author=F.%20Lombardo&amp;author=H.%20Miwa&amp;publication_year=2006&amp;journal=Plant%20Physiol.&amp;volume=142&amp;pages=1739-1750" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.106.089508" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.106.089508" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.106.089508"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17071642" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2006. Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=464686&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">463</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000244930400002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">89</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsl017"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="464686"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000244930400002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f13%2f6%2f255%2f464686">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000244930400002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (89)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F13%2F6%2F255%2F464686" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/464686'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Positional Cloning Identifies Lotus japonicus NSP2, A Putative Transcription Factor of the GRAS Family, Required for NIN and ENOD40 Gene Expression in Nodule Initiation" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsl017");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f13%2f6%2f255%2f464686">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

