

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/4/325/450749","@type":"ScholarlyArticle","name":"Transcriptome Analysis of a North American Songbird,  Melospiza melodia ","datePublished":"2012-05-29","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2012-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss015","keywords":["EST","genetic markers","next generation sequencing","songbird speciation","SNP characterization"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/4/325/450749","author":[{"name":"Srivastava, Anuj","affiliation":"1  Institute of Bioinformatics, Davidson Life Sciences, University of Georgia ,  Athens, GA 30602 ,  USA","@type":"Person"},{"name":"Winker, Kevin","affiliation":"2  University of Alaska Museum ,  907 Yukon Drive, Fairbanks, AK 99775 ,  USA","@type":"Person"},{"name":"Shaw, Timothy I.","affiliation":"1  Institute of Bioinformatics, Davidson Life Sciences, University of Georgia ,  Athens, GA 30602 ,  USA","@type":"Person"},{"name":"Jones, Kenneth L.","affiliation":"3  Georgia Genomics Facility ,  University of Georgia ,  110 Riverbend Road, Athens, GA 30602 ,  USA    4  Department of Environmental Health Science ,  University of Georgia ,  EHS Building, Athens, GA 30602 ,  USA    5  Department of Biochemistry and Molecular Genetics ,  University of Colorado School of Medicine ,  Aurora, CO 80045 ,  USA","@type":"Person"},{"name":"Glenn, Travis C.","affiliation":"1  Institute of Bioinformatics, Davidson Life Sciences, University of Georgia ,  Athens, GA 30602 ,  USA    3  Georgia Genomics Facility ,  University of Georgia ,  110 Riverbend Road, Athens, GA 30602 ,  USA    4  Department of Environmental Health Science ,  University of Georgia ,  EHS Building, Athens, GA 30602 ,  USA","@type":"Person"}],"description":"Abstract.  An effective way to understand the genomics of divergence in non-model organisms is to use the transcriptome to identify genes associated with diverg","pageStart":"325","pageEnd":"333","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/4/1/m_cover.gif?Expires=2147483647&Signature=eprZ0s37mFZTW8q04XyXQPhslj5EsmNEiVd2xsNgTNSRMFCtDBeg3Jl-WpIZ04VSSHYdf4drJ9cM4eXR~wAjVX4Ka1lGx5jT4Aan8~ek1e8YFczc~efvkNtRr1j4OBLyYeTJMiA~vza484yAcFT5QrseynX-Et0Amq8jonZzJ4uDq5hZrvVMLdx8~LCdEtaG8K8Th5wGC0jLW5Dvd--ISvvgHuZMFgwQgbYRdBL~2OYDNhG3CqoT4aVgmKKF8B3aG~dK1DqjtB8y6Zeh2up0vNZSY74Ozsnuq5BVobk3Bk2WTZDMIWq3CmTTmKd9YkkZCnPAEOdYujN8-TWtB1D4Tw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Transcriptome Analysis of a North American Songbird,  Melospiza melodia ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/4/1/m_cover.gif?Expires=2147483647&Signature=eprZ0s37mFZTW8q04XyXQPhslj5EsmNEiVd2xsNgTNSRMFCtDBeg3Jl-WpIZ04VSSHYdf4drJ9cM4eXR~wAjVX4Ka1lGx5jT4Aan8~ek1e8YFczc~efvkNtRr1j4OBLyYeTJMiA~vza484yAcFT5QrseynX-Et0Amq8jonZzJ4uDq5hZrvVMLdx8~LCdEtaG8K8Th5wGC0jLW5Dvd--ISvvgHuZMFgwQgbYRdBL~2OYDNhG3CqoT4aVgmKKF8B3aG~dK1DqjtB8y6Zeh2up0vNZSY74Ozsnuq5BVobk3Bk2WTZDMIWq3CmTTmKd9YkkZCnPAEOdYujN8-TWtB1D4Tw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Transcriptome Analysis of a North American Songbird,  Melospiza melodia " />
    <meta property="og:description" content="Abstract.  An effective way to understand the genomics of divergence in non-model organisms is to use the transcriptome to identify genes associated with diverg" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/4/325/450749" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2012-05-29" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Srivastava, Anuj" /><meta name="citation_author_institution" content="Institute of Bioinformatics, Davidson Life Sciences, University of Georgia, Athens, GA 30602, USA" /><meta name="citation_author" content="Winker, Kevin" /><meta name="citation_author_institution" content="University of Alaska Museum, 907 Yukon Drive, Fairbanks, AK 99775, USA" /><meta name="citation_author" content="Shaw, Timothy I." /><meta name="citation_author_institution" content="Institute of Bioinformatics, Davidson Life Sciences, University of Georgia, Athens, GA 30602, USA" /><meta name="citation_author" content="Jones, Kenneth L." /><meta name="citation_author_institution" content="Georgia Genomics Facility, University of Georgia, 110 Riverbend Road, Athens, GA 30602, USA" /><meta name="citation_author_institution" content="Department of Environmental Health Science, University of Georgia, EHS Building, Athens, GA 30602, USA" /><meta name="citation_author_institution" content="Department of Biochemistry and Molecular Genetics, University of Colorado School of Medicine, Aurora, CO 80045, USA" /><meta name="citation_author" content="Glenn, Travis C." /><meta name="citation_author_institution" content="Institute of Bioinformatics, Davidson Life Sciences, University of Georgia, Athens, GA 30602, USA" /><meta name="citation_author_institution" content="Georgia Genomics Facility, University of Georgia, 110 Riverbend Road, Athens, GA 30602, USA" /><meta name="citation_author_institution" content="Department of Environmental Health Science, University of Georgia, EHS Building, Athens, GA 30602, USA" /><meta name="citation_title" content="Transcriptome Analysis of a North American Songbird, Melospiza melodia" /><meta name="citation_firstpage" content="325" /><meta name="citation_lastpage" content="333" /><meta name="citation_doi" content="10.1093/dnares/dss015" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2012/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=A golden age for evolutionary genetics? Genomic studies of adaptation in natural populations; Trends Genet.;  citation_year=2010;  citation_volume=26;  citation_pages=484-92; " /><meta name="citation_reference" content="citation_title=Adaptation genomics: the next generation; Trends Ecol. Evol.;  citation_year=2010;  citation_volume=25;  citation_pages=705-12; " /><meta name="citation_reference" content="citation_title=The molecular ecologist's guide to expressed sequence tags; Mol. Ecol.;  citation_year=2007;  citation_volume=16;  citation_pages=907-24; " /><meta name="citation_reference" content="citation_title=Rapidly developing functional genomics in ecological model systems via 454 transcriptome sequencing; Genetica;  citation_year=2010;  citation_volume=138;  citation_pages=433-51; " /><meta name="citation_reference" content="citation_title=Comparative genomics based on massive parallel transcriptome sequencing reveals patterns of substitution and selection across 10 bird species; Mol. Ecol.;  citation_year=2010;  citation_volume=19;  citation_issue=Suppl. 1);  citation_pages=266-76; " /><meta name="citation_reference" content="citation_title=Transcriptome sequencing in an ecologically important tree species: assembly, annotation, and marker discovery; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=180" /><meta name="citation_reference" content="citation_title=Characterisation of the transcriptome of a wild great tit Parus major population by next generation sequencing; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=283" /><meta name="citation_reference" content="citation_title=Nucleotide divergence vs. gene expression differentiation: comparative transcriptome sequencing in natural isolates from the carrion crow and its hybrid zone with the hooded crow; Mol. Ecol.;  citation_year=2010;  citation_volume=19;  citation_issue=Suppl. 1);  citation_pages=162-75; " /><meta name="citation_reference" content="citation_title=The song sparrow, Melospiza melodia, as a ring species: patterns of geographic variation, a revision of subspecies, and implications for speciation; Syst. Biodivers.;  citation_year=2009;  citation_volume=7;  citation_pages=33-62; " /><meta name="citation_reference" content="citation_title=Song sparrow (Melospiza melodia); citation_text=No. 704; citation_publisher=The Birds of North America, Inc., Philadelphia, PA; The Birds of North America;  citation_year=2002; " /><meta name="citation_reference" content="citation_title=A wing expressed sequence tag resource for Bicyclus anynana butterflies, an evo-devo model; BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=130" /><meta name="citation_reference" content="citation_title=Pyrobayes: an improved base caller for SNP discovery in pyrosequences; Nat. Methods;  citation_year=2008;  citation_volume=5;  citation_pages=179-81; " /><meta name="citation_reference" content="citation_title=Using the miraEST assembler for reliable and automated mRNA transcript assembly and SNP detection in sequenced ESTs; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=1147-59; " /><meta name="citation_reference" content="citation_title=A general approach to single-nucleotide polymorphism discovery; Nat. Genet.;  citation_year=1999;  citation_volume=23;  citation_pages=452-6; " /><meta name="citation_reference" content="citation_title=Phylogenetic and functional assessment of orthologs inference projects and methods; Plos Comput. Biol;  citation_year=2009;  citation_volume=5;  citation_pages=e1000262" /><meta name="citation_reference" content="citation_title=BLAT–the BLAST-like alignment tool; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=656-64; " /><meta name="citation_reference" content="citation_title=Next generation sequencing and analysis of a conserved transcriptome of New Zealand's kiwi; BMC Evol. Biol.;  citation_year=2010;  citation_volume=10;  citation_pages=387" /><meta name="citation_reference" content="citation_title=Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=3674-6; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=Clustal W and clustal X version 2.0; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2947-8; " /><meta name="citation_reference" content="citation_title=PAL2NAL: robust conversion of protein sequence alignments into the corresponding codon alignments; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=W609-12; " /><meta name="citation_reference" content="citation_title=PAML 4: phylogenetic analysis by maximum likelihood; Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=1586-91; " /><meta name="citation_reference" content="citation_title=Gene Ontology: tool for the unification of biology; citation_author=The Gene Ontology Consortium; Nat. Genet.;  citation_year=2000;  citation_volume=25;  citation_pages=25-9; " /><meta name="citation_reference" content="citation_title=Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources; Nat. Protoc.;  citation_year=2009;  citation_volume=4;  citation_pages=44-57; " /><meta name="citation_reference" content="citation_title=Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=1-13; " /><meta name="citation_reference" content="citation_title=Identification and validation of housekeeping genes as internal control for gene expression in an intravenous LPS inflammation model in chickens; Vet. Immunol. Immunopathol.;  citation_year=2008;  citation_volume=122;  citation_pages=312-7; " /><meta name="citation_reference" content="citation_title=Digital gene expression analysis of the zebra finch genome; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=219" /><meta name="citation_reference" content="citation_title=Evolutionary stasis: the stable chromosomes of birds; Trends Ecol. Evol.;  citation_year=2010;  citation_volume=25;  citation_pages=283-91; " /><meta name="citation_reference" content="citation_title=The evolution of the avian genome as revealed by comparative molecular cytogenetics; Cytogenet Genome. Res.;  citation_year=2007;  citation_volume=117;  citation_pages=64-77; " /><meta name="citation_reference" content="citation_title=Looking for Darwin in all the wrong places: the misguided quest for positive selection at the nucleotide sequence level; Heredity;  citation_year=2007;  citation_volume=99;  citation_pages=364-73; " /><meta name="citation_reference" content="citation_title=Interpopulation patterns of divergence and selection across the transcriptome of the copepod Tigriopus californicus; Mol. Ecol.;  citation_year=2011;  citation_volume=20;  citation_pages=560-72; " /><meta name="citation_reference" content="citation_title=Rapid evolution and selection inferred from the transcriptomes of sympatric crater lake cichlid fishes; Mol. Ecol.;  citation_year=2010;  citation_volume=19;  citation_issue=Suppl. 1);  citation_pages=197-211; " /><meta name="citation_reference" content="citation_title=Statistical methods for detecting molecular adaptation; Trends Ecol. Evol.;  citation_year=2000;  citation_volume=15;  citation_pages=496-503; " /><meta name="citation_reference" content="citation_title=Estimates of positive Darwinian selection are inflated by errors in sequencing, annotation, and alignment; Genome Biol. Evol.;  citation_year=2009;  citation_volume=1;  citation_pages=114-18; " /><meta name="citation_reference" content="citation_title=The population genetics of dN/dS; PLoS Genet.;  citation_year=2008;  citation_volume=4;  citation_pages=e1000304" /><meta name="citation_reference" content="citation_title=Comparisons of dN/dS are time dependent for closely related bacterial genomes; J. Theor. Biol.;  citation_year=2006;  citation_volume=239;  citation_pages=226-35; " /><meta name="citation_reference" content="citation_title=Nonlinear dynamics of nonsynonymous (dN) and synonymous (dS) substitution rates affects inference of selection; Genome Biol. Evol.;  citation_year=2009;  citation_volume=1;  citation_pages=308-19; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/4/325/1231724/dss015.pdf" /><meta name="description" content="Abstract.  An effective way to understand the genomics of divergence in non-model organisms is to use the transcriptome to identify genes associated with diverg" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/4/325/450749" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/4/325/450749" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/19/4/325/450749");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dss015';
    </script>
<title>
	Transcriptome Analysis of a North American Songbird, Melospiza melodia | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./450749" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bd9YPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTcml2YXN0YXZhLCBBbnVqIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEJpb2luZm9ybWF0aWNzLCBEYXZpZHNvbiBMaWZlIFNjaWVuY2VzLCBVbml2ZXJzaXR5IG9mIEdlb3JnaWEsIEF0aGVucywgR0EgMzA2MDIsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IldpbmtlciwgS2V2aW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJVbml2ZXJzaXR5IG9mIEFsYXNrYSBNdXNldW0sIDkwNyBZdWtvbiBEcml2ZSwgRmFpcmJhbmtzLCBBSyA5OTc3NSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hhdywgVGltb3RoeSBJLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBCaW9pbmZvcm1hdGljcywgRGF2aWRzb24gTGlmZSBTY2llbmNlcywgVW5pdmVyc2l0eSBvZiBHZW9yZ2lhLCBBdGhlbnMsIEdBIDMwNjAyLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJKb25lcywgS2VubmV0aCBMLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikdlb3JnaWEgR2Vub21pY3MgRmFjaWxpdHksIFVuaXZlcnNpdHkgb2YgR2VvcmdpYSwgMTEwIFJpdmVyYmVuZCBSb2FkLCBBdGhlbnMsIEdBIDMwNjAyLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEVudmlyb25tZW50YWwgSGVhbHRoIFNjaWVuY2UsIFVuaXZlcnNpdHkgb2YgR2VvcmdpYSwgRUhTIEJ1aWxkaW5nLCBBdGhlbnMsIEdBIDMwNjAyLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2NoZW1pc3RyeSBhbmQgTW9sZWN1bGFyIEdlbmV0aWNzLCBVbml2ZXJzaXR5IG9mIENvbG9yYWRvIFNjaG9vbCBvZiBNZWRpY2luZSwgQXVyb3JhLCBDTyA4MDA0NSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR2xlbm4sIFRyYXZpcyBDLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBCaW9pbmZvcm1hdGljcywgRGF2aWRzb24gTGlmZSBTY2llbmNlcywgVW5pdmVyc2l0eSBvZiBHZW9yZ2lhLCBBdGhlbnMsIEdBIDMwNjAyLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHZW9yZ2lhIEdlbm9taWNzIEZhY2lsaXR5LCBVbml2ZXJzaXR5IG9mIEdlb3JnaWEsIDExMCBSaXZlcmJlbmQgUm9hZCwgQXRoZW5zLCBHQSAzMDYwMiwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBFbnZpcm9ubWVudGFsIEhlYWx0aCBTY2llbmNlLCBVbml2ZXJzaXR5IG9mIEdlb3JnaWEsIEVIUyBCdWlsZGluZywgQXRoZW5zLCBHQSAzMDYwMiwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJUcmFuc2NyaXB0b21lIEFuYWx5c2lzIG9mIGEgTm9ydGggQW1lcmljYW4gU29uZ2JpcmQsIE1lbG9zcGl6YSBtZWxvZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMzI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIzMzMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3MwMTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMi8wOC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ29sZGVuIGFnZSBmb3IgZXZvbHV0aW9uYXJ5IGdlbmV0aWNzPyBHZW5vbWljIHN0dWRpZXMgb2YgYWRhcHRhdGlvbiBpbiBuYXR1cmFsIHBvcHVsYXRpb25zOyBUcmVuZHMgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9NDg0LTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZGFwdGF0aW9uIGdlbm9taWNzOiB0aGUgbmV4dCBnZW5lcmF0aW9uOyBUcmVuZHMgRWNvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz03MDUtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtb2xlY3VsYXIgZWNvbG9naXN0J3MgZ3VpZGUgdG8gZXhwcmVzc2VkIHNlcXVlbmNlIHRhZ3M7IE1vbC4gRWNvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz05MDctMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhcGlkbHkgZGV2ZWxvcGluZyBmdW5jdGlvbmFsIGdlbm9taWNzIGluIGVjb2xvZ2ljYWwgbW9kZWwgc3lzdGVtcyB2aWEgNDU0IHRyYW5zY3JpcHRvbWUgc2VxdWVuY2luZzsgR2VuZXRpY2E7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTM4OyAgY2l0YXRpb25fcGFnZXM9NDMzLTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBnZW5vbWljcyBiYXNlZCBvbiBtYXNzaXZlIHBhcmFsbGVsIHRyYW5zY3JpcHRvbWUgc2VxdWVuY2luZyByZXZlYWxzIHBhdHRlcm5zIG9mIHN1YnN0aXR1dGlvbiBhbmQgc2VsZWN0aW9uIGFjcm9zcyAxMCBiaXJkIHNwZWNpZXM7IE1vbC4gRWNvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9pc3N1ZT1TdXBwbC4gMSk7ICBjaXRhdGlvbl9wYWdlcz0yNjYtNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWUgc2VxdWVuY2luZyBpbiBhbiBlY29sb2dpY2FsbHkgaW1wb3J0YW50IHRyZWUgc3BlY2llczogYXNzZW1ibHksIGFubm90YXRpb24sIGFuZCBtYXJrZXIgZGlzY292ZXJ5OyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xODAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpc2F0aW9uIG9mIHRoZSB0cmFuc2NyaXB0b21lIG9mIGEgd2lsZCBncmVhdCB0aXQgUGFydXMgbWFqb3IgcG9wdWxhdGlvbiBieSBuZXh0IGdlbmVyYXRpb24gc2VxdWVuY2luZzsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MjgzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnVjbGVvdGlkZSBkaXZlcmdlbmNlIHZzLiBnZW5lIGV4cHJlc3Npb24gZGlmZmVyZW50aWF0aW9uOiBjb21wYXJhdGl2ZSB0cmFuc2NyaXB0b21lIHNlcXVlbmNpbmcgaW4gbmF0dXJhbCBpc29sYXRlcyBmcm9tIHRoZSBjYXJyaW9uIGNyb3cgYW5kIGl0cyBoeWJyaWQgem9uZSB3aXRoIHRoZSBob29kZWQgY3JvdzsgTW9sLiBFY29sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX2lzc3VlPVN1cHBsLiAxKTsgIGNpdGF0aW9uX3BhZ2VzPTE2Mi03NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHNvbmcgc3BhcnJvdywgTWVsb3NwaXphIG1lbG9kaWEsIGFzIGEgcmluZyBzcGVjaWVzOiBwYXR0ZXJucyBvZiBnZW9ncmFwaGljIHZhcmlhdGlvbiwgYSByZXZpc2lvbiBvZiBzdWJzcGVjaWVzLCBhbmQgaW1wbGljYXRpb25zIGZvciBzcGVjaWF0aW9uOyBTeXN0LiBCaW9kaXZlcnMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0zMy02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U29uZyBzcGFycm93IChNZWxvc3BpemEgbWVsb2RpYSk7IGNpdGF0aW9uX3RleHQ9Tm8uIDcwNDsgY2l0YXRpb25fcHVibGlzaGVyPVRoZSBCaXJkcyBvZiBOb3J0aCBBbWVyaWNhLCBJbmMuLCBQaGlsYWRlbHBoaWEsIFBBOyBUaGUgQmlyZHMgb2YgTm9ydGggQW1lcmljYTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSB3aW5nIGV4cHJlc3NlZCBzZXF1ZW5jZSB0YWcgcmVzb3VyY2UgZm9yIEJpY3ljbHVzIGFueW5hbmEgYnV0dGVyZmxpZXMsIGFuIGV2by1kZXZvIG1vZGVsOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTEzMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVB5cm9iYXllczogYW4gaW1wcm92ZWQgYmFzZSBjYWxsZXIgZm9yIFNOUCBkaXNjb3ZlcnkgaW4gcHlyb3NlcXVlbmNlczsgTmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xNzktODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzaW5nIHRoZSBtaXJhRVNUIGFzc2VtYmxlciBmb3IgcmVsaWFibGUgYW5kIGF1dG9tYXRlZCBtUk5BIHRyYW5zY3JpcHQgYXNzZW1ibHkgYW5kIFNOUCBkZXRlY3Rpb24gaW4gc2VxdWVuY2VkIEVTVHM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTE0Ny01OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnZW5lcmFsIGFwcHJvYWNoIHRvIHNpbmdsZS1udWNsZW90aWRlIHBvbHltb3JwaGlzbSBkaXNjb3Zlcnk7IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9NDUyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeWxvZ2VuZXRpYyBhbmQgZnVuY3Rpb25hbCBhc3Nlc3NtZW50IG9mIG9ydGhvbG9ncyBpbmZlcmVuY2UgcHJvamVjdHMgYW5kIG1ldGhvZHM7IFBsb3MgQ29tcHV0LiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMDI2MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJMQVTigJN0aGUgQkxBU1QtbGlrZSBhbGlnbm1lbnQgdG9vbDsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz02NTYtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5leHQgZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIGFuZCBhbmFseXNpcyBvZiBhIGNvbnNlcnZlZCB0cmFuc2NyaXB0b21lIG9mIE5ldyBaZWFsYW5kJ3Mga2l3aTsgQk1DIEV2b2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9Mzg3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Qmxhc3QyR086IGEgdW5pdmVyc2FsIHRvb2wgZm9yIGFubm90YXRpb24sIHZpc3VhbGl6YXRpb24gYW5kIGFuYWx5c2lzIGluIGZ1bmN0aW9uYWwgZ2Vub21pY3MgcmVzZWFyY2g7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MzY3NC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HYXBwZWQgQkxBU1QgYW5kIFBTSS1CTEFTVDogYSBuZXcgZ2VuZXJhdGlvbiBvZiBwcm90ZWluIGRhdGFiYXNlIHNlYXJjaCBwcm9ncmFtczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzM4OS00MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsdXN0YWwgVyBhbmQgY2x1c3RhbCBYIHZlcnNpb24gMi4wOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTI5NDctODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UEFMMk5BTDogcm9idXN0IGNvbnZlcnNpb24gb2YgcHJvdGVpbiBzZXF1ZW5jZSBhbGlnbm1lbnRzIGludG8gdGhlIGNvcnJlc3BvbmRpbmcgY29kb24gYWxpZ25tZW50czsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9VzYwOS0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UEFNTCA0OiBwaHlsb2dlbmV0aWMgYW5hbHlzaXMgYnkgbWF4aW11bSBsaWtlbGlob29kOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MTU4Ni05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBPbnRvbG9neTogdG9vbCBmb3IgdGhlIHVuaWZpY2F0aW9uIG9mIGJpb2xvZ3k7IGNpdGF0aW9uX2F1dGhvcj1UaGUgR2VuZSBPbnRvbG9neSBDb25zb3J0aXVtOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTI1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5c3RlbWF0aWMgYW5kIGludGVncmF0aXZlIGFuYWx5c2lzIG9mIGxhcmdlIGdlbmUgbGlzdHMgdXNpbmcgREFWSUQgYmlvaW5mb3JtYXRpY3MgcmVzb3VyY2VzOyBOYXQuIFByb3RvYy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTQ0LTU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW9pbmZvcm1hdGljcyBlbnJpY2htZW50IHRvb2xzOiBwYXRocyB0b3dhcmQgdGhlIGNvbXByZWhlbnNpdmUgZnVuY3Rpb25hbCBhbmFseXNpcyBvZiBsYXJnZSBnZW5lIGxpc3RzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz0xLTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBhbmQgdmFsaWRhdGlvbiBvZiBob3VzZWtlZXBpbmcgZ2VuZXMgYXMgaW50ZXJuYWwgY29udHJvbCBmb3IgZ2VuZSBleHByZXNzaW9uIGluIGFuIGludHJhdmVub3VzIExQUyBpbmZsYW1tYXRpb24gbW9kZWwgaW4gY2hpY2tlbnM7IFZldC4gSW1tdW5vbC4gSW1tdW5vcGF0aG9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMjI7ICBjaXRhdGlvbl9wYWdlcz0zMTItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlnaXRhbCBnZW5lIGV4cHJlc3Npb24gYW5hbHlzaXMgb2YgdGhlIHplYnJhIGZpbmNoIGdlbm9tZTsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MjE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uYXJ5IHN0YXNpczogdGhlIHN0YWJsZSBjaHJvbW9zb21lcyBvZiBiaXJkczsgVHJlbmRzIEVjb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MjgzLTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXZvbHV0aW9uIG9mIHRoZSBhdmlhbiBnZW5vbWUgYXMgcmV2ZWFsZWQgYnkgY29tcGFyYXRpdmUgbW9sZWN1bGFyIGN5dG9nZW5ldGljczsgQ3l0b2dlbmV0IEdlbm9tZS4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTc7ICBjaXRhdGlvbl9wYWdlcz02NC03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG9va2luZyBmb3IgRGFyd2luIGluIGFsbCB0aGUgd3JvbmcgcGxhY2VzOiB0aGUgbWlzZ3VpZGVkIHF1ZXN0IGZvciBwb3NpdGl2ZSBzZWxlY3Rpb24gYXQgdGhlIG51Y2xlb3RpZGUgc2VxdWVuY2UgbGV2ZWw7IEhlcmVkaXR5OyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTk5OyAgY2l0YXRpb25fcGFnZXM9MzY0LTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlcnBvcHVsYXRpb24gcGF0dGVybnMgb2YgZGl2ZXJnZW5jZSBhbmQgc2VsZWN0aW9uIGFjcm9zcyB0aGUgdHJhbnNjcmlwdG9tZSBvZiB0aGUgY29wZXBvZCBUaWdyaW9wdXMgY2FsaWZvcm5pY3VzOyBNb2wuIEVjb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9NTYwLTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXBpZCBldm9sdXRpb24gYW5kIHNlbGVjdGlvbiBpbmZlcnJlZCBmcm9tIHRoZSB0cmFuc2NyaXB0b21lcyBvZiBzeW1wYXRyaWMgY3JhdGVyIGxha2UgY2ljaGxpZCBmaXNoZXM7IE1vbC4gRWNvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9pc3N1ZT1TdXBwbC4gMSk7ICBjaXRhdGlvbl9wYWdlcz0xOTctMjExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdGF0aXN0aWNhbCBtZXRob2RzIGZvciBkZXRlY3RpbmcgbW9sZWN1bGFyIGFkYXB0YXRpb247IFRyZW5kcyBFY29sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTQ5Ni01MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzdGltYXRlcyBvZiBwb3NpdGl2ZSBEYXJ3aW5pYW4gc2VsZWN0aW9uIGFyZSBpbmZsYXRlZCBieSBlcnJvcnMgaW4gc2VxdWVuY2luZywgYW5ub3RhdGlvbiwgYW5kIGFsaWdubWVudDsgR2Vub21lIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0xMTQtMTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBwb3B1bGF0aW9uIGdlbmV0aWNzIG9mIGROL2RTOyBQTG9TIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDAzMDQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJpc29ucyBvZiBkTi9kUyBhcmUgdGltZSBkZXBlbmRlbnQgZm9yIGNsb3NlbHkgcmVsYXRlZCBiYWN0ZXJpYWwgZ2Vub21lczsgSi4gVGhlb3IuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTIzOTsgIGNpdGF0aW9uX3BhZ2VzPTIyNi0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm9ubGluZWFyIGR5bmFtaWNzIG9mIG5vbnN5bm9ueW1vdXMgKGROKSBhbmQgc3lub255bW91cyAoZFMpIHN1YnN0aXR1dGlvbiByYXRlcyBhZmZlY3RzIGluZmVyZW5jZSBvZiBzZWxlY3Rpb247IEdlbm9tZSBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MzA4LTE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTkvNC8zMjUvMTIzMTcyNC9kc3MwMTUucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEFuIGVmZmVjdGl2ZSB3YXkgdG8gdW5kZXJzdGFuZCB0aGUgZ2Vub21pY3Mgb2YgZGl2ZXJnZW5jZSBpbiBub24tbW9kZWwgb3JnYW5pc21zIGlzIHRvIHVzZSB0aGUgdHJhbnNjcmlwdG9tZSB0byBpZGVudGlmeSBnZW5lcyBhc3NvY2lhdGVkIHdpdGggZGl2ZXJnIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00NTA3NDkmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rkp79Fki/A8qQhfq0mVQP/KS5FtfY=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f4%2f325%2f450749" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f4%2f325%2f450749" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=17634" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/4/1/m_cover.gif?Expires=2147483647&amp;Signature=eprZ0s37mFZTW8q04XyXQPhslj5EsmNEiVd2xsNgTNSRMFCtDBeg3Jl-WpIZ04VSSHYdf4drJ9cM4eXR~wAjVX4Ka1lGx5jT4Aan8~ek1e8YFczc~efvkNtRr1j4OBLyYeTJMiA~vza484yAcFT5QrseynX-Et0Amq8jonZzJ4uDq5hZrvVMLdx8~LCdEtaG8K8Th5wGC0jLW5Dvd--ISvvgHuZMFgwQgbYRdBL~2OYDNhG3CqoT4aVgmKKF8B3aG~dK1DqjtB8y6Zeh2up0vNZSY74Ozsnuq5BVobk3Bk2WTZDMIWq3CmTTmKd9YkkZCnPAEOdYujN8-TWtB1D4Tw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/4">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6764008">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6764008">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6764012">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6764012">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6764018">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6764018">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6764056">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6764056">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6764143">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6764143">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6764147">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6764147">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6764151">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6764151">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6764155">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6764155">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/19/4/317/450529">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/19/4/335/451025">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Transcriptome Analysis of a North American Songbird, <em>Melospiza melodia</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Anuj Srivastava</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Institute of Bioinformatics, Davidson Life Sciences, University of Georgia</div>, <div class="addr-line">Athens, GA 30602</div>, <div class="country">USA</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +1 207-288-6000. Fax. +1 907-474-5469. E-mail: <a href="mailto:anuj.srivastava@jax.org" target="_blank">anuj.srivastava@jax.org</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Anuj+Srivastava">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Srivastava A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Srivastava A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Anuj Srivastava</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kevin Winker</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">University of Alaska Museum</div>, <div class="addr-line">907 Yukon Drive, Fairbanks, AK 99775</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kevin+Winker">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Winker K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Winker K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kevin Winker</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Timothy I. Shaw</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Institute of Bioinformatics, Davidson Life Sciences, University of Georgia</div>, <div class="addr-line">Athens, GA 30602</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Timothy+I.+Shaw">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shaw T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shaw T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Timothy I. Shaw</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenneth L. Jones</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Georgia Genomics Facility</div>, <div class="insititution">University of Georgia</div>, <div class="addr-line">110 Riverbend Road, Athens, GA 30602</div>, <div class="country">USA</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Environmental Health Science</div>, <div class="insititution">University of Georgia</div>, <div class="addr-line">EHS Building, Athens, GA 30602</div>, <div class="country">USA</div></div><div class="aff"><div class="label -label">5</div><div class="addr-line">Department of Biochemistry and Molecular Genetics</div>, <div class="insititution">University of Colorado School of Medicine</div>, <div class="addr-line">Aurora, CO 80045</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenneth+L.+Jones">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jones K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jones K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kenneth L. Jones</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Travis C. Glenn</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Institute of Bioinformatics, Davidson Life Sciences, University of Georgia</div>, <div class="addr-line">Athens, GA 30602</div>, <div class="country">USA</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Georgia Genomics Facility</div>, <div class="insititution">University of Georgia</div>, <div class="addr-line">110 Riverbend Road, Athens, GA 30602</div>, <div class="country">USA</div></div><div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Environmental Health Science</div>, <div class="insititution">University of Georgia</div>, <div class="addr-line">EHS Building, Athens, GA 30602</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Travis+C.+Glenn">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Glenn T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Glenn T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Travis C. Glenn</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="label -label">‡</div><div class="footnote-content">Present address: The Jackson Laboratory, 600 Main Street, Bar Harbor, ME 04609, USA.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Minoru Yoshida</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 4, 1 August 2012, Pages 325–333, <a href='https://doi.org/10.1093/dnares/dss015'>https://doi.org/10.1093/dnares/dss015</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">29 May 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">08 January 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">19 April 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="450749" href="/dnaresearch/article-pdf/19/4/325/1231724/dss015.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Anuj Srivastava, Kevin Winker, Timothy I. Shaw, Kenneth L. Jones, Travis C. Glenn;  Transcriptome Analysis of a North American Songbird, <em>Melospiza melodia</em>, <em>DNA Research</em>, Volume 19, Issue 4, 1 August 2012, Pages 325–333, <a href='https://doi.org/10.1093/dnares/dss015'>https://doi.org/10.1093/dnares/dss015</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/450749?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/450749?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/450749?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/450749?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/450749?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/450749?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/450749?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Transcriptome%20Analysis%20of%20a%20North%20American%20Songbird%2C%20%26lt%3Bem%26gt%3BMelospiza%20melodia%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2012-05-29&amp;volumeNum=19&amp;issueNum=4&amp;author=Srivastava%2C%20Anuj%3B%20Winker%2C%20Kevin&amp;startPage=325&amp;endPage=333&amp;contentId=10.1093%2Fdnares%2Fdss015&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f4%2f325%2f450749" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6764008 id="6764008" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>An effective way to understand the genomics of divergence in non-model organisms is to use the transcriptome to identify genes associated with divergence. We examine the transcriptome of the song sparrow (<em>Melospiza melodia</em>) and contrast it with the avian models zebra finch (<em>Taeniopygia guttata</em>) and chicken (<em>Gallus gallus</em>). We aimed to (i) obtain a functional annotation of a substantial portion of the song sparrow transcriptome; (ii) compare transcript divergence; (iii) efficiently characterize single nucleotide polymorphism/indel markers possibly fixed between song sparrow subspecies; and (iv) identify the most common set of transcripts in birds using the zebra finch as a reference. Using two individuals from each of three populations, whole-body mRNA was normalized and sequenced (110Mb total). The assembly yielded 38 539 contigs [N50 (the length–weighted median) = 482 bp]; 4574 were orthologous to both model genomes and 3680 are functionally annotated. This low-coverage scan of the song sparrow transcriptome revealed 29 982 SNPs/indels, 1402 fixed between populations and subspecies. Referencing zebra finch and chicken, we identified 43 and 5 fast-evolving genes, respectively. We also identified the most common set of transcripts present in birds with respect to zebra finch. This study provides new insight into songbird transcriptomes, and candidate markers identified here may help research in songbirds (oscine Passeriformes), a frequently studied group.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="EST">EST</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genetic markers&quot;">genetic markers</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;next generation sequencing&quot;">next generation sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;songbird speciation&quot;">songbird speciation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;SNP characterization&quot;">SNP characterization</a></div>                            <h2 scrollto-destination=6764012 id="6764012" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Determining the genetic underpinnings of organismal divergence and speciation will provide insight into the evolutionary generation of biodiversity, and next-generation sequencing is propelling such studies in non-model organisms.<sup><a reveal-id="DSS015C1" data-open="DSS015C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSS015C2" data-open="DSS015C2" class="link link-ref link-reveal xref-bibr">2</a></sup> An effective way to initiate genomic-wide data sets in non-model organisms is to focus on the transcriptome, or expressed sequence, which, unlike a whole-genome approach, increases the data's focus on functional genomic attributes.<sup><a reveal-id="DSS015C3" data-open="DSS015C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSS015C4" data-open="DSS015C4" class="link link-ref link-reveal xref-bibr">4</a></sup> As these data become available, evolutionary biologists will be able to make contrasts within and among lineages to identify genes associated with divergence.<sup><a reveal-id="DSS015C5  DSS015C8" data-open="DSS015C5  DSS015C8" class="link link-ref link-reveal xref-bibr">5–8</a></sup> To gain insight into the genes associated with avian diversification, we examine the transcriptome of the song sparrow (<em>Melospiza melodia</em>) and contrast it with the model birds zebra finch (<em>Taeniopygia guttata</em>) and chicken (<em>G. gallus</em>).</p><p>The song sparrow is broadly distributed across North America and exhibits pronounced morphological variation, with 25 subspecies recognized (of 52 described<sup><a reveal-id="DSS015C9" data-open="DSS015C9" class="link link-ref link-reveal xref-bibr">9</a></sup>). It has been extensively studied over the past 70 yrs; it is considered a model vertebrate species for field research; and it will continue to be a focus for questions about the causes of population variation in behaviour, demographics, and morphology.<sup><a reveal-id="DSS015C10" data-open="DSS015C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Our goals in this study were to (i) obtain a functional annotation of a substantial portion of the song sparrow transcriptome; (ii) compare transcript divergence between the song sparrow and the two bird genomes sequenced and assembled to the highest quality thus far, zebra finch (<em>T. guttata</em>) and chicken (<em>G. gallus</em>); (iii) efficiently characterize a set of single nucleotide polymorphism (SNP)/indel markers that may be fixed between song sparrow subspecies; and (iv) identify the most common set of transcripts present in bird species using the zebra finch as a reference. Achieving these goals will establish important baseline data for a non-model organism in a speciose group (passerines or songbirds) frequently studied.</p>                            <h2 scrollto-destination=6764018 id="6764018" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6764021 id="6764021" class="section-title" data-legacy-id=s2a>2.1. Samples, cDNA library, and sequencing</h3>
<p>Two song sparrows still undergoing growth (from embryo to just-fledged) were sampled from each of three Alaska populations (the northwestern most distribution of the species), chosen because they span some of the most pronounced morphological diversity that occurs in the species (Fig. <a reveal-id="DSS015F1" data-open="DSS015F1" class="link link-reveal link-table xref-fig">1</a>): two island populations of <em>M. m. maxima</em> (from Attu and Adak islands; an egg and a very young nestling from Attu Island, unvouchered; and vouchers UAM 27831 and 27832 from Adak Island) and one mainland population of <em>M. m. caurina</em> (from Cordova, vouchers UAM 27829 and 27830). The Attu and Adak populations of <em>Melospiza m. maxima</em> are the largest in the species and also have different plumage coloration; in addition, they are non-migratory, unlike the population from Cordova, which is also smaller and darker (Fig. <a reveal-id="DSS015F1" data-open="DSS015F1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="6764026" scrollto-destination="6764026"></a>
<div id="DSS015F1" data-id="DSS015F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/m_dss01501.jpeg?Expires=2147483647&Signature=0dp5owPp750Xi9lA3RrRH~Ut6IMRYVuE5ij7B4aFf4xvzQujkJoRPGupqiztkaPqcfxMgLgsS-KUB7VIb3fHN0YWW817LCWyK8FvUsz5kwL3-1m-x9cxq6oOTw3icyV1niQYRAUQeG3ZL5XVHSgFNbau1JTM5dtfKtcz4xJnQW2Md7J7JWayCGziFVe1tXRqdfHpWqqJMhFIypcxxQXsV1ZY8sZcOYOu5GkF~Np6xqQ46ocWV77Uz0SPmtP7Kl8AStmSk2nIPradqp~UFidp-JU-7je6Y9ARzWsfIUmVgquDw2LN3TfeQ7-5c4c3sRJojCkyfA2gRDdTGcB1uzYTZg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Samples in this study came from Cordova (Melospiza melodia caurina, right in inset) and Adak and Attu islands (M. m. maxima, left in inset); grey shading indicates the species' range." path-from-xml="dss01501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6764026/dss01501.jpeg" path-from-xml="dss01501.jpeg" target="_blank">View large</a><a section="6764026" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/dss01501.jpeg?Expires=2147483647&Signature=kZZLltrcrk3Gp4OKxlAc4j9d0c0By354P2Dadz~wcUQvkfAtP-kr6ewM1rpQkhUfKC6b~lBh8eQgCjudfR1ThjI5B0beEluAz1~WzKb5Q3jrwFBHZ0DB0FenLycQ8DEcgxqejiVzIGjTuCRkA0mxryNxUFHjJ7i8Ra1bGt3k2S2vbrIlUoZYehP6Lr5qrsHhNYI9z6nJ~~9KtSMBqSHQgrYoRHMcASDFWlTyG9DndPvsLyhQfDUhwLiC-Bul1HRPw62mgTMOB6Rlkgx3Lh9rWlh~hY4WlJiZQsWPMqRXJmysquSBnONIBhRjraIIsyFt8~wx5ftyVHOttAR3EtJ8TA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6764026&ar=450749&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Samples in this study came from Cordova (<em>Melospiza melodia caurina</em>, right in inset) and Adak and Attu islands (<em>M. m. maxima</em>, left in inset); grey shading indicates the species' range.</p></div></div><div content-id="DSS015F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/m_dss01501.jpeg?Expires=2147483647&Signature=0dp5owPp750Xi9lA3RrRH~Ut6IMRYVuE5ij7B4aFf4xvzQujkJoRPGupqiztkaPqcfxMgLgsS-KUB7VIb3fHN0YWW817LCWyK8FvUsz5kwL3-1m-x9cxq6oOTw3icyV1niQYRAUQeG3ZL5XVHSgFNbau1JTM5dtfKtcz4xJnQW2Md7J7JWayCGziFVe1tXRqdfHpWqqJMhFIypcxxQXsV1ZY8sZcOYOu5GkF~Np6xqQ46ocWV77Uz0SPmtP7Kl8AStmSk2nIPradqp~UFidp-JU-7je6Y9ARzWsfIUmVgquDw2LN3TfeQ7-5c4c3sRJojCkyfA2gRDdTGcB1uzYTZg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Samples in this study came from Cordova (Melospiza melodia caurina, right in inset) and Adak and Attu islands (M. m. maxima, left in inset); grey shading indicates the species' range." path-from-xml="dss01501.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6764026/dss01501.jpeg" path-from-xml="dss01501.jpeg" target="_blank">View large</a><a section="6764026" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/dss01501.jpeg?Expires=2147483647&Signature=kZZLltrcrk3Gp4OKxlAc4j9d0c0By354P2Dadz~wcUQvkfAtP-kr6ewM1rpQkhUfKC6b~lBh8eQgCjudfR1ThjI5B0beEluAz1~WzKb5Q3jrwFBHZ0DB0FenLycQ8DEcgxqejiVzIGjTuCRkA0mxryNxUFHjJ7i8Ra1bGt3k2S2vbrIlUoZYehP6Lr5qrsHhNYI9z6nJ~~9KtSMBqSHQgrYoRHMcASDFWlTyG9DndPvsLyhQfDUhwLiC-Bul1HRPw62mgTMOB6Rlkgx3Lh9rWlh~hY4WlJiZQsWPMqRXJmysquSBnONIBhRjraIIsyFt8~wx5ftyVHOttAR3EtJ8TA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6764026&ar=450749&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01501.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Samples in this study came from Cordova (<em>Melospiza melodia caurina</em>, right in inset) and Adak and Attu islands (<em>M. m. maxima</em>, left in inset); grey shading indicates the species' range.</p></div></div><p>All samples were obtained in June (spring) at a very young age and only two were sexable (both females, one each from Cordova and Adak). The egg was homogenized, whereas from the others six tissues (brain, liver, heart, muscle, bone, and pancreas) were taken, minced and placed in RNAlater (Qiagen, Valencia, CA) within minutes of death and then frozen. In the laboratory, tissues were homogenized and total RNA was isolated using Trizol (Invitrogen, Carlsbad, CA) and subsequently cleaned using a Qiagen RNeasy column.</p><p>Equal amounts of RNA from individuals of each population were pooled and an MINT universal cDNA kit (Evrogen, Moscow, Russia) with primers modified specifically for 454 procedures<sup><a reveal-id="DSS015C11" data-open="DSS015C11" class="link link-ref link-reveal xref-bibr">11</a></sup> was used to create cDNA libraries enriched for full-length transcripts. We then normalized the three cDNA libraries using the TRIMMER cDNA normalization kit (Evrogen) to substantially decrease the relative abundance of common transcripts. The normalized cDNA was fragmented and prepared for sequencing using standard 454 procedures, including independent molecular identifiers [MID tags: Cordova (MID 13), Attu (MID 18) and Adak (MID 19)] for each of the three populations. As each library contained a unique MID tag, libraries were pooled and sequenced as a single sample. Sequencing was performed at the University of Georgia's Georgia Genomics Facility on a Roche 454 FLX using Titanium chemistry.</p>                            <h3 scrollto-destination=6764036 id="6764036" class="section-title" data-legacy-id=s2b>2.2. Assembly, polymorphism, and ortholog identification</h3>
<p>Bases were called from the 454-generated sff file using Pyrobayes,<sup><a reveal-id="DSS015C12" data-open="DSS015C12" class="link link-ref link-reveal xref-bibr">12</a></sup> which provides improved accuracy in the estimation of base qualities for pyrosequences. We removed MINT primer sequences, short sequences, and other contaminatants using SeqClean (<a class="link link-uri" href="http://compbio.dfci.harvard.edu">http://compbio.dfci.harvard.edu</a>), and reads from all three populations were combined. We performed a combined assembly of reads using MIRA,<sup><a reveal-id="DSS015C13" data-open="DSS015C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and then used GigaBayes,<sup><a reveal-id="DSS015C14" data-open="DSS015C14" class="link link-ref link-reveal xref-bibr">14</a></sup> a short-read SNP and short indel discovery program, to detect polymorphisms. To make the SNP/indel predictions more reliable, we used the more stringent criteria that the minor allele must occur at least three times and be present at ≥10% relative to the major allele frequency when &gt;30 reads per locus were obtained (after combining all the reads for particular alleles among different subspecies; sequences with fewer reads are considered the minor allele and sequences with more reads are considered the major allele). We identified orthologous contigs (against the zebra finch and chicken genomes) using the reciprocal blast approach, because it has been found to be superior to sophisticated orthology detection algorithms.<sup><a reveal-id="DSS015C15" data-open="DSS015C15" class="link link-ref link-reveal xref-bibr">15</a></sup> A stringent cutoff of 1e−20 was used to separate paralogues from orthologues. The cDNA sequences from the zebra finch (taeGut3.2.4.60.cdna.all.fa) and chicken (WASHUC2.60.cdna.all.fa) were obtained from the Biomart database (<a class="link link-uri" href="http://www.biomart.org">www.biomart.org</a>). Although the zebra finch is a passerine and thus more closely related to the song sparrow, the chicken database contains sequences from whole growing chicks, whereas that of the zebra finch emphasizes neural transcripts.</p><p>To identify likely genomic positions of the song sparrow contigs, we mapped them against genomic sequences of the zebra finch (taeGut3.2.4.60.dna_rm.toplevel.fa) and chicken (WASHUC2.60.dna_rm.toplevel.fa) using BLAT<sup><a reveal-id="DSS015C16" data-open="DSS015C16" class="link link-ref link-reveal xref-bibr">16</a></sup> with default criteria. We obtained feature information for protein-coding genes and ncRNA using the Ensemble (<a class="link link-uri" href="http://uswest.ensembl.org/index.html/">http://uswest.ensembl.org/index.html/</a>) Xenoref and gtf files, respectively.</p>                            <h3 scrollto-destination=6764042 id="6764042" class="section-title" data-legacy-id=s2c>2.3. Most common set of transcripts in birds</h3>
<p>To find the most common set of transcripts in birds with respect to zebra finch, we collected and assembled (454 GS assembler version 2.5) the transcriptome sequence of 12 bird species (publicly available sequence<sup><a reveal-id="DSS015C5" data-open="DSS015C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSS015C7" data-open="DSS015C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSS015C8" data-open="DSS015C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS015C17" data-open="DSS015C17" class="link link-ref link-reveal xref-bibr">17</a></sup>). The orthologous sequence with respect to zebra finch was determined using the bidirectional blast best hit method (1e−20). Only contigs &gt;200 bp were used in the analysis. After determining the orthologous sequences, we sorted them in decreasing order and added orthologous sequences from other species sequentially to find the most common set.</p>                            <h3 scrollto-destination=6764047 id="6764047" class="section-title" data-legacy-id=s2d>2.4. Functional annotation of contigs</h3>
<p>We used Blast2GO<sup><a reveal-id="DSS015C18" data-open="DSS015C18" class="link link-ref link-reveal xref-bibr">18</a></sup> (B2G) to functionally annotate the contigs. A combined graph was generated for each gene ontology (GO) category. For the molecular function division, a graph was obtained using default criteria and for the other two divisions (cellular component and biological process), seq/node filter values were changed to 4/10 to prevent overloading the graphs.</p>                            <h3 scrollto-destination=6764051 id="6764051" class="section-title" data-legacy-id=s2e>2.5. Estimation of substitution rates</h3>
<p>Substitution rates were estimated for contigs that were orthologous to both zebra finch and chicken. Reading frames for these contigs were identified using BLASTX<sup><a reveal-id="DSS015C19" data-open="DSS015C19" class="link link-ref link-reveal xref-bibr">19</a></sup> against protein sequences of zebra finch (taeGut3.2.4.60.pep.all.fa) and chicken (WASHUC2.60.pep.all.fa) obtained from Biomart (<a class="link link-uri" href="http://www.biomart.org">www.biomart.org</a>). Sequences that produced significant alignments were extracted (using their coordinates), translated, and aligned using CLUSTALW.<sup><a reveal-id="DSS015C20" data-open="DSS015C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Sequences that contained frame shifts were excluded from the analysis. Corresponding codon alignments were produced using PAL2NAL,<sup><a reveal-id="DSS015C21" data-open="DSS015C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and, finally, rates were estimated using a maximum likelihood method implemented in the CODEML program of the PAML package Version 4.1.<sup><a reveal-id="DSS015C22" data-open="DSS015C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Pairwise maximum likelihood analyses were performed in runmode-2. The estimated rates of non-synonymous to synonymous substitutions (<em>K</em><sub>a</sub><em>/K</em><sub>s</sub> values) were plotted as a scatter plot in the range of 0–2.0.</p>                            <h2 scrollto-destination=6764056 id="6764056" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=6764058 id="6764058" class="section-title" data-legacy-id=s3a>3.1. Sequence assembly</h3>
<p>The pooled reads from all three populations yielded 131 Mb (458 808 sequences) of raw data, which was reduced to 110Mb (381 474 sequences) after the use of SeqClean (Table <a reveal-id="DSS015TB1" data-open="DSS015TB1" class="link link-reveal link-table xref-fig">1</a>). The mean raw and cleaned read lengths were 286 and 290 bp, respectively. Poor-quality reads were often very short and were purged entirely prior to assembly. Without a reference genome for the song sparrow, <em>de novo</em> assembly was required. Cleaned sequences were assembled into 38 539 contigs with N50 and N90 values of 482 and 317 bp, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data</a>). There were 1417 singletons. The mean coverage per contig was 3.93 X and the mean GC content per contig was 43.6%. </p>                        <a id="6764063" scrollto-destination="6764063"></a>
<div content-id="DSS015TB1" class="table-modal"><div class="table-wrap"><div id="DSS015TB1" class="table-wrap-title" data-id="DSS015TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Number of reads and assembly statistics for three song sparrow populations (SRA 048516)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Subspecies </th><th>Locality </th><th><em>n</em><sup>a</sup> </th><th>MID </th><th>Raw reads </th><th>Cleaned reads </th><th>Cleaned bases (MB) </th></tr></thead><tbody><tr><td><em>M. m. caurina</em> </td><td>Cordova </td><td>2 </td><td>13 </td><td>138 439 </td><td>114 098 </td><td>32.5 </td></tr><tr><td><em>M. m. maxima</em> </td><td>Adak </td><td>2 </td><td>19 </td><td>135 588 </td><td>117 166 </td><td>34.7 </td></tr><tr><td><em>M. m. maxima</em> </td><td>Attu </td><td>2 </td><td>18 </td><td>184 781 </td><td>150 210 </td><td>42.8 </td></tr><tr><td><em>Combined</em> </td><td>— </td><td>6 </td><td>— </td><td>458 808 </td><td>381 474 </td><td>110 </td></tr></tbody></table></div><div content-id="DSS015TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Subspecies </th><th>Locality </th><th><em>n</em><sup>a</sup> </th><th>MID </th><th>Raw reads </th><th>Cleaned reads </th><th>Cleaned bases (MB) </th></tr></thead><tbody><tr><td><em>M. m. caurina</em> </td><td>Cordova </td><td>2 </td><td>13 </td><td>138 439 </td><td>114 098 </td><td>32.5 </td></tr><tr><td><em>M. m. maxima</em> </td><td>Adak </td><td>2 </td><td>19 </td><td>135 588 </td><td>117 166 </td><td>34.7 </td></tr><tr><td><em>M. m. maxima</em> </td><td>Attu </td><td>2 </td><td>18 </td><td>184 781 </td><td>150 210 </td><td>42.8 </td></tr><tr><td><em>Combined</em> </td><td>— </td><td>6 </td><td>— </td><td>458 808 </td><td>381 474 </td><td>110 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Number of individuals pooled prior to sequencing.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6764063" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS015TB1" class="table-wrap-title" data-id="DSS015TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Number of reads and assembly statistics for three song sparrow populations (SRA 048516)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Subspecies </th><th>Locality </th><th><em>n</em><sup>a</sup> </th><th>MID </th><th>Raw reads </th><th>Cleaned reads </th><th>Cleaned bases (MB) </th></tr></thead><tbody><tr><td><em>M. m. caurina</em> </td><td>Cordova </td><td>2 </td><td>13 </td><td>138 439 </td><td>114 098 </td><td>32.5 </td></tr><tr><td><em>M. m. maxima</em> </td><td>Adak </td><td>2 </td><td>19 </td><td>135 588 </td><td>117 166 </td><td>34.7 </td></tr><tr><td><em>M. m. maxima</em> </td><td>Attu </td><td>2 </td><td>18 </td><td>184 781 </td><td>150 210 </td><td>42.8 </td></tr><tr><td><em>Combined</em> </td><td>— </td><td>6 </td><td>— </td><td>458 808 </td><td>381 474 </td><td>110 </td></tr></tbody></table></div><div content-id="DSS015TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Subspecies </th><th>Locality </th><th><em>n</em><sup>a</sup> </th><th>MID </th><th>Raw reads </th><th>Cleaned reads </th><th>Cleaned bases (MB) </th></tr></thead><tbody><tr><td><em>M. m. caurina</em> </td><td>Cordova </td><td>2 </td><td>13 </td><td>138 439 </td><td>114 098 </td><td>32.5 </td></tr><tr><td><em>M. m. maxima</em> </td><td>Adak </td><td>2 </td><td>19 </td><td>135 588 </td><td>117 166 </td><td>34.7 </td></tr><tr><td><em>M. m. maxima</em> </td><td>Attu </td><td>2 </td><td>18 </td><td>184 781 </td><td>150 210 </td><td>42.8 </td></tr><tr><td><em>Combined</em> </td><td>— </td><td>6 </td><td>— </td><td>458 808 </td><td>381 474 </td><td>110 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Number of individuals pooled prior to sequencing.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6764063" target="_blank">View Large</a></div></div><p>We acknowledge that the amount of sequencing presented is insufficient to allow a high-quality assembly of the extremely diverse transcriptome that we have sampled. A large number of tissues were sampled, and these clearly contain a large and diverse set of transcripts (see Section 3.2). Simulations indicate that transcriptomes sequenced with 454 Titanium chemistry will quickly lead to about twice as many contigs as transcripts, and additional sequences only gradually cause the number of contigs to reach the number of transcripts (i.e. the point when contigs = transcripts; data not shown). Thus, quite large numbers of additional sequences will be necessary to fully assemble the transcripts contained in these cDNA libraries. Given the relatively high cost of 454 sequencing, it would be more economical to obtain the additional sequences as paired-end reads on Illumina or Ion Torrent platforms.</p>                            <h3 scrollto-destination=6764068 id="6764068" class="section-title" data-legacy-id=s3b>3.2. Functional annotation</h3>
<p>B2G, which we used to functionally annotate the contigs, has three annotation steps involving (i) a blast against databases, (ii) mapping against GO resources, and (iii) annotation to generate reliable functional assignments. In our data, 12 880 of the contigs (33.46% overall, of which 8540 were unique hits) had significant matches to currently known proteins in the NCBI non-redundant protein database. Because one-third of the contigs hit the same proteins as other contigs in our data, this indicates that large transcripts were often split among multiple contigs in our assembly. Although it is possible to use the zebra finch or chicken proteins as a reference to scaffold the song sparrow contigs, we did not do this because it could make chimeras, and assembly of full-length genes was not a major goal of this work.</p><p>As expected, zebra finch and chicken were identified as the top two species with the best blast hits for our song sparrow contigs (Table <a reveal-id="DSS015TB2" data-open="DSS015TB2" class="link link-reveal link-table xref-fig">2</a>). Contigs with significant blast matches were functionally annotated. GO resource assignment was found for 3949 (10.2%) of the total contigs (with 24 363 GO terms; there can be multiple terms per contig), of which 3367 (8.7% of all contigs) were functionally annotated (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Sheet 1</a>). </p>                        <a id="6764075" scrollto-destination="6764075"></a>
<div content-id="DSS015TB2" class="table-modal"><div class="table-wrap"><div id="DSS015TB2" class="table-wrap-title" data-id="DSS015TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Species with ≥100 top hits from B2G</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Hits </th></tr></thead><tbody><tr><td><em>T. guttata</em> </td><td>7820 </td></tr><tr><td><em>G. gallus</em> </td><td>2222 </td></tr><tr><td><em>Homo sapiens</em> </td><td>235 </td></tr><tr><td><em>Monodelphis domestica</em> </td><td>193 </td></tr><tr><td><em>Mus musculus</em> </td><td>187 </td></tr><tr><td><em>Ailuropoda melanoleuca</em> </td><td>177 </td></tr><tr><td><em>Ornithorhynchus anatinus</em> </td><td>149 </td></tr><tr><td><em>Canis familiaris</em> </td><td>119 </td></tr><tr><td><em>M. melodia</em> </td><td>113 </td></tr><tr><td><em>Rattus norvegicus</em> </td><td>100 </td></tr></tbody></table></div><div content-id="DSS015TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Hits </th></tr></thead><tbody><tr><td><em>T. guttata</em> </td><td>7820 </td></tr><tr><td><em>G. gallus</em> </td><td>2222 </td></tr><tr><td><em>Homo sapiens</em> </td><td>235 </td></tr><tr><td><em>Monodelphis domestica</em> </td><td>193 </td></tr><tr><td><em>Mus musculus</em> </td><td>187 </td></tr><tr><td><em>Ailuropoda melanoleuca</em> </td><td>177 </td></tr><tr><td><em>Ornithorhynchus anatinus</em> </td><td>149 </td></tr><tr><td><em>Canis familiaris</em> </td><td>119 </td></tr><tr><td><em>M. melodia</em> </td><td>113 </td></tr><tr><td><em>Rattus norvegicus</em> </td><td>100 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6764075" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS015TB2" class="table-wrap-title" data-id="DSS015TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Species with ≥100 top hits from B2G</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Species </th><th>Hits </th></tr></thead><tbody><tr><td><em>T. guttata</em> </td><td>7820 </td></tr><tr><td><em>G. gallus</em> </td><td>2222 </td></tr><tr><td><em>Homo sapiens</em> </td><td>235 </td></tr><tr><td><em>Monodelphis domestica</em> </td><td>193 </td></tr><tr><td><em>Mus musculus</em> </td><td>187 </td></tr><tr><td><em>Ailuropoda melanoleuca</em> </td><td>177 </td></tr><tr><td><em>Ornithorhynchus anatinus</em> </td><td>149 </td></tr><tr><td><em>Canis familiaris</em> </td><td>119 </td></tr><tr><td><em>M. melodia</em> </td><td>113 </td></tr><tr><td><em>Rattus norvegicus</em> </td><td>100 </td></tr></tbody></table></div><div content-id="DSS015TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Species </th><th>Hits </th></tr></thead><tbody><tr><td><em>T. guttata</em> </td><td>7820 </td></tr><tr><td><em>G. gallus</em> </td><td>2222 </td></tr><tr><td><em>Homo sapiens</em> </td><td>235 </td></tr><tr><td><em>Monodelphis domestica</em> </td><td>193 </td></tr><tr><td><em>Mus musculus</em> </td><td>187 </td></tr><tr><td><em>Ailuropoda melanoleuca</em> </td><td>177 </td></tr><tr><td><em>Ornithorhynchus anatinus</em> </td><td>149 </td></tr><tr><td><em>Canis familiaris</em> </td><td>119 </td></tr><tr><td><em>M. melodia</em> </td><td>113 </td></tr><tr><td><em>Rattus norvegicus</em> </td><td>100 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6764075" target="_blank">View Large</a></div></div><p>In the first GO division, ‘biological process’,<sup><a reveal-id="DSS015C23" data-open="DSS015C23" class="link link-ref link-reveal xref-bibr">23</a></sup> 22 categories were identified. Most contigs (3578 = 53.1%) were involved in ‘cellular and metabolic processes’. The second most abundant category was ‘biological regulation and localization’ (1253 = 18.6%; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>A). Within the second division, ‘molecular function’,<sup><a reveal-id="DSS015C23" data-open="DSS015C23" class="link link-ref link-reveal xref-bibr">23</a></sup> nine major categories were identified. Most of the contigs were functionally related to ‘nucleotide binding’ (1966 = 43.9%) and ‘catalytic activity’ (1266 28.2%; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>B). Finally, the last division, ‘cellular component’,<sup><a reveal-id="DSS015C23" data-open="DSS015C23" class="link link-ref link-reveal xref-bibr">23</a></sup> also had nine categories. Gene products were primarily expressed intracellularly (2322 = 41.9%) or in the membrane bound/non-membrane bound organelle (1787 = 32.3%; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>C).</p><p>All of the GO results should be viewed with caution because the depth of the available sequences ensures that most highly expressed transcripts will have been sequenced but many low-expression transcripts will not have been detected. The normalization techniques used substantially increased the number of low-expression transcripts sequenced, but the number of sequences obtained is insufficient to overcome the bias toward highly expressed transcripts.</p>                            <h3 scrollto-destination=6764081 id="6764081" class="section-title" data-legacy-id=s3c>3.3. Polymorphism detection</h3>
<p>We detected a total of 29 982 SNPs/indels that were spread relatively evenly within, between, and among all three populations (Fig. <a reveal-id="DSS015F2" data-open="DSS015F2" class="link link-reveal link-table xref-fig">2</a>, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Sheet 2</a>). A total of 1402 SNPs/indels were fixed between populations and subspecies (Fig. <a reveal-id="DSS015F3" data-open="DSS015F3" class="link link-reveal link-table xref-fig">3</a>; the sum of all pairwise comparisons is 1635 because some pairwise SNPs are found in more than one pair). Out of the 1402, there were 392 and 410 SNPs/indels between subspecies and within-subspecies, respectively. This provides many SNPs/indels for further study (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Sheet 2</a>), although given our limited sampling of individuals within populations (<em>n</em> = 2) many will not be true fixed differences (i.e. they are false positives, other individuals contain these variants). We also note that we have used quite stringent criteria for SNP/indel assignment. By requiring at least three reads for the minor allele, a minimum of six times coverage is required to call a SNP. Because our average assembly depth is only about four times, most polymorphic nucleotides in our contigs will not pass our criteria for SNP discovery. Because of this, we have biased the SNPs to be from the relatively highly expressed transcripts. Many additional SNPs/indels occur in song sparrows, we describe only those with a high probability of being real, not sequencing artefacts. None of these issues limits our ability to achieve our stated goals, but we note them so that it is understood that we have made appropriately cautious interpretations of our results. </p>                        <a id="6764085" scrollto-destination="6764085"></a>
<div id="DSS015F2" data-id="DSS015F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/m_dss01502.jpeg?Expires=2147483647&Signature=avfQsf8qWFezvlwuQ7ZsAX-nmTHOhEye5UNyJLYOZzPQPSZTAWVlFABf4yuEZ93kjtK0k2Nnn7Bqx6yVqxsaKUTdpaUv316xsKzL6lspkjjR5VN4CHcFZFfOmzbzEsC-Nm63Qm1IMo8Vu6AGtrvvFacAn90N32unVsyuwsMUK8jpXEyhy726Mbw4TObkTvLn2BPkQXIhWBjHNuZLfdpuLNb22zepRtJg5qnoAA3blxuBP0xGH-V6xHSA8kH4shmNFP8pMc3R-hKjY-pph6SaqIzlENPzTPbZJhLOYBj78hgxjxKQA2wZvscUBXX7vVcF0B2n3VhcLwPfFasJQiGwbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Numbers of SNPs and indels that are within and shared between and among three populations of song sparrows." path-from-xml="dss01502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6764085/dss01502.jpeg" path-from-xml="dss01502.jpeg" target="_blank">View large</a><a section="6764085" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/dss01502.jpeg?Expires=2147483647&Signature=Vr2dNI7puupa3e6nTXUZh4sEk4QwrblCJcxUqtPAs9W0qS3hJ1hjqtaC0PjOzDYSSu8qKfXHIuqbtsQw3AMKJDyHd0QbEZsCHTVz6wtNEoTKnCDDNSgQGx43Y8ykQzkbm3-0m1YvHHijDuiBcgEo1XpJ1IMt9MAeAnbWFDoj9QjNxjdz2PEKwYBQOTAMe81~ehqRSn4lLLk9y0vfRNh9korKaIMnb4tpCeKptHOp3lWZeLwORWhitXDG50vBO70J5U4c-QruooEa~O19QGynvqomDIesPtocOCu3LG~6-0LIqsQcrECcAC6xNbGbjlCAfybI-0h8~vT3Gp5kf9P8Ug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6764085&ar=450749&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Numbers of SNPs and indels that are within and shared between and among three populations of song sparrows.</p></div></div><div content-id="DSS015F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/m_dss01502.jpeg?Expires=2147483647&Signature=avfQsf8qWFezvlwuQ7ZsAX-nmTHOhEye5UNyJLYOZzPQPSZTAWVlFABf4yuEZ93kjtK0k2Nnn7Bqx6yVqxsaKUTdpaUv316xsKzL6lspkjjR5VN4CHcFZFfOmzbzEsC-Nm63Qm1IMo8Vu6AGtrvvFacAn90N32unVsyuwsMUK8jpXEyhy726Mbw4TObkTvLn2BPkQXIhWBjHNuZLfdpuLNb22zepRtJg5qnoAA3blxuBP0xGH-V6xHSA8kH4shmNFP8pMc3R-hKjY-pph6SaqIzlENPzTPbZJhLOYBj78hgxjxKQA2wZvscUBXX7vVcF0B2n3VhcLwPfFasJQiGwbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Numbers of SNPs and indels that are within and shared between and among three populations of song sparrows." path-from-xml="dss01502.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6764085/dss01502.jpeg" path-from-xml="dss01502.jpeg" target="_blank">View large</a><a section="6764085" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/dss01502.jpeg?Expires=2147483647&Signature=Vr2dNI7puupa3e6nTXUZh4sEk4QwrblCJcxUqtPAs9W0qS3hJ1hjqtaC0PjOzDYSSu8qKfXHIuqbtsQw3AMKJDyHd0QbEZsCHTVz6wtNEoTKnCDDNSgQGx43Y8ykQzkbm3-0m1YvHHijDuiBcgEo1XpJ1IMt9MAeAnbWFDoj9QjNxjdz2PEKwYBQOTAMe81~ehqRSn4lLLk9y0vfRNh9korKaIMnb4tpCeKptHOp3lWZeLwORWhitXDG50vBO70J5U4c-QruooEa~O19QGynvqomDIesPtocOCu3LG~6-0LIqsQcrECcAC6xNbGbjlCAfybI-0h8~vT3Gp5kf9P8Ug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6764085&ar=450749&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01502.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Numbers of SNPs and indels that are within and shared between and among three populations of song sparrows.</p></div></div>                        <a id="6764090" scrollto-destination="6764090"></a>
<div id="DSS015F3" data-id="DSS015F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/m_dss01503.jpeg?Expires=2147483647&Signature=2nHIFJV22NzVQyiKgv-AtDPx8wJBf2TXtQ1AC3DWtbuDEq36cyOmQ5Q05UdO6vBv9b-J-gsqrig7LugwI4vIGAPESNBxozQwHSpoV4keHTZbQifs1jrRXborTSsYMprdWyxFpZVHQniIjBWaGk-CZYAyiaEc1VNZzyzctDh8M0z5OYjx5jE28XrwzJvs6UgM3CYQ9T6Y5ZEZtLVavkiJnc0rzXvFJqX8jk4CrYZGE9tLhP0lkRdqKDntd-SA-10WuXc-DB8Mh7D7oYmcJXXKMnYGObyaIzxetdJG9tRU0IxO3ReA9Xer3EIUe3cI8PvWB5kzJYpdKSxaf7aw3rEg~w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="SNPs and indels that are fixed between and among three populations of song sparrows. There are 392 SNPs/indels that are identical in Attu and Adak, but different from Cordova. Because sample sizes are small, these figures include false positives." path-from-xml="dss01503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6764090/dss01503.jpeg" path-from-xml="dss01503.jpeg" target="_blank">View large</a><a section="6764090" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/dss01503.jpeg?Expires=2147483647&Signature=az6Q03922MlPp5tLdCkSFGRMVKBZ0LNCDfL1UWr7ck8V3LAhb17EmPJFEzQrXTsKHnbMn5oKYs-qot43X3XarQKAFOGkrqrPFF6krEoSAUyUzg4pvYJ~uQ6aew1YG8T1t-zSnlyxU7YUF61MMnfFF0Ek5aXe9ekPeZiQjAbuk8-tv-HSZwH0ni~Nong5EtKpxqHvwDFT5rexxtkO5As23i-dLQMQlOKOd4xSKGlYjSor2dbFekUBVZccFuKrYe-PtW2ByCvgb121bLKae93~s0lr7w5BT~B3XopJTHg0MsK9OUVK8Dq4pYCdZb4v1lfghj9-tI-rzL9I7Zc1kBgbew__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6764090&ar=450749&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>SNPs and indels that are fixed between and among three populations of song sparrows. There are 392 SNPs/indels that are identical in Attu and Adak, but different from Cordova. Because sample sizes are small, these figures include false positives.</p></div></div><div content-id="DSS015F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/m_dss01503.jpeg?Expires=2147483647&Signature=2nHIFJV22NzVQyiKgv-AtDPx8wJBf2TXtQ1AC3DWtbuDEq36cyOmQ5Q05UdO6vBv9b-J-gsqrig7LugwI4vIGAPESNBxozQwHSpoV4keHTZbQifs1jrRXborTSsYMprdWyxFpZVHQniIjBWaGk-CZYAyiaEc1VNZzyzctDh8M0z5OYjx5jE28XrwzJvs6UgM3CYQ9T6Y5ZEZtLVavkiJnc0rzXvFJqX8jk4CrYZGE9tLhP0lkRdqKDntd-SA-10WuXc-DB8Mh7D7oYmcJXXKMnYGObyaIzxetdJG9tRU0IxO3ReA9Xer3EIUe3cI8PvWB5kzJYpdKSxaf7aw3rEg~w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="SNPs and indels that are fixed between and among three populations of song sparrows. There are 392 SNPs/indels that are identical in Attu and Adak, but different from Cordova. Because sample sizes are small, these figures include false positives." path-from-xml="dss01503.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6764090/dss01503.jpeg" path-from-xml="dss01503.jpeg" target="_blank">View large</a><a section="6764090" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/dss01503.jpeg?Expires=2147483647&Signature=az6Q03922MlPp5tLdCkSFGRMVKBZ0LNCDfL1UWr7ck8V3LAhb17EmPJFEzQrXTsKHnbMn5oKYs-qot43X3XarQKAFOGkrqrPFF6krEoSAUyUzg4pvYJ~uQ6aew1YG8T1t-zSnlyxU7YUF61MMnfFF0Ek5aXe9ekPeZiQjAbuk8-tv-HSZwH0ni~Nong5EtKpxqHvwDFT5rexxtkO5As23i-dLQMQlOKOd4xSKGlYjSor2dbFekUBVZccFuKrYe-PtW2ByCvgb121bLKae93~s0lr7w5BT~B3XopJTHg0MsK9OUVK8Dq4pYCdZb4v1lfghj9-tI-rzL9I7Zc1kBgbew__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6764090&ar=450749&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01503.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>SNPs and indels that are fixed between and among three populations of song sparrows. There are 392 SNPs/indels that are identical in Attu and Adak, but different from Cordova. Because sample sizes are small, these figures include false positives.</p></div></div>                            <h3 scrollto-destination=6764094 id="6764094" class="section-title" data-legacy-id=s3d>3.4. Orthology with zebra finch and chicken</h3>
<p>The reciprocal blast approach identified 4574 contigs as orthologous to both zebra finch and chicken. As expected because of phylogenetic relationships, more contigs were identified as orthologous to the zebra finch than the chicken: the set [unique song sparrow (orthologues) unique zebra finch] was [32 435 (6104) 12 493], whereas the set [unique song sparrow (orthologues) unique chicken] was [32 767 (5772) 16 518]. A substantial number of orthologous contigs (3894) were found to have the same chromosome location in the zebra finch and chicken (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Sheet 1</a>).</p>                            <h3 scrollto-destination=6764099 id="6764099" class="section-title" data-legacy-id=s3e>3.5. Localization of contigs</h3>
<p>The zebra finch and chicken genomes were used as references to locate the contigs. BLAT mapping of our assemblies against these genomes showed sequences that uniquely mapped to particular features of the reference genomes [5′UTR (untranslated region), 3′UTR, CDS (coding sequence), 1 kb upstream, 1 kb downstream; Fig. <a reveal-id="DSS015F4" data-open="DSS015F4" class="link link-reveal link-table xref-fig">4</a>A]. Based on the zebra finch genome annotation, nearly 34% of mapped contigs (2890 of 8561) were found to be in CDS regions. Even with the use of the MINT cDNA construction kit, which is meant only to allow amplification of full-length transcripts, we still observed a substantial bias toward contigs mapping to 3′UTR and 1 kb downstream _relative to 5′UTR and 1 kb upstream. The normalized distributions clearly indicate that our libraries contain relatively few transcripts that are full length (Fig. <a reveal-id="DSS015F4" data-open="DSS015F4" class="link link-reveal link-table xref-fig">4</a>B). Similar patterns, although with slightly fewer hits, were obtained from mapping to the chicken genome. The localization of contigs containing SNPs/indels mapped against the zebra finch and chicken genomes showed that a major proportion of polymorphisms belongs to coding sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>A and B). Contigs with SNPs/indels had more blast hits to the zebra finch than to the chicken, reflecting the overall pattern of all contigs. Few RNA genes were also found by BLAT mapping (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>A and B). </p>                        <a id="6764103" scrollto-destination="6764103"></a>
<div id="DSS015F4" data-id="DSS015F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/m_dss01504.jpeg?Expires=2147483647&Signature=DWgKD3dcUx06DFHCFpWJp5NIbk7C5bDMmWy109wbCAET55k8NtrCjOx48bkDhlDUXw~XKxKcbVWD82bedXvd0bVAzaCmAJXKWq6RkdBc5OI6SrKcyPxub6IBxBzdLnMmZfLUcme1Xd~xXv~9SFTO~IoakwQrYRTvn14ak3-sFdW-WID4DZltoibzgEcQqIBIlQsXjCtbD9mBsDomZXXWhNjIDA~uTenecovPcTXW8p6PvqoKKBndVQtFW79zILiJfWu16O1q2e3keo5Ya3l8ZcxgNvu3zU8Cg8pC91FOZkinVHrthynDHCndCAUqfa~VioUqabKB4m7t-kSgw68FnA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Histogram displaying the proportion of contigs mapped to particular features of protein coding genes of zebra finch and chicken (UTR is the untranslated region, and CDS is the coding sequence). The upper panel displays the raw count and the lower panel normalized values (the proportion discovered relative to how many could be discovered within each category)." path-from-xml="dss01504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6764103/dss01504.jpeg" path-from-xml="dss01504.jpeg" target="_blank">View large</a><a section="6764103" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/dss01504.jpeg?Expires=2147483647&Signature=vE8j50HyZwzjpRQQgb~iAhlvRVbPFklZ5ZrrQ6x5PEWO8OkZxGCiTuykAkSf2TOFfp1BNSPUyYwDUxgsfreiyLzSESl0GLneJljYC42lTMZq~7aWZnIfqCKzNqDRYYRRr0zzMNIZJSM3yvitByasgZdVXzhfNfeBxzd7YOl6TJtqaylwg9JlRy8F5YiGKdHO0HxUYbkrjG3uRLwvloEsqyrTy-o~4wYMJKb8vmxrqhGweqP1knmaWjSRkqNb9PKQsscJfX9GkAkSprnm8ZZqvlQy-QzVKYUR5UADE9bihcFbmXnq-6w2jh1k7Sppl0mDlDmu7kqPEf1-cyqqOWteoQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6764103&ar=450749&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Histogram displaying the proportion of contigs mapped to particular features of protein coding genes of zebra finch and chicken (UTR is the untranslated region, and CDS is the coding sequence). The upper panel displays the raw count and the lower panel normalized values (the proportion discovered relative to how many could be discovered within each category).</p></div></div><div content-id="DSS015F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/m_dss01504.jpeg?Expires=2147483647&Signature=DWgKD3dcUx06DFHCFpWJp5NIbk7C5bDMmWy109wbCAET55k8NtrCjOx48bkDhlDUXw~XKxKcbVWD82bedXvd0bVAzaCmAJXKWq6RkdBc5OI6SrKcyPxub6IBxBzdLnMmZfLUcme1Xd~xXv~9SFTO~IoakwQrYRTvn14ak3-sFdW-WID4DZltoibzgEcQqIBIlQsXjCtbD9mBsDomZXXWhNjIDA~uTenecovPcTXW8p6PvqoKKBndVQtFW79zILiJfWu16O1q2e3keo5Ya3l8ZcxgNvu3zU8Cg8pC91FOZkinVHrthynDHCndCAUqfa~VioUqabKB4m7t-kSgw68FnA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Histogram displaying the proportion of contigs mapped to particular features of protein coding genes of zebra finch and chicken (UTR is the untranslated region, and CDS is the coding sequence). The upper panel displays the raw count and the lower panel normalized values (the proportion discovered relative to how many could be discovered within each category)." path-from-xml="dss01504.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6764103/dss01504.jpeg" path-from-xml="dss01504.jpeg" target="_blank">View large</a><a section="6764103" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/dss01504.jpeg?Expires=2147483647&Signature=vE8j50HyZwzjpRQQgb~iAhlvRVbPFklZ5ZrrQ6x5PEWO8OkZxGCiTuykAkSf2TOFfp1BNSPUyYwDUxgsfreiyLzSESl0GLneJljYC42lTMZq~7aWZnIfqCKzNqDRYYRRr0zzMNIZJSM3yvitByasgZdVXzhfNfeBxzd7YOl6TJtqaylwg9JlRy8F5YiGKdHO0HxUYbkrjG3uRLwvloEsqyrTy-o~4wYMJKb8vmxrqhGweqP1knmaWjSRkqNb9PKQsscJfX9GkAkSprnm8ZZqvlQy-QzVKYUR5UADE9bihcFbmXnq-6w2jh1k7Sppl0mDlDmu7kqPEf1-cyqqOWteoQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6764103&ar=450749&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01504.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Histogram displaying the proportion of contigs mapped to particular features of protein coding genes of zebra finch and chicken (UTR is the untranslated region, and CDS is the coding sequence). The upper panel displays the raw count and the lower panel normalized values (the proportion discovered relative to how many could be discovered within each category).</p></div></div>                            <h3 scrollto-destination=6764108 id="6764108" class="section-title" data-legacy-id=s3f>3.6. Common set transcripts in birds</h3>
<p>We determined the orthologous transcripts with respect to zebra finch using the bidirectional blast best hit method in 12 bird species. From the orthologous sequences, we determined the most common set of transcripts of zebra finch which is present in all species or most of the species. The first big set of transcripts (1004 zebra finch sequences) was present in seven bird species. The second largest set comprised 219 and 126 sequences present in 10 and 12 bird species, respectively, and, finally, 19 sequences were present in all 13 species. Detailed information regarding species used and orthologous sequences is given in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Sheet 3</a>. Further, we checked the pathways in which these common transcripts might be involved using DAVID<sup><a reveal-id="DSS015C24" data-open="DSS015C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSS015C25" data-open="DSS015C25" class="link link-ref link-reveal xref-bibr">25</a></sup> and found that they mainly related to oxidative phosphorylation, ribosome biogenesis, and cardiac muscle contraction. These are housekeeping genes<sup><a reveal-id="DSS015C26" data-open="DSS015C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSS015C27" data-open="DSS015C27" class="link link-ref link-reveal xref-bibr">27</a></sup> which explains the frequent occurrence of these in all avian species. With respect to the chromosomal location of common transcripts, we did not find any significant bias related to any particular chromosome.</p>                            <h3 scrollto-destination=6764113 id="6764113" class="section-title" data-legacy-id=s3g>3.7. Estimation of K<sub><em>a</em></sub>/K<sub><em>s</em></sub></h3>
<p>Substitution rates were estimated for the 4574 contigs orthologous to both zebra finch and chicken. After filtering (based on the length of alignment and removing frame shifts), the number of contigs was reduced to 3821. We excluded contigs that were either identical or which had <em>K</em><sub>s</sub> = 0 (which made <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> incalculable). Thus, <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> was estimated for 3252 (zebra finch) and 3127 (chicken) contigs. Rate estimation with zebra finch identified 43 contigs with <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> ≥1 and 283 with values of 0.5–1.0 (Fig. <a reveal-id="DSS015F5" data-open="DSS015F5" class="link link-reveal link-table xref-fig">5</a>A). Rate estimations with chicken yielded 5 and 58 contigs with <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> ≥1 and between 0.5 and 1.0, respectively (Fig. <a reveal-id="DSS015F5" data-open="DSS015F5" class="link link-reveal link-table xref-fig">5</a>B). Afterwards, assuming the song sparrow contigs have the same chromosome organization as zebra finch and chicken, the calculated ratios were organized into chromosomes (Table <a reveal-id="DSS015TB3" data-open="DSS015TB3" class="link link-reveal link-table xref-fig">3</a>); this is not an unrealistic assumption considering the high degree of chromosomal conservation among avian genomes<sup><a reveal-id="DSS015C28" data-open="DSS015C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSS015C29" data-open="DSS015C29" class="link link-ref link-reveal xref-bibr">29</a></sup> and the fact that such a high proportion (85.1%) of our orthologous contigs was found to have shared chromosomal locations with zebra finch and chicken. </p>                        <a id="6764118" scrollto-destination="6764118"></a>
<div content-id="DSS015TB3" class="table-modal"><div class="table-wrap"><div id="DSS015TB3" class="table-wrap-title" data-id="DSS015TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Number of contgis orthologous to particular zebra finch and chicken chromosomes, and mean <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> ratio for each chromosome, assuming the orthologous contigs have the same chromosomal location as zebra finch and chicken</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chr </th><th>Contigs orthologous to particular zebra finch chromosome </th><th>Total number of transcripts from particular zebra finch chromosome in Biomart file </th><th><em>K</em><sub>a</sub><em>/K</em><sub>s</sub> (mean ± SD) </th><th>Contigs orthologous to particular chicken chromosome </th><th>Total number of transcripts from particular chicken chromosome in Biomart file </th><th><em>K</em><sub>a</sub><em>/K</em><sub>s</sub> (mean ± SD) </th></tr></thead><tbody><tr><td>1 </td><td>261 </td><td>1124 </td><td>0.2552 ± 0.2733 </td><td>492 </td><td>2994 </td><td>0.1528 ± 0.1694 </td></tr><tr><td>2 </td><td>338 </td><td>1345 </td><td>0.2434 ± 0.2465 </td><td>339 </td><td>1995 </td><td>0.1457 ± 0.1326 </td></tr><tr><td>3 </td><td>309 </td><td>1169 </td><td>0.2434 ± 0.2807 </td><td>314 </td><td>1672 </td><td>0.1565 ± 0.1497 </td></tr><tr><td>4 </td><td>188 </td><td>741 </td><td>0.2258 ± 0.3347 </td><td>252 </td><td>1516 </td><td>0.1374 ± 0.1274 </td></tr><tr><td>5 </td><td>229 </td><td>936 </td><td>0.2103 ± 0.2184 </td><td>234 </td><td>1299 </td><td>0.1280 ± 0.1219 </td></tr><tr><td>6 </td><td>107 </td><td>562 </td><td>0.2447 ± 0.2112 </td><td>106 </td><td>781 </td><td>0.1486 ± 0.1187 </td></tr><tr><td>7 </td><td>124 </td><td>521 </td><td>0.2220 ± 0.2103 </td><td>120 </td><td>767 </td><td>0.1361 ± 0.1235 </td></tr><tr><td>8 </td><td>111 </td><td>416 </td><td>0.2581 ± 0.2196 </td><td>127 </td><td>723 </td><td>0.1436 ± 0.1251 </td></tr><tr><td>9 </td><td>90 </td><td>458 </td><td>0.2286 ± 0.3839 </td><td>86 </td><td>598 </td><td>0.1045 ± 0.1087 </td></tr><tr><td>10 </td><td>86 </td><td>394 </td><td>0.1784 ± 0.1738 </td><td>90 </td><td>599 </td><td>0.1220 ± 0.1890 </td></tr><tr><td>11 </td><td>68 </td><td>371 </td><td>0.2330 ± 0.2978 </td><td>61 </td><td>499 </td><td>0.1429 ± 0.1439 </td></tr><tr><td>12 </td><td>73 </td><td>349 </td><td>0.1799 ± 0.2206 </td><td>68 </td><td>427 </td><td>0.1076 ± 0.1122 </td></tr><tr><td>13 </td><td>77 </td><td>321 </td><td>0.1845 ± 0.2319 </td><td>83 </td><td>499 </td><td>0.0994 ± 0.1225 </td></tr><tr><td>14 </td><td>80 </td><td>390 </td><td>0.2541 ± 0.3448 </td><td>79 </td><td>578 </td><td>0.1333 ± 0.1288 </td></tr><tr><td>15 </td><td>76 </td><td>350 </td><td>0.1817 ± 0.2299 </td><td>73 </td><td>531 </td><td>0.0925 ± 0.1207 </td></tr><tr><td>17 </td><td>49 </td><td>300 </td><td>0.1705 ± 0.1597 </td><td>46 </td><td>432 </td><td>0.0967 ± 0.0861 </td></tr><tr><td>18 </td><td>54 </td><td>309 </td><td>0.2230 ± 0.1950 </td><td>55 </td><td>428 </td><td>0.1085 ± 0.0907 </td></tr><tr><td>19 </td><td>68 </td><td>313 </td><td>0.2004 ± 0.2982 </td><td>66 </td><td>443 </td><td>0.0858 ± 0.0952 </td></tr><tr><td>20 </td><td>50 </td><td>329 </td><td>0.2419 ± 0.2444 </td><td>51 </td><td>476 </td><td>0.1336 ± 0.1277 </td></tr><tr><td>21 </td><td>34 </td><td>192 </td><td>0.1470 ± 0.1569 </td><td>44 </td><td>346 </td><td>0.0847 ± 0.1058 </td></tr><tr><td>22 </td><td>16 </td><td>98 </td><td>0.1000 ± 0.0976 </td><td>11 </td><td>160 </td><td>0.0441 ± 0.0593 </td></tr><tr><td>23 </td><td>34 </td><td>205 </td><td>0.1783 ± 0.1828 </td><td>33 </td><td>288 </td><td>0.0782 ± 0.0920 </td></tr><tr><td>24 </td><td>27 </td><td>181 </td><td>0.1961 ± 0.1906 </td><td>24 </td><td>270 </td><td>0.1000 ± 0.0982 </td></tr><tr><td>25 </td><td>7 </td><td>92 </td><td>0.1161 ± 0.1069 </td><td>6 </td><td>169 </td><td>0.0711 ± 0.1017 </td></tr><tr><td>26 </td><td>31 </td><td>176 </td><td>0.1148 ± 0.1081 </td><td>29 </td><td>341 </td><td>0.0824 ± 0.0927 </td></tr><tr><td>27 </td><td>31 </td><td>252 </td><td>0.1471 ± 0.1438 </td><td>28 </td><td>345 </td><td>0.0698 ± 0.0727 </td></tr><tr><td>28 </td><td>27 </td><td>227 </td><td>0.1102 ± 0.1256 </td><td>23 </td><td>284 </td><td>0.0476 ± 0.0414 </td></tr><tr><td>Z </td><td>149 </td><td>745 </td><td>0.2321 ± 0.2293 </td><td>146 </td><td>990 </td><td>0.1381 ± 0.1174 </td></tr></tbody></table></div><div content-id="DSS015TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Chr </th><th>Contigs orthologous to particular zebra finch chromosome </th><th>Total number of transcripts from particular zebra finch chromosome in Biomart file </th><th><em>K</em><sub>a</sub><em>/K</em><sub>s</sub> (mean ± SD) </th><th>Contigs orthologous to particular chicken chromosome </th><th>Total number of transcripts from particular chicken chromosome in Biomart file </th><th><em>K</em><sub>a</sub><em>/K</em><sub>s</sub> (mean ± SD) </th></tr></thead><tbody><tr><td>1 </td><td>261 </td><td>1124 </td><td>0.2552 ± 0.2733 </td><td>492 </td><td>2994 </td><td>0.1528 ± 0.1694 </td></tr><tr><td>2 </td><td>338 </td><td>1345 </td><td>0.2434 ± 0.2465 </td><td>339 </td><td>1995 </td><td>0.1457 ± 0.1326 </td></tr><tr><td>3 </td><td>309 </td><td>1169 </td><td>0.2434 ± 0.2807 </td><td>314 </td><td>1672 </td><td>0.1565 ± 0.1497 </td></tr><tr><td>4 </td><td>188 </td><td>741 </td><td>0.2258 ± 0.3347 </td><td>252 </td><td>1516 </td><td>0.1374 ± 0.1274 </td></tr><tr><td>5 </td><td>229 </td><td>936 </td><td>0.2103 ± 0.2184 </td><td>234 </td><td>1299 </td><td>0.1280 ± 0.1219 </td></tr><tr><td>6 </td><td>107 </td><td>562 </td><td>0.2447 ± 0.2112 </td><td>106 </td><td>781 </td><td>0.1486 ± 0.1187 </td></tr><tr><td>7 </td><td>124 </td><td>521 </td><td>0.2220 ± 0.2103 </td><td>120 </td><td>767 </td><td>0.1361 ± 0.1235 </td></tr><tr><td>8 </td><td>111 </td><td>416 </td><td>0.2581 ± 0.2196 </td><td>127 </td><td>723 </td><td>0.1436 ± 0.1251 </td></tr><tr><td>9 </td><td>90 </td><td>458 </td><td>0.2286 ± 0.3839 </td><td>86 </td><td>598 </td><td>0.1045 ± 0.1087 </td></tr><tr><td>10 </td><td>86 </td><td>394 </td><td>0.1784 ± 0.1738 </td><td>90 </td><td>599 </td><td>0.1220 ± 0.1890 </td></tr><tr><td>11 </td><td>68 </td><td>371 </td><td>0.2330 ± 0.2978 </td><td>61 </td><td>499 </td><td>0.1429 ± 0.1439 </td></tr><tr><td>12 </td><td>73 </td><td>349 </td><td>0.1799 ± 0.2206 </td><td>68 </td><td>427 </td><td>0.1076 ± 0.1122 </td></tr><tr><td>13 </td><td>77 </td><td>321 </td><td>0.1845 ± 0.2319 </td><td>83 </td><td>499 </td><td>0.0994 ± 0.1225 </td></tr><tr><td>14 </td><td>80 </td><td>390 </td><td>0.2541 ± 0.3448 </td><td>79 </td><td>578 </td><td>0.1333 ± 0.1288 </td></tr><tr><td>15 </td><td>76 </td><td>350 </td><td>0.1817 ± 0.2299 </td><td>73 </td><td>531 </td><td>0.0925 ± 0.1207 </td></tr><tr><td>17 </td><td>49 </td><td>300 </td><td>0.1705 ± 0.1597 </td><td>46 </td><td>432 </td><td>0.0967 ± 0.0861 </td></tr><tr><td>18 </td><td>54 </td><td>309 </td><td>0.2230 ± 0.1950 </td><td>55 </td><td>428 </td><td>0.1085 ± 0.0907 </td></tr><tr><td>19 </td><td>68 </td><td>313 </td><td>0.2004 ± 0.2982 </td><td>66 </td><td>443 </td><td>0.0858 ± 0.0952 </td></tr><tr><td>20 </td><td>50 </td><td>329 </td><td>0.2419 ± 0.2444 </td><td>51 </td><td>476 </td><td>0.1336 ± 0.1277 </td></tr><tr><td>21 </td><td>34 </td><td>192 </td><td>0.1470 ± 0.1569 </td><td>44 </td><td>346 </td><td>0.0847 ± 0.1058 </td></tr><tr><td>22 </td><td>16 </td><td>98 </td><td>0.1000 ± 0.0976 </td><td>11 </td><td>160 </td><td>0.0441 ± 0.0593 </td></tr><tr><td>23 </td><td>34 </td><td>205 </td><td>0.1783 ± 0.1828 </td><td>33 </td><td>288 </td><td>0.0782 ± 0.0920 </td></tr><tr><td>24 </td><td>27 </td><td>181 </td><td>0.1961 ± 0.1906 </td><td>24 </td><td>270 </td><td>0.1000 ± 0.0982 </td></tr><tr><td>25 </td><td>7 </td><td>92 </td><td>0.1161 ± 0.1069 </td><td>6 </td><td>169 </td><td>0.0711 ± 0.1017 </td></tr><tr><td>26 </td><td>31 </td><td>176 </td><td>0.1148 ± 0.1081 </td><td>29 </td><td>341 </td><td>0.0824 ± 0.0927 </td></tr><tr><td>27 </td><td>31 </td><td>252 </td><td>0.1471 ± 0.1438 </td><td>28 </td><td>345 </td><td>0.0698 ± 0.0727 </td></tr><tr><td>28 </td><td>27 </td><td>227 </td><td>0.1102 ± 0.1256 </td><td>23 </td><td>284 </td><td>0.0476 ± 0.0414 </td></tr><tr><td>Z </td><td>149 </td><td>745 </td><td>0.2321 ± 0.2293 </td><td>146 </td><td>990 </td><td>0.1381 ± 0.1174 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6764118" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS015TB3" class="table-wrap-title" data-id="DSS015TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Number of contgis orthologous to particular zebra finch and chicken chromosomes, and mean <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> ratio for each chromosome, assuming the orthologous contigs have the same chromosomal location as zebra finch and chicken</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chr </th><th>Contigs orthologous to particular zebra finch chromosome </th><th>Total number of transcripts from particular zebra finch chromosome in Biomart file </th><th><em>K</em><sub>a</sub><em>/K</em><sub>s</sub> (mean ± SD) </th><th>Contigs orthologous to particular chicken chromosome </th><th>Total number of transcripts from particular chicken chromosome in Biomart file </th><th><em>K</em><sub>a</sub><em>/K</em><sub>s</sub> (mean ± SD) </th></tr></thead><tbody><tr><td>1 </td><td>261 </td><td>1124 </td><td>0.2552 ± 0.2733 </td><td>492 </td><td>2994 </td><td>0.1528 ± 0.1694 </td></tr><tr><td>2 </td><td>338 </td><td>1345 </td><td>0.2434 ± 0.2465 </td><td>339 </td><td>1995 </td><td>0.1457 ± 0.1326 </td></tr><tr><td>3 </td><td>309 </td><td>1169 </td><td>0.2434 ± 0.2807 </td><td>314 </td><td>1672 </td><td>0.1565 ± 0.1497 </td></tr><tr><td>4 </td><td>188 </td><td>741 </td><td>0.2258 ± 0.3347 </td><td>252 </td><td>1516 </td><td>0.1374 ± 0.1274 </td></tr><tr><td>5 </td><td>229 </td><td>936 </td><td>0.2103 ± 0.2184 </td><td>234 </td><td>1299 </td><td>0.1280 ± 0.1219 </td></tr><tr><td>6 </td><td>107 </td><td>562 </td><td>0.2447 ± 0.2112 </td><td>106 </td><td>781 </td><td>0.1486 ± 0.1187 </td></tr><tr><td>7 </td><td>124 </td><td>521 </td><td>0.2220 ± 0.2103 </td><td>120 </td><td>767 </td><td>0.1361 ± 0.1235 </td></tr><tr><td>8 </td><td>111 </td><td>416 </td><td>0.2581 ± 0.2196 </td><td>127 </td><td>723 </td><td>0.1436 ± 0.1251 </td></tr><tr><td>9 </td><td>90 </td><td>458 </td><td>0.2286 ± 0.3839 </td><td>86 </td><td>598 </td><td>0.1045 ± 0.1087 </td></tr><tr><td>10 </td><td>86 </td><td>394 </td><td>0.1784 ± 0.1738 </td><td>90 </td><td>599 </td><td>0.1220 ± 0.1890 </td></tr><tr><td>11 </td><td>68 </td><td>371 </td><td>0.2330 ± 0.2978 </td><td>61 </td><td>499 </td><td>0.1429 ± 0.1439 </td></tr><tr><td>12 </td><td>73 </td><td>349 </td><td>0.1799 ± 0.2206 </td><td>68 </td><td>427 </td><td>0.1076 ± 0.1122 </td></tr><tr><td>13 </td><td>77 </td><td>321 </td><td>0.1845 ± 0.2319 </td><td>83 </td><td>499 </td><td>0.0994 ± 0.1225 </td></tr><tr><td>14 </td><td>80 </td><td>390 </td><td>0.2541 ± 0.3448 </td><td>79 </td><td>578 </td><td>0.1333 ± 0.1288 </td></tr><tr><td>15 </td><td>76 </td><td>350 </td><td>0.1817 ± 0.2299 </td><td>73 </td><td>531 </td><td>0.0925 ± 0.1207 </td></tr><tr><td>17 </td><td>49 </td><td>300 </td><td>0.1705 ± 0.1597 </td><td>46 </td><td>432 </td><td>0.0967 ± 0.0861 </td></tr><tr><td>18 </td><td>54 </td><td>309 </td><td>0.2230 ± 0.1950 </td><td>55 </td><td>428 </td><td>0.1085 ± 0.0907 </td></tr><tr><td>19 </td><td>68 </td><td>313 </td><td>0.2004 ± 0.2982 </td><td>66 </td><td>443 </td><td>0.0858 ± 0.0952 </td></tr><tr><td>20 </td><td>50 </td><td>329 </td><td>0.2419 ± 0.2444 </td><td>51 </td><td>476 </td><td>0.1336 ± 0.1277 </td></tr><tr><td>21 </td><td>34 </td><td>192 </td><td>0.1470 ± 0.1569 </td><td>44 </td><td>346 </td><td>0.0847 ± 0.1058 </td></tr><tr><td>22 </td><td>16 </td><td>98 </td><td>0.1000 ± 0.0976 </td><td>11 </td><td>160 </td><td>0.0441 ± 0.0593 </td></tr><tr><td>23 </td><td>34 </td><td>205 </td><td>0.1783 ± 0.1828 </td><td>33 </td><td>288 </td><td>0.0782 ± 0.0920 </td></tr><tr><td>24 </td><td>27 </td><td>181 </td><td>0.1961 ± 0.1906 </td><td>24 </td><td>270 </td><td>0.1000 ± 0.0982 </td></tr><tr><td>25 </td><td>7 </td><td>92 </td><td>0.1161 ± 0.1069 </td><td>6 </td><td>169 </td><td>0.0711 ± 0.1017 </td></tr><tr><td>26 </td><td>31 </td><td>176 </td><td>0.1148 ± 0.1081 </td><td>29 </td><td>341 </td><td>0.0824 ± 0.0927 </td></tr><tr><td>27 </td><td>31 </td><td>252 </td><td>0.1471 ± 0.1438 </td><td>28 </td><td>345 </td><td>0.0698 ± 0.0727 </td></tr><tr><td>28 </td><td>27 </td><td>227 </td><td>0.1102 ± 0.1256 </td><td>23 </td><td>284 </td><td>0.0476 ± 0.0414 </td></tr><tr><td>Z </td><td>149 </td><td>745 </td><td>0.2321 ± 0.2293 </td><td>146 </td><td>990 </td><td>0.1381 ± 0.1174 </td></tr></tbody></table></div><div content-id="DSS015TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Chr </th><th>Contigs orthologous to particular zebra finch chromosome </th><th>Total number of transcripts from particular zebra finch chromosome in Biomart file </th><th><em>K</em><sub>a</sub><em>/K</em><sub>s</sub> (mean ± SD) </th><th>Contigs orthologous to particular chicken chromosome </th><th>Total number of transcripts from particular chicken chromosome in Biomart file </th><th><em>K</em><sub>a</sub><em>/K</em><sub>s</sub> (mean ± SD) </th></tr></thead><tbody><tr><td>1 </td><td>261 </td><td>1124 </td><td>0.2552 ± 0.2733 </td><td>492 </td><td>2994 </td><td>0.1528 ± 0.1694 </td></tr><tr><td>2 </td><td>338 </td><td>1345 </td><td>0.2434 ± 0.2465 </td><td>339 </td><td>1995 </td><td>0.1457 ± 0.1326 </td></tr><tr><td>3 </td><td>309 </td><td>1169 </td><td>0.2434 ± 0.2807 </td><td>314 </td><td>1672 </td><td>0.1565 ± 0.1497 </td></tr><tr><td>4 </td><td>188 </td><td>741 </td><td>0.2258 ± 0.3347 </td><td>252 </td><td>1516 </td><td>0.1374 ± 0.1274 </td></tr><tr><td>5 </td><td>229 </td><td>936 </td><td>0.2103 ± 0.2184 </td><td>234 </td><td>1299 </td><td>0.1280 ± 0.1219 </td></tr><tr><td>6 </td><td>107 </td><td>562 </td><td>0.2447 ± 0.2112 </td><td>106 </td><td>781 </td><td>0.1486 ± 0.1187 </td></tr><tr><td>7 </td><td>124 </td><td>521 </td><td>0.2220 ± 0.2103 </td><td>120 </td><td>767 </td><td>0.1361 ± 0.1235 </td></tr><tr><td>8 </td><td>111 </td><td>416 </td><td>0.2581 ± 0.2196 </td><td>127 </td><td>723 </td><td>0.1436 ± 0.1251 </td></tr><tr><td>9 </td><td>90 </td><td>458 </td><td>0.2286 ± 0.3839 </td><td>86 </td><td>598 </td><td>0.1045 ± 0.1087 </td></tr><tr><td>10 </td><td>86 </td><td>394 </td><td>0.1784 ± 0.1738 </td><td>90 </td><td>599 </td><td>0.1220 ± 0.1890 </td></tr><tr><td>11 </td><td>68 </td><td>371 </td><td>0.2330 ± 0.2978 </td><td>61 </td><td>499 </td><td>0.1429 ± 0.1439 </td></tr><tr><td>12 </td><td>73 </td><td>349 </td><td>0.1799 ± 0.2206 </td><td>68 </td><td>427 </td><td>0.1076 ± 0.1122 </td></tr><tr><td>13 </td><td>77 </td><td>321 </td><td>0.1845 ± 0.2319 </td><td>83 </td><td>499 </td><td>0.0994 ± 0.1225 </td></tr><tr><td>14 </td><td>80 </td><td>390 </td><td>0.2541 ± 0.3448 </td><td>79 </td><td>578 </td><td>0.1333 ± 0.1288 </td></tr><tr><td>15 </td><td>76 </td><td>350 </td><td>0.1817 ± 0.2299 </td><td>73 </td><td>531 </td><td>0.0925 ± 0.1207 </td></tr><tr><td>17 </td><td>49 </td><td>300 </td><td>0.1705 ± 0.1597 </td><td>46 </td><td>432 </td><td>0.0967 ± 0.0861 </td></tr><tr><td>18 </td><td>54 </td><td>309 </td><td>0.2230 ± 0.1950 </td><td>55 </td><td>428 </td><td>0.1085 ± 0.0907 </td></tr><tr><td>19 </td><td>68 </td><td>313 </td><td>0.2004 ± 0.2982 </td><td>66 </td><td>443 </td><td>0.0858 ± 0.0952 </td></tr><tr><td>20 </td><td>50 </td><td>329 </td><td>0.2419 ± 0.2444 </td><td>51 </td><td>476 </td><td>0.1336 ± 0.1277 </td></tr><tr><td>21 </td><td>34 </td><td>192 </td><td>0.1470 ± 0.1569 </td><td>44 </td><td>346 </td><td>0.0847 ± 0.1058 </td></tr><tr><td>22 </td><td>16 </td><td>98 </td><td>0.1000 ± 0.0976 </td><td>11 </td><td>160 </td><td>0.0441 ± 0.0593 </td></tr><tr><td>23 </td><td>34 </td><td>205 </td><td>0.1783 ± 0.1828 </td><td>33 </td><td>288 </td><td>0.0782 ± 0.0920 </td></tr><tr><td>24 </td><td>27 </td><td>181 </td><td>0.1961 ± 0.1906 </td><td>24 </td><td>270 </td><td>0.1000 ± 0.0982 </td></tr><tr><td>25 </td><td>7 </td><td>92 </td><td>0.1161 ± 0.1069 </td><td>6 </td><td>169 </td><td>0.0711 ± 0.1017 </td></tr><tr><td>26 </td><td>31 </td><td>176 </td><td>0.1148 ± 0.1081 </td><td>29 </td><td>341 </td><td>0.0824 ± 0.0927 </td></tr><tr><td>27 </td><td>31 </td><td>252 </td><td>0.1471 ± 0.1438 </td><td>28 </td><td>345 </td><td>0.0698 ± 0.0727 </td></tr><tr><td>28 </td><td>27 </td><td>227 </td><td>0.1102 ± 0.1256 </td><td>23 </td><td>284 </td><td>0.0476 ± 0.0414 </td></tr><tr><td>Z </td><td>149 </td><td>745 </td><td>0.2321 ± 0.2293 </td><td>146 </td><td>990 </td><td>0.1381 ± 0.1174 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6764118" target="_blank">View Large</a></div></div>                        <a id="6764120" scrollto-destination="6764120"></a>
<div id="DSS015F5" data-id="DSS015F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/m_dss01505.jpeg?Expires=2147483647&Signature=AzxeHVNWO-Ll2nPo6SpcUc9RynBZJrL2US2gI7ZRg3GJZsJwmRPwKmDIE2G6tS~BFM86RLKn86lYqzTXjg2hvpxfSm0yLhK7o5Kixh41kddk6xrlXxnJF6rHnn3FOK0erhg8E~pP73rRgH8-tv-yFoGM8sX3q6GR4wkEXg8kqt~jD~kca-V-JG4M8WcYx8vwdKI7OTGfhO~lIlCHHMlzotRr3D1tUdG6NB2jlRSZTbwi6jNA9sGW76eQdHbZh6f5yuxlxkM1WTKK6-yG2g~xvi-E4Tm8GIcG5lsU9pcK8to3SbQP7VX2LoU3hnkGUv9X3CcrRS~5d6cA547tZBfzhw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of Ka/Ks ratio for the contigs orthologous to both zebra finch (A) and chicken (B). Contigs with Ka/Ks values of 0.5–1.0 fall above the grey line and values &gt;1.0 fall above the black line." path-from-xml="dss01505.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6764120/dss01505.jpeg" path-from-xml="dss01505.jpeg" target="_blank">View large</a><a section="6764120" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/dss01505.jpeg?Expires=2147483647&Signature=yXTN5Rve3fOhERClG44iyKQJMHHO~j~hgw8Qyj5DtqrIG3B9Mrm0pniGKQj2V1-HQwuG4mGbk4tRtqWeRt34uGevIf-IKvUopZNpdz1QX4uxEI4d1yXzFrPwWoUmlIpnPgZjpO~Y1ecUrX1Y1kgHgaMbhfNVA4HKe48sEJbuT3L2jb-re15Ty-hWgF8lvQr9UOSaJNURBug6nQqQLlOiM5F2bTXgikvtrX2GMpwpF25MvzOAurPwydQnKLdzt5ywOuMcysmk-VfpSyyoGPfpeLw3no3opYmFuRHvjgvzydEqt3A64D2QpcDS6XElbrAYH5-7JAZJAtdMdAUmLq-Zng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6764120&ar=450749&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01505.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution of <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> ratio for the contigs orthologous to both zebra finch (A) and chicken (B). Contigs with <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> values of 0.5–1.0 fall above the grey line and values &gt;1.0 fall above the black line.</p></div></div><div content-id="DSS015F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/m_dss01505.jpeg?Expires=2147483647&Signature=AzxeHVNWO-Ll2nPo6SpcUc9RynBZJrL2US2gI7ZRg3GJZsJwmRPwKmDIE2G6tS~BFM86RLKn86lYqzTXjg2hvpxfSm0yLhK7o5Kixh41kddk6xrlXxnJF6rHnn3FOK0erhg8E~pP73rRgH8-tv-yFoGM8sX3q6GR4wkEXg8kqt~jD~kca-V-JG4M8WcYx8vwdKI7OTGfhO~lIlCHHMlzotRr3D1tUdG6NB2jlRSZTbwi6jNA9sGW76eQdHbZh6f5yuxlxkM1WTKK6-yG2g~xvi-E4Tm8GIcG5lsU9pcK8to3SbQP7VX2LoU3hnkGUv9X3CcrRS~5d6cA547tZBfzhw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of Ka/Ks ratio for the contigs orthologous to both zebra finch (A) and chicken (B). Contigs with Ka/Ks values of 0.5–1.0 fall above the grey line and values &gt;1.0 fall above the black line." path-from-xml="dss01505.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6764120/dss01505.jpeg" path-from-xml="dss01505.jpeg" target="_blank">View large</a><a section="6764120" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/dss01505.jpeg?Expires=2147483647&Signature=yXTN5Rve3fOhERClG44iyKQJMHHO~j~hgw8Qyj5DtqrIG3B9Mrm0pniGKQj2V1-HQwuG4mGbk4tRtqWeRt34uGevIf-IKvUopZNpdz1QX4uxEI4d1yXzFrPwWoUmlIpnPgZjpO~Y1ecUrX1Y1kgHgaMbhfNVA4HKe48sEJbuT3L2jb-re15Ty-hWgF8lvQr9UOSaJNURBug6nQqQLlOiM5F2bTXgikvtrX2GMpwpF25MvzOAurPwydQnKLdzt5ywOuMcysmk-VfpSyyoGPfpeLw3no3opYmFuRHvjgvzydEqt3A64D2QpcDS6XElbrAYH5-7JAZJAtdMdAUmLq-Zng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6764120&ar=450749&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss01505.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution of <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> ratio for the contigs orthologous to both zebra finch (A) and chicken (B). Contigs with <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> values of 0.5–1.0 fall above the grey line and values &gt;1.0 fall above the black line.</p></div></div><p>Although <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> (sometimes calculated as <em>d</em><sub>N</sub><em>/d</em><sub>S</sub> or <em>ω</em>) is commonly misinterpreted,<sup><a reveal-id="DSS015C30" data-open="DSS015C30" class="link link-ref link-reveal xref-bibr">30</a></sup> this ratio of rates of non-synonymous to synonymous substitutions can give some context to candidate genes and allows for subsequent hypothesis testing.<sup><a reveal-id="DSS015C31" data-open="DSS015C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSS015C32" data-open="DSS015C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Data organized into chromosomes suggest that contigs may have undergone more selection with respect to the zebra finch than the chicken (as high <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> values are typically interpreted, though see ref. <a reveal-id="DSS015C30" data-open="DSS015C30" class="link link-ref link-reveal xref-bibr">30</a>).</p><p>The fact that <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> values were higher on average for the zebra finch than for the chicken (Table <a reveal-id="DSS015TB3" data-open="DSS015TB3" class="link link-reveal link-table xref-fig">3</a>) is likely a methodological artefact. The zebra finch is in the same taxonomic order as the song sparrow (Passeriformes), whereas the chicken is taxonomically distant (Galliformes). Estimates of <em>ω</em> necessarily classify sites with differences as non-synonymous or synonymous, and errors in the estimation of either can profoundly affect the outcome of these analyses.<sup><a reveal-id="DSS015C33" data-open="DSS015C33" class="link link-ref link-reveal xref-bibr">33</a></sup> Taxonomic or lineage distance (longer branches) will affect the reconstruction of synonymous substitution rates especially (through an expected increase in repeated mutations, or multiple hits), and we consider this to be a likely source of the consistent differences in apparent molecular selection between our song-sparrow-to-zebra-finch and song-sparrow-to-chicken contrasts (Table <a reveal-id="DSS015TB3" data-open="DSS015TB3" class="link link-reveal link-table xref-fig">3</a>; see also ref. <a reveal-id="DSS015C34" data-open="DSS015C34" class="link link-ref link-reveal xref-bibr">34</a>). Nevertheless, these contrasts are valuable in highlighting the chromosomal distributions (assuming chromosomal stability<sup><a reveal-id="DSS015C28" data-open="DSS015C28" class="link link-ref link-reveal xref-bibr">28</a></sup>) and relative values of <em>ω</em> between closer and more distant relatives of the song sparrow, providing insights into attributes of selection in the coding genome across these scales. Unfortunately, this approach is not valid within species.<sup><a reveal-id="DSS015C35" data-open="DSS015C35" class="link link-ref link-reveal xref-bibr">35</a>–<a reveal-id="DSS015C37" data-open="DSS015C37" class="link link-ref link-reveal xref-bibr">37</a></sup></p><p>Chromosomes 22 and 26 showed the greatest differences between the zebra finch and the chicken in the percentage of song sparrow contigs mapped (relative to the number of genes available in the Biomart database for the zebra finch and chicken). Both of these chromosomes had significantly different frequencies of mapped-song-sparrow versus Biomart data-available genes between the zebra finch and the chicken (<em>G</em><sub>adj</sub> = 4.4, <em>P</em>&lt; 0.05, and <em>G</em><sub>adj</sub> = 6.9, <em>P</em>&lt; 0.01, respectively at 1 d.f., <em>G</em>-test with Williams' correction; Table <a reveal-id="DSS015TB3" data-open="DSS015TB3" class="link link-reveal link-table xref-fig">3</a>). In both cases, proportionally more contigs were mapped to the zebra finch than to the chicken given the sizes of the respective databases (Table <a reveal-id="DSS015TB3" data-open="DSS015TB3" class="link link-reveal link-table xref-fig">3</a>).</p>                            <h3 scrollto-destination=6764131 id="6764131" class="section-title" data-legacy-id=s3h>3.8. Chromosomal distributions of between-subspecies SNPs/indels</h3>
<p>Two findings emerged in comparing the among-chromosome locations (mapped against the zebra finch) of the between-subspecies SNPs/indels that were mapped to chromosomes (218 SNP/indel-bearing, between-subspecies song sparrow contigs; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Sheet 2</a>) versus all orthologous song sparrow contigs (Table <a reveal-id="DSS015TB3" data-open="DSS015TB3" class="link link-reveal link-table xref-fig">3</a>). First, the chromosomal distribution of the candidate loci was significantly different from the distribution of all orthologous contigs (<em>G</em><sub>adj</sub> = 51.5, 27 d.f., <em>P</em>&lt; 0.005), indicative of a non-random process (e.g. selection). Importantly, the chromosomal distribution of the 199 unique, mappable SNP/indel-bearing contigs between Attu and Adak islands (within the subspecies <em>maxima</em>), where we expected drift rather than selection to be more pronounced, was not significantly different from the chromosomal distribution of all orthologous contigs (<em>G</em><sub>adj</sub> = 35.1, 27 d.f., <em>P</em>&gt; 0.1). Secondly, the greatest differences in the distribution of between-subspecies candidate loci from the distribution of all contigs occurred among chromosomes 2, 5, and Z (where proportionally fewer SNP/indel-bearing contigs occurred than expected) and chromosomes 3 and 11 (where relatively more SNP/indel-bearing contigs occurred than expected).</p><p>Finally, in contrasting our between-subspecies results with those of our between-species comparisons above, we found that seven of the SNP/indel-bearing contigs between subspecies were also contigs that exhibited evidence suggestive of selection (high <em>K</em><sub>a</sub><em>/K</em><sub>s</sub> values) when compared with the zebra finch and the chicken. Each contig has one between-subspecies SNP, and the functions of these loci are variable (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Sheet 4</a>). Three of these seven occurred on chromosome 3 and one on chromosome 11, where the between-subspecies contrasts suggested elevated levels of SNPs/indels. These contigs and their chromosomal locations may thus be important in songbird divergence, but we do not yet know why.</p>                            <h3 scrollto-destination=6764138 id="6764138" class="section-title" data-legacy-id=s3i>3.9. Summary</h3>
<p>In summary, our analysis identified the major categories of song sparrow genes and orthologous loci between song sparrow/zebra finch and song sparrow/chicken. Substitution rate estimation yielded the fastest evolving loci, and some of the loci that were fixed between subspecies were also highlighted as possibly under selection between the song sparrow and the zebra finch. Although additional sequencing of these libraries and validation of within-species SNPs/indels in multiple populations and lineages is required, we consider that the loci described here will include some of broad utility for studying the genomics of songbird divergence.</p>                            <h2 scrollto-destination=6764143 id="6764143" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6764147 id="6764147" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This study was supported in part by resources and technical expertise from the University of Georgia, Georgia Advanced Computing Resource Center, a partnership between the Office of the Vice President for Research and the Office of the Chief Information Office.</p>                            <h2 scrollto-destination=6764151 id="6764151" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank NSF Alaska EPSCoR (EPS-0701898) and the University of Alaska Museum for supporting this research. Jack Withrow assisted in fieldwork, Roger Nilsen made the normalized cDNA libraries, Jeff Wagner sequenced the libraries, and Christin Pruett and Erik Postma provided helpful comments. We are also grateful for support from the UGA ARCS foundation.</p>                            <h2 scrollto-destination=6764155 id="6764155" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSS015C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nadeau</div> <div class="given-names">N.J.</div></div>,  <div class="name"><div class="surname">Jiggins</div> <div class="given-names">C.D.</div></div>. </div><div class="article-title">A golden age for evolutionary genetics? Genomic studies of adaptation in natural populations</div>, <div class="source">Trends Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">484</div>-<div class="lpage">92</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20golden%20age%20for%20evolutionary%20genetics%3F%20Genomic%20studies%20of%20adaptation%20in%20natural%20populations&amp;author=N.J.%20Nadeau&amp;author=C.D.%20Jiggins&amp;publication_year=2010&amp;journal=Trends%20Genet.&amp;volume=26&amp;pages=484-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2010.08.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2010.08.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2010.08.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20851493" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stapley</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Reger</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Feulner</div> <div class="given-names">P.G.</div></div>, et al. </div><div class="article-title">Adaptation genomics: the next generation</div>, <div class="source">Trends Ecol. Evol.</div> , <div class="year">2010</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">705</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adaptation%20genomics%3A%20the%20next%20generation&amp;author=J.%20Stapley&amp;author=J.%20Reger&amp;author=P.G.%20Feulner&amp;publication_year=2010&amp;journal=Trends%20Ecol.%20Evol.&amp;volume=25&amp;pages=705-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tree.2010.09.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tree.2010.09.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tree.2010.09.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20952088" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bouck</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Vision</div> <div class="given-names">T.</div></div>. </div><div class="article-title">The molecular ecologist's guide to expressed sequence tags</div>, <div class="source">Mol. Ecol.</div> , <div class="year">2007</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">907</div>-<div class="lpage">24</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20molecular%20ecologist's%20guide%20to%20expressed%20sequence%20tags&amp;author=A.%20Bouck&amp;author=T.%20Vision&amp;publication_year=2007&amp;journal=Mol.%20Ecol.&amp;volume=16&amp;pages=907-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-294X.2006.03195.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-294X.2006.03195.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-294X.2006.03195.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17305850" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wheat</div> <div class="given-names">C.W.</div></div>. </div><div class="article-title">Rapidly developing functional genomics in ecological model systems via 454 transcriptome sequencing</div>, <div class="source">Genetica</div> , <div class="year">2010</div>, vol. <div class="volume">138</div> (pg. <div class="fpage">433</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapidly%20developing%20functional%20genomics%20in%20ecological%20model%20systems%20via%20454%20transcriptome%20sequencing&amp;author=C.W.%20Wheat&amp;publication_year=2010&amp;journal=Genetica&amp;volume=138&amp;pages=433-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10709-008-9326-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10709-008-9326-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10709-008-9326-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18931921" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kunstner</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Wolf</div> <div class="given-names">J.B.</div></div>,  <div class="name"><div class="surname">Backstrom</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Comparative genomics based on massive parallel transcriptome sequencing reveals patterns of substitution and selection across 10 bird species</div>, <div class="source">Mol. Ecol.</div> , <div class="year">2010</div>, vol. <div class="volume">19</div> <div class="issue">Suppl. 1)</div>(pg. <div class="fpage">266</div>-<div class="lpage">76</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomics%20based%20on%20massive%20parallel%20transcriptome%20sequencing%20reveals%20patterns%20of%20substitution%20and%20selection%20across%2010%20bird%20species&amp;author=A.%20Kunstner&amp;author=J.B.%20Wolf&amp;author=N.%20Backstrom&amp;publication_year=2010&amp;journal=Mol.%20Ecol.&amp;volume=19&amp;pages=266-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.2010.19.issue-s1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.2010.19.issue-s1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.2010.19.issue-s1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20331785" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parchman</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Geist</div> <div class="given-names">K.S.</div></div>,  <div class="name"><div class="surname">Grahnen</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Benkman</div> <div class="given-names">C.W.</div></div>,  <div class="name"><div class="surname">Buerkle</div> <div class="given-names">C.A.</div></div>. </div><div class="article-title">Transcriptome sequencing in an ecologically important tree species: assembly, annotation, and marker discovery</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">180</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20sequencing%20in%20an%20ecologically%20important%20tree%20species%3A%20assembly%2C%20annotation%2C%20and%20marker%20discovery&amp;author=T.L.%20Parchman&amp;author=K.S.%20Geist&amp;author=J.A.%20Grahnen&amp;author=C.W.%20Benkman&amp;author=C.A.%20Buerkle&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=180" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-180" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-180" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-180"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20233449" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Santure</div> <div class="given-names">A.W.</div></div>,  <div class="name"><div class="surname">Gratten</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Mossman</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Sheldon</div> <div class="given-names">B.C.</div></div>,  <div class="name"><div class="surname">Slate</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Characterisation of the transcriptome of a wild great tit <em>Parus major</em> population by next generation sequencing</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">283</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterisation%20of%20the%20transcriptome%20of%20a%20wild%20great%20tit%20Parus%20major%20population%20by%20next%20generation%20sequencing&amp;author=A.W.%20Santure&amp;author=J.%20Gratten&amp;author=J.A.%20Mossman&amp;author=B.C.%20Sheldon&amp;author=J.%20Slate&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=283" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-283" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-283" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-283"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21635727" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wolf</div> <div class="given-names">J.B.</div></div>,  <div class="name"><div class="surname">Bayer</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Haubold</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Schilhabel</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Rosenstiel</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Tautz</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Nucleotide divergence vs. gene expression differentiation: comparative transcriptome sequencing in natural isolates from the carrion crow and its hybrid zone with the hooded crow</div>, <div class="source">Mol. Ecol.</div> , <div class="year">2010</div>, vol. <div class="volume">19</div> <div class="issue">Suppl. 1)</div>(pg. <div class="fpage">162</div>-<div class="lpage">75</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleotide%20divergence%20vs.%20gene%20expression%20differentiation%3A%20comparative%20transcriptome%20sequencing%20in%20natural%20isolates%20from%20the%20carrion%20crow%20and%20its%20hybrid%20zone%20with%20the%20hooded%20crow&amp;author=J.B.%20Wolf&amp;author=T.%20Bayer&amp;author=B.%20Haubold&amp;author=M.%20Schilhabel&amp;author=P.%20Rosenstiel&amp;author=D.%20Tautz&amp;publication_year=2010&amp;journal=Mol.%20Ecol.&amp;volume=19&amp;pages=162-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.2010.19.issue-s1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.2010.19.issue-s1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.2010.19.issue-s1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20331778" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Patten</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Pruett</div> <div class="given-names">C.L.</div></div>. </div><div class="article-title">The song sparrow, <em>Melospiza melodia</em>, as a ring species: patterns of geographic variation, a revision of subspecies, and implications for speciation</div>, <div class="source">Syst. Biodivers.</div> , <div class="year">2009</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">33</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20song%20sparrow%2C%20Melospiza%20melodia%2C%20as%20a%20ring%20species%3A%20patterns%20of%20geographic%20variation%2C%20a%20revision%20of%20subspecies%2C%20and%20implications%20for%20speciation&amp;author=M.A.%20Patten&amp;author=C.L.%20Pruett&amp;publication_year=2009&amp;journal=Syst.%20Biodivers.&amp;volume=7&amp;pages=33-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S1477200008002867" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS1477200008002867" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS1477200008002867"> </span></div> </div></div></div></div></div><div content-id="DSS015C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Arcese</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Sogge</div> <div class="given-names">M.K.</div></div>,  <div class="name"><div class="surname">Marr</div> <div class="given-names">A.B.</div></div>,  <div class="name"><div class="surname">Patten</div> <div class="given-names">M.A.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Poole</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Gill</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Song sparrow (<em>Melospiza melodia</em>)</div>, <div class="source">The Birds of North America</div> , <div class="year">2002</div><div class="publisher-loc">Philadelphia, PA</div><div class="publisher-name">The Birds of North America, Inc.</div> <div class="comment">No. 704</div></div></div></div></div><div content-id="DSS015C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Beldade</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Rudd</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gruber</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Long</div> <div class="given-names">A.D.</div></div>. </div><div class="article-title">A wing expressed sequence tag resource for <em>Bicyclus anynana</em> butterflies, an evo-devo model</div>, <div class="source">BMC Genomics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">130</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20wing%20expressed%20sequence%20tag%20resource%20for%20Bicyclus%20anynana%20butterflies%2C%20an%20evo-devo%20model&amp;author=P.%20Beldade&amp;author=S.%20Rudd&amp;author=J.D.%20Gruber&amp;author=A.D.%20Long&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=130" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-130" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-130" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-130"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16737530" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Quinlan</div> <div class="given-names">A.R.</div></div>,  <div class="name"><div class="surname">Stewart</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Stromberg</div> <div class="given-names">M.P.</div></div>,  <div class="name"><div class="surname">Marth</div> <div class="given-names">G.T.</div></div>. </div><div class="article-title">Pyrobayes: an improved base caller for SNP discovery in pyrosequences</div>, <div class="source">Nat. Methods</div> , <div class="year">2008</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">179</div>-<div class="lpage">81</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pyrobayes%3A%20an%20improved%20base%20caller%20for%20SNP%20discovery%20in%20pyrosequences&amp;author=A.R.%20Quinlan&amp;author=D.A.%20Stewart&amp;author=M.P.%20Stromberg&amp;author=G.T.%20Marth&amp;publication_year=2008&amp;journal=Nat.%20Methods&amp;volume=5&amp;pages=179-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1172" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1172" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1172"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18193056" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chevreux</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Pfisterer</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Drescher</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Using the miraEST assembler for reliable and automated mRNA transcript assembly and SNP detection in sequenced ESTs</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">1147</div>-<div class="lpage">59</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20the%20miraEST%20assembler%20for%20reliable%20and%20automated%20mRNA%20transcript%20assembly%20and%20SNP%20detection%20in%20sequenced%20ESTs&amp;author=B.%20Chevreux&amp;author=T.%20Pfisterer&amp;author=B.%20Drescher&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=1147-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1917404" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1917404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1917404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15140833" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marth</div> <div class="given-names">G.T.</div></div>,  <div class="name"><div class="surname">Korf</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Yandell</div> <div class="given-names">M.D.</div></div>, et al. </div><div class="article-title">A general approach to single-nucleotide polymorphism discovery</div>, <div class="source">Nat. Genet.</div> , <div class="year">1999</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">452</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20general%20approach%20to%20single-nucleotide%20polymorphism%20discovery&amp;author=G.T.%20Marth&amp;author=I.%20Korf&amp;author=M.D.%20Yandell&amp;publication_year=1999&amp;journal=Nat.%20Genet.&amp;volume=23&amp;pages=452-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/70570" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F70570" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F70570"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10581034" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altenhoff</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Dessimoz</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Phylogenetic and functional assessment of orthologs inference projects and methods</div>, <div class="source">Plos Comput. Biol</div> , <div class="year">2009</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e1000262</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20and%20functional%20assessment%20of%20orthologs%20inference%20projects%20and%20methods&amp;author=A.M.%20Altenhoff&amp;author=C.%20Dessimoz&amp;publication_year=2009&amp;journal=Plos%20Comput.%20Biol&amp;volume=5&amp;pages=e1000262" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000262" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1000262" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1000262"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19148271" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kent</div> <div class="given-names">W.J.</div></div>. </div><div class="article-title">BLAT–the BLAST-like alignment tool</div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">656</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLAT%E2%80%93the%20BLAST-like%20alignment%20tool&amp;author=W.J.%20Kent&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=656-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.229202. Article published online before March 2002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.229202. Article published online before March 2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.229202. Article published online before March 2002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11932250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Subramanian</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Huynen</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Millar</div> <div class="given-names">C.D.</div></div>,  <div class="name"><div class="surname">Lambert</div> <div class="given-names">D.M.</div></div>. </div><div class="article-title">Next generation sequencing and analysis of a conserved transcriptome of New Zealand's kiwi</div>, <div class="source">BMC Evol. Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">10</div> pg. <div class="fpage">387</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next%20generation%20sequencing%20and%20analysis%20of%20a%20conserved%20transcriptome%20of%20New%20Zealand's%20kiwi&amp;author=S.%20Subramanian&amp;author=L.%20Huynen&amp;author=C.D.%20Millar&amp;author=D.M.%20Lambert&amp;publication_year=2010&amp;journal=BMC%20Evol.%20Biol.&amp;volume=10&amp;pages=387" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-10-387" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-10-387" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-10-387"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21156082" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Conesa</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Götz</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Garcia-Gomez</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Terol</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Talon</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Robles</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">3674</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Blast2GO%3A%20a%20universal%20tool%20for%20annotation%2C%20visualization%20and%20analysis%20in%20functional%20genomics%20research&amp;author=A.%20Conesa&amp;author=S.%20G%C3%B6tz&amp;author=J.M.%20Garcia-Gomez&amp;author=J.%20Terol&amp;author=M.%20Talon&amp;author=M.%20Robles&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=3674-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti610" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti610" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti610"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16081474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Schaffer</div> <div class="given-names">A.A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Higgins</div> <div class="given-names">D.G.</div></div>,  <div class="name"><div class="surname">Larkin</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Blackshields</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">Clustal W and clustal X version 2.0</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">2947</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Clustal%20W%20and%20clustal%20X%20version%202.0&amp;author=D.G.%20Higgins&amp;author=M.A.%20Larkin&amp;author=G.%20Blackshields&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2947-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm404" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17846036" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suyama</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Torrents</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Bork</div> <div class="given-names">P.</div></div>. </div><div class="article-title">PAL2NAL: robust conversion of protein sequence alignments into the corresponding codon alignments</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">W609</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PAL2NAL%3A%20robust%20conversion%20of%20protein%20sequence%20alignments%20into%20the%20corresponding%20codon%20alignments&amp;author=M.%20Suyama&amp;author=D.%20Torrents&amp;author=P.%20Bork&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=W609-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl315" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl315" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl315"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845082" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">PAML 4: phylogenetic analysis by maximum likelihood</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">1586</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PAML%204%3A%20phylogenetic%20analysis%20by%20maximum%20likelihood&amp;author=Z.%20Yang&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=1586-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm088" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm088" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm088"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17483113" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="collab">The Gene Ontology Consortium</div><div class="article-title">Gene Ontology: tool for the unification of biology</div>, <div class="source">Nat. Genet.</div> , <div class="year">2000</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">25</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/75556" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F75556" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F75556"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10802651" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">D.W.</div></div>,  <div class="name"><div class="surname">Sherman</div> <div class="given-names">B.T.</div></div>,  <div class="name"><div class="surname">Lempicki</div> <div class="given-names">R.A.</div></div>. </div><div class="article-title">Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources</div>, <div class="source">Nat. Protoc.</div> , <div class="year">2009</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">44</div>-<div class="lpage">57</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20and%20integrative%20analysis%20of%20large%20gene%20lists%20using%20DAVID%20bioinformatics%20resources&amp;author=D.W.%20Huang&amp;author=B.T.%20Sherman&amp;author=R.A.%20Lempicki&amp;publication_year=2009&amp;journal=Nat.%20Protoc.&amp;volume=4&amp;pages=44-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2008.211" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2008.211" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2008.211"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19131956" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lempicki</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">D.W.</div></div>,  <div class="name"><div class="surname">Sherman</div> <div class="given-names">B.T.</div></div>. </div><div class="article-title">Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">1</div>-<div class="lpage">13</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bioinformatics%20enrichment%20tools%3A%20paths%20toward%20the%20comprehensive%20functional%20analysis%20of%20large%20gene%20lists&amp;author=R.A.%20Lempicki&amp;author=D.W.%20Huang&amp;author=B.T.%20Sherman&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=1-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19033363" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">De Boever</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Vangestel</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">De Backer</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Croubels</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sys</div> <div class="given-names">S.U.</div></div>. </div><div class="article-title">Identification and validation of housekeeping genes as internal control for gene expression in an intravenous LPS inflammation model in chickens</div>, <div class="source">Vet. Immunol. Immunopathol.</div> , <div class="year">2008</div>, vol. <div class="volume">122</div> (pg. <div class="fpage">312</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20validation%20of%20housekeeping%20genes%20as%20internal%20control%20for%20gene%20expression%20in%20an%20intravenous%20LPS%20inflammation%20model%20in%20chickens&amp;author=S.%20De%20Boever&amp;author=C.%20Vangestel&amp;author=P.%20De%20Backer&amp;author=S.%20Croubels&amp;author=S.U.%20Sys&amp;publication_year=2008&amp;journal=Vet.%20Immunol.%20Immunopathol.&amp;volume=122&amp;pages=312-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.vetimm.2007.12.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.vetimm.2007.12.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.vetimm.2007.12.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18272235" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ekblom</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Balakrishnan</div> <div class="given-names">C.N.</div></div>,  <div class="name"><div class="surname">Burke</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Slate</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Digital gene expression analysis of the zebra finch genome</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">219</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Digital%20gene%20expression%20analysis%20of%20the%20zebra%20finch%20genome&amp;author=R.%20Ekblom&amp;author=C.N.%20Balakrishnan&amp;author=T.%20Burke&amp;author=J.%20Slate&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=219" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-219" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-219" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-219"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20359325" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ellegren</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Evolutionary stasis: the stable chromosomes of birds</div>, <div class="source">Trends Ecol. Evol.</div> , <div class="year">2010</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">283</div>-<div class="lpage">91</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20stasis%3A%20the%20stable%20chromosomes%20of%20birds&amp;author=H.%20Ellegren&amp;publication_year=2010&amp;journal=Trends%20Ecol.%20Evol.&amp;volume=25&amp;pages=283-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tree.2009.12.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tree.2009.12.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tree.2009.12.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20363047" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Griffin</div> <div class="given-names">D.K.</div></div>,  <div class="name"><div class="surname">Robertson</div> <div class="given-names">L.B.W.</div></div>,  <div class="name"><div class="surname">Tempest</div> <div class="given-names">H.G.</div></div>,  <div class="name"><div class="surname">Skinner</div> <div class="given-names">B.M.</div></div>. </div><div class="article-title">The evolution of the avian genome as revealed by comparative molecular cytogenetics</div>, <div class="source">Cytogenet Genome. Res.</div> , <div class="year">2007</div>, vol. <div class="volume">117</div> (pg. <div class="fpage">64</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20the%20avian%20genome%20as%20revealed%20by%20comparative%20molecular%20cytogenetics&amp;author=D.K.%20Griffin&amp;author=L.B.W.%20Robertson&amp;author=H.G.%20Tempest&amp;author=B.M.%20Skinner&amp;publication_year=2007&amp;journal=Cytogenet%20Genome.%20Res.&amp;volume=117&amp;pages=64-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000103166" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000103166" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000103166"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17675846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hughes</div> <div class="given-names">A.L.</div></div>. </div><div class="article-title">Looking for Darwin in all the wrong places: the misguided quest for positive selection at the nucleotide sequence level</div>, <div class="source">Heredity</div> , <div class="year">2007</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">364</div>-<div class="lpage">73</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Looking%20for%20Darwin%20in%20all%20the%20wrong%20places%3A%20the%20misguided%20quest%20for%20positive%20selection%20at%20the%20nucleotide%20sequence%20level&amp;author=A.L.%20Hughes&amp;publication_year=2007&amp;journal=Heredity&amp;volume=99&amp;pages=364-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.hdy.6801031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.hdy.6801031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.hdy.6801031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17622265" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barreto</div> <div class="given-names">F.S.</div></div>,  <div class="name"><div class="surname">Moy</div> <div class="given-names">G.W.</div></div>,  <div class="name"><div class="surname">Burton</div> <div class="given-names">R.S.</div></div>. </div><div class="article-title">Interpopulation patterns of divergence and selection across the transcriptome of the copepod <em>Tigriopus californicus</em></div>, <div class="source">Mol. Ecol.</div> , <div class="year">2011</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">560</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Interpopulation%20patterns%20of%20divergence%20and%20selection%20across%20the%20transcriptome%20of%20the%20copepod%20Tigriopus%20californicus&amp;author=F.S.%20Barreto&amp;author=G.W.%20Moy&amp;author=R.S.%20Burton&amp;publication_year=2011&amp;journal=Mol.%20Ecol.&amp;volume=20&amp;pages=560-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.2011.20.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.2011.20.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.2011.20.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21199025" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Elmer</div> <div class="given-names">K.R.</div></div>,  <div class="name"><div class="surname">Fan</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gunter</div> <div class="given-names">H.M.</div></div>, et al. </div><div class="article-title">Rapid evolution and selection inferred from the transcriptomes of sympatric crater lake cichlid fishes</div>, <div class="source">Mol. Ecol.</div> , <div class="year">2010</div>, vol. <div class="volume">19</div> <div class="issue">Suppl. 1)</div>(pg. <div class="fpage">197</div>-<div class="lpage">211</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20evolution%20and%20selection%20inferred%20from%20the%20transcriptomes%20of%20sympatric%20crater%20lake%20cichlid%20fishes&amp;author=K.R.%20Elmer&amp;author=S.%20Fan&amp;author=H.M.%20Gunter&amp;publication_year=2010&amp;journal=Mol.%20Ecol.&amp;volume=19&amp;pages=197-211" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.2010.19.issue-s1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.2010.19.issue-s1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.2010.19.issue-s1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20331780" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yang</div> <div class="given-names">Z.H.</div></div>,  <div class="name"><div class="surname">Bielawski</div> <div class="given-names">J.P.</div></div>. </div><div class="article-title">Statistical methods for detecting molecular adaptation</div>, <div class="source">Trends Ecol. Evol.</div> , <div class="year">2000</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">496</div>-<div class="lpage">503</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistical%20methods%20for%20detecting%20molecular%20adaptation&amp;author=Z.H.%20Yang&amp;author=J.P.%20Bielawski&amp;publication_year=2000&amp;journal=Trends%20Ecol.%20Evol.&amp;volume=15&amp;pages=496-503" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0169-5347(00)01994-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0169-5347(00)01994-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0169-5347(00)01994-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11114436" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schneider</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Souvorov</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sabath</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Landan</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Gonnet</div> <div class="given-names">G.H.</div></div>,  <div class="name"><div class="surname">Graur</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Estimates of positive Darwinian selection are inflated by errors in sequencing, annotation, and alignment</div>, <div class="source">Genome Biol. Evol.</div> , <div class="year">2009</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">114</div>-<div class="lpage">18</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimates%20of%20positive%20Darwinian%20selection%20are%20inflated%20by%20errors%20in%20sequencing%2C%20annotation%2C%20and%20alignment&amp;author=A.%20Schneider&amp;author=A.%20Souvorov&amp;author=N.%20Sabath&amp;author=G.%20Landan&amp;author=G.H.%20Gonnet&amp;author=D.%20Graur&amp;publication_year=2009&amp;journal=Genome%20Biol.%20Evol.&amp;volume=1&amp;pages=114-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evp012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevp012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevp012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20333182" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kryazhimskiy</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Plotkin</div> <div class="given-names">J.B.</div></div>. </div><div class="article-title">The population genetics of dN/dS</div>, <div class="source">PLoS Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e1000304</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20population%20genetics%20of%20dN%2FdS&amp;author=S.%20Kryazhimskiy&amp;author=J.B.%20Plotkin&amp;publication_year=2008&amp;journal=PLoS%20Genet.&amp;volume=4&amp;pages=e1000304" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000304" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000304" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000304"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19081788" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rocha</div> <div class="given-names">E.P.</div></div>,  <div class="name"><div class="surname">Smith</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Hurst</div> <div class="given-names">L.D.</div></div>, et al. </div><div class="article-title">Comparisons of dN/dS are time dependent for closely related bacterial genomes</div>, <div class="source">J. Theor. Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">239</div> (pg. <div class="fpage">226</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparisons%20of%20dN%2FdS%20are%20time%20dependent%20for%20closely%20related%20bacterial%20genomes&amp;author=E.P.%20Rocha&amp;author=J.M.%20Smith&amp;author=L.D.%20Hurst&amp;publication_year=2006&amp;journal=J.%20Theor.%20Biol.&amp;volume=239&amp;pages=226-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jtbi.2005.08.037" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jtbi.2005.08.037" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jtbi.2005.08.037"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16239014" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS015C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS015C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wolf</div> <div class="given-names">J.B.</div></div>,  <div class="name"><div class="surname">Kunstner</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Nam</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Jakobsson</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ellegren</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Nonlinear dynamics of nonsynonymous (dN) and synonymous (dS) substitution rates affects inference of selection</div>, <div class="source">Genome Biol. Evol.</div> , <div class="year">2009</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">308</div>-<div class="lpage">19</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nonlinear%20dynamics%20of%20nonsynonymous%20(dN)%20and%20synonymous%20(dS)%20substitution%20rates%20affects%20inference%20of%20selection&amp;author=J.B.%20Wolf&amp;author=A.%20Kunstner&amp;author=K.%20Nam&amp;author=M.%20Jakobsson&amp;author=H.%20Ellegren&amp;publication_year=2009&amp;journal=Genome%20Biol.%20Evol.&amp;volume=1&amp;pages=308-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evp030" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevp030" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevp030"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20333200" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=450749&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/dss015-dss015supp_table4.xlsx?Expires=2147483647&amp;Signature=gpHVV692OwaPJ-PuZ~ZrnUim8Y-RNYBjhZSLwEy1dvIeue78iamZqO-ACNhOKZDS~epmQbHnETIky0hd116jqDJd6TeB5nnEA4mfMg6Vv0cOmYp9UicutQ~dAGu6KRtjbIh9GxsJDHQCkB2Hd8b-U6Vtq3C6Kgukx50I62AcRFECt3cy6~HM4E4NKXGH36zNnfodkFQZp1Nyok0JBWzpsDumRU6QjO07LnYKu~WzYtWe3qtTTuJUsaD26MYxkjDV1PK0PXpc8LRGb0rwGgvPrA3QpamnsKwkerJycyqaRh9yQEyMAInynTyICkGcuoA8frhWVWMisQVJ38kZyUe4Fg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-12</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/dss015-dss015supp_table3.xlsx?Expires=2147483647&amp;Signature=HP64pZYcxtsdrKXlCTOyCvKboQLmHYQ~L8fOKRivEC~RE6Ivu24xeOUsIY8sNGvzSSy400k0iiLLSA~gkuqU-5oMKpjLLVSTb3Pc2ozrfsZfrwiuMThgvB9dSRKN8IvGKoSoJ0GfR1eKWmM2KhtK2n8CdydigTN8Zc6mI5lKOJUvI-~sMbXi6sak7I7tbuJfSO2ctWiOfr~-wctbx9p-gGSmZXL2krTuXr3cDicBubhEnI1q~V~4MFabk4vVQ61GikI3MgwRmtJZfqWD80ignruYKy-ZEqxOfzWVWYePZwqE2H6uAoFjtDtuFRXhgPNx7yPrYBjI9cYdPDqdsfXYAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-11</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/dss015-dss015supp_table2.xlsx?Expires=2147483647&amp;Signature=kqI~DD8EOrpqgLjWxaKhaH8NywJWmkn3~hqYRgc0wnAEPaAFLIGYK9Y~6IefV0wQRPOx1C30F7O7Ec-EPyz7HL09EH-yA13bhJm4vG~41AG9MrxWi5rlbvjJ4h3Nq7~oO203ECK7iGsscN03OT2alElQlL7GxJAxSOP9tz6v4LXbK4Op4aLBs5TkiOdEq64M8TS2PEPcPwu0i2xa1m3YsRP~t00h5xmIVVjOeUA65T0pRW7Fm1ddHHLk4BtIdRZwZQcdvHPHFUl2d3jrff3Ie50yzicyaLXOeBPBxW0M56~xUDEm6LBFkw62bQR96~NPdKPeLMgbzk2SjIi8ZXVfKA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-10</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/dss015-dss015supp_table1.xlsx?Expires=2147483647&amp;Signature=zwBMfw7pKx6W~NLIH-frB0Pte3AN~u4SzKsSBcrEDZqQKHouJxziVHEXW0clombAjJb90uPCu4xI60gLwQgAPE5R3zkSY-jwufMPavuuT4XuAl5yCx3wtETlWRJVgO8wKKgc3t-1I3BMZEkGWyWItCa3OI9GKbxVI-t5x3E26YcM0g5i0WVxY8Zp9aDn04mmt~AfE0iBR9VxyrIcuGE574zUnO8eY0~y0yU-q7CZdnsOe5maRHoH0-tO7bJ08R2m5KaR6rX-pVKwF8nZgy4Gypp8Rd72R4OkENDP0mdH6fzsSeSX9MTFWKPc99oflqeNVa~-Z6MjyPQaWXRVKlXPMQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-9</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/dss015-dss015supp_fig3B.png?Expires=2147483647&amp;Signature=V0hLSh7pC~LVpNnKCBzQOBJMNln5wzc~GtrLBAiQOPB6F7s2aNkASnq8i0l4hiTxl6OCOS25-KYJm58Ezxllt1vq8WiChRhRBIOfjRacOkX513wASlWGFdoLZrfoSStzUhQnYsE-gq3JmikL0gqJroZhugucYEvasjJ~glSWePczDqCQLWi6pXzJZFHCnF4KmKD7ZpklgKehuEWvPp6hh4E~XZqy50yz-27LsuasnFX3lYzBldeT4uMpn9i-10oN4yag734HEMnOrEtkgvznhU5tNO0-GsRdRkrJ98ZSQ1JqwEykyosHFDfQxVdlm5LbNrQCJjc6EZlx6IJQjQfAmw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/dss015-dss015supp_fig3A.png?Expires=2147483647&amp;Signature=476tOxt6kKkQTPaSXiRo9Ah3vKW4kTu4LJvVXWEORA4S68b~FdWI8VCK3yLzS7Dv11qGjX79pAUZux66LnqL5g8HglIdkCtYg-p5~8LoVvpZ990wpR9pbgC3MKPu4q15liB~XoOuapCE4WDio3dHvXGdFMo3EQtH-Y~HPJw5iUGp-KMR91hMZmQJKOqfuACIJZ3T1jtxFjKJ1L9b20-5B0jajWlStiGF47jwtU--9xQa4vtklt7QOnzy8B6noaCA-SFEzkA-WAQda0MDpIVU3-sxkm-Qw95kR9a5PO8RMdSM-scNDTmnq5QknMchcGmurtmzH2O6Q5Msn8ILZLNuNQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/dss015-dss015supp_fig2B.png?Expires=2147483647&amp;Signature=2-1ZgzdlBaHtcW8ZwT3sBLd-pFnj5ahsxshxFkABBOIE9XXnFyclMLQemF2~X2d1thpjqh1G-SXIsFSDTrD5-bHRCGQfRmxBNEtBPQo3zHeDvNM-BmpMy56iBbQbc~hRHtnWe8dck8-j56jHPFBhUlLel7kOwzDB7Hzt9SpNyiGRYLHkVLHkAmzOLQ1ii4nNCkxTXXRQEfL9cnBvshCxWoh4XZmwYuwQ6G56-8W10~7Wv8olbkuORkCwCC6hoK-2V7gM3qvo41nwE9aAWLaAnw6fdcP0J-IxTB56YR~xNkvsw~LnWFk0u1TPs0klPFLwh~ghum90czQm5lCfNxIzVw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/dss015-dss015supp_fig2A.png?Expires=2147483647&amp;Signature=T~rVajVNcCsxPsCMjtc6j~AB-oMijlIMKW7Wcjo3zyAWx9z-8rWdcgvXclwaie8ykUTKh70h5wM-aMPDekvhBvp0rksZANV6u1hVIiwtajTRRYQVXT8apShK5S~4oGaKP5j2wYn7cgFL8Jzw~AoE0jvxNqVYxq6m~HPe0cnXJqmX5aQRf~ZdZsNf6vgbHEFYBaX-zkfrytaThaaCAP3DJJ~-inNYrSSoQvcEZT0BXolf3WEu3a4ZN2CSiMDRn3I-k1RcBQfGWBF-vIzYSVN4sDeBeX5e89xM7twhRIe3eEdXlwWGH4ZhtaDEcZ4NZO2Pt3yzZtwWT0MjjlimVupvYQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/dss015-dss015supp_fig1C.png?Expires=2147483647&amp;Signature=HTCaer87QCMw9lqMwHostYfWC6bJIVECyVaPiT2S-4QQwCLk6D8DaRJeL~ct7IEiPguSFPidsh2bbqXCRzCnV2DxQufW~rAoCESzDL13Bft-fVfbaHalXCDQlyxHZAQYaetVCvhPflYld3d-yZmpPdbKVx4SCP5RaezaxGulgoSu~pkCkgzyZbWdJ2cOTgZT1WOiyLhS19bhhfiu3xPH3Vbd~~iEmTNngjxpX~N2Msb4o5ZvIu2srrpXmrSUymgHyI0xR19wrAgODUAiBoSNfcoVVit7hvvvQnYlr~dSORpQHc8LAQwIbKiDXrMWUJ4aF6wN-HzRFO8wgptsWGlVSw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/dss015-dss015supp_fig1B.png?Expires=2147483647&amp;Signature=cdlAteQbzYXUfNBy85UcwjZ8KxNqHDAdSjMe6hQ7ZD9vSYiJoEbZrmAMOwKiCB0jqsH3oILtVBUJ5uG0wBpYyJJuyHnw-Abszd57qpMWDM7~~O4x5p1pksduhGpoBcB2ke3QUhHAxAqTQX8r4vC9TwrRZWswmRAhiSlK4RzTdUvSdH3xm8X~JYrEitOznKEGD1IxCziNgRVG3RUEIP3trraz9Df6uVKMZ2V4xWEZMm2KrCOItsxLesOaJ3Bs2EbPpSPar3AuGVl61f-mYuiQvJTEGbb-D0zyKd4MQ3q3YDXpSys0~AXQrOdj6mAVMsNx248c0i53IjvYmSyTdHyNWA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/4/10.1093/dnares/dss015/2/dss015-dss015supp_fig1A.png?Expires=2147483647&amp;Signature=EzS4gXTUPhnOKMJuhGEphYBsis0VS8U~lOe3awmiBcrQLatUHZpfSXWZws5v2FNsAP0D4UHzah24QDLaXEjUqxrYLnp5zyudzE68fMhUDSrJsmPyzPGGnex5Xtl2IhMx3aOKMP~VtWEbg4DTdpRm-iN-opBRhnvxpm2h2qykuCALJ2olL8Teh0VQeEN0cOykBPpsxKGoXSFy7-hK3fysndURMw-FTN9RshBIcQSpaJy6lyHtGinVjybP28efi9rDPmmKuC9TBkiLS59stZFsR3h6hWyWyhk01SasIzcnx6xkD-C1PHI6QJXQc6QpJHoPtSZm-n4a2U-AlCDjAwbMdg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - png file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">133</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000308385400003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">11</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dss015"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="450749"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000308385400003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f4%2f325%2f450749">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000308385400003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (11)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F4%2F325%2F450749" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/450749'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Transcriptome Analysis of a North American Songbird, Melospiza melodia" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dss015");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f4%2f325%2f450749">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

