

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":1,"applicationTime":2169,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/3/327/4898128","@type":"ScholarlyArticle","name":"Functional divergence of duplicate genes several million years after gene duplication in  Arabidopsis ","datePublished":"2018-02-21","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2018-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy005","keywords":["plant evolution","gene duplication","<italic>Arabidopsis</italic>","selection pressure","functional divergence"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/3/327/4898128","author":[{"name":"Hanada, Kousuke","affiliation":"Department of Bioscience and Bioinformatics, Frontier Research Academy for Young Researchers, Kyusyu Institute of Technology, Iizuka, Fukuoka 820-8502, Japan  RIKEN Center for Sustainable Resource Science, RIKEN, Yokohama, Kanagawa 230-0045, Japan  CREST, Japan Science and Technology Agency, Kawaguchi, Saitama 332-0012, Japan","@type":"Person"},{"name":"Tezuka, Ayumi","affiliation":"Department of Bioscience and Bioinformatics, Frontier Research Academy for Young Researchers, Kyusyu Institute of Technology, Iizuka, Fukuoka 820-8502, Japan","@type":"Person"},{"name":"Nozawa, Masafumi","affiliation":"Center for Information Biology, National Institute of Genetics, Mishima, Shizuoka 411-8540, Japan  Department of Genetics, SOKENDAI, Mishima, Shizuoka 411-8540, Japan  Department of Biological Sciences, Tokyo Metropolitan University, Hachioji, Tokyo 192-0397, Japan","@type":"Person"},{"name":"Suzuki, Yutaka","affiliation":"Graduate School of Frontier Science, The University of Tokyo, Kashiwa, Chiba 277-8562, Japan","@type":"Person"},{"name":"Sugano, Sumio","affiliation":"Graduate School of Frontier Science, The University of Tokyo, Kashiwa, Chiba 277-8562, Japan","@type":"Person"},{"name":"Nagano, Atsushi J","affiliation":"CREST, Japan Science and Technology Agency, Kawaguchi, Saitama 332-0012, Japan  Center of Ecological Research, Kyoto University, Hirano, Otsu, Shiga 520-2113, Japan","@type":"Person"},{"name":"Ito, Motomi","affiliation":"Graduate School of Arts and Sciences, The University of Tokyo, Tokyo 153-8902, Japan","@type":"Person"},{"name":"Morinaga, Shin-Ichi","affiliation":"CREST, Japan Science and Technology Agency, Kawaguchi, Saitama 332-0012, Japan  Graduate School of Arts and Sciences, The University of Tokyo, Tokyo 153-8902, Japan  College of Bioresource Sciences, Nihon University, Fujisawa, Kanagawa 252-0880, Japan","@type":"Person"}],"description":"Abstract.  Lineage-specific duplicated genes likely contribute to the phenotypic divergence in closely related species. However, neither the frequency of duplic","pageStart":"327","pageEnd":"339","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/3/1/m_dnares_25_3cover.png?Expires=2147483647&Signature=zOskx939n215x6LsaNaPxmag9A6Tt6e7bP~Kt5lfxwaH8FjX1dKLUJ0mdj-Sm0uq9hmCpw3OAnAQFLS-GTY4CsXXX4aRhm5J1IbPv51CcTKXj828y94I4tx-BuFB-EA4WwEvEot4P8hDcnFzu~xWV52k2Sm6IHcCABqAv5wSWve6oMrdsQXhpN0srJqhN5UIcLjzws~g9qv24XhgtRb~Nw7X9IiEo1EM5V47od3bkv5yHlkUT1xmh3tJ4-ZjHw-HyAXqIp8eeHds4Cfjy9xqh8e3n5ynt516a1ZMF0hjA3wYH5X0-y3lqTHnWmgUvQv1L6dLXdRWhki~KqxTetYsJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Functional divergence of duplicate genes several million years after gene duplication in  Arabidopsis ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/3/1/m_dnares_25_3cover.png?Expires=2147483647&Signature=zOskx939n215x6LsaNaPxmag9A6Tt6e7bP~Kt5lfxwaH8FjX1dKLUJ0mdj-Sm0uq9hmCpw3OAnAQFLS-GTY4CsXXX4aRhm5J1IbPv51CcTKXj828y94I4tx-BuFB-EA4WwEvEot4P8hDcnFzu~xWV52k2Sm6IHcCABqAv5wSWve6oMrdsQXhpN0srJqhN5UIcLjzws~g9qv24XhgtRb~Nw7X9IiEo1EM5V47od3bkv5yHlkUT1xmh3tJ4-ZjHw-HyAXqIp8eeHds4Cfjy9xqh8e3n5ynt516a1ZMF0hjA3wYH5X0-y3lqTHnWmgUvQv1L6dLXdRWhki~KqxTetYsJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Functional divergence of duplicate genes several million years after gene duplication in  Arabidopsis " />
    <meta property="og:description" content="Abstract.  Lineage-specific duplicated genes likely contribute to the phenotypic divergence in closely related species. However, neither the frequency of duplic" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/3/327/4898128" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2018-02-21" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Hanada, Kousuke" /><meta name="citation_author_institution" content="Department of Bioscience and Bioinformatics, Frontier Research Academy for Young Researchers, Kyusyu Institute of Technology, Iizuka, Fukuoka 820-8502, Japan" /><meta name="citation_author_institution" content="RIKEN Center for Sustainable Resource Science, RIKEN, Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author_institution" content="CREST, Japan Science and Technology Agency, Kawaguchi, Saitama 332-0012, Japan" /><meta name="citation_author" content="Tezuka, Ayumi" /><meta name="citation_author_institution" content="Department of Bioscience and Bioinformatics, Frontier Research Academy for Young Researchers, Kyusyu Institute of Technology, Iizuka, Fukuoka 820-8502, Japan" /><meta name="citation_author" content="Nozawa, Masafumi" /><meta name="citation_author_institution" content="Center for Information Biology, National Institute of Genetics, Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author_institution" content="Department of Genetics, SOKENDAI, Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author_institution" content="Department of Biological Sciences, Tokyo Metropolitan University, Hachioji, Tokyo 192-0397, Japan" /><meta name="citation_author" content="Suzuki, Yutaka" /><meta name="citation_author_institution" content="Graduate School of Frontier Science, The University of Tokyo, Kashiwa, Chiba 277-8562, Japan" /><meta name="citation_author" content="Sugano, Sumio" /><meta name="citation_author_institution" content="Graduate School of Frontier Science, The University of Tokyo, Kashiwa, Chiba 277-8562, Japan" /><meta name="citation_author" content="Nagano, Atsushi J" /><meta name="citation_author_institution" content="CREST, Japan Science and Technology Agency, Kawaguchi, Saitama 332-0012, Japan" /><meta name="citation_author_institution" content="Center of Ecological Research, Kyoto University, Hirano, Otsu, Shiga 520-2113, Japan" /><meta name="citation_author" content="Ito, Motomi" /><meta name="citation_author_institution" content="Graduate School of Arts and Sciences, The University of Tokyo, Tokyo 153-8902, Japan" /><meta name="citation_author" content="Morinaga, Shin-Ichi" /><meta name="citation_author_institution" content="CREST, Japan Science and Technology Agency, Kawaguchi, Saitama 332-0012, Japan" /><meta name="citation_author_institution" content="Graduate School of Arts and Sciences, The University of Tokyo, Tokyo 153-8902, Japan" /><meta name="citation_author_institution" content="College of Bioresource Sciences, Nihon University, Fujisawa, Kanagawa 252-0880, Japan" /><meta name="citation_title" content="Functional divergence of duplicate genes several million years after gene duplication in Arabidopsis" /><meta name="citation_firstpage" content="327" /><meta name="citation_lastpage" content="339" /><meta name="citation_doi" content="10.1093/dnares/dsy005" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2018/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Plant conserved non-coding sequences and paralogue evolution;  citation_journal_title=Trends Genet;  citation_year=2005;  citation_volume=21;  citation_pages=60-5; " /><meta name="citation_reference" content="citation_title=Analysis of 41 plant genomes supports a wave of successful genome duplications in association with the Cretaceous-Paleogene boundary;  citation_journal_title=Genome Res;  citation_year=2014;  citation_volume=24;  citation_pages=1334-47; " /><meta name="citation_reference" content="citation_title=The functional role of pack-MULEs in rice inferred from purifying selection and expression profile;  citation_journal_title=Plant Cell;  citation_year=2009;  citation_volume=21;  citation_pages=25-38; " /><meta name="citation_reference" content="citation_title=Importance of lineage-specific expansion of plant tandem duplicates in the adaptive response to environmental stimuli;  citation_journal_title=Plant Physiol;  citation_year=2008;  citation_volume=148;  citation_pages=993-1003; " /><meta name="citation_reference" content="citation_title=Lineage-specific gene duplication and loss in human and great ape evolution;  citation_journal_title=PLoS Biol;  citation_year=2004;  citation_volume=2;  citation_pages=E207" /><meta name="citation_reference" content="citation_title=Single-feature polymorphism discovery in the barley transcriptome;  citation_journal_title=Genome Biol;  citation_year=2005;  citation_volume=6;  citation_pages=R54" /><meta name="citation_reference" content="citation_title=Common sequence polymorphisms shaping genetic diversity in Arabidopsis thaliana;  citation_journal_title=Science;  citation_year=2007;  citation_volume=317;  citation_pages=338-42; " /><meta name="citation_reference" content="citation_title=Striking similarities in the genomic distribution of tandemly arrayed genes in Arabidopsis and rice;  citation_journal_title=PLoS Comput Biol;  citation_year=2006;  citation_volume=2;  citation_pages=e115" /><meta name="citation_reference" content="citation_title=Contrasting patterns of evolution following whole genome versus tandem duplication events in Populus;  citation_journal_title=Genome Res;  citation_year=2012;  citation_volume=22;  citation_pages=95-105; " /><meta name="citation_reference" content="citation_title=Tandem and segmental gene duplication and recombination in the evolution of plant disease resistance gene;  citation_journal_title=Trends Genet;  citation_year=2004;  citation_volume=20;  citation_pages=116-22; " /><meta name="citation_reference" content="citation_publisher=Springer-Verlag, New York;  citation_title=Evolution by Gene Duplication;  citation_year=1970; " /><meta name="citation_reference" content="citation_title=Preservation of duplicate genes by complementary, degenerative mutations;  citation_journal_title=Genetics;  citation_year=1999;  citation_volume=151;  citation_pages=1531-45; " /><meta name="citation_reference" content="citation_title=Evolutionary and expression signatures of pseudogenes in Arabidopsis and rice;  citation_journal_title=Plant Physiol;  citation_year=2009;  citation_volume=151;  citation_pages=3-15; " /><meta name="citation_reference" content="citation_title=The ambiguous boundary between genes and pseudogenes: the dead rise up, or do they?;  citation_journal_title=Trends Genet;  citation_year=2007;  citation_volume=23;  citation_pages=219-24; " /><meta name="citation_reference" content="citation_title=Turning a hobby into a job: how duplicated genes find new functions;  citation_journal_title=Nat. Rev. Genet;  citation_year=2008;  citation_volume=9;  citation_pages=938-50; " /><meta name="citation_reference" content="citation_title=Gene duplication as a mechanism of genomic adaptation to a changing environment;  citation_journal_title=Proc. Biol. Sci;  citation_year=2012;  citation_volume=279;  citation_pages=5048-57; " /><meta name="citation_reference" content="citation_title=Evolutionary persistence of functional compensation by duplicate genes in Arabidopsis;  citation_journal_title=Genome Biol. Evol;  citation_year=2009;  citation_volume=1;  citation_pages=409-14; " /><meta name="citation_reference" content="citation_title=Functional compensation of primary and secondary metabolites by duplicate genes in Arabidopsis thaliana;  citation_journal_title=Mol. Biol. Evol;  citation_year=2011;  citation_volume=28;  citation_pages=377-82; " /><meta name="citation_reference" content="citation_title=Evolution of genetic redundancy;  citation_journal_title=Nature;  citation_year=1997;  citation_volume=388;  citation_pages=167-71; " /><meta name="citation_reference" content="citation_title=Increased expression and protein divergence in duplicate genes is associated with morphological diversification;  citation_journal_title=PLoS Genet;  citation_year=2009;  citation_volume=5;  citation_pages=e1000781" /><meta name="citation_reference" content="citation_title=Evolutionary rate and duplicability in the Arabidopsis thaliana protein-protein interaction network;  citation_journal_title=Genome Biol. Evol;  citation_year=2012;  citation_volume=4;  citation_pages=1263-74; " /><meta name="citation_reference" content="citation_title=Functional bias in molecular evolution rate of Arabidopsis thaliana;  citation_journal_title=BMC Evol. Biol;  citation_year=2010;  citation_volume=10;  citation_pages=125" /><meta name="citation_reference" content="citation_title=Divergent evolutionary and expression patterns between lineage specific new duplicate genes and their parental paralogs in Arabidopsis thaliana;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e72362" /><meta name="citation_reference" content="citation_title=Factors that contribute to variation in evolutionary rate among Arabidopsis genes;  citation_journal_title=Mol. Biol. Evol;  citation_year=2011;  citation_volume=28;  citation_pages=2359-69; " /><meta name="citation_reference" content="citation_title=Evolution of plant genome architecture;  citation_journal_title=Genome Biol.;  citation_year=2016;  citation_volume=17;  citation_pages=37" /><meta name="citation_reference" content="citation_title=A survey of innovation through duplication in the reduced genomes of twelve parasites;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e99213" /><meta name="citation_reference" content="citation_title=Improved gap size estimation for scaffolding algorithms;  citation_journal_title=Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=2215-22; " /><meta name="citation_reference" content="citation_title=Analysis of the genome sequence of the flowering plant Arabidopsis thaliana; citation_author=Arabidopsis_Genome_Initiative;  citation_journal_title=Nature;  citation_year=2000;  citation_volume=408;  citation_pages=796-815; " /><meta name="citation_reference" content="citation_title=The Arabidopsis lyrata genome sequence and the basis of rapid genome size change;  citation_journal_title=Nature Genetics;  citation_year=2011;  citation_volume=43;  citation_pages=476-81; " /><meta name="citation_reference" content="citation_title=Dated molecular phylogenies indicate a Miocene origin for Arabidopsis thaliana;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2010;  citation_volume=107;  citation_pages=18724-8; " /><meta name="citation_reference" content="citation_title=Consequences of whole-genome triplication as revealed by comparative genomic analyses of the wild radish raphanus raphanistrum and three other Brassicaceae species;  citation_journal_title=Plant Cell;  citation_year=2014;  citation_volume=26;  citation_pages=1925-37; " /><meta name="citation_reference" content="citation_title=Beyond the thale: comparative genomics and genetics of Arabidopsis relatives;  citation_journal_title=Nat. Rev. Genet;  citation_year=2015;  citation_volume=16;  citation_pages=285-98; " /><meta name="citation_reference" content="citation_title=Metal hyperaccumulation in plants;  citation_journal_title=Ann. Rev. Plant Biol;  citation_year=2010;  citation_volume=61;  citation_pages=517-34; " /><meta name="citation_reference" content="citation_title=Molecular mechanisms of metal hyperaccumulation in plants;  citation_journal_title=New Phytol;  citation_year=2009;  citation_volume=181;  citation_pages=759-76; " /><meta name="citation_reference" content="citation_title=Evolutionary and ecological genomics of Arabidopsis;  citation_journal_title=Plant Physiol;  citation_year=2005;  citation_volume=138;  citation_pages=578-84; " /><meta name="citation_reference" content="citation_title=Population resequencing reveals local adaptation of Arabidopsis lyrata to serpentine soils;  citation_journal_title=Nat. Genet;  citation_year=2010;  citation_volume=42;  citation_pages=260-3; " /><meta name="citation_reference" content="citation_title=Sequencing of the genus Arabidopsis identifies a complex history of nonbifurcating speciation and abundant trans-specific polymorphism;  citation_journal_title=Nat. Genet;  citation_year=2016;  citation_volume=48;  citation_pages=1077-82; " /><meta name="citation_reference" content="citation_title=Genome assembly and annotation of Arabidopsis halleri, a model for heavy metal hyperaccumulation and evolutionary ecology;  citation_journal_title=Mol. Ecol. Resour;  citation_year=2017;  citation_volume=17;  citation_pages=1025-36; " /><meta name="citation_reference" content="citation_title=Genome-wide quantification of homeolog expression ratio revealed nonstochastic gene regulation in synthetic allopolyploid Arabidopsis;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=e46" /><meta name="citation_reference" content="citation_title=Fine-scale genetic differentiation of a temperate herb: relevance of local environments and demographic change;  citation_journal_title=AoB Plants;  citation_year=2014;  citation_volume=6;  citation_pages=plu070" /><meta name="citation_reference" content="citation_title=Evolution of metal hyperaccumulation required cis-regulatory changes and triplication of HMA4;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=453;  citation_pages=391-5; " /><meta name="citation_reference" content="Al-Shehbaz I. A., O'Kane S. L.  2002 , Taxonomy and phylogeny of Arabidopsis (Brassicaceae). In The Arabidopsis Book/American Society of Plant Biologists , Vol. 1. " /><meta name="citation_reference" content="citation_title=BLAST: a more efficient report with usability improvements;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=W29-33; " /><meta name="citation_reference" content="citation_title=Evolution of genome size in Brassicaceae;  citation_journal_title=Ann. Bot;  citation_year=2005;  citation_volume=95;  citation_pages=229-35; " /><meta name="citation_reference" content="citation_title=Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads;  citation_journal_title=Genome Res;  citation_year=2014;  citation_volume=24;  citation_pages=1384-95; " /><meta name="citation_reference" content="citation_title=ABySS: a parallel assembler for short read sequence data;  citation_journal_title=Genome Res;  citation_year=2009;  citation_volume=19;  citation_pages=1117-23; " /><meta name="citation_reference" content="citation_title=SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler;  citation_journal_title=Gigasci;  citation_year=2012;  citation_volume=1;  citation_pages=18" /><meta name="citation_reference" content="citation_title=GMAP: a genomic mapping and alignment program for mRNA and EST sequences;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=1859-75; " /><meta name="citation_reference" content="citation_title=Performance of genetic programming optimised Bowtie2 on genome comparison and analytic testing (GCAT) benchmarks;  citation_journal_title=BioData Min;  citation_year=2015;  citation_volume=8;  citation_pages=1" /><meta name="citation_reference" content="citation_title=Primer3Plus, an enhanced web interface to Primer3;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=W71-4; " /><meta name="citation_reference" content="citation_title=Automatic clustering of orthologs and in-paralogs from pairwise species comparisons;  citation_journal_title=J. Mol. Biol;  citation_year=2001;  citation_volume=314;  citation_pages=1041-52; " /><meta name="citation_reference" content="citation_title=MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform;  citation_journal_title=Nucleic Acids Res;  citation_year=2002;  citation_volume=30;  citation_pages=3059-66; " /><meta name="citation_reference" content="citation_title=PAML 4: phylogenetic analysis by maximum likelihood;  citation_journal_title=Mol. Biol. Evol;  citation_year=2007;  citation_volume=24;  citation_pages=1586-91; " /><meta name="citation_reference" content="citation_title=The evolutionary fate and consequences of duplicate genes;  citation_journal_title=Science;  citation_year=2000;  citation_volume=290;  citation_pages=1151-5; " /><meta name="citation_reference" content="citation_title=Early evolutionary history and genomic features of gene duplicates in the human genome;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=621" /><meta name="citation_reference" content="citation_title=Very low gene duplication rate in the yeast genome;  citation_journal_title=Science;  citation_year=2004;  citation_volume=306;  citation_pages=1367-70; " /><meta name="citation_reference" content="citation_title=Droplet Digital PCR quantitation of HER2 expression in FFPE breast cancer samples;  citation_journal_title=Methods;  citation_year=2013;  citation_volume=59;  citation_pages=S20-3; " /><meta name="citation_reference" content="citation_title=Two genes encoding Arabidopsis halleri MTP1 metal transport proteins co-segregate with zinc tolerance and account for high MTP1 transcript levels;  citation_journal_title=Plant J;  citation_year=2004;  citation_volume=39;  citation_pages=425-39; " /><meta name="citation_reference" content="citation_title=Identification of shared single copy nuclear genes in Arabidopsis, Populus, Vitis and Oryza and their phylogenetic utility across various taxonomic levels;  citation_journal_title=BMC Evol. Biol;  citation_year=2010;  citation_volume=10;  citation_pages=61" /><meta name="citation_reference" content="citation_title=Inferring evolutionary trees with PAUP*;  citation_journal_title=Curr. Protoc. Bioinformatics;  citation_year=2003; " /><meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees;  citation_journal_title=Mol. Biol. Evol;  citation_year=1987;  citation_volume=4;  citation_pages=406-25; " /><meta name="citation_reference" content="citation_title=The genetic basis of zinc tolerance in the metallophyte Arabidopsis halleri ssp. halleri (Brassicaceae): an analysis of quantitative trait loci;  citation_journal_title=Genetics;  citation_year=2007;  citation_volume=176;  citation_pages=659-74; " /><meta name="citation_reference" content="citation_title=Genetic architecture of zinc hyperaccumulation in Arabidopsis halleri: the essential role of QTL x environment interactions;  citation_journal_title=New Phytol;  citation_year=2010;  citation_volume=187;  citation_pages=355-67; " /><meta name="citation_reference" content="citation_title=Young proteins experience more variable selection pressures than old proteins;  citation_journal_title=Genome Res;  citation_year=2010;  citation_volume=20;  citation_pages=1574-81; " /><meta name="citation_reference" content="citation_title=Duplicated genes evolve slower than singletons despite the initial rate increase;  citation_journal_title=BMC Evol. Biol;  citation_year=2004;  citation_volume=4;  citation_pages=22" /><meta name="citation_reference" content="citation_title=Inverse relationship between evolutionary rate and age of mammalian genes;  citation_journal_title=Mol. Biol. Evol;  citation_year=2005;  citation_volume=22;  citation_pages=598-606; " /><meta name="citation_reference" content="citation_title=The universal distribution of evolutionary rates of genes and distinct characteristics of eukaryotic genes of different apparent ages;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2009;  citation_volume=106;  citation_pages=7273-80; " /><meta name="citation_reference" content="citation_title=Preferential duplication of conserved proteins in eukaryotic genomes;  citation_journal_title=PLoS Biol;  citation_year=2004;  citation_volume=2;  citation_pages=E55" /><meta name="citation_reference" content="citation_title=Rate of protein evolution versus fitness effect of gene deletion;  citation_journal_title=Mol. Biol. Evol;  citation_year=2003;  citation_volume=20;  citation_pages=772-4; " /><meta name="citation_reference" content="citation_title=Evolution of vertebrate tissues driven by differential modes of gene duplication;  citation_journal_title=DNA Res;  citation_year=2012;  citation_volume=19;  citation_pages=305-16; " /><meta name="citation_reference" content="citation_title=Coregulation of tandem duplicate genes slows evolution of subfunctionalization in mammals;  citation_journal_title=Science;  citation_year=2016;  citation_volume=352;  citation_pages=1009-13; " /><meta name="citation_reference" content="citation_title=The Physcomitrella genome reveals evolutionary insights into the conquest of land by plants;  citation_journal_title=Science;  citation_year=2008;  citation_volume=319;  citation_pages=64-9; " /><meta name="citation_reference" content="citation_title=The coevolution of galaxies and supermassive black holes: a local perspective;  citation_journal_title=Science;  citation_year=2011;  citation_volume=333;  citation_pages=182-5; " /><meta name="citation_reference" content="citation_title=The genome of black cottonwood, Populus trichocarpa (Torr. &amp; Gray);  citation_journal_title=Science;  citation_year=2006;  citation_volume=313;  citation_pages=1596-604; " /><meta name="citation_reference" content="citation_title=Date of the monocot-dicot divergence estimated from chloroplast DNA sequence data;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=1989;  citation_volume=86;  citation_pages=6201-5; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/3/327/25072143/dsy005.pdf" /><meta name="description" content="Abstract.  Lineage-specific duplicated genes likely contribute to the phenotypic divergence in closely related species. However, neither the frequency of duplic" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/3/327/4898128" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/3/327/4898128" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/3/327/4898128");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsy005';
    </script>
<title>
	Functional divergence of duplicate genes several million years after gene duplication in Arabidopsis | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4898128" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BaWvATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGFuYWRhLCBLb3VzdWtlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9zY2llbmNlIGFuZCBCaW9pbmZvcm1hdGljcywgRnJvbnRpZXIgUmVzZWFyY2ggQWNhZGVteSBmb3IgWW91bmcgUmVzZWFyY2hlcnMsIEt5dXN5dSBJbnN0aXR1dGUgb2YgVGVjaG5vbG9neSwgSWl6dWthLCBGdWt1b2thIDgyMC04NTAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlJJS0VOIENlbnRlciBmb3IgU3VzdGFpbmFibGUgUmVzb3VyY2UgU2NpZW5jZSwgUklLRU4sIFlva29oYW1hLCBLYW5hZ2F3YSAyMzAtMDA0NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDUkVTVCwgSmFwYW4gU2NpZW5jZSBhbmQgVGVjaG5vbG9neSBBZ2VuY3ksIEthd2FndWNoaSwgU2FpdGFtYSAzMzItMDAxMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUZXp1a2EsIEF5dW1pIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9zY2llbmNlIGFuZCBCaW9pbmZvcm1hdGljcywgRnJvbnRpZXIgUmVzZWFyY2ggQWNhZGVteSBmb3IgWW91bmcgUmVzZWFyY2hlcnMsIEt5dXN5dSBJbnN0aXR1dGUgb2YgVGVjaG5vbG9neSwgSWl6dWthLCBGdWt1b2thIDgyMC04NTAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5vemF3YSwgTWFzYWZ1bWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50ZXIgZm9yIEluZm9ybWF0aW9uIEJpb2xvZ3ksIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBHZW5ldGljcywgTWlzaGltYSwgU2hpenVva2EgNDExLTg1NDAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBHZW5ldGljcywgU09LRU5EQUksIE1pc2hpbWEsIFNoaXp1b2thIDQxMS04NTQwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9naWNhbCBTY2llbmNlcywgVG9reW8gTWV0cm9wb2xpdGFuIFVuaXZlcnNpdHksIEhhY2hpb2ppLCBUb2t5byAxOTItMDM5NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTdXp1a2ksIFl1dGFrYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBGcm9udGllciBTY2llbmNlLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgS2FzaGl3YSwgQ2hpYmEgMjc3LTg1NjIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3VnYW5vLCBTdW1pbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBGcm9udGllciBTY2llbmNlLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgS2FzaGl3YSwgQ2hpYmEgMjc3LTg1NjIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFnYW5vLCBBdHN1c2hpIEoiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDUkVTVCwgSmFwYW4gU2NpZW5jZSBhbmQgVGVjaG5vbG9neSBBZ2VuY3ksIEthd2FndWNoaSwgU2FpdGFtYSAzMzItMDAxMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50ZXIgb2YgRWNvbG9naWNhbCBSZXNlYXJjaCwgS3lvdG8gVW5pdmVyc2l0eSwgSGlyYW5vLCBPdHN1LCBTaGlnYSA1MjAtMjExMywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJdG8sIE1vdG9taSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBBcnRzIGFuZCBTY2llbmNlcywgVGhlIFVuaXZlcnNpdHkgb2YgVG9reW8sIFRva3lvIDE1My04OTAyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1vcmluYWdhLCBTaGluLUljaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDUkVTVCwgSmFwYW4gU2NpZW5jZSBhbmQgVGVjaG5vbG9neSBBZ2VuY3ksIEthd2FndWNoaSwgU2FpdGFtYSAzMzItMDAxMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgQXJ0cyBhbmQgU2NpZW5jZXMsIFRoZSBVbml2ZXJzaXR5IG9mIFRva3lvLCBUb2t5byAxNTMtODkwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDb2xsZWdlIG9mIEJpb3Jlc291cmNlIFNjaWVuY2VzLCBOaWhvbiBVbml2ZXJzaXR5LCBGdWppc2F3YSwgS2FuYWdhd2EgMjUyLTA4ODAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJGdW5jdGlvbmFsIGRpdmVyZ2VuY2Ugb2YgZHVwbGljYXRlIGdlbmVzIHNldmVyYWwgbWlsbGlvbiB5ZWFycyBhZnRlciBnZW5lIGR1cGxpY2F0aW9uIGluIEFyYWJpZG9wc2lzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMzI3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIzMzkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3kwMDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxOC8wNi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYW50IGNvbnNlcnZlZCBub24tY29kaW5nIHNlcXVlbmNlcyBhbmQgcGFyYWxvZ3VlIGV2b2x1dGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NjAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgNDEgcGxhbnQgZ2Vub21lcyBzdXBwb3J0cyBhIHdhdmUgb2Ygc3VjY2Vzc2Z1bCBnZW5vbWUgZHVwbGljYXRpb25zIGluIGFzc29jaWF0aW9uIHdpdGggdGhlIENyZXRhY2VvdXMtUGFsZW9nZW5lIGJvdW5kYXJ5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MTMzNC00NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGZ1bmN0aW9uYWwgcm9sZSBvZiBwYWNrLU1VTEVzIGluIHJpY2UgaW5mZXJyZWQgZnJvbSBwdXJpZnlpbmcgc2VsZWN0aW9uIGFuZCBleHByZXNzaW9uIHByb2ZpbGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0yNS0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1wb3J0YW5jZSBvZiBsaW5lYWdlLXNwZWNpZmljIGV4cGFuc2lvbiBvZiBwbGFudCB0YW5kZW0gZHVwbGljYXRlcyBpbiB0aGUgYWRhcHRpdmUgcmVzcG9uc2UgdG8gZW52aXJvbm1lbnRhbCBzdGltdWxpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE0ODsgIGNpdGF0aW9uX3BhZ2VzPTk5My0xMDAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MaW5lYWdlLXNwZWNpZmljIGdlbmUgZHVwbGljYXRpb24gYW5kIGxvc3MgaW4gaHVtYW4gYW5kIGdyZWF0IGFwZSBldm9sdXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9RTIwNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbmdsZS1mZWF0dXJlIHBvbHltb3JwaGlzbSBkaXNjb3ZlcnkgaW4gdGhlIGJhcmxleSB0cmFuc2NyaXB0b21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9UjU0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tbW9uIHNlcXVlbmNlIHBvbHltb3JwaGlzbXMgc2hhcGluZyBnZW5ldGljIGRpdmVyc2l0eSBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTc7ICBjaXRhdGlvbl9wYWdlcz0zMzgtNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cmlraW5nIHNpbWlsYXJpdGllcyBpbiB0aGUgZ2Vub21pYyBkaXN0cmlidXRpb24gb2YgdGFuZGVtbHkgYXJyYXllZCBnZW5lcyBpbiBBcmFiaWRvcHNpcyBhbmQgcmljZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBDb21wdXQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9ZTExNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnRyYXN0aW5nIHBhdHRlcm5zIG9mIGV2b2x1dGlvbiBmb2xsb3dpbmcgd2hvbGUgZ2Vub21lIHZlcnN1cyB0YW5kZW0gZHVwbGljYXRpb24gZXZlbnRzIGluIFBvcHVsdXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz05NS0xMDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRhbmRlbSBhbmQgc2VnbWVudGFsIGdlbmUgZHVwbGljYXRpb24gYW5kIHJlY29tYmluYXRpb24gaW4gdGhlIGV2b2x1dGlvbiBvZiBwbGFudCBkaXNlYXNlIHJlc2lzdGFuY2UgZ2VuZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTE2LTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXItVmVybGFnLCBOZXcgWW9yazsgIGNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBieSBHZW5lIER1cGxpY2F0aW9uOyAgY2l0YXRpb25feWVhcj0xOTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVzZXJ2YXRpb24gb2YgZHVwbGljYXRlIGdlbmVzIGJ5IGNvbXBsZW1lbnRhcnksIGRlZ2VuZXJhdGl2ZSBtdXRhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTE1MTsgIGNpdGF0aW9uX3BhZ2VzPTE1MzEtNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbmFyeSBhbmQgZXhwcmVzc2lvbiBzaWduYXR1cmVzIG9mIHBzZXVkb2dlbmVzIGluIEFyYWJpZG9wc2lzIGFuZCByaWNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE1MTsgIGNpdGF0aW9uX3BhZ2VzPTMtMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBhbWJpZ3VvdXMgYm91bmRhcnkgYmV0d2VlbiBnZW5lcyBhbmQgcHNldWRvZ2VuZXM6IHRoZSBkZWFkIHJpc2UgdXAsIG9yIGRvIHRoZXk/OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0yMTktMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVR1cm5pbmcgYSBob2JieSBpbnRvIGEgam9iOiBob3cgZHVwbGljYXRlZCBnZW5lcyBmaW5kIG5ldyBmdW5jdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9OTM4LTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIGR1cGxpY2F0aW9uIGFzIGEgbWVjaGFuaXNtIG9mIGdlbm9taWMgYWRhcHRhdGlvbiB0byBhIGNoYW5naW5nIGVudmlyb25tZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBCaW9sLiBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mjc5OyAgY2l0YXRpb25fcGFnZXM9NTA0OC01NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uYXJ5IHBlcnNpc3RlbmNlIG9mIGZ1bmN0aW9uYWwgY29tcGVuc2F0aW9uIGJ5IGR1cGxpY2F0ZSBnZW5lcyBpbiBBcmFiaWRvcHNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTQwOS0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb25hbCBjb21wZW5zYXRpb24gb2YgcHJpbWFyeSBhbmQgc2Vjb25kYXJ5IG1ldGFib2xpdGVzIGJ5IGR1cGxpY2F0ZSBnZW5lcyBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9Mzc3LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24gb2YgZ2VuZXRpYyByZWR1bmRhbmN5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9Mzg4OyAgY2l0YXRpb25fcGFnZXM9MTY3LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmNyZWFzZWQgZXhwcmVzc2lvbiBhbmQgcHJvdGVpbiBkaXZlcmdlbmNlIGluIGR1cGxpY2F0ZSBnZW5lcyBpcyBhc3NvY2lhdGVkIHdpdGggbW9ycGhvbG9naWNhbCBkaXZlcnNpZmljYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAwNzgxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uYXJ5IHJhdGUgYW5kIGR1cGxpY2FiaWxpdHkgaW4gdGhlIEFyYWJpZG9wc2lzIHRoYWxpYW5hIHByb3RlaW4tcHJvdGVpbiBpbnRlcmFjdGlvbiBuZXR3b3JrOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MTI2My03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb25hbCBiaWFzIGluIG1vbGVjdWxhciBldm9sdXRpb24gcmF0ZSBvZiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEV2b2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXZlcmdlbnQgZXZvbHV0aW9uYXJ5IGFuZCBleHByZXNzaW9uIHBhdHRlcm5zIGJldHdlZW4gbGluZWFnZSBzcGVjaWZpYyBuZXcgZHVwbGljYXRlIGdlbmVzIGFuZCB0aGVpciBwYXJlbnRhbCBwYXJhbG9ncyBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWU3MjM2MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhY3RvcnMgdGhhdCBjb250cmlidXRlIHRvIHZhcmlhdGlvbiBpbiBldm9sdXRpb25hcnkgcmF0ZSBhbW9uZyBBcmFiaWRvcHNpcyBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MjM1OS02OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIHBsYW50IGdlbm9tZSBhcmNoaXRlY3R1cmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTM3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzdXJ2ZXkgb2YgaW5ub3ZhdGlvbiB0aHJvdWdoIGR1cGxpY2F0aW9uIGluIHRoZSByZWR1Y2VkIGdlbm9tZXMgb2YgdHdlbHZlIHBhcmFzaXRlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWU5OTIxMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcHJvdmVkIGdhcCBzaXplIGVzdGltYXRpb24gZm9yIHNjYWZmb2xkaW5nIGFsZ29yaXRobXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MjIxNS0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgdGhlIGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgZmxvd2VyaW5nIHBsYW50IEFyYWJpZG9wc2lzIHRoYWxpYW5hOyBjaXRhdGlvbl9hdXRob3I9QXJhYmlkb3BzaXNfR2Vub21lX0luaXRpYXRpdmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT00MDg7ICBjaXRhdGlvbl9wYWdlcz03OTYtODE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQXJhYmlkb3BzaXMgbHlyYXRhIGdlbm9tZSBzZXF1ZW5jZSBhbmQgdGhlIGJhc2lzIG9mIHJhcGlkIGdlbm9tZSBzaXplIGNoYW5nZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlIEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9NDc2LTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EYXRlZCBtb2xlY3VsYXIgcGh5bG9nZW5pZXMgaW5kaWNhdGUgYSBNaW9jZW5lIG9yaWdpbiBmb3IgQXJhYmlkb3BzaXMgdGhhbGlhbmE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS4gUy4gQTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDc7ICBjaXRhdGlvbl9wYWdlcz0xODcyNC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zZXF1ZW5jZXMgb2Ygd2hvbGUtZ2Vub21lIHRyaXBsaWNhdGlvbiBhcyByZXZlYWxlZCBieSBjb21wYXJhdGl2ZSBnZW5vbWljIGFuYWx5c2VzIG9mIHRoZSB3aWxkIHJhZGlzaCByYXBoYW51cyByYXBoYW5pc3RydW0gYW5kIHRocmVlIG90aGVyIEJyYXNzaWNhY2VhZSBzcGVjaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MTkyNS0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmV5b25kIHRoZSB0aGFsZTogY29tcGFyYXRpdmUgZ2Vub21pY3MgYW5kIGdlbmV0aWNzIG9mIEFyYWJpZG9wc2lzIHJlbGF0aXZlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9Mjg1LTk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZXRhbCBoeXBlcmFjY3VtdWxhdGlvbiBpbiBwbGFudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubi4gUmV2LiBQbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTYxOyAgY2l0YXRpb25fcGFnZXM9NTE3LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgbWVjaGFuaXNtcyBvZiBtZXRhbCBoeXBlcmFjY3VtdWxhdGlvbiBpbiBwbGFudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5ldyBQaHl0b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTgxOyAgY2l0YXRpb25fcGFnZXM9NzU5LTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgYW5kIGVjb2xvZ2ljYWwgZ2Vub21pY3Mgb2YgQXJhYmlkb3BzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTM4OyAgY2l0YXRpb25fcGFnZXM9NTc4LTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3B1bGF0aW9uIHJlc2VxdWVuY2luZyByZXZlYWxzIGxvY2FsIGFkYXB0YXRpb24gb2YgQXJhYmlkb3BzaXMgbHlyYXRhIHRvIHNlcnBlbnRpbmUgc29pbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz0yNjAtMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2luZyBvZiB0aGUgZ2VudXMgQXJhYmlkb3BzaXMgaWRlbnRpZmllcyBhIGNvbXBsZXggaGlzdG9yeSBvZiBub25iaWZ1cmNhdGluZyBzcGVjaWF0aW9uIGFuZCBhYnVuZGFudCB0cmFucy1zcGVjaWZpYyBwb2x5bW9ycGhpc207ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NDg7ICBjaXRhdGlvbl9wYWdlcz0xMDc3LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgYXNzZW1ibHkgYW5kIGFubm90YXRpb24gb2YgQXJhYmlkb3BzaXMgaGFsbGVyaSwgYSBtb2RlbCBmb3IgaGVhdnkgbWV0YWwgaHlwZXJhY2N1bXVsYXRpb24gYW5kIGV2b2x1dGlvbmFyeSBlY29sb2d5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEVjb2wuIFJlc291cjsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTEwMjUtMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIHF1YW50aWZpY2F0aW9uIG9mIGhvbWVvbG9nIGV4cHJlc3Npb24gcmF0aW8gcmV2ZWFsZWQgbm9uc3RvY2hhc3RpYyBnZW5lIHJlZ3VsYXRpb24gaW4gc3ludGhldGljIGFsbG9wb2x5cGxvaWQgQXJhYmlkb3BzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9ZTQ2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmluZS1zY2FsZSBnZW5ldGljIGRpZmZlcmVudGlhdGlvbiBvZiBhIHRlbXBlcmF0ZSBoZXJiOiByZWxldmFuY2Ugb2YgbG9jYWwgZW52aXJvbm1lbnRzIGFuZCBkZW1vZ3JhcGhpYyBjaGFuZ2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFvQiBQbGFudHM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPXBsdTA3MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBvZiBtZXRhbCBoeXBlcmFjY3VtdWxhdGlvbiByZXF1aXJlZCBjaXMtcmVndWxhdG9yeSBjaGFuZ2VzIGFuZCB0cmlwbGljYXRpb24gb2YgSE1BNDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ1MzsgIGNpdGF0aW9uX3BhZ2VzPTM5MS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJBbC1TaGVoYmF6IEkuIEEuLCBPJ0thbmUgUy4gTC4gIDIwMDIgLCBUYXhvbm9teSBhbmQgcGh5bG9nZW55IG9mIEFyYWJpZG9wc2lzIChCcmFzc2ljYWNlYWUpLiBJbiBUaGUgQXJhYmlkb3BzaXMgQm9vay9BbWVyaWNhbiBTb2NpZXR5IG9mIFBsYW50IEJpb2xvZ2lzdHMgLCBWb2wuIDEuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJMQVNUOiBhIG1vcmUgZWZmaWNpZW50IHJlcG9ydCB3aXRoIHVzYWJpbGl0eSBpbXByb3ZlbWVudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9VzI5LTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24gb2YgZ2Vub21lIHNpemUgaW4gQnJhc3NpY2FjZWFlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm4uIEJvdDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT05NTsgIGNpdGF0aW9uX3BhZ2VzPTIyOS0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmaWNpZW50IGRlIG5vdm8gYXNzZW1ibHkgb2YgaGlnaGx5IGhldGVyb3p5Z291cyBnZW5vbWVzIGZyb20gd2hvbGUtZ2Vub21lIHNob3RndW4gc2hvcnQgcmVhZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xMzg0LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BQnlTUzogYSBwYXJhbGxlbCBhc3NlbWJsZXIgZm9yIHNob3J0IHJlYWQgc2VxdWVuY2UgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTExMTctMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNPQVBkZW5vdm8yOiBhbiBlbXBpcmljYWxseSBpbXByb3ZlZCBtZW1vcnktZWZmaWNpZW50IHNob3J0LXJlYWQgZGUgbm92byBhc3NlbWJsZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdpZ2FzY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTE4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R01BUDogYSBnZW5vbWljIG1hcHBpbmcgYW5kIGFsaWdubWVudCBwcm9ncmFtIGZvciBtUk5BIGFuZCBFU1Qgc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTE4NTktNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBlcmZvcm1hbmNlIG9mIGdlbmV0aWMgcHJvZ3JhbW1pbmcgb3B0aW1pc2VkIEJvd3RpZTIgb24gZ2Vub21lIGNvbXBhcmlzb24gYW5kIGFuYWx5dGljIHRlc3RpbmcgKEdDQVQpIGJlbmNobWFya3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb0RhdGEgTWluOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJpbWVyM1BsdXMsIGFuIGVuaGFuY2VkIHdlYiBpbnRlcmZhY2UgdG8gUHJpbWVyMzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz1XNzEtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXV0b21hdGljIGNsdXN0ZXJpbmcgb2Ygb3J0aG9sb2dzIGFuZCBpbi1wYXJhbG9ncyBmcm9tIHBhaXJ3aXNlIHNwZWNpZXMgY29tcGFyaXNvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0zMTQ7ICBjaXRhdGlvbl9wYWdlcz0xMDQxLTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NQUZGVDogYSBub3ZlbCBtZXRob2QgZm9yIHJhcGlkIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCBiYXNlZCBvbiBmYXN0IEZvdXJpZXIgdHJhbnNmb3JtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTMwNTktNjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBBTUwgNDogcGh5bG9nZW5ldGljIGFuYWx5c2lzIGJ5IG1heGltdW0gbGlrZWxpaG9vZDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MTU4Ni05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGV2b2x1dGlvbmFyeSBmYXRlIGFuZCBjb25zZXF1ZW5jZXMgb2YgZHVwbGljYXRlIGdlbmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTI5MDsgIGNpdGF0aW9uX3BhZ2VzPTExNTEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWFybHkgZXZvbHV0aW9uYXJ5IGhpc3RvcnkgYW5kIGdlbm9taWMgZmVhdHVyZXMgb2YgZ2VuZSBkdXBsaWNhdGVzIGluIHRoZSBodW1hbiBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTYyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZlcnkgbG93IGdlbmUgZHVwbGljYXRpb24gcmF0ZSBpbiB0aGUgeWVhc3QgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMwNjsgIGNpdGF0aW9uX3BhZ2VzPTEzNjctNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyb3BsZXQgRGlnaXRhbCBQQ1IgcXVhbnRpdGF0aW9uIG9mIEhFUjIgZXhwcmVzc2lvbiBpbiBGRlBFIGJyZWFzdCBjYW5jZXIgc2FtcGxlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT01OTsgIGNpdGF0aW9uX3BhZ2VzPVMyMC0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ud28gZ2VuZXMgZW5jb2RpbmcgQXJhYmlkb3BzaXMgaGFsbGVyaSBNVFAxIG1ldGFsIHRyYW5zcG9ydCBwcm90ZWlucyBjby1zZWdyZWdhdGUgd2l0aCB6aW5jIHRvbGVyYW5jZSBhbmQgYWNjb3VudCBmb3IgaGlnaCBNVFAxIHRyYW5zY3JpcHQgbGV2ZWxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9NDI1LTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBzaGFyZWQgc2luZ2xlIGNvcHkgbnVjbGVhciBnZW5lcyBpbiBBcmFiaWRvcHNpcywgUG9wdWx1cywgVml0aXMgYW5kIE9yeXphIGFuZCB0aGVpciBwaHlsb2dlbmV0aWMgdXRpbGl0eSBhY3Jvc3MgdmFyaW91cyB0YXhvbm9taWMgbGV2ZWxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgRXZvbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTYxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5mZXJyaW5nIGV2b2x1dGlvbmFyeSB0cmVlcyB3aXRoIFBBVVAqOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBQcm90b2MuIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbmVpZ2hib3Itam9pbmluZyBtZXRob2Q6IGEgbmV3IG1ldGhvZCBmb3IgcmVjb25zdHJ1Y3RpbmcgcGh5bG9nZW5ldGljIHRyZWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTQwNi0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbmV0aWMgYmFzaXMgb2YgemluYyB0b2xlcmFuY2UgaW4gdGhlIG1ldGFsbG9waHl0ZSBBcmFiaWRvcHNpcyBoYWxsZXJpIHNzcC4gaGFsbGVyaSAoQnJhc3NpY2FjZWFlKTogYW4gYW5hbHlzaXMgb2YgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE3NjsgIGNpdGF0aW9uX3BhZ2VzPTY1OS03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBhcmNoaXRlY3R1cmUgb2YgemluYyBoeXBlcmFjY3VtdWxhdGlvbiBpbiBBcmFiaWRvcHNpcyBoYWxsZXJpOiB0aGUgZXNzZW50aWFsIHJvbGUgb2YgUVRMIHggZW52aXJvbm1lbnQgaW50ZXJhY3Rpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OZXcgUGh5dG9sOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE4NzsgIGNpdGF0aW9uX3BhZ2VzPTM1NS02NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9WW91bmcgcHJvdGVpbnMgZXhwZXJpZW5jZSBtb3JlIHZhcmlhYmxlIHNlbGVjdGlvbiBwcmVzc3VyZXMgdGhhbiBvbGQgcHJvdGVpbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0xNTc0LTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EdXBsaWNhdGVkIGdlbmVzIGV2b2x2ZSBzbG93ZXIgdGhhbiBzaW5nbGV0b25zIGRlc3BpdGUgdGhlIGluaXRpYWwgcmF0ZSBpbmNyZWFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEV2b2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW52ZXJzZSByZWxhdGlvbnNoaXAgYmV0d2VlbiBldm9sdXRpb25hcnkgcmF0ZSBhbmQgYWdlIG9mIG1hbW1hbGlhbiBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9NTk4LTYwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHVuaXZlcnNhbCBkaXN0cmlidXRpb24gb2YgZXZvbHV0aW9uYXJ5IHJhdGVzIG9mIGdlbmVzIGFuZCBkaXN0aW5jdCBjaGFyYWN0ZXJpc3RpY3Mgb2YgZXVrYXJ5b3RpYyBnZW5lcyBvZiBkaWZmZXJlbnQgYXBwYXJlbnQgYWdlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLiBTLiBBOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwNjsgIGNpdGF0aW9uX3BhZ2VzPTcyNzMtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWZlcmVudGlhbCBkdXBsaWNhdGlvbiBvZiBjb25zZXJ2ZWQgcHJvdGVpbnMgaW4gZXVrYXJ5b3RpYyBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPUU1NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhdGUgb2YgcHJvdGVpbiBldm9sdXRpb24gdmVyc3VzIGZpdG5lc3MgZWZmZWN0IG9mIGdlbmUgZGVsZXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTc3Mi00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24gb2YgdmVydGVicmF0ZSB0aXNzdWVzIGRyaXZlbiBieSBkaWZmZXJlbnRpYWwgbW9kZXMgb2YgZ2VuZSBkdXBsaWNhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTMwNS0xNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29yZWd1bGF0aW9uIG9mIHRhbmRlbSBkdXBsaWNhdGUgZ2VuZXMgc2xvd3MgZXZvbHV0aW9uIG9mIHN1YmZ1bmN0aW9uYWxpemF0aW9uIGluIG1hbW1hbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MzUyOyAgY2l0YXRpb25fcGFnZXM9MTAwOS0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFBoeXNjb21pdHJlbGxhIGdlbm9tZSByZXZlYWxzIGV2b2x1dGlvbmFyeSBpbnNpZ2h0cyBpbnRvIHRoZSBjb25xdWVzdCBvZiBsYW5kIGJ5IHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zMTk7ICBjaXRhdGlvbl9wYWdlcz02NC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY29ldm9sdXRpb24gb2YgZ2FsYXhpZXMgYW5kIHN1cGVybWFzc2l2ZSBibGFjayBob2xlczogYSBsb2NhbCBwZXJzcGVjdGl2ZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzM7ICBjaXRhdGlvbl9wYWdlcz0xODItNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBvZiBibGFjayBjb3R0b253b29kLCBQb3B1bHVzIHRyaWNob2NhcnBhIChUb3JyLiAmYW1wOyBHcmF5KTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zMTM7ICBjaXRhdGlvbl9wYWdlcz0xNTk2LTYwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGF0ZSBvZiB0aGUgbW9ub2NvdC1kaWNvdCBkaXZlcmdlbmNlIGVzdGltYXRlZCBmcm9tIGNobG9yb3BsYXN0IEROQSBzZXF1ZW5jZSBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUuIFMuIEE7ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl92b2x1bWU9ODY7ICBjaXRhdGlvbl9wYWdlcz02MjAxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNS8zLzMyNy8yNTA3MjE0My9kc3kwMDUucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIExpbmVhZ2Utc3BlY2lmaWMgZHVwbGljYXRlZCBnZW5lcyBsaWtlbHkgY29udHJpYnV0ZSB0byB0aGUgcGhlbm90eXBpYyBkaXZlcmdlbmNlIGluIGNsb3NlbHkgcmVsYXRlZCBzcGVjaWVzLiBIb3dldmVyLCBuZWl0aGVyIHRoZSBmcmVxdWVuY3kgb2YgZHVwbGljIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00ODk4MTI4JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZMt2OPwH/xOsqbU1gcrG6hBtpa+a" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f3%2f327%2f4898128" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f3%2f327%2f4898128" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=109141" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/3/1/m_dnares_25_3cover.png?Expires=2147483647&amp;Signature=zOskx939n215x6LsaNaPxmag9A6Tt6e7bP~Kt5lfxwaH8FjX1dKLUJ0mdj-Sm0uq9hmCpw3OAnAQFLS-GTY4CsXXX4aRhm5J1IbPv51CcTKXj828y94I4tx-BuFB-EA4WwEvEot4P8hDcnFzu~xWV52k2Sm6IHcCABqAv5wSWve6oMrdsQXhpN0srJqhN5UIcLjzws~g9qv24XhgtRb~Nw7X9IiEo1EM5V47od3bkv5yHlkUT1xmh3tJ4-ZjHw-HyAXqIp8eeHds4Cfjy9xqh8e3n5ynt516a1ZMF0hjA3wYH5X0-y3lqTHnWmgUvQv1L6dLXdRWhki~KqxTetYsJQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/3">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="118057098">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057098">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057100">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057100">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057111">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057111">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057132">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057132">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057166">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057166">4. Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057168">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057168">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057174">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057174">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/3/317/4898127">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/3/341/4934930">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Functional divergence of duplicate genes several million years after gene duplication in <em>Arabidopsis</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kousuke Hanada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Bioscience and Bioinformatics, Frontier Research Academy for Young Researchers, Kyusyu Institute of Technology, Iizuka, Fukuoka 820-8502, Japan</div><div class="aff">RIKEN Center for Sustainable Resource Science, RIKEN, Yokohama, Kanagawa 230-0045, Japan</div><div class="aff">CREST, Japan Science and Technology Agency, Kawaguchi, Saitama 332-0012, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy005-cor1" data-open="dsy005-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +81 948 29 7842. Fax. +81 948 29 7801. Email: <a href="mailto:kohanada@bio.kyutech.ac.jp" target="_blank">kohanada@bio.kyutech.ac.jp</a> (K.H.); Tel. +81 466 84 3724. Fax. +81 466 84 3724. Email: <a href="mailto:morinaga.shinichi@nihon-u.ac.jp" target="_blank">morinaga.shinichi@nihon-u.ac.jp</a> (S.-I.M.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kousuke+Hanada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hanada K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hanada K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kousuke Hanada</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ayumi Tezuka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Bioscience and Bioinformatics, Frontier Research Academy for Young Researchers, Kyusyu Institute of Technology, Iizuka, Fukuoka 820-8502, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ayumi+Tezuka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tezuka A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tezuka A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ayumi Tezuka</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masafumi Nozawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Center for Information Biology, National Institute of Genetics, Mishima, Shizuoka 411-8540, Japan</div><div class="aff">Department of Genetics, SOKENDAI, Mishima, Shizuoka 411-8540, Japan</div><div class="aff">Department of Biological Sciences, Tokyo Metropolitan University, Hachioji, Tokyo 192-0397, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masafumi+Nozawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nozawa M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nozawa M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masafumi Nozawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yutaka Suzuki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Frontier Science, The University of Tokyo, Kashiwa, Chiba 277-8562, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yutaka+Suzuki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suzuki Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suzuki Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yutaka Suzuki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sumio Sugano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Frontier Science, The University of Tokyo, Kashiwa, Chiba 277-8562, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sumio+Sugano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sugano S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sugano S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sumio Sugano</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Atsushi J Nagano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">CREST, Japan Science and Technology Agency, Kawaguchi, Saitama 332-0012, Japan</div><div class="aff">Center of Ecological Research, Kyoto University, Hirano, Otsu, Shiga 520-2113, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Atsushi+J+Nagano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nagano A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nagano A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Atsushi J Nagano</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Motomi Ito</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Arts and Sciences, The University of Tokyo, Tokyo 153-8902, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Motomi+Ito">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ito M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ito M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Motomi Ito</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shin-Ichi Morinaga</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">CREST, Japan Science and Technology Agency, Kawaguchi, Saitama 332-0012, Japan</div><div class="aff">Graduate School of Arts and Sciences, The University of Tokyo, Tokyo 153-8902, Japan</div><div class="aff">College of Bioresource Sciences, Nihon University, Fujisawa, Kanagawa 252-0880, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy005-cor1" data-open="dsy005-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +81 948 29 7842. Fax. +81 948 29 7801. Email: <a href="mailto:kohanada@bio.kyutech.ac.jp" target="_blank">kohanada@bio.kyutech.ac.jp</a> (K.H.); Tel. +81 466 84 3724. Fax. +81 466 84 3724. Email: <a href="mailto:morinaga.shinichi@nihon-u.ac.jp" target="_blank">morinaga.shinichi@nihon-u.ac.jp</a> (S.-I.M.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shin-Ichi+Morinaga">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Morinaga S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Morinaga S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shin-Ichi Morinaga</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy005-FM1"><div class="fn"><div class="footnote-content">Kousuke Hanada and Ayumi Tezuka authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy005-FM2"><div class="fn"><div class="footnote-content">Edited by Prof. Kazuhiro Sato</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 3, 1 June 2018, Pages 327–339, <a href='https://doi.org/10.1093/dnares/dsy005'>https://doi.org/10.1093/dnares/dsy005</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">21 February 2018</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">25 July 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">02 February 2018</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4898128" href="/dnaresearch/article-pdf/25/3/327/25072143/dsy005.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Kousuke Hanada, Ayumi Tezuka, Masafumi Nozawa, Yutaka Suzuki, Sumio Sugano, Atsushi J Nagano, Motomi Ito, Shin-Ichi Morinaga;  Functional divergence of duplicate genes several million years after gene duplication in <em>Arabidopsis</em>, <em>DNA Research</em>, Volume 25, Issue 3, 1 June 2018, Pages 327–339, <a href='https://doi.org/10.1093/dnares/dsy005'>https://doi.org/10.1093/dnares/dsy005</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4898128?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4898128?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4898128?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4898128?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4898128?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4898128?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4898128?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Functional%20divergence%20of%20duplicate%20genes%20several%20million%20years%20after%20gene%20duplication%20in%20%26lt%3Bem%26gt%3BArabidopsis%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2018-02-21&amp;volumeNum=25&amp;issueNum=3&amp;author=Hanada%2C%20Kousuke%3B%20Tezuka%2C%20Ayumi&amp;startPage=327&amp;endPage=339&amp;contentId=10.1093%2Fdnares%2Fdsy005&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f3%2f327%2f4898128" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=118057098 id="118057098" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Lineage-specific duplicated genes likely contribute to the phenotypic divergence in closely related species. However, neither the frequency of duplication events nor the degree of selection pressures immediately after gene duplication is clear in the speciation process. Here, using Illumina DNA-sequencing reads from <em>Arabidopsis halleri</em>, which has multiple closely related species with high-quality genome assemblies (<em>A. thaliana</em> and <em>A. lyrata</em>), we succeeded in generating orthologous gene groups in <em>Brassicaceae</em>. The duplication frequency of retained genes in the <em>Arabidopsis</em> lineage was ∼10 times higher than the duplication frequency inferred by comparative genomics of <em>Arabidopsis</em>, poplar, rice and moss (Physcomitrella patens). The difference of duplication frequencies can be explained by a rapid decay of anciently duplicated genes. To examine the degree of selection pressure on genes duplicated in either the <em>A. halleri-lyrata</em> or the <em>A. halleri</em> lineage, we examined positive and purifying selection in the <em>A. halleri-lyrata</em> and <em>A. halleri</em> lineages throughout the ratios of nonsynonymous to synonymous substitution rates (<em>K</em><sub>A</sub>/<em>K</em><sub>S</sub>). Duplicate genes tended to have a higher proportion of positive selection compared with non-duplicated genes. Interestingly, we found that functional divergence of duplicated genes was accelerated several million years after gene duplication compared with immediately after gene duplication.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;plant evolution&quot;">plant evolution</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene duplication&quot;">gene duplication</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Arabidopsis"><em>Arabidopsis</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;selection pressure&quot;">selection pressure</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;functional divergence&quot;">functional divergence</a></div>                            <h2 scrollto-destination=118057100 id="118057100" class="section-title" >1. Introduction</h2>
<p>Plant genomes have experienced more gene duplication events than most other eukaryotes.<a reveal-id="dsy005-B1" data-open="dsy005-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> These duplications are largely classified into whole genome duplications (WGDs) and small-scale duplications (SSDs). WGD events are concentrated in the Cretaceous-Paleogene extinction period.<a reveal-id="dsy005-B2" data-open="dsy005-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> SSDs such as retroduplication and tandem duplication have occurred continuously at a high rate during plant evolution, indicating that SSDs may be a key factor for plant speciation or phenotypic differences in close relatives.<a reveal-id="dsy005-B3 dsy005-B4 dsy005-B5 dsy005-B6 dsy005-B7 dsy005-B8" data-open="dsy005-B3 dsy005-B4 dsy005-B5 dsy005-B6 dsy005-B7 dsy005-B8" class="link link-ref link-reveal xref-bibr"><sup>3–8</sup></a> SSDs tend to be lineage-specific in land plants.<a reveal-id="dsy005-B4" data-open="dsy005-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> There is a clear functional bias between WGD and SSD in plants.<a reveal-id="dsy005-B4" data-open="dsy005-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsy005-B9" data-open="dsy005-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> SSDs tend to be associated with stress responses,<a reveal-id="dsy005-B4" data-open="dsy005-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsy005-B8 dsy005-B9 dsy005-B10" data-open="dsy005-B8 dsy005-B9 dsy005-B10" class="link link-ref link-reveal xref-bibr"><sup>8–10</sup></a> and are likely important for adaptive evolution to rapidly changing environments in close relatives.</p><p>SSD genes functionally diverge through either neo-functionalization or sub-functionalization, which may contribute to speciation.<a reveal-id="dsy005-B11" data-open="dsy005-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsy005-B12" data-open="dsy005-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> SSD genes can also retain functional redundancy through the long-term fate of becoming pseudogenes,<a reveal-id="dsy005-B13" data-open="dsy005-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsy005-B14" data-open="dsy005-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> dosage selection<a reveal-id="dsy005-B15" data-open="dsy005-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsy005-B16" data-open="dsy005-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> or avoiding developmental errors.<a reveal-id="dsy005-B17 dsy005-B18 dsy005-B19" data-open="dsy005-B17 dsy005-B18 dsy005-B19" class="link link-ref link-reveal xref-bibr"><sup>17–19</sup></a> Functional divergence of duplicated genes can be inferred by selection pressures based on the non-synonymous substitution rate (<em>K</em><sub>A</sub>) versus the synonymous substitution rate (<em>K</em><sub>S</sub>). Genes undergoing positive selection (<em>K</em><sub>A</sub>/<em>K</em><sub>S</sub> &gt; 1) and purifying selection (<em>K</em><sub>A</sub>/<em>K</em><sub>S</sub> &lt; 1) are associated with functional divergence and constraint, respectively.<a reveal-id="dsy005-B20" data-open="dsy005-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Most earlier studies compared selection pressures in WGDs with those in SSDs, or selection pressures in anciently duplicated genes (several hundred million years [MY] ago) in plants.<a reveal-id="dsy005-B21 dsy005-B22 dsy005-B23 dsy005-B24" data-open="dsy005-B21 dsy005-B22 dsy005-B23 dsy005-B24" class="link link-ref link-reveal xref-bibr"><sup>21–24</sup></a> There are little data with which to examine selection pressures in recent SSDs in plants. Because selection pressure likely varies during the speciation process, it is important to examine the selection pressures immediately after gene duplications among close relatives.</p><p>Recently, there are many plant genomes assembled by Illumina DNA-sequencing reads.<a reveal-id="dsy005-B25" data-open="dsy005-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> However, SSDs tend to be underestimated in contigs generated by only Illumina DNA-sequencing.<a reveal-id="dsy005-B26" data-open="dsy005-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a><sup>,</sup><a reveal-id="dsy005-B27" data-open="dsy005-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> It is likely that genomes generated by BAC are useful to examine SSDs. However, construction of such a library takes much budget and time to generate highly qualified genomes. In the present study, we tried to generate reliable orthologous genes by only paired-end Illumina DNA-sequencing reads using available genomes of close relatives. We then inferred SSDs in a species after the divergence of closed species throughout the reading depth of Illumina DNA sequencing.</p><p>The <em>Arabidopsis</em> lineage seems to be the best lineage to examine recent SSDs in plants because <em>Arabidopsis</em> has two BAC library-based genomes in <em>Arabidopsis thaliana</em> and <em>Arabidopsis lyrata.</em><a reveal-id="dsy005-B28" data-open="dsy005-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a><sup>,</sup><a reveal-id="dsy005-B29" data-open="dsy005-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> In particular, <em>A. thaliana</em> has a large amount of functional data for its annotated genes. To comprehensively examine SSDs in <em>Arabidopsis</em>, we investigated an additional <em>Arabidopsis</em> species—<em>A. halleri</em>. The divergence time between <em>A. lyrata</em> and <em>A. halleri</em> is supposed to be ∼2 MYA,<a reveal-id="dsy005-B30" data-open="dsy005-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> and the divergence time between either <em>A. halleri</em> or <em>A. lyrata</em> and <em>A. thaliana</em> is ∼10–11 MYA.<a reveal-id="dsy005-B31" data-open="dsy005-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> Among recently diverged three <em>Arabidopsis</em> species, there are phenotypic variations such as self-compatibility and heavy-metal tolerance. <em>A. thaliana</em> has self-compatibility, but the others are self-incompatible.<a reveal-id="dsy005-B32 dsy005-B33 dsy005-B34" data-open="dsy005-B32 dsy005-B33 dsy005-B34" class="link link-ref link-reveal xref-bibr"><sup>32–34</sup></a> Therefore, <em>A. halleri</em> and <em>A. lyrata</em> have large petals to attract pollinator insects, and the anthers are separated from the stigma to avoid autopollination.<a reveal-id="dsy005-B35" data-open="dsy005-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a><sup>,</sup><em>A. halleri</em> is known as a Zn/Cd hyper-accumulator<a reveal-id="dsy005-B33" data-open="dsy005-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a><sup>,</sup><a reveal-id="dsy005-B34" data-open="dsy005-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> and some wild populations of <em>A. lyrata</em> are tolerant of serpentine soils, which are characterized by a high heavy-metal content,<a reveal-id="dsy005-B36" data-open="dsy005-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> while <em>A. thaliana</em> is not.</p><p><em>A. halleri</em> has highly variable morphologies with respect to leaf, flower colour and development of stolons among the subspecies. To examine SSDs in the <em>Arabidopsis</em> lineage, we focused on <em>A. halleri</em> subsp<em>. gemmifera</em> which grows in the Russian Far East, northeastern China, Korea, Taiwan and Japan across lowland and highland areas. We collected the plants in Mt. Ibuki, Shiga, Japan. We then extracted the plant DNAs, and performed the paired-end Illumina DNA sequencing. After generating contigs from the Illumina DNA-sequencing reads, we inferred <em>A. halleri</em> orthologous genes against the <em>A. thaliana</em> and <em>A. lyrata</em> genes. The number of SSDs was inferred both from phylogenetic analysis of orthologous genes and the reading depth of Illumina DNA sequencing. As outgroup species for the three <em>Arabidopsis</em> species, we used five non-<em>Arabidopsis</em> species in <em>Brassicaceae</em> for which genome sequences are already determined. There is trans-specific polymorphism in <em>Arabidopsis</em>; in particular, some genes have undergone gene flow in <em>Arabidopsis</em>.<a reveal-id="dsy005-B37" data-open="dsy005-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> Here, excluding genes that have undergone gene flow in <em>Arabidopsis</em>, we generated three sets of orthologous gene groups (OGGs) among five non-<em>Arabidopsis</em> species, <em>A. thaliana</em>, <em>A. lyrata</em> and <em>A. halleri</em> in Brassicaceae (<a reveal-id="dsy005-F1" data-open="dsy005-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). Each OGG represent genes derived from one ancestral gene in each speciation event. By identifying the genes in each OGG, we examined the evolutionary process in the <em>Arabidopsis</em> lineage. Although a BAC library-based genome for <em>A. halleri</em> is unavailable, there is an available <em>A. halleri</em> genome based on long-insert mate paired reads and short-insert paired-end reads in Illumina DNA sequencing.<a reveal-id="dsy005-B38" data-open="dsy005-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a><sup>,</sup><a reveal-id="dsy005-B39" data-open="dsy005-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> The available <em>A. halleri</em> genome was originated from <em>A. halleri</em> subsp<em>. gemmifera</em> which was the same subspecies in our plant material. The collection site was in Tada mine, Hyogo, Japan. Microsatellite analyses suggested that our plant material collected from Mt. Ibuki, Shiga, Japan was genetically closed to the plant with the available genome.<a reveal-id="dsy005-B40" data-open="dsy005-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> Thus, it is likely that <em>A. halleri</em> genes inferred from the available genome are similar to our inferred <em>A. halleri</em> genes. To validate our overall results, we also examined SSDs based on the available <em>A. halleri</em> genome, </p>                        <a id="118057106" scrollto-destination="118057106"></a>
<div id="dsy005-F1" data-id="dsy005-F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/m_dsy005f1.png?Expires=2147483647&Signature=TUdVHeBP91NDGkzlDhRsceKDvs2o8K9zqLBni4izRnKVLIMrher5pyJ3Auv3lrWn~EuUcfXcD4dlzNDufz7MhnqyRsxSgvMfvLnDlSo-230suFJHBht9Pt1ggPPcRG1dA8tYkQCrebHpSNRmuImokBa24Esj3lcF-hREyOjK1DCQwFE-CKqrmR9BGT6keUIl1wjYWSCcdyrqLHLXvL36gda6r7diFuga0sDzsyWRcU2W~Dnofb3bfzO8pNVA7GcEzASaapIkA~e4c4zdYPFUMBdi5yhCygsnDihugRWs-EBKq36Ohm9POQI5T-AdfjEdxN2RjW0~P2cb9pNnFN6vtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Three sets of OGGs among non-Arabidopsis species, A. thaliana, A. lyrata and A. halleri. OGGs between A. lyrata and A. halleri were defined as AL–AH OGGs. There were 25,833, 26,428 and 26,007 AL–AH OGGs based on Illumina paired-end DNA-sequencing reads without any pseudogene-like genes, Illumina paired-end DNA-sequencing reads including pseudogene-like genes and the available A. halleri genome, respectively. OGGs among A. thaliana, A. lyrata and A. halleri were defined as AT–AL–AH OGGs. There were 22,105, 22,684 and 21,537 AT–AL–AH OGGs based on Illumina paired-end DNA-sequencing reads without any pseudogene-like genes, Illumina paired-end DNA-sequencing reads including pseudogene-like genes and the available A. halleri genome, respectively. OGGs among non-Arabidopsis species (B. rapa, B. stricta, C. grandiflora, C. rubella, E. salsugineum), A. thaliana, A. lyrata and A. halleri were defined as nonA-AT–AL–AH OGGs. There were 17,669 and 17,925 non-A-AT–AL–AH OGGs based on Illumina paired-end DNA-sequencing reads without any pseudogene-like genes and the available A. halleri genome, respectively." path-from-xml="dsy005f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057106/dsy005f1.tif" path-from-xml="dsy005f1.tif" target="_blank">View large</a><a section="118057106" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005f1.png?Expires=2147483647&Signature=ZtBE2wZ42cIljGZ2X~86bsvGPsdpoRde~D0GFBuW4MR0f20shMQytcbK2f8b6fWpOH1aqwVIBOhUg-VDpHzKa49AhPskshn3MhBRCM4NlfcMOq7v1ZZDIz9h~rof86WcvvaDZW4Hqg4bzzhuN8K3NEj-nIzb4YWqp5VZq7CjfYlJoRXugL2e9XJ3Xfz~I2bPB9nba5Iw3Fb8IaqAh8z63g1Q2FpT5cyQY4cIpglI~ctcO5A7rsRdj2Uu9iIS5UkCo7RgPpb7Z-kFH70H4QhI7pYRFOI3zvGt~R97447~yaHswr8xPehyeVbWAukYb7xjuhSwkXGbNf42fc~VaR5mDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057106&ar=4898128&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy005f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Three sets of OGGs among non-<em>Arabidopsis</em> species, <em>A. thaliana</em>, <em>A. lyrata</em> and <em>A. halleri.</em> OGGs between <em>A. lyrata</em> and <em>A. halleri</em> were defined as <em>AL–AH</em> OGGs. There were 25,833, 26,428 and 26,007 <em>AL–AH</em> OGGs based on Illumina paired-end DNA-sequencing reads without any pseudogene-like genes, Illumina paired-end DNA-sequencing reads including pseudogene-like genes and the available <em>A. halleri</em> genome, respectively. OGGs among <em>A. thaliana</em>, <em>A. lyrata</em> and <em>A. halleri</em> were defined as <em>AT–AL–AH</em> OGGs. There were 22,105, 22,684 and 21,537 <em>AT–AL–AH</em> OGGs based on Illumina paired-end DNA-sequencing reads without any pseudogene-like genes, Illumina paired-end DNA-sequencing reads including pseudogene-like genes and the available <em>A. halleri</em> genome, respectively. OGGs among non-<em>Arabidopsis</em> species (<em>B. rapa</em>, <em>B. stricta</em>, <em>C. grandiflora</em>, <em>C. rubella</em>, <em>E. salsugineum</em>), <em>A. thaliana</em>, <em>A. lyrata</em> and <em>A. halleri</em> were defined as <em>nonA-AT–AL–AH</em> OGGs. There were 17,669 and 17,925 <em>non-A-AT–AL–AH</em> OGGs based on Illumina paired-end DNA-sequencing reads without any pseudogene-like genes and the available <em>A. halleri</em> genome, respectively.</p></div></div><div content-id="dsy005-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/m_dsy005f1.png?Expires=2147483647&Signature=TUdVHeBP91NDGkzlDhRsceKDvs2o8K9zqLBni4izRnKVLIMrher5pyJ3Auv3lrWn~EuUcfXcD4dlzNDufz7MhnqyRsxSgvMfvLnDlSo-230suFJHBht9Pt1ggPPcRG1dA8tYkQCrebHpSNRmuImokBa24Esj3lcF-hREyOjK1DCQwFE-CKqrmR9BGT6keUIl1wjYWSCcdyrqLHLXvL36gda6r7diFuga0sDzsyWRcU2W~Dnofb3bfzO8pNVA7GcEzASaapIkA~e4c4zdYPFUMBdi5yhCygsnDihugRWs-EBKq36Ohm9POQI5T-AdfjEdxN2RjW0~P2cb9pNnFN6vtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Three sets of OGGs among non-Arabidopsis species, A. thaliana, A. lyrata and A. halleri. OGGs between A. lyrata and A. halleri were defined as AL–AH OGGs. There were 25,833, 26,428 and 26,007 AL–AH OGGs based on Illumina paired-end DNA-sequencing reads without any pseudogene-like genes, Illumina paired-end DNA-sequencing reads including pseudogene-like genes and the available A. halleri genome, respectively. OGGs among A. thaliana, A. lyrata and A. halleri were defined as AT–AL–AH OGGs. There were 22,105, 22,684 and 21,537 AT–AL–AH OGGs based on Illumina paired-end DNA-sequencing reads without any pseudogene-like genes, Illumina paired-end DNA-sequencing reads including pseudogene-like genes and the available A. halleri genome, respectively. OGGs among non-Arabidopsis species (B. rapa, B. stricta, C. grandiflora, C. rubella, E. salsugineum), A. thaliana, A. lyrata and A. halleri were defined as nonA-AT–AL–AH OGGs. There were 17,669 and 17,925 non-A-AT–AL–AH OGGs based on Illumina paired-end DNA-sequencing reads without any pseudogene-like genes and the available A. halleri genome, respectively." path-from-xml="dsy005f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057106/dsy005f1.tif" path-from-xml="dsy005f1.tif" target="_blank">View large</a><a section="118057106" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005f1.png?Expires=2147483647&Signature=ZtBE2wZ42cIljGZ2X~86bsvGPsdpoRde~D0GFBuW4MR0f20shMQytcbK2f8b6fWpOH1aqwVIBOhUg-VDpHzKa49AhPskshn3MhBRCM4NlfcMOq7v1ZZDIz9h~rof86WcvvaDZW4Hqg4bzzhuN8K3NEj-nIzb4YWqp5VZq7CjfYlJoRXugL2e9XJ3Xfz~I2bPB9nba5Iw3Fb8IaqAh8z63g1Q2FpT5cyQY4cIpglI~ctcO5A7rsRdj2Uu9iIS5UkCo7RgPpb7Z-kFH70H4QhI7pYRFOI3zvGt~R97447~yaHswr8xPehyeVbWAukYb7xjuhSwkXGbNf42fc~VaR5mDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057106&ar=4898128&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy005f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Three sets of OGGs among non-<em>Arabidopsis</em> species, <em>A. thaliana</em>, <em>A. lyrata</em> and <em>A. halleri.</em> OGGs between <em>A. lyrata</em> and <em>A. halleri</em> were defined as <em>AL–AH</em> OGGs. There were 25,833, 26,428 and 26,007 <em>AL–AH</em> OGGs based on Illumina paired-end DNA-sequencing reads without any pseudogene-like genes, Illumina paired-end DNA-sequencing reads including pseudogene-like genes and the available <em>A. halleri</em> genome, respectively. OGGs among <em>A. thaliana</em>, <em>A. lyrata</em> and <em>A. halleri</em> were defined as <em>AT–AL–AH</em> OGGs. There were 22,105, 22,684 and 21,537 <em>AT–AL–AH</em> OGGs based on Illumina paired-end DNA-sequencing reads without any pseudogene-like genes, Illumina paired-end DNA-sequencing reads including pseudogene-like genes and the available <em>A. halleri</em> genome, respectively. OGGs among non-<em>Arabidopsis</em> species (<em>B. rapa</em>, <em>B. stricta</em>, <em>C. grandiflora</em>, <em>C. rubella</em>, <em>E. salsugineum</em>), <em>A. thaliana</em>, <em>A. lyrata</em> and <em>A. halleri</em> were defined as <em>nonA-AT–AL–AH</em> OGGs. There were 17,669 and 17,925 <em>non-A-AT–AL–AH</em> OGGs based on Illumina paired-end DNA-sequencing reads without any pseudogene-like genes and the available <em>A. halleri</em> genome, respectively.</p></div></div><p>In a previous report, selection pressures of genes duplicated after the divergence of <em>A. thaliana</em> and <em>A. lyrata</em> were examined in the <em>A. thaliana</em> lineage.<a reveal-id="dsy005-B23" data-open="dsy005-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> These lineage-specific duplicated genes tend to have undergone positive selection. Here, we examined the selection pressures in duplicated genes in the other lineage after the split between <em>A. thaliana</em> and <em>A. lyrata</em>. This lineage was further split into the <em>A. halleri–lyrata</em> lineage and <em>A. halleri</em> lineage (<a reveal-id="dsy005-F2" data-open="dsy005-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). We then examined whether gene duplications in either the <em>A. halleri–lyrata</em> or <em>A. halleri</em> lineage contributed to functional divergence in the retained duplicate genes. </p>                        <a id="118057108" scrollto-destination="118057108"></a>
<div id="dsy005-F2" data-id="dsy005-F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/m_dsy005f2.png?Expires=2147483647&Signature=wqh-XhAUinhhGdifYqE64IZO1iV88mB-mkjPj-GpC~QXa~5tsGxSJyFgYBcIlVzt0PJeDvWHedAPmramWmZuU-HR~3~ljkHCvA3oKXI8Hi5ALDJW6IaN2bk3l98xsROxAD~lcpxHrSSbV00RX8eO0ec9U45ErF3T6fGIAYwhQvGZO38Azn-H2xsG1fLfoHBaw12koU05CKWZni2J6AEatXY6HOY8ylkpaCfIzQmCy8LliJqvyPDrT0j6-KLTfUck9GhpVEPk7QGpYHEOz3UxaUAwKOb5LAPur-BfH8tBJLQExbQJgKyc54IFuRWZK3dfXynj7uHiLU27ie1CxkXl8g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gain rates through gene duplication in land plants. The divergence times among moss (P. patens), rice (O. sativa), poplar (P. trichocarpa), B. rapa, A. thaliana, A. lyrata and A. halleri were taken from previous reports.30,31,72–75 Gene gains through gene duplication were estimated for each branch. The gene gains in two branches were estimated in this study. One was the A. halleri-lyrata lineage, which represents the evolutionary process of the A. halleri lineage between the divergence of A. thaliana and the divergence of A. lyrata. The other was the A. halleri lineage, which represents the evolutionary process of the A. halleri lineage after the divergence of A. lyrata. Gene gains were divided by ancestral gene numbers and branch lengths corresponding to times (MY). The rate of gene gain was defined as the number of genes gained through gene duplication per gene per MY." path-from-xml="dsy005f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057108/dsy005f2.tif" path-from-xml="dsy005f2.tif" target="_blank">View large</a><a section="118057108" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005f2.png?Expires=2147483647&Signature=bVspsvEvpADIAlxHpo8YojfVUn9M~L3Evmm7jJwLNJSgWzcx1hcT1k9vIv5H4d3UaSyeHjZ-VwEn-QuKEHQYBeGpcFafbfNcqp5Hm3hMWd-4P0-rpCBObwHPn3bdqPU274OKJA9Q8Cffa1bQ8w4XZIE9QCGF9X8Ae2Metm1qjKUdzgTtKyVLfi~ZR1mswa040XkNG16kd55OtopuqxD6cbitj3Z55qAq8cjZzCMDMpgos3fRIxpN4ZjpDTyUcRf-9cywsK3jW4u2qFwOJpMCQquE4~5Cf69IyJTKoPJyzWvWJRy2K0~TBnT12r56JNJ12uzNs-yoQ3yd6lMz8D7vYg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057108&ar=4898128&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy005f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gain rates through gene duplication in land plants. The divergence times among moss (<em>P. patens</em>), rice (<em>O. sativa</em>), poplar (<em>P. trichocarpa</em>), <em>B. rapa</em>, <em>A. thaliana</em>, <em>A. lyrata</em> and <em>A. halleri</em> were taken from previous reports.<a reveal-id="dsy005-B30" data-open="dsy005-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a><sup>,</sup><a reveal-id="dsy005-B31" data-open="dsy005-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a><sup>,</sup><a reveal-id="dsy005-B72 dsy005-B73 dsy005-B74 dsy005-B75" data-open="dsy005-B72 dsy005-B73 dsy005-B74 dsy005-B75" class="link link-ref link-reveal xref-bibr"><sup>72–75</sup></a> Gene gains through gene duplication were estimated for each branch. The gene gains in two branches were estimated in this study. One was the <em>A. halleri-lyrata</em> lineage, which represents the evolutionary process of the <em>A. halleri</em> lineage between the divergence of <em>A. thaliana</em> and the divergence of <em>A. lyrata</em>. The other was the <em>A. halleri</em> lineage, which represents the evolutionary process of the <em>A. halleri</em> lineage after the divergence of <em>A. lyrata</em>. Gene gains were divided by ancestral gene numbers and branch lengths corresponding to times (MY). The rate of gene gain was defined as the number of genes gained through gene duplication per gene per MY.</p></div></div><div content-id="dsy005-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/m_dsy005f2.png?Expires=2147483647&Signature=wqh-XhAUinhhGdifYqE64IZO1iV88mB-mkjPj-GpC~QXa~5tsGxSJyFgYBcIlVzt0PJeDvWHedAPmramWmZuU-HR~3~ljkHCvA3oKXI8Hi5ALDJW6IaN2bk3l98xsROxAD~lcpxHrSSbV00RX8eO0ec9U45ErF3T6fGIAYwhQvGZO38Azn-H2xsG1fLfoHBaw12koU05CKWZni2J6AEatXY6HOY8ylkpaCfIzQmCy8LliJqvyPDrT0j6-KLTfUck9GhpVEPk7QGpYHEOz3UxaUAwKOb5LAPur-BfH8tBJLQExbQJgKyc54IFuRWZK3dfXynj7uHiLU27ie1CxkXl8g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gain rates through gene duplication in land plants. The divergence times among moss (P. patens), rice (O. sativa), poplar (P. trichocarpa), B. rapa, A. thaliana, A. lyrata and A. halleri were taken from previous reports.30,31,72–75 Gene gains through gene duplication were estimated for each branch. The gene gains in two branches were estimated in this study. One was the A. halleri-lyrata lineage, which represents the evolutionary process of the A. halleri lineage between the divergence of A. thaliana and the divergence of A. lyrata. The other was the A. halleri lineage, which represents the evolutionary process of the A. halleri lineage after the divergence of A. lyrata. Gene gains were divided by ancestral gene numbers and branch lengths corresponding to times (MY). The rate of gene gain was defined as the number of genes gained through gene duplication per gene per MY." path-from-xml="dsy005f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057108/dsy005f2.tif" path-from-xml="dsy005f2.tif" target="_blank">View large</a><a section="118057108" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005f2.png?Expires=2147483647&Signature=bVspsvEvpADIAlxHpo8YojfVUn9M~L3Evmm7jJwLNJSgWzcx1hcT1k9vIv5H4d3UaSyeHjZ-VwEn-QuKEHQYBeGpcFafbfNcqp5Hm3hMWd-4P0-rpCBObwHPn3bdqPU274OKJA9Q8Cffa1bQ8w4XZIE9QCGF9X8Ae2Metm1qjKUdzgTtKyVLfi~ZR1mswa040XkNG16kd55OtopuqxD6cbitj3Z55qAq8cjZzCMDMpgos3fRIxpN4ZjpDTyUcRf-9cywsK3jW4u2qFwOJpMCQquE4~5Cf69IyJTKoPJyzWvWJRy2K0~TBnT12r56JNJ12uzNs-yoQ3yd6lMz8D7vYg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057108&ar=4898128&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy005f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gain rates through gene duplication in land plants. The divergence times among moss (<em>P. patens</em>), rice (<em>O. sativa</em>), poplar (<em>P. trichocarpa</em>), <em>B. rapa</em>, <em>A. thaliana</em>, <em>A. lyrata</em> and <em>A. halleri</em> were taken from previous reports.<a reveal-id="dsy005-B30" data-open="dsy005-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a><sup>,</sup><a reveal-id="dsy005-B31" data-open="dsy005-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a><sup>,</sup><a reveal-id="dsy005-B72 dsy005-B73 dsy005-B74 dsy005-B75" data-open="dsy005-B72 dsy005-B73 dsy005-B74 dsy005-B75" class="link link-ref link-reveal xref-bibr"><sup>72–75</sup></a> Gene gains through gene duplication were estimated for each branch. The gene gains in two branches were estimated in this study. One was the <em>A. halleri-lyrata</em> lineage, which represents the evolutionary process of the <em>A. halleri</em> lineage between the divergence of <em>A. thaliana</em> and the divergence of <em>A. lyrata</em>. The other was the <em>A. halleri</em> lineage, which represents the evolutionary process of the <em>A. halleri</em> lineage after the divergence of <em>A. lyrata</em>. Gene gains were divided by ancestral gene numbers and branch lengths corresponding to times (MY). The rate of gene gain was defined as the number of genes gained through gene duplication per gene per MY.</p></div></div><p>Duplicated genes may have undergone purifying selection in the <em>Arabidopsis</em> lineage. The gene dosage hypothesis proposes that duplicate genes can be fixed for increased gene dosage, keeping redundant functions among duplicated genes. In <em>A. halleri</em>, the expression of <em>HEAVY METAL ATPASE 4</em> (<em>HMA4</em>) has been enhanced by cis-regulatory mutations and increased gene copy number for metal hyperaccumulation.<a reveal-id="dsy005-B41" data-open="dsy005-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> This is an example of increased gene dosage by gene duplication. Consequently, duplicated genes with purifying selection tend to be associated with increased gene dosage.</p><p>To understand the contribution of duplicated genes to the speciation process in <em>Arabidopsis</em> within the last 10 MY, we first examined the frequency of gene duplications in the <em>A. halleri–lyrata</em> and <em>A. halleri</em> lineages. We then examined the relationship between gene duplication and selection pressure based on the <em>K</em><sub>A</sub>/<em>K</em><sub>S</sub> ratio. Finally, we examined the over-represented functional categories of genes under positive and purifying selection.</p>                            <h2 scrollto-destination=118057111 id="118057111" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=118057112 id="118057112" class="section-title" >2.1. Sampling and illumina paired-end DNA-sequencing analysis</h3>
<p><em>Arabidopsis halleri</em> subsp. <em>gemmifera</em> is one of the three subspecies of <em>A. halleri</em>, and grows in the Russian Far East, northeastern China, Korea, Taiwan and Japan.<a reveal-id="dsy005-B42" data-open="dsy005-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> In 2008, a leaf sample was collected from an individual of <em>A. halleri</em> subsp. <em>gemmifera</em> on Mt. Ibuki, Shiga, Japan. DNA was extracted from the leaf using a DNeasy Plant Mini Kit (QIAGEN, Venlo, The Netherlands). A 300-bp paired-end DNA library was constructed according to the paired-End Genomic DNA Sample Preparation Kit (Illumina, San Diego, CA, USA), and 93-bp paired-end reads were obtained from the Illumina GAIIx sequencer.</p><p>A total of 44.5 Gb reads were generated by Illumina DNA paired-end sequencing. Using the Trim Galore (<a class="link link-uri" href="http://www.bioinformatics.babraham.ac.uk" target="_blank">www.bioinformatics.babraham.ac.uk</a>) software, sequences with low-quality base calls (Phred score &lt; 20) were trimmed off from the 3ʹ end of the reads. Adapter sequences started with ‘AGATCGGAAGAGC’ were completely removed from the reads. Approximately 28% of reads had either low-quality scores or adapters and were trimmed by Trim Galore. When a paired-end read was completely removed by this procedure, the other read was used as a single-end read. Given that mitochondrial and chloroplast genomes have much higher copy numbers than the nuclear genome, sequencing reads mapped to either the mitochondrial or chloroplast genome in <em>A. thaliana</em> (<a class="link link-uri" href="https://www.arabidopsis.org" target="_blank">https://www.arabidopsis.org</a>, TAIR10) or <em>A. lyrata</em> (<a class="link link-uri" href="http://genome.jgi.doe.gov" target="_blank">http://genome.jgi.doe.gov</a>, FilteredModels6) by BLASTN version 2.2.29 (<em>e</em>-value &lt; 1.0) were excluded from the following procedures.<a reveal-id="dsy005-B43" data-open="dsy005-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> Assuming that the genome size of <em>A. halleri</em> was 255 Mb,<a reveal-id="dsy005-B44" data-open="dsy005-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> the sequencing coverage was estimated to be ∼135× (34.4/0.255 Gb).</p>                            <h3 scrollto-destination=118057115 id="118057115" class="section-title" >2.2. Gene sequences of <em>Brassica rapa, Boechera stricta</em>, <em>Capsella grandiflora</em>, <em>Capsella rubella</em>, <em>Eutrema salsugineum</em>, <em>A. thaliana</em> and <em>A. lyrata</em></h3>
<p>Gene sequences in <em>B. rapa</em> (BrapaFPsc_277_v1.3), <em>B. stricta</em> (Bstricta_278_v1.2), <em>C. grandiflora</em> (Cgrandiflora_266_v1.1), <em>C. rubella</em> (Crubella_183_v1.0) and <em>E. salsugineum</em> (Esalsugineum_173_v1.0) were collected from Phytozome (<a class="link link-uri" href="https://phytozome.jgi.doe.gov" target="_blank">https://phytozome.jgi.doe.gov</a>, version 10). Gene sequences in <em>A. thaliana</em> and <em>A. lyrata</em> were collected from TAIR (<a class="link link-uri" href="https://www.arabidopsis.org/" target="_blank">https://www.arabidopsis.org/</a>, version 10, TAIR10_cds_20110103) and JGI (<a class="link link-uri" href="http://genome.jgi.doe.gov" target="_blank">http://genome.jgi.doe.gov</a>, FilteredModels6), respectively.</p>                            <h3 scrollto-destination=118057117 id="118057117" class="section-title" >2.3. Assembly of <em>A. halleri</em> genes orthologous to <em>A. lyrata</em> genes based on illumina paired-end DNA-sequencing reads</h3>
<p>We first generated <em>A. halleri</em> contigs from a total of 44.5 Gb Illumina DNA-sequencing reads using several assembly software tools such as Platanus 1.2.4,<a reveal-id="dsy005-B45" data-open="dsy005-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> ABySS 1.5.2,<a reveal-id="dsy005-B46" data-open="dsy005-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> SOAP-denovo2<a reveal-id="dsy005-B47" data-open="dsy005-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> and CLC Genomics Workbench 7.0.3 (<a class="link link-uri" href="https://www.qiagenbioinformatics.com/" target="_blank">https://www.qiagenbioinformatics.com/</a>) software. A higher proportion of genes in closely related species tend to be mapped to reliable assembled sequences. As the closest species, 32,670 annotated <em>A. lyrata</em> genes were mapped to each type of assembled DNA segment by the gmap (version 2014-08-20) software with default parameters, which takes into account exon–intron junctions.<a reveal-id="dsy005-B48" data-open="dsy005-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> The <em>A. halleri</em> contigs generated by the ABySS software with 63 <em>K</em>-mer size (N50 size = 5,331 bp) had the highest mapping rate to the 32,670 annotated <em>A. lyrata</em> genes with &gt;80% coverage (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Some of the matched regions were truncated by terminal codons. When coding sequences up to the terminal codon had &gt;80% similarity to and 80% coverage for an <em>A. lyrata</em> gene, the coding sequence was defined as the orthologous <em>A. halleri</em> gene sequence to an <em>A. lyrata</em> gene. Following this procedure, we generated 22,727 orthologous gene pairs between <em>A. halleri</em> and <em>A. lyrata</em>. However, an <em>A. lyrata</em> gene sequence was frequently mapped to different <em>A. halleri</em> contigs depending on the region of the <em>A. lyrata</em> gene because the whole sequence of each <em>A. halleri</em> gene was split into several contigs. To further identify <em>A. halleri</em> genes orthologous to <em>A. lyrata</em> genes, we re-assembled the <em>A. halleri</em> contigs based on the mapping information of <em>A. lyrata</em> genes. To collect additional <em>A. halleri</em> genes orthologous to <em>A. lyrata</em>, unmapped <em>A. lyrata</em> genes were re-mapped to the assembled DNA segments by TBLASTN version 2.2.29 (<em>e</em>-value &lt; 1 × 10<sup>−5</sup>).<a reveal-id="dsy005-B43" data-open="dsy005-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> However, the contigs mapped separately by <em>A. lyrata</em> genes may not be orthologous syntenic regions. To focus on only syntenic contigs between <em>A. halleri</em> and <em>A. lyrata</em>, we defined syntenic contigs to which more than two <em>A. lyrata</em> genes within less than five spacer genes in <em>A. lyrata</em> genome were mapped. When an <em>A. lyrata</em> gene was separately mapped to two syntenic contigs between <em>A. lyrata</em> and <em>A</em>. <em>halleri</em>, the contigs were concatenated following the direction of the <em>A. lyrata</em> gene. The <em>A. lyrata</em> gene was then mapped to the concatenated contigs. We thus obtained an additional 3,106 <em>A. halleri</em> gene sequences with &gt;80% similarity and 80% coverage against <em>A. lyrata</em> genes. Finally, we succeeded in identifying 25,833 (79.1%) <em>A. halleri</em> genes orthologous to 32,670 <em>A. lyrata</em> genes. Given that all identified <em>A. halleri</em> genes were paired with <em>A. lyrata</em> genes, a total of 25,833 pairs were defined as <em>A. lyrata–A. halleri</em> (<em>AL</em>–<em>AH</em>) OGGs.</p><p>In the above procedures, we did not identify pseudogene-like <em>A. halleri</em> genes which are orthologous to <em>A. lyrata</em> genes. Consequently, duplication frequency may be underestimated in either the <em>A. lyrata–A. halleri</em> or the <em>A. halleri</em> lineage. Therefore, we also identified 26,428 orthologous <em>A. halleri</em> regions truncated by terminal codons with &gt;80% similarity and 80% coverage against <em>A. lyrate</em> genes.</p><p>These analysis procedures and findings are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S1</a></span>.</p>                            <h3 scrollto-destination=118057121 id="118057121" class="section-title" >2.4. Validation of <em>A. halleri</em> lineage-specific duplicated genes by droplet digital PCR</h3>
<p>To calculate the read coverage of each <em>A. halleri</em> gene, the mapped count was divided by the number of genes to which a read was mapped by BOWTIE2 version 2.2.3.<a reveal-id="dsy005-B49" data-open="dsy005-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> Reads per kilobase of exon model per million (RPKM) values were calculated for each <em>A. halleri</em> gene. For 12 <em>A. halleri</em> genes whose copy numbers were known (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>), we designed primer pairs using the following parameters in the Primer3Plus software:<a reveal-id="dsy005-B44" data-open="dsy005-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> primer length of 18–24 bases, amplicon length of 70–150 bp, <em>T</em><sub>m</sub> value of 57–63°C and GC composition of 40–60%.<a reveal-id="dsy005-B50" data-open="dsy005-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> To obtain enough genomic DNA for droplet digital PCR (ddPCR), we mixed the genomic DNAs from four individuals of <em>A. halleri</em> from Mt. Ibuki. The genomic DNA was sonicated with the Covaris M220 system (25 s in frequency sweeping mode). The concentration of the sonicated genomic DNA sample was 6 ng/μl. The peak size of sonicated DNA fragments was 2,382 bp according to the Fragment Analyzer system (Advanced Analytical, Ankeny, IA, USA) with the High Sensitivity Genomic DNA Analysis Kit (Advanced Analytical). Each ddPCR reaction was performed with ddPCR EvaGreen Supermix (Bio-Rad, Hercules, CA, USA). Each reagent was divided into ∼20,000 droplets with a droplet generator (Bio-Rad QX-200). Cycled droplets were measured with a QX200 droplet reader (Bio-Rad).</p><p>To find <em>A. halleri</em> genes whose DNA concentrations were robustly inferred by ddPCR, we first identified uniquely mapped regions (&gt;80 bp) in <em>A. halleri</em> genes from the Illumina DNA-sequencing reads. Among the <em>A. halleri</em> genes with uniquely mapped regions, we manually chose 50 <em>A. halleri</em> genes with a variety of RPKMs. We designed a pair of primers for each gene in the Primer3Plus software using the parameters described earlier. To examine the specificity of the targeted DNAs, we performed real-time PCR analysis using the protocol for the Mx3000P qPCR System (Agilent Technologies). The PCR analyses were performed using SsoFast EvaGreen Supermix (Bio-Rad) and the products were analysed using the Mx3000P multiplex quantitative PCR system (Agilent Technologies). Specific and multiple reactions should result in a single and multiple melting peaks corresponding to the PCR products. Of the 50 primer pairs, 25 showed a clear raised curve for all three replicates. Thus, for the copy number assay by ddPCR, we used 13 primer pairs for unknown copy number genes, 10 pairs for single-copy genes in a broad range of plant species and 2 pairs for a three-copy gene (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>).</p>                            <h3 scrollto-destination=118057124 id="118057124" class="section-title" >2.5. Assembly of <em>A. halleri</em> genes orthologous to <em>A. lyrata</em> genes based on the available draft <em>A. halleri</em> genome</h3>
<p>Coding genes of <em>A. halleri</em> were generated on the draft <em>A. halleri</em> genome using long-insert mate paired reads and short-insert paired-end reads in Illumina DNA sequencing.<a reveal-id="dsy005-B38" data-open="dsy005-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> On the genome, we used annotated coding genes. We performed BLAST search between the annotated coding <em>A. halleri</em> and <em>A. lyrata</em> genes by BLASTP. The best matching pairs were defined to be <em>AL</em>–<em>AH</em> OGGs with more than &gt;80% similarity to and 80% coverage at amino acid level. We succeeded in identifying 26,007 (79.6%) <em>A. halleri</em> genes which were orthologous to 32,670 <em>A. lyrata</em> genes. Out of 26,007 orthologous pairs of <em>A. halleri</em> and <em>A. lyrata</em> genes, 22,105 (22,105/26,007 = 85%) <em>A. lyrata</em> genes were inferred in the <em>AL</em>–<em>AH</em> OGGs based on only Illumina paired-end DNA-sequencing reads. The number of specifically identified <em>A. halleri</em> genes based on Illumina paired-end DNA sequencing and the available <em>A. helleri</em> genome was 3,728 and 3,902, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Thus, either method had a particular advantage to infer the orthologous genes.</p><p><em>A. halleri</em> genes duplicated in the <em>A. lyrata</em> lineage were identified with the Inparanoid algorithm<a reveal-id="dsy005-B51" data-open="dsy005-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> Given the best-match pair, A1 and B1, an <em>A. halleri</em> gene that had a smaller sequence distance to A1 (or B1) than the distance between A1 and B1, was added to the <em>AL–AH</em> OGG containing A1 and B1 (seeds). This process was continued until all qualified <em>A. halleri</em> genes were assigned to seed pairs of the genes. </p>                            <h3 scrollto-destination=118057127 id="118057127" class="section-title" >2.6. Selection pressures in the <em>A. halleri-lyrata</em> and <em>A. halleri</em> lineages</h3>
<p>To infer selection pressure in the <em>A. halleri–lyrata</em> lineage, we focussed on orthologous groups that followed the speciation process among the genes of the five non-Arabidopsis species (<em>B. rapa, B. stricta</em>, <em>C. grandiflora, C. rubella, E. salsugineum</em>), <em>A. thaliana</em>, <em>A. lyrata</em> and <em>A. halleri</em>. In each orthologous group, a multiple alignment was made to match the coding regions with the computer programme MAFFT v7.215 with default parameters.<a reveal-id="dsy005-B52" data-open="dsy005-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> Using the phylogenetic tree and multiple alignment, we constructed the common ancestral sequences among <em>A. thaliana</em>, <em>A. halleri</em> and <em>A. lyrata</em>, and the common ancestral sequence between <em>A. halleri</em> and <em>A. lyrata</em> with codeml (model = 1, NSsites = 0) in PAML. For each pair of ancestral sequences, the synonymous (<em>K</em><sub>S</sub>) and non-synonymous substitution rates (<em>K</em><sub>A</sub>) were calculated using yn00 (code = 0, weighting = 0, commonf3x4 = 0) in PAML. To determine whether the <em>K</em><sub>A</sub>/<em>K</em><sub>S</sub> ratio was significantly &lt;1, two maximum likelihood values were calculated with the <em>K</em><sub>A</sub>/<em>K</em><sub>S</sub> ratio fixed at 1 and with the <em>K</em><sub>A</sub>/<em>K</em><sub>S</sub> ratio as a free parameter. The ratio of the maximum likelihood values was then compared with the χ<sup>2</sup> distribution. A <em>P</em>-value representing the deviation of the two models was then calculated for the <em>A. halleri-lyrata</em> lineage.</p><p>To infer selection pressure in the <em>A. halleri</em> lineage, we generated a tree file for the <em>A. thaliana</em>, <em>A. lyrata</em> and <em>A. halleri</em> genes. In each of the orthologous groups, a multiple alignment was made to match the coding regions by the computer programme MAFFT.<a reveal-id="dsy005-B52" data-open="dsy005-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> Using the tree file and multiple alignment file, we constructed the common ancestral sequences among <em>A. thaliana</em>, <em>A. halleri</em>, and <em>A. lyrata</em> with codeml (model = 1, NSsites = 0) in PAML. Although we used a representative <em>A. halleri</em> gene sequence to infer the ancestral sequence, proper <em>A. halleri</em> genes had sequence variation when they were lineage-specific duplicated after the split from <em>A. lyrata</em>. From the Illumina DNA-sequencing reads mapped to <em>A. halleri</em> genes by BOWTIE2 version 2.2.3 with default parameters,<a reveal-id="dsy005-B49" data-open="dsy005-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> the sequence variation was examined in each <em>A. halleri</em> gene by the Picard software with default parameters (<a class="link link-uri" href="http://broadinstitute.github.io/picard/" target="_blank">http://broadinstitute.github.io/picard/</a>). When a variable sequence was observed in the <em>A. halleri</em> genes, the codon sequence including the variable nucleotide was concatenated into the representative <em>A. halleri</em> genes. Alignments between <em>A. halleri</em> genes including codons with variable nucleotides and the ancestral sequence were modified by adding codons of the ancestral sequence against concatenated codons including variable nucleotides. <em>K</em><sub>A</sub> and <em>K</em><sub>S</sub> were calculated in each pair of ancestral and <em>A. halleri</em> gene sequences including variable sequences using yn00 in PAML. To determine whether the <em>K</em><sub>A</sub>/<em>K</em><sub>S</sub> ratio was significantly &lt;1, two maximum likelihood values were calculated with the <em>K</em><sub>A</sub>/<em>K</em><sub>S</sub> ratio fixed at 1 and with the <em>K</em><sub>A</sub>/<em>K</em><sub>S</sub> ratio as a free parameter using codeml in PAML. The ratio of the maximum likelihood values was then compared with the χ<sup>2</sup> distribution. A <em>P</em>-value representing the deviation of the two models was then calculated for each <em>A. halleri</em> gene. These analysis procedures are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S3</a></span>.</p>                            <h3 scrollto-destination=118057130 id="118057130" class="section-title" >2.7. Inference of over-represented gene ontologies</h3>
<p>Assuming that <em>A. halleri</em> and <em>A. lyrata</em> genes have similar GO assignments in <em>A. thaliana</em> in the same OGG, we obtained gene ontology (GO) assignments for the <em>A. thaliana</em> genes from The <em>Arabidopsis</em> Information Resource (<a class="link link-uri" href="http://www.arabidopsis.org" target="_blank">www.arabidopsis.org</a>). Among the top three GO categories (cellular components, molecular functions, and biological processes), we analysed only biological processes. For each GO category in the biological processes category, the expected ratio was inferred to be the ratio between the number of genes assigned to the GO category and the number of genes not assigned to the GO category among all annotated <em>A. thaliana</em> genes. In each GO category, the observed ratio in each category of OGGs was compared with the expected ratio by a <em>χ</em><sup>2</sup> test for different categories of OGGs. The ratios were processed in the R software environment (<a class="link link-uri" href="http://www.r-project.org" target="_blank">www.r-project.org</a>) and normalized among different arrays using Z-scores calculated using genescale in the R library. A heatmap was generated using heatmap.2 in the R library. To correct for multiple testing, the false discovery rate (FDR) was estimated by the R-library Q-VALUE software. The null hypothesis was rejected if FDR values were &lt; 0.01.</p>                            <h2 scrollto-destination=118057132 id="118057132" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=118057133 id="118057133" class="section-title" >3.1. Duplication frequency in the <em>A. halleri-lyrata</em> lineage</h3>
<p>To examine the frequency of duplication events in the <em>A. halleri-lyrata</em> lineage (<a reveal-id="dsy005-F2" data-open="dsy005-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>), we first used 25,833 <em>AL</em>–<em>AH</em> OGGs based on Illumina paired-end DNA-sequencing reads. We searched for an orthologous <em>A. thaliana</em> gene for each <em>AL</em>–<em>AH</em> OGG by BLASTP searches between <em>AL</em> and <em>AH</em> OGG protein sequences and annotated <em>A. thaliana</em> protein sequences.<a reveal-id="dsy005-B43" data-open="dsy005-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> When both the <em>A. lyrata</em> and <em>A. halleri</em> genes in an <em>AL</em>–<em>AH</em> OGG had the best hit to the same <em>A. thaliana</em> gene, the <em>A. thaliana</em> gene was considered an orthologous candidate gene. Of 25,833 <em>AL</em>–<em>AH</em> OGGs, 93.3% (24,019/25,833) had orthologous candidate genes in <em>A. thaliana</em>. Among these, we searched for orthologous groups that were consistent with the species tree. To do this, the synonymous substitution rates (<em>K</em><sub>S</sub>) were calculated among the <em>A. thaliana</em>, <em>A. lyrata</em>, and <em>A. halleri</em> genes in each orthologous group using yn00 in PAML.<a reveal-id="dsy005-B53" data-open="dsy005-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> When the <em>K</em><sub>S</sub> between <em>A. lyrata</em> and <em>A. halleri</em> was lower than both the <em>K</em><sub>S</sub> between <em>A. thaliana</em> and <em>A. lyrata</em> and the <em>K</em><sub>S</sub> between <em>A. thaliana</em> and <em>A. halleri</em>, we assumed that the topology of the orthologous group was consistent with the species tree. Among the 24,019 <em>AL–AH</em> OGGs, 22,602 (22,602/24,019 = 94.1%) had the same topology as the species tree. In the 22,602 <em>AL–AH</em> OGGs, 16,704 and 2,370 <em>A. thaliana</em> genes were uniquely and multiply assigned to <em>AL–AH</em> OGGs, respectively. To examine the duplication frequency in <em>A. halleri-lyrata</em> lineage, we considered two evolutionary scenarios for the 2,370 <em>A. thaliana</em> genes multiply assigned to 5,898 OGGs. One is that gene duplication events occurred in the <em>A. halleri-lyrata</em> lineage. That is, the assigned <em>A. thaliana</em> gene is orthologous to the <em>AL–AH</em> OGGs. The other is that gene loss events occurred in the <em>A. thaliana</em> lineage after gene duplication in the common ancestor among <em>A. thaliana</em>, <em>A. lyrata</em> and <em>A. halleri</em>. In this case, the assigned <em>A. thaliana</em> gene is not orthologous to the <em>AL–AH</em> OGGs. To examine these two possible scenarios, we collected <em>AL–AH</em> OGGs to which an <em>A. thaliana</em> gene was assigned as an orthologous gene. We then calculated the <em>K</em><sub>S</sub> between the <em>A. thaliana</em> gene and <em>AL–AH</em> OGG, and searched for the closest pair of the <em>AL–AH</em> OGG against the <em>A. thaliana</em> gene. When the <em>K<sub>S</sub></em> between the chosen <em>AL-AH</em> OGG and the other <em>AL-AH</em> OGG was lower than the <em>K<sub>S</sub></em> in the closest pair, the other <em>AL–AH</em> OGG was defined as sharing the <em>A. thaliana</em> gene as an orthologous gene. Of the 5,898 <em>AL–AH</em> OGGs, 497 had no orthologous <em>A. thaliana</em> genes and 5,401 had 2,370 orthologous <em>A. thaliana</em> genes. In the following analyses, 22,105 (16,704 + 5,401) <em>AL</em>–<em>AH</em> OGG with 19,074 (16,704 + 2,370) orthologous <em>A. thaliana</em> genes were defined as <em>A. thaliana</em>–<em>A. lyrata–A. halleri</em> (<em>AT</em>–<em>AL</em>–<em>AH)</em> OGGs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). These analysis procedures are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S4</a></span>.</p><p>In <em>A. lyrata-halleri</em> lineage, 22,105 <em>AT–AL</em>–<em>AH</em> OGGs were derived from the 19,074 genes, which represent the number of genes in the common ancestor of <em>A. thaliana</em>, <em>A. lyrata</em> and <em>A. halleri</em>. Thus, 3,031 (22,105−19,074) gains through gene duplication were supposed to have occurred in the <em>A. halleri-lyrata</em> lineage. The gain rate (total gene duplication gains during a given time period divided by the estimated duration per gene) was inferred to be 1.8–2.0 × 10<sup>−2</sup> (3,031 gains/19,074 genes/8–9 MY) in the <em>A. halleri-lyrata</em> lineage. Additionally, we examined 26,428 <em>AL</em>–<em>AH</em> OGGs including pseudogene-like <em>A. halleri</em> genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>), and identified 22,684 <em>AT–AL</em>–<em>AH</em> OGGs derived from the 19,318 orthologous <em>A. thaliana</em> genes. The gain rate was inferred to be 1.9–2.2 × 10<sup>−2</sup> (3,366 gains/19,318 genes/8–9 MY) in the <em>A. halleri-lyrata</em> lineage. The gain rate in OGGs including pseudogene-like genes (<a reveal-id="dsy005-F4" data-open="dsy005-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a>) is similar to OGGs without pseudogene-like genes, indicating that addition of pseudogene-like <em>A. halleri</em> genes did not affect duplication frequency in the <em>A. halleri-lyrata</em> lineage.</p><p>Second, we used 26,007 <em>AL</em>–<em>AH</em> OGGs based on the available <em>A. halleri</em> genome. Of 26,007 <em>AL</em>–<em>AH</em> OGGs, 91.0% (23,682/26,007) had orthologous candidate genes in <em>A. thaliana</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Among the 23,682 <em>AL–AH</em> OGGs, 21,984 (21,984/23,682 = 92.8%) had the same topology as the species tree. In the 21,984 <em>AL–AH</em> OGGs, 16,800 and 2,058 <em>A. thaliana</em> genes were uniquely and multiply assigned to <em>AL–AH</em> OGGs, respectively. Since 2,058 <em>A. thaliana</em> genes multiply assigned to 5,183 OGGs, 5,183 OGGs were classified into 446 OGGs which had no orthologous <em>A. thaliana</em> genes and 4,737 OGGs which had 2,058 orthologous <em>A. thaliana</em> genes. Following the above procedure, 21,537 (16,800 + 4,737) <em>AL</em>–<em>AH</em> OGG with 18,858 (16,800 + 2,058) orthologous <em>A. thaliana</em> genes were defined as <em>AT</em>–<em>AL</em>–<em>AH</em> OGGs. The gain rate was inferred to be 1.6–1.8 × 10<sup>−2</sup> (2,679 gains/18,858 genes/8–9 MY) in the <em>A. halleri-lyrata</em> lineage. Taken together, the gain rate without pseudogene-like <em>A. halleri</em> genes was 1.6–2.0 × 10<sup>−2</sup> in the <em>A. halleri-lyrata</em> lineage (<a reveal-id="dsy005-F2" data-open="dsy005-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>).</p><p>In our previous study, we inferred the gain rate of gene duplication in the lineage leading to <em>Arabidopsis</em> after the divergence of moss.<a reveal-id="dsy005-B4" data-open="dsy005-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Using the same method, we re-estimated the gain rates in three times periods—after the divergence of moss (<em>Physcomitrella patens</em>), rice (<em>Oryza sativa</em>) and poplar (<em>Populus trichocarpa</em>). The gain rates were 1.8–3.5 × 10<sup>−3</sup> in the three branches, ∼10 times lower than in the <em>A. halleri-lyrata</em> lineage (<a reveal-id="dsy005-F2" data-open="dsy005-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). One explanation for this gain rate is that even though many genes were fixed and retained, a large number of them did not survive in the long run. This explanation is consistent with the gradual decay of paralog synonymous substitution rates observed in several eukaryotes over time.<a reveal-id="dsy005-B54" data-open="dsy005-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a><sup>,</sup><a reveal-id="dsy005-B55" data-open="dsy005-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a></p><p>Note that the effect of tandemly duplicated genes on these gain rates needs to be considered because tandemly duplicated genes have undergone gene conversion, which leads to identical sequences among tandemly duplicated genes in the same species.<a reveal-id="dsy005-B56" data-open="dsy005-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> Consequently, our procedure may have failed to identify some orthologous <em>A. halleri</em> genes among the <em>A. lyrata</em> tandemly duplicated genes. In such cases, our method would have missed the duplication event in the <em>A. halleri-lyrata</em> lineage, meaning our gain rate is underestimated in the <em>A. lyrata-halleri</em> lineage. Thus, tandemly duplicated genes did not disturb the trend of a higher gain rate in <em>A. halleri-lyrata</em> in comparison with other land plants.</p>                            <h3 scrollto-destination=118057139 id="118057139" class="section-title" >3.2. Duplication frequency in the <em>A. halleri</em> lineage</h3>
<p>The copy numbers of the <em>A. halleri</em> genes were unclear from <em>AT–AL–AH</em> OGGs. However, the absolute copy number of an <em>A. halleri</em> gene could be experimentally inferred by the absolute DNA concentration of the gene by ddPCR.<a reveal-id="dsy005-B57" data-open="dsy005-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> First, to examine the relationship between copy number and DNA concentration, we focussed on known three-copy genes, <em>HMA4</em><a reveal-id="dsy005-B41" data-open="dsy005-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> and <em>MTP1</em>,<a reveal-id="dsy005-B58" data-open="dsy005-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> and 10 singleton genes that share a single copy in a broad range of plants.<a reveal-id="dsy005-B59" data-open="dsy005-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> The concentration of <em>HMA4</em> was 627.5 ± 40.39 and 626.75 ± 39.38 copies/μl (four replicates for each of the two primer pairs, average of each primer pair ± 95% CI). The concentration of <em>MTP1</em> was 605.0 ± 22.39. Conversely, the concentration of the 10 singleton genes was 193.30 ± 9.06 copies/μl (four replicates for each of the 10 primer pairs, average of all 10 primer pairs ± 95% CI). The average DNA concentration was ∼3.2 times higher for the three-copy genes compared with the single-copy genes (<a reveal-id="dsy005-F3" data-open="dsy005-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>), indicating that the copy number corresponded to the DNA concentration inferred by ddPCR. </p>                        <a id="118057141" scrollto-destination="118057141"></a>
<div id="dsy005-F3" data-id="dsy005-F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/m_dsy005f3.png?Expires=2147483647&Signature=hwhhnhziELnucDZCSavdSSiUo2vjr~EvHbpuAWxWzPspqO-PIhUrD-3mW8h43FknNG624rKLBLQPNwlAnYMHc8cKGUNVLv2SsH8aeGrgQQf5MFL7bhMClHpvnrWb83PMrCdI0QaMFItUbqSQfHCjhuSKZQpIhTnOPl0mu~f8ugqorcZC3nERr7VkQAQlI0GbX-jGFlXCqXQLpdLwGj73pemIcduJ9C59ZbNSHjF8N9wCxPXFlKpMtdxS41dZDjrFJxJT5t2ax6hQWUHXdlzK479WnkCySJhwcMuwZpb6Pl0ozETxLjWnSTQ~oHvB6qDgGp1LPxwRELjbUnpEV~5EkA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relationship between the Illumina DNA-sequencing read depth and the copy number inferred by ddPCR. (A) The Y-axis represents copy numbers per μl inferred by ddPCR. Black circles (gray background) and open circles (black background) indicate three-copy genes and single-copy genes, respectively. All points and error bars represent averages of four replicates and 95% CIs. (B) Each dot represents an A. halleri gene. The X-axis represents the Illumina DNA-sequencing read depth, which is the number of reads per 1 Kbp per 1 million reads. The Y-axis represents copy numbers per μl, which were inferred by ddPCR. The regression line was calculated with the simple formula Y = αX; α was inferred by the least squares method." path-from-xml="dsy005f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057141/dsy005f3.tif" path-from-xml="dsy005f3.tif" target="_blank">View large</a><a section="118057141" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005f3.png?Expires=2147483647&Signature=hbPcdLjIHE4ZOcz~hqgFtB74rUiHQm2ml-CAX~c-HrchJUZo9sWLLTO4dfxTHCYltfmQt~-RwcHaAJSHuLgcFNwnfrztmzNF8c~UcvWHoL0ns-cssdV9qHVStpgZfwJck7005VCXBrn5YYyh65PsDEDf2I5KxP1ZIlZrNDlMEWuEAHd5rtns~MWkib2K6orCd2E3iU42sYfRWV2Vjxb6ESrYJUfvlg0JgMFvX3HPHng1xf3OlfKTaI3co-BdKxaHWA74SvS~SENL9QKrfFXDxzE7jqvWVdYJeRreaR~DbkEo5bbN6IoNWCDfpVNd4THW4a2oiGrZ-NuUUb~nqFRBMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057141&ar=4898128&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy005f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relationship between the Illumina DNA-sequencing read depth and the copy number inferred by ddPCR. (A) The Y-axis represents copy numbers per μl inferred by ddPCR. Black circles (gray background) and open circles (black background) indicate three-copy genes and single-copy genes, respectively. All points and error bars represent averages of four replicates and 95% CIs. (B) Each dot represents an <em>A. halleri</em> gene. The X-axis represents the Illumina DNA-sequencing read depth, which is the number of reads per 1 Kbp per 1 million reads. The Y-axis represents copy numbers per μl, which were inferred by ddPCR. The regression line was calculated with the simple formula Y = αX; α was inferred by the least squares method.</p></div></div><div content-id="dsy005-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/m_dsy005f3.png?Expires=2147483647&Signature=hwhhnhziELnucDZCSavdSSiUo2vjr~EvHbpuAWxWzPspqO-PIhUrD-3mW8h43FknNG624rKLBLQPNwlAnYMHc8cKGUNVLv2SsH8aeGrgQQf5MFL7bhMClHpvnrWb83PMrCdI0QaMFItUbqSQfHCjhuSKZQpIhTnOPl0mu~f8ugqorcZC3nERr7VkQAQlI0GbX-jGFlXCqXQLpdLwGj73pemIcduJ9C59ZbNSHjF8N9wCxPXFlKpMtdxS41dZDjrFJxJT5t2ax6hQWUHXdlzK479WnkCySJhwcMuwZpb6Pl0ozETxLjWnSTQ~oHvB6qDgGp1LPxwRELjbUnpEV~5EkA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relationship between the Illumina DNA-sequencing read depth and the copy number inferred by ddPCR. (A) The Y-axis represents copy numbers per μl inferred by ddPCR. Black circles (gray background) and open circles (black background) indicate three-copy genes and single-copy genes, respectively. All points and error bars represent averages of four replicates and 95% CIs. (B) Each dot represents an A. halleri gene. The X-axis represents the Illumina DNA-sequencing read depth, which is the number of reads per 1 Kbp per 1 million reads. The Y-axis represents copy numbers per μl, which were inferred by ddPCR. The regression line was calculated with the simple formula Y = αX; α was inferred by the least squares method." path-from-xml="dsy005f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057141/dsy005f3.tif" path-from-xml="dsy005f3.tif" target="_blank">View large</a><a section="118057141" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005f3.png?Expires=2147483647&Signature=hbPcdLjIHE4ZOcz~hqgFtB74rUiHQm2ml-CAX~c-HrchJUZo9sWLLTO4dfxTHCYltfmQt~-RwcHaAJSHuLgcFNwnfrztmzNF8c~UcvWHoL0ns-cssdV9qHVStpgZfwJck7005VCXBrn5YYyh65PsDEDf2I5KxP1ZIlZrNDlMEWuEAHd5rtns~MWkib2K6orCd2E3iU42sYfRWV2Vjxb6ESrYJUfvlg0JgMFvX3HPHng1xf3OlfKTaI3co-BdKxaHWA74SvS~SENL9QKrfFXDxzE7jqvWVdYJeRreaR~DbkEo5bbN6IoNWCDfpVNd4THW4a2oiGrZ-NuUUb~nqFRBMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057141&ar=4898128&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy005f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relationship between the Illumina DNA-sequencing read depth and the copy number inferred by ddPCR. (A) The Y-axis represents copy numbers per μl inferred by ddPCR. Black circles (gray background) and open circles (black background) indicate three-copy genes and single-copy genes, respectively. All points and error bars represent averages of four replicates and 95% CIs. (B) Each dot represents an <em>A. halleri</em> gene. The X-axis represents the Illumina DNA-sequencing read depth, which is the number of reads per 1 Kbp per 1 million reads. The Y-axis represents copy numbers per μl, which were inferred by ddPCR. The regression line was calculated with the simple formula Y = αX; α was inferred by the least squares method.</p></div></div><p>The copy numbers of the <em>A. halleri</em> genes could also be inferred by the reading depth of the Illumina DNA-sequencing reads. The reading depth was defined as the RPKM. To examine whether RPKM values corresponded with copy numbers, we focussed on 25 <em>A. halleri</em> genes that had a wide variety of RPKMs (see section 2). For these genes, the copy number estimated by digital droplet PCR (ddPCR) was compared with the RPKM. Consequently, we found that RPKM was significantly correlated with concentration (<a reveal-id="dsy005-F3" data-open="dsy005-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>, <em>R</em><sup>2</sup> = 0.94). This result indicated that RPKM values could be used an index of copy numbers for <em>A. halleri</em> genes.</p><p>To infer duplication frequencies by RPKM values, we focussed on 285 <em>AT–AL–AH</em> OGGs with single-copy genes in a broad range of plant species such as <em>Arabidopsis</em>, <em>Carica</em>, <em>Populus</em>, <em>Vitis</em>, <em>Oryza</em>, <em>Selaginella</em> and <em>Physcomitrella</em>.<a reveal-id="dsy005-B59" data-open="dsy005-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> The RPKM values of the <em>A. halleri</em> genes showed a normal distribution (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5A</a></span>). To call duplicated genes, the top 5% (309) of RPKMs was defined as the threshold for duplicated genes. That is, <em>A. halleri</em> genes with an RPKM &lt; 309 were defined as non-duplicated genes, <em>A. halleri</em> genes with RPKM values from 309 to 618 (309 × 2) were defined as two-copy genes, and <em>A. halleri</em> genes with RPKM values of 618–927 (309 × 3) were defined as three-copy genes. Following this rule, in 22,105 <em>AT–AL</em>–<em>AH</em> OGGs based on Illumina paired-end DNA-sequencing reads, we identified 2,488 multiply duplicated <em>A. halleri</em> genes and 3,378 gain events through gene duplications in the <em>A. halleri</em> lineage (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). The gain rate (total gains from gene duplication during a given time period divided by the estimated duration per gene) was 7.6 × 10<sup>−2</sup> (3,378 gains/22,105 genes/2 MY).</p><p>However, the gain events may have been over-estimated because duplication events occurring in the <em>A. halleri-lyrata</em> lineage caused an increase of RPKMs. Therefore, excluding <em>A. halleri</em> genes that were duplicated in the <em>A. halleri-lyrata</em> lineage, we counted the number of gain events in the <em>A. halleri</em> lineage. Among the currently retained <em>A. halleri</em> genes, we focussed on 16,946 <em>A. halleri</em> genes that had not undergone gene duplication in the <em>A.halleri-lyrata</em> lineage. These 16,946 genes had undergone 1,958 gain events through gene duplication in the <em>A. halleri</em> lineage. The re-estimated gain rate was 5.8 × 10<sup>−2</sup> (1,958 gains/16,946 genes/2 MY). The gain rate (5.8–7.6 × 10<sup>−2</sup>) of duplicate genes in the <em>A. halleri</em> lineage was approximately four times higher than the gain rate (1.6–2.0 × 10<sup>−2</sup>) in the <em>A. halleri-lyrata</em> lineage (<a reveal-id="dsy005-F2" data-open="dsy005-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). As mentioned earlier, the gain rate difference can be explained by a rapid decay of duplicated genes. However, this gain rate did not include gene duplications derived from pseudogenes which were on the earlier process of decayed genes. To determine the decayed effect in the <em>A. halleri</em> lineage, we focussed on 22,684 <em>AT–AL</em>–<em>AH</em> OGGs including pseudogene-like genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). Out of 22,684 OGGs, the gain rate in the <em>A. halleri</em> lineage was 1.2 × 10<sup>−1</sup> (5,665 gains/22,684 genes/2 MY). Thus, the gain rate including pseudogenes-like <em>A. halleri genes</em> was approximately two times higher than the gain rate (7.6 × 10<sup>−2</sup>) without any pseudogene-like <em>A. halleri</em> genes. This result indicates that pseudogene-like <em>A. halleri</em> genes accelerated the gain rates in the <em>A. halleri</em> lineage. To determine whether pseudogenes tended to have a higher duplication rate in comparison with non-pseudogene-like genes, we fosused on 1,159 <em>AT–AL</em>–<em>AH</em> OGGs composing only pseudogene-like <em>A. halleri</em> genes among 22,684 <em>AT–AL</em>–<em>AH</em> OGGs including pseudogene-like genes. We then identified 713 pseudogene-like <em>A. halleri</em> genes experiencing gene duplications in either the <em>A. halleri-lyrata</em> or the <em>A. halleri</em> lineage. In the case of 22,105 <em>AT–AL</em>–<em>AH</em> OGGs without any pseudogene-like <em>A. halleri</em> genes, we identified 6,752 <em>A. haller</em>i genes experiencing gene duplications in either the <em>A. halleri-lyrata</em> or the <em>A. halleri</em> lineage. The proportion (713/1,159 = 62%) of duplicated genes in pseudogene-like <em>A. halleri</em> genes was significantly higher than that (6,752/22,105 = 31%) of duplicated genes in the other <em>A. halleri</em> genes (<em>P</em>-value = 1.9 × 10<sup>−107</sup> by χ<sup>2</sup> test, <a reveal-id="dsy005-T1" data-open="dsy005-T1" class="link link-reveal link-table xref-fig">Table 1</a>), indicating that most of pseudogenes tended to appear via gene duplication in <em>Arabidopsis</em>. Taken together, it is likely that most of recently duplicated genes in <em>Arabidopsis</em> may be on the process of decayed genes but some of duplicated genes significantly contributed to functional divergence among <em>Arabidopsis</em> species. <div content-id="dsy005-T1" class="table-modal"><div class="table-wrap"><div id="dsy005-T1" data-id="dsy005-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Comparison of gene duplication to non-gene duplication ratio in either <em>A. halleri-lyrata</em> or <em>A. halleri</em> lineage between OGGs including pseudogene-like <em>A. halleri</em> genes and OGGs without any pseudogene-like <em>A. halleri</em> genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Gene duplication in either <em>A. halleri- lyrata</em> or <em>A. halleri</em> lineage (D) </th><th>No gene duplication in either <em>A. halleri- lyrata</em> or <em>A. halleri lineage</em> (N) </th><th>D/N ratio </th><th><em>P</em> value (χ<sup>2</sup> test) </th></tr></thead><tbody><tr><td>OGGs derived from only pseudogenes </td><td>713 </td><td>446 </td><td>1.60 </td><td rowspan="2">1.9 × 10<sup>−107</sup> </td></tr><tr><td>OGGs without any pseudogenes </td><td>6,752 </td><td>15,353 </td><td>0.44 </td></tr></tbody></table></div><div content-id="dsy005-T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Gene duplication in either <em>A. halleri- lyrata</em> or <em>A. halleri</em> lineage (D) </th><th>No gene duplication in either <em>A. halleri- lyrata</em> or <em>A. halleri lineage</em> (N) </th><th>D/N ratio </th><th><em>P</em> value (χ<sup>2</sup> test) </th></tr></thead><tbody><tr><td>OGGs derived from only pseudogenes </td><td>713 </td><td>446 </td><td>1.60 </td><td rowspan="2">1.9 × 10<sup>−107</sup> </td></tr><tr><td>OGGs without any pseudogenes </td><td>6,752 </td><td>15,353 </td><td>0.44 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsy005-T1" data-id="dsy005-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Comparison of gene duplication to non-gene duplication ratio in either <em>A. halleri-lyrata</em> or <em>A. halleri</em> lineage between OGGs including pseudogene-like <em>A. halleri</em> genes and OGGs without any pseudogene-like <em>A. halleri</em> genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Gene duplication in either <em>A. halleri- lyrata</em> or <em>A. halleri</em> lineage (D) </th><th>No gene duplication in either <em>A. halleri- lyrata</em> or <em>A. halleri lineage</em> (N) </th><th>D/N ratio </th><th><em>P</em> value (χ<sup>2</sup> test) </th></tr></thead><tbody><tr><td>OGGs derived from only pseudogenes </td><td>713 </td><td>446 </td><td>1.60 </td><td rowspan="2">1.9 × 10<sup>−107</sup> </td></tr><tr><td>OGGs without any pseudogenes </td><td>6,752 </td><td>15,353 </td><td>0.44 </td></tr></tbody></table></div><div content-id="dsy005-T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Gene duplication in either <em>A. halleri- lyrata</em> or <em>A. halleri</em> lineage (D) </th><th>No gene duplication in either <em>A. halleri- lyrata</em> or <em>A. halleri lineage</em> (N) </th><th>D/N ratio </th><th><em>P</em> value (χ<sup>2</sup> test) </th></tr></thead><tbody><tr><td>OGGs derived from only pseudogenes </td><td>713 </td><td>446 </td><td>1.60 </td><td rowspan="2">1.9 × 10<sup>−107</sup> </td></tr><tr><td>OGGs without any pseudogenes </td><td>6,752 </td><td>15,353 </td><td>0.44 </td></tr></tbody></table></div></div></p><p>Using 21,537 <em>AT–AL</em>–<em>AH</em> OGGs based on the available <em>A. halleri</em> genome, we identified 1,248 gain events in 838 genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). The gain rate was 2.9 × 10<sup>−2</sup> (1,248 gains/21,537/2MY). The gain rate was approximately half in comparison with the gain rate (5.8–7.6 × 10<sup>−2</sup>) inferred by RPKMs. Therefore, copy number information inferred by ddPCR was compared with the number of <em>A. halleri</em> lineage-specific duplicated genes. Most of OGGs had single-copy genes in <em>A. halleri</em> although various duplication frequencies were inferred by ddPCR (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplemental Fig. S5B</a></span>). These results indicate that some of gene duplication tended to be missed in a genome assembly based only on Illumina short reads. This shows that the reading depth of raw Illumina reads may be informative for inferring the number of recently duplicated genes.</p>                            <h3 scrollto-destination=118057146 id="118057146" class="section-title" >3.3. Selection pressures in the <em>A. halleri-lyrata</em> lineage</h3>
<p>We found that 23–30% of OGGs had undergone gene duplications in the <em>Arabidopsis</em> lineage at high gain rates (1.6–7.6 × 10<sup>−2</sup> gains/gene/MY), which were 10 times higher than the rates inferred by comparative genomics among <em>Arabidopsis</em>, poplar, rice and moss (<a reveal-id="dsy005-F2" data-open="dsy005-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). Therefore, we were interested in investigating the functional divergence of duplicated genes in <em>Arabidopsis</em>. To infer the functional divergence of duplicated genes in the <em>A. halleri-lyrata</em> lineage, we tried to infer the ancestral sequences of the most recent common ancestor of <em>A. thaliana</em>, <em>A. lyrata</em> and <em>A. halleri</em>. To define the node of the most recent common ancestor among <em>A. thaliana</em>, <em>A. lyrata</em>, and <em>A. halleri</em>, we used an orthologous non-<em>Arabidopsis</em> gene as an outgroup sequence for each of <em>AT</em>–<em>AL</em>–<em>AH</em> OGGs and performed BLASTP searches of <em>AT</em>–<em>AL</em>–<em>AH</em> protein sequences against all five non-<em>Arabidopsis</em> (<em>B. rapa</em>, <em>B. stricta</em>, <em>C. grandiflora</em>, <em>C. rubella</em>, <em>E. salsugineum</em>) protein sequences.<a reveal-id="dsy005-B43" data-open="dsy005-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> When the best-hit non-<em>Arabidopsis</em> gene was the same for the <em>A. thaliana</em>, <em>A. lyrata</em> and <em>A. halleri</em> genes, the non-<em>Arabidopsis</em> gene was considered a candidate orthologous gene to the <em>AT</em>–<em>AL</em>–<em>AH</em> OGG. For each of candidate genes, we searched for candidate orthologous groups that were consistent with the species tree. To do this, we generated a phylogenetic tree by the neighbour-joining method using the PAUP software (set outroot = mono, dset distance = hky).<a reveal-id="dsy005-B60" data-open="dsy005-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a><sup>,</sup><a reveal-id="dsy005-B61" data-open="dsy005-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> When the topology of the gene tree was the same as that of the species tree, we assumed that the topology of the orthologous group was consistent with the species tree. There were 22,105 and 21,537 <em>AT–AL</em>–<em>AH</em> OGGs based on Illumina paired-end DNA-sequencing reads and the available <em>A. halleri</em> genomes, respectively. Out of 22,105 and 21,537 <em>AL</em>–<em>AH</em> OGGs, we identified 17,669 and 17,925 orthologous groups that followed the speciation process among non-<em>Arabidopsis</em>, <em>A. thaliana</em>, <em>A. lyrata</em>, and <em>A. halleri</em> genes, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). These OGGs were defined as <em>non-A-AT–AL–AH</em> OGGs. These analysis procedures are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S6</a></span>.</p><p>Based on the phylogenetic tree in each <em>non-A</em>-<em>AT–AL–AH</em> OGG, we inferred the ancestor sequences of all nodes using codeml in the PAML package,<a reveal-id="dsy005-B53" data-open="dsy005-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> and calculated <em>K</em><sub>A</sub> and <em>K</em><sub>S</sub> in the <em>A. halleri-lyrata</em> lineage. First, among the 17,669 OGGs base on Illumina paired-end DNA-sequencing reads, we found 481, 8,313 and 8,875 genes with positive selection (<em>K</em><sub>A</sub>/<em>K</em><sub>S</sub> &gt; 1), purifying selection (<em>K</em><sub>A</sub>/<em>K</em><sub>S</sub> &lt; 1) and unclear selection respectively, by the maximum likelihood approach (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). To address whether the duplicated genes contributed to functional divergence in the <em>A. halleri-lyrata</em> lineage, the proportions of positive selection and purifying selection in 1,782 duplicated genes were compared with those in 15,887 non-duplicated genes. In this test, the null model was the ratio of duplicated genes to non-duplicated genes without any particular selection pressure in the <em>A. halleri-lyrata</em> lineage. The proportions of positive selection in the duplicated genes was significantly higher than those in the non-duplicated genes (<em>P</em>-value = 2.8 × 10<sup>−57</sup> by χ<sup>2</sup> test, <a reveal-id="dsy005-F4" data-open="dsy005-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). The proportion of purifying selection in the non-duplicated genes was significantly higher than in the duplicated genes in the <em>A. halleri-lyrata</em> lineage (<em>P</em>-value = 1.2 × 10<sup>−33</sup> by χ<sup>2</sup> test, <a reveal-id="dsy005-F4" data-open="dsy005-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). Furthermore, we did the same analysis in 17,925 OOGs based on the available <em>A. halleri</em> genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). We found the same trend for only positive selection in the OOGs (<em>P</em>-value = 3.0 × 10<sup>−36</sup> for positive selection, <em>P</em>-value = 0.05 for purifying selection by χ<sup>2</sup> test, <a reveal-id="dsy005-F4" data-open="dsy005-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>). These results indicate that gene duplication induced functional divergence in the <em>A. halleri-lyrata</em> lineage. </p>                        <a id="118057149" scrollto-destination="118057149"></a>
<div id="dsy005-F4" data-id="dsy005-F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/m_dsy005f4.png?Expires=2147483647&Signature=VVxu5I16buWiYFIXmysWh9u1z1h3hfJECqTDzriI5SwDghk91yzgMJG0eBxYhuNa~5mYb-QjWLEy-fVqcV70us1BEmuW26I65uRzS2UUyE2A0BU6qShY~g6Zht5zrTGTq3DgvwDwhrpnB2r6~IEMb0g47jaXCdZ9Wsnp--NWT11JIiXlkJj577xQ66NU0dqZ2DfkDNzOgzDo7GrITbcGZJzexI1LITc-flVrC3STD5Wk90ID1ZdYbwn9E36JeSNkA~CiLZj7UUhzObdDZcQlCTiiM28JfN8fE23~Z2yJEW2cDLIku8nZRKVHhPYwwC~z7G9l43sVSXEXGg5i~s0LbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene duplication and selection pressure in the A. halleri-lyrata and A. halleri lineages. Genes were classified as being under positive selection (KA/KS &gt; 1), unclear selection or purifying selection (KA/KS &lt; 1) in the A. halleri-lyrata and A. halleri lineages. Asterisks (*) represent significant differences by the chi-square test (P &lt; 0.05). (A) The relationship between gene duplication and selection pressure in the A. lyrata-halleri lineage in 17,669 OGGs base on Illumina paired-end DNA-sequencing reads. (B) The relationship between gene duplication and selection pressure in the A. lyrata-halleri lineage in 17,925 OOGs based on the available A. halleri genome. (C) The relationship between gene duplication and selection pressure in the A. halleri lineage in 22,105 AT–AL–AH OGGs based on Illumina paired-end DNA-sequencing reads. (D) The relationship between gene duplication and selection pressure in the A. halleri lineage in 21,537 AT–AL–AH OOGs based on the available A. halleri genome. (E) The relationship between gene duplication in the A. lyrata-halleri lineage and selection pressure in the A. halleri lineage in 22,105 AT–AL–AH OGGs based on Illumina paired-end DNA-sequencing reads. (F) The relationship between gene duplication in the A. lyrata-halleri lineage and selection pressure in the A. halleri lineage in 21,537 AT–AL–AH OOGs based on the available A. halleri genome. (G) Comparison of selection pressure in the A. halleri lineage between gene duplication in only the A. lyrata-halleri lineage and gene duplication in only the A. halleri lineage in OGGs base on Illumina paired-end DNA-sequencing reads. (H) Comparison of selection pressure in the A. halleri lineage between gene duplication in only the A. lyrata-halleri lineage and gene duplication in only the A. halleri lineage in OGGs based on the available A. halleri genome." path-from-xml="dsy005f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057149/dsy005f4.tif" path-from-xml="dsy005f4.tif" target="_blank">View large</a><a section="118057149" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005f4.png?Expires=2147483647&Signature=NX3DjFroZReyJdupMdCUuTnQyn7cU~h4IgokXh7usaS0m7HI-tXX5mDsGrDB5ocJtoSWMfz0fFVEIJFpr7vqMh2rnP12msWViZ~mGAXkd2YgmpfLX9PEInEUTJ51oX6ySXLNgONduxWmF-PEL4H0Z3PDrRYnKSq2~nLC1EcqNTOmMTRHWbNv35cW9yUtQSVykoKnaJ16F9vZcDfpKBlA1UgurDU3p7Dfqgcgcn5Jc6WPVII331yU~rB0nsq9~z4w0ZBo8iYXoXuN2EbREQ4-q0MOiNIMp1aAuDkh~n2AKAxtaoj46LZOP1bn-G-c9qyBMpaDlL3S6F1i2U4Ju9jQEA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057149&ar=4898128&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy005f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene duplication and selection pressure in the <em>A. halleri-lyrata</em> and <em>A. halleri</em> lineages. Genes were classified as being under positive selection (K<sub>A</sub>/K<sub>S</sub> &gt; 1), unclear selection or purifying selection (K<sub>A</sub>/K<sub>S</sub> &lt; 1) in the <em>A. halleri-lyrata</em> and <em>A. halleri</em> lineages. Asterisks (*) represent significant differences by the chi-square test (<em>P</em> &lt; 0.05). (A) The relationship between gene duplication and selection pressure in the <em>A. lyrata-halleri</em> lineage in 17,669 OGGs base on Illumina paired-end DNA-sequencing reads. (B) The relationship between gene duplication and selection pressure in the <em>A. lyrata-halleri</em> lineage in 17,925 OOGs based on the available <em>A. halleri</em> genome. (C) The relationship between gene duplication and selection pressure in the <em>A. halleri</em> lineage in 22,105 <em>AT–AL</em>–<em>AH</em> OGGs based on Illumina paired-end DNA-sequencing reads. (D) The relationship between gene duplication and selection pressure in the <em>A. halleri</em> lineage in 21,537 <em>AT–AL</em>–<em>AH</em> OOGs based on the available <em>A. halleri</em> genome. (E) The relationship between gene duplication in the <em>A. lyrata-halleri</em> lineage and selection pressure in the <em>A. halleri</em> lineage in 22,105 <em>AT–AL</em>–<em>AH</em> OGGs based on Illumina paired-end DNA-sequencing reads. (F) The relationship between gene duplication in the <em>A. lyrata-halleri</em> lineage and selection pressure in the <em>A. halleri</em> lineage in 21,537 <em>AT–AL</em>–<em>AH</em> OOGs based on the available <em>A. halleri</em> genome. (G) Comparison of selection pressure in the <em>A. halleri</em> lineage between gene duplication in only the <em>A. lyrata-halleri</em> lineage and gene duplication in only the <em>A. halleri</em> lineage in OGGs base on Illumina paired-end DNA-sequencing reads. (H) Comparison of selection pressure in the <em>A. halleri</em> lineage between gene duplication in only the <em>A. lyrata-halleri</em> lineage and gene duplication in only the <em>A. halleri</em> lineage in OGGs based on the available <em>A. halleri</em> genome.</p></div></div><div content-id="dsy005-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/m_dsy005f4.png?Expires=2147483647&Signature=VVxu5I16buWiYFIXmysWh9u1z1h3hfJECqTDzriI5SwDghk91yzgMJG0eBxYhuNa~5mYb-QjWLEy-fVqcV70us1BEmuW26I65uRzS2UUyE2A0BU6qShY~g6Zht5zrTGTq3DgvwDwhrpnB2r6~IEMb0g47jaXCdZ9Wsnp--NWT11JIiXlkJj577xQ66NU0dqZ2DfkDNzOgzDo7GrITbcGZJzexI1LITc-flVrC3STD5Wk90ID1ZdYbwn9E36JeSNkA~CiLZj7UUhzObdDZcQlCTiiM28JfN8fE23~Z2yJEW2cDLIku8nZRKVHhPYwwC~z7G9l43sVSXEXGg5i~s0LbQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene duplication and selection pressure in the A. halleri-lyrata and A. halleri lineages. Genes were classified as being under positive selection (KA/KS &gt; 1), unclear selection or purifying selection (KA/KS &lt; 1) in the A. halleri-lyrata and A. halleri lineages. Asterisks (*) represent significant differences by the chi-square test (P &lt; 0.05). (A) The relationship between gene duplication and selection pressure in the A. lyrata-halleri lineage in 17,669 OGGs base on Illumina paired-end DNA-sequencing reads. (B) The relationship between gene duplication and selection pressure in the A. lyrata-halleri lineage in 17,925 OOGs based on the available A. halleri genome. (C) The relationship between gene duplication and selection pressure in the A. halleri lineage in 22,105 AT–AL–AH OGGs based on Illumina paired-end DNA-sequencing reads. (D) The relationship between gene duplication and selection pressure in the A. halleri lineage in 21,537 AT–AL–AH OOGs based on the available A. halleri genome. (E) The relationship between gene duplication in the A. lyrata-halleri lineage and selection pressure in the A. halleri lineage in 22,105 AT–AL–AH OGGs based on Illumina paired-end DNA-sequencing reads. (F) The relationship between gene duplication in the A. lyrata-halleri lineage and selection pressure in the A. halleri lineage in 21,537 AT–AL–AH OOGs based on the available A. halleri genome. (G) Comparison of selection pressure in the A. halleri lineage between gene duplication in only the A. lyrata-halleri lineage and gene duplication in only the A. halleri lineage in OGGs base on Illumina paired-end DNA-sequencing reads. (H) Comparison of selection pressure in the A. halleri lineage between gene duplication in only the A. lyrata-halleri lineage and gene duplication in only the A. halleri lineage in OGGs based on the available A. halleri genome." path-from-xml="dsy005f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057149/dsy005f4.tif" path-from-xml="dsy005f4.tif" target="_blank">View large</a><a section="118057149" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005f4.png?Expires=2147483647&Signature=NX3DjFroZReyJdupMdCUuTnQyn7cU~h4IgokXh7usaS0m7HI-tXX5mDsGrDB5ocJtoSWMfz0fFVEIJFpr7vqMh2rnP12msWViZ~mGAXkd2YgmpfLX9PEInEUTJ51oX6ySXLNgONduxWmF-PEL4H0Z3PDrRYnKSq2~nLC1EcqNTOmMTRHWbNv35cW9yUtQSVykoKnaJ16F9vZcDfpKBlA1UgurDU3p7Dfqgcgcn5Jc6WPVII331yU~rB0nsq9~z4w0ZBo8iYXoXuN2EbREQ4-q0MOiNIMp1aAuDkh~n2AKAxtaoj46LZOP1bn-G-c9qyBMpaDlL3S6F1i2U4Ju9jQEA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057149&ar=4898128&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy005f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene duplication and selection pressure in the <em>A. halleri-lyrata</em> and <em>A. halleri</em> lineages. Genes were classified as being under positive selection (K<sub>A</sub>/K<sub>S</sub> &gt; 1), unclear selection or purifying selection (K<sub>A</sub>/K<sub>S</sub> &lt; 1) in the <em>A. halleri-lyrata</em> and <em>A. halleri</em> lineages. Asterisks (*) represent significant differences by the chi-square test (<em>P</em> &lt; 0.05). (A) The relationship between gene duplication and selection pressure in the <em>A. lyrata-halleri</em> lineage in 17,669 OGGs base on Illumina paired-end DNA-sequencing reads. (B) The relationship between gene duplication and selection pressure in the <em>A. lyrata-halleri</em> lineage in 17,925 OOGs based on the available <em>A. halleri</em> genome. (C) The relationship between gene duplication and selection pressure in the <em>A. halleri</em> lineage in 22,105 <em>AT–AL</em>–<em>AH</em> OGGs based on Illumina paired-end DNA-sequencing reads. (D) The relationship between gene duplication and selection pressure in the <em>A. halleri</em> lineage in 21,537 <em>AT–AL</em>–<em>AH</em> OOGs based on the available <em>A. halleri</em> genome. (E) The relationship between gene duplication in the <em>A. lyrata-halleri</em> lineage and selection pressure in the <em>A. halleri</em> lineage in 22,105 <em>AT–AL</em>–<em>AH</em> OGGs based on Illumina paired-end DNA-sequencing reads. (F) The relationship between gene duplication in the <em>A. lyrata-halleri</em> lineage and selection pressure in the <em>A. halleri</em> lineage in 21,537 <em>AT–AL</em>–<em>AH</em> OOGs based on the available <em>A. halleri</em> genome. (G) Comparison of selection pressure in the <em>A. halleri</em> lineage between gene duplication in only the <em>A. lyrata-halleri</em> lineage and gene duplication in only the <em>A. halleri</em> lineage in OGGs base on Illumina paired-end DNA-sequencing reads. (H) Comparison of selection pressure in the <em>A. halleri</em> lineage between gene duplication in only the <em>A. lyrata-halleri</em> lineage and gene duplication in only the <em>A. halleri</em> lineage in OGGs based on the available <em>A. halleri</em> genome.</p></div></div>                            <h3 scrollto-destination=118057150 id="118057150" class="section-title" >3.4. Selection pressure in the <em>A. halleri</em> lineage</h3>
<p>To infer selection pressure in the <em>A. halleri</em> lineage, we generated ancestral sequences of the <em>A. lyrata</em> and <em>A. halleri</em> genes using <em>A. thaliana</em> genes as outgroups. When an <em>A. halleri</em> gene did not have any sequence variation in the Illumina DNA-sequencing reads, <em>K</em><sub>S</sub> and <em>K</em><sub>A</sub> were simply calculated by comparing the ancestral sequence to the representative <em>A. halleri</em> gene sequence. When sequence variation for a representative <em>A. halleri</em> gene sequence was identified from the Illumina DNA-sequencing reads, <em>K</em><sub>S</sub> and <em>K</em><sub>A</sub> were separately calculated for the varied sequences (see Materials and Methods, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). Among the 22,105 <em>AT–AL</em>–<em>AH</em> OGGs based on Illumina paired-end DNA-sequencing reads, we found 568, 7,717, and 13,820 genes with positive selection (<em>K</em><sub>A</sub>/<em>K</em><sub>S</sub> &gt; 1), purifying selection (<em>K</em><sub>A</sub>/<em>K</em><sub>S</sub> &lt; 1) and unclear selection, respectively, in the <em>A. halleri</em> lineage by the maximum likelihood approach (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). To examine the relationship between the effect of gene duplication and selection pressures in the <em>A. halleri</em> lineage, the proportions of positive selection and purifying selection in 2,488 duplicated genes were compared with those in 19,617 non-duplicated genes. In this test, the null model was the ratio of duplicated genes to non-duplicated genes in the <em>A. halleri</em> lineage. As observed in the <em>A. halleri-lyrata</em> lineage, the proportions of positive selection in the duplicated genes were significantly higher than those in the non-duplicated genes (<em>P</em>-value = 2.3 × 10<sup>−6</sup> for positive selection by χ<sup>2</sup> test, <a reveal-id="dsy005-F4" data-open="dsy005-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a>), while the proportion of purifying selection in the non-duplicated genes was significantly higher than in the duplicated genes (<em>P</em>-value = 1.9 × 10<sup>−26</sup> by χ<sup>2</sup> test, <a reveal-id="dsy005-F4" data-open="dsy005-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a>). We did the same analysis in 21,537 <em>AT–AL</em>–<em>AH</em> OOGs based on the available <em>A. halleri</em> genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). We found the same trend in the OOGs (<em>P</em>-value = 9.9 × 10<sup>−15</sup> for positive selection, <em>P</em>-value = 2.2 × 10<sup>−18</sup> for purifying selection by χ<sup>2</sup> test, <a reveal-id="dsy005-F4" data-open="dsy005-F4" class="link link-reveal link-table xref-fig">Fig. 4D</a>). These results indicate that gene duplication induced functional divergence in the <em>A. halleri</em> lineage as well.</p><p>Functional divergence may not occur immediately after gene duplication. To determine whether gene duplication in the <em>A. halleri-lyrata</em> lineage affected selection pressures in the <em>A. halleri</em> lineage, we classified the 22,105 <em>AT–AL</em>–<em>AH</em> OGGs based on Illumina paired-end DNA-sequencing reads into 5,159 OGGs with gene duplications and 16,946 OGGs without any gene duplications, focusing on the <em>A. halleri-lyrata</em> lineage. The proportions of positive selection and purifying selection in the duplicated genes in the <em>A. halleri</em> lineage were compared with those in the non-duplicated genes. In this test, the null model was the ratio of duplicated genes to non-duplicated genes in the <em>A. halleri-lyrata</em> lineage. The proportion of positive selection in the duplicated genes was significantly higher than those in the non-duplicated genes (<em>P</em>-value = 1.3 × 10<sup>−72</sup> for positive selection by χ<sup>2</sup> test, <a reveal-id="dsy005-F4" data-open="dsy005-F4" class="link link-reveal link-table xref-fig">Fig. 4E</a>). The proportion of purifying selection in the non-duplicated genes was significantly higher than in the duplicated genes (<em>P</em>-value = 1.9 × 10<sup>−85</sup> by χ<sup>2</sup> test, <a reveal-id="dsy005-F4" data-open="dsy005-F4" class="link link-reveal link-table xref-fig">Fig. 4E</a>). We did the same analysis in 21,537 based on the available <em>A. halleri</em> genome. We again found the same trend (<em>P</em>-value = 4.8 × 10<sup>−81</sup> for positive selection, <em>P</em>-value = 1.1 × 10<sup>−39</sup> for purifying selection by χ<sup>2</sup> test, <a reveal-id="dsy005-F4" data-open="dsy005-F4" class="link link-reveal link-table xref-fig">Fig. 4F</a>). These results indicate that gene duplication in the <em>A. halleri-lyrata</em> lineage contributed to functional divergence in the <em>A. halleri</em> lineage.</p><p>To determine whether gene duplications in the <em>A. halleri-lyrata</em> or <em>A. halleri</em> lineage contributed to functional divergence in the <em>A. halleri</em> lineage, we focussed on two categories of OGGs—genes not duplicated in the <em>A. halleri-lyrata</em> lineage but duplicated in the <em>A. halleri</em> lineage (1,593 OGGs), and genes duplicated in the <em>A. halleri-lyrata</em> lineage but not duplicated in the <em>A. halleri</em> lineage (4,264 OGGs). The proportions of positive selection and purifying selection in the <em>A. halleri</em> lineage were compared in the two categories. In this test, the null model was the ratio of the two categories of OGGs (1,593 and 4,264 OGGs). The proportions of positive selection in genes duplicated only in the <em>A. halleri-lyrata</em> lineage were significantly higher than in genes duplicated only in the <em>A. halleri</em> lineage (<em>P</em>-value = 7.3 × 10<sup>−9</sup> for positive selection by χ<sup>2</sup> test, <a reveal-id="dsy005-F4" data-open="dsy005-F4" class="link link-reveal link-table xref-fig">Fig. 4G</a>). Conversely, the proportion of purifying selection in genes duplicated only in the <em>A. halleri-lyrata</em> lineage was significantly lower than in genes duplicated only in the <em>A. halleri</em> lineage (<em>P</em>-value = 2.9 × 10<sup>−8</sup> by χ<sup>2</sup> test, <a reveal-id="dsy005-F4" data-open="dsy005-F4" class="link link-reveal link-table xref-fig">Fig. 4G</a>). We did the same analysis in 21,537 based on based on the available <em>A. halleri</em> genome. We found the same trend (<em>P</em>-value = 0.04 for positive selection, <em>P</em>-value = 0.02 for purifying selection in the available genome by χ<sup>2</sup> test, <a reveal-id="dsy005-F4" data-open="dsy005-F4" class="link link-reveal link-table xref-fig">Fig. 4H</a>). These results indicate that gene duplication in the <em>A. halleri-lyrata</em> lineage was the main determinant of the elevated proportion of positive selection in the <em>A. halleri</em> lineage.</p>                            <h3 scrollto-destination=118057154 id="118057154" class="section-title" >3.5. Functional bias of genes under positive or purifying selection in the <em>A. halleri-lyrata</em> and <em>A. halleri</em> lineages</h3>
<p>We found that gene duplication contributed to functional divergence in comparison with non-duplicated genes in <em>Arabidopsis</em> but many duplicated genes had been retained with purifying selection, which induces an increase of gene dosage. Thus, we were interested in investigating the functional bias in duplicated and non-duplicated genes with positive/purifying selection in the <em>A. halleri-lyrata</em> and <em>A. halleri</em> lineages. OGGs based on Illumina paired-end DNA-sequencing reads were classified into duplicated and non-duplicated genes in the <em>A. halleri-lyrata</em> and <em>A. halleri</em> lineages. The OGGs were then further classified by positive and purifying selection. This gave a total of eight classes. In each class, we examined the degree of over-representation in 1,504 GO categories compared with the number of GO categories assigned to <em>A. thaliana</em> genes (see Section 2) (<a reveal-id="dsy005-F5" data-open="dsy005-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Although two classes with no duplication and purifying selection were clustered into one group, the other six classes were not essentially clustered with each other. These results indicate that the evolutionary direction of <em>A. halleri</em> was quite different in the <em>A. halleri-lyrata</em> and <em>A. halleri</em> lineages with respect to either gene dosage by gene duplication or functional divergence by positive selection. </p>                        <a id="118057156" scrollto-destination="118057156"></a>
<div id="dsy005-F5" data-id="dsy005-F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/m_dsy005f5.png?Expires=2147483647&Signature=PATBp-wmZ4Dkh02AXMDfZPfrisnK7PQd7OAjXabd9jZpm3fufsBj380Kn5-erRHTJaE6I-YSdCN8mWKO34rG6nK0fvxHkcuJCAn3aJKwop31vCYxqBiPdiCPVdf4cAF0uCVlSxawyAagwIWq3a7L7LHzCbh0N1u-7XLEYdddqz7zO7I~m7enwXTovuty~IXtTOP4JDvGU9I2wR1-DKU0x18pm2wmSwSXf7GscI6PIJth3byLcE2HgoYsDfu9knkJzOE6XcM2fmwDvzaGteMsCvO53Roz~UlChvmfVB-LDRqbrGDdjI1M2Bmiz137Ze5bC9EsI8fiDg5QC36HTJuvAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Over-represented functional categories. The X-axis represents different kinds of OGGs. OGGs were classified into duplicated genes and non-duplicated genes in the A. halleri-lyrata (AHL) and A. halleri (AH) lineages. The OGGs were then further classified by positive or purifying selection. The Y-axis represents 1,504 GO categories (biological processes) assigned to A. thaliana genes belonging to the OGGs. The key shows the relationship between colour and the z-score of over-representation in the GO categories. Red and green indicate low and high over-representation, respectively. The ratio of observed gene numbers in selected OGGs to expected gene numbers inferred from the data for all annotated genes was calculated in each GO category. " path-from-xml="dsy005f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057156/dsy005f5.tif" path-from-xml="dsy005f5.tif" target="_blank">View large</a><a section="118057156" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005f5.png?Expires=2147483647&Signature=L9W6GptHes~P4fC97-kYs~gYQYtWlc8Xhtr0YaXDADy1fDfYTl-qkklzRnnplyrQPs13jBHXzuY2zB7sUjRnPHY0KIiYA2Qj4jemlKjjFKzZy4NEBHd1xPVp13yqw98LraqzsXEVrLFsLGABDHgN3OAvV-Gk1X4bhu3o-Lo1c1GQ3NPMdhdKXVO9MkAbiYDm10BeznO9TXzXXhYLo2O1ZS5Q~lXQLW2~F24aDw2nhgV1LXQvGnLs4c-N9-gpMsywfeuGBy~uonfJx~kMFh4sNrUHKODuZbNf4LlBzJP1rfOzt~08CKaqFwuWWVF1VRdPjhnKBLt8oXL~G5iu3Ble-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057156&ar=4898128&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy005f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Over-represented functional categories. The X-axis represents different kinds of OGGs. OGGs were classified into duplicated genes and non-duplicated genes in the <em>A. halleri-lyrata</em> (AHL) and <em>A. halleri</em> (AH) lineages. The OGGs were then further classified by positive or purifying selection. The Y-axis represents 1,504 GO categories (biological processes) assigned to <em>A. thaliana</em> genes belonging to the OGGs. The key shows the relationship between colour and the z-score of over-representation in the GO categories. Red and green indicate low and high over-representation, respectively. The ratio of observed gene numbers in selected OGGs to expected gene numbers inferred from the data for all annotated genes was calculated in each GO category. </p></div></div><div content-id="dsy005-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/m_dsy005f5.png?Expires=2147483647&Signature=PATBp-wmZ4Dkh02AXMDfZPfrisnK7PQd7OAjXabd9jZpm3fufsBj380Kn5-erRHTJaE6I-YSdCN8mWKO34rG6nK0fvxHkcuJCAn3aJKwop31vCYxqBiPdiCPVdf4cAF0uCVlSxawyAagwIWq3a7L7LHzCbh0N1u-7XLEYdddqz7zO7I~m7enwXTovuty~IXtTOP4JDvGU9I2wR1-DKU0x18pm2wmSwSXf7GscI6PIJth3byLcE2HgoYsDfu9knkJzOE6XcM2fmwDvzaGteMsCvO53Roz~UlChvmfVB-LDRqbrGDdjI1M2Bmiz137Ze5bC9EsI8fiDg5QC36HTJuvAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Over-represented functional categories. The X-axis represents different kinds of OGGs. OGGs were classified into duplicated genes and non-duplicated genes in the A. halleri-lyrata (AHL) and A. halleri (AH) lineages. The OGGs were then further classified by positive or purifying selection. The Y-axis represents 1,504 GO categories (biological processes) assigned to A. thaliana genes belonging to the OGGs. The key shows the relationship between colour and the z-score of over-representation in the GO categories. Red and green indicate low and high over-representation, respectively. The ratio of observed gene numbers in selected OGGs to expected gene numbers inferred from the data for all annotated genes was calculated in each GO category. " path-from-xml="dsy005f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057156/dsy005f5.tif" path-from-xml="dsy005f5.tif" target="_blank">View large</a><a section="118057156" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005f5.png?Expires=2147483647&Signature=L9W6GptHes~P4fC97-kYs~gYQYtWlc8Xhtr0YaXDADy1fDfYTl-qkklzRnnplyrQPs13jBHXzuY2zB7sUjRnPHY0KIiYA2Qj4jemlKjjFKzZy4NEBHd1xPVp13yqw98LraqzsXEVrLFsLGABDHgN3OAvV-Gk1X4bhu3o-Lo1c1GQ3NPMdhdKXVO9MkAbiYDm10BeznO9TXzXXhYLo2O1ZS5Q~lXQLW2~F24aDw2nhgV1LXQvGnLs4c-N9-gpMsywfeuGBy~uonfJx~kMFh4sNrUHKODuZbNf4LlBzJP1rfOzt~08CKaqFwuWWVF1VRdPjhnKBLt8oXL~G5iu3Ble-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057156&ar=4898128&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy005f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Over-represented functional categories. The X-axis represents different kinds of OGGs. OGGs were classified into duplicated genes and non-duplicated genes in the <em>A. halleri-lyrata</em> (AHL) and <em>A. halleri</em> (AH) lineages. The OGGs were then further classified by positive or purifying selection. The Y-axis represents 1,504 GO categories (biological processes) assigned to <em>A. thaliana</em> genes belonging to the OGGs. The key shows the relationship between colour and the z-score of over-representation in the GO categories. Red and green indicate low and high over-representation, respectively. The ratio of observed gene numbers in selected OGGs to expected gene numbers inferred from the data for all annotated genes was calculated in each GO category. </p></div></div><p>To examine the kinds of genes associated with phenotypic differences among <em>A. thaliana</em>, <em>A. lyrata</em> and <em>A. halleri</em>, we identified significantly over-represented GO categories in OGGs with gene duplication and purifying selection, OGGs with gene duplication and positive selection and OGGs with non-duplication and positive selection in the <em>A. halleri–A. lyrata</em> and <em>A. halleri</em> lineages. OGGs with non-duplication and purifying selection were disregarded in the analysis because such genes tended to have the same functions in <em>A. halleri</em>, <em>A. lyrata</em> and <em>A. halleri</em>. From the GO categories assigned to the <em>A. thaliana</em> genes belonging to the OGGs, over-represented GO categories were identified (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>; see Section 2, FDR &lt; 0.01).</p><p>When we focussed on metal and zinc tolerance or accumulation among <em>A. thaliana</em>, <em>A. lyrata</em> and <em>A. halleri</em>, both <em>A. lyrata</em> and <em>A. halleri</em> had a higher tolerance to metal and zinc than <em>A. thaliana.</em><a reveal-id="dsy005-B36" data-open="dsy005-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> In particular, both metal and zinc tend to be accumulated in <em>A. halleri</em> compared with <em>A. lyrata</em>.<a reveal-id="dsy005-B62" data-open="dsy005-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a><sup>,</sup><a reveal-id="dsy005-B63" data-open="dsy005-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> This observation suggests that metal tolerance is enhanced in the <em>A. halleri–A. lyrata</em> lineage, and that metal accumulation is enhanced in the <em>A. halleri</em> lineage. Genes associated with metal and zinc transporters/responses were highly duplicated with purifying selection in the <em>A. halleri-lyrata</em> lineage, indicating that the dosages of genes associated with metal responses and transporters had been enhanced in the <em>A. halleri-lyrata</em> lineage. Furthermore, genes associated with metal and zinc responses were highly duplicated with purifying selection in the <em>A. halleri-lyrata</em> lineage, indicating that the dosages of genes associated with metal and zinc responses had been enhanced in the <em>A. halleri-lyrata</em> lineage. Thus, tolerance or accumulation of metal and zinc was enhanced in the <em>A. halleri-lyrata</em> and <em>A. halleri</em> lineages through gene duplication.</p><p>Genes associated with the reproductive system, cell cycle, various developments, various metabolites, epigenetics, metabolites, abiotic and biotic responses were subject to positive selection in either the <em>A. halleri-lyrata</em> or <em>A. halleri</em> lineage (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). However, we do not have any clear idea why such genes were subject to positive selection. Additionally, genes associated with various ubiquitous processes tended to be duplicated in either the <em>A. halleri-lyrata</em> or the <em>A. halleri</em> lineage with purifying selection (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). These over-represented functional categories may contribute to phenotypic differences between <em>A. thaliana</em> and either <em>A. lyrata</em> or <em>A. halleri</em> through high dosages. However, we do not know the phenotypic differences associated with these functional categories. These duplicated genes might have been retained because increased gene dosages associated with these functional categories were not too disadvantageous for <em>A. halleri</em>. To avoid gaining novel functions, these genes may be under purifying selection. In the future, these duplicated genes may be lost if disadvantageous functions appear. Indeed, the <em>A. halleri-lyrata</em> and <em>A. halleri</em> lineages have extraordinarily high retention rates of duplicated genes in comparison with earlier plant lineages. These observations indicate that most of the duplicated genes in the <em>A. halleri-lyrata</em> and <em>A. halleri</em> lineages may be lost in future evolution.</p>                            <h3 scrollto-destination=118057160 id="118057160" class="section-title" >3.6. Concluding remarks</h3>
<p>In this analysis, we generated 25,833 <em>A. halleri</em> genes that were orthologous to 79.1% of the annotated <em>A. lyrata</em> genes from contigs generated from only paired-end reads of Illumina DNA-sequencing. On the other hands, we inferred 26,007 AH-AL OOGs based on the available <em>A. halleri</em> genome. Out of 32,670 <em>A. lyrata</em> genes, 79.6% were identified as orthologous genes to <em>A. halleri</em> genes. Thus, our method inferring orthologous genes is compatible to a method to infer orthologous genes based on the available genome. However, it has significant limitations for examining gene loss, exon shuffling and heterozygosity because it does not infer any new genes in the genomes. Nevertheless, the number of duplicated genes inferred by reading depth of Illumina DNA-sequencing reads is likely to be more reliable in comparison with duplicated genes identified on scaffolds generated by Illumina DNA-sequencing reads (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5B</a></span>). Therefore, this procedure is useful for inferring lineage-specific duplicated genes resulting from such events as SSDs.</p><p>The gain rates based on 25, 833 <em>AL</em>–<em>AH</em> OGGs based on both Illumina paired-end DNA-sequencing reads and the available <em>A. halleri</em> genome were 1.6–2.0 × 10<sup>−2</sup> per gene per MY in the <em>A. halleri-lyrata</em> lineage (<a reveal-id="dsy005-F2" data-open="dsy005-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). Furthermore, using the only mapping coverage of the Illumina DNA-sequencing reads, the gain rate was inferred to be 5.7–7.6 × 10<sup>−2</sup> in the <em>A. halleri</em> lineage because gene duplication tended to be missed in a genome assembly based on Illumina short reads. That is, the gain rates were inferred to be 1.6–2.0 and 5.7–7.6 × 10<sup>−2</sup> per gene per MY in the <em>A. halleri-lyrata</em> and <em>A. halleri</em> lineages, respectively. The gain rate in the <em>A. halleri</em> lineage was approximately four times higher than in the <em>A. halleri-lyrata</em> lineage. Using our previous data, we re-estimated the gain rates in the three time periods after the divergence of mosses, rice and poplar (<a reveal-id="dsy005-F2" data-open="dsy005-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). The inferred gain rates (1.8–3.0 × 10<sup>−3</sup>) were ∼10 times lower than in the <em>Arabidopsis</em> lineage (<a reveal-id="dsy005-F2" data-open="dsy005-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). Thus, gain rates tend to increase as the evolutionary period gets younger. One explanation for this gain rate difference is that duplicated genes tend to rapidly decay over time. This explanation is supported by a higher rate of pseudogenization in recently duplicated genes in comparison with non-duplicated genes (<a reveal-id="dsy005-T1" data-open="dsy005-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Also, several previous reports showed that younger duplicated genes tended to be relaxed compared with older duplicated genes.<a reveal-id="dsy005-B54" data-open="dsy005-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a><sup>,</sup><a reveal-id="dsy005-B55" data-open="dsy005-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a><sup>,</sup><a reveal-id="dsy005-B64 dsy005-B65 dsy005-B66 dsy005-B67" data-open="dsy005-B64 dsy005-B65 dsy005-B66 dsy005-B67" class="link link-ref link-reveal xref-bibr"><sup>64–67</sup></a> That is, most anciently duplicated genes tend not to be retained in current species. Consequently, the gain rates inferred in earlier evolutional periods tend to decrease.</p><p>To investigate the functional divergence of duplicated genes in the <em>A. halleri-lyrata</em> and <em>A. halleri</em> lineages, we identified OGGs under either positive or purifying selection in these lineages based on the ratio of nonsynonymous and synonymous substitution rates (<em>K</em><sub>A</sub>/<em>K</em><sub>S</sub>). Interestingly, the proportions of positive and purifying selection tended to increase and decrease, respectively, when gene duplication occurred in either the <em>A. halleri-lyrata</em> or <em>A. halleri</em> lineage. This result indicates that gene duplication tends to enhance functional divergence in comparison with non-duplicated genes in the <em>Arabidopsis</em> lineage. In contrast, the general observation is that duplicated genes tend to have less functional divergence in yeasts, plants and mammals.<a reveal-id="dsy005-B24" data-open="dsy005-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a><sup>,</sup><a reveal-id="dsy005-B68" data-open="dsy005-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a><sup>,</sup><a reveal-id="dsy005-B69" data-open="dsy005-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a> This is because functionally important genes are more likely to be retained as duplicates.<a reveal-id="dsy005-B68" data-open="dsy005-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> This contradictory relationship might derive from the duplication ages. Most previous analyses have examined recently observed selection pressures in anciently duplicated genes. When functional divergence was examined in recently duplicated genes, the duplicated genes tended to have higher functional divergence than singletons.<a reveal-id="dsy005-B70" data-open="dsy005-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a> Together, these results suggest duplicated genes tend to have higher functional divergence immediately after duplication than singletons.</p><p>How long gene duplication accelerates functional divergence remains an open question. To address this, we examined whether gene duplication in the <em>A. halleri-lyrata</em> lineage (2–10 MYA) accelerated functional divergence in the <em>A. halleri</em> lineage (&lt;2 MYA). Interestingly, we found that gene duplication in the <em>A. halleri</em>-<em>lyrata</em> lineage enhanced the proportion of positive selection in the <em>A. halleri</em> lineage (<a reveal-id="dsy005-F4" data-open="dsy005-F4" class="link link-reveal link-table xref-fig">Fig. 4E–H</a>). This result indicated that the functional divergence of duplicated genes was accelerated several MY after gene duplication. If gene duplication is too deleterious for a gene, the gene tends to be lost immediately after duplication. If not, duplicated genes may be retained for a long period without functional divergence because functional divergence may be evolutionarily disadvantageous. Therefore, immediately after duplication, most duplicated genes might be under functional constraints in comparison with genes duplicated several MY earlier. Indeed, many recently duplicated genes have functional redundancy in <em>A. thaliana</em><a reveal-id="dsy005-B17" data-open="dsy005-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsy005-B18" data-open="dsy005-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> and in mammals.<a reveal-id="dsy005-B71" data-open="dsy005-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> These young duplicated genes tend to be less functionally constrained than singletons, and may have the potential to obtain an essential function to survive in new environments.</p><p>Finally, we examined the kinds of genes that were duplicated and/or under positive selection in the <em>A. halleri-lyrata</em> and <em>A. halleri</em> lineages. Different functional categories tended to have experienced gene duplication and/or selection pressure in the <em>A. halleri-lyrata</em> and <em>A. halleri</em> lineages. For example, <em>A. halleri</em> is known as a heavy metal hyper-accumulator with high metal tolerance. <em>A. lyrata</em> is tolerant of heavy-metal ions in the soil to some degree but <em>A. thaliana</em> is not. Genes related to heavy-metal tolerance and accumulation tended to be highly duplicated with purifying selection in the <em>A. halleri-lyrata</em> and <em>A. halleri</em> lineages. Earlier studies reported that metal tolerance was enhanced by increasing gene dosage through gene duplication.<a reveal-id="dsy005-B41" data-open="dsy005-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> Our results supported this trend at a genomic scale. Taken together, the results of our study reveal that lineage-specific duplicated genes have contributed to species-specific evolution in <em>Arabidopsis</em>.</p>                            <h2 scrollto-destination=118057166 id="118057166" class="section-title" >4. Availability</h2>
<p>Illumina DNA-sequencing data (DRA004564) have been deposited in the DDBJ Sequence Read Archive (<a class="link link-uri" href="https://trace.ddbj.nig.ac.jp/dra/" target="_blank">https://trace.ddbj.nig.ac.jp/dra/</a>). Contig sequences (BFAE01000001-BFAE01344622) assembled from the Illumina DNA-sequencing data have been deposited in the DDBJ Mass Submission System. <em>A. halleri</em> gene sequences determined in this study are included in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S3 and S4</a></span>. </p>                            <h2 scrollto-destination=118057168 id="118057168" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Kiyomi Imamura, Makiko Tosaka, Taiji Kikuchi, Terumi Horiuchi, Kanako Onizuka and Miu Kubota for Illumina DNA-sequencing analyses and ddPCR analyses. We also thank the National Institute of Genetics of the Research Organization of Information and Systems for providing excellent supercomputer services. This work was supported by the Research for Evolutional Science and Technology (CREST) programme ‘Creation of essential technologies to utilize carbon dioxide as a resource through the enhancement of plant productivity and the exploitation of plant products’ of the Japan Science and Technology Agency (JST) (JPMJCR11B3 to K.H. and S.I.M.); Grants-in-Aid for Scientific Research (15K14421, 15H02433, K.H. and S.I.M.) and research grants from the Takeda Science Foundation, the Sumitomo Foundation, the Kurume Research Park and the Asahi Glass Foundation (to K.H.).</p>                            <h3 scrollto-destination=118057170 id="118057170" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=118057172 id="118057172" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h2 scrollto-destination=118057174 id="118057174" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsy005-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lockton</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Gaut</div> <div class="given-names">B. S.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Plant conserved non-coding sequences and paralogue evolution</div>, <div class="source">Trends Genet</div> ., <div class="volume">21</div>, <div class="fpage">60</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20conserved%20non-coding%20sequences%20and%20paralogue%20evolution&amp;author=S.%20Lockton&amp;author=B.%20S.%20Gaut&amp;publication_year=2005&amp;journal=Trends%20Genet&amp;volume=21&amp;pages=60-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2004.11.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2004.11.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2004.11.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15680516" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vanneste</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Baele</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Maere</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Van de Peer</div> <div class="given-names">Y.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Analysis of 41 plant genomes supports a wave of successful genome duplications in association with the Cretaceous-Paleogene boundary</div>, <div class="source">Genome Res</div> ., <div class="volume">24</div>, <div class="fpage">1334</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%2041%20plant%20genomes%20supports%20a%20wave%20of%20successful%20genome%20duplications%20in%20association%20with%20the%20Cretaceous-Paleogene%20boundary&amp;author=K.%20Vanneste&amp;author=G.%20Baele&amp;author=S.%20Maere&amp;author=Y.%20Van%20de%20Peer&amp;publication_year=2014&amp;journal=Genome%20Res&amp;volume=24&amp;pages=1334-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.168997.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.168997.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.168997.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24835588" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hanada</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Vallejo</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Nobuta</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">The functional role of pack-MULEs in rice inferred from purifying selection and expression profile</div>, <div class="source">Plant Cell</div> , <div class="volume">21</div>, <div class="fpage">25</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20functional%20role%20of%20pack-MULEs%20in%20rice%20inferred%20from%20purifying%20selection%20and%20expression%20profile&amp;author=K.%20Hanada&amp;author=V.%20Vallejo&amp;author=K.%20Nobuta&amp;publication_year=2009&amp;journal=Plant%20Cell&amp;volume=21&amp;pages=25-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.108.063206" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.108.063206" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.108.063206"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19136648" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hanada</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Zou</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Lehti-Shiu</div> <div class="given-names">M. D.</div></div>, <div class="name string-name"><div class="surname">Shinozaki</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Shiu</div> <div class="given-names">S. H.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Importance of lineage-specific expansion of plant tandem duplicates in the adaptive response to environmental stimuli</div>, <div class="source">Plant Physiol</div> ., <div class="volume">148</div>, <div class="fpage">993</div>–<div class="lpage">1003</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Importance%20of%20lineage-specific%20expansion%20of%20plant%20tandem%20duplicates%20in%20the%20adaptive%20response%20to%20environmental%20stimuli&amp;author=K.%20Hanada&amp;author=C.%20Zou&amp;author=M.%20D.%20Lehti-Shiu&amp;author=K.%20Shinozaki&amp;author=S.%20H.%20Shiu&amp;publication_year=2008&amp;journal=Plant%20Physiol&amp;volume=148&amp;pages=993-1003" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.108.122457" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.108.122457" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.108.122457"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18715958" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fortna</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">MacLaren</div> <div class="given-names">E.</div></div></div>, et al.   <div class="year">2004</div>, 
<div class="article-title">Lineage-specific gene duplication and loss in human and great ape evolution</div>, <div class="source">PLoS Biol</div> ., <div class="volume">2</div>, <div class="fpage">E207</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lineage-specific%20gene%20duplication%20and%20loss%20in%20human%20and%20great%20ape%20evolution&amp;author=A.%20Fortna&amp;author=Y.%20Kim&amp;author=E.%20MacLaren&amp;publication_year=2004&amp;journal=PLoS%20Biol&amp;volume=2&amp;pages=E207" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0020207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0020207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0020207"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15252450" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rostoks</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Borevitz</div> <div class="given-names">J. O.</div></div>, <div class="name string-name"><div class="surname">Hedley</div> <div class="given-names">P. E.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">Single-feature polymorphism discovery in the barley transcriptome</div>, <div class="source">Genome Biol</div> ., <div class="volume">6</div>, <div class="fpage">R54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single-feature%20polymorphism%20discovery%20in%20the%20barley%20transcriptome&amp;author=N.%20Rostoks&amp;author=J.%20O.%20Borevitz&amp;author=P.%20E.%20Hedley&amp;publication_year=2005&amp;journal=Genome%20Biol&amp;volume=6&amp;pages=R54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2005-6-6-r54" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2005-6-6-r54" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2005-6-6-r54"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15960806" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Clark</div> <div class="given-names">R. M.</div></div>, <div class="name string-name"><div class="surname">Schweikert</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Toomajian</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">Common sequence polymorphisms shaping genetic diversity in Arabidopsis thaliana</div>, <div class="source">Science</div> , <div class="volume">317</div>, <div class="fpage">338</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Common%20sequence%20polymorphisms%20shaping%20genetic%20diversity%20in%20Arabidopsis%20thaliana&amp;author=R.%20M.%20Clark&amp;author=G.%20Schweikert&amp;author=C.%20Toomajian&amp;publication_year=2007&amp;journal=Science&amp;volume=317&amp;pages=338-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1138632" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1138632" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1138632"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17641193" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rizzon</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Ponger</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Gaut</div> <div class="given-names">B. S.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Striking similarities in the genomic distribution of tandemly arrayed genes in Arabidopsis and rice</div>, <div class="source">PLoS Comput Biol</div> ., <div class="volume">2</div>, <div class="fpage">e115</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Striking%20similarities%20in%20the%20genomic%20distribution%20of%20tandemly%20arrayed%20genes%20in%20Arabidopsis%20and%20rice&amp;author=C.%20Rizzon&amp;author=L.%20Ponger&amp;author=B.%20S.%20Gaut&amp;publication_year=2006&amp;journal=PLoS%20Comput%20Biol&amp;volume=2&amp;pages=e115" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.0020115" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.0020115" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.0020115"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16948529" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rodgers-Melnick</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Mane</div> <div class="given-names">S. P.</div></div>, <div class="name string-name"><div class="surname">Dharmawardhana</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Contrasting patterns of evolution following whole genome versus tandem duplication events in Populus</div>, <div class="source">Genome Res</div> ., <div class="volume">22</div>, <div class="fpage">95</div>–<div class="lpage">105</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Contrasting%20patterns%20of%20evolution%20following%20whole%20genome%20versus%20tandem%20duplication%20events%20in%20Populus&amp;author=E.%20Rodgers-Melnick&amp;author=S.%20P.%20Mane&amp;author=P.%20Dharmawardhana&amp;publication_year=2012&amp;journal=Genome%20Res&amp;volume=22&amp;pages=95-105" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.125146.111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.125146.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.125146.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21974993" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Leister</div> <div class="given-names">D.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Tandem and segmental gene duplication and recombination in the evolution of plant disease resistance gene</div>, <div class="source">Trends Genet</div> ., <div class="volume">20</div>, <div class="fpage">116</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tandem%20and%20segmental%20gene%20duplication%20and%20recombination%20in%20the%20evolution%20of%20plant%20disease%20resistance%20gene&amp;author=D.%20Leister&amp;publication_year=2004&amp;journal=Trends%20Genet&amp;volume=20&amp;pages=116-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2004.01.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2004.01.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2004.01.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15049302" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ohno</div> <div class="given-names">S.</div></div></div> <div class="year">1970</div>, <div class="source">Evolution by Gene Duplication</div> . 
<div class="publisher-name">Springer-Verlag</div>: 
<div class="publisher-loc">New York</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20by%20Gene%20Duplication&amp;author=S.%20Ohno&amp;publication_year=1970&amp;book=Evolution%20by%20Gene%20Duplication" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/978-3-642-86659-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F978-3-642-86659-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2F978-3-642-86659-3"> </span></div> </div></div></div></div></div><div content-id="dsy005-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Force</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Lynch</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pickett</div> <div class="given-names">F. B.</div></div>, <div class="name string-name"><div class="surname">Amores</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Yan</div> <div class="given-names">Y. L.</div></div>, <div class="name string-name"><div class="surname">Postlethwait</div> <div class="given-names">J.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Preservation of duplicate genes by complementary, degenerative mutations</div>, <div class="source">Genetics</div> , <div class="volume">151</div>, <div class="fpage">1531</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Preservation%20of%20duplicate%20genes%20by%20complementary%2C%20degenerative%20mutations&amp;author=A.%20Force&amp;author=M.%20Lynch&amp;author=F.%20B.%20Pickett&amp;author=A.%20Amores&amp;author=Y.%20L.%20Yan&amp;author=J.%20Postlethwait&amp;publication_year=1999&amp;journal=Genetics&amp;volume=151&amp;pages=1531-45" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10101175" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zou</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Lehti-Shiu</div> <div class="given-names">M. D.</div></div>, <div class="name string-name"><div class="surname">Thibaud-Nissen</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Prakash</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Buell</div> <div class="given-names">C. R.</div></div>, <div class="name string-name"><div class="surname">Shiu</div> <div class="given-names">S. H.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Evolutionary and expression signatures of pseudogenes in Arabidopsis and rice</div>, <div class="source">Plant Physiol</div> ., <div class="volume">151</div>, <div class="fpage">3</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20and%20expression%20signatures%20of%20pseudogenes%20in%20Arabidopsis%20and%20rice&amp;author=C.%20Zou&amp;author=M.%20D.%20Lehti-Shiu&amp;author=F.%20Thibaud-Nissen&amp;author=T.%20Prakash&amp;author=C.%20R.%20Buell&amp;author=S.%20H.%20Shiu&amp;publication_year=2009&amp;journal=Plant%20Physiol&amp;volume=151&amp;pages=3-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.109.140632" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.109.140632" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.109.140632"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19641029" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zheng</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Gerstein</div> <div class="given-names">M. B.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">The ambiguous boundary between genes and pseudogenes: the dead rise up, or do they?</div> <div class="source">Trends Genet</div> ., <div class="volume">23</div>, <div class="fpage">219</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20ambiguous%20boundary%20between%20genes%20and%20pseudogenes%3A%20the%20dead%20rise%20up%2C%20or%20do%20they%3F&amp;author=D.%20Zheng&amp;author=M.%20B.%20Gerstein&amp;publication_year=2007&amp;journal=Trends%20Genet&amp;volume=23&amp;pages=219-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2007.03.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2007.03.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2007.03.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17382428" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Conant</div> <div class="given-names">G. C.</div></div>, <div class="name string-name"><div class="surname">Wolfe</div> <div class="given-names">K. H.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Turning a hobby into a job: how duplicated genes find new functions</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">9</div>, <div class="fpage">938</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Turning%20a%20hobby%20into%20a%20job%3A%20how%20duplicated%20genes%20find%20new%20functions&amp;author=G.%20C.%20Conant&amp;author=K.%20H.%20Wolfe&amp;publication_year=2008&amp;journal=Nat.%20Rev.%20Genet&amp;volume=9&amp;pages=938-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2482" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2482" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2482"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19015656" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kondrashov</div> <div class="given-names">F. A.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Gene duplication as a mechanism of genomic adaptation to a changing environment</div>, <div class="source">Proc. Biol. Sci</div> ., <div class="volume">279</div>, <div class="fpage">5048</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20duplication%20as%20a%20mechanism%20of%20genomic%20adaptation%20to%20a%20changing%20environment&amp;author=F.%20A.%20Kondrashov&amp;publication_year=2012&amp;journal=Proc.%20Biol.%20Sci&amp;volume=279&amp;pages=5048-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rspb.2012.1108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frspb.2012.1108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frspb.2012.1108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22977152" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hanada</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kuromori</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Myouga</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Toyoda</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">W. H.</div></div>, <div class="name string-name"><div class="surname">Shinozaki</div> <div class="given-names">K.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Evolutionary persistence of functional compensation by duplicate genes in Arabidopsis</div>, <div class="source">Genome Biol. Evol</div> ., <div class="volume">1</div>, <div class="fpage">409</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20persistence%20of%20functional%20compensation%20by%20duplicate%20genes%20in%20Arabidopsis&amp;author=K.%20Hanada&amp;author=T.%20Kuromori&amp;author=F.%20Myouga&amp;author=T.%20Toyoda&amp;author=W.%20H.%20Li&amp;author=K.%20Shinozaki&amp;publication_year=2009&amp;journal=Genome%20Biol.%20Evol&amp;volume=1&amp;pages=409-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evp043" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevp043" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevp043"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20333209" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hanada</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Sawada</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kuromori</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Functional compensation of primary and secondary metabolites by duplicate genes in Arabidopsis thaliana</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">28</div>, <div class="fpage">377</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20compensation%20of%20primary%20and%20secondary%20metabolites%20by%20duplicate%20genes%20in%20Arabidopsis%20thaliana&amp;author=K.%20Hanada&amp;author=Y.%20Sawada&amp;author=T.%20Kuromori&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol&amp;volume=28&amp;pages=377-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msq204" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsq204" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsq204"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20736450" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nowak</div> <div class="given-names">M. A.</div></div>, <div class="name string-name"><div class="surname">Boerlijst</div> <div class="given-names">M. C.</div></div>, <div class="name string-name"><div class="surname">Cooke</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Smith</div> <div class="given-names">J. M.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Evolution of genetic redundancy</div>, <div class="source">Nature</div> , <div class="volume">388</div>, <div class="fpage">167</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20genetic%20redundancy&amp;author=M.%20A.%20Nowak&amp;author=M.%20C.%20Boerlijst&amp;author=J.%20Cooke&amp;author=J.%20M.%20Smith&amp;publication_year=1997&amp;journal=Nature&amp;volume=388&amp;pages=167-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/40618" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F40618" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F40618"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9217155" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hanada</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kuromori</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Myouga</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Toyoda</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Shinozaki</div> <div class="given-names">K.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Increased expression and protein divergence in duplicate genes is associated with morphological diversification</div>, <div class="source">PLoS Genet</div> ., <div class="volume">5</div>, <div class="fpage">e1000781</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Increased%20expression%20and%20protein%20divergence%20in%20duplicate%20genes%20is%20associated%20with%20morphological%20diversification&amp;author=K.%20Hanada&amp;author=T.%20Kuromori&amp;author=F.%20Myouga&amp;author=T.%20Toyoda&amp;author=K.%20Shinozaki&amp;publication_year=2009&amp;journal=PLoS%20Genet&amp;volume=5&amp;pages=e1000781" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1000781" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1000781" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1000781"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20041196" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Alvarez-Ponce</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Fares</div> <div class="given-names">M. A.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Evolutionary rate and duplicability in the Arabidopsis thaliana protein-protein interaction network</div>, <div class="source">Genome Biol. Evol</div> ., <div class="volume">4</div>, <div class="fpage">1263</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20rate%20and%20duplicability%20in%20the%20Arabidopsis%20thaliana%20protein-protein%20interaction%20network&amp;author=D.%20Alvarez-Ponce&amp;author=M.%20A.%20Fares&amp;publication_year=2012&amp;journal=Genome%20Biol.%20Evol&amp;volume=4&amp;pages=1263-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evs101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevs101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevs101"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23160177" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Warren</div> <div class="given-names">A. S.</div></div>, <div class="name string-name"><div class="surname">Anandakrishnan</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Functional bias in molecular evolution rate of Arabidopsis thaliana</div>, <div class="source">BMC Evol. Biol</div> ., <div class="volume">10</div>, <div class="fpage">125</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20bias%20in%20molecular%20evolution%20rate%20of%20Arabidopsis%20thaliana&amp;author=A.%20S.%20Warren&amp;author=R.%20Anandakrishnan&amp;author=L.%20Zhang&amp;publication_year=2010&amp;journal=BMC%20Evol.%20Biol&amp;volume=10&amp;pages=125" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-10-125" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-10-125" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-10-125"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20433764" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Marowsky</div> <div class="given-names">N. C.</div></div>, <div class="name string-name"><div class="surname">Fan</div> <div class="given-names">C.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Divergent evolutionary and expression patterns between lineage specific new duplicate genes and their parental paralogs in Arabidopsis thaliana</div>, <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e72362</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Divergent%20evolutionary%20and%20expression%20patterns%20between%20lineage%20specific%20new%20duplicate%20genes%20and%20their%20parental%20paralogs%20in%20Arabidopsis%20thaliana&amp;author=J.%20Wang&amp;author=N.%20C.%20Marowsky&amp;author=C.%20Fan&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e72362" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0072362" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0072362" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0072362"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24009676" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Gaut</div> <div class="given-names">B. S.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Factors that contribute to variation in evolutionary rate among Arabidopsis genes</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">28</div>, <div class="fpage">2359</div>–<div class="lpage">69</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Factors%20that%20contribute%20to%20variation%20in%20evolutionary%20rate%20among%20Arabidopsis%20genes&amp;author=L.%20Yang&amp;author=B.%20S.%20Gaut&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol&amp;volume=28&amp;pages=2359-69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr058" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr058" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr058"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21389272" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wendel</div> <div class="given-names">J. F.</div></div>, <div class="name string-name"><div class="surname">Jackson</div> <div class="given-names">S. A.</div></div>, <div class="name string-name"><div class="surname">Meyers</div> <div class="given-names">B. C.</div></div>, <div class="name string-name"><div class="surname">Wing</div> <div class="given-names">R. A.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Evolution of plant genome architecture</div>, <div class="source">Genome Biol.</div>  , <div class="volume">17</div>, <div class="fpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20plant%20genome%20architecture&amp;author=J.%20F.%20Wendel&amp;author=S.%20A.%20Jackson&amp;author=B.%20C.%20Meyers&amp;author=R.%20A.%20Wing&amp;publication_year=2016&amp;journal=Genome%20Biol.&amp;volume=17&amp;pages=37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-016-0908-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-016-0908-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-016-0908-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26926526" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">DeBarry</div> <div class="given-names">J. D.</div></div>, <div class="name string-name"><div class="surname">Kissinger</div> <div class="given-names">J. C.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">A survey of innovation through duplication in the reduced genomes of twelve parasites</div>, <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e99213</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20survey%20of%20innovation%20through%20duplication%20in%20the%20reduced%20genomes%20of%20twelve%20parasites&amp;author=J.%20D.%20DeBarry&amp;author=J.%20C.%20Kissinger&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e99213" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0099213" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0099213" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0099213"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24919110" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sin</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Street</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Lundeberg</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Arvestad</div> <div class="given-names">L.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Improved gap size estimation for scaffolding algorithms</div>, <div class="source">Bioinformatics</div> , <div class="volume">28</div>, <div class="fpage">2215</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20gap%20size%20estimation%20for%20scaffolding%20algorithms&amp;author=K.%20Sin&amp;author=N.%20Street&amp;author=J.%20Lundeberg&amp;author=L.%20Arvestad&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=2215-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts441" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts441" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbts441"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22923455" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="collab">Arabidopsis_Genome_Initiative</div> <div class="year">2000</div>, 
<div class="article-title">Analysis of the genome sequence of the flowering plant Arabidopsis thaliana</div>, <div class="source">Nature</div> , <div class="volume">408</div>, <div class="fpage">796</div>–<div class="lpage">815</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35048692" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35048692" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35048692"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11130711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hu</div> <div class="given-names">T. T.</div></div>, <div class="name string-name"><div class="surname">Pattyn</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Bakker</div> <div class="given-names">E. G.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">The Arabidopsis lyrata genome sequence and the basis of rapid genome size change</div>, <div class="source">Nature Genetics</div> , <div class="volume">43</div>, <div class="fpage">476</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Arabidopsis%20lyrata%20genome%20sequence%20and%20the%20basis%20of%20rapid%20genome%20size%20change&amp;author=T.%20T.%20Hu&amp;author=P.%20Pattyn&amp;author=E.%20G.%20Bakker&amp;publication_year=2011&amp;journal=Nature%20Genetics&amp;volume=43&amp;pages=476-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.807" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.807" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.807"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21478890" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Beilstein</div> <div class="given-names">M. A.</div></div>, <div class="name string-name"><div class="surname">Nagalingum</div> <div class="given-names">N. S.</div></div>, <div class="name string-name"><div class="surname">Clements</div> <div class="given-names">M. D.</div></div>, <div class="name string-name"><div class="surname">Manchester</div> <div class="given-names">S. R.</div></div>, <div class="name string-name"><div class="surname">Mathews</div> <div class="given-names">S.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Dated molecular phylogenies indicate a Miocene origin for Arabidopsis thaliana</div>, <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">107</div>, <div class="fpage">18724</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dated%20molecular%20phylogenies%20indicate%20a%20Miocene%20origin%20for%20Arabidopsis%20thaliana&amp;author=M.%20A.%20Beilstein&amp;author=N.%20S.%20Nagalingum&amp;author=M.%20D.%20Clements&amp;author=S.%20R.%20Manchester&amp;author=S.%20Mathews&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=107&amp;pages=18724-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0909766107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0909766107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0909766107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20921408" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Moghe</div> <div class="given-names">G. D.</div></div>, <div class="name string-name"><div class="surname">Hufnagel</div> <div class="given-names">D. E.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Consequences of whole-genome triplication as revealed by comparative genomic analyses of the wild radish raphanus raphanistrum and three other Brassicaceae species</div>, <div class="source">Plant Cell</div> , <div class="volume">26</div>, <div class="fpage">1925</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Consequences%20of%20whole-genome%20triplication%20as%20revealed%20by%20comparative%20genomic%20analyses%20of%20the%20wild%20radish%20raphanus%20raphanistrum%20and%20three%20other%20Brassicaceae%20species&amp;author=G.%20D.%20Moghe&amp;author=D.%20E.%20Hufnagel&amp;author=H.%20Tang&amp;publication_year=2014&amp;journal=Plant%20Cell&amp;volume=26&amp;pages=1925-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.114.124297" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.114.124297" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.114.124297"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24876251" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Koenig</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Weigel</div> <div class="given-names">D.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Beyond the thale: comparative genomics and genetics of Arabidopsis relatives</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">16</div>, <div class="fpage">285</div>–<div class="lpage">98</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Beyond%20the%20thale%3A%20comparative%20genomics%20and%20genetics%20of%20Arabidopsis%20relatives&amp;author=D.%20Koenig&amp;author=D.%20Weigel&amp;publication_year=2015&amp;journal=Nat.%20Rev.%20Genet&amp;volume=16&amp;pages=285-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3883" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3883" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3883"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25854181" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kramer</div> <div class="given-names">U.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Metal hyperaccumulation in plants</div>, <div class="source">Ann. Rev. Plant Biol</div> ., <div class="volume">61</div>, <div class="fpage">517</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metal%20hyperaccumulation%20in%20plants&amp;author=U.%20Kramer&amp;publication_year=2010&amp;journal=Ann.%20Rev.%20Plant%20Biol&amp;volume=61&amp;pages=517-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-arplant-042809-112156" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-arplant-042809-112156" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-arplant-042809-112156"> </span></div> </div></div></div></div></div><div content-id="dsy005-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Verbruggen</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Hermans</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Schat</div> <div class="given-names">H.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Molecular mechanisms of metal hyperaccumulation in plants</div>, <div class="source">New Phytol</div> ., <div class="volume">181</div>, <div class="fpage">759</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20mechanisms%20of%20metal%20hyperaccumulation%20in%20plants&amp;author=N.%20Verbruggen&amp;author=C.%20Hermans&amp;author=H.%20Schat&amp;publication_year=2009&amp;journal=New%20Phytol&amp;volume=181&amp;pages=759-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2008.02748.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2008.02748.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2008.02748.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19192189" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shimizu</div> <div class="given-names">K. K.</div></div>, <div class="name string-name"><div class="surname">Purugganan</div> <div class="given-names">M. D.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Evolutionary and ecological genomics of Arabidopsis</div>, <div class="source">Plant Physiol</div> ., <div class="volume">138</div>, <div class="fpage">578</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20and%20ecological%20genomics%20of%20Arabidopsis&amp;author=K.%20K.%20Shimizu&amp;author=M.%20D.%20Purugganan&amp;publication_year=2005&amp;journal=Plant%20Physiol&amp;volume=138&amp;pages=578-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.105.061655" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.105.061655" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.105.061655"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15955921" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Turner</div> <div class="given-names">T. L.</div></div>, <div class="name string-name"><div class="surname">Bourne</div> <div class="given-names">E. C.</div></div>, <div class="name string-name"><div class="surname">Von Wettberg</div> <div class="given-names">E. J.</div></div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">T. T.</div></div>, <div class="name string-name"><div class="surname">Nuzhdin</div> <div class="given-names">S. V.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Population resequencing reveals local adaptation of Arabidopsis lyrata to serpentine soils</div>, <div class="source">Nat. Genet</div> ., <div class="volume">42</div>, <div class="fpage">260</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20resequencing%20reveals%20local%20adaptation%20of%20Arabidopsis%20lyrata%20to%20serpentine%20soils&amp;author=T.%20L.%20Turner&amp;author=E.%20C.%20Bourne&amp;author=E.%20J.%20Von%20Wettberg&amp;author=T.%20T.%20Hu&amp;author=S.%20V.%20Nuzhdin&amp;publication_year=2010&amp;journal=Nat.%20Genet&amp;volume=42&amp;pages=260-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.515" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.515" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.515"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20101244" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Novikova</div> <div class="given-names">P. Y.</div></div>, <div class="name string-name"><div class="surname">Hohmann</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Nizhynska</div> <div class="given-names">V.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Sequencing of the genus Arabidopsis identifies a complex history of nonbifurcating speciation and abundant trans-specific polymorphism</div>, <div class="source">Nat. Genet</div> , <div class="volume">48</div>, <div class="fpage">1077</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20of%20the%20genus%20Arabidopsis%20identifies%20a%20complex%20history%20of%20nonbifurcating%20speciation%20and%20abundant%20trans-specific%20polymorphism&amp;author=P.%20Y.%20Novikova&amp;author=N.%20Hohmann&amp;author=V.%20Nizhynska&amp;publication_year=2016&amp;journal=Nat.%20Genet&amp;volume=48&amp;pages=1077-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.3617" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.3617" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.3617"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27428747" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Briskine</div> <div class="given-names">R. V.</div></div>, <div class="name string-name"><div class="surname">Paape</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Shimizu-Inatsugi</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Genome assembly and annotation of Arabidopsis halleri, a model for heavy metal hyperaccumulation and evolutionary ecology</div>, <div class="source">Mol. Ecol. Resour</div> ., <div class="volume">17</div>, <div class="fpage">1025</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20assembly%20and%20annotation%20of%20Arabidopsis%20halleri%2C%20a%20model%20for%20heavy%20metal%20hyperaccumulation%20and%20evolutionary%20ecology&amp;author=R.%20V.%20Briskine&amp;author=T.%20Paape&amp;author=R.%20Shimizu-Inatsugi&amp;publication_year=2017&amp;journal=Mol.%20Ecol.%20Resour&amp;volume=17&amp;pages=1025-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/men.2017.17.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmen.2017.17.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmen.2017.17.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27671113" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Akama</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Shimizu-Inatsugi</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Shimizu</div> <div class="given-names">K. K.</div></div>, <div class="name string-name"><div class="surname">Sese</div> <div class="given-names">J.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Genome-wide quantification of homeolog expression ratio revealed nonstochastic gene regulation in synthetic allopolyploid Arabidopsis</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">e46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20quantification%20of%20homeolog%20expression%20ratio%20revealed%20nonstochastic%20gene%20regulation%20in%20synthetic%20allopolyploid%20Arabidopsis&amp;author=S.%20Akama&amp;author=R.%20Shimizu-Inatsugi&amp;author=K.%20K.%20Shimizu&amp;author=J.%20Sese&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=e46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24423873" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sato</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kudoh</div> <div class="given-names">H.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Fine-scale genetic differentiation of a temperate herb: relevance of local environments and demographic change</div>, <div class="source">AoB Plants</div> , <div class="volume">6</div>, <div class="fpage">plu070</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fine-scale%20genetic%20differentiation%20of%20a%20temperate%20herb%3A%20relevance%20of%20local%20environments%20and%20demographic%20change&amp;author=Y.%20Sato&amp;author=H.%20Kudoh&amp;publication_year=2014&amp;journal=AoB%20Plants&amp;volume=6&amp;pages=plu070" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/aobpla/plu070" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faobpla%2Fplu070" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Faobpla%2Fplu070"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25387749" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hanikenne</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Talke</div> <div class="given-names">I. N.</div></div>, <div class="name string-name"><div class="surname">Haydon</div> <div class="given-names">M. J.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">Evolution of metal hyperaccumulation required cis-regulatory changes and triplication of HMA4</div>, <div class="source">Nature</div> , <div class="volume">453</div>, <div class="fpage">391</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20metal%20hyperaccumulation%20required%20cis-regulatory%20changes%20and%20triplication%20of%20HMA4&amp;author=M.%20Hanikenne&amp;author=I.%20N.%20Talke&amp;author=M.%20J.%20Haydon&amp;publication_year=2008&amp;journal=Nature&amp;volume=453&amp;pages=391-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06877" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06877" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06877"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18425111" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Al-Shehbaz</div> <div class="given-names">I. A.</div></div>, <div class="name string-name"><div class="surname">O'Kane</div> <div class="given-names">S. L.</div>Jr</div></div> <div class="year">2002</div>, Taxonomy and phylogeny of Arabidopsis (Brassicaceae). In <em>The Arabidopsis Book/American Society of Plant Biologists</em>, Vol. 1.</div></div></div></div><div content-id="dsy005-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Boratyn</div> <div class="given-names">G. M.</div></div>, <div class="name string-name"><div class="surname">Camacho</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Cooper</div> <div class="given-names">P. S.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">BLAST: a more efficient report with usability improvements</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">W29</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLAST%3A%20a%20more%20efficient%20report%20with%20usability%20improvements&amp;author=G.%20M.%20Boratyn&amp;author=C.%20Camacho&amp;author=P.%20S.%20Cooper&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=W29-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt282" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt282" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt282"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23609542" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Johnston</div> <div class="given-names">J. S.</div></div>, <div class="name string-name"><div class="surname">Pepper</div> <div class="given-names">A. E.</div></div>, <div class="name string-name"><div class="surname">Hall</div> <div class="given-names">A. E.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">Evolution of genome size in Brassicaceae</div>, <div class="source">Ann. Bot</div> ., <div class="volume">95</div>, <div class="fpage">229</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20genome%20size%20in%20Brassicaceae&amp;author=J.%20S.%20Johnston&amp;author=A.%20E.%20Pepper&amp;author=A.%20E.%20Hall&amp;publication_year=2005&amp;journal=Ann.%20Bot&amp;volume=95&amp;pages=229-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/aob/mci016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmci016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Faob%2Fmci016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15596470" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kajitani</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Toshimoto</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Noguchi</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads</div>, <div class="source">Genome Res</div> ., <div class="volume">24</div>, <div class="fpage">1384</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20de%20novo%20assembly%20of%20highly%20heterozygous%20genomes%20from%20whole-genome%20shotgun%20short%20reads&amp;author=R.%20Kajitani&amp;author=K.%20Toshimoto&amp;author=H.%20Noguchi&amp;publication_year=2014&amp;journal=Genome%20Res&amp;volume=24&amp;pages=1384-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.170720.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.170720.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.170720.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24755901" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Simpson</div> <div class="given-names">J. T.</div></div>, <div class="name string-name"><div class="surname">Wong</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Jackman</div> <div class="given-names">S. D.</div></div>, <div class="name string-name"><div class="surname">Schein</div> <div class="given-names">J. E.</div></div>, <div class="name string-name"><div class="surname">Jones</div> <div class="given-names">S. J.</div></div>, <div class="name string-name"><div class="surname">Birol</div> <div class="given-names">I.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">ABySS: a parallel assembler for short read sequence data</div>, <div class="source">Genome Res</div> ., <div class="volume">19</div>, <div class="fpage">1117</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ABySS%3A%20a%20parallel%20assembler%20for%20short%20read%20sequence%20data&amp;author=J.%20T.%20Simpson&amp;author=K.%20Wong&amp;author=S.%20D.%20Jackman&amp;author=J.%20E.%20Schein&amp;author=S.%20J.%20Jones&amp;author=I.%20Birol&amp;publication_year=2009&amp;journal=Genome%20Res&amp;volume=19&amp;pages=1117-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.089532.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.089532.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.089532.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19251739" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Luo</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Xie</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler</div>, <div class="source">Gigasci</div> ., <div class="volume">1</div>, <div class="fpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SOAPdenovo2%3A%20an%20empirically%20improved%20memory-efficient%20short-read%20de%20novo%20assembler&amp;author=R.%20Luo&amp;author=B.%20Liu&amp;author=Y.%20Xie&amp;publication_year=2012&amp;journal=Gigasci&amp;volume=1&amp;pages=18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/2047-217X-1-18" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F2047-217X-1-18" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F2047-217X-1-18"> </span></div> </div></div></div></div></div><div content-id="dsy005-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">T. D.</div></div>, <div class="name string-name"><div class="surname">Watanabe</div> <div class="given-names">C. K.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">GMAP: a genomic mapping and alignment program for mRNA and EST sequences</div>, <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">1859</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GMAP%3A%20a%20genomic%20mapping%20and%20alignment%20program%20for%20mRNA%20and%20EST%20sequences&amp;author=T.%20D.%20Wu&amp;author=C.%20K.%20Watanabe&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=1859-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti310" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti310" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti310"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15728110" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Langdon</div> <div class="given-names">W. B.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Performance of genetic programming optimised Bowtie2 on genome comparison and analytic testing (GCAT) benchmarks</div>, <div class="source">BioData Min</div> ., <div class="volume">8</div>, <div class="fpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Performance%20of%20genetic%20programming%20optimised%20Bowtie2%20on%20genome%20comparison%20and%20analytic%20testing%20(GCAT)%20benchmarks&amp;author=W.%20B.%20Langdon&amp;publication_year=2015&amp;journal=BioData%20Min&amp;volume=8&amp;pages=1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13040-014-0034-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13040-014-0034-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13040-014-0034-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25621011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Untergasser</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Nijveen</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Rao</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Bisseling</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Geurts</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Leunissen</div> <div class="given-names">J. A.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Primer3Plus, an enhanced web interface to Primer3</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">35</div>, <div class="fpage">W71</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Primer3Plus%2C%20an%20enhanced%20web%20interface%20to%20Primer3&amp;author=A.%20Untergasser&amp;author=H.%20Nijveen&amp;author=X.%20Rao&amp;author=T.%20Bisseling&amp;author=R.%20Geurts&amp;author=J.%20A.%20Leunissen&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=W71-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm306" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm306" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm306"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17485472" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Remm</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Storm</div> <div class="given-names">C. E.</div></div>, <div class="name string-name"><div class="surname">Sonnhammer</div> <div class="given-names">E. L.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Automatic clustering of orthologs and in-paralogs from pairwise species comparisons</div>, <div class="source">J. Mol. Biol</div> ., <div class="volume">314</div>, <div class="fpage">1041</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automatic%20clustering%20of%20orthologs%20and%20in-paralogs%20from%20pairwise%20species%20comparisons&amp;author=M.%20Remm&amp;author=C.%20E.%20Storm&amp;author=E.%20L.%20Sonnhammer&amp;publication_year=2001&amp;journal=J.%20Mol.%20Biol&amp;volume=314&amp;pages=1041-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.2000.5197" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.2000.5197" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.2000.5197"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11743721" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Katoh</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Misawa</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kuma</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Miyata</div> <div class="given-names">T.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">MAFFT: a novel method for rapid multiple sequence alignment based on fast Fourier transform</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">30</div>, <div class="fpage">3059</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAFFT%3A%20a%20novel%20method%20for%20rapid%20multiple%20sequence%20alignment%20based%20on%20fast%20Fourier%20transform&amp;author=K.%20Katoh&amp;author=K.%20Misawa&amp;author=K.%20Kuma&amp;author=T.%20Miyata&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res&amp;volume=30&amp;pages=3059-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkf436" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkf436" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkf436"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12136088" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">PAML 4: phylogenetic analysis by maximum likelihood</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">24</div>, <div class="fpage">1586</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PAML%204%3A%20phylogenetic%20analysis%20by%20maximum%20likelihood&amp;author=Z.%20Yang&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol&amp;volume=24&amp;pages=1586-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm088" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm088" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm088"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17483113" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lynch</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Conery</div> <div class="given-names">J. S.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">The evolutionary fate and consequences of duplicate genes</div>, <div class="source">Science</div> , <div class="volume">290</div>, <div class="fpage">1151</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolutionary%20fate%20and%20consequences%20of%20duplicate%20genes&amp;author=M.%20Lynch&amp;author=J.%20S.%20Conery&amp;publication_year=2000&amp;journal=Science&amp;volume=290&amp;pages=1151-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.290.5494.1151" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.290.5494.1151" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.290.5494.1151"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11073452" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bu</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Katju</div> <div class="given-names">V.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Early evolutionary history and genomic features of gene duplicates in the human genome</div>, <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">621</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Early%20evolutionary%20history%20and%20genomic%20features%20of%20gene%20duplicates%20in%20the%20human%20genome&amp;author=L.%20Bu&amp;author=V.%20Katju&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=621" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1827-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1827-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1827-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26290067" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gao</div> <div class="given-names">L. Z.</div></div>, <div class="name string-name"><div class="surname">Innan</div> <div class="given-names">H.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Very low gene duplication rate in the yeast genome</div>, <div class="source">Science</div> , <div class="volume">306</div>, <div class="fpage">1367</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Very%20low%20gene%20duplication%20rate%20in%20the%20yeast%20genome&amp;author=L.%20Z.%20Gao&amp;author=H.%20Innan&amp;publication_year=2004&amp;journal=Science&amp;volume=306&amp;pages=1367-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1102033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1102033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1102033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15550669" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Heredia</div> <div class="given-names">N. J.</div></div>, <div class="name string-name"><div class="surname">Belgrader</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Droplet Digital PCR quantitation of HER2 expression in FFPE breast cancer samples</div>, <div class="source">Methods</div> , <div class="volume">59</div>, <div class="fpage">S20</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Droplet%20Digital%20PCR%20quantitation%20of%20HER2%20expression%20in%20FFPE%20breast%20cancer%20samples&amp;author=N.%20J.%20Heredia&amp;author=P.%20Belgrader&amp;author=S.%20Wang&amp;publication_year=2013&amp;journal=Methods&amp;volume=59&amp;pages=S20-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ymeth.2012.09.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ymeth.2012.09.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ymeth.2012.09.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23036330" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Drager</div> <div class="given-names">D. B.</div></div>, <div class="name string-name"><div class="surname">Desbrosses-Fonrouge</div> <div class="given-names">A. G.</div></div>, <div class="name string-name"><div class="surname">Krach</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2004</div>, 
<div class="article-title">Two genes encoding Arabidopsis halleri MTP1 metal transport proteins co-segregate with zinc tolerance and account for high MTP1 transcript levels</div>, <div class="source">Plant J</div> ., <div class="volume">39</div>, <div class="fpage">425</div>–<div class="lpage">39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20genes%20encoding%20Arabidopsis%20halleri%20MTP1%20metal%20transport%20proteins%20co-segregate%20with%20zinc%20tolerance%20and%20account%20for%20high%20MTP1%20transcript%20levels&amp;author=D.%20B.%20Drager&amp;author=A.%20G.%20Desbrosses-Fonrouge&amp;author=C.%20Krach&amp;publication_year=2004&amp;journal=Plant%20J&amp;volume=39&amp;pages=425-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2004.39.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2004.39.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2004.39.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15255871" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Duarte</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Wall</div> <div class="given-names">P. K.</div></div>, <div class="name string-name"><div class="surname">Edger</div> <div class="given-names">P. P.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Identification of shared single copy nuclear genes in Arabidopsis, Populus, Vitis and Oryza and their phylogenetic utility across various taxonomic levels</div>, <div class="source">BMC Evol. Biol</div> ., <div class="volume">10</div>, <div class="fpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20shared%20single%20copy%20nuclear%20genes%20in%20Arabidopsis%2C%20Populus%2C%20Vitis%20and%20Oryza%20and%20their%20phylogenetic%20utility%20across%20various%20taxonomic%20levels&amp;author=J.%20M.%20Duarte&amp;author=P.%20K.%20Wall&amp;author=P.%20P.%20Edger&amp;publication_year=2010&amp;journal=BMC%20Evol.%20Biol&amp;volume=10&amp;pages=61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-10-61" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-10-61" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-10-61"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20181251" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wilgenbusch</div> <div class="given-names">J. C.</div></div>, <div class="name string-name"><div class="surname">Swofford</div> <div class="given-names">D.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Inferring evolutionary trees with PAUP*</div>, <div class="source">Curr. Protoc. Bioinformatics</div> , <strong>Chapter 6</strong>, Unit 6 4.</div></div></div></div><div content-id="dsy005-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Saitou</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Nei</div> <div class="given-names">M.</div></div></div> <div class="year">1987</div>, 
<div class="article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">4</div>, <div class="fpage">406</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20neighbor-joining%20method%3A%20a%20new%20method%20for%20reconstructing%20phylogenetic%20trees&amp;author=N.%20Saitou&amp;author=M.%20Nei&amp;publication_year=1987&amp;journal=Mol.%20Biol.%20Evol&amp;volume=4&amp;pages=406-25" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3447015" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Willems</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Drager</div> <div class="given-names">D. B.</div></div>, <div class="name string-name"><div class="surname">Courbot</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Gode</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Verbruggen</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Saumitou-Laprade</div> <div class="given-names">P.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">The genetic basis of zinc tolerance in the metallophyte Arabidopsis halleri ssp. halleri (Brassicaceae): an analysis of quantitative trait loci</div>, <div class="source">Genetics</div> , <div class="volume">176</div>, <div class="fpage">659</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genetic%20basis%20of%20zinc%20tolerance%20in%20the%20metallophyte%20Arabidopsis%20halleri%20ssp.%20halleri%20(Brassicaceae)%3A%20an%20analysis%20of%20quantitative%20trait%20loci&amp;author=G.%20Willems&amp;author=D.%20B.%20Drager&amp;author=M.%20Courbot&amp;author=C.%20Gode&amp;author=N.%20Verbruggen&amp;author=P.%20Saumitou-Laprade&amp;publication_year=2007&amp;journal=Genetics&amp;volume=176&amp;pages=659-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.106.064485" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.106.064485" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.106.064485"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17409091" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Frerot</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Faucon</div> <div class="given-names">M. P.</div></div>, <div class="name string-name"><div class="surname">Willems</div> <div class="given-names">G.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Genetic architecture of zinc hyperaccumulation in Arabidopsis halleri: the essential role of QTL x environment interactions</div>, <div class="source">New Phytol</div> ., <div class="volume">187</div>, <div class="fpage">355</div>–<div class="lpage">67</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20architecture%20of%20zinc%20hyperaccumulation%20in%20Arabidopsis%20halleri%3A%20the%20essential%20role%20of%20QTL%20x%20environment%20interactions&amp;author=H.%20Frerot&amp;author=M.%20P.%20Faucon&amp;author=G.%20Willems&amp;publication_year=2010&amp;journal=New%20Phytol&amp;volume=187&amp;pages=355-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2010.03295.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2010.03295.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2010.03295.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20487314" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vishnoi</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Kryazhimskiy</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Bazykin</div> <div class="given-names">G. A.</div></div>, <div class="name string-name"><div class="surname">Hannenhalli</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Plotkin</div> <div class="given-names">J. B.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Young proteins experience more variable selection pressures than old proteins</div>, <div class="source">Genome Res</div> ., <div class="volume">20</div>, <div class="fpage">1574</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Young%20proteins%20experience%20more%20variable%20selection%20pressures%20than%20old%20proteins&amp;author=A.%20Vishnoi&amp;author=S.%20Kryazhimskiy&amp;author=G.%20A.%20Bazykin&amp;author=S.%20Hannenhalli&amp;author=J.%20B.%20Plotkin&amp;publication_year=2010&amp;journal=Genome%20Res&amp;volume=20&amp;pages=1574-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.109595.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.109595.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.109595.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20921233" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jordan</div> <div class="given-names">I. K.</div></div>, <div class="name string-name"><div class="surname">Wolf</div> <div class="given-names">Y. I.</div></div>, <div class="name string-name"><div class="surname">Koonin</div> <div class="given-names">E. V.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Duplicated genes evolve slower than singletons despite the initial rate increase</div>, <div class="source">BMC Evol. Biol</div> ., <div class="volume">4</div>, <div class="fpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Duplicated%20genes%20evolve%20slower%20than%20singletons%20despite%20the%20initial%20rate%20increase&amp;author=I.%20K.%20Jordan&amp;author=Y.%20I.%20Wolf&amp;author=E.%20V.%20Koonin&amp;publication_year=2004&amp;journal=BMC%20Evol.%20Biol&amp;volume=4&amp;pages=22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-4-22" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-4-22" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-4-22"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15238160" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Alba</div> <div class="given-names">M. M.</div></div>, <div class="name string-name"><div class="surname">Castresana</div> <div class="given-names">J.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Inverse relationship between evolutionary rate and age of mammalian genes</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">22</div>, <div class="fpage">598</div>–<div class="lpage">606</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inverse%20relationship%20between%20evolutionary%20rate%20and%20age%20of%20mammalian%20genes&amp;author=M.%20M.%20Alba&amp;author=J.%20Castresana&amp;publication_year=2005&amp;journal=Mol.%20Biol.%20Evol&amp;volume=22&amp;pages=598-606" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msi045" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsi045" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsi045"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15537804" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wolf</div> <div class="given-names">Y. I.</div></div>, <div class="name string-name"><div class="surname">Novichkov</div> <div class="given-names">P. S.</div></div>, <div class="name string-name"><div class="surname">Karev</div> <div class="given-names">G. P.</div></div>, <div class="name string-name"><div class="surname">Koonin</div> <div class="given-names">E. V.</div></div>, <div class="name string-name"><div class="surname">Lipman</div> <div class="given-names">D. J.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">The universal distribution of evolutionary rates of genes and distinct characteristics of eukaryotic genes of different apparent ages</div>, <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">106</div>, <div class="fpage">7273</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20universal%20distribution%20of%20evolutionary%20rates%20of%20genes%20and%20distinct%20characteristics%20of%20eukaryotic%20genes%20of%20different%20apparent%20ages&amp;author=Y.%20I.%20Wolf&amp;author=P.%20S.%20Novichkov&amp;author=G.%20P.%20Karev&amp;author=E.%20V.%20Koonin&amp;author=D.%20J.%20Lipman&amp;publication_year=2009&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=106&amp;pages=7273-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0901808106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0901808106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0901808106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19351897" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Davis</div> <div class="given-names">J. C.</div></div>, <div class="name string-name"><div class="surname">Petrov</div> <div class="given-names">D. A.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Preferential duplication of conserved proteins in eukaryotic genomes</div>, <div class="source">PLoS Biol</div> ., <div class="volume">2</div>, <div class="fpage">E55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Preferential%20duplication%20of%20conserved%20proteins%20in%20eukaryotic%20genomes&amp;author=J.%20C.%20Davis&amp;author=D.%20A.%20Petrov&amp;publication_year=2004&amp;journal=PLoS%20Biol&amp;volume=2&amp;pages=E55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0020055" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0020055" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0020055"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15024414" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Gu</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">W. H.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Rate of protein evolution versus fitness effect of gene deletion</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">20</div>, <div class="fpage">772</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rate%20of%20protein%20evolution%20versus%20fitness%20effect%20of%20gene%20deletion&amp;author=J.%20Yang&amp;author=Z.%20Gu&amp;author=W.%20H.%20Li&amp;publication_year=2003&amp;journal=Mol.%20Biol.%20Evol&amp;volume=20&amp;pages=772-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msg078" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsg078" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsg078"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12679525" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Satake</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kawata</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">McLysaght</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Makino</div> <div class="given-names">T.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Evolution of vertebrate tissues driven by differential modes of gene duplication</div>, <div class="source">DNA Res</div> ., <div class="volume">19</div>, <div class="fpage">305</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20vertebrate%20tissues%20driven%20by%20differential%20modes%20of%20gene%20duplication&amp;author=M.%20Satake&amp;author=M.%20Kawata&amp;author=A.%20McLysaght&amp;author=T.%20Makino&amp;publication_year=2012&amp;journal=DNA%20Res&amp;volume=19&amp;pages=305-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dss012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdss012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdss012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22490996" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lan</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Pritchard</div> <div class="given-names">J. K.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Coregulation of tandem duplicate genes slows evolution of subfunctionalization in mammals</div>, <div class="source">Science</div> , <div class="volume">352</div>, <div class="fpage">1009</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coregulation%20of%20tandem%20duplicate%20genes%20slows%20evolution%20of%20subfunctionalization%20in%20mammals&amp;author=X.%20Lan&amp;author=J.%20K.%20Pritchard&amp;publication_year=2016&amp;journal=Science&amp;volume=352&amp;pages=1009-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.aad8411" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.aad8411" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.aad8411"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27199432" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rensing</div> <div class="given-names">S. A.</div></div>, <div class="name string-name"><div class="surname">Lang</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Zimmer</div> <div class="given-names">A. D.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">The Physcomitrella genome reveals evolutionary insights into the conquest of land by plants</div>, <div class="source">Science</div> , <div class="volume">319</div>, <div class="fpage">64</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Physcomitrella%20genome%20reveals%20evolutionary%20insights%20into%20the%20conquest%20of%20land%20by%20plants&amp;author=S.%20A.%20Rensing&amp;author=D.%20Lang&amp;author=A.%20D.%20Zimmer&amp;publication_year=2008&amp;journal=Science&amp;volume=319&amp;pages=64-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1150646" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1150646" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1150646"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18079367" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Heckman</div> <div class="given-names">T. M.</div></div>, <div class="name string-name"><div class="surname">Kauffmann</div> <div class="given-names">G.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">The coevolution of galaxies and supermassive black holes: a local perspective</div>, <div class="source">Science</div> , <div class="volume">333</div>, <div class="fpage">182</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20coevolution%20of%20galaxies%20and%20supermassive%20black%20holes%3A%20a%20local%20perspective&amp;author=T.%20M.%20Heckman&amp;author=G.%20Kauffmann&amp;publication_year=2011&amp;journal=Science&amp;volume=333&amp;pages=182-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1200504" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1200504" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1200504"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21737734" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tuskan</div> <div class="given-names">G. A.</div></div>, <div class="name string-name"><div class="surname">Difazio</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Jansson</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">The genome of black cottonwood, Populus trichocarpa (Torr. &amp; Gray)</div>, <div class="source">Science</div> , <div class="volume">313</div>, <div class="fpage">1596</div>–<div class="lpage">604</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20black%20cottonwood%2C%20Populus%20trichocarpa%20(Torr.%20%26%20Gray)&amp;author=G.%20A.%20Tuskan&amp;author=S.%20Difazio&amp;author=S.%20Jansson&amp;publication_year=2006&amp;journal=Science&amp;volume=313&amp;pages=1596-604" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1128691" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1128691" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1128691"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16973872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy005-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy005-B75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wolfe</div> <div class="given-names">K. H.</div></div>, <div class="name string-name"><div class="surname">Gouy</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Y. W.</div></div>, <div class="name string-name"><div class="surname">Sharp</div> <div class="given-names">P. M.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">W. H.</div></div></div> <div class="year">1989</div>, 
<div class="article-title">Date of the monocot-dicot divergence estimated from chloroplast DNA sequence data</div>, <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">86</div>, <div class="fpage">6201</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Date%20of%20the%20monocot-dicot%20divergence%20estimated%20from%20chloroplast%20DNA%20sequence%20data&amp;author=K.%20H.%20Wolfe&amp;author=M.%20Gouy&amp;author=Y.%20W.%20Yang&amp;author=P.%20M.%20Sharp&amp;author=W.%20H.%20Li&amp;publication_year=1989&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=86&amp;pages=6201-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.86.16.6201" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.86.16.6201" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.86.16.6201"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2762323" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2018. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4898128&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy005/1/dsy005_supp.zip?Expires=2147483647&amp;Signature=Ccbka9nIYpgdtD9-1Aag~s~Vr~QmRXnIWLzEDYvgKoPqc~pxFxbJ3JEW83Ms67lAMZmAFFkUyqaIslI4ot4fICq3nsFEgIZTwJhLBymN9~RY1hPf2S6jWEe62gQGPBVsh7ddVPCPIuQUDrJvGeOxL27PSVl8qoh93Ocb6qYOeRisqa~0~4hEe0wflmJGuKCpzA7OL0zpHXTWtD1zDWJDdIWls-60sbMtxPmgYnedDaTEM~IBP5kiwJ2HT7InoI90i7LNXwvy4uBz3OBoeGLSGDJQ8CG2p2p8pkilbySW6IOmR1h29BqzhUvs9LvYXTRk-Qy-xTqv3eTWeNahk~-X3A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,251</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsy005"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4898128"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f3%2f327%2f4898128">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F3%2F327%2F4898128" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4898128'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Functional divergence of duplicate genes several million years after gene duplication in Arabidopsis" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy005");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f3%2f327%2f4898128">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

