

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/5/555/2754552","@type":"ScholarlyArticle","name":"A High-Density Genetic Map with Array-Based Markers Facilitates Structural and Quantitative Trait Locus Analyses of the Common Wheat Genome","datePublished":"2014-06-27","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2014-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu020","keywords":["array-based genotyping","chromosomal synteny","high-density genetic map","next-generation sequencing","QTL analysis"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/5/555/2754552","author":[{"name":"Iehisa, Julio Cesar Masaru","affiliation":"1  Laboratory of Plant Genetics, Graduate School of Agricultural Science ,  Kobe University ,  Nada-ku, Kobe 657-8501 ,  Japan","@type":"Person"},{"name":"Ohno, Ryoko","affiliation":"2  Core Research Division, Organization of Advanced Science and Technology ,  Kobe University ,  Kobe ,  Japan","@type":"Person"},{"name":"Kimura, Tatsuro","affiliation":"3  Bio Research Laboratory ,  TOYOTA Motor Corporation ,  Toyota, Aichi 471-8572 ,  Japan","@type":"Person"},{"name":"Enoki, Hiroyuki","affiliation":"3  Bio Research Laboratory ,  TOYOTA Motor Corporation ,  Toyota, Aichi 471-8572 ,  Japan","@type":"Person"},{"name":"Nishimura, Satoru","affiliation":"3  Bio Research Laboratory ,  TOYOTA Motor Corporation ,  Toyota, Aichi 471-8572 ,  Japan","@type":"Person"},{"name":"Okamoto, Yuki","affiliation":"1  Laboratory of Plant Genetics, Graduate School of Agricultural Science ,  Kobe University ,  Nada-ku, Kobe 657-8501 ,  Japan","@type":"Person"},{"name":"Nasuda, Shuhei","affiliation":"4  Graduate School of Agriculture ,  Kyoto University ,  Kyoto 606-8502 ,  Japan","@type":"Person"},{"name":"Takumi, Shigeo","affiliation":"1  Laboratory of Plant Genetics, Graduate School of Agricultural Science ,  Kobe University ,  Nada-ku, Kobe 657-8501 ,  Japan","@type":"Person"}],"description":"The large genome and allohexaploidy of common wheat have complicated construction of a high-density genetic map. Although improvements in the throughput of next","pageStart":"555","pageEnd":"567","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/5/1/m_cover.gif?Expires=2147483647&Signature=gPiJSb93IIOcIMTSkMAi7kcKhziIjvuRKEzK8XVp8CTVu4eB811a4N7c5iRqAkFTvycBeFOL1qu9DtcvWUQE-HFcU90uhCx7UXftpGVaIU0r-58rykRAQjr7OyCGXhRRXufrlhLC-AFU8QeGWWE7goVK9s00kVi5KaqA8BdZOl4ZUh6UGigqEAXS9fKMJcyv7L69eSvyhk8fr0u2lpE6Z~kKXJ-zph2mYgOTE6TZsnpM7S63PaSYvqdIQyt~fHxa7pkm4binASfSKcJvbv9WOoMwYew88jtLZv495c~AkAPvlFj827Es8r960R62hvuI-oun7Y-AfhyLYD5usT~QKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"A High-Density Genetic Map with Array-Based Markers Facilitates Structural and Quantitative Trait Locus Analyses of the Common Wheat Genome","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/5/1/m_cover.gif?Expires=2147483647&Signature=gPiJSb93IIOcIMTSkMAi7kcKhziIjvuRKEzK8XVp8CTVu4eB811a4N7c5iRqAkFTvycBeFOL1qu9DtcvWUQE-HFcU90uhCx7UXftpGVaIU0r-58rykRAQjr7OyCGXhRRXufrlhLC-AFU8QeGWWE7goVK9s00kVi5KaqA8BdZOl4ZUh6UGigqEAXS9fKMJcyv7L69eSvyhk8fr0u2lpE6Z~kKXJ-zph2mYgOTE6TZsnpM7S63PaSYvqdIQyt~fHxa7pkm4binASfSKcJvbv9WOoMwYew88jtLZv495c~AkAPvlFj827Es8r960R62hvuI-oun7Y-AfhyLYD5usT~QKA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="High-Density Genetic Map with Array-Based Markers Facilitates Structural and Quantitative Trait Locus Analyses of the Common Wheat Genome" />
    <meta property="og:description" content="The large genome and allohexaploidy of common wheat have complicated construction of a high-density genetic map. Although improvements in the throughput of next" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/5/555/2754552" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2014-06-27" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Iehisa, Julio Cesar Masaru" /><meta name="citation_author_institution" content="Laboratory of Plant Genetics, Graduate School of Agricultural Science, Kobe University, Nada-ku, Kobe 657-8501, Japan" /><meta name="citation_author" content="Ohno, Ryoko" /><meta name="citation_author_institution" content="Core Research Division, Organization of Advanced Science and Technology, Kobe University, Kobe, Japan" /><meta name="citation_author" content="Kimura, Tatsuro" /><meta name="citation_author_institution" content="Bio Research Laboratory, TOYOTA Motor Corporation, Toyota, Aichi 471-8572, Japan" /><meta name="citation_author" content="Enoki, Hiroyuki" /><meta name="citation_author_institution" content="Bio Research Laboratory, TOYOTA Motor Corporation, Toyota, Aichi 471-8572, Japan" /><meta name="citation_author" content="Nishimura, Satoru" /><meta name="citation_author_institution" content="Bio Research Laboratory, TOYOTA Motor Corporation, Toyota, Aichi 471-8572, Japan" /><meta name="citation_author" content="Okamoto, Yuki" /><meta name="citation_author_institution" content="Laboratory of Plant Genetics, Graduate School of Agricultural Science, Kobe University, Nada-ku, Kobe 657-8501, Japan" /><meta name="citation_author" content="Nasuda, Shuhei" /><meta name="citation_author_institution" content="Graduate School of Agriculture, Kyoto University, Kyoto 606-8502, Japan" /><meta name="citation_author" content="Takumi, Shigeo" /><meta name="citation_author_institution" content="Laboratory of Plant Genetics, Graduate School of Agricultural Science, Kobe University, Nada-ku, Kobe 657-8501, Japan" /><meta name="citation_title" content="A High-Density Genetic Map with Array-Based Markers Facilitates Structural and Quantitative Trait Locus Analyses of the Common Wheat Genome" /><meta name="citation_firstpage" content="555" /><meta name="citation_lastpage" content="567" /><meta name="citation_doi" content="10.1093/dnares/dsu020" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2014/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Discovery of the DD-analyser, one of the ancestors of Triticum vulgare;  citation_journal_title=Agric. Hortic.;  citation_year=1944;  citation_volume=19;  citation_pages=889-90; " /><meta name="citation_reference" content="citation_title=The artificial synthesis of Triticum spelta;  citation_journal_title=Rec. Genet. Soc. Am.;  citation_year=1944;  citation_volume=13;  citation_pages=26-7; " /><meta name="citation_reference" content="citation_title=Diversity arrays technology (DArT) for high-throughput profiling of the hexaploid wheat genome;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2006;  citation_volume=113;  citation_pages=1409-20; " /><meta name="citation_reference" content="citation_title=Diversity arrays technology (DArT) for whole-genome profiling of barley;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=9915-20; " /><meta name="citation_reference" content="citation_title=Reconstruction of the Synthetic W7984 Ã— Opata M85 wheat reference population;  citation_journal_title=Genome;  citation_year=2011;  citation_volume=54;  citation_pages=875-82; " /><meta name="citation_reference" content="citation_title=Investigation of genetic diversity and population structure of common wheat cultivars in northern China using DArT markers;  citation_journal_title=BMC Genetics;  citation_year=2011;  citation_volume=12;  citation_pages=42" /><meta name="citation_reference" content="citation_title=High-throughput SNP discovery and genotyping in durum wheat (Triticum durum Desf.);  citation_journal_title=Theor. Appl. Genet.;  citation_year=2011;  citation_volume=123;  citation_pages=555-69; " /><meta name="citation_reference" content="citation_title=Transcript-specific, single-nucleotide polymorphism discovery and linkage analysis in hexaploid bread wheat (Triticum aestivum L.);  citation_journal_title=Plant Biotechnol. J.;  citation_year=2011;  citation_volume=9;  citation_pages=1086-99; " /><meta name="citation_reference" content="citation_title=Single nucleotide polymorphism discovery from wheat next-generation sequence data;  citation_journal_title=Plant Biotechnol. J.;  citation_year=2012;  citation_volume=10;  citation_pages=743-9; " /><meta name="citation_reference" content="citation_title=Genome-wide comparative diversity uncovers multiple targets of selection for improvement in hexaploid wheat landraces and cultivars;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2013;  citation_volume=110;  citation_pages=8057-62; " /><meta name="citation_reference" content="citation_title=Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach;  citation_journal_title=PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e32253" /><meta name="citation_reference" content="citation_title=Sequence-based mapping of the polyploid wheat genome;  citation_journal_title=G3-Genes Genomes Genet.;  citation_year=2013;  citation_volume=3;  citation_pages=1105-14; " /><meta name="citation_reference" content="citation_title=Sequence-based SNP genotyping in durum wheat;  citation_journal_title=Plant Biotechnol. J.;  citation_year=2013;  citation_volume=11;  citation_pages=809-17; " /><meta name="citation_reference" content="citation_title=Targeted re-sequencing of the allohexaploid wheat exome;  citation_journal_title=Plant Biotechnol. J.;  citation_year=2012;  citation_volume=10;  citation_pages=733-42; " /><meta name="citation_reference" content="citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2011;  citation_volume=12;  citation_pages=499-510; " /><meta name="citation_reference" content="citation_title=Identification of quantitative trait loci for ABA responsiveness at the seedling stage associated with ABA-regulated gene expression in common wheat;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2010;  citation_volume=121;  citation_pages=629-41; " /><meta name="citation_reference" content="  Enoki H.  Nishimura S.  Murakami A.   Method for designing probe in DNA microarray, and DNA microarray provided with probe designed thereby  2011  International publication number: WO 2011/074510  " /><meta name="citation_reference" content="  Enoki H.  Nishimura S.  Murakami A.   Method for designing probe in DNA microarray, and DNA microarray provided with probe designed thereby  2012  US patent application 20120190582A1  " /><meta name="citation_reference" content="citation_title=Velvet: algorithms for de novo short read assembly using de Bruijn graphs;  citation_journal_title=Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=821-9; " /><meta name="citation_reference" content="citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome;  citation_journal_title=Genome Biol.;  citation_year=2009;  citation_volume=10;  citation_pages=R25" /><meta name="citation_reference" content="citation_title=A pseudo-response regulator is misexpressed in the photoperiod insensitive Ppd-D1a mutant of wheat (Triticum aestivum L.);  citation_journal_title=Theor. Appl. Genet.;  citation_year=2007;  citation_volume=115;  citation_pages=721-33; " /><meta name="citation_reference" content="citation_title=AntMap: constructing genetic linkage maps using an ant colony optimization algorithm;  citation_journal_title=Breed. Sci.;  citation_year=2006;  citation_volume=56;  citation_pages=371-7; " /><meta name="citation_reference" content="citation_title=MapDisto: fast and efficient computation of genetic linkage maps;  citation_journal_title=Mol. Breed.;  citation_year=2012;  citation_volume=30;  citation_pages=1231-5; " /><meta name="citation_reference" content="citation_title=The estimation of map distance from recombination values;  citation_journal_title=Ann. Eugen.;  citation_year=1944;  citation_volume=12;  citation_pages=172-5; " /><meta name="citation_reference" content="citation_title=Aegilops tauschii draft genome sequence reveals a gene repertoire for wheat adaptation;  citation_journal_title=Nature;  citation_year=2013;  citation_volume=496;  citation_pages=91-5; " /><meta name="citation_reference" content="citation_title=Draft genome of the wheat A-genome progenitor Triticum urartu;  citation_journal_title=Nature;  citation_year=2013;  citation_volume=496;  citation_pages=87-90; " /><meta name="citation_reference" content="citation_title=A physical, genetic and functional sequence assembly of the barley genome; citation_author=International Barley Genome Sequencing Consortium;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=491;  citation_pages=711-6; " /><meta name="citation_reference" content="citation_title=A 4-gigabase physical map unlocks the structure and evolution of the complex genome of Aegilops tauschii, the wheat D-genome progenitor;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2013;  citation_volume=110;  citation_pages=7940-5; " /><meta name="citation_reference" content="citation_title=Circos: an information aesthetic for comparative genomics;  citation_journal_title=Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=1639-45; " /><meta name="citation_reference" content="citation_title=R/qtl: QTL mapping in experimental crosses;  citation_journal_title=Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=889-90; " /><meta name="citation_reference" content="citation_title=Structural evolution of wheat chromosomes 4A, 5A, and 7B and its impact on recombination;  citation_journal_title=Theor. Appl. Genet.;  citation_year=1995;  citation_volume=91;  citation_pages=282-8; " /><meta name="citation_reference" content="citation_title=Sequence-based analysis of translocations and inversions in bread wheat (Triticum aestivum L.);  citation_journal_title=PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e79329" /><meta name="citation_reference" content="citation_title=Analysis of the bread wheat genome using whole-genome shotgun sequencing;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=491;  citation_pages=705-10; " /><meta name="citation_reference" content="citation_title=A pollen-killing gene in wheat;  citation_journal_title=Genetics;  citation_year=1961;  citation_volume=46;  citation_pages=897" /><meta name="citation_reference" content="citation_title=Segregation distortion caused by weak hybrid necrosis in recombinant inbred lines of common wheat;  citation_journal_title=Genetica;  citation_year=2013;  citation_volume=141;  citation_pages=463-70; " /><meta name="citation_reference" content="citation_title=Molecular mapping of hybrid necrosis genes Ne1 and Ne2 in hexaploid wheat using microsatellite markers;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2006;  citation_volume=112;  citation_pages=1374-81; " /><meta name="citation_reference" content="citation_title=Divergent evolution of duplicate genes leads to genetic incompatibilities within A. thaliana;  citation_journal_title=Science;  citation_year=2009;  citation_volume=323;  citation_pages=623-6; " /><meta name="citation_reference" content="citation_title=Rice pollen hybrid incompatibility caused by reciprocal gene loss of duplicated genes;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=20417-22; " /><meta name="citation_reference" content="citation_title=A killer-protector system regulates both hybrid sterility and segregation distortion;  citation_journal_title=Science;  citation_year=2012;  citation_volume=337;  citation_pages=1336-40; " /><meta name="citation_reference" content="citation_title=Gains in QTL detection using an ultra-high density SNP map based on population sequencing relative to traditional RFLP/SSR markers;  citation_journal_title=PLoS ONE;  citation_year=2011;  citation_volume=6;  citation_pages=e17595" /><meta name="citation_reference" content="citation_title=Copy number variation affecting the Photoperiod-B1 and Vernalization-A1 genes is associated with altered flowering time in wheat (Triticum aestivum);  citation_journal_title=PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e33234" /><meta name="citation_reference" content="citation_title=Differential contribution of two Ppd-1 homoeoalleles to early-flowering phenotype in Nepalese and Japanese varieties of common wheat;  citation_journal_title=Breed. Sci.;  citation_year=2013;  citation_volume=63;  citation_pages=374-83; " /><meta name="citation_reference" content="citation_title=Allelic variation at the VRN-1 promoter region in polyploid wheat;  citation_journal_title=Theor. Appl. Genet.;  citation_year=2004;  citation_volume=109;  citation_pages=1677-86; " /><meta name="citation_reference" content="citation_title=Predicting milling extraction rate by image analysis of wheat grains;  citation_journal_title=Asp. Appl. Biol.;  citation_year=1990;  citation_volume=25;  citation_pages=417-26; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/5/555/17463091/dsu020.pdf" /><meta name="description" content="The large genome and allohexaploidy of common wheat have complicated construction of a high-density genetic map. Although improvements in the throughput of next" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/5/555/2754552" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/5/555/2754552" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/5/555/2754552");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsu020';
    </script>
<title>
	High-Density Genetic Map with Array-Based Markers Facilitates Structural and Quantitative Trait Locus Analyses of the Common Wheat Genome | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2754552" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZhtPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJZWhpc2EsIEp1bGlvIENlc2FyIE1hc2FydSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgUGxhbnQgR2VuZXRpY3MsIEdyYWR1YXRlIFNjaG9vbCBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZSwgS29iZSBVbml2ZXJzaXR5LCBOYWRhLWt1LCBLb2JlIDY1Ny04NTAxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9obm8sIFJ5b2tvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ29yZSBSZXNlYXJjaCBEaXZpc2lvbiwgT3JnYW5pemF0aW9uIG9mIEFkdmFuY2VkIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIEtvYmUgVW5pdmVyc2l0eSwgS29iZSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLaW11cmEsIFRhdHN1cm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW8gUmVzZWFyY2ggTGFib3JhdG9yeSwgVE9ZT1RBIE1vdG9yIENvcnBvcmF0aW9uLCBUb3lvdGEsIEFpY2hpIDQ3MS04NTcyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkVub2tpLCBIaXJveXVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpbyBSZXNlYXJjaCBMYWJvcmF0b3J5LCBUT1lPVEEgTW90b3IgQ29ycG9yYXRpb24sIFRveW90YSwgQWljaGkgNDcxLTg1NzIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmlzaGltdXJhLCBTYXRvcnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW8gUmVzZWFyY2ggTGFib3JhdG9yeSwgVE9ZT1RBIE1vdG9yIENvcnBvcmF0aW9uLCBUb3lvdGEsIEFpY2hpIDQ3MS04NTcyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9rYW1vdG8sIFl1a2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIFBsYW50IEdlbmV0aWNzLCBHcmFkdWF0ZSBTY2hvb2wgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2UsIEtvYmUgVW5pdmVyc2l0eSwgTmFkYS1rdSwgS29iZSA2NTctODUwMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYXN1ZGEsIFNodWhlaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBBZ3JpY3VsdHVyZSwgS3lvdG8gVW5pdmVyc2l0eSwgS3lvdG8gNjA2LTg1MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFrdW1pLCBTaGlnZW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIFBsYW50IEdlbmV0aWNzLCBHcmFkdWF0ZSBTY2hvb2wgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2UsIEtvYmUgVW5pdmVyc2l0eSwgTmFkYS1rdSwgS29iZSA2NTctODUwMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkEgSGlnaC1EZW5zaXR5IEdlbmV0aWMgTWFwIHdpdGggQXJyYXktQmFzZWQgTWFya2VycyBGYWNpbGl0YXRlcyBTdHJ1Y3R1cmFsIGFuZCBRdWFudGl0YXRpdmUgVHJhaXQgTG9jdXMgQW5hbHlzZXMgb2YgdGhlIENvbW1vbiBXaGVhdCBHZW5vbWUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI1NTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjU2NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdTAyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE0LzEwLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzY292ZXJ5IG9mIHRoZSBERC1hbmFseXNlciwgb25lIG9mIHRoZSBhbmNlc3RvcnMgb2YgVHJpdGljdW0gdnVsZ2FyZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWdyaWMuIEhvcnRpYy47ICBjaXRhdGlvbl95ZWFyPTE5NDQ7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz04ODktOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBhcnRpZmljaWFsIHN5bnRoZXNpcyBvZiBUcml0aWN1bSBzcGVsdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVJlYy4gR2VuZXQuIFNvYy4gQW0uOyAgY2l0YXRpb25feWVhcj0xOTQ0OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MjYtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGl2ZXJzaXR5IGFycmF5cyB0ZWNobm9sb2d5IChEQXJUKSBmb3IgaGlnaC10aHJvdWdocHV0IHByb2ZpbGluZyBvZiB0aGUgaGV4YXBsb2lkIHdoZWF0IGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTM7ICBjaXRhdGlvbl9wYWdlcz0xNDA5LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXZlcnNpdHkgYXJyYXlzIHRlY2hub2xvZ3kgKERBclQpIGZvciB3aG9sZS1nZW5vbWUgcHJvZmlsaW5nIG9mIGJhcmxleTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTAxOyAgY2l0YXRpb25fcGFnZXM9OTkxNS0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjb25zdHJ1Y3Rpb24gb2YgdGhlIFN5bnRoZXRpYyBXNzk4NCDDlyBPcGF0YSBNODUgd2hlYXQgcmVmZXJlbmNlIHBvcHVsYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT01NDsgIGNpdGF0aW9uX3BhZ2VzPTg3NS04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW52ZXN0aWdhdGlvbiBvZiBnZW5ldGljIGRpdmVyc2l0eSBhbmQgcG9wdWxhdGlvbiBzdHJ1Y3R1cmUgb2YgY29tbW9uIHdoZWF0IGN1bHRpdmFycyBpbiBub3J0aGVybiBDaGluYSB1c2luZyBEQXJUIG1hcmtlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTQyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC10aHJvdWdocHV0IFNOUCBkaXNjb3ZlcnkgYW5kIGdlbm90eXBpbmcgaW4gZHVydW0gd2hlYXQgKFRyaXRpY3VtIGR1cnVtIERlc2YuKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjM7ICBjaXRhdGlvbl9wYWdlcz01NTUtNjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHQtc3BlY2lmaWMsIHNpbmdsZS1udWNsZW90aWRlIHBvbHltb3JwaGlzbSBkaXNjb3ZlcnkgYW5kIGxpbmthZ2UgYW5hbHlzaXMgaW4gaGV4YXBsb2lkIGJyZWFkIHdoZWF0IChUcml0aWN1bSBhZXN0aXZ1bSBMLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEJpb3RlY2hub2wuIEouOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0xMDg2LTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc20gZGlzY292ZXJ5IGZyb20gd2hlYXQgbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNlIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEJpb3RlY2hub2wuIEouOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NzQzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGNvbXBhcmF0aXZlIGRpdmVyc2l0eSB1bmNvdmVycyBtdWx0aXBsZSB0YXJnZXRzIG9mIHNlbGVjdGlvbiBmb3IgaW1wcm92ZW1lbnQgaW4gaGV4YXBsb2lkIHdoZWF0IGxhbmRyYWNlcyBhbmQgY3VsdGl2YXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTA7ICBjaXRhdGlvbl9wYWdlcz04MDU3LTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBoaWdoLWRlbnNpdHkgZ2VuZXRpYyBtYXBzIGZvciBiYXJsZXkgYW5kIHdoZWF0IHVzaW5nIGEgbm92ZWwgdHdvLWVuenltZSBnZW5vdHlwaW5nLWJ5LXNlcXVlbmNpbmcgYXBwcm9hY2g7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lMzIyNTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZS1iYXNlZCBtYXBwaW5nIG9mIHRoZSBwb2x5cGxvaWQgd2hlYXQgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HMy1HZW5lcyBHZW5vbWVzIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MTEwNS0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2UtYmFzZWQgU05QIGdlbm90eXBpbmcgaW4gZHVydW0gd2hlYXQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEJpb3RlY2hub2wuIEouOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9ODA5LTE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYXJnZXRlZCByZS1zZXF1ZW5jaW5nIG9mIHRoZSBhbGxvaGV4YXBsb2lkIHdoZWF0IGV4b21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBCaW90ZWNobm9sLiBKLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTczMy00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgZ2VuZXRpYyBtYXJrZXIgZGlzY292ZXJ5IGFuZCBnZW5vdHlwaW5nIHVzaW5nIG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NDk5LTUxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgZm9yIEFCQSByZXNwb25zaXZlbmVzcyBhdCB0aGUgc2VlZGxpbmcgc3RhZ2UgYXNzb2NpYXRlZCB3aXRoIEFCQS1yZWd1bGF0ZWQgZ2VuZSBleHByZXNzaW9uIGluIGNvbW1vbiB3aGVhdDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjE7ICBjaXRhdGlvbl9wYWdlcz02MjktNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IiAgRW5va2kgSC4gIE5pc2hpbXVyYSBTLiAgTXVyYWthbWkgQS4gICBNZXRob2QgZm9yIGRlc2lnbmluZyBwcm9iZSBpbiBETkEgbWljcm9hcnJheSwgYW5kIEROQSBtaWNyb2FycmF5IHByb3ZpZGVkIHdpdGggcHJvYmUgZGVzaWduZWQgdGhlcmVieSAgMjAxMSAgSW50ZXJuYXRpb25hbCBwdWJsaWNhdGlvbiBudW1iZXI6IFdPIDIwMTEvMDc0NTEwICAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSIgIEVub2tpIEguICBOaXNoaW11cmEgUy4gIE11cmFrYW1pIEEuICAgTWV0aG9kIGZvciBkZXNpZ25pbmcgcHJvYmUgaW4gRE5BIG1pY3JvYXJyYXksIGFuZCBETkEgbWljcm9hcnJheSBwcm92aWRlZCB3aXRoIHByb2JlIGRlc2lnbmVkIHRoZXJlYnkgIDIwMTIgIFVTIHBhdGVudCBhcHBsaWNhdGlvbiAyMDEyMDE5MDU4MkExICAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WZWx2ZXQ6IGFsZ29yaXRobXMgZm9yIGRlIG5vdm8gc2hvcnQgcmVhZCBhc3NlbWJseSB1c2luZyBkZSBCcnVpam4gZ3JhcGhzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTgyMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbHRyYWZhc3QgYW5kIG1lbW9yeS1lZmZpY2llbnQgYWxpZ25tZW50IG9mIHNob3J0IEROQSBzZXF1ZW5jZXMgdG8gdGhlIGh1bWFuIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9UjI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwc2V1ZG8tcmVzcG9uc2UgcmVndWxhdG9yIGlzIG1pc2V4cHJlc3NlZCBpbiB0aGUgcGhvdG9wZXJpb2QgaW5zZW5zaXRpdmUgUHBkLUQxYSBtdXRhbnQgb2Ygd2hlYXQgKFRyaXRpY3VtIGFlc3RpdnVtIEwuKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTU7ICBjaXRhdGlvbl9wYWdlcz03MjEtMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFudE1hcDogY29uc3RydWN0aW5nIGdlbmV0aWMgbGlua2FnZSBtYXBzIHVzaW5nIGFuIGFudCBjb2xvbnkgb3B0aW1pemF0aW9uIGFsZ29yaXRobTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QnJlZWQuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NTY7ICBjaXRhdGlvbl9wYWdlcz0zNzEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwRGlzdG86IGZhc3QgYW5kIGVmZmljaWVudCBjb21wdXRhdGlvbiBvZiBnZW5ldGljIGxpbmthZ2UgbWFwczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCcmVlZC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xMjMxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBlc3RpbWF0aW9uIG9mIG1hcCBkaXN0YW5jZSBmcm9tIHJlY29tYmluYXRpb24gdmFsdWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm4uIEV1Z2VuLjsgIGNpdGF0aW9uX3llYXI9MTk0NDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTE3Mi01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZWdpbG9wcyB0YXVzY2hpaSBkcmFmdCBnZW5vbWUgc2VxdWVuY2UgcmV2ZWFscyBhIGdlbmUgcmVwZXJ0b2lyZSBmb3Igd2hlYXQgYWRhcHRhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ5NjsgIGNpdGF0aW9uX3BhZ2VzPTkxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IGdlbm9tZSBvZiB0aGUgd2hlYXQgQS1nZW5vbWUgcHJvZ2VuaXRvciBUcml0aWN1bSB1cmFydHU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00OTY7ICBjaXRhdGlvbl9wYWdlcz04Ny05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwaHlzaWNhbCwgZ2VuZXRpYyBhbmQgZnVuY3Rpb25hbCBzZXF1ZW5jZSBhc3NlbWJseSBvZiB0aGUgYmFybGV5IGdlbm9tZTsgY2l0YXRpb25fYXV0aG9yPUludGVybmF0aW9uYWwgQmFybGV5IEdlbm9tZSBTZXF1ZW5jaW5nIENvbnNvcnRpdW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00OTE7ICBjaXRhdGlvbl9wYWdlcz03MTEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSA0LWdpZ2FiYXNlIHBoeXNpY2FsIG1hcCB1bmxvY2tzIHRoZSBzdHJ1Y3R1cmUgYW5kIGV2b2x1dGlvbiBvZiB0aGUgY29tcGxleCBnZW5vbWUgb2YgQWVnaWxvcHMgdGF1c2NoaWksIHRoZSB3aGVhdCBELWdlbm9tZSBwcm9nZW5pdG9yOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTA7ICBjaXRhdGlvbl9wYWdlcz03OTQwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNpcmNvczogYW4gaW5mb3JtYXRpb24gYWVzdGhldGljIGZvciBjb21wYXJhdGl2ZSBnZW5vbWljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xNjM5LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SL3F0bDogUVRMIG1hcHBpbmcgaW4gZXhwZXJpbWVudGFsIGNyb3NzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9ODg5LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmFsIGV2b2x1dGlvbiBvZiB3aGVhdCBjaHJvbW9zb21lcyA0QSwgNUEsIGFuZCA3QiBhbmQgaXRzIGltcGFjdCBvbiByZWNvbWJpbmF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTkxOyAgY2l0YXRpb25fcGFnZXM9MjgyLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNlLWJhc2VkIGFuYWx5c2lzIG9mIHRyYW5zbG9jYXRpb25zIGFuZCBpbnZlcnNpb25zIGluIGJyZWFkIHdoZWF0IChUcml0aWN1bSBhZXN0aXZ1bSBMLik7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNzkzMjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiB0aGUgYnJlYWQgd2hlYXQgZ2Vub21lIHVzaW5nIHdob2xlLWdlbm9tZSBzaG90Z3VuIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00OTE7ICBjaXRhdGlvbl9wYWdlcz03MDUtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcG9sbGVuLWtpbGxpbmcgZ2VuZSBpbiB3aGVhdDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5NjE7ICBjaXRhdGlvbl92b2x1bWU9NDY7ICBjaXRhdGlvbl9wYWdlcz04OTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWdyZWdhdGlvbiBkaXN0b3J0aW9uIGNhdXNlZCBieSB3ZWFrIGh5YnJpZCBuZWNyb3NpcyBpbiByZWNvbWJpbmFudCBpbmJyZWQgbGluZXMgb2YgY29tbW9uIHdoZWF0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljYTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDE7ICBjaXRhdGlvbl9wYWdlcz00NjMtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBtYXBwaW5nIG9mIGh5YnJpZCBuZWNyb3NpcyBnZW5lcyBOZTEgYW5kIE5lMiBpbiBoZXhhcGxvaWQgd2hlYXQgdXNpbmcgbWljcm9zYXRlbGxpdGUgbWFya2VyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTI7ICBjaXRhdGlvbl9wYWdlcz0xMzc0LTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXZlcmdlbnQgZXZvbHV0aW9uIG9mIGR1cGxpY2F0ZSBnZW5lcyBsZWFkcyB0byBnZW5ldGljIGluY29tcGF0aWJpbGl0aWVzIHdpdGhpbiBBLiB0aGFsaWFuYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zMjM7ICBjaXRhdGlvbl9wYWdlcz02MjMtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmljZSBwb2xsZW4gaHlicmlkIGluY29tcGF0aWJpbGl0eSBjYXVzZWQgYnkgcmVjaXByb2NhbCBnZW5lIGxvc3Mgb2YgZHVwbGljYXRlZCBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9MjA0MTctMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEga2lsbGVyLXByb3RlY3RvciBzeXN0ZW0gcmVndWxhdGVzIGJvdGggaHlicmlkIHN0ZXJpbGl0eSBhbmQgc2VncmVnYXRpb24gZGlzdG9ydGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMzc7ICBjaXRhdGlvbl9wYWdlcz0xMzM2LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HYWlucyBpbiBRVEwgZGV0ZWN0aW9uIHVzaW5nIGFuIHVsdHJhLWhpZ2ggZGVuc2l0eSBTTlAgbWFwIGJhc2VkIG9uIHBvcHVsYXRpb24gc2VxdWVuY2luZyByZWxhdGl2ZSB0byB0cmFkaXRpb25hbCBSRkxQL1NTUiBtYXJrZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTE3NTk1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29weSBudW1iZXIgdmFyaWF0aW9uIGFmZmVjdGluZyB0aGUgUGhvdG9wZXJpb2QtQjEgYW5kIFZlcm5hbGl6YXRpb24tQTEgZ2VuZXMgaXMgYXNzb2NpYXRlZCB3aXRoIGFsdGVyZWQgZmxvd2VyaW5nIHRpbWUgaW4gd2hlYXQgKFRyaXRpY3VtIGFlc3RpdnVtKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWUzMzIzNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZmZlcmVudGlhbCBjb250cmlidXRpb24gb2YgdHdvIFBwZC0xIGhvbW9lb2FsbGVsZXMgdG8gZWFybHktZmxvd2VyaW5nIHBoZW5vdHlwZSBpbiBOZXBhbGVzZSBhbmQgSmFwYW5lc2UgdmFyaWV0aWVzIG9mIGNvbW1vbiB3aGVhdDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QnJlZWQuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NjM7ICBjaXRhdGlvbl9wYWdlcz0zNzQtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsbGVsaWMgdmFyaWF0aW9uIGF0IHRoZSBWUk4tMSBwcm9tb3RlciByZWdpb24gaW4gcG9seXBsb2lkIHdoZWF0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwOTsgIGNpdGF0aW9uX3BhZ2VzPTE2NzctODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWRpY3RpbmcgbWlsbGluZyBleHRyYWN0aW9uIHJhdGUgYnkgaW1hZ2UgYW5hbHlzaXMgb2Ygd2hlYXQgZ3JhaW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bc3AuIEFwcGwuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9NDE3LTI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjEvNS81NTUvMTc0NjMwOTEvZHN1MDIwLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iVGhlIGxhcmdlIGdlbm9tZSBhbmQgYWxsb2hleGFwbG9pZHkgb2YgY29tbW9uIHdoZWF0IGhhdmUgY29tcGxpY2F0ZWQgY29uc3RydWN0aW9uIG9mIGEgaGlnaC1kZW5zaXR5IGdlbmV0aWMgbWFwLiBBbHRob3VnaCBpbXByb3ZlbWVudHMgaW4gdGhlIHRocm91Z2hwdXQgb2YgbmV4dCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9Mjc1NDU1MiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGSWsCz588DLR7fxNwF08VikcH9cFQ==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f5%2f555%2f2754552" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f5%2f555%2f2754552" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=15969" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/5/1/m_cover.gif?Expires=2147483647&amp;Signature=gPiJSb93IIOcIMTSkMAi7kcKhziIjvuRKEzK8XVp8CTVu4eB811a4N7c5iRqAkFTvycBeFOL1qu9DtcvWUQE-HFcU90uhCx7UXftpGVaIU0r-58rykRAQjr7OyCGXhRRXufrlhLC-AFU8QeGWWE7goVK9s00kVi5KaqA8BdZOl4ZUh6UGigqEAXS9fKMJcyv7L69eSvyhk8fr0u2lpE6Z~kKXJ-zph2mYgOTE6TZsnpM7S63PaSYvqdIQyt~fHxa7pkm4binASfSKcJvbv9WOoMwYew88jtLZv495c~AkAPvlFj827Es8r960R62hvuI-oun7Y-AfhyLYD5usT~QKA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/5">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87199028">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199028">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199033">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199033">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199058">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199058">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199082">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199082">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199091">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199091">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87199093">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87199093">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/5/541/2754549">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">A High-Density Genetic Map with Array-Based Markers Facilitates Structural and Quantitative Trait Locus Analyses of the Common Wheat Genome
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Julio Cesar Masaru Iehisa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratory of Plant Genetics, Graduate School of Agricultural Science</div>, <div class="insititution">Kobe University</div>, <div class="addr-line">Nada-ku, Kobe 657-8501</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Julio+Cesar+Masaru+Iehisa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Iehisa J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Iehisa J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Julio Cesar Masaru Iehisa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ryoko Ohno</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Core Research Division, Organization of Advanced Science and Technology</div>, <div class="insititution">Kobe University</div>, <div class="addr-line">Kobe</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ryoko+Ohno">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohno R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohno R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ryoko Ohno</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tatsuro Kimura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Bio Research Laboratory</div>, <div class="insititution">TOYOTA Motor Corporation</div>, <div class="addr-line">Toyota, Aichi 471-8572</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tatsuro+Kimura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kimura T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kimura T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tatsuro Kimura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroyuki Enoki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Bio Research Laboratory</div>, <div class="insititution">TOYOTA Motor Corporation</div>, <div class="addr-line">Toyota, Aichi 471-8572</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroyuki+Enoki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Enoki H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Enoki H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiroyuki Enoki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoru Nishimura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Bio Research Laboratory</div>, <div class="insititution">TOYOTA Motor Corporation</div>, <div class="addr-line">Toyota, Aichi 471-8572</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoru+Nishimura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nishimura S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nishimura S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Satoru Nishimura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuki Okamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratory of Plant Genetics, Graduate School of Agricultural Science</div>, <div class="insititution">Kobe University</div>, <div class="addr-line">Nada-ku, Kobe 657-8501</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuki+Okamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Okamoto Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Okamoto Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuki Okamoto</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shuhei Nasuda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Graduate School of Agriculture</div>, <div class="insititution">Kyoto University</div>, <div class="addr-line">Kyoto 606-8502</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shuhei+Nasuda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nasuda S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nasuda S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shuhei Nasuda</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigeo Takumi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Laboratory of Plant Genetics, Graduate School of Agricultural Science</div>, <div class="insititution">Kobe University</div>, <div class="addr-line">Nada-ku, Kobe 657-8501</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 78-803-5860. Fax. +81 78-803-5860. Email: <a href="mailto:takumi@kobe-u.ac.jp" target="_blank">takumi@kobe-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shigeo+Takumi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takumi S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takumi S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shigeo Takumi</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label"><sup>â€ </sup></div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="an2"><div class="fn"><div class="footnote-content">Edited by Prof. Kazuhiro Sato</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 5, 1 October 2014, Pages 555â€“567, <a href='https://doi.org/10.1093/dnares/dsu020'>https://doi.org/10.1093/dnares/dsu020</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">27 June 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">05 March 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">26 May 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2754552" href="/dnaresearch/article-pdf/21/5/555/17463091/dsu020.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Julio Cesar Masaru Iehisa, Ryoko Ohno, Tatsuro Kimura, Hiroyuki Enoki, Satoru Nishimura, Yuki Okamoto, Shuhei Nasuda, Shigeo Takumi;  A High-Density Genetic Map with Array-Based Markers Facilitates Structural and Quantitative Trait Locus Analyses of the Common Wheat Genome, <em>DNA Research</em>, Volume 21, Issue 5, 1 October 2014, Pages 555â€“567, <a href='https://doi.org/10.1093/dnares/dsu020'>https://doi.org/10.1093/dnares/dsu020</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2754552?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754552?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754552?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754552?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754552?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754552?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754552?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=A%20High-Density%20Genetic%20Map%20with%20Array-Based%20Markers%20Facilitates%20Structural%20and%20Quantitative%20Trait%20Locus%20Analyses%20of%20the%20Common%20Wheat%20Genome&amp;publicationDate=2014-06-27&amp;volumeNum=21&amp;issueNum=5&amp;author=Iehisa%2C%20Julio%20Cesar%20Masaru%3B%20Ohno%2C%20Ryoko&amp;startPage=555&amp;endPage=567&amp;contentId=10.1093%2Fdnares%2Fdsu020&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f5%2f555%2f2754552" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





<section class="abstract"><p>The large genome and allohexaploidy of common wheat have complicated construction of a high-density genetic map. Although improvements in the throughput of next-generation sequencing (NGS) technologies have made it possible to obtain a large amount of genotyping data for an entire mapping population by direct sequencing, including hexaploid wheat, a significant number of missing data points are often apparent due to the low coverage of sequencing. In the present study, a microarray-based polymorphism detection system was developed using NGS data obtained from complexity-reduced genomic DNA of two common wheat cultivars, Chinese Spring (CS) and Mironovskaya 808. After design and selection of polymorphic probes, 13,056 new markers were added to the linkage map of a recombinant inbred mapping population between CS and Mironovskaya 808. On average, 2.49 missing data points per marker were observed in the 201 recombinant inbred lines, with a maximum of 42. Around 40% of the new markers were derived from genic regions and 11% from repetitive regions. The low number of retroelements indicated that the new polymorphic markers were mainly derived from the less repetitive region of the wheat genome. Around 25% of the mapped sequences were useful for alignment with the physical map of barley. Quantitative trait locus (QTL) analyses of 14 agronomically important traits related to flowering, spikes, and seeds demonstrated that the new high-density map showed improved QTL detection, resolution, and accuracy over the original simple sequence repeat map.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;array-based genotyping&quot;">array-based genotyping</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;chromosomal synteny&quot;">chromosomal synteny</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;high-density genetic map&quot;">high-density genetic map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;next-generation sequencing&quot;">next-generation sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;QTL analysis&quot;">QTL analysis</a></div>                            <h2 scrollto-destination=87199028 id="87199028" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Common wheat (<em>Triticum aestivum</em> L., genome constitution AABBDD) is an allohexaploid species that originated from interspecific hybridization between tetraploid wheat (<em>Triticum turgidum</em> L., AABB) and the wild diploid relative, <em>Aegilops tauschii</em> Coss. (DD).<sup><a reveal-id="DSU020C1" data-open="DSU020C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSU020C2" data-open="DSU020C2" class="link link-ref link-reveal xref-bibr">2</a></sup> In common wheat, its large genome (17 Gb) and allopolyploidy have complicated the construction of high-resolution genetic maps. Diversity arrays technology (DArT), a microarray hybridization-based technique, has been developed in common wheat for high-throughput genotyping without relying on sequence information.<sup><a reveal-id="DSU020C3" data-open="DSU020C3" class="link link-ref link-reveal xref-bibr">3</a></sup> DArT generates whole-genome fingerprints by scoring the presence versus absence of DNA fragments in genomic representations generated by the digestion of genomic DNA with a combination of two restriction endonucleases.<sup><a reveal-id="DSU020C4" data-open="DSU020C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Using the wheat PstI(TaqI) v2.6 array, 1,348 DArT markers were mapped in a hexaploid Synthetic W7984 Ã— Opata M85 doubled haploid population.<sup><a reveal-id="DSU020C5" data-open="DSU020C5" class="link link-ref link-reveal xref-bibr">5</a></sup> The most recent common wheat PstI(TaqI) v3.0 array comprises about 7,000 markers that are polymorphic in a wide range of wheat cultivars.<sup><a reveal-id="DSU020C6" data-open="DSU020C6" class="link link-ref link-reveal xref-bibr">6</a></sup></p><p>Recent advances in next-generation sequencing (NGS) technology have made it possible to screen a huge number of polymorphic sites even in species without reference genome information. Single-nucleotide polymorphisms (SNPs) were discovered by sequencing of a complexity-reduced fraction of the tetraploid wheat genome<sup><a reveal-id="DSU020C7" data-open="DSU020C7" class="link link-ref link-reveal xref-bibr">7</a></sup> or cDNA of common wheat,<sup><a reveal-id="DSU020C8" data-open="DSU020C8" class="link link-ref link-reveal xref-bibr">8</a>â€“<a reveal-id="DSU020C10" data-open="DSU020C10" class="link link-ref link-reveal xref-bibr">10</a></sup> and have been used to develop high-throughput SNP-typing platforms such as BeadExpress,<sup><a reveal-id="DSU020C7" data-open="DSU020C7" class="link link-ref link-reveal xref-bibr">7</a></sup> KASPar,<sup><a reveal-id="DSU020C8" data-open="DSU020C8" class="link link-ref link-reveal xref-bibr">8</a></sup> and Infinium.<sup><a reveal-id="DSU020C10" data-open="DSU020C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Using an Illumina 9K iSelect SNP assay, 7,504 polymorphic loci were positioned in a consensus map constructed based on integrating information on seven common wheat mapping populations.<sup><a reveal-id="DSU020C10" data-open="DSU020C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p><p>Improvements in the throughput of NGS technologies made it possible to conduct SNP typing of an entire mapping population by direct sequencing, combining the process of polymorphism discovery and genotyping in a single experiment. In tetraploid and hexaploid wheat, sequence-based genotyping has been performed by sequencing of complexity-reduced genomic DNA<sup><a reveal-id="DSU020C11" data-open="DSU020C11" class="link link-ref link-reveal xref-bibr">11</a>â€“<a reveal-id="DSU020C13" data-open="DSU020C13" class="link link-ref link-reveal xref-bibr">13</a></sup> or a targeted genomic region.<sup><a reveal-id="DSU020C14" data-open="DSU020C14" class="link link-ref link-reveal xref-bibr">14</a></sup> The former, called complexity reduction of polymorphic sequences (CRoPS), restriction site-associated DNA sequencing (RAD-seq), or genotyping-by-sequencing (GBS),<sup><a reveal-id="DSU020C15" data-open="DSU020C15" class="link link-ref link-reveal xref-bibr">15</a></sup> was applied to the development of high-density genetic maps ranging from &gt;10,000<sup><a reveal-id="DSU020C11" data-open="DSU020C11" class="link link-ref link-reveal xref-bibr">11</a></sup> to &gt;400,000 markers<sup><a reveal-id="DSU020C12" data-open="DSU020C12" class="link link-ref link-reveal xref-bibr">12</a></sup> in the hexaploid wheat mapping population. In the latter, the targeted genomic region (for example, exome) can be captured and re-sequenced.<sup><a reveal-id="DSU020C14" data-open="DSU020C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Sequence-based genotyping is cost-effective and faster than array-based platforms, but often have a significant amount of missing data due to the low coverage of sequencing.<sup><a reveal-id="DSU020C15" data-open="DSU020C15" class="link link-ref link-reveal xref-bibr">15</a></sup></p><p>To efficiently construct a high-resolution map in common wheat with few missing data points, we developed a microarray-based polymorphism detection system using NGS data obtained from complexity-reduced genomic DNA samples of two common wheat cultivars, Chinese Spring (CS) and Mironovskaya 808 (M808), and genotyped 210 recombinant inbred lines (RILs) between the two cultivars. Moreover, quantitative trait locus (QTL) analysis was performed for 14 morphological and agronomically important traits using the high-density map constructed.</p>                            <h2 scrollto-destination=87199033 id="87199033" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87199034 id="87199034" class="section-title" data-legacy-id=s2a>2.1. Plant material</h3>
<p>CS and M808 were selected because of the difference in growth habit, geographical origin and abiotic stress responses, and the availability of RILs derived from them. M808 was bred in Mironovska Institute, Ukraine. It is reported to be one of the most freezing tolerant wheat cultivars tested.<sup><a reveal-id="DSU020C16" data-open="DSU020C16" class="link link-ref link-reveal xref-bibr">16</a></sup> In contrast, CS is a spring-type wheat landrace from Sichuan Province in China. A mapping population of 210 RILs derived from a cross between two common wheat cultivars, CS and M808, developed by Kobayashi <em>et al.</em>,<sup><a reveal-id="DSU020C16" data-open="DSU020C16" class="link link-ref link-reveal xref-bibr">16</a></sup> was used in this study. The plants were grown individually in pots arranged randomly in a field of Kobe University (34Â°43â€²N, 135Â°13â€²E) in two seasons (2009â€“10 [Kobe10, indicating location and growing season] and 2011â€“12 [Kobe12]), and in a glasshouse of Kyoto University (35Â°2â€²N, 135Â°47â€²E) in the 2011â€“12 season (Kyoto12). Total DNA from each RIL and the parents was extracted from leaves using standard procedures.<sup><a reveal-id="DSU020C16" data-open="DSU020C16" class="link link-ref link-reveal xref-bibr">16</a></sup></p>                            <h3 scrollto-destination=87199036 id="87199036" class="section-title" data-legacy-id=s2b>2.2. Library preparation and NGS</h3>
<p>DNA samples (180 ng) of CS and M808 were digested first with <em>Pst</em>I (New England BioLabs, Inc., Ipswich, MA, USA) at 37Â°C for 1 h and then with <em>Bst</em>NI (New England BioLabs) at 60Â°C for 1 h. We selected this combination of restriction enzymes based on <em>in silico</em> analysis in rice, our experience in sugarcane,<sup><a reveal-id="DSU020C17" data-open="DSU020C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSU020C18" data-open="DSU020C18" class="link link-ref link-reveal xref-bibr">18</a></sup> and its use in barley.<sup><a reveal-id="DSU020C4" data-open="DSU020C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Digested DNA was ligated to <em>Pst</em>I adapters (5â€²-CACGATGGATCCAGTGCA-3â€² and 5â€²-CTGGATCCATCGTGCA-3â€²) using T4 DNA ligase (Nippon Gene, Tokyo, Japan) and ATP (Wako Pure Chemical Industries, Osaka, Japan). The ligation reaction was run at 16Â°C for 16 h and the ligase was inactivated by holding at 60Â°C for 20 min.</p><p>Ligated samples were PCR-amplified for 30 cycles using 5â€²-GATGGATCCAGTGCAG-3â€² with PrimeSTAR HS DNA Polymerase (Takara Bio, Shiga, Japan) under the following conditions: 10 s at 98Â°C, 15 s at 55Â°C, and 60 s at 72Â°C. PCR products were purified using a MinElute PCR Purification Kit (Qiagen, Hilden, Germany). The constructed libraries were sequenced as 2 Ã— 100 nt paired-end reads on the Genome Analyzer IIx instrument (Illumina, San Diego, CA, USA). The sequences were deposited to the DNA Data Bank of Japan (DDBJ) Short Read Archive (DRA001257).</p>                            <h3 scrollto-destination=87199039 id="87199039" class="section-title" data-legacy-id=s2c>2.3. Processing of raw reads and design of probes</h3>
<p>First, trimming of adaptor and primer sequences and low-quality 3â€² ends (quality score â‰¤2) from raw reads was performed using an in-house developed software. Reads with the length of &lt;70 nt were discarded including pairs. CS and M808 libraries were <em>de novo</em> assembled separately using Velvet version 1.1.05.<sup><a reveal-id="DSU020C19" data-open="DSU020C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Velveth was run with a k-mer size of 65, and velvetg with expected coverage: â€˜autoâ€™, coverage cut-off: â€˜autoâ€™, insert length of 220 bp, minimum contig length of 100 bp, and allowing scaffolding. From contigs and unused reads (in assembly), tiling probes were designed with a <em>T</em><sub>m</sub> of âˆ¼76Â°C and the length of 50â€“70 bp.</p><p>To construct a primary array, the designed probes were first aligned to filtered reads of CS and M808 using Bowtie version 0.12.7,<sup><a reveal-id="DSU020C20" data-open="DSU020C20" class="link link-ref link-reveal xref-bibr">20</a></sup> where no more than three mismatches were allowed and up to 40 alignments were reported (-v 3, -k 40). CS-derived probes that met all the following five conditions were selected: (i) the number of aligned CS reads with no mismatch was â‰¥5, (ii) no aligned CS reads with 1â€“3 mismatches, (iii) no aligned M808 reads with a perfect match, (iv) no aligned M808 reads with one or two mismatches, and (v) the number of aligned M808 reads with three mismatches was â‰¤2. The last condition was included because this array-based polymorphism detection system has been demonstrated to accurately detect differences of more than three nucleotides.<sup><a reveal-id="DSU020C17" data-open="DSU020C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSU020C18" data-open="DSU020C18" class="link link-ref link-reveal xref-bibr">18</a></sup> A similar selection was performed with M808 probes. In total, 223,000 probes from both CS and M808 were spotted in triplicate on a NimbleGen 3 Ã— 1.4 M array (Roche Diagnostics, Mannheim, Germany).</p>                            <h3 scrollto-destination=87199042 id="87199042" class="section-title" data-legacy-id=s2d>2.4. Hybridization and construction of secondary array</h3>
<p>To select probes for the construction of a secondary array, CS, M808, and 14 randomly selected RILs were genotyped. After digestion of total DNA with <em>Pst</em>I and <em>Bst</em>NI, adapter ligation, PCR amplification and purification of PCR products as described in the library preparation section, they were labelled with Cy3 using the NimbleGen One-Color DNA Labeling Kit (Roche Diagnostics) according to the manufacturer's instructions. Hybridization was performed at 42Â°C for 72 h on a NimbleGen Hybridization System (Roche Diagnostics). Arrays were scanned using a NimbleGen MS200 Microarray Scanner (Roche Diagnostics) and genotype calling was performed based on the signal intensities.</p><p>For selection of probes, a <em>Ï‡</em><sup>2</sup> test was used to compute segregation of each probe in the 14 RILs. Those deviating from Mendelian 1 : 1 segregation at a 1% significance level were discarded. The secondary array was constructed using one selected probe per contig or read, consisting of 21,346 CS-derived probes, 21,205 M808-derived probes, and 2,303 probes for normalization. These 44,854 probes were spotted in triplicate on a NimbleGen 12 Ã— 135 K array (Roche Diagnostics). This array was used for genotyping of 210 RILs and the two parental lines.</p>                            <h3 scrollto-destination=87199045 id="87199045" class="section-title" data-legacy-id=s2e>2.5. Genetic map construction</h3>
<p>The genotyping data for 422 simple sequence repeat (SSR) markers were previously generated.<sup><a reveal-id="DSU020C16" data-open="DSU020C16" class="link link-ref link-reveal xref-bibr">16</a></sup> A total of 140 barc, 1 cfa, 34 cfd, 18 gdm, 109 gwm, 4 hbd, 15 hbe, 41 hbg, and 60 wmc loci were used for chromosomal assignment of wheat array-based markers (WABMs) developed in this study. <em>VRN-D1</em> was mapped using primer sets 5â€²-CTGGTTGTCTGCCTCATCA-3â€² and 5â€²-CTCTCTCCCCCTGCTAGT-3â€², and a second copy of <em>Ppd-B1</em><sup><a reveal-id="DSU020C21" data-open="DSU020C21" class="link link-ref link-reveal xref-bibr">21</a></sup> using 5â€²-TAACTGCTCGTCACAAGTGC-3â€² and 5â€²-CCGGAACCTGAGGATCATC-3â€². Loci were assembled into linkage groups using Antmap version 1.2<sup><a reveal-id="DSU020C22" data-open="DSU020C22" class="link link-ref link-reveal xref-bibr">22</a></sup> with the â€˜all combinationâ€™ method and different threshold distances (5.75, 12.0, 17.5, and 25.0 cM). Loci were ordered using MapDisto version 1.7.7,<sup><a reveal-id="DSU020C23" data-open="DSU020C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and the genetic distances were calculated with the Kosambi function.<sup><a reveal-id="DSU020C24" data-open="DSU020C24" class="link link-ref link-reveal xref-bibr">24</a></sup></p>                            <h3 scrollto-destination=87199047 id="87199047" class="section-title" data-legacy-id=s2f>2.6. BLAST searches of the mapped contigs and reads</h3>
<p>To search repeat elements (REs) in the mapped contigs and reads, they were used as queries against the Munich Information Center for Protein Sequences (MIPS) RE database version 9.0 for Poaceae (<a class="link link-uri" href="http://ftp://ftpmips.helmholtz-muenchen.de/plants/REdat/">ftp://ftpmips.helmholtz-muenchen.de/plants/REdat/</a>) using the blastn algorithm. A blastn search was also performed against <em>Ae. tauschii</em>,<sup><a reveal-id="DSU020C25" data-open="DSU020C25" class="link link-ref link-reveal xref-bibr">25</a></sup><em>Triticum urartu</em>,<sup><a reveal-id="DSU020C26" data-open="DSU020C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and barley high-confidence genes and genomic sequences.<sup><a reveal-id="DSU020C27" data-open="DSU020C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Contigs and reads mapping to the A-genome were searched by the blastn algorithm against the <em>T. urartu</em> draft genome sequence, while those mapping to the D-genome were searched against the <em>Ae. tauschii</em> draft genome sequence.<sup><a reveal-id="DSU020C25" data-open="DSU020C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSU020C26" data-open="DSU020C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSU020C28" data-open="DSU020C28" class="link link-ref link-reveal xref-bibr">28</a></sup> All BLAST hits were filtered with an <em>E</em>-value cut-off of 10<sup>âˆ’5</sup> and hit length of â‰¥50 bp. The genetic map and the results of the RE search were represented in a circular plot using the Circos version 0.63 software.<sup><a reveal-id="DSU020C29" data-open="DSU020C29" class="link link-ref link-reveal xref-bibr">29</a></sup></p>                            <h3 scrollto-destination=87199049 id="87199049" class="section-title" data-legacy-id=s2g>2.7. In silico digestion of the draft genome sequences of T. urartu and Ae. tauschii</h3>
<p>Using an in-house PERL script, genomic scaffolds of <em>T. urartu</em> and <em>Ae. tauschii</em> were fragmented at <em>Pst</em>I and <em>Bst</em>NI sites, and those having only <em>Pst</em>I sites at both ends but not <em>Bst</em>NI sites were extracted. A blastn search was performed against the MIPS RE database version 9.0 for Poaceae using as a query the extracted <em>T. urartu</em> and <em>Ae. tauschii</em> genomic fragments with lengths between 65 and 6,000 bp (<em>E</em>-value &lt;10<sup>âˆ’5</sup> and hit length â‰¥50 bp).</p>                            <h3 scrollto-destination=87199051 id="87199051" class="section-title" data-legacy-id=s2h>2.8. In silico mapping to the physical map of barley</h3>
<p>For reads and contigs mapping to the A- and D-genomes, a blastn search was first conducted against draft genomic sequences (<em>T. urartu</em> for the A-genome and <em>Ae. tauschii</em> for the D-genome)<sup><a reveal-id="DSU020C25" data-open="DSU020C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSU020C26" data-open="DSU020C26" class="link link-ref link-reveal xref-bibr">26</a></sup> as described above. The obtained scaffolds were mapped based on a blastn search of the genes contained in these scaffolds against the barley genome (<em>E</em>-value &lt;10<sup>âˆ’5</sup> and hit length â‰¥50 bp). The A- and D-genome scaffolds without genes as well as reads and contigs mapping to the B-genome were searched against the barley genome using the same blastn parameters.</p>                            <h3 scrollto-destination=87199053 id="87199053" class="section-title" data-legacy-id=s2i>2.9. Phenotypic measurements</h3>
<p>A list of traits, growing season, and location is presented in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>. Heading time (HT) and flowering time (FT) were recorded as days after sowing. Maturation time (MT) was evaluated as the number of days that had passed when the ear neck turned yellow. HT, FT, and MT were measured for the three earliest tillers of each plant, and mean values were calculated. Grain-filling period (GFP) was defined as the number of days from flowering to maturation.</p><p>Spike-related traits (such as spike length [SL], number of spikelets per spike [SpN], SL per number of spikelets [SLperSpN], and top five spikelets length [T5SpL]) and culm length (CL) were assessed for the three earliest tillers of each plant before anthesis. Seed-related traits such as seed length (SdL), seed width (SdW), seed height (SdH), and seed length-to-width ratio (SdLperW) were measured using seeds on the central spikelets of the first, second, and third spikes. For each spikelet, only the components of the first and second florets were used. The mean of the replicated measurements over three spikes was calculated for each trait and used as the trait value in subsequent analyses. SdL, SdW, SdH, and SdLperW were evaluated using at least 10 seeds per line. For tiller number (TN), the number of tillers bearing ears was counted at harvest.</p>                            <h3 scrollto-destination=87199056 id="87199056" class="section-title" data-legacy-id=s2j>2.10. QTL analysis</h3>
<p>QTLs were analysed by composite interval mapping using R/qtl package version 1.21-2<sup><a reveal-id="DSU020C30" data-open="DSU020C30" class="link link-ref link-reveal xref-bibr">30</a></sup> for the new map and for that constructed by Kobayashi <em>et al.</em><sup><a reveal-id="DSU020C16" data-open="DSU020C16" class="link link-ref link-reveal xref-bibr">16</a></sup> using SSR markers. This SSR map was constructed using 412 SSR markers. First, QTL genotype probability was calculated using the function calc.genoprob with a step size of 1 cM and Kosambi map function. QTL analysis was performed with cim function using three marker covariates and Kosambi map function. The log-likelihood (LOD) score threshold was determined by computing a 1,000 permutation test for all traits. The percentage of phenotypic variation (PV) explained by a QTL for a trait was estimated using the fitqtl function in R/qtl.</p>                            <h2 scrollto-destination=87199058 id="87199058" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87199059 id="87199059" class="section-title" data-legacy-id=s3a>3.1. Construction of a microarray</h3>
<div class="block-child-p">To design probes for the development of a microarray, we sequenced complexity-reduced genomic DNA samples from two common wheat cultivars, CS and M808. Genome complexity was reduced using a combination of two restriction enzymes, <em>Pst</em>I and <em>Bst</em>NI. After ligation of adapters with sticky ends complementary to the 3â€²-overhang created by <em>Pst</em>I, only the DNA fragments having a <em>Pst</em>I site at both ends, with no <em>Bst</em>NI site, were amplified by PCR. After trimming and filtering of raw reads, 63,563,396 (6.04 Gb, genome coverage of 0.36Ã— [<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&amp;Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>]) and 63,961,698 (6.09 Gb, 0.36Ã—) reads were generated for CS and M808, respectively (Fig.Â <a reveal-id="DSU020F1" data-open="DSU020F1" class="link link-reveal link-table xref-fig">1</a>). Both libraries were independently assembled using Velvet,<sup><a reveal-id="DSU020C19" data-open="DSU020C19" class="link link-ref link-reveal xref-bibr">19</a></sup> yielding 84,985 contigs and 39,324,373 unused reads during assembly for CS and 123,680 contigs and 33,006,701 unused reads for M808. <div data-id="DSU020F1" class="fig fig-section"><div class="label fig-label">FigureÂ 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/m_dsu02001.jpeg?Expires=2147483647&amp;Signature=ttE08aiD1P2oQnviXtlgj-6jIaXAbk91eC8TC~6tR~MkwmJLp3UGEsPjUim0jpwDT3ahl0s6jO7T7hNsFGcy-NX2dg1aT7MCcxuv1IYLfDuKNariZtfO4LxO2zcZXSQ9bvpt1SdQW53ayFWZMZHozBB6iWUhHXlzEkgR3ZIsHsobaZGFYqwXaTKsIjZ~76R2Sm-S3dF30WNHeszVaEfRdOBuYSV3fIi-48RZGMX9mmg~sh~71JArrN~Kbu6QoYIVF5kywd8kDykcFXppiqzko9CrBOnH17n9-KsfTHbx5tCc-PEFliye5OFObkPSLwM3ZUgLluUec0nwKs9bwC3Kng__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic of primary array development. Raw reads obtained from sequencing complexity-reduced CS and M808 genomic DNA were pre-processed for quality filtering prior to assembly with Velvet. Tiling probes were designed from contigs and unused reads. To predict non-polymorphic probes, alignment to filtered CS and M808 reads was performed. The primary array was constructed by spotting 446,000 filtered probes in triplicate. See Materials and methods for more details." path-from-xml="dsu02001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199060/dsu02001.jpeg" path-from-xml="dsu02001.jpeg" target="_blank">View large</a><a section="87199060" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu02001.jpeg?Expires=2147483647&amp;Signature=uwHW77gR8LLwXTg~~C9dI6BB~pWiStZzTd326vpxJIYWDppOYbPD7euV~nQ~qYxypeUyVV1ODQWlhtQ13ijtuWEKOyLs8jGNi-CvNhn9putdYodAxIQk3JV8VcSSAiDPIyNhOulZoV2nzJRMRnTUnczBGuYNXxbPL4ixh9GHoaoD-wv0SHFNusF2LIP6YfA-N9-7aWj3RGJV98~bML5s5HWv445iiJlZGaJ06sqHKijBpKcO5FMA~eQO6tLn-nzHdpkD3xkDn4gngZryvN-D~Zl40U560Nuq7R7bw6pwCBQG9VN8wpNd92Ml4N2PhbG0RdJvxaWKuFCJxrL1vouo0Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199060&amp;ar=2754552&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic of primary array development. Raw reads obtained from sequencing complexity-reduced CS and M808 genomic DNA were pre-processed for quality filtering prior to assembly with Velvet. Tiling probes were designed from contigs and unused reads. To predict non-polymorphic probes, alignment to filtered CS and M808 reads was performed. The primary array was constructed by spotting 446,000 filtered probes in triplicate. See Materials and methods for more details.</p></div></div><div content-id="DSU020F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">FigureÂ 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/m_dsu02001.jpeg?Expires=2147483647&amp;Signature=ttE08aiD1P2oQnviXtlgj-6jIaXAbk91eC8TC~6tR~MkwmJLp3UGEsPjUim0jpwDT3ahl0s6jO7T7hNsFGcy-NX2dg1aT7MCcxuv1IYLfDuKNariZtfO4LxO2zcZXSQ9bvpt1SdQW53ayFWZMZHozBB6iWUhHXlzEkgR3ZIsHsobaZGFYqwXaTKsIjZ~76R2Sm-S3dF30WNHeszVaEfRdOBuYSV3fIi-48RZGMX9mmg~sh~71JArrN~Kbu6QoYIVF5kywd8kDykcFXppiqzko9CrBOnH17n9-KsfTHbx5tCc-PEFliye5OFObkPSLwM3ZUgLluUec0nwKs9bwC3Kng__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic of primary array development. Raw reads obtained from sequencing complexity-reduced CS and M808 genomic DNA were pre-processed for quality filtering prior to assembly with Velvet. Tiling probes were designed from contigs and unused reads. To predict non-polymorphic probes, alignment to filtered CS and M808 reads was performed. The primary array was constructed by spotting 446,000 filtered probes in triplicate. See Materials and methods for more details." path-from-xml="dsu02001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199060/dsu02001.jpeg" path-from-xml="dsu02001.jpeg" target="_blank">View large</a><a section="87199060" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu02001.jpeg?Expires=2147483647&amp;Signature=uwHW77gR8LLwXTg~~C9dI6BB~pWiStZzTd326vpxJIYWDppOYbPD7euV~nQ~qYxypeUyVV1ODQWlhtQ13ijtuWEKOyLs8jGNi-CvNhn9putdYodAxIQk3JV8VcSSAiDPIyNhOulZoV2nzJRMRnTUnczBGuYNXxbPL4ixh9GHoaoD-wv0SHFNusF2LIP6YfA-N9-7aWj3RGJV98~bML5s5HWv445iiJlZGaJ06sqHKijBpKcO5FMA~eQO6tLn-nzHdpkD3xkDn4gngZryvN-D~Zl40U560Nuq7R7bw6pwCBQG9VN8wpNd92Ml4N2PhbG0RdJvxaWKuFCJxrL1vouo0Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199060&amp;ar=2754552&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic of primary array development. Raw reads obtained from sequencing complexity-reduced CS and M808 genomic DNA were pre-processed for quality filtering prior to assembly with Velvet. Tiling probes were designed from contigs and unused reads. To predict non-polymorphic probes, alignment to filtered CS and M808 reads was performed. The primary array was constructed by spotting 446,000 filtered probes in triplicate. See Materials and methods for more details.</p></div></div></div><p>From contigs and unused reads, tiling probes were designed. The probes were aligned with the filtered reads of CS and M808, and only the probes that predicted detection of â€˜presenceâ€™ or â€˜absenceâ€™ alleles were selected (see Section 2.3.). Thus, the primary array contained 1,338,000 probes per plex (446,000 probes in triplicate) on a 3 Ã— 1.4 M array (Fig.Â <a reveal-id="DSU020F1" data-open="DSU020F1" class="link link-reveal link-table xref-fig">1</a>) with a genome coverage of 4.51 Ã— 10<sup>âˆ’3</sup> fold.</p><p>The primary arrays were used for genotyping of CS, M808, and their 14 derived RILs, and of the 446,000 probes, 303,387 were polymorphic. Probes showing segregation distortion from a 1 : 1 ratio at a 1% significance level were discarded and only one probe per contig or read was selected. The final array contained 44,854 probes (including 2,303 probes for normalization, genome coverage of 7.45 Ã— 10<sup>âˆ’4</sup> fold) in triplicate on a 12 Ã— 135 K array and was used for genotyping of CS, M808, and 210 RILs. We named these microarray probes as WABMs.</p>                            <h3 scrollto-destination=87199063 id="87199063" class="section-title" data-legacy-id=s3b>3.2. Construction of a high-density linkage map</h3>
<p>In a previous study, a linkage map was constructed for 210 RILs of CS and M808 using 422 SSR loci.<sup><a reveal-id="DSU020C16" data-open="DSU020C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Of the 42,551 microarray probes, polymorphism was observed for 15,530 (36.5%) with a minor allele frequency of â‰¥10% among the 210 RILs. Missing data points were counted in each of the RILs and ranged from 17 to 1,074, with an average of 159.0 Â± 153.0 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). To use high-quality data in further analyses, nine RILs with â‰¥465.1 (mean + 2 SD) missing data points were excluded. Missing data points were also counted for each marker and ranged from 0 to 42, with an average of 2.49 Â± 5.84 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). Markers with &gt;20 missing data points were excluded, and 14,464 markers were used for genetic map construction.</p><p>In total, 13,480 markers, including 13,056 WABMs, 422 SSRs, <em>VRN-D1</em>, and <em>Ppd-B1</em>, were grouped into 21 linkage groups. The remaining markers did not form linkage group or the linkage group was small (&lt;10 cM) without chromosomal assignment. Of the mapped WABMs, 7,253 (55.6%) were probes derived from contigs. The probe sequences for these markers are presented in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>. The number of total markers per chromosome ranged from 1,256 on chromosome 2B to 85 on chromosome 4D (TableÂ <a reveal-id="DSU020TB1" data-open="DSU020TB1" class="link link-reveal link-table xref-fig">1</a>). The number of markers with a non-redundant (nr) genotype pattern was 5,243, comprising 2,744 loci (markers with the same position on genetic map). The total map length was 5,093.12 cM, with an average marker interval of 0.38 cM when considering all markers, 0.97 cM per nr marker, and 1.86 cM per locus. <div content-id="DSU020TB1" class="table-modal"><div class="table-wrap"><div id="DSU020TB1" data-id="DSU020TB1" class="table-wrap-title"><div class="label -label">TableÂ 1.</div><div class="caption"><p>Number of markers mapped to each chromosome of common wheat</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>ChromosomeÂ </th><th>Map length (cM)Â </th><th>Number of lociÂ </th><th>Number of nr markerÂ </th><th>Number of total markerÂ </th></tr></thead><tbody><tr><td>1AÂ </td><td>239.87Â </td><td>168Â </td><td>241Â </td><td>571Â </td></tr><tr><td>1BÂ </td><td>277.28Â </td><td>214Â </td><td>428Â </td><td>1,026Â </td></tr><tr><td>1DÂ </td><td>137.82Â </td><td>76Â </td><td>131Â </td><td>353Â </td></tr><tr><td>2AÂ </td><td>245.01Â </td><td>147Â </td><td>259Â </td><td>574Â </td></tr><tr><td>2BÂ </td><td>309.67Â </td><td>201Â </td><td>428Â </td><td>1,256Â </td></tr><tr><td>2DÂ </td><td>272.18Â </td><td>112Â </td><td>280Â </td><td>854Â </td></tr><tr><td>3AÂ </td><td>228.25Â </td><td>128Â </td><td>233Â </td><td>548Â </td></tr><tr><td>3BÂ </td><td>310.09Â </td><td>231Â </td><td>408Â </td><td>939Â </td></tr><tr><td>3DÂ </td><td>256.95Â </td><td>70Â </td><td>161Â </td><td>401Â </td></tr><tr><td>4AÂ </td><td>221.29Â </td><td>141Â </td><td>317Â </td><td>1,013Â </td></tr><tr><td>4BÂ </td><td>158.60Â </td><td>77Â </td><td>145Â </td><td>506Â </td></tr><tr><td>4DÂ </td><td>121.46Â </td><td>31Â </td><td>52Â </td><td>85Â </td></tr><tr><td>5AÂ </td><td>319.07Â </td><td>129Â </td><td>206Â </td><td>473Â </td></tr><tr><td>5BÂ </td><td>315.32Â </td><td>197Â </td><td>376Â </td><td>984Â </td></tr><tr><td>5DÂ </td><td>292.74Â </td><td>73Â </td><td>142Â </td><td>272Â </td></tr><tr><td>6AÂ </td><td>179.32Â </td><td>112Â </td><td>219Â </td><td>667Â </td></tr><tr><td>6BÂ </td><td>241.01Â </td><td>167Â </td><td>363Â </td><td>940Â </td></tr><tr><td>6DÂ </td><td>193.48Â </td><td>67Â </td><td>112Â </td><td>207Â </td></tr><tr><td>7AÂ </td><td>288.54Â </td><td>178Â </td><td>338Â </td><td>771Â </td></tr><tr><td>7BÂ </td><td>269.17Â </td><td>168Â </td><td>289Â </td><td>851Â </td></tr><tr><td>7DÂ </td><td>216.00Â </td><td>57Â </td><td>115Â </td><td>189Â </td></tr><tr><td>TotalÂ </td><td>5,093.12Â </td><td>2,744Â </td><td>5,243Â </td><td>13,480Â </td></tr><tr><td>A genomeÂ </td><td>1,721.35Â </td><td>1,003Â </td><td>1,813Â </td><td>4,617Â </td></tr><tr><td>B genomeÂ </td><td>1,881.14Â </td><td>1,255Â </td><td>2,437Â </td><td>6,502Â </td></tr><tr><td>D genomeÂ </td><td>1,490.63Â </td><td>486Â </td><td>993Â </td><td>2,361Â </td></tr></tbody></table></div><div content-id="DSU020TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>ChromosomeÂ </th><th>Map length (cM)Â </th><th>Number of lociÂ </th><th>Number of nr markerÂ </th><th>Number of total markerÂ </th></tr></thead><tbody><tr><td>1AÂ </td><td>239.87Â </td><td>168Â </td><td>241Â </td><td>571Â </td></tr><tr><td>1BÂ </td><td>277.28Â </td><td>214Â </td><td>428Â </td><td>1,026Â </td></tr><tr><td>1DÂ </td><td>137.82Â </td><td>76Â </td><td>131Â </td><td>353Â </td></tr><tr><td>2AÂ </td><td>245.01Â </td><td>147Â </td><td>259Â </td><td>574Â </td></tr><tr><td>2BÂ </td><td>309.67Â </td><td>201Â </td><td>428Â </td><td>1,256Â </td></tr><tr><td>2DÂ </td><td>272.18Â </td><td>112Â </td><td>280Â </td><td>854Â </td></tr><tr><td>3AÂ </td><td>228.25Â </td><td>128Â </td><td>233Â </td><td>548Â </td></tr><tr><td>3BÂ </td><td>310.09Â </td><td>231Â </td><td>408Â </td><td>939Â </td></tr><tr><td>3DÂ </td><td>256.95Â </td><td>70Â </td><td>161Â </td><td>401Â </td></tr><tr><td>4AÂ </td><td>221.29Â </td><td>141Â </td><td>317Â </td><td>1,013Â </td></tr><tr><td>4BÂ </td><td>158.60Â </td><td>77Â </td><td>145Â </td><td>506Â </td></tr><tr><td>4DÂ </td><td>121.46Â </td><td>31Â </td><td>52Â </td><td>85Â </td></tr><tr><td>5AÂ </td><td>319.07Â </td><td>129Â </td><td>206Â </td><td>473Â </td></tr><tr><td>5BÂ </td><td>315.32Â </td><td>197Â </td><td>376Â </td><td>984Â </td></tr><tr><td>5DÂ </td><td>292.74Â </td><td>73Â </td><td>142Â </td><td>272Â </td></tr><tr><td>6AÂ </td><td>179.32Â </td><td>112Â </td><td>219Â </td><td>667Â </td></tr><tr><td>6BÂ </td><td>241.01Â </td><td>167Â </td><td>363Â </td><td>940Â </td></tr><tr><td>6DÂ </td><td>193.48Â </td><td>67Â </td><td>112Â </td><td>207Â </td></tr><tr><td>7AÂ </td><td>288.54Â </td><td>178Â </td><td>338Â </td><td>771Â </td></tr><tr><td>7BÂ </td><td>269.17Â </td><td>168Â </td><td>289Â </td><td>851Â </td></tr><tr><td>7DÂ </td><td>216.00Â </td><td>57Â </td><td>115Â </td><td>189Â </td></tr><tr><td>TotalÂ </td><td>5,093.12Â </td><td>2,744Â </td><td>5,243Â </td><td>13,480Â </td></tr><tr><td>A genomeÂ </td><td>1,721.35Â </td><td>1,003Â </td><td>1,813Â </td><td>4,617Â </td></tr><tr><td>B genomeÂ </td><td>1,881.14Â </td><td>1,255Â </td><td>2,437Â </td><td>6,502Â </td></tr><tr><td>D genomeÂ </td><td>1,490.63Â </td><td>486Â </td><td>993Â </td><td>2,361Â </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="DSU020TB1" data-id="DSU020TB1" class="table-wrap-title"><div class="label -label">TableÂ 1.</div><div class="caption"><p>Number of markers mapped to each chromosome of common wheat</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>ChromosomeÂ </th><th>Map length (cM)Â </th><th>Number of lociÂ </th><th>Number of nr markerÂ </th><th>Number of total markerÂ </th></tr></thead><tbody><tr><td>1AÂ </td><td>239.87Â </td><td>168Â </td><td>241Â </td><td>571Â </td></tr><tr><td>1BÂ </td><td>277.28Â </td><td>214Â </td><td>428Â </td><td>1,026Â </td></tr><tr><td>1DÂ </td><td>137.82Â </td><td>76Â </td><td>131Â </td><td>353Â </td></tr><tr><td>2AÂ </td><td>245.01Â </td><td>147Â </td><td>259Â </td><td>574Â </td></tr><tr><td>2BÂ </td><td>309.67Â </td><td>201Â </td><td>428Â </td><td>1,256Â </td></tr><tr><td>2DÂ </td><td>272.18Â </td><td>112Â </td><td>280Â </td><td>854Â </td></tr><tr><td>3AÂ </td><td>228.25Â </td><td>128Â </td><td>233Â </td><td>548Â </td></tr><tr><td>3BÂ </td><td>310.09Â </td><td>231Â </td><td>408Â </td><td>939Â </td></tr><tr><td>3DÂ </td><td>256.95Â </td><td>70Â </td><td>161Â </td><td>401Â </td></tr><tr><td>4AÂ </td><td>221.29Â </td><td>141Â </td><td>317Â </td><td>1,013Â </td></tr><tr><td>4BÂ </td><td>158.60Â </td><td>77Â </td><td>145Â </td><td>506Â </td></tr><tr><td>4DÂ </td><td>121.46Â </td><td>31Â </td><td>52Â </td><td>85Â </td></tr><tr><td>5AÂ </td><td>319.07Â </td><td>129Â </td><td>206Â </td><td>473Â </td></tr><tr><td>5BÂ </td><td>315.32Â </td><td>197Â </td><td>376Â </td><td>984Â </td></tr><tr><td>5DÂ </td><td>292.74Â </td><td>73Â </td><td>142Â </td><td>272Â </td></tr><tr><td>6AÂ </td><td>179.32Â </td><td>112Â </td><td>219Â </td><td>667Â </td></tr><tr><td>6BÂ </td><td>241.01Â </td><td>167Â </td><td>363Â </td><td>940Â </td></tr><tr><td>6DÂ </td><td>193.48Â </td><td>67Â </td><td>112Â </td><td>207Â </td></tr><tr><td>7AÂ </td><td>288.54Â </td><td>178Â </td><td>338Â </td><td>771Â </td></tr><tr><td>7BÂ </td><td>269.17Â </td><td>168Â </td><td>289Â </td><td>851Â </td></tr><tr><td>7DÂ </td><td>216.00Â </td><td>57Â </td><td>115Â </td><td>189Â </td></tr><tr><td>TotalÂ </td><td>5,093.12Â </td><td>2,744Â </td><td>5,243Â </td><td>13,480Â </td></tr><tr><td>A genomeÂ </td><td>1,721.35Â </td><td>1,003Â </td><td>1,813Â </td><td>4,617Â </td></tr><tr><td>B genomeÂ </td><td>1,881.14Â </td><td>1,255Â </td><td>2,437Â </td><td>6,502Â </td></tr><tr><td>D genomeÂ </td><td>1,490.63Â </td><td>486Â </td><td>993Â </td><td>2,361Â </td></tr></tbody></table></div><div content-id="DSU020TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>ChromosomeÂ </th><th>Map length (cM)Â </th><th>Number of lociÂ </th><th>Number of nr markerÂ </th><th>Number of total markerÂ </th></tr></thead><tbody><tr><td>1AÂ </td><td>239.87Â </td><td>168Â </td><td>241Â </td><td>571Â </td></tr><tr><td>1BÂ </td><td>277.28Â </td><td>214Â </td><td>428Â </td><td>1,026Â </td></tr><tr><td>1DÂ </td><td>137.82Â </td><td>76Â </td><td>131Â </td><td>353Â </td></tr><tr><td>2AÂ </td><td>245.01Â </td><td>147Â </td><td>259Â </td><td>574Â </td></tr><tr><td>2BÂ </td><td>309.67Â </td><td>201Â </td><td>428Â </td><td>1,256Â </td></tr><tr><td>2DÂ </td><td>272.18Â </td><td>112Â </td><td>280Â </td><td>854Â </td></tr><tr><td>3AÂ </td><td>228.25Â </td><td>128Â </td><td>233Â </td><td>548Â </td></tr><tr><td>3BÂ </td><td>310.09Â </td><td>231Â </td><td>408Â </td><td>939Â </td></tr><tr><td>3DÂ </td><td>256.95Â </td><td>70Â </td><td>161Â </td><td>401Â </td></tr><tr><td>4AÂ </td><td>221.29Â </td><td>141Â </td><td>317Â </td><td>1,013Â </td></tr><tr><td>4BÂ </td><td>158.60Â </td><td>77Â </td><td>145Â </td><td>506Â </td></tr><tr><td>4DÂ </td><td>121.46Â </td><td>31Â </td><td>52Â </td><td>85Â </td></tr><tr><td>5AÂ </td><td>319.07Â </td><td>129Â </td><td>206Â </td><td>473Â </td></tr><tr><td>5BÂ </td><td>315.32Â </td><td>197Â </td><td>376Â </td><td>984Â </td></tr><tr><td>5DÂ </td><td>292.74Â </td><td>73Â </td><td>142Â </td><td>272Â </td></tr><tr><td>6AÂ </td><td>179.32Â </td><td>112Â </td><td>219Â </td><td>667Â </td></tr><tr><td>6BÂ </td><td>241.01Â </td><td>167Â </td><td>363Â </td><td>940Â </td></tr><tr><td>6DÂ </td><td>193.48Â </td><td>67Â </td><td>112Â </td><td>207Â </td></tr><tr><td>7AÂ </td><td>288.54Â </td><td>178Â </td><td>338Â </td><td>771Â </td></tr><tr><td>7BÂ </td><td>269.17Â </td><td>168Â </td><td>289Â </td><td>851Â </td></tr><tr><td>7DÂ </td><td>216.00Â </td><td>57Â </td><td>115Â </td><td>189Â </td></tr><tr><td>TotalÂ </td><td>5,093.12Â </td><td>2,744Â </td><td>5,243Â </td><td>13,480Â </td></tr><tr><td>A genomeÂ </td><td>1,721.35Â </td><td>1,003Â </td><td>1,813Â </td><td>4,617Â </td></tr><tr><td>B genomeÂ </td><td>1,881.14Â </td><td>1,255Â </td><td>2,437Â </td><td>6,502Â </td></tr><tr><td>D genomeÂ </td><td>1,490.63Â </td><td>486Â </td><td>993Â </td><td>2,361Â </td></tr></tbody></table></div></div></p><div class="block-child-p">Segregation distortion was computed for each marker and <em>P</em>-values of &lt;1.82 Ã— 10<sup>âˆ’5</sup> (Bonferroni corrected <em>P</em>-value: 0.05/2,744 loci) from a <em>Ï‡</em><sup>2</sup> test (1 : 1 ratio) were observed for markers of chromosomes 2B, 2D, 5A, 5B, 6B, and 7B (Fig.Â <a reveal-id="DSU020F2" data-open="DSU020F2" class="link link-reveal link-table xref-fig">2</a>, Tracks 6 and 7; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&amp;Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). The greatest distortion was observed at the telomeric region of chromosome 6BL (235.8 cM), where the allele frequency was 89.5% for CS and 10.5% for M808. <div data-id="DSU020F2" class="fig fig-section"><div class="label fig-label">FigureÂ 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/m_dsu02002.jpeg?Expires=2147483647&amp;Signature=mNAlhWRkQxN1eRMm~yoaVTAqQkYmLkPYK4oytZoyHtBRrgYr1TL5VnWPVrWzwX~0xcNahaa9Jb-W4JgKg0fx99SySue4rtHcYnC4FTJcwPivtZLfQxT1Ndjcx8txxvOI6CvLgNXBcqnFtOMZkzw0U9mjLt4dCjeICW7C6buQLD1Hk2e6LlCnUy2uhm75RcSSIdfk8~qhCPThE0EbCPp1qJD80l3aDRlwjIg~miPFm5GTnsVPmVvpuBftr9r23N1gx1Oz8PyYV4SmO8nMSS~YoJqJv4vZEmAVCkcJM1ewTI8I9WVUcODAYmgITj7P7Dsb2AbV4~cLikLW72LZGOQqmQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of WABMs, REs, missing data, and allele frequency on the constructed genetic map. From the outer to the inner circle, Track 1 shows the genetic map (scale in cM); Track 2 represents the distribution of sequence derived from genic regions; Tracks 3 and 4 represent, respectively, the distribution of retroelements and DNA transposons; Tracks 5 and 6, respectively, a heat map of missing data and P-values for segregation distortion; Track 7, the allele frequency of M808 (green) and CS (orange); and Track 8, a histogram of the number of markers (y-axis range 0â€“100)." path-from-xml="dsu02002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199066/dsu02002.jpeg" path-from-xml="dsu02002.jpeg" target="_blank">View large</a><a section="87199066" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu02002.jpeg?Expires=2147483647&amp;Signature=Tnp530fIknu1afqdiUtegx8whQwNwG60XVGEkuB5~BHNOfOYaUYQ6z3Ynv37E6gu1wWYTInxw4e5RpDSky8uh5a8o0I4IFsGI9faZri0Hm8erBlyjU0uysCNewC-A7zYp6lfWICS4RmbXe0kBkv6GqYxB8TZZ8kqBUJvzJxPxNn-bFJgnEl~BhQLvSblDI7veKO0tRvFO3zuf~eVCaKKap3lW42abVNFGwX2EJlfGA1DK~Q8Fqa7hLFzoa~z2Pff6oDWwRt-3qZwzUaSNELIQ4a8y7Qe~zqCRd4QfDskw7TjJYRY6bEwUg7wdpAquzJDZx9WytjXaSdmjkRTJ6KUtA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199066&amp;ar=2754552&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of WABMs, REs, missing data, and allele frequency on the constructed genetic map. From the outer to the inner circle, Track 1 shows the genetic map (scale in cM); Track 2 represents the distribution of sequence derived from genic regions; Tracks 3 and 4 represent, respectively, the distribution of retroelements and DNA transposons; Tracks 5 and 6, respectively, a heat map of missing data and <em>P</em>-values for segregation distortion; Track 7, the allele frequency of M808 (green) and CS (orange); and Track 8, a histogram of the number of markers (<em>y</em>-axis range 0â€“100).</p></div></div><div content-id="DSU020F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">FigureÂ 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/m_dsu02002.jpeg?Expires=2147483647&amp;Signature=mNAlhWRkQxN1eRMm~yoaVTAqQkYmLkPYK4oytZoyHtBRrgYr1TL5VnWPVrWzwX~0xcNahaa9Jb-W4JgKg0fx99SySue4rtHcYnC4FTJcwPivtZLfQxT1Ndjcx8txxvOI6CvLgNXBcqnFtOMZkzw0U9mjLt4dCjeICW7C6buQLD1Hk2e6LlCnUy2uhm75RcSSIdfk8~qhCPThE0EbCPp1qJD80l3aDRlwjIg~miPFm5GTnsVPmVvpuBftr9r23N1gx1Oz8PyYV4SmO8nMSS~YoJqJv4vZEmAVCkcJM1ewTI8I9WVUcODAYmgITj7P7Dsb2AbV4~cLikLW72LZGOQqmQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of WABMs, REs, missing data, and allele frequency on the constructed genetic map. From the outer to the inner circle, Track 1 shows the genetic map (scale in cM); Track 2 represents the distribution of sequence derived from genic regions; Tracks 3 and 4 represent, respectively, the distribution of retroelements and DNA transposons; Tracks 5 and 6, respectively, a heat map of missing data and P-values for segregation distortion; Track 7, the allele frequency of M808 (green) and CS (orange); and Track 8, a histogram of the number of markers (y-axis range 0â€“100)." path-from-xml="dsu02002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199066/dsu02002.jpeg" path-from-xml="dsu02002.jpeg" target="_blank">View large</a><a section="87199066" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu02002.jpeg?Expires=2147483647&amp;Signature=Tnp530fIknu1afqdiUtegx8whQwNwG60XVGEkuB5~BHNOfOYaUYQ6z3Ynv37E6gu1wWYTInxw4e5RpDSky8uh5a8o0I4IFsGI9faZri0Hm8erBlyjU0uysCNewC-A7zYp6lfWICS4RmbXe0kBkv6GqYxB8TZZ8kqBUJvzJxPxNn-bFJgnEl~BhQLvSblDI7veKO0tRvFO3zuf~eVCaKKap3lW42abVNFGwX2EJlfGA1DK~Q8Fqa7hLFzoa~z2Pff6oDWwRt-3qZwzUaSNELIQ4a8y7Qe~zqCRd4QfDskw7TjJYRY6bEwUg7wdpAquzJDZx9WytjXaSdmjkRTJ6KUtA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199066&amp;ar=2754552&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of WABMs, REs, missing data, and allele frequency on the constructed genetic map. From the outer to the inner circle, Track 1 shows the genetic map (scale in cM); Track 2 represents the distribution of sequence derived from genic regions; Tracks 3 and 4 represent, respectively, the distribution of retroelements and DNA transposons; Tracks 5 and 6, respectively, a heat map of missing data and <em>P</em>-values for segregation distortion; Track 7, the allele frequency of M808 (green) and CS (orange); and Track 8, a histogram of the number of markers (<em>y</em>-axis range 0â€“100).</p></div></div></div>                            <h3 scrollto-destination=87199067 id="87199067" class="section-title" data-legacy-id=s3c>3.3. Annotation of mapped contigs and reads</h3>
<div class="block-child-p">To annotate mapped contigs or reads, blastn searches were conducted against protein-encoding genes of <em>Ae. tauschii</em>, <em>T. urartu</em>, and barley, and against the RE database of the Poaceae of the MIPS. Of the 13,056 contigs/reads, 32.5, 27.0, and 23.9%, respectively, had hits to the <em>Ae. tauschii</em>, <em>T. urartu</em>, and barley genes (Fig.Â <a reveal-id="DSU020F3" data-open="DSU020F3" class="link link-reveal link-table xref-fig">3</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&amp;Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). Overall, 39.8% of the contigs/reads had at least one hit in one of these three databases. These contigs/reads derived from genic regions were distributed all across the wheat genome and chromosomes (Fig.Â <a reveal-id="DSU020F2" data-open="DSU020F2" class="link link-reveal link-table xref-fig">2</a>, Track 2; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&amp;Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). <div data-id="DSU020F3" class="fig fig-section"><div class="label fig-label">FigureÂ 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/m_dsu02003.jpeg?Expires=2147483647&amp;Signature=atKucQUANkhJUmB6B0lq-2CQcDuQM9eu0-X33ZPMy0lbhLTlC5T0b78p~XEoJunl8rhUmVytEKGbV~N5K9ZjbFK9mxoYQr8esFQpGapXVYLhKUgpSQkAb2sYOKFKHV5y8cAk2ApFD6TeWfmgmNmKewjCZOph~pG6AbyVnKELob0ZWYQmf1HY2xyK9Xoyh9eHqT4G1At0miNPzU7U9Dx7HJht60s2qYzFFlum0JF2ECHo~Ql5~~MZlNfd7TeFV4qSo2bLuvP8pWS~uAe45xd74822z2jnRRmTsso7YnAj~LPhnoBJnaUk98tpzEUKmB901b4H-SHhqBXe4lefoAkUSQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Histogram of annotated WABMs revealed by blastn analyses of genome databases. Reads/contigs anchored to the high-density genetic map were searched using the blastn algorithm against genes of Ae. tauschii, T. urartu, and barley and against Poaceae REs." path-from-xml="dsu02003.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199068/dsu02003.jpeg" path-from-xml="dsu02003.jpeg" target="_blank">View large</a><a section="87199068" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu02003.jpeg?Expires=2147483647&amp;Signature=3wVf5H77Pt96JsG991ck-M1t9qcigfmqR6dUGxHgDGj5jk3m5y8v51zL7wIsF~9MdmWdtRQ7qW2-jCMWFboy~vySdRT7GWl3TyqOhmACflWB-2Ll7FiSHCcMyED86~bmTyBmMJFAebDOb-YR1dsE7~ifKVWguZ0TIlWC3HvqoTCqResu1~du9oPtntB4th1USj~Z0bz621c7F~PWbTal8zKJ0PaW~SYDN-TPo3lPd9I-b~wyeP1dr1VpuTMFeKpmYhNoescMNSzlbthQiNxQKasrprZnH2UBZ7YPo1QnqadFriclfV4as2KIgZBMddefaLnx-XCfCt~QybOPJ-7lFQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199068&amp;ar=2754552&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02003.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Histogram of annotated WABMs revealed by blastn analyses of genome databases. Reads/contigs anchored to the high-density genetic map were searched using the blastn algorithm against genes of <em>Ae. tauschii</em>, <em>T. urartu</em>, and barley and against Poaceae REs.</p></div></div><div content-id="DSU020F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">FigureÂ 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/m_dsu02003.jpeg?Expires=2147483647&amp;Signature=atKucQUANkhJUmB6B0lq-2CQcDuQM9eu0-X33ZPMy0lbhLTlC5T0b78p~XEoJunl8rhUmVytEKGbV~N5K9ZjbFK9mxoYQr8esFQpGapXVYLhKUgpSQkAb2sYOKFKHV5y8cAk2ApFD6TeWfmgmNmKewjCZOph~pG6AbyVnKELob0ZWYQmf1HY2xyK9Xoyh9eHqT4G1At0miNPzU7U9Dx7HJht60s2qYzFFlum0JF2ECHo~Ql5~~MZlNfd7TeFV4qSo2bLuvP8pWS~uAe45xd74822z2jnRRmTsso7YnAj~LPhnoBJnaUk98tpzEUKmB901b4H-SHhqBXe4lefoAkUSQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Histogram of annotated WABMs revealed by blastn analyses of genome databases. Reads/contigs anchored to the high-density genetic map were searched using the blastn algorithm against genes of Ae. tauschii, T. urartu, and barley and against Poaceae REs." path-from-xml="dsu02003.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199068/dsu02003.jpeg" path-from-xml="dsu02003.jpeg" target="_blank">View large</a><a section="87199068" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu02003.jpeg?Expires=2147483647&amp;Signature=3wVf5H77Pt96JsG991ck-M1t9qcigfmqR6dUGxHgDGj5jk3m5y8v51zL7wIsF~9MdmWdtRQ7qW2-jCMWFboy~vySdRT7GWl3TyqOhmACflWB-2Ll7FiSHCcMyED86~bmTyBmMJFAebDOb-YR1dsE7~ifKVWguZ0TIlWC3HvqoTCqResu1~du9oPtntB4th1USj~Z0bz621c7F~PWbTal8zKJ0PaW~SYDN-TPo3lPd9I-b~wyeP1dr1VpuTMFeKpmYhNoescMNSzlbthQiNxQKasrprZnH2UBZ7YPo1QnqadFriclfV4as2KIgZBMddefaLnx-XCfCt~QybOPJ-7lFQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199068&amp;ar=2754552&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02003.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Histogram of annotated WABMs revealed by blastn analyses of genome databases. Reads/contigs anchored to the high-density genetic map were searched using the blastn algorithm against genes of <em>Ae. tauschii</em>, <em>T. urartu</em>, and barley and against Poaceae REs.</p></div></div></div><p>REs were found in 11.4% of the mapped contigs/reads (Fig.Â <a reveal-id="DSU020F3" data-open="DSU020F3" class="link link-reveal link-table xref-fig">3</a>). The proportion of DNA transposons and retroelements was similar (TableÂ <a reveal-id="DSU020TB2" data-open="DSU020TB2" class="link link-reveal link-table xref-fig">2</a>), and no bias in the distribution was found for these REs (Fig.Â <a reveal-id="DSU020F2" data-open="DSU020F2" class="link link-reveal link-table xref-fig">2</a>, Tracks 3 and 4; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). The <em>CACTA</em> superfamily (present in 38.7% of the probes) was mainly found among the DNA transposons followed by the miniature inverted-repeat transposable element (MITE) family (25.6%) and <em>Mariner</em> superfamily (17.5%). Among the retroelements, <em>Gypsy</em> (30%), <em>Copia</em> (23%), and other (29%) long terminal repeat (LTR) retrotransposons were the predominant classes. <div content-id="DSU020TB2" class="table-modal"><div class="table-wrap"><div id="DSU020TB2" data-id="DSU020TB2" class="table-wrap-title"><div class="label -label">TableÂ 2.</div><div class="caption"><p>REs found in the mapped contigs and reads</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Repeat elementsÂ </th><th>Number of contigs (%)Â </th></tr></thead><tbody><tr><td>DNA transposonsÂ </td><td>679 (45.8)Â </td></tr><tr><td>â€ƒ<em>CACTA</em>Â </td><td>263 (38.7)Â </td></tr><tr><td>â€ƒMITEÂ </td><td>174 (25.6)Â </td></tr><tr><td>â€ƒ<em>Tc1</em>/<em>Mariner</em>Â </td><td>119 (17.5)Â </td></tr><tr><td>â€ƒOther DNA transposonsÂ </td><td>57 (8.39)Â </td></tr><tr><td>â€ƒ<em>Mutator</em>Â </td><td>37 (5.4)Â </td></tr><tr><td>â€ƒ<em>Harbinger</em>Â </td><td>29 (4.3)Â </td></tr><tr><td>RetroelementsÂ </td><td>659 (44.4)Â </td></tr><tr><td>â€ƒLTR/<em>Gypsy</em>Â </td><td>196 (29.7)Â </td></tr><tr><td>â€ƒOther LTRsÂ </td><td>193 (29.3)Â </td></tr><tr><td>â€ƒLTR/<em>Copia</em>Â </td><td>152 (23.1)Â </td></tr><tr><td>â€ƒLINEÂ </td><td>116 (17.6)Â </td></tr><tr><td>â€ƒSINEÂ </td><td>2 (0.3)Â </td></tr><tr><td>OthersÂ </td><td>146 (9.8)Â </td></tr><tr><td>TotalÂ </td><td>1,484Â </td></tr></tbody></table></div><div content-id="DSU020TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Repeat elementsÂ </th><th>Number of contigs (%)Â </th></tr></thead><tbody><tr><td>DNA transposonsÂ </td><td>679 (45.8)Â </td></tr><tr><td>â€ƒ<em>CACTA</em>Â </td><td>263 (38.7)Â </td></tr><tr><td>â€ƒMITEÂ </td><td>174 (25.6)Â </td></tr><tr><td>â€ƒ<em>Tc1</em>/<em>Mariner</em>Â </td><td>119 (17.5)Â </td></tr><tr><td>â€ƒOther DNA transposonsÂ </td><td>57 (8.39)Â </td></tr><tr><td>â€ƒ<em>Mutator</em>Â </td><td>37 (5.4)Â </td></tr><tr><td>â€ƒ<em>Harbinger</em>Â </td><td>29 (4.3)Â </td></tr><tr><td>RetroelementsÂ </td><td>659 (44.4)Â </td></tr><tr><td>â€ƒLTR/<em>Gypsy</em>Â </td><td>196 (29.7)Â </td></tr><tr><td>â€ƒOther LTRsÂ </td><td>193 (29.3)Â </td></tr><tr><td>â€ƒLTR/<em>Copia</em>Â </td><td>152 (23.1)Â </td></tr><tr><td>â€ƒLINEÂ </td><td>116 (17.6)Â </td></tr><tr><td>â€ƒSINEÂ </td><td>2 (0.3)Â </td></tr><tr><td>OthersÂ </td><td>146 (9.8)Â </td></tr><tr><td>TotalÂ </td><td>1,484Â </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="DSU020TB2" data-id="DSU020TB2" class="table-wrap-title"><div class="label -label">TableÂ 2.</div><div class="caption"><p>REs found in the mapped contigs and reads</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Repeat elementsÂ </th><th>Number of contigs (%)Â </th></tr></thead><tbody><tr><td>DNA transposonsÂ </td><td>679 (45.8)Â </td></tr><tr><td>â€ƒ<em>CACTA</em>Â </td><td>263 (38.7)Â </td></tr><tr><td>â€ƒMITEÂ </td><td>174 (25.6)Â </td></tr><tr><td>â€ƒ<em>Tc1</em>/<em>Mariner</em>Â </td><td>119 (17.5)Â </td></tr><tr><td>â€ƒOther DNA transposonsÂ </td><td>57 (8.39)Â </td></tr><tr><td>â€ƒ<em>Mutator</em>Â </td><td>37 (5.4)Â </td></tr><tr><td>â€ƒ<em>Harbinger</em>Â </td><td>29 (4.3)Â </td></tr><tr><td>RetroelementsÂ </td><td>659 (44.4)Â </td></tr><tr><td>â€ƒLTR/<em>Gypsy</em>Â </td><td>196 (29.7)Â </td></tr><tr><td>â€ƒOther LTRsÂ </td><td>193 (29.3)Â </td></tr><tr><td>â€ƒLTR/<em>Copia</em>Â </td><td>152 (23.1)Â </td></tr><tr><td>â€ƒLINEÂ </td><td>116 (17.6)Â </td></tr><tr><td>â€ƒSINEÂ </td><td>2 (0.3)Â </td></tr><tr><td>OthersÂ </td><td>146 (9.8)Â </td></tr><tr><td>TotalÂ </td><td>1,484Â </td></tr></tbody></table></div><div content-id="DSU020TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Repeat elementsÂ </th><th>Number of contigs (%)Â </th></tr></thead><tbody><tr><td>DNA transposonsÂ </td><td>679 (45.8)Â </td></tr><tr><td>â€ƒ<em>CACTA</em>Â </td><td>263 (38.7)Â </td></tr><tr><td>â€ƒMITEÂ </td><td>174 (25.6)Â </td></tr><tr><td>â€ƒ<em>Tc1</em>/<em>Mariner</em>Â </td><td>119 (17.5)Â </td></tr><tr><td>â€ƒOther DNA transposonsÂ </td><td>57 (8.39)Â </td></tr><tr><td>â€ƒ<em>Mutator</em>Â </td><td>37 (5.4)Â </td></tr><tr><td>â€ƒ<em>Harbinger</em>Â </td><td>29 (4.3)Â </td></tr><tr><td>RetroelementsÂ </td><td>659 (44.4)Â </td></tr><tr><td>â€ƒLTR/<em>Gypsy</em>Â </td><td>196 (29.7)Â </td></tr><tr><td>â€ƒOther LTRsÂ </td><td>193 (29.3)Â </td></tr><tr><td>â€ƒLTR/<em>Copia</em>Â </td><td>152 (23.1)Â </td></tr><tr><td>â€ƒLINEÂ </td><td>116 (17.6)Â </td></tr><tr><td>â€ƒSINEÂ </td><td>2 (0.3)Â </td></tr><tr><td>OthersÂ </td><td>146 (9.8)Â </td></tr><tr><td>TotalÂ </td><td>1,484Â </td></tr></tbody></table></div></div></p><p>To test whether digestion with <em>Pst</em>I and <em>Bst</em>NI and selection of DNA fragments having only <em>Pst</em>I sites at both ends reduced the genomic fraction containing REs, <em>T. urartu</em> and <em>Ae. tauschii</em> genomic scaffolds were digested <em>in silico</em>. Overall, 136,189 and 106,724 fragments were, respectively, generated from the <em>T. urartu</em> and <em>Ae. tauschii</em> scaffolds. The size of A-genome fragments ranged from 6 to 23,779 bp with an average of 57.2 Â± 551.7 bp, and that of D-genome fragments varied from 6 to 28,677 bp with an average of 44.9 Â± 346.7 bp. The majority of these fragments were â‰¤80 bp (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). Because contigs and reads obtained from the NGS data of CS and M808 ranged from 65 bp to &lt;6,000 bp, a blastn search of the <em>in silico</em> digested genomic sequences against the MIPS RE database was performed using as query the A- and D-genome-derived fragments between 65 and 6,000 bp. REs were found in 84,410 (76.1%) of the 110,984 A-genome fragments, and in 60,522 (69.2%) of the 87,466 D-genome fragments. In contrast to the sequences mapping to the linkage map, retroelements predominated on DNA transposons in the fragments derived from the <em>in silico</em> digestion of both <em>T. urartu</em> and <em>Ae. tauschii</em> genomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). REs found were mainly members of the <em>CACTA</em> superfamily of DNA transposons or <em>Gypsy</em> retroelements.</p>                            <h3 scrollto-destination=87199071 id="87199071" class="section-title" data-legacy-id=s3d>3.4. In silico mapping of the contigs/reads to the physical map of barley</h3>
<div class="block-child-p">To determine how many of the mapped sequences could be used for the analysis of synteny between wheat and barley (<em>Hordeum vulgare</em> L., H-genome) chromosomes, they were mapped <em>in silico</em> to the physical map of barley.<sup><a reveal-id="DSU020C27" data-open="DSU020C27" class="link link-ref link-reveal xref-bibr">27</a></sup> For sequences mapping to the wheat A- and D-genomes, a blastn search was first conducted, respectively, against the <em>T. urartu</em> and <em>Ae. tauschii</em> draft genomic sequences;<sup><a reveal-id="DSU020C25" data-open="DSU020C25" class="link link-ref link-reveal xref-bibr">25</a>,<a reveal-id="DSU020C26" data-open="DSU020C26" class="link link-ref link-reveal xref-bibr">26</a></sup> the scaffolds obtained were mapped based on a blastn search of the genes contained in these scaffolds against the barley genome. Of the 4,462 WABMs mapping to the A-genome, hits against the <em>T. urartu</em> genomic scaffolds were obtained in 3,918 (87.8%) sequences, and 2,856 (64.0%) were anchored to the barley physical map (TableÂ <a reveal-id="DSU020TB3" data-open="DSU020TB3" class="link link-reveal link-table xref-fig">3</a>). In total, 1,658 (37.2%) markers were mapped to the same homoeologous group chromosomes (e.g. between 1A and 1H), accounting for âˆ¼40% for most of the chromosomes but only 8.5% for chromosome 4A. Collinearity between the A- and H-genomes was observed for many of these markers (Fig.Â <a reveal-id="DSU020F4" data-open="DSU020F4" class="link link-reveal link-table xref-fig">4</a>), except on chromosome 4A, where inversions and translocations have been reported.<sup><a reveal-id="DSU020C31" data-open="DSU020C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSU020C32" data-open="DSU020C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Translocations among chromosomes 4A, 5A, and 7B could also be observed, as previously reported (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&amp;Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). <div content-id="DSU020TB3" class="table-modal"><div class="table-wrap"><div id="DSU020TB3" data-id="DSU020TB3" class="table-wrap-title"><div class="label -label">TableÂ 3.</div><div class="caption"><p>Summary of blastn results against genome sequences</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Mapped sequencesÂ </th><th>Hits against draft genomeÂ </th><th>Hits against barley genomeÂ </th><th>Hits on the same homoeologous chromosomesÂ </th></tr></thead><tbody><tr><td>A genomeÂ </td><td>3,918 (87.8%)Â </td><td>2,856 (64.0%)Â </td><td>1,658 (37.2%)Â </td></tr><tr><td>â€ƒ1AÂ </td><td>464 (84.5%)Â </td><td>379 (69.0%)Â </td><td>265 (48.3%)Â </td></tr><tr><td>â€ƒ2AÂ </td><td>488 (91.6%)Â </td><td>338 (63.4%)Â </td><td>238 (44.7%)Â </td></tr><tr><td>â€ƒ3AÂ </td><td>458 (87.2%)Â </td><td>333 (63.4%)Â </td><td>238 (45.3%)Â </td></tr><tr><td>â€ƒ4AÂ </td><td>851 (85.2%)Â </td><td>608 (60.9%)Â </td><td>85 (8.5%)Â </td></tr><tr><td>â€ƒ5AÂ </td><td>416 (91.4%)Â </td><td>315 (69.2%)Â </td><td>184 (40.4%)Â </td></tr><tr><td>â€ƒ6AÂ </td><td>555 (85.6%)Â </td><td>409 (63.1%)Â </td><td>293 (45.2%)Â </td></tr><tr><td>â€ƒ7AÂ </td><td>686 (91.1%)Â </td><td>474 (62.9%)Â </td><td>355 (47.1%)Â </td></tr><tr><td>B genomeÂ </td><td>â€“Â </td><td>3,972 (62.4%)Â </td><td>1,772 (27.8%)Â </td></tr><tr><td>â€ƒ1BÂ </td><td>â€“Â </td><td>629 (63.4%)Â </td><td>279 (28.1%)Â </td></tr><tr><td>â€ƒ2BÂ </td><td>â€“Â </td><td>793 (64.4%)Â </td><td>389 (31.6%)Â </td></tr><tr><td>â€ƒ3BÂ </td><td>â€“Â </td><td>568 (61.3%)Â </td><td>249 (26.9%)Â </td></tr><tr><td>â€ƒ4BÂ </td><td>â€“Â </td><td>290 (58.7%)Â </td><td>150 (30.4%)Â </td></tr><tr><td>â€ƒ5BÂ </td><td>â€“Â </td><td>588 (60.9%)Â </td><td>268 (27.8%)Â </td></tr><tr><td>â€ƒ6BÂ </td><td>â€“Â </td><td>573 (61.9%)Â </td><td>207 (22.4%)Â </td></tr><tr><td>â€ƒ7BÂ </td><td>â€“Â </td><td>531 (63.8%)Â </td><td>230 (27.6%)Â </td></tr><tr><td>D genomeÂ </td><td>1,319 (59.2%)Â </td><td>1,006 (45.2%)Â </td><td>923 (41.4%)Â </td></tr><tr><td>â€ƒ1DÂ </td><td>216 (64.7%)Â </td><td>150 (44.9%)Â </td><td>110 (32.9%)Â </td></tr><tr><td>â€ƒ2DÂ </td><td>514 (61.6%)Â </td><td>436 (52.3%)Â </td><td>417 (50.0%)Â </td></tr><tr><td>â€ƒ3DÂ </td><td>230 (59.3%)Â </td><td>181 (46.6%)Â </td><td>176 (45.4%)Â </td></tr><tr><td>â€ƒ4DÂ </td><td>30 (41.7%)Â </td><td>23 (31.9%)Â </td><td>23 (31.9%)Â </td></tr><tr><td>â€ƒ5DÂ </td><td>131 (52.6%)Â </td><td>90 (36.1%)Â </td><td>89 (35.7%)Â </td></tr><tr><td>â€ƒ6DÂ </td><td>90 (50.3%)Â </td><td>68 (38.0%)Â </td><td>64 (35.8%)Â </td></tr><tr><td>â€ƒ7DÂ </td><td>108 (62.8%)Â </td><td>58 (33.7%)Â </td><td>44 (25.6%)Â </td></tr><tr><td>TotalÂ </td><td>5,237 (78.3%)Â </td><td>7,834 (60.0%)Â </td><td>4,353 (33.3%)Â </td></tr></tbody></table></div><div content-id="DSU020TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Mapped sequencesÂ </th><th>Hits against draft genomeÂ </th><th>Hits against barley genomeÂ </th><th>Hits on the same homoeologous chromosomesÂ </th></tr></thead><tbody><tr><td>A genomeÂ </td><td>3,918 (87.8%)Â </td><td>2,856 (64.0%)Â </td><td>1,658 (37.2%)Â </td></tr><tr><td>â€ƒ1AÂ </td><td>464 (84.5%)Â </td><td>379 (69.0%)Â </td><td>265 (48.3%)Â </td></tr><tr><td>â€ƒ2AÂ </td><td>488 (91.6%)Â </td><td>338 (63.4%)Â </td><td>238 (44.7%)Â </td></tr><tr><td>â€ƒ3AÂ </td><td>458 (87.2%)Â </td><td>333 (63.4%)Â </td><td>238 (45.3%)Â </td></tr><tr><td>â€ƒ4AÂ </td><td>851 (85.2%)Â </td><td>608 (60.9%)Â </td><td>85 (8.5%)Â </td></tr><tr><td>â€ƒ5AÂ </td><td>416 (91.4%)Â </td><td>315 (69.2%)Â </td><td>184 (40.4%)Â </td></tr><tr><td>â€ƒ6AÂ </td><td>555 (85.6%)Â </td><td>409 (63.1%)Â </td><td>293 (45.2%)Â </td></tr><tr><td>â€ƒ7AÂ </td><td>686 (91.1%)Â </td><td>474 (62.9%)Â </td><td>355 (47.1%)Â </td></tr><tr><td>B genomeÂ </td><td>â€“Â </td><td>3,972 (62.4%)Â </td><td>1,772 (27.8%)Â </td></tr><tr><td>â€ƒ1BÂ </td><td>â€“Â </td><td>629 (63.4%)Â </td><td>279 (28.1%)Â </td></tr><tr><td>â€ƒ2BÂ </td><td>â€“Â </td><td>793 (64.4%)Â </td><td>389 (31.6%)Â </td></tr><tr><td>â€ƒ3BÂ </td><td>â€“Â </td><td>568 (61.3%)Â </td><td>249 (26.9%)Â </td></tr><tr><td>â€ƒ4BÂ </td><td>â€“Â </td><td>290 (58.7%)Â </td><td>150 (30.4%)Â </td></tr><tr><td>â€ƒ5BÂ </td><td>â€“Â </td><td>588 (60.9%)Â </td><td>268 (27.8%)Â </td></tr><tr><td>â€ƒ6BÂ </td><td>â€“Â </td><td>573 (61.9%)Â </td><td>207 (22.4%)Â </td></tr><tr><td>â€ƒ7BÂ </td><td>â€“Â </td><td>531 (63.8%)Â </td><td>230 (27.6%)Â </td></tr><tr><td>D genomeÂ </td><td>1,319 (59.2%)Â </td><td>1,006 (45.2%)Â </td><td>923 (41.4%)Â </td></tr><tr><td>â€ƒ1DÂ </td><td>216 (64.7%)Â </td><td>150 (44.9%)Â </td><td>110 (32.9%)Â </td></tr><tr><td>â€ƒ2DÂ </td><td>514 (61.6%)Â </td><td>436 (52.3%)Â </td><td>417 (50.0%)Â </td></tr><tr><td>â€ƒ3DÂ </td><td>230 (59.3%)Â </td><td>181 (46.6%)Â </td><td>176 (45.4%)Â </td></tr><tr><td>â€ƒ4DÂ </td><td>30 (41.7%)Â </td><td>23 (31.9%)Â </td><td>23 (31.9%)Â </td></tr><tr><td>â€ƒ5DÂ </td><td>131 (52.6%)Â </td><td>90 (36.1%)Â </td><td>89 (35.7%)Â </td></tr><tr><td>â€ƒ6DÂ </td><td>90 (50.3%)Â </td><td>68 (38.0%)Â </td><td>64 (35.8%)Â </td></tr><tr><td>â€ƒ7DÂ </td><td>108 (62.8%)Â </td><td>58 (33.7%)Â </td><td>44 (25.6%)Â </td></tr><tr><td>TotalÂ </td><td>5,237 (78.3%)Â </td><td>7,834 (60.0%)Â </td><td>4,353 (33.3%)Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p>Sequences mapped on A- and D-genomes were first blast searched against <em>T. urartu</em> and <em>Ae. tauschii</em> genome sequences and then the obtained scaffolds were assigned to the barley physical map based on their genes. B-genome sequences were directly aligned to the barley genome.</p></div></div></div></div></div><div class="table-wrap"><div id="DSU020TB3" data-id="DSU020TB3" class="table-wrap-title"><div class="label -label">TableÂ 3.</div><div class="caption"><p>Summary of blastn results against genome sequences</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Mapped sequencesÂ </th><th>Hits against draft genomeÂ </th><th>Hits against barley genomeÂ </th><th>Hits on the same homoeologous chromosomesÂ </th></tr></thead><tbody><tr><td>A genomeÂ </td><td>3,918 (87.8%)Â </td><td>2,856 (64.0%)Â </td><td>1,658 (37.2%)Â </td></tr><tr><td>â€ƒ1AÂ </td><td>464 (84.5%)Â </td><td>379 (69.0%)Â </td><td>265 (48.3%)Â </td></tr><tr><td>â€ƒ2AÂ </td><td>488 (91.6%)Â </td><td>338 (63.4%)Â </td><td>238 (44.7%)Â </td></tr><tr><td>â€ƒ3AÂ </td><td>458 (87.2%)Â </td><td>333 (63.4%)Â </td><td>238 (45.3%)Â </td></tr><tr><td>â€ƒ4AÂ </td><td>851 (85.2%)Â </td><td>608 (60.9%)Â </td><td>85 (8.5%)Â </td></tr><tr><td>â€ƒ5AÂ </td><td>416 (91.4%)Â </td><td>315 (69.2%)Â </td><td>184 (40.4%)Â </td></tr><tr><td>â€ƒ6AÂ </td><td>555 (85.6%)Â </td><td>409 (63.1%)Â </td><td>293 (45.2%)Â </td></tr><tr><td>â€ƒ7AÂ </td><td>686 (91.1%)Â </td><td>474 (62.9%)Â </td><td>355 (47.1%)Â </td></tr><tr><td>B genomeÂ </td><td>â€“Â </td><td>3,972 (62.4%)Â </td><td>1,772 (27.8%)Â </td></tr><tr><td>â€ƒ1BÂ </td><td>â€“Â </td><td>629 (63.4%)Â </td><td>279 (28.1%)Â </td></tr><tr><td>â€ƒ2BÂ </td><td>â€“Â </td><td>793 (64.4%)Â </td><td>389 (31.6%)Â </td></tr><tr><td>â€ƒ3BÂ </td><td>â€“Â </td><td>568 (61.3%)Â </td><td>249 (26.9%)Â </td></tr><tr><td>â€ƒ4BÂ </td><td>â€“Â </td><td>290 (58.7%)Â </td><td>150 (30.4%)Â </td></tr><tr><td>â€ƒ5BÂ </td><td>â€“Â </td><td>588 (60.9%)Â </td><td>268 (27.8%)Â </td></tr><tr><td>â€ƒ6BÂ </td><td>â€“Â </td><td>573 (61.9%)Â </td><td>207 (22.4%)Â </td></tr><tr><td>â€ƒ7BÂ </td><td>â€“Â </td><td>531 (63.8%)Â </td><td>230 (27.6%)Â </td></tr><tr><td>D genomeÂ </td><td>1,319 (59.2%)Â </td><td>1,006 (45.2%)Â </td><td>923 (41.4%)Â </td></tr><tr><td>â€ƒ1DÂ </td><td>216 (64.7%)Â </td><td>150 (44.9%)Â </td><td>110 (32.9%)Â </td></tr><tr><td>â€ƒ2DÂ </td><td>514 (61.6%)Â </td><td>436 (52.3%)Â </td><td>417 (50.0%)Â </td></tr><tr><td>â€ƒ3DÂ </td><td>230 (59.3%)Â </td><td>181 (46.6%)Â </td><td>176 (45.4%)Â </td></tr><tr><td>â€ƒ4DÂ </td><td>30 (41.7%)Â </td><td>23 (31.9%)Â </td><td>23 (31.9%)Â </td></tr><tr><td>â€ƒ5DÂ </td><td>131 (52.6%)Â </td><td>90 (36.1%)Â </td><td>89 (35.7%)Â </td></tr><tr><td>â€ƒ6DÂ </td><td>90 (50.3%)Â </td><td>68 (38.0%)Â </td><td>64 (35.8%)Â </td></tr><tr><td>â€ƒ7DÂ </td><td>108 (62.8%)Â </td><td>58 (33.7%)Â </td><td>44 (25.6%)Â </td></tr><tr><td>TotalÂ </td><td>5,237 (78.3%)Â </td><td>7,834 (60.0%)Â </td><td>4,353 (33.3%)Â </td></tr></tbody></table></div><div content-id="DSU020TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Mapped sequencesÂ </th><th>Hits against draft genomeÂ </th><th>Hits against barley genomeÂ </th><th>Hits on the same homoeologous chromosomesÂ </th></tr></thead><tbody><tr><td>A genomeÂ </td><td>3,918 (87.8%)Â </td><td>2,856 (64.0%)Â </td><td>1,658 (37.2%)Â </td></tr><tr><td>â€ƒ1AÂ </td><td>464 (84.5%)Â </td><td>379 (69.0%)Â </td><td>265 (48.3%)Â </td></tr><tr><td>â€ƒ2AÂ </td><td>488 (91.6%)Â </td><td>338 (63.4%)Â </td><td>238 (44.7%)Â </td></tr><tr><td>â€ƒ3AÂ </td><td>458 (87.2%)Â </td><td>333 (63.4%)Â </td><td>238 (45.3%)Â </td></tr><tr><td>â€ƒ4AÂ </td><td>851 (85.2%)Â </td><td>608 (60.9%)Â </td><td>85 (8.5%)Â </td></tr><tr><td>â€ƒ5AÂ </td><td>416 (91.4%)Â </td><td>315 (69.2%)Â </td><td>184 (40.4%)Â </td></tr><tr><td>â€ƒ6AÂ </td><td>555 (85.6%)Â </td><td>409 (63.1%)Â </td><td>293 (45.2%)Â </td></tr><tr><td>â€ƒ7AÂ </td><td>686 (91.1%)Â </td><td>474 (62.9%)Â </td><td>355 (47.1%)Â </td></tr><tr><td>B genomeÂ </td><td>â€“Â </td><td>3,972 (62.4%)Â </td><td>1,772 (27.8%)Â </td></tr><tr><td>â€ƒ1BÂ </td><td>â€“Â </td><td>629 (63.4%)Â </td><td>279 (28.1%)Â </td></tr><tr><td>â€ƒ2BÂ </td><td>â€“Â </td><td>793 (64.4%)Â </td><td>389 (31.6%)Â </td></tr><tr><td>â€ƒ3BÂ </td><td>â€“Â </td><td>568 (61.3%)Â </td><td>249 (26.9%)Â </td></tr><tr><td>â€ƒ4BÂ </td><td>â€“Â </td><td>290 (58.7%)Â </td><td>150 (30.4%)Â </td></tr><tr><td>â€ƒ5BÂ </td><td>â€“Â </td><td>588 (60.9%)Â </td><td>268 (27.8%)Â </td></tr><tr><td>â€ƒ6BÂ </td><td>â€“Â </td><td>573 (61.9%)Â </td><td>207 (22.4%)Â </td></tr><tr><td>â€ƒ7BÂ </td><td>â€“Â </td><td>531 (63.8%)Â </td><td>230 (27.6%)Â </td></tr><tr><td>D genomeÂ </td><td>1,319 (59.2%)Â </td><td>1,006 (45.2%)Â </td><td>923 (41.4%)Â </td></tr><tr><td>â€ƒ1DÂ </td><td>216 (64.7%)Â </td><td>150 (44.9%)Â </td><td>110 (32.9%)Â </td></tr><tr><td>â€ƒ2DÂ </td><td>514 (61.6%)Â </td><td>436 (52.3%)Â </td><td>417 (50.0%)Â </td></tr><tr><td>â€ƒ3DÂ </td><td>230 (59.3%)Â </td><td>181 (46.6%)Â </td><td>176 (45.4%)Â </td></tr><tr><td>â€ƒ4DÂ </td><td>30 (41.7%)Â </td><td>23 (31.9%)Â </td><td>23 (31.9%)Â </td></tr><tr><td>â€ƒ5DÂ </td><td>131 (52.6%)Â </td><td>90 (36.1%)Â </td><td>89 (35.7%)Â </td></tr><tr><td>â€ƒ6DÂ </td><td>90 (50.3%)Â </td><td>68 (38.0%)Â </td><td>64 (35.8%)Â </td></tr><tr><td>â€ƒ7DÂ </td><td>108 (62.8%)Â </td><td>58 (33.7%)Â </td><td>44 (25.6%)Â </td></tr><tr><td>TotalÂ </td><td>5,237 (78.3%)Â </td><td>7,834 (60.0%)Â </td><td>4,353 (33.3%)Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p>Sequences mapped on A- and D-genomes were first blast searched against <em>T. urartu</em> and <em>Ae. tauschii</em> genome sequences and then the obtained scaffolds were assigned to the barley physical map based on their genes. B-genome sequences were directly aligned to the barley genome.</p></div></div></div></div><div data-id="DSU020F4" class="fig fig-section"><div class="label fig-label">FigureÂ 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/m_dsu02004.jpeg?Expires=2147483647&amp;Signature=ImQVs~JuCxWuaa3XoRg4kOMJzPJFr8u8dBTclcglcmdW2g88f9SVhhUvWkuyn14mFfc5hlr9GmsprQNxeiqtFhu5HXs~sj5R-ugUPFerS5wwtdavh5U7IpWlpQQM0k723q2mHkdhUL2QeUxu0BVd15jAxQ48xIhapn~d~w1J5hRx8eY8C4QXRtbzydoC8JCRSFO086~~J-MatRmpXZ943MUAsOnNZ1JSFJO7FUKDPkqUy9vkU5CI3PkcxOZodgnuC6Pc5LC4VQJ3SvolJs8OzfgxeWm1KzdbM8xldNgrOzUfT4dIqG6QyKw1vGbRVdfDuhhZOh0dpZ3Qk-oejULXtw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relationship between the wheat genetic and barley physical maps. Based on blastn search sequences anchored to A- (left column), B- (centre), and D-genomes (right column) were aligned to the barley genome. Only the sequences mapped on the same homoeologous group are presented in each row. The y-axis indicates the position on the physical map of barley in Mb and the x-axis the position on the genetic map of wheat in cM." path-from-xml="dsu02004.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199072/dsu02004.jpeg" path-from-xml="dsu02004.jpeg" target="_blank">View large</a><a section="87199072" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu02004.jpeg?Expires=2147483647&amp;Signature=ZqAU1VynJEJvfPikr9AOenHoyKkCPU3~yfbTzznVStkfMYgiBnDCINZ5mZqCH62Z5XGmIzUobl4PXn3j5lDOQT5yNHKioe9LJlZV0DUzDEYQnmU3d5JoTquMAMpEY31b97KiwTxV6fS-q6-XIYnCYoVF8JBdMWCuFdX5jXEtrR4UOUo6tOWhK-Y3IwaKnz20KCekAZpiVjcLVIu0ofHc6~Dn0QwVi2EhXj67ldLws9~MfCd6DW2KvhnveVEucsFgHAxNvk0D9J3HXorDqNBDK-OJgPJuCM9LLpSGSQ10nZCIISoYTNWXAo7n22il6Ewu7B7a-cSataQRLZ63cRdTYA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199072&amp;ar=2754552&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02004.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relationship between the wheat genetic and barley physical maps. Based on blastn search sequences anchored to A- (left column), B- (centre), and D-genomes (right column) were aligned to the barley genome. Only the sequences mapped on the same homoeologous group are presented in each row. The <em>y</em>-axis indicates the position on the physical map of barley in Mb and the <em>x</em>-axis the position on the genetic map of wheat in cM.</p></div></div><div content-id="DSU020F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">FigureÂ 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/m_dsu02004.jpeg?Expires=2147483647&amp;Signature=ImQVs~JuCxWuaa3XoRg4kOMJzPJFr8u8dBTclcglcmdW2g88f9SVhhUvWkuyn14mFfc5hlr9GmsprQNxeiqtFhu5HXs~sj5R-ugUPFerS5wwtdavh5U7IpWlpQQM0k723q2mHkdhUL2QeUxu0BVd15jAxQ48xIhapn~d~w1J5hRx8eY8C4QXRtbzydoC8JCRSFO086~~J-MatRmpXZ943MUAsOnNZ1JSFJO7FUKDPkqUy9vkU5CI3PkcxOZodgnuC6Pc5LC4VQJ3SvolJs8OzfgxeWm1KzdbM8xldNgrOzUfT4dIqG6QyKw1vGbRVdfDuhhZOh0dpZ3Qk-oejULXtw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relationship between the wheat genetic and barley physical maps. Based on blastn search sequences anchored to A- (left column), B- (centre), and D-genomes (right column) were aligned to the barley genome. Only the sequences mapped on the same homoeologous group are presented in each row. The y-axis indicates the position on the physical map of barley in Mb and the x-axis the position on the genetic map of wheat in cM." path-from-xml="dsu02004.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87199072/dsu02004.jpeg" path-from-xml="dsu02004.jpeg" target="_blank">View large</a><a section="87199072" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu02004.jpeg?Expires=2147483647&amp;Signature=ZqAU1VynJEJvfPikr9AOenHoyKkCPU3~yfbTzznVStkfMYgiBnDCINZ5mZqCH62Z5XGmIzUobl4PXn3j5lDOQT5yNHKioe9LJlZV0DUzDEYQnmU3d5JoTquMAMpEY31b97KiwTxV6fS-q6-XIYnCYoVF8JBdMWCuFdX5jXEtrR4UOUo6tOWhK-Y3IwaKnz20KCekAZpiVjcLVIu0ofHc6~Dn0QwVi2EhXj67ldLws9~MfCd6DW2KvhnveVEucsFgHAxNvk0D9J3HXorDqNBDK-OJgPJuCM9LLpSGSQ10nZCIISoYTNWXAo7n22il6Ewu7B7a-cSataQRLZ63cRdTYA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87199072&amp;ar=2754552&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu02004.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relationship between the wheat genetic and barley physical maps. Based on blastn search sequences anchored to A- (left column), B- (centre), and D-genomes (right column) were aligned to the barley genome. Only the sequences mapped on the same homoeologous group are presented in each row. The <em>y</em>-axis indicates the position on the physical map of barley in Mb and the <em>x</em>-axis the position on the genetic map of wheat in cM.</p></div></div></div><p>The proportion of D-genome markers with hits against the <em>Ae. tauschii</em> genomic sequence (1,319/2,228), and consequently, those anchored on the barley physical map (1,006/2,228), was much lower than the A-genome markers (TableÂ <a reveal-id="DSU020TB3" data-open="DSU020TB3" class="link link-reveal link-table xref-fig">3</a>). However, a similar percentage (41.4%) was observed for markers mapping to the same homoeologous group chromosomes.</p><p>For B-genome markers, the proportion of hits against the barley genome was similar to that in the A-genome (62.4%), but the number of markers mapping to the same homoeologous group was lower in the B-genome (27.8%, TableÂ <a reveal-id="DSU020TB3" data-open="DSU020TB3" class="link link-reveal link-table xref-fig">3</a>). Although collinearity with the barley genome was observed for many of these markers, the number of non-syntenic markers tended to be higher in the B-genome (Fig.Â <a reveal-id="DSU020F4" data-open="DSU020F4" class="link link-reveal link-table xref-fig">4</a>). In general, synteny with barley chromosomes at the centromeric region was clearly observed only in a few chromosomes such as 1A and 2A.</p>                            <h3 scrollto-destination=87199075 id="87199075" class="section-title" data-legacy-id=s3e>3.5. QTL analysis for 14 morphological and agricultural traits</h3>
<p>To compare the efficiency of the newly constructed map with the original SSR map developed by Kobayashi <em>et al.</em><sup><a reveal-id="DSU020C16" data-open="DSU020C16" class="link link-ref link-reveal xref-bibr">16</a></sup> for analysing complex traits, QTL analysis of 14 agronomically important traits was performed using both maps. Flowering-related traits such as HT, FT, MT, and GFP were evaluated over two growing seasons at Kobe10 and Kobe12 (see Section 2.1.), and at Kyoto12 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). Of the spike-related traits, SpN, SLperSpN (indicative of spike compactness), and T5SpL were evaluated for Kobe12; SL was also evaluated for Kyoto12. Seed-related traits such as SdL, SdW, SdH, and SdLperW were measured for Kobe12. Phenotype data for CL and TN were obtained for Kyoto12. The mean values of the parental lines and RILs are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>. Strong positive correlations were observed among HT, FT, and MT, and between SL and SLperSpN (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). Relatively, strong correlation was also found between T5SpL and SLperSpN, SdH and SdW, and SdLperW and SdW.</p><p>QTL analysis was performed using composite interval mapping, and only the QTLs with peaks above the LOD threshold computed by a 1,000-permutation test were selected. The LOD threshold values were significantly higher (<em>P</em> &lt; 0.001) in the new high-density map, with an average of 4.76 Â± 0.15, than in the original SSR map (3.76 Â± 0.08). In total, 29 QTLs were identified using the SSR map and 32 QTLs using the high-density map, of which 25 were identified in both maps (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>); seven QTLs were identified only in the high-density map and four QTLs only in the SSR map.</p><p>The 1.5-LOD support interval, meaning the interval in which the LOD score was within 1.5 units of its maximum, tended to be smaller in the high-density map (average of 14.62 Â± 18.03 cM) than in the SSR map (22.01 Â± 19.85 cM), but the difference was not significant (<em>P</em> = 0.129) (TableÂ <a reveal-id="DSU020TB4" data-open="DSU020TB4" class="link link-reveal link-table xref-fig">4</a>). Excluding the D-genome QTLs, where the map resolution was still low, the difference became significant (9.45 Â± 3.29 cM for the high-density map and 17.21 Â± 11.29 cM for the SSR map, <em>P</em> &lt; 0.01). The number of markers within the 1.5-LOD intervals was also significantly higher (<em>P</em> &lt; 0.05) in the high-density map (average of 16.40 Â± 22.97) than in the SSR map (average of 4.20 Â± 1.98) (TableÂ <a reveal-id="DSU020TB4" data-open="DSU020TB4" class="link link-reveal link-table xref-fig">4</a>). <div content-id="DSU020TB4" class="table-modal"><div class="table-wrap"><div id="DSU020TB4" data-id="DSU020TB4" class="table-wrap-title"><div class="label -label">TableÂ 4.</div><div class="caption"><p>Comparison of 1.5-LOD support intervals and number of markers within this interval between the newly constructed and original SSR maps</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">QTL<sup>a</sup>Â </th><th rowspan="2">ChrÂ </th><th colspan="2">Newly constructed map<hr />Â </th><th colspan="2">Original SSR map<sup>b</sup><hr />Â </th></tr><tr><th>1.5-LOD interval (cM)Â </th><th>Number of markersÂ </th><th>1.5-LOD interval (cM)Â </th><th>Number of markersÂ </th></tr></thead><tbody><tr><td>HT_Kobe10Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>FT_Kobe10Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>MT_Kobe10Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>GFP_Kobe10Â </td><td>1AÂ </td><td>9.53Â </td><td>14Â </td><td>9.47Â </td><td>4Â </td></tr><tr><td>HT_Kobe12Â </td><td>2BÂ </td><td>9.14Â </td><td>7Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>HT_Kobe12Â </td><td>5DÂ </td><td>73.64Â </td><td>5Â </td><td>76.43Â </td><td>5Â </td></tr><tr><td>FT_Kobe12Â </td><td>2BÂ </td><td>9.14Â </td><td>7Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>FT_Kobe12Â </td><td>5DÂ </td><td>73.64Â </td><td>5Â </td><td>76.43Â </td><td>5Â </td></tr><tr><td>MT_Kobe12Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>HT_Kyoto12Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>FT_Kyoto12Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>SL_Kobe12Â </td><td>3AÂ </td><td>14.68Â </td><td>10Â </td><td>31.42Â </td><td>4Â </td></tr><tr><td>SL_Kobe12Â </td><td>6AÂ </td><td>10.98Â </td><td>23Â </td><td>22.34Â </td><td>7Â </td></tr><tr><td>SL_Kyoto12Â </td><td>4AÂ </td><td>10.99Â </td><td>28Â </td><td>12.81Â </td><td>6Â </td></tr><tr><td>SpN_Kobe12Â </td><td>2AÂ </td><td>14.68Â </td><td>9Â </td><td>5.80Â </td><td>8Â </td></tr><tr><td>SpN_Kobe12Â </td><td>2BÂ </td><td>9.14Â </td><td>7Â </td><td>11.01Â </td><td>2Â </td></tr><tr><td>SLperSpN_Kobe12Â </td><td>4AÂ </td><td>9.80Â </td><td>21Â </td><td>12.81Â </td><td>6Â </td></tr><tr><td>SLperSpN_Kobe12Â </td><td>6AÂ </td><td>9.67Â </td><td>22Â </td><td>12.00Â </td><td>5Â </td></tr><tr><td>T5SpL_Kobe12Â </td><td>2BÂ </td><td>2.31Â </td><td>5Â </td><td>11.01Â </td><td>2Â </td></tr><tr><td>T5SpL_Kobe12Â </td><td>3AÂ </td><td>14.68Â </td><td>10Â </td><td>60.77Â </td><td>5Â </td></tr><tr><td>T5SpL_Kobe12Â </td><td>4AÂ </td><td>10.99Â </td><td>28Â </td><td>5.34Â </td><td>3Â </td></tr><tr><td>SdH_Kobe12Â </td><td>1BÂ </td><td>8.06Â </td><td>45Â </td><td>15.61Â </td><td>10Â </td></tr><tr><td>SdLperW_Kobe12Â </td><td>2DÂ </td><td>10.41Â </td><td>115Â </td><td>41.19Â </td><td>3Â </td></tr><tr><td>CL_Kyoto12Â </td><td>2BÂ </td><td>15.06Â </td><td>10Â </td><td>11.01Â </td><td>2Â </td></tr><tr><td>TN_Kyoto12Â </td><td>1AÂ </td><td>10.03Â </td><td>15Â </td><td>11.74Â </td><td>4Â </td></tr><tr><td>Total averageÂ </td><td>Â </td><td>14.62 Â± 18.03Â </td><td>16.40 Â± 22.97*Â </td><td>22.91 Â± 19.85Â </td><td>4.20 Â± 1.97Â </td></tr><tr><td>Average of AB-genomeÂ </td><td>Â </td><td>9.45 Â± 3.29**Â </td><td>12.95 Â± 10.75***Â </td><td>17.21 Â± 11.29Â </td><td>4.18 Â± 2.08Â </td></tr></tbody></table></div><div content-id="DSU020TB4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">QTL<sup>a</sup>Â </th><th rowspan="2">ChrÂ </th><th colspan="2">Newly constructed map<hr />Â </th><th colspan="2">Original SSR map<sup>b</sup><hr />Â </th></tr><tr><th>1.5-LOD interval (cM)Â </th><th>Number of markersÂ </th><th>1.5-LOD interval (cM)Â </th><th>Number of markersÂ </th></tr></thead><tbody><tr><td>HT_Kobe10Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>FT_Kobe10Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>MT_Kobe10Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>GFP_Kobe10Â </td><td>1AÂ </td><td>9.53Â </td><td>14Â </td><td>9.47Â </td><td>4Â </td></tr><tr><td>HT_Kobe12Â </td><td>2BÂ </td><td>9.14Â </td><td>7Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>HT_Kobe12Â </td><td>5DÂ </td><td>73.64Â </td><td>5Â </td><td>76.43Â </td><td>5Â </td></tr><tr><td>FT_Kobe12Â </td><td>2BÂ </td><td>9.14Â </td><td>7Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>FT_Kobe12Â </td><td>5DÂ </td><td>73.64Â </td><td>5Â </td><td>76.43Â </td><td>5Â </td></tr><tr><td>MT_Kobe12Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>HT_Kyoto12Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>FT_Kyoto12Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>SL_Kobe12Â </td><td>3AÂ </td><td>14.68Â </td><td>10Â </td><td>31.42Â </td><td>4Â </td></tr><tr><td>SL_Kobe12Â </td><td>6AÂ </td><td>10.98Â </td><td>23Â </td><td>22.34Â </td><td>7Â </td></tr><tr><td>SL_Kyoto12Â </td><td>4AÂ </td><td>10.99Â </td><td>28Â </td><td>12.81Â </td><td>6Â </td></tr><tr><td>SpN_Kobe12Â </td><td>2AÂ </td><td>14.68Â </td><td>9Â </td><td>5.80Â </td><td>8Â </td></tr><tr><td>SpN_Kobe12Â </td><td>2BÂ </td><td>9.14Â </td><td>7Â </td><td>11.01Â </td><td>2Â </td></tr><tr><td>SLperSpN_Kobe12Â </td><td>4AÂ </td><td>9.80Â </td><td>21Â </td><td>12.81Â </td><td>6Â </td></tr><tr><td>SLperSpN_Kobe12Â </td><td>6AÂ </td><td>9.67Â </td><td>22Â </td><td>12.00Â </td><td>5Â </td></tr><tr><td>T5SpL_Kobe12Â </td><td>2BÂ </td><td>2.31Â </td><td>5Â </td><td>11.01Â </td><td>2Â </td></tr><tr><td>T5SpL_Kobe12Â </td><td>3AÂ </td><td>14.68Â </td><td>10Â </td><td>60.77Â </td><td>5Â </td></tr><tr><td>T5SpL_Kobe12Â </td><td>4AÂ </td><td>10.99Â </td><td>28Â </td><td>5.34Â </td><td>3Â </td></tr><tr><td>SdH_Kobe12Â </td><td>1BÂ </td><td>8.06Â </td><td>45Â </td><td>15.61Â </td><td>10Â </td></tr><tr><td>SdLperW_Kobe12Â </td><td>2DÂ </td><td>10.41Â </td><td>115Â </td><td>41.19Â </td><td>3Â </td></tr><tr><td>CL_Kyoto12Â </td><td>2BÂ </td><td>15.06Â </td><td>10Â </td><td>11.01Â </td><td>2Â </td></tr><tr><td>TN_Kyoto12Â </td><td>1AÂ </td><td>10.03Â </td><td>15Â </td><td>11.74Â </td><td>4Â </td></tr><tr><td>Total averageÂ </td><td>Â </td><td>14.62 Â± 18.03Â </td><td>16.40 Â± 22.97*Â </td><td>22.91 Â± 19.85Â </td><td>4.20 Â± 1.97Â </td></tr><tr><td>Average of AB-genomeÂ </td><td>Â </td><td>9.45 Â± 3.29**Â </td><td>12.95 Â± 10.75***Â </td><td>17.21 Â± 11.29Â </td><td>4.18 Â± 2.08Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN2"><div class="fn"><p><sup>a</sup>Only the 25 QTLs identified in both maps were included.</p></div></div><div content-id="TN3"><div class="fn"><p><sup>b</sup>Kobayashi <em>et al.</em><sup><a reveal-id="DSU020C16" data-open="DSU020C16" class="link link-ref link-reveal xref-bibr">16</a></sup></p></div></div><div content-id="TN4"><div class="fn"><p>Student's <em>t</em>-test was used to test for statistical significance (*<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01; ***<em>P</em> &lt; 0.001).</p></div></div></div></div></div><div class="table-wrap"><div id="DSU020TB4" data-id="DSU020TB4" class="table-wrap-title"><div class="label -label">TableÂ 4.</div><div class="caption"><p>Comparison of 1.5-LOD support intervals and number of markers within this interval between the newly constructed and original SSR maps</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">QTL<sup>a</sup>Â </th><th rowspan="2">ChrÂ </th><th colspan="2">Newly constructed map<hr />Â </th><th colspan="2">Original SSR map<sup>b</sup><hr />Â </th></tr><tr><th>1.5-LOD interval (cM)Â </th><th>Number of markersÂ </th><th>1.5-LOD interval (cM)Â </th><th>Number of markersÂ </th></tr></thead><tbody><tr><td>HT_Kobe10Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>FT_Kobe10Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>MT_Kobe10Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>GFP_Kobe10Â </td><td>1AÂ </td><td>9.53Â </td><td>14Â </td><td>9.47Â </td><td>4Â </td></tr><tr><td>HT_Kobe12Â </td><td>2BÂ </td><td>9.14Â </td><td>7Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>HT_Kobe12Â </td><td>5DÂ </td><td>73.64Â </td><td>5Â </td><td>76.43Â </td><td>5Â </td></tr><tr><td>FT_Kobe12Â </td><td>2BÂ </td><td>9.14Â </td><td>7Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>FT_Kobe12Â </td><td>5DÂ </td><td>73.64Â </td><td>5Â </td><td>76.43Â </td><td>5Â </td></tr><tr><td>MT_Kobe12Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>HT_Kyoto12Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>FT_Kyoto12Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>SL_Kobe12Â </td><td>3AÂ </td><td>14.68Â </td><td>10Â </td><td>31.42Â </td><td>4Â </td></tr><tr><td>SL_Kobe12Â </td><td>6AÂ </td><td>10.98Â </td><td>23Â </td><td>22.34Â </td><td>7Â </td></tr><tr><td>SL_Kyoto12Â </td><td>4AÂ </td><td>10.99Â </td><td>28Â </td><td>12.81Â </td><td>6Â </td></tr><tr><td>SpN_Kobe12Â </td><td>2AÂ </td><td>14.68Â </td><td>9Â </td><td>5.80Â </td><td>8Â </td></tr><tr><td>SpN_Kobe12Â </td><td>2BÂ </td><td>9.14Â </td><td>7Â </td><td>11.01Â </td><td>2Â </td></tr><tr><td>SLperSpN_Kobe12Â </td><td>4AÂ </td><td>9.80Â </td><td>21Â </td><td>12.81Â </td><td>6Â </td></tr><tr><td>SLperSpN_Kobe12Â </td><td>6AÂ </td><td>9.67Â </td><td>22Â </td><td>12.00Â </td><td>5Â </td></tr><tr><td>T5SpL_Kobe12Â </td><td>2BÂ </td><td>2.31Â </td><td>5Â </td><td>11.01Â </td><td>2Â </td></tr><tr><td>T5SpL_Kobe12Â </td><td>3AÂ </td><td>14.68Â </td><td>10Â </td><td>60.77Â </td><td>5Â </td></tr><tr><td>T5SpL_Kobe12Â </td><td>4AÂ </td><td>10.99Â </td><td>28Â </td><td>5.34Â </td><td>3Â </td></tr><tr><td>SdH_Kobe12Â </td><td>1BÂ </td><td>8.06Â </td><td>45Â </td><td>15.61Â </td><td>10Â </td></tr><tr><td>SdLperW_Kobe12Â </td><td>2DÂ </td><td>10.41Â </td><td>115Â </td><td>41.19Â </td><td>3Â </td></tr><tr><td>CL_Kyoto12Â </td><td>2BÂ </td><td>15.06Â </td><td>10Â </td><td>11.01Â </td><td>2Â </td></tr><tr><td>TN_Kyoto12Â </td><td>1AÂ </td><td>10.03Â </td><td>15Â </td><td>11.74Â </td><td>4Â </td></tr><tr><td>Total averageÂ </td><td>Â </td><td>14.62 Â± 18.03Â </td><td>16.40 Â± 22.97*Â </td><td>22.91 Â± 19.85Â </td><td>4.20 Â± 1.97Â </td></tr><tr><td>Average of AB-genomeÂ </td><td>Â </td><td>9.45 Â± 3.29**Â </td><td>12.95 Â± 10.75***Â </td><td>17.21 Â± 11.29Â </td><td>4.18 Â± 2.08Â </td></tr></tbody></table></div><div content-id="DSU020TB4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">QTL<sup>a</sup>Â </th><th rowspan="2">ChrÂ </th><th colspan="2">Newly constructed map<hr />Â </th><th colspan="2">Original SSR map<sup>b</sup><hr />Â </th></tr><tr><th>1.5-LOD interval (cM)Â </th><th>Number of markersÂ </th><th>1.5-LOD interval (cM)Â </th><th>Number of markersÂ </th></tr></thead><tbody><tr><td>HT_Kobe10Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>FT_Kobe10Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>MT_Kobe10Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>GFP_Kobe10Â </td><td>1AÂ </td><td>9.53Â </td><td>14Â </td><td>9.47Â </td><td>4Â </td></tr><tr><td>HT_Kobe12Â </td><td>2BÂ </td><td>9.14Â </td><td>7Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>HT_Kobe12Â </td><td>5DÂ </td><td>73.64Â </td><td>5Â </td><td>76.43Â </td><td>5Â </td></tr><tr><td>FT_Kobe12Â </td><td>2BÂ </td><td>9.14Â </td><td>7Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>FT_Kobe12Â </td><td>5DÂ </td><td>73.64Â </td><td>5Â </td><td>76.43Â </td><td>5Â </td></tr><tr><td>MT_Kobe12Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>HT_Kyoto12Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>FT_Kyoto12Â </td><td>2BÂ </td><td>6.49Â </td><td>4Â </td><td>18.18Â </td><td>3Â </td></tr><tr><td>SL_Kobe12Â </td><td>3AÂ </td><td>14.68Â </td><td>10Â </td><td>31.42Â </td><td>4Â </td></tr><tr><td>SL_Kobe12Â </td><td>6AÂ </td><td>10.98Â </td><td>23Â </td><td>22.34Â </td><td>7Â </td></tr><tr><td>SL_Kyoto12Â </td><td>4AÂ </td><td>10.99Â </td><td>28Â </td><td>12.81Â </td><td>6Â </td></tr><tr><td>SpN_Kobe12Â </td><td>2AÂ </td><td>14.68Â </td><td>9Â </td><td>5.80Â </td><td>8Â </td></tr><tr><td>SpN_Kobe12Â </td><td>2BÂ </td><td>9.14Â </td><td>7Â </td><td>11.01Â </td><td>2Â </td></tr><tr><td>SLperSpN_Kobe12Â </td><td>4AÂ </td><td>9.80Â </td><td>21Â </td><td>12.81Â </td><td>6Â </td></tr><tr><td>SLperSpN_Kobe12Â </td><td>6AÂ </td><td>9.67Â </td><td>22Â </td><td>12.00Â </td><td>5Â </td></tr><tr><td>T5SpL_Kobe12Â </td><td>2BÂ </td><td>2.31Â </td><td>5Â </td><td>11.01Â </td><td>2Â </td></tr><tr><td>T5SpL_Kobe12Â </td><td>3AÂ </td><td>14.68Â </td><td>10Â </td><td>60.77Â </td><td>5Â </td></tr><tr><td>T5SpL_Kobe12Â </td><td>4AÂ </td><td>10.99Â </td><td>28Â </td><td>5.34Â </td><td>3Â </td></tr><tr><td>SdH_Kobe12Â </td><td>1BÂ </td><td>8.06Â </td><td>45Â </td><td>15.61Â </td><td>10Â </td></tr><tr><td>SdLperW_Kobe12Â </td><td>2DÂ </td><td>10.41Â </td><td>115Â </td><td>41.19Â </td><td>3Â </td></tr><tr><td>CL_Kyoto12Â </td><td>2BÂ </td><td>15.06Â </td><td>10Â </td><td>11.01Â </td><td>2Â </td></tr><tr><td>TN_Kyoto12Â </td><td>1AÂ </td><td>10.03Â </td><td>15Â </td><td>11.74Â </td><td>4Â </td></tr><tr><td>Total averageÂ </td><td>Â </td><td>14.62 Â± 18.03Â </td><td>16.40 Â± 22.97*Â </td><td>22.91 Â± 19.85Â </td><td>4.20 Â± 1.97Â </td></tr><tr><td>Average of AB-genomeÂ </td><td>Â </td><td>9.45 Â± 3.29**Â </td><td>12.95 Â± 10.75***Â </td><td>17.21 Â± 11.29Â </td><td>4.18 Â± 2.08Â </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN2"><div class="fn"><p><sup>a</sup>Only the 25 QTLs identified in both maps were included.</p></div></div><div content-id="TN3"><div class="fn"><p><sup>b</sup>Kobayashi <em>et al.</em><sup><a reveal-id="DSU020C16" data-open="DSU020C16" class="link link-ref link-reveal xref-bibr">16</a></sup></p></div></div><div content-id="TN4"><div class="fn"><p>Student's <em>t</em>-test was used to test for statistical significance (*<em>P</em> &lt; 0.05; **<em>P</em> &lt; 0.01; ***<em>P</em> &lt; 0.001).</p></div></div></div></div></p><p>Most of the identified QTLs were located in a similar chromosomal region to those previously reported (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). Of the flowering-related traits, the early flowering effect of the CS <em>Ppd-B1</em> allele was observed in all the datasets and explained &gt;30% of the PV. The early flowering effect of CS at 5B and 5D QTL for HT and FT was observed only in one environment (Kyoto12 and Kobe12, respectively), indicating a possible genotype Ã— environment interaction.</p><p>Of the spike-related traits, QTLs for SL were identified on chromosomes 3A, 4A, and 6A, and explained âˆ¼10% of the total PV. The CS allele increased SL at the 3A QTL, but decreased it at the 4A and 6A QTLs. The 3A QTL also exhibited an effect on T5SpL (explaining 7.77% of PV) and SLperSpN (9.82% of PV). Similarly, the 4A QTL also affected T5SpL (15.39% of PV) and SLperSpN (10.39% of PV) and might explain the positive correlation between these traits (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). The 6A QTL had an effect on SLperSpN explaining 15.01% of the PV. QTLs for SpN were identified on homoeologous group 2 chromosomes, of which 2B QTL (linked to the <em>Ppd-B1</em> locus) exhibited a major effect and explained 22.16% of the PV.</p><p>A QTL for SdL was detected on chromosome 3A (6.25% of PV), and another was detected for SdH on chromosome 1B (9.60% PV). QTLs for SdW and SdLperW were detected on the long arm of chromosome 2D, and explained, respectively, 5.94 and 8.78% of the total PV. This QTL also might explain the significant correlation between SdW and SdLperW (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>).</p>                            <h2 scrollto-destination=87199082 id="87199082" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>The construction of high-resolution genetic maps in common wheat has required extensive effort due to its large genome and allohexaploid nature. More than 60% of wheat and barley genomes are composed of REs, with retroelements found at least four times more frequently than DNA transposons.<sup><a reveal-id="DSU020C25" data-open="DSU020C25" class="link link-ref link-reveal xref-bibr">25</a>â€“<a reveal-id="DSU020C27" data-open="DSU020C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DSU020C33" data-open="DSU020C33" class="link link-ref link-reveal xref-bibr">33</a></sup> In this study, a microarray-based polymorphism detection system was developed using NGS data obtained from complexity-reduced genomic DNA samples of CS and M808. Although the reduction in complexity using a combination of two restriction enzymes, <em>Pst</em>I and <em>Bst</em>NI, did not affect the composition of REs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>), most of the array-based markers used for construction of the genetic map were derived from genic regions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). In addition, retroelements were under-represented in the array-based markers, indicating that the selection of useful polymorphic probes decreased the number of sequences derived from highly repetitive regions.</p><p>SNP typing of the entire mapping population using NGS technology has also been applied in polyploid wheat species.<sup><a reveal-id="DSU020C11" data-open="DSU020C11" class="link link-ref link-reveal xref-bibr">11</a>â€“<a reveal-id="DSU020C13" data-open="DSU020C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Although the GBS approach is cost-effective and fast, it also often leads to a significant amount of missing data due to the low coverage of sequencing.<sup><a reveal-id="DSU020C15" data-open="DSU020C15" class="link link-ref link-reveal xref-bibr">15</a></sup> These missing data points have mostly been replaced with estimated genotypes through statistical imputation.<sup><a reveal-id="DSU020C15" data-open="DSU020C15" class="link link-ref link-reveal xref-bibr">15</a></sup> An advantage of the array-based genotyping approach in wheat is the low generation of missing data, in contrast to the GBS approach. In 9 of the 210 RILs (F<sub>7</sub> generation) of CS and M808, the number of missing data points was higher than the mean + 2 SD, and they were thus excluded. These missing data points can be explained in part by the presence of heterozygous regions, because data of ambiguous intensity were converted into missing values. In the remaining 201 RILs, no missing data were observed for most of the markers (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>), indicating a high yield of data from this array-based genotyping system.</p><p>The high-density genetic map was composed of 13,480 SSR marker and WABMs, and the markers were distributed across the allohexaploid wheat genome, especially in the A- and B-genomes (Fig.Â <a reveal-id="DSU020F2" data-open="DSU020F2" class="link link-reveal link-table xref-fig">2</a>). The mapped marker number was similar to that constructed using the GBS approach.<sup><a reveal-id="DSU020C11" data-open="DSU020C11" class="link link-ref link-reveal xref-bibr">11</a></sup> As reported in previous studies, the number of markers in the D-genome was much lower than in the A- and B-genomes of common wheat (TableÂ <a reveal-id="DSU020TB1" data-open="DSU020TB1" class="link link-reveal link-table xref-fig">1</a>).<sup><a reveal-id="DSU020C8" data-open="DSU020C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSU020C10" data-open="DSU020C10" class="link link-ref link-reveal xref-bibr">10</a></sup> The reads and contigs anchored to the high-density map were also mapped <em>in silico</em> against the physical map of barley, because common wheat lacks any physical map. The draft genome sequences of <em>T. urartu</em><sup><a reveal-id="DSU020C26" data-open="DSU020C26" class="link link-ref link-reveal xref-bibr">26</a></sup> and <em>Ae. tauschii</em><sup><a reveal-id="DSU020C25" data-open="DSU020C25" class="link link-ref link-reveal xref-bibr">25</a></sup> were used for the anchoring of A- and D-genome markers, whereas B-genome markers were directly mapped to the barley genome. The use of <em>T. urartu</em> and <em>Ae. tauschii</em> genome information greatly increased the number of markers aligned on the same homoeologous group chromosomes of barley compared with the B-genome markers (TableÂ <a reveal-id="DSU020TB3" data-open="DSU020TB3" class="link link-reveal link-table xref-fig">3</a>). Because of the lower number of D-genome markers, collinearity with barley chromosomes was difficult to observe (Fig.Â <a reveal-id="DSU020F4" data-open="DSU020F4" class="link link-reveal link-table xref-fig">4</a>). In most cases, collinearity was not observed at the centromeric region. This might be due to the lower gene density and higher number of repetitive sequences present in the proximal region of barley and wheat chromosomes.<sup><a reveal-id="DSU020C27" data-open="DSU020C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DSU020C28" data-open="DSU020C28" class="link link-ref link-reveal xref-bibr">28</a></sup></p><p>In the high-density map, segregation distortion was observed in six chromosomal regions, with the long arm of chromosome 6B showing the greatest distortion (Fig.Â <a reveal-id="DSU020F2" data-open="DSU020F2" class="link link-reveal link-table xref-fig">2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). This might have been caused by the presence of the pollen killer gene, <em>Ki</em>, on the long arm of chromosome 6B of CS.<sup><a reveal-id="DSU020C34" data-open="DSU020C34" class="link link-ref link-reveal xref-bibr">34</a>,<a reveal-id="DSU020C35" data-open="DSU020C35" class="link link-ref link-reveal xref-bibr">35</a></sup><em>Ki</em> would affect marker segregation in a large chromosomal region of 6BL, though its position on 6BL has not been accurately determined yet.<sup><a reveal-id="DSU020C35" data-open="DSU020C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Distortions on chromosomes 2B and 5B are due to an epistatic interaction of hybrid necrosis alleles, <em>Ne1<sup>w</sup></em> on the long arm of chromosome 5B of CS, and <em>Ne2</em> on the short arm of chromosome 2B of M808.<sup><a reveal-id="DSU020C35" data-open="DSU020C35" class="link link-ref link-reveal xref-bibr">35</a></sup><em>Ne1</em> and <em>Ne2</em> have been roughly mapped to 5BL and 2BS, respectively.<sup><a reveal-id="DSU020C36" data-open="DSU020C36" class="link link-ref link-reveal xref-bibr">36</a></sup> The distorted regions of 2BS and 5BL were more limited around the causal genes than on 6BL. The three other segregation distortions on chromosomes 2A, 5A, and 7B were not observed in our previous study using the original SSR map of CS and M808.<sup><a reveal-id="DSU020C35" data-open="DSU020C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Thus, the increased map resolution should clarify the presence of additional regions showing segregation distortion. However, the causal genes for the segregation distortion on chromosomes 2A, 5A, and 7B are unknown. In general, segregation distortion is considered to occur due to the presence of a reproductive barrier gene,<sup><a reveal-id="DSU020C37" data-open="DSU020C37" class="link link-ref link-reveal xref-bibr">37</a>â€“<a reveal-id="DSU020C39" data-open="DSU020C39" class="link link-ref link-reveal xref-bibr">39</a></sup> suggesting that reproductive barrier genes might be located in these three chromosomal regions.</p><p>QTL analysis using the high-density and SSR maps identified a similar number of QTLs for both maps (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). Although most of the QTLs were commonly identified in the two maps, a higher number of QTLs was identified in the high-density map. In the high-density map, the LOD threshold values computed by a 1,000-permutation test were significantly higher, as reported in rice.<sup><a reveal-id="DSU020C40" data-open="DSU020C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Thus, the four QTLs identified only in the SSR map might be masked in the high-density map due to these higher threshold values. The increased map resolution resulted in identification of QTLs in a narrower chromosomal region and in a more accurate position than for the SSR map (TableÂ <a reveal-id="DSU020TB4" data-open="DSU020TB4" class="link link-reveal link-table xref-fig">4</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>).</p><p>In the high-density map, the QTL associated with flowering-related traits corresponded well to the <em>Ppd-B1</em> locus on chromosome 2B and the <em>VRN-D1</em> locus (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). The early flowering phenotype of CS (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>) can be explained mainly by the effect of the photoperiod-insensitive allele of <em>Ppd-B1</em> (probably due to its higher copy number)<sup><a reveal-id="DSU020C41" data-open="DSU020C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DSU020C42" data-open="DSU020C42" class="link link-ref link-reveal xref-bibr">42</a></sup> and dominant spring allele of <em>VRN-D1</em>.<sup><a reveal-id="DSU020C43" data-open="DSU020C43" class="link link-ref link-reveal xref-bibr">43</a></sup> The spike compactness (SL/SpN) and short SL of CS can be explained by the 6A and 4A QTLs. The 4A QTL also exhibited an effect on T5SpL, which might contribute to the square-head like spike morphology of CS. On the other hand, the CS allele at the 3A QTL contributed to a less compact spike, longer SL, and longer T5SpL. Grain size and shape are associated with milling quality and yield in wheat. Large and spherical grains have been predicted to be the optimum grain morphology to increase milling yield.<sup><a reveal-id="DSU020C44" data-open="DSU020C44" class="link link-ref link-reveal xref-bibr">44</a></sup> The CS allele at the 2D QTL, with an increasing effect on SdW and reduced SdLperW, and the M808 allele at the 1B QTL (increased SdH) contributed in seed roundness. Favourable alleles for increasing yield were also found in CS at the 2B QTL for CL (reduced plant height) and at the 1A QTL for TN (increased number of tillers with ears). Therefore, the results of QTL analyses indicate that the high-density map efficiently and precisely functions to identify QTLs for morphological and agricultural traits.</p><p>Because of the difference in geographical origin and growth habit of the common wheat cultivars CS (spring habit Chinese variety) and M808 (winter habit Ukrainian variety) used for the development of our array-based system, it is expected to work better between spring and winter wheat and/or Chinese/Asian and European variety. Further study is needed to test the usefulness of this genotyping system using a large wheat population set.</p><p>A number of tightly linked WABMs were found in most of the QTLs, which could facilitate marker-assisted selection or map-based cloning of genes. For this application, the marker of interest should be converted into a PCR marker. Our results demonstrated that this array-based polymorphism detection system can yield high-quality genotype data useful for high-density genetic map construction and QTL analysis of agronomically important traits.</p>                            <h2 scrollto-destination=87199091 id="87199091" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by a grant from the Ministry of Education, Culture, Sports, Science and Technology (MEXT) of Japan [Grant-in-Aid for Scientific Research (B) no. 25292008] to S.T., by cooperative research funds from TOYOTA Motor Co. Ltd., and by Special Coordination Funds for Promoting Science and Technology, Creation of Innovation Centers for Advanced Interdisciplinary Research Areas (Innovative BioProduction Kobe), MEXT, Japan. Funding to pay the Open Access publication charges for this article was provided by a grant from the Ministry of Education, Culture, Sports, Science and Technology of Japan [Grant-in-Aid for Scientific Research (B) No. 25292008] to S.T.</p>                            <h2 scrollto-destination=87199093 id="87199093" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSU020C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kihara</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="article-title">Discovery of the DD-analyser, one of the ancestors of <em>Triticum vulgare</em></div>
          <div class="source">Agric. Hortic.</div>Â 
          <div class="year">1944</div>
          <div class="volume">19</div>
          <div class="fpage">889</div>
          <div class="lpage">90</div>
          <div class="comment">[in Japanese]</div>
        </div></div></div></div><div content-id="DSU020C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">McFadden</div>
               <div class="given-names">E.S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sears</div>
               <div class="given-names">E.R.</div>
            </div>
          </div>
          <div class="article-title">The artificial synthesis of <em>Triticum spelta</em></div>
          <div class="source">Rec. Genet. Soc. Am.</div>Â 
          <div class="year">1944</div>
          <div class="volume">13</div>
          <div class="fpage">26</div>
          <div class="lpage">7</div>
        </div></div></div></div><div content-id="DSU020C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Akbari</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wenzl</div>
               <div class="given-names">P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Caig</div>
               <div class="given-names">V.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Diversity arrays technology (DArT) for high-throughput profiling of the hexaploid wheat genome</div>
          <div class="source">Theor. Appl. Genet.</div>Â 
          <div class="year">2006</div>
          <div class="volume">113</div>
          <div class="fpage">1409</div>
          <div class="lpage">20</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20arrays%20technology%20(DArT)%20for%20high-throughput%20profiling%20of%20the%20hexaploid%20wheat%20genome&amp;author=M.%20Akbari&amp;author=P.%20Wenzl&amp;author=V.%20Caig&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=113&amp;pages=1409-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0365-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0365-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0365-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17033786" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wenzl</div>
               <div class="given-names">P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Carling</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kudrna</div>
               <div class="given-names">D.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Diversity arrays technology (DArT) for whole-genome profiling of barley</div>
          <div class="source">Proc. Natl. Acad. Sci. USA</div>Â 
          <div class="year">2004</div>
          <div class="volume">101</div>
          <div class="fpage">9915</div>
          <div class="lpage">20</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20arrays%20technology%20(DArT)%20for%20whole-genome%20profiling%20of%20barley&amp;author=P.%20Wenzl&amp;author=J.%20Carling&amp;author=D.%20Kudrna&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=9915-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0401076101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0401076101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0401076101"> </span></div>Â </div></div></div></div></div><div content-id="DSU020C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sorrells</div>
               <div class="given-names">M.E.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gustafson</div>
               <div class="given-names">J.P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Somers</div>
               <div class="given-names">D.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Reconstruction of the Synthetic W7984 Ã— Opata M85 wheat reference population</div>
          <div class="source">Genome</div>Â 
          <div class="year">2011</div>
          <div class="volume">54</div>
          <div class="fpage">875</div>
          <div class="lpage">82</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reconstruction%20of%20the%20Synthetic%20W7984%20%C3%97%20Opata%20M85%20wheat%20reference%20population&amp;author=M.E.%20Sorrells&amp;author=J.P.%20Gustafson&amp;author=D.%20Somers&amp;publication_year=2011&amp;journal=Genome&amp;volume=54&amp;pages=875-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g11-054" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg11-054" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg11-054"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21999208" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">L.Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">D.C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Guo</div>
               <div class="given-names">X.L.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Investigation of genetic diversity and population structure of common wheat cultivars in northern China using DArT markers</div>
          <div class="source">BMC Genetics</div>Â 
          <div class="year">2011</div>
          <div class="volume">12</div>
          <div class="fpage">42</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Investigation%20of%20genetic%20diversity%20and%20population%20structure%20of%20common%20wheat%20cultivars%20in%20northern%20China%20using%20DArT%20markers&amp;author=L.Y.%20Zhang&amp;author=D.C.%20Liu&amp;author=X.L.%20Guo&amp;publication_year=2011&amp;journal=BMC%20Genetics&amp;volume=12&amp;pages=42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2156-12-42" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2156-12-42" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2156-12-42"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21569312" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Trebbi</div>
               <div class="given-names">D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Maccaferri</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">de Heer</div>
               <div class="given-names">P.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">High-throughput SNP discovery and genotyping in durum wheat (<em>Triticum durum</em> Desf.)</div>
          <div class="source">Theor. Appl. Genet.</div>Â 
          <div class="year">2011</div>
          <div class="volume">123</div>
          <div class="fpage">555</div>
          <div class="lpage">69</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20SNP%20discovery%20and%20genotyping%20in%20durum%20wheat%20(Triticum%20durum%20Desf.)&amp;author=D.%20Trebbi&amp;author=M.%20Maccaferri&amp;author=P.%20de%20Heer&amp;publication_year=2011&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=123&amp;pages=555-69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-011-1607-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-011-1607-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-011-1607-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21611761" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Allen</div>
               <div class="given-names">A.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Barker</div>
               <div class="given-names">G.L.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Berry</div>
               <div class="given-names">S.T.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Transcript-specific, single-nucleotide polymorphism discovery and linkage analysis in hexaploid bread wheat (<em>Triticum aestivum</em> L.)</div>
          <div class="source">Plant Biotechnol. J.</div>Â 
          <div class="year">2011</div>
          <div class="volume">9</div>
          <div class="fpage">1086</div>
          <div class="lpage">99</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcript-specific%2C%20single-nucleotide%20polymorphism%20discovery%20and%20linkage%20analysis%20in%20hexaploid%20bread%20wheat%20(Triticum%20aestivum%20L.)&amp;author=A.M.%20Allen&amp;author=G.L.A.%20Barker&amp;author=S.T.%20Berry&amp;publication_year=2011&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=9&amp;pages=1086-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.2011.9.issue-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2011.9.issue-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.2011.9.issue-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21627760" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lai</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Duran</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Berkman</div>
               <div class="given-names">P.J.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Single nucleotide polymorphism discovery from wheat next-generation sequence data</div>
          <div class="source">Plant Biotechnol. J.</div>Â 
          <div class="year">2012</div>
          <div class="volume">10</div>
          <div class="fpage">743</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single%20nucleotide%20polymorphism%20discovery%20from%20wheat%20next-generation%20sequence%20data&amp;author=K.%20Lai&amp;author=C.%20Duran&amp;author=P.J.%20Berkman&amp;publication_year=2012&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=10&amp;pages=743-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1467-7652.2012.00718.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1467-7652.2012.00718.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1467-7652.2012.00718.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22748104" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cavanagh</div>
               <div class="given-names">C.R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Chao</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">S.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Genome-wide comparative diversity uncovers multiple targets of selection for improvement in hexaploid wheat landraces and cultivars</div>
          <div class="source">Proc. Natl. Acad. Sci. USA</div>Â 
          <div class="year">2013</div>
          <div class="volume">110</div>
          <div class="fpage">8057</div>
          <div class="lpage">62</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20comparative%20diversity%20uncovers%20multiple%20targets%20of%20selection%20for%20improvement%20in%20hexaploid%20wheat%20landraces%20and%20cultivars&amp;author=C.R.%20Cavanagh&amp;author=S.%20Chao&amp;author=S.%20Wang&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=110&amp;pages=8057-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1217133110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1217133110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1217133110"> </span></div>Â </div></div></div></div></div><div content-id="DSU020C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Poland</div>
               <div class="given-names">J.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Brown</div>
               <div class="given-names">P.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sorrells</div>
               <div class="given-names">M.E.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Jannink</div>
               <div class="given-names">J.L.</div>
            </div>
          </div>
          <div class="article-title">Development of high-density genetic maps for barley and wheat using a novel two-enzyme genotyping-by-sequencing approach</div>
          <div class="source">PLoS ONE</div>Â 
          <div class="year">2012</div>
          <div class="volume">7</div>
          <div class="fpage">e32253</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20high-density%20genetic%20maps%20for%20barley%20and%20wheat%20using%20a%20novel%20two-enzyme%20genotyping-by-sequencing%20approach&amp;author=J.A.%20Poland&amp;author=P.J.%20Brown&amp;author=M.E.%20Sorrells&amp;author=J.L.%20Jannink&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e32253" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0032253" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0032253" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0032253"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22389690" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Saintenac</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Jiang</div>
               <div class="given-names">D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Akhunov</div>
               <div class="given-names">E.</div>
            </div>
          </div>
          <div class="article-title">Sequence-based mapping of the polyploid wheat genome</div>
          <div class="source">G3-Genes Genomes Genet.</div>Â 
          <div class="year">2013</div>
          <div class="volume">3</div>
          <div class="fpage">1105</div>
          <div class="lpage">14</div>
        </div></div></div></div><div content-id="DSU020C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">van Poecke</div>
               <div class="given-names">R.M.P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Maccaferri</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Tang</div>
               <div class="given-names">J.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Sequence-based SNP genotyping in durum wheat</div>
          <div class="source">Plant Biotechnol. J.</div>Â 
          <div class="year">2013</div>
          <div class="volume">11</div>
          <div class="fpage">809</div>
          <div class="lpage">17</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence-based%20SNP%20genotyping%20in%20durum%20wheat&amp;author=R.M.P.%20van%20Poecke&amp;author=M.%20Maccaferri&amp;author=J.%20Tang&amp;publication_year=2013&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=11&amp;pages=809-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.2013.11.issue-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2013.11.issue-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.2013.11.issue-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23639032" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Winfield</div>
               <div class="given-names">M.O.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wilkinson</div>
               <div class="given-names">P.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Allen</div>
               <div class="given-names">A.M.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Targeted re-sequencing of the allohexaploid wheat exome</div>
          <div class="source">Plant Biotechnol. J.</div>Â 
          <div class="year">2012</div>
          <div class="volume">10</div>
          <div class="fpage">733</div>
          <div class="lpage">42</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20re-sequencing%20of%20the%20allohexaploid%20wheat%20exome&amp;author=M.O.%20Winfield&amp;author=P.A.%20Wilkinson&amp;author=A.M.%20Allen&amp;publication_year=2012&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=10&amp;pages=733-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1467-7652.2012.00713.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1467-7652.2012.00713.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1467-7652.2012.00713.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22703335" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Davey</div>
               <div class="given-names">J.W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hohenlohe</div>
               <div class="given-names">P.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Etter</div>
               <div class="given-names">P.D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Boone</div>
               <div class="given-names">J.Q.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Catchen</div>
               <div class="given-names">J.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Blaxter</div>
               <div class="given-names">M.L.</div>
            </div>
          </div>
          <div class="article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing</div>
          <div class="source">Nat. Rev. Genet.</div>Â 
          <div class="year">2011</div>
          <div class="volume">12</div>
          <div class="fpage">499</div>
          <div class="lpage">510</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20genetic%20marker%20discovery%20and%20genotyping%20using%20next-generation%20sequencing&amp;author=J.W.%20Davey&amp;author=P.A.%20Hohenlohe&amp;author=P.D.%20Etter&amp;author=J.Q.%20Boone&amp;author=J.M.%20Catchen&amp;author=M.L.%20Blaxter&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=12&amp;pages=499-510" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21681211" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kobayashi</div>
               <div class="given-names">F.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Takumi</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Handa</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="article-title">Identification of quantitative trait loci for ABA responsiveness at the seedling stage associated with ABA-regulated gene expression in common wheat</div>
          <div class="source">Theor. Appl. Genet.</div>Â 
          <div class="year">2010</div>
          <div class="volume">121</div>
          <div class="fpage">629</div>
          <div class="lpage">41</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20quantitative%20trait%20loci%20for%20ABA%20responsiveness%20at%20the%20seedling%20stage%20associated%20with%20ABA-regulated%20gene%20expression%20in%20common%20wheat&amp;author=F.%20Kobayashi&amp;author=S.%20Takumi&amp;author=H.%20Handa&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=121&amp;pages=629-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-010-1335-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-010-1335-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-010-1335-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20401645" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Enoki</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Nishimura</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Murakami</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="article-title">Method for designing probe in DNA microarray, and DNA microarray provided with probe designed thereby</div>
          <div class="year">2011</div>
          <div class="comment">International publication number: WO 2011/074510</div>
        </div></div></div></div><div content-id="DSU020C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Enoki</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Nishimura</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Murakami</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="article-title">Method for designing probe in DNA microarray, and DNA microarray provided with probe designed thereby</div>
          <div class="year">2012</div>
          <div class="comment">US patent application 20120190582A1</div>
        </div></div></div></div><div content-id="DSU020C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zerbino</div>
               <div class="given-names">D.R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Birney</div>
               <div class="given-names">E.</div>
            </div>
          </div>
          <div class="article-title">Velvet: algorithms for de novo short read assembly using de Bruijn graphs</div>
          <div class="source">Genome Res.</div>Â 
          <div class="year">2008</div>
          <div class="volume">18</div>
          <div class="fpage">821</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Velvet%3A%20algorithms%20for%20de%20novo%20short%20read%20assembly%20using%20de%20Bruijn%20graphs&amp;author=D.R.%20Zerbino&amp;author=E.%20Birney&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=821-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.074492.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.074492.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.074492.107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18349386" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Langmead</div>
               <div class="given-names">B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Trapnell</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Pop</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Salzberg</div>
               <div class="given-names">S.L.</div>
            </div>
          </div>
          <div class="article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</div>
          <div class="source">Genome Biol.</div>Â 
          <div class="year">2009</div>
          <div class="volume">10</div>
          <div class="fpage">R25</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultrafast%20and%20memory-efficient%20alignment%20of%20short%20DNA%20sequences%20to%20the%20human%20genome&amp;author=B.%20Langmead&amp;author=C.%20Trapnell&amp;author=M.%20Pop&amp;author=S.L.%20Salzberg&amp;publication_year=2009&amp;journal=Genome%20Biol.&amp;volume=10&amp;pages=R25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-3-r25" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-3-r25" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-3-r25"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19261174" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Beales</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Turner</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Griffiths</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Snape</div>
               <div class="given-names">J.W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Laurie</div>
               <div class="given-names">D.A.</div>
            </div>
          </div>
          <div class="article-title">A <em>pseudo-response regulator</em> is misexpressed in the photoperiod insensitive <em>Ppd-D1a</em> mutant of wheat (<em>Triticum aestivum</em> L.)</div>
          <div class="source">Theor. Appl. Genet.</div>Â 
          <div class="year">2007</div>
          <div class="volume">115</div>
          <div class="fpage">721</div>
          <div class="lpage">33</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20pseudo-response%20regulator%20is%20misexpressed%20in%20the%20photoperiod%20insensitive%20Ppd-D1a%20mutant%20of%20wheat%20(Triticum%20aestivum%20L.)&amp;author=J.%20Beales&amp;author=A.%20Turner&amp;author=S.%20Griffiths&amp;author=J.W.%20Snape&amp;author=D.A.%20Laurie&amp;publication_year=2007&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=115&amp;pages=721-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-007-0603-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-007-0603-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-007-0603-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17634915" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Iwata</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ninomiya</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="article-title">AntMap: constructing genetic linkage maps using an ant colony optimization algorithm</div>
          <div class="source">Breed. Sci.</div>Â 
          <div class="year">2006</div>
          <div class="volume">56</div>
          <div class="fpage">371</div>
          <div class="lpage">7</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AntMap%3A%20constructing%20genetic%20linkage%20maps%20using%20an%20ant%20colony%20optimization%20algorithm&amp;author=H.%20Iwata&amp;author=S.%20Ninomiya&amp;publication_year=2006&amp;journal=Breed.%20Sci.&amp;volume=56&amp;pages=371-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.56.371" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.56.371" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.56.371"> </span></div>Â </div></div></div></div></div><div content-id="DSU020C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lorieux</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="article-title">MapDisto: fast and efficient computation of genetic linkage maps</div>
          <div class="source">Mol. Breed.</div>Â 
          <div class="year">2012</div>
          <div class="volume">30</div>
          <div class="fpage">1231</div>
          <div class="lpage">5</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MapDisto%3A%20fast%20and%20efficient%20computation%20of%20genetic%20linkage%20maps&amp;author=M.%20Lorieux&amp;publication_year=2012&amp;journal=Mol.%20Breed.&amp;volume=30&amp;pages=1231-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-012-9706-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-012-9706-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-012-9706-y"> </span></div>Â </div></div></div></div></div><div content-id="DSU020C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kosambi</div>
               <div class="given-names">D.D.</div>
            </div>
          </div>
          <div class="article-title">The estimation of map distance from recombination values</div>
          <div class="source">Ann. Eugen.</div>Â 
          <div class="year">1944</div>
          <div class="volume">12</div>
          <div class="fpage">172</div>
          <div class="lpage">5</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20estimation%20of%20map%20distance%20from%20recombination%20values&amp;author=D.D.%20Kosambi&amp;publication_year=1944&amp;journal=Ann.%20Eugen.&amp;volume=12&amp;pages=172-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-1809.1943.tb02321.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-1809.1943.tb02321.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-1809.1943.tb02321.x"> </span></div>Â </div></div></div></div></div><div content-id="DSU020C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jia</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Zhao</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kong</div>
               <div class="given-names">X.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">
            <em>Aegilops tauschii</em> draft genome sequence reveals a gene repertoire for wheat adaptation</div>
          <div class="source">Nature</div>Â 
          <div class="year">2013</div>
          <div class="volume">496</div>
          <div class="fpage">91</div>
          <div class="lpage">5</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aegilops%20tauschii%20draft%20genome%20sequence%20reveals%20a%20gene%20repertoire%20for%20wheat%20adaptation&amp;author=J.%20Jia&amp;author=S.%20Zhao&amp;author=X.%20Kong&amp;publication_year=2013&amp;journal=Nature&amp;volume=496&amp;pages=91-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature12028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature12028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature12028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23535592" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ling</div>
               <div class="given-names">H.Q.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Zhao</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">D.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Draft genome of the wheat A-genome progenitor <em>Triticum urartu</em></div>
          <div class="source">Nature</div>Â 
          <div class="year">2013</div>
          <div class="volume">496</div>
          <div class="fpage">87</div>
          <div class="lpage">90</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20of%20the%20wheat%20A-genome%20progenitor%20Triticum%20urartu&amp;author=H.Q.%20Ling&amp;author=S.%20Zhao&amp;author=D.%20Liu&amp;publication_year=2013&amp;journal=Nature&amp;volume=496&amp;pages=87-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11997" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11997" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11997"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23535596" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="collab">International Barley Genome Sequencing Consortium</div>
          <div class="article-title">A physical, genetic and functional sequence assembly of the barley genome</div>
          <div class="source">Nature</div>Â 
          <div class="year">2012</div>
          <div class="volume">491</div>
          <div class="fpage">711</div>
          <div class="lpage">6</div>
        <div class="citation-links"><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23075845" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Luo</div>
               <div class="given-names">M.C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gu</div>
               <div class="given-names">Y.Q.</div>
            </div>
            <div class="name string-name">
              <div class="surname">You</div>
               <div class="given-names">F.M.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">A 4-gigabase physical map unlocks the structure and evolution of the complex genome of <em>Aegilops tauschii</em>, the wheat D-genome progenitor</div>
          <div class="source">Proc. Natl. Acad. Sci. USA</div>Â 
          <div class="year">2013</div>
          <div class="volume">110</div>
          <div class="fpage">7940</div>
          <div class="lpage">5</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%204-gigabase%20physical%20map%20unlocks%20the%20structure%20and%20evolution%20of%20the%20complex%20genome%20of%20Aegilops%20tauschii%2C%20the%20wheat%20D-genome%20progenitor&amp;author=M.C.%20Luo&amp;author=Y.Q.%20Gu&amp;author=F.M.%20You&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=110&amp;pages=7940-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1219082110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1219082110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1219082110"> </span></div>Â </div></div></div></div></div><div content-id="DSU020C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Krzywinski</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Schein</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Birol</div>
               <div class="given-names">I.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Circos: an information aesthetic for comparative genomics</div>
          <div class="source">Genome Res.</div>Â 
          <div class="year">2009</div>
          <div class="volume">19</div>
          <div class="fpage">1639</div>
          <div class="lpage">45</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circos%3A%20an%20information%20aesthetic%20for%20comparative%20genomics&amp;author=M.%20Krzywinski&amp;author=J.%20Schein&amp;author=I.%20Birol&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=1639-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.092759.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.092759.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.092759.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19541911" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Broman</div>
               <div class="given-names">K.W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wu</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sen</div>
               <div class="given-names">Åš.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Churchill</div>
               <div class="given-names">G.A.</div>
            </div>
          </div>
          <div class="article-title">R/qtl: QTL mapping in experimental crosses</div>
          <div class="source">Bioinformatics</div>Â 
          <div class="year">2003</div>
          <div class="volume">19</div>
          <div class="fpage">889</div>
          <div class="lpage">90</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=R%2Fqtl%3A%20QTL%20mapping%20in%20experimental%20crosses&amp;author=K.W.%20Broman&amp;author=H.%20Wu&amp;author=%C5%9A.%20Sen&amp;author=G.A.%20Churchill&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=889-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12724300" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Devos</div>
               <div class="given-names">K.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Dubcovsky</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">DvoÅ™Ã¡k</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Chinoy</div>
               <div class="given-names">C.N.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gale</div>
               <div class="given-names">M.D.</div>
            </div>
          </div>
          <div class="article-title">Structural evolution of wheat chromosomes 4A, 5A, and 7B and its impact on recombination</div>
          <div class="source">Theor. Appl. Genet.</div>Â 
          <div class="year">1995</div>
          <div class="volume">91</div>
          <div class="fpage">282</div>
          <div class="lpage">8</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20evolution%20of%20wheat%20chromosomes%204A%2C%205A%2C%20and%207B%20and%20its%20impact%20on%20recombination&amp;author=K.M.%20Devos&amp;author=J.%20Dubcovsky&amp;author=J.%20Dvo%C5%99%C3%A1k&amp;author=C.N.%20Chinoy&amp;author=M.D.%20Gale&amp;publication_year=1995&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=91&amp;pages=282-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00220890" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00220890" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00220890"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24169776" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ma</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Stiller</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Berkman</div>
               <div class="given-names">P.J.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Sequence-based analysis of translocations and inversions in bread wheat (<em>Triticum aestivum</em> L.)</div>
          <div class="source">PLoS ONE</div>Â 
          <div class="year">2013</div>
          <div class="volume">8</div>
          <div class="fpage">e79329</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence-based%20analysis%20of%20translocations%20and%20inversions%20in%20bread%20wheat%20(Triticum%20aestivum%20L.)&amp;author=J.%20Ma&amp;author=J.%20Stiller&amp;author=P.J.%20Berkman&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e79329" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0079329" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0079329" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0079329"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24260197" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Brenchley</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Spannagl</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Pfeifer</div>
               <div class="given-names">M.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Analysis of the bread wheat genome using whole-genome shotgun sequencing</div>
          <div class="source">Nature</div>Â 
          <div class="year">2012</div>
          <div class="volume">491</div>
          <div class="fpage">705</div>
          <div class="lpage">10</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20bread%20wheat%20genome%20using%20whole-genome%20shotgun%20sequencing&amp;author=R.%20Brenchley&amp;author=M.%20Spannagl&amp;author=M.%20Pfeifer&amp;publication_year=2012&amp;journal=Nature&amp;volume=491&amp;pages=705-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11650" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11650" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11650"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23192148" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sears</div>
               <div class="given-names">E.R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Loegering</div>
               <div class="given-names">W.Q.</div>
            </div>
          </div>
          <div class="article-title">A pollen-killing gene in wheat</div>
          <div class="source">Genetics</div>Â 
          <div class="year">1961</div>
          <div class="volume">46</div>
          <div class="fpage">897</div>
        </div></div></div></div><div content-id="DSU020C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Takumi</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Motomura</div>
               <div class="given-names">Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Iehisa</div>
               <div class="given-names">J.C.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kobayashi</div>
               <div class="given-names">F.</div>
            </div>
          </div>
          <div class="article-title">Segregation distortion caused by weak hybrid necrosis in recombinant inbred lines of common wheat</div>
          <div class="source">Genetica</div>Â 
          <div class="year">2013</div>
          <div class="volume">141</div>
          <div class="fpage">463</div>
          <div class="lpage">70</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Segregation%20distortion%20caused%20by%20weak%20hybrid%20necrosis%20in%20recombinant%20inbred%20lines%20of%20common%20wheat&amp;author=S.%20Takumi&amp;author=Y.%20Motomura&amp;author=J.C.M.%20Iehisa&amp;author=F.%20Kobayashi&amp;publication_year=2013&amp;journal=Genetica&amp;volume=141&amp;pages=463-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10709-013-9745-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10709-013-9745-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10709-013-9745-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24146090" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Chu</div>
               <div class="given-names">C.G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Faris</div>
               <div class="given-names">J.D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Friesen</div>
               <div class="given-names">T.L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Xu</div>
               <div class="given-names">S.S.</div>
            </div>
          </div>
          <div class="article-title">Molecular mapping of hybrid necrosis genes <em>Ne1</em> and <em>Ne2</em> in hexaploid wheat using microsatellite markers</div>
          <div class="source">Theor. Appl. Genet.</div>Â 
          <div class="year">2006</div>
          <div class="volume">112</div>
          <div class="fpage">1374</div>
          <div class="lpage">81</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20mapping%20of%20hybrid%20necrosis%20genes%20Ne1%20and%20Ne2%20in%20hexaploid%20wheat%20using%20microsatellite%20markers&amp;author=C.G.%20Chu&amp;author=J.D.%20Faris&amp;author=T.L.%20Friesen&amp;author=S.S.%20Xu&amp;publication_year=2006&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=112&amp;pages=1374-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-006-0239-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-006-0239-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-006-0239-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16518615" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bikard</div>
               <div class="given-names">D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Patel</div>
               <div class="given-names">D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Le MettÃ©</div>
               <div class="given-names">C.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Divergent evolution of duplicate genes leads to genetic incompatibilities within <em>A. thaliana</em></div>
          <div class="source">Science</div>Â 
          <div class="year">2009</div>
          <div class="volume">323</div>
          <div class="fpage">623</div>
          <div class="lpage">6</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Divergent%20evolution%20of%20duplicate%20genes%20leads%20to%20genetic%20incompatibilities%20within%20A.%20thaliana&amp;author=D.%20Bikard&amp;author=D.%20Patel&amp;author=C.%20Le%20Mett%C3%A9&amp;publication_year=2009&amp;journal=Science&amp;volume=323&amp;pages=623-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1165917" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1165917" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1165917"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19179528" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mizuta</div>
               <div class="given-names">Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Harushima</div>
               <div class="given-names">Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kurata</div>
               <div class="given-names">N.</div>
            </div>
          </div>
          <div class="article-title">Rice pollen hybrid incompatibility caused by reciprocal gene loss of duplicated genes</div>
          <div class="source">Proc. Natl. Acad. Sci. USA</div>Â 
          <div class="year">2010</div>
          <div class="volume">107</div>
          <div class="fpage">20417</div>
          <div class="lpage">22</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rice%20pollen%20hybrid%20incompatibility%20caused%20by%20reciprocal%20gene%20loss%20of%20duplicated%20genes&amp;author=Y.%20Mizuta&amp;author=Y.%20Harushima&amp;author=N.%20Kurata&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=20417-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1003124107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1003124107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1003124107"> </span></div>Â </div></div></div></div></div><div content-id="DSU020C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yang</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Zhao</div>
               <div class="given-names">X.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Cheng</div>
               <div class="given-names">K.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">A killer-protector system regulates both hybrid sterility and segregation distortion</div>
          <div class="source">Science</div>Â 
          <div class="year">2012</div>
          <div class="volume">337</div>
          <div class="fpage">1336</div>
          <div class="lpage">40</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20killer-protector%20system%20regulates%20both%20hybrid%20sterility%20and%20segregation%20distortion&amp;author=J.%20Yang&amp;author=X.%20Zhao&amp;author=K.%20Cheng&amp;publication_year=2012&amp;journal=Science&amp;volume=337&amp;pages=1336-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1223702" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1223702" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1223702"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22984070" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yu</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Xie</div>
               <div class="given-names">W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">J.</div>
            </div>
            et al. Â 
          </div>
          <div class="article-title">Gains in QTL detection using an ultra-high density SNP map based on population sequencing relative to traditional RFLP/SSR markers</div>
          <div class="source">PLoS ONE</div>Â 
          <div class="year">2011</div>
          <div class="volume">6</div>
          <div class="fpage">e17595</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gains%20in%20QTL%20detection%20using%20an%20ultra-high%20density%20SNP%20map%20based%20on%20population%20sequencing%20relative%20to%20traditional%20RFLP%2FSSR%20markers&amp;author=H.%20Yu&amp;author=W.%20Xie&amp;author=J.%20Wang&amp;publication_year=2011&amp;journal=PLoS%20ONE&amp;volume=6&amp;pages=e17595" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0017595" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0017595" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0017595"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21390234" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">DÃ­az</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Zikhali</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Turner</div>
               <div class="given-names">A.S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Isaac</div>
               <div class="given-names">P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Laurie</div>
               <div class="given-names">D.A.</div>
            </div>
          </div>
          <div class="article-title">Copy number variation affecting the <em>Photoperiod-B1</em> and <em>Vernalization-A1</em> genes is associated with altered flowering time in wheat (<em>Triticum aestivum</em>)</div>
          <div class="source">PLoS ONE</div>Â 
          <div class="year">2012</div>
          <div class="volume">7</div>
          <div class="fpage">e33234</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Copy%20number%20variation%20affecting%20the%20Photoperiod-B1%20and%20Vernalization-A1%20genes%20is%20associated%20with%20altered%20flowering%20time%20in%20wheat%20(Triticum%20aestivum)&amp;author=A.%20D%C3%ADaz&amp;author=M.%20Zikhali&amp;author=A.S.%20Turner&amp;author=P.%20Isaac&amp;author=D.A.%20Laurie&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e33234" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0033234" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0033234" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0033234"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22457747" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nguyen</div>
               <div class="given-names">A.T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Iehisa</div>
               <div class="given-names">J.C.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Mizuno</div>
               <div class="given-names">N.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Nitta</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Nasuda</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Takumi</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="article-title">Differential contribution of two <em>Ppd-1</em> homoeoalleles to early-flowering phenotype in Nepalese and Japanese varieties of common wheat</div>
          <div class="source">Breed. Sci.</div>Â 
          <div class="year">2013</div>
          <div class="volume">63</div>
          <div class="fpage">374</div>
          <div class="lpage">83</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20contribution%20of%20two%20Ppd-1%20homoeoalleles%20to%20early-flowering%20phenotype%20in%20Nepalese%20and%20Japanese%20varieties%20of%20common%20wheat&amp;author=A.T.%20Nguyen&amp;author=J.C.M.%20Iehisa&amp;author=N.%20Mizuno&amp;author=M.%20Nitta&amp;author=S.%20Nasuda&amp;author=S.%20Takumi&amp;publication_year=2013&amp;journal=Breed.%20Sci.&amp;volume=63&amp;pages=374-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1270/jsbbs.63.374" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1270%2Fjsbbs.63.374" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1270%2Fjsbbs.63.374"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24399909" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yan</div>
               <div class="given-names">L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Helguera</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kato</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Fukuyama</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sherman</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Dubcovsky</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="article-title">Allelic variation at the <em>VRN-1</em> promoter region in polyploid wheat</div>
          <div class="source">Theor. Appl. Genet.</div>Â 
          <div class="year">2004</div>
          <div class="volume">109</div>
          <div class="fpage">1677</div>
          <div class="lpage">86</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Allelic%20variation%20at%20the%20VRN-1%20promoter%20region%20in%20polyploid%20wheat&amp;author=L.%20Yan&amp;author=M.%20Helguera&amp;author=K.%20Kato&amp;author=S.%20Fukuyama&amp;author=J.%20Sherman&amp;author=J.%20Dubcovsky&amp;publication_year=2004&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=109&amp;pages=1677-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-004-1796-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-004-1796-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-004-1796-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15480533" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSU020C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU020C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Evers</div>
               <div class="given-names">A.D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Cox</div>
               <div class="given-names">R.I.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Shaheedullah</div>
               <div class="given-names">M.Z.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Withey</div>
               <div class="given-names">R.P.</div>
            </div>
          </div>
          <div class="article-title">Predicting milling extraction rate by image analysis of wheat grains</div>
          <div class="source">Asp. Appl. Biol.</div>Â 
          <div class="year">1990</div>
          <div class="volume">25</div>
          <div class="fpage">417</div>
          <div class="lpage">26</div>
        </div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">Â© The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2754552&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/5/10.1093_dnares_dsu020/2/dsu020_Supplementary_Data.zip?Expires=2147483647&amp;Signature=vvvikub2UQhURv2xACBraGOSh-eNu7R3~prMItmS2ZEU9r-t4IN~JXX1FoiR~K7pCIrb1xt-9zUcKHSxsdYf5wpXzWiLrigDxT4mAZlpLAHpS-7l-zoxOfLkj9704ZRDbzRE3FyMD5fm7hnsK~yo9TxUFvkOh~93GRrwYksf4SAhouIA5BgZg8bEsr7szQGQbRDfkZONH5W3YYEk5hEaja6MAmRLh34R8eYwmqJrN-B1sH1lJqU-332sQkffjxR17363e7Ya9Zm7gJTmK1V2eDzfnPcMzN7ztKZin~1FQH7jUFNaeL6gjpiaJMfzW8S5YeRPssLJExIZkoBcv43ImQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">457</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000344244900009&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">14</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsu020"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2754552"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000344244900009&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f5%2f555%2f2754552">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000344244900009&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (14)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F5%2F555%2F2754552" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2754552'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transpositionâ€“duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="High-Density Genetic Map with Array-Based Markers Facilitates Structural and Quantitative Trait Locus Analyses of the Common Wheat Genome" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsu020");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f5%2f555%2f2754552">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

