

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/3/231/393171","@type":"ScholarlyArticle","name":"Sequence Analysis of the Genome of Carnation ( Dianthus caryophyllus  L.)","datePublished":"2013-12-17","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2014-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst053","keywords":["<italic>Dianthus caryophyllus</italic> L.","carnation","genome sequencing","gene prediction"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/3/231/393171","author":[{"name":"Yagi, Masafumi","affiliation":"1  NARO Institute of Floricultural Science (NIFS) ,  2-1 Fujimoto, Tsukuba, Ibaraki 305-8519 ,  Japan","@type":"Person"},{"name":"Kosugi, Shunichi","affiliation":"2  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Hirakawa, Hideki","affiliation":"2  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Ohmiya, Akemi","affiliation":"1  NARO Institute of Floricultural Science (NIFS) ,  2-1 Fujimoto, Tsukuba, Ibaraki 305-8519 ,  Japan","@type":"Person"},{"name":"Tanase, Koji","affiliation":"1  NARO Institute of Floricultural Science (NIFS) ,  2-1 Fujimoto, Tsukuba, Ibaraki 305-8519 ,  Japan","@type":"Person"},{"name":"Harada, Taro","affiliation":"1  NARO Institute of Floricultural Science (NIFS) ,  2-1 Fujimoto, Tsukuba, Ibaraki 305-8519 ,  Japan","@type":"Person"},{"name":"Kishimoto, Kyutaro","affiliation":"1  NARO Institute of Floricultural Science (NIFS) ,  2-1 Fujimoto, Tsukuba, Ibaraki 305-8519 ,  Japan","@type":"Person"},{"name":"Nakayama, Masayoshi","affiliation":"1  NARO Institute of Floricultural Science (NIFS) ,  2-1 Fujimoto, Tsukuba, Ibaraki 305-8519 ,  Japan","@type":"Person"},{"name":"Ichimura, Kazuo","affiliation":"1  NARO Institute of Floricultural Science (NIFS) ,  2-1 Fujimoto, Tsukuba, Ibaraki 305-8519 ,  Japan","@type":"Person"},{"name":"Onozaki, Takashi","affiliation":"1  NARO Institute of Floricultural Science (NIFS) ,  2-1 Fujimoto, Tsukuba, Ibaraki 305-8519 ,  Japan","@type":"Person"},{"name":"Yamaguchi, Hiroyasu","affiliation":"1  NARO Institute of Floricultural Science (NIFS) ,  2-1 Fujimoto, Tsukuba, Ibaraki 305-8519 ,  Japan","@type":"Person"},{"name":"Sasaki, Nobuhiro","affiliation":"3  Tokyo University of Agriculture and Technology ,  2-24-16 Naka-cho, Koganei, Tokyo 184-8588 ,  Japan","@type":"Person"},{"name":"Miyahara, Taira","affiliation":"3  Tokyo University of Agriculture and Technology ,  2-24-16 Naka-cho, Koganei, Tokyo 184-8588 ,  Japan","@type":"Person"},{"name":"Nishizaki, Yuzo","affiliation":"3  Tokyo University of Agriculture and Technology ,  2-24-16 Naka-cho, Koganei, Tokyo 184-8588 ,  Japan","@type":"Person"},{"name":"Ozeki, Yoshihiro","affiliation":"3  Tokyo University of Agriculture and Technology ,  2-24-16 Naka-cho, Koganei, Tokyo 184-8588 ,  Japan","@type":"Person"},{"name":"Nakamura, Noriko","affiliation":"4  Research Institute ,  Suntory Global Innovation Center ,  1-1-1 Wakayamadai, Shimamoto, Mishima, Osaka 618-8503 ,  Japan","@type":"Person"},{"name":"Suzuki, Takamasa","affiliation":"5  JST, ERATO, Higashiyama Live-Holonics Project ,  Nagoya University ,  Furo-cho, Chikusa-ku, Nagoya, Aichi 464-8602 ,  Japan    6  Division of Biological Science, Graduate School of Science ,  Nagoya University ,  Furo-cho, Chikusa-ku, Nagoya, Aichi 464-8602 ,  Japan","@type":"Person"},{"name":"Tanaka, Yoshikazu","affiliation":"4  Research Institute ,  Suntory Global Innovation Center ,  1-1-1 Wakayamadai, Shimamoto, Mishima, Osaka 618-8503 ,  Japan","@type":"Person"},{"name":"Sato, Shusei","affiliation":"2  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Shirasawa, Kenta","affiliation":"2  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Isobe, Sachiko","affiliation":"2  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Miyamura, Yoshinori","affiliation":"2  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Watanabe, Akiko","affiliation":"2  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Nakayama, Shinobu","affiliation":"2  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Kishida, Yoshie","affiliation":"2  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Kohara, Mitsuyo","affiliation":"2  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Tabata, Satoshi","affiliation":"2  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan","@type":"Person"}],"description":"Abstract.  The whole-genome sequence of carnation (Dianthus caryophyllus L.) cv. ‘Francesco’ was determined using a combination of different new-generation mult","pageStart":"231","pageEnd":"241","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/3/1/m_cover.gif?Expires=2147483647&Signature=eD74u2chqR7m7Q4xVEpu-B2CnS8a5UZybuUBj9IiQcs7vcn-sLThVTH2YkcAPlEiODArsSqGH041ZPg5RakmGwkyHxqVC4LdBdXxFui4WCeLWHefwauec7mA5UKsie6Bk-ZRp7yPg10FfeUTBrjN7buCiodIP00VlbrJgVNTJFuLyaUUxPgJ3yahCwwX4TAcK2CRCX-3Y~LqokrGvJCW5URLAAo4GxWdmNUAJqBhdMREkIh~McyOhMfYKJNLDTxW9vnvEbKMxNW5048gQKGgF7kMxJ3~NsfswZCCrwYr2quaHFHHORZM0v4Eh4XRYgKpFeDHa6jnY4OeJPxjcOPrMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Sequence Analysis of the Genome of Carnation ( Dianthus caryophyllus  L.)","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/3/1/m_cover.gif?Expires=2147483647&Signature=eD74u2chqR7m7Q4xVEpu-B2CnS8a5UZybuUBj9IiQcs7vcn-sLThVTH2YkcAPlEiODArsSqGH041ZPg5RakmGwkyHxqVC4LdBdXxFui4WCeLWHefwauec7mA5UKsie6Bk-ZRp7yPg10FfeUTBrjN7buCiodIP00VlbrJgVNTJFuLyaUUxPgJ3yahCwwX4TAcK2CRCX-3Y~LqokrGvJCW5URLAAo4GxWdmNUAJqBhdMREkIh~McyOhMfYKJNLDTxW9vnvEbKMxNW5048gQKGgF7kMxJ3~NsfswZCCrwYr2quaHFHHORZM0v4Eh4XRYgKpFeDHa6jnY4OeJPxjcOPrMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Sequence Analysis of the Genome of Carnation ( Dianthus caryophyllus  L.)" />
    <meta property="og:description" content="Abstract.  The whole-genome sequence of carnation (Dianthus caryophyllus L.) cv. ‘Francesco’ was determined using a combination of different new-generation mult" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/3/231/393171" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-12-17" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Yagi, Masafumi" /><meta name="citation_author_institution" content="NARO Institute of Floricultural Science (NIFS), 2-1 Fujimoto, Tsukuba, Ibaraki 305-8519, Japan" /><meta name="citation_author" content="Kosugi, Shunichi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Hirakawa, Hideki" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Ohmiya, Akemi" /><meta name="citation_author_institution" content="NARO Institute of Floricultural Science (NIFS), 2-1 Fujimoto, Tsukuba, Ibaraki 305-8519, Japan" /><meta name="citation_author" content="Tanase, Koji" /><meta name="citation_author_institution" content="NARO Institute of Floricultural Science (NIFS), 2-1 Fujimoto, Tsukuba, Ibaraki 305-8519, Japan" /><meta name="citation_author" content="Harada, Taro" /><meta name="citation_author_institution" content="NARO Institute of Floricultural Science (NIFS), 2-1 Fujimoto, Tsukuba, Ibaraki 305-8519, Japan" /><meta name="citation_author" content="Kishimoto, Kyutaro" /><meta name="citation_author_institution" content="NARO Institute of Floricultural Science (NIFS), 2-1 Fujimoto, Tsukuba, Ibaraki 305-8519, Japan" /><meta name="citation_author" content="Nakayama, Masayoshi" /><meta name="citation_author_institution" content="NARO Institute of Floricultural Science (NIFS), 2-1 Fujimoto, Tsukuba, Ibaraki 305-8519, Japan" /><meta name="citation_author" content="Ichimura, Kazuo" /><meta name="citation_author_institution" content="NARO Institute of Floricultural Science (NIFS), 2-1 Fujimoto, Tsukuba, Ibaraki 305-8519, Japan" /><meta name="citation_author" content="Onozaki, Takashi" /><meta name="citation_author_institution" content="NARO Institute of Floricultural Science (NIFS), 2-1 Fujimoto, Tsukuba, Ibaraki 305-8519, Japan" /><meta name="citation_author" content="Yamaguchi, Hiroyasu" /><meta name="citation_author_institution" content="NARO Institute of Floricultural Science (NIFS), 2-1 Fujimoto, Tsukuba, Ibaraki 305-8519, Japan" /><meta name="citation_author" content="Sasaki, Nobuhiro" /><meta name="citation_author_institution" content="Tokyo University of Agriculture and Technology, 2-24-16 Naka-cho, Koganei, Tokyo 184-8588, Japan" /><meta name="citation_author" content="Miyahara, Taira" /><meta name="citation_author_institution" content="Tokyo University of Agriculture and Technology, 2-24-16 Naka-cho, Koganei, Tokyo 184-8588, Japan" /><meta name="citation_author" content="Nishizaki, Yuzo" /><meta name="citation_author_institution" content="Tokyo University of Agriculture and Technology, 2-24-16 Naka-cho, Koganei, Tokyo 184-8588, Japan" /><meta name="citation_author" content="Ozeki, Yoshihiro" /><meta name="citation_author_institution" content="Tokyo University of Agriculture and Technology, 2-24-16 Naka-cho, Koganei, Tokyo 184-8588, Japan" /><meta name="citation_author" content="Nakamura, Noriko" /><meta name="citation_author_institution" content="Research Institute, Suntory Global Innovation Center, 1-1-1 Wakayamadai, Shimamoto, Mishima, Osaka 618-8503, Japan" /><meta name="citation_author" content="Suzuki, Takamasa" /><meta name="citation_author_institution" content="JST, ERATO, Higashiyama Live-Holonics Project, Nagoya University, Furo-cho, Chikusa-ku, Nagoya, Aichi 464-8602, Japan" /><meta name="citation_author_institution" content="Division of Biological Science, Graduate School of Science, Nagoya University, Furo-cho, Chikusa-ku, Nagoya, Aichi 464-8602, Japan" /><meta name="citation_author" content="Tanaka, Yoshikazu" /><meta name="citation_author_institution" content="Research Institute, Suntory Global Innovation Center, 1-1-1 Wakayamadai, Shimamoto, Mishima, Osaka 618-8503, Japan" /><meta name="citation_author" content="Sato, Shusei" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Shirasawa, Kenta" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Isobe, Sachiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Miyamura, Yoshinori" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Watanabe, Akiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Nakayama, Shinobu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kishida, Yoshie" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kohara, Mitsuyo" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Tabata, Satoshi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_title" content="Sequence Analysis of the Genome of Carnation (Dianthus caryophyllus L.)" /><meta name="citation_firstpage" content="231" /><meta name="citation_lastpage" content="241" /><meta name="citation_doi" content="10.1093/dnares/dst053" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2014/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_publisher=Timber Press, Portland; Carnations and Pinks for Garden and Greenhouse;  citation_year=1997; " /><meta name="citation_reference" content="citation_title=Dianthus Linnaeus; citation_publisher=Cambridge University Press, Cambridge; The European Garden Flora;  citation_year=1989;  citation_volume=vol. III;  citation_pages=185-91; " /><meta name="citation_reference" content="citation_title=Genetic diversity of Dianthus accessions as assessed using two molecular marker systems (SRAPs and ISSRs) and morphological traits; Sci. Hortic.;  citation_year=2008;  citation_volume=117;  citation_pages=263-70; " /><meta name="citation_reference" content="citation_title=An active hAT transposable element causing bud mutation of carnation by insertion into the flavonoid 3′-hydroxylase gene; Mol. Genet. Genomics;  citation_year=2013;  citation_volume=288;  citation_pages=175-84; " /><meta name="citation_reference" content="citation_title=Flower colour and cytochromes P450; Phil. Trans. R. Soc. B.;  citation_year=2013;  citation_volume=368;  citation_pages=20120432" /><meta name="citation_reference" content="citation_title=Biosynthesis of plant pigments: anthocyanins, betalains and carotenoids; Plant, J.;  citation_year=2008;  citation_volume=54;  citation_pages=733-49; " /><meta name="citation_reference" content="citation_title=Improvement of flower vase life using cross-breeding techniques in carnation (Dianthus caryophyllus L.); JARQ;  citation_year=2008;  citation_volume=42;  citation_pages=137-44; " /><meta name="citation_reference" content="citation_title=Senescence and postharvest physiology of cut flowers, part 2; Hort. Rev.;  citation_year=1981;  citation_volume=3;  citation_pages=59-143; " /><meta name="citation_reference" content="citation_title=Crossings and selections for six generations based on flower vase life to create lines with ethylene resistance or ultra-long vase life in carnations (Dianthus caryophyllus L.); J. Japan. Soc. Hort. Sci.;  citation_year=2011;  citation_volume=80;  citation_pages=486-98; " /><meta name="citation_reference" content="citation_title=Sucrose prevents up-regulation of senescence-associated genes in carnation petals; J. Exp. Bot.;  citation_year=2007;  citation_volume=58;  citation_pages=2873-85; " /><meta name="citation_reference" content="citation_title=Construction of a genetic linkage map for the carnation by using RAPD and SSR markers and mapping quantitative trait loci (QTL) for resistance to bacterial wilt caused by Burkholderia caryophylli; J. Japan. Soc. Hort. Sci.;  citation_year=2006;  citation_volume=75;  citation_pages=166-72; " /><meta name="citation_reference" content="citation_title=QTL analysis for resistance to bacterial wilt (Burkholderia caryophylli) in carnation (Dianthus caryophyllus) using an SSR-based genetic linkage map; Mol. Breed.;  citation_year=2012;  citation_volume=30;  citation_pages=495-509; " /><meta name="citation_reference" content="citation_title=Transcriptome analysis of carnation (Dianthus caryophyllus L.) based on next-generation sequencing technology; BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=292" /><meta name="citation_reference" content="citation_title=Construction of a reference genetic linkage map for carnation (Dianthus caryophyllus L.); BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=734" /><meta name="citation_reference" content="citation_title=Yellow pinks: interspecific hybridization between Dianthus plumarius and related species with yellow flowers; Sci. Hortic.;  citation_year=1998;  citation_volume=77;  citation_pages=207-18; " /><meta name="citation_reference" content="citation_title=Evaluation of ploidy level and endoreduplication in carnation (Dianthus spp.); Plant Sci.;  citation_year=2013;  citation_volume=20,1–202;  citation_pages=1-11; " /><meta name="citation_reference" content="citation_title=Carnation; citation_author=Japan. Soc. Hort. Sci; citation_publisher=Shoukadoh Publication, Dept. of Publishing of Nakanishi Printing Co., Ltd., Kyoto; Horticulture in Japan 2006;  citation_year=2006;  citation_pages=223-30; " /><meta name="citation_reference" content="citation_title=Application of DNA markers for breeding carnations resistant to bacterial wilt; JARQ;  citation_year=2013;  citation_volume=47;  citation_pages=29-35; " /><meta name="citation_reference" content="citation_title=Illumina mate-paired DNA sequencing-library preparation using Cre-Lox recombination; Nucleic Acids Res.;  citation_year=2012;  citation_volume=40;  citation_pages=e24" /><meta name="citation_reference" content="citation_title=Generation of long insert pairs using a Cre-LoxP inverse PCR approach; PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e29437" /><meta name="citation_reference" content="citation_title=COPE: an accurate k-mer-based pair-end reads connection tool to facilitate genome assembly; Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=2870-4; " /><meta name="citation_reference" content="citation_title=Inheritance in the carnation (Dianthus caryophyllus). III. Inheritance of flower color; Ann. Missouri Bot. Gard.;  citation_year=1947;  citation_volume=34;  citation_pages=39-74; " /><meta name="citation_reference" content="citation_title=Excision of transposable elements from the chalcone isomerase and dihydroflavonol 4-reductase genes may contribute to the variegation of the yellow-flowered carnation (Dianthus caryophyllus); Plant Cell Physiol.;  citation_year=2002;  citation_volume=43;  citation_pages=578-85; " /><meta name="citation_reference" content="citation_title=Identification of the glutathione S-transferase gene responsible for flower color intensity in carnations; Plant Biotechnol.;  citation_year=2012;  citation_volume=29;  citation_pages=223-7; " /><meta name="citation_reference" content="citation_title=TreeView: an application to display phylogenetic trees on personal computers; Comput. Appl. Biosci.;  citation_year=1996;  citation_volume=12;  citation_pages=357-8; " /><meta name="citation_reference" content="citation_title=Characterization and functional identification of a novel plant 4,5-extradiol dioxygenase involved in betalain pigment biosynthesis in Portulaca grandiflora; Plant Physiol.;  citation_year=2004;  citation_volume=134;  citation_pages=265-74; " /><meta name="citation_reference" content="citation_title=The beet R locus encodes a new cytochrome P450 required for red betalain production; Nat. Genet.;  citation_year=2012;  citation_volume=44;  citation_pages=816-20; " /><meta name="citation_reference" content="citation_title=Detection of UDP-glucose:cyclo-DOPA 5-O-glucosyltransferase activity in four o'clocks (Mirabilis jalapa L.); FEBS Lett.;  citation_year=2004;  citation_volume=568;  citation_pages=159-62; " /><meta name="citation_reference" content="citation_title=Isolation and characterization of cDNAs encoding an enzyme with glucosyltransferase activity for cyclo-DOPA from four o’clocks and feather cockscombs; Plant Cell Physiol.;  citation_year=2005;  citation_volume=46;  citation_pages=666-70; " /><meta name="citation_reference" content="citation_title=Molecular characterization of the Arabidopsis 9-cis-epoxycarotenoid dioxygenase gene family; Plant J.;  citation_year=2003;  citation_volume=35;  citation_pages=44-56; " /><meta name="citation_reference" content="citation_title=Analysis of carotenogenic gene expression in petals and leaves of carnation (Dianthus caryophyllus L.); Plant Breed.;  citation_year=2013;  citation_volume=132;  citation_pages=423-9; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of NBS-encoding disease resistance genes in Cucumis sativus and phylogenic study of NBS-encoding genes in Cucurbitaceae crops; BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=109" /><meta name="citation_reference" content="citation_title=The genome of melon (Cucumis melo L.); Proc. Natl. Acad. Sci. USA;  citation_year=2012;  citation_volume=109;  citation_pages=11872-77; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of Carica papaya reveals a small NBS resistance gene family; Mol. Genet. Genomics;  citation_year=2009;  citation_volume=281;  citation_pages=609-26; " /><meta name="citation_reference" content="citation_title=Overview of tomato (Solanum lycopersicum) candidate pathogen recognition genes reveals important Solanum R locus dynamics; New Phytol.;  citation_year=2013;  citation_volume=197;  citation_pages=223-37; " /><meta name="citation_reference" content="citation_title=A high quality draft consensus sequence of the genome of a heterozygous grapevine variety; PLoS ONE;  citation_year=2007;  citation_volume=2;  citation_pages=e1326" /><meta name="citation_reference" content="citation_title=Genome-wide identification of NBS genes in japonica rice reveals significant expansion of divergent non-TIR NBS-LRR genes; Mol. Genet. Genomics;  citation_year=2004;  citation_volume=271;  citation_pages=402-15; " /><meta name="citation_reference" content="citation_title=The absence of TIR-type resistance gene analogues in the sugar beet (Beta vulgaris L.) genome; J. Mol. Evol.;  citation_year=2004;  citation_volume=58;  citation_pages=40-53; " /><meta name="citation_reference" content="citation_title=Differential expression of three members of the 1-aminocyclopropane-1-carboxylate synthase gene family in carnation; Plant Physiol.;  citation_year=1999;  citation_volume=119;  citation_pages=755-64; " /><meta name="citation_reference" content="citation_title=Biochemical diversity among the 1-amino-cyclopropane-1-carboxylate synthase isozymes encoded by the Arabidopsis gene family; J. Biol. Chem.;  citation_year=2003;  citation_volume=278;  citation_pages=49102-12; " /><meta name="citation_reference" content="citation_title=Identification of pinitol as a main sugar constituent and changes in its content during flower bud development in carnation (Dianthus caryophyllus L.); J. Plant Physiol.;  citation_year=1998;  citation_volume=152;  citation_pages=363-7; " /><meta name="citation_reference" content="citation_title=Endo-xyloglucan transferase, a novel class of glycosyltransferase that catalyzes transfer of a segment of xyloglucan molecule to another xyloglucan molecule; , J. Biol. Chem.;  citation_year=1992;  citation_volume=267;  citation_pages=21058-64; " /><meta name="citation_reference" content="citation_title=Cell wall extensibility and effect of cell-wall-loosening proteins during rose flower opening; J. Japan. Soc. Hort. Sci.;  citation_year=2009;  citation_volume=78;  citation_pages=242-51; " /><meta name="citation_reference" content="citation_title=A comprehensive expression analysis of all members of a gene family encoding cell-wall enzymes allowed us to predict cis-regulatory regions involved in cell-wall construction in specific organs of Arabidopsis; Plant Cell Physiol.;  citation_year=2001;  citation_volume=42;  citation_pages=1025-33; " /><meta name="citation_reference" content="citation_title=A surprising diversity and abundance of xyloglucan endotransglucosylase/hydrolases in rice. Classification and expression analysis; Plant Physiol.;  citation_year=2004;  citation_volume=134;  citation_pages=1088-99; " /><meta name="citation_reference" content="citation_title=Cloning, characterization, and expression of xyloglucan endotransglucosylase/hydrolase and expansin genes associated with petal growth and development during carnation flower opening; J. Exp. Bot.;  citation_year=2011;  citation_volume=62;  citation_pages=815-23; " /><meta name="citation_reference" content="citation_title=Analysis of scents emitted from flowers of interspecific hybrids between carnation and fragrant wild Dianthus species; J. Japan. Soc. Hort. Sci.;  citation_year=2013;  citation_volume=82;  citation_pages=145-53; " /><meta name="citation_reference" content="citation_title=The SABATH family of methyltransferases in Arabidopsis thaliana and other plant species; citation_publisher=Elsevier Science Ltd, Oxford; Recent Advances in Phytochemistry;  citation_year=2003;  citation_volume=vol. 37;  citation_pages=253-83; " /><meta name="citation_reference" content="citation_title=Positive selection for single amino acid change promotes substrate discrimination of a plant volatile-producing enzyme; Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=1320-9; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/3/231/1107121/dst053.pdf" /><meta name="description" content="Abstract.  The whole-genome sequence of carnation (Dianthus caryophyllus L.) cv. ‘Francesco’ was determined using a combination of different new-generation mult" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/3/231/393171" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/3/231/393171" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/3/231/393171");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst053';
    </script>
<title>
	Sequence Analysis of the Genome of Carnation (Dianthus caryophyllus L.) | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./393171" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BdiWATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFnaSwgTWFzYWZ1bWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOQVJPIEluc3RpdHV0ZSBvZiBGbG9yaWN1bHR1cmFsIFNjaWVuY2UgKE5JRlMpLCAyLTEgRnVqaW1vdG8sIFRzdWt1YmEsIEliYXJha2kgMzA1LTg1MTksIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS29zdWdpLCBTaHVuaWNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIaXJha2F3YSwgSGlkZWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9obWl5YSwgQWtlbWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOQVJPIEluc3RpdHV0ZSBvZiBGbG9yaWN1bHR1cmFsIFNjaWVuY2UgKE5JRlMpLCAyLTEgRnVqaW1vdG8sIFRzdWt1YmEsIEliYXJha2kgMzA1LTg1MTksIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFuYXNlLCBLb2ppIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTkFSTyBJbnN0aXR1dGUgb2YgRmxvcmljdWx0dXJhbCBTY2llbmNlIChOSUZTKSwgMi0xIEZ1amltb3RvLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS04NTE5LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhhcmFkYSwgVGFybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5BUk8gSW5zdGl0dXRlIG9mIEZsb3JpY3VsdHVyYWwgU2NpZW5jZSAoTklGUyksIDItMSBGdWppbW90bywgVHN1a3ViYSwgSWJhcmFraSAzMDUtODUxOSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLaXNoaW1vdG8sIEt5dXRhcm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOQVJPIEluc3RpdHV0ZSBvZiBGbG9yaWN1bHR1cmFsIFNjaWVuY2UgKE5JRlMpLCAyLTEgRnVqaW1vdG8sIFRzdWt1YmEsIEliYXJha2kgMzA1LTg1MTksIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFrYXlhbWEsIE1hc2F5b3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5BUk8gSW5zdGl0dXRlIG9mIEZsb3JpY3VsdHVyYWwgU2NpZW5jZSAoTklGUyksIDItMSBGdWppbW90bywgVHN1a3ViYSwgSWJhcmFraSAzMDUtODUxOSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJY2hpbXVyYSwgS2F6dW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOQVJPIEluc3RpdHV0ZSBvZiBGbG9yaWN1bHR1cmFsIFNjaWVuY2UgKE5JRlMpLCAyLTEgRnVqaW1vdG8sIFRzdWt1YmEsIEliYXJha2kgMzA1LTg1MTksIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT25vemFraSwgVGFrYXNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5BUk8gSW5zdGl0dXRlIG9mIEZsb3JpY3VsdHVyYWwgU2NpZW5jZSAoTklGUyksIDItMSBGdWppbW90bywgVHN1a3ViYSwgSWJhcmFraSAzMDUtODUxOSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZYW1hZ3VjaGksIEhpcm95YXN1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTkFSTyBJbnN0aXR1dGUgb2YgRmxvcmljdWx0dXJhbCBTY2llbmNlIChOSUZTKSwgMi0xIEZ1amltb3RvLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS04NTE5LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNhc2FraSwgTm9idWhpcm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUb2t5byBVbml2ZXJzaXR5IG9mIEFncmljdWx0dXJlIGFuZCBUZWNobm9sb2d5LCAyLTI0LTE2IE5ha2EtY2hvLCBLb2dhbmVpLCBUb2t5byAxODQtODU4OCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNaXlhaGFyYSwgVGFpcmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUb2t5byBVbml2ZXJzaXR5IG9mIEFncmljdWx0dXJlIGFuZCBUZWNobm9sb2d5LCAyLTI0LTE2IE5ha2EtY2hvLCBLb2dhbmVpLCBUb2t5byAxODQtODU4OCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOaXNoaXpha2ksIFl1em8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUb2t5byBVbml2ZXJzaXR5IG9mIEFncmljdWx0dXJlIGFuZCBUZWNobm9sb2d5LCAyLTI0LTE2IE5ha2EtY2hvLCBLb2dhbmVpLCBUb2t5byAxODQtODU4OCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPemVraSwgWW9zaGloaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVG9reW8gVW5pdmVyc2l0eSBvZiBBZ3JpY3VsdHVyZSBhbmQgVGVjaG5vbG9neSwgMi0yNC0xNiBOYWthLWNobywgS29nYW5laSwgVG9reW8gMTg0LTg1ODgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFrYW11cmEsIE5vcmlrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlJlc2VhcmNoIEluc3RpdHV0ZSwgU3VudG9yeSBHbG9iYWwgSW5ub3ZhdGlvbiBDZW50ZXIsIDEtMS0xIFdha2F5YW1hZGFpLCBTaGltYW1vdG8sIE1pc2hpbWEsIE9zYWthIDYxOC04NTAzLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlN1enVraSwgVGFrYW1hc2EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJKU1QsIEVSQVRPLCBIaWdhc2hpeWFtYSBMaXZlLUhvbG9uaWNzIFByb2plY3QsIE5hZ295YSBVbml2ZXJzaXR5LCBGdXJvLWNobywgQ2hpa3VzYS1rdSwgTmFnb3lhLCBBaWNoaSA0NjQtODYwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEaXZpc2lvbiBvZiBCaW9sb2dpY2FsIFNjaWVuY2UsIEdyYWR1YXRlIFNjaG9vbCBvZiBTY2llbmNlLCBOYWdveWEgVW5pdmVyc2l0eSwgRnVyby1jaG8sIENoaWt1c2Eta3UsIE5hZ295YSwgQWljaGkgNDY0LTg2MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFuYWthLCBZb3NoaWthenUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSZXNlYXJjaCBJbnN0aXR1dGUsIFN1bnRvcnkgR2xvYmFsIElubm92YXRpb24gQ2VudGVyLCAxLTEtMSBXYWtheWFtYWRhaSwgU2hpbWFtb3RvLCBNaXNoaW1hLCBPc2FrYSA2MTgtODUwMywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXRvLCBTaHVzZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpcmFzYXdhLCBLZW50YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJc29iZSwgU2FjaGlrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNaXlhbXVyYSwgWW9zaGlub3JpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IldhdGFuYWJlLCBBa2lrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWtheWFtYSwgU2hpbm9idSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLaXNoaWRhLCBZb3NoaWUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS29oYXJhLCBNaXRzdXlvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRhYmF0YSwgU2F0b3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IlNlcXVlbmNlIEFuYWx5c2lzIG9mIHRoZSBHZW5vbWUgb2YgQ2FybmF0aW9uIChEaWFudGh1cyBjYXJ5b3BoeWxsdXMgTC4pIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjMxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyNDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3QwNTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNC8wNi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1UaW1iZXIgUHJlc3MsIFBvcnRsYW5kOyBDYXJuYXRpb25zIGFuZCBQaW5rcyBmb3IgR2FyZGVuIGFuZCBHcmVlbmhvdXNlOyAgY2l0YXRpb25feWVhcj0xOTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWFudGh1cyBMaW5uYWV1czsgY2l0YXRpb25fcHVibGlzaGVyPUNhbWJyaWRnZSBVbml2ZXJzaXR5IFByZXNzLCBDYW1icmlkZ2U7IFRoZSBFdXJvcGVhbiBHYXJkZW4gRmxvcmE7ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl92b2x1bWU9dm9sLiBJSUk7ICBjaXRhdGlvbl9wYWdlcz0xODUtOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgZGl2ZXJzaXR5IG9mIERpYW50aHVzIGFjY2Vzc2lvbnMgYXMgYXNzZXNzZWQgdXNpbmcgdHdvIG1vbGVjdWxhciBtYXJrZXIgc3lzdGVtcyAoU1JBUHMgYW5kIElTU1JzKSBhbmQgbW9ycGhvbG9naWNhbCB0cmFpdHM7IFNjaS4gSG9ydGljLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMTc7ICBjaXRhdGlvbl9wYWdlcz0yNjMtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGFjdGl2ZSBoQVQgdHJhbnNwb3NhYmxlIGVsZW1lbnQgY2F1c2luZyBidWQgbXV0YXRpb24gb2YgY2FybmF0aW9uIGJ5IGluc2VydGlvbiBpbnRvIHRoZSBmbGF2b25vaWQgM+KAsi1oeWRyb3h5bGFzZSBnZW5lOyBNb2wuIEdlbmV0LiBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yODg7ICBjaXRhdGlvbl9wYWdlcz0xNzUtODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZsb3dlciBjb2xvdXIgYW5kIGN5dG9jaHJvbWVzIFA0NTA7IFBoaWwuIFRyYW5zLiBSLiBTb2MuIEIuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTM2ODsgIGNpdGF0aW9uX3BhZ2VzPTIwMTIwNDMyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Qmlvc3ludGhlc2lzIG9mIHBsYW50IHBpZ21lbnRzOiBhbnRob2N5YW5pbnMsIGJldGFsYWlucyBhbmQgY2Fyb3Rlbm9pZHM7IFBsYW50LCBKLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT01NDsgIGNpdGF0aW9uX3BhZ2VzPTczMy00OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1wcm92ZW1lbnQgb2YgZmxvd2VyIHZhc2UgbGlmZSB1c2luZyBjcm9zcy1icmVlZGluZyB0ZWNobmlxdWVzIGluIGNhcm5hdGlvbiAoRGlhbnRodXMgY2FyeW9waHlsbHVzIEwuKTsgSkFSUTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTEzNy00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VuZXNjZW5jZSBhbmQgcG9zdGhhcnZlc3QgcGh5c2lvbG9neSBvZiBjdXQgZmxvd2VycywgcGFydCAyOyBIb3J0LiBSZXYuOyAgY2l0YXRpb25feWVhcj0xOTgxOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz01OS0xNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNyb3NzaW5ncyBhbmQgc2VsZWN0aW9ucyBmb3Igc2l4IGdlbmVyYXRpb25zIGJhc2VkIG9uIGZsb3dlciB2YXNlIGxpZmUgdG8gY3JlYXRlIGxpbmVzIHdpdGggZXRoeWxlbmUgcmVzaXN0YW5jZSBvciB1bHRyYS1sb25nIHZhc2UgbGlmZSBpbiBjYXJuYXRpb25zIChEaWFudGh1cyBjYXJ5b3BoeWxsdXMgTC4pOyBKLiBKYXBhbi4gU29jLiBIb3J0LiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTgwOyAgY2l0YXRpb25fcGFnZXM9NDg2LTk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdWNyb3NlIHByZXZlbnRzIHVwLXJlZ3VsYXRpb24gb2Ygc2VuZXNjZW5jZS1hc3NvY2lhdGVkIGdlbmVzIGluIGNhcm5hdGlvbiBwZXRhbHM7IEouIEV4cC4gQm90LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT01ODsgIGNpdGF0aW9uX3BhZ2VzPTI4NzMtODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBhIGdlbmV0aWMgbGlua2FnZSBtYXAgZm9yIHRoZSBjYXJuYXRpb24gYnkgdXNpbmcgUkFQRCBhbmQgU1NSIG1hcmtlcnMgYW5kIG1hcHBpbmcgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgKFFUTCkgZm9yIHJlc2lzdGFuY2UgdG8gYmFjdGVyaWFsIHdpbHQgY2F1c2VkIGJ5IEJ1cmtob2xkZXJpYSBjYXJ5b3BoeWxsaTsgSi4gSmFwYW4uIFNvYy4gSG9ydC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03NTsgIGNpdGF0aW9uX3BhZ2VzPTE2Ni03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UVRMIGFuYWx5c2lzIGZvciByZXNpc3RhbmNlIHRvIGJhY3RlcmlhbCB3aWx0IChCdXJraG9sZGVyaWEgY2FyeW9waHlsbGkpIGluIGNhcm5hdGlvbiAoRGlhbnRodXMgY2FyeW9waHlsbHVzKSB1c2luZyBhbiBTU1ItYmFzZWQgZ2VuZXRpYyBsaW5rYWdlIG1hcDsgTW9sLiBCcmVlZC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz00OTUtNTA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0b21lIGFuYWx5c2lzIG9mIGNhcm5hdGlvbiAoRGlhbnRodXMgY2FyeW9waHlsbHVzIEwuKSBiYXNlZCBvbiBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZyB0ZWNobm9sb2d5OyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0yOTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgYSByZWZlcmVuY2UgZ2VuZXRpYyBsaW5rYWdlIG1hcCBmb3IgY2FybmF0aW9uIChEaWFudGh1cyBjYXJ5b3BoeWxsdXMgTC4pOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz03MzQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ZZWxsb3cgcGlua3M6IGludGVyc3BlY2lmaWMgaHlicmlkaXphdGlvbiBiZXR3ZWVuIERpYW50aHVzIHBsdW1hcml1cyBhbmQgcmVsYXRlZCBzcGVjaWVzIHdpdGggeWVsbG93IGZsb3dlcnM7IFNjaS4gSG9ydGljLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT03NzsgIGNpdGF0aW9uX3BhZ2VzPTIwNy0xODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZhbHVhdGlvbiBvZiBwbG9pZHkgbGV2ZWwgYW5kIGVuZG9yZWR1cGxpY2F0aW9uIGluIGNhcm5hdGlvbiAoRGlhbnRodXMgc3BwLik7IFBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjAsMeKAkzIwMjsgIGNpdGF0aW9uX3BhZ2VzPTEtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNhcm5hdGlvbjsgY2l0YXRpb25fYXV0aG9yPUphcGFuLiBTb2MuIEhvcnQuIFNjaTsgY2l0YXRpb25fcHVibGlzaGVyPVNob3VrYWRvaCBQdWJsaWNhdGlvbiwgRGVwdC4gb2YgUHVibGlzaGluZyBvZiBOYWthbmlzaGkgUHJpbnRpbmcgQ28uLCBMdGQuLCBLeW90bzsgSG9ydGljdWx0dXJlIGluIEphcGFuIDIwMDY7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl9wYWdlcz0yMjMtMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFwcGxpY2F0aW9uIG9mIEROQSBtYXJrZXJzIGZvciBicmVlZGluZyBjYXJuYXRpb25zIHJlc2lzdGFudCB0byBiYWN0ZXJpYWwgd2lsdDsgSkFSUTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00NzsgIGNpdGF0aW9uX3BhZ2VzPTI5LTM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbGx1bWluYSBtYXRlLXBhaXJlZCBETkEgc2VxdWVuY2luZy1saWJyYXJ5IHByZXBhcmF0aW9uIHVzaW5nIENyZS1Mb3ggcmVjb21iaW5hdGlvbjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9ZTI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXJhdGlvbiBvZiBsb25nIGluc2VydCBwYWlycyB1c2luZyBhIENyZS1Mb3hQIGludmVyc2UgUENSIGFwcHJvYWNoOyBQTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTI5NDM3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q09QRTogYW4gYWNjdXJhdGUgay1tZXItYmFzZWQgcGFpci1lbmQgcmVhZHMgY29ubmVjdGlvbiB0b29sIHRvIGZhY2lsaXRhdGUgZ2Vub21lIGFzc2VtYmx5OyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTI4NzAtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5oZXJpdGFuY2UgaW4gdGhlIGNhcm5hdGlvbiAoRGlhbnRodXMgY2FyeW9waHlsbHVzKS4gSUlJLiBJbmhlcml0YW5jZSBvZiBmbG93ZXIgY29sb3I7IEFubi4gTWlzc291cmkgQm90LiBHYXJkLjsgIGNpdGF0aW9uX3llYXI9MTk0NzsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPTM5LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeGNpc2lvbiBvZiB0cmFuc3Bvc2FibGUgZWxlbWVudHMgZnJvbSB0aGUgY2hhbGNvbmUgaXNvbWVyYXNlIGFuZCBkaWh5ZHJvZmxhdm9ub2wgNC1yZWR1Y3Rhc2UgZ2VuZXMgbWF5IGNvbnRyaWJ1dGUgdG8gdGhlIHZhcmllZ2F0aW9uIG9mIHRoZSB5ZWxsb3ctZmxvd2VyZWQgY2FybmF0aW9uIChEaWFudGh1cyBjYXJ5b3BoeWxsdXMpOyBQbGFudCBDZWxsIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9NTc4LTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiB0aGUgZ2x1dGF0aGlvbmUgUy10cmFuc2ZlcmFzZSBnZW5lIHJlc3BvbnNpYmxlIGZvciBmbG93ZXIgY29sb3IgaW50ZW5zaXR5IGluIGNhcm5hdGlvbnM7IFBsYW50IEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MjIzLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyZWVWaWV3OiBhbiBhcHBsaWNhdGlvbiB0byBkaXNwbGF5IHBoeWxvZ2VuZXRpYyB0cmVlcyBvbiBwZXJzb25hbCBjb21wdXRlcnM7IENvbXB1dC4gQXBwbC4gQmlvc2NpLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTM1Ny04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIGFuZCBmdW5jdGlvbmFsIGlkZW50aWZpY2F0aW9uIG9mIGEgbm92ZWwgcGxhbnQgNCw1LWV4dHJhZGlvbCBkaW94eWdlbmFzZSBpbnZvbHZlZCBpbiBiZXRhbGFpbiBwaWdtZW50IGJpb3N5bnRoZXNpcyBpbiBQb3J0dWxhY2EgZ3JhbmRpZmxvcmE7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEzNDsgIGNpdGF0aW9uX3BhZ2VzPTI2NS03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGJlZXQgUiBsb2N1cyBlbmNvZGVzIGEgbmV3IGN5dG9jaHJvbWUgUDQ1MCByZXF1aXJlZCBmb3IgcmVkIGJldGFsYWluIHByb2R1Y3Rpb247IE5hdC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9ODE2LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlY3Rpb24gb2YgVURQLWdsdWNvc2U6Y3ljbG8tRE9QQSA1LU8tZ2x1Y29zeWx0cmFuc2ZlcmFzZSBhY3Rpdml0eSBpbiBmb3VyIG8nY2xvY2tzIChNaXJhYmlsaXMgamFsYXBhIEwuKTsgRkVCUyBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01Njg7ICBjaXRhdGlvbl9wYWdlcz0xNTktNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzb2xhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBjRE5BcyBlbmNvZGluZyBhbiBlbnp5bWUgd2l0aCBnbHVjb3N5bHRyYW5zZmVyYXNlIGFjdGl2aXR5IGZvciBjeWNsby1ET1BBIGZyb20gZm91ciBv4oCZY2xvY2tzIGFuZCBmZWF0aGVyIGNvY2tzY29tYnM7IFBsYW50IENlbGwgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NDY7ICBjaXRhdGlvbl9wYWdlcz02NjYtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBjaGFyYWN0ZXJpemF0aW9uIG9mIHRoZSBBcmFiaWRvcHNpcyA5LWNpcy1lcG94eWNhcm90ZW5vaWQgZGlveHlnZW5hc2UgZ2VuZSBmYW1pbHk7IFBsYW50IEouOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9NDQtNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIGNhcm90ZW5vZ2VuaWMgZ2VuZSBleHByZXNzaW9uIGluIHBldGFscyBhbmQgbGVhdmVzIG9mIGNhcm5hdGlvbiAoRGlhbnRodXMgY2FyeW9waHlsbHVzIEwuKTsgUGxhbnQgQnJlZWQuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTEzMjsgIGNpdGF0aW9uX3BhZ2VzPTQyMy05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBvZiBOQlMtZW5jb2RpbmcgZGlzZWFzZSByZXNpc3RhbmNlIGdlbmVzIGluIEN1Y3VtaXMgc2F0aXZ1cyBhbmQgcGh5bG9nZW5pYyBzdHVkeSBvZiBOQlMtZW5jb2RpbmcgZ2VuZXMgaW4gQ3VjdXJiaXRhY2VhZSBjcm9wczsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTA5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBvZiBtZWxvbiAoQ3VjdW1pcyBtZWxvIEwuKTsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTA5OyAgY2l0YXRpb25fcGFnZXM9MTE4NzItNzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFuYWx5c2lzIG9mIENhcmljYSBwYXBheWEgcmV2ZWFscyBhIHNtYWxsIE5CUyByZXNpc3RhbmNlIGdlbmUgZmFtaWx5OyBNb2wuIEdlbmV0LiBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yODE7ICBjaXRhdGlvbl9wYWdlcz02MDktMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU92ZXJ2aWV3IG9mIHRvbWF0byAoU29sYW51bSBseWNvcGVyc2ljdW0pIGNhbmRpZGF0ZSBwYXRob2dlbiByZWNvZ25pdGlvbiBnZW5lcyByZXZlYWxzIGltcG9ydGFudCBTb2xhbnVtIFIgbG9jdXMgZHluYW1pY3M7IE5ldyBQaHl0b2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE5NzsgIGNpdGF0aW9uX3BhZ2VzPTIyMy0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoaWdoIHF1YWxpdHkgZHJhZnQgY29uc2Vuc3VzIHNlcXVlbmNlIG9mIHRoZSBnZW5vbWUgb2YgYSBoZXRlcm96eWdvdXMgZ3JhcGV2aW5lIHZhcmlldHk7IFBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz1lMTMyNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGlkZW50aWZpY2F0aW9uIG9mIE5CUyBnZW5lcyBpbiBqYXBvbmljYSByaWNlIHJldmVhbHMgc2lnbmlmaWNhbnQgZXhwYW5zaW9uIG9mIGRpdmVyZ2VudCBub24tVElSIE5CUy1MUlIgZ2VuZXM7IE1vbC4gR2VuZXQuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTI3MTsgIGNpdGF0aW9uX3BhZ2VzPTQwMi0xNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGFic2VuY2Ugb2YgVElSLXR5cGUgcmVzaXN0YW5jZSBnZW5lIGFuYWxvZ3VlcyBpbiB0aGUgc3VnYXIgYmVldCAoQmV0YSB2dWxnYXJpcyBMLikgZ2Vub21lOyBKLiBNb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU4OyAgY2l0YXRpb25fcGFnZXM9NDAtNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZmZlcmVudGlhbCBleHByZXNzaW9uIG9mIHRocmVlIG1lbWJlcnMgb2YgdGhlIDEtYW1pbm9jeWNsb3Byb3BhbmUtMS1jYXJib3h5bGF0ZSBzeW50aGFzZSBnZW5lIGZhbWlseSBpbiBjYXJuYXRpb247IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTExOTsgIGNpdGF0aW9uX3BhZ2VzPTc1NS02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlvY2hlbWljYWwgZGl2ZXJzaXR5IGFtb25nIHRoZSAxLWFtaW5vLWN5Y2xvcHJvcGFuZS0xLWNhcmJveHlsYXRlIHN5bnRoYXNlIGlzb3p5bWVzIGVuY29kZWQgYnkgdGhlIEFyYWJpZG9wc2lzIGdlbmUgZmFtaWx5OyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yNzg7ICBjaXRhdGlvbl9wYWdlcz00OTEwMi0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgcGluaXRvbCBhcyBhIG1haW4gc3VnYXIgY29uc3RpdHVlbnQgYW5kIGNoYW5nZXMgaW4gaXRzIGNvbnRlbnQgZHVyaW5nIGZsb3dlciBidWQgZGV2ZWxvcG1lbnQgaW4gY2FybmF0aW9uIChEaWFudGh1cyBjYXJ5b3BoeWxsdXMgTC4pOyBKLiBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNTI7ICBjaXRhdGlvbl9wYWdlcz0zNjMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW5kby14eWxvZ2x1Y2FuIHRyYW5zZmVyYXNlLCBhIG5vdmVsIGNsYXNzIG9mIGdseWNvc3lsdHJhbnNmZXJhc2UgdGhhdCBjYXRhbHl6ZXMgdHJhbnNmZXIgb2YgYSBzZWdtZW50IG9mIHh5bG9nbHVjYW4gbW9sZWN1bGUgdG8gYW5vdGhlciB4eWxvZ2x1Y2FuIG1vbGVjdWxlOyAsIEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTI2NzsgIGNpdGF0aW9uX3BhZ2VzPTIxMDU4LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DZWxsIHdhbGwgZXh0ZW5zaWJpbGl0eSBhbmQgZWZmZWN0IG9mIGNlbGwtd2FsbC1sb29zZW5pbmcgcHJvdGVpbnMgZHVyaW5nIHJvc2UgZmxvd2VyIG9wZW5pbmc7IEouIEphcGFuLiBTb2MuIEhvcnQuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Nzg7ICBjaXRhdGlvbl9wYWdlcz0yNDItNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29tcHJlaGVuc2l2ZSBleHByZXNzaW9uIGFuYWx5c2lzIG9mIGFsbCBtZW1iZXJzIG9mIGEgZ2VuZSBmYW1pbHkgZW5jb2RpbmcgY2VsbC13YWxsIGVuenltZXMgYWxsb3dlZCB1cyB0byBwcmVkaWN0IGNpcy1yZWd1bGF0b3J5IHJlZ2lvbnMgaW52b2x2ZWQgaW4gY2VsbC13YWxsIGNvbnN0cnVjdGlvbiBpbiBzcGVjaWZpYyBvcmdhbnMgb2YgQXJhYmlkb3BzaXM7IFBsYW50IENlbGwgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz0xMDI1LTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHN1cnByaXNpbmcgZGl2ZXJzaXR5IGFuZCBhYnVuZGFuY2Ugb2YgeHlsb2dsdWNhbiBlbmRvdHJhbnNnbHVjb3N5bGFzZS9oeWRyb2xhc2VzIGluIHJpY2UuIENsYXNzaWZpY2F0aW9uIGFuZCBleHByZXNzaW9uIGFuYWx5c2lzOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzQ7ICBjaXRhdGlvbl9wYWdlcz0xMDg4LTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbG9uaW5nLCBjaGFyYWN0ZXJpemF0aW9uLCBhbmQgZXhwcmVzc2lvbiBvZiB4eWxvZ2x1Y2FuIGVuZG90cmFuc2dsdWNvc3lsYXNlL2h5ZHJvbGFzZSBhbmQgZXhwYW5zaW4gZ2VuZXMgYXNzb2NpYXRlZCB3aXRoIHBldGFsIGdyb3d0aCBhbmQgZGV2ZWxvcG1lbnQgZHVyaW5nIGNhcm5hdGlvbiBmbG93ZXIgb3BlbmluZzsgSi4gRXhwLiBCb3QuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTYyOyAgY2l0YXRpb25fcGFnZXM9ODE1LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBzY2VudHMgZW1pdHRlZCBmcm9tIGZsb3dlcnMgb2YgaW50ZXJzcGVjaWZpYyBoeWJyaWRzIGJldHdlZW4gY2FybmF0aW9uIGFuZCBmcmFncmFudCB3aWxkIERpYW50aHVzIHNwZWNpZXM7IEouIEphcGFuLiBTb2MuIEhvcnQuIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODI7ICBjaXRhdGlvbl9wYWdlcz0xNDUtNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBTQUJBVEggZmFtaWx5IG9mIG1ldGh5bHRyYW5zZmVyYXNlcyBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYSBhbmQgb3RoZXIgcGxhbnQgc3BlY2llczsgY2l0YXRpb25fcHVibGlzaGVyPUVsc2V2aWVyIFNjaWVuY2UgTHRkLCBPeGZvcmQ7IFJlY2VudCBBZHZhbmNlcyBpbiBQaHl0b2NoZW1pc3RyeTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT12b2wuIDM3OyAgY2l0YXRpb25fcGFnZXM9MjUzLTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3NpdGl2ZSBzZWxlY3Rpb24gZm9yIHNpbmdsZSBhbWlubyBhY2lkIGNoYW5nZSBwcm9tb3RlcyBzdWJzdHJhdGUgZGlzY3JpbWluYXRpb24gb2YgYSBwbGFudCB2b2xhdGlsZS1wcm9kdWNpbmcgZW56eW1lOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MTMyMC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjEvMy8yMzEvMTEwNzEyMS9kc3QwNTMucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFRoZSB3aG9sZS1nZW5vbWUgc2VxdWVuY2Ugb2YgY2FybmF0aW9uIChEaWFudGh1cyBjYXJ5b3BoeWxsdXMgTC4pIGN2LiDigJhGcmFuY2VzY2/igJkgd2FzIGRldGVybWluZWQgdXNpbmcgYSBjb21iaW5hdGlvbiBvZiBkaWZmZXJlbnQgbmV3LWdlbmVyYXRpb24gbXVsdCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzkzMTcxJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZJpSm171Qv4Fx388D/e9PjuVetAX" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f3%2f231%2f393171" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f3%2f231%2f393171" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=15369" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/3/1/m_cover.gif?Expires=2147483647&amp;Signature=eD74u2chqR7m7Q4xVEpu-B2CnS8a5UZybuUBj9IiQcs7vcn-sLThVTH2YkcAPlEiODArsSqGH041ZPg5RakmGwkyHxqVC4LdBdXxFui4WCeLWHefwauec7mA5UKsie6Bk-ZRp7yPg10FfeUTBrjN7buCiodIP00VlbrJgVNTJFuLyaUUxPgJ3yahCwwX4TAcK2CRCX-3Y~LqokrGvJCW5URLAAo4GxWdmNUAJqBhdMREkIh~McyOhMfYKJNLDTxW9vnvEbKMxNW5048gQKGgF7kMxJ3~NsfswZCCrwYr2quaHFHHORZM0v4Eh4XRYgKpFeDHa6jnY4OeJPxjcOPrMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/3">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5799038">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5799038">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5799042">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5799042">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5799051">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5799051">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5799082">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5799082">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5799152">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5799152">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5799155">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5799155">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5799158">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5799158">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/21/3/243/393500">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Sequence Analysis of the Genome of Carnation (<em>Dianthus caryophyllus</em> L.)
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masafumi Yagi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">NARO Institute of Floricultural Science (NIFS)</div>, <div class="addr-line">2-1 Fujimoto, Tsukuba, Ibaraki 305-8519</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masafumi+Yagi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yagi M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yagi M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Masafumi Yagi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shunichi Kosugi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shunichi+Kosugi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kosugi S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kosugi S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shunichi Kosugi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Hirakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Hirakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirakawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirakawa H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hideki Hirakawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akemi Ohmiya</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">NARO Institute of Floricultural Science (NIFS)</div>, <div class="addr-line">2-1 Fujimoto, Tsukuba, Ibaraki 305-8519</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akemi+Ohmiya">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohmiya A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohmiya A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Akemi Ohmiya</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Koji Tanase</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">NARO Institute of Floricultural Science (NIFS)</div>, <div class="addr-line">2-1 Fujimoto, Tsukuba, Ibaraki 305-8519</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Koji+Tanase">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tanase K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tanase K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Koji Tanase</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Taro Harada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">NARO Institute of Floricultural Science (NIFS)</div>, <div class="addr-line">2-1 Fujimoto, Tsukuba, Ibaraki 305-8519</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Taro+Harada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Harada T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Harada T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Taro Harada</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kyutaro Kishimoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">NARO Institute of Floricultural Science (NIFS)</div>, <div class="addr-line">2-1 Fujimoto, Tsukuba, Ibaraki 305-8519</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kyutaro+Kishimoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kishimoto K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kishimoto K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kyutaro Kishimoto</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masayoshi Nakayama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">NARO Institute of Floricultural Science (NIFS)</div>, <div class="addr-line">2-1 Fujimoto, Tsukuba, Ibaraki 305-8519</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masayoshi+Nakayama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakayama M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakayama M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Masayoshi Nakayama</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazuo Ichimura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">NARO Institute of Floricultural Science (NIFS)</div>, <div class="addr-line">2-1 Fujimoto, Tsukuba, Ibaraki 305-8519</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kazuo+Ichimura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ichimura K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ichimura K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kazuo Ichimura</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takashi Onozaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">NARO Institute of Floricultural Science (NIFS)</div>, <div class="addr-line">2-1 Fujimoto, Tsukuba, Ibaraki 305-8519</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takashi+Onozaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Onozaki T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Onozaki T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Takashi Onozaki</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroyasu Yamaguchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">NARO Institute of Floricultural Science (NIFS)</div>, <div class="addr-line">2-1 Fujimoto, Tsukuba, Ibaraki 305-8519</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroyasu+Yamaguchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamaguchi H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamaguchi H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Hiroyasu Yamaguchi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nobuhiro Sasaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Tokyo University of Agriculture and Technology</div>, <div class="addr-line">2-24-16 Naka-cho, Koganei, Tokyo 184-8588</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nobuhiro+Sasaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sasaki N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sasaki N&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Nobuhiro Sasaki</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Taira Miyahara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Tokyo University of Agriculture and Technology</div>, <div class="addr-line">2-24-16 Naka-cho, Koganei, Tokyo 184-8588</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Taira+Miyahara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Miyahara T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Miyahara T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Taira Miyahara</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuzo Nishizaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Tokyo University of Agriculture and Technology</div>, <div class="addr-line">2-24-16 Naka-cho, Koganei, Tokyo 184-8588</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuzo+Nishizaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nishizaki Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nishizaki Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yuzo Nishizaki</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshihiro Ozeki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Tokyo University of Agriculture and Technology</div>, <div class="addr-line">2-24-16 Naka-cho, Koganei, Tokyo 184-8588</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshihiro+Ozeki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ozeki Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ozeki Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yoshihiro Ozeki</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Noriko Nakamura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Research Institute</div>, <div class="insititution">Suntory Global Innovation Center</div>, <div class="addr-line">1-1-1 Wakayamadai, Shimamoto, Mishima, Osaka 618-8503</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Noriko+Nakamura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakamura N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakamura N&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Noriko Nakamura</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takamasa Suzuki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">JST, ERATO, Higashiyama Live-Holonics Project</div>, <div class="insititution">Nagoya University</div>, <div class="addr-line">Furo-cho, Chikusa-ku, Nagoya, Aichi 464-8602</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">6</div><div class="addr-line">Division of Biological Science, Graduate School of Science</div>, <div class="insititution">Nagoya University</div>, <div class="addr-line">Furo-cho, Chikusa-ku, Nagoya, Aichi 464-8602</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takamasa+Suzuki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suzuki T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suzuki T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Takamasa Suzuki</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshikazu Tanaka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Research Institute</div>, <div class="insititution">Suntory Global Innovation Center</div>, <div class="addr-line">1-1-1 Wakayamadai, Shimamoto, Mishima, Osaka 618-8503</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshikazu+Tanaka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tanaka Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tanaka Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yoshikazu Tanaka</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shusei Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shusei+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Shusei Sato</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenta Shirasawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenta+Shirasawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shirasawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shirasawa K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Kenta Shirasawa</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sachiko Isobe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sachiko+Isobe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Isobe S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Isobe S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Sachiko Isobe</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshinori Miyamura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshinori+Miyamura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Miyamura Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Miyamura Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yoshinori Miyamura</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akiko Watanabe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akiko+Watanabe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Watanabe A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Watanabe A&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Akiko Watanabe</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shinobu Nakayama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shinobu+Nakayama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakayama S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakayama S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Shinobu Nakayama</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshie Kishida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshie+Kishida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kishida Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kishida Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yoshie Kishida</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mitsuyo Kohara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mitsuyo+Kohara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kohara M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kohara M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Mitsuyo Kohara</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Tabata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81-438-52-3933. Fax. +81-438-52-3934. Email: <a href="mailto:tabata@kazusa.or.jp" target="_blank">tabata@kazusa.or.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Tabata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tabata S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tabata S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Satoshi Tabata</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors equally contributed to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="label -label">‡</div><div class="footnote-content">Present address: Iwate Biotechnology Research Center, 22-174-4 Narita, Kitakami, Iwate 024-0003, Japan.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN3"><div class="fn"><div class="label -label">§</div><div class="footnote-content">Present address: Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Aoba-ku, Sendai, Miyagi 980-8577, Japan.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 3, 1 June 2014, Pages 231–241, <a href='https://doi.org/10.1093/dnares/dst053'>https://doi.org/10.1093/dnares/dst053</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">17 December 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">11 October 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">04 November 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="393171" href="/dnaresearch/article-pdf/21/3/231/1107121/dst053.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Masafumi Yagi, Shunichi Kosugi, Hideki Hirakawa, Akemi Ohmiya, Koji Tanase, Taro Harada, Kyutaro Kishimoto, Masayoshi Nakayama, Kazuo Ichimura, Takashi Onozaki, Hiroyasu Yamaguchi, Nobuhiro Sasaki, Taira Miyahara, Yuzo Nishizaki, Yoshihiro Ozeki, Noriko Nakamura, Takamasa Suzuki, Yoshikazu Tanaka, Shusei Sato, Kenta Shirasawa, Sachiko Isobe, Yoshinori Miyamura, Akiko Watanabe, Shinobu Nakayama, Yoshie Kishida, Mitsuyo Kohara, Satoshi Tabata;  Sequence Analysis of the Genome of Carnation (<em>Dianthus caryophyllus</em> L.), <em>DNA Research</em>, Volume 21, Issue 3, 1 June 2014, Pages 231–241, <a href='https://doi.org/10.1093/dnares/dst053'>https://doi.org/10.1093/dnares/dst053</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/393171?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/393171?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/393171?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/393171?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/393171?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/393171?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/393171?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Sequence%20Analysis%20of%20the%20Genome%20of%20Carnation%20(%26lt%3Bem%26gt%3BDianthus%20caryophyllus%26lt%3B%2Fem%26gt%3B%20L.)&amp;publicationDate=2013-12-17&amp;volumeNum=21&amp;issueNum=3&amp;author=Yagi%2C%20Masafumi%3B%20Kosugi%2C%20Shunichi&amp;startPage=231&amp;endPage=241&amp;contentId=10.1093%2Fdnares%2Fdst053&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f3%2f231%2f393171" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5799038 id="5799038" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The whole-genome sequence of carnation (<em>Dianthus caryophyllus</em> L.) cv. ‘Francesco’ was determined using a combination of different new-generation multiplex sequencing platforms. The total length of the non-redundant sequences was 568 887 315 bp, consisting of 45 088 scaffolds, which covered 91% of the 622 Mb carnation genome estimated by k-mer analysis. The N50 values of contigs and scaffolds were 16 644 bp and 60 737 bp, respectively, and the longest scaffold was 1 287 144 bp. The average GC content of the contig sequences was 36%. A total of 1050, 13, 92 and 143 genes for tRNAs, rRNAs, snoRNA and miRNA, respectively, were identified in the assembled genomic sequences. For protein-encoding genes, 43 266 complete and partial gene structures excluding those in transposable elements were deduced. Gene coverage was ∼98%, as deduced from the coverage of the core eukaryotic genes. Intensive characterization of the assigned carnation genes and comparison with those of other plant species revealed characteristic features of the carnation genome. The results of this study will serve as a valuable resource for fundamental and applied research of carnation, especially for breeding new carnation varieties. Further information on the genomic sequences is available at <a class="link link-uri" href="http://carnation.kazusa.or.jp">http://carnation.kazusa.or.jp</a>.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Dianthus caryophyllus L.&quot;"><em>Dianthus caryophyllus</em> L.</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="carnation">carnation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome sequencing&quot;">genome sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene prediction&quot;">gene prediction</a></div>                            <h2 scrollto-destination=5799042 id="5799042" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Carnation (<em>Dianthus caryophyllus</em> L.) is one of the major floricultural crops in Japan and worldwide. It is a member of the family Caryophyllaceae and belongs to the genus <em>Dianthus</em>. More than 300 <em>Dianthus</em> species have been recorded.<sup><a reveal-id="DST053C1" data-open="DST053C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Many <em>Dianthus</em> species are distributed throughout Europe and Asia, and the distribution of this genus extends to arctic North America and to mountainous sites in Africa.<sup><a reveal-id="DST053C2" data-open="DST053C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Several species, including <em>D. caryophyllus</em>, <em>D. barbatus</em>, <em>D. chinensis</em>, <em>D. plumarius</em>, <em>D. superbus</em> and their hybrids are widely used as horticultural cultivars.<sup><a reveal-id="DST053C3" data-open="DST053C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Many new carnations have been bred for attractive characteristics such as flower colour, flower size, fragrance and flower longevity.</p><p>The pigments in carnation flowers are mainly anthocyanin and chalcone derivatives, and most of the genes involved in pigment biosynthesis in carnation have been identified.<sup><a reveal-id="DST053C4" data-open="DST053C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Due to the absence of flavonoid 3′,5′-hydroxylase (F3′,5′H; a key enzyme in the synthesis of delphinidin) in carnation, blue or violet flowers have never occurred in carnations. The introduction of a petunia or pansy <em>F3′,5′H</em> gene into carnation has led to the creation of blue or violet transgenic carnations, which are commercially available.<sup><a reveal-id="DST053C5" data-open="DST053C5" class="link link-ref link-reveal xref-bibr">5</a></sup> The plant pigments of species belonging to the families of Caryophyllales (except for Caryophyllaceae and Molluginaceae) are betalains, which have never been detected with anthocyanins in the same species.<sup><a reveal-id="DST053C6" data-open="DST053C6" class="link link-ref link-reveal xref-bibr">6</a></sup> The carnation, exceptionally bearing anthocyanins in Caryophyllales, is one of the attractive materials to study evolution of genetic systems for pigment synthesis.</p><p>The vase life of cut flowers, or flower longevity, is one of the most important characteristics to carnation.<sup><a reveal-id="DST053C7" data-open="DST053C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Carnation flowers are highly sensitive to ethylene, which induces autocatalytic ethylene production and wilting in carnation petals.<sup><a reveal-id="DST053C8" data-open="DST053C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Conventional cross-breeding techniques have succeeded in improving the vase life of the carnation flower,<sup><a reveal-id="DST053C9" data-open="DST053C9" class="link link-ref link-reveal xref-bibr">9</a></sup> which is a polygenic trait that is controlled by several genes involved in ethylene production and ethylene sensitivity.<sup><a reveal-id="DST053C9" data-open="DST053C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DST053C10" data-open="DST053C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p><p>To clarify the genetic and physiological mechanisms of agriculturally important traits, and to apply this information to actual breeding, a number of genetic and molecular tools have been developed. Genetic linkage maps of the carnation genome have been constructed and used to identify quantitative trait loci (QTL) responsible for resistance to carnation bacterial wilt.<sup><a reveal-id="DST053C11" data-open="DST053C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DST053C12" data-open="DST053C12" class="link link-ref link-reveal xref-bibr">12</a></sup> With the aid of next-generation sequencing (NGS) technology, large-scale transcriptome analysis (RNA-seq) has been conducted, revealing 300 740 unigenes consisting of 37 844 contigs and 262 896 singletons.<sup><a reveal-id="DST053C13" data-open="DST053C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Recently, we constructed a reference genetic linkage map for carnation using simple sequence repeat (SSR) markers derived from this RNA-seq analysis.<sup><a reveal-id="DST053C14" data-open="DST053C14" class="link link-ref link-reveal xref-bibr">14</a></sup></p><p>Most carnation cultivars are diploid, with a chromosome number of 2<em>n</em> = 2<em>x</em> = 30.<sup><a reveal-id="DST053C15" data-open="DST053C15" class="link link-ref link-reveal xref-bibr">15</a></sup> The reported nuclear DNA content of carnation is 1.23–1.48 pg/2C, which indicates that the carnation has a comparatively small nuclear genome approximately four times the size of the <em>Arabidopsis thaliana</em> nuclear genome.<sup><a reveal-id="DST053C16" data-open="DST053C16" class="link link-ref link-reveal xref-bibr">16</a></sup> The estimated genome size of carnation (670 Mb)<sup><a reveal-id="DST053C16" data-open="DST053C16" class="link link-ref link-reveal xref-bibr">16</a></sup> is small compared with those of other ornamental flowers, such as <em>Rosa hybrida</em> (1.1 Gb), <em>Antirrhinum majus</em> (1.5 Gb), <em>Petunia hybrida</em> (1.6 Gb), <em>Chrysanthemum morifolium</em> (9.4 Gb) and <em>Tulipa gesneriana</em> (26 Gb), according to the Plant C-values database (<a class="link link-uri" href="http://data.kew.org/cvalues/">http://data.kew.org/cvalues/</a>). To understand the genetic systems of carnation and to accelerate the process of molecular breeding, we performed structural analysis of the whole genome of carnation for the first time in ornamentals. The information and material resources for the carnation genome generated in this study should enhance both fundamental and applied studies of carnations and related plants.</p>                            <h2 scrollto-destination=5799051 id="5799051" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5799053 id="5799053" class="section-title" data-legacy-id=s2a>2.1. Plant materials</h3>
<p>The carnation cultivars, ‘Francesco’ (for genome sequencing) and ‘Karen Rouge’ (for BAC construction) were grown under natural daylight conditions in a greenhouse in NIFS. ‘Francesco’, a red Mediterranean standard-type cultivar, is the leading cultivar in Japan,<sup><a reveal-id="DST053C17" data-open="DST053C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and ‘Karen Rouge’ is a cultivar with bacterial wilt resistance derived from <em>D. capitatus</em> ssp. <em>andrzejowskianus</em>.<sup><a reveal-id="DST053C18" data-open="DST053C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p>                            <h3 scrollto-destination=5799056 id="5799056" class="section-title" data-legacy-id=s2b>2.2. Construction of BAC libraries and BAC DNA sequencing</h3>
<p>BAC libraries were constructed from nuclear DNA prepared from young leaves of ‘Karen Rouge’. Nuclear DNA was partially digested with <em>Hin</em>dIII and size-selected, and 100–180 kb DNA was ligated to the BAC vector pIndigoBAC5 (Epicentre Biotechnologies, WI, USA) and introduced into <em>Escherichia coli</em> ElectroMAX DH10B cells (Life Technologies Co., CA, USA) by electroporation.</p><p>For shotgun sequencing of BAC clones, BAC DNAs prepared according to the standard procedure were fragmented by nebulization, barcoded with a GS Titanium Rapid Library MID Adaptors Kit (Roche Diagnostics, IN, USA), and pooled for sequencing using the GS Titanium platform (Roche Diagnostics) according to the manufacturer's instructions.</p>                            <h3 scrollto-destination=5799060 id="5799060" class="section-title" data-legacy-id=s2c>2.3. Shotgun sequencing of the carnation genome</h3>
<p>Whole-genome shotgun sequencing of the cultivar ‘Francesco’ was performed using both HiSeq 1000 (Illumina Inc., CA, USA) and GS FLX+ (Roche Diagnostics) sequencers. Genomic DNA extracted from leaves was used for library construction according to standard protocols. The libraries included paired-end (PE) (insert size: 500 bp) and overlapping fragment (OF) (insert size: 180 bp) libraries for the HiSeq 1000 sequencer, and single-end (SE) and PE libraries (insert size: 4 kb) for the GS FLX+ system. In addition, two Illumina mate-pair (MP) libraries with 3 and 5 kb inserts, respectively, were constructed with GS Titanium Library Paired End Adaptors (Roche Diagnostics) as previously described.<sup><a reveal-id="DST053C19" data-open="DST053C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DST053C20" data-open="DST053C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The sequence data collection is summarized in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>.</p>                            <h3 scrollto-destination=5799062 id="5799062" class="section-title" data-legacy-id=s2d>2.4. Sequence assembly and evaluation of authenticity</h3>
<p>A method taken for assembly of the genomic sequences of carnation is summarized in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>. Sequence assembly for the BAC DNAs was performed using Newbler ver. 2.7 (Roche Diagnostics). Authenticity of the assembled genomic sequences described above was examined by aligning the sequences of subcontigs contained in the scaffolds of the genomic assemblies with the contig sequences of the BAC clones using the Bowtie 2 program. References for computer programs and databases are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>.</p>                            <h3 scrollto-destination=5799064 id="5799064" class="section-title" data-legacy-id=s2e>2.5. cDNA sequences</h3>
<p>Carnation cDNA sequences generated by Sanger sequencing (accession numbers: FY382825–FY405424)<sup><a reveal-id="DST053C12" data-open="DST053C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and GS FLX+ sequencing (accession numbers: FX296474–FX334317)<sup><a reveal-id="DST053C13" data-open="DST053C13" class="link link-ref link-reveal xref-bibr">13</a></sup> were retrieved from NCBI GenBank (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/genbank/">http://www.ncbi.nlm.nih.gov/genbank/</a>). In order to generate a non-redundant cDNA data set, redundant cDNA sequences were removed with a CD-HIT tool.</p>                            <h3 scrollto-destination=5799067 id="5799067" class="section-title" data-legacy-id=s2f>2.6. Detection of repetitive sequences</h3>
<p>Known repetitive sequences, including transposable elements (TEs), were detected with the RepeatMasker (<a class="link link-uri" href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>) and TransposonPSI (<a class="link link-uri" href="http://transposonpsi.sourceforge.net">http://transposonpsi.sourceforge.net</a>) programs, and novel repeats were detected with RepeatScout and Piler. To exclude protein-coding genes, the novel repeat library was searched against the SWISS-PROT database (<a class="link link-uri" href="http://www.uniprot.org">http://www.uniprot.org</a>) with BLASTX. The known and novel repeat sets were merged and redundant repeats were removed.</p>                            <h3 scrollto-destination=5799070 id="5799070" class="section-title" data-legacy-id=s2g>2.7. Assignment of RNA-coding genes</h3>
<p>Genes for tRNAs were assigned using the tRNAscan-SE program. The rRNA genes were identified based on sequence similarity with those of <em>A. thaliana</em>. Genes for small nucleolar RNA (snoRNA) were predicted using snoScan. Micro RNA (miRNA) genes were searched against a miRBase library that contained plant miRNA sequences detected by the MapMi program, which assigns miRNA precursor sequences in genomes by a combination of sequence alignment with Bowtie and prediction of RNA-secondary structures with RNAfold. References for computer programs and databases are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>.</p>                            <h3 scrollto-destination=5799073 id="5799073" class="section-title" data-legacy-id=s2h>2.8. Assignment of protein-encoding genes</h3>
<p>Two programs were used for the assignment of protein-encoding genes in the carnation genome: PASA (<a class="link link-uri" href="http://pasa.sourceforge.net">http://pasa.sourceforge.net</a>), based on cDNA alignment and Augustus, based on <em>ab initio</em> gene prediction incorporating cDNA alignment information. The protein-encoding genes were first predicted using PASA and then, Augustus was trained with a dataset comprising 300 PASA-predicted genes likely containing complete coding regions. The trained Augustus was implemented with a hint file, which was generated by alignment of the cDNA set to the genome with BLAT. Finally, both the PASA- and Augustus-predicted datasets were merged, and only genes with a single exon predicted in both datasets, not in either one of the two datasets, were selected because the prediction accuracy of genes with a single exon is generally low.</p><p>To deduce coding regions of the predicted genes, all possible amino acid sequences translated from three reading frames for multiple exon genes and six reading frames for single exon genes were similarity-searched against the Uniprot-Tremble database (<a class="link link-uri" href="http://www.ebi.ac.uk/uniprot/">http://www.ebi.ac.uk/uniprot/</a>) with BLASTP. Translated amino acid sequences that had a similarity to a protein in the database with <em>E</em>-value ≤ 1<em>E</em>–5, identity ≥ 30% and minimal length ≥ 50 amino acids were selected and defined as the coding regions. When all translated sequences of a gene exhibited no similar proteins, the longest coding region was selected. To confirm the coverage of the assembly and the accuracy of the annotated genes, core eukaryotic genes were mapped using CEGMA. References for computer programs and databases are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. The predicted genes related to TEs were excluded for further analyses.</p>                            <h3 scrollto-destination=5799076 id="5799076" class="section-title" data-legacy-id=s2i>2.9. Comparison of metabolic pathways</h3>
<p>For comparison of the metabolic pathway, <em>Beta vulgaris</em>, <em>A. thaliana</em> and <em>Oryza sativa</em> were chosen. <em>B. vulgaris,</em> which has numerous cultivated root vegetables such as table beet and sugar beet, is a member of the same order Caryophyllales with carnation and has relatively large number of registered genes among this order. <em>Arabidopsis thaliana</em> and <em>O. sativa</em> are typical models of dicot and monocot, respectively. The nucleotide sequences of gene repertoires of <em>A. thaliana</em> and <em>O. sativa</em> were retrieved from the genome databases of TAIR10 and IRGSP 1.0, respectively. For <em>B. vulgaris</em>, expressed sequence tag (EST) sequences were obtained from dbEST of the NCBI databases and trimmed using the CROSS_MATCH program for vector sequences provided in CROSS_MATCH and NCBI's Univec (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/tools/vecscreen/univec/">http://www.ncbi.nlm.nih.gov/tools/vecscreen/univec/</a>) reads longer than 100 bp were subjected to assembly by PHRED with default parameters. The sequences of genes and unigenes thus obtained, as well as those of the predicted carnation genes, were mapped onto the KEGG reference pathways by BLAST searches against genes in the KEGG database with <em>E</em>-value cutoff of 1<em>E</em>–10, length coverage ≥25% and identity ≥50%, and the status of mapping was compared among the four plant species. References for computer programs and databases are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>.</p>                            <h3 scrollto-destination=5799079 id="5799079" class="section-title" data-legacy-id=s2j>2.10. Functional classification of genes</h3>
<p>The predicted gene sequences of carnation were searched against NCBI's Clusters of Orthologous Groups of proteins (KOG) by BLAST searches with <em>E</em>-value cutoff of 1<em>E</em>-20. In addition, functional domains located in the translated sequences in genes and unigenes were searched against the InterPro databases using InterProScan, and the detected domains were further classified into plant GO slim categories using the map2slim program.</p>                            <h2 scrollto-destination=5799082 id="5799082" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5799083 id="5799083" class="section-title" data-legacy-id=s3a>3.1. Sequencing the carnation genome</h3>
                            <h4 scrollto-destination=5799084 id="5799084" class="section-title" data-legacy-id=s3a1>3.1.1. Shotgun sequencing and assembly of the carnation genome</h4>
<p>Shotgun sequencing of the genome of the carnation cv. ‘Francesco’ was carried out using a combination of different sequencing libraries for two NGSs, the HiSeq 1000 and the GS FLX+ systems. In the HiSeq 1000 system, a total of 1277.4 million (M), 1526.5, 442.6 and 475.3 M reads corresponding to 127.7, 152.6, 44.3 and 47.5 Gb sequence data were collected from the PE, OF, 3 kb MP and 5 kb MP libraries, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). In parallel, 5.9 M single reads (mean length: 663 bases) and 1.3 M PE reads (mean length: 436 bases) corresponding to 3.9 Gb and 589 Mb sequence data, respectively, were obtained using the GS FLX+ system (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). The genome size of carnation cv. ‘Francesco’, which was estimated by k-mer analysis<sup><a reveal-id="DST053C21" data-open="DST053C21" class="link link-ref link-reveal xref-bibr">21</a></sup> based on the HiSeq 1000 sequence data, was 622 Mb, which is 93% of the previous estimate (670 Mb)<sup><a reveal-id="DST053C16" data-open="DST053C16" class="link link-ref link-reveal xref-bibr">16</a></sup> this value was adopted for subsequent analyses. Total redundancy of the obtained sequence data (376.6 Gb) was equivalent to ∼604-times the estimated genome size.</p><p>The method used for genomic data assembly is described in the Materials and methods and summarized in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a> and <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dst053/-/DC1" target="_blank">Table S1</a>. The total length of the resulting genomic assemblies was 568.9 Mb, equivalent to 91% of the estimated genome size, containing 69 Mb gaps filled by N; the N50 values of the contigs and scaffolds were 16 644 bp and 60 737 bp, respectively (Table <a reveal-id="DST053TB1" data-open="DST053TB1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="5799088" scrollto-destination="5799088"></a>
<div content-id="DST053TB1" class="table-modal"><div class="table-wrap"><div id="DST053TB1" class="table-wrap-title" data-id="DST053TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistics of <em>D. caryophyllus</em> cv. ‘Francesco’ genome assemblies</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Limited length<sup>a</sup> </th><th>Contigs<sup>b</sup> </th><th>Scaffolds </th></tr></thead><tbody><tr><td>Total number </td><td rowspan="4">≥100 bp </td><td>88 654 </td><td>45 088 </td></tr><tr><td>Total length (kb) </td><td>50 0218 </td><td>568 887 </td></tr><tr><td>N50 (bp) </td><td>16 644 </td><td>60 737 </td></tr><tr><td>N90 (bp) </td><td>2036 </td><td>6700 </td></tr><tr><td>Total number </td><td rowspan="4">≥1 kb </td><td>64 159 </td><td>30 716 </td></tr><tr><td>Total length (Mb) </td><td>483 703 </td><td>558 518 </td></tr><tr><td>N50 (bp) </td><td>17 553 </td><td>62 616 </td></tr><tr><td>N90 (bp) </td><td>2678 </td><td>8036 </td></tr><tr><td>Maximum length (kb) </td><td> </td><td>363 </td><td>1287 </td></tr><tr><td>GC content (%) </td><td> </td><td>36.3 </td><td> </td></tr></tbody></table></div><div content-id="DST053TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Limited length<sup>a</sup> </th><th>Contigs<sup>b</sup> </th><th>Scaffolds </th></tr></thead><tbody><tr><td>Total number </td><td rowspan="4">≥100 bp </td><td>88 654 </td><td>45 088 </td></tr><tr><td>Total length (kb) </td><td>50 0218 </td><td>568 887 </td></tr><tr><td>N50 (bp) </td><td>16 644 </td><td>60 737 </td></tr><tr><td>N90 (bp) </td><td>2036 </td><td>6700 </td></tr><tr><td>Total number </td><td rowspan="4">≥1 kb </td><td>64 159 </td><td>30 716 </td></tr><tr><td>Total length (Mb) </td><td>483 703 </td><td>558 518 </td></tr><tr><td>N50 (bp) </td><td>17 553 </td><td>62 616 </td></tr><tr><td>N90 (bp) </td><td>2678 </td><td>8036 </td></tr><tr><td>Maximum length (kb) </td><td> </td><td>363 </td><td>1287 </td></tr><tr><td>GC content (%) </td><td> </td><td>36.3 </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn1"><div class="fn"><p><sup>a</sup>Contigs or scaffolds shorter than the indicated length were excluded from the statistics.</p></div></div><div content-id="tbnfn2"><div class="fn"><p><sup>b</sup>Subcontigs in scaffolds, which were split by gap regions with length &gt;4 bp.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5799088" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST053TB1" class="table-wrap-title" data-id="DST053TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistics of <em>D. caryophyllus</em> cv. ‘Francesco’ genome assemblies</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Limited length<sup>a</sup> </th><th>Contigs<sup>b</sup> </th><th>Scaffolds </th></tr></thead><tbody><tr><td>Total number </td><td rowspan="4">≥100 bp </td><td>88 654 </td><td>45 088 </td></tr><tr><td>Total length (kb) </td><td>50 0218 </td><td>568 887 </td></tr><tr><td>N50 (bp) </td><td>16 644 </td><td>60 737 </td></tr><tr><td>N90 (bp) </td><td>2036 </td><td>6700 </td></tr><tr><td>Total number </td><td rowspan="4">≥1 kb </td><td>64 159 </td><td>30 716 </td></tr><tr><td>Total length (Mb) </td><td>483 703 </td><td>558 518 </td></tr><tr><td>N50 (bp) </td><td>17 553 </td><td>62 616 </td></tr><tr><td>N90 (bp) </td><td>2678 </td><td>8036 </td></tr><tr><td>Maximum length (kb) </td><td> </td><td>363 </td><td>1287 </td></tr><tr><td>GC content (%) </td><td> </td><td>36.3 </td><td> </td></tr></tbody></table></div><div content-id="DST053TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Limited length<sup>a</sup> </th><th>Contigs<sup>b</sup> </th><th>Scaffolds </th></tr></thead><tbody><tr><td>Total number </td><td rowspan="4">≥100 bp </td><td>88 654 </td><td>45 088 </td></tr><tr><td>Total length (kb) </td><td>50 0218 </td><td>568 887 </td></tr><tr><td>N50 (bp) </td><td>16 644 </td><td>60 737 </td></tr><tr><td>N90 (bp) </td><td>2036 </td><td>6700 </td></tr><tr><td>Total number </td><td rowspan="4">≥1 kb </td><td>64 159 </td><td>30 716 </td></tr><tr><td>Total length (Mb) </td><td>483 703 </td><td>558 518 </td></tr><tr><td>N50 (bp) </td><td>17 553 </td><td>62 616 </td></tr><tr><td>N90 (bp) </td><td>2678 </td><td>8036 </td></tr><tr><td>Maximum length (kb) </td><td> </td><td>363 </td><td>1287 </td></tr><tr><td>GC content (%) </td><td> </td><td>36.3 </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn1"><div class="fn"><p><sup>a</sup>Contigs or scaffolds shorter than the indicated length were excluded from the statistics.</p></div></div><div content-id="tbnfn2"><div class="fn"><p><sup>b</sup>Subcontigs in scaffolds, which were split by gap regions with length &gt;4 bp.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5799088" target="_blank">View Large</a></div></div><p>The N50 value of the final scaffolds was relatively low, probably due to the heterozygotic nature (∼0.2% heterozygosity estimated in this study; data not shown) of the carnation genome. Because current methods for <em>de novo</em> assembly using de Bruijn graphs split assemblies at heterologous polymorphic sites, assemblies of heterozygous diploid genomes tend to be fragmented. However, the N50 value of the contigs contained in the scaffolds was 16.6 kb (Table <a reveal-id="DST053TB1" data-open="DST053TB1" class="link link-reveal link-table xref-fig">1</a>), which allowed reliable characterization and gene annotations. Mapping of 248 core eukaryotic genes by the CEGMA program indicated that 96% of the core genes were completely covered in the genome assemblies. Comparison of the independently determined sequences of the two BAC clones with the assembled genomic sequences showed perfect alignment with correct order and coverage, demonstrating that the coverage and quality of the assembled genomic sequences were high.</p>                            <h4 scrollto-destination=5799090 id="5799090" class="section-title" data-legacy-id=s3a2>3.1.2. Correlation of the genomic sequences with a genetic linkage map</h4>
<p>We constructed an SSR-based reference genetic linkage map of the carnation genome comprising 412 SSR loci on a total length of 969.6 cM.<sup><a reveal-id="DST053C14" data-open="DST053C14" class="link link-ref link-reveal xref-bibr">14</a></sup> To correlate the genomic sequences obtained in this study to their positions on this linkage map, we searched the assembled genomic sequences for sequences of these markers and their adjacent regions using the BLASTN program. All primer sequences, sequences of the flanking regions and SSR motifs were successfully mapped to the assembled genomic sequences, although there were single base substitutions or small deletions of several bases long. Single corresponding scaffolds could be identified for 378 (91.7%) of the 412 SSR loci (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>), and the remaining SSR loci were assigned to multiple scaffolds containing identical or highly similar sequences. Consequently, 268 scaffolds could be located on the genetic linkage map. Some of the scaffolds covered multiple marker loci, e.g. scaffold6 on LG 85P_5, with six SSR loci. The longest mapped scaffold was scaffold2 on LG 85P_9, which covered 1.2 Mb regions and three marker loci. The total length of the mapped scaffolds was 51.4 Mb, equivalent to 8.3% of the estimated genome size.</p>                            <h3 scrollto-destination=5799093 id="5799093" class="section-title" data-legacy-id=s3b>3.2. Characteristic features of the carnation genome</h3>
                            <h4 scrollto-destination=5799094 id="5799094" class="section-title" data-legacy-id=s3b1>3.2.1. Repetitive sequences</h4>
<p>The repetitive sequences found in the assembled genomic sequences comprised 112 078 known TEs, 354 221 simple repeats and low complexity sequences, and 331 831 novel repeats defined by <em>de novo</em> repeat finding, the sum of which correspond to 33% of the assembled genomic sequences (Table <a reveal-id="DST053TB2" data-open="DST053TB2" class="link link-reveal link-table xref-fig">2</a>). Comparison of the TE contents of the carnation genome with those of other plant species, such as <em>A. thaliana</em>, <em>Brassica rapa</em>, potato, tomato, cotton, soybean, sweet orange, rice, foxtail millet and sorghum (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>), showed that the relative content of known TEs was lower than those of unclassified repeats, and simple repeats appeared to be higher in carnation. The lower content of TEs in the assembled genomic sequences in carnation may be attributed to the escape of TE sequences during the process of sequence assembly, since the carnation assemblies in this study were highly fragmented at the positions of potential repetitive sequences. </p>                        <a id="5799097" scrollto-destination="5799097"></a>
<div content-id="DST053TB2" class="table-modal"><div class="table-wrap"><div id="DST053TB2" class="table-wrap-title" data-id="DST053TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Repetitive sequences identified in the carnation genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Repeat class </th><th>Number </th><th>Total length (kb) </th><th>Genome content (%)<sup>a</sup> </th><th>Repeat composition (%) </th></tr></thead><tbody><tr><td colspan="5">Class 1 TE </td></tr><tr><td> LTR </td><td>86 111 </td><td>36 558 </td><td>7.3 </td><td>22.1 </td></tr><tr><td> SINE </td><td>441 </td><td>35.7 </td><td>0.007 </td><td>0.02 </td></tr><tr><td> LINE </td><td>7362 </td><td>3447 </td><td>0.69 </td><td>2.1 </td></tr><tr><td> Other </td><td>10 </td><td>1.7 </td><td>0.0003 </td><td>0 </td></tr><tr><td colspan="5">Class 2 TE </td></tr><tr><td> DNA transposon </td><td>16 729 </td><td>3914 </td><td>0.78 </td><td>2.4 </td></tr><tr><td> Rolling circle </td><td>1425 </td><td>535 </td><td>0.11 </td><td>0.33 </td></tr><tr><td>Other type TE </td><td>48 </td><td>21.1 </td><td>0.004 </td><td>0.01 </td></tr><tr><td>Simple repeat </td><td>295 621 </td><td>16 847 </td><td>3.4 </td><td>10.3 </td></tr><tr><td>Low complexity </td><td>58 600 </td><td>3528 </td><td>0.71 </td><td>2.2 </td></tr><tr><td>Other repeat </td><td>411 </td><td>65 </td><td>0.004 </td><td>0.01 </td></tr><tr><td>Unclassified (novel) repeat </td><td>331 831 </td><td>100 097 </td><td>20 </td><td>60.6 </td></tr><tr><td>Total </td><td>794 297 </td><td>165 027 </td><td>33 </td><td>100 </td></tr></tbody></table></div><div content-id="DST053TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Repeat class </th><th>Number </th><th>Total length (kb) </th><th>Genome content (%)<sup>a</sup> </th><th>Repeat composition (%) </th></tr></thead><tbody><tr><td colspan="5">Class 1 TE </td></tr><tr><td> LTR </td><td>86 111 </td><td>36 558 </td><td>7.3 </td><td>22.1 </td></tr><tr><td> SINE </td><td>441 </td><td>35.7 </td><td>0.007 </td><td>0.02 </td></tr><tr><td> LINE </td><td>7362 </td><td>3447 </td><td>0.69 </td><td>2.1 </td></tr><tr><td> Other </td><td>10 </td><td>1.7 </td><td>0.0003 </td><td>0 </td></tr><tr><td colspan="5">Class 2 TE </td></tr><tr><td> DNA transposon </td><td>16 729 </td><td>3914 </td><td>0.78 </td><td>2.4 </td></tr><tr><td> Rolling circle </td><td>1425 </td><td>535 </td><td>0.11 </td><td>0.33 </td></tr><tr><td>Other type TE </td><td>48 </td><td>21.1 </td><td>0.004 </td><td>0.01 </td></tr><tr><td>Simple repeat </td><td>295 621 </td><td>16 847 </td><td>3.4 </td><td>10.3 </td></tr><tr><td>Low complexity </td><td>58 600 </td><td>3528 </td><td>0.71 </td><td>2.2 </td></tr><tr><td>Other repeat </td><td>411 </td><td>65 </td><td>0.004 </td><td>0.01 </td></tr><tr><td>Unclassified (novel) repeat </td><td>331 831 </td><td>100 097 </td><td>20 </td><td>60.6 </td></tr><tr><td>Total </td><td>794 297 </td><td>165 027 </td><td>33 </td><td>100 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn3"><div class="fn"><p>LTR, long terminal repeat; SINE, short interspersed nuclear element; LINE, long interspersed nuclear element.</p></div></div><div content-id="tbnfn4"><div class="fn"><p><sup>a</sup>Percentage of total length of repeats in the continuous sequences in the assembled genome.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5799097" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST053TB2" class="table-wrap-title" data-id="DST053TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Repetitive sequences identified in the carnation genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Repeat class </th><th>Number </th><th>Total length (kb) </th><th>Genome content (%)<sup>a</sup> </th><th>Repeat composition (%) </th></tr></thead><tbody><tr><td colspan="5">Class 1 TE </td></tr><tr><td> LTR </td><td>86 111 </td><td>36 558 </td><td>7.3 </td><td>22.1 </td></tr><tr><td> SINE </td><td>441 </td><td>35.7 </td><td>0.007 </td><td>0.02 </td></tr><tr><td> LINE </td><td>7362 </td><td>3447 </td><td>0.69 </td><td>2.1 </td></tr><tr><td> Other </td><td>10 </td><td>1.7 </td><td>0.0003 </td><td>0 </td></tr><tr><td colspan="5">Class 2 TE </td></tr><tr><td> DNA transposon </td><td>16 729 </td><td>3914 </td><td>0.78 </td><td>2.4 </td></tr><tr><td> Rolling circle </td><td>1425 </td><td>535 </td><td>0.11 </td><td>0.33 </td></tr><tr><td>Other type TE </td><td>48 </td><td>21.1 </td><td>0.004 </td><td>0.01 </td></tr><tr><td>Simple repeat </td><td>295 621 </td><td>16 847 </td><td>3.4 </td><td>10.3 </td></tr><tr><td>Low complexity </td><td>58 600 </td><td>3528 </td><td>0.71 </td><td>2.2 </td></tr><tr><td>Other repeat </td><td>411 </td><td>65 </td><td>0.004 </td><td>0.01 </td></tr><tr><td>Unclassified (novel) repeat </td><td>331 831 </td><td>100 097 </td><td>20 </td><td>60.6 </td></tr><tr><td>Total </td><td>794 297 </td><td>165 027 </td><td>33 </td><td>100 </td></tr></tbody></table></div><div content-id="DST053TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Repeat class </th><th>Number </th><th>Total length (kb) </th><th>Genome content (%)<sup>a</sup> </th><th>Repeat composition (%) </th></tr></thead><tbody><tr><td colspan="5">Class 1 TE </td></tr><tr><td> LTR </td><td>86 111 </td><td>36 558 </td><td>7.3 </td><td>22.1 </td></tr><tr><td> SINE </td><td>441 </td><td>35.7 </td><td>0.007 </td><td>0.02 </td></tr><tr><td> LINE </td><td>7362 </td><td>3447 </td><td>0.69 </td><td>2.1 </td></tr><tr><td> Other </td><td>10 </td><td>1.7 </td><td>0.0003 </td><td>0 </td></tr><tr><td colspan="5">Class 2 TE </td></tr><tr><td> DNA transposon </td><td>16 729 </td><td>3914 </td><td>0.78 </td><td>2.4 </td></tr><tr><td> Rolling circle </td><td>1425 </td><td>535 </td><td>0.11 </td><td>0.33 </td></tr><tr><td>Other type TE </td><td>48 </td><td>21.1 </td><td>0.004 </td><td>0.01 </td></tr><tr><td>Simple repeat </td><td>295 621 </td><td>16 847 </td><td>3.4 </td><td>10.3 </td></tr><tr><td>Low complexity </td><td>58 600 </td><td>3528 </td><td>0.71 </td><td>2.2 </td></tr><tr><td>Other repeat </td><td>411 </td><td>65 </td><td>0.004 </td><td>0.01 </td></tr><tr><td>Unclassified (novel) repeat </td><td>331 831 </td><td>100 097 </td><td>20 </td><td>60.6 </td></tr><tr><td>Total </td><td>794 297 </td><td>165 027 </td><td>33 </td><td>100 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn3"><div class="fn"><p>LTR, long terminal repeat; SINE, short interspersed nuclear element; LINE, long interspersed nuclear element.</p></div></div><div content-id="tbnfn4"><div class="fn"><p><sup>a</sup>Percentage of total length of repeats in the continuous sequences in the assembled genome.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5799097" target="_blank">View Large</a></div></div><p>In carnation, the mechanism underlying variegated flower colour has long been of major horticultural interest.<sup><a reveal-id="DST053C22" data-open="DST053C22" class="link link-ref link-reveal xref-bibr">22</a></sup> An excision event of Class II DNA TEs has been identified from the genes for enzymes involved in anthocyanin biosynthesis.<sup><a reveal-id="DST053C23" data-open="DST053C23" class="link link-ref link-reveal xref-bibr">23</a></sup><em>Tic101</em>, a member of <em>hAT</em> (<em>Ac</em>/<em>Ds</em>) elements, was identified as an autonomous TE encoding an active transposase protein.<sup><a reveal-id="DST053C4" data-open="DST053C4" class="link link-ref link-reveal xref-bibr">4</a></sup> A single homologue of <em>Tdic101</em>, designated <em>Tic104</em>, whose sequence is 99% (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>) identical to that of <em>Tdic101</em> was detected in the assembled genomic sequences.<sup><a reveal-id="DST053C4" data-open="DST053C4" class="link link-ref link-reveal xref-bibr">4</a></sup><em>Tic104</em> has one nucleotide substitution in a terminal-inverted repeat sequence and another in the coding region of the transposase gene to generate a stop codon (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). The CACTA element (<em>En</em>/<em>Spm</em>), <em>dTac1</em> was first identified in a gene for glutathione-<em>S</em>-transferase, here designated <em>Tac101</em>, which is not likely to encode notable proteins for transposases.<sup><a reveal-id="DST053C24" data-open="DST053C24" class="link link-ref link-reveal xref-bibr">24</a></sup> A similarity search against the ‘Francesco’ genomic sequences detected four types of CACTA elements, designated <em>Tac201</em>, <em>Tac301</em>, <em>Tac401</em> and <em>Tac501</em>, in addition to <em>Tac101</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a> and <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dst053/-/DC1" target="_blank">Fig. S3</a>). None of these is likely to encode intact transposases that are active for transposition.</p><p>The genome of ‘Francesco’ contains an <em>acyl-glucose-dependent anthocyanin 5-glucosyltransferase</em> (<em>AA5GT</em>) gene with an insertion by <em>Ty1-1</em> (<em>Ty1dic1</em>), resulting in synthesis and accumulation of pelargonidin 3-<em>O</em>-malylglucoside that lacks its glucose moiety at the 5 positions. <em>Ty1-1</em> in the ‘Francesco’ genome has one nucleotide substitution in the coding region of the transposase gene to generate a stop codon different from <em>Ty1dic1</em>. Similarity searching of the assembled genomic sequences indicated that there were six genes for <em>Ty1-1</em> longer than 1 kb (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4 and</a><a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dst053/-/DC1" target="_blank">Fig. S4</a>).</p>                            <h4 scrollto-destination=5799101 id="5799101" class="section-title" data-legacy-id=s3b2>3.2.2. Genes coding for RNAs</h4>
<p>Thirteen genes for rRNAs and 1050 intact genes for tRNAs were identified in the assembled genomic sequences. Comparison of the number of genes for tRNAs in the genomes of 16 plant species indicated that the carnation genome contains many more genes than other plant species (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). A total of 92 and 143 genes for snoRNA and miRNA, respectively, were also assigned.</p>                            <h4 scrollto-destination=5799104 id="5799104" class="section-title" data-legacy-id=s3b3>3.2.3. Prediction of protein-encoding genes</h4>
<p>We assigned protein-encoding genes in the carnation genome using two types of computer programs, PASA and Augustus. As a result, 10 519 protein-encoding genes predicted by PASA and 99 123 genes predicted by Augustus were merged, resulting in assignment of 56 137 protein-encoding genes including those in TEs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>).</p><p>A similarity search against the Uniprot-Tremble database indicated that the translated amino acid sequences of the 42 047 predicted protein-encoding genes showed significant sequence similarity to the registered genes, and 12 871 were similar to TEs. Consequently, 43 266 protein-encoding genes excluding those TEs were assigned in the assembled genomic sequences, ∼72% of which showed sequence similarity to registered genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). It is possible that the gene number was overestimated due to fragmentation of the assembled sequences and the heterozygotic nature of the genome.</p><p>To estimate the gene coverage of the assembled genomic sequences and the accuracy of our gene prediction, we mapped core eukaryotic genes onto the assembled genomic sequences. Of the 248 core eukaryotic genes, 238 (96%) matched the entire coding regions, which increased to 242 (98%) if partial matches were included.</p>                            <h3 scrollto-destination=5799109 id="5799109" class="section-title" data-legacy-id=s3c>3.3. Comparison of carnation gene repertoire with those of other plant species</h3>
<p>To compare metabolic pathway genes, we examined the genes of <em>B. vulgaris, A. thaliana</em> and <em>O. sativa</em>. Since the genomic sequences of <em>B. vulgaris</em> are not yet available, 29 830 EST sequences of <em>B. vulgaris</em> retrieved from dbEST were assembled into 14 058 unigenes consisting of 5810 contigs and 8248 singletons (total size: 9 781 432 bp) and used for comparison. For <em>A. thaliana</em> and <em>O. sativa</em>, the complete gene sets, 35 386 and 42 136 genes, respectively, were retrieved from the TAIR10 and IRGSP 1.0 databases, respectively.</p><p>The translated amino acid sequences of 43 491 coding sequences (CDSs) including splicing variants in carnation were searched against the unigenes of <em>B. vulgaris</em> and the complete gene sets of <em>A. thaliana</em> (TAIR10) and <em>O. sativa</em> (IGRSP 1.0) using BLAST with <em>E</em>-value cutoff of 1<em>E</em>–10. The distributions of the percentage of amino acid sequence identities are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>. The degree of similarity was in order <em>B. vulgaris</em>, <em>A. thaliana</em> and <em>O. sativa</em>, which is consistent with their phylogenetic relationship.</p><p>The unigenes of <em>B. vulgaris</em> and the genes of <em>A. thaliana</em> (TAIR10) and <em>O. sativa</em> (IRGSP 1.0) were classified into KOG categories (Fig. <a reveal-id="DST053F1" data-open="DST053F1" class="link link-reveal link-table xref-fig">1</a>). The number of unigenes classified into KOG was 6937 (49.3%) in <em>B. vulgaris</em>, and the number of genes classified into KOG was 19 005 (43.7%), 18 250 (51.6%), and 18 065 (42.9%) in carnation, <em>A. thaliana</em>, and <em>O. sativa</em>, respectively. The ratio of the genes in KOG Q (Secondary metabolites biosynthesis, transport and catabolism) was relatively high in carnation. For classification of the genes based on GO slim, on the other hand, the number of unigenes classified into GO category was 5676 (40.4%) in <em>B. vulgaris</em>, and that of genes classified into GO category was 16 423 (37.8%), 21 875 (61.8%) and 20 203 (47.9%) in carnation, <em>A. thaliana</em> and <em>O. sativa</em>, respectively. This result indicated that the genes in the categories ‘nucleobase, nucleoside, nucleotide and nucleic acid metabolic process (Biological Process: BP)’, ‘cell wall (Cellular Component: CC)’ and ‘transferase activity (Molecular Function: MF)’ were relatively high in carnation (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6</a>). </p>                        <a id="5799114" scrollto-destination="5799114"></a>
<div id="DST053F1" data-id="DST053F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst053/2/m_dst05301.png?Expires=2147483647&Signature=IzlONAwjEsKdBu73eLSTJbhq5sqYn3poBq6SJQ6EeSJyuXhxhomjT6h6WwUsB-9rjlWoaXG1MOY9qTj82SVJnHMVh~~-h~CWeEvCPwZwRAIQx~xLoen-ZkjmrYPoJgdmo-9Vk5GfOxfTo~OTuiFPIna-Qa13YGlWr0n054IAGASb9dW3NDoQb83nhPiUULgcOBgIXWk57oxcyUXinFwwdcsBoWqmWf2pCMjCDMFUVKKmsx1mm7JIr25PK48iAxTRnNANJU-mUBWPSH-CEoz0fgGc6AccOfvZGXhx2rp4vLeyidR9ule5kR69DLGJAnDBH5K8sM8QtcLuudC6gumVRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene assignment to KOG functional categories." path-from-xml="dst05301.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5799114/dst05301.tiff" path-from-xml="dst05301.tiff" target="_blank">View large</a><a section="5799114" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst053/2/dst05301.png?Expires=2147483647&Signature=AlNIuA7sTqt--oEKzvBtpKbIecN-UIyaw5jF09kX8nBP04LCA6hYFaFTYownmc7HyuS4PtRJL9TXDhXgaCHO3yy~PpzwxY~1cuXx-pT60sbnmigrmruB8YM5ufK59IRogTYAlm6W6ebnyHW0Ax5yN3OtIPQ---8uDn7g3be-YwQdUb8Uumpb6tgjpuYAkpRNgqhdQxeYF2vL8aUbQ3QSa82RuUBzxbxgCOH2HdkFZuKvdepNnsOapkMMYVcSQJ38eal324N96dgi~SEUcZ~rhN-zvz-luaJo5oLDhPYtZ1RLCCXIQIEzqk1PnjEYg9CAXTWAht87eEb5u5ppprss5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5799114&ar=393171&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05301.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene assignment to KOG functional categories.</p></div></div><div content-id="DST053F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst053/2/m_dst05301.png?Expires=2147483647&Signature=IzlONAwjEsKdBu73eLSTJbhq5sqYn3poBq6SJQ6EeSJyuXhxhomjT6h6WwUsB-9rjlWoaXG1MOY9qTj82SVJnHMVh~~-h~CWeEvCPwZwRAIQx~xLoen-ZkjmrYPoJgdmo-9Vk5GfOxfTo~OTuiFPIna-Qa13YGlWr0n054IAGASb9dW3NDoQb83nhPiUULgcOBgIXWk57oxcyUXinFwwdcsBoWqmWf2pCMjCDMFUVKKmsx1mm7JIr25PK48iAxTRnNANJU-mUBWPSH-CEoz0fgGc6AccOfvZGXhx2rp4vLeyidR9ule5kR69DLGJAnDBH5K8sM8QtcLuudC6gumVRQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene assignment to KOG functional categories." path-from-xml="dst05301.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5799114/dst05301.tiff" path-from-xml="dst05301.tiff" target="_blank">View large</a><a section="5799114" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst053/2/dst05301.png?Expires=2147483647&Signature=AlNIuA7sTqt--oEKzvBtpKbIecN-UIyaw5jF09kX8nBP04LCA6hYFaFTYownmc7HyuS4PtRJL9TXDhXgaCHO3yy~PpzwxY~1cuXx-pT60sbnmigrmruB8YM5ufK59IRogTYAlm6W6ebnyHW0Ax5yN3OtIPQ---8uDn7g3be-YwQdUb8Uumpb6tgjpuYAkpRNgqhdQxeYF2vL8aUbQ3QSa82RuUBzxbxgCOH2HdkFZuKvdepNnsOapkMMYVcSQJ38eal324N96dgi~SEUcZ~rhN-zvz-luaJo5oLDhPYtZ1RLCCXIQIEzqk1PnjEYg9CAXTWAht87eEb5u5ppprss5w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5799114&ar=393171&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst05301.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene assignment to KOG functional categories.</p></div></div><p>The unigenes of <em>B. vulgaris</em> and the complete gene sets comprising 35 386 and 42 136 genes of <em>A. thaliana</em> (TAIR10) and <em>O. sativa</em> (IGRSP 1.0), respectively, were mapped onto KEGG reference pathways. As a result, 11 030 of 43 491 translated amino acid sequences of CDSs in carnation, 13 979 of 14 058 unigenes in <em>B. vulgaris</em>, 13 154 of 35 386 genes in <em>A. thaliana</em> (TAIR10) and 12 082 of 42 136 in <em>O. sativa</em> (IGRSP 1.0) were successfully mapped onto the KEGG reference pathways (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>). The pathways including the genes mapped only in carnation are as follows: ‘Pentose phosphate pathway’, ‘Galactose metabolism’, ‘Ether lipid metabolism’, ‘Alanine, aspartate and glutamate metabolism’, ‘Glycine, serine and threonine metabolism’, ‘Cysteine and methionine metabolism’, ‘Metabolism of xenobiotics by cytochrome P450’ and several others.</p>                            <h3 scrollto-destination=5799118 id="5799118" class="section-title" data-legacy-id=s3d>3.4. Genes characteristic of carnation</h3>
                            <h4 scrollto-destination=5799120 id="5799120" class="section-title" data-legacy-id=s3d1>3.4.1. Genes for enzymes involved in phenylpropanoid biosynthetic pathways</h4>
<p>The phenylpropanoid biosynthetic pathway that begins with phenylalanine and results in the production of anthocyanin, flavonoid and lignin is one of the most well-studied secondary metabolic pathways (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7</a>). Similarity searches and phylogenetic analyses<sup><a reveal-id="DST053C25" data-open="DST053C25" class="link link-ref link-reveal xref-bibr">25</a></sup> for genes known to be involved in the synthesis of phenylpropanoid, flavonoid and lignin against the assembled genomic sequences of carnation was performed, and the results are summarized in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>.</p><p>Similarity searches detected a single possible flavonoid 3′-hydroxylase (F3′H) homologue among 241 genes homologous to cytochrome P450 in the genome of carnation cv. ‘Francesco’. The <em>F3′H</em> gene in cv. ‘Mrs. Purple’ is known to code for an active protein of 514 amino acid residues.<sup><a reveal-id="DST053C4" data-open="DST053C4" class="link link-ref link-reveal xref-bibr">4</a></sup> By contrast, the F3′H homologue in cv. ‘Francesco’ contains one extra guanidine nucleotide in the potential coding region, resulting in the production of a truncated 193 amino acid peptide.</p><p>In addition, a total of 84, 25, 120, 93 and 139 putative homologues of glutathione <em>S</em>-transferases, multidrug and toxic compound extrusion (MATE)-type transporter, UDP-sugar dependent glycosyltransferase, Myb transcription factors and bHLH transcription factors, which are known to be involved in the phenylpropanoid biosynthetic pathway, were assigned in the assembled genomic sequences. Details of these genes as well as the results of phylogenetic analysis are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Figs S8–S12</a>.</p>                            <h4 scrollto-destination=5799125 id="5799125" class="section-title" data-legacy-id=s3d2>3.4.2. Genes involved in betalain synthesis</h4>
<p>Betalains are commonly synthesized instead of anthocyanin in the order Caryophyllales, except for two families, Caryophyllaceae and Molluginaceae. Although carnation is classified under the family Caryophillaceae and produces anthocyanin as the main plant pigment in petals, and often in other organs, carnation may also contain the genes required to produce betalains. Three characteristic enzymes are involved in synthesis of betalains, i.e. <span class="small-caps">l</span>-dihydroxyphenylalanine (DOPA) 4,5-dioxygenase (DOD),<sup><a reveal-id="DST053C6" data-open="DST053C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DST053C26" data-open="DST053C26" class="link link-ref link-reveal xref-bibr">26</a></sup> cytochrome P450 monooxygenase (CYP76AD)<sup><a reveal-id="DST053C27" data-open="DST053C27" class="link link-ref link-reveal xref-bibr">27</a></sup> and <em>cyclo</em>-DOPA glucosyltransferase (<em>c</em>D5GT), in the UDP-dependent glycosyltransferase (UGT) family.<sup><a reveal-id="DST053C28" data-open="DST053C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DST053C29" data-open="DST053C29" class="link link-ref link-reveal xref-bibr">29</a></sup></p><p>One copy of a DOD homologue (<em>Dca8668</em>) was found in the carnation genomic sequences by similarity searching. Multiple alignments revealed that the amino acid sequence of the carnation DOD homologue contains the conserved motif typical of non-betalain-producing plants (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S13</a>). This result suggests that the carnation DOD homologue does not catalyze the formation of betalamic acid.</p><p>CYP76AD belongs to the cytochrome P450 monooxygenase (CYP) family, which is one of the most divergent families in higher plants,<sup><a reveal-id="DST053C27" data-open="DST053C27" class="link link-ref link-reveal xref-bibr">27</a></sup> with 241 homologues of CYP detected in the carnation genomic sequences. Dca32662 showed the highest amino acid identity (66.4%) with that of <em>B. vulgaris</em> CTP76AD1. However, the fact that this protein belongs to CYP76C implied that carnation genome does not encode betalain-related CYP family protein. A single homologue of <em>Mirabilis jalapa cD5GT</em> (<em>MjcD5GT</em>) gene was found. The deduced amino acid sequence of this UGT homolog showed 58.4% identity to MjcD5GT and 62.5% to <em>Celosia cristata</em> cD5GT.</p>                            <h4 scrollto-destination=5799131 id="5799131" class="section-title" data-legacy-id=s3d3>3.4.3. Genes involved in chlorophyll and carotenoid synthesis</h4>
<p>Similarity searches against the assembled genomic sequences in carnation identified all of the genes involved in the chlorophyll metabolic pathway in the carnation genome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>). We found multiple genes encoding putative isozymes, including glutamate-1-semialdehyde 2,1-aminotransferase (<em>GSA</em>), 5-aminolevulinate dehydrogenase (<em>HEMB</em>), porphobilinogen deaminase (<em>HEMC</em>), uroporphyrinogen III decarboxylase (<em>HEME</em>), magnesium chelatase D subunit (<em>CHLD</em>)<em>,</em> Mg-protoporphyrin IX methyltransferase (<em>CHLM</em>), Mg-protoporphyrin IX monomethylester cyclase (<em>CRD</em>), protochlorophyllide oxidoreductase (<em>PORA</em>) and divinyl chlorophyllide <em>a</em> 8-vinyl reductase (<em>DVR</em>), which are involved in chlorophyll synthesis in carnation. By contrast, all of the enzymes involved in the chlorophyll cycle and chlorophyll degradation are likely to be encoded by a single gene. STAY-GREEN (<em>SGR</em>), a protein involved in senescence-related chlorophyll degradation, is encoded by three genes.</p><p>Similarity searches revealed that most of the enzymes involved in carotenoid biosynthesis are encoded by single genes in the carnation genome (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S10</a>). In addition, two homologues of 9-<em>cis</em>-epoxycarotenoid dioxygenase (<em>NCED</em>) genes involved in abscisic acid biosynthesis in <em>A. thaliana</em>,<sup><a reveal-id="DST053C30" data-open="DST053C30" class="link link-ref link-reveal xref-bibr">30</a></sup> two homologs of carotenoid cleavage dioxygenase (<em>CCD4</em>) genes as well as single homologues each of <em>CCD1</em>, <em>CCD7</em> and <em>CCD8</em>, genes related to different enzyme activities and substrate specificities,<sup><a reveal-id="DST053C30" data-open="DST053C30" class="link link-ref link-reveal xref-bibr">30</a></sup> were assigned (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S14</a>). No ESTs encoding CCDs were found in the carnation EST dataset (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S10</a>),<sup><a reveal-id="DST053C13" data-open="DST053C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DST053C31" data-open="DST053C31" class="link link-ref link-reveal xref-bibr">31</a></sup> suggesting that their expression levels are low.</p>                            <h4 scrollto-destination=5799134 id="5799134" class="section-title" data-legacy-id=s3d4>3.4.4. Disease resistance genes</h4>
<p>We searched for nucleotide-binding site-leucine-rich repeat (NBS-LRR) genes in the assembled genomic sequences of carnation and assigned 217 NBS-containing potential Resistance (<em>R</em>) genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S11</a>). The number of NBS-LRR genes in carnation is larger than that of cucumber,<sup><a reveal-id="DST053C32" data-open="DST053C32" class="link link-ref link-reveal xref-bibr">32</a></sup> melon<sup><a reveal-id="DST053C33" data-open="DST053C33" class="link link-ref link-reveal xref-bibr">33</a></sup> and papaya,<sup><a reveal-id="DST053C34" data-open="DST053C34" class="link link-ref link-reveal xref-bibr">34</a></sup> but smaller than that of tomato,<sup><a reveal-id="DST053C35" data-open="DST053C35" class="link link-ref link-reveal xref-bibr">35</a></sup> grape<sup><a reveal-id="DST053C36" data-open="DST053C36" class="link link-ref link-reveal xref-bibr">36</a></sup> and rice.<sup><a reveal-id="DST053C37" data-open="DST053C37" class="link link-ref link-reveal xref-bibr">37</a></sup> In this study, only three potential genes with Toll/interleukin-1 receptor (TIR) domains at the N-terminal were identified, two of which lack LRRs, while 69 genes containing a coiled-coil motif were assigned, 52 of which lack LRRs. This result is consistent with the previous observation that the genome of <em>B. vulgaris</em> lacks TIR-type resistance genes despite the fact that this plant is a dicot.<sup><a reveal-id="DST053C38" data-open="DST053C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Our present results support the assumption that the loss of TIR-type resistance genes is not restricted to cereals or monocots in general,<sup><a reveal-id="DST053C38" data-open="DST053C38" class="link link-ref link-reveal xref-bibr">38</a></sup> indicating the unique feature of <em>R</em> gene evolution in Caryophyllales. It should also be noted that more than 50% of the 114 NBS-containing genes lack N- and C-terminal domains (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S11</a>), with 30 genes (NL) possessing only NBS-LRR domains.</p><p>The 217 potential NBS-type <em>R</em> genes predicted in this study were assigned to 125 scaffolds. Of the 125 scaffolds, 87 contain single NBS genes, while the other scaffolds contain multiple NBS genes. The positions of eight scaffolds containing five or more NBS genes are shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S15</a>. Approximately 30% of the NBS-type <em>R</em> genes are clustered in these scaffolds, and the scaffold most abundant in NBS-type <em>R</em> genes is No.146, which contains 11 genes. These results suggest that <em>R</em> genes are unevenly distributed in the carnation genome, as was reported for a wide range of plants.<sup><a reveal-id="DST053C33  DST053C35" data-open="DST053C33  DST053C35" class="link link-ref link-reveal xref-bibr">33–35</a>,<a reveal-id="DST053C37" data-open="DST053C37" class="link link-ref link-reveal xref-bibr">37</a></sup></p>                            <h4 scrollto-destination=5799138 id="5799138" class="section-title" data-legacy-id=s3d5>3.4.5. Genes involved in ethylene metabolism</h4>
<p>Carnation has been used as a model system to study the mechanism of ethylene-induced flower senescence.<sup><a reveal-id="DST053C10" data-open="DST053C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Components of the ethylene signal transduction pathway, such as the ethylene receptors, Constitutive Triple Response (CTR), Ethylene-Insensitive 2 (EIN2) and EIN3, regulate a series of senescence-related genes.<sup><a reveal-id="DST053C10" data-open="DST053C10" class="link link-ref link-reveal xref-bibr">10</a></sup> By searching the carnation genomic sequences, six putative genes for ethylene receptors (DcETR1, DcETR2, DcETR3, DcETR4, DcERS1 and DcERS2), two genes for CTR, two genes for EIN2 and three EIN3-like genes were identified (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S12</a>). Phylogenetic analysis classified six ethylene receptors in carnation into two subfamilies; DcETR1, DcERS1 and DcERS2 are in Subfamily 1, which contains conserved histidine kinase domains, and DcETR2, DcETR3 and DcETR4 are in Subfamily 2 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S16</a>).</p><p>With respect to ethylene biosynthesis, three 1-aminocyclopropane-1-carboxylic acid (ACC) synthase genes (<em>DcACS1</em>, <em>DcACS2</em> and <em>DcACS3</em>) and one ACC oxidase gene (<em>DcACO1</em>) have been identified.<sup><a reveal-id="DST053C39" data-open="DST053C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Based on sequence similarity, we found six more genes for ACS (<em>DcACS</em>4 –<em>DcACS9</em>) and four more genes for ACO (<em>DcACO2</em>–<em>DcACO5</em>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S12</a>). Notably, the putative gene products of DcACS4, DcACS5 and DcACS6 lack the motifs BOX6 and BOX7, BOX1 and BOX2, and BOX6, respectively, while those of other ACSs contain all seven conserved BOXs found in ACS isozymes in <em>A. thaliana</em>.<sup><a reveal-id="DST053C40" data-open="DST053C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Since only eight out of 12 ACS genes are catalytically active in <em>A. thaliana</em>,<sup><a reveal-id="DST053C40" data-open="DST053C40" class="link link-ref link-reveal xref-bibr">40</a></sup> it is probable that all of the <em>ACS</em> identified in the carnation genome do not encode active ACS for synthesizing ethylene.</p>                            <h4 scrollto-destination=5799142 id="5799142" class="section-title" data-legacy-id=s3d6>3.4.6. Genes involved in carbohydrate metabolism and cell wall modification during flower opening</h4>
<p>Large amounts of soluble carbohydrate are required for flower opening as substrates for respiration and cell wall synthesis. Accumulation of substantial amounts of pinitol, one of the rare sugars potentially involved in salinity tolerance, is a unique aspect of sugar metabolism during flower opening in carnation.<sup><a reveal-id="DST053C41" data-open="DST053C41" class="link link-ref link-reveal xref-bibr">41</a></sup> One gene (<em>Dca24344</em>) showing strong homology to known genes encoding <em>myo</em>-inositol methyl transferase (IMT), which catalyzes the conversion of <em>myo</em>-inositol to pinitol, was found in the carnation genome.</p><p>Xyloglucan endotransglucosylase/hydrolase (XTH), which is involved in the hydrolysis and reconstruction of xyloglucan in the matrix polysaccharides of the cell wall,<sup><a reveal-id="DST053C42" data-open="DST053C42" class="link link-ref link-reveal xref-bibr">42</a></sup> is believed to play an essential role in increasing cell wall extensibility followed by water uptake during the process of petal cell expansion.<sup><a reveal-id="DST053C43" data-open="DST053C43" class="link link-ref link-reveal xref-bibr">43</a></sup> Genome-wide analysis has revealed 33 and 29 genes in the XTH family in <em>A. thaliana</em><sup><a reveal-id="DST053C44" data-open="DST053C44" class="link link-ref link-reveal xref-bibr">44</a></sup> and <em>O. sativa</em>,<sup><a reveal-id="DST053C45" data-open="DST053C45" class="link link-ref link-reveal xref-bibr">45</a></sup> respectively. In the carnation genomic sequences, 32 genes putatively encoding XTH were detected (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S13 and</a><a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dst053/-/DC1" target="_blank">Figs. S17, S18</a>), 11 of which were reported to be expressed in flowers and in some vegetative tissues.<sup><a reveal-id="DST053C13" data-open="DST053C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DST053C46" data-open="DST053C46" class="link link-ref link-reveal xref-bibr">46</a></sup></p>                            <h4 scrollto-destination=5799146 id="5799146" class="section-title" data-legacy-id=s3d7>3.4.7. Genes related to floral scent</h4>
<p>Floral scent is a notable breeding target for carnation.<sup><a reveal-id="DST053C47" data-open="DST053C47" class="link link-ref link-reveal xref-bibr">47</a></sup> Methylation by <em>S</em>-adenosyl-<span class="small-caps">l</span>-methionine-dependent methyltransferases belonging to SABATH family,<sup><a reveal-id="DST053C48" data-open="DST053C48" class="link link-ref link-reveal xref-bibr">48</a></sup> is an important catalytic process for emission of scent components from flowers. Methyl benzoate that is a major scent component of modern carnation cultivars,<sup><a reveal-id="DST053C47" data-open="DST053C47" class="link link-ref link-reveal xref-bibr">47</a></sup> is also derived from the methylation of benzoic acid. A similarity search against the carnation genomic sequences detected 11 genes in the SABATH family (<em>DcSABATH1-11</em>), which are candidate genes of benzoic acid methyltransferase in carnation (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S14</a>). Phylogenetic analysis of functionally characterized members of SABATH family showed that benzoic acid and salicylic acid methyltransferase form a monophyletic lineage irrespective of plant species (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S19</a>).<sup><a reveal-id="DST053C49" data-open="DST053C49" class="link link-ref link-reveal xref-bibr">49</a></sup> In contrast, DcSABATHs, SABATH members in carnation, did not strongly associate with this lineage.</p>                            <h3 scrollto-destination=5799149 id="5799149" class="section-title" data-legacy-id=s3e>3.5. Database and data retrieval</h3>
<p>All of the information about the assembled scaffold sequences, known and novel repetitive sequences, genes for non-coding RNA (tRNA, rRNA, snoRNA and miRNA) and potential protein-encoding genes is available through the Carnation DB (<a class="link link-uri" href="http://carnation.kazusa.or.jp">http://carnation.kazusa.or.jp</a>).</p><p>All the sequence data obtained in this study is available under the BioProject ID, PRJDB1491. The accession number of the reads sequenced by Illumina HiSeq 1000 are as follows: PE (insert size = 500 bp): DRX012625, OF (insert size = 180 bp): DRX012624, MP (insert size = 3 kb): DRX012626, MP (insert size = 5 kb): DRX012627. The accession numbers of the reads sequenced by Roche GS FLX+ are as follows: SE: DRX012628, PE (insert size = 4 kb): DRX012629. The nucleotide sequences of assembled scaffolds can be retrieved under the accession numbers DF340864-DF357213 (16 350 entries).</p>                            <h2 scrollto-destination=5799152 id="5799152" class="section-title" data-legacy-id=s4>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5799155 id="5799155" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was funded by grants from the Yoshio Itoh Fund of Tokyo University of Agriculture and Technology, and the Kazusa DNA Research Institute Foundation. Construction of the BAC library was supported by a genome-support grant from the National Institute of Agrobiological Sciences (NIAS) and JSPS KAKENHI grant number 24780038.</p>                            <h2 scrollto-destination=5799158 id="5799158" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST053C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Galbally</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Galbally</div> <div class="given-names">E.</div></div>. </div>, <div class="source">Carnations and Pinks for Garden and Greenhouse</div> , <div class="year">1997</div><div class="publisher-loc">Portland</div><div class="publisher-name">Timber Press</div></div></div></div></div><div content-id="DST053C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hamilton</div> <div class="given-names">R.F.L.</div></div>,  <div class="name"><div class="surname">Walters</div> <div class="given-names">S.M.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Walters</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Alexander</div> <div class="given-names">J.C.M.</div></div>,  <div class="name"><div class="surname">Brady</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title"><em>Dianthus</em> Linnaeus</div>, <div class="source">The European Garden Flora</div> , <div class="year">1989</div>, vol. <div class="volume">vol. III</div> <div class="publisher-loc">Cambridge</div><div class="publisher-name">Cambridge University Press</div>(pg. <div class="fpage">185</div>-<div class="lpage">91</div>)</div></div></div></div><div content-id="DST053C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fu</div> <div class="given-names">X.P.</div></div>,  <div class="name"><div class="surname">Ning</div> <div class="given-names">G.G.</div></div>,  <div class="name"><div class="surname">Gao</div> <div class="given-names">L.P.</div></div>,  <div class="name"><div class="surname">Bao</div> <div class="given-names">M.Z.</div></div>. </div><div class="article-title">Genetic diversity of <em>Dianthus</em> accessions as assessed using two molecular marker systems (SRAPs and ISSRs) and morphological traits</div>, <div class="source">Sci. Hortic.</div> , <div class="year">2008</div>, vol. <div class="volume">117</div> (pg. <div class="fpage">263</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20of%20Dianthus%20accessions%20as%20assessed%20using%20two%20molecular%20marker%20systems%20(SRAPs%20and%20ISSRs)%20and%20morphological%20traits&amp;author=X.P.%20Fu&amp;author=G.G.%20Ning&amp;author=L.P.%20Gao&amp;author=M.Z.%20Bao&amp;publication_year=2008&amp;journal=Sci.%20Hortic.&amp;volume=117&amp;pages=263-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.scienta.2008.04.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.scienta.2008.04.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.scienta.2008.04.001"> </span></div> </div></div></div></div></div><div content-id="DST053C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Momose</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nakayama</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Itoh</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Umemoto</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Toguri</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ozeki</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">An active <em>hAT</em> transposable element causing bud mutation of carnation by insertion into the <em>flavonoid 3′-hydroxylase</em> gene</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="year">2013</div>, vol. <div class="volume">288</div> (pg. <div class="fpage">175</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20active%20hAT%20transposable%20element%20causing%20bud%20mutation%20of%20carnation%20by%20insertion%20into%20the%20flavonoid%203%E2%80%B2-hydroxylase%20gene&amp;author=M.%20Momose&amp;author=M.%20Nakayama&amp;author=Y.%20Itoh&amp;author=N.%20Umemoto&amp;author=T.%20Toguri&amp;author=Y.%20Ozeki&amp;publication_year=2013&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=288&amp;pages=175-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-013-0742-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-013-0742-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-013-0742-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23543146" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tanaka</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Brugliera</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Flower colour and cytochromes P450</div>, <div class="source">Phil. Trans. R. Soc. B.</div> , <div class="year">2013</div>, vol. <div class="volume">368</div> pg. <div class="fpage">20120432</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Flower%20colour%20and%20cytochromes%20P450&amp;author=Y.%20Tanaka&amp;author=F.%20Brugliera&amp;publication_year=2013&amp;journal=Phil.%20Trans.%20R.%20Soc.%20B.&amp;volume=368&amp;pages=20120432" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rstb.2012.0432" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frstb.2012.0432" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frstb.2012.0432"> </span></div> </div></div></div></div></div><div content-id="DST053C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tanaka</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Sasaki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Ohmiya</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Biosynthesis of plant pigments: anthocyanins, betalains and carotenoids</div>, <div class="source">Plant, J.</div> , <div class="year">2008</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">733</div>-<div class="lpage">49</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biosynthesis%20of%20plant%20pigments%3A%20anthocyanins%2C%20betalains%20and%20carotenoids&amp;author=Y.%20Tanaka&amp;author=N.%20Sasaki&amp;author=A.%20Ohmiya&amp;publication_year=2008&amp;journal=Plant%2C%20J.&amp;volume=54&amp;pages=733-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.2008.03447.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.2008.03447.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.2008.03447.x"> </span></div> </div></div></div></div></div><div content-id="DST053C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Onozaki</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Improvement of flower vase life using cross-breeding techniques in carnation (<em>Dianthus caryophyllus</em> L.)</div>, <div class="source">JARQ</div> , <div class="year">2008</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">137</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improvement%20of%20flower%20vase%20life%20using%20cross-breeding%20techniques%20in%20carnation%20(Dianthus%20caryophyllus%20L.)&amp;author=T.%20Onozaki&amp;publication_year=2008&amp;journal=JARQ&amp;volume=42&amp;pages=137-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.6090/jarq.42.137" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.6090%2Fjarq.42.137" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.6090%2Fjarq.42.137"> </span></div> </div></div></div></div></div><div content-id="DST053C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Halevy</div> <div class="given-names">A.H.</div></div>,  <div class="name"><div class="surname">Mayak</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Senescence and postharvest physiology of cut flowers, part 2</div>, <div class="source">Hort. Rev.</div> , <div class="year">1981</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">59</div>-<div class="lpage">143</div>)</div></div></div></div><div content-id="DST053C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Onozaki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yagi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tanase</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Shibata</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Crossings and selections for six generations based on flower vase life to create lines with ethylene resistance or ultra-long vase life in carnations (<em>Dianthus caryophyllus</em> L.)</div>, <div class="source">J. Japan. Soc. Hort. Sci.</div> , <div class="year">2011</div>, vol. <div class="volume">80</div> (pg. <div class="fpage">486</div>-<div class="lpage">98</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Crossings%20and%20selections%20for%20six%20generations%20based%20on%20flower%20vase%20life%20to%20create%20lines%20with%20ethylene%20resistance%20or%20ultra-long%20vase%20life%20in%20carnations%20(Dianthus%20caryophyllus%20L.)&amp;author=T.%20Onozaki&amp;author=M.%20Yagi&amp;author=K.%20Tanase&amp;author=M.%20Shibata&amp;publication_year=2011&amp;journal=J.%20Japan.%20Soc.%20Hort.%20Sci.&amp;volume=80&amp;pages=486-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2503/jjshs1.80.486" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2503%2Fjjshs1.80.486" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2503%2Fjjshs1.80.486"> </span></div> </div></div></div></div></div><div content-id="DST053C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hoeberichts</div> <div class="given-names">F.A.</div></div>,  <div class="name"><div class="surname">van Doorn</div> <div class="given-names">W.G.</div></div>,  <div class="name"><div class="surname">Vorst</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Hall</div> <div class="given-names">R.D.</div></div>,  <div class="name"><div class="surname">van Wordragen</div> <div class="given-names">M.F.</div></div>. </div><div class="article-title">Sucrose prevents up-regulation of senescence-associated genes in carnation petals</div>, <div class="source">J. Exp. Bot.</div> , <div class="year">2007</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">2873</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sucrose%20prevents%20up-regulation%20of%20senescence-associated%20genes%20in%20carnation%20petals&amp;author=F.A.%20Hoeberichts&amp;author=W.G.%20van%20Doorn&amp;author=O.%20Vorst&amp;author=R.D.%20Hall&amp;author=M.F.%20van%20Wordragen&amp;publication_year=2007&amp;journal=J.%20Exp.%20Bot.&amp;volume=58&amp;pages=2873-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erm076" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferm076" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferm076"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17630294" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yagi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Onozaki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Taneya</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Construction of a genetic linkage map for the carnation by using RAPD and SSR markers and mapping quantitative trait loci (QTL) for resistance to bacterial wilt caused by <em>Burkholderia caryophylli</em></div>, <div class="source">J. Japan. Soc. Hort. Sci.</div> , <div class="year">2006</div>, vol. <div class="volume">75</div> (pg. <div class="fpage">166</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20genetic%20linkage%20map%20for%20the%20carnation%20by%20using%20RAPD%20and%20SSR%20markers%20and%20mapping%20quantitative%20trait%20loci%20(QTL)%20for%20resistance%20to%20bacterial%20wilt%20caused%20by%20Burkholderia%20caryophylli&amp;author=M.%20Yagi&amp;author=T.%20Onozaki&amp;author=M.%20Taneya&amp;publication_year=2006&amp;journal=J.%20Japan.%20Soc.%20Hort.%20Sci.&amp;volume=75&amp;pages=166-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2503/jjshs.75.166" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2503%2Fjjshs.75.166" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2503%2Fjjshs.75.166"> </span></div> </div></div></div></div></div><div content-id="DST053C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yagi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kimura</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tabata</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Onozaki</div> <div class="given-names">T.</div></div>. </div><div class="article-title">QTL analysis for resistance to bacterial wilt (<em>Burkholderia caryophylli</em>) in carnation (<em>Dianthus caryophyllus</em>) using an SSR-based genetic linkage map</div>, <div class="source">Mol. Breed.</div> , <div class="year">2012</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">495</div>-<div class="lpage">509</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL%20analysis%20for%20resistance%20to%20bacterial%20wilt%20(Burkholderia%20caryophylli)%20in%20carnation%20(Dianthus%20caryophyllus)%20using%20an%20SSR-based%20genetic%20linkage%20map&amp;author=M.%20Yagi&amp;author=T.%20Kimura&amp;author=T.%20Yamamoto&amp;author=S.%20Isobe&amp;author=S.%20Tabata&amp;author=T.%20Onozaki&amp;publication_year=2012&amp;journal=Mol.%20Breed.&amp;volume=30&amp;pages=495-509" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-011-9639-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-011-9639-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-011-9639-x"> </span></div> </div></div></div></div></div><div content-id="DST053C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tanase</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nishitani</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Transcriptome analysis of carnation (<em>Dianthus caryophyllus</em> L.) based on next-generation sequencing technology</div>, <div class="source">BMC Genomics</div> , <div class="year">2012</div>, vol. <div class="volume">13</div> pg. <div class="fpage">292</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20analysis%20of%20carnation%20(Dianthus%20caryophyllus%20L.)%20based%20on%20next-generation%20sequencing%20technology&amp;author=K.%20Tanase&amp;author=C.%20Nishitani&amp;author=H.%20Hirakawa&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=292" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-292" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-292" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-292"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22747974" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yagi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Construction of a reference genetic linkage map for carnation (<em>Dianthus caryophyllus</em> L.)</div>, <div class="source">BMC Genomics</div> , <div class="year">2013</div>, vol. <div class="volume">14</div> pg. <div class="fpage">734</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20reference%20genetic%20linkage%20map%20for%20carnation%20(Dianthus%20caryophyllus%20L.)&amp;author=M.%20Yagi&amp;author=T.%20Yamamoto&amp;author=S.%20Isobe&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=734" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-734" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-734" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-734"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24160306" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gatt</div> <div class="given-names">M.K.</div></div>,  <div class="name"><div class="surname">Hammett</div> <div class="given-names">K.R.W.</div></div>,  <div class="name"><div class="surname">Markham</div> <div class="given-names">K.R.</div></div>,  <div class="name"><div class="surname">Murray</div> <div class="given-names">B.G.</div></div>. </div><div class="article-title">Yellow pinks: interspecific hybridization between <em>Dianthus plumarius</em> and related species with yellow flowers</div>, <div class="source">Sci. Hortic.</div> , <div class="year">1998</div>, vol. <div class="volume">77</div> (pg. <div class="fpage">207</div>-<div class="lpage">18</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Yellow%20pinks%3A%20interspecific%20hybridization%20between%20Dianthus%20plumarius%20and%20related%20species%20with%20yellow%20flowers&amp;author=M.K.%20Gatt&amp;author=K.R.W.%20Hammett&amp;author=K.R.%20Markham&amp;author=B.G.%20Murray&amp;publication_year=1998&amp;journal=Sci.%20Hortic.&amp;volume=77&amp;pages=207-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0304-4238(98)00177-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0304-4238(98)00177-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0304-4238(98)00177-0"> </span></div> </div></div></div></div></div><div content-id="DST053C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Agullo-Anton</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Olmos</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Perez-Perez</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Acosta</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Evaluation of ploidy level and endoreduplication in carnation (<em>Dianthus</em> spp.)</div>, <div class="source">Plant Sci.</div> , <div class="year">2013</div>, vol. <div class="volume">20,1–202</div> (pg. <div class="fpage">1</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20ploidy%20level%20and%20endoreduplication%20in%20carnation%20(Dianthus%20spp.)&amp;author=M.A.%20Agullo-Anton&amp;author=E.%20Olmos&amp;author=J.M.%20Perez-Perez&amp;author=M.%20Acosta&amp;publication_year=2013&amp;journal=Plant%20Sci.&amp;volume=20,1–202&amp;pages=1-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.plantsci.2012.11.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plantsci.2012.11.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.plantsci.2012.11.006"> </span></div> </div></div></div></div></div><div content-id="DST053C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Onozaki</div> <div class="given-names">T.</div></div>. </div><div class="collab">Japan. Soc. Hort. Sci</div><div class="article-title">Carnation</div>, <div class="source">Horticulture in Japan 2006</div> , <div class="year">2006</div><div class="publisher-loc">Kyoto</div><div class="publisher-name">Shoukadoh Publication, Dept. of Publishing of Nakanishi Printing Co., Ltd.</div>(pg. <div class="fpage">223</div>-<div class="lpage">30</div>)</div></div></div></div><div content-id="DST053C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yagi</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Application of DNA markers for breeding carnations resistant to bacterial wilt</div>, <div class="source">JARQ</div> , <div class="year">2013</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">29</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Application%20of%20DNA%20markers%20for%20breeding%20carnations%20resistant%20to%20bacterial%20wilt&amp;author=M.%20Yagi&amp;publication_year=2013&amp;journal=JARQ&amp;volume=47&amp;pages=29-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.6090/jarq.47.29" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.6090%2Fjarq.47.29" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.6090%2Fjarq.47.29"> </span></div> </div></div></div></div></div><div content-id="DST053C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van Nieuwerburgh</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Thompson</div> <div class="given-names">R.C.</div></div>,  <div class="name"><div class="surname">Ledesma</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Illumina mate-paired DNA sequencing-library preparation using Cre-Lox recombination</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2012</div>, vol. <div class="volume">40</div> pg. <div class="fpage">e24</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Illumina%20mate-paired%20DNA%20sequencing-library%20preparation%20using%20Cre-Lox%20recombination&amp;author=F.%20Van%20Nieuwerburgh&amp;author=R.C.%20Thompson&amp;author=J.%20Ledesma&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=e24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22127871" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Peng</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Nath</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">Generation of long insert pairs using a Cre-LoxP inverse PCR approach</div>, <div class="source">PLoS ONE</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e29437</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Generation%20of%20long%20insert%20pairs%20using%20a%20Cre-LoxP%20inverse%20PCR%20approach&amp;author=Z.%20Peng&amp;author=Z.%20Zhao&amp;author=N.%20Nath&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e29437" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0029437" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0029437" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0029437"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22253722" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Yuan</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Yiu</div> <div class="given-names">S.M.</div></div>, et al. </div><div class="article-title">COPE: an accurate k-mer-based pair-end reads connection tool to facilitate genome assembly</div>, <div class="source">Bioinformatics</div> , <div class="year">2012</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">2870</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=COPE%3A%20an%20accurate%20k-mer-based%20pair-end%20reads%20connection%20tool%20to%20facilitate%20genome%20assembly&amp;author=B.%20Liu&amp;author=J.%20Yuan&amp;author=S.M.%20Yiu&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=2870-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts563" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts563" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbts563"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23044551" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mehlquist</div> <div class="given-names">G.A.L.</div></div>,  <div class="name"><div class="surname">Geissman</div> <div class="given-names">T.A.</div></div>. </div><div class="article-title">Inheritance in the carnation (<em>Dianthus caryophyllus</em>). III. Inheritance of flower color</div>, <div class="source">Ann. Missouri Bot. Gard.</div> , <div class="year">1947</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">39</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inheritance%20in%20the%20carnation%20(Dianthus%20caryophyllus).%20III.%20Inheritance%20of%20flower%20color&amp;author=G.A.L.%20Mehlquist&amp;author=T.A.%20Geissman&amp;publication_year=1947&amp;journal=Ann.%20Missouri%20Bot.%20Gard.&amp;volume=34&amp;pages=39-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2394439" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2394439" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2394439"> </span></div> </div></div></div></div></div><div content-id="DST053C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Itoh</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Higeta</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Yoshida</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ozeki</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Excision of transposable elements from the chalcone isomerase and dihydroflavonol 4-reductase genes may contribute to the variegation of the yellow-flowered carnation (<em>Dianthus caryophyllus</em>)</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2002</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">578</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Excision%20of%20transposable%20elements%20from%20the%20chalcone%20isomerase%20and%20dihydroflavonol%204-reductase%20genes%20may%20contribute%20to%20the%20variegation%20of%20the%20yellow-flowered%20carnation%20(Dianthus%20caryophyllus)&amp;author=Y.%20Itoh&amp;author=D.%20Higeta&amp;author=A.%20Suzuki&amp;author=H.%20Yoshida&amp;author=Y.%20Ozeki&amp;publication_year=2002&amp;journal=Plant%20Cell%20Physiol.&amp;volume=43&amp;pages=578-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcf065" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcf065" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcf065"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12040106" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sasaki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Nishizaki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Uchida</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Identification of the <em>glutathione</em> S-<em>transferase</em> gene responsible for flower color intensity in carnations</div>, <div class="source">Plant Biotechnol.</div> , <div class="year">2012</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">223</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20the%20glutathione%20S-transferase%20gene%20responsible%20for%20flower%20color%20intensity%20in%20carnations&amp;author=N.%20Sasaki&amp;author=Y.%20Nishizaki&amp;author=Y.%20Uchida&amp;publication_year=2012&amp;journal=Plant%20Biotechnol.&amp;volume=29&amp;pages=223-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.5511/plantbiotechnology.12.0120a" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.5511%2Fplantbiotechnology.12.0120a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.5511%2Fplantbiotechnology.12.0120a"> </span></div> </div></div></div></div></div><div content-id="DST053C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Page</div> <div class="given-names">R.D.</div></div>. </div><div class="article-title">TreeView: an application to display phylogenetic trees on personal computers</div>, <div class="source">Comput. Appl. Biosci.</div> , <div class="year">1996</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">357</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TreeView%3A%20an%20application%20to%20display%20phylogenetic%20trees%20on%20personal%20computers&amp;author=R.D.%20Page&amp;publication_year=1996&amp;journal=Comput.%20Appl.%20Biosci.&amp;volume=12&amp;pages=357-8" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8902363" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Christinet</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Burdet</div> <div class="given-names">F.X.</div></div>,  <div class="name"><div class="surname">Zaiko</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hinz</div> <div class="given-names">U.</div></div>,  <div class="name"><div class="surname">Zryd</div> <div class="given-names">J.P.</div></div>. </div><div class="article-title">Characterization and functional identification of a novel plant 4,5-extradiol dioxygenase involved in betalain pigment biosynthesis in <em>Portulaca grandiflora</em></div>, <div class="source">Plant Physiol.</div> , <div class="year">2004</div>, vol. <div class="volume">134</div> (pg. <div class="fpage">265</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20and%20functional%20identification%20of%20a%20novel%20plant%204%2C5-extradiol%20dioxygenase%20involved%20in%20betalain%20pigment%20biosynthesis%20in%20Portulaca%20grandiflora&amp;author=L.%20Christinet&amp;author=F.X.%20Burdet&amp;author=M.%20Zaiko&amp;author=U.%20Hinz&amp;author=J.P.%20Zryd&amp;publication_year=2004&amp;journal=Plant%20Physiol.&amp;volume=134&amp;pages=265-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.103.031914" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.103.031914" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.103.031914"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14730069" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hatlestad</div> <div class="given-names">G.J.</div></div>,  <div class="name"><div class="surname">Sunnadeniya</div> <div class="given-names">R.M.</div></div>,  <div class="name"><div class="surname">Akhavan</div> <div class="given-names">N.A.</div></div>, et al. </div><div class="article-title">The beet <em>R</em> locus encodes a new cytochrome P450 required for red betalain production</div>, <div class="source">Nat. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">816</div>-<div class="lpage">20</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20beet%20R%20locus%20encodes%20a%20new%20cytochrome%20P450%20required%20for%20red%20betalain%20production&amp;author=G.J.%20Hatlestad&amp;author=R.M.%20Sunnadeniya&amp;author=N.A.%20Akhavan&amp;publication_year=2012&amp;journal=Nat.%20Genet.&amp;volume=44&amp;pages=816-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2297" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2297" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2297"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22660548" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sasaki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Adachi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Koda</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ozeki</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Detection of UDP-glucose:<em>cyclo</em>-DOPA 5-<em>O</em>-glucosyltransferase activity in four o'clocks (<em>Mirabilis jalapa</em> L.)</div>, <div class="source">FEBS Lett.</div> , <div class="year">2004</div>, vol. <div class="volume">568</div> (pg. <div class="fpage">159</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20UDP-glucose%3Acyclo-DOPA%205-O-glucosyltransferase%20activity%20in%20four%20o'clocks%20(Mirabilis%20jalapa%20L.)&amp;author=N.%20Sasaki&amp;author=T.%20Adachi&amp;author=T.%20Koda&amp;author=Y.%20Ozeki&amp;publication_year=2004&amp;journal=FEBS%20Lett.&amp;volume=568&amp;pages=159-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.febslet.2004.04.097" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.febslet.2004.04.097" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.febslet.2004.04.097"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15196939" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sasaki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Wada</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Koda</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kasahara</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Adachi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ozeki</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Isolation and characterization of cDNAs encoding an enzyme with glucosyltransferase activity for <em>cyclo</em>-DOPA from four o’clocks and feather cockscombs</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2005</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">666</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20cDNAs%20encoding%20an%20enzyme%20with%20glucosyltransferase%20activity%20for%20cyclo-DOPA%20from%20four%20o%E2%80%99clocks%20and%20feather%20cockscombs&amp;author=N.%20Sasaki&amp;author=K.%20Wada&amp;author=T.%20Koda&amp;author=K.%20Kasahara&amp;author=T.%20Adachi&amp;author=Y.%20Ozeki&amp;publication_year=2005&amp;journal=Plant%20Cell%20Physiol.&amp;volume=46&amp;pages=666-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pci064" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpci064" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpci064"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15695438" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tan</div> <div class="given-names">B.C.</div></div>,  <div class="name"><div class="surname">Joseph</div> <div class="given-names">L.M.</div></div>,  <div class="name"><div class="surname">Deng</div> <div class="given-names">W.T.</div></div>, et al. </div><div class="article-title">Molecular characterization of the <em>Arabidopsis</em> 9-<em>cis</em>-epoxycarotenoid dioxygenase gene family</div>, <div class="source">Plant J.</div> , <div class="year">2003</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">44</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20characterization%20of%20the%20Arabidopsis%209-cis-epoxycarotenoid%20dioxygenase%20gene%20family&amp;author=B.C.%20Tan&amp;author=L.M.%20Joseph&amp;author=W.T.%20Deng&amp;publication_year=2003&amp;journal=Plant%20J.&amp;volume=35&amp;pages=44-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.2003.01786.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.2003.01786.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.2003.01786.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12834401" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohmiya</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Tanase</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hirashima</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yamamizo</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Yagi</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Analysis of carotenogenic gene expression in petals and leaves of carnation (<em>Dianthus caryophyllus</em> L.)</div>, <div class="source">Plant Breed.</div> , <div class="year">2013</div>, vol. <div class="volume">132</div> (pg. <div class="fpage">423</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20carotenogenic%20gene%20expression%20in%20petals%20and%20leaves%20of%20carnation%20(Dianthus%20caryophyllus%20L.)&amp;author=A.%20Ohmiya&amp;author=K.%20Tanase&amp;author=M.%20Hirashima&amp;author=C.%20Yamamizo&amp;author=M.%20Yagi&amp;publication_year=2013&amp;journal=Plant%20Breed.&amp;volume=132&amp;pages=423-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbr.2013.132.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbr.2013.132.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbr.2013.132.issue-4"> </span></div> </div></div></div></div></div><div content-id="DST053C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wan</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Yuan</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Bo</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Shen</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Pang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Genome-wide analysis of NBS-encoding disease resistance genes in <em>Cucumis sativus</em> and phylogenic study of NBS-encoding genes in Cucurbitaceae crops</div>, <div class="source">BMC Genomics</div> , <div class="year">2013</div>, vol. <div class="volume">14</div> pg. <div class="fpage">109</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20NBS-encoding%20disease%20resistance%20genes%20in%20Cucumis%20sativus%20and%20phylogenic%20study%20of%20NBS-encoding%20genes%20in%20Cucurbitaceae%20crops&amp;author=H.%20Wan&amp;author=W.%20Yuan&amp;author=K.%20Bo&amp;author=J.%20Shen&amp;author=X.%20Pang&amp;author=J.%20Chen&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=109" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23418910" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garcia-Mas</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Benjak</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Sanseverino</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title">The genome of melon (<em>Cucumis melo</em> L.)</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2012</div>, vol. <div class="volume">109</div> (pg. <div class="fpage">11872</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20melon%20(Cucumis%20melo%20L.)&amp;author=J.%20Garcia-Mas&amp;author=A.%20Benjak&amp;author=W.%20Sanseverino&amp;publication_year=2012&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=109&amp;pages=11872-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1205415109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1205415109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1205415109"> </span></div> </div></div></div></div></div><div content-id="DST053C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Porter</div> <div class="given-names">B.W.</div></div>,  <div class="name"><div class="surname">Paidi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ming</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Alam</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nishijima</div> <div class="given-names">W.T.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">Y.J.</div></div>. </div><div class="article-title">Genome-wide analysis of <em>Carica papaya</em> reveals a small <em>NBS</em> resistance gene family</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">281</div> (pg. <div class="fpage">609</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20Carica%20papaya%20reveals%20a%20small%20NBS%20resistance%20gene%20family&amp;author=B.W.%20Porter&amp;author=M.%20Paidi&amp;author=R.%20Ming&amp;author=M.%20Alam&amp;author=W.T.%20Nishijima&amp;author=Y.J.%20Zhu&amp;publication_year=2009&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=281&amp;pages=609-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-009-0434-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-009-0434-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-009-0434-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19263082" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Andolfo</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Sanseverino</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Rombauts</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Overview of tomato (<em>Solanum lycopersicum</em>) candidate pathogen recognition genes reveals important <em>Solanum</em> R locus dynamics</div>, <div class="source">New Phytol.</div> , <div class="year">2013</div>, vol. <div class="volume">197</div> (pg. <div class="fpage">223</div>-<div class="lpage">37</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Overview%20of%20tomato%20(Solanum%20lycopersicum)%20candidate%20pathogen%20recognition%20genes%20reveals%20important%20Solanum%20R%20locus%20dynamics&amp;author=G.%20Andolfo&amp;author=W.%20Sanseverino&amp;author=S.%20Rombauts&amp;publication_year=2013&amp;journal=New%20Phytol.&amp;volume=197&amp;pages=223-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2012.04380.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2012.04380.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2012.04380.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23163550" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Velasco</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Zharkikh</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Troggio</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">A high quality draft consensus sequence of the genome of a heterozygous grapevine variety</div>, <div class="source">PLoS ONE</div> , <div class="year">2007</div>, vol. <div class="volume">2</div> pg. <div class="fpage">e1326</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high%20quality%20draft%20consensus%20sequence%20of%20the%20genome%20of%20a%20heterozygous%20grapevine%20variety&amp;author=R.%20Velasco&amp;author=A.%20Zharkikh&amp;author=M.%20Troggio&amp;publication_year=2007&amp;journal=PLoS%20ONE&amp;volume=2&amp;pages=e1326" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0001326" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0001326" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0001326"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18094749" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhou</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">J.Q.</div></div>, et al. </div><div class="article-title">Genome-wide identification of NBS genes in <em>japonica</em> rice reveals significant expansion of divergent non-TIR NBS-LRR genes</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="year">2004</div>, vol. <div class="volume">271</div> (pg. <div class="fpage">402</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20identification%20of%20NBS%20genes%20in%20japonica%20rice%20reveals%20significant%20expansion%20of%20divergent%20non-TIR%20NBS-LRR%20genes&amp;author=T.%20Zhou&amp;author=Y.%20Wang&amp;author=J.Q.%20Chen&amp;publication_year=2004&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=271&amp;pages=402-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-004-0990-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-004-0990-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-004-0990-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15014983" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tian</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Fan</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Thurau</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Jung</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Cai</div> <div class="given-names">D.</div></div>. </div><div class="article-title">The absence of TIR-type resistance gene analogues in the sugar beet (<em>Beta vulgaris</em> L.) genome</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">2004</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">40</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20absence%20of%20TIR-type%20resistance%20gene%20analogues%20in%20the%20sugar%20beet%20(Beta%20vulgaris%20L.)%20genome&amp;author=Y.%20Tian&amp;author=L.%20Fan&amp;author=T.%20Thurau&amp;author=C.%20Jung&amp;author=D.%20Cai&amp;publication_year=2004&amp;journal=J.%20Mol.%20Evol.&amp;volume=58&amp;pages=40-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-003-2524-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-003-2524-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-003-2524-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14743313" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jones</div> <div class="given-names">M.L.</div></div>,  <div class="name"><div class="surname">Woodson</div> <div class="given-names">W.R.</div></div>. </div><div class="article-title">Differential expression of three members of the 1-aminocyclopropane-1-carboxylate synthase gene family in carnation</div>, <div class="source">Plant Physiol.</div> , <div class="year">1999</div>, vol. <div class="volume">119</div> (pg. <div class="fpage">755</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20expression%20of%20three%20members%20of%20the%201-aminocyclopropane-1-carboxylate%20synthase%20gene%20family%20in%20carnation&amp;author=M.L.%20Jones&amp;author=W.R.%20Woodson&amp;publication_year=1999&amp;journal=Plant%20Physiol.&amp;volume=119&amp;pages=755-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.119.2.755" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.119.2.755" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.119.2.755"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9952472" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamagami</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Tsuchisaka</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Yamada</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Haddon</div> <div class="given-names">W.F.</div></div>,  <div class="name"><div class="surname">Harden</div> <div class="given-names">L.A.</div></div>,  <div class="name"><div class="surname">Theologis</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Biochemical diversity among the 1-amino-cyclopropane-1-carboxylate synthase isozymes encoded by the <em>Arabidopsis</em> gene family</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2003</div>, vol. <div class="volume">278</div> (pg. <div class="fpage">49102</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biochemical%20diversity%20among%20the%201-amino-cyclopropane-1-carboxylate%20synthase%20isozymes%20encoded%20by%20the%20Arabidopsis%20gene%20family&amp;author=T.%20Yamagami&amp;author=A.%20Tsuchisaka&amp;author=K.%20Yamada&amp;author=W.F.%20Haddon&amp;author=L.A.%20Harden&amp;author=A.%20Theologis&amp;publication_year=2003&amp;journal=J.%20Biol.%20Chem.&amp;volume=278&amp;pages=49102-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M308297200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M308297200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M308297200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12968022" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ichimura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kohata</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Koketsu</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Identification of pinitol as a main sugar constituent and changes in its content during flower bud development in carnation (<em>Dianthus caryophyllus</em> L.)</div>, <div class="source">J. Plant Physiol.</div> , <div class="year">1998</div>, vol. <div class="volume">152</div> (pg. <div class="fpage">363</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20pinitol%20as%20a%20main%20sugar%20constituent%20and%20changes%20in%20its%20content%20during%20flower%20bud%20development%20in%20carnation%20(Dianthus%20caryophyllus%20L.)&amp;author=K.%20Ichimura&amp;author=K.%20Kohata&amp;author=M.%20Koketsu&amp;publication_year=1998&amp;journal=J.%20Plant%20Physiol.&amp;volume=152&amp;pages=363-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0176-1617(98)80248-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0176-1617(98)80248-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0176-1617(98)80248-4"> </span></div> </div></div></div></div></div><div content-id="DST053C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nishitani</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tominaga</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Endo-xyloglucan transferase, a novel class of glycosyltransferase that catalyzes transfer of a segment of xyloglucan molecule to another xyloglucan molecule</div>, <div class="source">, J. Biol. Chem.</div> , <div class="year">1992</div>, vol. <div class="volume">267</div> (pg. <div class="fpage">21058</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Endo-xyloglucan%20transferase%2C%20a%20novel%20class%20of%20glycosyltransferase%20that%20catalyzes%20transfer%20of%20a%20segment%20of%20xyloglucan%20molecule%20to%20another%20xyloglucan%20molecule&amp;author=K.%20Nishitani&amp;author=R.%20Tominaga&amp;publication_year=1992&amp;journal=%2C%20J.%20Biol.%20Chem.&amp;volume=267&amp;pages=21058-64" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1400418" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamada</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Takahashi</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Fujitani</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Cell wall extensibility and effect of cell-wall-loosening proteins during rose flower opening</div>, <div class="source">J. Japan. Soc. Hort. Sci.</div> , <div class="year">2009</div>, vol. <div class="volume">78</div> (pg. <div class="fpage">242</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cell%20wall%20extensibility%20and%20effect%20of%20cell-wall-loosening%20proteins%20during%20rose%20flower%20opening&amp;author=K.%20Yamada&amp;author=R.%20Takahashi&amp;author=C.%20Fujitani&amp;publication_year=2009&amp;journal=J.%20Japan.%20Soc.%20Hort.%20Sci.&amp;volume=78&amp;pages=242-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2503/jjshs1.78.242" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2503%2Fjjshs1.78.242" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2503%2Fjjshs1.78.242"> </span></div> </div></div></div></div></div><div content-id="DST053C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yokoyama</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Nishitani</div> <div class="given-names">K.</div></div>. </div><div class="article-title">A comprehensive expression analysis of all members of a gene family encoding cell-wall enzymes allowed us to predict <em>cis</em>-regulatory regions involved in cell-wall construction in specific organs of Arabidopsis</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2001</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">1025</div>-<div class="lpage">33</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comprehensive%20expression%20analysis%20of%20all%20members%20of%20a%20gene%20family%20encoding%20cell-wall%20enzymes%20allowed%20us%20to%20predict%20cis-regulatory%20regions%20involved%20in%20cell-wall%20construction%20in%20specific%20organs%20of%20Arabidopsis&amp;author=R.%20Yokoyama&amp;author=K.%20Nishitani&amp;publication_year=2001&amp;journal=Plant%20Cell%20Physiol.&amp;volume=42&amp;pages=1025-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pce154" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpce154" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpce154"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11673616" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yokoyama</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Rose</div> <div class="given-names">J.K.</div></div>,  <div class="name"><div class="surname">Nishitani</div> <div class="given-names">K.</div></div>. </div><div class="article-title">A surprising diversity and abundance of xyloglucan endotransglucosylase/hydrolases in rice. Classification and expression analysis</div>, <div class="source">Plant Physiol.</div> , <div class="year">2004</div>, vol. <div class="volume">134</div> (pg. <div class="fpage">1088</div>-<div class="lpage">99</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20surprising%20diversity%20and%20abundance%20of%20xyloglucan%20endotransglucosylase%2Fhydrolases%20in%20rice.%20Classification%20and%20expression%20analysis&amp;author=R.%20Yokoyama&amp;author=J.K.%20Rose&amp;author=K.%20Nishitani&amp;publication_year=2004&amp;journal=Plant%20Physiol.&amp;volume=134&amp;pages=1088-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.103.035261" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.103.035261" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.103.035261"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14988479" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Harada</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Torii</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Morita</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Cloning, characterization, and expression of xyloglucan endotransglucosylase/hydrolase and expansin genes associated with petal growth and development during carnation flower opening</div>, <div class="source">J. Exp. Bot.</div> , <div class="year">2011</div>, vol. <div class="volume">62</div> (pg. <div class="fpage">815</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%2C%20characterization%2C%20and%20expression%20of%20xyloglucan%20endotransglucosylase%2Fhydrolase%20and%20expansin%20genes%20associated%20with%20petal%20growth%20and%20development%20during%20carnation%20flower%20opening&amp;author=T.%20Harada&amp;author=Y.%20Torii&amp;author=S.%20Morita&amp;publication_year=2011&amp;journal=J.%20Exp.%20Bot.&amp;volume=62&amp;pages=815-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erq319" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferq319" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferq319"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20959626" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST053C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kishimoto</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yagi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Onozaki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yamaguchi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nakayama</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Oyama-Okubo</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Analysis of scents emitted from flowers of interspecific hybrids between carnation and fragrant wild <em>Dianthus</em> species</div>, <div class="source">J. Japan. Soc. Hort. Sci.</div> , <div class="year">2013</div>, vol. <div class="volume">82</div> (pg. <div class="fpage">145</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20scents%20emitted%20from%20flowers%20of%20interspecific%20hybrids%20between%20carnation%20and%20fragrant%20wild%20Dianthus%20species&amp;author=K.%20Kishimoto&amp;author=M.%20Yagi&amp;author=T.%20Onozaki&amp;author=H.%20Yamaguchi&amp;author=M.%20Nakayama&amp;author=N.%20Oyama-Okubo&amp;publication_year=2013&amp;journal=J.%20Japan.%20Soc.%20Hort.%20Sci.&amp;volume=82&amp;pages=145-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2503/jjshs1.82.145" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2503%2Fjjshs1.82.145" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2503%2Fjjshs1.82.145"> </span></div> </div></div></div></div></div><div content-id="DST053C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">D'Auria</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Pichersky</div> <div class="given-names">E.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Romeos</div> <div class="given-names">J.T.</div></div>. </div><div class="article-title">The SABATH family of methyltransferases in <em>Arabidopsis thaliana</em> and other plant species</div>, <div class="source">Recent Advances in Phytochemistry</div> , <div class="year">2003</div>, vol. <div class="volume">vol. 37</div> <div class="publisher-loc">Oxford</div><div class="publisher-name">Elsevier Science Ltd</div>(pg. <div class="fpage">253</div>-<div class="lpage">83</div>)</div></div></div></div><div content-id="DST053C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST053C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barkman</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Martins</div> <div class="given-names">T.R.</div></div>,  <div class="name"><div class="surname">Sutton</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Stout</div> <div class="given-names">J.T.</div></div>. </div><div class="article-title">Positive selection for single amino acid change promotes substrate discrimination of a plant volatile-producing enzyme</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">1320</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Positive%20selection%20for%20single%20amino%20acid%20change%20promotes%20substrate%20discrimination%20of%20a%20plant%20volatile-producing%20enzyme&amp;author=T.J.%20Barkman&amp;author=T.R.%20Martins&amp;author=E.%20Sutton&amp;author=J.T.%20Stout&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=1320-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm053" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm053" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm053"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17374877" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id="fn1"><div class="fn"><div class="footnote-content">Edited by Dr Katsumi Isono</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=393171&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst053/2/dst053-dst053supp_tables.xlsx?Expires=2147483647&amp;Signature=rMAsmBKFQ6oW9DpostwfFZM4sR7NUk3zv8Y6MSbOegWeruSCTM~xuj7cLIPkvenAVbbK7K3u2NozcpApM~DrZqnV~mHw7dk7GtIg7UMTZMV1qmPBUvjWGOusPP2oliOZRQvGrEhW96-T0PWd~GtHIXfAb8UyYMPQ~lkuHWhnnAQMEe0bRyYSTAJtfUrYF8vYsHJhXh1uQgSkmgd5CcZNp11b3uOqqzCM5gP66xlGEuQX5KflZfE18xOph4ShDOFBkWyKnh4FfHuwBtfD~lLsjLwAjgQTLlGBtGbDNmdHMYqDNQpgXPU~6rWkvTY8hkSaMZvP3tFYTNGQ3HmJ2xoktQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/3/10.1093/dnares/dst053/2/dst053-dst053supp_figures.pdf?Expires=2147483647&amp;Signature=xINLjOr0dAm6Vqs8oH41~2TfadWgptFr4NCa1NNzr003rqS3506ZopvwpZQ5PjFUCwCi9-YbOKL1QLTrBLj72gwcyt4lN9pamMF1sQzrWAMFa5CqmR3I1L79r22KG6CXDFfTwId~ODuFDEq0PYMO6DAOITcVZbL7OY0pc25YAn6OwafZgfiUvRAjUiIJEhQOhvDMxT9D4IBsiInPwHwYZAGBKgUhD95I8jg-N-Hz9Zwdh~75FlBC21wes5CKTAFLRA~Y0PHDTcOzIKGnZUr8xGB56SMHEx-q4E7TNMRdT0-L3TFoq9szHNViwC7f0EapaW2u7eWT~U4j3ldFEXDtJg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">920</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897900001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">29</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst053"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="393171"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000339897900001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f3%2f231%2f393171">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000339897900001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (29)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F3%2F231%2F393171" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/393171'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Sequence Analysis of the Genome of Carnation (Dianthus caryophyllus L.)" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst053");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f3%2f231%2f393171">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

