

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/18/1/65/377199","@type":"ScholarlyArticle","name":"Sequence Analysis of the Genome of an Oil-Bearing Tree,  Jatropha curcas  L.","datePublished":"2010-12-13","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/18/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2011-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsq030","keywords":["<italic>Jatropha curcas</italic> L.","genome sequencing","cDNA sequencing","microsatellite markers"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/18/1/65/377199","author":[{"name":"Sato, Shusei","affiliation":"1  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Hirakawa, Hideki","affiliation":"1  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Isobe, Sachiko","affiliation":"1  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Fukai, Eigo","affiliation":"1  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Watanabe, Akiko","affiliation":"1  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kato, Midori","affiliation":"1  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kawashima, Kumiko","affiliation":"1  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Minami, Chiharu","affiliation":"1  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Muraki, Akiko","affiliation":"1  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Nakazaki, Naomi","affiliation":"1  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Takahashi, Chika","affiliation":"1  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Nakayama, Shinobu","affiliation":"1  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kishida, Yoshie","affiliation":"1  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kohara, Mitsuyo","affiliation":"1  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Yamada, Manabu","affiliation":"1  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Tsuruoka, Hisano","affiliation":"1  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Sasamoto, Shigemi","affiliation":"1  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Tabata, Satoshi","affiliation":"1  Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Aizu, Tomoyuki","affiliation":"2  Center for Genetic Resource Information ,  National Institute of Genetics ,  Yata 1111, Mishima, Shizuoka 411-8540 ,  Japan","@type":"Person"},{"name":"Toyoda, Atsushi","affiliation":"2  Center for Genetic Resource Information ,  National Institute of Genetics ,  Yata 1111, Mishima, Shizuoka 411-8540 ,  Japan","@type":"Person"},{"name":"Shin-i, Tadasu","affiliation":"2  Center for Genetic Resource Information ,  National Institute of Genetics ,  Yata 1111, Mishima, Shizuoka 411-8540 ,  Japan","@type":"Person"},{"name":"Minakuchi, Yohei","affiliation":"2  Center for Genetic Resource Information ,  National Institute of Genetics ,  Yata 1111, Mishima, Shizuoka 411-8540 ,  Japan","@type":"Person"},{"name":"Kohara, Yuji","affiliation":"2  Center for Genetic Resource Information ,  National Institute of Genetics ,  Yata 1111, Mishima, Shizuoka 411-8540 ,  Japan","@type":"Person"},{"name":"Fujiyama, Asao","affiliation":"2  Center for Genetic Resource Information ,  National Institute of Genetics ,  Yata 1111, Mishima, Shizuoka 411-8540 ,  Japan    3  Informatics Research Division ,  National Institute of Informatics ,  2-1-2 Hitotsubashi, Chiyoda-ku, Tokyo 101-8430 ,  Japan","@type":"Person"},{"name":"Tsuchimoto, Suguru","affiliation":"4  Institute of Molecular and Cellular Biosciences ,  The University of Tokyo ,  1-1-1, Yayoi, Bunkyo-ku, Tokyo 113-0032 ,  Japan","@type":"Person"},{"name":"Kajiyama, Shin'ichiro","affiliation":"5  Graduate School of Biology Oriented Science and Technology ,  Kinki University ,  930 Nishi-Mitani, Kinokawa, Wakayama 649-6493 ,  Japan","@type":"Person"},{"name":"Makigano, Eri","affiliation":"6  Graduate School of Human Development and Environment ,  Kobe University ,  Tsurukabuto 3-11, Nada, Kobe 657-8501 ,  Japan","@type":"Person"},{"name":"Ohmido, Nobuko","affiliation":"6  Graduate School of Human Development and Environment ,  Kobe University ,  Tsurukabuto 3-11, Nada, Kobe 657-8501 ,  Japan","@type":"Person"},{"name":"Shibagaki, Nakako","affiliation":"7  Bioengineering for the Interest of Environmental Sustainability (Sumitomo Electric Industries) ,  Graduate School of Engineering, Osaka University ,  1-1 Yamadaoka, Suita, Osaka 565-0871 ,  Japan","@type":"Person"},{"name":"Cartagena, Joyce A.","affiliation":"7  Bioengineering for the Interest of Environmental Sustainability (Sumitomo Electric Industries) ,  Graduate School of Engineering, Osaka University ,  1-1 Yamadaoka, Suita, Osaka 565-0871 ,  Japan","@type":"Person"},{"name":"Wada, Naoki","affiliation":"7  Bioengineering for the Interest of Environmental Sustainability (Sumitomo Electric Industries) ,  Graduate School of Engineering, Osaka University ,  1-1 Yamadaoka, Suita, Osaka 565-0871 ,  Japan","@type":"Person"},{"name":"Kohinata, Tsutomu","affiliation":"8  Laboratory of Dynamic Cell Biology, Department of Biotechnology ,  Graduate School of Engineering, Osaka University ,  1-1 Yamadaoka, Suita, Osaka 565-0871 ,  Japan","@type":"Person"},{"name":"Atefeh, Alipour","affiliation":"8  Laboratory of Dynamic Cell Biology, Department of Biotechnology ,  Graduate School of Engineering, Osaka University ,  1-1 Yamadaoka, Suita, Osaka 565-0871 ,  Japan","@type":"Person"},{"name":"Yuasa, Shota","affiliation":"8  Laboratory of Dynamic Cell Biology, Department of Biotechnology ,  Graduate School of Engineering, Osaka University ,  1-1 Yamadaoka, Suita, Osaka 565-0871 ,  Japan","@type":"Person"},{"name":"Matsunaga, Sachihiro","affiliation":"8  Laboratory of Dynamic Cell Biology, Department of Biotechnology ,  Graduate School of Engineering, Osaka University ,  1-1 Yamadaoka, Suita, Osaka 565-0871 ,  Japan","@type":"Person"},{"name":"Fukui, Kiichi","affiliation":"8  Laboratory of Dynamic Cell Biology, Department of Biotechnology ,  Graduate School of Engineering, Osaka University ,  1-1 Yamadaoka, Suita, Osaka 565-0871 ,  Japan","@type":"Person"}],"description":"Abstract.  The whole genome of Jatropha curcas was sequenced, using a combination of the conventional Sanger method and new-generation multiplex sequencing meth","pageStart":"65","pageEnd":"76","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/1/1/m_cover.gif?Expires=2147483647&Signature=ClGcwi7mnwSCPup8xdZdf5ipPpPJFPkMnb8H8urPYhEwpzwvlLkxmUmUtTsd0O4KGqDmjee3iAP~cMOV4NNqFj~GhFPiWnYh~s6OnHI~uE0SigPMO8DkCoxQVVyGjpmFT3VyUCd1Rnhz0tzOY7qjLvTA13Zb9KBld2OsYWDgF6DdaTNIs1umhGa0yRi4S7UVqdSOPa2DP4QXoLG6JKhXqOt4B5uTWFLVPp9R~IrmcfN-N7uO1eXmxMYjpGkprLlRRW5~1~apyMzzJDfdqIyDGFVHMcKv4iliXAPqZ1VMUaMvUFuiZyal2HpblFNoF8oq40FBm2x~DVSMvIEBHFMHhQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Sequence Analysis of the Genome of an Oil-Bearing Tree,  Jatropha curcas  L.","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/1/1/m_cover.gif?Expires=2147483647&Signature=ClGcwi7mnwSCPup8xdZdf5ipPpPJFPkMnb8H8urPYhEwpzwvlLkxmUmUtTsd0O4KGqDmjee3iAP~cMOV4NNqFj~GhFPiWnYh~s6OnHI~uE0SigPMO8DkCoxQVVyGjpmFT3VyUCd1Rnhz0tzOY7qjLvTA13Zb9KBld2OsYWDgF6DdaTNIs1umhGa0yRi4S7UVqdSOPa2DP4QXoLG6JKhXqOt4B5uTWFLVPp9R~IrmcfN-N7uO1eXmxMYjpGkprLlRRW5~1~apyMzzJDfdqIyDGFVHMcKv4iliXAPqZ1VMUaMvUFuiZyal2HpblFNoF8oq40FBm2x~DVSMvIEBHFMHhQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Sequence Analysis of the Genome of an Oil-Bearing Tree,  Jatropha curcas  L." />
    <meta property="og:description" content="Abstract.  The whole genome of Jatropha curcas was sequenced, using a combination of the conventional Sanger method and new-generation multiplex sequencing meth" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/18/1/65/377199" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2010-12-13" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Sato, Shusei" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Hirakawa, Hideki" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Isobe, Sachiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Fukai, Eigo" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Watanabe, Akiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kato, Midori" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kawashima, Kumiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Minami, Chiharu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Muraki, Akiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Nakazaki, Naomi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Takahashi, Chika" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Nakayama, Shinobu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kishida, Yoshie" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kohara, Mitsuyo" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Yamada, Manabu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Tsuruoka, Hisano" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Sasamoto, Shigemi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Tabata, Satoshi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Aizu, Tomoyuki" /><meta name="citation_author_institution" content="Center for Genetic Resource Information, National Institute of Genetics, Yata 1111, Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author" content="Toyoda, Atsushi" /><meta name="citation_author_institution" content="Center for Genetic Resource Information, National Institute of Genetics, Yata 1111, Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author" content="Shin-i, Tadasu" /><meta name="citation_author_institution" content="Center for Genetic Resource Information, National Institute of Genetics, Yata 1111, Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author" content="Minakuchi, Yohei" /><meta name="citation_author_institution" content="Center for Genetic Resource Information, National Institute of Genetics, Yata 1111, Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author" content="Kohara, Yuji" /><meta name="citation_author_institution" content="Center for Genetic Resource Information, National Institute of Genetics, Yata 1111, Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author" content="Fujiyama, Asao" /><meta name="citation_author_institution" content="Center for Genetic Resource Information, National Institute of Genetics, Yata 1111, Mishima, Shizuoka 411-8540, Japan" /><meta name="citation_author_institution" content="Informatics Research Division, National Institute of Informatics, 2-1-2 Hitotsubashi, Chiyoda-ku, Tokyo 101-8430, Japan" /><meta name="citation_author" content="Tsuchimoto, Suguru" /><meta name="citation_author_institution" content="Institute of Molecular and Cellular Biosciences, The University of Tokyo, 1-1-1, Yayoi, Bunkyo-ku, Tokyo 113-0032, Japan" /><meta name="citation_author" content="Kajiyama, Shin'ichiro" /><meta name="citation_author_institution" content="Graduate School of Biology Oriented Science and Technology, Kinki University, 930 Nishi-Mitani, Kinokawa, Wakayama 649-6493, Japan" /><meta name="citation_author" content="Makigano, Eri" /><meta name="citation_author_institution" content="Graduate School of Human Development and Environment, Kobe University, Tsurukabuto 3-11, Nada, Kobe 657-8501, Japan" /><meta name="citation_author" content="Ohmido, Nobuko" /><meta name="citation_author_institution" content="Graduate School of Human Development and Environment, Kobe University, Tsurukabuto 3-11, Nada, Kobe 657-8501, Japan" /><meta name="citation_author" content="Shibagaki, Nakako" /><meta name="citation_author_institution" content="Bioengineering for the Interest of Environmental Sustainability (Sumitomo Electric Industries), Graduate School of Engineering, Osaka University, 1-1 Yamadaoka, Suita, Osaka 565-0871, Japan" /><meta name="citation_author" content="Cartagena, Joyce A." /><meta name="citation_author_institution" content="Bioengineering for the Interest of Environmental Sustainability (Sumitomo Electric Industries), Graduate School of Engineering, Osaka University, 1-1 Yamadaoka, Suita, Osaka 565-0871, Japan" /><meta name="citation_author" content="Wada, Naoki" /><meta name="citation_author_institution" content="Bioengineering for the Interest of Environmental Sustainability (Sumitomo Electric Industries), Graduate School of Engineering, Osaka University, 1-1 Yamadaoka, Suita, Osaka 565-0871, Japan" /><meta name="citation_author" content="Kohinata, Tsutomu" /><meta name="citation_author_institution" content="Laboratory of Dynamic Cell Biology, Department of Biotechnology, Graduate School of Engineering, Osaka University, 1-1 Yamadaoka, Suita, Osaka 565-0871, Japan" /><meta name="citation_author" content="Atefeh, Alipour" /><meta name="citation_author_institution" content="Laboratory of Dynamic Cell Biology, Department of Biotechnology, Graduate School of Engineering, Osaka University, 1-1 Yamadaoka, Suita, Osaka 565-0871, Japan" /><meta name="citation_author" content="Yuasa, Shota" /><meta name="citation_author_institution" content="Laboratory of Dynamic Cell Biology, Department of Biotechnology, Graduate School of Engineering, Osaka University, 1-1 Yamadaoka, Suita, Osaka 565-0871, Japan" /><meta name="citation_author" content="Matsunaga, Sachihiro" /><meta name="citation_author_institution" content="Laboratory of Dynamic Cell Biology, Department of Biotechnology, Graduate School of Engineering, Osaka University, 1-1 Yamadaoka, Suita, Osaka 565-0871, Japan" /><meta name="citation_author" content="Fukui, Kiichi" /><meta name="citation_author_institution" content="Laboratory of Dynamic Cell Biology, Department of Biotechnology, Graduate School of Engineering, Osaka University, 1-1 Yamadaoka, Suita, Osaka 565-0871, Japan" /><meta name="citation_title" content="Sequence Analysis of the Genome of an Oil-Bearing Tree, Jatropha curcas L." /><meta name="citation_firstpage" content="65" /><meta name="citation_lastpage" content="76" /><meta name="citation_doi" content="10.1093/dnares/dsq030" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="18" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2011/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=A review of J. curcas: an oil plant of unfulfilled promise; Biomass Bioenergy;  citation_year=2000;  citation_volume=19;  citation_pages=1-15; " /><meta name="citation_reference" content="Wouter H.M. Wouter M.J.A. Bart M. Use of inadequate data and methodological errors lead to a dramatic overestimation of the water footprint of Jatropha curcas 2009 Nature Precedings hdl:10101/npre.2009.3410.1 http://precedings.nature.com/documents/3410/version/1 " /><meta name="citation_reference" content="citation_title=Biofuel: the little shrub that could—maybe; Nature;  citation_year=2007;  citation_volume=449;  citation_pages=652-655; " /><meta name="citation_reference" content="citation_title=Green fuels for jets; Sci. Am.;  citation_year=2009;  citation_volume=19;  citation_pages=68-69; " /><meta name="citation_reference" content="citation_title=Genome size, base composition and karyotype of Jatropha curcas L., an important biofuel plant; Plant Sci.;  citation_year=2008;  citation_volume=174;  citation_pages=613-617; " /><meta name="citation_reference" content="citation_title=Transcriptome analysis of the oil-rich seed of the bioenergy crop Jatropha curcas L.; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=462" /><meta name="citation_reference" content="citation_title=Genome structure of the legume, Lotus japonicus; DNA Res.;  citation_year=2008;  citation_volume=15;  citation_pages=227-239; " /><meta name="citation_reference" content="citation_title=Figaro: a novel statistical method for vector sequence removal; Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=462-467; " /><meta name="citation_reference" content="citation_title=Application of a superword array in genome assembly; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=201-205; " /><meta name="citation_reference" content="citation_title=Pyrobayes: an improved base caller for SNP discovery in pyrosequences; Nat. Methods;  citation_year=2008;  citation_volume=5;  citation_pages=179-181; " /><meta name="citation_reference" content="citation_title=Systematic artifacts in metagenomes from complex microbial communities; ISME J.;  citation_year=2009;  citation_volume=3;  citation_pages=1314-1317; " /><meta name="citation_reference" content="citation_title=Genome sequence assembly using trace signals and additional sequence information; citation_text=99; Computer Science and Biology: Proceedings of the German Conference on Bioinformatics (GCB);  citation_year=1999;  citation_pages=45-56; " /><meta name="citation_reference" content="citation_title=The mitochondrial genome of Arabidopsis thaliana contains 57 genes in 366,924 nucleotides; Nat. Genet.;  citation_year=1997;  citation_volume=15;  citation_pages=57-61; " /><meta name="citation_reference" content="citation_title=Greedy algorithm for aligning DNA sequences; J. Comput. Biol.;  citation_year=2000;  citation_volume=7;  citation_pages=203-214; " /><meta name="citation_reference" content="citation_title=Velvet: algorithms for de novo short read assembly using de Bruijn graphs; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=821-829; " /><meta name="citation_reference" content="citation_title=BLAT—the BLAST-like alignment tool; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=656-664; " /><meta name="citation_reference" content="citation_title=GeneMark.hmm: new solutions for gene finding; Nucleic Acids Res.;  citation_year=1998;  citation_volume=26;  citation_pages=1107-1115; " /><meta name="citation_reference" content="citation_title=Prediction of complete gene structures in human genomic DNA; J. Mol. Biol.;  citation_year=1997;  citation_volume=268;  citation_pages=78-94; " /><meta name="citation_reference" content="citation_title=Splice site prediction in Arabidopsis thaliana pre-mRNA by combining local and global sequence information; Nucleic Acids Res.;  citation_year=1996;  citation_volume=24;  citation_pages=3439-3452; " /><meta name="citation_reference" content="citation_title=Prediction of locally optimal splice sites in plant pre-mRNA with applications to gene identification in Arabidopsis thaliana genomic DNA; Nucleic Acids Res.;  citation_year=1998;  citation_volume=26;  citation_pages=4748-4757; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-3402; " /><meta name="citation_reference" content="citation_title=Methods for comparing a DNA sequence with a protein sequence; Comput. Appl. Biosci.;  citation_year=1996;  citation_volume=12;  citation_pages=497-506; " /><meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium; Nat. Genet.;  citation_year=2000;  citation_volume=25;  citation_pages=25-29; " /><meta name="citation_reference" content="citation_title=InterPro: the integrative protein signature database; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=D211-215; " /><meta name="citation_reference" content="citation_title=KEGG: Kyoto encyclopedia of genes and genomes; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=29-34; " /><meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice; Nucleic Acids Res.;  citation_year=1994;  citation_volume=122;  citation_pages=4673-4680; " /><meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees; Mol. Biol. Evol.;  citation_year=1987;  citation_volume=4;  citation_pages=406-425; " /><meta name="citation_reference" content="citation_title=MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0; Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=1596-1599; " /><meta name="citation_reference" content="citation_title=Computational and experimental analysis of microsatellites in rice (Oryza sativa L.): frequency, length variation, transposon associations, and genetic marker potential; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=1441-1452; " /><meta name="citation_reference" content="citation_title=Primer3 on the WWW for general users and for biologist programmers; Methods Mol. Biol.;  citation_year=2000;  citation_volume=132;  citation_pages=365-386; " /><meta name="citation_reference" content="citation_title=Automated de novo identification of repeat sequence families in sequenced genomes; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=1269-1276; " /><meta name="citation_reference" content="citation_title=The ASRG database: identification and survey of Arabidopsis thaliana genes involved in pre-mRNA splicing; Genome Biol.;  citation_year=2004;  citation_volume=5;  citation_pages=R102" /><meta name="citation_reference" content="citation_title=The SWISS-PROT protein sequence data bank and its new supplement TREMBL; Nucleic Acids Res.;  citation_year=1996;  citation_volume=24;  citation_pages=21-25; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of the oilseed species Ricinus communis; Nat. Biotechnol.;  citation_year=2010;  citation_volume=28;  citation_pages=951-956; " /><meta name="citation_reference" content="citation_title=AmiGO: online access to ontology and annotation data; Bioinformatics;  citation_year=2009;  citation_volume=15;  citation_pages=288-289; " /><meta name="citation_reference" content="citation_title=Characterization of a new stearoyl-acyl carrier protein desaturase gene from Jatropha curcas; Biotechnol. Lett.;  citation_year=2006;  citation_volume=28;  citation_pages=657-662; " /><meta name="citation_reference" content="citation_title=Rapid analysis of Jatropha curcas gene functions by virus-induced gene silencing; Plant Biotechnol. J.;  citation_year=2009;  citation_volume=7;  citation_pages=964-976; " /><meta name="citation_reference" content="citation_title=Cytosolic triacylglycerol biosynthetic pathway in oilseeds. Molecular cloning and expression of peanut cytosolic diacylglycerol acyltransferase; Plant Physiol.;  citation_year=2006;  citation_volume=141;  citation_pages=1533-1543; " /><meta name="citation_reference" content="citation_title=Novel 12-deoxy-16-hydroxy phorbol diesters isolated from the seed oil of J. curcas; J. Nat. Prod.;  citation_year=2002;  citation_volume=65;  citation_pages=1434-1440; " /><meta name="citation_reference" content="citation_title=Molecular cloning and functional analysis of the gene encoding geranylgeranyl diphosphate synthase from J. curcas; Afr. J. Biotechnol.;  citation_year=2010;  citation_volume=9;  citation_pages=3342-3351; " /><meta name="citation_reference" content="citation_title=Genomic analysis of the terpenoid synthase (AtTPS) gene family of Arabidopsis thaliana; Mol. Genet. Genomics;  citation_year=2002;  citation_volume=267;  citation_pages=730-745; " /><meta name="citation_reference" content="citation_title=Studies on the proteins from the seeds of Croton tiglium and of Jatropha curcas. Toxic properties and inhibition of protein synthesis in vitro; Biochem. J.;  citation_year=1976;  citation_volume=156;  citation_pages=1-6; " /><meta name="citation_reference" content="citation_title=Expression, purification and anti-tumor activity of curcin; Acta Biochim. Biophys. Sin.;  citation_year=2006;  citation_volume=38;  citation_pages=663-668; " /><meta name="citation_reference" content="citation_title=Antitumor effects of curcin from seeds of Jatropha curcas; Acta Pharmacol. Sin.;  citation_year=2003;  citation_volume=24;  citation_pages=241-246; " /><meta name="citation_reference" content="citation_title=Stress-induced curcin-L promoter in leaves of Jatropha curcas L. and characterization in transgenic tobacco; Planta;  citation_year=2009;  citation_volume=230;  citation_pages=387-395; " /><meta name="citation_reference" content="citation_title=Expression of a ribosome inactivating protein (curcin 2) in Jatropha curcas is induced by stress; J. Biosci.;  citation_year=2005;  citation_volume=30;  citation_pages=351-357; " /><meta name="citation_reference" content="citation_title=A ribosome-inactivating protein (curcin 2) induced from Jatropha curcas can reduce viral and fungal infection in transgenic tobacco; Plant Growth Regul.;  citation_year=2008;  citation_volume=54;  citation_pages=115-123; " /><meta name="citation_reference" content="citation_title=NB-LRR proteins: pairs, pieces, perception, partners, and pathways; Curr. Opin. Plant Biol.;  citation_year=2010;  citation_volume=13;  citation_pages=472-477; " /><meta name="citation_reference" content="citation_title=An ancestral MADS-box duplication occurred before the divergence of plants and animals; Proc. Natl Acad. Sci. USA;  citation_year=2000;  citation_volume=97;  citation_pages=5328-5333; " /><meta name="citation_reference" content="citation_title=Evolutionary complexity of MADS complexes; Curr. Opin. Plant Biol.;  citation_year=2007;  citation_volume=10;  citation_pages=32-38; " /><meta name="citation_reference" content="citation_title=Role of SVP in the control of flowering time by ambient temperature in Arabidopsis; Genes Dev.;  citation_year=2007;  citation_volume=21;  citation_pages=397-402; " /><meta name="citation_reference" content="citation_title=Molecular cloning of SVP: a negative regulator of the floral transition in Arabidopsis; Plant J.;  citation_year=2000;  citation_volume=21;  citation_pages=351-360; " /><meta name="citation_reference" content="citation_title=Rice SVP-group MADS-box proteins, OsMADS22 and OsMADS55, are negative regulators of brassinosteroid responses; Plant J.;  citation_year=2008;  citation_volume=54;  citation_pages=93-105; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of the MADS-box gene family in Populus trichocarpa; Gene;  citation_year=2006;  citation_volume=378;  citation_pages=84-94; " /><meta name="citation_reference" content="citation_title=Morphology and infrageneric relationships of the genus J. curcas; citation_publisher=University of California Press, Berkeley, CA, USA, ;  citation_year=1992; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/18/1/65/1072092/dsq030.pdf" /><meta name="description" content="Abstract.  The whole genome of Jatropha curcas was sequenced, using a combination of the conventional Sanger method and new-generation multiplex sequencing meth" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/18/1/65/377199" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/18/1/65/377199" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/18/1/65/377199");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsq030';
    </script>
<title>
	Sequence Analysis of the Genome of an Oil-Bearing Tree, Jatropha curcas L. | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./377199" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Ba6qATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2F0bywgU2h1c2VpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhpcmFrYXdhLCBIaWRla2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSXNvYmUsIFNhY2hpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRnVrYWksIEVpZ28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV2F0YW5hYmUsIEFraWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkthdG8sIE1pZG9yaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYXdhc2hpbWEsIEt1bWlrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNaW5hbWksIENoaWhhcnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTXVyYWtpLCBBa2lrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWthemFraSwgTmFvbWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFrYWhhc2hpLCBDaGlrYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWtheWFtYSwgU2hpbm9idSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLaXNoaWRhLCBZb3NoaWUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS29oYXJhLCBNaXRzdXlvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllhbWFkYSwgTWFuYWJ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRzdXJ1b2thLCBIaXNhbm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2FzYW1vdG8sIFNoaWdlbWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFiYXRhLCBTYXRvc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFpenUsIFRvbW95dWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudGVyIGZvciBHZW5ldGljIFJlc291cmNlIEluZm9ybWF0aW9uLCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgR2VuZXRpY3MsIFlhdGEgMTExMSwgTWlzaGltYSwgU2hpenVva2EgNDExLTg1NDAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVG95b2RhLCBBdHN1c2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudGVyIGZvciBHZW5ldGljIFJlc291cmNlIEluZm9ybWF0aW9uLCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgR2VuZXRpY3MsIFlhdGEgMTExMSwgTWlzaGltYSwgU2hpenVva2EgNDExLTg1NDAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpbi1pLCBUYWRhc3UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50ZXIgZm9yIEdlbmV0aWMgUmVzb3VyY2UgSW5mb3JtYXRpb24sIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBHZW5ldGljcywgWWF0YSAxMTExLCBNaXNoaW1hLCBTaGl6dW9rYSA0MTEtODU0MCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNaW5ha3VjaGksIFlvaGVpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudGVyIGZvciBHZW5ldGljIFJlc291cmNlIEluZm9ybWF0aW9uLCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgR2VuZXRpY3MsIFlhdGEgMTExMSwgTWlzaGltYSwgU2hpenVva2EgNDExLTg1NDAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS29oYXJhLCBZdWppIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudGVyIGZvciBHZW5ldGljIFJlc291cmNlIEluZm9ybWF0aW9uLCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgR2VuZXRpY3MsIFlhdGEgMTExMSwgTWlzaGltYSwgU2hpenVva2EgNDExLTg1NDAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRnVqaXlhbWEsIEFzYW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50ZXIgZm9yIEdlbmV0aWMgUmVzb3VyY2UgSW5mb3JtYXRpb24sIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBHZW5ldGljcywgWWF0YSAxMTExLCBNaXNoaW1hLCBTaGl6dW9rYSA0MTEtODU0MCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbmZvcm1hdGljcyBSZXNlYXJjaCBEaXZpc2lvbiwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEluZm9ybWF0aWNzLCAyLTEtMiBIaXRvdHN1YmFzaGksIENoaXlvZGEta3UsIFRva3lvIDEwMS04NDMwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRzdWNoaW1vdG8sIFN1Z3VydSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBNb2xlY3VsYXIgYW5kIENlbGx1bGFyIEJpb3NjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgMS0xLTEsIFlheW9pLCBCdW5reW8ta3UsIFRva3lvIDExMy0wMDMyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikthaml5YW1hLCBTaGluJ2ljaGlybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBCaW9sb2d5IE9yaWVudGVkIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIEtpbmtpIFVuaXZlcnNpdHksIDkzMCBOaXNoaS1NaXRhbmksIEtpbm9rYXdhLCBXYWtheWFtYSA2NDktNjQ5MywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYWtpZ2FubywgRXJpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEh1bWFuIERldmVsb3BtZW50IGFuZCBFbnZpcm9ubWVudCwgS29iZSBVbml2ZXJzaXR5LCBUc3VydWthYnV0byAzLTExLCBOYWRhLCBLb2JlIDY1Ny04NTAxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9obWlkbywgTm9idWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEh1bWFuIERldmVsb3BtZW50IGFuZCBFbnZpcm9ubWVudCwgS29iZSBVbml2ZXJzaXR5LCBUc3VydWthYnV0byAzLTExLCBOYWRhLCBLb2JlIDY1Ny04NTAxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNoaWJhZ2FraSwgTmFrYWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvZW5naW5lZXJpbmcgZm9yIHRoZSBJbnRlcmVzdCBvZiBFbnZpcm9ubWVudGFsIFN1c3RhaW5hYmlsaXR5IChTdW1pdG9tbyBFbGVjdHJpYyBJbmR1c3RyaWVzKSwgR3JhZHVhdGUgU2Nob29sIG9mIEVuZ2luZWVyaW5nLCBPc2FrYSBVbml2ZXJzaXR5LCAxLTEgWWFtYWRhb2thLCBTdWl0YSwgT3Nha2EgNTY1LTA4NzEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2FydGFnZW5hLCBKb3ljZSBBLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb2VuZ2luZWVyaW5nIGZvciB0aGUgSW50ZXJlc3Qgb2YgRW52aXJvbm1lbnRhbCBTdXN0YWluYWJpbGl0eSAoU3VtaXRvbW8gRWxlY3RyaWMgSW5kdXN0cmllcyksIEdyYWR1YXRlIFNjaG9vbCBvZiBFbmdpbmVlcmluZywgT3Nha2EgVW5pdmVyc2l0eSwgMS0xIFlhbWFkYW9rYSwgU3VpdGEsIE9zYWthIDU2NS0wODcxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IldhZGEsIE5hb2tpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvZW5naW5lZXJpbmcgZm9yIHRoZSBJbnRlcmVzdCBvZiBFbnZpcm9ubWVudGFsIFN1c3RhaW5hYmlsaXR5IChTdW1pdG9tbyBFbGVjdHJpYyBJbmR1c3RyaWVzKSwgR3JhZHVhdGUgU2Nob29sIG9mIEVuZ2luZWVyaW5nLCBPc2FrYSBVbml2ZXJzaXR5LCAxLTEgWWFtYWRhb2thLCBTdWl0YSwgT3Nha2EgNTY1LTA4NzEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS29oaW5hdGEsIFRzdXRvbXUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIER5bmFtaWMgQ2VsbCBCaW9sb2d5LCBEZXBhcnRtZW50IG9mIEJpb3RlY2hub2xvZ3ksIEdyYWR1YXRlIFNjaG9vbCBvZiBFbmdpbmVlcmluZywgT3Nha2EgVW5pdmVyc2l0eSwgMS0xIFlhbWFkYW9rYSwgU3VpdGEsIE9zYWthIDU2NS0wODcxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkF0ZWZlaCwgQWxpcG91ciIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgRHluYW1pYyBDZWxsIEJpb2xvZ3ksIERlcGFydG1lbnQgb2YgQmlvdGVjaG5vbG9neSwgR3JhZHVhdGUgU2Nob29sIG9mIEVuZ2luZWVyaW5nLCBPc2FrYSBVbml2ZXJzaXR5LCAxLTEgWWFtYWRhb2thLCBTdWl0YSwgT3Nha2EgNTY1LTA4NzEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWXVhc2EsIFNob3RhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTGFib3JhdG9yeSBvZiBEeW5hbWljIENlbGwgQmlvbG9neSwgRGVwYXJ0bWVudCBvZiBCaW90ZWNobm9sb2d5LCBHcmFkdWF0ZSBTY2hvb2wgb2YgRW5naW5lZXJpbmcsIE9zYWthIFVuaXZlcnNpdHksIDEtMSBZYW1hZGFva2EsIFN1aXRhLCBPc2FrYSA1NjUtMDg3MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYXRzdW5hZ2EsIFNhY2hpaGlybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgRHluYW1pYyBDZWxsIEJpb2xvZ3ksIERlcGFydG1lbnQgb2YgQmlvdGVjaG5vbG9neSwgR3JhZHVhdGUgU2Nob29sIG9mIEVuZ2luZWVyaW5nLCBPc2FrYSBVbml2ZXJzaXR5LCAxLTEgWWFtYWRhb2thLCBTdWl0YSwgT3Nha2EgNTY1LTA4NzEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRnVrdWksIEtpaWNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgRHluYW1pYyBDZWxsIEJpb2xvZ3ksIERlcGFydG1lbnQgb2YgQmlvdGVjaG5vbG9neSwgR3JhZHVhdGUgU2Nob29sIG9mIEVuZ2luZWVyaW5nLCBPc2FrYSBVbml2ZXJzaXR5LCAxLTEgWWFtYWRhb2thLCBTdWl0YSwgT3Nha2EgNTY1LTA4NzEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJTZXF1ZW5jZSBBbmFseXNpcyBvZiB0aGUgR2Vub21lIG9mIGFuIE9pbC1CZWFyaW5nIFRyZWUsIEphdHJvcGhhIGN1cmNhcyBMLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjY1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI3NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcTAzMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDExLzAyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSByZXZpZXcgb2YgSi4gY3VyY2FzOiBhbiBvaWwgcGxhbnQgb2YgdW5mdWxmaWxsZWQgcHJvbWlzZTsgQmlvbWFzcyBCaW9lbmVyZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xLTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJXb3V0ZXIgSC5NLiBXb3V0ZXIgTS5KLkEuIEJhcnQgTS4gVXNlIG9mIGluYWRlcXVhdGUgZGF0YSBhbmQgbWV0aG9kb2xvZ2ljYWwgZXJyb3JzIGxlYWQgdG8gYSBkcmFtYXRpYyBvdmVyZXN0aW1hdGlvbiBvZiB0aGUgd2F0ZXIgZm9vdHByaW50IG9mIEphdHJvcGhhIGN1cmNhcyAyMDA5IE5hdHVyZSBQcmVjZWRpbmdzIGhkbDoxMDEwMS9ucHJlLjIwMDkuMzQxMC4xIGh0dHA6Ly9wcmVjZWRpbmdzLm5hdHVyZS5jb20vZG9jdW1lbnRzLzM0MTAvdmVyc2lvbi8xICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpb2Z1ZWw6IHRoZSBsaXR0bGUgc2hydWIgdGhhdCBjb3VsZOKAlG1heWJlOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDQ5OyAgY2l0YXRpb25fcGFnZXM9NjUyLTY1NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R3JlZW4gZnVlbHMgZm9yIGpldHM7IFNjaS4gQW0uOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9NjgtNjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzaXplLCBiYXNlIGNvbXBvc2l0aW9uIGFuZCBrYXJ5b3R5cGUgb2YgSmF0cm9waGEgY3VyY2FzIEwuLCBhbiBpbXBvcnRhbnQgYmlvZnVlbCBwbGFudDsgUGxhbnQgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNzQ7ICBjaXRhdGlvbl9wYWdlcz02MTMtNjE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0b21lIGFuYWx5c2lzIG9mIHRoZSBvaWwtcmljaCBzZWVkIG9mIHRoZSBiaW9lbmVyZ3kgY3JvcCBKYXRyb3BoYSBjdXJjYXMgTC47IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTQ2MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzdHJ1Y3R1cmUgb2YgdGhlIGxlZ3VtZSwgTG90dXMgamFwb25pY3VzOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTIyNy0yMzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZpZ2FybzogYSBub3ZlbCBzdGF0aXN0aWNhbCBtZXRob2QgZm9yIHZlY3RvciBzZXF1ZW5jZSByZW1vdmFsOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTQ2Mi00Njc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFwcGxpY2F0aW9uIG9mIGEgc3VwZXJ3b3JkIGFycmF5IGluIGdlbm9tZSBhc3NlbWJseTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9MjAxLTIwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHlyb2JheWVzOiBhbiBpbXByb3ZlZCBiYXNlIGNhbGxlciBmb3IgU05QIGRpc2NvdmVyeSBpbiBweXJvc2VxdWVuY2VzOyBOYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTE3OS0xODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5c3RlbWF0aWMgYXJ0aWZhY3RzIGluIG1ldGFnZW5vbWVzIGZyb20gY29tcGxleCBtaWNyb2JpYWwgY29tbXVuaXRpZXM7IElTTUUgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTEzMTQtMTMxNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNlIGFzc2VtYmx5IHVzaW5nIHRyYWNlIHNpZ25hbHMgYW5kIGFkZGl0aW9uYWwgc2VxdWVuY2UgaW5mb3JtYXRpb247IGNpdGF0aW9uX3RleHQ9OTk7IENvbXB1dGVyIFNjaWVuY2UgYW5kIEJpb2xvZ3k6IFByb2NlZWRpbmdzIG9mIHRoZSBHZXJtYW4gQ29uZmVyZW5jZSBvbiBCaW9pbmZvcm1hdGljcyAoR0NCKTsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3BhZ2VzPTQ1LTU2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbWl0b2Nob25kcmlhbCBnZW5vbWUgb2YgQXJhYmlkb3BzaXMgdGhhbGlhbmEgY29udGFpbnMgNTcgZ2VuZXMgaW4gMzY2LDkyNCBudWNsZW90aWRlczsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz01Ny02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R3JlZWR5IGFsZ29yaXRobSBmb3IgYWxpZ25pbmcgRE5BIHNlcXVlbmNlczsgSi4gQ29tcHV0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MjAzLTIxNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmVsdmV0OiBhbGdvcml0aG1zIGZvciBkZSBub3ZvIHNob3J0IHJlYWQgYXNzZW1ibHkgdXNpbmcgZGUgQnJ1aWpuIGdyYXBoczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz04MjEtODI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CTEFU4oCUdGhlIEJMQVNULWxpa2UgYWxpZ25tZW50IHRvb2w7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NjU2LTY2NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZU1hcmsuaG1tOiBuZXcgc29sdXRpb25zIGZvciBnZW5lIGZpbmRpbmc7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTExMDctMTExNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlZGljdGlvbiBvZiBjb21wbGV0ZSBnZW5lIHN0cnVjdHVyZXMgaW4gaHVtYW4gZ2Vub21pYyBETkE7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjY4OyAgY2l0YXRpb25fcGFnZXM9NzgtOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwbGljZSBzaXRlIHByZWRpY3Rpb24gaW4gQXJhYmlkb3BzaXMgdGhhbGlhbmEgcHJlLW1STkEgYnkgY29tYmluaW5nIGxvY2FsIGFuZCBnbG9iYWwgc2VxdWVuY2UgaW5mb3JtYXRpb247IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTM0MzktMzQ1MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlZGljdGlvbiBvZiBsb2NhbGx5IG9wdGltYWwgc3BsaWNlIHNpdGVzIGluIHBsYW50IHByZS1tUk5BIHdpdGggYXBwbGljYXRpb25zIHRvIGdlbmUgaWRlbnRpZmljYXRpb24gaW4gQXJhYmlkb3BzaXMgdGhhbGlhbmEgZ2Vub21pYyBETkE7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTQ3NDgtNDc1NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2FwcGVkIEJMQVNUIGFuZCBQU0ktQkxBU1Q6IGEgbmV3IGdlbmVyYXRpb24gb2YgcHJvdGVpbiBkYXRhYmFzZSBzZWFyY2ggcHJvZ3JhbXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTMzODktMzQwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWV0aG9kcyBmb3IgY29tcGFyaW5nIGEgRE5BIHNlcXVlbmNlIHdpdGggYSBwcm90ZWluIHNlcXVlbmNlOyBDb21wdXQuIEFwcGwuIEJpb3NjaS47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz00OTctNTA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIG9udG9sb2d5OiB0b29sIGZvciB0aGUgdW5pZmljYXRpb24gb2YgYmlvbG9neS4gVGhlIEdlbmUgT250b2xvZ3kgQ29uc29ydGl1bTsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yNS0yOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZXJQcm86IHRoZSBpbnRlZ3JhdGl2ZSBwcm90ZWluIHNpZ25hdHVyZSBkYXRhYmFzZTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9RDIxMS0yMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUtFR0c6IEt5b3RvIGVuY3ljbG9wZWRpYSBvZiBnZW5lcyBhbmQgZ2Vub21lczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MjktMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNMVVNUQUwgVzogaW1wcm92aW5nIHRoZSBzZW5zaXRpdml0eSBvZiBwcm9ncmVzc2l2ZSBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgdGhyb3VnaCBzZXF1ZW5jZSB3ZWlnaHRpbmcsIHBvc2l0aW9uLXNwZWNpZmljIGdhcCBwZW5hbHRpZXMgYW5kIHdlaWdodCBtYXRyaXggY2hvaWNlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTIyOyAgY2l0YXRpb25fcGFnZXM9NDY3My00NjgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbmVpZ2hib3Itam9pbmluZyBtZXRob2Q6IGEgbmV3IG1ldGhvZCBmb3IgcmVjb25zdHJ1Y3RpbmcgcGh5bG9nZW5ldGljIHRyZWVzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz00MDYtNDI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NRUdBNDogTW9sZWN1bGFyIEV2b2x1dGlvbmFyeSBHZW5ldGljcyBBbmFseXNpcyAoTUVHQSkgc29mdHdhcmUgdmVyc2lvbiA0LjA7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xNTk2LTE1OTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXB1dGF0aW9uYWwgYW5kIGV4cGVyaW1lbnRhbCBhbmFseXNpcyBvZiBtaWNyb3NhdGVsbGl0ZXMgaW4gcmljZSAoT3J5emEgc2F0aXZhIEwuKTogZnJlcXVlbmN5LCBsZW5ndGggdmFyaWF0aW9uLCB0cmFuc3Bvc29uIGFzc29jaWF0aW9ucywgYW5kIGdlbmV0aWMgbWFya2VyIHBvdGVudGlhbDsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xNDQxLTE0NTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByaW1lcjMgb24gdGhlIFdXVyBmb3IgZ2VuZXJhbCB1c2VycyBhbmQgZm9yIGJpb2xvZ2lzdCBwcm9ncmFtbWVyczsgTWV0aG9kcyBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTEzMjsgIGNpdGF0aW9uX3BhZ2VzPTM2NS0zODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUF1dG9tYXRlZCBkZSBub3ZvIGlkZW50aWZpY2F0aW9uIG9mIHJlcGVhdCBzZXF1ZW5jZSBmYW1pbGllcyBpbiBzZXF1ZW5jZWQgZ2Vub21lczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xMjY5LTEyNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBBU1JHIGRhdGFiYXNlOiBpZGVudGlmaWNhdGlvbiBhbmQgc3VydmV5IG9mIEFyYWJpZG9wc2lzIHRoYWxpYW5hIGdlbmVzIGludm9sdmVkIGluIHByZS1tUk5BIHNwbGljaW5nOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPVIxMDIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgU1dJU1MtUFJPVCBwcm90ZWluIHNlcXVlbmNlIGRhdGEgYmFuayBhbmQgaXRzIG5ldyBzdXBwbGVtZW50IFRSRU1CTDsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MjEtMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgb2lsc2VlZCBzcGVjaWVzIFJpY2ludXMgY29tbXVuaXM7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz05NTEtOTU2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbWlHTzogb25saW5lIGFjY2VzcyB0byBvbnRvbG9neSBhbmQgYW5ub3RhdGlvbiBkYXRhOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTI4OC0yODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgYSBuZXcgc3RlYXJveWwtYWN5bCBjYXJyaWVyIHByb3RlaW4gZGVzYXR1cmFzZSBnZW5lIGZyb20gSmF0cm9waGEgY3VyY2FzOyBCaW90ZWNobm9sLiBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTY1Ny02NjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJhcGlkIGFuYWx5c2lzIG9mIEphdHJvcGhhIGN1cmNhcyBnZW5lIGZ1bmN0aW9ucyBieSB2aXJ1cy1pbmR1Y2VkIGdlbmUgc2lsZW5jaW5nOyBQbGFudCBCaW90ZWNobm9sLiBKLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9OTY0LTk3NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3l0b3NvbGljIHRyaWFjeWxnbHljZXJvbCBiaW9zeW50aGV0aWMgcGF0aHdheSBpbiBvaWxzZWVkcy4gTW9sZWN1bGFyIGNsb25pbmcgYW5kIGV4cHJlc3Npb24gb2YgcGVhbnV0IGN5dG9zb2xpYyBkaWFjeWxnbHljZXJvbCBhY3lsdHJhbnNmZXJhc2U7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE0MTsgIGNpdGF0aW9uX3BhZ2VzPTE1MzMtMTU0MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm92ZWwgMTItZGVveHktMTYtaHlkcm94eSBwaG9yYm9sIGRpZXN0ZXJzIGlzb2xhdGVkIGZyb20gdGhlIHNlZWQgb2lsIG9mIEouIGN1cmNhczsgSi4gTmF0LiBQcm9kLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT02NTsgIGNpdGF0aW9uX3BhZ2VzPTE0MzQtMTQ0MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGNsb25pbmcgYW5kIGZ1bmN0aW9uYWwgYW5hbHlzaXMgb2YgdGhlIGdlbmUgZW5jb2RpbmcgZ2VyYW55bGdlcmFueWwgZGlwaG9zcGhhdGUgc3ludGhhc2UgZnJvbSBKLiBjdXJjYXM7IEFmci4gSi4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTMzNDItMzM1MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBhbmFseXNpcyBvZiB0aGUgdGVycGVub2lkIHN5bnRoYXNlIChBdFRQUykgZ2VuZSBmYW1pbHkgb2YgQXJhYmlkb3BzaXMgdGhhbGlhbmE7IE1vbC4gR2VuZXQuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI2NzsgIGNpdGF0aW9uX3BhZ2VzPTczMC03NDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0dWRpZXMgb24gdGhlIHByb3RlaW5zIGZyb20gdGhlIHNlZWRzIG9mIENyb3RvbiB0aWdsaXVtIGFuZCBvZiBKYXRyb3BoYSBjdXJjYXMuIFRveGljIHByb3BlcnRpZXMgYW5kIGluaGliaXRpb24gb2YgcHJvdGVpbiBzeW50aGVzaXMgaW4gdml0cm87IEJpb2NoZW0uIEouOyAgY2l0YXRpb25feWVhcj0xOTc2OyAgY2l0YXRpb25fdm9sdW1lPTE1NjsgIGNpdGF0aW9uX3BhZ2VzPTEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwcmVzc2lvbiwgcHVyaWZpY2F0aW9uIGFuZCBhbnRpLXR1bW9yIGFjdGl2aXR5IG9mIGN1cmNpbjsgQWN0YSBCaW9jaGltLiBCaW9waHlzLiBTaW4uOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9NjYzLTY2ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW50aXR1bW9yIGVmZmVjdHMgb2YgY3VyY2luIGZyb20gc2VlZHMgb2YgSmF0cm9waGEgY3VyY2FzOyBBY3RhIFBoYXJtYWNvbC4gU2luLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTI0MS0yNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cmVzcy1pbmR1Y2VkIGN1cmNpbi1MIHByb21vdGVyIGluIGxlYXZlcyBvZiBKYXRyb3BoYSBjdXJjYXMgTC4gYW5kIGNoYXJhY3Rlcml6YXRpb24gaW4gdHJhbnNnZW5pYyB0b2JhY2NvOyBQbGFudGE7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjMwOyAgY2l0YXRpb25fcGFnZXM9Mzg3LTM5NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwcmVzc2lvbiBvZiBhIHJpYm9zb21lIGluYWN0aXZhdGluZyBwcm90ZWluIChjdXJjaW4gMikgaW4gSmF0cm9waGEgY3VyY2FzIGlzIGluZHVjZWQgYnkgc3RyZXNzOyBKLiBCaW9zY2kuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MzUxLTM1NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSByaWJvc29tZS1pbmFjdGl2YXRpbmcgcHJvdGVpbiAoY3VyY2luIDIpIGluZHVjZWQgZnJvbSBKYXRyb3BoYSBjdXJjYXMgY2FuIHJlZHVjZSB2aXJhbCBhbmQgZnVuZ2FsIGluZmVjdGlvbiBpbiB0cmFuc2dlbmljIHRvYmFjY287IFBsYW50IEdyb3d0aCBSZWd1bC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz0xMTUtMTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OQi1MUlIgcHJvdGVpbnM6IHBhaXJzLCBwaWVjZXMsIHBlcmNlcHRpb24sIHBhcnRuZXJzLCBhbmQgcGF0aHdheXM7IEN1cnIuIE9waW4uIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NDcyLTQ3NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gYW5jZXN0cmFsIE1BRFMtYm94IGR1cGxpY2F0aW9uIG9jY3VycmVkIGJlZm9yZSB0aGUgZGl2ZXJnZW5jZSBvZiBwbGFudHMgYW5kIGFuaW1hbHM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9OTc7ICBjaXRhdGlvbl9wYWdlcz01MzI4LTUzMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbmFyeSBjb21wbGV4aXR5IG9mIE1BRFMgY29tcGxleGVzOyBDdXJyLiBPcGluLiBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTMyLTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Sb2xlIG9mIFNWUCBpbiB0aGUgY29udHJvbCBvZiBmbG93ZXJpbmcgdGltZSBieSBhbWJpZW50IHRlbXBlcmF0dXJlIGluIEFyYWJpZG9wc2lzOyBHZW5lcyBEZXYuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9Mzk3LTQwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGNsb25pbmcgb2YgU1ZQOiBhIG5lZ2F0aXZlIHJlZ3VsYXRvciBvZiB0aGUgZmxvcmFsIHRyYW5zaXRpb24gaW4gQXJhYmlkb3BzaXM7IFBsYW50IEouOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MzUxLTM2MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmljZSBTVlAtZ3JvdXAgTUFEUy1ib3ggcHJvdGVpbnMsIE9zTUFEUzIyIGFuZCBPc01BRFM1NSwgYXJlIG5lZ2F0aXZlIHJlZ3VsYXRvcnMgb2YgYnJhc3Npbm9zdGVyb2lkIHJlc3BvbnNlczsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz05My0xMDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFuYWx5c2lzIG9mIHRoZSBNQURTLWJveCBnZW5lIGZhbWlseSBpbiBQb3B1bHVzIHRyaWNob2NhcnBhOyBHZW5lOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM3ODsgIGNpdGF0aW9uX3BhZ2VzPTg0LTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb3JwaG9sb2d5IGFuZCBpbmZyYWdlbmVyaWMgcmVsYXRpb25zaGlwcyBvZiB0aGUgZ2VudXMgSi4gY3VyY2FzOyBjaXRhdGlvbl9wdWJsaXNoZXI9VW5pdmVyc2l0eSBvZiBDYWxpZm9ybmlhIFByZXNzLCBCZXJrZWxleSwgQ0EsIFVTQSwgOyAgY2l0YXRpb25feWVhcj0xOTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTgvMS82NS8xMDcyMDkyL2RzcTAzMC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVGhlIHdob2xlIGdlbm9tZSBvZiBKYXRyb3BoYSBjdXJjYXMgd2FzIHNlcXVlbmNlZCwgdXNpbmcgYSBjb21iaW5hdGlvbiBvZiB0aGUgY29udmVudGlvbmFsIFNhbmdlciBtZXRob2QgYW5kIG5ldy1nZW5lcmF0aW9uIG11bHRpcGxleCBzZXF1ZW5jaW5nIG1ldGgiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM3NzE5OSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQrNZkTVR5OI4ap+KM/xoCl6z9yyg==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f1%2f65%2f377199" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f1%2f65%2f377199" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=14695" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/18/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/1/1/m_cover.gif?Expires=2147483647&amp;Signature=ClGcwi7mnwSCPup8xdZdf5ipPpPJFPkMnb8H8urPYhEwpzwvlLkxmUmUtTsd0O4KGqDmjee3iAP~cMOV4NNqFj~GhFPiWnYh~s6OnHI~uE0SigPMO8DkCoxQVVyGjpmFT3VyUCd1Rnhz0tzOY7qjLvTA13Zb9KBld2OsYWDgF6DdaTNIs1umhGa0yRi4S7UVqdSOPa2DP4QXoLG6JKhXqOt4B5uTWFLVPp9R~IrmcfN-N7uO1eXmxMYjpGkprLlRRW5~1~apyMzzJDfdqIyDGFVHMcKv4iliXAPqZ1VMUaMvUFuiZyal2HpblFNoF8oq40FBm2x~DVSMvIEBHFMHhQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/18/1">
                        <div class="volume trailing-comma">Volume 18</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2011        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5500855">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5500855">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5500857">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5500857">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5500861">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5500861">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5500889">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5500889">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5500940">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5500940">4. Databases</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5500942">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5500942">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5500944">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5500944">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5500946">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5500946">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5500948">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5500948">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/18/1/53/377009">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Sequence Analysis of the Genome of an Oil-Bearing Tree, <em>Jatropha curcas</em> L.
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shusei Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shusei+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shusei Sato</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Hirakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Hirakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirakawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirakawa H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hideki Hirakawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sachiko Isobe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sachiko+Isobe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Isobe S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Isobe S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Sachiko Isobe</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eigo Fukai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eigo+Fukai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fukai E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fukai E&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Eigo Fukai</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akiko Watanabe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akiko+Watanabe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Watanabe A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Watanabe A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Akiko Watanabe</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Midori Kato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Midori+Kato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kato M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kato M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Midori Kato</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kumiko Kawashima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kumiko+Kawashima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawashima K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kawashima K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kumiko Kawashima</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chiharu Minami</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chiharu+Minami">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Minami C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Minami C&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Chiharu Minami</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akiko Muraki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akiko+Muraki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Muraki A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Muraki A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Akiko Muraki</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Naomi Nakazaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Naomi+Nakazaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakazaki N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakazaki N&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Naomi Nakazaki</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Chika Takahashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Chika+Takahashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takahashi C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takahashi C&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Chika Takahashi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shinobu Nakayama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shinobu+Nakayama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakayama S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakayama S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Shinobu Nakayama</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshie Kishida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshie+Kishida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kishida Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kishida Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yoshie Kishida</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mitsuyo Kohara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mitsuyo+Kohara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kohara M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kohara M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Mitsuyo Kohara</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Manabu Yamada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Manabu+Yamada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamada M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamada M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Manabu Yamada</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hisano Tsuruoka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hisano+Tsuruoka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tsuruoka H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tsuruoka H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Hisano Tsuruoka</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shigemi Sasamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shigemi+Sasamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sasamoto S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sasamoto S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Shigemi Sasamoto</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Tabata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 438-52-3935. Fax. +81 438-52-3934. E-mail: <a href="mailto:tabata@kazusa.or.jp" target="_blank">tabata@kazusa.or.jp</a> (S.T.); Tel. +81 6-6879-7440. Fax. +81 6-6879-7441. E-mail: <a href="mailto:kfukui@bio.eng.osaka-u.ac.jp" target="_blank">kfukui@bio.eng.osaka-u.ac.jp</a> (K.F.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Tabata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tabata S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tabata S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Satoshi Tabata</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tomoyuki Aizu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Center for Genetic Resource Information</div>, <div class="insititution">National Institute of Genetics</div>, <div class="addr-line">Yata 1111, Mishima, Shizuoka 411-8540</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tomoyuki+Aizu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Aizu T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Aizu T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tomoyuki Aizu</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Atsushi Toyoda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Center for Genetic Resource Information</div>, <div class="insititution">National Institute of Genetics</div>, <div class="addr-line">Yata 1111, Mishima, Shizuoka 411-8540</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Atsushi+Toyoda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Toyoda A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Toyoda A&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Atsushi Toyoda</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tadasu Shin-i</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Center for Genetic Resource Information</div>, <div class="insititution">National Institute of Genetics</div>, <div class="addr-line">Yata 1111, Mishima, Shizuoka 411-8540</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tadasu+Shin-i">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shin-i T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shin-i T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tadasu Shin-i</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yohei Minakuchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Center for Genetic Resource Information</div>, <div class="insititution">National Institute of Genetics</div>, <div class="addr-line">Yata 1111, Mishima, Shizuoka 411-8540</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yohei+Minakuchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Minakuchi Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Minakuchi Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yohei Minakuchi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuji Kohara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Center for Genetic Resource Information</div>, <div class="insititution">National Institute of Genetics</div>, <div class="addr-line">Yata 1111, Mishima, Shizuoka 411-8540</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuji+Kohara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kohara Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kohara Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yuji Kohara</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Asao Fujiyama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Center for Genetic Resource Information</div>, <div class="insititution">National Institute of Genetics</div>, <div class="addr-line">Yata 1111, Mishima, Shizuoka 411-8540</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Informatics Research Division</div>, <div class="insititution">National Institute of Informatics</div>, <div class="addr-line">2-1-2 Hitotsubashi, Chiyoda-ku, Tokyo 101-8430</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Asao+Fujiyama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fujiyama A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fujiyama A&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Asao Fujiyama</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Suguru Tsuchimoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Institute of Molecular and Cellular Biosciences</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">1-1-1, Yayoi, Bunkyo-ku, Tokyo 113-0032</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Suguru+Tsuchimoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tsuchimoto S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tsuchimoto S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Suguru Tsuchimoto</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shin&#39;ichiro Kajiyama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Graduate School of Biology Oriented Science and Technology</div>, <div class="insititution">Kinki University</div>, <div class="addr-line">930 Nishi-Mitani, Kinokawa, Wakayama 649-6493</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shin%27ichiro+Kajiyama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kajiyama S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kajiyama S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Shin'ichiro Kajiyama</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eri Makigano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="addr-line">Graduate School of Human Development and Environment</div>, <div class="insititution">Kobe University</div>, <div class="addr-line">Tsurukabuto 3-11, Nada, Kobe 657-8501</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eri+Makigano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Makigano E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Makigano E&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Eri Makigano</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nobuko Ohmido</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="addr-line">Graduate School of Human Development and Environment</div>, <div class="insititution">Kobe University</div>, <div class="addr-line">Tsurukabuto 3-11, Nada, Kobe 657-8501</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nobuko+Ohmido">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohmido N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohmido N&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Nobuko Ohmido</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nakako Shibagaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="addr-line">Bioengineering for the Interest of Environmental Sustainability (Sumitomo Electric Industries)</div>, <div class="insititution">Graduate School of Engineering, Osaka University</div>, <div class="addr-line">1-1 Yamadaoka, Suita, Osaka 565-0871</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nakako+Shibagaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shibagaki N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shibagaki N&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Nakako Shibagaki</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Joyce A. Cartagena</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="addr-line">Bioengineering for the Interest of Environmental Sustainability (Sumitomo Electric Industries)</div>, <div class="insititution">Graduate School of Engineering, Osaka University</div>, <div class="addr-line">1-1 Yamadaoka, Suita, Osaka 565-0871</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Joyce+A.+Cartagena">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cartagena J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cartagena J&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Joyce A. Cartagena</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Naoki Wada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="addr-line">Bioengineering for the Interest of Environmental Sustainability (Sumitomo Electric Industries)</div>, <div class="insititution">Graduate School of Engineering, Osaka University</div>, <div class="addr-line">1-1 Yamadaoka, Suita, Osaka 565-0871</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Naoki+Wada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wada N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wada N&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Naoki Wada</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tsutomu Kohinata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">8</div><div class="addr-line">Laboratory of Dynamic Cell Biology, Department of Biotechnology</div>, <div class="insititution">Graduate School of Engineering, Osaka University</div>, <div class="addr-line">1-1 Yamadaoka, Suita, Osaka 565-0871</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tsutomu+Kohinata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kohinata T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kohinata T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tsutomu Kohinata</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Alipour Atefeh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">8</div><div class="addr-line">Laboratory of Dynamic Cell Biology, Department of Biotechnology</div>, <div class="insititution">Graduate School of Engineering, Osaka University</div>, <div class="addr-line">1-1 Yamadaoka, Suita, Osaka 565-0871</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Alipour+Atefeh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Atefeh A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Atefeh A&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Alipour Atefeh</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shota Yuasa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">8</div><div class="addr-line">Laboratory of Dynamic Cell Biology, Department of Biotechnology</div>, <div class="insititution">Graduate School of Engineering, Osaka University</div>, <div class="addr-line">1-1 Yamadaoka, Suita, Osaka 565-0871</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shota+Yuasa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yuasa S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yuasa S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Shota Yuasa</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sachihiro Matsunaga</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">8</div><div class="addr-line">Laboratory of Dynamic Cell Biology, Department of Biotechnology</div>, <div class="insititution">Graduate School of Engineering, Osaka University</div>, <div class="addr-line">1-1 Yamadaoka, Suita, Osaka 565-0871</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sachihiro+Matsunaga">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Matsunaga S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Matsunaga S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Sachihiro Matsunaga</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kiichi Fukui</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">8</div><div class="addr-line">Laboratory of Dynamic Cell Biology, Department of Biotechnology</div>, <div class="insititution">Graduate School of Engineering, Osaka University</div>, <div class="addr-line">1-1 Yamadaoka, Suita, Osaka 565-0871</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 438-52-3935. Fax. +81 438-52-3934. E-mail: <a href="mailto:tabata@kazusa.or.jp" target="_blank">tabata@kazusa.or.jp</a> (S.T.); Tel. +81 6-6879-7440. Fax. +81 6-6879-7441. E-mail: <a href="mailto:kfukui@bio.eng.osaka-u.ac.jp" target="_blank">kfukui@bio.eng.osaka-u.ac.jp</a> (K.F.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kiichi+Fukui">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fukui K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fukui K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Kiichi Fukui</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 18, Issue 1, 1 February 2011, Pages 65–76, <a href='https://doi.org/10.1093/dnares/dsq030'>https://doi.org/10.1093/dnares/dsq030</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">13 December 2010</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">21 October 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">12 November 2010</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="377199" href="/dnaresearch/article-pdf/18/1/65/1072092/dsq030.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Shusei Sato, Hideki Hirakawa, Sachiko Isobe, Eigo Fukai, Akiko Watanabe, Midori Kato, Kumiko Kawashima, Chiharu Minami, Akiko Muraki, Naomi Nakazaki, Chika Takahashi, Shinobu Nakayama, Yoshie Kishida, Mitsuyo Kohara, Manabu Yamada, Hisano Tsuruoka, Shigemi Sasamoto, Satoshi Tabata, Tomoyuki Aizu, Atsushi Toyoda, Tadasu Shin-i, Yohei Minakuchi, Yuji Kohara, Asao Fujiyama, Suguru Tsuchimoto, Shin'ichiro Kajiyama, Eri Makigano, Nobuko Ohmido, Nakako Shibagaki, Joyce A. Cartagena, Naoki Wada, Tsutomu Kohinata, Alipour Atefeh, Shota Yuasa, Sachihiro Matsunaga, Kiichi Fukui;  Sequence Analysis of the Genome of an Oil-Bearing Tree, <em>Jatropha curcas</em> L., <em>DNA Research</em>, Volume 18, Issue 1, 1 February 2011, Pages 65–76, <a href='https://doi.org/10.1093/dnares/dsq030'>https://doi.org/10.1093/dnares/dsq030</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/377199?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/377199?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/377199?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/377199?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/377199?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/377199?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/377199?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Sequence%20Analysis%20of%20the%20Genome%20of%20an%20Oil-Bearing%20Tree%2C%20%26lt%3Bem%26gt%3BJatropha%20curcas%26lt%3B%2Fem%26gt%3B%20L.&amp;publicationDate=2010-12-13&amp;volumeNum=18&amp;issueNum=1&amp;author=Sato%2C%20Shusei%3B%20Hirakawa%2C%20Hideki&amp;startPage=65&amp;endPage=76&amp;contentId=10.1093%2Fdnares%2Fdsq030&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f1%2f65%2f377199" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5500855 id="5500855" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The whole genome of <em>Jatropha curcas</em> was sequenced, using a combination of the conventional Sanger method and new-generation multiplex sequencing methods. Total length of the non-redundant sequences thus obtained was 285 858 490 bp consisting of 120 586 contigs and 29 831 singlets. They accounted for ∼95% of the gene-containing regions with the average G + C content was 34.3%. A total of 40 929 complete and partial structures of protein encoding genes have been deduced. Comparison with genes of other plant species indicated that 1529 (4%) of the putative protein-encoding genes are specific to the Euphorbiaceae family. A high degree of microsynteny was observed with the genome of castor bean and, to a lesser extent, with those of soybean and <em>Arabidopsis thaliana</em>. In parallel with genome sequencing, cDNAs derived from leaf and callus tissues were subjected to pyrosequencing, and a total of 21 225 unigene data have been generated. Polymorphism analysis using microsatellite markers developed from the genomic sequence data obtained was performed with 12 <em>J. curcas</em> lines collected from various parts of the world to estimate their genetic diversity. The genomic sequence and accompanying information presented here are expected to serve as valuable resources for the acceleration of fundamental and applied research with <em>J. curcas</em>, especially in the fields of environment-related research such as biofuel production. Further information on the genomic sequences and DNA markers is available at <a class="link link-uri" href="http://www.kazusa.or.jp/jatropha/">http://www.kazusa.or.jp/jatropha/</a>.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Jatropha curcas L.&quot;"><em>Jatropha curcas</em> L.</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome sequencing&quot;">genome sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;cDNA sequencing&quot;">cDNA sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;microsatellite markers&quot;">microsatellite markers</a></div>                            <h2 scrollto-destination=5500857 id="5500857" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>To reconcile increasing energy consumption with worsening global environmental conditions is a fundamental concern of the contemporary society. Fossil fuel deposits are rapidly diminishing, and their consumption raises carbon dioxide discharge levels. Alternative fuels, such as bioethanol and biodiesel, show great promise for alleviating the problems caused by the consumption of fossil fuel.</p><p><em>Jatropha curcas</em> L. is a plant belonging to the family Euphorbiaceae that is endemic to tropical America. It is now grown commercially in tropical and subtropical Africa and Asia. Jatropha has considerable potential for various uses including biofuels.<sup><a reveal-id="DSQ030C1" data-open="DSQ030C1" class="link link-ref link-reveal xref-bibr">1</a></sup> The plant can grow at rainfall levels as low as 200 mm per annum.<sup><a reveal-id="DSQ030C2" data-open="DSQ030C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Medicinal compounds are found in various parts of the plant,<sup><a reveal-id="DSQ030C1" data-open="DSQ030C1" class="link link-ref link-reveal xref-bibr">1</a></sup> but it is the potentially high yield of oil per unit land area, which is second only to oil palm,<sup><a reveal-id="DSQ030C3" data-open="DSQ030C3" class="link link-ref link-reveal xref-bibr">3</a></sup> that makes Jatropha an outstanding biofuel plant. Furthermore, the quality of oil in its seeds is suitable for production of biodiesel as they contain more than 75% unsaturated fatty acids.<sup><a reveal-id="DSQ030C4" data-open="DSQ030C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Despite its cultivation throughout the tropical and subtropical world, the positive attributes of this plant are not fully understood in terms of breeding and utilization.<sup><a reveal-id="DSQ030C3" data-open="DSQ030C3" class="link link-ref link-reveal xref-bibr">3</a></sup> This can be attributed mainly to the lack of information on its genetics and genomics. The genome size (∼410 Mb) and the base composition have been estimated by flow cytometry, and karyotypes have been characterized.<sup><a reveal-id="DSQ030C5" data-open="DSQ030C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Expressed sequence tags (ESTs) from developing and germinating Jatropha seeds have been reported.<sup><a reveal-id="DSQ030C6" data-open="DSQ030C6" class="link link-ref link-reveal xref-bibr">6</a></sup> However, no further information on the genomic structure of <em>J. curcas</em> is available.</p><p>To understand the genetic system of this plant and to accelerate the process of molecular breeding, we analysed the structure of the whole genome of <em>J. curcas</em>. For genome sequencing, we adopted a combination of BAC end sequencing and shotgun sequencing by the conventional Sanger method and the new-generation multiplex methods, which was followed by information analyses. In addition, microsatellite markers have been developed using the sequence information, and polymorphism among various <em>J. curcas</em> varieties was examined. The information and material resources for the Jatropha genome generated in this study will enhance both fundamental and applied research with <em>J. curcas</em> and related plants.</p>                            <h2 scrollto-destination=5500861 id="5500861" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5500862 id="5500862" class="section-title" data-legacy-id=s2a>2.1. Plant materials</h3>
<p>A <em>J. curcas</em> line originating from the Palawan Island in the Philippines was subjected to genome sequencing. The following 12 lines were used for diversity analysis: Palawan, Indonesia, Indonesia IS, Thai, Chinese, Mexico 2b, Guatemala 1, Guatemala 2, Tanzania, Madagascar, Cape Verde, and Uganda. The Indonesia IS and Thai lines were purchased from IS Co. Ltd. (Tokyo, Japan) and Nikko-Seed Co. Ltd. (Tochigi, Japan), respectively. The Uganda and the remaining nine lines were kindly provided by BBL International (Osaka, Japan) and Nippon Biodiesel Fuel Co., Ltd. (Tokyo, Japan), respectively.</p>                            <h3 scrollto-destination=5500864 id="5500864" class="section-title" data-legacy-id=s2b>2.2. Construction of BAC libraries</h3>
<p>BAC genomic libraries were constructed using the genomic DNA of <em>J. curcas</em> partially digested with either <em>Mbo</em>I or <em>Hin</em>dIII and Copy Control pCC1BAC as a cloning vector. The average insert size of these libraries was 80.2 kb for the <em>Mbo</em>I library and 94.9 and 63.4 kb for two independent preparations of the <em>Hin</em>dIII libraries. Both libraries covered the haploid genome 9.2 times in total.</p>                            <h3 scrollto-destination=5500866 id="5500866" class="section-title" data-legacy-id=s2c>2.3. BAC sequencing</h3>
<p>To analyse end sequences, BAC DNAs were amplified using a TempliPhi large construction kit (GE Healthcare, UK), and the end sequences were analysed according to the Sanger method using a cycle sequencing kit (Big Dye-terminator kit, Applied Biosystems, USA) with DNA sequencer type 3730xl (Applied Biosystems). High-quality BAC sequences were determined by the shotgun method using the Sanger sequencing protocol, as described previously.<sup><a reveal-id="DSQ030C7" data-open="DSQ030C7" class="link link-ref link-reveal xref-bibr">7</a></sup></p>                            <h3 scrollto-destination=5500868 id="5500868" class="section-title" data-legacy-id=s2d>2.4. Shotgun genomic sequencing</h3>
<p>For sequencing by the Sanger method, shotgun libraries with average insert sizes of 2.5 kb were generated using pBluescript SK- as a cloning vector, and these were used to transform <em>Escherichia coli</em> ElectroTen-Blue (Agilent Technologies, Santa Clara, CA, USA). The shotgun clones were propagated in microtiter plates, and the plasmid DNA was amplified using a TempliPhi kit (GE Healthcare). Sequencing was performed using a cycle sequencing kit (Big Dye-terminator Cycle Sequencing kit, Applied Biosystems) with DNA sequencer type 3730xl (Applied Biosystems) or DeNOVA-5000HT (Shimadzu Co., Japan) according to the protocols recommended by the manufacturers.</p><p>High-throughput multiplex sequencing was carried out using a Genome Sequencer (GS) FLX Instrument (Roche Diagnostics, USA) and Genome Analyzer II (Illumina Inc., USA) sequencers. A 5-µg sample of Jatropha total cellular DNA was sheared by nebulization and subjected to library preparation followed by shotgun sequencing using the GS FLX platform. For the 3-kb paired-end sequencing, the library was prepared using GS Titanium Library Paired End Adaptors according to the manufacturer's instructions. For sequencing by an Illumina-solexa GAII sequencer, the sample was prepared according to the manufacturer's manual. Briefly, 1 µg of the total cellular genomic DNA was fragmented by the Covaris S1 instrument (Covaris Inc.). The fragmented DNA was repaired, and the adapters for paired-end sequencing (36, 51, and 76 cycles) were then ligated to the repaired DNA fragment. The size-selected fragment (300–350 bp) by agarose gel electrophoresis was PCR amplified, and the PCR product was validated using a 2100 Bioanalyzer (Agilent Technologies) and a 7900HT Fast Real-Time PCR system (ABI). The sample was then run on a Genome Analyzer II using the 36 cycles sequencing kits. Base-calling was performed using the Genome Analyzer Pipeline.</p>                            <h3 scrollto-destination=5500871 id="5500871" class="section-title" data-legacy-id=s2e>2.5. cDNA sequencing</h3>
<p>Total RNA was extracted from leaf and callus tissue using an RNeasy Plant Mini Kit (Qiagen, Germany). mRNA was purified from the total RNA using Oligotex-dT30 (Takara Bio Inc., Japan). Sequencing was performed with a GS FLX Instrument (Roche Diagnostics) using the cDNA rapid library method according to the manufacturer's instructions.</p>                            <h3 scrollto-destination=5500873 id="5500873" class="section-title" data-legacy-id=s2f>2.6. Assembly of sequence data</h3>
<p>Reconstruction of the genome sequence of <em>J. curcas</em> was performed in the following two steps: assembly of sequence data generated by different types of DNA sequencers, and scaffolding and base correction.</p><p>The sequence data collected according to the Sanger protocol using a 3730xl capillary sequencer were subjected to trimming of sequences derived from cloning vectors with the Figaro and Lucy programs,<sup><a reveal-id="DSQ030C8" data-open="DSQ030C8" class="link link-ref link-reveal xref-bibr">8</a></sup> followed by assembly with the PCAP.rep program.<sup><a reveal-id="DSQ030C9" data-open="DSQ030C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Base-calling of the sequence data generated by pyrosequencing using a GS FLX sequencer was performed using the Pyrobayes program.<sup><a reveal-id="DSQ030C10" data-open="DSQ030C10" class="link link-ref link-reveal xref-bibr">10</a></sup> The sequence reads artificially replicated during an emulsion PCR were removed by a 454 replicate filter,<sup><a reveal-id="DSQ030C11" data-open="DSQ030C11" class="link link-ref link-reveal xref-bibr">11</a></sup> and the remaining reads were assembled using MIRA version 3 rc4 software.<sup><a reveal-id="DSQ030C12" data-open="DSQ030C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Contigs and singlets generated by assembly using the Sanger protocol and pyrosequencing were separately subjected to similarity searches for sequences of the chloroplast (GenBank: FJ695500) of <em>J. curcas</em> and the mitochondria (GenBank: Y08501) of <em>Arabidopsis thaliana</em><sup><a reveal-id="DSQ030C13" data-open="DSQ030C13" class="link link-ref link-reveal xref-bibr">13</a></sup> using the Megablast program.<sup><a reveal-id="DSQ030C14" data-open="DSQ030C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Matching sequences were then removed. All remaining contigs and singlets were assembled using the PCAP.rep program.<sup><a reveal-id="DSQ030C9" data-open="DSQ030C9" class="link link-ref link-reveal xref-bibr">9</a></sup> BAC end sequences in which the vector sequences were trimmed by Figaro and Lucy programs<sup><a reveal-id="DSQ030C8" data-open="DSQ030C8" class="link link-ref link-reveal xref-bibr">8</a></sup> in advance were further integrated in the resulting sequences using PCAP.rep.<sup><a reveal-id="DSQ030C9" data-open="DSQ030C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Then, sequences 99 bp and shorter were removed.</p><p>The resulting contigs and singlets were designated as follows. The contigs containing sequences from the Sanger sequencing, pyrosequencing, and BAC end sequencing were prefixed with ‘JcCA’ followed by a seven-digit number. The contigs containing sequences from both the Sanger and 454 sequencing were prefixed with ‘JcCB’ followed by a seven-digit number. The contigs containing sequences from the Sanger sequencing and the pyrosequencing were prefixed with ‘JcCC’ and ‘JcCD', respectively. The singlets from the Sanger sequencing and pyrosequencing that were not assembled into other sequences throughout the whole process were prefixed with ‘JcSR’ and ‘JcPR', respectively.</p><p>For improvement of data quality, both single and mate-pair reads by an Illumina GAII sequencer were collected and assembled using the Velvet program.<sup><a reveal-id="DSQ030C15" data-open="DSQ030C15" class="link link-ref link-reveal xref-bibr">15</a></sup> The resulting contig sequences were mapped onto the contigs generated by hybrid assembly to correct the short insertion–deletion (indels) errors.</p><p>Both paired-end reads of the genomic DNA and single reads of cDNAs by the GS FLX sequencer were used for scaffolding. Paired-end reads of the genomic DNA were assembled with the MIRA program<sup><a reveal-id="DSQ030C12" data-open="DSQ030C12" class="link link-ref link-reveal xref-bibr">12</a></sup> according to the manufacturer's instructions, and the resulting sequences were used for scaffolding of the contig sequences generated by the Sanger sequencing and pyrosequencing using the GS reference mapper ver. 2.3 program. In parallel, a mixture of cDNAs derived from leaf and callus tissue of Jatropha was subjected to sequencing by GS FLX, and the data obtained were subjected to assembly with the MIRA ver. 3.0.5 program in the EST mode.<sup><a reveal-id="DSQ030C12" data-open="DSQ030C12" class="link link-ref link-reveal xref-bibr">12</a></sup> In addition, the resulting cDNA sequences as well as the Jatropha ESTs retrieved from public DNA databases were used for scaffolding using the Blat program.<sup><a reveal-id="DSQ030C16" data-open="DSQ030C16" class="link link-ref link-reveal xref-bibr">16</a></sup></p>                            <h3 scrollto-destination=5500879 id="5500879" class="section-title" data-legacy-id=s2g>2.7. Gene assignment</h3>
<p>Gene prediction and modelling were performed by automatic gene assignment programs that employ <em>ab initio</em> gene finding and similarity searches. For <em>ab initio</em> gene finding, predictions of protein-coding regions were carried out using GeneMark.hmm<sup><a reveal-id="DSQ030C17" data-open="DSQ030C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and Genescan<sup><a reveal-id="DSQ030C18" data-open="DSQ030C18" class="link link-ref link-reveal xref-bibr">18</a></sup> programs with the matrix trained by an <em>A. thaliana</em> gene set, and predictions of exon–intron structure were performed using NetGene2<sup><a reveal-id="DSQ030C19" data-open="DSQ030C19" class="link link-ref link-reveal xref-bibr">19</a></sup> and SplicePredictor<sup><a reveal-id="DSQ030C20" data-open="DSQ030C20" class="link link-ref link-reveal xref-bibr">20</a></sup> programs. Similarity searches for potential protein-coding regions and all contigs were performed against a Uniref database (<a class="link link-uri" href="http://www.ebi.ac.uk/uniref/">http://www.ebi.ac.uk/uniref/</a>) using Blastp and Blastx programs<sup><a reveal-id="DSQ030C21" data-open="DSQ030C21" class="link link-ref link-reveal xref-bibr">21</a></sup> with a cut-off (<em>E</em>-value ≤ 1e−3). The exon–intron structure of potential protein-coding regions and the contigs homologous to the Uniref database (<a class="link link-uri" href="http://www.ebi.ac.uk/uniref/">http://www.ebi.ac.uk/uniref/</a>) were predicted using the Nap program.<sup><a reveal-id="DSQ030C22" data-open="DSQ030C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Suitable exon-intron structures were determined by considering all the information above. The predicted gene structures were further confirmed by comparison to cDNA sequences analysed in this study. The protein-coding genes assigned in this manner were denoted by IDs with the contig names followed by sequential numbers from one end to another. They were classified into four categories based on sequence similarity to registered genes: genes with complete structure, pseudogenes, genes with partial structure, and transposons/retrotranspons.</p>                            <h3 scrollto-destination=5500881 id="5500881" class="section-title" data-legacy-id=s2h>2.8. Functional assignment and classification of potential protein-coding genes</h3>
<p>To assign the gene families, functional domains, GO terms, and GO accession numbers,<sup><a reveal-id="DSQ030C23" data-open="DSQ030C23" class="link link-ref link-reveal xref-bibr">23</a></sup> the predicted genes were searched against InterPro using InterProScan<sup><a reveal-id="DSQ030C24" data-open="DSQ030C24" class="link link-ref link-reveal xref-bibr">24</a></sup> software. Genes with an <em>E</em>-value of &lt;1.0 were taken into account. GO terms were grouped into plant GO slim categories using the map2slim program (<a class="link link-uri" href="http://www.geneontology.org/GO.slims.shtml">http://www.geneontology.org/GO.slims.shtml</a>).</p><p>The predicted protein-encoding genes were mapped onto KEGG metabolic pathways<sup><a reveal-id="DSQ030C25" data-open="DSQ030C25" class="link link-ref link-reveal xref-bibr">25</a></sup> using the Blastp program<sup><a reveal-id="DSQ030C21" data-open="DSQ030C21" class="link link-ref link-reveal xref-bibr">21</a></sup> against the GENES database.<sup><a reveal-id="DSQ030C25" data-open="DSQ030C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Thresholds of amino acid sequence identity ≥25% and of length coverage of the query sequence ≥50% with a cut-off (<em>E</em>-value ≤ 1e−10) were applied.</p>                            <h3 scrollto-destination=5500884 id="5500884" class="section-title" data-legacy-id=s2i>2.9. Phylogenetic analysis</h3>
<p>Evolutionary relationships of proteins of casben synthase genes, disease resistance genes, MADS-box genes, flowering genes, and COL genes were analysed using predicted amino acid sequences from different databases aligned with the program CLUSTALW (Ver. 1.83).<sup><a reveal-id="DSQ030C26" data-open="DSQ030C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Evolutionary relationships were inferred using a neighbour-joining algorithm.<sup><a reveal-id="DSQ030C27" data-open="DSQ030C27" class="link link-ref link-reveal xref-bibr">27</a></sup> All positions containing alignment gaps and missing data were eliminated only in pairwise sequence comparisons. Phylogenetic trees were constructed with MEGA4 software.<sup><a reveal-id="DSQ030C28" data-open="DSQ030C28" class="link link-ref link-reveal xref-bibr">28</a></sup></p>                            <h3 scrollto-destination=5500886 id="5500886" class="section-title" data-legacy-id=s2j>2.10. Polymorphism analysis</h3>
<p>Microsatellite or simple sequence repeats (SSRs) 15 nucleotides in length, containing all possible combinations of di-nucleotide (NN), tri-nucleotide (NNN) and tetra-nucleotide (NNNN) repeat, were identified from the Jatropha genome sequences using the SSRIT (SSR Identification Tool) program.<sup><a reveal-id="DSQ030C29" data-open="DSQ030C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Primer pairs for amplification of SSR-containing regions were designed based on the flanking sequences of each SSR with the Primer 3 program<sup><a reveal-id="DSQ030C30" data-open="DSQ030C30" class="link link-ref link-reveal xref-bibr">30</a></sup> so that amplified fragment sizes were between 90 and 300 bp in length. One hundred microsatellite markers were subjected to examination of polymorphisms among 12 lines of <em>J. curcas</em>.</p><div class="block-child-p">PCR amplifications (5 µl) were performed on 0.7 ng of Jatropha genomic DNA in 1 × PCR buffer (BIOLINE, London, UK), 3 mM MgCl<sub>2</sub>, 0.04 U BIOTAQ™ DNA Polymerase (BIOLINE), 0.8 mM dNTPs, and 0.4 μM of each primer, using the modified ‘Touchdown PCR’ protocol described by Sato <em>et al</em>.<sup><a reveal-id="DSQ030C7" data-open="DSQ030C7" class="link link-ref link-reveal xref-bibr">7</a></sup> PCR products were separated by 10% polyacrylamide gel electrophoresis using TBE buffer, and data were collected as described previously. Allele detection and genotype code typing were performed using the Polyans program (ver.1.1; <a class="link link-uri" href="http://www.kazusa.or.jp/polyans">http://www.kazusa.or.jp/polyans</a>). The presence or the absence of amplification and the number of different-sized fragments, which was taken as the number of alleles, were recorded. Loci for which there was no amplification were designated as null alleles. PIC was calculated using the following equation: <a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq030ueq1.gif?Expires=2147483647&Signature=2GL9EFI5jHjWc4d16dNM22l~jxaQftx52VLBXwFOir9cPMwiHaDlxNwbDxzgEnaFt4-BWZkF8qNoAwMi9JDkooeTVk7xjjFwMht2Pt68LIGWnoFEfuGvqElsQScfsj1VYBUzPxJ1vfDLpCpvd9abZsI6bGQlbD2SYTyyc7lQ8p2xz03F0Zl1aflMa~9aD2ePElobiErOPAe42vg6NKdZo1~Lpl9CGr7J8ZXYSV3aEYUkJwoUCC0C2vtcaBFkIzD88GhyQ6~cZTEJn~oub5ZTVSND9zXb1rdRwEvjTdM17C63cZ~AmqRd0J4a2g6z7MALwttlYm-v0xjEEUYZ7Nze3Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsq030ueq1.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/m_dsq030ueq1.gif?Expires=2147483647&Signature=38Et69GKPPB2mMWSofyDUtD7kXa4Y-3kikvJLrHdMZF6xFEP7uvw8s9oghd0O8GJ27Ck~JBSeWHoj5pza-tnb56S9~JtHR6ym5TDmloz03Nahf-aUJ0eTAWz~iCCgch7Blxh1Nx-ODxFueKvJNBr3b83zEjmIXHY31LavvOXANNLk12gb4jMydJkPhXVy0pMNei59XuzJ2r2CBo2R76cjfXJySxDs5Ams50zcKGMsU1Yt89lpdOhS4JLS9YpHUw4vvCrYEbkpFaVNldEgv1c39bZPE~VMuJDC7e4li7w5Qb624YYBsDE0VSdbTKqmbqLW1~cMDW3GdjZVX9kOMYV6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsq030ueq1.gif" /></a></div></div></div> where <em>P<sub>ij</sub></em> is the frequency of the <em>j</em>th allele for the <em>i</em>th locus. NTSYSpc ver. 2.21c software (Applied Biostatistics Inc., New York, USA) was employed to perform cluster analysis. The SimQual and SAHN modules were used for estimation of genetic distance and a genetic tree, respectively, with the coefficient in SimQual set to SM, and the clustering method set to UPGMA.</div>                            <h2 scrollto-destination=5500889 id="5500889" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5500890 id="5500890" class="section-title" data-legacy-id=s3a>3.1. Sequence analysis of the Jatropha genome</h3>
<p>The strategy and the status of sequencing and assembly are summarized in Fig. <a reveal-id="DSQ030F1" data-open="DSQ030F1" class="link link-reveal link-table xref-fig">1</a>. Briefly, the 1 025 000 reads of the Sanger sequencing and the 2 312 828 reads of pyrosequencing, which were appropriately processed in advance as indicated in Fig. <a reveal-id="DSQ030F1" data-open="DSQ030F1" class="link link-reveal link-table xref-fig">1</a>, were independently assembled using the PCAP.rep<sup><a reveal-id="DSQ030C9" data-open="DSQ030C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and MIRA programs,<sup><a reveal-id="DSQ030C12" data-open="DSQ030C12" class="link link-ref link-reveal xref-bibr">12</a></sup> respectively. The resulting contigs and singlets were subjected to hybrid assembly by PCAP.rep,<sup><a reveal-id="DSQ030C9" data-open="DSQ030C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and the 53 000 BAC end sequences were further integrated. </p>                        <a id="5500892" scrollto-destination="5500892"></a>
<div id="DSQ030F1" data-id="DSQ030F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/m_dsq03001.jpeg?Expires=2147483647&Signature=khumzQvDtFN1nv02dHcfCA8f7-34ZOQT~LgIQR7C4RDzGkO-oT-RvTFxMH8KLPFGY~9VJ1YFSak6ps6piVg40YsbAxcHNulS9~bvqoQ~ZvAPEeMlUcTFocry4M9uUWhcfu8lwX43M~CQ0IpdQBHWngmnKcvEFh1xWbczW3Pc1EwTDBrTTlTrvN34vtIYWJDzOmoQc-teUiFfzasjkxcuDacUglrxVvY7s3aBMJB6il9YlDKiAwOw6vnx0wMmlWIkoxN5BBYxLnYNXdodSuFTlpLdk0wN0~N~gGmZS8cA9vLWmzTjM2K0ZfdYYQAqVP40TeQA8rq438-3k7EfsooQtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The strategy and status of sequencing and assembly." path-from-xml="dsq03001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5500892/dsq03001.jpeg" path-from-xml="dsq03001.jpeg" target="_blank">View large</a><a section="5500892" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq03001.jpeg?Expires=2147483647&Signature=MfjmRezjiPpVJ2T0FykdgIA9XCH8HuQN1JHB9moAtEH8WjGmiRWeaESVB6J-nRa6tlQkxjM~QIWyqsRcrQl8zP-8tyCb8sPsCjJ~EZ6qab10QIshrpLUnmom7DkrXFPiYZQC0eoyodsmwi46X0bwHWEBE1bGCPKzBMKL~dK2PunCqNgx4gp5U187oan62eSezsGZn~NgSkdebkELLkISUB9jdouYqKyvr-G3rhfJvPmGrUgEyVCVYqm4k4c63x8MJqfXth~7NfDgwNSETgV3~tlxZX5NB6WDBORT84lCGFor0wWREzQQfn5YwosGQzz~MdRQ5qf8vFUl7g45tLBurA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5500892&ar=377199&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq03001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The strategy and status of sequencing and assembly.</p></div></div><div content-id="DSQ030F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/m_dsq03001.jpeg?Expires=2147483647&Signature=khumzQvDtFN1nv02dHcfCA8f7-34ZOQT~LgIQR7C4RDzGkO-oT-RvTFxMH8KLPFGY~9VJ1YFSak6ps6piVg40YsbAxcHNulS9~bvqoQ~ZvAPEeMlUcTFocry4M9uUWhcfu8lwX43M~CQ0IpdQBHWngmnKcvEFh1xWbczW3Pc1EwTDBrTTlTrvN34vtIYWJDzOmoQc-teUiFfzasjkxcuDacUglrxVvY7s3aBMJB6il9YlDKiAwOw6vnx0wMmlWIkoxN5BBYxLnYNXdodSuFTlpLdk0wN0~N~gGmZS8cA9vLWmzTjM2K0ZfdYYQAqVP40TeQA8rq438-3k7EfsooQtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The strategy and status of sequencing and assembly." path-from-xml="dsq03001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5500892/dsq03001.jpeg" path-from-xml="dsq03001.jpeg" target="_blank">View large</a><a section="5500892" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq03001.jpeg?Expires=2147483647&Signature=MfjmRezjiPpVJ2T0FykdgIA9XCH8HuQN1JHB9moAtEH8WjGmiRWeaESVB6J-nRa6tlQkxjM~QIWyqsRcrQl8zP-8tyCb8sPsCjJ~EZ6qab10QIshrpLUnmom7DkrXFPiYZQC0eoyodsmwi46X0bwHWEBE1bGCPKzBMKL~dK2PunCqNgx4gp5U187oan62eSezsGZn~NgSkdebkELLkISUB9jdouYqKyvr-G3rhfJvPmGrUgEyVCVYqm4k4c63x8MJqfXth~7NfDgwNSETgV3~tlxZX5NB6WDBORT84lCGFor0wWREzQQfn5YwosGQzz~MdRQ5qf8vFUl7g45tLBurA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5500892&ar=377199&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq03001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The strategy and status of sequencing and assembly.</p></div></div><p>For improvement of data quality, 86 028 428 (36 bases long from each end for each read) and 96 580 336 short-reads (50 and 31 bases long from each end for each read) by mate-pair sequencing with the Illumina GAII sequencer were assembled into 569 576 contigs (total length: 75 539 079 bp) by the Velvet program.<sup><a reveal-id="DSQ030C15" data-open="DSQ030C15" class="link link-ref link-reveal xref-bibr">15</a></sup> The resulting contig sequences were mapped onto those generated by hybrid assembly to correct short indels errors. These indels were probably attributed to classified insertions, deletions, and mismatches by their association with miscall from homopolymer effects. As a result of mapping, 7459 loci on the 5025 contigs were revised.</p><p>A total of 695 928 3-kb paired-end reads by the GS FLX sequencer were used for scaffolding of the generated contigs and singlets using the GS reference mapper ver. 2.3 program, as described in the ‘Materials and Methods’ section. In parallel, 991 050 reads of cDNA sequences by pyrosequencing were collected; 534 137 were derived from leaf tissue and 456 913 from callus tissue. The cDNA sequences were assembled with MIRA 3.05 in the EST mode,<sup><a reveal-id="DSQ030C12" data-open="DSQ030C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and 21 225 unigene sets were generated consisting of 13 610 contigs and 7615 singlets used for scaffolding by the BLAT program.<sup><a reveal-id="DSQ030C16" data-open="DSQ030C16" class="link link-ref link-reveal xref-bibr">16</a></sup> In addition, unigenes generated from 26 447 ESTs registered in public DNA databases (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/dbEST/">http://www.ncbi.nlm.nih.gov/dbEST/</a>) were also used for scaffolding. As a result of scaffolding, the 44 153 contigs and singlets constructed by hybrid assembly were integrated into 15 300 scaffolds. The total length of the scaffolds was 129 291 074 bp. The longest scaffold (JcS_100001) had 56 042 bp, and the average scaffold length was 8450 bp. The constructed scaffolds were designated as JcS followed by sequential numbers.</p><p>The total length of the final genomic sequences of <em>J. curcas</em> obtained was 285 858 490 bp, consisting of 120 586 contigs (276 710 623 bp total) and 29 831 singlets (9 147 867 bp total), which is ∼70 and 75% of the whole genome of 410<sup><a reveal-id="DSQ030C5" data-open="DSQ030C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and 380 Mb (N. Wada, unpublished result), respectively, estimated by flow cytometry. The average length of contigs and singlets was 1900 bp. Statistics of the assembly are summarized in Table <a reveal-id="DSQ030TB1" data-open="DSQ030TB1" class="link link-reveal link-table xref-fig">1</a>. The longest contig was 29 744 bp, and N50 length was 3833 bp. The distribution of contig lengths is shown in the <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>. The average G + C content of the contigs was 34.3%. </p>                        <a id="5500896" scrollto-destination="5500896"></a>
<div content-id="DSQ030TB1" class="table-modal"><div class="table-wrap"><div id="DSQ030TB1" class="table-wrap-title" data-id="DSQ030TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Assembly statistics</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Total length of contigs and singlets </td><td>285 858 490 </td></tr><tr><td>Total number of contigs and singlets </td><td>150 417 </td></tr><tr><td>Average length of contigs and singlets </td><td>1900 </td></tr><tr><td>Maximum length of contigs and singlets </td><td>29 744 </td></tr><tr><td>N50 </td><td>3833 </td></tr><tr><td>G + C content (%) </td><td>34.3 </td></tr><tr><td>Number of contigs (JcCA) </td><td>32 212 </td></tr><tr><td>Number of contigs (JcCB) </td><td>60 363 </td></tr><tr><td>Number of contigs (JcCC) </td><td>2483 </td></tr><tr><td>Number of contigs (JcCD) </td><td>25 528 </td></tr><tr><td>Number of singlets (JcSR) </td><td>26 819 </td></tr><tr><td>Number of singlets (JcPR) </td><td>1347 </td></tr><tr><td>Number of BAC end sequences (JHL/JHS/JMS) </td><td>1665 </td></tr><tr><td colspan="2">Contigs </td></tr><tr><td> Total number of contigs </td><td>120 586 </td></tr><tr><td> Total length of contigs </td><td>276 710 623 </td></tr><tr><td> Average length of contigs </td><td>2295 </td></tr><tr><td colspan="2">Singlets </td></tr><tr><td> Total number of singlets </td><td>29 831 </td></tr><tr><td> Total length of singlets </td><td>9 147 867 </td></tr><tr><td> Average length of singlets </td><td>307 </td></tr></tbody></table></div><div content-id="DSQ030TB1" class="table-modal"><table><tbody><tr><td>Total length of contigs and singlets </td><td>285 858 490 </td></tr><tr><td>Total number of contigs and singlets </td><td>150 417 </td></tr><tr><td>Average length of contigs and singlets </td><td>1900 </td></tr><tr><td>Maximum length of contigs and singlets </td><td>29 744 </td></tr><tr><td>N50 </td><td>3833 </td></tr><tr><td>G + C content (%) </td><td>34.3 </td></tr><tr><td>Number of contigs (JcCA) </td><td>32 212 </td></tr><tr><td>Number of contigs (JcCB) </td><td>60 363 </td></tr><tr><td>Number of contigs (JcCC) </td><td>2483 </td></tr><tr><td>Number of contigs (JcCD) </td><td>25 528 </td></tr><tr><td>Number of singlets (JcSR) </td><td>26 819 </td></tr><tr><td>Number of singlets (JcPR) </td><td>1347 </td></tr><tr><td>Number of BAC end sequences (JHL/JHS/JMS) </td><td>1665 </td></tr><tr><td colspan="2">Contigs </td></tr><tr><td> Total number of contigs </td><td>120 586 </td></tr><tr><td> Total length of contigs </td><td>276 710 623 </td></tr><tr><td> Average length of contigs </td><td>2295 </td></tr><tr><td colspan="2">Singlets </td></tr><tr><td> Total number of singlets </td><td>29 831 </td></tr><tr><td> Total length of singlets </td><td>9 147 867 </td></tr><tr><td> Average length of singlets </td><td>307 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5500896" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ030TB1" class="table-wrap-title" data-id="DSQ030TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Assembly statistics</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Total length of contigs and singlets </td><td>285 858 490 </td></tr><tr><td>Total number of contigs and singlets </td><td>150 417 </td></tr><tr><td>Average length of contigs and singlets </td><td>1900 </td></tr><tr><td>Maximum length of contigs and singlets </td><td>29 744 </td></tr><tr><td>N50 </td><td>3833 </td></tr><tr><td>G + C content (%) </td><td>34.3 </td></tr><tr><td>Number of contigs (JcCA) </td><td>32 212 </td></tr><tr><td>Number of contigs (JcCB) </td><td>60 363 </td></tr><tr><td>Number of contigs (JcCC) </td><td>2483 </td></tr><tr><td>Number of contigs (JcCD) </td><td>25 528 </td></tr><tr><td>Number of singlets (JcSR) </td><td>26 819 </td></tr><tr><td>Number of singlets (JcPR) </td><td>1347 </td></tr><tr><td>Number of BAC end sequences (JHL/JHS/JMS) </td><td>1665 </td></tr><tr><td colspan="2">Contigs </td></tr><tr><td> Total number of contigs </td><td>120 586 </td></tr><tr><td> Total length of contigs </td><td>276 710 623 </td></tr><tr><td> Average length of contigs </td><td>2295 </td></tr><tr><td colspan="2">Singlets </td></tr><tr><td> Total number of singlets </td><td>29 831 </td></tr><tr><td> Total length of singlets </td><td>9 147 867 </td></tr><tr><td> Average length of singlets </td><td>307 </td></tr></tbody></table></div><div content-id="DSQ030TB1" class="table-modal"><table><tbody><tr><td>Total length of contigs and singlets </td><td>285 858 490 </td></tr><tr><td>Total number of contigs and singlets </td><td>150 417 </td></tr><tr><td>Average length of contigs and singlets </td><td>1900 </td></tr><tr><td>Maximum length of contigs and singlets </td><td>29 744 </td></tr><tr><td>N50 </td><td>3833 </td></tr><tr><td>G + C content (%) </td><td>34.3 </td></tr><tr><td>Number of contigs (JcCA) </td><td>32 212 </td></tr><tr><td>Number of contigs (JcCB) </td><td>60 363 </td></tr><tr><td>Number of contigs (JcCC) </td><td>2483 </td></tr><tr><td>Number of contigs (JcCD) </td><td>25 528 </td></tr><tr><td>Number of singlets (JcSR) </td><td>26 819 </td></tr><tr><td>Number of singlets (JcPR) </td><td>1347 </td></tr><tr><td>Number of BAC end sequences (JHL/JHS/JMS) </td><td>1665 </td></tr><tr><td colspan="2">Contigs </td></tr><tr><td> Total number of contigs </td><td>120 586 </td></tr><tr><td> Total length of contigs </td><td>276 710 623 </td></tr><tr><td> Average length of contigs </td><td>2295 </td></tr><tr><td colspan="2">Singlets </td></tr><tr><td> Total number of singlets </td><td>29 831 </td></tr><tr><td> Total length of singlets </td><td>9 147 867 </td></tr><tr><td> Average length of singlets </td><td>307 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5500896" target="_blank">View Large</a></div></div><p>Coverage of gene space in the Jatropha genomic sequences was estimated roughly by surveying the matched non-redundant cDNA sequences obtained in this study. Of 21 225 non-redundant cDNA sequences and 26 447 EST sequences in the public databases, 45 029 matched Jatropha genomic sequences with an identity of 95% or more for a stretch of 50 nucleotides, suggesting that 95% of the gene space in the Jatropha genome was covered by the genomic sequences in this study.</p><p>We adopted here the sequencing strategy that combines the conventional Sanger method and the new-generation multiplex sequencing methods with the aid of various computer software for assembly. This strategy is superior in that shortcomings of respective methods are compensated by each other, enabling acquisition of sequences of higher quality in lower cost within a shorter period of time, thus is becoming popular for genome sequencing in both bacteria and eukaryotes.</p>                            <h3 scrollto-destination=5500899 id="5500899" class="section-title" data-legacy-id=s3b>3.2. Characteristic features of the genome</h3>
                            <h4 scrollto-destination=5500900 id="5500900" class="section-title" data-legacy-id=s3b1>3.2.1. Repetitive sequences</h4>
<p>A total of 41 428 di-, tri-, and tetra-nucleotide SSRs ≥15 bp were identified in the Jatropha genomic sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The frequency of the occurrence of these SSRs was estimated to be one SSR in every 7.0 kb in the 289 Mb sequences of the Jatropha genome. The di-, tri-, and tetra-nucleotide SSRs accounted for 46.3, 34.3, and 19.4% of the identified SSRs, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The SSR patterns that appeared frequently were (AT)n, (AAT)n, and (AAAT)n, each representing 71% of di-nucleotide, 60% of tri-nucleotide, and 58% of tetra-nucleotide repeat units, respectively. The tri-nucleotide SSRs, particularly (AAG)n and (AGC)n, were preferentially found in exons. (AT)n, (AG)n, and (AAT)n were enriched in 5′ and 3′ untranslated regions, and (AC)n frequently occurred in introns (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>).</p><p>A search of the Jatropha genomic sequences using the repeat sequence finding program RECON<sup><a reveal-id="DSQ030C31" data-open="DSQ030C31" class="link link-ref link-reveal xref-bibr">31</a></sup> unravelled the occurrence of a variety of repeat elements including class I and class II transposable element (TE) subfamilies and some that were difficult to classify into known subfamilies. Composition of these repeat sequences was analysed with the RepeatMasker program (<a class="link link-uri" href="http://repeatmasker.org/">http://repeatmasker.org/</a>); the results are summarized in Table <a reveal-id="DSQ030TB2" data-open="DSQ030TB2" class="link link-reveal link-table xref-fig">2</a>. The identified repetitive sequences in total occupied 36.6% of the Jatropha genomic sequences. The most abundant repeat category was class I TE (29.9%), in which Gypsy type (19.6%) and Copia type (8.0%) LTR retroelements constituted major components. </p>                        <a id="5500903" scrollto-destination="5500903"></a>
<div content-id="DSQ030TB2" class="table-modal"><div class="table-wrap"><div id="DSQ030TB2" class="table-wrap-title" data-id="DSQ030TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Repetitive sequences in the Jatropha genomic sequences</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Repeat type </th><th colspan="3">Jatropha genomic sequences<hr /> </th></tr><tr><th> </th><th>Number of elements </th><th>Coverage (kb) </th><th>Percentage of sequence </th></tr></thead><tbody><tr><td colspan="4">Class I </td></tr><tr><td> LINEs </td><td>195  </td><td>136.9 </td><td>0.05 </td></tr><tr><td> LTR: Copia </td><td>31 740 </td><td>22 318.2 </td><td>8.03 </td></tr><tr><td> LTR: Gypsy </td><td>67 658 </td><td>56 655.7 </td><td>19.60 </td></tr><tr><td> LTR: other </td><td>13 454 </td><td>6436.6 </td><td>2.23 </td></tr><tr><td>Total class I </td><td>113 047 </td><td>86 447.4 </td><td>29.91 </td></tr><tr><td colspan="4">Class II </td></tr><tr><td> Coding class II </td><td>5709 </td><td>4102.9 </td><td>1.42 </td></tr><tr><td> MITE </td><td>5980 </td><td>1802.8 </td><td>0.62 </td></tr><tr><td>Total class II </td><td>11 689 </td><td>5905.7 </td><td>2.04 </td></tr><tr><td>Short tandem repeats </td><td>2092 </td><td>148.1 </td><td>0.05 </td></tr><tr><td>Unclassified </td><td>25 977 </td><td>14 953.3 </td><td>5.17 </td></tr></tbody></table></div><div content-id="DSQ030TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Repeat type </th><th colspan="3">Jatropha genomic sequences<hr /> </th></tr><tr><th> </th><th>Number of elements </th><th>Coverage (kb) </th><th>Percentage of sequence </th></tr></thead><tbody><tr><td colspan="4">Class I </td></tr><tr><td> LINEs </td><td>195  </td><td>136.9 </td><td>0.05 </td></tr><tr><td> LTR: Copia </td><td>31 740 </td><td>22 318.2 </td><td>8.03 </td></tr><tr><td> LTR: Gypsy </td><td>67 658 </td><td>56 655.7 </td><td>19.60 </td></tr><tr><td> LTR: other </td><td>13 454 </td><td>6436.6 </td><td>2.23 </td></tr><tr><td>Total class I </td><td>113 047 </td><td>86 447.4 </td><td>29.91 </td></tr><tr><td colspan="4">Class II </td></tr><tr><td> Coding class II </td><td>5709 </td><td>4102.9 </td><td>1.42 </td></tr><tr><td> MITE </td><td>5980 </td><td>1802.8 </td><td>0.62 </td></tr><tr><td>Total class II </td><td>11 689 </td><td>5905.7 </td><td>2.04 </td></tr><tr><td>Short tandem repeats </td><td>2092 </td><td>148.1 </td><td>0.05 </td></tr><tr><td>Unclassified </td><td>25 977 </td><td>14 953.3 </td><td>5.17 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5500903" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ030TB2" class="table-wrap-title" data-id="DSQ030TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Repetitive sequences in the Jatropha genomic sequences</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Repeat type </th><th colspan="3">Jatropha genomic sequences<hr /> </th></tr><tr><th> </th><th>Number of elements </th><th>Coverage (kb) </th><th>Percentage of sequence </th></tr></thead><tbody><tr><td colspan="4">Class I </td></tr><tr><td> LINEs </td><td>195  </td><td>136.9 </td><td>0.05 </td></tr><tr><td> LTR: Copia </td><td>31 740 </td><td>22 318.2 </td><td>8.03 </td></tr><tr><td> LTR: Gypsy </td><td>67 658 </td><td>56 655.7 </td><td>19.60 </td></tr><tr><td> LTR: other </td><td>13 454 </td><td>6436.6 </td><td>2.23 </td></tr><tr><td>Total class I </td><td>113 047 </td><td>86 447.4 </td><td>29.91 </td></tr><tr><td colspan="4">Class II </td></tr><tr><td> Coding class II </td><td>5709 </td><td>4102.9 </td><td>1.42 </td></tr><tr><td> MITE </td><td>5980 </td><td>1802.8 </td><td>0.62 </td></tr><tr><td>Total class II </td><td>11 689 </td><td>5905.7 </td><td>2.04 </td></tr><tr><td>Short tandem repeats </td><td>2092 </td><td>148.1 </td><td>0.05 </td></tr><tr><td>Unclassified </td><td>25 977 </td><td>14 953.3 </td><td>5.17 </td></tr></tbody></table></div><div content-id="DSQ030TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Repeat type </th><th colspan="3">Jatropha genomic sequences<hr /> </th></tr><tr><th> </th><th>Number of elements </th><th>Coverage (kb) </th><th>Percentage of sequence </th></tr></thead><tbody><tr><td colspan="4">Class I </td></tr><tr><td> LINEs </td><td>195  </td><td>136.9 </td><td>0.05 </td></tr><tr><td> LTR: Copia </td><td>31 740 </td><td>22 318.2 </td><td>8.03 </td></tr><tr><td> LTR: Gypsy </td><td>67 658 </td><td>56 655.7 </td><td>19.60 </td></tr><tr><td> LTR: other </td><td>13 454 </td><td>6436.6 </td><td>2.23 </td></tr><tr><td>Total class I </td><td>113 047 </td><td>86 447.4 </td><td>29.91 </td></tr><tr><td colspan="4">Class II </td></tr><tr><td> Coding class II </td><td>5709 </td><td>4102.9 </td><td>1.42 </td></tr><tr><td> MITE </td><td>5980 </td><td>1802.8 </td><td>0.62 </td></tr><tr><td>Total class II </td><td>11 689 </td><td>5905.7 </td><td>2.04 </td></tr><tr><td>Short tandem repeats </td><td>2092 </td><td>148.1 </td><td>0.05 </td></tr><tr><td>Unclassified </td><td>25 977 </td><td>14 953.3 </td><td>5.17 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5500903" target="_blank">View Large</a></div></div>                            <h4 scrollto-destination=5500904 id="5500904" class="section-title" data-legacy-id=s3b2>3.2.2. RNA-coding genes</h4>
<p>A combination of computer prediction and similarity searches of the structural RNA sequence library resulted in identification of 597 putative genes for transfer RNAs in the Jatropha genomic sequences. Although 80 of these were likely to be pseudogenes, the remaining 517 could code for intact tRNAs with 54 species of anticodons (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). This is sufficient for translation of all the amino acids based on the universal codon table.</p><p>A total of 65 genes for snRNAs were assigned by referring to the list of <em>A. thaliana</em> snRNAs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>).<sup><a reveal-id="DSQ030C32" data-open="DSQ030C32" class="link link-ref link-reveal xref-bibr">32</a></sup> Some of these genes were found on the same contigs and scaffolds; thus, they are likely to form clusters in the genome, as they do in <em>A. thaliana</em>.</p>                            <h3 scrollto-destination=5500907 id="5500907" class="section-title" data-legacy-id=s3c>3.3. Characteristic features of protein-encoding genes</h3>
                            <h4 scrollto-destination=5500908 id="5500908" class="section-title" data-legacy-id=s3c1>3.3.1. Prediction of protein-encoding genes</h4>
<p>The Jatropha genomic sequences were subjected to an automatic assignment of protein-encoding genes, and a total of 40 929 genes, besides 16 447 transposon-related genes, were assigned. Complete structures were predicted for 9870 genes, but only partial structures were predicted for 17 863 genes. In addition, 1960 and 11 236 genes were likely to be pseudogenes with complete and truncated structures, respectively. Of the 40 929 presumptive protein-encoding genes, 15 573 (38.0%) carried ESTs with sequence identity of 95% or more for a stretch of 50 nucleotides.</p><p>Structural features of the protein-encoding genes in <em>J. curcas</em> were investigated in detail for 146 genes predicted on the 17 BAC clones (1.36 Mb in total) for which high-quality sequences were obtained by manual finishing and annotation (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). As shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>, the basic structures of the protein-encoding genes in <em>J. curcas</em> are similar to those of <em>A. thaliana</em> except for the average lengths of genes and introns: 3064 versus 1918 bp and 356 versus 157 bp in <em>J. curcas</em> and <em>A. thaliana</em>, respectively.</p>                            <h4 scrollto-destination=5500911 id="5500911" class="section-title" data-legacy-id=s3c2>3.3.2. Gene components</h4>
<p>A similarity search of translated amino acid sequences of the 40 929 presumptive protein-encoding genes was performed using the TrEMBL database as a protein sequence library.<sup><a reveal-id="DSQ030C33" data-open="DSQ030C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The results indicated that 31 822 (77.7%) genes had significant (<em>E</em>-value ≤ 1e−20) sequence similarity to those in this database. Of these genes, 13 067 (41.0%) genes showed sequence similarities to those in a public EST database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/dbEST/">http://www.ncbi.nlm.nih.gov/dbEST/</a>) with a cut-off (<em>E</em>-value ≤ 1e−20) using tBLASTN.</p><p>The 40 929 presumptive protein-encoding genes assigned in <em>J. curcas</em> and those in castor bean (<em>Ricinus communis</em>; 31 221 genes),<sup><a reveal-id="DSQ030C34" data-open="DSQ030C34" class="link link-ref link-reveal xref-bibr">34</a></sup> which belongs to the same family as Jatropha, and <em>A. thaliana</em> (32 615 genes), were classified into plant GO slim categories<sup><a reveal-id="DSQ030C35" data-open="DSQ030C35" class="link link-ref link-reveal xref-bibr">35</a></sup> for comparison (Fig. <a reveal-id="DSQ030F2" data-open="DSQ030F2" class="link link-reveal link-table xref-fig">2</a>). The percentage of the number of genes classified into each GO slim category (i.e. ‘biological process’, ‘cellular component', and ‘molecular function’) was calculated for <em>J. curcas</em>, <em>R. communis</em>, and <em>A. thaliana</em> (Fig. <a reveal-id="DSQ030F2" data-open="DSQ030F2" class="link link-reveal link-table xref-fig">2</a>). </p>                        <a id="5500914" scrollto-destination="5500914"></a>
<div id="DSQ030F2" data-id="DSQ030F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/m_dsq03002.jpeg?Expires=2147483647&Signature=yFMmeT~wLyaiMEUMzX3Zv~yV~1ir9NN8xkOjUPd3kcPWlXFtLDHiDFHbbEavA6eg5tK8DrdtekUCqUmTr3BP~YRLjoxuj1YaFzyUIye6nNX60t0nh3RFXXRQC45ocda9WQa0jzk9ZH8YeIap9hm9ceoS5U-GKkZvkvho9k-sUvbp4UFKs1T4e~7W3AVeMJSnwVVxA653UWJNnOLE7Xs4mobNQCE7HPYIWgFcgs2b6jQ9VUonQNsyOZiPBMclEjJOUBHS324ML-YfgjoiETrEYEdgboK2eSARr5K0PIqqrrWkuOHoIvvJcCf2LGzoGyYW2SAA4tSutOYjQTLG1XgCkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GO category classification. The percentages of number of genes classified into each GO slim category in J. curcas, R. communis, and A. thaliana are, respectively, shown in blue, red, and yellow bars. (A) GO terms; (B) biological process; (C) cellular component; and (D) molecular function." path-from-xml="dsq03002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5500914/dsq03002.jpeg" path-from-xml="dsq03002.jpeg" target="_blank">View large</a><a section="5500914" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq03002.jpeg?Expires=2147483647&Signature=ZFO3ot57COyBcOA-VZbDe-jlJPWGBwePryz09HmhgM8Xiq0aHZ4~lPrCsRZrVSGNEw6Pj7kN1yivvzeYleVs64vcSCXw3DnqdrHlUvW5MVi3gndxefvPCzccVScBMM9b9w6xt9CCIGqHIv-3shQWdf5DHQ28lc0srOu1vPL75rTXMy7mXT4J6F33InxtUAA94rPqhot377Ex5tM8s5KqzBRE0A~fqKOK-nFr9XDA-GYxpwDGKpCKD4SNOOsGX9~uU~vrUnR72Lycj~OdMaTEe7cyv2RBY6l2LRwI-WJM80S~YPsidiAU-70eu-F6TRWKN1hulYonP~nAXQR4yQ3xbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5500914&ar=377199&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq03002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GO category classification. The percentages of number of genes classified into each GO slim category in <em>J. curcas</em>, <em>R. communis</em>, and <em>A. thaliana</em> are, respectively, shown in blue, red, and yellow bars. (A) GO terms; (B) biological process; (C) cellular component; and (D) molecular function.</p></div></div><div content-id="DSQ030F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/m_dsq03002.jpeg?Expires=2147483647&Signature=yFMmeT~wLyaiMEUMzX3Zv~yV~1ir9NN8xkOjUPd3kcPWlXFtLDHiDFHbbEavA6eg5tK8DrdtekUCqUmTr3BP~YRLjoxuj1YaFzyUIye6nNX60t0nh3RFXXRQC45ocda9WQa0jzk9ZH8YeIap9hm9ceoS5U-GKkZvkvho9k-sUvbp4UFKs1T4e~7W3AVeMJSnwVVxA653UWJNnOLE7Xs4mobNQCE7HPYIWgFcgs2b6jQ9VUonQNsyOZiPBMclEjJOUBHS324ML-YfgjoiETrEYEdgboK2eSARr5K0PIqqrrWkuOHoIvvJcCf2LGzoGyYW2SAA4tSutOYjQTLG1XgCkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GO category classification. The percentages of number of genes classified into each GO slim category in J. curcas, R. communis, and A. thaliana are, respectively, shown in blue, red, and yellow bars. (A) GO terms; (B) biological process; (C) cellular component; and (D) molecular function." path-from-xml="dsq03002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5500914/dsq03002.jpeg" path-from-xml="dsq03002.jpeg" target="_blank">View large</a><a section="5500914" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq03002.jpeg?Expires=2147483647&Signature=ZFO3ot57COyBcOA-VZbDe-jlJPWGBwePryz09HmhgM8Xiq0aHZ4~lPrCsRZrVSGNEw6Pj7kN1yivvzeYleVs64vcSCXw3DnqdrHlUvW5MVi3gndxefvPCzccVScBMM9b9w6xt9CCIGqHIv-3shQWdf5DHQ28lc0srOu1vPL75rTXMy7mXT4J6F33InxtUAA94rPqhot377Ex5tM8s5KqzBRE0A~fqKOK-nFr9XDA-GYxpwDGKpCKD4SNOOsGX9~uU~vrUnR72Lycj~OdMaTEe7cyv2RBY6l2LRwI-WJM80S~YPsidiAU-70eu-F6TRWKN1hulYonP~nAXQR4yQ3xbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5500914&ar=377199&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq03002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GO category classification. The percentages of number of genes classified into each GO slim category in <em>J. curcas</em>, <em>R. communis</em>, and <em>A. thaliana</em> are, respectively, shown in blue, red, and yellow bars. (A) GO terms; (B) biological process; (C) cellular component; and (D) molecular function.</p></div></div><p>Of 40 929 presumptive genes in the Jatropha genomic sequences, 2213 genes could be mapped onto 134 of the 155 metabolic pathways in the KEGG database,<sup><a reveal-id="DSQ030C25" data-open="DSQ030C25" class="link link-ref link-reveal xref-bibr">25</a></sup> whereas the 2975 and 4115 genes of <em>R. communis</em> and <em>A. thaliana</em> were mapped onto 140 and 135 pathways, respectively. Twenty-nine pathways, including ‘fatty acid metabolism’ in lipid metabolism, ‘methionine metabolism’ and ‘lysine degradation’ in amino acid metabolism, and ‘benzoate degradation via hydroxylation’ in xenobiotics biodegradation and metabolism, contained enzyme(s) on which the genes in the Jatropha genome were solely mapped (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>).</p>                            <h3 scrollto-destination=5500916 id="5500916" class="section-title" data-legacy-id=s3d>3.4. Characteristic features of the genes in <em>J. curcas</em></h3>
                            <h4 scrollto-destination=5500917 id="5500917" class="section-title" data-legacy-id=s3d1>3.4.1. Genes involved in synthesis of triacylglycerols</h4>
<p><em>Jatropha curcas</em> is expected to contribute to biodiesel production through its ability to biosynthesize and accumulate considerable amounts of triacylglycerols (TAGs) in seeds. For this reason, the genes involved in TAG biosynthesis are of great interest and some of those genes have already been cloned from <em>J. curcas</em>.<sup><a reveal-id="DSQ030C36" data-open="DSQ030C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DSQ030C37" data-open="DSQ030C37" class="link link-ref link-reveal xref-bibr">37</a></sup> Recently, the collection of ESTs from developing and germinating Jatropha seeds has been reported.<sup><a reveal-id="DSQ030C6" data-open="DSQ030C6" class="link link-ref link-reveal xref-bibr">6</a></sup> We manually annotated and summarized the gene models for fatty acid and TAG biosynthesis that were predicted in this work, together with related data that have been deposited to GenBank (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>). The Jatropha genome appears to contain basically one gene for each enzyme isoform, and no obvious gene duplication particular to this plant was identified in this category. One gene model for a recently identified soluble type of DGAT<sup><a reveal-id="DSQ030C38" data-open="DSQ030C38" class="link link-ref link-reveal xref-bibr">38</a></sup> also existed in the Jatropha genome. To improve Jatropha oil quality for biodiesel, its fatty acid composition could be changed by altering the expression of some of the genes listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>.</p>                            <h4 scrollto-destination=5500919 id="5500919" class="section-title" data-legacy-id=s3d2>3.4.2. Genes related to phorbol ester biosynthesis</h4>
<p><em>Jatropha curcas</em> is known to produce tumour-promoting phorbol esters.<sup><a reveal-id="DSQ030C39" data-open="DSQ030C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Accordingly, depression of the phorbol ester biosynthetic gene in high oil content lines would be a step towards safe utilization of this plant. To our knowledge, genes involved in biosynthesis of phorbol esters have not been reported in <em>J. curcas</em>, with the exception of the gene for geranylgeranyl diphosphate synthase (GGPPS).<sup><a reveal-id="DSQ030C40" data-open="DSQ030C40" class="link link-ref link-reveal xref-bibr">40</a></sup> In the current study, we searched genes for GGPPS, casbene synthase (CS), terpene hydroxylase (cytochrome P450-dependent monooxygenase), and acyltransferase in the Jatropha genome with the tBLASTN program<sup><a reveal-id="DSQ030C21" data-open="DSQ030C21" class="link link-ref link-reveal xref-bibr">21</a></sup> using the corresponding amino acid sequences in diterpene-producing plants as queries (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>).</p><p>One (JcCS1), two (JcCS2 and JcCS3), and six (JcCS4–JcCS9) homologues of a gene for CS in <em>R. communis</em> were identified in the BAC clones, JHL23C09, JHL22C18, and JHL17M24, respectively. JcCS2 is a pseudogene because there are several stop codons in the putative open reading frame (ORF). Interestingly, JcCS4–JcCS9 are tandemly aligned and are likely to be active because their ORFs seem to be intact. The phylogenetic tree demonstrates that JcCS4–JcCS9 forms a cluster, suggesting that continuous duplication of the original JcCS gene occurred recently (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2 and Supplementary Table S9</a>). There are 40 genes for terpenoid synthase (AtTPS) in <em>A. thaliana</em> that are most closely related to JcCS phylogenetically.<sup><a reveal-id="DSQ030C41" data-open="DSQ030C41" class="link link-ref link-reveal xref-bibr">41</a></sup> They form clusters consisting of two or three tandem repeats at six loci in the genome. The clustered organization of JcCS may be an implication of the evolutionary process of genes related to the synthesis of terpenoid natural products.</p>                            <h4 scrollto-destination=5500922 id="5500922" class="section-title" data-legacy-id=s3d3>3.4.3. Genes encoding curcin</h4>
<p>Curcin is a Type I ribosome-inactivating protein (RIP) common among the members of the Euphobiaceae family. Curcin in <em>J. curcas</em> is analogous to ricin, a Type II RIP, in <em>R. communis,</em> although the toxicity of curcin is significantly lower than that of ricin.<sup><a reveal-id="DSQ030C42" data-open="DSQ030C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Research on curcin has been extensive,<sup><a reveal-id="DSQ030C42" data-open="DSQ030C42" class="link link-ref link-reveal xref-bibr">42</a></sup> and it has revealed antitumour activity.<sup><a reveal-id="DSQ030C43" data-open="DSQ030C43" class="link link-ref link-reveal xref-bibr">43</a>,<a reveal-id="DSQ030C44" data-open="DSQ030C44" class="link link-ref link-reveal xref-bibr">44</a></sup> The activity of a curcin protein isoform against viral and fungal diseases has been proven by heterologous expression in tobacco; the expression of this curcin gene was induced by abiotic and biotic stresses in leaves.<sup><a reveal-id="DSQ030C45  DSQ030C47" data-open="DSQ030C45  DSQ030C47" class="link link-ref link-reveal xref-bibr">45–47</a></sup> So far, Jatropha genes encoding three isoforms of curcin have been reported and deposited in public DNA databases. In our Jatropha genome sequence, only three contigs were identified to encode amino acid sequences highly similar to those coding for curcin, confirming that the Jatropha genome contains three curcin genes. However, there are four more contigs with presumptive genes predicted to encode curcin-like proteins with <em>E</em>-values from 1e−117 to 1e−91, as listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S10</a>, suggesting that at least two more curcin isoforms are encoded in the Jatropha genome because these four additional genes make two pairs with highly similar counterparts. Data from proteomic analysis of developing seeds that is briefly mentioned in Costa <em>et al.</em><sup><a reveal-id="DSQ030C6" data-open="DSQ030C6" class="link link-ref link-reveal xref-bibr">6</a></sup> appear to support this observation as they identified five isoforms of curcin.</p>                            <h4 scrollto-destination=5500924 id="5500924" class="section-title" data-legacy-id=s3d4>3.4.4. Disease resistance genes</h4>
<p>In response to pathogens, plants have evolved disease resistance (R) genes. Most of them are NBS-LRR (nucleotide-binding site and leucine-rich repeat) proteins, which are classified into two groups on the basis of the presence of Toll and human interleukin receptors (TIR) at their amino termini.<sup><a reveal-id="DSQ030C48" data-open="DSQ030C48" class="link link-ref link-reveal xref-bibr">48</a></sup> We identified 42 TIR NBS-LRR proteins and 50 non-TIR NBS-LRR proteins. We analysed five BAC clones (JHL06P13, JHS03A10, JHL25H03, JHL25P11, and JMS10C05) including R genes to reveal their gene structure (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). Two BAC clones (JHS03A10 and JMS10C05) include singletons of JcTIR-NBS-LRR1 and JcNBS-LRR9, whereas three clones (JHL06P13, JHL25H03, and JHL25P11) contain gene clusters as tandem repeats of R genes, JcNBS-LRR1 and JcNBS-LRR2, JcNBS-LRR3–5, or JcNBS-LRR6–8. JcNBS-LRR8 is a pseudogene with a stop codon in the ORF. The phylogenetic tree of R genes including eight R genes in <em>J. curcas</em> demonstrated that JcNBS-LRR3–5 or JcNBS-LRR6 and JcNBS-LRR7 are closely related, suggesting that these gene clusters evolved recently by the way of gene duplication (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3 and Supplementary Table S11</a>). Interestingly, JcNBS-LRR1 and JcNBS-LRR2 belong to different clades. This relationship indicates that gene duplication was not recent and that these gene segments were conserved after evolutionary diversification of <em>J. curcas</em>.</p>                            <h4 scrollto-destination=5500926 id="5500926" class="section-title" data-legacy-id=s3d5>3.4.5. MADS-box genes</h4>
<p>MADS-box genes, typical homeotic genes coding for transcription factors, form a family and are involved in several aspects of plant development.<sup><a reveal-id="DSQ030C49" data-open="DSQ030C49" class="link link-ref link-reveal xref-bibr">49</a></sup> Many plant species are known to harbour multiple MADS-box genes that belong to a range of functionally divergent subfamilies.<sup><a reveal-id="DSQ030C50" data-open="DSQ030C50" class="link link-ref link-reveal xref-bibr">50</a></sup> We searched for MIKC type II MADS-box genes in the genome of <em>J. curcas</em> using amino acid sequences of PI in <em>A. thaliana</em> as a query. A total of 28 potential MADS-box genes (JcMADS01–JcMADS28) were identified (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S12</a>). The phylogenetic analysis classified these genes into several subfamilies (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>).</p><p>SVP controls flowering time by negatively regulating the expression of a floral integrator, <em>FLOWERING LOCUS T</em> in response to ambient temperature changes in <em>A. thaliana</em>.<sup><a reveal-id="DSQ030C51" data-open="DSQ030C51" class="link link-ref link-reveal xref-bibr">51</a></sup> Interestingly, there are five paralogs of <em>SVP</em> in Jatropha, yet only a single copy and three copies were identified in <em>A. thaliana</em> and <em>Oryza sativa</em>, respectively.<sup><a reveal-id="DSQ030C52" data-open="DSQ030C52" class="link link-ref link-reveal xref-bibr">52</a>,<a reveal-id="DSQ030C53" data-open="DSQ030C53" class="link link-ref link-reveal xref-bibr">53</a></sup> Eight paralogs of <em>SVP</em> copies have been found in 57 MIKC type II MADS-box genes of <em>Populus trichocarta</em>,<sup><a reveal-id="DSQ030C54" data-open="DSQ030C54" class="link link-ref link-reveal xref-bibr">54</a></sup> suggesting amplification and functional diversification of the <em>SVP</em> gene in woody plants.</p>                            <h4 scrollto-destination=5500929 id="5500929" class="section-title" data-legacy-id=s3d6>3.4.6. Flowering-related genes</h4>
<p>Flowering in <em>J. curcas</em> is closely related to the production of seeds. <em>Jatropha curcas</em> is a monoecious species, which forms unisexual flowers, male and female flowers, separately in an individual plant. The unisexual flowers are produced on the same inflorescence, with the ratio of male flowers to female flowers ranging from 10:1 to 30:1.<sup><a reveal-id="DSQ030C55" data-open="DSQ030C55" class="link link-ref link-reveal xref-bibr">55</a></sup> The male bias ratio within an inflorescence limits seed production because more female flowers mean more fruits. Accordingly, modification of floral identity genes involved in organ identity could change the number or size of male and female organs or flowers.</p><p>In the Jatropha genomic sequences, we identified eight orthologs of flowering-related genes including five flowering regulators, <em>CONSTANS, FLOWERING LOCUS D, FLOWERING LOCUS F, LEAFY,</em> and <em>SUPPRESSOR OF OVEREXPRESSION OF CONSTANS 1,</em> designated as <em>JcCO, JcFD, JcFT, JcLFY, and JcSOC1,</em> respectively, and three floral identity genes, <em>APETALA2, APETALA3,</em> and <em>PISTILLATA,</em> designated as <em>JcAP2, JcAP3-1,</em> and <em>JcPI</em>, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S13</a>). Phylogenetic analysis indicated that all Jatropha flowering-related genes except JcCO are closely related to those of woody plants, including <em>Betula pendula</em>, <em>Hevea brasiliensis</em>, <em>R. communis</em>, and <em>Vitis vinifera</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5 and Supplementary Table S14</a>). JcCO belonged to evolutionary lineages that differ from its homologues in monocot and dicot species. Further phylogenetic analysis indicated that JcCO is not related to any flowering-related genes including CO paralogs, a rice CO orthologue Hd1 and light-signalling genes of AtCOLs, which are CO-like genes in <em>A. thaliana</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S6 and Supplementary Table S15</a>). This finding suggests that JcCO is not directly involved in flowering regulation, although JcCO has all CO-conservative domains as a transcription factor including B-box and CCT motif. There were other CO homologues in the Jatropha genome; for example, JcCOL2 in JcCB0217351.10 and JcCOL9 in JcCA0317951.10, which suggests that different components participate in the response to light in <em>J. curcas.</em></p>                            <h3 scrollto-destination=5500932 id="5500932" class="section-title" data-legacy-id=s3e>3.5. Comparative analysis</h3>
                            <h4 scrollto-destination=5500933 id="5500933" class="section-title" data-legacy-id=s3e1>3.5.1. Genes conserved in the Euphorbiaceae</h4>
<p>To identify genes conserved specifically in the family Euphorbiaceae, amino acid sequences translated from the putative Jatropha genes predicted in this study were compared with those of genes in the genomes of <em>A. thaliana</em>, <em>O. sativa</em>, <em>P. trichocarpa, V. vinifera</em>, <em>L. japonicus</em>, and <em>Glycine max,</em> as well as protein sequences in the TrEMBL protein database.<sup><a reveal-id="DSQ030C33" data-open="DSQ030C33" class="link link-ref link-reveal xref-bibr">33</a></sup> Sequences from the predicted genes in the <em>R. communis</em> genome<sup><a reveal-id="DSQ030C34" data-open="DSQ030C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and the gene index database for cassava (<em>Manihot esculenta</em>) were used as references for Euphorbiaceae protein-encoding genes. BLAST searches with a cut-off (<em>E</em>-value ≤ 1e−20) indicated that 1529 genes (4% of the predicted protein-encoding genes) were found only in the Euphorbiaceae. The InterPro annotations of these Euphorbiaceae-specific genes were surveyed to find conserved motifs in these genes, and consequently, 22 InterPro motifs were likely to be conserved in five or more genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S16</a>). Of these, the C1-like motif (IPR011424), the pentatricopeptide repeat motif (IPR002885), and the cytochrome P450 motif (IPR001128) were found in 10, 10, and 9 genes, respectively.</p><p>Furthermore, 1176 of the genes predicted in the Jatropha genome assembly had matching sequences only in the Jatropha cDNA database suggesting that these genes are specific to <em>J. curcas</em>. The most common InterPro motifs found in these genes were the protein kinase-like domain (IPR011009) detected in six genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S17</a>). The entire list of the Euphorbiaceae- and Jatropha-specific genes is provided in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S18 and S19</a>, respectively.</p>                            <h4 scrollto-destination=5500936 id="5500936" class="section-title" data-legacy-id=s3e2>3.5.2. Microsynteny</h4>
<p>To investigate the syntenic relations between the Jatropha and the other plant genomes, status of conservation of relative gene positions was surveyed using the scaffolds of Jatropha genomic sequences. Among the 1556 scaffolds with five or more predicted genes, conservation of the relative positions of three or more genes was observed in 829 scaffolds (53%) against genes predicted in the <em>R. communis</em> genomic sequences<sup><a reveal-id="DSQ030C34" data-open="DSQ030C34" class="link link-ref link-reveal xref-bibr">34</a></sup> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S20 and S21</a>). It appears that a significant degree of synteny can be expected within the family Euphorbiaceae. A syntenic relationship was also detected against the genomes of <em>G. max</em> and <em>A. thaliana</em> to a lesser degree. Microsyntenic relations have been observed in 178 (11%) and 256 (16%) of the 1556 scaffolds of the Jatropha genomic sequences, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S20 and S21</a>). The microsyntenic relationships between these plant species may provide useful information for predicting gene organization in the ancestral genome of dicots.</p>                            <h4 scrollto-destination=5500938 id="5500938" class="section-title" data-legacy-id=s3e3>3.5.3. Genetic diversity among Jatropha lines</h4>
<p>Five SSR motives were found in the 100 genome-derived microsatellite markers tested. Most of the SSRs were poly (AT)<sub>n</sub> (83 SSRs), followed by poly (AAT)<sub>n</sub> (8 SSRs), poly (AG)<sub>n</sub> (5 SSRs), poly (AAG)<sub>n</sub> (3 SSRs), and poly (AC)<sub>n</sub> (1 SSR). A total of 88 markers generated specific amplicons, whereas the other eight and four markers showed no amplification and non-specific amplification, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S7</a>). The small number of markers detecting non-specific amplification suggested less redundancy of SSR regions in the Jatropha genome. The number of alleles per locus ranged from one to four with a mean value of 1.31. Markers showed no polymorphisms; those detecting a single allele were most frequent. PIC values ranged from 0 to 0.45 with a mean value of 0.06 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S8</a>). The large number of markers detecting no polymorphisms and the low mean value of the PIC indicated that genetic diversity in Jatropha lines is generally narrow. An UPGMA genetic tree of the 12 lines of <em>J. curcas</em> illustrated that the three lines derived from meso-America regions (Guatemala1, Guatemala2, and Mexico2b) are genetically distinct from the other lines derived from Asia and Africa, whereas no significant difference was observed between the Asian and African lines (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S9</a>).</p>                            <h2 scrollto-destination=5500940 id="5500940" class="section-title" data-legacy-id=s4>4. Databases</h2>
<p>Information about the genomic sequences (contigs and singlets) and BAC clone sequences is available through international databases (DDBJ/GenBank/EMBL) under accession numbers BABX01000001–BABX01150417 (150 417 entries) and AP011961–AP011977 (17 entries), respectively. Single reads of cDNA by GS FLX sequencer derived from leaf and callus tissue are available through DDBJ Sequence Read Archive under accession numbers DRA000303 and DRA000304, respectively. Paired-end reads of genome by GAIIx sequencer with 36 bp long, and 50 and 31 bp long are available through DDBJ Sequence Read Archive under accession numbers DRA000305 and DRA000306, respectively. An online database that provides the nucleotide sequences and the predicted genes is available at <a class="link link-uri" href="http://www.kazusa.or.jp/jatropha/">http://www.kazusa.or.jp/jatropha/</a>.</p>                            <h2 scrollto-destination=5500942 id="5500942" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5500944 id="5500944" class="section-title" data-legacy-id=s6>Funding</h2>
<p>We thank the Sumitomo Electric Industries, Ltd. for their philanthropic donation of funds to aid the Jatropha genome project celebrating the 110th anniversary of their foundation. This work was also supported by the Kazusa DNA Research Institute Foundation.</p>                            <h2 scrollto-destination=5500946 id="5500946" class="backacknowledgements-title" >Acknowledgements</h2>
<p>Special thanks are extended to Profs. Kazuyoshi Itoh and Yasuo Kanematsu of the Graduate School of Engineering, Osaka University, for their guidance and support during the genome project.</p>                            <h2 scrollto-destination=5500948 id="5500948" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSQ030C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Openshaw</div> <div class="given-names">K.</div></div>. </div><div class="article-title">A review of <em>J. curcas</em>: an oil plant of unfulfilled promise</div>, <div class="source">Biomass Bioenergy</div> , <div class="year">2000</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20review%20of%20J.%20curcas%3A%20an%20oil%20plant%20of%20unfulfilled%20promise&amp;author=K.%20Openshaw&amp;publication_year=2000&amp;journal=Biomass%20Bioenergy&amp;volume=19&amp;pages=1-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0961-9534(00)00019-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0961-9534(00)00019-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0961-9534(00)00019-2"> </span></div> </div></div></div></div></div><div content-id="DSQ030C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wouter</div> <div class="given-names">H.M.</div></div>,  <div class="name"><div class="surname">Wouter</div> <div class="given-names">M.J.A.</div></div>,  <div class="name"><div class="surname">Bart</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Use of inadequate data and methodological errors lead to a dramatic overestimation of the water footprint of <em>Jatropha curcas</em></div>, <div class="year">2009</div><div class="publisher-name">Nature Precedings</div> <div class="comment">hdl:10101/npre.2009.3410.1 <a class="link link-uri" href="http://precedings.nature.com/documents/3410/version/1">http://precedings.nature.com/documents/3410/version/1</a></div></div></div></div></div><div content-id="DSQ030C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fairless</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Biofuel: the little shrub that could—maybe</div>, <div class="source">Nature</div> , <div class="year">2007</div>, vol. <div class="volume">449</div> (pg. <div class="fpage">652</div>-<div class="lpage">655</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biofuel%3A%20the%20little%20shrub%20that%20could%E2%80%94maybe&amp;author=D.%20Fairless&amp;publication_year=2007&amp;journal=Nature&amp;volume=449&amp;pages=652-655" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/449652a" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F449652a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F449652a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17968401" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Biello</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Green fuels for jets</div>, <div class="source">Sci. Am.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">68</div>-<div class="lpage">69</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Green%20fuels%20for%20jets&amp;author=D.%20Biello&amp;publication_year=2009&amp;journal=Sci.%20Am.&amp;volume=19&amp;pages=68-69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/scientificamericanearth0309-68" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fscientificamericanearth0309-68" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fscientificamericanearth0309-68"> </span></div> </div></div></div></div></div><div content-id="DSQ030C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carvalhoa</div> <div class="given-names">C.R.</div></div>,  <div class="name"><div class="surname">Clarindoa</div> <div class="given-names">W.R.</div></div>,  <div class="name"><div class="surname">Praça</div> <div class="given-names">M.M.</div></div>,  <div class="name"><div class="surname">Araújoa</div> <div class="given-names">F.S.</div></div>,  <div class="name"><div class="surname">Carels</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Genome size, base composition and karyotype of <em>Jatropha curcas</em> L., an important biofuel plant</div>, <div class="source">Plant Sci.</div> , <div class="year">2008</div>, vol. <div class="volume">174</div> (pg. <div class="fpage">613</div>-<div class="lpage">617</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20size%2C%20base%20composition%20and%20karyotype%20of%20Jatropha%20curcas%20L.%2C%20an%20important%20biofuel%20plant&amp;author=C.R.%20Carvalhoa&amp;author=W.R.%20Clarindoa&amp;author=M.M.%20Pra%C3%A7a&amp;author=F.S.%20Ara%C3%BAjoa&amp;author=N.%20Carels&amp;publication_year=2008&amp;journal=Plant%20Sci.&amp;volume=174&amp;pages=613-617" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.plantsci.2008.03.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plantsci.2008.03.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.plantsci.2008.03.010"> </span></div> </div></div></div></div></div><div content-id="DSQ030C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Costa</div> <div class="given-names">G.G.</div></div>,  <div class="name"><div class="surname">Cardoso</div> <div class="given-names">K.C.</div></div>,  <div class="name"><div class="surname">Del Bem</div> <div class="given-names">L.E.</div></div>, et al. </div><div class="article-title">Transcriptome analysis of the oil-rich seed of the bioenergy crop <em>Jatropha curcas</em> L.</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">462</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20analysis%20of%20the%20oil-rich%20seed%20of%20the%20bioenergy%20crop%20Jatropha%20curcas%20L.&amp;author=G.G.%20Costa&amp;author=K.C.%20Cardoso&amp;author=L.E.%20Del%20Bem&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=462" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-462" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-462" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-462"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20691070" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kaneko</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Genome structure of the legume, <em>Lotus japonicus</em></div>, <div class="source">DNA Res.</div> , <div class="year">2008</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">227</div>-<div class="lpage">239</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20structure%20of%20the%20legume%2C%20Lotus%20japonicus&amp;author=S.%20Sato&amp;author=Y.%20Nakamura&amp;author=T.%20Kaneko&amp;publication_year=2008&amp;journal=DNA%20Res.&amp;volume=15&amp;pages=227-239" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18511435" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">White</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Roberts</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yorke</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Pop</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Figaro: a novel statistical method for vector sequence removal</div>, <div class="source">Bioinformatics</div> , <div class="year">2008</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">462</div>-<div class="lpage">467</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Figaro%3A%20a%20novel%20statistical%20method%20for%20vector%20sequence%20removal&amp;author=J.R.%20White&amp;author=M.%20Roberts&amp;author=J.A.%20Yorke&amp;author=M.%20Pop&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=462-467" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm632" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm632" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm632"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18202027" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">S.P.</div></div>,  <div class="name"><div class="surname">Chinwalla</div> <div class="given-names">A.T.</div></div>, et al. </div><div class="article-title">Application of a superword array in genome assembly</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">201</div>-<div class="lpage">205</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Application%20of%20a%20superword%20array%20in%20genome%20assembly&amp;author=X.%20Huang&amp;author=S.P.%20Yang&amp;author=A.T.%20Chinwalla&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=201-205" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj419" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj419" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj419"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16397298" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Quinlan</div> <div class="given-names">A.R.</div></div>,  <div class="name"><div class="surname">Stewart</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Strömberg</div> <div class="given-names">M.P.</div></div>,  <div class="name"><div class="surname">Marth</div> <div class="given-names">G.T.</div></div>. </div><div class="article-title">Pyrobayes: an improved base caller for SNP discovery in pyrosequences</div>, <div class="source">Nat. Methods</div> , <div class="year">2008</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">179</div>-<div class="lpage">181</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pyrobayes%3A%20an%20improved%20base%20caller%20for%20SNP%20discovery%20in%20pyrosequences&amp;author=A.R.%20Quinlan&amp;author=D.A.%20Stewart&amp;author=M.P.%20Str%C3%B6mberg&amp;author=G.T.%20Marth&amp;publication_year=2008&amp;journal=Nat.%20Methods&amp;volume=5&amp;pages=179-181" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1172" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1172" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1172"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18193056" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gomez-Alvarez</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Teal</div> <div class="given-names">T.K.</div></div>,  <div class="name"><div class="surname">Schmidt</div> <div class="given-names">T.M.</div></div>. </div><div class="article-title">Systematic artifacts in metagenomes from complex microbial communities</div>, <div class="source">ISME J.</div> , <div class="year">2009</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">1314</div>-<div class="lpage">1317</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20artifacts%20in%20metagenomes%20from%20complex%20microbial%20communities&amp;author=V.%20Gomez-Alvarez&amp;author=T.K.%20Teal&amp;author=T.M.%20Schmidt&amp;publication_year=2009&amp;journal=ISME%20J.&amp;volume=3&amp;pages=1314-1317" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2009.72" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2009.72" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2009.72"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19587772" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chevreux</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Wetter</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Suhai</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Genome sequence assembly using trace signals and additional sequence information</div>, <div class="source">Computer Science and Biology: Proceedings of the German Conference on Bioinformatics (GCB)</div> , <div class="year">1999</div>(pg. <div class="fpage">45</div>-<div class="lpage">56</div>) <div class="comment">99</div></div></div></div></div><div content-id="DSQ030C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Unseld</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Marienfeld</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Brandt</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Brennicke</div> <div class="given-names">A.</div></div>. </div><div class="article-title">The mitochondrial genome of <em>Arabidopsis thaliana</em> contains 57 genes in 366,924 nucleotides</div>, <div class="source">Nat. Genet.</div> , <div class="year">1997</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">57</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20mitochondrial%20genome%20of%20Arabidopsis%20thaliana%20contains%2057%20genes%20in%20366%2C924%20nucleotides&amp;author=M.%20Unseld&amp;author=J.R.%20Marienfeld&amp;author=P.%20Brandt&amp;author=A.%20Brennicke&amp;publication_year=1997&amp;journal=Nat.%20Genet.&amp;volume=15&amp;pages=57-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng0197-57" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng0197-57" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng0197-57"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8988169" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Schwartz</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Wagner</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Miller</div> <div class="given-names">W.A.</div></div>. </div><div class="article-title">Greedy algorithm for aligning DNA sequences</div>, <div class="source">J. Comput. Biol.</div> , <div class="year">2000</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">203</div>-<div class="lpage">214</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Greedy%20algorithm%20for%20aligning%20DNA%20sequences&amp;author=Z.%20Zhang&amp;author=S.%20Schwartz&amp;author=L.%20Wagner&amp;author=W.A.%20Miller&amp;publication_year=2000&amp;journal=J.%20Comput.%20Biol.&amp;volume=7&amp;pages=203-214" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/10665270050081478" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2F10665270050081478" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2F10665270050081478"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10890397" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zerbino</div> <div class="given-names">D.R.</div></div>,  <div class="name"><div class="surname">Birney</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Velvet: algorithms for <em>de novo</em> short read assembly using de Bruijn graphs</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">821</div>-<div class="lpage">829</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Velvet%3A%20algorithms%20for%20de%20novo%20short%20read%20assembly%20using%20de%20Bruijn%20graphs&amp;author=D.R.%20Zerbino&amp;author=E.%20Birney&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=821-829" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.074492.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.074492.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.074492.107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18349386" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kent</div> <div class="given-names">W.J.</div></div>. </div><div class="article-title">BLAT—the BLAST-like alignment tool</div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">656</div>-<div class="lpage">664</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLAT%E2%80%94the%20BLAST-like%20alignment%20tool&amp;author=W.J.%20Kent&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=656-664" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.229202. Article published online before March 2002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.229202. Article published online before March 2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.229202. Article published online before March 2002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11932250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lukashin</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div>. </div><div class="article-title">GeneMark.hmm: new solutions for gene finding</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1998</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">1107</div>-<div class="lpage">1115</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GeneMark.hmm%3A%20new%20solutions%20for%20gene%20finding&amp;author=A.%20Lukashin&amp;author=M.%20Borodovsky&amp;publication_year=1998&amp;journal=Nucleic%20Acids%20Res.&amp;volume=26&amp;pages=1107-1115" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.4.1107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.4.1107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.4.1107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9461475" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Burge</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Karlin</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Prediction of complete gene structures in human genomic DNA</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1997</div>, vol. <div class="volume">268</div> (pg. <div class="fpage">78</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20complete%20gene%20structures%20in%20human%20genomic%20DNA&amp;author=C.%20Burge&amp;author=S.%20Karlin&amp;publication_year=1997&amp;journal=J.%20Mol.%20Biol.&amp;volume=268&amp;pages=78-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1997.0951" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1997.0951" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1997.0951"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9149143" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hebsgaard</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Korning</div> <div class="given-names">P.G.</div></div>,  <div class="name"><div class="surname">Tolstrup</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Engelbrecht</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Rouzé</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Brunak</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Splice site prediction in <em>Arabidopsis thaliana</em> pre-mRNA by combining local and global sequence information</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1996</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">3439</div>-<div class="lpage">3452</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Splice%20site%20prediction%20in%20Arabidopsis%20thaliana%20pre-mRNA%20by%20combining%20local%20and%20global%20sequence%20information&amp;author=S.M.%20Hebsgaard&amp;author=P.G.%20Korning&amp;author=N.%20Tolstrup&amp;author=J.%20Engelbrecht&amp;author=P.%20Rouz%C3%A9&amp;author=S.%20Brunak&amp;publication_year=1996&amp;journal=Nucleic%20Acids%20Res.&amp;volume=24&amp;pages=3439-3452" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/24.17.3439" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F24.17.3439" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F24.17.3439"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8811101" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brendel</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Kleffe</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Prediction of locally optimal splice sites in plant pre-mRNA with applications to gene identification in <em>Arabidopsis thaliana</em> genomic DNA</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1998</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">4748</div>-<div class="lpage">4757</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20locally%20optimal%20splice%20sites%20in%20plant%20pre-mRNA%20with%20applications%20to%20gene%20identification%20in%20Arabidopsis%20thaliana%20genomic%20DNA&amp;author=V.%20Brendel&amp;author=J.%20Kleffe&amp;publication_year=1998&amp;journal=Nucleic%20Acids%20Res.&amp;volume=26&amp;pages=4748-4757" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.20.4748" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.20.4748" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.20.4748"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9753745" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Schäffer</div> <div class="given-names">A.A.</div></div>. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">3402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Sch%C3%A4ffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-3402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Methods for comparing a DNA sequence with a protein sequence</div>, <div class="source">Comput. Appl. Biosci.</div> , <div class="year">1996</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">497</div>-<div class="lpage">506</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Methods%20for%20comparing%20a%20DNA%20sequence%20with%20a%20protein%20sequence&amp;author=X.%20Huang&amp;author=J.%20Zhang&amp;publication_year=1996&amp;journal=Comput.%20Appl.%20Biosci.&amp;volume=12&amp;pages=497-506" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9021268" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ashburner</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ball</div> <div class="given-names">C.A.</div></div>,  <div class="name"><div class="surname">Blake</div> <div class="given-names">J.A.</div></div>, et al. </div><div class="article-title">Gene ontology: tool for the unification of biology. The Gene Ontology Consortium</div>, <div class="source">Nat. Genet.</div> , <div class="year">2000</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">25</div>-<div class="lpage">29</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20ontology%3A%20tool%20for%20the%20unification%20of%20biology.%20The%20Gene%20Ontology%20Consortium&amp;author=M.%20Ashburner&amp;author=C.A.%20Ball&amp;author=J.A.%20Blake&amp;publication_year=2000&amp;journal=Nat.%20Genet.&amp;volume=25&amp;pages=25-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/75556" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F75556" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F75556"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10802651" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hunter</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Apweiler</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Attwood</div> <div class="given-names">T.K.</div></div>, et al. </div><div class="article-title">InterPro: the integrative protein signature database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">D211</div>-<div class="lpage">215</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterPro%3A%20the%20integrative%20protein%20signature%20database&amp;author=S.%20Hunter&amp;author=R.%20Apweiler&amp;author=T.K.%20Attwood&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=D211-215" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn785" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn785" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn785"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18940856" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ogata</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Goto</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Fujibuchi</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Bono</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div>. </div><div class="article-title">KEGG: Kyoto encyclopedia of genes and genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">29</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KEGG%3A%20Kyoto%20encyclopedia%20of%20genes%20and%20genomes&amp;author=H.%20Ogata&amp;author=S.%20Goto&amp;author=K.%20Sato&amp;author=W.%20Fujibuchi&amp;author=H.%20Bono&amp;author=M.%20Kanehisa&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=29-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.1.29" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.1.29" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.1.29"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9847135" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">D.G.</div></div>,  <div class="name"><div class="surname">Gibson</div> <div class="given-names">T.J.</div></div>. </div><div class="article-title">CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1994</div>, vol. <div class="volume">122</div> (pg. <div class="fpage">4673</div>-<div class="lpage">4680</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CLUSTAL%20W%3A%20improving%20the%20sensitivity%20of%20progressive%20multiple%20sequence%20alignment%20through%20sequence%20weighting%2C%20position-specific%20gap%20penalties%20and%20weight%20matrix%20choice&amp;author=J.D.%20Thompson&amp;author=D.G.%20Higgins&amp;author=T.J.%20Gibson&amp;publication_year=1994&amp;journal=Nucleic%20Acids%20Res.&amp;volume=122&amp;pages=4673-4680" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F22.22.4673" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F22.22.4673"> </span></div> </div></div></div></div></div><div content-id="DSQ030C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saitou</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>. </div><div class="article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1987</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">406</div>-<div class="lpage">425</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20neighbor-joining%20method%3A%20a%20new%20method%20for%20reconstructing%20phylogenetic%20trees&amp;author=N.%20Saitou&amp;author=M.%20Nei&amp;publication_year=1987&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=4&amp;pages=406-425" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3447015" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Dudley</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>. </div><div class="article-title">MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">1596</div>-<div class="lpage">1599</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA4%3A%20Molecular%20Evolutionary%20Genetics%20Analysis%20(MEGA)%20software%20version%204.0&amp;author=K.%20Tamura&amp;author=J.%20Dudley&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=1596-1599" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm092" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm092" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm092"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17488738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Temnykh</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">DeClerck</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Lukashova</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lipovich</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Cartinhour</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">McCouch</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Computational and experimental analysis of microsatellites in rice (<em>Oryza sativa</em> L.): frequency, length variation, transposon associations, and genetic marker potential</div>, <div class="source">Genome Res.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">1441</div>-<div class="lpage">1452</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20and%20experimental%20analysis%20of%20microsatellites%20in%20rice%20(Oryza%20sativa%20L.)%3A%20frequency%2C%20length%20variation%2C%20transposon%20associations%2C%20and%20genetic%20marker%20potential&amp;author=S.%20Temnykh&amp;author=G.%20DeClerck&amp;author=A.%20Lukashova&amp;author=L.%20Lipovich&amp;author=S.%20Cartinhour&amp;author=S.%20McCouch&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=1441-1452" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.184001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.184001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.184001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11483586" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Steve</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Helen</div> <div class="given-names">J.S.</div></div>. </div><div class="article-title">Primer3 on the WWW for general users and for biologist programmers</div>, <div class="source">Methods Mol. Biol.</div> , <div class="year">2000</div>, vol. <div class="volume">132</div> (pg. <div class="fpage">365</div>-<div class="lpage">386</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Primer3%20on%20the%20WWW%20for%20general%20users%20and%20for%20biologist%20programmers&amp;author=R.%20Steve&amp;author=J.S.%20Helen&amp;publication_year=2000&amp;journal=Methods%20Mol.%20Biol.&amp;volume=132&amp;pages=365-386" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10547847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bao</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div>. </div><div class="article-title">Automated <em>de novo</em> identification of repeat sequence families in sequenced genomes</div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">1269</div>-<div class="lpage">1276</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20de%20novo%20identification%20of%20repeat%20sequence%20families%20in%20sequenced%20genomes&amp;author=Z.%20Bao&amp;author=S.R.%20Eddy&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=1269-1276" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.88502" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.88502" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.88502"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12176934" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">B.B.</div></div>,  <div class="name"><div class="surname">Brendel</div> <div class="given-names">V.</div></div>. </div><div class="article-title">The ASRG database: identification and survey of <em>Arabidopsis thaliana</em> genes involved in pre-mRNA splicing</div>, <div class="source">Genome Biol.</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> pg. <div class="fpage">R102</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20ASRG%20database%3A%20identification%20and%20survey%20of%20Arabidopsis%20thaliana%20genes%20involved%20in%20pre-mRNA%20splicing&amp;author=B.B.%20Wang&amp;author=V.%20Brendel&amp;publication_year=2004&amp;journal=Genome%20Biol.&amp;volume=5&amp;pages=R102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2004-5-12-r102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2004-5-12-r102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2004-5-12-r102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15575968" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bairoch</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Apweiler</div> <div class="given-names">R.</div></div>. </div><div class="article-title">The SWISS-PROT protein sequence data bank and its new supplement TREMBL</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1996</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">21</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20SWISS-PROT%20protein%20sequence%20data%20bank%20and%20its%20new%20supplement%20TREMBL&amp;author=A.%20Bairoch&amp;author=R.%20Apweiler&amp;publication_year=1996&amp;journal=Nucleic%20Acids%20Res.&amp;volume=24&amp;pages=21-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/24.1.21" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F24.1.21" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F24.1.21"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8594581" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chan</div> <div class="given-names">A.P.</div></div>,  <div class="name"><div class="surname">Crabtree</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">Q.</div></div>, et al. </div><div class="article-title">Draft genome sequence of the oilseed species <em>Ricinus communis</em></div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2010</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">951</div>-<div class="lpage">956</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20the%20oilseed%20species%20Ricinus%20communis&amp;author=A.P.%20Chan&amp;author=J.%20Crabtree&amp;author=Q.%20Zhao&amp;publication_year=2010&amp;journal=Nat.%20Biotechnol.&amp;volume=28&amp;pages=951-956" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1674" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1674" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1674"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20729833" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carbon</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ireland</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Mungall</div> <div class="given-names">C.J.</div></div>,  <div class="name"><div class="surname">Shu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Marshall</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Lewis</div> <div class="given-names">S.</div></div>. </div><div class="article-title">AmiGO: online access to ontology and annotation data</div>, <div class="source">Bioinformatics</div> , <div class="year">2009</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">288</div>-<div class="lpage">289</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AmiGO%3A%20online%20access%20to%20ontology%20and%20annotation%20data&amp;author=S.%20Carbon&amp;author=A.%20Ireland&amp;author=C.J.%20Mungall&amp;author=S.%20Shu&amp;author=B.%20Marshall&amp;author=S.%20Lewis&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=15&amp;pages=288-289" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn615" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn615" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn615"> </span></div> </div></div></div></div></div><div content-id="DSQ030C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tong</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Shu-Ming</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Wu-Yuan</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Characterization of a new stearoyl-acyl carrier protein desaturase gene from <em>Jatropha curcas</em></div>, <div class="source">Biotechnol. Lett.</div> , <div class="year">2006</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">657</div>-<div class="lpage">662</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20a%20new%20stearoyl-acyl%20carrier%20protein%20desaturase%20gene%20from%20Jatropha%20curcas&amp;author=L.%20Tong&amp;author=P.%20Shu-Ming&amp;author=D.%20Wu-Yuan&amp;publication_year=2006&amp;journal=Biotechnol.%20Lett.&amp;volume=28&amp;pages=657-662" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10529-006-0034-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10529-006-0034-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10529-006-0034-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16642305" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ye</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Qu</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bui</div> <div class="given-names">H.T.N.</div></div>,  <div class="name"><div class="surname">Chua</div> <div class="given-names">N.H.</div></div>. </div><div class="article-title">Rapid analysis of <em>Jatropha curcas</em> gene functions by virus-induced gene silencing</div>, <div class="source">Plant Biotechnol. J.</div> , <div class="year">2009</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">964</div>-<div class="lpage">976</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20analysis%20of%20Jatropha%20curcas%20gene%20functions%20by%20virus-induced%20gene%20silencing&amp;author=J.%20Ye&amp;author=J.%20Qu&amp;author=H.T.N.%20Bui&amp;author=N.H.%20Chua&amp;publication_year=2009&amp;journal=Plant%20Biotechnol.%20J.&amp;volume=7&amp;pages=964-976" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pbi.2009.7.issue-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpbi.2009.7.issue-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpbi.2009.7.issue-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19906247" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saha</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Enugutti</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Rajakumari</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Rajasekharan</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Cytosolic triacylglycerol biosynthetic pathway in oilseeds. Molecular cloning and expression of peanut cytosolic diacylglycerol acyltransferase</div>, <div class="source">Plant Physiol.</div> , <div class="year">2006</div>, vol. <div class="volume">141</div> (pg. <div class="fpage">1533</div>-<div class="lpage">1543</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytosolic%20triacylglycerol%20biosynthetic%20pathway%20in%20oilseeds.%20Molecular%20cloning%20and%20expression%20of%20peanut%20cytosolic%20diacylglycerol%20acyltransferase&amp;author=S.%20Saha&amp;author=B.%20Enugutti&amp;author=S.%20Rajakumari&amp;author=R.%20Rajasekharan&amp;publication_year=2006&amp;journal=Plant%20Physiol.&amp;volume=141&amp;pages=1533-1543" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.106.082198" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.106.082198" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.106.082198"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16798944" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Haas</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Sterk</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Mittelbach</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Novel 12-deoxy-16-hydroxy phorbol diesters isolated from the seed oil of <em>J. curcas</em></div>, <div class="source">J. Nat. Prod.</div> , <div class="year">2002</div>, vol. <div class="volume">65</div> (pg. <div class="fpage">1434</div>-<div class="lpage">1440</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%2012-deoxy-16-hydroxy%20phorbol%20diesters%20isolated%20from%20the%20seed%20oil%20of%20J.%20curcas&amp;author=W.%20Haas&amp;author=H.%20Sterk&amp;author=M.%20Mittelbach&amp;publication_year=2002&amp;journal=J.%20Nat.%20Prod.&amp;volume=65&amp;pages=1434-1440" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/np020060d" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fnp020060d" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fnp020060d"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12398539" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lin</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Jin</div> <div class="given-names">Y.J.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.Y.</div></div>. </div><div class="article-title">Molecular cloning and functional analysis of the gene encoding geranylgeranyl diphosphate synthase from <em>J. curcas</em></div>, <div class="source">Afr. J. Biotechnol.</div> , <div class="year">2010</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">3342</div>-<div class="lpage">3351</div>)</div></div></div></div><div content-id="DSQ030C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aubourg</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Lecharny</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bohlmann</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Genomic analysis of the terpenoid synthase (AtTPS) gene family of <em>Arabidopsis thaliana</em></div>, <div class="source">Mol. Genet. Genomics</div> , <div class="year">2002</div>, vol. <div class="volume">267</div> (pg. <div class="fpage">730</div>-<div class="lpage">745</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20analysis%20of%20the%20terpenoid%20synthase%20(AtTPS)%20gene%20family%20of%20Arabidopsis%20thaliana&amp;author=S.%20Aubourg&amp;author=A.%20Lecharny&amp;author=J.%20Bohlmann&amp;publication_year=2002&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=267&amp;pages=730-745" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-002-0709-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-002-0709-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-002-0709-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12207221" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stirpe</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Pession-Brizzi</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lorenzoni</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Strocchi</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Montanaro</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Sperti</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Studies on the proteins from the seeds of <em>Croton tiglium</em> and of <em>Jatropha curcas</em>. Toxic properties and inhibition of protein synthesis <em>in vitro</em></div>, <div class="source">Biochem. J.</div> , <div class="year">1976</div>, vol. <div class="volume">156</div> (pg. <div class="fpage">1</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Studies%20on%20the%20proteins%20from%20the%20seeds%20of%20Croton%20tiglium%20and%20of%20Jatropha%20curcas.%20Toxic%20properties%20and%20inhibition%20of%20protein%20synthesis%20in%20vitro&amp;author=F.%20Stirpe&amp;author=A.%20Pession-Brizzi&amp;author=E.%20Lorenzoni&amp;author=P.%20Strocchi&amp;author=L.%20Montanaro&amp;author=S.%20Sperti&amp;publication_year=1976&amp;journal=Biochem.%20J.&amp;volume=156&amp;pages=1-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/bj1560001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2Fbj1560001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2Fbj1560001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/942394" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Luo</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">X.Y.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">W.X.</div></div>, et al. </div><div class="article-title">Expression, purification and anti-tumor activity of curcin</div>, <div class="source">Acta Biochim. Biophys. Sin.</div> , <div class="year">2006</div>, vol. <div class="volume">38</div> (pg. <div class="fpage">663</div>-<div class="lpage">668</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%2C%20purification%20and%20anti-tumor%20activity%20of%20curcin&amp;author=M.J.%20Luo&amp;author=X.Y.%20Yang&amp;author=W.X.%20Liu&amp;publication_year=2006&amp;journal=Acta%20Biochim.%20Biophys.%20Sin.&amp;volume=38&amp;pages=663-668" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/abbs.2006.38.issue-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fabbs.2006.38.issue-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fabbs.2006.38.issue-9"> </span></div> </div></div></div></div></div><div content-id="DSQ030C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lin</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Yan</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Tang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Antitumor effects of curcin from seeds of <em>Jatropha curcas</em></div>, <div class="source">Acta Pharmacol. Sin.</div> , <div class="year">2003</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">241</div>-<div class="lpage">246</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Antitumor%20effects%20of%20curcin%20from%20seeds%20of%20Jatropha%20curcas&amp;author=J.%20Lin&amp;author=F.%20Yan&amp;author=L.%20Tang&amp;author=F.%20Chen&amp;publication_year=2003&amp;journal=Acta%20Pharmacol.%20Sin.&amp;volume=24&amp;pages=241-246" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12617773" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qin</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Zheng</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Shao</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Stress-induced curcin-L promoter in leaves of <em>Jatropha curcas</em> L. and characterization in transgenic tobacco</div>, <div class="source">Planta</div> , <div class="year">2009</div>, vol. <div class="volume">230</div> (pg. <div class="fpage">387</div>-<div class="lpage">395</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stress-induced%20curcin-L%20promoter%20in%20leaves%20of%20Jatropha%20curcas%20L.%20and%20characterization%20in%20transgenic%20tobacco&amp;author=X.%20Qin&amp;author=X.%20Zheng&amp;author=C.%20Shao&amp;publication_year=2009&amp;journal=Planta&amp;volume=230&amp;pages=387-395" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00425-009-0956-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00425-009-0956-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00425-009-0956-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19475419" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qin</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Ming-Xing</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ying</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Xin-Shen</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Fang</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Expression of a ribosome inactivating protein (curcin 2) in <em>Jatropha curcas</em> is induced by stress</div>, <div class="source">J. Biosci.</div> , <div class="year">2005</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">351</div>-<div class="lpage">357</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20of%20a%20ribosome%20inactivating%20protein%20(curcin%202)%20in%20Jatropha%20curcas%20is%20induced%20by%20stress&amp;author=W.%20Qin&amp;author=H.%20Ming-Xing&amp;author=X.%20Ying&amp;author=Z.%20Xin-Shen&amp;author=C.%20Fang&amp;publication_year=2005&amp;journal=J.%20Biosci.&amp;volume=30&amp;pages=351-357" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02703672" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02703672" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02703672"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16052073" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">M.-X.</div></div>,  <div class="name"><div class="surname">Hou</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Wei</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Xu</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">F.</div></div>. </div><div class="article-title">A ribosome-inactivating protein (curcin 2) induced from <em>Jatropha curcas</em> can reduce viral and fungal infection in transgenic tobacco</div>, <div class="source">Plant Growth Regul.</div> , <div class="year">2008</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">115</div>-<div class="lpage">123</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20ribosome-inactivating%20protein%20(curcin%202)%20induced%20from%20Jatropha%20curcas%20can%20reduce%20viral%20and%20fungal%20infection%20in%20transgenic%20tobacco&amp;author=M.-X.%20Huang&amp;author=P.%20Hou&amp;author=Q.%20Wei&amp;author=Y.%20Xu&amp;author=F.%20Chen&amp;publication_year=2008&amp;journal=Plant%20Growth%20Regul.&amp;volume=54&amp;pages=115-123" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10725-007-9234-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10725-007-9234-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10725-007-9234-7"> </span></div> </div></div></div></div></div><div content-id="DSQ030C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eitas</div> <div class="given-names">T.K.</div></div>,  <div class="name"><div class="surname">Dangl</div> <div class="given-names">J.L.</div></div>. </div><div class="article-title">NB-LRR proteins: pairs, pieces, perception, partners, and pathways</div>, <div class="source">Curr. Opin. Plant Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">472</div>-<div class="lpage">477</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NB-LRR%20proteins%3A%20pairs%2C%20pieces%2C%20perception%2C%20partners%2C%20and%20pathways&amp;author=T.K.%20Eitas&amp;author=J.L.%20Dangl&amp;publication_year=2010&amp;journal=Curr.%20Opin.%20Plant%20Biol.&amp;volume=13&amp;pages=472-477" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2010.04.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2010.04.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2010.04.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20483655" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Alvarez-Buylla</div> <div class="given-names">E.R.</div></div>,  <div class="name"><div class="surname">Pelaz</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Liljegren</div> <div class="given-names">S.J.</div></div>, et al. </div><div class="article-title">An ancestral MADS-box duplication occurred before the divergence of plants and animals</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2000</div>, vol. <div class="volume">97</div> (pg. <div class="fpage">5328</div>-<div class="lpage">5333</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20ancestral%20MADS-box%20duplication%20occurred%20before%20the%20divergence%20of%20plants%20and%20animals&amp;author=E.R.%20Alvarez-Buylla&amp;author=S.%20Pelaz&amp;author=S.J.%20Liljegren&amp;publication_year=2000&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=97&amp;pages=5328-5333" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.97.10.5328" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.97.10.5328" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.97.10.5328"> </span></div> </div></div></div></div></div><div content-id="DSQ030C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rijpkema</div> <div class="given-names">A.S.</div></div>,  <div class="name"><div class="surname">Gerats</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Vandenbussche</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Evolutionary complexity of MADS complexes</div>, <div class="source">Curr. Opin. Plant Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">32</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20complexity%20of%20MADS%20complexes&amp;author=A.S.%20Rijpkema&amp;author=T.%20Gerats&amp;author=M.%20Vandenbussche&amp;publication_year=2007&amp;journal=Curr.%20Opin.%20Plant%20Biol.&amp;volume=10&amp;pages=32-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2006.11.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2006.11.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2006.11.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17140839" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Yoo</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Park</div> <div class="given-names">S.H.</div></div>,  <div class="name"><div class="surname">Hwang</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Ahn</div> <div class="given-names">J.H.</div></div>. </div><div class="article-title">Role of SVP in the control of flowering time by ambient temperature in Arabidopsis</div>, <div class="source">Genes Dev.</div> , <div class="year">2007</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">397</div>-<div class="lpage">402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20SVP%20in%20the%20control%20of%20flowering%20time%20by%20ambient%20temperature%20in%20Arabidopsis&amp;author=J.H.%20Lee&amp;author=S.J.%20Yoo&amp;author=S.H.%20Park&amp;author=I.%20Hwang&amp;author=J.S.%20Lee&amp;author=J.H.%20Ahn&amp;publication_year=2007&amp;journal=Genes%20Dev.&amp;volume=21&amp;pages=397-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1518407" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1518407" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1518407"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17322399" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hartmann</div> <div class="given-names">U.</div></div>,  <div class="name"><div class="surname">Höhmann</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nettesheim</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Wisman</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Saedler</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Huijser</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Molecular cloning of SVP: a negative regulator of the floral transition in <em>Arabidopsis</em></div>, <div class="source">Plant J.</div> , <div class="year">2000</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">351</div>-<div class="lpage">360</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20cloning%20of%20SVP%3A%20a%20negative%20regulator%20of%20the%20floral%20transition%20in%20Arabidopsis&amp;author=U.%20Hartmann&amp;author=S.%20H%C3%B6hmann&amp;author=K.%20Nettesheim&amp;author=E.%20Wisman&amp;author=H.%20Saedler&amp;author=P.%20Huijser&amp;publication_year=2000&amp;journal=Plant%20J.&amp;volume=21&amp;pages=351-360" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313x.2000.00682.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313x.2000.00682.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313x.2000.00682.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10758486" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lee</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Choi</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">An</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Rice SVP-group MADS-box proteins, OsMADS22 and OsMADS55, are negative regulators of brassinosteroid responses</div>, <div class="source">Plant J.</div> , <div class="year">2008</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">93</div>-<div class="lpage">105</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rice%20SVP-group%20MADS-box%20proteins%2C%20OsMADS22%20and%20OsMADS55%2C%20are%20negative%20regulators%20of%20brassinosteroid%20responses&amp;author=S.%20Lee&amp;author=S.C.%20Choi&amp;author=G.%20An&amp;publication_year=2008&amp;journal=Plant%20J.&amp;volume=54&amp;pages=93-105" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.2008.03406.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.2008.03406.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.2008.03406.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18182025" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Leseberg</div> <div class="given-names">C.H.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Duvall</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Mao</div> <div class="given-names">L.</div></div>. </div><div class="article-title">Genome-wide analysis of the MADS-box gene family in <em>Populus trichocarpa</em></div>, <div class="source">Gene</div> , <div class="year">2006</div>, vol. <div class="volume">378</div> (pg. <div class="fpage">84</div>-<div class="lpage">94</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20the%20MADS-box%20gene%20family%20in%20Populus%20trichocarpa&amp;author=C.H.%20Leseberg&amp;author=A.%20Li&amp;author=H.%20Kang&amp;author=M.%20Duvall&amp;author=L.%20Mao&amp;publication_year=2006&amp;journal=Gene&amp;volume=378&amp;pages=84-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2006.05.022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2006.05.022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2006.05.022"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16831523" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ030C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ030C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dehgan</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Webster</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Morphology and infrageneric relationships of the genus <em>J. curcas</em></div>, <div class="year">1992</div><div class="publisher-name">University of California Press, Berkeley, CA, USA</div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2010. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=377199&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq030-dsq030_suppl_table_35.xls?Expires=2147483647&amp;Signature=1joRpI5vbEU-ZA8xaXpxXOgFN1XP6PbdCr928HdWM9OQcP~BTyHESw~m5UwfRVZkXaCX~FujlnX0MNwY4wNgFinHROE2Hq5FK0D6gQYImncfzrt9iZ26vvo8f98CIEh7oND49ytl38MvVAiq2klHxMGSQJgG0oR2haj7u5BuNz0V6H3FheCkAHtb2u56dZy4amvEzTWOalUoMk~lI6sEpTn-xIAqxKPDS3Og~Qv8z3II6dDbwu72CK~i~z6U~Wx~fnaYdDCz~zZd2NARs9iVFaJ7BFySWMg79C~UtZMI8suAigDRiw81I6AmX9JWP3AFFWkPsafPbfc62wI7ls93XQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-22</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq030-dsq030_suppl_table_34.xls?Expires=2147483647&amp;Signature=aRh-e2E0mT-l-oOhVqAbWTGIPBZnogoVSgmGeKnhSQXKhzDnVkILcIKCwCJigmtUitT4FMpVLMuCMxLbllxG1r4YtnJ1pScZCuz1uaDxN9Rhj41m1UzBS0KCmj528hvtyhwXYhAKlQJ9J4MWWqjqa5lQHlY5YunYYalLfHCMS1hcg5LziVX0yYKf67bggwrTclXPo2bVk57O5Q-xAitIPVuoKROQhxN2QkArShJci84MTvSnVByT7rbWzVB4k8WeU2Vzs9sXhYZMK8VFy-NaJqLFLEwfsds63rmJX0v-RAUgzNDanyME2WIVhaq73AET0iXjYn2fS247cyLozTNu3w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-21</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq030-dsq030_suppl_table_33.xls?Expires=2147483647&amp;Signature=FtBeWuROyHOh3YtOi2L52UU7Hav56b1ZdXl1TkFphh3jjgWTucIJlCo507cxASc2lEJSnYBW1Ie7tPfUgbVzn~xY5ggXv6ybh3XFfjo8ySD5tfp6qqd99YWCgfC8a67gJBpV4ED5ayX7KCbYcE1G6wGN5YOxsZNcUUMwdA1ERgs3y5ruK6WDbrApGeYuh-WO23s1u9ByYWecU9NpNB5GO8KofS2OX1P-vdMUbNaZWZuyTJXS9-WhexzWZEtGc44GAm4C9GZBiveaeB4OTm~wg7KdD2dWoIIsvGF6sPRvuG9oE8YXvSRLFageN3n3H4CjJdOlhRl0M4nSxxSPcRD1lw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-20</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq030-dsq030_suppl_table_32.xls?Expires=2147483647&amp;Signature=x3dtYGTPOFtwnx1iTHsBBiKWm9f17JK7NYzy5zEUdTn1VR1RG-OkN5Y6nxTEeiDD1bOhECIv4VpQUGePETSXE8rsjcp1aRq0Kn5c8KImjDwDzosXO4BQeV2jIdhQdcZ1UkV4c6LXe8oKb1Ro3boImBUHHkTdjZGXuZ8Rj-esQCUzbiV0GK8rPUexLnnp~9fMBZaiVkYW2TcQ2YS7Cf5QTnSj2UVUSsOlekLMjQ5pFFx1d0YQ5kI5cJMZULvfW3i56JQPelSScFv~NbJ3kgmhv8SC6sBKEf4uMrrAX4W0wuJ4w6i8FcVXosFIdDcGdy9~jmzzB7MnOL~gvKfDDTKEFw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-19</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq030-dsq030_suppl_table_31.xls?Expires=2147483647&amp;Signature=kcLg1sUBJnCmTFT0yLiwWa7C~vamfS0qvKLru0kQFEOke0zOe6agIcpBXp4C561J~JJ8xpbWOtPgMdrzlMRDf9ZEWuT8uncdZ6y577XBL-eLslrhezSX0HnsDGK~MAIKN1wgIgxxdBO--FAgzZNNBRq1vpShGdJquNiyVPw2ffVbyeX4w5tuTBuhPJmtORMe8y0RxTBDzq5QTTxPv88k0TC1BKluGJjHCfXEsly077AAewOQEK6pUpqg7GkmsJk2l5OHb8j3pkHj0AKG32z2jSHYa~oTisBjVughBWQUPd9X1xEbTCb6bGJ0x5JASHH-v3uO1gpzvmTsZhaz3YTk2g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-18</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq030-dsq030_suppl_table_30.xls?Expires=2147483647&amp;Signature=MqH-b2A7D7fd3RBZhJsSVo~Jn8OS09mRudA6AmpChgLPt83m8KLL0LQWRZcq4FHDu8KIC8fkYQNL~ZLnj3ANHXx34EjNM2n~TqeisejDDhktObGcLr05zu0isgBtTP4ibknf7qxlWiK7bRwy7oI2no3T6R91mc83ZL0oc85oU8Fc0v-InQDWmh~iPpgBv8PhJGzmOqquPkeR5b5L9cs~RRaBAXGczaqkRD7DKTKi5h-sO~QPWySi-CzbLN9WAe4EPAMVFPwKZTELnB4yrOF3iBU44R4ielUHmN~LmIRD4BCE93B7P9x4RFVf3wAKPaBZveQXKiERjFVhESdN1cE9cg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-17</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq030-dsq030_suppl_table_29.xls?Expires=2147483647&amp;Signature=jr1VFI1QsmzMADSX-JhBwX~7qpIMATzh-XYHIe4uRPsioW7Vxy5BCdPichy~7we0rTq2ViZO9KhQPv6cqHVN8atwOcZcsGWo-IAEqH40QK0HTGBIEtrRIxpLUfR14VhT7pOs3MIRkwvr2HfMh5EuMTJkpkzT4a38M4ZNdZ37ZFgnVVimNAEHiEMR3VSxkALdL0SoXA1B1Qg~aeGOsU0L4ttcuLxeFyGEoI-sKnRD3oj-RwTnbofh0rlFuHy3p~iTU~cl1UIkYUGE8ELVV64oywX4xDaEWQQeymc9nZFTBsUywgtT5zpdQz2PGLhe3iO7dLIpQBFZphcx~W5~inxXYg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-16</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq030-dsq030_suppl_table_28.xls?Expires=2147483647&amp;Signature=z0X4vrcSMwO6HKuFemIwBgoqGDvY4Pi7cnMthJKd1oL1wVivOK0wijyeAyOUi8yxLDCrghk-n8jRuJAAeLhS2KMllvuJoNF4Kd2C5RXActh6fIK-tOfslkCDvRt7TM5tYIYZNgF01402iA3lcBA6xvSwM5ZRSz-cqHCeFqaTijLtC2BFPcXXSpn3EsUlBu-Goc~M8IQiL3x4pKNN3DXsioGCJxvIK3BxenNDqZdCsfaNSxu4YyWrnrgMw4o6SVBE2AucwY7JQhx3UKBWDOC65OGmOiZR1VTXN-fuF31XjEKGaa7lo6encap9QV5LicBTyLTs~U5Vwfmq-0aHMcY3NA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-15</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq030-dsq030_suppl_table_27.xls?Expires=2147483647&amp;Signature=wRsuZMt9tH83LZonoUzHafzapPlXVKuFhkKxLJlZokmbtO6FZl6NAdt05acn~c1AIDtY2ttL4p2cNahXS-fF~~UUfsHMCiDFGqU2F~0QKxAO-8LTb0dtmEB6zs3Jt6z4dy9gql55BtQul5fe-gIeUsRbildGjXSJjvgTnuxLhx7Zr9Q1sJ9JvFTcfeGNZ7MeKF12nBJ1OQG2o6P6cmrXIf4I4Mk8LzISGpFx4f0p3FFgpWPc5J4e1vU0oqIcSy0Z9fajMajhzLaQUe4IaE1hZlhaXBqjMjLDNAOLvQrHDj~p~8NVjEsDIh2ZLL~6tjxG0S0joow1r7naA8hcnh0QNw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-14</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq030-dsq030_suppl_table_26.xls?Expires=2147483647&amp;Signature=1o3mQHd9XFAqtKXrSHuAJrlZPec4kjsOnBnwoWO2FwXsuVW~AVDcHz2c1ET8vXnacyNPP5ww4Tv4cJpIpt4iXnFR6zm2nhQ6cfwjhWBkSXU5fk0zd9q437mdvdhjrWbtSPDTPGeVzwzWwCyn8prcn1PXaLnD0Xi6t5yk2s2TWB9jQCdtmwdkjPpw2~7fAfxwzjyJba6xEMYqv1~FaK9qqR1cgeBLwP~AfrJo71QucpuaesZMoDCJQwi6UH9hbJUhzh9anlxMvHv7OUHT8mgZ-en5jYTlVARlJqmyZEz5UHPwu4jPVhzOyvsWrvrSeORH0ZFR~4UsHuOs0hfAid5xtw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-13</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq030-dsq030_suppl_table_25.xls?Expires=2147483647&amp;Signature=hmWlHERMCAribZ56dHqqHZrKD1gLXDQ9l8DSYgqlamKQf5rzFXywCDYs8PMGx~bWVzMtZQdiwdTRZiaamKqOaDhXI30bzunZX-5SyDafLpbO6S-r6seN2G4rVI2EzdAk-PQbPcZg7t2cPU3BApCHP5ulqw-wN1I1GYN5EgRYcpUoLxurVHZ-1TazyJDNXV1C9twauScoHEculzsGudWUqZDN1Hn0ZOoncitrq03VV4i1KcFOo5IMiV14pgkDyURrDbwFCCPrUD1zDUw4qATIfL7Y2w5AE5WuTeU1Jonj4Y6ac7m8wP9w6PQL5ppGPeY~hE28OrBXGuNn3SSeKf56pg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-12</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq030-dsq030_suppl_table_24.xls?Expires=2147483647&amp;Signature=un0X1o12-Rfe5Z3xYxbWSUUXdjJIdhY5UVFscM795~HtYNg93Zy1dhPPXMJ5bmUmWRgVIS4IVRE-jbDjLvy7QWs75gkEA80Bur4SqqVYZPPBlobZ~x~J9l0lCk0czSFJXZs6NphjM6181uDpvpCKIg3UwYYkWSu1kdnxPG7A-2LYW7pJuDcOuG3u7Km7zwZJQfNXTTU1k~CcSmd-4yJLc4a5jwEVA5KokkfHIcmBO-ixewTX4HR3z~z5ifDdxr3r9EX48Uvy7UmBowE8gfXkp4BYrsRwIuK~1wpCLzVftYSLXFQBaXY5E-ozq5nM6v6YFJtcRCTTh2nvuB2YMrrhBw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-11</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq030-dsq030_suppl_table_23.xls?Expires=2147483647&amp;Signature=caurndxDqBs~tjrLOmOPWlpORQjHssKOlNpJoEEyo5G3iNAh4rFbjrDJZ5czvTY8IYxQf652Xm8H~n4ihIyXwaOArieA2P39tdtcpG-RtWXj-ne13AfsrE17fESse4GuNXvPcJmdLdINy5xDTX2xbRmlWDSX27nMPfE8vtm0WZdMwTISqfqq~P6ADhJZAojjaJ2JlfsiBj~QqIzxyToyFkkWJmImsXg1kyNcTHfLYiPAfzHWlYH9W6l7TalyOKlSbMPKgn4WW~0QbRFnAYH~xknWuHmjvhectltuDsj7tsbHJPKa93KpwajTBujz-qbsk-adgDEf~7VKNDCeGtslYA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-10</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq030-dsq030_suppl_table_22.xls?Expires=2147483647&amp;Signature=xDHh~~VEofcFtrcONMHtG~BguIMQLWtI4Cc6l0WpmgguFJmOzr8jsTD5g8u0L0PRWzvpc2tyx-cbg5cl9sEAHRz5Ykg9HysRmiL9bgoM6xWRuhbsYcI0FKjvfUckl34ysnnofjJBzJP~BsCJKr4Nmz8tbWwFFC0xzp14qZzUo6qkhO8Wbsq2719yRmcDn-jGTczi4GddEhAqkZpkKrOUPCYfnqUGv35ymx~k~c8JPKXvk08CnCc8HjEnZAz5FmDAjqNipir1F4Xmwuz0B7WRbFMtzuqCOYnLgP9mZ-7~HoeI962ZqtnafQ0nArmC1D6qPfKCafQ~iWZAAeeGA0dwDQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-9</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq030-dsq030_suppl_table_21.xls?Expires=2147483647&amp;Signature=ZtfvC9nWsq3FHsZSVm5LOo0oyhxy8EBBT87yxTy-nfa5fQxBrsWhgKo8E9aHu~g9oDQvX~wdMYhE8GKBO47j4R4nMT6XQXGI1~1btZ2p83ohuH4GSPyGIZW4r0hJ6C9tpcvzXi5r~hL3IIVRSHFgMFJvQwjkN-~MJ58KV1z0ACTas4bT5GxApFmdQ-ZJSDmevoe4Gxepdo0BSBA9nQBdjmEEq-NWv5eJJkrOr1SuBUWf8lyjiWYcPXZhPiTOupMtuC~jfF5NxNYufHLBi5bj10nJUgbWKSjJYaKvivbb29j7o81T--CDCf3B-nG3VEbrvpuz3Kk~TgL2YAEhs6nRSQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq030-dsq030_suppl_table_20.xls?Expires=2147483647&amp;Signature=lUggAvdfc5FMQ2UR~fdyx-drgT-Ip~U-u1x6ozRa~~M0yc~SbWIGYtxX-rnCTLGSHzOETVfojp2QR43mKrOGLloZZRPfj8usCBvE3GrmNVJ9iQ9JSsa3F~Rvw1-EkQAdAA5wEe8Rk1tnM1pp8mmQzRrK3grTGDsB9X08cyuiWFm7DdfSIJY5X-yJ~pCLhVqgWqkmRj3KBfQtBKhN00Lv4HcIa1lakYN90jzF1dLiaaA4r8bxzx8aoFD91ten6K-FtR0h-iTFZNde2PJydhwNAYCAHYslO3JHnStc2-swJ-6vb2lya6IlUQO6p7fytqu7v-4VONLcI4WmNnu8bo~gBw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq030-dsq030_suppl_table_19.xls?Expires=2147483647&amp;Signature=z8J-ztoP-g0d0MRM0jP6XRjgg0mFTQQydv~c5cYP~B15PaKtS9K9kfX0ZFH1llBb6k7ATO2kVvDzJPS4ba8x4i0c6go-S-r9eubcxKv7BBTAHniSaaLu2QSmncXCYA01BW7i79b56zG~ZvebloPZp6rIXCm-9FpnMklekfgJJNYe0AT9gHnFSS-~58Yvo8fCHefSm4-LA94eVtXnjsFbXb78NGn43YtjYqxkM4BBdiDhOAl14ujlp3g41MUt0stlQx5sCav1922iu~s6UbKIfniaEm977SdBjOPkpQLL~ozldfgHrRkjSQQ4-v3Qs3JvnsmpPIVcHJUpdLpgRrV9ew__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq030-dsq030_suppl_table_18.xls?Expires=2147483647&amp;Signature=J0wCK8E4u8KnXQhsKuIfPvaFXww-dCKQ1i80NFVL6e3bX6vRvnwTHSeHVqUTQ4FG8YD9zuEduljh7ZkCDZuuy1o0qiSp~j79kdgrlrq~cG4CvEesBmuva84aicAzHcnhtCDylVkHgT35wGRzzQJdv0PcWv9119jvmLU4klwtEAR4I9anmdQ~K5k625EDU0MPHXiegBQ1z32r5p9lyMmLDILSJvB9l~MNw46X4MGLuS2LOxT9MYL7kNUFXW~O~Qj9ushc~jR55~f~SI3ZiS84as0YIHAlAzuyEgWJ41JDEjtzHcV5-Q~ADUirFiXT2u3E0yozO5GSWgZyQA7B~igHSw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq030-dsq030_suppl_table_17.xls?Expires=2147483647&amp;Signature=zcfaCx9kgylCNuR~ZIlGFaWl4hiMtHDw6J6HEe0nqwe1AmEXQ3yPO4wyzJ2TDRwT4I9WRRyvTL5hCm0CdIa3W1E-j9f0Et1AV-nQYAbNSBDoX5DHY1evbwdCH6YIfSgPi5i5judPrqIfdBwEvkpVCaI1UVMS5rwoY0xkR8xB12ODJokE9KzgBsm4LGFd7IrhPhm2uCy3fd67N3Vt0T~ii0FnuhLkwn2YaDwFqPOX5dLONiBxoK7vsaiA7bIh~ergmdSf7q9qu5~~E8Q7aMf57IkgrLJsExZHKgIYnYETJYLJ2LrrYC0NmoWudbynrfcC2ZbamBzzCXc1jofm8LnUNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq030-dsq030_suppl_table_16.xls?Expires=2147483647&amp;Signature=4hn51YcVGpzvRpnaPGV639tdsdDnykC8SP2P1TduoCKl6gGx3q9qE51U7UYb4aLUI6nJB2R-xtTQItGCCMjH8AsE7n8KmP1qJTpAIgw78MRUv-dW7K2iLmWPzkVnwTAv8LxjFGmxViJi4tfWbpqtAw4ie4RqxjoU7-ciGkSlqoLo9g5J2NXkE~DvF6D6HKA~fH-K3Zot3xBJGOxjdB~EYu2rVag5BJnqbYmTVgj8VBLEhHAyxxhfT0NNxL~VRnFHZDmetaPK~smYlrr9CpsXEu4OkFNRHZczNRU5Xr79gny3TRQyqW2HnvsuBvlWyW7Bak3Oft4q5z2WpmyqGchasw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq030-dsq030_suppl_table_15.xls?Expires=2147483647&amp;Signature=ahr2CtBxOK1T0vBle0R8bRZ-x1uRbu2-5vz97N821H0Mp4zPA7gbBQALt9t8dOfFr81TumNBNVFO9uJrccSEAVZRsqbtSr~Acueq3L3DOAVHglsqAmj~vmd7Tdcpw1ZaQ2z19dCaE0Ip30ncC96mci2SxkylsgVoRSejEOXen4F8iA0hmBZqHSyN0U2Ln3M3NYlSOynacLDEP~NJNjkjdF7WEgEjvxY7McO4DxpPp5Z7Vfs0HCvETqfp9thHA4N0pB-0CM7cQDvl0H0Vq~m-V0WrNtbSqBAtk362HepmXDhtf8MlNgzCEzMz4JXlQDJLuZQ6kIJDqRiaZ5bVj4fmxg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq030/2/dsq030-dsq030suppl.pdf?Expires=2147483647&amp;Signature=np6-f6vGV~O3ESOHaRqkwAaPyag9dYgR7NWYT1sMRpx~r7kXYYK98-daB5Oxu0Y2XvyF-FUQvAWYrIK8LSH0biQukvCwlgiFO-hEQFPaMs1o8zWR1TUt49ERvTbHyR~iVMGc0sLPura4CiSRQG3U0ZKu7YMEvujv10ZWUNeSSNXFtCgjebJ-t8QKGVxmldAUZkp5Cz2u4silVhsMElMowBGiWH97nzVzRkZFBMWSee9-n-DZHGVNpFHEEGDpxAE4ckphsOQHeEmovNAQycOljwml2oz7rEWVuTirvS3dlg3RjZsd5JQeOvIiUoCcJpDNwkqDkCyduvpdEC5xVYxuHQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">861</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000288188700006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">144</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsq030"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="377199"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000288188700006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f18%2f1%2f65%2f377199">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000288188700006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (144)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F18%2F1%2F65%2F377199" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/377199'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Sequence Analysis of the Genome of an Oil-Bearing Tree, Jatropha curcas L." />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsq030");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f18%2f1%2f65%2f377199">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

