

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":1991,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/13/5/197/400871","@type":"ScholarlyArticle","name":"Pattern and Rate of Indel Evolution Inferred from Whole Chloroplast Intergenic Regions in Sugarcane, Maize and Rice","datePublished":"2006-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/13/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2006-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsl012","keywords":["chloroplast genome","evolutionary rate","indels","non-coding DNA sequence","intergenic region"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/13/5/197/400871","author":[{"name":"Yamane, Kyoko","affiliation":"1  Laboratory of Crop evolution, Graduate School of Agriculture, Kyoto University  Nakajoh, Mozume, Mukoh 617-0001, Japan    2  Plant resource Laboratory, Graduate School and School of Life and Environmental, Osaka Prefectural University  1-1 Gakuen-cho, Nakaku, Sakai, Osaka 599-8531, Japan","@type":"Person"},{"name":"Yano, Kentaro","affiliation":"3  Plant Breeding Laboratory, Graduate School of Agriculture, Kyoto University  Nakajoh, Oiwake-cho, Kitashirakawa, Sakyo-ku, Kyoto 606-8502, Japan    4  Kazusa DNA Research Institute  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Kawahara, Taihachi","affiliation":"1  Laboratory of Crop evolution, Graduate School of Agriculture, Kyoto University  Nakajoh, Mozume, Mukoh 617-0001, Japan","@type":"Person"}],"description":"Abstract.  Microstructural changes such as insertions and deletions (=indels) are a major driving force in the evolution of non-coding DNA sequences. To better ","pageStart":"197","pageEnd":"204","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/5/0/m_dnaresearch13_5.cover.gif?Expires=2147483647&Signature=mud4uh-30-GlHhdYLUg7ju5UAULSV1q0NE0Wrnu3RkEfHq-bQsyIw-lCORVdItpQPaM1Yqh7Mlu1ZRFevXZzXbVfRNUVlJab8lK-GCnEhdWYhEGef0xZWaCkBQCPrWvFt7U3DQUW~N39pDu-W62zATjEI5H8weIBmZySoS7ld8JJHNoZ~KRFmJwMZ~vay8M9zJZLeiRO8~3Z29cer7cu1n7PWnH2IdAgSUN6NGoLNlAwxp8MJUVjiUx9vdSEaj5zwY9CII2P1JOFiUJHY8pRgIGGQLQdF1uHKlUcELu9skGdb8-~LToWZ5KjifePPIlv7WMkw03h8gvh~iZhhNJb6A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Pattern and Rate of Indel Evolution Inferred from Whole Chloroplast Intergenic Regions in Sugarcane, Maize and Rice","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/5/0/m_dnaresearch13_5.cover.gif?Expires=2147483647&Signature=mud4uh-30-GlHhdYLUg7ju5UAULSV1q0NE0Wrnu3RkEfHq-bQsyIw-lCORVdItpQPaM1Yqh7Mlu1ZRFevXZzXbVfRNUVlJab8lK-GCnEhdWYhEGef0xZWaCkBQCPrWvFt7U3DQUW~N39pDu-W62zATjEI5H8weIBmZySoS7ld8JJHNoZ~KRFmJwMZ~vay8M9zJZLeiRO8~3Z29cer7cu1n7PWnH2IdAgSUN6NGoLNlAwxp8MJUVjiUx9vdSEaj5zwY9CII2P1JOFiUJHY8pRgIGGQLQdF1uHKlUcELu9skGdb8-~LToWZ5KjifePPIlv7WMkw03h8gvh~iZhhNJb6A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Pattern and Rate of Indel Evolution Inferred from Whole Chloroplast Intergenic Regions in Sugarcane, Maize and Rice" />
    <meta property="og:description" content="Abstract.  Microstructural changes such as insertions and deletions (=indels) are a major driving force in the evolution of non-coding DNA sequences. To better " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/13/5/197/400871" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2006-01-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Yamane, Kyoko" /><meta name="citation_author_institution" content="1Laboratory of Crop evolution, Graduate School of Agriculture, Kyoto UniversityNakajoh, Mozume, Mukoh 617-0001, Japan" /><meta name="citation_author_institution" content="2Plant resource Laboratory, Graduate School and School of Life and Environmental, Osaka Prefectural University1-1 Gakuen-cho, Nakaku, Sakai, Osaka 599-8531, Japan" /><meta name="citation_author" content="Yano, Kentaro" /><meta name="citation_author_institution" content="3Plant Breeding Laboratory, Graduate School of Agriculture, Kyoto UniversityNakajoh, Oiwake-cho, Kitashirakawa, Sakyo-ku, Kyoto 606-8502, Japan" /><meta name="citation_author_institution" content="4Kazusa DNA Research Institute2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kawahara, Taihachi" /><meta name="citation_author_institution" content="1Laboratory of Crop evolution, Graduate School of Agriculture, Kyoto UniversityNakajoh, Mozume, Mukoh 617-0001, Japan" /><meta name="citation_title" content="Pattern and Rate of Indel Evolution Inferred from Whole Chloroplast Intergenic Regions in Sugarcane, Maize and Rice" /><meta name="citation_firstpage" content="197" /><meta name="citation_lastpage" content="204" /><meta name="citation_doi" content="10.1093/dnares/dsl012" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="13" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2006/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Majority of divergence between closely related DNA samples is due to indels; Proc. Natl Acad. Sci. USA;  citation_year=2003;  citation_volume=100;  citation_pages=4661-4665; " /><meta name="citation_reference" content="citation_title=The evolution of non-coding chloroplast DNA and its application in plant systematics; Ann. Missouri Bot. Gard.;  citation_year=2000;  citation_volume=87;  citation_pages=499-527; " /><meta name="citation_reference" content="citation_title=Molecular evolution of insertions and deletions in the chloroplast genome of Silene; Mol. Biol. Evol.;  citation_year=2003;  citation_volume=20;  citation_pages=1737-1740; " /><meta name="citation_reference" content="citation_title=Patterns and relative rates of nucleotide and insertion/deletion evolution at six chloroplast intergenic regions in New World species of the Lecythidaceae; Mol. Biol. Evol.;  citation_year=2003;  citation_volume=20;  citation_pages=1710-1721; " /><meta name="citation_reference" content="citation_title=The complete sequence of the rice (Oryza sativa) chloroplast genome: intermolecular recombination between distinct tRNA genes accounts for a major plastid DNA inversion during the evolution of the cereals; Mol. Gen. Genet.;  citation_year=1989;  citation_volume=217;  citation_pages=185-194; " /><meta name="citation_reference" content="citation_title=Complete sequence of the maize chloroplast genome: gene content, hotspots of divergence and fine tuning of genetic information by transcript editing; J. Mol. Biol.;  citation_year=1995;  citation_volume=251;  citation_pages=614-628; " /><meta name="citation_reference" content="citation_title=Structural features of a wheat plastome as revealed by complete sequencing of chloroplast DNA; citation_text=(19 co-authors); Mol. Gen. Genet.;  citation_year=2002;  citation_volume=266;  citation_pages=740-746; " /><meta name="citation_reference" content="citation_title=Whole chloroplast genome comparison of rice, maize, and wheat: implications for chloroplast gene diversification and phylogeny of cereals; Mol. Biol. Evol.;  citation_year=2002;  citation_volume=19;  citation_pages=2084-2091; " /><meta name="citation_reference" content="citation_title=Complete nucleotide sequence of the sugarcane (Saccharum officinarum) chloroplast genome: a comparative analysis of four monocot chloroplast genomes; DNA res.;  citation_year=2004;  citation_volume=11;  citation_pages=93-99; " /><meta name="citation_reference" content="citation_title=A search for markers of sugarcane evolution; Genet. Mol. Biol.;  citation_year=2001;  citation_volume=24;  citation_pages=169-174; " /><meta name="citation_reference" content="citation_title=Molecular phylogeny of Miscanthus s.l., Saccharum and related genera (Saccharinae, Andropogoneae, Poaceae) using DNA sequences from ITS nuclear ribosomal DNA and the plastid trnL-F regions; J. Plant Res.;  citation_year=2002;  citation_volume=115;  citation_pages=381-392; " /><meta name="citation_reference" content="citation_title=A comparison of rice chloroplast genomes; Plant Phys.;  citation_year=2004;  citation_volume=135;  citation_pages=412-420; " /><meta name="citation_reference" content="citation_title=Uniparental inheritance of mitochondrial aid chloroplast genes: mechanisms and evolution; Proc. Natl Acad. Sci. USA;  citation_year=1995;  citation_volume=92;  citation_pages=11331-11338; " /><meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, positions-specific gap penalties and weight matrix choice; Nucleic Acids Res.;  citation_year=1994;  citation_volume=22;  citation_pages=4673-4680; " /><meta name="citation_reference" content="citation_title=The chloroplast genome; Plant Mol. Biol.;  citation_year=1992;  citation_volume=19;  citation_pages=149-168; " /><meta name="citation_reference" content="citation_title=Regulation of gene expression in chloroplasts of higher plants; Plant Mol. Biol.;  citation_year=1996;  citation_volume=32;  citation_pages=315-26; " /><meta name="citation_reference" content="citation_title=DnaSP version 3: an integrated program for molecular population genetics and molecular evolution analysis; Bioinformatics;  citation_year=1999;  citation_volume=15;  citation_pages=174-175; " /><meta name="citation_reference" content="citation_publisher=Evolutionary Genetics, Columbia University Press, New York; Molecular;  citation_year=1987; " /><meta name="citation_reference" content="citation_title=The size distribution of insertions and deletions in human and rodent pseudogenes suggests the logarithmic gap penalty for sequence alignment; J. Mol. Evol.;  citation_year=1995;  citation_volume=40;  citation_pages=464-473; " /><meta name="citation_reference" content="citation_title=Evolutionary rates of insertion and deletion in noncoding nucleotide sequences of primates; Mol. Biol. Evol.;  citation_year=1994;  citation_volume=11;  citation_pages=504-512; " /><meta name="citation_reference" content="citation_title=Deletions in processed pseudogenes accumulate faster in rodents than in humans; J. Mol. Evol.;  citation_year=1989;  citation_volume=28;  citation_pages=279-285; " /><meta name="citation_reference" content="citation_title=Evolution of a noncoding region of the chloroplast genome; Mol. Phylogenet. Evol.;  citation_year=1993;  citation_volume=2;  citation_pages=52-64; " /><meta name="citation_reference" content="citation_title=Slipped-strand mispairing: a major mechanism for DNA sequence evolution; Mol. Biol. Evol.;  citation_year=1987;  citation_volume=4;  citation_pages=203-221; " /><meta name="citation_reference" content="citation_title=The complete nucleotide sequence of wild rice (Oryza nivara) chloroplast genome: first genome wide comparative sequence analysis of wild and cultivated rice; Gene;  citation_year=2004;  citation_volume=340;  citation_pages=133-139; " /><meta name="citation_reference" content="citation_title=Neighboring base composition and transversion/transition bias in a comparison of rice and maize chloroplast noncoding regions; Proc. Natl Acad. Sci. USA;  citation_year=1995;  citation_volume=92;  citation_pages=9717-9721; " /><meta name="citation_reference" content="citation_title=Homopolymer tract length dependent enrichments in functional regions of 27 eukaryotes and their novel dependence on the organism DNA (G+C)% composition; BMC Genomics;  citation_year=2004;  citation_volume=5;  citation_pages=95" /><meta name="citation_reference" content="citation_title=Chloroplast gene sequences and the study of plant evolution; Proc. Natl Acad. Sci. USA;  citation_year=1993;  citation_volume=90;  citation_pages=363-367; " /><meta name="citation_reference" content="citation_title=Very close relationship of the chloroplast genomes among Saccharum species; Theor. Appl. Genet.;  citation_year=2005;  citation_volume=110;  citation_pages=1523-1529; " /><meta name="citation_reference" content="citation_title=Rapid recent growth and divergence of rice nuclear genomes; Proc. Natl Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=12404-12410; " /><meta name="citation_reference" content="citation_title=Phylogenetic relationships among A-genome species of the genus Oryza revealed by intron sequences of four nuclear genes; New Phytol.;  citation_year=2005;  citation_volume=167;  citation_pages=249-265; " /><meta name="citation_reference" content="citation_title=Molecular evolution of chloroplast DNA sequences; Mol. Biol. Evol.;  citation_year=1984;  citation_volume=1;  citation_pages=291-301; " /><meta name="citation_reference" content="citation_title=Comparative organization of chloroplast genomes; Annu. Rev. Genet.;  citation_year=1985;  citation_volume=19;  citation_pages=325-354; " /><meta name="citation_reference" content="citation_title=Rates of nucleotide substitution vary greatly across mitochondrial, chloroplast, and nuclear DNAs; Proc. Natl Acad. Sci. USA;  citation_year=1987;  citation_volume=84;  citation_pages=9054-9058; " /><meta name="citation_reference" content="citation_title=Biased gene conversion, copy number, and apparent mutation rate differences within chloroplast and bacterial genomes; Genetics;  citation_year=1992;  citation_volume=130;  citation_pages=677-783; " /><meta name="citation_reference" content="citation_title=Nucleotide substitution rates in legume chloroplast DNA depend on the presence of the inverted repeat; J. Mol. Evol.;  citation_year=2002;  citation_volume=55;  citation_pages=501-508; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/13/5/197/17462597/dsl012.pdf" /><meta name="description" content="Abstract.  Microstructural changes such as insertions and deletions (=indels) are a major driving force in the evolution of non-coding DNA sequences. To better " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/13/5/197/400871" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/13/5/197/400871" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/13/5/197/400871");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsl012';
    </script>
<title>
	Pattern and Rate of Indel Evolution Inferred from Whole Chloroplast Intergenic Regions in Sugarcane, Maize and Rice | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./400871" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZ1UPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZYW1hbmUsIEt5b2tvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMUxhYm9yYXRvcnkgb2YgQ3JvcCBldm9sdXRpb24sIEdyYWR1YXRlIFNjaG9vbCBvZiBBZ3JpY3VsdHVyZSwgS3lvdG8gVW5pdmVyc2l0eU5ha2Fqb2gsIE1venVtZSwgTXVrb2ggNjE3LTAwMDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMlBsYW50IHJlc291cmNlIExhYm9yYXRvcnksIEdyYWR1YXRlIFNjaG9vbCBhbmQgU2Nob29sIG9mIExpZmUgYW5kIEVudmlyb25tZW50YWwsIE9zYWthIFByZWZlY3R1cmFsIFVuaXZlcnNpdHkxLTEgR2FrdWVuLWNobywgTmFrYWt1LCBTYWthaSwgT3Nha2EgNTk5LTg1MzEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFubywgS2VudGFybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjNQbGFudCBCcmVlZGluZyBMYWJvcmF0b3J5LCBHcmFkdWF0ZSBTY2hvb2wgb2YgQWdyaWN1bHR1cmUsIEt5b3RvIFVuaXZlcnNpdHlOYWtham9oLCBPaXdha2UtY2hvLCBLaXRhc2hpcmFrYXdhLCBTYWt5by1rdSwgS3lvdG8gNjA2LTg1MDIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iNEthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2F3YWhhcmEsIFRhaWhhY2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMUxhYm9yYXRvcnkgb2YgQ3JvcCBldm9sdXRpb24sIEdyYWR1YXRlIFNjaG9vbCBvZiBBZ3JpY3VsdHVyZSwgS3lvdG8gVW5pdmVyc2l0eU5ha2Fqb2gsIE1venVtZSwgTXVrb2ggNjE3LTAwMDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJQYXR0ZXJuIGFuZCBSYXRlIG9mIEluZGVsIEV2b2x1dGlvbiBJbmZlcnJlZCBmcm9tIFdob2xlIENobG9yb3BsYXN0IEludGVyZ2VuaWMgUmVnaW9ucyBpbiBTdWdhcmNhbmUsIE1haXplIGFuZCBSaWNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMTk3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyMDQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc2wwMTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAwNi8wMS8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ham9yaXR5IG9mIGRpdmVyZ2VuY2UgYmV0d2VlbiBjbG9zZWx5IHJlbGF0ZWQgRE5BIHNhbXBsZXMgaXMgZHVlIHRvIGluZGVsczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDA7ICBjaXRhdGlvbl9wYWdlcz00NjYxLTQ2NjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBldm9sdXRpb24gb2Ygbm9uLWNvZGluZyBjaGxvcm9wbGFzdCBETkEgYW5kIGl0cyBhcHBsaWNhdGlvbiBpbiBwbGFudCBzeXN0ZW1hdGljczsgQW5uLiBNaXNzb3VyaSBCb3QuIEdhcmQuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTg3OyAgY2l0YXRpb25fcGFnZXM9NDk5LTUyNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGV2b2x1dGlvbiBvZiBpbnNlcnRpb25zIGFuZCBkZWxldGlvbnMgaW4gdGhlIGNobG9yb3BsYXN0IGdlbm9tZSBvZiBTaWxlbmU7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0xNzM3LTE3NDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhdHRlcm5zIGFuZCByZWxhdGl2ZSByYXRlcyBvZiBudWNsZW90aWRlIGFuZCBpbnNlcnRpb24vZGVsZXRpb24gZXZvbHV0aW9uIGF0IHNpeCBjaGxvcm9wbGFzdCBpbnRlcmdlbmljIHJlZ2lvbnMgaW4gTmV3IFdvcmxkIHNwZWNpZXMgb2YgdGhlIExlY3l0aGlkYWNlYWU7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0xNzEwLTE3MjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb21wbGV0ZSBzZXF1ZW5jZSBvZiB0aGUgcmljZSAoT3J5emEgc2F0aXZhKSBjaGxvcm9wbGFzdCBnZW5vbWU6IGludGVybW9sZWN1bGFyIHJlY29tYmluYXRpb24gYmV0d2VlbiBkaXN0aW5jdCB0Uk5BIGdlbmVzIGFjY291bnRzIGZvciBhIG1ham9yIHBsYXN0aWQgRE5BIGludmVyc2lvbiBkdXJpbmcgdGhlIGV2b2x1dGlvbiBvZiB0aGUgY2VyZWFsczsgTW9sLiBHZW4uIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTc7ICBjaXRhdGlvbl9wYWdlcz0xODUtMTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZSBzZXF1ZW5jZSBvZiB0aGUgbWFpemUgY2hsb3JvcGxhc3QgZ2Vub21lOiBnZW5lIGNvbnRlbnQsIGhvdHNwb3RzIG9mIGRpdmVyZ2VuY2UgYW5kIGZpbmUgdHVuaW5nIG9mIGdlbmV0aWMgaW5mb3JtYXRpb24gYnkgdHJhbnNjcmlwdCBlZGl0aW5nOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTI1MTsgIGNpdGF0aW9uX3BhZ2VzPTYxNC02Mjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cnVjdHVyYWwgZmVhdHVyZXMgb2YgYSB3aGVhdCBwbGFzdG9tZSBhcyByZXZlYWxlZCBieSBjb21wbGV0ZSBzZXF1ZW5jaW5nIG9mIGNobG9yb3BsYXN0IEROQTsgY2l0YXRpb25fdGV4dD0oMTkgY28tYXV0aG9ycyk7IE1vbC4gR2VuLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MjY2OyAgY2l0YXRpb25fcGFnZXM9NzQwLTc0NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hvbGUgY2hsb3JvcGxhc3QgZ2Vub21lIGNvbXBhcmlzb24gb2YgcmljZSwgbWFpemUsIGFuZCB3aGVhdDogaW1wbGljYXRpb25zIGZvciBjaGxvcm9wbGFzdCBnZW5lIGRpdmVyc2lmaWNhdGlvbiBhbmQgcGh5bG9nZW55IG9mIGNlcmVhbHM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0yMDg0LTIwOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIG51Y2xlb3RpZGUgc2VxdWVuY2Ugb2YgdGhlIHN1Z2FyY2FuZSAoU2FjY2hhcnVtIG9mZmljaW5hcnVtKSBjaGxvcm9wbGFzdCBnZW5vbWU6IGEgY29tcGFyYXRpdmUgYW5hbHlzaXMgb2YgZm91ciBtb25vY290IGNobG9yb3BsYXN0IGdlbm9tZXM7IEROQSByZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9OTMtOTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc2VhcmNoIGZvciBtYXJrZXJzIG9mIHN1Z2FyY2FuZSBldm9sdXRpb247IEdlbmV0LiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MTY5LTE3NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIHBoeWxvZ2VueSBvZiBNaXNjYW50aHVzIHMubC4sIFNhY2NoYXJ1bSBhbmQgcmVsYXRlZCBnZW5lcmEgKFNhY2NoYXJpbmFlLCBBbmRyb3BvZ29uZWFlLCBQb2FjZWFlKSB1c2luZyBETkEgc2VxdWVuY2VzIGZyb20gSVRTIG51Y2xlYXIgcmlib3NvbWFsIEROQSBhbmQgdGhlIHBsYXN0aWQgdHJuTC1GIHJlZ2lvbnM7IEouIFBsYW50IFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTE1OyAgY2l0YXRpb25fcGFnZXM9MzgxLTM5MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBjb21wYXJpc29uIG9mIHJpY2UgY2hsb3JvcGxhc3QgZ2Vub21lczsgUGxhbnQgUGh5cy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTM1OyAgY2l0YXRpb25fcGFnZXM9NDEyLTQyMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VW5pcGFyZW50YWwgaW5oZXJpdGFuY2Ugb2YgbWl0b2Nob25kcmlhbCBhaWQgY2hsb3JvcGxhc3QgZ2VuZXM6IG1lY2hhbmlzbXMgYW5kIGV2b2x1dGlvbjsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT05MjsgIGNpdGF0aW9uX3BhZ2VzPTExMzMxLTExMzM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DTFVTVEFMIFc6IGltcHJvdmluZyB0aGUgc2Vuc2l0aXZpdHkgb2YgcHJvZ3Jlc3NpdmUgbXVsdGlwbGUgc2VxdWVuY2UgYWxpZ25tZW50IHRocm91Z2ggc2VxdWVuY2Ugd2VpZ2h0aW5nLCBwb3NpdGlvbnMtc3BlY2lmaWMgZ2FwIHBlbmFsdGllcyBhbmQgd2VpZ2h0IG1hdHJpeCBjaG9pY2U7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTQ2NzMtNDY4MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNobG9yb3BsYXN0IGdlbm9tZTsgUGxhbnQgTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTE0OS0xNjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZ3VsYXRpb24gb2YgZ2VuZSBleHByZXNzaW9uIGluIGNobG9yb3BsYXN0cyBvZiBoaWdoZXIgcGxhbnRzOyBQbGFudCBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9MzE1LTI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EbmFTUCB2ZXJzaW9uIDM6IGFuIGludGVncmF0ZWQgcHJvZ3JhbSBmb3IgbW9sZWN1bGFyIHBvcHVsYXRpb24gZ2VuZXRpY3MgYW5kIG1vbGVjdWxhciBldm9sdXRpb24gYW5hbHlzaXM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MTc0LTE3NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUV2b2x1dGlvbmFyeSBHZW5ldGljcywgQ29sdW1iaWEgVW5pdmVyc2l0eSBQcmVzcywgTmV3IFlvcms7IE1vbGVjdWxhcjsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHNpemUgZGlzdHJpYnV0aW9uIG9mIGluc2VydGlvbnMgYW5kIGRlbGV0aW9ucyBpbiBodW1hbiBhbmQgcm9kZW50IHBzZXVkb2dlbmVzIHN1Z2dlc3RzIHRoZSBsb2dhcml0aG1pYyBnYXAgcGVuYWx0eSBmb3Igc2VxdWVuY2UgYWxpZ25tZW50OyBKLiBNb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9NDY0LTQ3MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uYXJ5IHJhdGVzIG9mIGluc2VydGlvbiBhbmQgZGVsZXRpb24gaW4gbm9uY29kaW5nIG51Y2xlb3RpZGUgc2VxdWVuY2VzIG9mIHByaW1hdGVzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NTA0LTUxMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGVsZXRpb25zIGluIHByb2Nlc3NlZCBwc2V1ZG9nZW5lcyBhY2N1bXVsYXRlIGZhc3RlciBpbiByb2RlbnRzIHRoYW4gaW4gaHVtYW5zOyBKLiBNb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9Mjc5LTI4NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIGEgbm9uY29kaW5nIHJlZ2lvbiBvZiB0aGUgY2hsb3JvcGxhc3QgZ2Vub21lOyBNb2wuIFBoeWxvZ2VuZXQuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz01Mi02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2xpcHBlZC1zdHJhbmQgbWlzcGFpcmluZzogYSBtYWpvciBtZWNoYW5pc20gZm9yIEROQSBzZXF1ZW5jZSBldm9sdXRpb247IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTIwMy0yMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb21wbGV0ZSBudWNsZW90aWRlIHNlcXVlbmNlIG9mIHdpbGQgcmljZSAoT3J5emEgbml2YXJhKSBjaGxvcm9wbGFzdCBnZW5vbWU6IGZpcnN0IGdlbm9tZSB3aWRlIGNvbXBhcmF0aXZlIHNlcXVlbmNlIGFuYWx5c2lzIG9mIHdpbGQgYW5kIGN1bHRpdmF0ZWQgcmljZTsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zNDA7ICBjaXRhdGlvbl9wYWdlcz0xMzMtMTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZWlnaGJvcmluZyBiYXNlIGNvbXBvc2l0aW9uIGFuZCB0cmFuc3ZlcnNpb24vdHJhbnNpdGlvbiBiaWFzIGluIGEgY29tcGFyaXNvbiBvZiByaWNlIGFuZCBtYWl6ZSBjaGxvcm9wbGFzdCBub25jb2RpbmcgcmVnaW9uczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT05MjsgIGNpdGF0aW9uX3BhZ2VzPTk3MTctOTcyMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SG9tb3BvbHltZXIgdHJhY3QgbGVuZ3RoIGRlcGVuZGVudCBlbnJpY2htZW50cyBpbiBmdW5jdGlvbmFsIHJlZ2lvbnMgb2YgMjcgZXVrYXJ5b3RlcyBhbmQgdGhlaXIgbm92ZWwgZGVwZW5kZW5jZSBvbiB0aGUgb3JnYW5pc20gRE5BIChHK0MpJSBjb21wb3NpdGlvbjsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz05NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNobG9yb3BsYXN0IGdlbmUgc2VxdWVuY2VzIGFuZCB0aGUgc3R1ZHkgb2YgcGxhbnQgZXZvbHV0aW9uOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTkwOyAgY2l0YXRpb25fcGFnZXM9MzYzLTM2NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmVyeSBjbG9zZSByZWxhdGlvbnNoaXAgb2YgdGhlIGNobG9yb3BsYXN0IGdlbm9tZXMgYW1vbmcgU2FjY2hhcnVtIHNwZWNpZXM7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTEwOyAgY2l0YXRpb25fcGFnZXM9MTUyMy0xNTI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXBpZCByZWNlbnQgZ3Jvd3RoIGFuZCBkaXZlcmdlbmNlIG9mIHJpY2UgbnVjbGVhciBnZW5vbWVzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTEyNDA0LTEyNDEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlsb2dlbmV0aWMgcmVsYXRpb25zaGlwcyBhbW9uZyBBLWdlbm9tZSBzcGVjaWVzIG9mIHRoZSBnZW51cyBPcnl6YSByZXZlYWxlZCBieSBpbnRyb24gc2VxdWVuY2VzIG9mIGZvdXIgbnVjbGVhciBnZW5lczsgTmV3IFBoeXRvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTY3OyAgY2l0YXRpb25fcGFnZXM9MjQ5LTI2NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGV2b2x1dGlvbiBvZiBjaGxvcm9wbGFzdCBETkEgc2VxdWVuY2VzOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTg0OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0yOTEtMzAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBvcmdhbml6YXRpb24gb2YgY2hsb3JvcGxhc3QgZ2Vub21lczsgQW5udS4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTE5ODU7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0zMjUtMzU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXRlcyBvZiBudWNsZW90aWRlIHN1YnN0aXR1dGlvbiB2YXJ5IGdyZWF0bHkgYWNyb3NzIG1pdG9jaG9uZHJpYWwsIGNobG9yb3BsYXN0LCBhbmQgbnVjbGVhciBETkFzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTg0OyAgY2l0YXRpb25fcGFnZXM9OTA1NC05MDU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaWFzZWQgZ2VuZSBjb252ZXJzaW9uLCBjb3B5IG51bWJlciwgYW5kIGFwcGFyZW50IG11dGF0aW9uIHJhdGUgZGlmZmVyZW5jZXMgd2l0aGluIGNobG9yb3BsYXN0IGFuZCBiYWN0ZXJpYWwgZ2Vub21lczsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MTMwOyAgY2l0YXRpb25fcGFnZXM9Njc3LTc4MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnVjbGVvdGlkZSBzdWJzdGl0dXRpb24gcmF0ZXMgaW4gbGVndW1lIGNobG9yb3BsYXN0IEROQSBkZXBlbmQgb24gdGhlIHByZXNlbmNlIG9mIHRoZSBpbnZlcnRlZCByZXBlYXQ7IEouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NTU7ICBjaXRhdGlvbl9wYWdlcz01MDEtNTA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTMvNS8xOTcvMTc0NjI1OTcvZHNsMDEyLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBNaWNyb3N0cnVjdHVyYWwgY2hhbmdlcyBzdWNoIGFzIGluc2VydGlvbnMgYW5kIGRlbGV0aW9ucyAoPWluZGVscykgYXJlIGEgbWFqb3IgZHJpdmluZyBmb3JjZSBpbiB0aGUgZXZvbHV0aW9uIG9mIG5vbi1jb2RpbmcgRE5BIHNlcXVlbmNlcy4gVG8gYmV0dGVyICIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDAwODcxJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZC4VUnRxgL8MS+M4lTP26W6UR7Do" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f5%2f197%2f400871" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f5%2f197%2f400871" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=15658" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/13/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/13/5/0/m_dnaresearch13_5.cover.gif?Expires=2147483647&amp;Signature=mud4uh-30-GlHhdYLUg7ju5UAULSV1q0NE0Wrnu3RkEfHq-bQsyIw-lCORVdItpQPaM1Yqh7Mlu1ZRFevXZzXbVfRNUVlJab8lK-GCnEhdWYhEGef0xZWaCkBQCPrWvFt7U3DQUW~N39pDu-W62zATjEI5H8weIBmZySoS7ld8JJHNoZ~KRFmJwMZ~vay8M9zJZLeiRO8~3Z29cer7cu1n7PWnH2IdAgSUN6NGoLNlAwxp8MJUVjiUx9vdSEaj5zwY9CII2P1JOFiUJHY8pRgIGGQLQdF1uHKlUcELu9skGdb8-~LToWZ5KjifePPIlv7WMkw03h8gvh~iZhhNJb6A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/13/5">
                        <div class="volume trailing-comma">Volume 13</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
2006        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87195060">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195060">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195062">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195062">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195067">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195067">2. Data and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195072">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195072">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195104">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195104">REFERENCES</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/13/5/185/400440">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/13/5/205/401161">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Pattern and Rate of Indel Evolution Inferred from Whole Chloroplast Intergenic Regions in Sugarcane, Maize and Rice
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kyoko Yamane</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Laboratory of Crop evolution, Graduate School of Agriculture, Kyoto University</div><div class="addr-line">Nakajoh, Mozume, Mukoh 617-0001, Japan</div></div><div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Plant resource Laboratory, Graduate School and School of Life and Environmental, Osaka Prefectural University</div><div class="addr-line">1-1 Gakuen-cho, Nakaku, Sakai, Osaka 599-8531, Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <sup>*</sup>To whom correspondence should be addressed. Tel. +81-75-921-0652, Fax. +81-75-932-8063, E-mail: <a href="mailto:m52297@mbox.kudpc.kyoto-u.ac.jp" target="_blank">m52297@mbox.kudpc.kyoto-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kyoko+Yamane">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamane K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamane K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kyoko Yamane</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kentaro Yano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><sup>3</sup><div class="insititution">Plant Breeding Laboratory, Graduate School of Agriculture, Kyoto University</div><div class="addr-line">Nakajoh, Oiwake-cho, Kitashirakawa, Sakyo-ku, Kyoto 606-8502, Japan</div></div><div class="aff"><div class="label -label">4</div><sup>4</sup><div class="insititution">Kazusa DNA Research Institute</div><div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kentaro+Yano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yano K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yano K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kentaro Yano</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Taihachi Kawahara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Laboratory of Crop evolution, Graduate School of Agriculture, Kyoto University</div><div class="addr-line">Nakajoh, Mozume, Mukoh 617-0001, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Taihachi+Kawahara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawahara T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kawahara T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Taihachi Kawahara</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Communicated by Masahiro Yano</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 13, Issue 5, 1 January 2006, Pages 197–204, <a href='https://doi.org/10.1093/dnares/dsl012'>https://doi.org/10.1093/dnares/dsl012</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 January 2006</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">03 August 2006</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">19 October 2006</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="400871" href="/dnaresearch/article-pdf/13/5/197/17462597/dsl012.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Kyoko Yamane, Kentaro Yano, Taihachi Kawahara;  Pattern and Rate of Indel Evolution Inferred from Whole Chloroplast Intergenic Regions in Sugarcane, Maize and Rice, <em>DNA Research</em>, Volume 13, Issue 5, 1 January 2006, Pages 197–204, <a href='https://doi.org/10.1093/dnares/dsl012'>https://doi.org/10.1093/dnares/dsl012</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/400871?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/400871?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/400871?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/400871?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/400871?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/400871?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/400871?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Pattern%20and%20Rate%20of%20Indel%20Evolution%20Inferred%20from%20Whole%20Chloroplast%20Intergenic%20Regions%20in%20Sugarcane%2C%20Maize%20and%20Rice&amp;publicationDate=2006-01-01&amp;volumeNum=13&amp;issueNum=5&amp;author=Yamane%2C%20Kyoko%3B%20Yano%2C%20Kentaro&amp;startPage=197&amp;endPage=204&amp;contentId=10.1093%2Fdnares%2Fdsl012&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f13%2f5%2f197%2f400871" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87195060 id="87195060" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Microstructural changes such as insertions and deletions (=indels) are a major driving force in the evolution of non-coding DNA sequences. To better understand the mechanisms by which indel mutations arise, as well as the molecular evolution of non-coding regions, the number and pattern of indels and nucleotide substitutions were compared in the whole chloroplast genomes. Comparisons were made for a total of over 38 kb non-coding DNA sequences from 126 intergenic regions in two data sets representing species with different divergence times: sugarcane and maize and <em>Oryza sativa</em> var. <em>indica</em> and <em>japonica</em>. The main findings of this study are: (i) Approximately half of all indels are single nucleotide indels. This observation agrees with previous studies in various organisms. (ii) The distribution and number of indels was different between two data sets, and different patterns were observed for tandem repeat and non-repeat indels. (iii) Distribution pattern of tandem repeat indels showed statistically significant bias towards A/T-rich. (iv) The rate of indel mutation was estimated to be ≈0.8 ± 0.04 × 10<sup>−9</sup> per site per year, which was similar to previous estimates in other organisms. (v) The frequencies of nucleotide substitutions and indels were significantly lower in inverted repeat (IR).</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;chloroplast genome&quot;">chloroplast genome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;evolutionary rate&quot;">evolutionary rate</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="indels">indels</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;non-coding DNA sequence&quot;">non-coding DNA sequence</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;intergenic region&quot;">intergenic region</a></div>                            <h2 scrollto-destination=87195062 id="87195062" class="section-title" >1. Introduction</h2>
<p>Chloroplast non-coding regions evolve through accumulated nucleotide substitutions and microstructural mutations such as insertions and deletions (=indels, represented by gaps) and inversions. Indels and inversions are thought to be a major driving force in sequence evolution,<a reveal-id="b1" data-open="b1" class="link link-ref link-reveal xref-bibr">1</a> and Kelchner,<a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a> who recently reviewed the evolution of chloroplast non-coding sequences, suggested that microstructural mutations would not be randomly distributed throughout chloroplast genomes, because the frequency at which such mutations arise often depends strongly on sequence context. Kelchner<a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a> claimed, therefore, that it is essential to understand the evolutionary features of microstructural mutations in order to use them in phylogenetic or other evolutionary studies.</p><p>Ingvarsson et al.<a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a> studied indel evolution in three chloroplast intergenic regions (<em>trnL-trnF</em>, <em>trnH-psbA</em> and <em>trnG-trnS</em>) of two <em>Silene</em> species to assess the phylogenetic utility of indels. Hamilton et al.<a reveal-id="b4" data-open="b4" class="link link-ref link-reveal xref-bibr">4</a> also investigated the frequency and the distribution of indels in six chloroplast intergenic regions (<em>psbB</em>–<em>psbH</em>, <em>atpB</em>–<em>rbcL</em>, <em>trnL</em>–<em>trnH</em>, <em>rpl20</em>–<em>5'rps12</em>, <em>trnS</em>–<em>trnG</em> and <em>trnH</em>–<em>psbA</em>) from Lecythidaceae. These data suggest that chloroplast indels behave as neutral variants that arise randomly. However, it is still unsettled whether indels have a uniformly random distribution throughout chloroplast genome or whether indel frequencies are influenced by base composition or other DNA sequence-dependent influences. Thus, a comprehensive study is needed of sequence context-dependent variation in the frequency of microstructural mutations in whole chloroplast genomes.</p><p>More than 30 chloroplast genomes have been completely sequenced from eukaryotic algae and land plants including the three economically important Poaceae crop plants rice<a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a>, maize<a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> and wheat<a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a>. Using these three chloroplast genome sequences, Matsuoka et al.<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> first conducted a comparative study of the nucleotide sequences of 106 chloroplast genes. In this study of coding regions, sequence alignments were unambiguous; however, the alignment of non-coding chloroplast sequences can be much more difficult and ambiguous, resulting in multiple alternative alignments and different patterns of indels. For this reason, there has been no large scale comparative study of entire non-coding regions. Thus, it is desirable to study closely related pairs of non-coding sequences, for which an unambiguous alignment can be made. Recently, the whole chloroplast genome sequence of a fourth grass plant, sugarcane, was determined.<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a> Both sugarcane and maize belong to the subtribe Saccharinae of the tribe Andropogoneae, Poaceae, and are phylogenetically very close.<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> For example, Asano et al.<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a> confirmed that the number and order of functional chloroplast genes are identical throughout the chloroplast genomes of sugarcane and maize. More recently, Tang et al.<a reveal-id="b12" data-open="b12" class="link link-ref link-reveal xref-bibr">12</a> determined the complete chloroplast genome sequence of cultivated rice var. <em>indica</em> and <em>japonica</em>. These data provided a good opportunity to conduct a comparative analysis of non-coding sequences in chloroplast genomes.</p><p>This study presents comparative analyses of number of indels and the distribution in two sequence data sets: data set (a) was the intergenic regions of the chloroplast genomes of sugarcane (<em>Saccharum offcinarum</em>) and maize (<em>Zea mays</em>); data set (b) includes <em>Oryza sativa</em> var. <em>indica</em> and var. <em>japonica</em>. These two data sets were chosen in part because the divergence time of DNA sequences in each data group is markedly different. Mutation frequencies were analyzed in &gt;38 kb non-coding DNA sequences from 126 intergenic regions in the chloroplast genomes of these species. The number of indels was counted in each intergenic region independently, and their frequencies and distribution patterns in chloroplast genomes were calculated en masse, based on the idea that a single linear linkage profile is associated with uniparental inheritance of chloroplast DNA.<a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> Furthermore, the evolutionary rate of indels in the chloroplast intergenic region was estimated. Based on these findings and previous reports, the evolutionary implications of indels are discussed.</p>                            <h2 scrollto-destination=87195067 id="87195067" class="section-title" >2. Data and methods</h2>
<p>The entire chloroplast genome sequence from sugarcane (<em>Saccharum offcinarum</em>, DDBJ/EMBL/GenBank accession no. AP006714), maize (<em>Zea mays</em>, X86563) and rice (<em>Oryza sativa</em> var. <em>japonica</em>; AY522331 and var. <em>indica</em>; AY522329)<a reveal-id="b12" data-open="b12" class="link link-ref link-reveal xref-bibr">12</a> were obtained from DDBJ. In this study, data for <em>O. sativa</em> var. <em>japonica</em> (GenBank accession no. X15901)<a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> was not analyzed, because Tang et al.<a reveal-id="b12" data-open="b12" class="link link-ref link-reveal xref-bibr">12</a> noticed that this accession is somewhat anomalous and may be a variant of the Nipponbare chloroplast.</p><p>Multiple sequences were aligned by Clustal W<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> using default settings which were adjusted manually. Indels were not classified as insertions or deletions, because an outgroup was not included in the data sets. The chloroplast genome is comprised of two large inverted repeat (IR) regions as well as large single copy (LSC) and small single copy (SSC) regions.<a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a> All mutations in IR, LSC and SSC were analyzed separately. Pseudogenes and their adjacent regions were excluded from all analyses, because the borders of pseudogenes tend to be ambiguous. Indel sequences were only analyzed when homologous genes were identified on both sides of the specific intergenic region. In vascular plants, introns are difficult to identify because most chloroplast genes are expressed as polycistronic transcription units (reviewed by Sugita and Sugiura<a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a>). In this study, therefore, intronic regions were not analyzed. The sequence alignments of multiple chloroplast genomes used in this study are available upon request.</p><p>DnaSP version 3.00<a reveal-id="b17" data-open="b17" class="link link-ref link-reveal xref-bibr">17</a> was used to estimate the number of net nucleotide substitutions between two populations (<em>D</em><sub>XY</sub>)<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> in aligned sequence. Indels were classified into tandem repeat or non-repeat indels, according to Ingvarsson et al.<a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a> (see Supplementary Material online Table 1-1 and 1-2). Tandem repeat indels include both perfect and imperfect repeats of a sequence motif ≥2 bp in length.</p>                            <h2 scrollto-destination=87195072 id="87195072" class="section-title" >3. Results and discussion</h2>
<p>Accurate DNA sequence alignments are essential when analyzing the evolution of DNA coding or non-coding sequences. In the present study, ambiguous sequence alignment of chloroplast non-coding regions and incorrect indel assignments were avoided by selecting input DNA sequences from plants belonging to closely related taxa. For the selected plant species and DNA regions, intergenic chloroplast DNA sequences were aligned without difficulty, and it was not necessary to exclude any DNA sequence regions due to ambiguity in the sequence alignment. This study analyzes indels and nucleotide substitutions in two data sets: data set (a) consisted of chloroplast DNA from sugarcane and maize; and data set (b) consisted of chloroplast DNA from <em>O. sativa</em> var. <em>indica</em> and <em>japonica</em>. The number and positions of all mutations in each data set were determined and are listed in Supplementary Tables 1-1 and 1-2, respectively. <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a> summarizes the number of nucleotide substitutions, tandem repeat indels, non-repeat indels and the mean and median indel length for each data set.</p>                        <a id="87195074" scrollto-destination="87195074"></a>
<div content-id="tbl1" class="table-modal"><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Number of substitutions and indels in whole chloroplast intergenic regions in sugarcane and maize [data set (a)] or <em>Oryza sativa</em> var. j<em>aponica</em> and var. <em>indica</em> [data set (b)].</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="3">Intergenic regions </th></tr><tr><th> </th><th colspan="3"><hr /> </th></tr><tr><th> </th><th>SC </th><th>IR </th><th>Total </th></tr><tr><th> </th><th>(LSC + SSC) </th><th>(a + b) </th><th> </th></tr></thead><tbody><tr><td colspan="4">(a) Sugarcane and maize </td></tr><tr><td>    Total length of intergenic regions (bp) </td><td>29 832 </td><td>8431 </td><td>38 263 </td></tr><tr><td>    Number of intergenic regions </td><td>82 </td><td>44 </td><td>126 </td></tr><tr><td>    Number of substitutions </td><td>354 (1.19) </td><td>19 (0.23) </td><td>373 (0.97) </td></tr><tr><td>    Total number of indels </td><td>178 (0.60) </td><td>23 (0.28) </td><td>201 (0.53) </td></tr><tr><td>         Mean length </td><td>7.6 </td><td>3.0 </td><td>7.1 </td></tr><tr><td>         Median length </td><td>2 </td><td>1 </td><td>2 </td></tr><tr><td>         % of single nucleotide indel </td><td>40.5 </td><td>78 </td><td>45 </td></tr><tr><td>         Maximum length of indel (bp) </td><td>264 </td><td>18 </td><td>— </td></tr><tr><td>    Number of tandem repeat indels </td><td>91 </td><td>13 </td><td>104 </td></tr><tr><td>         % of GC tandem repeat </td><td>16.5 </td><td>38.5 </td><td>19.2 </td></tr><tr><td>         Mean length </td><td>2.0 </td><td>1 </td><td>1.9 </td></tr><tr><td>        Median length </td><td>1 </td><td>1 </td><td>1 </td></tr><tr><td>         % of single nucleotide indel </td><td>60.4 </td><td>100 </td><td>65.4 </td></tr><tr><td>         Maximum length of indel (bp) </td><td>7 </td><td>1 </td><td>7 </td></tr><tr><td>    Number of non-repeat indels </td><td>87 </td><td>10 </td><td>97 </td></tr><tr><td>        Mean length </td><td>13.5 </td><td>5.6 </td><td>12.6 </td></tr><tr><td>        Median length </td><td>5 </td><td>3 </td><td>5 </td></tr><tr><td>        % of single nucleotide indel </td><td>19.5 </td><td>50 </td><td>22.7 </td></tr><tr><td>        Maximum length of indel (bp) </td><td>264 </td><td>18 </td><td>264 </td></tr><tr><td colspan="4">(b) <em>japonica</em> and <em>indica</em> </td></tr><tr><td>    Total length of intergenic regions (bp) </td><td>25 730 </td><td>11 376 </td><td>37 106 </td></tr><tr><td>    Number of intergenic regions </td><td>86 </td><td>50 </td><td>136 </td></tr><tr><td>    Number of substitutions </td><td>22 (0.086) </td><td>0 (0) </td><td>22 (0.086) </td></tr><tr><td>    Total number of indels </td><td>16 (0.069) </td><td>0 (0) </td><td>16 (0.069) </td></tr><tr><td>        Mean length </td><td>4.4 </td><td>— </td><td>4.4 </td></tr><tr><td>        Median length </td><td>1.5 </td><td>— </td><td>1.5 </td></tr><tr><td>        % of single nucleotide indel </td><td>50 </td><td>0 </td><td>50 </td></tr><tr><td>        Maximum length of indel (bp) </td><td>32 </td><td>— </td><td>32 </td></tr><tr><td>    Number of tandem repeat indels </td><td>14 </td><td>0 </td><td>14 </td></tr><tr><td>        Mean length </td><td>2.4 </td><td>— </td><td>2.4 </td></tr><tr><td>        Median length </td><td>1 </td><td>— </td><td>1 </td></tr><tr><td>        % of single nucleotide indel </td><td>57 </td><td>0 </td><td>57 </td></tr><tr><td>        Maximum length of indel (bp) </td><td>6 </td><td>— </td><td>6 </td></tr><tr><td>    Number of non-repeat indels </td><td>2 </td><td>0 </td><td>2 </td></tr><tr><td>        Mean length </td><td>18.5 </td><td>— </td><td>18.5 </td></tr><tr><td>        Median length </td><td>— </td><td>— </td><td>— </td></tr><tr><td>        % of single nucleotide indel </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>        Maximum length of indel (bp) </td><td>32 </td><td>— </td><td>32 </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="3">Intergenic regions </th></tr><tr><th> </th><th colspan="3"><hr /> </th></tr><tr><th> </th><th>SC </th><th>IR </th><th>Total </th></tr><tr><th> </th><th>(LSC + SSC) </th><th>(a + b) </th><th> </th></tr></thead><tbody><tr><td colspan="4">(a) Sugarcane and maize </td></tr><tr><td>    Total length of intergenic regions (bp) </td><td>29 832 </td><td>8431 </td><td>38 263 </td></tr><tr><td>    Number of intergenic regions </td><td>82 </td><td>44 </td><td>126 </td></tr><tr><td>    Number of substitutions </td><td>354 (1.19) </td><td>19 (0.23) </td><td>373 (0.97) </td></tr><tr><td>    Total number of indels </td><td>178 (0.60) </td><td>23 (0.28) </td><td>201 (0.53) </td></tr><tr><td>         Mean length </td><td>7.6 </td><td>3.0 </td><td>7.1 </td></tr><tr><td>         Median length </td><td>2 </td><td>1 </td><td>2 </td></tr><tr><td>         % of single nucleotide indel </td><td>40.5 </td><td>78 </td><td>45 </td></tr><tr><td>         Maximum length of indel (bp) </td><td>264 </td><td>18 </td><td>— </td></tr><tr><td>    Number of tandem repeat indels </td><td>91 </td><td>13 </td><td>104 </td></tr><tr><td>         % of GC tandem repeat </td><td>16.5 </td><td>38.5 </td><td>19.2 </td></tr><tr><td>         Mean length </td><td>2.0 </td><td>1 </td><td>1.9 </td></tr><tr><td>        Median length </td><td>1 </td><td>1 </td><td>1 </td></tr><tr><td>         % of single nucleotide indel </td><td>60.4 </td><td>100 </td><td>65.4 </td></tr><tr><td>         Maximum length of indel (bp) </td><td>7 </td><td>1 </td><td>7 </td></tr><tr><td>    Number of non-repeat indels </td><td>87 </td><td>10 </td><td>97 </td></tr><tr><td>        Mean length </td><td>13.5 </td><td>5.6 </td><td>12.6 </td></tr><tr><td>        Median length </td><td>5 </td><td>3 </td><td>5 </td></tr><tr><td>        % of single nucleotide indel </td><td>19.5 </td><td>50 </td><td>22.7 </td></tr><tr><td>        Maximum length of indel (bp) </td><td>264 </td><td>18 </td><td>264 </td></tr><tr><td colspan="4">(b) <em>japonica</em> and <em>indica</em> </td></tr><tr><td>    Total length of intergenic regions (bp) </td><td>25 730 </td><td>11 376 </td><td>37 106 </td></tr><tr><td>    Number of intergenic regions </td><td>86 </td><td>50 </td><td>136 </td></tr><tr><td>    Number of substitutions </td><td>22 (0.086) </td><td>0 (0) </td><td>22 (0.086) </td></tr><tr><td>    Total number of indels </td><td>16 (0.069) </td><td>0 (0) </td><td>16 (0.069) </td></tr><tr><td>        Mean length </td><td>4.4 </td><td>— </td><td>4.4 </td></tr><tr><td>        Median length </td><td>1.5 </td><td>— </td><td>1.5 </td></tr><tr><td>        % of single nucleotide indel </td><td>50 </td><td>0 </td><td>50 </td></tr><tr><td>        Maximum length of indel (bp) </td><td>32 </td><td>— </td><td>32 </td></tr><tr><td>    Number of tandem repeat indels </td><td>14 </td><td>0 </td><td>14 </td></tr><tr><td>        Mean length </td><td>2.4 </td><td>— </td><td>2.4 </td></tr><tr><td>        Median length </td><td>1 </td><td>— </td><td>1 </td></tr><tr><td>        % of single nucleotide indel </td><td>57 </td><td>0 </td><td>57 </td></tr><tr><td>        Maximum length of indel (bp) </td><td>6 </td><td>— </td><td>6 </td></tr><tr><td>    Number of non-repeat indels </td><td>2 </td><td>0 </td><td>2 </td></tr><tr><td>        Mean length </td><td>18.5 </td><td>— </td><td>18.5 </td></tr><tr><td>        Median length </td><td>— </td><td>— </td><td>— </td></tr><tr><td>        % of single nucleotide indel </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>        Maximum length of indel (bp) </td><td>32 </td><td>— </td><td>32 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Numbers in the parentheses indicate the values divided by total length of intergenic region (%).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195074" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Number of substitutions and indels in whole chloroplast intergenic regions in sugarcane and maize [data set (a)] or <em>Oryza sativa</em> var. j<em>aponica</em> and var. <em>indica</em> [data set (b)].</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="3">Intergenic regions </th></tr><tr><th> </th><th colspan="3"><hr /> </th></tr><tr><th> </th><th>SC </th><th>IR </th><th>Total </th></tr><tr><th> </th><th>(LSC + SSC) </th><th>(a + b) </th><th> </th></tr></thead><tbody><tr><td colspan="4">(a) Sugarcane and maize </td></tr><tr><td>    Total length of intergenic regions (bp) </td><td>29 832 </td><td>8431 </td><td>38 263 </td></tr><tr><td>    Number of intergenic regions </td><td>82 </td><td>44 </td><td>126 </td></tr><tr><td>    Number of substitutions </td><td>354 (1.19) </td><td>19 (0.23) </td><td>373 (0.97) </td></tr><tr><td>    Total number of indels </td><td>178 (0.60) </td><td>23 (0.28) </td><td>201 (0.53) </td></tr><tr><td>         Mean length </td><td>7.6 </td><td>3.0 </td><td>7.1 </td></tr><tr><td>         Median length </td><td>2 </td><td>1 </td><td>2 </td></tr><tr><td>         % of single nucleotide indel </td><td>40.5 </td><td>78 </td><td>45 </td></tr><tr><td>         Maximum length of indel (bp) </td><td>264 </td><td>18 </td><td>— </td></tr><tr><td>    Number of tandem repeat indels </td><td>91 </td><td>13 </td><td>104 </td></tr><tr><td>         % of GC tandem repeat </td><td>16.5 </td><td>38.5 </td><td>19.2 </td></tr><tr><td>         Mean length </td><td>2.0 </td><td>1 </td><td>1.9 </td></tr><tr><td>        Median length </td><td>1 </td><td>1 </td><td>1 </td></tr><tr><td>         % of single nucleotide indel </td><td>60.4 </td><td>100 </td><td>65.4 </td></tr><tr><td>         Maximum length of indel (bp) </td><td>7 </td><td>1 </td><td>7 </td></tr><tr><td>    Number of non-repeat indels </td><td>87 </td><td>10 </td><td>97 </td></tr><tr><td>        Mean length </td><td>13.5 </td><td>5.6 </td><td>12.6 </td></tr><tr><td>        Median length </td><td>5 </td><td>3 </td><td>5 </td></tr><tr><td>        % of single nucleotide indel </td><td>19.5 </td><td>50 </td><td>22.7 </td></tr><tr><td>        Maximum length of indel (bp) </td><td>264 </td><td>18 </td><td>264 </td></tr><tr><td colspan="4">(b) <em>japonica</em> and <em>indica</em> </td></tr><tr><td>    Total length of intergenic regions (bp) </td><td>25 730 </td><td>11 376 </td><td>37 106 </td></tr><tr><td>    Number of intergenic regions </td><td>86 </td><td>50 </td><td>136 </td></tr><tr><td>    Number of substitutions </td><td>22 (0.086) </td><td>0 (0) </td><td>22 (0.086) </td></tr><tr><td>    Total number of indels </td><td>16 (0.069) </td><td>0 (0) </td><td>16 (0.069) </td></tr><tr><td>        Mean length </td><td>4.4 </td><td>— </td><td>4.4 </td></tr><tr><td>        Median length </td><td>1.5 </td><td>— </td><td>1.5 </td></tr><tr><td>        % of single nucleotide indel </td><td>50 </td><td>0 </td><td>50 </td></tr><tr><td>        Maximum length of indel (bp) </td><td>32 </td><td>— </td><td>32 </td></tr><tr><td>    Number of tandem repeat indels </td><td>14 </td><td>0 </td><td>14 </td></tr><tr><td>        Mean length </td><td>2.4 </td><td>— </td><td>2.4 </td></tr><tr><td>        Median length </td><td>1 </td><td>— </td><td>1 </td></tr><tr><td>        % of single nucleotide indel </td><td>57 </td><td>0 </td><td>57 </td></tr><tr><td>        Maximum length of indel (bp) </td><td>6 </td><td>— </td><td>6 </td></tr><tr><td>    Number of non-repeat indels </td><td>2 </td><td>0 </td><td>2 </td></tr><tr><td>        Mean length </td><td>18.5 </td><td>— </td><td>18.5 </td></tr><tr><td>        Median length </td><td>— </td><td>— </td><td>— </td></tr><tr><td>        % of single nucleotide indel </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>        Maximum length of indel (bp) </td><td>32 </td><td>— </td><td>32 </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="3">Intergenic regions </th></tr><tr><th> </th><th colspan="3"><hr /> </th></tr><tr><th> </th><th>SC </th><th>IR </th><th>Total </th></tr><tr><th> </th><th>(LSC + SSC) </th><th>(a + b) </th><th> </th></tr></thead><tbody><tr><td colspan="4">(a) Sugarcane and maize </td></tr><tr><td>    Total length of intergenic regions (bp) </td><td>29 832 </td><td>8431 </td><td>38 263 </td></tr><tr><td>    Number of intergenic regions </td><td>82 </td><td>44 </td><td>126 </td></tr><tr><td>    Number of substitutions </td><td>354 (1.19) </td><td>19 (0.23) </td><td>373 (0.97) </td></tr><tr><td>    Total number of indels </td><td>178 (0.60) </td><td>23 (0.28) </td><td>201 (0.53) </td></tr><tr><td>         Mean length </td><td>7.6 </td><td>3.0 </td><td>7.1 </td></tr><tr><td>         Median length </td><td>2 </td><td>1 </td><td>2 </td></tr><tr><td>         % of single nucleotide indel </td><td>40.5 </td><td>78 </td><td>45 </td></tr><tr><td>         Maximum length of indel (bp) </td><td>264 </td><td>18 </td><td>— </td></tr><tr><td>    Number of tandem repeat indels </td><td>91 </td><td>13 </td><td>104 </td></tr><tr><td>         % of GC tandem repeat </td><td>16.5 </td><td>38.5 </td><td>19.2 </td></tr><tr><td>         Mean length </td><td>2.0 </td><td>1 </td><td>1.9 </td></tr><tr><td>        Median length </td><td>1 </td><td>1 </td><td>1 </td></tr><tr><td>         % of single nucleotide indel </td><td>60.4 </td><td>100 </td><td>65.4 </td></tr><tr><td>         Maximum length of indel (bp) </td><td>7 </td><td>1 </td><td>7 </td></tr><tr><td>    Number of non-repeat indels </td><td>87 </td><td>10 </td><td>97 </td></tr><tr><td>        Mean length </td><td>13.5 </td><td>5.6 </td><td>12.6 </td></tr><tr><td>        Median length </td><td>5 </td><td>3 </td><td>5 </td></tr><tr><td>        % of single nucleotide indel </td><td>19.5 </td><td>50 </td><td>22.7 </td></tr><tr><td>        Maximum length of indel (bp) </td><td>264 </td><td>18 </td><td>264 </td></tr><tr><td colspan="4">(b) <em>japonica</em> and <em>indica</em> </td></tr><tr><td>    Total length of intergenic regions (bp) </td><td>25 730 </td><td>11 376 </td><td>37 106 </td></tr><tr><td>    Number of intergenic regions </td><td>86 </td><td>50 </td><td>136 </td></tr><tr><td>    Number of substitutions </td><td>22 (0.086) </td><td>0 (0) </td><td>22 (0.086) </td></tr><tr><td>    Total number of indels </td><td>16 (0.069) </td><td>0 (0) </td><td>16 (0.069) </td></tr><tr><td>        Mean length </td><td>4.4 </td><td>— </td><td>4.4 </td></tr><tr><td>        Median length </td><td>1.5 </td><td>— </td><td>1.5 </td></tr><tr><td>        % of single nucleotide indel </td><td>50 </td><td>0 </td><td>50 </td></tr><tr><td>        Maximum length of indel (bp) </td><td>32 </td><td>— </td><td>32 </td></tr><tr><td>    Number of tandem repeat indels </td><td>14 </td><td>0 </td><td>14 </td></tr><tr><td>        Mean length </td><td>2.4 </td><td>— </td><td>2.4 </td></tr><tr><td>        Median length </td><td>1 </td><td>— </td><td>1 </td></tr><tr><td>        % of single nucleotide indel </td><td>57 </td><td>0 </td><td>57 </td></tr><tr><td>        Maximum length of indel (bp) </td><td>6 </td><td>— </td><td>6 </td></tr><tr><td>    Number of non-repeat indels </td><td>2 </td><td>0 </td><td>2 </td></tr><tr><td>        Mean length </td><td>18.5 </td><td>— </td><td>18.5 </td></tr><tr><td>        Median length </td><td>— </td><td>— </td><td>— </td></tr><tr><td>        % of single nucleotide indel </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td>        Maximum length of indel (bp) </td><td>32 </td><td>— </td><td>32 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Numbers in the parentheses indicate the values divided by total length of intergenic region (%).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195074" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87195077 id="87195077" class="section-title" >3.1. Dominance of single nucleotide indels in non-coding regions</h3>
<p>The distribution of indels of different length was determined in LSC and SSC regions for data set (a) and the results are presented graphically in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1A</a>. The number of indels decreases rapidly as indel length increases, such that short indels, especially 1 bp indels, are highly enriched in this data set (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1A and B</a>). A similar pattern was reported previously by Hamilton et al.<a reveal-id="b4" data-open="b4" class="link link-ref link-reveal xref-bibr">4</a>, who compared six chloroplast intergenic regions in species of Lecythidaceae. A strong linear relationship (<em>R</em><sup>2</sup> = 0.889, <em>P</em> &lt; 0.001) was detected in a plot of ln (indel number) versus ln (indel length). These results agree with a previous study in which indel size distribution was analyzed in human and chimpanzee processed pseudogenes.<a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> Furthermore, in the present study, single nucleotide indels represented 41% of total indels in LSC + SSC regions. Similar estimates have been reported previously in the following studies: 50% single nucleotide indels in human and rodent pseudogenes,<a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> 45% in the δ-globin region and 43% in the δ-globin spacer region,<a reveal-id="b20" data-open="b20" class="link link-ref link-reveal xref-bibr">20</a> 50% in rodent processed pseudogenes<a reveal-id="b21" data-open="b21" class="link link-ref link-reveal xref-bibr">21</a> and 41% in plant non-coding DNA.<a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> Thus, we noticed that single nucleotide indels are the predominant type of indel occurring between recently diverged DNA sequences from various organisms.</p>                        <a id="87195079" scrollto-destination="87195079"></a>
<div id="fig1" data-id="fig1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl012/1/m_dsl012f1.jpeg?Expires=2147483647&Signature=f3hyIkilx10NmmPntME3w0goOwwgtu-l2BUnm01hecPeQiNBoGZowqxc-8TPwk7-WIxpUJOrtkJW1MOj0p6bZjMZsLevLVsXILQ3vKof9VRWDfGW3MiUcW1a0SmW3tD6wpCAcq0pNikv~AcsK0cYRVaAhW3cTmYqO6Op4w2FV8sJ2Y0qbK3o16Nn~rpRQgrQ7Xtd9NNoxx6gQQ3wHvG78p26SZLrHRnEtu10zH2HX4Ju-4DmztEXqUcN8-nA~oZGsYCr4BaYkddLKQYlEa~dabJbJALt9LqivB8mwsfHOtr1k~1yUU1skUoFvrUGXU47exlP-7IM3euFJEGb5O7tAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of indels and length was counted in SC (LSC + SSC) regions in data set (a). [Note that there were too few indels for analysis in data set (b) or the IR (a + b) region.] (A) The distribution of indel length (= size) in the chloroplast intergenic regions. Number of indels (nk) is plotted versus indel length (k). The percent of single nucleotide indels is shown at top. The number of long indels was low, so the x-axis is truncated at 26 bp. (B) Relationship between indel length (bp) and number in SC (LSC + SSC) regions. Values are shown as a log–log plot.19 The dashed line shows the best-fit to the data truncated at 95% of total indel events." path-from-xml="dsl012f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195079/dsl012f1.jpeg" path-from-xml="dsl012f1.jpeg" target="_blank">View large</a><a section="87195079" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl012/1/dsl012f1.jpeg?Expires=2147483647&Signature=oZ96bR94JvOGPZukIE2CSa4FAgCSJjznjWjlT~XbVpkbVLOcXwcfUw4oZEK3cYwpIt9nz8Ojpg2QldX~BL3XxX6G5~vJRKa71tgW09YKB7HZ5QF53Jmdy1gzJFO-682Rd3ABpsdTlUgcL9jvQpQkEm0EKuxh6Ogytd4wD5E3DX-tbKd6uEGRDprOxzU0hF6aqHp7~cQ3nLvWLA-gArFbH5RRVaPw59GMveSktYHSS2cUNMWIU61j1RuztvWFiZC5jNjzXN1Gppqcz1k3hQz-NawOsW720KSW2GZ~mrUdQUeZ~-NwFKEavUYTXtnFppuUlKfBG8qkD~wXYk98WnQWMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195079&ar=400871&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl012f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of indels and length was counted in SC (LSC + SSC) regions in data set (a). [Note that there were too few indels for analysis in data set (b) or the IR (a + b) region.] (A) The distribution of indel length (= size) in the chloroplast intergenic regions. Number of indels (<em>n<sub>k</sub></em>) is plotted versus indel length (<em>k</em>). The percent of single nucleotide indels is shown at top. The number of long indels was low, so the <em>x</em>-axis is truncated at 26 bp. (B) Relationship between indel length (bp) and number in SC (LSC + SSC) regions. Values are shown as a log–log plot.<a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> The dashed line shows the best-fit to the data truncated at 95% of total indel events.</p></div></div><div content-id="fig1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl012/1/m_dsl012f1.jpeg?Expires=2147483647&Signature=f3hyIkilx10NmmPntME3w0goOwwgtu-l2BUnm01hecPeQiNBoGZowqxc-8TPwk7-WIxpUJOrtkJW1MOj0p6bZjMZsLevLVsXILQ3vKof9VRWDfGW3MiUcW1a0SmW3tD6wpCAcq0pNikv~AcsK0cYRVaAhW3cTmYqO6Op4w2FV8sJ2Y0qbK3o16Nn~rpRQgrQ7Xtd9NNoxx6gQQ3wHvG78p26SZLrHRnEtu10zH2HX4Ju-4DmztEXqUcN8-nA~oZGsYCr4BaYkddLKQYlEa~dabJbJALt9LqivB8mwsfHOtr1k~1yUU1skUoFvrUGXU47exlP-7IM3euFJEGb5O7tAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of indels and length was counted in SC (LSC + SSC) regions in data set (a). [Note that there were too few indels for analysis in data set (b) or the IR (a + b) region.] (A) The distribution of indel length (= size) in the chloroplast intergenic regions. Number of indels (nk) is plotted versus indel length (k). The percent of single nucleotide indels is shown at top. The number of long indels was low, so the x-axis is truncated at 26 bp. (B) Relationship between indel length (bp) and number in SC (LSC + SSC) regions. Values are shown as a log–log plot.19 The dashed line shows the best-fit to the data truncated at 95% of total indel events." path-from-xml="dsl012f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195079/dsl012f1.jpeg" path-from-xml="dsl012f1.jpeg" target="_blank">View large</a><a section="87195079" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl012/1/dsl012f1.jpeg?Expires=2147483647&Signature=oZ96bR94JvOGPZukIE2CSa4FAgCSJjznjWjlT~XbVpkbVLOcXwcfUw4oZEK3cYwpIt9nz8Ojpg2QldX~BL3XxX6G5~vJRKa71tgW09YKB7HZ5QF53Jmdy1gzJFO-682Rd3ABpsdTlUgcL9jvQpQkEm0EKuxh6Ogytd4wD5E3DX-tbKd6uEGRDprOxzU0hF6aqHp7~cQ3nLvWLA-gArFbH5RRVaPw59GMveSktYHSS2cUNMWIU61j1RuztvWFiZC5jNjzXN1Gppqcz1k3hQz-NawOsW720KSW2GZ~mrUdQUeZ~-NwFKEavUYTXtnFppuUlKfBG8qkD~wXYk98WnQWMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195079&ar=400871&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl012f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of indels and length was counted in SC (LSC + SSC) regions in data set (a). [Note that there were too few indels for analysis in data set (b) or the IR (a + b) region.] (A) The distribution of indel length (= size) in the chloroplast intergenic regions. Number of indels (<em>n<sub>k</sub></em>) is plotted versus indel length (<em>k</em>). The percent of single nucleotide indels is shown at top. The number of long indels was low, so the <em>x</em>-axis is truncated at 26 bp. (B) Relationship between indel length (bp) and number in SC (LSC + SSC) regions. Values are shown as a log–log plot.<a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> The dashed line shows the best-fit to the data truncated at 95% of total indel events.</p></div></div>                            <h3 scrollto-destination=87195081 id="87195081" class="section-title" >3.2. Number and distribution pattern of indels in chloroplast intergenic regions</h3>
<p>The low level of variation between <em>O. sativa</em> var. <em>indica</em> and <em>japonica</em> was previously reported by Tang et al.<a reveal-id="b12" data-open="b12" class="link link-ref link-reveal xref-bibr">12</a> In this study, a large number of mutations were identified in data set (a) (201 indels and 373 base substitutions), while ∼10-fold fewer mutations were identified in data set (b) (16 indels and 22 base substitutions), even though the total length of the intergenic sequences in the two data sets were very similar. In addition, in data set (a), the length distribution of indels was 1–264 bp, whereas which was 1–32 bp in data set (b). These different patterns of indel events between data sets (a) and (b) could be simply associated with the different level of divergence time. Interestingly, Golengerg et al.<a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> has reported in a study of chloroplast non-coding DNA sequences in monocot crops that the relative proportion of indels in the mutation spectrum is lower in more highly diverged species than in closely related species. In data set (a), the number of indels was approximately half (50.3%) the number of nucleotide substitutions, while in data set (b), the number of indels was 72.7% the number of nucleotide substitutions. Regarding this tendency, Golengerg et al.<a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> further suggested that ‘superimposed changes’ affects indel number depend on the divergence time increasing. Namely, ‘recurrent’ indel events could increase the size of an indel length with decreasing of indel number. Our results do not contradict the assumptions of Golengerg et al.<a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> but the preset data is too deficient to test them. More sequence data from complete chloroplast genomes are needed.</p><p>Ingvarsson et al.<a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a> investigated indel frequencies among closely related taxa and they concluded that repeat indels evolve at a higher rate than non-repeat indels. In this study, as shown in <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>, most indels in data set (b) were tandem repeat indels, whereas, the number and distribution of tandem repeat and non-repeat indels was similar in data set (a). This difference between data sets (a) and (b) is also attributable to the different divergence times for the two data sets. Why do the frequencies of tandem repeat and non-repeat indels differ as a function of divergence time? To explain this difference, the mechanism by which indels arise must be considered. It is not well established that tandem repeat indels, which involve adding or subtracting short repeat sequences, result from polymerase slippage (i.e. slipped strand mispairing) during DNA replication.<a reveal-id="b23" data-open="b23" class="link link-ref link-reveal xref-bibr">23</a> Although there have also been few evolutionary studies on non-repeat indels, Levinson and Gutman<a reveal-id="b23" data-open="b23" class="link link-ref link-reveal xref-bibr">23</a> proposed that non-repeat indels arise in DNA sequences whose flanking sequences include repeat sequences. This proposal implies that the mechanism by which tandem repeat sequences arise depends on the structure of an existing repeat at the site of the tandem repeat, whereas the mechanism by which arise depends on the presence of repeat sequences at a distant site in flanking DNA. Interestingly, if their mechanism is conceivable, their mechanism by which non-repeat indels arise increase a probability of ‘superimposed changes’. Briefly, it is likely that the rate of superimposed changes is accelerated by repeat sequences, which act as a mutational ‘trigger,’ explaining the fact that the rate of tandem repeat indels decrease as divergence time increases, as observed in data set (a).</p>                            <h3 scrollto-destination=87195085 id="87195085" class="section-title" >3.3. Relationship between indel number and base composition</h3>
<p>Here, the relationship between indel number and base composition in chloroplast DNA is considered. It is well known that the chloroplast genome in monocot plants is very AT-rich (≈61% A/T<a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a>). Morton<a reveal-id="b25" data-open="b25" class="link link-ref link-reveal xref-bibr">25</a> noted a positive correlation between the number of transversions and AT-richness in rice and maize chloroplast genomes, and he proposed that the AT-richness of chloroplast DNA might have an impact on patterns of nucleotide substitution. However, the relationships between base composition and ‘microstructural changes’ in plant whole genomes has not been systematically studied. Most of tandem repeat sequences in chloroplast DNA are composed of either poly A/T or poly G/C-containing repeats.<a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a> In data set (a), we observed that 20 of 104 tandem repeat indels include multiple Gs or Cs (Supplementary Table 1-1), indicating strong and statistically significant bias towards A/T-rich tandem repeat indels (Fisher's exact test, <em>P</em> &lt; 0.001). This result also suggests that AT-richness influences the distribution of indels in the chloroplast genome. Zhou et al.<a reveal-id="b26" data-open="b26" class="link link-ref link-reveal xref-bibr">26</a> investigated the distribution of poly A/T and poly G/C repeats in 27 eukaryotes with various GC contents from 20 to 60%. Interestingly, their results also indicated that overall GC contents affect the number of poly A/T or poly G/C. These findings support the implication that the AT-richness of chloroplast DNA influences both the dynamics of nucleotide substitution and the number of tandem repeat indels.</p>                            <h3 scrollto-destination=87195087 id="87195087" class="section-title" >3.4. Evolutionary rate of indels in chloroplast intergenic regions</h3>
<p><a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Figure 2</a> shows the number of nucleotide substitutions and indels in each intergenic region examined in this study. Although the length of indels was not considered in this analysis, the number of indels correlated significantly with the length of the intergenic region. Linear regression analysis suggested a strong linear relationship between the two variables (<em>P</em> &lt; 0.001), and a quadratic relationship did not provide a better fit for the data (quadratic fit for all indels: <em>R</em><sup>2</sup> adjusted = 0.6339; data not shown). These data indicate that the number of indels is proportionate to the length of the region and its divergence time. In addition, it can be concluded that the slope of each line is a reasonable estimate of the mutation number per base pair.</p>                        <a id="87195090" scrollto-destination="87195090"></a>
<div id="fig2" data-id="fig2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl012/1/m_dsl012f2.jpeg?Expires=2147483647&Signature=DPRT3Jtva~fD7foG9WbxAw4OjDLCdNus9GQ7puJj4hlPRZrr6USUdRRZf2Kw52RO-UAaHRLyu6L545u3AObnSrmAMp30eMiQp3Yeg6i4FdG1xNPDrpqlyQYB9D5ZNcItPrq0Kpf228KwJEDFyiPc5YgSLqh5cjLqSqSfuLOeHT2DovhuZ-PDp8kCpj~KZFp~It8K67x0B2rJxkgxEJpNakNSVhx5ileJ8uz8IB61F-PSsUpUIRMy3zwG2NTyXD7pMOW24W9VNlmnbi6wWXByG5Z231Q8MKLkTM29mhKYIxnA1fTqSCSe1UgGfw8~0vpOV1dCNp~CnW6qfD-WlWUu6Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of nucleotide substitutions and indels in data set (a) excluding IR (a + b). Total indels, tandem repeat indels and non-repeat indels are plotted versus length (bp) of each intergenic region. The x-axis is truncated at 90% of total indel events. The best-fit linear regression lines are shown." path-from-xml="dsl012f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195090/dsl012f2.jpeg" path-from-xml="dsl012f2.jpeg" target="_blank">View large</a><a section="87195090" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl012/1/dsl012f2.jpeg?Expires=2147483647&Signature=4WCjIbWAeS~8BSQuD-ovOe7fOR-Cju7oRVj16uptYcRRvUp3ZLSt8nNopAp-QqlqT8yEmFQURXTjgR0MLDYgUrYBsekzCkeI8Alr9zubzidGeVzDn-Mxx2XAn39KvMxin7-UeSfY6r4OhoU8NhxNBvngZyad-woiO6KYW75TTVHQfVsqd73UxLjSlKJtIvJWEmE-QuY8KuTF5-KVojy-b2lp70x4J7P1BtWL3xXY9G9A9eARpZZrK7vFc9L3EyK~6nh6bAUFJtbxFoHciV3KlRIPmdl5KsJqrtzxsgm6y6BxPRB3aq~gBwBOzwzWCozCSm2vFJGpjcl8l3wusQzTmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195090&ar=400871&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl012f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of nucleotide substitutions and indels in data set (a) excluding IR (a + b). Total indels, tandem repeat indels and non-repeat indels are plotted versus length (bp) of each intergenic region. The <em>x</em>-axis is truncated at 90% of total indel events. The best-fit linear regression lines are shown.</p></div></div><div content-id="fig2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl012/1/m_dsl012f2.jpeg?Expires=2147483647&Signature=DPRT3Jtva~fD7foG9WbxAw4OjDLCdNus9GQ7puJj4hlPRZrr6USUdRRZf2Kw52RO-UAaHRLyu6L545u3AObnSrmAMp30eMiQp3Yeg6i4FdG1xNPDrpqlyQYB9D5ZNcItPrq0Kpf228KwJEDFyiPc5YgSLqh5cjLqSqSfuLOeHT2DovhuZ-PDp8kCpj~KZFp~It8K67x0B2rJxkgxEJpNakNSVhx5ileJ8uz8IB61F-PSsUpUIRMy3zwG2NTyXD7pMOW24W9VNlmnbi6wWXByG5Z231Q8MKLkTM29mhKYIxnA1fTqSCSe1UgGfw8~0vpOV1dCNp~CnW6qfD-WlWUu6Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of nucleotide substitutions and indels in data set (a) excluding IR (a + b). Total indels, tandem repeat indels and non-repeat indels are plotted versus length (bp) of each intergenic region. The x-axis is truncated at 90% of total indel events. The best-fit linear regression lines are shown." path-from-xml="dsl012f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195090/dsl012f2.jpeg" path-from-xml="dsl012f2.jpeg" target="_blank">View large</a><a section="87195090" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl012/1/dsl012f2.jpeg?Expires=2147483647&Signature=4WCjIbWAeS~8BSQuD-ovOe7fOR-Cju7oRVj16uptYcRRvUp3ZLSt8nNopAp-QqlqT8yEmFQURXTjgR0MLDYgUrYBsekzCkeI8Alr9zubzidGeVzDn-Mxx2XAn39KvMxin7-UeSfY6r4OhoU8NhxNBvngZyad-woiO6KYW75TTVHQfVsqd73UxLjSlKJtIvJWEmE-QuY8KuTF5-KVojy-b2lp70x4J7P1BtWL3xXY9G9A9eARpZZrK7vFc9L3EyK~6nh6bAUFJtbxFoHciV3KlRIPmdl5KsJqrtzxsgm6y6BxPRB3aq~gBwBOzwzWCozCSm2vFJGpjcl8l3wusQzTmA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195090&ar=400871&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl012f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of nucleotide substitutions and indels in data set (a) excluding IR (a + b). Total indels, tandem repeat indels and non-repeat indels are plotted versus length (bp) of each intergenic region. The <em>x</em>-axis is truncated at 90% of total indel events. The best-fit linear regression lines are shown.</p></div></div><p>The following calculations were made to estimate the rate of indel mutations in this study. The divergence time, <em>T</em>, for two species is given by <em>T</em> = <em>D</em><sub>XY</sub>/2<em>l</em>,<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> where <em>l</em> is the rate of nucleotide substitution per site per year. Since there is no direct estimate of the divergence time between sugarcane and maize, we estimated the divergence time from analyzing variation in their <em>rbcL</em> sequences (GenBank accession nos AP006714 and ZMA86563, respectively). As a result, the number of nucleotide substitutions (<em>D</em><sub>XY</sub>) between sugarcane and maize was 0.00699 for the <em>rbcL</em> gene and the synonymous and non-synonymous substitution rates for <em>rbcL</em>, <em>I</em>, were obtained from a previous estimate for the grass family.<a reveal-id="b27" data-open="b27" class="link link-ref link-reveal xref-bibr">27</a> This calculation indicates a putative divergence time for sugarcane and maize of ≈7.6 million years (Mya). Takahashi et al.<a reveal-id="b28" data-open="b28" class="link link-ref link-reveal xref-bibr">28</a> also roughly estimated a divergence time of 5–6 Mya between <em>Saccharum</em> species and maize using 18 chloroplast regions. Therefore, we decided to use our estimated divergence time for the further analysis.</p><p>In this study, the rate of indel mutations was estimated using a novel method, as follows. The slope of the line in <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2</a> is the mutation rate per base pair. Using the fact that the region length and indel number have a positive linear relationship (<em>R</em><sup>2</sup> = 0.62, <em>P</em> &lt; 0.001), it can be calculated that the intergenic regions of chloroplast genomes accumulate indel mutations at an average rate of 0.006 ± 0.00037 per site (= slope of a line). As a result, the rate of indel mutation is estimated to be ≈0.8 ± 0.04 × 10<sup>−9</sup> mutations per site per year. Saitou and Ueda<a reveal-id="b20" data-open="b20" class="link link-ref link-reveal xref-bibr">20</a> calculated the rates of indels in non-coding sequences in primates as ≈2 × 10<sup>−9</sup> for mitochondrial DNA and ≈0.2 × 10<sup>−9</sup> for nuclear DNA, respectively. Thus, this study suggests that the indel mutation rate in chloroplast DNA is intermediate between the previous estimate for mitochondrial and nuclear DNA. However, Saitou and Ueda<a reveal-id="b20" data-open="b20" class="link link-ref link-reveal xref-bibr">20</a> also showed that the rate of nucleotide substitutions was ∼10 times higher than the rate of indels for nuclear and mitochondrial non-coding DNA. In this study, the average nucleotide substitution rate was 0.0116 ± 0.00052 per site (= slope of a line, see <a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2</a>), which is approximately half the average indel mutation rate (see <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3</a>). In order to validate the method used here, the same method was used to calculate that the nucleotide substitution rate in chloroplast DNA is ≈1.52 ± 0.06 × 10<sup>−9</sup> substitutions per site per year. This value is quite similar to the previously estimated substitution rate for <em>rbcL</em> in the grass family of ≈1.3 × 10<sup>−9</sup> substitutions per synonymous site per year.<a reveal-id="b27" data-open="b27" class="link link-ref link-reveal xref-bibr">27</a> These data suggest that the stated estimate for indel mutation rate in chloroplast intergenic DNA, 0.8 ± 0.04 × 10<sup>−9</sup> mutations per site per year, is a reasonable estimate, and that the novel method used here to arrive at this estimate is valid.</p>                        <a id="87195094" scrollto-destination="87195094"></a>
<div id="fig3" data-id="fig3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl012/1/m_dsl012f3.jpeg?Expires=2147483647&Signature=XLCHQXMuOxlxs~TgEOpNrWq8g0mEkWc5Pk1Vx8QTDhd-mgFrw2gnvVoYryHr3X9P-LgUQczc9h8JPoS8-zgckEZDg0lseSHDl01NJAEp9T-2IFqkCen~YQ-PiyYzAOLU~W7CzQrnh6-cL9pZ3TT-~nvicc6v7qI2SVw5Qc4kHXK4dAvyP-YRMfmuQEYFAgkXZ5CYfv4RGOS0~iwggQAfeiD-Q0yMK0SenWVIOIwGObc0oaEmPWU3qt12XlXJP7ecYAl5JA8y20Wc2u~S-e~LTqcnSyHIiqrWOKPTqPSDylzd1HSR9oY3Xfq211TcwQfDrEoh1urHY8BapG~B4sqKPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relationship between number of substitutions (x-axis) and that of indels (y-axis) in each intergenic region from data set (a) excluding IR (a + b). The best-fit line from linear regression analysis is shown." path-from-xml="dsl012f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195094/dsl012f3.jpeg" path-from-xml="dsl012f3.jpeg" target="_blank">View large</a><a section="87195094" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl012/1/dsl012f3.jpeg?Expires=2147483647&Signature=mtWPtdrLSwbqV4oX-9Dgf-tf4EWmua1lJqZ9pBc7tsuxsFQ5QIAy~4rXynoDLE7hkHj7K0MhdNflKhPd68-4S1W677hW0E27hmjp3IP2AlRl-1309yO3ebkIz81ocadA~d7kAMvg5FbwuCi-U09vqSkTywQTVva9QHaX1Wap2B-cBcYnuYMgghx4JOWII~KUbBfygIzhYPVd~aFwtolLgDUlmJtbE4LqkAJfonJqZKTOc3hPBftWoknV7KS0ukQ-ElT4OJ6RraJ7IgvzT3fMQqnU9ppcXP26wK9x6etVEMJ3J4Gp22NaVdb2es6GUjccF6mXxXh12HAiWGuo6NpaYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195094&ar=400871&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl012f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relationship between number of substitutions (<em>x</em>-axis) and that of indels (<em>y</em>-axis) in each intergenic region from data set (a) excluding IR (a + b). The best-fit line from linear regression analysis is shown.</p></div></div><div content-id="fig3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl012/1/m_dsl012f3.jpeg?Expires=2147483647&Signature=XLCHQXMuOxlxs~TgEOpNrWq8g0mEkWc5Pk1Vx8QTDhd-mgFrw2gnvVoYryHr3X9P-LgUQczc9h8JPoS8-zgckEZDg0lseSHDl01NJAEp9T-2IFqkCen~YQ-PiyYzAOLU~W7CzQrnh6-cL9pZ3TT-~nvicc6v7qI2SVw5Qc4kHXK4dAvyP-YRMfmuQEYFAgkXZ5CYfv4RGOS0~iwggQAfeiD-Q0yMK0SenWVIOIwGObc0oaEmPWU3qt12XlXJP7ecYAl5JA8y20Wc2u~S-e~LTqcnSyHIiqrWOKPTqPSDylzd1HSR9oY3Xfq211TcwQfDrEoh1urHY8BapG~B4sqKPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relationship between number of substitutions (x-axis) and that of indels (y-axis) in each intergenic region from data set (a) excluding IR (a + b). The best-fit line from linear regression analysis is shown." path-from-xml="dsl012f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195094/dsl012f3.jpeg" path-from-xml="dsl012f3.jpeg" target="_blank">View large</a><a section="87195094" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl012/1/dsl012f3.jpeg?Expires=2147483647&Signature=mtWPtdrLSwbqV4oX-9Dgf-tf4EWmua1lJqZ9pBc7tsuxsFQ5QIAy~4rXynoDLE7hkHj7K0MhdNflKhPd68-4S1W677hW0E27hmjp3IP2AlRl-1309yO3ebkIz81ocadA~d7kAMvg5FbwuCi-U09vqSkTywQTVva9QHaX1Wap2B-cBcYnuYMgghx4JOWII~KUbBfygIzhYPVd~aFwtolLgDUlmJtbE4LqkAJfonJqZKTOc3hPBftWoknV7KS0ukQ-ElT4OJ6RraJ7IgvzT3fMQqnU9ppcXP26wK9x6etVEMJ3J4Gp22NaVdb2es6GUjccF6mXxXh12HAiWGuo6NpaYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195094&ar=400871&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsl012f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relationship between number of substitutions (<em>x</em>-axis) and that of indels (<em>y</em>-axis) in each intergenic region from data set (a) excluding IR (a + b). The best-fit line from linear regression analysis is shown.</p></div></div><p>The validity of the estimated indel mutation rate was also tested by examining data set (b). Ma and Bennetzen<a reveal-id="b29" data-open="b29" class="link link-ref link-reveal xref-bibr">29</a> used variation in nuclear DNA to estimate that the <em>indica</em> and <em>japonica</em> genomes share a common ancestor ≈0.44 Mya. Using nuclear genes, Zhu and Ge<a reveal-id="b30" data-open="b30" class="link link-ref link-reveal xref-bibr">30</a> also estimated the divergence time between var. <em>japonica</em> and <em>indica</em> (≈0.4 Mya). Based on this divergence time, the expected number of indels between <em>indica</em> and <em>japonica</em> is 8.92 ± 0.446 in a total of 12 657 bp excluding the IR region (Supplementary Table 1-2). This estimate is about half of the observed number of indels (16) in data set (b) (<a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>). If we assume that indel mutations have a Poisson distribution, the probability that 16 indels arose in the chloroplast intergenic regions in 0.44 million years is not rejected (<em>P</em> &gt; 0.05). This means that the rate of indel mutations estimated here is at least possible in sugarcane, maize and rice, despite the fact that there are evolutionary heterogeneities for tandem repeat and non-repeat indels. This information will facilitate studies of genetic variation or differentiation in plant species and may be useful for management of germplasm, genotyping or phylogenetic analyses.</p>                            <h3 scrollto-destination=87195097 id="87195097" class="section-title" >3.5. Conserved nature of IR region</h3>
<p>In most of the chloroplast genomes, there are two large IR regions (≈30 kb each). The evolutionarily conserved nature of these large IR regions in chloroplast DNA has been reported by several authors.(<a reveal-id="b31  b33" data-open="b31  b33" class="link link-ref link-reveal xref-bibr">31–33</a>) Palmer<a reveal-id="b32" data-open="b32" class="link link-ref link-reveal xref-bibr">32</a> proposed that this highly conserved property may be attributed to the fact that approximately one-third of the IR region is occupied by rRNA genes. Recently, Matsuoka et al.<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> confirmed that rRNA genes are highly conserved compared with non-rRNA genes. <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a> shows that the number of nucleotide substitutions in SC (LSC + SSC) was ∼20 times higher than in the IR (a + b) region in data set (a). Furthermore, no substitutions or indels occurred in the IR (a + b) region in data set (b). These results clearly show that nucleotide substitutions and indels occur at a much lower rate in the IR (a + b) region than in SC (LSC + SSC) regions. Thus, the conserved nature of the IR (a + b) regions with respect to microstructural changes is confirmed in this study. Since functional constraints do not apply to non-coding regions and synonymous sites are implausible, another explanation for this phenomenon is needed. Wolfe et al.<a reveal-id="b33" data-open="b33" class="link link-ref link-reveal xref-bibr">33</a> already pointed out based on their sequence analyses that the highly conserved nature of IR sequences does depend on other dimension(s) for a reduced mutation rate in the IR region or a bias against mutations during DNA repair in IR DNA rather than on a functional constraint. Birky and Walsh<a reveal-id="b34" data-open="b34" class="link link-ref link-reveal xref-bibr">34</a> insisted from a theoretical study that a difference in the synonymous substitution rate for the IR and SC (LSC + SSC) regions was resulted from the fact that IR sequences are duplicated in the chloroplast genome. Perry and Wolfe<a reveal-id="b35" data-open="b35" class="link link-ref link-reveal xref-bibr">35</a> compared the synonymous substitution rate in chloroplast genes in duplicated IR (most plants are this type) and single-copy IR (only one single-copy IR in the chloroplast genome) in legume species. The result showed that the synonymous substitution rate in IR genes was 2.3-fold lower in duplicated IR than in single-copy IR. Based on these findings, consequently, it can be concluded that the highly conserved nature of duplicated IR is a function of DNA structure and not a function of primary DNA sequence.</p><p>We are indebted Dr K. Tanno, Research Institute for Humanity and Nature, for his helpful and valuable comments. The authors are also indebted to Prof. O. Ohnishi and Dr Y. Yasui, Kyoto University, Dr T. Ohsako, Kyoto Prefectural University and Dr Y. Matsuoka, Fukui Prefectural University, for their useful advices. We are also grateful to assistant Prof. Y. Yamazaki, National institute of Genetics, for her helpful suggestions. This is contribution No. 129 from the Plant Germplasm Institute, Graduate School of Agriculture, Kyoto University. This research was partly supported by a grant from the ministry of culture, education and technology, the National BioResource Project KOMUGI.</p><p><strong>Supplementary Data:</strong> Supplementary data are available online at <a class="link link-uri" href="http://dnaresearch.oxfordjournals.org" target="_blank"></a>.</p>                            <h2 scrollto-destination=87195104 id="87195104" class="backreferences-title" >REFERENCES</h2>
<div class="ref-list"><div content-id="b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Britten</div> <div class="given-names">R. J.</div></div>,  <div class="name"><div class="surname">Rowen</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Williams</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Cameron</div> <div class="given-names">R. A.</div></div>. </div><div class="article-title">Majority of divergence between closely related DNA samples is due to indels</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2003</div>, vol. <div class="volume">100</div> (pg. <div class="fpage">4661</div>-<div class="lpage">4665</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Majority%20of%20divergence%20between%20closely%20related%20DNA%20samples%20is%20due%20to%20indels&amp;author=R.%20J.%20Britten&amp;author=L.%20Rowen&amp;author=J.%20Williams&amp;author=R.%20A.%20Cameron&amp;publication_year=2003&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=100&amp;pages=4661-4665" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0330964100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0330964100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0330964100"> </span></div> </div></div></div></div></div><div content-id="b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kelchner</div> <div class="given-names">S. A.</div></div>. </div><div class="article-title">The evolution of non-coding chloroplast DNA and its application in plant systematics</div>, <div class="source">Ann. Missouri Bot. Gard.</div> , <div class="year">2000</div>, vol. <div class="volume">87</div> (pg. <div class="fpage">499</div>-<div class="lpage">527</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20non-coding%20chloroplast%20DNA%20and%20its%20application%20in%20plant%20systematics&amp;author=S.%20A.%20Kelchner&amp;publication_year=2000&amp;journal=Ann.%20Missouri%20Bot.%20Gard.&amp;volume=87&amp;pages=499-527" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2666142" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2666142" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2666142"> </span></div> </div></div></div></div></div><div content-id="b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ingvarsson</div> <div class="given-names">P. K.</div></div>,  <div class="name"><div class="surname">Ribstein</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Taylor</div> <div class="given-names">D. R.</div></div>. </div><div class="article-title">Molecular evolution of insertions and deletions in the chloroplast genome of <em>Silene</em></div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2003</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1737</div>-<div class="lpage">1740</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20evolution%20of%20insertions%20and%20deletions%20in%20the%20chloroplast%20genome%20of%20Silene&amp;author=P.%20K.%20Ingvarsson&amp;author=S.%20Ribstein&amp;author=D.%20R.%20Taylor&amp;publication_year=2003&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=20&amp;pages=1737-1740" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msg163" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsg163" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsg163"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12832644" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hamilton</div> <div class="given-names">M. B.</div></div>,  <div class="name"><div class="surname">Braverman</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Soria-Hernanz</div> <div class="given-names">D. F.</div></div>. </div><div class="article-title">Patterns and relative rates of nucleotide and insertion/deletion evolution at six chloroplast intergenic regions in New World species of the Lecythidaceae</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2003</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1710</div>-<div class="lpage">1721</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patterns%20and%20relative%20rates%20of%20nucleotide%20and%20insertion%2Fdeletion%20evolution%20at%20six%20chloroplast%20intergenic%20regions%20in%20New%20World%20species%20of%20the%20Lecythidaceae&amp;author=M.%20B.%20Hamilton&amp;author=J.%20M.%20Braverman&amp;author=D.%20F.%20Soria-Hernanz&amp;publication_year=2003&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=20&amp;pages=1710-1721" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msg190" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsg190" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsg190"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12832633" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hiratsuka</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Shimada</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Whittier</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">The complete sequence of the rice (<em>Oryza sativa</em>) chloroplast genome: intermolecular recombination between distinct tRNA genes accounts for a major plastid DNA inversion during the evolution of the cereals</div>, <div class="source">Mol. Gen. Genet.</div> , <div class="year">1989</div>, vol. <div class="volume">217</div> (pg. <div class="fpage">185</div>-<div class="lpage">194</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20sequence%20of%20the%20rice%20(Oryza%20sativa)%20chloroplast%20genome%3A%20intermolecular%20recombination%20between%20distinct%20tRNA%20genes%20accounts%20for%20a%20major%20plastid%20DNA%20inversion%20during%20the%20evolution%20of%20the%20cereals&amp;author=J.%20Hiratsuka&amp;author=H.%20Shimada&amp;author=R.%20Whittier&amp;publication_year=1989&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=217&amp;pages=185-194" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02464880" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02464880" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02464880"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2770692" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Maier</div> <div class="given-names">R. M.</div></div>,  <div class="name"><div class="surname">Neckermann</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Igloi</div> <div class="given-names">G. L.</div></div>,  <div class="name"><div class="surname">Kossel</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Complete sequence of the maize chloroplast genome: gene content, hotspots of divergence and fine tuning of genetic information by transcript editing</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1995</div>, vol. <div class="volume">251</div> (pg. <div class="fpage">614</div>-<div class="lpage">628</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20sequence%20of%20the%20maize%20chloroplast%20genome%3A%20gene%20content%2C%20hotspots%20of%20divergence%20and%20fine%20tuning%20of%20genetic%20information%20by%20transcript%20editing&amp;author=R.%20M.%20Maier&amp;author=K.%20Neckermann&amp;author=G.%20L.%20Igloi&amp;author=H.%20Kossel&amp;publication_year=1995&amp;journal=J.%20Mol.%20Biol.&amp;volume=251&amp;pages=614-628" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1995.0460" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1995.0460" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1995.0460"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7666415" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ogihara</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Isono</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kojim</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Structural features of a wheat plastome as revealed by complete sequencing of chloroplast DNA</div>, <div class="source">Mol. Gen. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">266</div> (pg. <div class="fpage">740</div>-<div class="lpage">746</div>) <div class="comment">(19 co-authors)</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20features%20of%20a%20wheat%20plastome%20as%20revealed%20by%20complete%20sequencing%20of%20chloroplast%20DNA&amp;author=Y.%20Ogihara&amp;author=K.%20Isono&amp;author=T.%20Kojim&amp;publication_year=2002&amp;journal=Mol.%20Gen.%20Genet.&amp;volume=266&amp;pages=740-746" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-001-0606-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-001-0606-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-001-0606-9"> </span></div> </div></div></div></div></div><div content-id="b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Matsuoka</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Yamazaki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ogihara</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tsunewaki</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Whole chloroplast genome comparison of rice, maize, and wheat: implications for chloroplast gene diversification and phylogeny of cereals</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2002</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">2084</div>-<div class="lpage">2091</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20chloroplast%20genome%20comparison%20of%20rice%2C%20maize%2C%20and%20wheat%3A%20implications%20for%20chloroplast%20gene%20diversification%20and%20phylogeny%20of%20cereals&amp;author=Y.%20Matsuoka&amp;author=Y.%20Yamazaki&amp;author=Y.%20Ogihara&amp;author=K.%20Tsunewaki&amp;publication_year=2002&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=19&amp;pages=2084-2091" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a004033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a004033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a004033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12446800" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Asano</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Tsudzuki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Takahashi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Shimada</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kadowaki</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Complete nucleotide sequence of the sugarcane (<em>Saccharum officinarum</em>) chloroplast genome: a comparative analysis of four monocot chloroplast genomes</div>, <div class="source">DNA res.</div> , <div class="year">2004</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">93</div>-<div class="lpage">99</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20nucleotide%20sequence%20of%20the%20sugarcane%20(Saccharum%20officinarum)%20chloroplast%20genome%3A%20a%20comparative%20analysis%20of%20four%20monocot%20chloroplast%20genomes&amp;author=T.%20Asano&amp;author=T.%20Tsudzuki&amp;author=S.%20Takahashi&amp;author=H.%20Shimada&amp;author=K.%20Kadowaki&amp;publication_year=2004&amp;journal=DNA%20res.&amp;volume=11&amp;pages=93-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/11.2.93" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F11.2.93" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F11.2.93"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15449542" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bacci</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Miranda</div> <div class="given-names">V. F. O.</div></div>,  <div class="name"><div class="surname">Martins</div> <div class="given-names">V. G.</div></div>, et al. </div><div class="article-title">A search for markers of sugarcane evolution</div>, <div class="source">Genet. Mol. Biol.</div> , <div class="year">2001</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">169</div>-<div class="lpage">174</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20search%20for%20markers%20of%20sugarcane%20evolution&amp;author=M.%20Bacci&amp;author=V.%20F.%20O.%20Miranda&amp;author=V.%20G.%20Martins&amp;publication_year=2001&amp;journal=Genet.%20Mol.%20Biol.&amp;volume=24&amp;pages=169-174" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1590/S1415-47572001000100023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1590%2FS1415-47572001000100023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1590%2FS1415-47572001000100023"> </span></div> </div></div></div></div></div><div content-id="b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hodkinson</div> <div class="given-names">T. R.l.</div></div>,  <div class="name"><div class="surname">Chase</div> <div class="given-names">M. W.</div></div>,  <div class="name"><div class="surname">Lledo</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Salamin</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Renvoize</div> <div class="given-names">S. A.</div></div>. </div><div class="article-title">Molecular phylogeny of <em>Miscanthus</em> s.l., <em>Saccharum</em> and related genera (Saccharinae, Andropogoneae, Poaceae) using DNA sequences from ITS nuclear ribosomal DNA and the plastid <em>trnL-F</em> regions</div>, <div class="source">J. Plant Res.</div> , <div class="year">2002</div>, vol. <div class="volume">115</div> (pg. <div class="fpage">381</div>-<div class="lpage">392</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20phylogeny%20of%20Miscanthus%20s.l.%2C%20Saccharum%20and%20related%20genera%20(Saccharinae%2C%20Andropogoneae%2C%20Poaceae)%20using%20DNA%20sequences%20from%20ITS%20nuclear%20ribosomal%20DNA%20and%20the%20plastid%20trnL-F%20regions&amp;author=T.%20R.l.%20Hodkinson&amp;author=M.%20W.%20Chase&amp;author=D.%20Lledo&amp;author=N.%20Salamin&amp;author=S.%20A.%20Renvoize&amp;publication_year=2002&amp;journal=J.%20Plant%20Res.&amp;volume=115&amp;pages=381-392" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10265-002-0049-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10265-002-0049-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10265-002-0049-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12579363" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Xia</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Cao</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">A comparison of rice chloroplast genomes</div>, <div class="source">Plant Phys.</div> , <div class="year">2004</div>, vol. <div class="volume">135</div> (pg. <div class="fpage">412</div>-<div class="lpage">420</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comparison%20of%20rice%20chloroplast%20genomes&amp;author=J.%20Tang&amp;author=H.%20Xia&amp;author=M.%20Cao&amp;publication_year=2004&amp;journal=Plant%20Phys.&amp;volume=135&amp;pages=412-420" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.103.031245" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.103.031245" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.103.031245"> </span></div> </div></div></div></div></div><div content-id="b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Birky</div> <div class="given-names">C. W. J.</div></div>. </div><div class="article-title">Uniparental inheritance of mitochondrial aid chloroplast genes: mechanisms and evolution</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1995</div>, vol. <div class="volume">92</div> (pg. <div class="fpage">11331</div>-<div class="lpage">11338</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Uniparental%20inheritance%20of%20mitochondrial%20aid%20chloroplast%20genes%3A%20mechanisms%20and%20evolution&amp;author=C.%20W.%20J.%20Birky&amp;publication_year=1995&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=92&amp;pages=11331-11338" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.92.25.11331" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.92.25.11331" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.92.25.11331"> </span></div> </div></div></div></div></div><div content-id="b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">J. D.</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">D. G.</div></div>,  <div class="name"><div class="surname">Gibson</div> <div class="given-names">T. J.</div></div>. </div><div class="article-title">CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, positions-specific gap penalties and weight matrix choice</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1994</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">4673</div>-<div class="lpage">4680</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CLUSTAL%20W%3A%20improving%20the%20sensitivity%20of%20progressive%20multiple%20sequence%20alignment%20through%20sequence%20weighting%2C%20positions-specific%20gap%20penalties%20and%20weight%20matrix%20choice&amp;author=J.%20D.%20Thompson&amp;author=D.%20G.%20Higgins&amp;author=T.%20J.%20Gibson&amp;publication_year=1994&amp;journal=Nucleic%20Acids%20Res.&amp;volume=22&amp;pages=4673-4680" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F22.22.4673" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F22.22.4673"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7984417" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sugiura</div> <div class="given-names">M.</div></div>. </div><div class="article-title">The chloroplast genome</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">1992</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">149</div>-<div class="lpage">168</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20chloroplast%20genome&amp;author=M.%20Sugiura&amp;publication_year=1992&amp;journal=Plant%20Mol.%20Biol.&amp;volume=19&amp;pages=149-168" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00015612" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00015612" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00015612"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1600166" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sugita</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sugiura</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Regulation of gene expression in chloroplasts of higher plants</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">1996</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">315</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20gene%20expression%20in%20chloroplasts%20of%20higher%20plants&amp;author=M.%20Sugita&amp;author=M.%20Sugiura&amp;publication_year=1996&amp;journal=Plant%20Mol.%20Biol.&amp;volume=32&amp;pages=315-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00039388" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00039388" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00039388"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8980485" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rozas</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Rozas</div> <div class="given-names">R.</div></div>. </div><div class="article-title">DnaSP version 3: an integrated program for molecular population genetics and molecular evolution analysis</div>, <div class="source">Bioinformatics</div> , <div class="year">1999</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">174</div>-<div class="lpage">175</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DnaSP%20version%203%3A%20an%20integrated%20program%20for%20molecular%20population%20genetics%20and%20molecular%20evolution%20analysis&amp;author=J.%20Rozas&amp;author=R.%20Rozas&amp;publication_year=1999&amp;journal=Bioinformatics&amp;volume=15&amp;pages=174-175" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/15.2.174" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F15.2.174" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F15.2.174"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10089204" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>. </div>, <div class="source">Molecular</div> , <div class="year">1987</div><div class="publisher-loc">New York</div><div class="publisher-name">Evolutionary Genetics, Columbia University Press</div></div></div></div></div><div content-id="b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gu</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.-H.</div></div>. </div><div class="article-title">The size distribution of insertions and deletions in human and rodent pseudogenes suggests the logarithmic gap penalty for sequence alignment</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1995</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">464</div>-<div class="lpage">473</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20size%20distribution%20of%20insertions%20and%20deletions%20in%20human%20and%20rodent%20pseudogenes%20suggests%20the%20logarithmic%20gap%20penalty%20for%20sequence%20alignment&amp;author=X.%20Gu&amp;author=W.-H.%20Li&amp;publication_year=1995&amp;journal=J.%20Mol.%20Evol.&amp;volume=40&amp;pages=464-473" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00164032" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00164032" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00164032"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7769622" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saitou</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Ueda</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Evolutionary rates of insertion and deletion in noncoding nucleotide sequences of primates</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1994</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">504</div>-<div class="lpage">512</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20rates%20of%20insertion%20and%20deletion%20in%20noncoding%20nucleotide%20sequences%20of%20primates&amp;author=N.%20Saitou&amp;author=S.%20Ueda&amp;publication_year=1994&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=11&amp;pages=504-512" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8015443" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Graur</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Shauali</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.-H.</div></div>. </div><div class="article-title">Deletions in processed pseudogenes accumulate faster in rodents than in humans</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1989</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">279</div>-<div class="lpage">285</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deletions%20in%20processed%20pseudogenes%20accumulate%20faster%20in%20rodents%20than%20in%20humans&amp;author=D.%20Graur&amp;author=Y.%20Shauali&amp;author=W.-H.%20Li&amp;publication_year=1989&amp;journal=J.%20Mol.%20Evol.&amp;volume=28&amp;pages=279-285" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02103423" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02103423" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02103423"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2499684" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Golenberg</div> <div class="given-names">E. M.</div></div>,  <div class="name"><div class="surname">Clegg</div> <div class="given-names">M. T.</div></div>,  <div class="name"><div class="surname">Durbin</div> <div class="given-names">M. L.</div></div>,  <div class="name"><div class="surname">Doebley</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Ma</div> <div class="given-names">D. P.</div></div>. </div><div class="article-title">Evolution of a noncoding region of the chloroplast genome</div>, <div class="source">Mol. Phylogenet. Evol.</div> , <div class="year">1993</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">52</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20a%20noncoding%20region%20of%20the%20chloroplast%20genome&amp;author=E.%20M.%20Golenberg&amp;author=M.%20T.%20Clegg&amp;author=M.%20L.%20Durbin&amp;author=J.%20Doebley&amp;author=D.%20P.%20Ma&amp;publication_year=1993&amp;journal=Mol.%20Phylogenet.%20Evol.&amp;volume=2&amp;pages=52-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/mpev.1993.1006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fmpev.1993.1006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fmpev.1993.1006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8081547" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Levinson</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Gutman</div> <div class="given-names">G. A.</div></div>. </div><div class="article-title">Slipped-strand mispairing: a major mechanism for DNA sequence evolution</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1987</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">203</div>-<div class="lpage">221</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Slipped-strand%20mispairing%3A%20a%20major%20mechanism%20for%20DNA%20sequence%20evolution&amp;author=G.%20Levinson&amp;author=G.%20A.%20Gutman&amp;publication_year=1987&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=4&amp;pages=203-221" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3328815" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Masood</div> <div class="given-names">M. S.</div></div>,  <div class="name"><div class="surname">Nishikawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Fukuoka</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Njenga</div> <div class="given-names">P. K.</div></div>,  <div class="name"><div class="surname">Tsudzuki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kadowaki</div> <div class="given-names">K.</div></div>. </div><div class="article-title">The complete nucleotide sequence of wild rice (<em>Oryza nivara</em>) chloroplast genome: first genome wide comparative sequence analysis of wild and cultivated rice</div>, <div class="source">Gene</div> , <div class="year">2004</div>, vol. <div class="volume">340</div> (pg. <div class="fpage">133</div>-<div class="lpage">139</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20nucleotide%20sequence%20of%20wild%20rice%20(Oryza%20nivara)%20chloroplast%20genome%3A%20first%20genome%20wide%20comparative%20sequence%20analysis%20of%20wild%20and%20cultivated%20rice&amp;author=M.%20S.%20Masood&amp;author=T.%20Nishikawa&amp;author=S.%20Fukuoka&amp;author=P.%20K.%20Njenga&amp;author=T.%20Tsudzuki&amp;author=K.%20Kadowaki&amp;publication_year=2004&amp;journal=Gene&amp;volume=340&amp;pages=133-139" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2004.06.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2004.06.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2004.06.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15556301" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morton</div> <div class="given-names">B. R.</div></div>. </div><div class="article-title">Neighboring base composition and transversion/transition bias in a comparison of rice and maize chloroplast noncoding regions</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1995</div>, vol. <div class="volume">92</div> (pg. <div class="fpage">9717</div>-<div class="lpage">9721</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Neighboring%20base%20composition%20and%20transversion%2Ftransition%20bias%20in%20a%20comparison%20of%20rice%20and%20maize%20chloroplast%20noncoding%20regions&amp;author=B.%20R.%20Morton&amp;publication_year=1995&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=92&amp;pages=9717-9721" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.92.21.9717" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.92.21.9717" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.92.21.9717"> </span></div> </div></div></div></div></div><div content-id="b26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhou</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Bizzaro</div> <div class="given-names">J. W.</div></div>,  <div class="name"><div class="surname">Marx</div> <div class="given-names">K. A.</div></div>. </div><div class="article-title">Homopolymer tract length dependent enrichments in functional regions of 27 eukaryotes and their novel dependence on the organism DNA (G+C)% composition</div>, <div class="source">BMC Genomics</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> pg. <div class="fpage">95</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Homopolymer%20tract%20length%20dependent%20enrichments%20in%20functional%20regions%20of%2027%20eukaryotes%20and%20their%20novel%20dependence%20on%20the%20organism%20DNA%20(G%2BC)%25%20composition&amp;author=Y.%20Zhou&amp;author=J.%20W.%20Bizzaro&amp;author=K.%20A.%20Marx&amp;publication_year=2004&amp;journal=BMC%20Genomics&amp;volume=5&amp;pages=95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-5-95" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-5-95" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-5-95"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15598342" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Clegg</div> <div class="given-names">M. T.</div></div>. </div><div class="article-title">Chloroplast gene sequences and the study of plant evolution</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1993</div>, vol. <div class="volume">90</div> (pg. <div class="fpage">363</div>-<div class="lpage">367</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chloroplast%20gene%20sequences%20and%20the%20study%20of%20plant%20evolution&amp;author=M.%20T.%20Clegg&amp;publication_year=1993&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=90&amp;pages=363-367" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.90.2.363" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.90.2.363" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.90.2.363"> </span></div> </div></div></div></div></div><div content-id="b28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takahashi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Furukawa</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Asano</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Very close relationship of the chloroplast genomes among <em>Saccharum</em> species</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">110</div> (pg. <div class="fpage">1523</div>-<div class="lpage">1529</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Very%20close%20relationship%20of%20the%20chloroplast%20genomes%20among%20Saccharum%20species&amp;author=S.%20Takahashi&amp;author=T.%20Furukawa&amp;author=T.%20Asano&amp;publication_year=2005&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=110&amp;pages=1523-1529" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-005-1990-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-005-1990-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-005-1990-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15818464" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ma</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J. L.</div></div>. </div><div class="article-title">Rapid recent growth and divergence of rice nuclear genomes</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">12404</div>-<div class="lpage">12410</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20recent%20growth%20and%20divergence%20of%20rice%20nuclear%20genomes&amp;author=J.%20Ma&amp;author=J.%20L.%20Bennetzen&amp;publication_year=2004&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=12404-12410" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0403715101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0403715101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0403715101"> </span></div> </div></div></div></div></div><div content-id="b30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhu</div> <div class="given-names">Q. H.</div></div>,  <div class="name"><div class="surname">Ge</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Phylogenetic relationships among A-genome species of the genus <em>Oryza</em> revealed by intron sequences of four nuclear genes</div>, <div class="source">New Phytol.</div> , <div class="year">2005</div>, vol. <div class="volume">167</div> (pg. <div class="fpage">249</div>-<div class="lpage">265</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20relationships%20among%20A-genome%20species%20of%20the%20genus%20Oryza%20revealed%20by%20intron%20sequences%20of%20four%20nuclear%20genes&amp;author=Q.%20H.%20Zhu&amp;author=S.%20Ge&amp;publication_year=2005&amp;journal=New%20Phytol.&amp;volume=167&amp;pages=249-265" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1469-8137.2005.01406.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1469-8137.2005.01406.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1469-8137.2005.01406.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15948847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Curtis</div> <div class="given-names">S. E.</div></div>,  <div class="name"><div class="surname">Clegg</div> <div class="given-names">M. T.</div></div>. </div><div class="article-title">Molecular evolution of chloroplast DNA sequences</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1984</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">291</div>-<div class="lpage">301</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20evolution%20of%20chloroplast%20DNA%20sequences&amp;author=S.%20E.%20Curtis&amp;author=M.%20T.%20Clegg&amp;publication_year=1984&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=1&amp;pages=291-301" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6152869" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Palmer</div> <div class="given-names">J. D.</div></div>. </div><div class="article-title">Comparative organization of chloroplast genomes</div>, <div class="source">Annu. Rev. Genet.</div> , <div class="year">1985</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">325</div>-<div class="lpage">354</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20organization%20of%20chloroplast%20genomes&amp;author=J.%20D.%20Palmer&amp;publication_year=1985&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=19&amp;pages=325-354" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.ge.19.120185.001545" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.ge.19.120185.001545" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.ge.19.120185.001545"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3936406" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wolfe</div> <div class="given-names">K. H.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.-H.</div></div>,  <div class="name"><div class="surname">Sharp</div> <div class="given-names">P. M.</div></div>. </div><div class="article-title">Rates of nucleotide substitution vary greatly across mitochondrial, chloroplast, and nuclear DNAs</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1987</div>, vol. <div class="volume">84</div> (pg. <div class="fpage">9054</div>-<div class="lpage">9058</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rates%20of%20nucleotide%20substitution%20vary%20greatly%20across%20mitochondrial%2C%20chloroplast%2C%20and%20nuclear%20DNAs&amp;author=K.%20H.%20Wolfe&amp;author=W.-H.%20Li&amp;author=P.%20M.%20Sharp&amp;publication_year=1987&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=84&amp;pages=9054-9058" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.84.24.9054" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.84.24.9054" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.84.24.9054"> </span></div> </div></div></div></div></div><div content-id="b34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Birky</div> <div class="given-names">C. W. J.</div></div>,  <div class="name"><div class="surname">Walsh</div> <div class="given-names">J. B.</div></div>. </div><div class="article-title">Biased gene conversion, copy number, and apparent mutation rate differences within chloroplast and bacterial genomes</div>, <div class="source">Genetics</div> , <div class="year">1992</div>, vol. <div class="volume">130</div> (pg. <div class="fpage">677</div>-<div class="lpage">783</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biased%20gene%20conversion%2C%20copy%20number%2C%20and%20apparent%20mutation%20rate%20differences%20within%20chloroplast%20and%20bacterial%20genomes&amp;author=C.%20W.%20J.%20Birky&amp;author=J.%20B.%20Walsh&amp;publication_year=1992&amp;journal=Genetics&amp;volume=130&amp;pages=677-783" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1551584" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Perry</div> <div class="given-names">A. S.</div></div>,  <div class="name"><div class="surname">Wolfe</div> <div class="given-names">K. H.</div></div>. </div><div class="article-title">Nucleotide substitution rates in legume chloroplast DNA depend on the presence of the inverted repeat</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">2002</div>, vol. <div class="volume">55</div> (pg. <div class="fpage">501</div>-<div class="lpage">508</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleotide%20substitution%20rates%20in%20legume%20chloroplast%20DNA%20depend%20on%20the%20presence%20of%20the%20inverted%20repeat&amp;author=A.%20S.%20Perry&amp;author=K.%20H.%20Wolfe&amp;publication_year=2002&amp;journal=J.%20Mol.%20Evol.&amp;volume=55&amp;pages=501-508" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-002-2333-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-002-2333-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-002-2333-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12399924" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2006. Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=400871&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/13/5/10.1093_dnares_dsl012/1/dsl012_Supplementary_Data.zip?Expires=2147483647&amp;Signature=wUWx8I0zumyp~b3E9oMc36lI6vTaHuwyLmFGhft8miz3HTBP~XUqnB~cuw2ZiiCU1Q5pZzmJb4ga5tXRc1OiX4wztADDaLVPRVq-i4p7IFjAM9x6wHmaYr98Jn2F1bJvmTJUDV5m0fe7nQF8FtnjplzRvtv7BLk1qRfnoZh7caXmmQhym9IUkO-8VbP6TFhQj1O-kzVVaYN36RksrjyP3Nyq0ynbpx0EHML5R1iWxlozX1f76c8Yx347FOZ~TNt7KaZgGQiknsm2jAa-dTp3qzVmmCS6tXLEYlajw45CrDhdDOVhhCbW9EiHl7yIMdu4tBGMcZcZaQXsC5QDUQ6f6g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">416</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000243280300002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">73</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsl012"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="400871"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000243280300002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f13%2f5%2f197%2f400871">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000243280300002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (73)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F13%2F5%2F197%2F400871" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/400871'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Pattern and Rate of Indel Evolution Inferred from Whole Chloroplast Intergenic Regions in Sugarcane, Maize and Rice" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsl012");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f13%2f5%2f197%2f400871">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

