

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/4/377/2469962","@type":"ScholarlyArticle","name":"Selection for reduced translation costs at the intronic 5′ end in fungi","datePublished":"2016-06-02","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2016-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw019","keywords":["mRNA translation","intron evolution","transcript evolution","splicing","silent mutations"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/4/377/2469962","author":[{"name":"Zafrir, Zohar","affiliation":"Department of Biomedical Engineering, Tel Aviv University, Tel Aviv, Israel","@type":"Person"},{"name":"Zur, Hadas","affiliation":"Department of Biomedical Engineering, Tel Aviv University, Tel Aviv, Israel","@type":"Person"},{"name":"Tuller, Tamir","affiliation":"Department of Biomedical Engineering, Tel Aviv University, Tel Aviv, Israel    The Sagol School of Neuroscience, Tel Aviv University, Tel Aviv 69978, Israel","@type":"Person"}],"description":"Abstract.   It is generally believed that introns are not translated; therefore, the potential intronic features that may be related to the translation step (oc","pageStart":"377","pageEnd":"394","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/4/0/m_cover.gif?Expires=2147483647&Signature=XWSbFJdI5lGDaRny1g7BRzfZg~ebzc~OT20sUUxiNUi3-W~7uvtu5IDIgO3L5~QjmCQzUopLMGz-3gEHtIKUOFbRCHM2eH4h7l2Wu3nqAQC9YYHygJi-TLC9x3FpQRq3lTBcdWmpj185LJbB-1H-PKlUoEU9gxYyCNvjJUg~GUaHNRtVxbV0a7bDIfdyclZJ~w1BhGonEiYqh-IZSS0VekFWZ7mnLia~WbOEtL8AzSLsoIVak5-aADczuqIYGRkdfxcEh8zTdO-NlGhKxLoi5zlJy0Y~XuIxHZNxQxDFoVA3s~0J2fujGZsWX~ElpEU3I2Vma3M6MgFM80P2xPWpTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Selection for reduced translation costs at the intronic 5′ end in fungi","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/4/0/m_cover.gif?Expires=2147483647&Signature=XWSbFJdI5lGDaRny1g7BRzfZg~ebzc~OT20sUUxiNUi3-W~7uvtu5IDIgO3L5~QjmCQzUopLMGz-3gEHtIKUOFbRCHM2eH4h7l2Wu3nqAQC9YYHygJi-TLC9x3FpQRq3lTBcdWmpj185LJbB-1H-PKlUoEU9gxYyCNvjJUg~GUaHNRtVxbV0a7bDIfdyclZJ~w1BhGonEiYqh-IZSS0VekFWZ7mnLia~WbOEtL8AzSLsoIVak5-aADczuqIYGRkdfxcEh8zTdO-NlGhKxLoi5zlJy0Y~XuIxHZNxQxDFoVA3s~0J2fujGZsWX~ElpEU3I2Vma3M6MgFM80P2xPWpTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Selection for reduced translation costs at the intronic 5′ end in fungi" />
    <meta property="og:description" content="Abstract.   It is generally believed that introns are not translated; therefore, the potential intronic features that may be related to the translation step (oc" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/4/377/2469962" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-06-02" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Zafrir, Zohar" /><meta name="citation_author_institution" content=" Department of Biomedical Engineering, Tel Aviv University, Tel Aviv, Israel " /><meta name="citation_author" content="Zur, Hadas" /><meta name="citation_author_institution" content=" Department of Biomedical Engineering, Tel Aviv University, Tel Aviv, Israel " /><meta name="citation_author" content="Tuller, Tamir" /><meta name="citation_author_institution" content=" Department of Biomedical Engineering, Tel Aviv University, Tel Aviv, Israel " /><meta name="citation_author_institution" content=" The Sagol School of Neuroscience, Tel Aviv University, Tel Aviv 69978, Israel " /><meta name="citation_title" content="Selection for reduced translation costs at the intronic 5′ end in fungi" /><meta name="citation_firstpage" content="377" /><meta name="citation_lastpage" content="394" /><meta name="citation_doi" content="10.1093/dnares/dsw019" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2016/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The spliceosome: the most complex macromolecular machine in the cell?; citation_author=Nilsen  T.W.;  citation_journal_title=BioEssays;  citation_year=2003;  citation_volume=25;  citation_pages=1147-9; " /><meta name="citation_reference" content="citation_title=The spliceosome: a flexible, reversible macromolecular machine; citation_author=Hoskins  A.A.; citation_author=Moore  M.J.;  citation_journal_title=Trends Biochem. Sci;  citation_year=2012;  citation_volume=37;  citation_pages=179-88; " /><meta name="citation_reference" content="citation_title=How introns influence and enhance eukaryotic gene expression; citation_author=Le Hir  H.; citation_author=Nott  A.; citation_author=Moore  M.J.;  citation_journal_title=Trends Biochem. Sci;  citation_year=2003;  citation_volume=28;  citation_pages=215-20; " /><meta name="citation_reference" content="citation_title=Splicing in disease: disruption of the splicing code and the decoding machinery; citation_author=Wang  G.S.; citation_author=Cooper  T.A.;  citation_journal_title=Nat. Rev. Genet;  citation_year=2007;  citation_volume=8;  citation_pages=749-61; " /><meta name="citation_reference" content="citation_title=The structure and function of proteins involved in mammalian pre-mRNA splicing; citation_author=Kramer  A.;  citation_journal_title=Annu. Rev. Biochem;  citation_year=1996;  citation_volume=65;  citation_pages=367-409; " /><meta name="citation_reference" content="citation_title=Systems perspectives on mRNA processing; citation_author=McKee  A.E.; citation_author=Silver  P.A.;  citation_journal_title=Cell Res;  citation_year=2007;  citation_volume=17;  citation_pages=581-90; " /><meta name="citation_reference" content="citation_title=Crystal structure of a self-spliced group II intron; citation_author=Toor  N.; citation_author=Keating  K.S.; citation_author=Taylor  S.D.; citation_author=Pyle  A.M.;  citation_journal_title=Science;  citation_year=2008;  citation_volume=320;  citation_pages=77-82; " /><meta name="citation_reference" content="citation_title=The spliceosome: design principles of a dynamic RNP machine; citation_author=Wahl  M.C.; citation_author=Will  C.L.; citation_author=Lührmann  R.;  citation_journal_title=Cell;  citation_year=2009;  citation_volume=136;  citation_pages=701-18; " /><meta name="citation_reference" content="citation_title=Origins and evolution of spliceosomal introns; citation_author=Rodríguez-Trelles  F.; citation_author=Tarrío  R.; citation_author=Ayala  F.J.;  citation_journal_title=Annu. Rev. Genet;  citation_year=2006;  citation_volume=40;  citation_pages=47-76; " /><meta name="citation_reference" content="citation_title=Origin and evolution of spliceosomal introns; citation_author=Rogozin  I.; citation_author=Carmel  L.; citation_author=Csuros  M.; citation_author=Koonin  E.;  citation_journal_title=Biol. Direct;  citation_year=2012;  citation_volume=7;  citation_pages=11" /><meta name="citation_reference" content="citation_title=Alternative pre-mRNA splicing and proteome expansion in metazoans; citation_author=Maniatis  T.; citation_author=Tasic  B.;  citation_journal_title=Nature;  citation_year=2002;  citation_volume=418;  citation_pages=236-43; " /><meta name="citation_reference" content="citation_title=Mechanisms of alternative pre-messenger RNA splicing; citation_author=Black  D.L.;  citation_journal_title=Annu. Rev. Biochem;  citation_year=2003;  citation_volume=72;  citation_pages=291-336; " /><meta name="citation_reference" content="citation_title=Genomewide comparative analysis of alternative splicing in plants; citation_author=Wang  B.B.; citation_author=Brendel  V.;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2006;  citation_volume=103;  citation_pages=7175-80; " /><meta name="citation_reference" content="citation_title=Different levels of alternative splicing among eukaryotes; citation_author=Kim  E.; citation_author=Magen  A.; citation_author=Ast  G.;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=125-31; " /><meta name="citation_reference" content="citation_title=Alternative splicing of PTC7 in Saccharomyces cerevisiae determines protein localization; citation_author=Juneau  K.; citation_author=Nislow  C.; citation_author=Davis  R.W.;  citation_journal_title=Genetics;  citation_year=2009;  citation_volume=183;  citation_pages=185-94; " /><meta name="citation_reference" content="citation_title=Comparative functional genomics of the fission yeasts; citation_author=Rhind  N.; citation_author=Chen  Z.; citation_author=Yassour  M.;  citation_journal_title=Science;  citation_year=2011;  citation_volume=332;  citation_pages=930-6; " /><meta name="citation_reference" content="citation_title=Alternative splicing and subfunctionalization generates functional diversity in fungal proteomes; citation_author=Marshall  A.N.; citation_author=Montealegre  M.C.; citation_author=Jiménez-López  C.; citation_author=Lorenz  M.C.; citation_author=van Hoof  A.;  citation_journal_title=PLoS Genet;  citation_year=2013;  citation_volume=9;  citation_pages=e1003376" /><meta name="citation_reference" content="citation_title=How did alternative splicing evolve?; citation_author=Ast  G.;  citation_journal_title=Nat. Rev. Genet;  citation_year=2004;  citation_volume=5;  citation_pages=773-82; " /><meta name="citation_reference" content="citation_title=Intron retention is a major phenomenon in alternative splicing in Arabidopsis; citation_author=Ner-Gaon  H.; citation_author=Halachmi  R.; citation_author=Savaldi-Goldstein  S.; citation_author=Rubin  E.; citation_author=Ophir  R.; citation_author=Fluhr  R.;  citation_journal_title=Plant J;  citation_year=2004;  citation_volume=39;  citation_pages=877-85; " /><meta name="citation_reference" content="citation_title=Alternative splicing: current perspectives; citation_author=Kim  E.; citation_author=Goren  A.; citation_author=Ast  G.;  citation_journal_title=BioEssays;  citation_year=2008;  citation_volume=30;  citation_pages=38-47; " /><meta name="citation_reference" content="citation_title=Genome-wide analyses of alternative splicing in plants: opportunities and challenges; citation_author=Barbazuk  W.B.; citation_author=Fu  Y.; citation_author=McGinnis  K.M.;  citation_journal_title=Genome Res;  citation_year=2008;  citation_volume=18;  citation_pages=1381-92; " /><meta name="citation_reference" content="citation_title=Alternative splicing and evolution: diversification, exon definition and function; citation_author=Keren  H.; citation_author=Lev-Maor  G.; citation_author=Ast  G.;  citation_journal_title=Nat. Rev. Genet;  citation_year=2010;  citation_volume=11;  citation_pages=345-55; " /><meta name="citation_reference" content="citation_title=RNA structure and the mechanisms of alternative splicing; citation_author=McManus  C.J.; citation_author=Graveley  B.R.;  citation_journal_title=Curr. Opin. Genet. Dev;  citation_year=2011;  citation_volume=21;  citation_pages=373-9; " /><meta name="citation_reference" content="citation_title=Translational control of intron splicing in eukaryotes; citation_author=Jaillon  O.; citation_author=Bouhouche  K.; citation_author=Gout  J.F.;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=451;  citation_pages=359-62; " /><meta name="citation_reference" content="citation_title=Nonsense-mediated mRNA decay: molecular insights and mechanistic variations across species; citation_author=Conti  E.; citation_author=Izaurralde  E.;  citation_journal_title=Curr. Opin. Cell Biol;  citation_year=2005;  citation_volume=17;  citation_pages=316-25; " /><meta name="citation_reference" content="citation_title=Upf1, an RNA helicase required for nonsense-mediated mRNA decay, modulates the transcriptional response to oxidative stress in fission yeast; citation_author=Rodríguez-Gabriel  M.A.; citation_author=Watt  S.; citation_author=Bähler  J.; citation_author=Russell  P.;  citation_journal_title=Mol. Cellular Biol;  citation_year=2006;  citation_volume=26;  citation_pages=6347-56; " /><meta name="citation_reference" content="citation_title=Nonsense-mediated mRNA decay mutation in Aspergillus nidulans; citation_author=Morozov  I.Y.; citation_author=Negrete-Urtasun  S.; citation_author=Tilburn  J.; citation_author=Jansen  C.A.; citation_author=Caddick  M.X.; citation_author=Arst  H.N.;  citation_journal_title=Eukaryot. Cell;  citation_year=2006;  citation_volume=5;  citation_pages=1838-46; " /><meta name="citation_reference" content="citation_title=Nonsense-mediated mRNA decay (NMD) mechanisms; citation_author=Brogna  S.; citation_author=Wen  J.;  citation_journal_title=Nat. Struct. Mol. Biol;  citation_year=2009;  citation_volume=16;  citation_pages=107-13; " /><meta name="citation_reference" content="citation_title=Regulation of natural mRNAs by the nonsense-mediated mRNA decay pathway; citation_author=Peccarelli  M.; citation_author=Kebaara  B.W.;  citation_journal_title=Eukaryot. Cell;  citation_year=2014;  citation_volume=13;  citation_pages=1126-35; " /><meta name="citation_reference" content="citation_title=Statistical analysis of the exon-intron structure of higher and lower eukaryote genes; citation_author=Kriventseva  E.V.; citation_author=Gelfand  M.S.;  citation_journal_title=J. Biomol. Struct. Dyn;  citation_year=1999;  citation_volume=17;  citation_pages=281-8; " /><meta name="citation_reference" content="citation_title=A computational analysis of sequence features involved in recognition of short introns; citation_author=Lim  L.P.; citation_author=Burge  C.B.;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=11193-8; " /><meta name="citation_reference" content="citation_title=Minimal introns are not “junk”; citation_author=Yu  J.; citation_author=Yang  Z.; citation_author=Kibukawa  M.; citation_author=Paddock  M.; citation_author=Passey  D.A.; citation_author=Wong  G.K.S.;  citation_journal_title=Genome Res;  citation_year=2002;  citation_volume=12;  citation_pages=1185-9; " /><meta name="citation_reference" content="citation_title=Statistical analysis of exon lengths in various eukaryotes; citation_author=Alexander  K.; citation_author=Anatoliy  I.; citation_author=Alexander  B.;  citation_journal_title=Open Access Bioinformatics;  citation_year=2011;  citation_volume=2011;  citation_pages=1-15; " /><meta name="citation_reference" content="citation_title=Genome-wide bioinformatic and molecular analysis of introns in Saccharomyces cerevisiae; citation_author=Spingola  M.; citation_author=Grate  L.; citation_author=Haussler  D.; citation_author=Ares  M.;  citation_journal_title=RNA;  citation_year=1999;  citation_volume=5;  citation_pages=221-34; " /><meta name="citation_reference" content="citation_author=Grate  L.; citation_author=Ares Jr  M.; citation_author=Christine  G.; citation_author=Gerald  R.F.; citation_publisher=Academic Press, imprint of Elsevier Science, San Diego, CA;  citation_title=Methods in Enzymology;  citation_year=2002;  citation_pages=380-92; " /><meta name="citation_reference" content="citation_title=Role of the ubiquitin-like protein Hub1 in splice-site usage and alternative splicing; citation_author=Mishra  S.K.; citation_author=Ammon  T.; citation_author=Popowicz  G.M.;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=474;  citation_pages=173-8; " /><meta name="citation_reference" content="citation_title=A handful of intron-containing genes produces the lion's share of yeast mRNA; citation_author=Ares  M.; citation_author=Grate  L.; citation_author=Pauling  M.H.;  citation_journal_title=RNA;  citation_year=1999;  citation_volume=5;  citation_pages=1138-9; " /><meta name="citation_reference" content="citation_title=Introns regulate RNA and protein abundance in yeast; citation_author=Juneau  K.; citation_author=Miranda  M.; citation_author=Hillenmeyer  M.E.; citation_author=Nislow  C.; citation_author=Davis  R.W.;  citation_journal_title=Genetics;  citation_year=2006;  citation_volume=174;  citation_pages=511-8; " /><meta name="citation_reference" content="citation_title=The genome sequence of Schizosaccharomyces pombe; citation_author=Wood  V.; citation_author=Gwilliam  R.; citation_author=Rajandream  M.A.;  citation_journal_title=Nature;  citation_year=2002;  citation_volume=415;  citation_pages=871-80; " /><meta name="citation_reference" content="citation_title=The fission yeast prp10+ gene involved in pre-mRNA splicing encodes a homologue of highly conserved splicing factor, SAP155; citation_author=Habara  Y.; citation_author=Urushiyama  S.; citation_author=Tani  T.; citation_author=Ohshima  Y.;  citation_journal_title=Nucleic Acids Res;  citation_year=1998;  citation_volume=26;  citation_pages=5662-9; " /><meta name="citation_reference" content="citation_title=mRNAs encoding zinc finger protein isoforms are expressed by alternative splicing of an in-frame intron in fission yeast; citation_author=Okazaki  K.; citation_author=Niwa  O.;  citation_journal_title=DNA Res;  citation_year=2000;  citation_volume=7;  citation_pages=27-30; " /><meta name="citation_reference" content="citation_title=Introns and splicing elements of five diverse fungi; citation_author=Kupfer  D.M.; citation_author=Drabenstot  S.D.; citation_author=Buchanan  K.L.;  citation_journal_title=Eukaryot. Cell;  citation_year=2004;  citation_volume=3;  citation_pages=1088-100; " /><meta name="citation_reference" content="citation_title=Computational and experimental approaches double the number of known introns in the pathogenic yeast Candida albicans; citation_author=Mitrovich  Q.M.; citation_author=Tuch  B.B.; citation_author=Guthrie  C.; citation_author=Johnson  A.D.;  citation_journal_title=Genome Res;  citation_year=2007;  citation_volume=17;  citation_pages=492-502; " /><meta name="citation_reference" content="citation_title=Comparative genomic analysis of fungal genomes reveals intron-rich ancestors; citation_author=Stajich  J.; citation_author=Dietrich  F.; citation_author=Roy  S.;  citation_journal_title=Genome Biol;  citation_year=2007;  citation_volume=8;  citation_pages=R223" /><meta name="citation_reference" content="citation_title=Patterns of intron gain and conservation in eukaryotic genes; citation_author=Carmel  L.; citation_author=Rogozin  I.B.; citation_author=Wolf  Y.I.; citation_author=Koonin  E.V.;  citation_journal_title=BMC Evol. Biol;  citation_year=2007;  citation_volume=7;  citation_pages=192" /><meta name="citation_reference" content="citation_title=Complex spliceosomal organization ancestral to extant eukaryotes; citation_author=Collins  L.; citation_author=Penny  D.;  citation_journal_title=Mol. Biol. Evol;  citation_year=2005;  citation_volume=22;  citation_pages=1053-66; " /><meta name="citation_reference" content="citation_title=Nucleomorph genome of Hemiselmis andersenii reveals complete intron loss and compaction as a driver of protein structure and function; citation_author=Lane  C.E.; citation_author=van den Heuvel  K.; citation_author=Kozera  C.;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2007;  citation_volume=104;  citation_pages=19908-13; " /><meta name="citation_reference" content="citation_title=Role of RNA structure in regulating pre-mRNA splicing; citation_author=Warf  M.B.; citation_author=Berglund  J.A.;  citation_journal_title=Trends Biochem. Sci;  citation_year=2010;  citation_volume=35;  citation_pages=169-78; " /><meta name="citation_reference" content="citation_title=Introns within ribosomal protein genes regulate the production and function of yeast ribosomes; citation_author=Parenteau  J.; citation_author=Durand  M.; citation_author=Morin  G.;  citation_journal_title=Cell;  citation_year=2011;  citation_volume=147;  citation_pages=320-31; " /><meta name="citation_reference" content="citation_author=Berbee  M.L.; citation_author=Taylor  J.W.; citation_publisher=Springer-Verlag, Berlin Heidelberg;  citation_title=Systematics Evolution;  citation_year=2001;  citation_pages=229-45; " /><meta name="citation_reference" content="citation_title=Dating the molecular clock in fungi—how close are we?; citation_author=Berbee  M.L.; citation_author=Taylor  J.W.;  citation_journal_title=Fungal Biol. Rev;  citation_year=2010;  citation_volume=24;  citation_pages=1-16; " /><meta name="citation_reference" content="citation_title=Dating divergences in the Fungal Tree of Life: review and new analyses; citation_author=Taylor  J.W.; citation_author=Berbee  M.L.;  citation_journal_title=Mycologia;  citation_year=2006;  citation_volume=98;  citation_pages=838-49; " /><meta name="citation_reference" content="citation_title=SGD: Saccharomyces Genome Database; citation_author=Cherry  J.M.; citation_author=Adler  C.; citation_author=Ball  C.;  citation_journal_title=Nucleic Acids Res;  citation_year=1998;  citation_volume=26;  citation_pages=73-9; " /><meta name="citation_reference" content="citation_title=PomBase: a comprehensive online resource for fission yeast; citation_author=Wood  V.; citation_author=Harris  M.A.; citation_author=McDowall  M.D.;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=D695-9; " /><meta name="citation_reference" content="citation_title=The Aspergillus Genome Database (AspGD): recent developments in comprehensive multispecies curation, comparative genomics and community resources; citation_author=Arnaud  M.B.; citation_author=Cerqueira  G.C.; citation_author=Inglis  D.O.;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=D653-9; " /><meta name="citation_reference" content="citation_title=The Candida genome database incorporates multiple Candida species: multispecies search and analysis tools with curated gene and protein information for Candida albicans and Candida glabrata; citation_author=Inglis  D.O.; citation_author=Arnaud  M.B.; citation_author=Binkley  J.;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=D667-D674; " /><meta name="citation_reference" content="citation_title=PaxDb, a database of protein abundance averages across all three domains of life; citation_author=Wang  M.; citation_author=Weiss  M.; citation_author=Simonovic  M.;  citation_journal_title=Mol. Cell. Proteomics;  citation_year=2012;  citation_volume=11;  citation_pages=492-500; " /><meta name="citation_reference" content="citation_title=Precision and functional specificity in mRNA decay; citation_author=Wang  Y.; citation_author=Liu  C.L.; citation_author=Storey  J.D.; citation_author=Tibshirani  R.J.; citation_author=Herschlag  D.; citation_author=Brown  P.O.;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=5860-5; " /><meta name="citation_reference" content="citation_title=The transcriptional landscape of the yeast genome defined by RNA sequencing; citation_author=Nagalakshmi  U.; citation_author=Wang  Z.; citation_author=Waern  K.;  citation_journal_title=Science;  citation_year=2008;  citation_volume=320;  citation_pages=1344-9; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling; citation_author=Ingolia  N.T.; citation_author=Ghaemmaghami  S.; citation_author=Newman  J.R.S.; citation_author=Weissman  J.S.;  citation_journal_title=Science;  citation_year=2009;  citation_volume=324;  citation_pages=218-23; " /><meta name="citation_reference" content="citation_title=Energy constraints on the evolution of gene expression; citation_author=Wagner  A.;  citation_journal_title=Mol. Biol. Evol;  citation_year=2005;  citation_volume=22;  citation_pages=1365-74; " /><meta name="citation_reference" content="citation_title=The codon adaptation index-a measure of directional synonymous codon usage bias, and its potential applications; citation_author=Sharp  P.M.; citation_author=Li  W.H.;  citation_journal_title=Nucleic Acids Res;  citation_year=1987;  citation_volume=15;  citation_pages=1281-95; " /><meta name="citation_reference" content="citation_title=The effect of tRNA levels on decoding times of mRNA codons; citation_author=Dana  A.; citation_author=Tuller  T.;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=9171-81; " /><meta name="citation_reference" content="citation_title=Mean of the typical decoding rates: a new translation efficiency index based on the analysis of ribosome profiling data; citation_author=Dana  A.; citation_author=Tuller  T.;  citation_journal_title=G3;  citation_year=2015;  citation_volume=5;  citation_pages=73-80; " /><meta name="citation_reference" content="citation_title=Translation efficiency is determined by both codon bias and folding energy; citation_author=Tuller  T.; citation_author=Waldman  Y.Y.; citation_author=Kupiec  M.; citation_author=Ruppin  E.;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=3645-50; " /><meta name="citation_reference" content="citation_title=GtRNAdb: a database of transfer RNA genes detected in genomic sequence; citation_author=Chan  P.P.; citation_author=Lowe  T.M.;  citation_journal_title=Nucleic Acids Res;  citation_year=2009;  citation_volume=37;  citation_pages=D93-7; " /><meta name="citation_reference" content="citation_title=Differential translation efficiency of orthologous genes is involved in phenotypic divergence of yeast species; citation_author=Man  O.; citation_author=Pilpel  Y.;  citation_journal_title=Nat. Genet;  citation_year=2007;  citation_volume=39;  citation_pages=415-21; " /><meta name="citation_reference" content="citation_title=Modelling the efficiency of codon–tRNA interactions based on codon usage bias; citation_author=Sabi  R.; citation_author=Tuller  T.;  citation_journal_title=DNA Res;  citation_year=2014;  citation_volume=21;  citation_pages=511-26; " /><meta name="citation_reference" content="citation_title=Gene expression omnibus: NCBI gene expression and hybridization array data repository; citation_author=Edgar  R.; citation_author=Domrachev  M.; citation_author=Lash  A.E.;  citation_journal_title=Nucleic Acids Res;  citation_year=2002;  citation_volume=30;  citation_pages=207-10; " /><meta name="citation_reference" content="citation_author=Ingolia  N.T.; citation_author=Jonathan  W.; citation_author=Christine  G.; citation_author=Gerald  R.F.; citation_publisher=Elsevier inc, ;  citation_title=Methods in Enzymology;  citation_year=2010;  citation_pages=119-42; " /><meta name="citation_reference" content="citation_title=BioMart: driving a paradigm change in biological data management; citation_author=Kasprzyk  A.;  citation_journal_title=Database;  citation_year=2011;  citation_volume=2011;  citation_pages=bar049" /><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2; citation_author=Langmead  B.; citation_author=Salzberg  S.L.;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=357-9; " /><meta name="citation_reference" content="citation_title=Global analysis of protein expression in yeast; citation_author=Ghaemmaghami  S.; citation_author=Huh  W.K.; citation_author=Bower  K.;  citation_journal_title=Nature;  citation_year=2003;  citation_volume=425;  citation_pages=737-41; " /><meta name="citation_reference" content="citation_title=Accurate, model-based tuning of synthetic gene expression using introns in S. cerevisiae; citation_author=Yofe  I.; citation_author=Zafrir  Z.; citation_author=Blau  R.;  citation_journal_title=PLoS Genet;  citation_year=2014;  citation_volume=10;  citation_pages=e1004407" /><meta name="citation_reference" content="citation_title=Recursive construction of perfect DNA molecules from imperfect oligonucleotides; citation_author=Linshiz  G.; citation_author=Yehezkel  T.B.; citation_author=Kaplan  S.;  citation_journal_title=Mol. Syst. Biol;  citation_year=2008;  citation_volume=4;  citation_pages=191-200; " /><meta name="citation_reference" content="citation_title=Processing DNA molecules as text; citation_author=Shabi  U.; citation_author=Kaplan  S.; citation_author=Linshiz  G.;  citation_journal_title=Syst. Synth. Biol;  citation_year=2010;  citation_volume=4;  citation_pages=227-36; " /><meta name="citation_reference" content="citation_title=Coding-sequence determinants of gene expression in Escherichia coli; citation_author=Kudla  G.; citation_author=Murray  A.W.; citation_author=Tollervey  D.; citation_author=Plotkin  J.B.;  citation_journal_title=Science;  citation_year=2009;  citation_volume=324;  citation_pages=255-8; " /><meta name="citation_reference" content="citation_title=Estimating Translational Selection in Eukaryotic Genomes; citation_author=dos Reis  M.; citation_author=Wernisch  L.;  citation_journal_title=Mol. Biol. Evol;  citation_year=2009;  citation_volume=26;  citation_pages=451-61; " /><meta name="citation_reference" content="citation_title=An evolutionarily conserved mechanism for controlling the efficiency of protein translation; citation_author=Tuller  T.; citation_author=Carmi  A.; citation_author=Vestsigian  K.;  citation_journal_title=Cell;  citation_year=2010;  citation_volume=141;  citation_pages=344-54; " /><meta name="citation_reference" content="citation_title=A role for codon order in translation dynamics; citation_author=Cannarozzi  G.; citation_author=Schraudolph  N.N.; citation_author=Faty  M.;  citation_journal_title=Cell;  citation_year=2010;  citation_volume=141;  citation_pages=355-67; " /><meta name="citation_reference" content="citation_title=Speeding with control: codon usage, tRNAs, and ribosomes; citation_author=Novoa  E.M.; citation_author=Ribas de Pouplana  L.;  citation_journal_title=Trends Genet;  citation_year=2012;  citation_volume=28;  citation_pages=574-81; " /><meta name="citation_reference" content="citation_title=Selection on codon bias; citation_author=Hershberg  R.; citation_author=Petrov  D.A.;  citation_journal_title=Annu. Rev. Genet;  citation_year=2008;  citation_volume=42;  citation_pages=287-99; " /><meta name="citation_reference" content="citation_title=Synonymous but not the same: the causes and consequences of codon bias; citation_author=Plotkin  J.B.; citation_author=Kudla  G.;  citation_journal_title=Nat. Rev. Genet;  citation_year=2011;  citation_volume=12;  citation_pages=32-42; " /><meta name="citation_reference" content="citation_title=Solving the riddle of codon usage preferences: a test for translational selection; citation_author=Reis  M.D.; citation_author=Savva  R.; citation_author=Wernisch  L.;  citation_journal_title=Nucleic Acids Res;  citation_year=2004;  citation_volume=32;  citation_pages=5036-44; " /><meta name="citation_reference" content="citation_title=The biology of intron gain and loss; citation_author=Jeffares  D.C.; citation_author=Mourier  T.; citation_author=Penny  D.;  citation_journal_title=Trends Genet;  citation_year=2006;  citation_volume=22;  citation_pages=16-22; " /><meta name="citation_reference" content="citation_title=The role of reverse transcriptase in intron gain and loss mechanisms; citation_author=Cohen  N.E.; citation_author=Shen  R.; citation_author=Carmel  L.;  citation_journal_title=Mol. Biol. Evol;  citation_year=2012;  citation_volume=29;  citation_pages=179-86; " /><meta name="citation_reference" content="citation_title=Codon usage and tRNA content in unicellular and multicellular organisms; citation_author=Ikemura  T.;  citation_journal_title=Mol. Biol. Evol;  citation_year=1985;  citation_volume=2;  citation_pages=13-34; " /><meta name="citation_reference" content="citation_title=Codon bias and gene expression; citation_author=Kurland  C.G.;  citation_journal_title=FEBS Lett;  citation_year=1991;  citation_volume=285;  citation_pages=165-9; " /><meta name="citation_reference" content="citation_title=Noisy splicing drives mRNA isoform diversity in human cells; citation_author=Pickrell  J.K.; citation_author=Pai  A.A.; citation_author=Gilad  Y.; citation_author=Pritchard  J.K.;  citation_journal_title=PLoS Genet;  citation_year=2010;  citation_volume=6;  citation_pages=e1001236" /><meta name="citation_reference" content="citation_title=Posttranscriptional gene regulation by spatial rearrangement of the 3? Untranslated region; citation_author=Eberle  A.B.; citation_author=Stalder  L.; citation_author=Mathys  H.; citation_author=Orozco  R.Z.; citation_author=Mühlemann  O.;  citation_journal_title=PLoS Biol;  citation_year=2008;  citation_volume=6;  citation_pages=e92" /><meta name="citation_reference" content="citation_title=Execution of nonsense-mediated mRNA decay: what defines a substrate?; citation_author=Rebbapragada  I.; citation_author=Lykke-Andersen  J.;  citation_journal_title=Curr. Opin. Cell Biol;  citation_year=2009;  citation_volume=21;  citation_pages=394-402; " /><meta name="citation_reference" content="citation_title=Rules that govern UPF1 binding to mRNA 3′ UTRs; citation_author=Kurosaki  T.; citation_author=Maquat  L.E.;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2013;  citation_volume=110;  citation_pages=3357-62; " /><meta name="citation_reference" content="citation_title=Global analyses of UPF1 binding and function reveal expanded scope of nonsense-mediated mRNA decay; citation_author=Hurt  J.A.; citation_author=Robertson  A.D.; citation_author=Burge  C.B.;  citation_journal_title=Genome Res;  citation_year=2013;  citation_volume=23;  citation_pages=1636-50; " /><meta name="citation_reference" content="citation_title=Identification of elements in human long 3′ UTRs that inhibit nonsense-mediated decay; citation_author=Toma  K.G.; citation_author=Rebbapragada  I.; citation_author=Durand  S.; citation_author=Lykke-Andersen  J.;  citation_journal_title=RNA;  citation_year=2015;  citation_volume=21;  citation_pages=887-97; " /><meta name="citation_reference" content="citation_title=Mechanism and regulation of the nonsense-mediated decay pathway; citation_author=Hug  N.; citation_author=Longman  D.; citation_author=Cáceres  J.F.;  citation_journal_title=Nucleic Acids Res;  citation_year=2016;  citation_volume=44;  citation_pages=1483-95; " /><meta name="citation_reference" content="citation_title=On relevance of codon usage to expression of synthetic and natural genes in Escherichia coli; citation_author=Supek  F.; citation_author=Šmuc  T.;  citation_journal_title=Genetics;  citation_year=2010;  citation_volume=185;  citation_pages=1129-34; " /><meta name="citation_reference" content="citation_title=Ribosome profiling of mouse embryonic stem cells reveals the complexity and dynamics of mammalian proteomes; citation_author=Ingolia  N.T.; citation_author=Lareau  L.F.; citation_author=Weissman  J.S.;  citation_journal_title=Cell;  citation_year=2011;  citation_volume=147;  citation_pages=789-802; " /><meta name="citation_reference" content="citation_title=The anti-Shine-Dalgarno sequence drives translational pausing and codon choice in bacteria; citation_author=Li  G.W.; citation_author=Oh  E.; citation_author=Weissman  J.S.;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=484;  citation_pages=538-41; " /><meta name="citation_reference" content="citation_title=Balanced codon usage optimizes eukaryotic translational efficiency; citation_author=Qian  W.; citation_author=Yang  J.-R.; citation_author=Pearson  N.M.; citation_author=Maclean  C.; citation_author=Zhang  J.;  citation_journal_title=PLoS Genet;  citation_year=2012;  citation_volume=8;  citation_pages=e1002603" /><meta name="citation_reference" content="citation_title=Determinants of translation elongation speed and ribosomal profiling biases in mouse embryonic stem cells; citation_author=Dana  A.; citation_author=Tuller  T.;  citation_journal_title=PLoS Comput. Biol;  citation_year=2012;  citation_volume=8;  citation_pages=e1002755" /><meta name="citation_reference" content="citation_title=Positively charged residues are the major determinants of ribosomal velocity; citation_author=Charneski  C.A.; citation_author=Hurst  L.D.;  citation_journal_title=PLoS Biol;  citation_year=2013;  citation_volume=11;  citation_pages=e1001508" /><meta name="citation_reference" content="citation_title=Evolution at two levels of gene expression in yeast; citation_author=Artieri  C.G.; citation_author=Fraser  H.B.;  citation_journal_title=Genome Res;  citation_year=2014;  citation_volume=24;  citation_pages=411-21; " /><meta name="citation_reference" content="citation_title=Measurement of average decoding rates of the 61 sense codons in vivo; citation_author=Gardin  J.; citation_author=Yeasmin  R.; citation_author=Yurovsky  A.; citation_author=Cai  Y.; citation_author=Skiena  S.; citation_author=Futcher  B.;  citation_journal_title=eLife;  citation_year=2014;  citation_volume=3; " /><meta name="citation_reference" content="citation_title=Improved ribosome-footprint and mRNA measurements provide insights into dynamics and regulation of yeast translation; citation_author=Weinberg  D.E.; citation_author=Shah  P.; citation_author=Eichhorn  S.W.; citation_author=Hussmann  J.A.; citation_author=Plotkin  J.B.; citation_author=Bartel  D.P.;  citation_journal_title=Cell Rep;  citation_year=2015;  citation_volume=14;  citation_pages=1787-99; " /><meta name="citation_reference" content="citation_title=Rationally designed, heterologous S. cerevisiae transcripts expose novel expression determinants; citation_author=Ben-Yehezkel  T.; citation_author=Atar  S.; citation_author=Zur  H;  citation_journal_title=RNA Biol;  citation_year=2015;  citation_volume=13;  citation_pages=972-84; " /><meta name="citation_reference" content="citation_title=Multiple roles of the coding sequence 5′ end in gene expression regulation; citation_author=Tuller  T.; citation_author=Zur  H.;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=13-28; " /><meta name="citation_reference" content="citation_title=Network motifs: simple building blocks of complex networks; citation_author=Milo  R.; citation_author=Shen-Orr  S.; citation_author=Itzkovitz  S.; citation_author=Kashtan  N.; citation_author=Chklovskii  D.; citation_author=Alon  U.;  citation_journal_title=Science;  citation_year=2002;  citation_volume=298;  citation_pages=824-7; " /><meta name="citation_reference" content="citation_title=Chromosomal organization is shaped by the transcription regulatory network; citation_author=Hershberg  R.; citation_author=Yeger-Lotem  E.; citation_author=Margalit  H.;  citation_journal_title=Trends Genet;  citation_year=2005;  citation_volume=21;  citation_pages=138-42; " /><meta name="citation_reference" content="citation_title=Telomere length as a quantitative trait: genome-wide survey and genetic mapping of telomere length-control genes in yeast; citation_author=Gatbonton  T.; citation_author=Imbesi  M.; citation_author=Nelson  M.;  citation_journal_title=PLoS Genet;  citation_year=2006;  citation_volume=2;  citation_pages=e35" /><meta name="citation_reference" content="citation_title=A universal trend of reduced mRNA stability near the translation-initiation site in prokaryotes and eukaryotes; citation_author=Gu  W.; citation_author=Zhou  T.; citation_author=Wilke  C.O.;  citation_journal_title=PLoS Comput. Biol;  citation_year=2010;  citation_volume=6;  citation_pages=e1000664" /><meta name="citation_reference" content="citation_title=Length bias correction in gene ontology enrichment analysis using logistic regression; citation_author=Mi  G.; citation_author=Di  Y.; citation_author=Emerson  S.; citation_author=Cumbie  J.S.; citation_author=Chang  J.H.;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e46128" /><meta name="citation_reference" content="citation_title=Assessing the fraction of short-distance tandem splice sites under purifying selection; citation_author=Hiller  M.; citation_author=Szafranski  K.; citation_author=Sinha  R.;  citation_journal_title=RNA;  citation_year=2008;  citation_volume=14;  citation_pages=616-29; " /><meta name="citation_reference" content="citation_title=Measuring the prevalence of regional mutation rates: an analysis of silent substitutions in mammals, fungi, and insects; citation_author=Fox  A.; citation_author=Tuch  B.; citation_author=Chuang  J.;  citation_journal_title=BMC Evol. Biol;  citation_year=2008;  citation_volume=8;  citation_pages=186" /><meta name="citation_reference" content="citation_title=The effect of variation in the effective population size on the rate of adaptive molecular evolution in eukaryotes; citation_author=Gossmann  T.I.; citation_author=Keightley  P.D.; citation_author=Eyre-Walker  A.;  citation_journal_title=Genome Biol. Evol;  citation_year=2012;  citation_volume=4;  citation_pages=658-67; " /><meta name="citation_reference" content="citation_title=Exon and intron definition in pre-mRNA splicing; citation_author=De Conti  L.; citation_author=Baralle  M.; citation_author=Buratti  E.;  citation_journal_title=Wiley Interdiscip. Rev;  citation_year=2013;  citation_volume=4;  citation_pages=49-60; " /><meta name="citation_reference" content="citation_title=Population size and the rate of evolution; citation_author=Lanfear  R.; citation_author=Kokko  H.; citation_author=Eyre-Walker  A.;  citation_journal_title=Trends Ecol. Evol;  citation_year=2014;  citation_volume=29;  citation_pages=33-41; " /><meta name="citation_reference" content="citation_title=Splice-site pairing is an intrinsically high fidelity process; citation_author=Fox-Walsh  K.L.; citation_author=Hertel  K.J.;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=1766-71; " /><meta name="citation_reference" content="citation_title=An abundance of ubiquitously expressed genes revealed by tissue transcriptome sequence data; citation_author=Ramsköld  D.; citation_author=Wang  E.T.; citation_author=Burge  C.B.; citation_author=Sandberg  R.;  citation_journal_title=PLoS Comput. Biol;  citation_year=2009;  citation_volume=5;  citation_pages=e1000598" /><meta name="citation_reference" content="citation_title=Global quantification of mammalian gene expression control; citation_author=Schwanhausser  B.; citation_author=Busse  D.; citation_author=Li  N.;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=473;  citation_pages=337-42; " /><meta name="citation_reference" content="citation_title=Insights into the regulation of protein abundance from proteomic and transcriptomic analyses; citation_author=Vogel  C.; citation_author=Marcotte  E.M.;  citation_journal_title=Nat. Rev. Genet;  citation_year=2012;  citation_volume=13;  citation_pages=227-32; " /><meta name="citation_reference" content="citation_title=Nonsense-mediated mRNA decay: terminating erroneous gene expression; citation_author=Baker  K.E.; citation_author=Parker  R.;  citation_journal_title=Curr. Opin. Cell Biol;  citation_year=2004;  citation_volume=16;  citation_pages=293-9; " /><meta name="citation_reference" content="citation_author=Mayr  E.; citation_publisher=Basic Books, Perseus Books Group, New York;  citation_title=What Evolution Is;  citation_year=2001; " /><meta name="citation_reference" content="citation_title=Evolution and tinkering; citation_author=Jacob  F.;  citation_journal_title=Science;  citation_year=1977;  citation_volume=196;  citation_pages=1161" /><meta name="citation_reference" content="citation_author=Pigliucci  M.; citation_author=Kaplan  J.; citation_publisher=University of Chicago Press, Chicago;  citation_title=Making Sense of Evolution: The Conceptual Foundations of Evolutionary Theory;  citation_year=2006; " /><meta name="citation_reference" content="citation_title=Hearing silence: non-neutral evolution at synonymous sites in mammals; citation_author=Chamary  J.V.; citation_author=Parmley  J.L.; citation_author=Hurst  L.D.;  citation_journal_title=Nat. Rev. Genet;  citation_year=2006;  citation_volume=7;  citation_pages=98-108; " /><meta name="citation_reference" content="citation_title=Understanding the contribution of synonymous mutations to human disease; citation_author=Sauna  Z.E.; citation_author=Kimchi-Sarfaty  C.;  citation_journal_title=Nat. Rev. Genet;  citation_year=2011;  citation_volume=12;  citation_pages=683-91; " /><meta name="citation_reference" content="citation_title=New universal rules of eukaryotic translation initiation fidelity; citation_author=Zur  H.; citation_author=Tuller  T.;  citation_journal_title=PLoS Comput. Biol;  citation_year=2013;  citation_volume=9;  citation_pages=e1003136" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/4/377/17462933/dsw019.pdf" /><meta name="description" content="Abstract.   It is generally believed that introns are not translated; therefore, the potential intronic features that may be related to the translation step (oc" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/4/377/2469962" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/4/377/2469962" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/4/377/2469962");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw019';
    </script>
<title>
	Selection for reduced translation costs at the intronic 5′ end in fungi | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2469962" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZ7NAjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWmFmcmlyLCBab2hhciIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBEZXBhcnRtZW50IG9mIEJpb21lZGljYWwgRW5naW5lZXJpbmcsIFRlbCBBdml2IFVuaXZlcnNpdHksIFRlbCBBdml2LCBJc3JhZWwgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWnVyLCBIYWRhcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBEZXBhcnRtZW50IG9mIEJpb21lZGljYWwgRW5naW5lZXJpbmcsIFRlbCBBdml2IFVuaXZlcnNpdHksIFRlbCBBdml2LCBJc3JhZWwgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVHVsbGVyLCBUYW1pciIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IiBEZXBhcnRtZW50IG9mIEJpb21lZGljYWwgRW5naW5lZXJpbmcsIFRlbCBBdml2IFVuaXZlcnNpdHksIFRlbCBBdml2LCBJc3JhZWwgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iIFRoZSBTYWdvbCBTY2hvb2wgb2YgTmV1cm9zY2llbmNlLCBUZWwgQXZpdiBVbml2ZXJzaXR5LCBUZWwgQXZpdiA2OTk3OCwgSXNyYWVsICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iU2VsZWN0aW9uIGZvciByZWR1Y2VkIHRyYW5zbGF0aW9uIGNvc3RzIGF0IHRoZSBpbnRyb25pYyA14oCyIGVuZCBpbiBmdW5naSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjM3NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMzk0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN3MDE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTYvMDgvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc3BsaWNlb3NvbWU6IHRoZSBtb3N0IGNvbXBsZXggbWFjcm9tb2xlY3VsYXIgbWFjaGluZSBpbiB0aGUgY2VsbD87IGNpdGF0aW9uX2F1dGhvcj1OaWxzZW4gIFQuVy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb0Vzc2F5czsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTExNDctOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHNwbGljZW9zb21lOiBhIGZsZXhpYmxlLCByZXZlcnNpYmxlIG1hY3JvbW9sZWN1bGFyIG1hY2hpbmU7IGNpdGF0aW9uX2F1dGhvcj1Ib3NraW5zICBBLkEuOyBjaXRhdGlvbl9hdXRob3I9TW9vcmUgIE0uSi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBCaW9jaGVtLiBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz0xNzktODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhvdyBpbnRyb25zIGluZmx1ZW5jZSBhbmQgZW5oYW5jZSBldWthcnlvdGljIGdlbmUgZXhwcmVzc2lvbjsgY2l0YXRpb25fYXV0aG9yPUxlIEhpciAgSC47IGNpdGF0aW9uX2F1dGhvcj1Ob3R0ICBBLjsgY2l0YXRpb25fYXV0aG9yPU1vb3JlICBNLkouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgQmlvY2hlbS4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MjE1LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TcGxpY2luZyBpbiBkaXNlYXNlOiBkaXNydXB0aW9uIG9mIHRoZSBzcGxpY2luZyBjb2RlIGFuZCB0aGUgZGVjb2RpbmcgbWFjaGluZXJ5OyBjaXRhdGlvbl9hdXRob3I9V2FuZyAgRy5TLjsgY2l0YXRpb25fYXV0aG9yPUNvb3BlciAgVC5BLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz03NDktNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBzdHJ1Y3R1cmUgYW5kIGZ1bmN0aW9uIG9mIHByb3RlaW5zIGludm9sdmVkIGluIG1hbW1hbGlhbiBwcmUtbVJOQSBzcGxpY2luZzsgY2l0YXRpb25fYXV0aG9yPUtyYW1lciAgQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubnUuIFJldi4gQmlvY2hlbTsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT02NTsgIGNpdGF0aW9uX3BhZ2VzPTM2Ny00MDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5c3RlbXMgcGVyc3BlY3RpdmVzIG9uIG1STkEgcHJvY2Vzc2luZzsgY2l0YXRpb25fYXV0aG9yPU1jS2VlICBBLkUuOyBjaXRhdGlvbl9hdXRob3I9U2lsdmVyICBQLkEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTU4MS05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3J5c3RhbCBzdHJ1Y3R1cmUgb2YgYSBzZWxmLXNwbGljZWQgZ3JvdXAgSUkgaW50cm9uOyBjaXRhdGlvbl9hdXRob3I9VG9vciAgTi47IGNpdGF0aW9uX2F1dGhvcj1LZWF0aW5nICBLLlMuOyBjaXRhdGlvbl9hdXRob3I9VGF5bG9yICBTLkQuOyBjaXRhdGlvbl9hdXRob3I9UHlsZSAgQS5NLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zMjA7ICBjaXRhdGlvbl9wYWdlcz03Ny04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHNwbGljZW9zb21lOiBkZXNpZ24gcHJpbmNpcGxlcyBvZiBhIGR5bmFtaWMgUk5QIG1hY2hpbmU7IGNpdGF0aW9uX2F1dGhvcj1XYWhsICBNLkMuOyBjaXRhdGlvbl9hdXRob3I9V2lsbCAgQy5MLjsgY2l0YXRpb25fYXV0aG9yPUzDvGhybWFubiAgUi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTM2OyAgY2l0YXRpb25fcGFnZXM9NzAxLTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcmlnaW5zIGFuZCBldm9sdXRpb24gb2Ygc3BsaWNlb3NvbWFsIGludHJvbnM7IGNpdGF0aW9uX2F1dGhvcj1Sb2Ryw61ndWV6LVRyZWxsZXMgIEYuOyBjaXRhdGlvbl9hdXRob3I9VGFycsOtbyAgUi47IGNpdGF0aW9uX2F1dGhvcj1BeWFsYSAgRi5KLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPTQ3LTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcmlnaW4gYW5kIGV2b2x1dGlvbiBvZiBzcGxpY2Vvc29tYWwgaW50cm9uczsgY2l0YXRpb25fYXV0aG9yPVJvZ296aW4gIEkuOyBjaXRhdGlvbl9hdXRob3I9Q2FybWVsICBMLjsgY2l0YXRpb25fYXV0aG9yPUNzdXJvcyAgTS47IGNpdGF0aW9uX2F1dGhvcj1Lb29uaW4gIEUuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9sLiBEaXJlY3Q7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTExIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWx0ZXJuYXRpdmUgcHJlLW1STkEgc3BsaWNpbmcgYW5kIHByb3Rlb21lIGV4cGFuc2lvbiBpbiBtZXRhem9hbnM7IGNpdGF0aW9uX2F1dGhvcj1NYW5pYXRpcyAgVC47IGNpdGF0aW9uX2F1dGhvcj1UYXNpYyAgQi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT00MTg7ICBjaXRhdGlvbl9wYWdlcz0yMzYtNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lY2hhbmlzbXMgb2YgYWx0ZXJuYXRpdmUgcHJlLW1lc3NlbmdlciBSTkEgc3BsaWNpbmc7IGNpdGF0aW9uX2F1dGhvcj1CbGFjayAgRC5MLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBCaW9jaGVtOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTcyOyAgY2l0YXRpb25fcGFnZXM9MjkxLTMzNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21ld2lkZSBjb21wYXJhdGl2ZSBhbmFseXNpcyBvZiBhbHRlcm5hdGl2ZSBzcGxpY2luZyBpbiBwbGFudHM7IGNpdGF0aW9uX2F1dGhvcj1XYW5nICBCLkIuOyBjaXRhdGlvbl9hdXRob3I9QnJlbmRlbCAgVi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTEwMzsgIGNpdGF0aW9uX3BhZ2VzPTcxNzUtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZmZlcmVudCBsZXZlbHMgb2YgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgYW1vbmcgZXVrYXJ5b3RlczsgY2l0YXRpb25fYXV0aG9yPUtpbSAgRS47IGNpdGF0aW9uX2F1dGhvcj1NYWdlbiAgQS47IGNpdGF0aW9uX2F1dGhvcj1Bc3QgIEcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPTEyNS0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWx0ZXJuYXRpdmUgc3BsaWNpbmcgb2YgUFRDNyBpbiBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWUgZGV0ZXJtaW5lcyBwcm90ZWluIGxvY2FsaXphdGlvbjsgY2l0YXRpb25fYXV0aG9yPUp1bmVhdSAgSy47IGNpdGF0aW9uX2F1dGhvcj1OaXNsb3cgIEMuOyBjaXRhdGlvbl9hdXRob3I9RGF2aXMgIFIuVy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE4MzsgIGNpdGF0aW9uX3BhZ2VzPTE4NS05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgZnVuY3Rpb25hbCBnZW5vbWljcyBvZiB0aGUgZmlzc2lvbiB5ZWFzdHM7IGNpdGF0aW9uX2F1dGhvcj1SaGluZCAgTi47IGNpdGF0aW9uX2F1dGhvcj1DaGVuICBaLjsgY2l0YXRpb25fYXV0aG9yPVlhc3NvdXIgIE0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTMzMjsgIGNpdGF0aW9uX3BhZ2VzPTkzMC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbHRlcm5hdGl2ZSBzcGxpY2luZyBhbmQgc3ViZnVuY3Rpb25hbGl6YXRpb24gZ2VuZXJhdGVzIGZ1bmN0aW9uYWwgZGl2ZXJzaXR5IGluIGZ1bmdhbCBwcm90ZW9tZXM7IGNpdGF0aW9uX2F1dGhvcj1NYXJzaGFsbCAgQS5OLjsgY2l0YXRpb25fYXV0aG9yPU1vbnRlYWxlZ3JlICBNLkMuOyBjaXRhdGlvbl9hdXRob3I9Smltw6luZXotTMOzcGV6ICBDLjsgY2l0YXRpb25fYXV0aG9yPUxvcmVueiAgTS5DLjsgY2l0YXRpb25fYXV0aG9yPXZhbiBIb29mICBBLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDMzNzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ib3cgZGlkIGFsdGVybmF0aXZlIHNwbGljaW5nIGV2b2x2ZT87IGNpdGF0aW9uX2F1dGhvcj1Bc3QgIEcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTc3My04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50cm9uIHJldGVudGlvbiBpcyBhIG1ham9yIHBoZW5vbWVub24gaW4gYWx0ZXJuYXRpdmUgc3BsaWNpbmcgaW4gQXJhYmlkb3BzaXM7IGNpdGF0aW9uX2F1dGhvcj1OZXItR2FvbiAgSC47IGNpdGF0aW9uX2F1dGhvcj1IYWxhY2htaSAgUi47IGNpdGF0aW9uX2F1dGhvcj1TYXZhbGRpLUdvbGRzdGVpbiAgUy47IGNpdGF0aW9uX2F1dGhvcj1SdWJpbiAgRS47IGNpdGF0aW9uX2F1dGhvcj1PcGhpciAgUi47IGNpdGF0aW9uX2F1dGhvcj1GbHVociAgUi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEo7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz04NzctODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdGVybmF0aXZlIHNwbGljaW5nOiBjdXJyZW50IHBlcnNwZWN0aXZlczsgY2l0YXRpb25fYXV0aG9yPUtpbSAgRS47IGNpdGF0aW9uX2F1dGhvcj1Hb3JlbiAgQS47IGNpdGF0aW9uX2F1dGhvcj1Bc3QgIEcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9Fc3NheXM7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0zOC00NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgYW5hbHlzZXMgb2YgYWx0ZXJuYXRpdmUgc3BsaWNpbmcgaW4gcGxhbnRzOiBvcHBvcnR1bml0aWVzIGFuZCBjaGFsbGVuZ2VzOyBjaXRhdGlvbl9hdXRob3I9QmFyYmF6dWsgIFcuQi47IGNpdGF0aW9uX2F1dGhvcj1GdSAgWS47IGNpdGF0aW9uX2F1dGhvcj1NY0dpbm5pcyAgSy5NLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTEzODEtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdGVybmF0aXZlIHNwbGljaW5nIGFuZCBldm9sdXRpb246IGRpdmVyc2lmaWNhdGlvbiwgZXhvbiBkZWZpbml0aW9uIGFuZCBmdW5jdGlvbjsgY2l0YXRpb25fYXV0aG9yPUtlcmVuICBILjsgY2l0YXRpb25fYXV0aG9yPUxldi1NYW9yICBHLjsgY2l0YXRpb25fYXV0aG9yPUFzdCAgRy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTM0NS01NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Uk5BIHN0cnVjdHVyZSBhbmQgdGhlIG1lY2hhbmlzbXMgb2YgYWx0ZXJuYXRpdmUgc3BsaWNpbmc7IGNpdGF0aW9uX2F1dGhvcj1NY01hbnVzICBDLkouOyBjaXRhdGlvbl9hdXRob3I9R3JhdmVsZXkgIEIuUi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIE9waW4uIEdlbmV0LiBEZXY7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0zNzMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNsYXRpb25hbCBjb250cm9sIG9mIGludHJvbiBzcGxpY2luZyBpbiBldWthcnlvdGVzOyBjaXRhdGlvbl9hdXRob3I9SmFpbGxvbiAgTy47IGNpdGF0aW9uX2F1dGhvcj1Cb3Vob3VjaGUgIEsuOyBjaXRhdGlvbl9hdXRob3I9R291dCAgSi5GLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ1MTsgIGNpdGF0aW9uX3BhZ2VzPTM1OS02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm9uc2Vuc2UtbWVkaWF0ZWQgbVJOQSBkZWNheTogbW9sZWN1bGFyIGluc2lnaHRzIGFuZCBtZWNoYW5pc3RpYyB2YXJpYXRpb25zIGFjcm9zcyBzcGVjaWVzOyBjaXRhdGlvbl9hdXRob3I9Q29udGkgIEUuOyBjaXRhdGlvbl9hdXRob3I9SXphdXJyYWxkZSAgRS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIE9waW4uIENlbGwgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTMxNi0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXBmMSwgYW4gUk5BIGhlbGljYXNlIHJlcXVpcmVkIGZvciBub25zZW5zZS1tZWRpYXRlZCBtUk5BIGRlY2F5LCBtb2R1bGF0ZXMgdGhlIHRyYW5zY3JpcHRpb25hbCByZXNwb25zZSB0byBveGlkYXRpdmUgc3RyZXNzIGluIGZpc3Npb24geWVhc3Q7IGNpdGF0aW9uX2F1dGhvcj1Sb2Ryw61ndWV6LUdhYnJpZWwgIE0uQS47IGNpdGF0aW9uX2F1dGhvcj1XYXR0ICBTLjsgY2l0YXRpb25fYXV0aG9yPULDpGhsZXIgIEouOyBjaXRhdGlvbl9hdXRob3I9UnVzc2VsbCAgUC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQ2VsbHVsYXIgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTYzNDctNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vbnNlbnNlLW1lZGlhdGVkIG1STkEgZGVjYXkgbXV0YXRpb24gaW4gQXNwZXJnaWxsdXMgbmlkdWxhbnM7IGNpdGF0aW9uX2F1dGhvcj1Nb3Jvem92ICBJLlkuOyBjaXRhdGlvbl9hdXRob3I9TmVncmV0ZS1VcnRhc3VuICBTLjsgY2l0YXRpb25fYXV0aG9yPVRpbGJ1cm4gIEouOyBjaXRhdGlvbl9hdXRob3I9SmFuc2VuICBDLkEuOyBjaXRhdGlvbl9hdXRob3I9Q2FkZGljayAgTS5YLjsgY2l0YXRpb25fYXV0aG9yPUFyc3QgIEguTi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV1a2FyeW90LiBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xODM4LTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob25zZW5zZS1tZWRpYXRlZCBtUk5BIGRlY2F5IChOTUQpIG1lY2hhbmlzbXM7IGNpdGF0aW9uX2F1dGhvcj1Ccm9nbmEgIFMuOyBjaXRhdGlvbl9hdXRob3I9V2VuICBKLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBTdHJ1Y3QuIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTEwNy0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVndWxhdGlvbiBvZiBuYXR1cmFsIG1STkFzIGJ5IHRoZSBub25zZW5zZS1tZWRpYXRlZCBtUk5BIGRlY2F5IHBhdGh3YXk7IGNpdGF0aW9uX2F1dGhvcj1QZWNjYXJlbGxpICBNLjsgY2l0YXRpb25fYXV0aG9yPUtlYmFhcmEgIEIuVy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV1a2FyeW90LiBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MTEyNi0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhdGlzdGljYWwgYW5hbHlzaXMgb2YgdGhlIGV4b24taW50cm9uIHN0cnVjdHVyZSBvZiBoaWdoZXIgYW5kIGxvd2VyIGV1a2FyeW90ZSBnZW5lczsgY2l0YXRpb25fYXV0aG9yPUtyaXZlbnRzZXZhICBFLlYuOyBjaXRhdGlvbl9hdXRob3I9R2VsZmFuZCAgTS5TLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvbW9sLiBTdHJ1Y3QuIER5bjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTI4MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbXB1dGF0aW9uYWwgYW5hbHlzaXMgb2Ygc2VxdWVuY2UgZmVhdHVyZXMgaW52b2x2ZWQgaW4gcmVjb2duaXRpb24gb2Ygc2hvcnQgaW50cm9uczsgY2l0YXRpb25fYXV0aG9yPUxpbSAgTC5QLjsgY2l0YXRpb25fYXV0aG9yPUJ1cmdlICBDLkIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT05ODsgIGNpdGF0aW9uX3BhZ2VzPTExMTkzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pbmltYWwgaW50cm9ucyBhcmUgbm90IOKAnGp1bmvigJ07IGNpdGF0aW9uX2F1dGhvcj1ZdSAgSi47IGNpdGF0aW9uX2F1dGhvcj1ZYW5nICBaLjsgY2l0YXRpb25fYXV0aG9yPUtpYnVrYXdhICBNLjsgY2l0YXRpb25fYXV0aG9yPVBhZGRvY2sgIE0uOyBjaXRhdGlvbl9hdXRob3I9UGFzc2V5ICBELkEuOyBjaXRhdGlvbl9hdXRob3I9V29uZyAgRy5LLlMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTE4NS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdGF0aXN0aWNhbCBhbmFseXNpcyBvZiBleG9uIGxlbmd0aHMgaW4gdmFyaW91cyBldWthcnlvdGVzOyBjaXRhdGlvbl9hdXRob3I9QWxleGFuZGVyICBLLjsgY2l0YXRpb25fYXV0aG9yPUFuYXRvbGl5ICBJLjsgY2l0YXRpb25fYXV0aG9yPUFsZXhhbmRlciAgQi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU9wZW4gQWNjZXNzIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTIwMTE7ICBjaXRhdGlvbl9wYWdlcz0xLTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBiaW9pbmZvcm1hdGljIGFuZCBtb2xlY3VsYXIgYW5hbHlzaXMgb2YgaW50cm9ucyBpbiBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWU7IGNpdGF0aW9uX2F1dGhvcj1TcGluZ29sYSAgTS47IGNpdGF0aW9uX2F1dGhvcj1HcmF0ZSAgTC47IGNpdGF0aW9uX2F1dGhvcj1IYXVzc2xlciAgRC47IGNpdGF0aW9uX2F1dGhvcj1BcmVzICBNLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Uk5BOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0yMjEtMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX2F1dGhvcj1HcmF0ZSAgTC47IGNpdGF0aW9uX2F1dGhvcj1BcmVzIEpyICBNLjsgY2l0YXRpb25fYXV0aG9yPUNocmlzdGluZSAgRy47IGNpdGF0aW9uX2F1dGhvcj1HZXJhbGQgIFIuRi47IGNpdGF0aW9uX3B1Ymxpc2hlcj1BY2FkZW1pYyBQcmVzcywgaW1wcmludCBvZiBFbHNldmllciBTY2llbmNlLCBTYW4gRGllZ28sIENBOyAgY2l0YXRpb25fdGl0bGU9TWV0aG9kcyBpbiBFbnp5bW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fcGFnZXM9MzgwLTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Sb2xlIG9mIHRoZSB1YmlxdWl0aW4tbGlrZSBwcm90ZWluIEh1YjEgaW4gc3BsaWNlLXNpdGUgdXNhZ2UgYW5kIGFsdGVybmF0aXZlIHNwbGljaW5nOyBjaXRhdGlvbl9hdXRob3I9TWlzaHJhICBTLksuOyBjaXRhdGlvbl9hdXRob3I9QW1tb24gIFQuOyBjaXRhdGlvbl9hdXRob3I9UG9wb3dpY3ogIEcuTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00NzQ7ICBjaXRhdGlvbl9wYWdlcz0xNzMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoYW5kZnVsIG9mIGludHJvbi1jb250YWluaW5nIGdlbmVzIHByb2R1Y2VzIHRoZSBsaW9uJ3Mgc2hhcmUgb2YgeWVhc3QgbVJOQTsgY2l0YXRpb25fYXV0aG9yPUFyZXMgIE0uOyBjaXRhdGlvbl9hdXRob3I9R3JhdGUgIEwuOyBjaXRhdGlvbl9hdXRob3I9UGF1bGluZyAgTS5ILjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Uk5BOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xMTM4LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludHJvbnMgcmVndWxhdGUgUk5BIGFuZCBwcm90ZWluIGFidW5kYW5jZSBpbiB5ZWFzdDsgY2l0YXRpb25fYXV0aG9yPUp1bmVhdSAgSy47IGNpdGF0aW9uX2F1dGhvcj1NaXJhbmRhICBNLjsgY2l0YXRpb25fYXV0aG9yPUhpbGxlbm1leWVyICBNLkUuOyBjaXRhdGlvbl9hdXRob3I9TmlzbG93ICBDLjsgY2l0YXRpb25fYXV0aG9yPURhdmlzICBSLlcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNzQ7ICBjaXRhdGlvbl9wYWdlcz01MTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBzZXF1ZW5jZSBvZiBTY2hpem9zYWNjaGFyb215Y2VzIHBvbWJlOyBjaXRhdGlvbl9hdXRob3I9V29vZCAgVi47IGNpdGF0aW9uX2F1dGhvcj1Hd2lsbGlhbSAgUi47IGNpdGF0aW9uX2F1dGhvcj1SYWphbmRyZWFtICBNLkEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NDE1OyAgY2l0YXRpb25fcGFnZXM9ODcxLTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZmlzc2lvbiB5ZWFzdCBwcnAxMCsgZ2VuZSBpbnZvbHZlZCBpbiBwcmUtbVJOQSBzcGxpY2luZyBlbmNvZGVzIGEgaG9tb2xvZ3VlIG9mIGhpZ2hseSBjb25zZXJ2ZWQgc3BsaWNpbmcgZmFjdG9yLCBTQVAxNTU7IGNpdGF0aW9uX2F1dGhvcj1IYWJhcmEgIFkuOyBjaXRhdGlvbl9hdXRob3I9VXJ1c2hpeWFtYSAgUy47IGNpdGF0aW9uX2F1dGhvcj1UYW5pICBULjsgY2l0YXRpb25fYXV0aG9yPU9oc2hpbWEgIFkuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTU2NjItOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9bVJOQXMgZW5jb2RpbmcgemluYyBmaW5nZXIgcHJvdGVpbiBpc29mb3JtcyBhcmUgZXhwcmVzc2VkIGJ5IGFsdGVybmF0aXZlIHNwbGljaW5nIG9mIGFuIGluLWZyYW1lIGludHJvbiBpbiBmaXNzaW9uIHllYXN0OyBjaXRhdGlvbl9hdXRob3I9T2themFraSAgSy47IGNpdGF0aW9uX2F1dGhvcj1OaXdhICBPLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MjctMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludHJvbnMgYW5kIHNwbGljaW5nIGVsZW1lbnRzIG9mIGZpdmUgZGl2ZXJzZSBmdW5naTsgY2l0YXRpb25fYXV0aG9yPUt1cGZlciAgRC5NLjsgY2l0YXRpb25fYXV0aG9yPURyYWJlbnN0b3QgIFMuRC47IGNpdGF0aW9uX2F1dGhvcj1CdWNoYW5hbiAgSy5MLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXVrYXJ5b3QuIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTEwODgtMTAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wdXRhdGlvbmFsIGFuZCBleHBlcmltZW50YWwgYXBwcm9hY2hlcyBkb3VibGUgdGhlIG51bWJlciBvZiBrbm93biBpbnRyb25zIGluIHRoZSBwYXRob2dlbmljIHllYXN0IENhbmRpZGEgYWxiaWNhbnM7IGNpdGF0aW9uX2F1dGhvcj1NaXRyb3ZpY2ggIFEuTS47IGNpdGF0aW9uX2F1dGhvcj1UdWNoICBCLkIuOyBjaXRhdGlvbl9hdXRob3I9R3V0aHJpZSAgQy47IGNpdGF0aW9uX2F1dGhvcj1Kb2huc29uICBBLkQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9NDkyLTUwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgZ2Vub21pYyBhbmFseXNpcyBvZiBmdW5nYWwgZ2Vub21lcyByZXZlYWxzIGludHJvbi1yaWNoIGFuY2VzdG9yczsgY2l0YXRpb25fYXV0aG9yPVN0YWppY2ggIEouOyBjaXRhdGlvbl9hdXRob3I9RGlldHJpY2ggIEYuOyBjaXRhdGlvbl9hdXRob3I9Um95ICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPVIyMjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYXR0ZXJucyBvZiBpbnRyb24gZ2FpbiBhbmQgY29uc2VydmF0aW9uIGluIGV1a2FyeW90aWMgZ2VuZXM7IGNpdGF0aW9uX2F1dGhvcj1DYXJtZWwgIEwuOyBjaXRhdGlvbl9hdXRob3I9Um9nb3ppbiAgSS5CLjsgY2l0YXRpb25fYXV0aG9yPVdvbGYgIFkuSS47IGNpdGF0aW9uX2F1dGhvcj1Lb29uaW4gIEUuVi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBFdm9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0xOTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV4IHNwbGljZW9zb21hbCBvcmdhbml6YXRpb24gYW5jZXN0cmFsIHRvIGV4dGFudCBldWthcnlvdGVzOyBjaXRhdGlvbl9hdXRob3I9Q29sbGlucyAgTC47IGNpdGF0aW9uX2F1dGhvcj1QZW5ueSAgRC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTEwNTMtNjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU51Y2xlb21vcnBoIGdlbm9tZSBvZiBIZW1pc2VsbWlzIGFuZGVyc2VuaWkgcmV2ZWFscyBjb21wbGV0ZSBpbnRyb24gbG9zcyBhbmQgY29tcGFjdGlvbiBhcyBhIGRyaXZlciBvZiBwcm90ZWluIHN0cnVjdHVyZSBhbmQgZnVuY3Rpb247IGNpdGF0aW9uX2F1dGhvcj1MYW5lICBDLkUuOyBjaXRhdGlvbl9hdXRob3I9dmFuIGRlbiBIZXV2ZWwgIEsuOyBjaXRhdGlvbl9hdXRob3I9S296ZXJhICBDLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTA0OyAgY2l0YXRpb25fcGFnZXM9MTk5MDgtMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJvbGUgb2YgUk5BIHN0cnVjdHVyZSBpbiByZWd1bGF0aW5nIHByZS1tUk5BIHNwbGljaW5nOyBjaXRhdGlvbl9hdXRob3I9V2FyZiAgTS5CLjsgY2l0YXRpb25fYXV0aG9yPUJlcmdsdW5kICBKLkEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgQmlvY2hlbS4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9MTY5LTc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRyb25zIHdpdGhpbiByaWJvc29tYWwgcHJvdGVpbiBnZW5lcyByZWd1bGF0ZSB0aGUgcHJvZHVjdGlvbiBhbmQgZnVuY3Rpb24gb2YgeWVhc3Qgcmlib3NvbWVzOyBjaXRhdGlvbl9hdXRob3I9UGFyZW50ZWF1ICBKLjsgY2l0YXRpb25fYXV0aG9yPUR1cmFuZCAgTS47IGNpdGF0aW9uX2F1dGhvcj1Nb3JpbiAgRy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTQ3OyAgY2l0YXRpb25fcGFnZXM9MzIwLTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9hdXRob3I9QmVyYmVlICBNLkwuOyBjaXRhdGlvbl9hdXRob3I9VGF5bG9yICBKLlcuOyBjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXItVmVybGFnLCBCZXJsaW4gSGVpZGVsYmVyZzsgIGNpdGF0aW9uX3RpdGxlPVN5c3RlbWF0aWNzIEV2b2x1dGlvbjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3BhZ2VzPTIyOS00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGF0aW5nIHRoZSBtb2xlY3VsYXIgY2xvY2sgaW4gZnVuZ2nigJRob3cgY2xvc2UgYXJlIHdlPzsgY2l0YXRpb25fYXV0aG9yPUJlcmJlZSAgTS5MLjsgY2l0YXRpb25fYXV0aG9yPVRheWxvciAgSi5XLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnVuZ2FsIEJpb2wuIFJldjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTEtMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURhdGluZyBkaXZlcmdlbmNlcyBpbiB0aGUgRnVuZ2FsIFRyZWUgb2YgTGlmZTogcmV2aWV3IGFuZCBuZXcgYW5hbHlzZXM7IGNpdGF0aW9uX2F1dGhvcj1UYXlsb3IgIEouVy47IGNpdGF0aW9uX2F1dGhvcj1CZXJiZWUgIE0uTC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU15Y29sb2dpYTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT05ODsgIGNpdGF0aW9uX3BhZ2VzPTgzOC00OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U0dEOiBTYWNjaGFyb215Y2VzIEdlbm9tZSBEYXRhYmFzZTsgY2l0YXRpb25fYXV0aG9yPUNoZXJyeSAgSi5NLjsgY2l0YXRpb25fYXV0aG9yPUFkbGVyICBDLjsgY2l0YXRpb25fYXV0aG9yPUJhbGwgIEMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTczLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvbUJhc2U6IGEgY29tcHJlaGVuc2l2ZSBvbmxpbmUgcmVzb3VyY2UgZm9yIGZpc3Npb24geWVhc3Q7IGNpdGF0aW9uX2F1dGhvcj1Xb29kICBWLjsgY2l0YXRpb25fYXV0aG9yPUhhcnJpcyAgTS5BLjsgY2l0YXRpb25fYXV0aG9yPU1jRG93YWxsICBNLkQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPUQ2OTUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEFzcGVyZ2lsbHVzIEdlbm9tZSBEYXRhYmFzZSAoQXNwR0QpOiByZWNlbnQgZGV2ZWxvcG1lbnRzIGluIGNvbXByZWhlbnNpdmUgbXVsdGlzcGVjaWVzIGN1cmF0aW9uLCBjb21wYXJhdGl2ZSBnZW5vbWljcyBhbmQgY29tbXVuaXR5IHJlc291cmNlczsgY2l0YXRpb25fYXV0aG9yPUFybmF1ZCAgTS5CLjsgY2l0YXRpb25fYXV0aG9yPUNlcnF1ZWlyYSAgRy5DLjsgY2l0YXRpb25fYXV0aG9yPUluZ2xpcyAgRC5PLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz1ENjUzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBDYW5kaWRhIGdlbm9tZSBkYXRhYmFzZSBpbmNvcnBvcmF0ZXMgbXVsdGlwbGUgQ2FuZGlkYSBzcGVjaWVzOiBtdWx0aXNwZWNpZXMgc2VhcmNoIGFuZCBhbmFseXNpcyB0b29scyB3aXRoIGN1cmF0ZWQgZ2VuZSBhbmQgcHJvdGVpbiBpbmZvcm1hdGlvbiBmb3IgQ2FuZGlkYSBhbGJpY2FucyBhbmQgQ2FuZGlkYSBnbGFicmF0YTsgY2l0YXRpb25fYXV0aG9yPUluZ2xpcyAgRC5PLjsgY2l0YXRpb25fYXV0aG9yPUFybmF1ZCAgTS5CLjsgY2l0YXRpb25fYXV0aG9yPUJpbmtsZXkgIEouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPUQ2NjctRDY3NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGF4RGIsIGEgZGF0YWJhc2Ugb2YgcHJvdGVpbiBhYnVuZGFuY2UgYXZlcmFnZXMgYWNyb3NzIGFsbCB0aHJlZSBkb21haW5zIG9mIGxpZmU7IGNpdGF0aW9uX2F1dGhvcj1XYW5nICBNLjsgY2l0YXRpb25fYXV0aG9yPVdlaXNzICBNLjsgY2l0YXRpb25fYXV0aG9yPVNpbW9ub3ZpYyAgTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQ2VsbC4gUHJvdGVvbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTQ5Mi01MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWNpc2lvbiBhbmQgZnVuY3Rpb25hbCBzcGVjaWZpY2l0eSBpbiBtUk5BIGRlY2F5OyBjaXRhdGlvbl9hdXRob3I9V2FuZyAgWS47IGNpdGF0aW9uX2F1dGhvcj1MaXUgIEMuTC47IGNpdGF0aW9uX2F1dGhvcj1TdG9yZXkgIEouRC47IGNpdGF0aW9uX2F1dGhvcj1UaWJzaGlyYW5pICBSLkouOyBjaXRhdGlvbl9hdXRob3I9SGVyc2NobGFnICBELjsgY2l0YXRpb25fYXV0aG9yPUJyb3duICBQLk8uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT05OTsgIGNpdGF0aW9uX3BhZ2VzPTU4NjAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHRyYW5zY3JpcHRpb25hbCBsYW5kc2NhcGUgb2YgdGhlIHllYXN0IGdlbm9tZSBkZWZpbmVkIGJ5IFJOQSBzZXF1ZW5jaW5nOyBjaXRhdGlvbl9hdXRob3I9TmFnYWxha3NobWkgIFUuOyBjaXRhdGlvbl9hdXRob3I9V2FuZyAgWi47IGNpdGF0aW9uX2F1dGhvcj1XYWVybiAgSy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzIwOyAgY2l0YXRpb25fcGFnZXM9MTM0NC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBpbiB2aXZvIG9mIHRyYW5zbGF0aW9uIHdpdGggbnVjbGVvdGlkZSByZXNvbHV0aW9uIHVzaW5nIHJpYm9zb21lIHByb2ZpbGluZzsgY2l0YXRpb25fYXV0aG9yPUluZ29saWEgIE4uVC47IGNpdGF0aW9uX2F1dGhvcj1HaGFlbW1hZ2hhbWkgIFMuOyBjaXRhdGlvbl9hdXRob3I9TmV3bWFuICBKLlIuUy47IGNpdGF0aW9uX2F1dGhvcj1XZWlzc21hbiAgSi5TLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zMjQ7ICBjaXRhdGlvbl9wYWdlcz0yMTgtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVuZXJneSBjb25zdHJhaW50cyBvbiB0aGUgZXZvbHV0aW9uIG9mIGdlbmUgZXhwcmVzc2lvbjsgY2l0YXRpb25fYXV0aG9yPVdhZ25lciAgQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTEzNjUtNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb2RvbiBhZGFwdGF0aW9uIGluZGV4LWEgbWVhc3VyZSBvZiBkaXJlY3Rpb25hbCBzeW5vbnltb3VzIGNvZG9uIHVzYWdlIGJpYXMsIGFuZCBpdHMgcG90ZW50aWFsIGFwcGxpY2F0aW9uczsgY2l0YXRpb25fYXV0aG9yPVNoYXJwICBQLk0uOyBjaXRhdGlvbl9hdXRob3I9TGkgIFcuSC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MTI4MS05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGVmZmVjdCBvZiB0Uk5BIGxldmVscyBvbiBkZWNvZGluZyB0aW1lcyBvZiBtUk5BIGNvZG9uczsgY2l0YXRpb25fYXV0aG9yPURhbmEgIEEuOyBjaXRhdGlvbl9hdXRob3I9VHVsbGVyICBULjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz05MTcxLTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWFuIG9mIHRoZSB0eXBpY2FsIGRlY29kaW5nIHJhdGVzOiBhIG5ldyB0cmFuc2xhdGlvbiBlZmZpY2llbmN5IGluZGV4IGJhc2VkIG9uIHRoZSBhbmFseXNpcyBvZiByaWJvc29tZSBwcm9maWxpbmcgZGF0YTsgY2l0YXRpb25fYXV0aG9yPURhbmEgIEEuOyBjaXRhdGlvbl9hdXRob3I9VHVsbGVyICBULjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RzM7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTczLTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2xhdGlvbiBlZmZpY2llbmN5IGlzIGRldGVybWluZWQgYnkgYm90aCBjb2RvbiBiaWFzIGFuZCBmb2xkaW5nIGVuZXJneTsgY2l0YXRpb25fYXV0aG9yPVR1bGxlciAgVC47IGNpdGF0aW9uX2F1dGhvcj1XYWxkbWFuICBZLlkuOyBjaXRhdGlvbl9hdXRob3I9S3VwaWVjICBNLjsgY2l0YXRpb25fYXV0aG9yPVJ1cHBpbiAgRS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEwNzsgIGNpdGF0aW9uX3BhZ2VzPTM2NDUtNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUd0Uk5BZGI6IGEgZGF0YWJhc2Ugb2YgdHJhbnNmZXIgUk5BIGdlbmVzIGRldGVjdGVkIGluIGdlbm9taWMgc2VxdWVuY2U7IGNpdGF0aW9uX2F1dGhvcj1DaGFuICBQLlAuOyBjaXRhdGlvbl9hdXRob3I9TG93ZSAgVC5NLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz1EOTMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZmVyZW50aWFsIHRyYW5zbGF0aW9uIGVmZmljaWVuY3kgb2Ygb3J0aG9sb2dvdXMgZ2VuZXMgaXMgaW52b2x2ZWQgaW4gcGhlbm90eXBpYyBkaXZlcmdlbmNlIG9mIHllYXN0IHNwZWNpZXM7IGNpdGF0aW9uX2F1dGhvcj1NYW4gIE8uOyBjaXRhdGlvbl9hdXRob3I9UGlscGVsICBZLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPTQxNS0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9kZWxsaW5nIHRoZSBlZmZpY2llbmN5IG9mIGNvZG9u4oCTdFJOQSBpbnRlcmFjdGlvbnMgYmFzZWQgb24gY29kb24gdXNhZ2UgYmlhczsgY2l0YXRpb25fYXV0aG9yPVNhYmkgIFIuOyBjaXRhdGlvbl9hdXRob3I9VHVsbGVyICBULjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTUxMS0yNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBleHByZXNzaW9uIG9tbmlidXM6IE5DQkkgZ2VuZSBleHByZXNzaW9uIGFuZCBoeWJyaWRpemF0aW9uIGFycmF5IGRhdGEgcmVwb3NpdG9yeTsgY2l0YXRpb25fYXV0aG9yPUVkZ2FyICBSLjsgY2l0YXRpb25fYXV0aG9yPURvbXJhY2hldiAgTS47IGNpdGF0aW9uX2F1dGhvcj1MYXNoICBBLkUuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTIwNy0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fYXV0aG9yPUluZ29saWEgIE4uVC47IGNpdGF0aW9uX2F1dGhvcj1Kb25hdGhhbiAgVy47IGNpdGF0aW9uX2F1dGhvcj1DaHJpc3RpbmUgIEcuOyBjaXRhdGlvbl9hdXRob3I9R2VyYWxkICBSLkYuOyBjaXRhdGlvbl9wdWJsaXNoZXI9RWxzZXZpZXIgaW5jLCA7ICBjaXRhdGlvbl90aXRsZT1NZXRob2RzIGluIEVuenltb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl9wYWdlcz0xMTktNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpb01hcnQ6IGRyaXZpbmcgYSBwYXJhZGlnbSBjaGFuZ2UgaW4gYmlvbG9naWNhbCBkYXRhIG1hbmFnZW1lbnQ7IGNpdGF0aW9uX2F1dGhvcj1LYXNwcnp5ayAgQS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPURhdGFiYXNlOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTIwMTE7ICBjaXRhdGlvbl9wYWdlcz1iYXIwNDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GYXN0IGdhcHBlZC1yZWFkIGFsaWdubWVudCB3aXRoIEJvd3RpZSAyOyBjaXRhdGlvbl9hdXRob3I9TGFuZ21lYWQgIEIuOyBjaXRhdGlvbl9hdXRob3I9U2FsemJlcmcgIFMuTC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MzU3LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdsb2JhbCBhbmFseXNpcyBvZiBwcm90ZWluIGV4cHJlc3Npb24gaW4geWVhc3Q7IGNpdGF0aW9uX2F1dGhvcj1HaGFlbW1hZ2hhbWkgIFMuOyBjaXRhdGlvbl9hdXRob3I9SHVoICBXLksuOyBjaXRhdGlvbl9hdXRob3I9Qm93ZXIgIEsuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDI1OyAgY2l0YXRpb25fcGFnZXM9NzM3LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY2N1cmF0ZSwgbW9kZWwtYmFzZWQgdHVuaW5nIG9mIHN5bnRoZXRpYyBnZW5lIGV4cHJlc3Npb24gdXNpbmcgaW50cm9ucyBpbiBTLiBjZXJldmlzaWFlOyBjaXRhdGlvbl9hdXRob3I9WW9mZSAgSS47IGNpdGF0aW9uX2F1dGhvcj1aYWZyaXIgIFouOyBjaXRhdGlvbl9hdXRob3I9QmxhdSAgUi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1lMTAwNDQwNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlY3Vyc2l2ZSBjb25zdHJ1Y3Rpb24gb2YgcGVyZmVjdCBETkEgbW9sZWN1bGVzIGZyb20gaW1wZXJmZWN0IG9saWdvbnVjbGVvdGlkZXM7IGNpdGF0aW9uX2F1dGhvcj1MaW5zaGl6ICBHLjsgY2l0YXRpb25fYXV0aG9yPVllaGV6a2VsICBULkIuOyBjaXRhdGlvbl9hdXRob3I9S2FwbGFuICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBTeXN0LiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0xOTEtMjAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm9jZXNzaW5nIEROQSBtb2xlY3VsZXMgYXMgdGV4dDsgY2l0YXRpb25fYXV0aG9yPVNoYWJpICBVLjsgY2l0YXRpb25fYXV0aG9yPUthcGxhbiAgUy47IGNpdGF0aW9uX2F1dGhvcj1MaW5zaGl6ICBHLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U3lzdC4gU3ludGguIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTIyNy0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kaW5nLXNlcXVlbmNlIGRldGVybWluYW50cyBvZiBnZW5lIGV4cHJlc3Npb24gaW4gRXNjaGVyaWNoaWEgY29saTsgY2l0YXRpb25fYXV0aG9yPUt1ZGxhICBHLjsgY2l0YXRpb25fYXV0aG9yPU11cnJheSAgQS5XLjsgY2l0YXRpb25fYXV0aG9yPVRvbGxlcnZleSAgRC47IGNpdGF0aW9uX2F1dGhvcj1QbG90a2luICBKLkIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTMyNDsgIGNpdGF0aW9uX3BhZ2VzPTI1NS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fc3RpbWF0aW5nIFRyYW5zbGF0aW9uYWwgU2VsZWN0aW9uIGluIEV1a2FyeW90aWMgR2Vub21lczsgY2l0YXRpb25fYXV0aG9yPWRvcyBSZWlzICBNLjsgY2l0YXRpb25fYXV0aG9yPVdlcm5pc2NoICBMLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9NDUxLTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBldm9sdXRpb25hcmlseSBjb25zZXJ2ZWQgbWVjaGFuaXNtIGZvciBjb250cm9sbGluZyB0aGUgZWZmaWNpZW5jeSBvZiBwcm90ZWluIHRyYW5zbGF0aW9uOyBjaXRhdGlvbl9hdXRob3I9VHVsbGVyICBULjsgY2l0YXRpb25fYXV0aG9yPUNhcm1pICBBLjsgY2l0YXRpb25fYXV0aG9yPVZlc3RzaWdpYW4gIEsuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE0MTsgIGNpdGF0aW9uX3BhZ2VzPTM0NC01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSByb2xlIGZvciBjb2RvbiBvcmRlciBpbiB0cmFuc2xhdGlvbiBkeW5hbWljczsgY2l0YXRpb25fYXV0aG9yPUNhbm5hcm96emkgIEcuOyBjaXRhdGlvbl9hdXRob3I9U2NocmF1ZG9scGggIE4uTi47IGNpdGF0aW9uX2F1dGhvcj1GYXR5ICBNLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDE7ICBjaXRhdGlvbl9wYWdlcz0zNTUtNjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwZWVkaW5nIHdpdGggY29udHJvbDogY29kb24gdXNhZ2UsIHRSTkFzLCBhbmQgcmlib3NvbWVzOyBjaXRhdGlvbl9hdXRob3I9Tm92b2EgIEUuTS47IGNpdGF0aW9uX2F1dGhvcj1SaWJhcyBkZSBQb3VwbGFuYSAgTC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTU3NC04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VsZWN0aW9uIG9uIGNvZG9uIGJpYXM7IGNpdGF0aW9uX2F1dGhvcj1IZXJzaGJlcmcgIFIuOyBjaXRhdGlvbl9hdXRob3I9UGV0cm92ICBELkEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9Mjg3LTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeW5vbnltb3VzIGJ1dCBub3QgdGhlIHNhbWU6IHRoZSBjYXVzZXMgYW5kIGNvbnNlcXVlbmNlcyBvZiBjb2RvbiBiaWFzOyBjaXRhdGlvbl9hdXRob3I9UGxvdGtpbiAgSi5CLjsgY2l0YXRpb25fYXV0aG9yPUt1ZGxhICBHLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MzItNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNvbHZpbmcgdGhlIHJpZGRsZSBvZiBjb2RvbiB1c2FnZSBwcmVmZXJlbmNlczogYSB0ZXN0IGZvciB0cmFuc2xhdGlvbmFsIHNlbGVjdGlvbjsgY2l0YXRpb25fYXV0aG9yPVJlaXMgIE0uRC47IGNpdGF0aW9uX2F1dGhvcj1TYXZ2YSAgUi47IGNpdGF0aW9uX2F1dGhvcj1XZXJuaXNjaCAgTC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9NTAzNi00NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGJpb2xvZ3kgb2YgaW50cm9uIGdhaW4gYW5kIGxvc3M7IGNpdGF0aW9uX2F1dGhvcj1KZWZmYXJlcyAgRC5DLjsgY2l0YXRpb25fYXV0aG9yPU1vdXJpZXIgIFQuOyBjaXRhdGlvbl9hdXRob3I9UGVubnkgIEQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0xNi0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJvbGUgb2YgcmV2ZXJzZSB0cmFuc2NyaXB0YXNlIGluIGludHJvbiBnYWluIGFuZCBsb3NzIG1lY2hhbmlzbXM7IGNpdGF0aW9uX2F1dGhvcj1Db2hlbiAgTi5FLjsgY2l0YXRpb25fYXV0aG9yPVNoZW4gIFIuOyBjaXRhdGlvbl9hdXRob3I9Q2FybWVsICBMLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MTc5LTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiB1c2FnZSBhbmQgdFJOQSBjb250ZW50IGluIHVuaWNlbGx1bGFyIGFuZCBtdWx0aWNlbGx1bGFyIG9yZ2FuaXNtczsgY2l0YXRpb25fYXV0aG9yPUlrZW11cmEgIFQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTE5ODU7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTEzLTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiBiaWFzIGFuZCBnZW5lIGV4cHJlc3Npb247IGNpdGF0aW9uX2F1dGhvcj1LdXJsYW5kICBDLkcuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRUJTIExldHQ7ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9Mjg1OyAgY2l0YXRpb25fcGFnZXM9MTY1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vaXN5IHNwbGljaW5nIGRyaXZlcyBtUk5BIGlzb2Zvcm0gZGl2ZXJzaXR5IGluIGh1bWFuIGNlbGxzOyBjaXRhdGlvbl9hdXRob3I9UGlja3JlbGwgIEouSy47IGNpdGF0aW9uX2F1dGhvcj1QYWkgIEEuQS47IGNpdGF0aW9uX2F1dGhvcj1HaWxhZCAgWS47IGNpdGF0aW9uX2F1dGhvcj1Qcml0Y2hhcmQgIEouSy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAxMjM2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9zdHRyYW5zY3JpcHRpb25hbCBnZW5lIHJlZ3VsYXRpb24gYnkgc3BhdGlhbCByZWFycmFuZ2VtZW50IG9mIHRoZSAzPyBVbnRyYW5zbGF0ZWQgcmVnaW9uOyBjaXRhdGlvbl9hdXRob3I9RWJlcmxlICBBLkIuOyBjaXRhdGlvbl9hdXRob3I9U3RhbGRlciAgTC47IGNpdGF0aW9uX2F1dGhvcj1NYXRoeXMgIEguOyBjaXRhdGlvbl9hdXRob3I9T3JvemNvICBSLlouOyBjaXRhdGlvbl9hdXRob3I9TcO8aGxlbWFubiAgTy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTkyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhlY3V0aW9uIG9mIG5vbnNlbnNlLW1lZGlhdGVkIG1STkEgZGVjYXk6IHdoYXQgZGVmaW5lcyBhIHN1YnN0cmF0ZT87IGNpdGF0aW9uX2F1dGhvcj1SZWJiYXByYWdhZGEgIEkuOyBjaXRhdGlvbl9hdXRob3I9THlra2UtQW5kZXJzZW4gIEouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBDZWxsIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0zOTQtNDAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SdWxlcyB0aGF0IGdvdmVybiBVUEYxIGJpbmRpbmcgdG8gbVJOQSAz4oCyIFVUUnM7IGNpdGF0aW9uX2F1dGhvcj1LdXJvc2FraSAgVC47IGNpdGF0aW9uX2F1dGhvcj1NYXF1YXQgIEwuRS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTMzNTctNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdsb2JhbCBhbmFseXNlcyBvZiBVUEYxIGJpbmRpbmcgYW5kIGZ1bmN0aW9uIHJldmVhbCBleHBhbmRlZCBzY29wZSBvZiBub25zZW5zZS1tZWRpYXRlZCBtUk5BIGRlY2F5OyBjaXRhdGlvbl9hdXRob3I9SHVydCAgSi5BLjsgY2l0YXRpb25fYXV0aG9yPVJvYmVydHNvbiAgQS5ELjsgY2l0YXRpb25fYXV0aG9yPUJ1cmdlICBDLkIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MTYzNi01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgZWxlbWVudHMgaW4gaHVtYW4gbG9uZyAz4oCyIFVUUnMgdGhhdCBpbmhpYml0IG5vbnNlbnNlLW1lZGlhdGVkIGRlY2F5OyBjaXRhdGlvbl9hdXRob3I9VG9tYSAgSy5HLjsgY2l0YXRpb25fYXV0aG9yPVJlYmJhcHJhZ2FkYSAgSS47IGNpdGF0aW9uX2F1dGhvcj1EdXJhbmQgIFMuOyBjaXRhdGlvbl9hdXRob3I9THlra2UtQW5kZXJzZW4gIEouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1STkE7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz04ODctOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lY2hhbmlzbSBhbmQgcmVndWxhdGlvbiBvZiB0aGUgbm9uc2Vuc2UtbWVkaWF0ZWQgZGVjYXkgcGF0aHdheTsgY2l0YXRpb25fYXV0aG9yPUh1ZyAgTi47IGNpdGF0aW9uX2F1dGhvcj1Mb25nbWFuICBELjsgY2l0YXRpb25fYXV0aG9yPUPDoWNlcmVzICBKLkYuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPTE0ODMtOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9uIHJlbGV2YW5jZSBvZiBjb2RvbiB1c2FnZSB0byBleHByZXNzaW9uIG9mIHN5bnRoZXRpYyBhbmQgbmF0dXJhbCBnZW5lcyBpbiBFc2NoZXJpY2hpYSBjb2xpOyBjaXRhdGlvbl9hdXRob3I9U3VwZWsgIEYuOyBjaXRhdGlvbl9hdXRob3I9xaBtdWMgIFQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xODU7ICBjaXRhdGlvbl9wYWdlcz0xMTI5LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SaWJvc29tZSBwcm9maWxpbmcgb2YgbW91c2UgZW1icnlvbmljIHN0ZW0gY2VsbHMgcmV2ZWFscyB0aGUgY29tcGxleGl0eSBhbmQgZHluYW1pY3Mgb2YgbWFtbWFsaWFuIHByb3Rlb21lczsgY2l0YXRpb25fYXV0aG9yPUluZ29saWEgIE4uVC47IGNpdGF0aW9uX2F1dGhvcj1MYXJlYXUgIEwuRi47IGNpdGF0aW9uX2F1dGhvcj1XZWlzc21hbiAgSi5TLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDc7ICBjaXRhdGlvbl9wYWdlcz03ODktODAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgYW50aS1TaGluZS1EYWxnYXJubyBzZXF1ZW5jZSBkcml2ZXMgdHJhbnNsYXRpb25hbCBwYXVzaW5nIGFuZCBjb2RvbiBjaG9pY2UgaW4gYmFjdGVyaWE7IGNpdGF0aW9uX2F1dGhvcj1MaSAgRy5XLjsgY2l0YXRpb25fYXV0aG9yPU9oICBFLjsgY2l0YXRpb25fYXV0aG9yPVdlaXNzbWFuICBKLlMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDg0OyAgY2l0YXRpb25fcGFnZXM9NTM4LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYWxhbmNlZCBjb2RvbiB1c2FnZSBvcHRpbWl6ZXMgZXVrYXJ5b3RpYyB0cmFuc2xhdGlvbmFsIGVmZmljaWVuY3k7IGNpdGF0aW9uX2F1dGhvcj1RaWFuICBXLjsgY2l0YXRpb25fYXV0aG9yPVlhbmcgIEouLVIuOyBjaXRhdGlvbl9hdXRob3I9UGVhcnNvbiAgTi5NLjsgY2l0YXRpb25fYXV0aG9yPU1hY2xlYW4gIEMuOyBjaXRhdGlvbl9hdXRob3I9WmhhbmcgIEouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMjYwMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldGVybWluYW50cyBvZiB0cmFuc2xhdGlvbiBlbG9uZ2F0aW9uIHNwZWVkIGFuZCByaWJvc29tYWwgcHJvZmlsaW5nIGJpYXNlcyBpbiBtb3VzZSBlbWJyeW9uaWMgc3RlbSBjZWxsczsgY2l0YXRpb25fYXV0aG9yPURhbmEgIEEuOyBjaXRhdGlvbl9hdXRob3I9VHVsbGVyICBULjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBDb21wdXQuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAyNzU1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9zaXRpdmVseSBjaGFyZ2VkIHJlc2lkdWVzIGFyZSB0aGUgbWFqb3IgZGV0ZXJtaW5hbnRzIG9mIHJpYm9zb21hbCB2ZWxvY2l0eTsgY2l0YXRpb25fYXV0aG9yPUNoYXJuZXNraSAgQy5BLjsgY2l0YXRpb25fYXV0aG9yPUh1cnN0ICBMLkQuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMTUwOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBhdCB0d28gbGV2ZWxzIG9mIGdlbmUgZXhwcmVzc2lvbiBpbiB5ZWFzdDsgY2l0YXRpb25fYXV0aG9yPUFydGllcmkgIEMuRy47IGNpdGF0aW9uX2F1dGhvcj1GcmFzZXIgIEguQi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz00MTEtMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lYXN1cmVtZW50IG9mIGF2ZXJhZ2UgZGVjb2RpbmcgcmF0ZXMgb2YgdGhlIDYxIHNlbnNlIGNvZG9ucyBpbiB2aXZvOyBjaXRhdGlvbl9hdXRob3I9R2FyZGluICBKLjsgY2l0YXRpb25fYXV0aG9yPVllYXNtaW4gIFIuOyBjaXRhdGlvbl9hdXRob3I9WXVyb3Zza3kgIEEuOyBjaXRhdGlvbl9hdXRob3I9Q2FpICBZLjsgY2l0YXRpb25fYXV0aG9yPVNraWVuYSAgUy47IGNpdGF0aW9uX2F1dGhvcj1GdXRjaGVyICBCLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9ZUxpZmU7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW1wcm92ZWQgcmlib3NvbWUtZm9vdHByaW50IGFuZCBtUk5BIG1lYXN1cmVtZW50cyBwcm92aWRlIGluc2lnaHRzIGludG8gZHluYW1pY3MgYW5kIHJlZ3VsYXRpb24gb2YgeWVhc3QgdHJhbnNsYXRpb247IGNpdGF0aW9uX2F1dGhvcj1XZWluYmVyZyAgRC5FLjsgY2l0YXRpb25fYXV0aG9yPVNoYWggIFAuOyBjaXRhdGlvbl9hdXRob3I9RWljaGhvcm4gIFMuVy47IGNpdGF0aW9uX2F1dGhvcj1IdXNzbWFubiAgSi5BLjsgY2l0YXRpb25fYXV0aG9yPVBsb3RraW4gIEouQi47IGNpdGF0aW9uX2F1dGhvcj1CYXJ0ZWwgIEQuUC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGwgUmVwOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTc4Ny05OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmF0aW9uYWxseSBkZXNpZ25lZCwgaGV0ZXJvbG9nb3VzIFMuIGNlcmV2aXNpYWUgdHJhbnNjcmlwdHMgZXhwb3NlIG5vdmVsIGV4cHJlc3Npb24gZGV0ZXJtaW5hbnRzOyBjaXRhdGlvbl9hdXRob3I9QmVuLVllaGV6a2VsICBULjsgY2l0YXRpb25fYXV0aG9yPUF0YXIgIFMuOyBjaXRhdGlvbl9hdXRob3I9WnVyICBIOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1STkEgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTk3Mi04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXVsdGlwbGUgcm9sZXMgb2YgdGhlIGNvZGluZyBzZXF1ZW5jZSA14oCyIGVuZCBpbiBnZW5lIGV4cHJlc3Npb24gcmVndWxhdGlvbjsgY2l0YXRpb25fYXV0aG9yPVR1bGxlciAgVC47IGNpdGF0aW9uX2F1dGhvcj1adXIgIEguOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTEzLTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXR3b3JrIG1vdGlmczogc2ltcGxlIGJ1aWxkaW5nIGJsb2NrcyBvZiBjb21wbGV4IG5ldHdvcmtzOyBjaXRhdGlvbl9hdXRob3I9TWlsbyAgUi47IGNpdGF0aW9uX2F1dGhvcj1TaGVuLU9yciAgUy47IGNpdGF0aW9uX2F1dGhvcj1JdHprb3ZpdHogIFMuOyBjaXRhdGlvbl9hdXRob3I9S2FzaHRhbiAgTi47IGNpdGF0aW9uX2F1dGhvcj1DaGtsb3Zza2lpICBELjsgY2l0YXRpb25fYXV0aG9yPUFsb24gIFUuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI5ODsgIGNpdGF0aW9uX3BhZ2VzPTgyNC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaHJvbW9zb21hbCBvcmdhbml6YXRpb24gaXMgc2hhcGVkIGJ5IHRoZSB0cmFuc2NyaXB0aW9uIHJlZ3VsYXRvcnkgbmV0d29yazsgY2l0YXRpb25fYXV0aG9yPUhlcnNoYmVyZyAgUi47IGNpdGF0aW9uX2F1dGhvcj1ZZWdlci1Mb3RlbSAgRS47IGNpdGF0aW9uX2F1dGhvcj1NYXJnYWxpdCAgSC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTEzOC00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGVsb21lcmUgbGVuZ3RoIGFzIGEgcXVhbnRpdGF0aXZlIHRyYWl0OiBnZW5vbWUtd2lkZSBzdXJ2ZXkgYW5kIGdlbmV0aWMgbWFwcGluZyBvZiB0ZWxvbWVyZSBsZW5ndGgtY29udHJvbCBnZW5lcyBpbiB5ZWFzdDsgY2l0YXRpb25fYXV0aG9yPUdhdGJvbnRvbiAgVC47IGNpdGF0aW9uX2F1dGhvcj1JbWJlc2kgIE0uOyBjaXRhdGlvbl9hdXRob3I9TmVsc29uICBNLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9ZTM1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSB1bml2ZXJzYWwgdHJlbmQgb2YgcmVkdWNlZCBtUk5BIHN0YWJpbGl0eSBuZWFyIHRoZSB0cmFuc2xhdGlvbi1pbml0aWF0aW9uIHNpdGUgaW4gcHJva2FyeW90ZXMgYW5kIGV1a2FyeW90ZXM7IGNpdGF0aW9uX2F1dGhvcj1HdSAgVy47IGNpdGF0aW9uX2F1dGhvcj1aaG91ICBULjsgY2l0YXRpb25fYXV0aG9yPVdpbGtlICBDLk8uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIENvbXB1dC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDA2NjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MZW5ndGggYmlhcyBjb3JyZWN0aW9uIGluIGdlbmUgb250b2xvZ3kgZW5yaWNobWVudCBhbmFseXNpcyB1c2luZyBsb2dpc3RpYyByZWdyZXNzaW9uOyBjaXRhdGlvbl9hdXRob3I9TWkgIEcuOyBjaXRhdGlvbl9hdXRob3I9RGkgIFkuOyBjaXRhdGlvbl9hdXRob3I9RW1lcnNvbiAgUy47IGNpdGF0aW9uX2F1dGhvcj1DdW1iaWUgIEouUy47IGNpdGF0aW9uX2F1dGhvcj1DaGFuZyAgSi5ILjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWU0NjEyOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc2Vzc2luZyB0aGUgZnJhY3Rpb24gb2Ygc2hvcnQtZGlzdGFuY2UgdGFuZGVtIHNwbGljZSBzaXRlcyB1bmRlciBwdXJpZnlpbmcgc2VsZWN0aW9uOyBjaXRhdGlvbl9hdXRob3I9SGlsbGVyICBNLjsgY2l0YXRpb25fYXV0aG9yPVN6YWZyYW5za2kgIEsuOyBjaXRhdGlvbl9hdXRob3I9U2luaGEgIFIuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1STkE7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz02MTYtMjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lYXN1cmluZyB0aGUgcHJldmFsZW5jZSBvZiByZWdpb25hbCBtdXRhdGlvbiByYXRlczogYW4gYW5hbHlzaXMgb2Ygc2lsZW50IHN1YnN0aXR1dGlvbnMgaW4gbWFtbWFscywgZnVuZ2ksIGFuZCBpbnNlY3RzOyBjaXRhdGlvbl9hdXRob3I9Rm94ICBBLjsgY2l0YXRpb25fYXV0aG9yPVR1Y2ggIEIuOyBjaXRhdGlvbl9hdXRob3I9Q2h1YW5nICBKLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEV2b2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTE4NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBlZmZlY3Qgb2YgdmFyaWF0aW9uIGluIHRoZSBlZmZlY3RpdmUgcG9wdWxhdGlvbiBzaXplIG9uIHRoZSByYXRlIG9mIGFkYXB0aXZlIG1vbGVjdWxhciBldm9sdXRpb24gaW4gZXVrYXJ5b3RlczsgY2l0YXRpb25fYXV0aG9yPUdvc3NtYW5uICBULkkuOyBjaXRhdGlvbl9hdXRob3I9S2VpZ2h0bGV5ICBQLkQuOyBjaXRhdGlvbl9hdXRob3I9RXlyZS1XYWxrZXIgIEEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9NjU4LTY3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeG9uIGFuZCBpbnRyb24gZGVmaW5pdGlvbiBpbiBwcmUtbVJOQSBzcGxpY2luZzsgY2l0YXRpb25fYXV0aG9yPURlIENvbnRpICBMLjsgY2l0YXRpb25fYXV0aG9yPUJhcmFsbGUgIE0uOyBjaXRhdGlvbl9hdXRob3I9QnVyYXR0aSAgRS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVdpbGV5IEludGVyZGlzY2lwLiBSZXY7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTQ5LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3B1bGF0aW9uIHNpemUgYW5kIHRoZSByYXRlIG9mIGV2b2x1dGlvbjsgY2l0YXRpb25fYXV0aG9yPUxhbmZlYXIgIFIuOyBjaXRhdGlvbl9hdXRob3I9S29ra28gIEguOyBjaXRhdGlvbl9hdXRob3I9RXlyZS1XYWxrZXIgIEEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgRWNvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTMzLTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TcGxpY2Utc2l0ZSBwYWlyaW5nIGlzIGFuIGludHJpbnNpY2FsbHkgaGlnaCBmaWRlbGl0eSBwcm9jZXNzOyBjaXRhdGlvbl9hdXRob3I9Rm94LVdhbHNoICBLLkwuOyBjaXRhdGlvbl9hdXRob3I9SGVydGVsICBLLkouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDY7ICBjaXRhdGlvbl9wYWdlcz0xNzY2LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBhYnVuZGFuY2Ugb2YgdWJpcXVpdG91c2x5IGV4cHJlc3NlZCBnZW5lcyByZXZlYWxlZCBieSB0aXNzdWUgdHJhbnNjcmlwdG9tZSBzZXF1ZW5jZSBkYXRhOyBjaXRhdGlvbl9hdXRob3I9UmFtc2vDtmxkICBELjsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIEUuVC47IGNpdGF0aW9uX2F1dGhvcj1CdXJnZSAgQy5CLjsgY2l0YXRpb25fYXV0aG9yPVNhbmRiZXJnICBSLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBDb21wdXQuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAwNTk4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIHF1YW50aWZpY2F0aW9uIG9mIG1hbW1hbGlhbiBnZW5lIGV4cHJlc3Npb24gY29udHJvbDsgY2l0YXRpb25fYXV0aG9yPVNjaHdhbmhhdXNzZXIgIEIuOyBjaXRhdGlvbl9hdXRob3I9QnVzc2UgIEQuOyBjaXRhdGlvbl9hdXRob3I9TGkgIE4uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDczOyAgY2l0YXRpb25fcGFnZXM9MzM3LTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnNpZ2h0cyBpbnRvIHRoZSByZWd1bGF0aW9uIG9mIHByb3RlaW4gYWJ1bmRhbmNlIGZyb20gcHJvdGVvbWljIGFuZCB0cmFuc2NyaXB0b21pYyBhbmFseXNlczsgY2l0YXRpb25fYXV0aG9yPVZvZ2VsICBDLjsgY2l0YXRpb25fYXV0aG9yPU1hcmNvdHRlICBFLk0uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0yMjctMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vbnNlbnNlLW1lZGlhdGVkIG1STkEgZGVjYXk6IHRlcm1pbmF0aW5nIGVycm9uZW91cyBnZW5lIGV4cHJlc3Npb247IGNpdGF0aW9uX2F1dGhvcj1CYWtlciAgSy5FLjsgY2l0YXRpb25fYXV0aG9yPVBhcmtlciAgUi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIE9waW4uIENlbGwgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTI5My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9hdXRob3I9TWF5ciAgRS47IGNpdGF0aW9uX3B1Ymxpc2hlcj1CYXNpYyBCb29rcywgUGVyc2V1cyBCb29rcyBHcm91cCwgTmV3IFlvcms7ICBjaXRhdGlvbl90aXRsZT1XaGF0IEV2b2x1dGlvbiBJczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIGFuZCB0aW5rZXJpbmc7IGNpdGF0aW9uX2F1dGhvcj1KYWNvYiAgRi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5Nzc7ICBjaXRhdGlvbl92b2x1bWU9MTk2OyAgY2l0YXRpb25fcGFnZXM9MTE2MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX2F1dGhvcj1QaWdsaXVjY2kgIE0uOyBjaXRhdGlvbl9hdXRob3I9S2FwbGFuICBKLjsgY2l0YXRpb25fcHVibGlzaGVyPVVuaXZlcnNpdHkgb2YgQ2hpY2FnbyBQcmVzcywgQ2hpY2FnbzsgIGNpdGF0aW9uX3RpdGxlPU1ha2luZyBTZW5zZSBvZiBFdm9sdXRpb246IFRoZSBDb25jZXB0dWFsIEZvdW5kYXRpb25zIG9mIEV2b2x1dGlvbmFyeSBUaGVvcnk7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhlYXJpbmcgc2lsZW5jZTogbm9uLW5ldXRyYWwgZXZvbHV0aW9uIGF0IHN5bm9ueW1vdXMgc2l0ZXMgaW4gbWFtbWFsczsgY2l0YXRpb25fYXV0aG9yPUNoYW1hcnkgIEouVi47IGNpdGF0aW9uX2F1dGhvcj1QYXJtbGV5ICBKLkwuOyBjaXRhdGlvbl9hdXRob3I9SHVyc3QgIEwuRC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9OTgtMTA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbmRlcnN0YW5kaW5nIHRoZSBjb250cmlidXRpb24gb2Ygc3lub255bW91cyBtdXRhdGlvbnMgdG8gaHVtYW4gZGlzZWFzZTsgY2l0YXRpb25fYXV0aG9yPVNhdW5hICBaLkUuOyBjaXRhdGlvbl9hdXRob3I9S2ltY2hpLVNhcmZhdHkgIEMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz02ODMtOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5ldyB1bml2ZXJzYWwgcnVsZXMgb2YgZXVrYXJ5b3RpYyB0cmFuc2xhdGlvbiBpbml0aWF0aW9uIGZpZGVsaXR5OyBjaXRhdGlvbl9hdXRob3I9WnVyICBILjsgY2l0YXRpb25fYXV0aG9yPVR1bGxlciAgVC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgQ29tcHV0LiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMzEzNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMy80LzM3Ny8xNzQ2MjkzMy9kc3cwMTkucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gICBJdCBpcyBnZW5lcmFsbHkgYmVsaWV2ZWQgdGhhdCBpbnRyb25zIGFyZSBub3QgdHJhbnNsYXRlZDsgdGhlcmVmb3JlLCB0aGUgcG90ZW50aWFsIGludHJvbmljIGZlYXR1cmVzIHRoYXQgbWF5IGJlIHJlbGF0ZWQgdG8gdGhlIHRyYW5zbGF0aW9uIHN0ZXAgKG9jIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0yNDY5OTYyJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZCXDNMpHMmerpqCRaoabmhuIvyNp" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f377%2f2469962" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f377%2f2469962" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=84049" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/4/0/m_cover.gif?Expires=2147483647&amp;Signature=XWSbFJdI5lGDaRny1g7BRzfZg~ebzc~OT20sUUxiNUi3-W~7uvtu5IDIgO3L5~QjmCQzUopLMGz-3gEHtIKUOFbRCHM2eH4h7l2Wu3nqAQC9YYHygJi-TLC9x3FpQRq3lTBcdWmpj185LJbB-1H-PKlUoEU9gxYyCNvjJUg~GUaHNRtVxbV0a7bDIfdyclZJ~w1BhGonEiYqh-IZSS0VekFWZ7mnLia~WbOEtL8AzSLsoIVak5-aADczuqIYGRkdfxcEh8zTdO-NlGhKxLoi5zlJy0Y~XuIxHZNxQxDFoVA3s~0J2fujGZsWX~ElpEU3I2Vma3M6MgFM80P2xPWpTw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/4">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87197572">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197572">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197576">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197576">Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197591">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197591">Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197647">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197647">Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197724">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197724">Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197735">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197735">Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197741">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197741">Conflict of interest</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197744">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197744">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197747">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197747">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/4/365/2469965">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/4/395/2469961">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Selection for reduced translation costs at the intronic 5′ end in fungi
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zohar Zafrir</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> Department of Biomedical Engineering, Tel Aviv University, Tel Aviv, Israel </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zohar+Zafrir">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zafrir Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zafrir Z&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Zohar Zafrir</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hadas Zur</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> Department of Biomedical Engineering, Tel Aviv University, Tel Aviv, Israel </div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hadas+Zur">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zur H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zur H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hadas Zur</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tamir Tuller</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div> Department of Biomedical Engineering, Tel Aviv University, Tel Aviv, Israel </div><div class="aff"><div class="label -label"><sup>2</sup></div> The Sagol School of Neuroscience, Tel Aviv University, Tel Aviv 69978, Israel </div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw019-cor1" data-open="dsw019-cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
             *To whom correspondence should be addressed. Tel. +972 3-640-5836. Fax. +972 3-640-8123. E-mail: <a href="mailto:tamirtul@post.tau.ac.il" target="_blank">tamirtul@post.tau.ac.il</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tamir+Tuller">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tuller T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tuller T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tamir Tuller</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw019-FN1"><div class="fn"><div class="footnote-content">Edited by Dr Mikita Suyama</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 4, 1 August 2016, Pages 377–394, <a href='https://doi.org/10.1093/dnares/dsw019'>https://doi.org/10.1093/dnares/dsw019</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">02 June 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">15 December 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">26 April 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2469962" href="/dnaresearch/article-pdf/23/4/377/17462933/dsw019.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Zohar Zafrir, Hadas Zur, Tamir Tuller;  Selection for reduced translation costs at the intronic 5′ end in fungi, <em>DNA Research</em>, Volume 23, Issue 4, 1 August 2016, Pages 377–394, <a href='https://doi.org/10.1093/dnares/dsw019'>https://doi.org/10.1093/dnares/dsw019</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2469962?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469962?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469962?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469962?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469962?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469962?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2469962?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Selection%20for%20reduced%20translation%20costs%20at%20the%20intronic%205%E2%80%B2%20end%20in%20fungi&amp;publicationDate=2016-06-02&amp;volumeNum=23&amp;issueNum=4&amp;author=Zafrir%2C%20Zohar%3B%20Zur%2C%20Hadas&amp;startPage=377&amp;endPage=394&amp;contentId=10.1093%2Fdnares%2Fdsw019&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f377%2f2469962" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87197572 id="87197572" class="abstract-title" >Abstract</h2>
<section class="abstract"><p> It is generally believed that introns are not translated; therefore, the potential intronic features that may be related to the translation step (occurring after splicing) have yet to be thoroughly studied. Here, focusing on four fungi, we performed for the first time a comprehensive study aimed at characterizing how translation efficiency is encoded in introns and affects their evolution. By analysing their intronome we provide evidence of selection for STOP codons close to the intronic 5′ end, and show that the beginning of introns are selected for significantly high translation, presumably to reduce translation and metabolic costs in cases of non-spliced introns. Ribosomal profiling data analysis in <em>Saccharomyces cerevisiae</em> supports the conjecture that in this organism intron retention frequently occurs, introns are partially translated, and their translation efficiency affects organismal fitness. We show that the reported results are more significant in highly translated and highly spliced genes, but are not associated only with genes with a specific function. We also discuss the potential relation of the reported signals to efficient nonsense-mediated decay due to splicing errors. These new discoveries are supported by population-genetics considerations. In addition, they are contributory steps towards a broader understanding of intron evolution and the effect of silent mutations on gene expression and organismal fitness. </p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;mRNA translation&quot;">mRNA translation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;intron evolution&quot;">intron evolution</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcript evolution&quot;">transcript evolution</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="splicing">splicing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;silent mutations&quot;">silent mutations</a></div>                            <h2 scrollto-destination=87197576 id="87197576" class="section-title" >Introduction</h2>
<p> RNA splicing is the process in which pre-mRNA transcripts mature and where introns, intervening fragments within transcripts, are recognized and removed, leaving retained regions termed exons that compose the mature mRNA. In eukaryotes spliceosomal introns are confined to the nucleus, and the splicing process is executed by the spliceosome, one of the largest molecular complexes in the cell. <a reveal-id="dsw019-B1" data-open="dsw019-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsw019-B2" data-open="dsw019-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> Accurate processing of introns is a crucial regulatory step in determining the cell expression profile, and is required before protein translation can be initiated. As such, splicing efficiency (SE; efficient recognition and proper splicing by the spliceosome) and specificity are used to regulate growth, development, and overall response to external signals. <a reveal-id="dsw019-B3" data-open="dsw019-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsw019-B4" data-open="dsw019-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Extensive studies over the past two decades have revealed the core chemical reactions, several sequence determinants, and the major protein component interactions during intron splicing. <a reveal-id="dsw019-B5 dsw019-B6 dsw019-B7 dsw019-B8" data-open="dsw019-B5 dsw019-B6 dsw019-B7 dsw019-B8" class="link link-ref link-reveal xref-bibr"><sup>5–8</sup></a> Nevertheless, the debate on intron origin and evolution has been ongoing intensively for decades. <a reveal-id="dsw019-B9" data-open="dsw019-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsw019-B10" data-open="dsw019-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a></p><p> The dynamic nature of spliceosome assembly and the complex interactions of both its proteins and RNA components with the pre-mRNA, give rise to a range of intronic SEs among different genes and within the same gene that may lead to altered SE, and facilitates the creation of different mature mRNA products from identical pre-mRNA transcripts; <a reveal-id="dsw019-B11" data-open="dsw019-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> in this process particular exons of a gene may be included within or excluded from (i.e. ‘skipped’) the final processed mRNA transcript. This phenomenon is termed alternative splicing (AS) and is the chief constituent of the increased number of proteins encoded by genomes of multicellular organisms. <a reveal-id="dsw019-B12" data-open="dsw019-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> Intron retention, in which an intron remains in the mature mRNA transcript, is widespread among plants and protozoa, but is infrequent in vertebrates and seldom found in fungi. <a reveal-id="dsw019-B13 dsw019-B14 dsw019-B15 dsw019-B16 dsw019-B17" data-open="dsw019-B13 dsw019-B14 dsw019-B15 dsw019-B16 dsw019-B17" class="link link-ref link-reveal xref-bibr"><sup>13–17</sup></a> As such, it is often considered the earliest version of AS to have evolved and might reflect mis-splicing, as the splicing machinery may fail to recognize weak splice sites flanking short introns. Additional AS events include alternative splice sites selection, whilst the combination of two or more events can generate more complex mature mRNA products. <a reveal-id="dsw019-B18 dsw019-B19 dsw019-B20 dsw019-B21 dsw019-B22 dsw019-B23" data-open="dsw019-B18 dsw019-B19 dsw019-B20 dsw019-B21 dsw019-B22 dsw019-B23" class="link link-ref link-reveal xref-bibr"><sup>18–23</sup></a></p><p> It was suggested that small introns in diverse organisms (such as <em>Paramecium tetraurelia</em> , <em>Homo sapiens</em> , <em>Caenorhabditis elegans</em> , and <em>Schizosaccharomyces pombe</em> ) are under selective pressure to cause premature translation termination in case an intron has not been spliced (i.e. ‘retained’). In particular, it has been proposed that the nonsense-mediated mRNA decay (NMD) pathway may have a proofreading role in gene expression, eliminating transcripts that have not been properly spliced; <a reveal-id="dsw019-B24" data-open="dsw019-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> moreover, the core components of the NMD machinery are evolutionarily conserved in all eukaryotic organisms tested so far, and their deletion or silencing prevents NMD in eukaryotic cells. <a reveal-id="dsw019-B25 dsw019-B26 dsw019-B27 dsw019-B28 dsw019-B29" data-open="dsw019-B25 dsw019-B26 dsw019-B27 dsw019-B28 dsw019-B29" class="link link-ref link-reveal xref-bibr"><sup>25–29</sup></a></p><p> The amount of intron-containing genes, intron density per gene or per kilobase pair, and intron length, varies from one eukaryote to another. <a reveal-id="dsw019-B30 dsw019-B31 dsw019-B32 dsw019-B33" data-open="dsw019-B30 dsw019-B31 dsw019-B32 dsw019-B33" class="link link-ref link-reveal xref-bibr"><sup>30–33</sup></a> In <em>Saccharomyces cerevisiae</em> there are &lt;300 intron-containing genes (5% of ∼6,000), of which few are mediated by two introns or more. <a reveal-id="dsw019-B34" data-open="dsw019-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a><sup>,</sup><a reveal-id="dsw019-B35" data-open="dsw019-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> In fact, until now only two AS or intron retention events have been identified in this organism. <a reveal-id="dsw019-B15" data-open="dsw019-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsw019-B36" data-open="dsw019-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Nevertheless, in <em>S. cerevisiae</em> intron-containing genes are particularly highly expressed and account for more than 70% of its proteome. Additionally, several intron sequences are found duplicated within ribosomal protein paralogs. <a reveal-id="dsw019-B37" data-open="dsw019-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a><sup>,</sup><a reveal-id="dsw019-B38" data-open="dsw019-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> Conversely, in <em>S. pombe</em> (an organisms with ∼5,000 genes) there are ∼5,000 introns spreading over a third of its genome, with up to 20 introns mediating a single gene <a reveal-id="dsw019-B39" data-open="dsw019-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> and with few known genes displaying AS. <a reveal-id="dsw019-B17" data-open="dsw019-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsw019-B40" data-open="dsw019-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a><sup>,</sup><a reveal-id="dsw019-B41" data-open="dsw019-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> Other fungi such as <em>Aspergillus nidulans</em> and <em>Candida albicans</em> show variations in intronic characteristics as well. <a reveal-id="dsw019-B42" data-open="dsw019-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a><sup>,</sup><a reveal-id="dsw019-B43" data-open="dsw019-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> Nonetheless, even though eukaryotic evolution has been generally characterized by widespread intron gain and loss events, <a reveal-id="dsw019-B10" data-open="dsw019-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsw019-B44" data-open="dsw019-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a><sup>,</sup><a reveal-id="dsw019-B45" data-open="dsw019-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> the ubiquity of introns and the core of the spliceosome is conserved in all well-characterized eukaryotes. <a reveal-id="dsw019-B1" data-open="dsw019-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsw019-B46" data-open="dsw019-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> Moreover, <em>Hemiselmis andersenii</em> is currently the only known eukaryotic organism without any introns or spliceosome subunit genes. <a reveal-id="dsw019-B47" data-open="dsw019-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a></p><p> The intron boundary signals at the donor splice site (or 5'SS) and acceptor splice site (or 3'SS), the branch site (BS), and the polypyrimidine tract, are canonical sequence elements which are essential for intron recognition and for splicing to occur. The factors that bind to these sequence motifs and the biochemical reactions they perform are relatively well known due to the extensive research in this field. Systematic investigations show that this process is highly regulated: from spliceosome assembly, through pre-mRNA recognition and binding, to the splicing reaction and complex disassembly. <a reveal-id="dsw019-B23" data-open="dsw019-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a><sup>,</sup><a reveal-id="dsw019-B48" data-open="dsw019-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> Additionally, it has been suggested that in yeast, introns regulate ribosome biogenesis and function, and affect cell fitness under stress. <a reveal-id="dsw019-B49" data-open="dsw019-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a></p><p>Despite such comprehensive studies, very little is known regarding the intronic features that are related to translation. Specifically, introns are believed to be the non-translated parts of the transcript; however, since splicing is not a perfect mechanism, it is possible that high enough splicing errors, such as intronic retention, will trigger selective pressure for translation efficiency (TE) adaptation at the 5′ end of introns. Here, we conduct a novel large scale study of four fungal intronomes, to characterize novel genomic level intronic features in the vicinity of the intronic splice-sites, related to post-splicing/translation regulation, and understand their evolution.</p>                            <h2 scrollto-destination=87197591 id="87197591" class="section-title" >Materials and Methods</h2>
                            <h3 scrollto-destination=87197593 id="87197593" class="section-title" >The analysed organisms</h3>
<p> The four fungi analysed here ( <em>S. cerevisiae, S. pombe</em><em>, A. nidulans</em> , and <em>C. albicans</em> ) were chosen based on the following considerations: <em>S. cerevisiae</em> and <em>S. pombe</em> are well studied organisms with well-established databases that are known to have diverged 350–900 million years ago. <a reveal-id="dsw019-B50" data-open="dsw019-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a><sup>,</sup><em>A. nidulans</em> and <em>C. albicans</em> are two additional fungi known to have diverged from <em>S</em><em>. pombe</em> about 650 million years ago <a reveal-id="dsw019-B51" data-open="dsw019-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> and from <em>S. cerevisiae</em> about 235 million years ago. <a reveal-id="dsw019-B52" data-open="dsw019-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> The genomes of these organisms are also fully sequenced and their introns are well annotated. In addition, <em>C. albicans</em> is a dimorphic fungus which can be a significant pathogen in humans. </p>                            <h3 scrollto-destination=87197595 id="87197595" class="section-title" >Intronic sequence information</h3>
<p><em>S. cerevisiae</em> open reading frames (ORFs) and intron-containing gene sequences (strain 288c) were taken from the <em>Saccharomyces</em> Genome Database (SGD); <a reveal-id="dsw019-B53" data-open="dsw019-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> BS location information was obtained from the Ares lab database. <a reveal-id="dsw019-B35" data-open="dsw019-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a><sup>,</sup><em>S</em><em>. pombe</em> genome information (Assembly 16) was taken from the PomBase database; <a reveal-id="dsw019-B54" data-open="dsw019-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> BS locations were calculated based on the position-specific scoring matrix information extracted from the Sanger Institute, and is based on the original full genome sequencing. <a reveal-id="dsw019-B39" data-open="dsw019-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a><sup>,</sup><em>A. nidulans</em> (FGSC A4) and <em>C. albicans</em> (SC5314 Assembly 21) genome information was taken from the <em>Aspergillus</em> Genome Database (AspGD) <a reveal-id="dsw019-B55" data-open="dsw019-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> and <em>Candida</em> Genome Database (CGD), <a reveal-id="dsw019-B56" data-open="dsw019-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> respectively; BS locations were calculated based on the fungal BS consensus sequence ( <strong><em>CURAY</em></strong> ). We used only introns taken from coding sequences (CDSs) and excluded 5′ untranslated region (UTR) and 3′ UTR introns. Introns associated with putative or alternatively spliced genes were also excluded; the full intron exclusion list can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Table S4</a></span> . Additional GC content and general information can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Table S5</a></span> ; exon–intron GC content was calculated using intron sequences and their flanking exon sequences (100 bp upstream and downstream). </p>                            <h3 scrollto-destination=87197597 id="87197597" class="section-title" >Protein abundance, mRNA levels, and metabolic cost</h3>
<p> Protein abundance (PA) information for <em>S. cerevisiae</em> and <em>S</em><em>. pombe</em> was taken from the PaxDb dataset, <a reveal-id="dsw019-B57" data-open="dsw019-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> which integrates information from various resources. The mRNA levels for <em>S. cerevisiae</em> are based on RNA-seq and DNA chip and were obtained by integrating three datasets <a reveal-id="dsw019-B58 dsw019-B59 dsw019-B60" data-open="dsw019-B58 dsw019-B59 dsw019-B60" class="link link-ref link-reveal xref-bibr"><sup>58–60</sup></a> in the following manner: first, we normalized each dataset by its average mRNA levels; next, for each gene we averaged all its normalized measurements (dataset with no value was ignored). Protein per mRNA ratio (PPR), i.e. the number of proteins produced on average from an mRNA molecule, was calculated by dividing PA by mRNA levels. PA levels are measured in parts per millions. The mRNA is based on RNA-seq and DNA chip and thus is proportional to the mRNA levels (number of molecules in the cell). Since our analysis is based on spearman correlation, ranking of genes according to their expression levels is enough to provide the required results <em>(</em> i.e. the actual levels will not change the results). Data for metabolic cost of amino acids (AAs) for <em>S. cerevisiae</em> were taken from Ref. <a reveal-id="dsw019-B61" data-open="dsw019-B61" class="link link-ref link-reveal xref-bibr">61</a> and were also used as an approximation for <em>S</em><em>. pombe</em> , as no <em>S</em><em>. pombe</em> data were available. </p>                            <h3 scrollto-destination=87197600 id="87197600" class="section-title" >Randomization models</h3>
<p> The randomized models were designed to conserve encoded protein information, in addition to the intronic properties. Specifically, we maintained codon frequencies [i.e. codon-usage bias (CUB)], canonical splicing signals, and GC content. To this end, the distribution of codons in the CDS was calculated for each gene, and the sequence was then randomized while maintaining these codon frequencies (i.e. for each AA in the gene, the probability of a certain codon was determined by its individual distribution in the CDS) and the amino acid content and order (i.e. the encoded protein); this was separately done for upstream and downstream exons. For each gene, intronic nucleotides were uniformly permutated, maintaining consensus sequences (5′SS, BS, and 3′SS) and the intronic GC content. We used the following randomization schemes to generate random sets: (i) Codon only and (ii) Intron only. Next, a combination of the basic schemes was applied, i.e. (i) + (ii), and is used throughout the study; the randomization models are also illustrated in <a reveal-id="dsw019-F1" data-open="dsw019-F1" class="link link-reveal link-table xref-fig">Fig. 1C and D</a> . In the models used in <a reveal-id="dsw019-F3 dsw019-F5" data-open="dsw019-F3 dsw019-F5" class="link link-reveal link-table xref-fig">Figs 3 and 5</a> , we randomized the intronic sequences while assuming introns as being translated. Specifically, we ‘continued’ the introns in the same frame of their upstream exons and considered the triplets of nucleotides in the introns as ‘artificial’ codons; next, we scrambled them while controlling for their CUB and GC content. </p>                        <a id="87197602" scrollto-destination="87197602"></a>
<div id="dsw019-F1" data-id="dsw019-F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/m_dsw019f1p.png?Expires=2147483647&amp;Signature=HGyCVqE1w2eN4Wt8iTH5yCBVY1GFGPwa3DdbNZX5LnCEXV4B2r3DeSz12x1j4MIBoesIWy1xqHrXkybzuG5oBUKB1AnM1~ekjD9tIvrCQQbYAq5YcYavdNqPXRLXbNIqPoiuoFtf2bFMnCzeh7bK~b4a-W1dlCo4H2rM6WLWBpQpnP0e-y-qQsJnuiz1WL4OnKiDZbdc6P1nIJnEbOZN29kzM~V4631WGTwvE-S4MhPAM34GhXH3Ucj7dfv~JCRrYIVV0mH~VugYXIhoHx-oEvbqdBQSfDDQ0l6wcGXrYMT-7v3rO-nyrLXpcSOX6wrrstON2yIJ8meDaISOgBEyEw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Pre-mRNA exonic and intronic regions, basic definitions, and randomization models. (A) The analysed fungal genes can be divided into three major regions: UTRs, exons, and introns (in the analysed organisms we did not consider UTR introns since they are not flanked by translated regions). The introns include three canonical consensus sequences: the donor (or 5′SS; subsequence GTATGT ) and acceptor (or 3′SS; subsequence YAG ) that define intronic boundaries, and the branch site (BS; subsequence CTRAY ) that is required for the lariat formation. (B) In our analyses, exons and introns were divided into three domains: Exonic Donor (up to 90 nt upstream from the 5′SS), pre-STOP (up to 90 nt upstream from the first intronic STOP codon), and post-STOP (up to 90 nt downstream from the first intronic STOP codon); the features surrounding the boundaries of these regions are studied here. (C and D) In order to demonstrate that the reported features are under selective pressure, we compared the potentially translated sequences to the ones obtained by the following randomized models: (C) encoded protein information is maintained; synonymous codons frequencies of each gene (separately) are maintained; (D) uniform permutation of intronic nucleotides (per intron); the randomization models preserve these consensus sequences (5′SS/BS/3′SS), as well as additional exonic and intronic characteristics (see Materials and methods and Supplementary information ). The results suggest adaptation to the translation process at the beginning of introns via preference for STOP codons close to the intronic 5′ end, and for CUB resembling the one appearing in annotated ORFs. This discovery is also supported by ribosomal profiling (RP) footprint measurements which are significantly higher in the 5′ end intronic region than in the intronic region downstream of the STOP codon. " path-from-xml="dsw019f1p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197602/dsw019f1p.png" path-from-xml="dsw019f1p.png" target="_blank">View large</a><a section="87197602" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019f1p.png?Expires=2147483647&amp;Signature=bsgZXf0kNnutpFC6g8MFxVOf-LhINm~Up~PBzThgZgn0IPhah564vU55114oR3yuN-sCJpf9yHKH28wDis~P-7GqjoGb1OVMUcgoofYIsdzc8VAif1f8SiadX6GLxUROF5fld1Ylfv1LGqSLEgyLvYuFIIskNXjxFhORYXrC-4mtvJutEDExCkeayvg8RgxuD6J-rHOIRCKY1BCtT4PkjIC9hB32BB3YYXA71~efGeMAyoc5pPk0neQjmdOhseq50NPz9Sz5vvMJLi8-TRL3p~LUpSUA5RbYfdnZzYgsCRqrg9YXgYec0TuKP467HUQbX0vcZ2sgOKzrIqViw45BoA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197602&amp;ar=2469962&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw019f1p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Pre-mRNA exonic and intronic regions, basic definitions, and randomization models. (A) The analysed fungal genes can be divided into three major regions: UTRs, exons, and introns (in the analysed organisms we did not consider UTR introns since they are not flanked by translated regions). The introns include three canonical consensus sequences: the donor (or 5′SS; subsequence <em>GTATGT</em> ) and acceptor (or 3′SS; subsequence <em>YAG</em> ) that define intronic boundaries, and the branch site (BS; subsequence <em>CTRAY</em> ) that is required for the lariat formation. (B) In our analyses, exons and introns were divided into three domains: Exonic Donor (up to 90 nt upstream from the 5′SS), pre-STOP (up to 90 nt upstream from the first intronic STOP codon), and post-STOP (up to 90 nt downstream from the first intronic STOP codon); the features surrounding the boundaries of these regions are studied here. (C and D) In order to demonstrate that the reported features are under selective pressure, we compared the potentially translated sequences to the ones obtained by the following randomized models: (C) encoded protein information is maintained; synonymous codons frequencies of each gene (separately) are maintained; (D) uniform permutation of intronic nucleotides (per intron); the randomization models preserve these consensus sequences (5′SS/BS/3′SS), as well as additional exonic and intronic characteristics (see Materials and methods and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&amp;Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary information</a></span> ). The results suggest adaptation to the translation process at the beginning of introns via preference for STOP codons close to the intronic 5′ end, and for CUB resembling the one appearing in annotated ORFs. This discovery is also supported by ribosomal profiling (RP) footprint measurements which are significantly higher in the 5′ end intronic region than in the intronic region downstream of the STOP codon. </p></div></div><div content-id="dsw019-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/m_dsw019f1p.png?Expires=2147483647&amp;Signature=HGyCVqE1w2eN4Wt8iTH5yCBVY1GFGPwa3DdbNZX5LnCEXV4B2r3DeSz12x1j4MIBoesIWy1xqHrXkybzuG5oBUKB1AnM1~ekjD9tIvrCQQbYAq5YcYavdNqPXRLXbNIqPoiuoFtf2bFMnCzeh7bK~b4a-W1dlCo4H2rM6WLWBpQpnP0e-y-qQsJnuiz1WL4OnKiDZbdc6P1nIJnEbOZN29kzM~V4631WGTwvE-S4MhPAM34GhXH3Ucj7dfv~JCRrYIVV0mH~VugYXIhoHx-oEvbqdBQSfDDQ0l6wcGXrYMT-7v3rO-nyrLXpcSOX6wrrstON2yIJ8meDaISOgBEyEw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" Pre-mRNA exonic and intronic regions, basic definitions, and randomization models. (A) The analysed fungal genes can be divided into three major regions: UTRs, exons, and introns (in the analysed organisms we did not consider UTR introns since they are not flanked by translated regions). The introns include three canonical consensus sequences: the donor (or 5′SS; subsequence GTATGT ) and acceptor (or 3′SS; subsequence YAG ) that define intronic boundaries, and the branch site (BS; subsequence CTRAY ) that is required for the lariat formation. (B) In our analyses, exons and introns were divided into three domains: Exonic Donor (up to 90 nt upstream from the 5′SS), pre-STOP (up to 90 nt upstream from the first intronic STOP codon), and post-STOP (up to 90 nt downstream from the first intronic STOP codon); the features surrounding the boundaries of these regions are studied here. (C and D) In order to demonstrate that the reported features are under selective pressure, we compared the potentially translated sequences to the ones obtained by the following randomized models: (C) encoded protein information is maintained; synonymous codons frequencies of each gene (separately) are maintained; (D) uniform permutation of intronic nucleotides (per intron); the randomization models preserve these consensus sequences (5′SS/BS/3′SS), as well as additional exonic and intronic characteristics (see Materials and methods and Supplementary information ). The results suggest adaptation to the translation process at the beginning of introns via preference for STOP codons close to the intronic 5′ end, and for CUB resembling the one appearing in annotated ORFs. This discovery is also supported by ribosomal profiling (RP) footprint measurements which are significantly higher in the 5′ end intronic region than in the intronic region downstream of the STOP codon. " path-from-xml="dsw019f1p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197602/dsw019f1p.png" path-from-xml="dsw019f1p.png" target="_blank">View large</a><a section="87197602" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019f1p.png?Expires=2147483647&amp;Signature=bsgZXf0kNnutpFC6g8MFxVOf-LhINm~Up~PBzThgZgn0IPhah564vU55114oR3yuN-sCJpf9yHKH28wDis~P-7GqjoGb1OVMUcgoofYIsdzc8VAif1f8SiadX6GLxUROF5fld1Ylfv1LGqSLEgyLvYuFIIskNXjxFhORYXrC-4mtvJutEDExCkeayvg8RgxuD6J-rHOIRCKY1BCtT4PkjIC9hB32BB3YYXA71~efGeMAyoc5pPk0neQjmdOhseq50NPz9Sz5vvMJLi8-TRL3p~LUpSUA5RbYfdnZzYgsCRqrg9YXgYec0TuKP467HUQbX0vcZ2sgOKzrIqViw45BoA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197602&amp;ar=2469962&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw019f1p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> Pre-mRNA exonic and intronic regions, basic definitions, and randomization models. (A) The analysed fungal genes can be divided into three major regions: UTRs, exons, and introns (in the analysed organisms we did not consider UTR introns since they are not flanked by translated regions). The introns include three canonical consensus sequences: the donor (or 5′SS; subsequence <em>GTATGT</em> ) and acceptor (or 3′SS; subsequence <em>YAG</em> ) that define intronic boundaries, and the branch site (BS; subsequence <em>CTRAY</em> ) that is required for the lariat formation. (B) In our analyses, exons and introns were divided into three domains: Exonic Donor (up to 90 nt upstream from the 5′SS), pre-STOP (up to 90 nt upstream from the first intronic STOP codon), and post-STOP (up to 90 nt downstream from the first intronic STOP codon); the features surrounding the boundaries of these regions are studied here. (C and D) In order to demonstrate that the reported features are under selective pressure, we compared the potentially translated sequences to the ones obtained by the following randomized models: (C) encoded protein information is maintained; synonymous codons frequencies of each gene (separately) are maintained; (D) uniform permutation of intronic nucleotides (per intron); the randomization models preserve these consensus sequences (5′SS/BS/3′SS), as well as additional exonic and intronic characteristics (see Materials and methods and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&amp;Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary information</a></span> ). The results suggest adaptation to the translation process at the beginning of introns via preference for STOP codons close to the intronic 5′ end, and for CUB resembling the one appearing in annotated ORFs. This discovery is also supported by ribosomal profiling (RP) footprint measurements which are significantly higher in the 5′ end intronic region than in the intronic region downstream of the STOP codon. </p></div></div>                            <h3 scrollto-destination=87197603 id="87197603" class="section-title" >Evaluation of significance levels</h3>
<p> For each genome, up to 1,000 random intronomes were generated (1,000 for <em>S. cerevisiae</em> and <em>C. albicans</em> , 200 for <em>S</em><em>. pombe</em> and <em>A. nidulans</em> ). The level of significance (i.e. empirical <em>P</em> -value) was determined based on a comparison between the random intronomes and the actual one as follows, when we expect the real to be significantly lower than the randomized version (e.g. in the case of the distance to STOP codon): Let <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>S</mi></mrow><mrow><mn>0</mn></mrow></msub></math></span> be the value of the actual intronome and <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">S</mi><mo xmlns="">=</mo><msub xmlns=""><mrow><mi>S</mi></mrow><mrow><mn>1</mn></mrow></msub><mo xmlns="">,</mo><mi xmlns=""> </mi><msub xmlns=""><mrow><mi>S</mi></mrow><mrow><mn>2</mn></mrow></msub><mo xmlns="">,</mo><mo xmlns="">⋯</mo><mo xmlns="">,</mo><msub xmlns=""><mrow><mi>S</mi></mrow><mrow><mi>N</mi></mrow></msub></math></span> a vector containing <em>N</em> random values sampled from the randomized models, where <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>S</mi></mrow><mrow><mi>i</mi></mrow></msub><mi xmlns=""> </mi></math></span> is the value of the <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>i</mi></mrow><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">h</mi></mrow></msub></math></span> sample; then <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">p</mi><mo xmlns="">=</mo><mfrac xmlns=""><mrow><mn>1</mn></mrow><mrow><mi>N</mi></mrow></mfrac><mrow xmlns=""><msubsup><mo stretchy="true">∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>N</mi></mrow></msubsup><mrow><mo stretchy="false">|</mo><mfenced close="}" open="{" separators="|"><mrow><msub><mrow><mi mathvariant="normal">S</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>&gt;</mo><msub><mrow><mi mathvariant="normal">S</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow></mfenced><mo stretchy="false">|</mo></mrow></mrow></math></span> , <em>N </em> = 1,000. When we expect the real to be significantly higher than the randomized version (e.g. the typical decoding rate (TDR)/tAI) the <em>P</em> -valued was computed as follows: <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">p</mi><mo xmlns="">=</mo><mfrac xmlns=""><mrow><mn>1</mn></mrow><mrow><mi>N</mi></mrow></mfrac><mrow xmlns=""><msubsup><mo stretchy="true">∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>N</mi></mrow></msubsup><mrow><mo stretchy="false">|</mo><mfenced close="}" open="{" separators="|"><mrow><msub><mrow><mi mathvariant="normal">S</mi></mrow><mrow><mi>i</mi></mrow></msub><mo>&lt;</mo><msub><mrow><mi mathvariant="normal">S</mi></mrow><mrow><mn>0</mn></mrow></msub></mrow></mfenced><mo stretchy="false">|</mo></mrow></mrow></math></span> , <em>N </em> = 1,000. </p><p> The mean distance to the first STOP codon (shown e.g. in <a reveal-id="dsw019-F2" data-open="dsw019-F2" class="link link-reveal link-table xref-fig">Fig. 2</a> ) was calculated over all introns and then compared to the distribution of means in the randomized sequences, which maintain the length and nucleotide distribution per intron. Note that performing the <em>P</em> -value computation separately for each intron cannot ‘mathematically’ work in this case: since three of the codons are STOP codons, the probability to get an in-frame intronic STOP codon in a certain position in the case of uniform nucleotide distribution is ∼0.05 (1–61/64). Similarly, the probability to get a STOP codon in the first or second positions is ∼0.09 (1–(61/64)*(61/64)). Thus, a significant intron (with <em>P</em> ∼ 0.05 or very close to this value) can only be observed if the STOP codon is exactly in the first ‘codon’ of the intron. Nevertheless, for a large set of introns (e.g <em>.</em> an entire intronome) we have enough ‘degrees of freedom’ to detect such signals. </p>                        <a id="87197607" scrollto-destination="87197607"></a>
<div id="dsw019-F2" data-id="dsw019-F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/m_dsw019f2p.png?Expires=2147483647&Signature=shNnBaXK0o0I3NeOvxB0O18teIPWwtC4TuaPiqY59XKbTD7EXLXlIvPFiXEF1bAZK0391Mx3x~Zw6T6AuXBwOWSA~6yDnmcTVhz1na6tYg325lD8OLADTCC2yKxmx37wqVNOnsDmZAtblLTSZxA~mtiaUqyGpmDvfFqEQIH0TuSTkvWoMNnuzi~XUFs1LDbUOqz8FVYXAcCdp72LHNdOg7mYzeHiPlccTP~uV-GzuyklSzGoMuEWy6plQLWQXJakcqwNldcOblFBzAmpreiG5wf2HmY8YDCk5lDhTcsaO6pvtf2oxuM0cSnwOA5WvM0XFC700jqovMq1wt07YKQeHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" First intronic STOP codon analysis. The position of the first STOP codon relatively to the beginning of the intron in the actual retained intronome tends to be closer to the 5′SS than in randomized intronome models. (A) Illustration of the translation process showing ribosomes on a transcript with a retained intron; the generated protein contains amino acids (AAs) that are encoded in the intronic nucleotide composition; as demonstrated in this case translation is usually terminated by a PTC, resulting in a truncated peptide. (B and C) Analysis of the average potential truncated peptide length distribution over the entire transcriptome (transcripts with introns) in the randomized models compared to the actual ones: (B) Average truncated peptide length distribution in S. cerevisiae suggests that there is preference for shorter retained protein length; the average length of the randomized model is 37% longer than the actual intronome (20.37AA vs. 14.89AA, respectively; empirical P  = 0.024; see Materials and methods; actual length displayed in broken line). (C) Average truncated peptide length distribution in S. pombe suggests that there is preference for shorter retained protein length; the average length of the randomized model is 8% longer than the actual intronome (21.49AA vs. 19.91AA, respectively; empirical P  = 0.025; actual length displayed by broken line); for comparison, the average truncated peptide length in the case of uniform nucleotide distribution is 21.33AA. (D and E) Average potential truncated peptide length over subsets of the actual transcriptome in comparison to the randomized models: (D) Intronome analysis of highly vs. lowly expressed genes in S. cerevisiae exhibits 76% longer truncated protein length in lowly expressed genes (9.13AA vs. 16.08AA; left and right, respectively); distribution analysis demonstrates evidence of selection in highly expressed genes but no significant selection in lowly expressed genes (empirical P  &lt;  1·10 −3 and empirical P  = 0.242; left and right, respectively; actual length displayed by broken line). (E) Intronome analysis of highly vs. lowly expressed genes in S. pombe exhibits 143% longer truncated protein length in lowly expressed genes (11.34AA vs. 27.52AA; left and right, respectively); distribution analysis demonstrates evidence of selection in highly expressed genes but no significant selection in lowly expressed genes (empirical P  &lt;  5·10 −3 and empirical P  = 0.665, respectively; actual length displayed by broken line). " path-from-xml="dsw019f2p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197607/dsw019f2p.png" path-from-xml="dsw019f2p.png" target="_blank">View large</a><a section="87197607" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019f2p.png?Expires=2147483647&Signature=KMW-0f1SIPz0LIfezcQ1S-86E3QTCNU8TuAT5w9XtpLccHgZRRRWxjZ3HpIGzTk7UWg8P32Nl0U6rSgD-OzvX~q5MIO1Zkz1J4Pdsn~2SqEhkiA5LmWEFZJZoEb6kGplvSA9M9BQ85kzWLpgMqbohoKW2RBJYwgrIt6N1R~HYkCDd~GDbNFVMtvABIonvEg~mfzt-I5hv-V9Wp2rlJMoTOLyjtSqCqfqUQx-H0VvFL~CKOE~bh~PSu3e-B~UmbM1uc8vRDkEuoTVW5uqE8pYAOU5OV0VXBUfYrjJKkY~QUIc6fTBYRhd5Hx9sQ2aUbYwmmgs1HQ2hrfAfY6TKY-YBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197607&ar=2469962&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw019f2p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> First intronic STOP codon analysis. The position of the first STOP codon relatively to the beginning of the intron in the actual retained intronome tends to be closer to the 5′SS than in randomized intronome models. (A) Illustration of the translation process showing ribosomes on a transcript with a retained intron; the generated protein contains amino acids (AAs) that are encoded in the intronic nucleotide composition; as demonstrated in this case translation is usually terminated by a PTC, resulting in a truncated peptide. (B and C) Analysis of the average potential truncated peptide length distribution over the entire transcriptome (transcripts with introns) in the randomized models compared to the actual ones: (B) Average truncated peptide length distribution in <em>S. cerevisiae</em> suggests that there is preference for shorter retained protein length; the average length of the randomized model is 37% longer than the actual intronome (20.37AA <em>vs.</em> 14.89AA, respectively; empirical <em>P</em>  = 0.024; see Materials and methods; actual length displayed in broken line). (C) Average truncated peptide length distribution in <em>S. pombe</em> suggests that there is preference for shorter retained protein length; the average length of the randomized model is 8% longer than the actual intronome (21.49AA <em>vs.</em> 19.91AA, respectively; empirical <em>P</em>  = 0.025; actual length displayed by broken line); for comparison, the average truncated peptide length in the case of uniform nucleotide distribution is 21.33AA. (D and E) Average potential truncated peptide length over subsets of the actual transcriptome in comparison to the randomized models: (D) Intronome analysis of highly <em>vs.</em> lowly expressed genes in <em>S. cerevisiae</em> exhibits 76% longer truncated protein length in lowly expressed genes (9.13AA <em>vs.</em> 16.08AA; left and right, respectively); distribution analysis demonstrates evidence of selection in highly expressed genes but no significant selection in lowly expressed genes (empirical <em>P</em>  &lt;  1·10 <sup>−3</sup> and empirical <em>P</em>  = 0.242; left and right, respectively; actual length displayed by broken line). (E) Intronome analysis of highly <em>vs.</em> lowly expressed genes in <em>S. pombe</em> exhibits 143% longer truncated protein length in lowly expressed genes (11.34AA <em>vs.</em> 27.52AA; left and right, respectively); distribution analysis demonstrates evidence of selection in highly expressed genes but no significant selection in lowly expressed genes (empirical <em>P</em>  &lt;  5·10 <sup>−3</sup> and empirical <em>P</em>  = 0.665, respectively; actual length displayed by broken line). </p></div></div><div content-id="dsw019-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/m_dsw019f2p.png?Expires=2147483647&Signature=shNnBaXK0o0I3NeOvxB0O18teIPWwtC4TuaPiqY59XKbTD7EXLXlIvPFiXEF1bAZK0391Mx3x~Zw6T6AuXBwOWSA~6yDnmcTVhz1na6tYg325lD8OLADTCC2yKxmx37wqVNOnsDmZAtblLTSZxA~mtiaUqyGpmDvfFqEQIH0TuSTkvWoMNnuzi~XUFs1LDbUOqz8FVYXAcCdp72LHNdOg7mYzeHiPlccTP~uV-GzuyklSzGoMuEWy6plQLWQXJakcqwNldcOblFBzAmpreiG5wf2HmY8YDCk5lDhTcsaO6pvtf2oxuM0cSnwOA5WvM0XFC700jqovMq1wt07YKQeHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" First intronic STOP codon analysis. The position of the first STOP codon relatively to the beginning of the intron in the actual retained intronome tends to be closer to the 5′SS than in randomized intronome models. (A) Illustration of the translation process showing ribosomes on a transcript with a retained intron; the generated protein contains amino acids (AAs) that are encoded in the intronic nucleotide composition; as demonstrated in this case translation is usually terminated by a PTC, resulting in a truncated peptide. (B and C) Analysis of the average potential truncated peptide length distribution over the entire transcriptome (transcripts with introns) in the randomized models compared to the actual ones: (B) Average truncated peptide length distribution in S. cerevisiae suggests that there is preference for shorter retained protein length; the average length of the randomized model is 37% longer than the actual intronome (20.37AA vs. 14.89AA, respectively; empirical P  = 0.024; see Materials and methods; actual length displayed in broken line). (C) Average truncated peptide length distribution in S. pombe suggests that there is preference for shorter retained protein length; the average length of the randomized model is 8% longer than the actual intronome (21.49AA vs. 19.91AA, respectively; empirical P  = 0.025; actual length displayed by broken line); for comparison, the average truncated peptide length in the case of uniform nucleotide distribution is 21.33AA. (D and E) Average potential truncated peptide length over subsets of the actual transcriptome in comparison to the randomized models: (D) Intronome analysis of highly vs. lowly expressed genes in S. cerevisiae exhibits 76% longer truncated protein length in lowly expressed genes (9.13AA vs. 16.08AA; left and right, respectively); distribution analysis demonstrates evidence of selection in highly expressed genes but no significant selection in lowly expressed genes (empirical P  &lt;  1·10 −3 and empirical P  = 0.242; left and right, respectively; actual length displayed by broken line). (E) Intronome analysis of highly vs. lowly expressed genes in S. pombe exhibits 143% longer truncated protein length in lowly expressed genes (11.34AA vs. 27.52AA; left and right, respectively); distribution analysis demonstrates evidence of selection in highly expressed genes but no significant selection in lowly expressed genes (empirical P  &lt;  5·10 −3 and empirical P  = 0.665, respectively; actual length displayed by broken line). " path-from-xml="dsw019f2p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197607/dsw019f2p.png" path-from-xml="dsw019f2p.png" target="_blank">View large</a><a section="87197607" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019f2p.png?Expires=2147483647&Signature=KMW-0f1SIPz0LIfezcQ1S-86E3QTCNU8TuAT5w9XtpLccHgZRRRWxjZ3HpIGzTk7UWg8P32Nl0U6rSgD-OzvX~q5MIO1Zkz1J4Pdsn~2SqEhkiA5LmWEFZJZoEb6kGplvSA9M9BQ85kzWLpgMqbohoKW2RBJYwgrIt6N1R~HYkCDd~GDbNFVMtvABIonvEg~mfzt-I5hv-V9Wp2rlJMoTOLyjtSqCqfqUQx-H0VvFL~CKOE~bh~PSu3e-B~UmbM1uc8vRDkEuoTVW5uqE8pYAOU5OV0VXBUfYrjJKkY~QUIc6fTBYRhd5Hx9sQ2aUbYwmmgs1HQ2hrfAfY6TKY-YBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197607&ar=2469962&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw019f2p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> First intronic STOP codon analysis. The position of the first STOP codon relatively to the beginning of the intron in the actual retained intronome tends to be closer to the 5′SS than in randomized intronome models. (A) Illustration of the translation process showing ribosomes on a transcript with a retained intron; the generated protein contains amino acids (AAs) that are encoded in the intronic nucleotide composition; as demonstrated in this case translation is usually terminated by a PTC, resulting in a truncated peptide. (B and C) Analysis of the average potential truncated peptide length distribution over the entire transcriptome (transcripts with introns) in the randomized models compared to the actual ones: (B) Average truncated peptide length distribution in <em>S. cerevisiae</em> suggests that there is preference for shorter retained protein length; the average length of the randomized model is 37% longer than the actual intronome (20.37AA <em>vs.</em> 14.89AA, respectively; empirical <em>P</em>  = 0.024; see Materials and methods; actual length displayed in broken line). (C) Average truncated peptide length distribution in <em>S. pombe</em> suggests that there is preference for shorter retained protein length; the average length of the randomized model is 8% longer than the actual intronome (21.49AA <em>vs.</em> 19.91AA, respectively; empirical <em>P</em>  = 0.025; actual length displayed by broken line); for comparison, the average truncated peptide length in the case of uniform nucleotide distribution is 21.33AA. (D and E) Average potential truncated peptide length over subsets of the actual transcriptome in comparison to the randomized models: (D) Intronome analysis of highly <em>vs.</em> lowly expressed genes in <em>S. cerevisiae</em> exhibits 76% longer truncated protein length in lowly expressed genes (9.13AA <em>vs.</em> 16.08AA; left and right, respectively); distribution analysis demonstrates evidence of selection in highly expressed genes but no significant selection in lowly expressed genes (empirical <em>P</em>  &lt;  1·10 <sup>−3</sup> and empirical <em>P</em>  = 0.242; left and right, respectively; actual length displayed by broken line). (E) Intronome analysis of highly <em>vs.</em> lowly expressed genes in <em>S. pombe</em> exhibits 143% longer truncated protein length in lowly expressed genes (11.34AA <em>vs.</em> 27.52AA; left and right, respectively); distribution analysis demonstrates evidence of selection in highly expressed genes but no significant selection in lowly expressed genes (empirical <em>P</em>  &lt;  5·10 <sup>−3</sup> and empirical <em>P</em>  = 0.665, respectively; actual length displayed by broken line). </p></div></div>                            <h4 scrollto-destination=87197610 id="87197610" class="section-title" >CUB calculation and randomization</h4>
<p> The mean of TDR scores were calculated using the modified Sharp and Li formulation: <a reveal-id="dsw019-B62" data-open="dsw019-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a><sup>,</sup><a reveal-id="dsw019-B63" data-open="dsw019-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a><sup>,</sup><span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">T</mi><mi mathvariant="normal" xmlns="">D</mi><mi mathvariant="normal" xmlns="">R</mi><mo xmlns="">=</mo><mrow xmlns=""><mrow><mi mathvariant="normal">exp</mi></mrow><mrow><mfenced separators="|"><mrow><mrow><mrow><mn>1</mn></mrow><mo>/</mo><mrow><mi>L</mi></mrow></mrow><mrow><munderover><mo stretchy="false">∑</mo><mrow><mi>l</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>L</mi></mrow></munderover><mrow><mrow><mrow><mi mathvariant="italic">ln</mi></mrow><mrow><mfenced separators="|"><mrow><mrow><mrow><mn>1</mn></mrow><mo>/</mo><mrow><msub><mrow><mi>μ</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow></mrow><mfenced separators="|"><mrow><mi>l</mi></mrow></mfenced></mrow></mfenced></mrow></mrow></mrow></mrow></mrow></mfenced></mrow></mrow></math></span> , where <em>L</em> is the AA sequence length, and <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>μ</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> is the typical codon decoding time of the <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>i</mi></mrow><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">h</mi></mrow></msub></math></span> codon; <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">μ</mi></math></span> values for <em>S. cerevisiae</em> were taken from Ref. <a reveal-id="dsw019-B64" data-open="dsw019-B64" class="link link-ref link-reveal xref-bibr">64</a> and are based on experimental measurements. <a reveal-id="dsw019-B60" data-open="dsw019-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> The tRNA adaptation index (tAI) scores were calculated using the scheme described in Ref. <a reveal-id="dsw019-B65" data-open="dsw019-B65" class="link link-ref link-reveal xref-bibr">65</a> ; genomic tRNA copy numbers for <em>S. cerevisiae</em> and <em>S.</em><em> pombe</em> were obtained from the genomic tRNA Database; <a reveal-id="dsw019-B66" data-open="dsw019-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> for <em>A. nidulans</em> , and <em>C. albicans</em> , we used the tRNA copy numbers reported in Ref. <a reveal-id="dsw019-B67" data-open="dsw019-B67" class="link link-ref link-reveal xref-bibr">67</a> while organism-specific <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>S</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></math></span> values were taken from Ref. <a reveal-id="dsw019-B68" data-open="dsw019-B68" class="link link-ref link-reveal xref-bibr">68</a> . </p><div class="block-child-p"> In the preparation of the mean profiles we used a single codon resolution (i.e. <em>L = 1</em> ; see an illustrative scheme in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span> ). The profiles of the TDR ( <a reveal-id="dsw019-F3" data-open="dsw019-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a> ) and tAI ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S4–S6</a></span> ; C and D) were computed in an identical manner; hence, here we use the tAI notations to describe the procedure. Let <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msup xmlns=""><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">A</mi><mi mathvariant="normal">I</mi><mo>(</mo><mi>i</mi><mo>)</mo></mrow><mrow><mi>j</mi></mrow></msup></math></span> denote the tAI value of a single codon, located in the <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>i</mi></mrow><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">h</mi></mrow></msub></math></span> AA of a gene <em>j</em> with length <em>n</em> . Gene tAI profile is defined as the vector of all the gene tAI values, i.e. <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">A</mi><mi mathvariant="normal">I</mi></mrow><mrow><msub><mrow><mi mathvariant="normal">G</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">e</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></msub><mo xmlns="">=</mo><mfenced separators="|" xmlns=""><mrow><msup><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">A</mi><mi mathvariant="normal">I</mi></mrow><mrow><mi>j</mi></mrow></msup><mo>(</mo><mn>1</mn><mo>)</mo><mo>,</mo><msup><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">A</mi><mi mathvariant="normal">I</mi></mrow><mrow><mi>j</mi></mrow></msup><mo>(</mo><mn>2</mn><mo>)</mo><mo>,</mo><mo>…</mo><mo>,</mo><msup><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">A</mi><mi mathvariant="normal">I</mi></mrow><mrow><mi>j</mi></mrow></msup><mo>(</mo><mi>n</mi><mo>)</mo></mrow></mfenced></math></span> . For each organism, all intron-containing genes were aligned once according to their 5′SS location and once according to their first intronic STOP codon location. Let <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>i</mi></mrow><mrow><mn>5</mn><mi>'</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">s</mi></mrow></msub></math></span> and <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>i</mi></mrow><mrow><mi mathvariant="normal">s</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">p</mi></mrow></msub></math></span> denote the positions of the 5′SS and first intronic STOP codon (for each of the analysed introns these indexes correspond to its 5′SS and STOP codon, respectively). The profiles of mean tAI were calculated based on window WS of 30 codons (i.e. 90 nt). Thus: <a id="E1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mover accent="true" xmlns=""><mrow><msub><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">A</mi><mi mathvariant="normal">I</mi></mrow><mrow><mn>5</mn><mi mathvariant="normal">'</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">s</mi></mrow></msub></mrow><mo>¯</mo></mover><mo xmlns="">=</mo><mfenced separators="|" xmlns=""><mrow><mover accent="false"><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">A</mi><mi mathvariant="normal">I</mi><mo>(</mo><msub><mrow><mi>i</mi></mrow><mrow><mn>5</mn><mi mathvariant="normal">'</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">s</mi></mrow></msub><mo>-</mo><mi mathvariant="normal">W</mi><mi mathvariant="normal">S</mi><mo>+</mo><mn>1</mn><mo>)</mo></mrow><mo>¯</mo></mover><mo>,</mo><mo>…</mo><mo>,</mo><mover accent="false"><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">A</mi><mi mathvariant="normal">I</mi><mfenced separators="|"><mrow><msub><mrow><mi>i</mi></mrow><mrow><mn>5</mn><mi>'</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">s</mi></mrow></msub></mrow></mfenced></mrow><mo>¯</mo></mover><mo>,</mo><mo>…</mo><mo>,</mo><mover accent="false"><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">A</mi><mi mathvariant="normal">I</mi><mo>(</mo><msub><mrow><mi>i</mi></mrow><mrow><mn>5</mn><mi>'</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">s</mi></mrow></msub><mo>+</mo><mi mathvariant="normal">W</mi><mi mathvariant="normal">S</mi><mo>)</mo></mrow><mo>¯</mo></mover></mrow></mfenced></math></div></div></div><a id="E2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E2"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mover accent="true" xmlns=""><mrow><msub><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">A</mi><mi mathvariant="normal">I</mi></mrow><mrow><mi mathvariant="normal">S</mi><mi mathvariant="normal">T</mi><mi mathvariant="normal">O</mi><mi mathvariant="normal">P</mi></mrow></msub></mrow><mo>¯</mo></mover><mo xmlns="">=</mo><mfenced separators="|" xmlns=""><mrow><mover accent="false"><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">A</mi><mi mathvariant="normal">I</mi><mo>(</mo><msub><mrow><mi>i</mi></mrow><mrow><mi mathvariant="normal">s</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">p</mi></mrow></msub><mo>-</mo><mi mathvariant="normal">W</mi><mi mathvariant="normal">S</mi><mo>+</mo><mn>1</mn><mo>)</mo></mrow><mo>¯</mo></mover><mo>,</mo><mo>…</mo><mo>,</mo><mover accent="false"><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">A</mi><mi mathvariant="normal">I</mi><mfenced separators="|"><mrow><msub><mrow><mi>i</mi></mrow><mrow><mi mathvariant="normal">s</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">p</mi></mrow></msub></mrow></mfenced></mrow><mo>¯</mo></mover><mo>,</mo><mo>…</mo><mo>,</mo><mover accent="false"><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">A</mi><mi mathvariant="normal">I</mi><mo>(</mo><msub><mrow><mi>i</mi></mrow><mrow><mi mathvariant="normal">s</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">p</mi></mrow></msub><mo>+</mo><mi mathvariant="normal">W</mi><mi mathvariant="normal">S</mi><mo>)</mo></mrow><mo>¯</mo></mover></mrow></mfenced><mo xmlns="">,</mo></math></div></div></div></div>                        <a id="87197615" scrollto-destination="87197615"></a>
<div id="dsw019-F3" data-id="dsw019-F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/m_dsw019f3p.png?Expires=2147483647&Signature=ZILpfaLolL9xGKQJHL2T9oPDpjraIh-WkBly2Zc12EIIR3Y0siFIzNoJrYtdApccvhOC-aKS0XUa9s8sQjLw0oQJ8UxN8iB4lMEHfRGOI61NFymGzDf3OE0KYeNWDV7MSiNqi2LN8GR3~llCIF0Q9BK0oBu7R6pwwiic5pThn4LgyySOKLe7BWLmiiFGz0Dkg0LtI9a~~f8K0VMRWLhRlI7BS4QKh80BiTJND5gPQJg80E8G7TqUa9zvrHNLeDQvt~eM~dqxm19a5Mza4owLz1ci15dCscxZMYVVVl8RvL5oNJBxXn425NhA793IzC-9pPbam0tOARQGfrLTVwEVkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" CUB profiles for the S. cerevisiae intronome. The profiles show that the average of TDRs index values upstream from the first intronic STOP codon is higher than the average of the TDR values downstream of it, supporting the conjecture that the beginnings of introns undergo evolutionary selection for higher TE. (A) Actual and random average TDR values (blue and green, respectively; see Materials and methods) aligned to the beginning of the 5′SS (Exonic Donor) and the first intronic STOP codon (pre-STOP, post-STOP) locations. (B) Mean TDR profile aligned to the beginning of the 5′SS (top) and the first Intronic STOP codon (bottom). As expected, TDR values downstream from the 5′ end of the exon/intron boundary (right side of the 5′SS) are lower than upstream of it (top; P  = 2.04·10 −8 , Wilcoxon signed-rank test between the exonic and pre-STOP values, per gene). Mean TDR profile of the first Intronic STOP codon demonstrates that the TDR upstream from the first intronic STOP location is higher than the TDR downstream of it (bottom; P  = 4.91·10 −18 , Wilcoxon signed-rank test between the pre-STOP and post-STOP values, per gene). (C) Mean values over all regions of the actual intronome and randomized version (blue and green, respectively). The STD of the randomized genomes for these regions (light magenta) shows a significant signal in the pre-STOP domain; mean values are 5.239/5.167/4.929 for the actual intronome and 5.236/5.163/4.948 for the randomized intronome (left, middle, and right; respectively). (D) High standard normalization global Z-scores (red, over all intronic ‘codons’; see Materials and methods) in the pre-STOP domain and very low global Z-scores in the post-STOP domain support the conjecture that the intronic pre-STOP codon selective pressure hypothesis is true; genes with STOP codons positioned in the downstream exon and locations with &lt;15% of the intronome were ignored. " path-from-xml="dsw019f3p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197615/dsw019f3p.png" path-from-xml="dsw019f3p.png" target="_blank">View large</a><a section="87197615" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019f3p.png?Expires=2147483647&Signature=0a8gV9I1dmC19ahGIHgA2Fe1ssdhqpERtq-pJKOfaD2-lkQ4KguMmRMJAg0V3JwrHwegr32ajmTwcpn574Iy3jAlrhUWRcNlQoGLAV-9VzN4e9oGMUV5N8~S6N73MEyftaKW~wqQnxKVDu7VgaLYezZWvwpMevXfBvvNuc4c1x7qyx449KyT3xKlOY-MhVlBuzuDluCl1cqJWSxqg6Ple0~0CZpG1phu84xWNxfz1IhCXTpN3ty5PDGGR3aFWpTuK0~p5AOTh0W31lav1oUBbFwdzsnEbCuPs7poTipqm9BFeu~Uk6YN8HpwIPtN-gnvV9lH~AuVzvGg8Qm6AHzmLg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197615&ar=2469962&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw019f3p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> CUB profiles for the <em>S. cerevisiae</em> intronome. The profiles show that the average of TDRs index values upstream from the first intronic STOP codon is higher than the average of the TDR values downstream of it, supporting the conjecture that the beginnings of introns undergo evolutionary selection for higher TE. (A) Actual and random average TDR values (blue and green, respectively; see Materials and methods) aligned to the beginning of the 5′SS (Exonic Donor) and the first intronic STOP codon (pre-STOP, post-STOP) locations. (B) Mean TDR profile aligned to the beginning of the 5′SS (top) and the first Intronic STOP codon (bottom). As expected, TDR values downstream from the 5′ end of the exon/intron boundary (right side of the 5′SS) are lower than upstream of it (top; <em>P</em>  = 2.04·10 <sup>−8</sup> , Wilcoxon signed-rank test between the exonic and pre-STOP values, per gene). Mean TDR profile of the first Intronic STOP codon demonstrates that the TDR upstream from the first intronic STOP location is higher than the TDR downstream of it (bottom; <em>P</em>  = 4.91·10 <sup>−18</sup> , Wilcoxon signed-rank test between the pre-STOP and post-STOP values, per gene). (C) Mean values over all regions of the actual intronome and randomized version (blue and green, respectively). The STD of the randomized genomes for these regions (light magenta) shows a significant signal in the pre-STOP domain; mean values are 5.239/5.167/4.929 for the actual intronome and 5.236/5.163/4.948 for the randomized intronome (left, middle, and right; respectively). (D) High standard normalization global Z-scores (red, over all intronic ‘codons’; see Materials and methods) in the pre-STOP domain and very low global Z-scores in the post-STOP domain support the conjecture that the intronic pre-STOP codon selective pressure hypothesis is true; genes with STOP codons positioned in the downstream exon and locations with &lt;15% of the intronome were ignored. </p></div></div><div content-id="dsw019-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/m_dsw019f3p.png?Expires=2147483647&Signature=ZILpfaLolL9xGKQJHL2T9oPDpjraIh-WkBly2Zc12EIIR3Y0siFIzNoJrYtdApccvhOC-aKS0XUa9s8sQjLw0oQJ8UxN8iB4lMEHfRGOI61NFymGzDf3OE0KYeNWDV7MSiNqi2LN8GR3~llCIF0Q9BK0oBu7R6pwwiic5pThn4LgyySOKLe7BWLmiiFGz0Dkg0LtI9a~~f8K0VMRWLhRlI7BS4QKh80BiTJND5gPQJg80E8G7TqUa9zvrHNLeDQvt~eM~dqxm19a5Mza4owLz1ci15dCscxZMYVVVl8RvL5oNJBxXn425NhA793IzC-9pPbam0tOARQGfrLTVwEVkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" CUB profiles for the S. cerevisiae intronome. The profiles show that the average of TDRs index values upstream from the first intronic STOP codon is higher than the average of the TDR values downstream of it, supporting the conjecture that the beginnings of introns undergo evolutionary selection for higher TE. (A) Actual and random average TDR values (blue and green, respectively; see Materials and methods) aligned to the beginning of the 5′SS (Exonic Donor) and the first intronic STOP codon (pre-STOP, post-STOP) locations. (B) Mean TDR profile aligned to the beginning of the 5′SS (top) and the first Intronic STOP codon (bottom). As expected, TDR values downstream from the 5′ end of the exon/intron boundary (right side of the 5′SS) are lower than upstream of it (top; P  = 2.04·10 −8 , Wilcoxon signed-rank test between the exonic and pre-STOP values, per gene). Mean TDR profile of the first Intronic STOP codon demonstrates that the TDR upstream from the first intronic STOP location is higher than the TDR downstream of it (bottom; P  = 4.91·10 −18 , Wilcoxon signed-rank test between the pre-STOP and post-STOP values, per gene). (C) Mean values over all regions of the actual intronome and randomized version (blue and green, respectively). The STD of the randomized genomes for these regions (light magenta) shows a significant signal in the pre-STOP domain; mean values are 5.239/5.167/4.929 for the actual intronome and 5.236/5.163/4.948 for the randomized intronome (left, middle, and right; respectively). (D) High standard normalization global Z-scores (red, over all intronic ‘codons’; see Materials and methods) in the pre-STOP domain and very low global Z-scores in the post-STOP domain support the conjecture that the intronic pre-STOP codon selective pressure hypothesis is true; genes with STOP codons positioned in the downstream exon and locations with &lt;15% of the intronome were ignored. " path-from-xml="dsw019f3p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197615/dsw019f3p.png" path-from-xml="dsw019f3p.png" target="_blank">View large</a><a section="87197615" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019f3p.png?Expires=2147483647&Signature=0a8gV9I1dmC19ahGIHgA2Fe1ssdhqpERtq-pJKOfaD2-lkQ4KguMmRMJAg0V3JwrHwegr32ajmTwcpn574Iy3jAlrhUWRcNlQoGLAV-9VzN4e9oGMUV5N8~S6N73MEyftaKW~wqQnxKVDu7VgaLYezZWvwpMevXfBvvNuc4c1x7qyx449KyT3xKlOY-MhVlBuzuDluCl1cqJWSxqg6Ple0~0CZpG1phu84xWNxfz1IhCXTpN3ty5PDGGR3aFWpTuK0~p5AOTh0W31lav1oUBbFwdzsnEbCuPs7poTipqm9BFeu~Uk6YN8HpwIPtN-gnvV9lH~AuVzvGg8Qm6AHzmLg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197615&ar=2469962&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw019f3p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> CUB profiles for the <em>S. cerevisiae</em> intronome. The profiles show that the average of TDRs index values upstream from the first intronic STOP codon is higher than the average of the TDR values downstream of it, supporting the conjecture that the beginnings of introns undergo evolutionary selection for higher TE. (A) Actual and random average TDR values (blue and green, respectively; see Materials and methods) aligned to the beginning of the 5′SS (Exonic Donor) and the first intronic STOP codon (pre-STOP, post-STOP) locations. (B) Mean TDR profile aligned to the beginning of the 5′SS (top) and the first Intronic STOP codon (bottom). As expected, TDR values downstream from the 5′ end of the exon/intron boundary (right side of the 5′SS) are lower than upstream of it (top; <em>P</em>  = 2.04·10 <sup>−8</sup> , Wilcoxon signed-rank test between the exonic and pre-STOP values, per gene). Mean TDR profile of the first Intronic STOP codon demonstrates that the TDR upstream from the first intronic STOP location is higher than the TDR downstream of it (bottom; <em>P</em>  = 4.91·10 <sup>−18</sup> , Wilcoxon signed-rank test between the pre-STOP and post-STOP values, per gene). (C) Mean values over all regions of the actual intronome and randomized version (blue and green, respectively). The STD of the randomized genomes for these regions (light magenta) shows a significant signal in the pre-STOP domain; mean values are 5.239/5.167/4.929 for the actual intronome and 5.236/5.163/4.948 for the randomized intronome (left, middle, and right; respectively). (D) High standard normalization global Z-scores (red, over all intronic ‘codons’; see Materials and methods) in the pre-STOP domain and very low global Z-scores in the post-STOP domain support the conjecture that the intronic pre-STOP codon selective pressure hypothesis is true; genes with STOP codons positioned in the downstream exon and locations with &lt;15% of the intronome were ignored. </p></div></div><p> where <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns=""> </mi><mover accent="false" xmlns=""><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">A</mi><mi mathvariant="normal">I</mi><mfenced separators="|"><mrow><mi>i</mi></mrow></mfenced></mrow><mo>¯</mo></mover><mo xmlns="">=</mo><mrow xmlns=""><munder><mo stretchy="false">∑</mo><mrow><mi>j</mi></mrow></munder><mrow><mfrac><mrow><msup><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">A</mi><mi mathvariant="normal">I</mi></mrow><mrow><mi>j</mi></mrow></msup><mfenced separators="|"><mrow><mi>i</mi></mrow></mfenced></mrow><mrow><mfenced close="|" open="|" separators="|"><mrow><msub><mrow><mi mathvariant="normal">G</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">e</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mfenced></mrow></mfrac><mo>,</mo></mrow></mrow></math></span> and <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi mathvariant="normal">G</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">e</mi></mrow><mrow><mi>j</mi></mrow></msub></math></span> is the number of genes with translatable codons at their <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>i</mi></mrow><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">h</mi></mrow></msub></math></span> location. Additionally, downstream exons in 5′SS profiles and all exons in STOP profiles were excluded, as well as UTRs such as 5′UTR and 3′UTR; analysis of STOP profiles when downstream exons are included (as illustrated in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Fig. S8</a></span> ) showed similar results. Values for the entire domains (shown e.g. in <a reveal-id="dsw019-F3" data-open="dsw019-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a> ) were calculated as the geometric mean per intron and then averaged over all introns. Wilcoxon signed-rank (paired) tests and paired <em>t</em> -tests were performed between the upstream/downstream average values per gene on the TDR/tAI profiles; additional statistics, including Spearman correlations of TDR/tAI with PA levels, can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Table S1</a></span> . </p><p> The randomized models used maintained the CDS codon frequencies (i.e. CUB) and uniformly permutated the intronic nucleotides, maintaining consensus sequences (as previously described; see also <a reveal-id="dsw019-F1" data-open="dsw019-F1" class="link link-reveal link-table xref-fig">Fig. 1C and D</a> ). During randomization, the 5′SS alignment termination position ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Fig. S8A–C</a></span> ) was set according to the randomization results, i.e. randomized termination point location. In the case of termination point alignment ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Fig. S8F</a></span> ), the termination point alignment position was set to the original place of the retained intron in order to avoid location bias. The intronome fraction level which determines the edges of analysis was set to 15% of the overall intronome (the results are robust to reasonable changes in this cut-off). </p>                            <h3 scrollto-destination=87197620 id="87197620" class="section-title" >Computing Z scores based on the randomized models</h3>
<p> Standard normal distribution scoring (i.e. Z-score) is a statistical measure that can be used for quantitative selection level evaluation via the comparison of the real signal to a randomized one; higher Z-score means higher chance that the selective pressure hypothesis is true and vice versa. Absolute values higher than 1.96 are typically considered to be significant, based on 95% confidence level (i.e. <em>P</em>  &lt; 0.05). Z-score values were calculated according to the following equation: <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi> </mi><mi>Z</mi></mrow><mrow><mi mathvariant="normal">s</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi></mrow></msub><mo xmlns="">=</mo><mfrac xmlns=""><mrow><msub><mrow><mi>μ</mi></mrow><mrow><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">l</mi></mrow></msub><mo>-</mo><msub><mrow><mi>μ</mi></mrow><mrow><mi mathvariant="normal">r</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">d</mi></mrow></msub></mrow><mrow><msub><mrow><mi>σ</mi></mrow><mrow><mi mathvariant="normal">r</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">d</mi></mrow></msub></mrow></mfrac></math></span> where <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>μ</mi></mrow><mrow><mi mathvariant="normal">r</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">l</mi></mrow></msub></math></span> is the mean related to the actual intronome, <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>μ</mi></mrow><mrow><mi mathvariant="normal">r</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">d</mi></mrow></msub></math></span> is the random model mean, and <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>σ</mi></mrow><mrow><mi mathvariant="normal">r</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">d</mi></mrow></msub></math></span> is the random model standard deviation (STD). We used the following scheme related to the TDR/tAI profiles values: for each randomized model, a vector of average values was generated (i.e. an average for each random intronome) and the global Z-score was calculated. </p>                            <h3 scrollto-destination=87197624 id="87197624" class="section-title" >Ribosomal profiling analysis</h3>
<p> The ribosomal profiling (RP) method gives quantitative information of ribosome footprints and ribosomal density (RD) in a single nucleotide resolution. <a reveal-id="dsw019-B60" data-open="dsw019-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> The analysis was performed as follows: RP raw data were obtained from NCBI GEO database <a reveal-id="dsw019-B69" data-open="dsw019-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a> (accession GSE13750) for two footprinting experiment replicates in rich media and mapped in a similar manner to the one described in Refs <a reveal-id="dsw019-B60" data-open="dsw019-B60" class="link link-ref link-reveal xref-bibr">60</a> and <a reveal-id="dsw019-B70" data-open="dsw019-B70" class="link link-ref link-reveal xref-bibr">70</a> . It includes the following major steps: (i) Degenerate reads, i.e. reads in which 18 or more of the first 22 nt are A bases (meaning the entire read is composed of the homopolymer A tail), were eliminated. Reads mapped to ribosomal RNA genes and other non-coding genes, such as tRNA, snRNA, snoRNA, taken from Biomart (version R64-1-1), <a reveal-id="dsw019-B71" data-open="dsw019-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> were excluded. (ii) The entire pre-mRNA set of the <em>S. cerevisiae</em> genome was reconstructed utilizing BioMart (version R64-1-1); <a reveal-id="dsw019-B71" data-open="dsw019-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a><sup>,</sup><a reveal-id="dsw019-B59" data-open="dsw019-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> and each pre-mRNA included its 5′UTR, exons, introns, and 3′UTR, genes with no 5′UTR and 3′UTR annotations were supplemented with flanking genomic segments with the maximum length ensuring there is no overlap with existing coding regions. (iii) Reads of 36 nt were mapped using bowtie <a reveal-id="dsw019-B72" data-open="dsw019-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> to the pre-mRNA sequences allowing up to two mismatches on a seed length of 21 nt, to account for polyadenylation of footprint fragments. (iv) Non-uniquely mapped reads were extended from the mapped seed length of 21–28 nt to determine uniqueness, by first removing the poly-A tail of each 36 nt long read according to the following heuristic: we determine the poly-A tail from the read’s end, by identifying the longest stretch of As allowing two mismatches and removing it, reads longer than 28 nt were cropped, and reads shorter than the 21 nt seed length after poly-A tail removal were discarded, now the read with the lowest mismatch score was selected if it uniquely exists; otherwise, the remaining contender multi-reads were distributed proportionally to the density of the uniquely mapped reads in their vicinity (30 nt upstream and downstream). </p><p> Based on the RP footprint read counts (RCs) of the two aforementioned experiment replicates, and the 5′UTR, exon, intron, 3′UTR annotations of 282 <em>S. cerevisiae</em> intron-containing genes, we performed the following analysis: (i) the position of the ribosome A-site was related to 15 nt from the beginning of the read; (ii) we averaged the profiles obtained for the two replicates; (iii) averaged profiles were generated in a similar way to the tAI profiles previously described, however in a nucleotide-based resolution with the following adjustments: downstream exons from the 5′SS alignment were excluded, as well as upstream exons from the STOP alignment; in addition, genes with a STOP codon positioned in the downstream exon were excluded; in the second intronic STOP analysis, RCs upstream from the first intronic STOP codon were excluded as well. Wilcoxon signed-rank test and paired <em>t</em> -test were performed between the upstream/downstream average values per gene on the RP ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Table S1</a></span> ). Analysis scheme with a possible gene example is presented in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Fig. S10</a></span> . The estimation of the difference between the pre-STOP and post-STOP domain (shown in <a reveal-id="dsw019-F7" data-open="dsw019-F7" class="link link-reveal link-table xref-fig">Fig. 7G</a> ) was done using 3,804 genes with observed protein levels taken from Ref. <a reveal-id="dsw019-B73" data-open="dsw019-B73" class="link link-ref link-reveal xref-bibr">73</a> , while excluding putative genes, dubious ORFs, and genes with No RC values at all. List of RP genes with nonzero RC in their pre-STOP domain can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Table S6</a></span> . Additional A-site related offset alignments of 14 and 16 nt show similar results ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Fig. S12</a></span> ). </p>                            <h3 scrollto-destination=87197629 id="87197629" class="section-title" >Subgroups analysis and bias control</h3>
<p> Introns were divided into subgroup sets according to various criteria: ribosomal <em>vs.</em> non-ribosomal, and highly expressed <em>vs.</em> lowly expressed (based on PA, RD, mRNA, or PPR measurements). Subgroups of different size may cause bias in the strength/significance of the detected statistical signals; therefore (and in order to control for the effect of different group sizes on the Z-score), we ensured that the size of both subgroup pairs is identical (i.e. equal number of introns in each subgroup); in case of a difference, introns were chosen randomly form the larger group to match subgroup size (e.g. in <em>S. cerevisiae</em> 93 introns were randomly chosen from a set of 187 non-ribosomal introns in total to match the ribosomal ones). Two sided Wilcoxon rank-sum test and two sample <em>t</em> -test were performed on the TDR/tAI profiles. </p>                            <h3 scrollto-destination=87197632 id="87197632" class="section-title" >Synthetic YiFP reporter library building and analysis</h3>
<p> The synthetic YiFP reporter library was downloaded from Yofe <em>et al.</em><a reveal-id="dsw019-B74" data-open="dsw019-B74" class="link link-ref link-reveal xref-bibr"><sup>74</sup></a> Briefly, to create the synthetic intron reporter library, <em>S. cerevisiae</em> was transformed with a library of DNA transformation cassettes; each containing a different native yeast intron. The cassettes were assembled using the Y-operation <a reveal-id="dsw019-B75" data-open="dsw019-B75" class="link link-ref link-reveal xref-bibr"><sup>75</sup></a><sup>,</sup><a reveal-id="dsw019-B76" data-open="dsw019-B76" class="link link-ref link-reveal xref-bibr"><sup>76</sup></a> by which introns were embedded in a Yellow Fluorescent Protein (YFP) fragment and concatenated to a common selection marker in high throughput. In this manner 240 strains were created, termed YiFP strains, where the sole difference between all strains is the native <em>S. cerevisiae</em> intron intervening the YFP gene. The contribution of introns to the regulation of gene expression was assessed by dynamic measurements of YFP expression. Following normalization, the expression level of each intron strain was compared to that of the intron-less YFP strain to give a measure of its relative expression level, which is related to intronic SE. YiFP strains that had a signal to noise ratio (SNR) of 5 and above were classified as spliced (178 introns), while the others (SNR &lt; 5; 62 introns) were classified as un-spliced. </p>                            <h3 scrollto-destination=87197637 id="87197637" class="section-title" >Gene ontology function specific group analysis</h3>
<p> We used the gene ontology (GO) annotation database ( <a class="link link-uri" href="http://www.geneontology.org" target="_blank">http://www.geneontology.org</a> /, 13 May 2016, date last accessed) to derive all the functional gene groups related to the three ontology domains (Cellular component, Molecular function, and Biological process). We analysed a subset of the terms, i.e. GO Slim, evaluating 90 terms in the database with at least 50 introns (out of a total of 145 terms; each term contains at least 1% of the <em>S</em><em>. pombe</em> intronome), and built corresponding Z-score profiles for each subgroup. Profiles were then vertically aligned in three ontology clusters, to present exonic donor, pre-STOP and post STOP intervals. Significance level for each term was calculated using a two sided Wilcoxon rank-sum test on the tAI upstream and downstream intervals, aligned to the STOP codon. </p>                            <h3 scrollto-destination=87197641 id="87197641" class="section-title" >Considering cases when the first in-frame STOP codon is downstream of the introns</h3>
<p> Here, we provide some additional specific details regarding the manner in which we consider the cases when the first in-frame STOP is downstream of the introns: the cases where the first in-frame STOP codon is in the 3′ UTR are very rare (&lt;0.5%) and are ignored or do not exist; these cases are very probably related to sequencing/annotation errors (i.e. the main ORF does not include a STOP codon), and do not exist at all in well annotated organisms (e.g. <em>S. cerevisiae</em> ). We ignore the very few cases (around ∼2%) where the first in-frame STOP is a main ORF STOP codon since, by the definition of our random model, in these cases the random and real sequences are identical. It is easy to see that any reasonable consideration of these cases does not affect any conclusion of the study. </p><p>Also note that when we plot TE (e.g. adaptation to the tRNA pool/TDR) and Ribo-seq profiles we ignore non-intronic parts since we do not want to bias the results (the TDR/tAI/Ribo-seq in exons is expected to be very high in comparison to introns), and to perform the most conservative and relevant test possible.</p>                            <h2 scrollto-destination=87197647 id="87197647" class="section-title" >Results</h2>
<p> In this study, we analyse the intronome of four fungi: <em>S. cerevisiae</em> , <em>S</em><em>. pombe</em> , <em>A. nidulans</em> , and <em>C. albicans</em> ; <em>S. cerevisiae</em> ; and <em>S</em><em>. pombe</em> are both well studied and diverged ∼350–900 million years ago; <a reveal-id="dsw019-B50" data-open="dsw019-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a><em>A. nidulans</em> and <em>C. albicans</em> are two additional fungi with fully sequenced genomes and well annotated introns (see further details regarding these organisms in Materials and methods). Specifically, the analysed data include 277 introns from <em>S. cerevisiae</em> , 4,747 introns from <em>S</em><em>. pombe</em> , 2,427 introns from <em>A. nidulans</em> , and 391 introns from <em>C. albicans</em> . </p><p> Our objective was to evaluate systematically novel sequence features/patterns near the splice sites that promote splicing regulation and affect TE at the genomic level, towards a better understanding of intronic evolution. To this end, we defined three pre-mRNA exonic and intronic regions that will be used hereafter and are illustrated in <a reveal-id="dsw019-F1" data-open="dsw019-F1" class="link link-reveal link-table xref-fig">Fig. 1A and B :</a> Exonic Donor—exonic region 90 nt upstream from the 5′SS (excluding upstream 5′ UTR); pre-STOP—intronic region 90 nt upstream from the first STOP codon (excluding the STOP itself and upstream exons), aligned to the 5′SS or to the first STOP codon; post-STOP—intronic region 90 nt downstream from the first STOP codon (excluding the STOP itself, downstream exons, and 3′ UTR), aligned to the first STOP codon; the pre-mRNA exon–intron boundaries, consensus sequences, and STOP codon are illustrated as well. We focused on three related aspects: (i) The position of STOP codons in possibly retained parts of the introns. (ii) The possible adaptation of the intronic ‘codons’ to the translation-elongation step in potentially retained parts of the introns (i.e <em>.</em> the intronic beginning; in-frame of the preceding exon and before the first STOP codon). (iii) The density of ribosomes on possibly retained parts of the introns. In all cases, we performed a comparative adaption analysis of various intronic subsets and functions. </p>                            <h3 scrollto-destination=87197653 id="87197653" class="section-title" >Evidence of selective pressure for intronic STOP codons near the 5′SS presumably to decrease translation and metabolic costs of retained introns</h3>
<p> Splicing, like other biological processes, does not exhibit perfect fidelity. Thus, it is clear that there are cases where introns are not spliced, and translation-elongation continues through the retained beginning of the intronic sequence. <a reveal-id="dsw019-B14" data-open="dsw019-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> If such events are frequent enough, we presume that evolution will shape the beginning of retained introns for reduced translation and metabolic costs, and possibly for increased TE (see illustration of truncated peptide formation in <a reveal-id="dsw019-F2" data-open="dsw019-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a> ). In this subsection, we study one aspect related to this argument. </p><p> Assuming that retained introns’ translation is usually deleterious to the cell, we expect to see preference for close STOP codons or premature termination codons (PTCs) at the beginning of introns in-frame of the exons preceding them. To test this hypothesis, we compared the mean distance to the first STOP codon in possibly retained introns in actual genomes, to the ones obtained in randomized genomes that controlled for their exonic CUB and GC content, maintained intronic consensus sequences, and controlled for intronic GC content (per gene; see details in Materials and methods and in <a reveal-id="dsw019-F1" data-open="dsw019-F1" class="link link-reveal link-table xref-fig">Fig. 1C and D</a> ). Specifically, we calculated the average length of all truncated peptides originating from non-spliced transcripts (relative to the end of each preceding exon) and compared it with the average lengths obtained using the randomized models. As can be seen in <a reveal-id="dsw019-F2" data-open="dsw019-F2" class="link link-reveal link-table xref-fig">Fig. 2B and C</a> , the average truncated peptide length in the randomized models was 37% and 8% longer than the actual one for <em>S. cerevisiae</em> and <em>S. pombe</em> , respectively; evaluation of the average length distribution in the randomized models shows that it is indeed significantly longer than observed in the actual intronome (empirical <em>P</em> -value: <em>P</em>  = 2.3·10 <sup>−</sup><sup>2</sup> , <em>P</em>  = 2.4·10 <sup>−</sup><sup>2</sup> ; <em>P</em>  = 8.55·10 <sup>−</sup><sup>8</sup> and <em>P</em>  = 1.25·10 <sup>−</sup><sup>15</sup> , Wilcoxon signed-rank test between the actual and average randomized models values per gene, which is a paired test; <em>S. cerevisiae</em> and <em>S. pombe</em> , respectively; see details in Materials and methods and in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Fig. S14A and B</a></span> ). </p><p> We expect selection for intronic STOP codons near the 5′ end of the intron to be higher in highly expressed genes, which potentially consume more intercellular translation resources. Thus, we further examined two intronic subgroups of <em>S. cerevisiae</em> and <em>S. pombe</em> , originating from highly and lowly expressed genes; 60 and 500 introns were included in each subgroup for <em>S. cerevisiae</em> and <em>S. pombe</em> , respectively, constituting 21% and 11% of their respective intronome, based on their PA (see Materials and methods). Analysis confirmed that the average retained peptide length in lowly expressed genes was 76% and 143% longer than in highly expressed genes for <em>S. cerevisiae</em> and <em>S. pombe</em> , respectively ( <em>P</em>  = 3.61·10 <sup>−</sup><sup>4</sup> and <em>P</em>  = 1.54·10 <sup>−</sup><sup>2</sup> , Wilcoxon rank-sum test between the highly and lowly expressed genes; <a reveal-id="dsw019-F2" data-open="dsw019-F2" class="link link-reveal link-table xref-fig">Fig. 2D and E</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Fig. S14C and D</a></span> ). Evaluation of the average length distribution obtained using the randomized models showed that randomized highly expressed genes have significantly longer peptide length than observed in the actual intronome (empirical <em>P</em>  &lt; 1·10 <sup>−</sup><sup>3</sup> and empirical <em>P</em>  &lt;  5·10 <sup>−</sup><sup>3</sup> for <em>S. cerevisiae</em> and <em>S. pombe</em> , respectively). Contrary, lowly expressed genes did not exhibit evidence of significant selection (empirical <em>P </em> &gt;  0.24 and empirical <em>P</em>   &gt; 0.66, for <em>S. cerevisiae</em> and <em>S. pombe</em> , respectively; <a reveal-id="dsw019-F2" data-open="dsw019-F2" class="link link-reveal link-table xref-fig">Fig. 2D and E</a> ), as expected. Similar results and conclusions were obtained using randomized models that separately maintained GC content in the first and second intronic fragments ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span> ). </p><p> Furthermore, we considered the metabolic biosynthesis costs of the truncated peptides originating from the non-spliced transcripts (relative to the end of each preceding exon and up to the STOP codon; see Materials and methods). The results shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2A and B</a></span> demonstrate that the metabolic costs of the truncated peptides are 25% and 7% lower than in the randomized models for <em>S. cerevisiae</em> and <em>S. pombe</em> , respectively; analysis of the randomized models average cost shows that it is indeed significantly higher than observed in the actual intronome (empirical <em>P</em> -value: <em>P</em>  = 2.6·10 <sup>−</sup><sup>2</sup> , <em>P</em>  = 3.5·10 <sup>−</sup><sup>2</sup> , <em>P</em>  = 9.36·10 <sup>−</sup><sup>7</sup> , and <em>P</em>  = 8.5·10 <sup>−</sup><sup>15</sup> , Wilcoxon signed-rank test between the actual and average randomized models values per gene; <em>S. cerevisiae</em> and <em>S. pombe</em> , respectively; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Fig. S14E and F</a></span> ). Further analysis confirmed that the signal is stronger for highly expressed genes (highly: empirical <em>P</em>  &lt;  1·10 <sup>−</sup><sup>3</sup> and empirical <em>P</em>  &lt;  5 <strong>·</strong> 10 <sup>−</sup><sup>3</sup> ; lowly: empirical <em>P</em>  &gt; 0.26 and empirical <em>P</em>  &gt; 0.68, <em>S. cerevisiae</em> and <em>S. pombe</em> , respectively; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Fig. S2C and D</a></span> ). </p><p>Together, these results support the conjecture that introns are selected for a STOP codon close to the 5′SS in order to decrease the length and corresponding metabolic costs of undesired translated peptides in the case of intron retention, presumably to reduce the global cost of translating such peptides; not surprisingly, highly expressed genes show evidence of significantly higher levels of selection for these intronic features.</p>                            <h3 scrollto-destination=87197664 id="87197664" class="section-title" ><em>Evidence of selective pressure for elevated intronic</em> TE <em>near the 5</em> ′ <em>SS and upstream from the first STOP codon presumably to decrease translation costs and improve</em> TE <em>of retained introns</em></h3>
<p> It was suggested that selection for certain synonymous codons improves TE (in terms of rate and fidelity), reduces the cost of the translation process (e.g. via improving ribosomal allocation), and affects organismal fitness. <a reveal-id="dsw019-B77 dsw019-B78 dsw019-B79 dsw019-B80 dsw019-B81" data-open="dsw019-B77 dsw019-B78 dsw019-B79 dsw019-B80 dsw019-B81" class="link link-ref link-reveal xref-bibr"><sup>77–81</sup></a> Thus, a preference for codons that are translated in a more efficient manner may possibly reduce translational costs of retained introns, even if their product is not functional or entirely functional. Therefore, if a relatively large fraction of the intronome is occasionally retained, we can expect to see preference for codons with translation rates or TE relatively similar to typical ORFs in the intronic region upstream from the first intronic STOP codon. <a reveal-id="dsw019-B82" data-open="dsw019-B82" class="link link-ref link-reveal xref-bibr"><sup>82</sup></a><sup>,</sup><a reveal-id="dsw019-B83" data-open="dsw019-B83" class="link link-ref link-reveal xref-bibr"><sup>83</sup></a> Hence, and in order to provide evidence supporting this hypothesis, we analysed two measures that estimate the TE or speed of codons: the first is the tAI of a gene (or other genomic sequence), which estimates its adaptation to the tRNA pool. <a reveal-id="dsw019-B84" data-open="dsw019-B84" class="link link-ref link-reveal xref-bibr"><sup>84</sup></a> In the case of <em>S. cerevisiae</em> , we used an additional measure: the TDR (see Materials and methods ), which is expected to be more accurate since it incorporates direct experimental RP data; <a reveal-id="dsw019-B63" data-open="dsw019-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a><sup>,</sup><a reveal-id="dsw019-B64" data-open="dsw019-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> TDR is currently not available for the other studied organisms. To this end, the tAI/TDR profiles of all intron-containing genes were computed both in the exonic and the intronic regions (see details in Materials and methods  and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Fig. S3</a></span> ). The profiles were aligned around: (i) the 5′SS of the introns and (ii) the first intronic STOP codon, and a mean profile was generated. A summary of the analysis for <em>S. cerevisiae</em> is presented in <a reveal-id="dsw019-F3" data-open="dsw019-F3" class="link link-reveal link-table xref-fig">Fig. 3</a> . As anticipated, the average TDR values in the exonic regions upstream of the 5′SS were higher than in the intronic regions downstream from it ( <a reveal-id="dsw019-F3" data-open="dsw019-F3" class="link link-reveal link-table xref-fig">Fig. 3A and B ;</a><em>P</em>  = 2.04·10 <sup>−</sup><sup>8</sup> ; Wilcoxon signed-rank test between the upstream and downstream values per gene). Furthermore, the average TDR values of the intronic sequences tend to be higher before the first intronic STOP codon relatively to the intronic sequence downstream from it ( <em>P</em>  = 4.91·10  <sup>−</sup>   <sup>18</sup> , Wilcoxon signed-rank test between the upstream and downstream values per gene). Similarly, when the mean intronic TDR before the first intronic STOP codon was compared with the randomized models it was found to be significantly higher ( <a reveal-id="dsw019-F3" data-open="dsw019-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a> ). Finally, comparison to randomized models shows that the average Z-score value in the <strong><em>pre-STOP</em></strong> domain is significant, while not significant in the <strong><em>post-STOP</em></strong> domain ( <a reveal-id="dsw019-F3" data-open="dsw019-F3" class="link link-reveal link-table xref-fig">Fig. 3D ;</a> 4.013 <em>vs.</em> −0.078, respectively), supporting the suggested hypothesis that the region before the first intronic STOP codon undergoes evolutionary selection for higher TE (see details on Z-score calculation in Materials and methods). Similar results were obtained in <em>S. pombe</em> , <em>A. nidulans</em> , and <em>C. albicans</em> (using tAI; see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Figs S4–S6</a></span> , respectively). A summary of the average TDR/tAI and corresponding Z-scores shown in <a reveal-id="dsw019-F4" data-open="dsw019-F4" class="link link-reveal link-table xref-fig">Fig. 4A and B</a> demonstrates that in all studied organisms the <strong><em>pre-STOP</em></strong> domain exhibits higher translation adaptation levels in comparison to the <strong><em>post-STOP</em></strong> domain; full analysis and statistical information, including average random values and empirical <em>P</em> -value information, can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Table S1</a></span> . Similar results were obtained when using randomized models that separately maintained GC content in the first and second intronic fragments ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Fig. S7</a></span> ); profiles aligned around the 3′SS can be seen in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Fig. S13</a></span> . </p>                        <a id="87197668" scrollto-destination="87197668"></a>
<div id="dsw019-F4" data-id="dsw019-F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/m_dsw019f4p.png?Expires=2147483647&Signature=DxcW1NvI0UFjrd-b~TrZzz0R0Vdc2BOZi2R4LnTV1TuDlxt5e~mSSg8IXmxixSNv3H-ciO0rng3wQwfHuvJ0YZRn97pMy8vudZmuAqmgs7lOTkZ4wjvhdHLSl5f9lUKpZWbg71InAWCaLFwTgskvjhrZmTIqRy9hb5r0J6mSFvHGvBclwwiPpzyrASkpjvWQbPRu0o4jUJLqwbEz28D0YwDJAYn2SqBgCyuzNyNJDdGx9qwGxhnU-0aa4OkP4AKEgxQq4RKX6s6diJu1rY-Uy~toFMhbxMUab1teIs6iXfp9YCNvaGDK5YXkqO1F-m1SfGzaif3XqUVktzDZpvyTNg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" CUB and translation efficiency summary. (A) Average TDR ( S. cerevisiae ) and tAI ( S. pombe , A. nidulans , and C . albicans ) in the Exonic Donor, pre-STOP and post-STOP domains of the examined organisms exhibit monotonic decrease in CUB over these three domains. (B) In all of the examined organisms the global Z-score at the pre-STOP domain is &gt; 2 whereas the Z-score post-STOP domain is negative, supporting the hypothesis that there is selection for codons similar to some typical ORFs in the pre-STOP but not in the post-STOP domain, probably to optimize translation. (C) Distribution analysis of the S. cerevisiae whole genome demonstrate that the average TDR at the beginning of its introns is higher than in 23.87% of the genes (average value of 5.1); downstream from the first intronic STOP codon the values drop to be higher than only 0.07% of the genes (average value of 4.77); in comparison, average values of the top 300 highly expressed genes is 6.17. (D) Distribution analysis of the S. pombe whole genome demonstrate that the average tAI at the beginning of its introns is higher than in 31.02% of the genes (average value of 0.382); downstream from the first intronic STOP codon the values drop to be higher than only 0.1% of the genes (average value of 0.313); in comparison, the average value of the top 300 highly expressed genes is 0.542. " path-from-xml="dsw019f4p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197668/dsw019f4p.png" path-from-xml="dsw019f4p.png" target="_blank">View large</a><a section="87197668" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019f4p.png?Expires=2147483647&Signature=l-l-OHAugkSmhi0yLT47Xl4asJACe5dhsqq5UJGACIj-3Yv5qkg30a0HEhLgvjQfgABF1WG9ZO4ed~IkjjvHN7BJeZfSn95uSUthYo3UxIm4uYpbvGFZJP1Mf~-PrSSfGtMcPpeyF3EhssqiE3LtFP6w66YL9WAow41NLGIWpDIAF6r5CjWoFciz7wQFF5-FYsE0v9-oKE0fRG58cgUCmAsEIFkal35ignskta9Ygd1YYf5JGiiXLfk7mh28TirIL1JA9xQRYnB34JPvM-7c6Q4Y8qh~YwbNZWgrtJybakptMYVsxoi04vh6M-ig7EUibglh7N3dSRCgzGbAADYK4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197668&ar=2469962&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw019f4p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> CUB and translation efficiency summary. (A) Average TDR ( <em>S. cerevisiae</em> ) and tAI ( <em>S. pombe</em> , <em>A. nidulans</em> , and C <em>. albicans</em> ) in the Exonic Donor, pre-STOP and post-STOP domains of the examined organisms exhibit monotonic decrease in CUB over these three domains. (B) In all of the examined organisms the global Z-score at the pre-STOP domain is &gt; 2 whereas the Z-score post-STOP domain is negative, supporting the hypothesis that there is selection for codons similar to some typical ORFs in the pre-STOP but not in the post-STOP domain, probably to optimize translation. (C) Distribution analysis of the <em>S. cerevisiae</em> whole genome demonstrate that the average TDR at the beginning of its introns is higher than in 23.87% of the genes (average value of 5.1); downstream from the first intronic STOP codon the values drop to be higher than only 0.07% of the genes (average value of 4.77); in comparison, average values of the top 300 highly expressed genes is 6.17. (D) Distribution analysis of the <em>S. pombe</em> whole genome demonstrate that the average tAI at the beginning of its introns is higher than in 31.02% of the genes (average value of 0.382); downstream from the first intronic STOP codon the values drop to be higher than only 0.1% of the genes (average value of 0.313); in comparison, the average value of the top 300 highly expressed genes is 0.542. </p></div></div><div content-id="dsw019-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/m_dsw019f4p.png?Expires=2147483647&Signature=DxcW1NvI0UFjrd-b~TrZzz0R0Vdc2BOZi2R4LnTV1TuDlxt5e~mSSg8IXmxixSNv3H-ciO0rng3wQwfHuvJ0YZRn97pMy8vudZmuAqmgs7lOTkZ4wjvhdHLSl5f9lUKpZWbg71InAWCaLFwTgskvjhrZmTIqRy9hb5r0J6mSFvHGvBclwwiPpzyrASkpjvWQbPRu0o4jUJLqwbEz28D0YwDJAYn2SqBgCyuzNyNJDdGx9qwGxhnU-0aa4OkP4AKEgxQq4RKX6s6diJu1rY-Uy~toFMhbxMUab1teIs6iXfp9YCNvaGDK5YXkqO1F-m1SfGzaif3XqUVktzDZpvyTNg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" CUB and translation efficiency summary. (A) Average TDR ( S. cerevisiae ) and tAI ( S. pombe , A. nidulans , and C . albicans ) in the Exonic Donor, pre-STOP and post-STOP domains of the examined organisms exhibit monotonic decrease in CUB over these three domains. (B) In all of the examined organisms the global Z-score at the pre-STOP domain is &gt; 2 whereas the Z-score post-STOP domain is negative, supporting the hypothesis that there is selection for codons similar to some typical ORFs in the pre-STOP but not in the post-STOP domain, probably to optimize translation. (C) Distribution analysis of the S. cerevisiae whole genome demonstrate that the average TDR at the beginning of its introns is higher than in 23.87% of the genes (average value of 5.1); downstream from the first intronic STOP codon the values drop to be higher than only 0.07% of the genes (average value of 4.77); in comparison, average values of the top 300 highly expressed genes is 6.17. (D) Distribution analysis of the S. pombe whole genome demonstrate that the average tAI at the beginning of its introns is higher than in 31.02% of the genes (average value of 0.382); downstream from the first intronic STOP codon the values drop to be higher than only 0.1% of the genes (average value of 0.313); in comparison, the average value of the top 300 highly expressed genes is 0.542. " path-from-xml="dsw019f4p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197668/dsw019f4p.png" path-from-xml="dsw019f4p.png" target="_blank">View large</a><a section="87197668" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019f4p.png?Expires=2147483647&Signature=l-l-OHAugkSmhi0yLT47Xl4asJACe5dhsqq5UJGACIj-3Yv5qkg30a0HEhLgvjQfgABF1WG9ZO4ed~IkjjvHN7BJeZfSn95uSUthYo3UxIm4uYpbvGFZJP1Mf~-PrSSfGtMcPpeyF3EhssqiE3LtFP6w66YL9WAow41NLGIWpDIAF6r5CjWoFciz7wQFF5-FYsE0v9-oKE0fRG58cgUCmAsEIFkal35ignskta9Ygd1YYf5JGiiXLfk7mh28TirIL1JA9xQRYnB34JPvM-7c6Q4Y8qh~YwbNZWgrtJybakptMYVsxoi04vh6M-ig7EUibglh7N3dSRCgzGbAADYK4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197668&ar=2469962&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw019f4p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> CUB and translation efficiency summary. (A) Average TDR ( <em>S. cerevisiae</em> ) and tAI ( <em>S. pombe</em> , <em>A. nidulans</em> , and C <em>. albicans</em> ) in the Exonic Donor, pre-STOP and post-STOP domains of the examined organisms exhibit monotonic decrease in CUB over these three domains. (B) In all of the examined organisms the global Z-score at the pre-STOP domain is &gt; 2 whereas the Z-score post-STOP domain is negative, supporting the hypothesis that there is selection for codons similar to some typical ORFs in the pre-STOP but not in the post-STOP domain, probably to optimize translation. (C) Distribution analysis of the <em>S. cerevisiae</em> whole genome demonstrate that the average TDR at the beginning of its introns is higher than in 23.87% of the genes (average value of 5.1); downstream from the first intronic STOP codon the values drop to be higher than only 0.07% of the genes (average value of 4.77); in comparison, average values of the top 300 highly expressed genes is 6.17. (D) Distribution analysis of the <em>S. pombe</em> whole genome demonstrate that the average tAI at the beginning of its introns is higher than in 31.02% of the genes (average value of 0.382); downstream from the first intronic STOP codon the values drop to be higher than only 0.1% of the genes (average value of 0.313); in comparison, the average value of the top 300 highly expressed genes is 0.542. </p></div></div><p> We suggest that the beginning of introns tend to be translated frequently enough to trigger TDR/tAI preference resembling typical fungal ORFs. Specifically, when we compared the average TDR/tAI of the intronic upstream region to all of <em>S. cerevisiae</em> and <em>S. pombe</em> genes, we found that in these organisms 23.87% and 31.02% of the genes, respectively, have lower values, as can be seen in <a reveal-id="dsw019-F4" data-open="dsw019-F4" class="link link-reveal link-table xref-fig">Fig. 4C and D</a> . This result demonstrates that the TDR/tAI at the beginning of introns is similar to the TDR/tAI for typically expressed genes. </p>                            <h3 scrollto-destination=87197673 id="87197673" class="section-title" ><em>Evidence of higher selective pressure on</em> CUB <em>to improve</em> TE <em>at the beginning of introns for introns located in highly translated genes in S. cerevisiae</em></h3>
<p> In budding yeast, almost all introns were lost during evolution. <a reveal-id="dsw019-B85" data-open="dsw019-B85" class="link link-ref link-reveal xref-bibr"><sup>85</sup></a><sup>,</sup><a reveal-id="dsw019-B86" data-open="dsw019-B86" class="link link-ref link-reveal xref-bibr"><sup>86</sup></a> However, the ones that were preserved are known to be generally found in highly expressed genes; moreover, about a third of its intronome originates from genes encoding ribosomal proteins. <a reveal-id="dsw019-B37" data-open="dsw019-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> It is also known that highly expressed genes tend to undergo selection to include codons that are translated more efficiently; <a reveal-id="dsw019-B63" data-open="dsw019-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a><sup>,</sup><a reveal-id="dsw019-B64" data-open="dsw019-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a><sup>,</sup><a reveal-id="dsw019-B84" data-open="dsw019-B84" class="link link-ref link-reveal xref-bibr"><sup>84</sup></a><sup>,</sup><a reveal-id="dsw019-B87" data-open="dsw019-B87" class="link link-ref link-reveal xref-bibr"><sup>87</sup></a><sup>,</sup><a reveal-id="dsw019-B88" data-open="dsw019-B88" class="link link-ref link-reveal xref-bibr"><sup>88</sup></a> thus, we also expect higher selection pressure for ‘codons’ that are translated more efficiently at the 5′ end of introns in highly translated genes. Therefore, in order to determine whether there is a significant difference in the selection for TE between highly and lowly expressed genes at the beginning of introns, intron-containing genes in <em>S. cerevisiae</em> were divided into two subgroups: 93 introns originating from the very highly expressed ribosomal genes and 93 introns originating from non-ribosomal genes (arbitrarily selected out of 187 in total to control for the effect of different group sizes; see Materials and methods ), and TDR profiles were generated for each group. We further analysed the randomized model that combines randomizations of codons and introns for the aforementioned subgroups to identify evidence of selection based on Z-score profiles calculations. Moreover, we examined the corresponding TDR Z-score profiles of four additional subgroups of the <em>S. cerevisiae</em> intronome based on their PA, mRNA levels, RD measurements, <a reveal-id="dsw019-B60" data-open="dsw019-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> and the PPR, which is an estimation of the translation rate (271, 273, 279, and 266 introns, respectively; see Materials and methods ). This was accomplished via sorting the intronome according to each measure, and analysing its upper tertiary and lower tertiary, i.e. 85 introns in each subgroup. A summary of the TDR Z-score profiles shown in <a reveal-id="dsw019-F5" data-open="dsw019-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a> demonstrates that both ribosomal and non-ribosomal introns exhibit significant selective pressure in the pre-STOP domain (Z &gt; 3.5; see Materials and methods), but no substantial selection in the post-STOP domain. Likewise, introns with high PPR/PA/mRNA/RD exhibit higher and significant selective pressure in comparison to introns with low PPR/PA/mRNA/RD, in the pre-STOP domain, and no substantial selection in the post-STOP domain (|Z|&lt;1.34). This suggests a higher level of retained intron adaptation to the translation process in those subgroups (e.g. in the pre-STOP domain Z = 5.3 for high PPR <em>vs.</em> Z = −0.99 for low PPR; see Materials and methods; additional information can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Table S2</a></span> ). Furthermore, the selection signal is better related to PA rather than to mRNA levels (Z = 4.74 for high PA <em>vs.</em> Z = 3.79 for high mRNA): the expected intronic TE selection pressure is related to [ <em>mRNA concentration</em> • <em>translational initiation rate per mRNA</em> • <em>splicing error rate</em> ], whereas the protein level (PA) is related to [ <em>mRNA concentration</em> • <em>translational initiation rate per mRNA</em> ] and is expected to have higher correlation with selection levels. <a reveal-id="dsw019-B60" data-open="dsw019-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> Thus, the fact that the TE signal better correlate with PA rather than with mRNA supports the conjecture that indeed the selection for codons with high TDR at the 5′ end of introns is related to translation and not to pre-translation gene expression stages. </p>                        <a id="87197677" scrollto-destination="87197677"></a>
<div id="dsw019-F5" data-id="dsw019-F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/m_dsw019f5p.png?Expires=2147483647&amp;Signature=E1sJY-xvkeo1Sb5~FlyUntRW8sAAl6ExuSpECyya7fKTi411ZVPGUmgDJ8RAHSt7LR35fCzAvfApAZkcsDyN9jfyVXvfTiWoLH~I7NYm4lam4Hd89IjDoCgEgvV32IvesTibDq4wB-F-oCzzyDhoj3GC2hbu74DjfPP48g6lS5PYgEE7JZQSWhhcaBkwAtlZ-Jvhe4R0S76aUhfSRqGNKg88ZUjkkOmghzLpikPmLvmq~7vmXDzgqD-h7XipIVb3~~8d6m43rMDvCwunf30C8HvWp6WgV6jnJ8QS-LrPS9JWA72cGkwurrq~JZjKbxnodnWEuHUu7TCUWMzsOBXjCA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" CUB analysis in various subgroups of S. cerevisiae . (A) Average TDR Z-scores for various intronome subgroups examined in S. cerevisiae : ribosomal vs . non-ribosomal, high levels of PPR vs . low levels of PPR, high PA vs . low PA, high mRNA levels vs . low mRNA levels, and high RD vs . low RD. 60 Introns originating from highly translated genes (e.g. high PPR/PA; Z &gt; 4.74) exhibit higher TDR Z-score values than introns originating from lowly translated genes (e.g. low PPR/PA; |Z|&lt;0.99) in the pre-STOP domain (see Materials and methods); results in the post-STOP domain were not significant (|Z|&lt;1.34); see full information in  Supplementary Table S2 . (B and C) Standardized reporter approach for studying SE in S. cerevisiae : (B) Overview of a previously reported approach for studying splicing mediated gene expression of a reported gene as an assessment of SE (see details in Materials and methods and Ref. 74 ). (C) Average TDR Z-scores for high/low levels of SE in the synthetic library: results were significant in the pre-STOP domain for highly spliced genes (Z = 3.49), but not significant in the post-STOP domain (|Z|&lt;0.74; see full information in  Supplementary Table S2 ). These results support the hypothesis that there is higher selective pressure for TE in introns originating from highly expressed/translated and highly spliced genes ( vs. introns originating from lowly expressed/translated and lowly spliced genes); thus, they are consistent with the accepted hypothesis that highly expressed genes are generally more adapted to the tRNA pool, the translation process, and to various gene expression steps in general. 62,81,87,88" path-from-xml="dsw019f5p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197677/dsw019f5p.png" path-from-xml="dsw019f5p.png" target="_blank">View large</a><a section="87197677" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019f5p.png?Expires=2147483647&amp;Signature=dJpOT5~zRZS2c8t~CwFt4MH0OsmstVI3Axv~TBARS9mPI9rfDNeb-hcRualHQnT-om2~wXQIgaCl2rbeO4Z-~I7JZumcXaxINoEn3lTL5o799WAIwBM1sf8gVhHwJQj-8NwhJSt1x6c-f8Zg6IWpmekKFfiMtDSCA1M~iWrvGBCiMMJdNxm6zS5gmirfghwJAzzHenh3X3vtan7xkOCNhVyuu78xwr4i1SGJaN7Vj-c7OiMz1AEs5ouGlcq6snBYvMghrbj6s8dvh2xv6KOaKzE8RjG9BfTzjXyOet1bsS3eddilIHZqMmqpbRhiiMkVZTeKiK1PsGv-BJdmpbgsjQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197677&amp;ar=2469962&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw019f5p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> CUB analysis in various subgroups of <em>S. cerevisiae</em> . (A) Average TDR Z-scores for various intronome subgroups examined in <em>S. cerevisiae</em> : ribosomal <em>vs</em> . non-ribosomal, high levels of PPR <em>vs</em> . low levels of PPR, high PA <em>vs</em> . low PA, high mRNA levels <em>vs</em> . low mRNA levels, and high RD <em>vs</em> . low RD. <a reveal-id="dsw019-B60" data-open="dsw019-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> Introns originating from highly translated genes (e.g. high PPR/PA; Z &gt; 4.74) exhibit higher TDR Z-score values than introns originating from lowly translated genes (e.g. low PPR/PA; |Z|&lt;0.99) in the pre-STOP domain (see Materials and methods); results in the post-STOP domain were not significant (|Z|&lt;1.34); see full information in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&amp;Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Table S2</a></span> . (B and C) Standardized reporter approach for studying SE in <em>S. cerevisiae</em> : (B) Overview of a previously reported approach for studying splicing mediated gene expression of a reported gene as an assessment of SE (see details in Materials and methods and Ref. <a reveal-id="dsw019-B74" data-open="dsw019-B74" class="link link-ref link-reveal xref-bibr">74</a> ). (C) Average TDR Z-scores for high/low levels of SE in the synthetic library: results were significant in the pre-STOP domain for highly spliced genes (Z = 3.49), but not significant in the post-STOP domain (|Z|&lt;0.74; see full information in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&amp;Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Table S2</a></span> ). These results support the hypothesis that there is higher selective pressure for TE in introns originating from highly expressed/translated and highly spliced genes ( <em>vs.</em> introns originating from lowly expressed/translated and lowly spliced genes); thus, they are consistent with the accepted hypothesis that highly expressed genes are generally more adapted to the tRNA pool, the translation process, and to various gene expression steps in general. <a reveal-id="dsw019-B62" data-open="dsw019-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a><sup>,</sup><a reveal-id="dsw019-B81" data-open="dsw019-B81" class="link link-ref link-reveal xref-bibr"><sup>81</sup></a><sup>,</sup><a reveal-id="dsw019-B87" data-open="dsw019-B87" class="link link-ref link-reveal xref-bibr"><sup>87</sup></a><sup>,</sup><a reveal-id="dsw019-B88" data-open="dsw019-B88" class="link link-ref link-reveal xref-bibr"><sup>88</sup></a></p></div></div><div content-id="dsw019-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/m_dsw019f5p.png?Expires=2147483647&amp;Signature=E1sJY-xvkeo1Sb5~FlyUntRW8sAAl6ExuSpECyya7fKTi411ZVPGUmgDJ8RAHSt7LR35fCzAvfApAZkcsDyN9jfyVXvfTiWoLH~I7NYm4lam4Hd89IjDoCgEgvV32IvesTibDq4wB-F-oCzzyDhoj3GC2hbu74DjfPP48g6lS5PYgEE7JZQSWhhcaBkwAtlZ-Jvhe4R0S76aUhfSRqGNKg88ZUjkkOmghzLpikPmLvmq~7vmXDzgqD-h7XipIVb3~~8d6m43rMDvCwunf30C8HvWp6WgV6jnJ8QS-LrPS9JWA72cGkwurrq~JZjKbxnodnWEuHUu7TCUWMzsOBXjCA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" CUB analysis in various subgroups of S. cerevisiae . (A) Average TDR Z-scores for various intronome subgroups examined in S. cerevisiae : ribosomal vs . non-ribosomal, high levels of PPR vs . low levels of PPR, high PA vs . low PA, high mRNA levels vs . low mRNA levels, and high RD vs . low RD. 60 Introns originating from highly translated genes (e.g. high PPR/PA; Z &gt; 4.74) exhibit higher TDR Z-score values than introns originating from lowly translated genes (e.g. low PPR/PA; |Z|&lt;0.99) in the pre-STOP domain (see Materials and methods); results in the post-STOP domain were not significant (|Z|&lt;1.34); see full information in  Supplementary Table S2 . (B and C) Standardized reporter approach for studying SE in S. cerevisiae : (B) Overview of a previously reported approach for studying splicing mediated gene expression of a reported gene as an assessment of SE (see details in Materials and methods and Ref. 74 ). (C) Average TDR Z-scores for high/low levels of SE in the synthetic library: results were significant in the pre-STOP domain for highly spliced genes (Z = 3.49), but not significant in the post-STOP domain (|Z|&lt;0.74; see full information in  Supplementary Table S2 ). These results support the hypothesis that there is higher selective pressure for TE in introns originating from highly expressed/translated and highly spliced genes ( vs. introns originating from lowly expressed/translated and lowly spliced genes); thus, they are consistent with the accepted hypothesis that highly expressed genes are generally more adapted to the tRNA pool, the translation process, and to various gene expression steps in general. 62,81,87,88" path-from-xml="dsw019f5p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197677/dsw019f5p.png" path-from-xml="dsw019f5p.png" target="_blank">View large</a><a section="87197677" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019f5p.png?Expires=2147483647&amp;Signature=dJpOT5~zRZS2c8t~CwFt4MH0OsmstVI3Axv~TBARS9mPI9rfDNeb-hcRualHQnT-om2~wXQIgaCl2rbeO4Z-~I7JZumcXaxINoEn3lTL5o799WAIwBM1sf8gVhHwJQj-8NwhJSt1x6c-f8Zg6IWpmekKFfiMtDSCA1M~iWrvGBCiMMJdNxm6zS5gmirfghwJAzzHenh3X3vtan7xkOCNhVyuu78xwr4i1SGJaN7Vj-c7OiMz1AEs5ouGlcq6snBYvMghrbj6s8dvh2xv6KOaKzE8RjG9BfTzjXyOet1bsS3eddilIHZqMmqpbRhiiMkVZTeKiK1PsGv-BJdmpbgsjQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197677&amp;ar=2469962&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw019f5p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> CUB analysis in various subgroups of <em>S. cerevisiae</em> . (A) Average TDR Z-scores for various intronome subgroups examined in <em>S. cerevisiae</em> : ribosomal <em>vs</em> . non-ribosomal, high levels of PPR <em>vs</em> . low levels of PPR, high PA <em>vs</em> . low PA, high mRNA levels <em>vs</em> . low mRNA levels, and high RD <em>vs</em> . low RD. <a reveal-id="dsw019-B60" data-open="dsw019-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> Introns originating from highly translated genes (e.g. high PPR/PA; Z &gt; 4.74) exhibit higher TDR Z-score values than introns originating from lowly translated genes (e.g. low PPR/PA; |Z|&lt;0.99) in the pre-STOP domain (see Materials and methods); results in the post-STOP domain were not significant (|Z|&lt;1.34); see full information in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&amp;Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Table S2</a></span> . (B and C) Standardized reporter approach for studying SE in <em>S. cerevisiae</em> : (B) Overview of a previously reported approach for studying splicing mediated gene expression of a reported gene as an assessment of SE (see details in Materials and methods and Ref. <a reveal-id="dsw019-B74" data-open="dsw019-B74" class="link link-ref link-reveal xref-bibr">74</a> ). (C) Average TDR Z-scores for high/low levels of SE in the synthetic library: results were significant in the pre-STOP domain for highly spliced genes (Z = 3.49), but not significant in the post-STOP domain (|Z|&lt;0.74; see full information in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&amp;Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Table S2</a></span> ). These results support the hypothesis that there is higher selective pressure for TE in introns originating from highly expressed/translated and highly spliced genes ( <em>vs.</em> introns originating from lowly expressed/translated and lowly spliced genes); thus, they are consistent with the accepted hypothesis that highly expressed genes are generally more adapted to the tRNA pool, the translation process, and to various gene expression steps in general. <a reveal-id="dsw019-B62" data-open="dsw019-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a><sup>,</sup><a reveal-id="dsw019-B81" data-open="dsw019-B81" class="link link-ref link-reveal xref-bibr"><sup>81</sup></a><sup>,</sup><a reveal-id="dsw019-B87" data-open="dsw019-B87" class="link link-ref link-reveal xref-bibr"><sup>87</sup></a><sup>,</sup><a reveal-id="dsw019-B88" data-open="dsw019-B88" class="link link-ref link-reveal xref-bibr"><sup>88</sup></a></p></div></div>                            <h3 scrollto-destination=87197681 id="87197681" class="section-title" ><em>The connection between elevated intronic</em> TE <em>near the 5</em> ′ <em>SS and upstream from the first STOP codon and SE measured via a synthetic intron library system</em></h3>
<p> The aim of the current section is to provide evidence supporting the conjecture that the reported signals at the intronic 5′ are associated with SE. To this end, we analysed previously reported measurements of different introns transformed into a Yellow Fluorescence intron Library (YiFP) as a system for estimating SE in <em>S. cerevisiae</em> (Ref. <a reveal-id="dsw019-B74" data-open="dsw019-B74" class="link link-ref link-reveal xref-bibr">74</a> ; higher YiFP is related to higher SE; see <a reveal-id="dsw019-F5" data-open="dsw019-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a> , Materials and methods, and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Note S2</a></span> for additional details). First, we extracted the intronic SE library related measurements of 215 introns. Next, we looked at the TDR and TDR Z-score profiles of two subgroups of endogenous introns in correspondence to their YiFP expression levels: 80 genes with the highest levels and 80 genes with the lowest levels; each group accounts for ∼33% of the synthetic library intronome. As can be seen in <a reveal-id="dsw019-F5" data-open="dsw019-F5" class="link link-reveal link-table xref-fig">Fig. 5C</a> , in the pre-STOP domain there is evidence of significant relation between selection for codons with high TE (i.e. TDR) and high SE (Z = 3.49 for high YiFP <em>vs.</em> Z = −0.29 for low YiFP; additional information can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Table S2</a></span> ). This and the previous results demonstrate the coupling between the different gene expression steps and aspects: highly expressed genes are expected to undergo stronger selection for all gene expression aspects, including translation cost/efficiency and splicing cost/efficiency. Specifically, introns of highly expressed genes are expected to be more optimal in terms of their SE and their TE. </p>                            <h3 scrollto-destination=87197684 id="87197684" class="section-title" ><em>Evidence that selective pressure on</em> TE <em>near the 5</em> ′ <em>SS and upstream from the first STOP codon in S. pombe is not function specific</em></h3>
<p> An intriguing question is whether the previously shown intronic translation features are not related to the regulation of specific functional gene groups, and can be detected in various gene groups. To this end, we used GO and analysed the retained peptide length, tAI profiles, and tAI Z-scores mentioned in the previous sections in <em>S. pombe</em> for 90 different GO terms separately (see Materials and methods). As can be seen in <a reveal-id="dsw019-F6" data-open="dsw019-F6" class="link link-reveal link-table xref-fig">Fig. 6</a> for genes of various biological processes, analysis of the STOP codon distance from the 5′SS demonstrates that the average retained peptide length is lower in 61% of the gene functions for the actual intronome in comparison to the randomized ones ( <a reveal-id="dsw019-F6" data-open="dsw019-F6" class="link link-reveal link-table xref-fig">Fig . 6A ;</a> 79%/76% for molecular function and cellular component, respectively; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Fig. S9</a></span> ; 22% of the terms exhibit selection levels with empirical <em>P</em> -value <em>P</em>  &lt;  0.1; <a reveal-id="dsw019-F6" data-open="dsw019-F6" class="link link-reveal link-table xref-fig">Fig. 6B and C ;</a> 29% for both molecular function and cellular component, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Fig. S9</a></span> ). In addition, and consistent with previous results, the highest tAI values are found upstream from the 5′SS in most cases; naturally and as expected, the tAI values drop in the pre-STOP domain, however, these values are significant higher than the values in the post-STOP domain (which have the smallest tAI values). This result suggests a partial adaptation to the translation process at the pre-STOP domain and minimum or no adaptation to the translation process at the post-STOP domain. In addition, the Z-score profiles in the pre-STOP domain exhibit higher selection levels for most cases ( <a reveal-id="dsw019-F6" data-open="dsw019-F6" class="link link-reveal link-table xref-fig">Fig. 6D and E</a> ), supporting the conjecture above. Finally, 71%/61%/85% of the gene functions in biological process, molecular function, and cellular component, respectively, show significantly higher values in comparison of the pre-STOP and post-STOP domains ( <em>P</em>  &lt;  0.05 for all cases, Wilcoxon rank-sum test; results for molecular function and cellular component can be seen in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Fig. S9</a></span> ; see details in Materials and methods, and also a list of the included and excluded terms in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Table S3</a></span> ). Altogether, these results suggest that the signal reported in the previous sections is not function specific or limited to a small group of genes with specific function, but rather tends to appear in genes with various functions. </p>                        <a id="87197687" scrollto-destination="87197687"></a>
<div id="dsw019-F6" data-id="dsw019-F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/m_dsw019f6p.png?Expires=2147483647&Signature=pTh3r1V-1MRkJMNUD5t6U4O-DEkmr7zqaU3yLpdzYcl491UVqy4VL766auABrghg0CmXHmVO8rvHUOAwUm4bYIuqFSRn10W3xPJirqrb2Giju0jC2aggvRqywbzUGqa1h5Skn~JEL~SQqaYiFvNdbf~agiEn~amGgJNJyOv8JvsrtkQCyn9NmFMte7x51zs5ndjsAvT-G5qwe9oL1QKT0PhmmPLpmO5jKBfD~eNGeiB38bxZ9~b2wQCCZBFp1bPmxIXuoZKnCPs5bTc3x59KkVX9-zaZb1uZX6piwo3OHQuxyraZmlnbAl2Jtq3Sf-39VLp4kJB~gxVLv1mrqtehvQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" GO terms analysis in S. pombe reveals that retained introns translation preference is not unique to genes with a very specific function. (A--–C) Summary of retained peptide length (A) and Z-score corresponding to peptide length relatively to randomized models (B) for biological process demonstrates that in 61% of the gene groups there is preference for a STOP codon closer to the 5′SS. (C) Statistical analysis indicates that the empirical P -value related to the observed signal is &lt;0.1 in 22% of the examined gene groups (see Materials and methods). (D and E) Summary of average tAI (D) and Z-score corresponding to tAI levels relatively to randomized models (E; in absolute values) profiles in the pre-mRNA domains for biological process demonstrates that in various gene groups there is preference for higher tAI values at the beginning of introns in comparison to the region downstream from the first intronic STOP codon; statistical analysis indicates that the observed signal is significant in 71% of the examined gene groups ( P  &lt;  0.05 for all cases, Wilcoxon rank-sum test; see Materials and methods). Thus, the reported signal is universal and not related to any one functional gene group. " path-from-xml="dsw019f6p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197687/dsw019f6p.png" path-from-xml="dsw019f6p.png" target="_blank">View large</a><a section="87197687" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019f6p.png?Expires=2147483647&Signature=SCaBqYknowU3mYrfM~c0~Xf1WXEmYeDkd5k2Djhs2f8dLFlkx2Pci1gnPJ0pnnhabkuCIH~6-5pcM4TDciR4s5NBYippaYgvZDGPStjiOo2L8gzygkQLO2mxMOl2-GtynDGiyjTkgrcqIfCVVSNc8SicYHugi28rKvXMNouXbhQksoKvJ1oTm0lSr5gDCuMMczPNlvhJGOAOJkbDahdrcmp3FRATxISDFzngxvVyObVLn0JiQ2tcoXeP01C0PQ69XMH-5Rb4meRqfcXjw4YzMp2QtFc5S74VgNTBSYwmgmpZqNzI9-YkqXPDp50nR2pWXmKotOlHeX5OiKJ0qFG5uA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197687&ar=2469962&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw019f6p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> GO terms analysis in <em>S. pombe</em> reveals that retained introns translation preference is not unique to genes with a very specific function. (A--–C) Summary of retained peptide length (A) and Z-score corresponding to peptide length relatively to randomized models (B) for biological process demonstrates that in 61% of the gene groups there is preference for a STOP codon closer to the 5′SS. (C) Statistical analysis indicates that the empirical <em>P</em> -value related to the observed signal is &lt;0.1 in 22% of the examined gene groups (see Materials and methods). (D and E) Summary of average tAI (D) and Z-score corresponding to tAI levels relatively to randomized models (E; in absolute values) profiles in the pre-mRNA domains for biological process demonstrates that in various gene groups there is preference for higher tAI values at the beginning of introns in comparison to the region downstream from the first intronic STOP codon; statistical analysis indicates that the observed signal is significant in 71% of the examined gene groups ( <em>P</em>  &lt;  0.05 for all cases, Wilcoxon rank-sum test; see Materials and methods). Thus, the reported signal is universal and not related to any one functional gene group. </p></div></div><div content-id="dsw019-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/m_dsw019f6p.png?Expires=2147483647&Signature=pTh3r1V-1MRkJMNUD5t6U4O-DEkmr7zqaU3yLpdzYcl491UVqy4VL766auABrghg0CmXHmVO8rvHUOAwUm4bYIuqFSRn10W3xPJirqrb2Giju0jC2aggvRqywbzUGqa1h5Skn~JEL~SQqaYiFvNdbf~agiEn~amGgJNJyOv8JvsrtkQCyn9NmFMte7x51zs5ndjsAvT-G5qwe9oL1QKT0PhmmPLpmO5jKBfD~eNGeiB38bxZ9~b2wQCCZBFp1bPmxIXuoZKnCPs5bTc3x59KkVX9-zaZb1uZX6piwo3OHQuxyraZmlnbAl2Jtq3Sf-39VLp4kJB~gxVLv1mrqtehvQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" GO terms analysis in S. pombe reveals that retained introns translation preference is not unique to genes with a very specific function. (A--–C) Summary of retained peptide length (A) and Z-score corresponding to peptide length relatively to randomized models (B) for biological process demonstrates that in 61% of the gene groups there is preference for a STOP codon closer to the 5′SS. (C) Statistical analysis indicates that the empirical P -value related to the observed signal is &lt;0.1 in 22% of the examined gene groups (see Materials and methods). (D and E) Summary of average tAI (D) and Z-score corresponding to tAI levels relatively to randomized models (E; in absolute values) profiles in the pre-mRNA domains for biological process demonstrates that in various gene groups there is preference for higher tAI values at the beginning of introns in comparison to the region downstream from the first intronic STOP codon; statistical analysis indicates that the observed signal is significant in 71% of the examined gene groups ( P  &lt;  0.05 for all cases, Wilcoxon rank-sum test; see Materials and methods). Thus, the reported signal is universal and not related to any one functional gene group. " path-from-xml="dsw019f6p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197687/dsw019f6p.png" path-from-xml="dsw019f6p.png" target="_blank">View large</a><a section="87197687" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019f6p.png?Expires=2147483647&Signature=SCaBqYknowU3mYrfM~c0~Xf1WXEmYeDkd5k2Djhs2f8dLFlkx2Pci1gnPJ0pnnhabkuCIH~6-5pcM4TDciR4s5NBYippaYgvZDGPStjiOo2L8gzygkQLO2mxMOl2-GtynDGiyjTkgrcqIfCVVSNc8SicYHugi28rKvXMNouXbhQksoKvJ1oTm0lSr5gDCuMMczPNlvhJGOAOJkbDahdrcmp3FRATxISDFzngxvVyObVLn0JiQ2tcoXeP01C0PQ69XMH-5Rb4meRqfcXjw4YzMp2QtFc5S74VgNTBSYwmgmpZqNzI9-YkqXPDp50nR2pWXmKotOlHeX5OiKJ0qFG5uA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197687&ar=2469962&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw019f6p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> GO terms analysis in <em>S. pombe</em> reveals that retained introns translation preference is not unique to genes with a very specific function. (A--–C) Summary of retained peptide length (A) and Z-score corresponding to peptide length relatively to randomized models (B) for biological process demonstrates that in 61% of the gene groups there is preference for a STOP codon closer to the 5′SS. (C) Statistical analysis indicates that the empirical <em>P</em> -value related to the observed signal is &lt;0.1 in 22% of the examined gene groups (see Materials and methods). (D and E) Summary of average tAI (D) and Z-score corresponding to tAI levels relatively to randomized models (E; in absolute values) profiles in the pre-mRNA domains for biological process demonstrates that in various gene groups there is preference for higher tAI values at the beginning of introns in comparison to the region downstream from the first intronic STOP codon; statistical analysis indicates that the observed signal is significant in 71% of the examined gene groups ( <em>P</em>  &lt;  0.05 for all cases, Wilcoxon rank-sum test; see Materials and methods). Thus, the reported signal is universal and not related to any one functional gene group. </p></div></div>                            <h3 scrollto-destination=87197693 id="87197693" class="section-title" >RP <em>analysis in budding yeast demonstrates higher ribosome density at the beginning of introns compared to downstream from the first intronic STOP codon</em></h3>
<p> To further demonstrate that the beginning of introns tends to be partially but considerably translated, we utilized <em>S. cerevisiae</em> RP experimental information. The RP method is currently one of the most promising approaches for studying gene translation; <a reveal-id="dsw019-B60" data-open="dsw019-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> it provides a quantitative measure of ribosome translation status for every nucleotide in the genome at any given moment (see details in <a reveal-id="dsw019-F7" data-open="dsw019-F7" class="link link-reveal link-table xref-fig">Fig. 7A–D</a> and in the Materials and methods). In order to estimate the translation levels of retained introns we compared the RD before and after the first intronic STOP codon. Specifically, the analysis was performed based on the RP footprint RC information in these regions in a per nucleotide resolution, through alignments around the 5′SS and first intronic STOP codon, averaging over the entire intronome (see Materials and methods and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Fig. S10</a></span> ). As expected, the mean RC values in the exons upstream of the 5′SS are around two orders of magnitude higher than the values at the intronic downstream side ( <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Fig. S11</a></span> ). Analysis of the region surrounding the first intronic STOP codon shows that mean RC values upstream of the termination point were 2.56 times significantly higher than those downstream from it ( <a reveal-id="dsw019-F7" data-open="dsw019-F7" class="link link-reveal link-table xref-fig">Fig. 7E ;</a> mean RC of 6.39·10 <sup>−</sup><sup>2</sup> before <em>vs.</em> 2.54·10 <sup>−</sup><sup>2</sup> after the first intronic STOP codon; <em>P</em>  = 1.42·10 <sup>−</sup><sup>9</sup> , Wilcoxon signed-rank test between the upstream and downstream values per gene; see also <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Table S1</a></span> ). Interestingly, analysis of the mean RC in the 3′ UTR (3.47·10 <sup>−</sup><sup>2</sup> ; after the annotated STOP and the end of the coding region), found similar levels to the ones detected in the intronic post-STOP domain (2.54·10 <sup>−</sup><sup>2</sup> ). This result supports the conjecture that the intronic STOP codons function in a similar manner as the annotated STOP, in the case where introns are properly spliced; thus, the level of RC detected at the intronic post-STOP domain is similar to the ‘background noise’ of the RP method. Furthermore, examination of RD in alignment to the second intronic STOP codon does not show any significant difference ( <a reveal-id="dsw019-F7" data-open="dsw019-F7" class="link link-reveal link-table xref-fig">Fig. 7F ;</a> mean RC of 2.47·10 <sup>−</sup><sup>2</sup> before <em>vs.</em> 2.35·10 <sup>−</sup><sup>2</sup> after the second intronic STOP codon; <em>P</em>  = 0.21, Wilcoxon signed-rank test between the upstream and downstream values per gene; see Materials and methods), further supporting the conjecture that indeed the observed signal near the first intronic STOP codon is related to genuine intronic translation. </p>                        <a id="87197699" scrollto-destination="87197699"></a>
<div id="dsw019-F7" data-id="dsw019-F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/m_dsw019f7p.png?Expires=2147483647&Signature=30i-t81CjwTZlzRcFl9kQgvMe1o39YkDcIp5S7jekreeKE34Wlx5uc91WaOA5L~nOP7Cf3mYbthm8pwp45h4is~J6g7KPCfAhwJGAUIsVy4~2xnJNaoXUrCHeJ7fQBxM-GGtLe0WSGvxvm2TrBB0wtv5cBJKnfCPEXlFqweLZCj7UJRVin97osai9ROOCwJQQObcwHd-q4~OPe0-Fi38s34xrcDbhUICzSIzcD6Key1IfntW3QRp~GGRuNYZdcGpLooMefr4z17zbqHT3OOnAvYHdPZD77neOJn5u8WUOI9YVIklDE6fbmUwKi8EJXpSpXrXCE4W4-MY18lQID6Qfg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" RP analysis of S. cerevisiae . The RP method gives quantitative information of RD in a single nucleotide resolution. (A) Cells are treated (for example) with cycloheximide to arrest translating ribosomes; (B and C) RNA fragments protected from RNases by the ribosome are isolated and processed for Illumina high-throughput sequencing; (D) Based on the sequence reads, it is possible to computationally infer various biophysical properties related to the translation-elongation process. Ribosomal footprint RCs of a certain codon are generated when the codon is covered by ribosomes. Thus, highly translated genes tend to create a higher number of reads. (E) The mean RC values upstream from the first intronic STOP codon (pre-STOP side, left of the Termination Point) shows higher values compared with downstream mean RC values (post-STOP side, right of the Termination Point; 6.39 × 10 −2 before vs. 2.54 × 10 −2 after the first intronic STOP codon; P  = 1.42 × 10 −9 , Wilcoxon signed-rank test). (F) The mean RC profile shows no significant difference between the intronic pre-STOP and post-STOP domains (2.47 × 10 −2 before vs. 2.35 × 10 −2 after second intronic STOP codon; P  = 0.21, Wilcoxon signed-rank test); RC A-site offset is 15 nt. (G) RC distribution analysis of the S. cerevisiae genome demonstrates that as much as 40.8% of the yeast genes have lower mean RC values than introns in the pre-STOP domain, while merely 15.77% of the genes have lower mean RC values than introns in the post-STOP domain (see details in Materials and methods). " path-from-xml="dsw019f7p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197699/dsw019f7p.png" path-from-xml="dsw019f7p.png" target="_blank">View large</a><a section="87197699" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019f7p.png?Expires=2147483647&Signature=lv0yRdOPlzz6ycvvFc47IE9cYfnRfU4kicdFmLE-4mzs3ZqyDNhdqWkGeYwSYT2ko6RC2cSIMlR28qjF226KC8ouQyajFTLb3-nMrtf9m8Zjq9pxxXsehUpKNUQn8AxL6e0BLlX3e9EoWH56sOj7wSgN606X~RB9P49y7pLkRIy9Vqkt3clY1taNZqEnX-klCwnV~nI4LH0n0WCATs9wNgA-kASTqwvEaI9vH~KbRpIND3BxaQ8tK4Jq-SUrdPruAdkstJ8EpAAkeIyME2ayb5Zg0qDRDa8gcZidTQGOuyzuj~UU1hQ0NYAXW~BRA9IT7KGRENVq~cJoPGs7tf89aQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197699&ar=2469962&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw019f7p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> RP analysis of <em>S. cerevisiae</em> . The RP method gives quantitative information of RD in a single nucleotide resolution. (A) Cells are treated (for example) with cycloheximide to arrest translating ribosomes; (B and C) RNA fragments protected from RNases by the ribosome are isolated and processed for Illumina high-throughput sequencing; (D) Based on the sequence reads, it is possible to computationally infer various biophysical properties related to the translation-elongation process. Ribosomal footprint RCs of a certain codon are generated when the codon is covered by ribosomes. Thus, highly translated genes tend to create a higher number of reads. (E) The mean RC values upstream from the first intronic STOP codon (pre-STOP side, left of the Termination Point) shows higher values compared with downstream mean RC values (post-STOP side, right of the Termination Point; 6.39 × 10 <sup>−2</sup> before <em>vs.</em> 2.54 × 10 <sup>−2</sup> after the first intronic STOP codon; <em>P</em>  = 1.42 × 10 <sup>−9</sup> , Wilcoxon signed-rank test). (F) The mean RC profile shows no significant difference between the intronic pre-STOP and post-STOP domains (2.47 × 10 <sup>−2</sup> before <em>vs.</em> 2.35 × 10 <sup>−2</sup> after second intronic STOP codon; <em>P</em>  = 0.21, Wilcoxon signed-rank test); RC A-site offset is 15 nt. (G) RC distribution analysis of the <em>S. cerevisiae</em> genome demonstrates that as much as 40.8% of the yeast genes have lower mean RC values than introns in the pre-STOP domain, while merely 15.77% of the genes have lower mean RC values than introns in the post-STOP domain (see details in Materials and methods). </p></div></div><div content-id="dsw019-F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/m_dsw019f7p.png?Expires=2147483647&Signature=30i-t81CjwTZlzRcFl9kQgvMe1o39YkDcIp5S7jekreeKE34Wlx5uc91WaOA5L~nOP7Cf3mYbthm8pwp45h4is~J6g7KPCfAhwJGAUIsVy4~2xnJNaoXUrCHeJ7fQBxM-GGtLe0WSGvxvm2TrBB0wtv5cBJKnfCPEXlFqweLZCj7UJRVin97osai9ROOCwJQQObcwHd-q4~OPe0-Fi38s34xrcDbhUICzSIzcD6Key1IfntW3QRp~GGRuNYZdcGpLooMefr4z17zbqHT3OOnAvYHdPZD77neOJn5u8WUOI9YVIklDE6fbmUwKi8EJXpSpXrXCE4W4-MY18lQID6Qfg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt=" RP analysis of S. cerevisiae . The RP method gives quantitative information of RD in a single nucleotide resolution. (A) Cells are treated (for example) with cycloheximide to arrest translating ribosomes; (B and C) RNA fragments protected from RNases by the ribosome are isolated and processed for Illumina high-throughput sequencing; (D) Based on the sequence reads, it is possible to computationally infer various biophysical properties related to the translation-elongation process. Ribosomal footprint RCs of a certain codon are generated when the codon is covered by ribosomes. Thus, highly translated genes tend to create a higher number of reads. (E) The mean RC values upstream from the first intronic STOP codon (pre-STOP side, left of the Termination Point) shows higher values compared with downstream mean RC values (post-STOP side, right of the Termination Point; 6.39 × 10 −2 before vs. 2.54 × 10 −2 after the first intronic STOP codon; P  = 1.42 × 10 −9 , Wilcoxon signed-rank test). (F) The mean RC profile shows no significant difference between the intronic pre-STOP and post-STOP domains (2.47 × 10 −2 before vs. 2.35 × 10 −2 after second intronic STOP codon; P  = 0.21, Wilcoxon signed-rank test); RC A-site offset is 15 nt. (G) RC distribution analysis of the S. cerevisiae genome demonstrates that as much as 40.8% of the yeast genes have lower mean RC values than introns in the pre-STOP domain, while merely 15.77% of the genes have lower mean RC values than introns in the post-STOP domain (see details in Materials and methods). " path-from-xml="dsw019f7p.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197699/dsw019f7p.png" path-from-xml="dsw019f7p.png" target="_blank">View large</a><a section="87197699" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019f7p.png?Expires=2147483647&Signature=lv0yRdOPlzz6ycvvFc47IE9cYfnRfU4kicdFmLE-4mzs3ZqyDNhdqWkGeYwSYT2ko6RC2cSIMlR28qjF226KC8ouQyajFTLb3-nMrtf9m8Zjq9pxxXsehUpKNUQn8AxL6e0BLlX3e9EoWH56sOj7wSgN606X~RB9P49y7pLkRIy9Vqkt3clY1taNZqEnX-klCwnV~nI4LH0n0WCATs9wNgA-kASTqwvEaI9vH~KbRpIND3BxaQ8tK4Jq-SUrdPruAdkstJ8EpAAkeIyME2ayb5Zg0qDRDa8gcZidTQGOuyzuj~UU1hQ0NYAXW~BRA9IT7KGRENVq~cJoPGs7tf89aQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197699&ar=2469962&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw019f7p.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p> RP analysis of <em>S. cerevisiae</em> . The RP method gives quantitative information of RD in a single nucleotide resolution. (A) Cells are treated (for example) with cycloheximide to arrest translating ribosomes; (B and C) RNA fragments protected from RNases by the ribosome are isolated and processed for Illumina high-throughput sequencing; (D) Based on the sequence reads, it is possible to computationally infer various biophysical properties related to the translation-elongation process. Ribosomal footprint RCs of a certain codon are generated when the codon is covered by ribosomes. Thus, highly translated genes tend to create a higher number of reads. (E) The mean RC values upstream from the first intronic STOP codon (pre-STOP side, left of the Termination Point) shows higher values compared with downstream mean RC values (post-STOP side, right of the Termination Point; 6.39 × 10 <sup>−2</sup> before <em>vs.</em> 2.54 × 10 <sup>−2</sup> after the first intronic STOP codon; <em>P</em>  = 1.42 × 10 <sup>−9</sup> , Wilcoxon signed-rank test). (F) The mean RC profile shows no significant difference between the intronic pre-STOP and post-STOP domains (2.47 × 10 <sup>−2</sup> before <em>vs.</em> 2.35 × 10 <sup>−2</sup> after second intronic STOP codon; <em>P</em>  = 0.21, Wilcoxon signed-rank test); RC A-site offset is 15 nt. (G) RC distribution analysis of the <em>S. cerevisiae</em> genome demonstrates that as much as 40.8% of the yeast genes have lower mean RC values than introns in the pre-STOP domain, while merely 15.77% of the genes have lower mean RC values than introns in the post-STOP domain (see details in Materials and methods). </p></div></div><p> Further, we evaluated the mean TDR at the pre-STOP domain assuming that due to splicing error rate consequently 1% of the introns are translated (e.g. see Ref. <a reveal-id="dsw019-B89" data-open="dsw019-B89" class="link link-ref link-reveal xref-bibr">89</a> ) To this end, we compared the mean TDR at the pre-STOP domain of the introns to the mean TDR of ORFs with RC that are [0.01 ± 0.005] <strong>·</strong> [mean RC of ORFs with introns] and found them to be very similar (5.165 <em>vs.</em> 5.148; in comparison the average and STD of the mean TDR over all annotated ORFs are 5.27 and 0.274, respectively). We also compared the mean TDR at the pre-STOP of the introns to the mean TDR of ORFs with similar RC (±10%), to the RC that appears in the pre-STOP domain and found them to be very similar (5.165 <em>vs.</em> 5.231); see also <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Note S2</a></span> . </p><p> Finally, as can be seen in <a reveal-id="dsw019-F7" data-open="dsw019-F7" class="link link-reveal link-table xref-fig">Fig. 7G</a> , analysis of the whole <em>S. cerevisiae</em> genome demonstrates that 40.8% of the genes have lower mean RC values than introns in the pre-STOP domain (the relatively high number can be explained by the fact that genes with introns are extremely highly expressed in <em>S. cerevisiae</em> ). These findings are in agreement with our previous results, as they support the conjecture that the intronic regions upstream of the first intronic STOP undergo significantly higher translation than the intronic regions downstream of it. In addition, the intronic translation rate is comparable to translation rates of known ORFs. Thus, these results support the conjecture that the reported intronic sequence features are indeed related to translation. </p>                            <h3 scrollto-destination=87197707 id="87197707" class="section-title" >The reported results from the population-genetics point of view</h3>
<p> The aim of this section is to demonstrate that the reported results are supported by population-genetics considerations. We will start with the conjecture that the 5′ end of introns undergo selection for ‘codons’ with better TE, and continue with the conjecture that there is selection for an intronic STOP codon in-frame with the upstream exon and close to the intron 5′ end. It was estimated in yeast <a reveal-id="dsw019-B78" data-open="dsw019-B78" class="link link-ref link-reveal xref-bibr"><sup>78</sup></a> that the CUB selective coefficient is <em>S</em>  = 2.6 for highly expressed genes (100 copies of the mRNA). This means that the expected frequency <em>P</em> of the optimal codon is: <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">P</mi><mo xmlns="">=</mo><msup xmlns=""><mrow><mfenced separators="|"><mrow><mn>1</mn><mo>+</mo><mfrac><mrow><mi>u</mi></mrow><mrow><mi>v</mi></mrow></mfrac><msup><mrow><mi>e</mi></mrow><mrow><mo>-</mo><mi>S</mi></mrow></msup></mrow></mfenced></mrow><mrow><mo>-</mo><mn>1</mn></mrow></msup></math></span> , where <em>u</em> and <em>v</em> are the mutation rates from <em> c <sub>1</sub></em> to <em> c <sub>2</sub></em> and from <em> c <sub>2</sub></em> to <em> c <sub>1</sub></em> , respectively ( <em> c <sub>1</sub></em> and <em> c <sub>2</sub></em> are two synonymous codons of the same AA). In addition, <em>S</em>  =  <em>4</em><strong>·</strong><em>Ne</em><strong>·</strong><em>s</em> , where <em>Ne</em> is the effective population size and <em>s</em> is the selective pressure. If we assume that <em>u </em> = <em> v</em> we get <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">P</mi><mo xmlns="">=</mo><msup xmlns=""><mrow><mfenced separators="|"><mrow><mn>1</mn><mo>+</mo><msup><mrow><mi>e</mi></mrow><mrow><mo>-</mo><mi>S</mi></mrow></msup></mrow></mfenced></mrow><mrow><mo>-</mo><mn>1</mn></mrow></msup></math></span>  = 0.93, and since in <em>S. cerevisiae Ne</em>  = 10 <sup>7</sup> (see in Ref. <a reveal-id="dsw019-B78" data-open="dsw019-B78" class="link link-ref link-reveal xref-bibr">78</a> ) we get a selective pressure of <em>s</em>  = 6.5·10 <sup>−</sup><sup>8</sup> for highly expressed genes. </p><p> In Ref. <a reveal-id="dsw019-B78" data-open="dsw019-B78" class="link link-ref link-reveal xref-bibr">78</a> they also provide a relation between expression levels and the selection coefficient (see <a reveal-id="dsw019-F2" data-open="dsw019-F2" class="link link-reveal link-table xref-fig">Fig. 2</a> in this article), and suggest that ‘genes expressed at over one transcript per cell show noticeable, albeit small, signs of selection on codon-usage’. Since ribosomal proteins have the highest expression levels in the cell (e.g. Y over 100 copies), we expect that an error rate of a few percentages <em>x</em> will be comparable to the selection coefficient for codons in genes with expression levels <em>x</em><strong>·</strong><em>y</em> . As can be seen in Ref. <a reveal-id="dsw019-B78" data-open="dsw019-B78" class="link link-ref link-reveal xref-bibr">78</a> , for <em>x</em> ∼ 5% (similarly to the RC observed at the intronic 5′ end of <em>S. cerevisiae</em> ) the estimated <em>S</em> is around 0.5, and thus the corresponding selective pressure is around <em>s</em>  = 1.25×10 <sup>−</sup><sup>8</sup> . According to them, this <em>S</em> value is close to the ones estimated for genes in <em>H</em><em>. Sapiens</em> , <em>Encephalitozoon cuniculi</em> , and <em>Plasmodium falciparum</em> , and is higher than the <em>S</em> value of <em>M</em><em>us musculus</em> , where selection for CUB was observed (i.e. the <em>S</em> value is significantly higher than 0 according to Ref. <a reveal-id="dsw019-B78" data-open="dsw019-B78" class="link link-ref link-reveal xref-bibr">78</a> ). </p><p> Regarding the second signal, the conclusion is similar: The selection coefficient for a close STOP codon is expected to be higher than for CUB due to the following reasons: (i) Based on TDR/tAI/Metabolic-cost in <em>S. cerevisiae</em> the mean TE or metabolic cost difference between all pairs of (different) codons is 1.4/0.299/19.78, respectively, while the mean improvement in terms of translation rate/metabolic cost when removing a codon is expected to be equal to the mean translation rate/metabolic cost of one codon, which is 5.22/0.387/29.08, respectively. (ii) Mutations that decrease the distance to a STOP codon can decrease it by more than one codon. </p>                            <h3 scrollto-destination=87197713 id="87197713" class="section-title" >The relation of the reported results to efficient NMD regulation</h3>
<p> The fact that the reported signals correspond to a STOP codon close to the intronic 5′SS may also be (partially) related to selection for improved NMD. Specifically, recent studies suggested that longer 3′ UTRs following PTC is one factor related to the improvement of NMD efficiency. <a reveal-id="dsw019-B90 dsw019-B91 dsw019-B92 dsw019-B93" data-open="dsw019-B90 dsw019-B91 dsw019-B92 dsw019-B93" class="link link-ref link-reveal xref-bibr"><sup>90–93</sup></a> Nevertheless, various lines of evidence support the conjecture that the reported result is at least partially related to translation rather than NMD: </p><p>(i) According to our analyses of reduced metabolic costs and TE adaption in the beginning of introns, the signal of more ‘efficient’ amino acids and codons before the first intronic STOP is expected to be related to translation and is not expected to be related to NMD.</p><p> (ii) The average distance between the PTC and the annotated STOP codons is 934.41 nt for <em>S. cerevisiae</em> and 932.36 for <em>Sch. pombe</em> . These results indeed detect a significant signal related to longer 3′ UTRs that may be related to improved NMD ( <em>P</em>  = 8.55×10 <sup>−</sup><sup>8</sup> and <em>P</em>  = 1.25×10 <sup>−</sup><sup>15</sup> ; Wilcoxon signed-rank test between the actual and average randomized models values per gene). However, analyses and comparisons to the randomized models gave an average value of 918.38 nt for <em>S. cerevisiae</em> and 926.55 nt for <em>S. pombe</em> , i.e. 1.75% and 0.63% higher, respectively. We believe that the relative small change in the 3′ UTR length is not expected to affect the NMD efficiency (in comparison, evaluation of the ratios in the case of average truncated translated intronic length gave 36.78% and 7.95%, respectively; see <a reveal-id="dsw019-F2" data-open="dsw019-F2" class="link link-reveal link-table xref-fig">Fig. 2B and C</a> ). </p><p> (iii) The definition of what is considered a ‘long’ 3′ UTR is somewhat ambiguous and there is no agreement on the relation between the 3′ UTR length and the NMD pathway efficiency. Thus, the mechanism by which PTCs are detected and promote assembly of the UPF-SMG complex is still poorly understood, though seems to be related to the 3′ UTR composition, and the relation between the PTC recognition and the actual mRNA decay is not fully understood. <a reveal-id="dsw019-B28" data-open="dsw019-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a><sup>,</sup><a reveal-id="dsw019-B29" data-open="dsw019-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a><sup>,</sup><a reveal-id="dsw019-B94" data-open="dsw019-B94" class="link link-ref link-reveal xref-bibr"><sup>94</sup></a><sup>,</sup><a reveal-id="dsw019-B95" data-open="dsw019-B95" class="link link-ref link-reveal xref-bibr"><sup>95</sup></a></p><p> (iv) If NMD is the dominant mechanism we expect to see a stronger signal of an intronic STOP near intronic 5′SS in introns further away from the beginning of the coding region; in these introns the increased ratio of the 3′ UTR due to a closer intron is expected to be stronger, promoting stronger selection. However, comparison of the average retained peptide length and metabolic costs in <em>S. pombe</em> between the first intron in each gene to the rest of the introns (2,269 <em>vs.</em> 2,478 introns, respectively) showed that the last ones are 25% longer and that the average metabolic cost in the first ones is 22% lower ( <em>P</em>  = 1.94×10 <sup>−</sup><sup>6</sup> and <em>P</em>  = 3.11×10 <sup>−</sup><sup>7</sup> , respectively; Wilcoxon rank-sum test; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Fig. S15</a></span> ). This result demonstrates that the signal is significantly stronger for the first introns in comparison to the rest, supporting the conjecture that NMD is not the most dominant factor here. </p><p> (v) Finally, we would like to emphasize that as in many cases reported in the past, a signal or phenomena may be related to multiple rules or reasons. <a reveal-id="dsw019-B95" data-open="dsw019-B95" class="link link-ref link-reveal xref-bibr"><sup>95</sup></a> Thus, it is clearly possible that both explanations (translation and NMD) are related to the suggested selection for intronic STOP close to the 5′SS. </p>                            <h2 scrollto-destination=87197724 id="87197724" class="section-title" >Discussion</h2>
<p> Intron retention is seldom found in fungi, and has not been reported in the organisms analysed in this study. However, here, we analyse four fungal genomes and report various novel evidence supporting the conjecture that novel sequence features found near the intronic splice sites are under selective pressure, presumably to reduce the translation costs of retained introns. Specifically, we report the following major results that are observed at the intronome level: (i) We demonstrate that the intronic sequences are selected for a close STOP codon near the 5′SS, presumably to reduce the cost of translating undesired peptides encoded in retained introns. (ii) In association with this outcome, the beginning of introns are selected for codons that improve TE, which resemble the ones that appear in annotated fungal ORFs, and thus probably undergo co-evolution with the tRNA pool. Again, this signal probably improves the fitness of the organism via reducing the translation and metabolic costs of peptides encoded in retained introns. In all of these cases the reported signals were compared to the ones obtained in randomized genomes and were shown to be significantly stronger, supporting the conjecture that they are indeed under selective pressure. (iii) Our analyses demonstrate that in <em>S. cerevisiae</em> there is higher RD at the beginning of introns, before the first intronic STOP codon, rather than downstream of it, supporting the conjecture that this region tends to undergo partial translation. (iv) The reported patterns occur in genes with various cellular functions supporting the conjecture that they are not function specific. (v) We show that the reported patterns are stronger in highly translated/expressed genes and in introns located in the beginning of the ORF, and specifically related to measures directly associated with TE, supporting the conjecture that they are indeed related to TE selection, and not to alternative signals. (vi) We show that the reported signals do not appear at the intronic 3′ end or in the region surrounding the second intronic STOP codon. (G) Finally, the level of codons’ TE at the 5′ pre-STOP intronic domain is very similar to the one appearing in annotated ORFs with similar ribosome densities/expression levels as in those intronic regions. </p><p> While there is an ongoing debate regarding the relation between codon-usage and TE based on the analyses of RP data, <a reveal-id="dsw019-B63" data-open="dsw019-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a><sup>,</sup><a reveal-id="dsw019-B96 dsw019-B97 dsw019-B98 dsw019-B99 dsw019-B100 dsw019-B101 dsw019-B102 dsw019-B103 dsw019-B104 dsw019-B105" data-open="dsw019-B96 dsw019-B97 dsw019-B98 dsw019-B99 dsw019-B100 dsw019-B101 dsw019-B102 dsw019-B103 dsw019-B104 dsw019-B105" class="link link-ref link-reveal xref-bibr"><sup>96–105</sup></a> it is important to emphasize that the most recent studies clearly demonstrate the relation between codon-usage and translation-elongation. <a reveal-id="dsw019-B63" data-open="dsw019-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a><sup>,</sup><a reveal-id="dsw019-B96" data-open="dsw019-B96" class="link link-ref link-reveal xref-bibr"><sup>96</sup></a><sup>,</sup><a reveal-id="dsw019-B100" data-open="dsw019-B100" class="link link-ref link-reveal xref-bibr"><sup>100</sup></a><sup>,</sup><a reveal-id="dsw019-B103 dsw019-B104 dsw019-B105" data-open="dsw019-B103 dsw019-B104 dsw019-B105" class="link link-ref link-reveal xref-bibr"><sup>103–105</sup></a> The current understanding is that previous studies did not find such a relation due to various problems and biases, including the nature of RP data, inaccurate analysis of the data, low coverage of the data, and more. <a reveal-id="dsw019-B63" data-open="dsw019-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a><sup>,</sup><a reveal-id="dsw019-B100" data-open="dsw019-B100" class="link link-ref link-reveal xref-bibr"><sup>100</sup></a><sup>,</sup><a reveal-id="dsw019-B103 dsw019-B104 dsw019-B105 dsw019-B106" data-open="dsw019-B103 dsw019-B104 dsw019-B105 dsw019-B106" class="link link-ref link-reveal xref-bibr"><sup>103–106</sup></a></p><p> The reported signals are observed at the genomic level, but are harder to detect for single introns due to the nature of the studied signals (see further details in the Materials and methods). Therefore, a comparison of the entire intronome to its randomized versions should enable evaluating such a selection. Likewise, other important recent studies in the field have performed analogous analyses. <a reveal-id="dsw019-B65" data-open="dsw019-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a><sup>,</sup><a reveal-id="dsw019-B107 dsw019-B108 dsw019-B109 dsw019-B110 dsw019-B111" data-open="dsw019-B107 dsw019-B108 dsw019-B109 dsw019-B110 dsw019-B111" class="link link-ref link-reveal xref-bibr"><sup>107–111</sup></a> We believe that the relatively rare translation events accumulate over all introns to yield a phenomena substantial enough to trigger a selective pressure. </p><p> It is important to mention that the reported patterns cannot be completely (or trivially) explained by alternative mechanisms related to splicing instead of translation. For example, it was suggested that in some organisms the intronic boundaries include alternative/tandem splice sites; <a reveal-id="dsw019-B112" data-open="dsw019-B112" class="link link-ref link-reveal xref-bibr"><sup>112</sup></a> thus, it is possible that the increased TE and RD at the intronic 5′ end is due to this phenomenon. However, the study did not include Fungi, which are different in terms of their splicing regulation and effective population size than the analysed organisms; <a reveal-id="dsw019-B14" data-open="dsw019-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsw019-B113 dsw019-B114 dsw019-B115 dsw019-B116" data-open="dsw019-B113 dsw019-B114 dsw019-B115 dsw019-B116" class="link link-ref link-reveal xref-bibr"><sup>113–116</sup></a> thus, it is not known whether tandem splice sites tend to occur also in Fungi. In addition, the fact that we did not find similar patterns surrounding the second STOP codon or at the intronic 3′SS ( <a reveal-id="dsw019-F1 dsw019-F7" data-open="dsw019-F1 dsw019-F7" class="link link-reveal link-table xref-fig">Fig. 7F1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary S13</a></span> ) supports the conjecture that the reported signals are at least partially related to translation. These results and the fact that we used CUB measures that are directly related to translation (TDR/tAI) support the conjecture that the reported CUB at the intronic 5′ end cannot be fully explained via non-adaptive mechanisms or aspects not related to translation (e.g. minimizing splicing errors or nucleotide synthesis costs). Furthermore, by any means the selection for STOP codons near intronic 5′ end cannot be related to tandem splice site signals. </p><p> Some studies have suggested that the process of intron splicing is quite efficient (the probability of splicing error is ∼1%), that introns in highly expressed genes are spliced more accurately, and that splicing errors trigger NMD which trigger degradation of the mRNA. <a reveal-id="dsw019-B89" data-open="dsw019-B89" class="link link-ref link-reveal xref-bibr"><sup>89</sup></a><sup>,</sup><a reveal-id="dsw019-B117" data-open="dsw019-B117" class="link link-ref link-reveal xref-bibr"><sup>117</sup></a> However, only few genes were examined and only in mammals, and it is not clear whether the results are consistent in different conditions or tissues. Since there are changes at the range and order of magnitude in various aspects of gene expression, <a reveal-id="dsw019-B118 dsw019-B119 dsw019-B120" data-open="dsw019-B118 dsw019-B119 dsw019-B120" class="link link-ref link-reveal xref-bibr"><sup>118–120</sup></a> we expect that also in terms of SE the differences may be very high (up to several orders of magnitudes). Here, because many of the analysed genes are highly expressed it is possible that even a translation ratio of 1% (that usually results in NMD) may result in substantial selection pressure that will enable the TE adaptation. Additionally, since the NMD pathway is based on ribosome halting at the premature STOP codon, <a reveal-id="dsw019-B121" data-open="dsw019-B121" class="link link-ref link-reveal xref-bibr"><sup>121</sup></a> when considering very large numbers of introns it will ‘cost’ less if the STOP is be closer to the 5′ end of the intron. Moreover, our analyses demonstrate that the RD at the <em>S. cerevisiae</em> intronic pre-STOP 5′ end (but not in other parts of the introns) is in agreement with the ribosomal densities in a large fraction of the annotated ORFs in this organism; we also show that the mean TDR at the intronic pre-STOP 5′ end is very similar to the one obtained for annotated ORFs with ribosomal densities and expression levels (when considering the 1% splicing error rate) at the intronic pre-STOP 5′ end. Thus, our results make sense and are in very good agreement also with the experimental evidence suggesting that the splicing error rate is 1% (see also <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"> Supplementary Note S1</a></span> ). An intriguing question for further research is whether there is a difference between wild type and NMD mutants with regard to ribosome footprinting patterns and translation of retained introns. </p><p> One may also wonder why evolution does not optimize splicing such that there will be no splicing errors at all, and therefore no need for translation optimization of introns; specifically, this solution may sound energetically cheaper and more ‘elegant’. A possible explanation to this “quandary” includes the fact that ‘evolutionary selection’ works more like a ‘tinkerer’ (mutation → selection → mutation → selection, etc.) that converge to local solutions rather than like an ‘engineer’ (that can find the globally optimal solution), due to various constraints and multiple variables involved in splicing and translation. Therefore, in our opinion it should not be surprising that evolution will ‘converge’ to the suggested ‘solution’. <a reveal-id="dsw019-B106" data-open="dsw019-B106" class="link link-ref link-reveal xref-bibr"><sup>106</sup></a><sup>,</sup><a reveal-id="dsw019-B122 dsw019-B123 dsw019-B124" data-open="dsw019-B122 dsw019-B123 dsw019-B124" class="link link-ref link-reveal xref-bibr"><sup>122–124</sup></a> It is easy to see that various aspects in biological systems are not optimal or can be further optimized. Furthermore, the higher splicing error rate may not ‘be solved’ (in a cheaper manner than the current ‘solution’) due to biophysical constraints and limitations. For example, it is possible that there is a trade-off(s) (as in the case of many other aspects of gene expression) between the ‘speed’ of splicing and its accuracy (a lower error rate may be related to lower splicing ‘speed’, which may eventually be more deleterious than the current state). This fact could be the ‘selective force’ facilitating the partial codon-usage/translation cost optimization. </p><p>Specifically, we suggest that the improved translation cost of introns contributes towards better organismal fitness. Indeed, translating a larger amount of useless proteins from the aberrantly spliced mRNAs is not expected to be beneficial; however, translating it in a more efficient manner is ‘better’/‘cheaper’ than translating it using non-efficient codons; on the other hand, completely eliminating these cases is also impossible due to possible higher (in terms of organismal fitness) negative SE or other/additional costs (and the nature of the evolutionary process which tend to converges to local optima).</p>                            <h2 scrollto-destination=87197735 id="87197735" class="section-title" >Conclusions</h2>
<p>One central conclusion of this study is that the classification to exons and introns in fungi is continuous rather than discrete, i.e. introns may also be translated to a certain degree. Moreover, it could make sense to classify many of the fungal intron-containing transcripts as alternatively spliced; while in many cases introns are spliced as defined, there are considerably abundant examples (enough to trigger translational preference on introns) where an intron is retained and subsequently a shorter peptide or protein is translated. Therefore, it will be interesting to study various properties of these putative shorter proteins.</p><p> Another principle conclusion is that functional silent mutations related to translation may appear in regions traditionally not classified as translated regions (e.g. introns). Our results support various recent studies in the field that have suggested that synonymous and silent mutations are functional. <a reveal-id="dsw019-B83" data-open="dsw019-B83" class="link link-ref link-reveal xref-bibr"><sup>83</sup></a><sup>,</sup><a reveal-id="dsw019-B125 dsw019-B126 dsw019-B127" data-open="dsw019-B125 dsw019-B126 dsw019-B127" class="link link-ref link-reveal xref-bibr"><sup>125–127</sup></a> However, we emphasize for the first time, the role of CUB also in non-coding parts of the transcript such as introns. </p><p>Insights from the results reported here can be used for developing future models of intronic evolution that will consider the effect of mutations both on SE and the possibility of intronic translation.</p>                            <h2 scrollto-destination=87197741 id="87197741" class="section-title" >Conflict of interest</h2>
<p>None declared.</p>                            <h2 scrollto-destination=87197744 id="87197744" class="section-title" >Funding</h2>
<p>This study was supported in part by a fellowship from the Edmond J. Safra Center for Bioinformatics at Tel-Aviv University. Funding to pay the Open Access publication charges for this article was provided by Tel-Aviv university and Minerva ARCHES award.</p>                            <h2 scrollto-destination=87197747 id="87197747" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw019-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Nilsen</div>
               <div class="given-names">T.W.</div>
            </div>
          </div>
          <div class="year">2003</div>
            , 
            <div class="article-title">The spliceosome: the most complex macromolecular machine in the cell?</div><div class="source">BioEssays</div> 
            , 
            <div class="volume">25</div>
            , 
            <div class="fpage">1147</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20spliceosome%3A%20the%20most%20complex%20macromolecular%20machine%20in%20the%20cell%3F&amp;author=T.W.%20Nilsen&amp;publication_year=2003&amp;journal=BioEssays&amp;volume=25&amp;pages=1147-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1521-1878" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1521-1878" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1521-1878"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14635248" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Hoskins</div>
               <div class="given-names">A.A.</div>
            </div>
             <div class="name">
              <div class="surname">Moore</div>
               <div class="given-names">M.J.</div>
            </div>
          </div>
          <div class="year">2012</div>
            , 
            <div class="article-title">The spliceosome: a flexible, reversible macromolecular machine</div>
            . 
            <div class="source">Trends Biochem. Sci</div> 
            ., 
            <div class="volume">37</div>
            , 
            <div class="fpage">179</div>
            –
            <div class="lpage">88</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20spliceosome%3A%20a%20flexible%2C%20reversible%20macromolecular%20machine&amp;author=A.A.%20Hoskins&amp;author=M.J.%20Moore&amp;publication_year=2012&amp;journal=Trends%20Biochem.%20Sci&amp;volume=37&amp;pages=179-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibs.2012.02.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibs.2012.02.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibs.2012.02.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22480731" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Le Hir</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Nott</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Moore</div>
               <div class="given-names">M.J.</div>
            </div>
          </div>
          <div class="year">2003</div>
            , 
            <div class="article-title">How introns influence and enhance eukaryotic gene expression</div>
            . 
            <div class="source">Trends Biochem. Sci</div> 
            ., 
            <div class="volume">28</div>
            , 
            <div class="fpage">215</div>
            –
            <div class="lpage">20</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20introns%20influence%20and%20enhance%20eukaryotic%20gene%20expression&amp;author=H.%20Le%20Hir&amp;author=A.%20Nott&amp;author=M.J.%20Moore&amp;publication_year=2003&amp;journal=Trends%20Biochem.%20Sci&amp;volume=28&amp;pages=215-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0968-0004(03)00052-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0968-0004(03)00052-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0968-0004(03)00052-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12713906" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Wang</div>
               <div class="given-names">G.S.</div>
            </div>
             <div class="name">
              <div class="surname">Cooper</div>
               <div class="given-names">T.A.</div>
            </div>
          </div>
          <div class="year">2007</div>
            , 
            <div class="article-title">Splicing in disease: disruption of the splicing code and the decoding machinery</div>
            . 
            <div class="source">Nat. Rev. Genet</div> 
            ., 
            <div class="volume">8</div>
            , 
            <div class="fpage">749</div>
            –
            <div class="lpage">61</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Splicing%20in%20disease%3A%20disruption%20of%20the%20splicing%20code%20and%20the%20decoding%20machinery&amp;author=G.S.%20Wang&amp;author=T.A.%20Cooper&amp;publication_year=2007&amp;journal=Nat.%20Rev.%20Genet&amp;volume=8&amp;pages=749-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2164" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2164" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2164"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17726481" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kramer</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">1996</div>
            , 
            <div class="article-title">The structure and function of proteins involved in mammalian pre-mRNA splicing</div>
            . 
            <div class="source">Annu. Rev. Biochem</div> 
            ., 
            <div class="volume">65</div>
            , 
            <div class="fpage">367</div>
            –
            <div class="lpage">409</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20structure%20and%20function%20of%20proteins%20involved%20in%20mammalian%20pre-mRNA%20splicing&amp;author=A.%20Kramer&amp;publication_year=1996&amp;journal=Annu.%20Rev.%20Biochem&amp;volume=65&amp;pages=367-409" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.bi.65.070196.002055" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.bi.65.070196.002055" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.bi.65.070196.002055"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8811184" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">McKee</div>
               <div class="given-names">A.E.</div>
            </div>
             <div class="name">
              <div class="surname">Silver</div>
               <div class="given-names">P.A.</div>
            </div>
          </div>
          <div class="year">2007</div>
            , 
            <div class="article-title">Systems perspectives on mRNA processing</div>
            . 
            <div class="source">Cell Res</div> 
            ., 
            <div class="volume">17</div>
            , 
            <div class="fpage">581</div>
            –
            <div class="lpage">90</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systems%20perspectives%20on%20mRNA%20processing&amp;author=A.E.%20McKee&amp;author=P.A.%20Silver&amp;publication_year=2007&amp;journal=Cell%20Res&amp;volume=17&amp;pages=581-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/cr.2007.54" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fcr.2007.54" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fcr.2007.54"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17621309" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Toor</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Keating</div>
               <div class="given-names">K.S.</div>
            </div>
             <div class="name">
              <div class="surname">Taylor</div>
               <div class="given-names">S.D.</div>
            </div>
             <div class="name">
              <div class="surname">Pyle</div>
               <div class="given-names">A.M.</div>
            </div>
          </div>
          <div class="year">2008</div>
            , 
            <div class="article-title">Crystal structure of a self-spliced group II intron</div>
            . 
            <div class="source">Science</div> 
            , 
            <div class="volume">320</div>
            , 
            <div class="fpage">77</div>
            –
            <div class="lpage">82</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Crystal%20structure%20of%20a%20self-spliced%20group%20II%20intron&amp;author=N.%20Toor&amp;author=K.S.%20Keating&amp;author=S.D.%20Taylor&amp;author=A.M.%20Pyle&amp;publication_year=2008&amp;journal=Science&amp;volume=320&amp;pages=77-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1153803" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1153803" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1153803"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18388288" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Wahl</div>
               <div class="given-names">M.C.</div>
            </div>
             <div class="name">
              <div class="surname">Will</div>
               <div class="given-names">C.L.</div>
            </div>
             <div class="name">
              <div class="surname">Lührmann</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="year">2009</div>
            , 
            <div class="article-title">The spliceosome: design principles of a dynamic RNP machine</div>
            . 
            <div class="source">Cell</div> 
            , 
            <div class="volume">136</div>
            , 
            <div class="fpage">701</div>
            –
            <div class="lpage">18</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20spliceosome%3A%20design%20principles%20of%20a%20dynamic%20RNP%20machine&amp;author=M.C.%20Wahl&amp;author=C.L.%20Will&amp;author=R.%20L%C3%BChrmann&amp;publication_year=2009&amp;journal=Cell&amp;volume=136&amp;pages=701-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2009.02.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2009.02.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2009.02.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19239890" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Rodríguez-Trelles</div>
               <div class="given-names">F.</div>
            </div>
             <div class="name">
              <div class="surname">Tarrío</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Ayala</div>
               <div class="given-names">F.J.</div>
            </div>
          </div>
          <div class="year">2006</div>
            , 
            <div class="article-title">Origins and evolution of spliceosomal introns</div>
            . 
            <div class="source">Annu. Rev. Genet</div> 
            ., 
            <div class="volume">40</div>
            , 
            <div class="fpage">47</div>
            –
            <div class="lpage">76</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origins%20and%20evolution%20of%20spliceosomal%20introns&amp;author=F.%20Rodr%C3%ADguez-Trelles&amp;author=R.%20Tarr%C3%ADo&amp;author=F.J.%20Ayala&amp;publication_year=2006&amp;journal=Annu.%20Rev.%20Genet&amp;volume=40&amp;pages=47-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.40.110405.090625" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.40.110405.090625" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.40.110405.090625"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17094737" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Rogozin</div>
               <div class="given-names">I.</div>
            </div>
             <div class="name">
              <div class="surname">Carmel</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Csuros</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Koonin</div>
               <div class="given-names">E.</div>
            </div>
          </div>
          <div class="year">2012</div>
            , 
            <div class="article-title">Origin and evolution of spliceosomal introns</div>
            . 
            <div class="source">Biol. Direct</div> 
            , 
            <div class="volume">7</div>
            , 
            <div class="fpage">11</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origin%20and%20evolution%20of%20spliceosomal%20introns&amp;author=I.%20Rogozin&amp;author=L.%20Carmel&amp;author=M.%20Csuros&amp;author=E.%20Koonin&amp;publication_year=2012&amp;journal=Biol.%20Direct&amp;volume=7&amp;pages=11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1745-6150-7-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1745-6150-7-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1745-6150-7-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22507701" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Maniatis</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Tasic</div>
               <div class="given-names">B.</div>
            </div>
          </div>
          <div class="year">2002</div>
            , 
            <div class="article-title">Alternative pre-mRNA splicing and proteome expansion in metazoans</div>
            . 
            <div class="source">Nature</div> 
            , 
            <div class="volume">418</div>
            , 
            <div class="fpage">236</div>
            –
            <div class="lpage">43</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20pre-mRNA%20splicing%20and%20proteome%20expansion%20in%20metazoans&amp;author=T.%20Maniatis&amp;author=B.%20Tasic&amp;publication_year=2002&amp;journal=Nature&amp;volume=418&amp;pages=236-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/418236a" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F418236a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F418236a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12110900" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Black</div>
               <div class="given-names">D.L.</div>
            </div>
          </div>
          <div class="year">2003</div>
            , 
            <div class="article-title">Mechanisms of alternative pre-messenger RNA splicing</div>
            . 
            <div class="source">Annu. Rev. Biochem</div> 
            ., 
            <div class="volume">72</div>
            , 
            <div class="fpage">291</div>
            –
            <div class="lpage">336</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanisms%20of%20alternative%20pre-messenger%20RNA%20splicing&amp;author=D.L.%20Black&amp;publication_year=2003&amp;journal=Annu.%20Rev.%20Biochem&amp;volume=72&amp;pages=291-336" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.biochem.72.121801.161720" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.biochem.72.121801.161720" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.biochem.72.121801.161720"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12626338" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Wang</div>
               <div class="given-names">B.B.</div>
            </div>
             <div class="name">
              <div class="surname">Brendel</div>
               <div class="given-names">V.</div>
            </div>
          </div>
          <div class="year">2006</div>
            , 
            <div class="article-title">Genomewide comparative analysis of alternative splicing in plants</div>
            . 
            <div class="source">Proc. Natl. Acad. Sci. USA</div> 
            ., 
            <div class="volume">103</div>
            , 
            <div class="fpage">7175</div>
            –
            <div class="lpage">80</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomewide%20comparative%20analysis%20of%20alternative%20splicing%20in%20plants&amp;author=B.B.%20Wang&amp;author=V.%20Brendel&amp;publication_year=2006&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=103&amp;pages=7175-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0602039103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0602039103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0602039103"> </span></div> </div></div></div></div></div><div content-id="dsw019-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kim</div>
               <div class="given-names">E.</div>
            </div>
             <div class="name">
              <div class="surname">Magen</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Ast</div>
               <div class="given-names">G.</div>
            </div>
          </div>
          <div class="year">2007</div>
            , 
            <div class="article-title">Different levels of alternative splicing among eukaryotes</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">35</div>
            , 
            <div class="fpage">125</div>
            –
            <div class="lpage">31</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Different%20levels%20of%20alternative%20splicing%20among%20eukaryotes&amp;author=E.%20Kim&amp;author=A.%20Magen&amp;author=G.%20Ast&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=125-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl924" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl924" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl924"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17158149" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Juneau</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Nislow</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Davis</div>
               <div class="given-names">R.W.</div>
            </div>
          </div>
          <div class="year">2009</div>
            , 
            <div class="article-title">
              Alternative splicing of PTC7 in 
              <em>Saccharomyces cerevisiae</em>
               determines protein localization
            </div>
            . 
            <div class="source">Genetics</div> 
            , 
            <div class="volume">183</div>
            , 
            <div class="fpage">185</div>
            –
            <div class="lpage">94</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20of%20PTC7%20in%20Saccharomyces%20cerevisiae%20determines%20protein%20localization&amp;author=K.%20Juneau&amp;author=C.%20Nislow&amp;author=R.W.%20Davis&amp;publication_year=2009&amp;journal=Genetics&amp;volume=183&amp;pages=185-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.109.105155" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.109.105155" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.109.105155"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19564484" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Rhind</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Chen</div>
               <div class="given-names">Z.</div>
            </div>
             <div class="name">
              <div class="surname">Yassour</div>
               <div class="given-names">M.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2011</div>
            , 
            <div class="article-title">Comparative functional genomics of the fission yeasts</div>
            . 
            <div class="source">Science</div> 
            , 
            <div class="volume">332</div>
            , 
            <div class="fpage">930</div>
            –
            <div class="lpage">6</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20functional%20genomics%20of%20the%20fission%20yeasts&amp;author=N.%20Rhind&amp;author=Z.%20Chen&amp;author=M.%20Yassour&amp;publication_year=2011&amp;journal=Science&amp;volume=332&amp;pages=930-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1203357" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1203357" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1203357"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21511999" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Marshall</div>
               <div class="given-names">A.N.</div>
            </div>
             <div class="name">
              <div class="surname">Montealegre</div>
               <div class="given-names">M.C.</div>
            </div>
             <div class="name">
              <div class="surname">Jiménez-López</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Lorenz</div>
               <div class="given-names">M.C.</div>
            </div>
             <div class="name">
              <div class="surname">van Hoof</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">2013</div>
            , 
            <div class="article-title">Alternative splicing and subfunctionalization generates functional diversity in fungal proteomes</div>
            . 
            <div class="source">PLoS Genet</div> 
            ., 
            <div class="volume">9</div>
            , 
            <div class="fpage">e1003376</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20and%20subfunctionalization%20generates%20functional%20diversity%20in%20fungal%20proteomes&amp;author=A.N.%20Marshall&amp;author=M.C.%20Montealegre&amp;author=C.%20Jim%C3%A9nez-L%C3%B3pez&amp;author=M.C.%20Lorenz&amp;author=A.%20van%20Hoof&amp;publication_year=2013&amp;journal=PLoS%20Genet&amp;volume=9&amp;pages=e1003376" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1003376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1003376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1003376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23516382" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ast</div>
               <div class="given-names">G.</div>
            </div>
          </div>
          <div class="year">2004</div>
            , 
            <div class="article-title">How did alternative splicing evolve?</div><div class="source">Nat. Rev. Genet</div> 
            ., 
            <div class="volume">5</div>
            , 
            <div class="fpage">773</div>
            –
            <div class="lpage">82</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20did%20alternative%20splicing%20evolve%3F&amp;author=G.%20Ast&amp;publication_year=2004&amp;journal=Nat.%20Rev.%20Genet&amp;volume=5&amp;pages=773-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1451" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1451" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1451"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15510168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ner-Gaon</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Halachmi</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Savaldi-Goldstein</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Rubin</div>
               <div class="given-names">E.</div>
            </div>
             <div class="name">
              <div class="surname">Ophir</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Fluhr</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="year">2004</div>
            , 
            <div class="article-title">
              Intron retention is a major phenomenon in alternative splicing in 
              <em>Arabidopsis</em></div>
            . 
            <div class="source">Plant J</div> 
            ., 
            <div class="volume">39</div>
            , 
            <div class="fpage">877</div>
            –
            <div class="lpage">85</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intron%20retention%20is%20a%20major%20phenomenon%20in%20alternative%20splicing%20in%20Arabidopsis&amp;author=H.%20Ner-Gaon&amp;author=R.%20Halachmi&amp;author=S.%20Savaldi-Goldstein&amp;author=E.%20Rubin&amp;author=R.%20Ophir&amp;author=R.%20Fluhr&amp;publication_year=2004&amp;journal=Plant%20J&amp;volume=39&amp;pages=877-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2004.39.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2004.39.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2004.39.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15341630" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kim</div>
               <div class="given-names">E.</div>
            </div>
             <div class="name">
              <div class="surname">Goren</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Ast</div>
               <div class="given-names">G.</div>
            </div>
          </div>
          <div class="year">2008</div>
            , 
            <div class="article-title">Alternative splicing: current perspectives</div>
            . 
            <div class="source">BioEssays</div> 
            , 
            <div class="volume">30</div>
            , 
            <div class="fpage">38</div>
            –
            <div class="lpage">47</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%3A%20current%20perspectives&amp;author=E.%20Kim&amp;author=A.%20Goren&amp;author=G.%20Ast&amp;publication_year=2008&amp;journal=BioEssays&amp;volume=30&amp;pages=38-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1521-1878" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1521-1878" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1521-1878"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18081010" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Barbazuk</div>
               <div class="given-names">W.B.</div>
            </div>
             <div class="name">
              <div class="surname">Fu</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">McGinnis</div>
               <div class="given-names">K.M.</div>
            </div>
          </div>
          <div class="year">2008</div>
            , 
            <div class="article-title">Genome-wide analyses of alternative splicing in plants: opportunities and challenges</div>
            . 
            <div class="source">Genome Res</div> 
            ., 
            <div class="volume">18</div>
            , 
            <div class="fpage">1381</div>
            –
            <div class="lpage">92</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analyses%20of%20alternative%20splicing%20in%20plants%3A%20opportunities%20and%20challenges&amp;author=W.B.%20Barbazuk&amp;author=Y.%20Fu&amp;author=K.M.%20McGinnis&amp;publication_year=2008&amp;journal=Genome%20Res&amp;volume=18&amp;pages=1381-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.053678.106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.053678.106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.053678.106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18669480" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Keren</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Lev-Maor</div>
               <div class="given-names">G.</div>
            </div>
             <div class="name">
              <div class="surname">Ast</div>
               <div class="given-names">G.</div>
            </div>
          </div>
          <div class="year">2010</div>
            , 
            <div class="article-title">Alternative splicing and evolution: diversification, exon definition and function</div>
            . 
            <div class="source">Nat. Rev. Genet</div> 
            ., 
            <div class="volume">11</div>
            , 
            <div class="fpage">345</div>
            –
            <div class="lpage">55</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20and%20evolution%3A%20diversification%2C%20exon%20definition%20and%20function&amp;author=H.%20Keren&amp;author=G.%20Lev-Maor&amp;author=G.%20Ast&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Genet&amp;volume=11&amp;pages=345-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2776" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2776" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2776"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20376054" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">McManus</div>
               <div class="given-names">C.J.</div>
            </div>
             <div class="name">
              <div class="surname">Graveley</div>
               <div class="given-names">B.R.</div>
            </div>
          </div>
          <div class="year">2011</div>
            , 
            <div class="article-title">RNA structure and the mechanisms of alternative splicing</div>
            . 
            <div class="source">Curr. Opin. Genet. Dev</div> 
            ., 
            <div class="volume">21</div>
            , 
            <div class="fpage">373</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20structure%20and%20the%20mechanisms%20of%20alternative%20splicing&amp;author=C.J.%20McManus&amp;author=B.R.%20Graveley&amp;publication_year=2011&amp;journal=Curr.%20Opin.%20Genet.%20Dev&amp;volume=21&amp;pages=373-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gde.2011.04.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2011.04.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gde.2011.04.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21530232" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Jaillon</div>
               <div class="given-names">O.</div>
            </div>
             <div class="name">
              <div class="surname">Bouhouche</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Gout</div>
               <div class="given-names">J.F.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2008</div>
            , 
            <div class="article-title">Translational control of intron splicing in eukaryotes</div>
            . 
            <div class="source">Nature</div> 
            , 
            <div class="volume">451</div>
            , 
            <div class="fpage">359</div>
            –
            <div class="lpage">62</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translational%20control%20of%20intron%20splicing%20in%20eukaryotes&amp;author=O.%20Jaillon&amp;author=K.%20Bouhouche&amp;author=J.F.%20Gout&amp;publication_year=2008&amp;journal=Nature&amp;volume=451&amp;pages=359-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06495" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06495" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06495"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18202663" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Conti</div>
               <div class="given-names">E.</div>
            </div>
             <div class="name">
              <div class="surname">Izaurralde</div>
               <div class="given-names">E.</div>
            </div>
          </div>
          <div class="year">2005</div>
            , 
            <div class="article-title">Nonsense-mediated mRNA decay: molecular insights and mechanistic variations across species</div>
            . 
            <div class="source">Curr. Opin. Cell Biol</div> 
            ., 
            <div class="volume">17</div>
            , 
            <div class="fpage">316</div>
            –
            <div class="lpage">25</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nonsense-mediated%20mRNA%20decay%3A%20molecular%20insights%20and%20mechanistic%20variations%20across%20species&amp;author=E.%20Conti&amp;author=E.%20Izaurralde&amp;publication_year=2005&amp;journal=Curr.%20Opin.%20Cell%20Biol&amp;volume=17&amp;pages=316-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ceb.2005.04.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ceb.2005.04.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ceb.2005.04.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15901503" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Rodríguez-Gabriel</div>
               <div class="given-names">M.A.</div>
            </div>
             <div class="name">
              <div class="surname">Watt</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Bähler</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Russell</div>
               <div class="given-names">P.</div>
            </div>
          </div>
          <div class="year">2006</div>
            , 
            <div class="article-title">Upf1, an RNA helicase required for nonsense-mediated mRNA decay, modulates the transcriptional response to oxidative stress in fission yeast</div>
            . 
            <div class="source">Mol. Cellular Biol</div> 
            ., 
            <div class="volume">26</div>
            , 
            <div class="fpage">6347</div>
            –
            <div class="lpage">56</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Upf1%2C%20an%20RNA%20helicase%20required%20for%20nonsense-mediated%20mRNA%20decay%2C%20modulates%20the%20transcriptional%20response%20to%20oxidative%20stress%20in%20fission%20yeast&amp;author=M.A.%20Rodr%C3%ADguez-Gabriel&amp;author=S.%20Watt&amp;author=J.%20B%C3%A4hler&amp;author=P.%20Russell&amp;publication_year=2006&amp;journal=Mol.%20Cellular%20Biol&amp;volume=26&amp;pages=6347-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.00286-06" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.00286-06" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.00286-06"> </span></div> </div></div></div></div></div><div content-id="dsw019-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Morozov</div>
               <div class="given-names">I.Y.</div>
            </div>
             <div class="name">
              <div class="surname">Negrete-Urtasun</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Tilburn</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Jansen</div>
               <div class="given-names">C.A.</div>
            </div>
             <div class="name">
              <div class="surname">Caddick</div>
               <div class="given-names">M.X.</div>
            </div>
             <div class="name">
              <div class="surname">Arst</div>
               <div class="given-names">H.N.</div>
            </div>
          </div>
          <div class="year">2006</div>
            , 
            <div class="article-title">
              Nonsense-mediated mRNA decay mutation in 
              <em>Aspergillus nidulans</em></div>
            . 
            <div class="source">Eukaryot. Cell</div> 
            , 
            <div class="volume">5</div>
            , 
            <div class="fpage">1838</div>
            –
            <div class="lpage">46</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nonsense-mediated%20mRNA%20decay%20mutation%20in%20Aspergillus%20nidulans&amp;author=I.Y.%20Morozov&amp;author=S.%20Negrete-Urtasun&amp;author=J.%20Tilburn&amp;author=C.A.%20Jansen&amp;author=M.X.%20Caddick&amp;author=H.N.%20Arst&amp;publication_year=2006&amp;journal=Eukaryot.%20Cell&amp;volume=5&amp;pages=1838-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/EC.00220-06" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FEC.00220-06" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FEC.00220-06"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16963627" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Brogna</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Wen</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="year">2009</div>
            , 
            <div class="article-title">Nonsense-mediated mRNA decay (NMD) mechanisms</div>
            . 
            <div class="source">Nat. Struct. Mol. Biol</div> 
            ., 
            <div class="volume">16</div>
            , 
            <div class="fpage">107</div>
            –
            <div class="lpage">13</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nonsense-mediated%20mRNA%20decay%20(NMD)%20mechanisms&amp;author=S.%20Brogna&amp;author=J.%20Wen&amp;publication_year=2009&amp;journal=Nat.%20Struct.%20Mol.%20Biol&amp;volume=16&amp;pages=107-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.1550" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.1550" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.1550"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19190664" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Peccarelli</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Kebaara</div>
               <div class="given-names">B.W.</div>
            </div>
          </div>
          <div class="year">2014</div>
            , 
            <div class="article-title">Regulation of natural mRNAs by the nonsense-mediated mRNA decay pathway</div>
            . 
            <div class="source">Eukaryot. Cell</div> 
            , 
            <div class="volume">13</div>
            , 
            <div class="fpage">1126</div>
            –
            <div class="lpage">35</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20natural%20mRNAs%20by%20the%20nonsense-mediated%20mRNA%20decay%20pathway&amp;author=M.%20Peccarelli&amp;author=B.W.%20Kebaara&amp;publication_year=2014&amp;journal=Eukaryot.%20Cell&amp;volume=13&amp;pages=1126-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/EC.00090-14" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FEC.00090-14" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FEC.00090-14"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25038084" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kriventseva</div>
               <div class="given-names">E.V.</div>
            </div>
             <div class="name">
              <div class="surname">Gelfand</div>
               <div class="given-names">M.S.</div>
            </div>
          </div>
          <div class="year">1999</div>
            , 
            <div class="article-title">Statistical analysis of the exon-intron structure of higher and lower eukaryote genes</div>
            . 
            <div class="source">J. Biomol. Struct. Dyn</div> 
            ., 
            <div class="volume">17</div>
            , 
            <div class="fpage">281</div>
            –
            <div class="lpage">8</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Statistical%20analysis%20of%20the%20exon-intron%20structure%20of%20higher%20and%20lower%20eukaryote%20genes&amp;author=E.V.%20Kriventseva&amp;author=M.S.%20Gelfand&amp;publication_year=1999&amp;journal=J.%20Biomol.%20Struct.%20Dyn&amp;volume=17&amp;pages=281-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/07391102.1999.10508361" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F07391102.1999.10508361" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F07391102.1999.10508361"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10563578" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Lim</div>
               <div class="given-names">L.P.</div>
            </div>
             <div class="name">
              <div class="surname">Burge</div>
               <div class="given-names">C.B.</div>
            </div>
          </div>
          <div class="year">2001</div>
            , 
            <div class="article-title">A computational analysis of sequence features involved in recognition of short introns</div>
            . 
            <div class="source">Proc. Natl. Acad. Sci. USA</div> 
            ., 
            <div class="volume">98</div>
            , 
            <div class="fpage">11193</div>
            –
            <div class="lpage">8</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20computational%20analysis%20of%20sequence%20features%20involved%20in%20recognition%20of%20short%20introns&amp;author=L.P.%20Lim&amp;author=C.B.%20Burge&amp;publication_year=2001&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=11193-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.201407298" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.201407298" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.201407298"> </span></div> </div></div></div></div></div><div content-id="dsw019-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Yu</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Yang</div>
               <div class="given-names">Z.</div>
            </div>
             <div class="name">
              <div class="surname">Kibukawa</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Paddock</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Passey</div>
               <div class="given-names">D.A.</div>
            </div>
             <div class="name">
              <div class="surname">Wong</div>
               <div class="given-names">G.K.S.</div>
            </div>
          </div>
          <div class="year">2002</div>
            , 
            <div class="article-title">Minimal introns are not “junk”</div>
            . 
            <div class="source">Genome Res</div> 
            ., 
            <div class="volume">12</div>
            , 
            <div class="fpage">1185</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Minimal%20introns%20are%20not%20%E2%80%9Cjunk%E2%80%9D&amp;author=J.%20Yu&amp;author=Z.%20Yang&amp;author=M.%20Kibukawa&amp;author=M.%20Paddock&amp;author=D.A.%20Passey&amp;author=G.K.S.%20Wong&amp;publication_year=2002&amp;journal=Genome%20Res&amp;volume=12&amp;pages=1185-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.224602" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.224602" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.224602"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12176926" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Alexander</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Anatoliy</div>
               <div class="given-names">I.</div>
            </div>
             <div class="name">
              <div class="surname">Alexander</div>
               <div class="given-names">B.</div>
            </div>
          </div>
          <div class="year">2011</div>
            , 
            <div class="article-title">Statistical analysis of exon lengths in various eukaryotes</div>
            . 
            <div class="source">Open Access Bioinformatics</div> 
            , 
            <div class="volume">2011</div>
            , 
            <div class="fpage">1</div>
            –
            <div class="lpage">15</div>
            .
          </div></div></div></div><div content-id="dsw019-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Spingola</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Grate</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Haussler</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Ares</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="year">1999</div>
            , 
            <div class="article-title">
              Genome-wide bioinformatic and molecular analysis of introns in 
              <em>Saccharomyces cerevisiae</em></div>
            . 
            <div class="source">RNA</div> 
            , 
            <div class="volume">5</div>
            , 
            <div class="fpage">221</div>
            –
            <div class="lpage">34</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20bioinformatic%20and%20molecular%20analysis%20of%20introns%20in%20Saccharomyces%20cerevisiae&amp;author=M.%20Spingola&amp;author=L.%20Grate&amp;author=D.%20Haussler&amp;author=M.%20Ares&amp;publication_year=1999&amp;journal=RNA&amp;volume=5&amp;pages=221-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S1355838299981682" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS1355838299981682" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS1355838299981682"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10024174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Grate</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Ares Jr</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="year">2002</div>
            , 
            Searching yeast intron data at Ares lab web site
            . In: 
            <div class="person-group"><div class="name"><div class="surname">Christine</div> <div class="given-names">G.</div></div> <div class="name"><div class="surname">Gerald</div> <div class="given-names">R.F.</div></div></div>
             (eds), 
            <div class="source">Methods in Enzymology</div> 
            , pp. 
            <div class="fpage">380</div>
            –
            <div class="lpage">92</div>
            . 
            <div class="publisher-name">Academic Press, imprint of Elsevier Science</div>
            : 
            <div class="publisher-loc">San Diego, CA</div>
            .
          </div></div></div></div><div content-id="dsw019-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Mishra</div>
               <div class="given-names">S.K.</div>
            </div>
             <div class="name">
              <div class="surname">Ammon</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Popowicz</div>
               <div class="given-names">G.M.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2011</div>
            , 
            <div class="article-title">Role of the ubiquitin-like protein Hub1 in splice-site usage and alternative splicing</div>
            . 
            <div class="source">Nature</div> 
            , 
            <div class="volume">474</div>
            , 
            <div class="fpage">173</div>
            –
            <div class="lpage">8</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20the%20ubiquitin-like%20protein%20Hub1%20in%20splice-site%20usage%20and%20alternative%20splicing&amp;author=S.K.%20Mishra&amp;author=T.%20Ammon&amp;author=G.M.%20Popowicz&amp;publication_year=2011&amp;journal=Nature&amp;volume=474&amp;pages=173-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10143" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10143" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10143"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21614000" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ares</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Grate</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Pauling</div>
               <div class="given-names">M.H.</div>
            </div>
          </div>
          <div class="year">1999</div>
            , 
            <div class="article-title">A handful of intron-containing genes produces the lion's share of yeast mRNA</div>
            . 
            <div class="source">RNA</div> 
            , 
            <div class="volume">5</div>
            , 
            <div class="fpage">1138</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20handful%20of%20intron-containing%20genes%20produces%20the%20lion's%20share%20of%20yeast%20mRNA&amp;author=M.%20Ares&amp;author=L.%20Grate&amp;author=M.H.%20Pauling&amp;publication_year=1999&amp;journal=RNA&amp;volume=5&amp;pages=1138-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S1355838299991379" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS1355838299991379" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS1355838299991379"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10496214" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Juneau</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Miranda</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Hillenmeyer</div>
               <div class="given-names">M.E.</div>
            </div>
             <div class="name">
              <div class="surname">Nislow</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Davis</div>
               <div class="given-names">R.W.</div>
            </div>
          </div>
          <div class="year">2006</div>
            , 
            <div class="article-title">Introns regulate RNA and protein abundance in yeast</div>
            . 
            <div class="source">Genetics</div> 
            , 
            <div class="volume">174</div>
            , 
            <div class="fpage">511</div>
            –
            <div class="lpage">8</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Introns%20regulate%20RNA%20and%20protein%20abundance%20in%20yeast&amp;author=K.%20Juneau&amp;author=M.%20Miranda&amp;author=M.E.%20Hillenmeyer&amp;author=C.%20Nislow&amp;author=R.W.%20Davis&amp;publication_year=2006&amp;journal=Genetics&amp;volume=174&amp;pages=511-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.106.058560" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.106.058560" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.106.058560"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16816425" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Wood</div>
               <div class="given-names">V.</div>
            </div>
             <div class="name">
              <div class="surname">Gwilliam</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Rajandream</div>
               <div class="given-names">M.A.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2002</div>
            , 
            <div class="article-title">
              The genome sequence of 
              <em>Schizosaccharomyces pombe</em></div>
            . 
            <div class="source">Nature</div> 
            , 
            <div class="volume">415</div>
            , 
            <div class="fpage">871</div>
            –
            <div class="lpage">80</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20Schizosaccharomyces%20pombe&amp;author=V.%20Wood&amp;author=R.%20Gwilliam&amp;author=M.A.%20Rajandream&amp;publication_year=2002&amp;journal=Nature&amp;volume=415&amp;pages=871-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature724" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature724" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature724"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11859360" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Habara</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Urushiyama</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Tani</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Ohshima</div>
               <div class="given-names">Y.</div>
            </div>
          </div>
          <div class="year">1998</div>
            , 
            <div class="article-title">The fission yeast prp10+ gene involved in pre-mRNA splicing encodes a homologue of highly conserved splicing factor, SAP155</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">26</div>
            , 
            <div class="fpage">5662</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20fission%20yeast%20prp10%2B%20gene%20involved%20in%20pre-mRNA%20splicing%20encodes%20a%20homologue%20of%20highly%20conserved%20splicing%20factor%2C%20SAP155&amp;author=Y.%20Habara&amp;author=S.%20Urushiyama&amp;author=T.%20Tani&amp;author=Y.%20Ohshima&amp;publication_year=1998&amp;journal=Nucleic%20Acids%20Res&amp;volume=26&amp;pages=5662-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.24.5662" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.24.5662" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.24.5662"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9837997" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Okazaki</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Niwa</div>
               <div class="given-names">O.</div>
            </div>
          </div>
          <div class="year">2000</div>
            , 
            <div class="article-title">mRNAs encoding zinc finger protein isoforms are expressed by alternative splicing of an in-frame intron in fission yeast</div>
            . 
            <div class="source">DNA Res</div> 
            ., 
            <div class="volume">7</div>
            , 
            <div class="fpage">27</div>
            –
            <div class="lpage">30</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=mRNAs%20encoding%20zinc%20finger%20protein%20isoforms%20are%20expressed%20by%20alternative%20splicing%20of%20an%20in-frame%20intron%20in%20fission%20yeast&amp;author=K.%20Okazaki&amp;author=O.%20Niwa&amp;publication_year=2000&amp;journal=DNA%20Res&amp;volume=7&amp;pages=27-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/7.1.27" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F7.1.27" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F7.1.27"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10718196" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kupfer</div>
               <div class="given-names">D.M.</div>
            </div>
             <div class="name">
              <div class="surname">Drabenstot</div>
               <div class="given-names">S.D.</div>
            </div>
             <div class="name">
              <div class="surname">Buchanan</div>
               <div class="given-names">K.L.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2004</div>
            , 
            <div class="article-title">Introns and splicing elements of five diverse fungi</div>
            . 
            <div class="source">Eukaryot. Cell</div> 
            , 
            <div class="volume">3</div>
            , 
            <div class="fpage">1088</div>
            –
            <div class="lpage">100</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Introns%20and%20splicing%20elements%20of%20five%20diverse%20fungi&amp;author=D.M.%20Kupfer&amp;author=S.D.%20Drabenstot&amp;author=K.L.%20Buchanan&amp;publication_year=2004&amp;journal=Eukaryot.%20Cell&amp;volume=3&amp;pages=1088-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/EC.3.5.1088-1100.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FEC.3.5.1088-1100.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FEC.3.5.1088-1100.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15470237" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Mitrovich</div>
               <div class="given-names">Q.M.</div>
            </div>
             <div class="name">
              <div class="surname">Tuch</div>
               <div class="given-names">B.B.</div>
            </div>
             <div class="name">
              <div class="surname">Guthrie</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Johnson</div>
               <div class="given-names">A.D.</div>
            </div>
          </div>
          <div class="year">2007</div>
            , 
            <div class="article-title">
              Computational and experimental approaches double the number of known introns in the pathogenic yeast 
              <em>Candida albicans</em></div>
            . 
            <div class="source">Genome Res</div> 
            ., 
            <div class="volume">17</div>
            , 
            <div class="fpage">492</div>
            –
            <div class="lpage">502</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20and%20experimental%20approaches%20double%20the%20number%20of%20known%20introns%20in%20the%20pathogenic%20yeast%20Candida%20albicans&amp;author=Q.M.%20Mitrovich&amp;author=B.B.%20Tuch&amp;author=C.%20Guthrie&amp;author=A.D.%20Johnson&amp;publication_year=2007&amp;journal=Genome%20Res&amp;volume=17&amp;pages=492-502" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.6111907" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.6111907" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.6111907"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17351132" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Stajich</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Dietrich</div>
               <div class="given-names">F.</div>
            </div>
             <div class="name">
              <div class="surname">Roy</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">2007</div>
            , 
            <div class="article-title">Comparative genomic analysis of fungal genomes reveals intron-rich ancestors</div>
            . 
            <div class="source">Genome Biol</div> 
            ., 
            <div class="volume">8</div>
            , 
            <div class="fpage">R223</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomic%20analysis%20of%20fungal%20genomes%20reveals%20intron-rich%20ancestors&amp;author=J.%20Stajich&amp;author=F.%20Dietrich&amp;author=S.%20Roy&amp;publication_year=2007&amp;journal=Genome%20Biol&amp;volume=8&amp;pages=R223" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2007-8-10-r223" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2007-8-10-r223" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2007-8-10-r223"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17949488" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Carmel</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Rogozin</div>
               <div class="given-names">I.B.</div>
            </div>
             <div class="name">
              <div class="surname">Wolf</div>
               <div class="given-names">Y.I.</div>
            </div>
             <div class="name">
              <div class="surname">Koonin</div>
               <div class="given-names">E.V.</div>
            </div>
          </div>
          <div class="year">2007</div>
            , 
            <div class="article-title">Patterns of intron gain and conservation in eukaryotic genes</div>
            . 
            <div class="source">BMC Evol. Biol</div> 
            ., 
            <div class="volume">7</div>
            , 
            <div class="fpage">192</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patterns%20of%20intron%20gain%20and%20conservation%20in%20eukaryotic%20genes&amp;author=L.%20Carmel&amp;author=I.B.%20Rogozin&amp;author=Y.I.%20Wolf&amp;author=E.V.%20Koonin&amp;publication_year=2007&amp;journal=BMC%20Evol.%20Biol&amp;volume=7&amp;pages=192" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-7-192" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-7-192" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-7-192"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17935625" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Collins</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Penny</div>
               <div class="given-names">D.</div>
            </div>
          </div>
          <div class="year">2005</div>
            , 
            <div class="article-title">Complex spliceosomal organization ancestral to extant eukaryotes</div>
            . 
            <div class="source">Mol. Biol. Evol</div> 
            ., 
            <div class="volume">22</div>
            , 
            <div class="fpage">1053</div>
            –
            <div class="lpage">66</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complex%20spliceosomal%20organization%20ancestral%20to%20extant%20eukaryotes&amp;author=L.%20Collins&amp;author=D.%20Penny&amp;publication_year=2005&amp;journal=Mol.%20Biol.%20Evol&amp;volume=22&amp;pages=1053-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msi091" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsi091" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsi091"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15659557" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Lane</div>
               <div class="given-names">C.E.</div>
            </div>
             <div class="name">
              <div class="surname">van den Heuvel</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Kozera</div>
               <div class="given-names">C.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2007</div>
            , 
            <div class="article-title">
              Nucleomorph genome of 
              <em>Hemiselmis andersenii</em>
               reveals complete intron loss and compaction as a driver of protein structure and function
            </div>
            . 
            <div class="source">Proc. Natl. Acad. Sci. USA</div> 
            ., 
            <div class="volume">104</div>
            , 
            <div class="fpage">19908</div>
            –
            <div class="lpage">13</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleomorph%20genome%20of%20Hemiselmis%20andersenii%20reveals%20complete%20intron%20loss%20and%20compaction%20as%20a%20driver%20of%20protein%20structure%20and%20function&amp;author=C.E.%20Lane&amp;author=K.%20van%20den%20Heuvel&amp;author=C.%20Kozera&amp;publication_year=2007&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=104&amp;pages=19908-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0707419104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0707419104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0707419104"> </span></div> </div></div></div></div></div><div content-id="dsw019-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Warf</div>
               <div class="given-names">M.B.</div>
            </div>
             <div class="name">
              <div class="surname">Berglund</div>
               <div class="given-names">J.A.</div>
            </div>
          </div>
          <div class="year">2010</div>
            , 
            <div class="article-title">Role of RNA structure in regulating pre-mRNA splicing</div>
            . 
            <div class="source">Trends Biochem. Sci</div> 
            ., 
            <div class="volume">35</div>
            , 
            <div class="fpage">169</div>
            –
            <div class="lpage">78</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20RNA%20structure%20in%20regulating%20pre-mRNA%20splicing&amp;author=M.B.%20Warf&amp;author=J.A.%20Berglund&amp;publication_year=2010&amp;journal=Trends%20Biochem.%20Sci&amp;volume=35&amp;pages=169-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibs.2009.10.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibs.2009.10.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibs.2009.10.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19959365" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Parenteau</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Durand</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Morin</div>
               <div class="given-names">G.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2011</div>
            , 
            <div class="article-title">Introns within ribosomal protein genes regulate the production and function of yeast ribosomes</div>
            . 
            <div class="source">Cell</div> 
            , 
            <div class="volume">147</div>
            , 
            <div class="fpage">320</div>
            –
            <div class="lpage">31</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Introns%20within%20ribosomal%20protein%20genes%20regulate%20the%20production%20and%20function%20of%20yeast%20ribosomes&amp;author=J.%20Parenteau&amp;author=M.%20Durand&amp;author=G.%20Morin&amp;publication_year=2011&amp;journal=Cell&amp;volume=147&amp;pages=320-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2011.08.044" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2011.08.044" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2011.08.044"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22000012" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Berbee</div>
               <div class="given-names">M.L.</div>
            </div>
             <div class="name">
              <div class="surname">Taylor</div>
               <div class="given-names">J.W.</div>
            </div>
          </div>
          <div class="year">2001</div>
            , 
            Fungal molecular evolution: gene trees and geologic time
            . 
            <div class="source">Systematics Evolution</div> 
            , pp. 
            <div class="fpage">229</div>
            –
            <div class="lpage">45</div>
            . 
            <div class="publisher-name">Springer-Verlag</div>
            : 
            <div class="publisher-loc">Berlin Heidelberg</div>
            .
          </div></div></div></div><div content-id="dsw019-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Berbee</div>
               <div class="given-names">M.L.</div>
            </div>
             <div class="name">
              <div class="surname">Taylor</div>
               <div class="given-names">J.W.</div>
            </div>
          </div>
          <div class="year">2010</div>
            , 
            <div class="article-title">Dating the molecular clock in fungi—how close are we?</div><div class="source">Fungal Biol. Rev</div> 
            ., 
            <div class="volume">24</div>
            , 
            <div class="fpage">1</div>
            –
            <div class="lpage">16</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dating%20the%20molecular%20clock%20in%20fungi%E2%80%94how%20close%20are%20we%3F&amp;author=M.L.%20Berbee&amp;author=J.W.%20Taylor&amp;publication_year=2010&amp;journal=Fungal%20Biol.%20Rev&amp;volume=24&amp;pages=1-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fbr.2010.03.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fbr.2010.03.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fbr.2010.03.001"> </span></div> </div></div></div></div></div><div content-id="dsw019-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Taylor</div>
               <div class="given-names">J.W.</div>
            </div>
             <div class="name">
              <div class="surname">Berbee</div>
               <div class="given-names">M.L.</div>
            </div>
          </div>
          <div class="year">2006</div>
            , 
            <div class="article-title">Dating divergences in the Fungal Tree of Life: review and new analyses</div>
            . 
            <div class="source">Mycologia</div> 
            , 
            <div class="volume">98</div>
            , 
            <div class="fpage">838</div>
            –
            <div class="lpage">49</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dating%20divergences%20in%20the%20Fungal%20Tree%20of%20Life%3A%20review%20and%20new%20analyses&amp;author=J.W.%20Taylor&amp;author=M.L.%20Berbee&amp;publication_year=2006&amp;journal=Mycologia&amp;volume=98&amp;pages=838-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/15572536.2006.11832614" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F15572536.2006.11832614" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F15572536.2006.11832614"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17486961" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Cherry</div>
               <div class="given-names">J.M.</div>
            </div>
             <div class="name">
              <div class="surname">Adler</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Ball</div>
               <div class="given-names">C.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">1998</div>
            , 
            <div class="article-title">
              SGD: 
              <em>Saccharomyces</em>
               Genome Database
            </div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">26</div>
            , 
            <div class="fpage">73</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SGD%3A%20Saccharomyces%20Genome%20Database&amp;author=J.M.%20Cherry&amp;author=C.%20Adler&amp;author=C.%20Ball&amp;publication_year=1998&amp;journal=Nucleic%20Acids%20Res&amp;volume=26&amp;pages=73-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.1.73" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.1.73" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.1.73"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9399804" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Wood</div>
               <div class="given-names">V.</div>
            </div>
             <div class="name">
              <div class="surname">Harris</div>
               <div class="given-names">M.A.</div>
            </div>
             <div class="name">
              <div class="surname">McDowall</div>
               <div class="given-names">M.D.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2012</div>
            , 
            <div class="article-title">PomBase: a comprehensive online resource for fission yeast</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">40</div>
            , 
            <div class="fpage">D695</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PomBase%3A%20a%20comprehensive%20online%20resource%20for%20fission%20yeast&amp;author=V.%20Wood&amp;author=M.A.%20Harris&amp;author=M.D.%20McDowall&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=D695-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr853" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr853" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr853"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22039153" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Arnaud</div>
               <div class="given-names">M.B.</div>
            </div>
             <div class="name">
              <div class="surname">Cerqueira</div>
               <div class="given-names">G.C.</div>
            </div>
             <div class="name">
              <div class="surname">Inglis</div>
               <div class="given-names">D.O.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2012</div>
            , 
            <div class="article-title">
              The 
              <em>Aspergillus</em>
               Genome Database (AspGD): recent developments in comprehensive multispecies curation, comparative genomics and community resources
            </div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">40</div>
            , 
            <div class="fpage">D653</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Aspergillus%20Genome%20Database%20(AspGD)%3A%20recent%20developments%20in%20comprehensive%20multispecies%20curation%2C%20comparative%20genomics%20and%20community%20resources&amp;author=M.B.%20Arnaud&amp;author=G.C.%20Cerqueira&amp;author=D.O.%20Inglis&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=D653-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr875" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr875" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr875"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22080559" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Inglis</div>
               <div class="given-names">D.O.</div>
            </div>
             <div class="name">
              <div class="surname">Arnaud</div>
               <div class="given-names">M.B.</div>
            </div>
             <div class="name">
              <div class="surname">Binkley</div>
               <div class="given-names">J.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2012</div>
            , 
            <div class="article-title">
              The 
              <em>Candida</em>
               genome database incorporates multiple 
              <em>Candida</em>
               species: multispecies search and analysis tools with curated gene and protein information for 
              <em>Candida albicans</em>
               and 
              <em>Candida glabrata</em></div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">40</div>
            , 
            <div class="fpage">D667</div>
            –
            <div class="lpage">D674</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Candida%20genome%20database%20incorporates%20multiple%20Candida%20species%3A%20multispecies%20search%20and%20analysis%20tools%20with%20curated%20gene%20and%20protein%20information%20for%20Candida%20albicans%20and%20Candida%20glabrata&amp;author=D.O.%20Inglis&amp;author=M.B.%20Arnaud&amp;author=J.%20Binkley&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=D667-D674" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr945" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr945" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr945"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22064862" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Wang</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Weiss</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Simonovic</div>
               <div class="given-names">M.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2012</div>
            , 
            <div class="article-title">PaxDb, a database of protein abundance averages across all three domains of life</div>
            . 
            <div class="source">Mol. Cell. Proteomics</div> 
            , 
            <div class="volume">11</div>
            , 
            <div class="fpage">492</div>
            –
            <div class="lpage">500</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PaxDb%2C%20a%20database%20of%20protein%20abundance%20averages%20across%20all%20three%20domains%20of%20life&amp;author=M.%20Wang&amp;author=M.%20Weiss&amp;author=M.%20Simonovic&amp;publication_year=2012&amp;journal=Mol.%20Cell.%20Proteomics&amp;volume=11&amp;pages=492-500" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/mcp.O111.014704" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fmcp.O111.014704" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fmcp.O111.014704"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22535208" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Wang</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Liu</div>
               <div class="given-names">C.L.</div>
            </div>
             <div class="name">
              <div class="surname">Storey</div>
               <div class="given-names">J.D.</div>
            </div>
             <div class="name">
              <div class="surname">Tibshirani</div>
               <div class="given-names">R.J.</div>
            </div>
             <div class="name">
              <div class="surname">Herschlag</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Brown</div>
               <div class="given-names">P.O.</div>
            </div>
          </div>
          <div class="year">2002</div>
            , 
            <div class="article-title">Precision and functional specificity in mRNA decay</div>
            . 
            <div class="source">Proc. Natl. Acad. Sci. USA</div> 
            ., 
            <div class="volume">99</div>
            , 
            <div class="fpage">5860</div>
            –
            <div class="lpage">5</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Precision%20and%20functional%20specificity%20in%20mRNA%20decay&amp;author=Y.%20Wang&amp;author=C.L.%20Liu&amp;author=J.D.%20Storey&amp;author=R.J.%20Tibshirani&amp;author=D.%20Herschlag&amp;author=P.O.%20Brown&amp;publication_year=2002&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=5860-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.092538799" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.092538799" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.092538799"> </span></div> </div></div></div></div></div><div content-id="dsw019-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Nagalakshmi</div>
               <div class="given-names">U.</div>
            </div>
             <div class="name">
              <div class="surname">Wang</div>
               <div class="given-names">Z.</div>
            </div>
             <div class="name">
              <div class="surname">Waern</div>
               <div class="given-names">K.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2008</div>
            , 
            <div class="article-title">The transcriptional landscape of the yeast genome defined by RNA sequencing</div>
            . 
            <div class="source">Science</div> 
            , 
            <div class="volume">320</div>
            , 
            <div class="fpage">1344</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20transcriptional%20landscape%20of%20the%20yeast%20genome%20defined%20by%20RNA%20sequencing&amp;author=U.%20Nagalakshmi&amp;author=Z.%20Wang&amp;author=K.%20Waern&amp;publication_year=2008&amp;journal=Science&amp;volume=320&amp;pages=1344-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1158441" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1158441" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1158441"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18451266" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ingolia</div>
               <div class="given-names">N.T.</div>
            </div>
             <div class="name">
              <div class="surname">Ghaemmaghami</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Newman</div>
               <div class="given-names">J.R.S.</div>
            </div>
             <div class="name">
              <div class="surname">Weissman</div>
               <div class="given-names">J.S.</div>
            </div>
          </div>
          <div class="year">2009</div>
            , 
            <div class="article-title">Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling</div>
            . 
            <div class="source">Science</div> 
            , 
            <div class="volume">324</div>
            , 
            <div class="fpage">218</div>
            –
            <div class="lpage">23</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20in%20vivo%20of%20translation%20with%20nucleotide%20resolution%20using%20ribosome%20profiling&amp;author=N.T.%20Ingolia&amp;author=S.%20Ghaemmaghami&amp;author=J.R.S.%20Newman&amp;author=J.S.%20Weissman&amp;publication_year=2009&amp;journal=Science&amp;volume=324&amp;pages=218-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1168978" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1168978" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1168978"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19213877" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Wagner</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">2005</div>
            , 
            <div class="article-title">Energy constraints on the evolution of gene expression</div>
            . 
            <div class="source">Mol. Biol. Evol</div> 
            ., 
            <div class="volume">22</div>
            , 
            <div class="fpage">1365</div>
            –
            <div class="lpage">74</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Energy%20constraints%20on%20the%20evolution%20of%20gene%20expression&amp;author=A.%20Wagner&amp;publication_year=2005&amp;journal=Mol.%20Biol.%20Evol&amp;volume=22&amp;pages=1365-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msi126" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsi126" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsi126"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15758206" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sharp</div>
               <div class="given-names">P.M.</div>
            </div>
             <div class="name">
              <div class="surname">Li</div>
               <div class="given-names">W.H.</div>
            </div>
          </div>
          <div class="year">1987</div>
            , 
            <div class="article-title">The codon adaptation index-a measure of directional synonymous codon usage bias, and its potential applications</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">15</div>
            , 
            <div class="fpage">1281</div>
            –
            <div class="lpage">95</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20codon%20adaptation%20index-a%20measure%20of%20directional%20synonymous%20codon%20usage%20bias%2C%20and%20its%20potential%20applications&amp;author=P.M.%20Sharp&amp;author=W.H.%20Li&amp;publication_year=1987&amp;journal=Nucleic%20Acids%20Res&amp;volume=15&amp;pages=1281-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/15.3.1281" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F15.3.1281" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F15.3.1281"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3547335" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Dana</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Tuller</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">2014</div>
            , 
            <div class="article-title">The effect of tRNA levels on decoding times of mRNA codons</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">42</div>
            , 
            <div class="fpage">9171</div>
            –
            <div class="lpage">81</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20effect%20of%20tRNA%20levels%20on%20decoding%20times%20of%20mRNA%20codons&amp;author=A.%20Dana&amp;author=T.%20Tuller&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=9171-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku646" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku646" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku646"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25056313" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Dana</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Tuller</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">2015</div>
            , 
            <div class="article-title">Mean of the typical decoding rates: a new translation efficiency index based on the analysis of ribosome profiling data</div>
            . 
            <div class="source">G3</div> 
            , 
            <div class="volume">5</div>
            , 
            <div class="fpage">73</div>
            –
            <div class="lpage">80</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mean%20of%20the%20typical%20decoding%20rates%3A%20a%20new%20translation%20efficiency%20index%20based%20on%20the%20analysis%20of%20ribosome%20profiling%20data&amp;author=A.%20Dana&amp;author=T.%20Tuller&amp;publication_year=2015&amp;journal=G3&amp;volume=5&amp;pages=73-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.114.015099" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.114.015099" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.114.015099"> </span></div> </div></div></div></div></div><div content-id="dsw019-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Tuller</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Waldman</div>
               <div class="given-names">Y.Y.</div>
            </div>
             <div class="name">
              <div class="surname">Kupiec</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Ruppin</div>
               <div class="given-names">E.</div>
            </div>
          </div>
          <div class="year">2010</div>
            , 
            <div class="article-title">Translation efficiency is determined by both codon bias and folding energy</div>
            . 
            <div class="source">Proc. Natl. Acad. Sci. USA</div> 
            ., 
            <div class="volume">107</div>
            , 
            <div class="fpage">3645</div>
            –
            <div class="lpage">50</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translation%20efficiency%20is%20determined%20by%20both%20codon%20bias%20and%20folding%20energy&amp;author=T.%20Tuller&amp;author=Y.Y.%20Waldman&amp;author=M.%20Kupiec&amp;author=E.%20Ruppin&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=3645-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0909910107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0909910107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0909910107"> </span></div> </div></div></div></div></div><div content-id="dsw019-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Chan</div>
               <div class="given-names">P.P.</div>
            </div>
             <div class="name">
              <div class="surname">Lowe</div>
               <div class="given-names">T.M.</div>
            </div>
          </div>
          <div class="year">2009</div>
            , 
            <div class="article-title">GtRNAdb: a database of transfer RNA genes detected in genomic sequence</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">37</div>
            , 
            <div class="fpage">D93</div>
            –
            <div class="lpage">7</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GtRNAdb%3A%20a%20database%20of%20transfer%20RNA%20genes%20detected%20in%20genomic%20sequence&amp;author=P.P.%20Chan&amp;author=T.M.%20Lowe&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res&amp;volume=37&amp;pages=D93-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn787" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn787" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn787"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18984615" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Man</div>
               <div class="given-names">O.</div>
            </div>
             <div class="name">
              <div class="surname">Pilpel</div>
               <div class="given-names">Y.</div>
            </div>
          </div>
          <div class="year">2007</div>
            , 
            <div class="article-title">Differential translation efficiency of orthologous genes is involved in phenotypic divergence of yeast species</div>
            . 
            <div class="source">Nat. Genet</div> 
            ., 
            <div class="volume">39</div>
            , 
            <div class="fpage">415</div>
            –
            <div class="lpage">21</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20translation%20efficiency%20of%20orthologous%20genes%20is%20involved%20in%20phenotypic%20divergence%20of%20yeast%20species&amp;author=O.%20Man&amp;author=Y.%20Pilpel&amp;publication_year=2007&amp;journal=Nat.%20Genet&amp;volume=39&amp;pages=415-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1967" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1967" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1967"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17277776" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sabi</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Tuller</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">2014</div>
            , 
            <div class="article-title">Modelling the efficiency of codon–tRNA interactions based on codon usage bias</div>
            . 
            <div class="source">DNA Res</div> 
            ., 
            <div class="volume">21</div>
            , 
            <div class="fpage">511</div>
            –
            <div class="lpage">26</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Modelling%20the%20efficiency%20of%20codon%E2%80%93tRNA%20interactions%20based%20on%20codon%20usage%20bias&amp;author=R.%20Sabi&amp;author=T.%20Tuller&amp;publication_year=2014&amp;journal=DNA%20Res&amp;volume=21&amp;pages=511-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsu017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsu017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsu017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24906480" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Edgar</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Domrachev</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Lash</div>
               <div class="given-names">A.E.</div>
            </div>
          </div>
          <div class="year">2002</div>
            , 
            <div class="article-title">Gene expression omnibus: NCBI gene expression and hybridization array data repository</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">30</div>
            , 
            <div class="fpage">207</div>
            –
            <div class="lpage">10</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20expression%20omnibus%3A%20NCBI%20gene%20expression%20and%20hybridization%20array%20data%20repository&amp;author=R.%20Edgar&amp;author=M.%20Domrachev&amp;author=A.E.%20Lash&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res&amp;volume=30&amp;pages=207-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/30.1.207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F30.1.207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F30.1.207"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11752295" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ingolia</div>
               <div class="given-names">N.T.</div>
            </div>
          </div>
          <div class="year">2010</div>
            , 
            Chapter 6—genome-wide translational profiling by ribosome footprinting
            . In: 
            <div class="person-group"><div class="name"><div class="surname">Jonathan</div> <div class="given-names">W.</div></div> <div class="name"><div class="surname">Christine</div> <div class="given-names">G.</div></div> <div class="name"><div class="surname">Gerald</div> <div class="given-names">R.F.</div></div></div>
             (eds), 
            <div class="source">Methods in Enzymology</div> 
            , Vol. 470, pp. 
            <div class="fpage">119</div>
            –
            <div class="lpage">42</div>
            . 
            <div class="publisher-name">Elsevier inc</div>
            .
          </div></div></div></div><div content-id="dsw019-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kasprzyk</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">2011</div>
            , 
            <div class="article-title">BioMart: driving a paradigm change in biological data management</div>
            . 
            <div class="source">Database</div> 
            , 
            <div class="volume">2011</div>
            , 
            <div class="fpage">bar049</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BioMart%3A%20driving%20a%20paradigm%20change%20in%20biological%20data%20management&amp;author=A.%20Kasprzyk&amp;publication_year=2011&amp;journal=Database&amp;volume=2011&amp;pages=bar049" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/database/bar049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdatabase%2Fbar049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdatabase%2Fbar049"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22083790" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Langmead</div>
               <div class="given-names">B.</div>
            </div>
             <div class="name">
              <div class="surname">Salzberg</div>
               <div class="given-names">S.L.</div>
            </div>
          </div>
          <div class="year">2012</div>
            , 
            <div class="article-title">Fast gapped-read alignment with Bowtie 2</div>
            . 
            <div class="source">Nat. Methods</div> 
            ., 
            <div class="volume">9</div>
            , 
            <div class="fpage">357</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B.%20Langmead&amp;author=S.L.%20Salzberg&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=357-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ghaemmaghami</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Huh</div>
               <div class="given-names">W.K.</div>
            </div>
             <div class="name">
              <div class="surname">Bower</div>
               <div class="given-names">K.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2003</div>
            , 
            <div class="article-title">Global analysis of protein expression in yeast</div>
            . 
            <div class="source">Nature</div> 
            , 
            <div class="volume">425</div>
            , 
            <div class="fpage">737</div>
            –
            <div class="lpage">41</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20analysis%20of%20protein%20expression%20in%20yeast&amp;author=S.%20Ghaemmaghami&amp;author=W.K.%20Huh&amp;author=K.%20Bower&amp;publication_year=2003&amp;journal=Nature&amp;volume=425&amp;pages=737-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02046" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02046" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02046"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14562106" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Yofe</div>
               <div class="given-names">I.</div>
            </div>
             <div class="name">
              <div class="surname">Zafrir</div>
               <div class="given-names">Z.</div>
            </div>
             <div class="name">
              <div class="surname">Blau</div>
               <div class="given-names">R.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2014</div>
            , 
            <div class="article-title">
              Accurate, model-based tuning of synthetic gene expression using introns in 
              <em>S. cerevisiae</em></div>
            . 
            <div class="source">PLoS Genet</div> 
            ., 
            <div class="volume">10</div>
            , 
            <div class="fpage">e1004407</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accurate%2C%20model-based%20tuning%20of%20synthetic%20gene%20expression%20using%20introns%20in%20S.%20cerevisiae&amp;author=I.%20Yofe&amp;author=Z.%20Zafrir&amp;author=R.%20Blau&amp;publication_year=2014&amp;journal=PLoS%20Genet&amp;volume=10&amp;pages=e1004407" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1004407" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1004407" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1004407"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24968317" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Linshiz</div>
               <div class="given-names">G.</div>
            </div>
             <div class="name">
              <div class="surname">Yehezkel</div>
               <div class="given-names">T.B.</div>
            </div>
             <div class="name">
              <div class="surname">Kaplan</div>
               <div class="given-names">S.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2008</div>
            , 
            <div class="article-title">Recursive construction of perfect DNA molecules from imperfect oligonucleotides</div>
            . 
            <div class="source">Mol. Syst. Biol</div> 
            ., 
            <div class="volume">4</div>
            , 
            <div class="fpage">191</div>
            –
            <div class="lpage">200</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recursive%20construction%20of%20perfect%20DNA%20molecules%20from%20imperfect%20oligonucleotides&amp;author=G.%20Linshiz&amp;author=T.B.%20Yehezkel&amp;author=S.%20Kaplan&amp;publication_year=2008&amp;journal=Mol.%20Syst.%20Biol&amp;volume=4&amp;pages=191-200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb.2008.26" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb.2008.26" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb.2008.26"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18463615" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B76" class="ref-content "><div class="label -label">76</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Shabi</div>
               <div class="given-names">U.</div>
            </div>
             <div class="name">
              <div class="surname">Kaplan</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Linshiz</div>
               <div class="given-names">G.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2010</div>
            , 
            <div class="article-title">Processing DNA molecules as text</div>
            . 
            <div class="source">Syst. Synth. Biol</div> 
            ., 
            <div class="volume">4</div>
            , 
            <div class="fpage">227</div>
            –
            <div class="lpage">36</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Processing%20DNA%20molecules%20as%20text&amp;author=U.%20Shabi&amp;author=S.%20Kaplan&amp;author=G.%20Linshiz&amp;publication_year=2010&amp;journal=Syst.%20Synth.%20Biol&amp;volume=4&amp;pages=227-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11693-010-9059-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11693-010-9059-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11693-010-9059-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21189843" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B77" class="ref-content "><div class="label -label">77</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kudla</div>
               <div class="given-names">G.</div>
            </div>
             <div class="name">
              <div class="surname">Murray</div>
               <div class="given-names">A.W.</div>
            </div>
             <div class="name">
              <div class="surname">Tollervey</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Plotkin</div>
               <div class="given-names">J.B.</div>
            </div>
          </div>
          <div class="year">2009</div>
            , 
            <div class="article-title">
              Coding-sequence determinants of gene expression in 
              <em>Escherichia coli</em></div>
            . 
            <div class="source">Science</div> 
            , 
            <div class="volume">324</div>
            , 
            <div class="fpage">255</div>
            –
            <div class="lpage">8</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coding-sequence%20determinants%20of%20gene%20expression%20in%20Escherichia%20coli&amp;author=G.%20Kudla&amp;author=A.W.%20Murray&amp;author=D.%20Tollervey&amp;author=J.B.%20Plotkin&amp;publication_year=2009&amp;journal=Science&amp;volume=324&amp;pages=255-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1170160" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1170160" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1170160"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19359587" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B78" class="ref-content "><div class="label -label">78</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">dos Reis</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Wernisch</div>
               <div class="given-names">L.</div>
            </div>
          </div>
          <div class="year">2009</div>
            , 
            <div class="article-title">
              Estimating 
              <em>Translational Selection in Eukaryotic Genomes</em></div>
            . 
            <div class="source">Mol. Biol. Evol</div> 
            ., 
            <div class="volume">26</div>
            , 
            <div class="fpage">451</div>
            –
            <div class="lpage">61</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimating%20Translational%20Selection%20in%20Eukaryotic%20Genomes&amp;author=M.%20dos%20Reis&amp;author=L.%20Wernisch&amp;publication_year=2009&amp;journal=Mol.%20Biol.%20Evol&amp;volume=26&amp;pages=451-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msn272" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsn272" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsn272"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19033257" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B79" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B79" class="ref-content "><div class="label -label">79</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Tuller</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Carmi</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Vestsigian</div>
               <div class="given-names">K.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2010</div>
            , 
            <div class="article-title">An evolutionarily conserved mechanism for controlling the efficiency of protein translation</div>
            . 
            <div class="source">Cell</div> 
            , 
            <div class="volume">141</div>
            , 
            <div class="fpage">344</div>
            –
            <div class="lpage">54</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20evolutionarily%20conserved%20mechanism%20for%20controlling%20the%20efficiency%20of%20protein%20translation&amp;author=T.%20Tuller&amp;author=A.%20Carmi&amp;author=K.%20Vestsigian&amp;publication_year=2010&amp;journal=Cell&amp;volume=141&amp;pages=344-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2010.03.031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2010.03.031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2010.03.031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20403328" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B80" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B80" class="ref-content "><div class="label -label">80</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Cannarozzi</div>
               <div class="given-names">G.</div>
            </div>
             <div class="name">
              <div class="surname">Schraudolph</div>
               <div class="given-names">N.N.</div>
            </div>
             <div class="name">
              <div class="surname">Faty</div>
               <div class="given-names">M.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2010</div>
            , 
            <div class="article-title">A role for codon order in translation dynamics</div>
            . 
            <div class="source">Cell</div> 
            , 
            <div class="volume">141</div>
            , 
            <div class="fpage">355</div>
            –
            <div class="lpage">67</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20role%20for%20codon%20order%20in%20translation%20dynamics&amp;author=G.%20Cannarozzi&amp;author=N.N.%20Schraudolph&amp;author=M.%20Faty&amp;publication_year=2010&amp;journal=Cell&amp;volume=141&amp;pages=355-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2010.02.036" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2010.02.036" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2010.02.036"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20403329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B81" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B81" class="ref-content "><div class="label -label">81</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Novoa</div>
               <div class="given-names">E.M.</div>
            </div>
             <div class="name">
              <div class="surname">Ribas de Pouplana</div>
               <div class="given-names">L.</div>
            </div>
          </div>
          <div class="year">2012</div>
            , 
            <div class="article-title">Speeding with control: codon usage, tRNAs, and ribosomes</div>
            . 
            <div class="source">Trends Genet</div> 
            ., 
            <div class="volume">28</div>
            , 
            <div class="fpage">574</div>
            –
            <div class="lpage">81</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Speeding%20with%20control%3A%20codon%20usage%2C%20tRNAs%2C%20and%20ribosomes&amp;author=E.M.%20Novoa&amp;author=L.%20Ribas%20de%20Pouplana&amp;publication_year=2012&amp;journal=Trends%20Genet&amp;volume=28&amp;pages=574-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2012.07.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2012.07.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2012.07.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22921354" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B82" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B82" class="ref-content "><div class="label -label">82</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Hershberg</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Petrov</div>
               <div class="given-names">D.A.</div>
            </div>
          </div>
          <div class="year">2008</div>
            , 
            <div class="article-title">Selection on codon bias</div>
            . 
            <div class="source">Annu. Rev. Genet</div> 
            ., 
            <div class="volume">42</div>
            , 
            <div class="fpage">287</div>
            –
            <div class="lpage">99</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selection%20on%20codon%20bias&amp;author=R.%20Hershberg&amp;author=D.A.%20Petrov&amp;publication_year=2008&amp;journal=Annu.%20Rev.%20Genet&amp;volume=42&amp;pages=287-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.42.110807.091442" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.42.110807.091442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.42.110807.091442"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18983258" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B83" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B83" class="ref-content "><div class="label -label">83</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Plotkin</div>
               <div class="given-names">J.B.</div>
            </div>
             <div class="name">
              <div class="surname">Kudla</div>
               <div class="given-names">G.</div>
            </div>
          </div>
          <div class="year">2011</div>
            , 
            <div class="article-title">Synonymous but not the same: the causes and consequences of codon bias</div>
            . 
            <div class="source">Nat. Rev. Genet</div> 
            ., 
            <div class="volume">12</div>
            , 
            <div class="fpage">32</div>
            –
            <div class="lpage">42</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20but%20not%20the%20same%3A%20the%20causes%20and%20consequences%20of%20codon%20bias&amp;author=J.B.%20Plotkin&amp;author=G.%20Kudla&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet&amp;volume=12&amp;pages=32-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2899" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2899" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2899"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21102527" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B84" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B84" class="ref-content "><div class="label -label">84</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Reis</div>
               <div class="given-names">M.D.</div>
            </div>
             <div class="name">
              <div class="surname">Savva</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Wernisch</div>
               <div class="given-names">L.</div>
            </div>
          </div>
          <div class="year">2004</div>
            , 
            <div class="article-title">Solving the riddle of codon usage preferences: a test for translational selection</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">32</div>
            , 
            <div class="fpage">5036</div>
            –
            <div class="lpage">44</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Solving%20the%20riddle%20of%20codon%20usage%20preferences%3A%20a%20test%20for%20translational%20selection&amp;author=M.D.%20Reis&amp;author=R.%20Savva&amp;author=L.%20Wernisch&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;pages=5036-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh834" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh834" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh834"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15448185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B85" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B85" class="ref-content "><div class="label -label">85</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Jeffares</div>
               <div class="given-names">D.C.</div>
            </div>
             <div class="name">
              <div class="surname">Mourier</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Penny</div>
               <div class="given-names">D.</div>
            </div>
          </div>
          <div class="year">2006</div>
            , 
            <div class="article-title">The biology of intron gain and loss</div>
            . 
            <div class="source">Trends Genet</div> 
            ., 
            <div class="volume">22</div>
            , 
            <div class="fpage">16</div>
            –
            <div class="lpage">22</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20biology%20of%20intron%20gain%20and%20loss&amp;author=D.C.%20Jeffares&amp;author=T.%20Mourier&amp;author=D.%20Penny&amp;publication_year=2006&amp;journal=Trends%20Genet&amp;volume=22&amp;pages=16-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2005.10.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2005.10.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2005.10.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16290250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B86" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B86" class="ref-content "><div class="label -label">86</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Cohen</div>
               <div class="given-names">N.E.</div>
            </div>
             <div class="name">
              <div class="surname">Shen</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Carmel</div>
               <div class="given-names">L.</div>
            </div>
          </div>
          <div class="year">2012</div>
            , 
            <div class="article-title">The role of reverse transcriptase in intron gain and loss mechanisms</div>
            . 
            <div class="source">Mol. Biol. Evol</div> 
            ., 
            <div class="volume">29</div>
            , 
            <div class="fpage">179</div>
            –
            <div class="lpage">86</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20reverse%20transcriptase%20in%20intron%20gain%20and%20loss%20mechanisms&amp;author=N.E.%20Cohen&amp;author=R.%20Shen&amp;author=L.%20Carmel&amp;publication_year=2012&amp;journal=Mol.%20Biol.%20Evol&amp;volume=29&amp;pages=179-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr192" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr192" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr192"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21804076" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B87" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B87" class="ref-content "><div class="label -label">87</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ikemura</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">1985</div>
            , 
            <div class="article-title">Codon usage and tRNA content in unicellular and multicellular organisms</div>
            . 
            <div class="source">Mol. Biol. Evol</div> 
            ., 
            <div class="volume">2</div>
            , 
            <div class="fpage">13</div>
            –
            <div class="lpage">34</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20and%20tRNA%20content%20in%20unicellular%20and%20multicellular%20organisms&amp;author=T.%20Ikemura&amp;publication_year=1985&amp;journal=Mol.%20Biol.%20Evol&amp;volume=2&amp;pages=13-34" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3916708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B88" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B88" class="ref-content "><div class="label -label">88</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kurland</div>
               <div class="given-names">C.G.</div>
            </div>
          </div>
          <div class="year">1991</div>
            , 
            <div class="article-title">Codon bias and gene expression</div>
            . 
            <div class="source">FEBS Lett</div> 
            ., 
            <div class="volume">285</div>
            , 
            <div class="fpage">165</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20bias%20and%20gene%20expression&amp;author=C.G.%20Kurland&amp;publication_year=1991&amp;journal=FEBS%20Lett&amp;volume=285&amp;pages=165-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0014-5793(91)80797-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0014-5793(91)80797-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0014-5793(91)80797-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1855585" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B89" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B89" class="ref-content "><div class="label -label">89</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Pickrell</div>
               <div class="given-names">J.K.</div>
            </div>
             <div class="name">
              <div class="surname">Pai</div>
               <div class="given-names">A.A.</div>
            </div>
             <div class="name">
              <div class="surname">Gilad</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Pritchard</div>
               <div class="given-names">J.K.</div>
            </div>
          </div>
          <div class="year">2010</div>
            , 
            <div class="article-title">Noisy splicing drives mRNA isoform diversity in human cells</div>
            . 
            <div class="source">PLoS Genet</div> 
            ., 
            <div class="volume">6</div>
            , 
            <div class="fpage">e1001236</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Noisy%20splicing%20drives%20mRNA%20isoform%20diversity%20in%20human%20cells&amp;author=J.K.%20Pickrell&amp;author=A.A.%20Pai&amp;author=Y.%20Gilad&amp;author=J.K.%20Pritchard&amp;publication_year=2010&amp;journal=PLoS%20Genet&amp;volume=6&amp;pages=e1001236" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1001236" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1001236" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1001236"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21151575" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B90" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B90" class="ref-content "><div class="label -label">90</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Eberle</div>
               <div class="given-names">A.B.</div>
            </div>
             <div class="name">
              <div class="surname">Stalder</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Mathys</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Orozco</div>
               <div class="given-names">R.Z.</div>
            </div>
             <div class="name">
              <div class="surname">Mühlemann</div>
               <div class="given-names">O.</div>
            </div>
          </div>
          <div class="year">2008</div>
            , 
            <div class="article-title">Posttranscriptional gene regulation by spatial rearrangement of the 3? Untranslated region</div>
            . 
            <div class="source">PLoS Biol</div> 
            ., 
            <div class="volume">6</div>
            , 
            <div class="fpage">e92</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Posttranscriptional%20gene%20regulation%20by%20spatial%20rearrangement%20of%20the%203%3F%20Untranslated%20region&amp;author=A.B.%20Eberle&amp;author=L.%20Stalder&amp;author=H.%20Mathys&amp;author=R.Z.%20Orozco&amp;author=O.%20M%C3%BChlemann&amp;publication_year=2008&amp;journal=PLoS%20Biol&amp;volume=6&amp;pages=e92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0060092" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0060092" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0060092"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18447580" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B91" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B91" class="ref-content "><div class="label -label">91</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Rebbapragada</div>
               <div class="given-names">I.</div>
            </div>
             <div class="name">
              <div class="surname">Lykke-Andersen</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="year">2009</div>
            , 
            <div class="article-title">Execution of nonsense-mediated mRNA decay: what defines a substrate?</div><div class="source">Curr. Opin. Cell Biol</div> 
            ., 
            <div class="volume">21</div>
            , 
            <div class="fpage">394</div>
            –
            <div class="lpage">402</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Execution%20of%20nonsense-mediated%20mRNA%20decay%3A%20what%20defines%20a%20substrate%3F&amp;author=I.%20Rebbapragada&amp;author=J.%20Lykke-Andersen&amp;publication_year=2009&amp;journal=Curr.%20Opin.%20Cell%20Biol&amp;volume=21&amp;pages=394-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ceb.2009.02.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ceb.2009.02.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ceb.2009.02.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19359157" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B92" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B92" class="ref-content "><div class="label -label">92</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kurosaki</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Maquat</div>
               <div class="given-names">L.E.</div>
            </div>
          </div>
          <div class="year">2013</div>
            , 
            <div class="article-title">Rules that govern UPF1 binding to mRNA 3′ UTRs</div>
            . 
            <div class="source">Proc. Natl. Acad. Sci. USA</div> 
            ., 
            <div class="volume">110</div>
            , 
            <div class="fpage">3357</div>
            –
            <div class="lpage">62</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rules%20that%20govern%20UPF1%20binding%20to%20mRNA%203%E2%80%B2%20UTRs&amp;author=T.%20Kurosaki&amp;author=L.E.%20Maquat&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=110&amp;pages=3357-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1219908110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1219908110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1219908110"> </span></div> </div></div></div></div></div><div content-id="dsw019-B93" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B93" class="ref-content "><div class="label -label">93</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Hurt</div>
               <div class="given-names">J.A.</div>
            </div>
             <div class="name">
              <div class="surname">Robertson</div>
               <div class="given-names">A.D.</div>
            </div>
             <div class="name">
              <div class="surname">Burge</div>
               <div class="given-names">C.B.</div>
            </div>
          </div>
          <div class="year">2013</div>
            , 
            <div class="article-title">Global analyses of UPF1 binding and function reveal expanded scope of nonsense-mediated mRNA decay</div>
            . 
            <div class="source">Genome Res</div> 
            ., 
            <div class="volume">23</div>
            , 
            <div class="fpage">1636</div>
            –
            <div class="lpage">50</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20analyses%20of%20UPF1%20binding%20and%20function%20reveal%20expanded%20scope%20of%20nonsense-mediated%20mRNA%20decay&amp;author=J.A.%20Hurt&amp;author=A.D.%20Robertson&amp;author=C.B.%20Burge&amp;publication_year=2013&amp;journal=Genome%20Res&amp;volume=23&amp;pages=1636-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.157354.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.157354.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.157354.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23766421" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B94" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B94" class="ref-content "><div class="label -label">94</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Toma</div>
               <div class="given-names">K.G.</div>
            </div>
             <div class="name">
              <div class="surname">Rebbapragada</div>
               <div class="given-names">I.</div>
            </div>
             <div class="name">
              <div class="surname">Durand</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Lykke-Andersen</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="year">2015</div>
            , 
            <div class="article-title">Identification of elements in human long 3′ UTRs that inhibit nonsense-mediated decay</div>
            . 
            <div class="source">RNA</div> 
            , 
            <div class="volume">21</div>
            , 
            <div class="fpage">887</div>
            –
            <div class="lpage">97</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20elements%20in%20human%20long%203%E2%80%B2%20UTRs%20that%20inhibit%20nonsense-mediated%20decay&amp;author=K.G.%20Toma&amp;author=I.%20Rebbapragada&amp;author=S.%20Durand&amp;author=J.%20Lykke-Andersen&amp;publication_year=2015&amp;journal=RNA&amp;volume=21&amp;pages=887-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.048637.114" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.048637.114" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.048637.114"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25805855" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B95" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B95" class="ref-content "><div class="label -label">95</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Hug</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Longman</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Cáceres</div>
               <div class="given-names">J.F.</div>
            </div>
          </div>
          <div class="year">2016</div>
            , 
            <div class="article-title">Mechanism and regulation of the nonsense-mediated decay pathway</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">44</div>
            , 
            <div class="fpage">1483</div>
            –
            <div class="lpage">95</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanism%20and%20regulation%20of%20the%20nonsense-mediated%20decay%20pathway&amp;author=N.%20Hug&amp;author=D.%20Longman&amp;author=J.F.%20C%C3%A1ceres&amp;publication_year=2016&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;pages=1483-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkw010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkw010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkw010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26773057" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B96" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B96" class="ref-content "><div class="label -label">96</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Supek</div>
               <div class="given-names">F.</div>
            </div>
             <div class="name">
              <div class="surname">Šmuc</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">2010</div>
            , 
            <div class="article-title">
              On relevance of codon usage to expression of synthetic and natural genes in 
              <em>Escherichia coli</em></div>
            . 
            <div class="source">Genetics</div> 
            , 
            <div class="volume">185</div>
            , 
            <div class="fpage">1129</div>
            –
            <div class="lpage">34</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On%20relevance%20of%20codon%20usage%20to%20expression%20of%20synthetic%20and%20natural%20genes%20in%20Escherichia%20coli&amp;author=F.%20Supek&amp;author=T.%20%C5%A0muc&amp;publication_year=2010&amp;journal=Genetics&amp;volume=185&amp;pages=1129-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.110.115477" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.110.115477" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.110.115477"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20421604" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B97" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B97" class="ref-content "><div class="label -label">97</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ingolia</div>
               <div class="given-names">N.T.</div>
            </div>
             <div class="name">
              <div class="surname">Lareau</div>
               <div class="given-names">L.F.</div>
            </div>
             <div class="name">
              <div class="surname">Weissman</div>
               <div class="given-names">J.S.</div>
            </div>
          </div>
          <div class="year">2011</div>
            , 
            <div class="article-title">Ribosome profiling of mouse embryonic stem cells reveals the complexity and dynamics of mammalian proteomes</div>
            . 
            <div class="source">Cell</div> 
            , 
            <div class="volume">147</div>
            , 
            <div class="fpage">789</div>
            –
            <div class="lpage">802</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ribosome%20profiling%20of%20mouse%20embryonic%20stem%20cells%20reveals%20the%20complexity%20and%20dynamics%20of%20mammalian%20proteomes&amp;author=N.T.%20Ingolia&amp;author=L.F.%20Lareau&amp;author=J.S.%20Weissman&amp;publication_year=2011&amp;journal=Cell&amp;volume=147&amp;pages=789-802" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2011.10.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2011.10.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2011.10.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22056041" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B98" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B98" class="ref-content "><div class="label -label">98</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Li</div>
               <div class="given-names">G.W.</div>
            </div>
             <div class="name">
              <div class="surname">Oh</div>
               <div class="given-names">E.</div>
            </div>
             <div class="name">
              <div class="surname">Weissman</div>
               <div class="given-names">J.S.</div>
            </div>
          </div>
          <div class="year">2012</div>
            , 
            <div class="article-title">The anti-Shine-Dalgarno sequence drives translational pausing and codon choice in bacteria</div>
            . 
            <div class="source">Nature</div> 
            , 
            <div class="volume">484</div>
            , 
            <div class="fpage">538</div>
            –
            <div class="lpage">41</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20anti-Shine-Dalgarno%20sequence%20drives%20translational%20pausing%20and%20codon%20choice%20in%20bacteria&amp;author=G.W.%20Li&amp;author=E.%20Oh&amp;author=J.S.%20Weissman&amp;publication_year=2012&amp;journal=Nature&amp;volume=484&amp;pages=538-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10965" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10965" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10965"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22456704" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B99" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B99" class="ref-content "><div class="label -label">99</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Qian</div>
               <div class="given-names">W.</div>
            </div>
             <div class="name">
              <div class="surname">Yang</div>
               <div class="given-names">J.-R.</div>
            </div>
             <div class="name">
              <div class="surname">Pearson</div>
               <div class="given-names">N.M.</div>
            </div>
             <div class="name">
              <div class="surname">Maclean</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Zhang</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="year">2012</div>
            , 
            <div class="article-title">Balanced codon usage optimizes eukaryotic translational efficiency</div>
            . 
            <div class="source">PLoS Genet</div> 
            ., 
            <div class="volume">8</div>
            , 
            <div class="fpage">e1002603</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Balanced%20codon%20usage%20optimizes%20eukaryotic%20translational%20efficiency&amp;author=W.%20Qian&amp;author=J.-R.%20Yang&amp;author=N.M.%20Pearson&amp;author=C.%20Maclean&amp;author=J.%20Zhang&amp;publication_year=2012&amp;journal=PLoS%20Genet&amp;volume=8&amp;pages=e1002603" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002603" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002603" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002603"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22479199" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B100" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B100" class="ref-content "><div class="label -label">100</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Dana</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Tuller</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">2012</div>
            , 
            <div class="article-title">Determinants of translation elongation speed and ribosomal profiling biases in mouse embryonic stem cells</div>
            . 
            <div class="source">PLoS Comput. Biol</div> 
            ., 
            <div class="volume">8</div>
            , 
            <div class="fpage">e1002755</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Determinants%20of%20translation%20elongation%20speed%20and%20ribosomal%20profiling%20biases%20in%20mouse%20embryonic%20stem%20cells&amp;author=A.%20Dana&amp;author=T.%20Tuller&amp;publication_year=2012&amp;journal=PLoS%20Comput.%20Biol&amp;volume=8&amp;pages=e1002755" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1002755" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1002755" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1002755"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23133360" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B101" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B101" class="ref-content "><div class="label -label">101</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Charneski</div>
               <div class="given-names">C.A.</div>
            </div>
             <div class="name">
              <div class="surname">Hurst</div>
               <div class="given-names">L.D.</div>
            </div>
          </div>
          <div class="year">2013</div>
            , 
            <div class="article-title">Positively charged residues are the major determinants of ribosomal velocity</div>
            . 
            <div class="source">PLoS Biol</div> 
            ., 
            <div class="volume">11</div>
            , 
            <div class="fpage">e1001508</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Positively%20charged%20residues%20are%20the%20major%20determinants%20of%20ribosomal%20velocity&amp;author=C.A.%20Charneski&amp;author=L.D.%20Hurst&amp;publication_year=2013&amp;journal=PLoS%20Biol&amp;volume=11&amp;pages=e1001508" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.1001508" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.1001508" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.1001508"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23554576" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B102" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B102" class="ref-content "><div class="label -label">102</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Artieri</div>
               <div class="given-names">C.G.</div>
            </div>
             <div class="name">
              <div class="surname">Fraser</div>
               <div class="given-names">H.B.</div>
            </div>
          </div>
          <div class="year">2014</div>
            , 
            <div class="article-title">Evolution at two levels of gene expression in yeast</div>
            . 
            <div class="source">Genome Res</div> 
            ., 
            <div class="volume">24</div>
            , 
            <div class="fpage">411</div>
            –
            <div class="lpage">21</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20at%20two%20levels%20of%20gene%20expression%20in%20yeast&amp;author=C.G.%20Artieri&amp;author=H.B.%20Fraser&amp;publication_year=2014&amp;journal=Genome%20Res&amp;volume=24&amp;pages=411-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.165522.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.165522.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.165522.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24318729" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B103" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B103" class="ref-content "><div class="label -label">103</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Gardin</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Yeasmin</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Yurovsky</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Cai</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Skiena</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Futcher</div>
               <div class="given-names">B.</div>
            </div>
          </div>
          <div class="year">2014</div>
            , 
            <div class="article-title">Measurement of average decoding rates of the 61 sense codons in vivo</div>
            . 
            <div class="source">eLife</div> 
            , 
            <div class="volume">3</div>
            .
          </div></div></div></div><div content-id="dsw019-B104" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B104" class="ref-content "><div class="label -label">104</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Weinberg</div>
               <div class="given-names">D.E.</div>
            </div>
             <div class="name">
              <div class="surname">Shah</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">Eichhorn</div>
               <div class="given-names">S.W.</div>
            </div>
             <div class="name">
              <div class="surname">Hussmann</div>
               <div class="given-names">J.A.</div>
            </div>
             <div class="name">
              <div class="surname">Plotkin</div>
               <div class="given-names">J.B.</div>
            </div>
             <div class="name">
              <div class="surname">Bartel</div>
               <div class="given-names">D.P.</div>
            </div>
          </div>
          <div class="year">2015</div>
            , 
            <div class="article-title">Improved ribosome-footprint and mRNA measurements provide insights into dynamics and regulation of yeast translation</div>
            . 
            <div class="source">Cell Rep</div> 
            ., 
            <div class="volume">14</div>
            , 
            <div class="fpage">1787</div>
            –
            <div class="lpage">99</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20ribosome-footprint%20and%20mRNA%20measurements%20provide%20insights%20into%20dynamics%20and%20regulation%20of%20yeast%20translation&amp;author=D.E.%20Weinberg&amp;author=P.%20Shah&amp;author=S.W.%20Eichhorn&amp;author=J.A.%20Hussmann&amp;author=J.B.%20Plotkin&amp;author=D.P.%20Bartel&amp;publication_year=2015&amp;journal=Cell%20Rep&amp;volume=14&amp;pages=1787-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.celrep.2016.01.043" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.celrep.2016.01.043" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.celrep.2016.01.043"> </span></div> </div></div></div></div></div><div content-id="dsw019-B105" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B105" class="ref-content "><div class="label -label">105</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ben-Yehezkel</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Atar</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Zur</div>
               <div class="given-names">H</div>
            </div>
          </div>
            ., 
            et al.  
            . 
            <div class="year">2015</div>
            , 
            <div class="article-title">
              Rationally designed, heterologous 
              <em>S. cerevisiae</em>
               transcripts expose novel expression determinants
            </div>
            . 
            <div class="source">RNA Biol</div> 
            ., 
            <div class="volume">13</div>
            , 
            <div class="fpage">972</div>
            –
            <div class="lpage">84</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rationally%20designed%2C%20heterologous%20S.%20cerevisiae%20transcripts%20expose%20novel%20expression%20determinants&amp;author=T.%20Ben-Yehezkel&amp;author=S.%20Atar&amp;author=H%20Zur&amp;publication_year=2015&amp;journal=RNA%20Biol&amp;volume=13&amp;pages=972-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/15476286.2015.1071762" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F15476286.2015.1071762" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F15476286.2015.1071762"> </span></div> </div></div></div></div></div><div content-id="dsw019-B106" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B106" class="ref-content "><div class="label -label">106</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Tuller</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Zur</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="year">2015</div>
            , 
            <div class="article-title">Multiple roles of the coding sequence 5′ end in gene expression regulation</div>
            . 
            <div class="source">Nucleic Acids Res</div> 
            ., 
            <div class="volume">43</div>
            , 
            <div class="fpage">13</div>
            –
            <div class="lpage">28</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20roles%20of%20the%20coding%20sequence%205%E2%80%B2%20end%20in%20gene%20expression%20regulation&amp;author=T.%20Tuller&amp;author=H.%20Zur&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=13-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku1313" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku1313" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku1313"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25505165" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B107" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B107" class="ref-content "><div class="label -label">107</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Milo</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Shen-Orr</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Itzkovitz</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Kashtan</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Chklovskii</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Alon</div>
               <div class="given-names">U.</div>
            </div>
          </div>
          <div class="year">2002</div>
            , 
            <div class="article-title">Network motifs: simple building blocks of complex networks</div>
            . 
            <div class="source">Science</div> 
            , 
            <div class="volume">298</div>
            , 
            <div class="fpage">824</div>
            –
            <div class="lpage">7</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Network%20motifs%3A%20simple%20building%20blocks%20of%20complex%20networks&amp;author=R.%20Milo&amp;author=S.%20Shen-Orr&amp;author=S.%20Itzkovitz&amp;author=N.%20Kashtan&amp;author=D.%20Chklovskii&amp;author=U.%20Alon&amp;publication_year=2002&amp;journal=Science&amp;volume=298&amp;pages=824-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.298.5594.824" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.298.5594.824" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.298.5594.824"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12399590" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B108" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B108" class="ref-content "><div class="label -label">108</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Hershberg</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Yeger-Lotem</div>
               <div class="given-names">E.</div>
            </div>
             <div class="name">
              <div class="surname">Margalit</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="year">2005</div>
            , 
            <div class="article-title">Chromosomal organization is shaped by the transcription regulatory network</div>
            . 
            <div class="source">Trends Genet</div> 
            ., 
            <div class="volume">21</div>
            , 
            <div class="fpage">138</div>
            –
            <div class="lpage">42</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosomal%20organization%20is%20shaped%20by%20the%20transcription%20regulatory%20network&amp;author=R.%20Hershberg&amp;author=E.%20Yeger-Lotem&amp;author=H.%20Margalit&amp;publication_year=2005&amp;journal=Trends%20Genet&amp;volume=21&amp;pages=138-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2005.01.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2005.01.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2005.01.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15734572" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B109" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B109" class="ref-content "><div class="label -label">109</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Gatbonton</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Imbesi</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Nelson</div>
               <div class="given-names">M.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2006</div>
            , 
            <div class="article-title">Telomere length as a quantitative trait: genome-wide survey and genetic mapping of telomere length-control genes in yeast</div>
            . 
            <div class="source">PLoS Genet</div> 
            ., 
            <div class="volume">2</div>
            , 
            <div class="fpage">e35</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Telomere%20length%20as%20a%20quantitative%20trait%3A%20genome-wide%20survey%20and%20genetic%20mapping%20of%20telomere%20length-control%20genes%20in%20yeast&amp;author=T.%20Gatbonton&amp;author=M.%20Imbesi&amp;author=M.%20Nelson&amp;publication_year=2006&amp;journal=PLoS%20Genet&amp;volume=2&amp;pages=e35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.0020035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.0020035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.0020035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16552446" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B110" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B110" class="ref-content "><div class="label -label">110</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Gu</div>
               <div class="given-names">W.</div>
            </div>
             <div class="name">
              <div class="surname">Zhou</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Wilke</div>
               <div class="given-names">C.O.</div>
            </div>
          </div>
          <div class="year">2010</div>
            , 
            <div class="article-title">A universal trend of reduced mRNA stability near the translation-initiation site in prokaryotes and eukaryotes</div>
            . 
            <div class="source">PLoS Comput. Biol</div> 
            ., 
            <div class="volume">6</div>
            , 
            <div class="fpage">e1000664</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20universal%20trend%20of%20reduced%20mRNA%20stability%20near%20the%20translation-initiation%20site%20in%20prokaryotes%20and%20eukaryotes&amp;author=W.%20Gu&amp;author=T.%20Zhou&amp;author=C.O.%20Wilke&amp;publication_year=2010&amp;journal=PLoS%20Comput.%20Biol&amp;volume=6&amp;pages=e1000664" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000664" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1000664" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1000664"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20140241" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B111" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B111" class="ref-content "><div class="label -label">111</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Mi</div>
               <div class="given-names">G.</div>
            </div>
             <div class="name">
              <div class="surname">Di</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Emerson</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Cumbie</div>
               <div class="given-names">J.S.</div>
            </div>
             <div class="name">
              <div class="surname">Chang</div>
               <div class="given-names">J.H.</div>
            </div>
          </div>
          <div class="year">2012</div>
            , 
            <div class="article-title">Length bias correction in gene ontology enrichment analysis using logistic regression</div>
            . 
            <div class="source">PLoS One</div> 
            , 
            <div class="volume">7</div>
            , 
            <div class="fpage">e46128</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Length%20bias%20correction%20in%20gene%20ontology%20enrichment%20analysis%20using%20logistic%20regression&amp;author=G.%20Mi&amp;author=Y.%20Di&amp;author=S.%20Emerson&amp;author=J.S.%20Cumbie&amp;author=J.H.%20Chang&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e46128" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0046128" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0046128" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0046128"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23056249" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B112" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B112" class="ref-content "><div class="label -label">112</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Hiller</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Szafranski</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Sinha</div>
               <div class="given-names">R.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2008</div>
            , 
            <div class="article-title">Assessing the fraction of short-distance tandem splice sites under purifying selection</div>
            . 
            <div class="source">RNA</div> 
            , 
            <div class="volume">14</div>
            , 
            <div class="fpage">616</div>
            –
            <div class="lpage">29</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessing%20the%20fraction%20of%20short-distance%20tandem%20splice%20sites%20under%20purifying%20selection&amp;author=M.%20Hiller&amp;author=K.%20Szafranski&amp;author=R.%20Sinha&amp;publication_year=2008&amp;journal=RNA&amp;volume=14&amp;pages=616-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.883908" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.883908" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.883908"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18268022" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B113" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B113" class="ref-content "><div class="label -label">113</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Fox</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Tuch</div>
               <div class="given-names">B.</div>
            </div>
             <div class="name">
              <div class="surname">Chuang</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="year">2008</div>
            , 
            <div class="article-title">Measuring the prevalence of regional mutation rates: an analysis of silent substitutions in mammals, fungi, and insects</div>
            . 
            <div class="source">BMC Evol. Biol</div> 
            ., 
            <div class="volume">8</div>
            , 
            <div class="fpage">186</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Measuring%20the%20prevalence%20of%20regional%20mutation%20rates%3A%20an%20analysis%20of%20silent%20substitutions%20in%20mammals%2C%20fungi%2C%20and%20insects&amp;author=A.%20Fox&amp;author=B.%20Tuch&amp;author=J.%20Chuang&amp;publication_year=2008&amp;journal=BMC%20Evol.%20Biol&amp;volume=8&amp;pages=186" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-8-186" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-8-186" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-8-186"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18588686" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B114" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B114" class="ref-content "><div class="label -label">114</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Gossmann</div>
               <div class="given-names">T.I.</div>
            </div>
             <div class="name">
              <div class="surname">Keightley</div>
               <div class="given-names">P.D.</div>
            </div>
             <div class="name">
              <div class="surname">Eyre-Walker</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">2012</div>
            , 
            <div class="article-title">The effect of variation in the effective population size on the rate of adaptive molecular evolution in eukaryotes</div>
            . 
            <div class="source">Genome Biol. Evol</div> 
            ., 
            <div class="volume">4</div>
            , 
            <div class="fpage">658</div>
            –
            <div class="lpage">67</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20effect%20of%20variation%20in%20the%20effective%20population%20size%20on%20the%20rate%20of%20adaptive%20molecular%20evolution%20in%20eukaryotes&amp;author=T.I.%20Gossmann&amp;author=P.D.%20Keightley&amp;author=A.%20Eyre-Walker&amp;publication_year=2012&amp;journal=Genome%20Biol.%20Evol&amp;volume=4&amp;pages=658-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evs027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevs027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevs027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22436998" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B115" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B115" class="ref-content "><div class="label -label">115</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">De Conti</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Baralle</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Buratti</div>
               <div class="given-names">E.</div>
            </div>
          </div>
          <div class="year">2013</div>
            , 
            <div class="article-title">Exon and intron definition in pre-mRNA splicing</div>
            . 
            <div class="source">Wiley Interdiscip. Rev</div> 
            ., 
            <div class="volume">4</div>
            , 
            <div class="fpage">49</div>
            –
            <div class="lpage">60</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exon%20and%20intron%20definition%20in%20pre-mRNA%20splicing&amp;author=L.%20De%20Conti&amp;author=M.%20Baralle&amp;author=E.%20Buratti&amp;publication_year=2013&amp;journal=Wiley%20Interdiscip.%20Rev&amp;volume=4&amp;pages=49-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/wrna.1140" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fwrna.1140" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fwrna.1140"> </span></div> </div></div></div></div></div><div content-id="dsw019-B116" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B116" class="ref-content "><div class="label -label">116</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Lanfear</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Kokko</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Eyre-Walker</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">2014</div>
            , 
            <div class="article-title">Population size and the rate of evolution</div>
            . 
            <div class="source">Trends Ecol. Evol</div> 
            ., 
            <div class="volume">29</div>
            , 
            <div class="fpage">33</div>
            –
            <div class="lpage">41</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20size%20and%20the%20rate%20of%20evolution&amp;author=R.%20Lanfear&amp;author=H.%20Kokko&amp;author=A.%20Eyre-Walker&amp;publication_year=2014&amp;journal=Trends%20Ecol.%20Evol&amp;volume=29&amp;pages=33-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tree.2013.09.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tree.2013.09.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tree.2013.09.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24148292" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B117" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B117" class="ref-content "><div class="label -label">117</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Fox-Walsh</div>
               <div class="given-names">K.L.</div>
            </div>
             <div class="name">
              <div class="surname">Hertel</div>
               <div class="given-names">K.J.</div>
            </div>
          </div>
          <div class="year">2009</div>
            , 
            <div class="article-title">Splice-site pairing is an intrinsically high fidelity process</div>
            . 
            <div class="source">Proc. Natl. Acad. Sci. USA</div> 
            ., 
            <div class="volume">106</div>
            , 
            <div class="fpage">1766</div>
            –
            <div class="lpage">71</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Splice-site%20pairing%20is%20an%20intrinsically%20high%20fidelity%20process&amp;author=K.L.%20Fox-Walsh&amp;author=K.J.%20Hertel&amp;publication_year=2009&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=1766-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0813128106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0813128106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0813128106"> </span></div> </div></div></div></div></div><div content-id="dsw019-B118" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B118" class="ref-content "><div class="label -label">118</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Ramsköld</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Wang</div>
               <div class="given-names">E.T.</div>
            </div>
             <div class="name">
              <div class="surname">Burge</div>
               <div class="given-names">C.B.</div>
            </div>
             <div class="name">
              <div class="surname">Sandberg</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="year">2009</div>
            , 
            <div class="article-title">An abundance of ubiquitously expressed genes revealed by tissue transcriptome sequence data</div>
            . 
            <div class="source">PLoS Comput. Biol</div> 
            ., 
            <div class="volume">5</div>
            , 
            <div class="fpage">e1000598</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20abundance%20of%20ubiquitously%20expressed%20genes%20revealed%20by%20tissue%20transcriptome%20sequence%20data&amp;author=D.%20Ramsk%C3%B6ld&amp;author=E.T.%20Wang&amp;author=C.B.%20Burge&amp;author=R.%20Sandberg&amp;publication_year=2009&amp;journal=PLoS%20Comput.%20Biol&amp;volume=5&amp;pages=e1000598" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000598" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1000598" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1000598"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20011106" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B119" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B119" class="ref-content "><div class="label -label">119</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Schwanhausser</div>
               <div class="given-names">B.</div>
            </div>
             <div class="name">
              <div class="surname">Busse</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Li</div>
               <div class="given-names">N.</div>
            </div>
          </div>
            , 
            et al.  
            . 
            <div class="year">2011</div>
            , 
            <div class="article-title">Global quantification of mammalian gene expression control</div>
            . 
            <div class="source">Nature</div> 
            , 
            <div class="volume">473</div>
            , 
            <div class="fpage">337</div>
            –
            <div class="lpage">42</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20quantification%20of%20mammalian%20gene%20expression%20control&amp;author=B.%20Schwanhausser&amp;author=D.%20Busse&amp;author=N.%20Li&amp;publication_year=2011&amp;journal=Nature&amp;volume=473&amp;pages=337-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10098" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10098" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10098"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21593866" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B120" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B120" class="ref-content "><div class="label -label">120</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Vogel</div>
               <div class="given-names">C.</div>
            </div>
             <div class="name">
              <div class="surname">Marcotte</div>
               <div class="given-names">E.M.</div>
            </div>
          </div>
          <div class="year">2012</div>
            , 
            <div class="article-title">Insights into the regulation of protein abundance from proteomic and transcriptomic analyses</div>
            . 
            <div class="source">Nat. Rev. Genet</div> 
            ., 
            <div class="volume">13</div>
            , 
            <div class="fpage">227</div>
            –
            <div class="lpage">32</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insights%20into%20the%20regulation%20of%20protein%20abundance%20from%20proteomic%20and%20transcriptomic%20analyses&amp;author=C.%20Vogel&amp;author=E.M.%20Marcotte&amp;publication_year=2012&amp;journal=Nat.%20Rev.%20Genet&amp;volume=13&amp;pages=227-32" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22411467" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B121" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B121" class="ref-content "><div class="label -label">121</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Baker</div>
               <div class="given-names">K.E.</div>
            </div>
             <div class="name">
              <div class="surname">Parker</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="year">2004</div>
            , 
            <div class="article-title">Nonsense-mediated mRNA decay: terminating erroneous gene expression</div>
            . 
            <div class="source">Curr. Opin. Cell Biol</div> 
            ., 
            <div class="volume">16</div>
            , 
            <div class="fpage">293</div>
            –
            <div class="lpage">9</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nonsense-mediated%20mRNA%20decay%3A%20terminating%20erroneous%20gene%20expression&amp;author=K.E.%20Baker&amp;author=R.%20Parker&amp;publication_year=2004&amp;journal=Curr.%20Opin.%20Cell%20Biol&amp;volume=16&amp;pages=293-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ceb.2004.03.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ceb.2004.03.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ceb.2004.03.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15145354" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B122" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B122" class="ref-content "><div class="label -label">122</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Mayr</div>
               <div class="given-names">E.</div>
            </div>
          </div>
          <div class="year">2001</div>
            , 
            <div class="source">What Evolution Is</div> 
            . 
            <div class="publisher-name">Basic Books, Perseus Books Group</div>
            : 
            <div class="publisher-loc">New York</div>
            .
          </div></div></div></div><div content-id="dsw019-B123" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B123" class="ref-content "><div class="label -label">123</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Jacob</div>
               <div class="given-names">F.</div>
            </div>
          </div>
          <div class="year">1977</div>
            , 
            <div class="article-title">Evolution and tinkering</div>
            . 
            <div class="source">Science</div> 
            , 
            <div class="volume">196</div>
            , 
            <div class="fpage">1161</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20and%20tinkering&amp;author=F.%20Jacob&amp;publication_year=1977&amp;journal=Science&amp;volume=196&amp;pages=1161" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.860134" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.860134" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.860134"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/860134" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B124" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B124" class="ref-content "><div class="label -label">124</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Pigliucci</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Kaplan</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="year">2006</div>
            , 
            <div class="source">Making Sense of Evolution: The Conceptual Foundations of Evolutionary Theory</div> 
            . 
            <div class="publisher-name">University of Chicago Press</div>
            : 
            <div class="publisher-loc">Chicago</div>
            .
          </div></div></div></div><div content-id="dsw019-B125" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B125" class="ref-content "><div class="label -label">125</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Chamary</div>
               <div class="given-names">J.V.</div>
            </div>
             <div class="name">
              <div class="surname">Parmley</div>
               <div class="given-names">J.L.</div>
            </div>
             <div class="name">
              <div class="surname">Hurst</div>
               <div class="given-names">L.D.</div>
            </div>
          </div>
          <div class="year">2006</div>
            , 
            <div class="article-title">Hearing silence: non-neutral evolution at synonymous sites in mammals</div>
            . 
            <div class="source">Nat. Rev. Genet</div> 
            ., 
            <div class="volume">7</div>
            , 
            <div class="fpage">98</div>
            –
            <div class="lpage">108</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hearing%20silence%3A%20non-neutral%20evolution%20at%20synonymous%20sites%20in%20mammals&amp;author=J.V.%20Chamary&amp;author=J.L.%20Parmley&amp;author=L.D.%20Hurst&amp;publication_year=2006&amp;journal=Nat.%20Rev.%20Genet&amp;volume=7&amp;pages=98-108" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1770" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1770" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1770"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16418745" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B126" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B126" class="ref-content "><div class="label -label">126</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sauna</div>
               <div class="given-names">Z.E.</div>
            </div>
             <div class="name">
              <div class="surname">Kimchi-Sarfaty</div>
               <div class="given-names">C.</div>
            </div>
          </div>
          <div class="year">2011</div>
            , 
            <div class="article-title">Understanding the contribution of synonymous mutations to human disease</div>
            . 
            <div class="source">Nat. Rev. Genet</div> 
            ., 
            <div class="volume">12</div>
            , 
            <div class="fpage">683</div>
            –
            <div class="lpage">91</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Understanding%20the%20contribution%20of%20synonymous%20mutations%20to%20human%20disease&amp;author=Z.E.%20Sauna&amp;author=C.%20Kimchi-Sarfaty&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet&amp;volume=12&amp;pages=683-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3051" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3051" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3051"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21878961" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw019-B127" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw019-B127" class="ref-content "><div class="label -label">127</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Zur</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Tuller</div>
               <div class="given-names">T.</div>
            </div>
          </div>
          <div class="year">2013</div>
            , 
            <div class="article-title">New universal rules of eukaryotic translation initiation fidelity</div>
            . 
            <div class="source">PLoS Comput. Biol</div> 
            ., 
            <div class="volume">9</div>
            , 
            <div class="fpage">e1003136</div>
            .
          <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20universal%20rules%20of%20eukaryotic%20translation%20initiation%20fidelity&amp;author=H.%20Zur&amp;author=T.%20Tuller&amp;publication_year=2013&amp;journal=PLoS%20Comput.%20Biol&amp;volume=9&amp;pages=e1003136" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1003136" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1003136" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1003136"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23874179" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p"> This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License ( <a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a> ), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com </div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2469962&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/4/10.1093_dnares_dsw019/2/dsw019_Supplementary_Data.zip?Expires=2147483647&amp;Signature=1Q3bJLc0nWGrMCil6yfHgI41W-Eb6d0IkJyRDZBMhwTgaHZHFzaN6owH6Ja-5JPyFjGmug4EGcVTtwIiu34AT5c~a~N6g9Um02mTBCIB26thfTpuRYdRbJrEP6X1dD419oJiTtu8FA7DFqVRRwQw2L2LmpCg7DtWPsgKmys7If5oSz-XB5uiApPTO~muqF47DIByzOXcglaf0XmhUKMszNJp3VVz~UX9q~MdRIw3SIq2H-ykRMuGMsEy-Wurs8KjA4Kex2ty-nisnMvdy0OMTCR7oq6BtNPmHmbEHHSNRhpp4ZW0sClklY2CqSoJ8-wc-UBVZLKv7~Fzbh2tQjHTZw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">224</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000384768700007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">3</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw019"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2469962"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000384768700007&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f4%2f377%2f2469962">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000384768700007&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (3)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F4%2F377%2F2469962" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2469962'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Selection for reduced translation costs at the intronic 5′ end in fungi" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw019");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f4%2f377%2f2469962">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

