

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/4/439/5035912","@type":"ScholarlyArticle","name":"Highly dense linkage maps from 31 full-sibling families of turbot ( Scophthalmus maximus ) provide insights into recombination patterns and chromosome rearrangements throughout a newly refined genome assembly","datePublished":"2018-06-11","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2018-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy015","keywords":["<italic>Scophthalmus maximus</italic>","2b-RAD sequencing","linkage mapping","genome assembly","chromosome reorganisations"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/4/439/5035912","author":[{"name":"Maroso, F","affiliation":"GENEAQUA S.L, 2007 Lugo, Spain","@type":"Person","sameAs":"http://orcid.org/0000-0003-3524-5916"},{"name":"Hermida, M","affiliation":"Departamento de Zoología, Genética y Antropología Física, Facultad de Veterinaria, Universidade de Santiago de Compostela, Lugo, Spain","@type":"Person"},{"name":"Millán, A","affiliation":"GENEAQUA S.L, 2007 Lugo, Spain","@type":"Person"},{"name":"Blanco, A","affiliation":"Departamento de Zoología, Genética y Antropología Física, Facultad de Veterinaria, Universidade de Santiago de Compostela, Lugo, Spain","@type":"Person"},{"name":"Saura, M","affiliation":"Departamento de Mejora Genética Animal, Instituto Nacional de Investigación y Tecnología Agraria y Alimentaria (INIA), Madrid, Spain","@type":"Person"},{"name":"Fernández, A","affiliation":"Departamento de Mejora Genética Animal, Instituto Nacional de Investigación y Tecnología Agraria y Alimentaria (INIA), Madrid, Spain","@type":"Person"},{"name":"Dalla Rovere, G","affiliation":"Department of Comparative Biomedicine and Food Science, University of Padova, Legnaro, Padova, Italy","@type":"Person"},{"name":"Bargelloni, L","affiliation":"Department of Comparative Biomedicine and Food Science, University of Padova, Legnaro, Padova, Italy","@type":"Person"},{"name":"Cabaleiro, S","affiliation":"Cluster de Acuicultura de Galicia (Punta do Couso), Aguiño-Ribeira, Spain","@type":"Person"},{"name":"Villanueva, B","affiliation":"Departamento de Mejora Genética Animal, Instituto Nacional de Investigación y Tecnología Agraria y Alimentaria (INIA), Madrid, Spain","@type":"Person"},{"name":"Bouza, C","affiliation":"Departamento de Zoología, Genética y Antropología Física, Facultad de Veterinaria, Universidade de Santiago de Compostela, Lugo, Spain","@type":"Person"},{"name":"Martínez, P","affiliation":"Departamento de Zoología, Genética y Antropología Física, Facultad de Veterinaria, Universidade de Santiago de Compostela, Lugo, Spain","@type":"Person"}],"description":"Abstract.  Highly dense linkage maps enable positioning thousands of landmarks useful for anchoring the whole genome and for analysing genome properties. Turbot","pageStart":"439","pageEnd":"450","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/4/1/m_dnares_25_4cover.png?Expires=2147483647&Signature=lD6dhAB99HpTltGbT4-zO-h9Z6msKpBpjdlZuCIFAdO0RvvEM~IgMElNp5bE6Rdf~OSfdWzB2y8SIAkqvb6Fc-0GRLo0HVHIHIXLjJF8ZJ94ZrTCEDv2Gn-92BvjK~Ebx3LV-ePsMx8j4josbKy1FscEJ3-qj3zB2TEHnK~iAQqommn5aOw82hhC1dOH5f1P15MUh9o1QIOhJ9WAnJwCFYDxRE0ZmdX~qCpKaFm8XPRL-O71Rxc1eeV8iTb9yQzgpbw2OwX1g4mQ5iALY~7Bk3L2RgUkQZIzgNqb8ANqfHjAsIlrzFHcKAl7tTiKtoGbsHvDqkXNPcKjgjrdKdIXQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Highly dense linkage maps from 31 full-sibling families of turbot ( Scophthalmus maximus ) provide insights into recombination patterns and chromosome rearrangements throughout a newly refined genome assembly","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/4/1/m_dnares_25_4cover.png?Expires=2147483647&Signature=lD6dhAB99HpTltGbT4-zO-h9Z6msKpBpjdlZuCIFAdO0RvvEM~IgMElNp5bE6Rdf~OSfdWzB2y8SIAkqvb6Fc-0GRLo0HVHIHIXLjJF8ZJ94ZrTCEDv2Gn-92BvjK~Ebx3LV-ePsMx8j4josbKy1FscEJ3-qj3zB2TEHnK~iAQqommn5aOw82hhC1dOH5f1P15MUh9o1QIOhJ9WAnJwCFYDxRE0ZmdX~qCpKaFm8XPRL-O71Rxc1eeV8iTb9yQzgpbw2OwX1g4mQ5iALY~7Bk3L2RgUkQZIzgNqb8ANqfHjAsIlrzFHcKAl7tTiKtoGbsHvDqkXNPcKjgjrdKdIXQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Highly dense linkage maps from 31 full-sibling families of turbot ( Scophthalmus maximus ) provide insights into recombination patterns and chromosome rearrangements throughout a newly refined genome assembly" />
    <meta property="og:description" content="Abstract.  Highly dense linkage maps enable positioning thousands of landmarks useful for anchoring the whole genome and for analysing genome properties. Turbot" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/4/439/5035912" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2018-06-11" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Maroso, F" /><meta name="citation_author_institution" content="GENEAQUA S.L, 2007 Lugo, Spain" /><meta name="citation_author" content="Hermida, M" /><meta name="citation_author_institution" content="Departamento de Zoología, Genética y Antropología Física, Facultad de Veterinaria, Universidade de Santiago de Compostela, Lugo, Spain" /><meta name="citation_author" content="Millán, A" /><meta name="citation_author_institution" content="GENEAQUA S.L, 2007 Lugo, Spain" /><meta name="citation_author" content="Blanco, A" /><meta name="citation_author_institution" content="Departamento de Zoología, Genética y Antropología Física, Facultad de Veterinaria, Universidade de Santiago de Compostela, Lugo, Spain" /><meta name="citation_author" content="Saura, M" /><meta name="citation_author_institution" content="Departamento de Mejora Genética Animal, Instituto Nacional de Investigación y Tecnología Agraria y Alimentaria (INIA), Madrid, Spain" /><meta name="citation_author" content="Fernández, A" /><meta name="citation_author_institution" content="Departamento de Mejora Genética Animal, Instituto Nacional de Investigación y Tecnología Agraria y Alimentaria (INIA), Madrid, Spain" /><meta name="citation_author" content="Dalla Rovere, G" /><meta name="citation_author_institution" content="Department of Comparative Biomedicine and Food Science, University of Padova, Legnaro, Padova, Italy" /><meta name="citation_author" content="Bargelloni, L" /><meta name="citation_author_institution" content="Department of Comparative Biomedicine and Food Science, University of Padova, Legnaro, Padova, Italy" /><meta name="citation_author" content="Cabaleiro, S" /><meta name="citation_author_institution" content="Cluster de Acuicultura de Galicia (Punta do Couso), Aguiño-Ribeira, Spain" /><meta name="citation_author" content="Villanueva, B" /><meta name="citation_author_institution" content="Departamento de Mejora Genética Animal, Instituto Nacional de Investigación y Tecnología Agraria y Alimentaria (INIA), Madrid, Spain" /><meta name="citation_author" content="Bouza, C" /><meta name="citation_author_institution" content="Departamento de Zoología, Genética y Antropología Física, Facultad de Veterinaria, Universidade de Santiago de Compostela, Lugo, Spain" /><meta name="citation_author" content="Martínez, P" /><meta name="citation_author_institution" content="Departamento de Zoología, Genética y Antropología Física, Facultad de Veterinaria, Universidade de Santiago de Compostela, Lugo, Spain" /><meta name="citation_title" content="Highly dense linkage maps from 31 full-sibling families of turbot (Scophthalmus maximus) provide insights into recombination patterns and chromosome rearrangements throughout a newly refined genome assembly" /><meta name="citation_firstpage" content="439" /><meta name="citation_lastpage" content="450" /><meta name="citation_doi" content="10.1093/dnares/dsy015" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2018/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The impact of next-generation sequencing technology on genetics;  citation_journal_title=Trends Genet;  citation_year=2008;  citation_volume=24;  citation_pages=133-41; " /><meta name="citation_reference" content="citation_title=Construction of a high-density genetic map and quantitative trait locus mapping in the Manila clam Ruditapes philippinarum;  citation_journal_title=Sci. Rep;  citation_year=2017;  citation_volume=7;  citation_pages=229" /><meta name="citation_reference" content="citation_title=Large-scale SNP discovery and construction of a high-density genetic map of Colossoma macropomum through genotyping-by-sequencing;  citation_journal_title=Sci. Rep;  citation_year=2017;  citation_volume=7;  citation_pages=46112" /><meta name="citation_reference" content="citation_title=The next-generation sequencing revolution and its impact on genomics;  citation_journal_title=Cell;  citation_year=2013;  citation_volume=155;  citation_pages=27-38; " /><meta name="citation_reference" content="citation_title=Ten years of next-generation sequencing technology;  citation_journal_title=Trends Genet;  citation_year=2014;  citation_volume=30;  citation_pages=418-26; " /><meta name="citation_reference" content="citation_title=Strategies for optimizing BioNano and Dovetail explored through a second reference quality assembly for the legume model, Medicago truncatula;  citation_journal_title=BMC Genomics;  citation_year=2017;  citation_volume=18;  citation_pages=578" /><meta name="citation_reference" content="citation_title=Recombination in the threespine stickleback genome—patterns and consequences;  citation_journal_title=Mol. Ecol;  citation_year=2013;  citation_volume=22;  citation_pages=3014-27; " /><meta name="citation_reference" content="citation_title=Using linkage maps to correct and scaffold de novo genome assemblies: methods, challenges, and computational tools;  citation_journal_title=Front. Genet;  citation_year=2015;  citation_volume=6;  citation_pages=220" /><meta name="citation_reference" content="Robledo D., Palaiokostas C., Bargelloni L., Martínez P., Houston R.  2017 , Applications of genotyping by sequencing in aquaculture breeding and genetics. Rev. Aquac. , doi: 10.1111/raq.12193. " /><meta name="citation_reference" content="citation_title=From 2R to 3R: evidence for a fish-specific genome duplication (FSGD);  citation_journal_title=Bioessays;  citation_year=2005;  citation_volume=27;  citation_pages=937-45; " /><meta name="citation_reference" content="citation_title=Rapid and cost-effective polymorphism identification and genotyping using restriction site associated DNA (RAD) markers;  citation_journal_title=Genome Res;  citation_year=2007;  citation_volume=17;  citation_pages=240-8; " /><meta name="citation_reference" content="citation_title=Rapid SNP discovery and genetic mapping using sequenced RAD markers;  citation_journal_title=PLoS One;  citation_year=2008;  citation_volume=3;  citation_pages=e3376" /><meta name="citation_reference" content="citation_title=Genome-wide genetic marker discovery and genotyping using next-generation sequencing;  citation_journal_title=Nat. Rev. Genet;  citation_year=2011;  citation_volume=12;  citation_pages=499-510; " /><meta name="citation_reference" content="citation_title=Quantitative genetic analysis of retinal degeneration in the blind cavefish Astyanax mexicanus;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e57281" /><meta name="citation_reference" content="citation_title=Linkage maps of the Atlantic salmon (Salmo salar) genome derived from RAD sequencing;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=166" /><meta name="citation_reference" content="citation_title=A high-density linkage map for Astyanax mexicanus using genotyping-by-sequencing technology;  citation_journal_title=G3 Genes Genomes Genet;  citation_year=2015;  citation_volume=5;  citation_pages=241-51; " /><meta name="citation_reference" content="citation_title=A high-density genetic map and growth related QTL mapping in bighead carp (Hypophthalmichthys nobilis);  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=28679" /><meta name="citation_reference" content="citation_title=Fine mapping QTL for resistance to VNN disease using a high-density linkage map in Asian seabass;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=32122" /><meta name="citation_reference" content="citation_title=Construction of high-resolution recombination maps in Asian seabass;  citation_journal_title=BMC Genomics;  citation_year=2017;  citation_volume=18;  citation_pages=63" /><meta name="citation_reference" content="citation_title=Construction of high-density genetic linkage maps and mapping of growth-related quantitative trail loci in the Japanese flounder (Paralichthys olivaceus);  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e50404" /><meta name="citation_reference" content="citation_title=High-density genetic linkage mapping in turbot (Scophthalmus maximus L.) based on SNP markers and major sex-and growth-related regions detection;  citation_journal_title=PLoS One;  citation_year=2015;  citation_volume=10;  citation_pages=e0120410" /><meta name="citation_reference" content="citation_title=Construction of a high-density linkage map and mapping of sex determination and growth-related loci in the mandarin fish (Siniperca chuatsi);  citation_journal_title=BMC Genomics;  citation_year=2017;  citation_volume=18;  citation_pages=446" /><meta name="citation_reference" content="citation_title=Construction and annotation of a high density SNP linkage map of the Atlantic salmon (Salmo salar) genome;  citation_journal_title=G3 Genes Genomes Genet;  citation_year=2016;  citation_volume=6;  citation_pages=2173-9; " /><meta name="citation_reference" content="citation_title=Genomic prediction of resistance to pasteurellosis in gilthead sea bream (Sparus aurata) using 2b-RAD sequencing;  citation_journal_title=G3 Genes Genomes Genet;  citation_year=2016;  citation_volume=6;  citation_pages=3693-700; " /><meta name="citation_reference" content="citation_title=Criteria to optimize designs for detection and estimation of linkage between marker loci from segregating populations containing several families;  citation_journal_title=Theoret. Appl. Genetics;  citation_year=1993;  citation_volume=86-86;  citation_pages=269-80; " /><meta name="citation_reference" content="citation_publisher=John Wiley &amp; Sons, Ltd, Chichester, UK;  citation_title=Bioinformatics in Aquaculture: Principles and Methods;  citation_year=2017;  citation_pages=356-79; " /><meta name="citation_reference" content="citation_title=in the Atlantic Cod genome represent polymorphic chromosomal rearrangements;  citation_journal_title=Genome Biol. Evol;  citation_year=2016;  citation_volume=8;  citation_pages=1012-22; " /><meta name="citation_reference" content="citation_title=The genomic basis of adaptive evolution in threespine sticklebacks;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=484;  citation_pages=55-61; " /><meta name="citation_reference" content="citation_title=Turbot. AquaTrace Species Leaflet;  citation_year=2014; " /><meta name="citation_reference" content="citation_title=Identification of the major sex-determining region of turbot (Scophthalmus maximus);  citation_journal_title=Genetics;  citation_year=2009;  citation_volume=183;  citation_pages=1443-52; " /><meta name="citation_reference" content="citation_title=Improving the inference of population genetic structure in the presence of related individuals;  citation_journal_title=Genet. Res;  citation_year=2014;  citation_volume=96;  citation_pages=e003" /><meta name="citation_reference" content="citation_title=Fine mapping and evolution of the major sex determining region in turbot (Scophthalmus maximus);  citation_journal_title=G3 Genes Genomes Genet;  citation_year=2014;  citation_volume=4;  citation_pages=1871-80; " /><meta name="citation_reference" content="citation_title=Gene expression analysis at the onset of sex differentiation in turbot (Scophthalmus maximus);  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=973" /><meta name="citation_reference" content="citation_title=Whole genome sequencing of turbot (Scophthalmus maximus; Pleuronectiformes): a fish adapted to demersal life;  citation_journal_title=DNA Res;  citation_year=2016;  citation_volume=23;  citation_pages=181-92; " /><meta name="citation_reference" content="citation_title=Integrative transcriptome, genome and quantitative trait loci resources identify single nucleotide polymorphisms in candidate genes for growth traits in turbot;  citation_journal_title=IJMS;  citation_year=2016;  citation_volume=17;  citation_pages=243" /><meta name="citation_reference" content="citation_title=A microsatellite genetic map of the turbot (Scophthalmus maximus);  citation_journal_title=Genetics;  citation_year=2007;  citation_volume=177;  citation_pages=2457-67; " /><meta name="citation_reference" content="citation_title=Genetic linkage mapping of turbot (Scophthalmus maximus L.) using microsatellite markers and its application in QTL analysis;  citation_journal_title=Aquaculture;  citation_year=2010;  citation_volume=308;  citation_pages=89-100; " /><meta name="citation_reference" content="citation_title=An Expressed Sequence Tag (EST)-enriched genetic map of turbot (Scophthalmus maximus): a useful framework for comparative genomics across model and farmed teleosts;  citation_journal_title=BMC Genet;  citation_year=2012;  citation_volume=13;  citation_pages=54" /><meta name="citation_reference" content="citation_title=Compilation of mapping resources in turbot (Scophthalmus maximus): a new integrated consensus genetic map;  citation_journal_title=Aquaculture;  citation_year=2013;  citation_volume=414–415;  citation_pages=19-25; " /><meta name="citation_reference" content="citation_title=Centromere-linkage in the turbot (Scophthalmus maximus) through half-tetrad analysis in diploid meiogynogenetics;  citation_journal_title=Aquaculture;  citation_year=2008;  citation_volume=280;  citation_pages=81-8; " /><meta name="citation_reference" content="citation_title=Complete nucleotide sequence of Japanese flounder (Paralichthys olivaceus) mitochondrial genome: structural properties and cue for resolving teleostean relationship;  citation_journal_title=J. Hered;  citation_year=2000;  citation_volume=91;  citation_pages=271-8; " /><meta name="citation_reference" content="citation_title=Whole-genome sequence of a flatfish provides insights into ZW sex chromosome evolution and adaptation to a benthic lifestyle;  citation_journal_title=Nat. Genet;  citation_year=2014;  citation_volume=46;  citation_pages=253-60; " /><meta name="citation_reference" content="citation_title=The genome and transcriptome of Japanese flounder provide insights into flatfish asymmetry;  citation_journal_title=Nat. Genet;  citation_year=2017;  citation_volume=49;  citation_pages=119-24; " /><meta name="citation_reference" content="citation_title=Identification and validation of single nucleotide polymorphisms as tools to detect hybridization and population structure in freshwater stingrays;  citation_journal_title=Mol. Ecol. Resour;  citation_year=2017;  citation_volume=17;  citation_pages=550-6; " /><meta name="citation_reference" content="citation_title=2b-RAD: a simple and flexible method for genome-wide genotyping;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=808-10; " /><meta name="citation_reference" content="citation_title=Stacks: building and genotyping loci de novo from short-read sequences;  citation_journal_title=G3 (Bethesda) );  citation_year=2011;  citation_volume=1;  citation_pages=171-82; " /><meta name="citation_reference" content="citation_title=Stacks: an analysis tool set for population genomics;  citation_journal_title=Mol. Ecol;  citation_year=2013;  citation_volume=22;  citation_pages=3124-40; " /><meta name="citation_reference" content="citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome;  citation_journal_title=Genome Biol;  citation_year=2009;  citation_volume=10;  citation_pages=R25" /><meta name="citation_reference" content="Van Ooijen J. W.  2006 , JoinMap ® 4. Software for the calculation of genetic linkage maps in experimental populations, vol. 33. Kyazma BV: Wageningen. " /><meta name="citation_reference" content="citation_title=Accurate construction of consensus genetic maps via integer linear programming;  citation_journal_title=IEEE/ACM Trans. Comput. Biol. Bioinform;  citation_year=2011;  citation_volume=8;  citation_pages=381-94; " /><meta name="citation_reference" content="citation_title=MapChart: software for the graphical presentation of linkage maps and QTLs;  citation_journal_title=J. Hered;  citation_year=2002;  citation_volume=93;  citation_pages=77-8; " /><meta name="citation_reference" content="citation_title=Karotypic characterization of turbot (Scophthalmus maximus) with conventional, fluorochrome and restriction endonuclease-banding techniques;  citation_journal_title=Mar. Biol;  citation_year=1994;  citation_volume=120;  citation_pages=609-13; " /><meta name="citation_reference" content="Team R. C.  2013 , R: a language and environment for statistical computing. " /><meta name="citation_reference" content="Harris R. S.  2007 , Improved pairwise alignment of genomic DNA. PhD Thesis, The Pennsylvania State University, Pennsylvania. " /><meta name="citation_reference" content="citation_title=Validation of single nucleotide polymorphism (SNP) markers from an immune Expressed Sequence Tag (EST) turbot, Scophthalmus maximus, database;  citation_journal_title=Aquaculture;  citation_year=2011;  citation_volume=313;  citation_pages=31-41; " /><meta name="citation_reference" content="citation_title=Development and validation of single nucleotide polymorphisms (SNPs) markers from two transcriptome 454-runs of turbot (Scophthalmus maximus) using high-throughput genotyping;  citation_journal_title=IJMS;  citation_year=2013;  citation_volume=14;  citation_pages=5694-711; " /><meta name="citation_reference" content="citation_title=Evolution from fish to mammals by gene duplication;  citation_journal_title=Hereditas;  citation_year=1968;  citation_volume=59;  citation_pages=169-87; " /><meta name="citation_reference" content="citation_title=Construction of a high-density, high-resolution genetic map and its integration with BAC-based physical map in channel catfish;  citation_journal_title=DNA Res;  citation_year=2015;  citation_volume=22;  citation_pages=39-52; " /><meta name="citation_reference" content="citation_title=A new SNP-based vision of the genetics of sex determination in European sea bass (Dicentrarchus labrax);  citation_journal_title=Genet. Sel. Evol;  citation_year=2015;  citation_volume=47;  citation_pages=68" /><meta name="citation_reference" content="citation_title=Integration of linkage maps for the amphidiploid Brassica napus and comparative mapping with Arabidopsis and Brassica rapa;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=101" /><meta name="citation_reference" content="citation_title=A multi-population consensus genetic map reveals inconsistent marker order among maps likely attributed to structural variations in the apple genome;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e47864" /><meta name="citation_reference" content="citation_title=Development and implementation of high-throughput SNP genotyping in barley;  citation_journal_title=BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=582" /><meta name="citation_reference" content="citation_publisher=Elsevier, London;  citation_title=Genomics in Aquaculture;  citation_year=2016;  citation_pages=131-63; " /><meta name="citation_reference" content="citation_title=Difference in the behaviour of eu-and hetero-chromatin: crossing-over;  citation_journal_title=Nature;  citation_year=1965;  citation_volume=205;  citation_pages=725-6; " /><meta name="citation_reference" content="citation_title=Integration of high-resolution physical and genetic map reveals differential recombination frequency between chromosomes and the genome assembling quality in cucumber;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e62676" /><meta name="citation_reference" content="citation_title=Sex-specific recombination rates in zebrafish (Danio rerio);  citation_journal_title=Genetics;  citation_year=2002;  citation_volume=160;  citation_pages=649-57; " /><meta name="citation_reference" content="citation_title=A linkage map of the Atlantic salmon (Salmo salar) based on EST-derived SNP markers;  citation_journal_title=BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=223" /><meta name="citation_reference" content="citation_title=Environmental selection on transcriptome-derived SNPs in a high gene flow marine fish, the Atlantic herring (Clupea harengus);  citation_journal_title=Mol. Ecol;  citation_year=2012;  citation_volume=21;  citation_pages=3686-703; " /><meta name="citation_reference" content="citation_title=Recombination difference between sexes: a role for haploid selection;  citation_journal_title=PLoS Biol;  citation_year=2005;  citation_volume=3;  citation_pages=e63" /><meta name="citation_reference" content="citation_title=Cytological studies of human meiosis: sex-specific differences in recombination originate at, or prior to, establishment of double-strand breaks;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e85075" /><meta name="citation_reference" content="citation_title=Phylogenetic analysis of flatfish (Order Pleuronectiformes) based on mitochondrial 16s rDNA sequences;  citation_journal_title=Sci. Mar;  citation_year=2005;  citation_volume=69;  citation_pages=531-43; " /><meta name="citation_reference" content="citation_title=Are flatfishes (Pleuronectiformes) monophyletic?,;  citation_journal_title=Mol. Phylogenet. Evol;  citation_year=2013;  citation_volume=69;  citation_pages=664-73; " /><meta name="citation_reference" content="citation_title=The evolutionary origin of flatfish asymmetry;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=454;  citation_pages=209-12; " /><meta name="citation_reference" content="citation_title=Phylogenomic analysis of carangimorph fishes reveals flatfish asymmetry arose in a blink of the evolutionary eye;  citation_journal_title=BMC Evol. Biol;  citation_year=2016;  citation_volume=16;  citation_pages=224" /><meta name="citation_reference" content="citation_title=Origin of human chromosome 2: an ancestral telomere-telomere fusion;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=1991;  citation_volume=88;  citation_pages=9051-5; " /><meta name="citation_reference" content="citation_title=Cytogenetic characterization of six species of flatfishes with comments to karyotype differentiation patterns in Pleuronectiformes (Teleostei);  citation_journal_title=J. Fish Biology;  citation_year=2007;  citation_volume=70;  citation_pages=1-15; " /><meta name="citation_reference" content="citation_title=Cold fusion: massive karyotype evolution in the Antarctic Bullhead Notothen Notothenia coriiceps;  citation_journal_title=G3 Genes Genomes Genet;  citation_year=2017;  citation_volume=7;  citation_pages=2195-207; " /><meta name="citation_reference" content="citation_title=Chromosomal evolution and speciation: a recombination-based approach;  citation_journal_title=New Phytol;  citation_year=2004;  citation_volume=161;  citation_pages=107-12; " /><meta name="citation_reference" content="citation_title=Chromosomal speciation and molecular divergence–accelerated evolution in rearranged chromosomes;  citation_journal_title=Science;  citation_year=2003;  citation_volume=300;  citation_pages=321-4; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/4/439/25501673/dsy015.pdf" /><meta name="description" content="Abstract.  Highly dense linkage maps enable positioning thousands of landmarks useful for anchoring the whole genome and for analysing genome properties. Turbot" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/4/439/5035912" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/4/439/5035912" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/4/439/5035912");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsy015';
    </script>
<title>
	Highly dense linkage maps from 31 full-sibling families of turbot (Scophthalmus maximus) provide insights into recombination patterns and chromosome rearrangements throughout a newly refined genome assembly | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./5035912" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZC8ATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWFyb3NvLCBGIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR0VORUFRVUEgUy5MLCAyMDA3IEx1Z28sIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGVybWlkYSwgTSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydGFtZW50byBkZSBab29sb2fDrWEsIEdlbsOpdGljYSB5IEFudHJvcG9sb2fDrWEgRsOtc2ljYSwgRmFjdWx0YWQgZGUgVmV0ZXJpbmFyaWEsIFVuaXZlcnNpZGFkZSBkZSBTYW50aWFnbyBkZSBDb21wb3N0ZWxhLCBMdWdvLCBTcGFpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1pbGzDoW4sIEEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHRU5FQVFVQSBTLkwsIDIwMDcgTHVnbywgU3BhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCbGFuY28sIEEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRhbWVudG8gZGUgWm9vbG9nw61hLCBHZW7DqXRpY2EgeSBBbnRyb3BvbG9nw61hIEbDrXNpY2EsIEZhY3VsdGFkIGRlIFZldGVyaW5hcmlhLCBVbml2ZXJzaWRhZGUgZGUgU2FudGlhZ28gZGUgQ29tcG9zdGVsYSwgTHVnbywgU3BhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXVyYSwgTSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydGFtZW50byBkZSBNZWpvcmEgR2Vuw6l0aWNhIEFuaW1hbCwgSW5zdGl0dXRvIE5hY2lvbmFsIGRlIEludmVzdGlnYWNpw7NuIHkgVGVjbm9sb2fDrWEgQWdyYXJpYSB5IEFsaW1lbnRhcmlhIChJTklBKSwgTWFkcmlkLCBTcGFpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZlcm7DoW5kZXosIEEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRhbWVudG8gZGUgTWVqb3JhIEdlbsOpdGljYSBBbmltYWwsIEluc3RpdHV0byBOYWNpb25hbCBkZSBJbnZlc3RpZ2FjacOzbiB5IFRlY25vbG9nw61hIEFncmFyaWEgeSBBbGltZW50YXJpYSAoSU5JQSksIE1hZHJpZCwgU3BhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEYWxsYSBSb3ZlcmUsIEciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIENvbXBhcmF0aXZlIEJpb21lZGljaW5lIGFuZCBGb29kIFNjaWVuY2UsIFVuaXZlcnNpdHkgb2YgUGFkb3ZhLCBMZWduYXJvLCBQYWRvdmEsIEl0YWx5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQmFyZ2VsbG9uaSwgTCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQ29tcGFyYXRpdmUgQmlvbWVkaWNpbmUgYW5kIEZvb2QgU2NpZW5jZSwgVW5pdmVyc2l0eSBvZiBQYWRvdmEsIExlZ25hcm8sIFBhZG92YSwgSXRhbHkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDYWJhbGVpcm8sIFMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDbHVzdGVyIGRlIEFjdWljdWx0dXJhIGRlIEdhbGljaWEgKFB1bnRhIGRvIENvdXNvKSwgQWd1acOxby1SaWJlaXJhLCBTcGFpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlZpbGxhbnVldmEsIEIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRhbWVudG8gZGUgTWVqb3JhIEdlbsOpdGljYSBBbmltYWwsIEluc3RpdHV0byBOYWNpb25hbCBkZSBJbnZlc3RpZ2FjacOzbiB5IFRlY25vbG9nw61hIEFncmFyaWEgeSBBbGltZW50YXJpYSAoSU5JQSksIE1hZHJpZCwgU3BhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCb3V6YSwgQyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydGFtZW50byBkZSBab29sb2fDrWEsIEdlbsOpdGljYSB5IEFudHJvcG9sb2fDrWEgRsOtc2ljYSwgRmFjdWx0YWQgZGUgVmV0ZXJpbmFyaWEsIFVuaXZlcnNpZGFkZSBkZSBTYW50aWFnbyBkZSBDb21wb3N0ZWxhLCBMdWdvLCBTcGFpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1hcnTDrW5leiwgUCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydGFtZW50byBkZSBab29sb2fDrWEsIEdlbsOpdGljYSB5IEFudHJvcG9sb2fDrWEgRsOtc2ljYSwgRmFjdWx0YWQgZGUgVmV0ZXJpbmFyaWEsIFVuaXZlcnNpZGFkZSBkZSBTYW50aWFnbyBkZSBDb21wb3N0ZWxhLCBMdWdvLCBTcGFpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iSGlnaGx5IGRlbnNlIGxpbmthZ2UgbWFwcyBmcm9tIDMxIGZ1bGwtc2libGluZyBmYW1pbGllcyBvZiB0dXJib3QgKFNjb3BodGhhbG11cyBtYXhpbXVzKSBwcm92aWRlIGluc2lnaHRzIGludG8gcmVjb21iaW5hdGlvbiBwYXR0ZXJucyBhbmQgY2hyb21vc29tZSByZWFycmFuZ2VtZW50cyB0aHJvdWdob3V0IGEgbmV3bHkgcmVmaW5lZCBnZW5vbWUgYXNzZW1ibHkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI0MzkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjQ1MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzeTAxNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE4LzA4LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGltcGFjdCBvZiBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZyB0ZWNobm9sb2d5IG9uIGdlbmV0aWNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xMzMtNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBhIGhpZ2gtZGVuc2l0eSBnZW5ldGljIG1hcCBhbmQgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY3VzIG1hcHBpbmcgaW4gdGhlIE1hbmlsYSBjbGFtIFJ1ZGl0YXBlcyBwaGlsaXBwaW5hcnVtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2kuIFJlcDsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MjI5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGFyZ2Utc2NhbGUgU05QIGRpc2NvdmVyeSBhbmQgY29uc3RydWN0aW9uIG9mIGEgaGlnaC1kZW5zaXR5IGdlbmV0aWMgbWFwIG9mIENvbG9zc29tYSBtYWNyb3BvbXVtIHRocm91Z2ggZ2Vub3R5cGluZy1ieS1zZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2kuIFJlcDsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NDYxMTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmcgcmV2b2x1dGlvbiBhbmQgaXRzIGltcGFjdCBvbiBnZW5vbWljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNTU7ICBjaXRhdGlvbl9wYWdlcz0yNy0zODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGVuIHllYXJzIG9mIG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIHRlY2hub2xvZ3k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTQxOC0yNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RyYXRlZ2llcyBmb3Igb3B0aW1pemluZyBCaW9OYW5vIGFuZCBEb3ZldGFpbCBleHBsb3JlZCB0aHJvdWdoIGEgc2Vjb25kIHJlZmVyZW5jZSBxdWFsaXR5IGFzc2VtYmx5IGZvciB0aGUgbGVndW1lIG1vZGVsLCBNZWRpY2FnbyB0cnVuY2F0dWxhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz01NzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNvbWJpbmF0aW9uIGluIHRoZSB0aHJlZXNwaW5lIHN0aWNrbGViYWNrIGdlbm9tZeKAlHBhdHRlcm5zIGFuZCBjb25zZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gRWNvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTMwMTQtMjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzaW5nIGxpbmthZ2UgbWFwcyB0byBjb3JyZWN0IGFuZCBzY2FmZm9sZCBkZSBub3ZvIGdlbm9tZSBhc3NlbWJsaWVzOiBtZXRob2RzLCBjaGFsbGVuZ2VzLCBhbmQgY29tcHV0YXRpb25hbCB0b29sczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnJvbnQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0yMjAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJSb2JsZWRvIEQuLCBQYWxhaW9rb3N0YXMgQy4sIEJhcmdlbGxvbmkgTC4sIE1hcnTDrW5leiBQLiwgSG91c3RvbiBSLiAgMjAxNyAsIEFwcGxpY2F0aW9ucyBvZiBnZW5vdHlwaW5nIGJ5IHNlcXVlbmNpbmcgaW4gYXF1YWN1bHR1cmUgYnJlZWRpbmcgYW5kIGdlbmV0aWNzLiBSZXYuIEFxdWFjLiAsIGRvaTogMTAuMTExMS9yYXEuMTIxOTMuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZyb20gMlIgdG8gM1I6IGV2aWRlbmNlIGZvciBhIGZpc2gtc3BlY2lmaWMgZ2Vub21lIGR1cGxpY2F0aW9uIChGU0dEKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvZXNzYXlzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9OTM3LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXBpZCBhbmQgY29zdC1lZmZlY3RpdmUgcG9seW1vcnBoaXNtIGlkZW50aWZpY2F0aW9uIGFuZCBnZW5vdHlwaW5nIHVzaW5nIHJlc3RyaWN0aW9uIHNpdGUgYXNzb2NpYXRlZCBETkEgKFJBRCkgbWFya2VyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTI0MC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXBpZCBTTlAgZGlzY292ZXJ5IGFuZCBnZW5ldGljIG1hcHBpbmcgdXNpbmcgc2VxdWVuY2VkIFJBRCBtYXJrZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9ZTMzNzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBnZW5ldGljIG1hcmtlciBkaXNjb3ZlcnkgYW5kIGdlbm90eXBpbmcgdXNpbmcgbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTQ5OS01MTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVF1YW50aXRhdGl2ZSBnZW5ldGljIGFuYWx5c2lzIG9mIHJldGluYWwgZGVnZW5lcmF0aW9uIGluIHRoZSBibGluZCBjYXZlZmlzaCBBc3R5YW5heCBtZXhpY2FudXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNTcyODEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MaW5rYWdlIG1hcHMgb2YgdGhlIEF0bGFudGljIHNhbG1vbiAoU2FsbW8gc2FsYXIpIGdlbm9tZSBkZXJpdmVkIGZyb20gUkFEIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTE2NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgaGlnaC1kZW5zaXR5IGxpbmthZ2UgbWFwIGZvciBBc3R5YW5heCBtZXhpY2FudXMgdXNpbmcgZ2Vub3R5cGluZy1ieS1zZXF1ZW5jaW5nIHRlY2hub2xvZ3k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUczIEdlbmVzIEdlbm9tZXMgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTI0MS01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoaWdoLWRlbnNpdHkgZ2VuZXRpYyBtYXAgYW5kIGdyb3d0aCByZWxhdGVkIFFUTCBtYXBwaW5nIGluIGJpZ2hlYWQgY2FycCAoSHlwb3BodGhhbG1pY2h0aHlzIG5vYmlsaXMpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2kuIFJlcDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9Mjg2NzkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GaW5lIG1hcHBpbmcgUVRMIGZvciByZXNpc3RhbmNlIHRvIFZOTiBkaXNlYXNlIHVzaW5nIGEgaGlnaC1kZW5zaXR5IGxpbmthZ2UgbWFwIGluIEFzaWFuIHNlYWJhc3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gUmVwOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0zMjEyMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBoaWdoLXJlc29sdXRpb24gcmVjb21iaW5hdGlvbiBtYXBzIGluIEFzaWFuIHNlYWJhc3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTYzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIGhpZ2gtZGVuc2l0eSBnZW5ldGljIGxpbmthZ2UgbWFwcyBhbmQgbWFwcGluZyBvZiBncm93dGgtcmVsYXRlZCBxdWFudGl0YXRpdmUgdHJhaWwgbG9jaSBpbiB0aGUgSmFwYW5lc2UgZmxvdW5kZXIgKFBhcmFsaWNodGh5cyBvbGl2YWNldXMpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTUwNDA0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1kZW5zaXR5IGdlbmV0aWMgbGlua2FnZSBtYXBwaW5nIGluIHR1cmJvdCAoU2NvcGh0aGFsbXVzIG1heGltdXMgTC4pIGJhc2VkIG9uIFNOUCBtYXJrZXJzIGFuZCBtYWpvciBzZXgtYW5kIGdyb3d0aC1yZWxhdGVkIHJlZ2lvbnMgZGV0ZWN0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPWUwMTIwNDEwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc3RydWN0aW9uIG9mIGEgaGlnaC1kZW5zaXR5IGxpbmthZ2UgbWFwIGFuZCBtYXBwaW5nIG9mIHNleCBkZXRlcm1pbmF0aW9uIGFuZCBncm93dGgtcmVsYXRlZCBsb2NpIGluIHRoZSBtYW5kYXJpbiBmaXNoIChTaW5pcGVyY2EgY2h1YXRzaSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTQ0NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBhbmQgYW5ub3RhdGlvbiBvZiBhIGhpZ2ggZGVuc2l0eSBTTlAgbGlua2FnZSBtYXAgb2YgdGhlIEF0bGFudGljIHNhbG1vbiAoU2FsbW8gc2FsYXIpIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RzMgR2VuZXMgR2Vub21lcyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MjE3My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljIHByZWRpY3Rpb24gb2YgcmVzaXN0YW5jZSB0byBwYXN0ZXVyZWxsb3NpcyBpbiBnaWx0aGVhZCBzZWEgYnJlYW0gKFNwYXJ1cyBhdXJhdGEpIHVzaW5nIDJiLVJBRCBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HMyBHZW5lcyBHZW5vbWVzIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0zNjkzLTcwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3JpdGVyaWEgdG8gb3B0aW1pemUgZGVzaWducyBmb3IgZGV0ZWN0aW9uIGFuZCBlc3RpbWF0aW9uIG9mIGxpbmthZ2UgYmV0d2VlbiBtYXJrZXIgbG9jaSBmcm9tIHNlZ3JlZ2F0aW5nIHBvcHVsYXRpb25zIGNvbnRhaW5pbmcgc2V2ZXJhbCBmYW1pbGllczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VGhlb3JldC4gQXBwbC4gR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9ODYtODY7ICBjaXRhdGlvbl9wYWdlcz0yNjktODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1Kb2huIFdpbGV5ICZhbXA7IFNvbnMsIEx0ZCwgQ2hpY2hlc3RlciwgVUs7ICBjaXRhdGlvbl90aXRsZT1CaW9pbmZvcm1hdGljcyBpbiBBcXVhY3VsdHVyZTogUHJpbmNpcGxlcyBhbmQgTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3BhZ2VzPTM1Ni03OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9aW4gdGhlIEF0bGFudGljIENvZCBnZW5vbWUgcmVwcmVzZW50IHBvbHltb3JwaGljIGNocm9tb3NvbWFsIHJlYXJyYW5nZW1lbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MTAxMi0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9taWMgYmFzaXMgb2YgYWRhcHRpdmUgZXZvbHV0aW9uIGluIHRocmVlc3BpbmUgc3RpY2tsZWJhY2tzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDg0OyAgY2l0YXRpb25fcGFnZXM9NTUtNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVR1cmJvdC4gQXF1YVRyYWNlIFNwZWNpZXMgTGVhZmxldDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgdGhlIG1ham9yIHNleC1kZXRlcm1pbmluZyByZWdpb24gb2YgdHVyYm90IChTY29waHRoYWxtdXMgbWF4aW11cyk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE4MzsgIGNpdGF0aW9uX3BhZ2VzPTE0NDMtNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcHJvdmluZyB0aGUgaW5mZXJlbmNlIG9mIHBvcHVsYXRpb24gZ2VuZXRpYyBzdHJ1Y3R1cmUgaW4gdGhlIHByZXNlbmNlIG9mIHJlbGF0ZWQgaW5kaXZpZHVhbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0LiBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9OTY7ICBjaXRhdGlvbl9wYWdlcz1lMDAzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmluZSBtYXBwaW5nIGFuZCBldm9sdXRpb24gb2YgdGhlIG1ham9yIHNleCBkZXRlcm1pbmluZyByZWdpb24gaW4gdHVyYm90IChTY29waHRoYWxtdXMgbWF4aW11cyk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUczIEdlbmVzIEdlbm9tZXMgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTE4NzEtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgZXhwcmVzc2lvbiBhbmFseXNpcyBhdCB0aGUgb25zZXQgb2Ygc2V4IGRpZmZlcmVudGlhdGlvbiBpbiB0dXJib3QgKFNjb3BodGhhbG11cyBtYXhpbXVzKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9OTczIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hvbGUgZ2Vub21lIHNlcXVlbmNpbmcgb2YgdHVyYm90IChTY29waHRoYWxtdXMgbWF4aW11czsgUGxldXJvbmVjdGlmb3JtZXMpOiBhIGZpc2ggYWRhcHRlZCB0byBkZW1lcnNhbCBsaWZlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MTgxLTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlZ3JhdGl2ZSB0cmFuc2NyaXB0b21lLCBnZW5vbWUgYW5kIHF1YW50aXRhdGl2ZSB0cmFpdCBsb2NpIHJlc291cmNlcyBpZGVudGlmeSBzaW5nbGUgbnVjbGVvdGlkZSBwb2x5bW9ycGhpc21zIGluIGNhbmRpZGF0ZSBnZW5lcyBmb3IgZ3Jvd3RoIHRyYWl0cyBpbiB0dXJib3Q7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUlKTVM7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0yNDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG1pY3Jvc2F0ZWxsaXRlIGdlbmV0aWMgbWFwIG9mIHRoZSB0dXJib3QgKFNjb3BodGhhbG11cyBtYXhpbXVzKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTc3OyAgY2l0YXRpb25fcGFnZXM9MjQ1Ny02NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBsaW5rYWdlIG1hcHBpbmcgb2YgdHVyYm90IChTY29waHRoYWxtdXMgbWF4aW11cyBMLikgdXNpbmcgbWljcm9zYXRlbGxpdGUgbWFya2VycyBhbmQgaXRzIGFwcGxpY2F0aW9uIGluIFFUTCBhbmFseXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXF1YWN1bHR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MzA4OyAgY2l0YXRpb25fcGFnZXM9ODktMTAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBFeHByZXNzZWQgU2VxdWVuY2UgVGFnIChFU1QpLWVucmljaGVkIGdlbmV0aWMgbWFwIG9mIHR1cmJvdCAoU2NvcGh0aGFsbXVzIG1heGltdXMpOiBhIHVzZWZ1bCBmcmFtZXdvcmsgZm9yIGNvbXBhcmF0aXZlIGdlbm9taWNzIGFjcm9zcyBtb2RlbCBhbmQgZmFybWVkIHRlbGVvc3RzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz01NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBpbGF0aW9uIG9mIG1hcHBpbmcgcmVzb3VyY2VzIGluIHR1cmJvdCAoU2NvcGh0aGFsbXVzIG1heGltdXMpOiBhIG5ldyBpbnRlZ3JhdGVkIGNvbnNlbnN1cyBnZW5ldGljIG1hcDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXF1YWN1bHR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDE04oCTNDE1OyAgY2l0YXRpb25fcGFnZXM9MTktMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNlbnRyb21lcmUtbGlua2FnZSBpbiB0aGUgdHVyYm90IChTY29waHRoYWxtdXMgbWF4aW11cykgdGhyb3VnaCBoYWxmLXRldHJhZCBhbmFseXNpcyBpbiBkaXBsb2lkIG1laW9neW5vZ2VuZXRpY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFxdWFjdWx0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI4MDsgIGNpdGF0aW9uX3BhZ2VzPTgxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIG51Y2xlb3RpZGUgc2VxdWVuY2Ugb2YgSmFwYW5lc2UgZmxvdW5kZXIgKFBhcmFsaWNodGh5cyBvbGl2YWNldXMpIG1pdG9jaG9uZHJpYWwgZ2Vub21lOiBzdHJ1Y3R1cmFsIHByb3BlcnRpZXMgYW5kIGN1ZSBmb3IgcmVzb2x2aW5nIHRlbGVvc3RlYW4gcmVsYXRpb25zaGlwOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBIZXJlZDsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT05MTsgIGNpdGF0aW9uX3BhZ2VzPTI3MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZS1nZW5vbWUgc2VxdWVuY2Ugb2YgYSBmbGF0ZmlzaCBwcm92aWRlcyBpbnNpZ2h0cyBpbnRvIFpXIHNleCBjaHJvbW9zb21lIGV2b2x1dGlvbiBhbmQgYWRhcHRhdGlvbiB0byBhIGJlbnRoaWMgbGlmZXN0eWxlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9MjUzLTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIGFuZCB0cmFuc2NyaXB0b21lIG9mIEphcGFuZXNlIGZsb3VuZGVyIHByb3ZpZGUgaW5zaWdodHMgaW50byBmbGF0ZmlzaCBhc3ltbWV0cnk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9NDk7ICBjaXRhdGlvbl9wYWdlcz0xMTktMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIGFuZCB2YWxpZGF0aW9uIG9mIHNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbXMgYXMgdG9vbHMgdG8gZGV0ZWN0IGh5YnJpZGl6YXRpb24gYW5kIHBvcHVsYXRpb24gc3RydWN0dXJlIGluIGZyZXNod2F0ZXIgc3RpbmdyYXlzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEVjb2wuIFJlc291cjsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTU1MC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT0yYi1SQUQ6IGEgc2ltcGxlIGFuZCBmbGV4aWJsZSBtZXRob2QgZm9yIGdlbm9tZS13aWRlIGdlbm90eXBpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ODA4LTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdGFja3M6IGJ1aWxkaW5nIGFuZCBnZW5vdHlwaW5nIGxvY2kgZGUgbm92byBmcm9tIHNob3J0LXJlYWQgc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HMyAoQmV0aGVzZGEpICk7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTE3MS04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhY2tzOiBhbiBhbmFseXNpcyB0b29sIHNldCBmb3IgcG9wdWxhdGlvbiBnZW5vbWljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBFY29sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MzEyNC00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VWx0cmFmYXN0IGFuZCBtZW1vcnktZWZmaWNpZW50IGFsaWdubWVudCBvZiBzaG9ydCBETkEgc2VxdWVuY2VzIHRvIHRoZSBodW1hbiBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9UjI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iVmFuIE9vaWplbiBKLiBXLiAgMjAwNiAsIEpvaW5NYXAgwq4gNC4gU29mdHdhcmUgZm9yIHRoZSBjYWxjdWxhdGlvbiBvZiBnZW5ldGljIGxpbmthZ2UgbWFwcyBpbiBleHBlcmltZW50YWwgcG9wdWxhdGlvbnMsIHZvbC4gMzMuIEt5YXptYSBCVjogV2FnZW5pbmdlbi4gIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWNjdXJhdGUgY29uc3RydWN0aW9uIG9mIGNvbnNlbnN1cyBnZW5ldGljIG1hcHMgdmlhIGludGVnZXIgbGluZWFyIHByb2dyYW1taW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JRUVFL0FDTSBUcmFucy4gQ29tcHV0LiBCaW9sLiBCaW9pbmZvcm07ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTM4MS05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwQ2hhcnQ6IHNvZnR3YXJlIGZvciB0aGUgZ3JhcGhpY2FsIHByZXNlbnRhdGlvbiBvZiBsaW5rYWdlIG1hcHMgYW5kIFFUTHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEhlcmVkOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTkzOyAgY2l0YXRpb25fcGFnZXM9NzctODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9S2Fyb3R5cGljIGNoYXJhY3Rlcml6YXRpb24gb2YgdHVyYm90IChTY29waHRoYWxtdXMgbWF4aW11cykgd2l0aCBjb252ZW50aW9uYWwsIGZsdW9yb2Nocm9tZSBhbmQgcmVzdHJpY3Rpb24gZW5kb251Y2xlYXNlLWJhbmRpbmcgdGVjaG5pcXVlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWFyLiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTEyMDsgIGNpdGF0aW9uX3BhZ2VzPTYwOS0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iVGVhbSBSLiBDLiAgMjAxMyAsIFI6IGEgbGFuZ3VhZ2UgYW5kIGVudmlyb25tZW50IGZvciBzdGF0aXN0aWNhbCBjb21wdXRpbmcuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkhhcnJpcyBSLiBTLiAgMjAwNyAsIEltcHJvdmVkIHBhaXJ3aXNlIGFsaWdubWVudCBvZiBnZW5vbWljIEROQS4gUGhEIFRoZXNpcywgVGhlIFBlbm5zeWx2YW5pYSBTdGF0ZSBVbml2ZXJzaXR5LCBQZW5uc3lsdmFuaWEuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZhbGlkYXRpb24gb2Ygc2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtIChTTlApIG1hcmtlcnMgZnJvbSBhbiBpbW11bmUgRXhwcmVzc2VkIFNlcXVlbmNlIFRhZyAoRVNUKSB0dXJib3QsIFNjb3BodGhhbG11cyBtYXhpbXVzLCBkYXRhYmFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXF1YWN1bHR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MzEzOyAgY2l0YXRpb25fcGFnZXM9MzEtNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IGFuZCB2YWxpZGF0aW9uIG9mIHNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbXMgKFNOUHMpIG1hcmtlcnMgZnJvbSB0d28gdHJhbnNjcmlwdG9tZSA0NTQtcnVucyBvZiB0dXJib3QgKFNjb3BodGhhbG11cyBtYXhpbXVzKSB1c2luZyBoaWdoLXRocm91Z2hwdXQgZ2Vub3R5cGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SUpNUzsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTU2OTQtNzExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24gZnJvbSBmaXNoIHRvIG1hbW1hbHMgYnkgZ2VuZSBkdXBsaWNhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SGVyZWRpdGFzOyAgY2l0YXRpb25feWVhcj0xOTY4OyAgY2l0YXRpb25fdm9sdW1lPTU5OyAgY2l0YXRpb25fcGFnZXM9MTY5LTg3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gb2YgYSBoaWdoLWRlbnNpdHksIGhpZ2gtcmVzb2x1dGlvbiBnZW5ldGljIG1hcCBhbmQgaXRzIGludGVncmF0aW9uIHdpdGggQkFDLWJhc2VkIHBoeXNpY2FsIG1hcCBpbiBjaGFubmVsIGNhdGZpc2g7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0zOS01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBuZXcgU05QLWJhc2VkIHZpc2lvbiBvZiB0aGUgZ2VuZXRpY3Mgb2Ygc2V4IGRldGVybWluYXRpb24gaW4gRXVyb3BlYW4gc2VhIGJhc3MgKERpY2VudHJhcmNodXMgbGFicmF4KTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXQuIFNlbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT00NzsgIGNpdGF0aW9uX3BhZ2VzPTY4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZWdyYXRpb24gb2YgbGlua2FnZSBtYXBzIGZvciB0aGUgYW1waGlkaXBsb2lkIEJyYXNzaWNhIG5hcHVzIGFuZCBjb21wYXJhdGl2ZSBtYXBwaW5nIHdpdGggQXJhYmlkb3BzaXMgYW5kIEJyYXNzaWNhIHJhcGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTEwMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbXVsdGktcG9wdWxhdGlvbiBjb25zZW5zdXMgZ2VuZXRpYyBtYXAgcmV2ZWFscyBpbmNvbnNpc3RlbnQgbWFya2VyIG9yZGVyIGFtb25nIG1hcHMgbGlrZWx5IGF0dHJpYnV0ZWQgdG8gc3RydWN0dXJhbCB2YXJpYXRpb25zIGluIHRoZSBhcHBsZSBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lNDc4NjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBhbmQgaW1wbGVtZW50YXRpb24gb2YgaGlnaC10aHJvdWdocHV0IFNOUCBnZW5vdHlwaW5nIGluIGJhcmxleTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NTgyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUVsc2V2aWVyLCBMb25kb247ICBjaXRhdGlvbl90aXRsZT1HZW5vbWljcyBpbiBBcXVhY3VsdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3BhZ2VzPTEzMS02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZmVyZW5jZSBpbiB0aGUgYmVoYXZpb3VyIG9mIGV1LWFuZCBoZXRlcm8tY2hyb21hdGluOiBjcm9zc2luZy1vdmVyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5NjU7ICBjaXRhdGlvbl92b2x1bWU9MjA1OyAgY2l0YXRpb25fcGFnZXM9NzI1LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVncmF0aW9uIG9mIGhpZ2gtcmVzb2x1dGlvbiBwaHlzaWNhbCBhbmQgZ2VuZXRpYyBtYXAgcmV2ZWFscyBkaWZmZXJlbnRpYWwgcmVjb21iaW5hdGlvbiBmcmVxdWVuY3kgYmV0d2VlbiBjaHJvbW9zb21lcyBhbmQgdGhlIGdlbm9tZSBhc3NlbWJsaW5nIHF1YWxpdHkgaW4gY3VjdW1iZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNjI2NzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXgtc3BlY2lmaWMgcmVjb21iaW5hdGlvbiByYXRlcyBpbiB6ZWJyYWZpc2ggKERhbmlvIHJlcmlvKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTYwOyAgY2l0YXRpb25fcGFnZXM9NjQ5LTU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGxpbmthZ2UgbWFwIG9mIHRoZSBBdGxhbnRpYyBzYWxtb24gKFNhbG1vIHNhbGFyKSBiYXNlZCBvbiBFU1QtZGVyaXZlZCBTTlAgbWFya2VyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0yMjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbnZpcm9ubWVudGFsIHNlbGVjdGlvbiBvbiB0cmFuc2NyaXB0b21lLWRlcml2ZWQgU05QcyBpbiBhIGhpZ2ggZ2VuZSBmbG93IG1hcmluZSBmaXNoLCB0aGUgQXRsYW50aWMgaGVycmluZyAoQ2x1cGVhIGhhcmVuZ3VzKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBFY29sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MzY4Ni03MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlY29tYmluYXRpb24gZGlmZmVyZW5jZSBiZXR3ZWVuIHNleGVzOiBhIHJvbGUgZm9yIGhhcGxvaWQgc2VsZWN0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPWU2MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN5dG9sb2dpY2FsIHN0dWRpZXMgb2YgaHVtYW4gbWVpb3Npczogc2V4LXNwZWNpZmljIGRpZmZlcmVuY2VzIGluIHJlY29tYmluYXRpb24gb3JpZ2luYXRlIGF0LCBvciBwcmlvciB0bywgZXN0YWJsaXNobWVudCBvZiBkb3VibGUtc3RyYW5kIGJyZWFrczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWU4NTA3NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeWxvZ2VuZXRpYyBhbmFseXNpcyBvZiBmbGF0ZmlzaCAoT3JkZXIgUGxldXJvbmVjdGlmb3JtZXMpIGJhc2VkIG9uIG1pdG9jaG9uZHJpYWwgMTZzIHJETkEgc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2kuIE1hcjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT02OTsgIGNpdGF0aW9uX3BhZ2VzPTUzMS00MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXJlIGZsYXRmaXNoZXMgKFBsZXVyb25lY3RpZm9ybWVzKSBtb25vcGh5bGV0aWM/LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBQaHlsb2dlbmV0LiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTY5OyAgY2l0YXRpb25fcGFnZXM9NjY0LTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXZvbHV0aW9uYXJ5IG9yaWdpbiBvZiBmbGF0ZmlzaCBhc3ltbWV0cnk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00NTQ7ICBjaXRhdGlvbl9wYWdlcz0yMDktMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeWxvZ2Vub21pYyBhbmFseXNpcyBvZiBjYXJhbmdpbW9ycGggZmlzaGVzIHJldmVhbHMgZmxhdGZpc2ggYXN5bW1ldHJ5IGFyb3NlIGluIGEgYmxpbmsgb2YgdGhlIGV2b2x1dGlvbmFyeSBleWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBFdm9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MjI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3JpZ2luIG9mIGh1bWFuIGNocm9tb3NvbWUgMjogYW4gYW5jZXN0cmFsIHRlbG9tZXJlLXRlbG9tZXJlIGZ1c2lvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9ODg7ICBjaXRhdGlvbl9wYWdlcz05MDUxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN5dG9nZW5ldGljIGNoYXJhY3Rlcml6YXRpb24gb2Ygc2l4IHNwZWNpZXMgb2YgZmxhdGZpc2hlcyB3aXRoIGNvbW1lbnRzIHRvIGthcnlvdHlwZSBkaWZmZXJlbnRpYXRpb24gcGF0dGVybnMgaW4gUGxldXJvbmVjdGlmb3JtZXMgKFRlbGVvc3RlaSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEZpc2ggQmlvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT03MDsgIGNpdGF0aW9uX3BhZ2VzPTEtMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbGQgZnVzaW9uOiBtYXNzaXZlIGthcnlvdHlwZSBldm9sdXRpb24gaW4gdGhlIEFudGFyY3RpYyBCdWxsaGVhZCBOb3RvdGhlbiBOb3RvdGhlbmlhIGNvcmlpY2VwczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RzMgR2VuZXMgR2Vub21lcyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MjE5NS0yMDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNocm9tb3NvbWFsIGV2b2x1dGlvbiBhbmQgc3BlY2lhdGlvbjogYSByZWNvbWJpbmF0aW9uLWJhc2VkIGFwcHJvYWNoOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OZXcgUGh5dG9sOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE2MTsgIGNpdGF0aW9uX3BhZ2VzPTEwNy0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hyb21vc29tYWwgc3BlY2lhdGlvbiBhbmQgbW9sZWN1bGFyIGRpdmVyZ2VuY2XigJNhY2NlbGVyYXRlZCBldm9sdXRpb24gaW4gcmVhcnJhbmdlZCBjaHJvbW9zb21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMDA7ICBjaXRhdGlvbl9wYWdlcz0zMjEtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI1LzQvNDM5LzI1NTAxNjczL2RzeTAxNS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgSGlnaGx5IGRlbnNlIGxpbmthZ2UgbWFwcyBlbmFibGUgcG9zaXRpb25pbmcgdGhvdXNhbmRzIG9mIGxhbmRtYXJrcyB1c2VmdWwgZm9yIGFuY2hvcmluZyB0aGUgd2hvbGUgZ2Vub21lIGFuZCBmb3IgYW5hbHlzaW5nIGdlbm9tZSBwcm9wZXJ0aWVzLiBUdXJib3QiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTUwMzU5MTImeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rkv1l6UW1VKcZAAfEcjfu5vy0yZv8=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f439%2f5035912" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f439%2f5035912" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=109685" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/4/1/m_dnares_25_4cover.png?Expires=2147483647&amp;Signature=lD6dhAB99HpTltGbT4-zO-h9Z6msKpBpjdlZuCIFAdO0RvvEM~IgMElNp5bE6Rdf~OSfdWzB2y8SIAkqvb6Fc-0GRLo0HVHIHIXLjJF8ZJ94ZrTCEDv2Gn-92BvjK~Ebx3LV-ePsMx8j4josbKy1FscEJ3-qj3zB2TEHnK~iAQqommn5aOw82hhC1dOH5f1P15MUh9o1QIOhJ9WAnJwCFYDxRE0ZmdX~qCpKaFm8XPRL-O71Rxc1eeV8iTb9yQzgpbw2OwX1g4mQ5iALY~7Bk3L2RgUkQZIzgNqb8ANqfHjAsIlrzFHcKAl7tTiKtoGbsHvDqkXNPcKjgjrdKdIXQw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/4">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="120122068">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122068">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122070">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122070">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122076">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122076">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122097">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122097">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122122">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122122">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122130">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122130">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122132">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122132">Accession numbers</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122134">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122134">Conflict of interest</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122136">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122136">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/4/421/5020793">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Highly dense linkage maps from 31 full-sibling families of turbot (<em>Scophthalmus maximus</em>) provide insights into recombination patterns and chromosome rearrangements throughout a newly refined genome assembly
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">F Maroso</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">GENEAQUA S.L, 2007 Lugo, Spain</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy015-cor1" data-open="dsy015-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +34 982 822423. Email: <a href="mailto:francesco.maroso@gmail.com" target="_blank">francesco.maroso@gmail.com</a> (F.M.); Tel. +34982822425. Fax. +982822001. Email: <a href="mailto:paulino.martinez@usc.es" target="_blank">paulino.martinez@usc.es</a> (P.M.)
        </div>


        <div class="info-card-location">
            <a href="http://orcid.org/0000-0003-3524-5916">
                <img class="orchid-icon" alt="orchid id" src="/UI/app/img/mini-icon.png"/>&nbsp;&nbsp;http://orcid.org/0000-0003-3524-5916
           </a>
        </div>

        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=F+Maroso">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Maroso F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Maroso F&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">F Maroso</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">M Hermida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Departamento de Zoología, Genética y Antropología Física, Facultad de Veterinaria, Universidade de Santiago de Compostela, Lugo, Spain</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=M+Hermida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hermida M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hermida M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">M Hermida</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">A Mill&#225;n</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">GENEAQUA S.L, 2007 Lugo, Spain</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=A+Mill%c3%a1n">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mill&#225;n A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mill&#225;n A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">A Millán</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">A Blanco</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Departamento de Zoología, Genética y Antropología Física, Facultad de Veterinaria, Universidade de Santiago de Compostela, Lugo, Spain</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=A+Blanco">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Blanco A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Blanco A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">A Blanco</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">M Saura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Departamento de Mejora Genética Animal, Instituto Nacional de Investigación y Tecnología Agraria y Alimentaria (INIA), Madrid, Spain</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=M+Saura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Saura M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Saura M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">M Saura</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">A Fern&#225;ndez</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Departamento de Mejora Genética Animal, Instituto Nacional de Investigación y Tecnología Agraria y Alimentaria (INIA), Madrid, Spain</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=A+Fern%c3%a1ndez">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fern&#225;ndez A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fern&#225;ndez A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">A Fernández</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">G Dalla Rovere</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Comparative Biomedicine and Food Science, University of Padova, Legnaro, Padova, Italy</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=G+Dalla+Rovere">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dalla Rovere G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dalla Rovere G&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">G Dalla Rovere</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">L Bargelloni</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Comparative Biomedicine and Food Science, University of Padova, Legnaro, Padova, Italy</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=L+Bargelloni">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bargelloni L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bargelloni L&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">L Bargelloni</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">S Cabaleiro</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Cluster de Acuicultura de Galicia (Punta do Couso), Aguiño-Ribeira, Spain</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=S+Cabaleiro">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cabaleiro S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cabaleiro S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">S Cabaleiro</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">B Villanueva</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Departamento de Mejora Genética Animal, Instituto Nacional de Investigación y Tecnología Agraria y Alimentaria (INIA), Madrid, Spain</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=B+Villanueva">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Villanueva B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Villanueva B&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">B Villanueva</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">C Bouza</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Departamento de Zoología, Genética y Antropología Física, Facultad de Veterinaria, Universidade de Santiago de Compostela, Lugo, Spain</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=C+Bouza">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bouza C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bouza C&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">C Bouza</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">P Mart&#237;nez</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Departamento de Zoología, Genética y Antropología Física, Facultad de Veterinaria, Universidade de Santiago de Compostela, Lugo, Spain</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy015-cor1" data-open="dsy015-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +34 982 822423. Email: <a href="mailto:francesco.maroso@gmail.com" target="_blank">francesco.maroso@gmail.com</a> (F.M.); Tel. +34982822425. Fax. +982822001. Email: <a href="mailto:paulino.martinez@usc.es" target="_blank">paulino.martinez@usc.es</a> (P.M.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=P+Mart%c3%adnez">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mart&#237;nez P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mart&#237;nez P&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">P Martínez</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy015-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Toshihiko Shiroishi</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 4, 1 August 2018, Pages 439–450, <a href='https://doi.org/10.1093/dnares/dsy015'>https://doi.org/10.1093/dnares/dsy015</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">11 June 2018</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">15 February 2018</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">05 May 2018</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5035912" href="/dnaresearch/article-pdf/25/4/439/25501673/dsy015.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>F Maroso, M Hermida, A Millán, A Blanco, M Saura, A Fernández, G Dalla Rovere, L Bargelloni, S Cabaleiro, B Villanueva, C Bouza, P Martínez;  Highly dense linkage maps from 31 full-sibling families of turbot (<em>Scophthalmus maximus</em>) provide insights into recombination patterns and chromosome rearrangements throughout a newly refined genome assembly, <em>DNA Research</em>, Volume 25, Issue 4, 1 August 2018, Pages 439–450, <a href='https://doi.org/10.1093/dnares/dsy015'>https://doi.org/10.1093/dnares/dsy015</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/5035912?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/5035912?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/5035912?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/5035912?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/5035912?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/5035912?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/5035912?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Highly%20dense%20linkage%20maps%20from%2031%20full-sibling%20families%20of%20turbot%20(%26lt%3Bem%26gt%3BScophthalmus%20maximus%26lt%3B%2Fem%26gt%3B)%20provide%20insights%20into%20recombination%20patterns%20and%20chromosome%20rearrangements%20throughout%20a%20newly%20refined%20genome%20assembly&amp;publicationDate=2018-06-11&amp;volumeNum=25&amp;issueNum=4&amp;author=Maroso%2C%20F%3B%20Hermida%2C%20M&amp;startPage=439&amp;endPage=450&amp;contentId=10.1093%2Fdnares%2Fdsy015&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f439%2f5035912" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=120122068 id="120122068" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Highly dense linkage maps enable positioning thousands of landmarks useful for anchoring the whole genome and for analysing genome properties. Turbot is the most important cultured flatfish worldwide and breeding programs in the fifth generation of selection are targeted to improve growth rate, obtain disease resistant broodstock and understand sex determination to control sex ratio. Using a Restriction-site Associated DNA approach, we genotyped 18,214 single nucleotide polymorphism in 1,268 turbot individuals from 31 full-sibling families. Individual linkage maps were combined to obtain a male, female and species consensus maps. The turbot consensus map contained 11,845 markers distributed across 22 linkage groups representing a total normalised length of 3,753.9 cM. The turbot genome was anchored to this map, and scaffolds representing 96% of the assembly were ordered and oriented to obtain the expected 22 megascaffolds according to its karyotype. Recombination rate was lower in males, especially around centromeres, and pairwise comparison of 44 individual maps suggested chromosome polymorphism at specific genomic regions. Genome comparison across flatfish provided new evidence on karyotype reorganisations occurring across the evolution of this fish group.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Scophthalmus maximus&quot;"><em>Scophthalmus maximus</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;2b-RAD sequencing&quot;">2b-RAD sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;linkage mapping&quot;">linkage mapping</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome assembly&quot;">genome assembly</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;chromosome reorganisations&quot;">chromosome reorganisations</a></div>                            <h2 scrollto-destination=120122070 id="120122070" class="section-title" >1. Introduction</h2>
<p>The information provided by genomes has become invaluable for research fields that deal with animal production, conservation biology or evolution.<a reveal-id="dsy015-B1" data-open="dsy015-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> High-density (HD) genetic maps provide information on genome organisation by establishing the cartography of thousands of markers, which can aid to disentangle the genetic architecture of productive or evolutionary traits. Moreover, these maps represent an essential tool for comparative genomics and can facilitate whole genome assembly.<a reveal-id="dsy015-B2" data-open="dsy015-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsy015-B3" data-open="dsy015-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> The advent of next generation sequencing (NGS) has dramatically reduced the cost and time required for genomic analysis,<a reveal-id="dsy015-B4" data-open="dsy015-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsy015-B5" data-open="dsy015-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> thus contributing to a fast increase of genomic resources. Nevertheless, while DNA sequencing has become an automated task at platforms, genome assembling still relies on the hierarchical ordering of short DNA fragments (contigs, scaffolds), currently being facilitated by the novel long-read assembling approaches (Nanopore, PacBio, Dovetail, Bionano).<a reveal-id="dsy015-B6" data-open="dsy015-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> HD genetic mapping can assist the process of genome assembly through positioning and ordering scaffolds or validating bioinformatic assembly.<a reveal-id="dsy015-B7" data-open="dsy015-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsy015-B8" data-open="dsy015-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsy015-B9" data-open="dsy015-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> This is a particularly challenging task in teleosts, considering the ancestral teleost-specific whole genome duplication event in this group of fish.<a reveal-id="dsy015-B10" data-open="dsy015-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a></p><p>A huge number of markers and samples can now be analysed at very low cost using genotyping by sequencing (GBS) methods,<a reveal-id="dsy015-B4" data-open="dsy015-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsy015-B5" data-open="dsy015-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsy015-B9" data-open="dsy015-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> which enables to simultaneously identify and genotype thousands of single nucleotide polymorphisms (SNPs) across samples. A popular GBS method is Restriction Associated DNA sequencing (RADseq), a technique that combines the power of NGS with the simplification of genomes through restriction enzyme digestion.<a reveal-id="dsy015-B11 dsy015-B12 dsy015-B13" data-open="dsy015-B11 dsy015-B12 dsy015-B13" class="link link-ref link-reveal xref-bibr"><sup>11–13</sup></a> RADseq is being increasingly used for the construction of HD genetic maps in fish comprising thousands of markers (e.g. Refs <a reveal-id="dsy015-B14 dsy015-B15 dsy015-B16 dsy015-B17 dsy015-B18 dsy015-B19" data-open="dsy015-B14 dsy015-B15 dsy015-B16 dsy015-B17 dsy015-B18 dsy015-B19" class="link link-ref link-reveal xref-bibr"><sup>14–19</sup></a>). These maps have allowed the identification of genomic regions associated with productive traits<a reveal-id="dsy015-B20" data-open="dsy015-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsy015-B21" data-open="dsy015-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> or with adaptation to specific environments.<a reveal-id="dsy015-B14" data-open="dsy015-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsy015-B16" data-open="dsy015-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> The pattern of samples and families used in fish for map construction has been quite similar, and only very recently, the number of families has increased.<a reveal-id="dsy015-B19" data-open="dsy015-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><sup>,</sup><a reveal-id="dsy015-B22" data-open="dsy015-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> However, when many families have been analysed, the number of offspring per family has been much smaller (e.g. Tsai et al. 2016<a reveal-id="dsy015-B23" data-open="dsy015-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a>: 60 families with less than 10 offspring each; Palaiokostas et al. 2016<a reveal-id="dsy015-B24" data-open="dsy015-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a>: 75 families with 10 offspring on average). This is a way to achieve a consensus reference map at species level by averaging inter-individual variation, but the low family sampling size can result in suboptimal estimation of inter-marker distances and mapping order.<a reveal-id="dsy015-B25" data-open="dsy015-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a><sup>,</sup><a reveal-id="dsy015-B26" data-open="dsy015-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a></p><p>The presence of genomic reorganisations and variation of mapping parameters across individual or sex-specific maps has been hardly addressed to date in fish despite its importance from both evolutionary and practical perspectives. For example, inversions might aid to maintain blocks of coadapted genetic variants at specific environments, such as reported in fish like cod<a reveal-id="dsy015-B27" data-open="dsy015-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> or stickleback.<a reveal-id="dsy015-B7" data-open="dsy015-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsy015-B28" data-open="dsy015-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> Moreover, sex-specific mapping features are important when tracking markers associated to economically relevant traits since recombination patterns in males and females may differ and thus, they should be considered for accurate evaluations. NGS technologies now enable studying intra-specific variation on genome organisation and recombination with enough depth to improve the analysis of associations with traits of interest. This would aid for a more accurate mining to identify responsible genes and mutations and choose the best strategy for marker assisted selection.</p><p>The turbot (<em>Scophthalmus maximus</em>) is the main worldwide aquaculture flatfish,<a reveal-id="dsy015-B9" data-open="dsy015-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> People’s Republic of China being currently the most important producer (&gt;60,000 tons/year<a reveal-id="dsy015-B29" data-open="dsy015-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a>). Genetic breeding programs in this species began in the 90s. Nowadays, a broad battery of genetic markers and genomic tools are available in turbot, which can aid to understand the genetic basis of the main economically important traits (growth rate, disease resistance, sex control) and for applying a more efficient selection.<a reveal-id="dsy015-B30 dsy015-B31 dsy015-B32 dsy015-B33 dsy015-B34 dsy015-B35" data-open="dsy015-B30 dsy015-B31 dsy015-B32 dsy015-B33 dsy015-B34 dsy015-B35" class="link link-ref link-reveal xref-bibr"><sup>30–35</sup></a> The first genetic maps in turbot, based on anonymous or gene-associated microsatellites and SNPs,<a reveal-id="dsy015-B36 dsy015-B37 dsy015-B38" data-open="dsy015-B36 dsy015-B37 dsy015-B38" class="link link-ref link-reveal xref-bibr"><sup>36–38</sup></a> were later integrated in a consensus map.<a reveal-id="dsy015-B38" data-open="dsy015-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> More recently, a high-density map was reported by Wang et al.,<a reveal-id="dsy015-B21" data-open="dsy015-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> but the scarce information provided on genetic marker-associated sequences made it not feasible to integrate with previously reported maps.<a reveal-id="dsy015-B9" data-open="dsy015-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Centromeres have been located in the turbot genetic map using half-tetrad analysis<a reveal-id="dsy015-B40" data-open="dsy015-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a>, and recently, a high coverage draft genome has been assembled and anchored to the reported consensus map.<a reveal-id="dsy015-B36" data-open="dsy015-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Moreover, the development of genomic resources for other important flatfish species<a reveal-id="dsy015-B41 dsy015-B42 dsy015-B43" data-open="dsy015-B41 dsy015-B42 dsy015-B43" class="link link-ref link-reveal xref-bibr"><sup>41–43</sup></a> and their integration in a common framework has allowed comparative analyses to transfer genomic information on production-related traits between species and to look for insights into flatfish evolution.<a reveal-id="dsy015-B9" data-open="dsy015-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsy015-B38" data-open="dsy015-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a></p><p>In this study, we genotyped 18,214 SNPs using the 2b-RAD technology in 1,268 individuals from 31 turbot full-sibling (sib) families with the aim of: (i) obtaining a robust consensus genetic map for the species; (ii) integrating the genetic and physical maps in a common framework to facilitate transference of segregation data into the physical genome; (iii) refining the turbot genome assembly and comparative mapping with other flatfish; and (iv) assessing the variation in genome organisation and recombination between sexes and individuals. Our results rendered much deeper information on intra-specific mapping variation than previously reported in fish and provided useful integrated genomic resources to understand the architecture of economically important traits for further applications in industrial breeding programs.</p>                            <h2 scrollto-destination=120122076 id="120122076" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=120122077 id="120122077" class="section-title" >2.1. Families for mapping</h3>
<p>The broodstock of CETGA (Aquaculture Cluster of Galicia, Ribeira, Spain), a population representative of turbot from the Atlantic area, was used to found a set of 44 families within the goals of the FISHBOOST project (EU 613611). This project is focused on improving the efficiency and profitability of finfish European aquaculture by fostering selective breeding to a next level, including the application of genomic selection. This goal also involved the refinement and improvement of existing mapping resources and their integration in a common framework within the reported turbot genome assembly.<a reveal-id="dsy015-B34" data-open="dsy015-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> The experiment was performed under approval of the ethics committee of CETGA to adjust to animal welfare legislation.</p><p>A total of 31 full-sib families were selected among the 44 families considering both successful genotyping and informativeness (<a reveal-id="dsy015-F1" data-open="dsy015-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). The partly factorial mating of 22 males × 22 females performed made available 27, either maternal of paternal, half-sibs families (eight males and nine females contributed to more than one family). A total of 1,224 offspring were used for linkage analysis representing on average 39.5 individuals per family (range: 36–45). </p>                        <a id="120122080" scrollto-destination="120122080"></a>
<div id="dsy015-F1" data-id="dsy015-F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/m_dsy015f1.png?Expires=2147483647&Signature=JK22x9cmVbih-71eZeH6z6GQbocGiIk~MIVHnd9RuQIrGNajKBjktCXK9gvhMoOuJFiU84d3y2NFxkr7cbYE5wKLWBr8SAqcvwmBHf-lGB37ADCyQoBSNFhxtGaY0WsrbCEobSmvhyocYHGbtSWZtooRSjJRTqfoj4Z4sn3sX41quf7VXXtA4yfY4ENXx55IkA6-HZDEbcc1fjBd42BTK7pY4kPfhL2N5uRLs1-95VPoxxL2rwFR4DGCU9-BKGTcLXW6TWCqQlytbWNOyrxlVNorEiwLT~ayBlxV4zH7qSpXj9P8KVQx9KERay3icdpcVwplj~wyAeW2SrvywDBbDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Families selected for linkage analysis in S. maximus. Male (M) × female (F) crosses and the number of offspring per family are indicated. Note the presence of several half-sib families sharing the father or the mother." path-from-xml="dsy015f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122080/dsy015f1.tif" path-from-xml="dsy015f1.tif" target="_blank">View large</a><a section="120122080" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015f1.png?Expires=2147483647&Signature=tRdauuJsNu7hTxkNK~luvli83cDE3we2YPsl~QttaQrIsGC1vdMYuJfCn~0bhuwhMF8Z1CB3L2GcRG1qlX4nQ1sRbS1gnbZLNv7hym~EGVFs-BPJ6cUjCsshNDcscuP9S2XznOzff60cxgB3deEqwupfCieeC~R3vvMsWT85rjTGmX-K0hEjhVuxONmpd7Fcb70hgHc5CWvBHQbZESRUR3PzubbgoZBQRYPdQvmn9zPrj6gNDi12pWGddSr5TZhqq3FoH13rCWvTCxBzd0PbTB~M2z46MJQ~1OyCuPAYQcSyd3N9mXmMSXIBvno7Odp3xv7QwH5OH7bXtVKVASddeQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122080&ar=5035912&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy015f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Families selected for linkage analysis in <em>S. maximus</em>. Male (M) × female (F) crosses and the number of offspring per family are indicated. Note the presence of several half-sib families sharing the father or the mother.</p></div></div><div content-id="dsy015-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/m_dsy015f1.png?Expires=2147483647&Signature=JK22x9cmVbih-71eZeH6z6GQbocGiIk~MIVHnd9RuQIrGNajKBjktCXK9gvhMoOuJFiU84d3y2NFxkr7cbYE5wKLWBr8SAqcvwmBHf-lGB37ADCyQoBSNFhxtGaY0WsrbCEobSmvhyocYHGbtSWZtooRSjJRTqfoj4Z4sn3sX41quf7VXXtA4yfY4ENXx55IkA6-HZDEbcc1fjBd42BTK7pY4kPfhL2N5uRLs1-95VPoxxL2rwFR4DGCU9-BKGTcLXW6TWCqQlytbWNOyrxlVNorEiwLT~ayBlxV4zH7qSpXj9P8KVQx9KERay3icdpcVwplj~wyAeW2SrvywDBbDw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Families selected for linkage analysis in S. maximus. Male (M) × female (F) crosses and the number of offspring per family are indicated. Note the presence of several half-sib families sharing the father or the mother." path-from-xml="dsy015f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122080/dsy015f1.tif" path-from-xml="dsy015f1.tif" target="_blank">View large</a><a section="120122080" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015f1.png?Expires=2147483647&Signature=tRdauuJsNu7hTxkNK~luvli83cDE3we2YPsl~QttaQrIsGC1vdMYuJfCn~0bhuwhMF8Z1CB3L2GcRG1qlX4nQ1sRbS1gnbZLNv7hym~EGVFs-BPJ6cUjCsshNDcscuP9S2XznOzff60cxgB3deEqwupfCieeC~R3vvMsWT85rjTGmX-K0hEjhVuxONmpd7Fcb70hgHc5CWvBHQbZESRUR3PzubbgoZBQRYPdQvmn9zPrj6gNDi12pWGddSr5TZhqq3FoH13rCWvTCxBzd0PbTB~M2z46MJQ~1OyCuPAYQcSyd3N9mXmMSXIBvno7Odp3xv7QwH5OH7bXtVKVASddeQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122080&ar=5035912&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy015f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Families selected for linkage analysis in <em>S. maximus</em>. Male (M) × female (F) crosses and the number of offspring per family are indicated. Note the presence of several half-sib families sharing the father or the mother.</p></div></div>                            <h3 scrollto-destination=120122081 id="120122081" class="section-title" >2.2. SNP calling and genotyping</h3>
                            <h4 scrollto-destination=120122082 id="120122082" class="section-title" >2.2.1. DNA extraction and library preparation</h4>
<p>Genomic DNA was extracted from fin clips using SSTNE buffer (a TNE buffer modified by adding spermidine and spermine) and a standard NaCl isopropanol precipitation.<a reveal-id="dsy015-B44" data-open="dsy015-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> Library preparation followed the 2b-RAD protocol<a reveal-id="dsy015-B45" data-open="dsy015-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> slightly modified to include replicated polymerase chain reactions (PCRs) per sample (to reduce PCR duplicate bias) and to quantify DNA sampling before pooling (to increase sample coverage homogeneity). Detailed information about the protocol can be found in <span class="content-section supplementary-material"><a path-from-xml="sup10" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_maroso_dnares_supplementary_legends.doc?Expires=2147483647&amp;Signature=4hzbB9uSqpQDDD7vw8L0TRE5ThRl3nk0KTJ3oCaczcxIKHTj1juyPXhWQXDo1c5oPjGCvc1W4TR~UCqvy8dzh1qNRxn88hP1FtGm0ptR7P4AWp4AGrSstgtTaHixmmyUydvoSHw4n02V5wWpte7t2lVRjlXRLsWBmUnUUZu5zk0hA4A3Cw3-uFfx9NLiKP6tYDyvmZKLkdoyvhheKoe1c8XKgbmZN-gmMkvphRTKM93ODHpO7UhBZv4AgQ27VfOP0wC8Hm8ostjS4iqV2Eu70xToU5bu66wyH1LGPNpFm7lQsFkda027ezgH7HI0WjCOeYHxHgclv9VysWLwW9saIw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> text. Briefly, DNA of each sample was separately digested using <em>Alf</em>I restriction enzyme, adaptors including specific sample barcodes ligated, and the resulting fragments amplified. After PCR purification, samples were merged in pools of 128 samples for sequencing and genotyping the offspring, while the 44 parents were genotyped in a single pool, to increase coverage and genotyping accuracy. A total of 11 pools (1 for the parents and 10 for the offspring) were sequenced on an Illumina NextSeq sequencer using a 50 bp single end protocol at BMR Genomics (Padova, Italy).</p>                            <h4 scrollto-destination=120122084 id="120122084" class="section-title" >2.2.2. Raw data filtering and bioinformatics</h4>
<p>Raw data were demultiplexed at the sequencing facility. All reads were trimmed at 36 bp and centred on the enzyme recognition site. Stacks' <em>process_radtag</em> pipeline<a reveal-id="dsy015-B46" data-open="dsy015-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a><sup>,</sup><a reveal-id="dsy015-B47" data-open="dsy015-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> was used to filter raw reads according to sequencing quality. All reads with an uncalled base were removed, and quality was checked using a sliding window of 25% of the sequence length. If the mean quality score of this window dropped below 20, the sequence was removed.</p><p>Filtered reads were mapped against the turbot genome<a reveal-id="dsy015-B34" data-open="dsy015-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> using Bowtie 1.2.0<a reveal-id="dsy015-B48" data-open="dsy015-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> and allowing a maximum of three mismatches in the first 33 bases. Only a unique match on turbot genome was permitted, otherwise reads were discarded. Mapped reads from parent samples were fed into Stacks' module <em>ref_map.pl</em> using the bounded model with a significance threshold of 0.05 and an upper bound of 0.05. As a further filter, only SNPs genotyped in 80% of the parents with a minimum read depth of 10 reads were retained using the <em>populations</em> module. When a RAD-tag contained more than one SNP, only the first SNP from the 5' end was retained. The set of SNPs obtained from parents was used as a reference for offspring genotyping using the Stacks module <em>sstacks.</em> The obtained dataset (parents and offspring) was further filtered according to population parameters and Mendelian segregation using the 26 most informative families. Those markers showing consistent deviations from Mendelian segregation (<em>P</em> &lt; 0.01) across families or extreme departures from Hardy–Weinberg equilibrium (HWE) in the parental population (<em>P</em> &lt; 0.001) due to heterozygote excess (<em>F</em><sub>IS</sub> &lt; −0.5, <em>P</em> &lt; 0.0001; paralogous genes) or deficit (<em>F</em><sub>IS</sub> &gt; 0.5, <em>P</em> &lt; 0.0001; null alleles) were filtered out. Finally, RAD-tags containing SNPs were mapped against the turbot gene catalog<a reveal-id="dsy015-B38" data-open="dsy015-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> using Blastn (E-value &lt; e−20) to obtain sequence annotations.</p>                            <h3 scrollto-destination=120122087 id="120122087" class="section-title" >2.3. Construction of genetic maps</h3>
<p>Genotypes of filtered SNP markers were properly coded as JOINMAP Cross Pollinator (CP) cross type with unknown linkage phase. Markers genotyped in less than 90% of the offspring were excluded for linkage analysis in each family. All genetic maps (individual, family, male consensus, female consensus and species consensus) were constructed using JoinMap 4.1.<a reveal-id="dsy015-B49" data-open="dsy015-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> First, markers were associated to their linkage groups (LGs) using the <em>Grouping</em> function of JoinMap based on a series of LOD scores (logarithm of the likelihood ratio that two genes are linked regarding that they segregate independently) increasing by one, from 3 to 10. The LOD score was then selected in each family based on the number of turbot chromosomes (<em>n</em> = 22); in all cases, a LOD &gt; 6.0 was used. Unlinked markers and markers assigned to very small LGs exceeding the haploid number of 22 chromosomes were excluded from further analysis. Second, marker ordering was performed using the Maximum Likelihood algorithm with default parameters, and the recombination frequencies converted into map distances in centi-Morgans (cM) using the Kosambi mapping function. For each family, individual female and male maps, as well as a consensus family map, were generated. Due to computational limitations, only the three most informative families were selected to construct the species consensus, the female consensus and the male consensus maps using MergeMap.<a reveal-id="dsy015-B50" data-open="dsy015-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> All maps constructed were drawn using MapChart ver. 2.3.<a reveal-id="dsy015-B51" data-open="dsy015-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> In order to investigate the heterogeneity of recombination frequency (RF) between sexes, RFs were extracted from JoinMap analysis. The pattern of RF in the father and the mother could not be straightforwardly compared within each family due to the different panel of informative SNPs; consequently, the set of markers shared between both parents was selected in each family for a detailed evaluation of RF variation between sexes across LGs within each family (heterozygous in both parents with 1:2:1 segregation).</p>                            <h3 scrollto-destination=120122089 id="120122089" class="section-title" >2.4. Genetic versus physical map: anchoring scaffolds to linkage groups</h3>
<p>The consensus genetic map was used to refine the turbot genome assembly<a reveal-id="dsy015-B34" data-open="dsy015-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> and to obtain a physical map consistent on 22 megascaffolds, representing the complete sequence of each of the 22 LGs of turbot map.<a reveal-id="dsy015-B32" data-open="dsy015-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a><sup>,</sup><a reveal-id="dsy015-B39" data-open="dsy015-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> This task implied (i) to assign previously unlinked scaffolds to specific LGs in the new high-dense (HD) genetic map; (ii) to correct scaffold misassemblies by comparing the genetic and physical maps; and (iii) to establish the correct orientation of scaffolds within each megascaffold. All contigs including at least one marker significantly assigned to a LG were anchored in the new map. When markers of a specific scaffold were assigned to more than one LG, the scaffold was split into fragments pertaining to different LGs or to the same LG but in different positions (see Results). To establish the most confident splitting of these scaffolds, we compared the sequence of the gap between the flanking markers assigned to different LGs with the two model fish genomes, <em>Gasterosteus aculeatus</em> and <em>Oryzias latipes</em>, which had demonstrated large synteny with turbot.<a reveal-id="dsy015-B34" data-open="dsy015-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a><sup>,</sup><a reveal-id="dsy015-B38" data-open="dsy015-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> Then, we looked for the turbot genes in the gap assigned to different LGs in the model species in order to narrow down as much as possible the gap (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures1.pdf?Expires=2147483647&Signature=iRjYbKf0EsHectagtCG899YTSLSdL5P9bjXND9ypL1k5V6yJ9N7kaummLyzV8M9Aq~97B5XDuw7YQqAnSpYUQavEVl1qOsU6u8C6qNnsT2dgK1Diph~sWCQiex5kgT5xg3OakOagsJ38En7mNF5NSVezfdYwRnXdmMXhqXoeHpAI~aVtZ9Io8XCLXDkKcp6ngL0Ho8nHSnewzAz7BfLAYUAaXJrHfCwagjaNh4~5QhTV2JP4MIWNYMq4t8cxYezdidf2K0JxD~cRLan2DrLA5G2GDs1P4SeeqOrFjjLMecaT8r8pnM1MQk2TpmdSuiaIzhK9IymwzStqwCzsHinLOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). Following this approach, the narrowest gap was finally cut in the middle and halves assigned to each LG. Reorganisations were only validated when information was consistent with both model species. To find out the correct order and orientation of scaffolds within LGs, we obtained the average map position of the markers belonging to each scaffold using the maps of the 10 most informative families. Then, we checked for the correlation between the physical position of markers and their genetic position. When the correlation was positive, the original orientation was maintained, and when negative, the reverse complementary sequence of the scaffold was retrieved for assembling.</p>                            <h3 scrollto-destination=120122091 id="120122091" class="section-title" >2.5. Integrating genetic and physical maps with cytogenetic data</h3>
<p>Genetic and physical maps were then integrated with the turbot cytogenetic map using previous information.<a reveal-id="dsy015-B32" data-open="dsy015-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a><sup>,</sup><a reveal-id="dsy015-B40" data-open="dsy015-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a><sup>,</sup><a reveal-id="dsy015-B52" data-open="dsy015-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> Starting from centromere positions<a reveal-id="dsy015-B40" data-open="dsy015-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> and considering the relationship between genetic and cytogenetic maps,<a reveal-id="dsy015-B32" data-open="dsy015-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> we established the correspondence between LGs, megascaffolds and chromosomes as far as possible. Then, we compared the consensus genetic map of each chromosome with its physical assembly (megascaffold) to assess their correspondence with centromere, telomere and interstitial heterochromatin locations. When no marker was available close to the centromeric region, the closest markers around the centromere were used as reference or, in a few cases, the farthest marker from the centromere was used for its rough location at the other chromosome end, considering the acrocentric morphology of most turbot chromosomes.<a reveal-id="dsy015-B52" data-open="dsy015-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> Finally, this comparison was also carried out between sexes using the sex-specific consensus maps, in order to assess RF differences across chromosomes between males and females.</p>                            <h3 scrollto-destination=120122093 id="120122093" class="section-title" >2.6. Inter-individual genomic reorganisations in turbot</h3>
<p>The availability of 31 family maps and 44 parental maps (<a reveal-id="dsy015-F1" data-open="dsy015-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>), enabled an exhaustive comparison of genetic maps between individuals and sexes to assess inter-individual variation on RF across the genome. This variation, which might be related to chromosome polymorphisms existing in turbot populations, had not been previously addressed in fish with the number of markers and families here evaluated. In order to identify putative genomic reorganisation between individuals, we performed a pair-wise comparison between all parental maps by plotting the positions of each marker. Plots were created with R<a reveal-id="dsy015-B53" data-open="dsy015-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> and the different LGs were labelled with different colours for a better visualisation. In order to handle the amount of data obtained, 62 individual maps at 22 LGs (due to half-sib families; <a reveal-id="dsy015-F1" data-open="dsy015-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>), each LG was divided in stretches of 50 SNPs representing ∼1.6 Mb on average (e.g. 23 blocks for LG02, the longest LG with 1,132 SNPs). Then, the correlation between the physical and genetic maps was analysed for each block at each individual map in order to identify potential inversions. A heat map was generated to provide a more straightforward view of the results.</p>                            <h3 scrollto-destination=120122095 id="120122095" class="section-title" >2.7. Interspecific genomic reorganisations between turbot and other flatfish species</h3>
<p>The improved turbot genome was further exploited to complete and refine previous information about genomic reorganisations and chromosome evolution of flatfish.<a reveal-id="dsy015-B9" data-open="dsy015-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsy015-B34" data-open="dsy015-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> For this, we compared the improved turbot genome with those of Japanese flounder (<em>Paralichthys olivaceus</em><a reveal-id="dsy015-B43" data-open="dsy015-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a>) and half-smooth tongue sole (<em>Cynoglossus semilaevis</em><a reveal-id="dsy015-B42" data-open="dsy015-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a>) at chromosome level using LASTZ<a reveal-id="dsy015-B54" data-open="dsy015-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> (using options ‘–notransition – step = 400 – nogapped – format = rdotplot’) and compared each turbot chromosome with the orthologues of the other two species. Further, in order to assess the relation between flatfish megascaffolds to a model fish genome outside Pleuronectiformes, an additional LASTZ analysis was carried out with stickleback (<em>Gasterosteus aculeatus</em>) using the same parameters.</p>                            <h2 scrollto-destination=120122097 id="120122097" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=120122098 id="120122098" class="section-title" >3.1. SNP calling, filtering and genotyping</h3>
<p>A total of 3,946,354,960 reads (3.9 Gb) were produced at the BMR sequencing platform, corresponding to 1,282 individuals represented by 3,078,280 reads on average (range: 312 – 9,610,380). Fourteen offspring were removed due to the low number of usable reads (&lt; 75,000), leaving a total of 1,268 individuals (44 parents and 1,224 offspring) pertaining to 31 full-sib families for subsequent analyses.</p><p>After quality filtering and alignment to the turbot genome, the average number of reads per sample was 6,663,809 (range: 2,601,955–8,946,621) for parents and 2,774,579 (range: 283,203–5,596,610) for offspring. After Stacks' <em>populations</em> filter, 139,293 RAD-tags were retained, of which 25,511 (18.3%) contained at least one SNP. Since only one SNP was retained per RAD-tag, a total of 25,511 SNPs were considered after eliminating the remaining SNPs in the same RAD-tag (<span class="content-section supplementary-material"><a path-from-xml="sup11" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_tables1.xlsx?Expires=2147483647&amp;Signature=iu-s0ucyh61Jqm6Al68IyTnp0EwFC7zFFYSo0Z3cPTfltnRkboXvIDx076CZIb5BEt6jLoHAmQfgwmD~i9kS2Y-catWPu0lQ4QDwsKqAkbCWMHHdmDAC2~bW6Immj2-K3vlVHNbUu8ro6byRUgCLiCUlLpNGoPFmB6-VJqFQx-RcsXQLNQFTCBwIuTMihcwQtJEjlUB5CtJSNGma04LlRv4IHvvQtZxz2~hdm7o2VgohCrtQrsA1Ip~EuVwq56pYaqNwhw2WKH2-~XS4j9KlLjWPc1VJwobAhvRpgiQq2xKc16Y~XOVMteY2rJXBkfxKKn-i59DfiGAuXNvV8U~W0Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Among these 25,511 SNPs, 18,214 were finally retained after filtering loci by: (i) consistent deviations from Mendelian segregation (5,866 loci: <em>P</em> &lt; 0.01 in at least two families and representing more than 20% of the informative families); (ii) very low minimum allele frequency in the parental population (&lt; 0.015) suggestive of genotyping errors (991 SNPs); and (iii) extreme deviations from HWE suggesting either paralogous genes (364 loci) or high null allele frequency (91 loci) (<span class="content-section supplementary-material"><a path-from-xml="sup11" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_tables1.xlsx?Expires=2147483647&amp;Signature=iu-s0ucyh61Jqm6Al68IyTnp0EwFC7zFFYSo0Z3cPTfltnRkboXvIDx076CZIb5BEt6jLoHAmQfgwmD~i9kS2Y-catWPu0lQ4QDwsKqAkbCWMHHdmDAC2~bW6Immj2-K3vlVHNbUu8ro6byRUgCLiCUlLpNGoPFmB6-VJqFQx-RcsXQLNQFTCBwIuTMihcwQtJEjlUB5CtJSNGma04LlRv4IHvvQtZxz2~hdm7o2VgohCrtQrsA1Ip~EuVwq56pYaqNwhw2WKH2-~XS4j9KlLjWPc1VJwobAhvRpgiQq2xKc16Y~XOVMteY2rJXBkfxKKn-i59DfiGAuXNvV8U~W0Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). This set of 18,214 SNPs represented a frequency of one SNP every 163 bp in the reduced portion of the genome sequenced with 2b-RAD, a frequency slightly lower than that reported in other studies in turbot,<a reveal-id="dsy015-B55" data-open="dsy015-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a><sup>,</sup><a reveal-id="dsy015-B56" data-open="dsy015-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> and a density of one SNP every 31,130 bp in the whole turbot genome (567 Mb<a reveal-id="dsy015-B34" data-open="dsy015-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a>). The average proportion of genotyping errors (Mendelian inconsistencies in families) of our 2b-RAD genotyping approach was 0.68% throughout all families in the filtered SNP dataset.</p><p>Out of the 18,214 SNPs finally retained, nearly half matched to turbot genes (9,028). Among those lying within exons (834), 400 corresponded to non-synonymous and 430 to synonymous allelic variants (<span class="content-section supplementary-material"><a path-from-xml="sup11" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_tables1.xlsx?Expires=2147483647&amp;Signature=iu-s0ucyh61Jqm6Al68IyTnp0EwFC7zFFYSo0Z3cPTfltnRkboXvIDx076CZIb5BEt6jLoHAmQfgwmD~i9kS2Y-catWPu0lQ4QDwsKqAkbCWMHHdmDAC2~bW6Immj2-K3vlVHNbUu8ro6byRUgCLiCUlLpNGoPFmB6-VJqFQx-RcsXQLNQFTCBwIuTMihcwQtJEjlUB5CtJSNGma04LlRv4IHvvQtZxz2~hdm7o2VgohCrtQrsA1Ip~EuVwq56pYaqNwhw2WKH2-~XS4j9KlLjWPc1VJwobAhvRpgiQq2xKc16Y~XOVMteY2rJXBkfxKKn-i59DfiGAuXNvV8U~W0Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>).</p>                            <h3 scrollto-destination=120122102 id="120122102" class="section-title" >3.2. Construction of genetic maps</h3>
<p>On average, 5,385 SNPs were used for map construction in each of the 31 families analysed (range: 2,079 in Fam55 to 6,757 in Fam30; <span class="content-section supplementary-material"><a path-from-xml="sup12" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_tables2.xlsx?Expires=2147483647&amp;Signature=ih0SleqhQTp8cq1tZKwJOVoRcuIGZHurnaOzK1GA5pTH97D9Ym~SBbhLMS-OY68qtfA2PCaXKMK-Pyqd5qvqRqisq-pVVMyc0GzcRrT2GrZ067SkF2xyd1OV-hxLof-q2zFkbkYU5paBMjPiOA39LSKEZ4W4CGLwDzH5CfM8K0AxmpLV8JMk24LxnT7chbBXLGXl9eXxu2tmxnPyREtr3gpZX3OvFH47xSLafdvVrBlSJw~LZmkAC97D1d0SOjFV75O-DKGr~mgTgYfoygF1Tn-ZOP~ClcShwFcREzpiUFdSjFs9H7tIotRDJS7hbXtgbwfc7X7HjIQ0qcm0WWK8kw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Maps consisted of 22 LGs corresponding to the 22 chromosomes of the turbot haploid karyotype, averaging 140.5 cM and 245 markers per LG and family (<span class="content-section supplementary-material"><a path-from-xml="sup12" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_tables2.xlsx?Expires=2147483647&amp;Signature=ih0SleqhQTp8cq1tZKwJOVoRcuIGZHurnaOzK1GA5pTH97D9Ym~SBbhLMS-OY68qtfA2PCaXKMK-Pyqd5qvqRqisq-pVVMyc0GzcRrT2GrZ067SkF2xyd1OV-hxLof-q2zFkbkYU5paBMjPiOA39LSKEZ4W4CGLwDzH5CfM8K0AxmpLV8JMk24LxnT7chbBXLGXl9eXxu2tmxnPyREtr3gpZX3OvFH47xSLafdvVrBlSJw~LZmkAC97D1d0SOjFV75O-DKGr~mgTgYfoygF1Tn-ZOP~ClcShwFcREzpiUFdSjFs9H7tIotRDJS7hbXtgbwfc7X7HjIQ0qcm0WWK8kw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S2</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup13" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_tables3.xlsx?Expires=2147483647&amp;Signature=3iDrLGKcOPMCA9dyE1~NI5ar1ra7h6xEt7v1vZO8Zf1Y4mocW~QAc7-Yp-i71Wwlz0H1ON2cWX0ec4iuNnLIEyvZD9Xeu~Cc7kSIU9tXD5bSIsgCoQ9EskNAUmzC646kpN3ZxhTvVpbxOXhNTsd21N2swt5lAbJaGcAZRpgxVjChm9M8~1iq~Gs0G9sk-ePT0N3q1wgtSnR8vbVpCQneehCkbiRAnNcziMAyIccRLJk2L7IBrcO4aQXJrgJpVvaWTSOcOl0fw8ZSkjCy77cTpjLTL1XcldgT2WeV2LKo~DDHshvzasu5tLcV5FxIDO1iuIlCwXPrqz38FC6cztkS8g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S3</a></span>). Note that LGs nomenclature follow previous publications<a reveal-id="dsy015-B32" data-open="dsy015-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> and accordingly, LG18 is missing due to its merging with LG08 (LG08 + 18, here LG08). LG22 was on average the shortest LG (94.9 cM; range: 28.1 cM in Fam55 to 223.5 in Fam19) and included the lowest number of markers (189.9; range: 56 in Fam16 to 257 in Fam39). LG02 was on average the longest LG (207.7 cM; range: 49.9 cM in Fam59 to 414.0 cM in Fam19) with the highest number of markers (341.7; range: 97 in Fam55 to 457 in Fam39). A total of 17,169 SNPs were successfully mapped at least in one family, 1,045 remaining unmapped. Of the last, 1,029 were located in scaffolds where other markers were anchored to the linkage map and could therefore be assigned to specific LGs. Only 16 SNPs could not be assigned to any LG. Marker interval across all families averaged 0.6 cM and ranged from 0.43 cM in Fam11 to 1.08 cM in Fam19. If unique positions instead of markers were accounted for, marker interval averaged 2.0 cM and ranged from 1.81 in Fam13 cM to 2.74 cM in Fam55.</p><p>The turbot consensus map constructed with the three most informative families using MergeMap included 11,845 markers distributed over 22 LGs representing a total length of 8,532.6 cM (<span class="content-section supplementary-material"><a path-from-xml="sup13" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_tables3.xlsx?Expires=2147483647&amp;Signature=3iDrLGKcOPMCA9dyE1~NI5ar1ra7h6xEt7v1vZO8Zf1Y4mocW~QAc7-Yp-i71Wwlz0H1ON2cWX0ec4iuNnLIEyvZD9Xeu~Cc7kSIU9tXD5bSIsgCoQ9EskNAUmzC646kpN3ZxhTvVpbxOXhNTsd21N2swt5lAbJaGcAZRpgxVjChm9M8~1iq~Gs0G9sk-ePT0N3q1wgtSnR8vbVpCQneehCkbiRAnNcziMAyIccRLJk2L7IBrcO4aQXJrgJpVvaWTSOcOl0fw8ZSkjCy77cTpjLTL1XcldgT2WeV2LKo~DDHshvzasu5tLcV5FxIDO1iuIlCwXPrqz38FC6cztkS8g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3A</a></span>; <a reveal-id="dsy015-F2" data-open="dsy015-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>). LG length varied from 282.9 cM (LG22) to 588.7 cM (LG02), with an average length of 387.8 cM. The average marker interval was 0.72 cM and ranged from 0.57 cM in LG01 to 0.89 cM in LG4. Because of the inflated length observed for this consensus map, we also estimated the average LG lengths from the three families used to construct the consensus, as these values represent more realistic lengths than those rendered by the consensus MergeMap algorithm (see Discussion) (<a reveal-id="dsy015-T1" data-open="dsy015-T1" class="link link-reveal link-table xref-fig">Table 1</a>). These LG lengths sum up to 3,753.9 (i.e. an average marker interval of 0.32 cM). <div content-id="dsy015-T1" class="table-modal"><div class="table-wrap"><div id="dsy015-T1" data-id="dsy015-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Average length and constitution (SNP and scaffold number) of LGs in the species, female and male <em>S. maximus</em> consensus maps</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Karyotype </th><th> </th><th colspan="4">Physical map<hr /> </th><th colspan="3">Species consensus map<hr /> </th><th colspan="3">Female consensus map<hr /> </th><th colspan="3">Male consensus map<hr /> </th></tr><tr><th>Chromosome ID </th><th>LG </th><th>Megascaffold length (bp) </th><th>No SNPs </th><th>No scaffolds </th><th>Average contig length (bp) </th><th>Mapped markers </th><th>Average length </th><th>Mb/cM </th><th>Mapped markers </th><th>Average length </th><th>Mb/cM </th><th>Mapped markers </th><th>Average length </th><th>Mb/cM </th></tr></thead><tbody><tr><td>C7 </td><td>01 </td><td>26,874,605 </td><td>1,031 </td><td>32 </td><td>839,831.4 </td><td>671 </td><td>178.3 </td><td>0.151 </td><td>487 </td><td>176.6 </td><td>0.152 </td><td>409 </td><td>162.9 </td><td>0.165 </td></tr><tr><td>C1 </td><td>02 </td><td>31,897,078 </td><td>1,144 </td><td>46 </td><td>708,824.0 </td><td>761 </td><td>255.8 </td><td>0.125 </td><td>513 </td><td>261.6 </td><td>0.122 </td><td>546 </td><td>215.0 </td><td>0.148 </td></tr><tr><td>C17 </td><td>03 </td><td>21,400,935 </td><td>785 </td><td>23 </td><td>930,475.4 </td><td>506 </td><td>180.7 </td><td>0.118 </td><td>317 </td><td>179.1 </td><td>0.119 </td><td>395 </td><td>156.3 </td><td>0.137 </td></tr><tr><td>C5 </td><td>04 </td><td>29,497,696 </td><td>907 </td><td>31 </td><td>951,538.6 </td><td>610 </td><td>220.8 </td><td>0.134 </td><td>412 </td><td>251.0 </td><td>0.118 </td><td>439 </td><td>171.7 </td><td>0.172 </td></tr><tr><td>C12 </td><td>05 </td><td>24,809,483 </td><td>707 </td><td>20 </td><td>1,240,474.2 </td><td>443 </td><td>125.0 </td><td>0.198 </td><td>300 </td><td>129.1 </td><td>0.192 </td><td>275 </td><td>107.2 </td><td>0.231 </td></tr><tr><td>C10 </td><td>06 </td><td>25,205,370 </td><td>851 </td><td>26 </td><td>969,437.3 </td><td>587 </td><td>186.1 </td><td>0.135 </td><td>395 </td><td>205.3 </td><td>0.123 </td><td>455 </td><td>153.3 </td><td>0.164 </td></tr><tr><td>C13 </td><td>07 </td><td>24,510,356 </td><td>699 </td><td>20 </td><td>1,225,517.7 </td><td>412 </td><td>168.3 </td><td>0.146 </td><td>317 </td><td>179.4 </td><td>0.137 </td><td>301 </td><td>143.3 </td><td>0.171 </td></tr><tr><td>C4 </td><td>08 </td><td>30,930,439 </td><td>1,066 </td><td>38 </td><td>813,958.9 </td><td>640 </td><td>266.3 </td><td>0.116 </td><td>434 </td><td>303.3 </td><td>0.102 </td><td>427 </td><td>196.0 </td><td>0.158 </td></tr><tr><td>C8 </td><td>09 </td><td>25,788,341 </td><td>929 </td><td>20 </td><td>1,289,417.1 </td><td>593 </td><td>200.0 </td><td>0.129 </td><td>415 </td><td>209.6 </td><td>0.123 </td><td>406 </td><td>165.1 </td><td>0.156 </td></tr><tr><td>C11 </td><td>10 </td><td>25,098,921 </td><td>922 </td><td>25 </td><td>1,003,956.8 </td><td>650 </td><td>169.1 </td><td>0.148 </td><td>419 </td><td>166.8 </td><td>0.150 </td><td>497 </td><td>156.5 </td><td>0.160 </td></tr><tr><td>C6 </td><td>11 </td><td>27,408,095 </td><td>808 </td><td>33 </td><td>830,548.3 </td><td>494 </td><td>150.9 </td><td>0.182 </td><td>406 </td><td>174.1 </td><td>0.157 </td><td>323 </td><td>108.9 </td><td>0.252 </td></tr><tr><td>C9 </td><td>12 </td><td>25,240,469 </td><td>796 </td><td>21 </td><td>1,201,927.1 </td><td>536 </td><td>194.9 </td><td>0.130 </td><td>395 </td><td>204.2 </td><td>0.124 </td><td>376 </td><td>162.2 </td><td>0.156 </td></tr><tr><td>C19 </td><td>13 </td><td>20,256,549 </td><td>726 </td><td>20 </td><td>1,012,827.5 </td><td>472 </td><td>154.0 </td><td>0.132 </td><td>349 </td><td>176.3 </td><td>0.115 </td><td>346 </td><td>119.2 </td><td>0.170 </td></tr><tr><td>C16 </td><td>14 </td><td>21,450,603 </td><td>778 </td><td>20 </td><td>1,072,530.2 </td><td>514 </td><td>145.1 </td><td>0.148 </td><td>374 </td><td>158.2 </td><td>0.136 </td><td>402 </td><td>114.3 </td><td>0.188 </td></tr><tr><td>C3 </td><td>15 </td><td>24,127,324 </td><td>887 </td><td>33 </td><td>731,131.0 </td><td>593 </td><td>146.4 </td><td>0.165 </td><td>413 </td><td>156.7 </td><td>0.154 </td><td>383 </td><td>113.0 </td><td>0.214 </td></tr><tr><td>C2 </td><td>16 </td><td>23,833,596 </td><td>813 </td><td>32 </td><td>768,825.7 </td><td>531 </td><td>197.8 </td><td>0.120 </td><td>375 </td><td>201.1 </td><td>0.119 </td><td>371 </td><td>171.1 </td><td>0.139 </td></tr><tr><td>C21 </td><td>17 </td><td>15,881,475 </td><td>680 </td><td>16 </td><td>992,591.8 </td><td>467 </td><td>127.6 </td><td>0.124 </td><td>335 </td><td>141.3 </td><td>0.112 </td><td>332 </td><td>103.9 </td><td>0.153 </td></tr><tr><td>C15 </td><td>19 </td><td>21,781,057 </td><td>755 </td><td>22 </td><td>990,048.0 </td><td>492 </td><td>128.7 </td><td>0.169 </td><td>372 </td><td>123.0 </td><td>0.177 </td><td>345 </td><td>118.0 </td><td>0.185 </td></tr><tr><td>C14 </td><td>20 </td><td>22,751,219 </td><td>776 </td><td>23 </td><td>989,183.4 </td><td>456 </td><td>145.5 </td><td>0.156 </td><td>304 </td><td>163.1 </td><td>0.139 </td><td>322 </td><td>116.4 </td><td>0.195 </td></tr><tr><td>C18 </td><td>21 </td><td>21,354,662 </td><td>760 </td><td>21 </td><td>1,016,888.7 </td><td>501 </td><td>136.2 </td><td>0.157 </td><td>356 </td><td>138.6 </td><td>0.154 </td><td>386 </td><td>123.8 </td><td>0.172 </td></tr><tr><td>C22 </td><td>22 </td><td>14,917,408 </td><td>578 </td><td>14 </td><td>1,065,529.1 </td><td>408 </td><td>121.0 </td><td>0.123 </td><td>291 </td><td>128.6 </td><td>0.116 </td><td>259 </td><td>102.4 </td><td>0.146 </td></tr><tr><td>C20 </td><td>23 </td><td>19,910,129 </td><td>796 </td><td>22 </td><td>905,005.9 </td><td>508 </td><td>155.4 </td><td>0.128 </td><td>378 </td><td>151.4 </td><td>0.132 </td><td>378 </td><td>129.4 </td><td>0.154 </td></tr><tr><td>Sub-total </td><td> </td><td>524,925,810 </td><td>18,194 </td><td>558 </td><td> </td><td>11,845 </td><td>3,753.9 </td><td>0.140 </td><td>8,357 </td><td>3,978.4 </td><td>0.132 </td><td>8,373 </td><td>3,109.9 </td><td>0.169 </td></tr><tr><td>Not anchored </td><td> </td><td>19,312,992 </td><td>16 </td><td>15,936 </td><td>1,211.9 </td><td>6,369 </td><td> </td><td> </td><td>9,857 </td><td> </td><td> </td><td>9,841 </td><td> </td><td> </td></tr><tr><td>Total </td><td>  </td><td>544,238,802 </td><td>18,214 </td><td>16,494 </td><td>  </td><td>18,214 </td><td> </td><td> </td><td>18,214 </td><td>  </td><td>  </td><td>18,214 </td><td>  </td><td>  </td></tr></tbody></table></div><div content-id="dsy015-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Karyotype </th><th> </th><th colspan="4">Physical map<hr /> </th><th colspan="3">Species consensus map<hr /> </th><th colspan="3">Female consensus map<hr /> </th><th colspan="3">Male consensus map<hr /> </th></tr><tr><th>Chromosome ID </th><th>LG </th><th>Megascaffold length (bp) </th><th>No SNPs </th><th>No scaffolds </th><th>Average contig length (bp) </th><th>Mapped markers </th><th>Average length </th><th>Mb/cM </th><th>Mapped markers </th><th>Average length </th><th>Mb/cM </th><th>Mapped markers </th><th>Average length </th><th>Mb/cM </th></tr></thead><tbody><tr><td>C7 </td><td>01 </td><td>26,874,605 </td><td>1,031 </td><td>32 </td><td>839,831.4 </td><td>671 </td><td>178.3 </td><td>0.151 </td><td>487 </td><td>176.6 </td><td>0.152 </td><td>409 </td><td>162.9 </td><td>0.165 </td></tr><tr><td>C1 </td><td>02 </td><td>31,897,078 </td><td>1,144 </td><td>46 </td><td>708,824.0 </td><td>761 </td><td>255.8 </td><td>0.125 </td><td>513 </td><td>261.6 </td><td>0.122 </td><td>546 </td><td>215.0 </td><td>0.148 </td></tr><tr><td>C17 </td><td>03 </td><td>21,400,935 </td><td>785 </td><td>23 </td><td>930,475.4 </td><td>506 </td><td>180.7 </td><td>0.118 </td><td>317 </td><td>179.1 </td><td>0.119 </td><td>395 </td><td>156.3 </td><td>0.137 </td></tr><tr><td>C5 </td><td>04 </td><td>29,497,696 </td><td>907 </td><td>31 </td><td>951,538.6 </td><td>610 </td><td>220.8 </td><td>0.134 </td><td>412 </td><td>251.0 </td><td>0.118 </td><td>439 </td><td>171.7 </td><td>0.172 </td></tr><tr><td>C12 </td><td>05 </td><td>24,809,483 </td><td>707 </td><td>20 </td><td>1,240,474.2 </td><td>443 </td><td>125.0 </td><td>0.198 </td><td>300 </td><td>129.1 </td><td>0.192 </td><td>275 </td><td>107.2 </td><td>0.231 </td></tr><tr><td>C10 </td><td>06 </td><td>25,205,370 </td><td>851 </td><td>26 </td><td>969,437.3 </td><td>587 </td><td>186.1 </td><td>0.135 </td><td>395 </td><td>205.3 </td><td>0.123 </td><td>455 </td><td>153.3 </td><td>0.164 </td></tr><tr><td>C13 </td><td>07 </td><td>24,510,356 </td><td>699 </td><td>20 </td><td>1,225,517.7 </td><td>412 </td><td>168.3 </td><td>0.146 </td><td>317 </td><td>179.4 </td><td>0.137 </td><td>301 </td><td>143.3 </td><td>0.171 </td></tr><tr><td>C4 </td><td>08 </td><td>30,930,439 </td><td>1,066 </td><td>38 </td><td>813,958.9 </td><td>640 </td><td>266.3 </td><td>0.116 </td><td>434 </td><td>303.3 </td><td>0.102 </td><td>427 </td><td>196.0 </td><td>0.158 </td></tr><tr><td>C8 </td><td>09 </td><td>25,788,341 </td><td>929 </td><td>20 </td><td>1,289,417.1 </td><td>593 </td><td>200.0 </td><td>0.129 </td><td>415 </td><td>209.6 </td><td>0.123 </td><td>406 </td><td>165.1 </td><td>0.156 </td></tr><tr><td>C11 </td><td>10 </td><td>25,098,921 </td><td>922 </td><td>25 </td><td>1,003,956.8 </td><td>650 </td><td>169.1 </td><td>0.148 </td><td>419 </td><td>166.8 </td><td>0.150 </td><td>497 </td><td>156.5 </td><td>0.160 </td></tr><tr><td>C6 </td><td>11 </td><td>27,408,095 </td><td>808 </td><td>33 </td><td>830,548.3 </td><td>494 </td><td>150.9 </td><td>0.182 </td><td>406 </td><td>174.1 </td><td>0.157 </td><td>323 </td><td>108.9 </td><td>0.252 </td></tr><tr><td>C9 </td><td>12 </td><td>25,240,469 </td><td>796 </td><td>21 </td><td>1,201,927.1 </td><td>536 </td><td>194.9 </td><td>0.130 </td><td>395 </td><td>204.2 </td><td>0.124 </td><td>376 </td><td>162.2 </td><td>0.156 </td></tr><tr><td>C19 </td><td>13 </td><td>20,256,549 </td><td>726 </td><td>20 </td><td>1,012,827.5 </td><td>472 </td><td>154.0 </td><td>0.132 </td><td>349 </td><td>176.3 </td><td>0.115 </td><td>346 </td><td>119.2 </td><td>0.170 </td></tr><tr><td>C16 </td><td>14 </td><td>21,450,603 </td><td>778 </td><td>20 </td><td>1,072,530.2 </td><td>514 </td><td>145.1 </td><td>0.148 </td><td>374 </td><td>158.2 </td><td>0.136 </td><td>402 </td><td>114.3 </td><td>0.188 </td></tr><tr><td>C3 </td><td>15 </td><td>24,127,324 </td><td>887 </td><td>33 </td><td>731,131.0 </td><td>593 </td><td>146.4 </td><td>0.165 </td><td>413 </td><td>156.7 </td><td>0.154 </td><td>383 </td><td>113.0 </td><td>0.214 </td></tr><tr><td>C2 </td><td>16 </td><td>23,833,596 </td><td>813 </td><td>32 </td><td>768,825.7 </td><td>531 </td><td>197.8 </td><td>0.120 </td><td>375 </td><td>201.1 </td><td>0.119 </td><td>371 </td><td>171.1 </td><td>0.139 </td></tr><tr><td>C21 </td><td>17 </td><td>15,881,475 </td><td>680 </td><td>16 </td><td>992,591.8 </td><td>467 </td><td>127.6 </td><td>0.124 </td><td>335 </td><td>141.3 </td><td>0.112 </td><td>332 </td><td>103.9 </td><td>0.153 </td></tr><tr><td>C15 </td><td>19 </td><td>21,781,057 </td><td>755 </td><td>22 </td><td>990,048.0 </td><td>492 </td><td>128.7 </td><td>0.169 </td><td>372 </td><td>123.0 </td><td>0.177 </td><td>345 </td><td>118.0 </td><td>0.185 </td></tr><tr><td>C14 </td><td>20 </td><td>22,751,219 </td><td>776 </td><td>23 </td><td>989,183.4 </td><td>456 </td><td>145.5 </td><td>0.156 </td><td>304 </td><td>163.1 </td><td>0.139 </td><td>322 </td><td>116.4 </td><td>0.195 </td></tr><tr><td>C18 </td><td>21 </td><td>21,354,662 </td><td>760 </td><td>21 </td><td>1,016,888.7 </td><td>501 </td><td>136.2 </td><td>0.157 </td><td>356 </td><td>138.6 </td><td>0.154 </td><td>386 </td><td>123.8 </td><td>0.172 </td></tr><tr><td>C22 </td><td>22 </td><td>14,917,408 </td><td>578 </td><td>14 </td><td>1,065,529.1 </td><td>408 </td><td>121.0 </td><td>0.123 </td><td>291 </td><td>128.6 </td><td>0.116 </td><td>259 </td><td>102.4 </td><td>0.146 </td></tr><tr><td>C20 </td><td>23 </td><td>19,910,129 </td><td>796 </td><td>22 </td><td>905,005.9 </td><td>508 </td><td>155.4 </td><td>0.128 </td><td>378 </td><td>151.4 </td><td>0.132 </td><td>378 </td><td>129.4 </td><td>0.154 </td></tr><tr><td>Sub-total </td><td> </td><td>524,925,810 </td><td>18,194 </td><td>558 </td><td> </td><td>11,845 </td><td>3,753.9 </td><td>0.140 </td><td>8,357 </td><td>3,978.4 </td><td>0.132 </td><td>8,373 </td><td>3,109.9 </td><td>0.169 </td></tr><tr><td>Not anchored </td><td> </td><td>19,312,992 </td><td>16 </td><td>15,936 </td><td>1,211.9 </td><td>6,369 </td><td> </td><td> </td><td>9,857 </td><td> </td><td> </td><td>9,841 </td><td> </td><td> </td></tr><tr><td>Total </td><td>  </td><td>544,238,802 </td><td>18,214 </td><td>16,494 </td><td>  </td><td>18,214 </td><td> </td><td> </td><td>18,214 </td><td>  </td><td>  </td><td>18,214 </td><td>  </td><td>  </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsy015-T1" data-id="dsy015-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Average length and constitution (SNP and scaffold number) of LGs in the species, female and male <em>S. maximus</em> consensus maps</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Karyotype </th><th> </th><th colspan="4">Physical map<hr /> </th><th colspan="3">Species consensus map<hr /> </th><th colspan="3">Female consensus map<hr /> </th><th colspan="3">Male consensus map<hr /> </th></tr><tr><th>Chromosome ID </th><th>LG </th><th>Megascaffold length (bp) </th><th>No SNPs </th><th>No scaffolds </th><th>Average contig length (bp) </th><th>Mapped markers </th><th>Average length </th><th>Mb/cM </th><th>Mapped markers </th><th>Average length </th><th>Mb/cM </th><th>Mapped markers </th><th>Average length </th><th>Mb/cM </th></tr></thead><tbody><tr><td>C7 </td><td>01 </td><td>26,874,605 </td><td>1,031 </td><td>32 </td><td>839,831.4 </td><td>671 </td><td>178.3 </td><td>0.151 </td><td>487 </td><td>176.6 </td><td>0.152 </td><td>409 </td><td>162.9 </td><td>0.165 </td></tr><tr><td>C1 </td><td>02 </td><td>31,897,078 </td><td>1,144 </td><td>46 </td><td>708,824.0 </td><td>761 </td><td>255.8 </td><td>0.125 </td><td>513 </td><td>261.6 </td><td>0.122 </td><td>546 </td><td>215.0 </td><td>0.148 </td></tr><tr><td>C17 </td><td>03 </td><td>21,400,935 </td><td>785 </td><td>23 </td><td>930,475.4 </td><td>506 </td><td>180.7 </td><td>0.118 </td><td>317 </td><td>179.1 </td><td>0.119 </td><td>395 </td><td>156.3 </td><td>0.137 </td></tr><tr><td>C5 </td><td>04 </td><td>29,497,696 </td><td>907 </td><td>31 </td><td>951,538.6 </td><td>610 </td><td>220.8 </td><td>0.134 </td><td>412 </td><td>251.0 </td><td>0.118 </td><td>439 </td><td>171.7 </td><td>0.172 </td></tr><tr><td>C12 </td><td>05 </td><td>24,809,483 </td><td>707 </td><td>20 </td><td>1,240,474.2 </td><td>443 </td><td>125.0 </td><td>0.198 </td><td>300 </td><td>129.1 </td><td>0.192 </td><td>275 </td><td>107.2 </td><td>0.231 </td></tr><tr><td>C10 </td><td>06 </td><td>25,205,370 </td><td>851 </td><td>26 </td><td>969,437.3 </td><td>587 </td><td>186.1 </td><td>0.135 </td><td>395 </td><td>205.3 </td><td>0.123 </td><td>455 </td><td>153.3 </td><td>0.164 </td></tr><tr><td>C13 </td><td>07 </td><td>24,510,356 </td><td>699 </td><td>20 </td><td>1,225,517.7 </td><td>412 </td><td>168.3 </td><td>0.146 </td><td>317 </td><td>179.4 </td><td>0.137 </td><td>301 </td><td>143.3 </td><td>0.171 </td></tr><tr><td>C4 </td><td>08 </td><td>30,930,439 </td><td>1,066 </td><td>38 </td><td>813,958.9 </td><td>640 </td><td>266.3 </td><td>0.116 </td><td>434 </td><td>303.3 </td><td>0.102 </td><td>427 </td><td>196.0 </td><td>0.158 </td></tr><tr><td>C8 </td><td>09 </td><td>25,788,341 </td><td>929 </td><td>20 </td><td>1,289,417.1 </td><td>593 </td><td>200.0 </td><td>0.129 </td><td>415 </td><td>209.6 </td><td>0.123 </td><td>406 </td><td>165.1 </td><td>0.156 </td></tr><tr><td>C11 </td><td>10 </td><td>25,098,921 </td><td>922 </td><td>25 </td><td>1,003,956.8 </td><td>650 </td><td>169.1 </td><td>0.148 </td><td>419 </td><td>166.8 </td><td>0.150 </td><td>497 </td><td>156.5 </td><td>0.160 </td></tr><tr><td>C6 </td><td>11 </td><td>27,408,095 </td><td>808 </td><td>33 </td><td>830,548.3 </td><td>494 </td><td>150.9 </td><td>0.182 </td><td>406 </td><td>174.1 </td><td>0.157 </td><td>323 </td><td>108.9 </td><td>0.252 </td></tr><tr><td>C9 </td><td>12 </td><td>25,240,469 </td><td>796 </td><td>21 </td><td>1,201,927.1 </td><td>536 </td><td>194.9 </td><td>0.130 </td><td>395 </td><td>204.2 </td><td>0.124 </td><td>376 </td><td>162.2 </td><td>0.156 </td></tr><tr><td>C19 </td><td>13 </td><td>20,256,549 </td><td>726 </td><td>20 </td><td>1,012,827.5 </td><td>472 </td><td>154.0 </td><td>0.132 </td><td>349 </td><td>176.3 </td><td>0.115 </td><td>346 </td><td>119.2 </td><td>0.170 </td></tr><tr><td>C16 </td><td>14 </td><td>21,450,603 </td><td>778 </td><td>20 </td><td>1,072,530.2 </td><td>514 </td><td>145.1 </td><td>0.148 </td><td>374 </td><td>158.2 </td><td>0.136 </td><td>402 </td><td>114.3 </td><td>0.188 </td></tr><tr><td>C3 </td><td>15 </td><td>24,127,324 </td><td>887 </td><td>33 </td><td>731,131.0 </td><td>593 </td><td>146.4 </td><td>0.165 </td><td>413 </td><td>156.7 </td><td>0.154 </td><td>383 </td><td>113.0 </td><td>0.214 </td></tr><tr><td>C2 </td><td>16 </td><td>23,833,596 </td><td>813 </td><td>32 </td><td>768,825.7 </td><td>531 </td><td>197.8 </td><td>0.120 </td><td>375 </td><td>201.1 </td><td>0.119 </td><td>371 </td><td>171.1 </td><td>0.139 </td></tr><tr><td>C21 </td><td>17 </td><td>15,881,475 </td><td>680 </td><td>16 </td><td>992,591.8 </td><td>467 </td><td>127.6 </td><td>0.124 </td><td>335 </td><td>141.3 </td><td>0.112 </td><td>332 </td><td>103.9 </td><td>0.153 </td></tr><tr><td>C15 </td><td>19 </td><td>21,781,057 </td><td>755 </td><td>22 </td><td>990,048.0 </td><td>492 </td><td>128.7 </td><td>0.169 </td><td>372 </td><td>123.0 </td><td>0.177 </td><td>345 </td><td>118.0 </td><td>0.185 </td></tr><tr><td>C14 </td><td>20 </td><td>22,751,219 </td><td>776 </td><td>23 </td><td>989,183.4 </td><td>456 </td><td>145.5 </td><td>0.156 </td><td>304 </td><td>163.1 </td><td>0.139 </td><td>322 </td><td>116.4 </td><td>0.195 </td></tr><tr><td>C18 </td><td>21 </td><td>21,354,662 </td><td>760 </td><td>21 </td><td>1,016,888.7 </td><td>501 </td><td>136.2 </td><td>0.157 </td><td>356 </td><td>138.6 </td><td>0.154 </td><td>386 </td><td>123.8 </td><td>0.172 </td></tr><tr><td>C22 </td><td>22 </td><td>14,917,408 </td><td>578 </td><td>14 </td><td>1,065,529.1 </td><td>408 </td><td>121.0 </td><td>0.123 </td><td>291 </td><td>128.6 </td><td>0.116 </td><td>259 </td><td>102.4 </td><td>0.146 </td></tr><tr><td>C20 </td><td>23 </td><td>19,910,129 </td><td>796 </td><td>22 </td><td>905,005.9 </td><td>508 </td><td>155.4 </td><td>0.128 </td><td>378 </td><td>151.4 </td><td>0.132 </td><td>378 </td><td>129.4 </td><td>0.154 </td></tr><tr><td>Sub-total </td><td> </td><td>524,925,810 </td><td>18,194 </td><td>558 </td><td> </td><td>11,845 </td><td>3,753.9 </td><td>0.140 </td><td>8,357 </td><td>3,978.4 </td><td>0.132 </td><td>8,373 </td><td>3,109.9 </td><td>0.169 </td></tr><tr><td>Not anchored </td><td> </td><td>19,312,992 </td><td>16 </td><td>15,936 </td><td>1,211.9 </td><td>6,369 </td><td> </td><td> </td><td>9,857 </td><td> </td><td> </td><td>9,841 </td><td> </td><td> </td></tr><tr><td>Total </td><td>  </td><td>544,238,802 </td><td>18,214 </td><td>16,494 </td><td>  </td><td>18,214 </td><td> </td><td> </td><td>18,214 </td><td>  </td><td>  </td><td>18,214 </td><td>  </td><td>  </td></tr></tbody></table></div><div content-id="dsy015-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Karyotype </th><th> </th><th colspan="4">Physical map<hr /> </th><th colspan="3">Species consensus map<hr /> </th><th colspan="3">Female consensus map<hr /> </th><th colspan="3">Male consensus map<hr /> </th></tr><tr><th>Chromosome ID </th><th>LG </th><th>Megascaffold length (bp) </th><th>No SNPs </th><th>No scaffolds </th><th>Average contig length (bp) </th><th>Mapped markers </th><th>Average length </th><th>Mb/cM </th><th>Mapped markers </th><th>Average length </th><th>Mb/cM </th><th>Mapped markers </th><th>Average length </th><th>Mb/cM </th></tr></thead><tbody><tr><td>C7 </td><td>01 </td><td>26,874,605 </td><td>1,031 </td><td>32 </td><td>839,831.4 </td><td>671 </td><td>178.3 </td><td>0.151 </td><td>487 </td><td>176.6 </td><td>0.152 </td><td>409 </td><td>162.9 </td><td>0.165 </td></tr><tr><td>C1 </td><td>02 </td><td>31,897,078 </td><td>1,144 </td><td>46 </td><td>708,824.0 </td><td>761 </td><td>255.8 </td><td>0.125 </td><td>513 </td><td>261.6 </td><td>0.122 </td><td>546 </td><td>215.0 </td><td>0.148 </td></tr><tr><td>C17 </td><td>03 </td><td>21,400,935 </td><td>785 </td><td>23 </td><td>930,475.4 </td><td>506 </td><td>180.7 </td><td>0.118 </td><td>317 </td><td>179.1 </td><td>0.119 </td><td>395 </td><td>156.3 </td><td>0.137 </td></tr><tr><td>C5 </td><td>04 </td><td>29,497,696 </td><td>907 </td><td>31 </td><td>951,538.6 </td><td>610 </td><td>220.8 </td><td>0.134 </td><td>412 </td><td>251.0 </td><td>0.118 </td><td>439 </td><td>171.7 </td><td>0.172 </td></tr><tr><td>C12 </td><td>05 </td><td>24,809,483 </td><td>707 </td><td>20 </td><td>1,240,474.2 </td><td>443 </td><td>125.0 </td><td>0.198 </td><td>300 </td><td>129.1 </td><td>0.192 </td><td>275 </td><td>107.2 </td><td>0.231 </td></tr><tr><td>C10 </td><td>06 </td><td>25,205,370 </td><td>851 </td><td>26 </td><td>969,437.3 </td><td>587 </td><td>186.1 </td><td>0.135 </td><td>395 </td><td>205.3 </td><td>0.123 </td><td>455 </td><td>153.3 </td><td>0.164 </td></tr><tr><td>C13 </td><td>07 </td><td>24,510,356 </td><td>699 </td><td>20 </td><td>1,225,517.7 </td><td>412 </td><td>168.3 </td><td>0.146 </td><td>317 </td><td>179.4 </td><td>0.137 </td><td>301 </td><td>143.3 </td><td>0.171 </td></tr><tr><td>C4 </td><td>08 </td><td>30,930,439 </td><td>1,066 </td><td>38 </td><td>813,958.9 </td><td>640 </td><td>266.3 </td><td>0.116 </td><td>434 </td><td>303.3 </td><td>0.102 </td><td>427 </td><td>196.0 </td><td>0.158 </td></tr><tr><td>C8 </td><td>09 </td><td>25,788,341 </td><td>929 </td><td>20 </td><td>1,289,417.1 </td><td>593 </td><td>200.0 </td><td>0.129 </td><td>415 </td><td>209.6 </td><td>0.123 </td><td>406 </td><td>165.1 </td><td>0.156 </td></tr><tr><td>C11 </td><td>10 </td><td>25,098,921 </td><td>922 </td><td>25 </td><td>1,003,956.8 </td><td>650 </td><td>169.1 </td><td>0.148 </td><td>419 </td><td>166.8 </td><td>0.150 </td><td>497 </td><td>156.5 </td><td>0.160 </td></tr><tr><td>C6 </td><td>11 </td><td>27,408,095 </td><td>808 </td><td>33 </td><td>830,548.3 </td><td>494 </td><td>150.9 </td><td>0.182 </td><td>406 </td><td>174.1 </td><td>0.157 </td><td>323 </td><td>108.9 </td><td>0.252 </td></tr><tr><td>C9 </td><td>12 </td><td>25,240,469 </td><td>796 </td><td>21 </td><td>1,201,927.1 </td><td>536 </td><td>194.9 </td><td>0.130 </td><td>395 </td><td>204.2 </td><td>0.124 </td><td>376 </td><td>162.2 </td><td>0.156 </td></tr><tr><td>C19 </td><td>13 </td><td>20,256,549 </td><td>726 </td><td>20 </td><td>1,012,827.5 </td><td>472 </td><td>154.0 </td><td>0.132 </td><td>349 </td><td>176.3 </td><td>0.115 </td><td>346 </td><td>119.2 </td><td>0.170 </td></tr><tr><td>C16 </td><td>14 </td><td>21,450,603 </td><td>778 </td><td>20 </td><td>1,072,530.2 </td><td>514 </td><td>145.1 </td><td>0.148 </td><td>374 </td><td>158.2 </td><td>0.136 </td><td>402 </td><td>114.3 </td><td>0.188 </td></tr><tr><td>C3 </td><td>15 </td><td>24,127,324 </td><td>887 </td><td>33 </td><td>731,131.0 </td><td>593 </td><td>146.4 </td><td>0.165 </td><td>413 </td><td>156.7 </td><td>0.154 </td><td>383 </td><td>113.0 </td><td>0.214 </td></tr><tr><td>C2 </td><td>16 </td><td>23,833,596 </td><td>813 </td><td>32 </td><td>768,825.7 </td><td>531 </td><td>197.8 </td><td>0.120 </td><td>375 </td><td>201.1 </td><td>0.119 </td><td>371 </td><td>171.1 </td><td>0.139 </td></tr><tr><td>C21 </td><td>17 </td><td>15,881,475 </td><td>680 </td><td>16 </td><td>992,591.8 </td><td>467 </td><td>127.6 </td><td>0.124 </td><td>335 </td><td>141.3 </td><td>0.112 </td><td>332 </td><td>103.9 </td><td>0.153 </td></tr><tr><td>C15 </td><td>19 </td><td>21,781,057 </td><td>755 </td><td>22 </td><td>990,048.0 </td><td>492 </td><td>128.7 </td><td>0.169 </td><td>372 </td><td>123.0 </td><td>0.177 </td><td>345 </td><td>118.0 </td><td>0.185 </td></tr><tr><td>C14 </td><td>20 </td><td>22,751,219 </td><td>776 </td><td>23 </td><td>989,183.4 </td><td>456 </td><td>145.5 </td><td>0.156 </td><td>304 </td><td>163.1 </td><td>0.139 </td><td>322 </td><td>116.4 </td><td>0.195 </td></tr><tr><td>C18 </td><td>21 </td><td>21,354,662 </td><td>760 </td><td>21 </td><td>1,016,888.7 </td><td>501 </td><td>136.2 </td><td>0.157 </td><td>356 </td><td>138.6 </td><td>0.154 </td><td>386 </td><td>123.8 </td><td>0.172 </td></tr><tr><td>C22 </td><td>22 </td><td>14,917,408 </td><td>578 </td><td>14 </td><td>1,065,529.1 </td><td>408 </td><td>121.0 </td><td>0.123 </td><td>291 </td><td>128.6 </td><td>0.116 </td><td>259 </td><td>102.4 </td><td>0.146 </td></tr><tr><td>C20 </td><td>23 </td><td>19,910,129 </td><td>796 </td><td>22 </td><td>905,005.9 </td><td>508 </td><td>155.4 </td><td>0.128 </td><td>378 </td><td>151.4 </td><td>0.132 </td><td>378 </td><td>129.4 </td><td>0.154 </td></tr><tr><td>Sub-total </td><td> </td><td>524,925,810 </td><td>18,194 </td><td>558 </td><td> </td><td>11,845 </td><td>3,753.9 </td><td>0.140 </td><td>8,357 </td><td>3,978.4 </td><td>0.132 </td><td>8,373 </td><td>3,109.9 </td><td>0.169 </td></tr><tr><td>Not anchored </td><td> </td><td>19,312,992 </td><td>16 </td><td>15,936 </td><td>1,211.9 </td><td>6,369 </td><td> </td><td> </td><td>9,857 </td><td> </td><td> </td><td>9,841 </td><td> </td><td> </td></tr><tr><td>Total </td><td>  </td><td>544,238,802 </td><td>18,214 </td><td>16,494 </td><td>  </td><td>18,214 </td><td> </td><td> </td><td>18,214 </td><td>  </td><td>  </td><td>18,214 </td><td>  </td><td>  </td></tr></tbody></table></div></div></p>                        <a id="120122105" scrollto-destination="120122105"></a>
<div id="dsy015-F2" data-id="dsy015-F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/m_dsy015f2.png?Expires=2147483647&Signature=PZKCfaV3mOguN8~BTELmTJSd9XYxD1Q06dugBiLx9Ktpbypr-~xSXdomblz3EeFOFOCCf0gi1AL1Xg2Vb7mBhCnwGb9II4tUP-ReY4xf0zxR4dboy8kolyu~FFYTDehHs0aHNujCaErswXIBjsoog5zimIFSzMlxYNQI5lNbkIvoy0gjPLe0myeM~LitWMg4qQJnlKGuprerzLoUMl3nxzraAAgVdmDZyLkOPA2W440iwcmbyQ5qA9GMRC0nGB3CrSIYsMAC-0VhO9TiQwmLotbBsvthTdLAMTK04QVfMHLhF8ZJJwUsqBBHxJVzQcg5OjZsB7~1CN4CQIobabwISQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Consensus genetic map of S. maximus using the three most informative full-sib families; genetic distance in cM in the left bar; the position of centromeres (C) or genetic markers far from centromere (T) in acrocentric chromosomes are indicated." path-from-xml="dsy015f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122105/dsy015f2.tif" path-from-xml="dsy015f2.tif" target="_blank">View large</a><a section="120122105" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015f2.png?Expires=2147483647&Signature=Ftg~tHaILDWRQEjkEBDM1uOYENGzwDCcaS8dXYkNUm46qw1ZgrJNBIcyMtxDoAf05B73dzq3IH7TsfYCiPH~bEa9u1GdoDlFnUMOJ-DUghnYIpkFZgRegszxoNbAsDsXwB9x4MYjDOqYEfXkdGDRieL4Ji2ejqpZSv586yXOToKpy0NF8PUGzuEXRMmap8DzQh-eTcKt1M7n21IEIaofDXvLO6oetrkDRmzu1XsqAZjXe5TrDqlAQgEJeVov6Sy5l0JSpF9w2Ij69Yt6kTHTNkptIZFu4FzCMKqKBUEKn1GKz3APYI45b3ru~YXuh2VWkpZVCGu513y6TCGejlLLDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122105&ar=5035912&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy015f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Consensus genetic map of <em>S. maximus</em> using the three most informative full-sib families; genetic distance in cM in the left bar; the position of centromeres (C) or genetic markers far from centromere (T) in acrocentric chromosomes are indicated.</p></div></div><div content-id="dsy015-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/m_dsy015f2.png?Expires=2147483647&Signature=PZKCfaV3mOguN8~BTELmTJSd9XYxD1Q06dugBiLx9Ktpbypr-~xSXdomblz3EeFOFOCCf0gi1AL1Xg2Vb7mBhCnwGb9II4tUP-ReY4xf0zxR4dboy8kolyu~FFYTDehHs0aHNujCaErswXIBjsoog5zimIFSzMlxYNQI5lNbkIvoy0gjPLe0myeM~LitWMg4qQJnlKGuprerzLoUMl3nxzraAAgVdmDZyLkOPA2W440iwcmbyQ5qA9GMRC0nGB3CrSIYsMAC-0VhO9TiQwmLotbBsvthTdLAMTK04QVfMHLhF8ZJJwUsqBBHxJVzQcg5OjZsB7~1CN4CQIobabwISQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Consensus genetic map of S. maximus using the three most informative full-sib families; genetic distance in cM in the left bar; the position of centromeres (C) or genetic markers far from centromere (T) in acrocentric chromosomes are indicated." path-from-xml="dsy015f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122105/dsy015f2.tif" path-from-xml="dsy015f2.tif" target="_blank">View large</a><a section="120122105" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015f2.png?Expires=2147483647&Signature=Ftg~tHaILDWRQEjkEBDM1uOYENGzwDCcaS8dXYkNUm46qw1ZgrJNBIcyMtxDoAf05B73dzq3IH7TsfYCiPH~bEa9u1GdoDlFnUMOJ-DUghnYIpkFZgRegszxoNbAsDsXwB9x4MYjDOqYEfXkdGDRieL4Ji2ejqpZSv586yXOToKpy0NF8PUGzuEXRMmap8DzQh-eTcKt1M7n21IEIaofDXvLO6oetrkDRmzu1XsqAZjXe5TrDqlAQgEJeVov6Sy5l0JSpF9w2Ij69Yt6kTHTNkptIZFu4FzCMKqKBUEKn1GKz3APYI45b3ru~YXuh2VWkpZVCGu513y6TCGejlLLDA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122105&ar=5035912&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy015f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Consensus genetic map of <em>S. maximus</em> using the three most informative full-sib families; genetic distance in cM in the left bar; the position of centromeres (C) or genetic markers far from centromere (T) in acrocentric chromosomes are indicated.</p></div></div><p>Female and male consensus genetic maps differed significantly totalling 8,274.8 cM and 6,791.6 cM (F:M ratio = 1.22), with average LG length of 376.1 and 308.7 cM, respectively. In both females and males LG02 was the largest LG (545.8 vs 474.9 cM) and LG22 the shortest one (292.1 vs 198.7 cM) (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures2.pdf?Expires=2147483647&Signature=VNgFStr9FLU8w1BECvjvHbs6Q26TzCR6B1ziQ6sJxWURy7XAlwpn3g9bbDcrQiVXBi4NAD7fXthcDR0qffugWaNxSrhXEXB0DuPWW7PuBcLRzyCyiz9QdkztnuHURL7K4xvSWQFl8gEoMEDmYNrOMqrsFiripYaamTx6AWd08hzYgoUauSwicWTOVYE2E4F1meK2m~7OZ2q52Am3yTrOQjKojSPLA3yQM-wXfEnDHEF90~GMGRg7I3v7SmAG~1GpFXhv0roDaymH5ygPAn8ZGgwKgpXpk2wVVIvtO2BLUTJY1F6LyQjSZiQIOoRZr9jPoOm0L4b75wRaQQ7eLp9XUA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). As for the species consensus map, <a reveal-id="dsy015-T1" data-open="dsy015-T1" class="link link-reveal link-table xref-fig">Table 1</a> reports average LG lengths also for female and male consensus maps.</p><p>Individual female and male genetic maps (62 in total; <span class="content-section supplementary-material"><a path-from-xml="sup12" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_tables2.xlsx?Expires=2147483647&amp;Signature=ih0SleqhQTp8cq1tZKwJOVoRcuIGZHurnaOzK1GA5pTH97D9Ym~SBbhLMS-OY68qtfA2PCaXKMK-Pyqd5qvqRqisq-pVVMyc0GzcRrT2GrZ067SkF2xyd1OV-hxLof-q2zFkbkYU5paBMjPiOA39LSKEZ4W4CGLwDzH5CfM8K0AxmpLV8JMk24LxnT7chbBXLGXl9eXxu2tmxnPyREtr3gpZX3OvFH47xSLafdvVrBlSJw~LZmkAC97D1d0SOjFV75O-DKGr~mgTgYfoygF1Tn-ZOP~ClcShwFcREzpiUFdSjFs9H7tIotRDJS7hbXtgbwfc7X7HjIQ0qcm0WWK8kw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S2</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup13" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_tables3.xlsx?Expires=2147483647&amp;Signature=3iDrLGKcOPMCA9dyE1~NI5ar1ra7h6xEt7v1vZO8Zf1Y4mocW~QAc7-Yp-i71Wwlz0H1ON2cWX0ec4iuNnLIEyvZD9Xeu~Cc7kSIU9tXD5bSIsgCoQ9EskNAUmzC646kpN3ZxhTvVpbxOXhNTsd21N2swt5lAbJaGcAZRpgxVjChm9M8~1iq~Gs0G9sk-ePT0N3q1wgtSnR8vbVpCQneehCkbiRAnNcziMAyIccRLJk2L7IBrcO4aQXJrgJpVvaWTSOcOl0fw8ZSkjCy77cTpjLTL1XcldgT2WeV2LKo~DDHshvzasu5tLcV5FxIDO1iuIlCwXPrqz38FC6cztkS8g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S3</a></span>) were also constructed for a more detailed comparison of RF between individuals and sexes, and to look for putative chromosome polymorphisms in the turbot population (44 parents). The length of female genetic maps was higher than that of male maps in all families and for all LGs (average F:M ratio = 1.36). For a more detailed comparison, 10,168 markers shared between sexes and mapped in at least one family were selected (<span class="content-section supplementary-material"><a path-from-xml="sup14" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_tables4.xlsx?Expires=2147483647&amp;Signature=1gGP~lDt-SdNPSMaZCPLgvZlyfZgnR91WAiTr9Rpd-pu6pCFc-q2DS4fMgT9TgkpWu73KPXCjrLoizxfiRL8ULtttUTypNbHqkAYe~zQagdhe02JpOsPD7~UDpIOCnoWMUrOt7AwSDyPhX~rlLIwuApVVi5A19dq7-LpZFg08IOjIZ-V9rHTpIT8X0pv0MFMKAlT2UPp8sGt8gSLZP8gPlfkZptJQjDo6qGyKBh40QPIHbm~pxY2N9nmVjYsp0Rc7CO8~SKSvt5tbpU-zgZM7yMoxzvHRJM~Rol5KVy3pRdBefzMe20AnFrYRiEbP8cuD8R4iADBMu5ZfAZbywgwjA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). As expected, the average inter-marker distance for the same pair of markers was significantly higher in females than in males (2.2 and 1.7 cM, respectively; <em>P</em> &lt; 0.001, <em>t</em>-test for related samples). When F:M intervals were compared at each LG, the difference was significant for all groups excluding LG02, LG17 and LG22 (<em>P</em> &gt; 0.05) (<span class="content-section supplementary-material"><a path-from-xml="sup14" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_tables4.xlsx?Expires=2147483647&amp;Signature=1gGP~lDt-SdNPSMaZCPLgvZlyfZgnR91WAiTr9Rpd-pu6pCFc-q2DS4fMgT9TgkpWu73KPXCjrLoizxfiRL8ULtttUTypNbHqkAYe~zQagdhe02JpOsPD7~UDpIOCnoWMUrOt7AwSDyPhX~rlLIwuApVVi5A19dq7-LpZFg08IOjIZ-V9rHTpIT8X0pv0MFMKAlT2UPp8sGt8gSLZP8gPlfkZptJQjDo6qGyKBh40QPIHbm~pxY2N9nmVjYsp0Rc7CO8~SKSvt5tbpU-zgZM7yMoxzvHRJM~Rol5KVy3pRdBefzMe20AnFrYRiEbP8cuD8R4iADBMu5ZfAZbywgwjA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). When F:M intervals were compared across all LGs within each family, again inter-marker distances were higher and significant in females in all but four families (families 16, 22, 47 and 55, <em>P</em> &gt; 0.05; <span class="content-section supplementary-material"><a path-from-xml="sup14" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_tables4.xlsx?Expires=2147483647&amp;Signature=1gGP~lDt-SdNPSMaZCPLgvZlyfZgnR91WAiTr9Rpd-pu6pCFc-q2DS4fMgT9TgkpWu73KPXCjrLoizxfiRL8ULtttUTypNbHqkAYe~zQagdhe02JpOsPD7~UDpIOCnoWMUrOt7AwSDyPhX~rlLIwuApVVi5A19dq7-LpZFg08IOjIZ-V9rHTpIT8X0pv0MFMKAlT2UPp8sGt8gSLZP8gPlfkZptJQjDo6qGyKBh40QPIHbm~pxY2N9nmVjYsp0Rc7CO8~SKSvt5tbpU-zgZM7yMoxzvHRJM~Rol5KVy3pRdBefzMe20AnFrYRiEbP8cuD8R4iADBMu5ZfAZbywgwjA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>).</p>                            <h3 scrollto-destination=120122108 id="120122108" class="section-title" >3.3. Anchoring turbot genetic and physical maps: megascaffolds</h3>
<p>Scaffolds were assigned to the 22 LGs of the turbot consensus map, then ordered and oriented, thus constituting 22 megascaffolds. Series of 100 Ns were used in the megascaffold sequence to show the gap between adjacent scaffolds denoting missing information. Additionally, the short uncertain regions around the boundaries of split scaffolds were written in lowercase to highlight their lower confidence. The complete nucleotide sequence of the 22 megascaffolds with their correspondent genes located and annotated has been uploaded to NCBI (SUB3239805).</p><p>A total of 28 scaffolds were split and reallocated according to the very consistent mapping information in all the 31 families analysed (<span class="content-section supplementary-material"><a path-from-xml="sup15" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_tables5.xlsx?Expires=2147483647&amp;Signature=XtZ9-NcPrMejnhLh1bBfGmsnYP7AoctNMpQ2k5hU9c8c0CN9WboCPUhDUujBy3unr6Y1T4YArj0XI3YlI~Iu7fNMhBykpn0smO22y2fYG77Th5K7ehJQKKzwEwMBzdVlBlyleavqjvG0VmheTpcs3TAaH8pJP29E3oChXSRzMSvF7ogMm6ZLzteHh2R4u95m-XuYvS~tl~MVyo2EkqQEEfoCl~RuU77nS8Rvpl1UVmnXA9leWZJy6TU9XSUndCWOKS4WchwL-k5no1g2CS6KFP0KEJMfaJzyzrRhjWxK4AeEDbibmXjBgPL21HK8D6oB1hRKXqIeSJvOmgaoT1CpYA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Most scaffolds (22) were split into two fragments and reallocated to different LGs. In three cases, scaffolds were split in three different LGs (sm5_s00043, sm5_s00056 and sm5_s00084), and in other three cases, the two fragments of the scaffold were reallocated within the same LG (sm5_s00006, sm5_s00007 and sm5_s00049). In all cases, the boundaries of the breakage point were assigned with notable accuracy (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures1.pdf?Expires=2147483647&amp;Signature=iRjYbKf0EsHectagtCG899YTSLSdL5P9bjXND9ypL1k5V6yJ9N7kaummLyzV8M9Aq~97B5XDuw7YQqAnSpYUQavEVl1qOsU6u8C6qNnsT2dgK1Diph~sWCQiex5kgT5xg3OakOagsJ38En7mNF5NSVezfdYwRnXdmMXhqXoeHpAI~aVtZ9Io8XCLXDkKcp6ngL0Ho8nHSnewzAz7BfLAYUAaXJrHfCwagjaNh4~5QhTV2JP4MIWNYMq4t8cxYezdidf2K0JxD~cRLan2DrLA5G2GDs1P4SeeqOrFjjLMecaT8r8pnM1MQk2TpmdSuiaIzhK9IymwzStqwCzsHinLOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>), and uncertain gap regions averaged 38,010 bp, ranging from 549 bp (sm5_s00086) to 160,155 bp (sm5_s00084) (<span class="content-section supplementary-material"><a path-from-xml="sup15" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_tables5.xlsx?Expires=2147483647&amp;Signature=XtZ9-NcPrMejnhLh1bBfGmsnYP7AoctNMpQ2k5hU9c8c0CN9WboCPUhDUujBy3unr6Y1T4YArj0XI3YlI~Iu7fNMhBykpn0smO22y2fYG77Th5K7ehJQKKzwEwMBzdVlBlyleavqjvG0VmheTpcs3TAaH8pJP29E3oChXSRzMSvF7ogMm6ZLzteHh2R4u95m-XuYvS~tl~MVyo2EkqQEEfoCl~RuU77nS8Rvpl1UVmnXA9leWZJy6TU9XSUndCWOKS4WchwL-k5no1g2CS6KFP0KEJMfaJzyzrRhjWxK4AeEDbibmXjBgPL21HK8D6oB1hRKXqIeSJvOmgaoT1CpYA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Rearranged scaffolds represented a total of 108.5 Mb, to say ∼20% of the entire turbot genome.</p><p>A total of 17,169 SNPs corresponding to 558 scaffolds (after splitting 28) rendered a significant match to the turbot genetic map (<span class="content-section supplementary-material"><a path-from-xml="sup11" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_tables1.xlsx?Expires=2147483647&amp;Signature=iu-s0ucyh61Jqm6Al68IyTnp0EwFC7zFFYSo0Z3cPTfltnRkboXvIDx076CZIb5BEt6jLoHAmQfgwmD~i9kS2Y-catWPu0lQ4QDwsKqAkbCWMHHdmDAC2~bW6Immj2-K3vlVHNbUu8ro6byRUgCLiCUlLpNGoPFmB6-VJqFQx-RcsXQLNQFTCBwIuTMihcwQtJEjlUB5CtJSNGma04LlRv4IHvvQtZxz2~hdm7o2VgohCrtQrsA1Ip~EuVwq56pYaqNwhw2WKH2-~XS4j9KlLjWPc1VJwobAhvRpgiQq2xKc16Y~XOVMteY2rJXBkfxKKn-i59DfiGAuXNvV8U~W0Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). On average each scaffold contained 32.7 SNPs ranging from 610 SNPs in sm5_s00003_17 (LG17) to one SNP in 204 scaffolds. The total length of the anchored sequences was 524,925,810 bp, representing 96.5% of the turbot genome assembly (544 Mb). The average megascaffold length was 23,860,264 bp, ranging from 14,917,408 bp (LG22) to 31,897,078 bp (LG02) (<a reveal-id="dsy015-T1" data-open="dsy015-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Megascaffolds contained on average 827 SNPs (range 578–1144) and 25.4 scaffolds (range 14–46) (<a reveal-id="dsy015-T1" data-open="dsy015-T1" class="link link-reveal link-table xref-fig">Table 1</a>). All scaffolds assigned to chromosomes were consistently ordered within the megascaffold according to the mapping information across families; the orientation within each chromosome could be verified for those 336 that contained at least two informative markers. The remaining 204 scaffolds with only one marker could not be oriented, but they represented a very minor proportion of the total assembly (2.0%) because of their small size. Interestingly, they were mostly located at the end of chromosomes (telomeres), suggesting assembling problems caused by repetitive motifs.</p>                            <h3 scrollto-destination=120122112 id="120122112" class="section-title" >3.4. Integrating cytogenetic information: chromosomes vs LGs vs megascaffolds</h3>
<p>Turbot genetic, physical and cytogenetic information obtained in this work and from previous studies was integrated as far as possible, and accordingly, a unified nomenclature is proposed in this study (<a reveal-id="dsy015-T1" data-open="dsy015-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Four chromosomes (C1, C2, C3 and C22)<a reveal-id="dsy015-B34" data-open="dsy015-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> were unequivocally assigned to specific LGs: the two metacentric C1 (LG02) and C2 (LG16), the submetacentric nucleolus organiser region (NOR)-bearing chromosome C3 (LG15) and the smallest acrocentric chromosome C22 (LG22), constituting the four marker chromosomes identifiable in the turbot karyotype. The remaining megascaffolds, which correspond to acrocentric chromosomes of decreasing size, were termed according to their physical length from C4 to C21 (<a reveal-id="dsy015-T1" data-open="dsy015-T1" class="link link-reveal link-table xref-fig">Table 1</a>).</p><p>The consensus genetic and physical maps showed a linear correspondence along most chromosome length in all LGs, excluding those regions around centromeres and telomeres (<a reveal-id="dsy015-F3" data-open="dsy015-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a> and <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures3.pdf?Expires=2147483647&amp;Signature=rze0qaBj8BUmDYGTmlsnfb3I0AFDqsNCedVFBj0IQHFQK0cY1b9yr0DeEewu5JIhnFlgKS8h3lySXwBvBDsVs3JUA54eHOv55brf~pp6RA8pErLbudARu-2B9Zq5hm-bgKpDWdQJA13njpUy07wgyRflkC9Mn8nqQho041JIztv8QPCCaDY7lt2PtzYWNLmUYEjMlLdrUjnyQ1GaK0H8tRFLmex39MmH9VzrtGPDk8rfxN4E~Fl~DwXwlYcp4KNZp0MbUoSqQs4W4dbJHD3Cq~CXaWO~Wq~cr-1nYJpwunmjZEfKbPsixBm0eqxAhe-NUpsYdoAM-Qcnz3hvKgK7UA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). A cloud of dots indicative of poor mapping accuracy were observed around most mapped centromeres<a reveal-id="dsy015-B40" data-open="dsy015-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> (i.e. LG01, LG10, LG11, LG19 and LG20). Dot clouds were located in intermediate regions of the metacentric chromosomes (LG02 and LG16) or at the terminal regions in the acrocentrics in accordance with their cytogenetic structure. LG15, the NOR-bearing chromosome, showed a moderate dispersion of dots throughout the proximal centromere chromosome half likely related to the tandem rDNA clusters and their associated heterochromatin in that region.<a reveal-id="dsy015-B52" data-open="dsy015-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> Also, close to telomeres, although not so markedly, dots were either dispersed (i.e. LG03, LG08, LG11 and LG14) or showing particular drawings (LG02, LG06 and LG15) suggestive of wrong physical assembly or genetic mapping. Finally, the pattern of the acrocentric chromosome C5 (LG04) was remarkable because of the presence of an internal cloud of dots in addition to that associated around the centromere at the chromosome end. This LG very likely correspond to the chromosome 5 of turbot karyotype, which showed an interstitial heterochromatic block with C-banding.<a reveal-id="dsy015-B52" data-open="dsy015-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> When the same comparison between genetic and physical maps was performed for the female and male consensus maps, dot clouds around centromeres were very prominent in the male map and subtle, and sometimes inexistent, in the female maps (<a reveal-id="dsy015-F3" data-open="dsy015-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a> and <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures4.pdf?Expires=2147483647&amp;Signature=ZXzMONe2mxWzVNQBOd2IHH33n-9uVtGWjdlB4dJQlBdrH48tT-wLdDI~ST7OwRB~mB2sZqvJ2iwqAV7B1hKIPnQkw4Ed8pgphCHI1FKv0R0gr3eSps4YQxOeZW-GPkkT8-7qrhgpp~HYY-ZiFOVFtetq7ELUrjEgi~Yshldr1hVUIA6Y~mnCp~rf8VJgZ291wC17wanN-Y0A5jTh~jOnDKaJPtZwVkBB6NCvCeYx7sw6H0SdTK2-UEDZELyYVZRF0belrdmBj4Dz96j2wZtK8tMoFloZ0hYiHLm5f25YF8~-RmWzqyHLkiMUjqAThQ~1q~Vso3qMTMJu4xn5Vbyzsg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>; i.e. LG01, LG05, LG09, LG15, LG19 and LG22). So, it seems that the length difference between male and female maps is mainly related to the lower recombination rates around centromeres characterising male maps. </p>                        <a id="120122115" scrollto-destination="120122115"></a>
<div id="dsy015-F3" data-id="dsy015-F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/m_dsy015f3.png?Expires=2147483647&Signature=u5DBGNH3gTwEuCB27EHxNaprGtDZkIrr-cgAHlr8n6~c1EbmOPP1Gvp7Vm8C4HAyVC2b~buPFE6thypDuKBWqVCqECU~kKHsMkUXVg4PkY3Tode2DEp7tWrY6aAHXkPHtyADhJv2sKnUtoZRqIZ7FMFQKg9N5xwdbU3olxoexg-BzCN9nwYHiIf7BSF4HQo1dSFsicTHQwS4v2fyh5lCQlA5gtKbvrYhnf-Zbi6X516nHBQXvX-XiKfNwqO5oudq7iHyMgwZ~Nh5k09mFo33eapAKZYs5Z1MA2~ICKRLrj66zRqe14W-~1yXKjIzeANTFl4G0ip3yjFyZu~ldMB3lQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correspondence between physical (abscissae) and genetic (ordinates) maps showing the dispersion of dots around the centromere at LG11 (A) and the difference between male and female maps at LG11 (B). The vertical bar in both figures show the position of the centromere according to Martínez et al. (2008)42. Genetic markers corresponding to different scaffolds are shown in different colors." path-from-xml="dsy015f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122115/dsy015f3.tif" path-from-xml="dsy015f3.tif" target="_blank">View large</a><a section="120122115" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015f3.png?Expires=2147483647&Signature=BtXG-eYy5c3HIYioz4GI5x0Ld6cc~mVGQKLUIIKYJ24k-sj7RmV1nYVKdWVk6eec8M04kzFOmtW~YXdb49502HhcUQK0BMh8uvYraiYRGsYU8dnpp~eB0Qe0DL8CFwPf2U3osEiOwT34vSRzhQ6LpJRuG2SdoeoY5Kp-O27Kgdh1B~e3o939vphixBK5w2ExGSdk9cdbqEnZgFkuwCQYZFs66gLUXV7PkZnufr63rnKfZ-u9KcNfHsNBUEt6-544Kas1PDDw~XbNa7tzCRkMjKCvtG~gVM6HqVG1frZlXFYrbVoUILdGbsYqrGmMkYOwpU0LYSvwaiwnfIF0XFLrvg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122115&ar=5035912&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy015f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correspondence between physical (abscissae) and genetic (ordinates) maps showing the dispersion of dots around the centromere at LG11 (A) and the difference between male and female maps at LG11 (B). The vertical bar in both figures show the position of the centromere according to Martínez et al. (2008)<a reveal-id="dsy015-B42" data-open="dsy015-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a>. Genetic markers corresponding to different scaffolds are shown in different colors.</p></div></div><div content-id="dsy015-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/m_dsy015f3.png?Expires=2147483647&Signature=u5DBGNH3gTwEuCB27EHxNaprGtDZkIrr-cgAHlr8n6~c1EbmOPP1Gvp7Vm8C4HAyVC2b~buPFE6thypDuKBWqVCqECU~kKHsMkUXVg4PkY3Tode2DEp7tWrY6aAHXkPHtyADhJv2sKnUtoZRqIZ7FMFQKg9N5xwdbU3olxoexg-BzCN9nwYHiIf7BSF4HQo1dSFsicTHQwS4v2fyh5lCQlA5gtKbvrYhnf-Zbi6X516nHBQXvX-XiKfNwqO5oudq7iHyMgwZ~Nh5k09mFo33eapAKZYs5Z1MA2~ICKRLrj66zRqe14W-~1yXKjIzeANTFl4G0ip3yjFyZu~ldMB3lQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correspondence between physical (abscissae) and genetic (ordinates) maps showing the dispersion of dots around the centromere at LG11 (A) and the difference between male and female maps at LG11 (B). The vertical bar in both figures show the position of the centromere according to Martínez et al. (2008)42. Genetic markers corresponding to different scaffolds are shown in different colors." path-from-xml="dsy015f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122115/dsy015f3.tif" path-from-xml="dsy015f3.tif" target="_blank">View large</a><a section="120122115" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015f3.png?Expires=2147483647&Signature=BtXG-eYy5c3HIYioz4GI5x0Ld6cc~mVGQKLUIIKYJ24k-sj7RmV1nYVKdWVk6eec8M04kzFOmtW~YXdb49502HhcUQK0BMh8uvYraiYRGsYU8dnpp~eB0Qe0DL8CFwPf2U3osEiOwT34vSRzhQ6LpJRuG2SdoeoY5Kp-O27Kgdh1B~e3o939vphixBK5w2ExGSdk9cdbqEnZgFkuwCQYZFs66gLUXV7PkZnufr63rnKfZ-u9KcNfHsNBUEt6-544Kas1PDDw~XbNa7tzCRkMjKCvtG~gVM6HqVG1frZlXFYrbVoUILdGbsYqrGmMkYOwpU0LYSvwaiwnfIF0XFLrvg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122115&ar=5035912&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy015f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correspondence between physical (abscissae) and genetic (ordinates) maps showing the dispersion of dots around the centromere at LG11 (A) and the difference between male and female maps at LG11 (B). The vertical bar in both figures show the position of the centromere according to Martínez et al. (2008)<a reveal-id="dsy015-B42" data-open="dsy015-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a>. Genetic markers corresponding to different scaffolds are shown in different colors.</p></div></div>                            <h3 scrollto-destination=120122116 id="120122116" class="section-title" >3.5. Intraspecific chromosome reorganisations</h3>
<p>Marker positions showed a good correspondence between the 62 parental genetic maps (including ‘replicates’ from half-sib families) although particular drawings were suggestive of different marker order between individual maps (<a reveal-id="dsy015-F4" data-open="dsy015-F4" class="link link-reveal link-table xref-fig">Fig. 4</a> and <span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures5.pdf?Expires=2147483647&amp;Signature=KCOzgUfV01oiE4OE5USGGzbMSVkItECEhE27atsiYrZogtIP5h~pEP2y~m1Fl5qz4HUVrropQd4vc273vI~NVd3mzLaAb5WzrV-VZ2RSOKcVJTF6rnk3zKiL7rf5mFv0FD5FjFXrijiCM6uLlEKUZikOa7chth75m4Nl02dHuWjhqHi0YITrJvJHvUwQ9tx1yJYE8MmG73FYc2U4ayk5YIc9T2WHIJXZwBSIij4oIEl91DF-qIwsbEiD8J~FXnqw775w6Kv~6vy6f-WgESFRuDD7rlIt~OwRSaAOeyf-tSfSIn1SSGI5TBYbwojEKziv46L2BRYj3U6Ie6qFEiXQxg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). The negative correlations between physical and genetic maps at these regions supported this observation, and the presence of putative chromosome rearrangements that involved either single blocks (50 SNPs, ∼1.6 Mb) or longer chromosome tracts was identified (<span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures6.pdf?Expires=2147483647&amp;Signature=BzbimyjEMuBvB5cpAH3JQaykawpE8WAtNc06q-CwLlXUcLTFlmY9va00Bm-rhD2WbEGZvpZOH~jmdPi0nb0o6liq1eIVVu-GB78MnXe58YKKbBRfqaE3XsV2B7rPtNhoZs4LTm85Exa6GmnKMQPUe1o~~6fBHDSlI8TzXFRQXvpzyH3TZE2aWTMl9sJ0fN8hz7TgGM~6LPr39-60xogiHY~yCVkwoo~kJhSjTTuvX0F3MW1EoHWr9DwCitIfJCFjnNboyHlQVWfBgs5B9FtKXlWJuYYXY8TyCVAfyCO~DBwVADhWZHUBZC2v~yrsdgyM3KUbvshZMYguSBaK2klcag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). Inverted single blocks were very frequent at LG02 (blocks 7 and 23), LG05 (block 12), LG08 (block 2), LG13 (blocks 14 and 15) and LG22 (block 12), while larger inverted chromosome tracts were observed in some individuals at LG06 (blocks 5–9) and LG09 (blocks 13–16) (<span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures5.pdf?Expires=2147483647&amp;Signature=KCOzgUfV01oiE4OE5USGGzbMSVkItECEhE27atsiYrZogtIP5h~pEP2y~m1Fl5qz4HUVrropQd4vc273vI~NVd3mzLaAb5WzrV-VZ2RSOKcVJTF6rnk3zKiL7rf5mFv0FD5FjFXrijiCM6uLlEKUZikOa7chth75m4Nl02dHuWjhqHi0YITrJvJHvUwQ9tx1yJYE8MmG73FYc2U4ayk5YIc9T2WHIJXZwBSIij4oIEl91DF-qIwsbEiD8J~FXnqw775w6Kv~6vy6f-WgESFRuDD7rlIt~OwRSaAOeyf-tSfSIn1SSGI5TBYbwojEKziv46L2BRYj3U6Ie6qFEiXQxg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S5</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures6.pdf?Expires=2147483647&amp;Signature=BzbimyjEMuBvB5cpAH3JQaykawpE8WAtNc06q-CwLlXUcLTFlmY9va00Bm-rhD2WbEGZvpZOH~jmdPi0nb0o6liq1eIVVu-GB78MnXe58YKKbBRfqaE3XsV2B7rPtNhoZs4LTm85Exa6GmnKMQPUe1o~~6fBHDSlI8TzXFRQXvpzyH3TZE2aWTMl9sJ0fN8hz7TgGM~6LPr39-60xogiHY~yCVkwoo~kJhSjTTuvX0F3MW1EoHWr9DwCitIfJCFjnNboyHlQVWfBgs5B9FtKXlWJuYYXY8TyCVAfyCO~DBwVADhWZHUBZC2v~yrsdgyM3KUbvshZMYguSBaK2klcag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S6</a></span>). However, these putative inversions, particularly those related to single blocks, were found close to telomeres or centromeres, where mapping accuracy is poorer both due to assembling problems and to the lower ratio of genetic/physical distance. This suggests that they are more likely related to technical devices than to true reorganisations. Otherwise, the longer tracts observed at LG06 and LG09 were located at interstitial regions, and thus, signatures of inversions should be more consistent. </p>                        <a id="120122118" scrollto-destination="120122118"></a>
<div id="dsy015-F4" data-id="dsy015-F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/m_dsy015f4.png?Expires=2147483647&Signature=DSCxsYuP6-iAeyCqoO66uPAQ9~Hwtf6ddl7bUwP3IQlJPfKp8~2QB1-T9EzIGvmfqR~626KEQMwM37dIQn4wSDzrpOEtQvpSkAsqwaji~7cPTUidMunHP~CPh48cA0ZZA9xLcdKtnw8KP3d1gJY78XtKH968XSEj0CMLQjXV62WnlW~FoqDZc568zbRqy5tM2yPZb7PCJvyXIOEkTs0dB~xWsOJYnKfP32ShWmGKPSfGzFEUx19vJDuZVJwLOlR5XZnZs~Vem~3tfJh5gdovbY~Cm79GoUqbcTbq3nohuMSoeGavHOD2efiCL~ABtUMPZ~jytdJv2PzchR1V7G83mQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Particular drawings denoting the lack of collinearity between two individual genetic maps of S. maximus: Female Fam09 vs Male Fam07 maps; consecutive LGs are represented in different colours from LG01 (left bottom) to LG23 (right top)." path-from-xml="dsy015f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122118/dsy015f4.tif" path-from-xml="dsy015f4.tif" target="_blank">View large</a><a section="120122118" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015f4.png?Expires=2147483647&Signature=P60hFK6PuFBRtGLpjtsKwMmUO5x2XGtCuQryDgh9uHhZ-6rS5Hkn1KbbBSRCB9Ahejk0OMy5CEuUqSAi9Ch0ZrBd90KfS4IjcJhObI6LmJh1oYIXbGEk9yuKTsGQc0nK6t3oDg-dsGx32mQSE7sP9ubOo7Re9pOSRMwT88eEfWNUeXdYt2YITyLIhP-KtmpLA4wr2PB9TPz4oYWAhQfaK8I6Yt-KEoalUZ-4i5QjoXz1LEv-rmlaeGS77ft0Up7dIXEh7sVAi54pavP~scZVPoBEVf-hFCaZIPRRJF8Lnr7XYcxgXdSN0xrMMFTA2wTU41jrbkRAiTeWeDvQMyhlhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122118&ar=5035912&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy015f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Particular drawings denoting the lack of collinearity between two individual genetic maps of <em>S. maximus</em>: Female Fam09 vs Male Fam07 maps; consecutive LGs are represented in different colours from LG01 (left bottom) to LG23 (right top).</p></div></div><div content-id="dsy015-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/m_dsy015f4.png?Expires=2147483647&Signature=DSCxsYuP6-iAeyCqoO66uPAQ9~Hwtf6ddl7bUwP3IQlJPfKp8~2QB1-T9EzIGvmfqR~626KEQMwM37dIQn4wSDzrpOEtQvpSkAsqwaji~7cPTUidMunHP~CPh48cA0ZZA9xLcdKtnw8KP3d1gJY78XtKH968XSEj0CMLQjXV62WnlW~FoqDZc568zbRqy5tM2yPZb7PCJvyXIOEkTs0dB~xWsOJYnKfP32ShWmGKPSfGzFEUx19vJDuZVJwLOlR5XZnZs~Vem~3tfJh5gdovbY~Cm79GoUqbcTbq3nohuMSoeGavHOD2efiCL~ABtUMPZ~jytdJv2PzchR1V7G83mQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Particular drawings denoting the lack of collinearity between two individual genetic maps of S. maximus: Female Fam09 vs Male Fam07 maps; consecutive LGs are represented in different colours from LG01 (left bottom) to LG23 (right top)." path-from-xml="dsy015f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122118/dsy015f4.tif" path-from-xml="dsy015f4.tif" target="_blank">View large</a><a section="120122118" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015f4.png?Expires=2147483647&Signature=P60hFK6PuFBRtGLpjtsKwMmUO5x2XGtCuQryDgh9uHhZ-6rS5Hkn1KbbBSRCB9Ahejk0OMy5CEuUqSAi9Ch0ZrBd90KfS4IjcJhObI6LmJh1oYIXbGEk9yuKTsGQc0nK6t3oDg-dsGx32mQSE7sP9ubOo7Re9pOSRMwT88eEfWNUeXdYt2YITyLIhP-KtmpLA4wr2PB9TPz4oYWAhQfaK8I6Yt-KEoalUZ-4i5QjoXz1LEv-rmlaeGS77ft0Up7dIXEh7sVAi54pavP~scZVPoBEVf-hFCaZIPRRJF8Lnr7XYcxgXdSN0xrMMFTA2wTU41jrbkRAiTeWeDvQMyhlhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122118&ar=5035912&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy015f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Particular drawings denoting the lack of collinearity between two individual genetic maps of <em>S. maximus</em>: Female Fam09 vs Male Fam07 maps; consecutive LGs are represented in different colours from LG01 (left bottom) to LG23 (right top).</p></div></div>                            <h3 scrollto-destination=120122119 id="120122119" class="section-title" >3.6. Interspecific chromosome reorganisations</h3>
<p>The whole turbot megascaffold sequences were aligned against the genomes of Japanese flounder and tongue sole using the LASTZ program and a 1:1 macrosyntenic pattern was observed (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures7a.pdf?Expires=2147483647&amp;Signature=BG9oRJnYtqu01TUzVT74Llro4Y78oayaa8C3f7A5g0UfzR0MWX4Zkp25RWo5TfU702CVdbedz5BZ0psae1LZepJaLFpAoDdaeoXwwBqgrgCNbTSi1pT6YeriYkYUP75~imqtMHTTW7pIRid49xCYr1uc7e7hWIEkLIYltgliP4I3abhI-yvwVGSjFGuQoMoPHU9T-tw634brhzoJ7OGBx1GiG6MOyqA0reGrvfYL93dXHBmGNMeVCzyFLPTQwVGGdzmtUGDtM6PAW45Kvsyuc~vUFY4Jqr-hSj9U3Yt7O0fNMrHCOHSCxY4smJOdntavjDuVLB5SNBaEknszi9znIA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7A</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures7b.pdf?Expires=2147483647&amp;Signature=qXtL7nyzxZ4bS3ILiX36gnIZf9BazsbIoOV3jwT25ef8-ke0ReoNkcR8QlOqmG5Hb5gYYRGPbp96AR8~nJkPBi57WDRd-9eFYqQFLgCDp2jDEMf3ykqIJmlA1QefKPLkkF1AbrJJwhw9ufXVSxVDiABwSHnmMtTL8ZhlPVGUVhDPpMSRifXDV5jFLy9pNRNNrA24QzYsaPTFBjk1sFcUVam94hnVFs3rYKGSuTd6vn40c6Kb7Vtojr0KZzPtD308sOow0vJ6dXQRpAWedFAxxnyCSsYd2QNx2oV5arCTmxcR-eOsznoWhcYe5gb798kMLsUsw0Rambrs1tn~hmpsQg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">B</a></span>), as reported previously.<a reveal-id="dsy015-B9" data-open="dsy015-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> However, some Robertsonian translocations (fusions or fissions) and other minor inter-chromosomal rearrangements were observed. The most notable cases when comparing turbot (<em>n</em> = 22) and Japanese flounder (<em>n</em> = 24) genomes were the correspondence of turbot LG02 with Japanese flounder Chr06 and Chr14 (<a reveal-id="dsy015-F5" data-open="dsy015-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>) and that of LG16 with Chr09 and Chr16 (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures7a.pdf?Expires=2147483647&amp;Signature=BG9oRJnYtqu01TUzVT74Llro4Y78oayaa8C3f7A5g0UfzR0MWX4Zkp25RWo5TfU702CVdbedz5BZ0psae1LZepJaLFpAoDdaeoXwwBqgrgCNbTSi1pT6YeriYkYUP75~imqtMHTTW7pIRid49xCYr1uc7e7hWIEkLIYltgliP4I3abhI-yvwVGSjFGuQoMoPHU9T-tw634brhzoJ7OGBx1GiG6MOyqA0reGrvfYL93dXHBmGNMeVCzyFLPTQwVGGdzmtUGDtM6PAW45Kvsyuc~vUFY4Jqr-hSj9U3Yt7O0fNMrHCOHSCxY4smJOdntavjDuVLB5SNBaEknszi9znIA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7A</a></span>). In the same way, turbot LG02 was related to tongue sole Chr13 and Chr14 (<a reveal-id="dsy015-F5" data-open="dsy015-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>), which suggests that LG02 fusion is a derived condition (apomorphy) in the turbot lineage. The position of turbot centromeres at LG02 was the expected one if a centric fusion between two chromosomes had occurred in the turbot lineage (<a reveal-id="dsy015-F5" data-open="dsy015-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>), as also occurred for LG16 (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures7a.pdf?Expires=2147483647&amp;Signature=BG9oRJnYtqu01TUzVT74Llro4Y78oayaa8C3f7A5g0UfzR0MWX4Zkp25RWo5TfU702CVdbedz5BZ0psae1LZepJaLFpAoDdaeoXwwBqgrgCNbTSi1pT6YeriYkYUP75~imqtMHTTW7pIRid49xCYr1uc7e7hWIEkLIYltgliP4I3abhI-yvwVGSjFGuQoMoPHU9T-tw634brhzoJ7OGBx1GiG6MOyqA0reGrvfYL93dXHBmGNMeVCzyFLPTQwVGGdzmtUGDtM6PAW45Kvsyuc~vUFY4Jqr-hSj9U3Yt7O0fNMrHCOHSCxY4smJOdntavjDuVLB5SNBaEknszi9znIA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7A</a></span>). These chromosome fusions, starting from the ancestral teleost karyotype (2<em>n</em> = 48),<a reveal-id="dsy015-B57" data-open="dsy015-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> would explain the karyotype differences between turbot and Japanese flounder, as suggested by Robledo et al.<a reveal-id="dsy015-B9" data-open="dsy015-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Despite the general macrosyntenic pattern observed, pairwise alignment of chromosomal sequences indicated that extensive intrachromosomal reorganisations (mainly inversions) would have taken place throughout the evolution of these species (<span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures7a.pdf?Expires=2147483647&amp;Signature=BG9oRJnYtqu01TUzVT74Llro4Y78oayaa8C3f7A5g0UfzR0MWX4Zkp25RWo5TfU702CVdbedz5BZ0psae1LZepJaLFpAoDdaeoXwwBqgrgCNbTSi1pT6YeriYkYUP75~imqtMHTTW7pIRid49xCYr1uc7e7hWIEkLIYltgliP4I3abhI-yvwVGSjFGuQoMoPHU9T-tw634brhzoJ7OGBx1GiG6MOyqA0reGrvfYL93dXHBmGNMeVCzyFLPTQwVGGdzmtUGDtM6PAW45Kvsyuc~vUFY4Jqr-hSj9U3Yt7O0fNMrHCOHSCxY4smJOdntavjDuVLB5SNBaEknszi9znIA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7A</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures7b.pdf?Expires=2147483647&amp;Signature=qXtL7nyzxZ4bS3ILiX36gnIZf9BazsbIoOV3jwT25ef8-ke0ReoNkcR8QlOqmG5Hb5gYYRGPbp96AR8~nJkPBi57WDRd-9eFYqQFLgCDp2jDEMf3ykqIJmlA1QefKPLkkF1AbrJJwhw9ufXVSxVDiABwSHnmMtTL8ZhlPVGUVhDPpMSRifXDV5jFLy9pNRNNrA24QzYsaPTFBjk1sFcUVam94hnVFs3rYKGSuTd6vn40c6Kb7Vtojr0KZzPtD308sOow0vJ6dXQRpAWedFAxxnyCSsYd2QNx2oV5arCTmxcR-eOsznoWhcYe5gb798kMLsUsw0Rambrs1tn~hmpsQg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">B</a></span>). Reorganisations were much more extensive when comparing turbot to half-smooth tongue sole than to Japanese flounder (<a reveal-id="dsy015-F5" data-open="dsy015-F5" class="link link-reveal link-table xref-fig">Fig. 5C and D</a>). The results of the comparison with stickleback are presented in <span class="content-section supplementary-material"><a path-from-xml="sup9" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures7c.pdf?Expires=2147483647&amp;Signature=jLFpXGWltyztBbLsB9yLRRlx5Atwpr0RN3aFtDhsQubxXF1KX2H1mYSK6Jqs3dFZBR19IzXCcvPCj0tzkL7-bJPTcKGwKGwutmPVdr6cqWO7pV7Tg2q4H4oun3uqkAPTxhURXHz-pSH19k41EB0XSSupdkzjrTid2iBkEXcxfsJkBIkP4UPfijO0xcDQDvfqBWVptovHgg1CT1CLwrkmIzmbuZcUR3TK2gFpnN60u8LAA4ffez8~SdjD6lTVvzdL1LN-74hnmWpNz-GStBFv28EM3FskzwX41PFtP-3YOy5ROQEUNF6Qq19j6dFC4NkiLQa~h5odyWuAryWzDLyekg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S7C</a></span>. Similar fusion patterns as those identified in the comparison with the other flatfish species are visible, as for example at LG16, that showed macrosyntenic patterns with stickleback’s ChrVII and ChrIV. </p>                        <a id="120122121" scrollto-destination="120122121"></a>
<div id="dsy015-F5" data-id="dsy015-F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/m_dsy015f5.png?Expires=2147483647&Signature=b0o9TuxA1CuxA2NOqh3C-gG2KmoSfWCJzsfuZsx8F~nfOaCPyV6Y9R~PojXBBewXgAD3lkDJFWBitg2XX4V6INPi91O4yd6Ln---nr-PulxE1-9Z3Dha0bvNtB-ndTk5xhD3epHA-OE77TxU7XNw73dDylaEZ3q6w5BCvyciaGcMhpvyt1VrXdvTeWyPd~4RBjXNOgnDW8EaNRBNpaZKIpGQ7n0QuQLB-DaaA4V-Z8VqI0OR7xavf9WIBtmsxLOCF-GYjJ9jL3p6zWIqzyu9wO1TsO6AKtx367wFUjr9mScMrwdGKq3rLeKB8XQZEkoT2ltHuoumfd0h1gMv2qPC2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="LASTZ plots between S. maximus linkage groups (LG) and chromosomes (Chr) of P. olivaceus and C. semilaevis: (A) LG02 vs Chr06 and 14 of P. olivaceus; (B) LG02 vs Chr13 and 14 of C. semilaevis; (C) LG04 vs Chr07 of P. olivaceus; (D) LG04 vs Chr05 C. semilaevis. The gross vertical bar points the position of centromeres according to Martínez et al. (2008)42 and the light ones the limits between consecutive scaffolds. " path-from-xml="dsy015f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122121/dsy015f5.tif" path-from-xml="dsy015f5.tif" target="_blank">View large</a><a section="120122121" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015f5.png?Expires=2147483647&Signature=CTpCwfJnqRebvTeqgYT-NKzBp0gWqRJVj43nCTrG5~lxTIVVYkDL-Em9xUI9VQ0sqGxNJI8kmzM18hJiuUDahT6Kx3NFFJdKRvX4CVaorPvykvxUqRiahVn2kLFkUC~Zq9yRMuyflWycINe-4z89Zw9tUEG3ERORioY3iAGcj0IJZGb8cUMdgKwkf68eptmtf7N~g74uoxQ7IKaP24u3mYeaWjCLwkK50ZiCYsPwR5tEKaBvt21NX~R1icfyN5EY4XxzP4CQL9kw2YQcNNlAduJSHdy8yAwrbR6wBz9b9yNY2yPRw16WxOaUBdCwZpJVYcXyENsJx~iIadv311xlwA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122121&ar=5035912&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy015f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>LASTZ plots between <em>S. maximus</em> linkage groups (LG) and chromosomes (Chr) of <em>P. olivaceus</em> and <em>C. semilaevis</em>: (A) LG02 vs Chr06 and 14 of <em>P. olivaceus</em>; (B) LG02 vs Chr13 and 14 of <em>C. semilaevis</em>; (C) LG04 vs Chr07 of <em>P. olivaceus</em>; (D) LG04 vs Chr05 <em>C. semilaevis</em>. The gross vertical bar points the position of centromeres according to Martínez et al. (2008)<a reveal-id="dsy015-B42" data-open="dsy015-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> and the light ones the limits between consecutive scaffolds. </p></div></div><div content-id="dsy015-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/m_dsy015f5.png?Expires=2147483647&Signature=b0o9TuxA1CuxA2NOqh3C-gG2KmoSfWCJzsfuZsx8F~nfOaCPyV6Y9R~PojXBBewXgAD3lkDJFWBitg2XX4V6INPi91O4yd6Ln---nr-PulxE1-9Z3Dha0bvNtB-ndTk5xhD3epHA-OE77TxU7XNw73dDylaEZ3q6w5BCvyciaGcMhpvyt1VrXdvTeWyPd~4RBjXNOgnDW8EaNRBNpaZKIpGQ7n0QuQLB-DaaA4V-Z8VqI0OR7xavf9WIBtmsxLOCF-GYjJ9jL3p6zWIqzyu9wO1TsO6AKtx367wFUjr9mScMrwdGKq3rLeKB8XQZEkoT2ltHuoumfd0h1gMv2qPC2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="LASTZ plots between S. maximus linkage groups (LG) and chromosomes (Chr) of P. olivaceus and C. semilaevis: (A) LG02 vs Chr06 and 14 of P. olivaceus; (B) LG02 vs Chr13 and 14 of C. semilaevis; (C) LG04 vs Chr07 of P. olivaceus; (D) LG04 vs Chr05 C. semilaevis. The gross vertical bar points the position of centromeres according to Martínez et al. (2008)42 and the light ones the limits between consecutive scaffolds. " path-from-xml="dsy015f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122121/dsy015f5.tif" path-from-xml="dsy015f5.tif" target="_blank">View large</a><a section="120122121" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015f5.png?Expires=2147483647&Signature=CTpCwfJnqRebvTeqgYT-NKzBp0gWqRJVj43nCTrG5~lxTIVVYkDL-Em9xUI9VQ0sqGxNJI8kmzM18hJiuUDahT6Kx3NFFJdKRvX4CVaorPvykvxUqRiahVn2kLFkUC~Zq9yRMuyflWycINe-4z89Zw9tUEG3ERORioY3iAGcj0IJZGb8cUMdgKwkf68eptmtf7N~g74uoxQ7IKaP24u3mYeaWjCLwkK50ZiCYsPwR5tEKaBvt21NX~R1icfyN5EY4XxzP4CQL9kw2YQcNNlAduJSHdy8yAwrbR6wBz9b9yNY2yPRw16WxOaUBdCwZpJVYcXyENsJx~iIadv311xlwA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122121&ar=5035912&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy015f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>LASTZ plots between <em>S. maximus</em> linkage groups (LG) and chromosomes (Chr) of <em>P. olivaceus</em> and <em>C. semilaevis</em>: (A) LG02 vs Chr06 and 14 of <em>P. olivaceus</em>; (B) LG02 vs Chr13 and 14 of <em>C. semilaevis</em>; (C) LG04 vs Chr07 of <em>P. olivaceus</em>; (D) LG04 vs Chr05 <em>C. semilaevis</em>. The gross vertical bar points the position of centromeres according to Martínez et al. (2008)<a reveal-id="dsy015-B42" data-open="dsy015-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> and the light ones the limits between consecutive scaffolds. </p></div></div>                            <h2 scrollto-destination=120122122 id="120122122" class="section-title" >4. Discussion</h2>
<p>The reduced cost of the new genotyping by sequencing (GBS) techniques has facilitated the development of highly dense genetic maps in aquaculture fish species,<a reveal-id="dsy015-B3" data-open="dsy015-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsy015-B17 dsy015-B18 dsy015-B19" data-open="dsy015-B17 dsy015-B18 dsy015-B19" class="link link-ref link-reveal xref-bibr"><sup>17–19</sup></a> including turbot.<a reveal-id="dsy015-B21" data-open="dsy015-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> Here, we combined for the first time a high number of markers and families for a deep mapping analysis aimed at integrating genetic and physical map resources in turbot to understand its genome organisation and variation, a study not performed to date in fish.<a reveal-id="dsy015-B23" data-open="dsy015-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a><sup>,</sup><a reveal-id="dsy015-B58" data-open="dsy015-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> The average number of SNPs in the 31 full-sib families analysed in our work was in the upper range reported in the most recent linkage mapping studies (5–6k SNPs) and ∼40 individuals were genotyped at each family. The turbot consensus map (8,532.6 cM) was much longer than any previous linkage map reported for the species (6,647 SNPs, 2,622.1 cM<a reveal-id="dsy015-B21" data-open="dsy015-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a>), but this elongation is the consequence of combining a high number of markers with several mapping families and the MergeMaps software, an artefact highlighted previously by different authors.<a reveal-id="dsy015-B24" data-open="dsy015-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a><sup>,</sup><a reveal-id="dsy015-B59" data-open="dsy015-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a><sup>,</sup><sup>–</sup><a reveal-id="dsy015-B61" data-open="dsy015-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> A direct consequence of this ‘inflated’ map length is that the average ratio between physical and genetic distance is much lower than usual (0.066 Mb/cM vs ∼0.33 Mb/cM in teleost). A way to compensate this bias is to normalise for each LG consensus map coordinates to the average genetic distance obtained from the individual maps used to construct the consensus<a reveal-id="dsy015-B62" data-open="dsy015-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> (see <a reveal-id="dsy015-T1" data-open="dsy015-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Accordingly, the average length and average inter-marker distance of the three single family maps used to construct the consensus was 3753.9 and 0.32, respectively, figures much closer to those found by Wang et al.<a reveal-id="dsy015-B21" data-open="dsy015-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> using RADseq. Interestingly, the sex-determining region-bearing chromosome (LG05<a reveal-id="dsy015-B30" data-open="dsy015-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a>) showed one of the lowest RF ratio (cM/Mb), which agrees with theoretical expectations on the evolution of the sex-determining chromosome.<a reveal-id="dsy015-B32" data-open="dsy015-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a></p><p>The anchorage of the assembled turbot genome to a robust and dense genetic map is essential to transfer marker associations detected by segregation analysis in families to physical positions in the genome for gene mining.<a reveal-id="dsy015-B63" data-open="dsy015-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> In our study, ∼20% of turbot genome assembly<a reveal-id="dsy015-B34" data-open="dsy015-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> was refined, thanks to the mapping information obtained across the 10 most informative families. Furthermore, scaffolds representing 96.5% of turbot genome assembly were anchored to chromosomes, ordered and oriented, and thus, a megascaffold was set up for each LG. This represents a significant improvement from the 80% anchored with the previous medium density turbot map.<a reveal-id="dsy015-B34" data-open="dsy015-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> Also, the available cytogenetic information<a reveal-id="dsy015-B32" data-open="dsy015-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a><sup>,</sup><a reveal-id="dsy015-B40" data-open="dsy015-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a><sup>,</sup><a reveal-id="dsy015-B52" data-open="dsy015-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> was integrated as much as possible in a common framework with genetic and physical maps, and four marker chromosomes of the turbot karyotype were associated with their respective megascaffolds, and the remaining 18 roughly assigned according to their physical length. Using this information, a new nomenclature is here proposed for the 22 chromosomes/LGs of the species, which will facilitate the integration of the different levels of genome organisation for comparative studies within flatfish and teleosts.<a reveal-id="dsy015-B9" data-open="dsy015-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a></p><p>The high correlation observed between genetic and physical maps supports the confidence of marker ordering and physical assembling of turbot genome. However, this general trend was lost at specific chromosome structures such as centromeres and telomeres. Around centromeres extensive dot clouds were visualised, suggestive of poor mapping accuracy very likely related to the decay of RF in the vicinity of heterochromatic blocks.<a reveal-id="dsy015-B64" data-open="dsy015-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a><sup>,</sup><a reveal-id="dsy015-B65" data-open="dsy015-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> This was observed not only for pericentromeric heterochromatin but also for interstitial heterochromatic blocks such as the most conspicuous one detected at LG04 (C5). A remarkable RF difference was observed between males and females around heterochromatic structures, especially the centromeres, where females hardly showed RF decay. This fact is the main responsible for the length difference between male and female maps of turbot, as reported previously in other fish species.<a reveal-id="dsy015-B58" data-open="dsy015-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a><sup>,</sup><a reveal-id="dsy015-B66 dsy015-B67 dsy015-B68" data-open="dsy015-B66 dsy015-B67 dsy015-B68" class="link link-ref link-reveal xref-bibr"><sup>66–68</sup></a> It is tempting to ascribe the different RF between males and females observed in fish<a reveal-id="dsy015-B19" data-open="dsy015-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><sup>,</sup><a reveal-id="dsy015-B66" data-open="dsy015-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> to different crossing-over patterns around heterochromatic blocks. However, other explanations have been proposed for different species, such as the GC content in Asian sea bass<a reveal-id="dsy015-B19" data-open="dsy015-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a>; stronger selection pressure in male gametes during the haploid life stage in zebrafish<a reveal-id="dsy015-B66" data-open="dsy015-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> or in some plants<a reveal-id="dsy015-B69" data-open="dsy015-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a>; and differences in chromatin distribution between sexes as for example in humans <a reveal-id="dsy015-B70" data-open="dsy015-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a>. More information is needed for a comprehensive explanation to these observations, and indeed, NGS technologies can provide suitable data for this task.</p><p>The lack of correspondence between physical and genetic maps observed at telomeres in several families and LGs is likely a consequence of a wrong assembly and/or mapping at chromosome ends. These regions were characterised by the presence of a number of short contigs, suggestive of assembling problems likely due to the presence of repetitive DNA. Similarly, when many markers and a comparatively small offspring number per family are handled, mapping programs throw the least confident markers to LG ends, which lead to the loss of collinearity between physical and genetic maps.<a reveal-id="dsy015-B15" data-open="dsy015-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> Improving genomic assembly at chromosome ends by using long-read sequencing methods and increasing progeny number per family will be needed to solve this problem.</p><p>Genetic maps of bony fish reported to date have been based on a few families,<a reveal-id="dsy015-B19" data-open="dsy015-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><sup>,</sup><a reveal-id="dsy015-B38" data-open="dsy015-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a><sup>,</sup><a reveal-id="dsy015-B58" data-open="dsy015-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> and thus, the presence of inter-individual rearrangements affecting marker order has been scarcely addressed. Previous information showed that, despite a general macrosyntenic 1:1 pattern in Acanthopterygii<a reveal-id="dsy015-B38" data-open="dsy015-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> and specifically in Pleuronectiformes, a huge intrachromosomal reorganisation (mainly inversions) has taken place along flatfish evolution.<a reveal-id="dsy015-B9" data-open="dsy015-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsy015-B34" data-open="dsy015-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> This observation was confirmed in our study, where the genomic comparison between turbot and tongue sole, species pertaining to the distant families Scophthalmidae and Cynoglossidae,<a reveal-id="dsy015-B71" data-open="dsy015-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a><sup>,</sup><a reveal-id="dsy015-B72" data-open="dsy015-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> revealed great intrachromosomal reorganisations. Accordingly, polymorphic inversions could occur within flatfish species, and the comparison between individual maps could shed some light on this issue. Using nine families and a much lower amount of markers, Bouza et al.<a reveal-id="dsy015-B38" data-open="dsy015-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> suggested a general collinearity among individual maps, and only minor discrepancies were detected at 11 LGs involving markers at short distances (&lt; 3 cM). In our study, two medium-sized chromosome tracts (∼7 Mb) at LG06 and LG09 showed features that could be compatible with polymorphic inversions although further work will be necessary to confirm this observation. Combining fluorescence in situ hybridization with BAC probes on these chromosomes or checking linkage between markers within and around the potential inversions with a much higher offspring number could be possible strategies. If confirmed, this information should be considered not only for evolutionary studies but also for marker assisted selection in turbot breeding programs.</p><p>Flatfish represent one of the most extraordinary examples of anatomical specialisation to a particular lifestyle in vertebrates, and a bitter evolutionary controversy took place in the past about their origin and phylogeny.<a reveal-id="dsy015-B73" data-open="dsy015-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a><sup>,</sup><a reveal-id="dsy015-B74" data-open="dsy015-B74" class="link link-ref link-reveal xref-bibr"><sup>74</sup></a> The 1:1 macrosyntenic pattern observed between Acanthopterygii<a reveal-id="dsy015-B38" data-open="dsy015-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> was confirmed when comparing the available flatfish genomes: turbot, tongue sole and Japanese flounder.<a reveal-id="dsy015-B9" data-open="dsy015-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsy015-B21" data-open="dsy015-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a><sup>,</sup><a reveal-id="dsy015-B42" data-open="dsy015-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> The integration of our data with previous information, which includes the recently assembled genome of <em>P. olivaceus</em>,<a reveal-id="dsy015-B43" data-open="dsy015-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> enabled us to confirm and refine the suggested chromosome fusions along flatfish evolution.<a reveal-id="dsy015-B9" data-open="dsy015-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Our results highlight that the fusion of ancestral chromosomes in the evolution of turbot genome resulted in the formation of new chromosomes, whose centromeres should correspond to the ancestral centromere of one of the fusing chromosomes, as reported in the origin of human chromosome 2.<a reveal-id="dsy015-B75" data-open="dsy015-B75" class="link link-ref link-reveal xref-bibr"><sup>75</sup></a> The precise location of centromere-associated markers in the genetic and physical maps strongly supports two main centric fusions in the origin of turbot karyotype (LG02 and LG16) from the ancestral 2<em>n</em> = 48 observed in Japanese flounder.<a reveal-id="dsy015-B57" data-open="dsy015-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a><sup>,</sup><a reveal-id="dsy015-B76" data-open="dsy015-B76" class="link link-ref link-reveal xref-bibr"><sup>76</sup></a> Flatfish show large variation in chromosome number even across closely related taxa and more than half of flatfish species present a chromosome number lower than the ancestral, suggestive of several fusions, with the minimum diploid number (2<em>n</em> = 26) reported in <em>Citharichthys spilopterus.</em><a reveal-id="dsy015-B76" data-open="dsy015-B76" class="link link-ref link-reveal xref-bibr"><sup>76</sup></a> It would be worth investigating in more detail if the high number of chromosome fusions occurring in the families with lower diploid numbers (e.g. Soleidae, Bothidae and Achiridae and some Paralichthydae) show a preferential fusion pattern, similar to what recently found in notothenioids.<a reveal-id="dsy015-B77" data-open="dsy015-B77" class="link link-ref link-reveal xref-bibr"><sup>77</sup></a> In addition to chromosome fusions, other minor reorganizations between non-orthologous chromosomes of turbot, Japanese flounder and half-smooth tongue sole were identified in our study. These chromosome rearrangements are thought to be important in karyotypic evolution and species differentiation,<a reveal-id="dsy015-B77 dsy015-B78 dsy015-B79" data-open="dsy015-B77 dsy015-B78 dsy015-B79" class="link link-ref link-reveal xref-bibr"><sup>77–79</sup></a> and thus, further comparative mapping studies are encouraged to understand the reasons behind these rearrangements in the evolution of flatfish.</p><p>The turbot HD consensus genetic map here constructed using a set of 18,124 SNPs was a suitable reference to anchor the turbot genome and to integrate all previous genomic information. The comparison of the male and female consensus maps enabled to identify sharp RF differences around centromeres that explained their length differences. The 62 individual genetic maps constructed allowed to detect suggestive polymorphic rearrangements in the species putatively related to coadapted gene blocks. Finally, inter- and intra-chromosomal reorganisations in Pleuronectiformes were identified comparing the chromosome sequences available for three flatfish species taking the turbot map as reference. The consistent HD genetic map reported in turbot represents an invaluable genomic tool for further genome-wide association and evolutionary genomic studies. </p>                            <h2 scrollto-destination=120122130 id="120122130" class="section-title" >Acknowledgements</h2>
<p>This study has been supported by the FISHBOOST project (ref. 613611) from the European Community's Seventh Framework Programme (FP7/2007-2013) and by Consellería de Cultura, Educación e Ordenación Universitaria, Xunta de Galicia local government (ref. GRC2014/010). Computational support for bioinformatic analysis were provided by Centro de Supercomputaciòn de Galicia (CESGA).</p>                            <h2 scrollto-destination=120122132 id="120122132" class="section-title" >Accession numbers</h2>
<p>CP026243-CP026264.</p>                            <h2 scrollto-destination=120122134 id="120122134" class="section-title" >Conflict of interest</h2>
<p>None declared.</p>                            <h2 scrollto-destination=120122136 id="120122136" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsy015-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mardis</div> <div class="given-names">E. R.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">The impact of next-generation sequencing technology on genetics</div>, <div class="source">Trends Genet</div> ., <div class="volume">24</div>, <div class="fpage">133</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20impact%20of%20next-generation%20sequencing%20technology%20on%20genetics&amp;author=E.%20R.%20Mardis&amp;publication_year=2008&amp;journal=Trends%20Genet&amp;volume=24&amp;pages=133-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2007.12.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2007.12.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2007.12.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18262675" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nie</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Yan</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Huo</div> <div class="given-names">Z.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Construction of a high-density genetic map and quantitative trait locus mapping in the Manila clam <em>Ruditapes philippinarum</em></div>, <div class="source">Sci. Rep</div> ., <div class="volume">7</div>, <div class="fpage">229</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20high-density%20genetic%20map%20and%20quantitative%20trait%20locus%20mapping%20in%20the%20Manila%20clam%20Ruditapes%20philippinarum&amp;author=H.%20Nie&amp;author=X.%20Yan&amp;author=Z.%20Huo&amp;publication_year=2017&amp;journal=Sci.%20Rep&amp;volume=7&amp;pages=229" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/s41598-017-00246-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fs41598-017-00246-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fs41598-017-00246-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28331182" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nunes</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">de</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Pértille</div> <div class="given-names">F.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Large-scale SNP discovery and construction of a high-density genetic map of <em>Colossoma macropomum</em> through genotyping-by-sequencing</div>, <div class="source">Sci. Rep</div> ., <div class="volume">7</div>, <div class="fpage">46112</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20SNP%20discovery%20and%20construction%20of%20a%20high-density%20genetic%20map%20of%20Colossoma%20macropomum%20through%20genotyping-by-sequencing&amp;author=J.%20Nunes&amp;author=R.%20de&amp;author=S.%20Liu&amp;author=F.%20P%C3%A9rtille&amp;publication_year=2017&amp;journal=Sci.%20Rep&amp;volume=7&amp;pages=46112" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep46112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep46112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep46112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28387238" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Koboldt</div> <div class="given-names">D. C.</div></div>, <div class="name string-name"><div class="surname">Steinberg</div> <div class="given-names">K. M.</div></div>, <div class="name string-name"><div class="surname">Larson</div> <div class="given-names">D. E.</div></div>, <div class="name string-name"><div class="surname">Wilson</div> <div class="given-names">R. K.</div></div>, <div class="name string-name"><div class="surname">Mardis</div> <div class="given-names">E. R.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">The next-generation sequencing revolution and its impact on genomics</div>, <div class="source">Cell</div> , <div class="volume">155</div>, <div class="fpage">27</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20next-generation%20sequencing%20revolution%20and%20its%20impact%20on%20genomics&amp;author=D.%20C.%20Koboldt&amp;author=K.%20M.%20Steinberg&amp;author=D.%20E.%20Larson&amp;author=R.%20K.%20Wilson&amp;author=E.%20R.%20Mardis&amp;publication_year=2013&amp;journal=Cell&amp;volume=155&amp;pages=27-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2013.09.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2013.09.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2013.09.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24074859" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">van Dijk</div> <div class="given-names">E. L.</div></div>, <div class="name string-name"><div class="surname">Auger</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Jaszczyszyn</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Thermes</div> <div class="given-names">C.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Ten years of next-generation sequencing technology</div>, <div class="source">Trends Genet</div> ., <div class="volume">30</div>, <div class="fpage">418</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ten%20years%20of%20next-generation%20sequencing%20technology&amp;author=E.%20L.%20van%20Dijk&amp;author=H.%20Auger&amp;author=Y.%20Jaszczyszyn&amp;author=C.%20Thermes&amp;publication_year=2014&amp;journal=Trends%20Genet&amp;volume=30&amp;pages=418-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2014.07.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2014.07.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2014.07.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25108476" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Moll</div> <div class="given-names">K. M.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Ramaraj</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Strategies for optimizing BioNano and Dovetail explored through a second reference quality assembly for the legume model, <em>Medicago truncatula</em></div>, <div class="source">BMC Genomics</div> , <div class="volume">18</div>, <div class="fpage">578</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strategies%20for%20optimizing%20BioNano%20and%20Dovetail%20explored%20through%20a%20second%20reference%20quality%20assembly%20for%20the%20legume%20model%2C%20Medicago%20truncatula&amp;author=K.%20M.%20Moll&amp;author=P.%20Zhou&amp;author=T.%20Ramaraj&amp;publication_year=2017&amp;journal=BMC%20Genomics&amp;volume=18&amp;pages=578" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-017-3971-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-017-3971-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-017-3971-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28778149" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Roesti</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Moser</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Berner</div> <div class="given-names">D.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Recombination in the threespine stickleback genome—patterns and consequences</div>, <div class="source">Mol. Ecol</div> ., <div class="volume">22</div>, <div class="fpage">3014</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recombination%20in%20the%20threespine%20stickleback%20genome%E2%80%94patterns%20and%20consequences&amp;author=M.%20Roesti&amp;author=D.%20Moser&amp;author=D.%20Berner&amp;publication_year=2013&amp;journal=Mol.%20Ecol&amp;volume=22&amp;pages=3014-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.12322" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.12322" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.12322"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23601112" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fierst</div> <div class="given-names">J. L.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Using linkage maps to correct and scaffold de novo genome assemblies: methods, challenges, and computational tools</div>, <div class="source">Front. Genet</div> ., <div class="volume">6</div>, <div class="fpage">220</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20linkage%20maps%20to%20correct%20and%20scaffold%20de%20novo%20genome%20assemblies%3A%20methods%2C%20challenges%2C%20and%20computational%20tools&amp;author=J.%20L.%20Fierst&amp;publication_year=2015&amp;journal=Front.%20Genet&amp;volume=6&amp;pages=220" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fgene.2015.00220" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffgene.2015.00220" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffgene.2015.00220"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26150829" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Robledo</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Palaiokostas</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Bargelloni</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Martínez</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Houston</div> <div class="given-names">R.</div></div></div> <div class="year">2017</div>, Applications of genotyping by sequencing in aquaculture breeding and genetics. <em>Rev. Aquac.</em>, doi: 10.1111/raq.12193.</div></div></div></div><div content-id="dsy015-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Meyer</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Van de Peer</div> <div class="given-names">Y.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">From 2R to 3R: evidence for a fish-specific genome duplication (FSGD)</div>, <div class="source">Bioessays</div>  , <div class="volume">27</div>, <div class="fpage">937</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=From%202R%20to%203R%3A%20evidence%20for%20a%20fish-specific%20genome%20duplication%20(FSGD)&amp;author=A.%20Meyer&amp;author=Y.%20Van%20de%20Peer&amp;publication_year=2005&amp;journal=Bioessays&amp;volume=27&amp;pages=937-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1521-1878" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1521-1878" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1521-1878"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16108068" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Miller</div> <div class="given-names">M. R.</div></div>, <div class="name string-name"><div class="surname">Dunham</div> <div class="given-names">J. P.</div></div>, <div class="name string-name"><div class="surname">Amores</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Cresko</div> <div class="given-names">W. A.</div></div>, <div class="name string-name"><div class="surname">Johnson</div> <div class="given-names">E. A.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Rapid and cost-effective polymorphism identification and genotyping using restriction site associated DNA (RAD) markers</div>, <div class="source">Genome Res</div> ., <div class="volume">17</div>, <div class="fpage">240</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20and%20cost-effective%20polymorphism%20identification%20and%20genotyping%20using%20restriction%20site%20associated%20DNA%20(RAD)%20markers&amp;author=M.%20R.%20Miller&amp;author=J.%20P.%20Dunham&amp;author=A.%20Amores&amp;author=W.%20A.%20Cresko&amp;author=E.%20A.%20Johnson&amp;publication_year=2007&amp;journal=Genome%20Res&amp;volume=17&amp;pages=240-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.5681207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.5681207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.5681207"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17189378" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Baird</div> <div class="given-names">N. A.</div></div>, <div class="name string-name"><div class="surname">Etter</div> <div class="given-names">P. D.</div></div>, <div class="name string-name"><div class="surname">Atwood</div> <div class="given-names">T. S.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">Rapid SNP discovery and genetic mapping using sequenced RAD markers</div>, <div class="source">PLoS One</div> , <div class="volume">3</div>, <div class="fpage">e3376</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20SNP%20discovery%20and%20genetic%20mapping%20using%20sequenced%20RAD%20markers&amp;author=N.%20A.%20Baird&amp;author=P.%20D.%20Etter&amp;author=T.%20S.%20Atwood&amp;publication_year=2008&amp;journal=PLoS%20One&amp;volume=3&amp;pages=e3376" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0003376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0003376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0003376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18852878" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Davey</div> <div class="given-names">J. W.</div></div>, <div class="name string-name"><div class="surname">Hohenlohe</div> <div class="given-names">P. A.</div></div>, <div class="name string-name"><div class="surname">Etter</div> <div class="given-names">P. D.</div></div>, <div class="name string-name"><div class="surname">Boone</div> <div class="given-names">J. Q.</div></div>, <div class="name string-name"><div class="surname">Catchen</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Blaxter</div> <div class="given-names">M. L.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Genome-wide genetic marker discovery and genotyping using next-generation sequencing</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">12</div>, <div class="fpage">499</div>–<div class="lpage">510</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20genetic%20marker%20discovery%20and%20genotyping%20using%20next-generation%20sequencing&amp;author=J.%20W.%20Davey&amp;author=P.%20A.%20Hohenlohe&amp;author=P.%20D.%20Etter&amp;author=J.%20Q.%20Boone&amp;author=J.%20M.%20Catchen&amp;author=M.%20L.%20Blaxter&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet&amp;volume=12&amp;pages=499-510" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21681211" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">O'Quin</div> <div class="given-names">K. E.</div></div>, <div class="name string-name"><div class="surname">Yoshizawa</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Doshi</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Jeffery</div> <div class="given-names">W. R.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Quantitative genetic analysis of retinal degeneration in the blind cavefish <em>Astyanax mexicanus</em></div>, <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e57281</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20genetic%20analysis%20of%20retinal%20degeneration%20in%20the%20blind%20cavefish%20Astyanax%20mexicanus&amp;author=K.%20E.%20O'Quin&amp;author=M.%20Yoshizawa&amp;author=P.%20Doshi&amp;author=W.%20R.%20Jeffery&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e57281" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0057281" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0057281" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0057281"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23437360" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gonen</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Lowe</div> <div class="given-names">N. R.</div></div>, <div class="name string-name"><div class="surname">Cezard</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Gharbi</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Bishop</div> <div class="given-names">S. C.</div></div>, <div class="name string-name"><div class="surname">Houston</div> <div class="given-names">R. D.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Linkage maps of the Atlantic salmon (<em>Salmo salar</em>) genome derived from RAD sequencing</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">166</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linkage%20maps%20of%20the%20Atlantic%20salmon%20(Salmo%20salar)%20genome%20derived%20from%20RAD%20sequencing&amp;author=S.%20Gonen&amp;author=N.%20R.%20Lowe&amp;author=T.%20Cezard&amp;author=K.%20Gharbi&amp;author=S.%20C.%20Bishop&amp;author=R.%20D.%20Houston&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=166" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-166" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-166" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-166"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24571138" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Carlson</div> <div class="given-names">B. M.</div></div>, <div class="name string-name"><div class="surname">Onusko</div> <div class="given-names">S. W.</div></div>, <div class="name string-name"><div class="surname">Gross</div> <div class="given-names">J. B.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">A high-density linkage map for <em>Astyanax mexicanus</em> using genotyping-by-sequencing technology</div>, <div class="source">G3 Genes Genomes Genet</div> ., <div class="volume">5</div>, <div class="fpage">241</div>–<div class="lpage">51</div>.</div></div></div></div><div content-id="dsy015-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fu</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Tong</div> <div class="given-names">J.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">A high-density genetic map and growth related QTL mapping in bighead carp (<em>Hypophthalmichthys nobilis</em>)</div>, <div class="source">Sci. Rep</div> ., <div class="volume">6</div>, <div class="fpage">28679</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-density%20genetic%20map%20and%20growth%20related%20QTL%20mapping%20in%20bighead%20carp%20(Hypophthalmichthys%20nobilis)&amp;author=B.%20Fu&amp;author=H.%20Liu&amp;author=X.%20Yu&amp;author=J.%20Tong&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=6&amp;pages=28679" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep28679" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep28679" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep28679"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27345016" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Wong</div> <div class="given-names">S.-M.</div></div>, <div class="name string-name"><div class="surname">Yue</div> <div class="given-names">G. H.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Fine mapping QTL for resistance to VNN disease using a high-density linkage map in Asian seabass</div>, <div class="source">Sci. Rep</div> ., <div class="volume">6</div>, <div class="fpage">32122</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fine%20mapping%20QTL%20for%20resistance%20to%20VNN%20disease%20using%20a%20high-density%20linkage%20map%20in%20Asian%20seabass&amp;author=P.%20Liu&amp;author=L.%20Wang&amp;author=S.-M.%20Wong&amp;author=G.%20H.%20Yue&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=6&amp;pages=32122" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep32122" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep32122" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep32122"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27555039" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Bai</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Construction of high-resolution recombination maps in Asian seabass</div>, <div class="source">BMC Genomics</div> , <div class="volume">18</div>, <div class="fpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20high-resolution%20recombination%20maps%20in%20Asian%20seabass&amp;author=L.%20Wang&amp;author=B.%20Bai&amp;author=P.%20Liu&amp;publication_year=2017&amp;journal=BMC%20Genomics&amp;volume=18&amp;pages=63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-016-3462-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-016-3462-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-016-3462-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28068919" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Song</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Pang</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Niu</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Construction of high-density genetic linkage maps and mapping of growth-related quantitative trail loci in the Japanese flounder (<em>Paralichthys olivaceus</em>)</div>, <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e50404</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20high-density%20genetic%20linkage%20maps%20and%20mapping%20of%20growth-related%20quantitative%20trail%20loci%20in%20the%20Japanese%20flounder%20(Paralichthys%20olivaceus)&amp;author=W.%20Song&amp;author=R.%20Pang&amp;author=Y.%20Niu&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e50404" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0050404" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0050404" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0050404"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23209734" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Ma</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Guan</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Kong</div> <div class="given-names">J.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">High-density genetic linkage mapping in turbot (<em>Scophthalmus maximus</em> L.) based on SNP markers and major sex-and growth-related regions detection</div>, <div class="source">PLoS One</div>  , <div class="volume">10</div>, <div class="fpage">e0120410</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-density%20genetic%20linkage%20mapping%20in%20turbot%20(Scophthalmus%20maximus%20L.)%20based%20on%20SNP%20markers%20and%20major%20sex-and%20growth-related%20regions%20detection&amp;author=W.%20Wang&amp;author=Y.%20Hu&amp;author=Y.%20Ma&amp;author=L.%20Xu&amp;author=J.%20Guan&amp;author=J.%20Kong&amp;publication_year=2015&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0120410" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0120410" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0120410" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0120410"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25775256" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sun</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Niu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Ye</div> <div class="given-names">X.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Construction of a high-density linkage map and mapping of sex determination and growth-related loci in the mandarin fish (<em>Siniperca chuatsi</em>)</div>, <div class="source">BMC Genomics</div> , <div class="volume">18</div>, <div class="fpage">446</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20high-density%20linkage%20map%20and%20mapping%20of%20sex%20determination%20and%20growth-related%20loci%20in%20the%20mandarin%20fish%20(Siniperca%20chuatsi)&amp;author=C.%20Sun&amp;author=Y.%20Niu&amp;author=X.%20Ye&amp;publication_year=2017&amp;journal=BMC%20Genomics&amp;volume=18&amp;pages=446" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-017-3830-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-017-3830-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-017-3830-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28587594" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tsai</div> <div class="given-names">H. Y.</div></div>, <div class="name string-name"><div class="surname">Robledo</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Lowe</div> <div class="given-names">N. R.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Construction and annotation of a high density SNP linkage map of the Atlantic salmon (<em>Salmo salar</em>) genome</div>, <div class="source">G3 Genes Genomes Genet</div> ., <div class="volume">6</div>, <div class="fpage">2173</div>–<div class="lpage">9</div>.</div></div></div></div><div content-id="dsy015-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Palaiokostas</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Ferraresso</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Franch</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Houston</div> <div class="given-names">R. D.</div></div>, <div class="name string-name"><div class="surname">Bargelloni</div> <div class="given-names">L.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Genomic prediction of resistance to pasteurellosis in gilthead sea bream (<em>Sparus aurata</em>) using 2b-RAD sequencing</div>, <div class="source">G3 Genes Genomes Genet</div> ., <div class="volume">6</div>, <div class="fpage">3693</div>–<div class="lpage">700</div>.</div></div></div></div><div content-id="dsy015-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">van der Beek</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">van Arendonk</div> <div class="given-names">J. A.</div></div></div> <div class="year">1993</div>, 
<div class="article-title">Criteria to optimize designs for detection and estimation of linkage between marker loci from segregating populations containing several families</div>, <div class="source">Theoret. Appl. Genetics</div> , <div class="volume">86-86</div>, <div class="fpage">269</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Criteria%20to%20optimize%20designs%20for%20detection%20and%20estimation%20of%20linkage%20between%20marker%20loci%20from%20segregating%20populations%20containing%20several%20families&amp;author=S.%20van%20der%20Beek&amp;author=J.%20A.%20van%20Arendonk&amp;publication_year=1993&amp;journal=Theoret.%20Appl.%20Genetics&amp;volume=86-86&amp;pages=269-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00222088" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00222088" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00222088"> </span></div> </div></div></div></div></div><div content-id="dsy015-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Qin</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Z.</div></div></div> <div class="year">2017</div>, Bioinformatics considerations and approaches for high-density linkage mapping in aquaculture, In: <div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Z.</div></div></div>, (ed.) <div class="source">Bioinformatics in Aquaculture: Principles and Methods</div> , 
<div class="publisher-name">John Wiley &amp; Sons, Ltd</div>, 
<div class="publisher-loc">Chichester, UK</div>, pp. <div class="fpage">356</div>–<div class="lpage">79</div>.</div></div></div></div><div content-id="dsy015-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sodeland</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Jorde</div> <div class="given-names">P. E.</div></div>, <div class="name string-name"><div class="surname">Lien</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2016</div>, ‘Islands of divergence’ 
<div class="article-title">in the Atlantic Cod genome represent polymorphic chromosomal rearrangements</div>, <div class="source">Genome Biol. Evol</div> ., <div class="volume">8</div>, <div class="fpage">1012</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=in%20the%20Atlantic%20Cod%20genome%20represent%20polymorphic%20chromosomal%20rearrangements&amp;author=M.%20Sodeland&amp;author=P.%20E.%20Jorde&amp;author=S.%20Lien&amp;publication_year=2016&amp;journal=Genome%20Biol.%20Evol&amp;volume=8&amp;pages=1012-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evw057" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevw057" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevw057"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26983822" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jones</div> <div class="given-names">F. C.</div></div>, <div class="name string-name"><div class="surname">Grabherr</div> <div class="given-names">M. G.</div></div>, <div class="name string-name"><div class="surname">Chan</div> <div class="given-names">Y. F.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">The genomic basis of adaptive evolution in threespine sticklebacks</div>, <div class="source">Nature</div> , <div class="volume">484</div>, <div class="fpage">55</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genomic%20basis%20of%20adaptive%20evolution%20in%20threespine%20sticklebacks&amp;author=F.%20C.%20Jones&amp;author=M.%20G.%20Grabherr&amp;author=Y.%20F.%20Chan&amp;publication_year=2012&amp;journal=Nature&amp;volume=484&amp;pages=55-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10944" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10944" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10944"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22481358" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bouza</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Vandamme</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hermida</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Cabaleiro</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Volkaert</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Mart´nez</div> <div class="given-names">P.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Turbot. AquaTrace Species Leaflet</div>. <a class="link link-uri" href="https://aquatrace.eu/leaflets/turbot" target="_blank">https://aquatrace.eu/leaflets/turbot</a>.</div></div></div></div><div content-id="dsy015-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Martínez</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Bouza</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Hermida</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">Identification of the major sex-determining region of turbot (<em>Scophthalmus maximus</em>)</div>, <div class="source">Genetics</div> , <div class="volume">183</div>, <div class="fpage">1443</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20the%20major%20sex-determining%20region%20of%20turbot%20(Scophthalmus%20maximus)&amp;author=P.%20Mart%C3%ADnez&amp;author=C.%20Bouza&amp;author=M.%20Hermida&amp;publication_year=2009&amp;journal=Genetics&amp;volume=183&amp;pages=1443-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.109.107979" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.109.107979" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.109.107979"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19786621" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rodrìguez-Ramilo</div> <div class="given-names">S. T.</div></div>, <div class="name string-name"><div class="surname">Toro</div> <div class="given-names">M. A.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Fernandez</div> <div class="given-names">J.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Improving the inference of population genetic structure in the presence of related individuals</div>, <div class="source">Genet. Res</div> ., <div class="volume">96</div>, <div class="fpage">e003</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improving%20the%20inference%20of%20population%20genetic%20structure%20in%20the%20presence%20of%20related%20individuals&amp;author=S.%20T.%20Rodr%C3%ACguez-Ramilo&amp;author=M.%20A.%20Toro&amp;author=J.%20Wang&amp;author=J.%20Fernandez&amp;publication_year=2014&amp;journal=Genet.%20Res&amp;volume=96&amp;pages=e003" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S0016672314000068" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS0016672314000068" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS0016672314000068"> </span></div> </div></div></div></div></div><div content-id="dsy015-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Taboada</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Hermida</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pardo</div> <div class="given-names">B. G.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Fine mapping and evolution of the major sex determining region in turbot (<em>Scophthalmus maximus</em>)</div>, <div class="source">G3 Genes Genomes Genet</div> ., <div class="volume">4</div>, <div class="fpage">1871</div>–<div class="lpage">80</div>.</div></div></div></div><div content-id="dsy015-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Robledo</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Ribas</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Cal</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Gene expression analysis at the onset of sex differentiation in turbot (<em>Scophthalmus maximus</em>)</div>, <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">973</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20expression%20analysis%20at%20the%20onset%20of%20sex%20differentiation%20in%20turbot%20(Scophthalmus%20maximus)&amp;author=D.%20Robledo&amp;author=L.%20Ribas&amp;author=R.%20Cal&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=973" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-2142-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-2142-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-2142-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26581195" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Figueras</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Robledo</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Corvelo</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Whole genome sequencing of turbot (<em>Scophthalmus maximus</em>; Pleuronectiformes): a fish adapted to demersal life</div>, <div class="source">DNA Res</div> , <div class="volume">23</div>, <div class="fpage">181</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole%20genome%20sequencing%20of%20turbot%20(Scophthalmus%20maximus%3B%20Pleuronectiformes)%3A%20a%20fish%20adapted%20to%20demersal%20life&amp;author=A.%20Figueras&amp;author=D.%20Robledo&amp;author=A.%20Corvelo&amp;publication_year=2016&amp;journal=DNA%20Res&amp;volume=23&amp;pages=181-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsw007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsw007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsw007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26951068" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Robledo</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Fernández</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Hermida</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Integrative transcriptome, genome and quantitative trait loci resources identify single nucleotide polymorphisms in candidate genes for growth traits in turbot</div>, <div class="source">IJMS</div> , <div class="volume">17</div>, <div class="fpage">243</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrative%20transcriptome%2C%20genome%20and%20quantitative%20trait%20loci%20resources%20identify%20single%20nucleotide%20polymorphisms%20in%20candidate%20genes%20for%20growth%20traits%20in%20turbot&amp;author=D.%20Robledo&amp;author=C.%20Fern%C3%A1ndez&amp;author=M.%20Hermida&amp;publication_year=2016&amp;journal=IJMS&amp;volume=17&amp;pages=243" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3390/ijms17020243" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3390%2Fijms17020243" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3390%2Fijms17020243"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26901189" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bouza</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Hermida</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pardo</div> <div class="given-names">B. G.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">A microsatellite genetic map of the turbot (<em>Scophthalmus maximus</em>)</div>, <div class="source">Genetics</div> , <div class="volume">177</div>, <div class="fpage">2457</div>–<div class="lpage">67</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20microsatellite%20genetic%20map%20of%20the%20turbot%20(Scophthalmus%20maximus)&amp;author=C.%20Bouza&amp;author=M.%20Hermida&amp;author=B.%20G.%20Pardo&amp;publication_year=2007&amp;journal=Genetics&amp;volume=177&amp;pages=2457-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.107.075416" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.107.075416" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.107.075416"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18073440" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ruan</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Kong</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">X.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Genetic linkage mapping of turbot (<em>Scophthalmus maximus</em> L.) using microsatellite markers and its application in QTL analysis</div>, <div class="source">Aquaculture</div> , <div class="volume">308</div>, <div class="fpage">89</div>–<div class="lpage">100</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20linkage%20mapping%20of%20turbot%20(Scophthalmus%20maximus%20L.)%20using%20microsatellite%20markers%20and%20its%20application%20in%20QTL%20analysis&amp;author=X.%20Ruan&amp;author=W.%20Wang&amp;author=J.%20Kong&amp;author=F.%20Yu&amp;author=X.%20Huang&amp;publication_year=2010&amp;journal=Aquaculture&amp;volume=308&amp;pages=89-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.aquaculture.2010.08.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.aquaculture.2010.08.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.aquaculture.2010.08.010"> </span></div> </div></div></div></div></div><div content-id="dsy015-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bouza</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Hermida</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pardo</div> <div class="given-names">B. G.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">An Expressed Sequence Tag (EST)-enriched genetic map of turbot (<em>Scophthalmus maximus</em>): a useful framework for comparative genomics across model and farmed teleosts</div>, <div class="source">BMC Genet</div> ., <div class="volume">13</div>, <div class="fpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20Expressed%20Sequence%20Tag%20(EST)-enriched%20genetic%20map%20of%20turbot%20(Scophthalmus%20maximus)%3A%20a%20useful%20framework%20for%20comparative%20genomics%20across%20model%20and%20farmed%20teleosts&amp;author=C.%20Bouza&amp;author=M.%20Hermida&amp;author=B.%20G.%20Pardo&amp;publication_year=2012&amp;journal=BMC%20Genet&amp;volume=13&amp;pages=54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2156-13-54" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2156-13-54" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2156-13-54"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22747677" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hermida</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Bouza</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Fernández</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Compilation of mapping resources in turbot (<em>Scophthalmus maximus</em>): a new integrated consensus genetic map</div>, <div class="source">Aquaculture</div> , <div class="volume">414–415</div>, <div class="fpage">19</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Compilation%20of%20mapping%20resources%20in%20turbot%20(Scophthalmus%20maximus)%3A%20a%20new%20integrated%20consensus%20genetic%20map&amp;author=M.%20Hermida&amp;author=C.%20Bouza&amp;author=C.%20Fern%C3%A1ndez&amp;publication_year=2013&amp;journal=Aquaculture&amp;volume=414–415&amp;pages=19-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.aquaculture.2013.07.040" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.aquaculture.2013.07.040" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.aquaculture.2013.07.040"> </span></div> </div></div></div></div></div><div content-id="dsy015-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Martínez</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Hermida</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pardo</div> <div class="given-names">B. G.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">Centromere-linkage in the turbot (<em>Scophthalmus maximus</em>) through half-tetrad analysis in diploid meiogynogenetics</div>, <div class="source">Aquaculture</div> , <div class="volume">280</div>, <div class="fpage">81</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Centromere-linkage%20in%20the%20turbot%20(Scophthalmus%20maximus)%20through%20half-tetrad%20analysis%20in%20diploid%20meiogynogenetics&amp;author=P.%20Mart%C3%ADnez&amp;author=M.%20Hermida&amp;author=B.%20G.%20Pardo&amp;publication_year=2008&amp;journal=Aquaculture&amp;volume=280&amp;pages=81-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.aquaculture.2008.05.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.aquaculture.2008.05.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.aquaculture.2008.05.011"> </span></div> </div></div></div></div></div><div content-id="dsy015-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Saitoh</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Hayashizaki</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Yokoyama</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Asahida</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Toyohara</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Yamashita</div> <div class="given-names">Y.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Complete nucleotide sequence of Japanese flounder (<em>Paralichthys olivaceus</em>) mitochondrial genome: structural properties and cue for resolving teleostean relationship</div>, <div class="source">J. Hered</div> ., <div class="volume">91</div>, <div class="fpage">271</div>–<div class="lpage">8</div>.,<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20nucleotide%20sequence%20of%20Japanese%20flounder%20(Paralichthys%20olivaceus)%20mitochondrial%20genome%3A%20structural%20properties%20and%20cue%20for%20resolving%20teleostean%20relationship&amp;author=K.%20Saitoh&amp;author=K.%20Hayashizaki&amp;author=Y.%20Yokoyama&amp;author=T.%20Asahida&amp;author=H.%20Toyohara&amp;author=Y.%20Yamashita&amp;publication_year=2000&amp;journal=J.%20Hered&amp;volume=91&amp;pages=271-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/91.4.271" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2F91.4.271" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2F91.4.271"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10912672" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Shao</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Whole-genome sequence of a flatfish provides insights into ZW sex chromosome evolution and adaptation to a benthic lifestyle</div>, <div class="source">Nat. Genet</div> ., <div class="volume">46</div>, <div class="fpage">253</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20sequence%20of%20a%20flatfish%20provides%20insights%20into%20ZW%20sex%20chromosome%20evolution%20and%20adaptation%20to%20a%20benthic%20lifestyle&amp;author=S.%20Chen&amp;author=G.%20Zhang&amp;author=C.%20Shao&amp;publication_year=2014&amp;journal=Nat.%20Genet&amp;volume=46&amp;pages=253-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2890" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2890" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2890"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24487278" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shao</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Bao</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Xie</div> <div class="given-names">Z.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">The genome and transcriptome of Japanese flounder provide insights into flatfish asymmetry</div>, <div class="source">Nat. Genet</div> ., <div class="volume">49</div>, <div class="fpage">119</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20and%20transcriptome%20of%20Japanese%20flounder%20provide%20insights%20into%20flatfish%20asymmetry&amp;author=C.%20Shao&amp;author=B.%20Bao&amp;author=Z.%20Xie&amp;publication_year=2017&amp;journal=Nat.%20Genet&amp;volume=49&amp;pages=119-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.3732" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.3732" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.3732"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27918537" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cruz</div> <div class="given-names">V. P.</div></div>, <div class="name string-name"><div class="surname">Vera</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pardo</div> <div class="given-names">B. G.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Identification and validation of single nucleotide polymorphisms as tools to detect hybridization and population structure in freshwater stingrays</div>, <div class="source">Mol. Ecol. Resour</div> ., <div class="volume">17</div>, <div class="fpage">550</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20validation%20of%20single%20nucleotide%20polymorphisms%20as%20tools%20to%20detect%20hybridization%20and%20population%20structure%20in%20freshwater%20stingrays&amp;author=V.%20P.%20Cruz&amp;author=M.%20Vera&amp;author=B.%20G.%20Pardo&amp;publication_year=2017&amp;journal=Mol.%20Ecol.%20Resour&amp;volume=17&amp;pages=550-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/men.2017.17.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmen.2017.17.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmen.2017.17.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27470580" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Meyer</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">McKay</div> <div class="given-names">J. K.</div></div>, <div class="name string-name"><div class="surname">Matz</div> <div class="given-names">M. V.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">2b-RAD: a simple and flexible method for genome-wide genotyping</div>, <div class="source">Nat. Methods</div> , <div class="volume">9</div>, <div class="fpage">808</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=2b-RAD%3A%20a%20simple%20and%20flexible%20method%20for%20genome-wide%20genotyping&amp;author=S.%20Wang&amp;author=E.%20Meyer&amp;author=J.%20K.%20McKay&amp;author=M.%20V.%20Matz&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=808-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.2023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.2023"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22609625" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Catchen</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Amores</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Hohenlohe</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Cresko</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Postlethwait</div> <div class="given-names">J. H.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Stacks: building and genotyping loci de novo from short-read sequences</div>, <div class="source">G3 (Bethesda) )</div> , <div class="volume">1</div>, <div class="fpage">171</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stacks%3A%20building%20and%20genotyping%20loci%20de%20novo%20from%20short-read%20sequences&amp;author=J.%20M.%20Catchen&amp;author=A.%20Amores&amp;author=P.%20Hohenlohe&amp;author=W.%20Cresko&amp;author=J.%20H.%20Postlethwait&amp;publication_year=2011&amp;journal=G3%20(Bethesda)%20)&amp;volume=1&amp;pages=171-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.111.000240" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.111.000240" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.111.000240"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22384329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Catchen</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Hohenlohe</div> <div class="given-names">P. A.</div></div>, <div class="name string-name"><div class="surname">Bassham</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Amores</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Cresko</div> <div class="given-names">W. A.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Stacks: an analysis tool set for population genomics</div>, <div class="source">Mol. Ecol</div> ., <div class="volume">22</div>, <div class="fpage">3124</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stacks%3A%20an%20analysis%20tool%20set%20for%20population%20genomics&amp;author=J.%20Catchen&amp;author=P.%20A.%20Hohenlohe&amp;author=S.%20Bassham&amp;author=A.%20Amores&amp;author=W.%20A.%20Cresko&amp;publication_year=2013&amp;journal=Mol.%20Ecol&amp;volume=22&amp;pages=3124-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.12354" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.12354" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.12354"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23701397" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Langmead</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pop</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</div>, <div class="source">Genome Biol</div> ., <div class="volume">10</div>, <div class="fpage">R25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultrafast%20and%20memory-efficient%20alignment%20of%20short%20DNA%20sequences%20to%20the%20human%20genome&amp;author=B.%20Langmead&amp;author=C.%20Trapnell&amp;author=M.%20Pop&amp;author=S.%20L.%20Salzberg&amp;publication_year=2009&amp;journal=Genome%20Biol&amp;volume=10&amp;pages=R25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-3-r25" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-3-r25" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-3-r25"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19261174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Van Ooijen</div> <div class="given-names">J. W.</div></div></div> <div class="year">2006</div>, JoinMap<sup>®</sup>4. <em>Software for the calculation of genetic linkage maps in experimental populations, vol. 33.</em> Kyazma BV: Wageningen.</div></div></div></div><div content-id="dsy015-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Close</div> <div class="given-names">T. J.</div></div>, <div class="name string-name"><div class="surname">Lonardi</div> <div class="given-names">S.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Accurate construction of consensus genetic maps via integer linear programming</div>, <div class="source">IEEE/ACM Trans. Comput. Biol. Bioinform</div> ., <div class="volume">8</div>, <div class="fpage">381</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accurate%20construction%20of%20consensus%20genetic%20maps%20via%20integer%20linear%20programming&amp;author=Y.%20Wu&amp;author=T.%20J.%20Close&amp;author=S.%20Lonardi&amp;publication_year=2011&amp;journal=IEEE%2FACM%20Trans.%20Comput.%20Biol.%20Bioinform&amp;volume=8&amp;pages=381-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1109/TCBB.2010.35" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1109%2FTCBB.2010.35" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1109%2FTCBB.2010.35"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20479505" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Voorrips</div> <div class="given-names">R. E.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">MapChart: software for the graphical presentation of linkage maps and QTLs</div>, <div class="source">J. Hered</div> ., <div class="volume">93</div>, <div class="fpage">77</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MapChart%3A%20software%20for%20the%20graphical%20presentation%20of%20linkage%20maps%20and%20QTLs&amp;author=R.%20E.%20Voorrips&amp;publication_year=2002&amp;journal=J.%20Hered&amp;volume=93&amp;pages=77-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/93.1.77" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2F93.1.77" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2F93.1.77"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12011185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bouza</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Sanchez</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Martínez</div> <div class="given-names">P.</div></div></div> <div class="year">1994</div>, 
<div class="article-title">Karotypic characterization of turbot (<em>Scophthalmus maximus</em>) with conventional, fluorochrome and restriction endonuclease-banding techniques</div>, <div class="source">Mar. Biol</div> ., <div class="volume">120</div>, <div class="fpage">609</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Karotypic%20characterization%20of%20turbot%20(Scophthalmus%20maximus)%20with%20conventional%2C%20fluorochrome%20and%20restriction%20endonuclease-banding%20techniques&amp;author=C.%20Bouza&amp;author=L.%20Sanchez&amp;author=P.%20Mart%C3%ADnez&amp;publication_year=1994&amp;journal=Mar.%20Biol&amp;volume=120&amp;pages=609-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00350082" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00350082" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00350082"> </span></div> </div></div></div></div></div><div content-id="dsy015-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Team</div> <div class="given-names">R. C.</div></div></div> <div class="year">2013</div>, R: a language and environment for statistical computing.</div></div></div></div><div content-id="dsy015-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Harris</div> <div class="given-names">R. S.</div></div></div> <div class="year">2007</div>, Improved pairwise alignment of genomic DNA. PhD Thesis, The Pennsylvania State University, Pennsylvania.</div></div></div></div><div content-id="dsy015-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vera</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Álvarez-Dios</div> <div class="given-names">J. A.</div></div>, <div class="name string-name"><div class="surname">Millán</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Validation of single nucleotide polymorphism (SNP) markers from an immune Expressed Sequence Tag (EST) turbot, <em>Scophthalmus maximus, database</em></div>, <div class="source">Aquaculture</div> , <div class="volume">313</div>, <div class="fpage">31</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Validation%20of%20single%20nucleotide%20polymorphism%20(SNP)%20markers%20from%20an%20immune%20Expressed%20Sequence%20Tag%20(EST)%20turbot%2C%20Scophthalmus%20maximus%2C%20database&amp;author=M.%20Vera&amp;author=J.%20A.%20%C3%81lvarez-Dios&amp;author=A.%20Mill%C3%A1n&amp;publication_year=2011&amp;journal=Aquaculture&amp;volume=313&amp;pages=31-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.aquaculture.2011.01.038" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.aquaculture.2011.01.038" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.aquaculture.2011.01.038"> </span></div> </div></div></div></div></div><div content-id="dsy015-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vera</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Alvarez-Dios</div> <div class="given-names">J. A.</div></div>, <div class="name string-name"><div class="surname">Fernandez</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Development and validation of single nucleotide polymorphisms (SNPs) markers from two transcriptome 454-runs of turbot (<em>Scophthalmus maximus</em>) using high-throughput genotyping</div>, <div class="source">IJMS</div> , <div class="volume">14</div>, <div class="fpage">5694</div>–<div class="lpage">711</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20validation%20of%20single%20nucleotide%20polymorphisms%20(SNPs)%20markers%20from%20two%20transcriptome%20454-runs%20of%20turbot%20(Scophthalmus%20maximus)%20using%20high-throughput%20genotyping&amp;author=M.%20Vera&amp;author=J.%20A.%20Alvarez-Dios&amp;author=C.%20Fernandez&amp;publication_year=2013&amp;journal=IJMS&amp;volume=14&amp;pages=5694-711" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3390/ijms14035694" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3390%2Fijms14035694" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3390%2Fijms14035694"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23481633" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ohno</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Wolf</div> <div class="given-names">U.</div></div>, <div class="name string-name"><div class="surname">Atkin</div> <div class="given-names">N. B.</div></div></div> <div class="year">1968</div>, 
<div class="article-title">Evolution from fish to mammals by gene duplication</div>, <div class="source">Hereditas</div> , <div class="volume">59</div>, <div class="fpage">169</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20from%20fish%20to%20mammals%20by%20gene%20duplication&amp;author=S.%20Ohno&amp;author=U.%20Wolf&amp;author=N.%20B.%20Atkin&amp;publication_year=1968&amp;journal=Hereditas&amp;volume=59&amp;pages=169-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1601-5223.1968.tb02169.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1601-5223.1968.tb02169.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1601-5223.1968.tb02169.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5662632" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Qin</div> <div class="given-names">Z.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Construction of a high-density, high-resolution genetic map and its integration with BAC-based physical map in channel catfish</div>, <div class="source">DNA Res</div> ., <div class="volume">22</div>, <div class="fpage">39</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20high-density%2C%20high-resolution%20genetic%20map%20and%20its%20integration%20with%20BAC-based%20physical%20map%20in%20channel%20catfish&amp;author=Y.%20Li&amp;author=S.%20Liu&amp;author=Z.%20Qin&amp;publication_year=2015&amp;journal=DNA%20Res&amp;volume=22&amp;pages=39-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsu038" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsu038" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsu038"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25428894" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Palaiokostas</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Bekaert</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Taggart</div> <div class="given-names">J. B.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">A new SNP-based vision of the genetics of sex determination in European sea bass (<em>Dicentrarchus labrax</em>)</div>, <div class="source">Genet. Sel. Evol</div> ., <div class="volume">47</div>, <div class="fpage">68</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20SNP-based%20vision%20of%20the%20genetics%20of%20sex%20determination%20in%20European%20sea%20bass%20(Dicentrarchus%20labrax)&amp;author=C.%20Palaiokostas&amp;author=M.%20Bekaert&amp;author=J.%20B.%20Taggart&amp;publication_year=2015&amp;journal=Genet.%20Sel.%20Evol&amp;volume=47&amp;pages=68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12711-015-0148-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12711-015-0148-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12711-015-0148-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26337592" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Lydiate</div> <div class="given-names">D. J.</div></div>, <div class="name string-name"><div class="surname">Parkin</div> <div class="given-names">I. A.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Integration of linkage maps for the amphidiploid Brassica napus and comparative mapping with Arabidopsis and <em>Brassica rapa</em></div>, <div class="source">BMC Genomics</div> ., <div class="volume">12</div>, <div class="fpage">101</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integration%20of%20linkage%20maps%20for%20the%20amphidiploid%20Brassica%20napus%20and%20comparative%20mapping%20with%20Arabidopsis%20and%20Brassica%20rapa&amp;author=J.%20Wang&amp;author=D.%20J.%20Lydiate&amp;author=I.%20A.%20Parkin&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=101" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-101"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21306613" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Khan</div> <div class="given-names">M. A.</div></div>, <div class="name string-name"><div class="surname">Han</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">Y. F.</div></div>, <div class="name string-name"><div class="surname">Troggio</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Korban</div> <div class="given-names">S. S.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">A multi-population consensus genetic map reveals inconsistent marker order among maps likely attributed to structural variations in the apple genome</div>, <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e47864</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20multi-population%20consensus%20genetic%20map%20reveals%20inconsistent%20marker%20order%20among%20maps%20likely%20attributed%20to%20structural%20variations%20in%20the%20apple%20genome&amp;author=M.%20A.%20Khan&amp;author=Y.%20Han&amp;author=Y.%20F.%20Zhao&amp;author=M.%20Troggio&amp;author=S.%20S.%20Korban&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e47864" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0047864" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0047864" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0047864"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23144832" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Close</div> <div class="given-names">T. J.</div></div>, <div class="name string-name"><div class="surname">Bhat</div> <div class="given-names">P. R.</div></div>, <div class="name string-name"><div class="surname">Lonardi</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">Development and implementation of high-throughput SNP genotyping in barley</div>, <div class="source">BMC Genomics</div> , <div class="volume">10</div>, <div class="fpage">582</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20implementation%20of%20high-throughput%20SNP%20genotyping%20in%20barley&amp;author=T.%20J.%20Close&amp;author=P.%20R.%20Bhat&amp;author=S.%20Lonardi&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=582" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-582" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-582" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-582"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19961604" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Martínez</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Robledo</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Rodríguez-Ramilo</div> <div class="given-names">S. T.</div></div></div>, et al.   <div class="year">2016</div>, Turbot (<em>Scophthalmus maximus</em>) genomic resources: application for boosting aquaculture production, In: <div class="person-group"><div class="name string-name"><div class="surname">MacKenzie</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Jentoft</div> <div class="given-names">S.</div></div></div> (eds) <div class="source">Genomics in Aquaculture</div> , 
<div class="publisher-name">Elsevier</div>, 
<div class="publisher-loc">London</div>, pp. <div class="fpage">131</div>–<div class="lpage">63</div>.</div></div></div></div><div content-id="dsy015-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Roberts</div> <div class="given-names">P. A.</div></div></div> <div class="year">1965</div>, 
<div class="article-title">Difference in the behaviour of eu-and hetero-chromatin: crossing-over</div>, <div class="source">Nature</div> , <div class="volume">205</div>, <div class="fpage">725</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Difference%20in%20the%20behaviour%20of%20eu-and%20hetero-chromatin%3A%20crossing-over&amp;author=P.%20A.%20Roberts&amp;publication_year=1965&amp;journal=Nature&amp;volume=205&amp;pages=725-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/205725b0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F205725b0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F205725b0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14287435" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lou</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">He</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Integration of high-resolution physical and genetic map reveals differential recombination frequency between chromosomes and the genome assembling quality in cucumber</div>, <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e62676</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integration%20of%20high-resolution%20physical%20and%20genetic%20map%20reveals%20differential%20recombination%20frequency%20between%20chromosomes%20and%20the%20genome%20assembling%20quality%20in%20cucumber&amp;author=Q.%20Lou&amp;author=Y.%20He&amp;author=C.%20Cheng&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e62676" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0062676" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0062676" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0062676"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23671621" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Singer</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Perlman</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Yan</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2002</div>, 
<div class="article-title">Sex-specific recombination rates in zebrafish (<em>Danio rerio</em>)</div>, <div class="source">Genetics</div> , <div class="volume">160</div>, <div class="fpage">649</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sex-specific%20recombination%20rates%20in%20zebrafish%20(Danio%20rerio)&amp;author=A.%20Singer&amp;author=H.%20Perlman&amp;author=Y.%20Yan&amp;publication_year=2002&amp;journal=Genetics&amp;volume=160&amp;pages=649-57" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11861568" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Moen</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Hayes</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Baranski</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">A linkage map of the Atlantic salmon (<em>Salmo salar</em>) based on EST-derived SNP markers</div>, <div class="source">BMC Genomics</div> ., <div class="volume">9</div>, <div class="fpage">223</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20linkage%20map%20of%20the%20Atlantic%20salmon%20(Salmo%20salar)%20based%20on%20EST-derived%20SNP%20markers&amp;author=T.%20Moen&amp;author=B.%20Hayes&amp;author=M.%20Baranski&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=223" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-223" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-223" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-223"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18482444" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Limborg</div> <div class="given-names">M. T.</div></div>, <div class="name string-name"><div class="surname">Helyar</div> <div class="given-names">S. J.</div></div>, <div class="name string-name"><div class="surname">de Bruyn</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Environmental selection on transcriptome-derived SNPs in a high gene flow marine fish, the Atlantic herring (<em>Clupea harengus</em>)</div>, <div class="source">Mol. Ecol</div> ., <div class="volume">21</div>, <div class="fpage">3686</div>–<div class="lpage">703</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Environmental%20selection%20on%20transcriptome-derived%20SNPs%20in%20a%20high%20gene%20flow%20marine%20fish%2C%20the%20Atlantic%20herring%20(Clupea%20harengus)&amp;author=M.%20T.%20Limborg&amp;author=S.%20J.%20Helyar&amp;author=M.%20de%20Bruyn&amp;publication_year=2012&amp;journal=Mol.%20Ecol&amp;volume=21&amp;pages=3686-703" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-294X.2012.05639.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-294X.2012.05639.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-294X.2012.05639.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22694661" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lenormand</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Dutheil</div> <div class="given-names">J.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Recombination difference between sexes: a role for haploid selection</div>, <div class="source">PLoS Biol</div> ., <div class="volume">3</div>, <div class="fpage">e63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recombination%20difference%20between%20sexes%3A%20a%20role%20for%20haploid%20selection&amp;author=T.%20Lenormand&amp;author=J.%20Dutheil&amp;publication_year=2005&amp;journal=PLoS%20Biol&amp;volume=3&amp;pages=e63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0030063" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0030063" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0030063"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15736976" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gruhn</div> <div class="given-names">J. R.</div></div>, <div class="name string-name"><div class="surname">Rubio</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Broman</div> <div class="given-names">K. W.</div></div>, <div class="name string-name"><div class="surname">Hunt</div> <div class="given-names">P. A.</div></div>, <div class="name string-name"><div class="surname">Hassold</div> <div class="given-names">T.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Cytological studies of human meiosis: sex-specific differences in recombination originate at, or prior to, establishment of double-strand breaks</div>, <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e85075</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytological%20studies%20of%20human%20meiosis%3A%20sex-specific%20differences%20in%20recombination%20originate%20at%2C%20or%20prior%20to%2C%20establishment%20of%20double-strand%20breaks&amp;author=J.%20R.%20Gruhn&amp;author=C.%20Rubio&amp;author=K.%20W.%20Broman&amp;author=P.%20A.%20Hunt&amp;author=T.%20Hassold&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e85075" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0085075" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0085075" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0085075"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24376867" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pardo</div> <div class="given-names">B. G.</div></div>, <div class="name string-name"><div class="surname">Machordom</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Foresti</div> <div class="given-names">F.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">Phylogenetic analysis of flatfish (Order Pleuronectiformes) based on mitochondrial 16s rDNA sequences</div>, <div class="source">Sci. Mar</div> ., <div class="volume">69</div>, <div class="fpage">531</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20analysis%20of%20flatfish%20(Order%20Pleuronectiformes)%20based%20on%20mitochondrial%2016s%20rDNA%20sequences&amp;author=B.%20G.%20Pardo&amp;author=A.%20Machordom&amp;author=F.%20Foresti&amp;publication_year=2005&amp;journal=Sci.%20Mar&amp;volume=69&amp;pages=531-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3989/scimar.2005.69n4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3989%2Fscimar.2005.69n4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3989%2Fscimar.2005.69n4"> </span></div> </div></div></div></div></div><div content-id="dsy015-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Campbell</div> <div class="given-names">M. A.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">W.-J.</div></div>, <div class="name string-name"><div class="surname">López</div> <div class="given-names">J. A.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Are flatfishes (Pleuronectiformes) monophyletic?,</div> <div class="source">Mol. Phylogenet. Evol</div> ., <div class="volume">69</div>, <div class="fpage">664</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Are%20flatfishes%20(Pleuronectiformes)%20monophyletic%3F%2C&amp;author=M.%20A.%20Campbell&amp;author=W.-J.%20Chen&amp;author=J.%20A.%20L%C3%B3pez&amp;publication_year=2013&amp;journal=Mol.%20Phylogenet.%20Evol&amp;volume=69&amp;pages=664-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2013.07.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2013.07.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ympev.2013.07.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23876291" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Friedman</div> <div class="given-names">M.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">The evolutionary origin of flatfish asymmetry</div>, <div class="source">Nature</div> , <div class="volume">454</div>, <div class="fpage">209</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolutionary%20origin%20of%20flatfish%20asymmetry&amp;author=M.%20Friedman&amp;publication_year=2008&amp;journal=Nature&amp;volume=454&amp;pages=209-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18615083" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Harrington</div> <div class="given-names">R. C.</div></div>, <div class="name string-name"><div class="surname">Faircloth</div> <div class="given-names">B. C.</div></div>, <div class="name string-name"><div class="surname">Eytan</div> <div class="given-names">R. I.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Phylogenomic analysis of carangimorph fishes reveals flatfish asymmetry arose in a blink of the evolutionary eye</div>, <div class="source">BMC Evol. Biol</div> ., <div class="volume">16</div>, <div class="fpage">224</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenomic%20analysis%20of%20carangimorph%20fishes%20reveals%20flatfish%20asymmetry%20arose%20in%20a%20blink%20of%20the%20evolutionary%20eye&amp;author=R.%20C.%20Harrington&amp;author=B.%20C.%20Faircloth&amp;author=R.%20I.%20Eytan&amp;publication_year=2016&amp;journal=BMC%20Evol.%20Biol&amp;volume=16&amp;pages=224" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12862-016-0786-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12862-016-0786-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12862-016-0786-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27769164" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ijdo</div> <div class="given-names">J. W.</div></div>, <div class="name string-name"><div class="surname">Baldini</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ward</div> <div class="given-names">D. C.</div></div>, <div class="name string-name"><div class="surname">Reeders</div> <div class="given-names">S. T.</div></div>, <div class="name string-name"><div class="surname">Wells</div> <div class="given-names">R. A.</div></div></div> <div class="year">1991</div>, 
<div class="article-title">Origin of human chromosome 2: an ancestral telomere-telomere fusion</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> ., <div class="volume">88</div>, <div class="fpage">9051</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origin%20of%20human%20chromosome%202%3A%20an%20ancestral%20telomere-telomere%20fusion&amp;author=J.%20W.%20Ijdo&amp;author=A.%20Baldini&amp;author=D.%20C.%20Ward&amp;author=S.%20T.%20Reeders&amp;author=R.%20A.%20Wells&amp;publication_year=1991&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=88&amp;pages=9051-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.88.20.9051" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.88.20.9051" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.88.20.9051"> </span></div> </div></div></div></div></div><div content-id="dsy015-B76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B76" class="ref-content "><div class="label -label">76</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Azevedo</div> <div class="given-names">M. F. C.</div></div>, <div class="name string-name"><div class="surname">Oliveira</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pardo</div> <div class="given-names">B. G.</div></div>, <div class="name string-name"><div class="surname">Martínez</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Foresti</div> <div class="given-names">F.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Cytogenetic characterization of six species of flatfishes with comments to karyotype differentiation patterns in Pleuronectiformes (Teleostei)</div>, <div class="source">J. Fish Biology</div> , <div class="volume">70</div>, <div class="fpage">1</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytogenetic%20characterization%20of%20six%20species%20of%20flatfishes%20with%20comments%20to%20karyotype%20differentiation%20patterns%20in%20Pleuronectiformes%20(Teleostei)&amp;author=M.%20F.%20C.%20Azevedo&amp;author=C.%20Oliveira&amp;author=B.%20G.%20Pardo&amp;author=P.%20Mart%C3%ADnez&amp;author=F.%20Foresti&amp;publication_year=2007&amp;journal=J.%20Fish%20Biology&amp;volume=70&amp;pages=1-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jfb.2007.70.issue-sa" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjfb.2007.70.issue-sa" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjfb.2007.70.issue-sa"> </span></div> </div></div></div></div></div><div content-id="dsy015-B77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B77" class="ref-content "><div class="label -label">77</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Amores</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Wilson</div> <div class="given-names">C. A.</div></div>, <div class="name string-name"><div class="surname">Allard</div> <div class="given-names">C. A.</div></div>, <div class="name string-name"><div class="surname">Detrich</div> <div class="given-names">H. W.</div></div>, <div class="name string-name"><div class="surname">Postlethwait</div> <div class="given-names">J. H.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Cold fusion: massive karyotype evolution in the Antarctic Bullhead Notothen <em>Notothenia coriiceps</em></div>, <div class="source">G3 Genes Genomes Genet</div> ., <div class="volume">7</div>, <div class="fpage">2195</div>–<div class="lpage">207</div>.</div></div></div></div><div content-id="dsy015-B78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B78" class="ref-content "><div class="label -label">78</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Livingstone</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Rieseberg</div> <div class="given-names">L.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Chromosomal evolution and speciation: a recombination-based approach</div>, <div class="source">New Phytol</div> ., <div class="volume">161</div>, <div class="fpage">107</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosomal%20evolution%20and%20speciation%3A%20a%20recombination-based%20approach&amp;author=K.%20Livingstone&amp;author=L.%20Rieseberg&amp;publication_year=2004&amp;journal=New%20Phytol&amp;volume=161&amp;pages=107-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1469-8137.2003.00942.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1469-8137.2003.00942.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1469-8137.2003.00942.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19079643" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy015-B79" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy015-B79" class="ref-content "><div class="label -label">79</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Navarro</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Barton</div> <div class="given-names">N. H.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Chromosomal speciation and molecular divergence–accelerated evolution in rearranged chromosomes</div>, <div class="source">Science</div> , <div class="volume">300</div>, <div class="fpage">321</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosomal%20speciation%20and%20molecular%20divergence%E2%80%93accelerated%20evolution%20in%20rearranged%20chromosomes&amp;author=A.%20Navarro&amp;author=N.%20H.%20Barton&amp;publication_year=2003&amp;journal=Science&amp;volume=300&amp;pages=321-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1080600" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1080600" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1080600"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12690198" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2018. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=5035912&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures1.pdf?Expires=2147483647&amp;Signature=iRjYbKf0EsHectagtCG899YTSLSdL5P9bjXND9ypL1k5V6yJ9N7kaummLyzV8M9Aq~97B5XDuw7YQqAnSpYUQavEVl1qOsU6u8C6qNnsT2dgK1Diph~sWCQiex5kgT5xg3OakOagsJ38En7mNF5NSVezfdYwRnXdmMXhqXoeHpAI~aVtZ9Io8XCLXDkKcp6ngL0Ho8nHSnewzAz7BfLAYUAaXJrHfCwagjaNh4~5QhTV2JP4MIWNYMq4t8cxYezdidf2K0JxD~cRLan2DrLA5G2GDs1P4SeeqOrFjjLMecaT8r8pnM1MQk2TpmdSuiaIzhK9IymwzStqwCzsHinLOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary FigureS1</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures2.pdf?Expires=2147483647&amp;Signature=VNgFStr9FLU8w1BECvjvHbs6Q26TzCR6B1ziQ6sJxWURy7XAlwpn3g9bbDcrQiVXBi4NAD7fXthcDR0qffugWaNxSrhXEXB0DuPWW7PuBcLRzyCyiz9QdkztnuHURL7K4xvSWQFl8gEoMEDmYNrOMqrsFiripYaamTx6AWd08hzYgoUauSwicWTOVYE2E4F1meK2m~7OZ2q52Am3yTrOQjKojSPLA3yQM-wXfEnDHEF90~GMGRg7I3v7SmAG~1GpFXhv0roDaymH5ygPAn8ZGgwKgpXpk2wVVIvtO2BLUTJY1F6LyQjSZiQIOoRZr9jPoOm0L4b75wRaQQ7eLp9XUA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary FigureS2</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures3.pdf?Expires=2147483647&amp;Signature=rze0qaBj8BUmDYGTmlsnfb3I0AFDqsNCedVFBj0IQHFQK0cY1b9yr0DeEewu5JIhnFlgKS8h3lySXwBvBDsVs3JUA54eHOv55brf~pp6RA8pErLbudARu-2B9Zq5hm-bgKpDWdQJA13njpUy07wgyRflkC9Mn8nqQho041JIztv8QPCCaDY7lt2PtzYWNLmUYEjMlLdrUjnyQ1GaK0H8tRFLmex39MmH9VzrtGPDk8rfxN4E~Fl~DwXwlYcp4KNZp0MbUoSqQs4W4dbJHD3Cq~CXaWO~Wq~cr-1nYJpwunmjZEfKbPsixBm0eqxAhe-NUpsYdoAM-Qcnz3hvKgK7UA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary FigureS3</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures4.pdf?Expires=2147483647&amp;Signature=ZXzMONe2mxWzVNQBOd2IHH33n-9uVtGWjdlB4dJQlBdrH48tT-wLdDI~ST7OwRB~mB2sZqvJ2iwqAV7B1hKIPnQkw4Ed8pgphCHI1FKv0R0gr3eSps4YQxOeZW-GPkkT8-7qrhgpp~HYY-ZiFOVFtetq7ELUrjEgi~Yshldr1hVUIA6Y~mnCp~rf8VJgZ291wC17wanN-Y0A5jTh~jOnDKaJPtZwVkBB6NCvCeYx7sw6H0SdTK2-UEDZELyYVZRF0belrdmBj4Dz96j2wZtK8tMoFloZ0hYiHLm5f25YF8~-RmWzqyHLkiMUjqAThQ~1q~Vso3qMTMJu4xn5Vbyzsg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary FigureS4</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures5.pdf?Expires=2147483647&amp;Signature=KCOzgUfV01oiE4OE5USGGzbMSVkItECEhE27atsiYrZogtIP5h~pEP2y~m1Fl5qz4HUVrropQd4vc273vI~NVd3mzLaAb5WzrV-VZ2RSOKcVJTF6rnk3zKiL7rf5mFv0FD5FjFXrijiCM6uLlEKUZikOa7chth75m4Nl02dHuWjhqHi0YITrJvJHvUwQ9tx1yJYE8MmG73FYc2U4ayk5YIc9T2WHIJXZwBSIij4oIEl91DF-qIwsbEiD8J~FXnqw775w6Kv~6vy6f-WgESFRuDD7rlIt~OwRSaAOeyf-tSfSIn1SSGI5TBYbwojEKziv46L2BRYj3U6Ie6qFEiXQxg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary FigureS5</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures6.pdf?Expires=2147483647&amp;Signature=BzbimyjEMuBvB5cpAH3JQaykawpE8WAtNc06q-CwLlXUcLTFlmY9va00Bm-rhD2WbEGZvpZOH~jmdPi0nb0o6liq1eIVVu-GB78MnXe58YKKbBRfqaE3XsV2B7rPtNhoZs4LTm85Exa6GmnKMQPUe1o~~6fBHDSlI8TzXFRQXvpzyH3TZE2aWTMl9sJ0fN8hz7TgGM~6LPr39-60xogiHY~yCVkwoo~kJhSjTTuvX0F3MW1EoHWr9DwCitIfJCFjnNboyHlQVWfBgs5B9FtKXlWJuYYXY8TyCVAfyCO~DBwVADhWZHUBZC2v~yrsdgyM3KUbvshZMYguSBaK2klcag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary FigureS6</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures7a.pdf?Expires=2147483647&amp;Signature=BG9oRJnYtqu01TUzVT74Llro4Y78oayaa8C3f7A5g0UfzR0MWX4Zkp25RWo5TfU702CVdbedz5BZ0psae1LZepJaLFpAoDdaeoXwwBqgrgCNbTSi1pT6YeriYkYUP75~imqtMHTTW7pIRid49xCYr1uc7e7hWIEkLIYltgliP4I3abhI-yvwVGSjFGuQoMoPHU9T-tw634brhzoJ7OGBx1GiG6MOyqA0reGrvfYL93dXHBmGNMeVCzyFLPTQwVGGdzmtUGDtM6PAW45Kvsyuc~vUFY4Jqr-hSj9U3Yt7O0fNMrHCOHSCxY4smJOdntavjDuVLB5SNBaEknszi9znIA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary FigureS7A</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures7b.pdf?Expires=2147483647&amp;Signature=qXtL7nyzxZ4bS3ILiX36gnIZf9BazsbIoOV3jwT25ef8-ke0ReoNkcR8QlOqmG5Hb5gYYRGPbp96AR8~nJkPBi57WDRd-9eFYqQFLgCDp2jDEMf3ykqIJmlA1QefKPLkkF1AbrJJwhw9ufXVSxVDiABwSHnmMtTL8ZhlPVGUVhDPpMSRifXDV5jFLy9pNRNNrA24QzYsaPTFBjk1sFcUVam94hnVFs3rYKGSuTd6vn40c6Kb7Vtojr0KZzPtD308sOow0vJ6dXQRpAWedFAxxnyCSsYd2QNx2oV5arCTmxcR-eOsznoWhcYe5gb798kMLsUsw0Rambrs1tn~hmpsQg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary FigureS7B</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_figures7c.pdf?Expires=2147483647&amp;Signature=jLFpXGWltyztBbLsB9yLRRlx5Atwpr0RN3aFtDhsQubxXF1KX2H1mYSK6Jqs3dFZBR19IzXCcvPCj0tzkL7-bJPTcKGwKGwutmPVdr6cqWO7pV7Tg2q4H4oun3uqkAPTxhURXHz-pSH19k41EB0XSSupdkzjrTid2iBkEXcxfsJkBIkP4UPfijO0xcDQDvfqBWVptovHgg1CT1CLwrkmIzmbuZcUR3TK2gFpnN60u8LAA4ffez8~SdjD6lTVvzdL1LN-74hnmWpNz-GStBFv28EM3FskzwX41PFtP-3YOy5ROQEUNF6Qq19j6dFC4NkiLQa~h5odyWuAryWzDLyekg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary FigureS7C</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_maroso_dnares_supplementary_legends.doc?Expires=2147483647&amp;Signature=4hzbB9uSqpQDDD7vw8L0TRE5ThRl3nk0KTJ3oCaczcxIKHTj1juyPXhWQXDo1c5oPjGCvc1W4TR~UCqvy8dzh1qNRxn88hP1FtGm0ptR7P4AWp4AGrSstgtTaHixmmyUydvoSHw4n02V5wWpte7t2lVRjlXRLsWBmUnUUZu5zk0hA4A3Cw3-uFfx9NLiKP6tYDyvmZKLkdoyvhheKoe1c8XKgbmZN-gmMkvphRTKM93ODHpO7UhBZv4AgQ27VfOP0wC8Hm8ostjS4iqV2Eu70xToU5bu66wyH1LGPNpFm7lQsFkda027ezgH7HI0WjCOeYHxHgclv9VysWLwW9saIw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Legends</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_tables1.xlsx?Expires=2147483647&amp;Signature=iu-s0ucyh61Jqm6Al68IyTnp0EwFC7zFFYSo0Z3cPTfltnRkboXvIDx076CZIb5BEt6jLoHAmQfgwmD~i9kS2Y-catWPu0lQ4QDwsKqAkbCWMHHdmDAC2~bW6Immj2-K3vlVHNbUu8ro6byRUgCLiCUlLpNGoPFmB6-VJqFQx-RcsXQLNQFTCBwIuTMihcwQtJEjlUB5CtJSNGma04LlRv4IHvvQtZxz2~hdm7o2VgohCrtQrsA1Ip~EuVwq56pYaqNwhw2WKH2-~XS4j9KlLjWPc1VJwobAhvRpgiQq2xKc16Y~XOVMteY2rJXBkfxKKn-i59DfiGAuXNvV8U~W0Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary TableS1</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_tables2.xlsx?Expires=2147483647&amp;Signature=ih0SleqhQTp8cq1tZKwJOVoRcuIGZHurnaOzK1GA5pTH97D9Ym~SBbhLMS-OY68qtfA2PCaXKMK-Pyqd5qvqRqisq-pVVMyc0GzcRrT2GrZ067SkF2xyd1OV-hxLof-q2zFkbkYU5paBMjPiOA39LSKEZ4W4CGLwDzH5CfM8K0AxmpLV8JMk24LxnT7chbBXLGXl9eXxu2tmxnPyREtr3gpZX3OvFH47xSLafdvVrBlSJw~LZmkAC97D1d0SOjFV75O-DKGr~mgTgYfoygF1Tn-ZOP~ClcShwFcREzpiUFdSjFs9H7tIotRDJS7hbXtgbwfc7X7HjIQ0qcm0WWK8kw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary TableS2</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_tables3.xlsx?Expires=2147483647&amp;Signature=3iDrLGKcOPMCA9dyE1~NI5ar1ra7h6xEt7v1vZO8Zf1Y4mocW~QAc7-Yp-i71Wwlz0H1ON2cWX0ec4iuNnLIEyvZD9Xeu~Cc7kSIU9tXD5bSIsgCoQ9EskNAUmzC646kpN3ZxhTvVpbxOXhNTsd21N2swt5lAbJaGcAZRpgxVjChm9M8~1iq~Gs0G9sk-ePT0N3q1wgtSnR8vbVpCQneehCkbiRAnNcziMAyIccRLJk2L7IBrcO4aQXJrgJpVvaWTSOcOl0fw8ZSkjCy77cTpjLTL1XcldgT2WeV2LKo~DDHshvzasu5tLcV5FxIDO1iuIlCwXPrqz38FC6cztkS8g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary TableS3</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_tables4.xlsx?Expires=2147483647&amp;Signature=1gGP~lDt-SdNPSMaZCPLgvZlyfZgnR91WAiTr9Rpd-pu6pCFc-q2DS4fMgT9TgkpWu73KPXCjrLoizxfiRL8ULtttUTypNbHqkAYe~zQagdhe02JpOsPD7~UDpIOCnoWMUrOt7AwSDyPhX~rlLIwuApVVi5A19dq7-LpZFg08IOjIZ-V9rHTpIT8X0pv0MFMKAlT2UPp8sGt8gSLZP8gPlfkZptJQjDo6qGyKBh40QPIHbm~pxY2N9nmVjYsp0Rc7CO8~SKSvt5tbpU-zgZM7yMoxzvHRJM~Rol5KVy3pRdBefzMe20AnFrYRiEbP8cuD8R4iADBMu5ZfAZbywgwjA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary TableS4</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy015/1/dsy015_tables5.xlsx?Expires=2147483647&amp;Signature=XtZ9-NcPrMejnhLh1bBfGmsnYP7AoctNMpQ2k5hU9c8c0CN9WboCPUhDUujBy3unr6Y1T4YArj0XI3YlI~Iu7fNMhBykpn0smO22y2fYG77Th5K7ehJQKKzwEwMBzdVlBlyleavqjvG0VmheTpcs3TAaH8pJP29E3oChXSRzMSvF7ogMm6ZLzteHh2R4u95m-XuYvS~tl~MVyo2EkqQEEfoCl~RuU77nS8Rvpl1UVmnXA9leWZJy6TU9XSUndCWOKS4WchwL-k5no1g2CS6KFP0KEJMfaJzyzrRhjWxK4AeEDbibmXjBgPL21HK8D6oB1hRKXqIeSJvOmgaoT1CpYA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary TableS5</div></a> - xlsx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">593</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsy015"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5035912"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f4%2f439%2f5035912">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F4%2F439%2F5035912" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/5035912'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Highly dense linkage maps from 31 full-sibling families of turbot (Scophthalmus maximus) provide insights into recombination patterns and chromosome rearrangements throughout a newly refined genome assembly" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy015");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f439%2f5035912">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

