

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/1/67/2462472","@type":"ScholarlyArticle","name":"Next-generation sequencing analysis of lager brewing yeast strains reveals the evolutionary history of interspecies hybridization","datePublished":"2016-01-04","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2016-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv037","keywords":["lager beer yeast","interspecies hybrid","<italic>Saccharomyces pastorianus</italic>","loss of heterozygosity","allopolyploid"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/1/67/2462472","author":[{"name":"Okuno, Miki","affiliation":"1  Department of Biological Information ,  Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology ,  2-12-1 Ookayama, Meguro-ku, Tokyo 152-8550 ,  Japan","@type":"Person"},{"name":"Kajitani, Rei","affiliation":"1  Department of Biological Information ,  Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology ,  2-12-1 Ookayama, Meguro-ku, Tokyo 152-8550 ,  Japan","@type":"Person"},{"name":"Ryusui, Rie","affiliation":"1  Department of Biological Information ,  Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology ,  2-12-1 Ookayama, Meguro-ku, Tokyo 152-8550 ,  Japan","@type":"Person"},{"name":"Morimoto, Hiroya","affiliation":"1  Department of Biological Information ,  Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology ,  2-12-1 Ookayama, Meguro-ku, Tokyo 152-8550 ,  Japan","@type":"Person"},{"name":"Kodama, Yukiko","affiliation":"2  Suntory Global Innovation Center Limited ,  8-1-1 Seikadai, Seika-cho, Soraku-gun, Kyoto 619-0284 ,  Japan","@type":"Person"},{"name":"Itoh, Takehiko","affiliation":"1  Department of Biological Information ,  Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology ,  2-12-1 Ookayama, Meguro-ku, Tokyo 152-8550 ,  Japan","@type":"Person"}],"description":"Abstract.  The lager beer yeast Saccharomyces pastorianus is considered an allopolyploid hybrid species between S. cerevisiae and S. eubayanus. Many S. pastoria","pageStart":"67","pageEnd":"80","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/1/1/m_cover.gif?Expires=2147483647&Signature=KARGGT3434AhJvd0livqaaljLmMbBUo3zfp1TqsESiO3YlY~tkN7g3APCzu-WSs6e8Nl3TqpcrmHNHH0dQCdcITSOeE~aAPITj~eX0Yge-o3-8ykKxYb~OvG6Tl68sbE8kb9MDj8-DVfaGgSNHoeCfxrY9GfQxxlAheNwGlc1dEbYs6TpQ1qqETGqeV3fGp~5ePhsu4hP22hUcJ1Z06TJ5-2W4IJPC8QUHrse0jdAO7iAQAiMxLNxFuiLgIQ4xVlRKAVyu8pGxiw2RvdR2ib~ISf7YQpT2ikZjdevDSe5J5xnrmr4aL9l8xCPdqY1Y38fIiBcp0jIC7kA0-uvGAJYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Next-generation sequencing analysis of lager brewing yeast strains reveals the evolutionary history of interspecies hybridization","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/1/1/m_cover.gif?Expires=2147483647&Signature=KARGGT3434AhJvd0livqaaljLmMbBUo3zfp1TqsESiO3YlY~tkN7g3APCzu-WSs6e8Nl3TqpcrmHNHH0dQCdcITSOeE~aAPITj~eX0Yge-o3-8ykKxYb~OvG6Tl68sbE8kb9MDj8-DVfaGgSNHoeCfxrY9GfQxxlAheNwGlc1dEbYs6TpQ1qqETGqeV3fGp~5ePhsu4hP22hUcJ1Z06TJ5-2W4IJPC8QUHrse0jdAO7iAQAiMxLNxFuiLgIQ4xVlRKAVyu8pGxiw2RvdR2ib~ISf7YQpT2ikZjdevDSe5J5xnrmr4aL9l8xCPdqY1Y38fIiBcp0jIC7kA0-uvGAJYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Next-generation sequencing analysis of lager brewing yeast strains reveals the evolutionary history of interspecies hybridization" />
    <meta property="og:description" content="Abstract.  The lager beer yeast Saccharomyces pastorianus is considered an allopolyploid hybrid species between S. cerevisiae and S. eubayanus. Many S. pastoria" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/1/67/2462472" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-01-04" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Okuno, Miki" /><meta name="citation_author_institution" content="Department of Biological Information, Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology, 2-12-1 Ookayama, Meguro-ku, Tokyo 152-8550, Japan" /><meta name="citation_author" content="Kajitani, Rei" /><meta name="citation_author_institution" content="Department of Biological Information, Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology, 2-12-1 Ookayama, Meguro-ku, Tokyo 152-8550, Japan" /><meta name="citation_author" content="Ryusui, Rie" /><meta name="citation_author_institution" content="Department of Biological Information, Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology, 2-12-1 Ookayama, Meguro-ku, Tokyo 152-8550, Japan" /><meta name="citation_author" content="Morimoto, Hiroya" /><meta name="citation_author_institution" content="Department of Biological Information, Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology, 2-12-1 Ookayama, Meguro-ku, Tokyo 152-8550, Japan" /><meta name="citation_author" content="Kodama, Yukiko" /><meta name="citation_author_institution" content="Suntory Global Innovation Center Limited, 8-1-1 Seikadai, Seika-cho, Soraku-gun, Kyoto 619-0284, Japan" /><meta name="citation_author" content="Itoh, Takehiko" /><meta name="citation_author_institution" content="Department of Biological Information, Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology, 2-12-1 Ookayama, Meguro-ku, Tokyo 152-8550, Japan" /><meta name="citation_title" content="Next-generation sequencing analysis of lager brewing yeast strains reveals the evolutionary history of interspecies hybridization" /><meta name="citation_firstpage" content="67" /><meta name="citation_lastpage" content="80" /><meta name="citation_doi" content="10.1093/dnares/dsv037" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2016/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content=" P, L.  1909 , Mikroskopische Betriebskontrolle in den Garungswerben 6th edn. " /><meta name="citation_reference" content="citation_title=Comparative physiology and fermentation performance of Saaz and Frohberg lager yeast strains and the parental species Saccharomyces eubayanus;  citation_journal_title=Yeast;  citation_year=2013;  citation_volume=30;  citation_pages=255-66; " /><meta name="citation_reference" content="citation_title=Microbe domestication and the identification of the wild genetic stock of lager-brewing yeast;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2011;  citation_volume=108;  citation_pages=14539-44; " /><meta name="citation_reference" content="citation_title=Evidence for a Far East Asian origin of lager beer yeast;  citation_journal_title=Curr. Biol.;  citation_year=2014;  citation_volume=24;  citation_pages=R380-1; " /><meta name="citation_reference" content="citation_title=Sequencing and characterisation of rearrangements in three S. pastorianus strains reveals the presence of chimeric genes and gives evidence of breakpoint reuse;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e92203" /><meta name="citation_reference" content="citation_title=Genome sequence of the lager brewing yeast, an interspecies hybrid;  citation_journal_title=DNA Res.;  citation_year=2009;  citation_volume=16;  citation_pages=115-29; " /><meta name="citation_reference" content="citation_title=Pichia sorbitophila, an interspecies yeast hybrid, reveals early steps of genome resolution after polyploidization;  citation_journal_title=G3 Genes, Genomes, Genet.;  citation_year=2012;  citation_volume=2;  citation_pages=299-311; " /><meta name="citation_reference" content="citation_title=The genome sequence of the highly acetic acid-tolerant Zygosaccharomyces bailii-derived interspecies hybrid strain ISA1307, isolated from a sparkling wine plant,;  citation_journal_title=DNA Res.;  citation_year=2014;  citation_volume=21;  citation_pages=299-313; " /><meta name="citation_reference" content="citation_title=The genome sequence of the wine yeast VIN7 reveals an allotriploid hybrid genome with Saccharomyces cerevisiae and Saccharomyces kudriavzevii origins;  citation_journal_title=FEMS Yeast Res.;  citation_year=2012;  citation_volume=12;  citation_pages=88-96; " /><meta name="citation_reference" content="citation_title=Aneuploidy, copy number variation and unique chromosomal structures in bottom-fermenting yeast revealed by array-CGH;  citation_journal_title=J. Inst. Brew.;  citation_year=2014;  citation_volume=120;  citation_pages=27-37; " /><meta name="citation_reference" content="citation_title=Reconstruction of the genome origins and evolution of the hybrid lager yeast Saccharomyces pastorianus;  citation_journal_title=Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=1610-23; " /><meta name="citation_reference" content="citation_title=Inferences of evolutionary relationships from a population survey of LTR-retrotransposons and telomeric-associated sequences in the Saccharomyces sensu stricto complex;  citation_journal_title=Yeast;  citation_year=2005;  citation_volume=22;  citation_pages=177-92; " /><meta name="citation_reference" content="citation_title=Loss of lager specific genes and subtelomeric regions define two different Saccharomyces cerevisiae lineages for Saccharomyces pastorianus Group I and II strains;  citation_journal_title=FEMS Yeast Res.;  citation_year=2015;  citation_volume=15;  citation_pages=fou008" /><meta name="citation_reference" content="citation_title=Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads;  citation_journal_title=Genome Res.;  citation_year=2014;  citation_volume=24;  citation_pages=1384-95; " /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs;  citation_journal_title=Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=S. cerevisiae × S. eubayanus interspecific hybrid, the best of both worlds and beyond;  citation_journal_title=FEMS Yeast Res.;  citation_year=2015;  citation_volume=15;  citation_pages=fov005" /><meta name="citation_reference" content="citation_title=Circos: an information aesthetic for comparative genomics;  citation_journal_title=Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=1639-45; " /><meta name="citation_reference" content="citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data;  citation_journal_title=Genome Res.;  citation_year=2010;  citation_volume=20;  citation_pages=1297-303; " /><meta name="citation_reference" content="citation_title=The Sequence Alignment/Map format and SAMtools;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=PHYML Online—a web server for fast maximum likelihood-based phylogenetic inference;  citation_journal_title=Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=557-9; " /><meta name="citation_reference" content="citation_title=Genome sequence of Saccharomyces carlsbergensis, the world's first pure culture lager yeast;  citation_journal_title=G3 Genes, Genomes, Genet.;  citation_year=2014;  citation_volume=4;  citation_pages=783-93; " /><meta name="citation_reference" content="citation_title=Whole-genome comparison reveals novel genetic elements that characterize the genome of industrial strains of Saccharomyces cerevisiae;  citation_journal_title=PLoS Genet.;  citation_year=2011;  citation_volume=7;  citation_pages=e1001287" /><meta name="citation_reference" content="citation_title=Whole-genome sequencing of sake yeast Saccharomyces cerevisiae Kyokai no. 7;  citation_journal_title=DNA Res.;  citation_year=2011;  citation_volume=18;  citation_pages=423-34; " /><meta name="citation_reference" content="citation_title=Whole-genome sequencing of the efficient industrial fuel-ethanol fermentative Saccharomyces cerevisiae strain CAT-1;  citation_journal_title=Mol. Genet. Genomics;  citation_year=2012;  citation_volume=287;  citation_pages=485-94; " /><meta name="citation_reference" content="citation_title=The inheritance of mtDNA in lager brewing strains;  citation_journal_title=FEMS Yeast Res.;  citation_year=2008;  citation_volume=8;  citation_pages=586-96; " /><meta name="citation_reference" content="citation_title=Spontaneous loss of heterozygosity in diploid Saccharomyces cerevisiae cells;  citation_journal_title=Genetics;  citation_year=2000;  citation_volume=156;  citation_pages=1531-48; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/1/67/17463233/dsv037.pdf" /><meta name="description" content="Abstract.  The lager beer yeast Saccharomyces pastorianus is considered an allopolyploid hybrid species between S. cerevisiae and S. eubayanus. Many S. pastoria" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/1/67/2462472" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/1/67/2462472" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/1/67/2462472");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv037';
    </script>
<title>
	Next-generation sequencing analysis of lager brewing yeast strains reveals the evolutionary history of interspecies hybridization | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2462472" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BcJMPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPa3VubywgTWlraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9naWNhbCBJbmZvcm1hdGlvbiwgR3JhZHVhdGUgU2Nob29sIG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIFRva3lvIEluc3RpdHV0ZSBvZiBUZWNobm9sb2d5LCAyLTEyLTEgT29rYXlhbWEsIE1lZ3Vyby1rdSwgVG9reW8gMTUyLTg1NTAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2FqaXRhbmksIFJlaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9naWNhbCBJbmZvcm1hdGlvbiwgR3JhZHVhdGUgU2Nob29sIG9mIEJpb3NjaWVuY2UgYW5kIEJpb3RlY2hub2xvZ3ksIFRva3lvIEluc3RpdHV0ZSBvZiBUZWNobm9sb2d5LCAyLTEyLTEgT29rYXlhbWEsIE1lZ3Vyby1rdSwgVG9reW8gMTUyLTg1NTAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUnl1c3VpLCBSaWUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ2ljYWwgSW5mb3JtYXRpb24sIEdyYWR1YXRlIFNjaG9vbCBvZiBCaW9zY2llbmNlIGFuZCBCaW90ZWNobm9sb2d5LCBUb2t5byBJbnN0aXR1dGUgb2YgVGVjaG5vbG9neSwgMi0xMi0xIE9va2F5YW1hLCBNZWd1cm8ta3UsIFRva3lvIDE1Mi04NTUwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1vcmltb3RvLCBIaXJveWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ2ljYWwgSW5mb3JtYXRpb24sIEdyYWR1YXRlIFNjaG9vbCBvZiBCaW9zY2llbmNlIGFuZCBCaW90ZWNobm9sb2d5LCBUb2t5byBJbnN0aXR1dGUgb2YgVGVjaG5vbG9neSwgMi0xMi0xIE9va2F5YW1hLCBNZWd1cm8ta3UsIFRva3lvIDE1Mi04NTUwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktvZGFtYSwgWXVraWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3VudG9yeSBHbG9iYWwgSW5ub3ZhdGlvbiBDZW50ZXIgTGltaXRlZCwgOC0xLTEgU2Vpa2FkYWksIFNlaWthLWNobywgU29yYWt1LWd1biwgS3lvdG8gNjE5LTAyODQsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSXRvaCwgVGFrZWhpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ2ljYWwgSW5mb3JtYXRpb24sIEdyYWR1YXRlIFNjaG9vbCBvZiBCaW9zY2llbmNlIGFuZCBCaW90ZWNobm9sb2d5LCBUb2t5byBJbnN0aXR1dGUgb2YgVGVjaG5vbG9neSwgMi0xMi0xIE9va2F5YW1hLCBNZWd1cm8ta3UsIFRva3lvIDE1Mi04NTUwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iTmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmcgYW5hbHlzaXMgb2YgbGFnZXIgYnJld2luZyB5ZWFzdCBzdHJhaW5zIHJldmVhbHMgdGhlIGV2b2x1dGlvbmFyeSBoaXN0b3J5IG9mIGludGVyc3BlY2llcyBoeWJyaWRpemF0aW9uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjgwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN2MDM3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTYvMDIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSIgUCwgTC4gIDE5MDkgLCBNaWtyb3Nrb3Bpc2NoZSBCZXRyaWVic2tvbnRyb2xsZSBpbiBkZW4gR2FydW5nc3dlcmJlbiA2dGggZWRuLiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBwaHlzaW9sb2d5IGFuZCBmZXJtZW50YXRpb24gcGVyZm9ybWFuY2Ugb2YgU2FheiBhbmQgRnJvaGJlcmcgbGFnZXIgeWVhc3Qgc3RyYWlucyBhbmQgdGhlIHBhcmVudGFsIHNwZWNpZXMgU2FjY2hhcm9teWNlcyBldWJheWFudXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVllYXN0OyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MjU1LTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb2JlIGRvbWVzdGljYXRpb24gYW5kIHRoZSBpZGVudGlmaWNhdGlvbiBvZiB0aGUgd2lsZCBnZW5ldGljIHN0b2NrIG9mIGxhZ2VyLWJyZXdpbmcgeWVhc3Q7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTA4OyAgY2l0YXRpb25fcGFnZXM9MTQ1MzktNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2aWRlbmNlIGZvciBhIEZhciBFYXN0IEFzaWFuIG9yaWdpbiBvZiBsYWdlciBiZWVyIHllYXN0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPVIzODAtMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2luZyBhbmQgY2hhcmFjdGVyaXNhdGlvbiBvZiByZWFycmFuZ2VtZW50cyBpbiB0aHJlZSBTLiBwYXN0b3JpYW51cyBzdHJhaW5zIHJldmVhbHMgdGhlIHByZXNlbmNlIG9mIGNoaW1lcmljIGdlbmVzIGFuZCBnaXZlcyBldmlkZW5jZSBvZiBicmVha3BvaW50IHJldXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ZTkyMjAzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNlIG9mIHRoZSBsYWdlciBicmV3aW5nIHllYXN0LCBhbiBpbnRlcnNwZWNpZXMgaHlicmlkOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTExNS0yOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGljaGlhIHNvcmJpdG9waGlsYSwgYW4gaW50ZXJzcGVjaWVzIHllYXN0IGh5YnJpZCwgcmV2ZWFscyBlYXJseSBzdGVwcyBvZiBnZW5vbWUgcmVzb2x1dGlvbiBhZnRlciBwb2x5cGxvaWRpemF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HMyBHZW5lcywgR2Vub21lcywgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0yOTktMzExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIHNlcXVlbmNlIG9mIHRoZSBoaWdobHkgYWNldGljIGFjaWQtdG9sZXJhbnQgWnlnb3NhY2NoYXJvbXljZXMgYmFpbGlpLWRlcml2ZWQgaW50ZXJzcGVjaWVzIGh5YnJpZCBzdHJhaW4gSVNBMTMwNywgaXNvbGF0ZWQgZnJvbSBhIHNwYXJrbGluZyB3aW5lIHBsYW50LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0yOTktMzEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIHNlcXVlbmNlIG9mIHRoZSB3aW5lIHllYXN0IFZJTjcgcmV2ZWFscyBhbiBhbGxvdHJpcGxvaWQgaHlicmlkIGdlbm9tZSB3aXRoIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZSBhbmQgU2FjY2hhcm9teWNlcyBrdWRyaWF2emV2aWkgb3JpZ2luczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RkVNUyBZZWFzdCBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9ODgtOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuZXVwbG9pZHksIGNvcHkgbnVtYmVyIHZhcmlhdGlvbiBhbmQgdW5pcXVlIGNocm9tb3NvbWFsIHN0cnVjdHVyZXMgaW4gYm90dG9tLWZlcm1lbnRpbmcgeWVhc3QgcmV2ZWFsZWQgYnkgYXJyYXktQ0dIOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBJbnN0LiBCcmV3LjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjA7ICBjaXRhdGlvbl9wYWdlcz0yNy0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjb25zdHJ1Y3Rpb24gb2YgdGhlIGdlbm9tZSBvcmlnaW5zIGFuZCBldm9sdXRpb24gb2YgdGhlIGh5YnJpZCBsYWdlciB5ZWFzdCBTYWNjaGFyb215Y2VzIHBhc3RvcmlhbnVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTE2MTAtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluZmVyZW5jZXMgb2YgZXZvbHV0aW9uYXJ5IHJlbGF0aW9uc2hpcHMgZnJvbSBhIHBvcHVsYXRpb24gc3VydmV5IG9mIExUUi1yZXRyb3RyYW5zcG9zb25zIGFuZCB0ZWxvbWVyaWMtYXNzb2NpYXRlZCBzZXF1ZW5jZXMgaW4gdGhlIFNhY2NoYXJvbXljZXMgc2Vuc3Ugc3RyaWN0byBjb21wbGV4OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ZZWFzdDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTE3Ny05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG9zcyBvZiBsYWdlciBzcGVjaWZpYyBnZW5lcyBhbmQgc3VidGVsb21lcmljIHJlZ2lvbnMgZGVmaW5lIHR3byBkaWZmZXJlbnQgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIGxpbmVhZ2VzIGZvciBTYWNjaGFyb215Y2VzIHBhc3RvcmlhbnVzIEdyb3VwIEkgYW5kIElJIHN0cmFpbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZFTVMgWWVhc3QgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPWZvdTAwOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmljaWVudCBkZSBub3ZvIGFzc2VtYmx5IG9mIGhpZ2hseSBoZXRlcm96eWdvdXMgZ2Vub21lcyBmcm9tIHdob2xlLWdlbm9tZSBzaG90Z3VuIHNob3J0IHJlYWRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTEzODQtOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QgYW5kIGFjY3VyYXRlIHNob3J0IHJlYWQgYWxpZ25tZW50IHdpdGggQnVycm93cy1XaGVlbGVyIHRyYW5zZm9ybTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xNzU0LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HYXBwZWQgQkxBU1QgYW5kIFBTSS1CTEFTVDogYSBuZXcgZ2VuZXJhdGlvbiBvZiBwcm90ZWluIGRhdGFiYXNlIHNlYXJjaCBwcm9ncmFtczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzM4OS00MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVMuIGNlcmV2aXNpYWUgw5cgUy4gZXViYXlhbnVzIGludGVyc3BlY2lmaWMgaHlicmlkLCB0aGUgYmVzdCBvZiBib3RoIHdvcmxkcyBhbmQgYmV5b25kOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRU1TIFllYXN0IFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz1mb3YwMDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaXJjb3M6IGFuIGluZm9ybWF0aW9uIGFlc3RoZXRpYyBmb3IgY29tcGFyYXRpdmUgZ2Vub21pY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MTYzOS00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEdlbm9tZSBBbmFseXNpcyBUb29sa2l0OiBhIE1hcFJlZHVjZSBmcmFtZXdvcmsgZm9yIGFuYWx5emluZyBuZXh0LWdlbmVyYXRpb24gRE5BIHNlcXVlbmNpbmcgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0xMjk3LTMwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFNlcXVlbmNlIEFsaWdubWVudC9NYXAgZm9ybWF0IGFuZCBTQU10b29sczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yMDc4LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBIWU1MIE9ubGluZeKAlGEgd2ViIHNlcnZlciBmb3IgZmFzdCBtYXhpbXVtIGxpa2VsaWhvb2QtYmFzZWQgcGh5bG9nZW5ldGljIGluZmVyZW5jZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9NTU3LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jZSBvZiBTYWNjaGFyb215Y2VzIGNhcmxzYmVyZ2Vuc2lzLCB0aGUgd29ybGQncyBmaXJzdCBwdXJlIGN1bHR1cmUgbGFnZXIgeWVhc3Q7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUczIEdlbmVzLCBHZW5vbWVzLCBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTc4My05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hvbGUtZ2Vub21lIGNvbXBhcmlzb24gcmV2ZWFscyBub3ZlbCBnZW5ldGljIGVsZW1lbnRzIHRoYXQgY2hhcmFjdGVyaXplIHRoZSBnZW5vbWUgb2YgaW5kdXN0cmlhbCBzdHJhaW5zIG9mIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAxMjg3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hvbGUtZ2Vub21lIHNlcXVlbmNpbmcgb2Ygc2FrZSB5ZWFzdCBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWUgS3lva2FpIG5vLiA3OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTQyMy0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hvbGUtZ2Vub21lIHNlcXVlbmNpbmcgb2YgdGhlIGVmZmljaWVudCBpbmR1c3RyaWFsIGZ1ZWwtZXRoYW5vbCBmZXJtZW50YXRpdmUgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIHN0cmFpbiBDQVQtMTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBHZW5ldC4gR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mjg3OyAgY2l0YXRpb25fcGFnZXM9NDg1LTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgaW5oZXJpdGFuY2Ugb2YgbXRETkEgaW4gbGFnZXIgYnJld2luZyBzdHJhaW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRU1TIFllYXN0IFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTU4Ni05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3BvbnRhbmVvdXMgbG9zcyBvZiBoZXRlcm96eWdvc2l0eSBpbiBkaXBsb2lkIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZSBjZWxsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTU2OyAgY2l0YXRpb25fcGFnZXM9MTUzMS00ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIzLzEvNjcvMTc0NjMyMzMvZHN2MDM3LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBUaGUgbGFnZXIgYmVlciB5ZWFzdCBTYWNjaGFyb215Y2VzIHBhc3RvcmlhbnVzIGlzIGNvbnNpZGVyZWQgYW4gYWxsb3BvbHlwbG9pZCBoeWJyaWQgc3BlY2llcyBiZXR3ZWVuIFMuIGNlcmV2aXNpYWUgYW5kIFMuIGV1YmF5YW51cy4gTWFueSBTLiBwYXN0b3JpYSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MjQ2MjQ3MiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRTmfJvelTfGLozMXnls/Ia7gYqQA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f1%2f67%2f2462472" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f1%2f67%2f2462472" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=83835" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/1/1/m_cover.gif?Expires=2147483647&amp;Signature=KARGGT3434AhJvd0livqaaljLmMbBUo3zfp1TqsESiO3YlY~tkN7g3APCzu-WSs6e8Nl3TqpcrmHNHH0dQCdcITSOeE~aAPITj~eX0Yge-o3-8ykKxYb~OvG6Tl68sbE8kb9MDj8-DVfaGgSNHoeCfxrY9GfQxxlAheNwGlc1dEbYs6TpQ1qqETGqeV3fGp~5ePhsu4hP22hUcJ1Z06TJ5-2W4IJPC8QUHrse0jdAO7iAQAiMxLNxFuiLgIQ4xVlRKAVyu8pGxiw2RvdR2ib~ISf7YQpT2ikZjdevDSe5J5xnrmr4aL9l8xCPdqY1Y38fIiBcp0jIC7kA0-uvGAJYQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/1">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87200183">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200183">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200185">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200185">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200191">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200191">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200213">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200213">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200257">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200257">4. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200259">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200259">5. Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200261">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200261">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200263">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200263">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200265">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200265">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/1/53/2462457">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Next-generation sequencing analysis of lager brewing yeast strains reveals the evolutionary history of interspecies hybridization
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Miki Okuno</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biological Information</div>, <div class="insititution">Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology</div>, <div class="addr-line">2-12-1 Ookayama, Meguro-ku, Tokyo 152-8550</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Miki+Okuno">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Okuno M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Okuno M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Miki Okuno</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rei Kajitani</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biological Information</div>, <div class="insititution">Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology</div>, <div class="addr-line">2-12-1 Ookayama, Meguro-ku, Tokyo 152-8550</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rei+Kajitani">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kajitani R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kajitani R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Rei Kajitani</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rie Ryusui</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biological Information</div>, <div class="insititution">Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology</div>, <div class="addr-line">2-12-1 Ookayama, Meguro-ku, Tokyo 152-8550</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rie+Ryusui">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ryusui R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ryusui R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Rie Ryusui</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroya Morimoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biological Information</div>, <div class="insititution">Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology</div>, <div class="addr-line">2-12-1 Ookayama, Meguro-ku, Tokyo 152-8550</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroya+Morimoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Morimoto H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Morimoto H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiroya Morimoto</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yukiko Kodama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Suntory Global Innovation Center Limited</div>, <div class="addr-line">8-1-1 Seikadai, Seika-cho, Soraku-gun, Kyoto 619-0284</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yukiko+Kodama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kodama Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kodama Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yukiko Kodama</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takehiko Itoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biological Information</div>, <div class="insititution">Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology</div>, <div class="addr-line">2-12-1 Ookayama, Meguro-ku, Tokyo 152-8550</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 3-5734-3430. Fax. +81 3-5734-3630. E-mail: <a href="mailto:takehiko@bio.titech.ac.jp" target="_blank">takehiko@bio.titech.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takehiko+Itoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Itoh T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Itoh T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takehiko Itoh</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Dr Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 1, 1 February 2016, Pages 67–80, <a href='https://doi.org/10.1093/dnares/dsv037'>https://doi.org/10.1093/dnares/dsv037</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">04 January 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">02 August 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">19 November 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2462472" href="/dnaresearch/article-pdf/23/1/67/17463233/dsv037.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Miki Okuno, Rei Kajitani, Rie Ryusui, Hiroya Morimoto, Yukiko Kodama, Takehiko Itoh;  Next-generation sequencing analysis of lager brewing yeast strains reveals the evolutionary history of interspecies hybridization, <em>DNA Research</em>, Volume 23, Issue 1, 1 February 2016, Pages 67–80, <a href='https://doi.org/10.1093/dnares/dsv037'>https://doi.org/10.1093/dnares/dsv037</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2462472?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462472?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462472?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462472?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462472?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462472?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462472?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Next-generation%20sequencing%20analysis%20of%20lager%20brewing%20yeast%20strains%20reveals%20the%20evolutionary%20history%20of%20interspecies%20hybridization&amp;publicationDate=2016-01-04&amp;volumeNum=23&amp;issueNum=1&amp;author=Okuno%2C%20Miki%3B%20Kajitani%2C%20Rei&amp;startPage=67&amp;endPage=80&amp;contentId=10.1093%2Fdnares%2Fdsv037&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f1%2f67%2f2462472" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87200183 id="87200183" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The lager beer yeast <em>Saccharomyces pastorianus</em> is considered an allopolyploid hybrid species between <em>S. cerevisiae</em> and <em>S. eubayanus</em>. Many <em>S. pastorianus</em> strains have been isolated and classified into two groups according to geographical origin, but this classification remains controversial. Hybridization analyses and partial PCR-based sequence data have indicated a separate origin of these two groups, whereas a recent intertranslocation analysis suggested a single origin. To clarify the evolutionary history of this species, we analysed 10 <em>S. pastorianus</em> strains and the <em>S. eubayanus</em> type strain as a likely parent by Illumina next-generation sequencing. In addition to assembling the genomes of five of the strains, we obtained information on interchromosomal translocation, ploidy, and single-nucleotide variants (SNVs). Collectively, these results indicated that the two groups of strains share <em>S. cerevisiae</em> haploid chromosomes. We therefore conclude that both groups of <em>S. pastorianus</em> strains share at least one interspecific hybridization event and originated from a common parental species and that differences in ploidy and SNVs between the groups can be explained by chromosomal deletion or loss of heterozygosity.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;lager beer yeast&quot;">lager beer yeast</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;interspecies hybrid&quot;">interspecies hybrid</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Saccharomyces pastorianus&quot;"><em>Saccharomyces pastorianus</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;loss of heterozygosity&quot;">loss of heterozygosity</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="allopolyploid">allopolyploid</a></div>                            <h2 scrollto-destination=87200185 id="87200185" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Bottom-fermenting strains of brewing yeast—also known as lager beer yeast—represent a large portion of the beer market and have therefore been the focus of mycological research for centuries. Lager beer yeasts have been taxonomically classified as <em>Saccharomyces pastorianus</em> since the 19th century and are classified into two groups according to geographical origin: the Saaz type (Group 1) originally used in Bohemia and the Frohberg type (Group 2) in Germany.<sup><a reveal-id="DSV037C1" data-open="DSV037C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSV037C2" data-open="DSV037C2" class="link link-ref link-reveal xref-bibr">2</a></sup></p><p>Many pure strains have since been identified, including the type strains <em>S. monacensis</em> (CBS1503; Group 1), <em>S. carlsbergensis</em> (CBS1513; Group 1), and <em>S. pastorianus</em> (CBS1538; Group 1) as well as Weihenstephan 34/70 (Group 2), a widely used industrial strain. <em>Saccharomyces pastorianus</em> is an allopolyploid hybrid of two <em>Saccharomyces</em> species: the ale beer yeast <em>S. cerevisiae</em> and a hypothesized novel species similar to <em>S. bayanus</em>. In 2011, <em>S. eubayanus</em><sup><a reveal-id="DSV037C3" data-open="DSV037C3" class="link link-ref link-reveal xref-bibr">3</a></sup> was discovered on southern beech trees in Patagonia, South America, and identified as the absent parental species of lager beer yeast. <em>Saccharomyces eubayanus</em> has also recently been identified on trees in Tibet, Far East Asia, spurring a debate about the precise origin of this species.<sup><a reveal-id="DSV037C4" data-open="DSV037C4" class="link link-ref link-reveal xref-bibr">4</a></sup></p><p>Genomic analysis techniques, such as array comparative genome hybridization (array-CGH), have identified a difference in ploidy between the two groups of <em>S. pastorianus</em> strains. Group 1 (Saaz type) strains are approximately haploid for the <em>S. cerevisiae</em> genome, whereas Group 2 (Frohberg type) strains are diploid for the <em>S. cerevisiae</em> genome.<sup><a reveal-id="DSV037C5" data-open="DSV037C5" class="link link-ref link-reveal xref-bibr">5</a></sup> A Sanger sequencing-based genome sequence analysis of <em>S. pastorianus</em> Weihenstephan 34/70<sup><a reveal-id="DSV037C6" data-open="DSV037C6" class="link link-ref link-reveal xref-bibr">6</a></sup> conducted by our group revealed various genetic features, such as whole-genome structure and genomic divergence, at the single-nucleotide level. The genomes of many interspecies hybrid strains have also been sequenced, including the osmotolerant yeast species <em>Pichia sorbitolphila</em>,<sup><a reveal-id="DSV037C7" data-open="DSV037C7" class="link link-ref link-reveal xref-bibr">7</a></sup> the <em>Zygosaccharomyces bailii</em> hybrid strain ISA1307,<sup><a reveal-id="DSV037C8" data-open="DSV037C8" class="link link-ref link-reveal xref-bibr">8</a></sup> and the wine yeast strain VIN7.<sup><a reveal-id="DSV037C9" data-open="DSV037C9" class="link link-ref link-reveal xref-bibr">9</a></sup> As in our previous study of <em>S. pastorianus</em>,<sup><a reveal-id="DSV037C6" data-open="DSV037C6" class="link link-ref link-reveal xref-bibr">6</a></sup> most analyses of allopolyploid species genomes have been performed without the genome sequence of one or both parental species. For example, in the case of <em>P. sorbitolphila</em>,<sup><a reveal-id="DSV037C7" data-open="DSV037C7" class="link link-ref link-reveal xref-bibr">7</a></sup> neither parental species has been isolated; its sequence data are divided into two types based on the GC content of the expected parental species. The lack of information on one or both parental genomes has complicated downstream genomic analyses. Complete (or near-complete) genomes of parental species can provide more information on genomic variation in interspecies hybrids and facilitate the identification of translocation breakpoints between interparental homologous chromosomes at the single-nucleotide level, genome rearrangements, and differences in ploidy.</p><p>In <em>S. pastorianus</em>, ploidy differences, copy number variations, and single-nucleotide variants (SNVs) identified by array-CGH<sup><a reveal-id="DSV037C10" data-open="DSV037C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSV037C11" data-open="DSV037C11" class="link link-ref link-reveal xref-bibr">11</a></sup> or other approaches have suggested that the Saaz (Group 1) and Frohberg (Group 2) types originated from independent hybridization events,<sup><a reveal-id="DSV037C11  DSV037C13" data-open="DSV037C11  DSV037C13" class="link link-ref link-reveal xref-bibr">11–13</a></sup> a theory that has gained broad acceptance. In contrast, the recent confirmation of two breakpoints associated with interchromosomal translocation of <em>HSP82</em> and <em>KEM1</em> genes implies a common origin.<sup><a reveal-id="DSV037C5" data-open="DSV037C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Therefore, the evolutionary history of <em>S. pastorianus</em> remains controversial, primarily because of the lack of reference genome information for <em>S. eubayanus</em>. For example, array-CGH analysis was performed based on the <em>S. cerevisiae</em> S288C and <em>S. uvarum</em> CBS7001 genomes,<sup><a reveal-id="DSV037C11" data-open="DSV037C11" class="link link-ref link-reveal xref-bibr">11</a></sup> which are similar to non-Sc type <em>S. pastorianus</em>, but it was not possible to estimate differences in ploidy or sequence variations between <em>S. pastorianus</em> strains from the results.</p><p>In the present study, we obtained the whole-genome sequences of 10 <em>S. pastorianus</em> strains by Illumina next-generation sequencing (NGS) to gain insight into the evolutionary history of this species. The strains included five strains each from Groups 1 and 2, including three type strains [<em>S. monacensis</em> (CBS1503), <em>S. carlsbergensis</em> (CBS1513), and <em>S. pastorianus</em> (CBS1538)] and <em>S. pastorianus</em> Weihenstephan 34/70 (W34/70), which we sequenced previously by the Sanger method.<sup><a reveal-id="DSV037C6" data-open="DSV037C6" class="link link-ref link-reveal xref-bibr">6</a></sup> NGS technology enables more accurate sequencing and produces longer continuous genome sequences. In addition to paired-end sequences, long insert mate-pair libraries were also obtained for the three type strains and W34/70 and were applied to <em>de novo</em> assemblies to obtain overall genomic structures, which were then compared. However, as in the case of previous studies of interspecies hybrid strains,<sup><a reveal-id="DSV037C7  DSV037C9" data-open="DSV037C7  DSV037C9" class="link link-ref link-reveal xref-bibr">7–9</a></sup> a more detailed analysis of ploidy and evolutionary relationships based on sequence differences from the assembled sequences was difficult. To precisely determine the sequence differences between strains, we adopted mapping-based analysis using the two parent genomes as reference sequences, including the available <em>S. cerevisiae</em> (Sc) (S288C) genome. Because a reference sequence was not available for <em>S. eubayanus</em> (Se), we assembled the draft genome sequence of <em>S. eubayanus</em> CBS12357 using NGS data. These two reference genomes were used for mapping-based genome comparisons among the 10 <em>S. pastorianus</em> strains that considered the relatively high heterozygosity of <em>S. pastorianus</em>. Here, heterozygosity refers to the sequence difference between intrahomologous (Sc/Sc or Se/Se) chromosomes; loss of heterozygosity (LOH) refers to the loss of heterogeneity between intrahomologous chromosomes. The results provide information on ploidy, novel chromosomal translocations, mitochondrial (mt)DNA sequences, and the phylogenetic relationships among Sc and Se types that provides new insights into the origin and evolutionary history of <em>S. pastorianus</em>.</p>                            <h2 scrollto-destination=87200191 id="87200191" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87200192 id="87200192" class="section-title" data-legacy-id=s2a>2.1. Strain and sequence information</h3>
<p>The following strains were used in this study: <em>S. pastorianus</em> CBS1503 (<em>S. monacensis</em> type strain), CBS1513 (<em>S. carlsbergensis</em> type strain), CBS1538 (<em>S. pastorianus</em> type strain), CBS1174, CBS2440, Weihenstephan 34/70 (W34/70), CBS1483, CBS1484, CBS2156, and CBS5832; <em>S. eubayanus</em> CBS12357 (type strain) and BaiFY1; <em>S. bayanus</em> NBRC1948 and CBS380; and <em>S. cerevisiae</em> S288C. <em>Saccharomyces pastorianus</em> Weihenstephan 34/70 was provided by Fachhochschule Weihenstephan (Freising, Germany); the other strains, except <em>S. eubayanus</em> BaiFY1, were obtained from the CBS-KNAW Culture Collection Center (Utrecht, The Netherland). Sequence data for <em>S. eubayanus</em> BaiFY1 were downloaded from the Sequence Read Archive, NCBI.</p>                            <h3 scrollto-destination=87200194 id="87200194" class="section-title" data-legacy-id=s2b>2.2. Genome sequencing and assembly</h3>
<p>We prepared paired-end sequencing data with the Illumina platform (Hayward, CA, USA) for 14 strains of three species: <em>S. pastorianus</em> Group 1 (CBS1503, CBS1513, CBS1538, CBS1174, and CBS2440) and Group 2 (W34/70, CBS1483, CBS1484, CBS2156, and CBS5832); <em>S. eubayanus</em> (CBS12357 and BaiFY1); and <em>S. bayanus</em> (NBRC1948 and CBS380). With the exception of <em>S. eubayanus</em> BaiFY1, all sequence data were determined in this study by Illumina Miseq. <em>Saccharomyces bayanus</em> sequences were used only for mtDNA analysis. Libraries (insert size = 600 bp) were prepared using the TruSeq PCR-free DNA Sample Prep kit (Illumina) with a read length of 300 bp. Other sequence data for <em>S. eubayanus</em> BaiFY1 obtained from the Hiseq2000 platform with 151-bp paired-end libraries were available from the Sequence Read Archive (accession no. SRX646335). For <em>de novo</em> assembly, mate-pair libraries for CBS1503, CBS1513, CBS1538, W34/70, and CBS12357 were prepared using the Nextera Mate Pair Sample Prep kit (Illumina) and 3,000- to 15,000-bp fragments extracted from an agarose gel for each sample. All libraries were also sequenced using Illumina Miseq. Low-quality regions and adaptor sequences in the reads were trimmed with Platanus_trim v.1.0.7 for subsequent analyses.</p><p>The genomes of five strains sequenced from the paired-end and mate-pair libraries were assembled using Platanus v.1.2.4 software.<sup><a reveal-id="DSV037C14" data-open="DSV037C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Contig assembly was performed using only paired-end libraries, and scaffolding and gap closing were performed using both libraries. Prior to scaffolding, mate-pair reads were mapped on contigs using the BWA program<sup><a reveal-id="DSV037C15" data-open="DSV037C15" class="link link-ref link-reveal xref-bibr">15</a></sup> to remove duplicate reads generated during PCR amplification. Contig assembly for interspecies hybrid strains (CBS1503, CBS1513, CBS1538, and W34/70) was conducted with parameter −n 15 (for W34/70) or 20 (for other strains), and scaffolding was performed with parameter −u 0 to avoid removing low-coverage scaffolds derived from haploid chromosomes. The results of the assembly were evaluated by confirming the physical coverage of 6-kb mate-pair reads. Finally, contamination and mtDNA sequences were removed from scaffolds ≥500 bp by alignment with NCBI Bacteria DB, RefSeq viral DB, and BLASTN<sup><a reveal-id="DSV037C16" data-open="DSV037C16" class="link link-ref link-reveal xref-bibr">16</a></sup> for mtDNA sequences (minimum identity = 90% and minimum query coverage = 50%). The assembly of mitochondrial genomes is described below. CBS12357 was assembled using Platanus v.1.2.4 with default settings.</p><p>To construct a CBS12357 draft genome for hybrid strains in subsequent analyses, scaffolds were aligned with the <em>S. cerevisiae</em> S288C complete genome sequence using BLASTN. Only one locus, covering the rDNA region in chromosome XII, was connected by <em>N</em>-runs sequences manually, and two known interchromosomal translocations<sup><a reveal-id="DSV037C6" data-open="DSV037C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSV037C17" data-open="DSV037C17" class="link link-ref link-reveal xref-bibr">17</a></sup> (namely, II–IV and VIII–XV) that were structurally distinct from S288C were not divided and maintained. Finally, we obtained super-scaffolds that were adequate to call chromosome-level reference sequences. To estimate the genome size of the hybrid strains, paired-end reads were mapped to the parental species' genomes (S288C complete and CBS12357 draft genomes) and to the mtDNA of each strain, and the length of regions covering ≥10 reads was calculated.</p>                            <h3 scrollto-destination=87200198 id="87200198" class="section-title" data-legacy-id=s2c>2.3. Detection of interchromosomal translocations in hybrid strains</h3>
<p>Interchromosomal translocations between Sc and Se types were detected based on mate-pair reads, which were mapped to the reference (<em>S. cerevisiae</em> S288C complete and <em>S. eubayanus</em> CBS12357 draft) genomes using BWA after removing low-identity and multihit reads. Chromosomes were divided into 10,000-bp blocks, and the number of mate-pair reads linking the blocks was tabulated. Links consisting of ≥100 pairs between Sc- and Se-type blocks were displayed in a circus plot.<sup><a reveal-id="DSV037C18" data-open="DSV037C18" class="link link-ref link-reveal xref-bibr">18</a></sup> To reveal breakpoints at the nucleotide level, local assembly was performed using paired-end reads; when one of these was mapped near a breakpoint (as estimated by the procedure described above) and the opposite side was not mapped, the read corresponding to the latter was identified as a candidate covering the breakpoint region. These reads were collected and assembled.</p>                            <h3 scrollto-destination=87200200 id="87200200" class="section-title" data-legacy-id=s2d>2.4. Estimation of ploidy in hybrid strains</h3>
<p>Sequence coverage for the 10 <em>S. pastorianus strains</em> was calculated based on the number of reads mapped to the two parental species [<em>S. cerevisiae</em> (S288C) and <em>S. eubayanus</em> (CBS12357)] to confirm the accuracy of the total assembly size and estimate the ploidy of each chromosome. Sequence reads were mapped to the reference genomes using BWA. Mapped reads with &lt;90% identity or with multiple hits were removed. Following realignment using GATK realigner,<sup><a reveal-id="DSV037C19" data-open="DSV037C19" class="link link-ref link-reveal xref-bibr">19</a></sup> sequence coverage at each locus was calculated using SAMtools mpileup.<sup><a reveal-id="DSV037C20" data-open="DSV037C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Ploidy was estimated as the ratio of the sequence coverage of the focus region to the haploid coverage obtained from the whole-genome coverage distribution. The moving average of the calculated ploidy (window size = 10,000 bp and step size = 1,000 bp) was then plotted.</p>                            <h3 scrollto-destination=87200202 id="87200202" class="section-title" data-legacy-id=s2e>2.5. SNV calling in 10 <em>S. pastorianus</em> strains</h3>
<p>Mapping of paired-end reads and filtering were conducted as described above, and SNV calling was performed using SAMtools mpileup and an original Perl script that calculated the number of mapped reads, their strands, variants, and allele frequencies (AFs; i.e. the ratio of reads supporting the variant to mapped reads at the locus) from the mpileup format. SNVs were identified according to the following criteria: ≥20 reads were mapped to the locus; variants were supported by two or more mapped reads on both the forward and reverse strands; and insertion/deletion (InDel) mutations were excluded. SNVs were classified as either homo (0.8 ≤ AF) or hetero (0.2 ≤ AF &lt; 0.8) type.</p>                            <h3 scrollto-destination=87200204 id="87200204" class="section-title" data-legacy-id=s2f>2.6. Phylogenetic analysis</h3>
<p>Phylogenetic analysis of the 10 <em>S. pastorianus</em> strains and 2 parental species was performed based on SNV sites for which ≥20 reads were mapped in all strains, excluding InDel mutations. Analyses were conducted separately for Sc and Se types to identify differences between the types. Heterozygous sites were also considered, and each allele at each SNV site was treated separately. Phylogenetic estimates were calculated by the maximum likelihood approach using phyml (−b 1,000),<sup><a reveal-id="DSV037C21" data-open="DSV037C21" class="link link-ref link-reveal xref-bibr">21</a></sup> and phylogenetic trees were generated using FigTree (<a class="link link-uri" href="http://tree.bio.ed.ac.uk/software/figtree/">http://tree.bio.ed.ac.uk/software/figtree/</a>).</p>                            <h3 scrollto-destination=87200206 id="87200206" class="section-title" data-legacy-id=s2g>2.7. Assignment of hetero-SNVs and haplotype phasing</h3>
<p>To build two intrahomologous chromosomes from the diploid Sc-type chromosomes in Group 2 strains, the linkage relationships of adjacent hetero-SNVs were solved by pair reads mapped on a region bearing two or more hetero-SNVs. Regions in which pair reads solved the linkage relationships of consecutive adjacent hetero-SNVs were divided into two lines corresponding to chromosomal haplotypes and were defined as continuous haplotype-phased blocks. Blocks were divided by unphased (mainly caused by long no-hetero-SNVs) regions. W34/70 sequence reads were used in this analysis for the following reasons: the Sc-type genome of W34/70 has the highest heterozygosity among the five Group 2 strains; and mate-pair data (insert size 3 and 6 kb) were available. A Group 1 consensus sequence was constructed based on the major alleles in the five Group 1 strains. If the major allele was undefined at a locus, the locus was removed from comparison with W34/70 haplotypes.</p>                            <h3 scrollto-destination=87200208 id="87200208" class="section-title" data-legacy-id=s2h>2.8. Phylogenetic analysis based on haplotype-phased chromosomal SNVs</h3>
<p>The identities between each haplotype W34/70 sequence and Group 1 consensus sequences were calculated at each block. The W34/70 haplotyped chromosomes with higher homology to Group 1 were labelled as ‘W34/70_a’, and the others were labelled as ‘W34/70_b’. Finally, the identities between Group 1 vs. W34/70_a and Group 1 vs. W34/70_b were aggregated. For comparison, W34/70 artificial intrahomologous chromosomes were virtually constructed by randomly shuffling allele nucleotides and ignoring linkage information at every locus. The obtained artificial chromosomes were divided into two groups, W34/70_a and W34/70_b, according to the same methods for real intrahomologous chromosomes. This ‘shuffling test’ was repeated 100 times, and the average was calculated.</p>                            <h3 scrollto-destination=87200210 id="87200210" class="section-title" data-legacy-id=s2i>2.9. Assembly and phylogenetic analysis of mtDNA</h3>
<p>MtDNA assembly and phylogenetic analysis were performed for nine strains of four species, i.e. <em>S. cerevisiae</em> (S288C), <em>S. pastorianus</em> (CBS1503, CBS1513, CBS1538, and W34/70), <em>S. eubayanus</em> (CBS12357 and BaiFY1), and <em>S. bayanus</em> (NBRC1948 and CBS380). MtDNA sequences (with the exception of S288C) were assembled using only high-frequency <em>k</em>-mers (i.e. &gt;2-fold higher than the average <em>k</em>-mer frequency value obtained from sequence reads) because the copy number of mitochondria was much higher than that of chromosomes. After gap closing, scaffolds were aligned to <em>S. cerevisiae</em> S288C mtDNA using BLASTN and assembled with paired-end reads that were mapped to the edges of the scaffold to generate a circle.</p><p>The mtDNA of each strain was aligned with the open reading frames (ORFs) of <em>S. cerevisiae</em> S288C mtDNA using BLASTX to identify the ORFs of each strain. Multiple alignment with the ORFs was performed using ClustalW2. Phylogenetic estimates using the maximum likelihood method were obtained with phyml (−b 1,000), and a phylogenetic tree was generated using FigTree.</p>                            <h2 scrollto-destination=87200213 id="87200213" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=87200214 id="87200214" class="section-title" data-legacy-id=s3a>3.1. Genome sequencing</h3>
<p>The sequence sizes of each strain and library are presented in Table <a reveal-id="DSV037TB1" data-open="DSV037TB1" class="link link-reveal link-table xref-fig">1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv037_Supplementary_Data.zip?Expires=2147483647&Signature=TZn0N7h5~tgToQYT8xv52vTDsAS7RmRGsX8MFEc1uGykA0xtU8Y8Q47uIHEcB2odcIw3pk-fFfxaeW9AZDI0g2ENXJ9kQ8PFhfEiL~30bZk6EM8v6JHgz61ASkfbFS0oQRQDf1sIPANxIRcS6rMZfYgW9nSZ5jANdw72pHDijzsg2RaEuNcq-fPH~SrC1ZiXpzyoxDEXpnBjfgeznOVic37wKpVqA8xYPzyK8JzGFJwicJUJitpqbhMIiQ4lOPNoRcjq-UILR6AdTAIaGHupJNKrWfh~blOTsc3UGuebfB4-0KdbLxmGapsNbnaaghNuQSUqM~DKduF8-OAve~o3KQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>; 32-mer frequency distributions were calculated from the sequence reads of five strains each of <em>S. pastorianus</em> Groups 1 and 2 and of <em>S. eubayanus</em> CBS12357 (Fig. <a reveal-id="DSV037F1" data-open="DSV037F1" class="link link-reveal link-table xref-fig">1</a>A–C). In haploid or low-heterozygosity diploid cases, the <em>k-</em>mer frequency distribution exhibited a single peak at the average sequence frequency, as observed for <em>S. eubayanus</em> CBS12357 (Fig. <a reveal-id="DSV037F1" data-open="DSV037F1" class="link link-reveal link-table xref-fig">1</a>C). In contrast, most of the 32-mer distributions for <em>S. pastorianus</em> strains had two or more peaks (Fig. <a reveal-id="DSV037F1" data-open="DSV037F1" class="link link-reveal link-table xref-fig">1</a>A and B), which may have resulted from the aneuploidy of hybrid strains. Based on the previous array-CGH-based analysis,<sup><a reveal-id="DSV037C11" data-open="DSV037C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Group 1 strains are mainly derived from haploid <em>S. cerevisiae</em> and diploid <em>S. eubayanus</em>, and the left and right peaks in the double peak correspond to sequences originating from these two species, respectively. However, the presence of three or more peaks implies that some chromosomal regions are triploid or polyploid for parental strain chromosomes. Similarly, Group 2 strains are considered to be mainly from diploid <em>S. cerevisiae</em> and diploid <em>S. eubayanus</em>, yet the 32-mer distribution did not exhibit a pure single peak and, indeed, displayed double peaks in some strains (CBS1483) or low peaks resembling shoulders occurring at approximately half the frequency of the highest peak (W34/70, CBS1484, and CBS5832). This irregular distribution may have been due to heterozygosity in addition to polyploidy. The existence of SNVs between homologous chromosomes creates different 32-mers derived from each haplotype, and highly heterozygous genomes cause significant peaks at half the frequency of diploid peaks. The 32-mer distribution graph confirmed that <em>S. pastorianus</em> strains exhibit complex polyploidy or heterozygosity. The left-most peaks derived from the haploid genome range from 30 to 70, suggesting that the number of sequence reads for the hybrid strains was adequate. The total sequence size of the Illumina paired-end reads for <em>S. eubayanus</em> CBS12357 was 1.4 Gb with an estimated genome coverage of 120-fold, whereas that of the mate-pair reads was 2.8 Gb, with insert sizes of 3–15 kb. </p>                        <a id="87200216" scrollto-destination="87200216"></a>
<div content-id="DSV037TB1" class="table-modal"><div class="table-wrap"><div id="DSV037TB1" class="table-wrap-title" data-id="DSV037TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Strains used in this study and total sequence sizes (Mb)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Strains </th><th colspan="5">Libraries (target insert size, bp)<hr /> </th><th rowspan="2">Remarks </th></tr><tr><th>Paired end (600) </th><th>Mate pair (3k) </th><th>Mate pair (6k) </th><th>Mate pair (10k) </th><th>Mate pair (15k) </th></tr></thead><tbody><tr><td colspan="7"><em>S. pastorianus</em> </td></tr><tr><td colspan="7"> Group 1 </td></tr><tr><td>  CBS1503 </td><td>5,975.6 </td><td>913.6 </td><td>773.1 </td><td>— </td><td>— </td><td><em>S. monacensis</em> type strain </td></tr><tr><td>  CBS1513 </td><td>5,231.5 </td><td>882.9 </td><td>923.5 </td><td>— </td><td>— </td><td><em>S. carlsbergensis</em> type strain </td></tr><tr><td>  CBS1538 </td><td>5,400.5 </td><td>915.3 </td><td>766.3 </td><td>— </td><td>— </td><td><em>S. pastorianus</em> type strain </td></tr><tr><td>  CBS1174 </td><td>3,277.6 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td>  CBS2440 </td><td>3,126.8 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td colspan="7"> Group 2 </td></tr><tr><td>  W34/70 </td><td>3,947.7 </td><td>916.0 </td><td>824.2 </td><td>— </td><td>— </td><td>Widely used industrial strain </td></tr><tr><td>  CBS1483 </td><td>2,388.4 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td>  CBS1484 </td><td>2,559.7 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td>  CBS2156 </td><td>2,493.8 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td>  CBS5832 </td><td>3,113.3 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td colspan="7"><em>S. eubayanus</em> </td></tr><tr><td> CBS12357 </td><td>1,408.3 </td><td>670.1 </td><td>748.0 </td><td>719.0 </td><td>680.5 </td><td><em>S. eubayanus</em> type strain </td></tr><tr><td> BaiFY1 </td><td>1,570.3 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td colspan="7"><em>S. bayanus</em> </td></tr><tr><td> NBRC1948 </td><td>1,343.6 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td> CBS380 </td><td>3,778.9 </td><td>— </td><td>— </td><td>— </td><td>— </td><td><em>S. bayanus</em> type strain </td></tr></tbody></table></div><div content-id="DSV037TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Strains </th><th colspan="5">Libraries (target insert size, bp)<hr /> </th><th rowspan="2">Remarks </th></tr><tr><th>Paired end (600) </th><th>Mate pair (3k) </th><th>Mate pair (6k) </th><th>Mate pair (10k) </th><th>Mate pair (15k) </th></tr></thead><tbody><tr><td colspan="7"><em>S. pastorianus</em> </td></tr><tr><td colspan="7"> Group 1 </td></tr><tr><td>  CBS1503 </td><td>5,975.6 </td><td>913.6 </td><td>773.1 </td><td>— </td><td>— </td><td><em>S. monacensis</em> type strain </td></tr><tr><td>  CBS1513 </td><td>5,231.5 </td><td>882.9 </td><td>923.5 </td><td>— </td><td>— </td><td><em>S. carlsbergensis</em> type strain </td></tr><tr><td>  CBS1538 </td><td>5,400.5 </td><td>915.3 </td><td>766.3 </td><td>— </td><td>— </td><td><em>S. pastorianus</em> type strain </td></tr><tr><td>  CBS1174 </td><td>3,277.6 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td>  CBS2440 </td><td>3,126.8 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td colspan="7"> Group 2 </td></tr><tr><td>  W34/70 </td><td>3,947.7 </td><td>916.0 </td><td>824.2 </td><td>— </td><td>— </td><td>Widely used industrial strain </td></tr><tr><td>  CBS1483 </td><td>2,388.4 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td>  CBS1484 </td><td>2,559.7 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td>  CBS2156 </td><td>2,493.8 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td>  CBS5832 </td><td>3,113.3 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td colspan="7"><em>S. eubayanus</em> </td></tr><tr><td> CBS12357 </td><td>1,408.3 </td><td>670.1 </td><td>748.0 </td><td>719.0 </td><td>680.5 </td><td><em>S. eubayanus</em> type strain </td></tr><tr><td> BaiFY1 </td><td>1,570.3 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td colspan="7"><em>S. bayanus</em> </td></tr><tr><td> NBRC1948 </td><td>1,343.6 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td> CBS380 </td><td>3,778.9 </td><td>— </td><td>— </td><td>— </td><td>— </td><td><em>S. bayanus</em> type strain </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200216" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV037TB1" class="table-wrap-title" data-id="DSV037TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Strains used in this study and total sequence sizes (Mb)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Strains </th><th colspan="5">Libraries (target insert size, bp)<hr /> </th><th rowspan="2">Remarks </th></tr><tr><th>Paired end (600) </th><th>Mate pair (3k) </th><th>Mate pair (6k) </th><th>Mate pair (10k) </th><th>Mate pair (15k) </th></tr></thead><tbody><tr><td colspan="7"><em>S. pastorianus</em> </td></tr><tr><td colspan="7"> Group 1 </td></tr><tr><td>  CBS1503 </td><td>5,975.6 </td><td>913.6 </td><td>773.1 </td><td>— </td><td>— </td><td><em>S. monacensis</em> type strain </td></tr><tr><td>  CBS1513 </td><td>5,231.5 </td><td>882.9 </td><td>923.5 </td><td>— </td><td>— </td><td><em>S. carlsbergensis</em> type strain </td></tr><tr><td>  CBS1538 </td><td>5,400.5 </td><td>915.3 </td><td>766.3 </td><td>— </td><td>— </td><td><em>S. pastorianus</em> type strain </td></tr><tr><td>  CBS1174 </td><td>3,277.6 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td>  CBS2440 </td><td>3,126.8 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td colspan="7"> Group 2 </td></tr><tr><td>  W34/70 </td><td>3,947.7 </td><td>916.0 </td><td>824.2 </td><td>— </td><td>— </td><td>Widely used industrial strain </td></tr><tr><td>  CBS1483 </td><td>2,388.4 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td>  CBS1484 </td><td>2,559.7 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td>  CBS2156 </td><td>2,493.8 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td>  CBS5832 </td><td>3,113.3 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td colspan="7"><em>S. eubayanus</em> </td></tr><tr><td> CBS12357 </td><td>1,408.3 </td><td>670.1 </td><td>748.0 </td><td>719.0 </td><td>680.5 </td><td><em>S. eubayanus</em> type strain </td></tr><tr><td> BaiFY1 </td><td>1,570.3 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td colspan="7"><em>S. bayanus</em> </td></tr><tr><td> NBRC1948 </td><td>1,343.6 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td> CBS380 </td><td>3,778.9 </td><td>— </td><td>— </td><td>— </td><td>— </td><td><em>S. bayanus</em> type strain </td></tr></tbody></table></div><div content-id="DSV037TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Strains </th><th colspan="5">Libraries (target insert size, bp)<hr /> </th><th rowspan="2">Remarks </th></tr><tr><th>Paired end (600) </th><th>Mate pair (3k) </th><th>Mate pair (6k) </th><th>Mate pair (10k) </th><th>Mate pair (15k) </th></tr></thead><tbody><tr><td colspan="7"><em>S. pastorianus</em> </td></tr><tr><td colspan="7"> Group 1 </td></tr><tr><td>  CBS1503 </td><td>5,975.6 </td><td>913.6 </td><td>773.1 </td><td>— </td><td>— </td><td><em>S. monacensis</em> type strain </td></tr><tr><td>  CBS1513 </td><td>5,231.5 </td><td>882.9 </td><td>923.5 </td><td>— </td><td>— </td><td><em>S. carlsbergensis</em> type strain </td></tr><tr><td>  CBS1538 </td><td>5,400.5 </td><td>915.3 </td><td>766.3 </td><td>— </td><td>— </td><td><em>S. pastorianus</em> type strain </td></tr><tr><td>  CBS1174 </td><td>3,277.6 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td>  CBS2440 </td><td>3,126.8 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td colspan="7"> Group 2 </td></tr><tr><td>  W34/70 </td><td>3,947.7 </td><td>916.0 </td><td>824.2 </td><td>— </td><td>— </td><td>Widely used industrial strain </td></tr><tr><td>  CBS1483 </td><td>2,388.4 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td>  CBS1484 </td><td>2,559.7 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td>  CBS2156 </td><td>2,493.8 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td>  CBS5832 </td><td>3,113.3 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td colspan="7"><em>S. eubayanus</em> </td></tr><tr><td> CBS12357 </td><td>1,408.3 </td><td>670.1 </td><td>748.0 </td><td>719.0 </td><td>680.5 </td><td><em>S. eubayanus</em> type strain </td></tr><tr><td> BaiFY1 </td><td>1,570.3 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td colspan="7"><em>S. bayanus</em> </td></tr><tr><td> NBRC1948 </td><td>1,343.6 </td><td>— </td><td>— </td><td>— </td><td>— </td><td> </td></tr><tr><td> CBS380 </td><td>3,778.9 </td><td>— </td><td>— </td><td>— </td><td>— </td><td><em>S. bayanus</em> type strain </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200216" target="_blank">View Large</a></div></div>                        <a id="87200217" scrollto-destination="87200217"></a>
<div id="DSV037F1" data-id="DSV037F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/m_dsv03701.png?Expires=2147483647&Signature=h5RpmGDySrUZ40xWQk1aEj4WezaGxTof4zQeWglnF-eX53e8JULdP58i-eRkHrugVFDmPyLNR0CkY02UwQYtR~Q2aBSbt0ml1B7e6WwCaOxVl5tXcYhIoD~IiHb1TsXYnodBHa-K6yAPKsbhzAD7Himm~Mg082vXVO-nDlymXGYAa3Hf5XtnbpYsD3rc7E2T5d1HT5ziddrYyEomWsE1X5SCV7tdFIZtFWtVLqR5Uihss9fsJGNJyEiqGfyeElthniMmu~pcK0CxiOZLlzXhEjeAsT082aH07W6PQYO9vD2TyceB613iUk~4PbHvEGIwaE2opsIsu2WYW-L7gJIOTQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Frequency distribution of 32-mers in 10 S. pastorianus and S. eubayanus strains. Shown are the 32-mer distributions of (A) five Group 1 S. pastorianus strains; (B) five Group 2 S. pastorianus strains; and (C) the S. eubayanus strain. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03701.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200217/dsv03701.tiff" path-from-xml="dsv03701.tiff" target="_blank">View large</a><a section="87200217" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv03701.png?Expires=2147483647&Signature=fx5~YRGnbYgZxBo-3kE675vGqmRroiomWoXMdsXlVpysMKbM30lD2PsoiDm6e~PyaE-cTg8CMSy7vniJwKfmDAVP0TFPMe5iYlJUrw2f3OEn0yFJET0X-sEfGbXkprUHZDLFU9twBlEoQdK0~8c10W~3GsuTTwtlEH8PdqH067Hp5Bzs335vdxx9JjpxNnoVoeUFy4nAb~aLAohHDyfcdsurdAbyBs3210hgZc2DnMUaNWcZfZ14mTb70EWpEedXKTBh220KoDdApebmk4~UcUK7862J0TEO8Y6vEWXZZRG9HYT3ZCb5eXkfo9nsvqOr1pEjmdzSTTriJFmtAqnrpA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200217&ar=2462472&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03701.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Frequency distribution of 32-mers in 10 <em>S. pastorianus</em> and <em>S. eubayanus</em> strains. Shown are the 32-mer distributions of (A) five Group 1 <em>S. pastorianus</em> strains; (B) five Group 2 <em>S. pastorianus</em> strains; and (C) the <em>S. eubayanus</em> strain. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV037F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/m_dsv03701.png?Expires=2147483647&Signature=h5RpmGDySrUZ40xWQk1aEj4WezaGxTof4zQeWglnF-eX53e8JULdP58i-eRkHrugVFDmPyLNR0CkY02UwQYtR~Q2aBSbt0ml1B7e6WwCaOxVl5tXcYhIoD~IiHb1TsXYnodBHa-K6yAPKsbhzAD7Himm~Mg082vXVO-nDlymXGYAa3Hf5XtnbpYsD3rc7E2T5d1HT5ziddrYyEomWsE1X5SCV7tdFIZtFWtVLqR5Uihss9fsJGNJyEiqGfyeElthniMmu~pcK0CxiOZLlzXhEjeAsT082aH07W6PQYO9vD2TyceB613iUk~4PbHvEGIwaE2opsIsu2WYW-L7gJIOTQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Frequency distribution of 32-mers in 10 S. pastorianus and S. eubayanus strains. Shown are the 32-mer distributions of (A) five Group 1 S. pastorianus strains; (B) five Group 2 S. pastorianus strains; and (C) the S. eubayanus strain. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03701.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200217/dsv03701.tiff" path-from-xml="dsv03701.tiff" target="_blank">View large</a><a section="87200217" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv03701.png?Expires=2147483647&Signature=fx5~YRGnbYgZxBo-3kE675vGqmRroiomWoXMdsXlVpysMKbM30lD2PsoiDm6e~PyaE-cTg8CMSy7vniJwKfmDAVP0TFPMe5iYlJUrw2f3OEn0yFJET0X-sEfGbXkprUHZDLFU9twBlEoQdK0~8c10W~3GsuTTwtlEH8PdqH067Hp5Bzs335vdxx9JjpxNnoVoeUFy4nAb~aLAohHDyfcdsurdAbyBs3210hgZc2DnMUaNWcZfZ14mTb70EWpEedXKTBh220KoDdApebmk4~UcUK7862J0TEO8Y6vEWXZZRG9HYT3ZCb5eXkfo9nsvqOr1pEjmdzSTTriJFmtAqnrpA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200217&ar=2462472&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03701.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Frequency distribution of 32-mers in 10 <em>S. pastorianus</em> and <em>S. eubayanus</em> strains. Shown are the 32-mer distributions of (A) five Group 1 <em>S. pastorianus</em> strains; (B) five Group 2 <em>S. pastorianus</em> strains; and (C) the <em>S. eubayanus</em> strain. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87200218 id="87200218" class="section-title" data-legacy-id=s3b>3.2. Genome assembly</h3>
<p>The assembly statistics for <em>S. pastorianus</em> CBS1503, CBS1513, CBS1538, and W34/70 and <em>S. eubayanus</em> CBS12357 are presented in Table <a reveal-id="DSV037TB2" data-open="DSV037TB2" class="link link-reveal link-table xref-fig">2</a> and were, in general, very robust. For example, all <em>N</em>50 lengths exceeded 400 kb, and except in CBS1538, the longest scaffold lengths exceeded 1 Mb, comparable to the length of the longest chromosome of <em>S. cerevisiae</em> S288C (chromosome IV, 1.5 Mb). We previously reported an <em>N</em>50 length of 108 kb for W34/70,<sup><a reveal-id="DSV037C6" data-open="DSV037C6" class="link link-ref link-reveal xref-bibr">6</a></sup> only approximately one-seventh of the value observed here. In particular, the assembly for <em>S. eubayanus</em> CBS12357 yielded 17 scaffolds &gt;5,000 bp, one more than the chromosome number of 16. The <em>S. eubayanus</em> CBS12357 chromosome corresponding to chromosome XII of <em>S. cerevisiae</em> S288C was divided into two scaffolds that overlapped the rDNA region and were connected by ‘N run 5000 bp’. We ultimately obtained 16 super-scaffolds with a total genome size of 11.7 Mb—slightly smaller than that of S288C—with 30 gaps totalling 22,137 bp. Based on the total size of the scaffolds, the number of scaffolds, and the small number of gaps, the <em>S. eubayanus</em> CBS12357 draft genome was deemed adequate for use as a reference genome. </p>                        <a id="87200220" scrollto-destination="87200220"></a>
<div content-id="DSV037TB2" class="table-modal"><div class="table-wrap"><div id="DSV037TB2" class="table-wrap-title" data-id="DSV037TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Assembly statistics for five strains of two species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strains </th><th>Total length (bp) </th><th>Number of sequences </th><th><em>N</em>50 length (bp) </th><th><em>N</em>50 # </th><th>Minimum length (bp) </th><th>Maximum length (bp) </th></tr></thead><tbody><tr><td colspan="7"><em>S. pastorianus</em> </td></tr><tr><td colspan="7"> Group 1 </td></tr><tr><td>  CBS1503 </td><td>17,195,167 </td><td>631 </td><td>484,478 </td><td>12 </td><td>500 </td><td>1,060,739 </td></tr><tr><td>  CBS1513 </td><td>19,248,212 </td><td>178 </td><td>644,406 </td><td>12 </td><td>502 </td><td>1,050,489 </td></tr><tr><td>  CBS1538 </td><td>14,404,124 </td><td>277 </td><td>428,791 </td><td>13 </td><td>501 </td><td>760,567 </td></tr><tr><td colspan="7"> Group 2 </td></tr><tr><td>  W34/70 </td><td>22,500,926 </td><td>495 </td><td>723,289 </td><td>13 </td><td>500 </td><td>1,455,873 </td></tr><tr><td colspan="7"><em>S. eubayanus</em> </td></tr><tr><td rowspan="2"> CBS12357 (super-scaffolds) </td><td>11,666,993 </td><td>17 </td><td>833,488 </td><td>6 </td><td>196,426 </td><td>1,269,399 </td></tr><tr><td>(11,671,993) </td><td>16 </td><td>903,844 </td><td>6 </td><td>196,426 </td><td>1,269,399 </td></tr></tbody></table></div><div content-id="DSV037TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Strains </th><th>Total length (bp) </th><th>Number of sequences </th><th><em>N</em>50 length (bp) </th><th><em>N</em>50 # </th><th>Minimum length (bp) </th><th>Maximum length (bp) </th></tr></thead><tbody><tr><td colspan="7"><em>S. pastorianus</em> </td></tr><tr><td colspan="7"> Group 1 </td></tr><tr><td>  CBS1503 </td><td>17,195,167 </td><td>631 </td><td>484,478 </td><td>12 </td><td>500 </td><td>1,060,739 </td></tr><tr><td>  CBS1513 </td><td>19,248,212 </td><td>178 </td><td>644,406 </td><td>12 </td><td>502 </td><td>1,050,489 </td></tr><tr><td>  CBS1538 </td><td>14,404,124 </td><td>277 </td><td>428,791 </td><td>13 </td><td>501 </td><td>760,567 </td></tr><tr><td colspan="7"> Group 2 </td></tr><tr><td>  W34/70 </td><td>22,500,926 </td><td>495 </td><td>723,289 </td><td>13 </td><td>500 </td><td>1,455,873 </td></tr><tr><td colspan="7"><em>S. eubayanus</em> </td></tr><tr><td rowspan="2"> CBS12357 (super-scaffolds) </td><td>11,666,993 </td><td>17 </td><td>833,488 </td><td>6 </td><td>196,426 </td><td>1,269,399 </td></tr><tr><td>(11,671,993) </td><td>16 </td><td>903,844 </td><td>6 </td><td>196,426 </td><td>1,269,399 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200220" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV037TB2" class="table-wrap-title" data-id="DSV037TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Assembly statistics for five strains of two species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strains </th><th>Total length (bp) </th><th>Number of sequences </th><th><em>N</em>50 length (bp) </th><th><em>N</em>50 # </th><th>Minimum length (bp) </th><th>Maximum length (bp) </th></tr></thead><tbody><tr><td colspan="7"><em>S. pastorianus</em> </td></tr><tr><td colspan="7"> Group 1 </td></tr><tr><td>  CBS1503 </td><td>17,195,167 </td><td>631 </td><td>484,478 </td><td>12 </td><td>500 </td><td>1,060,739 </td></tr><tr><td>  CBS1513 </td><td>19,248,212 </td><td>178 </td><td>644,406 </td><td>12 </td><td>502 </td><td>1,050,489 </td></tr><tr><td>  CBS1538 </td><td>14,404,124 </td><td>277 </td><td>428,791 </td><td>13 </td><td>501 </td><td>760,567 </td></tr><tr><td colspan="7"> Group 2 </td></tr><tr><td>  W34/70 </td><td>22,500,926 </td><td>495 </td><td>723,289 </td><td>13 </td><td>500 </td><td>1,455,873 </td></tr><tr><td colspan="7"><em>S. eubayanus</em> </td></tr><tr><td rowspan="2"> CBS12357 (super-scaffolds) </td><td>11,666,993 </td><td>17 </td><td>833,488 </td><td>6 </td><td>196,426 </td><td>1,269,399 </td></tr><tr><td>(11,671,993) </td><td>16 </td><td>903,844 </td><td>6 </td><td>196,426 </td><td>1,269,399 </td></tr></tbody></table></div><div content-id="DSV037TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Strains </th><th>Total length (bp) </th><th>Number of sequences </th><th><em>N</em>50 length (bp) </th><th><em>N</em>50 # </th><th>Minimum length (bp) </th><th>Maximum length (bp) </th></tr></thead><tbody><tr><td colspan="7"><em>S. pastorianus</em> </td></tr><tr><td colspan="7"> Group 1 </td></tr><tr><td>  CBS1503 </td><td>17,195,167 </td><td>631 </td><td>484,478 </td><td>12 </td><td>500 </td><td>1,060,739 </td></tr><tr><td>  CBS1513 </td><td>19,248,212 </td><td>178 </td><td>644,406 </td><td>12 </td><td>502 </td><td>1,050,489 </td></tr><tr><td>  CBS1538 </td><td>14,404,124 </td><td>277 </td><td>428,791 </td><td>13 </td><td>501 </td><td>760,567 </td></tr><tr><td colspan="7"> Group 2 </td></tr><tr><td>  W34/70 </td><td>22,500,926 </td><td>495 </td><td>723,289 </td><td>13 </td><td>500 </td><td>1,455,873 </td></tr><tr><td colspan="7"><em>S. eubayanus</em> </td></tr><tr><td rowspan="2"> CBS12357 (super-scaffolds) </td><td>11,666,993 </td><td>17 </td><td>833,488 </td><td>6 </td><td>196,426 </td><td>1,269,399 </td></tr><tr><td>(11,671,993) </td><td>16 </td><td>903,844 </td><td>6 </td><td>196,426 </td><td>1,269,399 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200220" target="_blank">View Large</a></div></div><p>The assembly statistics revealed some interesting features. The total sequence size of the Group 2 W34/70 strain (22.5 Mb) was nearly equal to the sum of the <em>S. cerevisiae</em> S288C and <em>S. eubayanus</em> CBS12357 genome sizes (12.1 and 11.7 Mb, respectively). In contrast, the genome sizes of Group 1 strains—which ranged from 14.4 to 19.2 Mb—were much smaller than this value, suggesting partial or complete chromosomal deletion. Additionally, although the four <em>S. pastorianus</em> strains exhibited robust statistics (e.g. <em>N</em>50 and maximum scaffold length), the scaffold numbers were much higher than those of <em>S. eubayanus</em> CBS12357. Large <em>N</em>50 values and the considerable number of scaffolds suggest the existence of many short scaffolds, potentially due to the complexity of chromosome structure. During interchromosomal translocation, a homologous diploid or triploid chromosome may translocate to another interparental chromosome, leaving behind the remaining one or two homologous chromosomes. A junctional structure would then be present in the chromosome, and assembly results would be divided. Interchromosomal translocations have been reported,<sup><a reveal-id="DSV037C5" data-open="DSV037C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSV037C6" data-open="DSV037C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSV037C2" data-open="DSV037C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSV037C2" data-open="DSV037C2" class="link link-ref link-reveal xref-bibr">2</a></sup> and this scenario is also supported by the unusual 32<em>-</em>mer distribution (Fig. <a reveal-id="DSV037F1" data-open="DSV037F1" class="link link-reveal link-table xref-fig">1</a>). These complex chromosome structures may not be resolved by the assembly method; in the following sections, we describe our investigation of translocation structure performed by mapping mate-pair reads and the elucidation of ploidy distribution by mapping paired-end reads.</p>                            <h3 scrollto-destination=87200222 id="87200222" class="section-title" data-legacy-id=s3c>3.3. Estimation of ploidy in <em>S. pastorianus</em> strains</h3>
<p>The ploidy of the 10 <em>S. pastorianus</em> strains was estimated based on the sequence coverage of paired-end reads mapped onto the genomes of the parental species (Fig. <a reveal-id="DSV037F2" data-open="DSV037F2" class="link link-reveal link-table xref-fig">2</a>). Clear differences in ploidy between Groups 1 and 2 were observed. In the former, the Sc-type genome was haploid in most cases, and partial or complete chromosomal deletions were frequently observed; chromosomes I, IX, X, and the left arm of VII were present in these strains, and deletions of the right arm of chromosome IV, left arm of XIII, and the entire chromosome XII were common. Most Sc-type chromosomes in Group 1 were haploid or missing, whereas most Se-type chromosomes were diploid or triploid. In contrast, Group 2 Sc-type chromosomes were always haploid or diploid, whereas Se-type chromosomes ranged from haploid to triploid. Each parental type appeared aneuploid in each strain, but the sum of interhomologous chromosomes (between Sc- and Se-type) was approximately identical. Specifically, five strains in Group 1 and CBS2156 were basically triploid, and those in Group 2—except for CBS2156—were tetraploid. Despite the chromosomal deletion trails that were observed, a mechanism for maintaining a constant total number of intra- and interparental homologous chromosomes appears to exist in Sc and Se types. The differences in ploidy between the two groups provide additional evidence that both groups originated independently from the same parental species. </p>                        <a id="87200224" scrollto-destination="87200224"></a>
<div id="DSV037F2" data-id="DSV037F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/m_dsv03702.png?Expires=2147483647&Signature=lKBLIHMoAuA9KlOB-Jg7P9m0xDHnV4~5NFBOBSGwfBluzc609RcJrVkA3V0bKYfDgxMWCWcyXNWcl4ORYBnEdKRKphepMM51AtFLbtO7WUeNoJvBLvewhW38MhFzGhpna3XRgQ-~n-u3HHS6lD3Ddc5ym1a-7bK68vc0HnbqQfbJpRrOeDXS6NDpv7zXHzgc17IDC2mwJ6i4agJ8Jofj3iEnMxVfGhTVGA09l67gi4EJBzBf4HDrJAztuA7ylb2gQXG1asv5AidKNrKe1-WOuK48Vonl~~bDR0Hgi3iPHHD48DzRpuu35XJlVl30N7D0RRMGGlkfjwTzSfTZ-Wqqhw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Ploidy distribution in S. pastorianus based on sequence coverage mapped onto parental genomes for (A) Group 1 and (B) Group 2 strains. The blue and orange dots indicate the ploidy of Sc- (left) and Se- (right) type chromosomes, respectively. The vertical axis represents the estimated ploidy, which was calculated as the ratio of sequence coverage at each locus to the estimated haploid coverage; the horizontal axis represents chromosomal loci. In general, Group 1 strains had triploid genomes (haploid or non-Sc type, and diploid or triploid Se type), whereas Group 2 strains were tetraploid (haploid to triploid Sc and Se types). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03702.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200224/dsv03702.tiff" path-from-xml="dsv03702.tiff" target="_blank">View large</a><a section="87200224" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv03702.png?Expires=2147483647&Signature=mp5OPyEAUZGivAQs0daWjvZ6LcVf-F2wJXEqcIHqqdYB08oNK1FjlUHPxu8nSAvx9x46YE0sHLnYvVuBH4JwAwKrDgJQi4Ys54zBkrJlg9ifWbkGPsY1LZtX6UaG5ZqVJWTmAbKUTrRe3D1hCUjyl~SoEn9jJLPBckQXLTP~EkgVIWak9iOt02uInMX0UdsYi~AZIA7F6gTUQH3B6VTeCTJTSu-XHkcThbFcuBK8xTOg4nIzUvH6kT3toUWCFYWW0GYriAHHbLszi9Sh~65PvaOI5pHuDlE0rtKPAJNPQAksgTDc9-ekM6Eg0gGrnW9vWjU-6uZfrSxRQghSPLhuXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200224&ar=2462472&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03702.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Ploidy distribution in <em>S. pastorianus</em> based on sequence coverage mapped onto parental genomes for (A) Group 1 and (B) Group 2 strains. The blue and orange dots indicate the ploidy of Sc- (left) and Se- (right) type chromosomes, respectively. The vertical axis represents the estimated ploidy, which was calculated as the ratio of sequence coverage at each locus to the estimated haploid coverage; the horizontal axis represents chromosomal loci. In general, Group 1 strains had triploid genomes (haploid or non-Sc type, and diploid or triploid Se type), whereas Group 2 strains were tetraploid (haploid to triploid Sc and Se types). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV037F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/m_dsv03702.png?Expires=2147483647&Signature=lKBLIHMoAuA9KlOB-Jg7P9m0xDHnV4~5NFBOBSGwfBluzc609RcJrVkA3V0bKYfDgxMWCWcyXNWcl4ORYBnEdKRKphepMM51AtFLbtO7WUeNoJvBLvewhW38MhFzGhpna3XRgQ-~n-u3HHS6lD3Ddc5ym1a-7bK68vc0HnbqQfbJpRrOeDXS6NDpv7zXHzgc17IDC2mwJ6i4agJ8Jofj3iEnMxVfGhTVGA09l67gi4EJBzBf4HDrJAztuA7ylb2gQXG1asv5AidKNrKe1-WOuK48Vonl~~bDR0Hgi3iPHHD48DzRpuu35XJlVl30N7D0RRMGGlkfjwTzSfTZ-Wqqhw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Ploidy distribution in S. pastorianus based on sequence coverage mapped onto parental genomes for (A) Group 1 and (B) Group 2 strains. The blue and orange dots indicate the ploidy of Sc- (left) and Se- (right) type chromosomes, respectively. The vertical axis represents the estimated ploidy, which was calculated as the ratio of sequence coverage at each locus to the estimated haploid coverage; the horizontal axis represents chromosomal loci. In general, Group 1 strains had triploid genomes (haploid or non-Sc type, and diploid or triploid Se type), whereas Group 2 strains were tetraploid (haploid to triploid Sc and Se types). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03702.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200224/dsv03702.tiff" path-from-xml="dsv03702.tiff" target="_blank">View large</a><a section="87200224" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv03702.png?Expires=2147483647&Signature=mp5OPyEAUZGivAQs0daWjvZ6LcVf-F2wJXEqcIHqqdYB08oNK1FjlUHPxu8nSAvx9x46YE0sHLnYvVuBH4JwAwKrDgJQi4Ys54zBkrJlg9ifWbkGPsY1LZtX6UaG5ZqVJWTmAbKUTrRe3D1hCUjyl~SoEn9jJLPBckQXLTP~EkgVIWak9iOt02uInMX0UdsYi~AZIA7F6gTUQH3B6VTeCTJTSu-XHkcThbFcuBK8xTOg4nIzUvH6kT3toUWCFYWW0GYriAHHbLszi9Sh~65PvaOI5pHuDlE0rtKPAJNPQAksgTDc9-ekM6Eg0gGrnW9vWjU-6uZfrSxRQghSPLhuXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200224&ar=2462472&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03702.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Ploidy distribution in <em>S. pastorianus</em> based on sequence coverage mapped onto parental genomes for (A) Group 1 and (B) Group 2 strains. The blue and orange dots indicate the ploidy of Sc- (left) and Se- (right) type chromosomes, respectively. The vertical axis represents the estimated ploidy, which was calculated as the ratio of sequence coverage at each locus to the estimated haploid coverage; the horizontal axis represents chromosomal loci. In general, Group 1 strains had triploid genomes (haploid or non-Sc type, and diploid or triploid Se type), whereas Group 2 strains were tetraploid (haploid to triploid Sc and Se types). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87200225 id="87200225" class="section-title" data-legacy-id=s3d>3.4. Interchromosomal translocations between Sc- and Se-type genomes</h3>
<p>Interchromosomal translocations were detected by mate-pair reads in CBS1503, CBS1513, CBS1538, and W34/70 (Fig. <a reveal-id="DSV037F3" data-open="DSV037F3" class="link link-reveal link-table xref-fig">3</a>). Previous studies have detected 11, 9, and 8 Sc-/Se-type translocations in CBS1503,<sup><a reveal-id="DSV037C5" data-open="DSV037C5" class="link link-ref link-reveal xref-bibr">5</a></sup> CBS1513,<sup><a reveal-id="DSV037C5" data-open="DSV037C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSV037C2" data-open="DSV037C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSV037C2" data-open="DSV037C2" class="link link-ref link-reveal xref-bibr">2</a></sup> and W34/70,<sup><a reveal-id="DSV037C6" data-open="DSV037C6" class="link link-ref link-reveal xref-bibr">6</a></sup> respectively, and these translocations were verified by PCR amplification prior to the start of our study. Our analysis confirmed these known translocations in the three strains, demonstrating that this method is capable of detecting these events. Moreover, two translocations present at the <em>HSP82</em><sup><a reveal-id="DSV037C5" data-open="DSV037C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSV037C2" data-open="DSV037C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSV037C2" data-open="DSV037C2" class="link link-ref link-reveal xref-bibr">2</a></sup> and <em>KEM1</em> loci<sup><a reveal-id="DSV037C5" data-open="DSV037C5" class="link link-ref link-reveal xref-bibr">5</a></sup> were also confirmed in the other six <em>S. pastorianus</em> strains using paired-end sequences (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv037_Supplementary_Data.zip?Expires=2147483647&Signature=TZn0N7h5~tgToQYT8xv52vTDsAS7RmRGsX8MFEc1uGykA0xtU8Y8Q47uIHEcB2odcIw3pk-fFfxaeW9AZDI0g2ENXJ9kQ8PFhfEiL~30bZk6EM8v6JHgz61ASkfbFS0oQRQDf1sIPANxIRcS6rMZfYgW9nSZ5jANdw72pHDijzsg2RaEuNcq-fPH~SrC1ZiXpzyoxDEXpnBjfgeznOVic37wKpVqA8xYPzyK8JzGFJwicJUJitpqbhMIiQ4lOPNoRcjq-UILR6AdTAIaGHupJNKrWfh~blOTsc3UGuebfB4-0KdbLxmGapsNbnaaghNuQSUqM~DKduF8-OAve~o3KQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). The two breakpoints were completely matched at the nucleotide level among all Group 1 and 2 strains, with the exception of a chromosomal deletion (<em>HSP82</em> of CBS1538 and CBS1174). Novel interchromosomal translocations were also detected by our study—one each in CBS1503, CBS1513, and CBS1538 and two in W34/70. One of the novel translocations common to the four strains was further investigated by paired-end sequence alignment, revealing that all 10 <em>S. pastorianus</em> strains share a breakpoint within <em>ZUO1</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv037_Supplementary_Data.zip?Expires=2147483647&Signature=TZn0N7h5~tgToQYT8xv52vTDsAS7RmRGsX8MFEc1uGykA0xtU8Y8Q47uIHEcB2odcIw3pk-fFfxaeW9AZDI0g2ENXJ9kQ8PFhfEiL~30bZk6EM8v6JHgz61ASkfbFS0oQRQDf1sIPANxIRcS6rMZfYgW9nSZ5jANdw72pHDijzsg2RaEuNcq-fPH~SrC1ZiXpzyoxDEXpnBjfgeznOVic37wKpVqA8xYPzyK8JzGFJwicJUJitpqbhMIiQ4lOPNoRcjq-UILR6AdTAIaGHupJNKrWfh~blOTsc3UGuebfB4-0KdbLxmGapsNbnaaghNuQSUqM~DKduF8-OAve~o3KQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). The remaining translocation present in W34/70 spans from Sc-type chromosome VIII <em>PRP8</em> to Se-type chromosome XV–VIII (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv037_Supplementary_Data.zip?Expires=2147483647&Signature=TZn0N7h5~tgToQYT8xv52vTDsAS7RmRGsX8MFEc1uGykA0xtU8Y8Q47uIHEcB2odcIw3pk-fFfxaeW9AZDI0g2ENXJ9kQ8PFhfEiL~30bZk6EM8v6JHgz61ASkfbFS0oQRQDf1sIPANxIRcS6rMZfYgW9nSZ5jANdw72pHDijzsg2RaEuNcq-fPH~SrC1ZiXpzyoxDEXpnBjfgeznOVic37wKpVqA8xYPzyK8JzGFJwicJUJitpqbhMIiQ4lOPNoRcjq-UILR6AdTAIaGHupJNKrWfh~blOTsc3UGuebfB4-0KdbLxmGapsNbnaaghNuQSUqM~DKduF8-OAve~o3KQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). </p>                        <a id="87200227" scrollto-destination="87200227"></a>
<div id="DSV037F3" data-id="DSV037F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/m_dsv03703.png?Expires=2147483647&Signature=xM5p0gdWz~WMfan7W2yCUJBlVZ27q72VPVTPqQxskW8AX2KzZAufSll8vsDSgVpChVJSx26yrYE3X6Lo5lCqj8-mEgf1MYVNGEpsY0NDj3~1x-m1tk6jiQRAbrYYIDBwl0nNEdOdzrSdnq8xCCasgqwCV62D7zDR2zFwliQaS2~~9S4LGeEfk~gRkRX1Cf8B9nSHWShYGDDhvkEF~kXacF5Z-lg5e1hi8TL8nMv5sRPDyMjCFsZcWBlELbG38AZz03ozW8HIQQzfQqDMuWMuZyNEJRElDopu0corgBwtzVGd4eNMNx~qIc1hrTGkybWIcKcmpezlyh6FrHxBmFQMMw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Interchromosomal translocations between Sc- and Se-type genomes. The circular layouts show interchromosomal translocations between Sc and Se types detected by mate-pair links in Group 1 (CBS15103, CBS1513, and CBS1538) and Group 2 (W34/70) strains. Blue and orange bars represent Sc-type (left) and Se-type (right) chromosomes, respectively. The grey histograms within the circles represent sequence coverage, and regions without bars indicate chromosomal deletions. Grey intersecting lines indicate interchromosomal translocations supported by mate-pair reads bridging the Sc and Se types. Green lines indicate translocations shared by the two groups. Three interchromosomal translocations were common to both groups. The line thickness is proportional to the number of links supported by mate-pair reads. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03703.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200227/dsv03703.tiff" path-from-xml="dsv03703.tiff" target="_blank">View large</a><a section="87200227" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv03703.png?Expires=2147483647&Signature=Q1rC6jCfob~jkEa77NTG7-cxHNQ5XmF9zlppmQkefefzfLrnd4AJq181zEZIvWjER3~fqV9~Slj-pp5eL3guPmwY5acAO-LgcVGxhOg07F8o954wGd1j6JAWTtOEfGC7NXkqbk3KEpduUG0eDjbJO0J~1L15UaigY~1L7VFEfpA6O-4pT5lhqNukFgR6xZ9gPjMTmhNqfvTuWNRYHhBLn0~ddJT8EzYR-L8a5jdVsfp9yDl~13iirel7ZGZb6xyKy5E2WGjCL1U6BLOQFt5H9yr3S~gMUlTypnRHzJQq57jxaxHBBvFDRSU2CFzavTYLvoupfbOom7hZR~SQMSXN2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200227&ar=2462472&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03703.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Interchromosomal translocations between Sc- and Se-type genomes. The circular layouts show interchromosomal translocations between Sc and Se types detected by mate-pair links in Group 1 (CBS15103, CBS1513, and CBS1538) and Group 2 (W34/70) strains. Blue and orange bars represent Sc-type (left) and Se-type (right) chromosomes, respectively. The grey histograms within the circles represent sequence coverage, and regions without bars indicate chromosomal deletions. Grey intersecting lines indicate interchromosomal translocations supported by mate-pair reads bridging the Sc and Se types. Green lines indicate translocations shared by the two groups. Three interchromosomal translocations were common to both groups. The line thickness is proportional to the number of links supported by mate-pair reads. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV037F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/m_dsv03703.png?Expires=2147483647&Signature=xM5p0gdWz~WMfan7W2yCUJBlVZ27q72VPVTPqQxskW8AX2KzZAufSll8vsDSgVpChVJSx26yrYE3X6Lo5lCqj8-mEgf1MYVNGEpsY0NDj3~1x-m1tk6jiQRAbrYYIDBwl0nNEdOdzrSdnq8xCCasgqwCV62D7zDR2zFwliQaS2~~9S4LGeEfk~gRkRX1Cf8B9nSHWShYGDDhvkEF~kXacF5Z-lg5e1hi8TL8nMv5sRPDyMjCFsZcWBlELbG38AZz03ozW8HIQQzfQqDMuWMuZyNEJRElDopu0corgBwtzVGd4eNMNx~qIc1hrTGkybWIcKcmpezlyh6FrHxBmFQMMw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Interchromosomal translocations between Sc- and Se-type genomes. The circular layouts show interchromosomal translocations between Sc and Se types detected by mate-pair links in Group 1 (CBS15103, CBS1513, and CBS1538) and Group 2 (W34/70) strains. Blue and orange bars represent Sc-type (left) and Se-type (right) chromosomes, respectively. The grey histograms within the circles represent sequence coverage, and regions without bars indicate chromosomal deletions. Grey intersecting lines indicate interchromosomal translocations supported by mate-pair reads bridging the Sc and Se types. Green lines indicate translocations shared by the two groups. Three interchromosomal translocations were common to both groups. The line thickness is proportional to the number of links supported by mate-pair reads. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03703.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200227/dsv03703.tiff" path-from-xml="dsv03703.tiff" target="_blank">View large</a><a section="87200227" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv03703.png?Expires=2147483647&Signature=Q1rC6jCfob~jkEa77NTG7-cxHNQ5XmF9zlppmQkefefzfLrnd4AJq181zEZIvWjER3~fqV9~Slj-pp5eL3guPmwY5acAO-LgcVGxhOg07F8o954wGd1j6JAWTtOEfGC7NXkqbk3KEpduUG0eDjbJO0J~1L15UaigY~1L7VFEfpA6O-4pT5lhqNukFgR6xZ9gPjMTmhNqfvTuWNRYHhBLn0~ddJT8EzYR-L8a5jdVsfp9yDl~13iirel7ZGZb6xyKy5E2WGjCL1U6BLOQFt5H9yr3S~gMUlTypnRHzJQq57jxaxHBBvFDRSU2CFzavTYLvoupfbOom7hZR~SQMSXN2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200227&ar=2462472&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03703.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Interchromosomal translocations between Sc- and Se-type genomes. The circular layouts show interchromosomal translocations between Sc and Se types detected by mate-pair links in Group 1 (CBS15103, CBS1513, and CBS1538) and Group 2 (W34/70) strains. Blue and orange bars represent Sc-type (left) and Se-type (right) chromosomes, respectively. The grey histograms within the circles represent sequence coverage, and regions without bars indicate chromosomal deletions. Grey intersecting lines indicate interchromosomal translocations supported by mate-pair reads bridging the Sc and Se types. Green lines indicate translocations shared by the two groups. Three interchromosomal translocations were common to both groups. The line thickness is proportional to the number of links supported by mate-pair reads. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>The identification of both known and novel interchromosomal translocations with identical breakpoints at the nucleotide level in both <em>S. pastorianus</em> groups indicates that at least one hybridization event involving ancestral <em>S. cerevisiae</em> and <em>S. eubayanus</em> is shared by both <em>S. pastorianus</em> groups.</p>                            <h3 scrollto-destination=87200229 id="87200229" class="section-title" data-legacy-id=s3e>3.5. SNV analysis against reference genomes and their chromosomal distribution</h3>
<p>The SNVs were analysed using the <em>S. cerevisiae</em> S288C complete and <em>S. eubayanus</em> CBS12357 draft genomes (Sc and Se types, respectively) as references and classified as homo- and hetero-SNVs (Table <a reveal-id="DSV037TB3" data-open="DSV037TB3" class="link link-reveal link-table xref-fig">3</a>). The number of homo-SNVs was similar among the strains, and the homo-SNVs were present at a density of 1/200 bp in both the Sc and Se types, indicating that the reference genomes of both species were sufficiently similar to be used for mapping-based analysis. For the Sc-type, SNVs were less prevalent in Group 1 than in Group 2; however, this did not reflect a lower SNV density because many Group 1 Sc chromosomes were missing. In contrast, the number of hetero-Sc-type SNVs in Group 2 was markedly increased compared with Se-type SNVs, along with a 10-fold higher density. In Group 1, fewer hetero-SNVs were present because most Sc-type chromosomes were haploid or missing in this group. </p>                        <a id="87200231" scrollto-destination="87200231"></a>
<div content-id="DSV037TB3" class="table-modal"><div class="table-wrap"><div id="DSV037TB3" class="table-wrap-title" data-id="DSV037TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Number of SNVs in 10 <em>S. pastorianus</em> strains</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Strains </th><th colspan="2">Sc type<hr /> </th><th colspan="2">Se type<hr /> </th></tr><tr><th>Homo-SNV </th><th>Hetero-SNV </th><th>Homo-SNV </th><th>Hetero-SNV </th></tr></thead><tbody><tr><td colspan="5">Group 1 </td></tr><tr><td> CBS1503 </td><td>29,698 </td><td>465 </td><td>62,382 </td><td>663 </td></tr><tr><td> CBS1513 </td><td>36,371 </td><td>258 </td><td>62,520 </td><td>1,636 </td></tr><tr><td> CBS1538 </td><td>15,233 </td><td>283 </td><td>62,521 </td><td>2,340 </td></tr><tr><td> CBS1174 </td><td>15,671 </td><td>264 </td><td>62,044 </td><td>2,539 </td></tr><tr><td> CBS2440 </td><td>19,283 </td><td>325 </td><td>61,720 </td><td>2,180 </td></tr><tr><td colspan="5">Group 2 </td></tr><tr><td> W34/70 </td><td>48,892 </td><td>10,900 </td><td>59,226 </td><td>979 </td></tr><tr><td> CBS1483 </td><td>49,352 </td><td>6,459 </td><td>58,932 </td><td>683 </td></tr><tr><td> CBS1484 </td><td>47,551 </td><td>7,269 </td><td>59,479 </td><td>887 </td></tr><tr><td> CBS2156 </td><td>50,819 </td><td>5,871 </td><td>58,920 </td><td>507 </td></tr><tr><td> CBS5832 </td><td>49,849 </td><td>8,022 </td><td>60,466 </td><td>780 </td></tr></tbody></table></div><div content-id="DSV037TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Strains </th><th colspan="2">Sc type<hr /> </th><th colspan="2">Se type<hr /> </th></tr><tr><th>Homo-SNV </th><th>Hetero-SNV </th><th>Homo-SNV </th><th>Hetero-SNV </th></tr></thead><tbody><tr><td colspan="5">Group 1 </td></tr><tr><td> CBS1503 </td><td>29,698 </td><td>465 </td><td>62,382 </td><td>663 </td></tr><tr><td> CBS1513 </td><td>36,371 </td><td>258 </td><td>62,520 </td><td>1,636 </td></tr><tr><td> CBS1538 </td><td>15,233 </td><td>283 </td><td>62,521 </td><td>2,340 </td></tr><tr><td> CBS1174 </td><td>15,671 </td><td>264 </td><td>62,044 </td><td>2,539 </td></tr><tr><td> CBS2440 </td><td>19,283 </td><td>325 </td><td>61,720 </td><td>2,180 </td></tr><tr><td colspan="5">Group 2 </td></tr><tr><td> W34/70 </td><td>48,892 </td><td>10,900 </td><td>59,226 </td><td>979 </td></tr><tr><td> CBS1483 </td><td>49,352 </td><td>6,459 </td><td>58,932 </td><td>683 </td></tr><tr><td> CBS1484 </td><td>47,551 </td><td>7,269 </td><td>59,479 </td><td>887 </td></tr><tr><td> CBS2156 </td><td>50,819 </td><td>5,871 </td><td>58,920 </td><td>507 </td></tr><tr><td> CBS5832 </td><td>49,849 </td><td>8,022 </td><td>60,466 </td><td>780 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200231" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV037TB3" class="table-wrap-title" data-id="DSV037TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Number of SNVs in 10 <em>S. pastorianus</em> strains</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Strains </th><th colspan="2">Sc type<hr /> </th><th colspan="2">Se type<hr /> </th></tr><tr><th>Homo-SNV </th><th>Hetero-SNV </th><th>Homo-SNV </th><th>Hetero-SNV </th></tr></thead><tbody><tr><td colspan="5">Group 1 </td></tr><tr><td> CBS1503 </td><td>29,698 </td><td>465 </td><td>62,382 </td><td>663 </td></tr><tr><td> CBS1513 </td><td>36,371 </td><td>258 </td><td>62,520 </td><td>1,636 </td></tr><tr><td> CBS1538 </td><td>15,233 </td><td>283 </td><td>62,521 </td><td>2,340 </td></tr><tr><td> CBS1174 </td><td>15,671 </td><td>264 </td><td>62,044 </td><td>2,539 </td></tr><tr><td> CBS2440 </td><td>19,283 </td><td>325 </td><td>61,720 </td><td>2,180 </td></tr><tr><td colspan="5">Group 2 </td></tr><tr><td> W34/70 </td><td>48,892 </td><td>10,900 </td><td>59,226 </td><td>979 </td></tr><tr><td> CBS1483 </td><td>49,352 </td><td>6,459 </td><td>58,932 </td><td>683 </td></tr><tr><td> CBS1484 </td><td>47,551 </td><td>7,269 </td><td>59,479 </td><td>887 </td></tr><tr><td> CBS2156 </td><td>50,819 </td><td>5,871 </td><td>58,920 </td><td>507 </td></tr><tr><td> CBS5832 </td><td>49,849 </td><td>8,022 </td><td>60,466 </td><td>780 </td></tr></tbody></table></div><div content-id="DSV037TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Strains </th><th colspan="2">Sc type<hr /> </th><th colspan="2">Se type<hr /> </th></tr><tr><th>Homo-SNV </th><th>Hetero-SNV </th><th>Homo-SNV </th><th>Hetero-SNV </th></tr></thead><tbody><tr><td colspan="5">Group 1 </td></tr><tr><td> CBS1503 </td><td>29,698 </td><td>465 </td><td>62,382 </td><td>663 </td></tr><tr><td> CBS1513 </td><td>36,371 </td><td>258 </td><td>62,520 </td><td>1,636 </td></tr><tr><td> CBS1538 </td><td>15,233 </td><td>283 </td><td>62,521 </td><td>2,340 </td></tr><tr><td> CBS1174 </td><td>15,671 </td><td>264 </td><td>62,044 </td><td>2,539 </td></tr><tr><td> CBS2440 </td><td>19,283 </td><td>325 </td><td>61,720 </td><td>2,180 </td></tr><tr><td colspan="5">Group 2 </td></tr><tr><td> W34/70 </td><td>48,892 </td><td>10,900 </td><td>59,226 </td><td>979 </td></tr><tr><td> CBS1483 </td><td>49,352 </td><td>6,459 </td><td>58,932 </td><td>683 </td></tr><tr><td> CBS1484 </td><td>47,551 </td><td>7,269 </td><td>59,479 </td><td>887 </td></tr><tr><td> CBS2156 </td><td>50,819 </td><td>5,871 </td><td>58,920 </td><td>507 </td></tr><tr><td> CBS5832 </td><td>49,849 </td><td>8,022 </td><td>60,466 </td><td>780 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200231" target="_blank">View Large</a></div></div><p>The higher proportion of hetero-SNVs in Sc-type genomes in Group 2 compared with Se-type genomes suggests that the high heterozygosity in the former originated from an ancestral ale beer yeast.<sup><a reveal-id="DSV037C2" data-open="DSV037C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSV037C3" data-open="DSV037C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Indeed, ale beer strains—for instance, FostersB and FostersO—reportedly have high numbers of hetero-SNVs (up to 37,784 and 32,600, respectively). In comparison, lager beer yeast Sc-type genomes exhibit low heterozygosity but an uneven distribution of hetero-SNVs along Sc-type chromosomes, with some regions exhibiting many and others few or no hetero-SNVs (Fig. <a reveal-id="DSV037F4" data-open="DSV037F4" class="link link-reveal link-table xref-fig">4</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv037_Supplementary_Data.zip?Expires=2147483647&Signature=TZn0N7h5~tgToQYT8xv52vTDsAS7RmRGsX8MFEc1uGykA0xtU8Y8Q47uIHEcB2odcIw3pk-fFfxaeW9AZDI0g2ENXJ9kQ8PFhfEiL~30bZk6EM8v6JHgz61ASkfbFS0oQRQDf1sIPANxIRcS6rMZfYgW9nSZ5jANdw72pHDijzsg2RaEuNcq-fPH~SrC1ZiXpzyoxDEXpnBjfgeznOVic37wKpVqA8xYPzyK8JzGFJwicJUJitpqbhMIiQ4lOPNoRcjq-UILR6AdTAIaGHupJNKrWfh~blOTsc3UGuebfB4-0KdbLxmGapsNbnaaghNuQSUqM~DKduF8-OAve~o3KQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). An even distribution of hetero-SNVs has been reported for the <em>S. cerevisiae</em> sake yeast strain Kyokai no. 7<sup><a reveal-id="DSV037C2" data-open="DSV037C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSV037C4" data-open="DSV037C4" class="link link-ref link-reveal xref-bibr">4</a></sup> and the industrial fuel-ethanol fermentative strain CAT-1,<sup><a reveal-id="DSV037C2" data-open="DSV037C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSV037C5" data-open="DSV037C5" class="link link-ref link-reveal xref-bibr">5</a></sup> presumably as the result of frequent LOH. </p>                        <a id="87200233" scrollto-destination="87200233"></a>
<div id="DSV037F4" data-id="DSV037F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/m_dsv03704.png?Expires=2147483647&Signature=bOkahI7jjbqRK96CSx4Ie0xwWLhmsXhQ-EE5kxz740z7Z~wrBvQ5KitO-wJnG-suK7NLlamlwV8d3~aQI4jFtOzp0m4R-SZXc8Jf6C8CdTBVd1noWLrX2K68kC8Uq~8MK4zZgXHNSXIzae15UEm62Cfuux2GxPBAptTdwTo8cjkQiwYKUC3bwxR2L0wB94UbM65ttTUZa5rtq9ue5iWHAY5YKJBsW0VDlVVGxUFWajXu5xL2XcTVfg14vA5jrFW9AsjYRIVMgPQ8H8OdDCTCeay-BTj5tNp1RMBERSjTQWEnebsh1n-g6uPFEnWqAohQ11U8SrM2JAuOnnPoDWffiA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of hetero-SNVs in Sc-type chromosomes IV and VIII of Group 2 strains. The distribution of heterozygosity along Sc-type chromosomes IV and VIII in five Group 2 strains (CBS1483, CBS1484, CBS2156, CBS5832, and W34/70 from top to bottom for each chromosome) was determined by calculating a moving average of SNV density (window size = 10 kb and step size = 1 kb). Hetero-SNVs were unevenly distributed, possibly due to LOH. In many regions, low/no heterozygosity was observed in the five strains, whereas in other regions, the distribution pattern of low/no heterozygosity regions differed between strains. LOH was therefore considered a relatively frequent event, with shared low/no heterozygosity regions likely shaped in the common ancestor or by overlapping LOH events. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03704.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200233/dsv03704.tiff" path-from-xml="dsv03704.tiff" target="_blank">View large</a><a section="87200233" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv03704.png?Expires=2147483647&Signature=XCAQEm47LoBF2l13fxrXtlDWgTZq7wQl9fYWZjmrru6aumUDuWMwQfvRS7aWj9emUeEIbEsvHaPLcG7vfETZtl1ywzs-zmZ0hTCIeFlbPbQPWUj6WIoUYQsKUofp-vGwWc4gnFiH5Y26az6GalbJb69qOXbM0LeIJjrerAR4KvA7JxlLPeKP2kTP7ovpYX4V3U1AdyG3x2iogD1ZBLLSVtXQb0GgJxJzP-cdmfWvPqTN7sKGq2gaPUqbpRpkGPuZ7z-~DeR9D2q97xGtZaJl19Nld8o6ts6Xevyi4kITknHz7zE6lvsQQ-Smx8L-OnZEg~2jtszTerKLKQUK4Zul2Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200233&ar=2462472&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03704.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of hetero-SNVs in Sc-type chromosomes IV and VIII of Group 2 strains. The distribution of heterozygosity along Sc-type chromosomes IV and VIII in five Group 2 strains (CBS1483, CBS1484, CBS2156, CBS5832, and W34/70 from top to bottom for each chromosome) was determined by calculating a moving average of SNV density (window size = 10 kb and step size = 1 kb). Hetero-SNVs were unevenly distributed, possibly due to LOH. In many regions, low/no heterozygosity was observed in the five strains, whereas in other regions, the distribution pattern of low/no heterozygosity regions differed between strains. LOH was therefore considered a relatively frequent event, with shared low/no heterozygosity regions likely shaped in the common ancestor or by overlapping LOH events. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV037F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/m_dsv03704.png?Expires=2147483647&Signature=bOkahI7jjbqRK96CSx4Ie0xwWLhmsXhQ-EE5kxz740z7Z~wrBvQ5KitO-wJnG-suK7NLlamlwV8d3~aQI4jFtOzp0m4R-SZXc8Jf6C8CdTBVd1noWLrX2K68kC8Uq~8MK4zZgXHNSXIzae15UEm62Cfuux2GxPBAptTdwTo8cjkQiwYKUC3bwxR2L0wB94UbM65ttTUZa5rtq9ue5iWHAY5YKJBsW0VDlVVGxUFWajXu5xL2XcTVfg14vA5jrFW9AsjYRIVMgPQ8H8OdDCTCeay-BTj5tNp1RMBERSjTQWEnebsh1n-g6uPFEnWqAohQ11U8SrM2JAuOnnPoDWffiA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of hetero-SNVs in Sc-type chromosomes IV and VIII of Group 2 strains. The distribution of heterozygosity along Sc-type chromosomes IV and VIII in five Group 2 strains (CBS1483, CBS1484, CBS2156, CBS5832, and W34/70 from top to bottom for each chromosome) was determined by calculating a moving average of SNV density (window size = 10 kb and step size = 1 kb). Hetero-SNVs were unevenly distributed, possibly due to LOH. In many regions, low/no heterozygosity was observed in the five strains, whereas in other regions, the distribution pattern of low/no heterozygosity regions differed between strains. LOH was therefore considered a relatively frequent event, with shared low/no heterozygosity regions likely shaped in the common ancestor or by overlapping LOH events. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03704.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200233/dsv03704.tiff" path-from-xml="dsv03704.tiff" target="_blank">View large</a><a section="87200233" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv03704.png?Expires=2147483647&Signature=XCAQEm47LoBF2l13fxrXtlDWgTZq7wQl9fYWZjmrru6aumUDuWMwQfvRS7aWj9emUeEIbEsvHaPLcG7vfETZtl1ywzs-zmZ0hTCIeFlbPbQPWUj6WIoUYQsKUofp-vGwWc4gnFiH5Y26az6GalbJb69qOXbM0LeIJjrerAR4KvA7JxlLPeKP2kTP7ovpYX4V3U1AdyG3x2iogD1ZBLLSVtXQb0GgJxJzP-cdmfWvPqTN7sKGq2gaPUqbpRpkGPuZ7z-~DeR9D2q97xGtZaJl19Nld8o6ts6Xevyi4kITknHz7zE6lvsQQ-Smx8L-OnZEg~2jtszTerKLKQUK4Zul2Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200233&ar=2462472&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03704.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of hetero-SNVs in Sc-type chromosomes IV and VIII of Group 2 strains. The distribution of heterozygosity along Sc-type chromosomes IV and VIII in five Group 2 strains (CBS1483, CBS1484, CBS2156, CBS5832, and W34/70 from top to bottom for each chromosome) was determined by calculating a moving average of SNV density (window size = 10 kb and step size = 1 kb). Hetero-SNVs were unevenly distributed, possibly due to LOH. In many regions, low/no heterozygosity was observed in the five strains, whereas in other regions, the distribution pattern of low/no heterozygosity regions differed between strains. LOH was therefore considered a relatively frequent event, with shared low/no heterozygosity regions likely shaped in the common ancestor or by overlapping LOH events. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>Based on these observations and previous reports, the uneven distribution of heterozygosity may be due to the loss of variation between intrahomologous chromosomes. After a hybridization event between <em>S. cerevisiae</em> and <em>S. eubayanus</em>, variations between intrahomologous chromosomes in Sc types are thought to have gradually disappeared via LOH. Regions of high hetero-SNV density likely originated from the ancestral ale beer yeast, whereas low-density regions or those with no hetero-SNVs are likely the imprint of LOH. Low/no heterozygosity was observed in various regions in all five Group 2 strains, albeit with different distribution patterns. Therefore, LOH is presumed to have been a relatively frequent event, with low/no heterozygosity regions arising in the common ancestor or as a result of overlapping LOH events.</p>                            <h3 scrollto-destination=87200235 id="87200235" class="section-title" data-legacy-id=s3f>3.6. Phylogenetic analysis based on chromosomal SNVs against reference genomes</h3>
<p>Phylogenetic trees based on SNVs in chromosomes queried against reference genomes were constructed separately for the Sc and Se types (Fig. <a reveal-id="DSV037F5" data-open="DSV037F5" class="link link-reveal link-table xref-fig">5</a>). In the Sc-type tree, the strains were segregated into two clades, Groups 1 and 2, in agreement with previous studies<sup><a reveal-id="DSV037C11" data-open="DSV037C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSV037C12" data-open="DSV037C12" class="link link-ref link-reveal xref-bibr">12</a></sup> based on partial PCR-based sequence data. In contrast, the Se-type phylogenetic tree revealed that <em>S. pastorianus</em> strains were mixed in nearly a single clade, with no significant phylogenetic difference between Groups 1 and 2. Indeed, the Se-type variant rate among the 10 <em>S. pastorianus</em> strains was only 0.063%, which was much lower than that of the Sc-type (0.348%). For the Sc-type, the tree suggests that the two groups—which are separated by geographical location—have independent origins. However, the Se-type phylogenetic tree suggests that the different aspect from that of Sc-type. Se-type genomes of the two groups look like originated from a common ancestor. Thus, the conflicting characteristics of the phylogenetic trees may be attributable to differences in genetic features between the Sc and Se type strains, namely ploidy and hetero-SNV density. </p>                        <a id="87200237" scrollto-destination="87200237"></a>
<div id="DSV037F5" data-id="DSV037F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/m_dsv03705.png?Expires=2147483647&Signature=FstBUz15P5SfxJ22fBmH6GNQjSdYt-QW7AayUAwF8qkk-9EgMoGvpqr00~Sf-jj6AnDReMP7rzx-KpQuJwOtwlxufOAi-hcSoEbAGmf3COxFYwlnpFrBnzrEpBxZh6VOSNDpB~gHMtYCR1VBi3LRdu2WRbiepQNh4Y5WHrWvePw58B88Qeiq1timy19YCgFADNFoN-20XQffGc3lk~yu42InnX26XC12i8X~ShWCIepxKi-YJuN3BF8V--AOxAy~6yZzltgBpdaNBt30eZghrGKtCOQDI34ksET-IbOFUIZ2r1FpLEKJ1DI7UFh0fk1dMK0X2XcUGH4eUzAkqrDccw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic trees based on SNVs in chromosomes. (A and B) Phylogenetic tree of the Sc type in 10 S. pastorianus strains with S. cerevisiae S288C as a reference genome. (B) Enlarged view. (C and D) Phylogenetic tree of the Se type in S. pastorianus strains with S. eubayanus CBS12357 as a reference genome. (D) Enlarged view. In the Sc type, S. pastorianus strains are divided into two clades (Groups 1 and 2); however, in the Se type, the strains form a single mixed clade. Bootstrap values of 1,000 trials are indicated as blue numbers. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03705.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200237/dsv03705.tiff" path-from-xml="dsv03705.tiff" target="_blank">View large</a><a section="87200237" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv03705.png?Expires=2147483647&Signature=H-tWy-N2QMfeMCRLWHAsZIcXy~fLBLOsDk1esWapimy3kaC49I4zocvPYqAWsxUqcC2CGoSoKNVrqE8Uz92hLWQGwvYCkMfP4J7d-6l0jWdGGZu2wDdXDiuxEfb6D59e3gizB~JFfzC~FedCowAQs9EBGOvRH5AuVE8NQq8MmB~RNCp05hq~xDOrHRphT4oYhLbGSrJCCv3O7V982d0UCotMCNS4OEiKZ5ep29p2iXq4xbOqlEkHzcsB12G86jZ6VwmU5sR6~jSDKSl3WRS1vjnNlcf45oqGl5rE5WXsj2XdjNvovnFAYDtI0xa~dL8yhJewIFUrVXrQ0zEQPeoBhw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200237&ar=2462472&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03705.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic trees based on SNVs in chromosomes. (A and B) Phylogenetic tree of the Sc type in 10 <em>S. pastorianus</em> strains with <em>S. cerevisiae</em> S288C as a reference genome. (B) Enlarged view. (C and D) Phylogenetic tree of the Se type in <em>S. pastorianus</em> strains with <em>S. eubayanus</em> CBS12357 as a reference genome. (D) Enlarged view. In the Sc type, <em>S. pastorianus</em> strains are divided into two clades (Groups 1 and 2); however, in the Se type, the strains form a single mixed clade. Bootstrap values of 1,000 trials are indicated as blue numbers. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV037F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/m_dsv03705.png?Expires=2147483647&Signature=FstBUz15P5SfxJ22fBmH6GNQjSdYt-QW7AayUAwF8qkk-9EgMoGvpqr00~Sf-jj6AnDReMP7rzx-KpQuJwOtwlxufOAi-hcSoEbAGmf3COxFYwlnpFrBnzrEpBxZh6VOSNDpB~gHMtYCR1VBi3LRdu2WRbiepQNh4Y5WHrWvePw58B88Qeiq1timy19YCgFADNFoN-20XQffGc3lk~yu42InnX26XC12i8X~ShWCIepxKi-YJuN3BF8V--AOxAy~6yZzltgBpdaNBt30eZghrGKtCOQDI34ksET-IbOFUIZ2r1FpLEKJ1DI7UFh0fk1dMK0X2XcUGH4eUzAkqrDccw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic trees based on SNVs in chromosomes. (A and B) Phylogenetic tree of the Sc type in 10 S. pastorianus strains with S. cerevisiae S288C as a reference genome. (B) Enlarged view. (C and D) Phylogenetic tree of the Se type in S. pastorianus strains with S. eubayanus CBS12357 as a reference genome. (D) Enlarged view. In the Sc type, S. pastorianus strains are divided into two clades (Groups 1 and 2); however, in the Se type, the strains form a single mixed clade. Bootstrap values of 1,000 trials are indicated as blue numbers. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03705.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200237/dsv03705.tiff" path-from-xml="dsv03705.tiff" target="_blank">View large</a><a section="87200237" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv03705.png?Expires=2147483647&Signature=H-tWy-N2QMfeMCRLWHAsZIcXy~fLBLOsDk1esWapimy3kaC49I4zocvPYqAWsxUqcC2CGoSoKNVrqE8Uz92hLWQGwvYCkMfP4J7d-6l0jWdGGZu2wDdXDiuxEfb6D59e3gizB~JFfzC~FedCowAQs9EBGOvRH5AuVE8NQq8MmB~RNCp05hq~xDOrHRphT4oYhLbGSrJCCv3O7V982d0UCotMCNS4OEiKZ5ep29p2iXq4xbOqlEkHzcsB12G86jZ6VwmU5sR6~jSDKSl3WRS1vjnNlcf45oqGl5rE5WXsj2XdjNvovnFAYDtI0xa~dL8yhJewIFUrVXrQ0zEQPeoBhw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200237&ar=2462472&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03705.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic trees based on SNVs in chromosomes. (A and B) Phylogenetic tree of the Sc type in 10 <em>S. pastorianus</em> strains with <em>S. cerevisiae</em> S288C as a reference genome. (B) Enlarged view. (C and D) Phylogenetic tree of the Se type in <em>S. pastorianus</em> strains with <em>S. eubayanus</em> CBS12357 as a reference genome. (D) Enlarged view. In the Sc type, <em>S. pastorianus</em> strains are divided into two clades (Groups 1 and 2); however, in the Se type, the strains form a single mixed clade. Bootstrap values of 1,000 trials are indicated as blue numbers. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87200238 id="87200238" class="section-title" data-legacy-id=s3g>3.7. Assignment of hetero-SNVs and haplotype phasing</h3>
<p>We further validated these SNVs to address the conflicting phylogenic analysis by focusing on the hetero-SNVs in Group 2. This assessment revealed interesting results for the SNV numbers for Group 1 and Group 2 strains (Table <a reveal-id="DSV037TB4" data-open="DSV037TB4" class="link link-reveal link-table xref-fig">4</a>). Specifically, nearly all the homo–hetero-SNVs observed between Group 1 and Group 2 are consistent relationships (i.e. Group 1's homo allele is equal to either allele of Group 2's hetero-SNV at the corresponding locus), and there are few conflicting SNVs. With few exceptions, either allele of the Group 2 hetero-type SNVs consists of the same sequence as the corresponding positions' haploid Group 1 allele. To confirm that this relationship was not coincidental, we conducted an additional analysis to solve the linkage relationships of W34/70 adjacent hetero-SNVs (‘phasing’) based on sequence data to construct two separate intrahomologous haploid chromosomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv037_Supplementary_Data.zip?Expires=2147483647&Signature=TZn0N7h5~tgToQYT8xv52vTDsAS7RmRGsX8MFEc1uGykA0xtU8Y8Q47uIHEcB2odcIw3pk-fFfxaeW9AZDI0g2ENXJ9kQ8PFhfEiL~30bZk6EM8v6JHgz61ASkfbFS0oQRQDf1sIPANxIRcS6rMZfYgW9nSZ5jANdw72pHDijzsg2RaEuNcq-fPH~SrC1ZiXpzyoxDEXpnBjfgeznOVic37wKpVqA8xYPzyK8JzGFJwicJUJitpqbhMIiQ4lOPNoRcjq-UILR6AdTAIaGHupJNKrWfh~blOTsc3UGuebfB4-0KdbLxmGapsNbnaaghNuQSUqM~DKduF8-OAve~o3KQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5A and B</a></span>). We obtained 304 blocks containing five or more consecutive hetero-SNVs each. The total size of the blocks was 1,876,249 bp (15.6% of the whole Sc type genome), and the blocks consisted of 9,765 hetero-SNVs (89.6% of all hetero-SNVs in W34/70). To maximize the length of sequences comparable to phased haplotypes of W34/70, a consensus sequence was built for the Group 1 strains. As shown in Fig. <a reveal-id="DSV037F5" data-open="DSV037F5" class="link link-reveal link-table xref-fig">5</a>A, the differences between Group 1 strains are quite few and seemingly negligible for this analysis. Finally, we constructed a total of 274 blocks (1,705,313 bp including 8,731 hetero-SNVs) that could be compared with the Group 1 consensus sequence, covering 80.1% of all hetero-SNVs in W34/70. </p>                        <a id="87200240" scrollto-destination="87200240"></a>
<div content-id="DSV037TB4" class="table-modal"><div class="table-wrap"><div id="DSV037TB4" class="table-wrap-title" data-id="DSV037TB4"><div class="label -label">Table 4.</div><div class="caption"><p>SNV numbers between Group 1 and 2 strains</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="5">Group 2<hr /> </th></tr><tr><th>W34/70 </th><th>CBS1483 </th><th>CBS1484 </th><th>CBS2156 </th><th>CBS5832 </th></tr></thead><tbody><tr><td colspan="6">Group 1 </td></tr><tr><td colspan="6"> CBS1503 </td></tr><tr><td>  SNV site # *1 </td><td>18,783 </td><td>17,769 </td><td>17,323 </td><td>17,414 </td><td>17,673 </td></tr><tr><td>  Homo/hetero site # *2 </td><td>6,444 </td><td>5,053 </td><td>4,006 </td><td>3,723 </td><td>4,709 </td></tr><tr><td>  Consistent site # *3 </td><td>6,438 </td><td>5,049 </td><td>4,005 </td><td>3,720 </td><td>4,705 </td></tr><tr><td>  Conflict site # *4 </td><td>6 </td><td>4 </td><td>1 </td><td>3 </td><td>4 </td></tr><tr><td colspan="6"> CBS1513 </td></tr><tr><td>  SNV site # *1 </td><td>23,791 </td><td>21,765 </td><td>22,912 </td><td>21,669 </td><td>22,308 </td></tr><tr><td>  Homo/hetero site # *2 </td><td>9,323 </td><td>5,739 </td><td>6,632 </td><td>5,503 </td><td>7,289 </td></tr><tr><td>  Consistent site # *3 </td><td>9,318 </td><td>5,738 </td><td>6,631 </td><td>5,500 </td><td>7,286 </td></tr><tr><td>  Conflict site # *4 </td><td>5 </td><td>1 </td><td>1 </td><td>3 </td><td>3 </td></tr><tr><td colspan="6"> CBS1538 </td></tr><tr><td>  SNV site # *1 </td><td>9,117 </td><td>8,402 </td><td>8,638 </td><td>8,437 </td><td>9,225 </td></tr><tr><td>  Homo/hetero site # *2 </td><td>2,883 </td><td>1,664 </td><td>2,300 </td><td>1,786 </td><td>3,058 </td></tr><tr><td>  Consistent site # *3 </td><td>2,881 </td><td>1,664 </td><td>2,300 </td><td>1,785 </td><td>3,056 </td></tr><tr><td>  Conflict site # *4 </td><td>2 </td><td>0 </td><td>0 </td><td>1 </td><td>2 </td></tr><tr><td colspan="6"> CBS1174 </td></tr><tr><td>  SNV site # *1 </td><td>9,550 </td><td>8,833 </td><td>9,111 </td><td>8,901 </td><td>9,641 </td></tr><tr><td>  Homo/hetero site # *2 </td><td>2,865 </td><td>1,651 </td><td>2,298 </td><td>1,785 </td><td>3,037 </td></tr><tr><td>  Consistent site # *3 </td><td>2,862 </td><td>1,651 </td><td>2,297 </td><td>1,783 </td><td>3,034 </td></tr><tr><td>  Conflict site # *4 </td><td>3 </td><td>0 </td><td>1 </td><td>2 </td><td>3 </td></tr><tr><td colspan="6"> CBS2440 </td></tr><tr><td>  SNV site # *1 </td><td>12,223 </td><td>11,176 </td><td>10,287 </td><td>11,084 </td><td>12,242 </td></tr><tr><td>  Homo/hetero site # *2 </td><td>4,120 </td><td>2,302 </td><td>2,272 </td><td>1,886 </td><td>4,071 </td></tr><tr><td>  Consistent site # *3 </td><td>4,118 </td><td>2,302 </td><td>2,272 </td><td>1,885 </td><td>4,069 </td></tr><tr><td>  Conflict site # *4 </td><td>2 </td><td>0 </td><td>0 </td><td>1 </td><td>2 </td></tr></tbody></table></div><div content-id="DSV037TB4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="5">Group 2<hr /> </th></tr><tr><th>W34/70 </th><th>CBS1483 </th><th>CBS1484 </th><th>CBS2156 </th><th>CBS5832 </th></tr></thead><tbody><tr><td colspan="6">Group 1 </td></tr><tr><td colspan="6"> CBS1503 </td></tr><tr><td>  SNV site # *1 </td><td>18,783 </td><td>17,769 </td><td>17,323 </td><td>17,414 </td><td>17,673 </td></tr><tr><td>  Homo/hetero site # *2 </td><td>6,444 </td><td>5,053 </td><td>4,006 </td><td>3,723 </td><td>4,709 </td></tr><tr><td>  Consistent site # *3 </td><td>6,438 </td><td>5,049 </td><td>4,005 </td><td>3,720 </td><td>4,705 </td></tr><tr><td>  Conflict site # *4 </td><td>6 </td><td>4 </td><td>1 </td><td>3 </td><td>4 </td></tr><tr><td colspan="6"> CBS1513 </td></tr><tr><td>  SNV site # *1 </td><td>23,791 </td><td>21,765 </td><td>22,912 </td><td>21,669 </td><td>22,308 </td></tr><tr><td>  Homo/hetero site # *2 </td><td>9,323 </td><td>5,739 </td><td>6,632 </td><td>5,503 </td><td>7,289 </td></tr><tr><td>  Consistent site # *3 </td><td>9,318 </td><td>5,738 </td><td>6,631 </td><td>5,500 </td><td>7,286 </td></tr><tr><td>  Conflict site # *4 </td><td>5 </td><td>1 </td><td>1 </td><td>3 </td><td>3 </td></tr><tr><td colspan="6"> CBS1538 </td></tr><tr><td>  SNV site # *1 </td><td>9,117 </td><td>8,402 </td><td>8,638 </td><td>8,437 </td><td>9,225 </td></tr><tr><td>  Homo/hetero site # *2 </td><td>2,883 </td><td>1,664 </td><td>2,300 </td><td>1,786 </td><td>3,058 </td></tr><tr><td>  Consistent site # *3 </td><td>2,881 </td><td>1,664 </td><td>2,300 </td><td>1,785 </td><td>3,056 </td></tr><tr><td>  Conflict site # *4 </td><td>2 </td><td>0 </td><td>0 </td><td>1 </td><td>2 </td></tr><tr><td colspan="6"> CBS1174 </td></tr><tr><td>  SNV site # *1 </td><td>9,550 </td><td>8,833 </td><td>9,111 </td><td>8,901 </td><td>9,641 </td></tr><tr><td>  Homo/hetero site # *2 </td><td>2,865 </td><td>1,651 </td><td>2,298 </td><td>1,785 </td><td>3,037 </td></tr><tr><td>  Consistent site # *3 </td><td>2,862 </td><td>1,651 </td><td>2,297 </td><td>1,783 </td><td>3,034 </td></tr><tr><td>  Conflict site # *4 </td><td>3 </td><td>0 </td><td>1 </td><td>2 </td><td>3 </td></tr><tr><td colspan="6"> CBS2440 </td></tr><tr><td>  SNV site # *1 </td><td>12,223 </td><td>11,176 </td><td>10,287 </td><td>11,084 </td><td>12,242 </td></tr><tr><td>  Homo/hetero site # *2 </td><td>4,120 </td><td>2,302 </td><td>2,272 </td><td>1,886 </td><td>4,071 </td></tr><tr><td>  Consistent site # *3 </td><td>4,118 </td><td>2,302 </td><td>2,272 </td><td>1,885 </td><td>4,069 </td></tr><tr><td>  Conflict site # *4 </td><td>2 </td><td>0 </td><td>0 </td><td>1 </td><td>2 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>*1, Number of SNV sites between Group 1 and 2 strains.</p></div></div><div content-id="tblfn2"><div class="fn"><p>*2, Number of SNV sites between Group 1 homo vs. Group 2 hetero-types.</p></div></div><div content-id="tblfn3"><div class="fn"><p>*3, Number of consistent SNV sites, i.e. a Group 1 homo allele is included with either allele of Group 2 hetero-SNVs.</p></div></div><div content-id="tblfn4"><div class="fn"><p>*4, Number of conflicting SNV sites.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200240" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV037TB4" class="table-wrap-title" data-id="DSV037TB4"><div class="label -label">Table 4.</div><div class="caption"><p>SNV numbers between Group 1 and 2 strains</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="5">Group 2<hr /> </th></tr><tr><th>W34/70 </th><th>CBS1483 </th><th>CBS1484 </th><th>CBS2156 </th><th>CBS5832 </th></tr></thead><tbody><tr><td colspan="6">Group 1 </td></tr><tr><td colspan="6"> CBS1503 </td></tr><tr><td>  SNV site # *1 </td><td>18,783 </td><td>17,769 </td><td>17,323 </td><td>17,414 </td><td>17,673 </td></tr><tr><td>  Homo/hetero site # *2 </td><td>6,444 </td><td>5,053 </td><td>4,006 </td><td>3,723 </td><td>4,709 </td></tr><tr><td>  Consistent site # *3 </td><td>6,438 </td><td>5,049 </td><td>4,005 </td><td>3,720 </td><td>4,705 </td></tr><tr><td>  Conflict site # *4 </td><td>6 </td><td>4 </td><td>1 </td><td>3 </td><td>4 </td></tr><tr><td colspan="6"> CBS1513 </td></tr><tr><td>  SNV site # *1 </td><td>23,791 </td><td>21,765 </td><td>22,912 </td><td>21,669 </td><td>22,308 </td></tr><tr><td>  Homo/hetero site # *2 </td><td>9,323 </td><td>5,739 </td><td>6,632 </td><td>5,503 </td><td>7,289 </td></tr><tr><td>  Consistent site # *3 </td><td>9,318 </td><td>5,738 </td><td>6,631 </td><td>5,500 </td><td>7,286 </td></tr><tr><td>  Conflict site # *4 </td><td>5 </td><td>1 </td><td>1 </td><td>3 </td><td>3 </td></tr><tr><td colspan="6"> CBS1538 </td></tr><tr><td>  SNV site # *1 </td><td>9,117 </td><td>8,402 </td><td>8,638 </td><td>8,437 </td><td>9,225 </td></tr><tr><td>  Homo/hetero site # *2 </td><td>2,883 </td><td>1,664 </td><td>2,300 </td><td>1,786 </td><td>3,058 </td></tr><tr><td>  Consistent site # *3 </td><td>2,881 </td><td>1,664 </td><td>2,300 </td><td>1,785 </td><td>3,056 </td></tr><tr><td>  Conflict site # *4 </td><td>2 </td><td>0 </td><td>0 </td><td>1 </td><td>2 </td></tr><tr><td colspan="6"> CBS1174 </td></tr><tr><td>  SNV site # *1 </td><td>9,550 </td><td>8,833 </td><td>9,111 </td><td>8,901 </td><td>9,641 </td></tr><tr><td>  Homo/hetero site # *2 </td><td>2,865 </td><td>1,651 </td><td>2,298 </td><td>1,785 </td><td>3,037 </td></tr><tr><td>  Consistent site # *3 </td><td>2,862 </td><td>1,651 </td><td>2,297 </td><td>1,783 </td><td>3,034 </td></tr><tr><td>  Conflict site # *4 </td><td>3 </td><td>0 </td><td>1 </td><td>2 </td><td>3 </td></tr><tr><td colspan="6"> CBS2440 </td></tr><tr><td>  SNV site # *1 </td><td>12,223 </td><td>11,176 </td><td>10,287 </td><td>11,084 </td><td>12,242 </td></tr><tr><td>  Homo/hetero site # *2 </td><td>4,120 </td><td>2,302 </td><td>2,272 </td><td>1,886 </td><td>4,071 </td></tr><tr><td>  Consistent site # *3 </td><td>4,118 </td><td>2,302 </td><td>2,272 </td><td>1,885 </td><td>4,069 </td></tr><tr><td>  Conflict site # *4 </td><td>2 </td><td>0 </td><td>0 </td><td>1 </td><td>2 </td></tr></tbody></table></div><div content-id="DSV037TB4" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="5">Group 2<hr /> </th></tr><tr><th>W34/70 </th><th>CBS1483 </th><th>CBS1484 </th><th>CBS2156 </th><th>CBS5832 </th></tr></thead><tbody><tr><td colspan="6">Group 1 </td></tr><tr><td colspan="6"> CBS1503 </td></tr><tr><td>  SNV site # *1 </td><td>18,783 </td><td>17,769 </td><td>17,323 </td><td>17,414 </td><td>17,673 </td></tr><tr><td>  Homo/hetero site # *2 </td><td>6,444 </td><td>5,053 </td><td>4,006 </td><td>3,723 </td><td>4,709 </td></tr><tr><td>  Consistent site # *3 </td><td>6,438 </td><td>5,049 </td><td>4,005 </td><td>3,720 </td><td>4,705 </td></tr><tr><td>  Conflict site # *4 </td><td>6 </td><td>4 </td><td>1 </td><td>3 </td><td>4 </td></tr><tr><td colspan="6"> CBS1513 </td></tr><tr><td>  SNV site # *1 </td><td>23,791 </td><td>21,765 </td><td>22,912 </td><td>21,669 </td><td>22,308 </td></tr><tr><td>  Homo/hetero site # *2 </td><td>9,323 </td><td>5,739 </td><td>6,632 </td><td>5,503 </td><td>7,289 </td></tr><tr><td>  Consistent site # *3 </td><td>9,318 </td><td>5,738 </td><td>6,631 </td><td>5,500 </td><td>7,286 </td></tr><tr><td>  Conflict site # *4 </td><td>5 </td><td>1 </td><td>1 </td><td>3 </td><td>3 </td></tr><tr><td colspan="6"> CBS1538 </td></tr><tr><td>  SNV site # *1 </td><td>9,117 </td><td>8,402 </td><td>8,638 </td><td>8,437 </td><td>9,225 </td></tr><tr><td>  Homo/hetero site # *2 </td><td>2,883 </td><td>1,664 </td><td>2,300 </td><td>1,786 </td><td>3,058 </td></tr><tr><td>  Consistent site # *3 </td><td>2,881 </td><td>1,664 </td><td>2,300 </td><td>1,785 </td><td>3,056 </td></tr><tr><td>  Conflict site # *4 </td><td>2 </td><td>0 </td><td>0 </td><td>1 </td><td>2 </td></tr><tr><td colspan="6"> CBS1174 </td></tr><tr><td>  SNV site # *1 </td><td>9,550 </td><td>8,833 </td><td>9,111 </td><td>8,901 </td><td>9,641 </td></tr><tr><td>  Homo/hetero site # *2 </td><td>2,865 </td><td>1,651 </td><td>2,298 </td><td>1,785 </td><td>3,037 </td></tr><tr><td>  Consistent site # *3 </td><td>2,862 </td><td>1,651 </td><td>2,297 </td><td>1,783 </td><td>3,034 </td></tr><tr><td>  Conflict site # *4 </td><td>3 </td><td>0 </td><td>1 </td><td>2 </td><td>3 </td></tr><tr><td colspan="6"> CBS2440 </td></tr><tr><td>  SNV site # *1 </td><td>12,223 </td><td>11,176 </td><td>10,287 </td><td>11,084 </td><td>12,242 </td></tr><tr><td>  Homo/hetero site # *2 </td><td>4,120 </td><td>2,302 </td><td>2,272 </td><td>1,886 </td><td>4,071 </td></tr><tr><td>  Consistent site # *3 </td><td>4,118 </td><td>2,302 </td><td>2,272 </td><td>1,885 </td><td>4,069 </td></tr><tr><td>  Conflict site # *4 </td><td>2 </td><td>0 </td><td>0 </td><td>1 </td><td>2 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>*1, Number of SNV sites between Group 1 and 2 strains.</p></div></div><div content-id="tblfn2"><div class="fn"><p>*2, Number of SNV sites between Group 1 homo vs. Group 2 hetero-types.</p></div></div><div content-id="tblfn3"><div class="fn"><p>*3, Number of consistent SNV sites, i.e. a Group 1 homo allele is included with either allele of Group 2 hetero-SNVs.</p></div></div><div content-id="tblfn4"><div class="fn"><p>*4, Number of conflicting SNV sites.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200240" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87200241 id="87200241" class="section-title" data-legacy-id=s3h>3.8. Phylogenetic analysis based on haplotype-phased chromosomal SNVs</h3>
<p>As the result of the analysis described above, we obtained 274 blocks consisting of three Sc-type chromosome sequences each (i.e. one Group 1 consensus chromosome and two W34/70 haplotype-phased intrahomologous chromosomes). We divided the phased chromosomes into two groups, W34/70_a and W34/70_b, based on sequence identity with Group 1 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv037_Supplementary_Data.zip?Expires=2147483647&Signature=TZn0N7h5~tgToQYT8xv52vTDsAS7RmRGsX8MFEc1uGykA0xtU8Y8Q47uIHEcB2odcIw3pk-fFfxaeW9AZDI0g2ENXJ9kQ8PFhfEiL~30bZk6EM8v6JHgz61ASkfbFS0oQRQDf1sIPANxIRcS6rMZfYgW9nSZ5jANdw72pHDijzsg2RaEuNcq-fPH~SrC1ZiXpzyoxDEXpnBjfgeznOVic37wKpVqA8xYPzyK8JzGFJwicJUJitpqbhMIiQ4lOPNoRcjq-UILR6AdTAIaGHupJNKrWfh~blOTsc3UGuebfB4-0KdbLxmGapsNbnaaghNuQSUqM~DKduF8-OAve~o3KQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5C</a></span> and Fig. <a reveal-id="DSV037F6" data-open="DSV037F6" class="link link-reveal link-table xref-fig">6</a>A). The same analysis results for the randomly shuffled hetero alleles of W34/70 dropping off the linkage information (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv037_Supplementary_Data.zip?Expires=2147483647&Signature=TZn0N7h5~tgToQYT8xv52vTDsAS7RmRGsX8MFEc1uGykA0xtU8Y8Q47uIHEcB2odcIw3pk-fFfxaeW9AZDI0g2ENXJ9kQ8PFhfEiL~30bZk6EM8v6JHgz61ASkfbFS0oQRQDf1sIPANxIRcS6rMZfYgW9nSZ5jANdw72pHDijzsg2RaEuNcq-fPH~SrC1ZiXpzyoxDEXpnBjfgeznOVic37wKpVqA8xYPzyK8JzGFJwicJUJitpqbhMIiQ4lOPNoRcjq-UILR6AdTAIaGHupJNKrWfh~blOTsc3UGuebfB4-0KdbLxmGapsNbnaaghNuQSUqM~DKduF8-OAve~o3KQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5D and E</a></span>) are presented in Fig. <a reveal-id="DSV037F6" data-open="DSV037F6" class="link link-reveal link-table xref-fig">6</a>B. As shown in Fig. <a reveal-id="DSV037F6" data-open="DSV037F6" class="link link-reveal link-table xref-fig">6</a>A, the Group 1 and W34/70_a identity distribution was significantly higher than that between Group 1 and W34/70_b, but no significant difference in the two distributions was observed based on randomly divided allele information (Fig. <a reveal-id="DSV037F6" data-open="DSV037F6" class="link link-reveal link-table xref-fig">6</a>B). In addition, a phylogenetic tree of the consensus sequence of the Group 1 strains, the two haplotypes of W34/70 (W34/70_a and W34/70_b), and S288c is presented in Fig. <a reveal-id="DSV037F6" data-open="DSV037F6" class="link link-reveal link-table xref-fig">6</a>C. The tree was described based on 14,322 sites containing 8,731 hetero-SNVs within 274 blocks. As shown in Fig. <a reveal-id="DSV037F6" data-open="DSV037F6" class="link link-reveal link-table xref-fig">6</a>C, Group 1 and W34/70_a reside in the same clade. Similarly, the phylogenetic tree of the other four Group 2 strains revealed that all Group 2 ‘type a’ chromosomes reside in the same clade as Group 1 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv037_Supplementary_Data.zip?Expires=2147483647&Signature=TZn0N7h5~tgToQYT8xv52vTDsAS7RmRGsX8MFEc1uGykA0xtU8Y8Q47uIHEcB2odcIw3pk-fFfxaeW9AZDI0g2ENXJ9kQ8PFhfEiL~30bZk6EM8v6JHgz61ASkfbFS0oQRQDf1sIPANxIRcS6rMZfYgW9nSZ5jANdw72pHDijzsg2RaEuNcq-fPH~SrC1ZiXpzyoxDEXpnBjfgeznOVic37wKpVqA8xYPzyK8JzGFJwicJUJitpqbhMIiQ4lOPNoRcjq-UILR6AdTAIaGHupJNKrWfh~blOTsc3UGuebfB4-0KdbLxmGapsNbnaaghNuQSUqM~DKduF8-OAve~o3KQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). These results support the non-coincidental nature of the observation that either of the Group 2 hetero-type SNV alleles has the same sequence with the corresponding positions' haploid Group 1 allele, thus strongly suggesting that one of Group 2's chromosomes is shared with Group 1's chromosomes. </p>                        <a id="87200243" scrollto-destination="87200243"></a>
<div id="DSV037F6" data-id="DSV037F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/m_dsv03706.png?Expires=2147483647&Signature=RYgvKvzF9IsTmjlzDNr3E3ue-L~RyR6MYND8KTZIYHcylu7kJskG-4nR67adiVHuvkQpCPt5OpnlJypwgu19E5sJPf3nR-XsLCTEJ9d31fjOPDVtmXfcnEcFU3KdbyzK-3XwmvRwA0cnx7OQK7lndLorImxXKipw9zvYUpsTSjTQr6hPGd4A70EQHcAm470D7zPQ-e6mAkTJNPiG-NLpYO04oNNUGv-VDm4nDEBHIrQ66ZYDAqzgU0F3q28Yg8Stg1rK0kjWC~z0k32j1asscxXhm2X2oic5fepo-I5YUhLF5JYZ4bojmGAEoFonk2HrXDMIt9gTH3~vZhT3poHjKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic relationship between haplotype-phased W34/70 sequences and Group 1 consensus sequences. (A) Identity distribution between haplotype-phased W34/70 chromosome sequences (W34/70_a, close with Group 1 sequences; W34/70_b, another chromosome sequence) and Group 1 sequences. (B) Identity distribution between haplotype-phased W34/70 chromosome sequences (hetero alleles are randomly shuffled dropping off the linkage information, average of 100 trials). (C) Phylogenetic tree based on haplotype-phased W34/70 sequences and Group 1 sequences. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03706.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200243/dsv03706.tiff" path-from-xml="dsv03706.tiff" target="_blank">View large</a><a section="87200243" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv03706.png?Expires=2147483647&Signature=QJFwPpAoE4Vf4jl44ka1Wmq4vRJq7bvH3qxSG33JQbAPEDnY3sYHRCowPjIocB-aYSKHwbsn0wWydK~f8WJyy1wBGj3sIiTHE3ZHtTtKIHgqpElTjsLyE4Z9bbvXRaVzAFV9HkrkHYN~zVYsiHqW25ido9gfuypO8innXmUXFEVlL3bgYmrf~kb0GsGGrC8ERU1BtbqxjjxA4Gpzxo7DyMeLbRNHIH5cCil3vaR9u~RY0Xmm6rAh6-I8Fl~hMXNX2pqwoMIdlVIAW60YY6jlMJ26-hXmqime3vrfEF6XHN25Y8qIrp-g8rYO0Pp7jGiMOUxazpPIXk71AXNEZQ-65Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200243&ar=2462472&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03706.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic relationship between haplotype-phased W34/70 sequences and Group 1 consensus sequences. (A) Identity distribution between haplotype-phased W34/70 chromosome sequences (W34/70_a, close with Group 1 sequences; W34/70_b, another chromosome sequence) and Group 1 sequences. (B) Identity distribution between haplotype-phased W34/70 chromosome sequences (hetero alleles are randomly shuffled dropping off the linkage information, average of 100 trials). (C) Phylogenetic tree based on haplotype-phased W34/70 sequences and Group 1 sequences. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV037F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/m_dsv03706.png?Expires=2147483647&Signature=RYgvKvzF9IsTmjlzDNr3E3ue-L~RyR6MYND8KTZIYHcylu7kJskG-4nR67adiVHuvkQpCPt5OpnlJypwgu19E5sJPf3nR-XsLCTEJ9d31fjOPDVtmXfcnEcFU3KdbyzK-3XwmvRwA0cnx7OQK7lndLorImxXKipw9zvYUpsTSjTQr6hPGd4A70EQHcAm470D7zPQ-e6mAkTJNPiG-NLpYO04oNNUGv-VDm4nDEBHIrQ66ZYDAqzgU0F3q28Yg8Stg1rK0kjWC~z0k32j1asscxXhm2X2oic5fepo-I5YUhLF5JYZ4bojmGAEoFonk2HrXDMIt9gTH3~vZhT3poHjKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic relationship between haplotype-phased W34/70 sequences and Group 1 consensus sequences. (A) Identity distribution between haplotype-phased W34/70 chromosome sequences (W34/70_a, close with Group 1 sequences; W34/70_b, another chromosome sequence) and Group 1 sequences. (B) Identity distribution between haplotype-phased W34/70 chromosome sequences (hetero alleles are randomly shuffled dropping off the linkage information, average of 100 trials). (C) Phylogenetic tree based on haplotype-phased W34/70 sequences and Group 1 sequences. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03706.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200243/dsv03706.tiff" path-from-xml="dsv03706.tiff" target="_blank">View large</a><a section="87200243" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv03706.png?Expires=2147483647&Signature=QJFwPpAoE4Vf4jl44ka1Wmq4vRJq7bvH3qxSG33JQbAPEDnY3sYHRCowPjIocB-aYSKHwbsn0wWydK~f8WJyy1wBGj3sIiTHE3ZHtTtKIHgqpElTjsLyE4Z9bbvXRaVzAFV9HkrkHYN~zVYsiHqW25ido9gfuypO8innXmUXFEVlL3bgYmrf~kb0GsGGrC8ERU1BtbqxjjxA4Gpzxo7DyMeLbRNHIH5cCil3vaR9u~RY0Xmm6rAh6-I8Fl~hMXNX2pqwoMIdlVIAW60YY6jlMJ26-hXmqime3vrfEF6XHN25Y8qIrp-g8rYO0Pp7jGiMOUxazpPIXk71AXNEZQ-65Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200243&ar=2462472&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03706.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic relationship between haplotype-phased W34/70 sequences and Group 1 consensus sequences. (A) Identity distribution between haplotype-phased W34/70 chromosome sequences (W34/70_a, close with Group 1 sequences; W34/70_b, another chromosome sequence) and Group 1 sequences. (B) Identity distribution between haplotype-phased W34/70 chromosome sequences (hetero alleles are randomly shuffled dropping off the linkage information, average of 100 trials). (C) Phylogenetic tree based on haplotype-phased W34/70 sequences and Group 1 sequences. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87200244 id="87200244" class="section-title" data-legacy-id=s3i>3.9. Traces of frequent LOH events observed in haplotype-phased Sc-type chromosomes</h3>
<p>As mentioned above, the uneven distribution of hetero-SNVs might be evidence of frequent LOH events (Fig. <a reveal-id="DSV037F4" data-open="DSV037F4" class="link link-reveal link-table xref-fig">4</a>). In addition, we identified two examples of evidence of LOH in haplotype-phased intrahomologous Sc-type chromosomes. We also observed consecutive regions with hetero-SNVs and both types of homo alleles constituting corresponding hetero nucleotides in the Group 2 strains (Fig. <a reveal-id="DSV037F7" data-open="DSV037F7" class="link link-reveal link-table xref-fig">7</a>A). All SNVs of Group 2 in these regions—which span 16.0 kb—exhibited this pattern, and there were no exclusive SNVs. Moreover, the Group 1 strains exhibited either type of homo allele. Genomes with hetero-SNVs may be more ancestral type, with LOH events likely occurring independently in each strain, thus yielding two types of homo alleles at those loci. Another example is presented in Fig. <a reveal-id="DSV037F7" data-open="DSV037F7" class="link link-reveal link-table xref-fig">7</a>B; blocks spanning ∼17 kb indicate that the Group 1 sequences are nearly identical to either sequence in the hetero Group 2 strains (W34/70, CBS1483, and CBS1484). In addition, CBS2156 and CBS5832 lost heterozygosity and have the same sequences as Group 1 strains, with the exception of two continuous regions of homo-SNVs between Group 1 strains and Group 2 strains (∼3 and ∼5 kb regions shown in green boxes). The full elucidation of the evolutionary history of this 17-kb block is nearly impossible, but these regions suggest the existence of other LOH events for common Group 2 ancestor strains. These homo-SNV ‘island regions’ in the solved linkage blocks are the difference between the Group 1 and W34/70_a-type chromosome sequences. Therefore, Group 1–W34/70_a has different sequences and branch lengths remaining in the phylogenetic tree (Fig. <a reveal-id="DSV037F6" data-open="DSV037F6" class="link link-reveal link-table xref-fig">6</a>A). Here, we set a threshold length and divided into separate blocks if the distance between adjacent SNV loci exceeded the threshold. The phylogenetic trees resulting from threshold lengths of 1000 and 300 bp are presented in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv037_Supplementary_Data.zip?Expires=2147483647&Signature=TZn0N7h5~tgToQYT8xv52vTDsAS7RmRGsX8MFEc1uGykA0xtU8Y8Q47uIHEcB2odcIw3pk-fFfxaeW9AZDI0g2ENXJ9kQ8PFhfEiL~30bZk6EM8v6JHgz61ASkfbFS0oQRQDf1sIPANxIRcS6rMZfYgW9nSZ5jANdw72pHDijzsg2RaEuNcq-fPH~SrC1ZiXpzyoxDEXpnBjfgeznOVic37wKpVqA8xYPzyK8JzGFJwicJUJitpqbhMIiQ4lOPNoRcjq-UILR6AdTAIaGHupJNKrWfh~blOTsc3UGuebfB4-0KdbLxmGapsNbnaaghNuQSUqM~DKduF8-OAve~o3KQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>. The threshold length correlates with branch length in Group 1–W34/70_a. This phenomenon is further evidence of frequent LOH. If an LOH event occurred for a W34/70 chromosome, a homo-SNV region was established. If the opposite allele from Group 1 remained, a homo-SNV is observed in this region and caused branch length in the phylogenetic tree. If the same allele as Group 1 remained, no SNVs were detected in this continuous region. For example, in Fig. <a reveal-id="DSV037F7" data-open="DSV037F7" class="link link-reveal link-table xref-fig">7</a>B, no SNV regions are observed immediately right of the left region from 396,036 to 397,000 bp, which may be the result of an LOH with the corresponding remaining alleles in Group 1. When the threshold was set shorter than this silent SNV region, the blocks were divided. Thus, shorter thresholds facilitate the detection of shorter LOH regions, resulting in shorter branch lengths. </p>                        <a id="87200246" scrollto-destination="87200246"></a>
<div id="DSV037F7" data-id="DSV037F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/m_dsv03707.png?Expires=2147483647&Signature=ocuzrFbD6TNQ06Flb0tZOgCL6ASQnGI3zH1AUK9vmNQWWe9LqqMd9T6qqq4-n9AebF2xLMxK0bgpSoATxKlisGxdEVfk7zvI1lgHAQRMH3GWUrvX20J2LyRuoottNHavjZn2LXgprq5YLsBFjKxicJc74GZnDFJDhJbiuPTJgwuKAyqJDsxePjYsjC2vyNUFf~R-1r3e4CTSuktSVUfOOg09QnH6S31nZ4qL7xHWdvovdAfzAKLhLEmB7IZLS~7hj-8lLKk82qNBqCJHO4iEKG-1w1jk1~1g~43r7r2d3Mw6QYjcyAk2DZDCKpkDjfhYePMs2oZOzJ4oZJfAPp614g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evidence of LOH observed in haplotype-phased chromosomes. Two examples of regions in which LOH is expected to have occurred in some strains. (A) The common ancestor of the strains is considered to have had hetero-SNVs in these regions, and some strains are suspected to have undergone LOH, thus switching SNVs from the hetero to the homo type. (B) In this block, nested LOH events likely occurred. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03707.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200246/dsv03707.tiff" path-from-xml="dsv03707.tiff" target="_blank">View large</a><a section="87200246" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv03707.png?Expires=2147483647&Signature=4VKqFNaxQIdCDO3oFFUfzwstfBHzC6GEBlCRt0nA~Zi-c64W0mPaPrlqHOKngTxf9MeHitX9UqsgjqhHQizbsj0vk6lZG-GxWgmdGRfmx9ROHuQrF1Ujg2PgHwWKhiN1GoQuZTHggp4aVZL0NJ0GstyF9UM8hhxhrVLZiZlx9JeGBtY01Y6HOuQGHlUZYr9pQjOaTHUmVVkWfiGSTeViy1p-fs-sep5s8VkgVHJxPcnBkpI55djOGXbhNGMIFpFyRysXWWO5Iyu-k1dIqxbyfT-CKeM6c2gLsk2IycI4Bt-THynUMS8uw7amHuINhjwnjYRbbItj~54gpZn9D6oyyA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200246&ar=2462472&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03707.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evidence of LOH observed in haplotype-phased chromosomes. Two examples of regions in which LOH is expected to have occurred in some strains. (A) The common ancestor of the strains is considered to have had hetero-SNVs in these regions, and some strains are suspected to have undergone LOH, thus switching SNVs from the hetero to the homo type. (B) In this block, nested LOH events likely occurred. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV037F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/m_dsv03707.png?Expires=2147483647&Signature=ocuzrFbD6TNQ06Flb0tZOgCL6ASQnGI3zH1AUK9vmNQWWe9LqqMd9T6qqq4-n9AebF2xLMxK0bgpSoATxKlisGxdEVfk7zvI1lgHAQRMH3GWUrvX20J2LyRuoottNHavjZn2LXgprq5YLsBFjKxicJc74GZnDFJDhJbiuPTJgwuKAyqJDsxePjYsjC2vyNUFf~R-1r3e4CTSuktSVUfOOg09QnH6S31nZ4qL7xHWdvovdAfzAKLhLEmB7IZLS~7hj-8lLKk82qNBqCJHO4iEKG-1w1jk1~1g~43r7r2d3Mw6QYjcyAk2DZDCKpkDjfhYePMs2oZOzJ4oZJfAPp614g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evidence of LOH observed in haplotype-phased chromosomes. Two examples of regions in which LOH is expected to have occurred in some strains. (A) The common ancestor of the strains is considered to have had hetero-SNVs in these regions, and some strains are suspected to have undergone LOH, thus switching SNVs from the hetero to the homo type. (B) In this block, nested LOH events likely occurred. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03707.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200246/dsv03707.tiff" path-from-xml="dsv03707.tiff" target="_blank">View large</a><a section="87200246" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv03707.png?Expires=2147483647&Signature=4VKqFNaxQIdCDO3oFFUfzwstfBHzC6GEBlCRt0nA~Zi-c64W0mPaPrlqHOKngTxf9MeHitX9UqsgjqhHQizbsj0vk6lZG-GxWgmdGRfmx9ROHuQrF1Ujg2PgHwWKhiN1GoQuZTHggp4aVZL0NJ0GstyF9UM8hhxhrVLZiZlx9JeGBtY01Y6HOuQGHlUZYr9pQjOaTHUmVVkWfiGSTeViy1p-fs-sep5s8VkgVHJxPcnBkpI55djOGXbhNGMIFpFyRysXWWO5Iyu-k1dIqxbyfT-CKeM6c2gLsk2IycI4Bt-THynUMS8uw7amHuINhjwnjYRbbItj~54gpZn9D6oyyA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200246&ar=2462472&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03707.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evidence of LOH observed in haplotype-phased chromosomes. Two examples of regions in which LOH is expected to have occurred in some strains. (A) The common ancestor of the strains is considered to have had hetero-SNVs in these regions, and some strains are suspected to have undergone LOH, thus switching SNVs from the hetero to the homo type. (B) In this block, nested LOH events likely occurred. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87200247 id="87200247" class="section-title" data-legacy-id=s3j>3.10. The origin of <em>S. pastorianus</em> mtDNA</h3>
<p>To determine the origin of <em>S. pastorianus</em> mtDNA, the mtDNA of eight strains from three species was assembled into a circular scaffold. The mtDNA sizes and GC contents are presented in Table <a reveal-id="DSV037TB5" data-open="DSV037TB5" class="link link-reveal link-table xref-fig">5</a>. In <em>S. pastorianus</em> strains CBS1503, CBS1513, CBS1538, and W34/70, the mtDNA was ∼69 kb, smaller than that of <em>S. cerevisiae</em> S288C (85,779 bp) but slightly larger than those of <em>S. eubayanus</em> CBS12357 (64,290 bp) and BaiFY1 (67,205 bp). The <em>S. pastorianus</em> strain mtDNA exhibited conserved synteny with the mtDNA of <em>S. eubayanus</em> and <em>S. bayanus</em>. </p>                        <a id="87200249" scrollto-destination="87200249"></a>
<div content-id="DSV037TB5" class="table-modal"><div class="table-wrap"><div id="DSV037TB5" class="table-wrap-title" data-id="DSV037TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Mitochondrial genome in <em>S. pastorianus</em> strains and related species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Length (bp) </th><th>GC content (%) </th></tr></thead><tbody><tr><td colspan="3"><em>S. pastorianus</em> </td></tr><tr><td> CBS1503 </td><td>68,790 </td><td>18.97 </td></tr><tr><td> CBS1513 </td><td>69,294 </td><td>19.41 </td></tr><tr><td> CBS1538 </td><td>68,699 </td><td>19.24 </td></tr><tr><td> W34/70 </td><td>68,862 </td><td>19.04 </td></tr><tr><td colspan="3"><em>S. eubayanus</em> </td></tr><tr><td> CBS12357 </td><td>64,290 </td><td>17.48 </td></tr><tr><td> BaiFY1 </td><td>67,205 </td><td>18.43 </td></tr><tr><td colspan="3"><em>S.bayanus</em> </td></tr><tr><td> CBS380 </td><td>64,736 </td><td>16.32 </td></tr><tr><td> NBRC1948 </td><td>65,795 </td><td>19.51 </td></tr><tr><td colspan="3"><em>S. cerevisiae</em> </td></tr><tr><td> S288c </td><td>85,779 </td><td>17.11 </td></tr></tbody></table></div><div content-id="DSV037TB5" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Length (bp) </th><th>GC content (%) </th></tr></thead><tbody><tr><td colspan="3"><em>S. pastorianus</em> </td></tr><tr><td> CBS1503 </td><td>68,790 </td><td>18.97 </td></tr><tr><td> CBS1513 </td><td>69,294 </td><td>19.41 </td></tr><tr><td> CBS1538 </td><td>68,699 </td><td>19.24 </td></tr><tr><td> W34/70 </td><td>68,862 </td><td>19.04 </td></tr><tr><td colspan="3"><em>S. eubayanus</em> </td></tr><tr><td> CBS12357 </td><td>64,290 </td><td>17.48 </td></tr><tr><td> BaiFY1 </td><td>67,205 </td><td>18.43 </td></tr><tr><td colspan="3"><em>S.bayanus</em> </td></tr><tr><td> CBS380 </td><td>64,736 </td><td>16.32 </td></tr><tr><td> NBRC1948 </td><td>65,795 </td><td>19.51 </td></tr><tr><td colspan="3"><em>S. cerevisiae</em> </td></tr><tr><td> S288c </td><td>85,779 </td><td>17.11 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200249" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV037TB5" class="table-wrap-title" data-id="DSV037TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Mitochondrial genome in <em>S. pastorianus</em> strains and related species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Length (bp) </th><th>GC content (%) </th></tr></thead><tbody><tr><td colspan="3"><em>S. pastorianus</em> </td></tr><tr><td> CBS1503 </td><td>68,790 </td><td>18.97 </td></tr><tr><td> CBS1513 </td><td>69,294 </td><td>19.41 </td></tr><tr><td> CBS1538 </td><td>68,699 </td><td>19.24 </td></tr><tr><td> W34/70 </td><td>68,862 </td><td>19.04 </td></tr><tr><td colspan="3"><em>S. eubayanus</em> </td></tr><tr><td> CBS12357 </td><td>64,290 </td><td>17.48 </td></tr><tr><td> BaiFY1 </td><td>67,205 </td><td>18.43 </td></tr><tr><td colspan="3"><em>S.bayanus</em> </td></tr><tr><td> CBS380 </td><td>64,736 </td><td>16.32 </td></tr><tr><td> NBRC1948 </td><td>65,795 </td><td>19.51 </td></tr><tr><td colspan="3"><em>S. cerevisiae</em> </td></tr><tr><td> S288c </td><td>85,779 </td><td>17.11 </td></tr></tbody></table></div><div content-id="DSV037TB5" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Length (bp) </th><th>GC content (%) </th></tr></thead><tbody><tr><td colspan="3"><em>S. pastorianus</em> </td></tr><tr><td> CBS1503 </td><td>68,790 </td><td>18.97 </td></tr><tr><td> CBS1513 </td><td>69,294 </td><td>19.41 </td></tr><tr><td> CBS1538 </td><td>68,699 </td><td>19.24 </td></tr><tr><td> W34/70 </td><td>68,862 </td><td>19.04 </td></tr><tr><td colspan="3"><em>S. eubayanus</em> </td></tr><tr><td> CBS12357 </td><td>64,290 </td><td>17.48 </td></tr><tr><td> BaiFY1 </td><td>67,205 </td><td>18.43 </td></tr><tr><td colspan="3"><em>S.bayanus</em> </td></tr><tr><td> CBS380 </td><td>64,736 </td><td>16.32 </td></tr><tr><td> NBRC1948 </td><td>65,795 </td><td>19.51 </td></tr><tr><td colspan="3"><em>S. cerevisiae</em> </td></tr><tr><td> S288c </td><td>85,779 </td><td>17.11 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200249" target="_blank">View Large</a></div></div><p>Phylogenetic analysis based on the mitochondrial ORFs revealed that the mtDNA of <em>S. pastorianus</em> strains shares a common origin between Groups 1 and 2, with BaiFY1 exhibiting greater similarity in mtDNA to <em>S. pastorianus</em> (Fig. <a reveal-id="DSV037F8" data-open="DSV037F8" class="link link-reveal link-table xref-fig">8</a>). The mtDNA of interspecies hybrids can be inherited from either parental species. Although it was previously demonstrated that <em>S. pastorianus</em> mtDNA is not of the Sc type,<sup><a reveal-id="DSV037C2" data-open="DSV037C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSV037C6" data-open="DSV037C6" class="link link-ref link-reveal xref-bibr">6</a></sup> it could not be confirmed as Se type because the mtDNA sequence of <em>S. eubayanus</em> had not been determined. Interestingly, BaiFY1 mtDNA (isolated in Tibet)<sup><a reveal-id="DSV037C4" data-open="DSV037C4" class="link link-ref link-reveal xref-bibr">4</a></sup> differs considerably from that of CBS12357 (isolated in Patagonia),<sup><a reveal-id="DSV037C3" data-open="DSV037C3" class="link link-ref link-reveal xref-bibr">3</a></sup> with a sequence identity of 90–95% between the two strains despite being pure strains of the same species. Therefore, the Se-type genome of <em>S. pastorianus</em> strains is thought to originate from Tibet or its surrounding regions, consistent with previous studies.<sup><a reveal-id="DSV037C4" data-open="DSV037C4" class="link link-ref link-reveal xref-bibr">4</a></sup></p>                        <a id="87200251" scrollto-destination="87200251"></a>
<div id="DSV037F8" data-id="DSV037F8" class="fig fig-section"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/m_dsv03708.png?Expires=2147483647&Signature=aA7YbGenvXoR9ro8W6FvF12d~PfRSVp5hB6XPnHdWw5z0C3CL-HCtO3pRYoXZUGHtKT1d5p-Z70yCZ7wfD43p3rGqnZl7wyvNtOKz2cNCDHiVFgNpYI0UnvJq1jXnfSDQE1f9KWdQFJ0MxWwMr85t88SMAM7BKu2fXKfVRQH-ZpGt83VyVSVqxh6JQb740P9k5MrFrALqdZzx6Uhz0i5dc7dbs0NNzkdQuZBS13hPO5GFJo8PjPjMztdeAafPvSOgBjWo-J22Ek9xQuVKfEYpKITDFiDnIY46xib4my~svchR4zRWhO65GcWzwKqGu1imxX4LBEXlzwdiSKTvZ40HQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree based on mitochondrial ORFs. Shown are the phylogenetic relationships among S. pastorianus (CBS1503, CBS1513, CBS1538, and W34/70), its parental species S. cerevisiae (S288C) and S. eubayanus (CBS12357 and BaiFY1), and the related species S. bayanus (CBS380 and NBRC1948) based on mitochondrial ORFs. Four S. pastorianus strains (from Groups 1 and 2) are located in the same clade. Additionally, the mtDNA of S. eubayanus BaiFY1 is most closely related to that of S. pastorianus, suggesting that the latter originated from a single parental species (S. eubayanus). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03708.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200251/dsv03708.tiff" path-from-xml="dsv03708.tiff" target="_blank">View large</a><a section="87200251" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv03708.png?Expires=2147483647&Signature=anKFX6kJToaV4YrjNKZByTgztpjnZVFGuKa1PRA1-qHBLOWRHgv0ZaeOyvIhtP7G~4k5vUajqTfEGI03vhw5CBrd-y45QDTaCB2x4M9RIhNMCruNZAgQg62hadax4NadUw0cZGI7xu3wc01aNqVO~4mQHTH1eOSsgblAb2Llquzfm0OVaIqdX~WM1Upajliqia03WqJMOgF8qjbhbsyfTN7y~9oOcXAyLxV5SY8hMIMYBJUr6Kfz8zz3tb1vbetlRs8sgyFibmZ55zxOQFekWrqhiKmMIiibeVvQ35uBn5kYofOPWLrJxv2GWRITbkVeHWj-5IK9ivWW-RAhKhATVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200251&ar=2462472&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03708.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree based on mitochondrial ORFs. Shown are the phylogenetic relationships among <em>S. pastorianus</em> (CBS1503, CBS1513, CBS1538, and W34/70), its parental species <em>S. cerevisiae</em> (S288C) and <em>S. eubayanus</em> (CBS12357 and BaiFY1), and the related species <em>S. bayanus</em> (CBS380 and NBRC1948) based on mitochondrial ORFs. Four <em>S. pastorianus</em> strains (from Groups 1 and 2) are located in the same clade. Additionally, the mtDNA of <em>S. eubayanus</em> BaiFY1 is most closely related to that of <em>S. pastorianus</em>, suggesting that the latter originated from a single parental species (<em>S. eubayanus</em>). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV037F8" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/m_dsv03708.png?Expires=2147483647&Signature=aA7YbGenvXoR9ro8W6FvF12d~PfRSVp5hB6XPnHdWw5z0C3CL-HCtO3pRYoXZUGHtKT1d5p-Z70yCZ7wfD43p3rGqnZl7wyvNtOKz2cNCDHiVFgNpYI0UnvJq1jXnfSDQE1f9KWdQFJ0MxWwMr85t88SMAM7BKu2fXKfVRQH-ZpGt83VyVSVqxh6JQb740P9k5MrFrALqdZzx6Uhz0i5dc7dbs0NNzkdQuZBS13hPO5GFJo8PjPjMztdeAafPvSOgBjWo-J22Ek9xQuVKfEYpKITDFiDnIY46xib4my~svchR4zRWhO65GcWzwKqGu1imxX4LBEXlzwdiSKTvZ40HQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic tree based on mitochondrial ORFs. Shown are the phylogenetic relationships among S. pastorianus (CBS1503, CBS1513, CBS1538, and W34/70), its parental species S. cerevisiae (S288C) and S. eubayanus (CBS12357 and BaiFY1), and the related species S. bayanus (CBS380 and NBRC1948) based on mitochondrial ORFs. Four S. pastorianus strains (from Groups 1 and 2) are located in the same clade. Additionally, the mtDNA of S. eubayanus BaiFY1 is most closely related to that of S. pastorianus, suggesting that the latter originated from a single parental species (S. eubayanus). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03708.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200251/dsv03708.tiff" path-from-xml="dsv03708.tiff" target="_blank">View large</a><a section="87200251" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv03708.png?Expires=2147483647&Signature=anKFX6kJToaV4YrjNKZByTgztpjnZVFGuKa1PRA1-qHBLOWRHgv0ZaeOyvIhtP7G~4k5vUajqTfEGI03vhw5CBrd-y45QDTaCB2x4M9RIhNMCruNZAgQg62hadax4NadUw0cZGI7xu3wc01aNqVO~4mQHTH1eOSsgblAb2Llquzfm0OVaIqdX~WM1Upajliqia03WqJMOgF8qjbhbsyfTN7y~9oOcXAyLxV5SY8hMIMYBJUr6Kfz8zz3tb1vbetlRs8sgyFibmZ55zxOQFekWrqhiKmMIiibeVvQ35uBn5kYofOPWLrJxv2GWRITbkVeHWj-5IK9ivWW-RAhKhATVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200251&ar=2462472&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03708.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree based on mitochondrial ORFs. Shown are the phylogenetic relationships among <em>S. pastorianus</em> (CBS1503, CBS1513, CBS1538, and W34/70), its parental species <em>S. cerevisiae</em> (S288C) and <em>S. eubayanus</em> (CBS12357 and BaiFY1), and the related species <em>S. bayanus</em> (CBS380 and NBRC1948) based on mitochondrial ORFs. Four <em>S. pastorianus</em> strains (from Groups 1 and 2) are located in the same clade. Additionally, the mtDNA of <em>S. eubayanus</em> BaiFY1 is most closely related to that of <em>S. pastorianus</em>, suggesting that the latter originated from a single parental species (<em>S. eubayanus</em>). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div>                            <h3 scrollto-destination=87200252 id="87200252" class="section-title" data-legacy-id=s3k>3.11. Evolutionary history of <em>S. pastorianus</em></h3>
<p>On the basis of these results, we discuss whether the <em>S. pastorianus</em> strains originated from independent hybridization events or a common event. Translocation analysis revealed that the Group 1 and 2 strains share at least three interchromosomal translocations at the nucleotide level, indicative of common ancestors for both the Sc- and Se-type parental strains. The phylogenetic mtDNA genome analysis provided further support for a common origin (Tibet). In contrast, ploidy and phylogenetic analyses using SNVs against reference sequences suggested that the groups have different Sc-type ancestors and therefore independent evolutionary origins. The frequency of translocations is known to be much higher than that of point mutations;<sup><a reveal-id="DSV037C2" data-open="DSV037C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSV037C7" data-open="DSV037C7" class="link link-ref link-reveal xref-bibr">7</a></sup> therefore, SNVs provide more robust evidence. However, hetero-SNV analysis indicated that 17.0–39.2% of SNV sites between Groups 1 and 2 could be accounted for by consistent SNV relationships (Table <a reveal-id="DSV037TB4" data-open="DSV037TB4" class="link link-reveal link-table xref-fig">4</a>). Furthermore, phylogenetic analysis using haplotype-phased sequences suggested that Group 1 and Group 2 strains may share haploid Sc-type chromosomes as well as a common ancestor.</p><p>Based on these analyses, we hypothesized that at least one hybridization event was shared between Group 1 and 2 strains. Following this hypothesis, the difference in ploidy between the two groups can be explained by the chromosomal deletions in Group 1 (Fig. <a reveal-id="DSV037F9" data-open="DSV037F9" class="link link-reveal link-table xref-fig">9</a>A) or an additional hybrid Sc-type genome in Group 2 (Fig. <a reveal-id="DSV037F9" data-open="DSV037F9" class="link link-reveal link-table xref-fig">9</a>B). In the former scenario, a common ancestor originating from both Sc- and Se-types had a diploid genome, with subsequent chromosomal translocations; the ancestral Group 1 strains then experienced numerous chromosomal deletions in the Sc-type genome, leaving only some haploid chromosomes in the current Group 1 strains. In the latter scenario, a common ancestor originated from the first hybrid between Sc-type haploid and Se-type diploid genomes, followed by the formation of two groups after chromosomal translocations. In the Group 1 strains, chromosomal deletions in the Sc-type genome occurred, whereas the ancestral Group 2 strain interbred with another Sc-type species to produce a second hybrid. </p>                        <a id="87200255" scrollto-destination="87200255"></a>
<div id="DSV037F9" data-id="DSV037F9" class="fig fig-section"><div class="label fig-label">Figure 9.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/m_dsv03709.png?Expires=2147483647&Signature=XF3TpzXRNmOPBRgCEIF9I-KUo18q4DCxTSOGeCD6sm9tZJZ3yRwOQfWhUlV~6qz5tr7MR9ZSyjcqumDe~oWQ7U1pW2diMdnxYMQu-qoRB3HanWsQmkTIKRtVW2mBJGnUBANrcJaIVKIba5ZO7-3VrXsERiYM4LMNDwNn6tfkLJgyySzeSrZlqkWY1f6ruthRIwNlgTZa-mMEoSxvlyJsvfflvsberSWNl~uxcEBcjJ10U6oEMe~WpUqZ95G8SenKoLgGwJYeT4ApWJA-r4IMeUa2KYZh-LAw6689ngL49mPRo18AuzaGxvlMNegwgeohSW-Oh4d1T0lLfIHxM9TfTQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Two hypotheses regarding S. pastorianus origins based on shared chromosomal translocations and differences in ploidy between Groups 1 and 2. (A) Hybridization between diploid Sc and Se types occurred before chromosomal translocations, whereas chromosomal deletions occurred only in ancestral Group 1 strains. (B) After hybridization between haploid Sc and diploid Se types and chromosomal translocations, ancestral Group 2 strains gained another Sc type (i.e. a second hybridization event occurred). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03709.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200255/dsv03709.tiff" path-from-xml="dsv03709.tiff" target="_blank">View large</a><a section="87200255" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv03709.png?Expires=2147483647&Signature=OgEqNQgxQwmmahA63bcDZIKG8Zee-EbituC-ObDmywJ2LrwbSy5NPFFqU48uwjpZA5gjYg-b6JG57XMRJhxykADVIX6wGnMuT3fL~PjiHzLojpbZ8SWGz~WF18m6v7EQ1bwBdv4q4TKG7DNs-E465CHE4H24GAye7-eQkKZUwFfgvWoc0xs5BJHL-MP6PJhGaGerUcodyVk-ae~J6LVbq622UrznW~6u64vgPYpMh6hPGjLyEEF643H6-hHO~ss~w-8B6RXSQ9RvMwUS0-mb1OCqiAuCrTEjU4R0D574miiA4haqU4rJ8-v68MsRP~jz7eUtEcZjiBONFzjEPT1Aeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200255&ar=2462472&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03709.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Two hypotheses regarding <em>S. pastorianus</em> origins based on shared chromosomal translocations and differences in ploidy between Groups 1 and 2. (A) Hybridization between diploid Sc and Se types occurred before chromosomal translocations, whereas chromosomal deletions occurred only in ancestral Group 1 strains. (B) After hybridization between haploid Sc and diploid Se types and chromosomal translocations, ancestral Group 2 strains gained another Sc type (i.e. a second hybridization event occurred). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV037F9" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 9.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/m_dsv03709.png?Expires=2147483647&Signature=XF3TpzXRNmOPBRgCEIF9I-KUo18q4DCxTSOGeCD6sm9tZJZ3yRwOQfWhUlV~6qz5tr7MR9ZSyjcqumDe~oWQ7U1pW2diMdnxYMQu-qoRB3HanWsQmkTIKRtVW2mBJGnUBANrcJaIVKIba5ZO7-3VrXsERiYM4LMNDwNn6tfkLJgyySzeSrZlqkWY1f6ruthRIwNlgTZa-mMEoSxvlyJsvfflvsberSWNl~uxcEBcjJ10U6oEMe~WpUqZ95G8SenKoLgGwJYeT4ApWJA-r4IMeUa2KYZh-LAw6689ngL49mPRo18AuzaGxvlMNegwgeohSW-Oh4d1T0lLfIHxM9TfTQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Two hypotheses regarding S. pastorianus origins based on shared chromosomal translocations and differences in ploidy between Groups 1 and 2. (A) Hybridization between diploid Sc and Se types occurred before chromosomal translocations, whereas chromosomal deletions occurred only in ancestral Group 1 strains. (B) After hybridization between haploid Sc and diploid Se types and chromosomal translocations, ancestral Group 2 strains gained another Sc type (i.e. a second hybridization event occurred). This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03709.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200255/dsv03709.tiff" path-from-xml="dsv03709.tiff" target="_blank">View large</a><a section="87200255" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv03709.png?Expires=2147483647&Signature=OgEqNQgxQwmmahA63bcDZIKG8Zee-EbituC-ObDmywJ2LrwbSy5NPFFqU48uwjpZA5gjYg-b6JG57XMRJhxykADVIX6wGnMuT3fL~PjiHzLojpbZ8SWGz~WF18m6v7EQ1bwBdv4q4TKG7DNs-E465CHE4H24GAye7-eQkKZUwFfgvWoc0xs5BJHL-MP6PJhGaGerUcodyVk-ae~J6LVbq622UrznW~6u64vgPYpMh6hPGjLyEEF643H6-hHO~ss~w-8B6RXSQ9RvMwUS0-mb1OCqiAuCrTEjU4R0D574miiA4haqU4rJ8-v68MsRP~jz7eUtEcZjiBONFzjEPT1Aeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200255&ar=2462472&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03709.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Two hypotheses regarding <em>S. pastorianus</em> origins based on shared chromosomal translocations and differences in ploidy between Groups 1 and 2. (A) Hybridization between diploid Sc and Se types occurred before chromosomal translocations, whereas chromosomal deletions occurred only in ancestral Group 1 strains. (B) After hybridization between haploid Sc and diploid Se types and chromosomal translocations, ancestral Group 2 strains gained another Sc type (i.e. a second hybridization event occurred). This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>However, the remaining 60–80% of SNVs between these two groups are difficult to explain when only accounting for hetero–homo-SNV relationships. To address this point, we assumed that LOH events occurred with high frequency and could produce two homozygous alleles when occurring at a heterozygous location. Therefore, if Group 1 and 2 strains share common chromosomes (dark blue in Fig. <a reveal-id="DSV037F9" data-open="DSV037F9" class="link link-reveal link-table xref-fig">9</a>) and LOH occurred in the diploid ancestor of Group 2 strains (early <em>S. pastorianus</em> in Fig. <a reveal-id="DSV037F9" data-open="DSV037F9" class="link link-reveal link-table xref-fig">9</a>A or after the second hybridization strain in Fig. <a reveal-id="DSV037F9" data-open="DSV037F9" class="link link-reveal link-table xref-fig">9</a>B), these two alleles may be observed randomly. If the opposite allele of Group 1 strains was retained, this site would display a homo-SNV/homo-SNV relationship in the two groups (light blue region for Group 2 in Fig. <a reveal-id="DSV037F9" data-open="DSV037F9" class="link link-reveal link-table xref-fig">9</a>). However, the difference at this site would disappear if the same allele of Group 1 was retained. Consequently, homo-SNVs between the two groups were observed despite their shared common ancestral chromosomes. This speculation is supported by the distribution of hetero- and homo-SNVs between the Group 1 and 2 strains (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv037_Supplementary_Data.zip?Expires=2147483647&Signature=TZn0N7h5~tgToQYT8xv52vTDsAS7RmRGsX8MFEc1uGykA0xtU8Y8Q47uIHEcB2odcIw3pk-fFfxaeW9AZDI0g2ENXJ9kQ8PFhfEiL~30bZk6EM8v6JHgz61ASkfbFS0oQRQDf1sIPANxIRcS6rMZfYgW9nSZ5jANdw72pHDijzsg2RaEuNcq-fPH~SrC1ZiXpzyoxDEXpnBjfgeznOVic37wKpVqA8xYPzyK8JzGFJwicJUJitpqbhMIiQ4lOPNoRcjq-UILR6AdTAIaGHupJNKrWfh~blOTsc3UGuebfB4-0KdbLxmGapsNbnaaghNuQSUqM~DKduF8-OAve~o3KQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>). Hetero-SNVs, homo-SNV regions, and regions with no SNVs co-segregated as continuous blocks, possibly as a result of LOH corresponding to these regions. Furthermore, when we examined individual SNV allele sequences present in the blocks (i.e. at least five continuous SNV sites) in each strain—for which at least one of the alleles of the Group 2 strain exhibited hetero-SNVs—we determined that despite 512 blocks with 11,178 sites expanding to 2.34 Mb, there were only nine conflicting sites (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv037_Supplementary_Data.zip?Expires=2147483647&Signature=TZn0N7h5~tgToQYT8xv52vTDsAS7RmRGsX8MFEc1uGykA0xtU8Y8Q47uIHEcB2odcIw3pk-fFfxaeW9AZDI0g2ENXJ9kQ8PFhfEiL~30bZk6EM8v6JHgz61ASkfbFS0oQRQDf1sIPANxIRcS6rMZfYgW9nSZ5jANdw72pHDijzsg2RaEuNcq-fPH~SrC1ZiXpzyoxDEXpnBjfgeznOVic37wKpVqA8xYPzyK8JzGFJwicJUJitpqbhMIiQ4lOPNoRcjq-UILR6AdTAIaGHupJNKrWfh~blOTsc3UGuebfB4-0KdbLxmGapsNbnaaghNuQSUqM~DKduF8-OAve~o3KQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). These observations strongly suggest that the common ancestor of Group 2 had highly heterozygous chromosomes, of which one pair was shared with Group 1. After repeated chromosomal deletions and LOH in continuous regions, some blocks exhibited the homo-SNV/homo-SNV relationship; other blocks displayed no SNVs, and hetero-SNV/homo-SNVs were observed in the remaining 17–39% of the genome.</p>                            <h2 scrollto-destination=87200257 id="87200257" class="section-title" data-legacy-id=s4>4. Conclusions</h2>
<p><em>Saccharomyces pastorianus</em> is a useful species for investigating the evolutionary history of an interspecies hybrid. The genome sequences of the two parental species, <em>S. cerevisiae</em> and <em>S. eubayanus</em>, are available as reference genomes for analysis and can provide details of genome rearrangement events, including interchromosomal translocations, as well as chromosomal deletions and duplications following hybridization. We sequenced 14 strains of three species, including 10 strains of <em>S. pastorianus</em>, by NGS, and we compared the sequences to determine the phylogenetic relationships between <em>S. pastorianus</em> and its relatives. There are two conflicting hypotheses about whether the two <em>S. pastorianus</em> groups are derived from separate or a single hybridization event. Our analyses provide strong evidence that the haploid Sc-type chromosomes of Group 1 genomes and one of the intrahomologous diploid Sc-type chromosomes of Group 2 genomes were shared. Thus, the two <em>S. pastorianus</em> groups are not independent; they may have originated from the same parental species (<em>S. cerevisiae</em> and <em>S. eubayanus</em>) and share at least one hybridization event. Many of the SNVs shared by the two groups may have resulted from chromosomal deletion with LOH as an adaptive strategy following hybridization. This evolutionary scenario resolves all of the apparent contradictions, yet the genome sequence alone cannot reveal the driving force behind chromosomal deletions and LOH. Experimental approaches—such as the detection of LOH after hybridization between genomes exhibiting high vs. low heterozygosity—may provide additional insights into these processes.</p>                            <h2 scrollto-destination=87200259 id="87200259" class="section-title" data-legacy-id=s5>5. Availability</h2>
<p>Genome sequences were submitted to the DDBJ, EMBL, and GenBank DNA databases. The accession numbers are BBYU01000000, BBYV01000000, BBYW01000000, and BBYY01000000.</p>                            <h2 scrollto-destination=87200261 id="87200261" class="section-title" data-legacy-id=s7>Funding</h2>
<p>Funding to pay the Open Access publication charges for this article was provided by the Ministry of Education, Culture, Sports, Science and Technology, Japan.</p>                            <h2 scrollto-destination=87200263 id="87200263" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank the members of the laboratories of H. Iwasaki and T. Itoh for their valuable participation in scientific discussions. The expert technical assistance of M. Ogawa is gratefully acknowledged.</p>                            <h2 scrollto-destination=87200265 id="87200265" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV037C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="comment">P, L.</div>
          <div class="year">1909</div>, <div class="comment">Mikroskopische Betriebskontrolle in den Garungswerben 6th edn</div>.</div></div></div></div><div content-id="DSV037C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gibson</div>
               <div class="given-names">B.R.</div>
            </div>, <div class="name string-name"><div class="surname">Storgårds</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Krogerus</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Vidgren</div> <div class="given-names">V.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Comparative physiology and fermentation performance of Saaz and Frohberg lager yeast strains and the parental species <em>Saccharomyces eubayanus</em></div>, <div class="source">Yeast</div> , <div class="volume">30</div>, <div class="fpage">255</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20physiology%20and%20fermentation%20performance%20of%20Saaz%20and%20Frohberg%20lager%20yeast%20strains%20and%20the%20parental%20species%20Saccharomyces%20eubayanus&amp;author=B.R.%20Gibson&amp;author=E.%20Storg%C3%A5rds&amp;author=K.%20Krogerus&amp;author=V.%20Vidgren&amp;publication_year=2013&amp;journal=Yeast&amp;volume=30&amp;pages=255-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/yea.v30.7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fyea.v30.7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fyea.v30.7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23695993" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV037C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Libkind</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Hittinger</div> <div class="given-names">C.T.</div></div>, <div class="name string-name"><div class="surname">Valério</div> <div class="given-names">E.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Microbe domestication and the identification of the wild genetic stock of lager-brewing yeast</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">108</div>, <div class="fpage">14539</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microbe%20domestication%20and%20the%20identification%20of%20the%20wild%20genetic%20stock%20of%20lager-brewing%20yeast&amp;author=D.%20Libkind&amp;author=C.T.%20Hittinger&amp;author=E.%20Val%C3%A9rio&amp;publication_year=2011&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=14539-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1105430108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1105430108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1105430108"> </span></div> </div></div></div></div></div><div content-id="DSV037C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bing</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Han</div> <div class="given-names">P.J.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">W.Q.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Q.M.</div></div>, <div class="name string-name"><div class="surname">Bai</div> <div class="given-names">F.Y.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Evidence for a Far East Asian origin of lager beer yeast</div>, <div class="source">Curr. Biol.</div> , <div class="volume">24</div>, <div class="fpage">R380</div>–<div class="lpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20a%20Far%20East%20Asian%20origin%20of%20lager%20beer%20yeast&amp;author=J.%20Bing&amp;author=P.J.%20Han&amp;author=W.Q.%20Liu&amp;author=Q.M.%20Wang&amp;author=F.Y.%20Bai&amp;publication_year=2014&amp;journal=Curr.%20Biol.&amp;volume=24&amp;pages=R380-1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cub.2014.04.031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cub.2014.04.031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cub.2014.04.031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24845661" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV037C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hewitt</div>
               <div class="given-names">S.K.</div>
            </div>, <div class="name string-name"><div class="surname">Donaldson</div> <div class="given-names">I.J.</div></div>, <div class="name string-name"><div class="surname">Lovell</div> <div class="given-names">S.C.</div></div>, <div class="name string-name"><div class="surname">Delneri</div> <div class="given-names">D.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Sequencing and characterisation of rearrangements in three <em>S. pastorianus</em> strains reveals the presence of chimeric genes and gives evidence of breakpoint reuse</div>, <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e92203</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20and%20characterisation%20of%20rearrangements%20in%20three%20S.%20pastorianus%20strains%20reveals%20the%20presence%20of%20chimeric%20genes%20and%20gives%20evidence%20of%20breakpoint%20reuse&amp;author=S.K.%20Hewitt&amp;author=I.J.%20Donaldson&amp;author=S.C.%20Lovell&amp;author=D.%20Delneri&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e92203" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0092203" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0092203" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0092203"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24643015" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV037C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nakao</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Kanamori</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Itoh</div> <div class="given-names">T.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">Genome sequence of the lager brewing yeast, an interspecies hybrid</div>, <div class="source">DNA Res.</div> , <div class="volume">16</div>, <div class="fpage">115</div>–<div class="lpage">29</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20lager%20brewing%20yeast%2C%20an%20interspecies%20hybrid&amp;author=Y.%20Nakao&amp;author=T.%20Kanamori&amp;author=T.%20Itoh&amp;publication_year=2009&amp;journal=DNA%20Res.&amp;volume=16&amp;pages=115-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsp003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsp003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsp003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19261625" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV037C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Louis</div>
               <div class="given-names">V.L.</div>
            </div>, <div class="name string-name"><div class="surname">Despons</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Friedrich</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title"><em>Pichia sorbitophila</em>, an interspecies yeast hybrid, reveals early steps of genome resolution after polyploidization</div>, <div class="source">G3 Genes, Genomes, Genet.</div> , <div class="volume">2</div>, <div class="fpage">299</div>–<div class="lpage">311</div>.</div></div></div></div><div content-id="DSV037C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mira</div>
               <div class="given-names">N.P.</div>
            </div>, <div class="name string-name"><div class="surname">Münsterkötter</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Dias-Valada</div> <div class="given-names">F.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">The genome sequence of the highly acetic acid-tolerant <em>Zygosaccharomyces bailii</em>-derived interspecies hybrid strain ISA1307, isolated from a sparkling wine plant,</div><div class="source">DNA Res.</div> , <div class="volume">21</div>, <div class="fpage">299</div>–<div class="lpage">313</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20the%20highly%20acetic%20acid-tolerant%20Zygosaccharomyces%20bailii-derived%20interspecies%20hybrid%20strain%20ISA1307%2C%20isolated%20from%20a%20sparkling%20wine%20plant%2C&amp;author=N.P.%20Mira&amp;author=M.%20M%C3%BCnsterk%C3%B6tter&amp;author=F.%20Dias-Valada&amp;publication_year=2014&amp;journal=DNA%20Res.&amp;volume=21&amp;pages=299-313" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst058" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst058" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst058"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24453040" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV037C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Borneman</div>
               <div class="given-names">A.R.</div>
            </div>, <div class="name string-name"><div class="surname">Desany</div> <div class="given-names">B.A.</div></div>, <div class="name string-name"><div class="surname">Riches</div> <div class="given-names">D.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">The genome sequence of the wine yeast VIN7 reveals an allotriploid hybrid genome with <em>Saccharomyces cerevisiae</em> and <em>Saccharomyces kudriavzevii</em> origins</div>, <div class="source">FEMS Yeast Res.</div> , <div class="volume">12</div>, <div class="fpage">88</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20the%20wine%20yeast%20VIN7%20reveals%20an%20allotriploid%20hybrid%20genome%20with%20Saccharomyces%20cerevisiae%20and%20Saccharomyces%20kudriavzevii%20origins&amp;author=A.R.%20Borneman&amp;author=B.A.%20Desany&amp;author=D.%20Riches&amp;publication_year=2012&amp;journal=FEMS%20Yeast%20Res.&amp;volume=12&amp;pages=88-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fyr.2012.12.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffyr.2012.12.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffyr.2012.12.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22136070" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV037C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tadami</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Shikata-Miyoshi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Ogata</div> <div class="given-names">T.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Aneuploidy, copy number variation and unique chromosomal structures in bottom-fermenting yeast revealed by array-CGH</div>, <div class="source">J. Inst. Brew.</div> , <div class="volume">120</div>, <div class="fpage">27</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aneuploidy%2C%20copy%20number%20variation%20and%20unique%20chromosomal%20structures%20in%20bottom-fermenting%20yeast%20revealed%20by%20array-CGH&amp;author=H.%20Tadami&amp;author=M.%20Shikata-Miyoshi&amp;author=T.%20Ogata&amp;publication_year=2014&amp;journal=J.%20Inst.%20Brew.&amp;volume=120&amp;pages=27-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/jib.v120.1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fjib.v120.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fjib.v120.1"> </span></div> </div></div></div></div></div><div content-id="DSV037C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dunn</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Sherlock</div> <div class="given-names">G.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Reconstruction of the genome origins and evolution of the hybrid lager yeast <em>Saccharomyces pastorianus</em></div>, <div class="source">Genome Res.</div> , <div class="volume">18</div>, <div class="fpage">1610</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reconstruction%20of%20the%20genome%20origins%20and%20evolution%20of%20the%20hybrid%20lager%20yeast%20Saccharomyces%20pastorianus&amp;author=B.%20Dunn&amp;author=G.%20Sherlock&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=1610-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.076075.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.076075.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.076075.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18787083" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV037C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liti</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Peruffo</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">James</div> <div class="given-names">S.A.</div></div>, <div class="name string-name"><div class="surname">Roberts</div> <div class="given-names">I.N.</div></div>, <div class="name string-name"><div class="surname">Louis</div> <div class="given-names">E.J.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">Inferences of evolutionary relationships from a population survey of LTR-retrotransposons and telomeric-associated sequences in the <em>Saccharomyces sensu stricto</em> complex</div>, <div class="source">Yeast</div> , <div class="volume">22</div>, <div class="fpage">177</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inferences%20of%20evolutionary%20relationships%20from%20a%20population%20survey%20of%20LTR-retrotransposons%20and%20telomeric-associated%20sequences%20in%20the%20Saccharomyces%20sensu%20stricto%20complex&amp;author=G.%20Liti&amp;author=A.%20Peruffo&amp;author=S.A.%20James&amp;author=I.N.%20Roberts&amp;author=E.J.%20Louis&amp;publication_year=2005&amp;journal=Yeast&amp;volume=22&amp;pages=177-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15704235" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV037C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Monerawela</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">James</div> <div class="given-names">T.C.</div></div>, <div class="name string-name"><div class="surname">Wolfe</div> <div class="given-names">K.H.</div></div>, <div class="name string-name"><div class="surname">Bond</div> <div class="given-names">U.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">Loss of lager specific genes and subtelomeric regions define two different <em>Saccharomyces cerevisiae</em> lineages for <em>Saccharomyces pastorianus</em> Group I and II strains</div>, <div class="source">FEMS Yeast Res.</div> , <div class="volume">15</div>, <div class="fpage">fou008</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Loss%20of%20lager%20specific%20genes%20and%20subtelomeric%20regions%20define%20two%20different%20Saccharomyces%20cerevisiae%20lineages%20for%20Saccharomyces%20pastorianus%20Group%20I%20and%20II%20strains&amp;author=C.%20Monerawela&amp;author=T.C.%20James&amp;author=K.H.%20Wolfe&amp;author=U.%20Bond&amp;publication_year=2015&amp;journal=FEMS%20Yeast%20Res.&amp;volume=15&amp;pages=fou008" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/femsyr/fou008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Ffemsyr%2Ffou008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Ffemsyr%2Ffou008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25673756" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV037C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kajitani</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Toshimoto</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Noguchi</div> <div class="given-names">H.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads</div>, <div class="source">Genome Res.</div> , <div class="volume">24</div>, <div class="fpage">1384</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20de%20novo%20assembly%20of%20highly%20heterozygous%20genomes%20from%20whole-genome%20shotgun%20short%20reads&amp;author=R.%20Kajitani&amp;author=K.%20Toshimoto&amp;author=H.%20Noguchi&amp;publication_year=2014&amp;journal=Genome%20Res.&amp;volume=24&amp;pages=1384-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.170720.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.170720.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.170720.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24755901" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV037C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV037C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Altschul</div>
               <div class="given-names">S.F.</div>
            </div>, <div class="name string-name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>, <div class="name string-name"><div class="surname">Schäffer</div> <div class="given-names">A.A.</div></div>et al.  </div>. <div class="year">1997</div>, <div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">25</div>, <div class="fpage">3389</div>–<div class="lpage">402</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Sch%C3%A4ffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV037C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hebly</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Brickwedde</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Bolat</div> <div class="given-names">I.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title"><em>S. cerevisiae</em> × <em>S. eubayanus</em> interspecific hybrid, the best of both worlds and beyond</div>, <div class="source">FEMS Yeast Res.</div> , <div class="volume">15</div>, <div class="fpage">fov005</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=S.%20cerevisiae%20%C3%97%20S.%20eubayanus%20interspecific%20hybrid%2C%20the%20best%20of%20both%20worlds%20and%20beyond&amp;author=M.%20Hebly&amp;author=A.%20Brickwedde&amp;author=I.%20Bolat&amp;publication_year=2015&amp;journal=FEMS%20Yeast%20Res.&amp;volume=15&amp;pages=fov005" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/femsyr/fov005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Ffemsyr%2Ffov005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Ffemsyr%2Ffov005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25743788" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV037C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Krzywinski</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Schein</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Birol</div> <div class="given-names">I.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">Circos: an information aesthetic for comparative genomics</div>, <div class="source">Genome Res.</div> , <div class="volume">19</div>, <div class="fpage">1639</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circos%3A%20an%20information%20aesthetic%20for%20comparative%20genomics&amp;author=M.%20Krzywinski&amp;author=J.%20Schein&amp;author=I.%20Birol&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=1639-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.092759.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.092759.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.092759.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19541911" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV037C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">McKenna</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Hanna</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Banks</div> <div class="given-names">E.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</div>, <div class="source">Genome Res.</div> , <div class="volume">20</div>, <div class="fpage">1297</div>–<div class="lpage">303</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Genome%20Analysis%20Toolkit%3A%20a%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&amp;author=A.%20McKenna&amp;author=M.%20Hanna&amp;author=E.%20Banks&amp;publication_year=2010&amp;journal=Genome%20Res.&amp;volume=20&amp;pages=1297-303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.107524.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.107524.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.107524.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20644199" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV037C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Handsaker</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wysoker</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">The Sequence Alignment/Map format and SAMtools</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2078</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Sequence%20Alignment%2FMap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV037C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Guindon</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Lethiec</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Duroux</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Gascuel</div> <div class="given-names">O.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">PHYML Online—a web server for fast maximum likelihood-based phylogenetic inference</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">33</div>, <div class="fpage">557</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PHYML%20Online%E2%80%94a%20web%20server%20for%20fast%20maximum%20likelihood-based%20phylogenetic%20inference&amp;author=S.%20Guindon&amp;author=F.%20Lethiec&amp;author=P.%20Duroux&amp;author=O.%20Gascuel&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=557-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki352"> </span></div> </div></div></div></div></div><div content-id="DSV037C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Walther</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Hesselbart</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Wendland</div> <div class="given-names">J.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Genome sequence of <em>Saccharomyces carlsbergensis</em>, the world's first pure culture lager yeast</div>, <div class="source">G3 Genes, Genomes, Genet.</div> , <div class="volume">4</div>, <div class="fpage">783</div>–<div class="lpage">93</div>.</div></div></div></div><div content-id="DSV037C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Borneman</div>
               <div class="given-names">A.R.</div>
            </div>, <div class="name string-name"><div class="surname">Desany</div> <div class="given-names">B.A.</div></div>, <div class="name string-name"><div class="surname">Riches</div> <div class="given-names">D.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Whole-genome comparison reveals novel genetic elements that characterize the genome of industrial strains of <em>Saccharomyces cerevisiae</em></div>, <div class="source">PLoS Genet.</div> , <div class="volume">7</div>, <div class="fpage">e1001287</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20comparison%20reveals%20novel%20genetic%20elements%20that%20characterize%20the%20genome%20of%20industrial%20strains%20of%20Saccharomyces%20cerevisiae&amp;author=A.R.%20Borneman&amp;author=B.A.%20Desany&amp;author=D.%20Riches&amp;publication_year=2011&amp;journal=PLoS%20Genet.&amp;volume=7&amp;pages=e1001287" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1001287" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1001287" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1001287"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21304888" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV037C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Akao</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Yashiro</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Hosoyama</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Whole-genome sequencing of sake yeast <em>Saccharomyces cerevisiae</em> Kyokai no. 7</div>, <div class="source">DNA Res.</div> , <div class="volume">18</div>, <div class="fpage">423</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20sequencing%20of%20sake%20yeast%20Saccharomyces%20cerevisiae%20Kyokai%20no.%207&amp;author=T.%20Akao&amp;author=I.%20Yashiro&amp;author=A.%20Hosoyama&amp;publication_year=2011&amp;journal=DNA%20Res.&amp;volume=18&amp;pages=423-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsr029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsr029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsr029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21900213" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV037C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Babrzadeh</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Jalili</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">C.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Whole-genome sequencing of the efficient industrial fuel-ethanol fermentative <em>Saccharomyces cerevisiae</em> strain CAT-1</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="volume">287</div>, <div class="fpage">485</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20sequencing%20of%20the%20efficient%20industrial%20fuel-ethanol%20fermentative%20Saccharomyces%20cerevisiae%20strain%20CAT-1&amp;author=F.%20Babrzadeh&amp;author=R.%20Jalili&amp;author=C.%20Wang&amp;publication_year=2012&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=287&amp;pages=485-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-012-0695-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-012-0695-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-012-0695-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22562254" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV037C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rainieri</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Kodama</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Nakao</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Pulvirenti</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Giudici</div> <div class="given-names">P.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">The inheritance of mtDNA in lager brewing strains</div>, <div class="source">FEMS Yeast Res.</div> , <div class="volume">8</div>, <div class="fpage">586</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20inheritance%20of%20mtDNA%20in%20lager%20brewing%20strains&amp;author=S.%20Rainieri&amp;author=Y.%20Kodama&amp;author=Y.%20Nakao&amp;author=A.%20Pulvirenti&amp;author=P.%20Giudici&amp;publication_year=2008&amp;journal=FEMS%20Yeast%20Res.&amp;volume=8&amp;pages=586-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fyr.2008.8.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffyr.2008.8.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffyr.2008.8.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18318709" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV037C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV037C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hiraoka</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Watanabe</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Umezu</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Maki</div> <div class="given-names">H.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Spontaneous loss of heterozygosity in diploid <em>Saccharomyces cerevisiae</em> cells</div>, <div class="source">Genetics</div> , <div class="volume">156</div>, <div class="fpage">1531</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Spontaneous%20loss%20of%20heterozygosity%20in%20diploid%20Saccharomyces%20cerevisiae%20cells&amp;author=M.%20Hiraoka&amp;author=K.%20Watanabe&amp;author=K.%20Umezu&amp;author=H.%20Maki&amp;publication_year=2000&amp;journal=Genetics&amp;volume=156&amp;pages=1531-48" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11102355" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2462472&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv037/1/dsv037_Supplementary_Data.zip?Expires=2147483647&amp;Signature=TZn0N7h5~tgToQYT8xv52vTDsAS7RmRGsX8MFEc1uGykA0xtU8Y8Q47uIHEcB2odcIw3pk-fFfxaeW9AZDI0g2ENXJ9kQ8PFhfEiL~30bZk6EM8v6JHgz61ASkfbFS0oQRQDf1sIPANxIRcS6rMZfYgW9nSZ5jANdw72pHDijzsg2RaEuNcq-fPH~SrC1ZiXpzyoxDEXpnBjfgeznOVic37wKpVqA8xYPzyK8JzGFJwicJUJitpqbhMIiQ4lOPNoRcjq-UILR6AdTAIaGHupJNKrWfh~blOTsc3UGuebfB4-0KdbLxmGapsNbnaaghNuQSUqM~DKduF8-OAve~o3KQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">769</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000370973100007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">30</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv037"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2462472"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000370973100007&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f1%2f67%2f2462472">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000370973100007&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (30)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F1%2F67%2F2462472" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2462472'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Next-generation sequencing analysis of lager brewing yeast strains reveals the evolutionary history of interspecies hybridization" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv037");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f1%2f67%2f2462472">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

