

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/5/395/629019","@type":"ScholarlyArticle","name":"Mate Pair Sequencing of Whole-Genome-Amplified DNA Following Laser Capture Microdissection of Prostate Cancer","datePublished":"2012-09-18","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2012-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss021","keywords":["mate pair sequencing","whole-genome amplified, laser capture microdissection","prostrate cancer"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/5/395/629019","author":[{"name":"Murphy, Stephen J.","affiliation":"1  Department of Molecular Medicine ,  Mayo Clinic ,  Medical Sciences Building 2, 200 First St., SW, Rochester, MN 55905 ,  USA","@type":"Person"},{"name":"Cheville, John C.","affiliation":"2  Department of Laboratory Medicine and Pathology ,  Mayo Clinic ,  Rochester, MN 55905 ,  USA","@type":"Person"},{"name":"Zarei, Shabnam","affiliation":"1  Department of Molecular Medicine ,  Mayo Clinic ,  Medical Sciences Building 2, 200 First St., SW, Rochester, MN 55905 ,  USA","@type":"Person"},{"name":"Johnson, Sarah H.","affiliation":"1  Department of Molecular Medicine ,  Mayo Clinic ,  Medical Sciences Building 2, 200 First St., SW, Rochester, MN 55905 ,  USA","@type":"Person"},{"name":"Sikkink, Robert A.","affiliation":"3  Advanced Genomics Technology Center ,  Mayo Clinic ,  Rochester, MN 55905 ,  USA","@type":"Person"},{"name":"Kosari, Farhad","affiliation":"1  Department of Molecular Medicine ,  Mayo Clinic ,  Medical Sciences Building 2, 200 First St., SW, Rochester, MN 55905 ,  USA","@type":"Person"},{"name":"Feldman, Andrew L.","affiliation":"2  Department of Laboratory Medicine and Pathology ,  Mayo Clinic ,  Rochester, MN 55905 ,  USA","@type":"Person"},{"name":"Eckloff, Bruce W.","affiliation":"3  Advanced Genomics Technology Center ,  Mayo Clinic ,  Rochester, MN 55905 ,  USA","@type":"Person"},{"name":"Karnes, R. Jeffrey","affiliation":"4  Department of Urology ,  Mayo Clinic ,  Rochester, MN 55905 ,  USA","@type":"Person"},{"name":"Vasmatzis, George","affiliation":"1  Department of Molecular Medicine ,  Mayo Clinic ,  Medical Sciences Building 2, 200 First St., SW, Rochester, MN 55905 ,  USA","@type":"Person"}],"description":"Abstract.  High-throughput next-generation sequencing provides a revolutionary platform to unravel the precise DNA aberrations concealed within subgroups of tum","pageStart":"395","pageEnd":"406","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/5/1/m_cover.gif?Expires=2147483647&Signature=oCF62cOF-RHSHHzNh6fqRIkUeFdGFZWfw3pKB6h-XrXTUmjPkfpfn8w~8bchsCbKMP2r~GVw22lZ1MNx0q-s71-AcQyMN0lvE0QjuCe~oMOmvkISrjvYiWPgB2qMv~4jOVZX-tDdrzrIeyUlp6-dE-d1BMbDpzvRnY6ImlMPnjamQ8tNFPiRg84G~Uknop7I8711Uikreov-Syq2nbs4ZI3p39Ios3nBrhdKb3cghqgrrGVSW3I0iXxFPXkjQY1cpzGh8rn-oZc~FVxVdjoBppF4WPcNdaRzbNDAFK0kliJ6Rs5rfCQy64JYiBPIa6tUr3wfBbp66X2fB6q0zAaV3g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Mate Pair Sequencing of Whole-Genome-Amplified DNA Following Laser Capture Microdissection of Prostate Cancer","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/5/1/m_cover.gif?Expires=2147483647&Signature=oCF62cOF-RHSHHzNh6fqRIkUeFdGFZWfw3pKB6h-XrXTUmjPkfpfn8w~8bchsCbKMP2r~GVw22lZ1MNx0q-s71-AcQyMN0lvE0QjuCe~oMOmvkISrjvYiWPgB2qMv~4jOVZX-tDdrzrIeyUlp6-dE-d1BMbDpzvRnY6ImlMPnjamQ8tNFPiRg84G~Uknop7I8711Uikreov-Syq2nbs4ZI3p39Ios3nBrhdKb3cghqgrrGVSW3I0iXxFPXkjQY1cpzGh8rn-oZc~FVxVdjoBppF4WPcNdaRzbNDAFK0kliJ6Rs5rfCQy64JYiBPIa6tUr3wfBbp66X2fB6q0zAaV3g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Mate Pair Sequencing of Whole-Genome-Amplified DNA Following Laser Capture Microdissection of Prostate Cancer" />
    <meta property="og:description" content="Abstract.  High-throughput next-generation sequencing provides a revolutionary platform to unravel the precise DNA aberrations concealed within subgroups of tum" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/5/395/629019" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2012-09-18" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Murphy, Stephen J." /><meta name="citation_author_institution" content="Department of Molecular Medicine, Mayo Clinic, Medical Sciences Building 2, 200 First St., SW, Rochester, MN 55905, USA" /><meta name="citation_author" content="Cheville, John C." /><meta name="citation_author_institution" content="Department of Laboratory Medicine and Pathology, Mayo Clinic, Rochester, MN 55905, USA" /><meta name="citation_author" content="Zarei, Shabnam" /><meta name="citation_author_institution" content="Department of Molecular Medicine, Mayo Clinic, Medical Sciences Building 2, 200 First St., SW, Rochester, MN 55905, USA" /><meta name="citation_author" content="Johnson, Sarah H." /><meta name="citation_author_institution" content="Department of Molecular Medicine, Mayo Clinic, Medical Sciences Building 2, 200 First St., SW, Rochester, MN 55905, USA" /><meta name="citation_author" content="Sikkink, Robert A." /><meta name="citation_author_institution" content="Advanced Genomics Technology Center, Mayo Clinic, Rochester, MN 55905, USA" /><meta name="citation_author" content="Kosari, Farhad" /><meta name="citation_author_institution" content="Department of Molecular Medicine, Mayo Clinic, Medical Sciences Building 2, 200 First St., SW, Rochester, MN 55905, USA" /><meta name="citation_author" content="Feldman, Andrew L." /><meta name="citation_author_institution" content="Department of Laboratory Medicine and Pathology, Mayo Clinic, Rochester, MN 55905, USA" /><meta name="citation_author" content="Eckloff, Bruce W." /><meta name="citation_author_institution" content="Advanced Genomics Technology Center, Mayo Clinic, Rochester, MN 55905, USA" /><meta name="citation_author" content="Karnes, R. Jeffrey" /><meta name="citation_author_institution" content="Department of Urology, Mayo Clinic, Rochester, MN 55905, USA" /><meta name="citation_author" content="Vasmatzis, George" /><meta name="citation_author_institution" content="Department of Molecular Medicine, Mayo Clinic, Medical Sciences Building 2, 200 First St., SW, Rochester, MN 55905, USA" /><meta name="citation_title" content="Mate Pair Sequencing of Whole-Genome-Amplified DNA Following Laser Capture Microdissection of Prostate Cancer" /><meta name="citation_firstpage" content="395" /><meta name="citation_lastpage" content="406" /><meta name="citation_doi" content="10.1093/dnares/dss021" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2012/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Common gene rearrangements in prostate cancer; J. Clin. Oncol.;  citation_year=2012;  citation_volume=30;  citation_pages=608-15; " /><meta name="citation_reference" content="citation_title=Clinical features and prognosis of de novo diffuse large B-cell lymphoma with t(14;18) and 8q24/c-MYC translocations; Leukemia;  citation_year=2009;  citation_volume=23;  citation_pages=777-83; " /><meta name="citation_reference" content="citation_title=BCL-ABL; a multifaceted promoter of DNA mutation in chronic myelogenous leukemia; Leukemia;  citation_year=2010;  citation_volume=24;  citation_pages=1105-12; " /><meta name="citation_reference" content="citation_title=Translocation (18;22) (q21;q11) in B-cell lymphomas: a report of 4 cases and review of the literature; Hum. Pathol.;  citation_year=2008;  citation_volume=39;  citation_pages=1664-72; " /><meta name="citation_reference" content="Illumina Mate Pair Library v2 Sample Preparation Guide for 2â€“5 kb Libraries 2009 http://nextgen.mgh.harvard.edu/attachments/MatePair_v2_2-5kb_SamplePrep_Guide_15008135_A.pdf " /><meta name="citation_reference" content="citation_title=Discovery of recurrent t(6;7)(p25.3;q32.3) translocations in ALK-negative anaplastic large cell lymphomas by massively parallel genomic sequencing; Blood;  citation_year=2011;  citation_volume=117;  citation_pages=915-9; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis reveals recurrent structural abnormalities of TP63 and other p53-related genes in peritheral T-cell lymphomas; citation_text=2012 Aug 1. [Epub ahead of print] PMID: 22855598; Blood;  citation_year=2012; " /><meta name="citation_reference" content="citation_title=An update of the Gleason grading system; J. Urol.;  citation_year=2010;  citation_volume=183;  citation_pages=433-40; " /><meta name="citation_reference" content="citation_title=Gleason score 7 prostate cancer on needle biopsy: relation of primary pattern 3 or 4 to pathological stage and progression after radical prostatectomy; J. Urol.;  citation_year=2011;  citation_volume=186;  citation_pages=1286-90; " /><meta name="citation_reference" content="citation_title=Prostate cancer-specific mortality after radical prostatectomy for patients treated in the prostate-specific antigen era; J. Clin. Oncol.;  citation_year=2009;  citation_volume=27;  citation_pages=4300-5; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/5/395/1752535/dss021.pdf" /><meta name="description" content="Abstract.  High-throughput next-generation sequencing provides a revolutionary platform to unravel the precise DNA aberrations concealed within subgroups of tum" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/5/395/629019" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/5/395/629019" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/19/5/395/629019");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dss021';
    </script>
<title>
	Mate Pair Sequencing of Whole-Genome-Amplified DNA Following Laser Capture Microdissection of Prostate Cancer | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./629019" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BdkrPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNdXJwaHksIFN0ZXBoZW4gSi4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1vbGVjdWxhciBNZWRpY2luZSwgTWF5byBDbGluaWMsIE1lZGljYWwgU2NpZW5jZXMgQnVpbGRpbmcgMiwgMjAwIEZpcnN0IFN0LiwgU1csIFJvY2hlc3RlciwgTU4gNTU5MDUsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNoZXZpbGxlLCBKb2huIEMuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBMYWJvcmF0b3J5IE1lZGljaW5lIGFuZCBQYXRob2xvZ3ksIE1heW8gQ2xpbmljLCBSb2NoZXN0ZXIsIE1OIDU1OTA1LCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJaYXJlaSwgU2hhYm5hbSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTW9sZWN1bGFyIE1lZGljaW5lLCBNYXlvIENsaW5pYywgTWVkaWNhbCBTY2llbmNlcyBCdWlsZGluZyAyLCAyMDAgRmlyc3QgU3QuLCBTVywgUm9jaGVzdGVyLCBNTiA1NTkwNSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSm9obnNvbiwgU2FyYWggSC4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1vbGVjdWxhciBNZWRpY2luZSwgTWF5byBDbGluaWMsIE1lZGljYWwgU2NpZW5jZXMgQnVpbGRpbmcgMiwgMjAwIEZpcnN0IFN0LiwgU1csIFJvY2hlc3RlciwgTU4gNTU5MDUsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNpa2tpbmssIFJvYmVydCBBLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkFkdmFuY2VkIEdlbm9taWNzIFRlY2hub2xvZ3kgQ2VudGVyLCBNYXlvIENsaW5pYywgUm9jaGVzdGVyLCBNTiA1NTkwNSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS29zYXJpLCBGYXJoYWQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1vbGVjdWxhciBNZWRpY2luZSwgTWF5byBDbGluaWMsIE1lZGljYWwgU2NpZW5jZXMgQnVpbGRpbmcgMiwgMjAwIEZpcnN0IFN0LiwgU1csIFJvY2hlc3RlciwgTU4gNTU5MDUsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZlbGRtYW4sIEFuZHJldyBMLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTGFib3JhdG9yeSBNZWRpY2luZSBhbmQgUGF0aG9sb2d5LCBNYXlvIENsaW5pYywgUm9jaGVzdGVyLCBNTiA1NTkwNSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRWNrbG9mZiwgQnJ1Y2UgVy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJBZHZhbmNlZCBHZW5vbWljcyBUZWNobm9sb2d5IENlbnRlciwgTWF5byBDbGluaWMsIFJvY2hlc3RlciwgTU4gNTU5MDUsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikthcm5lcywgUi4gSmVmZnJleSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgVXJvbG9neSwgTWF5byBDbGluaWMsIFJvY2hlc3RlciwgTU4gNTU5MDUsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlZhc21hdHppcywgR2VvcmdlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNb2xlY3VsYXIgTWVkaWNpbmUsIE1heW8gQ2xpbmljLCBNZWRpY2FsIFNjaWVuY2VzIEJ1aWxkaW5nIDIsIDIwMCBGaXJzdCBTdC4sIFNXLCBSb2NoZXN0ZXIsIE1OIDU1OTA1LCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ik1hdGUgUGFpciBTZXF1ZW5jaW5nIG9mIFdob2xlLUdlbm9tZS1BbXBsaWZpZWQgRE5BIEZvbGxvd2luZyBMYXNlciBDYXB0dXJlIE1pY3JvZGlzc2VjdGlvbiBvZiBQcm9zdGF0ZSBDYW5jZXIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIzOTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjQwNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzczAyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEyLzEwLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tbW9uIGdlbmUgcmVhcnJhbmdlbWVudHMgaW4gcHJvc3RhdGUgY2FuY2VyOyBKLiBDbGluLiBPbmNvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz02MDgtMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsaW5pY2FsIGZlYXR1cmVzIGFuZCBwcm9nbm9zaXMgb2YgZGUgbm92byBkaWZmdXNlIGxhcmdlIEItY2VsbCBseW1waG9tYSB3aXRoIHQoMTQ7MTgpIGFuZCA4cTI0L2MtTVlDIHRyYW5zbG9jYXRpb25zOyBMZXVrZW1pYTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTc3Ny04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QkNMLUFCTDsgYSBtdWx0aWZhY2V0ZWQgcHJvbW90ZXIgb2YgRE5BIG11dGF0aW9uIGluIGNocm9uaWMgbXllbG9nZW5vdXMgbGV1a2VtaWE7IExldWtlbWlhOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MTEwNS0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNsb2NhdGlvbiAoMTg7MjIpIChxMjE7cTExKSBpbiBCLWNlbGwgbHltcGhvbWFzOiBhIHJlcG9ydCBvZiA0IGNhc2VzIGFuZCByZXZpZXcgb2YgdGhlIGxpdGVyYXR1cmU7IEh1bS4gUGF0aG9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPTE2NjQtNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IklsbHVtaW5hIE1hdGUgUGFpciBMaWJyYXJ5IHYyIFNhbXBsZSBQcmVwYXJhdGlvbiBHdWlkZSBmb3IgMuKAkzUga2IgTGlicmFyaWVzIDIwMDkgaHR0cDovL25leHRnZW4ubWdoLmhhcnZhcmQuZWR1L2F0dGFjaG1lbnRzL01hdGVQYWlyX3YyXzItNWtiX1NhbXBsZVByZXBfR3VpZGVfMTUwMDgxMzVfQS5wZGYgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzY292ZXJ5IG9mIHJlY3VycmVudCB0KDY7NykocDI1LjM7cTMyLjMpIHRyYW5zbG9jYXRpb25zIGluIEFMSy1uZWdhdGl2ZSBhbmFwbGFzdGljIGxhcmdlIGNlbGwgbHltcGhvbWFzIGJ5IG1hc3NpdmVseSBwYXJhbGxlbCBnZW5vbWljIHNlcXVlbmNpbmc7IEJsb29kOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTExNzsgIGNpdGF0aW9uX3BhZ2VzPTkxNS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyByZXZlYWxzIHJlY3VycmVudCBzdHJ1Y3R1cmFsIGFibm9ybWFsaXRpZXMgb2YgVFA2MyBhbmQgb3RoZXIgcDUzLXJlbGF0ZWQgZ2VuZXMgaW4gcGVyaXRoZXJhbCBULWNlbGwgbHltcGhvbWFzOyBjaXRhdGlvbl90ZXh0PTIwMTIgQXVnIDEuIFtFcHViIGFoZWFkIG9mIHByaW50XSBQTUlEOiAyMjg1NTU5ODsgQmxvb2Q7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIHVwZGF0ZSBvZiB0aGUgR2xlYXNvbiBncmFkaW5nIHN5c3RlbTsgSi4gVXJvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTgzOyAgY2l0YXRpb25fcGFnZXM9NDMzLTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HbGVhc29uIHNjb3JlIDcgcHJvc3RhdGUgY2FuY2VyIG9uIG5lZWRsZSBiaW9wc3k6IHJlbGF0aW9uIG9mIHByaW1hcnkgcGF0dGVybiAzIG9yIDQgdG8gcGF0aG9sb2dpY2FsIHN0YWdlIGFuZCBwcm9ncmVzc2lvbiBhZnRlciByYWRpY2FsIHByb3N0YXRlY3RvbXk7IEouIFVyb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE4NjsgIGNpdGF0aW9uX3BhZ2VzPTEyODYtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb3N0YXRlIGNhbmNlci1zcGVjaWZpYyBtb3J0YWxpdHkgYWZ0ZXIgcmFkaWNhbCBwcm9zdGF0ZWN0b215IGZvciBwYXRpZW50cyB0cmVhdGVkIGluIHRoZSBwcm9zdGF0ZS1zcGVjaWZpYyBhbnRpZ2VuIGVyYTsgSi4gQ2xpbi4gT25jb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NDMwMC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTkvNS8zOTUvMTc1MjUzNS9kc3MwMjEucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEhpZ2gtdGhyb3VnaHB1dCBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZyBwcm92aWRlcyBhIHJldm9sdXRpb25hcnkgcGxhdGZvcm0gdG8gdW5yYXZlbCB0aGUgcHJlY2lzZSBETkEgYWJlcnJhdGlvbnMgY29uY2VhbGVkIHdpdGhpbiBzdWJncm91cHMgb2YgdHVtIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj02MjkwMTkmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkNF6+MO1uMeHXzdgBTZbPtTHgp74=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f5%2f395%2f629019" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f5%2f395%2f629019" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=25687" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/5/1/m_cover.gif?Expires=2147483647&amp;Signature=oCF62cOF-RHSHHzNh6fqRIkUeFdGFZWfw3pKB6h-XrXTUmjPkfpfn8w~8bchsCbKMP2r~GVw22lZ1MNx0q-s71-AcQyMN0lvE0QjuCe~oMOmvkISrjvYiWPgB2qMv~4jOVZX-tDdrzrIeyUlp6-dE-d1BMbDpzvRnY6ImlMPnjamQ8tNFPiRg84G~Uknop7I8711Uikreov-Syq2nbs4ZI3p39Ios3nBrhdKb3cghqgrrGVSW3I0iXxFPXkjQY1cpzGh8rn-oZc~FVxVdjoBppF4WPcNdaRzbNDAFK0kliJ6Rs5rfCQy64JYiBPIa6tUr3wfBbp66X2fB6q0zAaV3g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/5">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="10636836">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10636836">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10636838">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10636838">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10636842">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10636842">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10636853">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10636853">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10636880">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10636880">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10636889">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10636889">Contributions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10636891">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10636891">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10636893">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10636893">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="10636895">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#10636895">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/19/5/383/628723">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/19/5/407/629039">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Mate Pair Sequencing of Whole-Genome-Amplified DNA Following Laser Capture Microdissection of Prostate Cancer
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Stephen J. Murphy</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Molecular Medicine</div>, <div class="insititution">Mayo Clinic</div>, <div class="addr-line">Medical Sciences Building 2, 200 First St., SW, Rochester, MN 55905</div>, <div class="country">USA</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel: +1 507-266-4617. Fax. +1 507-284-4521. E-mail: <a href="mailto:murphy.stephen@mayo.edu" target="_blank">murphy.stephen@mayo.edu</a>Â or <a href="mailto:vasmatzis.george@mayo.edu" target="_blank">vasmatzis.george@mayo.edu</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Stephen+J.+Murphy">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Murphy S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Murphy S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Stephen J. Murphy</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">John C. Cheville</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Laboratory Medicine and Pathology</div>, <div class="insititution">Mayo Clinic</div>, <div class="addr-line">Rochester, MN 55905</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=John+C.+Cheville">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cheville J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cheville J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">John C. Cheville</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shabnam Zarei</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Molecular Medicine</div>, <div class="insititution">Mayo Clinic</div>, <div class="addr-line">Medical Sciences Building 2, 200 First St., SW, Rochester, MN 55905</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shabnam+Zarei">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zarei S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zarei S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shabnam Zarei</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sarah H. Johnson</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Molecular Medicine</div>, <div class="insititution">Mayo Clinic</div>, <div class="addr-line">Medical Sciences Building 2, 200 First St., SW, Rochester, MN 55905</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sarah+H.+Johnson">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Johnson S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Johnson S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sarah H. Johnson</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Robert A. Sikkink</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Advanced Genomics Technology Center</div>, <div class="insititution">Mayo Clinic</div>, <div class="addr-line">Rochester, MN 55905</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Robert+A.+Sikkink">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sikkink R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sikkink R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Robert A. Sikkink</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Farhad Kosari</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Molecular Medicine</div>, <div class="insititution">Mayo Clinic</div>, <div class="addr-line">Medical Sciences Building 2, 200 First St., SW, Rochester, MN 55905</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Farhad+Kosari">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kosari F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kosari F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Farhad Kosari</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Andrew L. Feldman</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Laboratory Medicine and Pathology</div>, <div class="insititution">Mayo Clinic</div>, <div class="addr-line">Rochester, MN 55905</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Andrew+L.+Feldman">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Feldman A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Feldman A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Andrew L. Feldman</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Bruce W. Eckloff</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Advanced Genomics Technology Center</div>, <div class="insititution">Mayo Clinic</div>, <div class="addr-line">Rochester, MN 55905</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Bruce+W.+Eckloff">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Eckloff B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Eckloff B&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Bruce W. Eckloff</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">R. Jeffrey Karnes</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Urology</div>, <div class="insititution">Mayo Clinic</div>, <div class="addr-line">Rochester, MN 55905</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=R.+Jeffrey+Karnes">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Karnes R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Karnes R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">R. Jeffrey Karnes</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">George Vasmatzis</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Molecular Medicine</div>, <div class="insititution">Mayo Clinic</div>, <div class="addr-line">Medical Sciences Building 2, 200 First St., SW, Rochester, MN 55905</div>, <div class="country">USA</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel: +1 507-266-4617. Fax. +1 507-284-4521. E-mail: <a href="mailto:murphy.stephen@mayo.edu" target="_blank">murphy.stephen@mayo.edu</a>Â or <a href="mailto:vasmatzis.george@mayo.edu" target="_blank">vasmatzis.george@mayo.edu</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=George+Vasmatzis">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Vasmatzis G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Vasmatzis G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">George Vasmatzis</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 5, 1 October 2012, Pages 395â€“406, <a href='https://doi.org/10.1093/dnares/dss021'>https://doi.org/10.1093/dnares/dss021</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">18 September 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">23 April 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">06 August 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="629019" href="/dnaresearch/article-pdf/19/5/395/1752535/dss021.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Stephen J. Murphy, John C. Cheville, Shabnam Zarei, Sarah H. Johnson, Robert A. Sikkink, Farhad Kosari, Andrew L. Feldman, Bruce W. Eckloff, R. Jeffrey Karnes, George Vasmatzis;  Mate Pair Sequencing of Whole-Genome-Amplified DNA Following Laser Capture Microdissection of Prostate Cancer, <em>DNA Research</em>, Volume 19, Issue 5, 1 October 2012, Pages 395â€“406, <a href='https://doi.org/10.1093/dnares/dss021'>https://doi.org/10.1093/dnares/dss021</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/629019?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/629019?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/629019?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/629019?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/629019?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/629019?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/629019?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Mate%20Pair%20Sequencing%20of%20Whole-Genome-Amplified%20DNA%20Following%20Laser%20Capture%20Microdissection%20of%20Prostate%20Cancer&amp;publicationDate=2012-09-18&amp;volumeNum=19&amp;issueNum=5&amp;author=Murphy%2C%20Stephen%20J.%3B%20Cheville%2C%20John%20C.&amp;startPage=395&amp;endPage=406&amp;contentId=10.1093%2Fdnares%2Fdss021&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f5%2f395%2f629019" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=10636836 id="10636836" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>High-throughput next-generation sequencing provides a revolutionary platform to unravel the precise DNA aberrations concealed within subgroups of tumour cells. However, in many instances, the limited number of cells makes the application of this technology in tumour heterogeneity studies a challenge. In order to address these limitations, we present a novel methodology to partner laser capture microdissection (LCM) with sequencing platforms, through a whole-genome amplification (WGA) protocol performed <em>in situ</em> directly on LCM engrafted cells. We further adapted current Illumina mate pair (MP) sequencing protocols to the input of WGA DNA and used this technology to investigate large genomic rearrangements in adjacent Gleason Pattern 3 and 4 prostate tumours separately collected by LCM. Sequencing data predicted genome coverage and depths similar to unamplified genomic DNA, with limited repetition and bias predicted in WGA protocols. Mapping algorithms developed in our laboratory predicted high-confidence rearrangements and selected events each demonstrated the predicted fusion junctions upon validation. Rearrangements were additionally confirmed in unamplified tissue and evaluated in adjacent benign-appearing tissues. A detailed understanding of gene fusions that characterize cancer will be critical in the development of biomarkers to predict the clinical outcome. The described methodology provides a mechanism of efficiently defining these events in limited pure populations of tumour tissue, aiding in the derivation of genomic aberrations that initiate cancer and drive cancer progression.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;mate pair sequencing&quot;">mate pair sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;whole-genome amplified, laser capture microdissection&quot;">whole-genome amplified, laser capture microdissection</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;prostrate cancer&quot;">prostrate cancer</a></div>                            <h2 scrollto-destination=10636838 id="10636838" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>In order to better stratify cancer patients for the best treatment or no treatment (active surveillance), characterization of the tumour will be required that can define the molecular changes associated with indolent or aggressive behaviour. A large effort in cancer research is aimed at identifying these cancer biomarkers that will augment the prognostic value of contemporary clinical and pathological features in stratifying risk of progression. Single-nucleotide variations (SNVs) affecting the gene expression of key regulatory genes provide both mechanism and markers for the progression of cancer. However, the vast numbers of genes affected by SNVs and the unknown impact of specific mutations on protein expression and function make the derivation of driver and passenger mutations difficult. Larger genomic rearrangements, such as amplifications, deletions and translocations, predict more distinct impacts on gene expression. The recent evaluation of the prevalence and reoccurrence of large genomic rearrangements and translocations in solid tumours has highlighted their value in defining cancer progression and prognosis.<sup><a reveal-id="DSS021C1" data-open="DSS021C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Significantly, clinical tests aimed at recurrent translocations/gene fusions involving <em>Bcl-Abl, TMPRSS2-ERG, c-myc and bcl2</em> already impact clinical decisions.<sup><a reveal-id="DSS021C1  DSS021C4" data-open="DSS021C1  DSS021C4" class="link link-ref link-reveal xref-bibr">1â€“4</a></sup></p><p>Massively parallel, next-generation DNA sequencing technologies represent a quantum advance in the ability to understand cancer at the genetic level. However, in order to accurately characterize genomic aberrations that define a tumour population, it is critical to obtain pure genetic material from the tumour with no or minimal contamination of surrounding cells. Tumours exist as outgrowths or repopulations of normal tissues with genetically modified cells, and as a result, malignant cells are often embedded and intermixed with normal cells making the sampling of pure tumour cells difficult. Specifically, the bulk sampling of a tumour mass is inevitably contaminated with varying degrees of non-neoplastic stromal, inflammatory, and epithelial cells, which complicate the genetic interrogation of the tumour cells. Laser capture microdissection (LCM) is a powerful tool to efficiently and accurately extract pure populations of tumour cells from clinical specimens, yet provides a limited number of cells and genetic material for analysis. The mate pair (MP) next-generation sequencing (NGS) protocol developed for the evaluation is capable of evaluating large genomic rearrangements using only a single lane of the Illumina GAIIx sequencer.<sup><a reveal-id="DSS021C5" data-open="DSS021C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSS021C6" data-open="DSS021C6" class="link link-ref link-reveal xref-bibr">6</a></sup> However, this protocol currently requires DNA quantities far greater than those that can be yielded from LCM extracted tissues (5â€“10 Î¼g), making the application of LCM to this NGS platform challenging.<sup><a reveal-id="DSS021C5" data-open="DSS021C5" class="link link-ref link-reveal xref-bibr">5</a></sup></p><p>In this study, we report a novel methodology that enables the efficient and reproducible whole-genome amplification (WGA) of DNA from small LCM collected clinical specimens for application in the MP NGS protocol. We demonstrate the use of this technique in evaluating two pathologically distinct adjacent tumour grades (Gleason patterns) in a patient with prostate cancer.</p>                            <h2 scrollto-destination=10636842 id="10636842" class="section-title" data-legacy-id=s2>2. Materials and Methods</h2>
                            <h3 scrollto-destination=10636843 id="10636843" class="section-title" data-legacy-id=s2a>2.1. Tissue selection and LCM</h3>
<p>H&amp;E stained sections of prostate cancer tissues were reviewed by a urologic pathologist (J.C.C.) and graded for tumour content. Frozen prostate tissue sections were cut into 10 Î¼m size and stained with Cresyl Violet (LCM Staining Kit, Ambion; AM1935). GP3 and GP4 tumour regions were separately excised by LCM using the Arcturus PixCell II microscope and CapSure Macro LCM caps (Arcturus; LCM 0211). LCM of adjacent histologically normal (aN) peripheral zone prostatic epithelial cells (in the same tissue section as GP3 and GP4) and distal normal (dN) peripheral zone prostatic epithelial cells (separate benign tissue block from the opposite side of the prostate gland from cancer) were used as controls.</p>                            <h3 scrollto-destination=10636845 id="10636845" class="section-title" data-legacy-id=s2b>2.2. Direct <em>in situ</em> DNA extraction and amplification</h3>
<p>LCM cap polymer regions containing engrafted cells were carefully excised and applied directly to WGA using a modified Repli-g protocol as follows (Qiagen, CA, USA). A modified Repli-g D2 buffer was first prepared (2.5 Î¼l phosphate buffered saline, 3 Î¼l Repli-g buffer (DLB) and 1 Î¼l ditiothreitol per sample) and 6.5 Î¼l was added to the LCM engrafted cells in 0.2 Î¼l capped tubes ensuring complete immersion of the polymer fragments. Samples were incubated for 10 min on ice. Reactions were subsequently neutralized upon addition of 3.5 Î¼l of Repli-g stop solution, prior to adding 40 Î¼l of the standard Repli-g mini kit master mix and incubating for 16 h at 30Â°C. Final heating to 65Â°C for 3 min inactivated the DNA polymerase. WGA was performed on at least four independent reactions from parallel frozen sections and equal volumes pooled to minimize the amplification bias. For smaller lesions excised LCM regions from multiple sections can be applied to a single WGA reaction. WGA DNA (1 Î¼l) was visualized on 1% agarose gels and quantitated using the Quant-iT PicoGreen dsDNA reagent (Invitrogen, P7581). Qualitative multiplex polymerase chain reaction (PCR) was performed according to the Sigma-Aldrich protocol and PCR products resolved on 4% agarose gels (Sigma-Aldrich; P0982).</p>                            <h3 scrollto-destination=10636847 id="10636847" class="section-title" data-legacy-id=s2c>2.3. Illumina MP Library Production</h3>
<p>A modified Illumina MP library protocol was used. Briefly 10 Î¼g of WGA DNA was fragmented to 3â€“5 kb using the Covaris E210 (Duty Cycle 20%, Intensity 1, 1000 cycles burst for 600 s.). End repair was extended to 30 min prior to the biotin end-labelling step and size selection on a 1% agarose gel. The biotin-labelled blunt-end fragments were then circularized and endonuclease treated, before the remaining DNA circles were fragmented again to 350â€“650 bp on the Covaris E210. The biotinylated terminal fragments were then immobilized on M-280 streptavidin beads (Dynal) and assembled into conventional Illumina adapter flanked paired-end (PE) libraries. Libraries were amplified (18 cycles) using conventional Illumina PCR adapter primer pairs (PE 1.0 and PE 2.0), purified and analyzed on an Agilent Bioanalyzer DNA 1000 chip. The library was loaded onto one lane of an Illumina flow cell at a concentration of 9 pM generating an average of 215 000 clusters/tile using the Illumina cluster station and PE cluster kit v4. The flow cell was sequenced as a 76 Ã— 2 PE read on an Illumina GAIIx using SBS sequencing kit v4 and SCS v2.5. Base calling was performed using Illumina Pipeline v1.5.</p>                            <h3 scrollto-destination=10636849 id="10636849" class="section-title" data-legacy-id=s2d>2.4. Bioinformatics protocols</h3>
<p>Bioinformatics protocols to rapidly and efficiently process NGS MP data using a 32-bit binary indexing of the Hg19 reference genome, to which consecutive 32-bit binary sequences from associated MP reads are aligned, have been previously published from our laboratory.<sup><a reveal-id="DSS021C6" data-open="DSS021C6" class="link link-ref link-reveal xref-bibr">6</a></sup> This algorithm has subsequently been further optimized and is presented in depth in a separate manuscript.<sup><a reveal-id="DSS021C7" data-open="DSS021C7" class="link link-ref link-reveal xref-bibr">7</a></sup> The algorithm maps both MP reads successively in the whole genome, thus minimizes miss mapping and reduces false positives<em>.</em> The algorithm aggressively tries to find mapping possibilities where the two reads map &lt;15 kb apart allowing up to 10 mismatches. If mapping gave multiple good possibilities then the one with the lowest cumulative mismatch count was sent to the output. MPs mapping &gt;15 kb apart or in different chromosomes were defined as large genomic rearrangements and selected for further analysis. When two reads map to the same chromosome more than 20 kb apart, they are defined as intra-chromosomal rearrangement (<em>r</em>) events. When they map to different chromosomes, they are defined as inter-chromosomal translocation (<em>t</em>) events. Filters, based on homology scores calculated during mapping, are applied to eliminate false positives. Fragments that represent the same event are identified as associate fragments and are clustered together. A mask table was created to further eliminate false-positive events that pass these filters defined from recurrent events in normal tissues or through BLAST-like alignment tool mapping of multiple aligning regions. After the removal of replicate read pairs, coverage calculations indicated the expected number of MPs covering a breakpoint. Replication was calculated for each chromosome and the replicate read pairs were removed. Bridged coverage was calculated as the sum of the fragment lengths (distance between read 1 and read 2) of correctly mapping MP and PE reads, divided by the mappable chromosome size. Base coverage was calculated from the total number of mappable reads, multiplied by the read length and divided by the mappable chromosome size. The coverage was calculated per chromosome and the average was found using chromosomes 1â€“22. For allelic coverage, the values must be divided by 2. A procedure generated count plots of the number of mapped read pairs in non-overlapping equal-length windows. The algorithm normalized each sample by calculating the window size such that windows in parts of the genome without deletions or amplifications contained an average of 300 mapped read pairs.</p>                            <h3 scrollto-destination=10636851 id="10636851" class="section-title" data-legacy-id=s2e>2.5. Validation of genomic rearrangements</h3>
<p>MP sequence reads were mapped to the human genome, and primers spanning the fusion junctions were used in validation PCRs (50 Î¼l, 35 cycles), on WGA tumour or normal DNA with human genomic DNA (gDNA) used as a control (G304A; Promega, WI, USA). Unique PCR products yielded from the WGA DNA, absent from control, were isolated by gel extraction (Minelute; Qiagen 28004) and Sanger sequenced. For secondary validation of genomic rearrangements on unamplified GP3 and GP4 DNA extracts, LCM was performed on fresh tissue sections and extracted by Arcturus Pico Pure DNA extraction reagents (20 Î¼l/lesion) after incubation at 65Â°C for 4 h. PCRs (25 Î¼l) were run in parallel to WGA G3/G4 DNA (100 ng, 35 cycles) but using 5 Î¼l of unamplified material (40 cycles).</p>                            <h2 scrollto-destination=10636853 id="10636853" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=10636854 id="10636854" class="section-title" data-legacy-id=s3a>3.1. Isolation of prostate GP3 and GP4 DNA</h3>
<p>A prostate cancer case containing two regions of adjacent but distinct GP3 and GP4 tumours (Fig.Â <a reveal-id="DSS021F1" data-open="DSS021F1" class="link link-reveal link-table xref-fig">1</a>a, central panel) was selected for the study. For the GP3 tumour (Fig.Â <a reveal-id="DSS021F1" data-open="DSS021F1" class="link link-reveal link-table xref-fig">1</a>a, left panel), the infiltrative glands are well formed, each with a discernible round to oval shape. The GP4 tumour (Fig.Â <a reveal-id="DSS021F1" data-open="DSS021F1" class="link link-reveal link-table xref-fig">1</a>a, right panel) shows a loss of distinct glandular differentiation in comparison with GP3, and cells are arranged in cribriform structures as well as sheets of tumour cells. LCM was used to isolate cells specific to the GP3 and GP4 populations minimizing the contamination of adjacent cells. FigureÂ <a reveal-id="DSS021F1" data-open="DSS021F1" class="link link-reveal link-table xref-fig">1</a>b demonstrates the power of the LCM technique, showing the section images before and after the laser-mediated extraction, as well as the cellular population of the GP3 and GP4 isolated on the LCM caps. </p>                        <a id="10636856" scrollto-destination="10636856"></a>
<div id="DSS021F1" data-id="DSS021F1" class="fig fig-section"><div class="label fig-label">FigureÂ 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss021/2/m_dss02101.jpeg?Expires=2147483647&Signature=o2qeKNl01oriFsu3YbUr90tN7EaXAL2z4-A8JuKDFzcMLAHeG7Fp9qXXHCj7SCVC09Gmb3ugC9JN~6aQhe-BvsvuOlZCp6L4NqokHz-l7Ysq1RKn96hNrMucfCQaQdEwSykQFBCNYLyVKv-cTZdNKAbiLeN5aWKeI3h-Qmzzbeb42kYuAsHwb7I8CQgxBFxbZ-kpxFRm8VdRYWA4tbzovKRrr2PGmAoS604aXOwzp65U684Mr5P65s2a~N5o1KhiVa3tkwg76VFYozYQvmfvkSzPul2tMR7eFF2MSBa3HMATPeo~y51u8u9hR7rAmkrA8AqtEo~39~fIDhecg67Miw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="LCM and WGA of prostate cancers. (a) H&amp;E stained frozen section of prostate tissues. The central panel highlights the adjacent G3 and G4 cancers, with the individual G3 and G4 regions magnified further in the left and right panels, respectively. (b) LCM images before (PreLCM) and after (PostLCM) capture and the G3 and G4 cancer cells engrafted on the LCM caps. Products of direct in situ WGA of LCM-isolated cells (c) from two adjacent prostate lesions from six parallel tissue sections (1â€“6) and (d) from LCM of additional prostate (12 cases) and lung (4 cases) tissues [upper panels, 1% agarose gels, 1 kb ladders (L)]. Multiplex PCR from the corresponding tissues (lower panels, 4% agarose gels). The corresponding multiplex PCR products are shown in the lower panels (4% agarose gel with 100 bp ladders). Controls utilized gDNA (10 ng) in the WGA reaction or unamplified gDNA (unA) in the multiplex PCR." path-from-xml="dss02101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10636856/dss02101.jpeg" path-from-xml="dss02101.jpeg" target="_blank">View large</a><a section="10636856" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss021/2/dss02101.jpeg?Expires=2147483647&Signature=OCSFlU5SF-MFImC7eTnnzkbeRsDfjXkg~GO1~TYtRP6pXiSnGQG8JSAMZL05wQaiiMC7qtmWjjdexp31fSnhobvQYncp5BEftSM~67-FjVRJ9GnYiKW~NaHCQHx8jCUgTbUh-bOc68sHEO6YuRnpAmDtxi6c8SLlVNHymMom-raosp~7RrXG6BDUS97HlSv3HEDYqLhzVgGVW9E4-eGUNAsCXhnQGyZc1vuKXJ07KShLnQX-FCKCBSY6ehlWpzQUKNW4uv-JtsI3g-WFaq3qzTScniSZRuMMbsdXXY~vqWKmNQ6GoLd7ju3OJdrmcjIpyLpqN71yI49a29i1789hOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10636856&ar=629019&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>LCM and WGA of prostate cancers. (a) H&amp;E stained frozen section of prostate tissues. The central panel highlights the adjacent G3 and G4 cancers, with the individual G3 and G4 regions magnified further in the left and right panels, respectively. (b) LCM images before (PreLCM) and after (PostLCM) capture and the G3 and G4 cancer cells engrafted on the LCM caps. Products of direct <em>in situ</em> WGA of LCM-isolated cells (c) from two adjacent prostate lesions from six parallel tissue sections (1â€“6) and (d) from LCM of additional prostate (12 cases) and lung (4 cases) tissues [upper panels, 1% agarose gels, 1 kb ladders (L)]. Multiplex PCR from the corresponding tissues (lower panels, 4% agarose gels). The corresponding multiplex PCR products are shown in the lower panels (4% agarose gel with 100 bp ladders). Controls utilized gDNA (10 ng) in the WGA reaction or unamplified gDNA (unA) in the multiplex PCR.</p></div></div><div content-id="DSS021F1" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss021/2/m_dss02101.jpeg?Expires=2147483647&Signature=o2qeKNl01oriFsu3YbUr90tN7EaXAL2z4-A8JuKDFzcMLAHeG7Fp9qXXHCj7SCVC09Gmb3ugC9JN~6aQhe-BvsvuOlZCp6L4NqokHz-l7Ysq1RKn96hNrMucfCQaQdEwSykQFBCNYLyVKv-cTZdNKAbiLeN5aWKeI3h-Qmzzbeb42kYuAsHwb7I8CQgxBFxbZ-kpxFRm8VdRYWA4tbzovKRrr2PGmAoS604aXOwzp65U684Mr5P65s2a~N5o1KhiVa3tkwg76VFYozYQvmfvkSzPul2tMR7eFF2MSBa3HMATPeo~y51u8u9hR7rAmkrA8AqtEo~39~fIDhecg67Miw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="LCM and WGA of prostate cancers. (a) H&amp;E stained frozen section of prostate tissues. The central panel highlights the adjacent G3 and G4 cancers, with the individual G3 and G4 regions magnified further in the left and right panels, respectively. (b) LCM images before (PreLCM) and after (PostLCM) capture and the G3 and G4 cancer cells engrafted on the LCM caps. Products of direct in situ WGA of LCM-isolated cells (c) from two adjacent prostate lesions from six parallel tissue sections (1â€“6) and (d) from LCM of additional prostate (12 cases) and lung (4 cases) tissues [upper panels, 1% agarose gels, 1 kb ladders (L)]. Multiplex PCR from the corresponding tissues (lower panels, 4% agarose gels). The corresponding multiplex PCR products are shown in the lower panels (4% agarose gel with 100 bp ladders). Controls utilized gDNA (10 ng) in the WGA reaction or unamplified gDNA (unA) in the multiplex PCR." path-from-xml="dss02101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10636856/dss02101.jpeg" path-from-xml="dss02101.jpeg" target="_blank">View large</a><a section="10636856" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss021/2/dss02101.jpeg?Expires=2147483647&Signature=OCSFlU5SF-MFImC7eTnnzkbeRsDfjXkg~GO1~TYtRP6pXiSnGQG8JSAMZL05wQaiiMC7qtmWjjdexp31fSnhobvQYncp5BEftSM~67-FjVRJ9GnYiKW~NaHCQHx8jCUgTbUh-bOc68sHEO6YuRnpAmDtxi6c8SLlVNHymMom-raosp~7RrXG6BDUS97HlSv3HEDYqLhzVgGVW9E4-eGUNAsCXhnQGyZc1vuKXJ07KShLnQX-FCKCBSY6ehlWpzQUKNW4uv-JtsI3g-WFaq3qzTScniSZRuMMbsdXXY~vqWKmNQ6GoLd7ju3OJdrmcjIpyLpqN71yI49a29i1789hOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10636856&ar=629019&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>LCM and WGA of prostate cancers. (a) H&amp;E stained frozen section of prostate tissues. The central panel highlights the adjacent G3 and G4 cancers, with the individual G3 and G4 regions magnified further in the left and right panels, respectively. (b) LCM images before (PreLCM) and after (PostLCM) capture and the G3 and G4 cancer cells engrafted on the LCM caps. Products of direct <em>in situ</em> WGA of LCM-isolated cells (c) from two adjacent prostate lesions from six parallel tissue sections (1â€“6) and (d) from LCM of additional prostate (12 cases) and lung (4 cases) tissues [upper panels, 1% agarose gels, 1 kb ladders (L)]. Multiplex PCR from the corresponding tissues (lower panels, 4% agarose gels). The corresponding multiplex PCR products are shown in the lower panels (4% agarose gel with 100 bp ladders). Controls utilized gDNA (10 ng) in the WGA reaction or unamplified gDNA (unA) in the multiplex PCR.</p></div></div><p>To obtain enough gDNA to enable high-throughput MP sequencing of the specific GP of prostate cancer, WGA of the isolated DNA was required. Conventional techniques of DNA extraction and purification prior to WGA resulted in considerable variation and bias in the output quality of the DNA in pilot studies (data not shown). In order to improve on the quality of the sequencing data, a modified direct <em>in situ</em> WGA protocol was developed. This novel technique involved the application of engrafted LCM cellular material directly into WGA reactions. The chemical conditions of an initial DNA denaturing step facilitated the lysis of the engrafted LCM cells and the release of the nuclear reservoir of gDNA to the subsequent WGA reaction. Significantly, the crude cellular lysate did not result in observable inhibition of the enzymatic amplification reaction, generating DNA yields equivalent to control-purified gDNA (Fig.Â <a reveal-id="DSS021F1" data-open="DSS021F1" class="link link-reveal link-table xref-fig">1</a>c). The reproducibility of this direct <em>in situ</em> WGA method is exemplified in parallel reactions on two sets of six consecutive LCM isolations of prostate tissue frozen sections (Fig.Â <a reveal-id="DSS021F1" data-open="DSS021F1" class="link link-reveal link-table xref-fig">1</a>c, upper panel) yielding WGA DNA products of similar quantity and size on an agarose gel. Yields of amplified gDNA were typically in the range of 8â€“10 Î¼g/50 Î¼l WGA reactions. A multiplex PCR was used to assess amplification bias and quality of the WGA DNA. The banding patterns of five amplicons (132â€“295 bp) for the WGA DNA samples were equivalent to control WGA and unamplified DNA (Fig.Â <a reveal-id="DSS021F1" data-open="DSS021F1" class="link link-reveal link-table xref-fig">1</a>c, lower panel). This result gave an initial positive indication of the representative genomic amplification in these WGA reactions. The reproducible amplification quality and quantity of WGA DNA yielded directly from LCM of 12 different prostate cases and 4 lung tissues are also presented in Fig.Â <a reveal-id="DSS021F1" data-open="DSS021F1" class="link link-reveal link-table xref-fig">1</a>d.</p>                            <h3 scrollto-destination=10636858 id="10636858" class="section-title" data-legacy-id=s3b>3.2. MP sequencing and bioinformatics mapping algorithm</h3>
<p>MP libraries were assembled for the prostate GP3 and GP4 WGA DNA samples using the modified MP library assembly protocol described. The conventional Illumina protocols were adapted to account for the different physical properties of the WGA DNA. The MP libraries were sequenced on single lanes of the Illumina GAIIx platform. MP sequencing of the GP3 and GP4 cancers generated 33.0 and 29.4 million mappable read pairs, respectively, consistent with results from our MP sequencing of conventional unamplified gDNA on the GAIIx sequencer, which yielded averages of 27.1 million reads. A set of algorithms was developed to detect large chromosomal aberrations with low false-positive rates. The algorithms were specifically designed to handle NGS from mate-pair protocols.<sup><a reveal-id="DSS021C7" data-open="DSS021C7" class="link link-ref link-reveal xref-bibr">7</a></sup> The algorithms describe coverage in two ways: conventional base-pair coverage from the 75â€“100-bp reads and the theoretical bridged-coverage considering the original 3â€“5-kb span separating two MP reads (Fig.Â <a reveal-id="DSS021F2" data-open="DSS021F2" class="link link-reveal link-table xref-fig">2</a>a). MP reads mapping with the expected region span allows us to infer that the correct sequence span lies between the reads. Fragment mapping profiles are presented in Fig.Â <a reveal-id="DSS021F2" data-open="DSS021F2" class="link link-reveal link-table xref-fig">2</a>b. The predominant fragment population reflects the MP reads with 3â€“5 kb bridged spans. The second narrower peaks at around âˆ’300 bp consist of co-purifying PE library fragments derived from internal fragments of the circles not spanning the ligation junctions (Fig.Â <a reveal-id="DSS021F2" data-open="DSS021F2" class="link link-reveal link-table xref-fig">2</a>a). The PE peaks map algorithmically with negative values due to the different polarity of the read pairs stemming from the circularization in the MP methodology.<sup><a reveal-id="DSS021C5" data-open="DSS021C5" class="link link-ref link-reveal xref-bibr">5</a></sup> The fragment mapping profile for unamplified (Fig.Â <a reveal-id="DSS021F2" data-open="DSS021F2" class="link link-reveal link-table xref-fig">2</a>b, i) and WGA (Fig.Â <a reveal-id="DSS021F2" data-open="DSS021F2" class="link link-reveal link-table xref-fig">2</a>b, ii) DNA are effectively equivalent, with the WGA DNA mapping with just slightly reduced span of 2.5â€“5 kb indicative of the starting input of the smaller DNA fragments (Fig.Â <a reveal-id="DSS021F1" data-open="DSS021F1" class="link link-reveal link-table xref-fig">1</a>c). Although the proportion of PE mapping fragments is still higher with the WGA DNA, additional refinements of the MP protocol could reduce this further. </p>                        <a id="10636860" scrollto-destination="10636860"></a>
<div id="DSS021F2" data-id="DSS021F2" class="fig fig-section"><div class="label fig-label">FigureÂ 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss021/2/m_dss02102.jpeg?Expires=2147483647&Signature=QsOj99he0gW088eVx1afUUMlEOv4d~Pjs0K~RJYSG~Qi9zB~dTaagO~ytCukA2UfpBPfpSzFS9c0S5PrTQsjhwKVmsJ20XqMAjRJlGUko~7e7JN1Hb9jkfflYrf3GrZOZ~PmAaTVX6Zmp6diNGUtW7Rzvz0aaO3ZoJ3oRE~22yFCcbbNU0EAz5Sm7lWMKg4BhUfpF5qNxCewqkcqPH-tU7OX93o-7DA2QOjdmjB5zU4TohLhB2w1AmZUkUYttj3xg64fmNYd9w0zEujVJPlvLyc2uFlCmaOpzdahJfk0xoaNVvQBVql7FudNLGMmwo3XvpRW8gBfRHUAwvSwFs-RQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="MP NGS of WGA DNA products. (a) A schematic of the MP library assembly, detailing the initial fragmentation of the gDNA to 3â€“5 kb and biotinylation of the termini of these fragments to enable isolation and sequencing of the ligated terminal regions after circularization and a second fragmentation to âˆ¼500 bp. The bridged coverage is demonstrated as the span of the initial 3â€“5 kb fragments and the resulting terminal MP fusion products and conventional PE fragments depicted. (b) The bridged length of paired reads mapping to the reference genome from (i) an unamplified gDNA sample of T-cell origin and (ii) the WGA GP4 sample. (c) Bridged (open squares) and base pair coverage (open triangles) for individual chromosomes from the WGA GP3 (green) and GP4 (blue) samples and the unamplified T-cell control (red). (d) Percentage replication in the GP3 (green square), GP4 (blue triangle) and unamplified (red star) samples MP data. (e) Percentage replication (blue), together with bridged (red) and sequence (green) fold coverage for 22 additional WGA prostate cancer DNA samples following LCM isolation on the HiSeq platform. (f) Frequency coverage plots for chromosomes 4 (i and iii) and 10 (ii and iv) of unamplified DNA (i and ii) and WGA G4 DNA (iii and iv) sequencing data." path-from-xml="dss02102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10636860/dss02102.jpeg" path-from-xml="dss02102.jpeg" target="_blank">View large</a><a section="10636860" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss021/2/dss02102.jpeg?Expires=2147483647&Signature=iuRZvjvqrcXqnUiphNYvPyhuHnAwG0w2ZCyxADEA2Z8hAN7b99XveBv4XG~ouR-Woamud74gvJz7wyPeN8V~gprcIVjNd6ozNtLdtK4CIePCN~wmRMqPojX1sTBoy4vsQjd94vkbf6EpunfvRlgTgVVl5Vcip4J1O3JdepZeEUuCgZ674jGnsHoerTYN3cyTNljQsnb-TP2U8FsR3GPjrBzthEwnKQJ0wKluAyjZ0G9PPB01K7vGdGkqBkMW7ktEG7y~sY5oWr-iGWmSzzrQH93~QdWmu0~VF9YxR1qoD0s4BmxkLP8A4LTDKPRtlZ985DkVcdp9hhb~mA1b6T4VIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10636860&ar=629019&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>MP NGS of WGA DNA products. (a) A schematic of the MP library assembly, detailing the initial fragmentation of the gDNA to 3â€“5 kb and biotinylation of the termini of these fragments to enable isolation and sequencing of the ligated terminal regions after circularization and a second fragmentation to âˆ¼500 bp. The bridged coverage is demonstrated as the span of the initial 3â€“5 kb fragments and the resulting terminal MP fusion products and conventional PE fragments depicted. (b) The bridged length of paired reads mapping to the reference genome from (i) an unamplified gDNA sample of T-cell origin and (ii) the WGA GP4 sample. (c) Bridged (open squares) and base pair coverage (open triangles) for individual chromosomes from the WGA GP3 (green) and GP4 (blue) samples and the unamplified T-cell control (red). (d) Percentage replication in the GP3 (green square), GP4 (blue triangle) and unamplified (red star) samples MP data. (e) Percentage replication (blue), together with bridged (red) and sequence (green) fold coverage for 22 additional WGA prostate cancer DNA samples following LCM isolation on the HiSeq platform. (f) Frequency coverage plots for chromosomes 4 (i and iii) and 10 (ii and iv) of unamplified DNA (i and ii) and WGA G4 DNA (iii and iv) sequencing data.</p></div></div><div content-id="DSS021F2" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss021/2/m_dss02102.jpeg?Expires=2147483647&Signature=QsOj99he0gW088eVx1afUUMlEOv4d~Pjs0K~RJYSG~Qi9zB~dTaagO~ytCukA2UfpBPfpSzFS9c0S5PrTQsjhwKVmsJ20XqMAjRJlGUko~7e7JN1Hb9jkfflYrf3GrZOZ~PmAaTVX6Zmp6diNGUtW7Rzvz0aaO3ZoJ3oRE~22yFCcbbNU0EAz5Sm7lWMKg4BhUfpF5qNxCewqkcqPH-tU7OX93o-7DA2QOjdmjB5zU4TohLhB2w1AmZUkUYttj3xg64fmNYd9w0zEujVJPlvLyc2uFlCmaOpzdahJfk0xoaNVvQBVql7FudNLGMmwo3XvpRW8gBfRHUAwvSwFs-RQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="MP NGS of WGA DNA products. (a) A schematic of the MP library assembly, detailing the initial fragmentation of the gDNA to 3â€“5 kb and biotinylation of the termini of these fragments to enable isolation and sequencing of the ligated terminal regions after circularization and a second fragmentation to âˆ¼500 bp. The bridged coverage is demonstrated as the span of the initial 3â€“5 kb fragments and the resulting terminal MP fusion products and conventional PE fragments depicted. (b) The bridged length of paired reads mapping to the reference genome from (i) an unamplified gDNA sample of T-cell origin and (ii) the WGA GP4 sample. (c) Bridged (open squares) and base pair coverage (open triangles) for individual chromosomes from the WGA GP3 (green) and GP4 (blue) samples and the unamplified T-cell control (red). (d) Percentage replication in the GP3 (green square), GP4 (blue triangle) and unamplified (red star) samples MP data. (e) Percentage replication (blue), together with bridged (red) and sequence (green) fold coverage for 22 additional WGA prostate cancer DNA samples following LCM isolation on the HiSeq platform. (f) Frequency coverage plots for chromosomes 4 (i and iii) and 10 (ii and iv) of unamplified DNA (i and ii) and WGA G4 DNA (iii and iv) sequencing data." path-from-xml="dss02102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10636860/dss02102.jpeg" path-from-xml="dss02102.jpeg" target="_blank">View large</a><a section="10636860" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss021/2/dss02102.jpeg?Expires=2147483647&Signature=iuRZvjvqrcXqnUiphNYvPyhuHnAwG0w2ZCyxADEA2Z8hAN7b99XveBv4XG~ouR-Woamud74gvJz7wyPeN8V~gprcIVjNd6ozNtLdtK4CIePCN~wmRMqPojX1sTBoy4vsQjd94vkbf6EpunfvRlgTgVVl5Vcip4J1O3JdepZeEUuCgZ674jGnsHoerTYN3cyTNljQsnb-TP2U8FsR3GPjrBzthEwnKQJ0wKluAyjZ0G9PPB01K7vGdGkqBkMW7ktEG7y~sY5oWr-iGWmSzzrQH93~QdWmu0~VF9YxR1qoD0s4BmxkLP8A4LTDKPRtlZ985DkVcdp9hhb~mA1b6T4VIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10636860&ar=629019&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>MP NGS of WGA DNA products. (a) A schematic of the MP library assembly, detailing the initial fragmentation of the gDNA to 3â€“5 kb and biotinylation of the termini of these fragments to enable isolation and sequencing of the ligated terminal regions after circularization and a second fragmentation to âˆ¼500 bp. The bridged coverage is demonstrated as the span of the initial 3â€“5 kb fragments and the resulting terminal MP fusion products and conventional PE fragments depicted. (b) The bridged length of paired reads mapping to the reference genome from (i) an unamplified gDNA sample of T-cell origin and (ii) the WGA GP4 sample. (c) Bridged (open squares) and base pair coverage (open triangles) for individual chromosomes from the WGA GP3 (green) and GP4 (blue) samples and the unamplified T-cell control (red). (d) Percentage replication in the GP3 (green square), GP4 (blue triangle) and unamplified (red star) samples MP data. (e) Percentage replication (blue), together with bridged (red) and sequence (green) fold coverage for 22 additional WGA prostate cancer DNA samples following LCM isolation on the HiSeq platform. (f) Frequency coverage plots for chromosomes 4 (i and iii) and 10 (ii and iv) of unamplified DNA (i and ii) and WGA G4 DNA (iii and iv) sequencing data.</p></div></div><p>The MP bridged and base-pair coverages for the WGA DNA were also equivalent to unamplified gDNA (Fig.Â <a reveal-id="DSS021F2" data-open="DSS021F2" class="link link-reveal link-table xref-fig">2</a>c). The average bridged coverage's of 18 and 21X were obtained for the GP3 and GP4 tumours, respectively, compared with an average of 17.5X for our unamplified DNA. The average sequence base coverages of 1â€“2X were observed for both DNA inputs. Base-pair and bridged coverages are presented for each chromosome in Fig.Â <a reveal-id="DSS021F2" data-open="DSS021F2" class="link link-reveal link-table xref-fig">2</a>c. Both measures of coverage are observed to be relatively consistent across the majority of chromosomes for both the unamplified and WGA samples. The level of replication indicates how many identical MP sequences are derived from sequencing clusters. This replication of MP sequences is expected due to the PCR amplification processes involved in library assembly and is bioinformatically removed to prevent bias in library mapping. The GP3 and GP4 samples were associated with averages of 6 and 10% replication, respectively, in a range identical to an unamplified DNA sample (Fig.Â <a reveal-id="DSS021F2" data-open="DSS021F2" class="link link-reveal link-table xref-fig">2</a>d).</p><p>While this report exemplifies a single prostate cancer case, multiple additional WGA samples have been processed using this methodology on the enhanced Illumina HiSeq platform. FigureÂ <a reveal-id="DSS021F2" data-open="DSS021F2" class="link link-reveal link-table xref-fig">2</a>e illustrates the coverage and replication from the MP sequencing of WGA DNA samples yielded from 22 additional prostate tissues. The average total number of reads increased to 76.7 million on the newer HiSeq platform, with the resultant base and bridged coverage levels averaging 3 and 39X, respectively. As expected, the mean levels of replication also increased to 37X, potentially due to sequence saturation on the flow cell. Together, these data provide us with further confidence in the reproducibility of the amplification methodology and the mapping algorithm described in this study.</p><p>FigureÂ <a reveal-id="DSS021F2" data-open="DSS021F2" class="link link-reveal link-table xref-fig">2</a>f (i and ii, respectively) shows MP count plots associated with an unamplified DNA sample on chromosomes 4 and 10. Although both chromosomes show very flat even profiles (Fig.Â <a reveal-id="DSS021F2" data-open="DSS021F2" class="link link-reveal link-table xref-fig">2</a>f, i), chromosome 10 shows a clear drop in frequency between 98 and 112 Mb (Fig.Â <a reveal-id="DSS021F2" data-open="DSS021F2" class="link link-reveal link-table xref-fig">2</a>f, ii), indicative of a deletion in this region. The centromeres are observed as sequence gaps due to the current lack of reference sequence for these regions due to the high levels of repetition. The equivalent WGA GP4 tumour frequency plots are shown in Fig.Â <a reveal-id="DSS021F2" data-open="DSS021F2" class="link link-reveal link-table xref-fig">2</a>f, iii and iv. Although coverages across the chromosomes are high, the frequency profiles are much more variable than the unamplified DNA. Thus, at the fine mapping level, the amplification process results in some level of differential amplification bias across the genome. This variability in the frequency profiles does unfortunately make it difficult to confidently call copy number variations. Nevertheless, these data demonstrate that the MP sequencing from WGA DNA does result in extensive genome coverage across chromosomes.</p>                            <h3 scrollto-destination=10636864 id="10636864" class="section-title" data-legacy-id=s3c>3.3. Large genetic aberrations in the GP3 and GP4 cell populations</h3>
<p>TableÂ <a reveal-id="DSS021TB1" data-open="DSS021TB1" class="link link-reveal link-table xref-fig">1</a> lists the large genomic aberrations detected in the GP3 and GP4 of this prostate cancer by our MP sequencing. The events recorded passed bioinformatics filters in the mapping algorithm detailed in Materials and methods. Initial MP mapping predicted many more events prior to filtering, but the majority were false positives due to miss-mapping from highly repetitive or currently undefined regions within the reference human genome. The tableÂ is split into potential inter-chromosomal translocations and intra-chromosomal rearrangements (amplifications, deletions or inversions). The first significant observation is that the majority of large genomic aberrations are present in both the GP3 and GP4 cell populations of this prostate cancer, presenting strong evidence of common origin or clonality between these two Gleason patterns. However, the numbers of associated MPs varies considerably, both between the same and different chromosomal events. PCR was used to validate events in TableÂ <a reveal-id="DSS021TB1" data-open="DSS021TB1" class="link link-reveal link-table xref-fig">1</a> assessing the accuracy of the algorithm used to call genomic rearrangements from WGA DNA. </p>                        <a id="10636866" scrollto-destination="10636866"></a>
<div content-id="DSS021TB1" class="table-modal"><div class="table-wrap"><div id="DSS021TB1" class="table-wrap-title" data-id="DSS021TB1"><div class="label -label">TableÂ 1.</div><div class="caption"><p>Large genomic rearrangements in the GP3 and GP4 cancers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>EventÂ </th><th>Loci 1Â </th><th>Loci 2Â </th><th>Position 1Â </th><th>Position 2Â </th><th>Gene 1Â </th><th>Gene 2Â </th><th>#MP G3Â </th><th>#MP G4Â </th></tr></thead><tbody><tr><td><em>t</em>(1-12)aÂ </td><td>1q32.3cÂ </td><td>12q21.33aÂ </td><td>213127619Â </td><td>89404575Â </td><td>VASH2Â </td><td>AK055062Â </td><td>11Â </td><td>21Â </td></tr><tr><td><em>t</em>(1-12)bÂ </td><td>1p36.13eÂ </td><td>12p11.22aÂ </td><td>17481805Â </td><td>30504168Â </td><td>no_geneÂ </td><td>no_geneÂ </td><td>0Â </td><td>3Â </td></tr><tr><td><em>t</em>(2-14)Â </td><td>2q24.1cÂ </td><td>14q32.12aÂ </td><td>157595999Â </td><td>92919912Â </td><td>no_geneÂ </td><td>SLC24A4Â </td><td>5Â </td><td>0Â </td></tr><tr><td><em>t</em>(2-23)Â </td><td>2q35cÂ </td><td>Xq22.3bÂ </td><td>217539908Â </td><td>105359151Â </td><td>IGFBP5Â </td><td>no_geneÂ </td><td>5Â </td><td>0Â </td></tr><tr><td><em>t</em>(5-11)Â </td><td>5p13.1cÂ </td><td>11q24.3bÂ </td><td>38567230Â </td><td>129589054Â </td><td>LIFRÂ </td><td>no_geneÂ </td><td>3Â </td><td>0Â </td></tr><tr><td><em>t</em>(7-17)Â </td><td>7p21.2aÂ </td><td>17p13.1cÂ </td><td>13910005Â </td><td>7973212Â </td><td>ETV1Â </td><td>ALOX12BÂ </td><td>10Â </td><td>10Â </td></tr><tr><td><em>t</em>(8-11)aÂ </td><td>8q22.2bÂ </td><td>11q13.2cÂ </td><td>100826483Â </td><td>67861296Â </td><td>VPS13BÂ </td><td>CHKAÂ </td><td>4Â </td><td>3Â </td></tr><tr><td><em>t</em>(8-11)bÂ </td><td>8q22.2bÂ </td><td>11q24.2bÂ </td><td>100473404Â </td><td>125522385Â </td><td>VPS13BÂ </td><td>CHEK1Â </td><td>2Â </td><td>15Â </td></tr><tr><td><em>t</em>(17-19)aÂ </td><td>17q22bÂ </td><td>19q13.32aÂ </td><td>53592582Â </td><td>45963102Â </td><td>no_geneÂ </td><td>BCAMÂ </td><td>1Â </td><td>9Â </td></tr><tr><td><em>t</em>(7-19)bÂ </td><td>17q21.31cÂ </td><td>19q13.32aÂ </td><td>43268092Â </td><td>45311607Â </td><td>no_geneÂ </td><td>ERCC1Â </td><td>1Â </td><td>4Â </td></tr><tr><td><em>r</em>(1-1)Â </td><td>1p33dÂ </td><td>1p34.1bÂ </td><td>47040276Â </td><td>46048889Â </td><td>MKNK1Â </td><td>NASPÂ </td><td>2Â </td><td>7Â </td></tr><tr><td><em>r</em>(2-2)aÂ </td><td>2q13bÂ </td><td>2q13bÂ </td><td>111600477Â </td><td>111523813Â </td><td>ACOXLÂ </td><td>ACOXLÂ </td><td>10Â </td><td>10Â </td></tr><tr><td><em>r</em>(2-2)bÂ </td><td>2q13dÂ </td><td>2q13bÂ </td><td>113373061Â </td><td>111874072Â </td><td>no_geneÂ </td><td>ACOXLÂ </td><td>6Â </td><td>4Â </td></tr><tr><td><em>r</em>(3-3)aÂ </td><td>3p22.3bÂ </td><td>3p22.3bÂ </td><td>34811976Â </td><td>34726774Â </td><td>no_geneÂ </td><td>no_geneÂ </td><td>12Â </td><td>7Â </td></tr><tr><td><em>r</em>(3-3)bÂ </td><td>3q24cÂ </td><td>3q11.2cÂ </td><td>145684839Â </td><td>97497658Â </td><td>no_geneÂ </td><td>ARL6Â </td><td>0Â </td><td>5Â </td></tr><tr><td><em>r</em>(4-4)aÂ </td><td>4q31.21cÂ </td><td>4q24aÂ </td><td>145217293Â </td><td>102291108Â </td><td>no_geneÂ </td><td>(PPP3CA)Â </td><td>3Â </td><td>6Â </td></tr><tr><td><em>r</em>(4-4)bÂ </td><td>4q31.22aÂ </td><td>4q27bÂ </td><td>146813724Â </td><td>121967686Â </td><td>ZNF827Â </td><td>C4orf31Â </td><td>3Â </td><td>9Â </td></tr><tr><td><em>r</em>(4-4)cÂ </td><td>4q28.3hÂ </td><td>4q24aÂ </td><td>137887074Â </td><td>101807230Â </td><td>no_geneÂ </td><td>no_geneÂ </td><td>3Â </td><td>21Â </td></tr><tr><td><em>r</em>(4-4)dÂ </td><td>4q28.1cÂ </td><td>4q27bÂ </td><td>126358971Â </td><td>121952037Â </td><td>FAT4Â </td><td>no_geneÂ </td><td>3Â </td><td>20Â </td></tr><tr><td><em>r</em>(4-4)eÂ </td><td>4q31.21cÂ </td><td>4q24eÂ </td><td>146126396Â </td><td>107031959Â </td><td>(OTUD4)Â </td><td>TBCKLÂ </td><td>3Â </td><td>11Â </td></tr><tr><td><em>r</em>(6-6)Â </td><td>6q16.3dÂ </td><td>6q16.1eÂ </td><td>104492242Â </td><td>97668534Â </td><td>no_geneÂ </td><td>C6orf167Â </td><td>2Â </td><td>16Â </td></tr><tr><td><em>r</em>(7-7)Â </td><td>7q36.2cÂ </td><td>7q36.2bÂ </td><td>154376229Â </td><td>154265528Â </td><td>DPP6Â </td><td>DPP6Â </td><td>1Â </td><td>5Â </td></tr><tr><td><em>r</em>(8-8)aÂ </td><td>8p23.1aÂ </td><td>8p23.1cÂ </td><td>12610164Â </td><td>10101851Â </td><td>LONRF1Â </td><td>MSRAÂ </td><td>8Â </td><td>21Â </td></tr><tr><td><em>r</em>(8-8)bÂ </td><td>8p22eÂ </td><td>8p23.1aÂ </td><td>12718032Â </td><td>12607655Â </td><td>no_geneÂ </td><td>LONRF1Â </td><td>3Â </td><td>19Â </td></tr><tr><td><em>r</em>(9-9)Â </td><td>9q32dÂ </td><td>9q32dÂ </td><td>117100468Â </td><td>116868162Â </td><td>AKNAÂ </td><td>(KIF12)Â </td><td>11Â </td><td>15Â </td></tr><tr><td><em>r</em>(11-11)aÂ </td><td>11q24.2bÂ </td><td>11q13.2cÂ </td><td>125525691Â </td><td>68317314Â </td><td>CHEK1Â </td><td>SAPS3Â </td><td>2Â </td><td>13Â </td></tr><tr><td><em>r</em>(11-11)bÂ </td><td>11q23.3eÂ </td><td>11q23.1dÂ </td><td>117933682Â </td><td>112273163Â </td><td>no_geneÂ </td><td>no_geneÂ </td><td>7Â </td><td>27Â </td></tr><tr><td><em>r</em>(12-12)aÂ </td><td>12q23.1dÂ </td><td>12q23.1dÂ </td><td>101157176Â </td><td>100665614Â </td><td>ANO4Â </td><td>SCYL2Â </td><td>0Â </td><td>7Â </td></tr><tr><td><em>r</em>(12-12)bÂ </td><td>12q21.1aÂ </td><td>12q15dÂ </td><td>71683589Â </td><td>71437383Â </td><td>(TSPAN8)Â </td><td>TSPAN8Â </td><td>1Â </td><td>11Â </td></tr><tr><td><em>r</em>(23-23)Â </td><td>Xq25hÂ </td><td>Xq25hÂ </td><td>128603069Â </td><td>128566668Â </td><td>(SMARCA1)Â </td><td>SMARCA1Â </td><td>6Â </td><td>5Â </td></tr></tbody></table></div><div content-id="DSS021TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>EventÂ </th><th>Loci 1Â </th><th>Loci 2Â </th><th>Position 1Â </th><th>Position 2Â </th><th>Gene 1Â </th><th>Gene 2Â </th><th>#MP G3Â </th><th>#MP G4Â </th></tr></thead><tbody><tr><td><em>t</em>(1-12)aÂ </td><td>1q32.3cÂ </td><td>12q21.33aÂ </td><td>213127619Â </td><td>89404575Â </td><td>VASH2Â </td><td>AK055062Â </td><td>11Â </td><td>21Â </td></tr><tr><td><em>t</em>(1-12)bÂ </td><td>1p36.13eÂ </td><td>12p11.22aÂ </td><td>17481805Â </td><td>30504168Â </td><td>no_geneÂ </td><td>no_geneÂ </td><td>0Â </td><td>3Â </td></tr><tr><td><em>t</em>(2-14)Â </td><td>2q24.1cÂ </td><td>14q32.12aÂ </td><td>157595999Â </td><td>92919912Â </td><td>no_geneÂ </td><td>SLC24A4Â </td><td>5Â </td><td>0Â </td></tr><tr><td><em>t</em>(2-23)Â </td><td>2q35cÂ </td><td>Xq22.3bÂ </td><td>217539908Â </td><td>105359151Â </td><td>IGFBP5Â </td><td>no_geneÂ </td><td>5Â </td><td>0Â </td></tr><tr><td><em>t</em>(5-11)Â </td><td>5p13.1cÂ </td><td>11q24.3bÂ </td><td>38567230Â </td><td>129589054Â </td><td>LIFRÂ </td><td>no_geneÂ </td><td>3Â </td><td>0Â </td></tr><tr><td><em>t</em>(7-17)Â </td><td>7p21.2aÂ </td><td>17p13.1cÂ </td><td>13910005Â </td><td>7973212Â </td><td>ETV1Â </td><td>ALOX12BÂ </td><td>10Â </td><td>10Â </td></tr><tr><td><em>t</em>(8-11)aÂ </td><td>8q22.2bÂ </td><td>11q13.2cÂ </td><td>100826483Â </td><td>67861296Â </td><td>VPS13BÂ </td><td>CHKAÂ </td><td>4Â </td><td>3Â </td></tr><tr><td><em>t</em>(8-11)bÂ </td><td>8q22.2bÂ </td><td>11q24.2bÂ </td><td>100473404Â </td><td>125522385Â </td><td>VPS13BÂ </td><td>CHEK1Â </td><td>2Â </td><td>15Â </td></tr><tr><td><em>t</em>(17-19)aÂ </td><td>17q22bÂ </td><td>19q13.32aÂ </td><td>53592582Â </td><td>45963102Â </td><td>no_geneÂ </td><td>BCAMÂ </td><td>1Â </td><td>9Â </td></tr><tr><td><em>t</em>(7-19)bÂ </td><td>17q21.31cÂ </td><td>19q13.32aÂ </td><td>43268092Â </td><td>45311607Â </td><td>no_geneÂ </td><td>ERCC1Â </td><td>1Â </td><td>4Â </td></tr><tr><td><em>r</em>(1-1)Â </td><td>1p33dÂ </td><td>1p34.1bÂ </td><td>47040276Â </td><td>46048889Â </td><td>MKNK1Â </td><td>NASPÂ </td><td>2Â </td><td>7Â </td></tr><tr><td><em>r</em>(2-2)aÂ </td><td>2q13bÂ </td><td>2q13bÂ </td><td>111600477Â </td><td>111523813Â </td><td>ACOXLÂ </td><td>ACOXLÂ </td><td>10Â </td><td>10Â </td></tr><tr><td><em>r</em>(2-2)bÂ </td><td>2q13dÂ </td><td>2q13bÂ </td><td>113373061Â </td><td>111874072Â </td><td>no_geneÂ </td><td>ACOXLÂ </td><td>6Â </td><td>4Â </td></tr><tr><td><em>r</em>(3-3)aÂ </td><td>3p22.3bÂ </td><td>3p22.3bÂ </td><td>34811976Â </td><td>34726774Â </td><td>no_geneÂ </td><td>no_geneÂ </td><td>12Â </td><td>7Â </td></tr><tr><td><em>r</em>(3-3)bÂ </td><td>3q24cÂ </td><td>3q11.2cÂ </td><td>145684839Â </td><td>97497658Â </td><td>no_geneÂ </td><td>ARL6Â </td><td>0Â </td><td>5Â </td></tr><tr><td><em>r</em>(4-4)aÂ </td><td>4q31.21cÂ </td><td>4q24aÂ </td><td>145217293Â </td><td>102291108Â </td><td>no_geneÂ </td><td>(PPP3CA)Â </td><td>3Â </td><td>6Â </td></tr><tr><td><em>r</em>(4-4)bÂ </td><td>4q31.22aÂ </td><td>4q27bÂ </td><td>146813724Â </td><td>121967686Â </td><td>ZNF827Â </td><td>C4orf31Â </td><td>3Â </td><td>9Â </td></tr><tr><td><em>r</em>(4-4)cÂ </td><td>4q28.3hÂ </td><td>4q24aÂ </td><td>137887074Â </td><td>101807230Â </td><td>no_geneÂ </td><td>no_geneÂ </td><td>3Â </td><td>21Â </td></tr><tr><td><em>r</em>(4-4)dÂ </td><td>4q28.1cÂ </td><td>4q27bÂ </td><td>126358971Â </td><td>121952037Â </td><td>FAT4Â </td><td>no_geneÂ </td><td>3Â </td><td>20Â </td></tr><tr><td><em>r</em>(4-4)eÂ </td><td>4q31.21cÂ </td><td>4q24eÂ </td><td>146126396Â </td><td>107031959Â </td><td>(OTUD4)Â </td><td>TBCKLÂ </td><td>3Â </td><td>11Â </td></tr><tr><td><em>r</em>(6-6)Â </td><td>6q16.3dÂ </td><td>6q16.1eÂ </td><td>104492242Â </td><td>97668534Â </td><td>no_geneÂ </td><td>C6orf167Â </td><td>2Â </td><td>16Â </td></tr><tr><td><em>r</em>(7-7)Â </td><td>7q36.2cÂ </td><td>7q36.2bÂ </td><td>154376229Â </td><td>154265528Â </td><td>DPP6Â </td><td>DPP6Â </td><td>1Â </td><td>5Â </td></tr><tr><td><em>r</em>(8-8)aÂ </td><td>8p23.1aÂ </td><td>8p23.1cÂ </td><td>12610164Â </td><td>10101851Â </td><td>LONRF1Â </td><td>MSRAÂ </td><td>8Â </td><td>21Â </td></tr><tr><td><em>r</em>(8-8)bÂ </td><td>8p22eÂ </td><td>8p23.1aÂ </td><td>12718032Â </td><td>12607655Â </td><td>no_geneÂ </td><td>LONRF1Â </td><td>3Â </td><td>19Â </td></tr><tr><td><em>r</em>(9-9)Â </td><td>9q32dÂ </td><td>9q32dÂ </td><td>117100468Â </td><td>116868162Â </td><td>AKNAÂ </td><td>(KIF12)Â </td><td>11Â </td><td>15Â </td></tr><tr><td><em>r</em>(11-11)aÂ </td><td>11q24.2bÂ </td><td>11q13.2cÂ </td><td>125525691Â </td><td>68317314Â </td><td>CHEK1Â </td><td>SAPS3Â </td><td>2Â </td><td>13Â </td></tr><tr><td><em>r</em>(11-11)bÂ </td><td>11q23.3eÂ </td><td>11q23.1dÂ </td><td>117933682Â </td><td>112273163Â </td><td>no_geneÂ </td><td>no_geneÂ </td><td>7Â </td><td>27Â </td></tr><tr><td><em>r</em>(12-12)aÂ </td><td>12q23.1dÂ </td><td>12q23.1dÂ </td><td>101157176Â </td><td>100665614Â </td><td>ANO4Â </td><td>SCYL2Â </td><td>0Â </td><td>7Â </td></tr><tr><td><em>r</em>(12-12)bÂ </td><td>12q21.1aÂ </td><td>12q15dÂ </td><td>71683589Â </td><td>71437383Â </td><td>(TSPAN8)Â </td><td>TSPAN8Â </td><td>1Â </td><td>11Â </td></tr><tr><td><em>r</em>(23-23)Â </td><td>Xq25hÂ </td><td>Xq25hÂ </td><td>128603069Â </td><td>128566668Â </td><td>(SMARCA1)Â </td><td>SMARCA1Â </td><td>6Â </td><td>5Â </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/10636866" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS021TB1" class="table-wrap-title" data-id="DSS021TB1"><div class="label -label">TableÂ 1.</div><div class="caption"><p>Large genomic rearrangements in the GP3 and GP4 cancers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>EventÂ </th><th>Loci 1Â </th><th>Loci 2Â </th><th>Position 1Â </th><th>Position 2Â </th><th>Gene 1Â </th><th>Gene 2Â </th><th>#MP G3Â </th><th>#MP G4Â </th></tr></thead><tbody><tr><td><em>t</em>(1-12)aÂ </td><td>1q32.3cÂ </td><td>12q21.33aÂ </td><td>213127619Â </td><td>89404575Â </td><td>VASH2Â </td><td>AK055062Â </td><td>11Â </td><td>21Â </td></tr><tr><td><em>t</em>(1-12)bÂ </td><td>1p36.13eÂ </td><td>12p11.22aÂ </td><td>17481805Â </td><td>30504168Â </td><td>no_geneÂ </td><td>no_geneÂ </td><td>0Â </td><td>3Â </td></tr><tr><td><em>t</em>(2-14)Â </td><td>2q24.1cÂ </td><td>14q32.12aÂ </td><td>157595999Â </td><td>92919912Â </td><td>no_geneÂ </td><td>SLC24A4Â </td><td>5Â </td><td>0Â </td></tr><tr><td><em>t</em>(2-23)Â </td><td>2q35cÂ </td><td>Xq22.3bÂ </td><td>217539908Â </td><td>105359151Â </td><td>IGFBP5Â </td><td>no_geneÂ </td><td>5Â </td><td>0Â </td></tr><tr><td><em>t</em>(5-11)Â </td><td>5p13.1cÂ </td><td>11q24.3bÂ </td><td>38567230Â </td><td>129589054Â </td><td>LIFRÂ </td><td>no_geneÂ </td><td>3Â </td><td>0Â </td></tr><tr><td><em>t</em>(7-17)Â </td><td>7p21.2aÂ </td><td>17p13.1cÂ </td><td>13910005Â </td><td>7973212Â </td><td>ETV1Â </td><td>ALOX12BÂ </td><td>10Â </td><td>10Â </td></tr><tr><td><em>t</em>(8-11)aÂ </td><td>8q22.2bÂ </td><td>11q13.2cÂ </td><td>100826483Â </td><td>67861296Â </td><td>VPS13BÂ </td><td>CHKAÂ </td><td>4Â </td><td>3Â </td></tr><tr><td><em>t</em>(8-11)bÂ </td><td>8q22.2bÂ </td><td>11q24.2bÂ </td><td>100473404Â </td><td>125522385Â </td><td>VPS13BÂ </td><td>CHEK1Â </td><td>2Â </td><td>15Â </td></tr><tr><td><em>t</em>(17-19)aÂ </td><td>17q22bÂ </td><td>19q13.32aÂ </td><td>53592582Â </td><td>45963102Â </td><td>no_geneÂ </td><td>BCAMÂ </td><td>1Â </td><td>9Â </td></tr><tr><td><em>t</em>(7-19)bÂ </td><td>17q21.31cÂ </td><td>19q13.32aÂ </td><td>43268092Â </td><td>45311607Â </td><td>no_geneÂ </td><td>ERCC1Â </td><td>1Â </td><td>4Â </td></tr><tr><td><em>r</em>(1-1)Â </td><td>1p33dÂ </td><td>1p34.1bÂ </td><td>47040276Â </td><td>46048889Â </td><td>MKNK1Â </td><td>NASPÂ </td><td>2Â </td><td>7Â </td></tr><tr><td><em>r</em>(2-2)aÂ </td><td>2q13bÂ </td><td>2q13bÂ </td><td>111600477Â </td><td>111523813Â </td><td>ACOXLÂ </td><td>ACOXLÂ </td><td>10Â </td><td>10Â </td></tr><tr><td><em>r</em>(2-2)bÂ </td><td>2q13dÂ </td><td>2q13bÂ </td><td>113373061Â </td><td>111874072Â </td><td>no_geneÂ </td><td>ACOXLÂ </td><td>6Â </td><td>4Â </td></tr><tr><td><em>r</em>(3-3)aÂ </td><td>3p22.3bÂ </td><td>3p22.3bÂ </td><td>34811976Â </td><td>34726774Â </td><td>no_geneÂ </td><td>no_geneÂ </td><td>12Â </td><td>7Â </td></tr><tr><td><em>r</em>(3-3)bÂ </td><td>3q24cÂ </td><td>3q11.2cÂ </td><td>145684839Â </td><td>97497658Â </td><td>no_geneÂ </td><td>ARL6Â </td><td>0Â </td><td>5Â </td></tr><tr><td><em>r</em>(4-4)aÂ </td><td>4q31.21cÂ </td><td>4q24aÂ </td><td>145217293Â </td><td>102291108Â </td><td>no_geneÂ </td><td>(PPP3CA)Â </td><td>3Â </td><td>6Â </td></tr><tr><td><em>r</em>(4-4)bÂ </td><td>4q31.22aÂ </td><td>4q27bÂ </td><td>146813724Â </td><td>121967686Â </td><td>ZNF827Â </td><td>C4orf31Â </td><td>3Â </td><td>9Â </td></tr><tr><td><em>r</em>(4-4)cÂ </td><td>4q28.3hÂ </td><td>4q24aÂ </td><td>137887074Â </td><td>101807230Â </td><td>no_geneÂ </td><td>no_geneÂ </td><td>3Â </td><td>21Â </td></tr><tr><td><em>r</em>(4-4)dÂ </td><td>4q28.1cÂ </td><td>4q27bÂ </td><td>126358971Â </td><td>121952037Â </td><td>FAT4Â </td><td>no_geneÂ </td><td>3Â </td><td>20Â </td></tr><tr><td><em>r</em>(4-4)eÂ </td><td>4q31.21cÂ </td><td>4q24eÂ </td><td>146126396Â </td><td>107031959Â </td><td>(OTUD4)Â </td><td>TBCKLÂ </td><td>3Â </td><td>11Â </td></tr><tr><td><em>r</em>(6-6)Â </td><td>6q16.3dÂ </td><td>6q16.1eÂ </td><td>104492242Â </td><td>97668534Â </td><td>no_geneÂ </td><td>C6orf167Â </td><td>2Â </td><td>16Â </td></tr><tr><td><em>r</em>(7-7)Â </td><td>7q36.2cÂ </td><td>7q36.2bÂ </td><td>154376229Â </td><td>154265528Â </td><td>DPP6Â </td><td>DPP6Â </td><td>1Â </td><td>5Â </td></tr><tr><td><em>r</em>(8-8)aÂ </td><td>8p23.1aÂ </td><td>8p23.1cÂ </td><td>12610164Â </td><td>10101851Â </td><td>LONRF1Â </td><td>MSRAÂ </td><td>8Â </td><td>21Â </td></tr><tr><td><em>r</em>(8-8)bÂ </td><td>8p22eÂ </td><td>8p23.1aÂ </td><td>12718032Â </td><td>12607655Â </td><td>no_geneÂ </td><td>LONRF1Â </td><td>3Â </td><td>19Â </td></tr><tr><td><em>r</em>(9-9)Â </td><td>9q32dÂ </td><td>9q32dÂ </td><td>117100468Â </td><td>116868162Â </td><td>AKNAÂ </td><td>(KIF12)Â </td><td>11Â </td><td>15Â </td></tr><tr><td><em>r</em>(11-11)aÂ </td><td>11q24.2bÂ </td><td>11q13.2cÂ </td><td>125525691Â </td><td>68317314Â </td><td>CHEK1Â </td><td>SAPS3Â </td><td>2Â </td><td>13Â </td></tr><tr><td><em>r</em>(11-11)bÂ </td><td>11q23.3eÂ </td><td>11q23.1dÂ </td><td>117933682Â </td><td>112273163Â </td><td>no_geneÂ </td><td>no_geneÂ </td><td>7Â </td><td>27Â </td></tr><tr><td><em>r</em>(12-12)aÂ </td><td>12q23.1dÂ </td><td>12q23.1dÂ </td><td>101157176Â </td><td>100665614Â </td><td>ANO4Â </td><td>SCYL2Â </td><td>0Â </td><td>7Â </td></tr><tr><td><em>r</em>(12-12)bÂ </td><td>12q21.1aÂ </td><td>12q15dÂ </td><td>71683589Â </td><td>71437383Â </td><td>(TSPAN8)Â </td><td>TSPAN8Â </td><td>1Â </td><td>11Â </td></tr><tr><td><em>r</em>(23-23)Â </td><td>Xq25hÂ </td><td>Xq25hÂ </td><td>128603069Â </td><td>128566668Â </td><td>(SMARCA1)Â </td><td>SMARCA1Â </td><td>6Â </td><td>5Â </td></tr></tbody></table></div><div content-id="DSS021TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>EventÂ </th><th>Loci 1Â </th><th>Loci 2Â </th><th>Position 1Â </th><th>Position 2Â </th><th>Gene 1Â </th><th>Gene 2Â </th><th>#MP G3Â </th><th>#MP G4Â </th></tr></thead><tbody><tr><td><em>t</em>(1-12)aÂ </td><td>1q32.3cÂ </td><td>12q21.33aÂ </td><td>213127619Â </td><td>89404575Â </td><td>VASH2Â </td><td>AK055062Â </td><td>11Â </td><td>21Â </td></tr><tr><td><em>t</em>(1-12)bÂ </td><td>1p36.13eÂ </td><td>12p11.22aÂ </td><td>17481805Â </td><td>30504168Â </td><td>no_geneÂ </td><td>no_geneÂ </td><td>0Â </td><td>3Â </td></tr><tr><td><em>t</em>(2-14)Â </td><td>2q24.1cÂ </td><td>14q32.12aÂ </td><td>157595999Â </td><td>92919912Â </td><td>no_geneÂ </td><td>SLC24A4Â </td><td>5Â </td><td>0Â </td></tr><tr><td><em>t</em>(2-23)Â </td><td>2q35cÂ </td><td>Xq22.3bÂ </td><td>217539908Â </td><td>105359151Â </td><td>IGFBP5Â </td><td>no_geneÂ </td><td>5Â </td><td>0Â </td></tr><tr><td><em>t</em>(5-11)Â </td><td>5p13.1cÂ </td><td>11q24.3bÂ </td><td>38567230Â </td><td>129589054Â </td><td>LIFRÂ </td><td>no_geneÂ </td><td>3Â </td><td>0Â </td></tr><tr><td><em>t</em>(7-17)Â </td><td>7p21.2aÂ </td><td>17p13.1cÂ </td><td>13910005Â </td><td>7973212Â </td><td>ETV1Â </td><td>ALOX12BÂ </td><td>10Â </td><td>10Â </td></tr><tr><td><em>t</em>(8-11)aÂ </td><td>8q22.2bÂ </td><td>11q13.2cÂ </td><td>100826483Â </td><td>67861296Â </td><td>VPS13BÂ </td><td>CHKAÂ </td><td>4Â </td><td>3Â </td></tr><tr><td><em>t</em>(8-11)bÂ </td><td>8q22.2bÂ </td><td>11q24.2bÂ </td><td>100473404Â </td><td>125522385Â </td><td>VPS13BÂ </td><td>CHEK1Â </td><td>2Â </td><td>15Â </td></tr><tr><td><em>t</em>(17-19)aÂ </td><td>17q22bÂ </td><td>19q13.32aÂ </td><td>53592582Â </td><td>45963102Â </td><td>no_geneÂ </td><td>BCAMÂ </td><td>1Â </td><td>9Â </td></tr><tr><td><em>t</em>(7-19)bÂ </td><td>17q21.31cÂ </td><td>19q13.32aÂ </td><td>43268092Â </td><td>45311607Â </td><td>no_geneÂ </td><td>ERCC1Â </td><td>1Â </td><td>4Â </td></tr><tr><td><em>r</em>(1-1)Â </td><td>1p33dÂ </td><td>1p34.1bÂ </td><td>47040276Â </td><td>46048889Â </td><td>MKNK1Â </td><td>NASPÂ </td><td>2Â </td><td>7Â </td></tr><tr><td><em>r</em>(2-2)aÂ </td><td>2q13bÂ </td><td>2q13bÂ </td><td>111600477Â </td><td>111523813Â </td><td>ACOXLÂ </td><td>ACOXLÂ </td><td>10Â </td><td>10Â </td></tr><tr><td><em>r</em>(2-2)bÂ </td><td>2q13dÂ </td><td>2q13bÂ </td><td>113373061Â </td><td>111874072Â </td><td>no_geneÂ </td><td>ACOXLÂ </td><td>6Â </td><td>4Â </td></tr><tr><td><em>r</em>(3-3)aÂ </td><td>3p22.3bÂ </td><td>3p22.3bÂ </td><td>34811976Â </td><td>34726774Â </td><td>no_geneÂ </td><td>no_geneÂ </td><td>12Â </td><td>7Â </td></tr><tr><td><em>r</em>(3-3)bÂ </td><td>3q24cÂ </td><td>3q11.2cÂ </td><td>145684839Â </td><td>97497658Â </td><td>no_geneÂ </td><td>ARL6Â </td><td>0Â </td><td>5Â </td></tr><tr><td><em>r</em>(4-4)aÂ </td><td>4q31.21cÂ </td><td>4q24aÂ </td><td>145217293Â </td><td>102291108Â </td><td>no_geneÂ </td><td>(PPP3CA)Â </td><td>3Â </td><td>6Â </td></tr><tr><td><em>r</em>(4-4)bÂ </td><td>4q31.22aÂ </td><td>4q27bÂ </td><td>146813724Â </td><td>121967686Â </td><td>ZNF827Â </td><td>C4orf31Â </td><td>3Â </td><td>9Â </td></tr><tr><td><em>r</em>(4-4)cÂ </td><td>4q28.3hÂ </td><td>4q24aÂ </td><td>137887074Â </td><td>101807230Â </td><td>no_geneÂ </td><td>no_geneÂ </td><td>3Â </td><td>21Â </td></tr><tr><td><em>r</em>(4-4)dÂ </td><td>4q28.1cÂ </td><td>4q27bÂ </td><td>126358971Â </td><td>121952037Â </td><td>FAT4Â </td><td>no_geneÂ </td><td>3Â </td><td>20Â </td></tr><tr><td><em>r</em>(4-4)eÂ </td><td>4q31.21cÂ </td><td>4q24eÂ </td><td>146126396Â </td><td>107031959Â </td><td>(OTUD4)Â </td><td>TBCKLÂ </td><td>3Â </td><td>11Â </td></tr><tr><td><em>r</em>(6-6)Â </td><td>6q16.3dÂ </td><td>6q16.1eÂ </td><td>104492242Â </td><td>97668534Â </td><td>no_geneÂ </td><td>C6orf167Â </td><td>2Â </td><td>16Â </td></tr><tr><td><em>r</em>(7-7)Â </td><td>7q36.2cÂ </td><td>7q36.2bÂ </td><td>154376229Â </td><td>154265528Â </td><td>DPP6Â </td><td>DPP6Â </td><td>1Â </td><td>5Â </td></tr><tr><td><em>r</em>(8-8)aÂ </td><td>8p23.1aÂ </td><td>8p23.1cÂ </td><td>12610164Â </td><td>10101851Â </td><td>LONRF1Â </td><td>MSRAÂ </td><td>8Â </td><td>21Â </td></tr><tr><td><em>r</em>(8-8)bÂ </td><td>8p22eÂ </td><td>8p23.1aÂ </td><td>12718032Â </td><td>12607655Â </td><td>no_geneÂ </td><td>LONRF1Â </td><td>3Â </td><td>19Â </td></tr><tr><td><em>r</em>(9-9)Â </td><td>9q32dÂ </td><td>9q32dÂ </td><td>117100468Â </td><td>116868162Â </td><td>AKNAÂ </td><td>(KIF12)Â </td><td>11Â </td><td>15Â </td></tr><tr><td><em>r</em>(11-11)aÂ </td><td>11q24.2bÂ </td><td>11q13.2cÂ </td><td>125525691Â </td><td>68317314Â </td><td>CHEK1Â </td><td>SAPS3Â </td><td>2Â </td><td>13Â </td></tr><tr><td><em>r</em>(11-11)bÂ </td><td>11q23.3eÂ </td><td>11q23.1dÂ </td><td>117933682Â </td><td>112273163Â </td><td>no_geneÂ </td><td>no_geneÂ </td><td>7Â </td><td>27Â </td></tr><tr><td><em>r</em>(12-12)aÂ </td><td>12q23.1dÂ </td><td>12q23.1dÂ </td><td>101157176Â </td><td>100665614Â </td><td>ANO4Â </td><td>SCYL2Â </td><td>0Â </td><td>7Â </td></tr><tr><td><em>r</em>(12-12)bÂ </td><td>12q21.1aÂ </td><td>12q15dÂ </td><td>71683589Â </td><td>71437383Â </td><td>(TSPAN8)Â </td><td>TSPAN8Â </td><td>1Â </td><td>11Â </td></tr><tr><td><em>r</em>(23-23)Â </td><td>Xq25hÂ </td><td>Xq25hÂ </td><td>128603069Â </td><td>128566668Â </td><td>(SMARCA1)Â </td><td>SMARCA1Â </td><td>6Â </td><td>5Â </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/10636866" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=10636867 id="10636867" class="section-title" data-legacy-id=s3d>3.4. Validation of a chromosome 1â€“12 balanced translocation event</h3>
<p>FigureÂ <a reveal-id="DSS021F3" data-open="DSS021F3" class="link link-reveal link-table xref-fig">3</a>aâ€“c presents the validation of a balanced translocation event between 1q32.3c and 12q21.33a, involving genes <em>VASH2</em> and <em>AK055062</em>, respectively (TableÂ <a reveal-id="DSS021TB1" data-open="DSS021TB1" class="link link-reveal link-table xref-fig">1</a>). The GP3 and GP4 cell populations recorded 11 and 21 MP reads, respectively (TableÂ <a reveal-id="DSS021TB1" data-open="DSS021TB1" class="link link-reveal link-table xref-fig">1</a>), which are mapped in Fig.Â <a reveal-id="DSS021F3" data-open="DSS021F3" class="link link-reveal link-table xref-fig">3</a>a. The upper and lower parts of the figures refer to chromosomes 1 and 12, respectively. MP reads are illustrated as either blue or red dots, depending on the polarity of the MP sequence mapping to the reference genome, with a line linking the paired reads. Central horizontally linked red-to-blue dots represent intra-chromosomal mapping MP fragments with the concordant bridged coverage spans (3â€“5 kb). Vertically linked red-to-blue dots are associated with MPs specific to this translocation event between chromosomes 1 and 12 (Fig.Â <a reveal-id="DSS021F3" data-open="DSS021F3" class="link link-reveal link-table xref-fig">3</a>a, i and ii). Two sets of red/blue-linked MP events spanning the two chromosomes are observed, red-to-blue and blue-to-red. This is indicative of a balanced translocation event between the two chromosomes, where genomic regions have been exchanged between the two chromosomes resulting in <em>t</em>(1-12) and <em>t</em>(12-1) translocation products (Fig.Â <a reveal-id="DSS021F3" data-open="DSS021F3" class="link link-reveal link-table xref-fig">3</a>c). The regions where the redâ€“blue and blueâ€“red mapping events converge indicate the location of the translocation breakpoints. </p>                        <a id="10636869" scrollto-destination="10636869"></a>
<div id="DSS021F3" data-id="DSS021F3" class="fig fig-section"><div class="label fig-label">FigureÂ 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss021/2/m_dss02103.jpeg?Expires=2147483647&Signature=44IqICPdKynq7ZmgpMBxLSdcBI5FEWVcG8sJZldbnf-AazptMNsulVIyJl6Xz9yPYdU4Oo6xIOVR86OpLDL12uAkoH3vJKvWSjuOVzWTP8FOfLVySfJSyPThdGmsjFnevWSRLHvNWn1zOzY1gTHSswKoOzWdP4uBIuPmWNJZBGxxqVt5AF0nomVTvoU1QTsomzY1dhgCI7Qwt7Q~2OF4e1YEhh6Cp6PH5d~oV5CR3BEV4OWtGn3aKOSdLiOPFzE65WDUOp~JBkEQxumrRKQqo7oq-AIBoKzqXHImbxIIQRaRjeWOh0k907G8m3ma7oBs8Mcf-lXRCHKsh8Lna46I4A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Validation of chromosome translocation events: mapping of two independent translocation events are depicted; a balanced translocation between chromosomes 1 and 12 (aâ€“c) and a complex rearrangement between chromosomes 8 and 11 (e and f). (a) MP sequences mapping to chromosomes 1 and 12 for GP3 and GP4 are depicted above and below the zero axis, respectively, as red or blue dots dependent on the direction of the sequence read mapping to the reference genome. Horizontally linked red and blue dots closest to this axis depict normal mapping MP sequences to chromosome 1 or 12 alone. MP sequences linked vertically depict translocations between these two chromosomes. (b) PCR validation using primers specific to the t(1-12) event (i) for GP3 and GP4 (lanes 5 and 6) and the t(12-1) event (ii) for GP3 and GP4 (lanes 9 and 10). gDNA (lanes 2 and 8) and different prostate tumour tissues (PR1 and PR2, lanes 3 and 4) were used as controls together with 1 kb ladder (lanes 1 and 7). (c) Schematic representation of predicted VASH2 and AK055062 fusion products resulting from the translocation events described. (d) Mapping of MP sequences to three inter-linked events at genomic loci 8q22.2b, 11q13.2c and 11q24.2b. Events 1 and 2 describe two t(8-11) translocation events linking VPS13B to CHKA (i) and CHEK1 (ii), respectively. Event 3 describes an r(11-11) intra-chromosomal rearrangement linking CHEK1 with SASP3 (iii). (e) PCR validation 1% agarose gels are presented for four different primer sets (1â€“4) for event 1 (i), five different primers sets (1â€“5) for event 2 (ii) and two primer sets (1 and 2) for event 3 (iii) for the GP3 and GP4 WGA DNA and gDNA as control. (f) Schematic representation of the three fusion events on chromosome 8 (red) and 11 (blue), describing the potential products and the impact on the gene regions involved. Arrows describe the coding direction of the genes and an X represents the loss of the promoter regions." path-from-xml="dss02103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10636869/dss02103.jpeg" path-from-xml="dss02103.jpeg" target="_blank">View large</a><a section="10636869" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss021/2/dss02103.jpeg?Expires=2147483647&Signature=mqC3cbxv5U3D-Ov4NgKw1avrfVnzQYd1S3WJvIurU6jPZNWgcUug5Nqj9BRC2QZO21vQHUPBGXWXAApRs6A1sLfD9dSEBJyUnZSUahbPPpJIjIohfW-j13iFsxzZzCY7EVFxVoLCkAFNcbl-8dURMq88hf9AKDc4rkSNRKPVYDaUFfkfZV5M7CCdnnOft70mo1MgxEg2PX3QALF158RdVvjSqf6t8z2Fun5R5nzoPwHgqRFyqB3GYbECouKHMNWNLcr0wSnyDbh229xl4Nc8nQlNg9fIGsSp-HZ4obwwXTDBqdvXHnmYZuUmXiMC8gczOob-uY18UUgYaZprf79Q2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10636869&ar=629019&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Validation of chromosome translocation events: mapping of two independent translocation events are depicted; a balanced translocation between chromosomes 1 and 12 (aâ€“c) and a complex rearrangement between chromosomes 8 and 11 (e and f). (a) MP sequences mapping to chromosomes 1 and 12 for GP3 and GP4 are depicted above and below the zero axis, respectively, as red or blue dots dependent on the direction of the sequence read mapping to the reference genome. Horizontally linked red and blue dots closest to this axis depict normal mapping MP sequences to chromosome 1 or 12 alone. MP sequences linked vertically depict translocations between these two chromosomes. (b) PCR validation using primers specific to the <em>t</em>(1-12) event (i) for GP3 and GP4 (lanes 5 and 6) and the <em>t</em>(12-1) event (ii) for GP3 and GP4 (lanes 9 and 10). gDNA (lanes 2 and 8) and different prostate tumour tissues (PR1 and PR2, lanes 3 and 4) were used as controls together with 1 kb ladder (lanes 1 and 7). (c) Schematic representation of predicted <em>VASH2</em> and <em>AK055062</em> fusion products resulting from the translocation events described. (d) Mapping of MP sequences to three inter-linked events at genomic loci 8q22.2b, 11q13.2c and 11q24.2b. Events 1 and 2 describe two <em>t</em>(8-11) translocation events linking <em>VPS13B</em> to <em>CHKA</em> (i) and <em>CHEK1</em> (ii), respectively. Event 3 describes an <em>r</em>(11-11) intra-chromosomal rearrangement linking <em>CHEK1</em> with <em>SASP3</em> (iii). (e) PCR validation 1% agarose gels are presented for four different primer sets (1â€“4) for event 1 (i), five different primers sets (1â€“5) for event 2 (ii) and two primer sets (1 and 2) for event 3 (iii) for the GP3 and GP4 WGA DNA and gDNA as control. (f) Schematic representation of the three fusion events on chromosome 8 (red) and 11 (blue), describing the potential products and the impact on the gene regions involved. Arrows describe the coding direction of the genes and an X represents the loss of the promoter regions.</p></div></div><div content-id="DSS021F3" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss021/2/m_dss02103.jpeg?Expires=2147483647&Signature=44IqICPdKynq7ZmgpMBxLSdcBI5FEWVcG8sJZldbnf-AazptMNsulVIyJl6Xz9yPYdU4Oo6xIOVR86OpLDL12uAkoH3vJKvWSjuOVzWTP8FOfLVySfJSyPThdGmsjFnevWSRLHvNWn1zOzY1gTHSswKoOzWdP4uBIuPmWNJZBGxxqVt5AF0nomVTvoU1QTsomzY1dhgCI7Qwt7Q~2OF4e1YEhh6Cp6PH5d~oV5CR3BEV4OWtGn3aKOSdLiOPFzE65WDUOp~JBkEQxumrRKQqo7oq-AIBoKzqXHImbxIIQRaRjeWOh0k907G8m3ma7oBs8Mcf-lXRCHKsh8Lna46I4A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Validation of chromosome translocation events: mapping of two independent translocation events are depicted; a balanced translocation between chromosomes 1 and 12 (aâ€“c) and a complex rearrangement between chromosomes 8 and 11 (e and f). (a) MP sequences mapping to chromosomes 1 and 12 for GP3 and GP4 are depicted above and below the zero axis, respectively, as red or blue dots dependent on the direction of the sequence read mapping to the reference genome. Horizontally linked red and blue dots closest to this axis depict normal mapping MP sequences to chromosome 1 or 12 alone. MP sequences linked vertically depict translocations between these two chromosomes. (b) PCR validation using primers specific to the t(1-12) event (i) for GP3 and GP4 (lanes 5 and 6) and the t(12-1) event (ii) for GP3 and GP4 (lanes 9 and 10). gDNA (lanes 2 and 8) and different prostate tumour tissues (PR1 and PR2, lanes 3 and 4) were used as controls together with 1 kb ladder (lanes 1 and 7). (c) Schematic representation of predicted VASH2 and AK055062 fusion products resulting from the translocation events described. (d) Mapping of MP sequences to three inter-linked events at genomic loci 8q22.2b, 11q13.2c and 11q24.2b. Events 1 and 2 describe two t(8-11) translocation events linking VPS13B to CHKA (i) and CHEK1 (ii), respectively. Event 3 describes an r(11-11) intra-chromosomal rearrangement linking CHEK1 with SASP3 (iii). (e) PCR validation 1% agarose gels are presented for four different primer sets (1â€“4) for event 1 (i), five different primers sets (1â€“5) for event 2 (ii) and two primer sets (1 and 2) for event 3 (iii) for the GP3 and GP4 WGA DNA and gDNA as control. (f) Schematic representation of the three fusion events on chromosome 8 (red) and 11 (blue), describing the potential products and the impact on the gene regions involved. Arrows describe the coding direction of the genes and an X represents the loss of the promoter regions." path-from-xml="dss02103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10636869/dss02103.jpeg" path-from-xml="dss02103.jpeg" target="_blank">View large</a><a section="10636869" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss021/2/dss02103.jpeg?Expires=2147483647&Signature=mqC3cbxv5U3D-Ov4NgKw1avrfVnzQYd1S3WJvIurU6jPZNWgcUug5Nqj9BRC2QZO21vQHUPBGXWXAApRs6A1sLfD9dSEBJyUnZSUahbPPpJIjIohfW-j13iFsxzZzCY7EVFxVoLCkAFNcbl-8dURMq88hf9AKDc4rkSNRKPVYDaUFfkfZV5M7CCdnnOft70mo1MgxEg2PX3QALF158RdVvjSqf6t8z2Fun5R5nzoPwHgqRFyqB3GYbECouKHMNWNLcr0wSnyDbh229xl4Nc8nQlNg9fIGsSp-HZ4obwwXTDBqdvXHnmYZuUmXiMC8gczOob-uY18UUgYaZprf79Q2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10636869&ar=629019&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Validation of chromosome translocation events: mapping of two independent translocation events are depicted; a balanced translocation between chromosomes 1 and 12 (aâ€“c) and a complex rearrangement between chromosomes 8 and 11 (e and f). (a) MP sequences mapping to chromosomes 1 and 12 for GP3 and GP4 are depicted above and below the zero axis, respectively, as red or blue dots dependent on the direction of the sequence read mapping to the reference genome. Horizontally linked red and blue dots closest to this axis depict normal mapping MP sequences to chromosome 1 or 12 alone. MP sequences linked vertically depict translocations between these two chromosomes. (b) PCR validation using primers specific to the <em>t</em>(1-12) event (i) for GP3 and GP4 (lanes 5 and 6) and the <em>t</em>(12-1) event (ii) for GP3 and GP4 (lanes 9 and 10). gDNA (lanes 2 and 8) and different prostate tumour tissues (PR1 and PR2, lanes 3 and 4) were used as controls together with 1 kb ladder (lanes 1 and 7). (c) Schematic representation of predicted <em>VASH2</em> and <em>AK055062</em> fusion products resulting from the translocation events described. (d) Mapping of MP sequences to three inter-linked events at genomic loci 8q22.2b, 11q13.2c and 11q24.2b. Events 1 and 2 describe two <em>t</em>(8-11) translocation events linking <em>VPS13B</em> to <em>CHKA</em> (i) and <em>CHEK1</em> (ii), respectively. Event 3 describes an <em>r</em>(11-11) intra-chromosomal rearrangement linking <em>CHEK1</em> with <em>SASP3</em> (iii). (e) PCR validation 1% agarose gels are presented for four different primer sets (1â€“4) for event 1 (i), five different primers sets (1â€“5) for event 2 (ii) and two primer sets (1 and 2) for event 3 (iii) for the GP3 and GP4 WGA DNA and gDNA as control. (f) Schematic representation of the three fusion events on chromosome 8 (red) and 11 (blue), describing the potential products and the impact on the gene regions involved. Arrows describe the coding direction of the genes and an X represents the loss of the promoter regions.</p></div></div><p>PCR primers spanning the indicated breakpoints on chromosomes 1 and 12 were used to validate the translocation events (Fig.Â <a reveal-id="DSS021F3" data-open="DSS021F3" class="link link-reveal link-table xref-fig">3</a>b, i and ii). Specific bands are observed for both the GP3 and GP4 samples (lanes 5, 6, 9 and 10) that are not present in the human gDNA control (lanes 2 and 8) or two additional prostate cancers from different patients (lanes 3 and 4). Sanger sequencing of excised PCR bands yielded unique fusion sites for both the <em>t</em>(1-12) and <em>t</em>(12-1) translocation events. Identical fusion junctions were observed in the GP3 and GP4 samples. The <em>t</em>(1-12) translocation resulted in the fusion [chr1:213 127 018] to [chr12:89 406 958]. The <em>t</em>(12-1) translocation mapped [chr12:89 407 157] to [chr1:213 127 057], but was more complex, involving an additional inversion of 40 nucleotides of chromosome 1 [chr1:213 127 017â€“213 127 056] with an overlap of four nucleotides (TAAG) at the 12â€“12 fusion point (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Both fusions are intronic, located between exons 2 and 3 of <em>VASH2</em> and exons 1 and 2 of <em>AK055062</em> resulting in two potential fusion proteins (Fig.Â <a reveal-id="DSS021F3" data-open="DSS021F3" class="link link-reveal link-table xref-fig">3</a>c). The consequence of these two fusion events, the translation products yielded and the impact on tumourigenesis still needs to be investigated and were beyond the scope of this current work.</p>                            <h3 scrollto-destination=10636871 id="10636871" class="section-title" data-legacy-id=s3e>3.5. Validation of chromosome 8â€“11 complex rearrangement</h3>
<p>FigureÂ <a reveal-id="DSS021F3" data-open="DSS021F3" class="link link-reveal link-table xref-fig">3</a>dâ€“f describes the mapping and validation of a more complex chromosomal rearrangement, involving two translocations from <em>VPS13B</em> on chromosome 8 to two distal sites on chromosome 11 involving <em>CHKA and CHEK1</em>. The MP mapping of these two events each predict a single junction, without associated balanced events (Fig.Â <a reveal-id="DSS021F3" data-open="DSS021F3" class="link link-reveal link-table xref-fig">3</a>d, i and ii). While one event predicts a direct translocation between the chromosomes 8 and 11 regions with blue-to-red MPs, the second event is more complex with red-to-red MPs predicting a sequence inversion. In order to understand this complex rearrangement, a third event involving an intra-chromosomal inversion (blue-to-blue MP) of a large segment of the q-arm of chromosome 11 linking <em>SASP2</em> to <em>CHEK1</em> needs to be considered (Fig.Â <a reveal-id="DSS021F3" data-open="DSS021F3" class="link link-reveal link-table xref-fig">3</a>d, iii). Considering the three events together the balanced recombination event can be predicted (Fig.Â <a reveal-id="DSS021F3" data-open="DSS021F3" class="link link-reveal link-table xref-fig">3</a>f). While the 8-11 translocation between <em>CHKA</em> and <em>VPS13B</em> describes one half of the rearrangement, events 2 and 3 together form the balanced event. Specifically, a large fragment of the q-arm of chromosome 11 inverted, linking the <em>CHKA</em> adjacent gene; <em>SASP3</em> to <em>CHEK1</em> at the termini of chromosome 11, with the rest of the <em>CHEK1</em> gene fusing with <em>VPS13B</em> on chromosome 8. Using primers spanning the predicted fusion sites, each event was validated by PCR generating unique bands in both the GP3 and GP4 cell populations that were not observed with control gDNA (Fig.Â <a reveal-id="DSS021F3" data-open="DSS021F3" class="link link-reveal link-table xref-fig">3</a>e). While events 1 and 3 would result in the fusion of two gene regions with the loss of promoter sequences, event 2 results in both <em>CHEK1</em> and <em>VPS13B</em> retaining their promoters and thus two truncated proteins may result with unknown cellular impact. Additionally, a âˆ¼300 kb fragment spanning from <em>CHKA</em> to <em>SAPS3</em> is unaccounted for in this description. However, the reduced coverage of MPs adjacent to the <em>CHKA</em> and <em>SASP3</em> MP regions in Fig.Â <a reveal-id="DSS021F3" data-open="DSS021F3" class="link link-reveal link-table xref-fig">3</a>d could indicate a deletion of these sequences.</p>                            <h3 scrollto-destination=10636873 id="10636873" class="section-title" data-legacy-id=s3f>3.6. Additional validations in the GP3 and GP4 cancers</h3>
<p>In order to further support this novel experimental methodology and bioinformatics algorithm, 12 additional events from TableÂ <a reveal-id="DSS021TB1" data-open="DSS021TB1" class="link link-reveal link-table xref-fig">1</a> were selected for PCR validation. All 12 events generated unique PCR products in the prostate cancer that were absent in control gDNA (Fig.Â <a reveal-id="DSS021F4" data-open="DSS021F4" class="link link-reveal link-table xref-fig">4</a>a). Significantly, all events but one validated in both the GP3 and GP4 samples; however, the intensity of the bands often varied between GP3 and GP4, indicating this did not result from LCM contamination. For several events, the intensity of the band was greater in the GP4 than the GP3, indicating a greater prevalence of the rearrangement in the GP4 cell population. Only the event <em>t</em>(2-23) validated predominantly in GP3 with no or potential very low levels observed in the GP4 (Fig.Â <a reveal-id="DSS021F4" data-open="DSS021F4" class="link link-reveal link-table xref-fig">4</a>aâ€“c). While it is difficult to confidently define the levels of each genomic aberration in these tumours considering the input of WGA DNA and the semi-quantitative PCR, the band intensities correlated to some extent with the numbers of associated MP reads (TableÂ <a reveal-id="DSS021TB1" data-open="DSS021TB1" class="link link-reveal link-table xref-fig">1</a>). Sanger sequencing of the validating PCR bands revealed the fusion breakpoints for each event (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Significantly, for each PCR band, the exact fusion junction was identified in both the GP3 and GP4 samples. These data present strong evidence for the clonality of GP3 and GP4 in this prostate cancer. Additionally, the presence of the exact junction breakpoints in both GP3 and GP4 makes these fusion events unlikely to be artefacts of WGA. </p>                        <a id="10636875" scrollto-destination="10636875"></a>
<div id="DSS021F4" data-id="DSS021F4" class="fig fig-section"><div class="label fig-label">FigureÂ 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss021/2/m_dss02104.jpeg?Expires=2147483647&Signature=BZw4SlvlC~YYpSoTzQ-Q9BDgpe6QelMcKMJXCZdhWGuIuMXNHxOIHNb4NMlq3lnkxt~rgzBnHM5O5OlCiuXrsmWfJZhGOBV448G9WK4fVy-DlthTc2BnIDTZiLwlcIck1MUrXb5Mo5dEegx7drcyduklPu8YoBeZCWHiW6f8HjlMWk8SbEue-K7WGASr0goDRGX~uvK4plElEixS1xalGl4aXiwHFnAAQeDOBt~~elZ~dGOCjRiexWPtvHz0BFWQ1h0txRhGGGmuapuXQKRKDJyOJ4FsyhFfln2W~41ZvFizhdVEyKsgfrTRzxTm8haLIwMYQDRntFKdd~cE8nKR2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Additional validations in the GP3 and GP4 cancers: (a) 1% agarose gels of PCR validation for 12 additional chromosomal rearrangement events in the GP3 and GP4 cancers, involving two primer sets and using gDNA as control. (b) PCR evaluation of aN and dN tissues for 17 previously validated events in GP3 and GP4 tumours of the same patient. (c) PCR validation in amplified (WGA) and unamplified (nonWGA) GP3 and GP4 tissues for four validated events." path-from-xml="dss02104.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10636875/dss02104.jpeg" path-from-xml="dss02104.jpeg" target="_blank">View large</a><a section="10636875" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss021/2/dss02104.jpeg?Expires=2147483647&Signature=vjZqLs~ML~iSIU8ObJgb3NN91yfmJQRfflnswb4D1dihaIw3fuvbjNv~rD4E~BYk9zNhN7LsypkNajcUr~XWETDjXmMUy-I88ySHq~BXTjZLa8MN0hIy-1ZVxb2PoJMeQe6xFnVxT33oR7cNPu1zroD3Y8CKp83m8O8mPIiDTiKcRdZTwXmy2jVcPXPSvIK2CbkNSHFNOyxFu~LAiobU4bfmd3S2zc9K-UY1qpr-oEydmPQW5uGeFeYXRW4A21SHVV6WZVwKIcjAl6PsavOcTsYTpZCWpTB325n3Zd9VbaDaakPCIVQMfp9iRv9YMPDsB1mTUzlRQ-4BLP7Gh~yppg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10636875&ar=629019&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02104.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Additional validations in the GP3 and GP4 cancers: (a) 1% agarose gels of PCR validation for 12 additional chromosomal rearrangement events in the GP3 and GP4 cancers, involving two primer sets and using gDNA as control. (b) PCR evaluation of aN and dN tissues for 17 previously validated events in GP3 and GP4 tumours of the same patient. (c) PCR validation in amplified (WGA) and unamplified (nonWGA) GP3 and GP4 tissues for four validated events.</p></div></div><div content-id="DSS021F4" class="fig fig-modal reveal-modal"><div class="label fig-label">FigureÂ 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss021/2/m_dss02104.jpeg?Expires=2147483647&Signature=BZw4SlvlC~YYpSoTzQ-Q9BDgpe6QelMcKMJXCZdhWGuIuMXNHxOIHNb4NMlq3lnkxt~rgzBnHM5O5OlCiuXrsmWfJZhGOBV448G9WK4fVy-DlthTc2BnIDTZiLwlcIck1MUrXb5Mo5dEegx7drcyduklPu8YoBeZCWHiW6f8HjlMWk8SbEue-K7WGASr0goDRGX~uvK4plElEixS1xalGl4aXiwHFnAAQeDOBt~~elZ~dGOCjRiexWPtvHz0BFWQ1h0txRhGGGmuapuXQKRKDJyOJ4FsyhFfln2W~41ZvFizhdVEyKsgfrTRzxTm8haLIwMYQDRntFKdd~cE8nKR2w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Additional validations in the GP3 and GP4 cancers: (a) 1% agarose gels of PCR validation for 12 additional chromosomal rearrangement events in the GP3 and GP4 cancers, involving two primer sets and using gDNA as control. (b) PCR evaluation of aN and dN tissues for 17 previously validated events in GP3 and GP4 tumours of the same patient. (c) PCR validation in amplified (WGA) and unamplified (nonWGA) GP3 and GP4 tissues for four validated events." path-from-xml="dss02104.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/10636875/dss02104.jpeg" path-from-xml="dss02104.jpeg" target="_blank">View large</a><a section="10636875" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss021/2/dss02104.jpeg?Expires=2147483647&Signature=vjZqLs~ML~iSIU8ObJgb3NN91yfmJQRfflnswb4D1dihaIw3fuvbjNv~rD4E~BYk9zNhN7LsypkNajcUr~XWETDjXmMUy-I88ySHq~BXTjZLa8MN0hIy-1ZVxb2PoJMeQe6xFnVxT33oR7cNPu1zroD3Y8CKp83m8O8mPIiDTiKcRdZTwXmy2jVcPXPSvIK2CbkNSHFNOyxFu~LAiobU4bfmd3S2zc9K-UY1qpr-oEydmPQW5uGeFeYXRW4A21SHVV6WZVwKIcjAl6PsavOcTsYTpZCWpTB325n3Zd9VbaDaakPCIVQMfp9iRv9YMPDsB1mTUzlRQ-4BLP7Gh~yppg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=10636875&ar=629019&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss02104.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Additional validations in the GP3 and GP4 cancers: (a) 1% agarose gels of PCR validation for 12 additional chromosomal rearrangement events in the GP3 and GP4 cancers, involving two primer sets and using gDNA as control. (b) PCR evaluation of aN and dN tissues for 17 previously validated events in GP3 and GP4 tumours of the same patient. (c) PCR validation in amplified (WGA) and unamplified (nonWGA) GP3 and GP4 tissues for four validated events.</p></div></div>                            <h3 scrollto-destination=10636876 id="10636876" class="section-title" data-legacy-id=s3g>3.7. Evaluation of genomic rearrangements in histologically normal adjacent tissues</h3>
<p>Adjacent peripheral gland normal (aN) from the same sections as the GP3 and GP4 cancers was isolated by LCM and WGA. Additional, distal peripheral normal (dN) from the opposite side of the patient's prostate and in a different frozen tissue block was similarly isolated. Fusion-specific PCRs evaluated 16 validated events in the aN and dN samples. Surprisingly, the majority of events were present in aN, although the intensity of the bands were consistently reduced (Fig.Â <a reveal-id="DSS021F4" data-open="DSS021F4" class="link link-reveal link-table xref-fig">4</a>b). All the events associated with the previously discussed 1â€“12 and 8â€“11 rearrangements validated in the aN tissue, as did a group of five rearrangements within a 50-Mb region of chromosome 4, suggesting potential early events in cancer progression. Just <em>r</em>(2-2)a, <em>r</em>(2-2)b and <em>r</em>(6-6) yielded no equivalent bands in aN. Sanger sequencing of 12 aN bands of sufficient intensity to sequence without further amplification, confirmed the events (data not shown). The absence of any of these events in the dN, while predicting a potential local field effect phenomenon for this patient, also further eliminated the possibility of WGA artefacts. Although LCM contamination cannot be excluded in these analyses, the range of different banding patterns observed between the GP3, GP4 and aN tissues, specifically the contrasting <em>t</em>(2-23) GP3-specific event, minimizes these concerns.</p>                            <h3 scrollto-destination=10636878 id="10636878" class="section-title" data-legacy-id=s3h>3.8. Validation in non-amplified tissues</h3>
<p>To further confirm that these rearrangements were not artefacts of the WGA process, additional validations on unamplified tissues were performed. Further LCM of the same GP3 and GP4 cell populations was performed and DNA extracted in small volumes of extraction buffer. Due to the limited quantities of DNA extracted, only a limited number of previously optimized validation PCRs could be performed. FigureÂ <a reveal-id="DSS021F4" data-open="DSS021F4" class="link link-reveal link-table xref-fig">4</a>c demonstrates that identical bands were reproduced in the unamplified tissues as were observed in the WGA samples, further indicating the reliability of the methodology.</p>                            <h2 scrollto-destination=10636880 id="10636880" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>High-throughput NGS provides a powerful tool to investigate the molecular changes associated with cancer progression. To confidently identify these changes, it is vital to have methodologies to determine genetic changes specific to tumour populations. Prostate cancer like many cancers is heterogeneous with various degrees of differentiation that are assessed in the Gleason scoring system.<sup><a reveal-id="DSS021C8" data-open="DSS021C8" class="link link-ref link-reveal xref-bibr">8</a></sup> A critical transition for patient prognosis and for dictating therapy is the presence of GP4 or higher.<sup><a reveal-id="DSS021C9" data-open="DSS021C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Men who have tumours composed entirely of GP3 have an excellent outcome regardless of treatment.<sup><a reveal-id="DSS021C10" data-open="DSS021C10" class="link link-ref link-reveal xref-bibr">10</a></sup> In contrast, the presence of GP4 in addition to GP3 has a significant negative impact on survival and raises the possibility that GP3 alone is molecularly different from GP3 associated with GP4. Therefore, to understand prostate cancer progression, stratify patients by risk of progression and to provide an optimal care, it is necessary to understand the molecular changes in GP3 and GP4, and those that drive progression to GP4. The methodology described enables the accurate collection of pure populations of clinical specimens and the subsequent direct WGA of the gDNA.</p><p>The reporting of the prevalence and major role of the <em>E-twenty six (ETS)/TMPRSS2</em> family of genomic rearrangements in prostate cancer has dramatically changed the perception of solid tumour biology.<sup><a reveal-id="DSS021C1" data-open="DSS021C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Elucidation of genomic rearrangements present in solid tumours provides a scaffold to study disease heterogeneity, as well as the relationships between distinct cell populations such as clonality and cancer progression. The NGS MP protocol provides a method to specifically interrogate a gDNA sample for large rearrangements and was thus modified for the input WGA DNA.<sup><a reveal-id="DSS021C5" data-open="DSS021C5" class="link link-ref link-reveal xref-bibr">5</a></sup> The application of this methodology to WGA DNA yielded MP sequencing of similar quality to unamplified gDNA. Specifically, MP sequencing coverage and replication levels for WGA DNA were equivalent to unamplified DNA. However, frequency plots did indicate more focal variation in the WGA MP profiles compared with unamplified DNA, indicating some bias in the regional amplification, which made copy number variation calling problematic with WGA DNA. We are currently investigating complex bioinformatics algorithms aimed at compensating for this natural frequency variation from WGA profiles to enable more confident copy variation number calling. It must also be noted that the prostate cancer case exemplified in this study was performed on the Illumina GAIIx platform and more recent sequencing of WGA MP libraries using the methodology described in this report on the newer Illumina HiSeq platform generated far greater depths of sequencing (Fig.Â <a reveal-id="DSS021F2" data-open="DSS021F2" class="link link-reveal link-table xref-fig">2</a>f and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplemental Table S2</a>).</p><p>In this study, LCM was used to collect pure populations of GP3 and GP4 tumour cells, together with benign-appearing prostatic cells from the same patient. Our novel methodology for WGA and MP sequencing enabled us to accurately define large genomic aberrations using algorithms developed in our laboratory. TableÂ <a reveal-id="DSS021TB1" data-open="DSS021TB1" class="link link-reveal link-table xref-fig">1</a> lists 30 high-confidence genomic rearrangements within the GP3 and GP4 tumour cell populations. The 30 events involved 15 chromosomes, with 20 intra-chromosomal rearrangements and 10 inter-chromosomal translocations. Each event selected for validation was successfully verified by PCR and the fusion junctions determined, emphasizing the validity of the technique and algorithm presented in this study. While many genomic rearrangements were predicted, conservative filtering and knowledge of recurrent false positives additionally observed in normal tissues enabled us to eliminate the majority of false positives. Significantly, the GP3 and GP4 of this patient did not contain the common <em>ERG-TMPRSS2</em> fusion event,<sup><a reveal-id="DSS021C1" data-open="DSS021C1" class="link link-ref link-reveal xref-bibr">1</a></sup> but the validated <em>t</em>(7-17) fusion site lies just 15 kb downstream of the ETS family member; <em>ETV1</em> gene, which is fused at a similar 3â€² distance downstream of the previously unreported ETS family fusion partner <em>ALOX12B</em>. Though no fusion product is predicted, this case was independently demonstrated to have elevated ETV1 expression (data not shown).</p><p>The presence of identical rearrangements within the adjacent Gleason patterns provides evidence of the common origin of GP3 and GP4 in this prostate cancer, indicating the clonal expansion of GP3 with progression to GP4, a critical step in disease progression with very important clinical implications. While we cannot extrapolate this single patient observation to prostate cancer in general, this report describes powerful methodology to address this question in multiple prostate tumour samples, which is currently underway in our laboratory. Additionally, while these data may provide clues to potential GP3 to GP4 progression in the ratio of MP numbers and the intensity of PCR validation bands, care must be taken considering the input of WGA amplified DNA in the studies. More extensive and direct experimentation is required on unamplified tissues to answer these questions, which was outside the scope of this study. Our findings also suggest the presence of a significant prostate cancer field effect in this patient with 13 of 16 validated events, also present in the aN tissues but not dN prostate samples (Fig.Â <a reveal-id="DSS021F4" data-open="DSS021F4" class="link link-reveal link-table xref-fig">4</a>b). While great care was taken in the LCM sampling, contamination cannot be ruled out. However, the varying levels of PCR banding observed for the GP3, GP4 and aN validations (Fig.Â <a reveal-id="DSS021F4" data-open="DSS021F4" class="link link-reveal link-table xref-fig">4</a>a and b) does not reflect a level of continuity expected from a constant level of contamination in each sample.</p><p>The main difference of our procedure from previous studies relates to handling and amplification of DNA from small numbers of captured cells. To circumvent limitations related to low DNA yields and quality resulting from LCM of a few hundred cells, we developed an <em>in situ</em> whole-genome DNA amplification procedure to bypass the DNA extraction steps. As described, this methodology produced a good quality DNA resulting in representative MP sequencing data spanning the whole genome of tumour cells. While MP sequencing of WGA DNA did result in significant increases in uneven profiles across the genome, it still provided adequate depth of sequencing to reveal a large set of rearrangements that were validated by independent experimental methods. We conclude that the ability to genetically characterize pure populations of histologically significant lesions far outweighs these limitations introduced by WGA of the DNA.</p><p>While the successful application of WGA to the MP protocol was demonstrated in this report, special attention must be made in both library assembly and bioinformatics analysis. The reduced size of the input WGA (âˆ¼15 kb) compared with unamplified DNA (âˆ¼50 kb) required specific modifications to the Illumina MP fragmentation protocols. Over fragmentation of WGA DNA using protocols designed for normal gDNA was observed to compromise the final sequencing data, resulting in reduced MP data coverage. While the bioinformatics algorithm applied to WGA and gDNA was identical, WGA DNA was observed to result in increased background noise in the MP data. A summary of MP data from eight cases each of WGA and unamplified gDNA are presented in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>. Large numbers of false-positive discordant MP reads were clearly a factor for both DNA inputs due to limitations in the Illumina MP library production protocols. The percentage of discordant MPs that passed initial filters, were as expected slightly greater with WGA DNA. This elevated false-positive fusion frequency potentially arises from artefacts of random re-priming of the 3â€² termini of displaced amplification fragments to non-specific sites. While significant in the output MP data, they predominantly present as single events with no associates and the majority are removed in the final filtering steps requiring three associated MP reads to pass filter. An indication of the false positive rates can thus be reflected in the number of observed MPs that present with no associates. As expected, this population is extensive for both groups, with 83% of discordant MP having no associates for the gDNA before initial filtering. This figure was almost 10% higher with WGA DNA (91%) reflecting a larger proportion of false positives stemming from the amplification protocol. However, upon filtering the number of unique discordant MPs from the two populations is similar, reflective of the effectiveness of the bioinformatics algorithms applied to eliminate these false positives. In conclusion, while an increased number of false positives are predicted with WGA, the presence of these events as unique random events enables them to be easily eliminated upon bioinformatics filtering. However, care must always be taken in interpreting potential fusion events in MP data and PCR validation is generally required to truly validate events. Nevertheless, the robustness of the algorithm is emphasized in the level of validation observed in this study. Conversely, a significant number of genomic breakpoints may be missed due to reduced depths of sequencing or underrepresentation in the final library preparations. However, defining the level of false-negative calls in the data is far more complex and would require absolute knowledge of all the translocations present in a case to be assessed.</p><p>The Illumina MP protocol enables the sequencing of 100 nucleotides from the termini of larger spanning (2â€“5 kb) DNA fragments compared with conventional PE libraries (0.2â€“0.5 kb). This extended DNA span increases the probability of spanning a genomic breakpoint. The â€˜bridged-coverageâ€™ of &gt;30X with a half lane of a HiSeq2000 data is highly effective at detecting these breakpoints. However, with â€˜base-coverageâ€™ in the range of 4â€“5X, the ability to call SNVs is limited. The reduced size of DNA yielded from formalin fixed paraffin embedded (FFPE) tissues (generally &lt;500 bp) also precludes its application to the MP protocol and most WGA techniques. Interrogation of large genomic rearrangements in DNA from FFPE tissues currently requires whole genome sequencing of PE libraries on several lanes of an Illumina HiSeq2000 flow cell in order to attain high enough sequencing depths to confidently call fusion breakpoints. While more robust methods are now available for WGA from FFPE tissues, these methodologies target the smaller species of DNA present in these tissues, generally &lt;500 bp which still precludes their application to current MP protocols.</p><p>In summary, this study shows the methodology of interrogating small pure cell populations by NGS technologies using LCM coupled with WGA and a modified MP library assembly protocol. The WGA DNA generated in this study is of sufficient quality and quantity for application to other next generation protocols, including whole-genome and exome capture sequencing protocols (data not shown and S.J. Murphy <em>et al.</em>, submitted). Sampling and sequencing of distinct tumour cell populations as well as histologically normal cell populations allows the characterization of genetic changes occurring between these cell populations providing insight into cancer development and progression. The results described for the prostate tissue exemplified in this study provide major clues to the origin of this patient's cancer. Current experimentation in our laboratory is using this methodology to sequence large panels of adjacent tumours and histologically normal tissues from a variety of cancer types to identify large genomic translocations suitable for clinical targeting.</p>                            <h2 scrollto-destination=10636889 id="10636889" class="section-title" data-legacy-id=s5>Contributions</h2>
<p>S.J.M., J.C.C., F.K., R.J.K. and G.V. planned the project, S.J.M., J.C.C., S.Z., R.A.S. and B.W.E. carried out the experiments. S.J.M., S.H.J. and G.V. analyzed the data and S.J.M., J.C.C. and G.V. wrote the manuscript. All authors edited and commented on the manuscript.</p>                            <h2 scrollto-destination=10636891 id="10636891" class="section-title" data-legacy-id=s6>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at</a><a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=10636893 id="10636893" class="section-title" data-legacy-id=s7>Funding</h2>
<p>This work was supported in part by a Waterman Biomarker Discovery grant from the Mayo Clinic Center for Individualized Medicine.</p>                            <h2 scrollto-destination=10636895 id="10636895" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSS021C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS021C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rubin</div> <div class="given-names">M.A.</div></div>,Â  <div class="name"><div class="surname">Maher</div> <div class="given-names">C.A.</div></div>,Â  <div class="name"><div class="surname">Chinnaiyan</div> <div class="given-names">A.M.</div></div>.Â </div><div class="article-title">Common gene rearrangements in prostate cancer</div>,Â <div class="source">J. Clin. Oncol.</div>Â ,Â <div class="year">2012</div>,Â vol.Â <div class="volume">30</div>Â (pg.Â <div class="fpage">608</div>-<div class="lpage">15</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1200/JCO.2011.38.9650" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1200/JCO.2011.38.9650</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Common%20gene%20rearrangements%20in%20prostate%20cancer&amp;author=M.A.%20Rubin&amp;author=C.A.%20Maher&amp;author=A.M.%20Chinnaiyan&amp;publication_year=2012&amp;journal=J.%20Clin.%20Oncol.&amp;volume=30&amp;pages=608-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1200/JCO.2011.38.9650" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1200%2FJCO.2011.38.9650" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1200%2FJCO.2011.38.9650"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22253466" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS021C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS021C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Niitsu</div> <div class="given-names">N.</div></div>,Â  <div class="name"><div class="surname">Okamoto</div> <div class="given-names">M.</div></div>,Â  <div class="name"><div class="surname">Miura</div> <div class="given-names">I.</div></div>,Â  <div class="name"><div class="surname">Hirano</div> <div class="given-names">M.</div></div>.Â </div><div class="article-title">Clinical features and prognosis of de novo diffuse large B-cell lymphoma with t(14;18) and 8q24/c-MYC translocations</div>,Â <div class="source">Leukemia</div>Â ,Â <div class="year">2009</div>,Â vol.Â <div class="volume">23</div>Â (pg.Â <div class="fpage">777</div>-<div class="lpage">83</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/leu.2008.344" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/leu.2008.344</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Clinical%20features%20and%20prognosis%20of%20de%20novo%20diffuse%20large%20B-cell%20lymphoma%20with%20t(14%3B18)%20and%208q24%2Fc-MYC%20translocations&amp;author=N.%20Niitsu&amp;author=M.%20Okamoto&amp;author=I.%20Miura&amp;author=M.%20Hirano&amp;publication_year=2009&amp;journal=Leukemia&amp;volume=23&amp;pages=777-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/leu.2008.344" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fleu.2008.344" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fleu.2008.344"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19151788" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS021C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS021C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Burke</div> <div class="given-names">B.A.</div></div>,Â  <div class="name"><div class="surname">Carroll</div> <div class="given-names">M.</div></div>.Â </div><div class="article-title">BCL-ABL; a multifaceted promoter of DNA mutation in chronic myelogenous leukemia</div>,Â <div class="source">Leukemia</div>Â ,Â <div class="year">2010</div>,Â vol.Â <div class="volume">24</div>Â (pg.Â <div class="fpage">1105</div>-<div class="lpage">12</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/leu.2010.67" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/leu.2010.67</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BCL-ABL%3B%20a%20multifaceted%20promoter%20of%20DNA%20mutation%20in%20chronic%20myelogenous%20leukemia&amp;author=B.A.%20Burke&amp;author=M.%20Carroll&amp;publication_year=2010&amp;journal=Leukemia&amp;volume=24&amp;pages=1105-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/leu.2010.67" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fleu.2010.67" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fleu.2010.67"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20445577" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS021C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS021C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lin</div> <div class="given-names">P.</div></div>,Â  <div class="name"><div class="surname">Jetly</div> <div class="given-names">R.</div></div>,Â  <div class="name"><div class="surname">Lennon</div> <div class="given-names">P.A.</div></div>,Â et al.Â </div><div class="article-title">Translocation (18;22) (q21;q11) in B-cell lymphomas: a report of 4 cases and review of the literature</div>,Â <div class="source">Hum. Pathol.</div>Â ,Â <div class="year">2008</div>,Â vol.Â <div class="volume">39</div>Â (pg.Â <div class="fpage">1664</div>-<div class="lpage">72</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.humpath.2008.04.007" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.humpath.2008.04.007</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translocation%20(18%3B22)%20(q21%3Bq11)%20in%20B-cell%20lymphomas%3A%20a%20report%20of%204%20cases%20and%20review%20of%20the%20literature&amp;author=P.%20Lin&amp;author=R.%20Jetly&amp;author=P.A.%20Lennon&amp;publication_year=2008&amp;journal=Hum.%20Pathol.&amp;volume=39&amp;pages=1664-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.humpath.2008.04.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.humpath.2008.04.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.humpath.2008.04.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18656237" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS021C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS021C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="collab">Illumina</div><div class="article-title">Mate Pair Library v2 Sample Preparation Guide for 2â€“5 kb Libraries</div>,Â <div class="year">2009</div>Â <div class="comment">http://nextgen.mgh.harvard.edu/attachments/MatePair_v2_2-5kb_SamplePrep_Guide_15008135_A.pdf</div></div></div></div></div><div content-id="DSS021C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS021C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Feldman</div> <div class="given-names">A.L.</div></div>,Â  <div class="name"><div class="surname">Dogan</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Smith</div> <div class="given-names">D.I.</div></div>,Â et al.Â </div><div class="article-title">Discovery of recurrent t(6;7)(p25.3;q32.3) translocations in ALK-negative anaplastic large cell lymphomas by massively parallel genomic sequencing</div>,Â <div class="source">Blood</div>Â ,Â <div class="year">2011</div>,Â vol.Â <div class="volume">117</div>Â (pg.Â <div class="fpage">915</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1182/blood-2010-08-303305" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1182/blood-2010-08-303305</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20of%20recurrent%20t(6%3B7)(p25.3%3Bq32.3)%20translocations%20in%20ALK-negative%20anaplastic%20large%20cell%20lymphomas%20by%20massively%20parallel%20genomic%20sequencing&amp;author=A.L.%20Feldman&amp;author=A.%20Dogan&amp;author=D.I.%20Smith&amp;publication_year=2011&amp;journal=Blood&amp;volume=117&amp;pages=915-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1182/blood-2010-08-303305" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1182%2Fblood-2010-08-303305" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1182%2Fblood-2010-08-303305"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21030553" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS021C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS021C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vasmatzis</div> <div class="given-names">G.</div></div>,Â  <div class="name"><div class="surname">Johnson</div> <div class="given-names">S.H.</div></div>,Â  <div class="name"><div class="surname">Knudson</div> <div class="given-names">R.A.</div></div>,Â et al.Â </div><div class="article-title">Genome-wide analysis reveals recurrent structural abnormalities of TP63 and other p53-related genes in peritheral T-cell lymphomas</div>,Â <div class="source">Blood</div>Â ,Â <div class="year">2012</div>Â <div class="comment">2012 Aug 1. [Epub ahead of print] PMID: 22855598</div></div></div></div></div><div content-id="DSS021C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS021C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Epstein</div> <div class="given-names">J.I.</div></div>.Â </div><div class="article-title">An update of the Gleason grading system</div>,Â <div class="source">J. Urol.</div>Â ,Â <div class="year">2010</div>,Â vol.Â <div class="volume">183</div>Â (pg.Â <div class="fpage">433</div>-<div class="lpage">40</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.juro.2009.10.046" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.juro.2009.10.046</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20update%20of%20the%20Gleason%20grading%20system&amp;author=J.I.%20Epstein&amp;publication_year=2010&amp;journal=J.%20Urol.&amp;volume=183&amp;pages=433-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.juro.2009.10.046" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.juro.2009.10.046" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.juro.2009.10.046"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20006878" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS021C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS021C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Amin</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Partin</div> <div class="given-names">A.</div></div>,Â  <div class="name"><div class="surname">Epstein</div> <div class="given-names">J.L.</div></div>.Â </div><div class="article-title">Gleason score 7 prostate cancer on needle biopsy: relation of primary pattern 3 or 4 to pathological stage and progression after radical prostatectomy</div>,Â <div class="source">J. Urol.</div>Â ,Â <div class="year">2011</div>,Â vol.Â <div class="volume">186</div>Â (pg.Â <div class="fpage">1286</div>-<div class="lpage">90</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.juro.2011.05.075" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.juro.2011.05.075</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gleason%20score%207%20prostate%20cancer%20on%20needle%20biopsy%3A%20relation%20of%20primary%20pattern%203%20or%204%20to%20pathological%20stage%20and%20progression%20after%20radical%20prostatectomy&amp;author=A.%20Amin&amp;author=A.%20Partin&amp;author=J.L.%20Epstein&amp;publication_year=2011&amp;journal=J.%20Urol.&amp;volume=186&amp;pages=1286-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.juro.2011.05.075" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.juro.2011.05.075" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.juro.2011.05.075"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21862072" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div><div content-id="DSS021C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS021C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stephenson</div> <div class="given-names">A.J.</div></div>,Â  <div class="name"><div class="surname">Kattan</div> <div class="given-names">M.W.</div></div>,Â  <div class="name"><div class="surname">Eastham</div> <div class="given-names">J.A.</div></div>,Â et al.Â </div><div class="article-title">Prostate cancer-specific mortality after radical prostatectomy for patients treated in the prostate-specific antigen era</div>,Â <div class="source">J. Clin. Oncol.</div>Â ,Â <div class="year">2009</div>,Â vol.Â <div class="volume">27</div>Â (pg.Â <div class="fpage">4300</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1200/JCO.2008.18.2501" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1200/JCO.2008.18.2501</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prostate%20cancer-specific%20mortality%20after%20radical%20prostatectomy%20for%20patients%20treated%20in%20the%20prostate-specific%20antigen%20era&amp;author=A.J.%20Stephenson&amp;author=M.W.%20Kattan&amp;author=J.A.%20Eastham&amp;publication_year=2009&amp;journal=J.%20Clin.%20Oncol.&amp;volume=27&amp;pages=4300-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1200/JCO.2008.18.2501" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1200%2FJCO.2008.18.2501" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1200%2FJCO.2008.18.2501"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19636023" class="link link-pub-id" target="_blank">PubMed</a></div>Â </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Masahira Hattori</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">Â© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=629019&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss021/2/dss021-dss021supp_Table2.pdf?Expires=2147483647&amp;Signature=tqfzuZbA0wqB5U-5iURzYGgMI54srApYuR7Mz6zSKB0ZpM8CqhBn0IiqM8WoTtBpRAEzTjARRRBiUzKQyoglyNmyC5S5NB3WtQtcQsOztFmxQu7UOzVa1Pyrc~LYgbbHrW8NEsJvmPR8vLoNvV9eFPkupmOMhLiYAMoQ0VK~OZlmVMCD-djxgI~f9Dz1NWH2ohzJLRGU1AJVgNe48ccVQWnVFj~GEKfMYbkUHnh9gF8~w5Jg3WhD7Ld9QCXF3bQoqvgdbQIIboIG4Mu57b~znPcxuosgUzsaWC-5tMLnuLRhLbdmf5rdnQWw6e6DIuCO1cEwh1ZIiEW~OvALVQjLBg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/5/10.1093/dnares/dss021/2/dss021-dss021supp_Table1.pdf?Expires=2147483647&amp;Signature=PUSyN7FGO~ISeJDVwGH6it4v6uw1hIlfrEeu4ll8FX6SmIMpQnjdDvfSAx3wOfjRucLsIGX6NtM3swcw9MIgD5M7Y7AzwfGJgCCs-jvB52mjnFb~v04ooAUymPVoRgg8-wnUMbLEpk0ER7JRsC6hLg87M~QqB1frakkhaUWePLEjf-9kySM-LxEhOK1pncTotgoyyp0ZF1G0O78xBdHP9U40kqdksGfvGb3M2l25IkhZwTvuVbBxh-0TzgIaiFGi-isKN1uQeWURJ5eIrpUB12VpVJb6vDAgMafQCgY-6mF2LNKZHK3pkzS6eQJPt8l4M3AhRTTdi03ImVJWzQEU-A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">385</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000310822200004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">37</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dss021"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="629019"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000310822200004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f5%2f395%2f629019">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000310822200004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (37)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F5%2F395%2F629019" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/629019'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transpositionâ€“duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Mate Pair Sequencing of Whole-Genome-Amplified DNA Following Laser Capture Microdissection of Prostate Cancer" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dss021");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f5%2f395%2f629019">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

