

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/4/361/4939498","@type":"ScholarlyArticle","name":"Unravelling the epigenomic interactions between parental inbreds resulting in an altered hybrid methylome in pigeonpea","datePublished":"2018-03-16","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2018-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy008","keywords":["DNA methylation","TcM","TcDM","pigeonpea","hybrid"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/4/361/4939498","author":[{"name":"Junaid, Alim","affiliation":"National Research Centre on Plant Biotechnology, LBS Centre, PUSA Campus, New Delhi, India","@type":"Person"},{"name":"Kumar, Himanshu","affiliation":"Indian Agricultural Statistics Research Institute, New Delhi, India","@type":"Person"},{"name":"Rao, A R","affiliation":"Indian Agricultural Statistics Research Institute, New Delhi, India","@type":"Person"},{"name":"Patil, A N","affiliation":"Pulse Reaserch Unit, Dr. Panjabrao Deshmukh Krishi Vidyapeeth, P.O. Krishi Nagar, Akola, Maharashtra, India","@type":"Person"},{"name":"Singh, N K","affiliation":"National Research Centre on Plant Biotechnology, LBS Centre, PUSA Campus, New Delhi, India","@type":"Person"},{"name":"Gaikwad, Kishor","affiliation":"National Research Centre on Plant Biotechnology, LBS Centre, PUSA Campus, New Delhi, India","@type":"Person"}],"description":"Abstract.  DNA methylation is an important heritable landmark conferring epigenetic changes in hybrids and has fascinated biologists and plant-breeders over the","pageStart":"361","pageEnd":"373","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/4/1/m_dnares_25_4cover.png?Expires=2147483647&Signature=lD6dhAB99HpTltGbT4-zO-h9Z6msKpBpjdlZuCIFAdO0RvvEM~IgMElNp5bE6Rdf~OSfdWzB2y8SIAkqvb6Fc-0GRLo0HVHIHIXLjJF8ZJ94ZrTCEDv2Gn-92BvjK~Ebx3LV-ePsMx8j4josbKy1FscEJ3-qj3zB2TEHnK~iAQqommn5aOw82hhC1dOH5f1P15MUh9o1QIOhJ9WAnJwCFYDxRE0ZmdX~qCpKaFm8XPRL-O71Rxc1eeV8iTb9yQzgpbw2OwX1g4mQ5iALY~7Bk3L2RgUkQZIzgNqb8ANqfHjAsIlrzFHcKAl7tTiKtoGbsHvDqkXNPcKjgjrdKdIXQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Unravelling the epigenomic interactions between parental inbreds resulting in an altered hybrid methylome in pigeonpea","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/4/1/m_dnares_25_4cover.png?Expires=2147483647&Signature=lD6dhAB99HpTltGbT4-zO-h9Z6msKpBpjdlZuCIFAdO0RvvEM~IgMElNp5bE6Rdf~OSfdWzB2y8SIAkqvb6Fc-0GRLo0HVHIHIXLjJF8ZJ94ZrTCEDv2Gn-92BvjK~Ebx3LV-ePsMx8j4josbKy1FscEJ3-qj3zB2TEHnK~iAQqommn5aOw82hhC1dOH5f1P15MUh9o1QIOhJ9WAnJwCFYDxRE0ZmdX~qCpKaFm8XPRL-O71Rxc1eeV8iTb9yQzgpbw2OwX1g4mQ5iALY~7Bk3L2RgUkQZIzgNqb8ANqfHjAsIlrzFHcKAl7tTiKtoGbsHvDqkXNPcKjgjrdKdIXQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Unravelling the epigenomic interactions between parental inbreds resulting in an altered hybrid methylome in pigeonpea" />
    <meta property="og:description" content="Abstract.  DNA methylation is an important heritable landmark conferring epigenetic changes in hybrids and has fascinated biologists and plant-breeders over the" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/4/361/4939498" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2018-03-16" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Junaid, Alim" /><meta name="citation_author_institution" content="National Research Centre on Plant Biotechnology, LBS Centre, PUSA Campus, New Delhi, India" /><meta name="citation_author" content="Kumar, Himanshu" /><meta name="citation_author_institution" content="Indian Agricultural Statistics Research Institute, New Delhi, India" /><meta name="citation_author" content="Rao, A R" /><meta name="citation_author_institution" content="Indian Agricultural Statistics Research Institute, New Delhi, India" /><meta name="citation_author" content="Patil, A N" /><meta name="citation_author_institution" content="Pulse Reaserch Unit, Dr. Panjabrao Deshmukh Krishi Vidyapeeth, P.O. Krishi Nagar, Akola, Maharashtra, India" /><meta name="citation_author" content="Singh, N K" /><meta name="citation_author_institution" content="National Research Centre on Plant Biotechnology, LBS Centre, PUSA Campus, New Delhi, India" /><meta name="citation_author" content="Gaikwad, Kishor" /><meta name="citation_author_institution" content="National Research Centre on Plant Biotechnology, LBS Centre, PUSA Campus, New Delhi, India" /><meta name="citation_title" content="Unravelling the epigenomic interactions between parental inbreds resulting in an altered hybrid methylome in pigeonpea" /><meta name="citation_firstpage" content="361" /><meta name="citation_lastpage" content="373" /><meta name="citation_doi" content="10.1093/dnares/dsy008" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2018/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Epigenetic inheritance in plants;  citation_journal_title=Nature;  citation_year=2007;  citation_volume=447;  citation_pages=418-24; " /><meta name="citation_reference" content="citation_title=Covering your bases: inheritance of DNA methylation in plant genomes;  citation_journal_title=Mol. Plant;  citation_year=2014;  citation_volume=7;  citation_pages=472-80; " /><meta name="citation_reference" content="citation_title=DNA methylation and human disease;  citation_journal_title=Nat. Rev. Genet;  citation_year=2005;  citation_volume=6;  citation_pages=597-610; " /><meta name="citation_reference" content="citation_title=DNA methylation landscapes: provocative insights from epigenomics;  citation_journal_title=Nat Rev Genet;  citation_year=2008;  citation_volume=9;  citation_pages=465-76; " /><meta name="citation_reference" content="citation_title=Perceptions of epigenetics;  citation_journal_title=Nature;  citation_year=2007;  citation_volume=447;  citation_pages=396-8; " /><meta name="citation_reference" content="citation_title=Establishing, maintaining and modifying DNA methylation patterns in plants and animals;  citation_journal_title=Nat. Rev. Genet;  citation_year=2010;  citation_volume=11;  citation_pages=204-20; " /><meta name="citation_reference" content="citation_title=Arabidopsis thaliana DNA methylation mutants;  citation_journal_title=Science;  citation_year=1993;  citation_volume=260;  citation_pages=1926-8; " /><meta name="citation_reference" content="citation_title=Arabidopsis MET1 cytosine methyltransferase mutants;  citation_journal_title=Genetics;  citation_year=2003;  citation_volume=163;  citation_pages=1109-22; " /><meta name="citation_reference" content="citation_title=Genome-wide evolutionary analysis of eukaryotic DNA methylation;  citation_journal_title=Science;  citation_year=2010;  citation_volume=328;  citation_pages=916-9; " /><meta name="citation_reference" content="citation_title=Role of the arabidopsis DRM methyltransferases in de novo DNA methylation and gene silencing;  citation_journal_title=Curr. Biol;  citation_year=2002;  citation_volume=12;  citation_pages=1138-44; " /><meta name="citation_reference" content="citation_title=siRNAs and DNA methylation: seedy epigenetics;  citation_journal_title=Trends Plant Sci;  citation_year=2010;  citation_volume=15;  citation_pages=204-10; " /><meta name="citation_reference" content="citation_title=Non-CG methylation patterns shape the epigenetic landscape in Arabidopsis;  citation_journal_title=Nat. Struct. Mol. Biol;  citation_year=2014;  citation_volume=21;  citation_pages=64-72; " /><meta name="citation_reference" content="Reed H. S.  1942 , A short history of the plant sciences. https://www.scribd.com/document/190746815/Reed-Howard-S-1942-a-Short-History-of-the-Plant-Sciences. " /><meta name="citation_reference" content="citation_title=Long noncoding RNAs in development and disease of the central nervous system;  citation_journal_title=Trends Genet;  citation_year=2013;  citation_volume=29;  citation_pages=461-8; " /><meta name="citation_reference" content="citation_title=Trans chromosomal methylation in Arabidopsis hybrids;  citation_journal_title=Proc. Natl. Acad. Sci. US. A;  citation_year=2012;  citation_volume=109;  citation_pages=3570-5; " /><meta name="citation_reference" content="citation_title=Inheritance of trans chromosomal methylation patterns from Arabidopsis F1 hybrids;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2014;  citation_volume=111;  citation_pages=2017-22; " /><meta name="citation_reference" content="citation_title=Epigenetic changes in hybrids;  citation_journal_title=Plant Physiol;  citation_year=2015;  citation_volume=168;  citation_pages=1197-205; " /><meta name="citation_reference" content="citation_title=Epigenetic natural variation in Arabidopsis thaliana;  citation_journal_title=PLoS Biol;  citation_year=2007;  citation_volume=5;  citation_pages=e174" /><meta name="citation_reference" content="citation_title=Patterns of population epigenomic diversity;  citation_journal_title=Nature;  citation_year=2013;  citation_volume=495;  citation_pages=193-8; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of DNA methylation and gene expression changes in two Arabidopsis ecotypes and their reciprocal hybrids;  citation_journal_title=Plant Cell;  citation_year=2012;  citation_volume=24;  citation_pages=875-92; " /><meta name="citation_reference" content="citation_title=The inheritance pattern of 24 nt siRNA clusters in arabidopsis hybrids is influenced by proximity to transposable elements;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e47043" /><meta name="citation_reference" content="citation_title=Changes in 24-nt siRNA levels in Arabidopsis hybrids suggest an epigenetic contribution to hybrid vigor;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2011;  citation_volume=108;  citation_pages=2617-22; " /><meta name="citation_reference" content="citation_title=Global epigenetic and transcriptional trends among two rice subspecies and their reciprocal hybrids;  citation_journal_title=Plant Cell;  citation_year=2010;  citation_volume=22;  citation_pages=17-33; " /><meta name="citation_reference" content="citation_title=Repeat associated small RNAs vary among parents and following hybridization in maize;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2012;  citation_volume=109;  citation_pages=10444-9; " /><meta name="citation_reference" content="citation_title=Transcriptome and methylome profiling reveals relics of genome dominance in the mesopolyploid Brassica oleracea;  citation_journal_title=Genome Biol;  citation_year=2014;  citation_volume=15;  citation_pages=R77" /><meta name="citation_reference" content="citation_title=Transcriptome and methylome interactions in rice hybrids;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2012;  citation_volume=109;  citation_pages=12040-5; " /><meta name="citation_reference" content="citation_title=The first draft of the pigeonpea genome sequence;  citation_journal_title=J. Plant Biochem. Biotechnol;  citation_year=2012;  citation_volume=21;  citation_pages=98-112; " /><meta name="citation_reference" content="citation_title=Low natural outcrossing in ′cleistogamous′ pigeonpea mutant;  citation_journal_title=Int. Pigeonpea Newslett;  citation_year=1990;  citation_volume=11;  citation_pages=9-10; " /><meta name="citation_reference" content="citation_title=Male sterility and fertility restoration in crops;  citation_journal_title=Annu. Rev. Plant Biol;  citation_year=2014;  citation_volume=65;  citation_pages=579-606; " /><meta name="citation_reference" content="citation_title=Cytoplasmic male sterility-associated chimeric open reading frames identified by mitochondrial genome sequencing of four Cajanus genotypes;  citation_journal_title=DNA Res;  citation_year=2013;  citation_volume=20;  citation_pages=485-95; " /><meta name="citation_reference" content="citation_title=Genetics of fertility restoration in A2 cytoplasm based hybrids of pigeonpea;  citation_journal_title=Int. J. Cur. Microbiol. Appl. Sci;  citation_year=2018; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of pigeonpea (Cajanus cajan), an orphan legume crop of resource-poor farmers;  citation_journal_title=Nat. Biotechnol;  citation_year=2011;  citation_volume=30;  citation_pages=83-9; " /><meta name="citation_reference" content="citation_title=BSMAP: whole genome bisulfite sequence MAPping program;  citation_journal_title=BMC Bioinformatics;  citation_year=2009;  citation_volume=10;  citation_pages=232" /><meta name="citation_reference" content="citation_title=′Leveling′ the playing field for analyses of single-base resolution DNA methylomes;  citation_journal_title=Trends Genet;  citation_year=2012;  citation_volume=28;  citation_pages=583-5; " /><meta name="citation_reference" content="citation_title=methylKit: a comprehensive R package for the analysis of genome-wide DNA methylation profiles;  citation_journal_title=Genome Bio.l;  citation_year=2012;  citation_volume=13;  citation_pages=R87" /><meta name="citation_reference" content="citation_title=MethPrimer: designing primers for methylation PCRs;  citation_journal_title=Bioinformatics;  citation_year=2002;  citation_volume=18;  citation_pages=1427-31; " /><meta name="citation_reference" content="citation_title=QUMA: quantification tool for methylation analysis;  citation_journal_title=Nucleic Acids Res;  citation_year=2008;  citation_volume=36;  citation_pages=W170-5; " /><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=357-9; " /><meta name="citation_reference" content="citation_title=RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome;  citation_journal_title=BMC Bioinformatics;  citation_year=2011;  citation_volume=12;  citation_pages=323" /><meta name="citation_reference" content="citation_title=edgeR: a bioconductor package for differential expression analysis of digital gene expression data;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=139-40; " /><meta name="citation_reference" content="citation_title=WEGO: a web tool for plotting GO annotations;  citation_journal_title=Nucleic Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=W293-7; " /><meta name="citation_reference" content="citation_title=Bis-SNP: combined DNA methylation and SNP calling for Bisulfite-seq data;  citation_journal_title=Genome Biol;  citation_year=2012;  citation_volume=13;  citation_pages=R61" /><meta name="citation_reference" content="citation_title=Conservation and divergence of methylation patterning in plants and animals;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2010;  citation_volume=107;  citation_pages=8689-94; " /><meta name="citation_reference" content="citation_title=Epigenetic regulation of transposable elements in plants;  citation_journal_title=Annu. Rev. Plant Biol;  citation_year=2009;  citation_volume=60;  citation_pages=43-66; " /><meta name="citation_reference" content="citation_title=Shotgun bisulphite sequencing of the Arabidopsis genome reveals DNA methylation patterning;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=452;  citation_pages=215" /><meta name="citation_reference" content="citation_title=Identification of a novel salt tolerance gene in wild soybean by whole-genome sequencing;  citation_journal_title=Nat. Commun;  citation_year=2014;  citation_volume=5;  citation_pages=4340" /><meta name="citation_reference" content="citation_title=Analysis of the genome sequence of the flowering plant Arabidopsis thaliana; citation_author=The Arabidopsis Genome, I;  citation_journal_title=Nature;  citation_year=2000;  citation_volume=408;  citation_pages=796" /><meta name="citation_reference" content="citation_title=Autotetraploid rice methylome analysis reveals methylation variation of transposable elements and their effects on gene expression;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2015;  citation_volume=112;  citation_pages=E7022-9; " /><meta name="citation_reference" content="citation_title=Single-base resolution maps of cultivated and wild rice methylomes and regulatory roles of DNA methylation in plant gene expression;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=300" /><meta name="citation_reference" content="citation_title=Variation and patterns of DNA methylation in maize C-type CMS lines and their maintainers;  citation_journal_title=J. Plant Biochem. Biotechnol;  citation_year=2010;  citation_volume=19;  citation_pages=43-50; " /><meta name="citation_reference" content="citation_title=DNA methylation changes in photoperiod-thermo-sensitive male sterile rice PA64S under two different conditions;  citation_journal_title=Gene;  citation_year=2014;  citation_volume=537;  citation_pages=143-8; " /><meta name="citation_reference" content="citation_title=Transgenerational epigenetic instability is a source of novel methylation variants;  citation_journal_title=Science;  citation_year=2011;  citation_volume=334;  citation_pages=369-73; " /><meta name="citation_reference" content="citation_title=Spontaneous epigenetic variation in the Arabidopsis thaliana methylome;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=480;  citation_pages=245-9; " /><meta name="citation_reference" content="citation_title=Epigenetic and genetic influences on DNA methylation variation in maize populations;  citation_journal_title=Plant Cell;  citation_year=2013;  citation_volume=25;  citation_pages=2783-97; " /><meta name="citation_reference" content="citation_title=Mitochondrial aldehyde dehydrogenase activity is required for male fertility in maize;  citation_journal_title=Plant Cell;  citation_year=2001;  citation_volume=13;  citation_pages=1063-78; " /><meta name="citation_reference" content="citation_title=Pentatricopeptide-repeat family protein RF6 functions with hexokinase 6 to rescue rice cytoplasmic male sterility;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2015;  citation_volume=112;  citation_pages=14984-9; " /><meta name="citation_reference" content="citation_title=Dynamic roles for small RNAs and DNA methylation during ovule and fiber development in allotetraploid cotton;  citation_journal_title=PLoS Genet;  citation_year=2015;  citation_volume=11;  citation_pages=e1005724" /><meta name="citation_reference" content="citation_title=Genomic DNA methylation analyses reveal the distinct profiles in castor bean seeds with persistent endosperms;  citation_journal_title=Plant Physiol;  citation_year=2016;  citation_volume=171;  citation_pages=1242-58; " /><meta name="citation_reference" content="citation_title=Genome-wide epigenetic regulation of gene transcription in maize seeds;  citation_journal_title=PLoS One;  citation_year=2015;  citation_volume=10;  citation_pages=e0139582" /><meta name="citation_reference" content="citation_title=Divergent DNA methylation patterns associated with gene expression in rice cultivars with contrasting drought and salinity stress response;  citation_journal_title=Sci. Rep;  citation_year=2015;  citation_volume=5;  citation_pages=14922" /><meta name="citation_reference" content="citation_title=Genome-wide analysis of DNA methylation in soybean;  citation_journal_title=Mol. Plant;  citation_year=2013;  citation_volume=6;  citation_pages=1961-74; " /><meta name="citation_reference" content="citation_title=RNA-mediated chromatin-based silencing in plants;  citation_journal_title=Curr. Opin. Cell Biol;  citation_year=2009;  citation_volume=21;  citation_pages=367-76; " /><meta name="citation_reference" content="citation_title=What triggers differential DNA methylation of genes and TEs: contribution of body methylation?;  citation_journal_title=Cold Spring Harb. Symp. Quant. Biol;  citation_year=2012;  citation_volume=77;  citation_pages=155-60; " /><meta name="citation_reference" content="citation_title=DNA methylation of retrotransposons, DNA transposons and genes in sugar beet (Beta vulgaris L.);  citation_journal_title=Plant J;  citation_year=2017;  citation_volume=90;  citation_pages=1156-75; " /><meta name="citation_reference" content="citation_title=Genomic distribution of H3K9me2 and DNA methylation in a maize genome;  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e105267" /><meta name="citation_reference" content="citation_title=CHH islands: de novo DNA methylation in near-gene chromatin regulation in maize;  citation_journal_title=Genome Res;  citation_year=2013;  citation_volume=23;  citation_pages=628-37; " /><meta name="citation_reference" content="citation_title=Somatically heritable switches in the DNA modification of Mu transposable elements monitored with a suppressible mutant in maize;  citation_journal_title=Genes Dev;  citation_year=1990;  citation_volume=4;  citation_pages=331-43; " /><meta name="citation_reference" content="citation_title=Epigenetic regulation of rice flowering and reproduction;  citation_journal_title=Front. Plant Sci;  citation_year=2014;  citation_volume=5;  citation_pages=803" /><meta name="citation_reference" content="citation_title=DDM1 (decrease in DNA methylation) genes in rice (Oryza sativa);  citation_journal_title=Mol. Genet. Genomics;  citation_year=2012;  citation_volume=287;  citation_pages=785-92; " /><meta name="citation_reference" content="citation_title=Trithorax group protein Oryza sativa Trithorax1 controls flowering time in rice via interaction with early heading date3;  citation_journal_title=Plant Physiol;  citation_year=2014;  citation_volume=164;  citation_pages=1326-37; " /><meta name="citation_reference" content="citation_title=Genome-wide high-resolution mapping and functional analysis of DNA methylation in arabidopsis;  citation_journal_title=Cell;  citation_year=2006;  citation_volume=126;  citation_pages=1189-201; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of Arabidopsis thaliana DNA methylation uncovers an interdependence between methylation and transcription;  citation_journal_title=Nat Genet;  citation_year=2007;  citation_volume=39;  citation_pages=61-9; " /><meta name="citation_reference" content="citation_title=Epigenome-wide inheritance of cytosine methylation variants in a recombinant inbred population;  citation_journal_title=Genome Res;  citation_year=2013;  citation_volume=23;  citation_pages=1663-74; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/4/361/25501718/dsy008.pdf" /><meta name="description" content="Abstract.  DNA methylation is an important heritable landmark conferring epigenetic changes in hybrids and has fascinated biologists and plant-breeders over the" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/4/361/4939498" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/4/361/4939498" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/4/361/4939498");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsy008';
    </script>
<title>
	Unravelling the epigenomic interactions between parental inbreds resulting in an altered hybrid methylome in pigeonpea | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4939498" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeWdATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSnVuYWlkLCBBbGltIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgUmVzZWFyY2ggQ2VudHJlIG9uIFBsYW50IEJpb3RlY2hub2xvZ3ksIExCUyBDZW50cmUsIFBVU0EgQ2FtcHVzLCBOZXcgRGVsaGksIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS3VtYXIsIEhpbWFuc2h1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5kaWFuIEFncmljdWx0dXJhbCBTdGF0aXN0aWNzIFJlc2VhcmNoIEluc3RpdHV0ZSwgTmV3IERlbGhpLCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlJhbywgQSBSIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5kaWFuIEFncmljdWx0dXJhbCBTdGF0aXN0aWNzIFJlc2VhcmNoIEluc3RpdHV0ZSwgTmV3IERlbGhpLCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBhdGlsLCBBIE4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJQdWxzZSBSZWFzZXJjaCBVbml0LCBEci4gUGFuamFicmFvIERlc2htdWtoIEtyaXNoaSBWaWR5YXBlZXRoLCBQLk8uIEtyaXNoaSBOYWdhciwgQWtvbGEsIE1haGFyYXNodHJhLCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNpbmdoLCBOIEsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBSZXNlYXJjaCBDZW50cmUgb24gUGxhbnQgQmlvdGVjaG5vbG9neSwgTEJTIENlbnRyZSwgUFVTQSBDYW1wdXMsIE5ldyBEZWxoaSwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHYWlrd2FkLCBLaXNob3IiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBSZXNlYXJjaCBDZW50cmUgb24gUGxhbnQgQmlvdGVjaG5vbG9neSwgTEJTIENlbnRyZSwgUFVTQSBDYW1wdXMsIE5ldyBEZWxoaSwgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IlVucmF2ZWxsaW5nIHRoZSBlcGlnZW5vbWljIGludGVyYWN0aW9ucyBiZXR3ZWVuIHBhcmVudGFsIGluYnJlZHMgcmVzdWx0aW5nIGluIGFuIGFsdGVyZWQgaHlicmlkIG1ldGh5bG9tZSBpbiBwaWdlb25wZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIzNjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjM3MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzeTAwOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE4LzA4LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXBpZ2VuZXRpYyBpbmhlcml0YW5jZSBpbiBwbGFudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00NDc7ICBjaXRhdGlvbl9wYWdlcz00MTgtMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvdmVyaW5nIHlvdXIgYmFzZXM6IGluaGVyaXRhbmNlIG9mIEROQSBtZXRoeWxhdGlvbiBpbiBwbGFudCBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFBsYW50OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz00NzItODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBtZXRoeWxhdGlvbiBhbmQgaHVtYW4gZGlzZWFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz01OTctNjEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgbWV0aHlsYXRpb24gbGFuZHNjYXBlczogcHJvdm9jYXRpdmUgaW5zaWdodHMgZnJvbSBlcGlnZW5vbWljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0IFJldiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9NDY1LTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QZXJjZXB0aW9ucyBvZiBlcGlnZW5ldGljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ0NzsgIGNpdGF0aW9uX3BhZ2VzPTM5Ni04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fc3RhYmxpc2hpbmcsIG1haW50YWluaW5nIGFuZCBtb2RpZnlpbmcgRE5BIG1ldGh5bGF0aW9uIHBhdHRlcm5zIGluIHBsYW50cyBhbmQgYW5pbWFsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MjA0LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcmFiaWRvcHNpcyB0aGFsaWFuYSBETkEgbWV0aHlsYXRpb24gbXV0YW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0yNjA7ICBjaXRhdGlvbl9wYWdlcz0xOTI2LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFyYWJpZG9wc2lzIE1FVDEgY3l0b3NpbmUgbWV0aHlsdHJhbnNmZXJhc2UgbXV0YW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTYzOyAgY2l0YXRpb25fcGFnZXM9MTEwOS0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgZXZvbHV0aW9uYXJ5IGFuYWx5c2lzIG9mIGV1a2FyeW90aWMgRE5BIG1ldGh5bGF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTMyODsgIGNpdGF0aW9uX3BhZ2VzPTkxNi05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Sb2xlIG9mIHRoZSBhcmFiaWRvcHNpcyBEUk0gbWV0aHlsdHJhbnNmZXJhc2VzIGluIGRlIG5vdm8gRE5BIG1ldGh5bGF0aW9uIGFuZCBnZW5lIHNpbGVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTExMzgtNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXNpUk5BcyBhbmQgRE5BIG1ldGh5bGF0aW9uOiBzZWVkeSBlcGlnZW5ldGljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIFBsYW50IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTIwNC0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm9uLUNHIG1ldGh5bGF0aW9uIHBhdHRlcm5zIHNoYXBlIHRoZSBlcGlnZW5ldGljIGxhbmRzY2FwZSBpbiBBcmFiaWRvcHNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBTdHJ1Y3QuIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTY0LTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJSZWVkIEguIFMuICAxOTQyICwgQSBzaG9ydCBoaXN0b3J5IG9mIHRoZSBwbGFudCBzY2llbmNlcy4gaHR0cHM6Ly93d3cuc2NyaWJkLmNvbS9kb2N1bWVudC8xOTA3NDY4MTUvUmVlZC1Ib3dhcmQtUy0xOTQyLWEtU2hvcnQtSGlzdG9yeS1vZi10aGUtUGxhbnQtU2NpZW5jZXMuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvbmcgbm9uY29kaW5nIFJOQXMgaW4gZGV2ZWxvcG1lbnQgYW5kIGRpc2Vhc2Ugb2YgdGhlIGNlbnRyYWwgbmVydm91cyBzeXN0ZW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTQ2MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFucyBjaHJvbW9zb21hbCBtZXRoeWxhdGlvbiBpbiBBcmFiaWRvcHNpcyBoeWJyaWRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTLiBBOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEwOTsgIGNpdGF0aW9uX3BhZ2VzPTM1NzAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5oZXJpdGFuY2Ugb2YgdHJhbnMgY2hyb21vc29tYWwgbWV0aHlsYXRpb24gcGF0dGVybnMgZnJvbSBBcmFiaWRvcHNpcyBGMSBoeWJyaWRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTE7ICBjaXRhdGlvbl9wYWdlcz0yMDE3LTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FcGlnZW5ldGljIGNoYW5nZXMgaW4gaHlicmlkczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjg7ICBjaXRhdGlvbl9wYWdlcz0xMTk3LTIwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXBpZ2VuZXRpYyBuYXR1cmFsIHZhcmlhdGlvbiBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lMTc0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGF0dGVybnMgb2YgcG9wdWxhdGlvbiBlcGlnZW5vbWljIGRpdmVyc2l0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ5NTsgIGNpdGF0aW9uX3BhZ2VzPTE5My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBvZiBETkEgbWV0aHlsYXRpb24gYW5kIGdlbmUgZXhwcmVzc2lvbiBjaGFuZ2VzIGluIHR3byBBcmFiaWRvcHNpcyBlY290eXBlcyBhbmQgdGhlaXIgcmVjaXByb2NhbCBoeWJyaWRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9ODc1LTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgaW5oZXJpdGFuY2UgcGF0dGVybiBvZiAyNCBudCBzaVJOQSBjbHVzdGVycyBpbiBhcmFiaWRvcHNpcyBoeWJyaWRzIGlzIGluZmx1ZW5jZWQgYnkgcHJveGltaXR5IHRvIHRyYW5zcG9zYWJsZSBlbGVtZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWU0NzA0MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYW5nZXMgaW4gMjQtbnQgc2lSTkEgbGV2ZWxzIGluIEFyYWJpZG9wc2lzIGh5YnJpZHMgc3VnZ2VzdCBhbiBlcGlnZW5ldGljIGNvbnRyaWJ1dGlvbiB0byBoeWJyaWQgdmlnb3I7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS5TLkE7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTA4OyAgY2l0YXRpb25fcGFnZXM9MjYxNy0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIGVwaWdlbmV0aWMgYW5kIHRyYW5zY3JpcHRpb25hbCB0cmVuZHMgYW1vbmcgdHdvIHJpY2Ugc3Vic3BlY2llcyBhbmQgdGhlaXIgcmVjaXByb2NhbCBoeWJyaWRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MTctMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlcGVhdCBhc3NvY2lhdGVkIHNtYWxsIFJOQXMgdmFyeSBhbW9uZyBwYXJlbnRzIGFuZCBmb2xsb3dpbmcgaHlicmlkaXphdGlvbiBpbiBtYWl6ZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLlMuQTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDk7ICBjaXRhdGlvbl9wYWdlcz0xMDQ0NC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0b21lIGFuZCBtZXRoeWxvbWUgcHJvZmlsaW5nIHJldmVhbHMgcmVsaWNzIG9mIGdlbm9tZSBkb21pbmFuY2UgaW4gdGhlIG1lc29wb2x5cGxvaWQgQnJhc3NpY2Egb2xlcmFjZWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9Ujc3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdG9tZSBhbmQgbWV0aHlsb21lIGludGVyYWN0aW9ucyBpbiByaWNlIGh5YnJpZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS5TLkE7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTA5OyAgY2l0YXRpb25fcGFnZXM9MTIwNDAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGZpcnN0IGRyYWZ0IG9mIHRoZSBwaWdlb25wZWEgZ2Vub21lIHNlcXVlbmNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBQbGFudCBCaW9jaGVtLiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9OTgtMTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Mb3cgbmF0dXJhbCBvdXRjcm9zc2luZyBpbiDigLJjbGVpc3RvZ2Ftb3Vz4oCyIHBpZ2VvbnBlYSBtdXRhbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUludC4gUGlnZW9ucGVhIE5ld3NsZXR0OyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9OS0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFsZSBzdGVyaWxpdHkgYW5kIGZlcnRpbGl0eSByZXN0b3JhdGlvbiBpbiBjcm9wczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBQbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTY1OyAgY2l0YXRpb25fcGFnZXM9NTc5LTYwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3l0b3BsYXNtaWMgbWFsZSBzdGVyaWxpdHktYXNzb2NpYXRlZCBjaGltZXJpYyBvcGVuIHJlYWRpbmcgZnJhbWVzIGlkZW50aWZpZWQgYnkgbWl0b2Nob25kcmlhbCBnZW5vbWUgc2VxdWVuY2luZyBvZiBmb3VyIENhamFudXMgZ2Vub3R5cGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9NDg1LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljcyBvZiBmZXJ0aWxpdHkgcmVzdG9yYXRpb24gaW4gQTIgY3l0b3BsYXNtIGJhc2VkIGh5YnJpZHMgb2YgcGlnZW9ucGVhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbnQuIEouIEN1ci4gTWljcm9iaW9sLiBBcHBsLiBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IGdlbm9tZSBzZXF1ZW5jZSBvZiBwaWdlb25wZWEgKENhamFudXMgY2FqYW4pLCBhbiBvcnBoYW4gbGVndW1lIGNyb3Agb2YgcmVzb3VyY2UtcG9vciBmYXJtZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz04My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CU01BUDogd2hvbGUgZ2Vub21lIGJpc3VsZml0ZSBzZXF1ZW5jZSBNQVBwaW5nIHByb2dyYW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTIzMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPeKAskxldmVsaW5n4oCyIHRoZSBwbGF5aW5nIGZpZWxkIGZvciBhbmFseXNlcyBvZiBzaW5nbGUtYmFzZSByZXNvbHV0aW9uIEROQSBtZXRoeWxvbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz01ODMtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9bWV0aHlsS2l0OiBhIGNvbXByZWhlbnNpdmUgUiBwYWNrYWdlIGZvciB0aGUgYW5hbHlzaXMgb2YgZ2Vub21lLXdpZGUgRE5BIG1ldGh5bGF0aW9uIHByb2ZpbGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvLmw7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz1SODciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZXRoUHJpbWVyOiBkZXNpZ25pbmcgcHJpbWVycyBmb3IgbWV0aHlsYXRpb24gUENSczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xNDI3LTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RVU1BOiBxdWFudGlmaWNhdGlvbiB0b29sIGZvciBtZXRoeWxhdGlvbiBhbmFseXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz1XMTcwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QgZ2FwcGVkLXJlYWQgYWxpZ25tZW50IHdpdGggQm93dGllIDI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MzU3LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJTRU06IGFjY3VyYXRlIHRyYW5zY3JpcHQgcXVhbnRpZmljYXRpb24gZnJvbSBSTkEtU2VxIGRhdGEgd2l0aCBvciB3aXRob3V0IGEgcmVmZXJlbmNlIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MzIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9ZWRnZVI6IGEgYmlvY29uZHVjdG9yIHBhY2thZ2UgZm9yIGRpZmZlcmVudGlhbCBleHByZXNzaW9uIGFuYWx5c2lzIG9mIGRpZ2l0YWwgZ2VuZSBleHByZXNzaW9uIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MTM5LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XRUdPOiBhIHdlYiB0b29sIGZvciBwbG90dGluZyBHTyBhbm5vdGF0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz1XMjkzLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpcy1TTlA6IGNvbWJpbmVkIEROQSBtZXRoeWxhdGlvbiBhbmQgU05QIGNhbGxpbmcgZm9yIEJpc3VsZml0ZS1zZXEgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz1SNjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zZXJ2YXRpb24gYW5kIGRpdmVyZ2VuY2Ugb2YgbWV0aHlsYXRpb24gcGF0dGVybmluZyBpbiBwbGFudHMgYW5kIGFuaW1hbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS5TLkE7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9ODY4OS05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXBpZ2VuZXRpYyByZWd1bGF0aW9uIG9mIHRyYW5zcG9zYWJsZSBlbGVtZW50cyBpbiBwbGFudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubnUuIFJldi4gUGxhbnQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT02MDsgIGNpdGF0aW9uX3BhZ2VzPTQzLTY2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaG90Z3VuIGJpc3VscGhpdGUgc2VxdWVuY2luZyBvZiB0aGUgQXJhYmlkb3BzaXMgZ2Vub21lIHJldmVhbHMgRE5BIG1ldGh5bGF0aW9uIHBhdHRlcm5pbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00NTI7ICBjaXRhdGlvbl9wYWdlcz0yMTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBhIG5vdmVsIHNhbHQgdG9sZXJhbmNlIGdlbmUgaW4gd2lsZCBzb3liZWFuIGJ5IHdob2xlLWdlbm9tZSBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIENvbW11bjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NDM0MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIHRoZSBnZW5vbWUgc2VxdWVuY2Ugb2YgdGhlIGZsb3dlcmluZyBwbGFudCBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgY2l0YXRpb25fYXV0aG9yPVRoZSBBcmFiaWRvcHNpcyBHZW5vbWUsIEk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT00MDg7ICBjaXRhdGlvbl9wYWdlcz03OTYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BdXRvdGV0cmFwbG9pZCByaWNlIG1ldGh5bG9tZSBhbmFseXNpcyByZXZlYWxzIG1ldGh5bGF0aW9uIHZhcmlhdGlvbiBvZiB0cmFuc3Bvc2FibGUgZWxlbWVudHMgYW5kIHRoZWlyIGVmZmVjdHMgb24gZ2VuZSBleHByZXNzaW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUuUy5BOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTExMjsgIGNpdGF0aW9uX3BhZ2VzPUU3MDIyLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbmdsZS1iYXNlIHJlc29sdXRpb24gbWFwcyBvZiBjdWx0aXZhdGVkIGFuZCB3aWxkIHJpY2UgbWV0aHlsb21lcyBhbmQgcmVndWxhdG9yeSByb2xlcyBvZiBETkEgbWV0aHlsYXRpb24gaW4gcGxhbnQgZ2VuZSBleHByZXNzaW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0zMDAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WYXJpYXRpb24gYW5kIHBhdHRlcm5zIG9mIEROQSBtZXRoeWxhdGlvbiBpbiBtYWl6ZSBDLXR5cGUgQ01TIGxpbmVzIGFuZCB0aGVpciBtYWludGFpbmVyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gUGxhbnQgQmlvY2hlbS4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTQzLTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgbWV0aHlsYXRpb24gY2hhbmdlcyBpbiBwaG90b3BlcmlvZC10aGVybW8tc2Vuc2l0aXZlIG1hbGUgc3RlcmlsZSByaWNlIFBBNjRTIHVuZGVyIHR3byBkaWZmZXJlbnQgY29uZGl0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01Mzc7ICBjaXRhdGlvbl9wYWdlcz0xNDMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNnZW5lcmF0aW9uYWwgZXBpZ2VuZXRpYyBpbnN0YWJpbGl0eSBpcyBhIHNvdXJjZSBvZiBub3ZlbCBtZXRoeWxhdGlvbiB2YXJpYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzQ7ICBjaXRhdGlvbl9wYWdlcz0zNjktNzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwb250YW5lb3VzIGVwaWdlbmV0aWMgdmFyaWF0aW9uIGluIHRoZSBBcmFiaWRvcHNpcyB0aGFsaWFuYSBtZXRoeWxvbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00ODA7ICBjaXRhdGlvbl9wYWdlcz0yNDUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXBpZ2VuZXRpYyBhbmQgZ2VuZXRpYyBpbmZsdWVuY2VzIG9uIEROQSBtZXRoeWxhdGlvbiB2YXJpYXRpb24gaW4gbWFpemUgcG9wdWxhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yNzgzLTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaXRvY2hvbmRyaWFsIGFsZGVoeWRlIGRlaHlkcm9nZW5hc2UgYWN0aXZpdHkgaXMgcmVxdWlyZWQgZm9yIG1hbGUgZmVydGlsaXR5IGluIG1haXplOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9MTA2My03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGVudGF0cmljb3BlcHRpZGUtcmVwZWF0IGZhbWlseSBwcm90ZWluIFJGNiBmdW5jdGlvbnMgd2l0aCBoZXhva2luYXNlIDYgdG8gcmVzY3VlIHJpY2UgY3l0b3BsYXNtaWMgbWFsZSBzdGVyaWxpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS5TLkE7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTEyOyAgY2l0YXRpb25fcGFnZXM9MTQ5ODQtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RHluYW1pYyByb2xlcyBmb3Igc21hbGwgUk5BcyBhbmQgRE5BIG1ldGh5bGF0aW9uIGR1cmluZyBvdnVsZSBhbmQgZmliZXIgZGV2ZWxvcG1lbnQgaW4gYWxsb3RldHJhcGxvaWQgY290dG9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9ZTEwMDU3MjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljIEROQSBtZXRoeWxhdGlvbiBhbmFseXNlcyByZXZlYWwgdGhlIGRpc3RpbmN0IHByb2ZpbGVzIGluIGNhc3RvciBiZWFuIHNlZWRzIHdpdGggcGVyc2lzdGVudCBlbmRvc3Blcm1zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTE3MTsgIGNpdGF0aW9uX3BhZ2VzPTEyNDItNTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGVwaWdlbmV0aWMgcmVndWxhdGlvbiBvZiBnZW5lIHRyYW5zY3JpcHRpb24gaW4gbWFpemUgc2VlZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9ZTAxMzk1ODIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXZlcmdlbnQgRE5BIG1ldGh5bGF0aW9uIHBhdHRlcm5zIGFzc29jaWF0ZWQgd2l0aCBnZW5lIGV4cHJlc3Npb24gaW4gcmljZSBjdWx0aXZhcnMgd2l0aCBjb250cmFzdGluZyBkcm91Z2h0IGFuZCBzYWxpbml0eSBzdHJlc3MgcmVzcG9uc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gUmVwOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xNDkyMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFuYWx5c2lzIG9mIEROQSBtZXRoeWxhdGlvbiBpbiBzb3liZWFuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFBsYW50OyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0xOTYxLTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1STkEtbWVkaWF0ZWQgY2hyb21hdGluLWJhc2VkIHNpbGVuY2luZyBpbiBwbGFudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUN1cnIuIE9waW4uIENlbGwgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTM2Ny03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2hhdCB0cmlnZ2VycyBkaWZmZXJlbnRpYWwgRE5BIG1ldGh5bGF0aW9uIG9mIGdlbmVzIGFuZCBURXM6IGNvbnRyaWJ1dGlvbiBvZiBib2R5IG1ldGh5bGF0aW9uPzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q29sZCBTcHJpbmcgSGFyYi4gU3ltcC4gUXVhbnQuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Nzc7ICBjaXRhdGlvbl9wYWdlcz0xNTUtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBtZXRoeWxhdGlvbiBvZiByZXRyb3RyYW5zcG9zb25zLCBETkEgdHJhbnNwb3NvbnMgYW5kIGdlbmVzIGluIHN1Z2FyIGJlZXQgKEJldGEgdnVsZ2FyaXMgTC4pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0yMDE3OyAgY2l0YXRpb25fdm9sdW1lPTkwOyAgY2l0YXRpb25fcGFnZXM9MTE1Ni03NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBkaXN0cmlidXRpb24gb2YgSDNLOW1lMiBhbmQgRE5BIG1ldGh5bGF0aW9uIGluIGEgbWFpemUgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ZTEwNTI2NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNISCBpc2xhbmRzOiBkZSBub3ZvIEROQSBtZXRoeWxhdGlvbiBpbiBuZWFyLWdlbmUgY2hyb21hdGluIHJlZ3VsYXRpb24gaW4gbWFpemU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz02MjgtMzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNvbWF0aWNhbGx5IGhlcml0YWJsZSBzd2l0Y2hlcyBpbiB0aGUgRE5BIG1vZGlmaWNhdGlvbiBvZiBNdSB0cmFuc3Bvc2FibGUgZWxlbWVudHMgbW9uaXRvcmVkIHdpdGggYSBzdXBwcmVzc2libGUgbXV0YW50IGluIG1haXplOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5lcyBEZXY7ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTMzMS00MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXBpZ2VuZXRpYyByZWd1bGF0aW9uIG9mIHJpY2UgZmxvd2VyaW5nIGFuZCByZXByb2R1Y3Rpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZyb250LiBQbGFudCBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTgwMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURETTEgKGRlY3JlYXNlIGluIEROQSBtZXRoeWxhdGlvbikgZ2VuZXMgaW4gcmljZSAoT3J5emEgc2F0aXZhKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBHZW5ldC4gR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mjg3OyAgY2l0YXRpb25fcGFnZXM9Nzg1LTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ucml0aG9yYXggZ3JvdXAgcHJvdGVpbiBPcnl6YSBzYXRpdmEgVHJpdGhvcmF4MSBjb250cm9scyBmbG93ZXJpbmcgdGltZSBpbiByaWNlIHZpYSBpbnRlcmFjdGlvbiB3aXRoIGVhcmx5IGhlYWRpbmcgZGF0ZTM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTY0OyAgY2l0YXRpb25fcGFnZXM9MTMyNi0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgaGlnaC1yZXNvbHV0aW9uIG1hcHBpbmcgYW5kIGZ1bmN0aW9uYWwgYW5hbHlzaXMgb2YgRE5BIG1ldGh5bGF0aW9uIGluIGFyYWJpZG9wc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTEyNjsgIGNpdGF0aW9uX3BhZ2VzPTExODktMjAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBvZiBBcmFiaWRvcHNpcyB0aGFsaWFuYSBETkEgbWV0aHlsYXRpb24gdW5jb3ZlcnMgYW4gaW50ZXJkZXBlbmRlbmNlIGJldHdlZW4gbWV0aHlsYXRpb24gYW5kIHRyYW5zY3JpcHRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdCBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPTYxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVwaWdlbm9tZS13aWRlIGluaGVyaXRhbmNlIG9mIGN5dG9zaW5lIG1ldGh5bGF0aW9uIHZhcmlhbnRzIGluIGEgcmVjb21iaW5hbnQgaW5icmVkIHBvcHVsYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xNjYzLTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjUvNC8zNjEvMjU1MDE3MTgvZHN5MDA4LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBETkEgbWV0aHlsYXRpb24gaXMgYW4gaW1wb3J0YW50IGhlcml0YWJsZSBsYW5kbWFyayBjb25mZXJyaW5nIGVwaWdlbmV0aWMgY2hhbmdlcyBpbiBoeWJyaWRzIGFuZCBoYXMgZmFzY2luYXRlZCBiaW9sb2dpc3RzIGFuZCBwbGFudC1icmVlZGVycyBvdmVyIHRoZSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDkzOTQ5OCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQDQbU/Df5Hp/8r1Z86a/IUOY2jPw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f361%2f4939498" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f361%2f4939498" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=109685" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/4/1/m_dnares_25_4cover.png?Expires=2147483647&amp;Signature=lD6dhAB99HpTltGbT4-zO-h9Z6msKpBpjdlZuCIFAdO0RvvEM~IgMElNp5bE6Rdf~OSfdWzB2y8SIAkqvb6Fc-0GRLo0HVHIHIXLjJF8ZJ94ZrTCEDv2Gn-92BvjK~Ebx3LV-ePsMx8j4josbKy1FscEJ3-qj3zB2TEHnK~iAQqommn5aOw82hhC1dOH5f1P15MUh9o1QIOhJ9WAnJwCFYDxRE0ZmdX~qCpKaFm8XPRL-O71Rxc1eeV8iTb9yQzgpbw2OwX1g4mQ5iALY~7Bk3L2RgUkQZIzgNqb8ANqfHjAsIlrzFHcKAl7tTiKtoGbsHvDqkXNPcKjgjrdKdIXQw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/4">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="120122238">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122238">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122240">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122240">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122246">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122246">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122264">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122264">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122291">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122291">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122300">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122300">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120122306">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120122306">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/4/353/4922629">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/4/375/4958748">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Unravelling the epigenomic interactions between parental inbreds resulting in an altered hybrid methylome in pigeonpea
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Alim Junaid</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">National Research Centre on Plant Biotechnology, LBS Centre, PUSA Campus, New Delhi, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Alim+Junaid">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Junaid A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Junaid A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Alim Junaid</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Himanshu Kumar</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Indian Agricultural Statistics Research Institute, New Delhi, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Himanshu+Kumar">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kumar H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kumar H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Himanshu Kumar</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">A R Rao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Indian Agricultural Statistics Research Institute, New Delhi, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=A+R+Rao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rao A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Rao A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">A R Rao</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">A N Patil</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Pulse Reaserch Unit, Dr. Panjabrao Deshmukh Krishi Vidyapeeth, P.O. Krishi Nagar, Akola, Maharashtra, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=A+N+Patil">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Patil A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Patil A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">A N Patil</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">N K Singh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">National Research Centre on Plant Biotechnology, LBS Centre, PUSA Campus, New Delhi, India</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=N+K+Singh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Singh N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Singh N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">N K Singh</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kishor Gaikwad</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">National Research Centre on Plant Biotechnology, LBS Centre, PUSA Campus, New Delhi, India</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy008-cor1" data-open="dsy008-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. 01125841787; ext. 246. Email: <a href="mailto:kish2012@nrcpb.org" target="_blank">kish2012@nrcpb.org</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kishor+Gaikwad">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gaikwad K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gaikwad K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kishor Gaikwad</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy008-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 4, 1 August 2018, Pages 361–373, <a href='https://doi.org/10.1093/dnares/dsy008'>https://doi.org/10.1093/dnares/dsy008</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">16 March 2018</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">21 November 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">21 February 2018</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4939498" href="/dnaresearch/article-pdf/25/4/361/25501718/dsy008.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Alim Junaid, Himanshu Kumar, A R Rao, A N Patil, N K Singh, Kishor Gaikwad;  Unravelling the epigenomic interactions between parental inbreds resulting in an altered hybrid methylome in pigeonpea, <em>DNA Research</em>, Volume 25, Issue 4, 1 August 2018, Pages 361–373, <a href='https://doi.org/10.1093/dnares/dsy008'>https://doi.org/10.1093/dnares/dsy008</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4939498?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4939498?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4939498?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4939498?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4939498?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4939498?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4939498?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Unravelling%20the%20epigenomic%20interactions%20between%20parental%20inbreds%20resulting%20in%20an%20altered%20hybrid%20methylome%20in%20pigeonpea&amp;publicationDate=2018-03-16&amp;volumeNum=25&amp;issueNum=4&amp;author=Junaid%2C%20Alim%3B%20Kumar%2C%20Himanshu&amp;startPage=361&amp;endPage=373&amp;contentId=10.1093%2Fdnares%2Fdsy008&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f361%2f4939498" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=120122238 id="120122238" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>DNA methylation is an important heritable landmark conferring epigenetic changes in hybrids and has fascinated biologists and plant-breeders over the years. Although epigenetic changes have been documented in rice and maize hybrids, such investigations have not been reported in pigeonpea. Here, we report genome-wide methylation profiles of pigeonpea sterile and fertile inbred lines and their fertile F1 hybrid at single base resolution. We found that pigeonpea genome is relatively enriched in CG methylation. Identification of differentially methylated regions (DMRs) in the sterile and fertile parent revealed remarkable differences between their methylation patterns. Investigation of methylation status of parental DMRs in hybrid revealed non-additive methylation patterns resulting from trans-chromosomal methylation and trans-chromosomal demethylation events. Furthermore, we discovered several DMRs negatively associated with gene expression in the hybrid and fertile parent. Interestingly, many of those DMRs belonged to transposable elements and genes encoding pentatricopeptide repeats associated proteins, which may mediate a role in modulating the genes impacting pollen fertility. Overall, our findings provide an understanding of two parental epigenomes interacting to give rise to an altered methylome in pigeonpea hybrids, from genome-wide point of view.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DNA methylation&quot;">DNA methylation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="TcM">TcM</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="TcDM">TcDM</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="pigeonpea">pigeonpea</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="hybrid">hybrid</a></div>                            <h2 scrollto-destination=120122240 id="120122240" class="section-title" >1. Introduction</h2>
<p>Epigenetic regulation has remained of fundamental interest in understanding gene regulation in eukaryotes. Epigenetic regulation involves differential methylation at cytosine residues and post-translational modifications of histones.<a reveal-id="dsy008-B1" data-open="dsy008-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> One of the most comprehensively investigated mechanism of chromatin modification in plants,<a reveal-id="dsy008-B2" data-open="dsy008-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> DNA methylation involves covalent transfer of a methyl group to C-5 position of the cytosine ring by DNA methyltransferases and is transmitted across generations.<a reveal-id="dsy008-B3" data-open="dsy008-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> DNA methylation acts to regulate transposons and repetitive genome sequences and often results in the suppression of gene activity. <a reveal-id="dsy008-B4" data-open="dsy008-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsy008-B5" data-open="dsy008-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> A major difference between DNA methylation in animals and plants is that in plant genome, DNA methylation occurs at CG, CHH and CHG sequence contexts (where H could be C, T or A).<a reveal-id="dsy008-B6" data-open="dsy008-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> It has been shown in <em>Arabidopsis</em> that different enzymes regulate methylation at different sequence contexts. Methyltransferase 1 (MET1) causes and maintains methylation at CG sites<a reveal-id="dsy008-B6 dsy008-B7 dsy008-B8" data-open="dsy008-B6 dsy008-B7 dsy008-B8" class="link link-ref link-reveal xref-bibr"><sup>6–8</sup></a> and chromomethylase 3 (CMT3) produces and maintains methylation at CHG sites<a reveal-id="dsy008-B6" data-open="dsy008-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsy008-B9" data-open="dsy008-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsy008-B10" data-open="dsy008-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> and this maintenance is further required to preserve methylation patterns over generations. On the other hand, domains rearranged methyltransferase 2 (DRM2) and 24 nt siRNAs, which guide methyltransfrase to specific DNA segments, regulate de novo methylation CHH sites,<a reveal-id="dsy008-B6" data-open="dsy008-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsy008-B10" data-open="dsy008-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsy008-B11" data-open="dsy008-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> which is maintained by CMT2.<a reveal-id="dsy008-B12" data-open="dsy008-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a></p><p>De novo methylation of all three different sites involves 24 nt smRNAs (small RNAs), DRM2 and the Dnmt3 homologue involved in RNA-directed DNA methylation (RdDM). In RdDM, RNA is transcribed by the RNA polymerase POLIV, which is then made double stranded by RNA-dependent RNA polymerases (RDRs), cleaved to 24 nt by Dicer-like enzymes and loaded onto argonaute proteins. This effector complex then interacts with other proteins and Polymerase V transcripts to target homologous regions of the genome for methylation by the recruitment of DRM2. The methylation state of CG is maintained during cell replication by MET1, which recognizes hemi methylated CG sites and methylate the newly formed DNA strand. CMT3 maintains CHG methylation, while CHH methylation is recognized by a histone methyltransferase kryptonite (KYP), which can then methylate the H3K9 residues. These two enzymes work in a feedback loop enabling the maintenance of both epigenetic marks, with the absence of either CMT3 or KYP leading to a reduction in CHG methylation. DNA methylation plays an important role during different aspects of plant development and growth. Of these heterosis or hybrid vigour is of vital importance in agriculture. Heterosis is a phenomenon where F1 progeny of a genetic cross exhibits superior characteristics comparedwith its inbred parental lines and has been documented nearly 300 years ago.<a reveal-id="dsy008-B13" data-open="dsy008-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> In recent years, several studies have focused on the role of epigenetic regulation in mediating hybrid vigour in model plant Arabidopsis.<a reveal-id="dsy008-B1" data-open="dsy008-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsy008-B14 dsy008-B15 dsy008-B16 dsy008-B17" data-open="dsy008-B14 dsy008-B15 dsy008-B16 dsy008-B17" class="link link-ref link-reveal xref-bibr"><sup>14–17</sup></a> Previous studies in <em>Arabidopsis</em> have suggested that interaction between similar genomes and different epigenomes of the two different parental ecotypes could result in new transcriptional profiles in hybrids that might contribute to hybrid vigour in the progeny.<a reveal-id="dsy008-B18" data-open="dsy008-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsy008-B19" data-open="dsy008-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Also, it has been proposed that epigenetic regulation via changes in methylation patterns might play a role in mediating differences in gene expression levels that may confer increased biomass in heterotic hybrid offspring of <em>Arabidopsis</em> ecotypes C24 and Landsberg erecta.<a reveal-id="dsy008-B15" data-open="dsy008-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> It has also been documented that both DNA methylation patterns and sRNA (small RNA) levels are altered in intraspecific hybrids of Arabidopsis,<a reveal-id="dsy008-B15" data-open="dsy008-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsy008-B20 dsy008-B21 dsy008-B22" data-open="dsy008-B20 dsy008-B21 dsy008-B22" class="link link-ref link-reveal xref-bibr"><sup>20–22</sup></a> rice<a reveal-id="dsy008-B23" data-open="dsy008-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> and maize.<a reveal-id="dsy008-B24" data-open="dsy008-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a></p><p>Recent advances in DNA methylation profiling techniques have enabled in understanding the correlation between methylation patterns and hybrid vigour in several crop species as well such as rice and Brassica<a reveal-id="dsy008-B25" data-open="dsy008-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a><sup>,</sup><a reveal-id="dsy008-B26" data-open="dsy008-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> but knowledge about interaction of methylation patterns and gene expression in legume plants such as pigeonpea has still remained elusive. Pigeonpea (<em>Cajanus cajan</em> (L) Millsp) is a important grain crop with high protein (20–22%) grain legume, mainly cultivated in the tropical and subtropical parts of the world. Pigeonpea contains 11 pairs of chromosomes (2<em>n</em> = 22) and has a genome size of 858 Mbp.<a reveal-id="dsy008-B27" data-open="dsy008-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> Globally pigeonpea is cultivated on 5.4 Mha, with an annual production of 4.4 MT and a yield of 8299 hg/ha. Pigeonpea is grown in about 50 countries with India occupying 66% of its area (FAO, 2016, <a class="link link-uri" href="http://www.fao.org/faostat/en/#data/QC" target="_blank">http://www.fao.org/faostat/en/#data/QC</a>). India is also the largest producer and consumer of pigeonpea with an annual production of 2.51 MT.</p><p>Pigeonpea is characterized by significant genetic variability. Genetic studies have also revealed the occurrence of non-additive genetic variation in pigeonpea hybrids.<a reveal-id="dsy008-B28" data-open="dsy008-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> This particular property has become popular among breeders to produce varieties with improved fertility and been utilized through cytoplasmic male sterility (CMs) system. CMS is a condition in which the ability of a plant to produce fertile pollen is severely impaired. In this condition, rearrangements in mitochondrial genome result in chimeric mitochondrial open reading frames (ORFs), generating non-functionality in anthers. However, the expression of such chimeric mitochondrial ORFs can be suppressed by fertility restorer gene in F1 hybrid, thus restoring fertility<em>.</em> This kind of hybrid breeding technology involves three parents— a male sterile parent, its maintainer and a fertility restorer line (commonly called—A, B and R lines, respectively; reviewed in detail in the literature).<a reveal-id="dsy008-B29" data-open="dsy008-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> Few CMS associated mitochondrial <em>orfs</em> have been identified in pigeonpea.<a reveal-id="dsy008-B30" data-open="dsy008-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a></p><p>In this study, we have investigated whether there are any epigenetic changes in F1 progeny of cross between 11 A (male sterile) and 303 R (fertile) inbred lines and whether those changes could potentially mediate an impact on the gene expression patterns including those involved in fertility restoration in the hybrid. We conducted whole genome bisulfite sequencing of 11 A, 303 R parental genotypes and their hybrid to generate whole genome cytosine methylation profiles and found that overall, hybrid methylome was remarkably altered compared with the parental methylome due to the events of transchrosomal methylation and demethylation in hybrid. We next carried out transcriptome profiling of the same plants and compared methylation status of the differentially expressed genes between parental inbreds and the F1 hybrids. We found differences in methylation patterns in the three genotypes, suggesting the involvement of DNA methylation in regulation of gene expression and possibly influencing sterility and fertility.</p>                            <h2 scrollto-destination=120122246 id="120122246" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=120122247 id="120122247" class="section-title" >2.1. Plant material and growth conditions</h3>
<p>Pigeonpea lines 11 A and 303 R<a reveal-id="dsy008-B31" data-open="dsy008-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> were crossed by hand pollination using 303 R as pollen donor to generate hybrid. Seeds were collected and F1 seed resulting from the cross along with seeds from parental lines were sown again together for further experimental work. Seeds were directly sown in the soil in green house under natural conditions. All the plants were grown at the same time under exactly similar growth conditions to reduce the impact of any environmental variations. Fertility status of all three genotypes was verified by pollen analysis. Pollens were fixed on a glass slide using 2% acetocarmine. Pollens stained with acetocarmine were considered to be fertile. After pollen verification, young leaves of same age from the three lines were harvested, frozen in liquid nitrogen and used to extract DNA and RNA for further work.</p>                            <h3 scrollto-destination=120122249 id="120122249" class="section-title" >2.2. Whole genome bisulfite sequencing library construction and data analysis</h3>
<p>High molecular weight genomic DNA was isolated from the leaves using cetyl trimethylammonium bromide method. DNA was fragmented into 100–300 bp using Covaris followed by end repair by adding ‘A’ and True seq methylated adapters were ligated. The ligated DNA fragments were subjected to bisulfite treatment using EZ DNA Methylation-Gold Kit, according to manufacturer’s instructions. The BS libraries were amplified, quality tested using Bio-analyzer (Agilent 2100) and subjected to Ilumina Hi-seq2000 for 90 cycles in paired-end mode. The adaptors and poor quality reads were removed from the raw data using TrimGalore (<a class="link link-uri" href="https://github.com/FelixKrueger/TrimGalore" target="_blank">https://github.com/FelixKrueger/TrimGalore</a>). High quality reads were mapped to the reference genome<a reveal-id="dsy008-B27" data-open="dsy008-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a><sup>,</sup><a reveal-id="dsy008-B32" data-open="dsy008-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> using BSMAP<a reveal-id="dsy008-B33" data-open="dsy008-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> at default parameters. Methylation information was extracted using Python script. Reads were mapped on chloroplast genome to calculate methylation efficiency and error rate. A bisulfite conversion rate of  &gt;99% was obtained for all three samples. In-order to determine true methylated cytosines, <em>P</em>-values were calculated using binomial distribution <em>P</em>= binomial (<em>m</em>, x, error rate), where <em>m</em>= number of methylated reads, <em>n</em>= number of non-methylated reads and <em>x</em>= m + n (coverage on a single C).<a reveal-id="dsy008-B15" data-open="dsy008-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> Cytosines with <em>P</em>-value &lt; 0.0025 were considered to be truly methylated. Fraction and mean methylation level was calculated as described previously.<a reveal-id="dsy008-B34" data-open="dsy008-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a></p>                            <h3 scrollto-destination=120122251 id="120122251" class="section-title" >2.3. Identification of differentially methylated regions</h3>
<p>To identify differentially methylated regions (DMRs) at each sequence context (CG, CHG and CHH), genomes were tiled using a sliding window approach with 100 bp window sliding at 100 bp interval for CG and CHG and 150 bp for CHH. Methylation information for each window was calculated using Methyl kit.<a reveal-id="dsy008-B35" data-open="dsy008-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a></p><p><em>P</em>-value for each window was calculated using Fisher’s exact test. To determine q-values, <em>P</em>-values for each window were corrected using sliding linear model . Following criteria were used for identification of DMRs. <ol class="number"><li><p>Windows contained at-least three methylated cytosines for CG and CHG; and six for CHH DMRs.</p></li><li><p>Bases having more than 10X coverage were considered.</p></li><li><p>Bases having more than 99.9th percentile coverage were discarded to minimize errors arising due to PCR bias.</p></li><li><p>Windows with methylation difference greater than 20% for CG, CHG and 25% for CHH were selected.</p></li><li><p>Fisher’s exact test corrected q-values &lt; 0.01.</p></li></ol></p>                            <h3 scrollto-destination=120122254 id="120122254" class="section-title" >2.4. Validation of methylation levels using bisulfite sequencing PCR</h3>
<p>Two different regions were chosen one for validating methylation levels, one in the intergenic region and the other one in the exonic region. For bisulfite PCR, genomic DNA was extracted from 11 A and F1 hybrid and subjected to bisulfite conversion using EZ DNA Methylation-Gold™ Kit. Primers were designed using METHPRIME<a reveal-id="dsy008-B36" data-open="dsy008-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008_supp.zip?Expires=2147483647&Signature=a5pxiGBiZuW6BLObg3zcGti-PLIrtW4lDnTZKtyw3m~A3SRkZcr0Y2ve6moKjtosqKCTHkcpfiWI1YiZ59ZYjagra1cBx2qHidDiS9YD76AyCdt-4~W-54Oj7mbqe~VnqVToy5n7EnQdmwYondGxoqfU4SGnxtqtHyNZnOY--6wL3vaY64N6y9Ol6VaEdmhxlHtZ9zskpG0jWypn5ZEvlgg-2NnJYoPL6EOKqxhPhKOweQ9WNBLLfKlYzWiqy7TPWCchHeatrk45nHlTmBeMjmhFGIHpav47NDhdJbGqGb2EIjKAOQuctS37uI7xAHUHDrLIQQ~VOnhtRcc1h0JCcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Bisulfite converted DNA was amplified and the fragments were cloned into pGEMT vector. Ten colonies were picked randomly for sequencing. The results were analyzed using QUMA.<a reveal-id="dsy008-B37" data-open="dsy008-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a></p>                            <h3 scrollto-destination=120122256 id="120122256" class="section-title" >2.5. RNA sequencing library construction and analysis</h3>
<p>Total RNA was extracted from the same plants used for BS Seq. Equal amount of total RNA from three biological replicates were pooled. About 1 µg of the total pooled RNA was used to purify poly-A containing mRNA molecules. The purified mRNA was fragmented into smaller pieces and reverse transcribed into cDNA. The cDNA fragments were subjected to end repair by adding single ‘A’ base. True seq adaptors were ligated and cDNA libraries were amplified for 15 cycles. Quality of the library was assessed using Bioanalyzer and the validated library was sequenced by Illumina Hi-Seq 1000 according to manufacturers’ instructions. The quality of the raw reads was determined using FastQc (<a class="link link-uri" href="https://www.illumina.com/products/by-type/informatics-products/basespace-sequence-hub/apps/fastqc.html" target="_blank">https://www.illumina.com/products/by-type/informatics-products/basespace-sequence-hub/apps/fastqc.html</a>). Adaptors and poor quality reads were cleaned using Trim Galore. High quality paired end reads were aligned to reference genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008_supp.zip?Expires=2147483647&Signature=a5pxiGBiZuW6BLObg3zcGti-PLIrtW4lDnTZKtyw3m~A3SRkZcr0Y2ve6moKjtosqKCTHkcpfiWI1YiZ59ZYjagra1cBx2qHidDiS9YD76AyCdt-4~W-54Oj7mbqe~VnqVToy5n7EnQdmwYondGxoqfU4SGnxtqtHyNZnOY--6wL3vaY64N6y9Ol6VaEdmhxlHtZ9zskpG0jWypn5ZEvlgg-2NnJYoPL6EOKqxhPhKOweQ9WNBLLfKlYzWiqy7TPWCchHeatrk45nHlTmBeMjmhFGIHpav47NDhdJbGqGb2EIjKAOQuctS37uI7xAHUHDrLIQQ~VOnhtRcc1h0JCcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>) using Bowtie alignment program,<a reveal-id="dsy008-B38" data-open="dsy008-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> then relative abundance of transcripts was calculated using RSEM.<a reveal-id="dsy008-B39" data-open="dsy008-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> Transcripts with zero Fragments Per Kilobase of transcript per Million mapped reads (FPKM) value were excluded from analysis. Raw reads count data were imported into R environment and data table containing fragment counts for each transcript were combined and matrix was formed to identify differentially expressed gene using edgeR.<a reveal-id="dsy008-B40" data-open="dsy008-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a></p>                            <h3 scrollto-destination=120122258 id="120122258" class="section-title" >2.6. GO enrichment analysis</h3>
<p>To identify and categorize cluster of differentially expressed genes (DEGs) and genes enriched for methylation in all the three categories (11 A vs. 303 R, 11 A vs. hybrid and 303 R vs. hybrid), GO enrichment analysis was performed using python based program GOATOOLS (<a class="link link-uri" href="https://github.com/tanghaibao/goatools" target="_blank">https://github.com/tanghaibao/goatools</a>) and Wego.<a reveal-id="dsy008-B41" data-open="dsy008-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> Genes with Bonferroni-corrected <em>P</em>-values &lt; 0.05 using GOATOOLS were considered to be enriched.</p>                            <h3 scrollto-destination=120122260 id="120122260" class="section-title" >2.7. Identification of transposable elements</h3>
<p>A <em>denovo</em> repeat library was generated using repeatmodeller (<a class="link link-uri" href="http://www.repeatmasker.org/RepeatModeler/" target="_blank">http://www.repeatmasker.org/RepeatModeler/</a>; version 1.0.11).This customized library was used to run RepeatMasker <strong>(</strong><a class="link link-uri" href="http://www.repeatmasker.org/" target="_blank">http://www.repeatmasker.org/</a>; version- 4.0.7) to identify and annotate transposable elements (TEs) in pigeonpea at default parameters. TE shorter than 100 bp were excluded from the analysis and the distance between TE, less than 50 bp were conconated (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008_supp.zip?Expires=2147483647&Signature=a5pxiGBiZuW6BLObg3zcGti-PLIrtW4lDnTZKtyw3m~A3SRkZcr0Y2ve6moKjtosqKCTHkcpfiWI1YiZ59ZYjagra1cBx2qHidDiS9YD76AyCdt-4~W-54Oj7mbqe~VnqVToy5n7EnQdmwYondGxoqfU4SGnxtqtHyNZnOY--6wL3vaY64N6y9Ol6VaEdmhxlHtZ9zskpG0jWypn5ZEvlgg-2NnJYoPL6EOKqxhPhKOweQ9WNBLLfKlYzWiqy7TPWCchHeatrk45nHlTmBeMjmhFGIHpav47NDhdJbGqGb2EIjKAOQuctS37uI7xAHUHDrLIQQ~VOnhtRcc1h0JCcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>).</p>                            <h3 scrollto-destination=120122262 id="120122262" class="section-title" >2.8. Allele specific methylation analysis using single nucleotide polymorphism</h3>
<p>Single nucleotide polymorphisms (SNPs) from parental lines and hybrid were identified using BIS–SNP,<a reveal-id="dsy008-B42" data-open="dsy008-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> a variant caller from BS reads. For this, the mapped methylated reads were used as input files along with pigeonpea reference genome.<a reveal-id="dsy008-B32" data-open="dsy008-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> SNPs detection was performed using the following parameters: ‘-T BisulfiteGenotyper -stand_call_conf 20 -stand_emit_conf 0 -mmq 30 -mbq 17 -minConv 20’. Spurious SNP with quality score less than 20, more than 120x read coverage, strand bias more than—0.02, quality score by depth less than 1.0, mapping quality zero reads, fraction more than 0.1 and 2 SNPs within the 20 bp bin were further filtered out using VCF post process command.</p>                            <h2 scrollto-destination=120122264 id="120122264" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=120122265 id="120122265" class="section-title" >3.1. Genome wide methylation analysis in pigeonpea</h3>
<p>To understand the possible roles of epigenetic regulation and the dynamics of DNA methylation during hybrid formation in pigeonpea, we crossed 303 R (Fertile, pollen donor) and 11 A(sterile) lines. Since their F1 progeny restored fertility (<a reveal-id="dsy008-F1" data-open="dsy008-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>), we hypothesised that it could possibly partially attributed to the new epigenome resulting from the interaction between the two parental epigenomes. Thus, in-order to gain deep insights into the epigenome of the parental lines and their F1 hybrid, we deciphered their respective whole genome methylation pattern at single base resolution. For this high molecular weight genomic DNA was isolated from the leaves of 10 days after flowering (DAF) old plants from three different genotypes, namely 11 A, 303 R and F1 hybrid progeny of 11 A and 303 R. The DNA samples were subjected to bisulfite treatment to generate genomic libraries for high throughput bisulfite sequencing. The reads obtained were processed for removal of adapter and low quality sequences. About 80–100 million cleaned reads (in each of the three genotypes) were mapped on to the reference genome.<a reveal-id="dsy008-B27" data-open="dsy008-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a><sup>,</sup><a reveal-id="dsy008-B32" data-open="dsy008-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> We found that more than 80% reads got mapped on to the reference genome. Thus eventually, 46 408 326, 54 030 096 and 43 763 469 reads were obtained for 11 A line, 303 R line and their F1 hybrid progeny, respectively, that mapped uniquely to the reference genome and providing a coverage more than 20X. These were subjected to further analysis of cytosine methylation at single base resolution. Further, to calculate the frequency of cytosine conversion, reads were mapped to un-methylated chloroplast genome. We obtained a conversion rate of more than 99% in all the three genotypes, indicating high efficiency of cytosine conversion in our experiments (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008_supp.zip?Expires=2147483647&Signature=a5pxiGBiZuW6BLObg3zcGti-PLIrtW4lDnTZKtyw3m~A3SRkZcr0Y2ve6moKjtosqKCTHkcpfiWI1YiZ59ZYjagra1cBx2qHidDiS9YD76AyCdt-4~W-54Oj7mbqe~VnqVToy5n7EnQdmwYondGxoqfU4SGnxtqtHyNZnOY--6wL3vaY64N6y9Ol6VaEdmhxlHtZ9zskpG0jWypn5ZEvlgg-2NnJYoPL6EOKqxhPhKOweQ9WNBLLfKlYzWiqy7TPWCchHeatrk45nHlTmBeMjmhFGIHpav47NDhdJbGqGb2EIjKAOQuctS37uI7xAHUHDrLIQQ~VOnhtRcc1h0JCcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). </p>                        <a id="120122267" scrollto-destination="120122267"></a>
<div id="dsy008-F1" data-id="dsy008-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/m_dsy008f1.png?Expires=2147483647&Signature=VX92eY0VDU7FXi~ST7MUNEWTTLAmedX5PcUfmGmuletGfHn2kWS-RQJyshp0axsRxvFb4~I1Nh4Rmr99~L-8M4XK3dbLmQvkSBzSNl7fjPJaQB5uNHbnjjnDxuUcPgDbPKYwq5ORTiuC-V27w71Z7cVW8nLGFV0z~-QECBKd~y5T3X7qStbyq6kK8mo4UhOZooaVoi~I08xNQMRjcMB8mifL3TSsYdZi8CSNcEhIvlus0TdoT82p8D1OUR3E8bCozUW~p5oEawtwxjcs-ux4tXVFU4iJfHI2CkGV1ZtDZJ9OgabQnsaAP1YlF~dJ13ZUsBLtShi37h5oZ~MV1pFdNg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNA methylation profiles of pigeonpea inbred lines (11 A, 303 R) and their hybrid. (A) Acetocarmine stained pollens of 11A (sterile parent), 303R (fertile parent) and fertile hybrid. Note the density of pollen is less in 11A compared to 303R and hybrid. Also note viable pollens in 303R and hybrid.(B) Genome-wide relative proportions of CG (blue), CHG (red) and CHH (green) methyl cytosine in 11A, 303R and F1 hybrid. (C and D) Fractional methylation levels in the three lines of pigeonpea in CG, CHG and CHH contexts (B) and comparisons with published reports in Soybean, Rice, Populus, Arabidopsis and Brassica (D). (E) A circos plot representing chromosome wise distribution of mCG, mCHG, mCHH, gene, transposons density and FPKM (outer to inner) in F1 hybrid. Yellow to red color relates to lower to higher methylation density in a 100 Kb window (Colored figure is available in the online version of the article). Scale bars 50µm (A)." path-from-xml="dsy008f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122267/dsy008f1.tif" path-from-xml="dsy008f1.tif" target="_blank">View large</a><a section="120122267" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008f1.png?Expires=2147483647&Signature=BvDBAOvWd0TBJziJ3cWfAlBKoPRAhku8Rqv2Mzz-wYQPC8X4w7BkYWC4lPMJFHuk2cAoYLPv3d0ryCvMwKu5hmAaAINbPe7zlxQKw6w-FKgGVyWkPS6NbYdW5Gk9s7cpEmsjAakY1wVB95zJOU-eblU-NZI67XL7-aB2RDPNTT7JmtD9WEwLyGpVbjNa6PgcGqeoP2EJs-VObGVSOxyrT4RAnZq9QyuU4r8BBO0QIOX7Lgs3FO2LAgThpVxmarKNWy2PzhniOzUA7~RtpVYh2HzyC0G8C7iITD-dn-nFkR6QCUZnGVWswQAelZmZ-ZdsFrI2P4uknTdcmd8rLeeHEw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122267&ar=4939498&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy008f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNA methylation profiles of pigeonpea inbred lines (11 A, 303 R) and their hybrid. (A) Acetocarmine stained pollens of 11A (sterile parent), 303R (fertile parent) and fertile hybrid. Note the density of pollen is less in 11A compared to 303R and hybrid. Also note viable pollens in 303R and hybrid.<strong>(</strong>B<strong>)</strong> Genome-wide relative proportions of CG (blue), CHG (red) and CHH (green) methyl cytosine in 11A, 303R and F1 hybrid. <strong>(</strong>C and D<strong>)</strong> Fractional methylation levels in the three lines of pigeonpea in CG, CHG and CHH contexts (B) and comparisons with published reports in Soybean, Rice, Populus, Arabidopsis and Brassica (D). <strong>(</strong>E<strong>)</strong> A circos plot representing chromosome wise distribution of mCG, mCHG, mCHH, gene, transposons density and FPKM (outer to inner) in F1 hybrid. Yellow to red color relates to lower to higher methylation density in a 100 Kb window (Colored figure is available in the online version of the article). Scale bars 50µm (A).</p></div></div><div content-id="dsy008-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/m_dsy008f1.png?Expires=2147483647&Signature=VX92eY0VDU7FXi~ST7MUNEWTTLAmedX5PcUfmGmuletGfHn2kWS-RQJyshp0axsRxvFb4~I1Nh4Rmr99~L-8M4XK3dbLmQvkSBzSNl7fjPJaQB5uNHbnjjnDxuUcPgDbPKYwq5ORTiuC-V27w71Z7cVW8nLGFV0z~-QECBKd~y5T3X7qStbyq6kK8mo4UhOZooaVoi~I08xNQMRjcMB8mifL3TSsYdZi8CSNcEhIvlus0TdoT82p8D1OUR3E8bCozUW~p5oEawtwxjcs-ux4tXVFU4iJfHI2CkGV1ZtDZJ9OgabQnsaAP1YlF~dJ13ZUsBLtShi37h5oZ~MV1pFdNg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNA methylation profiles of pigeonpea inbred lines (11 A, 303 R) and their hybrid. (A) Acetocarmine stained pollens of 11A (sterile parent), 303R (fertile parent) and fertile hybrid. Note the density of pollen is less in 11A compared to 303R and hybrid. Also note viable pollens in 303R and hybrid.(B) Genome-wide relative proportions of CG (blue), CHG (red) and CHH (green) methyl cytosine in 11A, 303R and F1 hybrid. (C and D) Fractional methylation levels in the three lines of pigeonpea in CG, CHG and CHH contexts (B) and comparisons with published reports in Soybean, Rice, Populus, Arabidopsis and Brassica (D). (E) A circos plot representing chromosome wise distribution of mCG, mCHG, mCHH, gene, transposons density and FPKM (outer to inner) in F1 hybrid. Yellow to red color relates to lower to higher methylation density in a 100 Kb window (Colored figure is available in the online version of the article). Scale bars 50µm (A)." path-from-xml="dsy008f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122267/dsy008f1.tif" path-from-xml="dsy008f1.tif" target="_blank">View large</a><a section="120122267" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008f1.png?Expires=2147483647&Signature=BvDBAOvWd0TBJziJ3cWfAlBKoPRAhku8Rqv2Mzz-wYQPC8X4w7BkYWC4lPMJFHuk2cAoYLPv3d0ryCvMwKu5hmAaAINbPe7zlxQKw6w-FKgGVyWkPS6NbYdW5Gk9s7cpEmsjAakY1wVB95zJOU-eblU-NZI67XL7-aB2RDPNTT7JmtD9WEwLyGpVbjNa6PgcGqeoP2EJs-VObGVSOxyrT4RAnZq9QyuU4r8BBO0QIOX7Lgs3FO2LAgThpVxmarKNWy2PzhniOzUA7~RtpVYh2HzyC0G8C7iITD-dn-nFkR6QCUZnGVWswQAelZmZ-ZdsFrI2P4uknTdcmd8rLeeHEw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122267&ar=4939498&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy008f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNA methylation profiles of pigeonpea inbred lines (11 A, 303 R) and their hybrid. (A) Acetocarmine stained pollens of 11A (sterile parent), 303R (fertile parent) and fertile hybrid. Note the density of pollen is less in 11A compared to 303R and hybrid. Also note viable pollens in 303R and hybrid.<strong>(</strong>B<strong>)</strong> Genome-wide relative proportions of CG (blue), CHG (red) and CHH (green) methyl cytosine in 11A, 303R and F1 hybrid. <strong>(</strong>C and D<strong>)</strong> Fractional methylation levels in the three lines of pigeonpea in CG, CHG and CHH contexts (B) and comparisons with published reports in Soybean, Rice, Populus, Arabidopsis and Brassica (D). <strong>(</strong>E<strong>)</strong> A circos plot representing chromosome wise distribution of mCG, mCHG, mCHH, gene, transposons density and FPKM (outer to inner) in F1 hybrid. Yellow to red color relates to lower to higher methylation density in a 100 Kb window (Colored figure is available in the online version of the article). Scale bars 50µm (A).</p></div></div><p>We then deciphered the fractional proportion of DNA methylation and found that CG methylation was highest in 11 A (41.7%) while CHH methylation was highest in F1 hybrid (28.82%). Methylation in CHG context were similar in all the three genotypes (∼34%) (<a reveal-id="dsy008-F1" data-open="dsy008-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>). Highest mean methylation was observed in CG context (74%-79%), closely followed by CHG (56%–63%) while mean methylation in CHH context (10–15%) was lowest in both the parents and the hybrid (<a reveal-id="dsy008-F1" data-open="dsy008-F1" class="link link-reveal link-table xref-fig">Fig. 1C</a>). Comparative analysis of fractional methylation levels in pigeonpea, soybean, brassica, oryza and populus revealed that pigeonpea genome was hypermethylated compared with the other crops. Interestingly, CHH methylation levels in pigeonpea genome were significantly higher than those observed in soybean, brassica, oryza and populus<a reveal-id="dsy008-B25" data-open="dsy008-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a><sup>,</sup><a reveal-id="dsy008-B43" data-open="dsy008-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> (<a reveal-id="dsy008-F1" data-open="dsy008-F1" class="link link-reveal link-table xref-fig">Fig. 1D</a>). It has been documented that the DNA repeats in the genomes are highly methylated in symmetric contexts.<a reveal-id="dsy008-B9" data-open="dsy008-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsy008-B44" data-open="dsy008-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a><sup>,</sup><a reveal-id="dsy008-B45" data-open="dsy008-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> Pigeonpea geneome is highly repetitive (47%) compared with soyabean<a reveal-id="dsy008-B46" data-open="dsy008-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> and Arabidopsis.<a reveal-id="dsy008-B47" data-open="dsy008-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> Thus, such high levels of DNA methylation in pigeonpea may arise from the occcurance highly repetive genome.</p><p>Visualization of chromosome wide distribution of CG, CHG and CHH methylation revealed higher CG and CHG methylation levels compared with CHH while relatively even distribution pattern of CHH methylation (<a reveal-id="dsy008-F1" data-open="dsy008-F1" class="link link-reveal link-table xref-fig">Fig. 1E</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008_supp.zip?Expires=2147483647&Signature=a5pxiGBiZuW6BLObg3zcGti-PLIrtW4lDnTZKtyw3m~A3SRkZcr0Y2ve6moKjtosqKCTHkcpfiWI1YiZ59ZYjagra1cBx2qHidDiS9YD76AyCdt-4~W-54Oj7mbqe~VnqVToy5n7EnQdmwYondGxoqfU4SGnxtqtHyNZnOY--6wL3vaY64N6y9Ol6VaEdmhxlHtZ9zskpG0jWypn5ZEvlgg-2NnJYoPL6EOKqxhPhKOweQ9WNBLLfKlYzWiqy7TPWCchHeatrk45nHlTmBeMjmhFGIHpav47NDhdJbGqGb2EIjKAOQuctS37uI7xAHUHDrLIQQ~VOnhtRcc1h0JCcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1A</a></span> and B). High CG and CHG methylation were correlated with transposon rich regions across the genome (<a reveal-id="dsy008-F1" data-open="dsy008-F1" class="link link-reveal link-table xref-fig">Fig. 1E</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008_supp.zip?Expires=2147483647&Signature=a5pxiGBiZuW6BLObg3zcGti-PLIrtW4lDnTZKtyw3m~A3SRkZcr0Y2ve6moKjtosqKCTHkcpfiWI1YiZ59ZYjagra1cBx2qHidDiS9YD76AyCdt-4~W-54Oj7mbqe~VnqVToy5n7EnQdmwYondGxoqfU4SGnxtqtHyNZnOY--6wL3vaY64N6y9Ol6VaEdmhxlHtZ9zskpG0jWypn5ZEvlgg-2NnJYoPL6EOKqxhPhKOweQ9WNBLLfKlYzWiqy7TPWCchHeatrk45nHlTmBeMjmhFGIHpav47NDhdJbGqGb2EIjKAOQuctS37uI7xAHUHDrLIQQ~VOnhtRcc1h0JCcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1A</a></span> and B).</p>                            <h3 scrollto-destination=120122270 id="120122270" class="section-title" >3.2. Patterns of DNA methylation in genic and transposable elements</h3>
<p>To precisely understand the methylation patterns of the protein coding genes, genes with inserted TE elements were excluded from analysis. We calculated the methylation levels in the genic region and 2 kb upstream and downstream regions of the gene bodies in the two parental lines and their hybrid and found significant methylation differences between the three lines (Wilcoxon rank sum test, <em>P</em>-value &lt; 2.2e-16, <em>n</em> = 26,774). These differences reflect differential methylation patterns in the genic regions of parental lines and their hybrid. Furthermore, upon a detailed investigation of methylation pattern, we observed the occurrence of a single CG methylation peak within the gene body and two troughs near transcriptional start and termination sites in pigeonpea, invariably to earlier reports in other plants.<a reveal-id="dsy008-B43" data-open="dsy008-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a><sup>,</sup><a reveal-id="dsy008-B48" data-open="dsy008-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a><sup>,</sup><a reveal-id="dsy008-B49" data-open="dsy008-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> However, contrary to a recent report in rice,<a reveal-id="dsy008-B48" data-open="dsy008-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> we found over all low CG methylation levels within the coding regions compared with their flanking regions. A similar trend was observed for CHG methylation, however, the average CHG methylation both within the coding regions and their flanking regions were comparatively lower than CG methylation. CHH methylation however showed a different trend to both CG and CHG methylation in terms of a plateau instead of a single peak within the gene body. Overall CHH methylation both within the genic regions and their flanking regions were significantly attenuated than CG and CHG methylation. However, the hybrid exhibited mid parental range for CG and CHG but a higher CHH methylation compared with parental lines, suggesting altered CHH methylation patterns in the genic regions in pigeonpea hybrids (<a reveal-id="dsy008-F2" data-open="dsy008-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). </p>                        <a id="120122272" scrollto-destination="120122272"></a>
<div id="dsy008-F2" data-id="dsy008-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/m_dsy008f2.png?Expires=2147483647&Signature=PjRnRr8jJtFSFnVfkf0hTrE1Y0DfmpaVuz-p~VQPNHlGZSIcdjNYhT-HA9GhLs9Iy1nbPJALbzlGrImphv5~Opvs-kj49v1CIzYUvc4mryNEJjbrpgeLg7Yp8Fgicp-FSWClOOBKSjgE4n61HbrVfGB7JUjTa65zGBRczePCGNmd8tcFXyLknPJBbD3LhrHvReJsqUybtRyB7LYGQ0Fnxi74K5KS53mdUtHo8nOtI4mfwxML~KNOS5CgBybhIR50ttmKKxPqeWSaQ4LDjE08PjiGV3Hx3j9db5LmhbsAJqT8z6QeLlGeNriHSD7BGUyM9qfrHpj5CW4japudyT8E-w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of methyl cytosines in genes and transposable elements in pigeonpea. (A) Line plots showing methylation level in 2Kb flanking (upstream and downstream) and within the gene body in 11A, 303R and F1 hybrid in CG, CHG and CHH contexts. (B–D) Line plots showing methylation level in 2Kb flanking (upstream and downstream) and within the gene body of transposons- long terminal repeats (B); LINE (C) and DNA Elements (D) in 11A, 303R and F1 hybrid in CG, CHG and CHH context. Note higher mCs throughout the transposons compared to genes (Colored figure is available in the online version of the article)." path-from-xml="dsy008f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122272/dsy008f2.tif" path-from-xml="dsy008f2.tif" target="_blank">View large</a><a section="120122272" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008f2.png?Expires=2147483647&Signature=x-ZHdET~8ydcZfyPPdOYtOqsTM6~7fjLdfoEgfI53QfVLZtPXDAI9dQ4CswIv4CUzvQYK6WGqY~Lbw2JbPs~phd9TH5mzBaOht8oKPqH9PrBRSCDOasNBMdScajn6TBaGP7TwqpHhknkDFV5em-sk4y2YNLH1T3cwgtuEDCxSB5lOKcL1Zrt0-4Qw~dOHS9dUeupmXV5HM60k36Ew9kRBj5wLy-p39PAsm7wrUR2J9mpKSNrlm~aKj8KKPPBbkHXCU1IHH-AknUTWec2vwzPfRoJWu18eRLy4TtsQcS4Lpf8YQLaJgLxVtb35GVUZLhKX6lnFzzHaFRLOF9Md-pXog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122272&ar=4939498&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy008f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of methyl cytosines in genes and transposable elements in pigeonpea. <strong>(</strong>A<strong>)</strong> Line plots showing methylation level in 2Kb flanking (upstream and downstream) and within the gene body in 11A, 303R and F1 hybrid in CG, CHG and CHH contexts. <strong>(</strong>B–D<strong>)</strong> Line plots showing methylation level in 2Kb flanking (upstream and downstream) and within the gene body of transposons- long terminal repeats (B); LINE (C) and DNA Elements (D) in 11A, 303R and F1 hybrid in CG, CHG and CHH context. Note higher mCs throughout the transposons compared to genes (Colored figure is available in the online version of the article).</p></div></div><div content-id="dsy008-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/m_dsy008f2.png?Expires=2147483647&Signature=PjRnRr8jJtFSFnVfkf0hTrE1Y0DfmpaVuz-p~VQPNHlGZSIcdjNYhT-HA9GhLs9Iy1nbPJALbzlGrImphv5~Opvs-kj49v1CIzYUvc4mryNEJjbrpgeLg7Yp8Fgicp-FSWClOOBKSjgE4n61HbrVfGB7JUjTa65zGBRczePCGNmd8tcFXyLknPJBbD3LhrHvReJsqUybtRyB7LYGQ0Fnxi74K5KS53mdUtHo8nOtI4mfwxML~KNOS5CgBybhIR50ttmKKxPqeWSaQ4LDjE08PjiGV3Hx3j9db5LmhbsAJqT8z6QeLlGeNriHSD7BGUyM9qfrHpj5CW4japudyT8E-w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of methyl cytosines in genes and transposable elements in pigeonpea. (A) Line plots showing methylation level in 2Kb flanking (upstream and downstream) and within the gene body in 11A, 303R and F1 hybrid in CG, CHG and CHH contexts. (B–D) Line plots showing methylation level in 2Kb flanking (upstream and downstream) and within the gene body of transposons- long terminal repeats (B); LINE (C) and DNA Elements (D) in 11A, 303R and F1 hybrid in CG, CHG and CHH context. Note higher mCs throughout the transposons compared to genes (Colored figure is available in the online version of the article)." path-from-xml="dsy008f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122272/dsy008f2.tif" path-from-xml="dsy008f2.tif" target="_blank">View large</a><a section="120122272" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008f2.png?Expires=2147483647&Signature=x-ZHdET~8ydcZfyPPdOYtOqsTM6~7fjLdfoEgfI53QfVLZtPXDAI9dQ4CswIv4CUzvQYK6WGqY~Lbw2JbPs~phd9TH5mzBaOht8oKPqH9PrBRSCDOasNBMdScajn6TBaGP7TwqpHhknkDFV5em-sk4y2YNLH1T3cwgtuEDCxSB5lOKcL1Zrt0-4Qw~dOHS9dUeupmXV5HM60k36Ew9kRBj5wLy-p39PAsm7wrUR2J9mpKSNrlm~aKj8KKPPBbkHXCU1IHH-AknUTWec2vwzPfRoJWu18eRLy4TtsQcS4Lpf8YQLaJgLxVtb35GVUZLhKX6lnFzzHaFRLOF9Md-pXog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122272&ar=4939498&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy008f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of methyl cytosines in genes and transposable elements in pigeonpea. <strong>(</strong>A<strong>)</strong> Line plots showing methylation level in 2Kb flanking (upstream and downstream) and within the gene body in 11A, 303R and F1 hybrid in CG, CHG and CHH contexts. <strong>(</strong>B–D<strong>)</strong> Line plots showing methylation level in 2Kb flanking (upstream and downstream) and within the gene body of transposons- long terminal repeats (B); LINE (C) and DNA Elements (D) in 11A, 303R and F1 hybrid in CG, CHG and CHH context. Note higher mCs throughout the transposons compared to genes (Colored figure is available in the online version of the article).</p></div></div><p>Eukaryote genomes are abundant in TEs, also called mobile elements. TEs are structurally divergent, epigenetically controlled and are mostly involved in genome evolution and gene activity regulation. Thus, we sought to gain a comprehensive insight of the influence of DNA methylation on the transposable elements within the pigeonpea genome. Since no complete dataset on pigeonpea TEs is available, we used repeat masker to identify and classify them. TEs shorter than 100 bp were excluded from analysis. In total, we found 105 733 TEs, which we classified into retro transposons and DNA elements. We next calculated their CG, CHG and CHH methylation levels in all the three genotypes and found that irrespective of their categories, methylation was highly enriched in all the contexts in TEs compared with the gene body. However, interestingly, we found relatively lower CG and CHG methylation levels in the TEs flanking regions than those of the genic regions. By comparing the methylation levels in long terminal repeats within the three genotypes, we found relatively higher methylation levels in F1 hybrid in all the three contexts, compared with the parental lines except for CG methylation in flanking regions where 11 A exhibited higher methylation levels (<a reveal-id="dsy008-F2" data-open="dsy008-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). While, in the case of LINE elements, 11 A displayed hyper methylation compared with the F1 hybrid and 303 R in CG and CHG contexts (<a reveal-id="dsy008-F2" data-open="dsy008-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>). DNA elements were relatively hypermethylated than the class 1 transposons. F1 hybrid displayed uniformly higher methylation irrespective of the mC contexts (<a reveal-id="dsy008-F2" data-open="dsy008-F2" class="link link-reveal link-table xref-fig">Fig. 2D</a>). Overall we observed comparatively lower methylation levels in 303 R, compared with 11 A and F1 hybrid (<a reveal-id="dsy008-F2" data-open="dsy008-F2" class="link link-reveal link-table xref-fig">Fig. 2B–D</a>), suggesting differences in the regulatory mechanisms of TE silencing via methylation in pigeonpea.</p>                            <h3 scrollto-destination=120122274 id="120122274" class="section-title" >3.3. Fertile and sterile pigeonpea inbred lines display locus specific differences in cytosine methylation</h3>
<p>Earlier work in crop plants such as rice and maize have documented differences in methylation levels between a male sterile plant and a fertile plant and revealed hyper-methylation in sterile lines compared with fertile lines.<a reveal-id="dsy008-B50" data-open="dsy008-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a><sup>,</sup><a reveal-id="dsy008-B51" data-open="dsy008-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> This suggests differential cytosine methylation could possibly influence fertility and sterility in plants by regulating gene expression. Therefore to investigate whether pigeonpea 11 A and 303 R inbreds exhibit differences in their methylation levels, we compared the methylome of the two lines. Several studies have documented that viewing differential methylation in windows of multiple cytosines seems to reflect more functional consequences rather than viewing at single sites.<a reveal-id="dsy008-B52" data-open="dsy008-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a><sup>,</sup><a reveal-id="dsy008-B53" data-open="dsy008-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> Also it is known that CG, CHG and CHH methylation levels are maintained through different regulatory mechanisms.<a reveal-id="dsy008-B6" data-open="dsy008-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> Thus, to identify DMRs between 11 A and 303 R lines, we decided to investigate and categorize differential methylation for all the three contexts, CG, CHG and CHH, separately. We averaged the DNA methylation for all the three contexts independently across non-overlapping 100 bp regions (windows), as done previously.<a reveal-id="dsy008-B54" data-open="dsy008-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> In-order to categorize a DMR window as CG, CGH or CHH, different conditions were applied. To assign a DMR to CG or CHG category, windows having at-least three cytosines with a minimum of 10X coverage were used. A minimum difference of 20% in CG and in CHG methylation each was chosen. Previously, it has been reported that CHH methylation patterns appeared to be more variable than CG and CHG.<a reveal-id="dsy008-B54" data-open="dsy008-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> Since CHH methylation sites are much abundant in genomes and still, the CHH methylation levels were observed to be much lower than the CG and CHG methylation levels (<a reveal-id="dsy008-F1" data-open="dsy008-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>), different criteria were applied for CHH DMR identification. To assign a DMR to CHH category, each window had at-least eight cytosines with minimum of 10X coverage and a minimum of 25% difference in CHH methylation. Eventually, we identified much more CG DMRs (11,300) than CHG (6988) and CHH (8061) DMRs between 303 R and 11 A parental lines (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008_supp.zip?Expires=2147483647&Signature=a5pxiGBiZuW6BLObg3zcGti-PLIrtW4lDnTZKtyw3m~A3SRkZcr0Y2ve6moKjtosqKCTHkcpfiWI1YiZ59ZYjagra1cBx2qHidDiS9YD76AyCdt-4~W-54Oj7mbqe~VnqVToy5n7EnQdmwYondGxoqfU4SGnxtqtHyNZnOY--6wL3vaY64N6y9Ol6VaEdmhxlHtZ9zskpG0jWypn5ZEvlgg-2NnJYoPL6EOKqxhPhKOweQ9WNBLLfKlYzWiqy7TPWCchHeatrk45nHlTmBeMjmhFGIHpav47NDhdJbGqGb2EIjKAOQuctS37uI7xAHUHDrLIQQ~VOnhtRcc1h0JCcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). We also found that the differential methylation between the two parents was more enriched in upstream flanking sequences of the genes than the down stream regions (<a reveal-id="dsy008-F3" data-open="dsy008-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). We next assessed relative methylation percentages of the obtained DMRs between the 11 A and 303 R parents (11 A vs. 303 R or Sterile vs. fertile) and found enrichment for hypermethylated CHH DMRs and hypomethylated CG DMRs (<a reveal-id="dsy008-F3" data-open="dsy008-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). This suggests that in 11 A line, genes involved in regulation of CG methylation are less active than the RdDM pathway genes and CMT2 that are involved in the regulation CHH methylation (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008_supp.zip?Expires=2147483647&Signature=a5pxiGBiZuW6BLObg3zcGti-PLIrtW4lDnTZKtyw3m~A3SRkZcr0Y2ve6moKjtosqKCTHkcpfiWI1YiZ59ZYjagra1cBx2qHidDiS9YD76AyCdt-4~W-54Oj7mbqe~VnqVToy5n7EnQdmwYondGxoqfU4SGnxtqtHyNZnOY--6wL3vaY64N6y9Ol6VaEdmhxlHtZ9zskpG0jWypn5ZEvlgg-2NnJYoPL6EOKqxhPhKOweQ9WNBLLfKlYzWiqy7TPWCchHeatrk45nHlTmBeMjmhFGIHpav47NDhdJbGqGb2EIjKAOQuctS37uI7xAHUHDrLIQQ~VOnhtRcc1h0JCcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). </p>                        <a id="120122276" scrollto-destination="120122276"></a>
<div id="dsy008-F3" data-id="dsy008-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/m_dsy008f3.png?Expires=2147483647&Signature=aC-tB2WOGq-cb7ce4O~KMQc3TY~ME0vFaTHgrrAlPhOEBFP5YXvmVF5U3ZDvuopU2RymzV7FCFnw1C0kk2dN3io8tfjCNvYQQHrD1YC~gWrqzApJKMVAk2z8OBofnLc5wy3hHTSARYd~MEF~Q~vxKg2I-AMv04Ig5I8EQewePHlDHaTCoYbQefhPnMVKdHPskrmdK5SA18oo~CphIDaioejCqwo-rjt-feXclf7IDVvpnGZ8pVkO4yoODlEVB4K-F086fRIIjq9IpS90jL567PNreSirqwSpplAPf7h2GqagFBJ7t0Ki5xneP8fN-yYw5NlfC6ihEjecCpysuBe~hA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparative analyses of DMRs between the inbred lines and their hybrid. (A) Pie charts showing the distribution of DMRs in the different genomic regions in 11A vs. 303R, 11A vs. hybrid and 303R vs. hybrid. (B) A bar plot showing number of hypo and hypermethylated CG, CHG and CHH DMRs in 11A vs. 303R, 11A vs. hybrid and 303R vs. hybrid. (C) A heat map showing enrichment of DMR associated genes in 11A vs. 303R, 11A vs. hybrid and 303R vs. hybrid. Color code represents low to high P-values (0.05 to 8.09e-10) (Colored figure is available in the online version of the article)." path-from-xml="dsy008f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122276/dsy008f3.tif" path-from-xml="dsy008f3.tif" target="_blank">View large</a><a section="120122276" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008f3.png?Expires=2147483647&Signature=Ko91F9ISNLzSJ70K1dMCpIGG1SGHR~IoyczD0EH0a5v961ulrnTjj420fGydlLtoM9Z3gidCbY51DWRIgRS~F16YEMjuGJERg97MIJT3GXJ3NiDKre2Er5meyz7l0ripcddRlBjtdnI~OyBy8eV2Cbesq~cbCzxkQSttjoMy6x2l2apMPRBz2SK7ePf7yfcCknEiCUEYN-BLZLYxURt5Yftyeomx8qsAJeqVhZ-TPoQc8hIqU90koxRXH1dnJl2lKXL4FIIS3QbtJCBl5QscdTZKx7gAjTvnKjtHM2saQso0zwid2868Bhj~GsmBncmdqqGqz~PYJpYjRHbgXTJOvQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122276&ar=4939498&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy008f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative analyses of DMRs between the inbred lines and their hybrid. <strong>(A)</strong> Pie charts showing the distribution of DMRs in the different genomic regions in 11A vs. 303R, 11A vs. hybrid and 303R vs. hybrid. (B) A bar plot showing number of hypo and hypermethylated CG, CHG and CHH DMRs in 11A vs. 303R, 11A vs. hybrid and 303R vs. hybrid. <strong>(</strong>C<strong>)</strong> A heat map showing enrichment of DMR associated genes in 11A vs. 303R, 11A vs. hybrid and 303R vs. hybrid. Color code represents low to high <em>P</em>-values (0.05 to 8.09e-10) (Colored figure is available in the online version of the article).</p></div></div><div content-id="dsy008-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/m_dsy008f3.png?Expires=2147483647&Signature=aC-tB2WOGq-cb7ce4O~KMQc3TY~ME0vFaTHgrrAlPhOEBFP5YXvmVF5U3ZDvuopU2RymzV7FCFnw1C0kk2dN3io8tfjCNvYQQHrD1YC~gWrqzApJKMVAk2z8OBofnLc5wy3hHTSARYd~MEF~Q~vxKg2I-AMv04Ig5I8EQewePHlDHaTCoYbQefhPnMVKdHPskrmdK5SA18oo~CphIDaioejCqwo-rjt-feXclf7IDVvpnGZ8pVkO4yoODlEVB4K-F086fRIIjq9IpS90jL567PNreSirqwSpplAPf7h2GqagFBJ7t0Ki5xneP8fN-yYw5NlfC6ihEjecCpysuBe~hA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparative analyses of DMRs between the inbred lines and their hybrid. (A) Pie charts showing the distribution of DMRs in the different genomic regions in 11A vs. 303R, 11A vs. hybrid and 303R vs. hybrid. (B) A bar plot showing number of hypo and hypermethylated CG, CHG and CHH DMRs in 11A vs. 303R, 11A vs. hybrid and 303R vs. hybrid. (C) A heat map showing enrichment of DMR associated genes in 11A vs. 303R, 11A vs. hybrid and 303R vs. hybrid. Color code represents low to high P-values (0.05 to 8.09e-10) (Colored figure is available in the online version of the article)." path-from-xml="dsy008f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122276/dsy008f3.tif" path-from-xml="dsy008f3.tif" target="_blank">View large</a><a section="120122276" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008f3.png?Expires=2147483647&Signature=Ko91F9ISNLzSJ70K1dMCpIGG1SGHR~IoyczD0EH0a5v961ulrnTjj420fGydlLtoM9Z3gidCbY51DWRIgRS~F16YEMjuGJERg97MIJT3GXJ3NiDKre2Er5meyz7l0ripcddRlBjtdnI~OyBy8eV2Cbesq~cbCzxkQSttjoMy6x2l2apMPRBz2SK7ePf7yfcCknEiCUEYN-BLZLYxURt5Yftyeomx8qsAJeqVhZ-TPoQc8hIqU90koxRXH1dnJl2lKXL4FIIS3QbtJCBl5QscdTZKx7gAjTvnKjtHM2saQso0zwid2868Bhj~GsmBncmdqqGqz~PYJpYjRHbgXTJOvQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122276&ar=4939498&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy008f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative analyses of DMRs between the inbred lines and their hybrid. <strong>(A)</strong> Pie charts showing the distribution of DMRs in the different genomic regions in 11A vs. 303R, 11A vs. hybrid and 303R vs. hybrid. (B) A bar plot showing number of hypo and hypermethylated CG, CHG and CHH DMRs in 11A vs. 303R, 11A vs. hybrid and 303R vs. hybrid. <strong>(</strong>C<strong>)</strong> A heat map showing enrichment of DMR associated genes in 11A vs. 303R, 11A vs. hybrid and 303R vs. hybrid. Color code represents low to high <em>P</em>-values (0.05 to 8.09e-10) (Colored figure is available in the online version of the article).</p></div></div><p>Interaction of two dissimilar parental epigenomes can result in an altered methylome in their hybrid progeny. Since fertility is restored in the F1 hybrid and earlier studies in <em>Arabidopsis</em> showed higher methylation levels in hybrid compared with parental lines,<a reveal-id="dsy008-B15" data-open="dsy008-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsy008-B20" data-open="dsy008-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> we therefore investigated whether there is differential methylation between individual parental lines and the fertile hybrid (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008_supp.zip?Expires=2147483647&Signature=a5pxiGBiZuW6BLObg3zcGti-PLIrtW4lDnTZKtyw3m~A3SRkZcr0Y2ve6moKjtosqKCTHkcpfiWI1YiZ59ZYjagra1cBx2qHidDiS9YD76AyCdt-4~W-54Oj7mbqe~VnqVToy5n7EnQdmwYondGxoqfU4SGnxtqtHyNZnOY--6wL3vaY64N6y9Ol6VaEdmhxlHtZ9zskpG0jWypn5ZEvlgg-2NnJYoPL6EOKqxhPhKOweQ9WNBLLfKlYzWiqy7TPWCchHeatrk45nHlTmBeMjmhFGIHpav47NDhdJbGqGb2EIjKAOQuctS37uI7xAHUHDrLIQQ~VOnhtRcc1h0JCcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Consistent with previous studies, we found a relative increase in the percentage of methylated cytosines in CG, CHG and CHH DMRs in the hybrid compared with parental lines (<a reveal-id="dsy008-F3" data-open="dsy008-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>) indicative of occurrence of non-additive methylation in pigeonpea hybrids. We randomly chose hyper and hypo methylated DMRs between sterile parent and F1 hybrid in intergenic region and coding regions. Analysis of their methylation levels using bisulfite sequencing PCR revealed relatively higher methylation levels in intergenic region associated DMR than in the coding region associated DMR (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008_supp.zip?Expires=2147483647&Signature=a5pxiGBiZuW6BLObg3zcGti-PLIrtW4lDnTZKtyw3m~A3SRkZcr0Y2ve6moKjtosqKCTHkcpfiWI1YiZ59ZYjagra1cBx2qHidDiS9YD76AyCdt-4~W-54Oj7mbqe~VnqVToy5n7EnQdmwYondGxoqfU4SGnxtqtHyNZnOY--6wL3vaY64N6y9Ol6VaEdmhxlHtZ9zskpG0jWypn5ZEvlgg-2NnJYoPL6EOKqxhPhKOweQ9WNBLLfKlYzWiqy7TPWCchHeatrk45nHlTmBeMjmhFGIHpav47NDhdJbGqGb2EIjKAOQuctS37uI7xAHUHDrLIQQ~VOnhtRcc1h0JCcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>) consistent with our <em>in-silico</em> data.</p><p>In-order to assess the functional categories of the genes associated with the DMRs between parents and hybrids and to investigate whether DMRs were associated with genes involved in DNA methylation, epigenetic processes or fertility regulation, gene ontology enrichment analysis was performed. We found genes ontology terms for regulation of flowering time, DNA methylation, regulation of gene silencing by miRNA, developmental processes, Histone H4 K4 methylation, DNA methyltransferase, transcription factors, proton transporting ATP synthase activity, regulation of post transcriptional gene silencing were enriched (<a reveal-id="dsy008-F3" data-open="dsy008-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a>). Interestingly, we also observed a strong enrichment of DMRs in gene is associated with aldehyde dehydrogenase activity. A previous study have reported that RF2 encoded aldehyde dehydrogenase activity is required to restore male fertility in maize and also plays a role in anther development.<a reveal-id="dsy008-B55" data-open="dsy008-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> Our results thus suggest a possible correlation between differential methylation and fertility or sterility in 11 A, 303 R pigeonpea inbred and their hybrid. Also, genes involved in cellular processes, biosynthetic processes and ATP binding were significantly enriched (Bonferroni corrected <em>P</em>-value &lt; 0.05; <a reveal-id="dsy008-F3" data-open="dsy008-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008_supp.zip?Expires=2147483647&Signature=a5pxiGBiZuW6BLObg3zcGti-PLIrtW4lDnTZKtyw3m~A3SRkZcr0Y2ve6moKjtosqKCTHkcpfiWI1YiZ59ZYjagra1cBx2qHidDiS9YD76AyCdt-4~W-54Oj7mbqe~VnqVToy5n7EnQdmwYondGxoqfU4SGnxtqtHyNZnOY--6wL3vaY64N6y9Ol6VaEdmhxlHtZ9zskpG0jWypn5ZEvlgg-2NnJYoPL6EOKqxhPhKOweQ9WNBLLfKlYzWiqy7TPWCchHeatrk45nHlTmBeMjmhFGIHpav47NDhdJbGqGb2EIjKAOQuctS37uI7xAHUHDrLIQQ~VOnhtRcc1h0JCcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> S1).</p><p>Overall, our results indicate that differential methylation could be involved in mediating and regulating several cellular and developmental processes in pigeonpea.</p>                            <h3 scrollto-destination=120122280 id="120122280" class="section-title" >3.4. Trans-chromosomal methylation events alter pigeonpea hybrid methylome</h3>
<p>Our results so far show altered methylome in pigeonpea hybrid compared with parents. The changes in hybrid methylome occur at DMRs between the parents and are mediated by either increase or decrease in DNA methylation termed as non-additive methylation.<a reveal-id="dsy008-B15" data-open="dsy008-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsy008-B20" data-open="dsy008-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> This increase or decrease occurs via altering methylation status of one parental allele to resemble that of the other parental allele, events known as trans-chromosomal methylation (TCM) and trans-chromosomal demethylation (TCdM).<a reveal-id="dsy008-B15" data-open="dsy008-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> We investigated non-additive methylation in F1 hybrid to understand its methylome dynamics. To understand this mechanism, DMRs between the two parental lines were identified and mapped onto the F1 hybrid genome. Average methylation of the mapped loci was calculated in F1 hybrid. Mean parental value (MPV) of the DMRs between parents were obtained and compared with observed methylation in hybrid. Loci having methylation difference of atleast 20% were considered as non-additive methylation sites. Subsequently, we found non-additive methylation in 28% DMRs suggesting methylation interaction at these regions in F1 hybrid. We further analyzed the nature of this non-additive methylation and observed patterns of Trans-chromosomal methylation at 3,911 DMRs and Trans-chromosomal demethylation in 5,377 DMRs (<a reveal-id="dsy008-F4" data-open="dsy008-F4" class="link link-reveal link-table xref-fig">Fig. 4A and B</a>). By investigating the genomic features of the DMRs showing non-additive methylation, we found a higher proportion resided in TE (25%) as compared with the genic regions (15%). These results are in accord with previous findings revealing the roles of TE in genome stability and their regulation by methylation. Furthermore we found that in all the three contexts, the methylome of F1 hybrid resembled more of the 303 R fertile parent than the 11 A sterile. CG and CHG methylation levels of 303 R fertile parent and hybrid were much higher than the 11 A sterile parent. In contrast, CHH methylation levels of 303 R fertile parent and hybrid were lower than that of the 11 A sterile parent (<a reveal-id="dsy008-F4" data-open="dsy008-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). Additionally, to decipher the number of direct TcM and TcDM non additive mC clusters, we used an alternative approach of SNPs as described previously.<a reveal-id="dsy008-B15" data-open="dsy008-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> For this, we tracked 75,340 mC clusters of parental origin and followed the methylation pattern of parental alleles with differences in methylation level (3,222 out of 75,340) in hybrid using SNPs. We observed 27% mC clusters were stably inherited in hybrids indicative of additive methylation. Among the clusters showing non additive methylation, we observed, TCM events in 18.5% cases where methylation of epi-alleles from low parent increased in the hybrid, whereas methylation of high parent epi-alleles remained unchanged in the hybrid, resulting in an over all increase in methylation in the hybrid. While in 24.1% clusters, TCdM was observed where methylation of the epi-allele from low parent remained unchanged but methylation of the epi-allele from high parent decreased in hybrid, resulting in an overall decrease in methylation in hybrid (<a reveal-id="dsy008-F4" data-open="dsy008-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a>). </p>                        <a id="120122282" scrollto-destination="120122282"></a>
<div id="dsy008-F4" data-id="dsy008-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/m_dsy008f4.png?Expires=2147483647&Signature=Otxzm1IcA3SRPNCdaA3R5HcfXMojT2PaHVdKCvlezjoxODvt0~XocT7zrIJdHtNj13r4eCByVxGRyjCjYVTOjqEhLsEsO9jneDFLTyCM7xXDQZdA8tapz-YzErWvF1Bh89N3lQOghWMWv2ySu7jlZH50UQxpVNFPWYyKuCwv-t9ARb08QaUNkiUk5FL6C7mc~CUEn3mnibpNOg7P4PRY81023A~OoFq-jTtrWgyteW8KEu-pCinM-ziE5wGlto9nj62x7vvYMg5gTSmWeZtrAstqOnsBudz--SdC7thVK8dtSjvfbtDvc7tcpNb-2zhaDU4dcT3qa-5FlKyPB2l8Bg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Heat maps showing the methylation status of DMRs of the parents and methylation pattern at those loci in the F1 hybrid. Note in all the mCs context, methylation patterns of F1 hybrid are similar to the fertile parent (303 R) and different to the sterile parent. (B) Bar plot showing non-additive methylation in F1 hybrid. Note increased average methylation in hybrid than the MPV in the upper graph indicating the occurrence of TCM. Similarly, note a decreased average methylation in hybrid than the MPV in the lower upper graph indicating TCdM. (C) Bar plots showing changes in parental allelic methylations in the hybrid. Majority of the non-additive methylation occurred through TCdM (Blue). Red bar represent TcM. Methylation levels of most parental alleles remained unchanged in hybrid (gray) (Colored figure is available in the online version of the article). HP: high parent; LP: low parent." path-from-xml="dsy008f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122282/dsy008f4.tif" path-from-xml="dsy008f4.tif" target="_blank">View large</a><a section="120122282" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008f4.png?Expires=2147483647&Signature=M0QtkWMX6CgnowwrMZ~VhJf5ZdljRqQ-iLXg3mSV7d9HognZagUgqNCY6mKumK9DCGIUL8X3ssp--rDPcle3XKzrDmWYbUfPfBIsFTFdE-JqXlJMqPW5JxhTIFLSKb0L6gzJb6th6wYaqPWSB~CEvoHVCxGhgz5rb77wpKwvWe7Yj0tfDcvfCZVya0OJDZWb9LWABYL48ifp8hMknM2QGlGT-EUb4i1H-noxnfMwFryicTfNIeBcZEVjo8rXhy6qAzKJ5gFjlAuCWrSIB6IqS6t9khIvqVzu4BBcyS803QMp518rvU431rlA~cvZjSVc91-MB1orBiSedvHFyWbZQg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122282&ar=4939498&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy008f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A<strong>)</strong> Heat maps showing the methylation status of DMRs of the parents and methylation pattern at those loci in the F1 hybrid. Note in all the mCs context, methylation patterns of F1 hybrid are similar to the fertile parent (303 R) and different to the sterile parent. <strong>(</strong>B<strong>)</strong> Bar plot showing non-additive methylation in F1 hybrid. Note increased average methylation in hybrid than the MPV in the upper graph indicating the occurrence of TCM. Similarly, note a decreased average methylation in hybrid than the MPV in the lower upper graph indicating TCdM. (C) Bar plots showing changes in parental allelic methylations in the hybrid. Majority of the non-additive methylation occurred through TCdM (Blue). Red bar represent TcM. Methylation levels of most parental alleles remained unchanged in hybrid (gray) (Colored figure is available in the online version of the article). HP: high parent; LP: low parent.</p></div></div><div content-id="dsy008-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/m_dsy008f4.png?Expires=2147483647&Signature=Otxzm1IcA3SRPNCdaA3R5HcfXMojT2PaHVdKCvlezjoxODvt0~XocT7zrIJdHtNj13r4eCByVxGRyjCjYVTOjqEhLsEsO9jneDFLTyCM7xXDQZdA8tapz-YzErWvF1Bh89N3lQOghWMWv2ySu7jlZH50UQxpVNFPWYyKuCwv-t9ARb08QaUNkiUk5FL6C7mc~CUEn3mnibpNOg7P4PRY81023A~OoFq-jTtrWgyteW8KEu-pCinM-ziE5wGlto9nj62x7vvYMg5gTSmWeZtrAstqOnsBudz--SdC7thVK8dtSjvfbtDvc7tcpNb-2zhaDU4dcT3qa-5FlKyPB2l8Bg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Heat maps showing the methylation status of DMRs of the parents and methylation pattern at those loci in the F1 hybrid. Note in all the mCs context, methylation patterns of F1 hybrid are similar to the fertile parent (303 R) and different to the sterile parent. (B) Bar plot showing non-additive methylation in F1 hybrid. Note increased average methylation in hybrid than the MPV in the upper graph indicating the occurrence of TCM. Similarly, note a decreased average methylation in hybrid than the MPV in the lower upper graph indicating TCdM. (C) Bar plots showing changes in parental allelic methylations in the hybrid. Majority of the non-additive methylation occurred through TCdM (Blue). Red bar represent TcM. Methylation levels of most parental alleles remained unchanged in hybrid (gray) (Colored figure is available in the online version of the article). HP: high parent; LP: low parent." path-from-xml="dsy008f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122282/dsy008f4.tif" path-from-xml="dsy008f4.tif" target="_blank">View large</a><a section="120122282" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008f4.png?Expires=2147483647&Signature=M0QtkWMX6CgnowwrMZ~VhJf5ZdljRqQ-iLXg3mSV7d9HognZagUgqNCY6mKumK9DCGIUL8X3ssp--rDPcle3XKzrDmWYbUfPfBIsFTFdE-JqXlJMqPW5JxhTIFLSKb0L6gzJb6th6wYaqPWSB~CEvoHVCxGhgz5rb77wpKwvWe7Yj0tfDcvfCZVya0OJDZWb9LWABYL48ifp8hMknM2QGlGT-EUb4i1H-noxnfMwFryicTfNIeBcZEVjo8rXhy6qAzKJ5gFjlAuCWrSIB6IqS6t9khIvqVzu4BBcyS803QMp518rvU431rlA~cvZjSVc91-MB1orBiSedvHFyWbZQg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122282&ar=4939498&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy008f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A<strong>)</strong> Heat maps showing the methylation status of DMRs of the parents and methylation pattern at those loci in the F1 hybrid. Note in all the mCs context, methylation patterns of F1 hybrid are similar to the fertile parent (303 R) and different to the sterile parent. <strong>(</strong>B<strong>)</strong> Bar plot showing non-additive methylation in F1 hybrid. Note increased average methylation in hybrid than the MPV in the upper graph indicating the occurrence of TCM. Similarly, note a decreased average methylation in hybrid than the MPV in the lower upper graph indicating TCdM. (C) Bar plots showing changes in parental allelic methylations in the hybrid. Majority of the non-additive methylation occurred through TCdM (Blue). Red bar represent TcM. Methylation levels of most parental alleles remained unchanged in hybrid (gray) (Colored figure is available in the online version of the article). HP: high parent; LP: low parent.</p></div></div>                            <h3 scrollto-destination=120122283 id="120122283" class="section-title" >3.5. Transcriptome profiling in pigeonpea</h3>
<p>In order to assess the transcript abundance and differential gene expression between the three genotypes, their whole transcriptome profiling was performed by RNA sequencing. We first compared the transcription between different genotypes (11 A vs. 303 R, 11 A vs. F1 hybrid and 303 R vs. F1 hybrid) to compute genes that were differentially expressed between these lines. The transcripts showing a minimum of 2-fold change and <em>P</em>-value &lt;0.05 were considered differentially expressed. We found 1,621 (460 up, 1,161 down), 1,790 (1,052 up, 738 down) and 1,380 (963 up, 417 down) genes that showed differential expression in 11 A vs. 303 R, 11 A vs. hybrid and 303 R vs. hybrid, respectively. We found that in all the three cases of comparison, the differentially expressed genes were well represented in gene categories like pentatricopeptide repeats (PPRs)- associated proteins involved in fertility restoration,<a reveal-id="dsy008-B56" data-open="dsy008-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> methylases, demethylases and histone modifying genes (that play a role in epigenetic regulation) and flowering-associated genes. We performed gene ontology enrichment analysis to identify functional categories of significantly enriched genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008_supp.zip?Expires=2147483647&Signature=a5pxiGBiZuW6BLObg3zcGti-PLIrtW4lDnTZKtyw3m~A3SRkZcr0Y2ve6moKjtosqKCTHkcpfiWI1YiZ59ZYjagra1cBx2qHidDiS9YD76AyCdt-4~W-54Oj7mbqe~VnqVToy5n7EnQdmwYondGxoqfU4SGnxtqtHyNZnOY--6wL3vaY64N6y9Ol6VaEdmhxlHtZ9zskpG0jWypn5ZEvlgg-2NnJYoPL6EOKqxhPhKOweQ9WNBLLfKlYzWiqy7TPWCchHeatrk45nHlTmBeMjmhFGIHpav47NDhdJbGqGb2EIjKAOQuctS37uI7xAHUHDrLIQQ~VOnhtRcc1h0JCcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S4–S6</a></span>). Interestingly, we found that gene ontology terms included reproductive processes, developmental processes, protein–DNA complexes and nucleic acid binding proteins that were significantly enriched (Fisher’s exact test <em>P</em>-value &lt;0.05) in all the three comparative scenarios 11 A vs. 303 R, 11 A vs. hybrid and 303 R vs. hybrid.</p>                            <h3 scrollto-destination=120122285 id="120122285" class="section-title" >3.6. Correlation between DMRs and gene expression in pigeonpea inbred and hybrid line</h3>
<p>We next asked if the differential methylation in the three different genotypes could potentially influence the gene expression in these plants. For this, first DEGs with DMRs localized in 2 kb flanking (upstream and downstream) and gene body regions were identified as DMR-associated DEGs. Then the correlation between whole genome differential methylation and gene expression was examined (<a reveal-id="dsy008-F5" data-open="dsy008-F5" class="link link-reveal link-table xref-fig">Fig. 5A–C</a>). We next determined the genes that were enriched for DMRs between the three genotypes and found that DMRs were localized in 3,019, 1,996 and 2,233 genes in 11 A vs. hybrid, 11 A vs. 303 R and 303 R vs. hybrid, respectively (<a reveal-id="dsy008-F5" data-open="dsy008-F5" class="link link-reveal link-table xref-fig">Fig. 5A–C</a>). Among these, 290, 586 and 578 genes were found to be both differentially expressed and associated with DMRs in 11 A vs. F, 11 A vs. 303 R and 303 R vs. F, respectively (<a reveal-id="dsy008-F5" data-open="dsy008-F5" class="link link-reveal link-table xref-fig">Fig. 5A–C</a>). Furthermore, we also investigated the correlation between methylation status and expression in the DEGs associated with DMRs and found that 127 (16 hypomethylated and up-regulated; 111 hyper-methylated and downregulated), 75 (29 hypo-methylated and up-regulated; 46 hypermethylated and down-regulated) and 229 (197 hypo-methylated and up-regulated; 22 hyper-methylated and down-regulated) genes showed a negative correlation between their expression and methylation in 11 A vs. R, 11 A vs. Hybrid and 303 R vs. Hybrid, respectively (<a reveal-id="dsy008-F5" data-open="dsy008-F5" class="link link-reveal link-table xref-fig">Fig. 5D</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008_supp.zip?Expires=2147483647&Signature=a5pxiGBiZuW6BLObg3zcGti-PLIrtW4lDnTZKtyw3m~A3SRkZcr0Y2ve6moKjtosqKCTHkcpfiWI1YiZ59ZYjagra1cBx2qHidDiS9YD76AyCdt-4~W-54Oj7mbqe~VnqVToy5n7EnQdmwYondGxoqfU4SGnxtqtHyNZnOY--6wL3vaY64N6y9Ol6VaEdmhxlHtZ9zskpG0jWypn5ZEvlgg-2NnJYoPL6EOKqxhPhKOweQ9WNBLLfKlYzWiqy7TPWCchHeatrk45nHlTmBeMjmhFGIHpav47NDhdJbGqGb2EIjKAOQuctS37uI7xAHUHDrLIQQ~VOnhtRcc1h0JCcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data S2</a></span>). In order to gain insights into the biological significance of the observed negative association between DEGs and DMRs, we investigated the functional categories of those genes. Interestingly we found many DMRs showing negative association with gene expression in fertile hybrid and 303 R lines, compared with 11 A (sterile) line belonged to transposable elements and genes that coded for PPR-associated proteins (<a reveal-id="dsy008-F5" data-open="dsy008-F5" class="link link-reveal link-table xref-fig">Fig. 5E–G</a>). In addition, we also found genes coding for histone deacytylase and methyltransferase (<a reveal-id="dsy008-F5" data-open="dsy008-F5" class="link link-reveal link-table xref-fig">Fig. 5E–G</a>). This suggests that the transcription of those genes might possibly be influenced by methylation. </p>                        <a id="120122287" scrollto-destination="120122287"></a>
<div id="dsy008-F5" data-id="dsy008-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/m_dsy008f5.png?Expires=2147483647&Signature=jjIxPuXRCe9Z8uyVYsGHf0mjxZ-cqYoQNX3eZi~sN60c6VxNi9FJruFVVGN1WolSz5VzH~iub9NYOUIt9nBAjTMnVMowctoIDwL6wsTEFZuq9Reu9DpLTO9sUhM52NM76PFl5juXi03dC3-qI5UU4Vi~OP1PR~-cteCmx-2GDz57QlLoJrxvVWZMVD~B18hm2izHKjDg7IB8PZ7obK~uCmAc5qXtYaOkbMdehyrj4nZD2tJIrX45A~j51hWcYP~jO8F0xc8u4zqck7EEkpfgA1GE-AXWrE5BMVPmFHwOGbxXgzOrBB2AsDWNLnp8mRGwTvVdR0lKaMyk4xRTmYx1MQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characterization of DMRs associated DEGs in pigeonpea. (A–C) Venn diagrams showing unique and common DEGs (differentially expressed genes) to the genes associated with DMRs in 11A vs. hybrid (A), 11A vs. 303R (B) and 303R vs. hybrid (C). (D) Bar plots showing number of DEGs that were differentially methylated in 11A vs. 303R, 11A vs. hybrid and 303R vs. hybrid. (E–G) Heat maps displaying comparative methylation levels of selective genes that showed negative correlation between their expression and methylation in 11A vs. 303R (E), 11A vs. hybrid (F) and 303R vs. hybrid (G). Color code indicates methylation level (0–1) (Colored figure is available in the online version of the article)." path-from-xml="dsy008f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122287/dsy008f5.tif" path-from-xml="dsy008f5.tif" target="_blank">View large</a><a section="120122287" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008f5.png?Expires=2147483647&Signature=jF0WaA-MtVaizjMb2sYDsXp7j7HOdaLfU3M-gdf5H~qNzkDsS3~5G6n6ygy7refxjRVjWX0nEPvZwt2U2z1mvGyjULuPXCKNIyMmir5CGO-qfcxWSJTQIpsA4bvOWGYIhpqb3qgXPlAfvT4K93YKbOqdDOrs-4Vjk6i-FCiwvnGVze23nUm~7fOk1IkOLkOOdeWxehvemWK2pgPdFfY0z4lvOVxmpSEs-1NPwi2fJ9dxKDGm8VHLTxK3GcWacc7~DPg~6zRC6UPdCbhe3EBFHF1dlm4aCzO0hsMows9GVUD0YqLf6gAkyeI-TfeES2kjbYJfNsB3d7ryH2XNOwXzag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122287&ar=4939498&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy008f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Characterization of DMRs associated DEGs in pigeonpea. (A–C) Venn diagrams showing unique and common DEGs (differentially expressed genes) to the genes associated with DMRs in 11A vs. hybrid (A), 11A vs. 303R (B) and 303R vs. hybrid (C). <strong>(D)</strong> Bar plots showing number of DEGs that were differentially methylated in 11A vs. 303R, 11A vs. hybrid and 303R vs. hybrid. <strong>(</strong>E–<strong>G)</strong> Heat maps displaying comparative methylation levels of selective genes that showed negative correlation between their expression and methylation in 11A vs. 303R (E), 11A vs. hybrid (F) and 303R vs. hybrid (G). Color code indicates methylation level (0–1) (Colored figure is available in the online version of the article).</p></div></div><div content-id="dsy008-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/m_dsy008f5.png?Expires=2147483647&Signature=jjIxPuXRCe9Z8uyVYsGHf0mjxZ-cqYoQNX3eZi~sN60c6VxNi9FJruFVVGN1WolSz5VzH~iub9NYOUIt9nBAjTMnVMowctoIDwL6wsTEFZuq9Reu9DpLTO9sUhM52NM76PFl5juXi03dC3-qI5UU4Vi~OP1PR~-cteCmx-2GDz57QlLoJrxvVWZMVD~B18hm2izHKjDg7IB8PZ7obK~uCmAc5qXtYaOkbMdehyrj4nZD2tJIrX45A~j51hWcYP~jO8F0xc8u4zqck7EEkpfgA1GE-AXWrE5BMVPmFHwOGbxXgzOrBB2AsDWNLnp8mRGwTvVdR0lKaMyk4xRTmYx1MQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characterization of DMRs associated DEGs in pigeonpea. (A–C) Venn diagrams showing unique and common DEGs (differentially expressed genes) to the genes associated with DMRs in 11A vs. hybrid (A), 11A vs. 303R (B) and 303R vs. hybrid (C). (D) Bar plots showing number of DEGs that were differentially methylated in 11A vs. 303R, 11A vs. hybrid and 303R vs. hybrid. (E–G) Heat maps displaying comparative methylation levels of selective genes that showed negative correlation between their expression and methylation in 11A vs. 303R (E), 11A vs. hybrid (F) and 303R vs. hybrid (G). Color code indicates methylation level (0–1) (Colored figure is available in the online version of the article)." path-from-xml="dsy008f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122287/dsy008f5.tif" path-from-xml="dsy008f5.tif" target="_blank">View large</a><a section="120122287" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008f5.png?Expires=2147483647&Signature=jF0WaA-MtVaizjMb2sYDsXp7j7HOdaLfU3M-gdf5H~qNzkDsS3~5G6n6ygy7refxjRVjWX0nEPvZwt2U2z1mvGyjULuPXCKNIyMmir5CGO-qfcxWSJTQIpsA4bvOWGYIhpqb3qgXPlAfvT4K93YKbOqdDOrs-4Vjk6i-FCiwvnGVze23nUm~7fOk1IkOLkOOdeWxehvemWK2pgPdFfY0z4lvOVxmpSEs-1NPwi2fJ9dxKDGm8VHLTxK3GcWacc7~DPg~6zRC6UPdCbhe3EBFHF1dlm4aCzO0hsMows9GVUD0YqLf6gAkyeI-TfeES2kjbYJfNsB3d7ryH2XNOwXzag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122287&ar=4939498&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy008f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Characterization of DMRs associated DEGs in pigeonpea. (A–C) Venn diagrams showing unique and common DEGs (differentially expressed genes) to the genes associated with DMRs in 11A vs. hybrid (A), 11A vs. 303R (B) and 303R vs. hybrid (C). <strong>(D)</strong> Bar plots showing number of DEGs that were differentially methylated in 11A vs. 303R, 11A vs. hybrid and 303R vs. hybrid. <strong>(</strong>E–<strong>G)</strong> Heat maps displaying comparative methylation levels of selective genes that showed negative correlation between their expression and methylation in 11A vs. 303R (E), 11A vs. hybrid (F) and 303R vs. hybrid (G). Color code indicates methylation level (0–1) (Colored figure is available in the online version of the article).</p></div></div>                            <h3 scrollto-destination=120122288 id="120122288" class="section-title" >3.7. Association between DNA methylation and transcript abundance</h3>
<p>Previous studies have reported a negative association between gene expression and CHG and CHH methylation in plants like cotton.<a reveal-id="dsy008-B57" data-open="dsy008-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> Therefore to investigate the occurrence of any relation between changes in methylation levels and gene expression in pigeonpea, we classified genes into four quartiles based on their FPKM values—undetectable expression (FPKM 0-1), low expression (FPKM 1-10), moderate expression (FPKM 10–100) and high expression (FPKM &gt; 100) according to the criteria described in earlier studies<a reveal-id="dsy008-B58" data-open="dsy008-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a><sup>,</sup><a reveal-id="dsy008-B59" data-open="dsy008-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> and correlated with changes in their methylation levels. Consistent with previous studies, we found CHG and CHH methylation were negatively correlated with gene expression levels. However, additionally, we also found negative association between CG methylation and gene expression (<a reveal-id="dsy008-F6" data-open="dsy008-F6" class="link link-reveal link-table xref-fig">Fig. 6A</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008_supp.zip?Expires=2147483647&Signature=a5pxiGBiZuW6BLObg3zcGti-PLIrtW4lDnTZKtyw3m~A3SRkZcr0Y2ve6moKjtosqKCTHkcpfiWI1YiZ59ZYjagra1cBx2qHidDiS9YD76AyCdt-4~W-54Oj7mbqe~VnqVToy5n7EnQdmwYondGxoqfU4SGnxtqtHyNZnOY--6wL3vaY64N6y9Ol6VaEdmhxlHtZ9zskpG0jWypn5ZEvlgg-2NnJYoPL6EOKqxhPhKOweQ9WNBLLfKlYzWiqy7TPWCchHeatrk45nHlTmBeMjmhFGIHpav47NDhdJbGqGb2EIjKAOQuctS37uI7xAHUHDrLIQQ~VOnhtRcc1h0JCcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>). Since transcription is importantly controlled by upstream and downstream regulatory sequences of the genes, we also sought to examine methylation levels in the gene flanking sequences. Similar methylation levels were observed in the sequences upstream and downstream of genes with high, moderate and low expression. However, contrary to this, genes with no expression exhibited high methylation in their flanking regions suggesting that a high methylation in upstream and downstream regulatory sequences might be critical for suppressing transcript levels (<a reveal-id="dsy008-F6" data-open="dsy008-F6" class="link link-reveal link-table xref-fig">Fig. 6A</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008_supp.zip?Expires=2147483647&Signature=a5pxiGBiZuW6BLObg3zcGti-PLIrtW4lDnTZKtyw3m~A3SRkZcr0Y2ve6moKjtosqKCTHkcpfiWI1YiZ59ZYjagra1cBx2qHidDiS9YD76AyCdt-4~W-54Oj7mbqe~VnqVToy5n7EnQdmwYondGxoqfU4SGnxtqtHyNZnOY--6wL3vaY64N6y9Ol6VaEdmhxlHtZ9zskpG0jWypn5ZEvlgg-2NnJYoPL6EOKqxhPhKOweQ9WNBLLfKlYzWiqy7TPWCchHeatrk45nHlTmBeMjmhFGIHpav47NDhdJbGqGb2EIjKAOQuctS37uI7xAHUHDrLIQQ~VOnhtRcc1h0JCcw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>). We also found that the presence of TE elements within the genes or in the vicinity resulted in a comparatively reduced expression relative to the genes without TE elements within or in the neighboring regions (<a reveal-id="dsy008-F6" data-open="dsy008-F6" class="link link-reveal link-table xref-fig">Fig. 6B</a>). </p>                        <a id="120122290" scrollto-destination="120122290"></a>
<div id="dsy008-F6" data-id="dsy008-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/m_dsy008f6.png?Expires=2147483647&Signature=VEWIHF1zXOwpr5mmOFW0CcU9elOz5O8-YEf08IMu21JMX6HN2hrWCJqNke1uWSCA1PlkiBR-4wVz2gsIK-yD~GSIdnOfqnWhfn3TqZ~0bJwnMFNzMsGTD~SZUoPq2tlW3PCWOpXGLQvyPGeU336xqv-AemVtlbHVRXuECJWMsVw2uabt7t~vz2ncX6x6gnDLZp7iXP7w0E101C-5vjFVRl6kzBwnoKeS5fHy~hS5akD5kmLF9J8HQTpfK6BV3B9uAQASZs1p-hP6HXqRDjlsgnDf6T8jKVs7Jer6C8mzy3EqetbXetwhBHn-Rlj4fRyh1UPJQq7YryvvOmLlCJK5DA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between gene expression and DMRs in pigeonpea. (A) Lineplots showing correlation between CG, CHG and CHH DMRs and the gene expression which was divided into four quartiles—no expression (purple); low expression (green); moderate expression (red) and high expression (cyan) in F1 hybrid. (Also see Figure S7 for correlation in 11A and 303R parental lines). Note a negative correlation between gene expression and methylation in all the three contexts. (B) Box plots showing expression levels of genes inserted with and without TE within their gene bodies and 2kb upstream and downstream flanking regions. Gene +TE indicates insertion of transposon. Gene-TE indicates no insertion of transposon (Colored figure is available in the online version of the article)." path-from-xml="dsy008f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122290/dsy008f6.tif" path-from-xml="dsy008f6.tif" target="_blank">View large</a><a section="120122290" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008f6.png?Expires=2147483647&Signature=iQCco7nRFQA0AZr8iT2N947i2ZGmxWRu9LysmfmZyV9aMJd-PBllMEl6sfDtlTxb5fHnoR7D~g0v0UMtyKg9v~tZ6Yla7ATc1B1Sm6wS~F06Jdw08~XAL6DEKlxGdYNmm2auodgOarnSA1s~MjX4a84LJ2vu5aEzkK6JZlDz3MAfV03wm2GE~bxCnK9czxTVFVTnJ93vSt0sOriYBgmNsDu7R0~XJ6wjYdaLFuk-2LFsVseSFfUj0LemVQKyipcaowt6H8BIOOIYak9qXCBd~6UGFoAghu3D91pJHelwlyxr2dhqvxaUEzThW6QOV9E5mPqtWkz12q2t2OlAQPNRfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122290&ar=4939498&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy008f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between gene expression and DMRs in pigeonpea. <strong>(</strong>A<strong>)</strong> Lineplots showing correlation between CG, CHG and CHH DMRs and the gene expression which was divided into four quartiles—no expression (purple); low expression (green); moderate expression (red) and high expression (cyan) in F1 hybrid. (Also see Figure S7 for correlation in 11A and 303R parental lines). Note a negative correlation between gene expression and methylation in all the three contexts. (B) Box plots showing expression levels of genes inserted with and without TE within their gene bodies and 2kb upstream and downstream flanking regions. Gene +TE indicates insertion of transposon. Gene-TE indicates no insertion of transposon (Colored figure is available in the online version of the article).</p></div></div><div content-id="dsy008-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/m_dsy008f6.png?Expires=2147483647&Signature=VEWIHF1zXOwpr5mmOFW0CcU9elOz5O8-YEf08IMu21JMX6HN2hrWCJqNke1uWSCA1PlkiBR-4wVz2gsIK-yD~GSIdnOfqnWhfn3TqZ~0bJwnMFNzMsGTD~SZUoPq2tlW3PCWOpXGLQvyPGeU336xqv-AemVtlbHVRXuECJWMsVw2uabt7t~vz2ncX6x6gnDLZp7iXP7w0E101C-5vjFVRl6kzBwnoKeS5fHy~hS5akD5kmLF9J8HQTpfK6BV3B9uAQASZs1p-hP6HXqRDjlsgnDf6T8jKVs7Jer6C8mzy3EqetbXetwhBHn-Rlj4fRyh1UPJQq7YryvvOmLlCJK5DA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between gene expression and DMRs in pigeonpea. (A) Lineplots showing correlation between CG, CHG and CHH DMRs and the gene expression which was divided into four quartiles—no expression (purple); low expression (green); moderate expression (red) and high expression (cyan) in F1 hybrid. (Also see Figure S7 for correlation in 11A and 303R parental lines). Note a negative correlation between gene expression and methylation in all the three contexts. (B) Box plots showing expression levels of genes inserted with and without TE within their gene bodies and 2kb upstream and downstream flanking regions. Gene +TE indicates insertion of transposon. Gene-TE indicates no insertion of transposon (Colored figure is available in the online version of the article)." path-from-xml="dsy008f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120122290/dsy008f6.tif" path-from-xml="dsy008f6.tif" target="_blank">View large</a><a section="120122290" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008f6.png?Expires=2147483647&Signature=iQCco7nRFQA0AZr8iT2N947i2ZGmxWRu9LysmfmZyV9aMJd-PBllMEl6sfDtlTxb5fHnoR7D~g0v0UMtyKg9v~tZ6Yla7ATc1B1Sm6wS~F06Jdw08~XAL6DEKlxGdYNmm2auodgOarnSA1s~MjX4a84LJ2vu5aEzkK6JZlDz3MAfV03wm2GE~bxCnK9czxTVFVTnJ93vSt0sOriYBgmNsDu7R0~XJ6wjYdaLFuk-2LFsVseSFfUj0LemVQKyipcaowt6H8BIOOIYak9qXCBd~6UGFoAghu3D91pJHelwlyxr2dhqvxaUEzThW6QOV9E5mPqtWkz12q2t2OlAQPNRfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120122290&ar=4939498&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy008f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between gene expression and DMRs in pigeonpea. <strong>(</strong>A<strong>)</strong> Lineplots showing correlation between CG, CHG and CHH DMRs and the gene expression which was divided into four quartiles—no expression (purple); low expression (green); moderate expression (red) and high expression (cyan) in F1 hybrid. (Also see Figure S7 for correlation in 11A and 303R parental lines). Note a negative correlation between gene expression and methylation in all the three contexts. (B) Box plots showing expression levels of genes inserted with and without TE within their gene bodies and 2kb upstream and downstream flanking regions. Gene +TE indicates insertion of transposon. Gene-TE indicates no insertion of transposon (Colored figure is available in the online version of the article).</p></div></div>                            <h2 scrollto-destination=120122291 id="120122291" class="section-title" >4. Discussion</h2>
<p>Rapid advancements in genome sequencing techniques have enabled genomic comparisons to understand genetic diversities between different plant species or different ecotypes of the same species. However, understanding of epigenetic variation among different ecotypes has become equally important to decipher its role in generating phenotypic variations. A fundamental question of great importance is that how the parental genome combination is regulated in hybrids to give rise to significantly different hybrid genome which results in altered genetic traits. It is widely speculated that many agriculturally important phenomena such as hybrid vigour and fertility restoration may arise due to epigenetic changes between inbred parents and hybrid lines. Thus the field of epigenetic research has expanded its horizon by adopting many crop species as model systems. However, there are still agriculturally important crops such as pigeonpea, in which existence of such mechanisms have remained unexplored. In this study, we investigated and compared epigenetic changes in three different pigeonpea genotypes. We have generated whole genome methylation profiles of sterile (11 A), fertile parents (303 R) and their fertile F1 hybrid, which were unavailable till date. Our analysis of relative fractions of methyl-cytosines in each methylation context in the three pigeonpea lines revealed highest fraction of mCs in CG context, followed by CHG and CHH. Similar pattern of distribution of methylation fractions has been observed previously in rice seedlings<a reveal-id="dsy008-B60" data-open="dsy008-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> and Soybean leaves.<a reveal-id="dsy008-B61" data-open="dsy008-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> We also found that average methylation levels of the fertile hybrid and the fertile parental line (303 R) were lower than the sterile parent (11 A) suggesting that comparative low DNA methylation of genes may play a role in restoring the fertility in the F1 hybrid.</p><p>Mean methylation has been reported to be highest in CG contexts in other plants such as Rice<a reveal-id="dsy008-B60" data-open="dsy008-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> and Arabidopsis,<a reveal-id="dsy008-B15" data-open="dsy008-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> and followed closely by CHG and CHH. However, our results show that in pigeonpea, mean methylation levels in CHH context are much lower than CG and CHG methylation levels. This suggests a possibility of overall low expression of genes involved in regulation of CHH methylations such as those involved in RdDM pathway that target homologous regions of the genome for methylation<a reveal-id="dsy008-B62" data-open="dsy008-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> in pigeonpea compared with Rice and Arabidopsis.</p><p>Previously, it has been reported that transcription of genes is affected by methylation in the promoter region but not in the internal regions. However, transposon silencing is regulated by methylation both in the promoter region as well as internal regions.<a reveal-id="dsy008-B63" data-open="dsy008-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> Our data demonstrate that fraction of mCs was higher in the transcribed regions of transposons than the transcribed regions of the genes. As shown in <a reveal-id="dsy008-F2" data-open="dsy008-F2" class="link link-reveal link-table xref-fig">Figure 2</a>, the fraction of methyl cytosines in upstream sequences of the transcribed regions of transposon was almost two times higher than fraction of mCs in upstream sequences of genes. Additionally, gene bodies of TEs exhibited higher methylation than their upstream and downstream regions. Contrary to this, genes had lower methylation in the transcribed regions than their flanking regions. As suggested previously for other crops,<a reveal-id="dsy008-B1" data-open="dsy008-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsy008-B64" data-open="dsy008-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> this could potentially be a consequence of reduced RdDM pathway activity within the gene body of the transcribed genes compared with TE. Furthermore, we also observed that the fraction of mCs in upstream regions of transposons in fertile 303 R parent was slightly less than sterile 11 A parental line and the fertile hybrid. These observations suggest that one possible epigenetic mechanism of regulating transposon inactivation in pigeonpea could be via methylation and this mechanism could be comparatively weaker in the fertile 303 R line than in the sterile parent and fertile hybrid. Over all, our results comply with previously documented findings that CHH methylation levels are usually lower than CG and CHG, both on a genomic scale and at specific loci.<a reveal-id="dsy008-B65" data-open="dsy008-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a><sup>,</sup><a reveal-id="dsy008-B66" data-open="dsy008-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a></p><p>Further, our findings also reveal the existence of non-additive methylation in F1 hybrid due TCM and TCdM events. TcM mediated increase in DNA methylation were more frequently observed in CG context and TcDM mediated decrease in DNA methylation were more frequently observed in CHH context, consistent with previous findings in Arabidopsis.<a reveal-id="dsy008-B15" data-open="dsy008-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> Furthermore, we also dissected the mC changes of differentially methylated parental epi-alleles in the hybrid, using SNPs for tracking the parental epi-alleles in the hybrid. Our analysis revealed that majority of the mC clusters having SNPs showed non- additive methylation in the hybrid when parents had different levels of methylation (HP or LP), consistent with our general mC cluster analysis.</p><p>Transposable elements can influence neighboring gene expression, contribute to inheritance of epi-alleles and thus represent the most striking and common example of epigenetic inheritance in plants.<a reveal-id="dsy008-B67" data-open="dsy008-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> Consistently, our data also revealed low expression of genes with inserted TEs suggesting similar roles of TEs in pigeonpea.</p><p>We also found a strong enrichment of DMRs in a wide spectrum of genes that are associated with chromatin modification and have been reported to regulate flowering, heterosis and reproduction in rice.<a reveal-id="dsy008-B68" data-open="dsy008-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> We also found genes involved in DNA methylation that are known to play a role in transposon repression<a reveal-id="dsy008-B69" data-open="dsy008-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a> and genes involved in histone methylation that help regulate flowering.<a reveal-id="dsy008-B70" data-open="dsy008-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a></p><p>DNA methylation is known to influence gene expression. However, as mentioned above, it has been proposed that DNA methylation and H3K9 methylation in the gene body regions do not affect gene abundance. Despite increased CHG and H3K9 methylation in loss of function mutants of a H3K9 demethylase gene <em>increased in bonsai methylation 1 (ibm1)</em>, the expression of their target genes remained unaffected.<a reveal-id="dsy008-B63" data-open="dsy008-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> Also, absence of CG gene body methylation in the loss of function MET1 mutants did not alter gene expression.<a reveal-id="dsy008-B71" data-open="dsy008-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> Surprisingly, we found a negative correlation between transcript abundance and methylation levels in all the three contexts in the gene body suggesting a more prominent role of DNA methylation in regulating gene expression in pigeonpea. Previous studies in Arabidopsis have reported that methylation in the promoter regions resulted in lower gene expression.<a reveal-id="dsy008-B71" data-open="dsy008-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a><sup>,</sup><a reveal-id="dsy008-B72" data-open="dsy008-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> Furthermore, another study in soybean revealed negative correlation between gene expression and DNA methylation in their promoters.<a reveal-id="dsy008-B73" data-open="dsy008-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a> We also found that high methylation in the 5’ upstream regions of the genes were negatively correlated with gene expression levels, consistent with the idea that DNA methylation in the promoter regions affects gene abundance.<a reveal-id="dsy008-B61" data-open="dsy008-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> Our results in pigeonpea together with previous results in soybean<a reveal-id="dsy008-B73" data-open="dsy008-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a> open new avenues for understanding the relationship between gene expression and gene body DNA methylation and whether they play a direct role in restoring fertility in pigeonpea hybrids requires further investigation.</p><p>The raw reads of bisulfite and transcriptomic study were submitted to NCBI Sequence Read Archive under bioproject accession PRJNA435649 and SRP133358 study.</p>                            <h2 scrollto-destination=120122300 id="120122300" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Council of Scientific and Industrial Research, Government of India for the award of a senior research fellowship, ICAR-IARI, ICAR-NRCPB and NRCPB in house project, New Delhi for supporting A.J. and providing other funds and facilities. We also thank Dr N. Bhatia, The University of Sydney for helpful feedback and ideas on the manuscript.</p>                            <h3 scrollto-destination=120122302 id="120122302" class="section-title" >Authors’ contributions</h3>
<p>A.J and K.G conceived and designed the study. A.J. performed all the experimental work and data analysis. H.K. helped in data analysis and interpreting the results. A.J., K.G. wrote the manuscript. A.R. and N.K.S. helped editing the manuscript.</p>                            <h3 scrollto-destination=120122304 id="120122304" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h2 scrollto-destination=120122306 id="120122306" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsy008-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Henderson</div> <div class="given-names">I. R.</div></div>, <div class="name string-name"><div class="surname">Jacobsen</div> <div class="given-names">S. E.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Epigenetic inheritance in plants</div>, <div class="source">Nature</div> , <div class="volume">447</div>, <div class="fpage">418</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenetic%20inheritance%20in%20plants&amp;author=I.%20R.%20Henderson&amp;author=S.%20E.%20Jacobsen&amp;publication_year=2007&amp;journal=Nature&amp;volume=447&amp;pages=418-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature05917" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05917" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature05917"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17522675" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Niederhuth</div> <div class="given-names">C. E.</div></div>, <div class="name string-name"><div class="surname">Schmitz</div> <div class="given-names">R. J.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Covering your bases: inheritance of DNA methylation in plant genomes</div>, <div class="source">Mol. Plant</div> ., <div class="volume">7</div>, <div class="fpage">472</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Covering%20your%20bases%3A%20inheritance%20of%20DNA%20methylation%20in%20plant%20genomes&amp;author=C.%20E.%20Niederhuth&amp;author=R.%20J.%20Schmitz&amp;publication_year=2014&amp;journal=Mol.%20Plant&amp;volume=7&amp;pages=472-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/mp/sst165" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmp%2Fsst165" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmp%2Fsst165"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24270503" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Robertson</div> <div class="given-names">K. D.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">DNA methylation and human disease</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">6</div>, <div class="fpage">597</div>–<div class="lpage">610</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20and%20human%20disease&amp;author=K.%20D.%20Robertson&amp;publication_year=2005&amp;journal=Nat.%20Rev.%20Genet&amp;volume=6&amp;pages=597-610" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1655" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1655" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1655"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16136652" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Suzuki</div> <div class="given-names">M. M.</div></div>, <div class="name string-name"><div class="surname">Bird</div> <div class="given-names">A.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">DNA methylation landscapes: provocative insights from epigenomics</div>, <div class="source">Nat Rev Genet</div> ., <div class="volume">9</div>, <div class="fpage">465</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20landscapes%3A%20provocative%20insights%20from%20epigenomics&amp;author=M.%20M.%20Suzuki&amp;author=A.%20Bird&amp;publication_year=2008&amp;journal=Nat%20Rev%20Genet&amp;volume=9&amp;pages=465-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2341" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2341" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2341"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18463664" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bird</div> <div class="given-names">A.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Perceptions of epigenetics</div>, <div class="source">Nature</div> , <div class="volume">447</div>, <div class="fpage">396</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Perceptions%20of%20epigenetics&amp;author=A.%20Bird&amp;publication_year=2007&amp;journal=Nature&amp;volume=447&amp;pages=396-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature05913" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05913" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature05913"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17522671" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Law</div> <div class="given-names">J. A.</div></div>, <div class="name string-name"><div class="surname">Jacobsen</div> <div class="given-names">S. E.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Establishing, maintaining and modifying DNA methylation patterns in plants and animals</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">11</div>, <div class="fpage">204</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Establishing%2C%20maintaining%20and%20modifying%20DNA%20methylation%20patterns%20in%20plants%20and%20animals&amp;author=J.%20A.%20Law&amp;author=S.%20E.%20Jacobsen&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Genet&amp;volume=11&amp;pages=204-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2719" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2719" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2719"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20142834" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vongs</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Kakutani</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Martienssen</div> <div class="given-names">R. A.</div></div>, <div class="name string-name"><div class="surname">Richards</div> <div class="given-names">E. J.</div></div></div> <div class="year">1993</div>, 
<div class="article-title">Arabidopsis thaliana DNA methylation mutants</div>, <div class="source">Science</div> , <div class="volume">260</div>, <div class="fpage">1926</div>–<div class="lpage">8</div>.,<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20thaliana%20DNA%20methylation%20mutants&amp;author=A.%20Vongs&amp;author=T.%20Kakutani&amp;author=R.%20A.%20Martienssen&amp;author=E.%20J.%20Richards&amp;publication_year=1993&amp;journal=Science&amp;volume=260&amp;pages=1926-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.8316832" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.8316832" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.8316832"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8316832" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kankel</div> <div class="given-names">M. W.</div></div>, <div class="name string-name"><div class="surname">Ramsey</div> <div class="given-names">D. E.</div></div>, <div class="name string-name"><div class="surname">Stokes</div> <div class="given-names">T. L.</div></div></div>, et al.   <div class="year">2003</div>, 
<div class="article-title">Arabidopsis MET1 cytosine methyltransferase mutants</div>, <div class="source">Genetics</div> , <div class="volume">163</div>, <div class="fpage">1109</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20MET1%20cytosine%20methyltransferase%20mutants&amp;author=M.%20W.%20Kankel&amp;author=D.%20E.%20Ramsey&amp;author=T.%20L.%20Stokes&amp;publication_year=2003&amp;journal=Genetics&amp;volume=163&amp;pages=1109-22" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12663548" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zemach</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">McDaniel</div> <div class="given-names">I. E.</div></div>, <div class="name string-name"><div class="surname">Silva</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Zilberman</div> <div class="given-names">D.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Genome-wide evolutionary analysis of eukaryotic DNA methylation</div>, <div class="source">Science</div> , <div class="volume">328</div>, <div class="fpage">916</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20evolutionary%20analysis%20of%20eukaryotic%20DNA%20methylation&amp;author=A.%20Zemach&amp;author=I.%20E.%20McDaniel&amp;author=P.%20Silva&amp;author=D.%20Zilberman&amp;publication_year=2010&amp;journal=Science&amp;volume=328&amp;pages=916-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1186366" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1186366" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1186366"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20395474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cao</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Jacobsen</div> <div class="given-names">S. E.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Role of the arabidopsis DRM methyltransferases in de novo DNA methylation and gene silencing</div>, <div class="source">Curr. Biol</div> ., <div class="volume">12</div>, <div class="fpage">1138</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20the%20arabidopsis%20DRM%20methyltransferases%20in%20de%20novo%20DNA%20methylation%20and%20gene%20silencing&amp;author=X.%20Cao&amp;author=S.%20E.%20Jacobsen&amp;publication_year=2002&amp;journal=Curr.%20Biol&amp;volume=12&amp;pages=1138-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0960-9822(02)00925-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0960-9822(02)00925-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0960-9822(02)00925-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12121623" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mosher</div> <div class="given-names">R. A.</div></div>, <div class="name string-name"><div class="surname">Melnyk</div> <div class="given-names">C. W.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">siRNAs and DNA methylation: seedy epigenetics</div>, <div class="source">Trends Plant Sci</div> ., <div class="volume">15</div>, <div class="fpage">204</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=siRNAs%20and%20DNA%20methylation%3A%20seedy%20epigenetics&amp;author=R.%20A.%20Mosher&amp;author=C.%20W.%20Melnyk&amp;publication_year=2010&amp;journal=Trends%20Plant%20Sci&amp;volume=15&amp;pages=204-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2010.01.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2010.01.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2010.01.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20129810" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stroud</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Do</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Du</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Non-CG methylation patterns shape the epigenetic landscape in Arabidopsis</div>, <div class="source">Nat. Struct. Mol. Biol</div> ., <div class="volume">21</div>, <div class="fpage">64</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Non-CG%20methylation%20patterns%20shape%20the%20epigenetic%20landscape%20in%20Arabidopsis&amp;author=H.%20Stroud&amp;author=T.%20Do&amp;author=J.%20Du&amp;publication_year=2014&amp;journal=Nat.%20Struct.%20Mol.%20Biol&amp;volume=21&amp;pages=64-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb.2735" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb.2735" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb.2735"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24336224" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Reed</div> <div class="given-names">H. S.</div></div></div> <div class="year">1942</div>, A short history of the plant sciences. <a class="link link-uri" href="https://www.scribd.com/document/190746815/Reed-Howard-S-1942-a-Short-History-of-the-Plant-Sciences" target="_blank">https://www.scribd.com/document/190746815/Reed-Howard-S-1942-a-Short-History-of-the-Plant-Sciences</a>.</div></div></div></div><div content-id="dsy008-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ng</div> <div class="given-names">S. Y.</div></div>, <div class="name string-name"><div class="surname">Lin</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Soh</div> <div class="given-names">B. S.</div></div>, <div class="name string-name"><div class="surname">Stanton</div> <div class="given-names">L. W.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Long noncoding RNAs in development and disease of the central nervous system</div>, <div class="source">Trends Genet</div> ., <div class="volume">29</div>, <div class="fpage">461</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Long%20noncoding%20RNAs%20in%20development%20and%20disease%20of%20the%20central%20nervous%20system&amp;author=S.%20Y.%20Ng&amp;author=L.%20Lin&amp;author=B.%20S.%20Soh&amp;author=L.%20W.%20Stanton&amp;publication_year=2013&amp;journal=Trends%20Genet&amp;volume=29&amp;pages=461-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2013.03.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2013.03.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2013.03.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23562612" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Greaves</div> <div class="given-names">I. K.</div></div>, <div class="name string-name"><div class="surname">Groszmann</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Ying</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Taylor</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Peacock</div> <div class="given-names">W. J.</div></div>, <div class="name string-name"><div class="surname">Dennis</div> <div class="given-names">E. S.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Trans chromosomal methylation in Arabidopsis hybrids</div>, <div class="source">Proc. Natl. Acad. Sci. US. A</div> , <div class="volume">109</div>, <div class="fpage">3570</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Trans%20chromosomal%20methylation%20in%20Arabidopsis%20hybrids&amp;author=I.%20K.%20Greaves&amp;author=M.%20Groszmann&amp;author=H.%20Ying&amp;author=J.%20M.%20Taylor&amp;author=W.%20J.%20Peacock&amp;author=E.%20S.%20Dennis&amp;publication_year=2012&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20US.%20A&amp;volume=109&amp;pages=3570-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1201043109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1201043109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1201043109"> </span></div> </div></div></div></div></div><div content-id="dsy008-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Greaves</div> <div class="given-names">I. K.</div></div>, <div class="name string-name"><div class="surname">Groszmann</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Peacock</div> <div class="given-names">W. J.</div></div>, <div class="name string-name"><div class="surname">Dennis</div> <div class="given-names">E. S.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Inheritance of trans chromosomal methylation patterns from Arabidopsis F1 hybrids</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">111</div>, <div class="fpage">2017</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inheritance%20of%20trans%20chromosomal%20methylation%20patterns%20from%20Arabidopsis%20F1%20hybrids&amp;author=I.%20K.%20Greaves&amp;author=M.%20Groszmann&amp;author=A.%20Wang&amp;author=W.%20J.%20Peacock&amp;author=E.%20S.%20Dennis&amp;publication_year=2014&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=111&amp;pages=2017-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1323656111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1323656111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1323656111"> </span></div> </div></div></div></div></div><div content-id="dsy008-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Greaves</div> <div class="given-names">I. K.</div></div>, <div class="name string-name"><div class="surname">Gonzalez-Bayon</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Epigenetic changes in hybrids</div>, <div class="source">Plant Physiol</div> ., <div class="volume">168</div>, <div class="fpage">1197</div>–<div class="lpage">205</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenetic%20changes%20in%20hybrids&amp;author=I.%20K.%20Greaves&amp;author=R.%20Gonzalez-Bayon&amp;author=L.%20Wang&amp;publication_year=2015&amp;journal=Plant%20Physiol&amp;volume=168&amp;pages=1197-205" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.15.00231" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.15.00231" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.15.00231"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26002907" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vaughn</div> <div class="given-names">M. W.</div></div>, <div class="name string-name"><div class="surname">Tanurdzic</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Lippman</div> <div class="given-names">Z.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">Epigenetic natural variation in Arabidopsis thaliana</div>, <div class="source">PLoS Biol</div> ., <div class="volume">5</div>, <div class="fpage">e174</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenetic%20natural%20variation%20in%20Arabidopsis%20thaliana&amp;author=M.%20W.%20Vaughn&amp;author=M.%20Tanurdzic&amp;author=Z.%20Lippman&amp;publication_year=2007&amp;journal=PLoS%20Biol&amp;volume=5&amp;pages=e174" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pbio.0050174" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pbio.0050174" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pbio.0050174"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17579518" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schmitz</div> <div class="given-names">R. J.</div></div>, <div class="name string-name"><div class="surname">Schultz</div> <div class="given-names">M. D.</div></div>, <div class="name string-name"><div class="surname">Urich</div> <div class="given-names">M. A.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Patterns of population epigenomic diversity</div>, <div class="source">Nature</div> , <div class="volume">495</div>, <div class="fpage">193</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patterns%20of%20population%20epigenomic%20diversity&amp;author=R.%20J.%20Schmitz&amp;author=M.%20D.%20Schultz&amp;author=M.%20A.%20Urich&amp;publication_year=2013&amp;journal=Nature&amp;volume=495&amp;pages=193-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11968" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11968" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11968"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23467092" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shen</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">He</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Genome-wide analysis of DNA methylation and gene expression changes in two Arabidopsis ecotypes and their reciprocal hybrids</div>, <div class="source">Plant Cell</div> , <div class="volume">24</div>, <div class="fpage">875</div>–<div class="lpage">92</div>.,<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20DNA%20methylation%20and%20gene%20expression%20changes%20in%20two%20Arabidopsis%20ecotypes%20and%20their%20reciprocal%20hybrids&amp;author=H.%20Shen&amp;author=H.%20He&amp;author=J.%20Li&amp;publication_year=2012&amp;journal=Plant%20Cell&amp;volume=24&amp;pages=875-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.111.094870" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.111.094870" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.111.094870"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22438023" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Varala</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Moose</div> <div class="given-names">S. P.</div></div>, <div class="name string-name"><div class="surname">Hudson</div> <div class="given-names">M. E.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">The inheritance pattern of 24 nt siRNA clusters in arabidopsis hybrids is influenced by proximity to transposable elements</div>, <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e47043</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20inheritance%20pattern%20of%2024%20nt%20siRNA%20clusters%20in%20arabidopsis%20hybrids%20is%20influenced%20by%20proximity%20to%20transposable%20elements&amp;author=Y.%20Li&amp;author=K.%20Varala&amp;author=S.%20P.%20Moose&amp;author=M.%20E.%20Hudson&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e47043" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0047043" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0047043" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0047043"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23118865" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Groszmann</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Greaves</div> <div class="given-names">I. K.</div></div>, <div class="name string-name"><div class="surname">Albertyn</div> <div class="given-names">Z. I.</div></div>, <div class="name string-name"><div class="surname">Scofield</div> <div class="given-names">G. N.</div></div>, <div class="name string-name"><div class="surname">Peacock</div> <div class="given-names">W. J.</div></div>, <div class="name string-name"><div class="surname">Dennis</div> <div class="given-names">E. S.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Changes in 24-nt siRNA levels in Arabidopsis hybrids suggest an epigenetic contribution to hybrid vigor</div>, <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> , <div class="volume">108</div>, <div class="fpage">2617</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Changes%20in%2024-nt%20siRNA%20levels%20in%20Arabidopsis%20hybrids%20suggest%20an%20epigenetic%20contribution%20to%20hybrid%20vigor&amp;author=M.%20Groszmann&amp;author=I.%20K.%20Greaves&amp;author=Z.%20I.%20Albertyn&amp;author=G.%20N.%20Scofield&amp;author=W.%20J.%20Peacock&amp;author=E.%20S.%20Dennis&amp;publication_year=2011&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=108&amp;pages=2617-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1019217108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1019217108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1019217108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21266545" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">He</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Elling</div> <div class="given-names">A. A.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Global epigenetic and transcriptional trends among two rice subspecies and their reciprocal hybrids</div>, <div class="source">Plant Cell</div> , <div class="volume">22</div>, <div class="fpage">17</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20epigenetic%20and%20transcriptional%20trends%20among%20two%20rice%20subspecies%20and%20their%20reciprocal%20hybrids&amp;author=G.%20He&amp;author=X.%20Zhu&amp;author=A.%20A.%20Elling&amp;publication_year=2010&amp;journal=Plant%20Cell&amp;volume=22&amp;pages=17-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.109.072041" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.109.072041" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.109.072041"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20086188" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Barber</div> <div class="given-names">W. T.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Win</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Repeat associated small RNAs vary among parents and following hybridization in maize</div>, <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> , <div class="volume">109</div>, <div class="fpage">10444</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repeat%20associated%20small%20RNAs%20vary%20among%20parents%20and%20following%20hybridization%20in%20maize&amp;author=W.%20T.%20Barber&amp;author=W.%20Zhang&amp;author=H.%20Win&amp;publication_year=2012&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=109&amp;pages=10444-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1202073109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1202073109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1202073109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22689990" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Parkin</div> <div class="given-names">I. A.</div></div>, <div class="name string-name"><div class="surname">Koh</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Transcriptome and methylome profiling reveals relics of genome dominance in the mesopolyploid Brassica oleracea</div>, <div class="source">Genome Biol</div> ., <div class="volume">15</div>, <div class="fpage">R77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20and%20methylome%20profiling%20reveals%20relics%20of%20genome%20dominance%20in%20the%20mesopolyploid%20Brassica%20oleracea&amp;author=I.%20A.%20Parkin&amp;author=C.%20Koh&amp;author=H.%20Tang&amp;publication_year=2014&amp;journal=Genome%20Biol&amp;volume=15&amp;pages=R77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2014-15-6-r77" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2014-15-6-r77" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2014-15-6-r77"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24916971" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chodavarapu</div> <div class="given-names">R. K.</div></div>, <div class="name string-name"><div class="surname">Feng</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ding</div> <div class="given-names">B.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Transcriptome and methylome interactions in rice hybrids</div>, <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> , <div class="volume">109</div>, <div class="fpage">12040</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20and%20methylome%20interactions%20in%20rice%20hybrids&amp;author=R.%20K.%20Chodavarapu&amp;author=S.%20Feng&amp;author=B.%20Ding&amp;publication_year=2012&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=109&amp;pages=12040-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1209297109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1209297109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1209297109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22778444" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Singh</div> <div class="given-names">N. K.</div></div>, <div class="name string-name"><div class="surname">Gupta</div> <div class="given-names">D. K.</div></div>, <div class="name string-name"><div class="surname">Jayaswal</div> <div class="given-names">P. K.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">The first draft of the pigeonpea genome sequence</div>, <div class="source">J. Plant Biochem. Biotechnol</div> ., <div class="volume">21</div>, <div class="fpage">98</div>–<div class="lpage">112</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20first%20draft%20of%20the%20pigeonpea%20genome%20sequence&amp;author=N.%20K.%20Singh&amp;author=D.%20K.%20Gupta&amp;author=P.%20K.%20Jayaswal&amp;publication_year=2012&amp;journal=J.%20Plant%20Biochem.%20Biotechnol&amp;volume=21&amp;pages=98-112" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s13562-011-0088-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs13562-011-0088-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs13562-011-0088-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24431589" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Saxena</div> <div class="given-names">K. B.</div></div>, <div class="name string-name"><div class="surname">Singh</div> <div class="given-names">L.</div></div></div> <div class="year">1990</div>, 
<div class="article-title">Low natural outcrossing in ′cleistogamous′ pigeonpea mutant</div>, <div class="source">Int. Pigeonpea Newslett</div> ., <div class="volume">11</div>, <div class="fpage">9</div>–<div class="lpage">10</div>. ISSN 0255-786X.</div></div></div></div><div content-id="dsy008-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Y. G.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Male sterility and fertility restoration in crops</div>, <div class="source">Annu. Rev. Plant Biol</div> ., <div class="volume">65</div>, <div class="fpage">579</div>–<div class="lpage">606</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Male%20sterility%20and%20fertility%20restoration%20in%20crops&amp;author=L.%20Chen&amp;author=Y.%20G.%20Liu&amp;publication_year=2014&amp;journal=Annu.%20Rev.%20Plant%20Biol&amp;volume=65&amp;pages=579-606" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-arplant-050213-040119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-arplant-050213-040119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-arplant-050213-040119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24313845" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tuteja</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Saxena</div> <div class="given-names">R. K.</div></div>, <div class="name string-name"><div class="surname">Davila</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Cytoplasmic male sterility-associated chimeric open reading frames identified by mitochondrial genome sequencing of four Cajanus genotypes</div>, <div class="source">DNA Res</div> ., <div class="volume">20</div>, <div class="fpage">485</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytoplasmic%20male%20sterility-associated%20chimeric%20open%20reading%20frames%20identified%20by%20mitochondrial%20genome%20sequencing%20of%20four%20Cajanus%20genotypes&amp;author=R.%20Tuteja&amp;author=R.%20K.%20Saxena&amp;author=J.%20Davila&amp;publication_year=2013&amp;journal=DNA%20Res&amp;volume=20&amp;pages=485-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst025"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23792890" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Meshram</div> <div class="given-names">M. P.</div></div>, <div class="name string-name"><div class="surname">Patil</div> <div class="given-names">A. N.</div></div></div> <div class="year">2018</div>, 
<div class="article-title">Genetics of fertility restoration in A2 cytoplasm based hybrids of pigeonpea</div>. <div class="source">Int. J. Cur. Microbiol. Appl. Sci</div> ., ISSN: 2319-7706 565-571.</div></div></div></div><div content-id="dsy008-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Varshney</div> <div class="given-names">R. K.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Draft genome sequence of pigeonpea (Cajanus cajan), an orphan legume crop of resource-poor farmers</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">30</div>, <div class="fpage">83</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20pigeonpea%20(Cajanus%20cajan)%2C%20an%20orphan%20legume%20crop%20of%20resource-poor%20farmers&amp;author=R.%20K.%20Varshney&amp;author=W.%20Chen&amp;author=Y.%20Li&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol&amp;volume=30&amp;pages=83-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2022"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22057054" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xi</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">W.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">BSMAP: whole genome bisulfite sequence MAPping program</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">10</div>, <div class="fpage">232</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BSMAP%3A%20whole%20genome%20bisulfite%20sequence%20MAPping%20program&amp;author=Y.%20Xi&amp;author=W.%20Li&amp;publication_year=2009&amp;journal=BMC%20Bioinformatics&amp;volume=10&amp;pages=232" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-10-232" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-10-232" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-10-232"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19635165" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schultz</div> <div class="given-names">M. D.</div></div>, <div class="name string-name"><div class="surname">Schmitz</div> <div class="given-names">R. J.</div></div>, <div class="name string-name"><div class="surname">Ecker</div> <div class="given-names">J. R.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">′Leveling′ the playing field for analyses of single-base resolution DNA methylomes</div>, <div class="source">Trends Genet</div> ., <div class="volume">28</div>, <div class="fpage">583</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=%E2%80%B2Leveling%E2%80%B2%20the%20playing%20field%20for%20analyses%20of%20single-base%20resolution%20DNA%20methylomes&amp;author=M.%20D.%20Schultz&amp;author=R.%20J.%20Schmitz&amp;author=J.%20R.%20Ecker&amp;publication_year=2012&amp;journal=Trends%20Genet&amp;volume=28&amp;pages=583-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2012.10.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2012.10.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2012.10.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23131467" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Akalin</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Kormaksson</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">methylKit: a comprehensive R package for the analysis of genome-wide DNA methylation profiles</div>, <div class="source">Genome Bio.l</div> , <div class="volume">13</div>, <div class="fpage">R87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=methylKit%3A%20a%20comprehensive%20R%20package%20for%20the%20analysis%20of%20genome-wide%20DNA%20methylation%20profiles&amp;author=A.%20Akalin&amp;author=M.%20Kormaksson&amp;author=S.%20Li&amp;publication_year=2012&amp;journal=Genome%20Bio.l&amp;volume=13&amp;pages=R87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2012-13-10-r87" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2012-13-10-r87" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2012-13-10-r87"> </span></div> </div></div></div></div></div><div content-id="dsy008-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">L. C.</div></div>, <div class="name string-name"><div class="surname">Dahiya</div> <div class="given-names">R.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">MethPrimer: designing primers for methylation PCRs</div>, <div class="source">Bioinformatics</div> , <div class="volume">18</div>, <div class="fpage">1427</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MethPrimer%3A%20designing%20primers%20for%20methylation%20PCRs&amp;author=L.%20C.%20Li&amp;author=R.%20Dahiya&amp;publication_year=2002&amp;journal=Bioinformatics&amp;volume=18&amp;pages=1427-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/18.11.1427" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F18.11.1427" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F18.11.1427"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12424112" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kumaki</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Oda</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Okano</div> <div class="given-names">M.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">QUMA: quantification tool for methylation analysis</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">36</div>, <div class="fpage">W170</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QUMA%3A%20quantification%20tool%20for%20methylation%20analysis&amp;author=Y.%20Kumaki&amp;author=M.%20Oda&amp;author=M.%20Okano&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res&amp;volume=36&amp;pages=W170-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn294" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn294" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn294"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18487274" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Langmead</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Fast gapped-read alignment with Bowtie 2</div>, <div class="source">Nat. Methods</div>  , <div class="volume">9</div>, <div class="fpage">357</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B.%20Langmead&amp;author=S.%20L.%20Salzberg&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=357-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Dewey</div> <div class="given-names">C. N.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">12</div>, <div class="fpage">323</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RSEM%3A%20accurate%20transcript%20quantification%20from%20RNA-Seq%20data%20with%20or%20without%20a%20reference%20genome&amp;author=B.%20Li&amp;author=C.%20N.%20Dewey&amp;publication_year=2011&amp;journal=BMC%20Bioinformatics&amp;volume=12&amp;pages=323" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-12-323" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-12-323" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-12-323"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21816040" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Robinson</div> <div class="given-names">M. D.</div></div>, <div class="name string-name"><div class="surname">McCarthy</div> <div class="given-names">D. J.</div></div>, <div class="name string-name"><div class="surname">Smyth</div> <div class="given-names">G. K.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">edgeR: a bioconductor package for differential expression analysis of digital gene expression data</div>, <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">139</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=edgeR%3A%20a%20bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20gene%20expression%20data&amp;author=M.%20D.%20Robinson&amp;author=D.%20J.%20McCarthy&amp;author=G.%20K.%20Smyth&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=139-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp616" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp616" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp616"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19910308" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ye</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Fang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Zheng</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">WEGO: a web tool for plotting GO annotations</div>, <div class="source">Nucleic Acids Res</div>  , <div class="volume">34</div>, <div class="fpage">W293</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=WEGO%3A%20a%20web%20tool%20for%20plotting%20GO%20annotations&amp;author=J.%20Ye&amp;author=L.%20Fang&amp;author=H.%20Zheng&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=W293-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845012" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Siegmund</div> <div class="given-names">K. D.</div></div>, <div class="name string-name"><div class="surname">Laird</div> <div class="given-names">P. W.</div></div>, <div class="name string-name"><div class="surname">Berman</div> <div class="given-names">B. P.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Bis-SNP: combined DNA methylation and SNP calling for Bisulfite-seq data</div>, <div class="source">Genome Biol</div> ., <div class="volume">13</div>, <div class="fpage">R61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bis-SNP%3A%20combined%20DNA%20methylation%20and%20SNP%20calling%20for%20Bisulfite-seq%20data&amp;author=Y.%20Liu&amp;author=K.%20D.%20Siegmund&amp;author=P.%20W.%20Laird&amp;author=B.%20P.%20Berman&amp;publication_year=2012&amp;journal=Genome%20Biol&amp;volume=13&amp;pages=R61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2012-13-7-r61" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2012-13-7-r61" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2012-13-7-r61"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22784381" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Feng</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Cokus</div> <div class="given-names">S. J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Conservation and divergence of methylation patterning in plants and animals</div>, <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> ., <div class="volume">107</div>, <div class="fpage">8689</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conservation%20and%20divergence%20of%20methylation%20patterning%20in%20plants%20and%20animals&amp;author=S.%20Feng&amp;author=S.%20J.%20Cokus&amp;author=X.%20Zhang&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=107&amp;pages=8689-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1002720107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1002720107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1002720107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20395551" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lisch</div> <div class="given-names">D.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Epigenetic regulation of transposable elements in plants</div>, <div class="source">Annu. Rev. Plant Biol</div> ., <div class="volume">60</div>, <div class="fpage">43</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenetic%20regulation%20of%20transposable%20elements%20in%20plants&amp;author=D.%20Lisch&amp;publication_year=2009&amp;journal=Annu.%20Rev.%20Plant%20Biol&amp;volume=60&amp;pages=43-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.59.032607.092744" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.59.032607.092744" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.59.032607.092744"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19007329" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cokus</div> <div class="given-names">S. J.</div></div>, <div class="name string-name"><div class="surname">Feng</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">Shotgun bisulphite sequencing of the Arabidopsis genome reveals DNA methylation patterning</div>, <div class="source">Nature</div> , <div class="volume">452</div>, <div class="fpage">215</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Shotgun%20bisulphite%20sequencing%20of%20the%20Arabidopsis%20genome%20reveals%20DNA%20methylation%20patterning&amp;author=S.%20J.%20Cokus&amp;author=S.%20Feng&amp;author=X.%20Zhang&amp;publication_year=2008&amp;journal=Nature&amp;volume=452&amp;pages=215" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06745" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06745" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06745"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18278030" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Qi</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">M.-W.</div></div>, <div class="name string-name"><div class="surname">Xie</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Identification of a novel salt tolerance gene in wild soybean by whole-genome sequencing</div>, <div class="source">Nat. Commun</div> ., <div class="volume">5</div>, <div class="fpage">4340</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20a%20novel%20salt%20tolerance%20gene%20in%20wild%20soybean%20by%20whole-genome%20sequencing&amp;author=X.%20Qi&amp;author=M.-W.%20Li&amp;author=M.%20Xie&amp;publication_year=2014&amp;journal=Nat.%20Commun&amp;volume=5&amp;pages=4340" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms5340" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms5340" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms5340"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25004933" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="collab">The Arabidopsis Genome, I</div>. <div class="year">2000</div>, 
<div class="article-title">Analysis of the genome sequence of the flowering plant Arabidopsis thaliana</div>, <div class="source">Nature</div> , <div class="volume">408</div>, <div class="fpage">796</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35048692" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35048692" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35048692"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11130711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Xia</div> <div class="given-names">E. H.</div></div>, <div class="name string-name"><div class="surname">Yao</div> <div class="given-names">Q. Y.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">X. D.</div></div>, <div class="name string-name"><div class="surname">Gao</div> <div class="given-names">L. Z.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Autotetraploid rice methylome analysis reveals methylation variation of transposable elements and their effects on gene expression</div>, <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> , <div class="volume">112</div>, <div class="fpage">E7022</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Autotetraploid%20rice%20methylome%20analysis%20reveals%20methylation%20variation%20of%20transposable%20elements%20and%20their%20effects%20on%20gene%20expression&amp;author=J.%20Zhang&amp;author=Y.%20Liu&amp;author=E.%20H.%20Xia&amp;author=Q.%20Y.%20Yao&amp;author=X.%20D.%20Liu&amp;author=L.%20Z.%20Gao&amp;publication_year=2015&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=112&amp;pages=E7022-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1515170112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1515170112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1515170112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26621743" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">F.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Single-base resolution maps of cultivated and wild rice methylomes and regulatory roles of DNA methylation in plant gene expression</div>, <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">300</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single-base%20resolution%20maps%20of%20cultivated%20and%20wild%20rice%20methylomes%20and%20regulatory%20roles%20of%20DNA%20methylation%20in%20plant%20gene%20expression&amp;author=X.%20Li&amp;author=J.%20Zhu&amp;author=F.%20Hu&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=300" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-300" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-300" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-300"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22747568" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Cao</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Rong</div> <div class="given-names">T.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Variation and patterns of DNA methylation in maize C-type CMS lines and their maintainers</div>, <div class="source">J. Plant Biochem. Biotechnol</div> ., <div class="volume">19</div>, <div class="fpage">43</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variation%20and%20patterns%20of%20DNA%20methylation%20in%20maize%20C-type%20CMS%20lines%20and%20their%20maintainers&amp;author=Y.%20Lu&amp;author=Y.%20Liu&amp;author=J.%20Wang&amp;author=M.%20Cao&amp;author=T.%20Rong&amp;publication_year=2010&amp;journal=J.%20Plant%20Biochem.%20Biotechnol&amp;volume=19&amp;pages=43-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF03323434" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF03323434" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF03323434"> </span></div> </div></div></div></div></div><div content-id="dsy008-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Ding</div> <div class="given-names">Y.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">DNA methylation changes in photoperiod-thermo-sensitive male sterile rice PA64S under two different conditions</div>, <div class="source">Gene</div> , <div class="volume">537</div>, <div class="fpage">143</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20changes%20in%20photoperiod-thermo-sensitive%20male%20sterile%20rice%20PA64S%20under%20two%20different%20conditions&amp;author=X.%20Chen&amp;author=J.%20Hu&amp;author=H.%20Zhang&amp;author=Y.%20Ding&amp;publication_year=2014&amp;journal=Gene&amp;volume=537&amp;pages=143-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2013.12.015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2013.12.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2013.12.015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24365594" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schmitz</div> <div class="given-names">R. J.</div></div>, <div class="name string-name"><div class="surname">Schultz</div> <div class="given-names">M. D.</div></div>, <div class="name string-name"><div class="surname">Lewsey</div> <div class="given-names">M. G.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Transgenerational epigenetic instability is a source of novel methylation variants</div>, <div class="source">Science</div> , <div class="volume">334</div>, <div class="fpage">369</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transgenerational%20epigenetic%20instability%20is%20a%20source%20of%20novel%20methylation%20variants&amp;author=R.%20J.%20Schmitz&amp;author=M.%20D.%20Schultz&amp;author=M.%20G.%20Lewsey&amp;publication_year=2011&amp;journal=Science&amp;volume=334&amp;pages=369-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1212959" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1212959" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1212959"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21921155" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Becker</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Hagmann</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Muller</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Spontaneous epigenetic variation in the Arabidopsis thaliana methylome</div>, <div class="source">Nature</div> , <div class="volume">480</div>, <div class="fpage">245</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Spontaneous%20epigenetic%20variation%20in%20the%20Arabidopsis%20thaliana%20methylome&amp;author=C.%20Becker&amp;author=J.%20Hagmann&amp;author=J.%20Muller&amp;publication_year=2011&amp;journal=Nature&amp;volume=480&amp;pages=245-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10555" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10555" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10555"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22057020" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Eichten</div> <div class="given-names">S. R.</div></div>, <div class="name string-name"><div class="surname">Briskine</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Song</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Epigenetic and genetic influences on DNA methylation variation in maize populations</div>, <div class="source">Plant Cell</div> , <div class="volume">25</div>, <div class="fpage">2783</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenetic%20and%20genetic%20influences%20on%20DNA%20methylation%20variation%20in%20maize%20populations&amp;author=S.%20R.%20Eichten&amp;author=R.%20Briskine&amp;author=J.%20Song&amp;publication_year=2013&amp;journal=Plant%20Cell&amp;volume=25&amp;pages=2783-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.113.114793" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.113.114793" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.113.114793"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23922207" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Cui</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Horner</div> <div class="given-names">H. T.</div></div>, <div class="name string-name"><div class="surname">Weiner</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Schnable</div> <div class="given-names">P. S.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Mitochondrial aldehyde dehydrogenase activity is required for male fertility in maize</div>, <div class="source">Plant Cell</div> , <div class="volume">13</div>, <div class="fpage">1063</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mitochondrial%20aldehyde%20dehydrogenase%20activity%20is%20required%20for%20male%20fertility%20in%20maize&amp;author=F.%20Liu&amp;author=X.%20Cui&amp;author=H.%20T.%20Horner&amp;author=H.%20Weiner&amp;author=P.%20S.%20Schnable&amp;publication_year=2001&amp;journal=Plant%20Cell&amp;volume=13&amp;pages=1063-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.13.5.1063" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.13.5.1063" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.13.5.1063"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11340182" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huang</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Pentatricopeptide-repeat family protein RF6 functions with hexokinase 6 to rescue rice cytoplasmic male sterility</div>, <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> , <div class="volume">112</div>, <div class="fpage">14984</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pentatricopeptide-repeat%20family%20protein%20RF6%20functions%20with%20hexokinase%206%20to%20rescue%20rice%20cytoplasmic%20male%20sterility&amp;author=W.%20Huang&amp;author=C.%20Yu&amp;author=J.%20Hu&amp;publication_year=2015&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=112&amp;pages=14984-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1511748112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1511748112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1511748112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26578814" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Song</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Guan</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">Z. J.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Dynamic roles for small RNAs and DNA methylation during ovule and fiber development in allotetraploid cotton</div>, <div class="source">PLoS Genet</div> ., <div class="volume">11</div>, <div class="fpage">e1005724</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20roles%20for%20small%20RNAs%20and%20DNA%20methylation%20during%20ovule%20and%20fiber%20development%20in%20allotetraploid%20cotton&amp;author=Q.%20Song&amp;author=X.%20Guan&amp;author=Z.%20J.%20Chen&amp;publication_year=2015&amp;journal=PLoS%20Genet&amp;volume=11&amp;pages=e1005724" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1005724" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1005724" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1005724"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26710171" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xu</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Dong</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">D. Z.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">A.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Genomic DNA methylation analyses reveal the distinct profiles in castor bean seeds with persistent endosperms</div>, <div class="source">Plant Physiol</div> ., <div class="volume">171</div>, <div class="fpage">1242</div>–<div class="lpage">58</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20DNA%20methylation%20analyses%20reveal%20the%20distinct%20profiles%20in%20castor%20bean%20seeds%20with%20persistent%20endosperms&amp;author=W.%20Xu&amp;author=T.%20Yang&amp;author=X.%20Dong&amp;author=D.%20Z.%20Li&amp;author=A.%20Liu&amp;publication_year=2016&amp;journal=Plant%20Physiol&amp;volume=171&amp;pages=1242-58" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27208275" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Ren</div> <div class="given-names">W.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Genome-wide epigenetic regulation of gene transcription in maize seeds</div>, <div class="source">PLoS One</div> , <div class="volume">10</div>, <div class="fpage">e0139582</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20epigenetic%20regulation%20of%20gene%20transcription%20in%20maize%20seeds&amp;author=X.%20Lu&amp;author=W.%20Wang&amp;author=W.%20Ren&amp;publication_year=2015&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0139582" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0139582" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0139582" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0139582"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26469520" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Garg</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Narayana Chevala</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Shankar</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Jain</div> <div class="given-names">M.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Divergent DNA methylation patterns associated with gene expression in rice cultivars with contrasting drought and salinity stress response</div>, <div class="source">Sci. Rep</div> ., <div class="volume">5</div>, <div class="fpage">14922</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Divergent%20DNA%20methylation%20patterns%20associated%20with%20gene%20expression%20in%20rice%20cultivars%20with%20contrasting%20drought%20and%20salinity%20stress%20response&amp;author=R.%20Garg&amp;author=V.%20Narayana%20Chevala&amp;author=R.%20Shankar&amp;author=M.%20Jain&amp;publication_year=2015&amp;journal=Sci.%20Rep&amp;volume=5&amp;pages=14922" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep14922" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep14922" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep14922"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26449881" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Song</div> <div class="given-names">Q. X.</div></div>, <div class="name string-name"><div class="surname">Lu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Q. T.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Genome-wide analysis of DNA methylation in soybean</div>, <div class="source">Mol. Plant</div> ., <div class="volume">6</div>, <div class="fpage">1961</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20DNA%20methylation%20in%20soybean&amp;author=Q.%20X.%20Song&amp;author=X.%20Lu&amp;author=Q.%20T.%20Li&amp;publication_year=2013&amp;journal=Mol.%20Plant&amp;volume=6&amp;pages=1961-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/mp/sst123" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmp%2Fsst123" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmp%2Fsst123"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23966636" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Matzke</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kanno</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Daxinger</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Huettel</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Matzke</div> <div class="given-names">A. J.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">RNA-mediated chromatin-based silencing in plants</div>, <div class="source">Curr. Opin. Cell Biol</div> ., <div class="volume">21</div>, <div class="fpage">367</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA-mediated%20chromatin-based%20silencing%20in%20plants&amp;author=M.%20Matzke&amp;author=T.%20Kanno&amp;author=L.%20Daxinger&amp;author=B.%20Huettel&amp;author=A.%20J.%20Matzke&amp;publication_year=2009&amp;journal=Curr.%20Opin.%20Cell%20Biol&amp;volume=21&amp;pages=367-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ceb.2009.01.025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ceb.2009.01.025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ceb.2009.01.025"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19243928" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Inagaki</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kakutani</div> <div class="given-names">T.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">What triggers differential DNA methylation of genes and TEs: contribution of body methylation?</div> <div class="source">Cold Spring Harb. Symp. Quant. Biol</div> ., <div class="volume">77</div>, <div class="fpage">155</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=What%20triggers%20differential%20DNA%20methylation%20of%20genes%20and%20TEs%3A%20contribution%20of%20body%20methylation%3F&amp;author=S.%20Inagaki&amp;author=T.%20Kakutani&amp;publication_year=2012&amp;journal=Cold%20Spring%20Harb.%20Symp.%20Quant.%20Biol&amp;volume=77&amp;pages=155-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/sqb.2013.77.016212" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fsqb.2013.77.016212" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fsqb.2013.77.016212"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23302809" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zakrzewski</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Schmidt</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Van Lijsebettens</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Schmidt</div> <div class="given-names">T.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">DNA methylation of retrotransposons, DNA transposons and genes in sugar beet (Beta vulgaris L.)</div>, <div class="source">Plant J</div> ., <div class="volume">90</div>, <div class="fpage">1156</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20of%20retrotransposons%2C%20DNA%20transposons%20and%20genes%20in%20sugar%20beet%20(Beta%20vulgaris%20L.)&amp;author=F.%20Zakrzewski&amp;author=M.%20Schmidt&amp;author=M.%20Van%20Lijsebettens&amp;author=T.%20Schmidt&amp;publication_year=2017&amp;journal=Plant%20J&amp;volume=90&amp;pages=1156-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2017.90.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2017.90.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2017.90.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28257158" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">West</div> <div class="given-names">P. T.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Ji</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Genomic distribution of H3K9me2 and DNA methylation in a maize genome</div>, <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e105267</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20distribution%20of%20H3K9me2%20and%20DNA%20methylation%20in%20a%20maize%20genome&amp;author=P.%20T.%20West&amp;author=Q.%20Li&amp;author=L.%20Ji&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e105267" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0105267" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0105267" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0105267"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25122127" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gent</div> <div class="given-names">J. I.</div></div>, <div class="name string-name"><div class="surname">Ellis</div> <div class="given-names">N. A.</div></div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">CHH islands: de novo DNA methylation in near-gene chromatin regulation in maize</div>, <div class="source">Genome Res</div> ., <div class="volume">23</div>, <div class="fpage">628</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CHH%20islands%3A%20de%20novo%20DNA%20methylation%20in%20near-gene%20chromatin%20regulation%20in%20maize&amp;author=J.%20I.%20Gent&amp;author=N.%20A.%20Ellis&amp;author=L.%20Guo&amp;publication_year=2013&amp;journal=Genome%20Res&amp;volume=23&amp;pages=628-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.146985.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.146985.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.146985.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23269663" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Martienssen</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Barkan</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Taylor</div> <div class="given-names">W. C.</div></div>, <div class="name string-name"><div class="surname">Freeling</div> <div class="given-names">M.</div></div></div> <div class="year">1990</div>, 
<div class="article-title">Somatically heritable switches in the DNA modification of Mu transposable elements monitored with a suppressible mutant in maize</div>, <div class="source">Genes Dev</div> ., <div class="volume">4</div>, <div class="fpage">331</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Somatically%20heritable%20switches%20in%20the%20DNA%20modification%20of%20Mu%20transposable%20elements%20monitored%20with%20a%20suppressible%20mutant%20in%20maize&amp;author=R.%20Martienssen&amp;author=A.%20Barkan&amp;author=W.%20C.%20Taylor&amp;author=M.%20Freeling&amp;publication_year=1990&amp;journal=Genes%20Dev&amp;volume=4&amp;pages=331-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.4.3.331" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.4.3.331" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.4.3.331"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2159936" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shi</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Dong</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Shen</div> <div class="given-names">W. H.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Epigenetic regulation of rice flowering and reproduction</div>, <div class="source">Front. Plant Sci</div> ., <div class="volume">5</div>, <div class="fpage">803</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenetic%20regulation%20of%20rice%20flowering%20and%20reproduction&amp;author=J.%20Shi&amp;author=A.%20Dong&amp;author=W.%20H.%20Shen&amp;publication_year=2014&amp;journal=Front.%20Plant%20Sci&amp;volume=5&amp;pages=803" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25674094" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Higo</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Tahir</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Takashima</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">DDM1 (decrease in DNA methylation) genes in rice (Oryza sativa)</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="volume">287</div>, <div class="fpage">785</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DDM1%20(decrease%20in%20DNA%20methylation)%20genes%20in%20rice%20(Oryza%20sativa)&amp;author=H.%20Higo&amp;author=M.%20Tahir&amp;author=K.%20Takashima&amp;publication_year=2012&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=287&amp;pages=785-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-012-0717-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-012-0717-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-012-0717-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22915302" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Choi</div> <div class="given-names">S. C.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">S. R.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Trithorax group protein Oryza sativa Trithorax1 controls flowering time in rice via interaction with early heading date3</div>, <div class="source">Plant Physiol</div> ., <div class="volume">164</div>, <div class="fpage">1326</div>–<div class="lpage">37</div>.,<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Trithorax%20group%20protein%20Oryza%20sativa%20Trithorax1%20controls%20flowering%20time%20in%20rice%20via%20interaction%20with%20early%20heading%20date3&amp;author=S.%20C.%20Choi&amp;author=S.%20Lee&amp;author=S.%20R.%20Kim&amp;publication_year=2014&amp;journal=Plant%20Physiol&amp;volume=164&amp;pages=1326-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.113.228049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.113.228049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.113.228049"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24420930" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Yazaki</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Sundaresan</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">Genome-wide high-resolution mapping and functional analysis of DNA methylation in arabidopsis</div>, <div class="source">Cell</div> , <div class="volume">126</div>, <div class="fpage">1189</div>–<div class="lpage">201</div>.,<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20high-resolution%20mapping%20and%20functional%20analysis%20of%20DNA%20methylation%20in%20arabidopsis&amp;author=X.%20Zhang&amp;author=J.%20Yazaki&amp;author=A.%20Sundaresan&amp;publication_year=2006&amp;journal=Cell&amp;volume=126&amp;pages=1189-201" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2006.08.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2006.08.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2006.08.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16949657" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zilberman</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Gehring</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Tran</div> <div class="given-names">R. K.</div></div>, <div class="name string-name"><div class="surname">Ballinger</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Henikoff</div> <div class="given-names">S.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Genome-wide analysis of Arabidopsis thaliana DNA methylation uncovers an interdependence between methylation and transcription</div>, <div class="source">Nat Genet</div> ., <div class="volume">39</div>, <div class="fpage">61</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20Arabidopsis%20thaliana%20DNA%20methylation%20uncovers%20an%20interdependence%20between%20methylation%20and%20transcription&amp;author=D.%20Zilberman&amp;author=M.%20Gehring&amp;author=R.%20K.%20Tran&amp;author=T.%20Ballinger&amp;author=S.%20Henikoff&amp;publication_year=2007&amp;journal=Nat%20Genet&amp;volume=39&amp;pages=61-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1929" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1929" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1929"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17128275" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy008-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy008-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schmitz</div> <div class="given-names">R. J.</div></div>, <div class="name string-name"><div class="surname">He</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Valdes-Lopez</div> <div class="given-names">O.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Epigenome-wide inheritance of cytosine methylation variants in a recombinant inbred population</div>, <div class="source">Genome Res</div> ., <div class="volume">23</div>, <div class="fpage">1663</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenome-wide%20inheritance%20of%20cytosine%20methylation%20variants%20in%20a%20recombinant%20inbred%20population&amp;author=R.%20J.%20Schmitz&amp;author=Y.%20He&amp;author=O.%20Valdes-Lopez&amp;publication_year=2013&amp;journal=Genome%20Res&amp;volume=23&amp;pages=1663-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.152538.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.152538.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.152538.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23739894" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2018. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4939498&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy008/1/dsy008_supp.zip?Expires=2147483647&amp;Signature=a5pxiGBiZuW6BLObg3zcGti-PLIrtW4lDnTZKtyw3m~A3SRkZcr0Y2ve6moKjtosqKCTHkcpfiWI1YiZ59ZYjagra1cBx2qHidDiS9YD76AyCdt-4~W-54Oj7mbqe~VnqVToy5n7EnQdmwYondGxoqfU4SGnxtqtHyNZnOY--6wL3vaY64N6y9Ol6VaEdmhxlHtZ9zskpG0jWypn5ZEvlgg-2NnJYoPL6EOKqxhPhKOweQ9WNBLLfKlYzWiqy7TPWCchHeatrk45nHlTmBeMjmhFGIHpav47NDhdJbGqGb2EIjKAOQuctS37uI7xAHUHDrLIQQ~VOnhtRcc1h0JCcw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,951</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsy008"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4939498"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f4%2f361%2f4939498">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F4%2F361%2F4939498" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4939498'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Unravelling the epigenomic interactions between parental inbreds resulting in an altered hybrid methylome in pigeonpea" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy008");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f361%2f4939498">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

