

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/3/271/2965793","@type":"ScholarlyArticle","name":"Prediction of bipartite transcriptional regulatory elements using transcriptome data of Arabidopsis","datePublished":"2017-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2017-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw065","keywords":["promoter prediction","transcriptome","plant genome"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/3/271/2965793","author":[{"name":"Yamamoto, Yoshiharu Y.","affiliation":"1 Faculty of Applied Biological Sciences, Gifu University, 1-1 Yanagido, Gifu City, Gifu 501-1193, Japan   2 United Graduate School of Agricultural Science, Gifu University, Yanagido 1-1, Gifu City, Gifu 501-1193, Japan   3 RIKEN CSRS, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama 230-0045, Japan   4 JST ALCA","@type":"Person"},{"name":"Ichida, Hiroyuki","affiliation":"5 RIKEN Nishina Center, 2-1 Hirosawa, Wako, Saitama 351-0198, Japan","@type":"Person"},{"name":"Hieno, Ayaka","affiliation":"2 United Graduate School of Agricultural Science, Gifu University, Yanagido 1-1, Gifu City, Gifu 501-1193, Japan","@type":"Person"},{"name":"Obata, Daichi","affiliation":"1 Faculty of Applied Biological Sciences, Gifu University, 1-1 Yanagido, Gifu City, Gifu 501-1193, Japan","@type":"Person"},{"name":"Tokizawa, Mutsutomo","affiliation":"2 United Graduate School of Agricultural Science, Gifu University, Yanagido 1-1, Gifu City, Gifu 501-1193, Japan","@type":"Person"},{"name":"Nomoto, Mika","affiliation":"6 Center for Gene Research, Nagoya University, Furo-cho, Chikusa, Nagoya 464-8602, Japan","@type":"Person"},{"name":"Tada, Yasuomi","affiliation":"6 Center for Gene Research, Nagoya University, Furo-cho, Chikusa, Nagoya 464-8602, Japan","@type":"Person"},{"name":"Kusunoki, Kazutaka","affiliation":"2 United Graduate School of Agricultural Science, Gifu University, Yanagido 1-1, Gifu City, Gifu 501-1193, Japan","@type":"Person"},{"name":"Koyama, Hiroyuki","affiliation":"1 Faculty of Applied Biological Sciences, Gifu University, 1-1 Yanagido, Gifu City, Gifu 501-1193, Japan   2 United Graduate School of Agricultural Science, Gifu University, Yanagido 1-1, Gifu City, Gifu 501-1193, Japan","@type":"Person"},{"name":"Hayami, Natsuki","affiliation":"2 United Graduate School of Agricultural Science, Gifu University, Yanagido 1-1, Gifu City, Gifu 501-1193, Japan","@type":"Person"}],"description":"Abstract.  In our previous study, a methodology was established to predict transcriptional regulatory elements in promoter sequences using transcriptome data ba","pageStart":"271","pageEnd":"278","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/3/1/m_dnares_24_3cover.png?Expires=2147483647&Signature=eoXc11KAMpHEsaqW0C6T2VF4hIzrR2AQ2xJtZ-I1SzuN8Ozcm3pfcXhSESBXVMgST8o0VjwPqT85KWv95Ep~BtVLeJis8VsXRFBVpMHhb~g9dvz54ejvU8kSReDsueZzPSdTwZuEswyhmEGVK~9fmtkzww9NnyhhDFzj14yR9QQPP3D5RCa40I2kb6S3cosdNYyeAUAcZmppIJGZquGgABfHKwfV6R3Ffo5Kcif39o76ejsFOOlH0X60OYvM4Yme1Vyjd2eLo4R5~a0mH9tf6MZMrQpFP4iVZGwKwAheMFv-ISdx3BcAb4Re6~N24GwZYpxXAywMLeRm98GmIDGpbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Prediction of bipartite transcriptional regulatory elements using transcriptome data of Arabidopsis","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/3/1/m_dnares_24_3cover.png?Expires=2147483647&Signature=eoXc11KAMpHEsaqW0C6T2VF4hIzrR2AQ2xJtZ-I1SzuN8Ozcm3pfcXhSESBXVMgST8o0VjwPqT85KWv95Ep~BtVLeJis8VsXRFBVpMHhb~g9dvz54ejvU8kSReDsueZzPSdTwZuEswyhmEGVK~9fmtkzww9NnyhhDFzj14yR9QQPP3D5RCa40I2kb6S3cosdNYyeAUAcZmppIJGZquGgABfHKwfV6R3Ffo5Kcif39o76ejsFOOlH0X60OYvM4Yme1Vyjd2eLo4R5~a0mH9tf6MZMrQpFP4iVZGwKwAheMFv-ISdx3BcAb4Re6~N24GwZYpxXAywMLeRm98GmIDGpbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Prediction of bipartite transcriptional regulatory elements using transcriptome data of Arabidopsis" />
    <meta property="og:description" content="Abstract.  In our previous study, a methodology was established to predict transcriptional regulatory elements in promoter sequences using transcriptome data ba" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/3/271/2965793" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-02-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Yamamoto, Yoshiharu Y." /><meta name="citation_author_institution" content="Faculty of Applied Biological Sciences, Gifu University, 1-1 Yanagido, Gifu City, Gifu 501-1193, Japan" /><meta name="citation_author_institution" content="United Graduate School of Agricultural Science, Gifu University, Yanagido 1-1, Gifu City, Gifu 501-1193, Japan" /><meta name="citation_author_institution" content="RIKEN CSRS, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama 230-0045, Japan" /><meta name="citation_author_institution" content="JST ALCA" /><meta name="citation_author" content="Ichida, Hiroyuki" /><meta name="citation_author_institution" content="RIKEN Nishina Center, 2-1 Hirosawa, Wako, Saitama 351-0198, Japan" /><meta name="citation_author" content="Hieno, Ayaka" /><meta name="citation_author_institution" content="United Graduate School of Agricultural Science, Gifu University, Yanagido 1-1, Gifu City, Gifu 501-1193, Japan" /><meta name="citation_author" content="Obata, Daichi" /><meta name="citation_author_institution" content="Faculty of Applied Biological Sciences, Gifu University, 1-1 Yanagido, Gifu City, Gifu 501-1193, Japan" /><meta name="citation_author" content="Tokizawa, Mutsutomo" /><meta name="citation_author_institution" content="United Graduate School of Agricultural Science, Gifu University, Yanagido 1-1, Gifu City, Gifu 501-1193, Japan" /><meta name="citation_author" content="Nomoto, Mika" /><meta name="citation_author_institution" content="Center for Gene Research, Nagoya University, Furo-cho, Chikusa, Nagoya 464-8602, Japan" /><meta name="citation_author" content="Tada, Yasuomi" /><meta name="citation_author_institution" content="Center for Gene Research, Nagoya University, Furo-cho, Chikusa, Nagoya 464-8602, Japan" /><meta name="citation_author" content="Kusunoki, Kazutaka" /><meta name="citation_author_institution" content="United Graduate School of Agricultural Science, Gifu University, Yanagido 1-1, Gifu City, Gifu 501-1193, Japan" /><meta name="citation_author" content="Koyama, Hiroyuki" /><meta name="citation_author_institution" content="Faculty of Applied Biological Sciences, Gifu University, 1-1 Yanagido, Gifu City, Gifu 501-1193, Japan" /><meta name="citation_author_institution" content="United Graduate School of Agricultural Science, Gifu University, Yanagido 1-1, Gifu City, Gifu 501-1193, Japan" /><meta name="citation_author" content="Hayami, Natsuki" /><meta name="citation_author_institution" content="United Graduate School of Agricultural Science, Gifu University, Yanagido 1-1, Gifu City, Gifu 501-1193, Japan" /><meta name="citation_title" content="Prediction of bipartite transcriptional regulatory elements using transcriptome data of Arabidopsis" /><meta name="citation_firstpage" content="271" /><meta name="citation_lastpage" content="278" /><meta name="citation_doi" content="10.1093/dnares/dsw065" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2017/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Detecting subtle sequence signals: a Gibbs sampling strategy for multiple alignment;  citation_journal_title=Science;  citation_year=1993;  citation_volume=262;  citation_pages=208-14; " /><meta name="citation_reference" content="citation_title=The value of prior knowledge in discovering motifs with MEME;  citation_journal_title=Proc. Int. Conf. Intell. Syst. Mol. Biol;  citation_year=1995;  citation_volume=3;  citation_pages=21-9; " /><meta name="citation_reference" content="citation_title=Prediction of transcriptional regulatory elements for plant hormone responses based on microarray data;  citation_journal_title=BMC Plant Biol;  citation_year=2011;  citation_volume=11;  citation_pages=39" /><meta name="citation_reference" content="citation_title=High REDOX RESPONSIVE TRANSCRIPTION FACTOR1 levels result in accumulation of reactive oxygen species in Arabidopsis thaliana shoots and roots;  citation_journal_title=Mol. Plant;  citation_year=2015;  citation_volume=8;  citation_pages=1253-73; " /><meta name="citation_reference" content="citation_title=SENSITIVE TO PROTON RHIZOTOXICITY1, CALMODULIN BINDING TRANSCRIPTION ACTIVATOR2, and other transcription factors are involved in ALUMINUM-ACTIVATED MALATE TRANSPORTER1 expression;  citation_journal_title=Plant Physiol;  citation_year=2015;  citation_volume=167;  citation_pages=991-1003; " /><meta name="citation_reference" content="citation_title=The responses of Arabidopsis ELIP2 to UV-B, high light, and cold stress are regulated by a transcriptional regulatory unit composed of two elements;  citation_journal_title=Plant Physiol;  citation_year=2015;  citation_volume=169;  citation_pages=840-55; " /><meta name="citation_reference" content="citation_title=JASPAR 2016: a major expansion and update of the open-access database of transcription factor binding profiles;  citation_journal_title=Nucleic Acids Res;  citation_year=2016;  citation_volume=44;  citation_pages=D110-115; " /><meta name="citation_reference" content="citation_title=Functional analysis of transcription factors in Arabidopsis;  citation_journal_title=Plant Cell Physiol;  citation_year=2009;  citation_volume=50;  citation_pages=1232-48; " /><meta name="citation_reference" content="citation_title=ppdb: plant promoter database version 3.0;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=D1188-92; " /><meta name="citation_reference" content="citation_title=Identification of plant promoter constituents by analysis of local distribution of short sequences;  citation_journal_title=BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=67" /><meta name="citation_reference" content="citation_title=Cluster analysis and display of genome-wide expression patterns;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=1998;  citation_volume=95;  citation_pages=14863-8; " /><meta name="citation_reference" content="citation_title=WebLogo: a sequence logo generator;  citation_journal_title=Genome Res;  citation_year=2004;  citation_volume=14;  citation_pages=1188-90; " /><meta name="citation_reference" content="citation_title=Arabidopsis IRE1 catalyses unconventional splicing of bZIP60 mRNA to produce the active transcription factor;  citation_journal_title=Sci. Rep;  citation_year=2011;  citation_volume=1;  citation_pages=29" /><meta name="citation_reference" content="citation_title=A membrane-bound NAC transcription factor, ANAC017, mediates mitochondrial retrograde signaling in Arabidopsis;  citation_journal_title=Plant Cell;  citation_year=2013;  citation_volume=25;  citation_pages=3450-71; " /><meta name="citation_reference" content="citation_title=Analysis of environmental stress in plants with the aid of marker genes for H2O2 responses;  citation_journal_title=Methods Enzymol;  citation_year=2013;  citation_volume=527;  citation_pages=221-37; " /><meta name="citation_reference" content="citation_title=Plant transducers of the endoplasmic reticulum unfolded protein response;  citation_journal_title=Trends Plant Sci;  citation_year=2012;  citation_volume=17;  citation_pages=720-7; " /><meta name="citation_reference" content="citation_title=Endoplasmic reticulum stress-induced formation of transcription factor complex ERSF including NF-Y (CBF) and activating transcription factors 6alpha and 6beta that activates the mammalian unfolded protein response;  citation_journal_title=Mol. Cell. Biol;  citation_year=2001;  citation_volume=21;  citation_pages=1239-48; " /><meta name="citation_reference" content="citation_title=bZIP28 and NF-Y transcription factors are activated by ER stress and assemble into a transcriptional complex to regulate stress response genes in Arabidopsis;  citation_journal_title=Plant Cell;  citation_year=2010;  citation_volume=22;  citation_pages=782-96; " /><meta name="citation_reference" content="citation_title=Genomic analysis of the unfolded protein response in Arabidopsis shows its connection to important cellular processes;  citation_journal_title=Plant Cell;  citation_year=2003;  citation_volume=15;  citation_pages=561-76; " /><meta name="citation_reference" content="citation_title=The membrane-bound NAC transcription factor ANAC013 functions in mitochondrial retrograde regulation of the oxidative stress response in Arabidopsis;  citation_journal_title=Plant Cell;  citation_year=2013;  citation_volume=25;  citation_pages=3472-90; " /><meta name="citation_reference" content="citation_title=Expression of an evolutionarily distinct novel BiP gene during the unfolded protein response in Arabidopsis thaliana;  citation_journal_title=Gene;  citation_year=2003;  citation_volume=311;  citation_pages=81-91; " /><meta name="citation_reference" content="citation_title=Melina II: a web tool for comparisons among several predictive algorithms to find potential motifs from promoter regions;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=W227-31; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/3/271/17743963/dsw065.pdf" /><meta name="description" content="Abstract.  In our previous study, a methodology was established to predict transcriptional regulatory elements in promoter sequences using transcriptome data ba" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/3/271/2965793" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/3/271/2965793" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/3/271/2965793");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw065';
    </script>
<title>
	Prediction of bipartite transcriptional regulatory elements using transcriptome data of Arabidopsis | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2965793" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYJMPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZYW1hbW90bywgWW9zaGloYXJ1IFkuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRmFjdWx0eSBvZiBBcHBsaWVkIEJpb2xvZ2ljYWwgU2NpZW5jZXMsIEdpZnUgVW5pdmVyc2l0eSwgMS0xIFlhbmFnaWRvLCBHaWZ1IENpdHksIEdpZnUgNTAxLTExOTMsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVW5pdGVkIEdyYWR1YXRlIFNjaG9vbCBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZSwgR2lmdSBVbml2ZXJzaXR5LCBZYW5hZ2lkbyAxLTEsIEdpZnUgQ2l0eSwgR2lmdSA1MDEtMTE5MywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSSUtFTiBDU1JTLCAxLTctMjIgU3VlaGlyby1jaG8sIFRzdXJ1bWkta3UsIFlva29oYW1hIDIzMC0wMDQ1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkpTVCBBTENBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSWNoaWRhLCBIaXJveXVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlJJS0VOIE5pc2hpbmEgQ2VudGVyLCAyLTEgSGlyb3Nhd2EsIFdha28sIFNhaXRhbWEgMzUxLTAxOTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGllbm8sIEF5YWthIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVW5pdGVkIEdyYWR1YXRlIFNjaG9vbCBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZSwgR2lmdSBVbml2ZXJzaXR5LCBZYW5hZ2lkbyAxLTEsIEdpZnUgQ2l0eSwgR2lmdSA1MDEtMTE5MywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPYmF0YSwgRGFpY2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRmFjdWx0eSBvZiBBcHBsaWVkIEJpb2xvZ2ljYWwgU2NpZW5jZXMsIEdpZnUgVW5pdmVyc2l0eSwgMS0xIFlhbmFnaWRvLCBHaWZ1IENpdHksIEdpZnUgNTAxLTExOTMsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVG9raXphd2EsIE11dHN1dG9tbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVuaXRlZCBHcmFkdWF0ZSBTY2hvb2wgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2UsIEdpZnUgVW5pdmVyc2l0eSwgWWFuYWdpZG8gMS0xLCBHaWZ1IENpdHksIEdpZnUgNTAxLTExOTMsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTm9tb3RvLCBNaWthIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudGVyIGZvciBHZW5lIFJlc2VhcmNoLCBOYWdveWEgVW5pdmVyc2l0eSwgRnVyby1jaG8sIENoaWt1c2EsIE5hZ295YSA0NjQtODYwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWRhLCBZYXN1b21pIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudGVyIGZvciBHZW5lIFJlc2VhcmNoLCBOYWdveWEgVW5pdmVyc2l0eSwgRnVyby1jaG8sIENoaWt1c2EsIE5hZ295YSA0NjQtODYwMiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLdXN1bm9raSwgS2F6dXRha2EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJVbml0ZWQgR3JhZHVhdGUgU2Nob29sIG9mIEFncmljdWx0dXJhbCBTY2llbmNlLCBHaWZ1IFVuaXZlcnNpdHksIFlhbmFnaWRvIDEtMSwgR2lmdSBDaXR5LCBHaWZ1IDUwMS0xMTkzLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktveWFtYSwgSGlyb3l1a2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGYWN1bHR5IG9mIEFwcGxpZWQgQmlvbG9naWNhbCBTY2llbmNlcywgR2lmdSBVbml2ZXJzaXR5LCAxLTEgWWFuYWdpZG8sIEdpZnUgQ2l0eSwgR2lmdSA1MDEtMTE5MywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJVbml0ZWQgR3JhZHVhdGUgU2Nob29sIG9mIEFncmljdWx0dXJhbCBTY2llbmNlLCBHaWZ1IFVuaXZlcnNpdHksIFlhbmFnaWRvIDEtMSwgR2lmdSBDaXR5LCBHaWZ1IDUwMS0xMTkzLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhheWFtaSwgTmF0c3VraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVuaXRlZCBHcmFkdWF0ZSBTY2hvb2wgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2UsIEdpZnUgVW5pdmVyc2l0eSwgWWFuYWdpZG8gMS0xLCBHaWZ1IENpdHksIEdpZnUgNTAxLTExOTMsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJQcmVkaWN0aW9uIG9mIGJpcGFydGl0ZSB0cmFuc2NyaXB0aW9uYWwgcmVndWxhdG9yeSBlbGVtZW50cyB1c2luZyB0cmFuc2NyaXB0b21lIGRhdGEgb2YgQXJhYmlkb3BzaXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyNzEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjI3OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdzA2NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE3LzA2LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZWN0aW5nIHN1YnRsZSBzZXF1ZW5jZSBzaWduYWxzOiBhIEdpYmJzIHNhbXBsaW5nIHN0cmF0ZWd5IGZvciBtdWx0aXBsZSBhbGlnbm1lbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MjYyOyAgY2l0YXRpb25fcGFnZXM9MjA4LTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgdmFsdWUgb2YgcHJpb3Iga25vd2xlZGdlIGluIGRpc2NvdmVyaW5nIG1vdGlmcyB3aXRoIE1FTUU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIEludC4gQ29uZi4gSW50ZWxsLiBTeXN0LiBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTIxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWRpY3Rpb24gb2YgdHJhbnNjcmlwdGlvbmFsIHJlZ3VsYXRvcnkgZWxlbWVudHMgZm9yIHBsYW50IGhvcm1vbmUgcmVzcG9uc2VzIGJhc2VkIG9uIG1pY3JvYXJyYXkgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0zOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2ggUkVET1ggUkVTUE9OU0lWRSBUUkFOU0NSSVBUSU9OIEZBQ1RPUjEgbGV2ZWxzIHJlc3VsdCBpbiBhY2N1bXVsYXRpb24gb2YgcmVhY3RpdmUgb3h5Z2VuIHNwZWNpZXMgaW4gQXJhYmlkb3BzaXMgdGhhbGlhbmEgc2hvb3RzIGFuZCByb290czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBQbGFudDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MTI1My03MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U0VOU0lUSVZFIFRPIFBST1RPTiBSSElaT1RPWElDSVRZMSwgQ0FMTU9EVUxJTiBCSU5ESU5HIFRSQU5TQ1JJUFRJT04gQUNUSVZBVE9SMiwgYW5kIG90aGVyIHRyYW5zY3JpcHRpb24gZmFjdG9ycyBhcmUgaW52b2x2ZWQgaW4gQUxVTUlOVU0tQUNUSVZBVEVEIE1BTEFURSBUUkFOU1BPUlRFUjEgZXhwcmVzc2lvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjc7ICBjaXRhdGlvbl9wYWdlcz05OTEtMTAwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHJlc3BvbnNlcyBvZiBBcmFiaWRvcHNpcyBFTElQMiB0byBVVi1CLCBoaWdoIGxpZ2h0LCBhbmQgY29sZCBzdHJlc3MgYXJlIHJlZ3VsYXRlZCBieSBhIHRyYW5zY3JpcHRpb25hbCByZWd1bGF0b3J5IHVuaXQgY29tcG9zZWQgb2YgdHdvIGVsZW1lbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OTsgIGNpdGF0aW9uX3BhZ2VzPTg0MC01NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SkFTUEFSIDIwMTY6IGEgbWFqb3IgZXhwYW5zaW9uIGFuZCB1cGRhdGUgb2YgdGhlIG9wZW4tYWNjZXNzIGRhdGFiYXNlIG9mIHRyYW5zY3JpcHRpb24gZmFjdG9yIGJpbmRpbmcgcHJvZmlsZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9RDExMC0xMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmN0aW9uYWwgYW5hbHlzaXMgb2YgdHJhbnNjcmlwdGlvbiBmYWN0b3JzIGluIEFyYWJpZG9wc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsIFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NTA7ICBjaXRhdGlvbl9wYWdlcz0xMjMyLTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1wcGRiOiBwbGFudCBwcm9tb3RlciBkYXRhYmFzZSB2ZXJzaW9uIDMuMDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz1EMTE4OC05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgcGxhbnQgcHJvbW90ZXIgY29uc3RpdHVlbnRzIGJ5IGFuYWx5c2lzIG9mIGxvY2FsIGRpc3RyaWJ1dGlvbiBvZiBzaG9ydCBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbHVzdGVyIGFuYWx5c2lzIGFuZCBkaXNwbGF5IG9mIGdlbm9tZS13aWRlIGV4cHJlc3Npb24gcGF0dGVybnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS4gUy4gQTsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT05NTsgIGNpdGF0aW9uX3BhZ2VzPTE0ODYzLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdlYkxvZ286IGEgc2VxdWVuY2UgbG9nbyBnZW5lcmF0b3I7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xMTg4LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BcmFiaWRvcHNpcyBJUkUxIGNhdGFseXNlcyB1bmNvbnZlbnRpb25hbCBzcGxpY2luZyBvZiBiWklQNjAgbVJOQSB0byBwcm9kdWNlIHRoZSBhY3RpdmUgdHJhbnNjcmlwdGlvbiBmYWN0b3I7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gUmVwOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0yOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbWVtYnJhbmUtYm91bmQgTkFDIHRyYW5zY3JpcHRpb24gZmFjdG9yLCBBTkFDMDE3LCBtZWRpYXRlcyBtaXRvY2hvbmRyaWFsIHJldHJvZ3JhZGUgc2lnbmFsaW5nIGluIEFyYWJpZG9wc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzQ1MC03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgZW52aXJvbm1lbnRhbCBzdHJlc3MgaW4gcGxhbnRzIHdpdGggdGhlIGFpZCBvZiBtYXJrZXIgZ2VuZXMgZm9yIEgyTzIgcmVzcG9uc2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NZXRob2RzIEVuenltb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NTI3OyAgY2l0YXRpb25fcGFnZXM9MjIxLTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFudCB0cmFuc2R1Y2VycyBvZiB0aGUgZW5kb3BsYXNtaWMgcmV0aWN1bHVtIHVuZm9sZGVkIHByb3RlaW4gcmVzcG9uc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBQbGFudCBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz03MjAtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW5kb3BsYXNtaWMgcmV0aWN1bHVtIHN0cmVzcy1pbmR1Y2VkIGZvcm1hdGlvbiBvZiB0cmFuc2NyaXB0aW9uIGZhY3RvciBjb21wbGV4IEVSU0YgaW5jbHVkaW5nIE5GLVkgKENCRikgYW5kIGFjdGl2YXRpbmcgdHJhbnNjcmlwdGlvbiBmYWN0b3JzIDZhbHBoYSBhbmQgNmJldGEgdGhhdCBhY3RpdmF0ZXMgdGhlIG1hbW1hbGlhbiB1bmZvbGRlZCBwcm90ZWluIHJlc3BvbnNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIENlbGwuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0xMjM5LTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1iWklQMjggYW5kIE5GLVkgdHJhbnNjcmlwdGlvbiBmYWN0b3JzIGFyZSBhY3RpdmF0ZWQgYnkgRVIgc3RyZXNzIGFuZCBhc3NlbWJsZSBpbnRvIGEgdHJhbnNjcmlwdGlvbmFsIGNvbXBsZXggdG8gcmVndWxhdGUgc3RyZXNzIHJlc3BvbnNlIGdlbmVzIGluIEFyYWJpZG9wc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9NzgyLTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljIGFuYWx5c2lzIG9mIHRoZSB1bmZvbGRlZCBwcm90ZWluIHJlc3BvbnNlIGluIEFyYWJpZG9wc2lzIHNob3dzIGl0cyBjb25uZWN0aW9uIHRvIGltcG9ydGFudCBjZWxsdWxhciBwcm9jZXNzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz01NjEtNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtZW1icmFuZS1ib3VuZCBOQUMgdHJhbnNjcmlwdGlvbiBmYWN0b3IgQU5BQzAxMyBmdW5jdGlvbnMgaW4gbWl0b2Nob25kcmlhbCByZXRyb2dyYWRlIHJlZ3VsYXRpb24gb2YgdGhlIG94aWRhdGl2ZSBzdHJlc3MgcmVzcG9uc2UgaW4gQXJhYmlkb3BzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zNDcyLTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHByZXNzaW9uIG9mIGFuIGV2b2x1dGlvbmFyaWx5IGRpc3RpbmN0IG5vdmVsIEJpUCBnZW5lIGR1cmluZyB0aGUgdW5mb2xkZWQgcHJvdGVpbiByZXNwb25zZSBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTE7ICBjaXRhdGlvbl9wYWdlcz04MS05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWVsaW5hIElJOiBhIHdlYiB0b29sIGZvciBjb21wYXJpc29ucyBhbW9uZyBzZXZlcmFsIHByZWRpY3RpdmUgYWxnb3JpdGhtcyB0byBmaW5kIHBvdGVudGlhbCBtb3RpZnMgZnJvbSBwcm9tb3RlciByZWdpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPVcyMjctMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNC8zLzI3MS8xNzc0Mzk2My9kc3cwNjUucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEluIG91ciBwcmV2aW91cyBzdHVkeSwgYSBtZXRob2RvbG9neSB3YXMgZXN0YWJsaXNoZWQgdG8gcHJlZGljdCB0cmFuc2NyaXB0aW9uYWwgcmVndWxhdG9yeSBlbGVtZW50cyBpbiBwcm9tb3RlciBzZXF1ZW5jZXMgdXNpbmcgdHJhbnNjcmlwdG9tZSBkYXRhIGJhIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0yOTY1NzkzJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZDumvKWEoy8bMhVbxV3EvnaPGOwB" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f271%2f2965793" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f271%2f2965793" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=97549" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/3/1/m_dnares_24_3cover.png?Expires=2147483647&amp;Signature=eoXc11KAMpHEsaqW0C6T2VF4hIzrR2AQ2xJtZ-I1SzuN8Ozcm3pfcXhSESBXVMgST8o0VjwPqT85KWv95Ep~BtVLeJis8VsXRFBVpMHhb~g9dvz54ejvU8kSReDsueZzPSdTwZuEswyhmEGVK~9fmtkzww9NnyhhDFzj14yR9QQPP3D5RCa40I2kb6S3cosdNYyeAUAcZmppIJGZquGgABfHKwfV6R3Ffo5Kcif39o76ejsFOOlH0X60OYvM4Yme1Vyjd2eLo4R5~a0mH9tf6MZMrQpFP4iVZGwKwAheMFv-ISdx3BcAb4Re6~N24GwZYpxXAywMLeRm98GmIDGpbg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/3">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="88473521">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473521">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473523">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473523">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473528">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473528">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473533">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473533">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473570">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473570">Conflict of interest</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473572">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473572">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473574">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473574">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473576">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473576">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/3/261/3052236">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/3/279/2972857">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Prediction of bipartite transcriptional regulatory elements using transcriptome data of Arabidopsis
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshiharu Y. Yamamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Faculty of Applied Biological Sciences, Gifu University, 1-1 Yanagido, Gifu City, Gifu 501-1193, Japan</div><div class="aff"><div class="label -label">2</div>United Graduate School of Agricultural Science, Gifu University, Yanagido 1-1, Gifu City, Gifu 501-1193, Japan</div><div class="aff"><div class="label -label">3</div>RIKEN CSRS, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama 230-0045, Japan</div><div class="aff"><div class="label -label">4</div>JST ALCA</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw065-cor1" data-open="dsw065-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 58 293 2848. Email: <a href="mailto:yyy@gifu-u.ac.jp" target="_blank">yyy@gifu-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshiharu+Y.+Yamamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamamoto Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamamoto Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshiharu Y. Yamamoto</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroyuki Ichida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div>RIKEN Nishina Center, 2-1 Hirosawa, Wako, Saitama 351-0198, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroyuki+Ichida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ichida H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ichida H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiroyuki Ichida</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ayaka Hieno</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>United Graduate School of Agricultural Science, Gifu University, Yanagido 1-1, Gifu City, Gifu 501-1193, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ayaka+Hieno">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hieno A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hieno A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ayaka Hieno</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Daichi Obata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Faculty of Applied Biological Sciences, Gifu University, 1-1 Yanagido, Gifu City, Gifu 501-1193, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Daichi+Obata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Obata D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Obata D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Daichi Obata</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mutsutomo Tokizawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>United Graduate School of Agricultural Science, Gifu University, Yanagido 1-1, Gifu City, Gifu 501-1193, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mutsutomo+Tokizawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tokizawa M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tokizawa M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mutsutomo Tokizawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mika Nomoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div>Center for Gene Research, Nagoya University, Furo-cho, Chikusa, Nagoya 464-8602, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mika+Nomoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nomoto M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nomoto M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mika Nomoto</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasuomi Tada</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div>Center for Gene Research, Nagoya University, Furo-cho, Chikusa, Nagoya 464-8602, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasuomi+Tada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tada Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tada Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yasuomi Tada</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazutaka Kusunoki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>United Graduate School of Agricultural Science, Gifu University, Yanagido 1-1, Gifu City, Gifu 501-1193, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kazutaka+Kusunoki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kusunoki K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kusunoki K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kazutaka Kusunoki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hiroyuki Koyama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Faculty of Applied Biological Sciences, Gifu University, 1-1 Yanagido, Gifu City, Gifu 501-1193, Japan</div><div class="aff"><div class="label -label">2</div>United Graduate School of Agricultural Science, Gifu University, Yanagido 1-1, Gifu City, Gifu 501-1193, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hiroyuki+Koyama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Koyama H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Koyama H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hiroyuki Koyama</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Natsuki Hayami</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>United Graduate School of Agricultural Science, Gifu University, Yanagido 1-1, Gifu City, Gifu 501-1193, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Natsuki+Hayami">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hayami N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hayami N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Natsuki Hayami</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw065-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Mikio Nishimura</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 3, 1 June 2017, Pages 271–278, <a href='https://doi.org/10.1093/dnares/dsw065'>https://doi.org/10.1093/dnares/dsw065</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 February 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">09 September 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">27 December 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2965793" href="/dnaresearch/article-pdf/24/3/271/17743963/dsw065.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yoshiharu Y. Yamamoto, Hiroyuki Ichida, Ayaka Hieno, Daichi Obata, Mutsutomo Tokizawa, Mika Nomoto, Yasuomi Tada, Kazutaka Kusunoki, Hiroyuki Koyama, Natsuki Hayami;  Prediction of bipartite transcriptional regulatory elements using transcriptome data of Arabidopsis, <em>DNA Research</em>, Volume 24, Issue 3, 1 June 2017, Pages 271–278, <a href='https://doi.org/10.1093/dnares/dsw065'>https://doi.org/10.1093/dnares/dsw065</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2965793?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2965793?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2965793?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2965793?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2965793?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2965793?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2965793?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Prediction%20of%20bipartite%20transcriptional%20regulatory%20elements%20using%20transcriptome%20data%20of%20Arabidopsis&amp;publicationDate=2017-02-01&amp;volumeNum=24&amp;issueNum=3&amp;author=Yamamoto%2C%20Yoshiharu%20Y.%3B%20Ichida%2C%20Hiroyuki&amp;startPage=271&amp;endPage=278&amp;contentId=10.1093%2Fdnares%2Fdsw065&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f271%2f2965793" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=88473521 id="88473521" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>In our previous study, a methodology was established to predict transcriptional regulatory elements in promoter sequences using transcriptome data based on a frequency comparison of octamers. Some transcription factors, including the NAC family, cannot be covered by this method because their binding sequences have non-specific spacers in the middle of the two binding sites. In order to remove this blind spot in promoter prediction, we have extended our analysis by including bipartite octamers that are composed of ‘4 bases—a spacer with a flexible length—4 bases’. 8,044 pre-selected bipartite octamers, which had an overrepresentation of specific spacer lengths in promoter sequences and sequences related to core elements removed, were subjected to frequency comparison analysis. Prediction of ER stress-responsive elements in the BiP/BiPL promoter and an ANAC017 target sequence resulted in precise detection of true positives, judged by functional analyses of a reported article and our own <em>in vitro</em> protein–DNA binding assays. These results demonstrate that incorporation of bipartite octamers with continuous ones improves promoter prediction significantly.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;promoter prediction&quot;">promoter prediction</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="transcriptome">transcriptome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;plant genome&quot;">plant genome</a></div>                            <h2 scrollto-destination=88473523 id="88473523" class="section-title" >1. Introduction</h2>
<p>The availability of large-scale gene expression data by microarray and RNASeq analyses provided the possibility of predicting transcriptional regulatory elements in promoters using these data. For such predictions, methods for extraction of consensus sequences from a set of promoter sequences have been used (e.g. Gibbs Sampler<a reveal-id="dsw065-B1" data-open="dsw065-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> and MEME<a reveal-id="dsw065-B2" data-open="dsw065-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a>). As the accuracy of these established methods is not sufficient for experimental validation, we have developed a novel, more accurate method based on a simple frequency comparison.<a reveal-id="dsw065-B3" data-open="dsw065-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> The new method shows considerably higher accuracy and sensitivity than conventional methods, judged by re-prediction of experimentally identified regulatory elements.<a reveal-id="dsw065-B3" data-open="dsw065-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> In addition, our recent studies demonstrated that it is also useful for predicting novel regulatory elements, and it has paved the way for prediction-oriented promoter analysis.<a reveal-id="dsw065-B4 dsw065-B5 dsw065-B6" data-open="dsw065-B4 dsw065-B5 dsw065-B6" class="link link-ref link-reveal xref-bibr"><sup>4–6</sup></a></p><p>Our prediction method compares the frequency of every octamer between focused and global promoter sets. This octamer-based methodology has worked well for recognition sites of many types of transcription factors including bZIP, AP2/DREB, zinc finger, CAMTA, and also PCF families. However, it is theoretically not good for the detection of spacer-containing bipartite motifs that are recognized by protein complexes including dimers. As a consequence, these promoter elements are considered a blind spot of our octamer-based prediction method.</p><p>There are several databases for detection of transcription factor-binding sites in the promoter region. Among them, JASPAR<a reveal-id="dsw065-B7" data-open="dsw065-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> is the most popular public database that covers higher plants. Detection of promoter elements is based on DNA motifs for each transcription factor proteins, and in the case of Arabidopsis which is the most intensely collected for data in higher plants, motifs for 48 factors are available out of ∼1,900 transcription factors in the genome.<a reveal-id="dsw065-B8" data-open="dsw065-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> The 48 factors include bipartite motifs for several MADS family proteins and an AP2 family protein, ANT. Obviously, the coverage is low for continuous and also bipartite motifs.</p><p>In this report, we have developed a supplemental method for promoter prediction that compensates for this blind spot by incorporating bipartite octamers (4 + 4 bases) to predictions that contain a spacer sequence in the middle. Our results using the pre-selected bipartite octamers showed improved prediction of the ER-stress responsive element (ERSE) as an element for tunicamycin-activated ER stress, and of ANAC017 target sites in promoters of the down-stream genes of ANAC017.</p>                            <h2 scrollto-destination=88473528 id="88473528" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=88473529 id="88473529" class="section-title" >2.1. Bioinformatics analysis</h3>
<p>24,956 Arabidopsis promoter sequences of 1 kb long, starting from the most major transcription start site (TSS), were prepared previously.<a reveal-id="dsw065-B9" data-open="dsw065-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Counting of each bipartite octamer, statistical analyses, and LDSS (local distribution of short sequences) analysis<a reveal-id="dsw065-B10" data-open="dsw065-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> were achieved with home-made Perl and C<sup>+ </sup>programs, and Excel (Microsoft Japan, Tokyo). Distribution profiles of bipartite octamers were subjected to smoothing with a 21 bin, which is a good width for regulatory sequences (REG),<a reveal-id="dsw065-B10" data-open="dsw065-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> clustered with Cluster<a reveal-id="dsw065-B11" data-open="dsw065-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> by the hierarchical method with correlation measurement, and visualized with TreeView<a reveal-id="dsw065-B11" data-open="dsw065-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> as described previously.<a reveal-id="dsw065-B10" data-open="dsw065-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Prediction of transcriptional regulatory elements based on microarray data with the aid of a list of bipartite octamers was achieved essentially according to our previous report<a reveal-id="dsw065-B3" data-open="dsw065-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> using modified software. Motifs of clustered sequences were expressed using WebLogo.<a reveal-id="dsw065-B12" data-open="dsw065-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> For prediction of ERSEs, 162 genes with a fold-change of over 2.5 were selected from up-regulated genes (E-MEXP-3186 from ArrayExpress, <a class="link link-uri" href="https://www.ebi.ac.uk/arrayexpress/" target="_blank">https://www.ebi.ac.uk/arrayexpress/</a> (11 January 2017, date last accessed)).<a reveal-id="dsw065-B13" data-open="dsw065-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> as a positive promoter group. Microarray data of ANAC017 mutants were also retrieved from ArrayExpress (E-GEOD-41136).<a reveal-id="dsw065-B14" data-open="dsw065-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a></p>                            <h3 scrollto-destination=88473531 id="88473531" class="section-title" >2.2. DNA–protein binding analysis</h3>
<p>Complementary DNA for ANAC017 was prepared by reverse transcriptase-polymerase chain reaction (RT-PCR) from total RNA of Arabidopsis seedlings using a conventional method as described elsewhere.<a reveal-id="dsw065-B15" data-open="dsw065-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> The coding sequence of the Flag tag and a T7 promoter were added to the ANAC017 CDS excluding the transmembrane domain (1–523 aa<a reveal-id="dsw065-B14" data-open="dsw065-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a>) by PCR. The prepared PCR product was subjected to <em>in vitro</em> transcription by T7 RNA polymerase and subsequent <em>in vitro</em> translation by a wheat germ system as described previously.<a reveal-id="dsw065-B6" data-open="dsw065-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> Binding assays of the Flag-tagged ANAC017 protein and biotinylated oligo-DNA probes were achieved using AlphaScreen (Perkin Elmer Japan, Tokyo) as described.<a reveal-id="dsw065-B6" data-open="dsw065-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> Sequences of a biotinylated DNA probe and non-biotinylated competitors are shown in <a reveal-id="dsw065-F6" data-open="dsw065-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>.</p>                            <h2 scrollto-destination=88473533 id="88473533" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=88473534 id="88473534" class="section-title" >3.1. Pre-selection of potential regulatory sequences</h3>
<p>In our previous study, promoter prediction using microarray data based on enumerative octamer analysis was developed.<a reveal-id="dsw065-B3" data-open="dsw065-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> The degree of overrepresentation for each octamer among a set of promoter sequences showing some transcriptional response over total promoters in a genome is calculated as the ‘Relative Appearance Ratio (RAR)’ . When predicting a specific promoter, an octamer is taken from the 5ʹ end of the promoter, the corresponding RAR value for the octamer is put back to the corresponding promoter position. Repeating these steps sliding 1 bp each time towards the 3ʹ direction provides a promoter scan with the RAR.<a reveal-id="dsw065-B3 dsw065-B4 dsw065-B5 dsw065-B6" data-open="dsw065-B3 dsw065-B4 dsw065-B5 dsw065-B6" class="link link-ref link-reveal xref-bibr"><sup>3–6</sup></a> Peak positions in the scanned results are predicted regulatory sites. We extended this procedure using a new type of bipartite octamers composed of ‘4 bases + a spacer + 4 bases’. In this article, we will refer to these octamers as ‘bipartite octamers (4 + 4)’.</p><p>As a pilot analysis, we surveyed 12 lengths of the spacer from 1 to 12 bases long. Analysis of all the possible bipartite octamers with a 12-spacer length gives 12 RAR values for each promoter point. While trials of this prediction method for cold- and high light-stress responsive elements detected novel promoter sequences that could not be detected by the previous octamer analysis without spacers, they also gave us highly complex results making their analysis difficult (data not shown). Therefore, we decided to pre-select potential regulatory sequences before carrying out frequency analysis. For this purpose, we set the priorities to high coverage over accuracy in the sequence selection, because an accurate, and thus small, selection at this point results in low sensitivity of prediction. Accuracy can be increased at the subsequent prediction steps.</p><p>During the analysis mentioned above, we noticed an overrepresentation of a specific spacer length over the others when the octamer (4 + 4) sequence is fixed. <a reveal-id="dsw065-F1" data-open="dsw065-F1" class="link link-reveal link-table xref-fig">Fig. 1</a> shows examples of the relationship between the spacer length and the counts in the total 24,956 promoters of the Arabidopsis genome. In the case of AACGnTCGT (<em>n</em> = 0–12), a spacer of three appears ∼4 times more frequently than the other spacer lengths. The probability of this count profile, represented by the peak and total counts, under the assumption of random occurrence is very low (<em>P</em> = 2.2E-195), suggesting a strong selection pressure towards this biased profile. Interestingly, a count profile of its complementary sequence (ACGAnCGTT) also shows the same characteristics where a spacer of three gives a single high peak.</p>                        <a id="88473538" scrollto-destination="88473538"></a>
<div id="dsw065-F1" data-id="dsw065-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/m_dsw065f1.png?Expires=2147483647&Signature=CEBM6H~IUDIoIRYRUGxirhs8gxL-BoHjH7WnnXUAuyV0kA5f3L4eSHl1xKgETUjW92lbUqi-yEooRcyFmxQYqeABofcIJWMVrPh16qoQZ2U5zmgJ6Y0Lqn1RoW5tIq3tJKrjMBXyjxN5xvtCnpQd5ZUY9VakbT8nqPrrfHQFv8BRpOfwjcixIqwgfLQ9EDxIIYZilUXH1v3Ux2yVO4Uud1hsR7cwBZcX~DQ8El0xmwMA2G-DEvadjSfJWRq-GTgA-4APiZWgBh8ldqDR7yyFpUEyP9RembdBepNYKPpZf9GfGx7MMSkha85IMakwDZAoZBbuiiZKqm6Lri-fCustSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overrepresentation of a specific length of spacer in promoter sequences. Counts of a bipartite-type octamer with a spacer in all the promoters of the Arabidopsis genome are shown. The horizontal axis indicates the number of ‘n’s, i.e. the length of the spacer between the two tetramers. The two octamers shown (AACGnTCGT and ACGAnCGTT) are mutually complementary." path-from-xml="dsw065f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473538/dsw065f1.tif" path-from-xml="dsw065f1.tif" target="_blank">View large</a><a section="88473538" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/dsw065f1.png?Expires=2147483647&Signature=kxdDE-x3VZoQOGzFVtNCF6Q~bUnf~Zk7cBig~FvGiInNTQhsdsqjsxF8VJdQKpxLaTqgpurzuDzd~kH-159fakTeyhW2sW~kzAsBam05z7u0q0N5ljdUBnxgvMtWwzcjl0Tm-zCin85RWXjn3W11gdV2Xz3d6JicSL8g7Oe~7eblve0ErmrP0q~Zw6hJSCVSP2X2ET45Ox7JCrYdAXkDyQHmZy47LIT6EyC1UcneFjfFTJDpDIprLS7SImoz9vFbvMfnNroVh4YQNEvivfahpVkfA8qjVITLQ6eMKl4l8b2NmX3tP3MHEWebA7CcRhvV8aRfdWXDnvn0nTvb8zhXvA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473538&ar=2965793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw065f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Overrepresentation of a specific length of spacer in promoter sequences. Counts of a bipartite-type octamer with a spacer in all the promoters of the Arabidopsis genome are shown. The horizontal axis indicates the number of ‘<em>n</em>’s, i.e. the length of the spacer between the two tetramers. The two octamers shown (AACGnTCGT and ACGAnCGTT) are mutually complementary.</p></div></div><div content-id="dsw065-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/m_dsw065f1.png?Expires=2147483647&Signature=CEBM6H~IUDIoIRYRUGxirhs8gxL-BoHjH7WnnXUAuyV0kA5f3L4eSHl1xKgETUjW92lbUqi-yEooRcyFmxQYqeABofcIJWMVrPh16qoQZ2U5zmgJ6Y0Lqn1RoW5tIq3tJKrjMBXyjxN5xvtCnpQd5ZUY9VakbT8nqPrrfHQFv8BRpOfwjcixIqwgfLQ9EDxIIYZilUXH1v3Ux2yVO4Uud1hsR7cwBZcX~DQ8El0xmwMA2G-DEvadjSfJWRq-GTgA-4APiZWgBh8ldqDR7yyFpUEyP9RembdBepNYKPpZf9GfGx7MMSkha85IMakwDZAoZBbuiiZKqm6Lri-fCustSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overrepresentation of a specific length of spacer in promoter sequences. Counts of a bipartite-type octamer with a spacer in all the promoters of the Arabidopsis genome are shown. The horizontal axis indicates the number of ‘n’s, i.e. the length of the spacer between the two tetramers. The two octamers shown (AACGnTCGT and ACGAnCGTT) are mutually complementary." path-from-xml="dsw065f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473538/dsw065f1.tif" path-from-xml="dsw065f1.tif" target="_blank">View large</a><a section="88473538" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/dsw065f1.png?Expires=2147483647&Signature=kxdDE-x3VZoQOGzFVtNCF6Q~bUnf~Zk7cBig~FvGiInNTQhsdsqjsxF8VJdQKpxLaTqgpurzuDzd~kH-159fakTeyhW2sW~kzAsBam05z7u0q0N5ljdUBnxgvMtWwzcjl0Tm-zCin85RWXjn3W11gdV2Xz3d6JicSL8g7Oe~7eblve0ErmrP0q~Zw6hJSCVSP2X2ET45Ox7JCrYdAXkDyQHmZy47LIT6EyC1UcneFjfFTJDpDIprLS7SImoz9vFbvMfnNroVh4YQNEvivfahpVkfA8qjVITLQ6eMKl4l8b2NmX3tP3MHEWebA7CcRhvV8aRfdWXDnvn0nTvb8zhXvA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473538&ar=2965793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw065f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Overrepresentation of a specific length of spacer in promoter sequences. Counts of a bipartite-type octamer with a spacer in all the promoters of the Arabidopsis genome are shown. The horizontal axis indicates the number of ‘<em>n</em>’s, i.e. the length of the spacer between the two tetramers. The two octamers shown (AACGnTCGT and ACGAnCGTT) are mutually complementary.</p></div></div><p>Preferential appearance in the promoter sequence and conservation of characteristics between forward and reverse sequences may suggest that they are transcriptional regulatory sequences. Therefore, we decided to select bipartite octamers that have a preference for the spacer length among the promoter sequences.</p><p>With spacer lengths from 0 to 12, the number of possible bipartite octamers (4 + 4) is 4<sup>8</sup> × 13 = 65,536 × 13 (<a reveal-id="dsw065-T1" data-open="dsw065-T1" class="link link-reveal link-table xref-fig">Table 1</a>). First, one spacer length from 0 to 12 bp for each octamer with the highest count among the promoter sequences was selected, extracting 65,536 sequences from 65,536 × 13. Second, sequences with spacers of 0 and 1 were removed from 65,536 sequences because they were thought to be detected by established analysis using octamers without a spacer. Then, the <em>P</em> value of the count profile with 12 spacer lengths for each bipartite octamer, as shown in <a reveal-id="dsw065-F1" data-open="dsw065-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, was considered, focusing on a peak height and the total count for 13 spacers, and 9,022 octamers with <em>P</em> &lt; 1E-5 were selected. <div content-id="dsw065-T1" class="table-modal"><div class="table-wrap"><div id="dsw065-T1" data-id="dsw065-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Selection of bipartite octamers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Selection </th><th>Number of octamers </th></tr></thead><tbody><tr><td>Total (0 ≤ <em>n</em> ≤ 12) </td><td>65,536 × 13 </td></tr><tr><td>Peak space </td><td>65,536 </td></tr><tr><td>Peak spacing: <em>n</em> ≠ 0, 1 &amp; single peak </td><td>41,158 </td></tr><tr><td>Dominance of peak space (<em>P</em> &lt; E-5) </td><td>9,022 </td></tr><tr><td>LDSS selection </td><td> </td></tr><tr><td> TATA or core </td><td>903 </td></tr><tr><td> Solid type </td><td>75 </td></tr><tr><td> Rest (REG &amp; non-REG) </td><td>8,044 </td></tr></tbody></table></div><div content-id="dsw065-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Selection </th><th>Number of octamers </th></tr></thead><tbody><tr><td>Total (0 ≤ <em>n</em> ≤ 12) </td><td>65,536 × 13 </td></tr><tr><td>Peak space </td><td>65,536 </td></tr><tr><td>Peak spacing: <em>n</em> ≠ 0, 1 &amp; single peak </td><td>41,158 </td></tr><tr><td>Dominance of peak space (<em>P</em> &lt; E-5) </td><td>9,022 </td></tr><tr><td>LDSS selection </td><td> </td></tr><tr><td> TATA or core </td><td>903 </td></tr><tr><td> Solid type </td><td>75 </td></tr><tr><td> Rest (REG &amp; non-REG) </td><td>8,044 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Of the 13 spacing lengths, one with the highest appearance in the promoters was selected (Peak space), and octamers with spacing lengths of 0 or 1, and octamers with the two spacings of highest appearance (twin peaks) were removed. Distribution profiles were subjected to statistical analysis and octamers with a profile of <em>P </em>&lt;<em> </em>E-5, as determined by the Chi-square test, were further selected. Selected octamers were subjected to LDSS analysis and TATA-, Core-, and Solid-type octamers were removed, providing a final 8,044 bipartite octamers.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw065-T1" data-id="dsw065-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Selection of bipartite octamers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Selection </th><th>Number of octamers </th></tr></thead><tbody><tr><td>Total (0 ≤ <em>n</em> ≤ 12) </td><td>65,536 × 13 </td></tr><tr><td>Peak space </td><td>65,536 </td></tr><tr><td>Peak spacing: <em>n</em> ≠ 0, 1 &amp; single peak </td><td>41,158 </td></tr><tr><td>Dominance of peak space (<em>P</em> &lt; E-5) </td><td>9,022 </td></tr><tr><td>LDSS selection </td><td> </td></tr><tr><td> TATA or core </td><td>903 </td></tr><tr><td> Solid type </td><td>75 </td></tr><tr><td> Rest (REG &amp; non-REG) </td><td>8,044 </td></tr></tbody></table></div><div content-id="dsw065-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Selection </th><th>Number of octamers </th></tr></thead><tbody><tr><td>Total (0 ≤ <em>n</em> ≤ 12) </td><td>65,536 × 13 </td></tr><tr><td>Peak space </td><td>65,536 </td></tr><tr><td>Peak spacing: <em>n</em> ≠ 0, 1 &amp; single peak </td><td>41,158 </td></tr><tr><td>Dominance of peak space (<em>P</em> &lt; E-5) </td><td>9,022 </td></tr><tr><td>LDSS selection </td><td> </td></tr><tr><td> TATA or core </td><td>903 </td></tr><tr><td> Solid type </td><td>75 </td></tr><tr><td> Rest (REG &amp; non-REG) </td><td>8,044 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Of the 13 spacing lengths, one with the highest appearance in the promoters was selected (Peak space), and octamers with spacing lengths of 0 or 1, and octamers with the two spacings of highest appearance (twin peaks) were removed. Distribution profiles were subjected to statistical analysis and octamers with a profile of <em>P </em>&lt;<em> </em>E-5, as determined by the Chi-square test, were further selected. Selected octamers were subjected to LDSS analysis and TATA-, Core-, and Solid-type octamers were removed, providing a final 8,044 bipartite octamers.</p></div></div></div></div></p><p>These sequences were further subjected to LDSS analysis, where preference of appearance according to promoter position was evaluated.<a reveal-id="dsw065-B10" data-open="dsw065-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> According to this analysis, 903 of them are revealed to be related to core promoter elements (<a reveal-id="dsw065-F2" data-open="dsw065-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>), including TATA box (<a reveal-id="dsw065-F3" data-open="dsw065-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>) and Y Patch (<a reveal-id="dsw065-F3" data-open="dsw065-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>), and 75 sequences with ‘Solid’ distribution profiles (<a reveal-id="dsw065-F2" data-open="dsw065-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>), which potentially include transposon-related sequences, will not be transcriptional regulatory elements. These 978 identified sequences (903 + 75) were removed for further analysis. A summary of the sequence extraction is shown in <a reveal-id="dsw065-T1" data-open="dsw065-T1" class="link link-reveal link-table xref-fig">Table 1</a>.</p>                        <a id="88473542" scrollto-destination="88473542"></a>
<div id="dsw065-F2" data-id="dsw065-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/m_dsw065f2.png?Expires=2147483647&Signature=XlgzJ3f0c-8M5ElSJLbuyPZpQbVm2gnuhCHQgEm3QdFAMTqr5tBkJ-Ot9IcQMjgp3IORcN8GRLoy5VyAudGBeD09MR~Zmebu2nZ2oIJnhD~csyxpJL4KVrlCgXzs7p~eoJIIzz~ZDe7XVqWQ4CZXLCk0refSC5b7BpImJTF0aqKFbOCnxCaIT--gp8WAL6F9tK8qRBGwAnvbUggyESp7vwjJRjps3lV~8jOeC511RxiicjMeoFM1BSN0C4IJfokpetZPhdFJI7OoxZqnGv5WAZjVOcnWXk1azROWpeyrlIlftlpcPlaXH~36W~QTaatpP5aSqV86BXnbPMhfK7BklQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="LDSS analysis of bipartite octamers. The distribution profiles of each octamer are shown. Each profile is shown as a colored horizontal line (rare = black, frequent = red), and the profiles of 903 (A), 75 (B), and 8,044 (C) octamers were subjected to hierarchical clustering. The cluster number of the ‘Rest’ group is indicated at the right end of the heat map (C). REG (1, 2), REG-like (3, 4), and non-REG (5–9)." path-from-xml="dsw065f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473542/dsw065f2.tif" path-from-xml="dsw065f2.tif" target="_blank">View large</a><a section="88473542" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/dsw065f2.png?Expires=2147483647&Signature=rLFs028o~xRIaqVXW~mlV4kO-3eOaSKnZF3h7YhIiBIHzx7ew--23w0XBjiMU6C05kdWzgUzQLgHVQpgbqwUjV6Y55e2CNaqGoPcUG7W9Bctt2HkKXzKuURagPG6pHS5wQI6e6dK2lf3Sb0dn1yEfw2qf-zKWkq0TDJE5B9wiqMT06BJ3IaQc44kW8BX04axHtmmESriGJuLVfY5v3eJ09H8KIA2mV7heyZ1~q2SrG2iE8VzEC1sxISDdf8ULvnZla0pO1vz97JCrhqDdhu5n8znmjFxG1sAGV-OUc-aJcA6eMizOEJW5eyBYDu6Lw6V3h-c-SSQWl2La0nuf7SHig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473542&ar=2965793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw065f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>LDSS analysis of bipartite octamers. The distribution profiles of each octamer are shown. Each profile is shown as a colored horizontal line (rare = black, frequent = red), and the profiles of 903 (A), 75 (B), and 8,044 (C) octamers were subjected to hierarchical clustering. The cluster number of the ‘Rest’ group is indicated at the right end of the heat map (C). REG (1, 2), REG-like (3, 4), and non-REG (5–9).</p></div></div><div content-id="dsw065-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/m_dsw065f2.png?Expires=2147483647&Signature=XlgzJ3f0c-8M5ElSJLbuyPZpQbVm2gnuhCHQgEm3QdFAMTqr5tBkJ-Ot9IcQMjgp3IORcN8GRLoy5VyAudGBeD09MR~Zmebu2nZ2oIJnhD~csyxpJL4KVrlCgXzs7p~eoJIIzz~ZDe7XVqWQ4CZXLCk0refSC5b7BpImJTF0aqKFbOCnxCaIT--gp8WAL6F9tK8qRBGwAnvbUggyESp7vwjJRjps3lV~8jOeC511RxiicjMeoFM1BSN0C4IJfokpetZPhdFJI7OoxZqnGv5WAZjVOcnWXk1azROWpeyrlIlftlpcPlaXH~36W~QTaatpP5aSqV86BXnbPMhfK7BklQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="LDSS analysis of bipartite octamers. The distribution profiles of each octamer are shown. Each profile is shown as a colored horizontal line (rare = black, frequent = red), and the profiles of 903 (A), 75 (B), and 8,044 (C) octamers were subjected to hierarchical clustering. The cluster number of the ‘Rest’ group is indicated at the right end of the heat map (C). REG (1, 2), REG-like (3, 4), and non-REG (5–9)." path-from-xml="dsw065f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473542/dsw065f2.tif" path-from-xml="dsw065f2.tif" target="_blank">View large</a><a section="88473542" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/dsw065f2.png?Expires=2147483647&Signature=rLFs028o~xRIaqVXW~mlV4kO-3eOaSKnZF3h7YhIiBIHzx7ew--23w0XBjiMU6C05kdWzgUzQLgHVQpgbqwUjV6Y55e2CNaqGoPcUG7W9Bctt2HkKXzKuURagPG6pHS5wQI6e6dK2lf3Sb0dn1yEfw2qf-zKWkq0TDJE5B9wiqMT06BJ3IaQc44kW8BX04axHtmmESriGJuLVfY5v3eJ09H8KIA2mV7heyZ1~q2SrG2iE8VzEC1sxISDdf8ULvnZla0pO1vz97JCrhqDdhu5n8znmjFxG1sAGV-OUc-aJcA6eMizOEJW5eyBYDu6Lw6V3h-c-SSQWl2La0nuf7SHig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473542&ar=2965793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw065f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>LDSS analysis of bipartite octamers. The distribution profiles of each octamer are shown. Each profile is shown as a colored horizontal line (rare = black, frequent = red), and the profiles of 903 (A), 75 (B), and 8,044 (C) octamers were subjected to hierarchical clustering. The cluster number of the ‘Rest’ group is indicated at the right end of the heat map (C). REG (1, 2), REG-like (3, 4), and non-REG (5–9).</p></div></div>                        <a id="88473543" scrollto-destination="88473543"></a>
<div id="dsw065-F3" data-id="dsw065-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/m_dsw065f3.png?Expires=2147483647&Signature=3mDPJjfkIz~cn8vpggHXq4g8ANp3q~UUSztJbVR0huhnIJijOdiNjA9IjEp4D3ylxHMsnEqcDoGAB5SLkzfr0pT2RXSMxqLc7J37dJWXRKPkrjYdSeLlHP9Az1gCOFX8~7oVvqMGAnfcsl1R0-8lnQr82qmqZEIL2e5ILcn7GeDvKHOClOewBKGuXXv0~bYYa8NxUEiP~aW6TuzIUFTgcHgt~EYCOqPaYUo6~XselsjCrYELcVeEybPdXd48gauVvCebfzL~wD28U~3MgfCf0v6DHqWLb1tC6JjEiIcUir7SOmNl3tXWWMpXdXXpu~8qSfDJss7QB6n2YNgY29g8yA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Typical LDSS profiles of bipartite octamers. Typical distribution profiles are shown: TATA type (A), Y Patch-related Core type (B), REG type (C), and non-REG type (D). Profiles were subjected to smoothing with a bin of 21 bp, and the maximum value was adjusted to 1.0. The peak position of Panel A is -47, which means the first T of the TATA within the sequence, ‘ACTT….TATA’, comes at -38. The two sequences in Panel C are complementary each other." path-from-xml="dsw065f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473543/dsw065f3.tif" path-from-xml="dsw065f3.tif" target="_blank">View large</a><a section="88473543" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/dsw065f3.png?Expires=2147483647&Signature=zwI-F9n1y8cfJNRARridfYOuHPId1ljCW95wXG25jMgNq4HFxl56U9cKqQqqgarX6H-kPdSfCgZsk~jxJjib5QaFWvhmi0DCXxubEqDhY4cisdpglhXRF654AXrMiQZyjjisnHRWlzzE5eTN4F4EJHbwFZL01BcoWXkoE9SfIzOJfjDTQEyHHGDDnin3HlfXf7UyzcZQbzEcr-W97AFrYGASsePlTsYPD-m6E0xLhnl~n1xgObUJZqe11Y22jCbqAGQO8pcOLGeuX3aIeBQwHk3HGrcB3rk4E1FCGDjTy2W7fHm3PYY8rLtbOUusTgj7AvkT8RD4~PPufwr9c3AHbw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473543&ar=2965793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw065f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Typical LDSS profiles of bipartite octamers. Typical distribution profiles are shown: TATA type (A), Y Patch-related Core type (B), REG type (C), and non-REG type (D). Profiles were subjected to smoothing with a bin of 21 bp, and the maximum value was adjusted to 1.0. The peak position of Panel A is -47, which means the first T of the TATA within the sequence, ‘ACTT….TATA’, comes at -38. The two sequences in Panel C are complementary each other.</p></div></div><div content-id="dsw065-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/m_dsw065f3.png?Expires=2147483647&Signature=3mDPJjfkIz~cn8vpggHXq4g8ANp3q~UUSztJbVR0huhnIJijOdiNjA9IjEp4D3ylxHMsnEqcDoGAB5SLkzfr0pT2RXSMxqLc7J37dJWXRKPkrjYdSeLlHP9Az1gCOFX8~7oVvqMGAnfcsl1R0-8lnQr82qmqZEIL2e5ILcn7GeDvKHOClOewBKGuXXv0~bYYa8NxUEiP~aW6TuzIUFTgcHgt~EYCOqPaYUo6~XselsjCrYELcVeEybPdXd48gauVvCebfzL~wD28U~3MgfCf0v6DHqWLb1tC6JjEiIcUir7SOmNl3tXWWMpXdXXpu~8qSfDJss7QB6n2YNgY29g8yA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Typical LDSS profiles of bipartite octamers. Typical distribution profiles are shown: TATA type (A), Y Patch-related Core type (B), REG type (C), and non-REG type (D). Profiles were subjected to smoothing with a bin of 21 bp, and the maximum value was adjusted to 1.0. The peak position of Panel A is -47, which means the first T of the TATA within the sequence, ‘ACTT….TATA’, comes at -38. The two sequences in Panel C are complementary each other." path-from-xml="dsw065f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473543/dsw065f3.tif" path-from-xml="dsw065f3.tif" target="_blank">View large</a><a section="88473543" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/dsw065f3.png?Expires=2147483647&Signature=zwI-F9n1y8cfJNRARridfYOuHPId1ljCW95wXG25jMgNq4HFxl56U9cKqQqqgarX6H-kPdSfCgZsk~jxJjib5QaFWvhmi0DCXxubEqDhY4cisdpglhXRF654AXrMiQZyjjisnHRWlzzE5eTN4F4EJHbwFZL01BcoWXkoE9SfIzOJfjDTQEyHHGDDnin3HlfXf7UyzcZQbzEcr-W97AFrYGASsePlTsYPD-m6E0xLhnl~n1xgObUJZqe11Y22jCbqAGQO8pcOLGeuX3aIeBQwHk3HGrcB3rk4E1FCGDjTy2W7fHm3PYY8rLtbOUusTgj7AvkT8RD4~PPufwr9c3AHbw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473543&ar=2965793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw065f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Typical LDSS profiles of bipartite octamers. Typical distribution profiles are shown: TATA type (A), Y Patch-related Core type (B), REG type (C), and non-REG type (D). Profiles were subjected to smoothing with a bin of 21 bp, and the maximum value was adjusted to 1.0. The peak position of Panel A is -47, which means the first T of the TATA within the sequence, ‘ACTT….TATA’, comes at -38. The two sequences in Panel C are complementary each other.</p></div></div><p>The remaining 8,044 sequences (<a reveal-id="dsw065-F2" data-open="dsw065-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>) were classified into three groups according to the LDSS profiles: the REG type (Clusters 1 and 2 in <a reveal-id="dsw065-F2 dsw065-F3" data-open="dsw065-F2 dsw065-F3" class="link link-reveal link-table xref-fig">Figs 2C and 3C</a>) that shows local distribution around -40 to -400 relative to the TSS and is a characteristic of a certain type of transcriptional regulatory element,<a reveal-id="dsw065-B10" data-open="dsw065-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> the non-REG type whose distributions are scattered evenly with regard to promoter position (Clusters 5–9 in <a reveal-id="dsw065-F2 dsw065-F3" data-open="dsw065-F2 dsw065-F3" class="link link-reveal link-table xref-fig">Figs 2C and 3D</a>) and the intermediate REG-like type (Clusters 3 and 4 in <a reveal-id="dsw065-F2" data-open="dsw065-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>).</p><p><a reveal-id="dsw065-F3" data-open="dsw065-F3" class="link link-reveal link-table xref-fig">Figure 3C</a> shows two distribution profiles of complementary sequences to each other, and both of them have very similar distribution profiles with peak positions around -100. These results demonstrate their distribution profile is direction-insensitive, and this feature is the same as the REG type of continuous octamers.<a reveal-id="dsw065-B10" data-open="dsw065-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Considering the distribution profiles, sequences in clusters 1 and 2 of <a reveal-id="dsw065-F2" data-open="dsw065-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a> are classified as the REG type.</p><p>Sequence lists of the removed core and solid groups, and also the 8,044 selected sequences containing REG and non-REG types can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/dsw065_Supp.xls?Expires=2147483647&Signature=F8Y4fwRwNGJil80iUVJPFsxswa910fM37~tNrinOWpfOAS3mOZ0zu1xYySI0Y5xPGZ4JefOBNzynYqecE-cNSs6u1fr4jY5w5lsi-1ivzNls9tCC9dVBK~QW6XjSm79JPPBqf5czmxS4mMAhyMJWB07mUe6MgE1FXlx8hE68tm85IwPkRN1j0iTWvjfJdrGXlY4xaSUM7N~vA-GNFAxWoLM4EwefAXMrh4QKIZjdqfXqu7R19Objxv9BO5tPnyP9sfhF6tvq5FmR3N4hhSLs6Z4IkaDo3Tm-s9nOHE4MyFmCIRyIa9YGCvp2So6IWosBn9nQ68Osbe91IsK0BsBG7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S1–S3</a></span>, respectively.</p><p>In order to pick up sequence groups from the extracted sequences, 1,053 selected REG-type sequences which are predicted to be regulatory sequences<a reveal-id="dsw065-B10" data-open="dsw065-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> were mapped to 24,714 Arabidopsis promoter sequences with 1 kb length, and a matrix of presence/absence (1,053 × 24,714) was subjected to 2D clustering analysis (<a reveal-id="dsw065-F4" data-open="dsw065-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). This analysis provided us the clearest classification of <em>cis</em> elements.<a reveal-id="dsw065-B10" data-open="dsw065-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> As shown in Panel A, several clusters have been identified each of which shares bipartite octamers and promoters.</p>                        <a id="88473548" scrollto-destination="88473548"></a>
<div id="dsw065-F4" data-id="dsw065-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/m_dsw065f4.png?Expires=2147483647&Signature=W9-YGh-ZACc9vtZcwu96DZM9SGor8d43HLixKfnOYBDKy57j195reZOfUdWjfIdSSYS-0hME6jBkzlBX9NF4psoI4KVbGMIygkMq5ugmnDBvrLDmfrXM9EldH1KhOJJOKv~W6s8y8sN2IWSS4CxUIAYb7po5mgPhowk~Owmfk5jvvwVTc9S9EbKmNBqcm3tq~CtdAl-bnSuLpOeAb8YGcdxZZU2bMAqe~xJ1Oq6qtVk-diVFIfcdZSICYstB1TNSn~e6efLd5OiW8cUZnfmfnzB9SMOyAj5PQKgLQ~zCbRw-ZxyItE4iCvQGbiXDKciCyDnTMHV~imXBDZxy6Kqs9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Several groups in the REG-type bipartite octamers. 1,053 bipartite octamers with distribution profiles of the REG type (Fig. 2C) were mapped to 24,714 Arabidopsis promoters, and a resultant presence/absence matrix (1,053 × 24,714) was subjected to 2D clustering analysis (A). Presence and absence of an octamer in a promoter is shown as red and black, respectively. Labels of the matrix in vertical and horizontal dimensions are omitted. Octamers of 20 largest clusters were subjected to WebLogo (B). Some pairs of cluster (1–9, 2–15, 3–4, 5–7, 10–17, 12–13) are complementary, respectively. Clusters 5 and 7 are ERSE (CCAAT-N9-CCACG)." path-from-xml="dsw065f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473548/dsw065f4.tif" path-from-xml="dsw065f4.tif" target="_blank">View large</a><a section="88473548" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/dsw065f4.png?Expires=2147483647&Signature=oyX3xW9N546YcwecEIcCZA9i3Qz8GqboXLGJlK8cOgKAbvh04umbJbWSgYba4I5LvDbNTIxcsf8e-dgTylOLiTFQroj0rXIj7H6bO4c3tUXmCNOiKJTVhcIAUVAgIa0Qs9VtNMlTIR4yWFu~4xDo1-FALP7UUSkAs-KB7gBHPcUjJiuHiLi2G-zSLIE-TEMIYOr45CvWi2QaH1BSz4MvsK4csN1yycYHi6aT2d-K-yRpmRuJIVtC6vhCKoCcbgumbAV5eyk3hzi6TgCqVoVZFpr-UutsHd6owJlVLqmkZrCHUWVd8x64V-8wrSAeIV0crs9VpZeLR7H72ldGFlltIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473548&ar=2965793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw065f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Several groups in the REG-type bipartite octamers. 1,053 bipartite octamers with distribution profiles of the REG type (<a reveal-id="dsw065-F2" data-open="dsw065-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>) were mapped to 24,714 Arabidopsis promoters, and a resultant presence/absence matrix (1,053 × 24,714) was subjected to 2D clustering analysis (A). Presence and absence of an octamer in a promoter is shown as red and black, respectively. Labels of the matrix in vertical and horizontal dimensions are omitted. Octamers of 20 largest clusters were subjected to WebLogo (B). Some pairs of cluster (1–9, 2–15, 3–4, 5–7, 10–17, 12–13) are complementary, respectively. Clusters 5 and 7 are ERSE (CCAAT-N<sub>9</sub>-CCACG).</p></div></div><div content-id="dsw065-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/m_dsw065f4.png?Expires=2147483647&Signature=W9-YGh-ZACc9vtZcwu96DZM9SGor8d43HLixKfnOYBDKy57j195reZOfUdWjfIdSSYS-0hME6jBkzlBX9NF4psoI4KVbGMIygkMq5ugmnDBvrLDmfrXM9EldH1KhOJJOKv~W6s8y8sN2IWSS4CxUIAYb7po5mgPhowk~Owmfk5jvvwVTc9S9EbKmNBqcm3tq~CtdAl-bnSuLpOeAb8YGcdxZZU2bMAqe~xJ1Oq6qtVk-diVFIfcdZSICYstB1TNSn~e6efLd5OiW8cUZnfmfnzB9SMOyAj5PQKgLQ~zCbRw-ZxyItE4iCvQGbiXDKciCyDnTMHV~imXBDZxy6Kqs9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Several groups in the REG-type bipartite octamers. 1,053 bipartite octamers with distribution profiles of the REG type (Fig. 2C) were mapped to 24,714 Arabidopsis promoters, and a resultant presence/absence matrix (1,053 × 24,714) was subjected to 2D clustering analysis (A). Presence and absence of an octamer in a promoter is shown as red and black, respectively. Labels of the matrix in vertical and horizontal dimensions are omitted. Octamers of 20 largest clusters were subjected to WebLogo (B). Some pairs of cluster (1–9, 2–15, 3–4, 5–7, 10–17, 12–13) are complementary, respectively. Clusters 5 and 7 are ERSE (CCAAT-N9-CCACG)." path-from-xml="dsw065f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473548/dsw065f4.tif" path-from-xml="dsw065f4.tif" target="_blank">View large</a><a section="88473548" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/dsw065f4.png?Expires=2147483647&Signature=oyX3xW9N546YcwecEIcCZA9i3Qz8GqboXLGJlK8cOgKAbvh04umbJbWSgYba4I5LvDbNTIxcsf8e-dgTylOLiTFQroj0rXIj7H6bO4c3tUXmCNOiKJTVhcIAUVAgIa0Qs9VtNMlTIR4yWFu~4xDo1-FALP7UUSkAs-KB7gBHPcUjJiuHiLi2G-zSLIE-TEMIYOr45CvWi2QaH1BSz4MvsK4csN1yycYHi6aT2d-K-yRpmRuJIVtC6vhCKoCcbgumbAV5eyk3hzi6TgCqVoVZFpr-UutsHd6owJlVLqmkZrCHUWVd8x64V-8wrSAeIV0crs9VpZeLR7H72ldGFlltIQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473548&ar=2965793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw065f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Several groups in the REG-type bipartite octamers. 1,053 bipartite octamers with distribution profiles of the REG type (<a reveal-id="dsw065-F2" data-open="dsw065-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>) were mapped to 24,714 Arabidopsis promoters, and a resultant presence/absence matrix (1,053 × 24,714) was subjected to 2D clustering analysis (A). Presence and absence of an octamer in a promoter is shown as red and black, respectively. Labels of the matrix in vertical and horizontal dimensions are omitted. Octamers of 20 largest clusters were subjected to WebLogo (B). Some pairs of cluster (1–9, 2–15, 3–4, 5–7, 10–17, 12–13) are complementary, respectively. Clusters 5 and 7 are ERSE (CCAAT-N<sub>9</sub>-CCACG).</p></div></div><p>We subsequently selected 20 largest clusters for summarization of the bipartite octamers for each cluster as sequence motifs. The motifs prepared with WebLogo<a reveal-id="dsw065-B12" data-open="dsw065-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> are shown in Panel B. Several tandem repeats (clusters 1, 6, 9, 12, and 13) and also palindromic sequence (clusters 8 and 11) are included. We suggest that they are recognized by a dimer or a trimer of the same protein family.</p>                            <h3 scrollto-destination=88473550 id="88473550" class="section-title" >3.2. Inclusion of known transcriptional regulatory elements in selected bipartite octamers</h3>
<p>The 8,044 selected sequences include REG-type groups. According to their LDSS profiles, they are strongly suggested to be transcriptional regulatory sequences.<a reveal-id="dsw065-B10" data-open="dsw065-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Their inclusion in the selected sequences is an indication of the ability of the procedure utilized to identify potential regulatory sequences.</p><p>In order to find other features that may be regulatory elements in the sequences selected, we surveyed known transcriptional regulatory elements within the sequences. One of the reported bipartite regulatory motifs is the endoplasmic reticulum stress-responsive element (ERSE, consensus sequence CCAAT-N<sub>9</sub>-CCACG) that is conserved between mammals and higher plants.<a reveal-id="dsw065-B16" data-open="dsw065-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> The ERSE in mammals is known to be recognized by a general transcription factor NF-Y for CCAAT and the stress responsive transcription factor ATF6 for CCACG, and spacing between the two elements is fixed to nine nucleotides.<a reveal-id="dsw065-B17" data-open="dsw065-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> An ERSE is also found in Arabidopsis promoters and is known to have a conserved function.<a reveal-id="dsw065-B18" data-open="dsw065-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a></p><p>When we searched the selected sequences for ERSEs, 28 related sequences were found (<a reveal-id="dsw065-T2" data-open="dsw065-T2" class="link link-reveal link-table xref-fig">Table 2</a>), indicating the inclusion of these elements in the extracted sequences. Then, microarray data of the transcriptional response to ER stress activated by tunicamycin treatment<a reveal-id="dsw065-B19" data-open="dsw065-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> were used to calculate the RAR values of the selected sequences. Typically, RAR values of over 3.0 are considered ‘positive’ as transcriptional regulatory sequences.<a reveal-id="dsw065-B3" data-open="dsw065-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> As shown in <a reveal-id="dsw065-T2" data-open="dsw065-T2" class="link link-reveal link-table xref-fig">Table 2</a>, the RAR values of most of the ERSE-related sequences are much &gt;3.0 (shown in bold in the table). These results mean that the bipartite ERSEs can be detected as ER stress (tunicamycin treatment)-responsive elements in our promoter prediction using the selected bipartite octamers (4 + 4). The application of such promoter prediction will be demonstrated later in this article. <div content-id="dsw065-T2" class="table-modal"><div class="table-wrap"><div id="dsw065-T2" data-id="dsw065-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>ERSE sequences in the selected bipartite octamers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LDSS cluster </th><th>Sequence </th><th>Motif </th><th>RAR (Tunicamycin_up) </th></tr></thead><tbody><tr><td>1_REG </td><td>A<span class="underline">CCA</span>…………<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.42</strong> </td></tr><tr><td>1_REG </td><td>T<span class="underline">CCA</span>…………<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>6.4</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CCAA</span>………G<span class="underline">CCA</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>3.52</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CCAA</span>……….<span class="underline">CCAC</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.65</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CCAA</span>……….<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>6.42</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CCAA</span>…………<span class="underline">ACG</span>T </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>5.64</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">CAAT</span>………<span class="underline">CCAC</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>5.84</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CAAT</span>……….<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>7.76</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CAAT</span>……….<span class="underline">ACG</span>T </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>6.83</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CAAT</span>…………<span class="underline">CG</span>TG </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.95</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">AAT</span>A………<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td>2.3 </td></tr><tr><td>2_REG </td><td><span class="underline">AAT</span>C………<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>6.8</strong> </td></tr><tr><td>4_REG-like </td><td>CA<span class="underline">CG</span>…………<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>8.53</strong> </td></tr><tr><td>2_REG </td><td>GA<span class="underline">CG</span>…………<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.39</strong> </td></tr><tr><td>1_REG </td><td>A<span class="underline">CGT</span>……….G<span class="underline">ATT</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>3.93</strong> </td></tr><tr><td>2_REG </td><td>A<span class="underline">CGT</span>……….C<span class="underline">ATT</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.07</strong> </td></tr><tr><td>1_REG </td><td>A<span class="underline">CGT</span>……….<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>8.69</strong> </td></tr><tr><td>1_REG </td><td>A<span class="underline">CGT</span>…………<span class="underline">TTGG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>10.25</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">CGTG</span>………T<span class="underline">ATT</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.37</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CGTG</span>………G<span class="underline">ATT</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.28</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">CGTG</span>………C<span class="underline">ATT</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>6.64</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CGTG</span>……….<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>11.96</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CGTG</span>……….<span class="underline">TTGG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>12.8</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CGTG</span>…………<span class="underline">TGG</span>T </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>10.88</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">GTGG</span>………<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>5.69</strong> </td></tr><tr><td>3_REG-like </td><td><span class="underline">GTG</span>T………<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.87</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">TGG</span>C………<span class="underline">TTGG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>5.74</strong> </td></tr><tr><td>4_REG-like </td><td><span class="underline">TG</span>TC………<span class="underline">TTGG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>5.09</strong> </td></tr></tbody></table></div><div content-id="dsw065-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>LDSS cluster </th><th>Sequence </th><th>Motif </th><th>RAR (Tunicamycin_up) </th></tr></thead><tbody><tr><td>1_REG </td><td>A<span class="underline">CCA</span>…………<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.42</strong> </td></tr><tr><td>1_REG </td><td>T<span class="underline">CCA</span>…………<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>6.4</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CCAA</span>………G<span class="underline">CCA</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>3.52</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CCAA</span>……….<span class="underline">CCAC</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.65</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CCAA</span>……….<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>6.42</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CCAA</span>…………<span class="underline">ACG</span>T </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>5.64</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">CAAT</span>………<span class="underline">CCAC</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>5.84</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CAAT</span>……….<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>7.76</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CAAT</span>……….<span class="underline">ACG</span>T </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>6.83</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CAAT</span>…………<span class="underline">CG</span>TG </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.95</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">AAT</span>A………<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td>2.3 </td></tr><tr><td>2_REG </td><td><span class="underline">AAT</span>C………<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>6.8</strong> </td></tr><tr><td>4_REG-like </td><td>CA<span class="underline">CG</span>…………<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>8.53</strong> </td></tr><tr><td>2_REG </td><td>GA<span class="underline">CG</span>…………<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.39</strong> </td></tr><tr><td>1_REG </td><td>A<span class="underline">CGT</span>……….G<span class="underline">ATT</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>3.93</strong> </td></tr><tr><td>2_REG </td><td>A<span class="underline">CGT</span>……….C<span class="underline">ATT</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.07</strong> </td></tr><tr><td>1_REG </td><td>A<span class="underline">CGT</span>……….<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>8.69</strong> </td></tr><tr><td>1_REG </td><td>A<span class="underline">CGT</span>…………<span class="underline">TTGG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>10.25</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">CGTG</span>………T<span class="underline">ATT</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.37</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CGTG</span>………G<span class="underline">ATT</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.28</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">CGTG</span>………C<span class="underline">ATT</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>6.64</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CGTG</span>……….<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>11.96</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CGTG</span>……….<span class="underline">TTGG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>12.8</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CGTG</span>…………<span class="underline">TGG</span>T </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>10.88</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">GTGG</span>………<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>5.69</strong> </td></tr><tr><td>3_REG-like </td><td><span class="underline">GTG</span>T………<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.87</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">TGG</span>C………<span class="underline">TTGG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>5.74</strong> </td></tr><tr><td>4_REG-like </td><td><span class="underline">TG</span>TC………<span class="underline">TTGG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>5.09</strong> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>LDSS cluster indicates the group number of LDSS clustering shown in <a reveal-id="dsw065-F2" data-open="dsw065-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>. Sequence matching with ERSE is indicated with underlining. A dot in a sequence means any nucleotide. The RAR values of each octamer for tunicamycin up-regulation are also shown. RAR values &gt;3.0 are shown in bold.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw065-T2" data-id="dsw065-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>ERSE sequences in the selected bipartite octamers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LDSS cluster </th><th>Sequence </th><th>Motif </th><th>RAR (Tunicamycin_up) </th></tr></thead><tbody><tr><td>1_REG </td><td>A<span class="underline">CCA</span>…………<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.42</strong> </td></tr><tr><td>1_REG </td><td>T<span class="underline">CCA</span>…………<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>6.4</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CCAA</span>………G<span class="underline">CCA</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>3.52</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CCAA</span>……….<span class="underline">CCAC</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.65</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CCAA</span>……….<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>6.42</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CCAA</span>…………<span class="underline">ACG</span>T </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>5.64</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">CAAT</span>………<span class="underline">CCAC</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>5.84</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CAAT</span>……….<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>7.76</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CAAT</span>……….<span class="underline">ACG</span>T </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>6.83</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CAAT</span>…………<span class="underline">CG</span>TG </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.95</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">AAT</span>A………<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td>2.3 </td></tr><tr><td>2_REG </td><td><span class="underline">AAT</span>C………<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>6.8</strong> </td></tr><tr><td>4_REG-like </td><td>CA<span class="underline">CG</span>…………<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>8.53</strong> </td></tr><tr><td>2_REG </td><td>GA<span class="underline">CG</span>…………<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.39</strong> </td></tr><tr><td>1_REG </td><td>A<span class="underline">CGT</span>……….G<span class="underline">ATT</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>3.93</strong> </td></tr><tr><td>2_REG </td><td>A<span class="underline">CGT</span>……….C<span class="underline">ATT</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.07</strong> </td></tr><tr><td>1_REG </td><td>A<span class="underline">CGT</span>……….<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>8.69</strong> </td></tr><tr><td>1_REG </td><td>A<span class="underline">CGT</span>…………<span class="underline">TTGG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>10.25</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">CGTG</span>………T<span class="underline">ATT</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.37</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CGTG</span>………G<span class="underline">ATT</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.28</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">CGTG</span>………C<span class="underline">ATT</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>6.64</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CGTG</span>……….<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>11.96</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CGTG</span>……….<span class="underline">TTGG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>12.8</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CGTG</span>…………<span class="underline">TGG</span>T </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>10.88</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">GTGG</span>………<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>5.69</strong> </td></tr><tr><td>3_REG-like </td><td><span class="underline">GTG</span>T………<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.87</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">TGG</span>C………<span class="underline">TTGG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>5.74</strong> </td></tr><tr><td>4_REG-like </td><td><span class="underline">TG</span>TC………<span class="underline">TTGG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>5.09</strong> </td></tr></tbody></table></div><div content-id="dsw065-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>LDSS cluster </th><th>Sequence </th><th>Motif </th><th>RAR (Tunicamycin_up) </th></tr></thead><tbody><tr><td>1_REG </td><td>A<span class="underline">CCA</span>…………<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.42</strong> </td></tr><tr><td>1_REG </td><td>T<span class="underline">CCA</span>…………<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>6.4</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CCAA</span>………G<span class="underline">CCA</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>3.52</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CCAA</span>……….<span class="underline">CCAC</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.65</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CCAA</span>……….<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>6.42</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CCAA</span>…………<span class="underline">ACG</span>T </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>5.64</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">CAAT</span>………<span class="underline">CCAC</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>5.84</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CAAT</span>……….<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>7.76</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CAAT</span>……….<span class="underline">ACG</span>T </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>6.83</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CAAT</span>…………<span class="underline">CG</span>TG </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.95</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">AAT</span>A………<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td>2.3 </td></tr><tr><td>2_REG </td><td><span class="underline">AAT</span>C………<span class="underline">CACG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>6.8</strong> </td></tr><tr><td>4_REG-like </td><td>CA<span class="underline">CG</span>…………<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>8.53</strong> </td></tr><tr><td>2_REG </td><td>GA<span class="underline">CG</span>…………<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.39</strong> </td></tr><tr><td>1_REG </td><td>A<span class="underline">CGT</span>……….G<span class="underline">ATT</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>3.93</strong> </td></tr><tr><td>2_REG </td><td>A<span class="underline">CGT</span>……….C<span class="underline">ATT</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.07</strong> </td></tr><tr><td>1_REG </td><td>A<span class="underline">CGT</span>……….<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>8.69</strong> </td></tr><tr><td>1_REG </td><td>A<span class="underline">CGT</span>…………<span class="underline">TTGG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>10.25</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">CGTG</span>………T<span class="underline">ATT</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.37</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CGTG</span>………G<span class="underline">ATT</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.28</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">CGTG</span>………C<span class="underline">ATT</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>6.64</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CGTG</span>……….<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>11.96</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CGTG</span>……….<span class="underline">TTGG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>12.8</strong> </td></tr><tr><td>1_REG </td><td><span class="underline">CGTG</span>…………<span class="underline">TGG</span>T </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>10.88</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">GTGG</span>………<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>5.69</strong> </td></tr><tr><td>3_REG-like </td><td><span class="underline">GTG</span>T………<span class="underline">ATTG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>4.87</strong> </td></tr><tr><td>2_REG </td><td><span class="underline">TGG</span>C………<span class="underline">TTGG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>5.74</strong> </td></tr><tr><td>4_REG-like </td><td><span class="underline">TG</span>TC………<span class="underline">TTGG</span> </td><td>ERSE: CCAAT-N<sub>9</sub>-CCACG </td><td><strong>5.09</strong> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>LDSS cluster indicates the group number of LDSS clustering shown in <a reveal-id="dsw065-F2" data-open="dsw065-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>. Sequence matching with ERSE is indicated with underlining. A dot in a sequence means any nucleotide. The RAR values of each octamer for tunicamycin up-regulation are also shown. RAR values &gt;3.0 are shown in bold.</p></div></div></div></div></p><p>Another reported bipartite motif of higher plants is the NAC-binding motif (CTTG-N5-CAAG) that is recognized by Arabidopsis ANAC013.<a reveal-id="dsw065-B20" data-open="dsw065-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> This is also called the mitochondrial dysfunction motif (MDM) which includes the NAC-binding site and has several sequence variations.<a reveal-id="dsw065-B20" data-open="dsw065-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Our extracted bipartite sequences included 13 sequences related to the NAC-binding motif (<a reveal-id="dsw065-T3" data-open="dsw065-T3" class="link link-reveal link-table xref-fig">Table 3</a>). The presence of these sequences in the extracted sequences also supports the idea that our method does extract bipartite transcriptional regulatory sequences. One feature of this group is non-REG-type distribution (<a reveal-id="dsw065-T3" data-open="dsw065-T3" class="link link-reveal link-table xref-fig">Table 3</a>). <div content-id="dsw065-T3" class="table-modal"><div class="table-wrap"><div id="dsw065-T3" data-id="dsw065-T3" class="table-wrap-title"><div class="label -label">Table 3</div><div class="caption"><p>NAC target sequences in the selected bipartite octamers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LDSS cluster </th><th>Sequence </th><th>Motif </th></tr></thead><tbody><tr><td>7_non-REG </td><td>A<span class="underline">CTT</span>……<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>3_REG-like </td><td>C<span class="underline">CTT</span>……<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td>G<span class="underline">CTT</span>……<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">CTTG</span>….TG<span class="underline">AA</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>3_REG-like </td><td><span class="underline">CTTG</span>….<span class="underline">CA</span>C<span class="underline">G</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">CTT</span>C….<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">CTTG</span>….G<span class="underline">AAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>7_non-REG </td><td><span class="underline">CTTG</span>….<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">C</span>G<span class="underline">TG</span>….<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>4_REG-like </td><td><span class="underline">CTTG</span>……<span class="underline">AAG</span>T </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">CTTG</span>……<span class="underline">AAG</span>G </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">CTTG</span>……<span class="underline">AAG</span>A </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">TTG</span>C…TG<span class="underline">AA</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr></tbody></table></div><div content-id="dsw065-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>LDSS cluster </th><th>Sequence </th><th>Motif </th></tr></thead><tbody><tr><td>7_non-REG </td><td>A<span class="underline">CTT</span>……<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>3_REG-like </td><td>C<span class="underline">CTT</span>……<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td>G<span class="underline">CTT</span>……<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">CTTG</span>….TG<span class="underline">AA</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>3_REG-like </td><td><span class="underline">CTTG</span>….<span class="underline">CA</span>C<span class="underline">G</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">CTT</span>C….<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">CTTG</span>….G<span class="underline">AAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>7_non-REG </td><td><span class="underline">CTTG</span>….<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">C</span>G<span class="underline">TG</span>….<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>4_REG-like </td><td><span class="underline">CTTG</span>……<span class="underline">AAG</span>T </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">CTTG</span>……<span class="underline">AAG</span>G </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">CTTG</span>……<span class="underline">AAG</span>A </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">TTG</span>C…TG<span class="underline">AA</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p>LDSS cluster indicates the group number of LDSS clustering shown in <a reveal-id="dsw065-F2" data-open="dsw065-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>. Sequence matching with the NAC target motif is underlined. A dot in a sequence means any nucleotide.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw065-T3" data-id="dsw065-T3" class="table-wrap-title"><div class="label -label">Table 3</div><div class="caption"><p>NAC target sequences in the selected bipartite octamers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LDSS cluster </th><th>Sequence </th><th>Motif </th></tr></thead><tbody><tr><td>7_non-REG </td><td>A<span class="underline">CTT</span>……<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>3_REG-like </td><td>C<span class="underline">CTT</span>……<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td>G<span class="underline">CTT</span>……<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">CTTG</span>….TG<span class="underline">AA</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>3_REG-like </td><td><span class="underline">CTTG</span>….<span class="underline">CA</span>C<span class="underline">G</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">CTT</span>C….<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">CTTG</span>….G<span class="underline">AAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>7_non-REG </td><td><span class="underline">CTTG</span>….<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">C</span>G<span class="underline">TG</span>….<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>4_REG-like </td><td><span class="underline">CTTG</span>……<span class="underline">AAG</span>T </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">CTTG</span>……<span class="underline">AAG</span>G </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">CTTG</span>……<span class="underline">AAG</span>A </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">TTG</span>C…TG<span class="underline">AA</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr></tbody></table></div><div content-id="dsw065-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>LDSS cluster </th><th>Sequence </th><th>Motif </th></tr></thead><tbody><tr><td>7_non-REG </td><td>A<span class="underline">CTT</span>……<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>3_REG-like </td><td>C<span class="underline">CTT</span>……<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td>G<span class="underline">CTT</span>……<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">CTTG</span>….TG<span class="underline">AA</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>3_REG-like </td><td><span class="underline">CTTG</span>….<span class="underline">CA</span>C<span class="underline">G</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">CTT</span>C….<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">CTTG</span>….G<span class="underline">AAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>7_non-REG </td><td><span class="underline">CTTG</span>….<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">C</span>G<span class="underline">TG</span>….<span class="underline">CAAG</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>4_REG-like </td><td><span class="underline">CTTG</span>……<span class="underline">AAG</span>T </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">CTTG</span>……<span class="underline">AAG</span>G </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">CTTG</span>……<span class="underline">AAG</span>A </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr><tr><td>9_non-REG </td><td><span class="underline">TTG</span>C…TG<span class="underline">AA</span> </td><td>NAC: CTTG-N<sub>5</sub>-CAAG </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p>LDSS cluster indicates the group number of LDSS clustering shown in <a reveal-id="dsw065-F2" data-open="dsw065-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>. Sequence matching with the NAC target motif is underlined. A dot in a sequence means any nucleotide.</p></div></div></div></div></p><p>As the NAC gene family is composed of 94–106 genes in Arabidopsis,<a reveal-id="dsw065-B8" data-open="dsw065-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> variations in the target sequence among this family and differentiation of their function could be present. Therefore, the sequences in <a reveal-id="dsw065-T3" data-open="dsw065-T3" class="link link-reveal link-table xref-fig">Table 3</a> may be recognized by different NAC proteins.</p>                            <h3 scrollto-destination=88473556 id="88473556" class="section-title" >3.3. Applications of bipartite octamer analysis to promoter prediction</h3>
<p>The selected 8,044 bipartite octamers were utilized for promoter prediction. An ER stress-responsive BiP3 promoter was selected for the prediction of tunicamycin-responsive elements. The results of this prediction were compared with those obtained using the established method using octamers without any spacers.</p><p><a reveal-id="dsw065-F5" data-open="dsw065-F5" class="link link-reveal link-table xref-fig">Figure 5A</a> shows the results of promoter scans of BiP3 for tunicamycin-responsive elements. The vertical axis shows the RAR, and typically peaks &gt;3.0 are selected as prediction sites. As shown, several peaks are &gt;3.0, and two of them match experimentally identified ERSEs, ERSE-A, and ERSE-B.<a reveal-id="dsw065-B21" data-open="dsw065-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> The RAR values of bipartite octamers are indicated with red X marks, and they show much higher RAR values than the continuous octamers at the positions of ERSEs, demonstrating the superiority of the bipartite octamers in detection of the ERSE.</p>                        <a id="88473559" scrollto-destination="88473559"></a>
<div id="dsw065-F5" data-id="dsw065-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/m_dsw065f5.png?Expires=2147483647&Signature=pPSPw1cDAPPCZVUTFs0dhnPQBCVgcjY0IvmQBXB2juT7KnE16S~ERa49~lyeFZnduyO2PBr5ydRsYQwjAQHotGxWBTrq48lRZWAb-C96iQaTBqwxz6IWItrM7YT-e1ZpcIY2ld8PscEMjMgSrgfmDz~v33tndtQw5xKgL1Ot2a2aXwkHF0f9Yb5KSysqdn9H4KKLQttnXb7IrDWnAzuXkhwoNVMieWoxW6otKPxa95-OrvkHLUGICyX32D61STmMXxHzGzrXt00P6c7KO6iAyvFuY22PUw~-JUgYfxQpAlfTSxR3ylW8l5pc~W3KwT5m9Efhy9ram3vRMdWD3HIlQQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Promoter prediction of BiP3/BiP-L for tunicamycin response. Application of the bipartite sequences for prediction of tunicamycin-responsive elements in a BiP/BiP-L (AT1G09080) promoter. (A) Results of promoter scans using octamers (green line graph) and bipartite octamers (red X mark) are shown. The Relative Appearance Ratio (RAR) means the degree of overrepresentation in tunicamycin-responsive promoters over the total promoters and thus high RAR sites predict corresponding regulatory elements. A height of RAR = 3.0, a conventional threshold,3 is indicated by a black horizontal line. Positions of experimentally identified ERSEs, ERSE-A and ERSE-B, are also shown in the graph. (B and C) Sequences from -247 to -198 (B) and from -147 to -98 (C), relative to the translation start site (ATG), are shown. Green letters show predictions based on continuous octamers, and red letters are predictions based on bipartite octamers. Underlining indicates sequences that match with the ERSE motif (CCAAT-N9-CCACG). Values in parentheses are RAR scores that show the ratio of overrepresentation in tunicamycin-responsive promoters over the global promoter set in the Arabidopsis genome. ERSE-A and ERSE-B are functionally confirmed ERSEs reported by Noh et al.19 Sequence in bold means an ERSE motif in ERSE-A and ERSE-B, as reported by Noh et al." path-from-xml="dsw065f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473559/dsw065f5.tif" path-from-xml="dsw065f5.tif" target="_blank">View large</a><a section="88473559" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/dsw065f5.png?Expires=2147483647&Signature=TfBFH10xwgrAZe01XE6fTh-NOiUPOyzO3arcrTdqT9S5I9Gn4UlauRs0GuRSgCv63zekR6C77F5MJoXYH7YpYnEOGRe0OVl166vXNudCacLZXg9VPEDTvZfV~T4P1H8E59kXCLe1NqUGEK3IdI9xQ8r4MGxHRyesJ~VqHump-oEtD208zNzThB471LnY0nxQNWmQuxXen1qr-bHbg66i6sXUBOtjxV9aj0T4XecLt~Es0J3s7uBN99Qiz8mBgUgYv9SOvecHrN8NEjwGY4G4OJ-7JPgTmxr-Qcss4Zc9IOATIIo37~JyPcc2EZq0I5KJE0CBrZsaSBVSgoU~DRybtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473559&ar=2965793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw065f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Promoter prediction of BiP3/BiP-L for tunicamycin response. Application of the bipartite sequences for prediction of tunicamycin-responsive elements in a BiP/BiP-L (AT1G09080) promoter. (A) Results of promoter scans using octamers (green line graph) and bipartite octamers (red X mark) are shown. The Relative Appearance Ratio (RAR) means the degree of overrepresentation in tunicamycin-responsive promoters over the total promoters and thus high RAR sites predict corresponding regulatory elements. A height of RAR = 3.0, a conventional threshold,<a reveal-id="dsw065-B3" data-open="dsw065-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> is indicated by a black horizontal line. Positions of experimentally identified ERSEs, ERSE-A and ERSE-B, are also shown in the graph. (B and C) Sequences from -247 to -198 (B) and from -147 to -98 (C), relative to the translation start site (ATG), are shown. Green letters show predictions based on continuous octamers, and red letters are predictions based on bipartite octamers. Underlining indicates sequences that match with the ERSE motif (CCAAT-N<sub>9</sub>-CCACG). Values in parentheses are RAR scores that show the ratio of overrepresentation in tunicamycin-responsive promoters over the global promoter set in the Arabidopsis genome. ERSE-A and ERSE-B are functionally confirmed ERSEs reported by Noh et al.<a reveal-id="dsw065-B19" data-open="dsw065-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Sequence in bold means an ERSE motif in ERSE-A and ERSE-B, as reported by Noh et al.</p></div></div><div content-id="dsw065-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/m_dsw065f5.png?Expires=2147483647&Signature=pPSPw1cDAPPCZVUTFs0dhnPQBCVgcjY0IvmQBXB2juT7KnE16S~ERa49~lyeFZnduyO2PBr5ydRsYQwjAQHotGxWBTrq48lRZWAb-C96iQaTBqwxz6IWItrM7YT-e1ZpcIY2ld8PscEMjMgSrgfmDz~v33tndtQw5xKgL1Ot2a2aXwkHF0f9Yb5KSysqdn9H4KKLQttnXb7IrDWnAzuXkhwoNVMieWoxW6otKPxa95-OrvkHLUGICyX32D61STmMXxHzGzrXt00P6c7KO6iAyvFuY22PUw~-JUgYfxQpAlfTSxR3ylW8l5pc~W3KwT5m9Efhy9ram3vRMdWD3HIlQQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Promoter prediction of BiP3/BiP-L for tunicamycin response. Application of the bipartite sequences for prediction of tunicamycin-responsive elements in a BiP/BiP-L (AT1G09080) promoter. (A) Results of promoter scans using octamers (green line graph) and bipartite octamers (red X mark) are shown. The Relative Appearance Ratio (RAR) means the degree of overrepresentation in tunicamycin-responsive promoters over the total promoters and thus high RAR sites predict corresponding regulatory elements. A height of RAR = 3.0, a conventional threshold,3 is indicated by a black horizontal line. Positions of experimentally identified ERSEs, ERSE-A and ERSE-B, are also shown in the graph. (B and C) Sequences from -247 to -198 (B) and from -147 to -98 (C), relative to the translation start site (ATG), are shown. Green letters show predictions based on continuous octamers, and red letters are predictions based on bipartite octamers. Underlining indicates sequences that match with the ERSE motif (CCAAT-N9-CCACG). Values in parentheses are RAR scores that show the ratio of overrepresentation in tunicamycin-responsive promoters over the global promoter set in the Arabidopsis genome. ERSE-A and ERSE-B are functionally confirmed ERSEs reported by Noh et al.19 Sequence in bold means an ERSE motif in ERSE-A and ERSE-B, as reported by Noh et al." path-from-xml="dsw065f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473559/dsw065f5.tif" path-from-xml="dsw065f5.tif" target="_blank">View large</a><a section="88473559" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/dsw065f5.png?Expires=2147483647&Signature=TfBFH10xwgrAZe01XE6fTh-NOiUPOyzO3arcrTdqT9S5I9Gn4UlauRs0GuRSgCv63zekR6C77F5MJoXYH7YpYnEOGRe0OVl166vXNudCacLZXg9VPEDTvZfV~T4P1H8E59kXCLe1NqUGEK3IdI9xQ8r4MGxHRyesJ~VqHump-oEtD208zNzThB471LnY0nxQNWmQuxXen1qr-bHbg66i6sXUBOtjxV9aj0T4XecLt~Es0J3s7uBN99Qiz8mBgUgYv9SOvecHrN8NEjwGY4G4OJ-7JPgTmxr-Qcss4Zc9IOATIIo37~JyPcc2EZq0I5KJE0CBrZsaSBVSgoU~DRybtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473559&ar=2965793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw065f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Promoter prediction of BiP3/BiP-L for tunicamycin response. Application of the bipartite sequences for prediction of tunicamycin-responsive elements in a BiP/BiP-L (AT1G09080) promoter. (A) Results of promoter scans using octamers (green line graph) and bipartite octamers (red X mark) are shown. The Relative Appearance Ratio (RAR) means the degree of overrepresentation in tunicamycin-responsive promoters over the total promoters and thus high RAR sites predict corresponding regulatory elements. A height of RAR = 3.0, a conventional threshold,<a reveal-id="dsw065-B3" data-open="dsw065-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> is indicated by a black horizontal line. Positions of experimentally identified ERSEs, ERSE-A and ERSE-B, are also shown in the graph. (B and C) Sequences from -247 to -198 (B) and from -147 to -98 (C), relative to the translation start site (ATG), are shown. Green letters show predictions based on continuous octamers, and red letters are predictions based on bipartite octamers. Underlining indicates sequences that match with the ERSE motif (CCAAT-N<sub>9</sub>-CCACG). Values in parentheses are RAR scores that show the ratio of overrepresentation in tunicamycin-responsive promoters over the global promoter set in the Arabidopsis genome. ERSE-A and ERSE-B are functionally confirmed ERSEs reported by Noh et al.<a reveal-id="dsw065-B19" data-open="dsw065-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Sequence in bold means an ERSE motif in ERSE-A and ERSE-B, as reported by Noh et al.</p></div></div><p><a reveal-id="dsw065-F5" data-open="dsw065-F5" class="link link-reveal link-table xref-fig">Figure 5B and C</a> shows sequences around ERSE-A (Panel B) and ERSE-B (Panel C), and the predictions using continuous octamers are shown in green letters. In both cases, the CCACG site of the ERSE, which is the recognition site (<span class="underline">CGTG</span>T, the underlining shows the match with the ERSE) of the ATF6 homolog of Arabidopsis, bZIP28,<a reveal-id="dsw065-B18" data-open="dsw065-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> was detected by continuous octamer analysis, but the NF-Y binding site (<span class="underline">ATTGG</span>, the underlining shows the match with ERSE)<a reveal-id="dsw065-B18" data-open="dsw065-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> was not. These results indicate that the bZIP binding site of the ERSE has some specificity in ER-stress responsive promoters, but the NF-Y site does not have any specificity in the same promoters, resulting in the failure to detect the NF-Y site as an ERSE.</p><p>For comparison, the same gene list from the microarray data of tunicamycin response has been applied to Gibbs Sampler, MEME, CONSENSUS and MD Scan provided by Melina II.<a reveal-id="dsw065-B22" data-open="dsw065-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> These four methods could not detect either ERSE-A or ERSE-B (data not shown).</p><p>Predictions using our bipartite octamers are shown in red letters in <a reveal-id="dsw065-F5" data-open="dsw065-F5" class="link link-reveal link-table xref-fig">Fig. 5B and C</a>. Both the bZIP and NF-Y binding sites were detected as putative ERSEs, indicating the advantage of using the bipartite octamer analysis to predict the bipartite ERSE. Comparison of the RAR signals between the two methods reveals that signals of the bipartite octamer analysis are much higher than those of the continuous octamer analysis (Panel A). This indicates higher sequence specificity of the detected ERSE sequences in the bipartite octamer method than in the continuous method.</p><p>These results, shown in <a reveal-id="dsw065-F5" data-open="dsw065-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>, demonstrate that detection of ERSEs among tunicamycin-stimulated ER-stress responsive promoters has been considerably improved by introducing the bipartite octamer analysis.</p><p>The second application of our new method is the prediction of ANAC017 target sites using microarray data of Arabidopsis ANAC017 mutants.<a reveal-id="dsw065-B14" data-open="dsw065-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> As ANAC017 is a mediator of H<sub>2</sub>O<sub>2</sub> signaling, the effect of a mutation was compared with wild type after H<sub>2</sub>O<sub>2</sub> treatments. As shown in <a reveal-id="dsw065-F6" data-open="dsw065-F6" class="link link-reveal link-table xref-fig">Fig. 6A, a</a> promoter region of <em>ATERF71</em> was predicted as a target site of ANAC017 (underlined) according to the following results: the expression level of <em>ATERF71</em> was reduced in the <em>ANAC017</em> mutants, and a target site of ANAC017 predicted by bipartite octamer analysis was mapped to the promoter region of <em>ATERF71</em>.</p>                        <a id="88473565" scrollto-destination="88473565"></a>
<div id="dsw065-F6" data-id="dsw065-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/m_dsw065f6.png?Expires=2147483647&Signature=O~QwhLwQ2n5xFXMdpBsqwh0sqCFqZmu0AmJLXRa1LOtcdUhGV7w~njs47yNiVFQMVllYZGC9Q1~E7PoWHDiTQTUmlv-FA~p~v6lw~2UGPiYK9KeE~f3EUuRwurlMVflENVYg6bk~M~ChpaKT9BkBTaXDUkDWMJgbiPkXfSk1W8ZADRUJrNnyam1oo-SEioPToyoLck55LAndAZ6ssglCUhadNZUe2RYQOPKKtEIV50tZqZ3WlWNayeJtcz6AxZEtDGCaQR-32J631n68y63kmf-uaLkMkYUFkV0gy8cWbqWLwLiDp1P-mndbW1bWvMHSJ8mkaRxoPXSXo9mXf0l6mA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Direct binding of ANAC017 to a bipartite octamer predicted for H2O2 response. (A) The predicted target site of ANAC017 in an ATERF71 (AT2G47520) promoter was used as a biotinylated DNA probe in a DNA–protein binding assay by AlphaScreen. Underlining shows the predicted target site identified by the bipartite analysis. The sequences of unbiotinylated competitor DNAs with base substitutions from the biotinylated probe are shown (M0–M8, unsubstituted bases are shown as dots). Asterisks on the top of the probe sequence indicate critical bases for binding by ANAC017 protein, revealed by Panel B. (B) Results of the AlphaScreen assay are shown as the relative AlphaScreen signal (the signal obtained with non-biotinylated probe/signal with biotinylated probe). The average and standard error of four assays are shown." path-from-xml="dsw065f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473565/dsw065f6.tif" path-from-xml="dsw065f6.tif" target="_blank">View large</a><a section="88473565" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/dsw065f6.png?Expires=2147483647&Signature=yx7Y9T1SpeVsw4tNk6hgXnsUEtORaF4eQQn6pRVItPusXAQ~ve2MwZX~s540g5RVSxRUwN53TXGu4tmgDHcXW7fGOeUFVqlQAwWm9xziEimbZ5JOYo9~Hm17-ZclRIQTEpjwe98xhZUyo2p1yrAn1~qGfNt-UOrwyRo3OFHV9tfVo4BaT~cgc2dznZJXL6Uh2qEGNcjOiNG8DyE0YFcLXwJcm1LiBcpWnTj-9tx5EYC6Dr3CMZ4hsrUynfDQwLSYReThRCr3EEHlByR4vvcA7NbHJkjGSuLKgNxB9eKmCvSfmpaHVZMz4GPP3pCAuYwxpoB9i68XE56X1HANx7fzJg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473565&ar=2965793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw065f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Direct binding of ANAC017 to a bipartite octamer predicted for H<sub>2</sub>O<sub>2</sub> response. (A) The predicted target site of ANAC017 in an ATERF71 (AT2G47520) promoter was used as a biotinylated DNA probe in a DNA–protein binding assay by AlphaScreen. Underlining shows the predicted target site identified by the bipartite analysis. The sequences of unbiotinylated competitor DNAs with base substitutions from the biotinylated probe are shown (M0–M8, unsubstituted bases are shown as dots). Asterisks on the top of the probe sequence indicate critical bases for binding by ANAC017 protein, revealed by Panel B. (B) Results of the AlphaScreen assay are shown as the relative AlphaScreen signal (the signal obtained with non-biotinylated probe/signal with biotinylated probe). The average and standard error of four assays are shown.</p></div></div><div content-id="dsw065-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/m_dsw065f6.png?Expires=2147483647&Signature=O~QwhLwQ2n5xFXMdpBsqwh0sqCFqZmu0AmJLXRa1LOtcdUhGV7w~njs47yNiVFQMVllYZGC9Q1~E7PoWHDiTQTUmlv-FA~p~v6lw~2UGPiYK9KeE~f3EUuRwurlMVflENVYg6bk~M~ChpaKT9BkBTaXDUkDWMJgbiPkXfSk1W8ZADRUJrNnyam1oo-SEioPToyoLck55LAndAZ6ssglCUhadNZUe2RYQOPKKtEIV50tZqZ3WlWNayeJtcz6AxZEtDGCaQR-32J631n68y63kmf-uaLkMkYUFkV0gy8cWbqWLwLiDp1P-mndbW1bWvMHSJ8mkaRxoPXSXo9mXf0l6mA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Direct binding of ANAC017 to a bipartite octamer predicted for H2O2 response. (A) The predicted target site of ANAC017 in an ATERF71 (AT2G47520) promoter was used as a biotinylated DNA probe in a DNA–protein binding assay by AlphaScreen. Underlining shows the predicted target site identified by the bipartite analysis. The sequences of unbiotinylated competitor DNAs with base substitutions from the biotinylated probe are shown (M0–M8, unsubstituted bases are shown as dots). Asterisks on the top of the probe sequence indicate critical bases for binding by ANAC017 protein, revealed by Panel B. (B) Results of the AlphaScreen assay are shown as the relative AlphaScreen signal (the signal obtained with non-biotinylated probe/signal with biotinylated probe). The average and standard error of four assays are shown." path-from-xml="dsw065f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473565/dsw065f6.tif" path-from-xml="dsw065f6.tif" target="_blank">View large</a><a section="88473565" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/dsw065f6.png?Expires=2147483647&Signature=yx7Y9T1SpeVsw4tNk6hgXnsUEtORaF4eQQn6pRVItPusXAQ~ve2MwZX~s540g5RVSxRUwN53TXGu4tmgDHcXW7fGOeUFVqlQAwWm9xziEimbZ5JOYo9~Hm17-ZclRIQTEpjwe98xhZUyo2p1yrAn1~qGfNt-UOrwyRo3OFHV9tfVo4BaT~cgc2dznZJXL6Uh2qEGNcjOiNG8DyE0YFcLXwJcm1LiBcpWnTj-9tx5EYC6Dr3CMZ4hsrUynfDQwLSYReThRCr3EEHlByR4vvcA7NbHJkjGSuLKgNxB9eKmCvSfmpaHVZMz4GPP3pCAuYwxpoB9i68XE56X1HANx7fzJg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473565&ar=2965793&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw065f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Direct binding of ANAC017 to a bipartite octamer predicted for H<sub>2</sub>O<sub>2</sub> response. (A) The predicted target site of ANAC017 in an ATERF71 (AT2G47520) promoter was used as a biotinylated DNA probe in a DNA–protein binding assay by AlphaScreen. Underlining shows the predicted target site identified by the bipartite analysis. The sequences of unbiotinylated competitor DNAs with base substitutions from the biotinylated probe are shown (M0–M8, unsubstituted bases are shown as dots). Asterisks on the top of the probe sequence indicate critical bases for binding by ANAC017 protein, revealed by Panel B. (B) Results of the AlphaScreen assay are shown as the relative AlphaScreen signal (the signal obtained with non-biotinylated probe/signal with biotinylated probe). The average and standard error of four assays are shown.</p></div></div><p>In order to confirm the predicted target site of ANAC017, direct binding of ANAC017 protein to the DNA sequence was examined <em>in vitro</em> by the AlphaScreen method. A FLAG-tagged ANAC017 protein was prepared <em>in vitro</em> using wheat germ and subjected to binding assays with a biotinylated oligo DNA probe. As shown in <a reveal-id="dsw065-F6" data-open="dsw065-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>, binding assays of the ANAC017 protein to the probe containing the predicted target site gave positive signals, indicating direct binding (no competitor, Panel B). Addition of a non-biotinylated competitor reduced the signal (competitor M0), but addition of mutated competitors of M1, M2, M3, and M6 failed to reduce the binding signals. The target sequence, revealed by these assays, that is necessary for the binding is shown with asterisks in Panel A. These results demonstrate that the bipartite octamer analysis developed here has predicted the exact target sequence of the ANAC017 protein based on microarray data of ANAC017 mutants.</p>                            <h3 scrollto-destination=88473567 id="88473567" class="section-title" >3.4. Possibility of further extension</h3>
<p>We have tried several patterns other than [4 + 4], but could not obtain good improvements so far. In an example, a zinc finger protein, STOP1, recognizes [3 + 3 + 2] in a target promoter,<a reveal-id="dsw065-B5" data-open="dsw065-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> but prediction of the target site using its knockout data was better with [8] than with [3 + 3 + 2] for some unknown reason. Currently, only results of [4 + 4] are worth reporting. We have given up to increase division patterns. We now think that extension of octamer analysis is enough with addition of [4 + 4].</p><p>In summary, we have successfully developed a <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/dsw065_Supp.xls?Expires=2147483647&Signature=F8Y4fwRwNGJil80iUVJPFsxswa910fM37~tNrinOWpfOAS3mOZ0zu1xYySI0Y5xPGZ4JefOBNzynYqecE-cNSs6u1fr4jY5w5lsi-1ivzNls9tCC9dVBK~QW6XjSm79JPPBqf5czmxS4mMAhyMJWB07mUe6MgE1FXlx8hE68tm85IwPkRN1j0iTWvjfJdrGXlY4xaSUM7N~vA-GNFAxWoLM4EwefAXMrh4QKIZjdqfXqu7R19Objxv9BO5tPnyP9sfhF6tvq5FmR3N4hhSLs6Z4IkaDo3Tm-s9nOHE4MyFmCIRyIa9YGCvp2So6IWosBn9nQ68Osbe91IsK0BsBG7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">supplementary method</a></span> for promoter prediction for spacer-containing transcriptional regulatory elements in Arabidopsis. This method should be used alongside the octamer analysis that was established year ago<a reveal-id="dsw065-B3" data-open="dsw065-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> and is applicable to any other genomes that are thought to contain bipartite transcriptional regulatory elements.</p>                            <h2 scrollto-destination=88473570 id="88473570" class="section-title" >Conflict of interest</h2>
<p>None declared.</p>                            <h2 scrollto-destination=88473572 id="88473572" class="section-title" >Supplementary data</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/dsw065_Supp.xls?Expires=2147483647&Signature=F8Y4fwRwNGJil80iUVJPFsxswa910fM37~tNrinOWpfOAS3mOZ0zu1xYySI0Y5xPGZ4JefOBNzynYqecE-cNSs6u1fr4jY5w5lsi-1ivzNls9tCC9dVBK~QW6XjSm79JPPBqf5czmxS4mMAhyMJWB07mUe6MgE1FXlx8hE68tm85IwPkRN1j0iTWvjfJdrGXlY4xaSUM7N~vA-GNFAxWoLM4EwefAXMrh4QKIZjdqfXqu7R19Objxv9BO5tPnyP9sfhF6tvq5FmR3N4hhSLs6Z4IkaDo3Tm-s9nOHE4MyFmCIRyIa9YGCvp2So6IWosBn9nQ68Osbe91IsK0BsBG7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S1–S3</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=88473574 id="88473574" class="section-title" >Funding</h2>
<p>This work is supported in part by JST ALCA (to Y.Y.Y.) and Grant-in-Aid for Scientific Research on Innovative Areas ‘Plant Perception’ (No. 23120511) (to Y.Y.Y.).</p>                            <h2 scrollto-destination=88473576 id="88473576" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw065-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw065-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lawrence</div> <div class="given-names">C. E.</div></div>, <div class="name string-name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>, <div class="name string-name"><div class="surname">Boguski</div> <div class="given-names">M. S.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">J. S.</div></div>, <div class="name string-name"><div class="surname">Neuwald</div> <div class="given-names">A. F.</div></div>, <div class="name string-name"><div class="surname">Wootton</div> <div class="given-names">J. C.</div></div></div> <div class="year">1993</div>, 
<div class="article-title">Detecting subtle sequence signals: a Gibbs sampling strategy for multiple alignment</div>. <div class="source">Science</div> , <div class="volume">262</div>, <div class="fpage">208</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detecting%20subtle%20sequence%20signals%3A%20a%20Gibbs%20sampling%20strategy%20for%20multiple%20alignment&amp;author=C.%20E.%20Lawrence&amp;author=S.%20F.%20Altschul&amp;author=M.%20S.%20Boguski&amp;author=J.%20S.%20Liu&amp;author=A.%20F.%20Neuwald&amp;author=J.%20C.%20Wootton&amp;publication_year=1993&amp;journal=Science&amp;volume=262&amp;pages=208-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.8211139" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.8211139" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.8211139"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8211139" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw065-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw065-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bailey</div> <div class="given-names">T. L.</div></div>, <div class="name string-name"><div class="surname">Elkan</div> <div class="given-names">C.</div></div></div> <div class="year">1995</div>, 
<div class="article-title">The value of prior knowledge in discovering motifs with MEME</div>. <div class="source">Proc. Int. Conf. Intell. Syst. Mol. Biol</div> ., <div class="volume">3</div>, <div class="fpage">21</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20value%20of%20prior%20knowledge%20in%20discovering%20motifs%20with%20MEME&amp;author=T.%20L.%20Bailey&amp;author=C.%20Elkan&amp;publication_year=1995&amp;journal=Proc.%20Int.%20Conf.%20Intell.%20Syst.%20Mol.%20Biol&amp;volume=3&amp;pages=21-9" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7584439" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw065-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw065-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yamamoto</div> <div class="given-names">Y. Y.</div></div>, <div class="name string-name"><div class="surname">Yoshioka</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Hyakumachi</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Prediction of transcriptional regulatory elements for plant hormone responses based on microarray data</div>. <div class="source">BMC Plant Biol</div> ., <div class="volume">11</div>, <div class="fpage">39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20transcriptional%20regulatory%20elements%20for%20plant%20hormone%20responses%20based%20on%20microarray%20data&amp;author=Y.%20Y.%20Yamamoto&amp;author=Y.%20Yoshioka&amp;author=M.%20Hyakumachi&amp;publication_year=2011&amp;journal=BMC%20Plant%20Biol&amp;volume=11&amp;pages=39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-11-39" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-11-39" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-11-39"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21349196" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw065-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw065-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Matsuo</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Johnson</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Hieno</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">High REDOX RESPONSIVE TRANSCRIPTION FACTOR1 levels result in accumulation of reactive oxygen species in Arabidopsis thaliana shoots and roots</div>. <div class="source">Mol. Plant</div> , <div class="volume">8</div>, <div class="fpage">1253</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20REDOX%20RESPONSIVE%20TRANSCRIPTION%20FACTOR1%20levels%20result%20in%20accumulation%20of%20reactive%20oxygen%20species%20in%20Arabidopsis%20thaliana%20shoots%20and%20roots&amp;author=M.%20Matsuo&amp;author=J.%20M.%20Johnson&amp;author=A.%20Hieno&amp;publication_year=2015&amp;journal=Mol.%20Plant&amp;volume=8&amp;pages=1253-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molp.2015.03.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molp.2015.03.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molp.2015.03.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25882345" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw065-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw065-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tokizawa</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kobayashi</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Saito</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">SENSITIVE TO PROTON RHIZOTOXICITY1, CALMODULIN BINDING TRANSCRIPTION ACTIVATOR2, and other transcription factors are involved in ALUMINUM-ACTIVATED MALATE TRANSPORTER1 expression</div>. <div class="source">Plant Physiol</div> ., <div class="volume">167</div>, <div class="fpage">991</div>–<div class="lpage">1003</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SENSITIVE%20TO%20PROTON%20RHIZOTOXICITY1%2C%20CALMODULIN%20BINDING%20TRANSCRIPTION%20ACTIVATOR2%2C%20and%20other%20transcription%20factors%20are%20involved%20in%20ALUMINUM-ACTIVATED%20MALATE%20TRANSPORTER1%20expression&amp;author=M.%20Tokizawa&amp;author=Y.%20Kobayashi&amp;author=T.%20Saito&amp;publication_year=2015&amp;journal=Plant%20Physiol&amp;volume=167&amp;pages=991-1003" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.114.256552" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.114.256552" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.114.256552"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25627216" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw065-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw065-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hayami</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Sakai</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kimura</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">The responses of Arabidopsis ELIP2 to UV-B, high light, and cold stress are regulated by a transcriptional regulatory unit composed of two elements</div>. <div class="source">Plant Physiol</div> ., <div class="volume">169</div>, <div class="fpage">840</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20responses%20of%20Arabidopsis%20ELIP2%20to%20UV-B%2C%20high%20light%2C%20and%20cold%20stress%20are%20regulated%20by%20a%20transcriptional%20regulatory%20unit%20composed%20of%20two%20elements&amp;author=N.%20Hayami&amp;author=Y.%20Sakai&amp;author=M.%20Kimura&amp;publication_year=2015&amp;journal=Plant%20Physiol&amp;volume=169&amp;pages=840-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.15.00398" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.15.00398" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.15.00398"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26175515" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw065-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw065-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mathelier</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Fornes</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Arenillas</div> <div class="given-names">D. J.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">JASPAR 2016: a major expansion and update of the open-access database of transcription factor binding profiles</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">44</div>, <div class="fpage">D110</div>–<div class="lpage">115</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=JASPAR%202016%3A%20a%20major%20expansion%20and%20update%20of%20the%20open-access%20database%20of%20transcription%20factor%20binding%20profiles&amp;author=A.%20Mathelier&amp;author=O.%20Fornes&amp;author=D.%20J.%20Arenillas&amp;publication_year=2016&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;pages=D110-115" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv1176" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv1176" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv1176"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26531826" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw065-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw065-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mitsuda</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Ohme-Takagi</div> <div class="given-names">M.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Functional analysis of transcription factors in Arabidopsis</div>. <div class="source">Plant Cell Physiol</div> ., <div class="volume">50</div>, <div class="fpage">1232</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20analysis%20of%20transcription%20factors%20in%20Arabidopsis&amp;author=N.%20Mitsuda&amp;author=M.%20Ohme-Takagi&amp;publication_year=2009&amp;journal=Plant%20Cell%20Physiol&amp;volume=50&amp;pages=1232-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcp075" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcp075" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcp075"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19478073" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw065-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw065-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hieno</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Naznin</div> <div class="given-names">H. A.</div></div>, <div class="name string-name"><div class="surname">Hyakumachi</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">ppdb: plant promoter database version 3.0</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">D1188</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ppdb%3A%20plant%20promoter%20database%20version%203.0&amp;author=A.%20Hieno&amp;author=H.%20A.%20Naznin&amp;author=M.%20Hyakumachi&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=D1188-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24194597" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw065-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw065-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yamamoto</div> <div class="given-names">Y. Y.</div></div>, <div class="name string-name"><div class="surname">Ichida</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Matsui</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">Identification of plant promoter constituents by analysis of local distribution of short sequences</div>. <div class="source">BMC Genomics</div> , <div class="volume">8</div>, <div class="fpage">67</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20plant%20promoter%20constituents%20by%20analysis%20of%20local%20distribution%20of%20short%20sequences&amp;author=Y.%20Y.%20Yamamoto&amp;author=H.%20Ichida&amp;author=M.%20Matsui&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-67" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-67" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-67"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17346352" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw065-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw065-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Eisen</div> <div class="given-names">M. B.</div></div>, <div class="name string-name"><div class="surname">Spellman</div> <div class="given-names">P. T.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">P. O.</div></div>, <div class="name string-name"><div class="surname">Botstein</div> <div class="given-names">D.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Cluster analysis and display of genome-wide expression patterns</div>. <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">95</div>, <div class="fpage">14863</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cluster%20analysis%20and%20display%20of%20genome-wide%20expression%20patterns&amp;author=M.%20B.%20Eisen&amp;author=P.%20T.%20Spellman&amp;author=P.%20O.%20Brown&amp;author=D.%20Botstein&amp;publication_year=1998&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=95&amp;pages=14863-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.25.14863" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.25.14863" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.25.14863"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9843981" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw065-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw065-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Crooks</div> <div class="given-names">G. E.</div></div>, <div class="name string-name"><div class="surname">Hon</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Chandonia</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Brenner</div> <div class="given-names">S. E.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">WebLogo: a sequence logo generator</div>. <div class="source">Genome Res</div> ., <div class="volume">14</div>, <div class="fpage">1188</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=WebLogo%3A%20a%20sequence%20logo%20generator&amp;author=G.%20E.%20Crooks&amp;author=G.%20Hon&amp;author=J.%20M.%20Chandonia&amp;author=S.%20E.%20Brenner&amp;publication_year=2004&amp;journal=Genome%20Res&amp;volume=14&amp;pages=1188-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.849004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.849004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.849004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15173120" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw065-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw065-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nagashima</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Mishiba</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Suzuki</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Shimada</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Iwata</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Koizumi</div> <div class="given-names">N.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Arabidopsis IRE1 catalyses unconventional splicing of bZIP60 mRNA to produce the active transcription factor</div>. <div class="source">Sci. Rep</div> ., <div class="volume">1</div>, <div class="fpage">29</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arabidopsis%20IRE1%20catalyses%20unconventional%20splicing%20of%20bZIP60%20mRNA%20to%20produce%20the%20active%20transcription%20factor&amp;author=Y.%20Nagashima&amp;author=K.%20Mishiba&amp;author=E.%20Suzuki&amp;author=Y.%20Shimada&amp;author=Y.%20Iwata&amp;author=N.%20Koizumi&amp;publication_year=2011&amp;journal=Sci.%20Rep&amp;volume=1&amp;pages=29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep00029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep00029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep00029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22355548" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw065-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw065-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ng</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ivanova</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Duncan</div> <div class="given-names">O.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">A membrane-bound NAC transcription factor, ANAC017, mediates mitochondrial retrograde signaling in Arabidopsis</div>. <div class="source">Plant Cell</div> , <div class="volume">25</div>, <div class="fpage">3450</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20membrane-bound%20NAC%20transcription%20factor%2C%20ANAC017%2C%20mediates%20mitochondrial%20retrograde%20signaling%20in%20Arabidopsis&amp;author=S.%20Ng&amp;author=A.%20Ivanova&amp;author=O.%20Duncan&amp;publication_year=2013&amp;journal=Plant%20Cell&amp;volume=25&amp;pages=3450-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.113.113985" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.113.113985" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.113.113985"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24045017" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw065-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw065-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hieno</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Naznin</div> <div class="given-names">H. A.</div></div>, <div class="name string-name"><div class="surname">Sawaki</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Analysis of environmental stress in plants with the aid of marker genes for H<sub>2</sub>O<sub>2</sub> responses</div>. <div class="source">Methods Enzymol</div> ., <div class="volume">527</div>, <div class="fpage">221</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20environmental%20stress%20in%20plants%20with%20the%20aid%20of%20marker%20genes%20for%20H2O2%20responses&amp;author=A.%20Hieno&amp;author=H.%20A.%20Naznin&amp;author=K.%20Sawaki&amp;publication_year=2013&amp;journal=Methods%20Enzymol&amp;volume=527&amp;pages=221-37" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23830634" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw065-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw065-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Iwata</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Koizumi</div> <div class="given-names">N.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Plant transducers of the endoplasmic reticulum unfolded protein response</div>. <div class="source">Trends Plant Sci</div> ., <div class="volume">17</div>, <div class="fpage">720</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20transducers%20of%20the%20endoplasmic%20reticulum%20unfolded%20protein%20response&amp;author=Y.%20Iwata&amp;author=N.%20Koizumi&amp;publication_year=2012&amp;journal=Trends%20Plant%20Sci&amp;volume=17&amp;pages=720-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2012.06.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2012.06.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2012.06.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22796463" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw065-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw065-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yoshida</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Okada</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Haze</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2001</div>, 
<div class="article-title">Endoplasmic reticulum stress-induced formation of transcription factor complex ERSF including NF-Y (CBF) and activating transcription factors 6alpha and 6beta that activates the mammalian unfolded protein response</div>. <div class="source">Mol. Cell. Biol</div> ., <div class="volume">21</div>, <div class="fpage">1239</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Endoplasmic%20reticulum%20stress-induced%20formation%20of%20transcription%20factor%20complex%20ERSF%20including%20NF-Y%20(CBF)%20and%20activating%20transcription%20factors%206alpha%20and%206beta%20that%20activates%20the%20mammalian%20unfolded%20protein%20response&amp;author=H.%20Yoshida&amp;author=T.%20Okada&amp;author=K.%20Haze&amp;publication_year=2001&amp;journal=Mol.%20Cell.%20Biol&amp;volume=21&amp;pages=1239-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.21.4.1239-1248.2001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.21.4.1239-1248.2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.21.4.1239-1248.2001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11158310" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw065-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw065-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">J. X.</div></div>, <div class="name string-name"><div class="surname">Howell</div> <div class="given-names">S. H.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">bZIP28 and NF-Y transcription factors are activated by ER stress and assemble into a transcriptional complex to regulate stress response genes in Arabidopsis</div>. <div class="source">Plant Cell</div> , <div class="volume">22</div>, <div class="fpage">782</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=bZIP28%20and%20NF-Y%20transcription%20factors%20are%20activated%20by%20ER%20stress%20and%20assemble%20into%20a%20transcriptional%20complex%20to%20regulate%20stress%20response%20genes%20in%20Arabidopsis&amp;author=J.%20X.%20Liu&amp;author=S.%20H.%20Howell&amp;publication_year=2010&amp;journal=Plant%20Cell&amp;volume=22&amp;pages=782-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.109.072173" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.109.072173" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.109.072173"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20207753" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw065-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw065-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Martinez</div> <div class="given-names">I. M.</div></div>, <div class="name string-name"><div class="surname">Chrispeels</div> <div class="given-names">M. J.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Genomic analysis of the unfolded protein response in Arabidopsis shows its connection to important cellular processes</div>. <div class="source">Plant Cell</div> , <div class="volume">15</div>, <div class="fpage">561</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20analysis%20of%20the%20unfolded%20protein%20response%20in%20Arabidopsis%20shows%20its%20connection%20to%20important%20cellular%20processes&amp;author=I.%20M.%20Martinez&amp;author=M.%20J.%20Chrispeels&amp;publication_year=2003&amp;journal=Plant%20Cell&amp;volume=15&amp;pages=561-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.007609" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.007609" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.007609"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12566592" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw065-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw065-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">De Clercq</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Vermeirssen</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Van Aken</div> <div class="given-names">O.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">The membrane-bound NAC transcription factor ANAC013 functions in mitochondrial retrograde regulation of the oxidative stress response in Arabidopsis</div>. <div class="source">Plant Cell</div> , <div class="volume">25</div>, <div class="fpage">3472</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20membrane-bound%20NAC%20transcription%20factor%20ANAC013%20functions%20in%20mitochondrial%20retrograde%20regulation%20of%20the%20oxidative%20stress%20response%20in%20Arabidopsis&amp;author=I.%20De%20Clercq&amp;author=V.%20Vermeirssen&amp;author=O.%20Van%20Aken&amp;publication_year=2013&amp;journal=Plant%20Cell&amp;volume=25&amp;pages=3472-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.113.117168" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.113.117168" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.113.117168"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24045019" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw065-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw065-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Noh</div> <div class="given-names">S. J.</div></div>, <div class="name string-name"><div class="surname">Kwon</div> <div class="given-names">C. S.</div></div>, <div class="name string-name"><div class="surname">Oh</div> <div class="given-names">D. H.</div></div>, <div class="name string-name"><div class="surname">Moon</div> <div class="given-names">J. S.</div></div>, <div class="name string-name"><div class="surname">Chung</div> <div class="given-names">W. I.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Expression of an evolutionarily distinct novel BiP gene during the unfolded protein response in Arabidopsis thaliana</div>. <div class="source">Gene</div> , <div class="volume">311</div>, <div class="fpage">81</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20of%20an%20evolutionarily%20distinct%20novel%20BiP%20gene%20during%20the%20unfolded%20protein%20response%20in%20Arabidopsis%20thaliana&amp;author=S.%20J.%20Noh&amp;author=C.%20S.%20Kwon&amp;author=D.%20H.%20Oh&amp;author=J.%20S.%20Moon&amp;author=W.%20I.%20Chung&amp;publication_year=2003&amp;journal=Gene&amp;volume=311&amp;pages=81-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(03)00559-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(03)00559-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(03)00559-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12853141" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw065-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw065-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Okumura</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Makiguchi</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Makita</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Yamashita</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Nakai</div> <div class="given-names">K.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Melina II: a web tool for comparisons among several predictive algorithms to find potential motifs from promoter regions</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">35</div>, <div class="fpage">W227</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Melina%20II%3A%20a%20web%20tool%20for%20comparisons%20among%20several%20predictive%20algorithms%20to%20find%20potential%20motifs%20from%20promoter%20regions&amp;author=T.%20Okumura&amp;author=H.%20Makiguchi&amp;author=Y.%20Makita&amp;author=R.%20Yamashita&amp;author=K.%20Nakai&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=W227-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm362" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm362" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm362"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17537821" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2965793&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw065/1/dsw065_Supp.xls?Expires=2147483647&amp;Signature=F8Y4fwRwNGJil80iUVJPFsxswa910fM37~tNrinOWpfOAS3mOZ0zu1xYySI0Y5xPGZ4JefOBNzynYqecE-cNSs6u1fr4jY5w5lsi-1ivzNls9tCC9dVBK~QW6XjSm79JPPBqf5czmxS4mMAhyMJWB07mUe6MgE1FXlx8hE68tm85IwPkRN1j0iTWvjfJdrGXlY4xaSUM7N~vA-GNFAxWoLM4EwefAXMrh4QKIZjdqfXqu7R19Objxv9BO5tPnyP9sfhF6tvq5FmR3N4hhSLs6Z4IkaDo3Tm-s9nOHE4MyFmCIRyIa9YGCvp2So6IWosBn9nQ68Osbe91IsK0BsBG7Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">544</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw065"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2965793"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f3%2f271%2f2965793">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F3%2F271%2F2965793" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2965793'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Prediction of bipartite transcriptional regulatory elements using transcriptome data of Arabidopsis" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw065");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f271%2f2965793">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

