

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/3/307/4831046","@type":"ScholarlyArticle","name":"The genomes of two  Eutrema  species provide insight into plant adaptation to high altitudes","datePublished":"2018-01-30","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2018-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy003","keywords":["<italic>Eutrema</italic>","high-altitude adaptation","<italic>de novo</italic> assembly","comparative genomics"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/3/307/4831046","author":[{"name":"Guo, Xinyi","affiliation":"Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University, Chengdu 610065, PR China","@type":"Person"},{"name":"Hu, Quanjun","affiliation":"Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University, Chengdu 610065, PR China","@type":"Person"},{"name":"Hao, Guoqian","affiliation":"Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University, Chengdu 610065, PR China  Management Committee for Emei Mountain Scenic Area, Biodiversity Institute of Emei Mountain, Leshan 614200, Sichuan, PR China","@type":"Person"},{"name":"Wang, Xiaojuan","affiliation":"Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University, Chengdu 610065, PR China","@type":"Person"},{"name":"Zhang, Dan","affiliation":"Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University, Chengdu 610065, PR China","@type":"Person"},{"name":"Ma, Tao","affiliation":"Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University, Chengdu 610065, PR China","@type":"Person","sameAs":"http://orcid.org/0000-0002-7094-6868"},{"name":"Liu, Jianquan","affiliation":"Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University, Chengdu 610065, PR China","@type":"Person","sameAs":"http://orcid.org/0000-0002-4237-7418"}],"description":"Abstract.  Eutrema is a genus in the Brassicaceae, which includes species of scientific and economic importance. Many Eutrema species are montane and/or alpine ","pageStart":"307","pageEnd":"315","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/3/1/m_dnares_25_3cover.png?Expires=2147483647&Signature=zOskx939n215x6LsaNaPxmag9A6Tt6e7bP~Kt5lfxwaH8FjX1dKLUJ0mdj-Sm0uq9hmCpw3OAnAQFLS-GTY4CsXXX4aRhm5J1IbPv51CcTKXj828y94I4tx-BuFB-EA4WwEvEot4P8hDcnFzu~xWV52k2Sm6IHcCABqAv5wSWve6oMrdsQXhpN0srJqhN5UIcLjzws~g9qv24XhgtRb~Nw7X9IiEo1EM5V47od3bkv5yHlkUT1xmh3tJ4-ZjHw-HyAXqIp8eeHds4Cfjy9xqh8e3n5ynt516a1ZMF0hjA3wYH5X0-y3lqTHnWmgUvQv1L6dLXdRWhki~KqxTetYsJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"The genomes of two  Eutrema  species provide insight into plant adaptation to high altitudes","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/3/1/m_dnares_25_3cover.png?Expires=2147483647&Signature=zOskx939n215x6LsaNaPxmag9A6Tt6e7bP~Kt5lfxwaH8FjX1dKLUJ0mdj-Sm0uq9hmCpw3OAnAQFLS-GTY4CsXXX4aRhm5J1IbPv51CcTKXj828y94I4tx-BuFB-EA4WwEvEot4P8hDcnFzu~xWV52k2Sm6IHcCABqAv5wSWve6oMrdsQXhpN0srJqhN5UIcLjzws~g9qv24XhgtRb~Nw7X9IiEo1EM5V47od3bkv5yHlkUT1xmh3tJ4-ZjHw-HyAXqIp8eeHds4Cfjy9xqh8e3n5ynt516a1ZMF0hjA3wYH5X0-y3lqTHnWmgUvQv1L6dLXdRWhki~KqxTetYsJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="genomes of two  Eutrema  species provide insight into plant adaptation to high altitudes" />
    <meta property="og:description" content="Abstract.  Eutrema is a genus in the Brassicaceae, which includes species of scientific and economic importance. Many Eutrema species are montane and/or alpine " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/3/307/4831046" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2018-01-30" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Guo, Xinyi" /><meta name="citation_author_institution" content="Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University, Chengdu 610065, PR China" /><meta name="citation_author" content="Hu, Quanjun" /><meta name="citation_author_institution" content="Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University, Chengdu 610065, PR China" /><meta name="citation_author" content="Hao, Guoqian" /><meta name="citation_author_institution" content="Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University, Chengdu 610065, PR China" /><meta name="citation_author_institution" content="Management Committee for Emei Mountain Scenic Area, Biodiversity Institute of Emei Mountain, Leshan 614200, Sichuan, PR China" /><meta name="citation_author" content="Wang, Xiaojuan" /><meta name="citation_author_institution" content="Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University, Chengdu 610065, PR China" /><meta name="citation_author" content="Zhang, Dan" /><meta name="citation_author_institution" content="Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University, Chengdu 610065, PR China" /><meta name="citation_author" content="Ma, Tao" /><meta name="citation_author_institution" content="Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University, Chengdu 610065, PR China" /><meta name="citation_author" content="Liu, Jianquan" /><meta name="citation_author_institution" content="Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University, Chengdu 610065, PR China" /><meta name="citation_title" content="The genomes of two Eutrema species provide insight into plant adaptation to high altitudes" /><meta name="citation_firstpage" content="307" /><meta name="citation_lastpage" content="315" /><meta name="citation_doi" content="10.1093/dnares/dsy003" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2018/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The ubiquity of alpine plant radiations: from the Andes to the Hengduan Mountains;  citation_journal_title=New Phytol;  citation_year=2015;  citation_volume=207;  citation_pages=275-82; " /><meta name="citation_reference" content="citation_title=Adaptation to high altitude: phenotypes and genotypes;  citation_journal_title=Annu. Rev. Anthropol;  citation_year=2014;  citation_volume=43;  citation_pages=251-72; " /><meta name="citation_reference" content="citation_title=Genome of plant maca (Lepidium meyenii) illuminates genomic basis for high-altitude adaptation in the central Andes;  citation_journal_title=Mol. Plant;  citation_year=2016;  citation_volume=9;  citation_pages=1066-77; " /><meta name="citation_reference" content="citation_title=Toward a global phylogeny of the Brassicaceae;  citation_journal_title=Mol. Biol. Evol;  citation_year=2006;  citation_volume=23;  citation_pages=2142-60; " /><meta name="citation_reference" content="citation_title=Supernetwork identifies multiple events of plastid trn F (GAA) pseudogene evolution in the Brassicaceae;  citation_journal_title=Mol. Biol. Evol;  citation_year=2006;  citation_volume=24;  citation_pages=63-73; " /><meta name="citation_reference" content="citation_title=Beyond the thale: comparative genomics and genetics of Arabidopsis relatives;  citation_journal_title=Nat. Rev. Genet;  citation_year=2015;  citation_volume=16;  citation_pages=285" /><meta name="citation_reference" content="citation_title=Using mustard genomes to explore the genetic basis of evolutionary change;  citation_journal_title=Curr. Opin. Plant Biol;  citation_year=2017;  citation_volume=36;  citation_pages=119-28; " /><meta name="citation_reference" content="citation_title=The Cardamine hirsuta genome offers insight into the evolution of morphological diversity;  citation_journal_title=Nat. Plants;  citation_year=2016;  citation_volume=2;  citation_pages=16167" /><meta name="citation_reference" content="citation_title=Improving and correcting the contiguity of long-read genome assemblies of three plant species using optical mapping and chromosome conformation capture data;  citation_journal_title=Genome Res;  citation_year=2017;  citation_volume=27;  citation_pages=778-86; " /><meta name="citation_reference" content="citation_title=Genome expansion of Arabis alpina linked with retrotransposition and reduced symmetric DNA methylation;  citation_journal_title=Nat. Plants;  citation_year=2015;  citation_volume=1; " /><meta name="citation_reference" content="citation_title=A synopsis of Eutrema (Brassicaceae);  citation_journal_title=Harvard Pap. Bot;  citation_year=2005;  citation_volume=10;  citation_pages=129-35; " /><meta name="citation_reference" content="citation_title=An integrative study of evolutionary diversification of Eutrema (Eutremeae, Brassicaceae);  citation_journal_title=Bot. J. Linn. Soc;  citation_year=2017;  citation_volume=184;  citation_pages=204-23; " /><meta name="citation_reference" content="citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=2114-20; " /><meta name="citation_reference" content="citation_title=BFC: correcting Illumina sequencing errors;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=2885-7; " /><meta name="citation_reference" content="citation_title=FastUniq: a fast de novo duplicates removal tool for paired short reads;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e52249" /><meta name="citation_reference" content="citation_title=SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler;  citation_journal_title=Gigascience;  citation_year=2012;  citation_volume=1;  citation_pages=18" /><meta name="citation_reference" content="citation_title=Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads;  citation_journal_title=Genome Res;  citation_year=2014;  citation_volume=24;  citation_pages=1384-95; " /><meta name="citation_reference" content="citation_title=CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=1061-7; " /><meta name="citation_reference" content="citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=3210-2; " /><meta name="citation_reference" content="citation_title=Considering transposable element diversification in de novo annotation approaches;  citation_journal_title=PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e16526" /><meta name="citation_reference" content="citation_title=Combined evidence annotation of transposable elements in genome sequences;  citation_journal_title=PLoS Comput. Biol;  citation_year=2005;  citation_volume=1;  citation_pages=166-75; " /><meta name="citation_reference" content="citation_title=LTRharvest, a efficient and flexible software for de novo detection of LTR retrotransposons;  citation_journal_title=BMC Bioinformatics;  citation_year=2008;  citation_volume=9;  citation_pages=18" /><meta name="citation_reference" content="citation_title=Automated de novo identification of repeat sequence families in sequenced genomes;  citation_journal_title=Genome Res;  citation_year=2002;  citation_volume=12;  citation_pages=1269-76; " /><meta name="citation_reference" content="citation_title=PILER: identification and classification of genomic repeats;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21(suppl_1);  citation_pages=i152-8; " /><meta name="citation_reference" content="citation_title=On global sequence alignment;  citation_journal_title=Comput. Appl. Biosci;  citation_year=1994;  citation_volume=10;  citation_pages=227-35; " /><meta name="citation_reference" content="citation_title=Repbase Update, a database of repetitive elements in eukaryotic genomes;  citation_journal_title=Mob. DNA;  citation_year=2015;  citation_volume=6;  citation_pages=11" /><meta name="citation_reference" content="citation_title=The Pfam protein families database: towards a more sustainable future;  citation_journal_title=Nucleic Acids Res;  citation_year=2016;  citation_volume=44;  citation_pages=D279-85; " /><meta name="citation_reference" content="citation_title=A probabilistic model of local sequence alignment that simplifies statistical significance estimation;  citation_journal_title=PLoS Comp. Biol;  citation_year=2008;  citation_volume=4;  citation_pages=e1000069" /><meta name="citation_reference" content="citation_title=Ancestral repeats have shaped epigenome and genome composition for millions of years in Arabidopsis thaliana;  citation_journal_title=Nat. Commun;  citation_year=2014;  citation_volume=5;  citation_pages=4104" /><meta name="citation_reference" content="citation_title=Automated eukaryotic gene structure annotation using EVidenceModeler and the Program to Assemble Spliced Alignments;  citation_journal_title=Genome Biol;  citation_year=2008;  citation_volume=9;  citation_pages=R7" /><meta name="citation_reference" content="citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome;  citation_journal_title=Nat. Biotechnol;  citation_year=2011;  citation_volume=29;  citation_pages=644-52; " /><meta name="citation_reference" content="citation_title=StringTie enables improved reconstruction of a transcriptome from RNA-seq reads;  citation_journal_title=Nat. Biotechnol;  citation_year=2015;  citation_volume=33;  citation_pages=290-5; " /><meta name="citation_reference" content="citation_title=Using native and syntenically mapped cDNA alignments to improve de novo gene finding;  citation_journal_title=Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=637-44; " /><meta name="citation_reference" content="citation_title=TigrScan and GlimmerHMM: two open source ab initio eukaryotic gene-finders;  citation_journal_title=Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=2878-9; " /><meta name="citation_reference" content="citation_title=The reference genome of the halophytic plant Eutrema salsugineum;  citation_journal_title=Front. Plant Sci;  citation_year=2013;  citation_volume=4;  citation_pages=1-14; " /><meta name="citation_reference" content="citation_title=UniRef clusters: a comprehensive and scalable alternative for improving sequence similarity searches;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=926-32; " /><meta name="citation_reference" content="citation_title=Benchmarking spliced alignment programs including Spaln2, an extended version of Spaln that incorporates additional species-specific features;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=e161" /><meta name="citation_reference" content="citation_title=BLAST+: architecture and applications;  citation_journal_title=BMC Bioinformatics;  citation_year=2009;  citation_volume=10;  citation_pages=421" /><meta name="citation_reference" content="citation_title=UniProt: a hub for protein information;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=D204-12; " /><meta name="citation_reference" content="citation_title=The Arabidopsis Information Resource (TAIR): improved gene annotation and new tools;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=D1202-10; " /><meta name="citation_reference" content="citation_title=InterProScan 5: genome-scale protein function classification;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=1236-40; " /><meta name="citation_reference" content="citation_title=The GOA database: gene Ontology annotation updates for 2015;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=D1057-63; " /><meta name="citation_reference" content="citation_title=OrthoMCL: identification of ortholog groups for eukaryotic genomes;  citation_journal_title=Genome Res;  citation_year=2003;  citation_volume=13;  citation_pages=2178-89; " /><meta name="citation_reference" content="citation_title=Estimating gene gain and loss rates in the presence of error in genome assembly and annotation using CAFE 3;  citation_journal_title=Mol. Biol. Evol;  citation_year=2013;  citation_volume=30;  citation_pages=1987-97; " /><meta name="citation_reference" content="citation_title=AgriGO v2.0: a GO analysis toolkit for the agricultural community, 2017 update;  citation_journal_title=Nucleic Acids Res;  citation_year=2017;  citation_volume=43;  citation_pages=W122-9; " /><meta name="citation_reference" content="citation_title=R: a language and environment for statistical computing; citation_author=R Core Team;  citation_journal_title=R Foundation for Statistical Computing;  citation_year=2016; " /><meta name="citation_reference" content="citation_title=RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=1312-3; " /><meta name="citation_reference" content="citation_title=Phylogeny-aware gap placement prevents errors in sequence alignment and evolutionary analysis;  citation_journal_title=Science;  citation_year=2008;  citation_volume=320;  citation_pages=1632-5; " /><meta name="citation_reference" content="citation_title=PAL2NAL: robust conversion of protein sequence alignments into the corresponding codon alignments;  citation_journal_title=Nucleic Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=W609-12; " /><meta name="citation_reference" content="citation_title=PAML 4: phylogenetic analysis by maximum likelihood;  citation_journal_title=Mol. Biol. Evol;  citation_year=2007;  citation_volume=24;  citation_pages=1586-91; " /><meta name="citation_reference" content="citation_title=Plastome phylogeny and early diversification of Brassicaceae;  citation_journal_title=BMC Genomics;  citation_year=2017;  citation_volume=176;  citation_pages=1-9; " /><meta name="citation_reference" content="citation_title=MCScanX: a toolkit for detection and evolutionary analysis of gene synteny and collinearity;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=e49" /><meta name="citation_reference" content="citation_title=Inference of human population history from individual whole-genome sequences;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=475;  citation_pages=493-6; " /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; " /><meta name="citation_reference" content="citation_title=A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2987-93; " /><meta name="citation_reference" content="citation_title=Comparative analysis of plant immune receptor architectures uncovers host proteins likely targeted by pathogens;  citation_journal_title=BMC Biol;  citation_year=2016;  citation_volume=14;  citation_pages=8" /><meta name="citation_reference" content="citation_title=Diversity of the superfamily of phloem lectins (phloem protein 2) in angiosperms;  citation_journal_title=Plant Physiol;  citation_year=2003;  citation_volume=131;  citation_pages=114-28; " /><meta name="citation_reference" content="citation_title=Mechanisms used by plants to cope with DNA damage;  citation_journal_title=Annu. Rev. Plant Biol;  citation_year=2016;  citation_volume=67;  citation_pages=439-62; " /><meta name="citation_reference" content="citation_title=Programmed induction of endoreduplication by DNA double-strand breaks in Arabidopsis;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2011;  citation_volume=108;  citation_pages=10004-9; " /><meta name="citation_reference" content="citation_title=SMC proteins and their multiple functions in higher plants;  citation_journal_title=Cytogenet. Genome Res;  citation_year=2009;  citation_volume=124;  citation_pages=202-14; " /><meta name="citation_reference" content="citation_title=Identification of ICE2, a gene involved in cold acclimation which determines freezing tolerance in Arabidopsis thaliana;  citation_journal_title=Gene;  citation_year=2009;  citation_volume=429;  citation_pages=98-103; " /><meta name="citation_reference" content="citation_title=Low-temperature damage and subsequent recovery of fab1 mutant Arabidopsis exposed to 2 [deg] C;  citation_journal_title=Plant Physiol;  citation_year=1997;  citation_volume=113;  citation_pages=347-56; " /><meta name="citation_reference" content="citation_title=Mutations in the prokaryotic pathway rescue the fatty acid biosynthesis1 mutant in the cold;  citation_journal_title=Plant Physiol;  citation_year=2015;  citation_volume=169;  citation_pages=442-52; " /><meta name="citation_reference" content="citation_title=Vernalization in Arabidopsis thaliana is mediated by the PHD finger protein VIN3;  citation_journal_title=Nature;  citation_year=2004;  citation_volume=427;  citation_pages=159-64; " /><meta name="citation_reference" content="citation_title=An atlas of over 90,000 conserved noncoding sequences provides insight into crucifer regulatory regions;  citation_journal_title=Nat. Genet;  citation_year=2013;  citation_volume=45;  citation_pages=891-8; " /><meta name="citation_reference" content="citation_title=Genomic and phenotypic differentiation of Arabidopsis thaliana along altitudinal gradients in the North Italian Alps;  citation_journal_title=Mol. Ecol;  citation_year=2016;  citation_volume=25;  citation_pages=3574-92; " /><meta name="citation_reference" content="citation_title=Convergent gene loss following gene and genome duplications creates single-copy families in flowering plants;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2013;  citation_volume=110;  citation_pages=2898-903; " /><meta name="citation_reference" content="citation_title=Identification and characterization of SA/Scc3p subunits in the Xenopus and human cohesin complexes;  citation_journal_title=J. Cell Biol;  citation_year=2000;  citation_volume=150;  citation_pages=405-16; " /><meta name="citation_reference" content="citation_title=Functional contribution of Pds5 to cohesin-mediated cohesion in human cells and Xenopus egg extracts;  citation_journal_title=J. Cell Sci;  citation_year=2005;  citation_volume=118;  citation_pages=2133-41; " /><meta name="citation_reference" content="citation_title=More to cohesin than meets the eye: complex diversity for fine-tuning of function;  citation_journal_title=Curr. Opin. Genet. Dev;  citation_year=2017;  citation_volume=43;  citation_pages=93-100; " /><meta name="citation_reference" content="citation_title=Characterization of vertebrate cohesin complexes and their regulation in prophase;  citation_journal_title=J. Cell Biol;  citation_year=2000;  citation_volume=151;  citation_pages=749-62; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/3/307/25072151/dsy003.pdf" /><meta name="description" content="Abstract.  Eutrema is a genus in the Brassicaceae, which includes species of scientific and economic importance. Many Eutrema species are montane and/or alpine " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/3/307/4831046" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/3/307/4831046" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/3/307/4831046");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsy003';
    </script>
<title>
	genomes of two Eutrema species provide insight into plant adaptation to high altitudes | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4831046" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeihATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR3VvLCBYaW55aSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEJpby1SZXNvdXJjZSBhbmQgRWNvLUVudmlyb25tZW50IG9mIE1pbmlzdHJ5IG9mIEVkdWNhdGlvbiwgQ29sbGVnZSBvZiBMaWZlIFNjaWVuY2VzLCBTaWNodWFuIFVuaXZlcnNpdHksIENoZW5nZHUgNjEwMDY1LCBQUiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikh1LCBRdWFuanVuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgQmlvLVJlc291cmNlIGFuZCBFY28tRW52aXJvbm1lbnQgb2YgTWluaXN0cnkgb2YgRWR1Y2F0aW9uLCBDb2xsZWdlIG9mIExpZmUgU2NpZW5jZXMsIFNpY2h1YW4gVW5pdmVyc2l0eSwgQ2hlbmdkdSA2MTAwNjUsIFBSIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGFvLCBHdW9xaWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgQmlvLVJlc291cmNlIGFuZCBFY28tRW52aXJvbm1lbnQgb2YgTWluaXN0cnkgb2YgRWR1Y2F0aW9uLCBDb2xsZWdlIG9mIExpZmUgU2NpZW5jZXMsIFNpY2h1YW4gVW5pdmVyc2l0eSwgQ2hlbmdkdSA2MTAwNjUsIFBSIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTWFuYWdlbWVudCBDb21taXR0ZWUgZm9yIEVtZWkgTW91bnRhaW4gU2NlbmljIEFyZWEsIEJpb2RpdmVyc2l0eSBJbnN0aXR1dGUgb2YgRW1laSBNb3VudGFpbiwgTGVzaGFuIDYxNDIwMCwgU2ljaHVhbiwgUFIgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXYW5nLCBYaWFvanVhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEJpby1SZXNvdXJjZSBhbmQgRWNvLUVudmlyb25tZW50IG9mIE1pbmlzdHJ5IG9mIEVkdWNhdGlvbiwgQ29sbGVnZSBvZiBMaWZlIFNjaWVuY2VzLCBTaWNodWFuIFVuaXZlcnNpdHksIENoZW5nZHUgNjEwMDY1LCBQUiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlpoYW5nLCBEYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLZXkgTGFib3JhdG9yeSBvZiBCaW8tUmVzb3VyY2UgYW5kIEVjby1FbnZpcm9ubWVudCBvZiBNaW5pc3RyeSBvZiBFZHVjYXRpb24sIENvbGxlZ2Ugb2YgTGlmZSBTY2llbmNlcywgU2ljaHVhbiBVbml2ZXJzaXR5LCBDaGVuZ2R1IDYxMDA2NSwgUFIgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYSwgVGFvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2V5IExhYm9yYXRvcnkgb2YgQmlvLVJlc291cmNlIGFuZCBFY28tRW52aXJvbm1lbnQgb2YgTWluaXN0cnkgb2YgRWR1Y2F0aW9uLCBDb2xsZWdlIG9mIExpZmUgU2NpZW5jZXMsIFNpY2h1YW4gVW5pdmVyc2l0eSwgQ2hlbmdkdSA2MTAwNjUsIFBSIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGl1LCBKaWFucXVhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktleSBMYWJvcmF0b3J5IG9mIEJpby1SZXNvdXJjZSBhbmQgRWNvLUVudmlyb25tZW50IG9mIE1pbmlzdHJ5IG9mIEVkdWNhdGlvbiwgQ29sbGVnZSBvZiBMaWZlIFNjaWVuY2VzLCBTaWNodWFuIFVuaXZlcnNpdHksIENoZW5nZHUgNjEwMDY1LCBQUiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iVGhlIGdlbm9tZXMgb2YgdHdvIEV1dHJlbWEgc3BlY2llcyBwcm92aWRlIGluc2lnaHQgaW50byBwbGFudCBhZGFwdGF0aW9uIHRvIGhpZ2ggYWx0aXR1ZGVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMzA3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIzMTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3kwMDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxOC8wNi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB1YmlxdWl0eSBvZiBhbHBpbmUgcGxhbnQgcmFkaWF0aW9uczogZnJvbSB0aGUgQW5kZXMgdG8gdGhlIEhlbmdkdWFuIE1vdW50YWluczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmV3IFBoeXRvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDc7ICBjaXRhdGlvbl9wYWdlcz0yNzUtODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFkYXB0YXRpb24gdG8gaGlnaCBhbHRpdHVkZTogcGhlbm90eXBlcyBhbmQgZ2Vub3R5cGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIEFudGhyb3BvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTI1MS03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIG9mIHBsYW50IG1hY2EgKExlcGlkaXVtIG1leWVuaWkpIGlsbHVtaW5hdGVzIGdlbm9taWMgYmFzaXMgZm9yIGhpZ2gtYWx0aXR1ZGUgYWRhcHRhdGlvbiBpbiB0aGUgY2VudHJhbCBBbmRlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBQbGFudDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MTA2Ni03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG93YXJkIGEgZ2xvYmFsIHBoeWxvZ2VueSBvZiB0aGUgQnJhc3NpY2FjZWFlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0yMTQyLTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdXBlcm5ldHdvcmsgaWRlbnRpZmllcyBtdWx0aXBsZSBldmVudHMgb2YgcGxhc3RpZCB0cm4gRiAoR0FBKSBwc2V1ZG9nZW5lIGV2b2x1dGlvbiBpbiB0aGUgQnJhc3NpY2FjZWFlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz02My03MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmV5b25kIHRoZSB0aGFsZTogY29tcGFyYXRpdmUgZ2Vub21pY3MgYW5kIGdlbmV0aWNzIG9mIEFyYWJpZG9wc2lzIHJlbGF0aXZlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9Mjg1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXNpbmcgbXVzdGFyZCBnZW5vbWVzIHRvIGV4cGxvcmUgdGhlIGdlbmV0aWMgYmFzaXMgb2YgZXZvbHV0aW9uYXJ5IGNoYW5nZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gUGxhbnQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPTExOS0yODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIENhcmRhbWluZSBoaXJzdXRhIGdlbm9tZSBvZmZlcnMgaW5zaWdodCBpbnRvIHRoZSBldm9sdXRpb24gb2YgbW9ycGhvbG9naWNhbCBkaXZlcnNpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUGxhbnRzOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0xNjE2NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcHJvdmluZyBhbmQgY29ycmVjdGluZyB0aGUgY29udGlndWl0eSBvZiBsb25nLXJlYWQgZ2Vub21lIGFzc2VtYmxpZXMgb2YgdGhyZWUgcGxhbnQgc3BlY2llcyB1c2luZyBvcHRpY2FsIG1hcHBpbmcgYW5kIGNocm9tb3NvbWUgY29uZm9ybWF0aW9uIGNhcHR1cmUgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTc3OC04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIGV4cGFuc2lvbiBvZiBBcmFiaXMgYWxwaW5hIGxpbmtlZCB3aXRoIHJldHJvdHJhbnNwb3NpdGlvbiBhbmQgcmVkdWNlZCBzeW1tZXRyaWMgRE5BIG1ldGh5bGF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFBsYW50czsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHN5bm9wc2lzIG9mIEV1dHJlbWEgKEJyYXNzaWNhY2VhZSk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUhhcnZhcmQgUGFwLiBCb3Q7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xMjktMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGludGVncmF0aXZlIHN0dWR5IG9mIGV2b2x1dGlvbmFyeSBkaXZlcnNpZmljYXRpb24gb2YgRXV0cmVtYSAoRXV0cmVtZWFlLCBCcmFzc2ljYWNlYWUpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Cb3QuIEouIExpbm4uIFNvYzsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT0xODQ7ICBjaXRhdGlvbl9wYWdlcz0yMDQtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyaW1tb21hdGljOiBhIGZsZXhpYmxlIHRyaW1tZXIgZm9yIElsbHVtaW5hIHNlcXVlbmNlIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MjExNC0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QkZDOiBjb3JyZWN0aW5nIElsbHVtaW5hIHNlcXVlbmNpbmcgZXJyb3JzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTI4ODUtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdFVuaXE6IGEgZmFzdCBkZSBub3ZvIGR1cGxpY2F0ZXMgcmVtb3ZhbCB0b29sIGZvciBwYWlyZWQgc2hvcnQgcmVhZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lNTIyNDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TT0FQZGVub3ZvMjogYW4gZW1waXJpY2FsbHkgaW1wcm92ZWQgbWVtb3J5LWVmZmljaWVudCBzaG9ydC1yZWFkIGRlIG5vdm8gYXNzZW1ibGVyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HaWdhc2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZmZpY2llbnQgZGUgbm92byBhc3NlbWJseSBvZiBoaWdobHkgaGV0ZXJvenlnb3VzIGdlbm9tZXMgZnJvbSB3aG9sZS1nZW5vbWUgc2hvdGd1biBzaG9ydCByZWFkczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTEzODQtOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNFR01BOiBhIHBpcGVsaW5lIHRvIGFjY3VyYXRlbHkgYW5ub3RhdGUgY29yZSBnZW5lcyBpbiBldWthcnlvdGljIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MTA2MS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CVVNDTzogYXNzZXNzaW5nIGdlbm9tZSBhc3NlbWJseSBhbmQgYW5ub3RhdGlvbiBjb21wbGV0ZW5lc3Mgd2l0aCBzaW5nbGUtY29weSBvcnRob2xvZ3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MzIxMC0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zaWRlcmluZyB0cmFuc3Bvc2FibGUgZWxlbWVudCBkaXZlcnNpZmljYXRpb24gaW4gZGUgbm92byBhbm5vdGF0aW9uIGFwcHJvYWNoZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1lMTY1MjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21iaW5lZCBldmlkZW5jZSBhbm5vdGF0aW9uIG9mIHRyYW5zcG9zYWJsZSBlbGVtZW50cyBpbiBnZW5vbWUgc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIENvbXB1dC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MTY2LTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MVFJoYXJ2ZXN0LCBhIGVmZmljaWVudCBhbmQgZmxleGlibGUgc29mdHdhcmUgZm9yIGRlIG5vdm8gZGV0ZWN0aW9uIG9mIExUUiByZXRyb3RyYW5zcG9zb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTE4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXV0b21hdGVkIGRlIG5vdm8gaWRlbnRpZmljYXRpb24gb2YgcmVwZWF0IHNlcXVlbmNlIGZhbWlsaWVzIGluIHNlcXVlbmNlZCBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTI2OS03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UElMRVI6IGlkZW50aWZpY2F0aW9uIGFuZCBjbGFzc2lmaWNhdGlvbiBvZiBnZW5vbWljIHJlcGVhdHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxKHN1cHBsXzEpOyAgY2l0YXRpb25fcGFnZXM9aTE1Mi04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbiBnbG9iYWwgc2VxdWVuY2UgYWxpZ25tZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Db21wdXQuIEFwcGwuIEJpb3NjaTsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTIyNy0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVwYmFzZSBVcGRhdGUsIGEgZGF0YWJhc2Ugb2YgcmVwZXRpdGl2ZSBlbGVtZW50cyBpbiBldWthcnlvdGljIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vYi4gRE5BOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0xMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBQZmFtIHByb3RlaW4gZmFtaWxpZXMgZGF0YWJhc2U6IHRvd2FyZHMgYSBtb3JlIHN1c3RhaW5hYmxlIGZ1dHVyZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz1EMjc5LTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHByb2JhYmlsaXN0aWMgbW9kZWwgb2YgbG9jYWwgc2VxdWVuY2UgYWxpZ25tZW50IHRoYXQgc2ltcGxpZmllcyBzdGF0aXN0aWNhbCBzaWduaWZpY2FuY2UgZXN0aW1hdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBDb21wLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMDA2OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuY2VzdHJhbCByZXBlYXRzIGhhdmUgc2hhcGVkIGVwaWdlbm9tZSBhbmQgZ2Vub21lIGNvbXBvc2l0aW9uIGZvciBtaWxsaW9ucyBvZiB5ZWFycyBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBDb21tdW47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTQxMDQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BdXRvbWF0ZWQgZXVrYXJ5b3RpYyBnZW5lIHN0cnVjdHVyZSBhbm5vdGF0aW9uIHVzaW5nIEVWaWRlbmNlTW9kZWxlciBhbmQgdGhlIFByb2dyYW0gdG8gQXNzZW1ibGUgU3BsaWNlZCBBbGlnbm1lbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9UjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdWxsLWxlbmd0aCB0cmFuc2NyaXB0b21lIGFzc2VtYmx5IGZyb20gUk5BLVNlcSBkYXRhIHdpdGhvdXQgYSByZWZlcmVuY2UgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz02NDQtNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cmluZ1RpZSBlbmFibGVzIGltcHJvdmVkIHJlY29uc3RydWN0aW9uIG9mIGEgdHJhbnNjcmlwdG9tZSBmcm9tIFJOQS1zZXEgcmVhZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTI5MC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vc2luZyBuYXRpdmUgYW5kIHN5bnRlbmljYWxseSBtYXBwZWQgY0ROQSBhbGlnbm1lbnRzIHRvIGltcHJvdmUgZGUgbm92byBnZW5lIGZpbmRpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9NjM3LTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaWdyU2NhbiBhbmQgR2xpbW1lckhNTTogdHdvIG9wZW4gc291cmNlIGFiIGluaXRpbyBldWthcnlvdGljIGdlbmUtZmluZGVyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0yODc4LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByZWZlcmVuY2UgZ2Vub21lIG9mIHRoZSBoYWxvcGh5dGljIHBsYW50IEV1dHJlbWEgc2Fsc3VnaW5ldW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZyb250LiBQbGFudCBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTEtMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVuaVJlZiBjbHVzdGVyczogYSBjb21wcmVoZW5zaXZlIGFuZCBzY2FsYWJsZSBhbHRlcm5hdGl2ZSBmb3IgaW1wcm92aW5nIHNlcXVlbmNlIHNpbWlsYXJpdHkgc2VhcmNoZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9OTI2LTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CZW5jaG1hcmtpbmcgc3BsaWNlZCBhbGlnbm1lbnQgcHJvZ3JhbXMgaW5jbHVkaW5nIFNwYWxuMiwgYW4gZXh0ZW5kZWQgdmVyc2lvbiBvZiBTcGFsbiB0aGF0IGluY29ycG9yYXRlcyBhZGRpdGlvbmFsIHNwZWNpZXMtc3BlY2lmaWMgZmVhdHVyZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9ZTE2MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJMQVNUKzogYXJjaGl0ZWN0dXJlIGFuZCBhcHBsaWNhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTQyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVuaVByb3Q6IGEgaHViIGZvciBwcm90ZWluIGluZm9ybWF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPUQyMDQtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBBcmFiaWRvcHNpcyBJbmZvcm1hdGlvbiBSZXNvdXJjZSAoVEFJUik6IGltcHJvdmVkIGdlbmUgYW5ub3RhdGlvbiBhbmQgbmV3IHRvb2xzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPUQxMjAyLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlclByb1NjYW4gNTogZ2Vub21lLXNjYWxlIHByb3RlaW4gZnVuY3Rpb24gY2xhc3NpZmljYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MTIzNi00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEdPQSBkYXRhYmFzZTogZ2VuZSBPbnRvbG9neSBhbm5vdGF0aW9uIHVwZGF0ZXMgZm9yIDIwMTU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9RDEwNTctNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9ydGhvTUNMOiBpZGVudGlmaWNhdGlvbiBvZiBvcnRob2xvZyBncm91cHMgZm9yIGV1a2FyeW90aWMgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTIxNzgtODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzdGltYXRpbmcgZ2VuZSBnYWluIGFuZCBsb3NzIHJhdGVzIGluIHRoZSBwcmVzZW5jZSBvZiBlcnJvciBpbiBnZW5vbWUgYXNzZW1ibHkgYW5kIGFubm90YXRpb24gdXNpbmcgQ0FGRSAzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xOTg3LTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZ3JpR08gdjIuMDogYSBHTyBhbmFseXNpcyB0b29sa2l0IGZvciB0aGUgYWdyaWN1bHR1cmFsIGNvbW11bml0eSwgMjAxNyB1cGRhdGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE3OyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9VzEyMi05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SOiBhIGxhbmd1YWdlIGFuZCBlbnZpcm9ubWVudCBmb3Igc3RhdGlzdGljYWwgY29tcHV0aW5nOyBjaXRhdGlvbl9hdXRob3I9UiBDb3JlIFRlYW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVIgRm91bmRhdGlvbiBmb3IgU3RhdGlzdGljYWwgQ29tcHV0aW5nOyAgY2l0YXRpb25feWVhcj0yMDE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SQXhNTCB2ZXJzaW9uIDg6IGEgdG9vbCBmb3IgcGh5bG9nZW5ldGljIGFuYWx5c2lzIGFuZCBwb3N0LWFuYWx5c2lzIG9mIGxhcmdlIHBoeWxvZ2VuaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTEzMTItMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW55LWF3YXJlIGdhcCBwbGFjZW1lbnQgcHJldmVudHMgZXJyb3JzIGluIHNlcXVlbmNlIGFsaWdubWVudCBhbmQgZXZvbHV0aW9uYXJ5IGFuYWx5c2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTMyMDsgIGNpdGF0aW9uX3BhZ2VzPTE2MzItNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UEFMMk5BTDogcm9idXN0IGNvbnZlcnNpb24gb2YgcHJvdGVpbiBzZXF1ZW5jZSBhbGlnbm1lbnRzIGludG8gdGhlIGNvcnJlc3BvbmRpbmcgY29kb24gYWxpZ25tZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz1XNjA5LTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QQU1MIDQ6IHBoeWxvZ2VuZXRpYyBhbmFseXNpcyBieSBtYXhpbXVtIGxpa2VsaWhvb2Q7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTE1ODYtOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYXN0b21lIHBoeWxvZ2VueSBhbmQgZWFybHkgZGl2ZXJzaWZpY2F0aW9uIG9mIEJyYXNzaWNhY2VhZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE3OyAgY2l0YXRpb25fdm9sdW1lPTE3NjsgIGNpdGF0aW9uX3BhZ2VzPTEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUNTY2FuWDogYSB0b29sa2l0IGZvciBkZXRlY3Rpb24gYW5kIGV2b2x1dGlvbmFyeSBhbmFseXNpcyBvZiBnZW5lIHN5bnRlbnkgYW5kIGNvbGxpbmVhcml0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDA7ICBjaXRhdGlvbl9wYWdlcz1lNDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmZlcmVuY2Ugb2YgaHVtYW4gcG9wdWxhdGlvbiBoaXN0b3J5IGZyb20gaW5kaXZpZHVhbCB3aG9sZS1nZW5vbWUgc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDc1OyAgY2l0YXRpb25fcGFnZXM9NDkzLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QgYW5kIGFjY3VyYXRlIHNob3J0IHJlYWQgYWxpZ25tZW50IHdpdGggQnVycm93cy1XaGVlbGVyIHRyYW5zZm9ybTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xNzU0LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHN0YXRpc3RpY2FsIGZyYW1ld29yayBmb3IgU05QIGNhbGxpbmcsIG11dGF0aW9uIGRpc2NvdmVyeSwgYXNzb2NpYXRpb24gbWFwcGluZyBhbmQgcG9wdWxhdGlvbiBnZW5ldGljYWwgcGFyYW1ldGVyIGVzdGltYXRpb24gZnJvbSBzZXF1ZW5jaW5nIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9Mjk4Ny05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgYW5hbHlzaXMgb2YgcGxhbnQgaW1tdW5lIHJlY2VwdG9yIGFyY2hpdGVjdHVyZXMgdW5jb3ZlcnMgaG9zdCBwcm90ZWlucyBsaWtlbHkgdGFyZ2V0ZWQgYnkgcGF0aG9nZW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXZlcnNpdHkgb2YgdGhlIHN1cGVyZmFtaWx5IG9mIHBobG9lbSBsZWN0aW5zIChwaGxvZW0gcHJvdGVpbiAyKSBpbiBhbmdpb3NwZXJtczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzE7ICBjaXRhdGlvbl9wYWdlcz0xMTQtMjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lY2hhbmlzbXMgdXNlZCBieSBwbGFudHMgdG8gY29wZSB3aXRoIEROQSBkYW1hZ2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubnUuIFJldi4gUGxhbnQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT02NzsgIGNpdGF0aW9uX3BhZ2VzPTQzOS02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvZ3JhbW1lZCBpbmR1Y3Rpb24gb2YgZW5kb3JlZHVwbGljYXRpb24gYnkgRE5BIGRvdWJsZS1zdHJhbmQgYnJlYWtzIGluIEFyYWJpZG9wc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDg7ICBjaXRhdGlvbl9wYWdlcz0xMDAwNC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TTUMgcHJvdGVpbnMgYW5kIHRoZWlyIG11bHRpcGxlIGZ1bmN0aW9ucyBpbiBoaWdoZXIgcGxhbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DeXRvZ2VuZXQuIEdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTI0OyAgY2l0YXRpb25fcGFnZXM9MjAyLTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBJQ0UyLCBhIGdlbmUgaW52b2x2ZWQgaW4gY29sZCBhY2NsaW1hdGlvbiB3aGljaCBkZXRlcm1pbmVzIGZyZWV6aW5nIHRvbGVyYW5jZSBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00Mjk7ICBjaXRhdGlvbl9wYWdlcz05OC0xMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvdy10ZW1wZXJhdHVyZSBkYW1hZ2UgYW5kIHN1YnNlcXVlbnQgcmVjb3Zlcnkgb2YgZmFiMSBtdXRhbnQgQXJhYmlkb3BzaXMgZXhwb3NlZCB0byAyIFtkZWddIEM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MTEzOyAgY2l0YXRpb25fcGFnZXM9MzQ3LTU2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdXRhdGlvbnMgaW4gdGhlIHByb2thcnlvdGljIHBhdGh3YXkgcmVzY3VlIHRoZSBmYXR0eSBhY2lkIGJpb3N5bnRoZXNpczEgbXV0YW50IGluIHRoZSBjb2xkOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OTsgIGNpdGF0aW9uX3BhZ2VzPTQ0Mi01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmVybmFsaXphdGlvbiBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYSBpcyBtZWRpYXRlZCBieSB0aGUgUEhEIGZpbmdlciBwcm90ZWluIFZJTjM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT00Mjc7ICBjaXRhdGlvbl9wYWdlcz0xNTktNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGF0bGFzIG9mIG92ZXIgOTAsMDAwIGNvbnNlcnZlZCBub25jb2Rpbmcgc2VxdWVuY2VzIHByb3ZpZGVzIGluc2lnaHQgaW50byBjcnVjaWZlciByZWd1bGF0b3J5IHJlZ2lvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz04OTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBhbmQgcGhlbm90eXBpYyBkaWZmZXJlbnRpYXRpb24gb2YgQXJhYmlkb3BzaXMgdGhhbGlhbmEgYWxvbmcgYWx0aXR1ZGluYWwgZ3JhZGllbnRzIGluIHRoZSBOb3J0aCBJdGFsaWFuIEFscHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gRWNvbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTM1NzQtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnZlcmdlbnQgZ2VuZSBsb3NzIGZvbGxvd2luZyBnZW5lIGFuZCBnZW5vbWUgZHVwbGljYXRpb25zIGNyZWF0ZXMgc2luZ2xlLWNvcHkgZmFtaWxpZXMgaW4gZmxvd2VyaW5nIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTEwOyAgY2l0YXRpb25fcGFnZXM9Mjg5OC05MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIFNBL1NjYzNwIHN1YnVuaXRzIGluIHRoZSBYZW5vcHVzIGFuZCBodW1hbiBjb2hlc2luIGNvbXBsZXhlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQ2VsbCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE1MDsgIGNpdGF0aW9uX3BhZ2VzPTQwNS0xNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb25hbCBjb250cmlidXRpb24gb2YgUGRzNSB0byBjb2hlc2luLW1lZGlhdGVkIGNvaGVzaW9uIGluIGh1bWFuIGNlbGxzIGFuZCBYZW5vcHVzIGVnZyBleHRyYWN0czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQ2VsbCBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTE4OyAgY2l0YXRpb25fcGFnZXM9MjEzMy00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9yZSB0byBjb2hlc2luIHRoYW4gbWVldHMgdGhlIGV5ZTogY29tcGxleCBkaXZlcnNpdHkgZm9yIGZpbmUtdHVuaW5nIG9mIGZ1bmN0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBHZW5ldC4gRGV2OyAgY2l0YXRpb25feWVhcj0yMDE3OyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9OTMtMTAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIHZlcnRlYnJhdGUgY29oZXNpbiBjb21wbGV4ZXMgYW5kIHRoZWlyIHJlZ3VsYXRpb24gaW4gcHJvcGhhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIENlbGwgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTE7ICBjaXRhdGlvbl9wYWdlcz03NDktNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNS8zLzMwNy8yNTA3MjE1MS9kc3kwMDMucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEV1dHJlbWEgaXMgYSBnZW51cyBpbiB0aGUgQnJhc3NpY2FjZWFlLCB3aGljaCBpbmNsdWRlcyBzcGVjaWVzIG9mIHNjaWVudGlmaWMgYW5kIGVjb25vbWljIGltcG9ydGFuY2UuIE1hbnkgRXV0cmVtYSBzcGVjaWVzIGFyZSBtb250YW5lIGFuZC9vciBhbHBpbmUgIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00ODMxMDQ2JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZGqz2U5stcXtjQs9RHCyhlRCrClj" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f3%2f307%2f4831046" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f3%2f307%2f4831046" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=109141" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/3/1/m_dnares_25_3cover.png?Expires=2147483647&amp;Signature=zOskx939n215x6LsaNaPxmag9A6Tt6e7bP~Kt5lfxwaH8FjX1dKLUJ0mdj-Sm0uq9hmCpw3OAnAQFLS-GTY4CsXXX4aRhm5J1IbPv51CcTKXj828y94I4tx-BuFB-EA4WwEvEot4P8hDcnFzu~xWV52k2Sm6IHcCABqAv5wSWve6oMrdsQXhpN0srJqhN5UIcLjzws~g9qv24XhgtRb~Nw7X9IiEo1EM5V47od3bkv5yHlkUT1xmh3tJ4-ZjHw-HyAXqIp8eeHds4Cfjy9xqh8e3n5ynt516a1ZMF0hjA3wYH5X0-y3lqTHnWmgUvQv1L6dLXdRWhki~KqxTetYsJQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/3">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="118057181">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057181">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057183">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057183">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057187">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057187">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057206">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057206">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057225">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057225">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057230">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057230">Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057232">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057232">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118057242">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118057242">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/3/297/4838783">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/3/317/4898127">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">The genomes of two <em>Eutrema</em> species provide insight into plant adaptation to high altitudes
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xinyi Guo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University, Chengdu 610065, PR China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xinyi+Guo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Guo X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Guo X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xinyi Guo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Quanjun Hu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University, Chengdu 610065, PR China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Quanjun+Hu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hu Q">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hu Q&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Quanjun Hu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Guoqian Hao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University, Chengdu 610065, PR China</div><div class="aff">Management Committee for Emei Mountain Scenic Area, Biodiversity Institute of Emei Mountain, Leshan 614200, Sichuan, PR China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Guoqian+Hao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hao G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hao G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Guoqian Hao</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiaojuan Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University, Chengdu 610065, PR China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiaojuan+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xiaojuan Wang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Dan Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University, Chengdu 610065, PR China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Dan+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Dan Zhang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tao Ma</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University, Chengdu 610065, PR China</div>
        </div>




        <div class="info-card-location">
            <a href="http://orcid.org/0000-0002-7094-6868">
                <img class="orchid-icon" alt="orchid id" src="/UI/app/img/mini-icon.png"/>&nbsp;&nbsp;http://orcid.org/0000-0002-7094-6868
           </a>
        </div>

        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tao+Ma">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ma T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ma T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tao Ma</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jianquan Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Key Laboratory of Bio-Resource and Eco-Environment of Ministry of Education, College of Life Sciences, Sichuan University, Chengdu 610065, PR China</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy003-cor1" data-open="dsy003-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +028 8541 2053. Fax. +028 8541 2053. Email: <a href="mailto:liujq@nwipb.cas.cn" target="_blank">liujq@nwipb.cas.cn</a>
        </div>


        <div class="info-card-location">
            <a href="http://orcid.org/0000-0002-4237-7418">
                <img class="orchid-icon" alt="orchid id" src="/UI/app/img/mini-icon.png"/>&nbsp;&nbsp;http://orcid.org/0000-0002-4237-7418
           </a>
        </div>

        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jianquan+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jianquan Liu</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy003-FM1"><div class="fn"><div class="footnote-content">Xinyi Guo and Quanjun Hu Equal contributions to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy003-FM2"><div class="fn"><div class="footnote-content">Edited by Dr Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 3, 1 June 2018, Pages 307–315, <a href='https://doi.org/10.1093/dnares/dsy003'>https://doi.org/10.1093/dnares/dsy003</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">30 January 2018</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">16 October 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">24 January 2018</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4831046" href="/dnaresearch/article-pdf/25/3/307/25072151/dsy003.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Xinyi Guo, Quanjun Hu, Guoqian Hao, Xiaojuan Wang, Dan Zhang, Tao Ma, Jianquan Liu;  The genomes of two <em>Eutrema</em> species provide insight into plant adaptation to high altitudes, <em>DNA Research</em>, Volume 25, Issue 3, 1 June 2018, Pages 307–315, <a href='https://doi.org/10.1093/dnares/dsy003'>https://doi.org/10.1093/dnares/dsy003</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4831046?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4831046?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4831046?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4831046?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4831046?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4831046?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4831046?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=The%20genomes%20of%20two%20%26lt%3Bem%26gt%3BEutrema%26lt%3B%2Fem%26gt%3B%20species%20provide%20insight%20into%20plant%20adaptation%20to%20high%20altitudes&amp;publicationDate=2018-01-30&amp;volumeNum=25&amp;issueNum=3&amp;author=Guo%2C%20Xinyi%3B%20Hu%2C%20Quanjun&amp;startPage=307&amp;endPage=315&amp;contentId=10.1093%2Fdnares%2Fdsy003&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f3%2f307%2f4831046" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=118057181 id="118057181" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Eutrema</em> is a genus in the Brassicaceae, which includes species of scientific and economic importance. Many <em>Eutrema</em> species are montane and/or alpine species that arose very recently, making them ideal candidates for comparative studies to understand both ecological speciation and high-altitude adaptation in plants. Here we provide <em>de novo</em> whole-genome assemblies for a pair of recently diverged perennials with contrasting altitude preferences, the high-altitude <em>E. heterophyllum</em> from the eastern Qinghai-Tibet Plateau and its lowland congener <em>E. yunnanense</em>. The two assembled genomes are 350 Mb and 412 Mb, respectively, with 29,606 and 28,881 predicted genes. Comparative analysis of the two species revealed contrasting demographic trajectories and evolution of gene families. Gene family expansions shared between <em>E. heterophyllum</em> and other alpine species were identified, including the disease resistance R genes (NBS-LRRs or NLRs). Genes that are duplicated specifically in the high-altitude <em>E. heterophyllum</em> are involved mainly in reproduction, DNA damage repair and cold tolerance. The two <em>Eutrema</em> genomes reported here constitute important genetic resources for diverse studies, including the evolution of the genus <em>Eutrema</em>, of the Brassicaceae as a whole and of alpine plants across the world.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="Eutrema"><em>Eutrema</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;high-altitude adaptation&quot;">high-altitude adaptation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;de novo assembly&quot;"><em>de novo</em> assembly</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;comparative genomics&quot;">comparative genomics</a></div>                            <h2 scrollto-destination=118057183 id="118057183" class="section-title" >1. Introduction</h2>
<p>Adaptation to high altitude has contributed to the high level of plant and animal species biodiversity found on numerous mountains across the world.<a reveal-id="dsy003-B1" data-open="dsy003-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> It is well known that animals (including humans) living in highlands have evolved unique genetic adaptations in order to maintain oxygen homeostasis.<a reveal-id="dsy003-B2" data-open="dsy003-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> In plants, UV-B tolerance and cold tolerance appear to be critical for survival at high altitude.<a reveal-id="dsy003-B3" data-open="dsy003-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> However, the genetic mechanisms by which immobile plants adapt to high-altitude conditions remain largely unknown due to a lack of appropriate genome resources.</p><p>Species of the Brassicaceae family have a wide range of global distributions and adaptations, as well as different life histories and mating systems, and they therefore comprise an ideal system for many types of study.<a reveal-id="dsy003-B4" data-open="dsy003-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsy003-B5" data-open="dsy003-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> Genome resources are critically important for all these studies.<a reveal-id="dsy003-B6" data-open="dsy003-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsy003-B7" data-open="dsy003-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Recent advances in high-throughput sequencing technology have facilitated the ability of the scientific community to sequence the genomes of an increasing number of Brassicaceae species for multiple purposes.<a reveal-id="dsy003-B8 dsy003-B9 dsy003-B10" data-open="dsy003-B8 dsy003-B9 dsy003-B10" class="link link-ref link-reveal xref-bibr"><sup>8–10</sup></a> A previous genome study of an alpine polyploid Brassicaceae species suggested that expansion of numerous gene families was correlated with increased tolerance of cold and of UV-B stress.<a reveal-id="dsy003-B3" data-open="dsy003-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> However, such expansions may have arisen purely from genome duplications rather than because of high-altitude adaptations.</p><p><em>Eutrema</em> is a genus in the Brassicaceae that contains more than 30 species, most of which are distributed in eastern Asia. In addition to the model plant <em>E. salsugineum</em> (Pall.) Al-Shehbaz &amp; Warwick, which is used for studies of abiotic stress, and the commercial crop wasabi, <em>E. japonicum</em> (Miq.) Koidz, the genus contains numerous montane and/or alpine species.<a reveal-id="dsy003-B11" data-open="dsy003-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> Most of these species occur in the Qinghai-Tibet Plateau (QTP) and have originated recently in response to geologic and climatic changes since the Pliocene.<a reveal-id="dsy003-B12" data-open="dsy003-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> In this study, we report the <em>de novo</em> genome sequence assembly and comparative genomic analysis of an alpine plant, <em>E. heterophyllum</em>, from the eastern QTP, and its lowland congener, <em>E. yunnanense</em>, both of which originated from a very recent common ancestor and are diploid perennials with chromosome numbers of 2<em>n</em> = 14 and similar selfing breeding systems,<a reveal-id="dsy003-B12" data-open="dsy003-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> therefore providing an ideal basis for genetic comparisons without the complicating effects of either genome duplication or phylogenetic evolution. In addition, these two genome assemblies provide important genetic resources for future comparative studies of this genus and family and of high-altitude adaptation in alpine plants.</p>                            <h2 scrollto-destination=118057187 id="118057187" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=118057188 id="118057188" class="section-title" >2.1. Plant materials, genomic DNA and total RNA extraction</h3>
<p>We selected and sequenced a pair of <em>Eutrema</em> species that met three criteria: (1) diploid (2<em>n</em> = 14), (2) having diverged recently and (3) growing at contrasting altitudes in their natural distributions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). One individual of <em>E. heterophyllum</em> was collected in June 2015 at the Zhuodala pass (∼4,700 m above sea level; asl) in Ganzi County, Sichuan Province, China. Fresh and healthy leaves, flowers and root were harvested and immediately frozen in liquid nitrogen, followed by storage at −80 °C in the laboratory prior to DNA/RNA extraction. For comparison, fresh samples of leaf, leafstalk and root tissues were obtained from one <em>E. yunnanense</em> plant collected in April 2014, from Xinning County (∼600 m asl), Hunan Province, China. We used the CTAB method to extract high-quality genomic DNA from 5 g of leaf tissues for each species. The quality of the high-molecular weight DNA was checked by 1% agarose gel electrophoresis. Around 200 μg of genomic DNA was used for library construction. We extracted total RNA with RNAiso Plus reagent (Takara, Japan) for each tissue collected from both species (i.e. leaves, flowers and roots for <em>E. heterophyllum</em>, and leaves, leafstalks and roots for <em>E. yunnanense</em>). The quality of the extracted RNA was verified by 1% agarose gel. DNase treatment was performed before library construction. For each sample, 5 μg of the total high-quality RNA was used for sequencing.</p>                            <h3 scrollto-destination=118057190 id="118057190" class="section-title" >2.2. Genome sequencing and assembly</h3>
<p>We constructed Illumina libraries with small (200-, 500- and 800-bp) and large (2-, 5-, 10- and 20-kb) inserts and then sequenced them following the Illumina protocols (Illumina, San Diego, CA) using the Illumina HiSeq X Ten platform at Novogene (Tianjin, China). Short reads were first subjected to quality filtering using Trimmomatic v0.33,<a reveal-id="dsy003-B13" data-open="dsy003-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> error correction using BFC (version r181)<a reveal-id="dsy003-B14" data-open="dsy003-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> and mate-pair data deduplication using FastUniq v1.1.<a reveal-id="dsy003-B15" data-open="dsy003-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> We then used SOAPec v2.01<a reveal-id="dsy003-B16" data-open="dsy003-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> to estimate the genome size as well as the level of genome-wide heterozygosity, which was based on the 17-mer frequency distribution. We adopted the Platanus<a reveal-id="dsy003-B17" data-open="dsy003-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> pipeline, which can efficiently assemble genomes with various levels of heterozygosity. Both genomes were initially assembled into scaffolds using Platanus (version 1.2.2)<a reveal-id="dsy003-B17" data-open="dsy003-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> with the first round of gap closing. An additional gap closing procedure was performed with GapCloser (version 1.12).<a reveal-id="dsy003-B16" data-open="dsy003-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Finally we evaluated genome assemblies for gene coverage using the CEGMA<a reveal-id="dsy003-B18" data-open="dsy003-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> and BUSCO (version 2.0)<a reveal-id="dsy003-B19" data-open="dsy003-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> pipeline.</p>                            <h3 scrollto-destination=118057192 id="118057192" class="section-title" >2.3. Repeat identification</h3>
<p>To structurally annotate repeat sequences in the two <em>Eutrema</em> genomes, we used the TEdenovo pipeline from the REPET package (v2.5) with default parameters.<a reveal-id="dsy003-B20" data-open="dsy003-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> The pipeline performed a self-comparison for the input genome using the BLASTER software (version 2.25)<a reveal-id="dsy003-B21" data-open="dsy003-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> to identify and classify repeated elements. In order to attain more comprehensive prediction of TEs, we employed the pipeline to carry out structural detection analysis using LTRharvest (version 1.5.8).<a reveal-id="dsy003-B22" data-open="dsy003-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> RECON v1.0.8,<a reveal-id="dsy003-B23" data-open="dsy003-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> PILER v1.0<a reveal-id="dsy003-B24" data-open="dsy003-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> and GROUPER v2.27<a reveal-id="dsy003-B21" data-open="dsy003-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> were then employed to cluster the matches detected. By performing a multiple sequence alignment using the Map algorithm,<a reveal-id="dsy003-B25" data-open="dsy003-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> a consensus sequence for each cluster was generated to represent the ancestral TE. These consensus sequences were then classified by looking for characteristic structural features and similarities to known TEs in Repbase (20.05),<a reveal-id="dsy003-B26" data-open="dsy003-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> and by scanning against the Pfam library<a reveal-id="dsy003-B27" data-open="dsy003-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> with HMMER3.<a reveal-id="dsy003-B28" data-open="dsy003-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> To annotate all TEs across the genome, we further employed the TEannot pipeline<a reveal-id="dsy003-B21" data-open="dsy003-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> with the default parameters. This pipeline mines the genome sequence using repeated sequences identified in the previous TEdenovo pipeline to produce classified non-redundant consensus repeat sequences along with short simple repeats, which are exported in GFF3 format. We followed the method of Maumus and Quesneville<a reveal-id="dsy003-B29" data-open="dsy003-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> to calculate the age of repetitive DNAs.</p>                            <h3 scrollto-destination=118057194 id="118057194" class="section-title" >2.4. Gene prediction</h3>
<p>A combination of <em>ab initio</em>, homology-based and transcript-based methods was used for gene prediction. We first built a comprehensive transcriptome database with the PASA pipeline.<a reveal-id="dsy003-B30" data-open="dsy003-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> After quality control with Trimmomatic v0.33,<a reveal-id="dsy003-B13" data-open="dsy003-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Illumina RNA-seq reads were assembled into <em>de novo</em> transcripts using Trinity.<a reveal-id="dsy003-B31" data-open="dsy003-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> Then two sets of genome-guided transcripts were built using (1) the genome-guided mode implemented in Trinity and (2) the HISAT-StringTie pipeline.<a reveal-id="dsy003-B32" data-open="dsy003-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> We performed <em>ab initio</em> prediction with Augustus (v3.2.2)<a reveal-id="dsy003-B33" data-open="dsy003-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> using <em>Arabidopsis thaliana</em> as the species model and our comprehensive transcriptome dataset as the input cDNA hint. We used GlimmerHMM (v3.0.4)<a reveal-id="dsy003-B34" data-open="dsy003-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> with the pre-trained directory for <em>A. thaliana</em> genes to generate another set of <em>ab initio</em> predictions. For homology-based prediction, protein sequences of 26,531 <em>E. salsugineum</em> genes<a reveal-id="dsy003-B35" data-open="dsy003-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> as well as the UniRef90 dataset<a reveal-id="dsy003-B36" data-open="dsy003-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> for all Brassicaceae species were aligned to the genome assemblies using SPALN2.<a reveal-id="dsy003-B37" data-open="dsy003-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> Gene models from the three main sources (i.e. aligned transcripts, <em>ab initio</em> predictions and aligned proteins) were merged to produce consensus models by EVidenceModeler.<a reveal-id="dsy003-B30" data-open="dsy003-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> Weights of evidences for gene models were manually set as: <em>ab initio</em> predictions, weight (Augustus) = 1 and weight (GlimmerHMM) = 1; protein alignments, weight (<em>E. salsugineum</em>) = 2 and weight (Brassicaceae_UniRef90) = 1; transcripts, weight (PASA) = 10. The EVM consensus predictions were updated by performing an additional round of PASA annotation in order to add UTR and alternatively spliced isoform annotations to gene models.</p>                            <h3 scrollto-destination=118057196 id="118057196" class="section-title" >2.5. Functional annotation</h3>
<p>We used BlastP<a reveal-id="dsy003-B38" data-open="dsy003-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> (with <em>E</em>-value ≤ 1 × 10<sup>−5</sup>) to assign functional descriptions by carrying out sequence homology searches against different sources, including protein datasets from SwissProt<a reveal-id="dsy003-B39" data-open="dsy003-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> and the <em>Arabidopsis</em> genome annotation version TAIR10<a reveal-id="dsy003-B40" data-open="dsy003-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a>. Motifs and domains within gene models were identified by deploying InterProScan (version 5.20.29)<a reveal-id="dsy003-B41" data-open="dsy003-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> with the options -dp -goterms -iprlookup -pathways -t p against multiple publicly available databases (Pfam, PROSITE, PRINTS, SMART, TIGRFAMs, Gene3D, PANTHER, etc.). We used AHRD (<a class="link link-uri" href="https://github.com/groupschoof/AHRD" target="_blank">https://github.com/groupschoof/AHRD</a> (24 January 2018, date last accessed)) to gather information from the search results and added Gene Ontology (GO) information using the GO annotation database<a reveal-id="dsy003-B42" data-open="dsy003-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> for <em>A. thaliana</em> (<a class="link link-uri" href="https://www.ebi.ac.uk/GOA/arabidopsis_release" target="_blank">https://www.ebi.ac.uk/GOA/arabidopsis_release</a>; (24 January 2018, date last accessed)).</p>                            <h3 scrollto-destination=118057198 id="118057198" class="section-title" >2.6. Comparative genomic analysis</h3>
<p>We downloaded the protein-coding genes of 13 Brassicaceae species (<em>A. thaliana</em>, <em>A. lyrata</em>, <em>Brassica rapa</em>, <em>Capsella rubella</em>, <em>Leavenworthia alabamica</em>, <em>Sisymbrium irio</em>, <em>Aethionema arabicum</em>, <em>Arabis alpina</em>, <em>Raphanus raphanistrum</em>, <em>Cardamine hirsuta</em>, <em>Schrenkiella parvula</em>, <em>Thlaspi arvense</em>, <em>E. salsugineum</em>; see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). We used OrthoMCL<a reveal-id="dsy003-B43" data-open="dsy003-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> to delineate gene families and cluster all genes into paralogous and orthologous groups. Gene family expansion and contraction analysis was performed with CAFE3 software.<a reveal-id="dsy003-B44" data-open="dsy003-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> Functional enrichment analysis was carried out by agriGO v2.0.<a reveal-id="dsy003-B45" data-open="dsy003-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> We used R<a reveal-id="dsy003-B46" data-open="dsy003-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> to perform clustering analysis for the gene families of the greatest size in <em>E. heterophyllum</em>. The transformed <em>z</em>-score profile was obtained from the number of genes per species for each family, and the hierarchically clustered (complete linkage clustering) ones, using Pearson correlation as a distance measure. The 2,235 single-copy gene families were used to reconstruct phylogenies with RAxML.<a reveal-id="dsy003-B47" data-open="dsy003-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> Alignment of protein sequences was performed with PRANK<a reveal-id="dsy003-B48" data-open="dsy003-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> and they were then back translated into coding sequences with PAL2NAL.<a reveal-id="dsy003-B49" data-open="dsy003-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> The MCMCtree program within the PAML package v4.9a<a reveal-id="dsy003-B50" data-open="dsy003-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> was used to estimate divergence time. The split of the two major lineages within the Brassicaceae was constrained to be between 20 and 30 million years ago.<a reveal-id="dsy003-B51" data-open="dsy003-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> We used MCScanX<a reveal-id="dsy003-B52" data-open="dsy003-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> to call intra- and inter-species gene collinearity. Synonymous substitutions were calculated for aligned gene pairs within identified collinearity blocks using the codeml program implemented in PAML.</p>                            <h3 scrollto-destination=118057200 id="118057200" class="section-title" >2.7. SNP calling and demographic history reconstruction</h3>
<p>We used the pairwise sequentially Markovian coalescent (PSMC) model<a reveal-id="dsy003-B53" data-open="dsy003-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> to estimate the history of effective population sizes (Ne) based on genome-wide diploid sequence data. This method has been previously applied to human and other vertebrates, as well as plants, for inferring demographic histories over long evolutionary periods. The quality-filtered Illumina pair end reads with insert sizes of ∼500 bp for each species were mapped to the corresponding genome assemblies using BWA (version 0.7.10-r789).<a reveal-id="dsy003-B54" data-open="dsy003-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> For SNP calling, we used the SAMtools (v1.4) pipeline.<a reveal-id="dsy003-B55" data-open="dsy003-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> The settings for PSMC analysis (-p and -t options) were chosen manually according to suggestions given by the authors (<a class="link link-uri" href="https://github.com/lh3/psmc" target="_blank">https://github.com/lh3/psmc</a> (24 January 2018, date last accessed)). We also performed 100 rounds of bootstrapping with the same parameters. The results were combined and plotted using the plot tool in PSMC. A generation time of 3 years and a mutation rate of 7 × 10<sup>−9</sup> per site per year were applied.</p>                            <h3 scrollto-destination=118057202 id="118057202" class="section-title" >2.8. NLR genes and integrated domains</h3>
<p>NLR (nucleotide-binding site with leucine-rich repeat) genes in the <em>E. heterophyllum</em> and <em>E. yunnanense</em> genomes were directly extracted from the InterProScan results using the Pfam domain of the nucleotide-binding adaptor shared by APAF-1, R proteins and CED-4 (NB-ARC, PF00931) as query. We further applied the same strategy to identify NLRs in protein sequences from 13 cruciferous species with genomes available. After excluding typical Pfam domains contained in NLRs (i.e. PF01582 for Toll/interleukin-1 receptor [TIR], PF13855 for leucine-rich repeat [LRR] and PF05659 for Resistance to Powdery Mildew 8 [RPW8]), we obtained a list of additional domains that were likely to be fused with NLR proteins. We did not detect TIR2 (PF13676) in our annotated NLRs; this is a TIR domain newly identified among NLR proteins.<a reveal-id="dsy003-B56" data-open="dsy003-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> However, most previously characterized gene fusions were recaptured in this study, indicating that the approach we used for domain searching was conservative and generally reliable. A maximum likelihood tree of phloem protein 2 domains was constructed with RAxML under the PROTGAMMA model with BLOSUM62 matrix, specifying 500 bootstrap replicates. The gene tree was visualized in FigTree v1.4.2 (<a class="link link-uri" href="http://tree.bio.ed.ac.uk/software/figtree/" target="_blank">http://tree.bio.ed.ac.uk/software/figtree/</a> (24 January 2018, date last accessed)).</p>                            <h3 scrollto-destination=118057204 id="118057204" class="section-title" >2.9. Positive selection in <em>SCC3</em> genes</h3>
<p>The ratio of non-synonymous substitutions per non-synonymous site (dN) to synonymous substitutions per synonymous site (dS), or ω, is a commonly used indicator of selective pressure acting on protein-coding genes, with ω &gt; 1 representing positive selection, ω = 1 representing neutral evolution and ω &lt; 1 representing purifying selection. To identify signals of positive selection, we extracted <em>SCC3</em> sequences from all Brassicaceae genomes. A gene tree of aligned protein sequences was constructed using RAxML under the PROTGAMMA model with BLOSUM62 matrix, specifying 500 bootstrap replicates. We used codeml in the PAML software package (v4.9a) to perform the branch-site test among duplicated <em>SCC3</em> paralogs for a series of pairs of alternative versus null models, aiming to find out whether or not any proportion of coding sites within each paralog has ω &gt; 1 compared with the background level. Log-likelihood values for each pair of nested models were compared using the likelihood ratio test (LRT) to obtain the <em>P-</em>value of significance.</p>                            <h2 scrollto-destination=118057206 id="118057206" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=118057207 id="118057207" class="section-title" >3.1. Genome assembly and annotation</h3>
<p>After Illumina sequencing and quality control, we obtained 111.87 and 97.29 Gb of Illumina short reads from seven sequencing libraries, corresponding to an estimated 249× and 230× base-pair coverage, for <em>E. heterophyllum</em> and <em>E. yunnanense</em>, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). The estimated genome size of ∼400 Mb for both species is larger than that of their congener <em>E. salsugineum</em>,<a reveal-id="dsy003-B35" data-open="dsy003-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> with a higher level of genome-wide heterozygosity in the low-altitude species <em>E. yunnanense</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). A <em>de novo</em> assembly pipeline allowed us to generate genome assemblies that captured 350 and 413 Mb in 149,415 and 317,771 contigs for the two <em>Eutrema</em> genomes, of which 328 and 386 Mb (81% and 91% of the estimated genome sizes) were represented in scaffolds of 1 kb or greater (<a reveal-id="dsy003-T1" data-open="dsy003-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Repetitive DNA constitutes 67% and 70% of, respectively, the <em>E. heterophyllum</em> and the <em>E. yunnanense</em> genome. Long terminal repeat (LTR) retrotransposons are the most abundant among these repetitive sequences (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>), having contributed to a recent wave of TE burst (Kimura divergence ≤ 5, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). Gene completeness analysis revealed that we had achieved more than 95% completeness of gene coverage for both genomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). We predicted 29,606 genes for <em>E. heterophyllum</em> and 28,881 for <em>E. yunnanense</em> (<a reveal-id="dsy003-T1" data-open="dsy003-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Clustering of predicted proteomes for 15 Brassicaceae genomes yielded a total of 27,193 gene families, which comprised 380,484 genes (∼85% of all 448,886 genes; see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). <div content-id="dsy003-T1" class="table-modal"><div class="table-wrap"><div id="dsy003-T1" data-id="dsy003-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Genome assembly and annotation statistics for two <em>Eutrema</em> species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th><em>E. heterophyllum</em> </th><th><em>E. yunnanense</em> </th></tr></thead><tbody><tr><td>Assembly statistics </td><td> </td><td> </td></tr><tr><td> Estimated genome size (Mb) </td><td>405 </td><td>423 </td></tr><tr><td> Number of contigs </td><td>149,415 </td><td>317,771 </td></tr><tr><td> Contig length (Mb) </td><td>350.47 </td><td>412.62 </td></tr><tr><td> Longest contig (Mb) </td><td>0.67 </td><td>0.39 </td></tr><tr><td> Contig L50 (seqs) </td><td>1264 </td><td>3269 </td></tr><tr><td> Contig N50 (kb) </td><td>74.65 </td><td>32.56 </td></tr><tr><td> Number of scaffolds (&gt;1 kb) </td><td>7,690 </td><td>14,031 </td></tr><tr><td> Scaffold length (Mb) </td><td>328.16 </td><td>385.58 </td></tr><tr><td> Longest scaffold (Mb) </td><td>3.5 </td><td>2.07 </td></tr><tr><td> Scaffold L50 (seqs) </td><td>189 </td><td>341 </td></tr><tr><td> Scaffold N50 (kb) </td><td>535.32 </td><td>331.5 </td></tr><tr><td>Annotation statistics </td><td> </td><td> </td></tr><tr><td> Number of gene models </td><td>29,606 </td><td>28,881 </td></tr><tr><td> With InterPro annotations </td><td>22,593 </td><td>22 343 </td></tr><tr><td> With GO annotations </td><td>20,879 </td><td>20,254 </td></tr><tr><td> Mean CDS size (bp) </td><td>1201.3 </td><td>1222.3 </td></tr><tr><td> Mean exon size (bp) </td><td>291.5 </td><td>301.3 </td></tr><tr><td> Mean intron size (bp) </td><td>233.0 </td><td>210.6 </td></tr></tbody></table></div><div content-id="dsy003-T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th><em>E. heterophyllum</em> </th><th><em>E. yunnanense</em> </th></tr></thead><tbody><tr><td>Assembly statistics </td><td> </td><td> </td></tr><tr><td> Estimated genome size (Mb) </td><td>405 </td><td>423 </td></tr><tr><td> Number of contigs </td><td>149,415 </td><td>317,771 </td></tr><tr><td> Contig length (Mb) </td><td>350.47 </td><td>412.62 </td></tr><tr><td> Longest contig (Mb) </td><td>0.67 </td><td>0.39 </td></tr><tr><td> Contig L50 (seqs) </td><td>1264 </td><td>3269 </td></tr><tr><td> Contig N50 (kb) </td><td>74.65 </td><td>32.56 </td></tr><tr><td> Number of scaffolds (&gt;1 kb) </td><td>7,690 </td><td>14,031 </td></tr><tr><td> Scaffold length (Mb) </td><td>328.16 </td><td>385.58 </td></tr><tr><td> Longest scaffold (Mb) </td><td>3.5 </td><td>2.07 </td></tr><tr><td> Scaffold L50 (seqs) </td><td>189 </td><td>341 </td></tr><tr><td> Scaffold N50 (kb) </td><td>535.32 </td><td>331.5 </td></tr><tr><td>Annotation statistics </td><td> </td><td> </td></tr><tr><td> Number of gene models </td><td>29,606 </td><td>28,881 </td></tr><tr><td> With InterPro annotations </td><td>22,593 </td><td>22 343 </td></tr><tr><td> With GO annotations </td><td>20,879 </td><td>20,254 </td></tr><tr><td> Mean CDS size (bp) </td><td>1201.3 </td><td>1222.3 </td></tr><tr><td> Mean exon size (bp) </td><td>291.5 </td><td>301.3 </td></tr><tr><td> Mean intron size (bp) </td><td>233.0 </td><td>210.6 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsy003-T1" data-id="dsy003-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Genome assembly and annotation statistics for two <em>Eutrema</em> species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th><em>E. heterophyllum</em> </th><th><em>E. yunnanense</em> </th></tr></thead><tbody><tr><td>Assembly statistics </td><td> </td><td> </td></tr><tr><td> Estimated genome size (Mb) </td><td>405 </td><td>423 </td></tr><tr><td> Number of contigs </td><td>149,415 </td><td>317,771 </td></tr><tr><td> Contig length (Mb) </td><td>350.47 </td><td>412.62 </td></tr><tr><td> Longest contig (Mb) </td><td>0.67 </td><td>0.39 </td></tr><tr><td> Contig L50 (seqs) </td><td>1264 </td><td>3269 </td></tr><tr><td> Contig N50 (kb) </td><td>74.65 </td><td>32.56 </td></tr><tr><td> Number of scaffolds (&gt;1 kb) </td><td>7,690 </td><td>14,031 </td></tr><tr><td> Scaffold length (Mb) </td><td>328.16 </td><td>385.58 </td></tr><tr><td> Longest scaffold (Mb) </td><td>3.5 </td><td>2.07 </td></tr><tr><td> Scaffold L50 (seqs) </td><td>189 </td><td>341 </td></tr><tr><td> Scaffold N50 (kb) </td><td>535.32 </td><td>331.5 </td></tr><tr><td>Annotation statistics </td><td> </td><td> </td></tr><tr><td> Number of gene models </td><td>29,606 </td><td>28,881 </td></tr><tr><td> With InterPro annotations </td><td>22,593 </td><td>22 343 </td></tr><tr><td> With GO annotations </td><td>20,879 </td><td>20,254 </td></tr><tr><td> Mean CDS size (bp) </td><td>1201.3 </td><td>1222.3 </td></tr><tr><td> Mean exon size (bp) </td><td>291.5 </td><td>301.3 </td></tr><tr><td> Mean intron size (bp) </td><td>233.0 </td><td>210.6 </td></tr></tbody></table></div><div content-id="dsy003-T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th><em>E. heterophyllum</em> </th><th><em>E. yunnanense</em> </th></tr></thead><tbody><tr><td>Assembly statistics </td><td> </td><td> </td></tr><tr><td> Estimated genome size (Mb) </td><td>405 </td><td>423 </td></tr><tr><td> Number of contigs </td><td>149,415 </td><td>317,771 </td></tr><tr><td> Contig length (Mb) </td><td>350.47 </td><td>412.62 </td></tr><tr><td> Longest contig (Mb) </td><td>0.67 </td><td>0.39 </td></tr><tr><td> Contig L50 (seqs) </td><td>1264 </td><td>3269 </td></tr><tr><td> Contig N50 (kb) </td><td>74.65 </td><td>32.56 </td></tr><tr><td> Number of scaffolds (&gt;1 kb) </td><td>7,690 </td><td>14,031 </td></tr><tr><td> Scaffold length (Mb) </td><td>328.16 </td><td>385.58 </td></tr><tr><td> Longest scaffold (Mb) </td><td>3.5 </td><td>2.07 </td></tr><tr><td> Scaffold L50 (seqs) </td><td>189 </td><td>341 </td></tr><tr><td> Scaffold N50 (kb) </td><td>535.32 </td><td>331.5 </td></tr><tr><td>Annotation statistics </td><td> </td><td> </td></tr><tr><td> Number of gene models </td><td>29,606 </td><td>28,881 </td></tr><tr><td> With InterPro annotations </td><td>22,593 </td><td>22 343 </td></tr><tr><td> With GO annotations </td><td>20,879 </td><td>20,254 </td></tr><tr><td> Mean CDS size (bp) </td><td>1201.3 </td><td>1222.3 </td></tr><tr><td> Mean exon size (bp) </td><td>291.5 </td><td>301.3 </td></tr><tr><td> Mean intron size (bp) </td><td>233.0 </td><td>210.6 </td></tr></tbody></table></div></div></p>                            <h3 scrollto-destination=118057209 id="118057209" class="section-title" >3.2. Comparative phylogenomics and demographic histories</h3>
<p>We analysed the phylogenetic relationships of 15 Brassicaceae species (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>) using four-fold degenerate third-codon transversion (4DTv) sites in 2,193 one-to-one orthologous genes identified across their genomes. The nuclear-genome phylogeny (<a reveal-id="dsy003-F1" data-open="dsy003-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>) is congruent with the plastome phylogeny reported previously.<a reveal-id="dsy003-B51" data-open="dsy003-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> We dated the divergence of the two species to 3.5 (1.5–7.2) Mya (<a reveal-id="dsy003-F1" data-open="dsy003-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). Clustering analysis of gene family size revealed that many duplicated genes were retained in genomes known to be polyploid (<a reveal-id="dsy003-F1" data-open="dsy003-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). We failed to detect any additional whole genome duplication in the two <em>Eutrema</em> species when compared with <em>A. thaliana</em> using synonymous substitutions per synonymous site (dS) age distribution analysis (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). We identified 581,883 and 807,391 heterozygous sites in, respectively, the <em>E. heterophyllum</em> and the <em>E. yunnanense</em> genome. Consistent with the results of analysing K-mer frequency distribution (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>), <em>E. heterophyllum</em> had a lower heterozygous SNP rate (1.75 × 10<sup>−3</sup>) than <em>E. yunnanense</em> (2.18 × 10<sup>−3</sup>). A sharp peak was observed in the distribution of genome-wide SNP density for <em>E. heterophyllum</em> (<a reveal-id="dsy003-F2" data-open="dsy003-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>), suggesting that mutations within this genome are more evenly distributed than those in the <em>E. yunnanense</em> genome. PSMC analysis based on the local SNP densities of heterozygous sites revealed distinct demographic histories for the two species, in that the high-altitude <em>E. heterophyllum</em> showed a decrease-increase-decrease trend while the population size of the low-altitude <em>E. yunnanense</em> decreased continuously in the recent past (<a reveal-id="dsy003-F2" data-open="dsy003-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). </p>                        <a id="118057211" scrollto-destination="118057211"></a>
<div id="dsy003-F1" data-id="dsy003-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/m_dsy003f1.png?Expires=2147483647&Signature=w0h3zjPEZ5xdnNFbwh-ugwifpigRWHC5GhDUb21T-ZkFZvVBKpvd959xYMlPUxYr7xn2hmpW05NUIeAlQPmv3vtAJgKgMKfjY28T-yMCHARBIFRSMRoPsedFwcF1uwX-ZSE7Y1e02azbsagr9SziTaUFsLqu-0GSwINt7wmWlDPJo12Xe58LSK7UDXiXOBqoPbjjbPsQzq0KyiQn8V0QFJwmLi3RGaR3dNWS~Ox7z6uJDpjKkrlLT59V5bOkik77ML9qW~y350G~Ncd3twkAfy5QKMOUJYTrOXUcRckdfbYvXmFp73M-wAUV0xZwfLLmz5p5mFFB814QmBjGClinhw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dated phylogeny and gene family analysis of Brassicaceae species. Pie charts showing gains (red) and losses (blue) of gene families are indicated along branches and nodes. The numbers of gene families, orphans (single-copy gene families) and predicted genes, as well as a heat map depicting the hierarchically clustered z-score profile for gene numbers within each family, is indicated next to each species. Background colours on the phylogenetic tree (top to bottom) are basal clade, lineage II and lineage I Brassicaceae species, respectively. Full species names: Aethionema arabicum, Arabis alpina, Eutrema yunnanense, Eutrema heterophyllum, Eutrema salsugineum, Thlaspi arvense, Schrenkiella parvula, Sisymbrium irio, Raphanus raphanistrum, Brassica rapa, Cardamine hirsuta, Leavenworthia alabamica, Capsella rubella, Arabidopsis thaliana, Arabidopsis lyrata. The star and triangle denote lineage-specific whole-genome triplications. Mya, million years ago. See online version for color display." path-from-xml="dsy003f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057211/dsy003f1.tif" path-from-xml="dsy003f1.tif" target="_blank">View large</a><a section="118057211" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003f1.png?Expires=2147483647&Signature=ajen7DNTSOd6vGjcD-qby0wNAKLiM5N0~ZM5ZdK3nI1ESq4H5NHsoVvnwLGQ4U~S~PSca~RBd7vMHSFVzRpDa83QRxEQHOZCKShDSgi6nqdkZEEmfcND3qeL58laZnjnB97SFGRpspUKMGtqXYixArFmawMtX343Coi464uXulbfFO8lnpemon731VUd-kFzR8WkjrWFsiHHGkhmudCVvWxBqn5D2qwcjRCJWkJXb8wZkR3QbSGRCF4yRryBpeA4qW~IQ8gW-eE-Z9Q2XuJKWMD7LJSH04MCEybor2R7TdHmUsLC-dCceMiJQvoNd-SHJ3kDN6nALNvJpu9YFeGdQQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057211&ar=4831046&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy003f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Dated phylogeny and gene family analysis of Brassicaceae species. Pie charts showing gains (red) and losses (blue) of gene families are indicated along branches and nodes. The numbers of gene families, orphans (single-copy gene families) and predicted genes, as well as a heat map depicting the hierarchically clustered <em>z</em>-score profile for gene numbers within each family, is indicated next to each species. Background colours on the phylogenetic tree (top to bottom) are basal clade, lineage II and lineage I Brassicaceae species, respectively. Full species names: <em>Aethionema arabicum</em>, <em>Arabis alpina</em>, <em>Eutrema yunnanense</em>, <em>Eutrema heterophyllum</em>, <em>Eutrema salsugineum</em>, <em>Thlaspi arvense</em>, <em>Schrenkiella parvula</em>, <em>Sisymbrium irio</em>, <em>Raphanus raphanistrum</em>, <em>Brassica rapa</em>, <em>Cardamine hirsuta</em>, <em>Leavenworthia alabamica</em>, <em>Capsella rubella</em>, <em>Arabidopsis thaliana</em>, <em>Arabidopsis lyrata</em>. The star and triangle denote lineage-specific whole-genome triplications. Mya, million years ago. See online version for color display.</p></div></div><div content-id="dsy003-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/m_dsy003f1.png?Expires=2147483647&Signature=w0h3zjPEZ5xdnNFbwh-ugwifpigRWHC5GhDUb21T-ZkFZvVBKpvd959xYMlPUxYr7xn2hmpW05NUIeAlQPmv3vtAJgKgMKfjY28T-yMCHARBIFRSMRoPsedFwcF1uwX-ZSE7Y1e02azbsagr9SziTaUFsLqu-0GSwINt7wmWlDPJo12Xe58LSK7UDXiXOBqoPbjjbPsQzq0KyiQn8V0QFJwmLi3RGaR3dNWS~Ox7z6uJDpjKkrlLT59V5bOkik77ML9qW~y350G~Ncd3twkAfy5QKMOUJYTrOXUcRckdfbYvXmFp73M-wAUV0xZwfLLmz5p5mFFB814QmBjGClinhw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Dated phylogeny and gene family analysis of Brassicaceae species. Pie charts showing gains (red) and losses (blue) of gene families are indicated along branches and nodes. The numbers of gene families, orphans (single-copy gene families) and predicted genes, as well as a heat map depicting the hierarchically clustered z-score profile for gene numbers within each family, is indicated next to each species. Background colours on the phylogenetic tree (top to bottom) are basal clade, lineage II and lineage I Brassicaceae species, respectively. Full species names: Aethionema arabicum, Arabis alpina, Eutrema yunnanense, Eutrema heterophyllum, Eutrema salsugineum, Thlaspi arvense, Schrenkiella parvula, Sisymbrium irio, Raphanus raphanistrum, Brassica rapa, Cardamine hirsuta, Leavenworthia alabamica, Capsella rubella, Arabidopsis thaliana, Arabidopsis lyrata. The star and triangle denote lineage-specific whole-genome triplications. Mya, million years ago. See online version for color display." path-from-xml="dsy003f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057211/dsy003f1.tif" path-from-xml="dsy003f1.tif" target="_blank">View large</a><a section="118057211" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003f1.png?Expires=2147483647&Signature=ajen7DNTSOd6vGjcD-qby0wNAKLiM5N0~ZM5ZdK3nI1ESq4H5NHsoVvnwLGQ4U~S~PSca~RBd7vMHSFVzRpDa83QRxEQHOZCKShDSgi6nqdkZEEmfcND3qeL58laZnjnB97SFGRpspUKMGtqXYixArFmawMtX343Coi464uXulbfFO8lnpemon731VUd-kFzR8WkjrWFsiHHGkhmudCVvWxBqn5D2qwcjRCJWkJXb8wZkR3QbSGRCF4yRryBpeA4qW~IQ8gW-eE-Z9Q2XuJKWMD7LJSH04MCEybor2R7TdHmUsLC-dCceMiJQvoNd-SHJ3kDN6nALNvJpu9YFeGdQQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057211&ar=4831046&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy003f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Dated phylogeny and gene family analysis of Brassicaceae species. Pie charts showing gains (red) and losses (blue) of gene families are indicated along branches and nodes. The numbers of gene families, orphans (single-copy gene families) and predicted genes, as well as a heat map depicting the hierarchically clustered <em>z</em>-score profile for gene numbers within each family, is indicated next to each species. Background colours on the phylogenetic tree (top to bottom) are basal clade, lineage II and lineage I Brassicaceae species, respectively. Full species names: <em>Aethionema arabicum</em>, <em>Arabis alpina</em>, <em>Eutrema yunnanense</em>, <em>Eutrema heterophyllum</em>, <em>Eutrema salsugineum</em>, <em>Thlaspi arvense</em>, <em>Schrenkiella parvula</em>, <em>Sisymbrium irio</em>, <em>Raphanus raphanistrum</em>, <em>Brassica rapa</em>, <em>Cardamine hirsuta</em>, <em>Leavenworthia alabamica</em>, <em>Capsella rubella</em>, <em>Arabidopsis thaliana</em>, <em>Arabidopsis lyrata</em>. The star and triangle denote lineage-specific whole-genome triplications. Mya, million years ago. See online version for color display.</p></div></div>                        <a id="118057212" scrollto-destination="118057212"></a>
<div id="dsy003-F2" data-id="dsy003-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/m_dsy003f2.png?Expires=2147483647&Signature=HuCOtwoSYU4m3Twitu9Nnffmb0obUhW~OXVpc~xcR5xe8oQxMREBh6GMLaG5Japgk-3LzW8DwfjTXZ6I-UAyjRuP03~rmJ-CiQb0U5xEK~2yMKXebjzrHxEWWeC7L1XmfNAy1zoPk8Et7dkUvRyNAvO9RJ8UoylYtvhAJko-G5OHlsondZFkUCCGv3xIGnLmYAJChu61mZ6wkKGMuJR9XXOQnXOuDEP4ldnOtXZylJ-4IsHQ4sK6iC5PxngJJ48tacKeABbvh5dyCVFoRK2M0RZYpZjzlsrFzanFLOQfssLLz6G5XTsYMbh89PVVgpii0AP9xeaVHwRn0-hwLq9Msw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="SNP density distribution and demographic history of two Eutrema species. (A) Distribution of SNP density across each Eutrema genome. Heterozygous SNPs were counted and used to calculate heterozygosity density in non-overlapping 50-kb windows. (B) PSMC inference of ancestral effective population size change in the two species. The estimated ancestral population sizes are represented by central bold lines, and the PSMC estimations of 100 bootstraps resampled from the original sequence are represented by thin curves surrounding each line. A generation time of 3 years and a mutation rate of 7 × 10−9 per site per year were assumed for both species. Dark grey and abbreviations indicate glaciation periods, while light grey corresponds to interglaciation. BG (LGP), Baiyu Glaciation (last glaciation period, 70 ∼ 10 thousand years ago, kya); GG, Guxiang Glaciation (300 ∼ 130 kya); NG, Naynayxungla Glaciation (780 ∼ 500 kya); XG, Xixiabangma Glaciation (1,170 ∼ 800 kya). Red arrow denotes the estimated divergence time of the two species." path-from-xml="dsy003f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057212/dsy003f2.tif" path-from-xml="dsy003f2.tif" target="_blank">View large</a><a section="118057212" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003f2.png?Expires=2147483647&Signature=0NEblTBsrCkeSkQdN4y14gFr4ibM4ipY6Ej1TJABDDHVzw7L~rIndRatjbh~IIZLDDDY16ksYmOxST~A~j4Ca4OgwaTGBq~4uICvk-psvUAfsrmYCJSGCteLsTlauSDkYVV1KcqNR5pD7a6bSIrzGfaGajIKsdDLm4Y9bulVUIITsRVzowcFbe2lCqLZWDcEQzpX6ntQQWmI-B9FdF0TCX6dsGxdZzySgG-ZhSxb~cUbSpoBm4yFQRyM0Gx9-OUujeqoXurh0Oabd3RO7xeUjuBSyoK7BLSBrZ~tWcO0ZHzosGSdLFUjDWzI2tgg32aM8lzIdrpTe~cY2ZDM02JOUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057212&ar=4831046&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy003f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>SNP density distribution and demographic history of two <em>Eutrema</em> species. (A) Distribution of SNP density across each <em>Eutrema</em> genome. Heterozygous SNPs were counted and used to calculate heterozygosity density in non-overlapping 50-kb windows. (B) PSMC inference of ancestral effective population size change in the two species. The estimated ancestral population sizes are represented by central bold lines, and the PSMC estimations of 100 bootstraps resampled from the original sequence are represented by thin curves surrounding each line. A generation time of 3 years and a mutation rate of 7 × 10<sup>−9</sup> per site per year were assumed for both species. Dark grey and abbreviations indicate glaciation periods, while light grey corresponds to interglaciation. BG (LGP), Baiyu Glaciation (last glaciation period, 70 ∼ 10 thousand years ago, kya); GG, Guxiang Glaciation (300 ∼ 130 kya); NG, Naynayxungla Glaciation (780 ∼ 500 kya); XG, Xixiabangma Glaciation (1,170 ∼ 800 kya). Red arrow denotes the estimated divergence time of the two species.</p></div></div><div content-id="dsy003-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/m_dsy003f2.png?Expires=2147483647&Signature=HuCOtwoSYU4m3Twitu9Nnffmb0obUhW~OXVpc~xcR5xe8oQxMREBh6GMLaG5Japgk-3LzW8DwfjTXZ6I-UAyjRuP03~rmJ-CiQb0U5xEK~2yMKXebjzrHxEWWeC7L1XmfNAy1zoPk8Et7dkUvRyNAvO9RJ8UoylYtvhAJko-G5OHlsondZFkUCCGv3xIGnLmYAJChu61mZ6wkKGMuJR9XXOQnXOuDEP4ldnOtXZylJ-4IsHQ4sK6iC5PxngJJ48tacKeABbvh5dyCVFoRK2M0RZYpZjzlsrFzanFLOQfssLLz6G5XTsYMbh89PVVgpii0AP9xeaVHwRn0-hwLq9Msw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="SNP density distribution and demographic history of two Eutrema species. (A) Distribution of SNP density across each Eutrema genome. Heterozygous SNPs were counted and used to calculate heterozygosity density in non-overlapping 50-kb windows. (B) PSMC inference of ancestral effective population size change in the two species. The estimated ancestral population sizes are represented by central bold lines, and the PSMC estimations of 100 bootstraps resampled from the original sequence are represented by thin curves surrounding each line. A generation time of 3 years and a mutation rate of 7 × 10−9 per site per year were assumed for both species. Dark grey and abbreviations indicate glaciation periods, while light grey corresponds to interglaciation. BG (LGP), Baiyu Glaciation (last glaciation period, 70 ∼ 10 thousand years ago, kya); GG, Guxiang Glaciation (300 ∼ 130 kya); NG, Naynayxungla Glaciation (780 ∼ 500 kya); XG, Xixiabangma Glaciation (1,170 ∼ 800 kya). Red arrow denotes the estimated divergence time of the two species." path-from-xml="dsy003f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057212/dsy003f2.tif" path-from-xml="dsy003f2.tif" target="_blank">View large</a><a section="118057212" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003f2.png?Expires=2147483647&Signature=0NEblTBsrCkeSkQdN4y14gFr4ibM4ipY6Ej1TJABDDHVzw7L~rIndRatjbh~IIZLDDDY16ksYmOxST~A~j4Ca4OgwaTGBq~4uICvk-psvUAfsrmYCJSGCteLsTlauSDkYVV1KcqNR5pD7a6bSIrzGfaGajIKsdDLm4Y9bulVUIITsRVzowcFbe2lCqLZWDcEQzpX6ntQQWmI-B9FdF0TCX6dsGxdZzySgG-ZhSxb~cUbSpoBm4yFQRyM0Gx9-OUujeqoXurh0Oabd3RO7xeUjuBSyoK7BLSBrZ~tWcO0ZHzosGSdLFUjDWzI2tgg32aM8lzIdrpTe~cY2ZDM02JOUQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057212&ar=4831046&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy003f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>SNP density distribution and demographic history of two <em>Eutrema</em> species. (A) Distribution of SNP density across each <em>Eutrema</em> genome. Heterozygous SNPs were counted and used to calculate heterozygosity density in non-overlapping 50-kb windows. (B) PSMC inference of ancestral effective population size change in the two species. The estimated ancestral population sizes are represented by central bold lines, and the PSMC estimations of 100 bootstraps resampled from the original sequence are represented by thin curves surrounding each line. A generation time of 3 years and a mutation rate of 7 × 10<sup>−9</sup> per site per year were assumed for both species. Dark grey and abbreviations indicate glaciation periods, while light grey corresponds to interglaciation. BG (LGP), Baiyu Glaciation (last glaciation period, 70 ∼ 10 thousand years ago, kya); GG, Guxiang Glaciation (300 ∼ 130 kya); NG, Naynayxungla Glaciation (780 ∼ 500 kya); XG, Xixiabangma Glaciation (1,170 ∼ 800 kya). Red arrow denotes the estimated divergence time of the two species.</p></div></div>                            <h3 scrollto-destination=118057213 id="118057213" class="section-title" >3.3. Expansion of NLR genes and analysis of domain integration</h3>
<p>In comparison with other Brassicaceae species, we found that 141 and 178 gene families had significantly expanded (<em>P </em>&lt; 0.05) in the <em>E. heterophyllum</em> and <em>E. yunnanense</em> genomes, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S5–S8</a></span>). Among the gene families that had expanded within the high-altitude <em>E. heterophyllum</em>, GO terms including ‘response to stress’ and ‘nucleotide binding’ (adjusted <em>P</em> &lt; 0.05, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>) were overrepresented, while no particular GO term was enriched in the case of the low-altitude <em>E. yunnanense</em>. The terms enriched in <em>E. heterophyllum</em> could be largely attributed to 10 clusters of disease resistance genes encoding nucleotide-binding site with leucine-rich repeat (NLR) proteins (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). We noticed that most of these NLR gene families had also expanded in the genomes of three other Brassicaceae species, <em>A. alpina</em>, <em>A. lyrata</em> and <em>C. hirsuta</em>, of which the former two are well documented as alpine plants (<a reveal-id="dsy003-F3" data-open="dsy003-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). Two NLR families shared across these species account for ∼20% of the total number of NLR genes that differ between <em>E. heterophyllum</em> and <em>E. yunnanense</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Overall, the gene families expanded in <em>E. heterophyllum</em> overlap significantly with those expanded in each of these three species (<em>P</em> &lt; 0.01, Fisher’s exact test) (<a reveal-id="dsy003-F3" data-open="dsy003-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>). These results imply that parallel expansion of gene sets, particularly within the NLR genes, may have repeatedly occurred in alpine plants. </p>                        <a id="118057215" scrollto-destination="118057215"></a>
<div id="dsy003-F3" data-id="dsy003-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/m_dsy003f3.png?Expires=2147483647&Signature=WudvxqapI2fz4Qu68jl8CQILtKy17BmOeG1mT5zybGOrgaLocuWA2rCubd0fSexWtuCobkomBRHyXVLOSUZ3W4jNMCqm5PU~N0JvIqr1XGyxMbsrpl8oy7Tp-gbhAUAH8IX-5NjYHb9s4hOBqk4wMz0VcG8cOjVtmpUGr8MtEp~Sd0Jn5R8PASjSQBQ2gmsomDd1nL9CfH8wPJAuziJKwbisa0zX179edlU0vl8Gco-AQr8JA6ck3elAvMZLkPT1PFdRnwNOFZiotIng0dv-Gl-D9xcKkmluBNt6BsHXssdu6EAlyf00ImWv-qt1Pcm~LF9-ILZ862sISHLx4xZ0WA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Shared gene family expansions among four Brassicaceae species. Venn diagram showing overlap of shared gene families and of the nuclear-binding site with leucine-rich repeats (NLR) genes. Numbers of significantly expanded gene families among the four genomes that contain the largest number of NLRs are shown. Counts of the significantly expanded NLR gene families are given in parentheses." path-from-xml="dsy003f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057215/dsy003f3.tif" path-from-xml="dsy003f3.tif" target="_blank">View large</a><a section="118057215" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003f3.png?Expires=2147483647&Signature=i6ErqCPN2JoD2wCPhNjJS7w9YeKMlvzUmhKmK3~voRS~Ll8Z4edk-7-dNJYfGhtVlfCQzNiLd6qVt-L1agnfKUdPfiZBf5XpRchQrTtyyx~DzC0cKkEWs94SQq0YAiPYV2Rw7zrysUa4RKhPoHyunftYDw9sQ0kw8JbnBP0benngHSu1WjrkDaa0Gb-0ZwTCqQBPvukP1eF9qds~Lx4Dv5UhE7KdoTrVpWxP0-ZmlGoMaojvvmJMmsh6ChsDHUZZWRbmGO9ruXNTyfp1leNOuvTXtDodSqWNCRNeyrypD8NqEi-iSPHbB8KvIbKVIBkt8XG0Gc8JzI9lpZSvvOiyHA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057215&ar=4831046&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy003f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Shared gene family expansions among four Brassicaceae species. Venn diagram showing overlap of shared gene families and of the nuclear-binding site with leucine-rich repeats (NLR) genes. Numbers of significantly expanded gene families among the four genomes that contain the largest number of NLRs are shown. Counts of the significantly expanded NLR gene families are given in parentheses.</p></div></div><div content-id="dsy003-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/m_dsy003f3.png?Expires=2147483647&Signature=WudvxqapI2fz4Qu68jl8CQILtKy17BmOeG1mT5zybGOrgaLocuWA2rCubd0fSexWtuCobkomBRHyXVLOSUZ3W4jNMCqm5PU~N0JvIqr1XGyxMbsrpl8oy7Tp-gbhAUAH8IX-5NjYHb9s4hOBqk4wMz0VcG8cOjVtmpUGr8MtEp~Sd0Jn5R8PASjSQBQ2gmsomDd1nL9CfH8wPJAuziJKwbisa0zX179edlU0vl8Gco-AQr8JA6ck3elAvMZLkPT1PFdRnwNOFZiotIng0dv-Gl-D9xcKkmluBNt6BsHXssdu6EAlyf00ImWv-qt1Pcm~LF9-ILZ862sISHLx4xZ0WA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Shared gene family expansions among four Brassicaceae species. Venn diagram showing overlap of shared gene families and of the nuclear-binding site with leucine-rich repeats (NLR) genes. Numbers of significantly expanded gene families among the four genomes that contain the largest number of NLRs are shown. Counts of the significantly expanded NLR gene families are given in parentheses." path-from-xml="dsy003f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057215/dsy003f3.tif" path-from-xml="dsy003f3.tif" target="_blank">View large</a><a section="118057215" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003f3.png?Expires=2147483647&Signature=i6ErqCPN2JoD2wCPhNjJS7w9YeKMlvzUmhKmK3~voRS~Ll8Z4edk-7-dNJYfGhtVlfCQzNiLd6qVt-L1agnfKUdPfiZBf5XpRchQrTtyyx~DzC0cKkEWs94SQq0YAiPYV2Rw7zrysUa4RKhPoHyunftYDw9sQ0kw8JbnBP0benngHSu1WjrkDaa0Gb-0ZwTCqQBPvukP1eF9qds~Lx4Dv5UhE7KdoTrVpWxP0-ZmlGoMaojvvmJMmsh6ChsDHUZZWRbmGO9ruXNTyfp1leNOuvTXtDodSqWNCRNeyrypD8NqEi-iSPHbB8KvIbKVIBkt8XG0Gc8JzI9lpZSvvOiyHA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057215&ar=4831046&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy003f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Shared gene family expansions among four Brassicaceae species. Venn diagram showing overlap of shared gene families and of the nuclear-binding site with leucine-rich repeats (NLR) genes. Numbers of significantly expanded gene families among the four genomes that contain the largest number of NLRs are shown. Counts of the significantly expanded NLR gene families are given in parentheses.</p></div></div><p>Plant NLR genes can fuse with additional domains (hereafter referred to as NB-IDs) that serve as ‘baits’ for pathogen-derived molecules.<a reveal-id="dsy003-B56" data-open="dsy003-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> We identified 112 NB-IDs with 82 distinct integrated domains across the Brassicaceae genomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). Within the fused domains, 22 were present in at least two genomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>). Interestingly, in <em>E. heterophyllum</em>, we found that integration of one domain encoding phloem protein 2 (PP2, Pfam ID: PF14299), characteristic of a group of plant lectins involved in responses to various stresses,<a reveal-id="dsy003-B57" data-open="dsy003-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> was shared by two other species, <em>A. alpina</em> and <em>C. hirsuta</em> (<a reveal-id="dsy003-F4" data-open="dsy003-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). It is worth noting that the NLRs fused with this domain were clustered into the same gene family that had undergone expansion in all these genomes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). Obviously, the fusion of PP2 arose independently in these three species (<a reveal-id="dsy003-F4" data-open="dsy003-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>), probably in response to the same class of pathogens. </p>                        <a id="118057217" scrollto-destination="118057217"></a>
<div id="dsy003-F4" data-id="dsy003-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/m_dsy003f4.png?Expires=2147483647&Signature=QnF6UJsN2BX2DioSVG8c7vj0EHoGGpsBLEprqAiFMmn2v6lDkeKXQGySb4hCCtDJ8uwbacug-dwewfi63Hxf49rDG5amhMOrcRpntFjH9bjCYt4CMZfQT7hSrDvcT882UaSqvxq3Zprvr959irzmUHBLSRWNbaz2D6U8gmGHVWmiIIYYGpZOGJFYOj15TLGqlgqvv2mhlNc9AsrHjWjBhSwzZeM9XQ~-sU2QlmlyJZhuvcty556LvisxqaeVKLKQmUFld94YdZwxNBS1QSsz9WyPByndtbzmt2GSGZaGW306bVVL6eOf0LbM6eslOxbC0MLgV-bV6WExLnmVz5UQUg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogeny of the phloem protein 2 (PP2) domain fused with expanded NLR genes. Sequences from all A. thaliana proteins containing this domain and the integrated PP2 detected in three Brassicaceae genomes (labelled with numbers within circles) were analyzed. Branches are coloured red if a species has a documented high-altitude distribution, and green if not. The structure of NLR fused with PP2 is shown below the gene tree. See online version for color display." path-from-xml="dsy003f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057217/dsy003f4.tif" path-from-xml="dsy003f4.tif" target="_blank">View large</a><a section="118057217" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003f4.png?Expires=2147483647&Signature=SzuqVItRSHAg548sYCSp1cYrY5bqO4wlpprHurCduRfW8jdupEB4NeC~md6NYOsJ6ff5GPTNuldjGnynytJ8FpwUP0xnLZM4~ICntECP4~mrbI~Z7pmvdyTJqPymkae~FfisBTGJLLYmwxyLvaEE226ZZKDjjZHbuv2RSIQEvBRBb4tIae~Uy-nvT9I77NmqtLJ3Ziba74AFOrH0WwlktnQiSCu8fnMTExVJvRtkCcq9JqeDQWp5tlEFDJFCSJ9rao~e32LeBHy92V~nFcu12zk8cKJ2A8yLYIPt7AANEHUSIXCdcFCo685uJxaCCKXAO9IrOvS5wApTFs64gHws1A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057217&ar=4831046&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy003f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogeny of the phloem protein 2 (PP2) domain fused with expanded NLR genes. Sequences from all <em>A. thaliana</em> proteins containing this domain and the integrated PP2 detected in three Brassicaceae genomes (labelled with numbers within circles) were analyzed. Branches are coloured red if a species has a documented high-altitude distribution, and green if not. The structure of NLR fused with PP2 is shown below the gene tree. See online version for color display.</p></div></div><div content-id="dsy003-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/m_dsy003f4.png?Expires=2147483647&Signature=QnF6UJsN2BX2DioSVG8c7vj0EHoGGpsBLEprqAiFMmn2v6lDkeKXQGySb4hCCtDJ8uwbacug-dwewfi63Hxf49rDG5amhMOrcRpntFjH9bjCYt4CMZfQT7hSrDvcT882UaSqvxq3Zprvr959irzmUHBLSRWNbaz2D6U8gmGHVWmiIIYYGpZOGJFYOj15TLGqlgqvv2mhlNc9AsrHjWjBhSwzZeM9XQ~-sU2QlmlyJZhuvcty556LvisxqaeVKLKQmUFld94YdZwxNBS1QSsz9WyPByndtbzmt2GSGZaGW306bVVL6eOf0LbM6eslOxbC0MLgV-bV6WExLnmVz5UQUg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogeny of the phloem protein 2 (PP2) domain fused with expanded NLR genes. Sequences from all A. thaliana proteins containing this domain and the integrated PP2 detected in three Brassicaceae genomes (labelled with numbers within circles) were analyzed. Branches are coloured red if a species has a documented high-altitude distribution, and green if not. The structure of NLR fused with PP2 is shown below the gene tree. See online version for color display." path-from-xml="dsy003f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057217/dsy003f4.tif" path-from-xml="dsy003f4.tif" target="_blank">View large</a><a section="118057217" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003f4.png?Expires=2147483647&Signature=SzuqVItRSHAg548sYCSp1cYrY5bqO4wlpprHurCduRfW8jdupEB4NeC~md6NYOsJ6ff5GPTNuldjGnynytJ8FpwUP0xnLZM4~ICntECP4~mrbI~Z7pmvdyTJqPymkae~FfisBTGJLLYmwxyLvaEE226ZZKDjjZHbuv2RSIQEvBRBb4tIae~Uy-nvT9I77NmqtLJ3Ziba74AFOrH0WwlktnQiSCu8fnMTExVJvRtkCcq9JqeDQWp5tlEFDJFCSJ9rao~e32LeBHy92V~nFcu12zk8cKJ2A8yLYIPt7AANEHUSIXCdcFCo685uJxaCCKXAO9IrOvS5wApTFs64gHws1A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057217&ar=4831046&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy003f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogeny of the phloem protein 2 (PP2) domain fused with expanded NLR genes. Sequences from all <em>A. thaliana</em> proteins containing this domain and the integrated PP2 detected in three Brassicaceae genomes (labelled with numbers within circles) were analyzed. Branches are coloured red if a species has a documented high-altitude distribution, and green if not. The structure of NLR fused with PP2 is shown below the gene tree. See online version for color display.</p></div></div>                            <h3 scrollto-destination=118057218 id="118057218" class="section-title" >3.4. Species-specific gene duplications and high-altitude adaptation</h3>
<p>We also analysed the functional enrichment of genes that were duplicated specifically in the high- or low-altitude species. In the high-altitude <em>E. heterophyllum</em>, we observed 30 enriched GO terms within the biological process ontology, including ‘response to stimulus’, ‘response to stress’, ‘reproductive process’ and ‘cell cycle’. However, only four terms within the biological process ontology, ‘developmental process’, ‘anatomical structure development’, ‘multicellular organism development’ and ‘multicellular organismal process’, were recovered for the low-altitude <em>E. yunnanense</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S11–S14</a></span>).</p><p>A close examination of the families expanded in <em>E. heterophyllum</em> revealed multiple genes that are involved in plant reproduction and DNA damage repair (DDR), probably in response to UV-B radiation and other abiotic stresses imposed by the harsh alpine environment. For example, more copies were found for <em>NBS1</em>, <em>RPA32</em> and <em>SNI1</em>, which are involved in homologous recombination (HR) using the intact sister chromatid, and <em>KU80</em> and <em>XRCC4</em>, which participate in the non-homologous end-joining (NHEJ) process (<a reveal-id="dsy003-F5" data-open="dsy003-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>); HR and NHEJ are the two major pathways for the repair of double-strand breaks.<a reveal-id="dsy003-B58" data-open="dsy003-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> Duplicated genes involved in the regulation of the cell cycle (<em>SIM</em>, <em>ATXR5</em> and <em>ICK3</em>) were also detected for <em>E. heterophyllum</em> (<a reveal-id="dsy003-F5" data-open="dsy003-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>); SIM is a plant-specific CDK inhibitor binding CDKB1; 1, probably specifically controlling endocycle onset during the G2/M transition.<a reveal-id="dsy003-B59" data-open="dsy003-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a></p>                        <a id="118057221" scrollto-destination="118057221"></a>
<div id="dsy003-F5" data-id="dsy003-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/m_dsy003f5.png?Expires=2147483647&Signature=V2SA~ogbS5pv1kLA~YMl9Kmx1uN21MWo5HKWvWpT3e--Mmzrd5SV4ARwU2sCIk2GGN0xpohfb7wZ~~iRsmQGMImjxzMifloh6jTLzyZcwxy~zGGFBKek0cvFsdsGe0E7R1RVgf9RxIJKZnOpMVVFcRF~tS~S~l5XhhVaTGLtDDgDGILUaFyMwxPIMX78fc7GyG8wBlpcze-jDz21iMaAW4b~a9vEObFWRlQbqgVk3AHyL8zNUNljWDmd8XnTtlnfiU-FpSEiSTeo8Vbx2N7~zV5aLnlLyaASbXh2e3ZTFGciolclYTmCxwZ4KGZzxAKGL760iIom2YItI5ICr7BmQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Duplication of genes related to DDR in E. heterophyllum. (A) Genes involved in the DDR pathway and cell cycle control that are duplicated in E. heterophyllum. Duplicated genes are shown in light brown. HR, homologous recombination; NHEJ, non-homologous end-joining. (B) A schematic picture of the plant cohesin complex indicating duplication of genes encoding two cohesin subunits, SCC3 and SMC3. The numbers of the corresponding genes in E. yunnanense and E. heterophyllum are shown in square brackets in the format [E. yunnanense, E. heterophyllum]. The expression profiles in FPKM (fragments per kilobase per million reads mapped) of paralogs are plotted as log10 values. See online version for color display." path-from-xml="dsy003f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057221/dsy003f5.tif" path-from-xml="dsy003f5.tif" target="_blank">View large</a><a section="118057221" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003f5.png?Expires=2147483647&Signature=I~~qoFcGlgXA0sbrEB7IdIE5LUKDLowG5zfZPM~0exkvfbU8OmYVsd6RrYWoNDidJblRr7XSFPAZczejDvAjLYve7YsrFPV9ws~Udb-nddQiaWQ~TQJxivIogEolYrk3O65cmFYgigWNDqSXHGq2pjmXNeFVBRpybjRQCpDprSONBTOluUyIzRnNztTnHA756fUGmQGAoW624Tz99WorW3k~RwGdtJqW8GGUuUs67EDN5QcgiPqPg9CnfbpfzwTazPLaHddUGOUIQnbTxPxJ4j4y1gFJeS5ZdWsheFGojgRnfGiWX0ihl5SCkbRs1D6ai-KLHF6HUI7nWGP1y1pwXQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057221&ar=4831046&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy003f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Duplication of genes related to DDR in <em>E. heterophyllum</em>. (A) Genes involved in the DDR pathway and cell cycle control that are duplicated in <em>E. heterophyllum</em>. Duplicated genes are shown in light brown. HR, homologous recombination; NHEJ, non-homologous end-joining. (B) A schematic picture of the plant cohesin complex indicating duplication of genes encoding two cohesin subunits, <em>SCC3</em> and <em>SMC3</em>. The numbers of the corresponding genes in <em>E. yunnanense</em> and <em>E. heterophyllum</em> are shown in square brackets in the format [<em>E. yunnanense</em>, <em>E. heterophyllum</em>]. The expression profiles in FPKM (fragments per kilobase per million reads mapped) of paralogs are plotted as log<sub>10</sub> values. See online version for color display.</p></div></div><div content-id="dsy003-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/m_dsy003f5.png?Expires=2147483647&Signature=V2SA~ogbS5pv1kLA~YMl9Kmx1uN21MWo5HKWvWpT3e--Mmzrd5SV4ARwU2sCIk2GGN0xpohfb7wZ~~iRsmQGMImjxzMifloh6jTLzyZcwxy~zGGFBKek0cvFsdsGe0E7R1RVgf9RxIJKZnOpMVVFcRF~tS~S~l5XhhVaTGLtDDgDGILUaFyMwxPIMX78fc7GyG8wBlpcze-jDz21iMaAW4b~a9vEObFWRlQbqgVk3AHyL8zNUNljWDmd8XnTtlnfiU-FpSEiSTeo8Vbx2N7~zV5aLnlLyaASbXh2e3ZTFGciolclYTmCxwZ4KGZzxAKGL760iIom2YItI5ICr7BmQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Duplication of genes related to DDR in E. heterophyllum. (A) Genes involved in the DDR pathway and cell cycle control that are duplicated in E. heterophyllum. Duplicated genes are shown in light brown. HR, homologous recombination; NHEJ, non-homologous end-joining. (B) A schematic picture of the plant cohesin complex indicating duplication of genes encoding two cohesin subunits, SCC3 and SMC3. The numbers of the corresponding genes in E. yunnanense and E. heterophyllum are shown in square brackets in the format [E. yunnanense, E. heterophyllum]. The expression profiles in FPKM (fragments per kilobase per million reads mapped) of paralogs are plotted as log10 values. See online version for color display." path-from-xml="dsy003f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057221/dsy003f5.tif" path-from-xml="dsy003f5.tif" target="_blank">View large</a><a section="118057221" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003f5.png?Expires=2147483647&Signature=I~~qoFcGlgXA0sbrEB7IdIE5LUKDLowG5zfZPM~0exkvfbU8OmYVsd6RrYWoNDidJblRr7XSFPAZczejDvAjLYve7YsrFPV9ws~Udb-nddQiaWQ~TQJxivIogEolYrk3O65cmFYgigWNDqSXHGq2pjmXNeFVBRpybjRQCpDprSONBTOluUyIzRnNztTnHA756fUGmQGAoW624Tz99WorW3k~RwGdtJqW8GGUuUs67EDN5QcgiPqPg9CnfbpfzwTazPLaHddUGOUIQnbTxPxJ4j4y1gFJeS5ZdWsheFGojgRnfGiWX0ihl5SCkbRs1D6ai-KLHF6HUI7nWGP1y1pwXQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057221&ar=4831046&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy003f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Duplication of genes related to DDR in <em>E. heterophyllum</em>. (A) Genes involved in the DDR pathway and cell cycle control that are duplicated in <em>E. heterophyllum</em>. Duplicated genes are shown in light brown. HR, homologous recombination; NHEJ, non-homologous end-joining. (B) A schematic picture of the plant cohesin complex indicating duplication of genes encoding two cohesin subunits, <em>SCC3</em> and <em>SMC3</em>. The numbers of the corresponding genes in <em>E. yunnanense</em> and <em>E. heterophyllum</em> are shown in square brackets in the format [<em>E. yunnanense</em>, <em>E. heterophyllum</em>]. The expression profiles in FPKM (fragments per kilobase per million reads mapped) of paralogs are plotted as log<sub>10</sub> values. See online version for color display.</p></div></div><p>Of the genes with the largest family sizes (<em>z</em>-score normalized) in <em>E. heterophyllum</em>, two (<em>SCC3</em> and <em>SMC3</em>) encode subunits of the cohesin complex, a well-known ring-shaped molecule with a critical role in sister chromatid cohesion.<a reveal-id="dsy003-B60" data-open="dsy003-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> In <em>E. heterophyllum</em>, we found four copies of <em>SCC3</em> and three ones of <em>SMC3</em> (<a reveal-id="dsy003-F5" data-open="dsy003-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>). Most of these copies were expressed in leaf, root and/or flower tissue (<a reveal-id="dsy003-F5" data-open="dsy003-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>). A duplication of <em>SCC3</em> was also found in <em>A. lyrata</em>, another alpine crucifer species (<a reveal-id="dsy003-F6" data-open="dsy003-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). Using the branch-site model, we showed that duplicated <em>SCC3</em> copies were under positive selection (<a reveal-id="dsy003-F6" data-open="dsy003-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). The average pairwise identity between the amino acid sequences of the <em>SCC3</em> genes in the high-altitude <em>E. heterophyllum</em> is only 61.1%, indicating that the duplication of this gene may have occurred very early. Intriguingly, although only one intact <em>SCC3</em> gene was found in the low-altitude <em>E. yunnanense</em>, multiple <em>SCC3</em> fragments were identified (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10</a></span>). Of these, one fragment was found within gene pairs that are collinear between the two <em>Eutrema</em> species (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>). These findings suggest that duplication of <em>SCC3</em> arose before the divergence of these species and subsequent pseudogenization may have occurred in <em>E. yunnanense</em>. </p>                        <a id="118057223" scrollto-destination="118057223"></a>
<div id="dsy003-F6" data-id="dsy003-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/m_dsy003f6.png?Expires=2147483647&Signature=0tURfOe9AEDfmBcOHYVqiFKW9pmYWvweipf78Toksb5U5OmQolM1G2IldvQVqN8hOB9cWLBJ1U1u3XV6VHHSI0isBL0VeHD26RvzyOhREOk0aoAQCuPxGVVmlFiJQiXghshuZ4iI~3B~xBDo3u-pOPYZYt9bNQUPHmagsW4fwHBip6PGDw6bpLHtgBFpzB8w4o1gFZjZm4ZdZu30qL616UcDXgo8nAke5rKmyKQnjAP-pbd-sjwpcBPn0wGRo~W4Ou1cJR9uejC5XaT3YO8yFHV0LT4eKOvc1i~ikGnSiXCnzL77SlCeogMfoN1nOflZDS6nmb1fqRmqIN8YjH5iWw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic relationships and positive selection of SCC3 genes in Brassicaceae species. Values above branches indicates the support values obtained from 500 bootstrap replicates. Scale bar represents the number of substitutions per site. The branches tested in branch-sites tests of selection for the EhSCC3 and AlSCC3 genes are indicated. The inset table shows the results of the branch-sites tests, with a P-value based on 1 degrees of freedom. ln L, log-likelihood values; H0, null model; H1, alternative model; χ2, chi-square test likelihood ratio (2Δln L)." path-from-xml="dsy003f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057223/dsy003f6.tif" path-from-xml="dsy003f6.tif" target="_blank">View large</a><a section="118057223" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003f6.png?Expires=2147483647&Signature=xiAXt9cmLD3XvcUQIt3Hw5dhKGrfHqqDVAd5WwAwdUhVbL79-U~o25Sh7UtCBVNISEWHojhcvPn1XyD7sj9FQKA2sj6-xtPw0vYCCAX3Zyc6WY0WM4ygvPqwJofuOmlJc-ED1dUtXidPTDoLr-Hba7r35IbcpLEIACEel3WEqgMxCvkprfCbEhXklyAXk0a88IFHeQRkxq5h-08PXqmlIcxedbo9gu-rf6eQZTQvk~qcbdqDOUntp0vcxk3p2dfejGxacwXrZH16SLTikF1Or7XbBv1kRT6U-~T4YJsSbmuPHhnpOJWTa2qoub5xSMgqEYJASDvj1TbkDCxm~0RQpw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057223&ar=4831046&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy003f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic relationships and positive selection of <em>SCC3</em> genes in Brassicaceae species. Values above branches indicates the support values obtained from 500 bootstrap replicates. Scale bar represents the number of substitutions per site. The branches tested in branch-sites tests of selection for the <em>EhSCC3</em> and <em>AlSCC3</em> genes are indicated. The inset table shows the results of the branch-sites tests, with a <em>P</em>-value based on 1 degrees of freedom. ln <em>L</em>, log-likelihood values; H0, null model; H1, alternative model; χ<sup>2</sup>, chi-square test likelihood ratio (2Δln <em>L</em>).</p></div></div><div content-id="dsy003-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/m_dsy003f6.png?Expires=2147483647&Signature=0tURfOe9AEDfmBcOHYVqiFKW9pmYWvweipf78Toksb5U5OmQolM1G2IldvQVqN8hOB9cWLBJ1U1u3XV6VHHSI0isBL0VeHD26RvzyOhREOk0aoAQCuPxGVVmlFiJQiXghshuZ4iI~3B~xBDo3u-pOPYZYt9bNQUPHmagsW4fwHBip6PGDw6bpLHtgBFpzB8w4o1gFZjZm4ZdZu30qL616UcDXgo8nAke5rKmyKQnjAP-pbd-sjwpcBPn0wGRo~W4Ou1cJR9uejC5XaT3YO8yFHV0LT4eKOvc1i~ikGnSiXCnzL77SlCeogMfoN1nOflZDS6nmb1fqRmqIN8YjH5iWw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic relationships and positive selection of SCC3 genes in Brassicaceae species. Values above branches indicates the support values obtained from 500 bootstrap replicates. Scale bar represents the number of substitutions per site. The branches tested in branch-sites tests of selection for the EhSCC3 and AlSCC3 genes are indicated. The inset table shows the results of the branch-sites tests, with a P-value based on 1 degrees of freedom. ln L, log-likelihood values; H0, null model; H1, alternative model; χ2, chi-square test likelihood ratio (2Δln L)." path-from-xml="dsy003f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118057223/dsy003f6.tif" path-from-xml="dsy003f6.tif" target="_blank">View large</a><a section="118057223" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003f6.png?Expires=2147483647&Signature=xiAXt9cmLD3XvcUQIt3Hw5dhKGrfHqqDVAd5WwAwdUhVbL79-U~o25Sh7UtCBVNISEWHojhcvPn1XyD7sj9FQKA2sj6-xtPw0vYCCAX3Zyc6WY0WM4ygvPqwJofuOmlJc-ED1dUtXidPTDoLr-Hba7r35IbcpLEIACEel3WEqgMxCvkprfCbEhXklyAXk0a88IFHeQRkxq5h-08PXqmlIcxedbo9gu-rf6eQZTQvk~qcbdqDOUntp0vcxk3p2dfejGxacwXrZH16SLTikF1Or7XbBv1kRT6U-~T4YJsSbmuPHhnpOJWTa2qoub5xSMgqEYJASDvj1TbkDCxm~0RQpw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118057223&ar=4831046&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy003f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic relationships and positive selection of <em>SCC3</em> genes in Brassicaceae species. Values above branches indicates the support values obtained from 500 bootstrap replicates. Scale bar represents the number of substitutions per site. The branches tested in branch-sites tests of selection for the <em>EhSCC3</em> and <em>AlSCC3</em> genes are indicated. The inset table shows the results of the branch-sites tests, with a <em>P</em>-value based on 1 degrees of freedom. ln <em>L</em>, log-likelihood values; H0, null model; H1, alternative model; χ<sup>2</sup>, chi-square test likelihood ratio (2Δln <em>L</em>).</p></div></div><p>Three genes, <em>ICE2</em>, <em>FAB1</em> and <em>VIN3</em>, involved in cold stress was also observed to be duplicated in the high-altitude <em>E. heterophyllum</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). <em>ICE2</em> encodes a transcription factor that confers rapid freezing tolerance in <em>A. thaliana</em>.<a reveal-id="dsy003-B61" data-open="dsy003-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> The product of <em>FAB1</em> is involved in membrane lipid synthesis and photosynthesis collapses in the <em>Arabidopsis fab1</em> mutant due to degradation of chloroplasts when plants are exposed to low temperature for long periods.<a reveal-id="dsy003-B62" data-open="dsy003-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a><sup>,</sup><a reveal-id="dsy003-B63" data-open="dsy003-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a><sup>,</sup><em>VIN3</em>, which is induced by prolonged winter cold, is essential for the repression of <em>FLC</em> during vernalization.<a reveal-id="dsy003-B64" data-open="dsy003-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a></p>                            <h2 scrollto-destination=118057225 id="118057225" class="section-title" >4. Discussion</h2>
<p>In this study, we performed <em>de novo</em> genome assemblies for <em>E. heterophyllum</em> and <em>E. yunnanense</em>, a pair of diploid perennials, which have diverged recently but have contrasting altitude preferences. Compared with previously published Brassicaceae genomes that were assembled from Illumina short reads,<a reveal-id="dsy003-B8" data-open="dsy003-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a><sup>,</sup><a reveal-id="dsy003-B65" data-open="dsy003-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> our genome assemblies were of high quality in terms of continuity and gene coverage, making them suitable for subsequent comparative genomic analysis and thus representing important genetic resources. Phylogenomic analysis confirmed the close relationship and the diploid nature of these two <em>Eutrema</em> species. However, the contrasting demographic histories of the two species may reflect their different responses to the local climatic oscillations that occurred during the Quaternary. The dramatic genomic differences observed here may have partly contributed to the high- or low-altitude adaptations that they exhibit.</p><p>The NLRs are among the most variable genes across plant species. Variation in these genes between high- and low-altitude <em>A. thaliana</em> populations was discovered recently.<a reveal-id="dsy003-B66" data-open="dsy003-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> We found an increase in the number of gene copies for some NLR groups in <em>E. heterophyllum</em> and other alpine species of the family Brassicaceae. In addition, we identified a fused domain encoding PP2 among these expanded NLR genes and confirmed the independent acquisition of this domain in different species. According to a recent comprehensive study of fused domains in plant species, this domain also exists in NLRs of Malvaceae and Poaceae species.<a reveal-id="dsy003-B56" data-open="dsy003-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> Thus members of the PP2 family have fused with NLR genes multiple times in different angiosperm lineages. Further studies are needed to investigate whether and how plant immune receptors are involved in high-altitude adaptation in other alpine species.</p><p>In the high-altitude <em>E. heterophyllum</em>, we found that many genes related to reproduction and DDR had undergone duplications, probably in response to abiotic stress. It should be noted that most of these reproduction-related genes remain as single copies in most angiosperms.<a reveal-id="dsy003-B67" data-open="dsy003-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> In addition, two genes in this category (<em>SCC3</em> and <em>SMC3</em>) that have undergone duplication events encode cohesin subunits, and similar copy number variations have also been found in animals, where they may cause distinct functional changes in the cohesin complex.<a reveal-id="dsy003-B68 dsy003-B69 dsy003-B70 dsy003-B71" data-open="dsy003-B68 dsy003-B69 dsy003-B70 dsy003-B71" class="link link-ref link-reveal xref-bibr"><sup>68–71</sup></a> To the best of our knowledge, this is the first report of duplication and possible subfunctionalization of cohesin subunits in plants. We have further shown that duplications of <em>SCC3</em> in <em>Eutrema</em> species occurred early, before their divergence. Although we cannot exclude the possibility that the apparent multiple partial <em>SCC3</em> sequences identified in <em>E. yunnanense</em> may be due to the fragmented nature of the draft genome, the evidence for pseudogenization in this species is strong. The duplication, followed by subsequent loss, of <em>SCC3</em> in <em>E. yunnanense</em> may be correlated with its initial occurrence (or its ancestral distribution) in the high-altitude QTP and subsequent migration to the lower-altitude region.<a reveal-id="dsy003-B12" data-open="dsy003-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> As the cohesin complex has a profound impact on reproduction as well as on gene regulation, duplication of these subunits may have played an important role in plant adaptation to harsh alpine environments. In addition, duplication of genes related to cold acclimation in the high-altitude <em>E. heterophyllum</em> may be correlated with plant adaptation to the low and rapidly changing temperatures experienced in alpine habitats.</p><p>In summary, we assembled <em>de novo</em> genomes of two <em>Eutrema</em> species and inferred genetic changes in the alpine species that may underlie high-altitude adaptation, although further tests in more species using multiple approaches are needed. The genome resources that we have developed for the two <em>Eutrema</em> species will be very useful for many studies in this genus, the Brassicaceae as a whole, and alpine plants in general. </p>                            <h2 scrollto-destination=118057230 id="118057230" class="section-title" >Availability</h2>
<p>The short reads of Illumina DNAseq and RNAseq generated in this study are available from the National Center for Biotechnology Information (NCBI) Short Read Archive (SRA) under BioProject PRJNA419026 (BioSample SAMN08200758 for <em>E. heterophyllum</em> and SAMN08200758 for <em>E. yunnanense</em>). The genome assemblies of this project have been deposited at DDBJ/ENA/GenBank under the accession PKMM00000000 for <em>E. heterophyllum</em> and PKML00000000 for <em>E. yunnanense</em>.</p>                            <h2 scrollto-destination=118057232 id="118057232" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Bingbing Liu for his help in DNA extraction.</p>                            <h3 scrollto-destination=118057234 id="118057234" class="section-title" >Funding</h3>
<p>This work was supported by grants from the National Key Research and Development program (2017YFC0505203), National Natural Science Foundation of China (31590821), National Key Project for Basic Research (2014CB954100), Youth Science and Technology Innovation Team of Sichuan Province (2014TD003), and the 985 and 211 Projects of Sichuan University.</p>                            <h3 scrollto-destination=118057236 id="118057236" class="section-title" >Conflict of interest</h3>
<p>The authors declare that they have no competing interests.</p>                            <h3 scrollto-destination=118057238 id="118057238" class="section-title" >Authors’ contributions</h3>
<p>J.L. conceived the study. G.H. and X.G. collected samples. X.G. and Q.H. assembled the genomes and performed genome annotation and evolutionary analyses with the help of X.W, D.Z. and T.M. X.G. and J.L. wrote the manuscript. All authors read and approved the final manuscript.</p>                            <h3 scrollto-destination=118057240 id="118057240" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h2 scrollto-destination=118057242 id="118057242" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsy003-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hughes</div> <div class="given-names">C. E.</div></div>, <div class="name string-name"><div class="surname">Atchison</div> <div class="given-names">G. W.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">The ubiquity of alpine plant radiations: from the Andes to the Hengduan Mountains</div>, <div class="source">New Phytol</div> ., <div class="volume">207</div>, <div class="fpage">275</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20ubiquity%20of%20alpine%20plant%20radiations%3A%20from%20the%20Andes%20to%20the%20Hengduan%20Mountains&amp;author=C.%20E.%20Hughes&amp;author=G.%20W.%20Atchison&amp;publication_year=2015&amp;journal=New%20Phytol&amp;volume=207&amp;pages=275-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.2015.207.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.2015.207.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.2015.207.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25605002" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Beall</div> <div class="given-names">C. M.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Adaptation to high altitude: phenotypes and genotypes</div>, <div class="source">Annu. Rev. Anthropol</div> ., <div class="volume">43</div>, <div class="fpage">251</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adaptation%20to%20high%20altitude%3A%20phenotypes%20and%20genotypes&amp;author=C.%20M.%20Beall&amp;publication_year=2014&amp;journal=Annu.%20Rev.%20Anthropol&amp;volume=43&amp;pages=251-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-anthro-102313-030000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-anthro-102313-030000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-anthro-102313-030000"> </span></div> </div></div></div></div></div><div content-id="dsy003-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Tian</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Yan</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Genome of plant maca (<em>Lepidium meyenii</em>) illuminates genomic basis for high-altitude adaptation in the central Andes</div>, <div class="source">Mol. Plant</div> ., <div class="volume">9</div>, <div class="fpage">1066</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20of%20plant%20maca%20(Lepidium%20meyenii)%20illuminates%20genomic%20basis%20for%20high-altitude%20adaptation%20in%20the%20central%20Andes&amp;author=J.%20Zhang&amp;author=Y.%20Tian&amp;author=L.%20Yan&amp;publication_year=2016&amp;journal=Mol.%20Plant&amp;volume=9&amp;pages=1066-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molp.2016.04.016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molp.2016.04.016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molp.2016.04.016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27174404" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bailey</div> <div class="given-names">C. D.</div></div>, <div class="name string-name"><div class="surname">Koch</div> <div class="given-names">M. A.</div></div>, <div class="name string-name"><div class="surname">Mayer</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">Toward a global phylogeny of the Brassicaceae</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">23</div>, <div class="fpage">2142</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Toward%20a%20global%20phylogeny%20of%20the%20Brassicaceae&amp;author=C.%20D.%20Bailey&amp;author=M.%20A.%20Koch&amp;author=M.%20Mayer&amp;publication_year=2006&amp;journal=Mol.%20Biol.%20Evol&amp;volume=23&amp;pages=2142-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msl087" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsl087" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsl087"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16916944" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Koch</div> <div class="given-names">M. A.</div></div>, <div class="name string-name"><div class="surname">Dobeš</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Kiefer</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Schmickl</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Klimeš</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Lysak</div> <div class="given-names">M. A.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Supernetwork identifies multiple events of plastid trn F (GAA) pseudogene evolution in the Brassicaceae</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">24</div>, <div class="fpage">63</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Supernetwork%20identifies%20multiple%20events%20of%20plastid%20trn%20F%20(GAA)%20pseudogene%20evolution%20in%20the%20Brassicaceae&amp;author=M.%20A.%20Koch&amp;author=C.%20Dobe%C5%A1&amp;author=C.%20Kiefer&amp;author=R.%20Schmickl&amp;author=L.%20Klime%C5%A1&amp;author=M.%20A.%20Lysak&amp;publication_year=2006&amp;journal=Mol.%20Biol.%20Evol&amp;volume=24&amp;pages=63-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msl130" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsl130" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsl130"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16987951" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Koenig</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Weigel</div> <div class="given-names">D.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Beyond the thale: comparative genomics and genetics of Arabidopsis relatives</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">16</div>, <div class="fpage">285</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Beyond%20the%20thale%3A%20comparative%20genomics%20and%20genetics%20of%20Arabidopsis%20relatives&amp;author=D.%20Koenig&amp;author=D.%20Weigel&amp;publication_year=2015&amp;journal=Nat.%20Rev.%20Genet&amp;volume=16&amp;pages=285" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3883" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3883" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3883"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25854181" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nikolov</div> <div class="given-names">L. A.</div></div>, <div class="name string-name"><div class="surname">Tsiantis</div> <div class="given-names">M.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Using mustard genomes to explore the genetic basis of evolutionary change</div>, <div class="source">Curr. Opin. Plant Biol</div> ., <div class="volume">36</div>, <div class="fpage">119</div>–<div class="lpage">28</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20mustard%20genomes%20to%20explore%20the%20genetic%20basis%20of%20evolutionary%20change&amp;author=L.%20A.%20Nikolov&amp;author=M.%20Tsiantis&amp;publication_year=2017&amp;journal=Curr.%20Opin.%20Plant%20Biol&amp;volume=36&amp;pages=119-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2017.02.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2017.02.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2017.02.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28285128" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gan</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Hay</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Kwantes</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">The <em>Cardamine hirsuta</em> genome offers insight into the evolution of morphological diversity</div>, <div class="source">Nat. Plants</div> , <div class="volume">2</div>, <div class="fpage">16167</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Cardamine%20hirsuta%20genome%20offers%20insight%20into%20the%20evolution%20of%20morphological%20diversity&amp;author=X.%20Gan&amp;author=A.%20Hay&amp;author=M.%20Kwantes&amp;publication_year=2016&amp;journal=Nat.%20Plants&amp;volume=2&amp;pages=16167" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nplants.2016.167" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnplants.2016.167" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnplants.2016.167"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27797353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jiao</div> <div class="given-names">W. B.</div></div>, <div class="name string-name"><div class="surname">Accinelli</div> <div class="given-names">G. G.</div></div>, <div class="name string-name"><div class="surname">Hartwig</div> <div class="given-names">B.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Improving and correcting the contiguity of long-read genome assemblies of three plant species using optical mapping and chromosome conformation capture data</div>, <div class="source">Genome Res</div> ., <div class="volume">27</div>, <div class="fpage">778</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improving%20and%20correcting%20the%20contiguity%20of%20long-read%20genome%20assemblies%20of%20three%20plant%20species%20using%20optical%20mapping%20and%20chromosome%20conformation%20capture%20data&amp;author=W.%20B.%20Jiao&amp;author=G.%20G.%20Accinelli&amp;author=B.%20Hartwig&amp;publication_year=2017&amp;journal=Genome%20Res&amp;volume=27&amp;pages=778-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.213652.116" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.213652.116" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.213652.116"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28159771" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Willing</div> <div class="given-names">E. M.</div></div>, <div class="name string-name"><div class="surname">Rawat</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Mandáková</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Genome expansion of <em>Arabis alpina</em> linked with retrotransposition and reduced symmetric DNA methylation</div>, <div class="source">Nat. Plants</div> , <div class="volume">1</div>, nplants201423.</div></div></div></div><div content-id="dsy003-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Al-Shehbaz</div> <div class="given-names">I. A.</div></div>, <div class="name string-name"><div class="surname">Warwick</div> <div class="given-names">S. I.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">A synopsis of <em>Eutrema</em> (Brassicaceae)</div>, <div class="source">Harvard Pap. Bot</div> ., <div class="volume">10</div>, <div class="fpage">129</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20synopsis%20of%20Eutrema%20(Brassicaceae)&amp;author=I.%20A.%20Al-Shehbaz&amp;author=S.%20I.%20Warwick&amp;publication_year=2005&amp;journal=Harvard%20Pap.%20Bot&amp;volume=10&amp;pages=129-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3100/1043-4534(2005)10[129:ASOEB]2.0.CO;2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3100%2F1043-4534(2005)10[129:ASOEB]2.0.CO;2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3100%2F1043-4534(2005)10[129:ASOEB]2.0.CO;2"> </span></div> </div></div></div></div></div><div content-id="dsy003-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hao</div> <div class="given-names">G. Q.</div></div>, <div class="name string-name"><div class="surname">Al-Shehba</div> <div class="given-names">I. A.</div></div>, <div class="name string-name"><div class="surname">Ahani</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">An integrative study of evolutionary diversification of <em>Eutrema</em> (Eutremeae, Brassicaceae)</div>, <div class="source">Bot. J. Linn. Soc</div> ., <div class="volume">184</div>, <div class="fpage">204</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrative%20study%20of%20evolutionary%20diversification%20of%20Eutrema%20(Eutremeae%2C%20Brassicaceae)&amp;author=G.%20Q.%20Hao&amp;author=I.%20A.%20Al-Shehba&amp;author=H.%20Ahani&amp;publication_year=2017&amp;journal=Bot.%20J.%20Linn.%20Soc&amp;volume=184&amp;pages=204-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/botlinnean/box024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbotlinnean%2Fbox024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbotlinnean%2Fbox024"> </span></div> </div></div></div></div></div><div content-id="dsy003-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bolger</div> <div class="given-names">A. M.</div></div>, <div class="name string-name"><div class="surname">Lohse</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Usadel</div> <div class="given-names">B.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Trimmomatic: a flexible trimmer for Illumina sequence data</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">2114</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;author=A.%20M.%20Bolger&amp;author=M.%20Lohse&amp;author=B.%20Usadel&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=2114-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu170" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu170" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu170"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24695404" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">BFC: correcting Illumina sequencing errors</div>, <div class="source">Bioinformatics</div> , <div class="volume">31</div>, <div class="fpage">2885</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BFC%3A%20correcting%20Illumina%20sequencing%20errors&amp;author=H.%20Li&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=2885-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btv290" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtv290" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtv290"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25953801" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xu</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Luo</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Qian</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">FastUniq: a fast de novo duplicates removal tool for paired short reads</div>, <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e52249</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=FastUniq%3A%20a%20fast%20de%20novo%20duplicates%20removal%20tool%20for%20paired%20short%20reads&amp;author=H.%20Xu&amp;author=X.%20Luo&amp;author=J.%20Qian&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e52249" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0052249" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0052249" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0052249"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23284954" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Luo</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Xie</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">SOAPdenovo2: an empirically improved memory-efficient short-read de novo assembler</div>, <div class="source">Gigascience</div> , <div class="volume">1</div>, <div class="fpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SOAPdenovo2%3A%20an%20empirically%20improved%20memory-efficient%20short-read%20de%20novo%20assembler&amp;author=R.%20Luo&amp;author=B.%20Liu&amp;author=Y.%20Xie&amp;publication_year=2012&amp;journal=Gigascience&amp;volume=1&amp;pages=18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/2047-217X-1-18" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F2047-217X-1-18" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F2047-217X-1-18"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23587118" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kajitani</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Toshimoto</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Noguchi</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads</div>, <div class="source">Genome Res</div> ., <div class="volume">24</div>, <div class="fpage">1384</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20de%20novo%20assembly%20of%20highly%20heterozygous%20genomes%20from%20whole-genome%20shotgun%20short%20reads&amp;author=R.%20Kajitani&amp;author=K.%20Toshimoto&amp;author=H.%20Noguchi&amp;publication_year=2014&amp;journal=Genome%20Res&amp;volume=24&amp;pages=1384-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.170720.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.170720.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.170720.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24755901" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Parra</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Bradnam</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Korf</div> <div class="given-names">I.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes</div>, <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">1061</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CEGMA%3A%20a%20pipeline%20to%20accurately%20annotate%20core%20genes%20in%20eukaryotic%20genomes&amp;author=G.%20Parra&amp;author=K.%20Bradnam&amp;author=I.%20Korf&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=1061-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm071" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm071" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm071"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17332020" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Simao</div> <div class="given-names">F. A.</div></div>, <div class="name string-name"><div class="surname">Waterhouse</div> <div class="given-names">R. M.</div></div>, <div class="name string-name"><div class="surname">Ioannidis</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Kriventseva</div> <div class="given-names">E. V.</div></div>, <div class="name string-name"><div class="surname">Zdobnov</div> <div class="given-names">E. M.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs</div>, <div class="source">Bioinformatics</div> , <div class="volume">31</div>, <div class="fpage">3210</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BUSCO%3A%20assessing%20genome%20assembly%20and%20annotation%20completeness%20with%20single-copy%20orthologs&amp;author=F.%20A.%20Simao&amp;author=R.%20M.%20Waterhouse&amp;author=P.%20Ioannidis&amp;author=E.%20V.%20Kriventseva&amp;author=E.%20M.%20Zdobnov&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=3210-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btv351" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtv351" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtv351"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26059717" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Flutre</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Duprat</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Feuillet</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Quesneville</div> <div class="given-names">H.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Considering transposable element diversification in de novo annotation approaches</div>, <div class="source">PLoS One</div> , <div class="volume">6</div>, <div class="fpage">e16526</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Considering%20transposable%20element%20diversification%20in%20de%20novo%20annotation%20approaches&amp;author=T.%20Flutre&amp;author=E.%20Duprat&amp;author=C.%20Feuillet&amp;author=H.%20Quesneville&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e16526" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0016526" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0016526" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0016526"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21304975" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Quesneville</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Bergman</div> <div class="given-names">C. M.</div></div>, <div class="name string-name"><div class="surname">Andrieu</div> <div class="given-names">O.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">Combined evidence annotation of transposable elements in genome sequences</div>, <div class="source">PLoS Comput. Biol</div> ., <div class="volume">1</div>, <div class="fpage">166</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Combined%20evidence%20annotation%20of%20transposable%20elements%20in%20genome%20sequences&amp;author=H.%20Quesneville&amp;author=C.%20M.%20Bergman&amp;author=O.%20Andrieu&amp;publication_year=2005&amp;journal=PLoS%20Comput.%20Biol&amp;volume=1&amp;pages=166-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.0010022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.0010022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.0010022"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16110336" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ellinghaus</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Kurtz</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Willhoeft</div> <div class="given-names">U.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">LTRharvest, a efficient and flexible software for <em>de novo</em> detection of LTR retrotransposons</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">9</div>, <div class="fpage">18</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LTRharvest%2C%20a%20efficient%20and%20flexible%20software%20for%20de%20novo%20detection%20of%20LTR%20retrotransposons&amp;author=D.%20Ellinghaus&amp;author=S.%20Kurtz&amp;author=U.%20Willhoeft&amp;publication_year=2008&amp;journal=BMC%20Bioinformatics&amp;volume=9&amp;pages=18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-18" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-18" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-9-18"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18194517" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bao</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S. R.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Automated <em>de novo</em> identification of repeat sequence families in sequenced genomes</div>, <div class="source">Genome Res</div> ., <div class="volume">12</div>, <div class="fpage">1269</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20de%20novo%20identification%20of%20repeat%20sequence%20families%20in%20sequenced%20genomes&amp;author=Z.%20Bao&amp;author=S.%20R.%20Eddy&amp;publication_year=2002&amp;journal=Genome%20Res&amp;volume=12&amp;pages=1269-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.88502" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.88502" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.88502"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12176934" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Edgar</div> <div class="given-names">R. C.</div></div>, <div class="name string-name"><div class="surname">Myers</div> <div class="given-names">E. W.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">PILER: identification and classification of genomic repeats</div>, <div class="source">Bioinformatics</div> , <div class="volume">21(suppl_1)</div>, <div class="fpage">i152</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PILER%3A%20identification%20and%20classification%20of%20genomic%20repeats&amp;author=R.%20C.%20Edgar&amp;author=E.%20W.%20Myers&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21(suppl_1)&amp;pages=i152-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti1003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti1003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti1003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15961452" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huang</div> <div class="given-names">X.</div></div></div> <div class="year">1994</div>, 
<div class="article-title">On global sequence alignment</div>, <div class="source">Comput. Appl. Biosci</div> ., <div class="volume">10</div>, <div class="fpage">227</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On%20global%20sequence%20alignment&amp;author=X.%20Huang&amp;publication_year=1994&amp;journal=Comput.%20Appl.%20Biosci&amp;volume=10&amp;pages=227-35" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7922677" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bao</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Kojima</div> <div class="given-names">K. K.</div></div>, <div class="name string-name"><div class="surname">Kohany</div> <div class="given-names">O.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Repbase Update, a database of repetitive elements in eukaryotic genomes</div>, <div class="source">Mob. DNA</div> , <div class="volume">6</div>, <div class="fpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repbase%20Update%2C%20a%20database%20of%20repetitive%20elements%20in%20eukaryotic%20genomes&amp;author=W.%20Bao&amp;author=K.%20K.%20Kojima&amp;author=O.%20Kohany&amp;publication_year=2015&amp;journal=Mob.%20DNA&amp;volume=6&amp;pages=11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13100-015-0041-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13100-015-0041-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13100-015-0041-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26045719" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Finn</div> <div class="given-names">R. D.</div></div>, <div class="name string-name"><div class="surname">Coggill</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Eberhardt</div> <div class="given-names">R. Y.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">The Pfam protein families database: towards a more sustainable future</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">44</div>, <div class="fpage">D279</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Pfam%20protein%20families%20database%3A%20towards%20a%20more%20sustainable%20future&amp;author=R.%20D.%20Finn&amp;author=P.%20Coggill&amp;author=R.%20Y.%20Eberhardt&amp;publication_year=2016&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;pages=D279-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv1344" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv1344" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv1344"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26673716" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S. R.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">A probabilistic model of local sequence alignment that simplifies statistical significance estimation</div>, <div class="source">PLoS Comp. Biol</div> ., <div class="volume">4</div>, <div class="fpage">e1000069</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20probabilistic%20model%20of%20local%20sequence%20alignment%20that%20simplifies%20statistical%20significance%20estimation&amp;author=S.%20R.%20Eddy&amp;publication_year=2008&amp;journal=PLoS%20Comp.%20Biol&amp;volume=4&amp;pages=e1000069" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000069" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1000069" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1000069"> </span></div> </div></div></div></div></div><div content-id="dsy003-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Maumus</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Quesneville</div> <div class="given-names">H.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Ancestral repeats have shaped epigenome and genome composition for millions of years in <em>Arabidopsis thaliana</em></div>, <div class="source">Nat. Commun</div> ., <div class="volume">5</div>, <div class="fpage">4104</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ancestral%20repeats%20have%20shaped%20epigenome%20and%20genome%20composition%20for%20millions%20of%20years%20in%20Arabidopsis%20thaliana&amp;author=F.%20Maumus&amp;author=H.%20Quesneville&amp;publication_year=2014&amp;journal=Nat.%20Commun&amp;volume=5&amp;pages=4104" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms5104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms5104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms5104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24954583" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Haas</div> <div class="given-names">B. J.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">W.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">Automated eukaryotic gene structure annotation using EVidenceModeler and the Program to Assemble Spliced Alignments</div>, <div class="source">Genome Biol</div> ., <div class="volume">9</div>, <div class="fpage">R7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20eukaryotic%20gene%20structure%20annotation%20using%20EVidenceModeler%20and%20the%20Program%20to%20Assemble%20Spliced%20Alignments&amp;author=B.%20J.%20Haas&amp;author=S.%20L.%20Salzberg&amp;author=W.%20Zhu&amp;publication_year=2008&amp;journal=Genome%20Biol&amp;volume=9&amp;pages=R7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2008-9-1-r7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2008-9-1-r7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2008-9-1-r7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18190707" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Grabherr</div> <div class="given-names">M. G.</div></div>, <div class="name string-name"><div class="surname">Haas</div> <div class="given-names">B. J.</div></div>, <div class="name string-name"><div class="surname">Yassour</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">29</div>, <div class="fpage">644</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Full-length%20transcriptome%20assembly%20from%20RNA-Seq%20data%20without%20a%20reference%20genome&amp;author=M.%20G.%20Grabherr&amp;author=B.%20J.%20Haas&amp;author=M.%20Yassour&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol&amp;volume=29&amp;pages=644-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1883" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1883" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1883"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pertea</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Pertea</div> <div class="given-names">G. M.</div></div>, <div class="name string-name"><div class="surname">Antonescu</div> <div class="given-names">C. M.</div></div>, <div class="name string-name"><div class="surname">Chang</div> <div class="given-names">T. C.</div></div>, <div class="name string-name"><div class="surname">Mendell</div> <div class="given-names">J. T.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">StringTie enables improved reconstruction of a transcriptome from RNA-seq reads</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">33</div>, <div class="fpage">290</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=StringTie%20enables%20improved%20reconstruction%20of%20a%20transcriptome%20from%20RNA-seq%20reads&amp;author=M.%20Pertea&amp;author=G.%20M.%20Pertea&amp;author=C.%20M.%20Antonescu&amp;author=T.%20C.%20Chang&amp;author=J.%20T.%20Mendell&amp;author=S.%20L.%20Salzberg&amp;publication_year=2015&amp;journal=Nat.%20Biotechnol&amp;volume=33&amp;pages=290-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.3122" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.3122" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.3122"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25690850" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Diekhans</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Baertsch</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Haussler</div> <div class="given-names">D.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Using native and syntenically mapped cDNA alignments to improve <em>de novo</em> gene finding</div>, <div class="source">Bioinformatics</div> , <div class="volume">24</div>, <div class="fpage">637</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20native%20and%20syntenically%20mapped%20cDNA%20alignments%20to%20improve%20de%20novo%20gene%20finding&amp;author=M.%20Stanke&amp;author=M.%20Diekhans&amp;author=R.%20Baertsch&amp;author=D.%20Haussler&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=637-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18218656" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Majoros</div> <div class="given-names">W. H.</div></div>, <div class="name string-name"><div class="surname">Pertea</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">TigrScan and GlimmerHMM: two open source ab initio eukaryotic gene-finders</div>, <div class="source">Bioinformatics</div> , <div class="volume">20</div>, <div class="fpage">2878</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TigrScan%20and%20GlimmerHMM%3A%20two%20open%20source%20ab%20initio%20eukaryotic%20gene-finders&amp;author=W.%20H.%20Majoros&amp;author=M.%20Pertea&amp;author=S.%20L.%20Salzberg&amp;publication_year=2004&amp;journal=Bioinformatics&amp;volume=20&amp;pages=2878-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bth315" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbth315" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbth315"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15145805" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Jarvis</div> <div class="given-names">D. E.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">The reference genome of the halophytic plant <em>Eutrema salsugineum</em></div>, <div class="source">Front. Plant Sci</div> ., <div class="volume">4</div>, <div class="fpage">1</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20reference%20genome%20of%20the%20halophytic%20plant%20Eutrema%20salsugineum&amp;author=R.%20Yang&amp;author=D.%20E.%20Jarvis&amp;author=H.%20Chen&amp;publication_year=2013&amp;journal=Front.%20Plant%20Sci&amp;volume=4&amp;pages=1-14" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23346092" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Suzek</div> <div class="given-names">B. E.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">McGarvey</div> <div class="given-names">P. B.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">C. H.</div></div>, <div class="name string-name"><div class="surname">UniProt</div> <div class="given-names">C.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">UniRef clusters: a comprehensive and scalable alternative for improving sequence similarity searches</div>, <div class="source">Bioinformatics</div> , <div class="volume">31</div>, <div class="fpage">926</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=UniRef%20clusters%3A%20a%20comprehensive%20and%20scalable%20alternative%20for%20improving%20sequence%20similarity%20searches&amp;author=B.%20E.%20Suzek&amp;author=Y.%20Wang&amp;author=H.%20Huang&amp;author=P.%20B.%20McGarvey&amp;author=C.%20H.%20Wu&amp;author=C.%20UniProt&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=926-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu739" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu739" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu739"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25398609" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Iwata</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Gotoh</div> <div class="given-names">O.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Benchmarking spliced alignment programs including Spaln2, an extended version of Spaln that incorporates additional species-specific features</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">e161</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Benchmarking%20spliced%20alignment%20programs%20including%20Spaln2%2C%20an%20extended%20version%20of%20Spaln%20that%20incorporates%20additional%20species-specific%20features&amp;author=H.%20Iwata&amp;author=O.%20Gotoh&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=e161" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks708" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks708" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks708"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22848105" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Camacho</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Coulouris</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Avagyan</div> <div class="given-names">V.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">BLAST+: architecture and applications</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">10</div>, <div class="fpage">421</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLAST%2B%3A%20architecture%20and%20applications&amp;author=C.%20Camacho&amp;author=G.%20Coulouris&amp;author=V.%20Avagyan&amp;publication_year=2009&amp;journal=BMC%20Bioinformatics&amp;volume=10&amp;pages=421" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-10-421" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-10-421" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-10-421"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20003500" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">UniProt</div> <div class="given-names">C.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">UniProt: a hub for protein information</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">D204</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=UniProt%3A%20a%20hub%20for%20protein%20information&amp;author=C.%20UniProt&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=D204-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku989" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku989" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku989"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25348405" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lamesch</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Berardini</div> <div class="given-names">T. Z.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">The Arabidopsis Information Resource (TAIR): improved gene annotation and new tools</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">D1202</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Arabidopsis%20Information%20Resource%20(TAIR)%3A%20improved%20gene%20annotation%20and%20new%20tools&amp;author=P.%20Lamesch&amp;author=T.%20Z.%20Berardini&amp;author=D.%20Li&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=D1202-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1090" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1090" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1090"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22140109" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jones</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Binns</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Chang</div> <div class="given-names">H. Y.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">InterProScan 5: genome-scale protein function classification</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">1236</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterProScan%205%3A%20genome-scale%20protein%20function%20classification&amp;author=P.%20Jones&amp;author=D.%20Binns&amp;author=H.%20Y.%20Chang&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=1236-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24451626" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huntley</div> <div class="given-names">R. P.</div></div>, <div class="name string-name"><div class="surname">Sawford</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Mutowo-Meullenet</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">The GOA database: gene Ontology annotation updates for 2015</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">D1057</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20GOA%20database%3A%20gene%20Ontology%20annotation%20updates%20for%202015&amp;author=R.%20P.%20Huntley&amp;author=T.%20Sawford&amp;author=P.%20Mutowo-Meullenet&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=D1057-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku1113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku1113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku1113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25378336" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Stoeckert</div> <div class="given-names">C. J.</div></div>, <div class="name string-name"><div class="surname">Roos</div> <div class="given-names">D. S.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">OrthoMCL: identification of ortholog groups for eukaryotic genomes</div>, <div class="source">Genome Res</div> ., <div class="volume">13</div>, <div class="fpage">2178</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=OrthoMCL%3A%20identification%20of%20ortholog%20groups%20for%20eukaryotic%20genomes&amp;author=L.%20Li&amp;author=C.%20J.%20Stoeckert&amp;author=D.%20S.%20Roos&amp;publication_year=2003&amp;journal=Genome%20Res&amp;volume=13&amp;pages=2178-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1224503" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1224503" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1224503"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12952885" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Han</div> <div class="given-names">M. V.</div></div>, <div class="name string-name"><div class="surname">Thomas</div> <div class="given-names">G. W.</div></div>, <div class="name string-name"><div class="surname">Lugo-Martinez</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Hahn</div> <div class="given-names">M. W.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Estimating gene gain and loss rates in the presence of error in genome assembly and annotation using CAFE 3</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">30</div>, <div class="fpage">1987</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimating%20gene%20gain%20and%20loss%20rates%20in%20the%20presence%20of%20error%20in%20genome%20assembly%20and%20annotation%20using%20CAFE%203&amp;author=M.%20V.%20Han&amp;author=G.%20W.%20Thomas&amp;author=J.%20Lugo-Martinez&amp;author=M.%20W.%20Hahn&amp;publication_year=2013&amp;journal=Mol.%20Biol.%20Evol&amp;volume=30&amp;pages=1987-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mst100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmst100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmst100"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23709260" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tian</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Yan</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">AgriGO v2.0: a GO analysis toolkit for the agricultural community, 2017 update</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">W122</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AgriGO%20v2.0%3A%20a%20GO%20analysis%20toolkit%20for%20the%20agricultural%20community%2C%202017%20update&amp;author=T.%20Tian&amp;author=Y.%20Liu&amp;author=H.%20Yan&amp;publication_year=2017&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=W122-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkx382" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkx382" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkx382"> </span></div> </div></div></div></div></div><div content-id="dsy003-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="collab">R Core Team</div>. <div class="year">2016</div>, 
<div class="article-title">R: a language and environment for statistical computing</div>. In: <div class="source">R Foundation for Statistical Computing</div> . Vienna, Austria. <a class="link link-uri" href="https://www.R-project.org/" target="_blank">https://www.R-project.org/</a>.</div></div></div></div><div content-id="dsy003-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stamatakis</div> <div class="given-names">A.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">RAxML version 8: a tool for phylogenetic analysis and post-analysis of large phylogenies</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">1312</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RAxML%20version%208%3A%20a%20tool%20for%20phylogenetic%20analysis%20and%20post-analysis%20of%20large%20phylogenies&amp;author=A.%20Stamatakis&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=1312-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24451623" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Loytynoja</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Goldman</div> <div class="given-names">N.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Phylogeny-aware gap placement prevents errors in sequence alignment and evolutionary analysis</div>, <div class="source">Science</div> , <div class="volume">320</div>, <div class="fpage">1632</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogeny-aware%20gap%20placement%20prevents%20errors%20in%20sequence%20alignment%20and%20evolutionary%20analysis&amp;author=A.%20Loytynoja&amp;author=N.%20Goldman&amp;publication_year=2008&amp;journal=Science&amp;volume=320&amp;pages=1632-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1158395" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1158395" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1158395"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18566285" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Suyama</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Torrents</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Bork</div> <div class="given-names">P.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">PAL2NAL: robust conversion of protein sequence alignments into the corresponding codon alignments</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">34</div>, <div class="fpage">W609</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PAL2NAL%3A%20robust%20conversion%20of%20protein%20sequence%20alignments%20into%20the%20corresponding%20codon%20alignments&amp;author=M.%20Suyama&amp;author=D.%20Torrents&amp;author=P.%20Bork&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=W609-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl315" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl315" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl315"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845082" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">PAML 4: phylogenetic analysis by maximum likelihood</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">24</div>, <div class="fpage">1586</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PAML%204%3A%20phylogenetic%20analysis%20by%20maximum%20likelihood&amp;author=Z.%20Yang&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol&amp;volume=24&amp;pages=1586-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm088" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm088" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm088"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17483113" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guo</div> <div class="given-names">X. Y.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">J. Q.</div></div>, <div class="name string-name"><div class="surname">Hao</div> <div class="given-names">G. Q.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Plastome phylogeny and early diversification of Brassicaceae</div>, <div class="source">BMC Genomics</div> , <div class="volume">176</div>, <div class="fpage">1</div>–<div class="lpage">9</div>.</div></div></div></div><div content-id="dsy003-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">DeBarry</div> <div class="given-names">J. D.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">MCScanX: a toolkit for detection and evolutionary analysis of gene synteny and collinearity</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">e49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MCScanX%3A%20a%20toolkit%20for%20detection%20and%20evolutionary%20analysis%20of%20gene%20synteny%20and%20collinearity&amp;author=Y.%20Wang&amp;author=H.%20Tang&amp;author=J.%20D.%20DeBarry&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=e49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1293" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1293" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1293"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22217600" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Inference of human population history from individual whole-genome sequences</div>, <div class="source">Nature</div> , <div class="volume">475</div>, <div class="fpage">493</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inference%20of%20human%20population%20history%20from%20individual%20whole-genome%20sequences&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2011&amp;journal=Nature&amp;volume=475&amp;pages=493-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10231" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10231" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10231"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21753753" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">A statistical framework for SNP calling, mutation discovery, association mapping and population genetical parameter estimation from sequencing data</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">2987</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20statistical%20framework%20for%20SNP%20calling%2C%20mutation%20discovery%2C%20association%20mapping%20and%20population%20genetical%20parameter%20estimation%20from%20sequencing%20data&amp;author=H.%20Li&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2987-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr509" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr509" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr509"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21903627" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sarris</div> <div class="given-names">P. F.</div></div>, <div class="name string-name"><div class="surname">Cevik</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Dagdas</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Jones</div> <div class="given-names">J. D.</div></div>, <div class="name string-name"><div class="surname">Krasileva</div> <div class="given-names">K. V.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Comparative analysis of plant immune receptor architectures uncovers host proteins likely targeted by pathogens</div>, <div class="source">BMC Biol</div> ., <div class="volume">14</div>, <div class="fpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20plant%20immune%20receptor%20architectures%20uncovers%20host%20proteins%20likely%20targeted%20by%20pathogens&amp;author=P.%20F.%20Sarris&amp;author=V.%20Cevik&amp;author=G.%20Dagdas&amp;author=J.%20D.%20Jones&amp;author=K.%20V.%20Krasileva&amp;publication_year=2016&amp;journal=BMC%20Biol&amp;volume=14&amp;pages=8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12915-016-0228-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12915-016-0228-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12915-016-0228-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26891798" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dinant</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Clark</div> <div class="given-names">A. M.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2003</div>, 
<div class="article-title">Diversity of the superfamily of phloem lectins (phloem protein 2) in angiosperms</div>, <div class="source">Plant Physiol</div> ., <div class="volume">131</div>, <div class="fpage">114</div>–<div class="lpage">28</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20of%20the%20superfamily%20of%20phloem%20lectins%20(phloem%20protein%202)%20in%20angiosperms&amp;author=S.%20Dinant&amp;author=A.%20M.%20Clark&amp;author=Y.%20Zhu&amp;publication_year=2003&amp;journal=Plant%20Physiol&amp;volume=131&amp;pages=114-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.013086" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.013086" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.013086"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12529520" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hu</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Cools</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">De Veylder</div> <div class="given-names">L.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Mechanisms used by plants to cope with DNA damage</div>, <div class="source">Annu. Rev. Plant Biol</div> ., <div class="volume">67</div>, <div class="fpage">439</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanisms%20used%20by%20plants%20to%20cope%20with%20DNA%20damage&amp;author=Z.%20Hu&amp;author=T.%20Cools&amp;author=L.%20De%20Veylder&amp;publication_year=2016&amp;journal=Annu.%20Rev.%20Plant%20Biol&amp;volume=67&amp;pages=439-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev-arplant-043015-111902" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev-arplant-043015-111902" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev-arplant-043015-111902"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26653616" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Adachi</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Minamisawa</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Okushima</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Programmed induction of endoreduplication by DNA double-strand breaks in Arabidopsis</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> ., <div class="volume">108</div>, <div class="fpage">10004</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Programmed%20induction%20of%20endoreduplication%20by%20DNA%20double-strand%20breaks%20in%20Arabidopsis&amp;author=S.%20Adachi&amp;author=K.%20Minamisawa&amp;author=Y.%20Okushima&amp;publication_year=2011&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=10004-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1103584108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1103584108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1103584108"> </span></div> </div></div></div></div></div><div content-id="dsy003-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schubert</div> <div class="given-names">V.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">SMC proteins and their multiple functions in higher plants</div>, <div class="source">Cytogenet. Genome Res</div> ., <div class="volume">124</div>, <div class="fpage">202</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SMC%20proteins%20and%20their%20multiple%20functions%20in%20higher%20plants&amp;author=V.%20Schubert&amp;publication_year=2009&amp;journal=Cytogenet.%20Genome%20Res&amp;volume=124&amp;pages=202-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000218126" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000218126" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000218126"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19556774" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fursova</div> <div class="given-names">O. V.</div></div>, <div class="name string-name"><div class="surname">Pogorelko</div> <div class="given-names">G. V.</div></div>, <div class="name string-name"><div class="surname">Tarasov</div> <div class="given-names">V. A.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Identification of <em>ICE2</em>, a gene involved in cold acclimation which determines freezing tolerance in <em>Arabidopsis thaliana</em></div>, <div class="source">Gene</div> , <div class="volume">429</div>, <div class="fpage">98</div>–<div class="lpage">103</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20ICE2%2C%20a%20gene%20involved%20in%20cold%20acclimation%20which%20determines%20freezing%20tolerance%20in%20Arabidopsis%20thaliana&amp;author=O.%20V.%20Fursova&amp;author=G.%20V.%20Pogorelko&amp;author=V.%20A.%20Tarasov&amp;publication_year=2009&amp;journal=Gene&amp;volume=429&amp;pages=98-103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2008.10.016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2008.10.016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2008.10.016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19026725" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Lightner</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Warwick</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Browse</div> <div class="given-names">J.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Low-temperature damage and subsequent recovery of <em>fab1</em> mutant Arabidopsis exposed to 2 [deg] C</div>, <div class="source">Plant Physiol</div> ., <div class="volume">113</div>, <div class="fpage">347</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Low-temperature%20damage%20and%20subsequent%20recovery%20of%20fab1%20mutant%20Arabidopsis%20exposed%20to%202%20%5Bdeg%5D%20C&amp;author=J.%20Wu&amp;author=J.%20Lightner&amp;author=N.%20Warwick&amp;author=J.%20Browse&amp;publication_year=1997&amp;journal=Plant%20Physiol&amp;volume=113&amp;pages=347-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.113.2.347" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.113.2.347" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.113.2.347"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9046588" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gao</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Wallis</div> <div class="given-names">J. G.</div></div>, <div class="name string-name"><div class="surname">Browse</div> <div class="given-names">J.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Mutations in the prokaryotic pathway rescue the fatty acid biosynthesis1 mutant in the cold</div>, <div class="source">Plant Physiol</div> ., <div class="volume">169</div>, <div class="fpage">442</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutations%20in%20the%20prokaryotic%20pathway%20rescue%20the%20fatty%20acid%20biosynthesis1%20mutant%20in%20the%20cold&amp;author=J.%20Gao&amp;author=J.%20G.%20Wallis&amp;author=J.%20Browse&amp;publication_year=2015&amp;journal=Plant%20Physiol&amp;volume=169&amp;pages=442-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.15.00931" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.15.00931" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.15.00931"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26224803" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sung</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Amasino</div> <div class="given-names">R. M.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Vernalization in <em>Arabidopsis thaliana</em> is mediated by the PHD finger protein VIN3</div>, <div class="source">Nature</div> , <div class="volume">427</div>, <div class="fpage">159</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Vernalization%20in%20Arabidopsis%20thaliana%20is%20mediated%20by%20the%20PHD%20finger%20protein%20VIN3&amp;author=S.%20Sung&amp;author=R.%20M.%20Amasino&amp;publication_year=2004&amp;journal=Nature&amp;volume=427&amp;pages=159-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02195" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02195" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02195"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14712276" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Haudry</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Platts</div> <div class="given-names">A. E.</div></div>, <div class="name string-name"><div class="surname">Vello</div> <div class="given-names">E.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">An atlas of over 90,000 conserved noncoding sequences provides insight into crucifer regulatory regions</div>, <div class="source">Nat. Genet</div> ., <div class="volume">45</div>, <div class="fpage">891</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20atlas%20of%20over%2090%2C000%20conserved%20noncoding%20sequences%20provides%20insight%20into%20crucifer%20regulatory%20regions&amp;author=A.%20Haudry&amp;author=A.%20E.%20Platts&amp;author=E.%20Vello&amp;publication_year=2013&amp;journal=Nat.%20Genet&amp;volume=45&amp;pages=891-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2684" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2684" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2684"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23817568" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Günther</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Lampei</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Barilar</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Schmid</div> <div class="given-names">K. J.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Genomic and phenotypic differentiation of <em>Arabidopsis thaliana</em> along altitudinal gradients in the North Italian Alps</div>, <div class="source">Mol. Ecol</div> ., <div class="volume">25</div>, <div class="fpage">3574</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20and%20phenotypic%20differentiation%20of%20Arabidopsis%20thaliana%20along%20altitudinal%20gradients%20in%20the%20North%20Italian%20Alps&amp;author=T.%20G%C3%BCnther&amp;author=C.%20Lampei&amp;author=I.%20Barilar&amp;author=K.%20J.%20Schmid&amp;publication_year=2016&amp;journal=Mol.%20Ecol&amp;volume=25&amp;pages=3574-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mec.13705" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmec.13705" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmec.13705"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27220345" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">De Smet</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Adams</div> <div class="given-names">K. L.</div></div>, <div class="name string-name"><div class="surname">Vandepoele</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Van Montagu</div> <div class="given-names">M. C.</div></div>, <div class="name string-name"><div class="surname">Maere</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Van de Peer</div> <div class="given-names">Y.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Convergent gene loss following gene and genome duplications creates single-copy families in flowering plants</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> ., <div class="volume">110</div>, <div class="fpage">2898</div>–<div class="lpage">903</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Convergent%20gene%20loss%20following%20gene%20and%20genome%20duplications%20creates%20single-copy%20families%20in%20flowering%20plants&amp;author=R.%20De%20Smet&amp;author=K.%20L.%20Adams&amp;author=K.%20Vandepoele&amp;author=M.%20C.%20Van%20Montagu&amp;author=S.%20Maere&amp;author=Y.%20Van%20de%20Peer&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=110&amp;pages=2898-903" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1300127110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1300127110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1300127110"> </span></div> </div></div></div></div></div><div content-id="dsy003-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Losada</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Yokochi</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Kobayashi</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Hirano</div> <div class="given-names">T.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Identification and characterization of SA/Scc3p subunits in the <em>Xenopus</em> and human cohesin complexes</div>, <div class="source">J. Cell Biol</div> ., <div class="volume">150</div>, <div class="fpage">405</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20SA%2FScc3p%20subunits%20in%20the%20Xenopus%20and%20human%20cohesin%20complexes&amp;author=A.%20Losada&amp;author=T.%20Yokochi&amp;author=R.%20Kobayashi&amp;author=T.%20Hirano&amp;publication_year=2000&amp;journal=J.%20Cell%20Biol&amp;volume=150&amp;pages=405-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1083/jcb.150.3.405" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1083%2Fjcb.150.3.405" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1083%2Fjcb.150.3.405"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10931856" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Losada</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Yokochi</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Hirano</div> <div class="given-names">T.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Functional contribution of Pds5 to cohesin-mediated cohesion in human cells and <em>Xenopus</em> egg extracts</div>, <div class="source">J. Cell Sci</div> ., <div class="volume">118</div>, <div class="fpage">2133</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20contribution%20of%20Pds5%20to%20cohesin-mediated%20cohesion%20in%20human%20cells%20and%20Xenopus%20egg%20extracts&amp;author=A.%20Losada&amp;author=T.%20Yokochi&amp;author=T.%20Hirano&amp;publication_year=2005&amp;journal=J.%20Cell%20Sci&amp;volume=118&amp;pages=2133-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/jcs.02355" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fjcs.02355" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fjcs.02355"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15855230" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pezic</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Weeks</div> <div class="given-names">S. L.</div></div>, <div class="name string-name"><div class="surname">Hadjur</div> <div class="given-names">S.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">More to cohesin than meets the eye: complex diversity for fine-tuning of function</div>, <div class="source">Curr. Opin. Genet. Dev</div> ., <div class="volume">43</div>, <div class="fpage">93</div>–<div class="lpage">100</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=More%20to%20cohesin%20than%20meets%20the%20eye%3A%20complex%20diversity%20for%20fine-tuning%20of%20function&amp;author=D.%20Pezic&amp;author=S.%20L.%20Weeks&amp;author=S.%20Hadjur&amp;publication_year=2017&amp;journal=Curr.%20Opin.%20Genet.%20Dev&amp;volume=43&amp;pages=93-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gde.2017.01.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2017.01.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gde.2017.01.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28189962" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy003-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy003-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sumara</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Vorlaufer</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Gieffers</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Peters</div> <div class="given-names">B. H.</div></div>, <div class="name string-name"><div class="surname">Peters</div> <div class="given-names">J. M.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Characterization of vertebrate cohesin complexes and their regulation in prophase</div>, <div class="source">J. Cell Biol</div> ., <div class="volume">151</div>, <div class="fpage">749</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20vertebrate%20cohesin%20complexes%20and%20their%20regulation%20in%20prophase&amp;author=I.%20Sumara&amp;author=E.%20Vorlaufer&amp;author=C.%20Gieffers&amp;author=B.%20H.%20Peters&amp;author=J.%20M.%20Peters&amp;publication_year=2000&amp;journal=J.%20Cell%20Biol&amp;volume=151&amp;pages=749-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1083/jcb.151.4.749" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1083%2Fjcb.151.4.749" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1083%2Fjcb.151.4.749"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11076961" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2018. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4831046&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/3/10.1093_dnares_dsy003/1/dsy003_supp.zip?Expires=2147483647&amp;Signature=zH88b7wl~kiZ427neY67EtPi8OHq4HFD4VAIlep9NOVbwPo9~SnKnHM5lysisKxf-Isbn8bbFTLoUbNjo-sQJcbh4Bohg5r7s3dFKiTepQnhzWLy8FOX2dFtqRMtXoxwGkqZQhR2Y1o~dLL8VUDjKOE2tYprmqkhNaui6x74V84QvG1X4K6eyRpwu0b~Fv6GBE3jjPfDoZ7e~-4WxGneorvjE-eRthWmwqIkgQyIohIairwy6796yRbotisF-5mksuyVNnR3rdlLswk4wa0hv3SYqS0~lgDTtX54vYVW2cuxSYwKzmBgsk42U37RSJQCjOGaQxrJX6dXNDb2c9rtmg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,643</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsy003"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4831046"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f3%2f307%2f4831046">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F3%2F307%2F4831046" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4831046'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="genomes of two Eutrema species provide insight into plant adaptation to high altitudes" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy003");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f3%2f307%2f4831046">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

