

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/18/1/31/377154","@type":"ScholarlyArticle","name":"Multiplex Target Enrichment Using DNA Indexing for Ultra-High Throughput SNP Detection","datePublished":"2010-12-16","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/18/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2011-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsq029","keywords":["next-generation sequencing","enrichment","capture","SNP","index"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/18/1/31/377154","author":[{"name":"Kenny, Elaine M.","affiliation":"Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry ,  Institute of Molecular Medicine, Trinity College Dublin ,  Ireland","@type":"Person"},{"name":"Cormican, Paul","affiliation":"Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry ,  Institute of Molecular Medicine, Trinity College Dublin ,  Ireland","@type":"Person"},{"name":"Gilks, William P.","affiliation":"Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry ,  Institute of Molecular Medicine, Trinity College Dublin ,  Ireland","@type":"Person"},{"name":"Gates, Amy S.","affiliation":"Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry ,  Institute of Molecular Medicine, Trinity College Dublin ,  Ireland","@type":"Person"},{"name":"O'Dushlaine, Colm T.","affiliation":"Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry ,  Institute of Molecular Medicine, Trinity College Dublin ,  Ireland","@type":"Person"},{"name":"Pinto, Carlos","affiliation":"Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry ,  Institute of Molecular Medicine, Trinity College Dublin ,  Ireland","@type":"Person"},{"name":"Corvin, Aiden P.","affiliation":"Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry ,  Institute of Molecular Medicine, Trinity College Dublin ,  Ireland","@type":"Person"},{"name":"Gill, Michael","affiliation":"Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry ,  Institute of Molecular Medicine, Trinity College Dublin ,  Ireland","@type":"Person"},{"name":"Morris, Derek W.","affiliation":"Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry ,  Institute of Molecular Medicine, Trinity College Dublin ,  Ireland","@type":"Person"}],"description":"Abstract.  Screening large numbers of target regions in multiple DNA samples for sequence variation is an important application of next-generation sequencing bu","pageStart":"31","pageEnd":"38","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/1/1/m_cover.gif?Expires=2147483647&Signature=ClGcwi7mnwSCPup8xdZdf5ipPpPJFPkMnb8H8urPYhEwpzwvlLkxmUmUtTsd0O4KGqDmjee3iAP~cMOV4NNqFj~GhFPiWnYh~s6OnHI~uE0SigPMO8DkCoxQVVyGjpmFT3VyUCd1Rnhz0tzOY7qjLvTA13Zb9KBld2OsYWDgF6DdaTNIs1umhGa0yRi4S7UVqdSOPa2DP4QXoLG6JKhXqOt4B5uTWFLVPp9R~IrmcfN-N7uO1eXmxMYjpGkprLlRRW5~1~apyMzzJDfdqIyDGFVHMcKv4iliXAPqZ1VMUaMvUFuiZyal2HpblFNoF8oq40FBm2x~DVSMvIEBHFMHhQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Multiplex Target Enrichment Using DNA Indexing for Ultra-High Throughput SNP Detection","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/1/1/m_cover.gif?Expires=2147483647&Signature=ClGcwi7mnwSCPup8xdZdf5ipPpPJFPkMnb8H8urPYhEwpzwvlLkxmUmUtTsd0O4KGqDmjee3iAP~cMOV4NNqFj~GhFPiWnYh~s6OnHI~uE0SigPMO8DkCoxQVVyGjpmFT3VyUCd1Rnhz0tzOY7qjLvTA13Zb9KBld2OsYWDgF6DdaTNIs1umhGa0yRi4S7UVqdSOPa2DP4QXoLG6JKhXqOt4B5uTWFLVPp9R~IrmcfN-N7uO1eXmxMYjpGkprLlRRW5~1~apyMzzJDfdqIyDGFVHMcKv4iliXAPqZ1VMUaMvUFuiZyal2HpblFNoF8oq40FBm2x~DVSMvIEBHFMHhQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Multiplex Target Enrichment Using DNA Indexing for Ultra-High Throughput SNP Detection" />
    <meta property="og:description" content="Abstract.  Screening large numbers of target regions in multiple DNA samples for sequence variation is an important application of next-generation sequencing bu" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/18/1/31/377154" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2010-12-16" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Kenny, Elaine M." /><meta name="citation_author_institution" content="Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry, Institute of Molecular Medicine, Trinity College Dublin, Ireland" /><meta name="citation_author" content="Cormican, Paul" /><meta name="citation_author_institution" content="Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry, Institute of Molecular Medicine, Trinity College Dublin, Ireland" /><meta name="citation_author" content="Gilks, William P." /><meta name="citation_author_institution" content="Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry, Institute of Molecular Medicine, Trinity College Dublin, Ireland" /><meta name="citation_author" content="Gates, Amy S." /><meta name="citation_author_institution" content="Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry, Institute of Molecular Medicine, Trinity College Dublin, Ireland" /><meta name="citation_author" content="O'Dushlaine, Colm T." /><meta name="citation_author_institution" content="Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry, Institute of Molecular Medicine, Trinity College Dublin, Ireland" /><meta name="citation_author" content="Pinto, Carlos" /><meta name="citation_author_institution" content="Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry, Institute of Molecular Medicine, Trinity College Dublin, Ireland" /><meta name="citation_author" content="Corvin, Aiden P." /><meta name="citation_author_institution" content="Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry, Institute of Molecular Medicine, Trinity College Dublin, Ireland" /><meta name="citation_author" content="Gill, Michael" /><meta name="citation_author_institution" content="Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry, Institute of Molecular Medicine, Trinity College Dublin, Ireland" /><meta name="citation_author" content="Morris, Derek W." /><meta name="citation_author_institution" content="Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry, Institute of Molecular Medicine, Trinity College Dublin, Ireland" /><meta name="citation_title" content="Multiplex Target Enrichment Using DNA Indexing for Ultra-High Throughput SNP Detection" /><meta name="citation_firstpage" content="31" /><meta name="citation_lastpage" content="38" /><meta name="citation_doi" content="10.1093/dnares/dsq029" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="18" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2011/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Multiplex amplification enabled by selective circularization of large sets of genomic DNA fragments; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=e71" /><meta name="citation_reference" content="citation_title=Molecular tools for a molecular medicine: analyzing genes, transcripts and proteins using padlock and proximity probes; J. Mol. Recognit.;  citation_year=2004;  citation_volume=17;  citation_pages=194-7; " /><meta name="citation_reference" content="citation_title=Multiplex padlock targeted sequencing reveals human hypermutable CpG variations; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=1606-15; " /><meta name="citation_reference" content="citation_title=Direct selection: a method for the isolation of cDNAs encoded by large genomic regions; Proc. Natl Acad. Sci. USA;  citation_year=1991;  citation_volume=88;  citation_pages=9628-32; " /><meta name="citation_reference" content="citation_title=Multiplex amplification of large sets of human exons; Nat. Methods;  citation_year=2007;  citation_volume=4;  citation_pages=931-6; " /><meta name="citation_reference" content="citation_title=Sequencing the full-length of the phosphatase and tensin homolog (PTEN) gene in hepatocellular carcinoma (HCC) using the 454 GS20 and Illumina GA DNA sequencing platforms; World J. Surg.;  citation_year=2009;  citation_volume=33;  citation_pages=647-52; " /><meta name="citation_reference" content="citation_title=Microdroplet-based PCR enrichment for large-scale targeted sequencing; Nat. Biotechnol.;  citation_year=2009;  citation_volume=27;  citation_pages=1025-31; " /><meta name="citation_reference" content="citation_title=Massively parallel exon capture and library-free resequencing across 16 genomes; Nat. Methods;  citation_year=2009;  citation_volume=6;  citation_pages=315-6; " /><meta name="citation_reference" content="citation_title=Nested Patch PCR enables highly multiplexed mutation discovery in candidate genes; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=1844-50; " /><meta name="citation_reference" content="citation_title=Target-enrichment strategies for next-generation sequencing; Nat. Methods;  citation_year=2010;  citation_volume=7;  citation_pages=111-8; " /><meta name="citation_reference" content="citation_title=Identification of genetic variants using bar-coded multiplexed sequencing; Nat. Methods;  citation_year=2008;  citation_volume=5;  citation_pages=887-93; " /><meta name="citation_reference" content="citation_title=Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing; Nat. Biotechnol.;  citation_year=2009;  citation_volume=27;  citation_pages=182-9; " /><meta name="citation_reference" content="citation_title=Mapping short DNA sequencing reads and calling variants using mapping quality scores; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=1851-8; " /><meta name="citation_reference" content="citation_title=Enrichment of sequencing targets from the human genome by solution hybridization; Genome Biol.;  citation_year=2009;  citation_volume=10;  citation_pages=R116" /><meta name="citation_reference" content="citation_title=Targeted capture and massively parallel sequencing of 12 human exomes; Nature;  citation_year=2009;  citation_volume=461;  citation_pages=272-6; " /><meta name="citation_reference" content="citation_title=Genome-wide in situ exon capture for selective resequencing; Nat. Genet.;  citation_year=2007;  citation_volume=39;  citation_pages=1522-7; " /><meta name="citation_reference" content="citation_title=Direct selection of human genomic loci by microarray hybridization; Nat. Methods;  citation_year=2007;  citation_volume=4;  citation_pages=903-5; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/18/1/31/1071922/dsq029.pdf" /><meta name="description" content="Abstract.  Screening large numbers of target regions in multiple DNA samples for sequence variation is an important application of next-generation sequencing bu" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/18/1/31/377154" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/18/1/31/377154" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/18/1/31/377154");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsq029';
    </script>
<title>
	Multiplex Target Enrichment Using DNA Indexing for Ultra-High Throughput SNP Detection | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./377154" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bfg5PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLZW5ueSwgRWxhaW5lIE0uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVHJpbml0eSBHZW5vbWUgU2VxdWVuY2luZyBMYWJvcmF0b3J5LCBOZXVyb3BzeWNoaWF0cmljIEdlbmV0aWNzIFJlc2VhcmNoIEdyb3VwLCBEZXBhcnRtZW50IG9mIFBzeWNoaWF0cnksIEluc3RpdHV0ZSBvZiBNb2xlY3VsYXIgTWVkaWNpbmUsIFRyaW5pdHkgQ29sbGVnZSBEdWJsaW4sIElyZWxhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDb3JtaWNhbiwgUGF1bCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlRyaW5pdHkgR2Vub21lIFNlcXVlbmNpbmcgTGFib3JhdG9yeSwgTmV1cm9wc3ljaGlhdHJpYyBHZW5ldGljcyBSZXNlYXJjaCBHcm91cCwgRGVwYXJ0bWVudCBvZiBQc3ljaGlhdHJ5LCBJbnN0aXR1dGUgb2YgTW9sZWN1bGFyIE1lZGljaW5lLCBUcmluaXR5IENvbGxlZ2UgRHVibGluLCBJcmVsYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR2lsa3MsIFdpbGxpYW0gUC4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUcmluaXR5IEdlbm9tZSBTZXF1ZW5jaW5nIExhYm9yYXRvcnksIE5ldXJvcHN5Y2hpYXRyaWMgR2VuZXRpY3MgUmVzZWFyY2ggR3JvdXAsIERlcGFydG1lbnQgb2YgUHN5Y2hpYXRyeSwgSW5zdGl0dXRlIG9mIE1vbGVjdWxhciBNZWRpY2luZSwgVHJpbml0eSBDb2xsZWdlIER1YmxpbiwgSXJlbGFuZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkdhdGVzLCBBbXkgUy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUcmluaXR5IEdlbm9tZSBTZXF1ZW5jaW5nIExhYm9yYXRvcnksIE5ldXJvcHN5Y2hpYXRyaWMgR2VuZXRpY3MgUmVzZWFyY2ggR3JvdXAsIERlcGFydG1lbnQgb2YgUHN5Y2hpYXRyeSwgSW5zdGl0dXRlIG9mIE1vbGVjdWxhciBNZWRpY2luZSwgVHJpbml0eSBDb2xsZWdlIER1YmxpbiwgSXJlbGFuZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik8nRHVzaGxhaW5lLCBDb2xtIFQuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVHJpbml0eSBHZW5vbWUgU2VxdWVuY2luZyBMYWJvcmF0b3J5LCBOZXVyb3BzeWNoaWF0cmljIEdlbmV0aWNzIFJlc2VhcmNoIEdyb3VwLCBEZXBhcnRtZW50IG9mIFBzeWNoaWF0cnksIEluc3RpdHV0ZSBvZiBNb2xlY3VsYXIgTWVkaWNpbmUsIFRyaW5pdHkgQ29sbGVnZSBEdWJsaW4sIElyZWxhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQaW50bywgQ2FybG9zIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVHJpbml0eSBHZW5vbWUgU2VxdWVuY2luZyBMYWJvcmF0b3J5LCBOZXVyb3BzeWNoaWF0cmljIEdlbmV0aWNzIFJlc2VhcmNoIEdyb3VwLCBEZXBhcnRtZW50IG9mIFBzeWNoaWF0cnksIEluc3RpdHV0ZSBvZiBNb2xlY3VsYXIgTWVkaWNpbmUsIFRyaW5pdHkgQ29sbGVnZSBEdWJsaW4sIElyZWxhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDb3J2aW4sIEFpZGVuIFAuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVHJpbml0eSBHZW5vbWUgU2VxdWVuY2luZyBMYWJvcmF0b3J5LCBOZXVyb3BzeWNoaWF0cmljIEdlbmV0aWNzIFJlc2VhcmNoIEdyb3VwLCBEZXBhcnRtZW50IG9mIFBzeWNoaWF0cnksIEluc3RpdHV0ZSBvZiBNb2xlY3VsYXIgTWVkaWNpbmUsIFRyaW5pdHkgQ29sbGVnZSBEdWJsaW4sIElyZWxhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHaWxsLCBNaWNoYWVsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVHJpbml0eSBHZW5vbWUgU2VxdWVuY2luZyBMYWJvcmF0b3J5LCBOZXVyb3BzeWNoaWF0cmljIEdlbmV0aWNzIFJlc2VhcmNoIEdyb3VwLCBEZXBhcnRtZW50IG9mIFBzeWNoaWF0cnksIEluc3RpdHV0ZSBvZiBNb2xlY3VsYXIgTWVkaWNpbmUsIFRyaW5pdHkgQ29sbGVnZSBEdWJsaW4sIElyZWxhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNb3JyaXMsIERlcmVrIFcuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVHJpbml0eSBHZW5vbWUgU2VxdWVuY2luZyBMYWJvcmF0b3J5LCBOZXVyb3BzeWNoaWF0cmljIEdlbmV0aWNzIFJlc2VhcmNoIEdyb3VwLCBEZXBhcnRtZW50IG9mIFBzeWNoaWF0cnksIEluc3RpdHV0ZSBvZiBNb2xlY3VsYXIgTWVkaWNpbmUsIFRyaW5pdHkgQ29sbGVnZSBEdWJsaW4sIElyZWxhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ik11bHRpcGxleCBUYXJnZXQgRW5yaWNobWVudCBVc2luZyBETkEgSW5kZXhpbmcgZm9yIFVsdHJhLUhpZ2ggVGhyb3VnaHB1dCBTTlAgRGV0ZWN0aW9uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMzEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNxMDI5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTEvMDIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aXBsZXggYW1wbGlmaWNhdGlvbiBlbmFibGVkIGJ5IHNlbGVjdGl2ZSBjaXJjdWxhcml6YXRpb24gb2YgbGFyZ2Ugc2V0cyBvZiBnZW5vbWljIEROQSBmcmFnbWVudHM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPWU3MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciB0b29scyBmb3IgYSBtb2xlY3VsYXIgbWVkaWNpbmU6IGFuYWx5emluZyBnZW5lcywgdHJhbnNjcmlwdHMgYW5kIHByb3RlaW5zIHVzaW5nIHBhZGxvY2sgYW5kIHByb3hpbWl0eSBwcm9iZXM7IEouIE1vbC4gUmVjb2duaXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MTk0LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11bHRpcGxleCBwYWRsb2NrIHRhcmdldGVkIHNlcXVlbmNpbmcgcmV2ZWFscyBodW1hbiBoeXBlcm11dGFibGUgQ3BHIHZhcmlhdGlvbnM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9MTYwNi0xNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlyZWN0IHNlbGVjdGlvbjogYSBtZXRob2QgZm9yIHRoZSBpc29sYXRpb24gb2YgY0ROQXMgZW5jb2RlZCBieSBsYXJnZSBnZW5vbWljIHJlZ2lvbnM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9ODg7ICBjaXRhdGlvbl9wYWdlcz05NjI4LTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aXBsZXggYW1wbGlmaWNhdGlvbiBvZiBsYXJnZSBzZXRzIG9mIGh1bWFuIGV4b25zOyBOYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTkzMS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jaW5nIHRoZSBmdWxsLWxlbmd0aCBvZiB0aGUgcGhvc3BoYXRhc2UgYW5kIHRlbnNpbiBob21vbG9nIChQVEVOKSBnZW5lIGluIGhlcGF0b2NlbGx1bGFyIGNhcmNpbm9tYSAoSENDKSB1c2luZyB0aGUgNDU0IEdTMjAgYW5kIElsbHVtaW5hIEdBIEROQSBzZXF1ZW5jaW5nIHBsYXRmb3JtczsgV29ybGQgSi4gU3VyZy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz02NDctNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3JvZHJvcGxldC1iYXNlZCBQQ1IgZW5yaWNobWVudCBmb3IgbGFyZ2Utc2NhbGUgdGFyZ2V0ZWQgc2VxdWVuY2luZzsgTmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTEwMjUtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hc3NpdmVseSBwYXJhbGxlbCBleG9uIGNhcHR1cmUgYW5kIGxpYnJhcnktZnJlZSByZXNlcXVlbmNpbmcgYWNyb3NzIDE2IGdlbm9tZXM7IE5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MzE1LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5lc3RlZCBQYXRjaCBQQ1IgZW5hYmxlcyBoaWdobHkgbXVsdGlwbGV4ZWQgbXV0YXRpb24gZGlzY292ZXJ5IGluIGNhbmRpZGF0ZSBnZW5lczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xODQ0LTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYXJnZXQtZW5yaWNobWVudCBzdHJhdGVnaWVzIGZvciBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZzsgTmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0xMTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgZ2VuZXRpYyB2YXJpYW50cyB1c2luZyBiYXItY29kZWQgbXVsdGlwbGV4ZWQgc2VxdWVuY2luZzsgTmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz04ODctOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNvbHV0aW9uIGh5YnJpZCBzZWxlY3Rpb24gd2l0aCB1bHRyYS1sb25nIG9saWdvbnVjbGVvdGlkZXMgZm9yIG1hc3NpdmVseSBwYXJhbGxlbCB0YXJnZXRlZCBzZXF1ZW5jaW5nOyBOYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MTgyLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgc2hvcnQgRE5BIHNlcXVlbmNpbmcgcmVhZHMgYW5kIGNhbGxpbmcgdmFyaWFudHMgdXNpbmcgbWFwcGluZyBxdWFsaXR5IHNjb3JlczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xODUxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVucmljaG1lbnQgb2Ygc2VxdWVuY2luZyB0YXJnZXRzIGZyb20gdGhlIGh1bWFuIGdlbm9tZSBieSBzb2x1dGlvbiBoeWJyaWRpemF0aW9uOyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1SMTE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFyZ2V0ZWQgY2FwdHVyZSBhbmQgbWFzc2l2ZWx5IHBhcmFsbGVsIHNlcXVlbmNpbmcgb2YgMTIgaHVtYW4gZXhvbWVzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDYxOyAgY2l0YXRpb25fcGFnZXM9MjcyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGluIHNpdHUgZXhvbiBjYXB0dXJlIGZvciBzZWxlY3RpdmUgcmVzZXF1ZW5jaW5nOyBOYXQuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPTE1MjItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlyZWN0IHNlbGVjdGlvbiBvZiBodW1hbiBnZW5vbWljIGxvY2kgYnkgbWljcm9hcnJheSBoeWJyaWRpemF0aW9uOyBOYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTkwMy01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTgvMS8zMS8xMDcxOTIyL2RzcTAyOS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgU2NyZWVuaW5nIGxhcmdlIG51bWJlcnMgb2YgdGFyZ2V0IHJlZ2lvbnMgaW4gbXVsdGlwbGUgRE5BIHNhbXBsZXMgZm9yIHNlcXVlbmNlIHZhcmlhdGlvbiBpcyBhbiBpbXBvcnRhbnQgYXBwbGljYXRpb24gb2YgbmV4dC1nZW5lcmF0aW9uIHNlcXVlbmNpbmcgYnUiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM3NzE1NCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGT1KGmIdhDqDNIPHCsP86M+z5Cdcw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f1%2f31%2f377154" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f1%2f31%2f377154" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=14695" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/18/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/1/1/m_cover.gif?Expires=2147483647&amp;Signature=ClGcwi7mnwSCPup8xdZdf5ipPpPJFPkMnb8H8urPYhEwpzwvlLkxmUmUtTsd0O4KGqDmjee3iAP~cMOV4NNqFj~GhFPiWnYh~s6OnHI~uE0SigPMO8DkCoxQVVyGjpmFT3VyUCd1Rnhz0tzOY7qjLvTA13Zb9KBld2OsYWDgF6DdaTNIs1umhGa0yRi4S7UVqdSOPa2DP4QXoLG6JKhXqOt4B5uTWFLVPp9R~IrmcfN-N7uO1eXmxMYjpGkprLlRRW5~1~apyMzzJDfdqIyDGFVHMcKv4iliXAPqZ1VMUaMvUFuiZyal2HpblFNoF8oq40FBm2x~DVSMvIEBHFMHhQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/18/1">
                        <div class="volume trailing-comma">Volume 18</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2011        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5499730">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5499730">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5499733">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5499733">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5499743">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5499743">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5499770">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5499770">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5499799">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5499799">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5499806">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5499806">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5499808">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5499808">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5499810">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5499810">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/18/1/17/377919">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/18/1/39/378044">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Multiplex Target Enrichment Using DNA Indexing for Ultra-High Throughput SNP Detection
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Elaine M. Kenny</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry</div>, <div class="insititution">Institute of Molecular Medicine, Trinity College Dublin</div>, <div class="country">Ireland</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +353 1 896 8461. Fax. <div class="fax">+353 1 896 3405</div>. E-mail: <a href="mailto:elaine.kenny@tcd.ie" target="_blank">elaine.kenny@tcd.ie</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Elaine+M.+Kenny">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kenny E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kenny E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Elaine M. Kenny</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Paul Cormican</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry</div>, <div class="insititution">Institute of Molecular Medicine, Trinity College Dublin</div>, <div class="country">Ireland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Paul+Cormican">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cormican P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cormican P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Paul Cormican</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">William P. Gilks</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry</div>, <div class="insititution">Institute of Molecular Medicine, Trinity College Dublin</div>, <div class="country">Ireland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=William+P.+Gilks">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gilks W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gilks W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">William P. Gilks</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Amy S. Gates</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry</div>, <div class="insititution">Institute of Molecular Medicine, Trinity College Dublin</div>, <div class="country">Ireland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Amy+S.+Gates">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gates A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gates A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Amy S. Gates</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Colm T. O&#39;Dushlaine</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry</div>, <div class="insititution">Institute of Molecular Medicine, Trinity College Dublin</div>, <div class="country">Ireland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Colm+T.+O%27Dushlaine">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=O&#39;Dushlaine C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;O&#39;Dushlaine C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Colm T. O'Dushlaine</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Carlos Pinto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry</div>, <div class="insititution">Institute of Molecular Medicine, Trinity College Dublin</div>, <div class="country">Ireland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Carlos+Pinto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pinto C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pinto C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Carlos Pinto</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Aiden P. Corvin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry</div>, <div class="insititution">Institute of Molecular Medicine, Trinity College Dublin</div>, <div class="country">Ireland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Aiden+P.+Corvin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Corvin A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Corvin A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Aiden P. Corvin</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Michael Gill</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry</div>, <div class="insititution">Institute of Molecular Medicine, Trinity College Dublin</div>, <div class="country">Ireland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Michael+Gill">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gill M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gill M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Michael Gill</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Derek W. Morris</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Trinity Genome Sequencing Laboratory, Neuropsychiatric Genetics Research Group, Department of Psychiatry</div>, <div class="insititution">Institute of Molecular Medicine, Trinity College Dublin</div>, <div class="country">Ireland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Derek+W.+Morris">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Morris D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Morris D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Derek W. Morris</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Osamu Ohara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 18, Issue 1, 1 February 2011, Pages 31–38, <a href='https://doi.org/10.1093/dnares/dsq029'>https://doi.org/10.1093/dnares/dsq029</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">16 December 2010</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">02 September 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">08 November 2010</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="377154" href="/dnaresearch/article-pdf/18/1/31/1071922/dsq029.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Elaine M. Kenny, Paul Cormican, William P. Gilks, Amy S. Gates, Colm T. O'Dushlaine, Carlos Pinto, Aiden P. Corvin, Michael Gill, Derek W. Morris;  Multiplex Target Enrichment Using DNA Indexing for Ultra-High Throughput SNP Detection, <em>DNA Research</em>, Volume 18, Issue 1, 1 February 2011, Pages 31–38, <a href='https://doi.org/10.1093/dnares/dsq029'>https://doi.org/10.1093/dnares/dsq029</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/377154?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/377154?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/377154?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/377154?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/377154?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/377154?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/377154?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Multiplex%20Target%20Enrichment%20Using%20DNA%20Indexing%20for%20Ultra-High%20Throughput%20SNP%20Detection&amp;publicationDate=2010-12-16&amp;volumeNum=18&amp;issueNum=1&amp;author=Kenny%2C%20Elaine%20M.%3B%20Cormican%2C%20Paul&amp;startPage=31&amp;endPage=38&amp;contentId=10.1093%2Fdnares%2Fdsq029&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f1%2f31%2f377154" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5499730 id="5499730" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Screening large numbers of target regions in multiple DNA samples for sequence variation is an important application of next-generation sequencing but an efficient method to enrich the samples in parallel has yet to be reported. We describe an advanced method that combines DNA samples using indexes or barcodes prior to target enrichment to facilitate this type of experiment. Sequencing libraries for multiple individual DNA samples, each incorporating a unique 6-bp index, are combined in equal quantities, enriched using a single in-solution target enrichment assay and sequenced in a single reaction. Sequence reads are parsed based on the index, allowing sequence analysis of individual samples. We show that the use of indexed samples does not impact on the efficiency of the enrichment reaction. For three- and nine-indexed HapMap DNA samples, the method was found to be highly accurate for SNP identification. Even with sequence coverage as low as 8x, 99% of sequence SNP calls were concordant with known genotypes. Within a single experiment, this method can sequence the exonic regions of hundreds of genes in tens of samples for sequence and structural variation using as little as 1 μg of input DNA per sample.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;next-generation sequencing&quot;">next-generation sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="enrichment">enrichment</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="capture">capture</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SNP">SNP</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="index">index</a></div>                            <h2 scrollto-destination=5499733 id="5499733" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Next-generation sequencing technology has the potential to allow sequencing of whole genomes to be carried out in standard molecular genetics laboratories. However, an important current application is the sequencing of specific genomic regions, e.g. genes with known or suspected mutations in patient samples. In order to sequence parts of the genome of interest, a number of target enrichment procedures have been developed. These include standard PCR, long-range PCR, nested patch PCR, template circularization, the use of gapped molecular inversion probes, microarray capture, in-solution capture and microdroplet-based PCR enrichment.<sup><a reveal-id="DSQ029C1  DSQ029C9" data-open="DSQ029C1  DSQ029C9" class="link link-ref link-reveal xref-bibr">1–9</a></sup> All these methods work but each has its own advantages and disadvantages. A recent review on target enrichment strategies for next-generation sequencing, which discusses the methods listed above, concluded that the ability to combine DNA samples prior to enrichment would be an important advancement for targeted next-generation sequencing: ‘The logical extension of sample pooling is to perform multiplexed target enrichments in which many samples are barcoded before capture’.<sup><a reveal-id="DSQ029C10" data-open="DSQ029C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Here, we describe a method that delivers this advancement by combining DNA samples using indexes or barcodes prior to target enrichment.</p><p>We have adapted the indexing protocol published by Craig <em>et al.</em><sup><a reveal-id="DSQ029C11" data-open="DSQ029C11" class="link link-ref link-reveal xref-bibr">11</a></sup> and combined it with the Agilent Technologies SureSelect Target Enrichment System to develop a cost-efficient method for targeting smaller regions of the genome (e.g. 100 kb–1 Mb) in multiple DNA samples. SureSelect is a capture protocol based on the in-solution method developed by Gnirke <em>et al</em>.<sup><a reveal-id="DSQ029C12" data-open="DSQ029C12" class="link link-ref link-reveal xref-bibr">12</a></sup> This method allows targeting of regions of the genome in custom designed reactions by using cRNA baits. By enabling simultaneous enrichment of multiple samples with no impact on individual sample identification for downstream analysis, this method significantly reduces the cost and time required for this type of experiment. Our method is highly suited to target enrichment of smaller genomic regions. The method can accurately detect SNPs at enriched target sites. Within a single sequencing experiment, it has the capacity to analyze the exonic regions of hundreds of genes in tens of samples for sequence variation using as little as 1 μg of input DNA.</p>                            <h2 scrollto-destination=5499743 id="5499743" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5499745 id="5499745" class="section-title" data-legacy-id=s2a>2.1. Enrichment reaction eArray design</h3>
<p>We focused on two genes on chromosome 1 (PTBP2 and CDC42). The plan was to sequence the genes in full and include upstream and downstream regions. The target region for CDC42 was 108 139 bp, and for PTBP2, it was 184 850 bp. Bait libraries were designed and assessed for coverage across the target genomic regions using the Agilent eArray website (<a class="link link-uri" href="https://earray.chem.agilent.com/earray/">https://earray.chem.agilent.com/earray/</a>). The online design recommends repeat masking of the target sequence in order to minimize off-target capture. With the default repeat masking option turned on, it was only possible to design target capture baits for just under 40% of CDC42 and just under 43% for PTBP2 (see cRNA baits RM track in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. SA</a>, e.g. of coverage across PTBP2 with default options). In order to try and increase the percentage of target capture, we adopted an alternative repeat masking protocol: the repeat masking constraints were relaxed to allow relatively unique baits to remain in the pool of target capture baits. This was achieved by designing baits using the Agilent eArray website to the target region with the repeat masking option turned off. The resulting baits were searched against the human genome reference (hg18) for similarity. If a bait mapped to more than one location with greater than 90% sequence identity using BLAST (∼12 mismatches across the bait), this bait was removed from the design. Using this protocol, we were able to design baits for just over 54% of CDC42 and 77% of PTBP2 (see cRNA baits SRM in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. SA</a>, e.g. of coverage across PTBP2 with our alternative relaxed repeat masking). Because the bait library was a fraction of the total possible for eArray design, we replicated it four times using 92.18% of the available space on the array.</p>                            <h3 scrollto-destination=5499748 id="5499748" class="section-title" data-legacy-id=s2b>2.2. Index design</h3>
<p>The indexing methodology was taken from the study by Craig <em>et al</em>.<sup><a reveal-id="DSQ029C11" data-open="DSQ029C11" class="link link-ref link-reveal xref-bibr">11</a></sup> The specific 6 bp indexes used in this study (listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table SA</a>) were from a group of 19 indexes that performed most reliably in that study (Fig. <a reveal-id="DSQ029F2" data-open="DSQ029F2" class="link link-reveal link-table xref-fig">2</a> in Craig <em>et al</em><sub>.</sub><sup><a reveal-id="DSQ029C11" data-open="DSQ029C11" class="link link-ref link-reveal xref-bibr">11</a></sup>). Each index is designed such that it can tolerate at least 1 bp change as a result of a sequencing error without mutating into one of the other indexes used in the study. This helps guard against potential mix-up of individual sample data.</p>                            <h3 scrollto-destination=5499753 id="5499753" class="section-title" data-legacy-id=s2c>2.3. Library preparation, target enrichment and sequencing of genomic DNA samples</h3>
<p>DNA from each of nine CEU HapMap samples (three trios) was used in this study. A total of four enriched sequencing libraries were generated using these samples (Fig. <a reveal-id="DSQ029F1" data-open="DSQ029F1" class="link link-reveal link-table xref-fig">1</a>). Library 1 contained one target enriched non-indexed DNA sample. Library 2 contained one target enriched indexed DNA sample. Library 3 was a multiplex sequencing library of three equimolar indexed samples, which was enriched using one target enrichment reaction. Library 4 was a multiplex sequencing library of nine equimolar indexed samples, which was enriched using one target enrichment reaction. Figure <a reveal-id="DSQ029F1" data-open="DSQ029F1" class="link link-reveal link-table xref-fig">1</a> identifies the HapMap samples used in each lane of sequencing. The issue of potential differences in quantity of sequence reads between indexed samples due to the efficiency of indexed adapter ligation to input DNA, as previously reported by Craig <em>et al.</em>,<sup><a reveal-id="DSQ029C11" data-open="DSQ029C11" class="link link-ref link-reveal xref-bibr">11</a></sup> was addressed in this study by multiplexing the indexed in equal quantities pre-enrichment. </p>                        <a id="5499756" scrollto-destination="5499756"></a>
<div id="DSQ029F1" data-id="DSQ029F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq029/2/m_dsq02901.jpeg?Expires=2147483647&Signature=Krrz8zSJuQhr9Y9iHlubb~E2HBCbLXaT5Cz8Ix9Y0LV7Fq4S1rzGAejnbduGPckbuh2iyQBOYhA4gWLF2Wagsnn7DJUGz7JK4MGG4drwZPatb20PD~OlcwvG3kqTiARyEx-TrKJ-LGd4rYLl3IlR2z6Uu74MWSW0kyoWkszyu6-ffZea4rQece9iRoR8M4zHrmRwOSCK1GLtoJ-HSEGzF~Q2sKzLsy09wg8mNc6D8~3jg3-nsWL5e~f19FZaPsDzGkXBwHuc~iTwyFrChlHTZDkqoaajsxpPSHTPT50Y3Tr6nIb5COUEbu0v31GsYPIfdKRY9F0BHVbekDf11D5inw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Experimental Design. Genomic DNA from nine HapMap samples was chosen for the study (three trio families). DNA from one of the samples (NA11881) was prepared twice (with and without an indexed adapter), target enriched and sequenced separately as single samples (non-indexed sample and one indexed sample in Step 1 and enriched libraries 1 and 2 in Step 2). One trio family (NA11881, NA11882 and NA10859; all indexed) was pooled after the Illumina genomic DNA sample prep and enriched together using one SureSelect enrichment reaction to produce the enriched library 3 sample. Indexed DNA from all nine samples was also pooled after the Illumina genomic DNA sample prep and enriched together using one SureSelect enrichment reaction to produce the enriched library 4 sample. Note: enriched libraries 3 and 4 were also sequenced using 80 bp reads to generate additional data for validation of the method for SNP detection." path-from-xml="dsq02901.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5499756/dsq02901.jpeg" path-from-xml="dsq02901.jpeg" target="_blank">View large</a><a section="5499756" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq029/2/dsq02901.jpeg?Expires=2147483647&Signature=BVkOwlEOOUQ3Al-uUwg3PRFHXJVX55hXH9vGKZnvjjeCuJRbp8E-dUJLZjY0SI3qC-dEcYKlR5gfbGA8XOSwds4rpL4a7x77WIR1FJbzxsv0Vh5BZpmN2Mg7YglJSJzF6ZGC6BYqqh0xt9nWQ7o0-5Ke-eq0pSukmRycjD4~merKkvM8Vsb5qfWUK0hT-FjE3usv8WWg-8s4nZplhO7tSKxwLgQ1SRb1I8bPMNMQLXcA5IK9EfUL0-yZ4wgrMbj5xZwxFRM4h0hXoMOQ9oeUelh55H7v5eB3Z1LdN9OxCMF69MdF19Nv~-FOWuEynANhrovUNKi8PGE6BOAskHOzcA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5499756&ar=377154&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02901.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Experimental Design. Genomic DNA from nine HapMap samples was chosen for the study (three trio families). DNA from one of the samples (NA11881) was prepared twice (with and without an indexed adapter), target enriched and sequenced separately as single samples (non-indexed sample and one indexed sample in Step 1 and enriched libraries 1 and 2 in Step 2). One trio family (NA11881, NA11882 and NA10859; all indexed) was pooled after the Illumina genomic DNA sample prep and enriched together using one SureSelect enrichment reaction to produce the enriched library 3 sample. Indexed DNA from all nine samples was also pooled after the Illumina genomic DNA sample prep and enriched together using one SureSelect enrichment reaction to produce the enriched library 4 sample. Note: enriched libraries 3 and 4 were also sequenced using 80 bp reads to generate additional data for validation of the method for SNP detection.</p></div></div><div content-id="DSQ029F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq029/2/m_dsq02901.jpeg?Expires=2147483647&Signature=Krrz8zSJuQhr9Y9iHlubb~E2HBCbLXaT5Cz8Ix9Y0LV7Fq4S1rzGAejnbduGPckbuh2iyQBOYhA4gWLF2Wagsnn7DJUGz7JK4MGG4drwZPatb20PD~OlcwvG3kqTiARyEx-TrKJ-LGd4rYLl3IlR2z6Uu74MWSW0kyoWkszyu6-ffZea4rQece9iRoR8M4zHrmRwOSCK1GLtoJ-HSEGzF~Q2sKzLsy09wg8mNc6D8~3jg3-nsWL5e~f19FZaPsDzGkXBwHuc~iTwyFrChlHTZDkqoaajsxpPSHTPT50Y3Tr6nIb5COUEbu0v31GsYPIfdKRY9F0BHVbekDf11D5inw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Experimental Design. Genomic DNA from nine HapMap samples was chosen for the study (three trio families). DNA from one of the samples (NA11881) was prepared twice (with and without an indexed adapter), target enriched and sequenced separately as single samples (non-indexed sample and one indexed sample in Step 1 and enriched libraries 1 and 2 in Step 2). One trio family (NA11881, NA11882 and NA10859; all indexed) was pooled after the Illumina genomic DNA sample prep and enriched together using one SureSelect enrichment reaction to produce the enriched library 3 sample. Indexed DNA from all nine samples was also pooled after the Illumina genomic DNA sample prep and enriched together using one SureSelect enrichment reaction to produce the enriched library 4 sample. Note: enriched libraries 3 and 4 were also sequenced using 80 bp reads to generate additional data for validation of the method for SNP detection." path-from-xml="dsq02901.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5499756/dsq02901.jpeg" path-from-xml="dsq02901.jpeg" target="_blank">View large</a><a section="5499756" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq029/2/dsq02901.jpeg?Expires=2147483647&Signature=BVkOwlEOOUQ3Al-uUwg3PRFHXJVX55hXH9vGKZnvjjeCuJRbp8E-dUJLZjY0SI3qC-dEcYKlR5gfbGA8XOSwds4rpL4a7x77WIR1FJbzxsv0Vh5BZpmN2Mg7YglJSJzF6ZGC6BYqqh0xt9nWQ7o0-5Ke-eq0pSukmRycjD4~merKkvM8Vsb5qfWUK0hT-FjE3usv8WWg-8s4nZplhO7tSKxwLgQ1SRb1I8bPMNMQLXcA5IK9EfUL0-yZ4wgrMbj5xZwxFRM4h0hXoMOQ9oeUelh55H7v5eB3Z1LdN9OxCMF69MdF19Nv~-FOWuEynANhrovUNKi8PGE6BOAskHOzcA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5499756&ar=377154&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02901.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Experimental Design. Genomic DNA from nine HapMap samples was chosen for the study (three trio families). DNA from one of the samples (NA11881) was prepared twice (with and without an indexed adapter), target enriched and sequenced separately as single samples (non-indexed sample and one indexed sample in Step 1 and enriched libraries 1 and 2 in Step 2). One trio family (NA11881, NA11882 and NA10859; all indexed) was pooled after the Illumina genomic DNA sample prep and enriched together using one SureSelect enrichment reaction to produce the enriched library 3 sample. Indexed DNA from all nine samples was also pooled after the Illumina genomic DNA sample prep and enriched together using one SureSelect enrichment reaction to produce the enriched library 4 sample. Note: enriched libraries 3 and 4 were also sequenced using 80 bp reads to generate additional data for validation of the method for SNP detection.</p></div></div><p>The preparation of each sample is a two-step process; in the first step, the DNA is prepared as an Illumina sequencing library, and in the second step, the sequencing library is enriched for the desired target using the Agilent SureSelect enrichment protocol. The library preparation and enrichment methods were followed according to the Agilent Illumina Single-End Sequencing Platform Library Prep protocol (v1.2 April 2009) with the following modifications: (1) instead of shearing the DNA with a covaris system, a biorupter (Diagenode) was used. The samples were sonicated on high for 30 s and off for 30 s for a total 30 min with addition of ice after every 10 min to keep the samples cool. (2) For the indexed samples (libraries 2–4), the Illumina adapters were replaced with custom made indexed adapters supplied by IDT DNA. During the SureSelect enrichment process, blocking oligos are used (provided by Agilent) to temporarily block the Illumina adapter sequence and prevent off-target pull-down of genomic DNA due to similarity of sequence in the cRNA baits and Illumina adapter sequence. (3) For the pre-capture enrichment PCR, 1.3 µl of DNA was used as input with 11 cycles of PCR instead of 1 µl of input DNA and 14 cycles of PCR. (4) Post-enriched library DNA from the individual samples were combined (one, three or nine samples) to a total quantity of 500 ng per pool. This solution was allowed to evaporate off overnight instead of using a vacuum concentrator and resuspended in 3.4 µl of elution buffer to give a final concentration of ∼147 ng/µl. The individual sample combinations (one, three or nine-samples) were each enriched using one custom SureSelect enrichment reaction (library design ELID: 0236181). Target enriched libraries were stored at a stock concentration of 10 nM ready for sequencing. A total of 6/8 pM of target enriched libraries were sequenced on the Illumina Genome Analyzer II using 40 bp reads following the manufacturers protocol. Libraries 3 and 4 (three and nine samples) were also sequenced using 80 bp reads to generate additional data for the study.</p>                            <h3 scrollto-destination=5499765 id="5499765" class="section-title" data-legacy-id=s2d>2.4. Data analysis</h3>
<p>The base sequence data were called from the image files with the Illumina Bustard.py script and the Illumina GA pipeline version1.4. The RunInfo.xml file was edited to allow for the 6 bp index to be called as a separate index read in the analysis, with matrix and phasing estimated from the PhiX control lane. This allowed the 6 bp index to be ignored in the alignment of the sequences to the reference genome (hg18) to facilitate calculation of basic QC measures, i.e. % clusters passing filters and alignment to hg18 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table SB</a>). Both indexed and non-indexed sequence reads were treated in this way so that the effective read length for samples was 34 bp (for 40 bp reads) and 74 bp (for 80 bp reads). The sequenced reads were then parsed based on the index using an in-house Perl script to allow analysis of the data on an individual per sample basis. SNP detection and generation of data used to determine coverage were performed using MAQ<sup><a reveal-id="DSQ029C13" data-open="DSQ029C13" class="link link-ref link-reveal xref-bibr">13</a></sup> on the individual sample sequence data. The Illumina quality scores were converted to the standard Phred scores required by MAQ by using a modified version of the fq_all2std.pl script supplied with MAQ. A SNP masked reference genome was used for alignment of the reads.</p>                            <h2 scrollto-destination=5499770 id="5499770" class="section-title" data-legacy-id=s3>3. Results</h2>
<p>Figure <a reveal-id="DSQ029F1" data-open="DSQ029F1" class="link link-reveal link-table xref-fig">1</a> outlines the experimental design for this study. Four sequencing libraries were enriched for 377 kb of target sequence. Libraries 1 and 2 each contained the same HapMap DNA sample but differed because library 2 included a 6 bp index. Comparison of sequence data from these indexed and non-indexed samples determined the impact of the index on the efficiency of the enrichment reaction. The third and fourth libraries were multiplex libraries containing three- and nine-indexed HapMap samples, respectively. Sequence data from these libraries were used to assess the performance of the indexes in distributing the reads from a single sequencing reaction to multiple samples and to measure the accuracy of SNP calling in multiplex samples.</p>                            <h3 scrollto-destination=5499773 id="5499773" class="section-title" data-legacy-id=s3a>3.1. Comparison of on-target versus off-target sequence coverage in indexed and non-indexed samples</h3>
<p>Sequence data from the non-indexed DNA sample show that 20% of sequence reads were on-target, i.e. they mapped back to a target region of the enrichment reaction ±50 bp (Table <a reveal-id="DSQ029TB1" data-open="DSQ029TB1" class="link link-reveal link-table xref-fig">1</a>). Across the target regions, there was a 1708-fold enrichment of target DNA in this sample. Ninty-eight percent of the targeted bases were covered by at least one sequence read and on average the target regions were covered to a depth of 169x. We investigated the effect of including a 6 bp index in the adaptor sequence on the efficiency of the enrichment reaction by enriching and sequencing the same DNA sample with an index. The on-target (22%) and fold enrichment (1885) metrics are similar for both samples (Table <a reveal-id="DSQ029TB1" data-open="DSQ029TB1" class="link link-reveal link-table xref-fig">1</a>) indicating that the inclusion of the index did not compromise the performance of the enrichment reaction. Figure <a reveal-id="DSQ029F2" data-open="DSQ029F2" class="link link-reveal link-table xref-fig">2</a>A and B show that for both ‘on-target’ and ‘off-target’ locations, the pattern of sequence coverage obtained is consistent between the single indexed and single non-indexed samples. The higher sequence coverage observed for the non-indexed sample compared with the indexed sample reflects the larger number of clusters (and therefore sequence reads) that passed QC filtering for the non-indexed sample (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table SB</a>). The three-index sample and the nine-index sample libraries had medians of 21 and 18% on-target sequence and medians of 1689- and 1467-fold enrichment, respectively, based on combined 40 and 80 bp read data. Individual sample level data and further quality control information are detailed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables SB and SC</a>. </p>                        <a id="5499780" scrollto-destination="5499780"></a>
<div content-id="DSQ029TB1" class="table-modal"><div class="table-wrap"><div id="DSQ029TB1" class="table-wrap-title" data-id="DSQ029TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Percentage on-target and fold enrichment for each library</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Non-index sample </th><th>One-index sample </th><th>Three-index sample<sup>a</sup> </th><th>Nine-index sample<sup>a</sup> </th></tr></thead><tbody><tr><td>Percentage reads in targeted regions ± 50 bp (%)<sup>b</sup> </td><td>20 </td><td>22 </td><td>21 </td><td>18 </td></tr><tr><td>Fold enrichment in targeted regions<sup>c</sup> </td><td>1708 </td><td>1885 </td><td>1689 </td><td>1467 </td></tr><tr><td>Percentage target bases covered (%)<sup>d</sup> </td><td>98 </td><td>98 </td><td>98 </td><td>98 </td></tr><tr><td>Median coverage of target<sup>e</sup> </td><td>169x<sup>f</sup> </td><td>93x<sup>f</sup> </td><td>164x </td><td>46x </td></tr></tbody></table></div><div content-id="DSQ029TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Non-index sample </th><th>One-index sample </th><th>Three-index sample<sup>a</sup> </th><th>Nine-index sample<sup>a</sup> </th></tr></thead><tbody><tr><td>Percentage reads in targeted regions ± 50 bp (%)<sup>b</sup> </td><td>20 </td><td>22 </td><td>21 </td><td>18 </td></tr><tr><td>Fold enrichment in targeted regions<sup>c</sup> </td><td>1708 </td><td>1885 </td><td>1689 </td><td>1467 </td></tr><tr><td>Percentage target bases covered (%)<sup>d</sup> </td><td>98 </td><td>98 </td><td>98 </td><td>98 </td></tr><tr><td>Median coverage of target<sup>e</sup> </td><td>169x<sup>f</sup> </td><td>93x<sup>f</sup> </td><td>164x </td><td>46x </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Forty and 80 bp data were combined for the three-index and nine-index samples. Average values given for multisample libraries. Individual values are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table SB</a>.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Number of reads uniquely mapping to the target region (±50 bp) as a % of the number of reads uniquely mapping to hg18.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>(Sequence reads uniquely mapping to the target regions/Sequence reads mapping to hg18) × Maximum enrichment where maximum enrichment is a ratio of genome length (3 080 419 510 bp) to target length (377 388 bp).</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>Percentage of target bases covered by at least one sequence read.</p></div></div><div content-id=""><div class="fn"><p><sup>e</sup>(Number of 34 or 74 bp reads matching target × 34 or 74)/target length.</p></div></div><div content-id=""><div class="fn"><p><sup>f</sup>The difference in median read coverage between the non-indexed and indexed sample is reflective of the larger number of clusters on the flowcell and also the larger number of clusters passing QC filters in the non-indexed sample (83.48 versus 57.65%, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table SB</a>).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5499780" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ029TB1" class="table-wrap-title" data-id="DSQ029TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Percentage on-target and fold enrichment for each library</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Non-index sample </th><th>One-index sample </th><th>Three-index sample<sup>a</sup> </th><th>Nine-index sample<sup>a</sup> </th></tr></thead><tbody><tr><td>Percentage reads in targeted regions ± 50 bp (%)<sup>b</sup> </td><td>20 </td><td>22 </td><td>21 </td><td>18 </td></tr><tr><td>Fold enrichment in targeted regions<sup>c</sup> </td><td>1708 </td><td>1885 </td><td>1689 </td><td>1467 </td></tr><tr><td>Percentage target bases covered (%)<sup>d</sup> </td><td>98 </td><td>98 </td><td>98 </td><td>98 </td></tr><tr><td>Median coverage of target<sup>e</sup> </td><td>169x<sup>f</sup> </td><td>93x<sup>f</sup> </td><td>164x </td><td>46x </td></tr></tbody></table></div><div content-id="DSQ029TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Non-index sample </th><th>One-index sample </th><th>Three-index sample<sup>a</sup> </th><th>Nine-index sample<sup>a</sup> </th></tr></thead><tbody><tr><td>Percentage reads in targeted regions ± 50 bp (%)<sup>b</sup> </td><td>20 </td><td>22 </td><td>21 </td><td>18 </td></tr><tr><td>Fold enrichment in targeted regions<sup>c</sup> </td><td>1708 </td><td>1885 </td><td>1689 </td><td>1467 </td></tr><tr><td>Percentage target bases covered (%)<sup>d</sup> </td><td>98 </td><td>98 </td><td>98 </td><td>98 </td></tr><tr><td>Median coverage of target<sup>e</sup> </td><td>169x<sup>f</sup> </td><td>93x<sup>f</sup> </td><td>164x </td><td>46x </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Forty and 80 bp data were combined for the three-index and nine-index samples. Average values given for multisample libraries. Individual values are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table SB</a>.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Number of reads uniquely mapping to the target region (±50 bp) as a % of the number of reads uniquely mapping to hg18.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>(Sequence reads uniquely mapping to the target regions/Sequence reads mapping to hg18) × Maximum enrichment where maximum enrichment is a ratio of genome length (3 080 419 510 bp) to target length (377 388 bp).</p></div></div><div content-id=""><div class="fn"><p><sup>d</sup>Percentage of target bases covered by at least one sequence read.</p></div></div><div content-id=""><div class="fn"><p><sup>e</sup>(Number of 34 or 74 bp reads matching target × 34 or 74)/target length.</p></div></div><div content-id=""><div class="fn"><p><sup>f</sup>The difference in median read coverage between the non-indexed and indexed sample is reflective of the larger number of clusters on the flowcell and also the larger number of clusters passing QC filters in the non-indexed sample (83.48 versus 57.65%, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table SB</a>).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5499780" target="_blank">View Large</a></div></div>                        <a id="5499783" scrollto-destination="5499783"></a>
<div id="DSQ029F2" data-id="DSQ029F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq029/2/m_dsq02902.jpeg?Expires=2147483647&Signature=JMjuQrorbb9LP-dm2ebPpcPCpw~~5s7UJN6R0JyQpkKCnZCm-YQW0mIEI4qAkZtJY8ZC9z2tLMvyf2SnXlPvjO4aW-dGd8AjT3zeMQbW2dPUQQKDUWef53FxqigbuRg1~mH7Jwe44DSeaLQeLm~Pn8PwQTt0SY3s~jsInDIbw38HgYhZOn12gzeA~12gN-PYwf59SdroXbSGL~iiA0IqlnKhpWeLGB1fXX5kS4hRJM5dGEeKszryADEQX57-yptX5F9ukSTvNrJH9mNzNUkeA7RJZJXUDlFEUSNhh8g5hVmbjOwJFf5~4wqvOsnL49ddiwUnQh0pTolf05IsLamu4A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequence coverage across on-target and off-target regions. Sequence coverage is plotted for the single non-indexed and indexed samples at an on-target site (PTBP2 on chromosome 1; A) and at an off-target site (chromosome 12; B). Inclusion of the index does not dramatically change the pattern of sequence coverage at on-target or off-target regions. The higher sequence coverage observed for the non-indexed sample compared with the indexed sample reflects the larger number of clusters that passed QC filtering during the sequence run (Supplementary Table SB)." path-from-xml="dsq02902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5499783/dsq02902.jpeg" path-from-xml="dsq02902.jpeg" target="_blank">View large</a><a section="5499783" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq029/2/dsq02902.jpeg?Expires=2147483647&Signature=XvZ93wFYRIMGxewrscuZ~xDDq5WKV47QGx~4SxmUifSmTfk0OUamMGmsH6OMbRqBigJ0rCe6K27kNkNPZuNcX3YwZ2mDTUEuoN0AVWe1pLe8nt74AzAax5MCqZb18I5DXG5pECHir1UKhc2z2PAOe6LOjxkzSNsZiCrmtMPpVQvh2gzkHg1nWMAsrvKTHzoYaQh4dvifhBafgvVjjOiJvHlxNT68KpzaeJb9T2DO4GFFgRqDH11LU6TrGllSZvVKdMczRJGc~BfNvU0d5~~0jwzmekMcPL-tOstEkrKaBKJEUxpgJKBEfUoB~zjVQRwVXM5yjsU~KL9NWNq6xCKHpQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5499783&ar=377154&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequence coverage across on-target and off-target regions. Sequence coverage is plotted for the single non-indexed and indexed samples at an on-target site (PTBP2 on chromosome 1; A) and at an off-target site (chromosome 12; B). Inclusion of the index does not dramatically change the pattern of sequence coverage at on-target or off-target regions. The higher sequence coverage observed for the non-indexed sample compared with the indexed sample reflects the larger number of clusters that passed QC filtering during the sequence run (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table SB</a>).</p></div></div><div content-id="DSQ029F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq029/2/m_dsq02902.jpeg?Expires=2147483647&Signature=JMjuQrorbb9LP-dm2ebPpcPCpw~~5s7UJN6R0JyQpkKCnZCm-YQW0mIEI4qAkZtJY8ZC9z2tLMvyf2SnXlPvjO4aW-dGd8AjT3zeMQbW2dPUQQKDUWef53FxqigbuRg1~mH7Jwe44DSeaLQeLm~Pn8PwQTt0SY3s~jsInDIbw38HgYhZOn12gzeA~12gN-PYwf59SdroXbSGL~iiA0IqlnKhpWeLGB1fXX5kS4hRJM5dGEeKszryADEQX57-yptX5F9ukSTvNrJH9mNzNUkeA7RJZJXUDlFEUSNhh8g5hVmbjOwJFf5~4wqvOsnL49ddiwUnQh0pTolf05IsLamu4A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequence coverage across on-target and off-target regions. Sequence coverage is plotted for the single non-indexed and indexed samples at an on-target site (PTBP2 on chromosome 1; A) and at an off-target site (chromosome 12; B). Inclusion of the index does not dramatically change the pattern of sequence coverage at on-target or off-target regions. The higher sequence coverage observed for the non-indexed sample compared with the indexed sample reflects the larger number of clusters that passed QC filtering during the sequence run (Supplementary Table SB)." path-from-xml="dsq02902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5499783/dsq02902.jpeg" path-from-xml="dsq02902.jpeg" target="_blank">View large</a><a section="5499783" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq029/2/dsq02902.jpeg?Expires=2147483647&Signature=XvZ93wFYRIMGxewrscuZ~xDDq5WKV47QGx~4SxmUifSmTfk0OUamMGmsH6OMbRqBigJ0rCe6K27kNkNPZuNcX3YwZ2mDTUEuoN0AVWe1pLe8nt74AzAax5MCqZb18I5DXG5pECHir1UKhc2z2PAOe6LOjxkzSNsZiCrmtMPpVQvh2gzkHg1nWMAsrvKTHzoYaQh4dvifhBafgvVjjOiJvHlxNT68KpzaeJb9T2DO4GFFgRqDH11LU6TrGllSZvVKdMczRJGc~BfNvU0d5~~0jwzmekMcPL-tOstEkrKaBKJEUxpgJKBEfUoB~zjVQRwVXM5yjsU~KL9NWNq6xCKHpQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5499783&ar=377154&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequence coverage across on-target and off-target regions. Sequence coverage is plotted for the single non-indexed and indexed samples at an on-target site (PTBP2 on chromosome 1; A) and at an off-target site (chromosome 12; B). Inclusion of the index does not dramatically change the pattern of sequence coverage at on-target or off-target regions. The higher sequence coverage observed for the non-indexed sample compared with the indexed sample reflects the larger number of clusters that passed QC filtering during the sequence run (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table SB</a>).</p></div></div>                            <h3 scrollto-destination=5499791 id="5499791" class="section-title" data-legacy-id=s3b>3.2. Comparison of read count for three-index sample versus nine-index sample</h3>
<p>Figure <a reveal-id="DSQ029F3" data-open="DSQ029F3" class="link link-reveal link-table xref-fig">3</a> shows the performance of the indexing methodology in combination with target enrichment. The numbers of reads attributable to each index/sample are calculated as a percentage of the total number of sequence reads per lane of data. For both the three-index sample and nine-index sample libraries, there is a relatively even share of sequence reads for the individual samples, especially in the nine-index library. This indicates that the enrichment process has been consistent across all samples within each indexed library. The individual read counts uniquely aligning to the hg18 reference, percentage on-target and fold enrichment for each sample in the three- and nine-index sample libraries are detailed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table SC</a>. </p>                        <a id="5499795" scrollto-destination="5499795"></a>
<div id="DSQ029F3" data-id="DSQ029F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq029/2/m_dsq02903.jpeg?Expires=2147483647&Signature=gtDibmot0F1Bq6~U1zgFRh7lgnF9fg9e7jrB1~gRp8U9hPRAsH8CMQFpoUhqGuDvYinHVLcukKC1mjMCVz5R-c8VEZFEh-j3t51JnJWQFbPmbZ5zeimM2h01X~jlzmU4VvwdNL3jDtiTXUE8UuPYZO0gaIdfw1S6LWHmaX~2UaqNc0lzYXJ7NMwlpeGlTQ70zhYeqQGJNqyfglW3rArtnsJIuGHhX6TjyBd8t5J26G3VjWLG3jvhydO1uVlu6DDnGn3lbucToIjPWhW4l9~7fq4PcxSLElCtuWbisOhbmrjpnEG83ffGi1ybFj3ZykMX3JrzK53KYAU84WVHTLTaLw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Percentage of sequence reads per indexed sample in sequenced libraries. Percentage distribution per sample of sequence reads (pre-alignment to reference genome; 40 and 80 bp data combined) for the three-index (A) and the nine-index (B) sample libraries. The relative underperformance of sample NA10859 in the three-index library is not observed in the nine-index library and is unlikely to be due to a systemtaic problem with the ACACAT index." path-from-xml="dsq02903.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5499795/dsq02903.jpeg" path-from-xml="dsq02903.jpeg" target="_blank">View large</a><a section="5499795" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq029/2/dsq02903.jpeg?Expires=2147483647&Signature=fZrE6kgekt-R~6yrKMqYs5Vp4KdL11IwgOo~FAL2df0UOc0vky7lWpj4N~5dQeHHQlC3nmgZ-7dssfSBubY1qOro7~YXsdbzCn1v9bfZlFp7cufZIHzk2DxQz0vCRbb0idOLmz9r8vk-EuOfBD3NsjJo5Ncj1peehvA3t605aGs5PCPr~KaBS3uNjwNx~c7gRMatMEkrSUd1P70C9jQWkEqohtDioTTDBdZSJJKcdOIvAN7xmqk7ZUH85iEQUIHA1c~bD3Wl70yMTQQ~6OeC1Q~SVejXmKkbllHFUpdcy2IB9e0i1WwL7kFQBhLmEGiwvLGjv6JcE1e0kUc88bC~ow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5499795&ar=377154&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02903.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Percentage of sequence reads per indexed sample in sequenced libraries. Percentage distribution per sample of sequence reads (pre-alignment to reference genome; 40 and 80 bp data combined) for the three-index (A) and the nine-index (B) sample libraries. The relative underperformance of sample NA10859 in the three-index library is not observed in the nine-index library and is unlikely to be due to a systemtaic problem with the ACACAT index.</p></div></div><div content-id="DSQ029F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq029/2/m_dsq02903.jpeg?Expires=2147483647&Signature=gtDibmot0F1Bq6~U1zgFRh7lgnF9fg9e7jrB1~gRp8U9hPRAsH8CMQFpoUhqGuDvYinHVLcukKC1mjMCVz5R-c8VEZFEh-j3t51JnJWQFbPmbZ5zeimM2h01X~jlzmU4VvwdNL3jDtiTXUE8UuPYZO0gaIdfw1S6LWHmaX~2UaqNc0lzYXJ7NMwlpeGlTQ70zhYeqQGJNqyfglW3rArtnsJIuGHhX6TjyBd8t5J26G3VjWLG3jvhydO1uVlu6DDnGn3lbucToIjPWhW4l9~7fq4PcxSLElCtuWbisOhbmrjpnEG83ffGi1ybFj3ZykMX3JrzK53KYAU84WVHTLTaLw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Percentage of sequence reads per indexed sample in sequenced libraries. Percentage distribution per sample of sequence reads (pre-alignment to reference genome; 40 and 80 bp data combined) for the three-index (A) and the nine-index (B) sample libraries. The relative underperformance of sample NA10859 in the three-index library is not observed in the nine-index library and is unlikely to be due to a systemtaic problem with the ACACAT index." path-from-xml="dsq02903.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5499795/dsq02903.jpeg" path-from-xml="dsq02903.jpeg" target="_blank">View large</a><a section="5499795" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq029/2/dsq02903.jpeg?Expires=2147483647&Signature=fZrE6kgekt-R~6yrKMqYs5Vp4KdL11IwgOo~FAL2df0UOc0vky7lWpj4N~5dQeHHQlC3nmgZ-7dssfSBubY1qOro7~YXsdbzCn1v9bfZlFp7cufZIHzk2DxQz0vCRbb0idOLmz9r8vk-EuOfBD3NsjJo5Ncj1peehvA3t605aGs5PCPr~KaBS3uNjwNx~c7gRMatMEkrSUd1P70C9jQWkEqohtDioTTDBdZSJJKcdOIvAN7xmqk7ZUH85iEQUIHA1c~bD3Wl70yMTQQ~6OeC1Q~SVejXmKkbllHFUpdcy2IB9e0i1WwL7kFQBhLmEGiwvLGjv6JcE1e0kUc88bC~ow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5499795&ar=377154&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02903.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Percentage of sequence reads per indexed sample in sequenced libraries. Percentage distribution per sample of sequence reads (pre-alignment to reference genome; 40 and 80 bp data combined) for the three-index (A) and the nine-index (B) sample libraries. The relative underperformance of sample NA10859 in the three-index library is not observed in the nine-index library and is unlikely to be due to a systemtaic problem with the ACACAT index.</p></div></div>                            <h3 scrollto-destination=5499796 id="5499796" class="section-title" data-legacy-id=s3c>3.3. Concordance of SNP calls with known HapMap genotypes</h3>
<p>To illustrate the capacity of this method to detect SNPs with very high accuracy, we present data from the sequenced PTBP2 and CDC42 regions. Concordance rates for SNP calls compared with known HapMap genotypes are listed in Table <a reveal-id="DSQ029TB2" data-open="DSQ029TB2" class="link link-reveal link-table xref-fig">2</a>. Only SNPs that had at least one copy of the non-reference allele, a sequencing depth of ≥8x and MAQ base quality score &gt;30 were considered for analysis for each test sample. For the three-index sample library at a sequencing coverage of ≥8x, the concordance rate across the three samples was 99.1%. For the nine-index sample library at ≥8x coverage, the concordance rate was 98.9%. Combined these data give an overall concordance rate of 99% for SNP calls in the three- and nine-indexed samples. If we consider the concordance of SNP calls at all sites, the false-negative rate (i.e. the proportion of HapMap SNPs not detected in the sequence data irrespective of coverage) was 1.4% for the three-index sample library and 4.9% for the nine-index sample library. This assumes that the genotypes in the HapMap database are correct. </p>                        <a id="5499798" scrollto-destination="5499798"></a>
<div content-id="DSQ029TB2" class="table-modal"><div class="table-wrap"><div id="DSQ029TB2" class="table-wrap-title" data-id="DSQ029TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Concordance of SNPs called by MAQ in sequencing data with known HapMap genotypes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Individual ID (# SNPs with at least one non-reference allele in PTBP2 and CDC42 target region for this sample) </th><th colspan="2">Number of concordant SNP calls/number of SNPs with at least 8x coverage and Phred-like consensus quality &gt; 30 (% concordance call)<hr /> </th></tr><tr><th> </th><th>Three-index samples </th><th>Nine-index samples </th></tr></thead><tbody><tr><td>NA11881 (103) </td><td>102/103 (99.0%) </td><td>96/97<sup>a</sup> (98.9%) </td></tr><tr><td>NA11882 (136) </td><td>134/135 (99.3%) </td><td>132/134 (98.5%) </td></tr><tr><td>NA10859 (106) </td><td>104/105 (99.0%) </td><td>98/99 (98.9%) </td></tr><tr><td>NA12144 (103) </td><td> </td><td>96/97 (98.9%) </td></tr><tr><td>NA12239 (57) </td><td> </td><td>54/56 (96.4%) </td></tr><tr><td>NA12145 (111) </td><td> </td><td>107/107 (100%) </td></tr><tr><td>NA10846 (109) </td><td> </td><td>104/105 (99%) </td></tr><tr><td>NA10847 (109) </td><td> </td><td>106/107 (99%) </td></tr><tr><td>NA12146 (106) </td><td> </td><td>101/102 (99%) </td></tr></tbody></table></div><div content-id="DSQ029TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Individual ID (# SNPs with at least one non-reference allele in PTBP2 and CDC42 target region for this sample) </th><th colspan="2">Number of concordant SNP calls/number of SNPs with at least 8x coverage and Phred-like consensus quality &gt; 30 (% concordance call)<hr /> </th></tr><tr><th> </th><th>Three-index samples </th><th>Nine-index samples </th></tr></thead><tbody><tr><td>NA11881 (103) </td><td>102/103 (99.0%) </td><td>96/97<sup>a</sup> (98.9%) </td></tr><tr><td>NA11882 (136) </td><td>134/135 (99.3%) </td><td>132/134 (98.5%) </td></tr><tr><td>NA10859 (106) </td><td>104/105 (99.0%) </td><td>98/99 (98.9%) </td></tr><tr><td>NA12144 (103) </td><td> </td><td>96/97 (98.9%) </td></tr><tr><td>NA12239 (57) </td><td> </td><td>54/56 (96.4%) </td></tr><tr><td>NA12145 (111) </td><td> </td><td>107/107 (100%) </td></tr><tr><td>NA10846 (109) </td><td> </td><td>104/105 (99%) </td></tr><tr><td>NA10847 (109) </td><td> </td><td>106/107 (99%) </td></tr><tr><td>NA12146 (106) </td><td> </td><td>101/102 (99%) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Only 97 of the 103 SNPs had ≥8x coverage and a Phred-like consensus score &gt;30 and were included in concordance analysis.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5499798" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ029TB2" class="table-wrap-title" data-id="DSQ029TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Concordance of SNPs called by MAQ in sequencing data with known HapMap genotypes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Individual ID (# SNPs with at least one non-reference allele in PTBP2 and CDC42 target region for this sample) </th><th colspan="2">Number of concordant SNP calls/number of SNPs with at least 8x coverage and Phred-like consensus quality &gt; 30 (% concordance call)<hr /> </th></tr><tr><th> </th><th>Three-index samples </th><th>Nine-index samples </th></tr></thead><tbody><tr><td>NA11881 (103) </td><td>102/103 (99.0%) </td><td>96/97<sup>a</sup> (98.9%) </td></tr><tr><td>NA11882 (136) </td><td>134/135 (99.3%) </td><td>132/134 (98.5%) </td></tr><tr><td>NA10859 (106) </td><td>104/105 (99.0%) </td><td>98/99 (98.9%) </td></tr><tr><td>NA12144 (103) </td><td> </td><td>96/97 (98.9%) </td></tr><tr><td>NA12239 (57) </td><td> </td><td>54/56 (96.4%) </td></tr><tr><td>NA12145 (111) </td><td> </td><td>107/107 (100%) </td></tr><tr><td>NA10846 (109) </td><td> </td><td>104/105 (99%) </td></tr><tr><td>NA10847 (109) </td><td> </td><td>106/107 (99%) </td></tr><tr><td>NA12146 (106) </td><td> </td><td>101/102 (99%) </td></tr></tbody></table></div><div content-id="DSQ029TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Individual ID (# SNPs with at least one non-reference allele in PTBP2 and CDC42 target region for this sample) </th><th colspan="2">Number of concordant SNP calls/number of SNPs with at least 8x coverage and Phred-like consensus quality &gt; 30 (% concordance call)<hr /> </th></tr><tr><th> </th><th>Three-index samples </th><th>Nine-index samples </th></tr></thead><tbody><tr><td>NA11881 (103) </td><td>102/103 (99.0%) </td><td>96/97<sup>a</sup> (98.9%) </td></tr><tr><td>NA11882 (136) </td><td>134/135 (99.3%) </td><td>132/134 (98.5%) </td></tr><tr><td>NA10859 (106) </td><td>104/105 (99.0%) </td><td>98/99 (98.9%) </td></tr><tr><td>NA12144 (103) </td><td> </td><td>96/97 (98.9%) </td></tr><tr><td>NA12239 (57) </td><td> </td><td>54/56 (96.4%) </td></tr><tr><td>NA12145 (111) </td><td> </td><td>107/107 (100%) </td></tr><tr><td>NA10846 (109) </td><td> </td><td>104/105 (99%) </td></tr><tr><td>NA10847 (109) </td><td> </td><td>106/107 (99%) </td></tr><tr><td>NA12146 (106) </td><td> </td><td>101/102 (99%) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Only 97 of the 103 SNPs had ≥8x coverage and a Phred-like consensus score &gt;30 and were included in concordance analysis.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5499798" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=5499799 id="5499799" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>We present a method where multiple DNA samples are first indexed and combined into one sequencing library and then target enriched using a single reaction. By indexing prior to enrichment, this method (i) dramatically reduces the costs associated with target enrichment and (ii) introduces significant flexibility into the design of targeted next-generation sequencing studies.</p><p>The efficiency of the enrichment reaction is not compromised by the inclusion of the 6 bp index in adaptor sequence as determined by enrichment and sequencing of the same DNA sample with and without an index. We were able to show that use of index-specific blocking oligos during the enrichment process is not necessary. The enrichment process is reasonably consistent across all indexed samples in both the three-index sample and nine-index sample libraries analyzed in this study. The problem of differential ligation efficiency for individual indexes<sup><a reveal-id="DSQ029C11" data-open="DSQ029C11" class="link link-ref link-reveal xref-bibr">11</a></sup> has been resolved here by quantifying each ligated and PCR-enriched sequencing library and then combining individual indexed sample libraries in equimolar amounts prior to target enrichment. In the three-index sample library, the sample with the ACACAT index (NA10859) had approximately one-third less reads than the other two indexed samples in that library. However, in the nine-index sample library, where the read counts for all nine individual samples were within a tight range of each other, the same ACACAT index did not under perform compared with the other indexes. Differences in sample read counts are more likely due to pipetting or quantitation error when the sequencing libraries are initially combined prior to the enrichment process.</p><p>We demonstrate that multiplex target enrichment using DNA indexing can detect SNPs with high accuracy. In accordance with other next-generation sequencing studies, SNP detection is highly dependent on sequence coverage and is more difficult for heterozygous sites.<sup><a reveal-id="DSQ029C14" data-open="DSQ029C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Even at a relatively low coverage of ≥8x, 99% of SNPs were called correctly when data from the PTBP2 and CDC42 loci from the three- and nine-index samples were compared with online HapMap data. This concordance call rate compares well with rates reported for other target enrichment studies: 99.7% for variants with a sequencing coverage &gt;5x, MAQ quality score &gt;30<sup><a reveal-id="DSQ029C14" data-open="DSQ029C14" class="link link-ref link-reveal xref-bibr">14</a></sup>; 99.4% for high-quality calls inferred using Bayesian model<sup><a reveal-id="DSQ029C12" data-open="DSQ029C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and 99.57% for variants with a sequencing coverage &gt;8x, MAQ quality score &gt;30.<sup><a reveal-id="DSQ029C15" data-open="DSQ029C15" class="link link-ref link-reveal xref-bibr">15</a></sup></p><p>Alternative methods amenable to combining multiple target enrichment and sample indexing such as high-throughput microdroplet-based PCR technology<sup><a reveal-id="DSQ029C7" data-open="DSQ029C7" class="link link-ref link-reveal xref-bibr">7</a></sup> and on-array sequence capture<sup><a reveal-id="DSQ029C8" data-open="DSQ029C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSQ029C15" data-open="DSQ029C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSQ029C16" data-open="DSQ029C16" class="link link-ref link-reveal xref-bibr">16</a></sup> require up to 7.5 and 20 µg of input DNA, respectively. These large requirements for input DNA put a significant strain on studies with limited amounts of source biological material. We used 3 µg of input DNA for our method but this could be lowered to 1 µg or less because the input requirement for the enrichment reaction is only 500 ng of Illumina sequencing library. Therefore, we believe our method is ideally suited to multiplex target enrichment of samples with limited DNA resources, e.g. clinical samples.</p><p>The factors that affect sequence coverage, and consequently SNP detection, are the volume of aligned sequence data generated from an experiment and the efficiency of the enrichment reaction. The former can be increased by repeating the sequencing experiment and/or loading a higher quantity of library DNA onto the flowcell to generate more data, sequencing to a longer read length, performing paired-end sequencing and implementing newer calling algorithms that can identify more sequence reads on a flowcell. The efficiency of the enrichment reaction is largely dependent on its design. In this study, we used a relaxed repeat masking protocol in an effort to increase the proportion of the PTBP2 and CDC42 genes that could be sequenced. Although this did allow us to sequence more of these genes than would have been possible with the stricter Agilent default repeat masking settings, it came at a cost of generating more off-target sequence. On-target sequence using this enrichment reaction was ∼21%. This is lower than achieved in other studies that used the SureSelect system (42–50<sup><a reveal-id="DSQ029C12" data-open="DSQ029C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and 40–45%<sup><a reveal-id="DSQ029C14" data-open="DSQ029C14" class="link link-ref link-reveal xref-bibr">14</a></sup>) or Agilent microarray capture (36–76,<sup><a reveal-id="DSQ029C17" data-open="DSQ029C17" class="link link-ref link-reveal xref-bibr">17</a></sup> 36–55<sup><a reveal-id="DSQ029C16" data-open="DSQ029C16" class="link link-ref link-reveal xref-bibr">16</a></sup> and 37–54% for whole exome resequencing<sup><a reveal-id="DSQ029C14" data-open="DSQ029C14" class="link link-ref link-reveal xref-bibr">15</a></sup>). In this study, the reduction in on-target specificity is most likely due to the relaxed repeat masking used in the design of the target enrichment reaction, which resulted in a large proportion of reads mapping to repetitive regions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. SB</a>). The advantage of implementing the default repeat masking settings during SureSelect eArray design will be greater on-target specificity. The disadvantage of these settings is that it will not be possible to target some genomic regions. How punitive this will be depends on the proportion of target regions that contains repetitive sequences. It will certainly impact on studies that plan to sequence entire genes including introns. For example, in this study, just under 40% of CDC42 could be targeted with default repeat masking settings. However, studies targeting exonic sequence at multiple sites will be less influenced by repetitive sequence.</p><p>In summary, we have designed an effective strategy to allow target enrichment and sequencing of multiple samples in parallel using DNA indexing. This method is very accurate for SNP detection. By combining DNA samples prior to enrichment, this method dramatically saves on enrichment and sequencing costs, and the inclusion of the DNA index permits each sample to be analyzed individually downstream. The method is an important inclusion in the range of next-generation sequencing applications below the level of whole exome resequencing but with the capability to sequence the exonic regions of hundreds of genes in tens of samples in a single sequencing experiment.</p>                            <h2 scrollto-destination=5499806 id="5499806" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5499808 id="5499808" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This material is based on works supported by the Science Foundation Ireland (SFI/07/RFP/GEN/F327/EC07) and the Health Research Board (Ireland) (HRB/HRA/2009/45).</p>                            <h2 scrollto-destination=5499810 id="5499810" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSQ029C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ029C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dahl</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Gullberg</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Stenberg</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Landegren</div> <div class="given-names">U.</div></div>,  <div class="name"><div class="surname">Nilsson</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Multiplex amplification enabled by selective circularization of large sets of genomic DNA fragments</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> pg. <div class="fpage">e71</div> <div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gni070" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gni070</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiplex%20amplification%20enabled%20by%20selective%20circularization%20of%20large%20sets%20of%20genomic%20DNA%20fragments&amp;author=F.%20Dahl&amp;author=M.%20Gullberg&amp;author=J.%20Stenberg&amp;author=U.%20Landegren&amp;author=M.%20Nilsson&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=e71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gni070" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgni070" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgni070"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15860768" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ029C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ029C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Landegren</div> <div class="given-names">U.</div></div>,  <div class="name"><div class="surname">Schallmeiner</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Nilsson</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Molecular tools for a molecular medicine: analyzing genes, transcripts and proteins using padlock and proximity probes</div>, <div class="source">J. Mol. Recognit.</div> , <div class="year">2004</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">194</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1002/jmr.664" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1002/jmr.664</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20tools%20for%20a%20molecular%20medicine%3A%20analyzing%20genes%2C%20transcripts%20and%20proteins%20using%20padlock%20and%20proximity%20probes&amp;author=U.%20Landegren&amp;author=E.%20Schallmeiner&amp;author=M.%20Nilsson&amp;publication_year=2004&amp;journal=J.%20Mol.%20Recognit.&amp;volume=17&amp;pages=194-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1099-1352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1099-1352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1099-1352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15137029" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ029C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ029C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">J.B.</div></div>,  <div class="name"><div class="surname">Gao</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Aach</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Multiplex padlock targeted sequencing reveals human hypermutable CpG variations</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1606</div>-<div class="lpage">15</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.092213.109" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.092213.109</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiplex%20padlock%20targeted%20sequencing%20reveals%20human%20hypermutable%20CpG%20variations&amp;author=J.B.%20Li&amp;author=Y.%20Gao&amp;author=J.%20Aach&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=1606-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.092213.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.092213.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.092213.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19525355" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ029C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ029C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lovett</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kere</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Hinton</div> <div class="given-names">L.M.</div></div>. </div><div class="article-title">Direct selection: a method for the isolation of cDNAs encoded by large genomic regions</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1991</div>, vol. <div class="volume">88</div> (pg. <div class="fpage">9628</div>-<div class="lpage">32</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.88.21.9628" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.88.21.9628</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Direct%20selection%3A%20a%20method%20for%20the%20isolation%20of%20cDNAs%20encoded%20by%20large%20genomic%20regions&amp;author=M.%20Lovett&amp;author=J.%20Kere&amp;author=L.M.%20Hinton&amp;publication_year=1991&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=88&amp;pages=9628-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.88.21.9628" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.88.21.9628" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.88.21.9628"> </span></div> </div></div></div></div></div><div content-id="DSQ029C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ029C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Porreca</div> <div class="given-names">G.J.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">J.B.</div></div>, et al. </div><div class="article-title">Multiplex amplification of large sets of human exons</div>, <div class="source">Nat. Methods</div> , <div class="year">2007</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">931</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nmeth1110" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nmeth1110</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiplex%20amplification%20of%20large%20sets%20of%20human%20exons&amp;author=G.J.%20Porreca&amp;author=K.%20Zhang&amp;author=J.B.%20Li&amp;publication_year=2007&amp;journal=Nat.%20Methods&amp;volume=4&amp;pages=931-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth1110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth1110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth1110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17934468" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ029C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ029C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rodriguez</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Guiteau</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">Nazareth</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Sequencing the full-length of the phosphatase and tensin homolog (PTEN) gene in hepatocellular carcinoma (HCC) using the 454 GS20 and Illumina GA DNA sequencing platforms</div>, <div class="source">World J. Surg.</div> , <div class="year">2009</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">647</div>-<div class="lpage">52</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00268-008-9852-x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00268-008-9852-x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20the%20full-length%20of%20the%20phosphatase%20and%20tensin%20homolog%20(PTEN)%20gene%20in%20hepatocellular%20carcinoma%20(HCC)%20using%20the%20454%20GS20%20and%20Illumina%20GA%20DNA%20sequencing%20platforms&amp;author=J.A.%20Rodriguez&amp;author=J.J.%20Guiteau&amp;author=L.%20Nazareth&amp;publication_year=2009&amp;journal=World%20J.%20Surg.&amp;volume=33&amp;pages=647-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00268-008-9852-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00268-008-9852-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00268-008-9852-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19082655" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ029C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ029C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tewhey</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Warner</div> <div class="given-names">J.B.</div></div>,  <div class="name"><div class="surname">Nakano</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Microdroplet-based PCR enrichment for large-scale targeted sequencing</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2009</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">1025</div>-<div class="lpage">31</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nbt.1583" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nbt.1583</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microdroplet-based%20PCR%20enrichment%20for%20large-scale%20targeted%20sequencing&amp;author=R.%20Tewhey&amp;author=J.B.%20Warner&amp;author=M.%20Nakano&amp;publication_year=2009&amp;journal=Nat.%20Biotechnol.&amp;volume=27&amp;pages=1025-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1583" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1583" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1583"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19881494" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ029C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ029C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Turner</div> <div class="given-names">E.H.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Ng</div> <div class="given-names">S.B.</div></div>,  <div class="name"><div class="surname">Nickerson</div> <div class="given-names">D.A.</div></div>,  <div class="name"><div class="surname">Shendure</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Massively parallel exon capture and library-free resequencing across 16 genomes</div>, <div class="source">Nat. Methods</div> , <div class="year">2009</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">315</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nmeth.f.248" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nmeth.f.248</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Massively%20parallel%20exon%20capture%20and%20library-free%20resequencing%20across%2016%20genomes&amp;author=E.H.%20Turner&amp;author=C.%20Lee&amp;author=S.B.%20Ng&amp;author=D.A.%20Nickerson&amp;author=J.%20Shendure&amp;publication_year=2009&amp;journal=Nat.%20Methods&amp;volume=6&amp;pages=315-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.f.248" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.f.248" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.f.248"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19349981" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ029C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ029C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varley</div> <div class="given-names">K.E.</div></div>,  <div class="name"><div class="surname">Mitra</div> <div class="given-names">R.D.</div></div>. </div><div class="article-title">Nested Patch PCR enables highly multiplexed mutation discovery in candidate genes</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">1844</div>-<div class="lpage">50</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.078204.108" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.078204.108</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nested%20Patch%20PCR%20enables%20highly%20multiplexed%20mutation%20discovery%20in%20candidate%20genes&amp;author=K.E.%20Varley&amp;author=R.D.%20Mitra&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=1844-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.078204.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.078204.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.078204.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18849522" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ029C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ029C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mamanova</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Coffey</div> <div class="given-names">A.J.</div></div>,  <div class="name"><div class="surname">Scott</div> <div class="given-names">C.E.</div></div>, et al. </div><div class="article-title">Target-enrichment strategies for next-generation sequencing</div>, <div class="source">Nat. Methods</div> , <div class="year">2010</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">111</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nmeth.1419" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nmeth.1419</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Target-enrichment%20strategies%20for%20next-generation%20sequencing&amp;author=L.%20Mamanova&amp;author=A.J.%20Coffey&amp;author=C.E.%20Scott&amp;publication_year=2010&amp;journal=Nat.%20Methods&amp;volume=7&amp;pages=111-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1419" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1419" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1419"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20111037" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ029C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ029C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Craig</div> <div class="given-names">D.W.</div></div>,  <div class="name"><div class="surname">Pearson</div> <div class="given-names">J.V.</div></div>,  <div class="name"><div class="surname">Szelinger</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Identification of genetic variants using bar-coded multiplexed sequencing</div>, <div class="source">Nat. Methods</div> , <div class="year">2008</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">887</div>-<div class="lpage">93</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nmeth.1251" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nmeth.1251</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20genetic%20variants%20using%20bar-coded%20multiplexed%20sequencing&amp;author=D.W.%20Craig&amp;author=J.V.%20Pearson&amp;author=S.%20Szelinger&amp;publication_year=2008&amp;journal=Nat.%20Methods&amp;volume=5&amp;pages=887-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1251" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1251" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1251"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18794863" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ029C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ029C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gnirke</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Melnikov</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Maguire</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2009</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">182</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nbt.1523" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nbt.1523</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Solution%20hybrid%20selection%20with%20ultra-long%20oligonucleotides%20for%20massively%20parallel%20targeted%20sequencing&amp;author=A.%20Gnirke&amp;author=A.%20Melnikov&amp;author=J.%20Maguire&amp;publication_year=2009&amp;journal=Nat.%20Biotechnol.&amp;volume=27&amp;pages=182-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1523" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1523" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1523"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19182786" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ029C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ029C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ruan</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Mapping short DNA sequencing reads and calling variants using mapping quality scores</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">1851</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.078212.108" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.078212.108</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20short%20DNA%20sequencing%20reads%20and%20calling%20variants%20using%20mapping%20quality%20scores&amp;author=H.%20Li&amp;author=J.%20Ruan&amp;author=R.%20Durbin&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=1851-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.078212.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.078212.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.078212.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18714091" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ029C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ029C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tewhey</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Nakano</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Enrichment of sequencing targets from the human genome by solution hybridization</div>, <div class="source">Genome Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">R116</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/gb-2009-10-10-r116" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/gb-2009-10-10-r116</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enrichment%20of%20sequencing%20targets%20from%20the%20human%20genome%20by%20solution%20hybridization&amp;author=R.%20Tewhey&amp;author=M.%20Nakano&amp;author=X.%20Wang&amp;publication_year=2009&amp;journal=Genome%20Biol.&amp;volume=10&amp;pages=R116" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-10-r116" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-10-r116" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-10-r116"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19835619" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ029C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ029C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ng</div> <div class="given-names">S.B.</div></div>,  <div class="name"><div class="surname">Turner</div> <div class="given-names">E.H.</div></div>,  <div class="name"><div class="surname">Robertson</div> <div class="given-names">P.D.</div></div>, et al. </div><div class="article-title">Targeted capture and massively parallel sequencing of 12 human exomes</div>, <div class="source">Nature</div> , <div class="year">2009</div>, vol. <div class="volume">461</div> (pg. <div class="fpage">272</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature08250" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature08250</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20capture%20and%20massively%20parallel%20sequencing%20of%2012%20human%20exomes&amp;author=S.B.%20Ng&amp;author=E.H.%20Turner&amp;author=P.D.%20Robertson&amp;publication_year=2009&amp;journal=Nature&amp;volume=461&amp;pages=272-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08250" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08250" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08250"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19684571" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ029C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ029C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hodges</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Xuan</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Balija</div> <div class="given-names">V.</div></div>, et al. </div><div class="article-title">Genome-wide in situ exon capture for selective resequencing</div>, <div class="source">Nat. Genet.</div> , <div class="year">2007</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">1522</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/ng.2007.42" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/ng.2007.42</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20in%20situ%20exon%20capture%20for%20selective%20resequencing&amp;author=E.%20Hodges&amp;author=Z.%20Xuan&amp;author=V.%20Balija&amp;publication_year=2007&amp;journal=Nat.%20Genet.&amp;volume=39&amp;pages=1522-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2007.42" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2007.42" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2007.42"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17982454" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ029C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ029C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Albert</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Molla</div> <div class="given-names">M.N.</div></div>,  <div class="name"><div class="surname">Muzny</div> <div class="given-names">D.M.</div></div>, et al. </div><div class="article-title">Direct selection of human genomic loci by microarray hybridization</div>, <div class="source">Nat. Methods</div> , <div class="year">2007</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">903</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nmeth1111" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nmeth1111</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Direct%20selection%20of%20human%20genomic%20loci%20by%20microarray%20hybridization&amp;author=T.J.%20Albert&amp;author=M.N.%20Molla&amp;author=D.M.%20Muzny&amp;publication_year=2007&amp;journal=Nat.%20Methods&amp;volume=4&amp;pages=903-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth1111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth1111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth1111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17934467" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2010. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=377154&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/1/10.1093/dnares/dsq029/2/dsq029-dsq029suppl.pdf?Expires=2147483647&amp;Signature=BDxnMAP4eE8zh-DMV~LSGq2At7mu4H6m0Q-l3VPKzzxdBtZIQmszPseowDpsL5vmXtW8RuS3ABQC4oWTTnggGmGTufjHPktEKqi9cziTzWWJpbz1CmApwn7i7duqhxFeNt6i5FEsFY8qUjCu6nKhX7HVW2cc46zFM625xxmlhv~M4aHJADlJq8BZsl2vA1AmqBghs65~gHbm77xZNT8qsiHD9ZBLE~DTOt10qRGbKlVA72Ma9rPsEANV5I1gzVTHmxQk-bqV03M2WP62ad7su4kCEdR5oKWsnYYyUyMJt3UZtauzTkQN9y7iPg164qMPAs8R-LamrSDNKSv7rwppwg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">316</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000288188700003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">29</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsq029"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="377154"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000288188700003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f18%2f1%2f31%2f377154">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000288188700003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (29)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F18%2F1%2F31%2F377154" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/377154'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Multiplex Target Enrichment Using DNA Indexing for Ultra-High Throughput SNP Detection" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsq029");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f18%2f1%2f31%2f377154">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

