

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/5/487/2236185","@type":"ScholarlyArticle","name":"High-resolution genetic maps of  Lotus japonicus  and  L. burttii  based on re-sequencing of recombinant inbred lines","datePublished":"2016-07-03","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2016-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw033","keywords":["genetic map","recombinant inbred lines","QTL mapping","chromosomal translocations","assembly errors"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/5/487/2236185","author":[{"name":"Shah, Niraj","affiliation":"Center for Carbohydrate Recognition and Signalling, Department of Molecular Biology and Genetics, Aarhus University, DK-8000 Aarhus, Denmark","@type":"Person"},{"name":"Hirakawa, Hideki","affiliation":"Kazusa DNA Research Institute, Chiba 292-0818, Japan","@type":"Person"},{"name":"Kusakabe, Shohei","affiliation":"Graduate School of Life Sciences, Tohoku University, Aoba-ku, Sendai 980-8577, Japan","@type":"Person"},{"name":"Sandal, Niels","affiliation":"Center for Carbohydrate Recognition and Signalling, Department of Molecular Biology and Genetics, Aarhus University, DK-8000 Aarhus, Denmark","@type":"Person"},{"name":"Stougaard, Jens","affiliation":"Center for Carbohydrate Recognition and Signalling, Department of Molecular Biology and Genetics, Aarhus University, DK-8000 Aarhus, Denmark","@type":"Person"},{"name":"Schierup, Mikkel Heide","affiliation":"Bioinformatics Research Centre, Aarhus University, 8000 Aarhus C, Denmark","@type":"Person"},{"name":"Sato, Shusei","affiliation":"Kazusa DNA Research Institute, Chiba 292-0818, Japan  Graduate School of Life Sciences, Tohoku University, Aoba-ku, Sendai 980-8577, Japan","@type":"Person"},{"name":"Andersen, Stig Uggerhøj","affiliation":"Center for Carbohydrate Recognition and Signalling, Department of Molecular Biology and Genetics, Aarhus University, DK-8000 Aarhus, Denmark","@type":"Person"}],"description":"Abstract.  Recombinant inbred lines (RILs) derived from bi-parental populations are stable genetic resources, which are widely used for constructing genetic lin","pageStart":"487","pageEnd":"494","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/5/0/m_cover.gif?Expires=2147483647&Signature=rsT2hJfrKXp2JlrHjh2bKIVwIZog14lNPGdzKP8yhtKMx~Yuyq3H3JzLDrlX5KEjnZvseVd2bAxtAi~Y9hNZXxJ6WDv2KUi8zxPom2lHNNImaLKpGt835Y71zYWnxYx55P8jzDtg5CRs2Bz1K1RPpU1Oz9FnrSPv6bmhcU6uyQ~xYqmBlGBw4TXr9jDHuwu~pd1eot3j91hB3lzFOWU6HfT9gLTdOIn00hhFqmJajweAIiXeiPNSrA~WoW-T9Q1u91QADYwK007qBz-ilhO5nfEW4RyE76MIZCWUuY7cDK1U0tgU8uh~HoZ~1YWG3BLI7M3P2TkqOmJNELz~~k5Mow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"High-resolution genetic maps of  Lotus japonicus  and  L. burttii  based on re-sequencing of recombinant inbred lines","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/5/0/m_cover.gif?Expires=2147483647&Signature=rsT2hJfrKXp2JlrHjh2bKIVwIZog14lNPGdzKP8yhtKMx~Yuyq3H3JzLDrlX5KEjnZvseVd2bAxtAi~Y9hNZXxJ6WDv2KUi8zxPom2lHNNImaLKpGt835Y71zYWnxYx55P8jzDtg5CRs2Bz1K1RPpU1Oz9FnrSPv6bmhcU6uyQ~xYqmBlGBw4TXr9jDHuwu~pd1eot3j91hB3lzFOWU6HfT9gLTdOIn00hhFqmJajweAIiXeiPNSrA~WoW-T9Q1u91QADYwK007qBz-ilhO5nfEW4RyE76MIZCWUuY7cDK1U0tgU8uh~HoZ~1YWG3BLI7M3P2TkqOmJNELz~~k5Mow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="High-resolution genetic maps of  Lotus japonicus  and  L. burttii  based on re-sequencing of recombinant inbred lines" />
    <meta property="og:description" content="Abstract.  Recombinant inbred lines (RILs) derived from bi-parental populations are stable genetic resources, which are widely used for constructing genetic lin" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/5/487/2236185" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-07-03" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Shah, Niraj" /><meta name="citation_author_institution" content="Center for Carbohydrate Recognition and Signalling, Department of Molecular Biology and Genetics, Aarhus University, DK-8000 Aarhus, Denmark" /><meta name="citation_author" content="Hirakawa, Hideki" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kusakabe, Shohei" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, Aoba-ku, Sendai 980-8577, Japan" /><meta name="citation_author" content="Sandal, Niels" /><meta name="citation_author_institution" content="Center for Carbohydrate Recognition and Signalling, Department of Molecular Biology and Genetics, Aarhus University, DK-8000 Aarhus, Denmark" /><meta name="citation_author" content="Stougaard, Jens" /><meta name="citation_author_institution" content="Center for Carbohydrate Recognition and Signalling, Department of Molecular Biology and Genetics, Aarhus University, DK-8000 Aarhus, Denmark" /><meta name="citation_author" content="Schierup, Mikkel Heide" /><meta name="citation_author_institution" content="Bioinformatics Research Centre, Aarhus University, 8000 Aarhus C, Denmark" /><meta name="citation_author" content="Sato, Shusei" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Chiba 292-0818, Japan" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, Aoba-ku, Sendai 980-8577, Japan" /><meta name="citation_author" content="Andersen, Stig Uggerhøj" /><meta name="citation_author_institution" content="Center for Carbohydrate Recognition and Signalling, Department of Molecular Biology and Genetics, Aarhus University, DK-8000 Aarhus, Denmark" /><meta name="citation_title" content="High-resolution genetic maps of Lotus japonicus and L. burttii based on re-sequencing of recombinant inbred lines" /><meta name="citation_firstpage" content="487" /><meta name="citation_lastpage" content="494" /><meta name="citation_doi" content="10.1093/dnares/dsw033" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2016/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Nitrogen in crop production: an account of global flows; citation_author=Smil  V.;  citation_journal_title=Global Biogeochem. Cycles;  citation_year=1999;  citation_volume=13;  citation_issue=2;  citation_pages=647-62; " /><meta name="citation_reference" content="citation_title=Legumes: importance and constraints to greater use,; citation_author=Graham  P.H.; citation_author=Vance  C.P.;  citation_journal_title=Plant Physiol.;  citation_year=2003;  citation_volume=131;  citation_issue=3;  citation_pages=872-7; " /><meta name="citation_reference" content="citation_title=Lotus-Japonicus, an autogamous, diploid legume species for classical and molecular-genetics,; citation_author=Handberg  K.; citation_author=Stougaard  J.;  citation_journal_title=Plant J;  citation_year=1992;  citation_volume=2;  citation_issue=4;  citation_pages=487-96; " /><meta name="citation_reference" content="citation_title=Lotus japonicus as a platform for legume research,; citation_author=Sato  S.; citation_author=Tabata  S.;  citation_journal_title=Curr. Opin. Plant Biol;  citation_year=2006;  citation_volume=9;  citation_issue=2;  citation_pages=128-32; " /><meta name="citation_reference" content="citation_title=Genome analysis of Lotus japonicus,; citation_author=Kawasaki  S.; citation_author=Murakami  Y.;  citation_journal_title=J. Plant Res;  citation_year=2000;  citation_volume=113;  citation_issue=1112;  citation_pages=497-506; " /><meta name="citation_reference" content="citation_title=Structural analysis of a Lotus japonicus genome. I. Sequence features and mapping of fifty-six TAC clones which cover the 5.4 Mb regions of the genome,; citation_author=Sato  S.; citation_author=Kaneko  T.; citation_author=Nakamura  Y.; citation_author=Asamizu  E.; citation_author=Kato  T.; citation_author=Tabata  S.;  citation_journal_title=DNA Res;  citation_year=2001;  citation_volume=8;  citation_issue=6;  citation_pages=311-8; " /><meta name="citation_reference" content="citation_title=Establishment of a Lotus japonicus gene tagging population using the exon-targeting endogenous retrotransposon LORE1,; citation_author=Fukai  E.; citation_author=Soyano  T.; citation_author=Umehara  Y.;  citation_journal_title=Plant J;  citation_year=2012;  citation_volume=69;  citation_issue=4;  citation_pages=720-30; " /><meta name="citation_reference" content="citation_title=Genome-wide LORE1 retrotransposon mutagenesis and high-throughput insertion detection in Lotus japonicus,; citation_author=Urbanski  D.F.; citation_author=Małolepszy  A.; citation_author=Stougaard  J.; citation_author=Andersen  S.U.;  citation_journal_title=Plant J;  citation_year=2012;  citation_volume=69;  citation_issue=4;  citation_pages=731-41; " /><meta name="citation_reference" content="citation_title=A TILLING reverse genetics tool and a web-accessible collection of mutants of the legume Lotus japonicus,; citation_author=Perry  J.A.; citation_author=Wang  T.L.; citation_author=Welham  T.J.;  citation_journal_title=Plant Physiol;  citation_year=2003;  citation_volume=131;  citation_issue=3;  citation_pages=866-71; " /><meta name="citation_reference" content="citation_title=Large scale structural analysis of cDNAs in the model legume, Lotus japonicus,; citation_author=Asamizu  E.; citation_author=Watanabe  M.; citation_author=Tabata  S.;  citation_journal_title=J. Plant Res.;  citation_year=2000;  citation_volume=113;  citation_issue=1112;  citation_pages=451-5; " /><meta name="citation_reference" content="citation_title=Construction of a genetic linkage map of the model legume Lotus japonicus using an intraspecific F-2 population,; citation_author=Hayashi  M.; citation_author=Miyahara  A.; citation_author=Sato  S.;  citation_journal_title=DNA Res;  citation_year=2001;  citation_volume=8;  citation_issue=6;  citation_pages=301-10; " /><meta name="citation_reference" content="citation_title=A genetic linkage map of the model legume Lotus japonicus and strategies for fast mapping of new loci,; citation_author=Sandal  N.; citation_author=Krusell  L.; citation_author=Radutoiu  S.;  citation_journal_title=Genetics;  citation_year=2002;  citation_volume=161;  citation_issue=4;  citation_pages=1673-83; " /><meta name="citation_reference" content="citation_title=Genome structure of the legume, Lotus japonicus,; citation_author=Sato  S.; citation_author=Nakamura  Y.; citation_author=Kaneko  T.;  citation_journal_title=DNA Res;  citation_year=2008;  citation_volume=15;  citation_issue=4;  citation_pages=227-39; " /><meta name="citation_reference" content="citation_title=A set of Lotus japonicus Gifu x Lotus burttii recombinant inbred lines facilitates map-based cloning and QTL mapping,; citation_author=Sandal  N.; citation_author=Jin  H.; citation_author=Rodriguez-Navarro  D.N.;  citation_journal_title=DNA Res;  citation_year=2012;  citation_volume=19;  citation_issue=4;  citation_pages=317-23; " /><meta name="citation_reference" content="citation_title=Classical and molecular genetics of the model legume Lotus japonicus,; citation_author=Jiang  Q.Y.; citation_author=Gresshoff  P.M.;  citation_journal_title=Mol. Plant Microbe Interact;  citation_year=1997;  citation_volume=10;  citation_issue=1;  citation_pages=59-68; " /><meta name="citation_reference" content="citation_title=Genetics of symbiosis in Lotus japonicus: recombinant inbred lines, comparative genetic maps, and map position of 35 symbiotic loci,; citation_author=Sandal  N.; citation_author=Petersen  T.R.; citation_author=Murray  J.;  citation_journal_title=Mol Plant Microbe Interact;  citation_year=2006;  citation_volume=19;  citation_issue=1;  citation_pages=80-91; " /><meta name="citation_reference" content="citation_title=Quantitative trait locus analysis of multiple agronomic traits in the model legume Lotus japonicus,; citation_author=Gondo  T.; citation_author=Sato  S.; citation_author=Okumura  K.; citation_author=Tabata  S.; citation_author=Akashi  R.; citation_author=Isobe  S.;  citation_journal_title=Genome;  citation_year=2007;  citation_volume=50;  citation_issue=7;  citation_pages=627-37; " /><meta name="citation_reference" content="citation_title=A high density integrated genetic linkage map of soybean and the development of a 1536 universal soy linkage panel for quantitative trait locus mapping; citation_author=Hyten  D.L.; citation_author=Choi  I.-Y.; citation_author=Song  Q.; citation_author=Specht  J.E.; citation_author=Carter  T.E.;  citation_journal_title=Crop Sci;  citation_year=2010;  citation_volume=50;  citation_issue=3;  citation_pages=960-8; " /><meta name="citation_reference" content="citation_title=A saturated genetic linkage map of autotetraploid alfalfa (Medicago sativa L.) developed using genotyping-by-sequencing is highly syntenous with the medicago truncatula genome; citation_author=Li  X.H.; citation_author=Wei  Y.; citation_author=Acharya  A.; citation_author=Jiang  Q.; citation_author=Kang  J.; citation_author=Brummer  E.C.;  citation_journal_title=G3-Genes Genomes Genet;  citation_year=2014;  citation_volume=4;  citation_issue=10;  citation_pages=1971-9; " /><meta name="citation_reference" content="citation_title=High density linkage mapping of genomic and transcriptomic SNPs for synteny analysis and anchoring the genome sequence of chickpea,; citation_author=Gaur  R.; citation_author=Jeena  G.; citation_author=Shah  N.; citation_author=Gupta  S.; citation_author=Pradhan  S.; citation_author=Tyagi  A. K.; citation_author=Jain,  M.; citation_author=Chattopadhyay  D.; citation_author=Bhatia  S.;  citation_journal_title=Sci. Rep;  citation_year=2015;  citation_volume=5;  citation_pages=13387" /><meta name="citation_reference" content="citation_title=A high-resolution map of Arabidopsis recombinant inbred lines by whole-genome exon array hybridization,; citation_author=Singer  T.; citation_author=Fan  Y.; citation_author=Chang  H.S.; citation_author=Zhu  T.; citation_author=Hazen  S.P.; citation_author=Briggs  S.P.;  citation_journal_title=Plos Genet.;  citation_year=2006;  citation_volume=2;  citation_issue=9;  citation_pages=1352-61; " /><meta name="citation_reference" content="citation_title=QTL mapping of leafy heads by genome resequencing in the RIL population of Brassica rapa,; citation_author=Yu  X.; citation_author=Wang  H.; citation_author=Zhong  W.; citation_author=Bai  J.; citation_author=Liu  P.; citation_author=He  Y.;  citation_journal_title=Plos One;  citation_year=2013;  citation_volume=8;  citation_issue=10;  citation_pages=e76059" /><meta name="citation_reference" content="citation_title=High-throughput genotyping by whole-genome resequencing,; citation_author=Huang  X.H.; citation_author=Feng  Q.; citation_author=Qian  Q.;  citation_journal_title=Genome Res;  citation_year=2009;  citation_volume=19;  citation_issue=6;  citation_pages=1068-76; " /><meta name="citation_reference" content="citation_title=Lotus japonicus 'Miyakojima' MG-20: an early-flowering accession suitable for indoor handling; citation_author=Kawaguchi  M.;  citation_journal_title=J. Plant Res;  citation_year=2000;  citation_volume=113;  citation_issue=1112;  citation_pages=507-9; " /><meta name="citation_reference" content="citation_title=New diploid species of Lotus (Leguminosae) in Pakistan; citation_author=Borsos  O.S.; citation_author=Somaroo  B.H.; citation_author=Grant  W.F.;  citation_journal_title=Can. J. Botany;  citation_year=1972;  citation_volume=50;  citation_issue=9;  citation_pages=1865-1870; " /><meta name="citation_reference" content="citation_title=Lotus burttii takes a position of the third corner in the lotus molecular genetics triangle,; citation_author=Kawaguchi  M.; citation_author=Pedrosa-Harand  A.; citation_author=Yano  K.;  citation_journal_title=DNA Res;  citation_year=2005;  citation_volume=12;  citation_issue=1;  citation_pages=69-77; " /><meta name="citation_reference" content="citation_title=Rapid isolation of high molecular-weight plant DNA,; citation_author=Murray  M.G.; citation_author=Thompson  W.F.;  citation_journal_title=Nucleic Acids Res;  citation_year=1980;  citation_volume=8;  citation_issue=19;  citation_pages=4321-5; " /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform,; citation_author=Li  H.; citation_author=Durbin  R.;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_issue=14;  citation_pages=1754-60; " /><meta name="citation_reference" content="citation_title=The sequence alignment/map format and SAMtools,; citation_author=Li  H.; citation_author=Handsaker  B.; citation_author=Wysoke  A.;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_issue=16;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=R/qtl: QTL mapping in experimental crosses,; citation_author=Broman  K.W.; citation_author=Wu  H.; citation_author=Sen  S.; citation_author=Churchill  G.A.;  citation_journal_title=Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_issue=7;  citation_pages=889-90; " /><meta name="citation_reference" content="citation_title=Chromosomal map of the model legume Lotus japonicus; citation_author=Pedrosa  A.; citation_author=Sandal  N.; citation_author=Stougaard  J.; citation_author=Schweizer  D.; citation_author=Bachmair  A.;  citation_journal_title=Genetics;  citation_year=2002;  citation_volume=161;  citation_issue=4;  citation_pages=1661-72; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="description" content="Abstract.  Recombinant inbred lines (RILs) derived from bi-parental populations are stable genetic resources, which are widely used for constructing genetic lin" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/5/487/2236185" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/5/487/2236185" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/5/487/2236185");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw033';
    </script>
<title>
	High-resolution genetic maps of Lotus japonicus and L. burttii based on re-sequencing of recombinant inbred lines | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2236185" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BaZwPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGFoLCBOaXJhaiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNlbnRlciBmb3IgQ2FyYm9oeWRyYXRlIFJlY29nbml0aW9uIGFuZCBTaWduYWxsaW5nLCBEZXBhcnRtZW50IG9mIE1vbGVjdWxhciBCaW9sb2d5IGFuZCBHZW5ldGljcywgQWFyaHVzIFVuaXZlcnNpdHksIERLLTgwMDAgQWFyaHVzLCBEZW5tYXJrIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGlyYWthd2EsIEhpZGVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLdXNha2FiZSwgU2hvaGVpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIExpZmUgU2NpZW5jZXMsIFRvaG9rdSBVbml2ZXJzaXR5LCBBb2JhLWt1LCBTZW5kYWkgOTgwLTg1NzcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2FuZGFsLCBOaWVscyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNlbnRlciBmb3IgQ2FyYm9oeWRyYXRlIFJlY29nbml0aW9uIGFuZCBTaWduYWxsaW5nLCBEZXBhcnRtZW50IG9mIE1vbGVjdWxhciBCaW9sb2d5IGFuZCBHZW5ldGljcywgQWFyaHVzIFVuaXZlcnNpdHksIERLLTgwMDAgQWFyaHVzLCBEZW5tYXJrIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3RvdWdhYXJkLCBKZW5zIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudGVyIGZvciBDYXJib2h5ZHJhdGUgUmVjb2duaXRpb24gYW5kIFNpZ25hbGxpbmcsIERlcGFydG1lbnQgb2YgTW9sZWN1bGFyIEJpb2xvZ3kgYW5kIEdlbmV0aWNzLCBBYXJodXMgVW5pdmVyc2l0eSwgREstODAwMCBBYXJodXMsIERlbm1hcmsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTY2hpZXJ1cCwgTWlra2VsIEhlaWRlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvaW5mb3JtYXRpY3MgUmVzZWFyY2ggQ2VudHJlLCBBYXJodXMgVW5pdmVyc2l0eSwgODAwMCBBYXJodXMgQywgRGVubWFyayIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNhdG8sIFNodXNlaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgTGlmZSBTY2llbmNlcywgVG9ob2t1IFVuaXZlcnNpdHksIEFvYmEta3UsIFNlbmRhaSA5ODAtODU3NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBbmRlcnNlbiwgU3RpZyBVZ2dlcmjDuGoiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50ZXIgZm9yIENhcmJvaHlkcmF0ZSBSZWNvZ25pdGlvbiBhbmQgU2lnbmFsbGluZywgRGVwYXJ0bWVudCBvZiBNb2xlY3VsYXIgQmlvbG9neSBhbmQgR2VuZXRpY3MsIEFhcmh1cyBVbml2ZXJzaXR5LCBESy04MDAwIEFhcmh1cywgRGVubWFyayIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iSGlnaC1yZXNvbHV0aW9uIGdlbmV0aWMgbWFwcyBvZiBMb3R1cyBqYXBvbmljdXMgYW5kIEwuIGJ1cnR0aWkgYmFzZWQgb24gcmUtc2VxdWVuY2luZyBvZiByZWNvbWJpbmFudCBpbmJyZWQgbGluZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI0ODciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjQ5NCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdzAzMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE2LzEwLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tml0cm9nZW4gaW4gY3JvcCBwcm9kdWN0aW9uOiBhbiBhY2NvdW50IG9mIGdsb2JhbCBmbG93czsgY2l0YXRpb25fYXV0aG9yPVNtaWwgIFYuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HbG9iYWwgQmlvZ2VvY2hlbS4gQ3ljbGVzOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25faXNzdWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTY0Ny02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGVndW1lczogaW1wb3J0YW5jZSBhbmQgY29uc3RyYWludHMgdG8gZ3JlYXRlciB1c2UsOyBjaXRhdGlvbl9hdXRob3I9R3JhaGFtICBQLkguOyBjaXRhdGlvbl9hdXRob3I9VmFuY2UgIEMuUC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEzMTsgIGNpdGF0aW9uX2lzc3VlPTM7ICBjaXRhdGlvbl9wYWdlcz04NzItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG90dXMtSmFwb25pY3VzLCBhbiBhdXRvZ2Ftb3VzLCBkaXBsb2lkIGxlZ3VtZSBzcGVjaWVzIGZvciBjbGFzc2ljYWwgYW5kIG1vbGVjdWxhci1nZW5ldGljcyw7IGNpdGF0aW9uX2F1dGhvcj1IYW5kYmVyZyAgSy47IGNpdGF0aW9uX2F1dGhvcj1TdG91Z2FhcmQgIEouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9pc3N1ZT00OyAgY2l0YXRpb25fcGFnZXM9NDg3LTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Mb3R1cyBqYXBvbmljdXMgYXMgYSBwbGF0Zm9ybSBmb3IgbGVndW1lIHJlc2VhcmNoLDsgY2l0YXRpb25fYXV0aG9yPVNhdG8gIFMuOyBjaXRhdGlvbl9hdXRob3I9VGFiYXRhICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gUGxhbnQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25faXNzdWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTEyOC0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIGFuYWx5c2lzIG9mIExvdHVzIGphcG9uaWN1cyw7IGNpdGF0aW9uX2F1dGhvcj1LYXdhc2FraSAgUy47IGNpdGF0aW9uX2F1dGhvcj1NdXJha2FtaSAgWS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIFBsYW50IFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTM7ICBjaXRhdGlvbl9pc3N1ZT0xMTEyOyAgY2l0YXRpb25fcGFnZXM9NDk3LTUwNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJhbCBhbmFseXNpcyBvZiBhIExvdHVzIGphcG9uaWN1cyBnZW5vbWUuIEkuIFNlcXVlbmNlIGZlYXR1cmVzIGFuZCBtYXBwaW5nIG9mIGZpZnR5LXNpeCBUQUMgY2xvbmVzIHdoaWNoIGNvdmVyIHRoZSA1LjQgTWIgcmVnaW9ucyBvZiB0aGUgZ2Vub21lLDsgY2l0YXRpb25fYXV0aG9yPVNhdG8gIFMuOyBjaXRhdGlvbl9hdXRob3I9S2FuZWtvICBULjsgY2l0YXRpb25fYXV0aG9yPU5ha2FtdXJhICBZLjsgY2l0YXRpb25fYXV0aG9yPUFzYW1penUgIEUuOyBjaXRhdGlvbl9hdXRob3I9S2F0byAgVC47IGNpdGF0aW9uX2F1dGhvcj1UYWJhdGEgIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9pc3N1ZT02OyAgY2l0YXRpb25fcGFnZXM9MzExLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzdGFibGlzaG1lbnQgb2YgYSBMb3R1cyBqYXBvbmljdXMgZ2VuZSB0YWdnaW5nIHBvcHVsYXRpb24gdXNpbmcgdGhlIGV4b24tdGFyZ2V0aW5nIGVuZG9nZW5vdXMgcmV0cm90cmFuc3Bvc29uIExPUkUxLDsgY2l0YXRpb25fYXV0aG9yPUZ1a2FpICBFLjsgY2l0YXRpb25fYXV0aG9yPVNveWFubyAgVC47IGNpdGF0aW9uX2F1dGhvcj1VbWVoYXJhICBZLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgSjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT02OTsgIGNpdGF0aW9uX2lzc3VlPTQ7ICBjaXRhdGlvbl9wYWdlcz03MjAtMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIExPUkUxIHJldHJvdHJhbnNwb3NvbiBtdXRhZ2VuZXNpcyBhbmQgaGlnaC10aHJvdWdocHV0IGluc2VydGlvbiBkZXRlY3Rpb24gaW4gTG90dXMgamFwb25pY3VzLDsgY2l0YXRpb25fYXV0aG9yPVVyYmFuc2tpICBELkYuOyBjaXRhdGlvbl9hdXRob3I9TWHFgm9sZXBzenkgIEEuOyBjaXRhdGlvbl9hdXRob3I9U3RvdWdhYXJkICBKLjsgY2l0YXRpb25fYXV0aG9yPUFuZGVyc2VuICBTLlUuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTY5OyAgY2l0YXRpb25faXNzdWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTczMS00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBUSUxMSU5HIHJldmVyc2UgZ2VuZXRpY3MgdG9vbCBhbmQgYSB3ZWItYWNjZXNzaWJsZSBjb2xsZWN0aW9uIG9mIG11dGFudHMgb2YgdGhlIGxlZ3VtZSBMb3R1cyBqYXBvbmljdXMsOyBjaXRhdGlvbl9hdXRob3I9UGVycnkgIEouQS47IGNpdGF0aW9uX2F1dGhvcj1XYW5nICBULkwuOyBjaXRhdGlvbl9hdXRob3I9V2VsaGFtICBULkouOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEzMTsgIGNpdGF0aW9uX2lzc3VlPTM7ICBjaXRhdGlvbl9wYWdlcz04NjYtNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxhcmdlIHNjYWxlIHN0cnVjdHVyYWwgYW5hbHlzaXMgb2YgY0ROQXMgaW4gdGhlIG1vZGVsIGxlZ3VtZSwgTG90dXMgamFwb25pY3VzLDsgY2l0YXRpb25fYXV0aG9yPUFzYW1penUgIEUuOyBjaXRhdGlvbl9hdXRob3I9V2F0YW5hYmUgIE0uOyBjaXRhdGlvbl9hdXRob3I9VGFiYXRhICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gUGxhbnQgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTM7ICBjaXRhdGlvbl9pc3N1ZT0xMTEyOyAgY2l0YXRpb25fcGFnZXM9NDUxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBhIGdlbmV0aWMgbGlua2FnZSBtYXAgb2YgdGhlIG1vZGVsIGxlZ3VtZSBMb3R1cyBqYXBvbmljdXMgdXNpbmcgYW4gaW50cmFzcGVjaWZpYyBGLTIgcG9wdWxhdGlvbiw7IGNpdGF0aW9uX2F1dGhvcj1IYXlhc2hpICBNLjsgY2l0YXRpb25fYXV0aG9yPU1peWFoYXJhICBBLjsgY2l0YXRpb25fYXV0aG9yPVNhdG8gIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9pc3N1ZT02OyAgY2l0YXRpb25fcGFnZXM9MzAxLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbmV0aWMgbGlua2FnZSBtYXAgb2YgdGhlIG1vZGVsIGxlZ3VtZSBMb3R1cyBqYXBvbmljdXMgYW5kIHN0cmF0ZWdpZXMgZm9yIGZhc3QgbWFwcGluZyBvZiBuZXcgbG9jaSw7IGNpdGF0aW9uX2F1dGhvcj1TYW5kYWwgIE4uOyBjaXRhdGlvbl9hdXRob3I9S3J1c2VsbCAgTC47IGNpdGF0aW9uX2F1dGhvcj1SYWR1dG9pdSAgUy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE2MTsgIGNpdGF0aW9uX2lzc3VlPTQ7ICBjaXRhdGlvbl9wYWdlcz0xNjczLTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc3RydWN0dXJlIG9mIHRoZSBsZWd1bWUsIExvdHVzIGphcG9uaWN1cyw7IGNpdGF0aW9uX2F1dGhvcj1TYXRvICBTLjsgY2l0YXRpb25fYXV0aG9yPU5ha2FtdXJhICBZLjsgY2l0YXRpb25fYXV0aG9yPUthbmVrbyAgVC47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9pc3N1ZT00OyAgY2l0YXRpb25fcGFnZXM9MjI3LTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNldCBvZiBMb3R1cyBqYXBvbmljdXMgR2lmdSB4IExvdHVzIGJ1cnR0aWkgcmVjb21iaW5hbnQgaW5icmVkIGxpbmVzIGZhY2lsaXRhdGVzIG1hcC1iYXNlZCBjbG9uaW5nIGFuZCBRVEwgbWFwcGluZyw7IGNpdGF0aW9uX2F1dGhvcj1TYW5kYWwgIE4uOyBjaXRhdGlvbl9hdXRob3I9SmluICBILjsgY2l0YXRpb25fYXV0aG9yPVJvZHJpZ3Vlei1OYXZhcnJvICBELk4uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25faXNzdWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTMxNy0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2xhc3NpY2FsIGFuZCBtb2xlY3VsYXIgZ2VuZXRpY3Mgb2YgdGhlIG1vZGVsIGxlZ3VtZSBMb3R1cyBqYXBvbmljdXMsOyBjaXRhdGlvbl9hdXRob3I9SmlhbmcgIFEuWS47IGNpdGF0aW9uX2F1dGhvcj1HcmVzc2hvZmYgIFAuTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gUGxhbnQgTWljcm9iZSBJbnRlcmFjdDsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX2lzc3VlPTE7ICBjaXRhdGlvbl9wYWdlcz01OS02ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpY3Mgb2Ygc3ltYmlvc2lzIGluIExvdHVzIGphcG9uaWN1czogcmVjb21iaW5hbnQgaW5icmVkIGxpbmVzLCBjb21wYXJhdGl2ZSBnZW5ldGljIG1hcHMsIGFuZCBtYXAgcG9zaXRpb24gb2YgMzUgc3ltYmlvdGljIGxvY2ksOyBjaXRhdGlvbl9hdXRob3I9U2FuZGFsICBOLjsgY2l0YXRpb25fYXV0aG9yPVBldGVyc2VuICBULlIuOyBjaXRhdGlvbl9hdXRob3I9TXVycmF5ICBKLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sIFBsYW50IE1pY3JvYmUgSW50ZXJhY3Q7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9pc3N1ZT0xOyAgY2l0YXRpb25fcGFnZXM9ODAtOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVF1YW50aXRhdGl2ZSB0cmFpdCBsb2N1cyBhbmFseXNpcyBvZiBtdWx0aXBsZSBhZ3Jvbm9taWMgdHJhaXRzIGluIHRoZSBtb2RlbCBsZWd1bWUgTG90dXMgamFwb25pY3VzLDsgY2l0YXRpb25fYXV0aG9yPUdvbmRvICBULjsgY2l0YXRpb25fYXV0aG9yPVNhdG8gIFMuOyBjaXRhdGlvbl9hdXRob3I9T2t1bXVyYSAgSy47IGNpdGF0aW9uX2F1dGhvcj1UYWJhdGEgIFMuOyBjaXRhdGlvbl9hdXRob3I9QWthc2hpICBSLjsgY2l0YXRpb25fYXV0aG9yPUlzb2JlICBTLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTUwOyAgY2l0YXRpb25faXNzdWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTYyNy0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoaWdoIGRlbnNpdHkgaW50ZWdyYXRlZCBnZW5ldGljIGxpbmthZ2UgbWFwIG9mIHNveWJlYW4gYW5kIHRoZSBkZXZlbG9wbWVudCBvZiBhIDE1MzYgdW5pdmVyc2FsIHNveSBsaW5rYWdlIHBhbmVsIGZvciBxdWFudGl0YXRpdmUgdHJhaXQgbG9jdXMgbWFwcGluZzsgY2l0YXRpb25fYXV0aG9yPUh5dGVuICBELkwuOyBjaXRhdGlvbl9hdXRob3I9Q2hvaSAgSS4tWS47IGNpdGF0aW9uX2F1dGhvcj1Tb25nICBRLjsgY2l0YXRpb25fYXV0aG9yPVNwZWNodCAgSi5FLjsgY2l0YXRpb25fYXV0aG9yPUNhcnRlciAgVC5FLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3JvcCBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTA7ICBjaXRhdGlvbl9pc3N1ZT0zOyAgY2l0YXRpb25fcGFnZXM9OTYwLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc2F0dXJhdGVkIGdlbmV0aWMgbGlua2FnZSBtYXAgb2YgYXV0b3RldHJhcGxvaWQgYWxmYWxmYSAoTWVkaWNhZ28gc2F0aXZhIEwuKSBkZXZlbG9wZWQgdXNpbmcgZ2Vub3R5cGluZy1ieS1zZXF1ZW5jaW5nIGlzIGhpZ2hseSBzeW50ZW5vdXMgd2l0aCB0aGUgbWVkaWNhZ28gdHJ1bmNhdHVsYSBnZW5vbWU7IGNpdGF0aW9uX2F1dGhvcj1MaSAgWC5ILjsgY2l0YXRpb25fYXV0aG9yPVdlaSAgWS47IGNpdGF0aW9uX2F1dGhvcj1BY2hhcnlhICBBLjsgY2l0YXRpb25fYXV0aG9yPUppYW5nICBRLjsgY2l0YXRpb25fYXV0aG9yPUthbmcgIEouOyBjaXRhdGlvbl9hdXRob3I9QnJ1bW1lciAgRS5DLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RzMtR2VuZXMgR2Vub21lcyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25faXNzdWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xOTcxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2ggZGVuc2l0eSBsaW5rYWdlIG1hcHBpbmcgb2YgZ2Vub21pYyBhbmQgdHJhbnNjcmlwdG9taWMgU05QcyBmb3Igc3ludGVueSBhbmFseXNpcyBhbmQgYW5jaG9yaW5nIHRoZSBnZW5vbWUgc2VxdWVuY2Ugb2YgY2hpY2twZWEsOyBjaXRhdGlvbl9hdXRob3I9R2F1ciAgUi47IGNpdGF0aW9uX2F1dGhvcj1KZWVuYSAgRy47IGNpdGF0aW9uX2F1dGhvcj1TaGFoICBOLjsgY2l0YXRpb25fYXV0aG9yPUd1cHRhICBTLjsgY2l0YXRpb25fYXV0aG9yPVByYWRoYW4gIFMuOyBjaXRhdGlvbl9hdXRob3I9VHlhZ2kgIEEuIEsuOyBjaXRhdGlvbl9hdXRob3I9SmFpbiwgIE0uOyBjaXRhdGlvbl9hdXRob3I9Q2hhdHRvcGFkaHlheSAgRC47IGNpdGF0aW9uX2F1dGhvcj1CaGF0aWEgIFMuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2kuIFJlcDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTMzODciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGhpZ2gtcmVzb2x1dGlvbiBtYXAgb2YgQXJhYmlkb3BzaXMgcmVjb21iaW5hbnQgaW5icmVkIGxpbmVzIGJ5IHdob2xlLWdlbm9tZSBleG9uIGFycmF5IGh5YnJpZGl6YXRpb24sOyBjaXRhdGlvbl9hdXRob3I9U2luZ2VyICBULjsgY2l0YXRpb25fYXV0aG9yPUZhbiAgWS47IGNpdGF0aW9uX2F1dGhvcj1DaGFuZyAgSC5TLjsgY2l0YXRpb25fYXV0aG9yPVpodSAgVC47IGNpdGF0aW9uX2F1dGhvcj1IYXplbiAgUy5QLjsgY2l0YXRpb25fYXV0aG9yPUJyaWdncyAgUy5QLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxvcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX2lzc3VlPTk7ICBjaXRhdGlvbl9wYWdlcz0xMzUyLTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RVEwgbWFwcGluZyBvZiBsZWFmeSBoZWFkcyBieSBnZW5vbWUgcmVzZXF1ZW5jaW5nIGluIHRoZSBSSUwgcG9wdWxhdGlvbiBvZiBCcmFzc2ljYSByYXBhLDsgY2l0YXRpb25fYXV0aG9yPVl1ICBYLjsgY2l0YXRpb25fYXV0aG9yPVdhbmcgIEguOyBjaXRhdGlvbl9hdXRob3I9WmhvbmcgIFcuOyBjaXRhdGlvbl9hdXRob3I9QmFpICBKLjsgY2l0YXRpb25fYXV0aG9yPUxpdSAgUC47IGNpdGF0aW9uX2F1dGhvcj1IZSAgWS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsb3MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9pc3N1ZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPWU3NjA1OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2gtdGhyb3VnaHB1dCBnZW5vdHlwaW5nIGJ5IHdob2xlLWdlbm9tZSByZXNlcXVlbmNpbmcsOyBjaXRhdGlvbl9hdXRob3I9SHVhbmcgIFguSC47IGNpdGF0aW9uX2F1dGhvcj1GZW5nICBRLjsgY2l0YXRpb25fYXV0aG9yPVFpYW4gIFEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25faXNzdWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTEwNjgtNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvdHVzIGphcG9uaWN1cyAnTWl5YWtvamltYScgTUctMjA6IGFuIGVhcmx5LWZsb3dlcmluZyBhY2Nlc3Npb24gc3VpdGFibGUgZm9yIGluZG9vciBoYW5kbGluZzsgY2l0YXRpb25fYXV0aG9yPUthd2FndWNoaSAgTS47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIFBsYW50IFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTM7ICBjaXRhdGlvbl9pc3N1ZT0xMTEyOyAgY2l0YXRpb25fcGFnZXM9NTA3LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5ldyBkaXBsb2lkIHNwZWNpZXMgb2YgTG90dXMgKExlZ3VtaW5vc2FlKSBpbiBQYWtpc3RhbjsgY2l0YXRpb25fYXV0aG9yPUJvcnNvcyAgTy5TLjsgY2l0YXRpb25fYXV0aG9yPVNvbWFyb28gIEIuSC47IGNpdGF0aW9uX2F1dGhvcj1HcmFudCAgVy5GLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2FuLiBKLiBCb3Rhbnk7ICBjaXRhdGlvbl95ZWFyPTE5NzI7ICBjaXRhdGlvbl92b2x1bWU9NTA7ICBjaXRhdGlvbl9pc3N1ZT05OyAgY2l0YXRpb25fcGFnZXM9MTg2NS0xODcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Mb3R1cyBidXJ0dGlpIHRha2VzIGEgcG9zaXRpb24gb2YgdGhlIHRoaXJkIGNvcm5lciBpbiB0aGUgbG90dXMgbW9sZWN1bGFyIGdlbmV0aWNzIHRyaWFuZ2xlLDsgY2l0YXRpb25fYXV0aG9yPUthd2FndWNoaSAgTS47IGNpdGF0aW9uX2F1dGhvcj1QZWRyb3NhLUhhcmFuZCAgQS47IGNpdGF0aW9uX2F1dGhvcj1ZYW5vICBLLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX2lzc3VlPTE7ICBjaXRhdGlvbl9wYWdlcz02OS03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmFwaWQgaXNvbGF0aW9uIG9mIGhpZ2ggbW9sZWN1bGFyLXdlaWdodCBwbGFudCBETkEsOyBjaXRhdGlvbl9hdXRob3I9TXVycmF5ICBNLkcuOyBjaXRhdGlvbl9hdXRob3I9VGhvbXBzb24gIFcuRi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTgwOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9pc3N1ZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPTQzMjEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCBhbmQgYWNjdXJhdGUgc2hvcnQgcmVhZCBhbGlnbm1lbnQgd2l0aCBCdXJyb3dzLVdoZWVsZXIgdHJhbnNmb3JtLDsgY2l0YXRpb25fYXV0aG9yPUxpICBILjsgY2l0YXRpb25fYXV0aG9yPUR1cmJpbiAgUi47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25faXNzdWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xNzU0LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgc2VxdWVuY2UgYWxpZ25tZW50L21hcCBmb3JtYXQgYW5kIFNBTXRvb2xzLDsgY2l0YXRpb25fYXV0aG9yPUxpICBILjsgY2l0YXRpb25fYXV0aG9yPUhhbmRzYWtlciAgQi47IGNpdGF0aW9uX2F1dGhvcj1XeXNva2UgIEEuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX2lzc3VlPTE2OyAgY2l0YXRpb25fcGFnZXM9MjA3OC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SL3F0bDogUVRMIG1hcHBpbmcgaW4gZXhwZXJpbWVudGFsIGNyb3NzZXMsOyBjaXRhdGlvbl9hdXRob3I9QnJvbWFuICBLLlcuOyBjaXRhdGlvbl9hdXRob3I9V3UgIEguOyBjaXRhdGlvbl9hdXRob3I9U2VuICBTLjsgY2l0YXRpb25fYXV0aG9yPUNodXJjaGlsbCAgRy5BLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9pc3N1ZT03OyAgY2l0YXRpb25fcGFnZXM9ODg5LTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaHJvbW9zb21hbCBtYXAgb2YgdGhlIG1vZGVsIGxlZ3VtZSBMb3R1cyBqYXBvbmljdXM7IGNpdGF0aW9uX2F1dGhvcj1QZWRyb3NhICBBLjsgY2l0YXRpb25fYXV0aG9yPVNhbmRhbCAgTi47IGNpdGF0aW9uX2F1dGhvcj1TdG91Z2FhcmQgIEouOyBjaXRhdGlvbl9hdXRob3I9U2Nod2VpemVyICBELjsgY2l0YXRpb25fYXV0aG9yPUJhY2htYWlyICBBLjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTYxOyAgY2l0YXRpb25faXNzdWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTE2NjEtNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFJlY29tYmluYW50IGluYnJlZCBsaW5lcyAoUklMcykgZGVyaXZlZCBmcm9tIGJpLXBhcmVudGFsIHBvcHVsYXRpb25zIGFyZSBzdGFibGUgZ2VuZXRpYyByZXNvdXJjZXMsIHdoaWNoIGFyZSB3aWRlbHkgdXNlZCBmb3IgY29uc3RydWN0aW5nIGdlbmV0aWMgbGluIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0yMjM2MTg1JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZI8bDDFdB5Yx0gPIqr6PM4UQum54" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f5%2f487%2f2236185" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f5%2f487%2f2236185" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=79980" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/5/0/m_cover.gif?Expires=2147483647&amp;Signature=rsT2hJfrKXp2JlrHjh2bKIVwIZog14lNPGdzKP8yhtKMx~Yuyq3H3JzLDrlX5KEjnZvseVd2bAxtAi~Y9hNZXxJ6WDv2KUi8zxPom2lHNNImaLKpGt835Y71zYWnxYx55P8jzDtg5CRs2Bz1K1RPpU1Oz9FnrSPv6bmhcU6uyQ~xYqmBlGBw4TXr9jDHuwu~pd1eot3j91hB3lzFOWU6HfT9gLTdOIn00hhFqmJajweAIiXeiPNSrA~WoW-T9Q1u91QADYwK007qBz-ilhO5nfEW4RyE76MIZCWUuY7cDK1U0tgU8uh~HoZ~1YWG3BLI7M3P2TkqOmJNELz~~k5Mow__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/5">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87201538">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87201538">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87201541">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87201541">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87201548">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87201548">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87201563">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87201563">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87201594">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87201594">4. Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87201596">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87201596">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87201604">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87201604">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/5/477/2236181">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/5/495/2236213">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">High-resolution genetic maps of <em>Lotus japonicus</em> and <em>L. burttii</em> based on re-sequencing of recombinant inbred lines
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Niraj Shah</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div>Center for Carbohydrate Recognition and Signalling, Department of Molecular Biology and Genetics, Aarhus University, DK-8000 Aarhus, Denmark</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Niraj+Shah">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shah N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shah N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Niraj Shah</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Hirakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>2</sup></div>Kazusa DNA Research Institute, Chiba 292-0818, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Hirakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirakawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirakawa H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hideki Hirakawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shohei Kusakabe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>3</sup></div>Graduate School of Life Sciences, Tohoku University, Aoba-ku, Sendai 980-8577, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shohei+Kusakabe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kusakabe S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kusakabe S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shohei Kusakabe</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Niels Sandal</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div>Center for Carbohydrate Recognition and Signalling, Department of Molecular Biology and Genetics, Aarhus University, DK-8000 Aarhus, Denmark</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Niels+Sandal">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sandal N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sandal N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Niels Sandal</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jens Stougaard</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div>Center for Carbohydrate Recognition and Signalling, Department of Molecular Biology and Genetics, Aarhus University, DK-8000 Aarhus, Denmark</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jens+Stougaard">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Stougaard J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Stougaard J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jens Stougaard</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mikkel Heide Schierup</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>4</sup></div>Bioinformatics Research Centre, Aarhus University, 8000 Aarhus C, Denmark</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mikkel+Heide+Schierup">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Schierup M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Schierup M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mikkel Heide Schierup</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shusei Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>2</sup></div>Kazusa DNA Research Institute, Chiba 292-0818, Japan</div><div class="aff"><div class="label -label"><sup>3</sup></div>Graduate School of Life Sciences, Tohoku University, Aoba-ku, Sendai 980-8577, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw033-cor1" data-open="dsw033-cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            *To whom correspondence should be addressed. Email: <a href="mailto:sua@mbg.au.dk" target="_blank">sua@mbg.au.dk</a>. Phone: +45 87154937 (S.U.A.); <a href="mailto:shuseis@m.tohoku.ac.jp" target="_blank">shuseis@m.tohoku.ac.jp</a>. Phone: +81 22-217-5688 (S.S.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shusei+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shusei Sato</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Stig Uggerh&#248;j Andersen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label"><sup>1</sup></div>Center for Carbohydrate Recognition and Signalling, Department of Molecular Biology and Genetics, Aarhus University, DK-8000 Aarhus, Denmark</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw033-cor1" data-open="dsw033-cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            *To whom correspondence should be addressed. Email: <a href="mailto:sua@mbg.au.dk" target="_blank">sua@mbg.au.dk</a>. Phone: +45 87154937 (S.U.A.); <a href="mailto:shuseis@m.tohoku.ac.jp" target="_blank">shuseis@m.tohoku.ac.jp</a>. Phone: +81 22-217-5688 (S.S.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Stig+Uggerh%c3%b8j+Andersen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Andersen S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Andersen S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Stig Uggerhøj Andersen</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw033-FM1"><div class="fn"><div class="footnote-content">Edited by Prof. Kazuhiro Sato</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 5, 1 October 2016, Pages 487–494, <a href='https://doi.org/10.1093/dnares/dsw033'>https://doi.org/10.1093/dnares/dsw033</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">03 July 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">05 April 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">01 June 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                            
                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Niraj Shah, Hideki Hirakawa, Shohei Kusakabe, Niels Sandal, Jens Stougaard, Mikkel Heide Schierup, Shusei Sato, Stig Uggerhøj Andersen;  High-resolution genetic maps of <em>Lotus japonicus</em> and <em>L. burttii</em> based on re-sequencing of recombinant inbred lines, <em>DNA Research</em>, Volume 23, Issue 5, 1 October 2016, Pages 487–494, <a href='https://doi.org/10.1093/dnares/dsw033'>https://doi.org/10.1093/dnares/dsw033</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2236185?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236185?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236185?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236185?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236185?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236185?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2236185?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=High-resolution%20genetic%20maps%20of%20%26lt%3Bem%26gt%3BLotus%20japonicus%26lt%3B%2Fem%26gt%3B%20and%20%26lt%3Bem%26gt%3BL.%20burttii%26lt%3B%2Fem%26gt%3B%20based%20on%20re-sequencing%20of%20recombinant%20inbred%20lines&amp;publicationDate=2016-07-03&amp;volumeNum=23&amp;issueNum=5&amp;author=Shah%2C%20Niraj%3B%20Hirakawa%2C%20Hideki&amp;startPage=487&amp;endPage=494&amp;contentId=10.1093%2Fdnares%2Fdsw033&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f5%2f487%2f2236185" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87201538 id="87201538" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Recombinant inbred lines (RILs) derived from bi-parental populations are stable genetic resources, which are widely used for constructing genetic linkage maps. These genetic maps are essential for QTL mapping and can aid contig and scaffold anchoring in the final stages of genome assembly. In this study, two <em>Lotus</em> sp. RIL populations, <em>Lotus japonicus</em> MG-20 × Gifu and Gifu × <em>L. burttii</em>, were characterized by Illumina re-sequencing. Genotyping of 187 MG-20 × Gifu RILs at 87,140 marker positions and 96 Gifu × <em>L. burttii</em> RILs at 357,973 marker positions allowed us to accurately identify 1,929 recombination breakpoints in the MG-20 × Gifu RILs and 1,044 breakpoints in the Gifu × <em>L. burttii</em> population. The resulting high-density genetic maps now facilitate high-accuracy QTL mapping, identification of reference genome mis-assemblies, and characterization of structural variants.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genetic map&quot;">genetic map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;recombinant inbred lines&quot;">recombinant inbred lines</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;QTL mapping&quot;">QTL mapping</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;chromosomal translocations&quot;">chromosomal translocations</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;assembly errors&quot;">assembly errors</a></div>                            <h2 scrollto-destination=87201541 id="87201541" class="section-title" data-legacy-id=SEC1>1. Introduction</h2>
<p>Legumes are important in both agricultural and natural ecosystems because of their capacity for symbiotic nitrogen fixation, and many are rich sources of protein.<a reveal-id="dsw033-B1" data-open="dsw033-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsw033-B2" data-open="dsw033-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> The legume <em>Lotus japonicus</em> has been extensively used as a model for deciphering the molecular genetics governing the symbiotic interaction with rhizobia.<a reveal-id="dsw033-B3" data-open="dsw033-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsw033-B4" data-open="dsw033-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Comprehensive genetic and genomic resources have been developed in <em>L. japonicus</em>, including bacterial artificial chromosome libraries,<a reveal-id="dsw033-B5" data-open="dsw033-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsw033-B6" data-open="dsw033-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> a <em>LORE1</em> mutant population,<a reveal-id="dsw033-B7" data-open="dsw033-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsw033-B8" data-open="dsw033-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> a TILLING population,<a reveal-id="dsw033-B9" data-open="dsw033-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> ESTs,<a reveal-id="dsw033-B6" data-open="dsw033-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsw033-B10" data-open="dsw033-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> genetic and physical maps,<a reveal-id="dsw033-B11 dsw033-B12 dsw033-B13 dsw033-B14" data-open="dsw033-B11 dsw033-B12 dsw033-B13 dsw033-B14" class="link link-ref link-reveal xref-bibr"><sup>11–14</sup></a> and recombinant inbred line (RIL) populations.<a reveal-id="dsw033-B14 dsw033-B15 dsw033-B16 dsw033-B17" data-open="dsw033-B14 dsw033-B15 dsw033-B16 dsw033-B17" class="link link-ref link-reveal xref-bibr"><sup>14–17</sup></a></p><p>So far, <em>L. japonicus</em> genetic maps have been based on low-resolution simple sequence repeat (SSR) genotyping of F2 (∼900 markers) and RIL populations (∼96 markers).<a reveal-id="dsw033-B14" data-open="dsw033-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsw033-B17" data-open="dsw033-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> These maps have proved useful for anchoring large contigs to pseudomolecules<a reveal-id="dsw033-B13" data-open="dsw033-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> and for QTL mapping,<a reveal-id="dsw033-B14" data-open="dsw033-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsw033-B17" data-open="dsw033-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> but both approaches have been limited by the low marker densities of the SSR-based maps. The resolution of a genetic map depends on the number of recombination events in the mapping population and on the number of markers used to genotype each individual. The recombination breakpoint positions in the current <em>L. japonicus</em> maps are determined with limited accuracy due to the low marker density. With advances in sequencing technology, it has now become possible to genotype all single nucleotide polymorphism (SNP) positions in RIL populations using whole-genome re-sequencing. SNP genotyping using various methods has been successfully used in other plant species. In soybean and <em>Medicago sativa</em>, genetic maps were generated by genotyping ∼3,000 SNPs,<a reveal-id="dsw033-B18" data-open="dsw033-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsw033-B19" data-open="dsw033-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> in chickpea ∼2–4,000 SNPs were used,<a reveal-id="dsw033-B20" data-open="dsw033-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> whereas 16,000 single feature polymorphisms were used to generate a genetic map with 815 recombination breakpoints in Arabidopsis.<a reveal-id="dsw033-B21" data-open="dsw033-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> In <em>Brassica rapa</em> more than 1 million SNPs were discovered in 150 RILs using whole-genome re-sequencing, and a genetic map with 2,305 recombination events was constructed.<a reveal-id="dsw033-B22" data-open="dsw033-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> Similarly, a high-density genetic map with 5,074 breakpoints was generated in rice using low-coverage re-sequencing of 150 RILs.<a reveal-id="dsw033-B23" data-open="dsw033-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a></p><p>The choice of crossing partners and the population size is important for generation of high-quality genetic maps. The F2 populations developed earlier using Gifu as the common parent in crosses with Funakura or <em>L. filicaulis</em><a reveal-id="dsw033-B12" data-open="dsw033-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsw033-B15" data-open="dsw033-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> suffered drawbacks including low levels of polymorphism, low viability, severe segregation distortion and recombination suppression.<a reveal-id="dsw033-B12" data-open="dsw033-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsw033-B16" data-open="dsw033-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Although the MG-20 × Gifu cross has advantages in terms of good viability with little segregation distortion and a reasonable marker density,<a reveal-id="dsw033-B11" data-open="dsw033-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsw033-B13" data-open="dsw033-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsw033-B24" data-open="dsw033-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> it has the disadvantage that a large reciprocal translocation between the top of Gifu chromosome 1 and the bottom of MG-20 chromosome 2 causes severe suppression of recombination in this genomic region.<a reveal-id="dsw033-B11" data-open="dsw033-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a></p><p>In order to overcome the problems with suppression of recombination in the MG-20 × Gifu cross and to increase genetic variation and marker density, a more divergent accession originating from West Pakistan, <em>Lotus burttii</em>,<a reveal-id="dsw033-B25" data-open="dsw033-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a><sup>,</sup><a reveal-id="dsw033-B26" data-open="dsw033-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> was used for bi-parental crossing with Gifu. F2 populations derived from this cross showed good viability and did not suffer from suppression of recombination at the top of chromosome 1,<a reveal-id="dsw033-B24" data-open="dsw033-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> and a genetic map based on genotyping 97 SSR markers in 146 RILs has been constructed.<a reveal-id="dsw033-B14" data-open="dsw033-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a></p><p>Here, we have genotyped 187 MG-20 × Gifu and 96 Gifu × <em>L. burttii</em> RILs using whole-genome re-sequencing to accurately identify recombination breakpoints and construct high-resolution genetic maps for both crosses. These maps now facilitate identification of reference genome mis-assemblies, assignment of unanchored contigs onto pseudomolecules, and mapping of genomic translocations with high accuracy.</p>                            <h2 scrollto-destination=87201548 id="87201548" class="section-title" data-legacy-id=SEC2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87201550 id="87201550" class="section-title" data-legacy-id=SEC2.1>2.1. Sequencing and read alignment</h3>
<p>DNA from 187 MG-20 × Gifu (F9) and 96 RILs of Gifu × <em>L. burttii</em> (F8) RILs developed from F2 populations by single seed descent<a reveal-id="dsw033-B11" data-open="dsw033-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsw033-B14" data-open="dsw033-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> was extracted using the CTAB protocol.<a reveal-id="dsw033-B27" data-open="dsw033-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> Sequencing libraries were then constructed using the Illumina Nextera DNA library preparation kit (FC-121-1031) with a dual-index adapter system according to the manufacturer’s instructions. The libraries were sequenced on an Illumina HiSeq 2000 sequencer to generate 2×93 bp paired-end reads from ∼500 bp insert libraries. The reads were mapped to the MG-20 <em>L. japonicus</em> genome v.3.0 (<a class="link link-uri" href="http://www.kazusa.or.jp/lotus/" target="_blank">http://www.kazusa.or.jp/lotus/</a>, 17 June 2016, date last accessed) using Burrows–Wheeler Aligner <em>mem</em> v. 0.7 with default parameters .5a.<a reveal-id="dsw033-B28" data-open="dsw033-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a></p>                            <h3 scrollto-destination=87201552 id="87201552" class="section-title" data-legacy-id=SEC2.2>2.2. Genetic map construction</h3>
<p>Candidate polymorphic positions were available from genome sequences, and these were validated by verifying that the positions were polymorphic according to the RIL genotyping results. The candidate SNP markers were genotyped using the <em>mpileup</em> function of SAMtools software v0.1.19.<a reveal-id="dsw033-B29" data-open="dsw033-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> The command used to determine genotypes was <em>samtools mpileup –uD –b &lt;List of BAM files.txt&gt; –f &lt;reference_genome.fasta&gt; –I &lt;Known Polymorphic positions.txt&gt; | samtools bcftools view –cg - &gt; result.vcf’</em>, where <em>–u</em> is to generate uncompressed binary variant call format (BCF) output, <em>-D</em> is to output per sample depth. Because the reads were mapped to an MG-20 reference sequence, the Gifu × <em>L. burttii</em> RIL genotype calls were re-oriented with reference to Gifu, such that the Gifu, rather than the MG-20, genotype was considered the reference allele.</p><p>To generate genetic maps, we used a modified version of a previously described approach.<a reveal-id="dsw033-B23" data-open="dsw033-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> Because of the higher sequencing depth used in our study, we simplified the sliding window approach for calling genotype blocks, using overlapping windows with a single base offset spanning 20 markers upstream and downstream of each marker position. Genotype counts were calculated for each window, and the consensus genotype was called as the genotype with the highest abundance. Recombination breakpoints were then called at positions where the consensus genotype switched (<a reveal-id="dsw033-F2" data-open="dsw033-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>B). To ensure better accuracy, the genotype block calls were compared with the original genotyping data at each marker position for each RIL and a score was calculated. The score was +1 if the genotype block call and marker genotypes matched, 0 if there was a mismatch, and +0.5 if one of the genotypes was heterozygous. This marker score was used to identify inaccurate breakpoint positions, where the breakpoint had been miscalled by a few marker positions relative to its true position. Inaccurate breakpoint positions were then shifted to optimize the marker score. For the MG-20 × Gifu data, the marker score was additionally used for filtering out markers that showed poor concordance with the genotype block calls. Markers were removed if the cumulative score at each marker position was &lt;150. Finally, breakpoints that occurred at the same position in the majority of the RILs were removed manually because these very likely represented artifacts. In order to construct a genetic segment map for each of the RIL populations, the region between any two consecutive breakpoints in all RILs for each population was defined as one segment.<a reveal-id="dsw033-B23" data-open="dsw033-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a></p>                            <h3 scrollto-destination=87201557 id="87201557" class="section-title" data-legacy-id=SEC2.3>2.3. Identification of translocations and assembly errors based on the Gifu × <em>L. burttii</em> genetic map</h3>
<p>For the MG-20 × Gifu RILs, the segment map did not require any filtering, but for Gifu × <em>L. burttii</em>, a large number of aberrant segments were apparent (<a reveal-id="dsw033-F3" data-open="dsw033-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>B). Although the complex segment pattern presented was difficult to resolve accurately using automated approaches, we identified aberrant segments with high confidence by manual curation of the segment map, where the dominant genotype pattern for each genomic region was readily apparent and the aberrant segments stood out because of large deviations from the consensus genotype pattern. Next, the aberrant segments that were &lt;10 kb were discarded in order to focus on the largest and most well-supported segments. Each of the remaining aberrant segments was then matched with all other segments using a custom perl script to determine the genetic linkage. If ≥90 out of the 96 RIL genotypes were identical between two segments, then the aberrant segment was placed next to the matching segment, identifying it is misplaced in the original map. Unplaced aberrant segments were not included in the genetic map.</p><p>The displaced segments were then categorized as translocations or assembly errors. The RIL genotypes of the displaced segment in the Gifu × <em>L. burttii</em> map were compared with the MG-20 × Gifu RIL genotypes in the same genomic region by manual inspection. If the region in MG-20 × Gifu showed markers with discordant genotypes compared with neighboring regions, then the segment in Gifu × <em>L. burttii</em> was classified as an assembly error. In contrast, if the genotypes of the markers in the region matched the flanking markers, the segment was categorized as a translocation. The genetic maps were visualized by calculating pairwise logarithm of the odds (LOD) scores and recombination fractions using the plot.rf function of the R<em>/</em>qtl package.<a reveal-id="dsw033-B30" data-open="dsw033-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a></p>                            <h2 scrollto-destination=87201563 id="87201563" class="section-title" data-legacy-id=SEC3>3. Results and discussion</h2>
                            <h3 scrollto-destination=87201564 id="87201564" class="section-title" data-legacy-id=SEC3.1>3.1. RIL re-sequencing and genotyping</h3>
<p>Paired-end sequencing reads from 187 MG-20 × Gifu and 96 Gifu × <em>L. burttii</em> RILs were aligned to the <em>L. japonicus</em> MG-20 reference genome v. 3.0 using <em>bwa</em>. The resulting average coverage was 6.53 for the MG20 × Gifu RILs and 8.03 for the Gifu × <em>L. burttii</em> RILs (<a reveal-id="dsw033-F1" data-open="dsw033-F1" class="link link-reveal link-table xref-fig">Fig. 1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033_Supplementary_Data.zip?Expires=2147483647&Signature=Gr4d2if-3VqP-bvLTKyu-Y0FcYRshQGdUtKuZ0O~vOtGB6VneOJF21eKbrAvnB5OnzzacBR2lBUdIeR3y~Zp0xcOBQnybd78Qq-hPxC46PDxxVYGXnBqIDiai8bXQPVdS2UR-wqWEMmzJv5UTuthaLj3gA-wmpJhLU7TZdc0qESDGhflzS1LcE7CBnMcYW4fAzgxxAcb2115nW2l1-iBHXbEvONyuc7mTf~ElQMHuY9L1xfcjhU2Ct28TcmRkFHls8U-ZVhgqRskQDP63-mvOzzaz8QirEyZAjFhgsZDR45iFzTRGIddTzN70kcyuomKKXZuTlcZ0BlBr6LCAAP6bw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). Next, 87,140 positions polymorphic between Gifu and MG-20, and 357,973 positions polymorphic between Gifu and <em>L. burttii</em> were genotyped in the two sets of RILs on chromosomes 1–6 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033_Supplementary_Data.zip?Expires=2147483647&Signature=Gr4d2if-3VqP-bvLTKyu-Y0FcYRshQGdUtKuZ0O~vOtGB6VneOJF21eKbrAvnB5OnzzacBR2lBUdIeR3y~Zp0xcOBQnybd78Qq-hPxC46PDxxVYGXnBqIDiai8bXQPVdS2UR-wqWEMmzJv5UTuthaLj3gA-wmpJhLU7TZdc0qESDGhflzS1LcE7CBnMcYW4fAzgxxAcb2115nW2l1-iBHXbEvONyuc7mTf~ElQMHuY9L1xfcjhU2Ct28TcmRkFHls8U-ZVhgqRskQDP63-mvOzzaz8QirEyZAjFhgsZDR45iFzTRGIddTzN70kcyuomKKXZuTlcZ0BlBr6LCAAP6bw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Files S1–S2</a></span>). With ∼230 Mbp sequence anchored on these pseudomolecules, this yields average marker distances of 2.6 and 0.64 kbp in the two maps, allowing high-resolution mapping of recombination breakpoints. In addition, 64,011 positions polymorphic between MG-20 and Gifu and 203,540 positions polymorphic between Gifu and <em>L. burttii</em> were genotyped on 162 Mbp of unanchored contigs (chr0) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033_Supplementary_Data.zip?Expires=2147483647&Signature=Gr4d2if-3VqP-bvLTKyu-Y0FcYRshQGdUtKuZ0O~vOtGB6VneOJF21eKbrAvnB5OnzzacBR2lBUdIeR3y~Zp0xcOBQnybd78Qq-hPxC46PDxxVYGXnBqIDiai8bXQPVdS2UR-wqWEMmzJv5UTuthaLj3gA-wmpJhLU7TZdc0qESDGhflzS1LcE7CBnMcYW4fAzgxxAcb2115nW2l1-iBHXbEvONyuc7mTf~ElQMHuY9L1xfcjhU2Ct28TcmRkFHls8U-ZVhgqRskQDP63-mvOzzaz8QirEyZAjFhgsZDR45iFzTRGIddTzN70kcyuomKKXZuTlcZ0BlBr6LCAAP6bw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Files S3–S4</a></span>) in <em>L. japonicus</em> v 3.0. This genotype information can now be used to position chr0 contigs onto the <em>L. japonicus</em> v. 3.0 pseudomolecules. </p>                        <a id="87201567" scrollto-destination="87201567"></a>
<div id="dsw033-F1" data-id="dsw033-F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/m_dsw033f1p.png?Expires=2147483647&Signature=ELCUbybbAMzMXujItQ~fR6o77TpCjgYBb2LMvJzGqfZgo51n0J4ect6gKUFPCvOw8n--ZjwYLUKTRNgyyU9lrpaRsSxqm0kwyN11WPAChn7QpJ3FIj0xfaXTHyN56cY3sOCI~Dx8vXZxhoRweVAYyFTCVDtaGoTv5MQY-1xJjYlgL33uG176TIaW~7HHdrTXDmTfYQEciPDb6ion7GBAatCuzOPtNLa2eEmbZwQ-jYC8CvEba3viQHtXV7McAWhwUmnnUWsnggEtj9uQhBXuo2cVM2hItDTd3ThUCIAPflp8Z4TV0-njmSwc~bputYdG65-nXRiW7-DdpZL2vqbKBQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Average RIL read coverage for (A) MG-20 × Gifu and (B) Gifu × Lotus burttii." path-from-xml="dsw033f1p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201567/dsw033f1p.tiff" path-from-xml="dsw033f1p.tiff" target="_blank">View large</a><a section="87201567" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033f1p.png?Expires=2147483647&Signature=3WrT5do98tFpK5xdk1GoFzbGfd7Jo03w5i3eq~C1BCpy66vJyuq2itM5vOhF8CUSdU3Tz~8rqIQU5rOZdaOCy-rVjcR3ZIydWXjlfrdSbpqjgqBF74SlJqG8l6GtFDJ8qbfc0EXMdlq5aHLkK9neV45NunZYkWgqZepNHo1GaXeDBtJq2RDnJYG~h4-N~ZFVWFo~j7g8NNXiJyIoyq61vJmh7efoe11U-dDwaq1lMdlFowDjiz-voYiV4A9iTVGB9OjZu3CP6o7RCD8i5V3gkVJQ7tW~D35TLw8tNxZgjp2mdXdyW9Hydz8BeRF-bGZE0Vid-TsxXwjV28roo4ZZnQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201567&ar=2236185&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw033f1p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Average RIL read coverage for (<strong>A</strong>) MG-20 × Gifu and (<strong>B</strong>) Gifu × <em>Lotus burttii.</em></p></div></div><div content-id="dsw033-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/m_dsw033f1p.png?Expires=2147483647&Signature=ELCUbybbAMzMXujItQ~fR6o77TpCjgYBb2LMvJzGqfZgo51n0J4ect6gKUFPCvOw8n--ZjwYLUKTRNgyyU9lrpaRsSxqm0kwyN11WPAChn7QpJ3FIj0xfaXTHyN56cY3sOCI~Dx8vXZxhoRweVAYyFTCVDtaGoTv5MQY-1xJjYlgL33uG176TIaW~7HHdrTXDmTfYQEciPDb6ion7GBAatCuzOPtNLa2eEmbZwQ-jYC8CvEba3viQHtXV7McAWhwUmnnUWsnggEtj9uQhBXuo2cVM2hItDTd3ThUCIAPflp8Z4TV0-njmSwc~bputYdG65-nXRiW7-DdpZL2vqbKBQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Average RIL read coverage for (A) MG-20 × Gifu and (B) Gifu × Lotus burttii." path-from-xml="dsw033f1p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201567/dsw033f1p.tiff" path-from-xml="dsw033f1p.tiff" target="_blank">View large</a><a section="87201567" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033f1p.png?Expires=2147483647&Signature=3WrT5do98tFpK5xdk1GoFzbGfd7Jo03w5i3eq~C1BCpy66vJyuq2itM5vOhF8CUSdU3Tz~8rqIQU5rOZdaOCy-rVjcR3ZIydWXjlfrdSbpqjgqBF74SlJqG8l6GtFDJ8qbfc0EXMdlq5aHLkK9neV45NunZYkWgqZepNHo1GaXeDBtJq2RDnJYG~h4-N~ZFVWFo~j7g8NNXiJyIoyq61vJmh7efoe11U-dDwaq1lMdlFowDjiz-voYiV4A9iTVGB9OjZu3CP6o7RCD8i5V3gkVJQ7tW~D35TLw8tNxZgjp2mdXdyW9Hydz8BeRF-bGZE0Vid-TsxXwjV28roo4ZZnQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201567&ar=2236185&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw033f1p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Average RIL read coverage for (<strong>A</strong>) MG-20 × Gifu and (<strong>B</strong>) Gifu × <em>Lotus burttii.</em></p></div></div>                            <h3 scrollto-destination=87201569 id="87201569" class="section-title" data-legacy-id=SEC3.2>3.2. Identification of RIL genotype blocks and construction of genetic maps</h3>
<p>Because recombination events are relatively infrequent,<a reveal-id="dsw033-B23" data-open="dsw033-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> we expected to identify large genotype blocks based on the genotyping data. These blocks were apparent, but they were frequently interrupted by aberrant genotype calls (<a reveal-id="dsw033-F2" data-open="dsw033-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>A). This noise could be due to genotyping errors or to misplaced markers, and we used a sliding window approach to reduce the noise by calling consensus genotypes within 41-marker sliding windows (<a reveal-id="dsw033-F2" data-open="dsw033-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>B). In addition, we eliminated small blocks of markers, which showed very poor consistency in genotype calls with those of their neighbors (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033_Supplementary_Data.zip?Expires=2147483647&Signature=Gr4d2if-3VqP-bvLTKyu-Y0FcYRshQGdUtKuZ0O~vOtGB6VneOJF21eKbrAvnB5OnzzacBR2lBUdIeR3y~Zp0xcOBQnybd78Qq-hPxC46PDxxVYGXnBqIDiai8bXQPVdS2UR-wqWEMmzJv5UTuthaLj3gA-wmpJhLU7TZdc0qESDGhflzS1LcE7CBnMcYW4fAzgxxAcb2115nW2l1-iBHXbEvONyuc7mTf~ElQMHuY9L1xfcjhU2Ct28TcmRkFHls8U-ZVhgqRskQDP63-mvOzzaz8QirEyZAjFhgsZDR45iFzTRGIddTzN70kcyuomKKXZuTlcZ0BlBr6LCAAP6bw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Files S5–S6</a></span>). </p>                        <a id="87201572" scrollto-destination="87201572"></a>
<div id="dsw033-F2" data-id="dsw033-F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/m_dsw033f2p.png?Expires=2147483647&Signature=42DYVvk4M8Wa37T5lmvEs4nVSmY-kdwyCx6wTCTQKEvzDlO-E8vnGwWerICsIqBZT14A2on~Wz9LXpYYSX7mHR1oVCHZkTu2305OKXVPDC~vuiwbeo2uqIDBK9qzEjKhaBPBwtcdJ52CAGFGRmo9swWWvSlsgK7vHgS7CiDuGy2loZrZM1sqadQ1P1aBYtdPLj7ve2-adTfj1EmguOQhlzAS0nB1il7OtVWiytbtRNHcu8hXGRiPb8MsjF23UwufEpyFdEKjEAAuvLEfwdX9nJY9INvMTYOAjbcDB9BhUjdnXS3zvQX-Lc7qnvFCP0hTr0luluZlylxrR6Ai5bXI3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genotype block calling. Chromosome 1 is shown for the MG-20 × Gifu cross. Genotype counts in 41-marker windows is shown. (A) Unfiltered genotype cells and (B) Final genotype blocks." path-from-xml="dsw033f2p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201572/dsw033f2p.tiff" path-from-xml="dsw033f2p.tiff" target="_blank">View large</a><a section="87201572" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033f2p.png?Expires=2147483647&Signature=JIXkU~aXdNTpD5KHe1r-TcRDFKUGzgkIe721Ekp7fmn3G5qXMVozgGZgvp1L1Us8SHuNOrVfQkZJAOo9xGgP~m49yCcSpTXO-nVue3i80ZCprR2DlNf9BCwYuW4hy0DAl7eK0iTgI6HDMBluJC~g9HTjgtF31vKpNFNWqVpwoUQJ8W8pD4L~j-kZBC62eRkKUK8S~IADJogMgePd7shjLrVf7xc9jN9O64nI4Gj1RepAgZE1UbfGS9IIkkTcgckpxYlNpYyZhdHklqKvdgUeEkIJXgw1lH86nUVlCoKeqYwnI~5NvLZ4UCqCQWmbRgRoEnj0u~jaATfF4rpGmBTcBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201572&ar=2236185&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw033f2p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genotype block calling. Chromosome 1 is shown for the MG-20 × Gifu cross. Genotype counts in 41-marker windows is shown. (A) Unfiltered genotype cells and (B) Final genotype blocks.</p></div></div><div content-id="dsw033-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/m_dsw033f2p.png?Expires=2147483647&Signature=42DYVvk4M8Wa37T5lmvEs4nVSmY-kdwyCx6wTCTQKEvzDlO-E8vnGwWerICsIqBZT14A2on~Wz9LXpYYSX7mHR1oVCHZkTu2305OKXVPDC~vuiwbeo2uqIDBK9qzEjKhaBPBwtcdJ52CAGFGRmo9swWWvSlsgK7vHgS7CiDuGy2loZrZM1sqadQ1P1aBYtdPLj7ve2-adTfj1EmguOQhlzAS0nB1il7OtVWiytbtRNHcu8hXGRiPb8MsjF23UwufEpyFdEKjEAAuvLEfwdX9nJY9INvMTYOAjbcDB9BhUjdnXS3zvQX-Lc7qnvFCP0hTr0luluZlylxrR6Ai5bXI3w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genotype block calling. Chromosome 1 is shown for the MG-20 × Gifu cross. Genotype counts in 41-marker windows is shown. (A) Unfiltered genotype cells and (B) Final genotype blocks." path-from-xml="dsw033f2p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201572/dsw033f2p.tiff" path-from-xml="dsw033f2p.tiff" target="_blank">View large</a><a section="87201572" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033f2p.png?Expires=2147483647&Signature=JIXkU~aXdNTpD5KHe1r-TcRDFKUGzgkIe721Ekp7fmn3G5qXMVozgGZgvp1L1Us8SHuNOrVfQkZJAOo9xGgP~m49yCcSpTXO-nVue3i80ZCprR2DlNf9BCwYuW4hy0DAl7eK0iTgI6HDMBluJC~g9HTjgtF31vKpNFNWqVpwoUQJ8W8pD4L~j-kZBC62eRkKUK8S~IADJogMgePd7shjLrVf7xc9jN9O64nI4Gj1RepAgZE1UbfGS9IIkkTcgckpxYlNpYyZhdHklqKvdgUeEkIJXgw1lH86nUVlCoKeqYwnI~5NvLZ4UCqCQWmbRgRoEnj0u~jaATfF4rpGmBTcBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201572&ar=2236185&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw033f2p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genotype block calling. Chromosome 1 is shown for the MG-20 × Gifu cross. Genotype counts in 41-marker windows is shown. (A) Unfiltered genotype cells and (B) Final genotype blocks.</p></div></div><p>Following identification of genotype blocks for all RILs, we combined the results into genetic maps, where each segment constituted the region between any two consecutive breakpoints in the RIL population.<a reveal-id="dsw033-B23" data-open="dsw033-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> To visualize the resulting maps, we then plotted the pairwise recombination fractions and LOD scores for all segments (<a reveal-id="dsw033-F3" data-open="dsw033-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). The MG-20 × Gifu map contained 1,744 segments and appeared very consistent with previous results,<a reveal-id="dsw033-B11" data-open="dsw033-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsw033-B12" data-open="dsw033-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> with the strongest linkage along the diagonal and at the site of the Gifu – MG-20 chromosome 1–2 translocation (<a reveal-id="dsw033-F3" data-open="dsw033-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>A, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033_Supplementary_Data.zip?Expires=2147483647&Signature=Gr4d2if-3VqP-bvLTKyu-Y0FcYRshQGdUtKuZ0O~vOtGB6VneOJF21eKbrAvnB5OnzzacBR2lBUdIeR3y~Zp0xcOBQnybd78Qq-hPxC46PDxxVYGXnBqIDiai8bXQPVdS2UR-wqWEMmzJv5UTuthaLj3gA-wmpJhLU7TZdc0qESDGhflzS1LcE7CBnMcYW4fAzgxxAcb2115nW2l1-iBHXbEvONyuc7mTf~ElQMHuY9L1xfcjhU2Ct28TcmRkFHls8U-ZVhgqRskQDP63-mvOzzaz8QirEyZAjFhgsZDR45iFzTRGIddTzN70kcyuomKKXZuTlcZ0BlBr6LCAAP6bw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). In contrast, with 1,837 detected breakpoints, the Gifu × <em>L. burttii</em> map contained many more segments than expected for 96 RILs, and large numbers of aberrant linkage signals were apparent as numerous inconsistent horizontal and vertical lines in the LOD score and recombination fraction plot (<a reveal-id="dsw033-F3" data-open="dsw033-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>B). </p>                        <a id="87201576" scrollto-destination="87201576"></a>
<div id="dsw033-F3" data-id="dsw033-F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/m_dsw033f3p.png?Expires=2147483647&Signature=yv2Zg9eqsDAbR7cYQ7N34QekrDq0NJ1QED9EaCP9zXg7bY0EblDgSmjoCeOXJVZnjn0WJsSEZbJ63qKxyUTyHm4pMTzlE8DSc2-eu4cEmo~hoP7F9is14ve4o10auKEeBE3G0elaoTz0BHtkRWLUDORSQTqfk3nDofD3RRsAR5641OwPKJ8xcP1WNw7lW1xb5htzqbCqRzo7dNxcMoLkC3izWWUyGlQdZAw7s3g8dJPKv06hFRJC9d5Cv-MW-0v5W9T9KXj-SFjMc-xA2BtHwjiA820r8Ugmy8TeeNdgyYTudj1JQe6V3gdlZI3v7v2s2Pzm9UoiiuQFtGm4FSvhtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Recombination fraction and LOD scores for all segment pairs. Recombination fractions (rf) are shown above the diagonal, and the LOD scores below. Low recombination fractions and high LOD scores indicate strong segment linkage. (A) MG-20 × Gifu RILs, (B–D) Gifu × L. burttii RILs. (B) Including aberrant segments. (C) After replacing aberrant segments. (D) After replacing aberrant segments and the chromosome 1–2 translocation." path-from-xml="dsw033f3p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201576/dsw033f3p.tiff" path-from-xml="dsw033f3p.tiff" target="_blank">View large</a><a section="87201576" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033f3p.png?Expires=2147483647&Signature=y5Foo4CxbDXGYwvx8uyqEjLf1hCm2wt18I9DxLxQ9xEPJmroS6UtI9yUDcxjV4SCNKk69EZjGW~JBbjJKb6LCvC-h5wYjD3wBbvxC1DeMZdMcYnFhbXrYYjm-0CmqExPGldF6k~wAdSl0UrAYG71wQin0sLrNVDayxt2FpB0s-o-1c~efFQJH3KO3-7Qla6nLEfTDxNCk5RZPxZDi7z5tvQTmxltlYSo2-kiCMquW8ck-6-FbSe1kKixk3wI7YUpVI-nXHgSySBEBrWsb89JcccNCvThl1kho-M6bPboAKc67rtccg3Va1vRqzRJlPe6IcrGfLUy-IwrJrI2AbpYqg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201576&ar=2236185&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw033f3p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Recombination fraction and LOD scores for all segment pairs. Recombination fractions (rf) are shown above the diagonal, and the LOD scores below. Low recombination fractions and high LOD scores indicate strong segment linkage. (A) MG-20 × Gifu RILs, (B–D) Gifu × <em>L. burttii</em> RILs. (B) Including aberrant segments. (C) After replacing aberrant segments. (D) After replacing aberrant segments and the chromosome 1–2 translocation.</p></div></div><div content-id="dsw033-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/m_dsw033f3p.png?Expires=2147483647&Signature=yv2Zg9eqsDAbR7cYQ7N34QekrDq0NJ1QED9EaCP9zXg7bY0EblDgSmjoCeOXJVZnjn0WJsSEZbJ63qKxyUTyHm4pMTzlE8DSc2-eu4cEmo~hoP7F9is14ve4o10auKEeBE3G0elaoTz0BHtkRWLUDORSQTqfk3nDofD3RRsAR5641OwPKJ8xcP1WNw7lW1xb5htzqbCqRzo7dNxcMoLkC3izWWUyGlQdZAw7s3g8dJPKv06hFRJC9d5Cv-MW-0v5W9T9KXj-SFjMc-xA2BtHwjiA820r8Ugmy8TeeNdgyYTudj1JQe6V3gdlZI3v7v2s2Pzm9UoiiuQFtGm4FSvhtw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Recombination fraction and LOD scores for all segment pairs. Recombination fractions (rf) are shown above the diagonal, and the LOD scores below. Low recombination fractions and high LOD scores indicate strong segment linkage. (A) MG-20 × Gifu RILs, (B–D) Gifu × L. burttii RILs. (B) Including aberrant segments. (C) After replacing aberrant segments. (D) After replacing aberrant segments and the chromosome 1–2 translocation." path-from-xml="dsw033f3p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201576/dsw033f3p.tiff" path-from-xml="dsw033f3p.tiff" target="_blank">View large</a><a section="87201576" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033f3p.png?Expires=2147483647&Signature=y5Foo4CxbDXGYwvx8uyqEjLf1hCm2wt18I9DxLxQ9xEPJmroS6UtI9yUDcxjV4SCNKk69EZjGW~JBbjJKb6LCvC-h5wYjD3wBbvxC1DeMZdMcYnFhbXrYYjm-0CmqExPGldF6k~wAdSl0UrAYG71wQin0sLrNVDayxt2FpB0s-o-1c~efFQJH3KO3-7Qla6nLEfTDxNCk5RZPxZDi7z5tvQTmxltlYSo2-kiCMquW8ck-6-FbSe1kKixk3wI7YUpVI-nXHgSySBEBrWsb89JcccNCvThl1kho-M6bPboAKc67rtccg3Va1vRqzRJlPe6IcrGfLUy-IwrJrI2AbpYqg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201576&ar=2236185&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw033f3p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Recombination fraction and LOD scores for all segment pairs. Recombination fractions (rf) are shown above the diagonal, and the LOD scores below. Low recombination fractions and high LOD scores indicate strong segment linkage. (A) MG-20 × Gifu RILs, (B–D) Gifu × <em>L. burttii</em> RILs. (B) Including aberrant segments. (C) After replacing aberrant segments. (D) After replacing aberrant segments and the chromosome 1–2 translocation.</p></div></div>                            <h3 scrollto-destination=87201577 id="87201577" class="section-title" data-legacy-id=SEC3.3>3.3. Identification of assembly errors and correction of the Gifu × <em>L. burttii</em> map</h3>
<p>Because all reads were aligned to the MG-20 reference genome, the inconsistencies in the Gifu × <em>L. burttii</em> map could be caused either by assembly errors in the MG-20 reference sequence being detected because of the fourfold larger marker density in the Gifu × <em>L. burttii</em> cross or by the occurrence of translocations specific to MG-20. To distinguish between these possibilities, we returned to the unfiltered MG-20 × Gifu genotyping results (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033_Supplementary_Data.zip?Expires=2147483647&Signature=Gr4d2if-3VqP-bvLTKyu-Y0FcYRshQGdUtKuZ0O~vOtGB6VneOJF21eKbrAvnB5OnzzacBR2lBUdIeR3y~Zp0xcOBQnybd78Qq-hPxC46PDxxVYGXnBqIDiai8bXQPVdS2UR-wqWEMmzJv5UTuthaLj3gA-wmpJhLU7TZdc0qESDGhflzS1LcE7CBnMcYW4fAzgxxAcb2115nW2l1-iBHXbEvONyuc7mTf~ElQMHuY9L1xfcjhU2Ct28TcmRkFHls8U-ZVhgqRskQDP63-mvOzzaz8QirEyZAjFhgsZDR45iFzTRGIddTzN70kcyuomKKXZuTlcZ0BlBr6LCAAP6bw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S1</a></span>) and investigated the genotype consistencies in the aberrant regions identified in the Gifu × <em>L. burttii</em> map. In 179 out of 347 cases, we found markers within the same intervals in the MG-20 × Gifu map, which had been filtered away because of genotype inconsistency with neighboring markers, identifying these intervals as misplaced in the MG-20 reference assembly. This genetic information has potential for improving the quality of the reference genome assembly because an average marker distance of only 0.6 kbp in the Gifu × <em>L. burttii</em> map would reliably detect misplaced contigs with sizes down to a few kilobasepair.</p><p>We then proceeded to replace the aberrant segments in the Gifu × <em>L. burttii</em> map based on their genotype patterns, and we were able to unambiguously assign 201 out of 347 regions, resulting in a consistent map comprising 1,044 segments (<a reveal-id="dsw033-F3" data-open="dsw033-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>C, <a reveal-id="dsw033-T1" data-open="dsw033-T1" class="link link-reveal link-table xref-fig">Table 1</a>) Finally, the reciprocally translocated regions at the top of chromosome 1 and bottom of chromosome 2 were replaced to align the segment map with the genetic data (<a reveal-id="dsw033-F3" data-open="dsw033-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>D, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033_Supplementary_Data.zip?Expires=2147483647&Signature=Gr4d2if-3VqP-bvLTKyu-Y0FcYRshQGdUtKuZ0O~vOtGB6VneOJF21eKbrAvnB5OnzzacBR2lBUdIeR3y~Zp0xcOBQnybd78Qq-hPxC46PDxxVYGXnBqIDiai8bXQPVdS2UR-wqWEMmzJv5UTuthaLj3gA-wmpJhLU7TZdc0qESDGhflzS1LcE7CBnMcYW4fAzgxxAcb2115nW2l1-iBHXbEvONyuc7mTf~ElQMHuY9L1xfcjhU2Ct28TcmRkFHls8U-ZVhgqRskQDP63-mvOzzaz8QirEyZAjFhgsZDR45iFzTRGIddTzN70kcyuomKKXZuTlcZ0BlBr6LCAAP6bw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033_Supplementary_Data.zip?Expires=2147483647&Signature=Gr4d2if-3VqP-bvLTKyu-Y0FcYRshQGdUtKuZ0O~vOtGB6VneOJF21eKbrAvnB5OnzzacBR2lBUdIeR3y~Zp0xcOBQnybd78Qq-hPxC46PDxxVYGXnBqIDiai8bXQPVdS2UR-wqWEMmzJv5UTuthaLj3gA-wmpJhLU7TZdc0qESDGhflzS1LcE7CBnMcYW4fAzgxxAcb2115nW2l1-iBHXbEvONyuc7mTf~ElQMHuY9L1xfcjhU2Ct28TcmRkFHls8U-ZVhgqRskQDP63-mvOzzaz8QirEyZAjFhgsZDR45iFzTRGIddTzN70kcyuomKKXZuTlcZ0BlBr6LCAAP6bw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File S4</a></span>). </p>                        <a id="87201580" scrollto-destination="87201580"></a>
<div content-id="dsw033-T1" class="table-modal"><div class="table-wrap"><div id="dsw033-T1" class="table-wrap-title" data-id="dsw033-T1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of replaced segments in the Gifu × <em>L. burttii</em> segment map</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>Chr </th><th>chr1 </th><th>chr2 </th><th>chr3 </th><th>chr4 </th><th>chr5 </th><th>chr6 </th><th>Total </th></tr></thead><tbody><tr><td>chr1 </td><td>24 </td><td>9 </td><td>10 </td><td>8 </td><td>6 </td><td>1 </td><td>58 </td></tr><tr><td>chr2 </td><td>12 </td><td>8 </td><td>6 </td><td>6 </td><td>2 </td><td>2 </td><td>36 </td></tr><tr><td>chr3 </td><td>12 </td><td>6 </td><td>12 </td><td>3 </td><td>3 </td><td>2 </td><td>38 </td></tr><tr><td>chr4 </td><td>6 </td><td>9 </td><td>1 </td><td>8 </td><td>3 </td><td>2 </td><td>29 </td></tr><tr><td>chr5 </td><td>8 </td><td>1 </td><td>2 </td><td>1 </td><td>2 </td><td>0 </td><td>14 </td></tr><tr><td>chr6 </td><td>2 </td><td>13 </td><td>3 </td><td>2 </td><td>3 </td><td>3 </td><td>26 </td></tr><tr><td>Total </td><td>64 </td><td>46 </td><td>34 </td><td>28 </td><td>19 </td><td>10 </td><td>201 </td></tr></tbody></table></div><div content-id="dsw033-T1" class="table-modal"><table><thead name="thead" align="left"><tr><th>Chr </th><th>chr1 </th><th>chr2 </th><th>chr3 </th><th>chr4 </th><th>chr5 </th><th>chr6 </th><th>Total </th></tr></thead><tbody><tr><td>chr1 </td><td>24 </td><td>9 </td><td>10 </td><td>8 </td><td>6 </td><td>1 </td><td>58 </td></tr><tr><td>chr2 </td><td>12 </td><td>8 </td><td>6 </td><td>6 </td><td>2 </td><td>2 </td><td>36 </td></tr><tr><td>chr3 </td><td>12 </td><td>6 </td><td>12 </td><td>3 </td><td>3 </td><td>2 </td><td>38 </td></tr><tr><td>chr4 </td><td>6 </td><td>9 </td><td>1 </td><td>8 </td><td>3 </td><td>2 </td><td>29 </td></tr><tr><td>chr5 </td><td>8 </td><td>1 </td><td>2 </td><td>1 </td><td>2 </td><td>0 </td><td>14 </td></tr><tr><td>chr6 </td><td>2 </td><td>13 </td><td>3 </td><td>2 </td><td>3 </td><td>3 </td><td>26 </td></tr><tr><td>Total </td><td>64 </td><td>46 </td><td>34 </td><td>28 </td><td>19 </td><td>10 </td><td>201 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw033-TF1"><div class="fn"><p>The segments were moved from the chromosome indicated in the header row to the chromosome indicated in the leftmost column.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87201580" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="dsw033-T1" class="table-wrap-title" data-id="dsw033-T1"><div class="label -label">Table 1.</div><div class="caption"><p>Summary of replaced segments in the Gifu × <em>L. burttii</em> segment map</p></div> </div><div class="table-overflow"><table><thead name="thead" align="left"><tr><th>Chr </th><th>chr1 </th><th>chr2 </th><th>chr3 </th><th>chr4 </th><th>chr5 </th><th>chr6 </th><th>Total </th></tr></thead><tbody><tr><td>chr1 </td><td>24 </td><td>9 </td><td>10 </td><td>8 </td><td>6 </td><td>1 </td><td>58 </td></tr><tr><td>chr2 </td><td>12 </td><td>8 </td><td>6 </td><td>6 </td><td>2 </td><td>2 </td><td>36 </td></tr><tr><td>chr3 </td><td>12 </td><td>6 </td><td>12 </td><td>3 </td><td>3 </td><td>2 </td><td>38 </td></tr><tr><td>chr4 </td><td>6 </td><td>9 </td><td>1 </td><td>8 </td><td>3 </td><td>2 </td><td>29 </td></tr><tr><td>chr5 </td><td>8 </td><td>1 </td><td>2 </td><td>1 </td><td>2 </td><td>0 </td><td>14 </td></tr><tr><td>chr6 </td><td>2 </td><td>13 </td><td>3 </td><td>2 </td><td>3 </td><td>3 </td><td>26 </td></tr><tr><td>Total </td><td>64 </td><td>46 </td><td>34 </td><td>28 </td><td>19 </td><td>10 </td><td>201 </td></tr></tbody></table></div><div content-id="dsw033-T1" class="table-modal"><table><thead name="thead" align="left"><tr><th>Chr </th><th>chr1 </th><th>chr2 </th><th>chr3 </th><th>chr4 </th><th>chr5 </th><th>chr6 </th><th>Total </th></tr></thead><tbody><tr><td>chr1 </td><td>24 </td><td>9 </td><td>10 </td><td>8 </td><td>6 </td><td>1 </td><td>58 </td></tr><tr><td>chr2 </td><td>12 </td><td>8 </td><td>6 </td><td>6 </td><td>2 </td><td>2 </td><td>36 </td></tr><tr><td>chr3 </td><td>12 </td><td>6 </td><td>12 </td><td>3 </td><td>3 </td><td>2 </td><td>38 </td></tr><tr><td>chr4 </td><td>6 </td><td>9 </td><td>1 </td><td>8 </td><td>3 </td><td>2 </td><td>29 </td></tr><tr><td>chr5 </td><td>8 </td><td>1 </td><td>2 </td><td>1 </td><td>2 </td><td>0 </td><td>14 </td></tr><tr><td>chr6 </td><td>2 </td><td>13 </td><td>3 </td><td>2 </td><td>3 </td><td>3 </td><td>26 </td></tr><tr><td>Total </td><td>64 </td><td>46 </td><td>34 </td><td>28 </td><td>19 </td><td>10 </td><td>201 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="dsw033-TF1"><div class="fn"><p>The segments were moved from the chromosome indicated in the header row to the chromosome indicated in the leftmost column.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87201580" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87201581 id="87201581" class="section-title" data-legacy-id=SEC3.4>3.4. Segregation distortion and breakpoint distribution</h3>
<p>After establishing the genetic maps for both crosses, we characterized segregation distortion patterns. Although the MG-20 × Gifu population was well balanced, we observed more severe biases in the Gifu × <em>L. burttii</em> RILs. The most striking signal was found on chromosome 2 with a maxium at ∼25 Mbp. Here, the <em>L. burttii</em> allele frequency reached 96% (<a reveal-id="dsw033-F4" data-open="dsw033-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>), indicating very strong selection against Gifu alleles resulting in elimination of 50% of the possible progeny due to this incompatibility alone. </p>                        <a id="87201583" scrollto-destination="87201583"></a>
<div id="dsw033-F4" data-id="dsw033-F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/m_dsw033f4p.png?Expires=2147483647&Signature=cgApi7Pxp7oVJkIVTuY6ip56taHI96t3fSazOmp3NmpYbOzAdGfJp08BL6tDi2X4be6C0R6-nplrqwNu~eJw1Z0fhsOdIa4ggGBwjaBhJAQjLYIXQaCIHM7oXmaThDYnfdQxF~yfHLSHI6PI9DR2szmjuX1ikGIONfVb5ED2tTGrzuGt9nGdtLszDnwfASKQmQZZAAivNJh-0ZCXLYtp0uCx1-gobgnKpACB0WsaIlNWooUaRh4Y~V14YSRHW6k0I71VTV-9ufspKnv7r4MdkLvCBMXm3V5ra87iL8DwWtG13ybjgUZu-NVOIlVVRnQth~rUr2OWkabhLJNSQnWUqg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Segregation distortion. The Gifu genotype fractions for MG-20 × Gifu and Gifu × Lotus burttii are plotted for each segment in the genetic maps. The positions are relative to the MG-20 v. 3.0 reference sequence." path-from-xml="dsw033f4p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201583/dsw033f4p.tiff" path-from-xml="dsw033f4p.tiff" target="_blank">View large</a><a section="87201583" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033f4p.png?Expires=2147483647&Signature=Pjnmr4DGOWzQnyHbFy1b09I2NVO1PL2c0hdjmwoACJi3MQ2zUxpbqAeaBgPzihbw3ZJ2A39L4iZq1dqAF0pBnXulFPz1Laar95IBgl-IOo2roqL6rmSEUgIXv7KcudMBNbVHpBfoC1AWYZP8OgOEgzkOJiuQp6IO-wAaWzQbntZExt17aDVqJeF70zBE94HIplo12lgcvPs3NUy3ORGRYDPddQkRcRay7G0mF~8hkzeM56nvSMH1t-kHtsxjTzehVIPhHFR6E~gdnIh-m3Ork5qllSdnI3iYQUFQqXeLRlrgJnimR0iwk9s89K9buaU9D56wMllJhAwfzw-g2yERFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201583&ar=2236185&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw033f4p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Segregation distortion. The Gifu genotype fractions for MG-20 × Gifu and Gifu × <em>Lotus burttii</em> are plotted for each segment in the genetic maps. The positions are relative to the MG-20 v. 3.0 reference sequence.</p></div></div><div content-id="dsw033-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/m_dsw033f4p.png?Expires=2147483647&Signature=cgApi7Pxp7oVJkIVTuY6ip56taHI96t3fSazOmp3NmpYbOzAdGfJp08BL6tDi2X4be6C0R6-nplrqwNu~eJw1Z0fhsOdIa4ggGBwjaBhJAQjLYIXQaCIHM7oXmaThDYnfdQxF~yfHLSHI6PI9DR2szmjuX1ikGIONfVb5ED2tTGrzuGt9nGdtLszDnwfASKQmQZZAAivNJh-0ZCXLYtp0uCx1-gobgnKpACB0WsaIlNWooUaRh4Y~V14YSRHW6k0I71VTV-9ufspKnv7r4MdkLvCBMXm3V5ra87iL8DwWtG13ybjgUZu-NVOIlVVRnQth~rUr2OWkabhLJNSQnWUqg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Segregation distortion. The Gifu genotype fractions for MG-20 × Gifu and Gifu × Lotus burttii are plotted for each segment in the genetic maps. The positions are relative to the MG-20 v. 3.0 reference sequence." path-from-xml="dsw033f4p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201583/dsw033f4p.tiff" path-from-xml="dsw033f4p.tiff" target="_blank">View large</a><a section="87201583" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033f4p.png?Expires=2147483647&Signature=Pjnmr4DGOWzQnyHbFy1b09I2NVO1PL2c0hdjmwoACJi3MQ2zUxpbqAeaBgPzihbw3ZJ2A39L4iZq1dqAF0pBnXulFPz1Laar95IBgl-IOo2roqL6rmSEUgIXv7KcudMBNbVHpBfoC1AWYZP8OgOEgzkOJiuQp6IO-wAaWzQbntZExt17aDVqJeF70zBE94HIplo12lgcvPs3NUy3ORGRYDPddQkRcRay7G0mF~8hkzeM56nvSMH1t-kHtsxjTzehVIPhHFR6E~gdnIh-m3Ork5qllSdnI3iYQUFQqXeLRlrgJnimR0iwk9s89K9buaU9D56wMllJhAwfzw-g2yERFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201583&ar=2236185&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw033f4p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Segregation distortion. The Gifu genotype fractions for MG-20 × Gifu and Gifu × <em>Lotus burttii</em> are plotted for each segment in the genetic maps. The positions are relative to the MG-20 v. 3.0 reference sequence.</p></div></div><p>Both crosses showed very similar averages of 10.3 breakpoints per chromosome (<a reveal-id="dsw033-F5" data-open="dsw033-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033_Supplementary_Data.zip?Expires=2147483647&Signature=Gr4d2if-3VqP-bvLTKyu-Y0FcYRshQGdUtKuZ0O~vOtGB6VneOJF21eKbrAvnB5OnzzacBR2lBUdIeR3y~Zp0xcOBQnybd78Qq-hPxC46PDxxVYGXnBqIDiai8bXQPVdS2UR-wqWEMmzJv5UTuthaLj3gA-wmpJhLU7TZdc0qESDGhflzS1LcE7CBnMcYW4fAzgxxAcb2115nW2l1-iBHXbEvONyuc7mTf~ElQMHuY9L1xfcjhU2Ct28TcmRkFHls8U-ZVhgqRskQDP63-mvOzzaz8QirEyZAjFhgsZDR45iFzTRGIddTzN70kcyuomKKXZuTlcZ0BlBr6LCAAP6bw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>), and the genetic length of each chromosome corresponded to the previous genetic linkage map<a reveal-id="dsw033-B11" data-open="dsw033-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033_Supplementary_Data.zip?Expires=2147483647&Signature=Gr4d2if-3VqP-bvLTKyu-Y0FcYRshQGdUtKuZ0O~vOtGB6VneOJF21eKbrAvnB5OnzzacBR2lBUdIeR3y~Zp0xcOBQnybd78Qq-hPxC46PDxxVYGXnBqIDiai8bXQPVdS2UR-wqWEMmzJv5UTuthaLj3gA-wmpJhLU7TZdc0qESDGhflzS1LcE7CBnMcYW4fAzgxxAcb2115nW2l1-iBHXbEvONyuc7mTf~ElQMHuY9L1xfcjhU2Ct28TcmRkFHls8U-ZVhgqRskQDP63-mvOzzaz8QirEyZAjFhgsZDR45iFzTRGIddTzN70kcyuomKKXZuTlcZ0BlBr6LCAAP6bw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). In addition, the breakpoints and overall genotype patterns detected in the MG-20 × Gifu RILs corresponded well with previous genotype data generated using SSR markers,<a reveal-id="dsw033-B17" data-open="dsw033-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> with the exception of seven lines, RI-011, RI-068, RI-080, RI-097, RI-149, RI-152 and RI-153, where the line identities may have been interchanged. </p>                        <a id="87201585" scrollto-destination="87201585"></a>
<div id="dsw033-F5" data-id="dsw033-F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/m_dsw033f5p.png?Expires=2147483647&Signature=4B0NQ-DNC6aZZbz9L~QmEWk50nXycNViZMJ-5515wLQkb6uM~LGQwXDfbmoh9kG0UiiYKM3jbqCTtETDyG2fgiNUx~zXubzftjYbpEERjg5mtROFyCLV1WSEFevBqrcsKDfI0z6nGt1xNgVHtmcWG0tselCPN3yrMHC9GBzDbkxFGu-bKcUqoGuN4xQFOfWpOA5qkJkxd4EllIE1HLEiO7Gnf~Z7UycaHGXJMW2ed~3ZcYYSTKZuclctjtUM5g-q-zy0Aq3vd9Tu9h~X1ndH5EjmT9Xd9zmWJgcUnA8jfUGieyQ7bcOCe8UBNETJooz2UIod7-FTjMy0WhPf5GO1mw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Recombination breakpoint counts. (A and B) Histograms of breakpoint counts by chromosome. (A) MG-20 × Gifu RILs. (B) Gifu × Lotus burttii RILs. (C and D) Histogram of breakpoint counts averaged across chromosomes 1–6. (C) MG-20 × Gifu RILs. (D) Gifu × L. burttii RILs. Vertical lines indicate the average number of breakpoints." path-from-xml="dsw033f5p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201585/dsw033f5p.tiff" path-from-xml="dsw033f5p.tiff" target="_blank">View large</a><a section="87201585" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033f5p.png?Expires=2147483647&Signature=OobElFaQIUqP9eudrrj42j0WnNpIirchM~xehMEt9dz14lvekCHrG6erYVZ~S48OJuBLUW9nILU343HV3D0WeMACDKq04Y5SLS~-Kjag7zQlY~5aU2isnMUhE-5C6kH6YugwOud9aDcPxToGwZr73lqhvY-DORe4OGdNHLs2Krs6-XX~Xj2MYY0~gVuhpWOa9dVBDMCJpa3adm2pzArUcJtzB2xpKmJmSQsGAK5uKojqWabE7BXDZWE5I50QCjGcuLgJvewvSKJU-gp1jkrJLbXWhasdXz3eKvfiJNoawFwbNe7FaF-sypS8EA~9dbv2GctbNeyzMeISEj3HOMaxfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201585&ar=2236185&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw033f5p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Recombination breakpoint counts. (A and B) Histograms of breakpoint counts by chromosome. (A) MG-20 × Gifu RILs. (B) Gifu × <em>Lotus burttii</em> RILs. (C and D) Histogram of breakpoint counts averaged across chromosomes 1–6. (C) MG-20 × Gifu RILs. (D) Gifu × <em>L. burttii</em> RILs. Vertical lines indicate the average number of breakpoints.</p></div></div><div content-id="dsw033-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/m_dsw033f5p.png?Expires=2147483647&Signature=4B0NQ-DNC6aZZbz9L~QmEWk50nXycNViZMJ-5515wLQkb6uM~LGQwXDfbmoh9kG0UiiYKM3jbqCTtETDyG2fgiNUx~zXubzftjYbpEERjg5mtROFyCLV1WSEFevBqrcsKDfI0z6nGt1xNgVHtmcWG0tselCPN3yrMHC9GBzDbkxFGu-bKcUqoGuN4xQFOfWpOA5qkJkxd4EllIE1HLEiO7Gnf~Z7UycaHGXJMW2ed~3ZcYYSTKZuclctjtUM5g-q-zy0Aq3vd9Tu9h~X1ndH5EjmT9Xd9zmWJgcUnA8jfUGieyQ7bcOCe8UBNETJooz2UIod7-FTjMy0WhPf5GO1mw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Recombination breakpoint counts. (A and B) Histograms of breakpoint counts by chromosome. (A) MG-20 × Gifu RILs. (B) Gifu × Lotus burttii RILs. (C and D) Histogram of breakpoint counts averaged across chromosomes 1–6. (C) MG-20 × Gifu RILs. (D) Gifu × L. burttii RILs. Vertical lines indicate the average number of breakpoints." path-from-xml="dsw033f5p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201585/dsw033f5p.tiff" path-from-xml="dsw033f5p.tiff" target="_blank">View large</a><a section="87201585" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033f5p.png?Expires=2147483647&Signature=OobElFaQIUqP9eudrrj42j0WnNpIirchM~xehMEt9dz14lvekCHrG6erYVZ~S48OJuBLUW9nILU343HV3D0WeMACDKq04Y5SLS~-Kjag7zQlY~5aU2isnMUhE-5C6kH6YugwOud9aDcPxToGwZr73lqhvY-DORe4OGdNHLs2Krs6-XX~Xj2MYY0~gVuhpWOa9dVBDMCJpa3adm2pzArUcJtzB2xpKmJmSQsGAK5uKojqWabE7BXDZWE5I50QCjGcuLgJvewvSKJU-gp1jkrJLbXWhasdXz3eKvfiJNoawFwbNe7FaF-sypS8EA~9dbv2GctbNeyzMeISEj3HOMaxfA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201585&ar=2236185&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw033f5p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Recombination breakpoint counts. (A and B) Histograms of breakpoint counts by chromosome. (A) MG-20 × Gifu RILs. (B) Gifu × <em>Lotus burttii</em> RILs. (C and D) Histogram of breakpoint counts averaged across chromosomes 1–6. (C) MG-20 × Gifu RILs. (D) Gifu × <em>L. burttii</em> RILs. Vertical lines indicate the average number of breakpoints.</p></div></div><p>In the MG-20 × Gifu cross, extensive regions displaying suppression of recombination were located on chromosomes 1 and 6, whereas the Gifu × <em>L. burttii</em> cross showed suppression of recombination on chromosomes 5 and 6 (<a reveal-id="dsw033-F6" data-open="dsw033-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). Both crosses showed a reduced recombination frequency at the top of chromosome 2, which corresponds to a region rich in ribosomal RNA genes,<a reveal-id="dsw033-B31" data-open="dsw033-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> and near the centromere on chromosome 4. </p>                        <a id="87201587" scrollto-destination="87201587"></a>
<div id="dsw033-F6" data-id="dsw033-F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/m_dsw033f6p.png?Expires=2147483647&Signature=mGWfOKqWL10X0IewDoYl2a6KiT3pwneiRzePBJC4NhVjsyMBOI1rfp04BSyVWs9v2hR5MqkfFSCLAUfF-zF3GXdfAtZDQbeGlxFG6IYP7kt07Rgi4b~1ipFqA~yyFlLzILdXEKxh2cV8k3Mfznk1ZiXUdpC5cqeN5A4G1xmRPwsBufQJJ29W1s4dophzkCmzkKbHgPLLW0iiDBp-sC4FoQCAWWVkT~5mfjSFgAHgv5XGdiXA3YGGiQuSn00NdsYiz82VNbFkD1p3e2HAjeUXTWOlflrCLzDJP91vW8Kc3Hxu8nOCO-bzqvzEZgcAzGmKrDJbKhLITc2UW~0mKIQ3hw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Recombination breakpoint distribution. Vertical lines represent the positions of recombination breakpoints. The black dot on the grey bar indicates the centromere position for each chromosome. The horizontal bars indicate regions of recombination suppression. All breakpoint positions are shown relative to the MG-20 v.3.0 reference sequence." path-from-xml="dsw033f6p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201587/dsw033f6p.tiff" path-from-xml="dsw033f6p.tiff" target="_blank">View large</a><a section="87201587" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033f6p.png?Expires=2147483647&Signature=CuhgAqSi65WNodJO1XCq626vXCPBGfOVIRFTulJwEV2-cz~aOtDRdJNz0jh4A~SqAwAzKubTY8T4MHJqiogj46adQ8PKkxnWoFxk9fUMM2FaBnE9E9OnWDiaPB2HTEKnmXiOWpGNVG1KTtAhOSeGPl6P7O31pFwpJSDs22clacd4bFg3M-C3g-Es9ASGzwZudZS40WWr-~OA9rF-xQSYETJueyHYPEMx1FmiKDs3vtzuBELNZ1IG1XeGFcXmQfyBDndV5OJN7wzmZwbaw4Afb0F-O6cP2azPVVdpL0zXavb8OqA-EfV19AibgA29oKvDtyiYhTZz10a~AJAPZb8YWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201587&ar=2236185&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw033f6p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Recombination breakpoint distribution. Vertical lines represent the positions of recombination breakpoints. The black dot on the grey bar indicates the centromere position for each chromosome. The horizontal bars indicate regions of recombination suppression. All breakpoint positions are shown relative to the MG-20 v.3.0 reference sequence.</p></div></div><div content-id="dsw033-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/m_dsw033f6p.png?Expires=2147483647&Signature=mGWfOKqWL10X0IewDoYl2a6KiT3pwneiRzePBJC4NhVjsyMBOI1rfp04BSyVWs9v2hR5MqkfFSCLAUfF-zF3GXdfAtZDQbeGlxFG6IYP7kt07Rgi4b~1ipFqA~yyFlLzILdXEKxh2cV8k3Mfznk1ZiXUdpC5cqeN5A4G1xmRPwsBufQJJ29W1s4dophzkCmzkKbHgPLLW0iiDBp-sC4FoQCAWWVkT~5mfjSFgAHgv5XGdiXA3YGGiQuSn00NdsYiz82VNbFkD1p3e2HAjeUXTWOlflrCLzDJP91vW8Kc3Hxu8nOCO-bzqvzEZgcAzGmKrDJbKhLITc2UW~0mKIQ3hw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Recombination breakpoint distribution. Vertical lines represent the positions of recombination breakpoints. The black dot on the grey bar indicates the centromere position for each chromosome. The horizontal bars indicate regions of recombination suppression. All breakpoint positions are shown relative to the MG-20 v.3.0 reference sequence." path-from-xml="dsw033f6p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201587/dsw033f6p.tiff" path-from-xml="dsw033f6p.tiff" target="_blank">View large</a><a section="87201587" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033f6p.png?Expires=2147483647&Signature=CuhgAqSi65WNodJO1XCq626vXCPBGfOVIRFTulJwEV2-cz~aOtDRdJNz0jh4A~SqAwAzKubTY8T4MHJqiogj46adQ8PKkxnWoFxk9fUMM2FaBnE9E9OnWDiaPB2HTEKnmXiOWpGNVG1KTtAhOSeGPl6P7O31pFwpJSDs22clacd4bFg3M-C3g-Es9ASGzwZudZS40WWr-~OA9rF-xQSYETJueyHYPEMx1FmiKDs3vtzuBELNZ1IG1XeGFcXmQfyBDndV5OJN7wzmZwbaw4Afb0F-O6cP2azPVVdpL0zXavb8OqA-EfV19AibgA29oKvDtyiYhTZz10a~AJAPZb8YWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201587&ar=2236185&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw033f6p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Recombination breakpoint distribution. Vertical lines represent the positions of recombination breakpoints. The black dot on the grey bar indicates the centromere position for each chromosome. The horizontal bars indicate regions of recombination suppression. All breakpoint positions are shown relative to the MG-20 v.3.0 reference sequence.</p></div></div>                            <h3 scrollto-destination=87201588 id="87201588" class="section-title" data-legacy-id=SEC3.5>3.5. Mapping of MG-20-specific structural variants</h3>
<p>Because suppression of recombination can be a consequence of structural rearrangements and because a major reciprocal chromosomal translocation has occurred between chromosomes 1 and 2 in MG-20 with respect to Gifu and <em>L. burttii</em>,<a reveal-id="dsw033-B11" data-open="dsw033-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsw033-B26" data-open="dsw033-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> we returned to the aberrant segments in the Gifu × <em>L. burttii</em> map in order to precisely characterize the structural variation between MG-20 and Gifu/<em>L. burttii</em>. Only two aberrant segments were found in regions, which contained consistent marker genotypes in the MG-20 × Gifu population (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033_Supplementary_Data.zip?Expires=2147483647&Signature=Gr4d2if-3VqP-bvLTKyu-Y0FcYRshQGdUtKuZ0O~vOtGB6VneOJF21eKbrAvnB5OnzzacBR2lBUdIeR3y~Zp0xcOBQnybd78Qq-hPxC46PDxxVYGXnBqIDiai8bXQPVdS2UR-wqWEMmzJv5UTuthaLj3gA-wmpJhLU7TZdc0qESDGhflzS1LcE7CBnMcYW4fAzgxxAcb2115nW2l1-iBHXbEvONyuc7mTf~ElQMHuY9L1xfcjhU2Ct28TcmRkFHls8U-ZVhgqRskQDP63-mvOzzaz8QirEyZAjFhgsZDR45iFzTRGIddTzN70kcyuomKKXZuTlcZ0BlBr6LCAAP6bw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). These segments corresponded to the regions displaying suppressed recombination in the MG-20 × Gifu cross, the top of chromosome 1 and a region around 20 Mbp on chromosome 6, indicating that they likely represented structural variants (<a reveal-id="dsw033-F6" data-open="dsw033-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033_Supplementary_Data.zip?Expires=2147483647&Signature=Gr4d2if-3VqP-bvLTKyu-Y0FcYRshQGdUtKuZ0O~vOtGB6VneOJF21eKbrAvnB5OnzzacBR2lBUdIeR3y~Zp0xcOBQnybd78Qq-hPxC46PDxxVYGXnBqIDiai8bXQPVdS2UR-wqWEMmzJv5UTuthaLj3gA-wmpJhLU7TZdc0qESDGhflzS1LcE7CBnMcYW4fAzgxxAcb2115nW2l1-iBHXbEvONyuc7mTf~ElQMHuY9L1xfcjhU2Ct28TcmRkFHls8U-ZVhgqRskQDP63-mvOzzaz8QirEyZAjFhgsZDR45iFzTRGIddTzN70kcyuomKKXZuTlcZ0BlBr6LCAAP6bw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>).</p><p>Examining first the top of chromosome 1 and the chromosome 1–2 reciprocal translocation, we could accurately map the translocation breakpoints to between 1,730 and 1,806 kbp on chromosome 1 (<a reveal-id="dsw033-F7" data-open="dsw033-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>A) and between 32,017 kbp and 32,645 kbp on chromosome 2 (<a reveal-id="dsw033-F7" data-open="dsw033-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>C) using the Gifu × <em>L. burttii</em> map. A number of putative structural rearrangements were also detected within this translocated region based on the Gifu × <em>L. burttii</em> data, but these were all classified as assembly errors and were mostly caused by errors in scaffolding, as the aberrant segments corresponded to entire contigs created by short read assembly. </p>                        <a id="87201591" scrollto-destination="87201591"></a>
<div id="dsw033-F7" data-id="dsw033-F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/m_dsw033f7p.png?Expires=2147483647&Signature=0-Mm6qZM0D~9-iPJMS0lByvumEheGBD5X92KiDgskozwSsXg-SK-IN~3ySnwgdT3BZuIUpEa4ZdGg-C6Ta5TZZw02SfXC4CJbvCwwDxq1tXpKBv56kWvuxHRfcEqts4ynpUN7mY90YGp7nZ40~tHcspDq5FcIpkZeUpie-4F7L8vfzvUrQINj4XNC-hc-40CQd5a0IihK2CaBlGq8h6lTEpbhh7mAXjQQ43bLa7g1ktwspnxvleIvipAJCCiwbFDl1Q5dQwMC0Z~VSvyZvufX8EvEYWJxhGtPYlqPaCrwpaNc9HCYWLPhenNr7unzWCCcuJv28DnV8NypEAibUUV9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Mapping structural variants. (A) Translocation breakpoint at the top of MG-20 chromosome 1. (B) Translocated segment from chromosome 1 placed at the correct location at the bottom of chromosome 2. (C) Translocation breakpoint at the bottom of MG-20 chromosome 2. (D) Translocated segment from chromosome 2 placed at the correct location at the top of chromosome 1. (E) Translocation breakpoints on chromosome 6. (F) The translocated segment placed at the correct position on Gifu chromosome 6. Number ranges indicate possible breakpoint intervals. Numbers above arrows indicate segment endpoints." path-from-xml="dsw033f7p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201591/dsw033f7p.tiff" path-from-xml="dsw033f7p.tiff" target="_blank">View large</a><a section="87201591" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033f7p.png?Expires=2147483647&Signature=189Qvk1JTYRIm7rDP1sG2AsQrWcpaUkVCDSHhUQalIaNy-2Ox-2ANsD9XWkOdhCaYJNxf1ttCPMB-8oF2Sljm6wODVf7rHL18rnzHd8QErmIe0sLwE7SkJSms7YIN8hPpH-giIi0z5MpPlh3N5OCUoK56sTEkkFcb1AnZn6T4xZ-ZNWtgTkp55rpaA-Pnxj6ZbIfiqX8G~SVyHl8f~17TbqpQUqoSZjECqZfyJsiOVOUdbWQDqPA9Kl3qky8drHx7Eab8gkUc45p01MLr8D~tSO8lCUhEu3D5R1jQ0sr282vsyJSdfYIB6-Z~RV~WQ8PzIVF8O8Ouoao4v8iAKe1dA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201591&ar=2236185&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw033f7p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Mapping structural variants. (A) Translocation breakpoint at the top of MG-20 chromosome 1. (B) Translocated segment from chromosome 1 placed at the correct location at the bottom of chromosome 2. (C) Translocation breakpoint at the bottom of MG-20 chromosome 2. (D) Translocated segment from chromosome 2 placed at the correct location at the top of chromosome 1. (E) Translocation breakpoints on chromosome 6. (F) The translocated segment placed at the correct position on Gifu chromosome 6. Number ranges indicate possible breakpoint intervals. Numbers above arrows indicate segment endpoints.</p></div></div><div content-id="dsw033-F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/m_dsw033f7p.png?Expires=2147483647&Signature=0-Mm6qZM0D~9-iPJMS0lByvumEheGBD5X92KiDgskozwSsXg-SK-IN~3ySnwgdT3BZuIUpEa4ZdGg-C6Ta5TZZw02SfXC4CJbvCwwDxq1tXpKBv56kWvuxHRfcEqts4ynpUN7mY90YGp7nZ40~tHcspDq5FcIpkZeUpie-4F7L8vfzvUrQINj4XNC-hc-40CQd5a0IihK2CaBlGq8h6lTEpbhh7mAXjQQ43bLa7g1ktwspnxvleIvipAJCCiwbFDl1Q5dQwMC0Z~VSvyZvufX8EvEYWJxhGtPYlqPaCrwpaNc9HCYWLPhenNr7unzWCCcuJv28DnV8NypEAibUUV9w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Mapping structural variants. (A) Translocation breakpoint at the top of MG-20 chromosome 1. (B) Translocated segment from chromosome 1 placed at the correct location at the bottom of chromosome 2. (C) Translocation breakpoint at the bottom of MG-20 chromosome 2. (D) Translocated segment from chromosome 2 placed at the correct location at the top of chromosome 1. (E) Translocation breakpoints on chromosome 6. (F) The translocated segment placed at the correct position on Gifu chromosome 6. Number ranges indicate possible breakpoint intervals. Numbers above arrows indicate segment endpoints." path-from-xml="dsw033f7p.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87201591/dsw033f7p.tiff" path-from-xml="dsw033f7p.tiff" target="_blank">View large</a><a section="87201591" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033f7p.png?Expires=2147483647&Signature=189Qvk1JTYRIm7rDP1sG2AsQrWcpaUkVCDSHhUQalIaNy-2Ox-2ANsD9XWkOdhCaYJNxf1ttCPMB-8oF2Sljm6wODVf7rHL18rnzHd8QErmIe0sLwE7SkJSms7YIN8hPpH-giIi0z5MpPlh3N5OCUoK56sTEkkFcb1AnZn6T4xZ-ZNWtgTkp55rpaA-Pnxj6ZbIfiqX8G~SVyHl8f~17TbqpQUqoSZjECqZfyJsiOVOUdbWQDqPA9Kl3qky8drHx7Eab8gkUc45p01MLr8D~tSO8lCUhEu3D5R1jQ0sr282vsyJSdfYIB6-Z~RV~WQ8PzIVF8O8Ouoao4v8iAKe1dA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87201591&ar=2236185&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw033f7p.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Mapping structural variants. (A) Translocation breakpoint at the top of MG-20 chromosome 1. (B) Translocated segment from chromosome 1 placed at the correct location at the bottom of chromosome 2. (C) Translocation breakpoint at the bottom of MG-20 chromosome 2. (D) Translocated segment from chromosome 2 placed at the correct location at the top of chromosome 1. (E) Translocation breakpoints on chromosome 6. (F) The translocated segment placed at the correct position on Gifu chromosome 6. Number ranges indicate possible breakpoint intervals. Numbers above arrows indicate segment endpoints.</p></div></div><p>For the region on chromosome 6, genetic data from the Gifu × <em>L. burttii</em> map indicated translocation of a 186 kb region from ∼21.4 Mbp in MG-20 to ∼17.8 Mbp in Gifu (<a reveal-id="dsw033-F7" data-open="dsw033-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>E and F). Although there was no indication of a larger inversion, it appeared that this smaller translocation could be causing the suppression of recombination in the region around 20 Mbp (<a reveal-id="dsw033-F6" data-open="dsw033-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>), which was also observed in the existing genetic map of an MG-20 × Gifu F2 population, where more than 10 markers mapped to 46.6 cM, corresponding to the same region.<a reveal-id="dsw033-B13" data-open="dsw033-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a></p><p>In conclusion, the two high-density genetic maps are complementary and together provide adequate coverage of the <em>L. japonicus</em> genome for QTL mapping and genome assembly, and the very high marker density generated by whole-genome re-sequencing of the Gifu × <em>L. burttii</em> RILs is especially useful for identifying and correcting even minor reference genome mis-assemblies.</p>                            <h2 scrollto-destination=87201594 id="87201594" class="section-title" data-legacy-id=SEC4>4. Availability</h2>
<p>The RILs are available from LegumeBase (<a class="link link-uri" href="http://www.legumebase.brc.miyazaki-u.ac.jp/" target="_blank">http://www.legumebase.brc.miyazaki-u.ac.jp/</a>). The genetic maps and all marker genotypes are provided as supplemental files and tables. The raw sequencing data are available from the DDBJ Sequence Read Archive with accession numbers DRA004729, DRA002730 and DRA004731.</p>                            <h2 scrollto-destination=87201596 id="87201596" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank A. Muraki, Y. Kishida, S. Nakayama, and A. Watanabe for excellent technical assistance. MG-20 × Gifu RILs and Gifu × <em>Lotus burttii</em> RILs were provided by the National BioResource Project ‘Lotus/Glycine’.</p>                            <h3 scrollto-destination=87201598 id="87201598" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=87201600 id="87201600" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033_Supplementary_Data.zip?Expires=2147483647&Signature=Gr4d2if-3VqP-bvLTKyu-Y0FcYRshQGdUtKuZ0O~vOtGB6VneOJF21eKbrAvnB5OnzzacBR2lBUdIeR3y~Zp0xcOBQnybd78Qq-hPxC46PDxxVYGXnBqIDiai8bXQPVdS2UR-wqWEMmzJv5UTuthaLj3gA-wmpJhLU7TZdc0qESDGhflzS1LcE7CBnMcYW4fAzgxxAcb2115nW2l1-iBHXbEvONyuc7mTf~ElQMHuY9L1xfcjhU2Ct28TcmRkFHls8U-ZVhgqRskQDP63-mvOzzaz8QirEyZAjFhgsZDR45iFzTRGIddTzN70kcyuomKKXZuTlcZ0BlBr6LCAAP6bw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org." target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h3 scrollto-destination=87201602 id="87201602" class="section-title" >Funding</h3>
<p>This work was supported by the Genome Information Upgrading Program of the National BioResource Project in 2014 (SS), the Danish National Research Foundation grant no. DNRF79 (JS), the ERC Advanced Grant 268523 (JS), grant no. 10-081677 from The Danish Council for Independent Research|Technology and Production Sciences (SUA).</p>                            <h2 scrollto-destination=87201604 id="87201604" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw033-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Smil</div>
               <div class="given-names">V.</div>
            </div>
          </div>
          <div class="year">1999</div>, 
<div class="article-title">Nitrogen in crop production: an account of global flows</div>, <div class="source">Global Biogeochem. Cycles</div> , <div class="volume">13</div>(<div class="issue">2</div>), <div class="fpage">647</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nitrogen%20in%20crop%20production%3A%20an%20account%20of%20global%20flows&amp;author=V.%20Smil&amp;publication_year=1999&amp;journal=Global%20Biogeochem.%20Cycles&amp;volume=13&amp;pages=647-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1029/1999GB900015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1029%2F1999GB900015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1029%2F1999GB900015"> </span></div> </div></div></div></div></div><div content-id="dsw033-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Graham</div>
               <div class="given-names">P.H.</div>
            </div>
             <div class="name">
              <div class="surname">Vance</div>
               <div class="given-names">C.P.</div>
            </div>
          </div>
          <div class="year">2003</div>, 
<div class="article-title">Legumes: importance and constraints to greater use,</div><div class="source">Plant Physiol.</div> <div class="volume">131</div>(<div class="issue">3</div>), <div class="fpage">872</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Legumes%3A%20importance%20and%20constraints%20to%20greater%20use%2C&amp;author=P.H.%20Graham&amp;author=C.P.%20Vance&amp;publication_year=2003&amp;journal=Plant%20Physiol.&amp;volume=131&amp;pages=872-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.017004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.017004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.017004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12644639" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw033-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Handberg</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Stougaard</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="year">1992</div>, 
<div class="article-title">Lotus-Japonicus, an autogamous, diploid legume species for classical and molecular-genetics,</div><div class="source">Plant J</div> ., <div class="volume">2</div>(<div class="issue">4</div>), <div class="fpage">487</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lotus-Japonicus%2C%20an%20autogamous%2C%20diploid%20legume%20species%20for%20classical%20and%20molecular-genetics%2C&amp;author=K.%20Handberg&amp;author=J.%20Stougaard&amp;publication_year=1992&amp;journal=Plant%20J&amp;volume=2&amp;pages=487-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.1992.2.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.1992.2.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.1992.2.issue-4"> </span></div> </div></div></div></div></div><div content-id="dsw033-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sato</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Tabata</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">2006</div>, 
<div class="article-title"><em>Lotus japonicus</em> as a platform for legume research,</div><div class="source">Curr. Opin. Plant Biol</div> ., <div class="volume">9</div>(<div class="issue">2</div>), <div class="fpage">128</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lotus%20japonicus%20as%20a%20platform%20for%20legume%20research%2C&amp;author=S.%20Sato&amp;author=S.%20Tabata&amp;publication_year=2006&amp;journal=Curr.%20Opin.%20Plant%20Biol&amp;volume=9&amp;pages=128-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2006.01.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2006.01.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2006.01.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16480917" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw033-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kawasaki</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Murakami</div>
               <div class="given-names">Y.</div>
            </div>
          </div>
          <div class="year">2000</div>, 
<div class="article-title">Genome analysis of <em>Lotus japonicus</em>,</div><div class="source">J. Plant Res</div> ., <div class="volume">113</div>(<div class="issue">1112</div>), <div class="fpage">497</div>–<div class="lpage">506</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20analysis%20of%20Lotus%20japonicus%2C&amp;author=S.%20Kawasaki&amp;author=Y.%20Murakami&amp;publication_year=2000&amp;journal=J.%20Plant%20Res&amp;volume=113&amp;pages=497-506" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00013960" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00013960" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00013960"> </span></div> </div></div></div></div></div><div content-id="dsw033-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sato</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Kaneko</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Nakamura</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Asamizu</div>
               <div class="given-names">E.</div>
            </div>
             <div class="name">
              <div class="surname">Kato</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Tabata</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">2001</div>, 
<div class="article-title">Structural analysis of a <em>Lotus japonicus</em> genome. I. Sequence features and mapping of fifty-six TAC clones which cover the 5.4 Mb regions of the genome,</div><div class="source">DNA Res</div> ., <div class="volume">8</div>(<div class="issue">6</div>), <div class="fpage">311</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20analysis%20of%20a%20Lotus%20japonicus%20genome.%20I.%20Sequence%20features%20and%20mapping%20of%20fifty-six%20TAC%20clones%20which%20cover%20the%205.4%20Mb%20regions%20of%20the%20genome%2C&amp;author=S.%20Sato&amp;author=T.%20Kaneko&amp;author=Y.%20Nakamura&amp;author=E.%20Asamizu&amp;author=T.%20Kato&amp;author=S.%20Tabata&amp;publication_year=2001&amp;journal=DNA%20Res&amp;volume=8&amp;pages=311-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/8.6.311" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F8.6.311" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F8.6.311"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11853318" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw033-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Fukai</div>
               <div class="given-names">E.</div>
            </div>
             <div class="name">
              <div class="surname">Soyano</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Umehara</div>
               <div class="given-names">Y.</div>
            </div>
          </div>
          <div class="year">2012</div>, 
<div class="article-title">Establishment of a <em>Lotus japonicus</em> gene tagging population using the exon-targeting endogenous retrotransposon LORE1,</div><div class="source">Plant J</div> ., <div class="volume">69</div>(<div class="issue">4</div>), <div class="fpage">720</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Establishment%20of%20a%20Lotus%20japonicus%20gene%20tagging%20population%20using%20the%20exon-targeting%20endogenous%20retrotransposon%20LORE1%2C&amp;author=E.%20Fukai&amp;author=T.%20Soyano&amp;author=Y.%20Umehara&amp;publication_year=2012&amp;journal=Plant%20J&amp;volume=69&amp;pages=720-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2012.69.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2012.69.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2012.69.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22014259" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw033-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Urbanski</div>
               <div class="given-names">D.F.</div>
            </div>
             <div class="name">
              <div class="surname">Małolepszy</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Stougaard</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Andersen</div>
               <div class="given-names">S.U.</div>
            </div>
          </div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Genome-wide LORE1 retrotransposon mutagenesis and high-throughput insertion detection in <em>Lotus japonicus</em>,</div><div class="source">Plant J</div> ., <div class="volume">69</div>(<div class="issue">4</div>), <div class="fpage">731</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20LORE1%20retrotransposon%20mutagenesis%20and%20high-throughput%20insertion%20detection%20in%20Lotus%20japonicus%2C&amp;author=D.F.%20Urbanski&amp;author=A.%20Ma%C5%82olepszy&amp;author=J.%20Stougaard&amp;author=S.U.%20Andersen&amp;publication_year=2012&amp;journal=Plant%20J&amp;volume=69&amp;pages=731-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2012.69.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2012.69.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2012.69.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22014280" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw033-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Perry</div>
               <div class="given-names">J.A.</div>
            </div>
             <div class="name">
              <div class="surname">Wang</div>
               <div class="given-names">T.L.</div>
            </div>
             <div class="name">
              <div class="surname">Welham</div>
               <div class="given-names">T.J.</div>
            </div>
          </div>, et al.  . <div class="year">2003</div>, 
<div class="article-title">A TILLING reverse genetics tool and a web-accessible collection of mutants of the legume <em>Lotus japonicus</em>,</div><div class="source">Plant Physiol</div> ., <div class="volume">131</div>(<div class="issue">3</div>), <div class="fpage">866</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20TILLING%20reverse%20genetics%20tool%20and%20a%20web-accessible%20collection%20of%20mutants%20of%20the%20legume%20Lotus%20japonicus%2C&amp;author=J.A.%20Perry&amp;author=T.L.%20Wang&amp;author=T.J.%20Welham&amp;publication_year=2003&amp;journal=Plant%20Physiol&amp;volume=131&amp;pages=866-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.102.017384" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.102.017384" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.102.017384"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12644638" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw033-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Asamizu</div>
               <div class="given-names">E.</div>
            </div>
             <div class="name">
              <div class="surname">Watanabe</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Tabata</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">2000</div>, 
<div class="article-title">Large scale structural analysis of cDNAs in the model legume, <em>Lotus japonicus</em>,</div><div class="source">J. Plant Res.</div> <div class="volume">113</div>(<div class="issue">1112</div>), <div class="fpage">451</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large%20scale%20structural%20analysis%20of%20cDNAs%20in%20the%20model%20legume%2C%20Lotus%20japonicus%2C&amp;author=E.%20Asamizu&amp;author=M.%20Watanabe&amp;author=S.%20Tabata&amp;publication_year=2000&amp;journal=J.%20Plant%20Res.&amp;volume=113&amp;pages=451-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00013955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00013955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00013955"> </span></div> </div></div></div></div></div><div content-id="dsw033-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Hayashi</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Miyahara</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Sato</div>
               <div class="given-names">S.</div>
            </div>
          </div>, et al.  . <div class="year">2001</div>, 
<div class="article-title">Construction of a genetic linkage map of the model legume <em>Lotus japonicus</em> using an intraspecific F-2 population,</div><div class="source">DNA Res</div> ., <div class="volume">8</div>(<div class="issue">6</div>), <div class="fpage">301</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20a%20genetic%20linkage%20map%20of%20the%20model%20legume%20Lotus%20japonicus%20using%20an%20intraspecific%20F-2%20population%2C&amp;author=M.%20Hayashi&amp;author=A.%20Miyahara&amp;author=S.%20Sato&amp;publication_year=2001&amp;journal=DNA%20Res&amp;volume=8&amp;pages=301-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/8.6.301" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F8.6.301" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F8.6.301"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11853317" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw033-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sandal</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Krusell</div>
               <div class="given-names">L.</div>
            </div>
             <div class="name">
              <div class="surname">Radutoiu</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">2002</div>, 
<div class="article-title">A genetic linkage map of the model legume <em>Lotus japonicus</em> and strategies for fast mapping of new loci,</div><div class="source">Genetics</div> <div class="volume">161</div>(<div class="issue">4</div>), <div class="fpage">1673</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genetic%20linkage%20map%20of%20the%20model%20legume%20Lotus%20japonicus%20and%20strategies%20for%20fast%20mapping%20of%20new%20loci%2C&amp;author=N.%20Sandal&amp;author=L.%20Krusell&amp;author=S.%20Radutoiu&amp;publication_year=2002&amp;journal=Genetics&amp;volume=161&amp;pages=1673-83" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12196410" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw033-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sato</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Nakamura</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Kaneko</div>
               <div class="given-names">T.</div>
            </div>
          </div>, et al.  . <div class="year">2008</div>, 
<div class="article-title">Genome structure of the legume, <em>Lotus japonicus</em>,</div><div class="source">DNA Res</div> ., <div class="volume">15</div>(<div class="issue">4</div>), <div class="fpage">227</div>–<div class="lpage">39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20structure%20of%20the%20legume%2C%20Lotus%20japonicus%2C&amp;author=S.%20Sato&amp;author=Y.%20Nakamura&amp;author=T.%20Kaneko&amp;publication_year=2008&amp;journal=DNA%20Res&amp;volume=15&amp;pages=227-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18511435" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw033-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sandal</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Jin</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Rodriguez-Navarro</div>
               <div class="given-names">D.N.</div>
            </div>
          </div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">A set of <em>Lotus japonicus</em> Gifu x Lotus burttii recombinant inbred lines facilitates map-based cloning and QTL mapping,</div><div class="source">DNA Res</div> , <div class="volume">19</div>(<div class="issue">4</div>), <div class="fpage">317</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20set%20of%20Lotus%20japonicus%20Gifu%20x%20Lotus%20burttii%20recombinant%20inbred%20lines%20facilitates%20map-based%20cloning%20and%20QTL%20mapping%2C&amp;author=N.%20Sandal&amp;author=H.%20Jin&amp;author=D.N.%20Rodriguez-Navarro&amp;publication_year=2012&amp;journal=DNA%20Res&amp;volume=19&amp;pages=317-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dss014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdss014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdss014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22619310" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw033-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Jiang</div>
               <div class="given-names">Q.Y.</div>
            </div>
             <div class="name">
              <div class="surname">Gresshoff</div>
               <div class="given-names">P.M.</div>
            </div>
          </div>
          <div class="year">1997</div>, 
<div class="article-title">Classical and molecular genetics of the model legume <em>Lotus japonicus</em>,</div><div class="source">Mol. Plant Microbe Interact</div> ., <div class="volume">10</div>(<div class="issue">1</div>), <div class="fpage">59</div>–<div class="lpage">68</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Classical%20and%20molecular%20genetics%20of%20the%20model%20legume%20Lotus%20japonicus%2C&amp;author=Q.Y.%20Jiang&amp;author=P.M.%20Gresshoff&amp;publication_year=1997&amp;journal=Mol.%20Plant%20Microbe%20Interact&amp;volume=10&amp;pages=59-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI.1997.10.1.59" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI.1997.10.1.59" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI.1997.10.1.59"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9002271" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw033-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Sandal</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Petersen</div>
               <div class="given-names">T.R.</div>
            </div>
             <div class="name">
              <div class="surname">Murray</div>
               <div class="given-names">J.</div>
            </div>
          </div>, et al.  . <div class="year">2006</div>, 
<div class="article-title">Genetics of symbiosis in <em>Lotus japonicus</em>: recombinant inbred lines, comparative genetic maps, and map position of 35 symbiotic loci,</div><div class="source">Mol Plant Microbe Interact</div> ., <div class="volume">19</div>(<div class="issue">1</div>), <div class="fpage">80</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetics%20of%20symbiosis%20in%20Lotus%20japonicus%3A%20recombinant%20inbred%20lines%2C%20comparative%20genetic%20maps%2C%20and%20map%20position%20of%2035%20symbiotic%20loci%2C&amp;author=N.%20Sandal&amp;author=T.R.%20Petersen&amp;author=J.%20Murray&amp;publication_year=2006&amp;journal=Mol%20Plant%20Microbe%20Interact&amp;volume=19&amp;pages=80-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/MPMI-19-0080" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FMPMI-19-0080" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FMPMI-19-0080"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16404956" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw033-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Gondo</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Sato</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Okumura</div>
               <div class="given-names">K.</div>
            </div>
             <div class="name">
              <div class="surname">Tabata</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Akashi</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Isobe</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">2007</div>, 
<div class="article-title">Quantitative trait locus analysis of multiple agronomic traits in the model legume <em>Lotus japonicus</em>,</div><div class="source">Genome</div> , <div class="volume">50</div>(<div class="issue">7</div>), <div class="fpage">627</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20trait%20locus%20analysis%20of%20multiple%20agronomic%20traits%20in%20the%20model%20legume%20Lotus%20japonicus%2C&amp;author=T.%20Gondo&amp;author=S.%20Sato&amp;author=K.%20Okumura&amp;author=S.%20Tabata&amp;author=R.%20Akashi&amp;author=S.%20Isobe&amp;publication_year=2007&amp;journal=Genome&amp;volume=50&amp;pages=627-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/G07-040" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FG07-040" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FG07-040"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17893740" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw033-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Hyten</div>
               <div class="given-names">D.L.</div>
            </div>
             <div class="name">
              <div class="surname">Choi</div>
               <div class="given-names">I.-Y.</div>
            </div>
             <div class="name">
              <div class="surname">Song</div>
               <div class="given-names">Q.</div>
            </div>
             <div class="name">
              <div class="surname">Specht</div>
               <div class="given-names">J.E.</div>
            </div>
             <div class="name">
              <div class="surname">Carter</div>
               <div class="given-names">T.E.</div>
              Jr
            </div>
          </div>
          <div class="year">2010</div>, 
<div class="article-title">A high density integrated genetic linkage map of soybean and the development of a 1536 universal soy linkage panel for quantitative trait locus mapping</div>, <div class="source">Crop Sci</div> ., <div class="volume">50</div>(<div class="issue">3</div>), <div class="fpage">960</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high%20density%20integrated%20genetic%20linkage%20map%20of%20soybean%20and%20the%20development%20of%20a%201536%20universal%20soy%20linkage%20panel%20for%20quantitative%20trait%20locus%20mapping&amp;author=D.L.%20Hyten&amp;author=I.-Y.%20Choi&amp;author=Q.%20Song&amp;author=J.E.%20Specht&amp;author=T.E.%20Carter&amp;publication_year=2010&amp;journal=Crop%20Sci&amp;volume=50&amp;pages=960-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2009.06.0360" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2009.06.0360" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2009.06.0360"> </span></div> </div></div></div></div></div><div content-id="dsw033-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Li</div>
               <div class="given-names">X.H.</div>
            </div>
             <div class="name">
              <div class="surname">Wei</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Acharya</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Jiang</div>
               <div class="given-names">Q.</div>
            </div>
             <div class="name">
              <div class="surname">Kang</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Brummer</div>
               <div class="given-names">E.C.</div>
            </div>
          </div>
          <div class="year">2014</div>, 
<div class="article-title">A saturated genetic linkage map of autotetraploid alfalfa (<em>Medicago sativa</em> L.) developed using genotyping-by-sequencing is highly syntenous with the medicago truncatula genome</div>, <div class="source">G3-Genes Genomes Genet</div> ., <div class="volume">4</div>(<div class="issue">10</div>), <div class="fpage">1971</div>–<div class="lpage">9</div>.</div></div></div></div><div content-id="dsw033-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Gaur</div>
               <div class="given-names">R.</div>
            </div>
             <div class="name">
              <div class="surname">Jeena</div>
               <div class="given-names">G.</div>
            </div>
             <div class="name">
              <div class="surname">Shah</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Gupta</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Pradhan</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Tyagi</div>
               <div class="given-names">A. K.</div>
            </div>
             <div class="name">
              <div class="surname">Jain, </div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Chattopadhyay</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Bhatia</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="year">2015</div>, 
<div class="article-title">High density linkage mapping of genomic and transcriptomic SNPs for synteny analysis and anchoring the genome sequence of chickpea,</div><div class="source">Sci. Rep</div> ., <div class="volume">5</div>, <div class="fpage">13387</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20density%20linkage%20mapping%20of%20genomic%20and%20transcriptomic%20SNPs%20for%20synteny%20analysis%20and%20anchoring%20the%20genome%20sequence%20of%20chickpea%2C&amp;author=R.%20Gaur&amp;author=G.%20Jeena&amp;author=N.%20Shah&amp;author=S.%20Gupta&amp;author=S.%20Pradhan&amp;author=A.%20K.%20Tyagi&amp;author=M.%20Jain%2C&amp;author=D.%20Chattopadhyay&amp;author=S.%20Bhatia&amp;publication_year=2015&amp;journal=Sci.%20Rep&amp;volume=5&amp;pages=13387" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep13387" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep13387" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep13387"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26303721" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw033-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Singer</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Fan</div>
               <div class="given-names">Y.</div>
            </div>
             <div class="name">
              <div class="surname">Chang</div>
               <div class="given-names">H.S.</div>
            </div>
             <div class="name">
              <div class="surname">Zhu</div>
               <div class="given-names">T.</div>
            </div>
             <div class="name">
              <div class="surname">Hazen</div>
               <div class="given-names">S.P.</div>
            </div>
             <div class="name">
              <div class="surname">Briggs</div>
               <div class="given-names">S.P.</div>
            </div>
          </div>
          <div class="year">2006</div>, 
<div class="article-title">A high-resolution map of Arabidopsis recombinant inbred lines by whole-genome exon array hybridization,</div><div class="source">Plos Genet.</div> <div class="volume">2</div>(<div class="issue">9</div>), <div class="fpage">1352</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-resolution%20map%20of%20Arabidopsis%20recombinant%20inbred%20lines%20by%20whole-genome%20exon%20array%20hybridization%2C&amp;author=T.%20Singer&amp;author=Y.%20Fan&amp;author=H.S.%20Chang&amp;author=T.%20Zhu&amp;author=S.P.%20Hazen&amp;author=S.P.%20Briggs&amp;publication_year=2006&amp;journal=Plos%20Genet.&amp;volume=2&amp;pages=1352-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.0020144" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.0020144" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.0020144"> </span></div> </div></div></div></div></div><div content-id="dsw033-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Yu</div>
               <div class="given-names">X.</div>
            </div>
             <div class="name">
              <div class="surname">Wang</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Zhong</div>
               <div class="given-names">W.</div>
            </div>
             <div class="name">
              <div class="surname">Bai</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Liu</div>
               <div class="given-names">P.</div>
            </div>
             <div class="name">
              <div class="surname">He</div>
               <div class="given-names">Y.</div>
            </div>
          </div>
          <div class="year">2013</div>, 
<div class="article-title">QTL mapping of leafy heads by genome resequencing in the RIL population of <em>Brassica rapa</em>,</div><div class="source">Plos One</div> , <div class="volume">8</div>(<div class="issue">10</div>), <div class="fpage">e76059</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=QTL%20mapping%20of%20leafy%20heads%20by%20genome%20resequencing%20in%20the%20RIL%20population%20of%20Brassica%20rapa%2C&amp;author=X.%20Yu&amp;author=H.%20Wang&amp;author=W.%20Zhong&amp;author=J.%20Bai&amp;author=P.%20Liu&amp;author=Y.%20He&amp;publication_year=2013&amp;journal=Plos%20One&amp;volume=8&amp;pages=e76059" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0076059" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0076059" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0076059"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24204591" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw033-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Huang</div>
               <div class="given-names">X.H.</div>
            </div>
             <div class="name">
              <div class="surname">Feng</div>
               <div class="given-names">Q.</div>
            </div>
             <div class="name">
              <div class="surname">Qian</div>
               <div class="given-names">Q.</div>
            </div>
          </div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">High-throughput genotyping by whole-genome resequencing,</div><div class="source">Genome Res</div> ., <div class="volume">19</div>(<div class="issue">6</div>), <div class="fpage">1068</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20genotyping%20by%20whole-genome%20resequencing%2C&amp;author=X.H.%20Huang&amp;author=Q.%20Feng&amp;author=Q.%20Qian&amp;publication_year=2009&amp;journal=Genome%20Res&amp;volume=19&amp;pages=1068-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.089516.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.089516.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.089516.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19420380" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw033-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kawaguchi</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="year">2000</div>, 
<div class="article-title"><em>Lotus japonicus</em> 'Miyakojima' MG-20: an early-flowering accession suitable for indoor handling</div>. <div class="source">J. Plant Res</div> ., <div class="volume">113</div>(<div class="issue">1112</div>), <div class="fpage">507</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lotus%20japonicus%20'Miyakojima'%20MG-20%3A%20an%20early-flowering%20accession%20suitable%20for%20indoor%20handling&amp;author=M.%20Kawaguchi&amp;publication_year=2000&amp;journal=J.%20Plant%20Res&amp;volume=113&amp;pages=507-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00013961" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00013961" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00013961"> </span></div> </div></div></div></div></div><div content-id="dsw033-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Borsos</div>
               <div class="given-names">O.S.</div>
            </div>
             <div class="name">
              <div class="surname">Somaroo</div>
               <div class="given-names">B.H.</div>
            </div>
             <div class="name">
              <div class="surname">Grant</div>
               <div class="given-names">W.F.</div>
            </div>
          </div>
          <div class="year">1972</div>, 
<div class="article-title">New diploid species of Lotus (Leguminosae) in Pakistan</div>, <div class="source">Can. J. Botany</div> , <div class="volume">50</div>(<div class="issue">9</div>), <div class="fpage">1865</div>–<div class="lpage">1870</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20diploid%20species%20of%20Lotus%20(Leguminosae)%20in%20Pakistan&amp;author=O.S.%20Borsos&amp;author=B.H.%20Somaroo&amp;author=W.F.%20Grant&amp;publication_year=1972&amp;journal=Can.%20J.%20Botany&amp;volume=50&amp;pages=1865-1870" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/b72-235" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fb72-235" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fb72-235"> </span></div> </div></div></div></div></div><div content-id="dsw033-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Kawaguchi</div>
               <div class="given-names">M.</div>
            </div>
             <div class="name">
              <div class="surname">Pedrosa-Harand</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Yano</div>
               <div class="given-names">K.</div>
            </div>
          </div>, et al.  . <div class="year">2005</div>, 
<div class="article-title"><em>Lotus burttii</em> takes a position of the third corner in the lotus molecular genetics triangle,</div><div class="source">DNA Res</div> ., <div class="volume">12</div>(<div class="issue">1</div>), <div class="fpage">69</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lotus%20burttii%20takes%20a%20position%20of%20the%20third%20corner%20in%20the%20lotus%20molecular%20genetics%20triangle%2C&amp;author=M.%20Kawaguchi&amp;author=A.%20Pedrosa-Harand&amp;author=K.%20Yano&amp;publication_year=2005&amp;journal=DNA%20Res&amp;volume=12&amp;pages=69-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/12.1.69" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F12.1.69" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F12.1.69"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16106754" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw033-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Murray</div>
               <div class="given-names">M.G.</div>
            </div>
             <div class="name">
              <div class="surname">Thompson</div>
               <div class="given-names">W.F.</div>
            </div>
          </div>
          <div class="year">1980</div>, 
<div class="article-title">Rapid isolation of high molecular-weight plant DNA,</div><div class="source">Nucleic Acids Res</div> ., <div class="volume">8</div>(<div class="issue">19</div>), <div class="fpage">4321</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20isolation%20of%20high%20molecular-weight%20plant%20DNA%2C&amp;author=M.G.%20Murray&amp;author=W.F.%20Thompson&amp;publication_year=1980&amp;journal=Nucleic%20Acids%20Res&amp;volume=8&amp;pages=4321-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/8.19.4321" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F8.19.4321" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F8.19.4321"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7433111" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw033-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Durbin</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="year">2009</div>, 
<div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform,</div><div class="source">Bioinformatics</div> , <div class="volume">25</div>(<div class="issue">14</div>), <div class="fpage">1754</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform%2C&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw033-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Handsaker</div>
               <div class="given-names">B.</div>
            </div>
             <div class="name">
              <div class="surname">Wysoke</div>
               <div class="given-names">A.</div>
            </div>
          </div>, et al.  ., <div class="year">2009</div>, 
<div class="article-title">The sequence alignment/map format and SAMtools,</div><div class="source">Bioinformatics</div> , <div class="volume">25</div>(<div class="issue">16</div>), <div class="fpage">2078</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20alignment%2Fmap%20format%20and%20SAMtools%2C&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoke&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw033-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Broman</div>
               <div class="given-names">K.W.</div>
            </div>
             <div class="name">
              <div class="surname">Wu</div>
               <div class="given-names">H.</div>
            </div>
             <div class="name">
              <div class="surname">Sen</div>
               <div class="given-names">S.</div>
            </div>
             <div class="name">
              <div class="surname">Churchill</div>
               <div class="given-names">G.A.</div>
            </div>
          </div>
          <div class="year">2003</div>, 
<div class="article-title">R/qtl: QTL mapping in experimental crosses,</div><div class="source">Bioinformatics</div> , <div class="volume">19</div>(<div class="issue">7</div>), <div class="fpage">889</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=R%2Fqtl%3A%20QTL%20mapping%20in%20experimental%20crosses%2C&amp;author=K.W.%20Broman&amp;author=H.%20Wu&amp;author=S.%20Sen&amp;author=G.A.%20Churchill&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=889-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12724300" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw033-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw033-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name">
              <div class="surname">Pedrosa</div>
               <div class="given-names">A.</div>
            </div>
             <div class="name">
              <div class="surname">Sandal</div>
               <div class="given-names">N.</div>
            </div>
             <div class="name">
              <div class="surname">Stougaard</div>
               <div class="given-names">J.</div>
            </div>
             <div class="name">
              <div class="surname">Schweizer</div>
               <div class="given-names">D.</div>
            </div>
             <div class="name">
              <div class="surname">Bachmair</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">2002</div>, 
<div class="article-title">Chromosomal map of the model legume <em>Lotus japonicus</em></div>, <div class="source">Genetics</div> , <div class="volume">161</div>(<div class="issue">4</div>), <div class="fpage">1661</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosomal%20map%20of%20the%20model%20legume%20Lotus%20japonicus&amp;author=A.%20Pedrosa&amp;author=N.%20Sandal&amp;author=J.%20Stougaard&amp;author=D.%20Schweizer&amp;author=A.%20Bachmair&amp;publication_year=2002&amp;journal=Genetics&amp;volume=161&amp;pages=1661-72" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12196409" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2236185&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/5/10.1093_dnares_dsw033/1/dsw033_Supplementary_Data.zip?Expires=2147483647&amp;Signature=Gr4d2if-3VqP-bvLTKyu-Y0FcYRshQGdUtKuZ0O~vOtGB6VneOJF21eKbrAvnB5OnzzacBR2lBUdIeR3y~Zp0xcOBQnybd78Qq-hPxC46PDxxVYGXnBqIDiai8bXQPVdS2UR-wqWEMmzJv5UTuthaLj3gA-wmpJhLU7TZdc0qESDGhflzS1LcE7CBnMcYW4fAzgxxAcb2115nW2l1-iBHXbEvONyuc7mTf~ElQMHuY9L1xfcjhU2Ct28TcmRkFHls8U-ZVhgqRskQDP63-mvOzzaz8QirEyZAjFhgsZDR45iFzTRGIddTzN70kcyuomKKXZuTlcZ0BlBr6LCAAP6bw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">334</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw033"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2236185"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f5%2f487%2f2236185">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F5%2F487%2F2236185" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2236185'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="High-resolution genetic maps of Lotus japonicus and L. burttii based on re-sequencing of recombinant inbred lines" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw033");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f5%2f487%2f2236185">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

