

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/16/2/91/510591","@type":"ScholarlyArticle","name":"Codon Usages of Genes on Chromosome, and Surprisingly, Genes in Plasmid are Primarily Affected by Strand-specific Mutational Biases in  Lawsonia intracellularis ","datePublished":"2009-02-15","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/16/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2009-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsp001","keywords":["<italic>Lawsonia intracellularis</italic>","codon usage","strand-specific mutational bias","plasmid","replication mechanism"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/16/2/91/510591","author":[{"name":"Guo, Feng-Biao","affiliation":"School of Life Science and Technology ,  University of Electronic Science and Technology of China ,  Chengdu 610054 ,  People's Republic of China","@type":"Person"},{"name":"Yuan, Jian-Bo","affiliation":"School of Life Science and Technology ,  University of Electronic Science and Technology of China ,  Chengdu 610054 ,  People's Republic of China","@type":"Person"}],"description":"Abstract.  In this study, the factors driving genome-wide patterns of codon usages in Lawsonia intracellularis genome are determined. For genes on the chromosom","pageStart":"91","pageEnd":"104","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/2/1/m_cover.gif?Expires=2147483647&Signature=spYD3Y1ECWwq5jfgvLwgtJGBnOHAN-nZXtWf5jX0Idx1cbrJRV4FKB28EY3pe17T0gsGMxo5dA8Tel6c2xNGAqa1byNOzqnWfnM8ohlcCfLHlrfGOEn2zyCW-15VbU7SOoLTo3OmdKcdJ19GrDbXCyAEcU~VJnaBCNGoOwBtbTnh6-S6KzBaPxjqj1HfbfKuNRe7H3r-0uFeA~Y3Ph79tw8QP6q4D0FiiXdjGGWYTNp3-Ga~m-PPAcOogXTotI1YFUthxQJgvIKyhC8JLbL3Znx728-CpY2KbMeFQlltwFxSw7d-mO5IOuvD7Wc91P7OLYCdnpnsyiP0k-TqunqvaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Codon Usages of Genes on Chromosome, and Surprisingly, Genes in Plasmid are Primarily Affected by Strand-specific Mutational Biases in  Lawsonia intracellularis ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/2/1/m_cover.gif?Expires=2147483647&Signature=spYD3Y1ECWwq5jfgvLwgtJGBnOHAN-nZXtWf5jX0Idx1cbrJRV4FKB28EY3pe17T0gsGMxo5dA8Tel6c2xNGAqa1byNOzqnWfnM8ohlcCfLHlrfGOEn2zyCW-15VbU7SOoLTo3OmdKcdJ19GrDbXCyAEcU~VJnaBCNGoOwBtbTnh6-S6KzBaPxjqj1HfbfKuNRe7H3r-0uFeA~Y3Ph79tw8QP6q4D0FiiXdjGGWYTNp3-Ga~m-PPAcOogXTotI1YFUthxQJgvIKyhC8JLbL3Znx728-CpY2KbMeFQlltwFxSw7d-mO5IOuvD7Wc91P7OLYCdnpnsyiP0k-TqunqvaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Codon Usages of Genes on Chromosome, and Surprisingly, Genes in Plasmid are Primarily Affected by Strand-specific Mutational Biases in  Lawsonia intracellularis " />
    <meta property="og:description" content="Abstract.  In this study, the factors driving genome-wide patterns of codon usages in Lawsonia intracellularis genome are determined. For genes on the chromosom" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/16/2/91/510591" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2009-02-15" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Guo, Feng-Biao" /><meta name="citation_author_institution" content="School of Life Science and Technology, University of Electronic Science and Technology of China, Chengdu 610054, People's Republic of China" /><meta name="citation_author" content="Yuan, Jian-Bo" /><meta name="citation_author_institution" content="School of Life Science and Technology, University of Electronic Science and Technology of China, Chengdu 610054, People's Republic of China" /><meta name="citation_title" content="Codon Usages of Genes on Chromosome, and Surprisingly, Genes in Plasmid are Primarily Affected by Strand-specific Mutational Biases in Lawsonia intracellularis" /><meta name="citation_firstpage" content="91" /><meta name="citation_lastpage" content="104" /><meta name="citation_doi" content="10.1093/dnares/dsp001" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="16" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2009/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Codon catalog usage and the genome hypothesis; Nucleic Acids Res.;  citation_year=1980;  citation_volume=8;  citation_pages=r49-r62; " /><meta name="citation_reference" content="citation_title=Codon catalog usage is a genome strategy modulated for gene expressivity; Nucleic Acids Res.;  citation_year=1981;  citation_volume=9;  citation_pages=r43-r74; " /><meta name="citation_reference" content="citation_title=Codon usage in bacteria: correlation with gene expressivity; Nucleic Acids Res.;  citation_year=1982;  citation_volume=10;  citation_pages=7055-7074; " /><meta name="citation_reference" content="citation_title=Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the E. coli translational system; J. Mol. Biol.;  citation_year=1981;  citation_volume=151;  citation_pages=389-409; " /><meta name="citation_reference" content="citation_title=Co-variation of tRNA abundance and codon usage in Escherichia coli at different growth rates; J. Mol. Biol.;  citation_year=1996;  citation_volume=260;  citation_pages=649-663; " /><meta name="citation_reference" content="citation_title=Codon selection in yeast; J. Biol. Chem.;  citation_year=1982;  citation_volume=257;  citation_pages=3026-3031; " /><meta name="citation_reference" content="citation_title=Codon usage in the Mycobacterium tuberculosis complex; Microbiology;  citation_year=1996;  citation_volume=142;  citation_pages=915-925; " /><meta name="citation_reference" content="citation_title=Absence of translationally selected synonymous codon usage bias in Helicobacter pylori; Microbiology;  citation_year=2000;  citation_volume=146;  citation_pages=851-860; " /><meta name="citation_reference" content="citation_title=Studies of codon usage and tRNA genes of 18 unicellular organisms and quantification of Bacillus subtilis tRNAs: gene expression level and species-specific diversity of codon usage based on multivariate analysis; Gene;  citation_year=1999;  citation_volume=238;  citation_pages=143-155; " /><meta name="citation_reference" content="citation_title=Replicational and transcriptional selection on codon usage in Borrelia burgdorferi; Proc. Natl Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=10698-10703; " /><meta name="citation_reference" content="citation_title=Codon usage in Chlamydia trachomatis is the result of strand-specific mutational biases and a complex pattern of selective forces; Nucleic Acids Res.;  citation_year=2000;  citation_volume=28;  citation_pages=2084-2090; " /><meta name="citation_reference" content="citation_title=Synonymous codon usage in Pseudomonas aeruginosa PA01; Gene;  citation_year=2002;  citation_volume=289;  citation_pages=131-139; " /><meta name="citation_reference" content="citation_title=Horizontal gene transfer in bacterial and archaeal complete genomes; Genome Res.;  citation_year=2000;  citation_volume=10;  citation_pages=1719-1725; " /><meta name="citation_reference" content="citation_title=HGT-DB: a database of putative horizontally transferred genes in prokaryotic complete genomes; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=187-189; " /><meta name="citation_reference" content="citation_title=Evidence for horizontal gene transfer in Escherichia coli speciation; J. Mol. Biol.;  citation_year=1991;  citation_volume=222;  citation_pages=851-856; " /><meta name="citation_reference" content="citation_title=Codon usage and lateral gene transfer in Bacillus subtilis; Curr. Opin. Microbiol.;  citation_year=1999;  citation_volume=2;  citation_pages=524-528; " /><meta name="citation_reference" content="citation_title=Prokaryotic genome evolution as assessed by multivariate analysis of codon usage patterns; Microb. Comp. Genomics;  citation_year=1997;  citation_volume=2;  citation_pages=1-10; " /><meta name="citation_reference" content="citation_title=Base composition skews, replication orientation, and gene orientation in 12 prokaryote genomes; J. Mol. Evol.;  citation_year=1998;  citation_volume=47;  citation_pages=691-696; " /><meta name="citation_reference" content="citation_title=Asymmetric substitution patterns: a review of possible underlying mutational or selective mechanisms; Gene;  citation_year=1999;  citation_volume=238;  citation_pages=65-77; " /><meta name="citation_reference" content="citation_title=Strand compositional asymmetry in bacterial and large viral genomes; Proc. Natl Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=3720-3725; " /><meta name="citation_reference" content="citation_title=How does replication-associated mutational pressure influence amino acid composition of proteins?; Genome Res.;  citation_year=1999;  citation_volume=9;  citation_pages=409-416; " /><meta name="citation_reference" content="citation_title=Asymmetric directional mutation pressures in bacteria; citation_text=RESEARCH0058; Genome Biol.;  citation_year=2002;  citation_volume=3; " /><meta name="citation_reference" content="citation_title=Essentiality, not expressiveness, drives gene-strand bias in bacteria; Nat Genet.;  citation_year=2003;  citation_volume=34;  citation_pages=377-378; " /><meta name="citation_reference" content="citation_title=Interruptions in gene expression drive highly expressed operons to the leading strand of DNA replication; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=3224-3234; " /><meta name="citation_reference" content="citation_title=Strong asymmetric mutation bias in endosymbiont genomes coincide with loss of genes for replication restart pathways; Mol. Biol. Evol.;  citation_year=2006;  citation_volume=23;  citation_pages=1031-1039; " /><meta name="citation_reference" content="citation_title=Similar compositional biases are caused by very different mutational effects; Genome Res.;  citation_year=2006;  citation_volume=16;  citation_pages=1537-1547; " /><meta name="citation_reference" content="citation_title=A new method for assessing the effect of replication on DNA base composition asymmetry; Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=2169-2179; " /><meta name="citation_reference" content="citation_title=Origin of replication of Mycoplasma genitalium; Science;  citation_year=1996;  citation_volume=272;  citation_pages=745-746; " /><meta name="citation_reference" content="citation_title=Asymmetric substitution patterns in the two DNA strands of bacteria; Mol. Biol. Evol.;  citation_year=1996;  citation_volume=13;  citation_pages=660-665; " /><meta name="citation_reference" content="citation_title=Proteome composition and codon usage in spirochaetes: species-specific and DNA strand-specific mutational biases; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=1642-1649; " /><meta name="citation_reference" content="citation_title=Mutational and selective pressures on codon and amino acid usage in Buchnera, endosymbiotic bacteria of aphids; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=44-53; " /><meta name="citation_reference" content="citation_title=Evolutionary forces in shaping the codon and amino acid usages in Blochmannia floridanus; J. Biomol. Struct. Dyn.;  citation_year=2004;  citation_volume=22;  citation_pages=13-23; " /><meta name="citation_reference" content="citation_title=Codon and amino acid usage in two major human pathogens of genus Bartonella–optimization between replicational–transcriptional selection, translational control and cost minimization; DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=91-102; " /><meta name="citation_reference" content="citation_title=Evolutionary constraints on codon and amino acid usage in two strains of human pathogenic actinobacteria Tropheryma whipplei; J. Mol. Evol.;  citation_year=2006;  citation_volume=62;  citation_pages=645-658; " /><meta name="citation_reference" content="citation_title=Separate base usages of genes located on the leading and lagging strands in Chlamydia muridarum revealed by the Z curve method; BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=366" /><meta name="citation_reference" content="citation_title=The microbiota of pigs influenced by diet texture and severity of Lawsonia intracellularis infection; Vet Microbiol.;  citation_year=2008;  citation_volume=128;  citation_pages=96-107; " /><meta name="citation_reference" content="citation_title=Analysis of codon usage; citation_text=Ph.D. Thesis; citation_publisher=University of Nottingham, ;  citation_year=1999; " /><meta name="citation_reference" content="citation_title=The ‘effective number of codons’ used in a gene; Gene;  citation_year=1990;  citation_volume=87;  citation_pages=23-29; " /><meta name="citation_reference" content="citation_title=Use and misuse of correspondence analysis in codon usage studies; Nucleic Acids Res.;  citation_year=2002;  citation_volume=30;  citation_pages=4548-4555; " /><meta name="citation_reference" content="citation_publisher=Willey Press, New York, USA; Multivariate Analysis, Method and Application;  citation_year=1984; " /><meta name="citation_reference" content="citation_title=Compositional pressure and translational selection determine codon usage in the extremely GC-poor unicellular eukaryote Entamoeba histolytica; Gene;  citation_year=2000;  citation_volume=242;  citation_pages=307-311; " /><meta name="citation_reference" content="citation_title=DoriC: a database of oriC regions in bacterial genomes; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=1866-1867; " /><meta name="citation_reference" content="citation_title=Gene expression analysis of six GC-rich Gram-negative phytopathogens; Biochem. Biophys. Res. Commun.;  citation_year=2005;  citation_volume=332;  citation_pages=380-387; " /><meta name="citation_reference" content="citation_title=Synonymous codon usage in Lactococcus lactis: mutational bias versus translational selection; J. Biomol. Struct. Dyn.;  citation_year=2004;  citation_volume=21;  citation_pages=527-536; " /><meta name="citation_reference" content="citation_title=Asymmetries generated by transcription-coupled repair in enterobacterial genes; Science;  citation_year=1996;  citation_volume=272;  citation_pages=107-109; " /><meta name="citation_reference" content="citation_title=Transcription-induced mutations: increase in C to T mutations in the nontranscribed strand during transcription in Escherichia coli; Proc. Natl Acad. Sci. USA;  citation_year=1996;  citation_volume=93;  citation_pages=13919-13924; " /><meta name="citation_reference" content="citation_title=Asymmetrical DNA replication promotes evolution: disparity theory of evolution; Genetica;  citation_year=1998;  citation_volume=102–103;  citation_pages=333-347; " /><meta name="citation_reference" content="citation_title=Consequences of replication fork movement through transcription units in vivo; Science;  citation_year=1992;  citation_volume=258;  citation_pages=1362-1365; " /><meta name="citation_reference" content="citation_title=Analyzing DNA strand compositional asymmetry to identify candidate replication origins of Borrelia burgdorferi linear and circular plasmids; Genome Res.;  citation_year=2000;  citation_volume=10;  citation_pages=1594-1604; " /><meta name="citation_reference" content="citation_title=Linear DNA of Borrelia species and antigenic variation; Trends Microbiol;  citation_year=1993;  citation_volume=1;  citation_pages=236-239; " /><meta name="citation_reference" content="citation_title=Horizontal gene transfer: evidence and possible consequences; Annu. Rev. Genet.;  citation_year=1994;  citation_volume=28;  citation_pages=237-261; " /><meta name="citation_reference" content="citation_title=The source of laterally transferred genes in bacterial genomes; Genome Biol.;  citation_year=2003;  citation_volume=4;  citation_pages=R57" /><meta name="citation_reference" content="citation_title=Amelioration of bacterial genomes: rates of change and exchange; J. Mol. Evol.;  citation_year=1997;  citation_volume=44;  citation_pages=383-397; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/16/2/91/17462640/dsp001.pdf" /><meta name="description" content="Abstract.  In this study, the factors driving genome-wide patterns of codon usages in Lawsonia intracellularis genome are determined. For genes on the chromosom" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/16/2/91/510591" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/16/2/91/510591" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/16/2/91/510591");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsp001';
    </script>
<title>
	Codon Usages of Genes on Chromosome, and Surprisingly, Genes in Plasmid are Primarily Affected by Strand-specific Mutational Biases in Lawsonia intracellularis | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./510591" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeJsPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHdW8sIEZlbmctQmlhbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNjaG9vbCBvZiBMaWZlIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIFVuaXZlcnNpdHkgb2YgRWxlY3Ryb25pYyBTY2llbmNlIGFuZCBUZWNobm9sb2d5IG9mIENoaW5hLCBDaGVuZ2R1IDYxMDA1NCwgUGVvcGxlJ3MgUmVwdWJsaWMgb2YgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZdWFuLCBKaWFuLUJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU2Nob29sIG9mIExpZmUgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSwgVW5pdmVyc2l0eSBvZiBFbGVjdHJvbmljIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3kgb2YgQ2hpbmEsIENoZW5nZHUgNjEwMDU0LCBQZW9wbGUncyBSZXB1YmxpYyBvZiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQ29kb24gVXNhZ2VzIG9mIEdlbmVzIG9uIENocm9tb3NvbWUsIGFuZCBTdXJwcmlzaW5nbHksIEdlbmVzIGluIFBsYXNtaWQgYXJlIFByaW1hcmlseSBBZmZlY3RlZCBieSBTdHJhbmQtc3BlY2lmaWMgTXV0YXRpb25hbCBCaWFzZXMgaW4gTGF3c29uaWEgaW50cmFjZWxsdWxhcmlzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iOTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjEwNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcDAwMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDA5LzA0LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gY2F0YWxvZyB1c2FnZSBhbmQgdGhlIGdlbm9tZSBoeXBvdGhlc2lzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5ODA7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPXI0OS1yNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIGNhdGFsb2cgdXNhZ2UgaXMgYSBnZW5vbWUgc3RyYXRlZ3kgbW9kdWxhdGVkIGZvciBnZW5lIGV4cHJlc3Npdml0eTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTgxOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1yNDMtcjc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiB1c2FnZSBpbiBiYWN0ZXJpYTogY29ycmVsYXRpb24gd2l0aCBnZW5lIGV4cHJlc3Npdml0eTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTgyOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NzA1NS03MDc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db3JyZWxhdGlvbiBiZXR3ZWVuIHRoZSBhYnVuZGFuY2Ugb2YgRXNjaGVyaWNoaWEgY29saSB0cmFuc2ZlciBSTkFzIGFuZCB0aGUgb2NjdXJyZW5jZSBvZiB0aGUgcmVzcGVjdGl2ZSBjb2RvbnMgaW4gaXRzIHByb3RlaW4gZ2VuZXM6IGEgcHJvcG9zYWwgZm9yIGEgc3lub255bW91cyBjb2RvbiBjaG9pY2UgdGhhdCBpcyBvcHRpbWFsIGZvciB0aGUgRS4gY29saSB0cmFuc2xhdGlvbmFsIHN5c3RlbTsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk4MTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTE7ICBjaXRhdGlvbl9wYWdlcz0zODktNDA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Dby12YXJpYXRpb24gb2YgdFJOQSBhYnVuZGFuY2UgYW5kIGNvZG9uIHVzYWdlIGluIEVzY2hlcmljaGlhIGNvbGkgYXQgZGlmZmVyZW50IGdyb3d0aCByYXRlczsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0yNjA7ICBjaXRhdGlvbl9wYWdlcz02NDktNjYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiBzZWxlY3Rpb24gaW4geWVhc3Q7IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0xOTgyOyAgY2l0YXRpb25fdm9sdW1lPTI1NzsgIGNpdGF0aW9uX3BhZ2VzPTMwMjYtMzAzMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgaW4gdGhlIE15Y29iYWN0ZXJpdW0gdHViZXJjdWxvc2lzIGNvbXBsZXg7IE1pY3JvYmlvbG9neTsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDI7ICBjaXRhdGlvbl9wYWdlcz05MTUtOTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BYnNlbmNlIG9mIHRyYW5zbGF0aW9uYWxseSBzZWxlY3RlZCBzeW5vbnltb3VzIGNvZG9uIHVzYWdlIGJpYXMgaW4gSGVsaWNvYmFjdGVyIHB5bG9yaTsgTWljcm9iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE0NjsgIGNpdGF0aW9uX3BhZ2VzPTg1MS04NjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0dWRpZXMgb2YgY29kb24gdXNhZ2UgYW5kIHRSTkEgZ2VuZXMgb2YgMTggdW5pY2VsbHVsYXIgb3JnYW5pc21zIGFuZCBxdWFudGlmaWNhdGlvbiBvZiBCYWNpbGx1cyBzdWJ0aWxpcyB0Uk5BczogZ2VuZSBleHByZXNzaW9uIGxldmVsIGFuZCBzcGVjaWVzLXNwZWNpZmljIGRpdmVyc2l0eSBvZiBjb2RvbiB1c2FnZSBiYXNlZCBvbiBtdWx0aXZhcmlhdGUgYW5hbHlzaXM7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MjM4OyAgY2l0YXRpb25fcGFnZXM9MTQzLTE1NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVwbGljYXRpb25hbCBhbmQgdHJhbnNjcmlwdGlvbmFsIHNlbGVjdGlvbiBvbiBjb2RvbiB1c2FnZSBpbiBCb3JyZWxpYSBidXJnZG9yZmVyaTsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT05NTsgIGNpdGF0aW9uX3BhZ2VzPTEwNjk4LTEwNzAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiB1c2FnZSBpbiBDaGxhbXlkaWEgdHJhY2hvbWF0aXMgaXMgdGhlIHJlc3VsdCBvZiBzdHJhbmQtc3BlY2lmaWMgbXV0YXRpb25hbCBiaWFzZXMgYW5kIGEgY29tcGxleCBwYXR0ZXJuIG9mIHNlbGVjdGl2ZSBmb3JjZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTIwODQtMjA5MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3lub255bW91cyBjb2RvbiB1c2FnZSBpbiBQc2V1ZG9tb25hcyBhZXJ1Z2lub3NhIFBBMDE7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9Mjg5OyAgY2l0YXRpb25fcGFnZXM9MTMxLTEzOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SG9yaXpvbnRhbCBnZW5lIHRyYW5zZmVyIGluIGJhY3RlcmlhbCBhbmQgYXJjaGFlYWwgY29tcGxldGUgZ2Vub21lczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xNzE5LTE3MjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhHVC1EQjogYSBkYXRhYmFzZSBvZiBwdXRhdGl2ZSBob3Jpem9udGFsbHkgdHJhbnNmZXJyZWQgZ2VuZXMgaW4gcHJva2FyeW90aWMgY29tcGxldGUgZ2Vub21lczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MTg3LTE4OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZpZGVuY2UgZm9yIGhvcml6b250YWwgZ2VuZSB0cmFuc2ZlciBpbiBFc2NoZXJpY2hpYSBjb2xpIHNwZWNpYXRpb247IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9MjIyOyAgY2l0YXRpb25fcGFnZXM9ODUxLTg1NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgYW5kIGxhdGVyYWwgZ2VuZSB0cmFuc2ZlciBpbiBCYWNpbGx1cyBzdWJ0aWxpczsgQ3Vyci4gT3Bpbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9NTI0LTUyODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJva2FyeW90aWMgZ2Vub21lIGV2b2x1dGlvbiBhcyBhc3Nlc3NlZCBieSBtdWx0aXZhcmlhdGUgYW5hbHlzaXMgb2YgY29kb24gdXNhZ2UgcGF0dGVybnM7IE1pY3JvYi4gQ29tcC4gR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTEtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhc2UgY29tcG9zaXRpb24gc2tld3MsIHJlcGxpY2F0aW9uIG9yaWVudGF0aW9uLCBhbmQgZ2VuZSBvcmllbnRhdGlvbiBpbiAxMiBwcm9rYXJ5b3RlIGdlbm9tZXM7IEouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz02OTEtNjk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3ltbWV0cmljIHN1YnN0aXR1dGlvbiBwYXR0ZXJuczogYSByZXZpZXcgb2YgcG9zc2libGUgdW5kZXJseWluZyBtdXRhdGlvbmFsIG9yIHNlbGVjdGl2ZSBtZWNoYW5pc21zOyBHZW5lOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTIzODsgIGNpdGF0aW9uX3BhZ2VzPTY1LTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJhbmQgY29tcG9zaXRpb25hbCBhc3ltbWV0cnkgaW4gYmFjdGVyaWFsIGFuZCBsYXJnZSB2aXJhbCBnZW5vbWVzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTk1OyAgY2l0YXRpb25fcGFnZXM9MzcyMC0zNzI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ib3cgZG9lcyByZXBsaWNhdGlvbi1hc3NvY2lhdGVkIG11dGF0aW9uYWwgcHJlc3N1cmUgaW5mbHVlbmNlIGFtaW5vIGFjaWQgY29tcG9zaXRpb24gb2YgcHJvdGVpbnM/OyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9NDA5LTQxNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXN5bW1ldHJpYyBkaXJlY3Rpb25hbCBtdXRhdGlvbiBwcmVzc3VyZXMgaW4gYmFjdGVyaWE7IGNpdGF0aW9uX3RleHQ9UkVTRUFSQ0gwMDU4OyBHZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXNzZW50aWFsaXR5LCBub3QgZXhwcmVzc2l2ZW5lc3MsIGRyaXZlcyBnZW5lLXN0cmFuZCBiaWFzIGluIGJhY3RlcmlhOyBOYXQgR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9Mzc3LTM3ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZXJydXB0aW9ucyBpbiBnZW5lIGV4cHJlc3Npb24gZHJpdmUgaGlnaGx5IGV4cHJlc3NlZCBvcGVyb25zIHRvIHRoZSBsZWFkaW5nIHN0cmFuZCBvZiBETkEgcmVwbGljYXRpb247IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTMyMjQtMzIzNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3Ryb25nIGFzeW1tZXRyaWMgbXV0YXRpb24gYmlhcyBpbiBlbmRvc3ltYmlvbnQgZ2Vub21lcyBjb2luY2lkZSB3aXRoIGxvc3Mgb2YgZ2VuZXMgZm9yIHJlcGxpY2F0aW9uIHJlc3RhcnQgcGF0aHdheXM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xMDMxLTEwMzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNpbWlsYXIgY29tcG9zaXRpb25hbCBiaWFzZXMgYXJlIGNhdXNlZCBieSB2ZXJ5IGRpZmZlcmVudCBtdXRhdGlvbmFsIGVmZmVjdHM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTUzNy0xNTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5ldyBtZXRob2QgZm9yIGFzc2Vzc2luZyB0aGUgZWZmZWN0IG9mIHJlcGxpY2F0aW9uIG9uIEROQSBiYXNlIGNvbXBvc2l0aW9uIGFzeW1tZXRyeTsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTIxNjktMjE3OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3JpZ2luIG9mIHJlcGxpY2F0aW9uIG9mIE15Y29wbGFzbWEgZ2VuaXRhbGl1bTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0yNzI7ICBjaXRhdGlvbl9wYWdlcz03NDUtNzQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3ltbWV0cmljIHN1YnN0aXR1dGlvbiBwYXR0ZXJucyBpbiB0aGUgdHdvIEROQSBzdHJhbmRzIG9mIGJhY3RlcmlhOyBNb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTEzOyAgY2l0YXRpb25fcGFnZXM9NjYwLTY2NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvdGVvbWUgY29tcG9zaXRpb24gYW5kIGNvZG9uIHVzYWdlIGluIHNwaXJvY2hhZXRlczogc3BlY2llcy1zcGVjaWZpYyBhbmQgRE5BIHN0cmFuZC1zcGVjaWZpYyBtdXRhdGlvbmFsIGJpYXNlczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MTY0Mi0xNjQ5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdXRhdGlvbmFsIGFuZCBzZWxlY3RpdmUgcHJlc3N1cmVzIG9uIGNvZG9uIGFuZCBhbWlubyBhY2lkIHVzYWdlIGluIEJ1Y2huZXJhLCBlbmRvc3ltYmlvdGljIGJhY3RlcmlhIG9mIGFwaGlkczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz00NC01MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uYXJ5IGZvcmNlcyBpbiBzaGFwaW5nIHRoZSBjb2RvbiBhbmQgYW1pbm8gYWNpZCB1c2FnZXMgaW4gQmxvY2htYW5uaWEgZmxvcmlkYW51czsgSi4gQmlvbW9sLiBTdHJ1Y3QuIER5bi47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0xMy0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gYW5kIGFtaW5vIGFjaWQgdXNhZ2UgaW4gdHdvIG1ham9yIGh1bWFuIHBhdGhvZ2VucyBvZiBnZW51cyBCYXJ0b25lbGxh4oCTb3B0aW1pemF0aW9uIGJldHdlZW4gcmVwbGljYXRpb25hbOKAk3RyYW5zY3JpcHRpb25hbCBzZWxlY3Rpb24sIHRyYW5zbGF0aW9uYWwgY29udHJvbCBhbmQgY29zdCBtaW5pbWl6YXRpb247IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9OTEtMTAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgY29uc3RyYWludHMgb24gY29kb24gYW5kIGFtaW5vIGFjaWQgdXNhZ2UgaW4gdHdvIHN0cmFpbnMgb2YgaHVtYW4gcGF0aG9nZW5pYyBhY3Rpbm9iYWN0ZXJpYSBUcm9waGVyeW1hIHdoaXBwbGVpOyBKLiBNb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTYyOyAgY2l0YXRpb25fcGFnZXM9NjQ1LTY1ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VwYXJhdGUgYmFzZSB1c2FnZXMgb2YgZ2VuZXMgbG9jYXRlZCBvbiB0aGUgbGVhZGluZyBhbmQgbGFnZ2luZyBzdHJhbmRzIGluIENobGFteWRpYSBtdXJpZGFydW0gcmV2ZWFsZWQgYnkgdGhlIFogY3VydmUgbWV0aG9kOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTM2NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtaWNyb2Jpb3RhIG9mIHBpZ3MgaW5mbHVlbmNlZCBieSBkaWV0IHRleHR1cmUgYW5kIHNldmVyaXR5IG9mIExhd3NvbmlhIGludHJhY2VsbHVsYXJpcyBpbmZlY3Rpb247IFZldCBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTEyODsgIGNpdGF0aW9uX3BhZ2VzPTk2LTEwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgY29kb24gdXNhZ2U7IGNpdGF0aW9uX3RleHQ9UGguRC4gVGhlc2lzOyBjaXRhdGlvbl9wdWJsaXNoZXI9VW5pdmVyc2l0eSBvZiBOb3R0aW5naGFtLCA7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSDigJhlZmZlY3RpdmUgbnVtYmVyIG9mIGNvZG9uc+KAmSB1c2VkIGluIGEgZ2VuZTsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT04NzsgIGNpdGF0aW9uX3BhZ2VzPTIzLTI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Vc2UgYW5kIG1pc3VzZSBvZiBjb3JyZXNwb25kZW5jZSBhbmFseXNpcyBpbiBjb2RvbiB1c2FnZSBzdHVkaWVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz00NTQ4LTQ1NTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1XaWxsZXkgUHJlc3MsIE5ldyBZb3JrLCBVU0E7IE11bHRpdmFyaWF0ZSBBbmFseXNpcywgTWV0aG9kIGFuZCBBcHBsaWNhdGlvbjsgIGNpdGF0aW9uX3llYXI9MTk4NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcG9zaXRpb25hbCBwcmVzc3VyZSBhbmQgdHJhbnNsYXRpb25hbCBzZWxlY3Rpb24gZGV0ZXJtaW5lIGNvZG9uIHVzYWdlIGluIHRoZSBleHRyZW1lbHkgR0MtcG9vciB1bmljZWxsdWxhciBldWthcnlvdGUgRW50YW1vZWJhIGhpc3RvbHl0aWNhOyBHZW5lOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTI0MjsgIGNpdGF0aW9uX3BhZ2VzPTMwNy0zMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURvcmlDOiBhIGRhdGFiYXNlIG9mIG9yaUMgcmVnaW9ucyBpbiBiYWN0ZXJpYWwgZ2Vub21lczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xODY2LTE4Njc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgZXhwcmVzc2lvbiBhbmFseXNpcyBvZiBzaXggR0MtcmljaCBHcmFtLW5lZ2F0aXZlIHBoeXRvcGF0aG9nZW5zOyBCaW9jaGVtLiBCaW9waHlzLiBSZXMuIENvbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MzMyOyAgY2l0YXRpb25fcGFnZXM9MzgwLTM4NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3lub255bW91cyBjb2RvbiB1c2FnZSBpbiBMYWN0b2NvY2N1cyBsYWN0aXM6IG11dGF0aW9uYWwgYmlhcyB2ZXJzdXMgdHJhbnNsYXRpb25hbCBzZWxlY3Rpb247IEouIEJpb21vbC4gU3RydWN0LiBEeW4uOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NTI3LTUzNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXN5bW1ldHJpZXMgZ2VuZXJhdGVkIGJ5IHRyYW5zY3JpcHRpb24tY291cGxlZCByZXBhaXIgaW4gZW50ZXJvYmFjdGVyaWFsIGdlbmVzOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTI3MjsgIGNpdGF0aW9uX3BhZ2VzPTEwNy0xMDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRpb24taW5kdWNlZCBtdXRhdGlvbnM6IGluY3JlYXNlIGluIEMgdG8gVCBtdXRhdGlvbnMgaW4gdGhlIG5vbnRyYW5zY3JpYmVkIHN0cmFuZCBkdXJpbmcgdHJhbnNjcmlwdGlvbiBpbiBFc2NoZXJpY2hpYSBjb2xpOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTkzOyAgY2l0YXRpb25fcGFnZXM9MTM5MTktMTM5MjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzeW1tZXRyaWNhbCBETkEgcmVwbGljYXRpb24gcHJvbW90ZXMgZXZvbHV0aW9uOiBkaXNwYXJpdHkgdGhlb3J5IG9mIGV2b2x1dGlvbjsgR2VuZXRpY2E7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTAy4oCTMTAzOyAgY2l0YXRpb25fcGFnZXM9MzMzLTM0NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc2VxdWVuY2VzIG9mIHJlcGxpY2F0aW9uIGZvcmsgbW92ZW1lbnQgdGhyb3VnaCB0cmFuc2NyaXB0aW9uIHVuaXRzIGluIHZpdm87IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MjU4OyAgY2l0YXRpb25fcGFnZXM9MTM2Mi0xMzY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXppbmcgRE5BIHN0cmFuZCBjb21wb3NpdGlvbmFsIGFzeW1tZXRyeSB0byBpZGVudGlmeSBjYW5kaWRhdGUgcmVwbGljYXRpb24gb3JpZ2lucyBvZiBCb3JyZWxpYSBidXJnZG9yZmVyaSBsaW5lYXIgYW5kIGNpcmN1bGFyIHBsYXNtaWRzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTE1OTQtMTYwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGluZWFyIEROQSBvZiBCb3JyZWxpYSBzcGVjaWVzIGFuZCBhbnRpZ2VuaWMgdmFyaWF0aW9uOyBUcmVuZHMgTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0yMzYtMjM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ib3Jpem9udGFsIGdlbmUgdHJhbnNmZXI6IGV2aWRlbmNlIGFuZCBwb3NzaWJsZSBjb25zZXF1ZW5jZXM7IEFubnUuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MjM3LTI2MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHNvdXJjZSBvZiBsYXRlcmFsbHkgdHJhbnNmZXJyZWQgZ2VuZXMgaW4gYmFjdGVyaWFsIGdlbm9tZXM7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9UjU3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW1lbGlvcmF0aW9uIG9mIGJhY3RlcmlhbCBnZW5vbWVzOiByYXRlcyBvZiBjaGFuZ2UgYW5kIGV4Y2hhbmdlOyBKLiBNb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9MzgzLTM5NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE2LzIvOTEvMTc0NjI2NDAvZHNwMDAxLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBJbiB0aGlzIHN0dWR5LCB0aGUgZmFjdG9ycyBkcml2aW5nIGdlbm9tZS13aWRlIHBhdHRlcm5zIG9mIGNvZG9uIHVzYWdlcyBpbiBMYXdzb25pYSBpbnRyYWNlbGx1bGFyaXMgZ2Vub21lIGFyZSBkZXRlcm1pbmVkLiBGb3IgZ2VuZXMgb24gdGhlIGNocm9tb3NvbSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NTEwNTkxJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZFGgMA2SN91VoXIJWm0oBxeg6dER" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f2%2f91%2f510591" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f2%2f91%2f510591" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=20204" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/16/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/2/1/m_cover.gif?Expires=2147483647&amp;Signature=spYD3Y1ECWwq5jfgvLwgtJGBnOHAN-nZXtWf5jX0Idx1cbrJRV4FKB28EY3pe17T0gsGMxo5dA8Tel6c2xNGAqa1byNOzqnWfnM8ohlcCfLHlrfGOEn2zyCW-15VbU7SOoLTo3OmdKcdJ19GrDbXCyAEcU~VJnaBCNGoOwBtbTnh6-S6KzBaPxjqj1HfbfKuNRe7H3r-0uFeA~Y3Ph79tw8QP6q4D0FiiXdjGGWYTNp3-Ga~m-PPAcOogXTotI1YFUthxQJgvIKyhC8JLbL3Znx728-CpY2KbMeFQlltwFxSw7d-mO5IOuvD7Wc91P7OLYCdnpnsyiP0k-TqunqvaA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/16/2">
                        <div class="volume trailing-comma">Volume 16</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2009        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87195348">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195348">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195350">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195350">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195357">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195357">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195365">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195365">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195398">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195398">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195417">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195417">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87195419">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87195419">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/16/2/81/510450">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/16/2/105/510716">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Codon Usages of Genes on Chromosome, and Surprisingly, Genes in Plasmid are Primarily Affected by Strand-specific Mutational Biases in <em>Lawsonia intracellularis</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Feng-Biao Guo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">School of Life Science and Technology</div>, <div class="insititution">University of Electronic Science and Technology of China</div>, <div class="addr-line">Chengdu 610054</div>, <div class="country">People's Republic of China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Fax. <div class="fax">+86 28-83201018</div>. E-mail: <a href="mailto:fbguo@uestc.edu.cn" target="_blank">fbguo@uestc.edu.cn</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Feng-Biao+Guo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Guo F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Guo F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Feng-Biao Guo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jian-Bo Yuan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">School of Life Science and Technology</div>, <div class="insititution">University of Electronic Science and Technology of China</div>, <div class="addr-line">Chengdu 610054</div>, <div class="country">People's Republic of China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jian-Bo+Yuan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yuan J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yuan J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jian-Bo Yuan</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 16, Issue 2, 1 April 2009, Pages 91–104, <a href='https://doi.org/10.1093/dnares/dsp001'>https://doi.org/10.1093/dnares/dsp001</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">15 February 2009</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">13 November 2008</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">08 January 2009</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="510591" href="/dnaresearch/article-pdf/16/2/91/17462640/dsp001.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Feng-Biao Guo, Jian-Bo Yuan;  Codon Usages of Genes on Chromosome, and Surprisingly, Genes in Plasmid are Primarily Affected by Strand-specific Mutational Biases in <em>Lawsonia intracellularis</em>, <em>DNA Research</em>, Volume 16, Issue 2, 1 April 2009, Pages 91–104, <a href='https://doi.org/10.1093/dnares/dsp001'>https://doi.org/10.1093/dnares/dsp001</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/510591?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/510591?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/510591?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/510591?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/510591?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/510591?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/510591?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Codon%20Usages%20of%20Genes%20on%20Chromosome%2C%20and%20Surprisingly%2C%20Genes%20in%20Plasmid%20are%20Primarily%20Affected%20by%20Strand-specific%20Mutational%20Biases%20in%20%26lt%3Bem%26gt%3BLawsonia%20intracellularis%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2009-02-15&amp;volumeNum=16&amp;issueNum=2&amp;author=Guo%2C%20Feng-Biao%3B%20Yuan%2C%20Jian-Bo&amp;startPage=91&amp;endPage=104&amp;contentId=10.1093%2Fdnares%2Fdsp001&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f2%2f91%2f510591" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87195348 id="87195348" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>In this study, the factors driving genome-wide patterns of codon usages in <em>Lawsonia intracellularis</em> genome are determined. For genes on the chromosome of the bacterium, it is found that the most important source of variation results from strand-specific mutational biases. A lesser trend of variation is attributable to genes that are presumed as horizontally transferred. These putative alien genes are unusually GC richer than the other genes, whereas horizontally transferred genes have been observed to be AT rich in bacteria with medium and relatively low G + C contents. Hydropathy of encoded protein and expression level are also found to influence codon usage. Therefore, codon usage in <em>L</em>. <em>intracellularis</em> chromosome is the result of a complex balance among the different mutational and selectional factors. When analyzing genes in the largest plasmid, for the first time it is found that the strand-specific mutational biases are responsible for the primary variation of codon usages in plasmid. Genes, particularly highly expressed genes of this plasmid, are mainly located on the leading strands and this supposed to be the effects exerted by replicational–transcriptional selection. These facts suggest that this plasmid adopts the similar mechanism of replication as the chromosome in <em>L</em>. <em>intracellularis</em>. Common characters among the 10 bacteria in whose genomes the strand-specific mutational biases are the primary source of variation of codon usage are also investigated. For example, it is found that genes <em>dnaT</em> and <em>fis</em> that are involved in DNA replication initiation and re-initiation pathways are absent in all of the 10 bacteria.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Lawsonia intracellularis&quot;"><em>Lawsonia intracellularis</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;codon usage&quot;">codon usage</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;strand-specific mutational bias&quot;">strand-specific mutational bias</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="plasmid">plasmid</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;replication mechanism&quot;">replication mechanism</a></div>                            <h2 scrollto-destination=87195350 id="87195350" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>When sequences of hundreds of microbial protein-coding genes became available in 1980, Grantham et al.<sup><a reveal-id="DSP001C1" data-open="DSP001C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSP001C2" data-open="DSP001C2" class="link link-ref link-reveal xref-bibr">2</a></sup> analyzed frequencies of 61 codons of all these genes. Consequently, they found that a surprising consistency of choices exists among genes of the same or similar genome. The ‘genome hypothesis’ was hereby proposed.<sup><a reveal-id="DSP001C1" data-open="DSP001C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSP001C2" data-open="DSP001C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Soon after that, it was shown that evident intra-genomic variability existed in many microorganisms.<sup><a reveal-id="DSP001C3" data-open="DSP001C3" class="link link-ref link-reveal xref-bibr">3</a></sup> This variation was interpreted as the effect of nature selection acting at the level of translation, which resulted in the preferential usage of optimal codons.<sup><a reveal-id="DSP001C4" data-open="DSP001C4" class="link link-ref link-reveal xref-bibr">4</a></sup> The interpretation was reinforced by the finding that the preferred codons in highly expressed genes were recognized by the most abundant tRNA in <em>Escherichia coli</em><sup><a reveal-id="DSP001C5" data-open="DSP001C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and as well as in <em>Saccharomyces cerevisiae</em>.<sup><a reveal-id="DSP001C6" data-open="DSP001C6" class="link link-ref link-reveal xref-bibr">6</a></sup> The selective advantage of optimal codons seems to lie in maximizing the efficiency of translation, particularly during periods of competitive exponential growth.<sup><a reveal-id="DSP001C7" data-open="DSP001C7" class="link link-ref link-reveal xref-bibr">7</a></sup> In bacteria with slow growth rate, selected codon usage bias may be relatively weak.<sup><a reveal-id="DSP001C7" data-open="DSP001C7" class="link link-ref link-reveal xref-bibr">7</a></sup> There may be no such bias in those bacteria for which the competitive growth is unimportant.<sup><a reveal-id="DSP001C8" data-open="DSP001C8" class="link link-ref link-reveal xref-bibr">8</a></sup> On the other hand, the preferred codons vary among species based on the changes in the complement of tRNAs in that bacterium.<sup><a reveal-id="DSP001C9" data-open="DSP001C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p><p>Besides translational selection, replicational and transcriptional selection may also have influence on the codon usage of a gene.<sup><a reveal-id="DSP001C10" data-open="DSP001C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSP001C11" data-open="DSP001C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Replicational selection is responsible for the higher number of genes on the leading strands, and transcriptional selection appears to be responsible for the enrichment of highly expressed genes on these strands.</p><p>The effects of mutation may be superimposed on biases generated by natural selection.<sup><a reveal-id="DSP001C12" data-open="DSP001C12" class="link link-ref link-reveal xref-bibr">12</a></sup> In most bacteria, there are short chromosome segments of unusual base composition due to the relatively recent import of the region through horizontal transfer.<sup><a reveal-id="DSP001C13" data-open="DSP001C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSP001C14" data-open="DSP001C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Genes located in these regions possess distinct codon usage or nucleotide composition from other genes, for example, in <em>E. coli</em><sup><a reveal-id="DSP001C15" data-open="DSP001C15" class="link link-ref link-reveal xref-bibr">15</a></sup> and in <em>Bacillus subtilis</em>.<sup><a reveal-id="DSP001C16" data-open="DSP001C16" class="link link-ref link-reveal xref-bibr">16</a></sup> In a single known example, <em>Mycoplasma genitalium</em>, codon usage variation is continuous and associated very strongly with position on the chromosome, perhaps reflecting change in the spectrum of mutations around the genome.<sup><a reveal-id="DSP001C17" data-open="DSP001C17" class="link link-ref link-reveal xref-bibr">17</a></sup> In addition, many bacteria exhibit skewed base composition between the leading and lagging strands of replication, although the magnitude of this skew varies considerably among species.<sup><a reveal-id="DSP001C18" data-open="DSP001C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p>Based on the above narration, the variation of codon usage of genes within a species is due to the combined effect of mutation and selection.<sup><a reveal-id="DSP001C12" data-open="DSP001C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Among these factors, the bias from asymmetric replication mechanism received special attention of researchers in the past 10 years.<sup><a reveal-id="DSP001C19" data-open="DSP001C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Many researches have been performed to analyze the effect on the codon usage (and/or amino acid composition) exerted by the asymmetric mutation and to investigate the underlying mechanism of the different mutation.<sup><a reveal-id="DSP001C20  DSP001C27" data-open="DSP001C20  DSP001C27" class="link link-ref link-reveal xref-bibr">20–27</a></sup> The skewed base composition between two replicating strand was first observed in <em>E. coli</em>, <em>M. genitalium</em>, <em>Haemophilus influenzae</em> and <em>B. subtilis</em>.<sup><a reveal-id="DSP001C28" data-open="DSP001C28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="DSP001C29" data-open="DSP001C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Then similar observations were obtained in most of the other bacteria. In 1998, for the first time it was found that the asymmetric replication is the major source of codon usage variation.<sup><a reveal-id="DSP001C10" data-open="DSP001C10" class="link link-ref link-reveal xref-bibr">10</a></sup> This observation was obtained in <em>Borrelia burgdorferi</em> genome.<sup><a reveal-id="DSP001C10" data-open="DSP001C10" class="link link-ref link-reveal xref-bibr">10</a></sup> The effect of asymmetry was so strong that the codon usages of genes on the two replicating strands were separated, distinct. After that, the separated codon usages between two replicating strands were also observed in <em>Treponema pallidum</em>,<sup><a reveal-id="DSP001C30" data-open="DSP001C30" class="link link-ref link-reveal xref-bibr">30</a></sup><em>Chlamydia trachomatis</em>,<sup><a reveal-id="DSP001C11" data-open="DSP001C11" class="link link-ref link-reveal xref-bibr">11</a></sup><em>Buchnera aphidicola</em>,<sup><a reveal-id="DSP001C31" data-open="DSP001C31" class="link link-ref link-reveal xref-bibr">31</a></sup><em>Blochmannia floridanus</em>,<sup><a reveal-id="DSP001C32" data-open="DSP001C32" class="link link-ref link-reveal xref-bibr">32</a></sup><em>Bartonella henselae</em>,<sup><a reveal-id="DSP001C33" data-open="DSP001C33" class="link link-ref link-reveal xref-bibr">33</a></sup><em>Bartonella quintana</em>,<sup><a reveal-id="DSP001C33" data-open="DSP001C33" class="link link-ref link-reveal xref-bibr">33</a></sup><em>Tropheryma whipplei</em><sup><a reveal-id="DSP001C34" data-open="DSP001C34" class="link link-ref link-reveal xref-bibr">34</a></sup> and <em>Chlamydia muridarum</em>.<sup><a reveal-id="DSP001C35" data-open="DSP001C35" class="link link-ref link-reveal xref-bibr">35</a></sup></p><p><em>Lawsonia intracellularis</em> is an obligate intracellular Gram-negative bacterial pathogen.<sup><a reveal-id="DSP001C36" data-open="DSP001C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Though primarily recognized in pigs, <em>L. intracellularis</em> is spreading to a wide range of mammals such as horses, and hamsters in North America and elsewhere. The bacterial pathogen invades the intestinal epithelial cells, which causes hyperplasia of the infected cells and leads to the process of disease pathogenesis. The disease has two clinical manifestations: an acute hemorrhagic form often referred as porcine hemorrhagic enteropathy, and a more chronic proliferative form often called porcine intestinal adenomatosis. Genome of <em>L</em>. <em>intracellularis</em> PHE/MN1-00 was determined in 2006, which provided a wonderful opportunity to extract a wealth of information on biochemistry, genetics, evolutionary history and pathogenicity of this organism.</p><p>Traditionally, codon usage data have been used in a wide variety of areas.<sup><a reveal-id="DSP001C10" data-open="DSP001C10" class="link link-ref link-reveal xref-bibr">10</a></sup> It is often desirable to use codon usage information to reduce the redundancy of primers for the PCR. Optimizing the codon usage of a gene could increase its expression level. Codon usage tables have been used to identify those ORFs that may encode proteins. Codon usage patterns also have been used to identify ORFs that probably do not code for functional proteins. Because of the importance of the intracellular pathogen and the potential usage of codon usage patterns, the intra-genomic variation in codon usage in <em>L</em>. <em>intracellularis</em> PHE/MN1-00 has been investigated through multivariate analysis method in this study.</p>                            <h2 scrollto-destination=87195357 id="87195357" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87195358 id="87195358" class="section-title" data-legacy-id=s2a>2.1. The database</h3>
<p>The complete genome sequence of <em>L</em>. <em>intracellularis</em> PHE/MN1-00 was downloaded from GenBank ftp site. One chromosome and three plasmids are contained in the complete genome. In this work, Plasmid 1 and Plasmid 2 are not taken into account because they contain too little genes to be analyzed statistically. Plasmid 3 is the largest plasmid and also analyzed. Chromosome has 1 457 619 bp and Plasmid 3 has 194 553 bp. A total of 1180 and 104 protein-coding genes are listed in the annotations of the chromosome and the largest plasmid, respectively. No attempt was made to alter the sequences or to remove those genes of unknown function. The FASTA formatted files, which are used as input files for codonW software, are proved as <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/dsp001_Supplementary_Data.zip?Expires=2147483647&Signature=ry0FoQMtRGuC91OnCopbPDJiqf6~bUMZD80nNelv5u5C38b4qjzjnHQb0X7SizcFnPCm-cyYysIEMdWD39v5SoqCsRNgPKW2vwVOPwsM5rMqGLApbW73unT~AQangV~hg4UO0pNwkLN15OZC5Cp5cAUp16aZHHKd9HZ9GSWwV48xDH7Vd6lAp6dmiMTKvszxHcpbizuTSKnaJQqGqHl~q48tSg0VnTQooo1zQo-~WUczIgB~6f7CzFJ0kMtQrc5ApOtNWC~yirwiiybbGHqZQWt13PIc7eLK1J02qtGuZzHq~q6KkaFoksAcYxlohgmaa5HNgFaN31vT8x8qsLHsqw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span>. In <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/dsp001_Supplementary_Data.zip?Expires=2147483647&Signature=ry0FoQMtRGuC91OnCopbPDJiqf6~bUMZD80nNelv5u5C38b4qjzjnHQb0X7SizcFnPCm-cyYysIEMdWD39v5SoqCsRNgPKW2vwVOPwsM5rMqGLApbW73unT~AQangV~hg4UO0pNwkLN15OZC5Cp5cAUp16aZHHKd9HZ9GSWwV48xDH7Vd6lAp6dmiMTKvszxHcpbizuTSKnaJQqGqHl~q48tSg0VnTQooo1zQo-~WUczIgB~6f7CzFJ0kMtQrc5ApOtNWC~yirwiiybbGHqZQWt13PIc7eLK1J02qtGuZzHq~q6KkaFoksAcYxlohgmaa5HNgFaN31vT8x8qsLHsqw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File 1</a></span>, the DNA sequences of 1180 genes located on the chromosome of <em>L</em>. <em>intracellularis</em> are contained. The first 607 genes correspond to those located on the leading strands and the last 573 ones correspond to those on the lagging strands. In <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/dsp001_Supplementary_Data.zip?Expires=2147483647&Signature=ry0FoQMtRGuC91OnCopbPDJiqf6~bUMZD80nNelv5u5C38b4qjzjnHQb0X7SizcFnPCm-cyYysIEMdWD39v5SoqCsRNgPKW2vwVOPwsM5rMqGLApbW73unT~AQangV~hg4UO0pNwkLN15OZC5Cp5cAUp16aZHHKd9HZ9GSWwV48xDH7Vd6lAp6dmiMTKvszxHcpbizuTSKnaJQqGqHl~q48tSg0VnTQooo1zQo-~WUczIgB~6f7CzFJ0kMtQrc5ApOtNWC~yirwiiybbGHqZQWt13PIc7eLK1J02qtGuZzHq~q6KkaFoksAcYxlohgmaa5HNgFaN31vT8x8qsLHsqw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary File 2</a></span>, the DNA sequences of 104 genes in the largest plasmid (Plasmid 3) of <em>L</em>. <em>intracellularis</em> are contained. The first 68 genes correspond to those located on the leading strands and the last 36 ones correspond to those on the lagging strands.</p>                            <h3 scrollto-destination=87195360 id="87195360" class="section-title" data-legacy-id=s2b>2.2. Statistical analysis</h3>
<p>Most analyses were carried out by using codonW,<sup><a reveal-id="DSP001C37" data-open="DSP001C37" class="link link-ref link-reveal xref-bibr">37</a></sup> which can be freely downloaded from the website (<a class="link link-uri" href="http://sourceforge.net/projects/codonw/">http://sourceforge.net/projects/codonw/</a>). GC3<sub>S</sub> denotes the frequencies of G and C at the third synonymously variable coding position (excluding Met, Trp and termination codons). <em>N</em><sub>C</sub> means the ‘effective number of codons’ used in a gene.<sup><a reveal-id="DSP001C38" data-open="DSP001C38" class="link link-ref link-reveal xref-bibr">38</a></sup> When all sense codons are used randomly, <em>N</em><sub>C</sub> takes a value of 61. Lower values of <em>N</em><sub>C</sub> indicate stronger bias, with an extreme value of 20 when only one synonymous codon is used for each amino acid. After calculating <em>N</em><sub>C</sub> and GC3<sub>S</sub> for each gene, <em>N</em><sub>C</sub>–GC3<sub>S</sub> plot can be made, which shows whether there are genes whose codon usage is affected by genome composition pressure and natural selection or mutation. An expected curve is plotted through the formula: <em>N</em> = 2 + <em>s</em> + {29/[<em>s</em><sup>2</sup> + (1 − <em>s</em>)<sup>2</sup>]}. For each gene, codon adaptive index (CAI) and hydropathy values (gravy) are also calculated by codonW.</p><p>Correspondence analysis (COA), as implemented in codonW, was used to determine the major source of variation of codon usage among the genes on the chromosome and the genes in Plasmid 3. As suggested by Perrière and Thioulouse,<sup><a reveal-id="DSP001C39" data-open="DSP001C39" class="link link-ref link-reveal xref-bibr">39</a></sup> parallel COA on codon counts and on relative codon frequencies were performed and then the results were compared. In addition, COA was carried out for genes on the chromosome, for genes in the largest plasmid and for the genes located on the leading strands of the chromosome, respectively. Relative synonymous codon usage (RSCU) is defined as the observed frequency of a codon divided by that expected when all codons for that amino acid are used equally. Therefore, RSCU values close to 1.0 indicate a lack of bias for that codon. Compared with simple measurements of codon abundance, RSCU values are normalized and are much more independent of amino acid usage. Only those codons for which there is a synonymous alternative were used in the analysis. Hence, the three termination codons and the codons that encode methionine and tryptophan are excluded. Consequently, each gene is described by a vector of 59 variables (codons). COA maps all the genes analyzed into the 59-dimensional space and attempts to identify a series of new orthogonal axes accounting for the greatest variation among genes. The first principal axis is chosen to maximize the standard deviation of the derived variable and the second principal axis is the direction to maximize the standard deviation among directions un-correlated with the first, and so forth. For details about this method, refer to Dillon and Goldstein.<sup><a reveal-id="DSP001C40" data-open="DSP001C40" class="link link-ref link-reveal xref-bibr">40</a></sup></p><p>GC skew [(G − C)/(G + C)] was used to determine the origin and termination of replication for the chromosome and the plasmid.<sup><a reveal-id="DSP001C28" data-open="DSP001C28" class="link link-ref link-reveal xref-bibr">28</a></sup> A non-overlapping sliding window of 1000 bp was employed for the GC skew. For the chromosome, the origin site is assumed to lie between genes LI0775 and LI0776, whereas the termination between genes LI0227 and LI0228. For the largest plasmid, the origin lies before gene LIC020, whereas the terminus between genes LIC062 and LIC063.</p><p>In order to check whether there have significant differences of codon usage between genes on the leading strands and those on the lagging strands, a <em>χ</em><sup>2</sup> test was employed. Significance was examined at the 5% level (<em>χ</em><sup>2</sup> value of 3.841). Significance was evaluated for the 59 sense codons for which there are a synonymous alternative.</p>                            <h2 scrollto-destination=87195365 id="87195365" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87195366 id="87195366" class="section-title" data-legacy-id=s3a>3.1. Global codon usage of genes on the chromosome</h3>
<p>It is widely accepted that global codon usage in unicellular species that displays extremely biased genomic composition is predominantly shaped by the compositional pressure.<sup><a reveal-id="DSP001C41" data-open="DSP001C41" class="link link-ref link-reveal xref-bibr">41</a></sup> This viewpoint is confirmed again in <em>L</em>. <em>intracellularis</em> genome. As can be seen from Table <a reveal-id="DSP001TB1" data-open="DSP001TB1" class="link link-reveal link-table xref-fig">1</a>, the global codon usage in 1180 genes on the chromosome in <em>L</em>. <em>intracellularis</em> shows the expected bias toward AT-rich codons. This enrichment is much stronger at the third codon position than at the first two positions. For all of the 18 amino acids (excepting Met and Trp), the frequencies of A- or T-ending codons are much more than those of G- or C-ending synonyms.</p>                        <a id="87195368" scrollto-destination="87195368"></a>
<div content-id="DSP001TB1" class="table-modal"><div class="table-wrap"><div id="DSP001TB1" class="table-wrap-title" data-id="DSP001TB1"><div class="label -label">Table 1</div><div class="caption"><p>Global codon usage of 1180 genes on the chromosome of <em>L. intracellularis</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>AA </th><th>Codon </th><th><em>N</em> </th><th>RSCU </th><th>AA </th><th>Codon </th><th><em>N</em> </th><th>RSCU </th></tr></thead><tbody><tr><td>Phe </td><td>UUU </td><td>15 112 </td><td>(1.72) </td><td>Ser </td><td>UCU </td><td>10 541 </td><td>(2.23) </td></tr><tr><td> </td><td>UUC </td><td>2411 </td><td>(0.28) </td><td> </td><td>UCC </td><td>1463 </td><td>(0.31) </td></tr><tr><td>Leu </td><td>UUA </td><td>15 959 </td><td>(2.19) </td><td> </td><td>UCA </td><td>7238 </td><td>(1.53) </td></tr><tr><td> </td><td>UUG </td><td>2922 </td><td>(0.40) </td><td> </td><td>UCG </td><td>525 </td><td>(0.11) </td></tr><tr><td>Tyr </td><td>UAU </td><td>10 979 </td><td>(1.67) </td><td>Cys </td><td>UGU </td><td>4395 </td><td>(1.68) </td></tr><tr><td> </td><td>UAC </td><td>2160 </td><td>(0.33) </td><td> </td><td>UGC </td><td>841 </td><td>(0.32) </td></tr><tr><td>Ter </td><td>UAA </td><td>777 </td><td>(0.00) </td><td>Ter </td><td>UGA </td><td>152 </td><td>(0.00) </td></tr><tr><td> </td><td>UAG </td><td>246 </td><td>(0.00) </td><td>Trp </td><td>UGG </td><td>4624 </td><td>(1.00) </td></tr><tr><td>Leu </td><td>CUU </td><td>16 136 </td><td>(2.21) </td><td>Pro </td><td>CCU </td><td>9194 </td><td>(2.07) </td></tr><tr><td> </td><td>CUC </td><td>2328 </td><td>(0.32) </td><td> </td><td>CCC </td><td>1075 </td><td>(0.24) </td></tr><tr><td> </td><td>CUA </td><td>5306 </td><td>(0.73) </td><td> </td><td>CCA </td><td>7189 </td><td>(1.62) </td></tr><tr><td> </td><td>CUG </td><td>1076 </td><td>(0.15) </td><td> </td><td>CCG </td><td>334 </td><td>(0.08) </td></tr><tr><td>His </td><td>CAU </td><td>7724 </td><td>(1.68) </td><td>Arg </td><td>CGU </td><td>6148 </td><td>(2.21) </td></tr><tr><td> </td><td>CAC </td><td>1494 </td><td>(0.32) </td><td> </td><td>CGC </td><td>1033 </td><td>(0.37) </td></tr><tr><td>Gln </td><td>CAA </td><td>13 060 </td><td>(1.58) </td><td> </td><td>CGA </td><td>2413 </td><td>(0.87) </td></tr><tr><td> </td><td>CAG </td><td>3462 </td><td>(0.42) </td><td> </td><td>CGG </td><td>637 </td><td>(0.23) </td></tr><tr><td>Ile </td><td>AUU </td><td>19 619 </td><td>(1.65) </td><td>Thr </td><td>ACU </td><td>8245 </td><td>(1.36) </td></tr><tr><td> </td><td>AUC </td><td>4019 </td><td>(0.34) </td><td> </td><td>ACC </td><td>1853 </td><td>(0.31) </td></tr><tr><td> </td><td>AUA </td><td>12 077 </td><td>(1.01) </td><td> </td><td>ACA </td><td>13 220 </td><td>(2.18) </td></tr><tr><td>Met </td><td>AUG </td><td>8968 </td><td>(1.00) </td><td> </td><td>ACG </td><td>904 </td><td>(0.15) </td></tr><tr><td>Asn </td><td>AAU </td><td>14 715 </td><td>(1.57) </td><td>Ser </td><td>AGU </td><td>6659 </td><td>(1.41) </td></tr><tr><td> </td><td>AAC </td><td>4058 </td><td>(0.43) </td><td> </td><td>AGC </td><td>1911 </td><td>(0.40) </td></tr><tr><td>Lys </td><td>AAA </td><td>20 477 </td><td>(1.62) </td><td>Arg </td><td>AGA </td><td>5064 </td><td>(1.82) </td></tr><tr><td> </td><td>AAG </td><td>4835 </td><td>(0.38) </td><td> </td><td>AGG </td><td>1384 </td><td>(0.50) </td></tr><tr><td>Val </td><td>GUU </td><td>11 365 </td><td>(1.88) </td><td>Ala </td><td>GCU </td><td>12 191 </td><td>(1.85) </td></tr><tr><td> </td><td>GUC </td><td>2237 </td><td>(0.37) </td><td> </td><td>GCC </td><td>2009 </td><td>(0.30) </td></tr><tr><td> </td><td>GUA </td><td>8764 </td><td>(1.45) </td><td> </td><td>GCA </td><td>11 581 </td><td>(1.75) </td></tr><tr><td> </td><td>GUG </td><td>1793 </td><td>(0.30) </td><td> </td><td>GCG </td><td>639 </td><td>(0.10) </td></tr><tr><td>Asp </td><td>GAU </td><td>15 082 </td><td>(1.67) </td><td>Gly </td><td>GGU </td><td>10 262 </td><td>(1.55) </td></tr><tr><td> </td><td>GAC </td><td>3017 </td><td>(0.33) </td><td> </td><td>GGC </td><td>2397 </td><td>(0.36) </td></tr><tr><td>Glu </td><td>GAA </td><td>19 734 </td><td>(1.57) </td><td> </td><td>GGA </td><td>10 629 </td><td>(1.60) </td></tr><tr><td> </td><td>GAG </td><td>5331 </td><td>(0.43) </td><td> </td><td>GGG </td><td>3217 </td><td>(0.49) </td></tr></tbody></table></div><div content-id="DSP001TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>AA </th><th>Codon </th><th><em>N</em> </th><th>RSCU </th><th>AA </th><th>Codon </th><th><em>N</em> </th><th>RSCU </th></tr></thead><tbody><tr><td>Phe </td><td>UUU </td><td>15 112 </td><td>(1.72) </td><td>Ser </td><td>UCU </td><td>10 541 </td><td>(2.23) </td></tr><tr><td> </td><td>UUC </td><td>2411 </td><td>(0.28) </td><td> </td><td>UCC </td><td>1463 </td><td>(0.31) </td></tr><tr><td>Leu </td><td>UUA </td><td>15 959 </td><td>(2.19) </td><td> </td><td>UCA </td><td>7238 </td><td>(1.53) </td></tr><tr><td> </td><td>UUG </td><td>2922 </td><td>(0.40) </td><td> </td><td>UCG </td><td>525 </td><td>(0.11) </td></tr><tr><td>Tyr </td><td>UAU </td><td>10 979 </td><td>(1.67) </td><td>Cys </td><td>UGU </td><td>4395 </td><td>(1.68) </td></tr><tr><td> </td><td>UAC </td><td>2160 </td><td>(0.33) </td><td> </td><td>UGC </td><td>841 </td><td>(0.32) </td></tr><tr><td>Ter </td><td>UAA </td><td>777 </td><td>(0.00) </td><td>Ter </td><td>UGA </td><td>152 </td><td>(0.00) </td></tr><tr><td> </td><td>UAG </td><td>246 </td><td>(0.00) </td><td>Trp </td><td>UGG </td><td>4624 </td><td>(1.00) </td></tr><tr><td>Leu </td><td>CUU </td><td>16 136 </td><td>(2.21) </td><td>Pro </td><td>CCU </td><td>9194 </td><td>(2.07) </td></tr><tr><td> </td><td>CUC </td><td>2328 </td><td>(0.32) </td><td> </td><td>CCC </td><td>1075 </td><td>(0.24) </td></tr><tr><td> </td><td>CUA </td><td>5306 </td><td>(0.73) </td><td> </td><td>CCA </td><td>7189 </td><td>(1.62) </td></tr><tr><td> </td><td>CUG </td><td>1076 </td><td>(0.15) </td><td> </td><td>CCG </td><td>334 </td><td>(0.08) </td></tr><tr><td>His </td><td>CAU </td><td>7724 </td><td>(1.68) </td><td>Arg </td><td>CGU </td><td>6148 </td><td>(2.21) </td></tr><tr><td> </td><td>CAC </td><td>1494 </td><td>(0.32) </td><td> </td><td>CGC </td><td>1033 </td><td>(0.37) </td></tr><tr><td>Gln </td><td>CAA </td><td>13 060 </td><td>(1.58) </td><td> </td><td>CGA </td><td>2413 </td><td>(0.87) </td></tr><tr><td> </td><td>CAG </td><td>3462 </td><td>(0.42) </td><td> </td><td>CGG </td><td>637 </td><td>(0.23) </td></tr><tr><td>Ile </td><td>AUU </td><td>19 619 </td><td>(1.65) </td><td>Thr </td><td>ACU </td><td>8245 </td><td>(1.36) </td></tr><tr><td> </td><td>AUC </td><td>4019 </td><td>(0.34) </td><td> </td><td>ACC </td><td>1853 </td><td>(0.31) </td></tr><tr><td> </td><td>AUA </td><td>12 077 </td><td>(1.01) </td><td> </td><td>ACA </td><td>13 220 </td><td>(2.18) </td></tr><tr><td>Met </td><td>AUG </td><td>8968 </td><td>(1.00) </td><td> </td><td>ACG </td><td>904 </td><td>(0.15) </td></tr><tr><td>Asn </td><td>AAU </td><td>14 715 </td><td>(1.57) </td><td>Ser </td><td>AGU </td><td>6659 </td><td>(1.41) </td></tr><tr><td> </td><td>AAC </td><td>4058 </td><td>(0.43) </td><td> </td><td>AGC </td><td>1911 </td><td>(0.40) </td></tr><tr><td>Lys </td><td>AAA </td><td>20 477 </td><td>(1.62) </td><td>Arg </td><td>AGA </td><td>5064 </td><td>(1.82) </td></tr><tr><td> </td><td>AAG </td><td>4835 </td><td>(0.38) </td><td> </td><td>AGG </td><td>1384 </td><td>(0.50) </td></tr><tr><td>Val </td><td>GUU </td><td>11 365 </td><td>(1.88) </td><td>Ala </td><td>GCU </td><td>12 191 </td><td>(1.85) </td></tr><tr><td> </td><td>GUC </td><td>2237 </td><td>(0.37) </td><td> </td><td>GCC </td><td>2009 </td><td>(0.30) </td></tr><tr><td> </td><td>GUA </td><td>8764 </td><td>(1.45) </td><td> </td><td>GCA </td><td>11 581 </td><td>(1.75) </td></tr><tr><td> </td><td>GUG </td><td>1793 </td><td>(0.30) </td><td> </td><td>GCG </td><td>639 </td><td>(0.10) </td></tr><tr><td>Asp </td><td>GAU </td><td>15 082 </td><td>(1.67) </td><td>Gly </td><td>GGU </td><td>10 262 </td><td>(1.55) </td></tr><tr><td> </td><td>GAC </td><td>3017 </td><td>(0.33) </td><td> </td><td>GGC </td><td>2397 </td><td>(0.36) </td></tr><tr><td>Glu </td><td>GAA </td><td>19 734 </td><td>(1.57) </td><td> </td><td>GGA </td><td>10 629 </td><td>(1.60) </td></tr><tr><td> </td><td>GAG </td><td>5331 </td><td>(0.43) </td><td> </td><td>GGG </td><td>3217 </td><td>(0.49) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195368" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP001TB1" class="table-wrap-title" data-id="DSP001TB1"><div class="label -label">Table 1</div><div class="caption"><p>Global codon usage of 1180 genes on the chromosome of <em>L. intracellularis</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>AA </th><th>Codon </th><th><em>N</em> </th><th>RSCU </th><th>AA </th><th>Codon </th><th><em>N</em> </th><th>RSCU </th></tr></thead><tbody><tr><td>Phe </td><td>UUU </td><td>15 112 </td><td>(1.72) </td><td>Ser </td><td>UCU </td><td>10 541 </td><td>(2.23) </td></tr><tr><td> </td><td>UUC </td><td>2411 </td><td>(0.28) </td><td> </td><td>UCC </td><td>1463 </td><td>(0.31) </td></tr><tr><td>Leu </td><td>UUA </td><td>15 959 </td><td>(2.19) </td><td> </td><td>UCA </td><td>7238 </td><td>(1.53) </td></tr><tr><td> </td><td>UUG </td><td>2922 </td><td>(0.40) </td><td> </td><td>UCG </td><td>525 </td><td>(0.11) </td></tr><tr><td>Tyr </td><td>UAU </td><td>10 979 </td><td>(1.67) </td><td>Cys </td><td>UGU </td><td>4395 </td><td>(1.68) </td></tr><tr><td> </td><td>UAC </td><td>2160 </td><td>(0.33) </td><td> </td><td>UGC </td><td>841 </td><td>(0.32) </td></tr><tr><td>Ter </td><td>UAA </td><td>777 </td><td>(0.00) </td><td>Ter </td><td>UGA </td><td>152 </td><td>(0.00) </td></tr><tr><td> </td><td>UAG </td><td>246 </td><td>(0.00) </td><td>Trp </td><td>UGG </td><td>4624 </td><td>(1.00) </td></tr><tr><td>Leu </td><td>CUU </td><td>16 136 </td><td>(2.21) </td><td>Pro </td><td>CCU </td><td>9194 </td><td>(2.07) </td></tr><tr><td> </td><td>CUC </td><td>2328 </td><td>(0.32) </td><td> </td><td>CCC </td><td>1075 </td><td>(0.24) </td></tr><tr><td> </td><td>CUA </td><td>5306 </td><td>(0.73) </td><td> </td><td>CCA </td><td>7189 </td><td>(1.62) </td></tr><tr><td> </td><td>CUG </td><td>1076 </td><td>(0.15) </td><td> </td><td>CCG </td><td>334 </td><td>(0.08) </td></tr><tr><td>His </td><td>CAU </td><td>7724 </td><td>(1.68) </td><td>Arg </td><td>CGU </td><td>6148 </td><td>(2.21) </td></tr><tr><td> </td><td>CAC </td><td>1494 </td><td>(0.32) </td><td> </td><td>CGC </td><td>1033 </td><td>(0.37) </td></tr><tr><td>Gln </td><td>CAA </td><td>13 060 </td><td>(1.58) </td><td> </td><td>CGA </td><td>2413 </td><td>(0.87) </td></tr><tr><td> </td><td>CAG </td><td>3462 </td><td>(0.42) </td><td> </td><td>CGG </td><td>637 </td><td>(0.23) </td></tr><tr><td>Ile </td><td>AUU </td><td>19 619 </td><td>(1.65) </td><td>Thr </td><td>ACU </td><td>8245 </td><td>(1.36) </td></tr><tr><td> </td><td>AUC </td><td>4019 </td><td>(0.34) </td><td> </td><td>ACC </td><td>1853 </td><td>(0.31) </td></tr><tr><td> </td><td>AUA </td><td>12 077 </td><td>(1.01) </td><td> </td><td>ACA </td><td>13 220 </td><td>(2.18) </td></tr><tr><td>Met </td><td>AUG </td><td>8968 </td><td>(1.00) </td><td> </td><td>ACG </td><td>904 </td><td>(0.15) </td></tr><tr><td>Asn </td><td>AAU </td><td>14 715 </td><td>(1.57) </td><td>Ser </td><td>AGU </td><td>6659 </td><td>(1.41) </td></tr><tr><td> </td><td>AAC </td><td>4058 </td><td>(0.43) </td><td> </td><td>AGC </td><td>1911 </td><td>(0.40) </td></tr><tr><td>Lys </td><td>AAA </td><td>20 477 </td><td>(1.62) </td><td>Arg </td><td>AGA </td><td>5064 </td><td>(1.82) </td></tr><tr><td> </td><td>AAG </td><td>4835 </td><td>(0.38) </td><td> </td><td>AGG </td><td>1384 </td><td>(0.50) </td></tr><tr><td>Val </td><td>GUU </td><td>11 365 </td><td>(1.88) </td><td>Ala </td><td>GCU </td><td>12 191 </td><td>(1.85) </td></tr><tr><td> </td><td>GUC </td><td>2237 </td><td>(0.37) </td><td> </td><td>GCC </td><td>2009 </td><td>(0.30) </td></tr><tr><td> </td><td>GUA </td><td>8764 </td><td>(1.45) </td><td> </td><td>GCA </td><td>11 581 </td><td>(1.75) </td></tr><tr><td> </td><td>GUG </td><td>1793 </td><td>(0.30) </td><td> </td><td>GCG </td><td>639 </td><td>(0.10) </td></tr><tr><td>Asp </td><td>GAU </td><td>15 082 </td><td>(1.67) </td><td>Gly </td><td>GGU </td><td>10 262 </td><td>(1.55) </td></tr><tr><td> </td><td>GAC </td><td>3017 </td><td>(0.33) </td><td> </td><td>GGC </td><td>2397 </td><td>(0.36) </td></tr><tr><td>Glu </td><td>GAA </td><td>19 734 </td><td>(1.57) </td><td> </td><td>GGA </td><td>10 629 </td><td>(1.60) </td></tr><tr><td> </td><td>GAG </td><td>5331 </td><td>(0.43) </td><td> </td><td>GGG </td><td>3217 </td><td>(0.49) </td></tr></tbody></table></div><div content-id="DSP001TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>AA </th><th>Codon </th><th><em>N</em> </th><th>RSCU </th><th>AA </th><th>Codon </th><th><em>N</em> </th><th>RSCU </th></tr></thead><tbody><tr><td>Phe </td><td>UUU </td><td>15 112 </td><td>(1.72) </td><td>Ser </td><td>UCU </td><td>10 541 </td><td>(2.23) </td></tr><tr><td> </td><td>UUC </td><td>2411 </td><td>(0.28) </td><td> </td><td>UCC </td><td>1463 </td><td>(0.31) </td></tr><tr><td>Leu </td><td>UUA </td><td>15 959 </td><td>(2.19) </td><td> </td><td>UCA </td><td>7238 </td><td>(1.53) </td></tr><tr><td> </td><td>UUG </td><td>2922 </td><td>(0.40) </td><td> </td><td>UCG </td><td>525 </td><td>(0.11) </td></tr><tr><td>Tyr </td><td>UAU </td><td>10 979 </td><td>(1.67) </td><td>Cys </td><td>UGU </td><td>4395 </td><td>(1.68) </td></tr><tr><td> </td><td>UAC </td><td>2160 </td><td>(0.33) </td><td> </td><td>UGC </td><td>841 </td><td>(0.32) </td></tr><tr><td>Ter </td><td>UAA </td><td>777 </td><td>(0.00) </td><td>Ter </td><td>UGA </td><td>152 </td><td>(0.00) </td></tr><tr><td> </td><td>UAG </td><td>246 </td><td>(0.00) </td><td>Trp </td><td>UGG </td><td>4624 </td><td>(1.00) </td></tr><tr><td>Leu </td><td>CUU </td><td>16 136 </td><td>(2.21) </td><td>Pro </td><td>CCU </td><td>9194 </td><td>(2.07) </td></tr><tr><td> </td><td>CUC </td><td>2328 </td><td>(0.32) </td><td> </td><td>CCC </td><td>1075 </td><td>(0.24) </td></tr><tr><td> </td><td>CUA </td><td>5306 </td><td>(0.73) </td><td> </td><td>CCA </td><td>7189 </td><td>(1.62) </td></tr><tr><td> </td><td>CUG </td><td>1076 </td><td>(0.15) </td><td> </td><td>CCG </td><td>334 </td><td>(0.08) </td></tr><tr><td>His </td><td>CAU </td><td>7724 </td><td>(1.68) </td><td>Arg </td><td>CGU </td><td>6148 </td><td>(2.21) </td></tr><tr><td> </td><td>CAC </td><td>1494 </td><td>(0.32) </td><td> </td><td>CGC </td><td>1033 </td><td>(0.37) </td></tr><tr><td>Gln </td><td>CAA </td><td>13 060 </td><td>(1.58) </td><td> </td><td>CGA </td><td>2413 </td><td>(0.87) </td></tr><tr><td> </td><td>CAG </td><td>3462 </td><td>(0.42) </td><td> </td><td>CGG </td><td>637 </td><td>(0.23) </td></tr><tr><td>Ile </td><td>AUU </td><td>19 619 </td><td>(1.65) </td><td>Thr </td><td>ACU </td><td>8245 </td><td>(1.36) </td></tr><tr><td> </td><td>AUC </td><td>4019 </td><td>(0.34) </td><td> </td><td>ACC </td><td>1853 </td><td>(0.31) </td></tr><tr><td> </td><td>AUA </td><td>12 077 </td><td>(1.01) </td><td> </td><td>ACA </td><td>13 220 </td><td>(2.18) </td></tr><tr><td>Met </td><td>AUG </td><td>8968 </td><td>(1.00) </td><td> </td><td>ACG </td><td>904 </td><td>(0.15) </td></tr><tr><td>Asn </td><td>AAU </td><td>14 715 </td><td>(1.57) </td><td>Ser </td><td>AGU </td><td>6659 </td><td>(1.41) </td></tr><tr><td> </td><td>AAC </td><td>4058 </td><td>(0.43) </td><td> </td><td>AGC </td><td>1911 </td><td>(0.40) </td></tr><tr><td>Lys </td><td>AAA </td><td>20 477 </td><td>(1.62) </td><td>Arg </td><td>AGA </td><td>5064 </td><td>(1.82) </td></tr><tr><td> </td><td>AAG </td><td>4835 </td><td>(0.38) </td><td> </td><td>AGG </td><td>1384 </td><td>(0.50) </td></tr><tr><td>Val </td><td>GUU </td><td>11 365 </td><td>(1.88) </td><td>Ala </td><td>GCU </td><td>12 191 </td><td>(1.85) </td></tr><tr><td> </td><td>GUC </td><td>2237 </td><td>(0.37) </td><td> </td><td>GCC </td><td>2009 </td><td>(0.30) </td></tr><tr><td> </td><td>GUA </td><td>8764 </td><td>(1.45) </td><td> </td><td>GCA </td><td>11 581 </td><td>(1.75) </td></tr><tr><td> </td><td>GUG </td><td>1793 </td><td>(0.30) </td><td> </td><td>GCG </td><td>639 </td><td>(0.10) </td></tr><tr><td>Asp </td><td>GAU </td><td>15 082 </td><td>(1.67) </td><td>Gly </td><td>GGU </td><td>10 262 </td><td>(1.55) </td></tr><tr><td> </td><td>GAC </td><td>3017 </td><td>(0.33) </td><td> </td><td>GGC </td><td>2397 </td><td>(0.36) </td></tr><tr><td>Glu </td><td>GAA </td><td>19 734 </td><td>(1.57) </td><td> </td><td>GGA </td><td>10 629 </td><td>(1.60) </td></tr><tr><td> </td><td>GAG </td><td>5331 </td><td>(0.43) </td><td> </td><td>GGG </td><td>3217 </td><td>(0.49) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195368" target="_blank">View Large</a></div></div><p>As suggested by Wright,<sup><a reveal-id="DSP001C38" data-open="DSP001C38" class="link link-ref link-reveal xref-bibr">38</a></sup> a plot of <em>N</em><sub>C</sub> against GC3<sub>S</sub> can gives a useful visual display of the main features of codon usage patterns for a number of genes. If a gene is only subject to G + C-biased mutational pressure, it will lie on the GC3<sub>S</sub> curve. It will lie just below the GC3<sub>S</sub> curve if a gene is under selection (either negative or positive) for codons in C and/or G. In other words, the gene will lie over the GC3<sub>S</sub> curve if it is subject to other kinds of selection and/or any kinds of mutation pressure. Such a plot for genes on the chromosome is shown in Fig. <a reveal-id="DSP001F1" data-open="DSP001F1" class="link link-reveal link-table xref-fig">1</a>. According to the figure, there exists significant heterogeneity of codon usage among genes on the chromosome: GC3<sub>S</sub> values range from 0.07 to 0.61, whereas <em>N</em><sub>C</sub> values range from 31.1 to 60.0. It can also be seen that one-third of genes lie above the GC3<sub>S</sub> curve, and the retaining two-thirds lie below the GC3<sub>S</sub> curve. For genes that lie above the GC3<sub>S</sub> curve, there may exist mutational or selectional pressure that leads to A- and/or T-ending codons. On inspection, almost all of these genes are located on the lagging strands and most of these genes are those likely to be lowly expressed. It should also be noted that dozens of genes are located far from the majority. These genes have high GC3<sub>S</sub> values and high <em>N</em><sub>C</sub> values, which are marked by open triangles in the plot. Origin of these genes will be discussed in the later section.</p>                        <a id="87195370" scrollto-destination="87195370"></a>
<div id="DSP001F1" data-id="DSP001F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/m_dsp00101.jpeg?Expires=2147483647&Signature=J4c5ydCc4QvAsRoLdw2e4hbwVpg-D8uifrONQUoImYfeucDfDRZGeTCOkZwxbNHcxO9Pxzz0GzfnZtP1zN62iOlkk6QHJTQMjm3KW8dAPdTA9x7n2v1u3822EkbzD6~1dEaY5iZMlOdJzuECA-4qHPH60yfi6frFgMRViLEAjzgmzuh7GnMXUjbxHF~TsOMAwkiXUlkDu0TzClxdg~zSgbMCy5uMvrwT297QfKYJY-LyXSNis5ZVCX2lvBlR5hJd~vGQWcoxjcMI~ZFJQmotxsnAjgy6YSVF-kV69Be9vE3G1zanYSBsGQ2jqx9k1F8IdSFTpbMj3MPrCWXHgUBiew__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The effective number of codons (NC) plotted against the G + C content at the synonymously variable third position (GC3S), for the 1180 genes on the chromosome. Open triangles indicate genes with unusual G + C content, speculated as foreign genes. Open squares denote the other genes." path-from-xml="dsp00101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195370/dsp00101.jpeg" path-from-xml="dsp00101.jpeg" target="_blank">View large</a><a section="87195370" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/dsp00101.jpeg?Expires=2147483647&Signature=QJcvr958sgmynz3X5HuFpfoFb8KnLUbGz7Opr~gT9bSLWJmjOGmnNyK5H48wCvBfEiQ4s~VE6MfEAmkCuQ3JiuaHruFDmhvYhAWOI6wgouGjwn~d54u~kEVCRttNO6g-uwgUSHmfeKhrHdcoxQEmLv~AtcF7paV4B1yW1gb4b8dAo6VDL8AbjhqNWoxXh9LODebPEi3z8HV4J4wvccZiUbYsVoTiVrDOjPa4Xi4cw4XXeO6zNqjfPCZv0UuO6VmfJ8XpEO-vPMcLnHPC0cqGRXLVbnPi2ZHugn6rFP-DRviR9mKnhzpi-3B5V4sOnU9O-VXjgSlJOv-NxS-MQW5VaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195370&ar=510591&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The effective number of codons (<em>N</em><sub>C</sub>) plotted against the G + C content at the synonymously variable third position (GC3<sub>S</sub>), for the 1180 genes on the chromosome. Open triangles indicate genes with unusual G + C content, speculated as foreign genes. Open squares denote the other genes.</p></div></div><div content-id="DSP001F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/m_dsp00101.jpeg?Expires=2147483647&Signature=J4c5ydCc4QvAsRoLdw2e4hbwVpg-D8uifrONQUoImYfeucDfDRZGeTCOkZwxbNHcxO9Pxzz0GzfnZtP1zN62iOlkk6QHJTQMjm3KW8dAPdTA9x7n2v1u3822EkbzD6~1dEaY5iZMlOdJzuECA-4qHPH60yfi6frFgMRViLEAjzgmzuh7GnMXUjbxHF~TsOMAwkiXUlkDu0TzClxdg~zSgbMCy5uMvrwT297QfKYJY-LyXSNis5ZVCX2lvBlR5hJd~vGQWcoxjcMI~ZFJQmotxsnAjgy6YSVF-kV69Be9vE3G1zanYSBsGQ2jqx9k1F8IdSFTpbMj3MPrCWXHgUBiew__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The effective number of codons (NC) plotted against the G + C content at the synonymously variable third position (GC3S), for the 1180 genes on the chromosome. Open triangles indicate genes with unusual G + C content, speculated as foreign genes. Open squares denote the other genes." path-from-xml="dsp00101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195370/dsp00101.jpeg" path-from-xml="dsp00101.jpeg" target="_blank">View large</a><a section="87195370" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/dsp00101.jpeg?Expires=2147483647&Signature=QJcvr958sgmynz3X5HuFpfoFb8KnLUbGz7Opr~gT9bSLWJmjOGmnNyK5H48wCvBfEiQ4s~VE6MfEAmkCuQ3JiuaHruFDmhvYhAWOI6wgouGjwn~d54u~kEVCRttNO6g-uwgUSHmfeKhrHdcoxQEmLv~AtcF7paV4B1yW1gb4b8dAo6VDL8AbjhqNWoxXh9LODebPEi3z8HV4J4wvccZiUbYsVoTiVrDOjPa4Xi4cw4XXeO6zNqjfPCZv0UuO6VmfJ8XpEO-vPMcLnHPC0cqGRXLVbnPi2ZHugn6rFP-DRviR9mKnhzpi-3B5V4sOnU9O-VXjgSlJOv-NxS-MQW5VaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195370&ar=510591&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The effective number of codons (<em>N</em><sub>C</sub>) plotted against the G + C content at the synonymously variable third position (GC3<sub>S</sub>), for the 1180 genes on the chromosome. Open triangles indicate genes with unusual G + C content, speculated as foreign genes. Open squares denote the other genes.</p></div></div>                            <h3 scrollto-destination=87195371 id="87195371" class="section-title" data-legacy-id=s3b>3.2. Strand-specific composition bias at three codon positions of genes on the chromosome</h3>
<p>GC-skew analysis shows a clear polarity switch at two points, around 284 and 975 kb on the chromosome in <em>L</em>. <em>intracellularis</em>, suggesting that the putative replication terminus and origin sites might be located in these regions. According to the record for this bacterium in the Doric database,<sup><a reveal-id="DSP001C42" data-open="DSP001C42" class="link link-ref link-reveal xref-bibr">42</a></sup> GC disparity that is a component of Z curve also shows a clear minimum and maximum at these points for this genome. Comparisons with the consensus sequence for the non-perfect DnaA box motif (ttttcaaca) reveal that a non-translatable region between 983 356 and 984 329 bp possesses cluster of three putative DnaA boxes, thereby confirming the possible locations of the functional chromosomal origins between two genes, LI0775 (trkH) and LI0776 (psd). The existence of a DnaA gene around 980 kb furthermore confirms this location as replication origin.</p><p>Table <a reveal-id="DSP001TB2" data-open="DSP001TB2" class="link link-reveal link-table xref-fig">2</a> shows the frequencies of nucleotide A, C, G, T and the mean G + T contents at three codon positions of the genes located on the leading and lagging strands of the <em>L</em>. <em>intracellularis</em> chromosome. Strand-specific skews, known to influence codon usage in other bacteria,<sup><a reveal-id="DSP001C18" data-open="DSP001C18" class="link link-ref link-reveal xref-bibr">18</a></sup> are also found to have the same influences on <em>L</em>. <em>intracellularis</em>. The leading strand genes show an excess of G over C and T over A, whereas the case for the lagging strands is opposite. The <em>t</em>-test shows that the mean G + T content of each codon position of the genes located on the leading strands is significantly different from that on the lagging strands of replication. This indicates that the strand-specific compositional bias has significant influence on nucleotide selection not only at the third codon position but also at the first and second positions. The inter-strand variation in G + T content is highest in the third and lowest in the second codon position, whereas the intra-strand variation is highest in the second codon position, which is reflected by the highest standard deviations.</p>                        <a id="87195374" scrollto-destination="87195374"></a>
<div content-id="DSP001TB2" class="table-modal"><div class="table-wrap"><div id="DSP001TB2" class="table-wrap-title" data-id="DSP001TB2"><div class="label -label">Table 2</div><div class="caption"><p>Base compositions and medium G + T content at three codon positions for genes located on the leading and lagging strands of the chromosome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>A </th><th>C </th><th>G </th><th>T </th><th>G + T </th></tr><tr><th> </th><th> </th><th>Mean (SD) </th><th>Mean (SD) </th><th>Mean (SD) </th><th>Mean (SD) </th><th>Mean (SD) </th></tr></thead><tbody><tr><td rowspan="2">First codon position </td><td>Leading </td><td>0.304 (0.036) </td><td>0.177 (0.031) </td><td>0.309 (0.047) </td><td>0.209 (0.046) </td><td>0.519 (0.039) </td></tr><tr><td>Lagging </td><td>0.331 (0.040) </td><td>0.204 (0.032) </td><td>0.276 (0.054) </td><td>0.189 (0.040) </td><td>0.464 (0.043) </td></tr><tr><td rowspan="2">Second codon position </td><td>Leading </td><td>0.308 (0.058) </td><td>0.206 (0.035) </td><td>0.162 (0.032) </td><td>0.325 (0.054) </td><td>0.486 (0.055) </td></tr><tr><td>Lagging </td><td>0.319 (0.060) </td><td>0.220 (0.031) </td><td>0.141 (0.030) </td><td>0.320 (0.049) </td><td>0.461 (0.057) </td></tr><tr><td rowspan="2">Third codon position </td><td>Leading </td><td>0.364 (0.045) </td><td>0.065 (0.038) </td><td>0.127 (0.031) </td><td>0.444 (0.047) </td><td>0.571 (0.041) </td></tr><tr><td>Lagging </td><td>0.391 (0.040) </td><td>0.102 (0.024) </td><td>0.078 (0.022) </td><td>0.429 (0.037) </td><td>0.507 (0.037) </td></tr><tr><td rowspan="2">All codon positions </td><td>Leading </td><td>0.325 (0.032) </td><td>0.149 (0.022) </td><td>0.199 (0.033) </td><td>0.326 (0.036) </td><td>0.525 (0.033) </td></tr><tr><td>Lagging </td><td>0.347 (0.035) </td><td>0.175 (0.017) </td><td>0.165 (0.024) </td><td>0.313 (0.029) </td><td>0.478 (0.034) </td></tr></tbody></table></div><div content-id="DSP001TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>A </th><th>C </th><th>G </th><th>T </th><th>G + T </th></tr><tr><th> </th><th> </th><th>Mean (SD) </th><th>Mean (SD) </th><th>Mean (SD) </th><th>Mean (SD) </th><th>Mean (SD) </th></tr></thead><tbody><tr><td rowspan="2">First codon position </td><td>Leading </td><td>0.304 (0.036) </td><td>0.177 (0.031) </td><td>0.309 (0.047) </td><td>0.209 (0.046) </td><td>0.519 (0.039) </td></tr><tr><td>Lagging </td><td>0.331 (0.040) </td><td>0.204 (0.032) </td><td>0.276 (0.054) </td><td>0.189 (0.040) </td><td>0.464 (0.043) </td></tr><tr><td rowspan="2">Second codon position </td><td>Leading </td><td>0.308 (0.058) </td><td>0.206 (0.035) </td><td>0.162 (0.032) </td><td>0.325 (0.054) </td><td>0.486 (0.055) </td></tr><tr><td>Lagging </td><td>0.319 (0.060) </td><td>0.220 (0.031) </td><td>0.141 (0.030) </td><td>0.320 (0.049) </td><td>0.461 (0.057) </td></tr><tr><td rowspan="2">Third codon position </td><td>Leading </td><td>0.364 (0.045) </td><td>0.065 (0.038) </td><td>0.127 (0.031) </td><td>0.444 (0.047) </td><td>0.571 (0.041) </td></tr><tr><td>Lagging </td><td>0.391 (0.040) </td><td>0.102 (0.024) </td><td>0.078 (0.022) </td><td>0.429 (0.037) </td><td>0.507 (0.037) </td></tr><tr><td rowspan="2">All codon positions </td><td>Leading </td><td>0.325 (0.032) </td><td>0.149 (0.022) </td><td>0.199 (0.033) </td><td>0.326 (0.036) </td><td>0.525 (0.033) </td></tr><tr><td>Lagging </td><td>0.347 (0.035) </td><td>0.175 (0.017) </td><td>0.165 (0.024) </td><td>0.313 (0.029) </td><td>0.478 (0.034) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195374" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP001TB2" class="table-wrap-title" data-id="DSP001TB2"><div class="label -label">Table 2</div><div class="caption"><p>Base compositions and medium G + T content at three codon positions for genes located on the leading and lagging strands of the chromosome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>A </th><th>C </th><th>G </th><th>T </th><th>G + T </th></tr><tr><th> </th><th> </th><th>Mean (SD) </th><th>Mean (SD) </th><th>Mean (SD) </th><th>Mean (SD) </th><th>Mean (SD) </th></tr></thead><tbody><tr><td rowspan="2">First codon position </td><td>Leading </td><td>0.304 (0.036) </td><td>0.177 (0.031) </td><td>0.309 (0.047) </td><td>0.209 (0.046) </td><td>0.519 (0.039) </td></tr><tr><td>Lagging </td><td>0.331 (0.040) </td><td>0.204 (0.032) </td><td>0.276 (0.054) </td><td>0.189 (0.040) </td><td>0.464 (0.043) </td></tr><tr><td rowspan="2">Second codon position </td><td>Leading </td><td>0.308 (0.058) </td><td>0.206 (0.035) </td><td>0.162 (0.032) </td><td>0.325 (0.054) </td><td>0.486 (0.055) </td></tr><tr><td>Lagging </td><td>0.319 (0.060) </td><td>0.220 (0.031) </td><td>0.141 (0.030) </td><td>0.320 (0.049) </td><td>0.461 (0.057) </td></tr><tr><td rowspan="2">Third codon position </td><td>Leading </td><td>0.364 (0.045) </td><td>0.065 (0.038) </td><td>0.127 (0.031) </td><td>0.444 (0.047) </td><td>0.571 (0.041) </td></tr><tr><td>Lagging </td><td>0.391 (0.040) </td><td>0.102 (0.024) </td><td>0.078 (0.022) </td><td>0.429 (0.037) </td><td>0.507 (0.037) </td></tr><tr><td rowspan="2">All codon positions </td><td>Leading </td><td>0.325 (0.032) </td><td>0.149 (0.022) </td><td>0.199 (0.033) </td><td>0.326 (0.036) </td><td>0.525 (0.033) </td></tr><tr><td>Lagging </td><td>0.347 (0.035) </td><td>0.175 (0.017) </td><td>0.165 (0.024) </td><td>0.313 (0.029) </td><td>0.478 (0.034) </td></tr></tbody></table></div><div content-id="DSP001TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>A </th><th>C </th><th>G </th><th>T </th><th>G + T </th></tr><tr><th> </th><th> </th><th>Mean (SD) </th><th>Mean (SD) </th><th>Mean (SD) </th><th>Mean (SD) </th><th>Mean (SD) </th></tr></thead><tbody><tr><td rowspan="2">First codon position </td><td>Leading </td><td>0.304 (0.036) </td><td>0.177 (0.031) </td><td>0.309 (0.047) </td><td>0.209 (0.046) </td><td>0.519 (0.039) </td></tr><tr><td>Lagging </td><td>0.331 (0.040) </td><td>0.204 (0.032) </td><td>0.276 (0.054) </td><td>0.189 (0.040) </td><td>0.464 (0.043) </td></tr><tr><td rowspan="2">Second codon position </td><td>Leading </td><td>0.308 (0.058) </td><td>0.206 (0.035) </td><td>0.162 (0.032) </td><td>0.325 (0.054) </td><td>0.486 (0.055) </td></tr><tr><td>Lagging </td><td>0.319 (0.060) </td><td>0.220 (0.031) </td><td>0.141 (0.030) </td><td>0.320 (0.049) </td><td>0.461 (0.057) </td></tr><tr><td rowspan="2">Third codon position </td><td>Leading </td><td>0.364 (0.045) </td><td>0.065 (0.038) </td><td>0.127 (0.031) </td><td>0.444 (0.047) </td><td>0.571 (0.041) </td></tr><tr><td>Lagging </td><td>0.391 (0.040) </td><td>0.102 (0.024) </td><td>0.078 (0.022) </td><td>0.429 (0.037) </td><td>0.507 (0.037) </td></tr><tr><td rowspan="2">All codon positions </td><td>Leading </td><td>0.325 (0.032) </td><td>0.149 (0.022) </td><td>0.199 (0.033) </td><td>0.326 (0.036) </td><td>0.525 (0.033) </td></tr><tr><td>Lagging </td><td>0.347 (0.035) </td><td>0.175 (0.017) </td><td>0.165 (0.024) </td><td>0.313 (0.029) </td><td>0.478 (0.034) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195374" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87195375 id="87195375" class="section-title" data-legacy-id=s3c>3.3. The first trend is associated with strand-specific mutational biases</h3>
<p>COA of codon usage was used to study extensively and quantitatively the variation of codon usage among the 1180 genes on the chromosome. Because the use of relative codon frequency sometimes introduces other biases and often diminishes the quantity of information to analyze, occasionally resulting in interpretation errors, we compute in parallel COA on codon counts and on RSCU and then compare the results in this study. Fig. <a reveal-id="DSP001F2" data-open="DSP001F2" class="link link-reveal link-table xref-fig">2</a>(a) and (b) shows the positions of the genes along the first and second major axes produced by COA on codon counts and RSCU values, respectively. The closeness of any two genes on each plot reflects the similarities of their codon usages. In the following sections, the factors that drive variation of synonymous codon usage of genes on the chromosome are discussed.</p>                        <a id="87195377" scrollto-destination="87195377"></a>
<div id="DSP001F2" data-id="DSP001F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/m_dsp00102.jpeg?Expires=2147483647&Signature=3fhMF6ZCs2sJetvCUUM6IewzSI0cf9EuooUEO7jUC6PkohfpUwyRl~EF7ASIOW9FgOueYWXTtnVaHIUQxFSk2LDgK4Qf5QGXoz26zd3jjS081tlCFEj91J05yqcLieyxypRjOKMZtD85A41t2fQRTArH0oYLKXJZjKYVJazNxk2KpHNlRJWoqS~2Aky9DCQhsmLtOG8ffQ2GG~yvSJjXYyofhid2yIEiEfrS-nzUmHgBDAJ6XXSj7zDAqZSWrPj8fAbtViVa3Hhuv0MzzUrlUBgploxpBi7c3z63aQ~2vavvjktiq6YPWNlpcKZ86PN-m7aDXHauHQhovWKndjCSPg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Plot of the two most important axes after COA for the 1180 genes on the chromosome. The open circles indicate the genes that are transcribed on the leading strands of replication. The filled circles indicate the genes that are transcribed on the lagging strands of the replication. (a) Use codon count to compute COA (COA/counts), the first and second axes account for 10.5% and 7.5% of the total inertia of the 59-dimensional space. (b) Use RSCU to compute COA (COA/RSCU), the first and second axes account for 9.2% and 7.7% of the total inertia of the 59-dimensional space." path-from-xml="dsp00102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195377/dsp00102.jpeg" path-from-xml="dsp00102.jpeg" target="_blank">View large</a><a section="87195377" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/dsp00102.jpeg?Expires=2147483647&Signature=L63OwSPINoT~oeX5yMKzEDDU0Iv3h8tLTashiK8yBVviLRgRxv9WFxetCFjx8QzzteIYjOshX0PJQQxgvLEewt9Ky0dmT51HeApAquz1tpj2C54xzS1~3ZytmvdNuBQX0sbn354bIH7y~Vzh4nDM2l2MMra6GLaTvBP6kQQGys39Ze-hl-N6-LcgYGMRXLUoipAqwGhc7fhSy80bFSTyY-CqUvU-Km8XDvPcPtrL0TTj~~GM~~rHFeff-gdOy~Rk4wN7H5lbBdgy8DBLp0foPNM8P0QU7zHR1qSp-u3ujVOD75bw94L7oCjeuRu0-U2eib6QRZ61iI5IbDvMhky0Yw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195377&ar=510591&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Plot of the two most important axes after COA for the 1180 genes on the chromosome. The open circles indicate the genes that are transcribed on the leading strands of replication. The filled circles indicate the genes that are transcribed on the lagging strands of the replication. (<strong>a</strong>) Use codon count to compute COA (COA/counts), the first and second axes account for 10.5% and 7.5% of the total inertia of the 59-dimensional space. (<strong>b</strong>) Use RSCU to compute COA (COA/RSCU), the first and second axes account for 9.2% and 7.7% of the total inertia of the 59-dimensional space.</p></div></div><div content-id="DSP001F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/m_dsp00102.jpeg?Expires=2147483647&Signature=3fhMF6ZCs2sJetvCUUM6IewzSI0cf9EuooUEO7jUC6PkohfpUwyRl~EF7ASIOW9FgOueYWXTtnVaHIUQxFSk2LDgK4Qf5QGXoz26zd3jjS081tlCFEj91J05yqcLieyxypRjOKMZtD85A41t2fQRTArH0oYLKXJZjKYVJazNxk2KpHNlRJWoqS~2Aky9DCQhsmLtOG8ffQ2GG~yvSJjXYyofhid2yIEiEfrS-nzUmHgBDAJ6XXSj7zDAqZSWrPj8fAbtViVa3Hhuv0MzzUrlUBgploxpBi7c3z63aQ~2vavvjktiq6YPWNlpcKZ86PN-m7aDXHauHQhovWKndjCSPg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Plot of the two most important axes after COA for the 1180 genes on the chromosome. The open circles indicate the genes that are transcribed on the leading strands of replication. The filled circles indicate the genes that are transcribed on the lagging strands of the replication. (a) Use codon count to compute COA (COA/counts), the first and second axes account for 10.5% and 7.5% of the total inertia of the 59-dimensional space. (b) Use RSCU to compute COA (COA/RSCU), the first and second axes account for 9.2% and 7.7% of the total inertia of the 59-dimensional space." path-from-xml="dsp00102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195377/dsp00102.jpeg" path-from-xml="dsp00102.jpeg" target="_blank">View large</a><a section="87195377" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/dsp00102.jpeg?Expires=2147483647&Signature=L63OwSPINoT~oeX5yMKzEDDU0Iv3h8tLTashiK8yBVviLRgRxv9WFxetCFjx8QzzteIYjOshX0PJQQxgvLEewt9Ky0dmT51HeApAquz1tpj2C54xzS1~3ZytmvdNuBQX0sbn354bIH7y~Vzh4nDM2l2MMra6GLaTvBP6kQQGys39Ze-hl-N6-LcgYGMRXLUoipAqwGhc7fhSy80bFSTyY-CqUvU-Km8XDvPcPtrL0TTj~~GM~~rHFeff-gdOy~Rk4wN7H5lbBdgy8DBLp0foPNM8P0QU7zHR1qSp-u3ujVOD75bw94L7oCjeuRu0-U2eib6QRZ61iI5IbDvMhky0Yw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195377&ar=510591&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Plot of the two most important axes after COA for the 1180 genes on the chromosome. The open circles indicate the genes that are transcribed on the leading strands of replication. The filled circles indicate the genes that are transcribed on the lagging strands of the replication. (<strong>a</strong>) Use codon count to compute COA (COA/counts), the first and second axes account for 10.5% and 7.5% of the total inertia of the 59-dimensional space. (<strong>b</strong>) Use RSCU to compute COA (COA/RSCU), the first and second axes account for 9.2% and 7.7% of the total inertia of the 59-dimensional space.</p></div></div><p>Both in Fig. <a reveal-id="DSP001F2" data-open="DSP001F2" class="link link-reveal link-table xref-fig">2</a>(a) and (b), the first axis separate the genes into two clusters with little overlap between them. The following two results could indicate that the two clusters correspond to genes on the leading and lagging strands of replication. (i) The first axis is found to strongly correlate with GC and AT skews, particularly at the third position. At the left of the first axis, genes are characterized by richness in nucleotides G and T, whereas it is opposite at the right. On the other hand, it has been found that there is an excess of nucleotides G relative to C in the leading strands and of C to G in the lagging strands in most bacterial genomes, which is frequently accompanied by an abundance of T over A in the leading strand.<sup><a reveal-id="DSP001C18" data-open="DSP001C18" class="link link-ref link-reveal xref-bibr">18</a></sup> (ii) The coordinates of individual genes along the first axis are plotted against the chromosomal locations of the corresponding genes in Fig. <a reveal-id="DSP001F3" data-open="DSP001F3" class="link link-reveal link-table xref-fig">3</a>. Genes on the Watson strand and those on the Crick strand are denoted by black and gray squares, respectively. It is found that genes on the left side of Watson strand and those on the right side of Crick strand have low values of coordinates along the Axis 1, whereas, for the other genes, the case is opposite. In fact, genes on the left side of Watson strand and those on the right side of Crick strand just correspond to genes on the leading strands, the other ones correspond to lagging strands. Therefore, it is reasonable to say that two clusters in Fig. <a reveal-id="DSP001F2" data-open="DSP001F2" class="link link-reveal link-table xref-fig">2</a> correspond to genes on the leading strands and lagging strands, respectively. After marking genes located on the leading and lagging strands by different symbols in Fig. <a reveal-id="DSP001F2" data-open="DSP001F2" class="link link-reveal link-table xref-fig">2</a>, the speculation is confirmed.</p>                        <a id="87195379" scrollto-destination="87195379"></a>
<div id="DSP001F3" data-id="DSP001F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/m_dsp00103.jpeg?Expires=2147483647&Signature=Qol0MqsiYwA5unPdsAUOfvUlWbrYNbwSomSid3wIa6xfpdhG2waNGG0SHuksOLHa7Z-Ip2am92QgX3Az8LAfuXl9q8Vr-uoVOgUe6nfHY0g3YUZ~JiBWavcjEjvqUXpBYbKxjliSnky9WWiLC94PMLlkfuNfYufhpFM0FKxqqAdViuX5Ev9xzrQ7FGjHF4E9kAekTKV6fm5GMYqab88UVPbwvqpS~HAc6e~YWHwD36qJFek7edpHqUy7kV-11P5etL~iIw82o-uS9b6zz9mkypk9A36AblmtDn-7AuSTrxYi5vX~fKUeJw10NCOUBz8ovuFCw6~NvclsqGqMCnslAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Plot of Axis 1 values of chromosomal genes against their corresponding chromosomal locations in the L. intracellularis genome. Black squares represent genes located on the Watson strand and gray squares represent genes on the Crick strand. To allow convenient observation, the beginning point of the horizontal axis is shifted to the origin of replication." path-from-xml="dsp00103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195379/dsp00103.jpeg" path-from-xml="dsp00103.jpeg" target="_blank">View large</a><a section="87195379" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/dsp00103.jpeg?Expires=2147483647&Signature=dcCC4k~~yDEiU3kI0RV7ryel3ZCaIHvKQHDGZjlc8BdSctOwO888NUTd3823vqno9tJb55OUrJnVVO6TnFrpiM2h9gFHxrfqB-lm-LNqDMjE970TMMH-aRXquTgq-y7AU33U2NklmbJNeNzFWJC~6EiwqYAWlsfWD0mFTUtEnsq2w-yKO7SVdbDMG7wFjSsyLPk-Bt-3LaIasYI1ugJ1kOGd~-stlGdAtrQV1w5OQwdkgOAncsg2JRFHuu4wC7LuMn~XoT-e0QXUu4mSdwqQVAls6F0fqHgyffAcZJBfLltZDpMK4oze0rqMW95Pjog-mK9ine1ER-o-j-eL6Dg3Ig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195379&ar=510591&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Plot of Axis 1 values of chromosomal genes against their corresponding chromosomal locations in the <em>L</em>. <em>intracellularis</em> genome. Black squares represent genes located on the Watson strand and gray squares represent genes on the Crick strand. To allow convenient observation, the beginning point of the horizontal axis is shifted to the origin of replication.</p></div></div><div content-id="DSP001F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/m_dsp00103.jpeg?Expires=2147483647&Signature=Qol0MqsiYwA5unPdsAUOfvUlWbrYNbwSomSid3wIa6xfpdhG2waNGG0SHuksOLHa7Z-Ip2am92QgX3Az8LAfuXl9q8Vr-uoVOgUe6nfHY0g3YUZ~JiBWavcjEjvqUXpBYbKxjliSnky9WWiLC94PMLlkfuNfYufhpFM0FKxqqAdViuX5Ev9xzrQ7FGjHF4E9kAekTKV6fm5GMYqab88UVPbwvqpS~HAc6e~YWHwD36qJFek7edpHqUy7kV-11P5etL~iIw82o-uS9b6zz9mkypk9A36AblmtDn-7AuSTrxYi5vX~fKUeJw10NCOUBz8ovuFCw6~NvclsqGqMCnslAQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Plot of Axis 1 values of chromosomal genes against their corresponding chromosomal locations in the L. intracellularis genome. Black squares represent genes located on the Watson strand and gray squares represent genes on the Crick strand. To allow convenient observation, the beginning point of the horizontal axis is shifted to the origin of replication." path-from-xml="dsp00103.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195379/dsp00103.jpeg" path-from-xml="dsp00103.jpeg" target="_blank">View large</a><a section="87195379" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/dsp00103.jpeg?Expires=2147483647&Signature=dcCC4k~~yDEiU3kI0RV7ryel3ZCaIHvKQHDGZjlc8BdSctOwO888NUTd3823vqno9tJb55OUrJnVVO6TnFrpiM2h9gFHxrfqB-lm-LNqDMjE970TMMH-aRXquTgq-y7AU33U2NklmbJNeNzFWJC~6EiwqYAWlsfWD0mFTUtEnsq2w-yKO7SVdbDMG7wFjSsyLPk-Bt-3LaIasYI1ugJ1kOGd~-stlGdAtrQV1w5OQwdkgOAncsg2JRFHuu4wC7LuMn~XoT-e0QXUu4mSdwqQVAls6F0fqHgyffAcZJBfLltZDpMK4oze0rqMW95Pjog-mK9ine1ER-o-j-eL6Dg3Ig__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195379&ar=510591&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00103.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Plot of Axis 1 values of chromosomal genes against their corresponding chromosomal locations in the <em>L</em>. <em>intracellularis</em> genome. Black squares represent genes located on the Watson strand and gray squares represent genes on the Crick strand. To allow convenient observation, the beginning point of the horizontal axis is shifted to the origin of replication.</p></div></div><p>A <em>χ</em><sup>2</sup> test was performed for RSCU of genes located on the leading versus lagging strands and the results are listed in Table <a reveal-id="DSP001TB3" data-open="DSP001TB3" class="link link-reveal link-table xref-fig">3</a>. As can be seen, 49 among 59 codons are found to be significantly different between genes on the two strands of replication. The 23 codons used more frequently in the leading strands are G-ending or T-ending, except TTA, ACA, AGA and GCA. Among the 26 codons used more frequently in the lagging strands, 16 are C-ending, eight are A-ending and the exceptions are CTT and ACT. Results of the test confirm that there is a bias toward G, T in the leading strands and toward C, A in the lagging strands of replication. Therefore, it can be concluded that in <em>L</em>. <em>intracellularis</em>, the leading and lagging strands of replication display an asymmetry in the mutational biases and or the differential correction/repair rates, and as shown in several other bacteria,<sup><a reveal-id="DSP001C10" data-open="DSP001C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSP001C11" data-open="DSP001C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSP001C30  DSP001C35" data-open="DSP001C30  DSP001C35" class="link link-ref link-reveal xref-bibr">30–35</a></sup> this difference is the most important source of codon usage variation.</p>                        <a id="87195381" scrollto-destination="87195381"></a>
<div content-id="DSP001TB3" class="table-modal"><div class="table-wrap"><div id="DSP001TB3" class="table-wrap-title" data-id="DSP001TB3"><div class="label -label">Table 3</div><div class="caption"><p>Results of <em>χ</em><sup>2</sup> test for RSCU of genes on the leading and lagging strands of the chromosome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>AA </th><th>Codon </th><th>RSCU leading </th><th>Significant<sup>a</sup> </th><th>RSCU lagging </th><th>AA </th><th>Codon </th><th>RSCU Leading </th><th>Significant<sup>a</sup> </th><th>RSCU lagging </th></tr></thead><tbody><tr><td>Phe </td><td>UUU </td><td>1.81 </td><td>≫ </td><td>1.64 </td><td>Ser </td><td>UCU </td><td>2.21 </td><td>– </td><td>2.25 </td></tr><tr><td> </td><td>UUC </td><td>0.19 </td><td>≪ </td><td>0.36 </td><td> </td><td>UCC </td><td>0.23 </td><td>≪ </td><td>0.39 </td></tr><tr><td>Leu </td><td>UUA </td><td>2.53 </td><td>≫ </td><td>1.86 </td><td> </td><td>UCA </td><td>1.47 </td><td>≪ </td><td>1.60 </td></tr><tr><td> </td><td>UUG </td><td>0.60 </td><td>≫ </td><td>0.20 </td><td> </td><td>UCG </td><td>0.14 </td><td>≫ </td><td>0.08 </td></tr><tr><td>Tyr </td><td>UAU </td><td>1.74 </td><td>≫ </td><td>1.60 </td><td>Cys </td><td>UGU </td><td>1.77 </td><td>≫ </td><td>1.59 </td></tr><tr><td> </td><td>UAC </td><td>0.26 </td><td>≪ </td><td>0.40 </td><td> </td><td>UGC </td><td>0.23 </td><td>≪ </td><td>0.41 </td></tr><tr><td>Ter </td><td>UAA </td><td>0.00 </td><td>– </td><td>0.00 </td><td>Ter </td><td>UGA </td><td>0.00 </td><td>– </td><td>0.00 </td></tr><tr><td>Ter </td><td>UAG </td><td>0.00 </td><td>– </td><td>0.00 </td><td>Trp </td><td>UGG </td><td>1.00 </td><td>– </td><td>1.00 </td></tr><tr><td>Leu </td><td>CUU </td><td>1.98 </td><td>≪ </td><td>2.44 </td><td>Pro </td><td>CCU </td><td>2.08 </td><td>– </td><td>2.06 </td></tr><tr><td> </td><td>CUC </td><td>0.17 </td><td>≪ </td><td>0.46 </td><td> </td><td>CCC </td><td>0.18 </td><td>≪ </td><td>0.30 </td></tr><tr><td> </td><td>CUA </td><td>0.56 </td><td>≪ </td><td>0.89 </td><td> </td><td>CCA </td><td>1.64 </td><td>– </td><td>1.59 </td></tr><tr><td> </td><td>CUG </td><td>0.15 </td><td>– </td><td>0.14 </td><td> </td><td>CCG </td><td>0.10 </td><td>– </td><td>0.05 </td></tr><tr><td>His </td><td>CAU </td><td>1.75 </td><td>≫ </td><td>1.61 </td><td>Arg </td><td>CGU </td><td>2.24 </td><td>– </td><td>2.18 </td></tr><tr><td> </td><td>CAC </td><td>0.25 </td><td>≪ </td><td>0.39 </td><td> </td><td>CGC </td><td>0.26 </td><td>≪ </td><td>0.51 </td></tr><tr><td>Gln </td><td>CAA </td><td>1.41 </td><td>≪ </td><td>1.74 </td><td> </td><td>CGA </td><td>0.74 </td><td>≪ </td><td>1.02 </td></tr><tr><td> </td><td>CAG </td><td>0.59 </td><td>≫ </td><td>0.26 </td><td> </td><td>CGG </td><td>0.25 </td><td>– </td><td>0.20 </td></tr><tr><td>Ile </td><td>AUU </td><td>1.72 </td><td>≫ </td><td>1.59 </td><td>Thr </td><td>ACU </td><td>1.33 </td><td>≪ </td><td>1.39 </td></tr><tr><td> </td><td>AUC </td><td>0.26 </td><td>≪ </td><td>0.41 </td><td> </td><td>ACC </td><td>0.24 </td><td>≪ </td><td>0.36 </td></tr><tr><td> </td><td>AUA </td><td>1.02 </td><td>– </td><td>1.01 </td><td> </td><td>ACA </td><td>2.24 </td><td>≫ </td><td>2.14 </td></tr><tr><td>Met </td><td>AUG </td><td>1.00 </td><td>– </td><td>1.00 </td><td> </td><td>ACG </td><td>0.19 </td><td>≫ </td><td>0.12 </td></tr><tr><td>Asn </td><td>AAU </td><td>1.66 </td><td>≫ </td><td>1.49 </td><td>Ser </td><td>AGU </td><td>1.58 </td><td>≫ </td><td>1.24 </td></tr><tr><td> </td><td>AAC </td><td>0.34 </td><td>≪ </td><td>0.51 </td><td> </td><td>AGC </td><td>0.36 </td><td>≪ </td><td>0.45 </td></tr><tr><td>Lys </td><td>AAA </td><td>1.47 </td><td>≪ </td><td>1.76 </td><td>Arg </td><td>AGA </td><td>1.88 </td><td>≫ </td><td>1.75 </td></tr><tr><td> </td><td>AAG </td><td>0.53 </td><td>≫ </td><td>0.24 </td><td> </td><td>AGG </td><td>0.63 </td><td>≫ </td><td>0.34 </td></tr><tr><td>Val </td><td>GUU </td><td>1.93 </td><td>≫ </td><td>1.82 </td><td>Ala </td><td>GCU </td><td>1.85 </td><td>– </td><td>1.84 </td></tr><tr><td> </td><td>GUC </td><td>0.29 </td><td>≪ </td><td>0.47 </td><td> </td><td>GCC </td><td>0.25 </td><td>≪ </td><td>0.36 </td></tr><tr><td> </td><td>GUA </td><td>1.40 </td><td>≪ </td><td>1.51 </td><td> </td><td>GCA </td><td>1.78 </td><td>≫ </td><td>1.73 </td></tr><tr><td> </td><td>GUG </td><td>0.37 </td><td>≫ </td><td>0.20 </td><td> </td><td>GCG </td><td>0.12 </td><td>– </td><td>0.08 </td></tr><tr><td>Asp </td><td>GAU </td><td>1.73 </td><td>≫ </td><td>1.60 </td><td>Gly </td><td>GGU </td><td>1.67 </td><td>≫ </td><td>1.41 </td></tr><tr><td> </td><td>GAC </td><td>0.27 </td><td>≪ </td><td>0.40 </td><td> </td><td>GGC </td><td>0.32 </td><td>≪ </td><td>0.41 </td></tr><tr><td>Glu </td><td>GAA </td><td>1.43 </td><td>≪ </td><td>1.73 </td><td> </td><td>GGA </td><td>1.46 </td><td>≪ </td><td>1.76 </td></tr><tr><td> </td><td>GAG </td><td>0.57 </td><td>≫ </td><td>0.27 </td><td> </td><td>GGG </td><td>0.55 </td><td>≫ </td><td>0.42 </td></tr></tbody></table></div><div content-id="DSP001TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>AA </th><th>Codon </th><th>RSCU leading </th><th>Significant<sup>a</sup> </th><th>RSCU lagging </th><th>AA </th><th>Codon </th><th>RSCU Leading </th><th>Significant<sup>a</sup> </th><th>RSCU lagging </th></tr></thead><tbody><tr><td>Phe </td><td>UUU </td><td>1.81 </td><td>≫ </td><td>1.64 </td><td>Ser </td><td>UCU </td><td>2.21 </td><td>– </td><td>2.25 </td></tr><tr><td> </td><td>UUC </td><td>0.19 </td><td>≪ </td><td>0.36 </td><td> </td><td>UCC </td><td>0.23 </td><td>≪ </td><td>0.39 </td></tr><tr><td>Leu </td><td>UUA </td><td>2.53 </td><td>≫ </td><td>1.86 </td><td> </td><td>UCA </td><td>1.47 </td><td>≪ </td><td>1.60 </td></tr><tr><td> </td><td>UUG </td><td>0.60 </td><td>≫ </td><td>0.20 </td><td> </td><td>UCG </td><td>0.14 </td><td>≫ </td><td>0.08 </td></tr><tr><td>Tyr </td><td>UAU </td><td>1.74 </td><td>≫ </td><td>1.60 </td><td>Cys </td><td>UGU </td><td>1.77 </td><td>≫ </td><td>1.59 </td></tr><tr><td> </td><td>UAC </td><td>0.26 </td><td>≪ </td><td>0.40 </td><td> </td><td>UGC </td><td>0.23 </td><td>≪ </td><td>0.41 </td></tr><tr><td>Ter </td><td>UAA </td><td>0.00 </td><td>– </td><td>0.00 </td><td>Ter </td><td>UGA </td><td>0.00 </td><td>– </td><td>0.00 </td></tr><tr><td>Ter </td><td>UAG </td><td>0.00 </td><td>– </td><td>0.00 </td><td>Trp </td><td>UGG </td><td>1.00 </td><td>– </td><td>1.00 </td></tr><tr><td>Leu </td><td>CUU </td><td>1.98 </td><td>≪ </td><td>2.44 </td><td>Pro </td><td>CCU </td><td>2.08 </td><td>– </td><td>2.06 </td></tr><tr><td> </td><td>CUC </td><td>0.17 </td><td>≪ </td><td>0.46 </td><td> </td><td>CCC </td><td>0.18 </td><td>≪ </td><td>0.30 </td></tr><tr><td> </td><td>CUA </td><td>0.56 </td><td>≪ </td><td>0.89 </td><td> </td><td>CCA </td><td>1.64 </td><td>– </td><td>1.59 </td></tr><tr><td> </td><td>CUG </td><td>0.15 </td><td>– </td><td>0.14 </td><td> </td><td>CCG </td><td>0.10 </td><td>– </td><td>0.05 </td></tr><tr><td>His </td><td>CAU </td><td>1.75 </td><td>≫ </td><td>1.61 </td><td>Arg </td><td>CGU </td><td>2.24 </td><td>– </td><td>2.18 </td></tr><tr><td> </td><td>CAC </td><td>0.25 </td><td>≪ </td><td>0.39 </td><td> </td><td>CGC </td><td>0.26 </td><td>≪ </td><td>0.51 </td></tr><tr><td>Gln </td><td>CAA </td><td>1.41 </td><td>≪ </td><td>1.74 </td><td> </td><td>CGA </td><td>0.74 </td><td>≪ </td><td>1.02 </td></tr><tr><td> </td><td>CAG </td><td>0.59 </td><td>≫ </td><td>0.26 </td><td> </td><td>CGG </td><td>0.25 </td><td>– </td><td>0.20 </td></tr><tr><td>Ile </td><td>AUU </td><td>1.72 </td><td>≫ </td><td>1.59 </td><td>Thr </td><td>ACU </td><td>1.33 </td><td>≪ </td><td>1.39 </td></tr><tr><td> </td><td>AUC </td><td>0.26 </td><td>≪ </td><td>0.41 </td><td> </td><td>ACC </td><td>0.24 </td><td>≪ </td><td>0.36 </td></tr><tr><td> </td><td>AUA </td><td>1.02 </td><td>– </td><td>1.01 </td><td> </td><td>ACA </td><td>2.24 </td><td>≫ </td><td>2.14 </td></tr><tr><td>Met </td><td>AUG </td><td>1.00 </td><td>– </td><td>1.00 </td><td> </td><td>ACG </td><td>0.19 </td><td>≫ </td><td>0.12 </td></tr><tr><td>Asn </td><td>AAU </td><td>1.66 </td><td>≫ </td><td>1.49 </td><td>Ser </td><td>AGU </td><td>1.58 </td><td>≫ </td><td>1.24 </td></tr><tr><td> </td><td>AAC </td><td>0.34 </td><td>≪ </td><td>0.51 </td><td> </td><td>AGC </td><td>0.36 </td><td>≪ </td><td>0.45 </td></tr><tr><td>Lys </td><td>AAA </td><td>1.47 </td><td>≪ </td><td>1.76 </td><td>Arg </td><td>AGA </td><td>1.88 </td><td>≫ </td><td>1.75 </td></tr><tr><td> </td><td>AAG </td><td>0.53 </td><td>≫ </td><td>0.24 </td><td> </td><td>AGG </td><td>0.63 </td><td>≫ </td><td>0.34 </td></tr><tr><td>Val </td><td>GUU </td><td>1.93 </td><td>≫ </td><td>1.82 </td><td>Ala </td><td>GCU </td><td>1.85 </td><td>– </td><td>1.84 </td></tr><tr><td> </td><td>GUC </td><td>0.29 </td><td>≪ </td><td>0.47 </td><td> </td><td>GCC </td><td>0.25 </td><td>≪ </td><td>0.36 </td></tr><tr><td> </td><td>GUA </td><td>1.40 </td><td>≪ </td><td>1.51 </td><td> </td><td>GCA </td><td>1.78 </td><td>≫ </td><td>1.73 </td></tr><tr><td> </td><td>GUG </td><td>0.37 </td><td>≫ </td><td>0.20 </td><td> </td><td>GCG </td><td>0.12 </td><td>– </td><td>0.08 </td></tr><tr><td>Asp </td><td>GAU </td><td>1.73 </td><td>≫ </td><td>1.60 </td><td>Gly </td><td>GGU </td><td>1.67 </td><td>≫ </td><td>1.41 </td></tr><tr><td> </td><td>GAC </td><td>0.27 </td><td>≪ </td><td>0.40 </td><td> </td><td>GGC </td><td>0.32 </td><td>≪ </td><td>0.41 </td></tr><tr><td>Glu </td><td>GAA </td><td>1.43 </td><td>≪ </td><td>1.73 </td><td> </td><td>GGA </td><td>1.46 </td><td>≪ </td><td>1.76 </td></tr><tr><td> </td><td>GAG </td><td>0.57 </td><td>≫ </td><td>0.27 </td><td> </td><td>GGG </td><td>0.55 </td><td>≫ </td><td>0.42 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>≫ indicates that the leading strand genes use the codon more frequently than the lagging strand genes; ≪ indicates the lagging strand genes use the codon more frequently than the leading strand genes; – indicates that there is no significant difference in the usage of the codon on either strand. Significance is examined at the level of 5%.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195381" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP001TB3" class="table-wrap-title" data-id="DSP001TB3"><div class="label -label">Table 3</div><div class="caption"><p>Results of <em>χ</em><sup>2</sup> test for RSCU of genes on the leading and lagging strands of the chromosome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>AA </th><th>Codon </th><th>RSCU leading </th><th>Significant<sup>a</sup> </th><th>RSCU lagging </th><th>AA </th><th>Codon </th><th>RSCU Leading </th><th>Significant<sup>a</sup> </th><th>RSCU lagging </th></tr></thead><tbody><tr><td>Phe </td><td>UUU </td><td>1.81 </td><td>≫ </td><td>1.64 </td><td>Ser </td><td>UCU </td><td>2.21 </td><td>– </td><td>2.25 </td></tr><tr><td> </td><td>UUC </td><td>0.19 </td><td>≪ </td><td>0.36 </td><td> </td><td>UCC </td><td>0.23 </td><td>≪ </td><td>0.39 </td></tr><tr><td>Leu </td><td>UUA </td><td>2.53 </td><td>≫ </td><td>1.86 </td><td> </td><td>UCA </td><td>1.47 </td><td>≪ </td><td>1.60 </td></tr><tr><td> </td><td>UUG </td><td>0.60 </td><td>≫ </td><td>0.20 </td><td> </td><td>UCG </td><td>0.14 </td><td>≫ </td><td>0.08 </td></tr><tr><td>Tyr </td><td>UAU </td><td>1.74 </td><td>≫ </td><td>1.60 </td><td>Cys </td><td>UGU </td><td>1.77 </td><td>≫ </td><td>1.59 </td></tr><tr><td> </td><td>UAC </td><td>0.26 </td><td>≪ </td><td>0.40 </td><td> </td><td>UGC </td><td>0.23 </td><td>≪ </td><td>0.41 </td></tr><tr><td>Ter </td><td>UAA </td><td>0.00 </td><td>– </td><td>0.00 </td><td>Ter </td><td>UGA </td><td>0.00 </td><td>– </td><td>0.00 </td></tr><tr><td>Ter </td><td>UAG </td><td>0.00 </td><td>– </td><td>0.00 </td><td>Trp </td><td>UGG </td><td>1.00 </td><td>– </td><td>1.00 </td></tr><tr><td>Leu </td><td>CUU </td><td>1.98 </td><td>≪ </td><td>2.44 </td><td>Pro </td><td>CCU </td><td>2.08 </td><td>– </td><td>2.06 </td></tr><tr><td> </td><td>CUC </td><td>0.17 </td><td>≪ </td><td>0.46 </td><td> </td><td>CCC </td><td>0.18 </td><td>≪ </td><td>0.30 </td></tr><tr><td> </td><td>CUA </td><td>0.56 </td><td>≪ </td><td>0.89 </td><td> </td><td>CCA </td><td>1.64 </td><td>– </td><td>1.59 </td></tr><tr><td> </td><td>CUG </td><td>0.15 </td><td>– </td><td>0.14 </td><td> </td><td>CCG </td><td>0.10 </td><td>– </td><td>0.05 </td></tr><tr><td>His </td><td>CAU </td><td>1.75 </td><td>≫ </td><td>1.61 </td><td>Arg </td><td>CGU </td><td>2.24 </td><td>– </td><td>2.18 </td></tr><tr><td> </td><td>CAC </td><td>0.25 </td><td>≪ </td><td>0.39 </td><td> </td><td>CGC </td><td>0.26 </td><td>≪ </td><td>0.51 </td></tr><tr><td>Gln </td><td>CAA </td><td>1.41 </td><td>≪ </td><td>1.74 </td><td> </td><td>CGA </td><td>0.74 </td><td>≪ </td><td>1.02 </td></tr><tr><td> </td><td>CAG </td><td>0.59 </td><td>≫ </td><td>0.26 </td><td> </td><td>CGG </td><td>0.25 </td><td>– </td><td>0.20 </td></tr><tr><td>Ile </td><td>AUU </td><td>1.72 </td><td>≫ </td><td>1.59 </td><td>Thr </td><td>ACU </td><td>1.33 </td><td>≪ </td><td>1.39 </td></tr><tr><td> </td><td>AUC </td><td>0.26 </td><td>≪ </td><td>0.41 </td><td> </td><td>ACC </td><td>0.24 </td><td>≪ </td><td>0.36 </td></tr><tr><td> </td><td>AUA </td><td>1.02 </td><td>– </td><td>1.01 </td><td> </td><td>ACA </td><td>2.24 </td><td>≫ </td><td>2.14 </td></tr><tr><td>Met </td><td>AUG </td><td>1.00 </td><td>– </td><td>1.00 </td><td> </td><td>ACG </td><td>0.19 </td><td>≫ </td><td>0.12 </td></tr><tr><td>Asn </td><td>AAU </td><td>1.66 </td><td>≫ </td><td>1.49 </td><td>Ser </td><td>AGU </td><td>1.58 </td><td>≫ </td><td>1.24 </td></tr><tr><td> </td><td>AAC </td><td>0.34 </td><td>≪ </td><td>0.51 </td><td> </td><td>AGC </td><td>0.36 </td><td>≪ </td><td>0.45 </td></tr><tr><td>Lys </td><td>AAA </td><td>1.47 </td><td>≪ </td><td>1.76 </td><td>Arg </td><td>AGA </td><td>1.88 </td><td>≫ </td><td>1.75 </td></tr><tr><td> </td><td>AAG </td><td>0.53 </td><td>≫ </td><td>0.24 </td><td> </td><td>AGG </td><td>0.63 </td><td>≫ </td><td>0.34 </td></tr><tr><td>Val </td><td>GUU </td><td>1.93 </td><td>≫ </td><td>1.82 </td><td>Ala </td><td>GCU </td><td>1.85 </td><td>– </td><td>1.84 </td></tr><tr><td> </td><td>GUC </td><td>0.29 </td><td>≪ </td><td>0.47 </td><td> </td><td>GCC </td><td>0.25 </td><td>≪ </td><td>0.36 </td></tr><tr><td> </td><td>GUA </td><td>1.40 </td><td>≪ </td><td>1.51 </td><td> </td><td>GCA </td><td>1.78 </td><td>≫ </td><td>1.73 </td></tr><tr><td> </td><td>GUG </td><td>0.37 </td><td>≫ </td><td>0.20 </td><td> </td><td>GCG </td><td>0.12 </td><td>– </td><td>0.08 </td></tr><tr><td>Asp </td><td>GAU </td><td>1.73 </td><td>≫ </td><td>1.60 </td><td>Gly </td><td>GGU </td><td>1.67 </td><td>≫ </td><td>1.41 </td></tr><tr><td> </td><td>GAC </td><td>0.27 </td><td>≪ </td><td>0.40 </td><td> </td><td>GGC </td><td>0.32 </td><td>≪ </td><td>0.41 </td></tr><tr><td>Glu </td><td>GAA </td><td>1.43 </td><td>≪ </td><td>1.73 </td><td> </td><td>GGA </td><td>1.46 </td><td>≪ </td><td>1.76 </td></tr><tr><td> </td><td>GAG </td><td>0.57 </td><td>≫ </td><td>0.27 </td><td> </td><td>GGG </td><td>0.55 </td><td>≫ </td><td>0.42 </td></tr></tbody></table></div><div content-id="DSP001TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>AA </th><th>Codon </th><th>RSCU leading </th><th>Significant<sup>a</sup> </th><th>RSCU lagging </th><th>AA </th><th>Codon </th><th>RSCU Leading </th><th>Significant<sup>a</sup> </th><th>RSCU lagging </th></tr></thead><tbody><tr><td>Phe </td><td>UUU </td><td>1.81 </td><td>≫ </td><td>1.64 </td><td>Ser </td><td>UCU </td><td>2.21 </td><td>– </td><td>2.25 </td></tr><tr><td> </td><td>UUC </td><td>0.19 </td><td>≪ </td><td>0.36 </td><td> </td><td>UCC </td><td>0.23 </td><td>≪ </td><td>0.39 </td></tr><tr><td>Leu </td><td>UUA </td><td>2.53 </td><td>≫ </td><td>1.86 </td><td> </td><td>UCA </td><td>1.47 </td><td>≪ </td><td>1.60 </td></tr><tr><td> </td><td>UUG </td><td>0.60 </td><td>≫ </td><td>0.20 </td><td> </td><td>UCG </td><td>0.14 </td><td>≫ </td><td>0.08 </td></tr><tr><td>Tyr </td><td>UAU </td><td>1.74 </td><td>≫ </td><td>1.60 </td><td>Cys </td><td>UGU </td><td>1.77 </td><td>≫ </td><td>1.59 </td></tr><tr><td> </td><td>UAC </td><td>0.26 </td><td>≪ </td><td>0.40 </td><td> </td><td>UGC </td><td>0.23 </td><td>≪ </td><td>0.41 </td></tr><tr><td>Ter </td><td>UAA </td><td>0.00 </td><td>– </td><td>0.00 </td><td>Ter </td><td>UGA </td><td>0.00 </td><td>– </td><td>0.00 </td></tr><tr><td>Ter </td><td>UAG </td><td>0.00 </td><td>– </td><td>0.00 </td><td>Trp </td><td>UGG </td><td>1.00 </td><td>– </td><td>1.00 </td></tr><tr><td>Leu </td><td>CUU </td><td>1.98 </td><td>≪ </td><td>2.44 </td><td>Pro </td><td>CCU </td><td>2.08 </td><td>– </td><td>2.06 </td></tr><tr><td> </td><td>CUC </td><td>0.17 </td><td>≪ </td><td>0.46 </td><td> </td><td>CCC </td><td>0.18 </td><td>≪ </td><td>0.30 </td></tr><tr><td> </td><td>CUA </td><td>0.56 </td><td>≪ </td><td>0.89 </td><td> </td><td>CCA </td><td>1.64 </td><td>– </td><td>1.59 </td></tr><tr><td> </td><td>CUG </td><td>0.15 </td><td>– </td><td>0.14 </td><td> </td><td>CCG </td><td>0.10 </td><td>– </td><td>0.05 </td></tr><tr><td>His </td><td>CAU </td><td>1.75 </td><td>≫ </td><td>1.61 </td><td>Arg </td><td>CGU </td><td>2.24 </td><td>– </td><td>2.18 </td></tr><tr><td> </td><td>CAC </td><td>0.25 </td><td>≪ </td><td>0.39 </td><td> </td><td>CGC </td><td>0.26 </td><td>≪ </td><td>0.51 </td></tr><tr><td>Gln </td><td>CAA </td><td>1.41 </td><td>≪ </td><td>1.74 </td><td> </td><td>CGA </td><td>0.74 </td><td>≪ </td><td>1.02 </td></tr><tr><td> </td><td>CAG </td><td>0.59 </td><td>≫ </td><td>0.26 </td><td> </td><td>CGG </td><td>0.25 </td><td>– </td><td>0.20 </td></tr><tr><td>Ile </td><td>AUU </td><td>1.72 </td><td>≫ </td><td>1.59 </td><td>Thr </td><td>ACU </td><td>1.33 </td><td>≪ </td><td>1.39 </td></tr><tr><td> </td><td>AUC </td><td>0.26 </td><td>≪ </td><td>0.41 </td><td> </td><td>ACC </td><td>0.24 </td><td>≪ </td><td>0.36 </td></tr><tr><td> </td><td>AUA </td><td>1.02 </td><td>– </td><td>1.01 </td><td> </td><td>ACA </td><td>2.24 </td><td>≫ </td><td>2.14 </td></tr><tr><td>Met </td><td>AUG </td><td>1.00 </td><td>– </td><td>1.00 </td><td> </td><td>ACG </td><td>0.19 </td><td>≫ </td><td>0.12 </td></tr><tr><td>Asn </td><td>AAU </td><td>1.66 </td><td>≫ </td><td>1.49 </td><td>Ser </td><td>AGU </td><td>1.58 </td><td>≫ </td><td>1.24 </td></tr><tr><td> </td><td>AAC </td><td>0.34 </td><td>≪ </td><td>0.51 </td><td> </td><td>AGC </td><td>0.36 </td><td>≪ </td><td>0.45 </td></tr><tr><td>Lys </td><td>AAA </td><td>1.47 </td><td>≪ </td><td>1.76 </td><td>Arg </td><td>AGA </td><td>1.88 </td><td>≫ </td><td>1.75 </td></tr><tr><td> </td><td>AAG </td><td>0.53 </td><td>≫ </td><td>0.24 </td><td> </td><td>AGG </td><td>0.63 </td><td>≫ </td><td>0.34 </td></tr><tr><td>Val </td><td>GUU </td><td>1.93 </td><td>≫ </td><td>1.82 </td><td>Ala </td><td>GCU </td><td>1.85 </td><td>– </td><td>1.84 </td></tr><tr><td> </td><td>GUC </td><td>0.29 </td><td>≪ </td><td>0.47 </td><td> </td><td>GCC </td><td>0.25 </td><td>≪ </td><td>0.36 </td></tr><tr><td> </td><td>GUA </td><td>1.40 </td><td>≪ </td><td>1.51 </td><td> </td><td>GCA </td><td>1.78 </td><td>≫ </td><td>1.73 </td></tr><tr><td> </td><td>GUG </td><td>0.37 </td><td>≫ </td><td>0.20 </td><td> </td><td>GCG </td><td>0.12 </td><td>– </td><td>0.08 </td></tr><tr><td>Asp </td><td>GAU </td><td>1.73 </td><td>≫ </td><td>1.60 </td><td>Gly </td><td>GGU </td><td>1.67 </td><td>≫ </td><td>1.41 </td></tr><tr><td> </td><td>GAC </td><td>0.27 </td><td>≪ </td><td>0.40 </td><td> </td><td>GGC </td><td>0.32 </td><td>≪ </td><td>0.41 </td></tr><tr><td>Glu </td><td>GAA </td><td>1.43 </td><td>≪ </td><td>1.73 </td><td> </td><td>GGA </td><td>1.46 </td><td>≪ </td><td>1.76 </td></tr><tr><td> </td><td>GAG </td><td>0.57 </td><td>≫ </td><td>0.27 </td><td> </td><td>GGG </td><td>0.55 </td><td>≫ </td><td>0.42 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>≫ indicates that the leading strand genes use the codon more frequently than the lagging strand genes; ≪ indicates the lagging strand genes use the codon more frequently than the leading strand genes; – indicates that there is no significant difference in the usage of the codon on either strand. Significance is examined at the level of 5%.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87195381" target="_blank">View Large</a></div></div><p>Furthermore, there are more annotated genes on the leading strands than on the lagging strands. The numbers are 607 and 573, respectively. However, the two numbers in <em>L</em>. <em>intracellularis</em> are less different than that in <em>B</em>. <em>burgdorferi</em>.<sup><a reveal-id="DSP001C10" data-open="DSP001C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Therefore, the effect of replication selection is weaker than that in <em>B</em>. <em>burgdorferi</em>. Three sets of genes, including ribosomal proteins, translation/transcription processing factors and the major chaperones and degradation genes,<sup><a reveal-id="DSP001C43" data-open="DSP001C43" class="link link-ref link-reveal xref-bibr">43</a></sup> are chosen as representative of highly expressed genes. For these putative highly expressed genes, the distribution on the two replicating strands is much more skewed. More than 59% of the 61 putative highly expressed genes are transcribed on the leading strands. So, the differences between the leading and lagging strands indicate the combined effects of mutation and selection induced by the replication–transcription. Replicational selection, although weak, may be responsible for the higher number of genes located on the leading strands,<sup><a reveal-id="DSP001C10" data-open="DSP001C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSP001C11" data-open="DSP001C11" class="link link-ref link-reveal xref-bibr">11</a></sup> and transcriptional selection appears to be responsible for the enrichment of highly expressed genes on these strands.<sup><a reveal-id="DSP001C10" data-open="DSP001C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSP001C11" data-open="DSP001C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Replicational–transcriptional selection coupled with asymmetric mutational bias is, therefore, the most important cause of intra-chromosome variations of synonymous codon usages in <em>L</em>. <em>intracellularis</em>.</p><p>As mentioned above, the separated codon usages between the two replicating strands have been found in nine bacteria. Among these species, <em>B</em>. <em>burgdorferi</em> shows an extremely strong bias of codon usage.<sup><a reveal-id="DSP001C10" data-open="DSP001C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Lobry and Sueoka<sup><a reveal-id="DSP001C22" data-open="DSP001C22" class="link link-ref link-reveal xref-bibr">22</a></sup> once described a method of graphical display of the influence of replication bias on leading versus lagging strands. It would be interesting to compare the graphics, namely PR2-plots, obtained for <em>L</em>. <em>intracellularis</em> chromosome with those of <em>B</em>. <em>burgdorferi</em>. As can be seen from Fig. <a reveal-id="DSP001F4" data-open="DSP001F4" class="link link-reveal link-table xref-fig">4</a>(a) and (b), the strand biases of G/C, which could be reflected by the values of the horizontal axis of the plot, are slightly weaker in <em>L</em>. <em>intracellularis</em> than those in <em>B</em>. <em>burgdorferi</em>, whereas the strand biases of T/A (reflected by the values of the vertical axis) are much weaker in the former than those in the latter. In both figures, the strand-specific biases are strong enough to separate the genes on the two replicating strands.</p>                        <a id="87195384" scrollto-destination="87195384"></a>
<div id="DSP001F4" data-id="DSP001F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/m_dsp00104.jpeg?Expires=2147483647&Signature=1S5p4yoXQaU6fZpJmCrKsvu41W2tR3Pc9v5hZczwbWXB5k02JrMcyOA9gZ3uAu1aNOCdnpMKv9GxfyP2GcBJJ-3~uWKdTTGC6N-p5WI-rNIL9T-Uu3Xt1DqEZetdsZhtVkRfzA24Tr~2rxfmvVhHoD49yhjIEhy1J64V7hTufcnQuOVPPEmGwGBcZSNNGcndnKyn2RGJ3A0m3G0sSoDohjGgRfaV4y4bQcCWvgRKDL9-yj0WO-cj1t3t3vBdiqNfRcqepeLOSLtRcf7hBj258SM6E6Mv6AoAcaIhEtiSjiMuu9UR5~n~0KVN4QS-CSbOb5s2QVI~l5FdswAHTlrb1Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="PR2-plots for L. intracellularis chromosome and B. burgdorferi genome. G/(G + C) and A/(A + T) at the third codon positions of protein-coding genes are calculated and plotted as the horizontal and vertical axes, respectively. For details of the PR2-plot, please refer to Lobry and Sueoka.22" path-from-xml="dsp00104.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195384/dsp00104.jpeg" path-from-xml="dsp00104.jpeg" target="_blank">View large</a><a section="87195384" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/dsp00104.jpeg?Expires=2147483647&Signature=NKDb4MezLTYqpEOzfPKs9~4wcQu0F3Y6kcBYhedOkTy4iwNsVG33QxwcTpn1gnP7IVXELtjPE3Xv0Y4naym2eKaPD4pBQOJFEXVNhcacsu0k3XDP2gC2MnAe7Awi0Tp6UjFTyWqI9QpObOaT-TiCkZ493UlZ5z4gS6MVjhxOluEiWIStuzPYV71hmM5cMFjj8p8CemSmHO2byyHRMJPli1UvpdvknOSufLVY4dh0nKCMZA5t9zW93pn7424dvRXlGMOTYigTWqTI3AGdaY3QvQVaOCJ1uOEcGhZ3RsiXDaKwYMBqEeU5iAiXm5N-E6aWY8-h5NaDU4j5W4bYpB7otw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195384&ar=510591&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00104.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>PR2-plots for <em>L</em>. <em>intracellularis</em> chromosome and <em>B</em>. <em>burgdorferi</em> genome. G/(G + C) and A/(A + T) at the third codon positions of protein-coding genes are calculated and plotted as the horizontal and vertical axes, respectively. For details of the PR2-plot, please refer to Lobry and Sueoka.<sup><a reveal-id="DSP001C22" data-open="DSP001C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p></div></div><div content-id="DSP001F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/m_dsp00104.jpeg?Expires=2147483647&Signature=1S5p4yoXQaU6fZpJmCrKsvu41W2tR3Pc9v5hZczwbWXB5k02JrMcyOA9gZ3uAu1aNOCdnpMKv9GxfyP2GcBJJ-3~uWKdTTGC6N-p5WI-rNIL9T-Uu3Xt1DqEZetdsZhtVkRfzA24Tr~2rxfmvVhHoD49yhjIEhy1J64V7hTufcnQuOVPPEmGwGBcZSNNGcndnKyn2RGJ3A0m3G0sSoDohjGgRfaV4y4bQcCWvgRKDL9-yj0WO-cj1t3t3vBdiqNfRcqepeLOSLtRcf7hBj258SM6E6Mv6AoAcaIhEtiSjiMuu9UR5~n~0KVN4QS-CSbOb5s2QVI~l5FdswAHTlrb1Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="PR2-plots for L. intracellularis chromosome and B. burgdorferi genome. G/(G + C) and A/(A + T) at the third codon positions of protein-coding genes are calculated and plotted as the horizontal and vertical axes, respectively. For details of the PR2-plot, please refer to Lobry and Sueoka.22" path-from-xml="dsp00104.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195384/dsp00104.jpeg" path-from-xml="dsp00104.jpeg" target="_blank">View large</a><a section="87195384" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/dsp00104.jpeg?Expires=2147483647&Signature=NKDb4MezLTYqpEOzfPKs9~4wcQu0F3Y6kcBYhedOkTy4iwNsVG33QxwcTpn1gnP7IVXELtjPE3Xv0Y4naym2eKaPD4pBQOJFEXVNhcacsu0k3XDP2gC2MnAe7Awi0Tp6UjFTyWqI9QpObOaT-TiCkZ493UlZ5z4gS6MVjhxOluEiWIStuzPYV71hmM5cMFjj8p8CemSmHO2byyHRMJPli1UvpdvknOSufLVY4dh0nKCMZA5t9zW93pn7424dvRXlGMOTYigTWqTI3AGdaY3QvQVaOCJ1uOEcGhZ3RsiXDaKwYMBqEeU5iAiXm5N-E6aWY8-h5NaDU4j5W4bYpB7otw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195384&ar=510591&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00104.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>PR2-plots for <em>L</em>. <em>intracellularis</em> chromosome and <em>B</em>. <em>burgdorferi</em> genome. G/(G + C) and A/(A + T) at the third codon positions of protein-coding genes are calculated and plotted as the horizontal and vertical axes, respectively. For details of the PR2-plot, please refer to Lobry and Sueoka.<sup><a reveal-id="DSP001C22" data-open="DSP001C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p></div></div>                            <h3 scrollto-destination=87195385 id="87195385" class="section-title" data-legacy-id=s3d>3.4. The second trend may be associated with horizontal gene transfer</h3>
<p>When analyzing the second trend, it is found that there exists a strong negative correlation between the Axis 2 (COA/RSCU) and GC3<sub>S</sub> (<em>r</em> = −0.6559). If analysis is restricted to the 87 genes whose coordinates along Axis 2 are less than –0.2, the correlation is more significant. Marking these genes by open triangles in the <em>N</em><sub>C</sub>–GC3<sub>S</sub> plot, it is found that these genes are located far from the other ones and have higher <em>N</em><sub>C</sub> and GC3<sub>S</sub>. And this fact means the different codon usage of these genes with that of the majority. As well known, genes that were recently imported through horizontal transfer would have unusual codon usages and base compositions. In <em>Pseudomonas aeruginosa</em>, putative alien genes showed higher <em>N</em><sub>C</sub> values than the majority of genes.<sup><a reveal-id="DSP001C12" data-open="DSP001C12" class="link link-ref link-reveal xref-bibr">12</a></sup> From the widely used public database HGT-DB,<sup><a reveal-id="DSP001C14" data-open="DSP001C14" class="link link-ref link-reveal xref-bibr">14</a></sup> we downloaded the information for all the predicted horizontally transferred genes in <em>L</em>. <em>intracellularis</em>. Among the 10 genes that have the lowest values of Axis 2, nine are predicted as horizontally transferred genes according to the results in HGT-DB.<sup><a reveal-id="DSP001C14" data-open="DSP001C14" class="link link-ref link-reveal xref-bibr">14</a></sup> The exceptional gene is as short as 104 codons. Based on the above analysis, it is reasonable to come to the conclusion that most of the 87 genes may be transferred horizontally. It should be noted that 20 of these genes are located around the replication terminus, which has been shown to be a hot spot of mutation and chromosome recombination.</p>                            <h3 scrollto-destination=87195387 id="87195387" class="section-title" data-legacy-id=s3e>3.5. Hydropathy of encoded proteins influences the codon usage variation</h3>
<p>Fig. <a reveal-id="DSP001F5" data-open="DSP001F5" class="link link-reveal link-table xref-fig">5</a> shows the positions of the 1180 genes along the third and forth major axes produced by COA on codon counts. As can be seen from the figure, the third axis separates another group of genes from the main group. After computing the gravy score (a measure of hydrophobicity) of all the proteins encoded by the genes, it is found that the proteins separated from the others on the third axis have very high scores (gravy &gt; 0.3) and so are hydrophobic. These results are nothing but the superimposition of amino acid bias on codon usage bias. If we use RSCU to compute the COA, the third major axis could not discriminate the genes according to the hydrophobicity values. It should be noted that this is the same case as that in the genome of <em>Lactococcus lactis</em>.<sup><a reveal-id="DSP001C44" data-open="DSP001C44" class="link link-ref link-reveal xref-bibr">44</a></sup></p>                        <a id="87195389" scrollto-destination="87195389"></a>
<div id="DSP001F5" data-id="DSP001F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/m_dsp00105.jpeg?Expires=2147483647&Signature=cM1fcxNCQd9QYsKVnuShrASyPonI9ktRCGhfqd6nhl9HvV6QXHYunpsEdAc2vVqVLfywEwyPqaaRg6uLiysQYzDsUn9fNKbt1koPSHiUswL-HsSgndLzAitd5BTEFB~IWPuyz6L4uCB-aT2NVcV3JERcephtlj0qXYiBA8cp3P0ZIihHus5nX9mYKXEPlARxiPaTSa~riZRVcK0KW19Vse5II03hux6cUJ1wc-Cxl76RhBL2najnqTB7l09orbCiy0MquJ6c9JGoKiG0aTU~jKT1JbIklzkwU0igM7qBFWeqDSIX9Vdr-zP6CKdiUvOMLIUOyZMquTLJFtlXfTPPeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Plot of the third and fourth axes after COA of codon counts for the 1180 genes on the chromosome. Proteins having a gravy score &gt;0.3 are represented as filled circles and the other genes are represented as open circles." path-from-xml="dsp00105.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195389/dsp00105.jpeg" path-from-xml="dsp00105.jpeg" target="_blank">View large</a><a section="87195389" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/dsp00105.jpeg?Expires=2147483647&Signature=a6hlyeBYBbmE9VXzsPmL1Q3SswisdmVXpy2GtyXzjZoDPMiZKP7iASx-ZaFeO9CJXqGS~CyUb1zXnvAysnz1BcJUEvp4BCF~R3ZA6fhng~5r8XlwyoWBtAfhlgwr2PNVizraxZrNz-H44W92PtNC7Akf3yZSo52esTpvtmbpzdNIh7nc51RleuI-VFb0zlLt1lQCm3nNjPmMMqk0WlSGhIwzXFgwGmIBHSmi~azCKhg4JdMGTSuX2KADV7ts8HQTzncN0wIYZamkz1StFoCqeraQd0g~vwfyt5o4xmil7W6jvzgvMPLvoe7W1jqMZGGq0Zt9cxeL6Nr-M~RHa7NLBQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195389&ar=510591&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00105.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Plot of the third and fourth axes after COA of codon counts for the 1180 genes on the chromosome. Proteins having a gravy score &gt;0.3 are represented as filled circles and the other genes are represented as open circles.</p></div></div><div content-id="DSP001F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/m_dsp00105.jpeg?Expires=2147483647&Signature=cM1fcxNCQd9QYsKVnuShrASyPonI9ktRCGhfqd6nhl9HvV6QXHYunpsEdAc2vVqVLfywEwyPqaaRg6uLiysQYzDsUn9fNKbt1koPSHiUswL-HsSgndLzAitd5BTEFB~IWPuyz6L4uCB-aT2NVcV3JERcephtlj0qXYiBA8cp3P0ZIihHus5nX9mYKXEPlARxiPaTSa~riZRVcK0KW19Vse5II03hux6cUJ1wc-Cxl76RhBL2najnqTB7l09orbCiy0MquJ6c9JGoKiG0aTU~jKT1JbIklzkwU0igM7qBFWeqDSIX9Vdr-zP6CKdiUvOMLIUOyZMquTLJFtlXfTPPeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Plot of the third and fourth axes after COA of codon counts for the 1180 genes on the chromosome. Proteins having a gravy score &gt;0.3 are represented as filled circles and the other genes are represented as open circles." path-from-xml="dsp00105.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195389/dsp00105.jpeg" path-from-xml="dsp00105.jpeg" target="_blank">View large</a><a section="87195389" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/dsp00105.jpeg?Expires=2147483647&Signature=a6hlyeBYBbmE9VXzsPmL1Q3SswisdmVXpy2GtyXzjZoDPMiZKP7iASx-ZaFeO9CJXqGS~CyUb1zXnvAysnz1BcJUEvp4BCF~R3ZA6fhng~5r8XlwyoWBtAfhlgwr2PNVizraxZrNz-H44W92PtNC7Akf3yZSo52esTpvtmbpzdNIh7nc51RleuI-VFb0zlLt1lQCm3nNjPmMMqk0WlSGhIwzXFgwGmIBHSmi~azCKhg4JdMGTSuX2KADV7ts8HQTzncN0wIYZamkz1StFoCqeraQd0g~vwfyt5o4xmil7W6jvzgvMPLvoe7W1jqMZGGq0Zt9cxeL6Nr-M~RHa7NLBQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195389&ar=510591&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00105.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Plot of the third and fourth axes after COA of codon counts for the 1180 genes on the chromosome. Proteins having a gravy score &gt;0.3 are represented as filled circles and the other genes are represented as open circles.</p></div></div>                            <h3 scrollto-destination=87195390 id="87195390" class="section-title" data-legacy-id=s3f>3.6. Influence of the expression level on the codon usage</h3>
<p>In order to investigate whether codon usage patterns are further shaped by factors at the level of translation, we conduct COA of codon counts and RSCU values on the genes located on the leading strands of replication, because most of the 61 putative expressed genes are located in that strand. Fig. <a reveal-id="DSP001F6" data-open="DSP001F6" class="link link-reveal link-table xref-fig">6</a>(a) and (b) shows the positions of the genes along the first and second major axes produced by such COA on codon counts and RSCU, respectively. As can be seen from the two figures, almost all of the putative highly expressed genes have positive scores along the second axis, whereas genes presumably expressed at low level are scattered everywhere of the whole distribution area. In addition, we calculate the CAI value for each gene located on the lagging strands. The correlation between the values of Axis 2 and CAI is statistically significant (<em>r</em> = 0.494, <em>P</em> &lt; 0.0001). The above analyses suggest that the second axis is associated with expression level.</p>                        <a id="87195392" scrollto-destination="87195392"></a>
<div id="DSP001F6" data-id="DSP001F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/m_dsp00106.jpeg?Expires=2147483647&Signature=g7ncNq8t9ME7kFCd4HYT~TjbIE5d17~D8N~plZZiDMsgkyiLqQqZSnoTJoy4Ebo8QqMAqH1USQj06wiPWGgcxVRWLmZREWnrw9Y0sFGSZUy1mSZTg0Dn6QVtc6oIo9dlo9HAoWOFk7O4ND7HesRVXVW17-BtuIb8vUZ2OPA0LbRKMk67OoltXk8Shv6Cyh8TJMIugqGIAyl4oDjdf7IdlCJufTAqP-i~AuvCVc0G6iB333z2rD5JpHf23QuCaM-D-1VYuyzTL~vjJvW6pbCLYBOQ0BzdUNBRNfXDSajk7UUjEbIQf9j2AWHTxAxnMB72RZT2aWbDGnEtdl-51OLC-w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Plot of the two most important axes after COA for 607 chromosome genes that are located on the leading strands of replication. The filled circles indicate 36 putative highly expressed genes. The open circles denote other genes. (a) Use codon count to compute COA (COA/counts), the first and second axes account for 12.7% and 7.2% of the total inertia of the 59-dimensional space. (b) Use RSCU to compute COA (COA/RSCU), the first and second axes account for 13.5% and 6.3% of the total inertia of the 59-dimensional space." path-from-xml="dsp00106.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195392/dsp00106.jpeg" path-from-xml="dsp00106.jpeg" target="_blank">View large</a><a section="87195392" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/dsp00106.jpeg?Expires=2147483647&Signature=zN0F0Ea-NE4bEhljWPYbsRAhxCBEpSTN0B6dUVKnUxb0I8LUY-hD2KDxXyMg~mbAKl2jxv9UjsYKQssueMmRLH~NlfK-gb0h7nwzKMWPPh7Wxanp4z70AZdJTmxFr39PSkXIFBRJQBdSK3MNbvQYlammZQY5P9o9xiIAAJnRdqzjRRomNjR0SEurhO918VGIBaVG~9aaJsQq6XcqLh9mseyLh6t~fVpZoACYHBb2Tgr9R62A34T4Q9FuOb2Vmi1n46WdA1jdJLKlL5DtCrbFbNUN2K6xvnWf9XPEPFuvOBb1bSo5vDjTWyOQ8NdRVAsPfouHAfquEEmo35kUnGlbZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195392&ar=510591&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00106.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Plot of the two most important axes after COA for 607 chromosome genes that are located on the leading strands of replication. The filled circles indicate 36 putative highly expressed genes. The open circles denote other genes. (<strong>a</strong>) Use codon count to compute COA (COA/counts), the first and second axes account for 12.7% and 7.2% of the total inertia of the 59-dimensional space. (<strong>b</strong>) Use RSCU to compute COA (COA/RSCU), the first and second axes account for 13.5% and 6.3% of the total inertia of the 59-dimensional space.</p></div></div><div content-id="DSP001F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/m_dsp00106.jpeg?Expires=2147483647&Signature=g7ncNq8t9ME7kFCd4HYT~TjbIE5d17~D8N~plZZiDMsgkyiLqQqZSnoTJoy4Ebo8QqMAqH1USQj06wiPWGgcxVRWLmZREWnrw9Y0sFGSZUy1mSZTg0Dn6QVtc6oIo9dlo9HAoWOFk7O4ND7HesRVXVW17-BtuIb8vUZ2OPA0LbRKMk67OoltXk8Shv6Cyh8TJMIugqGIAyl4oDjdf7IdlCJufTAqP-i~AuvCVc0G6iB333z2rD5JpHf23QuCaM-D-1VYuyzTL~vjJvW6pbCLYBOQ0BzdUNBRNfXDSajk7UUjEbIQf9j2AWHTxAxnMB72RZT2aWbDGnEtdl-51OLC-w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Plot of the two most important axes after COA for 607 chromosome genes that are located on the leading strands of replication. The filled circles indicate 36 putative highly expressed genes. The open circles denote other genes. (a) Use codon count to compute COA (COA/counts), the first and second axes account for 12.7% and 7.2% of the total inertia of the 59-dimensional space. (b) Use RSCU to compute COA (COA/RSCU), the first and second axes account for 13.5% and 6.3% of the total inertia of the 59-dimensional space." path-from-xml="dsp00106.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195392/dsp00106.jpeg" path-from-xml="dsp00106.jpeg" target="_blank">View large</a><a section="87195392" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/dsp00106.jpeg?Expires=2147483647&Signature=zN0F0Ea-NE4bEhljWPYbsRAhxCBEpSTN0B6dUVKnUxb0I8LUY-hD2KDxXyMg~mbAKl2jxv9UjsYKQssueMmRLH~NlfK-gb0h7nwzKMWPPh7Wxanp4z70AZdJTmxFr39PSkXIFBRJQBdSK3MNbvQYlammZQY5P9o9xiIAAJnRdqzjRRomNjR0SEurhO918VGIBaVG~9aaJsQq6XcqLh9mseyLh6t~fVpZoACYHBb2Tgr9R62A34T4Q9FuOb2Vmi1n46WdA1jdJLKlL5DtCrbFbNUN2K6xvnWf9XPEPFuvOBb1bSo5vDjTWyOQ8NdRVAsPfouHAfquEEmo35kUnGlbZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195392&ar=510591&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00106.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Plot of the two most important axes after COA for 607 chromosome genes that are located on the leading strands of replication. The filled circles indicate 36 putative highly expressed genes. The open circles denote other genes. (<strong>a</strong>) Use codon count to compute COA (COA/counts), the first and second axes account for 12.7% and 7.2% of the total inertia of the 59-dimensional space. (<strong>b</strong>) Use RSCU to compute COA (COA/RSCU), the first and second axes account for 13.5% and 6.3% of the total inertia of the 59-dimensional space.</p></div></div>                            <h3 scrollto-destination=87195393 id="87195393" class="section-title" data-legacy-id=s3g>3.7. COA of RSCU of genes in the largest plasmid</h3>
<p>COA of RSCU of genes in the largest plasmid in <em>L</em>. <em>intracellularis</em> was performed to determine the most important factor that drives synonymous codon usage patterns. The other two plasmids are not analyzed because there are too little genes to perform multivariate statistical analysis. One plot of the two most important axes after the COA is shown in Fig. <a reveal-id="DSP001F7" data-open="DSP001F7" class="link link-reveal link-table xref-fig">7</a>. The first and second axes account for 12.7% and 8.2% of the total inertia of the 59-dimensional space, respectively. After determining replication origin and terminus sites by GC-skew analysis shown in Fig. <a reveal-id="DSP001F8" data-open="DSP001F8" class="link link-reveal link-table xref-fig">8</a>, 68 genes are found to lie on the leading strands and 36 ones on the lagging strands. There is more significant difference between the numbers of genes on the two replicating strands in the plasmid than those on the chromosome, suggesting that replicational selection in the former is stronger than that in the latter. Marking the two groups of genes by open and filled circles, and then it is found that genes on the leading strands lie on the right side of the Axis 1, whereas lagging strand genes lie on the left side. <em>χ</em><sup>2</sup> test was performed for RSCU of genes located on the leading versus lagging strands in this plasmid. Consequently, 23 among 59 codons are found to be significantly different between genes on the two strands of replication. Among the 13 codons used more frequently in the leading strands, nine are G-ending or T-ending, except TTA, ATA, CCA and GCA. Among the 10 codons used more frequently in the lagging strands, eight are C-ending or A-ending and the exceptions are CTT and TCT. Therefore, the strand-specific mutational biases are responsible for the major variation of synonymous codon usages of genes in the plasmid. No known factors are found to correlate with the second axis of this COA.</p>                        <a id="87195395" scrollto-destination="87195395"></a>
<div id="DSP001F7" data-id="DSP001F7" class="fig fig-section"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/m_dsp00107.jpeg?Expires=2147483647&Signature=cZMhhjyLH9JgoVbcjJVn7qvtdZR0olkEYAho3sQuY1737BMfckGcMTffUpm2g02~~GRrXWkECkUolUA9-wc70-TAvpOBNr4~FW7J~oNmglcbIz8yqSj8YD-oOUb4YyF8KlD0-b7N8LQSZJoMwNEd6~Zqsky765zCytN6MLHOP7ZKbpe1VYQbGAses203dFCcMx9Q6QKdgg8ad1r0MgGrkyMfCOK7MfuwtH89RAFKerSss2LaqYs2R8w73b1O0vLETVExh7S5ReARM0ZbImd9PNNCelKoPbeZ4epukO~pVhOyqlPahyaW~iXIdms0UKEAxp4kQI9wLrnlKmLPb2OVTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Plot of the two most important axes after COA of RSCU values for the 104 genes in the largest plasmid. The open circles indicate the genes that are transcribed on the leading strands of replication. The filled circles indicate the genes that are transcribed on the lagging strands of the replication." path-from-xml="dsp00107.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195395/dsp00107.jpeg" path-from-xml="dsp00107.jpeg" target="_blank">View large</a><a section="87195395" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/dsp00107.jpeg?Expires=2147483647&Signature=03vCbUuo9aTHKlKyX3ckwHB3sO09WvZtaaTlwcK6D3TL92ZzCtodgmVaXxIdQwx5XSbeHYw~5tOAB5lZgXok8YSevvZi2CjHLp4ogEc5seCyBz4qf5BHgb0ON~ZV0Z8V~Zheapqtc~LOG-ZrBLiO96SxCAIRQi32I4ay58ntoBTR1pt86sp0mIymjsguJABc5BV~YpoHgwGX9rW0F69QdE8BtXVFAgtEhOee~pBkyLGDbGooAdGg85bpzNjMz5RARVS6Tl5HWZFzhNMJq2-XhdXWTKjjyR8qzgZbYruqKDmmNIbnjNtzqc9f96pwj0kCtFWsvNNr7bNDptanNsoiqw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195395&ar=510591&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00107.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Plot of the two most important axes after COA of RSCU values for the 104 genes in the largest plasmid. The open circles indicate the genes that are transcribed on the leading strands of replication. The filled circles indicate the genes that are transcribed on the lagging strands of the replication.</p></div></div><div content-id="DSP001F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/m_dsp00107.jpeg?Expires=2147483647&Signature=cZMhhjyLH9JgoVbcjJVn7qvtdZR0olkEYAho3sQuY1737BMfckGcMTffUpm2g02~~GRrXWkECkUolUA9-wc70-TAvpOBNr4~FW7J~oNmglcbIz8yqSj8YD-oOUb4YyF8KlD0-b7N8LQSZJoMwNEd6~Zqsky765zCytN6MLHOP7ZKbpe1VYQbGAses203dFCcMx9Q6QKdgg8ad1r0MgGrkyMfCOK7MfuwtH89RAFKerSss2LaqYs2R8w73b1O0vLETVExh7S5ReARM0ZbImd9PNNCelKoPbeZ4epukO~pVhOyqlPahyaW~iXIdms0UKEAxp4kQI9wLrnlKmLPb2OVTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Plot of the two most important axes after COA of RSCU values for the 104 genes in the largest plasmid. The open circles indicate the genes that are transcribed on the leading strands of replication. The filled circles indicate the genes that are transcribed on the lagging strands of the replication." path-from-xml="dsp00107.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195395/dsp00107.jpeg" path-from-xml="dsp00107.jpeg" target="_blank">View large</a><a section="87195395" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/dsp00107.jpeg?Expires=2147483647&Signature=03vCbUuo9aTHKlKyX3ckwHB3sO09WvZtaaTlwcK6D3TL92ZzCtodgmVaXxIdQwx5XSbeHYw~5tOAB5lZgXok8YSevvZi2CjHLp4ogEc5seCyBz4qf5BHgb0ON~ZV0Z8V~Zheapqtc~LOG-ZrBLiO96SxCAIRQi32I4ay58ntoBTR1pt86sp0mIymjsguJABc5BV~YpoHgwGX9rW0F69QdE8BtXVFAgtEhOee~pBkyLGDbGooAdGg85bpzNjMz5RARVS6Tl5HWZFzhNMJq2-XhdXWTKjjyR8qzgZbYruqKDmmNIbnjNtzqc9f96pwj0kCtFWsvNNr7bNDptanNsoiqw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195395&ar=510591&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00107.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Plot of the two most important axes after COA of RSCU values for the 104 genes in the largest plasmid. The open circles indicate the genes that are transcribed on the leading strands of replication. The filled circles indicate the genes that are transcribed on the lagging strands of the replication.</p></div></div>                        <a id="87195396" scrollto-destination="87195396"></a>
<div id="DSP001F8" data-id="DSP001F8" class="fig fig-section"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/m_dsp00108.jpeg?Expires=2147483647&Signature=kkbOCmF2CcgdqSKQqM5KPbO-16d3vKpqwU85jPQa4TMxy8lIv4ZjfgeQbyjOKDIM9mBpmmtqtflgiXOCqEEqGnV13Id33pfGFPXYXKVH3T~WJtmza~vF8V8P2wgi61tGKNDgCh4wYmxYRRk42X79-dxUutpzlw3l7LYOrnFltAjKk1WxlHuM4eDCwxK-kdTsfnK05woKxWi1WEomuRxK0cD9Co79WyQmzSdn187hDOnZWYMtaUB4RYRK46hUAgnlIeHgV8~ReINN~ue3oELuVkPNDB-LxXUAPCkz6UCP1bilE~Yx-i1b7S~1ndGgH01joUtbTBEtLauqIILm1BNjGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GC-skew along the DNA sequence of the largest plasmid. It is calculated using a non-overlapping sliding window of 1000 bp. As can be seen, there are two clear polarity switches, suggested as putative origin and terminus of replication." path-from-xml="dsp00108.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195396/dsp00108.jpeg" path-from-xml="dsp00108.jpeg" target="_blank">View large</a><a section="87195396" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/dsp00108.jpeg?Expires=2147483647&Signature=B7ZND3BOawkFnPntLw4Jea8FP8JdZ04naq6EStYw8dnvfyI3f9kn4cEFvmDauCaqVFICQ15y4nHE8awZcgF~j2dqz4WnOYi5a6IfvjG8aPm5zyz6FXL6vb7OIg9wyuRldmNyccW6aYvl8z05NjtERxvT-zBD4aUNNpTucuX4zwW92riRh7pW3C9ywNpPFepD2znQrMKBWdpgZb1K5nxNxA3Jk5Kib5PNHewfCy51yJSZCXNu7AdF3v9sLM149TERfYY98FoUvlr-JbcuekPsF9r2M~1xiPg0CeMv8VN31bmN2R1NHLsFE~vOnEWCZADmKxYHS9yvbY46vSALst7~bA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195396&ar=510591&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00108.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GC-skew along the DNA sequence of the largest plasmid. It is calculated using a non-overlapping sliding window of 1000 bp. As can be seen, there are two clear polarity switches, suggested as putative origin and terminus of replication.</p></div></div><div content-id="DSP001F8" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 8</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/m_dsp00108.jpeg?Expires=2147483647&Signature=kkbOCmF2CcgdqSKQqM5KPbO-16d3vKpqwU85jPQa4TMxy8lIv4ZjfgeQbyjOKDIM9mBpmmtqtflgiXOCqEEqGnV13Id33pfGFPXYXKVH3T~WJtmza~vF8V8P2wgi61tGKNDgCh4wYmxYRRk42X79-dxUutpzlw3l7LYOrnFltAjKk1WxlHuM4eDCwxK-kdTsfnK05woKxWi1WEomuRxK0cD9Co79WyQmzSdn187hDOnZWYMtaUB4RYRK46hUAgnlIeHgV8~ReINN~ue3oELuVkPNDB-LxXUAPCkz6UCP1bilE~Yx-i1b7S~1ndGgH01joUtbTBEtLauqIILm1BNjGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GC-skew along the DNA sequence of the largest plasmid. It is calculated using a non-overlapping sliding window of 1000 bp. As can be seen, there are two clear polarity switches, suggested as putative origin and terminus of replication." path-from-xml="dsp00108.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87195396/dsp00108.jpeg" path-from-xml="dsp00108.jpeg" target="_blank">View large</a><a section="87195396" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/dsp00108.jpeg?Expires=2147483647&Signature=B7ZND3BOawkFnPntLw4Jea8FP8JdZ04naq6EStYw8dnvfyI3f9kn4cEFvmDauCaqVFICQ15y4nHE8awZcgF~j2dqz4WnOYi5a6IfvjG8aPm5zyz6FXL6vb7OIg9wyuRldmNyccW6aYvl8z05NjtERxvT-zBD4aUNNpTucuX4zwW92riRh7pW3C9ywNpPFepD2znQrMKBWdpgZb1K5nxNxA3Jk5Kib5PNHewfCy51yJSZCXNu7AdF3v9sLM149TERfYY98FoUvlr-JbcuekPsF9r2M~1xiPg0CeMv8VN31bmN2R1NHLsFE~vOnEWCZADmKxYHS9yvbY46vSALst7~bA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87195396&ar=510591&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsp00108.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GC-skew along the DNA sequence of the largest plasmid. It is calculated using a non-overlapping sliding window of 1000 bp. As can be seen, there are two clear polarity switches, suggested as putative origin and terminus of replication.</p></div></div><p>To test whether the transcriptional selection exerts influence on the genes, CAI values are calculated for 104 genes in the plasmid by using ribosomal protein-coding genes as reference set. Consequently, it is found that all of the seven genes that have the highest CAI values are located on the leading strands. This suggests that transcriptional selection do have influence on the genes of the plasmid. Replicational selection and transcriptional selection are two different kinds of selective pressures. Although the two selective pressures yield similar consequences, they are very distinct.<sup><a reveal-id="DSP001C10" data-open="DSP001C10" class="link link-ref link-reveal xref-bibr">10</a></sup> The asymmetric mechanism of replication in the plasmid may also be shown by the following result. CMR database at TIGR lists the function categories of known genes in this plasmid. After comparing the numbers of the known genes on the two replicating strands, it is found that 15 of the 22 cell envelope-related genes are located on the lagging strands. On the other hand, 10 of the 11 genes with mobile and extra-chromosomal element functions are located on the leading strands.</p>                            <h2 scrollto-destination=87195398 id="87195398" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=87195399 id="87195399" class="section-title" data-legacy-id=s4a>4.1. Underlying mechanisms of replicational–transcriptional mutation and selection</h3>
<p>It is widely accepted that codon usage variation in a species is the combined effect of mutation bias and nature selection.<sup><a reveal-id="DSP001C12" data-open="DSP001C12" class="link link-ref link-reveal xref-bibr">12</a></sup> In <em>L</em>. <em>intracellularis</em> chromosome, genes located on the two replicating strands are shown to have distinct codon usages. On the other hand, genes, particularly those are highly expressed, are mainly located on the leading strands. It is important to investigate the underlying mechanisms of the two effects. According to McInerney<sup><a reveal-id="DSP001C10" data-open="DSP001C10" class="link link-ref link-reveal xref-bibr">10</a></sup> and Romero et al.,<sup><a reveal-id="DSP001C11" data-open="DSP001C11" class="link link-ref link-reveal xref-bibr">11</a></sup> the former is caused by the strand mutational bias and the latter results from replication–transcription selection. Also, some researchers believe that the strand-specific compositional biases are not only the result of strand mutation biases but also the superimposition of differential mutation rate and differential correction/repair rates.<sup><a reveal-id="DSP001C45" data-open="DSP001C45" class="link link-ref link-reveal xref-bibr">45</a></sup></p><p>Among the theories aimed at explaining strand mutation biases, it seems that the cytosine deamination theory enjoys the most attention.<sup><a reveal-id="DSP001C19" data-open="DSP001C19" class="link link-ref link-reveal xref-bibr">19</a></sup> The deamination of cytosine results in the formation of uracil. In normal circumstance <em>in vivo</em>, cytosine is effectively protected against deamination because of the Watson–Crick base paring. But the rate of cytosine deamination increases 140 times when the DNA is single-stranded.<sup><a reveal-id="DSP001C46" data-open="DSP001C46" class="link link-ref link-reveal xref-bibr">46</a></sup> If the resulting uracil is not replaced with cytosine, C to T mutation occurs. During the replicating process, the leading strand is much more exposed in the single-stranded state. Therefore, the C to T mutation occurs more frequently in the leading strands than in the lagging strands and then the excesses of G relative to C and T relative to A are formed in the leading strands. According to Furusawa and Doi,<sup><a reveal-id="DSP001C47" data-open="DSP001C47" class="link link-ref link-reveal xref-bibr">47</a></sup> such fidelity difference between the leading and lagging strands may make it possible to accelerate the evolution of unicellular and multicellular organisms and avoid the extinction of the population.</p><p>As far as gene orientation biases are concerned, there exist similar explanations. For genes on the leading strands, RNA polymerase, when transcribing them, moves in the same direction as a replication fork would move during replication, whereas opposite for genes on the lagging strands. Transcription–replication should be more effective if one organism maintains most of its genes, particularly highly expressed genes on the leading strands. The high efficiency results from the three factors: (i) the same direction reduces the probability of head-on collisions between the polymerases involved in the replicational and transcriptional processes; (ii) transcription may not be aborted by the replication complex; and (iii) the inverse orientation is very disadvantageous because of the possible lack of solution mechanism for head-on collisions.<sup><a reveal-id="DSP001C48" data-open="DSP001C48" class="link link-ref link-reveal xref-bibr">48</a></sup> In highly expressed genes, the selective advantage of transposition to the leading strands is more significant than that of lowly expressed genes. Therefore, highly expressed genes are much more likely to overcome random genetic drift, and these genotypes become fixed more easily in the population. Lowly expressed genes do not interfere with replication to such an extent as highly expressed genes, and, also, the interruption of lowly expressed gene transcription is not nearly as deleterious. So, the selective advantage is not so great in lowly expressed genes. Transposition of a lowly expressed gene from a lagging strand to a leading strand may not offer a sufficient selective advantage and therefore may not become fixed so easily in the population.<sup><a reveal-id="DSP001C10" data-open="DSP001C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p>                            <h3 scrollto-destination=87195403 id="87195403" class="section-title" data-legacy-id=s4b>4.2. Codon usages of genes in the plasmid and asymmetric mechanism of replication</h3>
<p>Usually, bacterial plasmids replicate using a different mechanism than that of the chromosome of their host cell. As an exception, cumulative skew diagrams showed that plasmid and chromosome of <em>B</em>. <em>burgdorferi</em> adopted a similar bi-directional replication.<sup><a reveal-id="DSP001C49" data-open="DSP001C49" class="link link-ref link-reveal xref-bibr">49</a></sup> Such common replication mechanism was consistent with previous suggestions that <em>Borrelia</em> plasmids were actually mini-chromosomes.<sup><a reveal-id="DSP001C50" data-open="DSP001C50" class="link link-ref link-reveal xref-bibr">50</a></sup> In this work, GC-skew analysis in Fig. <a reveal-id="DSP001F8" data-open="DSP001F8" class="link link-reveal link-table xref-fig">8</a> shows clear polarity switch at two points, around 27 and 115 kb in the largest plasmid of <em>L</em>. <em>intracellularis</em>. This suggests that this plasmid replicates bi-directionally from an internal origin as the chromosome does. Leading strands and lagging strands are hence determined based on the putative origin and terminus. COA shows that genes on the leading and lagging strands have distinct codon usages. The same results are observed in genes on the chromosome, which is supposed to be caused by the strand-specific mutational bias. Similarly, the difference between codon usages of genes on the two replicating strands in the plasmid is very likely to result from the different mutation and/or repair rates. If this speculation is the real case, then common asymmetric replication would be involved in the chromosome and the largest plasmid of <em>L</em>. <em>intracellularis</em>. Not only both replicate bi-directionally from internal origin, but also have biased mutation/repair rate between the two replicating strands.</p><p>Replicational–transcriptional selection is also found to exert pressure in the largest plasmid of <em>L</em>. <em>intracellularis</em>. The fact that most (68/104 = 65.4%) genes are located on the leading strands suggests the existence of replicational selection. In addition, all of the seven genes that have the highest CAI values are located on the leading strands. This suggests that transcriptional selection has influence on the genes of the plasmid. All the above facts suggest that this plasmid adopts the similar mechanism of replication as the chromosome in <em>L</em>. <em>intracellularis</em>. Perhaps, the largest plasmid of <em>L</em>. <em>intracellularis</em> is one mini-chromosome, as that in <em>B</em>. <em>burgdorferi</em>.<sup><a reveal-id="DSP001C50" data-open="DSP001C50" class="link link-ref link-reveal xref-bibr">50</a></sup></p>                            <h3 scrollto-destination=87195406 id="87195406" class="section-title" data-legacy-id=s4c>4.3. The ‘GC-richness’ of the putative alien genes in L. intracellularis</h3>
<p>Over 10 years ago, the tendency of horizontally transferred genes to be A + T-rich had been noted in species having intermediate G + C contents.<sup><a reveal-id="DSP001C51" data-open="DSP001C51" class="link link-ref link-reveal xref-bibr">51</a></sup> After that, the same phenomenon was observed for <em>Helicobacter pylori</em> and <em>Streptococcus pneumoniae</em>,<sup><a reveal-id="DSP001C52" data-open="DSP001C52" class="link link-ref link-reveal xref-bibr">52</a></sup> which have low G + C contents. This striking pattern raises questions about the nature and the source of these horizontally transferred genes. Lawrence and Ochman<sup><a reveal-id="DSP001C53" data-open="DSP001C53" class="link link-ref link-reveal xref-bibr">53</a></sup> hypothesized that the recently transferred genes were adapted to the genomic context of other distant species. The results of Daubin et al.<sup><a reveal-id="DSP001C52" data-open="DSP001C52" class="link link-ref link-reveal xref-bibr">52</a></sup> suggest that either the donor genomes are always more A + T rich than the acceptor genomes or there is a bias toward the internalization of A + T-rich exogenous DNA in the genome.</p><p>However, the putative alien genes in <em>L</em>. <em>intracellularis</em> are found to have higher GC than the other genes. And even, some of the putative alien genes are GC-rich. Perhaps, someone will think this is an exception. But we do not think so. In contrast, we suppose that this may be a usual pattern of bacterial genomes with very low GC contents. Genomic GC contents of <em>H</em>. <em>pylori</em> and <em>S</em>. <em>pneumoniae</em>, in which AT-rich alien genes are found, are low but not very low. Their GC contents are still 6% higher than those of <em>L</em>. <em>intracellularis</em>, which is 33%. For bacteria with GC content low as 33%, it is difficult to obtain a donor species that have higher AT contents. Therefore, alien genes will be inclined to GC-richer than the other genes in the acceptor species.</p>                            <h3 scrollto-destination=87195409 id="87195409" class="section-title" data-legacy-id=s4d>4.4. Common genomic characters of bacteria in which strand-specific mutational biases are strong</h3>
<p>Up to now, the strand-specific mutational bias has been found to be the most important factor that affects codon usage in genomes of 10 bacteria. Names of the 10 bacteria are <em>B</em>. <em>aphidicola</em>, <em>B</em>. <em>burgdorferi</em>, <em>B</em>. <em>floridanus</em>, <em>B</em>. <em>henselae</em>, <em>B</em>. <em>quintana</em>, <em>C</em>. <em>muridarum</em>, <em>C</em>. <em>trachomatis</em>, <em>T</em>. <em>pallidum</em>, <em>T</em>. <em>whipplei</em> and <em>L</em>. <em>intracellularis</em>, which is found in this work. Investigation on the common genomic characters of these bacteria may be useful. Several characters are analyzed in the following sections.</p><p>First, chromosomes of the 10 bacteria are all shorter than 2000 kb. According to statistics on fully sequenced genomes, bacteria vary from 160 to &gt;10 000 kb in their chromosomal lengths. However, these species are all small bacteria based on their chromosomal length, although some of these bacteria are not endosymbiont. Hence, we hypothesize that the short length of chromosome is a necessary condition to generate strong enough strand-specific mutational bias. Perhaps, in bacteria with larger chromosome, the mutation pressure is hard to prevail translational selection. Alternatively, among genomes that have suffered reductive evolution, the repair mechanism of replication may be inefficient.</p><p>Secondly, all of the 10 bacteria have medium or low genomic G + C content. Among these species, <em>B</em>. <em>aphidicola</em> has the lowest G + C content as 26%, whereas <em>T</em>. <em>pallidum</em> has the highest ones as 52%. Perhaps, the environment of high G + C contents is adverse to the generation of strong strand mutation biases. Future experimental works are needed to clarify the relationship between replication mechanism and genomic GC content or genome size.</p><p>Thirdly, the strong mutation bias may be associated with presence or absence of certain genes involved in chromosome replication. As suggested by Klasson and Andersson,<sup><a reveal-id="DSP001C25" data-open="DSP001C25" class="link link-ref link-reveal xref-bibr">25</a></sup> the strong strand-specific mutational bias in endosymbiont genomes coincides with the absence of genes for replication restart pathways. They performed a comparative analysis of 20 γ-proteobacterial genomes and found that endosymbiont bacteria lacking <em>recA</em> and other genes involved in replication restart processes, such as <em>priA</em>, displayed the strongest strand bias.<sup><a reveal-id="DSP001C25" data-open="DSP001C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Driven by this viewpoint, we investigate the presence and absence of replication restart-related genes in the 10 genomes, in which the strand mutation biases are strong enough to generate distinct codon usages. The analysis is involved with genes <em>mutH</em>, <em>priA</em>, <em>topA</em>, <em>dnaT</em>, <em>fis</em> and <em>recA</em>, which are all initiation and re-initiation associated genes.<sup><a reveal-id="DSP001C25" data-open="DSP001C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Consequently, all of these genes are found to be absent in <em>B</em>. <em>floridanus</em> and five absent in <em>B</em>. <em>aphidicola</em>, with <em>mutH</em> as an exception. For the other eight bacteria, <em>priA</em>, <em>topA</em> and <em>recA</em> are present and the other three genes are absent. In a word, <em>dnaT</em> and <em>fis</em> are absent in all of the 10 genomes, whereas both the genes exist in <em>E</em>. <em>coli</em> and other γ-proteobacteria, which have not strong mutation biases. Klasson and Andersson hypothesized that cytosine deaminations accumulate during single-strand exposure at stalled replication forks and the extent of strand bias may depend on the time spent repairing such lesions. Inefficient restart mechanisms result in the long time of the replication forks being arrested and hereby lead to high DNA strand asymmetry.<sup><a reveal-id="DSP001C25" data-open="DSP001C25" class="link link-ref link-reveal xref-bibr">25</a></sup> As a common character of the genomes mentioned here, we believe that the absence of replication restart involved genes is very likely to appear in the other genomes, found in the future, with strong strand mutational bias.</p><p>Finally, the strong mutation bias may reflect as the strong cumulative excess of G over C plus T over A along the chromosome. In our previously published work,<sup><a reveal-id="DSP001C35" data-open="DSP001C35" class="link link-ref link-reveal xref-bibr">35</a></sup> the Z curve was used to compare the chromosome sequences between genomes with or without strand-specific codon usage. The <em>y</em>-component of the Z curve represents the plus of cumulative excess of G over C and T over A. An index is defined as the changing rate of <em>y</em>-component per unit base and denoted by symbol <em>k</em>. In fact, <em>k</em> equals to (G − C + T − A)/(G + C + T + A), where A, C, G and T denote the total number of the corresponding base appearing in the half of chromosome (from replication origin to terminus). After calculating <em>k</em> values for the 10 bacteria mentioned above, it is found that this values for these species are all larger than 0.035 and some even larger than 0.1. However, the value of <em>k</em> for <em>E</em>. <em>coli</em> K-12 is less than 0.02. Therefore, the <em>k</em> value that represents the changing rate of cumulative excess of keto (G + T) over amino (A + C) per unit base could be a good measurement of magnitude of strand composition biases and even strand mutation bias.</p>                            <h3 scrollto-destination=87195415 id="87195415" class="section-title" data-legacy-id=s4e>4.5. Conclusion</h3>
<p>Complex factors are found to be responsible for variation of codon usage in <em>L</em>. <em>intracellularis</em> chromosome. All of these factors could be interpreted by the paradigm ‘mutational bias-translational selection’. When analyzing genes in the largest plasmid of this bacterium, for the first time it is found that the strand-specific mutational biases are responsible for the primary variation of synonymous codon usages in plasmid. Genes, particularly highly expressed genes of this plasmid, are mainly located on the leading strands and this is supposed to be the effects exerted by replicational–transcriptional selection. These facts suggest that this plasmid may adopt the similar replication mechanism as the chromosome in <em>L</em>. <em>intracellularis</em>. Finally, common genomic characters are found among <em>L</em>. <em>intracellularis</em> and other bacteria in whose genomes the strand-specific mutational biases are the most important source of variation of codon usage.</p>                            <h2 scrollto-destination=87195417 id="87195417" class="section-title" data-legacy-id=s6>Funding</h2>
<p>The present study was supported by Doctoral Fund of Ministry of Education of China (20070614011) and National Natural Science Foundation of China (60801058).</p>                            <h2 scrollto-destination=87195419 id="87195419" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSP001C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grantham</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Gautier</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Gouy</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Mercier</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Pavé</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Codon catalog usage and the genome hypothesis</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1980</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">r49</div>-<div class="lpage">r62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20catalog%20usage%20and%20the%20genome%20hypothesis&amp;author=R.%20Grantham&amp;author=C.%20Gautier&amp;author=M.%20Gouy&amp;author=R.%20Mercier&amp;author=A.%20Pav%C3%A9&amp;publication_year=1980&amp;journal=Nucleic%20Acids%20Res.&amp;volume=8&amp;pages=r49-r62" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6986610" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grantham</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Gautier</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Gouy</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Jacobzone</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Mercier</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Codon catalog usage is a genome strategy modulated for gene expressivity</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1981</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">r43</div>-<div class="lpage">r74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20catalog%20usage%20is%20a%20genome%20strategy%20modulated%20for%20gene%20expressivity&amp;author=R.%20Grantham&amp;author=C.%20Gautier&amp;author=M.%20Gouy&amp;author=M.%20Jacobzone&amp;author=R.%20Mercier&amp;publication_year=1981&amp;journal=Nucleic%20Acids%20Res.&amp;volume=9&amp;pages=r43-r74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/9.1.213-b" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F9.1.213-b" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F9.1.213-b"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7208352" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gouy</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Gautier</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Codon usage in bacteria: correlation with gene expressivity</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1982</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">7055</div>-<div class="lpage">7074</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20in%20bacteria%3A%20correlation%20with%20gene%20expressivity&amp;author=M.%20Gouy&amp;author=C.%20Gautier&amp;publication_year=1982&amp;journal=Nucleic%20Acids%20Res.&amp;volume=10&amp;pages=7055-7074" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/10.22.7055" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F10.22.7055" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F10.22.7055"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6760125" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Correlation between the abundance of <em>Escherichia coli</em> transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the <em>E. coli</em> translational system</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1981</div>, vol. <div class="volume">151</div> (pg. <div class="fpage">389</div>-<div class="lpage">409</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correlation%20between%20the%20abundance%20of%20Escherichia%20coli%20transfer%20RNAs%20and%20the%20occurrence%20of%20the%20respective%20codons%20in%20its%20protein%20genes%3A%20a%20proposal%20for%20a%20synonymous%20codon%20choice%20that%20is%20optimal%20for%20the%20E.%20coli%20translational%20system&amp;author=T.%20Ikemura&amp;publication_year=1981&amp;journal=J.%20Mol.%20Biol.&amp;volume=151&amp;pages=389-409" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(81)90003-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(81)90003-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(81)90003-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6175758" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dong</div> <div class="given-names">H. J.</div></div>,  <div class="name"><div class="surname">Nilsson</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Kurland</div> <div class="given-names">C. G.</div></div>. </div><div class="article-title">Co-variation of tRNA abundance and codon usage in <em>Escherichia coli</em> at different growth rates</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1996</div>, vol. <div class="volume">260</div> (pg. <div class="fpage">649</div>-<div class="lpage">663</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Co-variation%20of%20tRNA%20abundance%20and%20codon%20usage%20in%20Escherichia%20coli%20at%20different%20growth%20rates&amp;author=H.%20J.%20Dong&amp;author=L.%20Nilsson&amp;author=C.%20G.%20Kurland&amp;publication_year=1996&amp;journal=J.%20Mol.%20Biol.&amp;volume=260&amp;pages=649-663" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1996.0428" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1996.0428" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1996.0428"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8709146" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bennetzen</div> <div class="given-names">J. L.</div></div>,  <div class="name"><div class="surname">Hall</div> <div class="given-names">B. D.</div></div>. </div><div class="article-title">Codon selection in yeast</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1982</div>, vol. <div class="volume">257</div> (pg. <div class="fpage">3026</div>-<div class="lpage">3031</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20selection%20in%20yeast&amp;author=J.%20L.%20Bennetzen&amp;author=B.%20D.%20Hall&amp;publication_year=1982&amp;journal=J.%20Biol.%20Chem.&amp;volume=257&amp;pages=3026-3031" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7037777" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Andersson</div> <div class="given-names">S. G. E.</div></div>,  <div class="name"><div class="surname">Sharp</div> <div class="given-names">P. M.</div></div>. </div><div class="article-title">Codon usage in the <em>Mycobacterium tuberculosis</em> complex</div>, <div class="source">Microbiology</div> , <div class="year">1996</div>, vol. <div class="volume">142</div> (pg. <div class="fpage">915</div>-<div class="lpage">925</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20in%20the%20Mycobacterium%20tuberculosis%20complex&amp;author=S.%20G.%20E.%20Andersson&amp;author=P.%20M.%20Sharp&amp;publication_year=1996&amp;journal=Microbiology&amp;volume=142&amp;pages=915-925" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00221287-142-4-915" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00221287-142-4-915" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00221287-142-4-915"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8936318" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lafay</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Atherton</div> <div class="given-names">J. C.</div></div>,  <div class="name"><div class="surname">Sharp</div> <div class="given-names">P. M.</div></div>. </div><div class="article-title">Absence of translationally selected synonymous codon usage bias in <em>Helicobacter pylori</em></div>, <div class="source">Microbiology</div> , <div class="year">2000</div>, vol. <div class="volume">146</div> (pg. <div class="fpage">851</div>-<div class="lpage">860</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Absence%20of%20translationally%20selected%20synonymous%20codon%20usage%20bias%20in%20Helicobacter%20pylori&amp;author=B.%20Lafay&amp;author=J.%20C.%20Atherton&amp;author=P.%20M.%20Sharp&amp;publication_year=2000&amp;journal=Microbiology&amp;volume=146&amp;pages=851-860" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00221287-146-4-851" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00221287-146-4-851" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00221287-146-4-851"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10784043" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yamada</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kudo</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Studies of codon usage and tRNA genes of 18 unicellular organisms and quantification of <em>Bacillus subtilis</em> tRNAs: gene expression level and species-specific diversity of codon usage based on multivariate analysis</div>, <div class="source">Gene</div> , <div class="year">1999</div>, vol. <div class="volume">238</div> (pg. <div class="fpage">143</div>-<div class="lpage">155</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Studies%20of%20codon%20usage%20and%20tRNA%20genes%20of%2018%20unicellular%20organisms%20and%20quantification%20of%20Bacillus%20subtilis%20tRNAs%3A%20gene%20expression%20level%20and%20species-specific%20diversity%20of%20codon%20usage%20based%20on%20multivariate%20analysis&amp;author=S.%20Kanaya&amp;author=Y.%20Yamada&amp;author=Y.%20Kudo&amp;author=T.%20Ikemura&amp;publication_year=1999&amp;journal=Gene&amp;volume=238&amp;pages=143-155" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(99)00225-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(99)00225-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(99)00225-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10570992" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McInerney</div> <div class="given-names">J. O.</div></div>. </div><div class="article-title">Replicational and transcriptional selection on codon usage in <em>Borrelia burgdorferi</em></div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">10698</div>-<div class="lpage">10703</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Replicational%20and%20transcriptional%20selection%20on%20codon%20usage%20in%20Borrelia%20burgdorferi&amp;author=J.%20O.%20McInerney&amp;publication_year=1998&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=10698-10703" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.18.10698" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.18.10698" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.18.10698"> </span></div> </div></div></div></div></div><div content-id="DSP001C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Romero</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Zavala</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Musto</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Codon usage in <em>Chlamydia trachomatis</em> is the result of strand-specific mutational biases and a complex pattern of selective forces</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2000</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">2084</div>-<div class="lpage">2090</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20in%20Chlamydia%20trachomatis%20is%20the%20result%20of%20strand-specific%20mutational%20biases%20and%20a%20complex%20pattern%20of%20selective%20forces&amp;author=H.%20Romero&amp;author=A.%20Zavala&amp;author=H.%20Musto&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res.&amp;volume=28&amp;pages=2084-2090" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.10.2084" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.10.2084" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.10.2084"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10773076" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grocock</div> <div class="given-names">R. J.</div></div>,  <div class="name"><div class="surname">Sharp</div> <div class="given-names">P. M.</div></div>. </div><div class="article-title">Synonymous codon usage in <em>Pseudomonas aeruginosa</em> PA01</div>, <div class="source">Gene</div> , <div class="year">2002</div>, vol. <div class="volume">289</div> (pg. <div class="fpage">131</div>-<div class="lpage">139</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20codon%20usage%20in%20Pseudomonas%20aeruginosa%20PA01&amp;author=R.%20J.%20Grocock&amp;author=P.%20M.%20Sharp&amp;publication_year=2002&amp;journal=Gene&amp;volume=289&amp;pages=131-139" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(02)00503-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(02)00503-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(02)00503-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12036591" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garcia-Vallve</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Romeu</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Palau</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Horizontal gene transfer in bacterial and archaeal complete genomes</div>, <div class="source">Genome Res.</div> , <div class="year">2000</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">1719</div>-<div class="lpage">1725</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Horizontal%20gene%20transfer%20in%20bacterial%20and%20archaeal%20complete%20genomes&amp;author=S.%20Garcia-Vallve&amp;author=A.%20Romeu&amp;author=J.%20Palau&amp;publication_year=2000&amp;journal=Genome%20Res.&amp;volume=10&amp;pages=1719-1725" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.130000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.130000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.130000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11076857" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garcia-Vallve</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Guzman</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Montero</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Romeu</div> <div class="given-names">A.</div></div>. </div><div class="article-title">HGT-DB: a database of putative horizontally transferred genes in prokaryotic complete genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">187</div>-<div class="lpage">189</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HGT-DB%3A%20a%20database%20of%20putative%20horizontally%20transferred%20genes%20in%20prokaryotic%20complete%20genomes&amp;author=S.%20Garcia-Vallve&amp;author=E.%20Guzman&amp;author=M.%20A.%20Montero&amp;author=A.%20Romeu&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=187-189" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12519978" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Médigue</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Rouxel</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Vigier</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Hénaut</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Danchin</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Evidence for horizontal gene transfer in <em>Escherichia coli</em> speciation</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1991</div>, vol. <div class="volume">222</div> (pg. <div class="fpage">851</div>-<div class="lpage">856</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20horizontal%20gene%20transfer%20in%20Escherichia%20coli%20speciation&amp;author=C.%20M%C3%A9digue&amp;author=T.%20Rouxel&amp;author=P.%20Vigier&amp;author=A.%20H%C3%A9naut&amp;author=A.%20Danchin&amp;publication_year=1991&amp;journal=J.%20Mol.%20Biol.&amp;volume=222&amp;pages=851-856" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(91)90575-Q" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(91)90575-Q" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(91)90575-Q"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1762151" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moszer</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Rocha</div> <div class="given-names">E. P.</div></div>,  <div class="name"><div class="surname">Danchin</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Codon usage and lateral gene transfer in <em>Bacillus subtilis</em></div>, <div class="source">Curr. Opin. Microbiol.</div> , <div class="year">1999</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">524</div>-<div class="lpage">528</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20and%20lateral%20gene%20transfer%20in%20Bacillus%20subtilis&amp;author=I.%20Moszer&amp;author=E.%20P.%20Rocha&amp;author=A.%20Danchin&amp;publication_year=1999&amp;journal=Curr.%20Opin.%20Microbiol.&amp;volume=2&amp;pages=524-528" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1369-5274(99)00011-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1369-5274(99)00011-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1369-5274(99)00011-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10508724" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McInerney</div> <div class="given-names">J. O.</div></div>. </div><div class="article-title">Prokaryotic genome evolution as assessed by multivariate analysis of codon usage patterns</div>, <div class="source">Microb. Comp. Genomics</div> , <div class="year">1997</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">1</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prokaryotic%20genome%20evolution%20as%20assessed%20by%20multivariate%20analysis%20of%20codon%20usage%20patterns&amp;author=J.%20O.%20McInerney&amp;publication_year=1997&amp;journal=Microb.%20Comp.%20Genomics&amp;volume=2&amp;pages=1-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/omi.1.1997.2.89" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fomi.1.1997.2.89" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Fomi.1.1997.2.89"> </span></div> </div></div></div></div></div><div content-id="DSP001C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McLean</div> <div class="given-names">M. J.</div></div>,  <div class="name"><div class="surname">Wolfe</div> <div class="given-names">K. H.</div></div>,  <div class="name"><div class="surname">Devine</div> <div class="given-names">K. M.</div></div>. </div><div class="article-title">Base composition skews, replication orientation, and gene orientation in 12 prokaryote genomes</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1998</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">691</div>-<div class="lpage">696</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Base%20composition%20skews%2C%20replication%20orientation%2C%20and%20gene%20orientation%20in%2012%20prokaryote%20genomes&amp;author=M.%20J.%20McLean&amp;author=K.%20H.%20Wolfe&amp;author=K.%20M.%20Devine&amp;publication_year=1998&amp;journal=J.%20Mol.%20Evol.&amp;volume=47&amp;pages=691-696" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00006428" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00006428" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00006428"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9847411" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Frank</div> <div class="given-names">A. C.</div></div>,  <div class="name"><div class="surname">Lobry</div> <div class="given-names">J. R.</div></div>. </div><div class="article-title">Asymmetric substitution patterns: a review of possible underlying mutational or selective mechanisms</div>, <div class="source">Gene</div> , <div class="year">1999</div>, vol. <div class="volume">238</div> (pg. <div class="fpage">65</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Asymmetric%20substitution%20patterns%3A%20a%20review%20of%20possible%20underlying%20mutational%20or%20selective%20mechanisms&amp;author=A.%20C.%20Frank&amp;author=J.%20R.%20Lobry&amp;publication_year=1999&amp;journal=Gene&amp;volume=238&amp;pages=65-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(99)00297-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(99)00297-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(99)00297-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10570985" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mrázek</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Karlin</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Strand compositional asymmetry in bacterial and large viral genomes</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">3720</div>-<div class="lpage">3725</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strand%20compositional%20asymmetry%20in%20bacterial%20and%20large%20viral%20genomes&amp;author=J.%20Mr%C3%A1zek&amp;author=S.%20Karlin&amp;publication_year=1998&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=3720-3725" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.7.3720" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.7.3720" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.7.3720"> </span></div> </div></div></div></div></div><div content-id="DSP001C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mackiewicz</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Gierlik</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kowalczuk</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Dudek</div> <div class="given-names">M. R.</div></div>,  <div class="name"><div class="surname">Cebrat</div> <div class="given-names">S.</div></div>. </div><div class="article-title">How does replication-associated mutational pressure influence amino acid composition of proteins?</div>, <div class="source">Genome Res.</div> , <div class="year">1999</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">409</div>-<div class="lpage">416</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20does%20replication-associated%20mutational%20pressure%20influence%20amino%20acid%20composition%20of%20proteins%3F&amp;author=P.%20Mackiewicz&amp;author=A.%20Gierlik&amp;author=M.%20Kowalczuk&amp;author=M.%20R.%20Dudek&amp;author=S.%20Cebrat&amp;publication_year=1999&amp;journal=Genome%20Res.&amp;volume=9&amp;pages=409-416" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10330120" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lobry</div> <div class="given-names">J. R.</div></div>,  <div class="name"><div class="surname">Sueoka</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Asymmetric directional mutation pressures in bacteria</div>, <div class="source">Genome Biol.</div> , <div class="year">2002</div>, vol. <div class="volume">3</div>  <div class="comment">RESEARCH0058</div></div></div></div></div><div content-id="DSP001C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rocha</div> <div class="given-names">E. P.</div></div>,  <div class="name"><div class="surname">Danchin</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Essentiality, not expressiveness, drives gene-strand bias in bacteria</div>, <div class="source">Nat Genet.</div> , <div class="year">2003</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">377</div>-<div class="lpage">378</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Essentiality%2C%20not%20expressiveness%2C%20drives%20gene-strand%20bias%20in%20bacteria&amp;author=E.%20P.%20Rocha&amp;author=A.%20Danchin&amp;publication_year=2003&amp;journal=Nat%20Genet.&amp;volume=34&amp;pages=377-378" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1209" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1209" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1209"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12847524" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Price</div> <div class="given-names">M. N.</div></div>,  <div class="name"><div class="surname">Alm</div> <div class="given-names">E. J.</div></div>,  <div class="name"><div class="surname">Arkin</div> <div class="given-names">A. P.</div></div>. </div><div class="article-title">Interruptions in gene expression drive highly expressed operons to the leading strand of DNA replication</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">3224</div>-<div class="lpage">3234</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Interruptions%20in%20gene%20expression%20drive%20highly%20expressed%20operons%20to%20the%20leading%20strand%20of%20DNA%20replication&amp;author=M.%20N.%20Price&amp;author=E.%20J.%20Alm&amp;author=A.%20P.%20Arkin&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=3224-3234" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki638" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki638" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki638"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15942025" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Klasson</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Andersson</div> <div class="given-names">S. G.</div></div>. </div><div class="article-title">Strong asymmetric mutation bias in endosymbiont genomes coincide with loss of genes for replication restart pathways</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2006</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">1031</div>-<div class="lpage">1039</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strong%20asymmetric%20mutation%20bias%20in%20endosymbiont%20genomes%20coincide%20with%20loss%20of%20genes%20for%20replication%20restart%20pathways&amp;author=L.%20Klasson&amp;author=S.%20G.%20Andersson&amp;publication_year=2006&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=23&amp;pages=1031-1039" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msj107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsj107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsj107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16476690" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rocha</div> <div class="given-names">E. P.</div></div>,  <div class="name"><div class="surname">Touchon</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Feil</div> <div class="given-names">E. J.</div></div>. </div><div class="article-title">Similar compositional biases are caused by very different mutational effects</div>, <div class="source">Genome Res.</div> , <div class="year">2006</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">1537</div>-<div class="lpage">1547</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Similar%20compositional%20biases%20are%20caused%20by%20very%20different%20mutational%20effects&amp;author=E.%20P.%20Rocha&amp;author=M.%20Touchon&amp;author=E.%20J.%20Feil&amp;publication_year=2006&amp;journal=Genome%20Res.&amp;volume=16&amp;pages=1537-1547" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.5525106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.5525106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.5525106"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17068325" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Necsulea</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lobry</div> <div class="given-names">J. R.</div></div>. </div><div class="article-title">A new method for assessing the effect of replication on DNA base composition asymmetry</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">2169</div>-<div class="lpage">2179</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20method%20for%20assessing%20the%20effect%20of%20replication%20on%20DNA%20base%20composition%20asymmetry&amp;author=A.%20Necsulea&amp;author=J.%20R.%20Lobry&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=2169-2179" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm148" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm148" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm148"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17646257" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lobry</div> <div class="given-names">J. R.</div></div>. </div><div class="article-title">Origin of replication of <em>Mycoplasma genitalium</em></div>, <div class="source">Science</div> , <div class="year">1996</div>, vol. <div class="volume">272</div> (pg. <div class="fpage">745</div>-<div class="lpage">746</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origin%20of%20replication%20of%20Mycoplasma%20genitalium&amp;author=J.%20R.%20Lobry&amp;publication_year=1996&amp;journal=Science&amp;volume=272&amp;pages=745-746" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.272.5262.745" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.272.5262.745" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.272.5262.745"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8614839" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lobry</div> <div class="given-names">J. R.</div></div>. </div><div class="article-title">Asymmetric substitution patterns in the two DNA strands of bacteria</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1996</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">660</div>-<div class="lpage">665</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Asymmetric%20substitution%20patterns%20in%20the%20two%20DNA%20strands%20of%20bacteria&amp;author=J.%20R.%20Lobry&amp;publication_year=1996&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=13&amp;pages=660-665" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025626" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a025626" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a025626"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8676740" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lafay</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Lloyd</div> <div class="given-names">A. T.</div></div>,  <div class="name"><div class="surname">McLean</div> <div class="given-names">M. J.</div></div>,  <div class="name"><div class="surname">Devine</div> <div class="given-names">K. M.</div></div>,  <div class="name"><div class="surname">Sharp</div> <div class="given-names">P. M.</div></div>,  <div class="name"><div class="surname">Wolfe</div> <div class="given-names">K. H.</div></div>. </div><div class="article-title">Proteome composition and codon usage in spirochaetes: species-specific and DNA strand-specific mutational biases</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">1642</div>-<div class="lpage">1649</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proteome%20composition%20and%20codon%20usage%20in%20spirochaetes%3A%20species-specific%20and%20DNA%20strand-specific%20mutational%20biases&amp;author=B.%20Lafay&amp;author=A.%20T.%20Lloyd&amp;author=M.%20J.%20McLean&amp;author=K.%20M.%20Devine&amp;author=P.%20M.%20Sharp&amp;author=K.%20H.%20Wolfe&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=1642-1649" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.7.1642" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.7.1642" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.7.1642"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10075995" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rispe</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Delmotte</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">van Ham</div> <div class="given-names">R. C.</div></div>,  <div class="name"><div class="surname">Moya</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Mutational and selective pressures on codon and amino acid usage in <em>Buchnera</em>, endosymbiotic bacteria of aphids</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">44</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutational%20and%20selective%20pressures%20on%20codon%20and%20amino%20acid%20usage%20in%20Buchnera%2C%20endosymbiotic%20bacteria%20of%20aphids&amp;author=C.%20Rispe&amp;author=F.%20Delmotte&amp;author=R.%20C.%20van%20Ham&amp;author=A.%20Moya&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=44-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1358104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1358104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1358104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14672975" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Banerjee</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Basak</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Gupta</div> <div class="given-names">S. K.</div></div>,  <div class="name"><div class="surname">Ghosh</div> <div class="given-names">T. C.</div></div>. </div><div class="article-title">Evolutionary forces in shaping the codon and amino acid usages in <em>Blochmannia floridanus</em></div>, <div class="source">J. Biomol. Struct. Dyn.</div> , <div class="year">2004</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">13</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20forces%20in%20shaping%20the%20codon%20and%20amino%20acid%20usages%20in%20Blochmannia%20floridanus&amp;author=T.%20Banerjee&amp;author=S.%20Basak&amp;author=S.%20K.%20Gupta&amp;author=T.%20C.%20Ghosh&amp;publication_year=2004&amp;journal=J.%20Biomol.%20Struct.%20Dyn.&amp;volume=22&amp;pages=13-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/07391102.2004.10506976" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F07391102.2004.10506976" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F07391102.2004.10506976"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15214801" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Das</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Paul</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Chatterjee</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Dutta</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Codon and amino acid usage in two major human pathogens of genus <em>Bartonella</em>–optimization between replicational–transcriptional selection, translational control and cost minimization</div>, <div class="source">DNA Res.</div> , <div class="year">2005</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">91</div>-<div class="lpage">102</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20and%20amino%20acid%20usage%20in%20two%20major%20human%20pathogens%20of%20genus%20Bartonella%E2%80%93optimization%20between%20replicational%E2%80%93transcriptional%20selection%2C%20translational%20control%20and%20cost%20minimization&amp;author=S.%20Das&amp;author=S.%20Paul&amp;author=S.%20Chatterjee&amp;author=C.%20Dutta&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=91-102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/12.2.91" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F12.2.91" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F12.2.91"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16303741" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Das</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Paul</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Dutta</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Evolutionary constraints on codon and amino acid usage in two strains of human pathogenic actinobacteria <em>Tropheryma whipplei</em></div>, <div class="source">J. Mol. Evol.</div> , <div class="year">2006</div>, vol. <div class="volume">62</div> (pg. <div class="fpage">645</div>-<div class="lpage">658</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20constraints%20on%20codon%20and%20amino%20acid%20usage%20in%20two%20strains%20of%20human%20pathogenic%20actinobacteria%20Tropheryma%20whipplei&amp;author=S.%20Das&amp;author=S.%20Paul&amp;author=C.%20Dutta&amp;publication_year=2006&amp;journal=J.%20Mol.%20Evol.&amp;volume=62&amp;pages=645-658" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-005-0164-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-005-0164-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-005-0164-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16557339" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">F. B.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">X. J.</div></div>. </div><div class="article-title">Separate base usages of genes located on the leading and lagging strands in <em>Chlamydia muridarum</em> revealed by the Z curve method</div>, <div class="source">BMC Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> pg. <div class="fpage">366</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Separate%20base%20usages%20of%20genes%20located%20on%20the%20leading%20and%20lagging%20strands%20in%20Chlamydia%20muridarum%20revealed%20by%20the%20Z%20curve%20method&amp;author=F.%20B.%20Guo&amp;author=X.%20J.%20Yu&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=366" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-366" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-366" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-366"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17925038" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mølbak</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Johnsen</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Boye</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Jensen</div> <div class="given-names">T. K.</div></div>,  <div class="name"><div class="surname">Johansen</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Møller</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Leser</div> <div class="given-names">T. D.</div></div>. </div><div class="article-title">The microbiota of pigs influenced by diet texture and severity of <em>Lawsonia intracellularis</em> infection</div>, <div class="source">Vet Microbiol.</div> , <div class="year">2008</div>, vol. <div class="volume">128</div> (pg. <div class="fpage">96</div>-<div class="lpage">107</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20microbiota%20of%20pigs%20influenced%20by%20diet%20texture%20and%20severity%20of%20Lawsonia%20intracellularis%20infection&amp;author=L.%20M%C3%B8lbak&amp;author=K.%20Johnsen&amp;author=M.%20Boye&amp;author=T.%20K.%20Jensen&amp;author=M.%20Johansen&amp;author=K.%20M%C3%B8ller&amp;author=T.%20D.%20Leser&amp;publication_year=2008&amp;journal=Vet%20Microbiol.&amp;volume=128&amp;pages=96-107" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.vetmic.2007.09.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.vetmic.2007.09.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.vetmic.2007.09.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17996403" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Peden</div> <div class="given-names">J. F.</div></div>. </div><div class="article-title">Analysis of codon usage</div>, <div class="year">1999</div><div class="publisher-name">University of Nottingham</div> <div class="comment">Ph.D. Thesis</div></div></div></div></div><div content-id="DSP001C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wright</div> <div class="given-names">F.</div></div>. </div><div class="article-title">The ‘effective number of codons’ used in a gene</div>, <div class="source">Gene</div> , <div class="year">1990</div>, vol. <div class="volume">87</div> (pg. <div class="fpage">23</div>-<div class="lpage">29</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20%E2%80%98effective%20number%20of%20codons%E2%80%99%20used%20in%20a%20gene&amp;author=F.%20Wright&amp;publication_year=1990&amp;journal=Gene&amp;volume=87&amp;pages=23-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(90)90491-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(90)90491-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(90)90491-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2110097" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Perrière</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Thioulouse</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Use and misuse of correspondence analysis in codon usage studies</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">4548</div>-<div class="lpage">4555</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Use%20and%20misuse%20of%20correspondence%20analysis%20in%20codon%20usage%20studies&amp;author=G.%20Perri%C3%A8re&amp;author=J.%20Thioulouse&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res.&amp;volume=30&amp;pages=4548-4555" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkf565" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkf565" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkf565"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12384602" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dillon</div> <div class="given-names">W. R.</div></div>,  <div class="name"><div class="surname">Goldstein</div> <div class="given-names">M.</div></div>. </div>, <div class="source">Multivariate Analysis, Method and Application</div> , <div class="year">1984</div><div class="publisher-loc">New York, USA</div><div class="publisher-name">Willey Press</div></div></div></div></div><div content-id="DSP001C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Romero</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Zavala</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Musto</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Compositional pressure and translational selection determine codon usage in the extremely GC-poor unicellular eukaryote <em>Entamoeba histolytica</em></div>, <div class="source">Gene</div> , <div class="year">2000</div>, vol. <div class="volume">242</div> (pg. <div class="fpage">307</div>-<div class="lpage">311</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Compositional%20pressure%20and%20translational%20selection%20determine%20codon%20usage%20in%20the%20extremely%20GC-poor%20unicellular%20eukaryote%20Entamoeba%20histolytica&amp;author=H.%20Romero&amp;author=A.%20Zavala&amp;author=H.%20Musto&amp;publication_year=2000&amp;journal=Gene&amp;volume=242&amp;pages=307-311" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(99)00491-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(99)00491-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(99)00491-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10721724" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gao</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">C. T.</div></div>. </div><div class="article-title">DoriC: a database of oriC regions in bacterial genomes</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">1866</div>-<div class="lpage">1867</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DoriC%3A%20a%20database%20of%20oriC%20regions%20in%20bacterial%20genomes&amp;author=F.%20Gao&amp;author=C.%20T.%20Zhang&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=1866-1867" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm255" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm255" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm255"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17496319" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fu</div> <div class="given-names">Q. S.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">L. L.</div></div>. </div><div class="article-title">Gene expression analysis of six GC-rich Gram-negative phytopathogens</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">2005</div>, vol. <div class="volume">332</div> (pg. <div class="fpage">380</div>-<div class="lpage">387</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20expression%20analysis%20of%20six%20GC-rich%20Gram-negative%20phytopathogens&amp;author=Q.%20S.%20Fu&amp;author=F.%20Li&amp;author=L.%20L.%20Chen&amp;publication_year=2005&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=332&amp;pages=380-387" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2005.04.128" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2005.04.128" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2005.04.128"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15910748" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gupta</div> <div class="given-names">S. K.</div></div>,  <div class="name"><div class="surname">Bhattacharyya</div> <div class="given-names">T. K.</div></div>,  <div class="name"><div class="surname">Ghosh</div> <div class="given-names">T. C.</div></div>. </div><div class="article-title">Synonymous codon usage in <em>Lactococcus lactis</em>: mutational bias versus translational selection</div>, <div class="source">J. Biomol. Struct. Dyn.</div> , <div class="year">2004</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">527</div>-<div class="lpage">536</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20codon%20usage%20in%20Lactococcus%20lactis%3A%20mutational%20bias%20versus%20translational%20selection&amp;author=S.%20K.%20Gupta&amp;author=T.%20K.%20Bhattacharyya&amp;author=T.%20C.%20Ghosh&amp;publication_year=2004&amp;journal=J.%20Biomol.%20Struct.%20Dyn.&amp;volume=21&amp;pages=527-536" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/07391102.2004.10506946" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F07391102.2004.10506946" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F07391102.2004.10506946"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14692797" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Francino</div> <div class="given-names">M. P.</div></div>,  <div class="name"><div class="surname">Chao</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Riley</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Ochman</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Asymmetries generated by transcription-coupled repair in enterobacterial genes</div>, <div class="source">Science</div> , <div class="year">1996</div>, vol. <div class="volume">272</div> (pg. <div class="fpage">107</div>-<div class="lpage">109</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Asymmetries%20generated%20by%20transcription-coupled%20repair%20in%20enterobacterial%20genes&amp;author=M.%20P.%20Francino&amp;author=L.%20Chao&amp;author=M.%20A.%20Riley&amp;author=H.%20Ochman&amp;publication_year=1996&amp;journal=Science&amp;volume=272&amp;pages=107-109" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.272.5258.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.272.5258.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.272.5258.107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8600517" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Beletskii</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bhagwat</div> <div class="given-names">A. S.</div></div>. </div><div class="article-title">Transcription-induced mutations: increase in C to T mutations in the nontranscribed strand during transcription in <em>Escherichia coli</em></div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1996</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">13919</div>-<div class="lpage">13924</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcription-induced%20mutations%3A%20increase%20in%20C%20to%20T%20mutations%20in%20the%20nontranscribed%20strand%20during%20transcription%20in%20Escherichia%20coli&amp;author=A.%20Beletskii&amp;author=A.%20S.%20Bhagwat&amp;publication_year=1996&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=93&amp;pages=13919-13924" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.93.24.13919" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.93.24.13919" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.93.24.13919"> </span></div> </div></div></div></div></div><div content-id="DSP001C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Furusawa</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Doi</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Asymmetrical DNA replication promotes evolution: disparity theory of evolution</div>, <div class="source">Genetica</div> , <div class="year">1998</div>, vol. <div class="volume">102–103</div> (pg. <div class="fpage">333</div>-<div class="lpage">347</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Asymmetrical%20DNA%20replication%20promotes%20evolution%3A%20disparity%20theory%20of%20evolution&amp;author=M.%20Furusawa&amp;author=H.%20Doi&amp;publication_year=1998&amp;journal=Genetica&amp;volume=102–103&amp;pages=333-347" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1017078924245" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1017078924245" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1017078924245"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9766961" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">French</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Consequences of replication fork movement through transcription units in vivo</div>, <div class="source">Science</div> , <div class="year">1992</div>, vol. <div class="volume">258</div> (pg. <div class="fpage">1362</div>-<div class="lpage">1365</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Consequences%20of%20replication%20fork%20movement%20through%20transcription%20units%20in%20vivo&amp;author=S.%20French&amp;publication_year=1992&amp;journal=Science&amp;volume=258&amp;pages=1362-1365" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1455232" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1455232" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1455232"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1455232" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Picardeau</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lobry</div> <div class="given-names">J. R.</div></div>,  <div class="name"><div class="surname">Hinnebusch</div> <div class="given-names">B. J.</div></div>. </div><div class="article-title">Analyzing DNA strand compositional asymmetry to identify candidate replication origins of <em>Borrelia burgdorferi</em> linear and circular plasmids</div>, <div class="source">Genome Res.</div> , <div class="year">2000</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">1594</div>-<div class="lpage">1604</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analyzing%20DNA%20strand%20compositional%20asymmetry%20to%20identify%20candidate%20replication%20origins%20of%20Borrelia%20burgdorferi%20linear%20and%20circular%20plasmids&amp;author=M.%20Picardeau&amp;author=J.%20R.%20Lobry&amp;author=B.%20J.%20Hinnebusch&amp;publication_year=2000&amp;journal=Genome%20Res.&amp;volume=10&amp;pages=1594-1604" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.124000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.124000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.124000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11042157" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barbour</div> <div class="given-names">A. G.</div></div>. </div><div class="article-title">Linear DNA of <em>Borrelia</em> species and antigenic variation</div>, <div class="source">Trends Microbiol</div> , <div class="year">1993</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">236</div>-<div class="lpage">239</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linear%20DNA%20of%20Borrelia%20species%20and%20antigenic%20variation&amp;author=A.%20G.%20Barbour&amp;publication_year=1993&amp;journal=Trends%20Microbiol&amp;volume=1&amp;pages=236-239" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0966-842X(93)90139-I" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0966-842X(93)90139-I" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0966-842X(93)90139-I"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8137122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Syvanen</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Horizontal gene transfer: evidence and possible consequences</div>, <div class="source">Annu. Rev. Genet.</div> , <div class="year">1994</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">237</div>-<div class="lpage">261</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Horizontal%20gene%20transfer%3A%20evidence%20and%20possible%20consequences&amp;author=M.%20Syvanen&amp;publication_year=1994&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=28&amp;pages=237-261" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.ge.28.120194.001321" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.ge.28.120194.001321" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.ge.28.120194.001321"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7893125" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Daubin</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Lerat</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Perrière</div> <div class="given-names">G.</div></div>. </div><div class="article-title">The source of laterally transferred genes in bacterial genomes</div>, <div class="source">Genome Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">4</div> pg. <div class="fpage">R57</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20source%20of%20laterally%20transferred%20genes%20in%20bacterial%20genomes&amp;author=V.%20Daubin&amp;author=E.%20Lerat&amp;author=G.%20Perri%C3%A8re&amp;publication_year=2003&amp;journal=Genome%20Biol.&amp;volume=4&amp;pages=R57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2003-4-9-r57" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2003-4-9-r57" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2003-4-9-r57"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12952536" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP001C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP001C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lawrence</div> <div class="given-names">J. G.</div></div>,  <div class="name"><div class="surname">Ochman</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Amelioration of bacterial genomes: rates of change and exchange</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1997</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">383</div>-<div class="lpage">397</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Amelioration%20of%20bacterial%20genomes%3A%20rates%20of%20change%20and%20exchange&amp;author=J.%20G.%20Lawrence&amp;author=H.%20Ochman&amp;publication_year=1997&amp;journal=J.%20Mol.%20Evol.&amp;volume=44&amp;pages=383-397" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00006158" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00006158" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00006158"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9089078" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Minoru Ko</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2009. Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=510591&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093_dnares_dsp001/1/dsp001_Supplementary_Data.zip?Expires=2147483647&amp;Signature=ry0FoQMtRGuC91OnCopbPDJiqf6~bUMZD80nNelv5u5C38b4qjzjnHQb0X7SizcFnPCm-cyYysIEMdWD39v5SoqCsRNgPKW2vwVOPwsM5rMqGLApbW73unT~AQangV~hg4UO0pNwkLN15OZC5Cp5cAUp16aZHHKd9HZ9GSWwV48xDH7Vd6lAp6dmiMTKvszxHcpbizuTSKnaJQqGqHl~q48tSg0VnTQooo1zQo-~WUczIgB~6f7CzFJ0kMtQrc5ApOtNWC~yirwiiybbGHqZQWt13PIc7eLK1J02qtGuZzHq~q6KkaFoksAcYxlohgmaa5HNgFaN31vT8x8qsLHsqw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">146</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000265770800002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">20</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsp001"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="510591"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000265770800002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f16%2f2%2f91%2f510591">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000265770800002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (20)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F16%2F2%2F91%2F510591" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/510591'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Codon Usages of Genes on Chromosome, and Surprisingly, Genes in Plasmid are Primarily Affected by Strand-specific Mutational Biases in Lawsonia intracellularis" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsp001");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f16%2f2%2f91%2f510591">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

