

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/2/171/1745492","@type":"ScholarlyArticle","name":"Sequencing and comparative analyses of the genomes of zoysiagrasses","datePublished":"2016-03-14","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2016-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw006","keywords":["<italic>Zoysia</italic>","turfgrass","genome sequencing","gene prediction","comparative analysis"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/2/171/1745492","author":[{"name":"Tanaka, Hidenori","affiliation":"1  Faculty of Agriculture ,  University of Miyazaki ,  1-1 Gakuenkibanadai-Nishi, Miyazaki 889-2192 ,  Japan","@type":"Person"},{"name":"Hirakawa, Hideki","affiliation":"2  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Kosugi, Shunichi","affiliation":"2  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Nakayama, Shinobu","affiliation":"2  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Ono, Akiko","affiliation":"2  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Watanabe, Akiko","affiliation":"2  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Hashiguchi, Masatsugu","affiliation":"1  Faculty of Agriculture ,  University of Miyazaki ,  1-1 Gakuenkibanadai-Nishi, Miyazaki 889-2192 ,  Japan","@type":"Person"},{"name":"Gondo, Takahiro","affiliation":"1  Faculty of Agriculture ,  University of Miyazaki ,  1-1 Gakuenkibanadai-Nishi, Miyazaki 889-2192 ,  Japan","@type":"Person"},{"name":"Ishigaki, Genki","affiliation":"1  Faculty of Agriculture ,  University of Miyazaki ,  1-1 Gakuenkibanadai-Nishi, Miyazaki 889-2192 ,  Japan","@type":"Person"},{"name":"Muguerza, Melody","affiliation":"1  Faculty of Agriculture ,  University of Miyazaki ,  1-1 Gakuenkibanadai-Nishi, Miyazaki 889-2192 ,  Japan","@type":"Person"},{"name":"Shimizu, Katsuya","affiliation":"3  Plant Technology Group, Biotechnology &amp; Afforestation Laboratory, New Business Planning Division ,  Toyota Motor Corporation ,  1099 Marune, Kurozasa-cho, Miyoshi, Aichi  470-0201 ,  Japan","@type":"Person"},{"name":"Sawamura, Noriko","affiliation":"3  Plant Technology Group, Biotechnology &amp; Afforestation Laboratory, New Business Planning Division ,  Toyota Motor Corporation ,  1099 Marune, Kurozasa-cho, Miyoshi, Aichi  470-0201 ,  Japan","@type":"Person"},{"name":"Inoue, Takayasu","affiliation":"4  Fuji Chemical Co., Ltd. ,  3-2-33 Higashi-nodamachi, Miyakojima-ku, Osaka 534-0024 ,  Japan","@type":"Person"},{"name":"Shigeki, Yuichi","affiliation":"4  Fuji Chemical Co., Ltd. ,  3-2-33 Higashi-nodamachi, Miyakojima-ku, Osaka 534-0024 ,  Japan","@type":"Person"},{"name":"Ohno, Naoki","affiliation":"4  Fuji Chemical Co., Ltd. ,  3-2-33 Higashi-nodamachi, Miyakojima-ku, Osaka 534-0024 ,  Japan","@type":"Person"},{"name":"Tabata, Satoshi","affiliation":"2  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan","@type":"Person"},{"name":"Akashi, Ryo","affiliation":"1  Faculty of Agriculture ,  University of Miyazaki ,  1-1 Gakuenkibanadai-Nishi, Miyazaki 889-2192 ,  Japan","@type":"Person"},{"name":"Sato, Shusei","affiliation":"2  Kazusa DNA Research Institute ,  2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818 ,  Japan    5  Graduate School of Life Sciences ,  Tohoku University ,  Katahira 2-1-1, Aoba-ku, Sendai 980-8577 ,  Japan","@type":"Person"}],"description":"Abstract.  Zoysia is a warm-season turfgrass, which comprises 11 allotetraploid species (2n = 4x = 40), each possessing different morphological and physiologica","pageStart":"171","pageEnd":"180","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/2/1/m_cover.gif?Expires=2147483647&Signature=H~-rl2t0~4icOqxuxQciOuaILaWLPqFGH3A~nroLlKomW3yjgtZ~qWZv6XD0n7MjtydtUqNoJUBHP0IS9FeSlJ7e824tplkVC9mcToG2lma9nT3MqHOrgXg-FmGm8R1Hko1D-xKC7m9goSRdTV0PpCgY514~J0A53r05NXU-LixEnWtvnkrSQlIKg2cNSOr5m2suFRYKdAKYSGWqTMps1eFybyDDmECiZJwPFJr9hkzFGuT0qtGzqG0CshylrK0bC2jCz4U60gjwvUfzNmkXc68xyAxGKYJwvJo89NhQiuWScVg0wHNDvMxjguS6KYmoQK6GthtPkW1VFz6XLn-4lQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Sequencing and comparative analyses of the genomes of zoysiagrasses","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/2/1/m_cover.gif?Expires=2147483647&Signature=H~-rl2t0~4icOqxuxQciOuaILaWLPqFGH3A~nroLlKomW3yjgtZ~qWZv6XD0n7MjtydtUqNoJUBHP0IS9FeSlJ7e824tplkVC9mcToG2lma9nT3MqHOrgXg-FmGm8R1Hko1D-xKC7m9goSRdTV0PpCgY514~J0A53r05NXU-LixEnWtvnkrSQlIKg2cNSOr5m2suFRYKdAKYSGWqTMps1eFybyDDmECiZJwPFJr9hkzFGuT0qtGzqG0CshylrK0bC2jCz4U60gjwvUfzNmkXc68xyAxGKYJwvJo89NhQiuWScVg0wHNDvMxjguS6KYmoQK6GthtPkW1VFz6XLn-4lQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Sequencing and comparative analyses of the genomes of zoysiagrasses" />
    <meta property="og:description" content="Abstract.  Zoysia is a warm-season turfgrass, which comprises 11 allotetraploid species (2n = 4x = 40), each possessing different morphological and physiologica" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/2/171/1745492" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-03-14" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Tanaka, Hidenori" /><meta name="citation_author_institution" content="Faculty of Agriculture, University of Miyazaki, 1-1 Gakuenkibanadai-Nishi, Miyazaki 889-2192, Japan" /><meta name="citation_author" content="Hirakawa, Hideki" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Kosugi, Shunichi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Nakayama, Shinobu" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Ono, Akiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Watanabe, Akiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Hashiguchi, Masatsugu" /><meta name="citation_author_institution" content="Faculty of Agriculture, University of Miyazaki, 1-1 Gakuenkibanadai-Nishi, Miyazaki 889-2192, Japan" /><meta name="citation_author" content="Gondo, Takahiro" /><meta name="citation_author_institution" content="Faculty of Agriculture, University of Miyazaki, 1-1 Gakuenkibanadai-Nishi, Miyazaki 889-2192, Japan" /><meta name="citation_author" content="Ishigaki, Genki" /><meta name="citation_author_institution" content="Faculty of Agriculture, University of Miyazaki, 1-1 Gakuenkibanadai-Nishi, Miyazaki 889-2192, Japan" /><meta name="citation_author" content="Muguerza, Melody" /><meta name="citation_author_institution" content="Faculty of Agriculture, University of Miyazaki, 1-1 Gakuenkibanadai-Nishi, Miyazaki 889-2192, Japan" /><meta name="citation_author" content="Shimizu, Katsuya" /><meta name="citation_author_institution" content="Plant Technology Group, Biotechnology &amp; Afforestation Laboratory, New Business Planning Division, Toyota Motor Corporation, 1099 Marune, Kurozasa-cho, Miyoshi, Aichi470-0201, Japan" /><meta name="citation_author" content="Sawamura, Noriko" /><meta name="citation_author_institution" content="Plant Technology Group, Biotechnology &amp; Afforestation Laboratory, New Business Planning Division, Toyota Motor Corporation, 1099 Marune, Kurozasa-cho, Miyoshi, Aichi470-0201, Japan" /><meta name="citation_author" content="Inoue, Takayasu" /><meta name="citation_author_institution" content="Fuji Chemical Co., Ltd., 3-2-33 Higashi-nodamachi, Miyakojima-ku, Osaka 534-0024, Japan" /><meta name="citation_author" content="Shigeki, Yuichi" /><meta name="citation_author_institution" content="Fuji Chemical Co., Ltd., 3-2-33 Higashi-nodamachi, Miyakojima-ku, Osaka 534-0024, Japan" /><meta name="citation_author" content="Ohno, Naoki" /><meta name="citation_author_institution" content="Fuji Chemical Co., Ltd., 3-2-33 Higashi-nodamachi, Miyakojima-ku, Osaka 534-0024, Japan" /><meta name="citation_author" content="Tabata, Satoshi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Akashi, Ryo" /><meta name="citation_author_institution" content="Faculty of Agriculture, University of Miyazaki, 1-1 Gakuenkibanadai-Nishi, Miyazaki 889-2192, Japan" /><meta name="citation_author" content="Sato, Shusei" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, Katahira 2-1-1, Aoba-ku, Sendai 980-8577, Japan" /><meta name="citation_title" content="Sequencing and comparative analyses of the genomes of zoysiagrasses" /><meta name="citation_firstpage" content="171" /><meta name="citation_lastpage" content="180" /><meta name="citation_doi" content="10.1093/dnares/dsw006" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2016/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Research advances in mechanisms of turfgrass tolerance to abiotic stresses: from physiology to molecular biology;  citation_journal_title=Crit. Rev. Plant Sci.;  citation_year=2014;  citation_volume=33;  citation_pages=141-89; " /><meta name="citation_reference" content="citation_title=Salinity tolerance mechanisms of grasses in the subfamily Chloridoideae;  citation_journal_title=Crop Sci.;  citation_year=1999;  citation_volume=39;  citation_pages=1153-60; " /><meta name="citation_reference" content="citation_title=The climate of Japan and its surrounding areas and the distribution and classification of zoysiagrasses;  citation_journal_title=Int. Turfgrass Soc. Res. J.;  citation_year=1989;  citation_volume=6;  citation_pages=17-21; " /><meta name="citation_reference" content="citation_title=Genetic relationships in Zoysia species and the identification of putative interspecific hybrids using simple sequence repeat markers and inflorescence traits;  citation_journal_title=Crop Sci.;  citation_year=2013;  citation_volume=53;  citation_pages=285-95; " /><meta name="citation_reference" content="citation_title=Genetic structure and speciation of zoysiagrass ecotypes collected in Japan;  citation_journal_title=Crop Sci.;  citation_year=2016;  citation_volume=56;  citation_pages=818-26; " /><meta name="citation_reference" content="citation_publisher=CRC Press, Florida;  citation_title=Genetics, Genomics and Breeding of Forage Crops;  citation_year=2014;  citation_pages=168-86; " /><meta name="citation_reference" content="citation_title=Genetic linkage map construction and QTL mapping of salt tolerance traits in zoysiagrass (Zoysia japonica);  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e107249" /><meta name="citation_reference" content="citation_title=Sequence-tagged high-density genetic maps of Zoysia japonica provide insights into genome evolution in Chloridoideae;  citation_journal_title=Plant J.;  citation_year=2015;  citation_volume=82;  citation_pages=744-57; " /><meta name="citation_reference" content="citation_title=Global transcriptome profiles of ‘Meyer’ zoysiagrass in response to cold stress;  citation_journal_title=PLoS One;  citation_year=2015;  citation_volume=10;  citation_pages=e0131153" /><meta name="citation_reference" content="citation_title=De novo assembly of the Japanese lawngrass (Zoysia japonica Steud.) root transcriptome and identification of candidate unigenes related to early responses under salt stress;  citation_journal_title=Front. Plant Sci.;  citation_year=2015;  citation_volume=6;  citation_pages=610" /><meta name="citation_reference" content="citation_title=De novo transcriptome analysis to identify anthocyanin biosynthesis genes responsible for tissue-specific pigmentation in zoysiagrass (Zoysia japonica Steud.);  citation_journal_title=PLoS One;  citation_year=2015;  citation_volume=10;  citation_pages=e0124497" /><meta name="citation_reference" content="citation_title=The complete chloroplast genome sequence of Zoysia matrella;  citation_journal_title=Crop Sci.;  citation_year=2016; " /><meta name="citation_reference" content="citation_title=Making next-generation sequencing work for you: Approaches and practical considerations for marker development and phylogenetics;  citation_journal_title=Plant Ecol. Diversity;  citation_year=2012;  citation_volume=5;  citation_pages=427-50; " /><meta name="citation_reference" content="citation_title=Sequencing depth and coverage: key considerations in genomic analyses;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2014;  citation_volume=15;  citation_pages=121-32; " /><meta name="citation_reference" content="citation_title=Grains of knowledge: Genomics of model cereals;  citation_journal_title=Genome Res.;  citation_year=2005;  citation_volume=15;  citation_pages=1643-50; " /><meta name="citation_reference" content="citation_title=A revised medium for rapid growth and bio assays with tobacco tissue cultures;  citation_journal_title=Physiol. Plant;  citation_year=1986;  citation_volume=15;  citation_pages=473-97; " /><meta name="citation_reference" content="citation_title=Estimations of genome size in Brachiaria species;  citation_journal_title=Grassl. Sci.;  citation_year=2010;  citation_volume=56;  citation_pages=240-2; " /><meta name="citation_reference" content="citation_title=Genome evolution in the genus Sorghum (Poaceae);  citation_journal_title=Ann. Bot.;  citation_year=2005;  citation_volume=95;  citation_pages=219-27; " /><meta name="citation_reference" content="citation_title=Sequence analysis of the genome of carnation (Dianthus caryophyllus L.);  citation_journal_title=DNA Res.;  citation_year=2014;  citation_volume=21;  citation_pages=231-41; " /><meta name="citation_reference" content="citation_title=Survey of error-correction methods for next-generation sequencing;  citation_journal_title=Brief Bioinform.;  citation_year=2013;  citation_volume=14;  citation_pages=56-66; " /><meta name="citation_reference" content="citation_title=Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads;  citation_journal_title=Genome Res.;  citation_year=2014;  citation_volume=24;  citation_pages=1384-95; " /><meta name="citation_reference" content="citation_title=Exploring single-sample SNP and INDEL calling with whole-genome de novo assembly;  citation_journal_title=Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=1838-44; " /><meta name="citation_reference" content="citation_title=Opera: reconstructing optimal genomic scaffolds with high-throughput paired-end sequences;  citation_journal_title=J. Comput. Biol.;  citation_year=2011;  citation_volume=18;  citation_pages=1681-91; " /><meta name="citation_reference" content="citation_title=GMcloser: closing gaps in assemblies accurately with a likelihood-based selection of contig or long-read alignments;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=3733-41; " /><meta name="citation_reference" content="citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome;  citation_journal_title=Nat. Biotechnol.;  citation_year=2011;  citation_volume=29;  citation_pages=644-52; " /><meta name="citation_reference" content="citation_title=CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=1061-7; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs;  citation_journal_title=Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=De novo identification of repeat families in large genomes;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_issue=(Suppl 1);  citation_pages=i351-8; " /><meta name="citation_reference" content="citation_title=SciRoKo: a new tool for whole genome microsatellite search and investigation;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=1683-5; " /><meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence;  citation_journal_title=Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=955-64; " /><meta name="citation_reference" content="citation_title=A new generation of homology search tools based on probabilistic inference;  citation_journal_title=Genome Inform.;  citation_year=2009;  citation_volume=23;  citation_pages=205-11; " /><meta name="citation_reference" content="citation_title=MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects;  citation_journal_title=BMC Bioinformatics;  citation_year=2011;  citation_volume=12;  citation_pages=491" /><meta name="citation_reference" content="citation_title=Automated de novo identification of repeat sequence families in sequenced genomes;  citation_journal_title=Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=1269-76; " /><meta name="citation_reference" content="citation_title=Gene prediction with a hidden Markov model and a new intron submodel;  citation_journal_title=Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_issue=(Suppl 2);  citation_pages=ii215-25; " /><meta name="citation_reference" content="citation_title=Gene finding in novel genomes;  citation_journal_title=BMC Bioinformatics;  citation_year=2004;  citation_volume=5;  citation_pages=59" /><meta name="citation_reference" content="citation_title=TopHat: discovering splice junctions with RNA-Seq;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1105-11; " /><meta name="citation_reference" content="citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation;  citation_journal_title=Nat. Biotechnol.;  citation_year=2010;  citation_volume=28;  citation_pages=511-5; " /><meta name="citation_reference" content="citation_title=Automated generation of heuristics for biological sequence comparison;  citation_journal_title=BMC Bioinformatics;  citation_year=2005;  citation_volume=6;  citation_pages=31" /><meta name="citation_reference" content="citation_title=The COG database: an updated version includes eukaryotes;  citation_journal_title=BMC Bioinformatics;  citation_year=2003;  citation_volume=4;  citation_pages=41" /><meta name="citation_reference" content="citation_title=InterProScan: protein domains identifier;  citation_journal_title=Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=W116-20; " /><meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology; citation_author=The Gene Ontology Consortium;  citation_journal_title=Nat. Genet.;  citation_year=2000;  citation_volume=25;  citation_pages=25-9; " /><meta name="citation_reference" content="citation_title=KEGG: Kyoto encyclopedia of genes and genomes;  citation_journal_title=Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=29-34; " /><meta name="citation_reference" content="citation_title=CD-HIT: accelerated for clustering the next-generation sequencing data;  citation_journal_title=Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=3150-2; " /><meta name="citation_reference" content="citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome;  citation_journal_title=Genome Biol.;  citation_year=2009;  citation_volume=10;  citation_pages=R25" /><meta name="citation_reference" content="citation_title=The Sequence Alignment/Map format and SAMtools;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=Mechanism of salinity tolerance in plants: physiological, biochemical, and molecular characterization;  citation_journal_title=Int. J. Genomics;  citation_year=2014;  citation_volume=2014;  citation_pages=701596" /><meta name="citation_reference" content="citation_title=The low-oxygen induced NAC domain transcription factor ANAC102 affects viability of Arabidopsis thaliana seeds following low-oxygen treatment;  citation_journal_title=Plant Physiol.;  citation_year=2009;  citation_volume=149;  citation_pages=1724-38; " /><meta name="citation_reference" content="citation_title=The BBX family of plant transcription factors;  citation_journal_title=Trends Plant Sci.;  citation_year=2014;  citation_volume=19;  citation_pages=460-70; " /><meta name="citation_reference" content="citation_title=Two classes of plant cDNA clones differentially complement yeast calcineurin mutants and increase salt tolerance of wild-type yeast;  citation_journal_title=J. Biol. Chem.;  citation_year=1996;  citation_volume=271;  citation_pages=12859-66; " /><meta name="citation_reference" content="citation_title=Salt tolerance-related protein STO binds to a Myb transcription factor homologue and confers salt tolerance in Arabidopsis;  citation_journal_title=J. Exp. Bot.;  citation_year=2003;  citation_volume=54;  citation_pages=2231-7; " /><meta name="citation_reference" content="citation_title=Salinity effects on zoysiagrass cultivars and experimental lines;  citation_journal_title=Crop Sci.;  citation_year=2000;  citation_volume=40;  citation_pages=488-92; " /><meta name="citation_reference" content="citation_title=bZIP transcription factors in Arabidopsis;  citation_journal_title=Trends Plant Sci.;  citation_year=2002;  citation_volume=7;  citation_pages=106-11; " /><meta name="citation_reference" content="citation_title=Genome-wide evolutionary characterization and analysis of bZIP transcription factors and their expression profiles in response to multiple abiotic stresses in Brachypodium distachyon;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=227" /><meta name="citation_reference" content="citation_title=Comprehensive analysis of transcriptome response to salinity stress in the halophytic turf grass Sporobolus virginicus;  citation_journal_title=Front. Plant Sci.;  citation_year=2015;  citation_volume=6;  citation_pages=241" /><meta name="citation_reference" content="citation_title=PlantTFDB 3.0: a portal for the functional and evolutionary study of plant transcription factors;  citation_journal_title=Nucleic Acids Res.;  citation_year=2014;  citation_volume=42;  citation_pages=D1182-7; " /><meta name="citation_reference" content="citation_title=Why some leaves are anthocyanic and why most anthocyanic leaves are red?;  citation_journal_title=Flora;  citation_year=2006;  citation_volume=201;  citation_pages=163-77; " /><meta name="citation_reference" content="citation_title=Molecular cloning and function analysis of an anthocyanidin synthase gene from Ginkgo biloba, and its expression in abiotic stress responses;  citation_journal_title=Mol. Cells;  citation_year=2008;  citation_volume=26;  citation_pages=536-47; " /><meta name="citation_reference" content="citation_title=Morphological characteristics of medium-leaf type zoysiagrasses (Zoysia spp.) and their classification using RAPDs;  citation_journal_title=Kor. Turfgrass Sci.;  citation_year=2010;  citation_volume=24;  citation_pages=88-96; " /><meta name="citation_reference" content="citation_title=Genetic variation and interspecific hybridization among natural populations of zoysiagrasses detected by RFLP analyses of chloroplast and nuclear DNA;  citation_journal_title=Genes Genet. Syst.;  citation_year=1997;  citation_volume=72;  citation_pages=173-9; " /><meta name="citation_reference" content="citation_title=Development and characterization of simple sequence repeat markers in Zoysia japonica Steud;  citation_journal_title=Grassl. Sci.;  citation_year=2005;  citation_volume=51;  citation_pages=249-57; " /><meta name="citation_reference" content="citation_title=Plant speciation;  citation_journal_title=Science;  citation_year=2007;  citation_volume=317;  citation_pages=910-4; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/2/171/17463339/dsw006.pdf" /><meta name="description" content="Abstract.  Zoysia is a warm-season turfgrass, which comprises 11 allotetraploid species (2n = 4x = 40), each possessing different morphological and physiologica" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/2/171/1745492" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/2/171/1745492" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/2/171/1745492");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw006';
    </script>
<title>
	Sequencing and comparative analyses of the genomes of zoysiagrasses | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./1745492" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYagATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFuYWthLCBIaWRlbm9yaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZhY3VsdHkgb2YgQWdyaWN1bHR1cmUsIFVuaXZlcnNpdHkgb2YgTWl5YXpha2ksIDEtMSBHYWt1ZW5raWJhbmFkYWktTmlzaGksIE1peWF6YWtpIDg4OS0yMTkyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhpcmFrYXdhLCBIaWRla2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgMi02LTcgS2F6dXNhLWthbWF0YXJpLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS29zdWdpLCBTaHVuaWNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWtheWFtYSwgU2hpbm9idSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPbm8sIEFraWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IldhdGFuYWJlLCBBa2lrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2Eta2FtYXRhcmksIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIYXNoaWd1Y2hpLCBNYXNhdHN1Z3UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGYWN1bHR5IG9mIEFncmljdWx0dXJlLCBVbml2ZXJzaXR5IG9mIE1peWF6YWtpLCAxLTEgR2FrdWVua2liYW5hZGFpLU5pc2hpLCBNaXlhemFraSA4ODktMjE5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHb25kbywgVGFrYWhpcm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGYWN1bHR5IG9mIEFncmljdWx0dXJlLCBVbml2ZXJzaXR5IG9mIE1peWF6YWtpLCAxLTEgR2FrdWVua2liYW5hZGFpLU5pc2hpLCBNaXlhemFraSA4ODktMjE5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJc2hpZ2FraSwgR2Vua2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGYWN1bHR5IG9mIEFncmljdWx0dXJlLCBVbml2ZXJzaXR5IG9mIE1peWF6YWtpLCAxLTEgR2FrdWVua2liYW5hZGFpLU5pc2hpLCBNaXlhemFraSA4ODktMjE5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNdWd1ZXJ6YSwgTWVsb2R5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRmFjdWx0eSBvZiBBZ3JpY3VsdHVyZSwgVW5pdmVyc2l0eSBvZiBNaXlhemFraSwgMS0xIEdha3VlbmtpYmFuYWRhaS1OaXNoaSwgTWl5YXpha2kgODg5LTIxOTIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpbWl6dSwgS2F0c3V5YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlBsYW50IFRlY2hub2xvZ3kgR3JvdXAsIEJpb3RlY2hub2xvZ3kgJmFtcDsgQWZmb3Jlc3RhdGlvbiBMYWJvcmF0b3J5LCBOZXcgQnVzaW5lc3MgUGxhbm5pbmcgRGl2aXNpb24sIFRveW90YSBNb3RvciBDb3Jwb3JhdGlvbiwgMTA5OSBNYXJ1bmUsIEt1cm96YXNhLWNobywgTWl5b3NoaSwgQWljaGk0NzAtMDIwMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXdhbXVyYSwgTm9yaWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUGxhbnQgVGVjaG5vbG9neSBHcm91cCwgQmlvdGVjaG5vbG9neSAmYW1wOyBBZmZvcmVzdGF0aW9uIExhYm9yYXRvcnksIE5ldyBCdXNpbmVzcyBQbGFubmluZyBEaXZpc2lvbiwgVG95b3RhIE1vdG9yIENvcnBvcmF0aW9uLCAxMDk5IE1hcnVuZSwgS3Vyb3phc2EtY2hvLCBNaXlvc2hpLCBBaWNoaTQ3MC0wMjAxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Iklub3VlLCBUYWtheWFzdSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZ1amkgQ2hlbWljYWwgQ28uLCBMdGQuLCAzLTItMzMgSGlnYXNoaS1ub2RhbWFjaGksIE1peWFrb2ppbWEta3UsIE9zYWthIDUzNC0wMDI0LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNoaWdla2ksIFl1aWNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkZ1amkgQ2hlbWljYWwgQ28uLCBMdGQuLCAzLTItMzMgSGlnYXNoaS1ub2RhbWFjaGksIE1peWFrb2ppbWEta3UsIE9zYWthIDUzNC0wMDI0LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9obm8sIE5hb2tpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRnVqaSBDaGVtaWNhbCBDby4sIEx0ZC4sIDMtMi0zMyBIaWdhc2hpLW5vZGFtYWNoaSwgTWl5YWtvamltYS1rdSwgT3Nha2EgNTM0LTAwMjQsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFiYXRhLCBTYXRvc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFrYXNoaSwgUnlvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRmFjdWx0eSBvZiBBZ3JpY3VsdHVyZSwgVW5pdmVyc2l0eSBvZiBNaXlhemFraSwgMS0xIEdha3VlbmtpYmFuYWRhaS1OaXNoaSwgTWl5YXpha2kgODg5LTIxOTIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2F0bywgU2h1c2VpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIDItNi03IEthenVzYS1rYW1hdGFyaSwgS2lzYXJhenUsIENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBMaWZlIFNjaWVuY2VzLCBUb2hva3UgVW5pdmVyc2l0eSwgS2F0YWhpcmEgMi0xLTEsIEFvYmEta3UsIFNlbmRhaSA5ODAtODU3NywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IlNlcXVlbmNpbmcgYW5kIGNvbXBhcmF0aXZlIGFuYWx5c2VzIG9mIHRoZSBnZW5vbWVzIG9mIHpveXNpYWdyYXNzZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxNzEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjE4MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdzAwNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE2LzA0LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVzZWFyY2ggYWR2YW5jZXMgaW4gbWVjaGFuaXNtcyBvZiB0dXJmZ3Jhc3MgdG9sZXJhbmNlIHRvIGFiaW90aWMgc3RyZXNzZXM6IGZyb20gcGh5c2lvbG9neSB0byBtb2xlY3VsYXIgYmlvbG9neTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3JpdC4gUmV2LiBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9MTQxLTg5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TYWxpbml0eSB0b2xlcmFuY2UgbWVjaGFuaXNtcyBvZiBncmFzc2VzIGluIHRoZSBzdWJmYW1pbHkgQ2hsb3JpZG9pZGVhZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3JvcCBTY2kuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9MTE1My02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNsaW1hdGUgb2YgSmFwYW4gYW5kIGl0cyBzdXJyb3VuZGluZyBhcmVhcyBhbmQgdGhlIGRpc3RyaWJ1dGlvbiBhbmQgY2xhc3NpZmljYXRpb24gb2Ygem95c2lhZ3Jhc3NlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW50LiBUdXJmZ3Jhc3MgU29jLiBSZXMuIEouOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0xNy0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyByZWxhdGlvbnNoaXBzIGluIFpveXNpYSBzcGVjaWVzIGFuZCB0aGUgaWRlbnRpZmljYXRpb24gb2YgcHV0YXRpdmUgaW50ZXJzcGVjaWZpYyBoeWJyaWRzIHVzaW5nIHNpbXBsZSBzZXF1ZW5jZSByZXBlYXQgbWFya2VycyBhbmQgaW5mbG9yZXNjZW5jZSB0cmFpdHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNyb3AgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT01MzsgIGNpdGF0aW9uX3BhZ2VzPTI4NS05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBzdHJ1Y3R1cmUgYW5kIHNwZWNpYXRpb24gb2Ygem95c2lhZ3Jhc3MgZWNvdHlwZXMgY29sbGVjdGVkIGluIEphcGFuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Dcm9wIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NTY7ICBjaXRhdGlvbl9wYWdlcz04MTgtMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1DUkMgUHJlc3MsIEZsb3JpZGE7ICBjaXRhdGlvbl90aXRsZT1HZW5ldGljcywgR2Vub21pY3MgYW5kIEJyZWVkaW5nIG9mIEZvcmFnZSBDcm9wczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3BhZ2VzPTE2OC04NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBsaW5rYWdlIG1hcCBjb25zdHJ1Y3Rpb24gYW5kIFFUTCBtYXBwaW5nIG9mIHNhbHQgdG9sZXJhbmNlIHRyYWl0cyBpbiB6b3lzaWFncmFzcyAoWm95c2lhIGphcG9uaWNhKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWUxMDcyNDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZS10YWdnZWQgaGlnaC1kZW5zaXR5IGdlbmV0aWMgbWFwcyBvZiBab3lzaWEgamFwb25pY2EgcHJvdmlkZSBpbnNpZ2h0cyBpbnRvIGdlbm9tZSBldm9sdXRpb24gaW4gQ2hsb3JpZG9pZGVhZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9ODI7ICBjaXRhdGlvbl9wYWdlcz03NDQtNTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdsb2JhbCB0cmFuc2NyaXB0b21lIHByb2ZpbGVzIG9mIOKAmE1leWVy4oCZIHpveXNpYWdyYXNzIGluIHJlc3BvbnNlIHRvIGNvbGQgc3RyZXNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPWUwMTMxMTUzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGUgbm92byBhc3NlbWJseSBvZiB0aGUgSmFwYW5lc2UgbGF3bmdyYXNzIChab3lzaWEgamFwb25pY2EgU3RldWQuKSByb290IHRyYW5zY3JpcHRvbWUgYW5kIGlkZW50aWZpY2F0aW9uIG9mIGNhbmRpZGF0ZSB1bmlnZW5lcyByZWxhdGVkIHRvIGVhcmx5IHJlc3BvbnNlcyB1bmRlciBzYWx0IHN0cmVzczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnJvbnQuIFBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTYxMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlIG5vdm8gdHJhbnNjcmlwdG9tZSBhbmFseXNpcyB0byBpZGVudGlmeSBhbnRob2N5YW5pbiBiaW9zeW50aGVzaXMgZ2VuZXMgcmVzcG9uc2libGUgZm9yIHRpc3N1ZS1zcGVjaWZpYyBwaWdtZW50YXRpb24gaW4gem95c2lhZ3Jhc3MgKFpveXNpYSBqYXBvbmljYSBTdGV1ZC4pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPWUwMTI0NDk3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNvbXBsZXRlIGNobG9yb3BsYXN0IGdlbm9tZSBzZXF1ZW5jZSBvZiBab3lzaWEgbWF0cmVsbGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNyb3AgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFraW5nIG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIHdvcmsgZm9yIHlvdTogQXBwcm9hY2hlcyBhbmQgcHJhY3RpY2FsIGNvbnNpZGVyYXRpb25zIGZvciBtYXJrZXIgZGV2ZWxvcG1lbnQgYW5kIHBoeWxvZ2VuZXRpY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEVjb2wuIERpdmVyc2l0eTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NDI3LTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jaW5nIGRlcHRoIGFuZCBjb3ZlcmFnZToga2V5IGNvbnNpZGVyYXRpb25zIGluIGdlbm9taWMgYW5hbHlzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xMjEtMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdyYWlucyBvZiBrbm93bGVkZ2U6IEdlbm9taWNzIG9mIG1vZGVsIGNlcmVhbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MTY0My01MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSByZXZpc2VkIG1lZGl1bSBmb3IgcmFwaWQgZ3Jvd3RoIGFuZCBiaW8gYXNzYXlzIHdpdGggdG9iYWNjbyB0aXNzdWUgY3VsdHVyZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBoeXNpb2wuIFBsYW50OyAgY2l0YXRpb25feWVhcj0xOTg2OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9NDczLTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fc3RpbWF0aW9ucyBvZiBnZW5vbWUgc2l6ZSBpbiBCcmFjaGlhcmlhIHNwZWNpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdyYXNzbC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01NjsgIGNpdGF0aW9uX3BhZ2VzPTI0MC0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgZXZvbHV0aW9uIGluIHRoZSBnZW51cyBTb3JnaHVtIChQb2FjZWFlKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5uLiBCb3QuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTk1OyAgY2l0YXRpb25fcGFnZXM9MjE5LTI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZSBhbmFseXNpcyBvZiB0aGUgZ2Vub21lIG9mIGNhcm5hdGlvbiAoRGlhbnRodXMgY2FyeW9waHlsbHVzIEwuKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0yMzEtNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN1cnZleSBvZiBlcnJvci1jb3JyZWN0aW9uIG1ldGhvZHMgZm9yIG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CcmllZiBCaW9pbmZvcm0uOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9NTYtNjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmljaWVudCBkZSBub3ZvIGFzc2VtYmx5IG9mIGhpZ2hseSBoZXRlcm96eWdvdXMgZ2Vub21lcyBmcm9tIHdob2xlLWdlbm9tZSBzaG90Z3VuIHNob3J0IHJlYWRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTEzODQtOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGxvcmluZyBzaW5nbGUtc2FtcGxlIFNOUCBhbmQgSU5ERUwgY2FsbGluZyB3aXRoIHdob2xlLWdlbm9tZSBkZSBub3ZvIGFzc2VtYmx5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTE4MzgtNDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9wZXJhOiByZWNvbnN0cnVjdGluZyBvcHRpbWFsIGdlbm9taWMgc2NhZmZvbGRzIHdpdGggaGlnaC10aHJvdWdocHV0IHBhaXJlZC1lbmQgc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBDb21wdXQuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MTY4MS05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R01jbG9zZXI6IGNsb3NpbmcgZ2FwcyBpbiBhc3NlbWJsaWVzIGFjY3VyYXRlbHkgd2l0aCBhIGxpa2VsaWhvb2QtYmFzZWQgc2VsZWN0aW9uIG9mIGNvbnRpZyBvciBsb25nLXJlYWQgYWxpZ25tZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0zNzMzLTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdWxsLWxlbmd0aCB0cmFuc2NyaXB0b21lIGFzc2VtYmx5IGZyb20gUk5BLVNlcSBkYXRhIHdpdGhvdXQgYSByZWZlcmVuY2UgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9NjQ0LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DRUdNQTogYSBwaXBlbGluZSB0byBhY2N1cmF0ZWx5IGFubm90YXRlIGNvcmUgZ2VuZXMgaW4gZXVrYXJ5b3RpYyBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTEwNjEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2FwcGVkIEJMQVNUIGFuZCBQU0ktQkxBU1Q6IGEgbmV3IGdlbmVyYXRpb24gb2YgcHJvdGVpbiBkYXRhYmFzZSBzZWFyY2ggcHJvZ3JhbXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTMzODktNDAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZSBub3ZvIGlkZW50aWZpY2F0aW9uIG9mIHJlcGVhdCBmYW1pbGllcyBpbiBsYXJnZSBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX2lzc3VlPShTdXBwbCAxKTsgIGNpdGF0aW9uX3BhZ2VzPWkzNTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2NpUm9LbzogYSBuZXcgdG9vbCBmb3Igd2hvbGUgZ2Vub21lIG1pY3Jvc2F0ZWxsaXRlIHNlYXJjaCBhbmQgaW52ZXN0aWdhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xNjgzLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXRSTkFzY2FuLVNFOiBhIHByb2dyYW0gZm9yIGltcHJvdmVkIGRldGVjdGlvbiBvZiB0cmFuc2ZlciBSTkEgZ2VuZXMgaW4gZ2Vub21pYyBzZXF1ZW5jZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9OTU1LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5ldyBnZW5lcmF0aW9uIG9mIGhvbW9sb2d5IHNlYXJjaCB0b29scyBiYXNlZCBvbiBwcm9iYWJpbGlzdGljIGluZmVyZW5jZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEluZm9ybS47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0yMDUtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1BS0VSMjogYW4gYW5ub3RhdGlvbiBwaXBlbGluZSBhbmQgZ2Vub21lLWRhdGFiYXNlIG1hbmFnZW1lbnQgdG9vbCBmb3Igc2Vjb25kLWdlbmVyYXRpb24gZ2Vub21lIHByb2plY3RzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz00OTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BdXRvbWF0ZWQgZGUgbm92byBpZGVudGlmaWNhdGlvbiBvZiByZXBlYXQgc2VxdWVuY2UgZmFtaWxpZXMgaW4gc2VxdWVuY2VkIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTI2OS03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBwcmVkaWN0aW9uIHdpdGggYSBoaWRkZW4gTWFya292IG1vZGVsIGFuZCBhIG5ldyBpbnRyb24gc3VibW9kZWw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25faXNzdWU9KFN1cHBsIDIpOyAgY2l0YXRpb25fcGFnZXM9aWkyMTUtMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgZmluZGluZyBpbiBub3ZlbCBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTU5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG9wSGF0OiBkaXNjb3ZlcmluZyBzcGxpY2UganVuY3Rpb25zIHdpdGggUk5BLVNlcTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xMTA1LTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0IGFzc2VtYmx5IGFuZCBxdWFudGlmaWNhdGlvbiBieSBSTkEtU2VxIHJldmVhbHMgdW5hbm5vdGF0ZWQgdHJhbnNjcmlwdHMgYW5kIGlzb2Zvcm0gc3dpdGNoaW5nIGR1cmluZyBjZWxsIGRpZmZlcmVudGlhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTUxMS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BdXRvbWF0ZWQgZ2VuZXJhdGlvbiBvZiBoZXVyaXN0aWNzIGZvciBiaW9sb2dpY2FsIHNlcXVlbmNlIGNvbXBhcmlzb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MzEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQ09HIGRhdGFiYXNlOiBhbiB1cGRhdGVkIHZlcnNpb24gaW5jbHVkZXMgZXVrYXJ5b3RlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz00MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVyUHJvU2NhbjogcHJvdGVpbiBkb21haW5zIGlkZW50aWZpZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPVcxMTYtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgb250b2xvZ3k6IHRvb2wgZm9yIHRoZSB1bmlmaWNhdGlvbiBvZiBiaW9sb2d5OyBjaXRhdGlvbl9hdXRob3I9VGhlIEdlbmUgT250b2xvZ3kgQ29uc29ydGl1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yNS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1LRUdHOiBLeW90byBlbmN5Y2xvcGVkaWEgb2YgZ2VuZXMgYW5kIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTI5LTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DRC1ISVQ6IGFjY2VsZXJhdGVkIGZvciBjbHVzdGVyaW5nIHRoZSBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZyBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTMxNTAtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VWx0cmFmYXN0IGFuZCBtZW1vcnktZWZmaWNpZW50IGFsaWdubWVudCBvZiBzaG9ydCBETkEgc2VxdWVuY2VzIHRvIHRoZSBodW1hbiBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPVIyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBTZXF1ZW5jZSBBbGlnbm1lbnQvTWFwIGZvcm1hdCBhbmQgU0FNdG9vbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MjA3OC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWNoYW5pc20gb2Ygc2FsaW5pdHkgdG9sZXJhbmNlIGluIHBsYW50czogcGh5c2lvbG9naWNhbCwgYmlvY2hlbWljYWwsIGFuZCBtb2xlY3VsYXIgY2hhcmFjdGVyaXphdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW50LiBKLiBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDE0OyAgY2l0YXRpb25fcGFnZXM9NzAxNTk2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGxvdy1veHlnZW4gaW5kdWNlZCBOQUMgZG9tYWluIHRyYW5zY3JpcHRpb24gZmFjdG9yIEFOQUMxMDIgYWZmZWN0cyB2aWFiaWxpdHkgb2YgQXJhYmlkb3BzaXMgdGhhbGlhbmEgc2VlZHMgZm9sbG93aW5nIGxvdy1veHlnZW4gdHJlYXRtZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDk7ICBjaXRhdGlvbl9wYWdlcz0xNzI0LTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQkJYIGZhbWlseSBvZiBwbGFudCB0cmFuc2NyaXB0aW9uIGZhY3RvcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9NDYwLTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ud28gY2xhc3NlcyBvZiBwbGFudCBjRE5BIGNsb25lcyBkaWZmZXJlbnRpYWxseSBjb21wbGVtZW50IHllYXN0IGNhbGNpbmV1cmluIG11dGFudHMgYW5kIGluY3JlYXNlIHNhbHQgdG9sZXJhbmNlIG9mIHdpbGQtdHlwZSB5ZWFzdDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MjcxOyAgY2l0YXRpb25fcGFnZXM9MTI4NTktNjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNhbHQgdG9sZXJhbmNlLXJlbGF0ZWQgcHJvdGVpbiBTVE8gYmluZHMgdG8gYSBNeWIgdHJhbnNjcmlwdGlvbiBmYWN0b3IgaG9tb2xvZ3VlIGFuZCBjb25mZXJzIHNhbHQgdG9sZXJhbmNlIGluIEFyYWJpZG9wc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBFeHAuIEJvdC47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz0yMjMxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNhbGluaXR5IGVmZmVjdHMgb24gem95c2lhZ3Jhc3MgY3VsdGl2YXJzIGFuZCBleHBlcmltZW50YWwgbGluZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNyb3AgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPTQ4OC05MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9YlpJUCB0cmFuc2NyaXB0aW9uIGZhY3RvcnMgaW4gQXJhYmlkb3BzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0xMDYtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGV2b2x1dGlvbmFyeSBjaGFyYWN0ZXJpemF0aW9uIGFuZCBhbmFseXNpcyBvZiBiWklQIHRyYW5zY3JpcHRpb24gZmFjdG9ycyBhbmQgdGhlaXIgZXhwcmVzc2lvbiBwcm9maWxlcyBpbiByZXNwb25zZSB0byBtdWx0aXBsZSBhYmlvdGljIHN0cmVzc2VzIGluIEJyYWNoeXBvZGl1bSBkaXN0YWNoeW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0yMjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wcmVoZW5zaXZlIGFuYWx5c2lzIG9mIHRyYW5zY3JpcHRvbWUgcmVzcG9uc2UgdG8gc2FsaW5pdHkgc3RyZXNzIGluIHRoZSBoYWxvcGh5dGljIHR1cmYgZ3Jhc3MgU3Bvcm9ib2x1cyB2aXJnaW5pY3VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Gcm9udC4gUGxhbnQgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MjQxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGxhbnRURkRCIDMuMDogYSBwb3J0YWwgZm9yIHRoZSBmdW5jdGlvbmFsIGFuZCBldm9sdXRpb25hcnkgc3R1ZHkgb2YgcGxhbnQgdHJhbnNjcmlwdGlvbiBmYWN0b3JzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz1EMTE4Mi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaHkgc29tZSBsZWF2ZXMgYXJlIGFudGhvY3lhbmljIGFuZCB3aHkgbW9zdCBhbnRob2N5YW5pYyBsZWF2ZXMgYXJlIHJlZD87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZsb3JhOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTIwMTsgIGNpdGF0aW9uX3BhZ2VzPTE2My03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGNsb25pbmcgYW5kIGZ1bmN0aW9uIGFuYWx5c2lzIG9mIGFuIGFudGhvY3lhbmlkaW4gc3ludGhhc2UgZ2VuZSBmcm9tIEdpbmtnbyBiaWxvYmEsIGFuZCBpdHMgZXhwcmVzc2lvbiBpbiBhYmlvdGljIHN0cmVzcyByZXNwb25zZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQ2VsbHM7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz01MzYtNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vcnBob2xvZ2ljYWwgY2hhcmFjdGVyaXN0aWNzIG9mIG1lZGl1bS1sZWFmIHR5cGUgem95c2lhZ3Jhc3NlcyAoWm95c2lhIHNwcC4pIGFuZCB0aGVpciBjbGFzc2lmaWNhdGlvbiB1c2luZyBSQVBEczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9S29yLiBUdXJmZ3Jhc3MgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTg4LTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIHZhcmlhdGlvbiBhbmQgaW50ZXJzcGVjaWZpYyBoeWJyaWRpemF0aW9uIGFtb25nIG5hdHVyYWwgcG9wdWxhdGlvbnMgb2Ygem95c2lhZ3Jhc3NlcyBkZXRlY3RlZCBieSBSRkxQIGFuYWx5c2VzIG9mIGNobG9yb3BsYXN0IGFuZCBudWNsZWFyIEROQTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXMgR2VuZXQuIFN5c3QuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTcyOyAgY2l0YXRpb25fcGFnZXM9MTczLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHNpbXBsZSBzZXF1ZW5jZSByZXBlYXQgbWFya2VycyBpbiBab3lzaWEgamFwb25pY2EgU3RldWQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdyYXNzbC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT01MTsgIGNpdGF0aW9uX3BhZ2VzPTI0OS01NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGxhbnQgc3BlY2lhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTc7ICBjaXRhdGlvbl9wYWdlcz05MTAtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIzLzIvMTcxLzE3NDYzMzM5L2RzdzAwNi5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgWm95c2lhIGlzIGEgd2FybS1zZWFzb24gdHVyZmdyYXNzLCB3aGljaCBjb21wcmlzZXMgMTEgYWxsb3RldHJhcGxvaWQgc3BlY2llcyAoMm4gPSA0eCA9IDQwKSwgZWFjaCBwb3NzZXNzaW5nIGRpZmZlcmVudCBtb3JwaG9sb2dpY2FsIGFuZCBwaHlzaW9sb2dpY2EiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTE3NDU0OTImeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2Rk2lN+0izsM6+mIHeCA9IVBh8t8oU=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f171%2f1745492" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f171%2f1745492" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=69060" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/2/1/m_cover.gif?Expires=2147483647&amp;Signature=H~-rl2t0~4icOqxuxQciOuaILaWLPqFGH3A~nroLlKomW3yjgtZ~qWZv6XD0n7MjtydtUqNoJUBHP0IS9FeSlJ7e824tplkVC9mcToG2lma9nT3MqHOrgXg-FmGm8R1Hko1D-xKC7m9goSRdTV0PpCgY514~J0A53r05NXU-LixEnWtvnkrSQlIKg2cNSOr5m2suFRYKdAKYSGWqTMps1eFybyDDmECiZJwPFJr9hkzFGuT0qtGzqG0CshylrK0bC2jCz4U60gjwvUfzNmkXc68xyAxGKYJwvJo89NhQiuWScVg0wHNDvMxjguS6KYmoQK6GthtPkW1VFz6XLn-4lQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/2">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87200842">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200842">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200844">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200844">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200848">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200848">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200871">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200871">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200926">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200926">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200928">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200928">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87200934">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87200934">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/2/155/1745431">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Sequencing and comparative analyses of the genomes of zoysiagrasses
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hidenori Tanaka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Faculty of Agriculture</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">1-1 Gakuenkibanadai-Nishi, Miyazaki 889-2192</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <div class="title -title">†</div><p>These two authors equally contributed to this work.</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hidenori+Tanaka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tanaka H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tanaka H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hidenori Tanaka</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Hirakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <div class="title -title">†</div><p>These two authors equally contributed to this work.</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Hirakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirakawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirakawa H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hideki Hirakawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shunichi Kosugi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shunichi+Kosugi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kosugi S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kosugi S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shunichi Kosugi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shinobu Nakayama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shinobu+Nakayama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakayama S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakayama S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shinobu Nakayama</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akiko Ono</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akiko+Ono">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ono A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ono A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Akiko Ono</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akiko Watanabe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akiko+Watanabe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Watanabe A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Watanabe A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Akiko Watanabe</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masatsugu Hashiguchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Faculty of Agriculture</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">1-1 Gakuenkibanadai-Nishi, Miyazaki 889-2192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masatsugu+Hashiguchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hashiguchi M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hashiguchi M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Masatsugu Hashiguchi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takahiro Gondo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Faculty of Agriculture</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">1-1 Gakuenkibanadai-Nishi, Miyazaki 889-2192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takahiro+Gondo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gondo T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gondo T&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Takahiro Gondo</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Genki Ishigaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Faculty of Agriculture</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">1-1 Gakuenkibanadai-Nishi, Miyazaki 889-2192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Genki+Ishigaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ishigaki G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ishigaki G&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Genki Ishigaki</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Melody Muguerza</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Faculty of Agriculture</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">1-1 Gakuenkibanadai-Nishi, Miyazaki 889-2192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Melody+Muguerza">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Muguerza M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Muguerza M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Melody Muguerza</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Katsuya Shimizu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Plant Technology Group, Biotechnology &amp; Afforestation Laboratory, New Business Planning Division</div>, <div class="insititution">Toyota Motor Corporation</div>, <div class="addr-line">1099 Marune, Kurozasa-cho, Miyoshi, Aichi</div><div class="addr-line">470-0201</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Katsuya+Shimizu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shimizu K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shimizu K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Katsuya Shimizu</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Noriko Sawamura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Plant Technology Group, Biotechnology &amp; Afforestation Laboratory, New Business Planning Division</div>, <div class="insititution">Toyota Motor Corporation</div>, <div class="addr-line">1099 Marune, Kurozasa-cho, Miyoshi, Aichi</div><div class="addr-line">470-0201</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Noriko+Sawamura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sawamura N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sawamura N&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Noriko Sawamura</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takayasu Inoue</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">Fuji Chemical Co., Ltd.</div>, <div class="addr-line">3-2-33 Higashi-nodamachi, Miyakojima-ku, Osaka 534-0024</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takayasu+Inoue">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Inoue T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Inoue T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Takayasu Inoue</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuichi Shigeki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">Fuji Chemical Co., Ltd.</div>, <div class="addr-line">3-2-33 Higashi-nodamachi, Miyakojima-ku, Osaka 534-0024</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuichi+Shigeki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shigeki Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shigeki Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yuichi Shigeki</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Naoki Ohno</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">Fuji Chemical Co., Ltd.</div>, <div class="addr-line">3-2-33 Higashi-nodamachi, Miyakojima-ku, Osaka 534-0024</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Naoki+Ohno">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohno N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohno N&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Naoki Ohno</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Tabata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Tabata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tabata S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tabata S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Satoshi Tabata</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ryo Akashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Faculty of Agriculture</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">1-1 Gakuenkibanadai-Nishi, Miyazaki 889-2192</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ryo+Akashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Akashi R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Akashi R&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Ryo Akashi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shusei Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">2-6-7 Kazusa-kamatari, Kisarazu, Chiba 292-0818</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">5</div><div class="addr-line">Graduate School of Life Sciences</div>, <div class="insititution">Tohoku University</div>, <div class="addr-line">Katahira 2-1-1, Aoba-ku, Sendai 980-8577</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Graduate School of Life Sciences, Tohoku University, Katahira 2-1-1, Aoba-ku, Sendai 980-8577, Japan. Tel. +81-22-217-5688. Fax. +81-22-217-5691. E-mail: <a href="mailto:shuseis@ige.tohoku.ac.jp" target="_blank">shuseis@ige.tohoku.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shusei+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Shusei Sato</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These two authors equally contributed to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN2"><div class="fn"><div class="footnote-content">Edited by Dr Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 2, 1 April 2016, Pages 171–180, <a href='https://doi.org/10.1093/dnares/dsw006'>https://doi.org/10.1093/dnares/dsw006</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">14 March 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">28 December 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">01 February 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="1745492" href="/dnaresearch/article-pdf/23/2/171/17463339/dsw006.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Hidenori Tanaka, Hideki Hirakawa, Shunichi Kosugi, Shinobu Nakayama, Akiko Ono, Akiko Watanabe, Masatsugu Hashiguchi, Takahiro Gondo, Genki Ishigaki, Melody Muguerza, Katsuya Shimizu, Noriko Sawamura, Takayasu Inoue, Yuichi Shigeki, Naoki Ohno, Satoshi Tabata, Ryo Akashi, Shusei Sato;  Sequencing and comparative analyses of the genomes of zoysiagrasses, <em>DNA Research</em>, Volume 23, Issue 2, 1 April 2016, Pages 171–180, <a href='https://doi.org/10.1093/dnares/dsw006'>https://doi.org/10.1093/dnares/dsw006</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/1745492?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745492?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745492?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745492?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745492?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745492?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745492?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Sequencing%20and%20comparative%20analyses%20of%20the%20genomes%20of%20zoysiagrasses&amp;publicationDate=2016-03-14&amp;volumeNum=23&amp;issueNum=2&amp;author=Tanaka%2C%20Hidenori%3B%20Hirakawa%2C%20Hideki&amp;startPage=171&amp;endPage=180&amp;contentId=10.1093%2Fdnares%2Fdsw006&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f171%2f1745492" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87200842 id="87200842" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Zoysia</em> is a warm-season turfgrass, which comprises 11 allotetraploid species (2<em>n</em> = 4<em>x</em> = 40), each possessing different morphological and physiological traits. To characterize the genetic systems of <em>Zoysia</em> plants and to analyse their structural and functional differences in individual species and accessions, we sequenced the genomes of <em>Zoysia</em> species using HiSeq and MiSeq platforms. As a reference sequence of <em>Zoysia</em> species, we generated a high-quality draft sequence of the genome of <em>Z. japonica</em> accession ‘Nagirizaki’ (334 Mb) in which 59,271 protein-coding genes were predicted. In parallel, draft genome sequences of <em>Z. matrella</em> ‘Wakaba’ and <em>Z. pacifica</em> ‘Zanpa’ were also generated for comparative analyses. To investigate the genetic diversity among the <em>Zoysia</em> species, genome sequence reads of three additional accessions, <em>Z. japonica</em> ‘Kyoto’, <em>Z. japonica</em> ‘Miyagi’ and <em>Z. matrella</em> ‘Chiba Fair Green’, were accumulated, and aligned against the reference genome of ‘Nagirizaki’ along with those from ‘Wakaba’ and ‘Zanpa’. As a result, we detected 7,424,163 single-nucleotide polymorphisms and 852,488 short indels among these species. The information obtained in this study will be valuable for basic studies on zoysiagrass evolution and genetics as well as for the breeding of zoysiagrasses, and is made available in the ‘Zoysia Genome Database’ at <a class="link link-uri" href="http://zoysia.kazusa.or.jp">http://zoysia.kazusa.or.jp</a>.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="Zoysia"><em>Zoysia</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="turfgrass">turfgrass</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome sequencing&quot;">genome sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene prediction&quot;">gene prediction</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;comparative analysis&quot;">comparative analysis</a></div>                            <h2 scrollto-destination=87200844 id="87200844" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Members of the genus <em>Zoysia</em> are some of the most cold- and salt-tolerant plants among warm-season turfgrasses.<sup><a reveal-id="DSW006C1" data-open="DSW006C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSW006C2" data-open="DSW006C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Commonly known as zoysiagrasses, <em>Zoysia</em> species offer good turf qualities, including hardiness, salt tolerance, resistance to drought, shade, and heat, and they are preferred for their aesthetic characteristics. <em>Zoysia</em> comprises 11 species, and all of which are allotetraploids (2<em>n</em> = 4<em>x</em> = 40). Originating from East Asia (Japan, Korea, China), these out-crossing and mat-forming perennial grasses have spread throughout the Pacific Rim and have been used for centuries as cover plants for burial mounds according to the customs in the Orient. Five species such as <em>Z. japonica</em> Steudel, <em>Z. matrella</em> (L.) Merrill, <em>Z. pacifica</em> (Goudswaard) M. Hotta &amp; Kuroki [syn. <em>Z. matrella</em> (L.) Merr. var. pacifica Goudswaard], <em>Z. sinica</em> Hance, and <em>Z. macrostachya</em> Franchet &amp; Savatier have been found in Japan, and they exhibit wide morphological and ecological differences.<sup><a reveal-id="DSW006C3" data-open="DSW006C3" class="link link-ref link-reveal xref-bibr">3</a></sup></p><p>Zoysiagrasses have received less attention from the research community than other Poaceae family members. However, several research groups have conducted genetic studies on zoysiagrasses in an attempt to evaluate their genetic variation and clarify their phylogenetic relationships. Studies comparing domestic zoysiagrass accessions and wild species have revealed high rates of introgression and hybridization, particularly among <em>Z. japonica</em>, <em>Z. matrella</em>, and <em>Z. pacifica</em>.<sup><a reveal-id="DSW006C4" data-open="DSW006C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSW006C5" data-open="DSW006C5" class="link link-ref link-reveal xref-bibr">5</a></sup> High-density genetic linkage maps have been constructed in many crop species, to promote map-based gene cloning and marker-enhanced breeding. Such maps have been produced for zoysiagrasses based on restriction and amplified fragment length polymorphisms, simple sequence repeats (SSRs), and combinations of these markers, and the maps have been used in quantitative trait loci analyses.<sup><a reveal-id="DSW006C4" data-open="DSW006C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSW006C6" data-open="DSW006C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSW006C7" data-open="DSW006C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Moreover, sequence-tagged high-density genetic maps of <em>Z. japonica</em> were constructed and compared with those of <em>Oryza sativa</em> and <em>Sorghum bicolor</em>, revealing a highly conserved collinearity in the gene order of these genomes.<sup><a reveal-id="DSW006C8" data-open="DSW006C8" class="link link-ref link-reveal xref-bibr">8</a></sup> In parallel with the construction of genetic maps, transcriptome analyses have been used to identify abiotic stress-related genes<sup><a reveal-id="DSW006C9" data-open="DSW006C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSW006C10" data-open="DSW006C10" class="link link-ref link-reveal xref-bibr">10</a></sup> and genes that control tissue-specific anthocyanin pigmentation in <em>Z. japonica</em>.<sup><a reveal-id="DSW006C11" data-open="DSW006C11" class="link link-ref link-reveal xref-bibr">11</a></sup> A complete sequence of the chloroplast genome of <em>Z. matrella</em> L. Merr has also been published.<sup><a reveal-id="DSW006C12" data-open="DSW006C12" class="link link-ref link-reveal xref-bibr">12</a></sup></p><p>Next-generation sequencing technologies have enabled the acquisition of large sequence datasets from plant nuclear and organelle genomes. These genetic resources offer significant impacts on plant research in all disciplines, including systematics, genetics, co-evolution, conservation genetics, and breeding.<sup><a reveal-id="DSW006C13" data-open="DSW006C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Whole-genome sequencing along with precise data analysis by means of bioinformatics has allowed the discovery of new genes and regulatory sequences, and can integrate variant types including indels, structural variants and copy number variations.<sup><a reveal-id="DSW006C14" data-open="DSW006C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Genome and gene sequences have been accumulated for many model and crop plant species. However, much more work is needed to clarify relationships and variations across the Poaceae family, particularly in the subfamily Chloridoideae, which consists of 1,601 species in 131 genera that possess the Kranz or C4 anatomy and chloridoid bicellular microhairs.<sup><a reveal-id="DSW006C15" data-open="DSW006C15" class="link link-ref link-reveal xref-bibr">15</a></sup> For example, there were only 1,004 sequences from <em>Zoysia</em> spp. found in GenBank at the time of this writing. Here, we report on the generation and comparison of high-quality draft sequences of zoysiagrass genomes derived from native accessions of <em>Z. japonica</em>, <em>Z. matrella</em> and <em>Z. pacifica</em> in Japan. Furthermore, we compared the zoysiagrass genomes with those of related species: <em>O. sativa, S. bicolor</em>, and <em>Brachypodium distachyon</em>.</p>                            <h2 scrollto-destination=87200848 id="87200848" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87200849 id="87200849" class="section-title" data-legacy-id=s2a>2.1. Plant materials</h3>
<p>Six accessions of three species (<em>Z. japonica</em>, <em>Z. matrella</em> and <em>Z. pacifica</em>) were used in this study. These accessions are maintained at the Kibana Agricultural Science Station, Field Science Center, University of Miyazaki, Japan. An accession of <em>Z. japonica, ‘</em>Nagirizaki’, was collected from Tsushima, Nagasaki. We obtained stolons from ‘Nagirizaki’ plants and cut them into 2–3 cm segments with nodes. The segments were surface sterilized using a standard protocol. The shoot apices were excised from the nodes and cultured on half-strength hormone-free Murashige and Skoog medium<sup><a reveal-id="DSW006C16" data-open="DSW006C16" class="link link-ref link-reveal xref-bibr">16</a></sup> solidified with 0.7% agar in test tubes for regeneration. The regenerated leaves were used for whole-genome shotgun sequencing. The other accessions were as follows: <em>Z. japonica ‘</em>Miyagi’ from Ishinomaki, Miyagi; <em>Z. japonica ‘</em>Kyoto’ from Kyoto, Kyoto; and <em>Z. matrella ‘</em>Wakaba’, <em>Z. matrella</em> ‘Chiba Fair Green’, and <em>Z. pacifica ‘</em>Zanpa’ from Yomitan-son, Okinawa. Shoots of these accessions were briefly sterilized in 2% (v/v) sodium hypochlorite solution followed by three rinses with sterile water, and transplanted into pots (diameter: 6.0 cm, depth: 5.5 cm) filled with sterilized vermiculite and covered with plastic containers. All accessions were incubated at 31°C under 16 h light (30 µmol m<sup>−2</sup> s<sup>−1</sup>) and 8 h dark condition, and their leaves were used for comparative studies.</p>                            <h3 scrollto-destination=87200851 id="87200851" class="section-title" data-legacy-id=s2b>2.2. Flow cytometry</h3>
<p>Flow cytometry was conducted to estimate the genome sizes of the six <em>Zoysia</em> accessions according to Ishigaki et al.<sup><a reveal-id="DSW006C17" data-open="DSW006C17" class="link link-ref link-reveal xref-bibr">17</a></sup><em>Sorghum bicolor</em> ‘Tx623’ was used as the internal standard with a DNA content of 1.67 pg 2C<sup>−1</sup>.<sup><a reveal-id="DSW006C18" data-open="DSW006C18" class="link link-ref link-reveal xref-bibr">18</a></sup> The fluorescent intensity of each nuclear suspension was measured using an EPICS XL equipped with a 488 nm argon laser and a long path filter (Beckman Coulter, Inc., CA, USA). The analysis was replicated three times for each sample.</p>                            <h3 scrollto-destination=87200853 id="87200853" class="section-title" data-legacy-id=s2c>2.3. Shotgun sequencing of zoysiagrass genomes</h3>
<p>Whole-genome shotgun sequencing of the <em>Zoysia</em> species was performed using both the HiSeq 2000 and MiSeq platforms (Illumina Inc., CA, USA). Genomic DNA extracted from leaves was used for library construction according to standard protocols. The paired-end (PE) libraries were constructed using the manufacture's protocol (Illumina), and the mate-pair (MP) libraries were constructed using GS Titanium Library Paired End Adaptors (Roche Diagnostics, Basel, Switzerland) as described previously.<sup><a reveal-id="DSW006C19" data-open="DSW006C19" class="link link-ref link-reveal xref-bibr">19</a></sup> The libraries sequenced and platforms used for each accession are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>. For sequencing the genomes of <em>Z. japonica</em> ‘Nagirizaki’, <em>Z. matrella</em> ‘Wakaba’, and <em>Z. pacifica</em> ‘Zanpa’, PE reads with insert sizes of 700 bp were sequenced using the MiSeq platform. For ‘Nagirizaki’, MP reads with insert sizes of 5, 10, and 15 kb were sequenced using the HiSeq 2000 platform. For ‘Wakaba’ and ‘Zanpa’, MP reads with insert sizes of 3 kb were sequenced using HiSeq 2000. For the other three accessions, such as <em>Z. japonica</em> ‘Kyoto’, <em>Z. japonica</em> ‘Miyagi’, and <em>Z. matrella</em> ‘Chiba Fair Green’, PE reads with insert sizes of 300 bp were sequenced using the HiSeq 2000 platform. The k-mer frequency analysis and estimation of genome sizes were performed using SOAPec, ver. 2.0.1,<sup><a reveal-id="DSW006C20" data-open="DSW006C20" class="link link-ref link-reveal xref-bibr">20</a></sup> using the PE reads of each accession.</p>                            <h3 scrollto-destination=87200855 id="87200855" class="section-title" data-legacy-id=s2d>2.4. Assembly of genome sequences and RNA-Seq analysis</h3>
<p>Genome sequence assembly was performed for <em>Z. japonica</em> ‘Nagirizaki’, <em>Z. matrella</em> ‘Wakaba’, and <em>Z. pacifica</em> ‘Zanpa’ using the MiSeq PE and HiSeq MP reads. The adaptor sequences in the PE and MP reads were trimmed, and polymerase chain reaction (PCR) duplicates in the MP reads were excluded. For ‘Nagirizaki’, the MiSeq and HiSeq 2000 reads were assembled into contigs using Platanus, v1.2.1.<sup><a reveal-id="DSW006C21" data-open="DSW006C21" class="link link-ref link-reveal xref-bibr">21</a></sup> The scaffolding in Platanus was conducted under strict conditions with options –l 5 and –s 72. For ‘Wakaba’ and ‘Zanpa’, contigs were assembled from the MiSeq PE reads using DISCOVAR <em>de novo</em> ver. 51598 (<a class="link link-uri" href="http://www.broadinstitute.org/software/discovar/blog/">http://www.broadinstitute.org/software/discovar/blog/</a>) and FERMI, v1.1.<sup><a reveal-id="DSW006C22" data-open="DSW006C22" class="link link-ref link-reveal xref-bibr">22</a></sup> After the comparison of the status of resulted contigs by two assemblers, the contig sets generated by DISCOVAR <em>de novo</em> were further assembled with the PE and MP reads to generate scaffolds using Opera, v2.0.<sup><a reveal-id="DSW006C23" data-open="DSW006C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Gaps in the ‘Wakaba’ and ‘Zanpa’ scaffolds were filled by a sequential treatment with the two contig sets (the DISCOVAR <em>de novo</em>-contig set and the FERMI-contig set) using GMcloser.<sup><a reveal-id="DSW006C24" data-open="DSW006C24" class="link link-ref link-reveal xref-bibr">24</a></sup></p><p>For RNA-Seq analysis, PE libraries were prepared using 1 µg of total RNA extracted from young leaves and roots of <em>‘</em>Nagirizaki’ using the TruSeq Stranded Total RNA with Ribo-Zero Plant kit (Illumina). The constructed libraries were sequenced using the HiSeq 2000 platform. The reads were assembled using Trinity, ver. 2.0,<sup><a reveal-id="DSW006C25" data-open="DSW006C25" class="link link-ref link-reveal xref-bibr">25</a></sup> with the default parameters except that the min_kmer_cov option was 3, and the resulting assembled contigs with &lt;500 bases were filtered out.</p>                            <h3 scrollto-destination=87200858 id="87200858" class="section-title" data-legacy-id=s2e>2.5. Validation of assembled sequences</h3>
<p>To confirm the genome coverage and the accuracy of the assembled genome sequences, the core eukaryotic genes were mapped using CEGMA, v2.5.<sup><a reveal-id="DSW006C26" data-open="DSW006C26" class="link link-ref link-reveal xref-bibr">26</a></sup> The quality of the assembled genome sequences was also confirmed using published DNA markers developed by Wang et al.<sup><a reveal-id="DSW006C8" data-open="DSW006C8" class="link link-ref link-reveal xref-bibr">8</a></sup> The marker sequences of <em>Z. japonica</em> ‘El Toro’ linkage groups were mapped onto the genome sequences of ‘Nagirizaki’ using BLASTN<sup><a reveal-id="DSW006C27" data-open="DSW006C27" class="link link-ref link-reveal xref-bibr">27</a></sup> with an E-value cut-off of 1E−4 considering the length (95 bp) of the marker sequences.</p>                            <h3 scrollto-destination=87200860 id="87200860" class="section-title" data-legacy-id=s2f>2.6. Detection of repetitive sequences</h3>
<p>We detected known and novel repetitive sequences including transposable elements using the following two steps: First, the repetitive sequences were detected by RepeatScout<sup><a reveal-id="DSW006C28" data-open="DSW006C28" class="link link-ref link-reveal xref-bibr">28</a></sup> with default parameters. Then, repetitive sequences were detected and masked by RepeatMasker (<a class="link link-uri" href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>) against known repetitive sequences in Repbase (<a class="link link-uri" href="http://www.girinst.org/repbase/">http://www.girinst.org/repbase/</a>) and novel repetitive sequences found in the first step. The repetitive sequences were classified according to the results of RepeatMasker. SSR motifs were extracted from the ‘Nagirizaki’, ‘Wakaba’, and ‘Zanpa’ genome sequences and from those of <em>O. sativa</em> (IRGSP-1.0), <em>S. bicolor</em> (Sbicolor_255_v2.0), and <em>B. distachyon</em> (Bdistachyon_283_v2.0) using SciRoKo<sup><a reveal-id="DSW006C29" data-open="DSW006C29" class="link link-ref link-reveal xref-bibr">29</a></sup> with the MISA mode. The minimum numbers of repeats for SSR motifs were 7, 5, 4, 4 and 4 for di-, tri-, tetra-, penta- and hexa-nucleotides, respectively.</p>                            <h3 scrollto-destination=87200862 id="87200862" class="section-title" data-legacy-id=s2g>2.7. Gene assignment and annotation</h3>
<p>Genes for tRNAs were predicted using tRNAscan-SE, v1.23,<sup><a reveal-id="DSW006C30" data-open="DSW006C30" class="link link-ref link-reveal xref-bibr">30</a></sup> and rRNA genes were predicted using hmmscan in HMMER, ver. 3.0<sup><a reveal-id="DSW006C31" data-open="DSW006C31" class="link link-ref link-reveal xref-bibr">31</a></sup> with a Rfam 11.0 (<a class="link link-uri" href="http://rfam.xfam.org">http://rfam.xfam.org</a>) database. Genomic protein-coding genes of ‘Nagirizaki’, ‘Wakaba’, and ‘Zanpa’ were predicted using a MAKER pipeline<sup><a reveal-id="DSW006C32" data-open="DSW006C32" class="link link-ref link-reveal xref-bibr">32</a></sup> as follows: Repetitive sequences were detected and masked using a combination of RepeatModeller (<a class="link link-uri" href="http://www.repeatmasker.org/RepeatModeler.html">http://www.repeatmasker.org/RepeatModeler.html</a>), RECON,<sup><a reveal-id="DSW006C33" data-open="DSW006C33" class="link link-ref link-reveal xref-bibr">33</a></sup> RepeatMasker (<a class="link link-uri" href="http://www.repeatmasker.org">http://www.repeatmasker.org</a>) and RepeatRunner (<a class="link link-uri" href="https://github.com/Yandell-Lab/RepeatRunner">https://github.com/Yandell-Lab/RepeatRunner</a>). The training set for ‘Nagirizaki’ was built using BRAKER1 (<a class="link link-uri" href="http://exon.gatech.edu/genemark/braker1.html">http://exon.gatech.edu/genemark/braker1.html</a>) with the RNA-Seq data. Prediction of the protein-coding genes was performed using Augustus 3.0.3<sup><a reveal-id="DSW006C34" data-open="DSW006C34" class="link link-ref link-reveal xref-bibr">34</a></sup> with the training set for ‘Nagirizaki’, and snap<sup><a reveal-id="DSW006C35" data-open="DSW006C35" class="link link-ref link-reveal xref-bibr">35</a></sup> using the hidden Markov model of <em>O. sativa</em>. In addition, the RNA-Seq reads for ‘Nagirizaki’ were mapped onto the masked genome sequences by TopHat,<sup><a reveal-id="DSW006C36" data-open="DSW006C36" class="link link-ref link-reveal xref-bibr">36</a></sup> and coding regions were detected by Cufflinks<sup><a reveal-id="DSW006C37" data-open="DSW006C37" class="link link-ref link-reveal xref-bibr">37</a></sup> to generate a gff file indicating transcript units. Also, consensus transcript sequences were constructed by assembly of the RNA-Seq reads with Trinity, and were mapped onto the genome sequences by a BLASTN search and exonerate.<sup><a reveal-id="DSW006C38" data-open="DSW006C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Finally, the gene models supported by the above lines of evidence were selected using the MAKER pipeline. The predicted genes of ‘Nagirizaki’, ‘Wakaba’, and ‘Zanpa’ were prefixed with ‘Zjn’, ‘Zmw’, and ‘Zpz’, respectively, followed by a scaffold number, sequence version of each scaffold, gene number, and sequence version of each gene, which were connected to each other by dot characters. In addition, information about the gene prediction method was attached with the following signatures: am, assigned by Augustus; sm, assigned by snap; cf, assigned by Cufflinks; es, assigned by exonerate; and cf-es, assigned by Cufflinks and exonerate. The prediction accuracy was attached using the following signatures: mkhc, assigned with high confidence by MAKER (supported by multiple prediction methods); mk, assigned as non-overlapping genes by MAKER; and br, assigned by BRAKER1.</p><p>For assumption of the functions of the predicted protein-coding genes, database searches were carried out against the NCBI's non-redundant (nr) protein database (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov">http://www.ncbi.nlm.nih.gov</a>) and the protein sequences of <em>Arabidopsis thaliana</em> (TAIR10) (<a class="link link-uri" href="https://www.arabidopsis.org">https://www.arabidopsis.org</a>), <em>O. sativa</em> (RAP-DB) (<a class="link link-uri" href="http://rapdb.dna.affrc.go.jp">http://rapdb.dna.affrc.go.jp</a>), <em>S. bicolor</em>, <em>B. distachyon</em>, <em>Setaria italica</em>, <em>Triticum aestivum, Zea mays</em> (Phytozome, v10) (<a class="link link-uri" href="http://phytozome.jgi.doe.gov">http://phytozome.jgi.doe.gov</a>), and <em>Hordeum vulgare</em> (Ensembl Plants) (<a class="link link-uri" href="http://plants.ensembl.org">http://plants.ensembl.org</a>). The searches were performed using the BLAST program with an E-value cut-off of 1E−10. The translated protein sequences were classified into NCBI's euKaryotic clusters of Orthologous Groups (KOG)<sup><a reveal-id="DSW006C39" data-open="DSW006C39" class="link link-ref link-reveal xref-bibr">39</a></sup> using BLASTP searches against the KOG database with an E-value cut-off of 1E−10. A KOG id was assigned to each gene based on the top hit of the BLASTP search. Domain searches were performed using InterProScan<sup><a reveal-id="DSW006C40" data-open="DSW006C40" class="link link-ref link-reveal xref-bibr">40</a></sup> against the InterPro (<a class="link link-uri" href="http://www.ebi.ac.uk/interpro/">http://www.ebi.ac.uk/interpro/</a>) database with an E-value cut-off of 1.0. The genes were classified into GO slim categories using the map2slim program<sup><a reveal-id="DSW006C41" data-open="DSW006C41" class="link link-ref link-reveal xref-bibr">41</a></sup> based on the domain search results. The genes of ‘Nagirizaki’, <em>S. bicolor</em> (Sbicolor_255_v2.0; Phytozome v10), <em>O. sativa</em> (IRGSP-1.0) and <em>B. distachyon</em> (Bdistachyon_283_v2.0; Phytozome v10) were then mapped onto the KEGG (Kyoto Encyclopedia of Genes and Genomes) reference metabolic pathways<sup><a reveal-id="DSW006C42" data-open="DSW006C42" class="link link-ref link-reveal xref-bibr">42</a></sup> using BLASTP searches against KEGG's KO database with an E-value cut-off of 1E-10, identity ≥25% and length coverage ≥50%. The genes were classified using CD-hit<sup><a reveal-id="DSW006C43" data-open="DSW006C43" class="link link-ref link-reveal xref-bibr">43</a></sup> with parameters –c = 0.4, -G = 0 and –aS = 0.5.</p>                            <h3 scrollto-destination=87200866 id="87200866" class="section-title" data-legacy-id=s2h>2.8. Comparison of genes among <em>Z. japonica</em> and three other Poaceae species</h3>
<p>To identify common genes among <em>Z. japonica</em> ‘Nagirizaki’ and the three Poaceae species, <em>O. sativa</em>, <em>S. bicolor</em> and <em>B. distachyon</em>, the translated protein sequences were compared by a clustering approach using CD-hit. To compare the gene order, the translated protein sequences on the pseudomolecules of <em>Z. japonica</em> ‘Nagirizaki’ were searched against <em>O. sativa</em>, <em>S. bicolor</em> and <em>B. distachyon</em> using BLAST searches with an E-value cut-off of 1E-100. Synteny plots were made using a perl script and the gnuplot program (<a class="link link-uri" href="http://www.gnuplot.info">http://www.gnuplot.info</a>).</p>                            <h3 scrollto-destination=87200869 id="87200869" class="section-title" data-legacy-id=s2i>2.9. Polymorphism analysis</h3>
<p>The PE reads of each <em>Zoysia</em> accession were aligned onto the genome sequence of ‘Nagirizaki’ using Bowtie 2, ver. 2.2.5<sup><a reveal-id="DSW006C44" data-open="DSW006C44" class="link link-ref link-reveal xref-bibr">44</a></sup> with default parameters, except for the use of an end-to-end mode. SNPs and indel candidates were detected using bam files generated using SAMtools, ver. 0.1.19, and BCFtools, ver. 0.1.19.<sup><a reveal-id="DSW006C45" data-open="DSW006C45" class="link link-ref link-reveal xref-bibr">45</a></sup> After mapping, the SNP candidates on <em>N</em> or <em>n</em> nucleotides in the reference sequence were excluded. The false-positive SNPs were excluded according to the DP4 field in the VCF format.<sup><a reveal-id="DSW006C45" data-open="DSW006C45" class="link link-ref link-reveal xref-bibr">45</a></sup></p>                            <h2 scrollto-destination=87200871 id="87200871" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=87200872 id="87200872" class="section-title" data-legacy-id=s3a>3.1 Sequencing of the <em>Zoysia</em> genomes</h3>
                            <h4 scrollto-destination=87200873 id="87200873" class="section-title" data-legacy-id=s3a1>3.1.1 Shotgun sequencing and assembly of the <em>Zoysia</em> genomes</h4>
<p>A high-quality draft genome sequence of <em>Z. japonica ‘</em>Nagirizaki’ was generated for use, as a reference sequence. PE read data (49.3 Gb) were accumulated using the MiSeq platform. In addition, mate-pair (MP) reads were collected from three MP libraries with insert sizes of 5, 10 and 15 kb using the HiSeq 2000 platform. For <em>Z. matrella</em> ‘Wakaba’ and <em>Z. pacifica</em> ‘Zanpa’, ∼17 Gb of data from PE reads and a single type of MP library (with 3 kb inserts) were accumulated for comparative analyses. The sequence data for each <em>Zoysia</em> accession are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>.</p><p>The kmer analysis for sequenced reads suggested that all the sequenced genomes had a high proportion of heterozygous alleles (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S1</a></span>). We used two computer programs for the primary assemblies: Platanus, which is suitable for assembly of highly heterozygous genomes, and the DISCOVAR <em>de novo</em> assembler, which is capable of generating contigs with high efficiency using PE reads with 250–300 bases of read length. The scaffolds with the highest N50 values were obtained in Platanus for <em>Z. japonica</em> ‘Nagirizaki’ (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). After gap-closing by GMcloser, 334 Mb of scaffolds with an N50 length of 2.4 Mb were obtained (Table <a reveal-id="DSW006TB1" data-open="DSW006TB1" class="link link-reveal link-table xref-fig">1</a>). Ninety percent of the scaffolds (301 Mb) were included in the 167 longest scaffolds. According to the flow cytometry analysis, the genome size of <em>Z. japonica</em> ‘Nagirizaki’ is ∼390 Mb (0.80 pg/2C). This is 15% larger than the estimation by k-mer analysis (340 Mb) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>), presumably due to the effect of conserved regions on homeologous chromosome pairs in the tetraploid genome. Thus the genome assembly of <em>Z. japonica</em> ‘Nagirizaki’, designated as ‘Zjn_r1.1’, covered 86% of the whole genome as estimated by flow cytometry. </p>                        <a id="87200876" scrollto-destination="87200876"></a>
<div content-id="DSW006TB1" class="table-modal"><div class="table-wrap"><div id="DSW006TB1" class="table-wrap-title" data-id="DSW006TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Assembly statistics of draft genome sequences of the <em>Zoysia</em> species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th><em>Z. japonica</em><br />‘Nagirizaki’ (ZJN_r1.1) </th><th><em>Z. matrella</em><br /><em>‘</em>Wakaba’ (ZMW_r1.0) </th><th><em>Z. pacifica</em><br /><em>‘</em>Zanpa’ (ZPZ_r1.0) </th><th><em>Z. japonica</em><br /><em>‘</em>Nagirizaki’ (ZJN_r1.1_pseudomol) </th></tr></thead><tbody><tr><td colspan="5">Total </td></tr><tr><td> Number of sequences </td><td>11,786 </td><td>13,609 </td><td>11,428 </td><td>20 </td></tr><tr><td> Total length (b) </td><td>334,384,427 </td><td>563,438,595 </td><td>397,009,957 </td><td>273,735,632 </td></tr><tr><td> Ave. length (b) </td><td>28,371 </td><td>41,402 </td><td>34,740 </td><td>13,686,782 </td></tr><tr><td> Max. length (b) </td><td>8,501,895 </td><td>1,041,506 </td><td>1,506,652 </td><td>21,132,631 </td></tr><tr><td> Min. length (b) </td><td>500 </td><td>500 </td><td>500 </td><td>7,690,864 </td></tr><tr><td> N50 length (b) </td><td>2,370,062 </td><td>108,897 </td><td>111,449 </td><td>17,914,192 </td></tr><tr><td> A </td><td>90,476,466 </td><td>156,060,073 </td><td>110,343,373 </td><td>73,600,141 </td></tr><tr><td> T </td><td>90,391,770 </td><td>155,985,841 </td><td>110,358,466 </td><td>73,593,144 </td></tr><tr><td> G </td><td>71,275,668 </td><td>121,996,426 </td><td>85,817,354 </td><td>58,170,503 </td></tr><tr><td> C </td><td>71,324,485 </td><td>122,016,838 </td><td>85,854,342 </td><td>58,222,241 </td></tr><tr><td> N </td><td>10,916,038 </td><td>7,379,417 </td><td>4,636,422 </td><td>10,149,603 </td></tr><tr><td> Total (ATGC) </td><td>323,468,389 </td><td>556,059,178 </td><td>392,373,535 </td><td>263,586,029 </td></tr><tr><td> G+C% (GC/ATGC) </td><td>44.1 </td><td>43.9 </td><td>43.8 </td><td>44.2 </td></tr><tr><td colspan="5">≥1 kb </td></tr><tr><td> Number of sequences </td><td>6,554 </td><td>11,227 </td><td>8,730 </td><td>– </td></tr><tr><td> Total length (b) </td><td>330,746,168 </td><td>561,855,676 </td><td>395,175,704 </td><td>– </td></tr><tr><td> Ave. length (b) </td><td>50,465 </td><td>50,045 </td><td>45,266 </td><td>– </td></tr><tr><td colspan="5">≥2 kb </td></tr><tr><td> Number of sequences </td><td>2,069 </td><td>10,304 </td><td>7,340 </td><td>– </td></tr><tr><td> Total length (b) </td><td>324,295,136 </td><td>560,567,885 </td><td>393,244,071 </td><td>– </td></tr><tr><td> Ave. length (b) </td><td>156,740 </td><td>54,403 </td><td>53,575 </td><td>– </td></tr><tr><td colspan="5">≥3 kb </td></tr><tr><td> Number of sequences </td><td>1,149 </td><td>9,902 </td><td>6,930 </td><td>– </td></tr><tr><td> Total length (b) </td><td>322,069,938 </td><td>559,570,268 </td><td>392,259,279 </td><td>– </td></tr><tr><td> Ave. length (b) </td><td>280,305 </td><td>56,511 </td><td>56,603 </td><td>– </td></tr><tr><td colspan="5">≥4 kb </td></tr><tr><td> Number of sequences </td><td>777 </td><td>9,524 </td><td>6,552 </td><td>– </td></tr><tr><td> Total length (b) </td><td>320,784,379 </td><td>558,247,617 </td><td>390,938,668 </td><td>– </td></tr><tr><td> Ave. length (b) </td><td>412,850 </td><td>58,615 </td><td>59,667 </td><td>– </td></tr><tr><td colspan="5">≥5 kb </td></tr><tr><td> Number of sequences </td><td>684 </td><td>9,188 </td><td>6,242 </td><td>20 </td></tr><tr><td> Total length (b) </td><td>320,368,645 </td><td>556,750,068 </td><td>389,557,068 </td><td>273,735,632 </td></tr><tr><td> Ave. length (b) </td><td>468,375 </td><td>60,595 </td><td>62,409 </td><td>13,686,782 </td></tr></tbody></table></div><div content-id="DSW006TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th><em>Z. japonica</em><br />‘Nagirizaki’ (ZJN_r1.1) </th><th><em>Z. matrella</em><br /><em>‘</em>Wakaba’ (ZMW_r1.0) </th><th><em>Z. pacifica</em><br /><em>‘</em>Zanpa’ (ZPZ_r1.0) </th><th><em>Z. japonica</em><br /><em>‘</em>Nagirizaki’ (ZJN_r1.1_pseudomol) </th></tr></thead><tbody><tr><td colspan="5">Total </td></tr><tr><td> Number of sequences </td><td>11,786 </td><td>13,609 </td><td>11,428 </td><td>20 </td></tr><tr><td> Total length (b) </td><td>334,384,427 </td><td>563,438,595 </td><td>397,009,957 </td><td>273,735,632 </td></tr><tr><td> Ave. length (b) </td><td>28,371 </td><td>41,402 </td><td>34,740 </td><td>13,686,782 </td></tr><tr><td> Max. length (b) </td><td>8,501,895 </td><td>1,041,506 </td><td>1,506,652 </td><td>21,132,631 </td></tr><tr><td> Min. length (b) </td><td>500 </td><td>500 </td><td>500 </td><td>7,690,864 </td></tr><tr><td> N50 length (b) </td><td>2,370,062 </td><td>108,897 </td><td>111,449 </td><td>17,914,192 </td></tr><tr><td> A </td><td>90,476,466 </td><td>156,060,073 </td><td>110,343,373 </td><td>73,600,141 </td></tr><tr><td> T </td><td>90,391,770 </td><td>155,985,841 </td><td>110,358,466 </td><td>73,593,144 </td></tr><tr><td> G </td><td>71,275,668 </td><td>121,996,426 </td><td>85,817,354 </td><td>58,170,503 </td></tr><tr><td> C </td><td>71,324,485 </td><td>122,016,838 </td><td>85,854,342 </td><td>58,222,241 </td></tr><tr><td> N </td><td>10,916,038 </td><td>7,379,417 </td><td>4,636,422 </td><td>10,149,603 </td></tr><tr><td> Total (ATGC) </td><td>323,468,389 </td><td>556,059,178 </td><td>392,373,535 </td><td>263,586,029 </td></tr><tr><td> G+C% (GC/ATGC) </td><td>44.1 </td><td>43.9 </td><td>43.8 </td><td>44.2 </td></tr><tr><td colspan="5">≥1 kb </td></tr><tr><td> Number of sequences </td><td>6,554 </td><td>11,227 </td><td>8,730 </td><td>– </td></tr><tr><td> Total length (b) </td><td>330,746,168 </td><td>561,855,676 </td><td>395,175,704 </td><td>– </td></tr><tr><td> Ave. length (b) </td><td>50,465 </td><td>50,045 </td><td>45,266 </td><td>– </td></tr><tr><td colspan="5">≥2 kb </td></tr><tr><td> Number of sequences </td><td>2,069 </td><td>10,304 </td><td>7,340 </td><td>– </td></tr><tr><td> Total length (b) </td><td>324,295,136 </td><td>560,567,885 </td><td>393,244,071 </td><td>– </td></tr><tr><td> Ave. length (b) </td><td>156,740 </td><td>54,403 </td><td>53,575 </td><td>– </td></tr><tr><td colspan="5">≥3 kb </td></tr><tr><td> Number of sequences </td><td>1,149 </td><td>9,902 </td><td>6,930 </td><td>– </td></tr><tr><td> Total length (b) </td><td>322,069,938 </td><td>559,570,268 </td><td>392,259,279 </td><td>– </td></tr><tr><td> Ave. length (b) </td><td>280,305 </td><td>56,511 </td><td>56,603 </td><td>– </td></tr><tr><td colspan="5">≥4 kb </td></tr><tr><td> Number of sequences </td><td>777 </td><td>9,524 </td><td>6,552 </td><td>– </td></tr><tr><td> Total length (b) </td><td>320,784,379 </td><td>558,247,617 </td><td>390,938,668 </td><td>– </td></tr><tr><td> Ave. length (b) </td><td>412,850 </td><td>58,615 </td><td>59,667 </td><td>– </td></tr><tr><td colspan="5">≥5 kb </td></tr><tr><td> Number of sequences </td><td>684 </td><td>9,188 </td><td>6,242 </td><td>20 </td></tr><tr><td> Total length (b) </td><td>320,368,645 </td><td>556,750,068 </td><td>389,557,068 </td><td>273,735,632 </td></tr><tr><td> Ave. length (b) </td><td>468,375 </td><td>60,595 </td><td>62,409 </td><td>13,686,782 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Contigs or scaffolds shorter than 499 bp were excluded.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200876" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSW006TB1" class="table-wrap-title" data-id="DSW006TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Assembly statistics of draft genome sequences of the <em>Zoysia</em> species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th><em>Z. japonica</em><br />‘Nagirizaki’ (ZJN_r1.1) </th><th><em>Z. matrella</em><br /><em>‘</em>Wakaba’ (ZMW_r1.0) </th><th><em>Z. pacifica</em><br /><em>‘</em>Zanpa’ (ZPZ_r1.0) </th><th><em>Z. japonica</em><br /><em>‘</em>Nagirizaki’ (ZJN_r1.1_pseudomol) </th></tr></thead><tbody><tr><td colspan="5">Total </td></tr><tr><td> Number of sequences </td><td>11,786 </td><td>13,609 </td><td>11,428 </td><td>20 </td></tr><tr><td> Total length (b) </td><td>334,384,427 </td><td>563,438,595 </td><td>397,009,957 </td><td>273,735,632 </td></tr><tr><td> Ave. length (b) </td><td>28,371 </td><td>41,402 </td><td>34,740 </td><td>13,686,782 </td></tr><tr><td> Max. length (b) </td><td>8,501,895 </td><td>1,041,506 </td><td>1,506,652 </td><td>21,132,631 </td></tr><tr><td> Min. length (b) </td><td>500 </td><td>500 </td><td>500 </td><td>7,690,864 </td></tr><tr><td> N50 length (b) </td><td>2,370,062 </td><td>108,897 </td><td>111,449 </td><td>17,914,192 </td></tr><tr><td> A </td><td>90,476,466 </td><td>156,060,073 </td><td>110,343,373 </td><td>73,600,141 </td></tr><tr><td> T </td><td>90,391,770 </td><td>155,985,841 </td><td>110,358,466 </td><td>73,593,144 </td></tr><tr><td> G </td><td>71,275,668 </td><td>121,996,426 </td><td>85,817,354 </td><td>58,170,503 </td></tr><tr><td> C </td><td>71,324,485 </td><td>122,016,838 </td><td>85,854,342 </td><td>58,222,241 </td></tr><tr><td> N </td><td>10,916,038 </td><td>7,379,417 </td><td>4,636,422 </td><td>10,149,603 </td></tr><tr><td> Total (ATGC) </td><td>323,468,389 </td><td>556,059,178 </td><td>392,373,535 </td><td>263,586,029 </td></tr><tr><td> G+C% (GC/ATGC) </td><td>44.1 </td><td>43.9 </td><td>43.8 </td><td>44.2 </td></tr><tr><td colspan="5">≥1 kb </td></tr><tr><td> Number of sequences </td><td>6,554 </td><td>11,227 </td><td>8,730 </td><td>– </td></tr><tr><td> Total length (b) </td><td>330,746,168 </td><td>561,855,676 </td><td>395,175,704 </td><td>– </td></tr><tr><td> Ave. length (b) </td><td>50,465 </td><td>50,045 </td><td>45,266 </td><td>– </td></tr><tr><td colspan="5">≥2 kb </td></tr><tr><td> Number of sequences </td><td>2,069 </td><td>10,304 </td><td>7,340 </td><td>– </td></tr><tr><td> Total length (b) </td><td>324,295,136 </td><td>560,567,885 </td><td>393,244,071 </td><td>– </td></tr><tr><td> Ave. length (b) </td><td>156,740 </td><td>54,403 </td><td>53,575 </td><td>– </td></tr><tr><td colspan="5">≥3 kb </td></tr><tr><td> Number of sequences </td><td>1,149 </td><td>9,902 </td><td>6,930 </td><td>– </td></tr><tr><td> Total length (b) </td><td>322,069,938 </td><td>559,570,268 </td><td>392,259,279 </td><td>– </td></tr><tr><td> Ave. length (b) </td><td>280,305 </td><td>56,511 </td><td>56,603 </td><td>– </td></tr><tr><td colspan="5">≥4 kb </td></tr><tr><td> Number of sequences </td><td>777 </td><td>9,524 </td><td>6,552 </td><td>– </td></tr><tr><td> Total length (b) </td><td>320,784,379 </td><td>558,247,617 </td><td>390,938,668 </td><td>– </td></tr><tr><td> Ave. length (b) </td><td>412,850 </td><td>58,615 </td><td>59,667 </td><td>– </td></tr><tr><td colspan="5">≥5 kb </td></tr><tr><td> Number of sequences </td><td>684 </td><td>9,188 </td><td>6,242 </td><td>20 </td></tr><tr><td> Total length (b) </td><td>320,368,645 </td><td>556,750,068 </td><td>389,557,068 </td><td>273,735,632 </td></tr><tr><td> Ave. length (b) </td><td>468,375 </td><td>60,595 </td><td>62,409 </td><td>13,686,782 </td></tr></tbody></table></div><div content-id="DSW006TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th><em>Z. japonica</em><br />‘Nagirizaki’ (ZJN_r1.1) </th><th><em>Z. matrella</em><br /><em>‘</em>Wakaba’ (ZMW_r1.0) </th><th><em>Z. pacifica</em><br /><em>‘</em>Zanpa’ (ZPZ_r1.0) </th><th><em>Z. japonica</em><br /><em>‘</em>Nagirizaki’ (ZJN_r1.1_pseudomol) </th></tr></thead><tbody><tr><td colspan="5">Total </td></tr><tr><td> Number of sequences </td><td>11,786 </td><td>13,609 </td><td>11,428 </td><td>20 </td></tr><tr><td> Total length (b) </td><td>334,384,427 </td><td>563,438,595 </td><td>397,009,957 </td><td>273,735,632 </td></tr><tr><td> Ave. length (b) </td><td>28,371 </td><td>41,402 </td><td>34,740 </td><td>13,686,782 </td></tr><tr><td> Max. length (b) </td><td>8,501,895 </td><td>1,041,506 </td><td>1,506,652 </td><td>21,132,631 </td></tr><tr><td> Min. length (b) </td><td>500 </td><td>500 </td><td>500 </td><td>7,690,864 </td></tr><tr><td> N50 length (b) </td><td>2,370,062 </td><td>108,897 </td><td>111,449 </td><td>17,914,192 </td></tr><tr><td> A </td><td>90,476,466 </td><td>156,060,073 </td><td>110,343,373 </td><td>73,600,141 </td></tr><tr><td> T </td><td>90,391,770 </td><td>155,985,841 </td><td>110,358,466 </td><td>73,593,144 </td></tr><tr><td> G </td><td>71,275,668 </td><td>121,996,426 </td><td>85,817,354 </td><td>58,170,503 </td></tr><tr><td> C </td><td>71,324,485 </td><td>122,016,838 </td><td>85,854,342 </td><td>58,222,241 </td></tr><tr><td> N </td><td>10,916,038 </td><td>7,379,417 </td><td>4,636,422 </td><td>10,149,603 </td></tr><tr><td> Total (ATGC) </td><td>323,468,389 </td><td>556,059,178 </td><td>392,373,535 </td><td>263,586,029 </td></tr><tr><td> G+C% (GC/ATGC) </td><td>44.1 </td><td>43.9 </td><td>43.8 </td><td>44.2 </td></tr><tr><td colspan="5">≥1 kb </td></tr><tr><td> Number of sequences </td><td>6,554 </td><td>11,227 </td><td>8,730 </td><td>– </td></tr><tr><td> Total length (b) </td><td>330,746,168 </td><td>561,855,676 </td><td>395,175,704 </td><td>– </td></tr><tr><td> Ave. length (b) </td><td>50,465 </td><td>50,045 </td><td>45,266 </td><td>– </td></tr><tr><td colspan="5">≥2 kb </td></tr><tr><td> Number of sequences </td><td>2,069 </td><td>10,304 </td><td>7,340 </td><td>– </td></tr><tr><td> Total length (b) </td><td>324,295,136 </td><td>560,567,885 </td><td>393,244,071 </td><td>– </td></tr><tr><td> Ave. length (b) </td><td>156,740 </td><td>54,403 </td><td>53,575 </td><td>– </td></tr><tr><td colspan="5">≥3 kb </td></tr><tr><td> Number of sequences </td><td>1,149 </td><td>9,902 </td><td>6,930 </td><td>– </td></tr><tr><td> Total length (b) </td><td>322,069,938 </td><td>559,570,268 </td><td>392,259,279 </td><td>– </td></tr><tr><td> Ave. length (b) </td><td>280,305 </td><td>56,511 </td><td>56,603 </td><td>– </td></tr><tr><td colspan="5">≥4 kb </td></tr><tr><td> Number of sequences </td><td>777 </td><td>9,524 </td><td>6,552 </td><td>– </td></tr><tr><td> Total length (b) </td><td>320,784,379 </td><td>558,247,617 </td><td>390,938,668 </td><td>– </td></tr><tr><td> Ave. length (b) </td><td>412,850 </td><td>58,615 </td><td>59,667 </td><td>– </td></tr><tr><td colspan="5">≥5 kb </td></tr><tr><td> Number of sequences </td><td>684 </td><td>9,188 </td><td>6,242 </td><td>20 </td></tr><tr><td> Total length (b) </td><td>320,368,645 </td><td>556,750,068 </td><td>389,557,068 </td><td>273,735,632 </td></tr><tr><td> Ave. length (b) </td><td>468,375 </td><td>60,595 </td><td>62,409 </td><td>13,686,782 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Contigs or scaffolds shorter than 499 bp were excluded.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200876" target="_blank">View Large</a></div></div><p>For <em>Z. matrella</em> ‘Wakaba’ and <em>Z. pacifica</em> ‘Zanpa’, the scaffolds were built by Opera using contigs generated by DISCOVAR <em>de novo</em> because DISCOVAR <em>de novo</em> constructed contigs with higher N50 values than the other assemblers. This may be because only a single MP library was generated for these species. The final assemblies for ‘Wakaba’ and ‘Zanpa’ were generated by gap-closing with GMcloser and resulted in 563 Mb (N50 length, 109 kb) and 397 Mb (N50 length, 111 kb), respectively (Table <a reveal-id="DSW006TB1" data-open="DSW006TB1" class="link link-reveal link-table xref-fig">1</a>). As with ‘Nagirizaki’, the genome size of ‘Zanpa’ estimated by flow cytometry (370 Mb; 0.76 pg/2C) was larger (by 23%) than the size estimated by k-mer analysis (302 Mb) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). On the other hand, the flow cytometry estimate for ‘Wakaba’ was ∼380 Mb (0.79 pg/2C), which is 11% smaller than that estimated by k-mer analysis (423 Mb) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). This difference was presumably due to the effect of heterozygosity exceeding the effect of tetraploidy. The genome assemblies of ‘Wakaba’ and ‘Zanpa’ were longer than those estimated by flow cytometry, suggesting that they contained redundant scaffolds corresponding to heterozygous alleles. This is likely because the assemblies of these two accessions were obtained with the DISCOVAR <em>de novo</em> and Opera assemblers, which are less able than Platanus to merge homologous sequences corresponding to the same allelic regions. Indeed, the total sizes of the scaffold assemblies of ‘Wakaba’ and ‘Zanpa’ generated by Platanus were 30 and 22% smaller, respectively, than those obtained by DISCOVAR <em>de novo</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Considering the purpose of sequencing for comparative genome analyses, we finally adopted the DISCOVAR <em>de novo</em> assemblies, which presumably contained redundant scaffolds corresponding to heterozygous alleles, for these two accessions. The <em>Z. matrella</em> ‘Wakaba’ and <em>Z. pacifica</em> ‘Zanpa’ genomes were designated as ‘Zmw_r1.0’ and ‘Zpz_r1.0’, respectively.</p>                            <h4 scrollto-destination=87200878 id="87200878" class="section-title" data-legacy-id=s3a2>3.1.2 Authenticity of the assembled sequences</h4>
<p>The CEGMA analyses of the assembled genome sequences showed that they completely cover 91.5–94.8% (the highest is for <em>Z. japonica</em> ‘Nagirizaki’), and partially cover over 97%, of the 248 conserved core eukaryotic genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). These high coverage data strongly indicate that the constructed genome sequences of the three <em>Zoysia</em> species were of high quality and sufficient for subsequent accurate gene modelling and annotation.</p><p>The authenticity of the assembled genome sequences was further evaluated by comparing the genome sequence of ‘Nagirizaki’ and the published <em>Z. japonica</em> linkage map of RAD markers.<sup><a reveal-id="DSW006C8" data-open="DSW006C8" class="link link-ref link-reveal xref-bibr">8</a></sup> We compared 1,230 marker-associated sequences on the <em>Z. japonica ‘</em>El Toro’ linkage map developed by Wang et al.<sup><a reveal-id="DSW006C8" data-open="DSW006C8" class="link link-ref link-reveal xref-bibr">8</a></sup> with the scaffold sequences of Zjn_r1.1 and found that 1,224 (99.5%) of the sequences were successfully located on Zjn_r1.1. The 135 scaffolds in Zjn_r1.1 were each anchored by three or more markers in the ‘El Toro’ linkage groups and the order of most of the markers was in good agreement with their physical positions on the scaffolds. This result indicates a high degree of authenticity of the Zjn_r1.1 genome assembly.</p><p>While comparing the ‘Nagirizaki’ scaffold sequences with the linkage map, we found that 15 scaffolds were anchored to two or three different locations on the linkage map, probably due to mis-scaffolding caused by the presence of repetitive sequences. We separated these scaffolds at the repetitive sequences at the boundaries of the anchored regions, and the latter halves of these scaffolds were designated using numbers starting with ‘2’ or ‘3’ followed by the original four-digit scaffold number (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). For example, the scaffold Zjn_sc00037 was anchored by RAD markers at three different locations on the linkage map, and thus it was separated into three scaffolds: Zjn_sc00037, Zjn_sc20037, and Zjn_sc30037. After these revisions, we constructed pseudomolecules of <em>Z. japonica</em> ‘Nagirizaki’ that we designated as Zjn_chr01 – Zjn_chr20 based on the ‘El Toro’ linkage group numbers. These pseudomolecules covered 82% of the Zjn_r1.1 sequence. The pseudomolecules of homeologous chromosome pairs had similar lengths, ranging from the shortest pair: Zjn_chr15 (7.7 Mb) and Zjn_chr16 (8.1 Mb) to the longest pair: Zjn_chr07 (20.3 Mb) and Zjn_chr08 (21.1 Mb). These results confirmed the even distribution of the RAD markers and indicated that the scaffolds in Zjn_r1.1 are sufficient to cover most of the <em>Z. japonica</em> genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). Therefore, the pseudomolecules of ‘Nagirizaki’ were subjected to subsequent analyses as the reference sequence of <em>Z. japonica</em>, together with the draft genome sequences.</p>                            <h3 scrollto-destination=87200885 id="87200885" class="section-title" data-legacy-id=s3b>3.2 Characteristic features of the <em>Zoysia</em> genome</h3>
                            <h4 scrollto-destination=87200889 id="87200889" class="section-title" data-legacy-id=s3b1>3.2.1 Repetitive sequences</h4>
<p>Known and novel repetitive sequences, including transposable elements, were identified using RepeatMasker and RepeatScout. A classification of the repetitive elements and their content in each genome is summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>. The genome sequences of ‘Nagirizaki’ (Zjn_r1.1), ‘Wakaba’ (Zmw_r1.0), and ‘Zanpa’ (Zpz_r1.0) had total repeat contents of 40.9, 45.9 and 44.7%, respectively. The total repeat content in the pseudomolecules of the <em>Z. japonica</em> ‘Nagirizaki’ genome (35.6%) was lower than those of the other draft sequences because the unanchored scaffolds, which were likely to contain more repetitive sequences, were not included in the pseudomolecules. Based on the methods for detection of repetitive sequences used in this study, the contents of repetitive sequences in the <em>Zoysia</em> species were similar to that of <em>B. distachyon</em> (37.7% of total genome size, 272 Mb in Bdistachyon_283_v2.0), and smaller than those of <em>O. sativa</em> (48.7% of 373 Mb in IRGSP-1.0) and <em>S. bicolor</em> (64.8% of 727 Mb in Sbicolor_255_v2.0).</p><p>The SSRs in the draft genome sequences were detected using SciRoKo software, and their contents in each draft genome are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>. The total numbers of the SSR motifs in the <em>Zoysia</em> species were 24,383 in <em>Z. japonica</em> ‘Nagirizaki’ (7.07 SSRs per 100 kb), 41,883 in <em>Z. matrella</em> ‘Wakaba’ (7.20 SSRs per 100 kb), and 29,271 in <em>Z. pacifica</em> ‘Zanpa’ (7.15 SSRs per 100 kb). These frequencies are similar to the frequency for <em>B. distachyon</em>: 7.80 SSRs per 100 kb (total number: 21,818). The frequencies of di-nucleotides in the three <em>Zoysia</em> species were nearly the same and were close to that of <em>S. bicolor</em>. On the other hand, the frequencies of tri-nucleotides in the three <em>Zoysia</em> species were fewer than those in <em>O. sativa</em>, <em>S. bicolor</em> and <em>B. distachyon</em>.</p>                            <h4 scrollto-destination=87200892 id="87200892" class="section-title" data-legacy-id=s3b2>3.2.2 RNA-coding genes</h4>
<p>Genes for tRNAs were predicted using tRNAscan-SE and rRNA genes were predicted using HMMER3 searches against the Rfam database. Totals of 514, 827 and 597 tRNA genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>), and 59, 47 and 52 rRNA genes were assigned to the draft genome sequences of ‘Nagirizaki’, ‘Wakaba’ and ‘Zanpa’, respectively. The numbers of predicted non-coding RNA genes were roughly proportional to the lengths of the assembled genome sequences of the three species. Although the genome size of <em>Z. japonica</em> ‘Nagirizaki’ (390 Mb) is similar to that of <em>O. sativa</em> (373 Mb), the number of predicted tRNA genes in ‘Nagirizaki’ (514) is ∼73% of that in <em>O. sativa</em> (705).</p>                            <h3 scrollto-destination=87200895 id="87200895" class="section-title" data-legacy-id=s3c>3.3 Characteristic features of the protein-coding genes</h3>
                            <h4 scrollto-destination=87200897 id="87200897" class="section-title" data-legacy-id=s3c1>3.3.1. Prediction of protein-coding genes and annotation</h4>
<p>The protein-coding genes were predicted for the three <em>Zoysia</em> species using multiple computer programs as described in the Materials and methods section. The number of coding sequences predicted in ‘Nagirizaki’ (Zjn_r1.1) was 59,271 with an N50 length of 1,119 bp (Table <a reveal-id="DSW006TB2" data-open="DSW006TB2" class="link link-reveal link-table xref-fig">2</a>). Of these, 49,103 sequences were assigned to the pseudomolecules of ‘Nagirizaki’. When the predicted genes on the homeologous chromosome pairs were compared in detail, more than half of the genes seemed to be lost from one of the pairs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>). This explained why the number of predicted genes in tetraploid <em>Z. japonica</em> is less than twice those in the diploid Poaceae species <em>O. sativa</em> (35,626 genes) and <em>S. bicolor</em> (33,032 genes). On the other hand, the total numbers of protein-coding genes predicted on the draft genome sequences of ‘Wakaba’ and ‘Zanpa’ were 95,079 and 65,252, with N50 lengths of 1,065 and 1,062 bp, respectively (Table <a reveal-id="DSW006TB2" data-open="DSW006TB2" class="link link-reveal link-table xref-fig">2</a>). The higher numbers of genes predicted for these two accessions presumably reflects the presence of redundant scaffolds corresponding to heterozygous alleles in Zmw_r1.0 and Zpz_r1.0. </p>                        <a id="87200900" scrollto-destination="87200900"></a>
<div content-id="DSW006TB2" class="table-modal"><div class="table-wrap"><div id="DSW006TB2" class="table-wrap-title" data-id="DSW006TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Statistics of genes predicted in the draft genome sequences of <em>Zoysia</em> species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="4"><em>Z. japonica</em> ‘Nagirizaki’<hr /> </th><th colspan="3"><em>Z. matrella</em> ‘Wakaba’<hr /> </th><th colspan="3"><em>Z. pacifica</em> ‘Zanpa’<hr /> </th></tr><tr><th>Total </th><th>mkhc </th><th>mk </th><th>br </th><th>Total </th><th>mkhc </th><th>mk </th><th>Total </th><th>mkhc </th><th>mk </th></tr></thead><tbody><tr><td colspan="11">Total </td></tr><tr><td> Number of sequences </td><td>59,271 </td><td>16,205 </td><td>37,605 </td><td>5,461 </td><td>95,079 </td><td>24,398 </td><td>70,681 </td><td>65,252 </td><td>16,455 </td><td>48,797 </td></tr><tr><td> Total length (b) </td><td>49,483,110 </td><td>16,398,294 </td><td>27,579,822 </td><td>5,504,994 </td><td>76,439,178 </td><td>23,712,921 </td><td>52,726,257 </td><td>52,368,507 </td><td>16,358,223 </td><td>36,010,284 </td></tr><tr><td> Ave. length (b) </td><td>835 </td><td>1,012 </td><td>733 </td><td>1,008 </td><td>804 </td><td>972 </td><td>746 </td><td>803 </td><td>994 </td><td>738 </td></tr><tr><td> Max. length (b) </td><td>11,838 </td><td>11,838 </td><td>6,666 </td><td>5,439 </td><td>13,647 </td><td>13,647 </td><td>6,240 </td><td>14,421 </td><td>14,421 </td><td>6,735 </td></tr><tr><td> Min. length (b) </td><td>108 </td><td>150 </td><td>150 </td><td>108 </td><td>150 </td><td>153 </td><td>150 </td><td>150 </td><td>150 </td><td>150 </td></tr><tr><td> N50 length (b) </td><td>1,119 </td><td>1,254 </td><td>987 </td><td>1,275 </td><td>1,065 </td><td>1,179 </td><td>999 </td><td>1,062 </td><td>1,206 </td><td>978 </td></tr><tr><td> G+C% </td><td>59.3 </td><td>53.2 </td><td>63.2 </td><td>58.3 </td><td>59.6 </td><td>52.6 </td><td>62.7 </td><td>59.6 </td><td>52.7 </td><td>62.8 </td></tr><tr><td colspan="11">≥100 b </td></tr><tr><td> Number of sequences </td><td>59,271 </td><td>16,205 </td><td>37,605 </td><td>5,461 </td><td>95,079 </td><td>24,398 </td><td>70,681 </td><td>65,252 </td><td>16,455 </td><td>48,797 </td></tr><tr><td> Total length (b) </td><td>49,483,110 </td><td>16,398,294 </td><td>27,579,822 </td><td>5,504,994 </td><td>76,439,178 </td><td>23,712,921 </td><td>52,726,257 </td><td>52,368,507 </td><td>16,358,223 </td><td>36,010,284 </td></tr><tr><td> Ave. length (b) </td><td>835 </td><td>1,012 </td><td>733 </td><td>1,008 </td><td>804 </td><td>972 </td><td>746 </td><td>803 </td><td>994 </td><td>738 </td></tr><tr><td colspan="11">≥500 b </td></tr><tr><td> Number of sequences </td><td>36,762 </td><td>12,446 </td><td>20,166 </td><td>4,150 </td><td>58,044 </td><td>18,851 </td><td>39,193 </td><td>39,673 </td><td>12,864 </td><td>26,809 </td></tr><tr><td> Total length (b) </td><td>41,915,830 </td><td>15,083,250 </td><td>21,825,114 </td><td>5,007,466 </td><td>63,962,067 </td><td>21,739,386 </td><td>42,222,681 </td><td>43,778,979 </td><td>15,084,588 </td><td>28,694,391 </td></tr><tr><td> Ave. length (b) </td><td>1,140 </td><td>1,212 </td><td>1,082 </td><td>1,207 </td><td>1,102 </td><td>1,153 </td><td>1,077 </td><td>1,103 </td><td>1,173 </td><td>1,070 </td></tr><tr><td colspan="11">≥1 kb </td></tr><tr><td> Number of sequences </td><td>17,091 </td><td>6,436 </td><td>8,605 </td><td>2,050 </td><td>25,866 </td><td>9,153 </td><td>16,713 </td><td>17,496 </td><td>6,342 </td><td>11,154 </td></tr><tr><td> Total length (b) </td><td>27,773,167 </td><td>10,647,366 </td><td>13,635,474 </td><td>3,490,327 </td><td>40,863,282 </td><td>14,576,319 </td><td>26,286,963 </td><td>27,907,539 </td><td>10,272,498 </td><td>17,635,041 </td></tr><tr><td> Ave. length (b) </td><td>1,625 </td><td>1,654 </td><td>1,585 </td><td>1,703 </td><td>1,580 </td><td>1,593 </td><td>1,573 </td><td>1,595 </td><td>1,620 </td><td>1,581 </td></tr><tr><td colspan="11">≥5 kb </td></tr><tr><td> Number of sequences </td><td>56 </td><td>41 </td><td>10 </td><td>5 </td><td>53 </td><td>41 </td><td>12 </td><td>42 </td><td>35 </td><td>7 </td></tr><tr><td> Total length (b) </td><td>342,564 </td><td>262,044 </td><td>54,009 </td><td>26,511 </td><td>350,736 </td><td>283,644 </td><td>67,092 </td><td>277,953 </td><td>239,226 </td><td>38,727 </td></tr><tr><td> Ave. length (b) </td><td>6,117 </td><td>6,391 </td><td>5,401 </td><td>5,302 </td><td>6,618 </td><td>6,918 </td><td>5,591 </td><td>6,618 </td><td>6,835 </td><td>5,532 </td></tr></tbody></table></div><div content-id="DSW006TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="4"><em>Z. japonica</em> ‘Nagirizaki’<hr /> </th><th colspan="3"><em>Z. matrella</em> ‘Wakaba’<hr /> </th><th colspan="3"><em>Z. pacifica</em> ‘Zanpa’<hr /> </th></tr><tr><th>Total </th><th>mkhc </th><th>mk </th><th>br </th><th>Total </th><th>mkhc </th><th>mk </th><th>Total </th><th>mkhc </th><th>mk </th></tr></thead><tbody><tr><td colspan="11">Total </td></tr><tr><td> Number of sequences </td><td>59,271 </td><td>16,205 </td><td>37,605 </td><td>5,461 </td><td>95,079 </td><td>24,398 </td><td>70,681 </td><td>65,252 </td><td>16,455 </td><td>48,797 </td></tr><tr><td> Total length (b) </td><td>49,483,110 </td><td>16,398,294 </td><td>27,579,822 </td><td>5,504,994 </td><td>76,439,178 </td><td>23,712,921 </td><td>52,726,257 </td><td>52,368,507 </td><td>16,358,223 </td><td>36,010,284 </td></tr><tr><td> Ave. length (b) </td><td>835 </td><td>1,012 </td><td>733 </td><td>1,008 </td><td>804 </td><td>972 </td><td>746 </td><td>803 </td><td>994 </td><td>738 </td></tr><tr><td> Max. length (b) </td><td>11,838 </td><td>11,838 </td><td>6,666 </td><td>5,439 </td><td>13,647 </td><td>13,647 </td><td>6,240 </td><td>14,421 </td><td>14,421 </td><td>6,735 </td></tr><tr><td> Min. length (b) </td><td>108 </td><td>150 </td><td>150 </td><td>108 </td><td>150 </td><td>153 </td><td>150 </td><td>150 </td><td>150 </td><td>150 </td></tr><tr><td> N50 length (b) </td><td>1,119 </td><td>1,254 </td><td>987 </td><td>1,275 </td><td>1,065 </td><td>1,179 </td><td>999 </td><td>1,062 </td><td>1,206 </td><td>978 </td></tr><tr><td> G+C% </td><td>59.3 </td><td>53.2 </td><td>63.2 </td><td>58.3 </td><td>59.6 </td><td>52.6 </td><td>62.7 </td><td>59.6 </td><td>52.7 </td><td>62.8 </td></tr><tr><td colspan="11">≥100 b </td></tr><tr><td> Number of sequences </td><td>59,271 </td><td>16,205 </td><td>37,605 </td><td>5,461 </td><td>95,079 </td><td>24,398 </td><td>70,681 </td><td>65,252 </td><td>16,455 </td><td>48,797 </td></tr><tr><td> Total length (b) </td><td>49,483,110 </td><td>16,398,294 </td><td>27,579,822 </td><td>5,504,994 </td><td>76,439,178 </td><td>23,712,921 </td><td>52,726,257 </td><td>52,368,507 </td><td>16,358,223 </td><td>36,010,284 </td></tr><tr><td> Ave. length (b) </td><td>835 </td><td>1,012 </td><td>733 </td><td>1,008 </td><td>804 </td><td>972 </td><td>746 </td><td>803 </td><td>994 </td><td>738 </td></tr><tr><td colspan="11">≥500 b </td></tr><tr><td> Number of sequences </td><td>36,762 </td><td>12,446 </td><td>20,166 </td><td>4,150 </td><td>58,044 </td><td>18,851 </td><td>39,193 </td><td>39,673 </td><td>12,864 </td><td>26,809 </td></tr><tr><td> Total length (b) </td><td>41,915,830 </td><td>15,083,250 </td><td>21,825,114 </td><td>5,007,466 </td><td>63,962,067 </td><td>21,739,386 </td><td>42,222,681 </td><td>43,778,979 </td><td>15,084,588 </td><td>28,694,391 </td></tr><tr><td> Ave. length (b) </td><td>1,140 </td><td>1,212 </td><td>1,082 </td><td>1,207 </td><td>1,102 </td><td>1,153 </td><td>1,077 </td><td>1,103 </td><td>1,173 </td><td>1,070 </td></tr><tr><td colspan="11">≥1 kb </td></tr><tr><td> Number of sequences </td><td>17,091 </td><td>6,436 </td><td>8,605 </td><td>2,050 </td><td>25,866 </td><td>9,153 </td><td>16,713 </td><td>17,496 </td><td>6,342 </td><td>11,154 </td></tr><tr><td> Total length (b) </td><td>27,773,167 </td><td>10,647,366 </td><td>13,635,474 </td><td>3,490,327 </td><td>40,863,282 </td><td>14,576,319 </td><td>26,286,963 </td><td>27,907,539 </td><td>10,272,498 </td><td>17,635,041 </td></tr><tr><td> Ave. length (b) </td><td>1,625 </td><td>1,654 </td><td>1,585 </td><td>1,703 </td><td>1,580 </td><td>1,593 </td><td>1,573 </td><td>1,595 </td><td>1,620 </td><td>1,581 </td></tr><tr><td colspan="11">≥5 kb </td></tr><tr><td> Number of sequences </td><td>56 </td><td>41 </td><td>10 </td><td>5 </td><td>53 </td><td>41 </td><td>12 </td><td>42 </td><td>35 </td><td>7 </td></tr><tr><td> Total length (b) </td><td>342,564 </td><td>262,044 </td><td>54,009 </td><td>26,511 </td><td>350,736 </td><td>283,644 </td><td>67,092 </td><td>277,953 </td><td>239,226 </td><td>38,727 </td></tr><tr><td> Ave. length (b) </td><td>6,117 </td><td>6,391 </td><td>5,401 </td><td>5,302 </td><td>6,618 </td><td>6,918 </td><td>5,591 </td><td>6,618 </td><td>6,835 </td><td>5,532 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200900" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSW006TB2" class="table-wrap-title" data-id="DSW006TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Statistics of genes predicted in the draft genome sequences of <em>Zoysia</em> species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="4"><em>Z. japonica</em> ‘Nagirizaki’<hr /> </th><th colspan="3"><em>Z. matrella</em> ‘Wakaba’<hr /> </th><th colspan="3"><em>Z. pacifica</em> ‘Zanpa’<hr /> </th></tr><tr><th>Total </th><th>mkhc </th><th>mk </th><th>br </th><th>Total </th><th>mkhc </th><th>mk </th><th>Total </th><th>mkhc </th><th>mk </th></tr></thead><tbody><tr><td colspan="11">Total </td></tr><tr><td> Number of sequences </td><td>59,271 </td><td>16,205 </td><td>37,605 </td><td>5,461 </td><td>95,079 </td><td>24,398 </td><td>70,681 </td><td>65,252 </td><td>16,455 </td><td>48,797 </td></tr><tr><td> Total length (b) </td><td>49,483,110 </td><td>16,398,294 </td><td>27,579,822 </td><td>5,504,994 </td><td>76,439,178 </td><td>23,712,921 </td><td>52,726,257 </td><td>52,368,507 </td><td>16,358,223 </td><td>36,010,284 </td></tr><tr><td> Ave. length (b) </td><td>835 </td><td>1,012 </td><td>733 </td><td>1,008 </td><td>804 </td><td>972 </td><td>746 </td><td>803 </td><td>994 </td><td>738 </td></tr><tr><td> Max. length (b) </td><td>11,838 </td><td>11,838 </td><td>6,666 </td><td>5,439 </td><td>13,647 </td><td>13,647 </td><td>6,240 </td><td>14,421 </td><td>14,421 </td><td>6,735 </td></tr><tr><td> Min. length (b) </td><td>108 </td><td>150 </td><td>150 </td><td>108 </td><td>150 </td><td>153 </td><td>150 </td><td>150 </td><td>150 </td><td>150 </td></tr><tr><td> N50 length (b) </td><td>1,119 </td><td>1,254 </td><td>987 </td><td>1,275 </td><td>1,065 </td><td>1,179 </td><td>999 </td><td>1,062 </td><td>1,206 </td><td>978 </td></tr><tr><td> G+C% </td><td>59.3 </td><td>53.2 </td><td>63.2 </td><td>58.3 </td><td>59.6 </td><td>52.6 </td><td>62.7 </td><td>59.6 </td><td>52.7 </td><td>62.8 </td></tr><tr><td colspan="11">≥100 b </td></tr><tr><td> Number of sequences </td><td>59,271 </td><td>16,205 </td><td>37,605 </td><td>5,461 </td><td>95,079 </td><td>24,398 </td><td>70,681 </td><td>65,252 </td><td>16,455 </td><td>48,797 </td></tr><tr><td> Total length (b) </td><td>49,483,110 </td><td>16,398,294 </td><td>27,579,822 </td><td>5,504,994 </td><td>76,439,178 </td><td>23,712,921 </td><td>52,726,257 </td><td>52,368,507 </td><td>16,358,223 </td><td>36,010,284 </td></tr><tr><td> Ave. length (b) </td><td>835 </td><td>1,012 </td><td>733 </td><td>1,008 </td><td>804 </td><td>972 </td><td>746 </td><td>803 </td><td>994 </td><td>738 </td></tr><tr><td colspan="11">≥500 b </td></tr><tr><td> Number of sequences </td><td>36,762 </td><td>12,446 </td><td>20,166 </td><td>4,150 </td><td>58,044 </td><td>18,851 </td><td>39,193 </td><td>39,673 </td><td>12,864 </td><td>26,809 </td></tr><tr><td> Total length (b) </td><td>41,915,830 </td><td>15,083,250 </td><td>21,825,114 </td><td>5,007,466 </td><td>63,962,067 </td><td>21,739,386 </td><td>42,222,681 </td><td>43,778,979 </td><td>15,084,588 </td><td>28,694,391 </td></tr><tr><td> Ave. length (b) </td><td>1,140 </td><td>1,212 </td><td>1,082 </td><td>1,207 </td><td>1,102 </td><td>1,153 </td><td>1,077 </td><td>1,103 </td><td>1,173 </td><td>1,070 </td></tr><tr><td colspan="11">≥1 kb </td></tr><tr><td> Number of sequences </td><td>17,091 </td><td>6,436 </td><td>8,605 </td><td>2,050 </td><td>25,866 </td><td>9,153 </td><td>16,713 </td><td>17,496 </td><td>6,342 </td><td>11,154 </td></tr><tr><td> Total length (b) </td><td>27,773,167 </td><td>10,647,366 </td><td>13,635,474 </td><td>3,490,327 </td><td>40,863,282 </td><td>14,576,319 </td><td>26,286,963 </td><td>27,907,539 </td><td>10,272,498 </td><td>17,635,041 </td></tr><tr><td> Ave. length (b) </td><td>1,625 </td><td>1,654 </td><td>1,585 </td><td>1,703 </td><td>1,580 </td><td>1,593 </td><td>1,573 </td><td>1,595 </td><td>1,620 </td><td>1,581 </td></tr><tr><td colspan="11">≥5 kb </td></tr><tr><td> Number of sequences </td><td>56 </td><td>41 </td><td>10 </td><td>5 </td><td>53 </td><td>41 </td><td>12 </td><td>42 </td><td>35 </td><td>7 </td></tr><tr><td> Total length (b) </td><td>342,564 </td><td>262,044 </td><td>54,009 </td><td>26,511 </td><td>350,736 </td><td>283,644 </td><td>67,092 </td><td>277,953 </td><td>239,226 </td><td>38,727 </td></tr><tr><td> Ave. length (b) </td><td>6,117 </td><td>6,391 </td><td>5,401 </td><td>5,302 </td><td>6,618 </td><td>6,918 </td><td>5,591 </td><td>6,618 </td><td>6,835 </td><td>5,532 </td></tr></tbody></table></div><div content-id="DSW006TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="4"><em>Z. japonica</em> ‘Nagirizaki’<hr /> </th><th colspan="3"><em>Z. matrella</em> ‘Wakaba’<hr /> </th><th colspan="3"><em>Z. pacifica</em> ‘Zanpa’<hr /> </th></tr><tr><th>Total </th><th>mkhc </th><th>mk </th><th>br </th><th>Total </th><th>mkhc </th><th>mk </th><th>Total </th><th>mkhc </th><th>mk </th></tr></thead><tbody><tr><td colspan="11">Total </td></tr><tr><td> Number of sequences </td><td>59,271 </td><td>16,205 </td><td>37,605 </td><td>5,461 </td><td>95,079 </td><td>24,398 </td><td>70,681 </td><td>65,252 </td><td>16,455 </td><td>48,797 </td></tr><tr><td> Total length (b) </td><td>49,483,110 </td><td>16,398,294 </td><td>27,579,822 </td><td>5,504,994 </td><td>76,439,178 </td><td>23,712,921 </td><td>52,726,257 </td><td>52,368,507 </td><td>16,358,223 </td><td>36,010,284 </td></tr><tr><td> Ave. length (b) </td><td>835 </td><td>1,012 </td><td>733 </td><td>1,008 </td><td>804 </td><td>972 </td><td>746 </td><td>803 </td><td>994 </td><td>738 </td></tr><tr><td> Max. length (b) </td><td>11,838 </td><td>11,838 </td><td>6,666 </td><td>5,439 </td><td>13,647 </td><td>13,647 </td><td>6,240 </td><td>14,421 </td><td>14,421 </td><td>6,735 </td></tr><tr><td> Min. length (b) </td><td>108 </td><td>150 </td><td>150 </td><td>108 </td><td>150 </td><td>153 </td><td>150 </td><td>150 </td><td>150 </td><td>150 </td></tr><tr><td> N50 length (b) </td><td>1,119 </td><td>1,254 </td><td>987 </td><td>1,275 </td><td>1,065 </td><td>1,179 </td><td>999 </td><td>1,062 </td><td>1,206 </td><td>978 </td></tr><tr><td> G+C% </td><td>59.3 </td><td>53.2 </td><td>63.2 </td><td>58.3 </td><td>59.6 </td><td>52.6 </td><td>62.7 </td><td>59.6 </td><td>52.7 </td><td>62.8 </td></tr><tr><td colspan="11">≥100 b </td></tr><tr><td> Number of sequences </td><td>59,271 </td><td>16,205 </td><td>37,605 </td><td>5,461 </td><td>95,079 </td><td>24,398 </td><td>70,681 </td><td>65,252 </td><td>16,455 </td><td>48,797 </td></tr><tr><td> Total length (b) </td><td>49,483,110 </td><td>16,398,294 </td><td>27,579,822 </td><td>5,504,994 </td><td>76,439,178 </td><td>23,712,921 </td><td>52,726,257 </td><td>52,368,507 </td><td>16,358,223 </td><td>36,010,284 </td></tr><tr><td> Ave. length (b) </td><td>835 </td><td>1,012 </td><td>733 </td><td>1,008 </td><td>804 </td><td>972 </td><td>746 </td><td>803 </td><td>994 </td><td>738 </td></tr><tr><td colspan="11">≥500 b </td></tr><tr><td> Number of sequences </td><td>36,762 </td><td>12,446 </td><td>20,166 </td><td>4,150 </td><td>58,044 </td><td>18,851 </td><td>39,193 </td><td>39,673 </td><td>12,864 </td><td>26,809 </td></tr><tr><td> Total length (b) </td><td>41,915,830 </td><td>15,083,250 </td><td>21,825,114 </td><td>5,007,466 </td><td>63,962,067 </td><td>21,739,386 </td><td>42,222,681 </td><td>43,778,979 </td><td>15,084,588 </td><td>28,694,391 </td></tr><tr><td> Ave. length (b) </td><td>1,140 </td><td>1,212 </td><td>1,082 </td><td>1,207 </td><td>1,102 </td><td>1,153 </td><td>1,077 </td><td>1,103 </td><td>1,173 </td><td>1,070 </td></tr><tr><td colspan="11">≥1 kb </td></tr><tr><td> Number of sequences </td><td>17,091 </td><td>6,436 </td><td>8,605 </td><td>2,050 </td><td>25,866 </td><td>9,153 </td><td>16,713 </td><td>17,496 </td><td>6,342 </td><td>11,154 </td></tr><tr><td> Total length (b) </td><td>27,773,167 </td><td>10,647,366 </td><td>13,635,474 </td><td>3,490,327 </td><td>40,863,282 </td><td>14,576,319 </td><td>26,286,963 </td><td>27,907,539 </td><td>10,272,498 </td><td>17,635,041 </td></tr><tr><td> Ave. length (b) </td><td>1,625 </td><td>1,654 </td><td>1,585 </td><td>1,703 </td><td>1,580 </td><td>1,593 </td><td>1,573 </td><td>1,595 </td><td>1,620 </td><td>1,581 </td></tr><tr><td colspan="11">≥5 kb </td></tr><tr><td> Number of sequences </td><td>56 </td><td>41 </td><td>10 </td><td>5 </td><td>53 </td><td>41 </td><td>12 </td><td>42 </td><td>35 </td><td>7 </td></tr><tr><td> Total length (b) </td><td>342,564 </td><td>262,044 </td><td>54,009 </td><td>26,511 </td><td>350,736 </td><td>283,644 </td><td>67,092 </td><td>277,953 </td><td>239,226 </td><td>38,727 </td></tr><tr><td> Ave. length (b) </td><td>6,117 </td><td>6,391 </td><td>5,401 </td><td>5,302 </td><td>6,618 </td><td>6,918 </td><td>5,591 </td><td>6,618 </td><td>6,835 </td><td>5,532 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200900" target="_blank">View Large</a></div></div><p>The predicted protein-coding genes of <em>Z. japonica ‘</em>Nagirizaki’, <em>O. sativa</em>, <em>S. bicolor</em>, and <em>B. distachyon</em> were assigned to the functional categories of NCBI's euKaryotic clusters of Orthologous Groups (KOG). The percentages of proteins in each KOG category are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S2</a></span>. The four species showed similar percentages of proteins in each functional category. The predicted genes were then classified into GO slim categories based on the results of the domain searches using InterProScan against the InterPro database. The percentages of genes in the root categories Biological Process (BP), Cellular Components (CC) and Molecular Function (MF) were also similar among the four related species (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S3</a></span>).</p><p>The metabolic pathways of <em>Z. japonica</em> ‘Nagirizaki’, <em>O. sativa</em>, <em>S. bicolor</em>, and <em>B. distachyon</em> were then compared by mapping the genes of each genome onto the reference map of pathways in the ‘Metabolism’ group of the KEGG database. As a result, most of the enzymes found in <em>Z. japonica</em> ‘Nagirizaki’ were conserved in other three Poaceae species (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). However, there were eight enzymes found only in <em>Z. japonica</em> ‘Nagirizaki’, which were classified into seven pathways including ‘Sesquiterpenoid and triterpenoid biosynthesis’, ‘Benzoxazinoid biosynthesis’, and ‘Benzoate degradation’ (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). The entire results from the mapping of the metabolic pathways are available on the ‘Zoysia Genome Database’ at <a class="link link-uri" href="http://zoysia.kazusa.or.jp">http://zoysia.kazusa.or.jp</a>.</p>                            <h4 scrollto-destination=87200904 id="87200904" class="section-title" data-legacy-id=s3c2>3.3.2 Gene characteristics of <em>Zoysia</em> species</h4>
<p>Plant adaptation to high saline environments is mediated by complex interactions among salt-stress responsive genes and transcription factors (TFs).<sup><a reveal-id="DSW006C46" data-open="DSW006C46" class="link link-ref link-reveal xref-bibr">46</a></sup> The NAC family is one of the largest groups of plant-specific TFs involved in plant development and abiotic stress. A member of the NAC family, ANAC102, was reported to play a role in salinity tolerance in <em>Arabidopsis</em>.<sup><a reveal-id="DSW006C47" data-open="DSW006C47" class="link link-ref link-reveal xref-bibr">47</a></sup> To elucidate the high salt tolerance mechanisms in <em>Zoysia</em> species, we searched for ANAC102-like genes in the three <em>Zoysia</em> genome sequences. Two ANAC102-like genes were identified in each <em>Zoysia</em> genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S4</a></span>). Among the first group (Zjn_sc00013.1.g08180.1, Zmw_sc04858.1.g00100.1, and Zpz_sc00127.1.g00380.1) we found high degrees of similarity in amino acid sequences and small number of mutations, and each gene was equally represented in the three species. The second group (Zjn_sc00014.1.g08280.1, Zmw_sc02298.1.g00200.1, Zmw_sc02531.1.g00150.1 and Zpz_sc01716.1.g00180.1) was similar to the first group in amino acid sequences but had differences in the mutation sites. However, <em>Z. japonica</em> and <em>Z. pacifica</em> each had a distinctive sequence feature in the corresponding 5′ upstream region, and both features were found in <em>Z. matrella</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S5</a></span>). Based on the positions of these ANAC102-like genes on the ‘Nagirizaki’ pseudomolecules, two copies were retained on both homeologous chromosomes (Zjn_chr01 and Zjn_chr02). These differences of ANAC102<em>-</em>like genes may provide some contribution to salt tolerance in the <em>Zoysia</em> species.</p><p>The B-box type zinc-finger family of TFs are also key factors in regulatory networks such as those related to salt stress.<sup><a reveal-id="DSW006C48" data-open="DSW006C48" class="link link-ref link-reveal xref-bibr">48</a></sup> Specifically, the Salt Tolerance/B-Box Zinc-Finger Protein 24 (STO/BBX24) of <em>A. thaliana</em> was found to increase salt tolerance in yeast (<em>Saccharomyces cerevisiae</em>),<sup><a reveal-id="DSW006C49" data-open="DSW006C49" class="link link-ref link-reveal xref-bibr">49</a></sup> and the overexpression of <em>STO/BBX24</em> enhances the root growth of <em>Arabidopsis</em> seedlings in a high salt medium.<sup><a reveal-id="DSW006C50" data-open="DSW006C50" class="link link-ref link-reveal xref-bibr">50</a></sup> To investigate the contribution of STO/BBX24 to salt tolerance in <em>Zoysia</em> species, we compared the structures of the <em>STO/BBX24</em> genes in the three <em>Zoysia</em> species. We found two structurally distinguishable sequences in corresponding regions of the <em>Z. japonica</em> and <em>Z. pacifica</em> genomes, while both alleles were found in <em>Z. matrella</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S6</a></span>). Moreover, mutations were found in the 5′ upstream region of each <em>STO/BBX24</em> gene (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S7</a></span>). <em>Zoysia matrella</em> tends to possess higher level of salt tolerance than <em>Z. japonica</em> and <em>Z. pacifica</em>.<sup><a reveal-id="DSW006C2" data-open="DSW006C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSW006C51" data-open="DSW006C51" class="link link-ref link-reveal xref-bibr">51</a></sup> The polymorphisms that we found in the <em>ANAC102</em> and <em>STO/BBX24</em> genes may imply that these genes contribute to the variation in salt tolerance levels among these three <em>Zoysia</em> species.</p><p>Another zinc-binding B-box domain protein that is involved in plant responses to salinity and dehydration is a leucine zipper (bZIP) TF.<sup><a reveal-id="DSW006C52" data-open="DSW006C52" class="link link-ref link-reveal xref-bibr">52</a>,<a reveal-id="DSW006C53" data-open="DSW006C53" class="link link-ref link-reveal xref-bibr">53</a></sup><em>Sporobolus virginicus</em> is another halophyte turfgrass in the tribe <em>Zoysiae</em>, along with the <em>Zoysia</em> species. A comprehensive analysis of transcripts involved in salt tolerance in, <em>S. virginicus</em> revealed five <em>bZIP</em> genes with root-specific salt-responsive expression.<sup><a reveal-id="DSW006C54" data-open="DSW006C54" class="link link-ref link-reveal xref-bibr">54</a></sup> Only two of these <em>bZIP</em> genes (PK07387.1 and evm_27.model.AmTr_v1.0_scaffold00081.79 in PlantTFDB3.0<sup><a reveal-id="DSW006C55" data-open="DSW006C55" class="link link-ref link-reveal xref-bibr">55</a></sup>) were conserved in the <em>Zoysia</em> genomes. This suggests that <em>Zoysia</em> species may have different mechanisms for salt tolerance than those in <em>S. virginicus</em>.</p><p>Anthocyanins are a class of phenolic compounds responsible for the red to purple pigmentation in plants. They are involved in plant defenses and stress avoidance due to their antioxidant properties.<sup><a reveal-id="DSW006C56" data-open="DSW006C56" class="link link-ref link-reveal xref-bibr">56</a></sup> Ahn et al.<sup><a reveal-id="DSW006C11" data-open="DSW006C11" class="link link-ref link-reveal xref-bibr">11</a></sup> reported that zoysiagrasses exhibit red to purple pigments in their spike and stolon tissues. A transcriptome analysis of <em>Z. japonica</em> accessions with either green or purple tissues was used to identify two genes related to anthocyanin biosynthesis: anthocyanidin synthase (<em>ANS</em>) and dihydroflavonol 4-reductase (<em>DFR</em>). In another study, a <em>Ginko biloba ANS</em> gene showed synergistic patterns of expression in anthocyanin accumulation and abiotic stress responses.<sup><a reveal-id="DSW006C57" data-open="DSW006C57" class="link link-ref link-reveal xref-bibr">57</a></sup> Therefore, <em>ANS</em> genes are implicated in responses to adverse environment stresses including high levels of salinity.<sup><a reveal-id="DSW006C57" data-open="DSW006C57" class="link link-ref link-reveal xref-bibr">57</a></sup> A single <em>ANS1</em> orthologue was identified in each of <em>Z. japonica</em> and <em>Z. pacifica</em>, and these genes were designated as <em>ZjANS1</em> and <em>ZpANS1</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S8</a></span>). A comparison of the <em>ZjANS1</em> and <em>ZpANS1</em> sequences revealed a number of SNPs and indels in the 5′ upstream regions and 27 SNPs in the coding regions, including an SNP in the termination codon of <em>ZjANS1</em> that extends the coding region for an additional 27 bp. These polymorphisms may reflect the diversity of anthocyanin pigmentation within and between species of the genus <em>Zoysia</em>, as described in previous reports.<sup><a reveal-id="DSW006C11" data-open="DSW006C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSW006C58" data-open="DSW006C58" class="link link-ref link-reveal xref-bibr">58</a></sup> As is the case for the <em>ANAC102</em> and <em>STO/BBX24</em>, the <em>ANS1</em> gene was heterozygous in <em>Z. matrella</em>, and both <em>ZjANS1</em> and <em>ZpANS1</em> were identified in the ‘Wakaba’ genome assembly (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S9</a></span>). <em>Zoysia matrella</em> shows wide variations in morphological traits, most of which are intermediate between those of <em>Z. japonica</em> and <em>Z. pacifica</em>. Tanaka et al.<sup><a reveal-id="DSW006C5" data-open="DSW006C5" class="link link-ref link-reveal xref-bibr">5</a></sup> conducted an SSR marker analysis and suggested that <em>Z. matrella</em> might represent an interspecific hybrid between <em>Z. japonica</em> and <em>Z. pacifica</em>. The combinations of the <em>ANAC102</em>, <em>STO/BBX24</em>, and <em>ANS1</em> alleles in the three <em>Zoysia</em> genomes provide additional supportive evidence for this hypothesis.</p>                            <h3 scrollto-destination=87200911 id="87200911" class="section-title" data-legacy-id=s3d>3.4 Comparative genome analysis</h3>
                            <h4 scrollto-destination=87200912 id="87200912" class="section-title" data-legacy-id=s3d1>3.4.1 Comparison among <em>Z. japonica</em> and other <em>Zoysia</em> species</h4>
<p>In order to investigate the phylogenetic relationships among the <em>Zoysia</em> species, we collected genome sequence reads from three additional accessions, <em>Z. japonica</em> ‘Kyoto’, <em>Z. japonica</em> ‘Miyagi’, and <em>Z. matrella</em> ‘Chiba Fair Green’. The reads from these three accessions and those from <em>Z. matrella</em> ‘Wakaba’ and <em>Z. pacifica</em> ‘Zanpa’ were aligned against the reference genome of <em>Z. japonica ‘</em>Nagirizaki’. We identified a total of 7,424,163 SNPs including 3,660,698 SNPs in two or more accessions (common SNPs) (Table <a reveal-id="DSW006TB3" data-open="DSW006TB3" class="link link-reveal link-table xref-fig">3</a>). Additionally, we found a total of 852,488 short indels of 1–60 bp; of which, 319,551 were common to two or more accessions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S12</a></span>). These total numbers of SNPs and indels are well correlated with the previously reported phylogenetic relationships among <em>Zoysia</em> species based on DNA marker analyses.<sup><a reveal-id="DSW006C4" data-open="DSW006C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSW006C59" data-open="DSW006C59" class="link link-ref link-reveal xref-bibr">59</a>,<a reveal-id="DSW006C60" data-open="DSW006C60" class="link link-ref link-reveal xref-bibr">60</a></sup> In <em>Z. pacifica</em> ‘Zanpa’, the majority of SNPs and indels were homozygous in each allele, whereas <em>Z. matrella</em> ‘Wakaba’ and ‘Chiba Fair Green’ had higher numbers of heterozygous SNPs and indels (Table <a reveal-id="DSW006TB3" data-open="DSW006TB3" class="link link-reveal link-table xref-fig">3</a>). </p>                        <a id="87200914" scrollto-destination="87200914"></a>
<div content-id="DSW006TB3" class="table-modal"><div class="table-wrap"><div id="DSW006TB3" class="table-wrap-title" data-id="DSW006TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Summary of SNPs between <em>Z. japonica</em> ‘Nagirizaki’ and other <em>Zoysia</em> species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th rowspan="2">All SNPs </th><th colspan="3">Common SNP sites<hr /> </th><th colspan="3">Specific SNP sites<hr /> </th></tr><tr><th>Total number </th><th>ALT/homo </th><th>Hetero </th><th>Total number </th><th>ALT/homo </th><th>Hetero </th></tr></thead><tbody><tr><td>Total of 5 accessions </td><td>7,424,163 </td><td>3,660,698 </td><td>– </td><td>– </td><td>3,763,465 </td><td>– </td><td>– </td></tr><tr><td colspan="8"><em>Z. japonica</em> </td></tr><tr><td> Kyoto </td><td>1,011,037 </td><td>749,624 </td><td>434,991 </td><td>314,633 </td><td>261,413 </td><td>90,960 </td><td>170,453 </td></tr><tr><td> Miyagi </td><td>1,055,665 </td><td>789,694 </td><td>627,166 </td><td>162,528 </td><td>265,971 </td><td>177,898 </td><td>88,073 </td></tr><tr><td colspan="8"><em>Z. matrella</em> </td></tr><tr><td> Wakaba </td><td>3,929,947 </td><td>2,896,705 </td><td>964,659 </td><td>1,932,046 </td><td>1,033,242 </td><td>384,846 </td><td>648,396 </td></tr><tr><td> Chiba Fair Green </td><td>3,182,483 </td><td>2,511,362 </td><td>813,474 </td><td>1,697,888 </td><td>671,121 </td><td>210,065 </td><td>461,056 </td></tr><tr><td colspan="8"><em>Z. pacifica</em> </td></tr><tr><td> Zanpa </td><td>4,228,938 </td><td>2,697,220 </td><td>2,315,529 </td><td>381,691 </td><td>1,531,718 </td><td>958,605 </td><td>573,113 </td></tr></tbody></table></div><div content-id="DSW006TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th rowspan="2">All SNPs </th><th colspan="3">Common SNP sites<hr /> </th><th colspan="3">Specific SNP sites<hr /> </th></tr><tr><th>Total number </th><th>ALT/homo </th><th>Hetero </th><th>Total number </th><th>ALT/homo </th><th>Hetero </th></tr></thead><tbody><tr><td>Total of 5 accessions </td><td>7,424,163 </td><td>3,660,698 </td><td>– </td><td>– </td><td>3,763,465 </td><td>– </td><td>– </td></tr><tr><td colspan="8"><em>Z. japonica</em> </td></tr><tr><td> Kyoto </td><td>1,011,037 </td><td>749,624 </td><td>434,991 </td><td>314,633 </td><td>261,413 </td><td>90,960 </td><td>170,453 </td></tr><tr><td> Miyagi </td><td>1,055,665 </td><td>789,694 </td><td>627,166 </td><td>162,528 </td><td>265,971 </td><td>177,898 </td><td>88,073 </td></tr><tr><td colspan="8"><em>Z. matrella</em> </td></tr><tr><td> Wakaba </td><td>3,929,947 </td><td>2,896,705 </td><td>964,659 </td><td>1,932,046 </td><td>1,033,242 </td><td>384,846 </td><td>648,396 </td></tr><tr><td> Chiba Fair Green </td><td>3,182,483 </td><td>2,511,362 </td><td>813,474 </td><td>1,697,888 </td><td>671,121 </td><td>210,065 </td><td>461,056 </td></tr><tr><td colspan="8"><em>Z. pacifica</em> </td></tr><tr><td> Zanpa </td><td>4,228,938 </td><td>2,697,220 </td><td>2,315,529 </td><td>381,691 </td><td>1,531,718 </td><td>958,605 </td><td>573,113 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200914" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSW006TB3" class="table-wrap-title" data-id="DSW006TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Summary of SNPs between <em>Z. japonica</em> ‘Nagirizaki’ and other <em>Zoysia</em> species</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th rowspan="2">All SNPs </th><th colspan="3">Common SNP sites<hr /> </th><th colspan="3">Specific SNP sites<hr /> </th></tr><tr><th>Total number </th><th>ALT/homo </th><th>Hetero </th><th>Total number </th><th>ALT/homo </th><th>Hetero </th></tr></thead><tbody><tr><td>Total of 5 accessions </td><td>7,424,163 </td><td>3,660,698 </td><td>– </td><td>– </td><td>3,763,465 </td><td>– </td><td>– </td></tr><tr><td colspan="8"><em>Z. japonica</em> </td></tr><tr><td> Kyoto </td><td>1,011,037 </td><td>749,624 </td><td>434,991 </td><td>314,633 </td><td>261,413 </td><td>90,960 </td><td>170,453 </td></tr><tr><td> Miyagi </td><td>1,055,665 </td><td>789,694 </td><td>627,166 </td><td>162,528 </td><td>265,971 </td><td>177,898 </td><td>88,073 </td></tr><tr><td colspan="8"><em>Z. matrella</em> </td></tr><tr><td> Wakaba </td><td>3,929,947 </td><td>2,896,705 </td><td>964,659 </td><td>1,932,046 </td><td>1,033,242 </td><td>384,846 </td><td>648,396 </td></tr><tr><td> Chiba Fair Green </td><td>3,182,483 </td><td>2,511,362 </td><td>813,474 </td><td>1,697,888 </td><td>671,121 </td><td>210,065 </td><td>461,056 </td></tr><tr><td colspan="8"><em>Z. pacifica</em> </td></tr><tr><td> Zanpa </td><td>4,228,938 </td><td>2,697,220 </td><td>2,315,529 </td><td>381,691 </td><td>1,531,718 </td><td>958,605 </td><td>573,113 </td></tr></tbody></table></div><div content-id="DSW006TB3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th rowspan="2">All SNPs </th><th colspan="3">Common SNP sites<hr /> </th><th colspan="3">Specific SNP sites<hr /> </th></tr><tr><th>Total number </th><th>ALT/homo </th><th>Hetero </th><th>Total number </th><th>ALT/homo </th><th>Hetero </th></tr></thead><tbody><tr><td>Total of 5 accessions </td><td>7,424,163 </td><td>3,660,698 </td><td>– </td><td>– </td><td>3,763,465 </td><td>– </td><td>– </td></tr><tr><td colspan="8"><em>Z. japonica</em> </td></tr><tr><td> Kyoto </td><td>1,011,037 </td><td>749,624 </td><td>434,991 </td><td>314,633 </td><td>261,413 </td><td>90,960 </td><td>170,453 </td></tr><tr><td> Miyagi </td><td>1,055,665 </td><td>789,694 </td><td>627,166 </td><td>162,528 </td><td>265,971 </td><td>177,898 </td><td>88,073 </td></tr><tr><td colspan="8"><em>Z. matrella</em> </td></tr><tr><td> Wakaba </td><td>3,929,947 </td><td>2,896,705 </td><td>964,659 </td><td>1,932,046 </td><td>1,033,242 </td><td>384,846 </td><td>648,396 </td></tr><tr><td> Chiba Fair Green </td><td>3,182,483 </td><td>2,511,362 </td><td>813,474 </td><td>1,697,888 </td><td>671,121 </td><td>210,065 </td><td>461,056 </td></tr><tr><td colspan="8"><em>Z. pacifica</em> </td></tr><tr><td> Zanpa </td><td>4,228,938 </td><td>2,697,220 </td><td>2,315,529 </td><td>381,691 </td><td>1,531,718 </td><td>958,605 </td><td>573,113 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87200914" target="_blank">View Large</a></div></div><p>To investigate the homozygous and heterozygous features of the <em>Zoysia</em> genomes, the genotype of each SNP locus was classified into one of the three types: homozygous with the reference allele (R), homozygous with an alternative allele (A), or heterozygous (H). The zoysiagrass accessions were compared in the following order: <em>Z. japonica</em> ‘Kyoto’, <em>Z. japonica</em> ‘Miyagi’, <em>Z. matrella</em> ‘Wakaba’, <em>Z. matrella</em> ‘Chiba Fair Green’, and <em>Z. pacifica</em> ‘Zanpa’. The most common genotype pattern was R-R-H-H-A, with 670,724 SNP sites (18.3% of total), followed by R-R-H-R-A (13.4%), R-R-A-R-A (5.8%), and R-R-R-H-A (5.8%) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S13</a></span>). A similar tendency was observed in the pattern of common indels (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S14</a></span>). The results indicate that most of the polymorphisms among the three <em>Zoysia</em> species are between <em>Z. japonica</em> and <em>Z. pacifica</em>, and these sites are often heterozygous in <em>Z. matrella</em>. These data, along with the heterozygosity of the <em>ANAC102</em>, <em>STO/BBX24</em>, and <em>ANS1</em> genes in <em>Z. matrella</em>, and the results from our previous study using SSR markers<sup><a reveal-id="DSW006C5" data-open="DSW006C5" class="link link-ref link-reveal xref-bibr">5</a></sup> all support the high probability that <em>Z. matrella</em> is an interspecific hybrid between <em>Z. japonica</em> and <em>Z. pacifica</em>. This possibility would further support a hypothesis that <em>Z. matrella</em> is a homoploid hybrid species since the nature of <em>Z. matrella</em> reflects the criteria for homoploid hybrid speciation. These criteria include: similar chromosome numbers with parental species, ability to backcross with parental species, intermediate phenotypic traits, and ecological divergence from parents.<sup><a reveal-id="DSW006C61" data-open="DSW006C61" class="link link-ref link-reveal xref-bibr">61</a></sup> The information obtained in this study will provide new clues about the origin of <em>Zoysia</em>. In addition, our results highlight the importance of hybridization and polyploidization in grass diversification, which have implications for conservation.</p>                            <h4 scrollto-destination=87200917 id="87200917" class="section-title" data-legacy-id=s3d2>3.4.2. Comparisons with other Poaceae species</h4>
<p>Synteny among the Poaceae genomes was investigated by comparing the positions of genes on the <em>Z. japonica</em> ‘Nagirizaki’ pseudomolecules with their positions in the genome sequences of <em>O. sativa</em>, <em>S. bicolor</em>, and <em>B. distachyon</em> (see Materials and methods section for details). As shown in a Venn diagram in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S10</a></span>, 8,596 of 53,625 genes (16.0%) were conserved in these four Poaceae species, while 17,288 genes (32.2%) occurred in clusters only found in <em>Z. japonica</em>. Based on the homologous genes thus identified and their positions on the pseudomolecules, we generated dot-plot diagrams between <em>Z. japonica</em> and the three other Poaceae species. The results revealed extensive collinearities in all cases, and remarkable collinearities between the entire genomes of <em>O. sativa</em> and <em>S. bicolor</em> (Fig. <a reveal-id="DSW006F1" data-open="DSW006F1" class="link link-reveal link-table xref-fig">1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S11 and S12</a></span>). The comparison between <em>Z. japonica</em> and <em>S. bicolor</em> clearly revealed the following rearrangements: the middle section of <em>S. bicolor</em> chromosome 1 corresponded with the middle sections of <em>Z. japonica</em> chromosomes 7 and 8, and the middle section of <em>S. bicolor</em> chromosome 2 corresponded with the middle sections of <em>Z. japonica</em> chromosomes 19 and 20 (Fig. <a reveal-id="DSW006F1" data-open="DSW006F1" class="link link-reveal link-table xref-fig">1</a>). The rearranged regions of chromosomes 7 and 8 and chromosomes 19 and 20 in <em>Z. japonica</em> corresponded to chromosomes 10 and 9 in <em>O. sativa</em>, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S11</a></span>)<em>.</em> These rearrangements between the <em>Z. japonica</em> and <em>S. bicolor</em> genomes support the possibility that ‘invading’ chromosomes, the paleo-ancestors of chromosomes 10 and 9, integrated into different sets of chromosomes in <em>Z. japonica</em> and <em>S. bicolor</em>, as discussed in a previous study using RAD markers of <em>Z. japonica.</em><sup><a reveal-id="DSW006C8" data-open="DSW006C8" class="link link-ref link-reveal xref-bibr">8</a></sup></p>                        <a id="87200920" scrollto-destination="87200920"></a>
<div id="DSW006F1" data-id="DSW006F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/m_dsw00601.jpeg?Expires=2147483647&Signature=V6XC~XNOeeOg7JxdAgI5xi-FYwhp3fF27yYmyk7ZpoYFg4lMJIhuuYKeZRrmb3EihvCUg-wbiUptcc4YxVGWoDGRCs0Fh0w5yV2A7UbP6NlagwWghcEe6H4YXPzPUAZ-Bb3zQI~-BgkP47SA73X2LZvjmT6qtSjm4yFw2GdELVChPumsmulCJZXwpwvVmpGokKyovvmOTu9VK7fZOT-Jfe7Oa06MHwEnpRcyMqKGg7HJVhoySaYT04Xc1zePBnd38GDrCalNmP4JxVbP1CEVXh-o120mUBIwkwPlaJ14wcQVraMiReL-28d5Wu7-2Q20CFFHtPU47Y78a-YtIXz6sQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison between the Z. japonica ‘Nagirizaki’ and S. bicolor genomes. The translated protein sequences on the pseudomolecules of Z. japonica ‘Nagirizaki’ and S. bicolor were compared by BLAST searches with an E-value cut-off of 1E−100, and the top hits were plotted. A hit on the same strand between both species is coloured in red, while hits on reverse strands are coloured in blue." path-from-xml="dsw00601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200920/dsw00601.jpeg" path-from-xml="dsw00601.jpeg" target="_blank">View large</a><a section="87200920" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw00601.jpeg?Expires=2147483647&Signature=bywT-KRoTpf2y~jJPTXw5agWBDvXA0yA075t36zIuzfSbzjskB-Tudp0ZzNLwPU34Fp65uOEpkRhtubsyZf4U2u3LNsq3hLFNzy~34kcXAh7xC7cYHIjlclrLx9rLpDxvybeMd9EyXxwtVob3brthBtP8hD9jUVYHibtAOkhYV867p3Q6twr~JlHWDSVYvRhql9EY8iBmyIBmeu7wnzFgEdZ73LajzXfiQM8V6M6mt~-7-0vjCBslh0V31Wpp4UUjUVjtgBXfo5mmy1ngLlJdYxUm4YnLfP65YScxcfmvwhdbsWlCfEGvO3eDui0puoxQF9HXORcF1ftIZRq7vJEHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200920&ar=1745492&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison between the <em>Z. japonica</em> ‘Nagirizaki’ and <em>S. bicolor</em> genomes. The translated protein sequences on the pseudomolecules of <em>Z. japonica</em> ‘Nagirizaki’ and <em>S. bicolor</em> were compared by BLAST searches with an E-value cut-off of 1E−100, and the top hits were plotted. A hit on the same strand between both species is coloured in red, while hits on reverse strands are coloured in blue.</p></div></div><div content-id="DSW006F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/m_dsw00601.jpeg?Expires=2147483647&Signature=V6XC~XNOeeOg7JxdAgI5xi-FYwhp3fF27yYmyk7ZpoYFg4lMJIhuuYKeZRrmb3EihvCUg-wbiUptcc4YxVGWoDGRCs0Fh0w5yV2A7UbP6NlagwWghcEe6H4YXPzPUAZ-Bb3zQI~-BgkP47SA73X2LZvjmT6qtSjm4yFw2GdELVChPumsmulCJZXwpwvVmpGokKyovvmOTu9VK7fZOT-Jfe7Oa06MHwEnpRcyMqKGg7HJVhoySaYT04Xc1zePBnd38GDrCalNmP4JxVbP1CEVXh-o120mUBIwkwPlaJ14wcQVraMiReL-28d5Wu7-2Q20CFFHtPU47Y78a-YtIXz6sQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison between the Z. japonica ‘Nagirizaki’ and S. bicolor genomes. The translated protein sequences on the pseudomolecules of Z. japonica ‘Nagirizaki’ and S. bicolor were compared by BLAST searches with an E-value cut-off of 1E−100, and the top hits were plotted. A hit on the same strand between both species is coloured in red, while hits on reverse strands are coloured in blue." path-from-xml="dsw00601.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87200920/dsw00601.jpeg" path-from-xml="dsw00601.jpeg" target="_blank">View large</a><a section="87200920" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw00601.jpeg?Expires=2147483647&Signature=bywT-KRoTpf2y~jJPTXw5agWBDvXA0yA075t36zIuzfSbzjskB-Tudp0ZzNLwPU34Fp65uOEpkRhtubsyZf4U2u3LNsq3hLFNzy~34kcXAh7xC7cYHIjlclrLx9rLpDxvybeMd9EyXxwtVob3brthBtP8hD9jUVYHibtAOkhYV867p3Q6twr~JlHWDSVYvRhql9EY8iBmyIBmeu7wnzFgEdZ73LajzXfiQM8V6M6mt~-7-0vjCBslh0V31Wpp4UUjUVjtgBXfo5mmy1ngLlJdYxUm4YnLfP65YScxcfmvwhdbsWlCfEGvO3eDui0puoxQF9HXORcF1ftIZRq7vJEHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87200920&ar=1745492&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw00601.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison between the <em>Z. japonica</em> ‘Nagirizaki’ and <em>S. bicolor</em> genomes. The translated protein sequences on the pseudomolecules of <em>Z. japonica</em> ‘Nagirizaki’ and <em>S. bicolor</em> were compared by BLAST searches with an E-value cut-off of 1E−100, and the top hits were plotted. A hit on the same strand between both species is coloured in red, while hits on reverse strands are coloured in blue.</p></div></div>                            <h3 scrollto-destination=87200922 id="87200922" class="section-title" data-legacy-id=s3e>3.5 Databases and data retrieval</h3>
<p>The draft genome sequences, gene and protein sequences, and annotation files (gff files) of <em>Z. japonica</em> ‘Nagirizaki’ (Zjn_r1.1), <em>Z. matrella</em> ‘Wakaba’ (Zmw_r1.0) and <em>Z. pacifica</em> ‘Zanpa’ (Zpz_r1.0) can be retrieved from the Zoysia Genome Database (<a class="link link-uri" href="http://zoysia.kazusa.or.jp">http://zoysia.kazusa.or.jp</a>). The pseudomolecule sequences of <em>Z. japonica</em> ‘Nagirizaki’ are also provided. In this database, the BLAST searches against genome sequences and genes are available by inputting query sequences in the BLAST search page. Users can then browse the BLAST search results against databases with annotations, such as NCBI's NR database and TAIR10 pep.</p><p>The accession number in NCBI's BioProject database for the raw sequence data obtained in this study is PRJDB4210. The scaffolds of ‘Nagirizaki’ (Zjn_r1.1), ‘Wakaba’ (Zmw_r1.0), and ‘Zanpa’ (Zpz_r1.0) are available from DDBJ/EMBL/NCBI under the accession numbers BCLF01000001–BCLF01011786 (11,786 entries), BCLG01000001–BCLG01013609 (13,609 entries), and BCLH01000001–BCLH01011428 (11,428 entries), respectively. The llumina PE and MP reads from MiSeq and HiSeq 2000 are available from DDBJ's DRA database under the accession number DRA004073. BioSample accession numbers of the species are as follows: SAMD00041943 (<em>Z. japonica</em> ‘Nagirizaki’), SAMD00041944 (<em>Z. matrella</em> ‘Wakaba’), SAMD00041945 (<em>Z. pacifica</em> ‘Zanpa’), SAMD00041942 (<em>Z. japonica</em> ‘Kyoto’), SAMD00041941 (<em>Z. japonica</em> ‘Miyagi’), and SAMD00041940 (<em>Z. matrella</em> ‘Chiba Fair Green’).</p>                            <h2 scrollto-destination=87200926 id="87200926" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by the Kazusa DNA Research Institute Foundation. Funding to pay the Open Access publication charges for this article was provided by the Kazusa DNA Research Institute Foundation.</p>                            <h2 scrollto-destination=87200928 id="87200928" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank T. Fujishiro, C. Jige, K. Kawashima, Y. Kishida, M. Kohara and N. Nakazaki for excellent technical assistance. Thanks are also due to the Chiba Prefectural Agriculture and Forestry Research Center and Mr. Ippei Katayama in Kyoto Prefectural Katsura Senior High School for kindly providing the plant materials.</p>                            <h2 scrollto-destination=87200934 id="87200934" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSW006C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Huang</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">DaCosta</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Jiang</div> <div class="given-names">Y.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Research advances in mechanisms of turfgrass tolerance to abiotic stresses: from physiology to molecular biology</div>, <div class="source">Crit. Rev. Plant Sci.</div> , <div class="volume">33</div>, <div class="fpage">141</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Research%20advances%20in%20mechanisms%20of%20turfgrass%20tolerance%20to%20abiotic%20stresses%3A%20from%20physiology%20to%20molecular%20biology&amp;author=B.%20Huang&amp;author=M.%20DaCosta&amp;author=Y.%20Jiang&amp;publication_year=2014&amp;journal=Crit.%20Rev.%20Plant%20Sci.&amp;volume=33&amp;pages=141-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/07352689.2014.870411" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F07352689.2014.870411" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F07352689.2014.870411"> </span></div> </div></div></div></div></div><div content-id="DSW006C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Marcum</div>
               <div class="given-names">K.B.</div>
            </div>
          </div>
          <div class="year">1999</div>, <div class="article-title">Salinity tolerance mechanisms of grasses in the subfamily Chloridoideae</div>, <div class="source">Crop Sci.</div> , <div class="volume">39</div>, <div class="fpage">1153</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Salinity%20tolerance%20mechanisms%20of%20grasses%20in%20the%20subfamily%20Chloridoideae&amp;author=K.B.%20Marcum&amp;publication_year=1999&amp;journal=Crop%20Sci.&amp;volume=39&amp;pages=1153-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci1999.0011183X003900040034x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci1999.0011183X003900040034x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci1999.0011183X003900040034x"> </span></div> </div></div></div></div></div><div content-id="DSW006C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kitamura</div>
               <div class="given-names">F.</div>
            </div>
          </div>
          <div class="year">1989</div>, <div class="article-title">The climate of Japan and its surrounding areas and the distribution and classification of zoysiagrasses</div>, <div class="source">Int. Turfgrass Soc. Res. J.</div> , <div class="volume">6</div>, <div class="fpage">17</div>–<div class="lpage">21</div>.</div></div></div></div><div content-id="DSW006C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kimball</div>
               <div class="given-names">J.A.</div>
            </div>, <div class="name string-name"><div class="surname">Zuleta</div> <div class="given-names">C.M.</div></div>, <div class="name string-name"><div class="surname">Kenworthy</div> <div class="given-names">K.E.</div></div>, <div class="name string-name"><div class="surname">Lehman</div> <div class="given-names">V.G.</div></div>, <div class="name string-name"><div class="surname">Harris-Shultz</div> <div class="given-names">K.R.</div></div>, <div class="name string-name"><div class="surname">Milla-Lewis</div> <div class="given-names">S.R.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Genetic relationships in <em>Zoysia</em> species and the identification of putative interspecific hybrids using simple sequence repeat markers and inflorescence traits</div>, <div class="source">Crop Sci.</div> , <div class="volume">53</div>, <div class="fpage">285</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20relationships%20in%20Zoysia%20species%20and%20the%20identification%20of%20putative%20interspecific%20hybrids%20using%20simple%20sequence%20repeat%20markers%20and%20inflorescence%20traits&amp;author=J.A.%20Kimball&amp;author=C.M.%20Zuleta&amp;author=K.E.%20Kenworthy&amp;author=V.G.%20Lehman&amp;author=K.R.%20Harris-Shultz&amp;author=S.R.%20Milla-Lewis&amp;publication_year=2013&amp;journal=Crop%20Sci.&amp;volume=53&amp;pages=285-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2012.04.0218" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2012.04.0218" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2012.04.0218"> </span></div> </div></div></div></div></div><div content-id="DSW006C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tanaka</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Tokunaga</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Muguerza</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2016</div>, <div class="article-title">Genetic structure and speciation of zoysiagrass ecotypes collected in Japan</div>, <div class="source">Crop Sci.</div> , <div class="volume">56</div>, <div class="fpage">818</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20structure%20and%20speciation%20of%20zoysiagrass%20ecotypes%20collected%20in%20Japan&amp;author=H.%20Tanaka&amp;author=R.%20Tokunaga&amp;author=M.%20Muguerza&amp;publication_year=2016&amp;journal=Crop%20Sci.&amp;volume=56&amp;pages=818-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2015.04.0249" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2015.04.0249" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2015.04.0249"> </span></div> </div></div></div></div></div><div content-id="DSW006C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Cai</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Yuyama</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Hirata</div> <div class="given-names">M.</div></div></div>
          <div class="year">2014</div>, In: <div class="person-group"><div class="name string-name"><div class="surname">Cai</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Yamada</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Kole</div> <div class="given-names">C.</div></div></div> (eds.) <div class="source">Genetics, Genomics and Breeding of Forage Crops</div> , <div class="publisher-name">CRC Press</div>, <div class="publisher-loc">Florida</div>, pp. <div class="fpage">168</div>–<div class="lpage">86</div>.</div></div></div></div><div content-id="DSW006C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Guo</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Ding</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Genetic linkage map construction and QTL mapping of salt tolerance traits in zoysiagrass (<em>Zoysia japonica</em>)</div>, <div class="source">PLoS One</div> , <div class="volume">9</div>, <div class="fpage">e107249</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20linkage%20map%20construction%20and%20QTL%20mapping%20of%20salt%20tolerance%20traits%20in%20zoysiagrass%20(Zoysia%20japonica)&amp;author=H.%20Guo&amp;author=W.%20Ding&amp;author=J.%20Chen&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e107249" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0107249" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0107249" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0107249"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25203715" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Singh</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Genovesi</div> <div class="given-names">A.D.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Sequence-tagged high-density genetic maps of <em>Zoysia japonica</em> provide insights into genome evolution in Chloridoideae</div>, <div class="source">Plant J.</div> , <div class="volume">82</div>, <div class="fpage">744</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence-tagged%20high-density%20genetic%20maps%20of%20Zoysia%20japonica%20provide%20insights%20into%20genome%20evolution%20in%20Chloridoideae&amp;author=F.%20Wang&amp;author=R.%20Singh&amp;author=A.D.%20Genovesi&amp;publication_year=2015&amp;journal=Plant%20J.&amp;volume=82&amp;pages=744-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2015.82.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2015.82.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2015.82.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25846381" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wei</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Du</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Gao</div> <div class="given-names">F.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Global transcriptome profiles of ‘Meyer’ zoysiagrass in response to cold stress</div>, <div class="source">PLoS One</div> , <div class="volume">10</div>, <div class="fpage">e0131153</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20transcriptome%20profiles%20of%20%E2%80%98Meyer%E2%80%99%20zoysiagrass%20in%20response%20to%20cold%20stress&amp;author=S.%20Wei&amp;author=Z.%20Du&amp;author=F.%20Gao&amp;publication_year=2015&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0131153" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0131153" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0131153" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0131153"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26115186" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Xie</div>
               <div class="given-names">Q.</div>
            </div>, <div class="name string-name"><div class="surname">Niu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">X.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title"><em>De novo</em> assembly of the Japanese lawngrass (<em>Zoysia japonica</em> Steud.) root transcriptome and identification of candidate unigenes related to early responses under salt stress</div>, <div class="source">Front. Plant Sci.</div> , <div class="volume">6</div>, <div class="fpage">610</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20the%20Japanese%20lawngrass%20(Zoysia%20japonica%20Steud.)%20root%20transcriptome%20and%20identification%20of%20candidate%20unigenes%20related%20to%20early%20responses%20under%20salt%20stress&amp;author=Q.%20Xie&amp;author=J.%20Niu&amp;author=X.%20Xu&amp;publication_year=2015&amp;journal=Front.%20Plant%20Sci.&amp;volume=6&amp;pages=610" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26347751" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ahn</div>
               <div class="given-names">J.H.</div>
            </div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">J.S.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title"><em>De novo</em> transcriptome analysis to identify anthocyanin biosynthesis genes responsible for tissue-specific pigmentation in zoysiagrass (<em>Zoysia japonica</em> Steud.)</div>, <div class="source">PLoS One</div> , <div class="volume">10</div>, <div class="fpage">e0124497</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20transcriptome%20analysis%20to%20identify%20anthocyanin%20biosynthesis%20genes%20responsible%20for%20tissue-specific%20pigmentation%20in%20zoysiagrass%20(Zoysia%20japonica%20Steud.)&amp;author=J.H.%20Ahn&amp;author=J.S.%20Kim&amp;author=S.%20Kim&amp;publication_year=2015&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0124497" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0124497" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0124497" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0124497"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25905914" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tanaka</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Muguerza</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2016</div>, <div class="article-title">The complete chloroplast genome sequence of <em>Zoysia matrella</em></div>, <div class="source">Crop Sci.</div> , <div class="pub-id-doi"><a href="https://doi.org/10.2135/cropsci2015.08.0517" class="link link-pub-id-doi" target="_blank">https://doi.org/10.2135/cropsci2015.08.0517</a></div>. </div></div></div></div><div content-id="DSW006C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Godden</div>
               <div class="given-names">G.T.</div>
            </div>, <div class="name string-name"><div class="surname">Jordon-Thaden</div> <div class="given-names">I.E.</div></div>, <div class="name string-name"><div class="surname">Chamala</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Making next-generation sequencing work for you: Approaches and practical considerations for marker development and phylogenetics</div>, <div class="source">Plant Ecol. Diversity</div> , <div class="volume">5</div>, <div class="fpage">427</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Making%20next-generation%20sequencing%20work%20for%20you%3A%20Approaches%20and%20practical%20considerations%20for%20marker%20development%20and%20phylogenetics&amp;author=G.T.%20Godden&amp;author=I.E.%20Jordon-Thaden&amp;author=S.%20Chamala&amp;publication_year=2012&amp;journal=Plant%20Ecol.%20Diversity&amp;volume=5&amp;pages=427-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/17550874.2012.745909" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F17550874.2012.745909" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F17550874.2012.745909"> </span></div> </div></div></div></div></div><div content-id="DSW006C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sims</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Sudbery</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Ilott</div> <div class="given-names">N.E.</div></div>, <div class="name string-name"><div class="surname">Heger</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ponting</div> <div class="given-names">C.P.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Sequencing depth and coverage: key considerations in genomic analyses</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="volume">15</div>, <div class="fpage">121</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20depth%20and%20coverage%3A%20key%20considerations%20in%20genomic%20analyses&amp;author=D.%20Sims&amp;author=I.%20Sudbery&amp;author=N.E.%20Ilott&amp;author=A.%20Heger&amp;author=C.P.%20Ponting&amp;publication_year=2014&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=15&amp;pages=121-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3642" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3642" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3642"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24434847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Paterson</div>
               <div class="given-names">A.H.</div>
            </div>, <div class="name string-name"><div class="surname">Freeling</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Sasaki</div> <div class="given-names">T.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">Grains of knowledge: Genomics of model cereals</div>, <div class="source">Genome Res.</div> , <div class="volume">15</div>, <div class="fpage">1643</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Grains%20of%20knowledge%3A%20Genomics%20of%20model%20cereals&amp;author=A.H.%20Paterson&amp;author=M.%20Freeling&amp;author=T.%20Sasaki&amp;publication_year=2005&amp;journal=Genome%20Res.&amp;volume=15&amp;pages=1643-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.3725905" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.3725905" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.3725905"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16339361" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Murashige</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Skoog</div> <div class="given-names">F.</div></div></div>
          <div class="year">1986</div>, <div class="article-title">A revised medium for rapid growth and bio assays with tobacco tissue cultures</div>, <div class="source">Physiol. Plant</div> , <div class="volume">15</div>, <div class="fpage">473</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20revised%20medium%20for%20rapid%20growth%20and%20bio%20assays%20with%20tobacco%20tissue%20cultures&amp;author=T.%20Murashige&amp;author=F.%20Skoog&amp;publication_year=1986&amp;journal=Physiol.%20Plant&amp;volume=15&amp;pages=473-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/ppl.1962.15.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fppl.1962.15.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fppl.1962.15.issue-3"> </span></div> </div></div></div></div></div><div content-id="DSW006C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ishigaki</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Gondo</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Ebina</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Suenaga</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Akashi</div> <div class="given-names">R.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Estimations of genome size in <em>Brachiaria</em> species</div>, <div class="source">Grassl. Sci.</div> , <div class="volume">56</div>, <div class="fpage">240</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimations%20of%20genome%20size%20in%20Brachiaria%20species&amp;author=G.%20Ishigaki&amp;author=T.%20Gondo&amp;author=M.%20Ebina&amp;author=K.%20Suenaga&amp;author=R.%20Akashi&amp;publication_year=2010&amp;journal=Grassl.%20Sci.&amp;volume=56&amp;pages=240-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/grs.2010.56.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fgrs.2010.56.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fgrs.2010.56.issue-4"> </span></div> </div></div></div></div></div><div content-id="DSW006C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Price</div>
               <div class="given-names">H.J.</div>
            </div>, <div class="name string-name"><div class="surname">Dillon</div> <div class="given-names">S.L.</div></div>, <div class="name string-name"><div class="surname">Hodnett</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Rooney</div> <div class="given-names">W.L.</div></div>, <div class="name string-name"><div class="surname">Ross</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Johnston</div> <div class="given-names">J.S.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">Genome evolution in the genus <em>Sorghum</em> (Poaceae)</div>, <div class="source">Ann. Bot.</div> , <div class="volume">95</div>, <div class="fpage">219</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20evolution%20in%20the%20genus%20Sorghum%20(Poaceae)&amp;author=H.J.%20Price&amp;author=S.L.%20Dillon&amp;author=G.%20Hodnett&amp;author=W.L.%20Rooney&amp;author=L.%20Ross&amp;author=J.S.%20Johnston&amp;publication_year=2005&amp;journal=Ann.%20Bot.&amp;volume=95&amp;pages=219-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/aob/mci015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmci015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Faob%2Fmci015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15596469" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yagi</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Kosugi</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Sequence analysis of the genome of carnation (<em>Dianthus caryophyllus</em> L.)</div>, <div class="source">DNA Res.</div> , <div class="volume">21</div>, <div class="fpage">231</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20analysis%20of%20the%20genome%20of%20carnation%20(Dianthus%20caryophyllus%20L.)&amp;author=M.%20Yagi&amp;author=S.%20Kosugi&amp;author=H.%20Hirakawa&amp;publication_year=2014&amp;journal=DNA%20Res.&amp;volume=21&amp;pages=231-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst053" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst053" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst053"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24344172" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yang</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Chockalingam</div> <div class="given-names">S.P.</div></div>, <div class="name string-name"><div class="surname">Aluru</div> <div class="given-names">S.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">Survey of error-correction methods for next-generation sequencing</div>, <div class="source">Brief Bioinform.</div> , <div class="volume">14</div>, <div class="fpage">56</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Survey%20of%20error-correction%20methods%20for%20next-generation%20sequencing&amp;author=X.%20Yang&amp;author=S.P.%20Chockalingam&amp;author=S.%20Aluru&amp;publication_year=2013&amp;journal=Brief%20Bioinform.&amp;volume=14&amp;pages=56-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/bbs015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbs015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2Fbbs015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22492192" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kajitani</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Toshimoto</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Noguchi</div> <div class="given-names">H.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Efficient <em>de novo</em> assembly of highly heterozygous genomes from whole-genome shotgun short reads</div>, <div class="source">Genome Res.</div> , <div class="volume">24</div>, <div class="fpage">1384</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20de%20novo%20assembly%20of%20highly%20heterozygous%20genomes%20from%20whole-genome%20shotgun%20short%20reads&amp;author=R.%20Kajitani&amp;author=K.%20Toshimoto&amp;author=H.%20Noguchi&amp;publication_year=2014&amp;journal=Genome%20Res.&amp;volume=24&amp;pages=1384-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.170720.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.170720.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.170720.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24755901" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>
          </div>
          <div class="year">2012</div>, <div class="article-title">Exploring single-sample SNP and INDEL calling with whole-genome <em>de novo</em> assembly</div>, <div class="source">Bioinformatics</div> , <div class="volume">28</div>, <div class="fpage">1838</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploring%20single-sample%20SNP%20and%20INDEL%20calling%20with%20whole-genome%20de%20novo%20assembly&amp;author=H.%20Li&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=1838-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts280" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts280" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbts280"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22569178" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gao</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Sung</div> <div class="given-names">W.K.</div></div>, <div class="name string-name"><div class="surname">Nagarajan</div> <div class="given-names">N.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Opera: reconstructing optimal genomic scaffolds with high-throughput paired-end sequences</div>, <div class="source">J. Comput. Biol.</div> , <div class="volume">18</div>, <div class="fpage">1681</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Opera%3A%20reconstructing%20optimal%20genomic%20scaffolds%20with%20high-throughput%20paired-end%20sequences&amp;author=S.%20Gao&amp;author=W.K.%20Sung&amp;author=N.%20Nagarajan&amp;publication_year=2011&amp;journal=J.%20Comput.%20Biol.&amp;volume=18&amp;pages=1681-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/cmb.2011.0170" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fcmb.2011.0170" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Fcmb.2011.0170"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21929371" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kosugi</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Tabata</div> <div class="given-names">S.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">GMcloser: closing gaps in assemblies accurately with a likelihood-based selection of contig or long-read alignments</div>, <div class="source">Bioinformatics</div> , <div class="volume">31</div>, <div class="fpage">3733</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GMcloser%3A%20closing%20gaps%20in%20assemblies%20accurately%20with%20a%20likelihood-based%20selection%20of%20contig%20or%20long-read%20alignments&amp;author=S.%20Kosugi&amp;author=H.%20Hirakawa&amp;author=S.%20Tabata&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=3733-41" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26261222" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Grabherr</div>
               <div class="given-names">M.G.</div>
            </div>, <div class="name string-name"><div class="surname">Haas</div> <div class="given-names">B.J.</div></div>, <div class="name string-name"><div class="surname">Yassour</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">29</div>, <div class="fpage">644</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Full-length%20transcriptome%20assembly%20from%20RNA-Seq%20data%20without%20a%20reference%20genome&amp;author=M.G.%20Grabherr&amp;author=B.J.%20Haas&amp;author=M.%20Yassour&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol.&amp;volume=29&amp;pages=644-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1883" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1883" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1883"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Parra</div>
               <div class="given-names">G.</div>
            </div>, <div class="name string-name"><div class="surname">Bradnam</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Korf</div> <div class="given-names">I.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes</div>, <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">1061</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CEGMA%3A%20a%20pipeline%20to%20accurately%20annotate%20core%20genes%20in%20eukaryotic%20genomes&amp;author=G.%20Parra&amp;author=K.%20Bradnam&amp;author=I.%20Korf&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=1061-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm071" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm071" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm071"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17332020" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Altschul</div>
               <div class="given-names">S.F.</div>
            </div>, <div class="name string-name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>, <div class="name string-name"><div class="surname">Schaffer</div> <div class="given-names">A.A.</div></div>et al.  </div>. <div class="year">1997</div>, <div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">25</div>, <div class="fpage">3389</div>–<div class="lpage">402</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Price</div>
               <div class="given-names">A.L.</div>
            </div>, <div class="name string-name"><div class="surname">Jones</div> <div class="given-names">N.C.</div></div>, <div class="name string-name"><div class="surname">Pevzner</div> <div class="given-names">P.A.</div></div></div>
          <div class="year">2005</div>, <div class="article-title"><em>De novo</em> identification of repeat families in large genomes</div>, <div class="source">Bioinformatics</div> , <div class="volume">21</div><div class="issue">(Suppl 1)</div>, <div class="fpage">i351</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20identification%20of%20repeat%20families%20in%20large%20genomes&amp;author=A.L.%20Price&amp;author=N.C.%20Jones&amp;author=P.A.%20Pevzner&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=i351-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti1018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti1018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti1018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15961478" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kofler</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Schlotterer</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Lelley</div> <div class="given-names">T.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">SciRoKo: a new tool for whole genome microsatellite search and investigation</div>, <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">1683</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SciRoKo%3A%20a%20new%20tool%20for%20whole%20genome%20microsatellite%20search%20and%20investigation&amp;author=R.%20Kofler&amp;author=C.%20Schlotterer&amp;author=T.%20Lelley&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=1683-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm157" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm157" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm157"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17463017" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lowe</div>
               <div class="given-names">T.M.</div>
            </div>, <div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div></div>
          <div class="year">1997</div>, <div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">25</div>, <div class="fpage">955</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence&amp;author=T.M.%20Lowe&amp;author=S.R.%20Eddy&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=955-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9023104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Eddy</div>
               <div class="given-names">S.R.</div>
            </div>
          </div>
          <div class="year">2009</div>, <div class="article-title">A new generation of homology search tools based on probabilistic inference</div>, <div class="source">Genome Inform.</div> , <div class="volume">23</div>, <div class="fpage">205</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20generation%20of%20homology%20search%20tools%20based%20on%20probabilistic%20inference&amp;author=S.R.%20Eddy&amp;publication_year=2009&amp;journal=Genome%20Inform.&amp;volume=23&amp;pages=205-11" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20180275" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Holt</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Yandell</div> <div class="given-names">M.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">MAKER2: an annotation pipeline and genome-database management tool for second-generation genome projects</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">12</div>, <div class="fpage">491</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAKER2%3A%20an%20annotation%20pipeline%20and%20genome-database%20management%20tool%20for%20second-generation%20genome%20projects&amp;author=C.%20Holt&amp;author=M.%20Yandell&amp;publication_year=2011&amp;journal=BMC%20Bioinformatics&amp;volume=12&amp;pages=491" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-12-491" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-12-491" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-12-491"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22192575" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bao</div>
               <div class="given-names">Z.</div>
            </div>, <div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">Automated <em>de novo</em> identification of repeat sequence families in sequenced genomes</div>, <div class="source">Genome Res.</div> , <div class="volume">12</div>, <div class="fpage">1269</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20de%20novo%20identification%20of%20repeat%20sequence%20families%20in%20sequenced%20genomes&amp;author=Z.%20Bao&amp;author=S.R.%20Eddy&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=1269-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.88502" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.88502" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.88502"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12176934" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stanke</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Waack</div> <div class="given-names">S.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">Gene prediction with a hidden Markov model and a new intron submodel</div>, <div class="source">Bioinformatics</div> , <div class="volume">19</div><div class="issue">(Suppl 2)</div>, <div class="fpage">ii215</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20prediction%20with%20a%20hidden%20Markov%20model%20and%20a%20new%20intron%20submodel&amp;author=M.%20Stanke&amp;author=S.%20Waack&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=ii215-25" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14534192" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Korf</div>
               <div class="given-names">I.</div>
            </div>
          </div>
          <div class="year">2004</div>, <div class="article-title">Gene finding in novel genomes</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">5</div>, <div class="fpage">59</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20finding%20in%20novel%20genomes&amp;author=I.%20Korf&amp;publication_year=2004&amp;journal=BMC%20Bioinformatics&amp;volume=5&amp;pages=59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-5-59" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-5-59" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-5-59"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15144565" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Trapnell</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Pachter</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">TopHat: discovering splice junctions with RNA-Seq</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1105</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TopHat%3A%20discovering%20splice%20junctions%20with%20RNA-Seq&amp;author=C.%20Trapnell&amp;author=L.%20Pachter&amp;author=S.L.%20Salzberg&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1105-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp120" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp120" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp120"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19289445" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Trapnell</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Williams</div> <div class="given-names">B.A.</div></div>, <div class="name string-name"><div class="surname">Pertea</div> <div class="given-names">G.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">28</div>, <div class="fpage">511</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcript%20assembly%20and%20quantification%20by%20RNA-Seq%20reveals%20unannotated%20transcripts%20and%20isoform%20switching%20during%20cell%20differentiation&amp;author=C.%20Trapnell&amp;author=B.A.%20Williams&amp;author=G.%20Pertea&amp;publication_year=2010&amp;journal=Nat.%20Biotechnol.&amp;volume=28&amp;pages=511-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1621" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1621" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1621"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20436464" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Slater</div>
               <div class="given-names">G.S.</div>
            </div>, <div class="name string-name"><div class="surname">Birney</div> <div class="given-names">E.</div></div></div>
          <div class="year">2005</div>, <div class="article-title">Automated generation of heuristics for biological sequence comparison</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">6</div>, <div class="fpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20generation%20of%20heuristics%20for%20biological%20sequence%20comparison&amp;author=G.S.%20Slater&amp;author=E.%20Birney&amp;publication_year=2005&amp;journal=BMC%20Bioinformatics&amp;volume=6&amp;pages=31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-6-31" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-6-31" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-6-31"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15713233" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tatusov</div>
               <div class="given-names">R.L.</div>
            </div>, <div class="name string-name"><div class="surname">Fedorova</div> <div class="given-names">N.D.</div></div>, <div class="name string-name"><div class="surname">Jackson</div> <div class="given-names">J.D.</div></div>et al.  </div>. <div class="year">2003</div>, <div class="article-title">The COG database: an updated version includes eukaryotes</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">4</div>, <div class="fpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20COG%20database%3A%20an%20updated%20version%20includes%20eukaryotes&amp;author=R.L.%20Tatusov&amp;author=N.D.%20Fedorova&amp;author=J.D.%20Jackson&amp;publication_year=2003&amp;journal=BMC%20Bioinformatics&amp;volume=4&amp;pages=41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-4-41" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-4-41" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-4-41"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12969510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Quevillon</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Silventoinen</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Pillai</div> <div class="given-names">S.</div></div>et al.  </div>. <div class="year">2005</div>, <div class="article-title">InterProScan: protein domains identifier</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">33</div>, <div class="fpage">W116</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterProScan%3A%20protein%20domains%20identifier&amp;author=E.%20Quevillon&amp;author=V.%20Silventoinen&amp;author=S.%20Pillai&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=W116-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki442" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki442"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15980438" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="collab">The Gene Ontology Consortium</div>. <div class="year">2000</div>, <div class="article-title">Gene ontology: tool for the unification of biology</div>, <div class="source">Nat. Genet.</div> , <div class="volume">25</div>, <div class="fpage">25</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/75556" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F75556" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F75556"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10802651" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ogata</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Goto</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sato</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Fujibuchi</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Bono</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div></div>
          <div class="year">1999</div>, <div class="article-title">KEGG: Kyoto encyclopedia of genes and genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">27</div>, <div class="fpage">29</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KEGG%3A%20Kyoto%20encyclopedia%20of%20genes%20and%20genomes&amp;author=H.%20Ogata&amp;author=S.%20Goto&amp;author=K.%20Sato&amp;author=W.%20Fujibuchi&amp;author=H.%20Bono&amp;author=M.%20Kanehisa&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=29-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.1.29" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.1.29" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.1.29"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9847135" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fu</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Niu</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">W.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">CD-HIT: accelerated for clustering the next-generation sequencing data</div>, <div class="source">Bioinformatics</div> , <div class="volume">28</div>, <div class="fpage">3150</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CD-HIT%3A%20accelerated%20for%20clustering%20the%20next-generation%20sequencing%20data&amp;author=L.%20Fu&amp;author=B.%20Niu&amp;author=Z.%20Zhu&amp;author=S.%20Wu&amp;author=W.%20Li&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=3150-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts565" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts565" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbts565"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23060610" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Langmead</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pop</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</div>, <div class="source">Genome Biol.</div> , <div class="volume">10</div>, <div class="fpage">R25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultrafast%20and%20memory-efficient%20alignment%20of%20short%20DNA%20sequences%20to%20the%20human%20genome&amp;author=B.%20Langmead&amp;author=C.%20Trapnell&amp;author=M.%20Pop&amp;author=S.L.%20Salzberg&amp;publication_year=2009&amp;journal=Genome%20Biol.&amp;volume=10&amp;pages=R25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-3-r25" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-3-r25" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-3-r25"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19261174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Handsaker</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wysoker</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">The Sequence Alignment/Map format and SAMtools</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2078</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Sequence%20Alignment%2FMap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gupta</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">B.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">Mechanism of salinity tolerance in plants: physiological, biochemical, and molecular characterization</div>, <div class="source">Int. J. Genomics</div> , <div class="volume">2014</div>, <div class="fpage">701596</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanism%20of%20salinity%20tolerance%20in%20plants%3A%20physiological%2C%20biochemical%2C%20and%20molecular%20characterization&amp;author=B.%20Gupta&amp;author=B.%20Huang&amp;publication_year=2014&amp;journal=Int.%20J.%20Genomics&amp;volume=2014&amp;pages=701596" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1155/2014/701596" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1155%2F2014%2F701596" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1155%2F2014%2F701596"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24804192" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Christianson</div>
               <div class="given-names">J.A.</div>
            </div>, <div class="name string-name"><div class="surname">Wilson</div> <div class="given-names">I.W.</div></div>, <div class="name string-name"><div class="surname">Llewellyn</div> <div class="given-names">D.J.</div></div>, <div class="name string-name"><div class="surname">Dennis</div> <div class="given-names">E.S.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">The low-oxygen induced NAC domain transcription factor ANAC102 affects viability of <em>Arabidopsis thaliana</em> seeds following low-oxygen treatment</div>, <div class="source">Plant Physiol.</div> , <div class="volume">149</div>, <div class="fpage">1724</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20low-oxygen%20induced%20NAC%20domain%20transcription%20factor%20ANAC102%20affects%20viability%20of%20Arabidopsis%20thaliana%20seeds%20following%20low-oxygen%20treatment&amp;author=J.A.%20Christianson&amp;author=I.W.%20Wilson&amp;author=D.J.%20Llewellyn&amp;author=E.S.%20Dennis&amp;publication_year=2009&amp;journal=Plant%20Physiol.&amp;volume=149&amp;pages=1724-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.108.131912" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.108.131912" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.108.131912"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19176720" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gangappa</div>
               <div class="given-names">S.N.</div>
            </div>, <div class="name string-name"><div class="surname">Botto</div> <div class="given-names">J.F.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">The BBX family of plant transcription factors</div>, <div class="source">Trends Plant Sci.</div> , <div class="volume">19</div>, <div class="fpage">460</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20BBX%20family%20of%20plant%20transcription%20factors&amp;author=S.N.%20Gangappa&amp;author=J.F.%20Botto&amp;publication_year=2014&amp;journal=Trends%20Plant%20Sci.&amp;volume=19&amp;pages=460-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2014.01.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2014.01.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2014.01.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24582145" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lippuner</div>
               <div class="given-names">V.</div>
            </div>, <div class="name string-name"><div class="surname">Cyert</div> <div class="given-names">M.S.</div></div>, <div class="name string-name"><div class="surname">Gasser</div> <div class="given-names">C.S.</div></div></div>
          <div class="year">1996</div>, <div class="article-title">Two classes of plant cDNA clones differentially complement yeast calcineurin mutants and increase salt tolerance of wild-type yeast</div>, <div class="source">J. Biol. Chem.</div> , <div class="volume">271</div>, <div class="fpage">12859</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20classes%20of%20plant%20cDNA%20clones%20differentially%20complement%20yeast%20calcineurin%20mutants%20and%20increase%20salt%20tolerance%20of%20wild-type%20yeast&amp;author=V.%20Lippuner&amp;author=M.S.%20Cyert&amp;author=C.S.%20Gasser&amp;publication_year=1996&amp;journal=J.%20Biol.%20Chem.&amp;volume=271&amp;pages=12859-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.271.22.12859" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.271.22.12859" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.271.22.12859"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8662738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nagaoka</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Takano</div> <div class="given-names">T.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">Salt tolerance-related protein STO binds to a Myb transcription factor homologue and confers salt tolerance in <em>Arabidopsis</em></div>, <div class="source">J. Exp. Bot.</div> , <div class="volume">54</div>, <div class="fpage">2231</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Salt%20tolerance-related%20protein%20STO%20binds%20to%20a%20Myb%20transcription%20factor%20homologue%20and%20confers%20salt%20tolerance%20in%20Arabidopsis&amp;author=S.%20Nagaoka&amp;author=T.%20Takano&amp;publication_year=2003&amp;journal=J.%20Exp.%20Bot.&amp;volume=54&amp;pages=2231-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erg241" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferg241" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferg241"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12909688" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Qian</div>
               <div class="given-names">Y.L.</div>
            </div>, <div class="name string-name"><div class="surname">Engelke</div> <div class="given-names">M.C.</div></div>, <div class="name string-name"><div class="surname">Foster</div> <div class="given-names">M.J.V.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Salinity effects on zoysiagrass cultivars and experimental lines</div>, <div class="source">Crop Sci.</div> , <div class="volume">40</div>, <div class="fpage">488</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Salinity%20effects%20on%20zoysiagrass%20cultivars%20and%20experimental%20lines&amp;author=Y.L.%20Qian&amp;author=M.C.%20Engelke&amp;author=M.J.V.%20Foster&amp;publication_year=2000&amp;journal=Crop%20Sci.&amp;volume=40&amp;pages=488-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2135/cropsci2000.402488x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2135%2Fcropsci2000.402488x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2135%2Fcropsci2000.402488x"> </span></div> </div></div></div></div></div><div content-id="DSW006C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jakoby</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Weisshaar</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Dröge-Laser</div> <div class="given-names">W.</div></div>et al.  </div>. <div class="year">2002</div>, <div class="article-title">bZIP transcription factors in <em>Arabidopsis</em></div>, <div class="source">Trends Plant Sci.</div> , <div class="volume">7</div>, <div class="fpage">106</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=bZIP%20transcription%20factors%20in%20Arabidopsis&amp;author=M.%20Jakoby&amp;author=B.%20Weisshaar&amp;author=W.%20Dr%C3%B6ge-Laser&amp;publication_year=2002&amp;journal=Trends%20Plant%20Sci.&amp;volume=7&amp;pages=106-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1360-1385(01)02223-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1360-1385(01)02223-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1360-1385(01)02223-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11906833" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">X.</div>
            </div>, <div class="name string-name"><div class="surname">Chu</div> <div class="given-names">Z.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">Genome-wide evolutionary characterization and analysis of bZIP transcription factors and their expression profiles in response to multiple abiotic stresses in <em>Brachypodium distachyon</em></div>, <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">227</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20evolutionary%20characterization%20and%20analysis%20of%20bZIP%20transcription%20factors%20and%20their%20expression%20profiles%20in%20response%20to%20multiple%20abiotic%20stresses%20in%20Brachypodium%20distachyon&amp;author=X.%20Liu&amp;author=Z.%20Chu&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=227" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1457-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1457-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1457-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25887221" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yamamoto</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Takano</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Tanaka</div> <div class="given-names">K.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Comprehensive analysis of transcriptome response to salinity stress in the halophytic turf grass <em>Sporobolus virginicus</em></div>, <div class="source">Front. Plant Sci.</div> , <div class="volume">6</div>, <div class="fpage">241</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20analysis%20of%20transcriptome%20response%20to%20salinity%20stress%20in%20the%20halophytic%20turf%20grass%20Sporobolus%20virginicus&amp;author=N.%20Yamamoto&amp;author=T.%20Takano&amp;author=K.%20Tanaka&amp;publication_year=2015&amp;journal=Front.%20Plant%20Sci.&amp;volume=6&amp;pages=241" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fpls.2015.00241" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2015.00241" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffpls.2015.00241"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25954282" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jin</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kong</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Gao</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Luo</div> <div class="given-names">J.</div></div></div>
          <div class="year">2014</div>, <div class="article-title">PlantTFDB 3.0: a portal for the functional and evolutionary study of plant transcription factors</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">42</div>, <div class="fpage">D1182</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PlantTFDB%203.0%3A%20a%20portal%20for%20the%20functional%20and%20evolutionary%20study%20of%20plant%20transcription%20factors&amp;author=J.%20Jin&amp;author=H.%20Zhang&amp;author=L.%20Kong&amp;author=G.%20Gao&amp;author=J.%20Luo&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res.&amp;volume=42&amp;pages=D1182-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24174544" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Manetas</div>
               <div class="given-names">Y.</div>
            </div>
          </div>
          <div class="year">2006</div>, <div class="article-title">Why some leaves are anthocyanic and why most anthocyanic leaves are red?</div>, <div class="source">Flora</div> , <div class="volume">201</div>, <div class="fpage">163</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Why%20some%20leaves%20are%20anthocyanic%20and%20why%20most%20anthocyanic%20leaves%20are%20red%3F&amp;author=Y.%20Manetas&amp;publication_year=2006&amp;journal=Flora&amp;volume=201&amp;pages=163-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.flora.2005.06.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.flora.2005.06.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.flora.2005.06.010"> </span></div> </div></div></div></div></div><div content-id="DSW006C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Xu</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Cai</div> <div class="given-names">R.</div></div>et al.  </div>. <div class="year">2008</div>, <div class="article-title">Molecular cloning and function analysis of an anthocyanidin synthase gene from <em>Ginkgo biloba</em>, and its expression in abiotic stress responses</div>, <div class="source">Mol. Cells</div> , <div class="volume">26</div>, <div class="fpage">536</div>–<div class="lpage">47</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20cloning%20and%20function%20analysis%20of%20an%20anthocyanidin%20synthase%20gene%20from%20Ginkgo%20biloba%2C%20and%20its%20expression%20in%20abiotic%20stress%20responses&amp;author=F.%20Xu&amp;author=H.%20Cheng&amp;author=R.%20Cai&amp;publication_year=2008&amp;journal=Mol.%20Cells&amp;volume=26&amp;pages=536-47" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18779661" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Choi</div>
               <div class="given-names">J.S.</div>
            </div>
          </div>
          <div class="year">2010</div>, <div class="article-title">Morphological characteristics of medium-leaf type zoysiagrasses (<em>Zoysia</em> spp.) and their classification using RAPDs</div>, <div class="source">Kor. Turfgrass Sci.</div> , <div class="volume">24</div>, <div class="fpage">88</div>–<div class="lpage">96</div>.</div></div></div></div><div content-id="DSW006C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yaneshita</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Nagasawa</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Engelke</div> <div class="given-names">M.C.</div></div>, <div class="name string-name"><div class="surname">Sasakuma</div> <div class="given-names">T.</div></div></div>
          <div class="year">1997</div>, <div class="article-title">Genetic variation and interspecific hybridization among natural populations of zoysiagrasses detected by RFLP analyses of chloroplast and nuclear DNA</div>, <div class="source">Genes Genet. Syst.</div> , <div class="volume">72</div>, <div class="fpage">173</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20variation%20and%20interspecific%20hybridization%20among%20natural%20populations%20of%20zoysiagrasses%20detected%20by%20RFLP%20analyses%20of%20chloroplast%20and%20nuclear%20DNA&amp;author=M.%20Yaneshita&amp;author=R.%20Nagasawa&amp;author=M.C.%20Engelke&amp;author=T.%20Sasakuma&amp;publication_year=1997&amp;journal=Genes%20Genet.%20Syst.&amp;volume=72&amp;pages=173-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1266/ggs.72.173" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1266%2Fggs.72.173" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1266%2Fggs.72.173"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9418258" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSW006C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tsuruta</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Hashiguchi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Ebina</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2005</div>, <div class="article-title">Development and characterization of simple sequence repeat markers in <em>Zoysia japonica</em> Steud</div>, <div class="source">Grassl. Sci.</div> , <div class="volume">51</div>, <div class="fpage">249</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20characterization%20of%20simple%20sequence%20repeat%20markers%20in%20Zoysia%20japonica%20Steud&amp;author=S.%20Tsuruta&amp;author=M.%20Hashiguchi&amp;author=M.%20Ebina&amp;publication_year=2005&amp;journal=Grassl.%20Sci.&amp;volume=51&amp;pages=249-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/grs.2005.51.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fgrs.2005.51.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fgrs.2005.51.issue-3"> </span></div> </div></div></div></div></div><div content-id="DSW006C61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSW006C61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rieseberg</div>
               <div class="given-names">L.H.</div>
            </div>, <div class="name string-name"><div class="surname">Willis</div> <div class="given-names">J.H.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">Plant speciation</div>, <div class="source">Science</div> , <div class="volume">317</div>, <div class="fpage">910</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20speciation&amp;author=L.H.%20Rieseberg&amp;author=J.H.%20Willis&amp;publication_year=2007&amp;journal=Science&amp;volume=317&amp;pages=910-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1137729" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1137729" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1137729"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17702935" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=1745492&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsw006/2/dsw006_Supplementary_Data.zip?Expires=2147483647&amp;Signature=2jXN7GFW3xO4BGFSZS5a7WEhceSJsswHAhwaOAr-3KEOZHRZvcw-NAg1-HQvfQVyFsYv2hnrUwbKBfk2~5HVycKnKpc-xVXVM2lJBLehx0c57keK6e4fHxO7B7FECa23vm6ycFepULkN9aV2qXF131XOyNbu1uafu7Tl-FOyaGVpx4gRnfFh0h6U8tR2pimDBtebNOWBmVx2AUh-MMfyp4Yobje4BN1-jenvw9LOvQABzbj7HgTK2Nmodl0SqIpFzgf5-iepFSitZ434rrkbgM0~mYR6rtoVZTaF6BbeYx5TYlcd2PAx2EOfiAend4SZzBowBgMFbZTB2prfYuKGow__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,071</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000375088800009&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">13</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw006"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="1745492"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000375088800009&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f2%2f171%2f1745492">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000375088800009&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (13)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F2%2F171%2F1745492" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/1745492'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Sequencing and comparative analyses of the genomes of zoysiagrasses" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw006");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f171%2f1745492">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

