

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/16/2/115/510928","@type":"ScholarlyArticle","name":"Genome Sequence of the Lager Brewing Yeast, an Interspecies Hybrid","datePublished":"2009-03-04","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/16/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2009-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsp003","keywords":["<italic>Saccharomyces pastorianus</italic>","beer","genome","interspecies hybrid","larger yeast"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/16/2/115/510928","author":[{"name":"Nakao, Yoshihiro","affiliation":"1  R&amp;D Planning Division ,  Suntory Limited ,  1-1-1, Wakayamadai, Shimamoto-cho, Mishima-gun, Osaka 618-8503 ,  Japan","@type":"Person"},{"name":"Kanamori, Takeshi","affiliation":"1  R&amp;D Planning Division ,  Suntory Limited ,  1-1-1, Wakayamadai, Shimamoto-cho, Mishima-gun, Osaka 618-8503 ,  Japan","@type":"Person"},{"name":"Itoh, Takehiko","affiliation":"2  Mitsubishi Research Institute ,  Advanced Science and Technology Group ,  3-6 Otemachi 2-chome Chiyoda-ku, Tokyo 100-8141 ,  Japan","@type":"Person"},{"name":"Kodama, Yukiko","affiliation":"1  R&amp;D Planning Division ,  Suntory Limited ,  1-1-1, Wakayamadai, Shimamoto-cho, Mishima-gun, Osaka 618-8503 ,  Japan","@type":"Person"},{"name":"Rainieri, Sandra","affiliation":"1  R&amp;D Planning Division ,  Suntory Limited ,  1-1-1, Wakayamadai, Shimamoto-cho, Mishima-gun, Osaka 618-8503 ,  Japan","@type":"Person"},{"name":"Nakamura, Norihisa","affiliation":"1  R&amp;D Planning Division ,  Suntory Limited ,  1-1-1, Wakayamadai, Shimamoto-cho, Mishima-gun, Osaka 618-8503 ,  Japan","@type":"Person"},{"name":"Shimonaga, Tomoko","affiliation":"1  R&amp;D Planning Division ,  Suntory Limited ,  1-1-1, Wakayamadai, Shimamoto-cho, Mishima-gun, Osaka 618-8503 ,  Japan","@type":"Person"},{"name":"Hattori, Masahira","affiliation":"3  Department of Computational Biology ,  Graduate School of Frontier Sciences, University of Tokyo ,  Kashiwa-no-ha 5-1-5, Kashiwa, Chiba 277-8561 ,  Japan","@type":"Person"},{"name":"Ashikari, Toshihiko","affiliation":"4  Florigene Pty, Ltd ,  1 Park Drive, Bundoora, VIC 3083 ,  Australia","@type":"Person"}],"description":"Abstract.  This work presents the genome sequencing of the lager brewing yeast (Saccharomyces pastorianus) Weihenstephan 34/70, a strain widely used in lager be","pageStart":"115","pageEnd":"129","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/2/1/m_cover.gif?Expires=2147483647&Signature=spYD3Y1ECWwq5jfgvLwgtJGBnOHAN-nZXtWf5jX0Idx1cbrJRV4FKB28EY3pe17T0gsGMxo5dA8Tel6c2xNGAqa1byNOzqnWfnM8ohlcCfLHlrfGOEn2zyCW-15VbU7SOoLTo3OmdKcdJ19GrDbXCyAEcU~VJnaBCNGoOwBtbTnh6-S6KzBaPxjqj1HfbfKuNRe7H3r-0uFeA~Y3Ph79tw8QP6q4D0FiiXdjGGWYTNp3-Ga~m-PPAcOogXTotI1YFUthxQJgvIKyhC8JLbL3Znx728-CpY2KbMeFQlltwFxSw7d-mO5IOuvD7Wc91P7OLYCdnpnsyiP0k-TqunqvaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome Sequence of the Lager Brewing Yeast, an Interspecies Hybrid","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/2/1/m_cover.gif?Expires=2147483647&Signature=spYD3Y1ECWwq5jfgvLwgtJGBnOHAN-nZXtWf5jX0Idx1cbrJRV4FKB28EY3pe17T0gsGMxo5dA8Tel6c2xNGAqa1byNOzqnWfnM8ohlcCfLHlrfGOEn2zyCW-15VbU7SOoLTo3OmdKcdJ19GrDbXCyAEcU~VJnaBCNGoOwBtbTnh6-S6KzBaPxjqj1HfbfKuNRe7H3r-0uFeA~Y3Ph79tw8QP6q4D0FiiXdjGGWYTNp3-Ga~m-PPAcOogXTotI1YFUthxQJgvIKyhC8JLbL3Znx728-CpY2KbMeFQlltwFxSw7d-mO5IOuvD7Wc91P7OLYCdnpnsyiP0k-TqunqvaA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome Sequence of the Lager Brewing Yeast, an Interspecies Hybrid" />
    <meta property="og:description" content="Abstract.  This work presents the genome sequencing of the lager brewing yeast (Saccharomyces pastorianus) Weihenstephan 34/70, a strain widely used in lager be" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/16/2/115/510928" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2009-03-04" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Nakao, Yoshihiro" /><meta name="citation_author_institution" content="R&amp;D Planning Division, Suntory Limited, 1-1-1, Wakayamadai, Shimamoto-cho, Mishima-gun, Osaka 618-8503, Japan" /><meta name="citation_author" content="Kanamori, Takeshi" /><meta name="citation_author_institution" content="R&amp;D Planning Division, Suntory Limited, 1-1-1, Wakayamadai, Shimamoto-cho, Mishima-gun, Osaka 618-8503, Japan" /><meta name="citation_author" content="Itoh, Takehiko" /><meta name="citation_author_institution" content="Mitsubishi Research Institute, Advanced Science and Technology Group, 3-6 Otemachi 2-chome Chiyoda-ku, Tokyo 100-8141, Japan" /><meta name="citation_author" content="Kodama, Yukiko" /><meta name="citation_author_institution" content="R&amp;D Planning Division, Suntory Limited, 1-1-1, Wakayamadai, Shimamoto-cho, Mishima-gun, Osaka 618-8503, Japan" /><meta name="citation_author" content="Rainieri, Sandra" /><meta name="citation_author_institution" content="R&amp;D Planning Division, Suntory Limited, 1-1-1, Wakayamadai, Shimamoto-cho, Mishima-gun, Osaka 618-8503, Japan" /><meta name="citation_author" content="Nakamura, Norihisa" /><meta name="citation_author_institution" content="R&amp;D Planning Division, Suntory Limited, 1-1-1, Wakayamadai, Shimamoto-cho, Mishima-gun, Osaka 618-8503, Japan" /><meta name="citation_author" content="Shimonaga, Tomoko" /><meta name="citation_author_institution" content="R&amp;D Planning Division, Suntory Limited, 1-1-1, Wakayamadai, Shimamoto-cho, Mishima-gun, Osaka 618-8503, Japan" /><meta name="citation_author" content="Hattori, Masahira" /><meta name="citation_author_institution" content="Department of Computational Biology, Graduate School of Frontier Sciences, University of Tokyo, Kashiwa-no-ha 5-1-5, Kashiwa, Chiba 277-8561, Japan" /><meta name="citation_author" content="Ashikari, Toshihiko" /><meta name="citation_author_institution" content="Florigene Pty, Ltd, 1 Park Drive, Bundoora, VIC 3083, Australia" /><meta name="citation_title" content="Genome Sequence of the Lager Brewing Yeast, an Interspecies Hybrid" /><meta name="citation_firstpage" content="115" /><meta name="citation_lastpage" content="129" /><meta name="citation_doi" content="10.1093/dnares/dsp003" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="16" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2009/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The barth report hops 2006/2007;  citation_url=http://www.barthhaasgroup.com/cmsdk/content/bhg/barth_report.htm;  citation_year=2007; " /><meta name="citation_reference" content="citation_title=Investigations on the physiology and morphology of budding yeast. XIII. New studies of bottom fermenting brewers yeasts; C. R. Trav. Lab. Carlsberg.;  citation_year=1908;  citation_volume=7;  citation_pages=179-217; " /><meta name="citation_reference" content="citation_text=Chapter 44; citation_publisher=Elsevier, Amsterdam; The Yeasts, a Taxonomic Study: Saccharomyces Myen ex Reessm;  citation_year=1998;  citation_pages=358-371; " /><meta name="citation_reference" content="citation_title=Three newly delimited species of Saccharomyces sensu stricto; Antonie Van Leeuwenhoek;  citation_year=1987;  citation_volume=53;  citation_pages=77-84; " /><meta name="citation_reference" content="citation_title=Deoxyribonucleic acid relatedness among species of the genus Saccharomyces sensu stricto; Int. J. Syst. Bacteriol.;  citation_year=1985;  citation_volume=35;  citation_pages=508-511; " /><meta name="citation_reference" content="citation_title=Co-existence of two types of chromosome in the bottom fermenting yeast, Saccharomyces pastorianus; Yeast;  citation_year=1998;  citation_volume=14;  citation_pages=923-933; " /><meta name="citation_reference" content="citation_title=Chromosomal structures of bottom fermenting yeasts; Syst. Appl. Microbiol.;  citation_year=1999;  citation_volume=22;  citation_pages=341-353; " /><meta name="citation_reference" content="citation_title=Transfer of chromosome III during kar mediated cytoduction in yeast; Carlsberg Res. Commun.;  citation_year=1980;  citation_volume=45;  citation_pages=113-117; " /><meta name="citation_reference" content="citation_title=Analysis of chromosome V and the ILV1 gene from Saccharomyces carlsbergensis; Carlsberg Res. Commun.;  citation_year=1986;  citation_volume=51;  citation_pages=309-326; " /><meta name="citation_reference" content="citation_title=Genetics of brewing yeasts;  citation_edition=2nd Ed.; citation_publisher=Academic Press, London; The Yeasts;  citation_year=1995;  citation_volume=Vol. 6;  citation_pages=223-254; " /><meta name="citation_reference" content="citation_title=Nucleotide sequences of alcohol acetyltransferase genes from lager brewing yeast Saccharomyces carlsbergensis; Yeast;  citation_year=1996;  citation_volume=12;  citation_pages=593-598; " /><meta name="citation_reference" content="citation_title=Diversity of the HO gene encoding an endonuclease for mating-type conversion in the bottom fermenting yeast Saccharomyces pastorianus; Yeast;  citation_year=2000;  citation_volume=16;  citation_pages=1335-1343; " /><meta name="citation_reference" content="citation_title=Isolation and characterization of a gene specific to lager brewing yeast that encodes a branched-chain amino acid permease; Appl. Environ. Microbiol.;  citation_year=2001;  citation_volume=67;  citation_pages=3455-3462; " /><meta name="citation_reference" content="citation_title=Saccharomyces carlsbergensis contains two functional MET2 alleles similar to homologues from S. cerevisiae and S. monacensis; Gene;  citation_year=1994;  citation_volume=140;  citation_pages=33-40; " /><meta name="citation_reference" content="citation_title=Diversity in organization and the origin of gene orders in the mitochondrial DNA molecules of the genus Saccharomyces; Mol. Biol. Evol.;  citation_year=2000;  citation_volume=17;  citation_pages=1833-1841; " /><meta name="citation_reference" content="citation_text=Chapter 4; citation_publisher=Blackwell Science Ltd, Oxford; Brewing Yeast and Fermentation: Brewing yeast;  citation_year=2001;  citation_pages=143-259; " /><meta name="citation_reference" content="citation_title=Factors influence maltotriose utilization during brewery wort fermentations; J. Am. Soc. Brew. Chem.;  citation_year=1994;  citation_volume=52;  citation_pages=41-47; " /><meta name="citation_reference" content="citation_title=Novel differences between ale and lager brewing yeasts; Proc. Congr. Eur. Brew. Conv.;  citation_year=1993;  citation_volume=24;  citation_pages=267-274; " /><meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using phred. I. Accuracy assessment; Genome Res.;  citation_year=1998;  citation_volume=8;  citation_pages=175-185; " /><meta name="citation_reference" content="citation_title=PCAP: a whole-genome assembly program; Genome Res.;  citation_year=2003;  citation_volume=13;  citation_pages=2164-2170; " /><meta name="citation_reference" content="citation_title=Whole-genome shotgun optical mapping of Rhodospirillum rubrum; Appl. Environ. Microbiol.;  citation_year=2005;  citation_volume=71;  citation_pages=5511-5522; " /><meta name="citation_reference" content="citation_title=Phylogenetic analysis of the Saccharomyces cerevisiae group based on polymorphisms of rDNA spacer sequences; Int. J. Syst. Bacteriol.;  citation_year=1998;  citation_volume=48;  citation_issue=Pt 1;  citation_pages=295-303; " /><meta name="citation_reference" content="citation_title=Genome-scale approaches to resolving incongruence in molecular phylogenies; Nature;  citation_year=2003;  citation_volume=425;  citation_pages=798-804; " /><meta name="citation_reference" content="citation_title=Two subgroups within the Saccharomyces bayanus species evidenced by PCR amplification and restriction polymorphism of the non-transcribed spacer2 in the ribosomal DNA unit; System. Appl. Microbiol.;  citation_year=1997;  citation_volume=20;  citation_pages=286-294; " /><meta name="citation_reference" content="citation_title=Two subgroups within the Saccharomyces bayanus species evidenced by PCR amplification and restriction polymorphism of the non-transcribed spacer2 in the ribosomal DNA unit; System. Appl. Microbiol.;  citation_year=1997;  citation_volume=20;  citation_pages=286-294; " /><meta name="citation_reference" content="citation_title=Sequencing and comparison of yeast species to identify genes and regulatory elements; Nature;  citation_year=2003;  citation_volume=423;  citation_pages=241-254; " /><meta name="citation_reference" content="citation_title=Quest for wine yeasts—An old story revisited; J. Ind. Microbiol. Biotechnol.;  citation_year=1996;  citation_volume=17;  citation_pages=303-313; " /><meta name="citation_reference" content="citation_title=Genome sequencing and comparative analysis of Saccharomyces cerevisiae strain YJM789; Proc. Natl Acad. Sci. USA;  citation_year=2007;  citation_volume=104;  citation_pages=12825-12830; " /><meta name="citation_reference" content="citation_title=Proof and evolutionary analysis of ancient genome duplication in the yeast Saccharomyces cerevisiae; Nature;  citation_year=2004;  citation_volume=428;  citation_pages=617-624; " /><meta name="citation_reference" content="citation_title=Different types of recombination events are controlled by the RAD1 and RAD52 genes of Saccharomyces cerevisiae; Genetics;  citation_year=1988;  citation_volume=120;  citation_pages=367-377; " /><meta name="citation_reference" content="citation_title=Recombination between repeated genes in microorganisms; Annu. Rev. Genet.;  citation_year=1988;  citation_volume=22;  citation_pages=147-168; " /><meta name="citation_reference" content="citation_title=Allelic and ectopic recombination between Ty elements in yeast; Genetics;  citation_year=1988;  citation_volume=119;  citation_pages=549-559; " /><meta name="citation_reference" content="citation_title=Dual roles for DNA sequence identity and the mismatch repair system in the regulation of mitotic crossing-over in yeast; Proc. Natl Acad. Sci. USA;  citation_year=1997;  citation_volume=94;  citation_pages=9757-9762; " /><meta name="citation_reference" content="citation_title=Double-strand breaks associated with repetitive DNA can reshape the genome; Proc. Natl Acad. Sci. USA;  citation_year=2008;  citation_volume=105;  citation_pages=11845-11850; " /><meta name="citation_reference" content="citation_publisher=Springer-Verlag, Heidelberg; Top. Curr. Genet.: Lager brewing yeast;  citation_year=2006;  citation_volume=Vol. 15;  citation_pages=145-164; " /><meta name="citation_reference" content="citation_title=Uniparental loss of ribosomal DNA in the allotetraploid grass Zingeria trichopoda (2n equals 8); Genome;  citation_year=2003;  citation_volume=46;  citation_pages=156-163; " /><meta name="citation_reference" content="citation_title=Evolution of mitochondrial DNA in yeast: gene order and structural organization of the mitochondrial genome of Saccharomyces uvarum; Curr. Genet.;  citation_year=1998;  citation_volume=33;  citation_pages=52-59; " /><meta name="citation_reference" content="citation_title=Functional profiling of the Saccharomyces cerevisiae genome; Nature;  citation_year=2002;  citation_volume=418;  citation_pages=387-391; " /><meta name="citation_reference" content="citation_title=Improvement of maltose fermentation efficiency: constitutive expression of MAL genes in brewing yeasts; J. Am. Soc. Brew. Chem.;  citation_year=1995;  citation_volume=53;  citation_pages=24-29; " /><meta name="citation_reference" content="citation_title=Maltotriose utilization in lager yeast strains: MTT1 encodes a maltotriose transporter; Yeast;  citation_year=2005;  citation_volume=22;  citation_pages=775-788; " /><meta name="citation_reference" content="citation_title=Multiple alpha-glucoside transporter genes in brewer’s yeast; Appl. Environ. Microbiol.;  citation_year=1999;  citation_volume=65;  citation_pages=450-456; " /><meta name="citation_reference" content="citation_title=Characterization and functional analysis of the MAL and MPH Loci for maltose utilization in some ale and lager yeast strains; Appl. Environ. Microbiol.;  citation_year=2005;  citation_volume=71;  citation_pages=7846-7857; " /><meta name="citation_reference" content="citation_title=Cloning, sequence and chromosomal location of a MEL gene from Saccharomyces carlsbergensis NCYC396; Gene;  citation_year=1991;  citation_volume=101;  citation_pages=97-104; " /><meta name="citation_reference" content="citation_title=FSY1, a novel gene encoding a specific fructose/H(+) symporter in the type strain of Saccharomyces carlsbergensis; J. Bacteriol.;  citation_year=2000;  citation_volume=182;  citation_pages=5628-5630; " /><meta name="citation_reference" content="citation_title=Characterization of a novel tyrosine permease of lager brewing yeast shared by Saccharomyces cerevisiae strain RM11-1a; FEMS Yeast Res.;  citation_year=2007;  citation_volume=7;  citation_pages=1350-1361; " /><meta name="citation_reference" content="citation_title=Identification and characterization of amidase- homologous AMI1 genes of bottom-fermenting yeast; Yeast;  citation_year=2007;  citation_volume=24;  citation_pages=1075-1084; " /><meta name="citation_reference" content="citation_title=The proton-linked monocarboxylate transporter (MCT) family: structure, function and regulation; Biochem. J.;  citation_year=1999;  citation_volume=343;  citation_issue=Pt 2;  citation_pages=281-299; " /><meta name="citation_reference" content="citation_title=Reconstruction of the genome origins and evolution of the hybrid lager yeast Saccharomyces pastorianus; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=1610-1623; " /><meta name="citation_reference" content="citation_title=Evidence for multiple interspecific hybridization in Saccharomyces sensu stricto species; FEMS Yeast Res.;  citation_year=2002;  citation_volume=1;  citation_pages=323-331; " /><meta name="citation_reference" content="citation_title=Pure and mixed genetic lines of Saccharomyces bayanus and Saccharomyces pastorianus and their contribution to the lager brewing strain genome; Appl. Environ. Microbiol.;  citation_year=2006;  citation_volume=72;  citation_pages=3968-3974; " /><meta name="citation_reference" content="citation_title=Inheritance of mitochondrial DNA in interspecific Saccharomyces hybrids; Ann. Microbiol.;  citation_year=2000;  citation_volume=50;  citation_pages=61-64; " /><meta name="citation_reference" content="citation_title=The inheritance of mtDNA in lager brewing strains; FEMS Yeast Res.;  citation_year=2008;  citation_volume=8;  citation_pages=586-596; " /><meta name="citation_reference" content="citation_title=Molecular evidence for an ancient duplication of the entire yeast genome; Nature;  citation_year=1997;  citation_volume=387;  citation_pages=708-713; " /><meta name="citation_reference" content="citation_title=Construction of model lager brewing strains by interspecific hybridization; Yeast;  citation_year=2007;  citation_volume=24;  citation_pages=01-41; " /><meta name="citation_reference" content="citation_title=Characteristic genome rearrangements in experimental evolution of Saccharomyces cerevisiae; Proc. Natl Acad. Sci. USA;  citation_year=2002;  citation_volume=99;  citation_pages=16144-16149; " /><meta name="citation_reference" content="citation_title=Genome-wide amplifications caused by chromosomal rearrangements play a major role in the adaptive evolution of natural yeast; Genetics;  citation_year=2003;  citation_volume=165;  citation_pages=1745-1759; " /><meta name="citation_reference" content="citation_title=Molecular characterization of a chromosomal rearrangement involved in the adaptive evolution of yeast strains; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=1533-1539; " /><meta name="citation_reference" content="citation_title=An age-induced switch to a hyper-recombinational state; Science;  citation_year=2003;  citation_volume=301;  citation_pages=1908-1911; " /><meta name="citation_reference" content="citation_title=Breeding of brewer’s yeast by hybridization between a top-fermenting yeast Saccharomyces cerevisiae and a cryophilic yeast Saccharomyces bayanus; J. Biosci. Bioeng.;  citation_year=2002;  citation_volume=93;  citation_pages=509-511; " /><meta name="citation_reference" content="citation_title=Gene expression analysis of lager brewing yeast under different oxygenation condition using newly developed DNA microarray; Proc. Eur. Conv. Congr.;  citation_year=2007;  citation_pages=406-419; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/16/2/115/1372622/dsp003.pdf" /><meta name="description" content="Abstract.  This work presents the genome sequencing of the lager brewing yeast (Saccharomyces pastorianus) Weihenstephan 34/70, a strain widely used in lager be" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/16/2/115/510928" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/16/2/115/510928" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/16/2/115/510928");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsp003';
    </script>
<title>
	Genome Sequence of the Lager Brewing Yeast, an Interspecies Hybrid | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./510928" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeiHATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFrYW8sIFlvc2hpaGlybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlImYW1wO0QgUGxhbm5pbmcgRGl2aXNpb24sIFN1bnRvcnkgTGltaXRlZCwgMS0xLTEsIFdha2F5YW1hZGFpLCBTaGltYW1vdG8tY2hvLCBNaXNoaW1hLWd1biwgT3Nha2EgNjE4LTg1MDMsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2FuYW1vcmksIFRha2VzaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSJmFtcDtEIFBsYW5uaW5nIERpdmlzaW9uLCBTdW50b3J5IExpbWl0ZWQsIDEtMS0xLCBXYWtheWFtYWRhaSwgU2hpbWFtb3RvLWNobywgTWlzaGltYS1ndW4sIE9zYWthIDYxOC04NTAzLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikl0b2gsIFRha2VoaWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTWl0c3ViaXNoaSBSZXNlYXJjaCBJbnN0aXR1dGUsIEFkdmFuY2VkIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3kgR3JvdXAsIDMtNiBPdGVtYWNoaSAyLWNob21lIENoaXlvZGEta3UsIFRva3lvIDEwMC04MTQxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktvZGFtYSwgWXVraWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUiZhbXA7RCBQbGFubmluZyBEaXZpc2lvbiwgU3VudG9yeSBMaW1pdGVkLCAxLTEtMSwgV2FrYXlhbWFkYWksIFNoaW1hbW90by1jaG8sIE1pc2hpbWEtZ3VuLCBPc2FrYSA2MTgtODUwMywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJSYWluaWVyaSwgU2FuZHJhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUiZhbXA7RCBQbGFubmluZyBEaXZpc2lvbiwgU3VudG9yeSBMaW1pdGVkLCAxLTEtMSwgV2FrYXlhbWFkYWksIFNoaW1hbW90by1jaG8sIE1pc2hpbWEtZ3VuLCBPc2FrYSA2MTgtODUwMywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWthbXVyYSwgTm9yaWhpc2EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJSJmFtcDtEIFBsYW5uaW5nIERpdmlzaW9uLCBTdW50b3J5IExpbWl0ZWQsIDEtMS0xLCBXYWtheWFtYWRhaSwgU2hpbWFtb3RvLWNobywgTWlzaGltYS1ndW4sIE9zYWthIDYxOC04NTAzLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNoaW1vbmFnYSwgVG9tb2tvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iUiZhbXA7RCBQbGFubmluZyBEaXZpc2lvbiwgU3VudG9yeSBMaW1pdGVkLCAxLTEtMSwgV2FrYXlhbWFkYWksIFNoaW1hbW90by1jaG8sIE1pc2hpbWEtZ3VuLCBPc2FrYSA2MTgtODUwMywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIYXR0b3JpLCBNYXNhaGlyYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQ29tcHV0YXRpb25hbCBCaW9sb2d5LCBHcmFkdWF0ZSBTY2hvb2wgb2YgRnJvbnRpZXIgU2NpZW5jZXMsIFVuaXZlcnNpdHkgb2YgVG9reW8sIEthc2hpd2Etbm8taGEgNS0xLTUsIEthc2hpd2EsIENoaWJhIDI3Ny04NTYxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFzaGlrYXJpLCBUb3NoaWhpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJGbG9yaWdlbmUgUHR5LCBMdGQsIDEgUGFyayBEcml2ZSwgQnVuZG9vcmEsIFZJQyAzMDgzLCBBdXN0cmFsaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ikdlbm9tZSBTZXF1ZW5jZSBvZiB0aGUgTGFnZXIgQnJld2luZyBZZWFzdCwgYW4gSW50ZXJzcGVjaWVzIEh5YnJpZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjExNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTI5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNwMDAzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDkvMDQvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgYmFydGggcmVwb3J0IGhvcHMgMjAwNi8yMDA3OyAgY2l0YXRpb25fdXJsPWh0dHA6Ly93d3cuYmFydGhoYWFzZ3JvdXAuY29tL2Ntc2RrL2NvbnRlbnQvYmhnL2JhcnRoX3JlcG9ydC5odG07ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludmVzdGlnYXRpb25zIG9uIHRoZSBwaHlzaW9sb2d5IGFuZCBtb3JwaG9sb2d5IG9mIGJ1ZGRpbmcgeWVhc3QuIFhJSUkuIE5ldyBzdHVkaWVzIG9mIGJvdHRvbSBmZXJtZW50aW5nIGJyZXdlcnMgeWVhc3RzOyBDLiBSLiBUcmF2LiBMYWIuIENhcmxzYmVyZy47ICBjaXRhdGlvbl95ZWFyPTE5MDg7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTE3OS0yMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RleHQ9Q2hhcHRlciA0NDsgY2l0YXRpb25fcHVibGlzaGVyPUVsc2V2aWVyLCBBbXN0ZXJkYW07IFRoZSBZZWFzdHMsIGEgVGF4b25vbWljIFN0dWR5OiBTYWNjaGFyb215Y2VzIE15ZW4gZXggUmVlc3NtOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fcGFnZXM9MzU4LTM3MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhyZWUgbmV3bHkgZGVsaW1pdGVkIHNwZWNpZXMgb2YgU2FjY2hhcm9teWNlcyBzZW5zdSBzdHJpY3RvOyBBbnRvbmllIFZhbiBMZWV1d2VuaG9lazsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT01MzsgIGNpdGF0aW9uX3BhZ2VzPTc3LTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZW94eXJpYm9udWNsZWljIGFjaWQgcmVsYXRlZG5lc3MgYW1vbmcgc3BlY2llcyBvZiB0aGUgZ2VudXMgU2FjY2hhcm9teWNlcyBzZW5zdSBzdHJpY3RvOyBJbnQuIEouIFN5c3QuIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODU7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz01MDgtNTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Dby1leGlzdGVuY2Ugb2YgdHdvIHR5cGVzIG9mIGNocm9tb3NvbWUgaW4gdGhlIGJvdHRvbSBmZXJtZW50aW5nIHllYXN0LCBTYWNjaGFyb215Y2VzIHBhc3RvcmlhbnVzOyBZZWFzdDsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTkyMy05MzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNocm9tb3NvbWFsIHN0cnVjdHVyZXMgb2YgYm90dG9tIGZlcm1lbnRpbmcgeWVhc3RzOyBTeXN0LiBBcHBsLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MzQxLTM1MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNmZXIgb2YgY2hyb21vc29tZSBJSUkgZHVyaW5nIGthciBtZWRpYXRlZCBjeXRvZHVjdGlvbiBpbiB5ZWFzdDsgQ2FybHNiZXJnIFJlcy4gQ29tbXVuLjsgIGNpdGF0aW9uX3llYXI9MTk4MDsgIGNpdGF0aW9uX3ZvbHVtZT00NTsgIGNpdGF0aW9uX3BhZ2VzPTExMy0xMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIGNocm9tb3NvbWUgViBhbmQgdGhlIElMVjEgZ2VuZSBmcm9tIFNhY2NoYXJvbXljZXMgY2FybHNiZXJnZW5zaXM7IENhcmxzYmVyZyBSZXMuIENvbW11bi47ICBjaXRhdGlvbl95ZWFyPTE5ODY7ICBjaXRhdGlvbl92b2x1bWU9NTE7ICBjaXRhdGlvbl9wYWdlcz0zMDktMzI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljcyBvZiBicmV3aW5nIHllYXN0czsgIGNpdGF0aW9uX2VkaXRpb249Mm5kIEVkLjsgY2l0YXRpb25fcHVibGlzaGVyPUFjYWRlbWljIFByZXNzLCBMb25kb247IFRoZSBZZWFzdHM7ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9Vm9sLiA2OyAgY2l0YXRpb25fcGFnZXM9MjIzLTI1NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnVjbGVvdGlkZSBzZXF1ZW5jZXMgb2YgYWxjb2hvbCBhY2V0eWx0cmFuc2ZlcmFzZSBnZW5lcyBmcm9tIGxhZ2VyIGJyZXdpbmcgeWVhc3QgU2FjY2hhcm9teWNlcyBjYXJsc2JlcmdlbnNpczsgWWVhc3Q7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz01OTMtNTk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXZlcnNpdHkgb2YgdGhlIEhPIGdlbmUgZW5jb2RpbmcgYW4gZW5kb251Y2xlYXNlIGZvciBtYXRpbmctdHlwZSBjb252ZXJzaW9uIGluIHRoZSBib3R0b20gZmVybWVudGluZyB5ZWFzdCBTYWNjaGFyb215Y2VzIHBhc3RvcmlhbnVzOyBZZWFzdDsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTEzMzUtMTM0MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGEgZ2VuZSBzcGVjaWZpYyB0byBsYWdlciBicmV3aW5nIHllYXN0IHRoYXQgZW5jb2RlcyBhIGJyYW5jaGVkLWNoYWluIGFtaW5vIGFjaWQgcGVybWVhc2U7IEFwcGwuIEVudmlyb24uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9Njc7ICBjaXRhdGlvbl9wYWdlcz0zNDU1LTM0NjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNhY2NoYXJvbXljZXMgY2FybHNiZXJnZW5zaXMgY29udGFpbnMgdHdvIGZ1bmN0aW9uYWwgTUVUMiBhbGxlbGVzIHNpbWlsYXIgdG8gaG9tb2xvZ3VlcyBmcm9tIFMuIGNlcmV2aXNpYWUgYW5kIFMuIG1vbmFjZW5zaXM7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTQwOyAgY2l0YXRpb25fcGFnZXM9MzMtNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpdmVyc2l0eSBpbiBvcmdhbml6YXRpb24gYW5kIHRoZSBvcmlnaW4gb2YgZ2VuZSBvcmRlcnMgaW4gdGhlIG1pdG9jaG9uZHJpYWwgRE5BIG1vbGVjdWxlcyBvZiB0aGUgZ2VudXMgU2FjY2hhcm9teWNlczsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTE4MzMtMTg0MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGV4dD1DaGFwdGVyIDQ7IGNpdGF0aW9uX3B1Ymxpc2hlcj1CbGFja3dlbGwgU2NpZW5jZSBMdGQsIE94Zm9yZDsgQnJld2luZyBZZWFzdCBhbmQgRmVybWVudGF0aW9uOiBCcmV3aW5nIHllYXN0OyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fcGFnZXM9MTQzLTI1OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFjdG9ycyBpbmZsdWVuY2UgbWFsdG90cmlvc2UgdXRpbGl6YXRpb24gZHVyaW5nIGJyZXdlcnkgd29ydCBmZXJtZW50YXRpb25zOyBKLiBBbS4gU29jLiBCcmV3LiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT01MjsgIGNpdGF0aW9uX3BhZ2VzPTQxLTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob3ZlbCBkaWZmZXJlbmNlcyBiZXR3ZWVuIGFsZSBhbmQgbGFnZXIgYnJld2luZyB5ZWFzdHM7IFByb2MuIENvbmdyLiBFdXIuIEJyZXcuIENvbnYuOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MjY3LTI3NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFzZS1jYWxsaW5nIG9mIGF1dG9tYXRlZCBzZXF1ZW5jZXIgdHJhY2VzIHVzaW5nIHBocmVkLiBJLiBBY2N1cmFjeSBhc3Nlc3NtZW50OyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MTc1LTE4NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UENBUDogYSB3aG9sZS1nZW5vbWUgYXNzZW1ibHkgcHJvZ3JhbTsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0yMTY0LTIxNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdob2xlLWdlbm9tZSBzaG90Z3VuIG9wdGljYWwgbWFwcGluZyBvZiBSaG9kb3NwaXJpbGx1bSBydWJydW07IEFwcGwuIEVudmlyb24uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NzE7ICBjaXRhdGlvbl9wYWdlcz01NTExLTU1MjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeWxvZ2VuZXRpYyBhbmFseXNpcyBvZiB0aGUgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIGdyb3VwIGJhc2VkIG9uIHBvbHltb3JwaGlzbXMgb2YgckROQSBzcGFjZXIgc2VxdWVuY2VzOyBJbnQuIEouIFN5c3QuIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9NDg7ICBjaXRhdGlvbl9pc3N1ZT1QdCAxOyAgY2l0YXRpb25fcGFnZXM9Mjk1LTMwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXNjYWxlIGFwcHJvYWNoZXMgdG8gcmVzb2x2aW5nIGluY29uZ3J1ZW5jZSBpbiBtb2xlY3VsYXIgcGh5bG9nZW5pZXM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT00MjU7ICBjaXRhdGlvbl9wYWdlcz03OTgtODA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ud28gc3ViZ3JvdXBzIHdpdGhpbiB0aGUgU2FjY2hhcm9teWNlcyBiYXlhbnVzIHNwZWNpZXMgZXZpZGVuY2VkIGJ5IFBDUiBhbXBsaWZpY2F0aW9uIGFuZCByZXN0cmljdGlvbiBwb2x5bW9ycGhpc20gb2YgdGhlIG5vbi10cmFuc2NyaWJlZCBzcGFjZXIyIGluIHRoZSByaWJvc29tYWwgRE5BIHVuaXQ7IFN5c3RlbS4gQXBwbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTI4Ni0yOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVR3byBzdWJncm91cHMgd2l0aGluIHRoZSBTYWNjaGFyb215Y2VzIGJheWFudXMgc3BlY2llcyBldmlkZW5jZWQgYnkgUENSIGFtcGxpZmljYXRpb24gYW5kIHJlc3RyaWN0aW9uIHBvbHltb3JwaGlzbSBvZiB0aGUgbm9uLXRyYW5zY3JpYmVkIHNwYWNlcjIgaW4gdGhlIHJpYm9zb21hbCBETkEgdW5pdDsgU3lzdGVtLiBBcHBsLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9Mjg2LTI5NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2luZyBhbmQgY29tcGFyaXNvbiBvZiB5ZWFzdCBzcGVjaWVzIHRvIGlkZW50aWZ5IGdlbmVzIGFuZCByZWd1bGF0b3J5IGVsZW1lbnRzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9NDIzOyAgY2l0YXRpb25fcGFnZXM9MjQxLTI1NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVlc3QgZm9yIHdpbmUgeWVhc3Rz4oCUQW4gb2xkIHN0b3J5IHJldmlzaXRlZDsgSi4gSW5kLiBNaWNyb2Jpb2wuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MzAzLTMxMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNpbmcgYW5kIGNvbXBhcmF0aXZlIGFuYWx5c2lzIG9mIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZSBzdHJhaW4gWUpNNzg5OyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEwNDsgIGNpdGF0aW9uX3BhZ2VzPTEyODI1LTEyODMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm9vZiBhbmQgZXZvbHV0aW9uYXJ5IGFuYWx5c2lzIG9mIGFuY2llbnQgZ2Vub21lIGR1cGxpY2F0aW9uIGluIHRoZSB5ZWFzdCBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWU7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT00Mjg7ICBjaXRhdGlvbl9wYWdlcz02MTctNjI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWZmZXJlbnQgdHlwZXMgb2YgcmVjb21iaW5hdGlvbiBldmVudHMgYXJlIGNvbnRyb2xsZWQgYnkgdGhlIFJBRDEgYW5kIFJBRDUyIGdlbmVzIG9mIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZTsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5ODg7ICBjaXRhdGlvbl92b2x1bWU9MTIwOyAgY2l0YXRpb25fcGFnZXM9MzY3LTM3NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjb21iaW5hdGlvbiBiZXR3ZWVuIHJlcGVhdGVkIGdlbmVzIGluIG1pY3Jvb3JnYW5pc21zOyBBbm51LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTE0Ny0xNjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsbGVsaWMgYW5kIGVjdG9waWMgcmVjb21iaW5hdGlvbiBiZXR3ZWVuIFR5IGVsZW1lbnRzIGluIHllYXN0OyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT0xMTk7ICBjaXRhdGlvbl9wYWdlcz01NDktNTU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EdWFsIHJvbGVzIGZvciBETkEgc2VxdWVuY2UgaWRlbnRpdHkgYW5kIHRoZSBtaXNtYXRjaCByZXBhaXIgc3lzdGVtIGluIHRoZSByZWd1bGF0aW9uIG9mIG1pdG90aWMgY3Jvc3Npbmctb3ZlciBpbiB5ZWFzdDsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT05NDsgIGNpdGF0aW9uX3BhZ2VzPTk3NTctOTc2MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RG91YmxlLXN0cmFuZCBicmVha3MgYXNzb2NpYXRlZCB3aXRoIHJlcGV0aXRpdmUgRE5BIGNhbiByZXNoYXBlIHRoZSBnZW5vbWU7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTA1OyAgY2l0YXRpb25fcGFnZXM9MTE4NDUtMTE4NTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1TcHJpbmdlci1WZXJsYWcsIEhlaWRlbGJlcmc7IFRvcC4gQ3Vyci4gR2VuZXQuOiBMYWdlciBicmV3aW5nIHllYXN0OyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPVZvbC4gMTU7ICBjaXRhdGlvbl9wYWdlcz0xNDUtMTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbmlwYXJlbnRhbCBsb3NzIG9mIHJpYm9zb21hbCBETkEgaW4gdGhlIGFsbG90ZXRyYXBsb2lkIGdyYXNzIFppbmdlcmlhIHRyaWNob3BvZGEgKDJuIGVxdWFscyA4KTsgR2Vub21lOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9MTU2LTE2MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIG1pdG9jaG9uZHJpYWwgRE5BIGluIHllYXN0OiBnZW5lIG9yZGVyIGFuZCBzdHJ1Y3R1cmFsIG9yZ2FuaXphdGlvbiBvZiB0aGUgbWl0b2Nob25kcmlhbCBnZW5vbWUgb2YgU2FjY2hhcm9teWNlcyB1dmFydW07IEN1cnIuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTUyLTU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5jdGlvbmFsIHByb2ZpbGluZyBvZiB0aGUgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIGdlbm9tZTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTQxODsgIGNpdGF0aW9uX3BhZ2VzPTM4Ny0zOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcHJvdmVtZW50IG9mIG1hbHRvc2UgZmVybWVudGF0aW9uIGVmZmljaWVuY3k6IGNvbnN0aXR1dGl2ZSBleHByZXNzaW9uIG9mIE1BTCBnZW5lcyBpbiBicmV3aW5nIHllYXN0czsgSi4gQW0uIFNvYy4gQnJldy4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9NTM7ICBjaXRhdGlvbl9wYWdlcz0yNC0yOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFsdG90cmlvc2UgdXRpbGl6YXRpb24gaW4gbGFnZXIgeWVhc3Qgc3RyYWluczogTVRUMSBlbmNvZGVzIGEgbWFsdG90cmlvc2UgdHJhbnNwb3J0ZXI7IFllYXN0OyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9Nzc1LTc4ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXVsdGlwbGUgYWxwaGEtZ2x1Y29zaWRlIHRyYW5zcG9ydGVyIGdlbmVzIGluIGJyZXdlcuKAmXMgeWVhc3Q7IEFwcGwuIEVudmlyb24uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9NjU7ICBjaXRhdGlvbl9wYWdlcz00NTAtNDU2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIGFuZCBmdW5jdGlvbmFsIGFuYWx5c2lzIG9mIHRoZSBNQUwgYW5kIE1QSCBMb2NpIGZvciBtYWx0b3NlIHV0aWxpemF0aW9uIGluIHNvbWUgYWxlIGFuZCBsYWdlciB5ZWFzdCBzdHJhaW5zOyBBcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTcxOyAgY2l0YXRpb25fcGFnZXM9Nzg0Ni03ODU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbG9uaW5nLCBzZXF1ZW5jZSBhbmQgY2hyb21vc29tYWwgbG9jYXRpb24gb2YgYSBNRUwgZ2VuZSBmcm9tIFNhY2NoYXJvbXljZXMgY2FybHNiZXJnZW5zaXMgTkNZQzM5NjsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MTk5MTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDE7ICBjaXRhdGlvbl9wYWdlcz05Ny0xMDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZTWTEsIGEgbm92ZWwgZ2VuZSBlbmNvZGluZyBhIHNwZWNpZmljIGZydWN0b3NlL0goKykgc3ltcG9ydGVyIGluIHRoZSB0eXBlIHN0cmFpbiBvZiBTYWNjaGFyb215Y2VzIGNhcmxzYmVyZ2Vuc2lzOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE4MjsgIGNpdGF0aW9uX3BhZ2VzPTU2MjgtNTYzMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBvZiBhIG5vdmVsIHR5cm9zaW5lIHBlcm1lYXNlIG9mIGxhZ2VyIGJyZXdpbmcgeWVhc3Qgc2hhcmVkIGJ5IFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZSBzdHJhaW4gUk0xMS0xYTsgRkVNUyBZZWFzdCBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0xMzUwLTEzNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGFtaWRhc2UtIGhvbW9sb2dvdXMgQU1JMSBnZW5lcyBvZiBib3R0b20tZmVybWVudGluZyB5ZWFzdDsgWWVhc3Q7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xMDc1LTEwODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBwcm90b24tbGlua2VkIG1vbm9jYXJib3h5bGF0ZSB0cmFuc3BvcnRlciAoTUNUKSBmYW1pbHk6IHN0cnVjdHVyZSwgZnVuY3Rpb24gYW5kIHJlZ3VsYXRpb247IEJpb2NoZW0uIEouOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTM0MzsgIGNpdGF0aW9uX2lzc3VlPVB0IDI7ICBjaXRhdGlvbl9wYWdlcz0yODEtMjk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNvbnN0cnVjdGlvbiBvZiB0aGUgZ2Vub21lIG9yaWdpbnMgYW5kIGV2b2x1dGlvbiBvZiB0aGUgaHlicmlkIGxhZ2VyIHllYXN0IFNhY2NoYXJvbXljZXMgcGFzdG9yaWFudXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MTYxMC0xNjIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmlkZW5jZSBmb3IgbXVsdGlwbGUgaW50ZXJzcGVjaWZpYyBoeWJyaWRpemF0aW9uIGluIFNhY2NoYXJvbXljZXMgc2Vuc3Ugc3RyaWN0byBzcGVjaWVzOyBGRU1TIFllYXN0IFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTMyMy0zMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVB1cmUgYW5kIG1peGVkIGdlbmV0aWMgbGluZXMgb2YgU2FjY2hhcm9teWNlcyBiYXlhbnVzIGFuZCBTYWNjaGFyb215Y2VzIHBhc3RvcmlhbnVzIGFuZCB0aGVpciBjb250cmlidXRpb24gdG8gdGhlIGxhZ2VyIGJyZXdpbmcgc3RyYWluIGdlbm9tZTsgQXBwbC4gRW52aXJvbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03MjsgIGNpdGF0aW9uX3BhZ2VzPTM5NjgtMzk3NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5oZXJpdGFuY2Ugb2YgbWl0b2Nob25kcmlhbCBETkEgaW4gaW50ZXJzcGVjaWZpYyBTYWNjaGFyb215Y2VzIGh5YnJpZHM7IEFubi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT01MDsgIGNpdGF0aW9uX3BhZ2VzPTYxLTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgaW5oZXJpdGFuY2Ugb2YgbXRETkEgaW4gbGFnZXIgYnJld2luZyBzdHJhaW5zOyBGRU1TIFllYXN0IFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTU4Ni01OTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBldmlkZW5jZSBmb3IgYW4gYW5jaWVudCBkdXBsaWNhdGlvbiBvZiB0aGUgZW50aXJlIHllYXN0IGdlbm9tZTsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTM4NzsgIGNpdGF0aW9uX3BhZ2VzPTcwOC03MTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBtb2RlbCBsYWdlciBicmV3aW5nIHN0cmFpbnMgYnkgaW50ZXJzcGVjaWZpYyBoeWJyaWRpemF0aW9uOyBZZWFzdDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTAxLTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpc3RpYyBnZW5vbWUgcmVhcnJhbmdlbWVudHMgaW4gZXhwZXJpbWVudGFsIGV2b2x1dGlvbiBvZiBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWU7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9OTk7ICBjaXRhdGlvbl9wYWdlcz0xNjE0NC0xNjE0OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgYW1wbGlmaWNhdGlvbnMgY2F1c2VkIGJ5IGNocm9tb3NvbWFsIHJlYXJyYW5nZW1lbnRzIHBsYXkgYSBtYWpvciByb2xlIGluIHRoZSBhZGFwdGl2ZSBldm9sdXRpb24gb2YgbmF0dXJhbCB5ZWFzdDsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTY1OyAgY2l0YXRpb25fcGFnZXM9MTc0NS0xNzU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgY2hhcmFjdGVyaXphdGlvbiBvZiBhIGNocm9tb3NvbWFsIHJlYXJyYW5nZW1lbnQgaW52b2x2ZWQgaW4gdGhlIGFkYXB0aXZlIGV2b2x1dGlvbiBvZiB5ZWFzdCBzdHJhaW5zOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTE1MzMtMTUzOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gYWdlLWluZHVjZWQgc3dpdGNoIHRvIGEgaHlwZXItcmVjb21iaW5hdGlvbmFsIHN0YXRlOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMwMTsgIGNpdGF0aW9uX3BhZ2VzPTE5MDgtMTkxMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QnJlZWRpbmcgb2YgYnJld2Vy4oCZcyB5ZWFzdCBieSBoeWJyaWRpemF0aW9uIGJldHdlZW4gYSB0b3AtZmVybWVudGluZyB5ZWFzdCBTYWNjaGFyb215Y2VzIGNlcmV2aXNpYWUgYW5kIGEgY3J5b3BoaWxpYyB5ZWFzdCBTYWNjaGFyb215Y2VzIGJheWFudXM7IEouIEJpb3NjaS4gQmlvZW5nLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT05MzsgIGNpdGF0aW9uX3BhZ2VzPTUwOS01MTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgZXhwcmVzc2lvbiBhbmFseXNpcyBvZiBsYWdlciBicmV3aW5nIHllYXN0IHVuZGVyIGRpZmZlcmVudCBveHlnZW5hdGlvbiBjb25kaXRpb24gdXNpbmcgbmV3bHkgZGV2ZWxvcGVkIEROQSBtaWNyb2FycmF5OyBQcm9jLiBFdXIuIENvbnYuIENvbmdyLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3BhZ2VzPTQwNi00MTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xNi8yLzExNS8xMzcyNjIyL2RzcDAwMy5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVGhpcyB3b3JrIHByZXNlbnRzIHRoZSBnZW5vbWUgc2VxdWVuY2luZyBvZiB0aGUgbGFnZXIgYnJld2luZyB5ZWFzdCAoU2FjY2hhcm9teWNlcyBwYXN0b3JpYW51cykgV2VpaGVuc3RlcGhhbiAzNC83MCwgYSBzdHJhaW4gd2lkZWx5IHVzZWQgaW4gbGFnZXIgYmUiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTUxMDkyOCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQllckcpA71A8HVreH8q6YECfVhgA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f2%2f115%2f510928" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f2%2f115%2f510928" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=20204" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/16/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/16/2/1/m_cover.gif?Expires=2147483647&amp;Signature=spYD3Y1ECWwq5jfgvLwgtJGBnOHAN-nZXtWf5jX0Idx1cbrJRV4FKB28EY3pe17T0gsGMxo5dA8Tel6c2xNGAqa1byNOzqnWfnM8ohlcCfLHlrfGOEn2zyCW-15VbU7SOoLTo3OmdKcdJ19GrDbXCyAEcU~VJnaBCNGoOwBtbTnh6-S6KzBaPxjqj1HfbfKuNRe7H3r-0uFeA~Y3Ph79tw8QP6q4D0FiiXdjGGWYTNp3-Ga~m-PPAcOogXTotI1YFUthxQJgvIKyhC8JLbL3Znx728-CpY2KbMeFQlltwFxSw7d-mO5IOuvD7Wc91P7OLYCdnpnsyiP0k-TqunqvaA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/16/2">
                        <div class="volume trailing-comma">Volume 16</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2009        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7937705">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7937705">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7937709">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7937709">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7937720">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7937720">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7937756">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7937756">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7937801">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7937801">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7937803">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7937803">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7937805">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7937805">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/16/2/105/510716">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/16/2/131/511114">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome Sequence of the Lager Brewing Yeast, an Interspecies Hybrid
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshihiro Nakao</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">R&amp;D Planning Division</div>, <div class="insititution">Suntory Limited</div>, <div class="addr-line">1-1-1, Wakayamadai, Shimamoto-cho, Mishima-gun, Osaka 618-8503</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 75-962-2365. Fax. <div class="fax">+81 75-962-3791</div>. E-mail: <a href="mailto:yoshihiro_nakao@suntory.co.jp" target="_blank">yoshihiro_nakao@suntory.co.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshihiro+Nakao">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakao Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakao Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshihiro Nakao</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takeshi Kanamori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">R&amp;D Planning Division</div>, <div class="insititution">Suntory Limited</div>, <div class="addr-line">1-1-1, Wakayamadai, Shimamoto-cho, Mishima-gun, Osaka 618-8503</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takeshi+Kanamori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kanamori T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kanamori T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takeshi Kanamori</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takehiko Itoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Mitsubishi Research Institute</div>, <div class="insititution">Advanced Science and Technology Group</div>, <div class="addr-line">3-6 Otemachi 2-chome Chiyoda-ku, Tokyo 100-8141</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takehiko+Itoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Itoh T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Itoh T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takehiko Itoh</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yukiko Kodama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">R&amp;D Planning Division</div>, <div class="insititution">Suntory Limited</div>, <div class="addr-line">1-1-1, Wakayamadai, Shimamoto-cho, Mishima-gun, Osaka 618-8503</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yukiko+Kodama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kodama Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kodama Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yukiko Kodama</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sandra Rainieri</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">R&amp;D Planning Division</div>, <div class="insititution">Suntory Limited</div>, <div class="addr-line">1-1-1, Wakayamadai, Shimamoto-cho, Mishima-gun, Osaka 618-8503</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sandra+Rainieri">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rainieri S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Rainieri S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sandra Rainieri</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Norihisa Nakamura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">R&amp;D Planning Division</div>, <div class="insititution">Suntory Limited</div>, <div class="addr-line">1-1-1, Wakayamadai, Shimamoto-cho, Mishima-gun, Osaka 618-8503</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Norihisa+Nakamura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakamura N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakamura N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Norihisa Nakamura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tomoko Shimonaga</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">R&amp;D Planning Division</div>, <div class="insititution">Suntory Limited</div>, <div class="addr-line">1-1-1, Wakayamadai, Shimamoto-cho, Mishima-gun, Osaka 618-8503</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tomoko+Shimonaga">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shimonaga T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shimonaga T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tomoko Shimonaga</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masahira Hattori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Computational Biology</div>, <div class="insititution">Graduate School of Frontier Sciences, University of Tokyo</div>, <div class="addr-line">Kashiwa-no-ha 5-1-5, Kashiwa, Chiba 277-8561</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masahira+Hattori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hattori M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hattori M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masahira Hattori</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Toshihiko Ashikari</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">Florigene Pty, Ltd</div>, <div class="addr-line">1 Park Drive, Bundoora, VIC 3083</div>, <div class="country">Australia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Toshihiko+Ashikari">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ashikari T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ashikari T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Toshihiko Ashikari</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="FN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Present address: AZTI-Tecnalia, Food Research Division, Parque Tecnológico de Bizkaia, Astondo Bidea 609, 48160, Derio, Spain</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 16, Issue 2, 1 April 2009, Pages 115–129, <a href='https://doi.org/10.1093/dnares/dsp003'>https://doi.org/10.1093/dnares/dsp003</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">04 March 2009</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">30 September 2008</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">07 February 2009</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="510928" href="/dnaresearch/article-pdf/16/2/115/1372622/dsp003.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yoshihiro Nakao, Takeshi Kanamori, Takehiko Itoh, Yukiko Kodama, Sandra Rainieri, Norihisa Nakamura, Tomoko Shimonaga, Masahira Hattori, Toshihiko Ashikari;  Genome Sequence of the Lager Brewing Yeast, an Interspecies Hybrid, <em>DNA Research</em>, Volume 16, Issue 2, 1 April 2009, Pages 115–129, <a href='https://doi.org/10.1093/dnares/dsp003'>https://doi.org/10.1093/dnares/dsp003</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/510928?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/510928?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/510928?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/510928?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/510928?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/510928?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/510928?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome%20Sequence%20of%20the%20Lager%20Brewing%20Yeast%2C%20an%20Interspecies%20Hybrid&amp;publicationDate=2009-03-04&amp;volumeNum=16&amp;issueNum=2&amp;author=Nakao%2C%20Yoshihiro%3B%20Kanamori%2C%20Takeshi&amp;startPage=115&amp;endPage=129&amp;contentId=10.1093%2Fdnares%2Fdsp003&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f16%2f2%2f115%2f510928" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7937705 id="7937705" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>This work presents the genome sequencing of the lager brewing yeast (<em>Saccharomyces pastorianus</em>) Weihenstephan 34/70, a strain widely used in lager beer brewing. The 25 Mb genome comprises two nuclear sub-genomes originating from <em>Saccharomyces cerevisiae</em> and <em>Saccharomyces bayanus</em> and one circular mitochondrial genome originating from <em>S</em>. <em>bayanus</em>. Thirty-six different types of chromosomes were found including eight chromosomes with translocations between the two sub-genomes, whose breakpoints are within the orthologous open reading frames. Several gene loci responsible for typical lager brewing yeast characteristics such as maltotriose uptake and sulfite production have been increased in number by chromosomal rearrangements. Despite an overall high degree of conservation of the synteny with <em>S</em>. <em>cerevisiae</em> and <em>S</em>. <em>bayanus</em>, the syntenies were not well conserved in the sub-telomeric regions that contain lager brewing yeast characteristic and specific genes. Deletion of larger chromosomal regions, a massive unilateral decrease of the ribosomal DNA cluster and bilateral truncations of over 60 genes reflect a post-hybridization evolution process. Truncations and deletions of less efficient maltose and maltotriose uptake genes may indicate the result of adaptation to brewing. The genome sequence of this interspecies hybrid yeast provides a new tool for better understanding of lager brewing yeast behavior in industrial beer production.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Saccharomyces pastorianus&quot;"><em>Saccharomyces pastorianus</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="beer">beer</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="genome">genome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;interspecies hybrid&quot;">interspecies hybrid</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;larger yeast&quot;">larger yeast</a></div>                            <h2 scrollto-destination=7937709 id="7937709" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Beer is one of the most common alcoholic beverages, and ∼1.7 × 10<sup>11</sup> L are produced annually worldwide.<sup><a reveal-id="DSP003C1" data-open="DSP003C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Brewing yeasts are mainly divided into two groups, ale brewing yeasts and lager brewing yeasts, according to their use for the production of ales and lagers, respectively. Lager beers are fermented by using lager brewing strains at low temperatures, ranging from 8 to 15°C, and they represent the largest part of the beer market. Therefore, most of the research on beer yeast has been conducted on lager brewing yeast.</p><p>Ale and lager brewing yeasts belong to the <em>Saccharomyces</em> genus. The first lager brewing yeast that was studied in cultures derived from a single cell was classified as <em>Saccharomyces carlsbergensis</em>, whereas ale brewing yeast was classified as <em>Saccharomyces cerevisiae</em>.<sup><a reveal-id="DSP003C2" data-open="DSP003C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSP003C3" data-open="DSP003C3" class="link link-ref link-reveal xref-bibr">3</a></sup> According to subsequent more sophisticated taxonomic techniques such as DNA–DNA hybridization studies, the species <em>S</em>. <em>carlsbergensis</em> was later included in the species <em>Saccharomyces pastorianus</em>,<sup><a reveal-id="DSP003C4" data-open="DSP003C4" class="link link-ref link-reveal xref-bibr">4</a></sup> and therefore, lager brewing yeast is now classified as <em>S</em>. <em>pastorianus</em>.<sup><a reveal-id="DSP003C3" data-open="DSP003C3" class="link link-ref link-reveal xref-bibr">3</a></sup> On the basis of the results of DNA–DNA hybridization studies, lager brewing yeast was described as an allopolyploid interspecies hybrid between <em>S</em>. <em>cerevisiae</em> and <em>Saccharomyces bayanus</em>;<sup><a reveal-id="DSP003C5" data-open="DSP003C5" class="link link-ref link-reveal xref-bibr">5</a></sup> this finding was later supported by some Southern hybridization studies using <em>S</em>. <em>cerevisiae</em> gene probes.<sup><a reveal-id="DSP003C6" data-open="DSP003C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSP003C7" data-open="DSP003C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Single-chromosome transfer from lager brewing yeast to <em>S</em>. <em>cerevisiae</em> and subsequent genetic analyses also showed that lager brewing yeast is a species hybrid between <em>S</em>. <em>cerevisiae</em> and another <em>Saccharomyces</em> species, possibly represented by <em>S</em>. <em>bayanus</em>.<sup><a reveal-id="DSP003C8  DSP003C10" data-open="DSP003C8  DSP003C10" class="link link-ref link-reveal xref-bibr">8–10</a></sup> As expected in the case of an interspecies hybrid, two divergent orthologous genes were found in this yeast for almost all the genes tested: in most cases, they were very similar to the corresponding <em>S</em>. <em>cerevisiae</em> and <em>S</em>. <em>bayanus</em> genes.<sup><a reveal-id="DSP003C10  DSP003C13" data-open="DSP003C10  DSP003C13" class="link link-ref link-reveal xref-bibr">10–13</a></sup> However, a significant sequence divergence was found between a part of the <em>S</em>. <em>bayanus</em>-type <em>MET2</em> gene of the lager brewing yeast and the corresponding <em>S</em>. <em>bayanus</em> sequence, suggesting that the Sb-type sub-genome is slightly different from <em>S</em>. <em>bayanus</em>.<sup><a reveal-id="DSP003C14" data-open="DSP003C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Furthermore, because only a few genes or chromosomes have been identified and analyzed in the lager brewing yeast so far, the overall picture of its genome (e.g. genome size and chromosome number and precise structure) has not been clarified yet.</p><p>Lager brewing yeast mitochondrial DNA (mtDNA) show a similar map of restriction sites and the same gene order as <em>S</em>. <em>bayanus</em>, and it differs from <em>S</em>. <em>cerevisiae</em> mtDNA, suggesting that in lager brewing yeast mtDNA has been inherited from the <em>S</em>. <em>bayanus</em> parental species.<sup><a reveal-id="DSP003C15" data-open="DSP003C15" class="link link-ref link-reveal xref-bibr">15</a></sup></p><p>There are several biological features that are characteristic of lager brewing yeast, such as a proper degree and timing of flocculation and the efficient ability to assimilate maltose and maltotriose at low temperature.<sup><a reveal-id="DSP003C16" data-open="DSP003C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSP003C17" data-open="DSP003C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Another characteristic of lager brewing yeast is the ability to produce a significant amount of sulfite, which is important for the flavor stability of beer in several ways, including antioxidant activity. Lager brewing yeasts produce a larger amount of sulfite [&gt;4 parts per million (ppm)] than ale brewing yeasts (&lt;2 ppm).<sup><a reveal-id="DSP003C18" data-open="DSP003C18" class="link link-ref link-reveal xref-bibr">18</a></sup> However, the complete understanding of all these characteristics has been limited so far, due to the lack of a full genome sequence of this yeast.</p><p>Here, we report the entire sequence of the hybrid genome of the lager brewing yeast <em>S</em>. <em>pastorianus</em> strain Weihenstephan 34/70, and compare it with the genome sequences of other <em>Saccharomyces</em> species in order to explore the genetic basis of the characteristics of lager brewing yeast relevant to beer production. The sequence of a lager beer production strain has implications for understanding not only the biological characteristics of lager brewing yeasts but also the post-hybridization evolutionary history of hybrid yeast and species. We believe that the comprehensive analysis of lager brewing yeasts using this sequence will be highly beneficial to the brewing industry in terms of facilitating beer quality control, promoting the development of breeding programs for the construction of new yeast cultures and the development of new alcoholic products.</p>                            <h2 scrollto-destination=7937720 id="7937720" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7937722 id="7937722" class="section-title" data-legacy-id=s2a>2.1. Strain and sequence information</h3>
<p>The lager brewing strain <em>S</em>. <em>pastorianus</em> Weihenstephan 34/70 was provided by Fachhochschule Weihenstephan, Freising, Germany. Genome sequences and annotations for <em>S</em>. <em>cerevisiae</em> S288C, <em>S</em>. <em>bayanus</em> CBS 7001, <em>Saccharomyces castellii</em> CBS 4309, <em>Saccharomyces kudriavzevii</em> CBS 8840, <em>Saccharomyces mikatae</em> CBS 8839 and <em>Saccharomyces paradoxus</em> CBS 432 were obtained from the <em>Saccharomyce</em>s Genome Database (SGD; <a class="link link-uri" href="http://www.yeastgenome.org/">http://www.yeastgenome.org/</a>) (30 August 2006). Genome sequences for <em>S</em>. <em>cerevisiae</em> YJM789 and RM11-1a were obtained from the GenBank database (accession no. AAFW02000000) and from the Broad Institute (<a class="link link-uri" href="http://www.broad.mit.edu/annotation/genome/saccharomyces_cerevisiae/Home.html">http://www.broad.mit.edu/annotation/genome/saccharomyces_cerevisiae/Home.html</a>), respectively.</p>                            <h3 scrollto-destination=7937726 id="7937726" class="section-title" data-legacy-id=s2b>2.2. Genome sequencing and assembly</h3>
<p>A total of 331 798 clone ends were sequenced by the whole genome shotgun method and from cosmid libraries (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary methods and Supplementary Table S1</a>). The sequences were assembled into contigs (continuous blocks of uninterrupted sequence) and supercontigs (contigs linked by paired forward–reverse reads from the same plasmid) using the Phred<sup><a reveal-id="DSP003C19" data-open="DSP003C19" class="link link-ref link-reveal xref-bibr">19</a></sup> and PCAP<sup><a reveal-id="DSP003C20" data-open="DSP003C20" class="link link-ref link-reveal xref-bibr">20</a></sup> software packages.</p>                            <h3 scrollto-destination=7937731 id="7937731" class="section-title" data-legacy-id=s2c>2.3. Phylogenetic analysis</h3>
<p>A NCBI-BLAST blastp search within <em>S</em>. <em>cerevisiae</em> S288C open reading frames (ORFs) was carried out with default parameters to find potential paralogs ORFs (query; <em>S</em>. <em>cerevisiae</em> ORFs DB; <em>S</em>. <em>cerevisiae</em> ORFs). The ORFs aligned against ORFs with high alignment lengths (match length of at least 50 amino acid) and high identities (≥60%) were listed as potential paralogous ORFs. An <em>S</em>. <em>cerevisiae</em> S288C orthologous ORFs set was prepared by removing the potential paralogous ORFs, and it was used for identification of orthologous ORFs.</p><p>For eight yeast strains (<em>S</em>. <em>pastorianus</em> Weihenstephan 34/70, <em>S</em>. <em>cerevisiae</em> YJM789, RM11-1a, <em>S</em>. <em>bayanus</em> CBS 7001, <em>S</em>. <em>castellii</em> CBS 4309, <em>S</em>. <em>kudriavzevii</em> CBS 8840, <em>S</em>. <em>mikatae</em> CBS 8839 and <em>S</em>. <em>paradoxus</em> CBS 432), the complete set of all predicted ORFs starting with a methionine and having a length of at least 50 amino acids were gathered.</p><p>Orthologous ORF sets for each strain were gathered by picking up bidirectional best hits. As for lager brewing yeast, each orthologous ORF of <em>S</em>. <em>cerevisiae</em> S288C was aligned to the two different ORFs of this yeast: the best matching ORFs were considered as class1 orthologous ORFs, whereas the second best matching ORFs (with scores &gt;60) were considered as class2 orthologous ORFs. Pairwise distances between two strains were calculated using the Kimura two-parameter model using all the aligned ORFs, and phylogenetic trees were constructed using the UPGMA method. The relative distance of the closest species was computed using the neighbor-joining method.</p>                            <h3 scrollto-destination=7937739 id="7937739" class="section-title" data-legacy-id=s2d>2.4. Gene finding and annotations</h3>
<p>ORFs that start with a methionine (ATG codon) and consist of more than 50 amino acids (150 bp) were predicted from the supercontigs of lager brewing yeast. Two sets of ORF data were used as references for gene finding and annotations. One consisted of <em>S</em>. <em>bayanus</em> CBS 7001 ORFs, which were also re-annotated from the genome sequence reported (GenBank accession no. AACG02000000). The other was a set of <em>S</em>. <em>cerevisiae</em> S288C annotated ORFs (SGD; <a class="link link-uri" href="http://www.yeastgenome.org/">http://www.yeastgenome.org/</a>). Each ORF from a given species was used as a query to compare with ORFs from the two other species using NCBI-BLAST blastp. Blast results with scores above 60 were considered as significant hits for each ORF and used for the following steps. First, clusters of the ORF were generated. The 2930 clusters of the lager brewing yeast were composed of two types of ORF, and each ORF cluster was annotated with the functional role according to the descriptions of the <em>S</em>. <em>cerevisiae</em> ORF. In addition, the two sets of ORF in the lager brewing yeast were classified as Sc-type ORF, having higher identity to the <em>S</em>. <em>cerevisiae</em> ORF, and as Sb-type ORF, having higher identity to <em>S</em>. <em>bayanus</em> ORF. Subsequently, the remaining clusters, which had more than two <em>S</em>. <em>cerevisiae</em> ORFs, were annotated by preferentially keeping matches within gene order (synteny) over non-syntenic matches in the supercontigs. The remaining ORFs that were not annotated by the above methods were annotated to best-hit homologs with an identity of ≥40% and marked as non-orthologs. Of the non-orthologs, the ORFs having no homologies with <em>S</em>. <em>cerevisiae</em> S288C or <em>S</em>. <em>bayanus</em> CBS 7001 ORFs were defined as specific for the lager brewing yeast. These ORFs were annotated to the best-hit homologs and their functions were assigned or predicted by BLASTP against the non-redundant database using three thresholds: i.e. an <em>E</em>-value of ≤1 × 10<sup>−5</sup>, an identity of ≥40% and a match length of at least 60% of both protein lengths, and the domains by Pfam using one threshold, i.e. an <em>E</em>-value of ≤1 × 10<sup>−5</sup>. Six mtDNA contigs were obtained and assembled into a circular contig by closing the gaps by further sequencing using specific PCR primers across each gap. Gene annotation of the lager brewing yeast mtDNA genome was carried out using a variation of NCBI-BLAST blastn and blastx against mtDNA genomes of <em>S</em>. <em>cerevisiae</em> and manual curation. The annotation of tRNA genes was performed using the program tRNAscan-SE 1.21.</p>                            <h3 scrollto-destination=7937743 id="7937743" class="section-title" data-legacy-id=s2e>2.5. Identification of chromosomal rearrangements</h3>
<p>Supercontigs, in which either the gene synteny was not conserved with respect to the <em>S</em>. <em>cerevisiae</em> genome or both Sc- and Sb-type ORFs were included, were considered as candidates for chromosomal rearrangements. The regions containing the cosmid clones, which had an Sc-type sequence at one end and an Sb-type sequence at the other end, were considered as candidates for the chromosomal translocations. These putative translocations were confirmed by PCR amplification across the breakpoints using genomic DNA from the lager brewing yeast as a template and by sequencing analyses of the amplicons.</p>                            <h3 scrollto-destination=7937747 id="7937747" class="section-title" data-legacy-id=s2f>2.6. Optical mapping</h3>
<p>Optical mapping (opmap; OpGen, Inc., Madison, WI, USA) was performed as reported earlier<sup><a reveal-id="DSP003C21" data-open="DSP003C21" class="link link-ref link-reveal xref-bibr">21</a></sup> using <em>Xho</em>I restriction endonuclease. To validate and align the assembled sequence by opmap, the predicted lager brewing yeast chromosomal sequences were constructed on the basis of information provided by the assembled sequences, gene annotations and chromosomal rearrangements identified above.</p>                            <h3 scrollto-destination=7937752 id="7937752" class="section-title" data-legacy-id=s2g>2.7. Data deposition</h3>
<p>This Whole Genome Shotgun project has been deposited at DDBJ/EMBL/GenBank under the project accession ABPO00000000. The version described in this paper is the first version, ABPO10000000. The accession number for the mitochondrial genome is EU852811.</p>                            <h2 scrollto-destination=7937756 id="7937756" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=7937757 id="7937757" class="section-title" data-legacy-id=s3a>3.1. Genome assembly</h3>
<p>We generated 7.8-fold redundant coverage in paired end sequences from the whole genome shotgun plasmids and additional cosmid clones. The sequences were assembled into 3184 contigs and 796 supercontigs using Phred<sup><a reveal-id="DSP003C19" data-open="DSP003C19" class="link link-ref link-reveal xref-bibr">19</a></sup> and PCAP<sup><a reveal-id="DSP003C20" data-open="DSP003C20" class="link link-ref link-reveal xref-bibr">20</a></sup> to yield a draft consensus sequence (Table <a reveal-id="DSP003TB1" data-open="DSP003TB1" class="link link-reveal link-table xref-fig">1</a>). The sequence has a long-range continuity (N50 supercontig length of 108 kb) and 0.26 kb sequence gaps; such a sequence is short compared with the average gene length. The sum of the supercontig lengths including gaps is 23.4 Mb, not including the ribosomal DNA (rDNA) locus.</p>                        <a id="7937759" scrollto-destination="7937759"></a>
<div content-id="DSP003TB1" class="table-modal"><div class="table-wrap"><div id="DSP003TB1" class="table-wrap-title" data-id="DSP003TB1"><div class="label -label">Table 1</div><div class="caption"><p>Genome assembly</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Sequence coverage (fold) </td><td>7.8 </td></tr><tr><td>Assembled sequence (Mb)<sup>a</sup> </td><td>22.5 </td></tr><tr><td>Assembled sequence, including gaps (Mb)<sup>b</sup> </td><td>23.4 </td></tr><tr><td>Number of contigs </td><td>3184 </td></tr><tr><td>Number of supercontigs </td><td>796 </td></tr><tr><td>N50 supercontig length (kb) </td><td>108 </td></tr><tr><td>Gaps per 100 kb </td><td>13.2 </td></tr><tr><td>Average gap length (bp) </td><td>258 </td></tr></tbody></table></div><div content-id="DSP003TB1" class="table-modal"><table><tbody><tr><td>Sequence coverage (fold) </td><td>7.8 </td></tr><tr><td>Assembled sequence (Mb)<sup>a</sup> </td><td>22.5 </td></tr><tr><td>Assembled sequence, including gaps (Mb)<sup>b</sup> </td><td>23.4 </td></tr><tr><td>Number of contigs </td><td>3184 </td></tr><tr><td>Number of supercontigs </td><td>796 </td></tr><tr><td>N50 supercontig length (kb) </td><td>108 </td></tr><tr><td>Gaps per 100 kb </td><td>13.2 </td></tr><tr><td>Average gap length (bp) </td><td>258 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Sum of contig lengths.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Sum of supercontig length, including contig lengths and estimated gap sizes between consecutive contigs.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7937759" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP003TB1" class="table-wrap-title" data-id="DSP003TB1"><div class="label -label">Table 1</div><div class="caption"><p>Genome assembly</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Sequence coverage (fold) </td><td>7.8 </td></tr><tr><td>Assembled sequence (Mb)<sup>a</sup> </td><td>22.5 </td></tr><tr><td>Assembled sequence, including gaps (Mb)<sup>b</sup> </td><td>23.4 </td></tr><tr><td>Number of contigs </td><td>3184 </td></tr><tr><td>Number of supercontigs </td><td>796 </td></tr><tr><td>N50 supercontig length (kb) </td><td>108 </td></tr><tr><td>Gaps per 100 kb </td><td>13.2 </td></tr><tr><td>Average gap length (bp) </td><td>258 </td></tr></tbody></table></div><div content-id="DSP003TB1" class="table-modal"><table><tbody><tr><td>Sequence coverage (fold) </td><td>7.8 </td></tr><tr><td>Assembled sequence (Mb)<sup>a</sup> </td><td>22.5 </td></tr><tr><td>Assembled sequence, including gaps (Mb)<sup>b</sup> </td><td>23.4 </td></tr><tr><td>Number of contigs </td><td>3184 </td></tr><tr><td>Number of supercontigs </td><td>796 </td></tr><tr><td>N50 supercontig length (kb) </td><td>108 </td></tr><tr><td>Gaps per 100 kb </td><td>13.2 </td></tr><tr><td>Average gap length (bp) </td><td>258 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Sum of contig lengths.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Sum of supercontig length, including contig lengths and estimated gap sizes between consecutive contigs.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7937759" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7937760 id="7937760" class="section-title" data-legacy-id=s3b>3.2. Phylogenetic relationship</h3>
<p>Previous works based on the sequences of the rDNA internal transcribed spacers (<em>ITS1</em>, <em>ITS2</em>) have led to the definition of a phylogenetic relationship between the <em>Saccharomyces</em> genus in which lager brewing yeast is closely related to <em>S</em>. <em>bayanus</em>.<sup><a reveal-id="DSP003C22" data-open="DSP003C22" class="link link-ref link-reveal xref-bibr">22</a></sup> However, this definition has led to the construction of inaccurate phylogenetic trees because of the hybrid nature of lager brewing yeast. In addition, single gene phylogenies are much more error-prone than multiple gene phylogenies.<sup><a reveal-id="DSP003C23" data-open="DSP003C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Recently, whole genome data from <em>Saccharomyces</em> yeasts have provided a greater resolving power by allowing trees to be constructed based on concatenated sets of genes.<sup><a reveal-id="DSP003C23" data-open="DSP003C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Using this approach, we studied the relationship between lager brewing yeast and the other <em>Saccharomyces</em> yeasts.</p><p>A set of 2080 ORFs among class1 and 2 of the lager brewing yeast and eight <em>Saccharomyces</em> strains were compared and concatenated for phylogenetic analysis (see Materials and methods). As expected, the constructed phylogenetic tree showed that class1 and 2 were close to <em>S</em>. <em>cerevisiae</em> and <em>S</em>. <em>bayanus</em>, respectively (Fig. <a reveal-id="DSP003F1" data-open="DSP003F1" class="link link-reveal link-table xref-fig">1</a>). This result supported the hypothesis that the lager brewing yeast is a hybrid between the two species, <em>S</em>. <em>cerevisiae</em> and <em>S</em>. <em>bayanus</em> at genomic level. However, evolutionarily, the set of class2 ORFs of the lager brewing yeast was clearly farther apart from <em>S</em>. <em>bayanus</em> CBS 7001 in comparison with the class1 ORFs and the <em>S</em>. <em>cerevisae</em> strains.</p>                        <a id="7937763" scrollto-destination="7937763"></a>
<div id="DSP003F1" data-id="DSP003F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="" alt="Phylogenetic tree of class 1 and 2 of the larger brewing yeast and eight yeast species. Branch lengths denote the number of substitutions per base." path-from-xml="dsp00301" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7937763/dsp00301" path-from-xml="dsp00301" target="_blank">View large</a><a section="7937763"  class="download-slide" path-from-xml="dsp00301" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of class 1 and 2 of the larger brewing yeast and eight yeast species. Branch lengths denote the number of substitutions per base.</p></div></div><div content-id="DSP003F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="" alt="Phylogenetic tree of class 1 and 2 of the larger brewing yeast and eight yeast species. Branch lengths denote the number of substitutions per base." path-from-xml="dsp00301" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7937763/dsp00301" path-from-xml="dsp00301" target="_blank">View large</a><a section="7937763"  class="download-slide" path-from-xml="dsp00301" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic tree of class 1 and 2 of the larger brewing yeast and eight yeast species. Branch lengths denote the number of substitutions per base.</p></div></div><p>To explain the distance between the class2 ORFs of the lager brewing yeast and <em>S</em>. <em>bayanus</em>, it is important to highlight that the <em>S</em>. <em>bayanus</em> species is taxonomically very complex. It contains at least two sub-groups of strains:<sup><a reveal-id="DSP003C24" data-open="DSP003C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSP003C25" data-open="DSP003C25" class="link link-ref link-reveal xref-bibr">25</a></sup><em>S</em>. <em>bayanus</em> var. <em>bayanus</em> represented by a variety of strains not currently isolated from natural environment and <em>S</em>. <em>bayanus</em> var. <em>uvarum</em>, commonly known as <em>Saccharomyces uvarum</em>, represented by strains that are typically and abundantly isolated from the enological environment. The only <em>S</em>. <em>bayanus</em> strain that has currently been fully sequenced and can therefore be used as a reference in comparative genomic studies is strain CBS 7001, which belong to the <em>S</em>. <em>uvarum</em> group.<sup><a reveal-id="DSP003C26" data-open="DSP003C26" class="link link-ref link-reveal xref-bibr">26</a></sup> It is reasonable to conclude that this strain is not an ideal reference for studying the lager brewing yeast genome; it is simply the best approximation available so far. Some of the main differences between this yeast and the corresponding sub-genome of the lager brewing yeast are described in this study.</p>                            <h3 scrollto-destination=7937765 id="7937765" class="section-title" data-legacy-id=s3c>3.3. Chromosomal structure</h3>
<p>After clarifying that lager brewing yeast was a hybrid between <em>S</em>. <em>cerevisiae</em> and <em>S</em>. <em>bayanus</em>, we annotated the ORFs of the lager brewing yeast and produced a large-scale alignment of genomic regions by aligning the lager brewing yeast genome with the <em>S</em>. <em>cerevisiae</em> S288C and <em>S</em>. <em>bayanus</em> CBS 7001 genomes (see Materials and methods). The Sb-type ORFs showed clearly lower sequence identities with the corresponding <em>S</em>. <em>bayanus</em> CBS 7001 ORFs (average 92.7%) in comparison with the Sc-type ORFs and the <em>S</em>. <em>cerevisae</em> S288C (99.2%) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>), and it was consistent with the phylogenetic analysis. Most of the supercontigs were classified into two groups, i.e. those consisting entirely of Sc-type ORFs and those consisting entirely of Sb-type ORFs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>); therefore, the two sub-genomes were designated Sc- and Sb-type, respectively. Except for the sub-telomeric regions, large regions of the Sc- and the Sb-type sub-genomes shared conserved synteny with the <em>S</em>. <em>cerevisiae</em> and <em>S</em>. <em>bayanus</em> genomes (Fig. <a reveal-id="DSP003F2" data-open="DSP003F2" class="link link-reveal link-table xref-fig">2</a>). On the sub-telomeric regions, where syntenies were not well conserved, ∼59% of non-orthologs (111/187) were found. It is in these sub-telomeric regions where many important genes for beer brewing (sugar transporters; <em>HXT</em> and <em>MAL</em> families and lectin-like protein involved in flocculation; <em>FLO</em> families etc.) were found (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>).</p>                        <a id="7937767" scrollto-destination="7937767"></a>
<div id="DSP003F2" data-id="DSP003F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="" alt="Chromosomal structure of lager brewing yeast. The types of lager brewing yeast chromosomes are summarized. The black horizontal bars indicate the S. cerevisiae genome with tick marks for every 100 kbp. Matching pairs of translocations between chromosomes with different numbers are indicated by the orientation of the arrowheads with their common numbering. The locations of chromosomal rearrangements (inversions and deletions) relative to the S. cerevisiae genome are indicated by colored triangles, and the locations and numbers of non-orthologs and lager brewing yeast-specific genes are indicated by colored dots. The non-orthologs that are split into more than two genes by stop codon or frameshift mutation are counted as one, and the genes whose characteristics are deemed dubious are not counted. The positions of these marks, whether they are shown above or below the chromosome bars, do not indicate the strand in which those rearrangements or genes are located in the lager brewing yeast genome." path-from-xml="dsp00302" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7937767/dsp00302" path-from-xml="dsp00302" target="_blank">View large</a><a section="7937767"  class="download-slide" path-from-xml="dsp00302" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Chromosomal structure of lager brewing yeast. The types of lager brewing yeast chromosomes are summarized. The black horizontal bars indicate the <em>S</em>. <em>cerevisiae</em> genome with tick marks for every 100 kbp. Matching pairs of translocations between chromosomes with different numbers are indicated by the orientation of the arrowheads with their common numbering. The locations of chromosomal rearrangements (inversions and deletions) relative to the <em>S</em>. <em>cerevisiae</em> genome are indicated by colored triangles, and the locations and numbers of non-orthologs and lager brewing yeast-specific genes are indicated by colored dots. The non-orthologs that are split into more than two genes by stop codon or frameshift mutation are counted as one, and the genes whose characteristics are deemed dubious are not counted. The positions of these marks, whether they are shown above or below the chromosome bars, do not indicate the strand in which those rearrangements or genes are located in the lager brewing yeast genome.</p></div></div><div content-id="DSP003F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="" alt="Chromosomal structure of lager brewing yeast. The types of lager brewing yeast chromosomes are summarized. The black horizontal bars indicate the S. cerevisiae genome with tick marks for every 100 kbp. Matching pairs of translocations between chromosomes with different numbers are indicated by the orientation of the arrowheads with their common numbering. The locations of chromosomal rearrangements (inversions and deletions) relative to the S. cerevisiae genome are indicated by colored triangles, and the locations and numbers of non-orthologs and lager brewing yeast-specific genes are indicated by colored dots. The non-orthologs that are split into more than two genes by stop codon or frameshift mutation are counted as one, and the genes whose characteristics are deemed dubious are not counted. The positions of these marks, whether they are shown above or below the chromosome bars, do not indicate the strand in which those rearrangements or genes are located in the lager brewing yeast genome." path-from-xml="dsp00302" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7937767/dsp00302" path-from-xml="dsp00302" target="_blank">View large</a><a section="7937767"  class="download-slide" path-from-xml="dsp00302" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Chromosomal structure of lager brewing yeast. The types of lager brewing yeast chromosomes are summarized. The black horizontal bars indicate the <em>S</em>. <em>cerevisiae</em> genome with tick marks for every 100 kbp. Matching pairs of translocations between chromosomes with different numbers are indicated by the orientation of the arrowheads with their common numbering. The locations of chromosomal rearrangements (inversions and deletions) relative to the <em>S</em>. <em>cerevisiae</em> genome are indicated by colored triangles, and the locations and numbers of non-orthologs and lager brewing yeast-specific genes are indicated by colored dots. The non-orthologs that are split into more than two genes by stop codon or frameshift mutation are counted as one, and the genes whose characteristics are deemed dubious are not counted. The positions of these marks, whether they are shown above or below the chromosome bars, do not indicate the strand in which those rearrangements or genes are located in the lager brewing yeast genome.</p></div></div><p>In addition to the sub-telomeric rearrangements, several internal rearrangements were found in comparison with <em>S</em>. <em>cerevisiae</em> S288C (Fig. <a reveal-id="DSP003F2" data-open="DSP003F2" class="link link-reveal link-table xref-fig">2</a>). One and three inversions were found in Sc- and Sb-type sub-genomes, respectively (Fig. <a reveal-id="DSP003F2" data-open="DSP003F2" class="link link-reveal link-table xref-fig">2</a>). The Sc-type sub-genome showed an inversion of 30 kb (570–600 kb of Sc-type chromosome XIV) (Fig. <a reveal-id="DSP003F2" data-open="DSP003F2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3A</a>). The sequence of <em>S</em>. <em>cerevisiae</em> RM11-1a<sup><a reveal-id="DSP003C27" data-open="DSP003C27" class="link link-ref link-reveal xref-bibr">27</a></sup> and YJM789<sup><a reveal-id="DSP003C28" data-open="DSP003C28" class="link link-ref link-reveal xref-bibr">28</a></sup> revealed that the same region was also inverted in comparison with <em>S</em>. <em>cerevisiae</em> S288C. Two of the inversions found in the Sb-type sub-genome (117–133 and 164–181 kb of Sb-type chromosomes V and VI, respectively) (Fig. <a reveal-id="DSP003F2" data-open="DSP003F2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3A</a>) have also been identified in <em>S</em>. <em>bayanus</em> CBS 7001.<sup><a reveal-id="DSP003C26" data-open="DSP003C26" class="link link-ref link-reveal xref-bibr">26</a></sup> The remaining inversion spans within the interval between 553 and 560 kb on the Sb-type chromosome XIV. Two translocations were found in the Sc-type sub-genomes. A novel chromosome comprising a combination of Sc-type chromosomes V and XI created by the translocation present in the Ty element was detected, and a fragment corresponding to 32 kb of the Sc-type chromosome XV of <em>S</em>. <em>cerevisiae</em> S288C was also found to be translocated to the Sc-type chromosome XI inside an identical area of 34 bp located between <em>FRE2</em> and <em>FRE3</em> (Table <a reveal-id="DSP003TB2" data-open="DSP003TB2" class="link link-reveal link-table xref-fig">2</a>). On the other hand, two reciprocal translocations (Sb-types II–IV and VIII–XV) were found in the Sb-type sub-genome (Table <a reveal-id="DSP003TB2" data-open="DSP003TB2" class="link link-reveal link-table xref-fig">2</a>); these translocations have been reported previously in lager brewing yeast<sup><a reveal-id="DSP003C6" data-open="DSP003C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSP003C7" data-open="DSP003C7" class="link link-ref link-reveal xref-bibr">7</a></sup> as well as in <em>S</em>. <em>bayanus</em>.<sup><a reveal-id="DSP003C26" data-open="DSP003C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Furthermore, the genome of <em>S</em>. <em>bayanus</em> CBS 7001 has three reciprocal translocations (right arm of II and left arm of II, VI–X and II–XIV).<sup><a reveal-id="DSP003C26" data-open="DSP003C26" class="link link-ref link-reveal xref-bibr">26</a></sup> However, the gene syntenies of the corresponding regions in the Sb-type sub-genome are the same as those of <em>S</em>. <em>cerevisiae</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3C</a>). Eight chimera chromosomes between the Sc- and the Sb-type sub-genomes were found in the lager brewing yeast, indicating a post-hybridization event (Fig. <a reveal-id="DSP003F2" data-open="DSP003F2" class="link link-reveal link-table xref-fig">2</a>). Although the breakpoints of the chromosomal translocations in the <em>Saccharomyces</em> lineage seem to be associated to Ty elements or highly similar pairs of ribosomal protein genes,<sup><a reveal-id="DSP003C29" data-open="DSP003C29" class="link link-ref link-reveal xref-bibr">29</a></sup> those in the lager brewing yeast were found between the orthologous non-ribosomal protein genes, with the exception of chromosome III (Table <a reveal-id="DSP003TB2" data-open="DSP003TB2" class="link link-reveal link-table xref-fig">2</a>). One was a reciprocal translocation between the orthologs, <em>ScYJR009C</em> (ScX) and <em>SbYJR009C</em> (SbX), which share the 96.1% sequence identity over an 840 bp homology region (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). The other seven translocations occurred by non-reciprocal recombinations. The non-reciprocal translocation between ScIII and SbIII was found to be at the <em>MAT</em>-locus, which involved a 604 bp homologous region. The remaining six non-reciprocal translocations (ScVII/SbVII, ScXI/SbXI, ScXIII/SbXIII, ScXVI/SbXVI, ScXVI/SbXVI and ScXVI/SbXVI) occurred between their orthologs ORFs (<em>YGL173C</em>, <em>YKL045W</em>, <em>YMR302C</em>, <em>YPL240C</em>, <em>YPR160W</em> and <em>YPR191W</em>), which share 81–87% sequence identity (Table <a reveal-id="DSP003TB2" data-open="DSP003TB2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). The sequencing of these translocation products showed that exchange occurred inside their identical short regions (8–23 bp) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>).</p>                        <a id="7937769" scrollto-destination="7937769"></a>
<div content-id="DSP003TB2" class="table-modal"><div class="table-wrap"><div id="DSP003TB2" class="table-wrap-title" data-id="DSP003TB2"><div class="label -label">Table 2</div><div class="caption"><p>List of chromosomal translocations</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Translocation </th><th>Breakpoint </th></tr></thead><tbody><tr><td colspan="2">Sc–Sc-type chromosomal translocation </td></tr><tr><td> ScV/ScXI </td><td>ScV Ty LTR &gt; ScXI Ty LTR </td></tr><tr><td> ScXI/ScXV </td><td>Sc<em>YKL220C</em> (<em>FRE2</em>) &gt; Sc<em>YOR381W</em>(<em>FRE3</em>) </td></tr><tr><td colspan="2">Sb–Sb-type chromosomal translocations </td></tr><tr><td> SbII/SbIV </td><td>Sb<em>YBR031W</em> (<em>RPL4A</em>) &gt; Sb<em>YDR012W</em> (<em>RPL4B</em>) </td></tr><tr><td> SbII/SbIV </td><td>Sb<em>YDR012W</em> (<em>RPL4B</em>) &gt; Sb<em>YBR031W</em> (<em>RPL4A</em>) </td></tr><tr><td> SbVIII/SbXV </td><td>Sb<em>YOR018W</em>-Sb<em>YOR019W</em> &gt; Sb<em>YHR014W</em>-Sb<em>YHR015W</em> </td></tr><tr><td> SbVIII/SbXV </td><td>Sb<em>YHR014W</em>-Sb<em>YHR015W</em> &gt; Sb<em>YOR018W</em>-Sb<em>YOR019W</em> </td></tr><tr><td colspan="2">Sc–Sb-type chromosomal translocations </td></tr><tr><td> ScIII/SbIII </td><td>SbMAT locus &gt; ScMAT locus </td></tr><tr><td> ScVII/SbVII </td><td>Sc<em>YGL173C</em> (<em>KEM1</em>) &gt; Sb<em>YGL173C</em> (<em>KEM1</em>) </td></tr><tr><td> ScX/SbX </td><td>Sc<em>YJR009C</em> (<em>TDH2</em>) &gt; Sb<em>YJR009C</em> (<em>TDH2</em>) </td></tr><tr><td> ScX/SbX </td><td>Sb<em>YJR009C</em> (<em>TDH2</em>) &gt; Sc<em>YJR009C</em> (<em>TDH2</em>) </td></tr><tr><td> ScXI/SbXI </td><td>Sc<em>YKL045W</em> (<em>PRI2</em>) &gt; Sb<em>YKL045W</em> (<em>PRI2</em>) </td></tr><tr><td> ScXIII/SbXIII </td><td>Sb<em>YMR302C</em> (<em>YME2</em>) &gt; Sc<em>YMR302C</em> (<em>YME2</em>) </td></tr><tr><td> ScXVI/SbXVI </td><td>Sc<em>YPL240C</em> (<em>HSP82</em>) &gt; Sb<em>YPL240C</em> (<em>HSP82</em>) </td></tr><tr><td> ScXVI/SbXVI </td><td>Sc<em>YPR160W</em> (<em>GPH1</em>) &gt; Sb<em>YPR160W</em> (<em>GPH1</em>) </td></tr><tr><td> ScXVI/SbXVI </td><td>Sb<em>YPR191W</em> (<em>QCR2</em>) &gt; Sc<em>YPR191W</em> (<em>QCR2</em>) </td></tr></tbody></table></div><div content-id="DSP003TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Translocation </th><th>Breakpoint </th></tr></thead><tbody><tr><td colspan="2">Sc–Sc-type chromosomal translocation </td></tr><tr><td> ScV/ScXI </td><td>ScV Ty LTR &gt; ScXI Ty LTR </td></tr><tr><td> ScXI/ScXV </td><td>Sc<em>YKL220C</em> (<em>FRE2</em>) &gt; Sc<em>YOR381W</em>(<em>FRE3</em>) </td></tr><tr><td colspan="2">Sb–Sb-type chromosomal translocations </td></tr><tr><td> SbII/SbIV </td><td>Sb<em>YBR031W</em> (<em>RPL4A</em>) &gt; Sb<em>YDR012W</em> (<em>RPL4B</em>) </td></tr><tr><td> SbII/SbIV </td><td>Sb<em>YDR012W</em> (<em>RPL4B</em>) &gt; Sb<em>YBR031W</em> (<em>RPL4A</em>) </td></tr><tr><td> SbVIII/SbXV </td><td>Sb<em>YOR018W</em>-Sb<em>YOR019W</em> &gt; Sb<em>YHR014W</em>-Sb<em>YHR015W</em> </td></tr><tr><td> SbVIII/SbXV </td><td>Sb<em>YHR014W</em>-Sb<em>YHR015W</em> &gt; Sb<em>YOR018W</em>-Sb<em>YOR019W</em> </td></tr><tr><td colspan="2">Sc–Sb-type chromosomal translocations </td></tr><tr><td> ScIII/SbIII </td><td>SbMAT locus &gt; ScMAT locus </td></tr><tr><td> ScVII/SbVII </td><td>Sc<em>YGL173C</em> (<em>KEM1</em>) &gt; Sb<em>YGL173C</em> (<em>KEM1</em>) </td></tr><tr><td> ScX/SbX </td><td>Sc<em>YJR009C</em> (<em>TDH2</em>) &gt; Sb<em>YJR009C</em> (<em>TDH2</em>) </td></tr><tr><td> ScX/SbX </td><td>Sb<em>YJR009C</em> (<em>TDH2</em>) &gt; Sc<em>YJR009C</em> (<em>TDH2</em>) </td></tr><tr><td> ScXI/SbXI </td><td>Sc<em>YKL045W</em> (<em>PRI2</em>) &gt; Sb<em>YKL045W</em> (<em>PRI2</em>) </td></tr><tr><td> ScXIII/SbXIII </td><td>Sb<em>YMR302C</em> (<em>YME2</em>) &gt; Sc<em>YMR302C</em> (<em>YME2</em>) </td></tr><tr><td> ScXVI/SbXVI </td><td>Sc<em>YPL240C</em> (<em>HSP82</em>) &gt; Sb<em>YPL240C</em> (<em>HSP82</em>) </td></tr><tr><td> ScXVI/SbXVI </td><td>Sc<em>YPR160W</em> (<em>GPH1</em>) &gt; Sb<em>YPR160W</em> (<em>GPH1</em>) </td></tr><tr><td> ScXVI/SbXVI </td><td>Sb<em>YPR191W</em> (<em>QCR2</em>) &gt; Sc<em>YPR191W</em> (<em>QCR2</em>) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7937769" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP003TB2" class="table-wrap-title" data-id="DSP003TB2"><div class="label -label">Table 2</div><div class="caption"><p>List of chromosomal translocations</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Translocation </th><th>Breakpoint </th></tr></thead><tbody><tr><td colspan="2">Sc–Sc-type chromosomal translocation </td></tr><tr><td> ScV/ScXI </td><td>ScV Ty LTR &gt; ScXI Ty LTR </td></tr><tr><td> ScXI/ScXV </td><td>Sc<em>YKL220C</em> (<em>FRE2</em>) &gt; Sc<em>YOR381W</em>(<em>FRE3</em>) </td></tr><tr><td colspan="2">Sb–Sb-type chromosomal translocations </td></tr><tr><td> SbII/SbIV </td><td>Sb<em>YBR031W</em> (<em>RPL4A</em>) &gt; Sb<em>YDR012W</em> (<em>RPL4B</em>) </td></tr><tr><td> SbII/SbIV </td><td>Sb<em>YDR012W</em> (<em>RPL4B</em>) &gt; Sb<em>YBR031W</em> (<em>RPL4A</em>) </td></tr><tr><td> SbVIII/SbXV </td><td>Sb<em>YOR018W</em>-Sb<em>YOR019W</em> &gt; Sb<em>YHR014W</em>-Sb<em>YHR015W</em> </td></tr><tr><td> SbVIII/SbXV </td><td>Sb<em>YHR014W</em>-Sb<em>YHR015W</em> &gt; Sb<em>YOR018W</em>-Sb<em>YOR019W</em> </td></tr><tr><td colspan="2">Sc–Sb-type chromosomal translocations </td></tr><tr><td> ScIII/SbIII </td><td>SbMAT locus &gt; ScMAT locus </td></tr><tr><td> ScVII/SbVII </td><td>Sc<em>YGL173C</em> (<em>KEM1</em>) &gt; Sb<em>YGL173C</em> (<em>KEM1</em>) </td></tr><tr><td> ScX/SbX </td><td>Sc<em>YJR009C</em> (<em>TDH2</em>) &gt; Sb<em>YJR009C</em> (<em>TDH2</em>) </td></tr><tr><td> ScX/SbX </td><td>Sb<em>YJR009C</em> (<em>TDH2</em>) &gt; Sc<em>YJR009C</em> (<em>TDH2</em>) </td></tr><tr><td> ScXI/SbXI </td><td>Sc<em>YKL045W</em> (<em>PRI2</em>) &gt; Sb<em>YKL045W</em> (<em>PRI2</em>) </td></tr><tr><td> ScXIII/SbXIII </td><td>Sb<em>YMR302C</em> (<em>YME2</em>) &gt; Sc<em>YMR302C</em> (<em>YME2</em>) </td></tr><tr><td> ScXVI/SbXVI </td><td>Sc<em>YPL240C</em> (<em>HSP82</em>) &gt; Sb<em>YPL240C</em> (<em>HSP82</em>) </td></tr><tr><td> ScXVI/SbXVI </td><td>Sc<em>YPR160W</em> (<em>GPH1</em>) &gt; Sb<em>YPR160W</em> (<em>GPH1</em>) </td></tr><tr><td> ScXVI/SbXVI </td><td>Sb<em>YPR191W</em> (<em>QCR2</em>) &gt; Sc<em>YPR191W</em> (<em>QCR2</em>) </td></tr></tbody></table></div><div content-id="DSP003TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Translocation </th><th>Breakpoint </th></tr></thead><tbody><tr><td colspan="2">Sc–Sc-type chromosomal translocation </td></tr><tr><td> ScV/ScXI </td><td>ScV Ty LTR &gt; ScXI Ty LTR </td></tr><tr><td> ScXI/ScXV </td><td>Sc<em>YKL220C</em> (<em>FRE2</em>) &gt; Sc<em>YOR381W</em>(<em>FRE3</em>) </td></tr><tr><td colspan="2">Sb–Sb-type chromosomal translocations </td></tr><tr><td> SbII/SbIV </td><td>Sb<em>YBR031W</em> (<em>RPL4A</em>) &gt; Sb<em>YDR012W</em> (<em>RPL4B</em>) </td></tr><tr><td> SbII/SbIV </td><td>Sb<em>YDR012W</em> (<em>RPL4B</em>) &gt; Sb<em>YBR031W</em> (<em>RPL4A</em>) </td></tr><tr><td> SbVIII/SbXV </td><td>Sb<em>YOR018W</em>-Sb<em>YOR019W</em> &gt; Sb<em>YHR014W</em>-Sb<em>YHR015W</em> </td></tr><tr><td> SbVIII/SbXV </td><td>Sb<em>YHR014W</em>-Sb<em>YHR015W</em> &gt; Sb<em>YOR018W</em>-Sb<em>YOR019W</em> </td></tr><tr><td colspan="2">Sc–Sb-type chromosomal translocations </td></tr><tr><td> ScIII/SbIII </td><td>SbMAT locus &gt; ScMAT locus </td></tr><tr><td> ScVII/SbVII </td><td>Sc<em>YGL173C</em> (<em>KEM1</em>) &gt; Sb<em>YGL173C</em> (<em>KEM1</em>) </td></tr><tr><td> ScX/SbX </td><td>Sc<em>YJR009C</em> (<em>TDH2</em>) &gt; Sb<em>YJR009C</em> (<em>TDH2</em>) </td></tr><tr><td> ScX/SbX </td><td>Sb<em>YJR009C</em> (<em>TDH2</em>) &gt; Sc<em>YJR009C</em> (<em>TDH2</em>) </td></tr><tr><td> ScXI/SbXI </td><td>Sc<em>YKL045W</em> (<em>PRI2</em>) &gt; Sb<em>YKL045W</em> (<em>PRI2</em>) </td></tr><tr><td> ScXIII/SbXIII </td><td>Sb<em>YMR302C</em> (<em>YME2</em>) &gt; Sc<em>YMR302C</em> (<em>YME2</em>) </td></tr><tr><td> ScXVI/SbXVI </td><td>Sc<em>YPL240C</em> (<em>HSP82</em>) &gt; Sb<em>YPL240C</em> (<em>HSP82</em>) </td></tr><tr><td> ScXVI/SbXVI </td><td>Sc<em>YPR160W</em> (<em>GPH1</em>) &gt; Sb<em>YPR160W</em> (<em>GPH1</em>) </td></tr><tr><td> ScXVI/SbXVI </td><td>Sb<em>YPR191W</em> (<em>QCR2</em>) &gt; Sc<em>YPR191W</em> (<em>QCR2</em>) </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7937769" target="_blank">View Large</a></div></div><p>Translocations usually occur between sequences at identical positions on homologous chromosomes in order to repair DNA for chromosome disjunction.<sup><a reveal-id="DSP003C30" data-open="DSP003C30" class="link link-ref link-reveal xref-bibr">30</a>,<a reveal-id="DSP003C31" data-open="DSP003C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Abbreviated translocations have been shown to be common in <em>S</em>. <em>cerevisiae</em>, but most of them are mediated through long regions of homologous Ty or sub-telomeric Y′ element recombination.<sup><a reveal-id="DSP003C32" data-open="DSP003C32" class="link link-ref link-reveal xref-bibr">32</a></sup> However, in the present study, we observed that most of the chromosomal translocations between the Sc- and the Sb-type chromosomes occurred between the short homologous regions of orthologous genes, which showed 81–96% sequence divergences. Sequence divergence acts as a potent barrier for translocations. The translocations found in the lager brewing yeast are rare since the recombination rates for diverged sequences are 2.2 × 10<sup>−9</sup> (82% sequence divergence) and 5.4 × 10<sup>−8</sup> (96%) in <em>Saccharomyces</em> yeast.<sup><a reveal-id="DSP003C33" data-open="DSP003C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The frequencies are increased in double-strand breaks (DBSs). When DSBs were induced by ionizing radiation in the diploid <em>S</em>. <em>cerevisiae</em> genome, a translocation was found between the short homology region (26 bp) of the two homologs (90.7% sequence identity).<sup><a reveal-id="DSP003C34" data-open="DSP003C34" class="link link-ref link-reveal xref-bibr">34</a></sup> Thus, it seems that the translocations found between Sc- and Sb-type chromosomes were mediated by the fortuitous appearance of broken chromosomes ends produced by DSBs in either of the two orthologs.</p><p>An optical mapping (opmap)<sup><a reveal-id="DSP003C21" data-open="DSP003C21" class="link link-ref link-reveal xref-bibr">21</a></sup> using <em>Xho</em>I restriction endonuclease was performed to determine the genome size and chromosomes number. The DNA from the lager brewing yeast yielded an optical map of 36 chromosomes of ∼26.1 Mb, not including the rDNA cluster (Table <a reveal-id="DSP003TB3" data-open="DSP003TB3" class="link link-reveal link-table xref-fig">3</a>). This genome size and the number of chromosomes are more than twice those of <em>S</em>. <em>cerevisiae</em> S288C (16 chromosomes and 12.1 Mb, respectively). In order to validate the predicted chromosomal structures, 36 chromosomal DNA sequences were constructed by connecting supercontigs without gaps (as shown in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>) and then compared with the opmap. All the 36 sequences showed clear one-to-one matches to the 36 corresponding chromosomes found by the opmap (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). The total size of the 36 chromosomal DNA sequences was 25.0 Mb, and hence, the coverage was 95.8% of the estimated genome size, as determined by opmap (Table <a reveal-id="DSP003TB3" data-open="DSP003TB3" class="link link-reveal link-table xref-fig">3</a>). On the basis of these results, we conclude that the lager brewing yeast has 36 chromosomes, whereas its parental species, <em>S</em>. <em>cerevisiae</em> and <em>S</em>. <em>bayanus</em>, have 16 chromosomes each.</p>                        <a id="7937772" scrollto-destination="7937772"></a>
<div content-id="DSP003TB3" class="table-modal"><div class="table-wrap"><div id="DSP003TB3" class="table-wrap-title" data-id="DSP003TB3"><div class="label -label">Table 3</div><div class="caption"><p>Opmap and comparison with the predicted chromosome</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Number of chromosomes found by the opmap </td><td>36 </td></tr><tr><td>Total size of the opmap chromosomes (Mb) </td><td>26.1 </td></tr><tr><td>Number of the predicted chromosomes </td><td>36 </td></tr><tr><td>Total size of the predicted chromosomes (Mb) </td><td>25.0 </td></tr><tr><td>Genome coverage (%) </td><td>95.8 </td></tr></tbody></table></div><div content-id="DSP003TB3" class="table-modal"><table><tbody><tr><td>Number of chromosomes found by the opmap </td><td>36 </td></tr><tr><td>Total size of the opmap chromosomes (Mb) </td><td>26.1 </td></tr><tr><td>Number of the predicted chromosomes </td><td>36 </td></tr><tr><td>Total size of the predicted chromosomes (Mb) </td><td>25.0 </td></tr><tr><td>Genome coverage (%) </td><td>95.8 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7937772" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP003TB3" class="table-wrap-title" data-id="DSP003TB3"><div class="label -label">Table 3</div><div class="caption"><p>Opmap and comparison with the predicted chromosome</p></div> </div><div class="table-overflow"><table><tbody><tr><td>Number of chromosomes found by the opmap </td><td>36 </td></tr><tr><td>Total size of the opmap chromosomes (Mb) </td><td>26.1 </td></tr><tr><td>Number of the predicted chromosomes </td><td>36 </td></tr><tr><td>Total size of the predicted chromosomes (Mb) </td><td>25.0 </td></tr><tr><td>Genome coverage (%) </td><td>95.8 </td></tr></tbody></table></div><div content-id="DSP003TB3" class="table-modal"><table><tbody><tr><td>Number of chromosomes found by the opmap </td><td>36 </td></tr><tr><td>Total size of the opmap chromosomes (Mb) </td><td>26.1 </td></tr><tr><td>Number of the predicted chromosomes </td><td>36 </td></tr><tr><td>Total size of the predicted chromosomes (Mb) </td><td>25.0 </td></tr><tr><td>Genome coverage (%) </td><td>95.8 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7937772" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7937773 id="7937773" class="section-title" data-legacy-id=s3d>3.4. rDNA locus</h3>
<p>The rDNA of <em>S</em>. <em>cerevisiae</em> is a region of ∼1.5 Mb, consisting of 150 tandem copies of a 9.1 kb repeat containing the ITS and is located on the right arm of chromosome XII. The lager brewing yeast we analyzed has two types of chromosomes XII and two types of ITS sequence: one is closely related to <em>S</em>. <em>cerevisiae</em> and the other to <em>S</em>. <em>bayanus</em> (Fig <a reveal-id="DSP003F2" data-open="DSP003F2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). Strains <em>S</em>. <em>pastorianus</em> CBS 1538 and <em>S</em>. <em>carlsbergensis</em> CBS 1513, historical lager brewing yeasts, not currently used in beer manufacturing, show only the Sb-type ITS sequence. Comparative genomic hybridization (CGH) of these two strains with <em>S</em>. <em>cerevisiae</em> yeast DNA microarrays revealed, in fact, that the Sc-type chromosome XII has been lost in both strains.<sup><a reveal-id="DSP003C35" data-open="DSP003C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Although the length of the rDNA region on the Sc-type chromosome XII was estimated to be &gt;350 kb, that on the Sb-type chromosome XII, according to the opmap and PCR amplification, was of 18 kb (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2 and Table S3C</a>). This result indicates that the rDNA region on the Sb-type chromosome XII decreased massively. Analysis carried out by genomic <em>in situ</em> hybridization with labeled rDNA sequences in the alloploid grass, <em>Zingeria trichopoda</em>, also revealed that the 45S rDNA from one of the parental genomes had undergone significant reduction.<sup><a reveal-id="DSP003C36" data-open="DSP003C36" class="link link-ref link-reveal xref-bibr">36</a></sup></p>                            <h3 scrollto-destination=7937775 id="7937775" class="section-title" data-legacy-id=s3e>3.5. Structure of the mitochondrial genome</h3>
<p>The mitochondrial DNA (mtDNA) of the lager brewing yeast was also sequenced and assembled into a circular contig. The size of the mtDNA was 70 578 bp, somewhat smaller than that of <em>S</em>. <em>cerevisiae</em> S288C (85 779 bp) and slightly larger than that of <em>S</em>. <em>bayanus</em> (64.3 kb) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). The mtDNA of the lager brewing yeast was also found to contain several differences (gene order, number of exons and intron–exon boundaries of <em>COX1</em>, <em>COB1</em> and 21S<em>rRNA</em>) in comparison with <em>S</em>. <em>cerevisiae</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). On the other hand, it showed many aspects of similarity with the mtDNA of <em>S</em>. <em>bayanus</em>. The gene order was the same as that of <em>S</em>. <em>bayanus</em>,<sup><a reveal-id="DSP003C37" data-open="DSP003C37" class="link link-ref link-reveal xref-bibr">37</a></sup> and it was completely consistent with the results of mapping obtained with restriction enzymes.<sup><a reveal-id="DSP003C15" data-open="DSP003C15" class="link link-ref link-reveal xref-bibr">15</a></sup> In addition, all mtDNA genes exhibited a higher sequence similarity with the corresponding genes of <em>S</em>. <em>bayanus</em> than with those of <em>S</em>. <em>cerevisiae</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). We concluded that the mtDNA of the lager brewing yeast was inherited from its Sb-type ancestor without shuffling between the Sc- and the Sb-type mtDNA genomes.</p>                            <h3 scrollto-destination=7937777 id="7937777" class="section-title" data-legacy-id=s3f>3.6. ORF deletion</h3>
<p>Twenty-eight and 33 ORFs were found to have been shortened (truncated) by frameshift or stop codon mutation in the Sc- and Sb-type sub-genomes, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>), whereas the ORF lengths between <em>Saccharomyces sensu stricto</em> are quite conserved.<sup><a reveal-id="DSP003C26" data-open="DSP003C26" class="link link-ref link-reveal xref-bibr">26</a></sup> This can be expected as the result of a yeast having two functional copies of each gene that releases the evolutionary pressure on one copy. Of these ORFs, three were found to be essential by a large-scale deletion study in <em>S</em>. <em>cerevisiae</em>,<sup><a reveal-id="DSP003C38" data-open="DSP003C38" class="link link-ref link-reveal xref-bibr">38</a></sup> indicating that they must have been truncated after the interspecies hybridization event. In addition, some ORFs involved in the characteristics of lager brewing yeast were also found to be truncated (see below).</p>                            <h3 scrollto-destination=7937779 id="7937779" class="section-title" data-legacy-id=s3g>3.7. Maltose and maltotriose uptake genes</h3>
<p>Maltose and maltotriose are the most abundant fermentable sugars in wort, and the rate at which brewing yeast takes up these sugars can have a major impact on the fermentation performance. The α-glucoside transporter genes, with nine predicted transmembrane domains, are responsible for the uptake of maltose and maltotriose. Increasing the expression of the α-glucoside transporters have been shown to increase the rate of maltose fermentation.<sup><a reveal-id="DSP003C39" data-open="DSP003C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Three α-glucoside transporter genes in each Sc- and Sb-type sub-genome were found on seven loci (Table <a reveal-id="DSP003TB4" data-open="DSP003TB4" class="link link-reveal link-table xref-fig">4</a>). LBYG03039 was identical to <em>MTT1</em>, which encodes an efficient maltotriose transporter in lager brewing yeasts,<sup><a reveal-id="DSP003C40" data-open="DSP003C40" class="link link-ref link-reveal xref-bibr">40</a></sup> and was found to be on both chromosomes ScVII and SbVII (Table <a reveal-id="DSP003TB4" data-open="DSP003TB4" class="link link-reveal link-table xref-fig">4</a>). This indicates that the gene copy number of <em>MTT1</em> in the lager brewing yeast has increased thanks to the chromosomal translocation. Another α-glucoside transporter gene (<em>MPH3</em>), which was found in <em>S</em>. <em>cerevisiae</em> S288C, was not found in the lager brewing yeast genome. This is consistent with previous studies carried out with Southern blotting.<sup><a reveal-id="DSP003C41" data-open="DSP003C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DSP003C42" data-open="DSP003C42" class="link link-ref link-reveal xref-bibr">42</a></sup> LBYG09472 (Sc-type <em>AGT1</em>) and LBYG14145 (Sb-type <em>MAL31</em>) showed truncated forms having only two and three predicted transmembrane domains and are, hence, unlikely to be functional. In total, six α-glucoside transporter genes were found in seven chromosomal regions in the lager brewing yeast sequenced in this work, but two of these were truncated.</p>                        <a id="7937781" scrollto-destination="7937781"></a>
<div content-id="DSP003TB4" class="table-modal"><div class="table-wrap"><div id="DSP003TB4" class="table-wrap-title" data-id="DSP003TB4"><div class="label -label">Table 4</div><div class="caption"><p>Genes related to lager brewing yeast characteristics</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene </th><th>Type </th><th>Location </th><th>Length (LBYG) (aa) </th><th>Predicted Protein </th><th>Systematic Name </th><th>Standard Name </th><th>Length (S288C) (aa) </th><th>Identity (%) </th></tr></thead><tbody><tr><td colspan="9">α-glucoside transporter genes </td></tr><tr><td>LBYG00616 </td><td>Sc </td><td>ScII right sub-telomere </td><td>614 </td><td>α-glucoside transporter </td><td><em>YBR298C</em> </td><td><em>MAL31</em> </td><td>614 </td><td>98 </td></tr><tr><td>LBYG03039 </td><td>Sb </td><td>ScVII right sub-telomere<br />SbVII right sub-telomere </td><td>615 </td><td>α-glucoside transporter </td><td><em>YBR298C</em> </td><td><em>MAL31</em> </td><td>614 </td><td>91 </td></tr><tr><td>LBYG13187 </td><td>Sb </td><td>SbXV-VIII left sub-telomere </td><td>610 </td><td>α-glucoside transporter </td><td><em>YGR289C</em> </td><td><em>AGT1</em> </td><td>616 </td><td>85 </td></tr><tr><td>LBYG00811-00812<sup>a</sup> </td><td>Sc </td><td>ScIV left sub-telomere </td><td>602 </td><td>α-glucoside transporter </td><td><em>YDL247W</em> </td><td><em>MPH2</em> </td><td>609 </td><td>94 </td></tr><tr><td>LBYG09472<sup>a</sup> </td><td>Sb </td><td>SbV right sub-telomere </td><td>231 (tr) </td><td>α-glucoside transporter </td><td><em>YBR298C</em> </td><td><em>MAL31</em> </td><td>614 </td><td>80 </td></tr><tr><td>LBYG14145<sup>a</sup> </td><td>Sc </td><td>Unkown </td><td>321 (tr) </td><td>α-glucoside transporter </td><td><em>YGR289C</em> </td><td><em>AGT1</em> </td><td>616 </td><td>96 </td></tr><tr><td colspan="9">Genes related to the sulfite production pathway </td></tr><tr><td>LBYG00610 </td><td>Sc </td><td>ScII right sub-telomere </td><td>183 (tr) </td><td>High affinity sulfate permease </td><td><em>YBR294W</em> </td><td><em>SUL1</em> </td><td>859 </td><td>100 </td></tr><tr><td>LBYG09125 </td><td>Sb </td><td>SbIV-II right sub-telomere </td><td>295 (tr) </td><td>High affinity sulfate permease </td><td><em>YBR294W</em> </td><td><em>SUL1</em> </td><td>859 </td><td>83 </td></tr><tr><td>LBYG04711-04712<sup>a</sup> </td><td>Sc </td><td>ScXII </td><td>893 </td><td>High affinity sulfate permease </td><td><em>YLR092W</em> </td><td><em>SUL2</em> </td><td>893 </td><td>99 </td></tr><tr><td>LBYG11823 </td><td>Sb </td><td>ScXII </td><td>893 </td><td>High affinity sulfate permease </td><td><em>YLR092W</em> </td><td><em>SUL2</em> </td><td>893 </td><td>87 </td></tr><tr><td>LBYG03911 </td><td>Sc </td><td>ScX<br />SbX-ScX </td><td>511 </td><td>ATP sulfurylase </td><td><em>YJR010W</em> </td><td><em>MET3</em> </td><td>511 </td><td>100 </td></tr><tr><td>LBYG11111 </td><td>Sb </td><td>SbX<br />ScX-SbX </td><td>511 </td><td>ATP sulfurylase </td><td><em>YJR010W</em> </td><td><em>MET3</em> </td><td>511 </td><td>89 </td></tr><tr><td>LBYG04364 </td><td>Sc </td><td>ScXI<br />ScV-ScXI </td><td>202 </td><td>Adenylylsulfate kinase </td><td><em>YKL001C</em> </td><td><em>MET14</em> </td><td>202 </td><td>100 </td></tr><tr><td>LBYG11543 </td><td>Sb </td><td>SbXI<br />ScXI-SbXI </td><td>202 </td><td>Adenylylsulfate kinase </td><td><em>YKL001C</em> </td><td><em>MET14</em> </td><td>202 </td><td>97 </td></tr><tr><td>LBYG07555 </td><td>Sb </td><td>mosaic-ScXVI<br />mosaic-SbXVI </td><td>261 </td><td>3′-phosphoadenylsulfate reductase </td><td><em>YPR167C</em> </td><td><em>MET16</em> </td><td>261 </td><td>92 </td></tr><tr><td>LBYG07205 </td><td>Sc </td><td>mosaic-ScXVI </td><td>458 </td><td>Plasma membrane sulfite pump </td><td><em>YPL092W</em> </td><td><em>SSU1</em> </td><td>458 </td><td>99 </td></tr><tr><td>LBYG13862 </td><td>Sb </td><td>mosaic-SbXVI </td><td>458 </td><td>Plasma membrane sulfite pump </td><td><em>YPL092W</em> </td><td><em>SSU1</em> </td><td>458 </td><td>79 </td></tr></tbody></table></div><div content-id="DSP003TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene </th><th>Type </th><th>Location </th><th>Length (LBYG) (aa) </th><th>Predicted Protein </th><th>Systematic Name </th><th>Standard Name </th><th>Length (S288C) (aa) </th><th>Identity (%) </th></tr></thead><tbody><tr><td colspan="9">α-glucoside transporter genes </td></tr><tr><td>LBYG00616 </td><td>Sc </td><td>ScII right sub-telomere </td><td>614 </td><td>α-glucoside transporter </td><td><em>YBR298C</em> </td><td><em>MAL31</em> </td><td>614 </td><td>98 </td></tr><tr><td>LBYG03039 </td><td>Sb </td><td>ScVII right sub-telomere<br />SbVII right sub-telomere </td><td>615 </td><td>α-glucoside transporter </td><td><em>YBR298C</em> </td><td><em>MAL31</em> </td><td>614 </td><td>91 </td></tr><tr><td>LBYG13187 </td><td>Sb </td><td>SbXV-VIII left sub-telomere </td><td>610 </td><td>α-glucoside transporter </td><td><em>YGR289C</em> </td><td><em>AGT1</em> </td><td>616 </td><td>85 </td></tr><tr><td>LBYG00811-00812<sup>a</sup> </td><td>Sc </td><td>ScIV left sub-telomere </td><td>602 </td><td>α-glucoside transporter </td><td><em>YDL247W</em> </td><td><em>MPH2</em> </td><td>609 </td><td>94 </td></tr><tr><td>LBYG09472<sup>a</sup> </td><td>Sb </td><td>SbV right sub-telomere </td><td>231 (tr) </td><td>α-glucoside transporter </td><td><em>YBR298C</em> </td><td><em>MAL31</em> </td><td>614 </td><td>80 </td></tr><tr><td>LBYG14145<sup>a</sup> </td><td>Sc </td><td>Unkown </td><td>321 (tr) </td><td>α-glucoside transporter </td><td><em>YGR289C</em> </td><td><em>AGT1</em> </td><td>616 </td><td>96 </td></tr><tr><td colspan="9">Genes related to the sulfite production pathway </td></tr><tr><td>LBYG00610 </td><td>Sc </td><td>ScII right sub-telomere </td><td>183 (tr) </td><td>High affinity sulfate permease </td><td><em>YBR294W</em> </td><td><em>SUL1</em> </td><td>859 </td><td>100 </td></tr><tr><td>LBYG09125 </td><td>Sb </td><td>SbIV-II right sub-telomere </td><td>295 (tr) </td><td>High affinity sulfate permease </td><td><em>YBR294W</em> </td><td><em>SUL1</em> </td><td>859 </td><td>83 </td></tr><tr><td>LBYG04711-04712<sup>a</sup> </td><td>Sc </td><td>ScXII </td><td>893 </td><td>High affinity sulfate permease </td><td><em>YLR092W</em> </td><td><em>SUL2</em> </td><td>893 </td><td>99 </td></tr><tr><td>LBYG11823 </td><td>Sb </td><td>ScXII </td><td>893 </td><td>High affinity sulfate permease </td><td><em>YLR092W</em> </td><td><em>SUL2</em> </td><td>893 </td><td>87 </td></tr><tr><td>LBYG03911 </td><td>Sc </td><td>ScX<br />SbX-ScX </td><td>511 </td><td>ATP sulfurylase </td><td><em>YJR010W</em> </td><td><em>MET3</em> </td><td>511 </td><td>100 </td></tr><tr><td>LBYG11111 </td><td>Sb </td><td>SbX<br />ScX-SbX </td><td>511 </td><td>ATP sulfurylase </td><td><em>YJR010W</em> </td><td><em>MET3</em> </td><td>511 </td><td>89 </td></tr><tr><td>LBYG04364 </td><td>Sc </td><td>ScXI<br />ScV-ScXI </td><td>202 </td><td>Adenylylsulfate kinase </td><td><em>YKL001C</em> </td><td><em>MET14</em> </td><td>202 </td><td>100 </td></tr><tr><td>LBYG11543 </td><td>Sb </td><td>SbXI<br />ScXI-SbXI </td><td>202 </td><td>Adenylylsulfate kinase </td><td><em>YKL001C</em> </td><td><em>MET14</em> </td><td>202 </td><td>97 </td></tr><tr><td>LBYG07555 </td><td>Sb </td><td>mosaic-ScXVI<br />mosaic-SbXVI </td><td>261 </td><td>3′-phosphoadenylsulfate reductase </td><td><em>YPR167C</em> </td><td><em>MET16</em> </td><td>261 </td><td>92 </td></tr><tr><td>LBYG07205 </td><td>Sc </td><td>mosaic-ScXVI </td><td>458 </td><td>Plasma membrane sulfite pump </td><td><em>YPL092W</em> </td><td><em>SSU1</em> </td><td>458 </td><td>99 </td></tr><tr><td>LBYG13862 </td><td>Sb </td><td>mosaic-SbXVI </td><td>458 </td><td>Plasma membrane sulfite pump </td><td><em>YPL092W</em> </td><td><em>SSU1</em> </td><td>458 </td><td>79 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>α-Glucoside transporter genes and genes related to the sulfite production pathway are shown. In <em>Saccharomyces</em> sensu stricto yeast, sulfite is produced in the methionine and cysteine biosynthesis pathway. Sulfate is imported into a cell by sulfate transporters, Sul1p and Sul2p and is converted into APS (adenylylsulfate) by ATP sulfurylase, Met3p. APS is phosphorylated, yielding PAPS (3′-phosphoadenylyl-sulfate) by adenylylsulfate kinase, Met14p. Sulfite is yielded by the reduction of PAPS by 3′-phosphoadenylylsulfate reductase, Met16p. Excess sulfite is passed out of the cell by a sulfite transporter, Ssu1p.</p></div></div><div content-id=""><div class="fn"><p>The Tr in brackets indicates the truncated genes. The genes were aligned against <em>S. cerevisiae</em> gene using Blastp, and each <em>S. cerevisiae</em> gene with the highest score is shown. Non-truncated and truncated protein sequence identities were calculated by the global alignment tool (needle) and local alignment tool, respectively, which were provided by EMBOSS. Genes that were found in lager brewing yeast but in which the specific chromosome could not be decided are shown as ‘unknown’.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Note that the genes were confirmed by PCR amplification and sequencing (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Sequence S1</a>).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7937781" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP003TB4" class="table-wrap-title" data-id="DSP003TB4"><div class="label -label">Table 4</div><div class="caption"><p>Genes related to lager brewing yeast characteristics</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene </th><th>Type </th><th>Location </th><th>Length (LBYG) (aa) </th><th>Predicted Protein </th><th>Systematic Name </th><th>Standard Name </th><th>Length (S288C) (aa) </th><th>Identity (%) </th></tr></thead><tbody><tr><td colspan="9">α-glucoside transporter genes </td></tr><tr><td>LBYG00616 </td><td>Sc </td><td>ScII right sub-telomere </td><td>614 </td><td>α-glucoside transporter </td><td><em>YBR298C</em> </td><td><em>MAL31</em> </td><td>614 </td><td>98 </td></tr><tr><td>LBYG03039 </td><td>Sb </td><td>ScVII right sub-telomere<br />SbVII right sub-telomere </td><td>615 </td><td>α-glucoside transporter </td><td><em>YBR298C</em> </td><td><em>MAL31</em> </td><td>614 </td><td>91 </td></tr><tr><td>LBYG13187 </td><td>Sb </td><td>SbXV-VIII left sub-telomere </td><td>610 </td><td>α-glucoside transporter </td><td><em>YGR289C</em> </td><td><em>AGT1</em> </td><td>616 </td><td>85 </td></tr><tr><td>LBYG00811-00812<sup>a</sup> </td><td>Sc </td><td>ScIV left sub-telomere </td><td>602 </td><td>α-glucoside transporter </td><td><em>YDL247W</em> </td><td><em>MPH2</em> </td><td>609 </td><td>94 </td></tr><tr><td>LBYG09472<sup>a</sup> </td><td>Sb </td><td>SbV right sub-telomere </td><td>231 (tr) </td><td>α-glucoside transporter </td><td><em>YBR298C</em> </td><td><em>MAL31</em> </td><td>614 </td><td>80 </td></tr><tr><td>LBYG14145<sup>a</sup> </td><td>Sc </td><td>Unkown </td><td>321 (tr) </td><td>α-glucoside transporter </td><td><em>YGR289C</em> </td><td><em>AGT1</em> </td><td>616 </td><td>96 </td></tr><tr><td colspan="9">Genes related to the sulfite production pathway </td></tr><tr><td>LBYG00610 </td><td>Sc </td><td>ScII right sub-telomere </td><td>183 (tr) </td><td>High affinity sulfate permease </td><td><em>YBR294W</em> </td><td><em>SUL1</em> </td><td>859 </td><td>100 </td></tr><tr><td>LBYG09125 </td><td>Sb </td><td>SbIV-II right sub-telomere </td><td>295 (tr) </td><td>High affinity sulfate permease </td><td><em>YBR294W</em> </td><td><em>SUL1</em> </td><td>859 </td><td>83 </td></tr><tr><td>LBYG04711-04712<sup>a</sup> </td><td>Sc </td><td>ScXII </td><td>893 </td><td>High affinity sulfate permease </td><td><em>YLR092W</em> </td><td><em>SUL2</em> </td><td>893 </td><td>99 </td></tr><tr><td>LBYG11823 </td><td>Sb </td><td>ScXII </td><td>893 </td><td>High affinity sulfate permease </td><td><em>YLR092W</em> </td><td><em>SUL2</em> </td><td>893 </td><td>87 </td></tr><tr><td>LBYG03911 </td><td>Sc </td><td>ScX<br />SbX-ScX </td><td>511 </td><td>ATP sulfurylase </td><td><em>YJR010W</em> </td><td><em>MET3</em> </td><td>511 </td><td>100 </td></tr><tr><td>LBYG11111 </td><td>Sb </td><td>SbX<br />ScX-SbX </td><td>511 </td><td>ATP sulfurylase </td><td><em>YJR010W</em> </td><td><em>MET3</em> </td><td>511 </td><td>89 </td></tr><tr><td>LBYG04364 </td><td>Sc </td><td>ScXI<br />ScV-ScXI </td><td>202 </td><td>Adenylylsulfate kinase </td><td><em>YKL001C</em> </td><td><em>MET14</em> </td><td>202 </td><td>100 </td></tr><tr><td>LBYG11543 </td><td>Sb </td><td>SbXI<br />ScXI-SbXI </td><td>202 </td><td>Adenylylsulfate kinase </td><td><em>YKL001C</em> </td><td><em>MET14</em> </td><td>202 </td><td>97 </td></tr><tr><td>LBYG07555 </td><td>Sb </td><td>mosaic-ScXVI<br />mosaic-SbXVI </td><td>261 </td><td>3′-phosphoadenylsulfate reductase </td><td><em>YPR167C</em> </td><td><em>MET16</em> </td><td>261 </td><td>92 </td></tr><tr><td>LBYG07205 </td><td>Sc </td><td>mosaic-ScXVI </td><td>458 </td><td>Plasma membrane sulfite pump </td><td><em>YPL092W</em> </td><td><em>SSU1</em> </td><td>458 </td><td>99 </td></tr><tr><td>LBYG13862 </td><td>Sb </td><td>mosaic-SbXVI </td><td>458 </td><td>Plasma membrane sulfite pump </td><td><em>YPL092W</em> </td><td><em>SSU1</em> </td><td>458 </td><td>79 </td></tr></tbody></table></div><div content-id="DSP003TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene </th><th>Type </th><th>Location </th><th>Length (LBYG) (aa) </th><th>Predicted Protein </th><th>Systematic Name </th><th>Standard Name </th><th>Length (S288C) (aa) </th><th>Identity (%) </th></tr></thead><tbody><tr><td colspan="9">α-glucoside transporter genes </td></tr><tr><td>LBYG00616 </td><td>Sc </td><td>ScII right sub-telomere </td><td>614 </td><td>α-glucoside transporter </td><td><em>YBR298C</em> </td><td><em>MAL31</em> </td><td>614 </td><td>98 </td></tr><tr><td>LBYG03039 </td><td>Sb </td><td>ScVII right sub-telomere<br />SbVII right sub-telomere </td><td>615 </td><td>α-glucoside transporter </td><td><em>YBR298C</em> </td><td><em>MAL31</em> </td><td>614 </td><td>91 </td></tr><tr><td>LBYG13187 </td><td>Sb </td><td>SbXV-VIII left sub-telomere </td><td>610 </td><td>α-glucoside transporter </td><td><em>YGR289C</em> </td><td><em>AGT1</em> </td><td>616 </td><td>85 </td></tr><tr><td>LBYG00811-00812<sup>a</sup> </td><td>Sc </td><td>ScIV left sub-telomere </td><td>602 </td><td>α-glucoside transporter </td><td><em>YDL247W</em> </td><td><em>MPH2</em> </td><td>609 </td><td>94 </td></tr><tr><td>LBYG09472<sup>a</sup> </td><td>Sb </td><td>SbV right sub-telomere </td><td>231 (tr) </td><td>α-glucoside transporter </td><td><em>YBR298C</em> </td><td><em>MAL31</em> </td><td>614 </td><td>80 </td></tr><tr><td>LBYG14145<sup>a</sup> </td><td>Sc </td><td>Unkown </td><td>321 (tr) </td><td>α-glucoside transporter </td><td><em>YGR289C</em> </td><td><em>AGT1</em> </td><td>616 </td><td>96 </td></tr><tr><td colspan="9">Genes related to the sulfite production pathway </td></tr><tr><td>LBYG00610 </td><td>Sc </td><td>ScII right sub-telomere </td><td>183 (tr) </td><td>High affinity sulfate permease </td><td><em>YBR294W</em> </td><td><em>SUL1</em> </td><td>859 </td><td>100 </td></tr><tr><td>LBYG09125 </td><td>Sb </td><td>SbIV-II right sub-telomere </td><td>295 (tr) </td><td>High affinity sulfate permease </td><td><em>YBR294W</em> </td><td><em>SUL1</em> </td><td>859 </td><td>83 </td></tr><tr><td>LBYG04711-04712<sup>a</sup> </td><td>Sc </td><td>ScXII </td><td>893 </td><td>High affinity sulfate permease </td><td><em>YLR092W</em> </td><td><em>SUL2</em> </td><td>893 </td><td>99 </td></tr><tr><td>LBYG11823 </td><td>Sb </td><td>ScXII </td><td>893 </td><td>High affinity sulfate permease </td><td><em>YLR092W</em> </td><td><em>SUL2</em> </td><td>893 </td><td>87 </td></tr><tr><td>LBYG03911 </td><td>Sc </td><td>ScX<br />SbX-ScX </td><td>511 </td><td>ATP sulfurylase </td><td><em>YJR010W</em> </td><td><em>MET3</em> </td><td>511 </td><td>100 </td></tr><tr><td>LBYG11111 </td><td>Sb </td><td>SbX<br />ScX-SbX </td><td>511 </td><td>ATP sulfurylase </td><td><em>YJR010W</em> </td><td><em>MET3</em> </td><td>511 </td><td>89 </td></tr><tr><td>LBYG04364 </td><td>Sc </td><td>ScXI<br />ScV-ScXI </td><td>202 </td><td>Adenylylsulfate kinase </td><td><em>YKL001C</em> </td><td><em>MET14</em> </td><td>202 </td><td>100 </td></tr><tr><td>LBYG11543 </td><td>Sb </td><td>SbXI<br />ScXI-SbXI </td><td>202 </td><td>Adenylylsulfate kinase </td><td><em>YKL001C</em> </td><td><em>MET14</em> </td><td>202 </td><td>97 </td></tr><tr><td>LBYG07555 </td><td>Sb </td><td>mosaic-ScXVI<br />mosaic-SbXVI </td><td>261 </td><td>3′-phosphoadenylsulfate reductase </td><td><em>YPR167C</em> </td><td><em>MET16</em> </td><td>261 </td><td>92 </td></tr><tr><td>LBYG07205 </td><td>Sc </td><td>mosaic-ScXVI </td><td>458 </td><td>Plasma membrane sulfite pump </td><td><em>YPL092W</em> </td><td><em>SSU1</em> </td><td>458 </td><td>99 </td></tr><tr><td>LBYG13862 </td><td>Sb </td><td>mosaic-SbXVI </td><td>458 </td><td>Plasma membrane sulfite pump </td><td><em>YPL092W</em> </td><td><em>SSU1</em> </td><td>458 </td><td>79 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>α-Glucoside transporter genes and genes related to the sulfite production pathway are shown. In <em>Saccharomyces</em> sensu stricto yeast, sulfite is produced in the methionine and cysteine biosynthesis pathway. Sulfate is imported into a cell by sulfate transporters, Sul1p and Sul2p and is converted into APS (adenylylsulfate) by ATP sulfurylase, Met3p. APS is phosphorylated, yielding PAPS (3′-phosphoadenylyl-sulfate) by adenylylsulfate kinase, Met14p. Sulfite is yielded by the reduction of PAPS by 3′-phosphoadenylylsulfate reductase, Met16p. Excess sulfite is passed out of the cell by a sulfite transporter, Ssu1p.</p></div></div><div content-id=""><div class="fn"><p>The Tr in brackets indicates the truncated genes. The genes were aligned against <em>S. cerevisiae</em> gene using Blastp, and each <em>S. cerevisiae</em> gene with the highest score is shown. Non-truncated and truncated protein sequence identities were calculated by the global alignment tool (needle) and local alignment tool, respectively, which were provided by EMBOSS. Genes that were found in lager brewing yeast but in which the specific chromosome could not be decided are shown as ‘unknown’.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Note that the genes were confirmed by PCR amplification and sequencing (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Sequence S1</a>).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7937781" target="_blank">View Large</a></div></div><p>Integration of an intact <em>AGT1</em> gene containing its promoter region from <em>S</em>. <em>cerevisiae</em> into a lager brewing yeast caused a reduction in sugar assimilation rates during lager beer fermentation in comparison with the parental strain (H. Hatanaka et al., manuscript in preparation). These results indicate that adaptation to brewing conditions resulted not only in the increase of the copy number of efficient α-glucoside transporter genes but also in the inactivation of less efficient genes, since the capacity of membrane-localized proteins may have been limited.</p>                            <h3 scrollto-destination=7937783 id="7937783" class="section-title" data-legacy-id=s3h>3.8. Sulfite production pathway</h3>
<p>Lager brewing yeasts produce relatively high levels of sulfite compared with ale brewing yeasts.<sup><a reveal-id="DSP003C18" data-open="DSP003C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Two distinct Sc- and Sb-type genes were found in the lager brewing yeast sulfite production pathway, with the exception of <em>MET16</em> (Table <a reveal-id="DSP003TB4" data-open="DSP003TB4" class="link link-reveal link-table xref-fig">4</a>). Lager brewing yeast, in fact, has only Sb-type <em>MET16</em> because of a deletion on chromosome XVI in the area (from 861 to 920 kb) that includes the Sc-type <em>MET16</em>. Lager brewing yeast has four sulfate transporter genes: Sc-type-<em>SUL1</em>, <em>SUL2</em> and Sb-type-<em>SUL1</em>, <em>SUL2</em>. The <em>SUL1</em> gene sequences from <em>S</em>. <em>cerevisiae</em> encode an 859 amino acid polypeptide with nine predicted transmembrane domains, but Sc- and Sb-type <em>SUL1</em> genes were found to encode truncated (183 and 295 amino acids) polypeptides with stop codon and frameshift mutations, respectively. These proteins are unlikely to be functional transporters because they have only one and three transmembrane domains. Five genes (Sc-<em>MET3</em>, Sb-<em>MET3</em>, Sc-<em>MET14</em>, Sb-<em>MET14</em> and Sb-<em>MET16</em>) were found on two chromosomes due to chromosome translocations (Table <a reveal-id="DSP003TB4" data-open="DSP003TB4" class="link link-reveal link-table xref-fig">4</a>), indicating that these gene loci were increased in copy number.</p>                            <h3 scrollto-destination=7937785 id="7937785" class="section-title" data-legacy-id=s3i>3.9. Lager brewing yeast-specific genes</h3>
<p>Eight genes were found to be present in the lager brewing yeast but absent in both <em>S</em>. <em>cerevisiae</em> S288C and <em>S</em>. <em>bayanus</em> CBS 7001. Except for an YJM789 homolog (LBYG13 665), these were all located in sub-telomeric regions (Table <a reveal-id="DSP003TB5" data-open="DSP003TB5" class="link link-reveal link-table xref-fig">5</a>). LBYG05796 (encoding a transcriptional regulatory protein) was also found in YJM789.<sup><a reveal-id="DSP003C28" data-open="DSP003C28" class="link link-ref link-reveal xref-bibr">28</a></sup> LBYG11275 (encoding a melibiase), LBYG08543 (a fructose symporter), LBYG05774, (a tyrosine permease), LBYG05783 (an amidase) and LBYG09147 (an amidase) have been identified and characterized in lager brewing yeast, and some of their functions are specific of lager brewing yeast.<sup><a reveal-id="DSP003C43  DSP003C46" data-open="DSP003C43  DSP003C46" class="link link-ref link-reveal xref-bibr">43–46</a></sup> Only one of the lager brewing yeast specific genes (LBYG09608) did not have any previous <em>Saccharomyce</em>s homologs. According to the results retrieved by the PSORT II system (<a class="link link-uri" href="http://psort.hgc.jp/form2.html">http://psort.hgc.jp/form2.html</a>), this gene is predicted to encode a monocarboxylate transporter (MCT) with major facilitator superfamily, and localize in the plasma membrane or in the endoplasmic reticulum. Subsequent tests demonstrated that it is expressed at the middle and late stages of lager beer fermentation (data not shown).</p>                        <a id="7937787" scrollto-destination="7937787"></a>
<div content-id="DSP003TB5" class="table-modal"><div class="table-wrap"><div id="DSP003TB5" class="table-wrap-title" data-id="DSP003TB5"><div class="label -label">Table 5</div><div class="caption"><p>Lager brewing yeast-specific genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene </th><th>Location </th><th>Length (aa) </th><th>Predicted Protein </th><th>E-value </th><th>Homolog (BLASTP) </th><th>Pfam Domains </th></tr></thead><tbody><tr><td>LBYG11275 </td><td>SbX right sub-telomere<br />ScX-SbX right sub-telomere </td><td>471 </td><td>Melibiase </td><td>0 </td><td>α-galactosidase (<em>S. pastorianus</em>) </td><td>Melibiase </td></tr><tr><td>LBYG08543 </td><td>SbIV-II left sub-telomere </td><td>570 </td><td>Fructose symporter </td><td>0 </td><td>fructose symporter (<em>S. pastorianus</em>) </td><td>Sugar (and other) transporter </td></tr><tr><td>LBYG05774 </td><td>SbXIII-ScXIII right sub-telomere<br />ScXIII right sub-telomere </td><td>557 </td><td>tyrosine permease </td><td>0 </td><td>tyrosine permease (<em>S. pastorianus</em>) </td><td>Amino acid permease </td></tr><tr><td>LBYG05783 </td><td>ScXIII right sub-telomere<br />SbXIII-ScXIII right sub-telomere </td><td>598 </td><td>amidase </td><td>0 </td><td>amidase homolog AMI1-A (<em>S. pastorianus</em>) </td><td>Amidase </td></tr><tr><td>LBYG09147 </td><td>SbV left sub-telomere </td><td>395 </td><td>amidase </td><td>0 </td><td>amidase homolog AMI1-B (<em>S. pastorianus</em>) </td><td>Amidase </td></tr><tr><td>LBYG13665 </td><td>SbXVI left arm </td><td>372 </td><td>hypothetical protein </td><td>e<sup>−138</sup> </td><td>hypothetical protein SCY_5476 (<em>S. cerevisiae</em> YJM789) </td><td>- </td></tr><tr><td>LBYG05796 </td><td>ScXIV left sub-telomere </td><td>795 </td><td>transcriptional regulatory protein </td><td>0 </td><td>hypothetical protein SCY_1426 (<em>S. cerevisiae</em> YJM789) </td><td>Fungal Zn(2)-Cys(6) binuclear cluster domain </td></tr><tr><td>LBYG09608 </td><td>SbVI right sub-telomere </td><td>356 </td><td>Monocarboxylate transporter </td><td>1e<sup>−55</sup> </td><td>MFS monocarboxylate transporter, putative (<em>Aspergillus clavatus</em> NRRL 1) </td><td>Major Facilitator Superfamily </td></tr></tbody></table></div><div content-id="DSP003TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene </th><th>Location </th><th>Length (aa) </th><th>Predicted Protein </th><th>E-value </th><th>Homolog (BLASTP) </th><th>Pfam Domains </th></tr></thead><tbody><tr><td>LBYG11275 </td><td>SbX right sub-telomere<br />ScX-SbX right sub-telomere </td><td>471 </td><td>Melibiase </td><td>0 </td><td>α-galactosidase (<em>S. pastorianus</em>) </td><td>Melibiase </td></tr><tr><td>LBYG08543 </td><td>SbIV-II left sub-telomere </td><td>570 </td><td>Fructose symporter </td><td>0 </td><td>fructose symporter (<em>S. pastorianus</em>) </td><td>Sugar (and other) transporter </td></tr><tr><td>LBYG05774 </td><td>SbXIII-ScXIII right sub-telomere<br />ScXIII right sub-telomere </td><td>557 </td><td>tyrosine permease </td><td>0 </td><td>tyrosine permease (<em>S. pastorianus</em>) </td><td>Amino acid permease </td></tr><tr><td>LBYG05783 </td><td>ScXIII right sub-telomere<br />SbXIII-ScXIII right sub-telomere </td><td>598 </td><td>amidase </td><td>0 </td><td>amidase homolog AMI1-A (<em>S. pastorianus</em>) </td><td>Amidase </td></tr><tr><td>LBYG09147 </td><td>SbV left sub-telomere </td><td>395 </td><td>amidase </td><td>0 </td><td>amidase homolog AMI1-B (<em>S. pastorianus</em>) </td><td>Amidase </td></tr><tr><td>LBYG13665 </td><td>SbXVI left arm </td><td>372 </td><td>hypothetical protein </td><td>e<sup>−138</sup> </td><td>hypothetical protein SCY_5476 (<em>S. cerevisiae</em> YJM789) </td><td>- </td></tr><tr><td>LBYG05796 </td><td>ScXIV left sub-telomere </td><td>795 </td><td>transcriptional regulatory protein </td><td>0 </td><td>hypothetical protein SCY_1426 (<em>S. cerevisiae</em> YJM789) </td><td>Fungal Zn(2)-Cys(6) binuclear cluster domain </td></tr><tr><td>LBYG09608 </td><td>SbVI right sub-telomere </td><td>356 </td><td>Monocarboxylate transporter </td><td>1e<sup>−55</sup> </td><td>MFS monocarboxylate transporter, putative (<em>Aspergillus clavatus</em> NRRL 1) </td><td>Major Facilitator Superfamily </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Lager brewing yeast-specific genes that were found to be present only in the lager brewing yeast strain but absent in <em>S. cerevisiae</em> S288C and <em>S. bayanus</em> CBS 7001. The genes were identified by Blastp search against the non-redundant database and by Pfam search, and each description with the highest score or nearly the highest score is shown.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7937787" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSP003TB5" class="table-wrap-title" data-id="DSP003TB5"><div class="label -label">Table 5</div><div class="caption"><p>Lager brewing yeast-specific genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene </th><th>Location </th><th>Length (aa) </th><th>Predicted Protein </th><th>E-value </th><th>Homolog (BLASTP) </th><th>Pfam Domains </th></tr></thead><tbody><tr><td>LBYG11275 </td><td>SbX right sub-telomere<br />ScX-SbX right sub-telomere </td><td>471 </td><td>Melibiase </td><td>0 </td><td>α-galactosidase (<em>S. pastorianus</em>) </td><td>Melibiase </td></tr><tr><td>LBYG08543 </td><td>SbIV-II left sub-telomere </td><td>570 </td><td>Fructose symporter </td><td>0 </td><td>fructose symporter (<em>S. pastorianus</em>) </td><td>Sugar (and other) transporter </td></tr><tr><td>LBYG05774 </td><td>SbXIII-ScXIII right sub-telomere<br />ScXIII right sub-telomere </td><td>557 </td><td>tyrosine permease </td><td>0 </td><td>tyrosine permease (<em>S. pastorianus</em>) </td><td>Amino acid permease </td></tr><tr><td>LBYG05783 </td><td>ScXIII right sub-telomere<br />SbXIII-ScXIII right sub-telomere </td><td>598 </td><td>amidase </td><td>0 </td><td>amidase homolog AMI1-A (<em>S. pastorianus</em>) </td><td>Amidase </td></tr><tr><td>LBYG09147 </td><td>SbV left sub-telomere </td><td>395 </td><td>amidase </td><td>0 </td><td>amidase homolog AMI1-B (<em>S. pastorianus</em>) </td><td>Amidase </td></tr><tr><td>LBYG13665 </td><td>SbXVI left arm </td><td>372 </td><td>hypothetical protein </td><td>e<sup>−138</sup> </td><td>hypothetical protein SCY_5476 (<em>S. cerevisiae</em> YJM789) </td><td>- </td></tr><tr><td>LBYG05796 </td><td>ScXIV left sub-telomere </td><td>795 </td><td>transcriptional regulatory protein </td><td>0 </td><td>hypothetical protein SCY_1426 (<em>S. cerevisiae</em> YJM789) </td><td>Fungal Zn(2)-Cys(6) binuclear cluster domain </td></tr><tr><td>LBYG09608 </td><td>SbVI right sub-telomere </td><td>356 </td><td>Monocarboxylate transporter </td><td>1e<sup>−55</sup> </td><td>MFS monocarboxylate transporter, putative (<em>Aspergillus clavatus</em> NRRL 1) </td><td>Major Facilitator Superfamily </td></tr></tbody></table></div><div content-id="DSP003TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene </th><th>Location </th><th>Length (aa) </th><th>Predicted Protein </th><th>E-value </th><th>Homolog (BLASTP) </th><th>Pfam Domains </th></tr></thead><tbody><tr><td>LBYG11275 </td><td>SbX right sub-telomere<br />ScX-SbX right sub-telomere </td><td>471 </td><td>Melibiase </td><td>0 </td><td>α-galactosidase (<em>S. pastorianus</em>) </td><td>Melibiase </td></tr><tr><td>LBYG08543 </td><td>SbIV-II left sub-telomere </td><td>570 </td><td>Fructose symporter </td><td>0 </td><td>fructose symporter (<em>S. pastorianus</em>) </td><td>Sugar (and other) transporter </td></tr><tr><td>LBYG05774 </td><td>SbXIII-ScXIII right sub-telomere<br />ScXIII right sub-telomere </td><td>557 </td><td>tyrosine permease </td><td>0 </td><td>tyrosine permease (<em>S. pastorianus</em>) </td><td>Amino acid permease </td></tr><tr><td>LBYG05783 </td><td>ScXIII right sub-telomere<br />SbXIII-ScXIII right sub-telomere </td><td>598 </td><td>amidase </td><td>0 </td><td>amidase homolog AMI1-A (<em>S. pastorianus</em>) </td><td>Amidase </td></tr><tr><td>LBYG09147 </td><td>SbV left sub-telomere </td><td>395 </td><td>amidase </td><td>0 </td><td>amidase homolog AMI1-B (<em>S. pastorianus</em>) </td><td>Amidase </td></tr><tr><td>LBYG13665 </td><td>SbXVI left arm </td><td>372 </td><td>hypothetical protein </td><td>e<sup>−138</sup> </td><td>hypothetical protein SCY_5476 (<em>S. cerevisiae</em> YJM789) </td><td>- </td></tr><tr><td>LBYG05796 </td><td>ScXIV left sub-telomere </td><td>795 </td><td>transcriptional regulatory protein </td><td>0 </td><td>hypothetical protein SCY_1426 (<em>S. cerevisiae</em> YJM789) </td><td>Fungal Zn(2)-Cys(6) binuclear cluster domain </td></tr><tr><td>LBYG09608 </td><td>SbVI right sub-telomere </td><td>356 </td><td>Monocarboxylate transporter </td><td>1e<sup>−55</sup> </td><td>MFS monocarboxylate transporter, putative (<em>Aspergillus clavatus</em> NRRL 1) </td><td>Major Facilitator Superfamily </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Lager brewing yeast-specific genes that were found to be present only in the lager brewing yeast strain but absent in <em>S. cerevisiae</em> S288C and <em>S. bayanus</em> CBS 7001. The genes were identified by Blastp search against the non-redundant database and by Pfam search, and each description with the highest score or nearly the highest score is shown.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7937787" target="_blank">View Large</a></div></div><p>MCTs form a large family of proton-linked carriers that have the ability to transport monocarboxylates such as lactate, pyruvate or acetate across the plasma membrane, and they can be found in bacteria, protozoa, fungi, invertebrates, as well as in vertebrates.<sup><a reveal-id="DSP003C47" data-open="DSP003C47" class="link link-ref link-reveal xref-bibr">47</a></sup> Pyruvate, acetate and lactate are by-products formed during yeast alcoholic fermentation. Therefore, LBYG09608 gene encoding MCT may be expressed at the middle and late stages of fermentation to control these monocarboxylates in the plasma membrane or in the endoplasmic reticulum.</p>                            <h3 scrollto-destination=7937789 id="7937789" class="section-title" data-legacy-id=s3j>3.10. Origins and evolution of lager brewing yeast</h3>
<p>The genome sequencing of <em>S</em>. <em>pastorianus</em> Weihenstephan 34/70 confirmed at genomic level that lager brewing yeasts are hybrids between the two species <em>S</em>. <em>cerevisiae</em> and <em>S</em>. <em>bayanus</em>. The phylogenetic tree shown in Fig. <a reveal-id="DSP003F1" data-open="DSP003F1" class="link link-reveal link-table xref-fig">1</a>, constructed on the basis of concatenated sets of genes of various <em>Saccharomyces</em> yeasts and the high sequence identities between Sc-type and <em>S</em>. <em>cerevisiae</em> ORFs show that the Sc-type sub-genome of the lager brewing yeast is clearly close to <em>S</em>. <em>cerevisiae</em>. Recently, Dunn and Sherlock,<sup><a reveal-id="DSP003C48" data-open="DSP003C48" class="link link-ref link-reveal xref-bibr">48</a></sup> on the basis of results obtained using two species CGH array analysis, reported that the most probable ancestral parent of lager brewing yeasts Sc-type sub-genome is related to ale brewing yeasts. The contribution of ale brewing yeasts to the shaping of the lager brewing yeast Sc-type sub-genome, already proposed by de Barros Lopes et al.,<sup><a reveal-id="DSP003C49" data-open="DSP003C49" class="link link-ref link-reveal xref-bibr">49</a></sup> is a very likely possibility.</p><p>Lager brewing yeast Sb-type sub-genome contributes to the most useful features typical of this yeast. However, the full identification of this sub-genome has never been achieved and this is probably due to the complexity of the <em>S</em>. <em>bayanus</em> species as well as the lack of a proper reference strain, as already described above. Our results show that the Sb-type sub-genome is clearly further apart from <em>S</em>. <em>bayanus</em> CBS 7001, which actually belongs to the <em>S</em>. <em>uvarum</em> group and is closely related to the enological environment. Additionally, although <em>S</em>. <em>bayanus</em> CBS 7001 genome has three reciprocal translocations (right arm of II and left arm of II, VI–X and II–XIV),<sup><a reveal-id="DSP003C26" data-open="DSP003C26" class="link link-ref link-reveal xref-bibr">26</a></sup> the gene syntenies of the corresponding Sb-type lager brewing yeast chromosomal regions are the same as those of <em>S</em>. <em>cerevisiae</em>. We demonstrated in a previous study that the yeast that best represent the Sb-type sub-genome is strain NBRC 1948.<sup><a reveal-id="DSP003C50" data-open="DSP003C50" class="link link-ref link-reveal xref-bibr">50</a></sup> This yeast, unlike <em>S</em>. <em>bayanus</em> CBS 7001, is currently classified as <em>S</em>. <em>bayanus</em> var. <em>bayanus</em> and was originally isolated from beer. The typing of 48 gene fragments in various lager brewing yeasts by RFLP analysis showed that the Sb-type sub-genome differed from the genome of strain NBRC 1948 only for a few genes. <em>Saccharomyces uvarum</em> sequences in lager brewing yeasts were limited to the few genes portions in which <em>S</em>. <em>bayanus</em> var. <em>bayanus</em> strains could not be distinguished from <em>S</em>. <em>uvarum</em> strains.<sup><a reveal-id="DSP003C50" data-open="DSP003C50" class="link link-ref link-reveal xref-bibr">50</a></sup> This indicates a common origin of <em>S</em>. <em>bayanus</em> and <em>S</em>. <em>uvarum</em> yeasts and a subsequent differentiated evolution process in the brewing and in the enological environment, respectively.</p><p>The origin of the Sb-type sub-genome can also be confirmed by studying mtDNA inheritance. Several works have demonstrated that in lager brewing yeast the mtDNA genome is originating from the <em>S</em>. <em>bayanus</em> parental strain. In artificial interspecies, hybrid mtDNA is inherited by only one of the parental strains.<sup><a reveal-id="DSP003C51" data-open="DSP003C51" class="link link-ref link-reveal xref-bibr">51</a></sup> Consequently, the sequence analysis of the mtDNA in lager brewing yeasts can be very useful to define the nature of the <em>S</em>. <em>bayanus</em> parental genome. According to our results, the nucleotide sequence of lager brewing yeast <em>COXII</em>, <em>COXIII</em>, <em>ATP8</em>, <em>ATP6</em> and <em>21S rRNA</em> showed a 2–4% sequence divergence with the corresponding <em>S</em>. <em>uvarum</em> genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). On the other hand, the nucleotide sequence of lager brewing yeast <em>COXII</em> gene was identical to <em>S</em>. <em>bayanus</em> var. <em>bayanus</em> NBRC 1948 <em>COXII</em> gene, and mtDNA restriction analysis carried out with three enzymes confirmed this result.<sup><a reveal-id="DSP003C52" data-open="DSP003C52" class="link link-ref link-reveal xref-bibr">52</a></sup> Considering our present and previous results, the Sb-type sub-genome did not originate from <em>S</em>. <em>uvarum</em>, but from an <em>S</em>. <em>bayanus</em> var. <em>bayanus</em> yeast closely related to strain NBRC 1948.</p><p>Several studies have been carried out to explain the origin and the evolutionary processes that lager brewing yeasts has undergone. Either a simple hybridization event between an <em>S</em>. <em>cerevisiae</em> and <em>S</em>. <em>bayanus</em> yeast, or more likely, a multiple hybridization event as proposed by de Barros Lopes et al.,<sup><a reveal-id="DSP003C49" data-open="DSP003C49" class="link link-ref link-reveal xref-bibr">49</a></sup> followed by chromosomal loss and rearrangements are the most likely processes to have occurred, since they are common mechanisms for yeast genome evolution.<sup><a reveal-id="DSP003C53" data-open="DSP003C53" class="link link-ref link-reveal xref-bibr">53</a></sup> The hybridization between haploid <em>S</em>. <em>bayanus</em> and <em>S</em>. <em>cerevisiae</em> spores proposed by Dunn and Sherlock<sup><a reveal-id="DSP003C48" data-open="DSP003C48" class="link link-ref link-reveal xref-bibr">48</a></sup> seems a less probable mechanism, given the fact that interspecies hybrid constructed in vitro by conjugating haploid spores are generally stable and do not undergo genome rearrangements.<sup><a reveal-id="DSP003C54" data-open="DSP003C54" class="link link-ref link-reveal xref-bibr">54</a></sup></p><p>According to our results, following the hybridization event, lager brewing yeast genome was shaped as follows: eight chromosomal translocations between the Sc- and the Sb-type sub-genomes occurred and due to chromosomal translocations, the number of chromosomes increased with respect to the parental species. We have demonstrated that the lager brewing yeast possess 36 chromosomes, whereas its putative parental species, <em>S</em>. <em>cerevisiae</em> and <em>S</em>. <em>bayanus</em>, possess 16. The translocations between diverged sequences are rare in <em>Saccharomyces</em> yeast as described above.<sup><a reveal-id="DSP003C33" data-open="DSP003C33" class="link link-ref link-reveal xref-bibr">33</a></sup> The stressful condition typical of alcoholic fermentation might have driven the evolution of lager brewing yeast and the translocation that occurred to shape its genome. An experimental study of <em>S</em>. <em>cerevisiae</em> in a continuous culture under glucose limitation, for example, revealed that the selection pressure imposed by the environmental conditions can result in the selection of strains having translocations, being aneuploid or increasing the copy number of the high-affinity hexose transporter genes.<sup><a reveal-id="DSP003C55" data-open="DSP003C55" class="link link-ref link-reveal xref-bibr">55</a></sup> Another study proposed that gross chromosomal translocations were enhanced by ethanol and acetaldehyde.<sup><a reveal-id="DSP003C56" data-open="DSP003C56" class="link link-ref link-reveal xref-bibr">56</a></sup> A chromosomal translocation mediated by short homology has also been found in wine strains which are also exposed to multiple stress factors during alcoholic fermentation.<sup><a reveal-id="DSP003C57" data-open="DSP003C57" class="link link-ref link-reveal xref-bibr">57</a></sup> Thus, it seems that the multiple stress factors that lager brewing yeast have been exposed to (such as low fermentation temperature, low pH, sugar depletion and high ethanol conditions) could have determined or facilitated chromosomal translocations, as a process of evolutionary adaptation. Most translocations after the hybridization event occurred by non-reciprocal recombination. It has been demonstrated that in <em>S</em>. <em>cerevisiae</em>, chromosomal translocations seem to increase as diploid yeast cells age. While in young cells, translocations occur by reciprocal recombination, in the old mother’s progeny, they occur by non-reciprocal recombination.<sup><a reveal-id="DSP003C58" data-open="DSP003C58" class="link link-ref link-reveal xref-bibr">58</a></sup> This phenomenon could explain the translocation that occurred in lager brewing yeast by non-reciprocal recombination. Lager brewing fermentation is traditionally carried out by transferring the yeast culture from one fermentation batch to the following; thus, it is possible that non-reciprocal translocations between the Sc- and the Sb-type chromosomes occurred in an aged lager brewing yeast cell that adapted to lager brewing conditions. Non-reciprocal translocations usually cause the change of gene dosage. In the lager brewing yeast we sequenced, one large region of the Sc-type chromosomes and five large regions of the Sb-type chromosomes have been deleted and unilateral chromosomal regions have been increased by these non-reciprocal translocations. The increased number of loci caused by the translocations of one gene associated to an efficient maltose and maltotriose uptakes and of genes associated to sulfite production might have been the result not only of adaptation to the lager brewing conditions, but also and especially to human-driven selection. More than any other yeast the evolution of lager brewing yeasts has been strongly influenced by human-imposed selection, which has favored strains with optimal fermentation characteristics and good organoleptic traits. The lager brewing yeast sequenced in this study, possess a non-functional Sc-type <em>AGT1</em> gene; however, it still showed an increase in sugar assimilation rate when compared with lager brewing yeast with functional <em>AGT1</em>, suggesting that not only chromosomal translocation but also gene deletion and truncation after the hybridization event have been beneficial for lager brewing yeast.</p><p>Dunn and Sherlock<sup><a reveal-id="DSP003C48" data-open="DSP003C48" class="link link-ref link-reveal xref-bibr">48</a></sup> classified lager brewing yeasts into two distinct groups based on their shared sets of chromosomes; group 1 included strains that had lost a significant portion of the <em>S</em>. <em>cerevisiae</em> genome, and group 2 included strains that had preserved all genomic contents of both <em>S</em>. <em>cerevisiae</em> and <em>S</em>. <em>bayanus</em>. The set of 17 lager brewing strains these authors tested did not include any current production strains. The strain sequenced in this work is currently used in lager beer production and could be considered as a representative of lager brewing production strains. According to this grouping, our representative production strain is part of Group 2, in fact it possess almost the two entire sets of sub-genomes. Furthermore, most of the breakpoints of non-reciprocal translocations that we found seem to be consistent with the CGH analysis these authors performed, although they have not been confirmed by sequencing analysis.</p>                            <h3 scrollto-destination=7937796 id="7937796" class="section-title" data-legacy-id=s3k>3.11. Implications</h3>
<p>Lager brewing yeast is economically the most important yeast for beer brewing, but its genome sequence had not yet been determined, partly due to its complexity and partly to the lack of knowledge of its genome structure. In this study, the genome sequence was determined by the whole genome shotgun sequencing, by the additional cosmid sequencing and subsequently confirmed by opmap. This work represents the first example of the whole genome sequencing of a lager brewing yeast strain and demonstrates that lager brewing yeasts comprises two sub-genomes, one originating from <em>S</em>. <em>cerevisiae</em> and the other from <em>S</em>. <em>bayanus</em>. Sequence comparison between Sb-type sub-genome and mtDNA of lager brewing yeast and those of <em>S</em>. <em>bayanus</em> CBS 7001 (<em>S</em>. <em>uvarum</em>) supports the notion that the Sb-type sub-genome in the current lager brewing strain is not originating from <em>S</em>. <em>uvarum</em>, but rather from <em>S</em>. <em>bayanus</em> var. <em>bayanus</em>.</p><p>There are several specific biological features of lager brewing yeast such as the production of a significant amount of sulfite and the ability to ferment at low temperatures (ranging from 8 to 15°C). The Sb-type sub-genome is possibly responsible for these features since <em>S</em>. <em>cerevisiae</em> produces a relatively small amount sulfite<sup><a reveal-id="DSP003C18" data-open="DSP003C18" class="link link-ref link-reveal xref-bibr">18</a></sup> and ferments at higher temperatures, whereas the hybrid strains between <em>S</em>. <em>cerevisiae</em> and <em>S</em>. <em>bayanus</em> ferment fast and efficiently at low-temperature.<sup><a reveal-id="DSP003C59" data-open="DSP003C59" class="link link-ref link-reveal xref-bibr">59</a></sup> However, a clear understanding of these characteristics has been limited because of the lack of a complete genome sequence of this yeast. We have made the lager brewing yeast genome a freely accessible resource, and this represent a significant step towards further analysis of the lager brewing yeast. This may benefit research programs of genetic engineering or breeding designed to improve brewing yeast strains. New technologies for comprehensive analyses (gene expression analysis, polymorphism analysis and proteome analysis) can also be developed. These technologies promise further advances in the science of lager brewing yeast, such as understanding why lager brewing yeast is good for lager beer brewing, and the detection of sequence polymorphisms among lager strains for reference of the strain-specific characteristics. Indeed, comprehensive gene expression analysis of lager brewing yeasts under lager beer production using lager brewing yeast DNA microarray has revealed that the expression level of the Sb-type <em>SSU1</em> gene, encoding a sulfite transporter, was causing the elevated sulfite production of lager brewing yeast.<sup><a reveal-id="DSP003C60" data-open="DSP003C60" class="link link-ref link-reveal xref-bibr">60</a></sup></p><p>To our knowledge, this work is the first example of a genome sequencing project of a hybrid species. In principle, whole genome duplication may allow for large-scale adaptation to new environments, and subsequent gene-loss events then led to current genome by a large number of rearrangement events and individual gene deletions, which persists until the genome returns to functionally normal ploidy through the mutation, gene loss and genomic rearrangements.<sup><a reveal-id="DSP003C29" data-open="DSP003C29" class="link link-ref link-reveal xref-bibr">29</a></sup> The same is the case in the lager brewing yeast, in which larger chromosomal regions have been deleted, the rDNA cluster of the Sb-type chromosome XII has been drastically reduced, and many truncated genes were found. This indicates that such rearrangements are subsequent to genome hybridization.</p><p>The genome information and the new technology involved will be useful not only for their possible industrial applications but also for academic basic research. In particular, lager brewing yeast provides an excellent case to understand whole genome duplication and multiple genomes evolution and organization.</p>                            <h2 scrollto-destination=7937801 id="7937801" class="section-title" data-legacy-id=s4>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Material is available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=7937803 id="7937803" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Y. Kaneko and H. Yoshikawa for their valuable participations in scientific discussions and H. Hatanaka for her valuable discussions on α-glucoside transporters. We are deeply appreciative of K. Olesen, M. C. Kielland-Brandt and F. Omura for critically reading the manuscript. We are also grateful to K. Togami for her assistance with this study and to R. Komatsu for his analysis of the opmap. The expert technical assistance of K. Yasui and H. Toyonaga is gratefully acknowledged.</p>                            <h2 scrollto-destination=7937805 id="7937805" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSP003C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Barth</div> <div class="given-names">S. J.</div></div>. </div><div class="article-title">The barth report hops 2006/2007</div>, <div class="year">2007</div> <div class="comment"><a class="link link-uri" href="http://www.barthhaasgroup.com/cmsdk/content/bhg/barth_report.htm">http://www.barthhaasgroup.com/cmsdk/content/bhg/barth_report.htm</a></div></div></div></div></div><div content-id="DSP003C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hansen</div> <div class="given-names">E. C.</div></div>. </div><div class="article-title">Investigations on the physiology and morphology of budding yeast. XIII. New studies of bottom fermenting brewers yeasts</div>, <div class="source">C. R. Trav. Lab. Carlsberg.</div> , <div class="year">1908</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">179</div>-<div class="lpage">217</div>)</div></div></div></div><div content-id="DSP003C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Martini</div> <div class="given-names">A. V.</div></div>,  <div class="name"><div class="surname">Martini</div> <div class="given-names">A.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Kurtzman</div> <div class="given-names">C. P.</div></div>,  <div class="name"><div class="surname">Fell</div> <div class="given-names">J. W.</div></div>. </div>, <div class="source">The Yeasts, a Taxonomic Study: Saccharomyces Myen ex Reessm</div> , <div class="year">1998</div><div class="publisher-loc">Amsterdam</div><div class="publisher-name">Elsevier</div>(pg. <div class="fpage">358</div>-<div class="lpage">371</div>) <div class="comment">Chapter 44</div></div></div></div></div><div content-id="DSP003C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Martini</div> <div class="given-names">A. V.</div></div>,  <div class="name"><div class="surname">Martini</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Three newly delimited species of <em>Saccharomyces sensu stricto</em></div>, <div class="source">Antonie Van Leeuwenhoek</div> , <div class="year">1987</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">77</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Three%20newly%20delimited%20species%20of%20Saccharomyces%20sensu%20stricto&amp;author=A.%20V.%20Martini&amp;author=A.%20Martini&amp;publication_year=1987&amp;journal=Antonie%20Van%20Leeuwenhoek&amp;volume=53&amp;pages=77-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00419503" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00419503" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00419503"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3662481" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Martini</div> <div class="given-names">A. V.</div></div>,  <div class="name"><div class="surname">Kurtzman</div> <div class="given-names">C. P.</div></div>. </div><div class="article-title">Deoxyribonucleic acid relatedness among species of the genus <em>Saccharomyces sensu stricto</em></div>, <div class="source">Int. J. Syst. Bacteriol.</div> , <div class="year">1985</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">508</div>-<div class="lpage">511</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deoxyribonucleic%20acid%20relatedness%20among%20species%20of%20the%20genus%20Saccharomyces%20sensu%20stricto&amp;author=A.%20V.%20Martini&amp;author=C.%20P.%20Kurtzman&amp;publication_year=1985&amp;journal=Int.%20J.%20Syst.%20Bacteriol.&amp;volume=35&amp;pages=508-511" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00207713-35-4-508" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00207713-35-4-508" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00207713-35-4-508"> </span></div> </div></div></div></div></div><div content-id="DSP003C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamai</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Momma</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yoshimoto</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kaneko</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Co-existence of two types of chromosome in the bottom fermenting yeast, <em>Saccharomyces pastorianus</em></div>, <div class="source">Yeast</div> , <div class="year">1998</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">923</div>-<div class="lpage">933</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Co-existence%20of%20two%20types%20of%20chromosome%20in%20the%20bottom%20fermenting%20yeast%2C%20Saccharomyces%20pastorianus&amp;author=Y.%20Tamai&amp;author=T.%20Momma&amp;author=H.%20Yoshimoto&amp;author=Y.%20Kaneko&amp;publication_year=1998&amp;journal=Yeast&amp;volume=14&amp;pages=923-933" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9717238" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yamagishi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ogata</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Chromosomal structures of bottom fermenting yeasts</div>, <div class="source">Syst. Appl. Microbiol.</div> , <div class="year">1999</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">341</div>-<div class="lpage">353</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosomal%20structures%20of%20bottom%20fermenting%20yeasts&amp;author=H.%20Yamagishi&amp;author=T.%20Ogata&amp;publication_year=1999&amp;journal=Syst.%20Appl.%20Microbiol.&amp;volume=22&amp;pages=341-353" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0723-2020(99)80041-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0723-2020(99)80041-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0723-2020(99)80041-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10553286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nilsson-Tillgren</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Petersen</div> <div class="given-names">J. G. L.</div></div>,  <div class="name"><div class="surname">Hohnberg</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kielland-Brandt</div> <div class="given-names">M. C.</div></div>. </div><div class="article-title">Transfer of chromosome III during kar mediated cytoduction in yeast</div>, <div class="source">Carlsberg Res. Commun.</div> , <div class="year">1980</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">113</div>-<div class="lpage">117</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transfer%20of%20chromosome%20III%20during%20kar%20mediated%20cytoduction%20in%20yeast&amp;author=T.%20Nilsson-Tillgren&amp;author=J.%20G.%20L.%20Petersen&amp;author=S.%20Hohnberg&amp;author=M.%20C.%20Kielland-Brandt&amp;publication_year=1980&amp;journal=Carlsberg%20Res.%20Commun.&amp;volume=45&amp;pages=113-117" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02906512" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02906512" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02906512"> </span></div> </div></div></div></div></div><div content-id="DSP003C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nilsson-TiIlgren</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Gjermansen</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Holmberg</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Petersen</div> <div class="given-names">J. G. L.</div></div>,  <div class="name"><div class="surname">Kielland-Brandt</div> <div class="given-names">M. C.</div></div>. </div><div class="article-title">Analysis of chromosome V and the ILV1 gene from <em>Saccharomyces carlsbergensis</em></div>, <div class="source">Carlsberg Res. Commun.</div> , <div class="year">1986</div>, vol. <div class="volume">51</div> (pg. <div class="fpage">309</div>-<div class="lpage">326</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20chromosome%20V%20and%20the%20ILV1%20gene%20from%20Saccharomyces%20carlsbergensis&amp;author=T.%20Nilsson-TiIlgren&amp;author=C.%20Gjermansen&amp;author=S.%20Holmberg&amp;author=J.%20G.%20L.%20Petersen&amp;author=M.%20C.%20Kielland-Brandt&amp;publication_year=1986&amp;journal=Carlsberg%20Res.%20Commun.&amp;volume=51&amp;pages=309-326" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02907164" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02907164" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02907164"> </span></div> </div></div></div></div></div><div content-id="DSP003C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kielland-Brandt</div> <div class="given-names">M. C.</div></div>,  <div class="name"><div class="surname">Nilsson-Tillgren</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Gjermansen</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Holmberg</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Pedersen</div> <div class="given-names">M. B.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Rose</div> <div class="given-names">A. H.</div></div>,  <div class="name"><div class="surname">Wheals</div> <div class="given-names">A. E.</div></div>,  <div class="name"><div class="surname">Harrison</div> <div class="given-names">J. S.</div></div>. </div><div class="article-title">Genetics of brewing yeasts</div>, <div class="source">The Yeasts</div> , <div class="year">1995</div>, vol. <div class="volume">Vol. 6</div> <div class="edition">2nd Ed.</div><div class="publisher-loc">London</div><div class="publisher-name">Academic Press</div>(pg. <div class="fpage">223</div>-<div class="lpage">254</div>)</div></div></div></div><div content-id="DSP003C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fujii</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yoshimoto</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nagasawa</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Bogaki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Tamai</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Hamachi</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Nucleotide sequences of alcohol acetyltransferase genes from lager brewing yeast <em>Saccharomyces carlsbergensis</em></div>, <div class="source">Yeast</div> , <div class="year">1996</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">593</div>-<div class="lpage">598</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleotide%20sequences%20of%20alcohol%20acetyltransferase%20genes%20from%20lager%20brewing%20yeast%20Saccharomyces%20carlsbergensis&amp;author=T.%20Fujii&amp;author=H.%20Yoshimoto&amp;author=N.%20Nagasawa&amp;author=T.%20Bogaki&amp;author=Y.%20Tamai&amp;author=M.%20Hamachi&amp;publication_year=1996&amp;journal=Yeast&amp;volume=12&amp;pages=593-598" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(SICI)1097-0061(199605)12:6&lt;593::AID-YEA593&gt;3.0.CO;2-B" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(SICI)1097-0061(199605)12:6&lt;593::AID-YEA593&gt;3.0.CO;2-B" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(SICI)1097-0061(199605)12:6&lt;593::AID-YEA593&gt;3.0.CO;2-B"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8771714" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamai</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tanaka</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Umemoto</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Tomizuka</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kaneko</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Diversity of the HO gene encoding an endonuclease for mating-type conversion in the bottom fermenting yeast <em>Saccharomyces pastorianus</em></div>, <div class="source">Yeast</div> , <div class="year">2000</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">1335</div>-<div class="lpage">1343</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20of%20the%20HO%20gene%20encoding%20an%20endonuclease%20for%20mating-type%20conversion%20in%20the%20bottom%20fermenting%20yeast%20Saccharomyces%20pastorianus&amp;author=Y.%20Tamai&amp;author=K.%20Tanaka&amp;author=N.%20Umemoto&amp;author=K.%20Tomizuka&amp;author=Y.%20Kaneko&amp;publication_year=2000&amp;journal=Yeast&amp;volume=16&amp;pages=1335-1343" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11015730" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kodama</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Omura</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Ashikari</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Isolation and characterization of a gene specific to lager brewing yeast that encodes a branched-chain amino acid permease</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2001</div>, vol. <div class="volume">67</div> (pg. <div class="fpage">3455</div>-<div class="lpage">3462</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20a%20gene%20specific%20to%20lager%20brewing%20yeast%20that%20encodes%20a%20branched-chain%20amino%20acid%20permease&amp;author=Y.%20Kodama&amp;author=F.%20Omura&amp;author=T.%20Ashikari&amp;publication_year=2001&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=67&amp;pages=3455-3462" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.67.8.3455-3462.2001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.67.8.3455-3462.2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.67.8.3455-3462.2001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11472919" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hansen</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Kielland-Brandt</div> <div class="given-names">M. C.</div></div>. </div><div class="article-title"><em>Saccharomyces carlsbergensis</em> contains two functional <em>MET2</em> alleles similar to homologues from <em>S. cerevisiae</em> and <em>S. monacensis</em></div>, <div class="source">Gene</div> , <div class="year">1994</div>, vol. <div class="volume">140</div> (pg. <div class="fpage">33</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Saccharomyces%20carlsbergensis%20contains%20two%20functional%20MET2%20alleles%20similar%20to%20homologues%20from%20S.%20cerevisiae%20and%20S.%20monacensis&amp;author=J.%20Hansen&amp;author=M.%20C.%20Kielland-Brandt&amp;publication_year=1994&amp;journal=Gene&amp;volume=140&amp;pages=33-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(94)90727-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(94)90727-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(94)90727-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8125336" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Groth</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Petersen</div> <div class="given-names">R. F.</div></div>,  <div class="name"><div class="surname">Piskur</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Diversity in organization and the origin of gene orders in the mitochondrial DNA molecules of the genus <em>Saccharomyces</em></div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2000</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">1833</div>-<div class="lpage">1841</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20in%20organization%20and%20the%20origin%20of%20gene%20orders%20in%20the%20mitochondrial%20DNA%20molecules%20of%20the%20genus%20Saccharomyces&amp;author=C.%20Groth&amp;author=R.%20F.%20Petersen&amp;author=J.%20Piskur&amp;publication_year=2000&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=17&amp;pages=1833-1841" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026284" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026284" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a026284"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11110899" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Boulton</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Quain</div> <div class="given-names">D.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Boulton</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Quain</div> <div class="given-names">D.</div></div>. </div>, <div class="source">Brewing Yeast and Fermentation: Brewing yeast</div> , <div class="year">2001</div><div class="publisher-loc">Oxford</div><div class="publisher-name">Blackwell Science Ltd</div>(pg. <div class="fpage">143</div>-<div class="lpage">259</div>) <div class="comment">Chapter 4</div></div></div></div></div><div content-id="DSP003C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zheng</div> <div class="given-names">X. T.</div></div>,  <div class="name"><div class="surname">D’Amore</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Russell</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Stewart</div> <div class="given-names">G. G.</div></div>. </div><div class="article-title">Factors influence maltotriose utilization during brewery wort fermentations</div>, <div class="source">J. Am. Soc. Brew. Chem.</div> , <div class="year">1994</div>, vol. <div class="volume">52</div> (pg. <div class="fpage">41</div>-<div class="lpage">47</div>)</div></div></div></div><div content-id="DSP003C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Crumplen</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">D’Amore</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Slaughter</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Stewart</div> <div class="given-names">G. G.</div></div>. </div><div class="article-title">Novel differences between ale and lager brewing yeasts</div>, <div class="source">Proc. Congr. Eur. Brew. Conv.</div> , <div class="year">1993</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">267</div>-<div class="lpage">274</div>)</div></div></div></div><div content-id="DSP003C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ewing</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Hillier</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Wendl</div> <div class="given-names">M. C.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Base-calling of automated sequencer traces using phred. I. Accuracy assessment</div>, <div class="source">Genome Res.</div> , <div class="year">1998</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">175</div>-<div class="lpage">185</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Base-calling%20of%20automated%20sequencer%20traces%20using%20phred.%20I.%20Accuracy%20assessment&amp;author=B.%20Ewing&amp;author=L.%20Hillier&amp;author=M.%20C.%20Wendl&amp;author=P.%20Green&amp;publication_year=1998&amp;journal=Genome%20Res.&amp;volume=8&amp;pages=175-185" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.8.3.175" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.8.3.175" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.8.3.175"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9521921" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Aluru</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">S. P.</div></div>,  <div class="name"><div class="surname">Hillier</div> <div class="given-names">L.</div></div>. </div><div class="article-title">PCAP: a whole-genome assembly program</div>, <div class="source">Genome Res.</div> , <div class="year">2003</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">2164</div>-<div class="lpage">2170</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PCAP%3A%20a%20whole-genome%20assembly%20program&amp;author=X.%20Huang&amp;author=J.%20Wang&amp;author=S.%20Aluru&amp;author=S.%20P.%20Yang&amp;author=L.%20Hillier&amp;publication_year=2003&amp;journal=Genome%20Res.&amp;volume=13&amp;pages=2164-2170" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1390403" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1390403" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1390403"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12952883" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Reslewic</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Place</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Whole-genome shotgun optical mapping of <em>Rhodospirillum rubrum</em></div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">5511</div>-<div class="lpage">5522</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20shotgun%20optical%20mapping%20of%20Rhodospirillum%20rubrum&amp;author=S.%20Reslewic&amp;author=S.%20Zhou&amp;author=M.%20Place&amp;publication_year=2005&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=71&amp;pages=5511-5522" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.71.9.5511-5522.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.71.9.5511-5522.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.71.9.5511-5522.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16151144" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Montrocher</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Verner</div> <div class="given-names">M. C.</div></div>,  <div class="name"><div class="surname">Briolay</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Gautier</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Marmeisse</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Phylogenetic analysis of the <em>Saccharomyces cerevisiae</em> group based on polymorphisms of rDNA spacer sequences</div>, <div class="source">Int. J. Syst. Bacteriol.</div> , <div class="year">1998</div>, vol. <div class="volume">48</div> <div class="issue">Pt 1</div>(pg. <div class="fpage">295</div>-<div class="lpage">303</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20analysis%20of%20the%20Saccharomyces%20cerevisiae%20group%20based%20on%20polymorphisms%20of%20rDNA%20spacer%20sequences&amp;author=R.%20Montrocher&amp;author=M.%20C.%20Verner&amp;author=J.%20Briolay&amp;author=C.%20Gautier&amp;author=R.%20Marmeisse&amp;publication_year=1998&amp;journal=Int.%20J.%20Syst.%20Bacteriol.&amp;volume=48&amp;pages=295-303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00207713-48-1-295" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00207713-48-1-295" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00207713-48-1-295"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9542100" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rokas</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Williams</div> <div class="given-names">B. L.</div></div>,  <div class="name"><div class="surname">King</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Carroll</div> <div class="given-names">S. B.</div></div>. </div><div class="article-title">Genome-scale approaches to resolving incongruence in molecular phylogenies</div>, <div class="source">Nature</div> , <div class="year">2003</div>, vol. <div class="volume">425</div> (pg. <div class="fpage">798</div>-<div class="lpage">804</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-scale%20approaches%20to%20resolving%20incongruence%20in%20molecular%20phylogenies&amp;author=A.%20Rokas&amp;author=B.%20L.%20Williams&amp;author=N.%20King&amp;author=S.%20B.%20Carroll&amp;publication_year=2003&amp;journal=Nature&amp;volume=425&amp;pages=798-804" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02053" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02053" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02053"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14574403" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nguen</div> <div class="given-names">H. V.</div></div>,  <div class="name"><div class="surname">Gaillardin</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Two subgroups within the <em>Saccharomyces bayanus</em> species evidenced by PCR amplification and restriction polymorphism of the non-transcribed spacer2 in the ribosomal DNA unit</div>, <div class="source">System. Appl. Microbiol.</div> , <div class="year">1997</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">286</div>-<div class="lpage">294</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20subgroups%20within%20the%20Saccharomyces%20bayanus%20species%20evidenced%20by%20PCR%20amplification%20and%20restriction%20polymorphism%20of%20the%20non-transcribed%20spacer2%20in%20the%20ribosomal%20DNA%20unit&amp;author=H.%20V.%20Nguen&amp;author=C.%20Gaillardin&amp;publication_year=1997&amp;journal=System.%20Appl.%20Microbiol.&amp;volume=20&amp;pages=286-294" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0723-2020(97)80075-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0723-2020(97)80075-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0723-2020(97)80075-6"> </span></div> </div></div></div></div></div><div content-id="DSP003C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nguyen</div> <div class="given-names">H. V.</div></div>,  <div class="name"><div class="surname">Gaillardin</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Two subgroups within the <em>Saccharomyces bayanus</em> species evidenced by PCR amplification and restriction polymorphism of the non-transcribed spacer2 in the ribosomal DNA unit</div>, <div class="source">System. Appl. Microbiol.</div> , <div class="year">1997</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">286</div>-<div class="lpage">294</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20subgroups%20within%20the%20Saccharomyces%20bayanus%20species%20evidenced%20by%20PCR%20amplification%20and%20restriction%20polymorphism%20of%20the%20non-transcribed%20spacer2%20in%20the%20ribosomal%20DNA%20unit&amp;author=H.%20V.%20Nguyen&amp;author=C.%20Gaillardin&amp;publication_year=1997&amp;journal=System.%20Appl.%20Microbiol.&amp;volume=20&amp;pages=286-294" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0723-2020(97)80075-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0723-2020(97)80075-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0723-2020(97)80075-6"> </span></div> </div></div></div></div></div><div content-id="DSP003C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kellis</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Patterson</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Endrizzi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Birren</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Lander</div> <div class="given-names">E. S.</div></div>. </div><div class="article-title">Sequencing and comparison of yeast species to identify genes and regulatory elements</div>, <div class="source">Nature</div> , <div class="year">2003</div>, vol. <div class="volume">423</div> (pg. <div class="fpage">241</div>-<div class="lpage">254</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20and%20comparison%20of%20yeast%20species%20to%20identify%20genes%20and%20regulatory%20elements&amp;author=M.%20Kellis&amp;author=N.%20Patterson&amp;author=M.%20Endrizzi&amp;author=B.%20Birren&amp;author=E.%20S.%20Lander&amp;publication_year=2003&amp;journal=Nature&amp;volume=423&amp;pages=241-254" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01644" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01644" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01644"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12748633" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Török</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Mortimer</div> <div class="given-names">R. K.</div></div>,  <div class="name"><div class="surname">Romano</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Suzzi</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Polsinelli</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Quest for wine yeasts—An old story revisited</div>, <div class="source">J. Ind. Microbiol. Biotechnol.</div> , <div class="year">1996</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">303</div>-<div class="lpage">313</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quest%20for%20wine%20yeasts%E2%80%94An%20old%20story%20revisited&amp;author=T.%20T%C3%B6r%C3%B6k&amp;author=R.%20K.%20Mortimer&amp;author=P.%20Romano&amp;author=G.%20Suzzi&amp;author=M.%20Polsinelli&amp;publication_year=1996&amp;journal=J.%20Ind.%20Microbiol.%20Biotechnol.&amp;volume=17&amp;pages=303-313" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01574705" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01574705" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF01574705"> </span></div> </div></div></div></div></div><div content-id="DSP003C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wei</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">McCusker</div> <div class="given-names">J. H.</div></div>,  <div class="name"><div class="surname">Hyman</div> <div class="given-names">R. W.</div></div>, et al. </div><div class="article-title">Genome sequencing and comparative analysis of <em>Saccharomyces cerevisiae</em> strain YJM789</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2007</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">12825</div>-<div class="lpage">12830</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequencing%20and%20comparative%20analysis%20of%20Saccharomyces%20cerevisiae%20strain%20YJM789&amp;author=W.%20Wei&amp;author=J.%20H.%20McCusker&amp;author=R.%20W.%20Hyman&amp;publication_year=2007&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=104&amp;pages=12825-12830" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0701291104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0701291104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0701291104"> </span></div> </div></div></div></div></div><div content-id="DSP003C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kellis</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Birren</div> <div class="given-names">B. W.</div></div>,  <div class="name"><div class="surname">Lander</div> <div class="given-names">E. S.</div></div>. </div><div class="article-title">Proof and evolutionary analysis of ancient genome duplication in the yeast <em>Saccharomyces cerevisiae</em></div>, <div class="source">Nature</div> , <div class="year">2004</div>, vol. <div class="volume">428</div> (pg. <div class="fpage">617</div>-<div class="lpage">624</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proof%20and%20evolutionary%20analysis%20of%20ancient%20genome%20duplication%20in%20the%20yeast%20Saccharomyces%20cerevisiae&amp;author=M.%20Kellis&amp;author=B.%20W.%20Birren&amp;author=E.%20S.%20Lander&amp;publication_year=2004&amp;journal=Nature&amp;volume=428&amp;pages=617-624" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02424" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02424" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02424"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15004568" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Klein</div> <div class="given-names">H. L.</div></div>. </div><div class="article-title">Different types of recombination events are controlled by the RAD1 and RAD52 genes of Saccharomyces cerevisiae</div>, <div class="source">Genetics</div> , <div class="year">1988</div>, vol. <div class="volume">120</div> (pg. <div class="fpage">367</div>-<div class="lpage">377</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Different%20types%20of%20recombination%20events%20are%20controlled%20by%20the%20RAD1%20and%20RAD52%20genes%20of%20Saccharomyces%20cerevisiae&amp;author=H.%20L.%20Klein&amp;publication_year=1988&amp;journal=Genetics&amp;volume=120&amp;pages=367-377" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3058548" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Petes</div> <div class="given-names">T. D.</div></div>,  <div class="name"><div class="surname">Hill</div> <div class="given-names">C. W.</div></div>. </div><div class="article-title">Recombination between repeated genes in microorganisms</div>, <div class="source">Annu. Rev. Genet.</div> , <div class="year">1988</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">147</div>-<div class="lpage">168</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recombination%20between%20repeated%20genes%20in%20microorganisms&amp;author=T.%20D.%20Petes&amp;author=C.%20W.%20Hill&amp;publication_year=1988&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=22&amp;pages=147-168" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.ge.22.120188.001051" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.ge.22.120188.001051" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.ge.22.120188.001051"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3071247" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kupiec</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Petes</div> <div class="given-names">T. D.</div></div>. </div><div class="article-title">Allelic and ectopic recombination between Ty elements in yeast</div>, <div class="source">Genetics</div> , <div class="year">1988</div>, vol. <div class="volume">119</div> (pg. <div class="fpage">549</div>-<div class="lpage">559</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Allelic%20and%20ectopic%20recombination%20between%20Ty%20elements%20in%20yeast&amp;author=M.%20Kupiec&amp;author=T.%20D.%20Petes&amp;publication_year=1988&amp;journal=Genetics&amp;volume=119&amp;pages=549-559" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2841187" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Datta</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hendrix</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lipsitch</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Jinks-Robertson</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Dual roles for DNA sequence identity and the mismatch repair system in the regulation of mitotic crossing-over in yeast</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1997</div>, vol. <div class="volume">94</div> (pg. <div class="fpage">9757</div>-<div class="lpage">9762</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dual%20roles%20for%20DNA%20sequence%20identity%20and%20the%20mismatch%20repair%20system%20in%20the%20regulation%20of%20mitotic%20crossing-over%20in%20yeast&amp;author=A.%20Datta&amp;author=M.%20Hendrix&amp;author=M.%20Lipsitch&amp;author=S.%20Jinks-Robertson&amp;publication_year=1997&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=94&amp;pages=9757-9762" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.94.18.9757" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.94.18.9757" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.94.18.9757"> </span></div> </div></div></div></div></div><div content-id="DSP003C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Argueso</div> <div class="given-names">J. L.</div></div>,  <div class="name"><div class="surname">Westmoreland</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Mieczkowski</div> <div class="given-names">P. A.</div></div>,  <div class="name"><div class="surname">Gawel</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Petes</div> <div class="given-names">T. D.</div></div>,  <div class="name"><div class="surname">Resnick</div> <div class="given-names">M. A.</div></div>. </div><div class="article-title">Double-strand breaks associated with repetitive DNA can reshape the genome</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2008</div>, vol. <div class="volume">105</div> (pg. <div class="fpage">11845</div>-<div class="lpage">11850</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Double-strand%20breaks%20associated%20with%20repetitive%20DNA%20can%20reshape%20the%20genome&amp;author=J.%20L.%20Argueso&amp;author=J.%20Westmoreland&amp;author=P.%20A.%20Mieczkowski&amp;author=M.%20Gawel&amp;author=T.%20D.%20Petes&amp;author=M.%20A.%20Resnick&amp;publication_year=2008&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=105&amp;pages=11845-11850" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0804529105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0804529105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0804529105"> </span></div> </div></div></div></div></div><div content-id="DSP003C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kodama</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kielland-Brandt</div> <div class="given-names">M. C.</div></div>,  <div class="name"><div class="surname">Hansen</div> <div class="given-names">J.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Sunnerhagen</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Piskur</div> <div class="given-names">J.</div></div>. </div>, <div class="source">Top. Curr. Genet.: Lager brewing yeast</div> , <div class="year">2006</div>, vol. <div class="volume">Vol. 15</div> <div class="publisher-loc">Heidelberg</div><div class="publisher-name">Springer-Verlag</div>(pg. <div class="fpage">145</div>-<div class="lpage">164</div>)</div></div></div></div><div content-id="DSP003C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kotseruba</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Gernand</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Meister</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Houben</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Uniparental loss of ribosomal DNA in the allotetraploid grass <em>Zingeria trichopoda</em> (2<em>n</em> equals 8)</div>, <div class="source">Genome</div> , <div class="year">2003</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">156</div>-<div class="lpage">163</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Uniparental%20loss%20of%20ribosomal%20DNA%20in%20the%20allotetraploid%20grass%20Zingeria%20trichopoda%20(2n%20equals%208)&amp;author=V.%20Kotseruba&amp;author=D.%20Gernand&amp;author=A.%20Meister&amp;author=A.%20Houben&amp;publication_year=2003&amp;journal=Genome&amp;volume=46&amp;pages=156-163" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/g02-104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fg02-104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fg02-104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12669808" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cardazzo</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Minuzzo</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sartori</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Grapputo</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Carignani</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Evolution of mitochondrial DNA in yeast: gene order and structural organization of the mitochondrial genome of <em>Saccharomyces uvarum</em></div>, <div class="source">Curr. Genet.</div> , <div class="year">1998</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">52</div>-<div class="lpage">59</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20mitochondrial%20DNA%20in%20yeast%3A%20gene%20order%20and%20structural%20organization%20of%20the%20mitochondrial%20genome%20of%20Saccharomyces%20uvarum&amp;author=B.%20Cardazzo&amp;author=S.%20Minuzzo&amp;author=G.%20Sartori&amp;author=A.%20Grapputo&amp;author=G.%20Carignani&amp;publication_year=1998&amp;journal=Curr.%20Genet.&amp;volume=33&amp;pages=52-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s002940050308" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs002940050308" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs002940050308"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9472080" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Giaever</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Chu</div> <div class="given-names">A. M.</div></div>,  <div class="name"><div class="surname">Ni</div> <div class="given-names">L.</div></div>, et al. </div><div class="article-title">Functional profiling of the <em>Saccharomyces cerevisiae</em> genome</div>, <div class="source">Nature</div> , <div class="year">2002</div>, vol. <div class="volume">418</div> (pg. <div class="fpage">387</div>-<div class="lpage">391</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20profiling%20of%20the%20Saccharomyces%20cerevisiae%20genome&amp;author=G.%20Giaever&amp;author=A.%20M.%20Chu&amp;author=L.%20Ni&amp;publication_year=2002&amp;journal=Nature&amp;volume=418&amp;pages=387-391" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature00935" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature00935" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature00935"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12140549" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kodama</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Fukui</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Ashikari</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Shibano</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Improvement of maltose fermentation efficiency: constitutive expression of MAL genes in brewing yeasts</div>, <div class="source">J. Am. Soc. Brew. Chem.</div> , <div class="year">1995</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">24</div>-<div class="lpage">29</div>)</div></div></div></div><div content-id="DSP003C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dietvorst</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Londesborough</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Steensma</div> <div class="given-names">H. Y.</div></div>. </div><div class="article-title">Maltotriose utilization in lager yeast strains: <em>MTT1</em> encodes a maltotriose transporter</div>, <div class="source">Yeast</div> , <div class="year">2005</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">775</div>-<div class="lpage">788</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Maltotriose%20utilization%20in%20lager%20yeast%20strains%3A%20MTT1%20encodes%20a%20maltotriose%20transporter&amp;author=J.%20Dietvorst&amp;author=J.%20Londesborough&amp;author=H.%20Y.%20Steensma&amp;publication_year=2005&amp;journal=Yeast&amp;volume=22&amp;pages=775-788" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16088872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jespersen</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Cesar</div> <div class="given-names">L. B.</div></div>,  <div class="name"><div class="surname">Meaden</div> <div class="given-names">P. G.</div></div>,  <div class="name"><div class="surname">Jakobsen</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Multiple alpha-glucoside transporter genes in brewer’s yeast</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">1999</div>, vol. <div class="volume">65</div> (pg. <div class="fpage">450</div>-<div class="lpage">456</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20alpha-glucoside%20transporter%20genes%20in%20brewer%E2%80%99s%20yeast&amp;author=L.%20Jespersen&amp;author=L.%20B.%20Cesar&amp;author=P.%20G.%20Meaden&amp;author=M.%20Jakobsen&amp;publication_year=1999&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=65&amp;pages=450-456" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9925567" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Vidgren</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Ruohonen</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Londesborough</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Characterization and functional analysis of the MAL and MPH Loci for maltose utilization in some ale and lager yeast strains</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2005</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">7846</div>-<div class="lpage">7857</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20and%20functional%20analysis%20of%20the%20MAL%20and%20MPH%20Loci%20for%20maltose%20utilization%20in%20some%20ale%20and%20lager%20yeast%20strains&amp;author=V.%20Vidgren&amp;author=L.%20Ruohonen&amp;author=J.%20Londesborough&amp;publication_year=2005&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=71&amp;pages=7846-7857" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.71.12.7846-7857.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.71.12.7846-7857.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.71.12.7846-7857.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16332759" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Turakainen</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Korhola</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Aho</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Cloning, sequence and chromosomal location of a MEL gene from <em>Saccharomyces carlsbergensis</em> NCYC396</div>, <div class="source">Gene</div> , <div class="year">1991</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">97</div>-<div class="lpage">104</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%2C%20sequence%20and%20chromosomal%20location%20of%20a%20MEL%20gene%20from%20Saccharomyces%20carlsbergensis%20NCYC396&amp;author=H.%20Turakainen&amp;author=M.%20Korhola&amp;author=S.%20Aho&amp;publication_year=1991&amp;journal=Gene&amp;volume=101&amp;pages=97-104" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(91)90229-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(91)90229-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(91)90229-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1711992" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Goncalves</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Rodrigues de Sousa</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Spencer-Martins</div> <div class="given-names">I.</div></div>. </div><div class="article-title">FSY1, a novel gene encoding a specific fructose/H(+) symporter in the type strain of <em>Saccharomyces carlsbergensis</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2000</div>, vol. <div class="volume">182</div> (pg. <div class="fpage">5628</div>-<div class="lpage">5630</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=FSY1%2C%20a%20novel%20gene%20encoding%20a%20specific%20fructose%2FH(%2B)%20symporter%20in%20the%20type%20strain%20of%20Saccharomyces%20carlsbergensis&amp;author=P.%20Goncalves&amp;author=H.%20Rodrigues%20de%20Sousa&amp;author=I.%20Spencer-Martins&amp;publication_year=2000&amp;journal=J.%20Bacteriol.&amp;volume=182&amp;pages=5628-5630" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.182.19.5628-5630.2000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.182.19.5628-5630.2000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.182.19.5628-5630.2000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10986274" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Omura</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Hatanaka</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Nakao</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Characterization of a novel tyrosine permease of lager brewing yeast shared by <em>Saccharomyces cerevisiae</em> strain RM11-1a</div>, <div class="source">FEMS Yeast Res.</div> , <div class="year">2007</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">1350</div>-<div class="lpage">1361</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20a%20novel%20tyrosine%20permease%20of%20lager%20brewing%20yeast%20shared%20by%20Saccharomyces%20cerevisiae%20strain%20RM11-1a&amp;author=F.%20Omura&amp;author=H.%20Hatanaka&amp;author=Y.%20Nakao&amp;publication_year=2007&amp;journal=FEMS%20Yeast%20Res.&amp;volume=7&amp;pages=1350-1361" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fyr.2007.7.issue-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffyr.2007.7.issue-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffyr.2007.7.issue-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17825063" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yoshida</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hashimoto</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tanaka-Kanai</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yoshimoto</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kobayashi</div> <div class="given-names">O.</div></div>. </div><div class="article-title">Identification and characterization of amidase- homologous <em>AMI1</em> genes of bottom-fermenting yeast</div>, <div class="source">Yeast</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">1075</div>-<div class="lpage">1084</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20amidase-%20homologous%20AMI1%20genes%20of%20bottom-fermenting%20yeast&amp;author=S.%20Yoshida&amp;author=K.%20Hashimoto&amp;author=K.%20Tanaka-Kanai&amp;author=H.%20Yoshimoto&amp;author=O.%20Kobayashi&amp;publication_year=2007&amp;journal=Yeast&amp;volume=24&amp;pages=1075-1084" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17924455" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Halestrap</div> <div class="given-names">A. P.</div></div>,  <div class="name"><div class="surname">Price</div> <div class="given-names">N. T.</div></div>. </div><div class="article-title">The proton-linked monocarboxylate transporter (MCT) family: structure, function and regulation</div>, <div class="source">Biochem. J.</div> , <div class="year">1999</div>, vol. <div class="volume">343</div> <div class="issue">Pt 2</div>(pg. <div class="fpage">281</div>-<div class="lpage">299</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20proton-linked%20monocarboxylate%20transporter%20(MCT)%20family%3A%20structure%2C%20function%20and%20regulation&amp;author=A.%20P.%20Halestrap&amp;author=N.%20T.%20Price&amp;publication_year=1999&amp;journal=Biochem.%20J.&amp;volume=343&amp;pages=281-299" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1042/bj3430281" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1042%2Fbj3430281" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1042%2Fbj3430281"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10510291" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dunn</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Sherlock</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Reconstruction of the genome origins and evolution of the hybrid lager yeast Saccharomyces pastorianus</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">1610</div>-<div class="lpage">1623</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reconstruction%20of%20the%20genome%20origins%20and%20evolution%20of%20the%20hybrid%20lager%20yeast%20Saccharomyces%20pastorianus&amp;author=B.%20Dunn&amp;author=G.%20Sherlock&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=1610-1623" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.076075.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.076075.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.076075.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18787083" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">de Barros Lopes</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Bellon</div> <div class="given-names">J. R.</div></div>,  <div class="name"><div class="surname">Shirley</div> <div class="given-names">N. J.</div></div>,  <div class="name"><div class="surname">Ganter</div> <div class="given-names">P. F.</div></div>. </div><div class="article-title">Evidence for multiple interspecific hybridization in <em>Saccharomyces sensu stricto</em> species</div>, <div class="source">FEMS Yeast Res.</div> , <div class="year">2002</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">323</div>-<div class="lpage">331</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20multiple%20interspecific%20hybridization%20in%20Saccharomyces%20sensu%20stricto%20species&amp;author=M.%20de%20Barros%20Lopes&amp;author=J.%20R.%20Bellon&amp;author=N.%20J.%20Shirley&amp;author=P.%20F.%20Ganter&amp;publication_year=2002&amp;journal=FEMS%20Yeast%20Res.&amp;volume=1&amp;pages=323-331" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fyr.2002.1.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffyr.2002.1.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffyr.2002.1.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12702336" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rainieri</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kodama</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kaneko</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Mikata</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nakao</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ashikari</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Pure and mixed genetic lines of <em>Saccharomyces bayanus</em> and <em>Saccharomyces pastorianus</em> and their contribution to the lager brewing strain genome</div>, <div class="source">Appl. Environ. Microbiol.</div> , <div class="year">2006</div>, vol. <div class="volume">72</div> (pg. <div class="fpage">3968</div>-<div class="lpage">3974</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pure%20and%20mixed%20genetic%20lines%20of%20Saccharomyces%20bayanus%20and%20Saccharomyces%20pastorianus%20and%20their%20contribution%20to%20the%20lager%20brewing%20strain%20genome&amp;author=S.%20Rainieri&amp;author=Y.%20Kodama&amp;author=Y.%20Kaneko&amp;author=K.%20Mikata&amp;author=Y.%20Nakao&amp;author=T.%20Ashikari&amp;publication_year=2006&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=72&amp;pages=3968-3974" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.02769-05" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.02769-05" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.02769-05"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16751504" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pulvirenti</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Caggia</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Restuccia</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Giudici</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Zambonelli</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Inheritance of mitochondrial DNA in interspecific <em>Saccharomyces</em> hybrids</div>, <div class="source">Ann. Microbiol.</div> , <div class="year">2000</div>, vol. <div class="volume">50</div> (pg. <div class="fpage">61</div>-<div class="lpage">64</div>)</div></div></div></div><div content-id="DSP003C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rainieri</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kodama</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nakao</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Pulvirenti</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Giudici</div> <div class="given-names">P.</div></div>. </div><div class="article-title">The inheritance of mtDNA in lager brewing strains</div>, <div class="source">FEMS Yeast Res.</div> , <div class="year">2008</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">586</div>-<div class="lpage">596</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20inheritance%20of%20mtDNA%20in%20lager%20brewing%20strains&amp;author=S.%20Rainieri&amp;author=Y.%20Kodama&amp;author=Y.%20Nakao&amp;author=A.%20Pulvirenti&amp;author=P.%20Giudici&amp;publication_year=2008&amp;journal=FEMS%20Yeast%20Res.&amp;volume=8&amp;pages=586-596" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fyr.2008.8.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffyr.2008.8.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffyr.2008.8.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18318709" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wolfe</div> <div class="given-names">K. H.</div></div>,  <div class="name"><div class="surname">Shields</div> <div class="given-names">D. C.</div></div>. </div><div class="article-title">Molecular evidence for an ancient duplication of the entire yeast genome</div>, <div class="source">Nature</div> , <div class="year">1997</div>, vol. <div class="volume">387</div> (pg. <div class="fpage">708</div>-<div class="lpage">713</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20evidence%20for%20an%20ancient%20duplication%20of%20the%20entire%20yeast%20genome&amp;author=K.%20H.%20Wolfe&amp;author=D.%20C.%20Shields&amp;publication_year=1997&amp;journal=Nature&amp;volume=387&amp;pages=708-713" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/42711" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F42711" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F42711"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9192896" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rainieri</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Pulvirenti</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Giudici</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Construction of model lager brewing strains by interspecific hybridization</div>, <div class="source">Yeast</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">01</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20model%20lager%20brewing%20strains%20by%20interspecific%20hybridization&amp;author=S.%20Rainieri&amp;author=A.%20Pulvirenti&amp;author=P.%20Giudici&amp;publication_year=2007&amp;journal=Yeast&amp;volume=24&amp;pages=01-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0061"> </span></div> </div></div></div></div></div><div content-id="DSP003C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dunham</div> <div class="given-names">M. J.</div></div>,  <div class="name"><div class="surname">Badrane</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ferea</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Characteristic genome rearrangements in experimental evolution of <em>Saccharomyces cerevisiae</em></div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2002</div>, vol. <div class="volume">99</div> (pg. <div class="fpage">16144</div>-<div class="lpage">16149</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characteristic%20genome%20rearrangements%20in%20experimental%20evolution%20of%20Saccharomyces%20cerevisiae&amp;author=M.%20J.%20Dunham&amp;author=H.%20Badrane&amp;author=T.%20Ferea&amp;publication_year=2002&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=99&amp;pages=16144-16149" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.242624799" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.242624799" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.242624799"> </span></div> </div></div></div></div></div><div content-id="DSP003C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Infante</div> <div class="given-names">J. J.</div></div>,  <div class="name"><div class="surname">Dombek</div> <div class="given-names">K. M.</div></div>,  <div class="name"><div class="surname">Rebordinos</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Cantoral</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Young</div> <div class="given-names">E. T.</div></div>. </div><div class="article-title">Genome-wide amplifications caused by chromosomal rearrangements play a major role in the adaptive evolution of natural yeast</div>, <div class="source">Genetics</div> , <div class="year">2003</div>, vol. <div class="volume">165</div> (pg. <div class="fpage">1745</div>-<div class="lpage">1759</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20amplifications%20caused%20by%20chromosomal%20rearrangements%20play%20a%20major%20role%20in%20the%20adaptive%20evolution%20of%20natural%20yeast&amp;author=J.%20J.%20Infante&amp;author=K.%20M.%20Dombek&amp;author=L.%20Rebordinos&amp;author=J.%20M.%20Cantoral&amp;author=E.%20T.%20Young&amp;publication_year=2003&amp;journal=Genetics&amp;volume=165&amp;pages=1745-1759" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14704163" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Perez-Ortin</div> <div class="given-names">J. E.</div></div>,  <div class="name"><div class="surname">Querol</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Puig</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Barrio</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Molecular characterization of a chromosomal rearrangement involved in the adaptive evolution of yeast strains</div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">1533</div>-<div class="lpage">1539</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20characterization%20of%20a%20chromosomal%20rearrangement%20involved%20in%20the%20adaptive%20evolution%20of%20yeast%20strains&amp;author=J.%20E.%20Perez-Ortin&amp;author=A.%20Querol&amp;author=S.%20Puig&amp;author=E.%20Barrio&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=1533-1539" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.436602" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.436602" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.436602"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12368245" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McMurray</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Gottschling</div> <div class="given-names">D. E.</div></div>. </div><div class="article-title">An age-induced switch to a hyper-recombinational state</div>, <div class="source">Science</div> , <div class="year">2003</div>, vol. <div class="volume">301</div> (pg. <div class="fpage">1908</div>-<div class="lpage">1911</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20age-induced%20switch%20to%20a%20hyper-recombinational%20state&amp;author=M.%20A.%20McMurray&amp;author=D.%20E.%20Gottschling&amp;publication_year=2003&amp;journal=Science&amp;volume=301&amp;pages=1908-1911" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1087706" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1087706" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1087706"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14512629" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sato</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kishimoto</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Watari</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Takashio</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Breeding of brewer’s yeast by hybridization between a top-fermenting yeast <em>Saccharomyces cerevisiae</em> and a cryophilic yeast <em>Saccharomyces bayanus</em></div>, <div class="source">J. Biosci. Bioeng.</div> , <div class="year">2002</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">509</div>-<div class="lpage">511</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Breeding%20of%20brewer%E2%80%99s%20yeast%20by%20hybridization%20between%20a%20top-fermenting%20yeast%20Saccharomyces%20cerevisiae%20and%20a%20cryophilic%20yeast%20Saccharomyces%20bayanus&amp;author=M.%20Sato&amp;author=M.%20Kishimoto&amp;author=J.%20Watari&amp;author=M.%20Takashio&amp;publication_year=2002&amp;journal=J.%20Biosci.%20Bioeng.&amp;volume=93&amp;pages=509-511" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1389-1723(02)80101-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1389-1723(02)80101-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1389-1723(02)80101-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16233241" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSP003C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSP003C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakao</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kodama</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Shimonaga</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Gene expression analysis of lager brewing yeast under different oxygenation condition using newly developed DNA microarray</div>, <div class="source">Proc. Eur. Conv. Congr.</div> , <div class="year">2007</div>(pg. <div class="fpage">406</div>-<div class="lpage">419</div>)</div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2009. Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=510928&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp003/2/dsp003-dsp003_Table5.xls?Expires=2147483647&amp;Signature=u-eIUalUWiyah13-~iPkxsXrgUxfQsCKFD0NtG3pG~t30-f2UO70j04nL3BJDBrK6~p9dQnOKQg3VAliAGeOrMPtT9WC6Q3XbKHPrjxY-5WyVQmnbVjfMZJfKyQM2yYd9Bp4kH7QvTUBuqrLpgFkFDEEquNAWaJ7ndXP3yhhEdw2WDY0mo0LA6HGNK~XuKKUDjWISrMoWKQHpV9x-dKiAnG-X5CCxwLVi2O4EaGWYTOVNFKkaFZLg75NFnvR7yplvIcNWBN0KQfmhWuKWGctTaTDx6OnjtsvDkTWzkap-Y02FvVRF3qwErMzNLv4oEgi-4ve6AiYdiZawQiEHWC6Rg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp003/2/dsp003-dsp003_Table4.pdf?Expires=2147483647&amp;Signature=KG7QcgOe5OCpoC6RUBe9HfRVpIm3wd6lL6OVCe3I1tbuh8z1S9ptADGzdz3kGnh~Bu72S8Fmdl-tnWGxyU9s6q-Rgof4eDTuwz3RBZ7rVkRLWpXwoiY0EuXGNWsEX2yKzunaP~NBFXgBJRBwoMP~kdcVeAiBro9F8QZneLNnxncPTzGoJznOWemqsZ3DsrKsdxR10M4g3OPwS0zrQbuy~aGWayqIF7tHGozH9PQuzHhbPflFns7Hos55sgwOgJL79JPfGJ03azZAhZD8fy51-tWFToeSgs7y3ahjPsBHQ4~bppa-b5HfI~v5UidcEBBMiDngS57IWRBCwNIWDk8vZw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp003/2/dsp003-dsp003_Table3.xls?Expires=2147483647&amp;Signature=oHRW~PDtqLfkg6XdkZhly~3pYzyke~hGWVZk7RVgorBk~IEKasHxy5s4FtUkT1-UOc9w6w1YLtqc8PHFClOeoKVlqkAu2qJpW29YkzyzMO0JCP8xQALuhzFFIMv6AYBYKkWxpudMT-lTqU0kD7fs69ahFBcu8A2ef6tLfRBADkix~S4aUi0CjXoxc2MjBLeSDKUyNK0EOagwdpMvCi7~LLTTo0hM57leUHndcbM25CycSwQDrpyOq~PBT5miU9fBCtdK7FE37xtpT8Wv0YwlPVdIvKq7eTtkP7~Dwe4kkka84dXREDckI58Zi4FiNz7pWAjCOb9QX1V5z-w7TVu8Hw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp003/2/dsp003-dsp003_Table2.xls?Expires=2147483647&amp;Signature=Oigd3JYvgzj9hgmfv7DS37ZGkhurRPw1S6KNtbEsLtBt77xX9~jzsIJ5CTgX22zeG10XI9rgPq7smgjQgkmBHccL14Dya9EH-d4yp9DVo3tbZSS7OtJNaGaNH8W1vwMutrUpCXhbVXYXOCSit~c1Rk2J~z0hvUwG5QAq-83l9yewQ4SzQkhTvKBju~dpz3zyEAqb7~aKZ66NiPKQgWNZ2kG3P3gEI1ZWWS-09tAkwHUKm22RF3NUnfEpWckKtZXVXbMv5XMgmjGVZDpYxZV3Fidy9AYx3cja5bkRQLHccIPHjIzU0XacdB6drM57f23Mna6MbVyvJLwjWaJbi5L3ag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp003/2/dsp003-dsp003_Table1.xls?Expires=2147483647&amp;Signature=cLNbONf8TwFIHKyOd08-pwg~lP-WOakPHoFLp4LUSCpvYzPCLPrXkiicSsmicex2gOovGa49vLSYBeN6w0tXBT8S2PnxXX4THfMx4p8JFRDeGUsVc409XAladEHuIGyAtL2QxNIQ-hano4zZr1CLY~bXVh8g2Zq1ZG6ljcNFYV0wER5d-rJFzILOi2sZ6HtXbt8ze5QVYneB8BRji8LmD3~19Fi54dh1bYKsIXnC2dguhmvszIYItTC3BzuCHcZBWwV51IiWQi3Fz8o3RkuECvrxyB-DpWWywct0qQ6gbJTgsSL6~~TgpTL5CA30~de9cUDZ~7aFOnTOIHsLQvEgew__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp003/2/dsp003-dsp003_supp_methods.pdf?Expires=2147483647&amp;Signature=uZe~4v9g5PWOlYL3EMcByUrwmMJ4cPaJOX7wWd8nJxjXz4ibppCop5VsviR4EMskZ6T~RMckGL0ek9K7YAVwRWUjow38r~CvSGzCT6PZAZMYpXHz61To9XNi5tEBNySBKYXs1VKamk6ldIc8ElBr-0UaqopxiDn9eBF8mX8AI35PgKF-9T5vsGTlOLwBrOeoCByJBh1kH6~Dx~gfZ9Dp9IgTcNr-hrxqX7K9IDUoEELYCOGCkazXQ4HqLr53~wjESDAyftXxmnGz6NsHeURPQoplFcaOAFBjaKcjK~Kf4H2jss3VZLPerqAVIyyYMsvPH77rYuZfVV-R0rPJWiEJ6Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp003/2/dsp003-dsp003_supp_figures.pdf?Expires=2147483647&amp;Signature=AlNJ5RV8loMK89gHl9CxI8uTKqzTXsb-us5AozxBliaB5lr5oEBf18e0EkucxkE0B7Wf9pNJf73Z09aKzyLINE2X1DxZTco6vr1HJ06Yu3TQEAUJthcWJ930uRC-Fbl284IzsnENzHj-xjcNMIG8yDfEvWfkJ5J6YWF~YwhpqLiTwaH3~TdskU4swYJgXmk5ZuUX8V6mWVZ~a8OCogWlj96fIMPyz1s4IWmU8DRE9iB95z7a2lWo5rrFG23CBHZG2wYnTUtRS2ldoiuldkfOG1LMXgceQXmIkuzv3W6Xl0VJfQYqSkHqghKBAEvXg8Q~D95mIeijNuir826~geKQEg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/16/2/10.1093/dnares/dsp003/2/dsp003-dsp003_supp.pdf?Expires=2147483647&amp;Signature=k07cwolcUu0l-CC5ifPnggG86g~tYUxPPX~gjSJ3UwUwKNQDlNDGtAqjKIMsjS~x39Cb53b1h0E9ME7Gqz82Sb3a6NkN9UKncZrgU8ZpxUIY9iS5iZ7P41d1h-1gv571hC0k6-c7nUjoJBObRbndBEsjKW0tHh9FC5EW6XjTNk5voiA2EJnTeyS-0FzttXV5JbSKKfp4F0lX4qOXad-Kc198p0R89eNVBx7X7H3fWklXP6XalACTmpLC0XDmXHIY1PxXFRM54Kk-yhZYklHtjxPcJE06MO2-SiiewJb72f9dIYVcIk-ySkqWpUOD6I01MBnrj7G0VOVia5u0uGTK~g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">781</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000265770800004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">161</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsp003"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="510928"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000265770800004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f16%2f2%2f115%2f510928">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000265770800004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (161)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F16%2F2%2F115%2F510928" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/510928'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome Sequence of the Lager Brewing Yeast, an Interspecies Hybrid" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsp003");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f16%2f2%2f115%2f510928">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

