

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/2/195/4638432","@type":"ScholarlyArticle","name":"Genome scale analysis of  Escherichia coli  with a comprehensive prokaryotic sequence-based biophysical model of translation initiation and elongation","datePublished":"2017-11-17","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2018-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx049","keywords":["translation","biophysical model","transcript evolution","UTR","mRNA folding"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/2/195/4638432","author":[{"name":"Shaham, Gilad","affiliation":"Department of Biomedical Engineering, The Engineering Faculty, Tel Aviv University, Israel","@type":"Person"},{"name":"Tuller, Tamir","affiliation":"Department of Biomedical Engineering, The Engineering Faculty, Tel Aviv University, Israel  The Sagol School of Neuroscience, Tel-Aviv University, Tel-Aviv, Israel","@type":"Person"}],"description":"Abstract.  Translation initiation in prokaryotes is affected by the mRNA folding and interaction of the ribosome binding site with the ribosomal RNA. The elonga","pageStart":"195","pageEnd":"205","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/2/1/m_dnares_25_2cover.png?Expires=2147483647&Signature=yfSnSDuEb8Ru1iEiczh9QiVIRujFlv5eQUGA2PixxhIP2ifTZWTC6Dh4YwZbNdfMcc6NjI5nmXhBBnXl~MO-9TlTyrmfE~hJJOwCiRzc0uAKzzN1WszCYLPzg4YPUGBBBvmjvYjtWeuxzT6HMUtRCVQ1Lykx1-FxL9MPbgabfK3LfM0lfiO9VFwha2rRByA4174pANV1EEHkTLYoj9VZAqG9NluBapcroduGQ77QCNh6n6sU6cwRapmFfSHO~HcCIdSas62HDXNUy3sCegaV5-lnkgDK34Vs8bE~hWbIr-4dZ69GanJYFX-rhZ4a7s6UpCy38dZKnOnrMjKCPyIBeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome scale analysis of  Escherichia coli  with a comprehensive prokaryotic sequence-based biophysical model of translation initiation and elongation","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/2/1/m_dnares_25_2cover.png?Expires=2147483647&Signature=yfSnSDuEb8Ru1iEiczh9QiVIRujFlv5eQUGA2PixxhIP2ifTZWTC6Dh4YwZbNdfMcc6NjI5nmXhBBnXl~MO-9TlTyrmfE~hJJOwCiRzc0uAKzzN1WszCYLPzg4YPUGBBBvmjvYjtWeuxzT6HMUtRCVQ1Lykx1-FxL9MPbgabfK3LfM0lfiO9VFwha2rRByA4174pANV1EEHkTLYoj9VZAqG9NluBapcroduGQ77QCNh6n6sU6cwRapmFfSHO~HcCIdSas62HDXNUy3sCegaV5-lnkgDK34Vs8bE~hWbIr-4dZ69GanJYFX-rhZ4a7s6UpCy38dZKnOnrMjKCPyIBeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome scale analysis of  Escherichia coli  with a comprehensive prokaryotic sequence-based biophysical model of translation initiation and elongation" />
    <meta property="og:description" content="Abstract.  Translation initiation in prokaryotes is affected by the mRNA folding and interaction of the ribosome binding site with the ribosomal RNA. The elonga" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/2/195/4638432" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-11-17" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Shaham, Gilad" /><meta name="citation_author_institution" content="Department of Biomedical Engineering, The Engineering Faculty, Tel Aviv University, Israel" /><meta name="citation_author" content="Tuller, Tamir" /><meta name="citation_author_institution" content="Department of Biomedical Engineering, The Engineering Faculty, Tel Aviv University, Israel" /><meta name="citation_author_institution" content="The Sagol School of Neuroscience, Tel-Aviv University, Tel-Aviv, Israel" /><meta name="citation_title" content="Genome scale analysis of Escherichia coli with a comprehensive prokaryotic sequence-based biophysical model of translation initiation and elongation" /><meta name="citation_firstpage" content="195" /><meta name="citation_lastpage" content="205" /><meta name="citation_doi" content="10.1093/dnares/dsx049" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2018/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Strategies for optimizing heterologous protein expression in Escherichia coli;  citation_journal_title=Trends Biotechnol;  citation_year=1998;  citation_volume=16;  citation_pages=54-60; " /><meta name="citation_reference" content="citation_title=Initiation of translation in prokaryotes and eukaryotes;  citation_journal_title=Gene;  citation_year=1999;  citation_volume=234;  citation_pages=187-208; " /><meta name="citation_reference" content="citation_title=Determinant of cistron specificity in bacterial ribosomes;  citation_journal_title=Nature;  citation_year=1975;  citation_volume=254;  citation_pages=34-8; " /><meta name="citation_reference" content="citation_title=Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the E. coli translational system;  citation_journal_title=J. Mol. Biol;  citation_year=1981;  citation_volume=151;  citation_pages=389-409; " /><meta name="citation_reference" content="citation_title=Codon usage in bacteria: correlation with gene expressivity;  citation_journal_title=Nucleic Acids Res;  citation_year=1982;  citation_volume=10;  citation_pages=7055-74; " /><meta name="citation_reference" content="citation_title=The effect of tRNA levels on decoding times of mRNA codons;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=9171-81; " /><meta name="citation_reference" content="citation_title=Multiple roles of the coding sequence 5’ end in gene expression regulation;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=13-28; " /><meta name="citation_reference" content="citation_title=Codon bias and heterologous protein expression;  citation_journal_title=Trends Biotechnol;  citation_year=2004;  citation_volume=22;  citation_pages=346-53; " /><meta name="citation_reference" content="citation_title=A “Silent” Polymorphism in the MDR1 gene changes substrate specificity;  citation_journal_title=Science;  citation_year=2007;  citation_volume=315;  citation_pages=525-8; " /><meta name="citation_reference" content="citation_title=Viral adaptation to host: a proteome-based analysis of codon usage and amino acid preferences;  citation_journal_title=Mol. Syst. Biol;  citation_year=2009;  citation_volume=5;  citation_pages=311" /><meta name="citation_reference" content="citation_title=Differential arginylation of actin isoforms is regulated by coding sequence-dependent degradation;  citation_journal_title=Science;  citation_year=2010;  citation_volume=329;  citation_pages=1534-7; " /><meta name="citation_reference" content="citation_title=How the sequence of a gene can tune its translation;  citation_journal_title=Cell;  citation_year=2010;  citation_volume=141;  citation_pages=227-9; " /><meta name="citation_reference" content="citation_title=Predictive biophysical modeling and understanding of the dynamics of mRNA translation and its evolution;  citation_journal_title=Nucleic Acids Res;  citation_year=2016;  citation_volume=44;  citation_pages=9031-49; " /><meta name="citation_reference" content="citation_title=Genome-scale analysis of translation elongation with a ribosome flow model wasserman,;  citation_journal_title=PLoS Comput. Biol;  citation_year=2011;  citation_volume=7;  citation_pages=e1002127" /><meta name="citation_reference" content="citation_title=The codon Adaptation Index—a measure of directional synonymous codon usage bias, and its potential applications;  citation_journal_title=Nucl. Acids Res;  citation_year=1987;  citation_volume=15;  citation_pages=1281-95; " /><meta name="citation_reference" content="citation_title=Totally asymmetric exclusion process with extended objects: a model for protein synthesis;  citation_journal_title=Phys. Rev. E;  citation_year=2003;  citation_volume=68;  citation_pages=021910" /><meta name="citation_reference" content="citation_title=Solving the riddle of codon usage preferences: a test for translational selection;  citation_journal_title=Nucleic Acids Res;  citation_year=2004;  citation_volume=32;  citation_pages=5036-44; " /><meta name="citation_reference" content="citation_title=Mean of the typical decoding rates: a new translation efficiency index based on the analysis of ribosome profiling data;  citation_journal_title=G3 Bethesda Md;  citation_year=2014;  citation_volume=5;  citation_pages=73-80; " /><meta name="citation_reference" content="citation_title=Automated design of synthetic ribosome binding sites to control protein expression;  citation_journal_title=Nat. Biotechnol;  citation_year=2009;  citation_volume=27;  citation_pages=946-50; " /><meta name="citation_reference" content="citation_title=A universal trend of reduced mRNA stability near the translation-initiation site in prokaryotes and eukaryotes;  citation_journal_title=PLoS Comput. Biol;  citation_year=2010;  citation_volume=6;  citation_pages=e1000664" /><meta name="citation_reference" content="citation_publisher=Elsevier, ;  citation_title=The Ribosome Binding Site CalculatorMethods in Enzymology;  citation_year=2011;  citation_pages=19-42; " /><meta name="citation_reference" content="citation_title=New universal rules of eukaryotic translation initiation fidelity;  citation_journal_title=PLoS Comput. Biol;  citation_year=2013;  citation_volume=9;  citation_pages=e1003136" /><meta name="citation_reference" content="citation_title=Sequence signatures and mRNA concentration can explain two‐thirds of protein abundance variation in a human cell line;  citation_journal_title=Mol. Syst. Biol;  citation_year=2010;  citation_volume=6;  citation_pages=400" /><meta name="citation_reference" content="citation_title=Analysis and prediction of translation rate based on sequence and functional features of the mRNA Kudla,;  citation_journal_title=PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e16036" /><meta name="citation_reference" content="citation_title=Transcript features alone enable accurate prediction and understanding of gene expression in S. cerevisiae;  citation_journal_title=BMC Bioinformatics;  citation_year=2013;  citation_volume=14;  citation_pages=S1" /><meta name="citation_reference" content="citation_title=ViennaRNA Package 2.0;  citation_journal_title=Algorithms Mol. Biol;  citation_year=2011;  citation_volume=6;  citation_pages=26" /><meta name="citation_reference" content="citation_title=Cell size and nucleoid organization of engineered Escherichia coli cells with a reduced genome: engineered E. coli with a reduced genome;  citation_journal_title=Mol. Microbiol;  citation_year=2004;  citation_volume=55;  citation_pages=137-49; " /><meta name="citation_reference" content="citation_title=Construction of consecutive deletions of the Escherichia coli chromosome;  citation_journal_title=Mol. Syst. Biol;  citation_year=2007;  citation_volume=3;  citation_pages=132" /><meta name="citation_reference" content="citation_title=A Comprehensive, quantitative, and genome-wide model of translation;  citation_journal_title=PLoS Comput. Biol;  citation_year=2010;  citation_volume=6;  citation_pages=e1000865" /><meta name="citation_reference" content="citation_title=Transimulation -protein biosynthesis web service,;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e73943" /><meta name="citation_reference" content="citation_title=Properties and determinants of codon decoding time distributions;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=S13" /><meta name="citation_reference" content="citation_title=Minimally invasive determination of mRNA concentration in single living bacteria;  citation_journal_title=Nucleic Acids Res;  citation_year=2008;  citation_volume=36;  citation_pages=e73" /><meta name="citation_reference" content="citation_title=RegulonDB version 9.0: high-level integration of gene regulation, coexpression, motif clustering and beyond;  citation_journal_title=Nucleic Acids Res;  citation_year=2016;  citation_volume=44;  citation_pages=D133-43; " /><meta name="citation_reference" content="citation_title=PaxDb, a database of protein abundance averages across all three domains of life;  citation_journal_title=Mol. Cell. Proteomics;  citation_year=2012;  citation_volume=11;  citation_pages=492-500; " /><meta name="citation_reference" content="citation_title=Quantifying E. coli proteome and transcriptome with single-molecule sensitivity in single cells;  citation_journal_title=Science;  citation_year=2010;  citation_volume=329;  citation_pages=533-8; " /><meta name="citation_reference" content="citation_title=The anti-Shine–Dalgarno sequence drives translational pausing and codon choice in bacteria;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=484;  citation_pages=538-41; " /><meta name="citation_reference" content="citation_title=Cutadapt removes adapter sequences from high-throughput sequencing reads;  citation_journal_title=Embnet. J;  citation_year=2011;  citation_volume=17;  citation_pages=10" /><meta name="citation_reference" content="citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome;  citation_journal_title=Genome Biol;  citation_year=2009;  citation_volume=10;  citation_pages=R25" /><meta name="citation_reference" content="citation_title=Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources;  citation_journal_title=Nat. Protoc;  citation_year=2008;  citation_volume=4;  citation_pages=44-57; " /><meta name="citation_reference" content="citation_title=Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists;  citation_journal_title=Nucleic Acids Res;  citation_year=2009;  citation_volume=37;  citation_pages=1-13; " /><meta name="citation_reference" content="citation_title=Global quantification of mammalian gene expression control;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=473;  citation_pages=337-42; " /><meta name="citation_reference" content="citation_title=Translation elongation can control translation initiation on eukaryotic mRNAs;  citation_journal_title=Embo J;  citation_year=2014;  citation_volume=33;  citation_pages=21-34; " /><meta name="citation_reference" content="citation_title=Translation initiation in Escherichia coli: old and new questions;  citation_journal_title=Mol. Microbiol;  citation_year=1990;  citation_volume=4;  citation_pages=1063-7; " /><meta name="citation_reference" content="citation_title=The efficiency of folding of some proteins is increased by controlled rates of translation in vivo. A hypothesis;  citation_journal_title=J. Mol. Biol;  citation_year=1987;  citation_volume=193;  citation_pages=413-7; " /><meta name="citation_reference" content="citation_title=Protein structure and the sequential structure of mRNA: alpha-helix and beta-sheet signals at the nucleotide level;  citation_journal_title=Proteins;  citation_year=1996;  citation_volume=25;  citation_pages=237-52; " /><meta name="citation_reference" content="citation_title=Ribosome-mediated translational pause and protein domain organization;  citation_journal_title=Protein Sci.;  citation_year=1996;  citation_volume=5;  citation_pages=1594-612; " /><meta name="citation_reference" content="citation_title=Protein secondary structural types are differentially coded on messenger RNA;  citation_journal_title=Protein Sci.;  citation_year=1996;  citation_volume=5;  citation_pages=1973-83; " /><meta name="citation_reference" content="citation_title=Non-random usage of “degenerate” codons is related to protein three-dimensional structure;  citation_journal_title=FEBS Lett;  citation_year=1996;  citation_volume=399;  citation_pages=78-82; " /><meta name="citation_reference" content="citation_title=A model for competition for ribosomes in the cell;  citation_journal_title=J. R Soc. Interface;  citation_year=2016;  citation_volume=13;  citation_pages=20151062" /><meta name="citation_reference" content="citation_title=Translation initiation in Escherichia coli: sequences within the ribosome-binding site;  citation_journal_title=Mol. Microbiol;  citation_year=1992;  citation_volume=6;  citation_pages=1219-29; " /><meta name="citation_reference" content="citation_title=Determination of the optimal aligned spacing between the Shine-Dalgarno sequence and the translation initiation codon of Escherichia coli mRNAs;  citation_journal_title=Nucl. Acids Res;  citation_year=1994;  citation_volume=22;  citation_pages=4953-7; " /><meta name="citation_reference" content="citation_title=Correlations between Shine-Dalgarno sequences and gene features such as predicted expression levels and operon structures;  citation_journal_title=J. Bacteriol;  citation_year=2002;  citation_volume=184;  citation_pages=5733-45; " /><meta name="citation_reference" content="citation_title=Specialized ribosome system: preferential translation of a single mRNA species by a subpopulation of mutated ribosomes in Escherichia coli;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=1987;  citation_volume=84;  citation_pages=4762-6; " /><meta name="citation_reference" content="citation_title=Translational initiation on structured messengers: another role for the shine-dalgarno interaction;  citation_journal_title=J. Mol. Biol;  citation_year=1994;  citation_volume=235;  citation_pages=173-84; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling;  citation_journal_title=Science;  citation_year=2009;  citation_volume=324;  citation_pages=218-23; " /><meta name="citation_reference" content="citation_title=An evolutionarily conserved mechanism for controlling the efficiency of protein translation;  citation_journal_title=Cell;  citation_year=2010;  citation_volume=141;  citation_pages=344-54; " /><meta name="citation_reference" content="citation_title=Co-translational folding;  citation_journal_title=Curr. Opin. Struct. Biol;  citation_year=1999;  citation_volume=9;  citation_pages=111-4; " /><meta name="citation_reference" content="citation_title=Discrete nascent chain lengths are required for the insertion of presecretory proteins into microsomal membranes;  citation_journal_title=J. Cell Biol;  citation_year=1993;  citation_volume=121;  citation_pages=1211-9; " /><meta name="citation_reference" content="citation_publisher=Garland Science, New York, NY;  citation_title=Molecular Biology of the Cell;  citation_year=2002; " /><meta name="citation_reference" content="citation_title=Translation arrest of SecM is essential for the basal and regulated expression of SecA;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=12330-5; " /><meta name="citation_reference" content="citation_title=Individual comparisons by ranking methods;  citation_journal_title=Biom. Bull;  citation_year=1945;  citation_volume=1;  citation_pages=80" /><meta name="citation_reference" content="citation_title=Real-time tRNA transit on single translating ribosomes at codon resolution;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=464;  citation_pages=1012-7; " /><meta name="citation_reference" content="citation_title=Most associations between transcript features and gene expression are monotonic;  citation_journal_title=Mol. Biosyst;  citation_year=2014;  citation_volume=10;  citation_pages=1426-40; " /><meta name="citation_reference" content="citation_title=A predictive biophysical model of translational coupling to coordinate and control protein expression in bacterial operons;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=7137-51; " /><meta name="citation_reference" content="citation_title=The mechanism of eukaryotic translation initiation and principles of its regulation;  citation_journal_title=Nat. Rev. Mol. Cell Biol;  citation_year=2010;  citation_volume=11;  citation_pages=113-27; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/2/195/25073567/dsx049.pdf" /><meta name="description" content="Abstract.  Translation initiation in prokaryotes is affected by the mRNA folding and interaction of the ribosome binding site with the ribosomal RNA. The elonga" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/2/195/4638432" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/2/195/4638432" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/2/195/4638432");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsx049';
    </script>
<title>
	Genome scale analysis of Escherichia coli with a comprehensive prokaryotic sequence-based biophysical model of translation initiation and elongation | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4638432" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZOMATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hhaGFtLCBHaWxhZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbWVkaWNhbCBFbmdpbmVlcmluZywgVGhlIEVuZ2luZWVyaW5nIEZhY3VsdHksIFRlbCBBdml2IFVuaXZlcnNpdHksIElzcmFlbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlR1bGxlciwgVGFtaXIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb21lZGljYWwgRW5naW5lZXJpbmcsIFRoZSBFbmdpbmVlcmluZyBGYWN1bHR5LCBUZWwgQXZpdiBVbml2ZXJzaXR5LCBJc3JhZWwiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJUaGUgU2Fnb2wgU2Nob29sIG9mIE5ldXJvc2NpZW5jZSwgVGVsLUF2aXYgVW5pdmVyc2l0eSwgVGVsLUF2aXYsIElzcmFlbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iR2Vub21lIHNjYWxlIGFuYWx5c2lzIG9mIEVzY2hlcmljaGlhIGNvbGkgd2l0aCBhIGNvbXByZWhlbnNpdmUgcHJva2FyeW90aWMgc2VxdWVuY2UtYmFzZWQgYmlvcGh5c2ljYWwgbW9kZWwgb2YgdHJhbnNsYXRpb24gaW5pdGlhdGlvbiBhbmQgZWxvbmdhdGlvbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjE5NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMjA1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN4MDQ5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTgvMDQvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJhdGVnaWVzIGZvciBvcHRpbWl6aW5nIGhldGVyb2xvZ291cyBwcm90ZWluIGV4cHJlc3Npb24gaW4gRXNjaGVyaWNoaWEgY29saTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz01NC02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5pdGlhdGlvbiBvZiB0cmFuc2xhdGlvbiBpbiBwcm9rYXJ5b3RlcyBhbmQgZXVrYXJ5b3RlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZTsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzQ7ICBjaXRhdGlvbl9wYWdlcz0xODctMjA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlcm1pbmFudCBvZiBjaXN0cm9uIHNwZWNpZmljaXR5IGluIGJhY3RlcmlhbCByaWJvc29tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MTk3NTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTQ7ICBjaXRhdGlvbl9wYWdlcz0zNC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db3JyZWxhdGlvbiBiZXR3ZWVuIHRoZSBhYnVuZGFuY2Ugb2YgRXNjaGVyaWNoaWEgY29saSB0cmFuc2ZlciBSTkFzIGFuZCB0aGUgb2NjdXJyZW5jZSBvZiB0aGUgcmVzcGVjdGl2ZSBjb2RvbnMgaW4gaXRzIHByb3RlaW4gZ2VuZXM6IGEgcHJvcG9zYWwgZm9yIGEgc3lub255bW91cyBjb2RvbiBjaG9pY2UgdGhhdCBpcyBvcHRpbWFsIGZvciB0aGUgRS4gY29saSB0cmFuc2xhdGlvbmFsIHN5c3RlbTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTgxOyAgY2l0YXRpb25fdm9sdW1lPTE1MTsgIGNpdGF0aW9uX3BhZ2VzPTM4OS00MDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIHVzYWdlIGluIGJhY3RlcmlhOiBjb3JyZWxhdGlvbiB3aXRoIGdlbmUgZXhwcmVzc2l2aXR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MTk4MjsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTcwNTUtNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBlZmZlY3Qgb2YgdFJOQSBsZXZlbHMgb24gZGVjb2RpbmcgdGltZXMgb2YgbVJOQSBjb2RvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9OTE3MS04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXVsdGlwbGUgcm9sZXMgb2YgdGhlIGNvZGluZyBzZXF1ZW5jZSA14oCZIGVuZCBpbiBnZW5lIGV4cHJlc3Npb24gcmVndWxhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz0xMy0yODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gYmlhcyBhbmQgaGV0ZXJvbG9nb3VzIHByb3RlaW4gZXhwcmVzc2lvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0zNDYtNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEg4oCcU2lsZW504oCdIFBvbHltb3JwaGlzbSBpbiB0aGUgTURSMSBnZW5lIGNoYW5nZXMgc3Vic3RyYXRlIHNwZWNpZmljaXR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTMxNTsgIGNpdGF0aW9uX3BhZ2VzPTUyNS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WaXJhbCBhZGFwdGF0aW9uIHRvIGhvc3Q6IGEgcHJvdGVvbWUtYmFzZWQgYW5hbHlzaXMgb2YgY29kb24gdXNhZ2UgYW5kIGFtaW5vIGFjaWQgcHJlZmVyZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gU3lzdC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MzExIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZmVyZW50aWFsIGFyZ2lueWxhdGlvbiBvZiBhY3RpbiBpc29mb3JtcyBpcyByZWd1bGF0ZWQgYnkgY29kaW5nIHNlcXVlbmNlLWRlcGVuZGVudCBkZWdyYWRhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zMjk7ICBjaXRhdGlvbl9wYWdlcz0xNTM0LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhvdyB0aGUgc2VxdWVuY2Ugb2YgYSBnZW5lIGNhbiB0dW5lIGl0cyB0cmFuc2xhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDE7ICBjaXRhdGlvbl9wYWdlcz0yMjctOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlZGljdGl2ZSBiaW9waHlzaWNhbCBtb2RlbGluZyBhbmQgdW5kZXJzdGFuZGluZyBvZiB0aGUgZHluYW1pY3Mgb2YgbVJOQSB0cmFuc2xhdGlvbiBhbmQgaXRzIGV2b2x1dGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz05MDMxLTQ5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtc2NhbGUgYW5hbHlzaXMgb2YgdHJhbnNsYXRpb24gZWxvbmdhdGlvbiB3aXRoIGEgcmlib3NvbWUgZmxvdyBtb2RlbCB3YXNzZXJtYW4sOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIENvbXB1dC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDIxMjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgY29kb24gQWRhcHRhdGlvbiBJbmRleOKAlGEgbWVhc3VyZSBvZiBkaXJlY3Rpb25hbCBzeW5vbnltb3VzIGNvZG9uIHVzYWdlIGJpYXMsIGFuZCBpdHMgcG90ZW50aWFsIGFwcGxpY2F0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbC4gQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9MTI4MS05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG90YWxseSBhc3ltbWV0cmljIGV4Y2x1c2lvbiBwcm9jZXNzIHdpdGggZXh0ZW5kZWQgb2JqZWN0czogYSBtb2RlbCBmb3IgcHJvdGVpbiBzeW50aGVzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBoeXMuIFJldi4gRTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT02ODsgIGNpdGF0aW9uX3BhZ2VzPTAyMTkxMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNvbHZpbmcgdGhlIHJpZGRsZSBvZiBjb2RvbiB1c2FnZSBwcmVmZXJlbmNlczogYSB0ZXN0IGZvciB0cmFuc2xhdGlvbmFsIHNlbGVjdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz01MDM2LTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWFuIG9mIHRoZSB0eXBpY2FsIGRlY29kaW5nIHJhdGVzOiBhIG5ldyB0cmFuc2xhdGlvbiBlZmZpY2llbmN5IGluZGV4IGJhc2VkIG9uIHRoZSBhbmFseXNpcyBvZiByaWJvc29tZSBwcm9maWxpbmcgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RzMgQmV0aGVzZGEgTWQ7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTczLTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BdXRvbWF0ZWQgZGVzaWduIG9mIHN5bnRoZXRpYyByaWJvc29tZSBiaW5kaW5nIHNpdGVzIHRvIGNvbnRyb2wgcHJvdGVpbiBleHByZXNzaW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz05NDYtNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgdW5pdmVyc2FsIHRyZW5kIG9mIHJlZHVjZWQgbVJOQSBzdGFiaWxpdHkgbmVhciB0aGUgdHJhbnNsYXRpb24taW5pdGlhdGlvbiBzaXRlIGluIHByb2thcnlvdGVzIGFuZCBldWthcnlvdGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIENvbXB1dC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDA2NjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9RWxzZXZpZXIsIDsgIGNpdGF0aW9uX3RpdGxlPVRoZSBSaWJvc29tZSBCaW5kaW5nIFNpdGUgQ2FsY3VsYXRvck1ldGhvZHMgaW4gRW56eW1vbG9neTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3BhZ2VzPTE5LTQyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXcgdW5pdmVyc2FsIHJ1bGVzIG9mIGV1a2FyeW90aWMgdHJhbnNsYXRpb24gaW5pdGlhdGlvbiBmaWRlbGl0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBDb21wdXQuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAzMTM2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2Ugc2lnbmF0dXJlcyBhbmQgbVJOQSBjb25jZW50cmF0aW9uIGNhbiBleHBsYWluIHR3b+KAkHRoaXJkcyBvZiBwcm90ZWluIGFidW5kYW5jZSB2YXJpYXRpb24gaW4gYSBodW1hbiBjZWxsIGxpbmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gU3lzdC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9NDAwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgYW5kIHByZWRpY3Rpb24gb2YgdHJhbnNsYXRpb24gcmF0ZSBiYXNlZCBvbiBzZXF1ZW5jZSBhbmQgZnVuY3Rpb25hbCBmZWF0dXJlcyBvZiB0aGUgbVJOQSBLdWRsYSw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1lMTYwMzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0IGZlYXR1cmVzIGFsb25lIGVuYWJsZSBhY2N1cmF0ZSBwcmVkaWN0aW9uIGFuZCB1bmRlcnN0YW5kaW5nIG9mIGdlbmUgZXhwcmVzc2lvbiBpbiBTLiBjZXJldmlzaWFlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz1TMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZpZW5uYVJOQSBQYWNrYWdlIDIuMDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWxnb3JpdGhtcyBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2VsbCBzaXplIGFuZCBudWNsZW9pZCBvcmdhbml6YXRpb24gb2YgZW5naW5lZXJlZCBFc2NoZXJpY2hpYSBjb2xpIGNlbGxzIHdpdGggYSByZWR1Y2VkIGdlbm9tZTogZW5naW5lZXJlZCBFLiBjb2xpIHdpdGggYSByZWR1Y2VkIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTU7ICBjaXRhdGlvbl9wYWdlcz0xMzctNDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnN0cnVjdGlvbiBvZiBjb25zZWN1dGl2ZSBkZWxldGlvbnMgb2YgdGhlIEVzY2hlcmljaGlhIGNvbGkgY2hyb21vc29tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBTeXN0LiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0xMzIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIENvbXByZWhlbnNpdmUsIHF1YW50aXRhdGl2ZSwgYW5kIGdlbm9tZS13aWRlIG1vZGVsIG9mIHRyYW5zbGF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIENvbXB1dC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDA4NjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2ltdWxhdGlvbiAtcHJvdGVpbiBiaW9zeW50aGVzaXMgd2ViIHNlcnZpY2UsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTczOTQzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvcGVydGllcyBhbmQgZGV0ZXJtaW5hbnRzIG9mIGNvZG9uIGRlY29kaW5nIHRpbWUgZGlzdHJpYnV0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9UzEzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWluaW1hbGx5IGludmFzaXZlIGRldGVybWluYXRpb24gb2YgbVJOQSBjb25jZW50cmF0aW9uIGluIHNpbmdsZSBsaXZpbmcgYmFjdGVyaWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9ZTczIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVndWxvbkRCIHZlcnNpb24gOS4wOiBoaWdoLWxldmVsIGludGVncmF0aW9uIG9mIGdlbmUgcmVndWxhdGlvbiwgY29leHByZXNzaW9uLCBtb3RpZiBjbHVzdGVyaW5nIGFuZCBiZXlvbmQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9RDEzMy00MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGF4RGIsIGEgZGF0YWJhc2Ugb2YgcHJvdGVpbiBhYnVuZGFuY2UgYXZlcmFnZXMgYWNyb3NzIGFsbCB0aHJlZSBkb21haW5zIG9mIGxpZmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQ2VsbC4gUHJvdGVvbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTQ5Mi01MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVF1YW50aWZ5aW5nIEUuIGNvbGkgcHJvdGVvbWUgYW5kIHRyYW5zY3JpcHRvbWUgd2l0aCBzaW5nbGUtbW9sZWN1bGUgc2Vuc2l0aXZpdHkgaW4gc2luZ2xlIGNlbGxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTMyOTsgIGNpdGF0aW9uX3BhZ2VzPTUzMy04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgYW50aS1TaGluZeKAk0RhbGdhcm5vIHNlcXVlbmNlIGRyaXZlcyB0cmFuc2xhdGlvbmFsIHBhdXNpbmcgYW5kIGNvZG9uIGNob2ljZSBpbiBiYWN0ZXJpYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ4NDsgIGNpdGF0aW9uX3BhZ2VzPTUzOC00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q3V0YWRhcHQgcmVtb3ZlcyBhZGFwdGVyIHNlcXVlbmNlcyBmcm9tIGhpZ2gtdGhyb3VnaHB1dCBzZXF1ZW5jaW5nIHJlYWRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FbWJuZXQuIEo7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0xMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVsdHJhZmFzdCBhbmQgbWVtb3J5LWVmZmljaWVudCBhbGlnbm1lbnQgb2Ygc2hvcnQgRE5BIHNlcXVlbmNlcyB0byB0aGUgaHVtYW4gZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPVIyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5c3RlbWF0aWMgYW5kIGludGVncmF0aXZlIGFuYWx5c2lzIG9mIGxhcmdlIGdlbmUgbGlzdHMgdXNpbmcgREFWSUQgYmlvaW5mb3JtYXRpY3MgcmVzb3VyY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFByb3RvYzsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9NDQtNTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpb2luZm9ybWF0aWNzIGVucmljaG1lbnQgdG9vbHM6IHBhdGhzIHRvd2FyZCB0aGUgY29tcHJlaGVuc2l2ZSBmdW5jdGlvbmFsIGFuYWx5c2lzIG9mIGxhcmdlIGdlbmUgbGlzdHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9MS0xMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIHF1YW50aWZpY2F0aW9uIG9mIG1hbW1hbGlhbiBnZW5lIGV4cHJlc3Npb24gY29udHJvbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQ3MzsgIGNpdGF0aW9uX3BhZ2VzPTMzNy00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNsYXRpb24gZWxvbmdhdGlvbiBjYW4gY29udHJvbCB0cmFuc2xhdGlvbiBpbml0aWF0aW9uIG9uIGV1a2FyeW90aWMgbVJOQXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVtYm8gSjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTIxLTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2xhdGlvbiBpbml0aWF0aW9uIGluIEVzY2hlcmljaGlhIGNvbGk6IG9sZCBhbmQgbmV3IHF1ZXN0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTEwNjMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGVmZmljaWVuY3kgb2YgZm9sZGluZyBvZiBzb21lIHByb3RlaW5zIGlzIGluY3JlYXNlZCBieSBjb250cm9sbGVkIHJhdGVzIG9mIHRyYW5zbGF0aW9uIGluIHZpdm8uIEEgaHlwb3RoZXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTE5MzsgIGNpdGF0aW9uX3BhZ2VzPTQxMy03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm90ZWluIHN0cnVjdHVyZSBhbmQgdGhlIHNlcXVlbnRpYWwgc3RydWN0dXJlIG9mIG1STkE6IGFscGhhLWhlbGl4IGFuZCBiZXRhLXNoZWV0IHNpZ25hbHMgYXQgdGhlIG51Y2xlb3RpZGUgbGV2ZWw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb3RlaW5zOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MjM3LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SaWJvc29tZS1tZWRpYXRlZCB0cmFuc2xhdGlvbmFsIHBhdXNlIGFuZCBwcm90ZWluIGRvbWFpbiBvcmdhbml6YXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb3RlaW4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTU5NC02MTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb3RlaW4gc2Vjb25kYXJ5IHN0cnVjdHVyYWwgdHlwZXMgYXJlIGRpZmZlcmVudGlhbGx5IGNvZGVkIG9uIG1lc3NlbmdlciBSTkE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb3RlaW4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTk3My04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm9uLXJhbmRvbSB1c2FnZSBvZiDigJxkZWdlbmVyYXRl4oCdIGNvZG9ucyBpcyByZWxhdGVkIHRvIHByb3RlaW4gdGhyZWUtZGltZW5zaW9uYWwgc3RydWN0dXJlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRUJTIExldHQ7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9Mzk5OyAgY2l0YXRpb25fcGFnZXM9NzgtODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbW9kZWwgZm9yIGNvbXBldGl0aW9uIGZvciByaWJvc29tZXMgaW4gdGhlIGNlbGw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIFIgU29jLiBJbnRlcmZhY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0yMDE1MTA2MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zbGF0aW9uIGluaXRpYXRpb24gaW4gRXNjaGVyaWNoaWEgY29saTogc2VxdWVuY2VzIHdpdGhpbiB0aGUgcmlib3NvbWUtYmluZGluZyBzaXRlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MTIxOS0yOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZXJtaW5hdGlvbiBvZiB0aGUgb3B0aW1hbCBhbGlnbmVkIHNwYWNpbmcgYmV0d2VlbiB0aGUgU2hpbmUtRGFsZ2Fybm8gc2VxdWVuY2UgYW5kIHRoZSB0cmFuc2xhdGlvbiBpbml0aWF0aW9uIGNvZG9uIG9mIEVzY2hlcmljaGlhIGNvbGkgbVJOQXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2wuIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTQ5NTMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29ycmVsYXRpb25zIGJldHdlZW4gU2hpbmUtRGFsZ2Fybm8gc2VxdWVuY2VzIGFuZCBnZW5lIGZlYXR1cmVzIHN1Y2ggYXMgcHJlZGljdGVkIGV4cHJlc3Npb24gbGV2ZWxzIGFuZCBvcGVyb24gc3RydWN0dXJlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE4NDsgIGNpdGF0aW9uX3BhZ2VzPTU3MzMtNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwZWNpYWxpemVkIHJpYm9zb21lIHN5c3RlbTogcHJlZmVyZW50aWFsIHRyYW5zbGF0aW9uIG9mIGEgc2luZ2xlIG1STkEgc3BlY2llcyBieSBhIHN1YnBvcHVsYXRpb24gb2YgbXV0YXRlZCByaWJvc29tZXMgaW4gRXNjaGVyaWNoaWEgY29saTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLlMuQTsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT04NDsgIGNpdGF0aW9uX3BhZ2VzPTQ3NjItNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNsYXRpb25hbCBpbml0aWF0aW9uIG9uIHN0cnVjdHVyZWQgbWVzc2VuZ2VyczogYW5vdGhlciByb2xlIGZvciB0aGUgc2hpbmUtZGFsZ2Fybm8gaW50ZXJhY3Rpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0yMzU7ICBjaXRhdGlvbl9wYWdlcz0xNzMtODQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFuYWx5c2lzIGluIHZpdm8gb2YgdHJhbnNsYXRpb24gd2l0aCBudWNsZW90aWRlIHJlc29sdXRpb24gdXNpbmcgcmlib3NvbWUgcHJvZmlsaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTMyNDsgIGNpdGF0aW9uX3BhZ2VzPTIxOC0yMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gZXZvbHV0aW9uYXJpbHkgY29uc2VydmVkIG1lY2hhbmlzbSBmb3IgY29udHJvbGxpbmcgdGhlIGVmZmljaWVuY3kgb2YgcHJvdGVpbiB0cmFuc2xhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDE7ICBjaXRhdGlvbl9wYWdlcz0zNDQtNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvLXRyYW5zbGF0aW9uYWwgZm9sZGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gU3RydWN0LiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0xMTEtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzY3JldGUgbmFzY2VudCBjaGFpbiBsZW5ndGhzIGFyZSByZXF1aXJlZCBmb3IgdGhlIGluc2VydGlvbiBvZiBwcmVzZWNyZXRvcnkgcHJvdGVpbnMgaW50byBtaWNyb3NvbWFsIG1lbWJyYW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQ2VsbCBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTEyMTsgIGNpdGF0aW9uX3BhZ2VzPTEyMTEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUdhcmxhbmQgU2NpZW5jZSwgTmV3IFlvcmssIE5ZOyAgY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIEJpb2xvZ3kgb2YgdGhlIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zbGF0aW9uIGFycmVzdCBvZiBTZWNNIGlzIGVzc2VudGlhbCBmb3IgdGhlIGJhc2FsIGFuZCByZWd1bGF0ZWQgZXhwcmVzc2lvbiBvZiBTZWNBOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDE7ICBjaXRhdGlvbl9wYWdlcz0xMjMzMC01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbmRpdmlkdWFsIGNvbXBhcmlzb25zIGJ5IHJhbmtpbmcgbWV0aG9kczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvbS4gQnVsbDsgIGNpdGF0aW9uX3llYXI9MTk0NTsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9ODAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWFsLXRpbWUgdFJOQSB0cmFuc2l0IG9uIHNpbmdsZSB0cmFuc2xhdGluZyByaWJvc29tZXMgYXQgY29kb24gcmVzb2x1dGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ2NDsgIGNpdGF0aW9uX3BhZ2VzPTEwMTItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9zdCBhc3NvY2lhdGlvbnMgYmV0d2VlbiB0cmFuc2NyaXB0IGZlYXR1cmVzIGFuZCBnZW5lIGV4cHJlc3Npb24gYXJlIG1vbm90b25pYzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9zeXN0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MTQyNi00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwcmVkaWN0aXZlIGJpb3BoeXNpY2FsIG1vZGVsIG9mIHRyYW5zbGF0aW9uYWwgY291cGxpbmcgdG8gY29vcmRpbmF0ZSBhbmQgY29udHJvbCBwcm90ZWluIGV4cHJlc3Npb24gaW4gYmFjdGVyaWFsIG9wZXJvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9NzEzNy01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIG1lY2hhbmlzbSBvZiBldWthcnlvdGljIHRyYW5zbGF0aW9uIGluaXRpYXRpb24gYW5kIHByaW5jaXBsZXMgb2YgaXRzIHJlZ3VsYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBNb2wuIENlbGwgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTExMy0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI1LzIvMTk1LzI1MDczNTY3L2RzeDA0OS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVHJhbnNsYXRpb24gaW5pdGlhdGlvbiBpbiBwcm9rYXJ5b3RlcyBpcyBhZmZlY3RlZCBieSB0aGUgbVJOQSBmb2xkaW5nIGFuZCBpbnRlcmFjdGlvbiBvZiB0aGUgcmlib3NvbWUgYmluZGluZyBzaXRlIHdpdGggdGhlIHJpYm9zb21hbCBSTkEuIFRoZSBlbG9uZ2EiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ2Mzg0MzImeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkWDfy2jfhU9BVT6Xh3dNAa4KsK6Q=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f195%2f4638432" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f195%2f4638432" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=108668" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/2/1/m_dnares_25_2cover.png?Expires=2147483647&amp;Signature=yfSnSDuEb8Ru1iEiczh9QiVIRujFlv5eQUGA2PixxhIP2ifTZWTC6Dh4YwZbNdfMcc6NjI5nmXhBBnXl~MO-9TlTyrmfE~hJJOwCiRzc0uAKzzN1WszCYLPzg4YPUGBBBvmjvYjtWeuxzT6HMUtRCVQ1Lykx1-FxL9MPbgabfK3LfM0lfiO9VFwha2rRByA4174pANV1EEHkTLYoj9VZAqG9NluBapcroduGQ77QCNh6n6sU6cwRapmFfSHO~HcCIdSas62HDXNUy3sCegaV5-lnkgDK34Vs8bE~hWbIr-4dZ69GanJYFX-rhZ4a7s6UpCy38dZKnOnrMjKCPyIBeg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/2">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="118067887">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118067887">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118067889">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118067889">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118067894">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118067894">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118067897">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118067897">3. Translation initiation efficiency calculator</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118067901">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118067901">4. Normalizing initiation rate</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118067903">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118067903">5. Modelling translation elongation</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118067906">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118067906">6. TASEP elongation rate</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118067909">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118067909">7. TASEP traffic jams</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118067911">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118067911">8. Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118067930">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118067930">9. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118067976">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118067976">10. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118067984">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118067984">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118067986">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118067986">Conflict of interest</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118067988">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118067988">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118067990">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118067990">Footnotes</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="118067992">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#118067992">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/2/183/4665695">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/2/207/4675083">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome scale analysis of <em>Escherichia coli</em> with a comprehensive prokaryotic sequence-based biophysical model of translation initiation and elongation
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Gilad Shaham</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biomedical Engineering, The Engineering Faculty, Tel Aviv University, Israel</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Gilad+Shaham">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shaham G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shaham G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Gilad Shaham</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tamir Tuller</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Biomedical Engineering, The Engineering Faculty, Tel Aviv University, Israel</div><div class="aff">The Sagol School of Neuroscience, Tel-Aviv University, Tel-Aviv, Israel</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx049-cor1" data-open="dsx049-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +972 3 6405839. Fax. +972 3 6407308. Email: <a href="mailto:tamirtul@post.tau.ac.il" target="_blank">tamirtul@post.tau.ac.il</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tamir+Tuller">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tuller T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tuller T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tamir Tuller</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 2, 1 April 2018, Pages 195–205, <a href='https://doi.org/10.1093/dnares/dsx049'>https://doi.org/10.1093/dnares/dsx049</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">17 November 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">15 March 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">04 November 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4638432" href="/dnaresearch/article-pdf/25/2/195/25073567/dsx049.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Gilad Shaham, Tamir Tuller;  Genome scale analysis of <em>Escherichia coli</em> with a comprehensive prokaryotic sequence-based biophysical model of translation initiation and elongation, <em>DNA Research</em>, Volume 25, Issue 2, 1 April 2018, Pages 195–205, <a href='https://doi.org/10.1093/dnares/dsx049'>https://doi.org/10.1093/dnares/dsx049</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4638432?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4638432?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4638432?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4638432?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4638432?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4638432?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4638432?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome%20scale%20analysis%20of%20%26lt%3Bem%26gt%3BEscherichia%20coli%26lt%3B%2Fem%26gt%3B%20with%20a%20comprehensive%20prokaryotic%20sequence-based%20biophysical%20model%20of%20translation%20initiation%20and%20elongation&amp;publicationDate=2017-11-17&amp;volumeNum=25&amp;issueNum=2&amp;author=Shaham%2C%20Gilad%3B%20Tuller%2C%20Tamir&amp;startPage=195&amp;endPage=205&amp;contentId=10.1093%2Fdnares%2Fdsx049&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f195%2f4638432" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=118067887 id="118067887" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Translation initiation in prokaryotes is affected by the mRNA folding and interaction of the ribosome binding site with the ribosomal RNA. The elongation rate is affected, among other factors, by the local biophysical properties of the coding regions, the decoding rates of different codons, and the interactions among ribosomes. Currently, there is no comprehensive biophysical model of translation that enables the prediction of mRNA translation dynamics based only on the transcript sequence and while considering all of these fundamental aspects of translation. In this study, we provide, for the first time, a computational simulative biophysical model of both translation initiation and elongation with all aspects mentioned above. We demonstrate our model performance and advantages focusing on <em>Escherichia coli</em> genes. We further show that the model enables prediction of translation rate, protein levels, and ribosome densities. In addition, our model enables quantifying the effect of silent mutations on translation rate in different parts of the transcript, the relative effect of mutations on translation initiation and elongation, and the effect of mutations on ribosome traffic jams. Thus, unlike previous models, the proposed one provides comprehensive information, facilitating future research in disciplines such as molecular evolution, synthetic biology, and functional genomics. A toolkit to estimate translation dynamics of transcripts is available at: <a class="link link-uri" href="https://www.cs.tau.ac.il/&amp;sim;tamirtul/transim" target="_blank">https://www.cs.tau.ac.il/∼tamirtul/transim</a></p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="translation">translation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;biophysical model&quot;">biophysical model</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcript evolution&quot;">transcript evolution</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="UTR">UTR</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;mRNA folding&quot;">mRNA folding</a></div>                            <h2 scrollto-destination=118067889 id="118067889" class="section-title" >1. Introduction</h2>
<p>All living cells share the ability to translate mRNA transcripts into proteins. The codes related to this process partially appear in different regions of the transcript including the UTRs and the coding region. For example, it was shown that the mRNA folding at the 5′UTR and the 5′end of the coding region regulates translation initiation efficiency.<a reveal-id="dsx049-B1" data-open="dsx049-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsx049-B2" data-open="dsx049-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> In prokaryotes, translation efficiency is related to the hybridization of the ribosomal RNA and the ribosome biding sites upstream of the start codon.<a reveal-id="dsx049-B3" data-open="dsx049-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> It was also shown that different codons have different decoding rate; thus, codon usage biase regulate translation elongation.<a reveal-id="dsx049-B4 dsx049-B5 dsx049-B6" data-open="dsx049-B4 dsx049-B5 dsx049-B6" class="link link-ref link-reveal xref-bibr"><sup>4–6</sup></a> In addition, it was shown that there is high concentration of functional silent codes related to various gene expression steps at the 5′end of the coding regions.<a reveal-id="dsx049-B7" data-open="dsx049-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a></p><p>Thus, having the ability to harness the nucleotide composition of the transcript to the biophysical dynamics of mRNA translation should have fundamental contribution to disciplines such as molecular evolution, functional genomics, synthetic biology, biotechnology, and human health.<a reveal-id="dsx049-B8 dsx049-B9 dsx049-B10 dsx049-B11 dsx049-B12 dsx049-B13" data-open="dsx049-B8 dsx049-B9 dsx049-B10 dsx049-B11 dsx049-B12 dsx049-B13" class="link link-ref link-reveal xref-bibr"><sup>8–13</sup></a></p><p>Yet, similar to other cellular processes, despite decades of study and various technological advancements, it is still very challenging to predict the dynamics of mRNA translation based only on the transcript nucleotide composition. Specifically, today there are biophysical predictive models that consider mostly translation elongation<a reveal-id="dsx049-B14 dsx049-B15 dsx049-B16 dsx049-B17 dsx049-B18" data-open="dsx049-B14 dsx049-B15 dsx049-B16 dsx049-B17 dsx049-B18" class="link link-ref link-reveal xref-bibr"><sup>14–18</sup></a> or translation initiation.<a reveal-id="dsx049-B19 dsx049-B20 dsx049-B21 dsx049-B22" data-open="dsx049-B19 dsx049-B20 dsx049-B21 dsx049-B22" class="link link-ref link-reveal xref-bibr"><sup>19–22</sup></a> In addition, there are machine learning based models that statistically integrate various features of the 5′UTR and the coding region to predict gene expression measurements such as protein levels.<a reveal-id="dsx049-B23 dsx049-B24 dsx049-B25" data-open="dsx049-B23 dsx049-B24 dsx049-B25" class="link link-ref link-reveal xref-bibr"><sup>23–25</sup></a> However, there is no complete user-friendly model/toolkit, which incorporates both translation initiation and elongation dynamics, based only on the transcript nucleotide composition.</p><p>In this study, we make an additional step towards developing a computational biophysical model that predicts the dynamics of translation based solely on the transcript nucleotide composition. As we show, the model is specifically important as it enables, for the first time, to study the relation between transcript mutations and the biophysics/dynamics of translation. Translation is a fundamental intracellular process directly related to organismal fitness; thus, we provide an approach that should promote improved and novel understanding of transcript evolution at a nucleotide resolution. We specifically demonstrate how our new model can provide important, and previously unpredicted, information.</p>                            <h2 scrollto-destination=118067894 id="118067894" class="section-title" >2. Materials and methods</h2>
<p>The research scheme is shown in <a reveal-id="dsx049-F1" data-open="dsx049-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>: Using the mRNA sequence, the mRNA folding and mRNA-rRNA interactions are calculated, resulting in the relevant Gibbs free energy predicted values (A). These free energy terms are described in the translation initiation efficiency calculator subsection and are then used to predict the initiation rates (B); which is the rate by which ribosomes approach the start codon. The codon translation elongation rates (C) are calculated using ribosome profiling data; each codon has its own elongation rate and when we later refer to the elongation rate of a gene, we consider the elongation rates of all its codons and the dynamic of the translation process (see Section 5). Rather than using a mean field approximation of the model totally asymmetric exclusion process (TASEP)<a reveal-id="dsx049-B16" data-open="dsx049-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> such as the ribosome flow model,<a reveal-id="dsx049-B14" data-open="dsx049-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> we decided to directly use the stochastic ‘high resolution’ TASEP model to simulate translation using the estimated initiation rates and elongation rates (D). TASEP provides the predictions of translation rate (which is the overall translation rate when considering both the initiation and the elongation steps), ribosomal density, termination counts and occurrences of ribosome jamming (E). Finally, we study various novel questions related to the relations between nucleotide composition of the transcript and translation aspects (F). </p>                        <a id="118067896" scrollto-destination="118067896"></a>
<div id="dsx049-F1" data-id="dsx049-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/m_dsx049f1.png?Expires=2147483647&Signature=Vz~8B90MlBcANoytEPuXWxVQiJH0RGcOQiHKlOqdDYVCJHauQp04BzZsCHvExdZcF3N5CoKwENyuPpLqeKEelHwB1uIXYDjJg9BDVl7yN-s5AReRq1mYVFcG49jztGHenDtax0rcUKEOfxPG40-IDLpeM1jRsVTTW2zF4SoZwsXHlHdIS~CDriLkocl1aG8mBDamRjJmuYnMsSP4pyD0Ks0l9D1O7ZvPiQtvmrFwElQuIdHopcIuhiTCv3RdUkyjEyLn-MnG0uDCHR2m3sLUoTNOuj3XIDgd38YnkokYJUjFOFGKfo4yRPPpFN1Emq9OEYfJfxehb0QGOtLtwJ4aFA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Based on the 5’UTR and the beginning of the ORF, the Gibbs free energy is calculated. (B) Summing together the free energies results with the expected initiation rate. (C) Using codon translation times and the mRNA ORF, the per-codon elongation rate is calculated. (D) TASEP calculation is performed using the initiation rates and elongation rates from the previous steps. (E) TASEP results with translation rate, ribosomal density, termination count and ribosome jamming. (F) We study the relation between transcript nucleotide composition and translation based on the simulation results. (G) Illustration, as described by the RBS calculator where ΔGmRNA is the folding energy. The location of ΔGstandby is depicted in purple, ΔGmRNA:rRNA is in green, ΔGspacing is in blue and ΔGstart is in red." path-from-xml="dsx049f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118067896/dsx049f1.tif" path-from-xml="dsx049f1.tif" target="_blank">View large</a><a section="118067896" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049f1.png?Expires=2147483647&Signature=063utCpG~mMzZkSmftMBp06fyM~2fC5yE9nw03fwjbZAYUcbcYGzyQdZlQjW5nINLv5KyTFPsDux-UiL~l2ccOZB4xo1WxPB0qNJb~FfS-seHEM~hxXYVcLcjcWLdG-mvlOjRL2LZoFiJcMgcmbdIPCJNY5n0yDiYPtAWm646X557RRAl7dKY38zn9TgXMmDTPfFlRiEdBNE-IkuHWzCwQVhQc0hqOPmNcykm0MXGYxel9LzXR6gJoTXbQx0P0~b6gj6B5g4mapdptt7lw8Y6CegBvveasfTxiyuCok7Ns9WRfN2E-FZVNOzPXhTf67~2vTy9T~ydapo2IPF4LqWtA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118067896&ar=4638432&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx049f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Based on the 5’UTR and the beginning of the ORF, the Gibbs free energy is calculated. (B) Summing together the free energies results with the expected initiation rate. (C) Using codon translation times and the mRNA ORF, the per-codon elongation rate is calculated. (D) TASEP calculation is performed using the initiation rates and elongation rates from the previous steps. (E) TASEP results with translation rate, ribosomal density, termination count and ribosome jamming. (F) We study the relation between transcript nucleotide composition and translation based on the simulation results. (G) Illustration, as described by the RBS calculator where <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">Δ</mi><msub xmlns=""><mrow><mi mathvariant="normal">G</mi></mrow><mrow><mi mathvariant="normal">m</mi><mi mathvariant="normal">R</mi><mi mathvariant="normal">N</mi><mi mathvariant="normal">A</mi></mrow></msub></math></span> is the folding energy. The location of <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">Δ</mi><msub xmlns=""><mrow><mi>G</mi></mrow><mrow><mi mathvariant="normal">s</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">d</mi><mi mathvariant="normal">b</mi><mi mathvariant="normal">y</mi></mrow></msub></math></span> is depicted in purple, <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">Δ</mi><msub xmlns=""><mrow><mi>G</mi></mrow><mrow><mi mathvariant="normal">m</mi><mi mathvariant="normal">R</mi><mi mathvariant="normal">N</mi><mi mathvariant="normal">A</mi><mo>:</mo><mi mathvariant="normal">r</mi><mi mathvariant="normal">R</mi><mi mathvariant="normal">N</mi><mi mathvariant="normal">A</mi></mrow></msub></math></span> is in green, <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">Δ</mi><msub xmlns=""><mrow><mi>G</mi></mrow><mrow><mi>s</mi><mi>p</mi><mi>a</mi><mi>c</mi><mi>i</mi><mi>n</mi><mi>g</mi></mrow></msub></math></span> is in blue and <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">Δ</mi><msub xmlns=""><mrow><mi>G</mi></mrow><mrow><mi>s</mi><mi>t</mi><mi>a</mi><mi>r</mi><mi>t</mi></mrow></msub></math></span> is in red.</p></div></div><div content-id="dsx049-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/m_dsx049f1.png?Expires=2147483647&Signature=Vz~8B90MlBcANoytEPuXWxVQiJH0RGcOQiHKlOqdDYVCJHauQp04BzZsCHvExdZcF3N5CoKwENyuPpLqeKEelHwB1uIXYDjJg9BDVl7yN-s5AReRq1mYVFcG49jztGHenDtax0rcUKEOfxPG40-IDLpeM1jRsVTTW2zF4SoZwsXHlHdIS~CDriLkocl1aG8mBDamRjJmuYnMsSP4pyD0Ks0l9D1O7ZvPiQtvmrFwElQuIdHopcIuhiTCv3RdUkyjEyLn-MnG0uDCHR2m3sLUoTNOuj3XIDgd38YnkokYJUjFOFGKfo4yRPPpFN1Emq9OEYfJfxehb0QGOtLtwJ4aFA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Based on the 5’UTR and the beginning of the ORF, the Gibbs free energy is calculated. (B) Summing together the free energies results with the expected initiation rate. (C) Using codon translation times and the mRNA ORF, the per-codon elongation rate is calculated. (D) TASEP calculation is performed using the initiation rates and elongation rates from the previous steps. (E) TASEP results with translation rate, ribosomal density, termination count and ribosome jamming. (F) We study the relation between transcript nucleotide composition and translation based on the simulation results. (G) Illustration, as described by the RBS calculator where ΔGmRNA is the folding energy. The location of ΔGstandby is depicted in purple, ΔGmRNA:rRNA is in green, ΔGspacing is in blue and ΔGstart is in red." path-from-xml="dsx049f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118067896/dsx049f1.tif" path-from-xml="dsx049f1.tif" target="_blank">View large</a><a section="118067896" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049f1.png?Expires=2147483647&Signature=063utCpG~mMzZkSmftMBp06fyM~2fC5yE9nw03fwjbZAYUcbcYGzyQdZlQjW5nINLv5KyTFPsDux-UiL~l2ccOZB4xo1WxPB0qNJb~FfS-seHEM~hxXYVcLcjcWLdG-mvlOjRL2LZoFiJcMgcmbdIPCJNY5n0yDiYPtAWm646X557RRAl7dKY38zn9TgXMmDTPfFlRiEdBNE-IkuHWzCwQVhQc0hqOPmNcykm0MXGYxel9LzXR6gJoTXbQx0P0~b6gj6B5g4mapdptt7lw8Y6CegBvveasfTxiyuCok7Ns9WRfN2E-FZVNOzPXhTf67~2vTy9T~ydapo2IPF4LqWtA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118067896&ar=4638432&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx049f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Based on the 5’UTR and the beginning of the ORF, the Gibbs free energy is calculated. (B) Summing together the free energies results with the expected initiation rate. (C) Using codon translation times and the mRNA ORF, the per-codon elongation rate is calculated. (D) TASEP calculation is performed using the initiation rates and elongation rates from the previous steps. (E) TASEP results with translation rate, ribosomal density, termination count and ribosome jamming. (F) We study the relation between transcript nucleotide composition and translation based on the simulation results. (G) Illustration, as described by the RBS calculator where <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">Δ</mi><msub xmlns=""><mrow><mi mathvariant="normal">G</mi></mrow><mrow><mi mathvariant="normal">m</mi><mi mathvariant="normal">R</mi><mi mathvariant="normal">N</mi><mi mathvariant="normal">A</mi></mrow></msub></math></span> is the folding energy. The location of <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">Δ</mi><msub xmlns=""><mrow><mi>G</mi></mrow><mrow><mi mathvariant="normal">s</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">d</mi><mi mathvariant="normal">b</mi><mi mathvariant="normal">y</mi></mrow></msub></math></span> is depicted in purple, <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">Δ</mi><msub xmlns=""><mrow><mi>G</mi></mrow><mrow><mi mathvariant="normal">m</mi><mi mathvariant="normal">R</mi><mi mathvariant="normal">N</mi><mi mathvariant="normal">A</mi><mo>:</mo><mi mathvariant="normal">r</mi><mi mathvariant="normal">R</mi><mi mathvariant="normal">N</mi><mi mathvariant="normal">A</mi></mrow></msub></math></span> is in green, <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">Δ</mi><msub xmlns=""><mrow><mi>G</mi></mrow><mrow><mi>s</mi><mi>p</mi><mi>a</mi><mi>c</mi><mi>i</mi><mi>n</mi><mi>g</mi></mrow></msub></math></span> is in blue and <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">Δ</mi><msub xmlns=""><mrow><mi>G</mi></mrow><mrow><mi>s</mi><mi>t</mi><mi>a</mi><mi>r</mi><mi>t</mi></mrow></msub></math></span> is in red.</p></div></div>                            <h2 scrollto-destination=118067897 id="118067897" class="section-title" >3. Translation initiation efficiency calculator</h2>
<div class="block-child-p">The translation initiation calculator is based on the ribosome binding site (RBS) mRNA folding calculator.<a reveal-id="dsx049-B19" data-open="dsx049-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><sup>,</sup><a reveal-id="dsx049-B21" data-open="dsx049-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> In general, to compute the nominal initiation rate, we implemented a thermodynamic model of bacterial initiation, which predicts the Gibbs free energy of ribosome binding. The input to the initiation rate calculator is the mRNA sequence and its output is the initiation rate, predicted on a proportional scale span over 8 orders of magnitude (0.001 to 100, 000+ au). The thermodynamic model calculates the difference in Gibbs free energy before and after the 30 S complex assembles onto an mRNA transcript. Five free energy terms are calculated and summed together: <a id="E1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">Δ</mi><msub xmlns=""><mrow><mi>G</mi></mrow><mrow><mi>t</mi><mi>o</mi><mi>t</mi></mrow></msub><mo xmlns="">=</mo><mi mathvariant="normal" xmlns="">Δ</mi><msub xmlns=""><mrow><mi>G</mi></mrow><mrow><mi mathvariant="normal">m</mi><mi mathvariant="normal">R</mi><mi mathvariant="normal">N</mi><mi mathvariant="normal">A</mi><mo>:</mo><mi mathvariant="normal">r</mi><mi mathvariant="normal">R</mi><mi mathvariant="normal">N</mi><mi mathvariant="normal">A</mi></mrow></msub><mo xmlns="">+</mo><mi mathvariant="normal" xmlns="">Δ</mi><msub xmlns=""><mrow><mi>G</mi></mrow><mrow><mi mathvariant="normal">s</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">t</mi></mrow></msub><mo xmlns="">+</mo><mi mathvariant="normal" xmlns="">Δ</mi><msub xmlns=""><mrow><mi>G</mi></mrow><mrow><mi mathvariant="normal">s</mi><mi mathvariant="normal">p</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">g</mi></mrow></msub><mo xmlns="">-</mo><mi mathvariant="normal" xmlns="">Δ</mi><msub xmlns=""><mrow><mi>G</mi></mrow><mrow><mi mathvariant="normal">s</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">d</mi><mi mathvariant="normal">b</mi><mi mathvariant="normal">y</mi></mrow></msub><mo xmlns="">-</mo><mi mathvariant="normal" xmlns="">Δ</mi><msub xmlns=""><mrow><mi>G</mi></mrow><mrow><mi mathvariant="normal">m</mi><mi mathvariant="normal">R</mi><mi mathvariant="normal">N</mi><mi mathvariant="normal">A</mi><mo>,</mo></mrow></msub></math></div></div></div></div><p>where <ul class="bullet"><li><p><span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">Δ</mi><msub xmlns=""><mrow><mi>G</mi></mrow><mrow><mi mathvariant="normal">m</mi><mi mathvariant="normal">R</mi><mi mathvariant="normal">N</mi><mi mathvariant="normal">A</mi></mrow></msub></math></span>—The folding energy of the mRNA subsequence prior to binding with the 30S complex.</p></li><li><p><span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">Δ</mi><msub xmlns=""><mrow><mi>G</mi></mrow><mrow><mi mathvariant="normal">m</mi><mi mathvariant="normal">R</mi><mi mathvariant="normal">N</mi><mi mathvariant="normal">A</mi><mo>:</mo><mi mathvariant="normal">r</mi><mi mathvariant="normal">R</mi><mi mathvariant="normal">N</mi><mi mathvariant="normal">A</mi></mrow></msub></math></span>— The energy released when the last nine nucleotides of the 16S rRNA cofolds and hybridizes with the mRNA sub-sequence at the 16S rRNA-binding site.</p></li><li><p><span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">Δ</mi><msub xmlns=""><mrow><mi mathvariant="normal">G</mi></mrow><mrow><mi mathvariant="normal">s</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">r</mi><mi mathvariant="normal">t</mi></mrow></msub></math></span>—The energy released when the tRNAfMet’s anticodon hybridizes to the start codon.</p></li><li><p><span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">Δ</mi><msub xmlns=""><mrow><mi>G</mi></mrow><mrow><mi mathvariant="normal">s</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">d</mi><mi mathvariant="normal">b</mi><mi mathvariant="normal">y</mi></mrow></msub></math></span>— The energy released when the standby site is folded.</p></li><li><p><span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">Δ</mi><msub xmlns=""><mrow><mi>G</mi></mrow><mrow><mi mathvariant="normal">s</mi><mi mathvariant="normal">p</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">c</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">g</mi></mrow></msub></math></span>—An energetic penalty for a non-optimal distance between the 16S rRNA-binding site and the start codon.</p></li></ul></p><p>These free energy terms are illustrated in <a reveal-id="dsx049-F1" data-open="dsx049-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>. See the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049_suppl_data.zip?Expires=2147483647&Signature=K-XxDsS-lclmJqaifgOD8cqmR8wuXpMiBAd~PLg8rmKVSkcu4VLkYLSfj9Ec7G1iFsozccEB12Zm6NMd-blJmaU~mfKzJJIBhOfD~BkeqoHWZXPqsiCbqz~nIceinOSshRIKx-MORqxUDdLkVEC7VwMwrc1aaigUPWIFkNQS9S6dHEqUGECnWpTEzDEW9SfkjjpvyxMMBBAumGj1p9snNuotWhPaqULXWTeTH-IBLbB5UILXZKBQ71hQjREmQZ~Ci03~4uCB46laWmlEEx7nL7bu3MvL72lMnXPZycil78Z8MWW3pDhIL-5DTBvrgZJtwi1yoknXPR4rmRVVwv4etQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Text</a></span> for more details regarding these energy terms. We used the Vienna RNA package<a reveal-id="dsx049-B26" data-open="dsx049-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> version 2.2.7 to perform the necessary folding and resultant free energy calculations.</p>                            <h2 scrollto-destination=118067901 id="118067901" class="section-title" >4. Normalizing initiation rate</h2>
<div class="block-child-p">The initiation rate of each transcript is calculated using the following exponential formula: <a id="E2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E2"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">K</mi><mrow xmlns=""><mrow><mi mathvariant="normal">exp</mi></mrow><mo>⁡</mo><mrow><mfenced separators="|"><mrow><mo>-</mo><mi>β</mi><mi>Δ</mi><msub><mrow><mi>G</mi></mrow><mrow><mi mathvariant="normal">t</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">t</mi></mrow></msub></mrow></mfenced><mo>,</mo></mrow></mrow></math></div></div></div> where <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns="">β</mi><mo xmlns="">=</mo><mn xmlns="">0.45</mn><mi xmlns=""> </mi><mi mathvariant="normal" xmlns="">m</mi><mi mathvariant="normal" xmlns="">o</mi><mi mathvariant="normal" xmlns="">l</mi><mo xmlns="">/</mo><mi mathvariant="normal" xmlns="">k</mi><mi mathvariant="normal" xmlns="">c</mi><mi mathvariant="normal" xmlns="">a</mi><mi mathvariant="normal" xmlns="">l</mi></math></span> and <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">K</mi><mo xmlns="">=</mo><mn xmlns="">2500</mn></math>⁠</span>. This initiation rate is predicted on a proportional scale from 0.001 to 100, 000+ au; we then normalize it to initiation time measured in seconds. We took the list of 302 essential genes from the <em>Escherichia coli</em> chromosome (PEC) database<a reveal-id="dsx049-B27" data-open="dsx049-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a><sup>,</sup><a reveal-id="dsx049-B28" data-open="dsx049-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> and we calculated the average initiation rate of 709.773 ([au] units which are proportional to rate) for 138 genes in our list that are classified as essential genes. We then took the mean initiation time of 178 out of the 302 essential genes available at the Transimulation Protein Biosynthesis Server,<a reveal-id="dsx049-B29" data-open="dsx049-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a><sup>,</sup><a reveal-id="dsx049-B30" data-open="dsx049-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> with an average 12.179 s. Hence, we multiply the two values to obtain a factor of approximately <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mn xmlns="">8</mn><mo xmlns="">,</mo><mi mathvariant="normal" xmlns=""> </mi><mn xmlns="">644</mn><mi mathvariant="normal" xmlns=""> </mi><mo xmlns="">[</mo><mi mathvariant="normal" xmlns="">a</mi><mi mathvariant="normal" xmlns="">u</mi><mo xmlns="">⋅</mo><mi mathvariant="normal" xmlns="">s</mi><mo xmlns="">]</mo></math></span> and divide each individual transcript initiation rate by this factor in order to obtain the initiation rate in <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mfenced open="[" close="]" separators="|" xmlns=""><mrow><msup><mrow><mi mathvariant="normal">s</mi></mrow><mrow><mo>-</mo><mn>1</mn></mrow></msup></mrow></mfenced></math></span> units.</div>                            <h2 scrollto-destination=118067903 id="118067903" class="section-title" >5. Modelling translation elongation</h2>
<p>The elongation modeling was based on a TASEP.<a reveal-id="dsx049-B14" data-open="dsx049-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsx049-B16" data-open="dsx049-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> The TASEP is a stochastic flow model of translation elongation, whose output is the predicted translation rate, the ribosomal density and the number of termination events. An mRNA transcript with <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">N</mi></math></span> codons is modeled as a chain of sites, each of which is labeled by the index <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">i</mi></math>⁠</span>, where <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">i</mi><mo xmlns="">=</mo><mn xmlns="">1</mn><mo xmlns="">…</mo><mi xmlns="">N</mi></math>⁠</span>. The first and last codons <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">i</mi><mo xmlns="">=</mo><mn xmlns="">1</mn></math></span> and <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">i</mi><mo xmlns="">=</mo><mi xmlns="">N</mi></math>⁠</span>, are associated with the start and stop codons, respectively. At any time <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">t</mi></math>⁠</span>, <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">M</mi><mo xmlns="">(</mo><mi xmlns="">t</mi><mo xmlns="">)</mo></math></span> ribosomes are bound to the mRNA.</p><p>The initiation time as well as the time a ribosome spends translating each codon are exponentially distributed with a codon dependent rate. In addition, ribosomes span over <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">l</mi></math></span> codons and if two ribosomes are adjacent, the trailing one is delayed until the ribosome in front of it has proceeded onwards. Unless otherwise specified, all the reported results in this paper use <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">l</mi><mo xmlns="">=</mo><mn xmlns="">11</mn></math>⁠</span>. A free ribosome will attach to codon <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">i</mi><mo xmlns="">=</mo><mn xmlns="">1</mn></math></span> with mean rate <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">λ</mi></math>⁠</span>, if the first <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo xmlns="">(</mo><mi xmlns="">l</mi><mo xmlns="">+</mo><mn xmlns="">1</mn><mo xmlns="">)</mo><mo xmlns="">/</mo><mn xmlns="">2</mn></math></span> codons on the mRNA are empty. An attached ribosome located at codon <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">i</mi></math></span> will move to the next codon <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">i</mi><mo xmlns="">+</mo><mn xmlns="">1</mn></math></span> with mean rate <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>λ</mi></mrow><mrow><mi>i</mi></mrow></msub></math>⁠</span>, provided codon <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">i</mi><mo xmlns="">+</mo><mo xmlns="">(</mo><mi xmlns="">l</mi><mo xmlns="">+</mo><mn xmlns="">1</mn><mo xmlns="">)</mo><mo xmlns="">/</mo><mn xmlns="">2</mn></math></span> is not covered by another ribosome. In case <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">i</mi><mo xmlns="">+</mo><mo xmlns="">(</mo><mi xmlns="">l</mi><mo xmlns="">+</mo><mn xmlns="">1</mn><mo xmlns="">)</mo><mo xmlns="">/</mo><mn xmlns="">2</mn><mo xmlns="">&gt;</mo><mi xmlns="">N</mi></math></span> (ribosome is bulging out of the mRNA strand) an attached ribosome will move to the next codon with mean rate <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>λ</mi></mrow><mrow><mi>i</mi></mrow></msub></math>⁠</span>. In the case of <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">i</mi><mo xmlns="">=</mo><mi xmlns="">N</mi></math></span> the jump attempt is in fact a termination step.</p>                            <h2 scrollto-destination=118067906 id="118067906" class="section-title" >6. TASEP elongation rate</h2>
<p>We obtained the per-codon <em>E. coli</em> typical decoding time (TDR).<a reveal-id="dsx049-B31" data-open="dsx049-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> We calculated the mean TDR of the codons by multiplying each codon TDR by the empirical probability of the codon to appear in the entire sequence data (sequence data source is described in <em>E. coli</em> Sequence Data subsection below). We then normalized the mean TDR to be 8 codons per second.<a reveal-id="dsx049-B32" data-open="dsx049-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a></p><p>We estimate the elongation rate for each transcript by calculating TASEP with a constant initiation rate. We chose the mean normalized initiation rate (0.2884) to be this constant.</p>                            <h2 scrollto-destination=118067909 id="118067909" class="section-title" >7. TASEP traffic jams</h2>
<p>A ribosome can only move if the ribosome ahead of it does not cover the next codon. During the simulation, we choose to either attempt initiation or, based on the existing ribosomes, perform translation and move a ribosome to the next position. In cases where such movement cannot occur, we consider this event as a traffic jam. If the simulation attempts initiation, but cannot, since the first <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo xmlns="">(</mo><mi xmlns="">l</mi><mo xmlns="">+</mo><mn xmlns="">1</mn><mo xmlns="">)</mo><mo xmlns="">/</mo><mn xmlns="">2</mn></math></span> codons are not empty, we count this as ‘initiation jam occurrence’ and we measure the total time difference between the failed initiation attempt and the next time an initiation is successfully performed as ‘initiation jam time’. If the simulation attempts to move a ribosome to site <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">i</mi></math>⁠</span>, but cannot—since the next <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo xmlns="">(</mo><mi xmlns="">l</mi><mo xmlns="">+</mo><mn xmlns="">1</mn><mo xmlns="">)</mo><mo xmlns="">/</mo><mn xmlns="">2</mn></math></span> codons are not empty, we count this as ‘jam occurrence’. We then ensure that a ribosome would move at the same time the subsequent ribosome clears the next <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mo xmlns="">(</mo><mi xmlns="">l</mi><mo xmlns="">+</mo><mn xmlns="">1</mn><mo xmlns="">)</mo><mo xmlns="">/</mo><mn xmlns="">2</mn></math></span> codon sites, i.e. when it is possible for ribosome to proceed. We also measure the mean time the delayed ribosome had to wait as ‘mean jam time’.</p>                            <h2 scrollto-destination=118067911 id="118067911" class="section-title" >8. Data</h2>
                            <h3 scrollto-destination=118067912 id="118067912" class="section-title" >8.1. <em>Escherichia coli</em> sequence data</h3>
<p>We took <em>E. coli</em> str. K-12 substr. MG1655 mRNA sequences, including 5′UTR and ORF annotations from RegulonDB version 9.1 (<a class="link link-uri" href="http://regulondb.ccg.unam.mx" target="_blank">http://regulondb.ccg.unam.mx</a>).<a reveal-id="dsx049-B33" data-open="dsx049-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> The two files, UTR_5_3_sequence.txt and Gene_sequence.txt, were downloaded directly from the Downloadable Experimental Datasets web page on 31 July 2016.</p><p>For our analysis of the initiation and translation rates, we chose either operons with a single gene or the first gene in operons that contain multiple genes. Overall, we found 1,500 genes with 2,035 unique 5′UTR and ORF sequences. For some of these transcripts, we could not calculate initiation rate (either the start codon is not ATG, CTG, GTG, or TTG or we did not find a suitable ribosome-binding site); after eliminating those, we were left with 1,904 transcripts. Finally, the initiation calculation sometimes reaches non-feasible initiation time (e.g. millions seconds); thus, to remove extreme outliers we discarded the transcripts with top 50 initiation times from our analysis. Out of the transcripts left, we took only those whose initiation time was lower or equal to 5 times standard deviation away from the initiation time median. Thus, we were left with 1,835 transcripts. Additional details regarding the transcripts discarded from the analysis are available in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049_suppl_data.zip?Expires=2147483647&Signature=K-XxDsS-lclmJqaifgOD8cqmR8wuXpMiBAd~PLg8rmKVSkcu4VLkYLSfj9Ec7G1iFsozccEB12Zm6NMd-blJmaU~mfKzJJIBhOfD~BkeqoHWZXPqsiCbqz~nIceinOSshRIKx-MORqxUDdLkVEC7VwMwrc1aaigUPWIFkNQS9S6dHEqUGECnWpTEzDEW9SfkjjpvyxMMBBAumGj1p9snNuotWhPaqULXWTeTH-IBLbB5UILXZKBQ71hQjREmQZ~Ci03~4uCB46laWmlEEx7nL7bu3MvL72lMnXPZycil78Z8MWW3pDhIL-5DTBvrgZJtwi1yoknXPR4rmRVVwv4etQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">supplementary material</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049_suppl_data.zip?Expires=2147483647&Signature=K-XxDsS-lclmJqaifgOD8cqmR8wuXpMiBAd~PLg8rmKVSkcu4VLkYLSfj9Ec7G1iFsozccEB12Zm6NMd-blJmaU~mfKzJJIBhOfD~BkeqoHWZXPqsiCbqz~nIceinOSshRIKx-MORqxUDdLkVEC7VwMwrc1aaigUPWIFkNQS9S6dHEqUGECnWpTEzDEW9SfkjjpvyxMMBBAumGj1p9snNuotWhPaqULXWTeTH-IBLbB5UILXZKBQ71hQjREmQZ~Ci03~4uCB46laWmlEEx7nL7bu3MvL72lMnXPZycil78Z8MWW3pDhIL-5DTBvrgZJtwi1yoknXPR4rmRVVwv4etQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>.</p><p>Unless otherwise specified, the results in this paper are calculated per transcript. When comparing the model results with per-gene expression levels, we used the mean results across the different transcripts for the same gene, prior to calculating the correlation.</p>                            <h3 scrollto-destination=118067916 id="118067916" class="section-title" >8.2. <em>Escherichia coli</em> protein abundance</h3>
<p>The <em>E. coli</em> str. K-12 substr. MG1655 protein abundance (PA) integrated dataset, a weighted average of all <em>E. coli</em> datasets, published in 2015, was downloaded from PaxDB,<a reveal-id="dsx049-B34" data-open="dsx049-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> accessed on 7 May 2016. We excluded any gene with zero reads. Note that similar conclusions were obtained when we analyzed different databases separately (refer to <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049_suppl_data.zip?Expires=2147483647&Signature=K-XxDsS-lclmJqaifgOD8cqmR8wuXpMiBAd~PLg8rmKVSkcu4VLkYLSfj9Ec7G1iFsozccEB12Zm6NMd-blJmaU~mfKzJJIBhOfD~BkeqoHWZXPqsiCbqz~nIceinOSshRIKx-MORqxUDdLkVEC7VwMwrc1aaigUPWIFkNQS9S6dHEqUGECnWpTEzDEW9SfkjjpvyxMMBBAumGj1p9snNuotWhPaqULXWTeTH-IBLbB5UILXZKBQ71hQjREmQZ~Ci03~4uCB46laWmlEEx7nL7bu3MvL72lMnXPZycil78Z8MWW3pDhIL-5DTBvrgZJtwi1yoknXPR4rmRVVwv4etQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">supplementary material</a></span> for more details).</p>                            <h3 scrollto-destination=118067918 id="118067918" class="section-title" >8.3. <em>Escherichia coli</em> mean mRNA levels</h3>
<p>The mean mRNA levels were obtained from Taniguchi <em>et al.</em><a reveal-id="dsx049-B35" data-open="dsx049-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> Data include mean mRNA levels and mRNA levels per cell cycle.</p>                            <h3 scrollto-destination=118067920 id="118067920" class="section-title" >8.4. <em>Escherichia coli</em> protein per mRNA levels</h3>
<p>We used the protein abundance, described above and divided it by the corresponding mean mRNA levels, described in the previous subsection to calculate the protein abundance per mean mRNA.</p>                            <h3 scrollto-destination=118067922 id="118067922" class="section-title" >8.5. <em>Escherichia coli</em> ribo-seq</h3>
<p>Two ribosomal profiling datasets were obtained from the Gene Expression Omnibus: GSE35641, <em>E. coli</em> MG1655 rep1-2.<a reveal-id="dsx049-B36" data-open="dsx049-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Transcript sequences were obtained from EnsEMBL for <em>E. coli</em> (K-12 MG1655 release 121, accessed 28 July 2015). We trimmed 3′ adaptors from the reads using Cutadapt<a reveal-id="dsx049-B37" data-open="dsx049-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> version 1.8.3, and utilized Bowtie<a reveal-id="dsx049-B38" data-open="dsx049-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> version 1.1.1 to map them to the <em>E. coli</em> transcriptome. In the first phase, we discarded reads that mapped to rRNA and tRNA sequences with Bowtie parameters ‘-n 2-seedlen 21-k 1–norc’. In the second phase, we mapped the remaining reads to the transcriptome with Bowtie parameters ‘-v 2 -a –strata –best –norc -m 200’. We filtered out reads longer than 34 nt and shorter than 26 nt. Unique alignments were first assigned to the ribosome occupancy profiles. For reads that are mapped to more than one position, the best alignments in terms of number of mismatches were kept. Then, multiple aligned reads were distributed between locations according to the distribution of unique ribosomal reads in the respective surrounding regions. To this end, for each position <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">i</mi></math>⁠</span>, a 100 nt window was used to compute the read count density <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mtext>RCD</mtext></mrow><mrow><mi>i</mi></mrow></msub></math></span> (total read counts in the window divided by length, based on unique reads) in vicinity of the <em>M</em> multiple aligned positions in the transcriptome, and the fraction of a read assigned to each position was <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mrow><msub><mrow><mtext>RCD</mtext></mrow><mrow><mi>i</mi></mrow></msub></mrow><mo>/</mo><mrow><mrow><msubsup><mo stretchy="false">∑</mo><mrow><mi>j</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>M</mi></mrow></msubsup><mrow><msub><mrow><mtext>RCD</mtext></mrow><mrow><mi>j</mi></mrow></msub></mrow></mrow></mrow></mrow></math>⁠</span>.</p>                            <h3 scrollto-destination=118067924 id="118067924" class="section-title" >8.6. Functional annotation</h3>
<p>We performed functional annotation of specific groups of genes using DAVID bioinformatics resources version 6.7 (<a class="link link-uri" href="https://david.ncifcrf.gov" target="_blank">https://david.ncifcrf.gov</a>/).<a reveal-id="dsx049-B39" data-open="dsx049-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a><sup>,</sup><a reveal-id="dsx049-B40" data-open="dsx049-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> We set the background list to be the genes in the selected dataset, as described in the <em>E. coli</em> Sequence Data subsection.</p>                            <h3 scrollto-destination=118067926 id="118067926" class="section-title" >8.7. Simulating single-nucleotide mutations</h3>
<p>For each transcript in our selected dataset, we simulate a single nucleotide mutation in the range of − 35 to − 1 and +3 to +34 (where 0 is the first nucleotide in the ORF). We excluded the start codon change, as most mutations would result with an invalid start codon. Furthermore, mutations that introduced premature STOP codons were not allowed.</p>                            <h3 scrollto-destination=118067928 id="118067928" class="section-title" >8.8. Correction for multiple comparisons</h3>
<p>The enrichment analysis <em>P</em>-values were for multiple comparisons using Bonferroni correction. All the correlation <em>P</em>-values pass Benjamini and Hochberg correction for false discovery rate.</p>                            <h2 scrollto-destination=118067930 id="118067930" class="section-title" >9. Results</h2>
                            <h3 scrollto-destination=118067931 id="118067931" class="section-title" >9.1. A toolkit to estimate translation dynamics of transcripts</h3>
<p>We created a software tool that enables the estimation of the initiation rate and translation rate, as well as the sequence features affecting those rates, such as folding energies, ribosome binding efficiencies, and elongation rate. The tool is executed via command line interface, where one should provide either a single mRNA sequence or a list of different mRNA sequences and execution parameters via an input file.</p><p>Special attention was given to optimize the code, thus allowing almost immediate results for a single mRNA calculation and performing large-scale calculations within reasonable computational time.</p><p>We named the toolkit Translation Simulator, or Transim, and it is available to download for academic use at the authors’ website—<a class="link link-uri" href="http://www.cs.tau.ac.il/&amp;sim;tamirtul/transim" target="_blank">http://www.cs.tau.ac.il/∼tamirtul/transim</a>.</p>                            <h3 scrollto-destination=118067935 id="118067935" class="section-title" >9.2. Higher correlation of predicted translation rate with experimental measurements than the correlation of predicted initiation rate with experimental measurements in endogenous genes</h3>
<p>As a first step, we aimed at checking how well our biophysical model, and different components of the model, can predict measured (i.e. ‘true’) gene expression levels. Today, there are no <em>direct</em> large-scale measurements of translation (measurements of protein levels or mRNA levels, e.g. are related to the transcription step, and the mRNA degradation step; protein levels are related also to the protein degradation step). Thus, we used the measure most correlated/related to translation rate that is available today, protein abundance (PA) divided by mRNA levels which is the estimation of the number of proteins generated per mRNA molecule.</p><p>We calculated Spearman correlation of protein abundance (PA) divided by mRNA levels (protein per mRNA) with the calculated initiation rates, elongation rates and translation rates. As can be seen in <a reveal-id="dsx049-F2" data-open="dsx049-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>, the correlation with the translation rate was higher and more significant than that with the initiation rate and elongation rate alone. In addition, the partial correlation of translation with PA divided by mRNA given the initiation or the elongation is significant: R(translation, PA/mRNA | initiation) = 0.265, (<em>P</em> = 4.23×10<sup>−7</sup>), R(translation, PA/mRNA | elongation) = 0.1932 (<em>P</em> = 2.55×10<sup>−4</sup>). These results support the usage of the complete model (which includes both the initiation and the elongation steps). </p>                        <a id="118067938" scrollto-destination="118067938"></a>
<div id="dsx049-F2" data-id="dsx049-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/m_dsx049f2.png?Expires=2147483647&Signature=xayv4rL3hfh4DJyan4K02RayH5SQehpmX271gYmxiOMgFCP8WNfdI7J4yJlaYHNDvQRkSwWh0bfqap-98bgkXo6DQXAlNecZqkke1fpf0T1tjss1FVzNoPqmv-i3p1KRHlOpEmfxOKlovCXB4tjJuLMdwarbE~WDGIPbfYnyIetIr4DZ4bwUMszDAJ2oZjzVzXlCwz6gGY81XhiYbcje~h3Fnl~C9vbPXrE4cA5n04AY2uqh~za1km-UapyFSVmJ4BTGa5e08yXjXVzr6nB0P3AKuOPFekHBCNlWSE5OuYci-h6ZY0jXZL88ICXO~LBgxg0e3P1vvWqsPt3MsEENNQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Spearman correlation of initiation rate, elongation and translation rate with protein abundance (PA) per mRNA levels. The values in parenthesis indicate the P-value. (B)–(E) Scatter plots for each of the cases. In the case of the complete translation model there are two ‘regions’ in the figure: one related to low initiation rate where the initiation rate is low and rate limiting and the density is low; the second region related to higher initiation rate where the elongation rate becomes more rate limiting and the density is higher." path-from-xml="dsx049f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118067938/dsx049f2.tif" path-from-xml="dsx049f2.tif" target="_blank">View large</a><a section="118067938" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049f2.png?Expires=2147483647&Signature=CZ8Z3j~Xs3oLhPjoAYHqxbetCZ7FMlRN0gLyGb3sCyrAAWJIx7fyci7mX0eebFJS3WI39JVALTGtUODTQ3ZWXQRUE0kGJWnZLBvWkhTFRr2LWmhKQhmuVpxS4fwKoZR8IXKWrirIVaB2tc4u9hoLHTSL~FrQpcnQgOJQNQGx15dIX3-S3opWLLKLYbTiTHE4P06kjTM8IHmpQUreS0461G-cNsq4IgzXb8i0Q76hCCUrCLFQuaDpegtmk7DXPFo2ZeiTd4jde0g4qv2z6Keq5m5pza1pFQxym-xIS1kuK7NzkEkc3fTqLv3ZwNFkVFC4r0n4BrBDbyUGqOePuPfGlg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118067938&ar=4638432&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx049f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Spearman correlation of initiation rate, elongation and translation rate with protein abundance (PA) per mRNA levels. The values in parenthesis indicate the <em>P</em>-value. (B)–(E) Scatter plots for each of the cases. In the case of the complete translation model there are two ‘regions’ in the figure: one related to low initiation rate where the initiation rate is low and rate limiting and the density is low; the second region related to higher initiation rate where the elongation rate becomes more rate limiting and the density is higher.</p></div></div><div content-id="dsx049-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/m_dsx049f2.png?Expires=2147483647&Signature=xayv4rL3hfh4DJyan4K02RayH5SQehpmX271gYmxiOMgFCP8WNfdI7J4yJlaYHNDvQRkSwWh0bfqap-98bgkXo6DQXAlNecZqkke1fpf0T1tjss1FVzNoPqmv-i3p1KRHlOpEmfxOKlovCXB4tjJuLMdwarbE~WDGIPbfYnyIetIr4DZ4bwUMszDAJ2oZjzVzXlCwz6gGY81XhiYbcje~h3Fnl~C9vbPXrE4cA5n04AY2uqh~za1km-UapyFSVmJ4BTGa5e08yXjXVzr6nB0P3AKuOPFekHBCNlWSE5OuYci-h6ZY0jXZL88ICXO~LBgxg0e3P1vvWqsPt3MsEENNQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Spearman correlation of initiation rate, elongation and translation rate with protein abundance (PA) per mRNA levels. The values in parenthesis indicate the P-value. (B)–(E) Scatter plots for each of the cases. In the case of the complete translation model there are two ‘regions’ in the figure: one related to low initiation rate where the initiation rate is low and rate limiting and the density is low; the second region related to higher initiation rate where the elongation rate becomes more rate limiting and the density is higher." path-from-xml="dsx049f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118067938/dsx049f2.tif" path-from-xml="dsx049f2.tif" target="_blank">View large</a><a section="118067938" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049f2.png?Expires=2147483647&Signature=CZ8Z3j~Xs3oLhPjoAYHqxbetCZ7FMlRN0gLyGb3sCyrAAWJIx7fyci7mX0eebFJS3WI39JVALTGtUODTQ3ZWXQRUE0kGJWnZLBvWkhTFRr2LWmhKQhmuVpxS4fwKoZR8IXKWrirIVaB2tc4u9hoLHTSL~FrQpcnQgOJQNQGx15dIX3-S3opWLLKLYbTiTHE4P06kjTM8IHmpQUreS0461G-cNsq4IgzXb8i0Q76hCCUrCLFQuaDpegtmk7DXPFo2ZeiTd4jde0g4qv2z6Keq5m5pza1pFQxym-xIS1kuK7NzkEkc3fTqLv3ZwNFkVFC4r0n4BrBDbyUGqOePuPfGlg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118067938&ar=4638432&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx049f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Spearman correlation of initiation rate, elongation and translation rate with protein abundance (PA) per mRNA levels. The values in parenthesis indicate the <em>P</em>-value. (B)–(E) Scatter plots for each of the cases. In the case of the complete translation model there are two ‘regions’ in the figure: one related to low initiation rate where the initiation rate is low and rate limiting and the density is low; the second region related to higher initiation rate where the elongation rate becomes more rate limiting and the density is higher.</p></div></div><p>In the future, it will be interesting to further validate our model with direct experimental measurements of translation.<a reveal-id="dsx049-B41" data-open="dsx049-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a></p>                            <h3 scrollto-destination=118067940 id="118067940" class="section-title" >9.3. About the distributions of initiation rates, elongation rates, and translation rates</h3>
<p>It is worth noting that the distribution of the translation rate is different from the initiation rate and elongation rate, as seen in <a reveal-id="dsx049-F3" data-open="dsx049-F3" class="link link-reveal link-table xref-fig">Fig. 3</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049_suppl_data.zip?Expires=2147483647&Signature=K-XxDsS-lclmJqaifgOD8cqmR8wuXpMiBAd~PLg8rmKVSkcu4VLkYLSfj9Ec7G1iFsozccEB12Zm6NMd-blJmaU~mfKzJJIBhOfD~BkeqoHWZXPqsiCbqz~nIceinOSshRIKx-MORqxUDdLkVEC7VwMwrc1aaigUPWIFkNQS9S6dHEqUGECnWpTEzDEW9SfkjjpvyxMMBBAumGj1p9snNuotWhPaqULXWTeTH-IBLbB5UILXZKBQ71hQjREmQZ~Ci03~4uCB46laWmlEEx7nL7bu3MvL72lMnXPZycil78Z8MWW3pDhIL-5DTBvrgZJtwi1yoknXPR4rmRVVwv4etQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>. Specifically, all distributions are not normal (KStest <em>P</em>-value &lt; 10<sup>−10</sup> for initiation and translation and <em>P</em>-value is <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mn xmlns="">5.86</mn><mo xmlns="">×</mo><mo xmlns="">⋅</mo><msup xmlns=""><mrow><mn>10</mn></mrow><mrow><mo>-</mo><mn>6</mn></mrow></msup></math></span> for elongation) with positive skewness of 6 and 6.6 for initiation and translation, respectively due to right tails, and small negative skewness (−1.28) for elongation. The positive skewness of the initiation and translation may be related to traffic jams and/or extremely low initiations rate in some genes at the initiation step (extreme and unexpected delays in the initiation times contribute to very high and non-typical initiation times, resulting in the right tail).<a reveal-id="dsx049-B18" data-open="dsx049-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a></p>                        <a id="118067942" scrollto-destination="118067942"></a>
<div id="dsx049-F3" data-id="dsx049-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/m_dsx049f3.png?Expires=2147483647&Signature=fJwWa3L5fOt1QWuS0uuuZGHHbYKn-9vQzL4BvzHLimg8avd5AjEyfiGrEW7IGn8VeqXPTXreXOp1zaw20Q8QOTHFRzarIrZCsp75Ybo13QbLYGhgPCGgydD2SRN2Rv7AiXLHdpgIFTrZ7QZeVK4RlVrzDzAq3dnbOPS72NJs-7HXIHW-dQQ9c5gTjSlKnLOcP--nl1Sf5ZL1wMNdcppYiCGVKuLT25JMykOdXucpJV9I47bXtYGDZQMTYWx4sra5tk7eLuIefPZfak-sp6mBOFrchbzqu0w0jIcvylJ1-bzdJSZrrPVLowu~jCLpVFDof3Pi3wblKgQiqXHA0qUtqw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of various translation variables: initiation rates (A), elongation rates (B), ribosome densities (C), translation rates (D), translation rate to initiation rate ratio (E). Note that the bin sizes are not uniform, but grow exponentially." path-from-xml="dsx049f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118067942/dsx049f3.tif" path-from-xml="dsx049f3.tif" target="_blank">View large</a><a section="118067942" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049f3.png?Expires=2147483647&Signature=ScrQgjwm3OeNcxK-idSH0GYmje0I-fn8n4qTOVP93juIIZfYydAZEQ~eKaTN8Odq4EW~Ax08FVy99FaHmu-yfGl-MCyWxXFpI18SOrYOdausqtlVwqJq4v2a5N~67R1rEgUCeddYrcN6B3vVq3n2pELqoES0DvxqvPSfvFZJ8kIHY5Q8QBIa6msqas0IWmpR9AeQdUrQrQSfowOIBaL5AXcUrNMCffI1SXylAvdwDFJbZS48bRk0pU5LtBXnpOjlKOIN7AU9ObqAqKhOejlktjh7BkJBndzq6~iO~vVnWAibmMVG7EHqyRORn8OFgSZpiBf6gBgpg4ntLi9p2dLO8Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118067942&ar=4638432&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx049f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution of various translation variables: initiation rates (A), elongation rates (B), ribosome densities (C), translation rates (D), translation rate to initiation rate ratio (E). Note that the bin sizes are not uniform, but grow exponentially.</p></div></div><div content-id="dsx049-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/m_dsx049f3.png?Expires=2147483647&Signature=fJwWa3L5fOt1QWuS0uuuZGHHbYKn-9vQzL4BvzHLimg8avd5AjEyfiGrEW7IGn8VeqXPTXreXOp1zaw20Q8QOTHFRzarIrZCsp75Ybo13QbLYGhgPCGgydD2SRN2Rv7AiXLHdpgIFTrZ7QZeVK4RlVrzDzAq3dnbOPS72NJs-7HXIHW-dQQ9c5gTjSlKnLOcP--nl1Sf5ZL1wMNdcppYiCGVKuLT25JMykOdXucpJV9I47bXtYGDZQMTYWx4sra5tk7eLuIefPZfak-sp6mBOFrchbzqu0w0jIcvylJ1-bzdJSZrrPVLowu~jCLpVFDof3Pi3wblKgQiqXHA0qUtqw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The distribution of various translation variables: initiation rates (A), elongation rates (B), ribosome densities (C), translation rates (D), translation rate to initiation rate ratio (E). Note that the bin sizes are not uniform, but grow exponentially." path-from-xml="dsx049f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118067942/dsx049f3.tif" path-from-xml="dsx049f3.tif" target="_blank">View large</a><a section="118067942" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049f3.png?Expires=2147483647&Signature=ScrQgjwm3OeNcxK-idSH0GYmje0I-fn8n4qTOVP93juIIZfYydAZEQ~eKaTN8Odq4EW~Ax08FVy99FaHmu-yfGl-MCyWxXFpI18SOrYOdausqtlVwqJq4v2a5N~67R1rEgUCeddYrcN6B3vVq3n2pELqoES0DvxqvPSfvFZJ8kIHY5Q8QBIa6msqas0IWmpR9AeQdUrQrQSfowOIBaL5AXcUrNMCffI1SXylAvdwDFJbZS48bRk0pU5LtBXnpOjlKOIN7AU9ObqAqKhOejlktjh7BkJBndzq6~iO~vVnWAibmMVG7EHqyRORn8OFgSZpiBf6gBgpg4ntLi9p2dLO8Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118067942&ar=4638432&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx049f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The distribution of various translation variables: initiation rates (A), elongation rates (B), ribosome densities (C), translation rates (D), translation rate to initiation rate ratio (E). Note that the bin sizes are not uniform, but grow exponentially.</p></div></div><p>As can be seen (<a reveal-id="dsx049-F3" data-open="dsx049-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>), the mean initiation time per transcript is 54.4 s while the mean elongation time is 138.1 s and the mean translation time is 174.2 s. This is expected, as translation includes both initiation and elongation. While there are cases with very low initiation time (the minimum is 0.04 s), the minimum elongation and translation times are 96.3 and 98.6 s, respectively. In the cases where initiation rates are very fast, we would expect elongation to be the rate-limiting factor. We also see the maximum initiation time found is 1,629 s, while the maximum translation time is 1,739.7 s. The maximum elongation time is 155.3 s, but 80% of the initiation rates are below 52.2 s and 80% of the translation rates are below 155.7 s and we expect the initiation to be rate-limiting factor when it is slow.</p><p>The fact that the typical initiation rate is significantly lower than the typical translation rate demonstrates that, in practice, not only the initiation step affects translation rate but that also the elongation rate tcontributes to the final translation rate;<a reveal-id="dsx049-B7" data-open="dsx049-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsx049-B42" data-open="dsx049-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> this insight provides a better understanding to previous models in the field.<a reveal-id="dsx049-B43" data-open="dsx049-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a></p><p>While translation rate can be significantly affected by the initiation rate, fast initiation rates (i.e. low initiation time), can increase the translation rate, but up to a point. In these cases, the elongation rate becomes a limiting factor, effectively creating a maximum threshold for the translation rate. This limit in translation rate may be important for biological processes, such as prevention of depletion of the ribosome pool. Hence, genes that have high and efficient initiation rate may have codons that partially limit their overall translation rate and ribosome consumption.<a reveal-id="dsx049-B7" data-open="dsx049-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Limiting the translation elongation rate may also assist in allowing the resultant protein to cotranslationally fold correctly.<a reveal-id="dsx049-B44 dsx049-B45 dsx049-B46 dsx049-B47 dsx049-B48" data-open="dsx049-B44 dsx049-B45 dsx049-B46 dsx049-B47 dsx049-B48" class="link link-ref link-reveal xref-bibr"><sup>44–48</sup></a></p><p>The results reported here suggest that changes in the codon composition of a gene should have significant effect on translation, even if the initiation rate is optimal/maximal and this can be further validated experimentally via the design of relevant libraries of gene variants and the measurements of translation of the different variants.</p>                            <h3 scrollto-destination=118067947 id="118067947" class="section-title" >9.4. Only specific locations of single nucleotide mutation affect initiation and translation rates</h3>
<p>At the next step, we aimed at understanding the effect of mutations in different regions surrounding the start codon on translation rate in endogenous transcripts. This was done under the assumption that the expression levels of each transcript is not extremely high such that the mutations should not affect the translation rate of other transcripts.<a reveal-id="dsx049-B49" data-open="dsx049-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> Thus, for each transcript in our dataset and each nucleotide in the range of −35 to −1 and +3 to +34 (where 0 is the first nucleotide of the ORF), we changed the actual nucleotide to all three other possible nucleotides. We then calculated the new initiation rate and translation rate per each change. For each position, we calculated the mean result and compared it to the mean of the unmodified transcript.</p><p>Results are available in <a reveal-id="dsx049-F4" data-open="dsx049-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>. On average, there is a clear reduction in initiation rate when mutations occur at the RBS site (which is generally 5–13 nucleotides upstream of the start codon, with a typical distance of about 8–10 bases relative to the start codon),<a reveal-id="dsx049-B50 dsx049-B51 dsx049-B52" data-open="dsx049-B50 dsx049-B51 dsx049-B52" class="link link-ref link-reveal xref-bibr"><sup>50–52</sup></a> although the translation rate is less affected. Apart from the RBS site, changes in the first nucleotides of the ORF also tend to slightly decrease mean initiation rate. When we examine this in more detail (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049_suppl_data.zip?Expires=2147483647&Signature=K-XxDsS-lclmJqaifgOD8cqmR8wuXpMiBAd~PLg8rmKVSkcu4VLkYLSfj9Ec7G1iFsozccEB12Zm6NMd-blJmaU~mfKzJJIBhOfD~BkeqoHWZXPqsiCbqz~nIceinOSshRIKx-MORqxUDdLkVEC7VwMwrc1aaigUPWIFkNQS9S6dHEqUGECnWpTEzDEW9SfkjjpvyxMMBBAumGj1p9snNuotWhPaqULXWTeTH-IBLbB5UILXZKBQ71hQjREmQZ~Ci03~4uCB46laWmlEEx7nL7bu3MvL72lMnXPZycil78Z8MWW3pDhIL-5DTBvrgZJtwi1yoknXPR4rmRVVwv4etQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S1–S4</a></span>), while we still see the same effect in the RBS site, we also see that for translation, only a small group of transcripts had an increase in translation rate, and some had a decrease, but for most transcripts, no change occurred in translation rate. For initiation, we also see several groups, and a large portion remains unchanged. These results demonstrate that translation rate tend to be immune to single point mutations in the transcript. We also see slight increase in codon rate in that region due to the mutation, this occurs since codons are usually ‘less optimal’ (in terms of their translation rate) in the beginning of the ORF.<a reveal-id="dsx049-B7" data-open="dsx049-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a></p>                        <a id="118067950" scrollto-destination="118067950"></a>
<div id="dsx049-F4" data-id="dsx049-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/m_dsx049f4.png?Expires=2147483647&Signature=PohK2W~t6lOCvfswSTw3v9G-VHRYynOS~IV6C8V4DRb986sM0lh50fLXYLhl3As1HKLQyWJqh2Ya6Jb5PTgYpgJF-fkb1ObJfgKIRAAOEc1FzQV3UJ17-ScvZwPBHPjxAG-Vel9eJc3LN9p31-qQd0RmVpNsOs7x939EiftPIBzpEXJxy3CJvDltJGJglFmT7~bWlSvN~ijeCO3qXMOcx1qtnzo6pMpAEiO62qZ0rGDVUNuEEC3pxjt3r7epq-v7SehNidR7UVwNG11D0MKr6qitqUQess-QeCvqeU15ia5AAAA5vZi47o2~SLCAUO9zftjHvbSfz0eqDqH-b-~Smw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Impact of single single nucleotide mutation on the mean initiation rate, the mean change in codon decoding rate for the position where the mutation occurs, and the mean translation rate." path-from-xml="dsx049f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118067950/dsx049f4.tif" path-from-xml="dsx049f4.tif" target="_blank">View large</a><a section="118067950" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049f4.png?Expires=2147483647&Signature=hlp1-4pCAyMv9epz9JP4o-rMZmhcj6CSlVh7Yh3cAZwHmZ2BOWWendS-yUHa91SFfpTnbGVqZT2HYum6-fKALj~r8O7ujZJAZQluREEZhDiZz8B5lm564cIoELpNVexNwJMiVTsP46vU8HZGIe1U02frPB~~UCLn78GDZ78n9v7CgQ879PQ7M8OE-aQCjcbLCjCuHK-LuRid0WsvunpMhBz3pMWC9PGb7Rr5Sgxp1Cx4aAuiIzpOrSBcuTMDsgFbENrCq1EmMEeDnI~H463NgJU6eDHZ3S92SAhY0SovJ12Kg3-doi0ZoIeA3WzUFamBWEgt-dC9SWfNaxExdjBueQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118067950&ar=4638432&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx049f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Impact of single single nucleotide mutation on the mean initiation rate, the mean change in codon decoding rate for the position where the mutation occurs, and the mean translation rate.</p></div></div><div content-id="dsx049-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/m_dsx049f4.png?Expires=2147483647&Signature=PohK2W~t6lOCvfswSTw3v9G-VHRYynOS~IV6C8V4DRb986sM0lh50fLXYLhl3As1HKLQyWJqh2Ya6Jb5PTgYpgJF-fkb1ObJfgKIRAAOEc1FzQV3UJ17-ScvZwPBHPjxAG-Vel9eJc3LN9p31-qQd0RmVpNsOs7x939EiftPIBzpEXJxy3CJvDltJGJglFmT7~bWlSvN~ijeCO3qXMOcx1qtnzo6pMpAEiO62qZ0rGDVUNuEEC3pxjt3r7epq-v7SehNidR7UVwNG11D0MKr6qitqUQess-QeCvqeU15ia5AAAA5vZi47o2~SLCAUO9zftjHvbSfz0eqDqH-b-~Smw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Impact of single single nucleotide mutation on the mean initiation rate, the mean change in codon decoding rate for the position where the mutation occurs, and the mean translation rate." path-from-xml="dsx049f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118067950/dsx049f4.tif" path-from-xml="dsx049f4.tif" target="_blank">View large</a><a section="118067950" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049f4.png?Expires=2147483647&Signature=hlp1-4pCAyMv9epz9JP4o-rMZmhcj6CSlVh7Yh3cAZwHmZ2BOWWendS-yUHa91SFfpTnbGVqZT2HYum6-fKALj~r8O7ujZJAZQluREEZhDiZz8B5lm564cIoELpNVexNwJMiVTsP46vU8HZGIe1U02frPB~~UCLn78GDZ78n9v7CgQ879PQ7M8OE-aQCjcbLCjCuHK-LuRid0WsvunpMhBz3pMWC9PGb7Rr5Sgxp1Cx4aAuiIzpOrSBcuTMDsgFbENrCq1EmMEeDnI~H463NgJU6eDHZ3S92SAhY0SovJ12Kg3-doi0ZoIeA3WzUFamBWEgt-dC9SWfNaxExdjBueQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118067950&ar=4638432&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx049f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Impact of single single nucleotide mutation on the mean initiation rate, the mean change in codon decoding rate for the position where the mutation occurs, and the mean translation rate.</p></div></div><p>The fact that the most prominent effect is seen in the RBS site is in agreement with previous studies, which have demonstrated that mutations in the Shine-Delgarno regions have the strongest effect on translation rate.<a reveal-id="dsx049-B53" data-open="dsx049-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a><sup>,</sup><a reveal-id="dsx049-B54" data-open="dsx049-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> This could be further validated in large-scale experiments in the future.</p><p>We further examined the impact of mutation type on the initiation rate (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049_suppl_data.zip?Expires=2147483647&Signature=K-XxDsS-lclmJqaifgOD8cqmR8wuXpMiBAd~PLg8rmKVSkcu4VLkYLSfj9Ec7G1iFsozccEB12Zm6NMd-blJmaU~mfKzJJIBhOfD~BkeqoHWZXPqsiCbqz~nIceinOSshRIKx-MORqxUDdLkVEC7VwMwrc1aaigUPWIFkNQS9S6dHEqUGECnWpTEzDEW9SfkjjpvyxMMBBAumGj1p9snNuotWhPaqULXWTeTH-IBLbB5UILXZKBQ71hQjREmQZ~Ci03~4uCB46laWmlEEx7nL7bu3MvL72lMnXPZycil78Z8MWW3pDhIL-5DTBvrgZJtwi1yoknXPR4rmRVVwv4etQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10</a></span>). The type of mutation tends to have significant position-based effect most prominent in the RBS: G to C and G to T seem to decrease the initiation rate while the inverse mutation C to G and T to G have the opposite effect and tend to increase the initiation rate in that region. This inverse effect does not always exist, for example, G to A tends to reduce the initiation rate, but A to G tends to have an almost neutral effect. It is likely these changes in the initiation rate are due to changes in the ribosome binding to the mRNA, since the mutation can change the binding site to be closer or farther from the Shine-Delgarno sequence.</p><p>The result reported here provides an estimation of different type of mutations in different regions surrounding the start codon on translation. This estimation can be further validated experimentally via the design of relevant libraries of gene variants and the measurements of translation of the different variants.</p>                            <h3 scrollto-destination=118067954 id="118067954" class="section-title" >9.5. Correlation between initiation rate, elongation rate, ribosome density, and translation rate</h3>
<p>Next, we aimed at understanding the relations between initiation rate, elongation rate, ribosome density and translation rate in endogenous transcripts. Thus, we calculated the Spearman correlation between these values. The results are shown in <a reveal-id="dsx049-T1" data-open="dsx049-T1" class="link link-reveal link-table xref-fig">Table 1</a>. <div content-id="dsx049-T1" class="table-modal"><div class="table-wrap"><div id="dsx049-T1" data-id="dsx049-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Spearman correlation between initiation rate, elongation rate, ribosome density, and translation rate</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Elongation rate </th><th>Ribosome density </th><th>Translation rate </th></tr></thead><tbody><tr><td rowspan="2">Initiation rate </td><td>0.1137 </td><td>0.7822 </td><td>0.6487 </td></tr><tr><td>(1.034 × 10<sup>−6</sup>) </td><td>(&lt;2.2 × 10<sup>−308</sup>) </td><td>(1.125 × 10<sup>−219</sup>) </td></tr><tr><td rowspan="2">Elongation rate </td><td rowspan="2">— </td><td>0.1155 </td><td>0.6454 </td></tr><tr><td>(6.984 × 10<sup>−7</sup>) </td><td>(1.046 × 10<sup>−216</sup>) </td></tr><tr><td rowspan="2">Ribosome density </td><td rowspan="2">— </td><td rowspan="2">— </td><td>0.6514 </td></tr><tr><td>(&lt;2.2 × 10<sup>−308</sup>) </td></tr></tbody></table></div><div content-id="dsx049-T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Elongation rate </th><th>Ribosome density </th><th>Translation rate </th></tr></thead><tbody><tr><td rowspan="2">Initiation rate </td><td>0.1137 </td><td>0.7822 </td><td>0.6487 </td></tr><tr><td>(1.034 × 10<sup>−6</sup>) </td><td>(&lt;2.2 × 10<sup>−308</sup>) </td><td>(1.125 × 10<sup>−219</sup>) </td></tr><tr><td rowspan="2">Elongation rate </td><td rowspan="2">— </td><td>0.1155 </td><td>0.6454 </td></tr><tr><td>(6.984 × 10<sup>−7</sup>) </td><td>(1.046 × 10<sup>−216</sup>) </td></tr><tr><td rowspan="2">Ribosome density </td><td rowspan="2">— </td><td rowspan="2">— </td><td>0.6514 </td></tr><tr><td>(&lt;2.2 × 10<sup>−308</sup>) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>The values in parenthesis indicate the respective <em>P</em>-value (all <em>P</em>-values pass correction for false discovery rate).</p></div></div></div></div></div><div class="table-wrap"><div id="dsx049-T1" data-id="dsx049-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Spearman correlation between initiation rate, elongation rate, ribosome density, and translation rate</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Elongation rate </th><th>Ribosome density </th><th>Translation rate </th></tr></thead><tbody><tr><td rowspan="2">Initiation rate </td><td>0.1137 </td><td>0.7822 </td><td>0.6487 </td></tr><tr><td>(1.034 × 10<sup>−6</sup>) </td><td>(&lt;2.2 × 10<sup>−308</sup>) </td><td>(1.125 × 10<sup>−219</sup>) </td></tr><tr><td rowspan="2">Elongation rate </td><td rowspan="2">— </td><td>0.1155 </td><td>0.6454 </td></tr><tr><td>(6.984 × 10<sup>−7</sup>) </td><td>(1.046 × 10<sup>−216</sup>) </td></tr><tr><td rowspan="2">Ribosome density </td><td rowspan="2">— </td><td rowspan="2">— </td><td>0.6514 </td></tr><tr><td>(&lt;2.2 × 10<sup>−308</sup>) </td></tr></tbody></table></div><div content-id="dsx049-T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Elongation rate </th><th>Ribosome density </th><th>Translation rate </th></tr></thead><tbody><tr><td rowspan="2">Initiation rate </td><td>0.1137 </td><td>0.7822 </td><td>0.6487 </td></tr><tr><td>(1.034 × 10<sup>−6</sup>) </td><td>(&lt;2.2 × 10<sup>−308</sup>) </td><td>(1.125 × 10<sup>−219</sup>) </td></tr><tr><td rowspan="2">Elongation rate </td><td rowspan="2">— </td><td>0.1155 </td><td>0.6454 </td></tr><tr><td>(6.984 × 10<sup>−7</sup>) </td><td>(1.046 × 10<sup>−216</sup>) </td></tr><tr><td rowspan="2">Ribosome density </td><td rowspan="2">— </td><td rowspan="2">— </td><td>0.6514 </td></tr><tr><td>(&lt;2.2 × 10<sup>−308</sup>) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>The values in parenthesis indicate the respective <em>P</em>-value (all <em>P</em>-values pass correction for false discovery rate).</p></div></div></div></div></p><p>We can see weak correlation between elongation rate and initiation rate and strong relationship between initiation rate and ribosome density. We also find that translation rate is highly correlated with all three other measurements. Please note that mathematically there can be cases where some of these correlations do not exist (refer to the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049_suppl_data.zip?Expires=2147483647&Signature=K-XxDsS-lclmJqaifgOD8cqmR8wuXpMiBAd~PLg8rmKVSkcu4VLkYLSfj9Ec7G1iFsozccEB12Zm6NMd-blJmaU~mfKzJJIBhOfD~BkeqoHWZXPqsiCbqz~nIceinOSshRIKx-MORqxUDdLkVEC7VwMwrc1aaigUPWIFkNQS9S6dHEqUGECnWpTEzDEW9SfkjjpvyxMMBBAumGj1p9snNuotWhPaqULXWTeTH-IBLbB5UILXZKBQ71hQjREmQZ~Ci03~4uCB46laWmlEEx7nL7bu3MvL72lMnXPZycil78Z8MWW3pDhIL-5DTBvrgZJtwi1yoknXPR4rmRVVwv4etQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">supplementary material</a></span> for more details). In endogenous/real sequences, a correlation can be explained by the fact that the translation process depends on ribosome initiation and elongation; in addition, sequences with high translation rates will require more ribosomes in average (due to higher initiation rate). The magnitude of the correlation, provided here for the first time, allows estimation of the relative ‘contribution’ of initiation and elongation to translation rate variability in endogenous/real transcripts. In general, when increasing the initiation rate while keeping the elongation rate constant, we expect to see an increase in ribosome density (as more ribosomes are ‘pumped’ into the coding region). Higher elongation rate, for a constant initiation rate, is expected to decrease the ribosome density (as it corresponds to faster removal of ribosomes from the coding region); indeed the partial Spearman correlation between translation elongation and ribosome density when controlling for the initiation rate <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">r</mi><mfenced separators="|" xmlns=""><mrow><mi mathvariant="normal">e</mi><mi mathvariant="normal">l</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">g</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">n</mi><mo>,</mo><mi mathvariant="normal"> </mi><mi mathvariant="normal">d</mi><mi mathvariant="normal">e</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">s</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">y</mi></mrow><mrow><mi mathvariant="normal">i</mi><mi mathvariant="normal">n</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">a</mi><mi mathvariant="normal">t</mi><mi mathvariant="normal">i</mi><mi mathvariant="normal">o</mi><mi mathvariant="normal">n</mi></mrow></mfenced></math></span> is negative (<em>r</em> = −0.044; <em>P</em>-value is borderline significant 0.0597). Increase in initiation and elongation rates (when they are rate limiting) is expected to correspond to an increase in the translation rate. Our analysis demonstrates that genes that tend to have increased/higher initiation rate tend to also have higher elongation rate; the increase in initiation rate in highly expressed genes is more ‘dominant’ in terms of the effect on ribosome density as the correlation between translation rate/protein levels and ribosome density is positive. This supports the approach of various studies that measure translation efficiency via measurements that are related to ribosome density.<a reveal-id="dsx049-B55" data-open="dsx049-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a></p><p>It should be interesting to study in the future other organisms to see if the positive relation between initiation and elongation rate vs. ribosome density is universal.</p>                            <h3 scrollto-destination=118067958 id="118067958" class="section-title" >9.6. Traffic jams study in different gene groups and the way they are effected by mutations</h3>
<p>One advantage of our model is the fact that it provides estimations related to variables such as ribosome collisions/traffic jams that cannot be accurately predicted based on models that estimate only the initiation or only the elongation. For native/endogenous transcripts, we see that most of the jam time can be attributed to the initiation step: across all transcripts, the mean time for initiation jamming is 8.7% of the total initiation time, while for the elongation jam time, the mean is up to 2.4% of the total time (<a reveal-id="dsx049-F5" data-open="dsx049-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Furthermore, the maximum initiation jam time over all gene is 96.9% of the total initiation time, whereas the maximum elongation jam time at a codon is at most 8%. When we measure the mean elongation jam time across all codons, we see the mean is almost 20% out of the elongation time and for 25% of the genes this value is above 27%. </p>                        <a id="118067960" scrollto-destination="118067960"></a>
<div id="dsx049-F5" data-id="dsx049-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/m_dsx049f5.png?Expires=2147483647&Signature=zYB1y1UuthllAogMOClBaaSUftTDYZXDW0Cia96GP8MGL3wJQ2FdQlgajZrH1sNYnFmTdK9yUr1QfMsO8vw2hVbaacpeCFArQeJNZQsbbNiuuK-iohNmaNpohcK5VAR5pyOCZUA7LzELGobQnPaOTvOBslFja0VUshFoLw~g~e04boj37jgaOxsO9zRCEqbFB8Ezixy8nx39zw2714rhrPlVnJu8CMS4~PobU9JPPzo8cEuXf~XSJUZrpawMKiHjkyTqXTzX7eK282QruAGGhma~zliUEFx2DZ~edVF9kL6XLClfZhd7aGS~frdST0Na5dBA0Tgje~0JM9lhCfLdRA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The fraction of time ribosome jam occurs for initiation, on start codon, and mean jam time across the entire ORF. Note that the bin sizes are not uniform but grow exponentially." path-from-xml="dsx049f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118067960/dsx049f5.tif" path-from-xml="dsx049f5.tif" target="_blank">View large</a><a section="118067960" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049f5.png?Expires=2147483647&Signature=b~W2UV5LnAWeUt~7eiMGWMmbu4RBSVOCEBxZy3l2shrHVMymFsPIP~4xNpAzMH0mB~AwAHYfg1N4L1njXIQQ7iamqRpMGut7J3~FWn9VA4oMah3O~gYjHV3fsGaw73cmvE~tRy7OAM6ssayJOPU-O-yZTxBt1Dl~mI~ZB-JMxUGezf7QlYhkXcD0G2ECF2Sh-4lfkvKOuXaJ66FEo0tspLzJtrGRVHNb8sbKeONOnpDofHS-F-BUEf10N4CSevk0Cej3xwg~brBUIhuYoFayJ61leCxccQEqMYYiSEUu4Dy2jzHcTZO0k~zI8dZj8uGSKT6695E0r8merygEI9kGeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118067960&ar=4638432&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx049f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The fraction of time ribosome jam occurs for initiation, on start codon, and mean jam time across the entire ORF. Note that the bin sizes are not uniform but grow exponentially.</p></div></div><div content-id="dsx049-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/m_dsx049f5.png?Expires=2147483647&Signature=zYB1y1UuthllAogMOClBaaSUftTDYZXDW0Cia96GP8MGL3wJQ2FdQlgajZrH1sNYnFmTdK9yUr1QfMsO8vw2hVbaacpeCFArQeJNZQsbbNiuuK-iohNmaNpohcK5VAR5pyOCZUA7LzELGobQnPaOTvOBslFja0VUshFoLw~g~e04boj37jgaOxsO9zRCEqbFB8Ezixy8nx39zw2714rhrPlVnJu8CMS4~PobU9JPPzo8cEuXf~XSJUZrpawMKiHjkyTqXTzX7eK282QruAGGhma~zliUEFx2DZ~edVF9kL6XLClfZhd7aGS~frdST0Na5dBA0Tgje~0JM9lhCfLdRA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The fraction of time ribosome jam occurs for initiation, on start codon, and mean jam time across the entire ORF. Note that the bin sizes are not uniform but grow exponentially." path-from-xml="dsx049f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/118067960/dsx049f5.tif" path-from-xml="dsx049f5.tif" target="_blank">View large</a><a section="118067960" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049f5.png?Expires=2147483647&Signature=b~W2UV5LnAWeUt~7eiMGWMmbu4RBSVOCEBxZy3l2shrHVMymFsPIP~4xNpAzMH0mB~AwAHYfg1N4L1njXIQQ7iamqRpMGut7J3~FWn9VA4oMah3O~gYjHV3fsGaw73cmvE~tRy7OAM6ssayJOPU-O-yZTxBt1Dl~mI~ZB-JMxUGezf7QlYhkXcD0G2ECF2Sh-4lfkvKOuXaJ66FEo0tspLzJtrGRVHNb8sbKeONOnpDofHS-F-BUEf10N4CSevk0Cej3xwg~brBUIhuYoFayJ61leCxccQEqMYYiSEUu4Dy2jzHcTZO0k~zI8dZj8uGSKT6695E0r8merygEI9kGeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=118067960&ar=4638432&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx049f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The fraction of time ribosome jam occurs for initiation, on start codon, and mean jam time across the entire ORF. Note that the bin sizes are not uniform but grow exponentially.</p></div></div><p>In addition, the Spearman correlation between initiation rate and initiation jam time is 0.72 (<em>P</em>-value = <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mn xmlns="">3.61</mn><mo xmlns="">×</mo><msup xmlns=""><mrow><mn>10</mn></mrow><mrow><mo>-</mo><mn>293</mn></mrow></msup></math>⁠</span>) and between initiation rate and mean elongation jam time it is 0.39 (<em>P</em>-value = <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mn xmlns="">1.7</mn><mo xmlns="">×</mo><msup xmlns=""><mrow><mn>10</mn></mrow><mrow><mo>-</mo><mn>69</mn></mrow></msup></math>⁠</span>). The correlation between translation rate and initiation jam time is 0.56 and with mean elongation jam time it is 0.54 (<em>P</em>-value &lt; <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mn xmlns="">2.2</mn><mo xmlns="">×</mo><msup xmlns=""><mrow><mn>10</mn></mrow><mrow><mo>-</mo><mn>308</mn></mrow></msup></math>⁠</span>).</p><p>At the next step, we analyzed the effect of mutations on jamming (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049_suppl_data.zip?Expires=2147483647&Signature=K-XxDsS-lclmJqaifgOD8cqmR8wuXpMiBAd~PLg8rmKVSkcu4VLkYLSfj9Ec7G1iFsozccEB12Zm6NMd-blJmaU~mfKzJJIBhOfD~BkeqoHWZXPqsiCbqz~nIceinOSshRIKx-MORqxUDdLkVEC7VwMwrc1aaigUPWIFkNQS9S6dHEqUGECnWpTEzDEW9SfkjjpvyxMMBBAumGj1p9snNuotWhPaqULXWTeTH-IBLbB5UILXZKBQ71hQjREmQZ~Ci03~4uCB46laWmlEEx7nL7bu3MvL72lMnXPZycil78Z8MWW3pDhIL-5DTBvrgZJtwi1yoknXPR4rmRVVwv4etQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S5–S8</a></span>). We found that the initiation jam time relative change has significantly high correlation with the initiation rate, where for the same position a change occurs—a similar number of transcripts decrease or increase in the same magnitude. We determined that the correlation between the relative change of initiation rate and initiation jam time is 0.93 (<em>P</em>-value &lt; <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mn xmlns="">2.2</mn><mo xmlns="">×</mo><msup xmlns=""><mrow><mn>10</mn></mrow><mrow><mo>-</mo><mn>308</mn></mrow></msup></math>⁠</span>). If we check the same correlation per each mutation position, the minimum correlation is 0.738, and if we exclude the two extreme mutation positions, we see the minimum correlation is 0.856 and a total of 61 out of 67 position have a correlation of above 0.9 (all <em>P</em>-values are lower than <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mn xmlns="">2.2</mn><mo xmlns="">×</mo><msup xmlns=""><mrow><mn>10</mn></mrow><mrow><mo>-</mo><mn>308</mn></mrow></msup></math>⁠</span>). While we have seen in previous sections that translation rate was mostly unaffected by single nucleotide mutations, there is greater impact on the jam time during elongation. This change may indeed further propagate towards changes in initiation rate and initiation jam time, as a change in jam time during elongation may free or occupy codons at the beginning of the ORF, depending on whether the jam time decreased or increased.</p><p>Previous studies have shown that a conserved region (codons 18–50) exists where the codons tend to be slower.<a reveal-id="dsx049-B7" data-open="dsx049-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsx049-B56" data-open="dsx049-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> It was previously suggested that both slower initiation and slower elongation rates at the beginning of the coding region should contribute (independently) to decreased jamming.<a reveal-id="dsx049-B7" data-open="dsx049-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsx049-B56" data-open="dsx049-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> To examine this, we performed the following simulation: first, we changed the codons at positions 18–50 to be the fastest synonymous codons; as a result, the mean traffic jam increased by 6.2%. This value is very significant since increase in traffic jams/ribosome-density should be proportional to the decrease in the number of free ribosomes, which consequently affects the organism growth rate.<a reveal-id="dsx049-B49" data-open="dsx049-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> The effect was even higher when we changed the codons in positions 51–100 to be the slowest (Traffic jam increased by 25.2%) or when we changed all codons downstream the first 50 codons to be slowest (Traffic jam increased by 68.2%).</p><p>In this sub-section, we provide some estimation of traffic jams and the effect of codon distribution/bias on traffic jams. These estimation and relations can further validated experimentally via the design of relevant libraries of gene variants and the measurements of the profile or ribosome densities (e.g. using ribo-seq) of the different variants.</p>                            <h3 scrollto-destination=118067965 id="118067965" class="section-title" >9.7. Genes with relatively extreme translation initiation and translation elongation rate</h3>
<p>Our model enables recognizing and studying groups of genes with unique translation features. In this subsection, we demonstrate this via the analyses of various group of genes with extreme translation elongation and initiation levels.</p><p>We first considered the group of genes with high initiation rate but with relatively low elongation rate. These genes are interesting because, in general, there is a <em>positive</em> correlation between translation initiation and elongation (see previous sections). Relative slower elongation for these genes may be related to additional constraints such as co-translational folding.<a reveal-id="dsx049-B57" data-open="dsx049-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> We found 16 genes to be in the top 10% of the initiation rate and the bottom 10% of the elongation rate. When we raised the threshold to 20%, we found 63 genes (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049_suppl_data.zip?Expires=2147483647&Signature=K-XxDsS-lclmJqaifgOD8cqmR8wuXpMiBAd~PLg8rmKVSkcu4VLkYLSfj9Ec7G1iFsozccEB12Zm6NMd-blJmaU~mfKzJJIBhOfD~BkeqoHWZXPqsiCbqz~nIceinOSshRIKx-MORqxUDdLkVEC7VwMwrc1aaigUPWIFkNQS9S6dHEqUGECnWpTEzDEW9SfkjjpvyxMMBBAumGj1p9snNuotWhPaqULXWTeTH-IBLbB5UILXZKBQ71hQjREmQZ~Ci03~4uCB46laWmlEEx7nL7bu3MvL72lMnXPZycil78Z8MWW3pDhIL-5DTBvrgZJtwi1yoknXPR4rmRVVwv4etQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) and 267 genes were found with a threshold of 40%. Using functional annotation (Gene Ontology: GO; <a class="link link-uri" href="http://www.geneontology.org" target="_blank">http://www.geneontology.org</a>) we found that, among others, the latest group of genes includes 37 genes that are related to organelle inner membrane (GO: 0019866; <em>P</em>-value = <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mn xmlns="">6.84</mn><mo xmlns="">×</mo><msup xmlns=""><mrow><mn>10</mn></mrow><mrow><mo>-</mo><mn>19</mn></mrow></msup></math>⁠</span>; all the <em>P</em>-values reported in this section were adjusted for multiple comparison—see the Methods section) and 57 genes that are related to ‘plasma membrane’ (GO: 0005886; <em>P</em>-value = <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mn xmlns="">6.84</mn><mo xmlns="">×</mo><msup xmlns=""><mrow><mn>10</mn></mrow><mrow><mo>-</mo><mn>9</mn></mrow></msup></math>⁠</span>).</p><p>One explanation for these results may be related to the suggestion that membrane proteins undergo various maturation aspects during the elongation step which requires slower elongation (while the initiation may remain efficient).<a reveal-id="dsx049-B7" data-open="dsx049-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsx049-B58 dsx049-B59 dsx049-B60" data-open="dsx049-B58 dsx049-B59 dsx049-B60" class="link link-ref link-reveal xref-bibr"><sup>58–60</sup></a></p><p>The second gene group we study includes genes with relatively low initiation rate (bottom 40%) and high elongation rate (40%). We found 170 such genes (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049_suppl_data.zip?Expires=2147483647&Signature=K-XxDsS-lclmJqaifgOD8cqmR8wuXpMiBAd~PLg8rmKVSkcu4VLkYLSfj9Ec7G1iFsozccEB12Zm6NMd-blJmaU~mfKzJJIBhOfD~BkeqoHWZXPqsiCbqz~nIceinOSshRIKx-MORqxUDdLkVEC7VwMwrc1aaigUPWIFkNQS9S6dHEqUGECnWpTEzDEW9SfkjjpvyxMMBBAumGj1p9snNuotWhPaqULXWTeTH-IBLbB5UILXZKBQ71hQjREmQZ~Ci03~4uCB46laWmlEEx7nL7bu3MvL72lMnXPZycil78Z8MWW3pDhIL-5DTBvrgZJtwi1yoknXPR4rmRVVwv4etQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>); the group was enriched with genes relate to purine nucleotide biosynthetic process (GO: 0006164/ecx00230; <em>P</em> = 2.52 × 10<sup>−3</sup>) and genes related to translation/ribosome (GO: 0006412; <em>P</em> = 0.03). It is possible that these are highly expressed genes with constraints on the nucleotide composition near the start codon that effect the evolution of and optimality of their initiation (e.g. due to specific amino acids at the 5′end or signals related to transcription); it is also possible that some of them have non-canonical initiation regulation.</p><p>The third gene group we study includes genes with relatively high initiation rate (bottom 40%) and high elongation rate (40%). We found 228 such genes (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049_suppl_data.zip?Expires=2147483647&Signature=K-XxDsS-lclmJqaifgOD8cqmR8wuXpMiBAd~PLg8rmKVSkcu4VLkYLSfj9Ec7G1iFsozccEB12Zm6NMd-blJmaU~mfKzJJIBhOfD~BkeqoHWZXPqsiCbqz~nIceinOSshRIKx-MORqxUDdLkVEC7VwMwrc1aaigUPWIFkNQS9S6dHEqUGECnWpTEzDEW9SfkjjpvyxMMBBAumGj1p9snNuotWhPaqULXWTeTH-IBLbB5UILXZKBQ71hQjREmQZ~Ci03~4uCB46laWmlEEx7nL7bu3MvL72lMnXPZycil78Z8MWW3pDhIL-5DTBvrgZJtwi1yoknXPR4rmRVVwv4etQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>); this group, naturally, includes highly expressed genes and enriched with housekeeping gene groups/functions such as amino acid metabolism (e.g. GO: 0008652 with <em>P</em> = 1.44 × 10<sup>−5</sup>) translation/ribosome (e.g. GO: 0003735 with <em>P</em> = 7.25 × 10<sup>−6</sup>), protein folding/chaperone (GO: 0006457 with <em>P</em> = 0.0067).</p><p>The groups of genes described here can be further experimentally studied in the future, by introducing silent mutations into the coding sequences and the UTRs of these genes and examining their effect on the improvement of translation rate, the functionality of the related proteins, and the organism fitness.</p>                            <h3 scrollto-destination=118067972 id="118067972" class="section-title" >9.8. Mutations with reciprocal effect on initiation and elongation</h3>
<p>In this section we aimed at further demonstrating how the new model can answer new important questions related to the effect on transcript evolution and mRNA translation regulation and biophysics. The (non intuitive) reciprocal effect of mutations on elongation and initiation in different regions is a simple example of such a question which was already discussed in the past<a reveal-id="dsx049-B7" data-open="dsx049-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> but with no clear quantitative analyses.</p><p>Thus, mutations causing reciprocal effect in initiation and elongation are of interest, given that we naturally see/expect significant correlation between elongation and initiation rate. To study these types of mutations, for each possible mutation occurring in the beginning of the ORF (positions +3 until +34), we looked for cases where the initiation rate was increased by at least 25%, while the codon translation rate was reduced by at least 25% (which we name ‘type 1 mutations’). We also looked for cases where initiation rate decreased by at least 25% and codon translation rate was increased by 25% (which we name ‘type 2 mutations’). For each of the above cases such occurrence happens in 4.6% of the ORF mutations. More details of per-position changes are available in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049_suppl_data.zip?Expires=2147483647&Signature=K-XxDsS-lclmJqaifgOD8cqmR8wuXpMiBAd~PLg8rmKVSkcu4VLkYLSfj9Ec7G1iFsozccEB12Zm6NMd-blJmaU~mfKzJJIBhOfD~BkeqoHWZXPqsiCbqz~nIceinOSshRIKx-MORqxUDdLkVEC7VwMwrc1aaigUPWIFkNQS9S6dHEqUGECnWpTEzDEW9SfkjjpvyxMMBBAumGj1p9snNuotWhPaqULXWTeTH-IBLbB5UILXZKBQ71hQjREmQZ~Ci03~4uCB46laWmlEEx7nL7bu3MvL72lMnXPZycil78Z8MWW3pDhIL-5DTBvrgZJtwi1yoknXPR4rmRVVwv4etQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>. Interestingly, we found that, on average, in positions 3–17 of the coding region there are higher levels of mutations of type 2. Conversely, there are more occurrences of type 1 mutations in positions 18–34. Using Wilcoxon signed-rank test<a reveal-id="dsx049-B61" data-open="dsx049-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> we compared, per each mutation position, the relative initiation rate and the relative codon decoding rate. In about half of the positions within the range of 3 and 17, and in all of the positions in the range of 18–34, we found significant differences. These results demonstrate that many point mutations can have reciprocal effect on initiation and elongation and that the first nucleotides of the coding region tend to be relatively more ‘optimized’ to the translation initiation stage than the elongation stage. Previous studies have suggested that the beginning of the coding sequence includes ‘late initiation’ signals, which are part of the initiation step but encoded in the coding region.<a reveal-id="dsx049-B7" data-open="dsx049-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Our model enables the prediction of the exact boundaries of this region.</p><p>In this sub-section, we provide estimations of the effect of each nucleotide/region on initiation and elongation. These estimations and relations can be further validated experimentally via the design of relevant libraries of gene variants and the measurements of the profile or ribosome densities (e.g. using ribo-seq) or better performing high resolution tracking of ribosome movements<a reveal-id="dsx049-B62" data-open="dsx049-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> over the different variants.</p>                            <h2 scrollto-destination=118067976 id="118067976" class="section-title" >10. Discussion</h2>
<p>The model described in this study allows predicting, for the first time, translation dynamics based on the transcript features alone. As we demonstrate here, the model enables performing analyses that were not possible within the framework of previous models (e.g. statistical/machine learning models, or models that consider only the initiation or elongation step). Specifically, we demonstrate how the model can be used for predicting expression levels of endogenous and heterologous genes and predicting various aspects related to the translation dynamics. In addition, the model is very useful for studying the molecular evolution of translation aspects via studying the effect of mutations on translation dynamics.</p><p>Among others, we demonstrate a close association between changes in initiation rate and jam time during initiation. Within the framework of our model, it is easy to explain such relations, as increase or decrease in initiation rate can directly alter jam time. Within the cell, it is also possible that changes affecting the initiation jam time (such as those occurring in the ORF) can influence the initiation rate. We also demonstrate that mutations at the RBS region have the strongest effect on translation and we show that membrane genes tend to have relatively slower elongation and high initiation. Furthermore, we show that around 4–5% of the mutations near the beginning of the ORF <em>increase</em> initiation rate by at least 25% while <em>decreasing</em> by at least 25% the elongation efficiency. We also find 4–5% of these mutations <em>increase</em> elongation while <em>reducing</em> initiation rate by the same factor. Our analysis provides an estimation of the ratio between translation rate and initiation rate in the <em>E. coli</em> endogenous genes and found that its median is 10.13%. This result further implies that elongation (and the coding region itself) and not only initiation has central effect on the translation rate.</p><p>It is important to emphasize that this model connects, for the first time, the impact of <em>mutations</em> (possibly silent) in the transcripts with <em>fundamental biophysical phenomena</em> (e.g., ribosome allocation, traffic jams, translation rate, etc.), which are directly related to the organism fitness. Thus, with this model (and additional relevant adjustments), we are able to understand the effect of silent mutations on fitness and thus estimate their corresponding selection pressure; this is an important aspect that should be integrated into any realistic molecular evolution models. While existing models (including non-physical ones such as regressors) may similarly predict the expression levels of endogenous genes,<a reveal-id="dsx049-B23" data-open="dsx049-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a><sup>,</sup><a reveal-id="dsx049-B25" data-open="dsx049-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a><sup>,</sup><a reveal-id="dsx049-B63" data-open="dsx049-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> it is important to emphasize that our model’s novel potential to predict the impact of single mutations on initiation and elongation biophysics, as well as other translation aspects, based on the mRNA sequence alone.</p><p>These results can only be derived from a combined model, such as the one presented here. The model’s application can be downloaded and used in future quantitative studies related to translation in the field.</p><p>Future work can extend the current model to support wider and more complex aspects of translation. For instance, whole cell simulations, taking into account additional constraints, such as the tRNA pool size and ribosome pool size may provide additional insights. mRNA and protein degradation rates can be incorporated to better correlate mRNA levels with protein abundance, etc.</p><p>We chose to calculate the initiation rates only for the first gene in each operon. Extending the calculations to genes in other positions of the operon may yield additional interesting results but this requires modifications in the model due to additional possible interactions of the mRNA molecules with itself and with the ribosome.<a reveal-id="dsx049-B64" data-open="dsx049-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> Furthermore, although we provide some initial results for <em>B. subtilis</em> (see in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049_suppl_data.zip?Expires=2147483647&Signature=K-XxDsS-lclmJqaifgOD8cqmR8wuXpMiBAd~PLg8rmKVSkcu4VLkYLSfj9Ec7G1iFsozccEB12Zm6NMd-blJmaU~mfKzJJIBhOfD~BkeqoHWZXPqsiCbqz~nIceinOSshRIKx-MORqxUDdLkVEC7VwMwrc1aaigUPWIFkNQS9S6dHEqUGECnWpTEzDEW9SfkjjpvyxMMBBAumGj1p9snNuotWhPaqULXWTeTH-IBLbB5UILXZKBQ71hQjREmQZ~Ci03~4uCB46laWmlEEx7nL7bu3MvL72lMnXPZycil78Z8MWW3pDhIL-5DTBvrgZJtwi1yoknXPR4rmRVVwv4etQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary material</a></span>), our analysis focused on <em>E. coli</em> simply since it is the prokaryote with the most abundance relevant large-scale measurements. It is important to emphasize that the model (with some small adjustment) can be used on other prokaryotes and in the future, it will be interesting to comprehensively study other prokaryotes with similar models. In addition, it will be interesting to develop similar model for Eukaryotes and in different tissues; specifically, for this domain currently there is no initiation model while elongation model exists and is relatively similar between the two domains.<a reveal-id="dsx049-B65" data-open="dsx049-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a></p><p>Finally, in this work, we assume steady state of the translation process. A more dynamic model, taking into account mRNA folding during ribosomes detachment and mRNA decay rate may result in simulations of higher accuracy and a better understanding of the biophysics of translation.</p>                            <h2 scrollto-destination=118067984 id="118067984" class="section-title" >Funding</h2>
<p>This research was partially supported by a research grant from the Israeli Ministry of Science, Technology and Space and by a grant from the Ela Kodesz Institute for Medical Physics and Engineering.</p>                            <h2 scrollto-destination=118067986 id="118067986" class="section-title" >Conflict of interest</h2>
<p>None declared.</p>                            <h2 scrollto-destination=118067988 id="118067988" class="section-title" >Supplementary data</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049_suppl_data.zip?Expires=2147483647&Signature=K-XxDsS-lclmJqaifgOD8cqmR8wuXpMiBAd~PLg8rmKVSkcu4VLkYLSfj9Ec7G1iFsozccEB12Zm6NMd-blJmaU~mfKzJJIBhOfD~BkeqoHWZXPqsiCbqz~nIceinOSshRIKx-MORqxUDdLkVEC7VwMwrc1aaigUPWIFkNQS9S6dHEqUGECnWpTEzDEW9SfkjjpvyxMMBBAumGj1p9snNuotWhPaqULXWTeTH-IBLbB5UILXZKBQ71hQjREmQZ~Ci03~4uCB46laWmlEEx7nL7bu3MvL72lMnXPZycil78Z8MWW3pDhIL-5DTBvrgZJtwi1yoknXPR4rmRVVwv4etQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h2 scrollto-destination=118067990 id="118067990" class="backfootnotegroup-title" >Footnotes</h2>
<div content-id="dsx049-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Mikita Suyama</div></div></div>                            <h2 scrollto-destination=118067992 id="118067992" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx049-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hannig</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Makrides</div> <div class="given-names">S.C.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Strategies for optimizing heterologous protein expression in Escherichia coli</div>, <div class="source">Trends Biotechnol</div> ., <div class="volume">16</div>, <div class="fpage">54</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strategies%20for%20optimizing%20heterologous%20protein%20expression%20in%20Escherichia%20coli&amp;author=G.%20Hannig&amp;author=S.C.%20Makrides&amp;publication_year=1998&amp;journal=Trends%20Biotechnol&amp;volume=16&amp;pages=54-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0167-7799(97)01155-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0167-7799(97)01155-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0167-7799(97)01155-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9487731" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kozak</div> <div class="given-names">M.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Initiation of translation in prokaryotes and eukaryotes</div>, <div class="source">Gene</div> , <div class="volume">234</div>, <div class="fpage">187</div>–<div class="lpage">208</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Initiation%20of%20translation%20in%20prokaryotes%20and%20eukaryotes&amp;author=M.%20Kozak&amp;publication_year=1999&amp;journal=Gene&amp;volume=234&amp;pages=187-208" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(99)00210-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(99)00210-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(99)00210-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10395892" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shine</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Dalgarno</div> <div class="given-names">L.</div></div></div> <div class="year">1975</div>, 
<div class="article-title">Determinant of cistron specificity in bacterial ribosomes</div>, <div class="source">Nature</div> , <div class="volume">254</div>, <div class="fpage">34</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Determinant%20of%20cistron%20specificity%20in%20bacterial%20ribosomes&amp;author=J.%20Shine&amp;author=L.%20Dalgarno&amp;publication_year=1975&amp;journal=Nature&amp;volume=254&amp;pages=34-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/254034a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F254034a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F254034a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/803646" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div></div> <div class="year">1981</div>, 
<div class="article-title">Correlation between the abundance of <em>Escherichia coli</em> transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the E. coli translational system</div>, <div class="source">J. Mol. Biol</div> ., <div class="volume">151</div>, <div class="fpage">389</div>–<div class="lpage">409</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correlation%20between%20the%20abundance%20of%20Escherichia%20coli%20transfer%20RNAs%20and%20the%20occurrence%20of%20the%20respective%20codons%20in%20its%20protein%20genes%3A%20a%20proposal%20for%20a%20synonymous%20codon%20choice%20that%20is%20optimal%20for%20the%20E.%20coli%20translational%20system&amp;author=T.%20Ikemura&amp;publication_year=1981&amp;journal=J.%20Mol.%20Biol&amp;volume=151&amp;pages=389-409" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(81)90003-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(81)90003-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(81)90003-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6175758" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gouy</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Gautier</div> <div class="given-names">C.</div></div></div> <div class="year">1982</div>, 
<div class="article-title">Codon usage in bacteria: correlation with gene expressivity</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">10</div>, <div class="fpage">7055</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20in%20bacteria%3A%20correlation%20with%20gene%20expressivity&amp;author=M.%20Gouy&amp;author=C.%20Gautier&amp;publication_year=1982&amp;journal=Nucleic%20Acids%20Res&amp;volume=10&amp;pages=7055-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/10.22.7055" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F10.22.7055" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F10.22.7055"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6760125" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dana</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">The effect of tRNA levels on decoding times of mRNA codons</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">9171</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20effect%20of%20tRNA%20levels%20on%20decoding%20times%20of%20mRNA%20codons&amp;author=A.%20Dana&amp;author=T.%20Tuller&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=9171-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku646" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku646" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku646"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25056313" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Zur</div> <div class="given-names">H.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Multiple roles of the coding sequence 5’ end in gene expression regulation</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">13</div>–<div class="lpage">28</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20roles%20of%20the%20coding%20sequence%205%E2%80%99%20end%20in%20gene%20expression%20regulation&amp;author=T.%20Tuller&amp;author=H.%20Zur&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=13-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku1313" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku1313" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku1313"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25505165" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gustafsson</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Govindarajan</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Minshull</div> <div class="given-names">J.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Codon bias and heterologous protein expression</div>, <div class="source">Trends Biotechnol</div> ., <div class="volume">22</div>, <div class="fpage">346</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20bias%20and%20heterologous%20protein%20expression&amp;author=C.%20Gustafsson&amp;author=S.%20Govindarajan&amp;author=J.%20Minshull&amp;publication_year=2004&amp;journal=Trends%20Biotechnol&amp;volume=22&amp;pages=346-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2004.04.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2004.04.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2004.04.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15245907" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kimchi-Sarfaty</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Oh</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">I.-W.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">A “Silent” Polymorphism in the MDR1 gene changes substrate specificity</div>, <div class="source">Science</div> , <div class="volume">315</div>, <div class="fpage">525</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20%E2%80%9CSilent%E2%80%9D%20Polymorphism%20in%20the%20MDR1%20gene%20changes%20substrate%20specificity&amp;author=C.%20Kimchi-Sarfaty&amp;author=J.%20M.%20Oh&amp;author=I.-W.%20Kim&amp;publication_year=2007&amp;journal=Science&amp;volume=315&amp;pages=525-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1135308" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1135308" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1135308"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17185560" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bahir</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Fromer</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Prat</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Linial</div> <div class="given-names">M.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Viral adaptation to host: a proteome-based analysis of codon usage and amino acid preferences</div>, <div class="source">Mol. Syst. Biol</div> ., <div class="volume">5</div>, <div class="fpage">311</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Viral%20adaptation%20to%20host%3A%20a%20proteome-based%20analysis%20of%20codon%20usage%20and%20amino%20acid%20preferences&amp;author=I.%20Bahir&amp;author=M.%20Fromer&amp;author=Y.%20Prat&amp;author=M.%20Linial&amp;publication_year=2009&amp;journal=Mol.%20Syst.%20Biol&amp;volume=5&amp;pages=311" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb.2009.71" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb.2009.71" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb.2009.71"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19888206" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Saha</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Shabalina</div> <div class="given-names">S.A.</div></div>, <div class="name string-name"><div class="surname">Kashina</div> <div class="given-names">A.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Differential arginylation of actin isoforms is regulated by coding sequence-dependent degradation</div>, <div class="source">Science</div> , <div class="volume">329</div>, <div class="fpage">1534</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20arginylation%20of%20actin%20isoforms%20is%20regulated%20by%20coding%20sequence-dependent%20degradation&amp;author=F.%20Zhang&amp;author=S.%20Saha&amp;author=S.A.%20Shabalina&amp;author=A.%20Kashina&amp;publication_year=2010&amp;journal=Science&amp;volume=329&amp;pages=1534-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1191701" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1191701" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1191701"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20847274" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fredrick</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Ibba</div> <div class="given-names">M.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">How the sequence of a gene can tune its translation</div>, <div class="source">Cell</div> , <div class="volume">141</div>, <div class="fpage">227</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20the%20sequence%20of%20a%20gene%20can%20tune%20its%20translation&amp;author=K.%20Fredrick&amp;author=M.%20Ibba&amp;publication_year=2010&amp;journal=Cell&amp;volume=141&amp;pages=227-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2010.03.033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2010.03.033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2010.03.033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20403320" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zur</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Predictive biophysical modeling and understanding of the dynamics of mRNA translation and its evolution</div>, <div class="source">Nucleic Acids Res</div> , <div class="volume">44</div>, <div class="fpage">9031</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predictive%20biophysical%20modeling%20and%20understanding%20of%20the%20dynamics%20of%20mRNA%20translation%20and%20its%20evolution&amp;author=H.%20Zur&amp;author=T.%20Tuller&amp;publication_year=2016&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;pages=9031-49" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27591251" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Reuveni</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Meilijson</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Kupiec</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Ruppin</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Genome-scale analysis of translation elongation with a ribosome flow model wasserman,</div> <div class="source">PLoS Comput. Biol</div> ., <div class="volume">7</div>, <div class="fpage">e1002127</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-scale%20analysis%20of%20translation%20elongation%20with%20a%20ribosome%20flow%20model%20wasserman%2C&amp;author=S.%20Reuveni&amp;author=I.%20Meilijson&amp;author=M.%20Kupiec&amp;author=E.%20Ruppin&amp;author=T.%20Tuller&amp;publication_year=2011&amp;journal=PLoS%20Comput.%20Biol&amp;volume=7&amp;pages=e1002127" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1002127" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1002127" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1002127"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21909250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div></div> <div class="year">1987</div>, 
<div class="article-title">The codon Adaptation Index—a measure of directional synonymous codon usage bias, and its potential applications</div>, <div class="source">Nucl. Acids Res</div> ., <div class="volume">15</div>, <div class="fpage">1281</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20codon%20Adaptation%20Index%E2%80%94a%20measure%20of%20directional%20synonymous%20codon%20usage%20bias%2C%20and%20its%20potential%20applications&amp;author=P.M.%20Sharp&amp;author=W.H.%20Li&amp;publication_year=1987&amp;journal=Nucl.%20Acids%20Res&amp;volume=15&amp;pages=1281-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/15.3.1281" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F15.3.1281" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F15.3.1281"> </span></div> </div></div></div></div></div><div content-id="dsx049-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shaw</div> <div class="given-names">L.B.</div></div>, <div class="name string-name"><div class="surname">Zia</div> <div class="given-names">R.K.P.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">K.H.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Totally asymmetric exclusion process with extended objects: a model for protein synthesis</div>, <div class="source">Phys. Rev. E</div> , <div class="volume">68</div>, <div class="fpage">021910</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Totally%20asymmetric%20exclusion%20process%20with%20extended%20objects%3A%20a%20model%20for%20protein%20synthesis&amp;author=L.B.%20Shaw&amp;author=R.K.P.%20Zia&amp;author=K.H.%20Lee&amp;publication_year=2003&amp;journal=Phys.%20Rev.%20E&amp;volume=68&amp;pages=021910" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1103/PhysRevE.68.021910" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1103%2FPhysRevE.68.021910" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1103%2FPhysRevE.68.021910"> </span></div> </div></div></div></div></div><div content-id="dsx049-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">dos Reis</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Savva</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Wernisch</div> <div class="given-names">L.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Solving the riddle of codon usage preferences: a test for translational selection</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">32</div>, <div class="fpage">5036</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Solving%20the%20riddle%20of%20codon%20usage%20preferences%3A%20a%20test%20for%20translational%20selection&amp;author=M.%20dos%20Reis&amp;author=R.%20Savva&amp;author=L.%20Wernisch&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;pages=5036-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh834" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh834" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh834"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15448185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dana</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Mean of the typical decoding rates: a new translation efficiency index based on the analysis of ribosome profiling data</div>, <div class="source">G3 Bethesda Md</div> , <div class="volume">5</div>, <div class="fpage">73</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mean%20of%20the%20typical%20decoding%20rates%3A%20a%20new%20translation%20efficiency%20index%20based%20on%20the%20analysis%20of%20ribosome%20profiling%20data&amp;author=A.%20Dana&amp;author=T.%20Tuller&amp;publication_year=2014&amp;journal=G3%20Bethesda%20Md&amp;volume=5&amp;pages=73-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.114.015099" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.114.015099" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.114.015099"> </span></div> </div></div></div></div></div><div content-id="dsx049-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Salis</div> <div class="given-names">H.M.</div></div>, <div class="name string-name"><div class="surname">Mirsky</div> <div class="given-names">E.A.</div></div>, <div class="name string-name"><div class="surname">Voigt</div> <div class="given-names">C.A.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Automated design of synthetic ribosome binding sites to control protein expression</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">27</div>, <div class="fpage">946</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20design%20of%20synthetic%20ribosome%20binding%20sites%20to%20control%20protein%20expression&amp;author=H.M.%20Salis&amp;author=E.A.%20Mirsky&amp;author=C.A.%20Voigt&amp;publication_year=2009&amp;journal=Nat.%20Biotechnol&amp;volume=27&amp;pages=946-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1568" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1568" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1568"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19801975" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gu</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Wilke</div> <div class="given-names">C.O.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">A universal trend of reduced mRNA stability near the translation-initiation site in prokaryotes and eukaryotes</div> , <div class="source">PLoS Comput. Biol</div> ., <div class="volume">6</div>, <div class="fpage">e1000664</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20universal%20trend%20of%20reduced%20mRNA%20stability%20near%20the%20translation-initiation%20site%20in%20prokaryotes%20and%20eukaryotes&amp;author=W.%20Gu&amp;author=T.%20Zhou&amp;author=C.O.%20Wilke&amp;publication_year=2010&amp;journal=PLoS%20Comput.%20Biol&amp;volume=6&amp;pages=e1000664" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000664" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1000664" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1000664"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20140241" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Salis</div> <div class="given-names">H.M.</div></div></div> <div class="year">2011</div>, <div class="source">The Ribosome Binding Site CalculatorMethods in Enzymology</div> . 
<div class="publisher-name">Elsevier</div>, pp. <div class="fpage">19</div>–<div class="lpage">42</div>.</div></div></div></div><div content-id="dsx049-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zur</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">New universal rules of eukaryotic translation initiation fidelity</div> <div class="source">PLoS Comput. Biol</div> ., <div class="volume">9</div>, <div class="fpage">e1003136</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20universal%20rules%20of%20eukaryotic%20translation%20initiation%20fidelity&amp;author=H.%20Zur&amp;author=T.%20Tuller&amp;publication_year=2013&amp;journal=PLoS%20Comput.%20Biol&amp;volume=9&amp;pages=e1003136" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1003136" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1003136" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1003136"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23874179" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vogel</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Abreu Rde</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ko</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Sequence signatures and mRNA concentration can explain two‐thirds of protein abundance variation in a human cell line</div>, <div class="source">Mol. Syst. Biol</div> ., <div class="volume">6</div>, <div class="fpage">400</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20signatures%20and%20mRNA%20concentration%20can%20explain%20two%E2%80%90thirds%20of%20protein%20abundance%20variation%20in%20a%20human%20cell%20line&amp;author=C.%20Vogel&amp;author=S.%20Abreu%20Rde&amp;author=D.%20Ko&amp;publication_year=2010&amp;journal=Mol.%20Syst.%20Biol&amp;volume=6&amp;pages=400" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb.2010.59" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb.2010.59" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb.2010.59"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20739923" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huang</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Wan</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Xu</div> <div class="given-names">Z.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Analysis and prediction of translation rate based on sequence and functional features of the mRNA Kudla,</div> <div class="source">PLoS One</div> , <div class="volume">6</div>, <div class="fpage">e16036</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20and%20prediction%20of%20translation%20rate%20based%20on%20sequence%20and%20functional%20features%20of%20the%20mRNA%20Kudla%2C&amp;author=T.%20Huang&amp;author=S.%20Wan&amp;author=Z.%20Xu&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e16036" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0016036" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0016036" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0016036"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21253596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zur</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Transcript features alone enable accurate prediction and understanding of gene expression in <em>S. cerevisiae</em></div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">14</div>, <div class="fpage">S1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcript%20features%20alone%20enable%20accurate%20prediction%20and%20understanding%20of%20gene%20expression%20in%20S.%20cerevisiae&amp;author=H.%20Zur&amp;author=T.%20Tuller&amp;publication_year=2013&amp;journal=BMC%20Bioinformatics&amp;volume=14&amp;pages=S1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-14-S15-S1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-14-S15-S1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-14-S15-S1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24564391" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lorenz</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Bernhart</div> <div class="given-names">S.H.</div></div>, <div class="name string-name"><div class="surname">Höner zu Siederdissen</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">ViennaRNA Package 2.0</div>, <div class="source">Algorithms Mol. Biol</div> ., <div class="volume">6</div>, <div class="fpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ViennaRNA%20Package%202.0&amp;author=R.%20Lorenz&amp;author=S.H.%20Bernhart&amp;author=C.%20H%C3%B6ner%20zu%20Siederdissen&amp;publication_year=2011&amp;journal=Algorithms%20Mol.%20Biol&amp;volume=6&amp;pages=26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1748-7188-6-26" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1748-7188-6-26" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1748-7188-6-26"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22115189" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hashimoto</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Ichimura</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Mizoguchi</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2004</div>, 
<div class="article-title">Cell size and nucleoid organization of engineered Escherichia coli cells with a reduced genome: engineered <em>E. coli</em> with a reduced genome</div>, <div class="source">Mol. Microbiol</div> ., <div class="volume">55</div>, <div class="fpage">137</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cell%20size%20and%20nucleoid%20organization%20of%20engineered%20Escherichia%20coli%20cells%20with%20a%20reduced%20genome%3A%20engineered%20E.%20coli%20with%20a%20reduced%20genome&amp;author=M.%20Hashimoto&amp;author=T.%20Ichimura&amp;author=H.%20Mizoguchi&amp;publication_year=2004&amp;journal=Mol.%20Microbiol&amp;volume=55&amp;pages=137-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-2958.2004.04386.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-2958.2004.04386.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-2958.2004.04386.x"> </span></div> </div></div></div></div></div><div content-id="dsx049-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kato</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Hashimoto</div> <div class="given-names">M.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Construction of consecutive deletions of the <em>Escherichia coli</em> chromosome</div>, <div class="source">Mol. Syst. Biol</div> ., <div class="volume">3</div>, <div class="fpage">132</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20of%20consecutive%20deletions%20of%20the%20Escherichia%20coli%20chromosome&amp;author=J.%20Kato&amp;author=M.%20Hashimoto&amp;publication_year=2007&amp;journal=Mol.%20Syst.%20Biol&amp;volume=3&amp;pages=132" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb4100174" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb4100174" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb4100174"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17700540" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Siwiak</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Zielenkiewicz</div> <div class="given-names">P.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">A Comprehensive, quantitative, and genome-wide model of translation</div>, <div class="source">PLoS Comput. Biol</div> ., <div class="volume">6</div>, <div class="fpage">e1000865</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20Comprehensive%2C%20quantitative%2C%20and%20genome-wide%20model%20of%20translation&amp;author=M.%20Siwiak&amp;author=P.%20Zielenkiewicz&amp;publication_year=2010&amp;journal=PLoS%20Comput.%20Biol&amp;volume=6&amp;pages=e1000865" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pcbi.1000865" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pcbi.1000865" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pcbi.1000865"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20686685" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Siwiak</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Zielenkiewicz</div> <div class="given-names">P.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Transimulation -protein biosynthesis web service,</div> <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e73943</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transimulation%20-protein%20biosynthesis%20web%20service%2C&amp;author=M.%20Siwiak&amp;author=P.%20Zielenkiewicz&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e73943" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0073943" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0073943" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0073943"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24040122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dana</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Properties and determinants of codon decoding time distributions</div>, <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">S13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Properties%20and%20determinants%20of%20codon%20decoding%20time%20distributions&amp;author=A.%20Dana&amp;author=T.%20Tuller&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=S13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-S6-S13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-S6-S13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-S6-S13"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25572668" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guet</div> <div class="given-names">C.C.</div></div>, <div class="name string-name"><div class="surname">Bruneaux</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Min</div> <div class="given-names">T.L.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">Minimally invasive determination of mRNA concentration in single living bacteria</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">36</div>, <div class="fpage">e73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Minimally%20invasive%20determination%20of%20mRNA%20concentration%20in%20single%20living%20bacteria&amp;author=C.C.%20Guet&amp;author=L.%20Bruneaux&amp;author=T.L.%20Min&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res&amp;volume=36&amp;pages=e73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn329" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn329" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn329"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18515347" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gama-Castro</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Salgado</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Santos-Zavaleta</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">RegulonDB version 9.0: high-level integration of gene regulation, coexpression, motif clustering and beyond</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">44</div>, <div class="fpage">D133</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RegulonDB%20version%209.0%3A%20high-level%20integration%20of%20gene%20regulation%2C%20coexpression%2C%20motif%20clustering%20and%20beyond&amp;author=S.%20Gama-Castro&amp;author=H.%20Salgado&amp;author=A.%20Santos-Zavaleta&amp;publication_year=2016&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;pages=D133-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv1156" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv1156" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv1156"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26527724" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Weiss</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Simonovic</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">PaxDb, a database of protein abundance averages across all three domains of life</div>, <div class="source">Mol. Cell. Proteomics</div> , <div class="volume">11</div>, <div class="fpage">492</div>–<div class="lpage">500</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PaxDb%2C%20a%20database%20of%20protein%20abundance%20averages%20across%20all%20three%20domains%20of%20life&amp;author=M.%20Wang&amp;author=M.%20Weiss&amp;author=M.%20Simonovic&amp;publication_year=2012&amp;journal=Mol.%20Cell.%20Proteomics&amp;volume=11&amp;pages=492-500" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/mcp.O111.014704" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fmcp.O111.014704" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fmcp.O111.014704"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22535208" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Taniguchi</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Choi</div> <div class="given-names">P.J.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">G.-W.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Quantifying <em>E. coli</em> proteome and transcriptome with single-molecule sensitivity in single cells</div>, <div class="source">Science</div> , <div class="volume">329</div>, <div class="fpage">533</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantifying%20E.%20coli%20proteome%20and%20transcriptome%20with%20single-molecule%20sensitivity%20in%20single%20cells&amp;author=Y.%20Taniguchi&amp;author=P.J.%20Choi&amp;author=G.-W.%20Li&amp;publication_year=2010&amp;journal=Science&amp;volume=329&amp;pages=533-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1188308" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1188308" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1188308"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20671182" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">G.-W.</div></div>, <div class="name string-name"><div class="surname">Oh</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Weissman</div> <div class="given-names">J. S.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">The anti-Shine–Dalgarno sequence drives translational pausing and codon choice in bacteria</div>, <div class="source">Nature</div> , <div class="volume">484</div>, <div class="fpage">538</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20anti-Shine%E2%80%93Dalgarno%20sequence%20drives%20translational%20pausing%20and%20codon%20choice%20in%20bacteria&amp;author=G.-W.%20Li&amp;author=E.%20Oh&amp;author=J.%20S.%20Weissman&amp;publication_year=2012&amp;journal=Nature&amp;volume=484&amp;pages=538-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10965" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10965" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10965"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22456704" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Martin</div> <div class="given-names">M.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads</div>, <div class="source">Embnet. J</div> ., <div class="volume">17</div>, <div class="fpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cutadapt%20removes%20adapter%20sequences%20from%20high-throughput%20sequencing%20reads&amp;author=M.%20Martin&amp;publication_year=2011&amp;journal=Embnet.%20J&amp;volume=17&amp;pages=10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.14806/ej.17.1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.14806%2Fej.17.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.14806%2Fej.17.1"> </span></div> </div></div></div></div></div><div content-id="dsx049-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Langmead</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pop</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</div>, <div class="source">Genome Biol</div> ., <div class="volume">10</div>, <div class="fpage">R25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultrafast%20and%20memory-efficient%20alignment%20of%20short%20DNA%20sequences%20to%20the%20human%20genome&amp;author=B.%20Langmead&amp;author=C.%20Trapnell&amp;author=M.%20Pop&amp;author=S.L.%20Salzberg&amp;publication_year=2009&amp;journal=Genome%20Biol&amp;volume=10&amp;pages=R25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-3-r25" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-3-r25" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-3-r25"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19261174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huang</div> <div class="given-names">D.W.</div></div>, <div class="name string-name"><div class="surname">Sherman</div> <div class="given-names">B.T.</div></div>, <div class="name string-name"><div class="surname">Lempicki</div> <div class="given-names">R.A.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources</div>, <div class="source">Nat. Protoc</div> ., <div class="volume">4</div>, <div class="fpage">44</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20and%20integrative%20analysis%20of%20large%20gene%20lists%20using%20DAVID%20bioinformatics%20resources&amp;author=D.W.%20Huang&amp;author=B.T.%20Sherman&amp;author=R.A.%20Lempicki&amp;publication_year=2008&amp;journal=Nat.%20Protoc&amp;volume=4&amp;pages=44-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2008.211" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2008.211" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2008.211"> </span></div> </div></div></div></div></div><div content-id="dsx049-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huang</div> <div class="given-names">D.W.</div></div>, <div class="name string-name"><div class="surname">Sherman</div> <div class="given-names">B.T.</div></div>, <div class="name string-name"><div class="surname">Lempicki</div> <div class="given-names">R.A.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">37</div>, <div class="fpage">1</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bioinformatics%20enrichment%20tools%3A%20paths%20toward%20the%20comprehensive%20functional%20analysis%20of%20large%20gene%20lists&amp;author=D.W.%20Huang&amp;author=B.T.%20Sherman&amp;author=R.A.%20Lempicki&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res&amp;volume=37&amp;pages=1-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19033363" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schwanhäusser</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Busse</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">N.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Global quantification of mammalian gene expression control</div>, <div class="source">Nature</div> , <div class="volume">473</div>, <div class="fpage">337</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20quantification%20of%20mammalian%20gene%20expression%20control&amp;author=B.%20Schwanh%C3%A4usser&amp;author=D.%20Busse&amp;author=N.%20Li&amp;publication_year=2011&amp;journal=Nature&amp;volume=473&amp;pages=337-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10098" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10098" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10098"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21593866" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chu</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Kazana</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Bellanger</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Singh</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Tuite</div> <div class="given-names">M. F.</div></div>, <div class="name string-name"><div class="surname">von der Haar</div> <div class="given-names">T.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Translation elongation can control translation initiation on eukaryotic mRNAs</div>, <div class="source">Embo J</div> ., <div class="volume">33</div>, <div class="fpage">21</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translation%20elongation%20can%20control%20translation%20initiation%20on%20eukaryotic%20mRNAs&amp;author=D.%20Chu&amp;author=E.%20Kazana&amp;author=N.%20Bellanger&amp;author=T.%20Singh&amp;author=M.%20F.%20Tuite&amp;author=T.%20von%20der%20Haar&amp;publication_year=2014&amp;journal=Embo%20J&amp;volume=33&amp;pages=21-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/embj.201385651" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fembj.201385651" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fembj.201385651"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24357599" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jacques</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Dreyfus</div> <div class="given-names">M.</div></div></div> <div class="year">1990</div>, 
<div class="article-title">Translation initiation in <em>Escherichia coli</em>: old and new questions</div>, <div class="source">Mol. Microbiol</div> ., <div class="volume">4</div>, <div class="fpage">1063</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translation%20initiation%20in%20Escherichia%20coli%3A%20old%20and%20new%20questions&amp;author=N.%20Jacques&amp;author=M.%20Dreyfus&amp;publication_year=1990&amp;journal=Mol.%20Microbiol&amp;volume=4&amp;pages=1063-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.1990.4.issue-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.1990.4.issue-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.1990.4.issue-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1700254" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Purvis</div> <div class="given-names">I.J.</div></div>, <div class="name string-name"><div class="surname">Bettany</div> <div class="given-names">A.J.</div></div>, <div class="name string-name"><div class="surname">Santiago</div> <div class="given-names">T.C.</div></div></div>, et al.   <div class="year">1987</div>, 
<div class="article-title">The efficiency of folding of some proteins is increased by controlled rates of translation in vivo. A hypothesis</div>, <div class="source">J. Mol. Biol</div> ., <div class="volume">193</div>, <div class="fpage">413</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20efficiency%20of%20folding%20of%20some%20proteins%20is%20increased%20by%20controlled%20rates%20of%20translation%20in%20vivo.%20A%20hypothesis&amp;author=I.J.%20Purvis&amp;author=A.J.%20Bettany&amp;author=T.C.%20Santiago&amp;publication_year=1987&amp;journal=J.%20Mol.%20Biol&amp;volume=193&amp;pages=413-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(87)90230-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(87)90230-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(87)90230-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3298659" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Brunak</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Engelbrecht</div> <div class="given-names">J.</div></div></div> <div class="year">1996</div>, 
<div class="article-title">Protein structure and the sequential structure of mRNA: alpha-helix and beta-sheet signals at the nucleotide level</div>, <div class="source">Proteins</div> , <div class="volume">25</div>, <div class="fpage">237</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20structure%20and%20the%20sequential%20structure%20of%20mRNA%3A%20alpha-helix%20and%20beta-sheet%20signals%20at%20the%20nucleotide%20level&amp;author=S.%20Brunak&amp;author=J.%20Engelbrecht&amp;publication_year=1996&amp;journal=Proteins&amp;volume=25&amp;pages=237-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(SICI)1097-0134(199606)25:2&lt;237::AID-PROT9&gt;3.3.CO;2-Y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(SICI)1097-0134(199606)25:2&lt;237::AID-PROT9&gt;3.3.CO;2-Y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(SICI)1097-0134(199606)25:2&lt;237::AID-PROT9&gt;3.3.CO;2-Y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8811739" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Thanaraj</div> <div class="given-names">T.A.</div></div>, <div class="name string-name"><div class="surname">Argos</div> <div class="given-names">P.</div></div></div> <div class="year">1996</div>, 
<div class="article-title">Ribosome-mediated translational pause and protein domain organization</div>, <div class="source">Protein Sci.</div>  , <div class="volume">5</div>, <div class="fpage">1594</div>–<div class="lpage">612</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ribosome-mediated%20translational%20pause%20and%20protein%20domain%20organization&amp;author=T.A.%20Thanaraj&amp;author=P.%20Argos&amp;publication_year=1996&amp;journal=Protein%20Sci.&amp;volume=5&amp;pages=1594-612" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/pro.v5:8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fpro.v5:8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fpro.v5:8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8844849" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Thanaraj</div> <div class="given-names">T.A.</div></div>, <div class="name string-name"><div class="surname">Argos</div> <div class="given-names">P.</div></div></div> <div class="year">1996</div>, 
<div class="article-title">Protein secondary structural types are differentially coded on messenger RNA</div>, <div class="source">Protein Sci.</div>  ., <div class="volume">5</div>, <div class="fpage">1973</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protein%20secondary%20structural%20types%20are%20differentially%20coded%20on%20messenger%20RNA&amp;author=T.A.%20Thanaraj&amp;author=P.%20Argos&amp;publication_year=1996&amp;journal=Protein%20Sci.&amp;volume=5&amp;pages=1973-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/pro.v5:10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fpro.v5:10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fpro.v5:10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8897597" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Adzhubei</div> <div class="given-names">A.A.</div></div>, <div class="name string-name"><div class="surname">Adzhubei</div> <div class="given-names">I.A.</div></div>, <div class="name string-name"><div class="surname">Krasheninnikov</div> <div class="given-names">I.A.</div></div>, <div class="name string-name"><div class="surname">Neidle</div> <div class="given-names">S.</div></div></div> <div class="year">1996</div>, 
<div class="article-title">Non-random usage of “degenerate” codons is related to protein three-dimensional structure</div>, <div class="source">FEBS Lett</div> ., <div class="volume">399</div>, <div class="fpage">78</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Non-random%20usage%20of%20%E2%80%9Cdegenerate%E2%80%9D%20codons%20is%20related%20to%20protein%20three-dimensional%20structure&amp;author=A.A.%20Adzhubei&amp;author=I.A.%20Adzhubei&amp;author=I.A.%20Krasheninnikov&amp;author=S.%20Neidle&amp;publication_year=1996&amp;journal=FEBS%20Lett&amp;volume=399&amp;pages=78-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0014-5793(96)01287-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0014-5793(96)01287-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0014-5793(96)01287-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8980124" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Raveh</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Margaliot</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Sontag</div> <div class="given-names">E.D.</div></div>, <div class="name string-name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">A model for competition for ribosomes in the cell</div>, <div class="source">J. R Soc. Interface</div> , <div class="volume">13</div>, <div class="fpage">20151062</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20model%20for%20competition%20for%20ribosomes%20in%20the%20cell&amp;author=A.%20Raveh&amp;author=M.%20Margaliot&amp;author=E.D.%20Sontag&amp;author=T.%20Tuller&amp;publication_year=2016&amp;journal=J.%20R%20Soc.%20Interface&amp;volume=13&amp;pages=20151062" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rsif.2015.1062" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frsif.2015.1062" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frsif.2015.1062"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26962028" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ringquist</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Shinedling</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Barrick</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">1992</div>, 
<div class="article-title">Translation initiation in <em>Escherichia coli</em>: sequences within the ribosome-binding site</div>, <div class="source">Mol. Microbiol</div> ., <div class="volume">6</div>, <div class="fpage">1219</div>–<div class="lpage">29</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translation%20initiation%20in%20Escherichia%20coli%3A%20sequences%20within%20the%20ribosome-binding%20site&amp;author=S.%20Ringquist&amp;author=S.%20Shinedling&amp;author=D.%20Barrick&amp;publication_year=1992&amp;journal=Mol.%20Microbiol&amp;volume=6&amp;pages=1219-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.1992.6.issue-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.1992.6.issue-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.1992.6.issue-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1375310" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Bjerknes</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Jay</div> <div class="given-names">E.</div></div></div> <div class="year">1994</div>, 
<div class="article-title">Determination of the optimal aligned spacing between the Shine-Dalgarno sequence and the translation initiation codon of <em>Escherichia coli</em> mRNAs</div>, <div class="source">Nucl. Acids Res</div> ., <div class="volume">22</div>, <div class="fpage">4953</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Determination%20of%20the%20optimal%20aligned%20spacing%20between%20the%20Shine-Dalgarno%20sequence%20and%20the%20translation%20initiation%20codon%20of%20Escherichia%20coli%20mRNAs&amp;author=H.%20Chen&amp;author=M.%20Bjerknes&amp;author=R.%20Kumar&amp;author=E.%20Jay&amp;publication_year=1994&amp;journal=Nucl.%20Acids%20Res&amp;volume=22&amp;pages=4953-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/22.23.4953" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F22.23.4953" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F22.23.4953"> </span></div> </div></div></div></div></div><div content-id="dsx049-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ma</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Campbell</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Karlin</div> <div class="given-names">S.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Correlations between Shine-Dalgarno sequences and gene features such as predicted expression levels and operon structures</div>, <div class="source">J. Bacteriol</div> ., <div class="volume">184</div>, <div class="fpage">5733</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correlations%20between%20Shine-Dalgarno%20sequences%20and%20gene%20features%20such%20as%20predicted%20expression%20levels%20and%20operon%20structures&amp;author=J.%20Ma&amp;author=A.%20Campbell&amp;author=S.%20Karlin&amp;publication_year=2002&amp;journal=J.%20Bacteriol&amp;volume=184&amp;pages=5733-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.184.20.5733-5745.2002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.184.20.5733-5745.2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.184.20.5733-5745.2002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12270832" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hui</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">de Boer</div> <div class="given-names">H.A.</div></div></div> <div class="year">1987</div>, 
<div class="article-title">Specialized ribosome system: preferential translation of a single mRNA species by a subpopulation of mutated ribosomes in Escherichia coli</div>, <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> , <div class="volume">84</div>, <div class="fpage">4762</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Specialized%20ribosome%20system%3A%20preferential%20translation%20of%20a%20single%20mRNA%20species%20by%20a%20subpopulation%20of%20mutated%20ribosomes%20in%20Escherichia%20coli&amp;author=A.%20Hui&amp;author=H.A.%20de%20Boer&amp;publication_year=1987&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=84&amp;pages=4762-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.84.14.4762" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.84.14.4762" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.84.14.4762"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2440028" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">de Smit</div> <div class="given-names">M.H.</div></div>, <div class="name string-name"><div class="surname">van Duin</div> <div class="given-names">J.</div></div></div> <div class="year">1994</div>, 
<div class="article-title">Translational initiation on structured messengers: another role for the shine-dalgarno interaction</div>, <div class="source">J. Mol. Biol</div> ., <div class="volume">235</div>, <div class="fpage">173</div>–<div class="lpage">84</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translational%20initiation%20on%20structured%20messengers%3A%20another%20role%20for%20the%20shine-dalgarno%20interaction&amp;author=M.H.%20de%20Smit&amp;author=J.%20van%20Duin&amp;publication_year=1994&amp;journal=J.%20Mol.%20Biol&amp;volume=235&amp;pages=173-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80024-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80024-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80024-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8289239" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ingolia</div> <div class="given-names">N. .</div></div>, <div class="name string-name"><div class="surname">Ghaemmaghami</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Newman</div> <div class="given-names">J.R.S.</div></div>, <div class="name string-name"><div class="surname">Weissman</div> <div class="given-names">J.S.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Genome-wide analysis in vivo of translation with nucleotide resolution using ribosome profiling</div>, <div class="source">Science</div> , <div class="volume">324</div>, <div class="fpage">218</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20in%20vivo%20of%20translation%20with%20nucleotide%20resolution%20using%20ribosome%20profiling&amp;author=N.%20.%20Ingolia&amp;author=S.%20Ghaemmaghami&amp;author=J.R.S.%20Newman&amp;author=J.S.%20Weissman&amp;publication_year=2009&amp;journal=Science&amp;volume=324&amp;pages=218-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1168978" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1168978" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1168978"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19213877" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Carmi</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Vestsigian</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">An evolutionarily conserved mechanism for controlling the efficiency of protein translation</div>, <div class="source">Cell</div> , <div class="volume">141</div>, <div class="fpage">344</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20evolutionarily%20conserved%20mechanism%20for%20controlling%20the%20efficiency%20of%20protein%20translation&amp;author=T.%20Tuller&amp;author=A.%20Carmi&amp;author=K.%20Vestsigian&amp;publication_year=2010&amp;journal=Cell&amp;volume=141&amp;pages=344-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2010.03.031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2010.03.031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2010.03.031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20403328" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hardesty</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Tsalkova</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Kramer</div> <div class="given-names">G.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Co-translational folding</div>, <div class="source">Curr. Opin. Struct. Biol</div> ., <div class="volume">9</div>, <div class="fpage">111</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Co-translational%20folding&amp;author=B.%20Hardesty&amp;author=T.%20Tsalkova&amp;author=G.%20Kramer&amp;publication_year=1999&amp;journal=Curr.%20Opin.%20Struct.%20Biol&amp;volume=9&amp;pages=111-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0959-440X(99)80014-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0959-440X(99)80014-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0959-440X(99)80014-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10047581" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wolin</div> <div class="given-names">S.L.</div></div>, <div class="name string-name"><div class="surname">Walter</div> <div class="given-names">P.</div></div></div> <div class="year">1993</div>, 
<div class="article-title">Discrete nascent chain lengths are required for the insertion of presecretory proteins into microsomal membranes</div>, <div class="source">J. Cell Biol</div> ., <div class="volume">121</div>, <div class="fpage">1211</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discrete%20nascent%20chain%20lengths%20are%20required%20for%20the%20insertion%20of%20presecretory%20proteins%20into%20microsomal%20membranes&amp;author=S.L.%20Wolin&amp;author=P.%20Walter&amp;publication_year=1993&amp;journal=J.%20Cell%20Biol&amp;volume=121&amp;pages=1211-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1083/jcb.121.6.1211" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1083%2Fjcb.121.6.1211" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1083%2Fjcb.121.6.1211"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8389768" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Alberts</div> <div class="given-names">B.</div></div></div> <div class="year">2002</div>, <div class="source">Molecular Biology of the Cell</div> , <div class="edition">4th ed</div>. 
<div class="publisher-name">Garland Science</div>, 
<div class="publisher-loc">New York, NY</div>.</div></div></div></div><div content-id="dsx049-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Murakami</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Nakatogawa</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Ito</div> <div class="given-names">K.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Translation arrest of SecM is essential for the basal and regulated expression of SecA</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">101</div>, <div class="fpage">12330</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translation%20arrest%20of%20SecM%20is%20essential%20for%20the%20basal%20and%20regulated%20expression%20of%20SecA&amp;author=A.%20Murakami&amp;author=H.%20Nakatogawa&amp;author=K.%20Ito&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=12330-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0404907101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0404907101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0404907101"> </span></div> </div></div></div></div></div><div content-id="dsx049-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wilcoxon</div> <div class="given-names">F.</div></div></div> <div class="year">1945</div>, 
<div class="article-title">Individual comparisons by ranking methods</div>, <div class="source">Biom. Bull</div> ., <div class="volume">1</div>, <div class="fpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Individual%20comparisons%20by%20ranking%20methods&amp;author=F.%20Wilcoxon&amp;publication_year=1945&amp;journal=Biom.%20Bull&amp;volume=1&amp;pages=80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/3001968" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F3001968" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F3001968"> </span></div> </div></div></div></div></div><div content-id="dsx049-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Uemura</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Aitken</div> <div class="given-names">C.E.</div></div>, <div class="name string-name"><div class="surname">Korlach</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Flusberg</div> <div class="given-names">B.A.</div></div>, <div class="name string-name"><div class="surname">Turner</div> <div class="given-names">S.W.</div></div>, <div class="name string-name"><div class="surname">Puglisi</div> <div class="given-names">J.D.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Real-time tRNA transit on single translating ribosomes at codon resolution</div>, <div class="source">Nature</div> , <div class="volume">464</div>, <div class="fpage">1012</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Real-time%20tRNA%20transit%20on%20single%20translating%20ribosomes%20at%20codon%20resolution&amp;author=S.%20Uemura&amp;author=C.E.%20Aitken&amp;author=J.%20Korlach&amp;author=B.A.%20Flusberg&amp;author=S.W.%20Turner&amp;author=J.D.%20Puglisi&amp;publication_year=2010&amp;journal=Nature&amp;volume=464&amp;pages=1012-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08925" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08925" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08925"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20393556" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shaham</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Most associations between transcript features and gene expression are monotonic</div>, <div class="source">Mol. Biosyst</div> , <div class="volume">10</div>, <div class="fpage">1426</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Most%20associations%20between%20transcript%20features%20and%20gene%20expression%20are%20monotonic&amp;author=G.%20Shaham&amp;author=T.%20Tuller&amp;publication_year=2014&amp;journal=Mol.%20Biosyst&amp;volume=10&amp;pages=1426-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1039/C3MB70617F" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1039%2FC3MB70617F" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1039%2FC3MB70617F"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24675795" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tian</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Salis</div> <div class="given-names">H.M.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">A predictive biophysical model of translational coupling to coordinate and control protein expression in bacterial operons</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">7137</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20predictive%20biophysical%20model%20of%20translational%20coupling%20to%20coordinate%20and%20control%20protein%20expression%20in%20bacterial%20operons&amp;author=T.%20Tian&amp;author=H.M.%20Salis&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=7137-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv635" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv635" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv635"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26117546" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx049-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx049-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jackson</div> <div class="given-names">R.J.</div></div>, <div class="name string-name"><div class="surname">Hellen</div> <div class="given-names">C.U.T.</div></div>, <div class="name string-name"><div class="surname">Pestova</div> <div class="given-names">T.V.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">The mechanism of eukaryotic translation initiation and principles of its regulation</div>, <div class="source">Nat. Rev. Mol. Cell Biol</div> ., <div class="volume">11</div>, <div class="fpage">113</div>–<div class="lpage">27</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20mechanism%20of%20eukaryotic%20translation%20initiation%20and%20principles%20of%20its%20regulation&amp;author=R.J.%20Jackson&amp;author=C.U.T.%20Hellen&amp;author=T.V.%20Pestova&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Mol.%20Cell%20Biol&amp;volume=11&amp;pages=113-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrm2838" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrm2838" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrm2838"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20094052" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4638432&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx049/2/dsx049_suppl_data.zip?Expires=2147483647&amp;Signature=K-XxDsS-lclmJqaifgOD8cqmR8wuXpMiBAd~PLg8rmKVSkcu4VLkYLSfj9Ec7G1iFsozccEB12Zm6NMd-blJmaU~mfKzJJIBhOfD~BkeqoHWZXPqsiCbqz~nIceinOSshRIKx-MORqxUDdLkVEC7VwMwrc1aaigUPWIFkNQS9S6dHEqUGECnWpTEzDEW9SfkjjpvyxMMBBAumGj1p9snNuotWhPaqULXWTeTH-IBLbB5UILXZKBQ71hQjREmQZ~Ci03~4uCB46laWmlEEx7nL7bu3MvL72lMnXPZycil78Z8MWW3pDhIL-5DTBvrgZJtwi1yoknXPR4rmRVVwv4etQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">776</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx049"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4638432"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f2%2f195%2f4638432">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F2%2F195%2F4638432" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4638432'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome scale analysis of Escherichia coli with a comprehensive prokaryotic sequence-based biophysical model of translation initiation and elongation" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx049");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f195%2f4638432">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

